<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/03/31/292672.atom" />
<meta name="HW.pisa" content="biorxiv;292672v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Receptor binding and proteolysis do not induce large conformational changes in the SARS-CoV spike" />
<meta name="DC.Identifier" content="10.1101/292672" />
<meta name="DC.Date" content="2018-03-31" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Severe acute respiratory syndrome coronavirus (SARS-CoV) emerged in 2002 as a highly transmissible pathogenic human betacoronavirus. The viral spike glycoprotein (S) utilizes angiotensin-converting enzyme 2 (ACE2) as a host protein receptor and mediates fusion of the viral and host membranes, making S essential to viral entry into host cells and host species tropism. As SARS-CoV enters host cells, the viral S undergoes two proteolytic cleavages at S1/S2 and S2’ sites necessary for efficient membrane fusion. Here, we present a cryo-EM analysis of the trimeric SARS-CoV S interactions with ACE2 and of the trypsin-cleaved S. Surprisingly, neither binding to ACE2 nor cleavage by trypsin at the S1/S2 cleavage site impart large conformational changes within S or expose the secondary cleavage site, S2’. These observations suggest that S2’ cleavage does not occur in the S prefusion conformation and that additional triggers may be required." />
<meta name="DC.Contributor" content="Robert N. Kirchdoerfer" />
<meta name="DC.Contributor" content="Nianshuang Wang" />
<meta name="DC.Contributor" content="Jesper Pallesen" />
<meta name="DC.Contributor" content="Daniel Wrapp" />
<meta name="DC.Contributor" content="Hannah L. Turner" />
<meta name="DC.Contributor" content="Christopher A. Cottrell" />
<meta name="DC.Contributor" content="Kizzmekia S. Corbett" />
<meta name="DC.Contributor" content="Barney S. Graham" />
<meta name="DC.Contributor" content="Jason S. McLellan" />
<meta name="DC.Contributor" content="Andrew B. Ward" />
<meta name="article:published_time" content="2018-03-31" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Receptor binding and proteolysis do not induce large conformational changes in the SARS-CoV spike" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Severe acute respiratory syndrome coronavirus (SARS-CoV) emerged in 2002 as a highly transmissible pathogenic human betacoronavirus. The viral spike glycoprotein (S) utilizes angiotensin-converting enzyme 2 (ACE2) as a host protein receptor and mediates fusion of the viral and host membranes, making S essential to viral entry into host cells and host species tropism. As SARS-CoV enters host cells, the viral S undergoes two proteolytic cleavages at S1/S2 and S2’ sites necessary for efficient membrane fusion. Here, we present a cryo-EM analysis of the trimeric SARS-CoV S interactions with ACE2 and of the trypsin-cleaved S. Surprisingly, neither binding to ACE2 nor cleavage by trypsin at the S1/S2 cleavage site impart large conformational changes within S or expose the secondary cleavage site, S2’. These observations suggest that S2’ cleavage does not occur in the S prefusion conformation and that additional triggers may be required.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;292672v1" />
<meta name="citation_id" content="292672v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/292672v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/292672v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/292672v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672.full.pdf" />
<meta name="citation_doi" content="10.1101/292672" />
<meta name="citation_num_pages" content="29" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="292672" />
<meta name="citation_author" content="Robert N. Kirchdoerfer" />
<meta name="citation_author_institution" content="Department of Integrative Structural and Computational Biology, The Scripps Research Institute" />
<meta name="citation_author" content="Nianshuang Wang" />
<meta name="citation_author_institution" content="Department of Biochemistry and Cellular Biology, Geisel School of Medicine at Dartmouth, Hanover" />
<meta name="citation_author" content="Jesper Pallesen" />
<meta name="citation_author_institution" content="Department of Integrative Structural and Computational Biology, The Scripps Research Institute" />
<meta name="citation_author" content="Daniel Wrapp" />
<meta name="citation_author_institution" content="Department of Biochemistry and Cellular Biology, Geisel School of Medicine at Dartmouth, Hanover" />
<meta name="citation_author" content="Hannah L. Turner" />
<meta name="citation_author_institution" content="Department of Integrative Structural and Computational Biology, The Scripps Research Institute" />
<meta name="citation_author" content="Christopher A. Cottrell" />
<meta name="citation_author_institution" content="Department of Integrative Structural and Computational Biology, The Scripps Research Institute" />
<meta name="citation_author" content="Kizzmekia S. Corbett" />
<meta name="citation_author_institution" content="Vaccine Research Center, National Institute of Allergy and Infectious Diseases" />
<meta name="citation_author" content="Barney S. Graham" />
<meta name="citation_author_institution" content="Vaccine Research Center, National Institute of Allergy and Infectious Diseases" />
<meta name="citation_author" content="Jason S. McLellan" />
<meta name="citation_author_institution" content="Department of Biochemistry and Cellular Biology, Geisel School of Medicine at Dartmouth, Hanover" />
<meta name="citation_author" content="Andrew B. Ward" />
<meta name="citation_author_institution" content="Department of Integrative Structural and Computational Biology, The Scripps Research Institute" />
<meta name="citation_author_email" content="andrew@scripps.edu" />
<meta name="citation_reference" content="de Wit, E., van Doremalen, N., Falzarano, D. &amp; Munster, V. J. SARS and MERS: recent insights into emerging coronaviruses. Nature reviews. Microbiology 14, 523–534, doi:10.1038/nrmicro.2016.81 (2016)." />
<meta name="citation_reference" content="Menachery, V. D. et al. SARS-like WIV1-CoV poised for human emergence. Proceedings of the National Academy of Sciences of the United States of America 113, 3048–3053, doi:10.1073/pnas.1517719113 (2016)." />
<meta name="citation_reference" content="Belouzard, S., Millet, J. K., Licitra, B. N. &amp; Whittaker, G. R. Mechanisms of coronavirus cell entry mediated by the viral spike protein. Viruses 4, 1011–1033, doi:10.3390/v4061011 (2012)." />
<meta name="citation_reference" content="Matsuyama, S. &amp;, Taguchi F. Two-step conformational changes in a coronavirus envelope glycoprotein mediated by receptor binding and proteolysis. Journal of virology 83, 11133–11141, doi:10.1128/jvi.00959-09 (2009)." />
<meta name="citation_reference" content="Park, J. E., Cruz, D. J. &amp; Shin, H. J. Receptor-bound porcine epidemic diarrhea virus spike protein cleaved by trypsin induces membrane fusion. Archives of virology 156, 1749–1756, doi:10.1007/s00705-011-1044-6 (2011)." />
<meta name="citation_reference" content="Park, J. E. et al. Proteolytic processing of Middle East respiratory syndrome coronavirus spikes expands virus tropism. Proceedings of the National Academy of Sciences of the United States of America 113, 12262–12267, doi:10.1073/pnas.1608147113 (2016)." />
<meta name="citation_reference" content="Simmons, G. et al. Inhibitors of cathepsin L prevent severe acute respiratory syndrome coronavirus entry. Proceedings of the National Academy of Sciences of the United States of America 102, 11876–11881, doi:10.1073/pnas.0505577102 (2005)." />
<meta name="citation_reference" content="Bosch, B. J., van der Zee, R., de Haan, C. A. &amp; Rottier, P. J. The coronavirus spike protein is a class I virus fusion protein: structural and functional characterization of the fusion core complex. Journal of virology 77, 8801–8811 (2003)." />
<meta name="citation_reference" content="Bullough, P. A., Hughson, F. M., Skehel, J. J. &amp; Wiley, D. C. Structure of influenza haemagglutinin at the pH of membrane fusion. Nature 371, 37–43, doi:10.1038/371037a0 (1994)." />
<meta name="citation_reference" content="Walls, A. C. et al. Tectonic conformational changes of a coronavirus spike glycoprotein promote membrane fusion. Proceedings of the National Academy of Sciences of the United States of America 114, 11157–11162, doi:10.1073/pnas.1708727114 (2017)." />
<meta name="citation_reference" content="Zhu, M. SARS Immunity and Vaccination. Cellular &amp; molecular immunology 1, 193–198 (2004)." />
<meta name="citation_reference" content="Graham, R. L. &amp; Baric, R. S. Recombination, reservoirs, and the modular spike: mechanisms of coronavirus cross-species transmission. Journal of virology 84, 3134–3146, doi:10.1128/jvi.01394-09 (2010)." />
<meta name="citation_reference" content="Li, W. et al. Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus. Nature 426, 450–454, doi:10.1038/nature02145 (2003)." />
<meta name="citation_reference" content="Wong, S. K., Li, W., Moore, M. J., Choe, H. &amp; Farzan, M. A 193-amino acid fragment of the SARS coronavirus S protein efficiently binds angiotensin-converting enzyme 2. The Journal of biological chemistry 279, 3197–3201, doi:10.1074/jbc.C300520200 (2004)." />
<meta name="citation_reference" content="Gui, M. et al. Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding. Cell research 27, 119–129, doi:10.1038/cr.2016.152 (2017)." />
<meta name="citation_reference" content="Kirchdoerfer, R. N. et al. Pre-fusion structure of a human coronavirus spike protein. Nature 531, 118–121, doi:10.1038/nature17200 (2016)." />
<meta name="citation_reference" content="Pallesen, J. et al. Immunogenicity and structures of a rationally designed prefusion MERS-CoV spike antigen. Proceedings of the National Academy of Sciences of the United States of America 114, E7348–e7357, doi:10.1073/pnas.1707304114 (2017)." />
<meta name="citation_reference" content="Shang, J. et al. Cryo-EM structure of porcine delta coronavirus spike protein in the pre-fusion state. Journal of virology, doi:10.1128/jvi.01556-17 (2017)." />
<meta name="citation_reference" content="Walls, A. C. et al. Cryo-electron microscopy structure of a coronavirus spike glycoprotein trimer. Nature 531, 114–117, doi:10.1038/nature16988 (2016)." />
<meta name="citation_reference" content="Walls, A. C. et al. Glycan shield and epitope masking of a coronavirus spike protein observed by cryo-electron microscopy. Nature structural &amp; molecular biology 23, 899–905, doi:10.1038/nsmb.3293 (2016)." />
<meta name="citation_reference" content="Xiong, X. et al. Glycan shield and fusion activation of a deltacoronavirus spike glycoprotein fine-tuned for enteric infections. Journal of virology, doi:10.1128/jvi.01628-17 (2017)." />
<meta name="citation_reference" content="Yuan, Y. et al. Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains. Nature communications 8, 15092, doi:10.1038/ncomms15092 (2017)." />
<meta name="citation_reference" content="Li, F., Li, W., Farzan, M. &amp; Harrison, S. C. Structure of SARS coronavirus spike receptor-binding domain complexed with receptor. Science (New York, N.Y.) 309, 1864–1868, doi:10.1126/science.1116480 (2005)." />
<meta name="citation_reference" content="Lai, A. L., Millet, J. K., Daniel, S., Freed, J. H. &amp; Whittaker, G. R. The SARS-CoV Fusion Peptide Forms an Extended Bipartite Fusion Platform that Perturbs Membrane Order in a Calcium-Dependent Manner. Journal of molecular biology, doi:10.1016/j.jmb.2017.10.017 (2017)." />
<meta name="citation_reference" content="Fukushi, S. et al. Vesicular stomatitis virus pseudotyped with severe acute respiratory syndrome coronavirus spike protein. The Journal of general virology 86, 2269–2274, doi:10.1099/vir.0.80955-0 (2005)." />
<meta name="citation_reference" content="Miura, H. S., Nakagaki, K. &amp; Taguchi, F. N-terminal domain of the murine coronavirus receptor CEACAM1 is responsible for fusogenic activation and conformational changes of the spike protein. Journal of virology 78, 216–223 (2004)." />
<meta name="citation_reference" content="Belouzard, S., Chu, V. C. &amp; Whittaker, G. R. Activation of the SARS coronavirus spike protein via sequential proteolytic cleavage at two distinct sites. Proceedings of the National Academy of Sciences of the United States of America 106, 5871–5876, doi:10.1073/pnas.0809524106 (2009)." />
<meta name="citation_reference" content="Matsuyama, S., Ujike, M., Morikawa, S., Tashiro, M. &amp; Taguchi, F. Protease-mediated enhancement of severe acute respiratory syndrome coronavirus infection. Proceedings of the National Academy of Sciences of the United States of America 102, 12543–12547, doi:10.1073/pnas.0503203102 (2005)." />
<meta name="citation_reference" content="Millet, J. K. &amp; Whittaker, G. R. Physiological and molecular triggers for SARS-CoV membrane fusion and entry into host cells. Virology 517, 3–8, doi:10.1016/j.virol.2017.12.015 (2018)." />
<meta name="citation_reference" content="Millet, J. K. &amp; Whittaker, G. R. Host cell proteases: Critical determinants of coronavirus tropism and pathogenesis. Virus research 202, 120–134, doi:10.1016/j.virusres.2014.11.021 (2015)." />
<meta name="citation_reference" content="Hu, B. et al. Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus. PLoS pathogens 13, e1006698, doi:10.1371/journal.ppat.1006698 (2017)." />
<meta name="citation_reference" content="Suloway, C. et al. Automated molecular microscopy: the new Leginon system. Journal of structural biology 151, 41–60, doi:10.1016/j.jsb.2005.03.010 (2005)." />
<meta name="citation_reference" content="Zheng, S. Q. et al. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Nature methods 14, 331–332, doi:10.1038/nmeth.4193 (2017)." />
<meta name="citation_reference" content="Lander, G. C. et al. Appion: an integrated, database-driven pipeline to facilitate EM image processing. Journal of structural biology 166, 95–102 (2009)." />
<meta name="citation_reference" content="Voss, N. R., Yoshioka, C. K., Radermacher, M., Potter, C. S. &amp; Carragher, B. DoG Picker and TiltPicker: software tools to facilitate particle selection in single particle electron microscopy. Journal of structural biology 166, 205–213 (2009)." />
<meta name="citation_reference" content="Berndsen, Z., Bowman, C., Jang, H. &amp; Ward, A. B. EMHP: an accurate automated hole masking algorithm for single-particle cryo-EM image processing. Bioinformatics (Oxford, England) 33, 3824–3826, doi:10.1093/bioinformatics/btx500 (2017)." />
<meta name="citation_reference" content="Zhang, K. Gctf: Real-time CTF determination and correction. Journal of structural biology 193, 1–12, doi:10.1016/j.jsb.2015.11.003 (2016)." />
<meta name="citation_reference" content="Kimanius, D., Forsberg, B. O., Scheres, S. H. &amp; Lindahl, E. Accelerated cryo-EM structure determination with parallelisation using GPUs in RELION-2. eLife 5, doi:10.7554/eLife.18722 (2016)." />
<meta name="citation_reference" content="Emsley, P. &amp; Cowtan, K. Coot: model-building tools for molecular graphics. Acta crystallographica. Section D, Biological crystallography 60, 2126–2132, doi:10.1107/s0907444904019158 (2004)." />
<meta name="citation_reference" content="DiMaio, F. et al. Atomic-accuracy models from 4.5-A cryo-electron microscopy data with density-guided iterative local refinement. Nature methods 12, 361–365, doi:10.1038/nmeth.3286 (2015)." />
<meta name="citation_reference" content="Adams, P. D. et al. PHENIX: a comprehensive Python-based system for macromolecular structure solution. Acta crystallographica. Section D, Biological crystallography 66, 213–221, doi:10.1107/s0907444909052925 (2010)." />
<meta name="twitter:title" content="Receptor binding and proteolysis do not induce large conformational changes in the SARS-CoV spike" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Severe acute respiratory syndrome coronavirus (SARS-CoV) emerged in 2002 as a highly transmissible pathogenic human betacoronavirus. The viral spike glycoprotein (S) utilizes angiotensin-converting enzyme 2 (ACE2) as a host protein receptor and mediates fusion of the viral and host membranes, making S essential to viral entry into host cells and host species tropism. As SARS-CoV enters host cells, the viral S undergoes two proteolytic cleavages at S1/S2 and S2’ sites necessary for efficient membrane fusion. Here, we present a cryo-EM analysis of the trimeric SARS-CoV S interactions with ACE2 and of the trypsin-cleaved S. Surprisingly, neither binding to ACE2 nor cleavage by trypsin at the S1/S2 cleavage site impart large conformational changes within S or expose the secondary cleavage site, S2’. These observations suggest that S2’ cleavage does not occur in the S prefusion conformation and that additional triggers may be required." />
<meta name="og-title" property="og:title" content="Receptor binding and proteolysis do not induce large conformational changes in the SARS-CoV spike" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/292672v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Severe acute respiratory syndrome coronavirus (SARS-CoV) emerged in 2002 as a highly transmissible pathogenic human betacoronavirus. The viral spike glycoprotein (S) utilizes angiotensin-converting enzyme 2 (ACE2) as a host protein receptor and mediates fusion of the viral and host membranes, making S essential to viral entry into host cells and host species tropism. As SARS-CoV enters host cells, the viral S undergoes two proteolytic cleavages at S1/S2 and S2’ sites necessary for efficient membrane fusion. Here, we present a cryo-EM analysis of the trimeric SARS-CoV S interactions with ACE2 and of the trypsin-cleaved S. Surprisingly, neither binding to ACE2 nor cleavage by trypsin at the S1/S2 cleavage site impart large conformational changes within S or expose the secondary cleavage site, S2’. These observations suggest that S2’ cleavage does not occur in the S prefusion conformation and that additional triggers may be required." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-03-31" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/292672v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/292672v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/292672v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/292672v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/90133" />
    <title>Receptor binding and proteolysis do not induce large conformational changes in the SARS-CoV spike | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"G8TW3iX5_L49pLgOEP1Kh6LdL6K76b9X4NaHDGTfkgM","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"90133","apath":"\/biorxiv\/early\/2018\/03\/31\/292672.atom","pisa":"biorxiv;292672v1","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;292672\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;292672\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;292672v1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;292672\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;292672\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/292672v1","title":"Receptor binding and proteolysis do not induce large conformational changes in the SARS-CoV spike","identifier":"node\/90133"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lIZT9qmlPUEvc9MTRUQcKig+FnvSIKQMaKiYEskjsAh0UoeT8NBQtaCRTEcgzEGQqNTYqPZaSDDrlKrYuIutWc5OfgA+Gggcqdr\\\\\\\/M23isTW3++oOAlNIFhKDZQSqDxG+SiXCkvPtZQutAttyi7tEZdAps8fUjR1p4\\\\\\\/rKcxabX3lzAAUlC6qCbMZ3KYRA62JSPF6bvMrjO4NNIhw0NTkmMk+fL0nvPIgCdWH8LQuTCWNJ8Jh6Ij22gMNvgT\\\\\\\/PyTJ6vlOVMZDtvGPPPXUehcw\\\\\\\/cxQ5Q7rU0RU9MRlYO5sQElXtZOiliqyI3Oyu7UM9xT+5j5RQVydEPYZ78uTEgTSf1NbZqODScPVt3bAKa6QpaEUT41m\\\\\\\/u4U36ANPeJV9t\\\\\\\/2Hi5sGCvUeYKTYOdVPpkHCFcx\\\\\\\/MLDzNnoCs6Kuo\\\\\\\/VgQ7q+2m401RqZiGixDeH1D76GsUu6rF3UpvWjQ0t3ZVDYiqnsejVrOrbwzE2SG1uJ3hVcno1t5rx8uhvBA5ujEkTlQB7cq2VQo\\\\\\\/4VZNB01Tuux7kyuIVp0XVUVsayQMGT6JBc\\\\\\\/n+cFBcOjEwzPUQNcvWREYfIraD9kfTEcKyje29s1vn3yl2BG1h+0ABMXYgWlyKN\\\\\\\/o8v2jLbb8gRhfMf+vZa8S8cLzVxZXFzT2+auNoBIE8\\\\\\\/lp94\\\\\\\/4CbADRriWEAxt8BtnueZ9rl3arEC2mb19OVSgO7xEquYeNnLxCL7wmn78kCB73l9NcnTfIqJpy\\\\\\\/yuE+TqWrM8Lhg4\\\\\\\/FvQYnRgrPVtL1ZKebJwCxunmaQTM64X2yOI+mS0dCmtTeoNx9lqr43p+Ip7wGtBH5ikN9iUex8R5BUbPVBFPohjSJWOVJBPzbBkXB0k2JMqPirwsjaDWaReNAqo4t1xDhTD1\\\\\\\/G5dQzmHwO9uXFfSzF+iPVl5NHwmYfwO6vXC5tFF08v13cs0ebM7XtIyJQF2lujEOBADeNuXNO\\\\\\\/VYdsPnUvopHg56IFzNrFEwYVvGl6z5Gb4y+qf1RLqhuXmz4QmqPtID+D8zHBtcLCNWQl5GLIM49uLicM6N8eo6Djgkmwz7e\\\\\\\/TsjAeciDtpI\\\\\\\/6Y4XDTCIhfeZtKg6F1LyeS9MJ2FiSYyCmTRXHNDL6RzvJrH9yDTHuOA7FNPQ85lGm57B8mUo5vPlpUcurRBY3yipiwjVyR5pH2CIdd6qEVClIXDJFwrNAZszzVmYdpb2A37j22Ur0ew\\\\\\\/GcCe1mz2BFBfBWE\\\\\\\/o3Cqy58BHonxa6H4XVndiP3DkEmMYdzYQAppEd3jgKZJqXm8hxVc0ExlumhdHwg8B7V\\\\\\\/VopAmPN8LcuZdyByPDFv0RQN8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022nbNawRk7r\\\\\\\/OMDshs2+9VxQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022e9c1bb7c65e95c68bde6b69d4ce19703c9dbce6705451fbbac99c0915c8892f5\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/292672v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-90133 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/292672v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-lh7fl3ZnpNBGGx3l48r3gcRaockP5FRkYgPmgMSoDnc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="90133" id="node90133" data-pisa="biorxiv;292672v1" data-pisa-master="biorxiv;292672" data-apath="/biorxiv/early/2018/03/31/292672.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Receptor binding and proteolysis do not induce large conformational changes in the SARS-CoV spike</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Robert N.</span> <span class="nlm-surname">Kirchdoerfer</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Nianshuang</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jesper</span> <span class="nlm-surname">Pallesen</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Wrapp</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Hannah L.</span> <span class="nlm-surname">Turner</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Christopher A.</span> <span class="nlm-surname">Cottrell</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Kizzmekia S.</span> <span class="nlm-surname">Corbett</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Barney S.</span> <span class="nlm-surname">Graham</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Jason S.</span> <span class="nlm-surname">McLellan</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Andrew B.</span> <span class="nlm-surname">Ward</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/292672 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node90133" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Robert N. Kirchdoerfer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Integrative Structural and Computational Biology, The Scripps Research Institute</span>, La Jolla, CA 92037</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BN.%2BKirchdoerfer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kirchdoerfer%20RN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BN.%2BKirchdoerfer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nianshuang Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Biochemistry and Cellular Biology, Geisel School of Medicine at Dartmouth, Hanover</span>, NH 03755</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nianshuang%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANianshuang%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jesper Pallesen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Integrative Structural and Computational Biology, The Scripps Research Institute</span>, La Jolla, CA 92037</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesper%2BPallesen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pallesen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesper%2BPallesen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel Wrapp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Biochemistry and Cellular Biology, Geisel School of Medicine at Dartmouth, Hanover</span>, NH 03755</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BWrapp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wrapp%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BWrapp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hannah L. Turner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Integrative Structural and Computational Biology, The Scripps Research Institute</span>, La Jolla, CA 92037</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hannah%2BL.%2BTurner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Turner%20HL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHannah%2BL.%2BTurner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Christopher A. Cottrell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Integrative Structural and Computational Biology, The Scripps Research Institute</span>, La Jolla, CA 92037</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BA.%2BCottrell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cottrell%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BA.%2BCottrell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Kizzmekia S. Corbett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Vaccine Research Center, National Institute of Allergy and Infectious Diseases</span>, Bethesda, MD 20814</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kizzmekia%2BS.%2BCorbett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Corbett%20KS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKizzmekia%2BS.%2BCorbett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Barney S. Graham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Vaccine Research Center, National Institute of Allergy and Infectious Diseases</span>, Bethesda, MD 20814</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Barney%2BS.%2BGraham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graham%20BS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABarney%2BS.%2BGraham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jason S. McLellan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Biochemistry and Cellular Biology, Geisel School of Medicine at Dartmouth, Hanover</span>, NH 03755</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BS.%2BMcLellan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McLellan%20JS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BS.%2BMcLellan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Andrew B. Ward </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Integrative Structural and Computational Biology, The Scripps Research Institute</span>, La Jolla, CA 92037</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BB.%2BWard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ward%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BB.%2BWard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:andrew@scripps.edu" class="" data-icon-position="" data-hide-link-title="0">andrew@scripps.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/292672v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:90133" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:90133/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/292672v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:90133" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:90133/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/292672v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:90133" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:90133/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/292672v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:90133" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:90133/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/292672v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:90133" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:90133/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/292672v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:90133" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:90133/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Severe acute respiratory syndrome coronavirus (SARS-CoV) emerged in 2002 as a highly transmissible pathogenic human betacoronavirus. The viral spike glycoprotein (S) utilizes angiotensin-converting enzyme 2 (ACE2) as a host protein receptor and mediates fusion of the viral and host membranes, making S essential to viral entry into host cells and host species tropism. As SARS-CoV enters host cells, the viral S undergoes two proteolytic cleavages at S1/S2 and S2’ sites necessary for efficient membrane fusion. Here, we present a cryo-EM analysis of the trimeric SARS-CoV S interactions with ACE2 and of the trypsin-cleaved S. Surprisingly, neither binding to ACE2 nor cleavage by trypsin at the S1/S2 cleavage site impart large conformational changes within S or expose the secondary cleavage site, S2’. These observations suggest that S2’ cleavage does not occur in the S prefusion conformation and that additional triggers may be required.</p></div><p id="p-4">Severe acute respiratory syndrome coronavirus (SARS-CoV) emerged in humans in 2002 and rapidly spread globally causing 8,096 cases and 774 associated deaths in 26 countries through July 2003<sup>1</sup>. SARS-CoV reappeared in a second smaller outbreak in 2004, but has since disappeared from human circulation. However, closely related coronaviruses, such as WIV1, currently circulate in bat reservoirs and are capable of utilizing human receptors to enter cells<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>. The more recent emergence of Middle East respiratory syndrome coronavirus (MERS-CoV)<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup> and the likelihood of future zoonotic transmission of novel coronaviruses to humans from animal reservoirs make understanding the coronavirus infection cycle of great importance to human health.</p><p id="p-5">Coronaviruses are enveloped viruses possessing large, trimeric spike glycoproteins (S) required for the recognition of host receptors for many coronaviruses as well as the fusion of viral and host cell membranes for viral entry into cells<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup>. During viral egress from infected host cells, some coronavirus S proteins are cleaved into S1 and S2 subunits. The S1 subunit is responsible for host-receptor binding while the S2 subunit contains the membrane-fusion machinery. During viral entry, the S1 subunit binds host receptors in an interaction thought to expose a secondary cleavage site within S2 (S2’) adjacent to the fusion peptide for cleavage by host proteases<sup><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>–<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. This S2’ proteolysis has been hypothesized to facilitate insertion of the fusion peptide into host membranes after the first heptad repeat region (HR1) of the S2 subunit rearranges into an extended α-helix<sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>–<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>. Subsequent conformational changes in the second heptad repeat region (HR2) of S2 form a six-helix bundle with HR1, fusing the viral and host membranes and allowing for release of the viral genome into host cells. Coronavirus S is also the target of neutralizing antibodies<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup>, making an understanding of S structure and conformational transitions pertinent for investigating S antigenic surfaces and designing vaccines.</p><p id="p-6">The SARS-CoV S1 subunit is composed of two distinct domains: an N-terminal domain (S1 NTD) and a receptor-binding domain (S1 RBD) also referred to as the S1 CTD or domain B. Each of these domains have been implicated in binding to host receptors, depending on the coronavirus in question. However, most coronaviruses are not known to utilize both the S1 NTD and S1 RBD for viral entry<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>. SARS-CoV makes use of its S1 RBD to bind to the human angiotensin-converting enzyme 2 (ACE2) as its host receptor<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>.</p><p id="p-7">Recent examination using cryo-electron microscopy (cryo-EM) has illuminated the prefusion structures of coronavirus spikes<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>–<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>. Initial examination of HCoV-HKU1 S showed that the receptor-binding site on the S1 RBD was occluded when the RBD was in a ‘down’ conformation and it was hypothesized that conformational changes were required to access this site<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>. Subsequent studies of the highly pathogenic human coronavirus S proteins of SARS-CoV<sup><a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a></sup> and MERS-CoV<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>,<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a></sup> showed that these viral S1 RBD do indeed sample an ‘up’ conformation where the receptor-binding site is accessible. These structural studies also located the positions of the S1/S2 and S2’ cleavage sites on the prefusion spike. The S1/S2 site lies within a surface exposed loop in the second subdomain of S1<sup><a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a></sup>. However, the S2’ site lies closer to base of the spike and though this region is located on the surface of the spike, cleavage at this site is prevented by surrounding protein elements<sup><a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a></sup>.</p><p id="p-8">To examine the hypothesized conformational transitions induced by proteolysis and receptor binding, we used single-particle cryo-EM to determine structures of S in uncleaved, S1/S2 cleaved and ACE2-bound states. Three-dimensional classification of the S1 RBD positions and corresponding atomic protein models revealed that neither ACE2-binding nor trypsin cleavage at the S1/S2 boundary induced substantial conformational changes in the prefusion protein conformation to expose the S2’ proteolysis site, suggesting additional requirements for fusion-associated conformational changes and secondary protease cleavage.</p><div class="section" id="sec-1"><h2 class="">Results</h2><div id="sec-2" class="subsection"><h3>Structural description of SARS-CoV S 2P ectodomain</h3><p id="p-9">To take advantage of improved protein expression and increased prefusion conformational stability, the SARS-CoV S ectodomain used for cryo-EM studies contained two proline substitutions (2P) in the S2 region<sup><a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a></sup>. These mutations had similar prefusion stabilizing effects on MERS-CoV S where their inclusion did not alter presentation of known epitopes or receptor recognition compared with wild-type MERS-CoV S. To establish a baseline for any potential conformational changes, we determined structures of the SARS-CoV S 2P ectodomain. This included a 3.2 Å resolution C3-symmetric reconstruction and several asymmetric reconstructions (3.9-4.5 Å, see below) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a> and Supplementary Table 1 and 4 and Supplementary Fig. 1). The S1 NTD and some of the S1 RBD, while clearly present, appear poorly resolved in the cryo-EM maps, precluding a <em>de novo</em> build of these regions. For this reason, available SARS-CoV S crystal structures<sup><a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">22</a>,<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup> of corresponding domains were placed into these densities. Overall, the SARS-CoV S 2P structure resembles that of other coronaviruses, particularly those of the betacoronavirus genus<sup><a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">17</a>,<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>,<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">22</a></sup> and the previously published SARS-CoV wild-type prefusion ectodomains<sup><a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">22</a></sup>. The S1 domains surround the helical S2 subunits and interdigitate at the membrane-distal apex of the trimeric spike. The structure of the S2 subunit appears to be highly conserved across coronavirus genera<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>. The SARS-CoV prefusion wild-type and 2P structures are nearly identical, including regions proximal to the 2P mutation site (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1b</a> and <a id="xref-fig-1-3" class="xref-fig" href="#F1">c</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structure of the SARS-CoV S 2P ectotodomain a) The C3 symmetrized reconstruction of all particles within the dataset resembles that of other betacoronaviruses. Side and membrane-distal top views (90&#x2DA; rotation about x-axis) are shown. Coordinate models derived from cryo-EM reconstructions of b) the wild-type SARS-CoV S ectodomain22 and c) prefusion stabilized SARS-CoV S 2P ectodomain adopt identical conformations near the 2P mutation site. Coordinate models derived from C3 symmetry cryo-EM reconstructions are shown. d) Classification of heterogeneity within the S1 RBD reveals a distribution of RBD configurations with the single-&#x2018;up&#x2019; conformation being most prevalent. S1 regions are shown in blue and S2 regions are shown in green." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2088464672" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Structure of the SARS-CoV S 2P ectotodomain&lt;/span&gt; a) The C3 symmetrized reconstruction of all particles within the dataset resembles that of other betacoronaviruses. Side and membrane-distal top views (90&#x2DA; rotation about x-axis) are shown. Coordinate models derived from cryo-EM reconstructions of b) the wild-type SARS-CoV S ectodomain&lt;sup&gt;22&lt;/sup&gt; and c) prefusion stabilized SARS-CoV S 2P ectodomain adopt identical conformations near the 2P mutation site. Coordinate models derived from C3 symmetry cryo-EM reconstructions are shown. d) Classification of heterogeneity within the S1 RBD reveals a distribution of RBD configurations with the single-&#x2018;up&#x2019; conformation being most prevalent. S1 regions are shown in blue and S2 regions are shown in green.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F1.medium.gif" width="440" height="214"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F1.medium.gif" width="440" height="214"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <span class="caption-title">Structure of the SARS-CoV S 2P ectotodomain</span><p id="p-10" class="first-child">a) The C3 symmetrized reconstruction of all particles within the dataset resembles that of other betacoronaviruses. Side and membrane-distal top views (90˚ rotation about x-axis) are shown. Coordinate models derived from cryo-EM reconstructions of b) the wild-type SARS-CoV S ectodomain<sup><a id="xref-ref-22-7" class="xref-bibr" href="#ref-22">22</a></sup> and c) prefusion stabilized SARS-CoV S 2P ectodomain adopt identical conformations near the 2P mutation site. Coordinate models derived from C3 symmetry cryo-EM reconstructions are shown. d) Classification of heterogeneity within the S1 RBD reveals a distribution of RBD configurations with the single-‘up’ conformation being most prevalent. S1 regions are shown in blue and S2 regions are shown in green.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">Although the overall structures of SARS-CoV S are similar, we were able to build several S2 protein regions excluded from the previous SARS-CoV models, including a fusion-peptide-adjacent region as well as additional amino acids in the S2 C-terminal connector domain. The region C-terminal to the fusion peptide, known as FP2, has been suggested to act as the second part of a bipartite fusion peptide for SARS-CoV S<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup>. Comparison of this FP2 region in the structures presented here with the MHV<sup><a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a></sup>, HCoV-HKU1<sup><a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">16</a></sup>, MERS-CoV<sup><a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">17</a>,<a id="xref-ref-22-8" class="xref-bibr" href="#ref-22">22</a></sup>, HCoV-NL63<sup><a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a></sup> and Porcine deltacoronavirus<sup><a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a></sup> S structures indicates that SARS-CoV adopts a unique conformation in this region (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). Whereas most other coronaviruses possess two short α-helices with a cross-linking disulfide bond, in SARS-CoV a similar disulfide bond bridges the two strands of a beta-hairpin. The unique SARS-CoV S conformation of FP2 and the lack of cross-group sequence conservation beyond the disulfide bond in this region suggest that SARS-CoV may use a distinct mechanism of FP2 membrane insertion.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of putative bi-partite fusion peptide regions a) The SARS-CoV S FP2 region of the bi-partite fusion peptide24 adopts a conformation distinct from equivalent regions in b) alpha-(HuCoV-NL63, 5SZS.pdb20) and c) deltacoronavirus spikes (PDCV S, 6BFU.pdb21) as well as other d-f) betacoronaviruses (MERS-CoV S, 5W9I.pdb17, HuCoV-HKU1 S, 5I08.pdb16, MHV S, 3JCL.pdb19)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2088464672" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Comparison of putative bi-partite fusion peptide regions&lt;/span&gt; a) The SARS-CoV S FP2 region of the bi-partite fusion peptide&lt;sup&gt;24&lt;/sup&gt; adopts a conformation distinct from equivalent regions in b) alpha-(HuCoV-NL63, 5SZS.pdb&lt;sup&gt;20&lt;/sup&gt;) and c) deltacoronavirus spikes (PDCV S, 6BFU.pdb&lt;sup&gt;21&lt;/sup&gt;) as well as other d-f) betacoronaviruses (MERS-CoV S, 5W9I.pdb&lt;sup&gt;17&lt;/sup&gt;, HuCoV-HKU1 S, 5I08.pdb&lt;sup&gt;16&lt;/sup&gt;, MHV S, 3JCL.pdb&lt;sup&gt;19&lt;/sup&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F2.medium.gif" width="440" height="321"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F2.medium.gif" width="440" height="321"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <span class="caption-title">Comparison of putative bi-partite fusion peptide regions</span><p id="p-12" class="first-child">a) The SARS-CoV S FP2 region of the bi-partite fusion peptide<sup><a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a></sup> adopts a conformation distinct from equivalent regions in b) alpha-(HuCoV-NL63, 5SZS.pdb<sup><a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20</a></sup>) and c) deltacoronavirus spikes (PDCV S, 6BFU.pdb<sup><a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">21</a></sup>) as well as other d-f) betacoronaviruses (MERS-CoV S, 5W9I.pdb<sup><a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">17</a></sup>, HuCoV-HKU1 S, 5I08.pdb<sup><a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">16</a></sup>, MHV S, 3JCL.pdb<sup><a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">19</a></sup>).</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Compositional and conformational heterogeneity of ACE2-bound SARS-CoV spikes Classification of heterogeneity within the SARS-CoV S &#x2013; ACE2 complex reveals that 55% of the particles are not bound by ACE2. Of those particles which are bound by ACE2, each configuration of the S1 RBD is observed with classes containing triple &#x2018;up&#x2019; conformation S1 RBD being the most poorly represented. The labeled description of each class begins at the lower RBD when viewed at the membrane distal apex and proceeds clockwise. Side views and membrane-distal top views are shown for each reconstruction. S1 regions are shown in blue, S2 regions are shown in green and soluble ACE2 is shown in orange." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2088464672" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Compositional and conformational heterogeneity of ACE2-bound SARS-CoV spikes&lt;/span&gt; Classification of heterogeneity within the SARS-CoV S &#x2013; ACE2 complex reveals that 55% of the particles are not bound by ACE2. Of those particles which are bound by ACE2, each configuration of the S1 RBD is observed with classes containing triple &#x2018;up&#x2019; conformation S1 RBD being the most poorly represented. The labeled description of each class begins at the lower RBD when viewed at the membrane distal apex and proceeds clockwise. Side views and membrane-distal top views are shown for each reconstruction. S1 regions are shown in blue, S2 regions are shown in green and soluble ACE2 is shown in orange.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F3.medium.gif" width="440" height="413"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F3.medium.gif" width="440" height="413"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <span class="caption-title">Compositional and conformational heterogeneity of ACE2-bound SARS-CoV spikes</span><p id="p-13" class="first-child">Classification of heterogeneity within the SARS-CoV S – ACE2 complex reveals that 55% of the particles are not bound by ACE2. Of those particles which are bound by ACE2, each configuration of the S1 RBD is observed with classes containing triple ‘up’ conformation S1 RBD being the most poorly represented. The labeled description of each class begins at the lower RBD when viewed at the membrane distal apex and proceeds clockwise. Side views and membrane-distal top views are shown for each reconstruction. S1 regions are shown in blue, S2 regions are shown in green and soluble ACE2 is shown in orange.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">As observed in the previous SARS-CoV and MERS-CoV S structures<sup><a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-22-9" class="xref-bibr" href="#ref-22">22</a></sup>, the trimeric S adopts two distinct conformations related to each of the S1 RBD. The ‘down’ conformation caps the S2 helices and makes extensive contacts with the S1 NTD. The ‘up’ conformation of the S1 RBD exposes the S1 RBD receptor-binding site. It has been previously reported that for wild-type SARS-CoV S, 56% of the particles contained three ‘down’ RBD conformations while 44% contained a single ‘up’ S1 RBD conformation<sup><a id="xref-ref-22-10" class="xref-bibr" href="#ref-22">22</a></sup>. To examine the conformation of the S1 RBD among our SARS-CoV S 2P ectodomains, we used a local masking and 3-D sorting strategy<sup><a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">17</a></sup> to more accurately classify the conformations as being either ‘down’ or ‘up’ at each of the three S1 RBD positions within the trimer. This analysis revealed that the majority of the S 2P proteins were in the single-‘up’ conformation (58%) with lesser amounts of double- and triple-‘up’ conformations (39% and 3% respectively) and with no all-‘down’ conformation observed. The increased propensity to adopt the ‘up’ S1 RBD conformation may indicate a difference in the coronavirus S containing the 2P mutations, however other differences in sample preparation cannot be ruled out.</p></div><div id="sec-3" class="subsection"><h3>ACE2 and S1 C-terminal domains</h3><p id="p-15">To examine the structure of SARS-CoV S bound to its receptor, ACE2, we combined SARS-CoV S 2P ectodomain with an excess of soluble human ACE2 with subsequent purification by size-exclusion chromatography and immediate cryo-EM specimen preparation. Initial sorting of particle heterogeneity indicated spikes could be split into ACE2-bound (45%) and unbound (55%) classes. Using a similar masking and 3-D sorting strategy as above we sorted the unbound S class further into classes with S1 conformations of one or two ‘up’ S1 RBDs (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig 3</a> and Supplementary Tables 2-4 and Supplementary Fig. 2-4). We did not observe an all-‘down’ class nor a three ‘up’ S1 RBD class indicating a low prevalence of these conformations among the unbound spikes. Expanding our 3D sorting strategy, we classified our ACE2-bound particles at each S1 RBD position and identified single, double and triple ACE2-bound S. We were further able to identify S1 RBD conformations at the non-ACE2 occupied RBD positions to represent each population of S1 RBD conformations among ACE2-bound S.</p><p id="p-16">As hypothesized by previous structural work<sup><a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">15</a>–<a id="xref-ref-17-8" class="xref-bibr" href="#ref-17">17</a>,<a id="xref-ref-22-11" class="xref-bibr" href="#ref-22">22</a></sup>, the S1 RBD recognizes ACE2 with an ‘up’ S1 RBD conformation. The proportion of total ‘up’ S1 RBD conformations within the ACE2-bound and -unbound classes is nearly identical within this dataset (58% ‘up’ S1 RBD), similar to the proportion of total ‘up’ S1 RBD in the SARS S 2P ectodomain dataset (48%). This strongly suggests that binding of a single ACE2 receptor does not induce adjacent S1 RBDs to transition from a ‘down’ to ‘up’ conformation. Hence, ACE2 is more likely to bind to an already ‘up’ S1 RBD rather than inducing the conformational changes that are required for the S1 RBD to become accessible to ACE2.</p><p id="p-17">It is noteworthy that despite prolonged co-incubation and an excess of ACE2, we had difficulties in saturating the S1 RBD with ACE2 in the context of trimeric S ectodomain. This poor saturation is illustrated by the small proportion of triple-bound ACE2 and the majority of spikes that are unbound by receptor. In contrast, isolated recombinant S1 RBD easily binds ACE2 and is capable saturating ACE2 on target cells to block S-mediated entry<sup><a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a></sup>. Our observed sub-stoichiometric ACE2 binding to trimeric spikes is consistent with the difficulty in using soluble ACE2 receptor to neutralize SARS-CoV S pseudotyped onto VSV<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup>. The reduced binding of ACE2 to trimeric spikes is likely due to the incomplete exposure and conformational flexibility of the S1 RBD. Incomplete neutralization with soluble receptor was not encountered for MHV which binds CEACAM1a via its S1 NTD, which does not undergo conformational changes<sup><a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">19</a>,<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>.</p><p id="p-18">Similar to recently published MERS-CoV S structures<sup><a id="xref-ref-17-9" class="xref-bibr" href="#ref-17">17</a></sup>, the ACE2-bound RBD adopts a much more extended and rotated conformation compared to S1 RBD modeled in previous SARS-CoV S structures<sup><a id="xref-ref-22-12" class="xref-bibr" href="#ref-22">22</a></sup>. This difference is likely due to poor density in the hinge regions between the S1 RBD and subdomain 1 (SD-1) in these previous reconstructions<sup><a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-22-13" class="xref-bibr" href="#ref-22">22</a></sup> rather than the presentation of a unique receptor-bound conformation. Indeed, the bound ACE2 receptor and S1 RBD for all reconstructions here show poorer density quality than the less mobile regions of the SARS-CoV S (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig 4</a>). To improve the density for ACE2-bound S1 RBD, we used focused refinement on this region to overcome the flexibility of these domains relative to the rest of S. This yielded a 7.9 Å resolution reconstruction with improved local density quality (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig 4b</a> and <a id="xref-fig-4-3" class="xref-fig" href="#F4">c</a>). We successfully placed the crystal structure of the SARS-CoV S1 RBD bound to ACE2 (2AJF.pdb<sup><a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a></sup>) into this density as a rigid body indicating that the previously determined crystal structure accurately recapitulates the conformation between the ACE2-bound S1 RBD in the trimeric spike.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ACE2-receptor binding and induced conformational changes a) ACE2 binds the SARS-CoV S1 RBD in an &#x2018;up&#x2019; conformation. b) Focused refinement was used on the S1 RBD &#x2013; ACE2 portion of the particles to improve the density of this flexible region. c) Rigid-body fitting of the SARS-CoV S1 RBD &#x2013; ACE2 complex crystal structure (2AJF.pdb23) indicates that the crystal structure is representative of the ACE2-bound trimeric spike. Comparison of the d) SARS-CoV S 2P central helices with the e) SARS-CoV S 2P central helix which has been uncapped by an ACE2-bound S1 RBD demonstrates a transition to a short 310 helix. Colors are as in Fig. 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2088464672" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;ACE2-receptor binding and induced conformational changes&lt;/span&gt; a) ACE2 binds the SARS-CoV S1 RBD in an &#x2018;up&#x2019; conformation. b) Focused refinement was used on the S1 RBD &#x2013; ACE2 portion of the particles to improve the density of this flexible region. c) Rigid-body fitting of the SARS-CoV S1 RBD &#x2013; ACE2 complex crystal structure (2AJF.pdb&lt;sup&gt;23&lt;/sup&gt;) indicates that the crystal structure is representative of the ACE2-bound trimeric spike. Comparison of the d) SARS-CoV S 2P central helices with the e) SARS-CoV S 2P central helix which has been uncapped by an ACE2-bound S1 RBD demonstrates a transition to a short 3&lt;sub&gt;10&lt;/sub&gt; helix. Colors are as in Fig. 3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F4.medium.gif" width="440" height="338"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F4.medium.gif" width="440" height="338"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <span class="caption-title">ACE2-receptor binding and induced conformational changes</span><p id="p-19" class="first-child">a) ACE2 binds the SARS-CoV S1 RBD in an ‘up’ conformation. b) Focused refinement was used on the S1 RBD – ACE2 portion of the particles to improve the density of this flexible region. c) Rigid-body fitting of the SARS-CoV S1 RBD – ACE2 complex crystal structure (2AJF.pdb<sup><a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a></sup>) indicates that the crystal structure is representative of the ACE2-bound trimeric spike. Comparison of the d) SARS-CoV S 2P central helices with the e) SARS-CoV S 2P central helix which has been uncapped by an ACE2-bound S1 RBD demonstrates a transition to a short 3<sub>10</sub> helix. Colors are as in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">The ACE2-bound, S1 RBD extends upwards and rotates away from contacts with nearby amino acids. Hence, any conformational changes induced by receptor binding to the S1 RBD are more likely to be caused by the absence of the S1 RBD contacts in the ‘up’ conformation, rather than the formation of additional contacts (Supplemental Figure 5). This model provides a flexible mechanism for how different coronavirus spikes can bind to different protein receptors with their S1 RBD and facilitate fusion with host cells. Moreover, movements of the S1 RBD to the ‘up’ conformation disrupt molecular interactions with the S2 central α-helices which may play a key role in receptor-induced conformational changes leading to membrane fusion.</p></div><div id="sec-4" class="subsection"><h3>SARS-CoV S2 fusion machinery and receptor binding</h3><p id="p-21">Several biochemical and virological studies have suggested that binding of a host protein receptor to coronavirus spikes induces conformational changes leading to fusion and exposure of the S2’ protease site for cleavage<sup><a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup>. Comparison of our ACE2-bound 2P-stabilized spike structures with those of both the wild-type SARS-CoV S and the 2P-stabilized SARS-CoV S indicate that overall, these structures are surprisingly similar and do not exhibit large conformational differences. The absence of conformational changes includes regions near the S2’ cleavage site, fusion peptide and S2 heptad repeat 1 (HR1). However, fine examination of the ACE2-bound S structure revealed a modest conformational change in the S2 central helix where the S2 had been uncapped by a receptor-bound RBD. In both the wild-type<sup><a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-22-14" class="xref-bibr" href="#ref-22">22</a></sup> and 2P unbound structures, the central helix presents as an α-helix. However, in the ACE2-bound structure, the upper portion of this helix has transitioned to a 3<sub>10</sub>-helix (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4d</a> and <a id="xref-fig-4-5" class="xref-fig" href="#F4">e</a>). This subtle transition may indicate that conformational changes towards fusion are initiated in the central helix. N-terminal to the central helix are the 2P mutations that stabilize the spike in its prefusion conformation. These stabilizing mutations may block the propagation of conformational changes beyond the subtle change to a short 3<sub>10</sub>-helix, providing a possible mechanism for their prefusion stabilizing effect.</p></div><div id="sec-5" class="subsection"><h3>Trypsin-cleaved SARS-CoV spike</h3><p id="p-22">Unlike some other coronavirus spikes such as MERS-CoV S, SARS-CoV S lacks an amino acid sequence capable of recognition by host furin proteases at the S1/S2 cleavage site. However, SARS-CoV S can be cleaved <em>in vitro</em> by exogenous trypsin at an equivalent site, and it has been proposed that a similar cleavage event may occur <em>in vivo</em> by trypsin-like proteases during viral entry<sup><a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup>. Moreover, trypsin cleavage of the SARS-CoV S potentiates S for more efficient membrane fusion<sup><a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>,<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup>. For MERS-CoV S, cleavage at S1/S2 enables more efficient receptor recognition<sup><a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a></sup>. In addition, coronaviruses possess a secondary cleavage site in S2 called S2’<sup><a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">27</a></sup>. This site is thought to be cleaved by host proteases after receptor recognition<sup><a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">4</a>–<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a></sup> and liberates the viral fusion peptide to insert into host membranes in a S2 pre-hairpin intermediate<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>. The S2’ cleavage has been proposed to occur after an arginine residue conserved across coronavirus genera (SARS-CoV S Arg797)<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup>.</p><p id="p-23">To examine the trypsin protease sensitivity of our coronavirus spikes, we carried out limited proteolysis experiments of both wild-type and 2P SARS-CoV S ectodomains in the presence and absence of soluble ACE2 receptor. A time course of the proteolysis revealed that all four samples are cleaved to S1 and S2 products at equivalent rates at similar sites (Supplementary Fig. 6). Nearing the end of the time course additional lower molecular weight bands are observed which we interpret to be degradation of the S1 subunit. Regardless of which construct was used or whether ACE2 was bound to the S ectodomains, there is no prominent band that corresponds to a S2’ cleavage product (approximately 52 kDa).</p><p id="p-24">To analyze the cleavage products in detail, we performed cryo-EM analysis on the trypsin-cleaved SARS-CoV S 2P ectodomain. Using all-particles and C3 symmetry yielded a reconstruction at 3.3 Å resolution (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5</a>, Supplementary Tables 1 and 4 and Supplementary Fig. 7). The short loop containing the S1/S2 cleavage site is disordered in the uncleaved spike reconstruction and remains disordered in the trypsin cleaved reconstruction. Moreover, examination of the structure models indicates no significant differences between the trypsin-cleaved and uncleaved SARS-CoV S (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5b</a>). Fine sorting of S1 RBD positions of the trypsin-cleaved S reveals a very similar distribution of ‘up’ S1 RBD conformations available for receptor binding as in the uncleaved samples, although we additionally observe a small proportion of S1 RBD in the all-‘down’ conformation (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5c</a>). These results indicate that trypsin-cleavage at S1/S2 does not impart large conformational changes on the SARS-CoV S and justifies the removal of S1/S2 cleavage sites for the production of more homogeneous material as vaccine immunogens. This suggests that although cleavage at S1/S2 may remove an obstacle for conformational changes leading to fusion, S1/S2 cleavage alone does not produce significant conformational changes.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Trypsin cleavage does not impart conformational changes in prefusion SARS-CoV spike a) An all particle reconstruction of trypsin-cleaved SARS-CoV S 2P with C3 symmetry. b) Comparison of the trypsin-cleaved and uncleaved SARS-CoV S 2P coordinate models reveals no differences in protein structure near the cleavage site. The last residue in S1 (T662) and the first residue in S2 (K672) visualized in the coordinates are labeled. c) Classification of the heterogeneous S1 RBD in the trypsin-cleaved S shows a predominance of the single-&#x2018;up&#x2019; conformation as well as the observation of an all-&#x2018;down&#x2019; conformation not seen in the uncleaved S 2P sample. Colors are as in Fig. 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2088464672" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Trypsin cleavage does not impart conformational changes in prefusion SARS-CoV spike&lt;/span&gt; a) An all particle reconstruction of trypsin-cleaved SARS-CoV S 2P with C3 symmetry. b) Comparison of the trypsin-cleaved and uncleaved SARS-CoV S 2P coordinate models reveals no differences in protein structure near the cleavage site. The last residue in S1 (T662) and the first residue in S2 (K672) visualized in the coordinates are labeled. c) Classification of the heterogeneous S1 RBD in the trypsin-cleaved S shows a predominance of the single-&#x2018;up&#x2019; conformation as well as the observation of an all-&#x2018;down&#x2019; conformation not seen in the uncleaved S 2P sample. Colors are as in Fig. 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F5.medium.gif" width="412" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F5.medium.gif" width="412" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5:</span> <span class="caption-title">Trypsin cleavage does not impart conformational changes in prefusion SARS-CoV spike</span><p id="p-25" class="first-child">a) An all particle reconstruction of trypsin-cleaved SARS-CoV S 2P with C3 symmetry. b) Comparison of the trypsin-cleaved and uncleaved SARS-CoV S 2P coordinate models reveals no differences in protein structure near the cleavage site. The last residue in S1 (T662) and the first residue in S2 (K672) visualized in the coordinates are labeled. c) Classification of the heterogeneous S1 RBD in the trypsin-cleaved S shows a predominance of the single-‘up’ conformation as well as the observation of an all-‘down’ conformation not seen in the uncleaved S 2P sample. Colors are as in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">In agreement with the <em>in vitro</em> limited proteolysis experiments, the S2’ site remains uncleaved in the trypsin-cleaved spike ectodomain structures. The arginine at the proposed S2’ cleavage site in all coronavirus spike structures published to date and presented here is blocked from cleavage by a loop protruding from just N-terminal to the fusion peptide and by an N-terminal helix of S2 HR1 (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig 6</a>). Exposure of this site for cleavage may require remodeling of this penultimate loop or HR1 beyond the conformation observed in the prefusion state. We hypothesize that additional triggers beyond cleavage at the S1/S2 site or protein-receptor binding are needed to transition the spike from its prefusion state to a yet to be observed intermediate. Further, this intermediate may be the target for S2’ cleavage before final adoption of the postfusion six-helix bundle conformation and membrane fusion. Indeed, it has been suggested that for some strains of MHV, the spike undergoes a S2’-like cleavage after transitioning to a SDS-resistant trimer in the presence of its protein receptor<sup><a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">4</a></sup>.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="S2&#x2019; secondary cleavage site is occluded Superimposition of the SARS-CoV S 2P, uncleaved, cleaved and ACE2-bound structures reveals no structural changes within this region. The S2&#x2019; site after amino acid R797 is shielded from cleavage by an adjacent loop and the fusion peptide which packs against an N-terminal region of S2 HR1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2088464672" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;S2&#x2019; secondary cleavage site is occluded&lt;/span&gt; Superimposition of the SARS-CoV S 2P, uncleaved, cleaved and ACE2-bound structures reveals no structural changes within this region. The S2&#x2019; site after amino acid R797 is shielded from cleavage by an adjacent loop and the fusion peptide which packs against an N-terminal region of S2 HR1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F6.medium.gif" width="440" height="394"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F6.medium.gif" width="440" height="394"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6:</span> <span class="caption-title">S2’ secondary cleavage site is occluded</span><p id="p-27" class="first-child">Superimposition of the SARS-CoV S 2P, uncleaved, cleaved and ACE2-bound structures reveals no structural changes within this region. The S2’ site after amino acid R797 is shielded from cleavage by an adjacent loop and the fusion peptide which packs against an N-terminal region of S2 HR1.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-6"><h2 class="">Discussion</h2><p id="p-28">SARS-CoV is a highly transmissible pathogen, which emerged from an animal reservoir and rapidly spread around the world<sup><a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a></sup>. Though eventually removed from human circulation through public health measures, SARS-like coronaviruses continue to circulate in animal reservoirs<sup><a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup> and there are no vaccines or virus-specific treatments available for human use. Understanding how coronavirus S glycoproteins are processed and bind to host receptors is key to the development of coronavirus vaccines and therapeutics.</p><p id="p-29">Here, we present the first structures of a trimeric coronavirus spike ectodomain bound to a protein receptor as well as the first structures of prefusion S1/S2 cleaved coronavirus spikes. Our structures of the SARS-CoV S 2P ectodomain bound to a soluble form of human ACE2 receptor show that any conformational changes induced in S by receptor binding are more likely to be due to the disruption of protein-protein interactions rather than the formation of additional contacts between the S1 RBD and other regions of S (Supplemental Fig. 5). The only conformational change that we observe in the ACE2-bound structures is the transition to a short 3<sub>10</sub>-helix at the top of the S2 central α-helix when uncapped by receptor-bound S1 RBD suggesting that more extensive conformational changes may be initiated here. The 2P prefusion stabilizing mutations adjacent to this region may block more extensive transitions from occurring. However, both wild-type and 2P SARS-CoV S ectodomains bound to ACE2 were cleaved equally by trypsin, indicating that at least at the level of protease susceptibility, these two receptor-bound complexes are equivalent.</p><p id="p-30">Our examinations of both receptor-bound and trypsin-cleaved prefusion SARS-CoV S proteins indicate that neither receptor binding nor trypsin cleavage induces large conformational changes in S and do not expose the S2’ cleavage site for proteolysis. These data suggest that SARS coronavirus spikes may require an additional factor to undergo such conformational changes and that the S2’ proteolysis does not occur in the S prefusion state (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 7</a>). This hypothesis is favored by the packing of the S2’ cleavage site next to S2 HR1. If HR1 were to extend towards the pre-hairpin intermediate, this may remove restrictions on the conformation of the fusion peptide and adjacent S2’ cleavage site to allow for this secondary cleavage. Possible additional factors may include the presence of apposing membranes, the shedding of the S1 subunits or the involvement of yet to be identified host factors.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The coronavirus fusion process The prefusion SARS-CoV S RBD conformationally samples both the down and up states prior to binding the ACE2 receptor. We hypothesize that there is an unobserved intermediate which is not directly trigged by either S1/S2 protease cleavage or ACE2 binding that exposes the S2&#x2019; cleavage site. Cleavage at the S2&#x2019; site allows membrane insertion of the fusion peptide and subsequent adoption of the postfusion S conformation and membrane fusion. Colors are as in Fig. 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2088464672" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The coronavirus fusion process&lt;/span&gt; The prefusion SARS-CoV S RBD conformationally samples both the down and up states prior to binding the ACE2 receptor. We hypothesize that there is an unobserved intermediate which is not directly trigged by either S1/S2 protease cleavage or ACE2 binding that exposes the S2&#x2019; cleavage site. Cleavage at the S2&#x2019; site allows membrane insertion of the fusion peptide and subsequent adoption of the postfusion S conformation and membrane fusion. Colors are as in Fig. 3.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F7.medium.gif" width="440" height="181"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7:" src="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F7.medium.gif" width="440" height="181"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/31/292672/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7:</span> <span class="caption-title">The coronavirus fusion process</span><p id="p-31" class="first-child">The prefusion SARS-CoV S RBD conformationally samples both the down and up states prior to binding the ACE2 receptor. We hypothesize that there is an unobserved intermediate which is not directly trigged by either S1/S2 protease cleavage or ACE2 binding that exposes the S2’ cleavage site. Cleavage at the S2’ site allows membrane insertion of the fusion peptide and subsequent adoption of the postfusion S conformation and membrane fusion. Colors are as in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-7"><h2 class="">Methods</h2><div id="sec-8" class="subsection"><h3>Plasmid construction, Protein expression and purification</h3><p id="p-32">A mammalian-codon-optimized gene encoding SARS-CoV S (Tor2 strain) residues 1– 1190 with a C-terminal T4 fibritin trimerization domain, an HRV3C cleavage site, an 8xHis-tag and a Twin-Strep-tag was synthesized and subcloned into the eukaryotic-expression vector pαH. Proline-substituted variant harboring K968P and V969P mutations was generated based on this construct. The resulting plasmids, designated SARS S WT and SARS S 2P respectively, were transfected into 1 L FreeStyle 293-F cells (Life Technologies). 293-F cells were used without validation and were not tested for mycoplasma. Three hours after transfection, kifunensine was added to a final concentration of 5 µM. Cultures were harvested after 6 d, and protein was purified from the supernatant using Strep-Tactin resin (IBA). HRV3C protease (1% wt/wt) was added to the protein and the reaction was incubated overnight at 4 °C. The digested protein was further purified using a Superose 6 16/70 column (GE Healthcare Life Sciences).</p><p id="p-33">A gene encoding human ACE2 residues 1-615 with an HRV3C cleavage site, an 8xHis-tag and a Twin-Strep-tag was synthesized and subcloned into the eukaryotic-expression vector pαH. This protein was expressed as described above for S proteins. The HRV3C digested ACE2 protein was further purified using a Superdex 200 column (GE Healthcare Life Sciences).</p><p id="p-34">To make complexes of SARS S 2P with soluble ACE2, the two proteins were combined in a 1:1.1 molar ratio prior to overnight incubation and size-exclusion chromatography on a Superose 6 Increase 10/300 (GE Healthcare Life Sciences).</p></div><div id="sec-9" class="subsection"><h3>Electron microscopy data collection</h3><p id="p-35">3 µL of 0.47 mg/mL of protein or protein complexes were mixed with 1 µL of 0.04 % (w/v) amphipol A8-35 (Anatrace) just prior to grid preparation. 3 µL of the protein mixture, final concentration 0.35 mg/mL, was spotted on to plasma cleaned C-flat grids with a 2 µm/2 µm spacing. Grids were blotted for 3 seconds before plunge freezing in liquid ethane on a Vitrobot. Grids were loaded onto a Titan Krios and data was collected using Leginon<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup> at a total dose of 65 e<sup>-</sup>/Å<sup><a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a></sup>. Frames were aligned with MotionCor2 (UCSF)<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup> implemented in the Appion workflow<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>. Particles were selected using DoG picker<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>. Images were assessed and particle picks were masked using EM Hole Punch<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup>. The CTF for each image was estimated using Gctf<sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup>.</p></div><div id="sec-10" class="subsection"><h3>Electron microscopy data processing</h3><p id="p-36">Initial particle stacks were cleaned using multiple rounds of 2D classification in RELION<sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup>. Good particles were selected as resembling prefusion coronavirus spikes. For the SARS S 2P and trypsin-treated SARS S 2P, all particles from the clean stacks were used for reconstruction with C3 symmetry. All datasets were extensively sorted using 3D classification to examine heterogeneity in the S1 RBDs as described previously<sup><a id="xref-ref-17-10" class="xref-bibr" href="#ref-17">17</a></sup>. Briefly, 3D masks were defined to encompass the possible heterogeneity at each S1 RBD position. The density within these masks was then removed from unfiltered, unsharpened reconstructions. We then used relion_project with image subtraction to create a particle stack containing only the signal arising from the masked density. Finally, we used focused 3D classification to identify compositional and conformational states at each S1 RBD position. All 3D reconstructions were produced with RELION<sup><a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a></sup> and final refinements were performed with a six-pixel soft-edge solvent mask. Post-processing was applied to each reconstruction to apply B-factor sharpening and amplitude corrections as well as to calculate local resolution maps.</p><p id="p-37">Coordinate models were built for several of the high-resolution reconstructions using 5I08.pdb<sup><a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">16</a></sup>, 2AJF.pdb<sup><a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">23</a></sup> and 5X4S.pdb<sup><a id="xref-ref-22-15" class="xref-bibr" href="#ref-22">22</a></sup> as template models with reference to a recently published wild-type SARS S ectodomain (5X58.pdb<sup><a id="xref-ref-22-16" class="xref-bibr" href="#ref-22">22</a></sup>). Manual model building was preformed in Coot<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup> with real-space refinement refinement in Rosetta<sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup> and PHENIX<sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup> with final rounds of refinement performed in PHENIX.</p></div><div id="sec-11" class="subsection"><h3>Limited proteolysis</h3><p id="p-38">10 µg of either SARS-CoV S 2P or SARS-CoV S WT ectodomains were mixed with soluble ACE2 in a 1:1.1 ratio and incubated overnight at 4˚C. Proteins were digested with 0.1% (w/w) TPCK trypsin at room temperature. For SDS-PAGE analysis, samples were removed at indicated time points, immediately mixed with SDS loading buffer and incubated at 95˚C for 1 minute. For EM analysis, the proteolysis reaction was stopped after two hours using phenyl-methyl-sulfonyl-floride (PMSF) at a final concentration of 1 mM and then immediately loaded onto a Superose 6 Increase 10/300 size-exclusion column.</p></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-39">Plasmid DNA used in this study is available upon request. Electron microscopy maps are deposited in the Electron Microscopy Data Bank with accession codes: 7573, 7574, 7575, 7576, 7577, 7578, 7579, 7580, 7581, 7582, 7584, 7585, 7586, 7601, 7602, 7603, 7604, 7605, 7606, 7607 and 7608. Atomic models for select maps are deposited in the Protein Data Bank with accession codes 6CRV, 6CRW, 6CRX, 6CRZ, 6CS0, 6CS1 and 6CS2.</p></div></div><div class="section" id="sec-13"><h2 class="">Author Contributions</h2><p id="p-41">R.N.K, J.S.M and A.B.W conceived and designed experiments. N.W. and D.W. expressed and purified protein. R.N.K. and H.L.T. collected electron microscopy data. R.N.K. and J.P. processed electron microscopy data. R.N.K and C.A.C. built and refined atomic models. B.S.G. and K.S.C. contributed support and advice. All authors contributed to writing the manuscript.</p></div><div class="section COI-statement" id="sec-14"><h2 class="">Competing financial interests</h2><p id="p-42">R.N.K, N.W., J.P., H.L.T., C.A.C, K.S.C, B.S.G., J.S.M., and A.B.W. are inventors on US patent application no. 62/412,703, entitled “Prefusion Coronavirus Spike Proteins and Their Use.”</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-40">We gratefully acknowledge Travis Nieusma, Charles Bowman, Jean-Christophe Ducom and Bill Anderson for microscopy and computational support. We also thank Lauren Holden for a critical reading of this manuscript. This work was supported by grants from NIH/NIAID to A.B.W and J.S.M. (AI127521) and to R.N.K (AI123498). Computational resources for electron microscopy at The Scripps Research Institute are supported by NIH grant OD021634.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list"><li><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.1" data-doi="10.1038/nrmicro.2016.81"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Wit</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Doremalen</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Falzarano</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Munster</span>,  <span class="cit-name-given-names">V. J.</span></span> <span class="cit-article-title">SARS and MERS: recent insights into emerging coronaviruses</span>. <abbr class="cit-jnl-abbrev">Nature reviews. Microbiology</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">523</span>–<span class="cit-lpage">534</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nrmicro.2016.81</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.volume%253D14%26rft.spage%253D523%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro.2016.81%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro.2016.81&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.2" data-doi="10.1073/pnas.1517719113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Menachery</span>,  <span class="cit-name-given-names">V. D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">SARS-like WIV1-CoV poised for human emergence</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">113</span>, <span class="cit-fpage">3048</span>–<span class="cit-lpage">3053</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1517719113</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1517719113%26rft_id%253Dinfo%253Apmid%252F26976607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzExLzMwNDgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMy8zMS8yOTI2NzIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.3" data-doi="10.3390/v4061011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Belouzard</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Millet</span>,  <span class="cit-name-given-names">J. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Licitra</span>,  <span class="cit-name-given-names">B. N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Whittaker</span>,  <span class="cit-name-given-names">G. R.</span></span> <span class="cit-article-title">Mechanisms of coronavirus cell entry mediated by the viral spike protein</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">1011</span>–<span class="cit-lpage">1033</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3390/v4061011</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DBelouzard%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253D1011%26rft.epage%253D1033%26rft.atitle%253DMechanisms%2Bof%2Bcoronavirus%2Bcell%2Bentry%2Bmediated%2Bby%2Bthe%2Bviral%2Bspike%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv4061011%26rft_id%253Dinfo%253Apmid%252F22816037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v4061011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22816037&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F31%2F292672.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305801900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4 in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.4" data-doi="10.1128/JVI.00959-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matsuyama</span>,  <span class="cit-name-given-names">S.</span></span> &amp;, <span class="cit-auth"><span class="cit-name-surname">Taguchi</span>  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Two-step conformational changes in a coronavirus envelope glycoprotein mediated by receptor binding and proteolysis</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">83</span>, <span class="cit-fpage">11133</span>–<span class="cit-lpage">11141</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.00959-09</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00959-09%26rft_id%253Dinfo%253Apmid%252F19706706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4My8yMS8xMTEzMyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzAzLzMxLzI5MjY3Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-292672v1.5" data-doi="10.1007/s00705-011-1044-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">J. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cruz</span>,  <span class="cit-name-given-names">D. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Shin</span>,  <span class="cit-name-given-names">H. J.</span></span> <span class="cit-article-title">Receptor-bound porcine epidemic diarrhea virus spike protein cleaved by trypsin induces membrane fusion</span>. <abbr class="cit-jnl-abbrev">Archives of virology</abbr> <span class="cit-vol">156</span>, <span class="cit-fpage">1749</span>–<span class="cit-lpage">1756</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s00705-011-1044-6</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Bvirology%26rft.stitle%253DArch%2BVirol%26rft.aulast%253DPark%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D156%26rft.issue%253D10%26rft.spage%253D1749%26rft.epage%253D1756%26rft.atitle%253DReceptor-bound%2Bporcine%2Bepidemic%2Bdiarrhea%2Bvirus%2Bspike%2Bprotein%2Bcleaved%2Bby%2Btrypsin%2Binduces%2Bmembrane%2Bfusion.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-011-1044-6%26rft_id%253Dinfo%253Apmid%252F21667287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-011-1044-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21667287&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F31%2F292672.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.6" data-doi="10.1073/pnas.1608147113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">J. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Proteolytic processing of Middle East respiratory syndrome coronavirus spikes expands virus tropism</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">113</span>, <span class="cit-fpage">12262</span>–<span class="cit-lpage">12267</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1608147113</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1608147113%26rft_id%253Dinfo%253Apmid%252F27791014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzQzLzEyMjYyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDMvMzEvMjkyNjcyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7 in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.7" data-doi="10.1073/pnas.0505577102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Simmons</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Inhibitors of cathepsin L prevent severe acute respiratory syndrome coronavirus entry</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">102</span>, <span class="cit-fpage">11876</span>–<span class="cit-lpage">11881</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0505577102</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0505577102%26rft_id%253Dinfo%253Apmid%252F16081529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzMzLzExODc2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDMvMzEvMjkyNjcyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.8" data-doi="10.1128/JVI.77.16.8801-8811.2003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bosch</span>,  <span class="cit-name-given-names">B. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Zee</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Haan</span>,  <span class="cit-name-given-names">C. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rottier</span>,  <span class="cit-name-given-names">P. J.</span></span> <span class="cit-article-title">The coronavirus spike protein is a class I virus fusion protein: structural and functional characterization of the fusion core complex</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">77</span>, <span class="cit-fpage">8801</span>–<span class="cit-lpage">8811</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DBosch%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D77%26rft.issue%253D16%26rft.spage%253D8801%26rft.epage%253D8811%26rft.atitle%253DThe%2BCoronavirus%2BSpike%2BProtein%2BIs%2Ba%2BClass%2BI%2BVirus%2BFusion%2BProtein%253A%2BStructural%2Band%2BFunctional%2BCharacterization%2Bof%2Bthe%2BFusion%2BCore%2BComplex%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.77.16.8801-8811.2003%26rft_id%253Dinfo%253Apmid%252F12885899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3Ny8xNi84ODAxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDMvMzEvMjkyNjcyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-292672v1.9" data-doi="10.1038/371037a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bullough</span>,  <span class="cit-name-given-names">P. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hughson</span>,  <span class="cit-name-given-names">F. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skehel</span>,  <span class="cit-name-given-names">J. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wiley</span>,  <span class="cit-name-given-names">D. C.</span></span> <span class="cit-article-title">Structure of influenza haemagglutinin at the pH of membrane fusion</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">371</span>, <span class="cit-fpage">37</span>–<span class="cit-lpage">43</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/371037a0</span> (<span class="cit-pub-date">1994</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBullough%26rft.auinit1%253DP.%2BA.%26rft.volume%253D371%26rft.issue%253D6492%26rft.spage%253D37%26rft.epage%253D43%26rft.atitle%253DStructure%2Bof%2Binfluenza%2Bhaemagglutinin%2Bat%2Bthe%2BpH%2Bof%2Bmembrane%2Bfusion.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F371037a0%26rft_id%253Dinfo%253Apmid%252F8072525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/371037a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8072525&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F31%2F292672.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PE38100042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.10" data-doi="10.1073/pnas.1708727114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>,  <span class="cit-name-given-names">A. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Tectonic conformational changes of a coronavirus spike glycoprotein promote membrane fusion</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">114</span>, <span class="cit-fpage">11157</span>–<span class="cit-lpage">11162</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1708727114</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1708727114%26rft_id%253Dinfo%253Apmid%252F29073020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzQyLzExMTU3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDMvMzEvMjkyNjcyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">SARS Immunity and Vaccination</span>. <abbr class="cit-jnl-abbrev">Cellular &amp; molecular immunology</abbr> <span class="cit-vol">1</span>, <span class="cit-fpage">193</span>–<span class="cit-lpage">198</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellular%2B%2526%2Bmolecular%2Bimmunology%26rft.volume%253D1%26rft.spage%253D193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.12" data-doi="10.1128/JVI.01394-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">R. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R. S.</span></span> <span class="cit-article-title">Recombination, reservoirs, and the modular spike: mechanisms of coronavirus cross-species transmission</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">84</span>, <span class="cit-fpage">3134</span>–<span class="cit-lpage">3146</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01394-09</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01394-09%26rft_id%253Dinfo%253Apmid%252F19906932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg0LzcvMzEzNCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzAzLzMxLzI5MjY3Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.13" data-doi="10.1038/nature02145"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">426</span>, <span class="cit-fpage">450</span>–<span class="cit-lpage">454</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature02145</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D426%26rft.issue%253D6965%26rft.spage%253D450%26rft.epage%253D454%26rft.atitle%253DAngiotensin-converting%2Benzyme%2B2%2Bis%2Ba%2Bfunctional%2Breceptor%2Bfor%2Bthe%2BSARS%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02145%26rft_id%253Dinfo%253Apmid%252F14647384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14647384&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F31%2F292672.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.14" data-doi="10.1074/jbc.C300520200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">S. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>,  <span class="cit-name-given-names">M. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choe</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Farzan</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">A 193-amino acid fragment of the SARS coronavirus S protein efficiently binds angiotensin-converting enzyme 2</span>. <abbr class="cit-jnl-abbrev">The Journal of biological chemistry</abbr> <span class="cit-vol">279</span>, <span class="cit-fpage">3197</span>–<span class="cit-lpage">3201</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1074/jbc.C300520200</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bbiological%2Bchemistry%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.C300520200%26rft_id%253Dinfo%253Apmid%252F14670965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzkvNS8zMTk3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDMvMzEvMjkyNjcyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.15" data-doi="10.1038/cr.2016.152"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gui</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cryo-electron microscopy structures of the SARS-CoV spike glycoprotein reveal a prerequisite conformational state for receptor binding</span>. <abbr class="cit-jnl-abbrev">Cell research</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">119</span>–<span class="cit-lpage">129</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/cr.2016.152</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bresearch%26rft.volume%253D27%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2016.152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2016.152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16 in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.16" data-doi="10.1038/nature17200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kirchdoerfer</span>,  <span class="cit-name-given-names">R. N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Pre-fusion structure of a human coronavirus spike protein</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">531</span>, <span class="cit-fpage">118</span>–<span class="cit-lpage">121</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature17200</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D531%26rft.spage%253D118%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature17200%26rft_id%253Dinfo%253Apmid%252F26935699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature17200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26935699&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F31%2F292672.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.17" data-doi="10.1073/pnas.1707304114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pallesen</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Immunogenicity and structures of a rationally designed prefusion MERS-CoV spike antigen</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">114</span>, <span class="cit-fpage">E7348</span>–<span class="cit-lpage">e7357</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1707304114</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1707304114%26rft_id%253Dinfo%253Apmid%252F28807998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzM1L0U3MzQ4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDMvMzEvMjkyNjcyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18 in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.18" data-doi="10.1128/jvi.01556-17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shang</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cryo-EM structure of porcine delta coronavirus spike protein in the pre-fusion state</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01556-17</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjvi.01556-17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/jvi.01556-17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19 in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.19" data-doi="10.1038/nature16988"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>,  <span class="cit-name-given-names">A. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cryo-electron microscopy structure of a coronavirus spike glycoprotein trimer</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">531</span>, <span class="cit-fpage">114</span>–<span class="cit-lpage">117</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature16988</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D531%26rft.spage%253D114%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature16988%26rft_id%253Dinfo%253Apmid%252F26855426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature16988&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26855426&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F31%2F292672.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20 in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.20" data-doi="10.1038/nsmb.3293"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>,  <span class="cit-name-given-names">A. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Glycan shield and epitope masking of a coronavirus spike protein observed by cryo-electron microscopy</span>. <abbr class="cit-jnl-abbrev">Nature structural &amp; molecular biology</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">899</span>–<span class="cit-lpage">905</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nsmb.3293</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.volume%253D23%26rft.spage%253D899%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.3293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.3293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21 in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.21" data-doi="10.1128/jvi.01628-17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xiong</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Glycan shield and fusion activation of a deltacoronavirus spike glycoprotein fine-tuned for enteric infections</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.01628-17</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjvi.01628-17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/jvi.01628-17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22 in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.22" data-doi="10.1038/ncomms15092"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cryo-EM structures of MERS-CoV and SARS-CoV spike glycoproteins reveal the dynamic receptor binding domains</span>. <abbr class="cit-jnl-abbrev">Nature communications</abbr> <span class="cit-vol">8</span>, <span class="cit-issue">15092</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/ncomms15092</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D8%26rft.spage%253D15092%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms15092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms15092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23 in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-292672v1.23" data-doi="10.1126/science.1116480"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farzan</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Harrison</span>,  <span class="cit-name-given-names">S. C.</span></span> <span class="cit-chapter-title">Structure of SARS coronavirus spike receptor-binding domain complexed with receptor</span>. <span class="cit-source">Science</span> (<span class="cit-publ-loc">New York, N.Y</span>.) <span class="cit-vol">309</span>, <span class="cit-fpage">1864</span>–<span class="cit-lpage">1868</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1116480</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLi%26rft.auinit1%253DF.%26rft.volume%253D309%26rft.issue%253D5742%26rft.spage%253D1864%26rft.epage%253D1868%26rft.atitle%253DStructure%2Bof%2BSARS%2BCoronavirus%2BSpike%2BReceptor-Binding%2BDomain%2BComplexed%2Bwith%2BReceptor%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1116480%26rft_id%253Dinfo%253Apmid%252F16166518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDkvNTc0Mi8xODY0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDMvMzEvMjkyNjcyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24 in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.24" data-doi="10.1016/j.jmb.2017.10.017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lai</span>,  <span class="cit-name-given-names">A. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Millet</span>,  <span class="cit-name-given-names">J. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daniel</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Freed</span>,  <span class="cit-name-given-names">J. H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Whittaker</span>,  <span class="cit-name-given-names">G. R.</span></span> <span class="cit-article-title">The SARS-CoV Fusion Peptide Forms an Extended Bipartite Fusion Platform that Perturbs Membrane Order in a Calcium-Dependent Manner</span>. <abbr class="cit-jnl-abbrev">Journal of molecular biology</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jmb.2017.10.017</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2017.10.017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2017.10.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.25" data-doi="10.1099/vir.0.80955-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fukushi</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Vesicular stomatitis virus pseudotyped with severe acute respiratory syndrome coronavirus spike protein</span>. <abbr class="cit-jnl-abbrev">The Journal of general virology</abbr> <span class="cit-vol">86</span>, <span class="cit-fpage">2269</span>–<span class="cit-lpage">2274</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/vir.0.80955-0</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DFukushi%26rft.auinit1%253DS.%26rft.volume%253D86%26rft.issue%253D8%26rft.spage%253D2269%26rft.epage%253D2274%26rft.atitle%253DVesicular%2Bstomatitis%2Bvirus%2Bpseudotyped%2Bwith%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%2Bspike%2Bprotein%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.80955-0%26rft_id%253Dinfo%253Apmid%252F16033974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.80955-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16033974&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F31%2F292672.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231033300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26 in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.26" data-doi="10.1128/JVI.78.1.216-223.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miura</span>,  <span class="cit-name-given-names">H. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nakagaki</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Taguchi</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">N-terminal domain of the murine coronavirus receptor CEACAM1 is responsible for fusogenic activation and conformational changes of the spike protein</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">78</span>, <span class="cit-fpage">216</span>–<span class="cit-lpage">223</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DMiura%26rft.auinit1%253DH.%2BS.%26rft.volume%253D78%26rft.issue%253D1%26rft.spage%253D216%26rft.epage%253D223%26rft.atitle%253DN-Terminal%2BDomain%2Bof%2Bthe%2BMurine%2BCoronavirus%2BReceptor%2BCEACAM1%2BIs%2BResponsible%2Bfor%2BFusogenic%2BActivation%2Band%2BConformational%2BChanges%2Bof%2Bthe%2BSpike%2BProtein%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.1.216-223.2004%26rft_id%253Dinfo%253Apmid%252F14671103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6Ijc4LzEvMjE2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDMvMzEvMjkyNjcyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27 in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.27" data-doi="10.1073/pnas.0809524106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Belouzard</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chu</span>,  <span class="cit-name-given-names">V. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Whittaker</span>,  <span class="cit-name-given-names">G. R.</span></span> <span class="cit-article-title">Activation of the SARS coronavirus spike protein via sequential proteolytic cleavage at two distinct sites</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">106</span>, <span class="cit-fpage">5871</span>–<span class="cit-lpage">5876</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0809524106</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0809524106%26rft_id%253Dinfo%253Apmid%252F19321428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzE0LzU4NzEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMy8zMS8yOTI2NzIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28 in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.28" data-doi="10.1073/pnas.0503203102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matsuyama</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ujike</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morikawa</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tashiro</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Taguchi</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Protease-mediated enhancement of severe acute respiratory syndrome coronavirus infection</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr> <span class="cit-vol">102</span>, <span class="cit-fpage">12543</span>–<span class="cit-lpage">12547</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0503203102</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0503203102%26rft_id%253Dinfo%253Apmid%252F16116101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzM1LzEyNTQzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDMvMzEvMjkyNjcyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29 in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.29" data-doi="10.1016/j.virol.2017.12.015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Millet</span>,  <span class="cit-name-given-names">J. K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Whittaker</span>,  <span class="cit-name-given-names">G. R.</span></span> <span class="cit-article-title">Physiological and molecular triggers for SARS-CoV membrane fusion and entry into host cells</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">517</span>, <span class="cit-fpage">3</span>–<span class="cit-lpage">8</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2017.12.015</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D517%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2017.12.015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2017.12.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30 in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.30" data-doi="10.1016/j.virusres.2014.11.021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Millet</span>,  <span class="cit-name-given-names">J. K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Whittaker</span>,  <span class="cit-name-given-names">G. R.</span></span> <span class="cit-article-title">Host cell proteases: Critical determinants of coronavirus tropism and pathogenesis</span>. <abbr class="cit-jnl-abbrev">Virus research</abbr> <span class="cit-vol">202</span>, <span class="cit-fpage">120</span>–<span class="cit-lpage">134</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2014.11.021</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2014.11.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2014.11.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31 in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.31" data-doi="10.1371/journal.ppat.1006698"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">e1006698</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1006698</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bpathogens%26rft.volume%253D13%26rft.spage%253De1006698%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32 in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.32" data-doi="10.1016/j.jsb.2005.03.010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Suloway</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Automated molecular microscopy: the new Leginon system</span>. <abbr class="cit-jnl-abbrev">Journal of structural biology</abbr> <span class="cit-vol">151</span>, <span class="cit-fpage">41</span>–<span class="cit-lpage">60</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jsb.2005.03.010</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Bbiology%26rft.stitle%253DJ%2BStruct%2BBiol%26rft.aulast%253DSuloway%26rft.auinit1%253DC.%26rft.volume%253D151%26rft.issue%253D1%26rft.spage%253D41%26rft.epage%253D60%26rft.atitle%253DAutomated%2Bmolecular%2Bmicroscopy%253A%2Bthe%2Bnew%2BLeginon%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jsb.2005.03.010%26rft_id%253Dinfo%253Apmid%252F15890530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jsb.2005.03.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15890530&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F31%2F292672.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230411800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33 in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.33" data-doi="10.1038/nmeth.4193"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">S. Q.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy</span>. <abbr class="cit-jnl-abbrev">Nature methods</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">331</span>–<span class="cit-lpage">332</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.4193</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D14%26rft.spage%253D331%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4193%26rft_id%253Dinfo%253Apmid%252F28250466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28250466&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F31%2F292672.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34 in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.34" data-doi="10.1016/j.jsb.2009.01.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lander</span>,  <span class="cit-name-given-names">G. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Appion: an integrated, database-driven pipeline to facilitate EM image processing</span>. <abbr class="cit-jnl-abbrev">Journal of structural biology</abbr> <span class="cit-vol">166</span>, <span class="cit-fpage">95</span>–<span class="cit-lpage">102</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Bbiology%26rft.stitle%253DJ%2BStruct%2BBiol%26rft.aulast%253DLander%26rft.auinit1%253DG.%2BC.%26rft.volume%253D166%26rft.issue%253D1%26rft.spage%253D95%26rft.epage%253D102%26rft.atitle%253DAppion%253A%2Ban%2Bintegrated%252C%2Bdatabase-driven%2Bpipeline%2Bto%2Bfacilitate%2BEM%2Bimage%2Bprocessing.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jsb.2009.01.002%26rft_id%253Dinfo%253Apmid%252F19263523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jsb.2009.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19263523&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F31%2F292672.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264081000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35 in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.35" data-doi="10.1016/j.jsb.2009.01.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Voss</span>,  <span class="cit-name-given-names">N. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yoshioka</span>,  <span class="cit-name-given-names">C. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Radermacher</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Potter</span>,  <span class="cit-name-given-names">C. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Carragher</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">DoG Picker and TiltPicker: software tools to facilitate particle selection in single particle electron microscopy</span>. <abbr class="cit-jnl-abbrev">Journal of structural biology</abbr> <span class="cit-vol">166</span>, <span class="cit-fpage">205</span>–<span class="cit-lpage">213</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Bbiology%26rft.stitle%253DJ%2BStruct%2BBiol%26rft.aulast%253DVoss%26rft.auinit1%253DN.%2BR.%26rft.volume%253D166%26rft.issue%253D2%26rft.spage%253D205%26rft.epage%253D213%26rft.atitle%253DDoG%2BPicker%2Band%2BTiltPicker%253A%2Bsoftware%2Btools%2Bto%2Bfacilitate%2Bparticle%2Bselection%2Bin%2Bsingle%2Bparticle%2Belectron%2Bmicroscopy.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jsb.2009.01.004%26rft_id%253Dinfo%253Apmid%252F19374019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jsb.2009.01.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19374019&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F31%2F292672.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36 in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.36" data-doi="10.1093/bioinformatics/btx500"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berndsen</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bowman</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jang</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ward</span>,  <span class="cit-name-given-names">A. B.</span></span> <span class="cit-article-title">EMHP: an accurate automated hole masking algorithm for single-particle cryo-EM image processing</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> (Oxford, England) <span class="cit-vol">33</span>, <span class="cit-fpage">3824</span>–<span class="cit-lpage">3826</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btx500</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D3824%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37 in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.37" data-doi="10.1016/j.jsb.2015.11.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">Gctf: Real-time CTF determination and correction</span>. <abbr class="cit-jnl-abbrev">Journal of structural biology</abbr> <span class="cit-vol">193</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jsb.2015.11.003</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Bbiology%26rft.volume%253D193%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jsb.2015.11.003%26rft_id%253Dinfo%253Apmid%252F26592709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jsb.2015.11.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26592709&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F31%2F292672.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38 in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.38" data-doi="10.7554/eLife.18722"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kimanius</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Forsberg</span>,  <span class="cit-name-given-names">B. O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scheres</span>,  <span class="cit-name-given-names">S. H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lindahl</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">Accelerated cryo-EM structure determination with parallelisation using GPUs in RELION-2</span>. <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">5</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.7554/eLife.18722</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.18722%26rft_id%253Dinfo%253Apmid%252F27845625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.18722&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27845625&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F31%2F292672.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39 in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.39" data-doi="10.1107/S0907444904019158"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Emsley</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Cowtan</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">Coot: model-building tools for molecular graphics</span>. <abbr class="cit-jnl-abbrev">Acta crystallographica. Section D, Biological crystallography</abbr> <span class="cit-vol">60</span>, <span class="cit-fpage">2126</span>–<span class="cit-lpage">2132</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1107/s0907444904019158</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.stitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.aulast%253DEmsley%26rft.auinit1%253DP.%26rft.volume%253D60%26rft.issue%253DPt%2B12%2BPt%2B1%26rft.spage%253D2126%26rft.epage%253D2132%26rft.atitle%253DCoot%253A%2Bmodel-building%2Btools%2Bfor%2Bmolecular%2Bgraphics.%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0907444904019158%26rft_id%253Dinfo%253Apmid%252F15572765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1107/S0907444904019158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15572765&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F31%2F292672.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225360500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40 in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.40" data-doi="10.1038/nmeth.3286"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">DiMaio</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Atomic-accuracy models from 4.5-A cryo-electron microscopy data with density-guided iterative local refinement</span>. <abbr class="cit-jnl-abbrev">Nature methods</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">361</span>–<span class="cit-lpage">365</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.3286</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3286%26rft_id%253Dinfo%253Apmid%252F25707030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3286&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25707030&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F31%2F292672.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41 in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-292672v1.41" data-doi="10.1107/S0907444909052925"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Adams</span>,  <span class="cit-name-given-names">P. D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <abbr class="cit-jnl-abbrev">Acta crystallographica. Section D, Biological crystallography</abbr> <span class="cit-vol">66</span>, <span class="cit-fpage">213</span>–<span class="cit-lpage">221</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1107/s0907444909052925</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.stitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.aulast%253DAdams%26rft.auinit1%253DP.%2BD.%26rft.volume%253D66%26rft.issue%253DPt%2B2%26rft.spage%253D213%26rft.epage%253D221%26rft.atitle%253DPHENIX%253A%2Ba%2Bcomprehensive%2BPython-based%2Bsystem%2Bfor%2Bmacromolecular%2Bstructure%2Bsolution.%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0907444909052925%26rft_id%253Dinfo%253Apmid%252F20124702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1107/S0907444909052925&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20124702&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F31%2F292672.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273820800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/292672v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F292672v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/292383v1" title="The Organizational Principles of Membranes Below 100 nm: Experimental Breakthroughs Occasion a “Modeling Manifesto”" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 31, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/292672v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/292672v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/292672v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/292672v1" class="active" data-icon-position="" data-hide-link-title="0">Receptor binding and proteolysis do not induce large conformational changes in the SARS-CoV spike</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/90133" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-z-fFncE9aFTd8zu4FO5wygEGqGgiYciywsfe-C1lgP4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="90133" id="node90133--2" data-pisa="biorxiv;292672v1" data-pisa-master="biorxiv;292672" data-seqnum="90133" data-apath="/biorxiv/early/2018/03/31/292672.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Receptor binding and proteolysis do not induce large conformational changes in the SARS-CoV spike</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Robert N.</span> <span class="nlm-surname">Kirchdoerfer</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Nianshuang</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jesper</span> <span class="nlm-surname">Pallesen</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Wrapp</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Hannah L.</span> <span class="nlm-surname">Turner</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Christopher A.</span> <span class="nlm-surname">Cottrell</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Kizzmekia S.</span> <span class="nlm-surname">Corbett</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Barney S.</span> <span class="nlm-surname">Graham</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Jason S.</span> <span class="nlm-surname">McLellan</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Andrew B.</span> <span class="nlm-surname">Ward</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">292672; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/292672 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/292672v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/292672v1&amp;title=Receptor%20binding%20and%20proteolysis%20do%20not%20induce%20large%20conformational%20changes%20in%20the%20SARS-CoV%20spike" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/292672v1&amp;title=Receptor%20binding%20and%20proteolysis%20do%20not%20induce%20large%20conformational%20changes%20in%20the%20SARS-CoV%20spike" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/292672v1&amp;text=Receptor%20binding%20and%20proteolysis%20do%20not%20induce%20large%20conformational%20changes%20in%20the%20SARS-CoV%20spike" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/292672v1&amp;title=Receptor%20binding%20and%20proteolysis%20do%20not%20induce%20large%20conformational%20changes%20in%20the%20SARS-CoV%20spike" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/292672v1&amp;t=Receptor%20binding%20and%20proteolysis%20do%20not%20induce%20large%20conformational%20changes%20in%20the%20SARS-CoV%20spike" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/292672v1&amp;title=Receptor%20binding%20and%20proteolysis%20do%20not%20induce%20large%20conformational%20changes%20in%20the%20SARS-CoV%20spike" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/292672v1&amp;title=Receptor%20binding%20and%20proteolysis%20do%20not%20induce%20large%20conformational%20changes%20in%20the%20SARS-CoV%20spike" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="90133" id="node90133--3" data-pisa="biorxiv;292672v1" data-pisa-master="biorxiv;292672" data-seqnum="90133" data-apath="/biorxiv/early/2018/03/31/292672.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Receptor binding and proteolysis do not induce large conformational changes in the SARS-CoV spike</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Robert N.</span> <span class="nlm-surname">Kirchdoerfer</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Nianshuang</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jesper</span> <span class="nlm-surname">Pallesen</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Wrapp</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Hannah L.</span> <span class="nlm-surname">Turner</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Christopher A.</span> <span class="nlm-surname">Cottrell</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Kizzmekia S.</span> <span class="nlm-surname">Corbett</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Barney S.</span> <span class="nlm-surname">Graham</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Jason S.</span> <span class="nlm-surname">McLellan</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Andrew B.</span> <span class="nlm-surname">Ward</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">292672; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/292672 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Receptor%20binding%20and%20proteolysis%20do%20not%20induce%20large%20conformational%20changes%20in%20the%20SARS-CoV%20spike&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Kirchdoerfer%2C+Robert+N.&amp;rft.au=Wang%2C+Nianshuang&amp;rft.au=Pallesen%2C+Jesper&amp;rft.au=Wrapp%2C+Daniel&amp;rft.au=Turner%2C+Hannah+L.&amp;rft.au=Cottrell%2C+Christopher+A.&amp;rft.au=Corbett%2C+Kizzmekia+S.&amp;rft.au=Graham%2C+Barney+S.&amp;rft.au=McLellan%2C+Jason+S.&amp;rft.au=Ward%2C+Andrew+B."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/90133/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/90133/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/90133/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/90133/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/90133/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/90133/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/90133/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/90133/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/90133/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/90133/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/90133/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/90133/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/292672v1&amp;count=horizontal&amp;via=&amp;text=Receptor%20binding%20and%20proteolysis%20do%20not%20induce%20large%20conformational%20changes%20in%20the%20SARS-CoV%20spike&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/292672v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/292672v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/292672v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/biochemistry" class="highlight" data-icon-position="" data-hide-link-title="0">Biochemistry<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
