<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Viruses</span><span class="iso-abbrev" title="iso-abbrev">Viruses</span><span class="publisher-id" title="publisher-id">viruses</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Viruses</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1999-4915</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">MDPI</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7232257">7232257</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3390/v12040446">10.3390/v12040446</a></span><span class="publisher-id" title="publisher-id">viruses-12-00446</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Conference Report</span></div> <div class="title-group"> <div class="article-title" title="article-title">
"French Phage Network�? Annual Conference—Fifth Meeting Report</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laumay</span><span class="given-names" tagx="given-names" title="given-names">Floriane</span></span><a href="#af1-viruses-12-00446">1</a><a href="#fn1-viruses-12-00446">†</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chaïb</span><span class="given-names" tagx="given-names" title="given-names">Amel</span></span><a href="#af2-viruses-12-00446">2</a><a href="#fn1-viruses-12-00446">†</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linares</span><span class="given-names" tagx="given-names" title="given-names">Romain</span></span><a href="#af3-viruses-12-00446">3</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-4382-0434</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Breyton</span><span class="given-names" tagx="given-names" title="given-names">Cécile</span></span><a href="#af3-viruses-12-00446">3</a><a href="#c1-viruses-12-00446">*</a></span></div><span class="citation_author_institution" id="af1-viruses-12-00446">[1], </span><span class="citation_author_institution" id="af2-viruses-12-00446">[2], </span><span class="citation_author_institution" id="af3-viruses-12-00446">[3], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span>Correspondence: <span class="email" tagx="email" title="email">Cecile.Breyton@ibs.fr</span></div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">†</span><p>These authors contributed equally to this work.</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-4-4</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-4</span></span><span class="volume" tagx="volume" title="volume">12</span><span class="issue" tagx="issue" title="issue">4</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">446</span><span class="history" title="history"><span class="received" title="received">received: 2020-4-05</span><span class="accepted" title="accepted">accepted: 2020-4-09</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>).</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Attracting about 100 participants, the fifth edition of our French Phages.fr annual conference was once more a success. This year’s conference took place at the Institute for Structural Biology on the European Electron and Photon Campus in Grenoble, 8–9 October 2019. Similar to previous years, our meeting gathered scientists mainly working in France, from academic labs and hospitals as well as from industry. We also had the pleasure of welcoming attendees from different European countries and even beyond. The conference was divided into four sessions: Ecology and Evolution, Phage Therapy and Biotechnology, Structure and Assembly and Phage–Host Interaction, each opened by a keynote lecture. The talks, selected from abstracts, gave the opportunity for young scientists (especially students and post-docs) to present their project and results in a friendly atmosphere. Poster sessions also favoured interactions and discussions between young researchers and more senior scientists. Here, we provide a summary of the topics developed during the conference.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
bacteriophages</div> <div class="kwd" title="kwd">
Ecology and Evolution</div> <div class="kwd" title="kwd">
Phage Therapy and Biotechnology</div> <div class="kwd" title="kwd">
Phage–Structure and Assembly</div> <div class="kwd" title="kwd">
Host Interaction</div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>Bacteriophages or phages, viruses that infect bacteria, are ubiquitous on Earth. Discovered just about 100 years ago, their potential to cure infectious diseases, later called phage therapy, was soon applied [<a href="#B1-viruses-12-00446">1</a>,<a href="#B2-viruses-12-00446">2</a>]. During the first half of the 20th century the study of phage by physicists interested in understanding the basis of life, lead to the birth of a new scientific field, Molecular Biology (for a review [<a href="#B3-viruses-12-00446">3</a>]). During the second half of the 20th century however, phage research declined, due to the increasing number of antibiotics and because many scientists shifted their interest toward more "complex�? living systems. Nowadays, with the alarming increase of animal and human pathogen resistance to antibiotics [<a href="#B4-viruses-12-00446">4</a>], a renewed interest for phage therapy is observed. Together with the rising awareness of the importance of phages in all ecosystems (e.g. [<a href="#B5-viruses-12-00446">5</a>,<a href="#B6-viruses-12-00446">6</a>,<a href="#B7-viruses-12-00446">7</a>]), the potentialities of phages in biotechnology (e.g. [<a href="#B8-viruses-12-00446">8</a>,<a href="#B9-viruses-12-00446">9</a>,<a href="#B10-viruses-12-00446">10</a>]), the progress in structural biology (e.g. [<a href="#B11-viruses-12-00446">11</a>,<a href="#B12-viruses-12-00446">12</a>]), and the fascinating complexity of phage ecology and evolution (e.g. [<a href="#B13-viruses-12-00446">13</a>,<a href="#B14-viruses-12-00446">14</a>]), the study of phages is back to the frontlines in a wide variety of scientific fields.</p> <p>The Phages.fr network aims at federating researchers mainly located in France who use bacteriophages or their applications in their research, in a very large range of projects (ecology and evolution, molecular mechanisms of infection, genetic regulation, industrial and therapeutic applications, etc.). Our yearly meeting gives them the opportunity to interact, discuss, share experiences and initiate new collaborations. This year, this community met at the Institute for Structural Biology, on the European Photon and Neutron Campus in Grenoble, 8–9 October 2019 (<a href="#viruses-12-00446-f001">Figure 1</a>). Attracting a total of 98 attendees coming from all over France, but also from Germany, Italy, Belgium, Switzerland, United Kingdom, Austria, Spain, Czech Republic, Hungary, Turkey and Canada, the international exposition of this annual meeting has expanded. However, we deeply regret that a Turkish colleague, coming from Italy, was refused crossing of the French border, thus, she could not attend the meeting and present her topic. The four sessions (21 talks that were selected from abstracts), all chaired by students or post-docs, were opened by a keynote lecture given by guests invited from Austria, France, Germany and the United Kingdom. Discussions were further extended during two speaking sessions (<a href="#viruses-12-00446-t001">Table 1</a>), lunches, dinners and coffee breaks. Half the attendees were either students (29) or post-docs (15), and gender parity was reached at the level of the attendees, with a majority of women speakers (16 versus 9 men). Two private companies, Myriade and Cellexus, presented their technologies related to a rapid method for counting viral particles and a disposable system for the production of phages, respectively.</p> </div> <div class="2.summaryofthescientificsessions" title="sec"> <div class="title" tagx="title" title="title">
2. Summary of the Scientific Sessions</div> <div class="2.1.ecologyandevolution" title="sec"> <div class="title" tagx="title" title="title">
2.1. Ecology and Evolution</div> <p><b>Evelien Adriaenssens</b> (Quadram Institute, Norwich, UK) opened this session, chaired by Damien Piel (PhD student, LM2E, Roscoff, France) and Jack Dorling (PhD student, I2BC, Gif–sur–Yvette, France), with a keynote lecture entitled "Phage Taxonomy: what, how and why?�?. Phage taxonomy recently experienced an important evolution linked to sequencing technologies and metagenomics. Although tailed phages were historically classified into families based on morphological analysis of their tail structure (Myoviridae, Siphoviridae and Podoviridae), for example, thousands of bacteriophage sequences recently deposited in public databases show little to no sequence homology between members of the same family, challenging the current classification scheme. Being chair of the ICTV (International Committee on Taxonomy of Viruses) Bacterial and Archaeal Viruses Subcommittee, Evelien described how the subcommittee is currently reorganizing the morphology-based families in favour of genome-based ones, using an ensemble of genomic, proteomic and phylogenetic methods. Using the case study of a group of large Myoviruses, the Spounaviruses, now grouped into a new family called Herelleviridae, she presented a large-scale, genome-based taxonomic classification that can robustly be carried into the future, fit for high-throughput and metagenomic analyses. She also discussed the rules for naming newly discovered phage (sub)families and suggested working with taxonomists upon these matters. Finally, she presented a series of useful tools for taxonomy (ViPTree server, GRAVity, vConTACT, VICTOR).</p> <p>Bacteriophages, as an important group of predators of bacteria in the ocean, are thought to play an important role in controlling fine-scale blooms within these communities. Using the case of the oyster pathogen <i>Vibrio crassostreae</i>, <b>Frédérique Le Roux</b> (Ifremer, Marine station of Roscoff, France) wanted to understand whether phages are major drivers of microbial abundance and diversity, and how they evolve in response to <i>Vibrio</i> resistance. Her team performed seawater and oyster tissue time series sampling over several months and at two distant sites (Brest and Sylt) to assess the dynamic of <i>Vibrio</i> and their phages. They showed that their abundances are inversely correlated. Investigating the host range of Vibriophages also strongly suggested local adaptation or co-evolution, the phage populations exhibiting a good connection with <i>Vibrio</i> clones. This work also allowed the characterisation of broad host phages to be used for the rational design of phage cocktails against oyster pathogens.</p> <p>Phages and plasmids are two major groups of mobile genetic elements (MGEs) shaping bacterial evolution. Although literature distinguishes them clearly, several MGEs were reported to be plasmids encoding <i>bona fide</i> phage genes (or <i>vice versa</i>), enterobacteriophages P1 and N15 being the most prominent examples of the so-called "phage–plasmids�?. <b>Eugen Pfeifer</b> (research fellow, Institut Pasteur, Paris, France) presented work aiming at better understanding those MGEs through the study of known phage–plasmids, the identification of novel ones and the evaluation of their impact on bacterial evolution. The screening of phage and plasmid databases for typical phage characteristics in plasmids and plasmid features in phages resulted in the prediction of around 500 putative phage-plasmids. Genome-based clustering allowed the determination of 29 phage–plasmid families, each containing two to fifty members, amongst which a few encode many virulence and resistance traits. Future work will aim at understanding the evolution of these phage–plasmids.</p> <p>Lactic acid bacteria (LAB) as <i>Streptococcus thermophilus</i> are widely used for industrial food fermentation, but also generate great interest for their food-grade expression capabilities (proteins or metabolites) [<a href="#B15-viruses-12-00446">15</a>,<a href="#B16-viruses-12-00446">16</a>]. To do so, replicating plasmids are often used to introduce genetic material into LAB cells [<a href="#B17-viruses-12-00446">17</a>,<a href="#B18-viruses-12-00446">18</a>], but these plasmids need to be maintained into the cells. This can be achieved by several methods, including recently the use of CRISPR–Cas technology [<a href="#B19-viruses-12-00446">19</a>,<a href="#B20-viruses-12-00446">20</a>]. <b>Cécile Philippe</b> (postdoctoral fellow, Laval University, Quebec City, Canada) demonstrated that streptococcal phage proteins also can be used to increase plasmid stability in LAB cultures and, thus, could be a new tool for food-grade vector applications. Moreover, she showed that this plasmid stability phenotype was not strain-dependant, using various industrial strains.</p> <p><b>Marie–Agnès Petit</b> (MICALIS, Jouy–en–Josas, France) and her team previously showed that antibiotic resistance genes (ARG) were infrequent in phage genome and virome contigs [<a href="#B21-viruses-12-00446">21</a>]. Conversely, recent studies suggested that pig fecal samples could be rich in viral ARG, although these conclusions could be impaired due to bacterial DNA contamination [<a href="#B22-viruses-12-00446">22</a>,<a href="#B23-viruses-12-00446">23</a>]. Fourteen pig (five piglets and nine adults) fecal samples were processed to obtain clean and pure pig viromes. Subsequent sequencing led to the conclusion that phage contigs reconstructed from these pig viromes were indeed poor in ARG, although some were found on phage contigs from four of the piglet samples. Also, Inoviridae appeared to be much more prevalent in viromes than previously thought [<a href="#B24-viruses-12-00446">24</a>], especially for pig fecal samples.</p> <p>The <i>Ralstonia solanacearum</i> species complex causes one of the most important plant diseases in the world, bacterial wilt. <b>Clara Torres–Barceló</b> (INRA–PACA, Monfavet, France) presented a study aiming at isolating and characterising new <i>Ralstonia</i> spp. phages to be used as biocontrol tools. The work also addressed evolutionary questions on the host range of phages, regarding a potential trade-off between the generality and efficiency of phages at controlling their hosts. Forty-two phages were isolated from agricultural samples in Mauritius and the Reunion islands and their host range was tested on 64 <i>Ralstonia</i> spp. strains, from local or international origins. Although phages preferentially attacked bacterial strains present in the islands, 18 of them were able to multiply in bacteria never detected in the islands. Phages also exhibited variations in their efficacy at decreasing bacterial growth. Finally, Reunion phages appeared to be quite specialised and efficient against genetically homogeneous local <i>Ralstonia</i> spp. strains, whereas Mauritius ones were more generalised but not as competent at controlling all of them. The results of this study suggest that the host range should not be the only selection criteria for phage therapy and that they should integrate the phylogenetic information of the targeted bacteria.</p> <p>To enable a phage epidemic to occur, phages need to bypass the host immunity, especially the widespread prokaryotic adaptive immune system CRISPR–Cas. <b>Hélène Chabas</b> (research fellow, ETH Zürich, Switzerland) investigated why, in some cases, phage infection results in large epidemics or in other cases in phage extinction. Using a stochastic epidemiological model describing CRISPR–phage interactions, she determined that, in the absence of phage evolution, higher probabilities of CRISPR-resistance acquisition increase the probability of phage extinction. However, when phage can escape CRISPR through single mutations, the probability of phage extinction is bimodal: below a certain epidemiological tipping point, phages always remain in the population, whereas above this point, phages are always led to extinction. She then explored whether phages can increase their probability of survival by increasing their mutation probability and found that this has only a minimal effect.</p> </div> <div class="2.2.phagetherapyandbiotechnologysession" title="sec"> <div class="title" tagx="title" title="title">
2.2. Phage Therapy and Biotechnology Session</div> <p>The high specificity of phage–host interactions has rapidly led to applications in the agricultural and food industry, as well as in the human clinics. This session was dedicated to the use of phages for human health and activities, and was chaired by <b>Fernando Clavijo</b> (PhD student, Laboratory of Bacterial Chemistry, Marseille, France).</p> <p>It opened with a keynote lecture by <b>Claire Geslin</b> (LM2E, Brest (UBO), France). Phages are a driving force in many evolutionary processes, even in extreme conditions such as deep-sea hydrothermal vents. Only a few viruses have been isolated from microorganisms living in these environments. Geslin’s group has characterised some of them. PAV1 and TPV1 are two lemon-shaped viruses from the Thermococcales order that infect hyperthermophilic anaerobic archaea; they have two homologous genes encoding proteins containing a concanavalin A-like lectin/glucanase domain, probably involved in virus–host recognition. MFV1 is the first head–tailed virus isolated from a deep-sea hyperthermophilic archaea. To contrast, MPV1 infects a thermophilic, anaerobic and piezophilic bacterium, and is able to package a 13.3 kb plasmid originating from its bacterial host (lateral gene transfer or molecular piracy events). Overall, Claire provided evidences of a morphological and genetic diversity among viruses of (hyper)thermophilic archaea and bacteria, with a prevalence of lysogeny that might contribute to viral DNA stability under extreme conditions [<a href="#B25-viruses-12-00446">25</a>].</p> <p><b>David Olivenza</b> (PhD student, Departamento de Genética, Sevilla, Spain) talked about the development of biosensors for phage detection and phage receptor discrimination. The rapid detection of phages may have important implications in many fields, such as assessing the presence of phages in industrial processes and following up the evolution of phage population during and after therapeutic treatment; however, only a few methods are available to detect specific phages in a given sample. Olivenza’s work consisted of using opvAB-<i>gfp</i>-based construction as a biosensor for detecting phages using the O-antigen as a receptor. <i>Salmonella enterica</i> opvAB operon allows the production of short chained-LPS and provides resistance to phages recognizing LPS as a receptor. Upon phage infection, cells expressing opvAB operon are the ones to be selected. The fusion of the operon with a <i>gfp</i> reporter gene allowed the high sensitivity monitoring of LPS-binding phages in various media.</p> <p>The session continued with a presentation of <b>Ildikó Nagy</b> (Enviroinvest Environmental and Biotechnological Corporation, Hungary). The bacterial leaf blight is responsible for important economic losses in rice production, especially in Asia and in western Africa. The challenge is to find pesticides against the causative agent, <i>Xanthomonas oryzae</i> pv. <i>oryzae</i> (<i>Xoo</i>), which are both effective and safe for the environment. Nagy’s laboratory characterised twelve <i>Xoo</i> bacteriophages from Asia (from water or soil in the Philippines and Vietnam) and from France (from a laboratory in Montpellier). Genome-based comparisons allowed the establishment of the phylogenetic relationship among different phages, with the identification of a 322 bp conserved intergenic region. Additionally, all phages were efficient against <i>Xoo</i> from Asia, but not against <i>Xoo</i> from Africa and <i>X. oryzae</i> pv. <i>citri</i> strains.</p> <p>Beyond the presence of multi-resistant bacteria, infections can be complicated to eradicate due to their location, the presence of prosthetic devices or the immunological context. <b>Alexandre Bleibtreu</b> (Hôpital Pitié Salpêtrière, Paris, France) described the first utilisation of phages combined with antibiotics for treating a patient suffering from an extradural empyema. The patient was a 29 year-old woman with a type I neurofibromatosis and a pilocytic astrocytoma (in 2007), further complicated by an extradural empyema. Despite the use of appropriate antibiotics and surgeries, the infection due to methicillin-sensitive <i>Staphylococcus aureus</i> persisted. Clinical and microbiological failure status was declared at the end of 2018. At the request of the patient and her family, she was allowed to receive a personalised cocktail of two phages produced by Pherecydes Pharma, in combination with intravenous injection of dalbavancin. The infection was successfully cured; she has left the hospital a few months after phage administration.</p> <p>This session ended with a lecture by <b>Raphaëlle Delattre</b> (research fellow, Pasteur Institute, Paris, France) presenting a summary of the diverse topics discussed this year by the Comité Scientifique Spécialisé Temporaire de l’Agence Nationale de Sécurité du Médicament et des Produits de Santé about phage therapy. Regarding the compassionate administrations of bacteriophages in France, the ten patients treated were in complex clinical situations, sometimes without an alternative. The majority of them suffered from joint and bone infections due to <i>Pseudomonas aeruginosa</i> or <i>Staphylococcus aureus</i>. The specificities of phage therapy, the difficulty to access phages, the lack of clinical trials (activity and fate of phages, modes of administration) and the misinformation about phage therapy were evoked. The committee was favourable to the creation of a national platform managing patients’ requests and a phage library, and to the promotion of academic phage production.</p> </div> <div class="2.3.structureandassemblysession" title="sec"> <div class="title" tagx="title" title="title">
2.3. Structure and Assembly Session</div> <p>This small session, chaired by <b>Müge Senarisoy</b> (research fellow, I2BC, Orsay, France), was introduced by the keynote lecture of <b>Stefanie Barbirz</b> (University of Potsdam, Germany), who dissected the biophysics of the interactions between the different families of phages (short-, long flexible- and long contractile-tailed phages) and the outer surface of Gram-negative bacteria, covered by lipopolysaccharide (LPS). Very elegant experiments were performed on either purified LPS preparations or LPS-containing outer-membrane vesicles to explore the effect of LPS composition and environment on the kinetics of binding and of DNA ejection of the different families of phages. Stefanie could show that tail morphology dictated DNA ejection velocity, short tailed phages being much slower than long tailed phages, and that binding and cell wall perforation can be dissociated in short-tailed phages, depending on temperature, whereas it is coupled for the other phages [<a href="#B26-viruses-12-00446">26</a>,<a href="#B27-viruses-12-00446">27</a>].</p> <p>The two following talks were given by researchers from the Institute for Structural Biology, Grenoble, studying the structure of the coli-siphophage T5. <b>Romain Linares</b> (research fellow), presented his latest data on the structural analysis of the tail of phage T5. Taken from cryo grids of purified T5 tails obtained from a capsid Amber mutant [<a href="#B28-viruses-12-00446">28</a>] and data collected on the high-end Krios electron microscope of the ESRF (European Synchrotron Radiation Facility), Romain could calculate a map of the tip (end of the tail tube) of the tail of phage T5 at 3.5 Å resolution (see also [<a href="#B29-viruses-12-00446">29</a>]). Structural analysis shows remarkable structure conservation between the different proteins, but also with proteins of homologous functions in tails of myophages, a type 6 Secretion System, and other phage-derived perforating apparatus.</p> <p>During the vulnerable time during which the bacterium is turned into a viral factory, phages protect their host from over-infection. Regarding T5, this mechanism is mediated by a periplasmic lipoprotein, Llp, targeted to the inner leaflet of the outer membrane, which binds the phage receptor FhuA. Llp’s main biological function probably also is to prevent the inactivation of progeny phage by active receptors present in the outer-membrane debris of lysed cells and increasing their chance of infecting a new host. <b>Séraphine Degroux</b> (PhD student) presented her project aiming at deciphering this immunity mechanism using a structural approach. She already produced the Llp protein as an acylated form and as a soluble form. The proteins behaved properly, auguring well for future studies.</p> </div> <div class="2.4.phage–hostinteractionsession" title="sec"> <div class="title" tagx="title" title="title">
2.4. Phage–Host Interaction Session</div> <p>This large session, chaired first by <b>Maud Billaud</b> (PhD student, MICALIS, Jouy–en–Josas, France) then by <b>Adelaïde Renard</b> (PhD student, Bactéries et risques materno-foetal, Tours, France), started with a keynote lecture by <b>Călin Guet</b> (Institute of Science and Technology, Austria) entitled "The cost of immigration control and the benefits of illegal Immigration�?, a metaphoric way to describe the impact of Restriction–Modification (RM) systems on prophage acquisition. This study lies on the assumption that RMs could be costly for their hosts. Conversely, it was noticed that the more bacterial genomes contain RMs, the more they contain prophages. It, therefore, was assessed whether RM efficacy could differ during lysogenic and lytic infections. Using <i>E. coli</i> and a Lambda phage, it was demonstrated that carrying RMs is costly for the host at the individual level, which relies on self-restriction phenomena. Efficiency of plating and lysogenisation measurements showed that RM activities do not differ within lytic and lysogenic infections, but that, at the population level, RMs promote prophage acquisition. An explanation would be that RMs delay population infection by the phage until bacteria reach high densities, where the lysogenic cycle is predominant [<a href="#B30-viruses-12-00446">30</a>]. Immigration control, therefore, is used for illegal immigration purposes that benefit the host by conferring upon it new traits.</p> <p>The second talk of the session was presented by <b>Marie Vasse</b> (research fellow, ETH, Switzerland), where <i>Myxococcus xanthus</i> was used to study the impact of phages on its host lifecycle. <i>M. xanthus</i> is a bacterium that displays a complex lifecycle alternating between vegetative cells and fruiting bodies that form upon starvation, leading to the production of spores [<a href="#B31-viruses-12-00446">31</a>]. When bacteria are challenged with the MX1 virulent phage, germination is delayed, indicating that spores might sense phage or infected/dead cell components. Moreover, the fruiting bodies aspect can be modified when exposed to different phages.</p> <p>The session continued with a presentation by <b>Pauline Misson</b> (PhD student, MICALIS, Jouy–en–Josas, France) about the impact of <i>E. coli</i> LF82 prophages on the macrophage-induced persistence of their host. <i>E. coli</i> LF82 is frequently associated with the microbiota of Crohn’s disease patients. This strain can survive within macrophages and produce persisters (i.e. phenotypic variants with low metabolic activity and transient resistance to antibiotics) [<a href="#B32-viruses-12-00446">32</a>]. LF82 harbours five prophages. One of them, Gally, a P22-homologue, is most frequently found in <i>E. coli</i> associated with Crohn’s disease [<a href="#B33-viruses-12-00446">33</a>] and seems to have a positive impact on the macrophage-induced persistence of its host <i>E. coli</i> LF82. Three Gally genes were shown to be overexpressed within macrophages, and their inactivation led to a 3,4-fold less persister production. These genes could be implicated in a non-canonical DNA recombination mechanism, combining homologous and non-homologous recombination to promote DNA repair in persister cells.</p> <p><b>Amel Chaïb</b> (PhD student, ISVV Œnologie, University of Bordeaux, France) made a presentation on lysogeny in <i>Oenococcus oeni</i>, the main agent responsible for malolactic fermentation of wine. Although prophage prevalence in <i>O. oeni</i> species is high, its roles on bacterial fitness are still poorly understood. An elegant, colorimetric method, to discriminate lysogenic derivatives of <i>O. oeni</i> was presented (lysogens grow red on red grape juice agar medium, while non-lysogenic clones are white). It simplified the construction of isogenic pairs of strains that do or do not carry prophages for further characterization. Different approaches demonstrated that a white to red colony colour switch is linked to prophage spontaneous induction within the intracolonial population. Red grape polyphenolic compounds were shown to interact with dead cell material. The method was further adapted to other LAB species [<a href="#B34-viruses-12-00446">34</a>].</p> <p><b>Mehdi El Sadek Fadel</b> (PhD student, I2BC, Gif–sur–Yvette, France) then reported a study on the molecular mechanism of selective viral DNA recognition and packaging of bacteriophage SPP1. Encapsidation depends on the recognition of a <i>pac</i> sequence present on the SPP1 genome by the viral terminase [<a href="#B35-viruses-12-00446">35</a>]. Despite the specificity of the terminase for a <i>pac</i> sequence, bacterial DNA encapsidation can occur and lead to generalised transduction. Interactions between SPP1 terminase and <i>pac</i> were characterized through sequential deletions and substitution mutagenesis of the <i>pac</i> region. Results show that a large part of a <i>pac</i> sequence is dispensable. Extensive degenerations of <i>pacL</i> and <i>pacR</i> sub-regions are compensated through suppressor mutations leading to overexpression of the terminase complex or changes in the gp1 protein, respectively, indicating the importance of the concerned sites for viral DNA recognition and cleavage.</p> <p><b>Luis Ramirez</b> (PhD student, I2BC, Orsay, France) presented his work on the <i>E. coli</i> T5 phage early gene function in host takeover. The T5 phage delivers its DNA in two sequential steps into the bacterial cell. First, only 8% of the T5 genome is injected, allowing the expression of 17 early genes, then injection resumes. Products of these early genes cause host genome degradation and deactivate its defence mechanisms. Among them, only A1 and A2, a putative DNase and a putative transcriptional regulator, respectively, are known to be essential [<a href="#B36-viruses-12-00446">36</a>]. As a first step in studying the significance of early genes in T5 infection, genetic engineering strategies were successfully developed to introduce amber mutations, as well as single and multiple gene deletions in T5. Several T5 mutants exhibited changes during the latent period, burst size and host lysis. These data help to identify critical T5 early genes needed for host takeover.</p> <p><b>Naoual Derdouri</b> (PhD student, Laboratoire de Chimie Bactérienne, Marseille, France) then presented her work on the AppY protein, encoded by an <i>E. coli</i> K-12 DLP12 defective prophage. AppY is a transcriptional regulator of the AraC family, also suspected to improve RNA polymerase sigma S factor (RpoS) stability under stress conditions [<a href="#B37-viruses-12-00446">37</a>]. During the exponential growth phase, RpoS interaction with RssB triggers its degradation by the ClpXP protease. When needed, the RpoS degradation pathway is inhibited by Ira proteins that titrate RssB. Mutation within the HTH motif of the AppY regulator indicated that its transcriptional regulator function is not needed for RpoS stabilisation. Different molecular approaches demonstrated that AppY interacts directly with RssB and that the full-length protein is required, its N-terminal conformation is particularly important. AppY might be a new member of the Ira proteins family. This work shows, for the first time, a prophage-encoded protein-stabilising RpoS.</p> <p><b>Audrey Labarde</b> (engineer, I2BC, Gif–sur–Yvette, France) presented her work about the sequential program of phage SPP1 genome replication during infection of <i>Bacillus subtilis</i>. She found that viral replication occurs in a specific focus in the cytoplasm [<a href="#B38-viruses-12-00446">38</a>] and that large quantities of SPP1 DNA are synthesised within the first thirty minutes following infection. This involves the rapid and massive recruitment of the bacterial replisome machinery at this focus. This process is orchestrated by the SPP1 helicase gp40, known to bind the DnaG primase as well as DnaX, a subunit of the DNA polymerase III [<a href="#B39-viruses-12-00446">39</a>,<a href="#B40-viruses-12-00446">40</a>]. Audrey showed that these three partners are found in the viral DNA focus a few minutes after infection and that their concentration increases over time. She finally observed that the viral DNA compartment is dynamic and takes different shapes during the infection process.</p> <p><b>Quentin Lamy–Besnier</b> (PhD student, Pasteur Institute, Paris, France) presented an overview of the utilisation of the Viral Host Range database (VHRdb) developed by his team. The identification of host–virus pairs can be difficult and time-consuming due to the accumulation of information and the variety of appellations in public databases. This lack of uniformity results in publication of host ranges that are often incomplete. Additionally, the results of host range experiences performed routinely in many laboratories are rarely publicly available. The ambition of the VHRdb is to provide a tool for the analysis of the host range of viruses and to become a public resource to centralise the experimental information linking viruses to hosts. The VHRdb will be launched during the first semester of 2020.</p> <p>The session ended with the presentation by <b>Kovacs Tamas</b> (Enviroinvest Environmental and Biotechnological Corporation, Hungary), who proposes a COST Action on bacteriophage research and applications. COST (European Cooperation in Science and Technology) is a funding organisation for research and innovation networks. It aims at financing conferences, workshops, training and short missions between members of the network. Thirty-nine countries are eligible, and at least seven should be included in a COST. COST actions are bottom-up networks, and the new COST Kovacs proposes to setup would include five working groups: Ecology, Evolution and Taxonomy; Phage–Host interaction; Structure (including genome); Applications in Plant Protection; Application in Human and Animal Health. To register, please follow the <span class="uri" title="uri"><a href="www.europhage.eu">www.europhage.eu</a></span> link.</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
3. Conclusions</div> <p>Similar to previous years, our French Phage Network annual conference favoured many discussions and exchanges in a very friendly atmosphere, and the conference covered a wide range of topics, as can be seen in the summary above. A notable trend is the increasing number of different phage–host systems, witnessing a very active and growing community in France. Furthermore, the activity of our network extends beyond those yearly meetings, and several workshops took place including "Viromathon�? organised by Annie Château and Marie–Agnès Petit, 18 October 2018 in Montpellier, "Phage ecology�? organised by Mai Huong Chatain and Xavier Bellanger, 27 March 2019 in Lyon, "Genome sequencing and annotation�? organised by Mireille Ansaldi and Marie–Agnès Petit, 8–9 October 2019 in Grenoble, and "Phage therapy applied to bone and joint infections�? organised by Rémy Froissart and Tristan Ferry, 3 December 2019 in Lyon. During 2020, two workshops are already planned: "Phage Ecology�? in Avignon, by Xavier Bellanger and Clara Tores–Barcelò, 27 March, and "Phage therapy�? in Paris by Rémy Froissart, Aleandre Bleibtreu and Charlotte Brives, 25 June. More details on past and coming workshops are on our web site <span class="uri" title="uri"><a href="http://phages.fr/workshops/">http://phages.fr/workshops/</a></span> Additionally, members of our community were present at a Specialised Scientific Committee of the French National Agency for Drug and Health Product Security to discuss "Phage therapy: feedback from experience and perspectives�? (see talk of R. Delattre). Our 2020 conference is scheduled for 7–8 October, at the Pasteur Institute in Paris.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>The fifth meeting of the French Phage Network (Phages in Grenoble), was supported by IBS (Institute for Structural Biology, Grenoble), CNRS (INEE and INSB institutes) and INRA. IBS acknowledges integration into the Interdisciplinary Research Institute of Grenoble (IRIG, CEA). We thank all the authors that contributed to the meeting.</p> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>All authors drafted, commented and edited the manuscript, which was in addition coordinated by C.B. All authors have read and agreed to the published version of the manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>This research received no external funding.</p> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare no conflict of interest. The funding sponsors had no role in the decision to publish this report.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="B1-viruses-12-00446" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">D’Herelle</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">G.H.</span></span></span><span class="source" tagx="source" title="source">The Bacteriophage, Its Rôle in Immunity</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Williams &amp;amp; Wilkins Company</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Baltimore, MD, USA</span><span class="year" tagx="year" title="year">1922</span></span></li> <li tag="ref"><a name="B2-viruses-12-00446" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">D’Herelle</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malone</span><span class="given-names" tagx="given-names" title="given-names">R.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A Preliminary Report of Work Carried out by the Cholera Bacteriophage Enquiry</span><span class="source" tagx="source" title="source">Indian Med. Gaz.</span><span class="year" tagx="year" title="year">1927</span><span class="volume" tagx="volume" title="volume">62</span><span class="fpage" tagx="fpage" title="fpage">614</span><span class="lpage" tagx="lpage" title="lpage">616</span></span></li> <li tag="ref"><a name="B3-viruses-12-00446" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keen</span><span class="given-names" tagx="given-names" title="given-names">E.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A century of phage research: Bacteriophages and the shaping of modern biology</span><span class="source" tagx="source" title="source">BioEssays News Rev. Mol. Cell. Dev. Biol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">37</span><span class="fpage" tagx="fpage" title="fpage">6</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/bies.201400152">10.1002/bies.201400152</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25521633">25521633</a></span></span></li> <li tag="ref"><a name="B4-viruses-12-00446" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="mixed-article-title" title="mixed-article-title">Antibiotic Resistance</span><span class="comment">Available online: <a href="https://www.who.int/news-room/fact-sheets/detail/antibiotic-resistance">https://www.who.int/news-room/fact-sheets/detail/antibiotic-resistance</a></span><span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">(accessed on 6 December 2019)</span></span></li> <li tag="ref"><a name="B5-viruses-12-00446" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Danovaro</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corinaldesi</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dell’anno</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fuhrman</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Middelburg</span><span class="given-names" tagx="given-names" title="given-names">J.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noble</span><span class="given-names" tagx="given-names" title="given-names">R.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suttle</span><span class="given-names" tagx="given-names" title="given-names">C.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Marine viruses and global climate change</span><span class="source" tagx="source" title="source">FEMS Microbiol. Rev.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">993</span><span class="lpage" tagx="lpage" title="lpage">1034</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1574-6976.2010.00258.x">10.1111/j.1574-6976.2010.00258.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21204862">21204862</a></span></span></li> <li tag="ref"><a name="B6-viruses-12-00446" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manrique</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bolduc</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walk</span><span class="given-names" tagx="given-names" title="given-names">S.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van der Oost</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Vos</span><span class="given-names" tagx="given-names" title="given-names">W.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Young</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Healthy human gut phageome</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">113</span><span class="fpage" tagx="fpage" title="fpage">10400</span><span class="lpage" tagx="lpage" title="lpage">10405</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1601060113">10.1073/pnas.1601060113</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27573828">27573828</a></span></span></li> <li tag="ref"><a name="B7-viruses-12-00446" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suttle</span><span class="given-names" tagx="given-names" title="given-names">C.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Marine viruses—Major players in the global ecosystem</span><span class="source" tagx="source" title="source">Nat. Rev. Microbiol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">801</span><span class="lpage" tagx="lpage" title="lpage">812</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrmicro1750">10.1038/nrmicro1750</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17853907">17853907</a></span></span></li> <li tag="ref"><a name="B8-viruses-12-00446" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dy</span><span class="given-names" tagx="given-names" title="given-names">R.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rigano</span><span class="given-names" tagx="given-names" title="given-names">L.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fineran</span><span class="given-names" tagx="given-names" title="given-names">P.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phage-based biocontrol strategies and their application in agriculture and aquaculture</span><span class="source" tagx="source" title="source">Biochem. Soc. Trans.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">1605</span><span class="lpage" tagx="lpage" title="lpage">1613</span><span class="pub-id"><a href="https://dx.doi.org/10.1042/BST20180178">10.1042/BST20180178</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30514766">30514766</a></span></span></li> <li tag="ref"><a name="B9-viruses-12-00446" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jafari</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamzeh-Mivehroud</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morris</span><span class="given-names" tagx="given-names" title="given-names">M.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dastmalchi</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Exploitation of phage display for the development of anti-cancer agents targeting fibroblast growth factor signaling pathways: New strategies to tackle an old challenge</span><span class="source" tagx="source" title="source">Cytokine Growth Factor Rev.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">54</span><span class="lpage" tagx="lpage" title="lpage">65</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cytogfr.2019.03.002">10.1016/j.cytogfr.2019.03.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30898440">30898440</a></span></span></li> <li tag="ref"><a name="B10-viruses-12-00446" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemire</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yehl</span><span class="given-names" tagx="given-names" title="given-names">K.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">T.K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phage-Based Applications in Synthetic Biology</span><span class="source" tagx="source" title="source">Annu. Rev. Virol.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">453</span><span class="lpage" tagx="lpage" title="lpage">476</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-virology-092917-043544">10.1146/annurev-virology-092917-043544</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30001182">30001182</a></span></span></li> <li tag="ref"><a name="B11-viruses-12-00446" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">V.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fitzpatrick</span><span class="given-names" tagx="given-names" title="given-names">A.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Islam</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maxwell</span><span class="given-names" tagx="given-names" title="given-names">K.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Diverse Impacts of Phage Morons on Bacterial Fitness and Virulence</span><span class="source" tagx="source" title="source">Advances in Virus Research</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Amsterdam, The Netherlands</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">Volume 103</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">31</span></span></li> <li tag="ref"><a name="B12-viruses-12-00446" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Margolin</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Molineux</span><span class="given-names" tagx="given-names" title="given-names">I.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural remodeling of bacteriophage T4 and host membranes during infection initiation</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">112</span><span class="fpage" tagx="fpage" title="fpage">E4919</span><span class="lpage" tagx="lpage" title="lpage">E4928</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1501064112">10.1073/pnas.1501064112</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26283379">26283379</a></span></span></li> <li tag="ref"><a name="B13-viruses-12-00446" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knowles</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silveira</span><span class="given-names" tagx="given-names" title="given-names">C.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bailey</span><span class="given-names" tagx="given-names" title="given-names">B.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barott</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cantu</span><span class="given-names" tagx="given-names" title="given-names">V.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cobián-Güemes</span><span class="given-names" tagx="given-names" title="given-names">A.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coutinho</span><span class="given-names" tagx="given-names" title="given-names">F.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dinsdale</span><span class="given-names" tagx="given-names" title="given-names">E.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Felts</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Furby</span><span class="given-names" tagx="given-names" title="given-names">K.A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Lytic to temperate switching of viral communities</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">531</span><span class="fpage" tagx="fpage" title="fpage">466</span><span class="lpage" tagx="lpage" title="lpage">470</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature17193">10.1038/nature17193</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26982729">26982729</a></span></span></li> <li tag="ref"><a name="B14-viruses-12-00446" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mavrich</span><span class="given-names" tagx="given-names" title="given-names">T.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hatfull</span><span class="given-names" tagx="given-names" title="given-names">G.F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bacteriophage evolution differs by host, lifestyle and genome</span><span class="source" tagx="source" title="source">Nat. Microbiol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">17112</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmicrobiol.2017.112">10.1038/nmicrobiol.2017.112</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28692019">28692019</a></span></span></li> <li tag="ref"><a name="B15-viruses-12-00446" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kong</span><span class="given-names" tagx="given-names" title="given-names">L.-H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiong</span><span class="given-names" tagx="given-names" title="given-names">Z.-Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">Y.-J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ai</span><span class="given-names" tagx="given-names" title="given-names">L.-Z.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of a Panel of Strong Constitutive Promoters from <i>Streptococcus thermophilus</i> for Fine-Tuning Gene Expression</span><span class="source" tagx="source" title="source">ACS Synth. Biol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1469</span><span class="lpage" tagx="lpage" title="lpage">1472</span><span class="pub-id"><a href="https://dx.doi.org/10.1021/acssynbio.9b00045">10.1021/acssynbio.9b00045</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31117359">31117359</a></span></span></li> <li tag="ref"><a name="B16-viruses-12-00446" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tagliavia</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicosia</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Advanced Strategies for Food-Grade Protein Production: A New <i>E. coli</i>/Lactic Acid Bacteria Shuttle Vector for Improved Cloning and Food-Grade Expression</span><span class="source" tagx="source" title="source">Microorganisms</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">116</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/microorganisms7050116">10.3390/microorganisms7050116</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31035573">31035573</a></span></span></li> <li tag="ref"><a name="B17-viruses-12-00446" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">El Demerdash</span><span class="given-names" tagx="given-names" title="given-names">H.A.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heller</span><span class="given-names" tagx="given-names" title="given-names">K.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geis</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Application of the <i>shsp</i> Gene, Encoding a Small Heat Shock Protein, as a Food-Grade Selection Marker for Lactic Acid Bacteria</span><span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">69</span><span class="fpage" tagx="fpage" title="fpage">4408</span><span class="lpage" tagx="lpage" title="lpage">4412</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.69.8.4408-4412.2003">10.1128/AEM.69.8.4408-4412.2003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12902223">12902223</a></span></span></li> <li tag="ref"><a name="B18-viruses-12-00446" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garay-Novillo</span><span class="given-names" tagx="given-names" title="given-names">J.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">García-Morena</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Masó</span><span class="given-names" tagx="given-names" title="given-names">J.�?.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barra</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Del Solar</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Combining Modules for Versatile and Optimal Labeling of Lactic Acid Bacteria: Two pMV158-Family Promiscuous Replicons, a Pneumococcal System for Constitutive or Inducible Gene Expression, and Two Fluorescent Proteins</span><span class="source" tagx="source" title="source">Front. Microbiol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">1431</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2019.01431">10.3389/fmicb.2019.01431</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31297101">31297101</a></span></span></li> <li tag="ref"><a name="B19-viruses-12-00446" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cho</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shin</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cho</span><span class="given-names" tagx="given-names" title="given-names">B.-K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Applications of CRISPR/Cas System to Bacterial Metabolic Engineering</span><span class="source" tagx="source" title="source">Int. J. Mol. Sci.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">19</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">1089</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/ijms19041089">10.3390/ijms19041089</a></span></span></li> <li tag="ref"><a name="B20-viruses-12-00446" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Landete</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A review of food-grade vectors in lactic acid bacteria: From the laboratory to their application</span><span class="source" tagx="source" title="source">Crit. Rev. Biotechnol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">37</span><span class="fpage" tagx="fpage" title="fpage">296</span><span class="lpage" tagx="lpage" title="lpage">308</span><span class="pub-id"><a href="https://dx.doi.org/10.3109/07388551.2016.1144044">10.3109/07388551.2016.1144044</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26918754">26918754</a></span></span></li> <li tag="ref"><a name="B21-viruses-12-00446" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enault</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Briet</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouteille</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roux</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sullivan</span><span class="given-names" tagx="given-names" title="given-names">M.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petit</span><span class="given-names" tagx="given-names" title="given-names">M.-A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phages rarely encode antibiotic resistance genes: A cautionary tale for virome analyses</span><span class="source" tagx="source" title="source">ISME J.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">237</span><span class="lpage" tagx="lpage" title="lpage">247</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ismej.2016.90">10.1038/ismej.2016.90</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27326545">27326545</a></span></span></li> <li tag="ref"><a name="B22-viruses-12-00446" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calero-Cáceres</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balcázar</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bacteriophages as Environmental Reservoirs of Antibiotic Resistance</span><span class="source" tagx="source" title="source">Trends Microbiol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">570</span><span class="lpage" tagx="lpage" title="lpage">577</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tim.2019.02.008">10.1016/j.tim.2019.02.008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30905524">30905524</a></span></span></li> <li tag="ref"><a name="B23-viruses-12-00446" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Debroas</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siguret</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Viruses as key reservoirs of antibiotic resistance genes in the environment</span><span class="source" tagx="source" title="source">ISME J.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">2856</span><span class="lpage" tagx="lpage" title="lpage">2867</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41396-019-0478-9">10.1038/s41396-019-0478-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31358910">31358910</a></span></span></li> <li tag="ref"><a name="B24-viruses-12-00446" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roux</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krupovic</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daly</span><span class="given-names" tagx="given-names" title="given-names">R.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borges</span><span class="given-names" tagx="given-names" title="given-names">A.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nayfach</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schulz</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharrar</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matheus Carnevali</span><span class="given-names" tagx="given-names" title="given-names">P.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">J.-F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ivanova</span><span class="given-names" tagx="given-names" title="given-names">N.N.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cryptic inoviruses revealed as pervasive in bacteria and archaea across Earth’s biomes</span><span class="source" tagx="source" title="source">Nat. Microbiol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">1895</span><span class="lpage" tagx="lpage" title="lpage">1906</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41564-019-0510-x">10.1038/s41564-019-0510-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31332386">31332386</a></span></span></li> <li tag="ref"><a name="B25-viruses-12-00446" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lossouarn</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dupont</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gorlas</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mercier</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bienvenu</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marguet</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Forterre</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geslin</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">An abyssal mobilome: Viruses, plasmids and vesicles from deep-sea hydrothermal vents</span><span class="source" tagx="source" title="source">Res. Microbiol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">166</span><span class="fpage" tagx="fpage" title="fpage">742</span><span class="lpage" tagx="lpage" title="lpage">752</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.resmic.2015.04.001">10.1016/j.resmic.2015.04.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25911507">25911507</a></span></span></li> <li tag="ref"><a name="B26-viruses-12-00446" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Broeker</span><span class="given-names" tagx="given-names" title="given-names">N.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kiele</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Casjens</span><span class="given-names" tagx="given-names" title="given-names">S.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilcrease</span><span class="given-names" tagx="given-names" title="given-names">E.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thalhammer</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koetz</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barbirz</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">In Vitro Studies of Lipopolysaccharide-Mediated DNA Release of Podovirus HK620</span><span class="source" tagx="source" title="source">Viruses</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">289</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/v10060289">10.3390/v10060289</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29843473">29843473</a></span></span></li> <li tag="ref"><a name="B27-viruses-12-00446" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Broeker</span><span class="given-names" tagx="given-names" title="given-names">N.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roske</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valleriani</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stephan</span><span class="given-names" tagx="given-names" title="given-names">M.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andres</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koetz</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heinemann</span><span class="given-names" tagx="given-names" title="given-names">U.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barbirz</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Time-resolved DNA release from an O-antigen–specific <i>Salmonella</i> bacteriophage with a contractile tail</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">294</span><span class="fpage" tagx="fpage" title="fpage">11751</span><span class="lpage" tagx="lpage" title="lpage">11761</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.RA119.008133">10.1074/jbc.RA119.008133</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31189652">31189652</a></span></span></li> <li tag="ref"><a name="B28-viruses-12-00446" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zivanovic</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Confalonieri</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ponchon</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lurz</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chami</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flayhan</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Renouard</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huet</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Decottignies</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davidson</span><span class="given-names" tagx="given-names" title="given-names">A.R.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Insights into bacteriophage T5 structure from the analysis of its morphogenesis genes and protein components</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">88</span><span class="fpage" tagx="fpage" title="fpage">1162</span><span class="lpage" tagx="lpage" title="lpage">1174</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02262-13">10.1128/JVI.02262-13</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24198424">24198424</a></span></span></li> <li tag="ref"><a name="B29-viruses-12-00446" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arnaud</span><span class="given-names" tagx="given-names" title="given-names">C.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Effantin</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vivès</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Engilberge</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bacia</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boulanger</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Girard</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schoehn</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Breyton</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bacteriophage T5 tail tube structure suggests a trigger mechanism for Siphoviridae DNA ejection</span><span class="source" tagx="source" title="source">Nat. Commun.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1953</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-017-02049-3">10.1038/s41467-017-02049-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29209037">29209037</a></span></span></li> <li tag="ref"><a name="B30-viruses-12-00446" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pleška</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lang</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Refardt</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levin</span><span class="given-names" tagx="given-names" title="given-names">B.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guet</span><span class="given-names" tagx="given-names" title="given-names">C.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phage–host population dynamics promotes prophage acquisition in bacteria with innate immunity</span><span class="source" tagx="source" title="source">Nat. Ecol. Evol.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">359</span><span class="lpage" tagx="lpage" title="lpage">366</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41559-017-0424-z">10.1038/s41559-017-0424-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29311700">29311700</a></span></span></li> <li tag="ref"><a name="B31-viruses-12-00446" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pathak</span><span class="given-names" tagx="given-names" title="given-names">D.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wall</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Myxobacterial tools for social interactions</span><span class="source" tagx="source" title="source">Res. Microbiol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">163</span><span class="fpage" tagx="fpage" title="fpage">579</span><span class="lpage" tagx="lpage" title="lpage">591</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.resmic.2012.10.022">10.1016/j.resmic.2012.10.022</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23123306">23123306</a></span></span></li> <li tag="ref"><a name="B32-viruses-12-00446" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Demarre</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prudent</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schenk</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rousseau</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bringer</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barnich</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tran Van Nhieu</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rimsky</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Monte</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Espeli</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The Crohn’s disease-associated <i>Escherichia coli</i> strain LF82 relies on SOS and stringent responses to survive, multiply and tolerate antibiotics within macrophages</span><span class="source" tagx="source" title="source">PLoS Pathog.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">14</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1008123</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1008123">10.1371/journal.ppat.1008123</a></span></span></li> <li tag="ref"><a name="B33-viruses-12-00446" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rakitina</span><span class="given-names" tagx="given-names" title="given-names">D.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manolov</span><span class="given-names" tagx="given-names" title="given-names">A.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanygina</span><span class="given-names" tagx="given-names" title="given-names">A.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garushyants</span><span class="given-names" tagx="given-names" title="given-names">S.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baikova</span><span class="given-names" tagx="given-names" title="given-names">J.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alexeev</span><span class="given-names" tagx="given-names" title="given-names">D.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ladygina</span><span class="given-names" tagx="given-names" title="given-names">V.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kostryukova</span><span class="given-names" tagx="given-names" title="given-names">E.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larin</span><span class="given-names" tagx="given-names" title="given-names">A.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Semashko</span><span class="given-names" tagx="given-names" title="given-names">T.A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genome analysis of <i>E. coli</i> isolated from Crohn’s disease patients</span><span class="source" tagx="source" title="source">BMC Genom.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">18</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">544</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12864-017-3917-x">10.1186/s12864-017-3917-x</a></span></span></li> <li tag="ref"><a name="B34-viruses-12-00446" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chaïb</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Philippe</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaomanjaka</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Claisse</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jourdes</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lucas</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cluzet</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le Marrec</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Lysogeny in the Lactic Acid Bacterium <i>Oenococcus oeni</i> Is Responsible for Modified Colony Morphology on Red Grape Juice Agar</span><span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">85</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.00997-19">10.1128/AEM.00997-19</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31375489">31375489</a></span></span></li> <li tag="ref"><a name="B35-viruses-12-00446" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oliveira</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tavares</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alonso</span><span class="given-names" tagx="given-names" title="given-names">J.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Headful DNA packaging: Bacteriophage SPP1 as a model system</span><span class="source" tagx="source" title="source">Virus Res.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">173</span><span class="fpage" tagx="fpage" title="fpage">247</span><span class="lpage" tagx="lpage" title="lpage">259</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virusres.2013.01.021">10.1016/j.virusres.2013.01.021</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23419885">23419885</a></span></span></li> <li tag="ref"><a name="B36-viruses-12-00446" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davison</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pre-early functions of bacteriophage T5 and its relatives</span><span class="source" tagx="source" title="source">Bacteriophage</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">e1086500</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/21597081.2015.1086500">10.1080/21597081.2015.1086500</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26904381">26904381</a></span></span></li> <li tag="ref"><a name="B37-viruses-12-00446" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bougdour</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wickner</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gottesman</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Modulating RssB activity: IraP, a novel regulator of σS stability in <i>Escherichia coli</i></span><span class="source" tagx="source" title="source">Genes Dev.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">884</span><span class="lpage" tagx="lpage" title="lpage">897</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gad.1400306">10.1101/gad.1400306</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16600914">16600914</a></span></span></li> <li tag="ref"><a name="B38-viruses-12-00446" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martínez-Jiménez</span><span class="given-names" tagx="given-names" title="given-names">M.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mesa</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alonso</span><span class="given-names" tagx="given-names" title="given-names">J.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bacillus subtilis τ subunit of DNA polymerase III interacts with bacteriophage SPP1 replicative DNA helicase G40P</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">5056</span><span class="lpage" tagx="lpage" title="lpage">5064</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkf650">10.1093/nar/gkf650</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12466528">12466528</a></span></span></li> <li tag="ref"><a name="B39-viruses-12-00446" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klein</span><span class="given-names" tagx="given-names" title="given-names">M.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tokonzaba</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holden</span><span class="given-names" tagx="given-names" title="given-names">L.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">X.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The structure of a DnaB-family replicative helicase and its interactions with primase</span><span class="source" tagx="source" title="source">Nat. Struct. Mol. Biol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">94</span><span class="lpage" tagx="lpage" title="lpage">100</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nsmb1356">10.1038/nsmb1356</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18157148">18157148</a></span></span></li> <li tag="ref"><a name="B40-viruses-12-00446" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ayora</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Missich</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mesa</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lurz</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Egelman</span><span class="given-names" tagx="given-names" title="given-names">E.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alonso</span><span class="given-names" tagx="given-names" title="given-names">J.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Homologous-pairing activity of the Bacillus subtilis bacteriophage SPP1 replication protein G35P</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">277</span><span class="fpage" tagx="fpage" title="fpage">35969</span><span class="lpage" tagx="lpage" title="lpage">35979</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M204467200">10.1074/jbc.M204467200</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12124388">12124388</a></span></span></li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Poster of the Phage in Grenoble Conference.</p>   </div> <div class="table-wrap_UNKNOWN" id="viruses-12-00446-t001" orientation="portrait" position="float">
viruses-12-00446-t001_Table 1<span class="label" tagx="label" title="label">Table 1</span> <p>List of topics presented during the French Phage Network in Grenoble in 2019.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Poster Title</th> <th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Authors. Presenters Underlined</th> </tr> <tr> <th colspan="2" align="center" valign="middle" style="border-bottom:solid thin" rowspan="1">Ecology and Evolution Session</th> </tr> </thead> <tbody> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">A molecular ecology approach using Stable Isotope Probing and metagenomics to study viruses of methanogens’ diversity</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><u>H. Ngo</u>, M. Sotomski, M. Krupovic, F. Enault, O. Chapleur, Théodore B., <u>A. Bize</u></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Detection of an archaeal-specific viral family, previously thought to infect only hyperthermophiles, in human gut metaviromes</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><u>H. Ngo</u>, C. Midoux, O. Rué, M. Mariadassou, V. Da Cunha, V. Loux, F. Enault, <u>A. Bize</u></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Genome analysis of <i>Pseudomonas</i> phage PPA2</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ZE. Aynur, E. Oryasin, G. Basbülbül, B. Ertugrul, <u>B. </u><u>Bozdo</u><u>ğ</u><u>an</u></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Proline and arginine metabolism at the interface of stationary phase physiology and bacteriophage infection in <i>Bacillus subtilis</i></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><u>J. Dorling</u>, A. Corral–Lugo, V. Cvirkaite-Krupovic, P. Tavares</td> </tr> <tr> <td colspan="2" align="center" valign="middle" style="border-bottom:solid thin" rowspan="1"><b>Phage Therapy and Biotechnology Session</b></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">DNA-free POETential, a synthetic biology project for the iGEM competition: Repurposing a DNA-less bacterium into an "RNA cell�? with a little help from phages</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">A. Boudigou, C. Diaz, H. Herrmann, N. Moné, L. Maroc, M. Sabeti Azad, P. Bouloc, S. Bury–Moné, <u>O. Rossier</u></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Bacteriophages active against Meticillin Resistant <i>Staphylococci</i> isolated from bovine mastitis infections</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Z. E. Aynur, <u>G. Basbülbül</u>, B. Bozdoğan</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Strategy using phages to control <i>Staphylococcus aureus</i> responsible for bovine mastitis</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><u>M. H. Chatain</u></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">The use of interferometric microscopy to quantify viral particles in complex samples such as fecal filtrates</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><u>R. Sausset</u>, M. Greffet, M–A. Petit, M. De Paepe</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Assessing phage therapy against the plant pest <i>Xylella fastidiosa</i></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><u>F. Clavijo</u>, M–A. Jacques, M. Ansaldi</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Study of the prophages of <i>Pseudomonas aeruginosa</i> strain PP001</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><u>M. Billaud</u>, M–A. Petit, P. Champion–Arnaud</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Optical Bacterial Susceptibility test by Surface Plasmon Resonance (SPR)</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><u>L. O’Connell</u></td> </tr> <tr> <td colspan="2" align="center" valign="middle" style="border-bottom:solid thin" rowspan="1"><b>Structure and Assembly Session</b></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Detection of viral particles in bacterial cultures of <i>Xanthomonas campestris</i> pv. <i>campestris</i></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><u>M. Kocanova</u>, M. Baránek, A. Eichmeier</td> </tr> <tr> <td colspan="2" align="center" valign="middle" style="border-bottom:solid thin" rowspan="1"><b>Phage–Host Interaction Session</b></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Exploring the mechanisms of host takeover by bacteriophage T5: role of the DNA-binding protein A2</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><u>M. Senarisoy</u>, P. Cuniasse, S. Zinn–Justin, P. Boulanger</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Investigation of A2 protein partners, an essential pre-early protein of bacteriophage T5</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><u>A. Djedid</u>, N. Ginet, A. Battesti, M. Ansaldi</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Filamentous phage translocation in <i>Escherichia coli</i> envelope does not require a functional TolQRA motor</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">P. Samire, B. Serrano, D. Duché, E. Lemarié, R. Lloubes, <u>L. Houot</u></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">How bacteria and bacteriophage coexist in the mammalian gut?</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">M. Mansos Lourenco, <u>L. Chaffringeon</u>, Q. Lamy–Besnier, C. Eberl, B. Stacher, L. Debarbieux, L. De Sordi</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Low efficiency of DNA mismatch repair system on lambda phage</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><u>M. De Paepe</u>, J. Cornuault, M. Elez</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">ATP-dependent formation of Sak4 filaments: a first step towards the single strand annealing</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">O. Son, S. Baconnais, M–A. Petit, E. Le Cam, <u>F. Lecointe</u></td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Role of H-NS in maintenance of Gifsy prophages lysogeny in <i>Salmonella enterica</i> ST4/74</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><u>J. A. Bulssico</u>, M. Ansaldi, A. Wahl, A. Boulanger</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Isolation and characterization of the bacteriophages infecting <i>Xanthomonas arboricola</i> pv. <i>juglandis</i></td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><u>I. Altin</u>, K. Gasic, M. Krivokapic, E. Stefani</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">An early expressed <i>Pseudomonas</i> phage protein increases host susceptibility to lysis and antibiotics</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><u>M. De Jode</u>, A. Chevallereau, M. Monot, E. Brambilla, G. Karimova, L. Debarbieux</td> </tr> <tr> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Functional impact of relB-metK region and <i>clpP</i> carried by 12/111phiA prophage on GBS pathogenicity</td> <td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><u>A. Renard</u>, M. Lacasse, S. Dos Santos Borges, N. van der Mee–Marquet</td> </tr> </tbody> </table> </div> </div>  </body></html>