<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Sci Rep</span><span class="iso-abbrev" title="iso-abbrev">Sci Rep</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Scientific Reports</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2045-2322</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6987159">6987159</a></span><span class="publisher-id" title="publisher-id">58135</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41598-020-58135-y">10.1038/s41598-020-58135-y</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Zika virus targets the human thymic epithelium</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Messias</span><span class="given-names" tagx="given-names" title="given-names">Carolina V.</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loss-Morais</span><span class="given-names" tagx="given-names" title="given-names">Guilherme</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carvalho</span><span class="given-names" tagx="given-names" title="given-names">Joseane Biso de</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">González</span><span class="given-names" tagx="given-names" title="given-names">Mariela N.</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cunha</span><span class="given-names" tagx="given-names" title="given-names">Daniela P.</span></span><a href="#Aff5">5</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-2193-2224</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vasconcelos</span><span class="given-names" tagx="given-names" title="given-names">Zilton</span></span><a href="#Aff5">5</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-0109-4809</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arge</span><span class="given-names" tagx="given-names" title="given-names">Luis W. P.</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farias-de-Oliveira</span><span class="given-names" tagx="given-names" title="given-names">Désio A.</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerber</span><span class="given-names" tagx="given-names" title="given-names">Alexandra L.</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Portari</span><span class="given-names" tagx="given-names" title="given-names">Elyzabeth A.</span></span><a href="#Aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferreira</span><span class="given-names" tagx="given-names" title="given-names">Nilma</span></span><a href="#Aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raphael</span><span class="given-names" tagx="given-names" title="given-names">Lidiane M. S.</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-9899-3386</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonaldo</span><span class="given-names" tagx="given-names" title="given-names">Myrna C.</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riederer</span><span class="given-names" tagx="given-names" title="given-names">Ingo</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-2034-0294</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lopes Moreira</span><span class="given-names" tagx="given-names" title="given-names">Maria E.</span></span><a href="#Aff5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cotta-de-Almeida</span><span class="given-names" tagx="given-names" title="given-names">Vinicius</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vasconcelos</span><span class="given-names" tagx="given-names" title="given-names">Ana T. R.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">atrv@lncc.br</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-3697-1506</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mendes-da-Cruz</span><span class="given-names" tagx="given-names" title="given-names">Daniella A.</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">daniella@ioc.fiocruz.br</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Savino</span><span class="given-names" tagx="given-names" title="given-names">Wilson</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">wilson.savino@fiocruz.br</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 0723 0931</span><span class="institution-id" title="institution-id">grid.418068.3</span><span class="institution" title="institution">Laboratory on Thymus Research, Oswaldo Cruz Institute, Oswaldo Cruz Foundation, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 0723 0931</span><span class="institution-id" title="institution-id">grid.418068.3</span><span class="institution" title="institution">National Institute of Science and Technology on Neuroimmunomodulation (INCT-NIM); Oswaldo Cruz Institute, Oswaldo Cruz Foundation, </span></div></span><span class="citation_author_institution" id="Aff3">[3], </span><span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 0723 0931</span><span class="institution-id" title="institution-id">grid.418068.3</span><span class="institution" title="institution">Laboratory of Molecular Biology of Flavivirus, Oswaldo Cruz Institute, Oswaldo Cruz Foundation, </span></div></span><span class="citation_author_institution" id="Aff5">[5], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 0723 0931</span><span class="institution-id" title="institution-id">grid.418068.3</span><span class="institution" title="institution">Laboratory of High Complexity, Fernandes Figueira National Institute for the Health of Mother, Child, and Adolescent, Oswaldo Cruz Foundation, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-1-1</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-1-1</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">1378</span><span class="history" title="history"><span class="received" title="received">received: 2019-9-11</span><span class="accepted" title="accepted">accepted: 2020-1-10</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">Previous work showed that the thymus can be infected by RNA viruses as HIV and HTLV-1. We thus hypothesized that the thymus might also be infected by the Zika virus (ZIKV). Herein we provide compelling evidence that ZIKV targets human thymic epithelial cells (TEC) <i>in vivo</i> and <i>in vitro</i>. ZIKV-infection enhances keratinization of TEC, with a decrease in proliferation and increase in cell death. Moreover, ZIKV modulates a high amount of coding RNAs with upregulation of genes related to cell adhesion and migration, as well as non-coding genes including miRNAs, circRNAs and lncRNAs. Moreover, we observed enhanced attachment of lymphoblastic T-cells to infected TEC, as well as virus transfer to those cells. Lastly, alterations in thymuses from babies congenitally infected were seen, with the presence of viral envelope protein in TEC. Taken together, our data reveals that the thymus, particularly the thymic epithelium, is a target for the ZIKV with changes in the expression of molecules that are relevant for interactions with developing thymocytes.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Subject terms</div> <div class="kwd" title="kwd">
Viral infection</div> <div class="kwd" title="kwd">
Lymphoid tissues</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Brazilian Funding Agency for Studies and Projects (FINEP) #01.16.0078.00; Brazilian National Council for Scientific and Technological Development (CNPq) #303170/2017-4, #306699/2018-4, and #307720/2016-0; Carlos Chagas Filho Foundation for Funding Research in the State of Rio de Janeiro (Faperj) #E-26/202.826/2018; the Oswaldo Cruz Foundation (Fiocruz) and National Laboratory for Scientific Computation; Coordination of Superior Level Staff Improvement (CAPES) - Brasil - Finance Code 001 #23038.010041/2013-13; Mercosur Fund for Structural Convergence (FOCEM, Mercosur #03/11) and the Program of Preventing and Combating the Zika Virus, MCTIC/FNDCT - CNPq/MEC-CAPES/MS-Decit (Grants 426767/2018-7 and 88881.130684/2016-01); Brazilian National Institute of Science and Technology on Neuroimmunomodulation (INCT-NIM) #465489/2014-1.</span></div> </div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="© The Author(s) 2020" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">Zika virus (ZIKV) epidemics in 2015-2016 resulted in devastating effects, causing microcephaly, other related congenital defects at birth and neurodevelopmental delay after two years in children born from mothers infected by the virus during pregnancy<sup><a href="#CR1">1</a>–<a href="#CR4">4</a></sup>. Additionally, ZIKV infection in adults correlated with a rise in the frequency of cases of Guillain-Barré syndrome<sup><a href="#CR5">5</a>,<a href="#CR6">6</a></sup>.</p> <p id="Par3">Although the knowledge on the cellular and molecular alterations caused by the ZIKV in the nervous tissue largely increased in the last few years<sup><a href="#CR2">2</a>,<a href="#CR7">7</a>,<a href="#CR8">8</a></sup>, the effects of this virus upon hematopoietic tissues are much less defined.</p> <p id="Par4">In terms of secondary lymphoid organs, ZIKV RNA and protein have been described in lymph nodes of Rhesus monkeys in both paracortex and germinal centers<sup><a href="#CR9">9</a>,<a href="#CR10">10</a></sup>. The virus was also found in macrophages, dendritic cells, and B-cells, in both spleen and axillary lymph nodes of this non-human primate. However, in the same study, the presence of ZIKV RNA in T-cells was observed only in axillary lymph nodes from one animal<sup><a href="#CR10">10</a></sup>.</p> <p id="Par5">Much less is known on the putative infection of primary lymphoid organs, and more particularly in the thymus. This central lymphoid organ is responsible for the generation of T lymphocytes under the control of the thymic microenvironment, a three-dimensional cellular network mainly composed by thymic epithelial cells – TEC<sup><a href="#CR11">11</a></sup>. In this respect, it is interesting to note the thymus as a target organ for other RNA viruses, such as HIV<sup><a href="#CR12">12</a></sup> and HTLV-1<sup><a href="#CR13">13</a>,<a href="#CR14">14</a></sup>. In fact, we showed that cultured human TEC can be infected by HTLV-1 and convey virus particles to lymphoblastic T cells<sup><a href="#CR15">15</a>,<a href="#CR16">16</a></sup>. We thus hypothesized that the thymic epithelium might also be infected by the Zika virus. Herein we provide compelling evidence that ZIKV targets human TEC both <i>in vivo</i> and <i>in vitro</i>. Data are provided showing that ZIKV-infection enhances keratinization of TEC, with a decrease in proliferation and increase in cell death. Moreover, <i>in vitro</i> data revealed that the virus could modulate a high amount of coding and non-coding genes (including miRNAs, circRNAs, and lncRNAs), with upregulation of genes encoding various cell adhesion and cell migration-related proteins. Accordingly, infected TEC enhanced adhesion of lymphoblastic lymphoma-derived T cells and conveyed the virus to this same cell type. Lastly, alterations in the thymic microenvironment from babies prenatally infected by ZIKV were observed, as well as the presence of the viral envelope protein in the thymus. Taken together, our data reveal that the thymus is a target for the ZIKV and may function as a reservoir of the virus during congenital infection.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="culturedhumanteccanbeinfectedbythezikavirus" title="sec"> <div class="title" tagx="title" title="title">
Cultured human TEC can be infected by the Zika virus</div> <p id="Par6">We first investigated the infectivity and growth capacity in ZIKV-infected human TEC, evaluating the virus yield in cell monolayers. The human postnatal TEC line used was obtained by explant technique and limiting dilution cloning, being derived from fragments of a postnatal thymus from a child undergoing cardiac surgery<sup><a href="#CR17">17</a></sup>. Functionally these cells are able to produce cytokines, chemokines, and extracellular matrix proteins, and can adhere to freshly-isolated thymocytes, as well as acute lymphoblastic leukemia derived T cells<sup><a href="#CR16">16</a>,<a href="#CR18">18</a></sup>. In the first group of experiments, we applied infection doses of 0.1 and 1.0 MOI, with the cells being then harvested 24, 48 and 72 hours post-infection, and subjected to cytofluorometry or immunofluorescence with the 4G2 antibody for intracellular detection of the viral envelope protein. The mouse monoclonal 4G2 antibody recognizes an epitope on the envelope protein conserved in the flavivirus family, including Dengue virus, West Nile virus, Japanese Encephalitis virus and Zika virus<sup><a href="#CR19">19</a></sup>. As shown in Fig. <a href="#Fig1">1a,b</a>, we found a progressive increase in the relative numbers of infected cells, as ascertained by the two applied viral doses. Yet, the use of 1.0 MOI was much more efficient in promoting infection, so that around 90% of the cells were 4G2-positive after 72 hours. Accordingly, all further experiments were done using this infection dose at 72 hours post-infection and MOCK cells were used as control, as the 4G2 antibody present a slight background in MOCK cells when compared to the isotype control. The same percentage of infection was detected by immunofluorescence on adhered cells (Fig. <a href="#Fig1">1c,d</a>), and the presence of the ZIKV envelope protein could be detected in the cytoplasm - especially around the nucleus (Fig. <a href="#Fig1">1e</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Human TEC can be infected <i>in vitro</i> by the Zika virus. (<b>a</b>) TEC were infected with ZIKV (MOI = 0.1 or 1) and the relative expression of 4G2<sup>+</sup> cells was detected 24, 28 and 72 hours post-infection (hpi) by flow cytometry. Each time point represents the mean ± standard error. Asterisks represent statistical significance between MOCK and MOI = 1 (24, 48 and 72 h) and between MOCK and MOI = 0.1 (72 h). Hash marks represent statistical significance between MOI = 0.1 and MOI = 1. (<b>b</b>) Representative histograms of the relative expression of 4G2<sup>+</sup> cells 24 and 72 hpi with MOI = 1. Orange curves represent isotype controls, blue curves represent MOCK and red curves represent ZIKV (n = 3). TEC were infected with ZIKV (MOI = 1) and (<b>c</b>) the percentage of 4G2<sup>+</sup> cells were detected 72 hpi by immunofluorescence. Results are represented by the mean percentage of infection of the three replicates of each independent experiment. Representative images of immunofluorescence for 4G2 (viral protein, in red), cytokeratin (CK, in green) and DAPI (nuclei, in blue) (<b>d</b>) in lower and (<b>e</b>) higher magnifications (n = 3, in triplicates). (<b>f</b>) Representative images of TEC culture 72 hpi (n = 3). (<b>g</b>) ZIKV growth curve in TEC culture. Supernatant of ZIKV-infected TEC culture was harvested, and the presence of infective viral particles was verified in Vero cells. Results are shown as plaque-forming units and each time point represents the mean ± SD, of at least 5 wells (n ≥ 5). (<b>h</b>) Relative expression of 4G2 <sup>+</sup> cells detected by flow cytometry in TEC culture 72 hpi with 1 mL of supernatant from MOCK or ZIKV-infected TEC cultures (n = 3). (<b>i</b>) Relative expression of AXL, TIM1, TIM4 e TYRO3 receptors detected by flow cytometry in TEC (n = 3). (<b>j</b>) Relative expression and (<b>k</b>) Mean Fluorescence Intensity (MFI) of 4G2<sup>+</sup> cells detected by flow cytometry 72 hpi (MOI = 1) after treatment with anti-AXL polyclonal antibody (10 µg/mL) (n = 3). *or p &amp;lt; 0.05, **or p &amp;lt; 0.01, ***p &amp;lt; 0.001, ****or p &amp;lt; 0.0001, (unpaired <i>t</i>-test or one-way ANOVA, followed by Tukey multiple comparison test).</p>   </div> <p /> <p id="Par7">ZIKV infection induced cytopathic effects in TEC with the presence of higher amounts of floating cells in the culture and vacuole-like structures around the nucleus (Fig. <a href="#Fig1">1f</a>).</p> <p id="Par8">We also showed that infection of growing TEC resulted in the generation and release of viral infectious particles in the culture supernatants. Following 24, 48 and 72 hours post-infection, the obtained viral titers from TEC cultures were 4.29 ± 0.16, 6.03 ± 0.25, and 6.96 ± 0.11 log10 PFU/mL, as ascertained in Vero cell cultures (Fig. <a href="#Fig1">1g</a>). Furthermore, infected TEC-derived supernatants were able to convey ZIKV to uninfected TEC cultures (Fig. <a href="#Fig1">1h</a>).</p> <p id="Par9">Since human TEC could be infected by ZIKV, we searched for the membrane expression of well-known ZIKV receptors, namely TIM1, TIM4, TYRO3, and AXL. Actually, cultured human TEC constitutively expresses AXL, but not TIM1, TIM4, TYRO3 receptors (Fig. <a href="#Fig1">1i</a>). Blocking the AXL receptor with the corresponding neutralizing antibody completely prevented TEC infection by ZIKV (Fig. <a href="#Fig1">1j,k</a>), showing that ZIKV uses the AXL receptor to invade human TEC.</p> </div> <div class="zikavirusinfectiondisturbscellgrowthinculturedhumantec" title="sec"> <div class="title" tagx="title" title="title">
Zika virus infection disturbs cell growth in cultured human TEC</div> <p id="Par10">Considering that human TEC were sensitive to ZIKV infection, we wondered whether such an infection could also modulate the growth of these cells <i>in vitro</i>. The TEC counting in infected versus uninfected cultures showed that ZIKV infection induced a progressive reduction in the numbers of growing TEC (Fig. <a href="#Fig2">2a</a>). Experiments aiming at defining whether such a diminution was due to a decrease in cell proliferation or increase in cell death, revealed a diminution in the proportion of dividing cells (defined by the high expression of the Ki67 marker) and an increase in the relative numbers of apoptotic (but not necrotic) cells (Fig. <a href="#Fig2">2b,c</a>), as ascertained by PI and annexin-V double labeling. Differences in the proportion of apoptotic and dividing cells begin to appear 48 hours post-infection, although significant differences are only detected 72 hours post-infection. Of note, in addition to the small, but significant increase in the relative numbers of apoptotic cells, we observed increased amounts of cellular debris in ZIKV-infected TEC cultures when compared to controls (data not shown). It is interesting to note that such modulation of cell growth was not seen when we infected the same human TEC line by HTLV-1 neither in cell proliferation nor cell death<sup><a href="#CR16">16</a></sup>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>Zika virus infection down modulates the growth of cultured human thymic epithelial cells. TECs were infected with ZIKV (MOI = 1) and the (<b>a</b>) numbers of cells in culture were detected 24, 48 and 72 hours post-infection (hpi). Each time point represents the mean ± SEM (n = 3). (<b>b</b>) Percentage of apoptotic cells (annexin-V<sup>+</sup>PI<sup>−</sup>) detected by flow cytometry 24, 48 and 72 hpi. Each time point represents the mean ± SEM (n = 3) (<b>c</b>) Percentage of Ki67 high<sup>+</sup> cells detected by flow cytometry 24, 48 and 72 hpi. Values represent the mean ± SEM (n = 3). **p &amp;lt; 0.01, ***p &amp;lt; 0.001 (unpaired <i>t</i>-test).</p>   </div> <p /> </div> <div class="zikainfectioninducesmajorchangesinhumantecgeneexpressionprofile" title="sec"> <div class="title" tagx="title" title="title">
Zika infection induces major changes in human TEC gene expression profile</div> <p id="Par11">The transcriptome analysis of ZIKV-infected TEC revealed 2,934 differentially expressed genes (DEGs) with p-adjusted &amp;lt; 0.01 and Log<sub>2</sub> Fold Change &amp;gt; |1|, distributed in protein-coding genes, long non-coding RNAs, circRNAs and other RNAs classes (Table <a href="#Tab1">1</a>, Supplemental Table <a href="#MOESM1">1</a>). <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Summary of the differentially Expressed Genes and miRNAs identified by RNAseq and small RNAseq in ZIKV-infected TEC.</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Molecule</th> <th>Expressed Genes</th> <th>Upregulated*</th> <th>Downregulated*</th> <th>Total*</th> </tr> </thead> <tbody> <tr> <td>Protein Coding</td> <td>15935</td> <td>1471</td> <td>106</td> <td><b>1577</b></td> </tr> <tr> <td>Long Non-Coding RNA</td> <td>4113</td> <td>629</td> <td>26</td> <td><b>655</b></td> </tr> <tr> <td>circRNAs</td> <td>13342</td> <td>562</td> <td>91</td> <td><b>653</b></td> </tr> <tr> <td>other RNA Classes**</td> <td>321</td> <td>3</td> <td>46</td> <td><b>49</b></td> </tr> <tr> <td>Total DEGs</td> <td>33711</td> <td>2665</td> <td>269</td> <td><b>2934</b></td> </tr> <tr> <td>miRNAs</td> <td>591</td> <td>42</td> <td>25</td> <td><b>67</b></td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>*Statistical significance of DESEq. 2 with p-value adjusted &amp;lt; 0.01 and Log2 Fold Change &amp;gt; |1.0|; **miscRNA, scaRNA, scRNA, snoRNA and snRNA were included in this category.</p> </div>  <p /> <p id="Par12">The enrichment analysis of DEGs in ZIKV-infected TEC allowed to identify upregulated pathways involved with interleukin and interferon signaling, extracellular matrix organization, integrin cell surface interactions, NCAM1 interactions, NCAM signaling and ECM proteoglycans in Reactome enrichment (Fig. <a href="#Fig3">3a,b</a>, Supplemental Table <a href="#MOESM1">2</a>). The GO enrichment analysis revealed categories also associated with extracellular matrix, response to virus and bacteria, cytokine production, defense response to virus (Supplemental Table <a href="#MOESM1">2</a>). In addition, we found genes with enhanced expression by ZIKV-infected TEC such as CCL4, CCL5, CCL20, CXCL10, GBP2, GBP4, GBP5, GBP6, IFNB1, IFNL1, 2 and 3, IL12B, IL18RAP, MX1, MX2, NOS2, NOD2, OAS2, OASL, RSAD2, and TNF. Other upregulated antiviral genes were found, including IFIH1 and DDX58, which act as cytoplasmic sensors of viral nucleic acids, and DHX58, which mediates antiviral signaling and induction of type I interferons and proinflammatory cytokines (Supplemental Tables <a href="#MOESM1">1</a> and <a href="#MOESM1">2</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <p>Enrichment Analyses of upregulated and downregulated genes from transcriptome of Zika virus-infected TEC. (<b>a</b>) Top 10 significant terms from the Reactome Enrichment Analysis of the upregulated genes. Horizontal red bars indicate the p-adjusted values (−Log<sub>10</sub> scale) from the Reactome Enrichment Analysis. Numbers in the top of the red bars indicate the gene counting of each Reactome Term. (<b>b</b>) Cnetplot showing both upregulated genes and enriched Reactome terms. Pale yellow circles indicate the size of Reactome terms. The Log<sub>2</sub> Fold Changes are indicated by the dot colors. Warmer colors indicate those most upregulated genes. (<b>c</b>) Top 10 significant terms from the Reactome Enrichment Analysis of the downregulated genes. Horizontal blue bars indicate the p-adjusted values (−Log<sub>10</sub> scale) from the Reactome Enrichment Analysis. Numbers in the top of the blue bars indicates the gene counting of each Reactome Term. (<b>d</b>) Cnetplot showing both downregulated genes and enriched Reactome terms. Pale yellow circles indicate the size of the Reactome terms. The Log<sub>2</sub> Fold Changes are indicated by the dot colors. Greener colors indicate the most downregulated genes.</p>   </div> <p /> <p id="Par13">The Enrichment Analysis of Reactome also revealed upregulated genes included in extracellular matrix organization, and recognized as adhesion molecules, such as ADAMTS4, ACAM, ICAM1, ICAM4, VCAM1, ITGAM, PECAM1, CEACAM1, JAM2, MMP8, ITGA9, LAMA2 LAMA3, LAMA4, LAMC3, COL2A1, COL20A1, and COL26A1 (Fig. <a href="#Fig3">3b</a>). Interestingly, the GO enrichment revealed the category named as collagen fibril organization enriched, containing upregulated collagen fibril forming genes (COL2A1, COL3A1, COL5A1, COL5A2 COL5A3, COL11A1, and COL11A2), as shown Fig. <a href="#Fig3">3</a> and Supplemental Table <a href="#MOESM1">2</a>.</p> <p id="Par14">We also found upregulated genes that integrate extracellular matrix organization, interferon signaling, interleukin signaling, and integrin cell surface interaction, including ITGAM, VCAM1, ICAM1 in Reactome enrichment (Fig. <a href="#Fig3">3</a>). The transcriptome results were coherent with the cell growth assay that results in progressive reduction in the numbers of growing TEC, reduced cell proliferation and cell division, with a slight increase of apoptosis after ZIKV infection (Supplemental Table <a href="#MOESM1">2</a> and Fig. <a href="#Fig2">2</a>).</p> <p id="Par15">The Enrichment Analysis also revealed downregulated protein-coding genes related to histone 1 and 2 cluster, ubiquitin, cyclin, E2F Transcription Factor 2 (Fig. <a href="#Fig3">3c</a>, Supplemental Table <a href="#MOESM1">2</a>). The downregulation of FBXO43, CKS2, AURKA, CCNB1, CDC20, UBE2C, CCNF, and CCNA2 genes could also contribute to the decreasing of cell growth (Fig. <a href="#Fig2">2</a>, Supplemental Tables <a href="#MOESM1">1</a> and <a href="#MOESM1">2</a>) corroborating the reduction of cell proliferation <i>in vitro</i>. Moreover, we found downregulated genes (CCNB1, AURKA, PMAIP, and PLK3), which are also associated with the response to DNA damage.</p> </div> <div class="longnon-codingrnasinvolvedwithanantiviralresponseinzikv-infectedtec" title="sec"> <div class="title" tagx="title" title="title">
Long non-coding RNAs involved with an antiviral response in ZIKV-infected TEC</div> <p id="Par16">The DGE analysis also identified 655 differentially expressed long non-coding RNAs (lncRNAs), with 629 upregulated and 26 downregulated in ZIKV-infected TEC (Table <a href="#Tab1">1</a>). Comprising among others, three upregulated lncRNAs, previously related with antiviral response (LUARIS, EGOT, TNFRSF14-AS1), two associated with regulation of apoptosis (PANDAR, MBNL1-AS1), two associated with both apoptosis and proliferation (UCA1, ZEB2-AS1), four with cell proliferation (MDC1-AS1, HOXB-AS3, TFAP2A-AS1, PICART1), one with cell cycle arrest (BDNF-AS) and one with cell adhesion (SENCR). Those lncRNAs showed log2 Fold Change ranging from 1.05 to 5.40 (Fig. <a href="#Fig4">4a</a>, Supplemental Table <a href="#MOESM1">1</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <p>Non-coding genes response in human TEC infected with the Zika virus. (<b>a</b>) Expression of Long non-coding genes related to Immune System, Apoptosis, Apoptosis and Proliferation, Proliferation, Cell Cycle Arrest and Cellular Adhesion. (<b>b</b>) Network formed by MicroRNA and respective target transcripts, showing upregulated microRNAs and down-regulated targets. (<b>c</b>) Network formed by MicroRNA and respective target transcripts, showing down regulated microRNAs and upregulated targets. The Log<sub>2</sub> Fold Changes were indicated by the continuous scaled colors. Warmer colors indicate upregulation, cooler colors indicate down regulation. (<b>d</b>) Expression analysis of Hsa-miR-542-3p and its putative circRNAs targets. (<b>e</b>) Expression analysis of Hsa-miR-92a-1-5p and its putative circRNAs targets. The data are expressed in Log2 Fold Change (Log2FC). Red and blue bars indicate up and downregulated genes, respectively.</p>   </div> <p /> </div> <div class="mirnasandcircrnasexpressionpatternsrevealafinecontroloftranscriptioninzikv-infectedtec" title="sec"> <div class="title" tagx="title" title="title">
miRNAs and circRNAs expression patterns reveal a fine control of transcription in ZIKV-infected TEC</div> <p id="Par17">Small RNA-seq analysis revealed 67 miRNAs differentially expressed in ZIKV-infected TEC, resulting in 42 upregulated and 25 downregulated microRNAs (Table 1 and Supplemental Table <a href="#MOESM1">1</a>). The hsa-miR-7-3-3p was the most upregulated miRNA (log2 FC = 5.16) after ZIKV infection. The majority of miRNAs presented a slightly upregulation or downregulation with log2 FC &amp;gt; |1| including hsa-miR-155-5p (1.55) and hsa-miR-203a-3p (1.19).</p> <p id="Par18">The microRNA-target network (Fig. <a href="#Fig4">4b,c</a>) revealed 13 upregulated microRNAs that can modulate the expression of 12 downregulated target transcripts, including the hsa-miR-3065-5p regulating HIST2H2BE and MYB, hsa-miR-155-5p regulating MYB and E2F2, hsa-miR-449a-5p regulating HIST1H4H and hsa-miR-215-5p regulating CDCA4 (Fig. <a href="#Fig4">4b</a>). This analysis also showed 20 downregulated microRNAs, which target upregulated genes involved with immune system, viral response and extracellular matrix, as shown in hsa-miR-4458-5p that regulates the IL-6, CXCL8, C2, COL3A1, COL5A2, ITGB8, TNFSF9 genes, hsa-miR19b-5p targeting ITGB8 and CXCL8, hsa-miR-100-3p targeting CXCL8 and hsa-miR-222-5p regulating HLA-B and DDX58 genes (Fig. <a href="#Fig4">4c</a>).</p> <p id="Par19">The transcriptome analysis also revealed 562 upregulated and 91 downregulated circRNAs in ZIKV-infected TEC (Table <a href="#Tab1">1</a>). In order to identify the possible role of microRNA sponge by circRNAs, we conducted a microRNA target prediction with miRanda tool, using the circRNAs as targets. This strategy revealed the upregulated hsa-miR-542-3p targeting 3 circRNAs (Fig. <a href="#Fig4">4d</a>) and the downregulated hsa-miR-92a-1-5p, possibly regulated by 12 circRNAs (Fig. <a href="#Fig4">4e</a>).</p> <p id="Par20">Taken together, the transcriptome and small RNA-seq analyses revealed complex molecular responses of TEC in response to ZIKV infection, showing an intricate regulatory network, including different layers of transcription regulation. In this context, ICAM-1 and VCAM appeared as a main hub connecting distinct clusters of genes whose expression was altered secondary to ZIKV infection.</p> </div> <div class="co-culturesofzikv-infectedtecwithlymphoblastictcellsenhanceheterocellularcelladhesionandvirustransfer" title="sec"> <div class="title" tagx="title" title="title">
Co-cultures of ZIKV-infected TEC with lymphoblastic T cells enhance heterocellular cell adhesion and virus transfer</div> <p id="Par21">Considering that the expression of cell adhesion related-genes was enhanced in ZIKV-infected TEC, and the fact that ICAM-1 as well as VCAM are adhesion molecules known to play a role in TEC thymocyte interaction<sup><a href="#CR20">20</a></sup>, we investigated whether these proteins were upregulated in the membrane of ZIKV-infected TEC. We found enhanced numbers of TEC expressing ICAM-1 (CD54 – data not shown), as well as enhanced ICAM-1 in infected cells (Fig. <a href="#Fig5">5a</a>). This was more clear-cut when we analyzed the cells expressing high amounts of ICAM-1 (5.2% (±1.2) of ZIKV infected cells and 2.6% (±0.5) for controls (p = 0.05). As regards the expression of VCAM-1 (CD106), we did find higher membrane expression levels in infected cultures, although the differences were not statistically significant (p-value = 0.1). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <p>Zika virus infection in human cultured epithelial cells enhances cell adhesion and conveys the virus to lymphoblastic T cells. (<b>a</b>) Human TEC were infected with ZIKV (MOI = 1) and 72 hpi the expression of ICAM-1 (CD54) and VCAM-1 (CD106) were analyzed by flow cytometry. Graphs show the Mean Fluorescence Intensity (MFI) of these molecules. MFI values were normalized by the mean of the control group (n = 3). Human TEC were infected with ZIKV (MOI = 1) and 72 hpi uninfected CEM acute T-cell leukemia cell line was added to the culture. (<b>b</b>) Association index between CEM and TEC after 1 h of co-culture (adhesion). Values in the upper panel represent mean ± SEM. Representative microscopic fields after Giemsa staining are shown in the lower panel (n = 4). Relative expression of 4G2<sup>+</sup> cells in (<b>c</b>) TEC and (<b>d</b>) CEM subpopulations were detected by flow cytometry after 24 h of co-culture. Values represent the mean ± SEM (n = 3, two experiments in duplicate). (<b>e</b>) CEM cells were not directly infected by ZIKV (MOI = 1). CEM cells were infected with ZIKV (MOI = 1) for 2 h and 72 hpi 4G2<sup>+</sup> cells were analyzed by flow cytometry. Values represent the mean ± SEM (n = 3). *p &amp;lt; 0.05, **p &amp;lt; 0.01, ****p &amp;lt; 0.0001 (as determined by the unpaired <i>t</i>-test).</p>   </div> <p /> <p id="Par22">We searched for a functional change on the ability of infected TEC to bind to lymphoblastic T cells. For that, the CEM acute T-cell leukemia cell line, previously known to adhere to human TEC<sup><a href="#CR16">16</a></sup>, was led to adhere to growing TEC 72 hours post-infection. Figure <a href="#Fig5">5b</a> shows that adhesion was significantly increased onto ZIKV-infected TEC cultures, as compared to controls. Importantly, we showed that 72 hours after TEC infection plus 24 hours of co-culture with CEM, a proportion (10.76% ± 1.98) of the T lymphoblasts were infected as revealed by immunolabeling with the 4G2 antibody (Fig. <a href="#Fig5">5d</a>). At this time point (96 hours post-infection), 86% of the TEC present in the co-culture were infected (Fig. <a href="#Fig5">5c</a>). Interestingly enough, we could not directly infect CEM cells with ZIKV (Fig. <a href="#Fig5">5e</a>) or with Zika virus-infected TEC supernatant (data not shown).</p> </div> <div class="thehumanthymicepitheliumisalteredaftercongenitalinvivozikvinfection" title="sec"> <div class="title" tagx="title" title="title">
The human thymic epithelium is altered after congenital <i>in vivo</i> ZIKV infection</div> <p id="Par23">We also studied sections of six thymuses from children that died in postnatal life born from mothers infected by ZIKV during pregnancy. The histological profile of these specimens varied from virtually normal cortico-medullary pattern to a severe disruption of the cortex in one thymus, with clear-cut decrease in thymocytes (Fig. <a href="#Fig6">6a</a>). Yet, in all cases, we observed an increase in the numbers of the Hassall’s corpuscles, indicating an over differentiation of the TEC network. This feature was further confirmed by immunohistochemistry for cytokeratin detection (applied herein as a pan-marker of the whole TEC network) in TEC cultures, as seen in Supplemental Fig. <a href="#MOESM1">1</a>. In fact, cytokeratin immunostaining revealed, in some thymus samples, a complete loss of the cortical/medullary distinction (Fig. <a href="#Fig6">6b</a>, case 2). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <p>Congenital Zika virus infection induces major changes in the human thymic epithelial network. Representative images of (<b>a</b>) Hematoxylin-eosin and (<b>b</b>) cytokeratin staining of paraffin-embedded human thymus sections. A control case, case 6 with intermediate alterations and case 2 with severe alterations in the human thymic epithelial network are represented (n = 2 for controls and n = 6 for congenital ZIKV infection). Arrows show some of the Hassal’s corpuscles. C: cortical region; M: medullary region.</p>   </div> <p /> <p id="Par24">Considering the transcriptome data, we also performed histological evaluations for ECM-containing fibers and fibrils. In the thymuses from ZIKV-infected newborns, there was an important increase in the reticulin network, indicating thickness of basement membranes in the thymic microenvironment. Additionally, Masson’s trichrome staining revealed a higher density of type I collagen-containing fibers in the thymic <i>septae</i>. These data are summarized in Fig. <a href="#Fig7">7</a>. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <p>Congenital Zika virus infection induces changes in the human thymic extracellular matrix. Representative images of (<b>a</b>) Gomori’s reticulin (dark blue fibers) and (<b>b</b>) Masson’s trichrome staining (blue fibers) of paraffin-embedded human thymus sections. A control case, case 6 (with intermediate alterations) and case 2 (with severe alterations) in the human thymic epithelial network are represented. Arrows show some collagen fibers present in the images. (n = 2 for controls and n = 6 for congenital ZIKV infection).</p>   </div> <p /> <p id="Par25">We also searched for the presence of the viral envelope protein in the same thymuses, using the 4G2 antibody, which recognizes any Flavivirus family envelope protein. As compared to normal thymuses (that were completely negative for 4G2 labeling), we did find specific immunostaining in thymus sections from 5 out of 6 thymuses evaluated. The presence of the envelope viral protein was seen in scattered TEC, both in the cortex and medulla of the thymic lobules, as well as in Hassall’s corpuscles (Fig. <a href="#Fig8">8</a>). It is important to point out that one thymus was positive for the presence of ZIKV ascertained by the RT-PCR (case 4 - Supplemental Table <a href="#MOESM1">4</a>), although we were not able to detect the viral envelope protein in this case (case 4 – Fig. <a href="#Fig8">8</a> and Supplemental Fig. <a href="#MOESM1">2a</a>). None of the thymus preparations were positive for the presence of Dengue as well as Chikungunya viruses (Supplemental Table <a href="#MOESM1">4</a>). Secondary antibodies were used as negative controls and did not generate any significant labeling (Supplemental Fig. <a href="#MOESM1">2b</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8</span></a></div>   <p>ZIKV envelope protein is found in human thymus of babies with congenital Zika virus infection. Representative images of immunofluorescence staining of 4G2 (viral protein, in red), cytokeratin (in green) and DAPI (nuclei, in blue) of paraffin-embedded human thymus sections (n = 2 for controls and n = 6 for congenital ZIKV infection). First column shows the merge of the three stainings. The last column shows the colocalization analysis for 4G2 and cytokeratin stainings; white points represent overlap between these two immunostainings. We could identify viral protein in five out of the six thymuses studied; most of cells labeled being epithelial. Arrows show some of the cells that were stained for both 4G2 and cytokeratin. Differences in bar sizes are due to the digital zoom used during image acquisition.</p>   </div> <p /> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par26">The behavior of the human immune system secondary to a congenital syndrome caused by the ZIKV remains largely unknown, although there is evidence that innate and acquired immune cells can be infected by the virus, as ascertained by studies on peripheral blood leukocytes<sup><a href="#CR21">21</a>,<a href="#CR22">22</a></sup>. Nevertheless, whether primary lymphoid organs are targeted by the ZIKV, as well as putative consequences upon interactions with B- and T-cell precursors and their corresponding microenvironments, remain to be demonstrated. Herein we tackled this issue by investigating whether or not the human thymic epithelium could be infected by ZIKV.</p> <p id="Par27">We first demonstrated that growing human TEC could indeed be infected, and that such infection occurred through the AXL receptor. Moreover, viral particles produced within TEC were infective. Interestingly, the human TEC line could also be infected with other RNA viruses, namely the retrovirus HTLV-1<sup><a href="#CR15">15</a></sup>. Moreover, pioneer work has shown that primary cultures of human TEC can also be infected with HIV<sup><a href="#CR12">12</a></sup>. This shows that human TEC is permissive to different types of RNA viruses.</p> <p id="Par28">Importantly, ZIKV did trigger an antiviral machinery response in human TEC, as ascertained by the upregulation in the expression of several genes related to antiviral response, including, among others, those related to the type 1 interferon signaling.</p> <p id="Par29">Major changes in TEC growth were also found after <i>in vitro</i> ZIKV infection, including a decrease in proliferation and increase in cell death, resulting in a reduction of total cell numbers after 3-day TEC cultures. RNA viruses such as ZIKV use strategies to modulate cell-cycle control and cell proliferation from the host and obtain cellular conditions favorable for their replication<sup><a href="#CR23">23</a>,<a href="#CR24">24</a></sup>. Accordingly, the data from RNA sequencing revealed few downregulated genes associated with the regulation of cell proliferation and apoptosis. These results are consistent with previous studies on ZIKV-infected human cell types, such as Sertoli cells in the seminiferous tubules, retinal pigment epithelium, neural progenitors, and astrocytes, which reported suppression of cell proliferation, as well as induction of antiviral defense pathways<sup><a href="#CR24">24</a>–<a href="#CR27">27</a></sup>.</p> <p id="Par30">It is known that adhesion of developing T-cells to the thymic epithelium is controlled by various cell-cell and cell-extracellular matrix interactions, including ICAM-1, VCAM-1, fibronectin, laminins, and their corresponding integrin receptors<sup><a href="#CR28">28</a>–<a href="#CR32">32</a></sup>. We found a large number of upregulated genes related to extracellular matrix, cell adhesion, and cell migration. Significant increase in the gene expression of various laminin alpha-chains as well as fibril forming collagen chains was seen in ZIKV-infected TEC, as well as higher levels of VCAM1 and ICAM-1 expression in TEC after ZIKV infection.</p> <p id="Par31">Enhanced expression of some matrix metalloproteinase genes, such as MMP8 and MMP19 (both having collagenolytic activity) was also observed, suggesting that the ECM-related metabolism of TEC is altered by ZIKV. Moreover, ZIKV augmented the gene expression of various chemokines; some of which (CCL4, CCL5, CCL20, and CXCL10, for example) being involved in intrathymic T-cell migration<sup><a href="#CR33">33</a>–<a href="#CR36">36</a></sup>.</p> <p id="Par32">In keeping with the transcriptome data related to extracellular matrix, cell adhesion, and cell migration, we demonstrated that infected TEC expressed higher levels of the ICAM-1 protein on their membranes. In keeping with the literature showing that ICAM-1 plays a major role in TEC adhesion to developing T-cells<sup><a href="#CR37">37</a></sup>, we found that lymphoblastic T cells adhered more to ZIKV-infected TEC than to uninfected TEC cultures. We can thus hypothesize that enhanced expression of adhesion molecules and chemokines would facilitate the encounter of developing T-cells with ZIKV-infected TEC, favoring in one hand the transfer of the virus to lymphocytes, and on the other hand, the establishment of an immunological synapse between the two cell types. This should also be placed in the context of the control of HLA-B gene expression, which was upregulated (Fig. <a href="#Fig3">3b</a>) in ZIKV-infected human TEC.</p> <p id="Par33">The control of gene expression in ZIKV-infected human TEC was also evaluated through the analysis of different non-coding RNAs. In fact, our robust transcriptome and microRNA expression analyses in ZIKV-infected TEC highlighted the molecular response of protein-coding genes and non-coding genes (lncRNAs, circRNAs and miRNAs). We observed miRNAs acting as probable regulators of genes associated with immune response, such as hsa-miR-222-5p (DDX58 and HLA-B), hsa-miR-1303-5p (HLA-DOA), hsa-miR-193b-5p (IFIT2), hsa-miR-100-3p (IFIT1B). Some miRNAs also have shown response to Cytokine Signaling in Immune system such as hsa-miR-503-3p (TNFRSF9) and hsa-miR-4458-5p (TNFSF9). The chemokines CXCL8 and CXCL11 could also be regulated by miRNAs in ZIKV-infected TEC. Taken together, these analyses indicate that miRNAs may play a relevant role in the intracellular control of antiviral immunity against ZIKV-infected TEC.</p> <p id="Par34">We also identified several cirRNAs which can serve as miRNA sponges and are crucial regulators of gene expression<sup><a href="#CR38">38</a></sup>. Some circRNAs present antiviral activity, interacting directly with the antiviral genes DDX58 (RIG-I), IFIH1 (MDA5) and TLR3<sup><a href="#CR39">39</a></sup>. The major class of non-coding lncRNAs also shows antiviral activity and involvement in cell proliferation and apoptosis. The LUARIS lncRNA expression level was decreased, indicating that it may be suppressed by IFNB1, which, in turn, was highly induced in ZIKV-infected human TEC. The expression of some interferon stimulated genes (ISGs) and chemokines showed altered levels of transcripts, indicating that LUARIS could act in antiviral immunity. The same regulation pattern of ISGs and chemokines was found in infection by other human viruses such as HBV, HCV, and EMCV<sup><a href="#CR40">40</a></sup>. Following ZIKV infection in TEC, we also detected an upregulation of the lncRNA EGOT, which is also induced after HCV infection, increasing viral replication<sup><a href="#CR41">41</a></sup>.</p> <p id="Par35">Studies involving lncRNA associated with cell proliferation and apoptosis and cell adhesion have been already described. We hypothesize that upregulation of PICART1 may also suppress proliferation and induce apoptosis in ZIKV-infected human TEC, as shown by other authors that this lncRNA impairing proliferation<sup><a href="#CR42">42</a></sup> and inducing apoptosis<sup><a href="#CR43">43</a></sup> in lung cancer cells. Another lncRNA, SENCR can stabilize the vascular endothelial cell adherent junctions<sup><a href="#CR44">44</a></sup>, which mediate cell adhesion and intracellular signals<sup><a href="#CR45">45</a></sup>. Accordingly, the upregulation of SENCR may be involved in regulating the enhancement of cell adhesion seen in ZIKV-infected TEC. The transcriptome and miRNA sequencing also revealed that genes associated with adhesion and ECM molecules were activated by infection by ZIKV, including coding and non-coding genes. Overall, our findings highlight a high complexity in the intracellular circuitry controlling ZIKV replication within human TEC.</p> <p id="Par36">In addition to extended <i>in vitro</i> studies showing that human TEC is infected and respond to ZIKV, we evaluated the presence of the virus as well as the general thymus microarchitecture in histological sections obtained from children who died early in life, whose mothers were infected with ZIKV during pregnancy. Although the number of specimens was low, thymuses were widely affected, varying from massive loss of cortical thymocytes, to preservation of the cortex in some cases. Within the thymic lobules and septae, we could see an increase in ECM deposition, as revealed by histological staining of collagen-containing fibrils. Most importantly, there was an alteration in the thymic epithelial network with a clear-cut increase in the formation of Hassall’s corpuscles, denoting a higher degree of keratinization (corresponding to terminal epithelial cell differentiation). In the same vein, we observed an increase in keratin contents in ZIKV-infected TEC cultures. It is important to point out that such enhancement of Hassall’s corpuscles is also found in cases of glucocorticoid treatment of patients with primary immunodeficiency<sup><a href="#CR46">46</a></sup>, as well as in mice treated with dexamethasone<sup><a href="#CR47">47</a></sup>.</p> <p id="Par37">In 5 out of the 6 thymuses analyzed, we detected the presence of viral protein colocalized with TEC, as ascertained by immunofluorescence using the 4G2 anti-flavivirus and anti-cytokeratin antibodies. Not only we detected the viral protein in Hassall’s corpuscles, as briefly reported by Valdespino-Vázquez and co-workers<sup><a href="#CR48">48</a></sup>, but also in TEC outside the corpuscles and that are located in close contact with developing thymocytes.</p> <p id="Par38">Rather surprisingly however, the virus genome was detected, by using qRT-PCR, only in one case. This can be explained by the fact the material applied was the bulk of thymic tissues whereas the immunohistochemical data revealed detectable amounts of the virus only in some microenvironmental cells (most of them epithelial). Moreover, in this particular case we did not see 4G2<sup>+</sup> cells in the thymus sections we examined. One conceivable explanation for the fact that the only thymic extract positive for qRT-PCR was negative for immunolabeling with the 4G2 monoclonal antibody is that in this particular case, the presence of the virus was restricted to one part of the thymus that was not evaluated by immunohistochemistry.</p> <p id="Par39">Finally, from a conceptual point of view, it is conceivable that the human thymic epithelium is a target for ZIKV infection, with general but also specific consequences, as for example the modulation of adhesion molecules, extracellular matrix and cytokines and chemokines known to modulate intrathymic T-cell development.</p> </div> <div class="methodsandpatients" title="sec"> <div class="title" tagx="title" title="title">
Methods and Patients</div> <div class="humanteclineandzikavirusinfection" title="sec"> <div class="title" tagx="title" title="title">
Human TEC line and Zika virus infection</div> <p id="Par40">The human postnatal TEC line<sup><a href="#CR17">17</a></sup> was maintained in RPMI-1640 medium - supplemented with 10% fetal calf serum, 2 g/L sodium bicarbonate, 2 g/L HEPES, 1X antimycotic solution (Sigma-Aldrich) and 5 µg/mL of plasmocin prophylactic (InvivoGen) at 37 °C, in an atmosphere containing 5% CO<sub>2</sub>.</p> <p id="Par41">The ZIKV strain used in this study was RIO-U1, which was isolated in 2016 from urine of a patient from Rio de Janeiro state, Brazil (GenBank Accession number: KU926309), as described elsewhere<sup><a href="#CR49">49</a></sup>. Viral stocks of ZIKV Rio-U1 were prepared by infecting Vero cell monolayers. The titer (PFU/ml) and the genome integrity were determined as described elsewhere<sup><a href="#CR49">49</a></sup>. Samples of the stock were used in the TEC infection assays. Vero cell cultures were also applied to test the infectivity of virus particles derived from infected TEC.</p> <p id="Par42">Unless described elsewhere, growing human TEC was plated in T-25 flasks (10<sup>5</sup> cells) and infected 24 h later with 0.1 or 1.0 multiplicity of infection (MOI) or with 1 mL of infected TEC-derived supernatants for 1 h at 37<b>°</b>C in an atmosphere containing 5% CO<sub>2</sub>. After infection, culture medium was changed and cells were maintained in culture for 72 h. Controls (MOCK group) corresponded to cell grown under the same culture conditions but not infected by ZIKV.</p> </div> <div class="cemcelllineandzikvinfection" title="sec"> <div class="title" tagx="title" title="title">
CEM cell line and ZIKV infection</div> <p id="Par43">CEM is an acute T-cell lymphoblastic leukemia cell line that was maintained in RPMI-1640 medium supplemented as described above. ZIKV Infection was performed with 1.0 MOI, as described previously for peripheral blood mononuclear cells<sup><a href="#CR50">50</a></sup>.</p> </div> <div class="viralmultiplicationintecline" title="sec"> <div class="title" tagx="title" title="title">
Viral Multiplication in TEC line</div> <p id="Par44">An aliquot of supernatant was collected every 24 h after infection for three days and frozen at −80 °C. Briefly, Vero cells were seeded in a 24-well plate (5 × 10<sup>4</sup> cells/cm<sup>2</sup>), 24 h before inoculation. Serial dilutions of the supernatant were used to infect cell monolayers. After 1 h incubation at 37 °C, the supernatant was replaced by 2.4% CMC (carboxymethyl cellulose) in Earle’s 199 complete medium supplemented with 5% FBS, followed by incubation for 7 days at 37 °C. Cells were fixed with 10% formaldehyde, washed, and dyed with 0.4% crystal violet. Viral titer was determined from the numbers of plaques visualized.</p> </div> <div class="inhibitionofinfection" title="sec"> <div class="title" tagx="title" title="title">
Inhibition of infection</div> <p id="Par45">TEC were plated and 24 h later incubated for 30 min at 37 °C in an atmosphere containing 5% CO<sub>2</sub> with culture media containing 10 µg/mL of goat anti-AXL polyclonal antibody (R&amp;amp;D Systems). Identical concentrations of IgG from normal goat serum were used as control. IgG amounts present in normal goat serum were quantified in Nanodrop™ 2000/2000c Spectrophotometers (Thermo Fisher Scientific). Cells were then infected as described previously in the presence of anti-AXL. Our experiments on the role of the AXL receptor in ZIKV entry in TECs were based on the paper of Hamel and co-workers, showing that exposure of the human skin fibroblast cell line HFF1 to ZIKV (and to Dengue virus, used as a positive control) resulted in comparable numbers of infected cells that were inhibited 70% and 50%, respectively, in the presence of the same neutralizing anti-AXL antibody<sup><a href="#CR51">51</a></sup>. Infection was quantified by flow cytometry as indicated below.</p> </div> <div class="antibodies" title="sec"> <div class="title" tagx="title" title="title">
Antibodies</div> <p id="Par46">See Supplementary Table <a href="#MOESM1">3</a>.</p> </div> <div class="flowcytometry" title="sec"> <div class="title" tagx="title" title="title">
Flow cytometry</div> <p id="Par47">Intracellular staining of viral protein was performed with True-Nuclear™ transcriptional factor buffer set (Biolegend). Cells were incubated with 1x Fix Concentrate buffer for 40 min at 4 °C, followed by incubation with normal human serum for 20 min at 4 °C. Then, cells were stained with 4G2 antibody for 1 h at 4 °C and stained with secondary antibody for 30 min at 4 °C. After, cells were washed with 1x Perm buffer, as well as between each incubation. Subsequently, cells were fixed with formaldehyde 2% and evaluated by flow cytometry.</p> <p id="Par48">Cell proliferation status was performed by intracellular staining of Ki67 protein as indicated above for detection of viral protein, with the exception of that a single antibody incubation was realized for 30 min. We further evaluated those cells which are Ki67 high expressers, which correspond to those leaving cells in the mitosis phase of the cell cycle.</p> <p id="Par49">Cell death was analyzed by staining with Annexin-V-APC (Immunotools) and PI (Propidium iodide– Sigma-Aldrich). Cells were resuspended in 100 µL of 1x Annexin-V binding buffer (BD Biosciences) and incubated with Annexin-V-APC (1 µL) for 10 min at room temperature. PI (10 µg/mL) was added to the suspension prior acquisition on the flow cytometer.</p> <p id="Par50">To analyze ICAM-1 (CD54) and VCAM-1 (CD106) expression cells were incubated with normal human serum for 20 min at 4 °C and then incubated with CD54-PECY5 or CD106-APC antibodies for 30 min at 4 °C. After incubations cells were washed with PBS. Then, the cells were fixed with formaldehyde 2% and evaluated by flow cytometry.</p> <p id="Par51">The fluorescence intensity of labeled TEC was determined with FACSCanto II flow cytometer (BD Biosciences) Analyses were carried out using the BD FACSDiva 6.1.3 software (BD Biosciences).</p> </div> <div class="immunofluorescence" title="sec"> <div class="title" tagx="title" title="title">
Immunofluorescence</div> <p id="Par52">TEC were seeded in 8 well (10<sup>3</sup> cells/well) Lab-Tek<sup>®</sup> Chamber Slide™ System and infected as described previously. After infection, the slides were washed twice with PBS and fixed in 4% paraformaldehyde for 15 min. Slides were washed again and treated with permeabilization buffer (0.1% saponin and 1% BSA – bovine albumin serum) for 15 min at 37 °C. Thereafter, samples were incubated for 40 min with blocking solution (0.2% normal goat serum, 1% FBS, 1% BSA, 0.3% X-100 triton) and submitted to staining with primary antibody for 1 h and with secondary antibody for 30 min. After staining, the slides were washed twice with permeabilization buffer, stained with DAPI for 10 min, washed and mounted with Dako Fluorescent Mounting Medium (DAKO).</p> <p id="Par53">Immunostained samples were analyzed by an AxioImager A2 device using the AxioVision Rel 4.8 software (Zeiss) or by Leica TSC SP8 Confocal device using LAS-X Software (Leica Microsystems). Negative controls, in which the secondary antibody was used alone, did not generate any significant labeling. The percentage of infection was achieved by the number of 4G2<sup>+</sup> cells divided by the total number of cells per field. We analyzed 5 fields from 3 independent experiments and each experiment was performed in triplicate. The quantitative fluorescence analysis of cytokeratin expression was performed by transforming the specific staining into an eight-bit grey image for analysis of the corrected total cell fluorescence (CTCF) through the ImageJ software (NIH). At least 50 cells per condition were analyzed CTCF is calculated by the integrated density minus the area of a given cell multiplies by the mean fluorescence of background readings. When necessary, brightness and contrast adjustments were performed in the entire images.</p> </div> <div class="tecadhesiontolymphoblastictcells" title="sec"> <div class="title" tagx="title" title="title">
TEC adhesion to lymphoblastic T cells</div> <p id="Par54">Human TEC were seeded (10<sup>4</sup> or 5 × 10<sup>4</sup> cells) and 24 h later infected as described previously. After 72 h in culture, CEM cells (human T-cell acute lymphoblastic leukemia cell line) were added to the infected culture in a proportion of 50 CEM cells per TEC. We led these cells to adhere onto ZIKV-infected or non-infected-TEC during 1 h in serum-free RPMI medium at 37 °C in an atmosphere containing 5% CO<sub>2</sub>. Non-adherent CEM cells were gentle washed out with PBS at 37 °C. Culture flasks were fixed with methanol for 7 min and stained with Giemsa. The number of adherent CEM cells per at least 500 TEC were counted and the association index (AI) was calculated by the following formula, previously validated for this type of analysis<sup><a href="#CR52">52</a>,<a href="#CR53">53</a></sup>. </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M1">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document}$${\rm{AI}}=\frac{{\rm{number}}\,{\rm{of}}\,{\rm{TEC}}\,{\rm{with}}\,{\rm{CEM}}\,{\rm{cells}}\,\times \,{\rm{number}}\,{\rm{of}}\,{\rm{CEM}}\,{\rm{cells}}\,{\rm{bound}}\,{\rm{to}}\,{\rm{TEC}}}{{\rm{total}}\,{\rm{TEC}}\,{\rm{number}}}\times 100$$\end{document}</div> </div><div class="math_UNKNOWN"> <div class="mi_UNKNOWN">
AI</div> <div class="mo_UNKNOWN">
=</div> <div class="mfrac_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
number</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
of</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
TEC</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
with</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
CEM</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
cells</div> <div class="mspace_UNKNOWN" /> <div class="mo_UNKNOWN">
×</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
number</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
of</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
CEM</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
cells</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
bound</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
to</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
TEC</div> </div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
total</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
TEC</div> <div class="mspace_UNKNOWN" /> <div class="mi_UNKNOWN">
number</div> </div> </div> <div class="mo_UNKNOWN">
×</div> <div class="mn_UNKNOWN">
100</div></div><div class="graphic_UNKNOWN" position="anchor" /> </div> </div> <p /> </div> <div class="thymicepithelialandlymphoblastictcellco-cultures" title="sec"> <div class="title" tagx="title" title="title">
Thymic epithelial and lymphoblastic T cell co-cultures</div> <p id="Par55">TEC were seeded in T-25 flasks (5 × 10<sup>4</sup> cells) and 24 h later infected as described previously. Seventy-two hours post-infection, the medium was removed and CEM cells were added to the culture in a proportion of 25 CEM cells per TEC with fresh RPMI medium supplemented with 15% FBS. After 24 h of co-culture, non-adherent cells were removed by washing the flasks with PBS. Remaining adherent cells were recovered after addition of trypsin to the flasks. Both adherent and non-adherent cells were stained with 4G2 and anti-cytokeratin antibody by flow cytometry, as described previously. Anti-cytokeratin antibody was used for distinguishing CEM cells and TEC in adherent and non-adherent cells suspensions.</p> </div> <div class="rnaextraction" title="sec"> <div class="title" tagx="title" title="title">
RNA extraction</div> <p id="Par56">TEC were seeded in T-25 flasks (10<sup>5</sup> cells) and 24 h later infected as described previously. Seventy-two hours post-infection, RNA extraction was performed with MOCK and ZIKV-infected TEC using RNeasy Plus Mini Kit (Qiagen, Hilden, Germany), following the manufacturer’s instructions for mRNA and small RNA extraction. Total RNA concentration and purity were determined by Nanodrop spectrophotometer. RNA integrity was verified through Agilent 2100 Bioanalyzer (Agilent Technologies).</p> </div> <div class="transcriptomeandsmallrnalibrarypreparationandsequencing" title="sec"> <div class="title" tagx="title" title="title">
Transcriptome and small RNA library preparation and sequencing</div> <p id="Par57">The 12 transcriptome libraries were prepared from 500 ng of total RNA using the TruSeq Stranded Total RNA Sample Preparation Kit with Ribo-Zero Gold (Illumina). Additionally, the 12 small RNA libraries were prepared from 1 μg from the same total RNA of transcriptome analysis, using the TruSeq Small RNA Library Prep Kit (Illumina). All the steps of the protocols were carried out without any modifications, with the exception of the cDNA construction purification step on the small RNA protocol, which was performed using 3% Agarose Gel Cassette for targets between 100 bp − 250 bp in a BluePippin system (Sage Science). The quality control of each library was performed using the 2100 Bioanalyzer System with the Agilent High Sensitivity DNA Kit (Agilent). The libraries were quantified via qPCR using a KAPA Library Quantification Kits for Illumina platforms (KAPA Biosystems). The Libraries were sequenced on a NextSeq. 500 sequencing system (Illumina), for the RNA-seq paired-end reads (2 × 75 bp) were obtained using NextSeq. 500/550 High Output v2 kit (150 cycles) and for the small RNA, single-end reads (1 × 50 bp) were obtained using a NextSeq. 500/550 High Output v2 kit (75 cycles).</p> </div> <div class="transcriptomeanalysis" title="sec"> <div class="title" tagx="title" title="title">
Transcriptome analysis</div> <p id="Par58">The 12 transcriptome and 12 small RNA libraries were quality checked with FASTQC tool (<a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>) and further trimmed with Trimmomatic tool<sup><a href="#CR54">54</a></sup>. For small RNAse libraries, only reads with length between 16-28 nucleotides were kept for posterior analysis. The high-quality transcriptome libraries were mapped separately on the Human genome (GRCH38.p12) from ENSEMBL databank (<a href="http://www.ensembl.org/info/data/ftp/index.html">http://www.ensembl.org/info/data/ftp/index.html</a>) using the STAR mapper (Version 2.6)<sup><a href="#CR55">55</a></sup>, with default parameters. The small RNA reads were mapped in the human microRNA precursors retrieved from miRBase (Release 22) (<a href="http://www.mirbase.org/">http://www.mirbase.org/</a>) with Bowtie2 tool (version 2.2.1)<sup><a href="#CR56">56</a></sup> running with default parameters.</p> <p id="Par59">The Differential Gene Expression (DGE) analysis was performed with DESeq. 2 package<sup><a href="#CR57">57</a></sup> from R. The genes (protein-coding, lncRNAs, circRNAs, and other transcripts) and microRNAs with adjusted p-value &amp;lt; 0.01 and Log2 Fold Change &amp;gt; |1.0| were considered for further analyses. Both transcriptome and microRNA expression analysis was carried out in six biological replicates.</p> <p id="Par60">In order to identify microRNAs and their real targets, we retrieved the experimentally validated microRNA-target interactions from miRTarBase (<a href="http://mirtarbase.mbc.nctu.edu.tw/php/index.php">http://mirtarbase.mbc.nctu.edu.tw/php/index.php</a>) plus the predicted ones from SpidermiR package<sup><a href="#CR58">58</a></sup>, which includes DIANA<sup><a href="#CR59">59</a></sup>, miRanda<sup><a href="#CR60">60</a></sup>, PicTar<sup><a href="#CR61">61</a></sup> and TargetScan<sup><a href="#CR62">62</a></sup> microRNA target datasets. We compared with differentially expressed microRNAs and target genes from both transcriptome and small RNA seq experiments.</p> </div> <div class="circularrna(circrnas)analysis" title="sec"> <div class="title" tagx="title" title="title">
Circular RNA (circRNAs) Analysis</div> <p id="Par61">Using the 12 transcriptome libraries, we performed a DGE analysis with DESEQ. 2 library<sup><a href="#CR57">57</a></sup> of the circular RNAs (circRNAs) from human genome, based on circBase (<a href="http://www.circbase.org/">http://www.circbase.org/</a>). The circRNAs with adjusted p-value lower than 0.01 and Log<sub>2</sub> Fold Change &amp;gt; |1.0| were used to characterize the interaction of microRNAs and circRNAs with miRanda tool<sup><a href="#CR60">60</a></sup>, using strict 5′ seed pairing, minimum score threshold, alignment length and identity of 140, 17 and 80%, respectively. The interactions were visualized with Cytoscape tool<sup><a href="#CR63">63</a></sup>.</p> </div> <div class="enrichmentanalysisofdegs" title="sec"> <div class="title" tagx="title" title="title">
Enrichment analysis of DEGs</div> <p id="Par62">The Gene Ontology (GO)<sup><a href="#CR64">64</a></sup> and ReactomePA<sup><a href="#CR65">65</a></sup> were used to determine enrichment of biological processes and pathways, respectively on DEGs. We used the ClusterProfiler package from R<sup><a href="#CR66">66</a></sup> for ReactomePA generation and GO enrichment by over-representation test. The enrichment was generated with Benjamini &amp;amp; Hochberg (BH) adjusted p-value method with a p-value cutoff of 0.01.</p> </div> <div class="humanthymushistologyandimmunostainings" title="sec"> <div class="title" tagx="title" title="title">
Human thymus histology and immunostainings</div> <p id="Par63">We evaluated human thymus fragments from children who died up to 6 months after birth, and whose mothers were infected with ZIKV during pregnancy. The clinical features of these newborns are summarized in Supplemental Table <a href="#MOESM1">4</a>. The use of organ histological sections for research has been approved by the Fernandes Figueira National Institute for the Health of Mother, Child, and Adolescent, Oswaldo Cruz Foundation review board under the number CAAE: 52675616.0.000.5269. All participants provided written informed consent. All methods were performed in accordance with the relevant guidelines and regulations. General histological profiles were evaluated through haematoxin-eosin staining. Gomori’s reticulin and Masson’s trichrome staining procedures were applied for revealing extracellular matrix containing compartments.</p> <p id="Par64">In brief, paraffin-embedded thymus sections (4 µm) were maintained at 65 °C overnight. The slides were deparaffinized in two passages in xylene and three passages in absolute alcohol for two min each. Samples were hydrated in distilled water and the peroxidase blockade was performed. The slides were washed with PBS and antigenic recovery was performed in water bath at 96 °C with 10% Trilogy solution for 40 min. After cooling for 20 min, the slides were washed with PBS and blocked with 2% BSA for 40 min. Thereafter, samples were incubated overnight at 4 °C with anti-cytokeratin and 4G2 antibody. The slides were washed twice with permeabilization buffer and incubated for 30 min with secondary antibodies for 30 min. After being washed twice with permeabilization buffer, samples were incubated for 10 min with DAPI and washed again. Immunostained samples were analyzed by the Leica TSC SP8 Confocal device using LAS-X Software.</p> <p id="Par65">Two thymuses derived from 2 infants were applied for controls. Although in both cases there was a pathological situation associated (pulmonary hypoplasia), their normal histological profile was confirmed by hematoxylin-eosin staining. Sections of these specimens were subjected to the same histological and immunofluorescence procedures.</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p id="Par66">Unless described elsewhere, the statistical significance was approached using unpaired one- or two-tailed <i>t</i>-test or One-way ANOVA followed by Tukey’s multiple comparison test.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary information</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2020_58135_MOESM1_ESM.docx">LINK</a> <p>Supplemental material.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2020_58135_MOESM2_ESM.pdf">LINK</a> <p>Supplementary Table 1.</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41598_2020_58135_MOESM3_ESM.pdf">LINK</a> <p>Supplementary Table 2.</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> <div class="fn-type-" title=""> <p>These authors contributed equally: Carolina V. Messias, Guilherme Loss Morais and Joseane Biso de Carvalho.</p> </div> <div class="fn-type-" title=""> <p>These authors jointly supervised this work: Ana T. R. Vasconcelos, Daniella A. Mendes-da-Cruz and Wilson Savino.</p> </div> </div> <div class="supplementaryinformation" title="sec"> <div class="title" tagx="title" title="title">
Supplementary information</div> <p>is available for this paper at 10.1038/s41598-020-58135-y.</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>The human postnatal TEC line was kindly provided by Dr. Maria Luiza Toríbio (Universidad Autonoma de Madrid, Madrid, Spain). The mouse monoclonal 4G2 antibody (anti-flavivirus) was gently provided by the Laboratory of Monoclonal Antibody Technology from Bio-Manguinhos (Oswaldo Cruz Foundation, Rio de Janeiro, Brazil). The authors thank Raquel de Souza Martins for confocal analysis assistance. This work was funded by Brazilian Funding Agency for Studies and Projects (FINEP) #01.16.0078.00; Brazilian National Council for Scientific and Technological Development (CNPq) ATRV supported by #303170/2017-4, WS by #306699/2018-4, DAMC by #307720/2016-0; Carlos Chagas Filho Foundation for Funding Research in the State of Rio de Janeiro (Faperj) ATRV supported by #E-26/202.826/2018; the Oswaldo Cruz Foundation (Fiocruz) and National Laboratory for Scientific Computation; Coordination of Superior Level Staff Improvement (CAPES) - Brazil - Finance Code 001 and GLM, JBC and LWPA supported #23038.010041/2013-13; Mercosur Fund for Structural Convergence (FOCEM, Mercosur #03/11) and the Program of Preventing and Combating the Zika Virus, MCTIC/FNDCT - CNPq/MEC-CAPES/MS-Decit (Grants 426767/2018-7 and 88881.130684/2016-01 for MCB). This work was developed in the framework of the Brazilian National Institute of Science and Technology on Neuroimmunomodulation (INCT-NIM) WS supported by #465489/2014-1.</p> </div> <div class="title" tagx="title" title="title">
Author contributions</div> <p>C.V.M., G.L.M. and J.B.C.: conceived and designed the experiments, performed the experiments, analyzed data and wrote the manuscript. M.N.G., D.P.C., D.A.F.O., A.L.G. and L.M.S.R.: conceived and designed the experiments, performed the experiments, analyzed data. Z.V., E.A.P., M.E.L.M., M.C.B., I.R. and V.C.A.: conceived and designed the experiments; L.W.P.A.: analyzed data; N.F: performed the experiments; A.T.R.V., D.A.M.C. and W.S.: conceived and designed the experiments, wrote the manuscript and coordinated the research.</p> <div class="title" tagx="title" title="title">
Data availability</div> <p>The RNA-seq and small-RNA-seq data that support the findings of this study have been deposited in SRA database from NCBI with the accession codes PRJNA560342 and PRJNA561078. All other data are available in the article and its supplementary information files or from the corresponding authors upon request.</p> <div class="title" tagx="title" title="title">
Code availability</div> <p>We use already published algorithms. See Methods section for full description of each analysis including input data, library, and algorithm version used, for which we provide corresponding web links or references.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par67">The authors declare no competing interests.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martines</span><span class="given-names" tagx="given-names" title="given-names">RB</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Pathology of congenital Zika syndrome in Brazil: a case series</span><span class="source" tagx="source" title="source">Lancet</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">388</span><span class="fpage" tagx="fpage" title="fpage">898</span><span class="lpage" tagx="lpage" title="lpage">904</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(16)30883-2">10.1016/S0140-6736(16)30883-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27372395">27372395</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carod-Artal</span><span class="given-names" tagx="given-names" title="given-names">FJ</span></span></span><span class="mixed-article-title" title="mixed-article-title">Neurological complications of Zika virus infection</span><span class="source" tagx="source" title="source">Expert Rev. Anti. Infect. Ther.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">399</span><span class="lpage" tagx="lpage" title="lpage">410</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/14787210.2018.1466702">10.1080/14787210.2018.1466702</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29668332">29668332</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lopes Moreira</span><span class="given-names" tagx="given-names" title="given-names">ME</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Neurodevelopment in Infants Exposed to Zika Virus In Utero</span><span class="source" tagx="source" title="source">N. Engl. J. Med.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">379</span><span class="fpage" tagx="fpage" title="fpage">2377</span><span class="lpage" tagx="lpage" title="lpage">2379</span><span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJMc1800098">10.1056/NEJMc1800098</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30575464">30575464</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen-Saines</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Delayed childhood neurodevelopment and neurosensory alterations in the second year of life in a prospective cohort of ZIKV-exposed children</span><span class="source" tagx="source" title="source">Nat. Med.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">1213</span><span class="lpage" tagx="lpage" title="lpage">1217</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41591-019-0496-1">10.1038/s41591-019-0496-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31285631">31285631</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Da Silva</span><span class="given-names" tagx="given-names" title="given-names">IRF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frontera</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Filippis</span><span class="given-names" tagx="given-names" title="given-names">AMB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Do Nascimento</span><span class="given-names" tagx="given-names" title="given-names">OJM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Neurologic complications associated with the Zika virus in Brazilian adults</span><span class="source" tagx="source" title="source">JAMA Neurol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">74</span><span class="fpage" tagx="fpage" title="fpage">1190</span><span class="lpage" tagx="lpage" title="lpage">1198</span><span class="pub-id"><a href="https://dx.doi.org/10.1001/jamaneurol.2017.1703">10.1001/jamaneurol.2017.1703</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28806453">28806453</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Savino</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Zika Virus Infection in the Elderly: Possible Relationship with Guillain-Barré Syndrome</span><span class="source" tagx="source" title="source">Gerontology</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">63</span><span class="fpage" tagx="fpage" title="fpage">210</span><span class="lpage" tagx="lpage" title="lpage">215</span><span class="pub-id"><a href="https://dx.doi.org/10.1159/000453579">10.1159/000453579</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28002820">28002820</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garcez</span><span class="given-names" tagx="given-names" title="given-names">PP</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Zika virus impairs growth in human neurospheres and brain organoids</span><span class="source" tagx="source" title="source">Science (80-.).</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">352</span><span class="fpage" tagx="fpage" title="fpage">816</span><span class="lpage" tagx="lpage" title="lpage">818</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.aaf6116">10.1126/science.aaf6116</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azouz</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arora</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krause</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nerurkar</span><span class="given-names" tagx="given-names" title="given-names">VR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Integrated MicroRNA and mRNA profiling in zika virus-infected neurons</span><span class="source" tagx="source" title="source">Viruses</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">22</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/v11020162">10.3390/v11020162</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aid</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Zika Virus Persistence in the Central Nervous System and Lymph Nodes of Rhesus Monkeys</span><span class="source" tagx="source" title="source">Cell</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">169</span><span class="fpage" tagx="fpage" title="fpage">610</span><span class="lpage" tagx="lpage" title="lpage">620</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2017.04.008">10.1016/j.cell.2017.04.008</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28457610">28457610</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hirsch</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Zika Virus infection of rhesus macaques leads to viral persistence in multiple tissues</span><span class="source" tagx="source" title="source">PLoS Pathog.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">e1006219</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1006219">10.1371/journal.ppat.1006219</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28278237">28278237</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Savino</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mendes-da-Cruz</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lepletier</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dardenne</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hormonal control of T-cell development in health and disease</span><span class="source" tagx="source" title="source">Nat. Rev. Endocrinol.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">77</span><span class="lpage" tagx="lpage" title="lpage">89</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nrendo.2015.168">10.1038/nrendo.2015.168</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26437623">26437623</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Braun</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Productive and Persistent Infection of Human Thymic Epithelial Cells <i>in Vitro</i> with HIV-1</span><span class="source" tagx="source" title="source">Virology</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">225</span><span class="fpage" tagx="fpage" title="fpage">413</span><span class="lpage" tagx="lpage" title="lpage">418</span><span class="pub-id"><a href="https://dx.doi.org/10.1006/viro.1996.0617">10.1006/viro.1996.0617</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8918929">8918929</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maguer-Satta</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gazzolo</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dodon</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span></span><span class="mixed-article-title" title="mixed-article-title">Human Immature Thymocytes as Target Cells of the Leukemogenic Activity of Human T-cell Leukemia Virus Type I</span><span class="source" tagx="source" title="source">Blood</span><span class="year" tagx="year" title="year">1995</span><span class="volume" tagx="volume" title="volume">86</span><span class="fpage" tagx="fpage" title="fpage">1444</span><span class="lpage" tagx="lpage" title="lpage">1452</span><span class="pub-id"><a href="https://dx.doi.org/10.1182/blood.V86.4.1444.bloodjournal8641444">10.1182/blood.V86.4.1444.bloodjournal8641444</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7632951">7632951</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manca</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Detection of HTLV-I tax-rex and pol gene sequences of thymus gland in a large group of patients with myasthenia gravis</span><span class="source" tagx="source" title="source">J. Acquir. immune Defic. Syndr.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">300</span><span class="lpage" tagx="lpage" title="lpage">306</span><span class="pub-id"><a href="https://dx.doi.org/10.1097/00126334-200203010-00012">10.1097/00126334-200203010-00012</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11873081">11873081</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moreira-Ramos</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castro</span><span class="given-names" tagx="given-names" title="given-names">FMM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linhares-Lacerda</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Savino</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="mixed-article-title" title="mixed-article-title">Can thymic epithelial cells be infected by human T-lymphotropic virus type 1?</span><span class="source" tagx="source" title="source">Mem. Inst. Oswaldo Cruz</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">106</span><span class="fpage" tagx="fpage" title="fpage">759</span><span class="lpage" tagx="lpage" title="lpage">762</span><span class="pub-id"><a href="https://dx.doi.org/10.1590/S0074-02762011000600018">10.1590/S0074-02762011000600018</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22012233">22012233</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carvalho Barros</span><span class="given-names" tagx="given-names" title="given-names">LR</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">HTLV-1-infected thymic epithelial cells convey the virus to CD4 + T lymphocytes</span><span class="source" tagx="source" title="source">Immunobiology</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">222</span><span class="fpage" tagx="fpage" title="fpage">1053</span><span class="lpage" tagx="lpage" title="lpage">1063</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.imbio.2017.08.001">10.1016/j.imbio.2017.08.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28888743">28888743</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernández</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Establishment and characterization of cloned human thymic epithelial cell lines</span><span class="source" tagx="source" title="source">Analysis of adhesion molecule expression and cytokine production. Blood</span><span class="year" tagx="year" title="year">1994</span><span class="volume" tagx="volume" title="volume">83</span><span class="fpage" tagx="fpage" title="fpage">3245</span><span class="lpage" tagx="lpage" title="lpage">3254</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7514905">7514905</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ribeiro-Carvalho</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Triiodothyronine modulates thymocyte migration</span><span class="source" tagx="source" title="source">Scand. J. Immunol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">66</span><span class="fpage" tagx="fpage" title="fpage">17</span><span class="lpage" tagx="lpage" title="lpage">25</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-3083.2007.01928.x">10.1111/j.1365-3083.2007.01928.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17587342">17587342</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Henchal</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gentry</span><span class="given-names" tagx="given-names" title="given-names">MK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">CCown</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brandt</span><span class="given-names" tagx="given-names" title="given-names">WE</span></span></span><span class="mixed-article-title" title="mixed-article-title">Dengue Virus-Specific and Flavivirus Group Determinants Identified with Monoclonal Antibodies by Indirect Immunofluorescence</span><span class="source" tagx="source" title="source">Am. J. Trop. Med. Hyg.</span><span class="year" tagx="year" title="year">1982</span><span class="volume" tagx="volume" title="volume">31</span><span class="fpage" tagx="fpage" title="fpage">830</span><span class="lpage" tagx="lpage" title="lpage">836</span><span class="pub-id"><a href="https://dx.doi.org/10.4269/ajtmh.1982.31.830">10.4269/ajtmh.1982.31.830</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6285749">6285749</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paessens</span><span class="given-names" tagx="given-names" title="given-names">LC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">SK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernandes</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Kooyk</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Vascular cell adhesion molecule-1 (VCAM-1) and intercellular adhesion molecule-1 (ICAM-1) provide co-stimulation in positive selection along with survival of selected thymocytes</span><span class="source" tagx="source" title="source">Mol. Immunol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">45</span><span class="fpage" tagx="fpage" title="fpage">42</span><span class="lpage" tagx="lpage" title="lpage">8</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.molimm.2007.05.016">10.1016/j.molimm.2007.05.016</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17604837">17604837</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foo</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Asian Zika virus strains target CD14+blood monocytes and induce M2-skewed immunosuppression during pregnancy</span><span class="source" tagx="source" title="source">Nat. Microbiol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">1558</span><span class="lpage" tagx="lpage" title="lpage">1570</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41564-017-0016-3">10.1038/s41564-017-0016-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28827581">28827581</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michlmayr</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andrade</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gonzalez</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balmaseda</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="mixed-article-title" title="mixed-article-title">CD14+CD16+monocytes are the main target of Zika virus infection in peripheral blood mononuclear cells in a paediatric study in Nicaragua</span><span class="source" tagx="source" title="source">Nat. Microbiol.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">1462</span><span class="lpage" tagx="lpage" title="lpage">1470</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41564-017-0035-0">10.1038/s41564-017-0035-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28970482">28970482</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kannan</span><span class="given-names" tagx="given-names" title="given-names">RP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hensley</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Evers</span><span class="given-names" tagx="given-names" title="given-names">LE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemon</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGivern</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span></span><span class="mixed-article-title" title="mixed-article-title">Hepatitis C Virus Infection Causes Cell Cycle Arrest at the Level of Initiation of Mitosis</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">85</span><span class="fpage" tagx="fpage" title="fpage">7989</span><span class="lpage" tagx="lpage" title="lpage">8001</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00280-11">10.1128/JVI.00280-11</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21680513">21680513</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Strange</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Green</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siemann</span><span class="given-names" tagx="given-names" title="given-names">DN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gale</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verma</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Immunoprofiles of human Sertoli cells infected with Zika virus reveals unique insights into host-pathogen crosstalk</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">15</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-018-27027-7">10.1038/s41598-018-27027-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29311619">29311619</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Limonta</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Human fetal astrocytes infected with zika virus exhibit delayed apoptosis and resistance to interferon: Implications for persistence</span><span class="source" tagx="source" title="source">Viruses</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">19</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/v10110646">10.3390/v10110646</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singh</span><span class="given-names" tagx="given-names" title="given-names">PK</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Determination of system level alterations in host transcriptome due to Zika virus (ZIKV) Infection in retinal pigment epithelium</span><span class="source" tagx="source" title="source">Sci. Rep.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">16</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-017-17765-5">10.1038/s41598-017-17765-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29311619">29311619</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Ling</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Zhenyu</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Xin</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Shun</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hui</span><span class="given-names" tagx="given-names" title="given-names">Yi</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">Hexi</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">Yanhua</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">Guohua</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Xiaohui</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Xiaoqing</span></span></span><span class="mixed-article-title" title="mixed-article-title">Protection of ZIKV infection-induced neuropathy by abrogation of acute antiviral response in human neural progenitors</span><span class="source" tagx="source" title="source">Cell Death &amp;amp; Differentiation</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">26</span><span class="issue" tagx="issue" title="issue">12</span><span class="fpage" tagx="fpage" title="fpage">2607</span><span class="lpage" tagx="lpage" title="lpage">2621</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41418-019-0324-7">10.1038/s41418-019-0324-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30952992">30952992</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bertoni</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alabiso</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galetto</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baldanzi</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Integrins in T cell physiology</span><span class="source" tagx="source" title="source">Int. J. Mol. Sci.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">E485</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/ijms19020485">10.3390/ijms19020485</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29415483">29415483</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lannes-Vieira</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Extracellular matrix components of the mouse thymic microenvironment. III. Thymic epithelial cells express the VLA6 complex that is involved in laminin-mediated interactions with thymocytes</span><span class="source" tagx="source" title="source">Int. Immunol.</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">1421</span><span class="lpage" tagx="lpage" title="lpage">1430</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/intimm/5.11.1421">10.1093/intimm/5.11.1421</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8260456">8260456</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Le</span><span class="given-names" tagx="given-names" title="given-names">PT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singer</span><span class="given-names" tagx="given-names" title="given-names">KH</span></span></span><span class="mixed-article-title" title="mixed-article-title">Human thymic epithelial cells: adhesion molecules and cytokine production</span><span class="source" tagx="source" title="source">Int. J. Clin. Lab. Res.</span><span class="year" tagx="year" title="year">1993</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">56</span><span class="lpage" tagx="lpage" title="lpage">60</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/BF02592284">10.1007/BF02592284</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8518415">8518415</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ocampo</span><span class="given-names" tagx="given-names" title="given-names">JSP</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Laminin-211 controls thymocyte-thymic epithelial cell interactions</span><span class="source" tagx="source" title="source">Cell. Immunol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">254</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cellimm.2008.06.005">10.1016/j.cellimm.2008.06.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18644586">18644586</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Savino</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mendes-da-Cruz</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Golbert</span><span class="given-names" tagx="given-names" title="given-names">DCF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riederer</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cotta-de-Almeida</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="mixed-article-title" title="mixed-article-title">Laminin-Mediated Interactions in Thymocyte Migration and Development</span><span class="source" tagx="source" title="source">Front. Immunol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">579</span><span class="pub-id"><a href="https://dx.doi.org/10.3389/fimmu.2015.00579">10.3389/fimmu.2015.00579</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26635793">26635793</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dairaghi</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Macrophage Inflammatory Protein-1 β Induces Migration and Activation of Human Thymocytes</span><span class="source" tagx="source" title="source">Blood</span><span class="year" tagx="year" title="year">1998</span><span class="volume" tagx="volume" title="volume">91</span><span class="fpage" tagx="fpage" title="fpage">2905</span><span class="lpage" tagx="lpage" title="lpage">2913</span><span class="pub-id"><a href="https://dx.doi.org/10.1182/blood.V91.8.2905.2905_2905_2913">10.1182/blood.V91.8.2905.2905_2905_2913</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9531601">9531601</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Savino</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mendes-da-Cruz</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silva</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dardenne</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cotta-de-Almeida</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="mixed-article-title" title="mixed-article-title">Intrathymic T-cell migration: a combinatorial interplay of extracellular matrix and chemokines?</span><span class="source" tagx="source" title="source">Trends Immunol.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">305</span><span class="lpage" tagx="lpage" title="lpage">313</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1471-4906(02)02224-X">10.1016/S1471-4906(02)02224-X</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12072370">12072370</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zamarchi</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Expression and functional activity of CXCR-4 and CCR-5 chemokine receptors in human thymocytes</span><span class="source" tagx="source" title="source">Clin. Exp. Immunol.</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">127</span><span class="fpage" tagx="fpage" title="fpage">321</span><span class="lpage" tagx="lpage" title="lpage">330</span><span class="pub-id"><a href="https://dx.doi.org/10.1046/j.1365-2249.2002.01775.x">10.1046/j.1365-2249.2002.01775.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11876757">11876757</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bunting</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Comerford</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kara</span><span class="given-names" tagx="given-names" title="given-names">EE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Korner</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McColl</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span></span><span class="mixed-article-title" title="mixed-article-title">CCR6 supports migration and differentiation of a subset of DN1 early thymocyte progenitors but is not required for thymic nTreg development</span><span class="source" tagx="source" title="source">Immunol. Cell Biol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">92</span><span class="fpage" tagx="fpage" title="fpage">489</span><span class="lpage" tagx="lpage" title="lpage">498</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/icb.2014.14">10.1038/icb.2014.14</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24638065">24638065</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singer</span><span class="given-names" tagx="given-names" title="given-names">KH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Denning</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whichard</span><span class="given-names" tagx="given-names" title="given-names">LP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haynes</span><span class="given-names" tagx="given-names" title="given-names">BF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Thymocyte LFA-1 and thymic epithelial cell ICAM-1 molecules mediate binding of activated human thymocytes to thymic epithelial cells</span><span class="source" tagx="source" title="source">J. Immunol.</span><span class="year" tagx="year" title="year">1990</span><span class="volume" tagx="volume" title="volume">144</span><span class="fpage" tagx="fpage" title="fpage">2931</span><span class="lpage" tagx="lpage" title="lpage">9</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1691223">1691223</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Circular RNAs: A novel type of non-coding RNA and their potential implications in antiviral immunity</span><span class="source" tagx="source" title="source">Int. J. Biol. Sci.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">1497</span><span class="lpage" tagx="lpage" title="lpage">1506</span><span class="pub-id"><a href="https://dx.doi.org/10.7150/ijbs.22531">10.7150/ijbs.22531</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29230098">29230098</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cadena</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hur</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Antiviral Immunity and Circular RNA: No End in Sight</span><span class="source" tagx="source" title="source">Mol. Cell</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">67</span><span class="fpage" tagx="fpage" title="fpage">163</span><span class="lpage" tagx="lpage" title="lpage">164</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.molcel.2017.07.005">10.1016/j.molcel.2017.07.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28732203">28732203</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishitsuji</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Long noncoding RNA #32 contributes to antiviral responses by controlling interferon-stimulated gene expression</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">113</span><span class="fpage" tagx="fpage" title="fpage">10388</span><span class="lpage" tagx="lpage" title="lpage">10393</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1525022113">10.1073/pnas.1525022113</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27582466">27582466</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carnero</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Long noncoding RNA EGOT negatively affects the antiviral response and favors HCV replication</span><span class="source" tagx="source" title="source">EMBO Rep.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">1013</span><span class="lpage" tagx="lpage" title="lpage">1028</span><span class="pub-id"><a href="https://dx.doi.org/10.15252/embr.201541763">10.15252/embr.201541763</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27283940">27283940</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Long non-coding RNA PICART1 suppresses proliferation and promotes apoptosis in lung cancer cells by inhibiting JAK2/STAT3 signaling</span><span class="source" tagx="source" title="source">Neoplasma</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">65</span><span class="fpage" tagx="fpage" title="fpage">779</span><span class="lpage" tagx="lpage" title="lpage">789</span><span class="pub-id"><a href="https://dx.doi.org/10.4149/neo_2018_171130N778">10.4149/neo_2018_171130N778</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29940776">29940776</a></span></span></li> <li tag="ref"><a name="CR43" /><span class="label" tagx="label" title="label">43.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">LncRNA PICART1 suppressed non-small cell lung cancer cells proliferation and invasion by targeting AKT1 signaling pathway</span><span class="source" tagx="source" title="source">Am. J. Clin. Res.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">4193</span><span class="lpage" tagx="lpage" title="lpage">4201</span></span></li> <li tag="ref"><a name="CR44" /><span class="label" tagx="label" title="label">44.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lyu</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">SENCR stabilizes vascular endothelial cell adherens junctions through interaction with CKAP4</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. USA</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">116</span><span class="fpage" tagx="fpage" title="fpage">546</span><span class="lpage" tagx="lpage" title="lpage">555</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1810729116">10.1073/pnas.1810729116</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30584103">30584103</a></span></span></li> <li tag="ref"><a name="CR45" /><span class="label" tagx="label" title="label">45.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dejana</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orsenigo</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">Endothelial adherens junctions at a glance</span><span class="source" tagx="source" title="source">J. Cell Sci.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">126</span><span class="fpage" tagx="fpage" title="fpage">2545</span><span class="lpage" tagx="lpage" title="lpage">2549</span><span class="pub-id"><a href="https://dx.doi.org/10.1242/jcs.124529">10.1242/jcs.124529</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23781019">23781019</a></span></span></li> <li tag="ref"><a name="CR46" /><span class="label" tagx="label" title="label">46.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hale</span><span class="given-names" tagx="given-names" title="given-names">LP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Markert</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span></span><span class="mixed-article-title" title="mixed-article-title">Corticosteroids Regulate Epithelial Cell Differentiation and Hassall Body Formation in the Human Thymus</span><span class="source" tagx="source" title="source">J. Immunol.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">172</span><span class="fpage" tagx="fpage" title="fpage">617</span><span class="lpage" tagx="lpage" title="lpage">624</span><span class="pub-id"><a href="https://dx.doi.org/10.4049/jimmunol.172.1.617">10.4049/jimmunol.172.1.617</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14688374">14688374</a></span></span></li> <li tag="ref"><a name="CR47" /><span class="label" tagx="label" title="label">47.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Savino</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Hydrocortisone increases the numbers of KL1+ cells, a discrete thymic epithelial cell subset characterized by high molecular weight cytokeratin expression</span><span class="source" tagx="source" title="source">Endocrinology</span><span class="year" tagx="year" title="year">1988</span><span class="volume" tagx="volume" title="volume">123</span><span class="fpage" tagx="fpage" title="fpage">2557</span><span class="lpage" tagx="lpage" title="lpage">64</span><span class="pub-id"><a href="https://dx.doi.org/10.1210/endo-123-5-2557">10.1210/endo-123-5-2557</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2458915">2458915</a></span></span></li> <li tag="ref"><a name="CR48" /><span class="label" tagx="label" title="label">48.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Valdespino-Vázquez</span><span class="given-names" tagx="given-names" title="given-names">MY</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Congenital Zika Syndrome and Extra-Central Nervous System Detection of Zika Virus in a Pre-term Newborn in Mexico</span><span class="source" tagx="source" title="source">Clin. Infect. Dis.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">68</span><span class="fpage" tagx="fpage" title="fpage">903</span><span class="lpage" tagx="lpage" title="lpage">912</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/cid/ciy616">10.1093/cid/ciy616</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30188990">30188990</a></span></span></li> <li tag="ref"><a name="CR49" /><span class="label" tagx="label" title="label">49.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonaldo</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation of Infective Zika Virus from Urine and Saliva of Patients in Brazil</span><span class="source" tagx="source" title="source">PLoS Negl. Trop. Dis.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">e0004816</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0004816">10.1371/journal.pntd.0004816</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27341420">27341420</a></span></span></li> <li tag="ref"><a name="CR50" /><span class="label" tagx="label" title="label">50.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Messias</span><span class="given-names" tagx="given-names" title="given-names">CV</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Zika virus infects human blood mononuclear cells</span><span class="source" tagx="source" title="source">BMC Infect. Dis.</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">986</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12879-019-4622-y">10.1186/s12879-019-4622-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31752731">31752731</a></span></span></li> <li tag="ref"><a name="CR51" /><span class="label" tagx="label" title="label">51.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamel</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Biology of Zika Virus Infection in Human Skin Cells</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">89</span><span class="fpage" tagx="fpage" title="fpage">8880</span><span class="lpage" tagx="lpage" title="lpage">8896</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00354-15">10.1128/JVI.00354-15</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26085147">26085147</a></span></span></li> <li tag="ref"><a name="CR52" /><span class="label" tagx="label" title="label">52.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ribeiro-Carvalho</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farias-de-Oliveira</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Villa-Verde</span><span class="given-names" tagx="given-names" title="given-names">DMS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Savino</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="mixed-article-title" title="mixed-article-title">Triiodothyronine modulates extracellular matrix-mediated interactions between thymocytes and thymic microenvironmental cells</span><span class="source" tagx="source" title="source">Neuroimmunomodulation</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">142</span><span class="lpage" tagx="lpage" title="lpage">152</span><span class="pub-id"><a href="https://dx.doi.org/10.1159/000067175">10.1159/000067175</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12481154">12481154</a></span></span></li> <li tag="ref"><a name="CR53" /><span class="label" tagx="label" title="label">53.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linhares-Lacerda</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">RNA interference-mediated knockdown of CD49e (α5 integrin chain) in human thymic epithelial cells modulates the expression of multiple genes and decreases thymocyte adhesion</span><span class="source" tagx="source" title="source">BMC Genomics</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">S2</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2164-11-S5-S2">10.1186/1471-2164-11-S5-S2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21210968">21210968</a></span></span></li> <li tag="ref"><a name="CR54" /><span class="label" tagx="label" title="label">54.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bolger</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lohse</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Usadel</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Trimmomatic: A flexible trimmer for Illumina sequence data</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">2114</span><span class="lpage" tagx="lpage" title="lpage">2120</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btu170">10.1093/bioinformatics/btu170</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404">24695404</a></span></span></li> <li tag="ref"><a name="CR55" /><span class="label" tagx="label" title="label">55.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dobin</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">STAR: Ultrafast universal RNA-seq aligner</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">15</span><span class="lpage" tagx="lpage" title="lpage">21</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/bts635">10.1093/bioinformatics/bts635</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23104886">23104886</a></span></span></li> <li tag="ref"><a name="CR56" /><span class="label" tagx="label" title="label">56.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langmead</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slazberg</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fast gapped-read alignmnet with Bowtie 2</span><span class="source" tagx="source" title="source">Nat. Methods</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">357</span><span class="lpage" tagx="lpage" title="lpage">359</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.1923">10.1038/nmeth.1923</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286">22388286</a></span></span></li> <li tag="ref"><a name="CR57" /><span class="label" tagx="label" title="label">57.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Love</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huber</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anders</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq. 2</span><span class="source" tagx="source" title="source">Genome Biol.</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">550</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13059-014-0550-8">10.1186/s13059-014-0550-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25516281">25516281</a></span></span></li> <li tag="ref"><a name="CR58" /><span class="label" tagx="label" title="label">58.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cava</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">SpidermiR: An R/bioconductor package for integrative analysis with miRNA data</span><span class="source" tagx="source" title="source">Int. J. Mol. Sci.</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">E274</span><span class="pub-id"><a href="https://dx.doi.org/10.3390/ijms18020274">10.3390/ijms18020274</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28134831">28134831</a></span></span></li> <li tag="ref"><a name="CR59" /><span class="label" tagx="label" title="label">59.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paraskevopoulou</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">DIANA-microT web server v5.0: service integration into miRNA functional analysis workflows</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">41</span><span class="fpage" tagx="fpage" title="fpage">169</span><span class="lpage" tagx="lpage" title="lpage">173</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkt393">10.1093/nar/gkt393</a></span></span></li> <li tag="ref"><a name="CR60" /><span class="label" tagx="label" title="label">60.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">John</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Human microRNA targets</span><span class="source" tagx="source" title="source">PLoS Biol.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">e363</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pbio.0020363">10.1371/journal.pbio.0020363</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15502875">15502875</a></span></span></li> <li tag="ref"><a name="CR61" /><span class="label" tagx="label" title="label">61.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krek</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Combinatorial microRNA target predictions</span><span class="source" tagx="source" title="source">Nat. Genet.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">37</span><span class="fpage" tagx="fpage" title="fpage">495</span><span class="lpage" tagx="lpage" title="lpage">500</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/ng1536">10.1038/ng1536</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15806104">15806104</a></span></span></li> <li tag="ref"><a name="CR62" /><span class="label" tagx="label" title="label">62.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agarwal</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bell</span><span class="given-names" tagx="given-names" title="given-names">GW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nam</span><span class="given-names" tagx="given-names" title="given-names">J-W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartel</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span></span><span class="mixed-article-title" title="mixed-article-title">Predicting effective microRNA target sites in mammalian mRNAs</span><span class="source" tagx="source" title="source">Elife</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">38</span><span class="pub-id"><a href="https://dx.doi.org/10.7554/eLife.05005">10.7554/eLife.05005</a></span></span></li> <li tag="ref"><a name="CR63" /><span class="label" tagx="label" title="label">63.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shannon</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks</span><span class="source" tagx="source" title="source">Genome Res.</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">2498</span><span class="lpage" tagx="lpage" title="lpage">2504</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/gr.1239303">10.1101/gr.1239303</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14597658">14597658</a></span></span></li> <li tag="ref"><a name="CR64" /><span class="label" tagx="label" title="label">64.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashburner</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Gene Ontology: tool for the unification of biology. The Gene Ontology Consortium</span><span class="source" tagx="source" title="source">Nat. Genet.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">25</span><span class="lpage" tagx="lpage" title="lpage">29</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/75556">10.1038/75556</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651">10802651</a></span></span></li> <li tag="ref"><a name="CR65" /><span class="label" tagx="label" title="label">65.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fabregat</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The Reactome Pathway Knowledgebase</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">46</span><span class="fpage" tagx="fpage" title="fpage">D649</span><span class="lpage" tagx="lpage" title="lpage">D655</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkx1132">10.1093/nar/gkx1132</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29145629">29145629</a></span></span></li> <li tag="ref"><a name="CR66" /><span class="label" tagx="label" title="label">66.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L-G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">Q-Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">clusterProfiler: an R package for comparing biological themes among gene clusters</span><span class="source" tagx="source" title="source">OMICS</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">284</span><span class="lpage" tagx="lpage" title="lpage">287</span><span class="pub-id"><a href="https://dx.doi.org/10.1089/omi.2011.0118">10.1089/omi.2011.0118</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22455463">22455463</a></span></span></li> </ul> </div> </div>  </body></html>