<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Wellcome Open Res</span><span class="iso-abbrev" title="iso-abbrev">Wellcome Open Res</span><span class="pmc" title="pmc">Wellcome Open Res</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Wellcome Open Research</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2398-502X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">F1000 Research Limited</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London, UK</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7001756">7001756</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/32090172">32090172</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.12688/wellcomeopenres.15518.2">10.12688/wellcomeopenres.15518.2</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Method Article</span>: <span class="subject" title="subject">Articles</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Rapid in-country sequencing of whole virus genomes to inform rabies elimination programmes</div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p>[version 2; peer review: 3 approved]</p> </div> </div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunker</span><span class="given-names" tagx="given-names" title="given-names">Kirstyn</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Data Curation</span><span class="role" tagx="role" title="role">Formal Analysis</span><span class="role" tagx="role" title="role">Funding Acquisition</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Project Administration</span><span class="role" tagx="role" title="role">Validation</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – Original Draft Preparation</span><span class="role" tagx="role" title="role">Writing – Review &amp;amp; Editing</span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-9990-6299</span><a href="#c1">a</a><a href="#a1">1</a><a href="#a2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaswant</span><span class="given-names" tagx="given-names" title="given-names">Gurdeep</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#a1">1</a><a href="#a3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thumbi</span><span class="given-names" tagx="given-names" title="given-names">S.M.</span></span><span class="role" tagx="role" title="role">Funding Acquisition</span><span class="role" tagx="role" title="role">Project Administration</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Supervision</span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-5754-0556</span><a href="#a3">3</a><a href="#a4">4</a><a href="#a5">5</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lushasi</span><span class="given-names" tagx="given-names" title="given-names">Kennedy</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-2060-4202</span><a href="#a6">6</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lugelo</span><span class="given-names" tagx="given-names" title="given-names">Ahmed</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#a7">7</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Czupryna</span><span class="given-names" tagx="given-names" title="given-names">Anna M.</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Project Administration</span><a href="#a1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ade</span><span class="given-names" tagx="given-names" title="given-names">Fred</span></span><span class="role" tagx="role" title="role">Resources</span><a href="#a4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wambura</span><span class="given-names" tagx="given-names" title="given-names">Gati</span></span><span class="role" tagx="role" title="role">Resources</span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-3137-2355</span><a href="#a4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chuchu</span><span class="given-names" tagx="given-names" title="given-names">Veronicah</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#a4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steenson</span><span class="given-names" tagx="given-names" title="given-names">Rachel</span></span><span class="role" tagx="role" title="role">Visualization</span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-1579-6596</span><a href="#a1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ngeleja</span><span class="given-names" tagx="given-names" title="given-names">Chanasa</span></span><span class="role" tagx="role" title="role">Resources</span><a href="#a8">8</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bautista</span><span class="given-names" tagx="given-names" title="given-names">Criselda</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#a9">9</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manalo</span><span class="given-names" tagx="given-names" title="given-names">Daria L.</span></span><span class="role" tagx="role" title="role">Resources</span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-9290-5146</span><a href="#a9">9</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomez</span><span class="given-names" tagx="given-names" title="given-names">Ma. Ricci R.</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#a9">9</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">Maria Yna Joyce V.</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-6709-2534</span><a href="#a9">9</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miranda</span><span class="given-names" tagx="given-names" title="given-names">Mary Elizabeth</span></span><span class="role" tagx="role" title="role">Resources</span><a href="#a9">9</a><a href="#a10">10</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamat</span><span class="given-names" tagx="given-names" title="given-names">Maya</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-7792-2084</span><a href="#a1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rysava</span><span class="given-names" tagx="given-names" title="given-names">Kristyna</span></span><span class="role" tagx="role" title="role">Resources</span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-3475-931X</span><a href="#a11">11</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Espineda</span><span class="given-names" tagx="given-names" title="given-names">Jason</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#a12">12</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silo</span><span class="given-names" tagx="given-names" title="given-names">Eva Angelica V.</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#a12">12</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aringo</span><span class="given-names" tagx="given-names" title="given-names">Ariane Mae</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#a12">12</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bernales</span><span class="given-names" tagx="given-names" title="given-names">Rona P.</span></span><span class="role" tagx="role" title="role">Data Curation</span><a href="#a12">12</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adonay</span><span class="given-names" tagx="given-names" title="given-names">Florencio F.</span></span><span class="role" tagx="role" title="role">Data Curation</span><a href="#a13">13</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tildesley</span><span class="given-names" tagx="given-names" title="given-names">Michael J.</span></span><span class="role" tagx="role" title="role">Resources</span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-6875-7232</span><a href="#a11">11</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marston</span><span class="given-names" tagx="given-names" title="given-names">Denise A.</span></span><span class="role" tagx="role" title="role">Resources</span><a href="#a14">14</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jennings</span><span class="given-names" tagx="given-names" title="given-names">Daisy L.</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#a14">14</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fooks</span><span class="given-names" tagx="given-names" title="given-names">Anthony R.</span></span><span class="role" tagx="role" title="role">Resources</span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-3243-6154</span><a href="#a14">14</a><a href="#a15">15</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Wenlong</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#a1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meredith</span><span class="given-names" tagx="given-names" title="given-names">Luke W.</span></span><span class="role" tagx="role" title="role">Resources</span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-3802-8290</span><a href="#a16">16</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hill</span><span class="given-names" tagx="given-names" title="given-names">Sarah C.</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#a17">17</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poplawski</span><span class="given-names" tagx="given-names" title="given-names">Radoslaw</span></span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Software</span><a href="#a18">18</a><a href="#a19">19</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gifford</span><span class="given-names" tagx="given-names" title="given-names">Robert J.</span></span><span class="role" tagx="role" title="role">Data Curation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Software</span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-4028-9884</span><a href="#a20">20</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singer</span><span class="given-names" tagx="given-names" title="given-names">Joshua B.</span></span><span class="role" tagx="role" title="role">Data Curation</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Software</span><span class="role" tagx="role" title="role">Writing – Review &amp;amp; Editing</span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0001-6989-475X</span><a href="#a20">20</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maturi</span><span class="given-names" tagx="given-names" title="given-names">Mathew</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#a21">21</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mwatondo</span><span class="given-names" tagx="given-names" title="given-names">Athman</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#a21">21</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biek</span><span class="given-names" tagx="given-names" title="given-names">Roman</span></span><span class="role" tagx="role" title="role">Writing – Review &amp;amp; Editing</span><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-3471-5357</span><a href="#a1">1</a><a href="#a2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hampson</span><span class="given-names" tagx="given-names" title="given-names">Katie</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Data Curation</span><span class="role" tagx="role" title="role">Funding Acquisition</span><span class="role" tagx="role" title="role">Project Administration</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Supervision</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – Original Draft Preparation</span><span class="role" tagx="role" title="role">Writing – Review &amp;amp; Editing</span><a href="#a1">1</a><a href="#a2">2</a></span><span class="citation_author_institution" id="a1">[1], </span><span class="citation_author_institution" id="a2">[2], </span><span class="citation_author_institution" id="a3">[3], </span><span class="citation_author_institution" id="a4">[4], </span><span class="citation_author_institution" id="a5">[5], </span><span class="citation_author_institution" id="a6">[6], </span><span class="citation_author_institution" id="a7">[7], </span><span class="citation_author_institution" id="a8">[8], </span><span class="citation_author_institution" id="a9">[9], </span><span class="citation_author_institution" id="a10">[10], </span><span class="citation_author_institution" id="a11">[11], </span><span class="citation_author_institution" id="a12">[12], </span><span class="citation_author_institution" id="a13">[13], </span><span class="citation_author_institution" id="a14">[14], </span><span class="citation_author_institution" id="a15">[15], </span><span class="citation_author_institution" id="a16">[16], </span><span class="citation_author_institution" id="a17">[17], </span><span class="citation_author_institution" id="a18">[18], </span><span class="citation_author_institution" id="a19">[19], </span><span class="citation_author_institution" id="a20">[20], </span><span class="citation_author_institution" id="a21">[21], </span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">a</span><span class="email" tagx="email" title="email">kirstyn.brunker@glasgow.ac.uk</span></div> <div class="fn-type-COI-statement" title="COI-statement"> <p>No competing interests were disclosed.</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-5-5</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">5</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">3</span><span class="history" title="history"><span class="accepted" title="accepted">accepted: 2020-5-12</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the Creative Commons Attribution Licence, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</span></span></div><a class="self-uri" href="wellcomeopenres-5-17517.pdf" title="self-uri">wellcomeopenres-5-17517.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Genomic surveillance is an important aspect of contemporary disease management but has yet to be used routinely to monitor endemic disease transmission and control in low- and middle-income countries. Rabies is an almost invariably fatal viral disease that causes a large public health and economic burden in Asia and Africa, despite being entirely vaccine preventable. With policy efforts now directed towards achieving a global goal of zero dog-mediated human rabies deaths by 2030, establishing effective surveillance tools is critical. Genomic data can provide important and unique insights into rabies spread and persistence that can direct control efforts. However, capacity for genomic research in low- and middle-income countries is held back by limited laboratory infrastructure, cost, supply chains and other logistical challenges. Here we present and validate an end-to-end workflow to facilitate affordable whole genome sequencing for rabies surveillance utilising nanopore technology. We used this workflow in Kenya, Tanzania and the Philippines to generate rabies virus genomes in two to three days, reducing costs to approximately £60 per genome. This is over half the cost of metagenomic sequencing previously conducted for Tanzanian samples, which involved exporting samples to the UK and a three- to six-month lag time. Ongoing optimization of workflows are likely to reduce these costs further. We also present tools to support routine whole genome sequencing and interpretation for genomic surveillance. Moreover, combined with training workshops to empower scientists in-country, we show that local sequencing capacity can be readily established and sustainable, negating the common misperception that cutting-edge genomic research can only be conducted in high resource laboratories. More generally, we argue that the capacity to harness genomic data is a game-changer for endemic disease surveillance and should precipitate a new wave of researchers from low- and middle-income countries.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
dog-mediated rabies</div> <div class="kwd" title="kwd">
field sequencing</div> <div class="kwd" title="kwd">
lyssavirus</div> <div class="kwd" title="kwd">
MinION</div> <div class="kwd" title="kwd">
nanopore</div> <div class="kwd" title="kwd">
neglected tropical diseases</div> <div class="kwd" title="kwd">
phylogenetic</div> <div class="kwd" title="kwd">
rabies virus</div> <div class="kwd" title="kwd">
whole genome sequencing</div> <div class="kwd" title="kwd">
zoonoses</div> <div class="kwd" title="kwd">
surveillance</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
New Partnership for Africa's Development</div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Scottish Government</div><span class="award-id" tagx="award-id" title="award-id">SE0431</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Royal Society of Tropical Medicine and Hygiene</div><span class="award-id" tagx="award-id" title="award-id">GR000892</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Department for International Development, UK Government</div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Department for Environment, Food and Rural Affairs</div><span class="award-id" tagx="award-id" title="award-id">SE0431</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Wellcome Trust</div><span class="award-id" tagx="award-id" title="award-id">207569</span><span class="award-id" tagx="award-id" title="award-id">204820</span><span class="award-id" tagx="award-id" title="award-id">110330</span><span class="award-id" tagx="award-id" title="award-id">107753</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
African Academy of Sciences</div><span class="award-id" tagx="award-id" title="award-id">DEL-15-008</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Scottish Funding Council</div><span class="award-id" tagx="award-id" title="award-id">EP/S51584X/1</span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
UK Research and Innovation</div><span class="award-id" tagx="award-id" title="award-id">EP/S51584X/1</span></div> <div class="funding-statement" title="funding-statement">
This work was supported by the Wellcome Trust through a Senior Research Fellowship to KH [207569], a Training Fellowship in Public Health and Tropical Medicine to SMT [110330] and Institutional Strategic Support Fund grants to the University of Glasgow [204820]. KB was supported by a Royal Society of Tropical Medicine and Hygiene Small Grant 2017 [GR000892] and the GCRF/Scottish Funding Council through a grant awarded to the University of Glasgow [EP/S51584X/1]. KL is supported by the DELTAS Africa Initiative [Afrique One-ASPIRE/DEL-15-008]. Afrique One-ASPIRE is funded by a consortium of donors including the African Academy of Sciences (AAS) Alliance for Accelerating Excellence in Science in Africa (AESA), the New Partnership for Africa’s Development Planning and Coordinating (NEPAD) Agency, the Wellcome Trust [107753] and the UK government. The work at APHA was supported by funding from Defra, the Scottish Government and Welsh Government [SE0431]. </div> <div class="funding-statement" title="funding-statement"><i>The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</i></div> </div> </div> <div class="version-changes" title="version-changes"><span class="label" tagx="label" title="label">Revised</span><div class="title" tagx="title" title="title">
Amendments from Version 1</div> <p>Version 2 of the manuscripts has been revised based on three reviewer's comments. We also provide more detailed phylogenetic trees in Figures 6 and 7.</p> </div> </div> <div class="body" title="body"> <div class="abbreviations" title="sec"> <div class="title" tagx="title" title="title">
Abbreviations</div> <p>RABV, rabies virus; LMICs, low- and middle-income countries; ONT, Oxford Nanopore Technology; APHA, Animal and Plant Health Agency; RITM, Research Institute for Tropical Medicine; UNITID, University of Nairobi Institute of Tropical and Infectious Diseases; WGS, whole genome sequencing; RIDT, rapid immunodiagnostic test.</p> </div> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Surveillance is critical to inform infectious disease management and control programmes ( <a href="#ref-41">Klepac <i>et al.</i>, 2013</a>; <a href="#ref-54">Molyneux <i>et al.</i>, 2004</a>) and genomic data is emerging as a powerful new surveillance tool ( <a href="#ref-21">Cotton <i>et al.</i>, 2018</a>). Pathogen sequencing has become increasingly routine in well-resourced public health laboratories, but in low- and middle-income countries (LMICs) where most emerging and endemic zoonoses occur, sequencing capacity is more limited and the direct application of genomics-informed surveillance is still at an early stage ( <a href="#ref-3">Balloux <i>et al.</i>, 2018</a>; <a href="#ref-28">Folarin <i>et al.</i>, 2014</a>; <a href="#ref-29">Gardy &amp;amp; Loman, 2018</a>).</p> <p>Oxford Nanopore Technologies’ (ONT) MinION machine has revolutionized genomic sequencing capacity ( <a href="#ref-38">Johnson <i>et al.</i>, 2017</a>). This portable, low-cost sequencer has supported innovative "lab-in-a-suitcase�? platforms to diagnose crop pathogens <i>in situ</i> ( <a href="#ref-8">Boykin <i>et al.</i>, 2018</a>; <a href="#ref-9">Boykin <i>et al.</i>, 2019</a>) and enable real-time outbreak surveillance of emerging pathogens, such as Ebola, Yellow Fever and Zika viruses ( <a href="#ref-90">Faria <i>et al.</i>, 2018</a>; <a href="#ref-27">Faria <i>et al.</i>, 2016</a>; <a href="#ref-59">Quick <i>et al.</i>, 2015</a>; <a href="#ref-61">Quick <i>et al.</i>, 2016</a>; <a href="#ref-60">Quick <i>et al.</i>, 2017</a>). There is potential to extend genomic technologies to endemic pathogens such as rabies, foot-and-mouth disease, brucellosis and bovine tuberculosis, which are responsible for major impacts on people’s health and livelihoods on a daily basis ( <a href="#ref-20">Cleaveland <i>et al.</i>, 2017</a>; <a href="#ref-22">Danielsen <i>et al.</i>, 2019</a>; <a href="#ref-33">Halliday <i>et al.</i>, 2015</a>). Such an approach could reduce the lag time from sample to sequence and epidemiological action that is typical for these neglected diseases, while simultaneously building genomic capacity in LMICs. Investing in the surveillance of these endemic pathogens builds core capacities necessary for the detection of unusual disease emergence events that precede epidemics ( <a href="#ref-20">Cleaveland <i>et al.</i>, 2017</a>; <a href="#ref-34">Halliday <i>et al.</i>, 2017</a>).</p> <p>Genomic surveillance has potential applications for management of a range of endemic zoonoses. Here we focus on the development and deployment of genomic surveillance tools to support rabies control and elimination programmes. Rabies is a devastating, invariably fatal zoonotic disease caused by a single stranded negative-sense RNA virus ( <a href="#ref-18">Brunker &amp;amp; Mollentze, 2018</a>). Rabies is globally widespread and can infect and be transmitted by all mammal species, but the vast majority of human deaths worldwide result from bites by infected domestic dogs ( <a href="#ref-73">WHO, 2018a</a>). Canine rabies elimination is feasible through sustained well-implemented mass dog vaccination programmes ( <a href="#ref-2">Abela-Ridder <i>et al.</i>, 2016</a>; <a href="#ref-44">Lankester <i>et al.</i>, 2014</a>; <a href="#ref-74">WHO, 2018b</a>). A global initiative to end human deaths from dog-mediated rabies by 2030 is now underway ( <a href="#ref-74">WHO, 2018b</a>) and practical, effective surveillance tools are required to support this campaign ( <a href="#ref-4">Benavides <i>et al.</i>, 2019</a>; <a href="#ref-5">Bitek <i>et al.</i>, 2019</a>; <a href="#ref-10">Broban <i>et al.</i>, 2018</a>; <a href="#ref-36">Hampson <i>et al.</i>, 2016</a>; <a href="#ref-55">Mtema <i>et al.</i>, 2016</a>).</p> <p>Genomic surveillance can inform global efforts to eliminate dog-mediated rabies and may be particularly valuable during the endgame, when the disease can circulate for extended periods at low levels ( <a href="#ref-41">Klepac <i>et al.</i>, 2013</a>). Endemic rabies typically circulates at low levels in dog populations (e.g. 58 to 384 cases per 100,000 dogs in low to high incidence settings in Tanzania, <a href="#ref-36">Hampson <i>et al.,</i> 2016</a>) and during endgame stages incidence can drop below detectable levels with traditional surveillance, often based on laboratory-based confirmation of submitted suspect cases. Enhanced surveillance including integrated bite case management can increase case detection up to ten-fold ( <a href="#ref-104">Suseno <i>et al.,</i> 2019</a>) and genetic data from cases detected during this period is critical in determining their role in maintaining circulation and dispersing infection. For example: differentiating dog rabies variants from other wildlife variants ( <a href="#ref-47">Lembo <i>et al.</i>, 2007</a>); distinguishing ongoing endemic transmission from incursions ( <a href="#ref-7">Bourhy <i>et al.</i>, 2016</a>; <a href="#ref-53">Mollentze <i>et al.</i>, 2014b</a>; <a href="#ref-76">Zinsstag <i>et al.</i>, 2017</a>); and identifying sources of incursions ( <a href="#ref-51">Mollentze <i>et al.</i>, 2013</a>; <a href="#ref-67">Tohma <i>et al.</i>, 2016</a>). Furthermore, the capacity for an informed rapid response to newly detected cases will be pivotal for control efforts to succeed ( <a href="#ref-37">Henderson, 2011</a>).</p> <p>Here we establish methods for rapid whole genome sequencing (WGS) of rabies viruses (RABV) using the MinION sequencer, including a workflow from sample to sequence and the associated bioinformatics pipeline. We also incorporate procedures for downstream analysis, interpretation, and management of genomic data in a surveillance context. Through testing and replication of methods across different locations in Kenya, Tanzania and the Philippines we demonstrate the feasibility of real-time sequencing of RABV to rapidly inform policy decisions and disease management. We further highlight the potential for cost savings that could make routine genomic surveillance of rabies affordable in low-resource endemic settings.</p> </div> <div class="methods" title="methods"> <div class="title" tagx="title" title="title">
Methods</div> <p>Rabies positive samples were processed under a range of conditions to generate near whole genome sequences for viral characterization and phylogenetic analysis. Here we describe the sequencing and bioinformatics pipelines and validation process, detailing adaptations required for different settings. Our aim was to establish a protocol that could be used under minimal laboratory infrastructure; therefore, the approach is supported by a lab-in-a-suitcase platform: a portable toolkit to support the steps required to get from sample to sequence ( <a href="#f1">Figure 1</a>). A comprehensive, up-to-date sample-to-sequence laboratory protocol is available on <a href="http://dx.doi.org/10.17504/protocols.io.ba4figtn">protocols.io</a> and supporting pipelines can be found on <a href="https://github.com/kirstyn/rapid-rabies">GitHub</a> will be updated with improvements as methods develop. The ARTIC network, a Wellcome funded project to develop the application of genomic surveillance for viral outbreak response, provides comprehensive open-source resources for laboratory and sequencing work, bioinformatics, phylogenetics and analysis. We take advantage of these resources for our rabies workflow. They can be found here on the <a href="https://artic.network/">ARTIC network website</a> and are published in <a href="#ref-60">Quick <i>et al.</i>, 2017</a>. We also utilise <a href="http://rabv.glue.cvr.ac.uk/#/home">RABV-GLUE</a> ( <a href="#ref-64">Singer <i>et al.</i>, 2018</a>), a general-purpose resource for analysis, interpretation, and management of RABV genome data, to organise and interpret RABV consensus sequences.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1. </span></a></div>   <div class="title" tagx="title" title="title">
Rabies "lab-in-a-suitcase�? setup for sequencing in low-resource settings.</div> <p><b>A</b>) Lab-in-a-suitcase being used at the Nelson Mandela African Institute of Science and Technology, Tanzania; <b>B</b>) RNA extractions performed in the back of a vehicle using a portable glove box and battery-powered centrifuge in Makueni District, Kenya; <b>C</b>) Laboratory setup for sample inactivations in a district livestock office in Mugumu, Serengeti District, Tanzania. </p>   </div> <div class="samples" title="sec"> <div class="title" tagx="title" title="title">
Samples</div> <p>Brain tissue samples were collected by field officers and veterinarians from suspect rabid animals as part of rabies surveillance and research projects across Kenya (n = 9), Tanzania (n = 26, with 2 archived samples processed in the UK) and the Philippines (n = 52). These samples were accessible through existing collaborations and time allowed for their processing whilst optimizing our protocols and delivering training workshops, therefore numbers do not necessarily represent the surveillance capacity in each locality. Samples were obtained predominantly from domestic dogs but also included livestock and wildlife samples; details are provided as underlying data ( <a href="#ref-12">Brunker, 2019b</a>). Samples were processed in settings with varying infrastructure and resources, from central diagnostic laboratories to field sites. Details of each site can be found in <a href="#T1">Table 1</a> and <a href="#f2">Figure 2</a>. Rapid immunodiagnostic tests (RIDT), manufactured by BioNote, Inc. (RG18.01, Gyeongi-do, Republic of Korea), were used to confirm rabies positive cases in the field in Tanzania. These kits have been validated as an effective rabies field diagnostic tool ( <a href="#ref-45">Léchenne <i>et al.</i>, 2016</a>). In Kenya, samples were confirmed positive by the Kenya Medical Research Institute (KEMRI) using the direct rapid immunohistochemical test (dRIT) and tested by RIDT prior to sequencing. In the Philippines samples were tested by the Research Institute for Tropical Medicine (RITM) using the fluorescent antibody test (FAT) ( <a href="#ref-1">Abela-Ridder, 2018</a>).</p> <div class="table-wrap_UNKNOWN" id="T1" orientation="portrait" position="anchor"><span class="label" tagx="label" title="label">Table 1. </span> <div class="title" tagx="title" title="title">
Locations across Kenya, Tanzania, the Philippines and UK where rabies virus sequencing was conducted.</div> <p>The number of samples sequenced, number of MinION sequencing runs and resources available at each location are detailed. (*six out of the 11 samples were repeats of FIELD2 samples).</p>  <table frame="hsides" rules="groups" content-type="article-table"> <thead> <tr> <th align="left" valign="top" rowspan="1" colspan="1">Site</th> <th align="left" valign="top" rowspan="1" colspan="1">Abbreviation</th> <th align="left" valign="top" rowspan="1" colspan="1">Location <br /><br />(City, <br /><br />Country)</th> <th align="left" valign="top" rowspan="1" colspan="1"># Samples <br /><br />sequenced <br /><br />(# MinION <br /><br />runs)</th> <th align="left" valign="top" rowspan="1" colspan="1">Year</th> <th align="left" valign="top" rowspan="1" colspan="1">Classification</th> <th align="center" colspan="6" valign="top" rowspan="1">Resources</th> </tr> <tr> <th rowspan="1" colspan="1" /> <th rowspan="1" colspan="1" /> <th rowspan="1" colspan="1" /> <th rowspan="1" colspan="1" /> <th rowspan="1" colspan="1" /> <th rowspan="1" colspan="1" /> <th align="left" valign="top" rowspan="1" colspan="1">Containment <br /><br />level 2</th> <th align="left" valign="top" rowspan="1" colspan="1">Freezer <br /><br />-80°C</th> <th align="left" valign="top" rowspan="1" colspan="1">Freezer <br /><br />-20°C</th> <th align="left" valign="top" rowspan="1" colspan="1">Laboratory</th> <th align="left" valign="top" rowspan="1" colspan="1">Back-up <br /><br />Generator</th> <th align="left" valign="top" rowspan="1" colspan="1">Standard lab <br /><br />equipment <sup><a href="#TFN1">1</a></sup></th> </tr> </thead> <tbody> <tr> <td align="left" valign="top" rowspan="1" colspan="1">Tanzania Veterinary <br /><br />Laboratories Agency</td> <td align="left" valign="top" rowspan="1" colspan="1">TVLA</td> <td align="left" valign="top" rowspan="1" colspan="1">Dar es <br /><br />Salaam, <br /><br />Tanzania</td> <td align="left" valign="top" rowspan="1" colspan="1">9 (2)</td> <td align="left" valign="top" rowspan="1" colspan="1">2017</td> <td align="left" valign="top" rowspan="1" colspan="1">Research/ <br /><br />diagnostic lab</td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> </tr> <tr> <td align="left" valign="top" rowspan="1" colspan="1">Research Institute for <br /><br />Tropical Medicine</td> <td align="left" valign="top" rowspan="1" colspan="1">RITM</td> <td align="left" valign="top" rowspan="1" colspan="1">Manilla, <br /><br />Philippines</td> <td align="left" valign="top" rowspan="1" colspan="1">52 (3)</td> <td align="left" valign="top" rowspan="1" colspan="1">2019</td> <td align="left" valign="top" rowspan="1" colspan="1">Research/ <br /><br />diagnostic lab</td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> </tr> <tr> <td align="left" valign="top" rowspan="1" colspan="1">Mugumu District field <br /><br />office</td> <td align="left" valign="top" rowspan="1" colspan="1">FIELD1</td> <td align="left" valign="top" rowspan="1" colspan="1">Mugumu, <br /><br />Tanzania</td> <td align="left" valign="top" rowspan="1" colspan="1">4 (2)</td> <td align="left" valign="top" rowspan="1" colspan="1">2017</td> <td align="left" valign="top" rowspan="1" colspan="1">Field site</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" valign="top" rowspan="1" colspan="1">University of Nairobi <br /><br />Institute of Tropical and <br /><br />Infectious Diseases</td> <td align="left" valign="top" rowspan="1" colspan="1">UNITID</td> <td align="left" valign="top" rowspan="1" colspan="1">Nairobi, <br /><br />Kenya</td> <td align="left" valign="top" rowspan="1" colspan="1">11* (1)</td> <td align="left" valign="top" rowspan="1" colspan="1">2019</td> <td align="left" valign="top" rowspan="1" colspan="1">Research <br /><br />laboratory</td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" valign="top" rowspan="1" colspan="1">Makeuni Field site</td> <td align="left" valign="top" rowspan="1" colspan="1">FIELD2</td> <td align="left" valign="top" rowspan="1" colspan="1">Makueni <br /><br />District, <br /><br />Kenya</td> <td align="left" valign="top" rowspan="1" colspan="1">4 (1)</td> <td align="left" valign="top" rowspan="1" colspan="1">2019</td> <td align="left" valign="top" rowspan="1" colspan="1">Field site (RNA <br /><br />extractions only)</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> </tr> <tr> <td align="left" valign="top" rowspan="1" colspan="1">Institute of Biodiversity, <br /><br />Animal Health &amp;amp; <br /><br />Comparative Medicine, <br /><br />University of Glasgow</td> <td align="left" valign="top" rowspan="1" colspan="1">IBAHCM</td> <td align="left" valign="top" rowspan="1" colspan="1">Glasgow, <br /><br />UK</td> <td align="left" valign="top" rowspan="1" colspan="1">1 (1)</td> <td align="left" valign="top" rowspan="1" colspan="1">2017</td> <td align="left" valign="top" rowspan="1" colspan="1">Research <br /><br />laboratory (from <br /><br />RNA stage)</td> <td align="left" rowspan="1" colspan="1" /> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> </tr> <tr> <td align="left" valign="top" rowspan="1" colspan="1">Animal &amp;amp; Plant Health <br /><br />Agency</td> <td align="left" valign="top" rowspan="1" colspan="1">APHA</td> <td align="left" valign="top" rowspan="1" colspan="1">Surrey, UK</td> <td align="left" valign="top" rowspan="1" colspan="1">1 (1)</td> <td align="left" valign="top" rowspan="1" colspan="1">2016</td> <td align="left" valign="top" rowspan="1" colspan="1">Research/ <br /><br />diagnostic lab</td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b><br /><br />CL3</td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> </tr> <tr> <td align="left" valign="top" rowspan="1" colspan="1">Nelson Mandela <br /><br />Institute of Biodiversity, <br /><br />Animal Health &amp;amp; <br /><br />Comparative Medicine</td> <td align="left" valign="top" rowspan="1" colspan="1">NMAIST</td> <td align="left" valign="top" rowspan="1" colspan="1">Arusha, <br /><br />Tanzania</td> <td align="left" valign="top" rowspan="1" colspan="1">12 (3)</td> <td align="left" valign="top" rowspan="1" colspan="1">2017, <br /><br />2018</td> <td align="left" valign="top" rowspan="1" colspan="1">Research <br /><br />laboratory</td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> <td align="left" rowspan="1" colspan="1" /> <td align="left" valign="top" rowspan="1" colspan="1"><b>✓</b></td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p id="TFN1">[1] Centrifuge and thermocycler available for use</p> </div> </div> </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2. </span></a></div>   <div class="title" tagx="title" title="title">
Locations in East Africa where sequencing of rabies viruses was performed.</div> <p>Circles are scaled according to numbers of samples sequenced. Details of laboratory capacity at each site are provided as underlying data ( <a href="#ref-12">Brunker, 2019b</a>).</p>   </div> </div> <div class="equipment" title="sec"> <div class="title" tagx="title" title="title">
Equipment</div> <p>A full suite of portable, battery-powered equipment and laboratory consumables was put together based on the ARTIC network’s published kit list. Reagents that required cold chain transportation were carried in polystyrene temperature control boxes packed with ice packs frozen at -20 or -80°C prior to use. This equipment, including reagent boxes, could be packed into standard airline sized checked luggage (lab-in-a-suitcase) for transportation ( <a href="#f1">Figure 1</a>).</p> </div> <div class="samplepreparationandrnaextraction" title="sec"> <div class="title" tagx="title" title="title">
Sample preparation and RNA extraction</div> <p>Methods to perform sample processing and RNA extraction varied according to resources available on site. In research or diagnostic laboratories, samples were handled in containment level (CL) 2+ or CL3 conditions until sample inactivation. At field sites, samples were handled in a portable glove box (UY-33666-50, Cole Parmer). RNA extraction was performed using the Zymo Quick RNA Miniprep kit (R1054, Zymo Research) according to the manufacturer’s protocol. Sample preparation varied according to the state of the sample (fresh/frozen, homogenised/non-homogenised) and its preservation (RNAlater [Invitrogen], DNA/RNA Shield [Zymo Research], 50% glycerol-saline solution). Recommended methods for different sample types can be found in the manufacturer’s instructions but generally the following protocol was used. A small amount of brain tissue (50–100mg) was transferred from the original sample tube, using a wooden applicator stick or toothpick, to a 2ml reinforced homogeniser tube containing 1.4mm ceramic beads (Qiagen) and ~1ml of Zymo kit RNA lysis buffer. The sample was homogenised using a Terralyzer (Zymo Research) with 30 second pulses until fully homogenised and left to inactivate for 2 minutes. When a Terralyzer was unavailable, samples were manually homogenised using a toothpick. Lysed samples were processed according to manufacturer’s column-based RNA purification protocol with in-column DNase I treatment to remove genomic DNA. In brief, lysed samples were added to filter columns and subject to a series of centrifugation steps to selectively bind RNA and purify (ethanol washes) and finally eluted. Purified viral RNA was eluted in 50µl DNase/RNase-Free Water. Aliquots of four samples stored in RNA later were shipped on dry ice from Tanzania to the Animal &amp;amp; Plant Health Agency (APHA, Weybridge UK) in 2017. These, and two samples used for validation (see below), were extracted according to a previously described Trizol-based method at APHA ( <a href="#ref-17">Brunker <i>et al.</i>, 2015</a>). Briefly, 50–100mg of brain tissue was homogenised in 1ml Trizol and 200µl chloroform added to enable phase separation. RNA separates into the upper aqueous phase, which is removed and RNA precipitated using isopropanol. This is followed by centrifugation to pellet RNA and ethanol washes to purify, with elution in DNase/RNase-Free Water.</p> </div> <div class="multiplexprimerdesignandvalidation" title="sec"> <div class="title" tagx="title" title="title">
Multiplex primer design and validation</div> <p>Multiplex primer schemes were designed for RABV using <a href="http://primal.zibraproject.org/">Primal Scheme</a>. For Tanzania, a set of 10 reference sequences from datasets curated and filtered in RABV-GLUE were chosen to represent the known RABV diversity in Tanzania (in file order submitted to Primal Scheme: <a href="https://www.ncbi.nlm.nih.gov/nuccore/1016870541/">KR906748.1</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KR906747.1">KR906747.1</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KY210263.1">KY210263.1</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KY210264.1">KY210264.1</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KY210227.1">KY210227.1</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KR906774.1">KR906774.1</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KR906763.1">KR906763.1</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KR906742.1">KR906742.1</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KR534218.2">KR534218.2</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KY210305.1">KY210305.1</a>). Settings were applied to generate primers for 400bp amplicons across the entire genome (~11923bp), with an overlap of 50bp. A short amplicon length was chosen due to previous experience of poor sample quality from field collected and stored samples resulting in RNA fragmentation and a low viral copy number ( <a href="#ref-17">Brunker <i>et al.</i>, 2015</a>; <a href="#ref-16">Brunker <i>et al.</i>, 2018</a>). The first scheme created, rabvTanzDg, was tested on archived Tanzanian RABV RNA samples at APHA following the protocol described in <a href="#ref-61">Quick <i>et al.</i> (2016)</a>. Two sequencing validations were performed: 1) Amplicons from one sample, RV3227, were sequenced on an Illumina NextSeq to confirm the amplification of individual products across the genome. This sample had already been sequenced using a PCR-free metagenomic approach, briefly described in section "Additional Sequencing�? (described in <a href="#ref-17">Brunker <i>et al.</i>, 2015</a>), providing a comparison between library preparations; 2) Subsequently, amplicons from another sample (RAB16025) were sequenced on both the MinION and NextSeq (fragmented before sequencing on NextSeq). In addition, metagenomic sequencing was conducted on the NextSeq for this sample. A second RABV scheme was produced for the Philippines (rabvSEasia) following the same approach and used directly for MinION sequencing (reference sequences: <a href="https://www.ncbi.nlm.nih.gov/nuccore/KX148260">KX148260</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KX148263">KX148263</a>, N-gene 99% consensus sequence of [ <a href="https://www.ncbi.nlm.nih.gov/nuccore/AB116581">AB116581</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/AB116582">AB116582</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/AB683592">AB683592</a>: <a href="https://www.ncbi.nlm.nih.gov/nuccore/AB683635">AB683635</a>], <a href="https://www.ncbi.nlm.nih.gov/nuccore/AB981664">AB981664</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KX148255">KX148255</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/JN786878">JN786878</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KX148250">KX148250</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KX148254">KX148254</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/EU293111">EU293111</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KX148248">KX148248</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KX148266">KX148266</a>).</p> </div> <div class="dnapreparation" title="sec"> <div class="title" tagx="title" title="title">
DNA preparation</div> <p>Protocols and reagents varied slightly due to availability and as protocols evolved; the latest version of the protocol is available on the GitHub resource and Figshare ( <a href="#ref-14">Brunker, 2019d</a>). Initially, extracted RNA was reverse transcribed using Protoscript II First Strand Sequencing kit (E6560, NEB) under manufacturer’s instructions. At later stages this was changed to Lunascript RT Supermix (E3010, NEB) in a reduced reaction volume of 10μl (2μl Lunascript + 3μl nuclease-free water +5μl RNA). Multiplex PCR was conducted with 2 to 2.5 µL of cDNA, the RABV primer scheme and Q5 High Fidelity Hot-Start DNA Polymerase (M0493, NEB) following the two-step PCR protocol ( <a href="#ref-60">Quick <i>et al.</i>, 2017</a>), with the following PCR cycling conditions: 98°C heat inactivation for 30sec, followed by 30–40 cycles of denaturation at 98°C for 15sec and a combined extension/annealing step of 65°C for 5min. PCR products were purified using a 1x solid-phase reversible immobilization bead (SPRI) cleanup (Ampure XP, Beckman-Coulter or PCRClean DX,Aline Biosciences) and concentrations measured using a Qubit dsDNA High Sensitivity kit (Q32851, Thermofisher) on a Qubit 3.0 fluorimeter (ThermoFisher). In field sites, or when standard thermocyclers in laboratories were unavailable, PCR was performed using a portable MiniPCR machine (Amplyus). This was powered by a lithium battery (PowerAdd Pilot Pro2 85Wh LiION 9-20V) for the full PCR cycle when power was intermittent or unavailable. A negative control (water) was included at the cDNA synthesis stage for each batch of samples and as part of the sequencing library. Some of the first runs in Tanzania did not include a negative control as part of the sequencing library as the protocol was still being finalised. However, a negative control up to the PCR stage was always included as part of sample preparation. If the negative control was &amp;gt;1ng/μl by Qubit quantification, sample preparation was repeated from RNA extraction.</p> </div> <div class="minionlibrarypreparationandsequencing" title="sec"> <div class="title" tagx="title" title="title">
MinION library preparation and sequencing</div> <p>Library preparation for the MinION was conducted using Ligation Sequencing 1D (SQK-LSK108/SQK-LSK109, ONT) and Native Barcoding kits (EXP-NBD103/EXP-NBD104/EXP-NBD114, ONT) on FLO-MIN106 (R9.4/R9.4.1, ONT) flowcells according to the manufacturer’s instructions and modifications as per <a href="#ref-60">Quick <i>et al.</i> (2017)</a>. Briefly, DNA prepared in the previous step (PCR amplicon pools for each sample were pooled) was end-repaired and dA-tailed using an UltraII End Prep Reaction Module (E7442, NEB) followed by ligation of barcodes using the NEBNext UltraII Ligation module (E7595, NEB) in a "one-pot�? reaction. Following barcode ligation, samples were purified using a SPRI bead cleanup and pooled together into a single tube before ONT adaptor ligation with the NEBNext UltraII Ligation module. Ligated DNA was then cleaned using a SPRI bead clean-up with ABB/SFB (SQK-LSK108/SQK-LSK109, ONT) washes and final library elution in ONT’s elution buffer (SQK-LSK108/SQK-LSK109, ONT). With the latest kit versions, up to 24 native barcodes are available, but multiplexing varied by run according to available samples and barcodes, with multiplexed MinION runs of three to 24 samples conducted. Sequencing libraries were prepared with a negative control (non-template control from cDNA synthesis), which was sequenced as a barcoded sample. Sequencing was performed on a Dell Latitude E5470 laptop (CPU: 6th Generation Intel Core i7-6820HQ [Quad Core, 2.7GHz, 8MB cache]; Memory: 16GB RAM; Storage: 1TB solid state drive) using MinKNOW (version available at the time, v1.7.3 to 18.12.9) with live basecalling turned off. The MinION functions as a "read-until�? platform, whereby the length of sequencing runs is determined by the user when sufficient data has been produced (maximum 48h); therefore, this varied between runs. An offline version of MinKNOW provided by ONT was used due to varying internet quality at different locations. To ease the installation and use of bioinformatic pipelines in the field, a lab-on-an-SSD approach was used to ensure a working bioinformatic environment was always available. The lab-on-an-SSD is a bootable USB3 drive containing a linux installation, ONT software and bioinformatic analysis software, available from the ARTIC network resources, that offers a plug-and-play bioinformatic environment for most laptops. SSD image available as an open-source resource here: <a href="https://github.com/artic-network/fieldbioinformatics/tree/master/lab-on-an-ssd">https://github.com/artic-network/fieldbioinformatics/tree/master/lab-on-an-ssd</a>),</p> </div> <div class="additionalsequencing" title="sec"> <div class="title" tagx="title" title="title">
Additional sequencing</div> <p>Four samples shipped to the UK and extracted at APHA were sequenced using the metagenomic approach previously described ( <a href="#ref-17">Brunker <i>et al.</i>, 2015</a>). Briefly, this involved host genomic DNA depletion of extracted total RNA using the on-column DNase treatment in RNeasy plus mini kit (74134, Qiagen) followed by first- and second-strand cDNA synthesis with a cDNA synthesis system kit (04379012001, Roche) and library preparation with a Nextera XT sample preparation kit (FC-131-1024, Illumina).</p> </div> <div class="costestimates" title="sec"> <div class="title" tagx="title" title="title">
Cost estimates</div> <p>The costs per sequenced sample were estimated for the MinION-based multiplex PCR approach and the previously used Illumina-based metagenomic approach ( <a href="#ref-17">Brunker <i>et al.</i>, 2015</a>; <a href="#ref-16">Brunker <i>et al.</i>, 2018</a>). These estimates include processing from sample to sequence, i.e. incorporating all costs from the workflow shown in <a href="#f3">Figure 3</a> and any transportation costs associated with the shipment of samples to laboratory facilities. Reagents and consumable costings assume a bulk buy for at least 100 samples. For metagenomic sequencing, costs were based on previous quotes from UK sequencing facilities to process RABV samples, including sample shipment from Tanzania to the UK.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 3. </span></a></div>   <div class="title" tagx="title" title="title">
Workflow from sample to sequence.</div> <p><b>A</b>) Table of laboratory workflow steps with estimated timings and details of important reagents/consumables; <b>B</b>) Bioinformatics workflow diagram; <b>C</b>) Estimated cost per sample for end-to-end rabies virus sequencing in-country using a PCR-based MinION approach versus the PCR-free metagenomic approach previously used in the UK. Costs assume a bulk buy of reagents for 100 reactions and a multiplex capacity of 24 samples per flowcell for MinION sequencing. </p>   </div> </div> <div class="bioinformaticsanddatamanagement" title="sec"> <div class="title" tagx="title" title="title">
Bioinformatics and data management</div> <p><b><i>i) Consensus sequence generation.</i></b> Bioinformatic pipelines were run on the same laptop used for sequencing. A RABV version of the ARTIC network open-source bioinformatics pipeline ( <a href="#ref-60">Quick <i>et al.</i>, 2017</a>) was used to generate consensus sequences for each barcoded sample ( <a href="#f3">Figure 3B</a>). Briefly, raw files were basecalled with the latest ONT software at the time (Albacore or Guppy) and demultiplexed with Porechop using stringent settings to select reads with barcodes at both ends of the read. Demultiplexed reads were then mapped with BWA ( <a href="#ref-48">Li &amp;amp; Durbin, 2009</a>) to a reference genome chosen according to the country of origin (Tanzania/Kenya: Genbank accession no. <a href="https://www.ncbi.nlm.nih.gov/nuccore/KF155002">KF155002</a>; Philippines: Genbank accession no. <a href="https://www.ncbi.nlm.nih.gov/nuccore/KX148260">KX148260</a>). Prior to variant calling, primer sequences were trimmed using their genome coordinates and their genome coverage normalised. Variants were called using Nanopolish ( <a href="#ref-100">Loman <i>et al.</i>, 2015</a>), which uses MinION signal-level data to "polish�? the genome assembly. Consensus sequences were generated under the condition that non-overlapping primer binding sites and sites for which coverage was &amp;lt;20X were masked by ambiguity code N. Illumina reads were processed according to a previously described method ( <a href="#ref-17">Brunker <i>et al.</i>, 2015</a>). Briefly, Genome Analysis Toolkit ( <a href="#ref-57">Poplin <i>et al.,</i> 2017</a>) was used to identify high quality SNPs using filters on strand bias (FS&amp;gt;60, SOR&amp;gt;4), mapping quality (MQ&amp;lt;40, MQRankSum≤12.5), read position (ReadPosRankSum≤8), and depth of coverage (DP&amp;lt;5). Consensus sequences were produced with a 75% majority basecall rule. Raw and processed sequence data files were stored locally on the laptop’s internal hard drive or external SSDs. Data was transferred to University of Glasgow servers for storage and backup on return to the UK.</p> <p><b><i>ii) Analysis/interpretation.</i></b> Major and minor clade assignments were obtained for newly generated RABV sequences using Maximum Likelihood Clade Assignment (MLCA) as implemented in RABV-GLUE ( <a href="#ref-64">Singer <i>et al.</i>, 2018</a>). MCLA uses RAxML's Evolutionary Placement Algorithm ( <a href="#ref-65">Stamatakis, 2014</a>) to select high-likelihood branch placements for a query virus sequence in a previously calculated phylogeny of virus reference strains. The reference phylogeny in RABV-GLUE was reconstructed using maximum likelihood, and contains a representative set of RABV sequences obtained from recent studies of RABV diversity ( <a href="#ref-43">Kuzmin <i>et al.</i>, 2012</a>; <a href="#ref-69">Troupin <i>et al.</i>, 2016</a>). Following MLCA, RABV-GLUE was used to import relevant sequences from GenBank to compare to the new sequence data from each area i.e. East Africa and the Philippines. GenBank sequences were curated from the same minor clade associated with each area according to MLCA and both GenBank and MinION sequences were filtered to include only sequences with &amp;gt;90% genome coverage. For sequences produced for this study that was 24 out of 34 sequences from East Africa and 44 out of 52 sequences from Philippines. Alignments were produced using MAFFT ( <a href="#ref-40">Katoh &amp;amp; Standley, 2013</a>) implemented in RABV-GLUE. Maximum likelihood phylogenies were produced for each of the alignment sets using RAxML implemented in RABV-GLUE with a General Time Reversible model of nucleotide substitution with the gamma model of rate heterogeneity and a proportion of invariant sites and 1000 bootstraps. Outgroups consisting of RABV genomes from other minor clades were included in tree building.</p> </div> <div class="traininginminionsequencing" title="sec"> <div class="title" tagx="title" title="title">
Training in MinION sequencing</div> <p>Workshops were conducted in Kenya (11 participants) and the Philippines (10 participants) to provide hands-on training in MinION library preparation, sequencing and basic bioinformatics. Participants typically had a background in molecular biology but most had never prepared sequencing libraries. RABV RNA from local archived samples was used for sequencing following the sample-to-sequence method described above. Basic bioinformatic training included an introduction to the command line and use of the lab-on-an-SSD resource with the RABV version of the pipeline. In addition to workshops, training of local researchers was conducted on a one-to-one or small group basis at every opportunity. Consent from participants was requested as part of a questionnaire to give feedback on training workshops. Only those that consented provided feedback (75% of participants). Given that no identifying or sensitive information was collected, no ethical approval was sought for this questionnaire.</p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="validationofpipelineinuklaboratory" title="sec"> <div class="title" tagx="title" title="title">
Validation of pipeline in UK laboratory</div> <p>Amplicons from sample RV3227 sequenced on the NextSeq platform produced over 22million mapped reads and an average read depth of 6971 with &amp;gt;99% genome coverage. The same sample sequenced using the PCR-free metagenomic approach on the NextSeq produced 9498 mapped reads (0.4% of the total reads sequenced) with &amp;gt;99% genome coverage. Consensus sequences from each approach were identical. Amplicons from sample RAB16025 were sequenced on both NextSeq and MinION platforms, and metagenomic sequencing was also performed. Comparison between the three consensus sequences revealed 100% identity between Illumina consensus sequences (metagenomic vs PCR) and 99.98% identity between Illumina and MinION sequences. There were two SNPs between Illumina and MinION consensus sequences, highlighted in <a href="#f4">Figure 4</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4. </span></a></div>   <div class="title" tagx="title" title="title">
Genome coverage profiles for a Tanzanian dog rabies virus sample sequenced by different methods.</div> <p>Blue: PCR amplicons (fragmented by Nextera library preparation) sequenced on Illumina NextSeq; orange: PCR amplicons sequenced on MinION sequencer; yellow: metagenomic sequencing on Illumina NextSeq. Asterisks indicate locations of single nucleotide polymorphisms between Illumina and MinION consensus sequences.</p>   </div> </div> <div class="in-countrygenomesequencing" title="sec"> <div class="title" tagx="title" title="title">
In-country genome sequencing</div> <p>RABV samples were sequenced across a range of settings in Kenya, Tanzania and the Philippines between 2017–2019 ( <a href="#T1">Table 1</a> and <a href="#f2">Figure 2</a>). Using a lab-in-a-suitcase approach provided the flexibility to ensure that the necessary resources were available for sequencing in resource-limited settings but also in well-resourced laboratories if particular equipment was not available. In total, 84 brain tissue samples and one salivary gland sample were processed from sample to sequence in their respective country of origin. For all samples, a consensus was generated for all sites across the genome where coverage was ≥20x ( <a href="#ref-11">Brunker, 2019a</a>). The majority of samples (~80%) achieved &amp;gt;90% genome coverage at a depth of at least 20x ( <a href="#ref-12">Brunker, 2019b</a>). Due to practical constraints, sequencing run times varied; therefore, the total throughput from each run was different. Instances where coverage dropped below 20x may have been improved with longer sequencing times.</p> <p>The total processing time from sample to sequence-ready library for a batch of 24 samples was ~15 hours ( <a href="#f3">Figure 3A</a>). Sequencing run length varied batch to batch and was determined by practical constraints and the expected output required for 100x read coverage for <i>n</i> samples in a run (and assuming loss of reads during demultiplexing). Typically, we found 5–8 hours was sufficient to get adequate coverage for multiplexed runs. For most samples, the time taken to process them end-to-end was 2–3 days when accounting for reasonable working hours (which are often restricted in laboratories) and processing a large number of samples (up to 24) for each run. The most time-consuming parts of the protocol were sample preparation for RNA extraction due to variation in the quality of brain tissue; PCR setup for a large number of samples (24 samples = 48 PCR reactions); and the normalisation of PCR products for sequencing libraries - samples with high concentrations of DNA often required an additional dilution step to provide suitable volumes (&amp;gt;1μl) for pooling after normalisation ( <a href="#f3">Figure 3</a>). Workflow times can be reduced with multiple workers (estimated timings assume only one person is doing the work) and good preparation e.g. prior tube labelling/setup, the preparation of next steps during incubations, preparation of stock solutions ahead of time etc, but careful work is required to follow workflow steps to avoid mistakes and introducing contamination.</p> <p>Negative controls were included in each batch of samples processed and sequenced in most runs. Contamination was found in some batches, particularly during group training exercises (e.g. Philippines workshop batch), highlighting the importance of this quality control step. When contamination was detected during PCR quantification steps (concentration &amp;gt;1ng/μl), sample preparation was repeated from RNA extraction. However, some cross-contamination was not apparent until sequencing; therefore, a negative control should always be included in the sequencing library. A low level of cross-contamination was noted in all sequencing runs (a few hundred reads), often caused by amplicons with particularly high levels of amplification.</p> </div> <div class="costings" title="sec"> <div class="title" tagx="title" title="title">
Costings</div> <p>We estimated the full cost of sample-to-sequence preparation for RABV samples to give a better understanding of the costs involved to process field collected samples ( <a href="#ref-13">Brunker, 2019c</a>). The cost reduced as improvements to the protocol and multiplexing capacity were made. Early runs, which had a low number of samples per flowcell and were used mainly for validation, were approximately £160 per sample. This reduced to ~£60 per sample at full multiplexing capacity, i.e. 24 samples per flowcell ( <a href="#f3">Figure 3C</a>). Furthermore, the development of an effective washing procedure for flowcell reuse ("Nuclease-flush�?, Oxford Nanopore) means that, potentially, multiple runs per flowcell could be achieved. Therefore, we anticipate that this will reduce even further to ~£35 to £45 per sample (based on 2–6 reuses of one MinION flowcell). For example, a similar PCR-based sequencing approach for Zika virus in Brazil was able to yield ~150 samples per flowcell by making use of the nuclease-flush between runs (S. Hill, University of Oxford, personal communication). In comparison, sample shipment to the UK followed by sample-to-sequence costs for an Illumina-based metagenomic approach used previously for RABV WGS costs an estimated ~£205 per sample ( <a href="#f3">Figure 3C</a>). In terms of bioinformatic/data management costs, the SSDs used were typically purchased for £75–80 but are an optional component of the workflow and therefore not included in the overall estimate.</p> </div> <div class="training" title="sec"> <div class="title" tagx="title" title="title">
Training</div> <p>21 participants from veterinary and public health laboratories across Tanzania, Kenya and the Philippines took part in four-day sequencing workshops held in-country. Each participant prepared one sample during group laboratory sessions, providing hands-on experience of end-to-end preparation for MinION sequencing. Participants typically had experience in molecular laboratory methods such as PCR but most had never prepared sequencing libraries before. Feedback questionnaires revealed that participants were overall very satisfied with training workshops but indicated a desire for more comprehensive bioinformatics training. At the end of each workshop, participants were asked to submit a mini research proposal and a commitment from their research institute to support MinION genomic work in order to obtain a MinION starter pack and additional reagents. In total, four MinION starter kits were provided to groups of researchers from the workshops to kick-start the application of their new training skills for genomics research. At the time of writing none of the groups had conducted sequencing mostly due to problems procuring additional reagents (either due to funding or supplier issues). However, we have offered ongoing support e.g. provision of extra reagents, linking up with local expertise to ensure projects can be completed and all are expected to sequence within the next few months.</p> </div> <div class="consensussequenceanalysis" title="sec"> <div class="title" tagx="title" title="title">
Consensus sequence analysis</div> <p>Major and minor clade assignments were obtained for all sequences using RABV-GLUE (see example in <a href="#f5">Figure 5</a>), even for sequences with very poor consensus genome coverage of coding regions (as little as 15.1%; see underlying data; <a href="#ref-12">Brunker, 2019b</a>). Philippines sequences were all identified as Asian SEA4, a minor clade so far detected exclusively in the Philippines ( <a href="#ref-62">Saito <i>et al.</i>, 2013</a>; <a href="#ref-67">Tohma <i>et al.</i>, 2016</a>, source: RABV-GLUE). Subdivision within the Philippines subclade indicates regional phylogeographic structure, as found previously ( <a href="#ref-62">Saito <i>et al.</i>, 2013</a>; <a href="#ref-67">Tohma <i>et al.</i>, 2016</a>) but also further subdivision that may indicate structure at a finer spatial resolution ( <a href="#f6">Figure 6</a>). Tanzanian sequences were all from the Africa 1b lineage, as found previously ( <a href="#ref-17">Brunker <i>et al.</i>, 2015</a>; <a href="#ref-16">Brunker <i>et al.</i>, 2018</a>). Interestingly, we detected two minor RABV subclades in Kenya; Africa 1a and Africa 1b ( <a href="#f7">Figure 7</a>). The limited RABV genetic data available for Kenya prior to our work (one WGS, nine complete/partial nucleoprotein sequences) are all Africa 1b subclade. </p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 5. </span></a></div>   <div class="title" tagx="title" title="title">
Example output from the online RABV-GLUE: an online tool for automated genotyping and interpretation of rabies virus (RABV) sequence data.</div> <p><b>A</b>) Major and minor clade assignments for a Tanzanian RABV whole genome sequence (sample id=Rab16031, GenBank accession=MN726824); <b>B</b>) Phylogenetic placement of the sequence within the RABV-GLUE rabies virus reference phylogeny. This figure has been reproduced with permission from the University of Glasgow.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 6. </span></a></div>   <div class="title" tagx="title" title="title">
Maximum likelihood tree of rabies virus genomes from the Philippines.</div> <p>Genomes from the rabies virus Asian minor clade SEA4, which were sequenced in this study (n=44) and curated from GenBank (n=5). Colours indicate the administrative region associated with each sequence as shown on the map and internal node symbols indicate bootstrap support ≥80. Genbank sequences GU358653 and GU647092 representing Asian minor clades SEA2a and SEA2b were used as an outgroup (branch not shown). Administrative shapefiles were obtained from <a href="https://www.diva-gis.org/datadown">https://www.diva-gis.org/datadown</a> and plotted in R.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 7. </span></a></div>   <div class="title" tagx="title" title="title">
Maximum likelihood tree of rabies virus genomes from Kenya and Tanzania.</div> <p>Genomes belonging to two minor clades in the rabies virus Cosmopolitan clade are highlighted, Africa 1a and Africa 1b, which were sequenced in this study (tip shapes, n=24) and curated from GenBank (n=220). The country of origin and number of sequences in each minor clade are annotated on a map of Africa and the location of Pemba sequences from a 2016/17 outbreak is annotated on the tree, indicating multiple introductions from the mainland. Genbank sequences representing other minor clades ME1a, ME1b and ME2 (KX148162, KX148190, KX148191) were used as an outgroup (branch not shown). Maps were generated with package rWorldMaps in R.</p>   </div> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Through a series of sequencing applications and trainings conducted in different settings between 2017–2019, we demonstrate the feasibility of a cost-effective, in-country pipeline for WGS of RABV using the MinION. Although challenges exist, we show they are not insurmountable. Our pipeline offers both a scalable, low cost workflow that can be particularly beneficial for sequencing a large set of archived samples and a rapid sample-to-sequence turnaround for priority samples to inform decision-making for disease management. We share our protocols and guidance to facilitate WGS of RABV, even in resource-limited settings, discuss its limitations and prospects for ongoing application.</p> <p>Genomic data has become an important aspect of contemporary infectious disease surveillance ( <a href="#ref-29">Gardy &amp;amp; Loman, 2018</a>). Yet for rabies and other neglected endemic diseases it is underused - largely limited to academic studies and retrospective analyses with more limited actionable outputs. Even well-funded research projects can take up to six months from sample collection to dissemination of results (e.g. <a href="#ref-16">Brunker <i>et al.</i>, 2018</a>; <a href="#ref-9">Boykin <i>et al</i>., 2019</a>), impeding the quick decision-making necessary for timely disease control. Using our MinION-based sequencing protocol, RABV samples can be sequenced in-country in a matter of days and at a cost much lower than third-party sequencing facilities ( <a href="#f3">Figure 3C</a>), which typically requires the export of samples. We envision two broad applications of our protocols, for: 1) cost-effective batch sequencing of archived samples to characterize circulating RABV diversity, and 2) rapid <i>in situ</i> genomic surveillance of new cases, particularly during the endgame when such information can target appropriate responses. Many countries have repositories of archived tissue samples from rabid animals, which could be sequenced in bulk and therefore at low cost. These samples are potentially very valuable to characterise historical RABV diversity in a locality from which to compare to contemporary sequences. This comparison can show whether lineages continue to circulate or have disappeared, perhaps as a result of rabies control efforts. The latter option of rapid <i>in-situ</i> sequencing makes less use of cost-cutting measures such as multiplex sequencing, but the benefit of genomic surveillance to gain time-critical insights into persisting or re-emerging RABV far outweigh the expense of re-emergence. Further options to reduce costs when sample numbers are low include combining runs with other pathogens or using ONT’s low-cost, lower throughput Flongle flowcells (e.g. <a href="#ref-31">Grädel <i>et al.,</i> 2019</a>).</p> <p>To facilitate replicable sequencing in different settings, we provide detailed instructions for each stage of sample processing. Each stage independently is relatively simple, requiring standard molecular techniques, such as PCR, or adherence to step-by-step protocols. However, as an end-to-end protocol, it is intensive and ideally undertaken by experienced laboratory workers. The workflow needs to accommodate constraints on working hours and fit with other workloads to enable routine implementation. Realistically, the estimated 15-hour workflow for sample-to-sequence (for a batch of 24 samples) should be conducted over a 2–3 day period, but could be performed rapidly when required. Feedback from our training workshops indicated that hands-on training was valued and provided an appreciation of what it takes to produce quality sequence data. However, while useful as an introduction to MinION sequencing, workshops should be followed by extended one-on-one training and continued use to achieve full competency in performing the workflow.</p> <div class="challenges" title="sec"> <div class="title" tagx="title" title="title">
Challenges</div> <p>Sub-optimal storage due to frequent power cuts and limited local storage facilities are a challenge in LMICs. Previously we have encountered difficulties in WGS of RABV samples archived in laboratories under variable conditions for up to a decade due to RNA damage ( <a href="#ref-17">Brunker <i>et al.</i>, 2015</a>). We found that the amplicon-based approach ensured not only efficient viral read throughput on the MinION but also that samples varying in quality could be sequenced in the same way and with the best chance of success (a worst-case scenario approach). Generating longer amplicons for RABV sequencing ( <a href="#ref-56">Nadin-davis <i>et al.</i>, 2017</a>) has proven difficult for samples typical of these conditions (K. Brunker, unpublished report) but may be an option for <i>in situ</i> sequencing when samples are fresh. We were unable to perform quantitative reverse-transcription PCR (qRT-PCR) on samples during field trips due to lack of equipment, but if available, this could inform the sequencing approach and aid as a diagnostic tool ( <a href="#ref-49">Marston <i>et al.</i>, 2019</a>). In addition, sequencing can usefully validate diagnostic results, e.g. a cat sample in the Philippines was determined positive by dRIT but shown to be negative during sequencing preparation (K. Brunker, unpublished report). Notes from the dRIT on this sample indicated it had been difficult to interpret, therefore the sequence information provided vital data to confirm the sample had been misdiagnosed as a false positive.</p> <p>More generally, logistical challenges impact on infectious disease research in LMICs, from sample collection difficulties to limited internet and reliable electricity supplies and sourcing of reagents and consumables. A recent study found only two thirds of hospitals providing surgical care in 21 LMICs have a continuous electricity source or an available generator ( <a href="#ref-19">Chawla <i>et al.</i>, 2018</a>), problems that only worsen in more remote areas. To overcome these issues, we used a lab-in-a-suitcase approach, with portable equipment to enable quick setup in different locations using battery power if required. However, maintaining a cold chain was problematic. Access to generator backed -20°C (for reagents) and -80°C (RNA storage) freezers was limited, even in research facilities, and minor issues often taken for granted in UK labs (e.g. access to ice for benchwork with reagents) were common. ONT have recently developed a field version of some of their sequencing kits, containing lyophilised reagents to mitigate cold chain constraints. Lyophilised reagents could be an option for future work but are expensive and would require significant bulk purchasing to be affordable. Moreover, few local suppliers can provide sequencing-specific reagents and a considerable mark-up is added to already monopolised prices in research-rich countries like the UK/USA. Ordering directly from the company comes with its own difficulties, including expensive shipment costs, import taxes and airway bills for the delivery of reagents under cold-chain conditions. Delays at customs can compromise reagent viability and occurred frequently on our sequencing trips. To promote in-country scientists and genomic research, better solutions are needed for sustaining a reliable, affordable supply chain for reagents and consumables.</p> <p>Bioinformatics and data management proved to be a bottleneck in our pipeline and are a primary target for optimisation. The lab-on-an-SSD method (ARTIC network) simplified the installation of bioinformatic software and available pipelines, but as this is a developing resource, we did encounter issues that stalled analysis. This was largely due to limited experience and lack of access to a dedicated bioinformatician to troubleshoot problems. In general there is a critical need for bioinformatics training in LMICs, which has been recognised as a priority area for development, e.g. the <a href="https://h3abionet.org/">Pan African Bioinformatics Network</a> for the Human Heredity and Health in Africa consortium and NIH-funded Application of Genomics and Modelling to the Control of Virus Pathogens (GeMVi). Future work will focus on support for bioinformatic pipeline development, including optimising barcoding assignment and consensus building thresholds for RABV samples. This may involve the integration of Read Assignment, Mapping, and Phylogenetic Analysis in Real Time ( <a href="https://github.com/artic-network/rampart">RAMPART</a>), a promising new tool to assess genome coverage and reference matching for each barcode concurrently as sequencing proceeds. We sequenced completely offline to avoid problems with internet connection (normally necessary to initialise MinION sequencing) and limit reliance on data transfer to servers for data storage. Sequence data was stored locally on either internal or external hard drives, with subsequent transfer to university servers in the UK at a later date. However, data transfer was problematic with the large number of files produced (in one instance corrupting a hard drive) and using one laptop for all sequencing and subsequent analysis was slow. Identifying local or cloud-based data solutions is advised to ensure data backup and take advantage of high-performance computing resources when available, but will come at additional cost.</p> <p>RNA extraction from brain tissue was time-consuming and typically yields a low viral proportion after depletion of host genomic DNA ( <a href="#ref-50">Marston <i>et al.</i>, 2013</a>). It also remains one of the most costly aspects of the workflow at ~£6 per sample. <a href="#ref-30">Goharriz <i>et al.</i> (2017)</a> have shown that complete viral genome sequence can be obtained from Whatman Flinders Technology Associates (FTA) card samples without RNA extraction, which opens avenues for refining this part of the protocol and simplifying sample field collection and storage. Efforts to sequence from FTA card samples in Tanzania have shown promise, producing partial genome data (K. Brunker, unpublished report), and may be an option in future for field surveillance.</p> <p>There is potential for cross-sample contamination, particularly when working with amplicon sequencing. This can occur both during library preparation and the bioinformatic stage of barcode demultiplexing following sequencing ( <a href="#ref-75">Xu <i>et al.</i>, 2018</a>). As advised in <a href="#ref-61">Quick <i>et al.</i> (2016)</a>, we took precautions to minimise amplicon contamination in pre-PCR areas, reagents and equipment, such as physical separation between pre- and post-PCR areas, regular decontamination and aliquoting of reagents. We also processed and sequenced negative controls (in most runs) to assess the degree of contamination, finding a low level of amplicon contamination in most sequencing runs. However, the proportion of reads in the negative control relative to the total number of reads per run was low (median [range]=0.03 [0-1.5]%) for runs prepared outside of training workshops i.e. in normal working conditions. Contamination was most problematic in regions where products overamplified in certain samples, increasing the risk of cross-over to other samples. This could be mitigated by limiting the number of PCR cycles for high viral titre samples, if they can be assessed by qPCR prior to sequencing. Unsurprisingly, contamination was higher during training exercises (see underlying data; <a href="#ref-12">Brunker, 2019b</a>) and any sequencing results from training should therefore be repeated for validation.</p> <p>PCR bias may preferentially amplify specific haplotypes in a sample ( <a href="#ref-32">Grubaugh <i>et al.</i>, 2019</a>). This has implications for the use of this type of data in intra-host diversity studies and our results indicate it can contribute to consensus sequence variation between platforms/approaches. Our comparisons between samples sequenced on different platforms (MinION vs Illumina) and with different methodology (PCR vs metagenomic) indicate that intra single nucleotide variation (iSNV) can change the nucleotides that hold the majority frequency in RABV consensus sequences. The extent to which this occurs may be influenced by the frequency of primer mismatches, depth of coverage and initial viral input ( <a href="#ref-32">Grubaugh <i>et al.</i>, 2019</a>). Minor consensus variation has no impact on our objective of using phylogenetic-based methods to identify variants and spatio-temporal-genetic patterns. However, it does indicate that this sequencing approach will require some adjustment to accommodate in-depth analysis, such as inferring who-infected-whom and measuring intra-host diversity. Grubaught <i>et al.</i>’s experimental approach to "correct�? such biases in amplicon-generated sequence data could be applied to limit false positive consensus assignments and accurately estimate within-host diversity in future.</p> <p>The main limitation of an amplicon-based approach is the requirement for prior knowledge of the pathogen for targeted sequencing. This can be an issue if little is known about circulating RABV diversity in an area e.g. following an emergence event, an incursion of a new variant or where multiple divergent lineages co-circulate. We have found that primer design can accommodate a reasonable level of RABV diversity, but its application is still limited. Recent discoveries of novel lyssaviruses ( <a href="#ref-63">Shipley <i>et al.</i>, 2019</a>), cross-species transmission events ( <a href="#ref-43">Kuzmin <i>et al.</i>, 2012</a>; <a href="#ref-52">Mollentze <i>et al.</i>, 2014a</a>) and the emergence of new variants ( <a href="#ref-42">Kotait <i>et al.</i>, 2019</a>) indicate that when an amplicon-based approach is insufficient, a less targeted method would be useful to monitor threats as dog-mediated rabies is eliminated. Metagenomic sequencing is considered the gold standard for viral pathogen sequencing, offering an unbiased, untargeted approach but suffers from sensitivity issues when working with low viral titres. Success with RABV metagenomic sequencing varies according to sample quality and viral titre, with often &amp;lt;1% of reads mapping to a RABV reference sequence ( <a href="#ref-17">Brunker <i>et al.</i>, 2015</a>). Therefore, a high sequencing throughput per sample is required. The MinION is not considered a high-throughput platform but recent and ongoing improvements to flowcell chemistry and data output from the MinION (10–20Gb in 48hours) have advanced the feasibility of conducting whole genome viral metagenomic sequencing directly from clinical samples, even with low viral titre samples ( <a href="#ref-39">Kafetzopoulou <i>et al.</i>, 2018</a>).</p> </div> <div class="opportunities" title="sec"> <div class="title" tagx="title" title="title">
Opportunities</div> <p>The value of genomic data cannot be understated and has yet to reach its full potential for RABV. While the emphasis of this paper is on the application of the workflow and not phylogenetic analysis, the genomic data has already provided new insights. For example, uncovering the presence of two RABV subclades in Kenya, including one that is closely related to the lineage found in Tanzania. The extent of cross-border transmission and the scale over which persisting RABV lineages circulate in East Africa will have important implications for national and regional control efforts. In <a href="#B1">Box 1</a>, we suggest how genetic data can inform control measures at different stages in the pathway to elimination. In addition, during the application of our workflow in Tanzania we were able to show directly how genomic data enhanced an investigation of a rabies outbreak on the island of Pemba (K. Lushasi, Ifakara Health Institute, unpublished report). Pemba is a small island (988 km <sup>2</sup>) off the coast of Tanzania. After four years of sustained vaccination campaigns, rabies appeared to have been eliminated from Pemba, with the last case detected mid-2014. However, after a lapse in vaccination efforts there was a rabies outbreak in 2016–2017. Samples collected in the early stages of the outbreak were exported to the UK as a priority for sequencing but still experienced a lag time of around three months. During our first field trip to validate in-country MinION sequencing in Tanzania (2017), we had the opportunity to sequence more samples from the outbreak. Eight samples were sequenced with a turnaround of two days, yielding whole genome consensus sequences that provided evidence of multiple incursions from mainland Tanzania (see <a href="#f7">Figure 7</a>). This emphasises the importance of maintaining vaccination coverage and continued surveillance on Pemba to contain incursions and sustain freedom from rabies.</p> <div class="boxed-text" tagx="boxed-text" title="boxed-text"><span class="label" tagx="label" title="label">Box 1. </span> <div class="title" tagx="title" title="title">
Surveillance insights from genomic data</div>  <p>Here we highlight the current and prospective ways in which genetic data can be used to answer questions at different stages along the complex pathway towards elimination of rabies ( <a href="#f8">Figure 8</a>).</p> <ul> <li><span class="label" tagx="label" title="label">1. </span><p>Characterize circulating variants and lineages:</p> <ul> <li><span class="label" tagx="label" title="label">a. </span><p>Identify host associations/biotypes to inform targeted vaccination, such as vampire bat versus dog rabies in Latin America ( <a href="#ref-42">Kotait <i>et al.,</i> 2019</a>) or mongoose versus dog rabies in parts of Southern Africa ( <a href="#ref-81">Nel <i>et al.,</i> 2005</a>), and the identification of domestic dogs as the reservoir host in a wildlife-rich ecosystem ( <a href="#ref-46">Lembo <i>et al.,</i> 2008</a>)</p> </li> <li><span class="label" tagx="label" title="label">b. </span><p>Assess lineage distributions in the context of political/administrative borders, such as provinces in the Philippines ( <a href="#ref-67">Tohma <i>et al.,</i> 2016</a>), states in North Africa ( <a href="#ref-82">Talbi <i>et al.,</i> 2010</a>) and East Africa ( <a href="#ref-17">Brunker <i>et al.,</i> 2015</a>); and quantifying transboundary risks as shown by <a href="#ref-68">Trewby <i>et al.,</i> 2017</a>.</p> </li> <li><span class="label" tagx="label" title="label">c. </span><p>Determine the socio-ecological processes underlying local and long-distance transmission, for example the identification of long-distance human-mediated movement of RABV lineages ( <a href="#ref-17">Brunker <i>et al.,</i> 2015</a>; <a href="#ref-80">Denduangboripant <i>et al.,</i> 2005</a>; <a href="#ref-68">Trewby <i>et al.,</i> 2017</a>) or associations with human density and landscape accessibility ( <a href="#ref-23">Dellicour <i>et al.,</i> 2017</a>; <a href="#ref-24">Dellicour <i>et al.,</i> 2019</a>).</p> </li> </ul> </li> <li><span class="label" tagx="label" title="label">2. </span><p>Monitor progress of control:</p> <ul> <li><span class="label" tagx="label" title="label">a. </span><p>Report emergence and extinction of lineages as evidence of control failures or successes ( <a href="#ref-25">Dudas <i>et al.</i>, 2017</a>)</p> </li> <li><span class="label" tagx="label" title="label">b. </span><p>Integrate with data streams (e.g. case detection, bite patients, vaccination coverage) to resolve complexities in dynamics ( <a href="#ref-7">Bourhy <i>et al.</i>, 2016</a>; <a href="#ref-46">Lembo <i>et al.</i>, 2008</a>; <a href="#ref-72">Viana <i>et al.</i>, 2016</a>)</p> </li> <li><span class="label" tagx="label" title="label">c. </span><p>Identify epidemiologically connected areas that RABV lineages circulate between once control underway e.g. RABV spread associated with road networks, ( <a href="#ref-16">Brunker <i>et al.,</i> 2018</a>; <a href="#ref-82">Talbi <i>et al.,</i> 2010</a>), spread between peri-urban and rural areas ( <a href="#ref-7">Bourhy <i>et al.,</i> 2016</a>).</p> </li> </ul> </li> <li><span class="label" tagx="label" title="label">3. </span><p>Enhance surveillance during endgame:</p> <ul> <li><span class="label" tagx="label" title="label">a. </span><p>Differentiate undetected endemic transmission versus incursions ( <a href="#ref-7">Bourhy <i>et al.</i>, 2016</a>; <a href="#ref-26">Faria <i>et al.</i>, 2017</a>; <a href="#ref-76">Zinsstag <i>et al.</i>, 2017</a>)</p> </li> <li><span class="label" tagx="label" title="label">b. </span><p>Discriminate variants to verify freedom from dog rabies ( <a href="#ref-70">Velasco-Villa <i>et al.,</i> 2017</a>) or identify emergence of new reservoirs, such as those seen in Latin America (primate/marmoset associated variants ( <a href="#ref-42">Kotait <i>et al.,</i> 2019</a>), circulation in crab-eating foxes ( <a href="#ref-79">Carnieli <i>et al.,</i> 2009</a>), terrestrial carnivores associations in Mexico ( <a href="#ref-71">Velasco-Villa <i>et al.</i>, 2005</a>))</p> </li> <li><span class="label" tagx="label" title="label">c. </span><p>Exclude possibility that cases are due to vaccine reversion, given vaccine quality concerns in some settings ( <a href="#ref-104">Suseno <i>et al.</i>, 2019</a>)</p> </li> </ul> </li> <li><span class="label" tagx="label" title="label">4. </span><p>Post-elimination surveillance:</p> <ul> <li><span class="label" tagx="label" title="label">a. </span><p>Identify the origin of imported cases (animal or human) or new outbreaks to direct control and prevention efforts and limit spread ( <a href="#ref-51">Mollentze <i>et al.</i>, 2013</a>; <a href="#ref-101">Zhang <i>et al.</i>, 2019</a>)</p> </li> </ul> </li> </ul> </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 8. </span></a></div>   <div class="title" tagx="title" title="title">
Progress towards elimination of dog rabies and role for genomic surveillance.</div> <p>Detected animal rabies cases are shown in grey and human rabies cases in red, based on surveillance data from Ecuador (from SIRVERA). Data were used to inform hypothetical scenarios relevant to rabies circulation in Latin America to illustrate examples of how genomic surveillance can add value. Stages of genomic surveillance corresponding to <a href="#B1">Box 1</a> are shown with examples of inferences from genomic surveillance indicated by asterisks.</p>   </div> <p>WGS using portable nanopore technology is a rapidly moving field, with protocols and pipeline developments evolving quickly. Although this comes with the caveat that protocols have to evolve to keep up with changes, these developments bring cost savings and improve accuracy. New refinements are expected to reduce costs further from a current estimate of ~£60 to £35 per sample, according to projections with flowcell re-use or the smaller low cost Flongle flowcell. Moreover, ongoing optimisation to reduce reagent use (K. Brunker, unpublished report) indicate that this can be reduced even further to the point where sample preparation/RNA extraction, rather than sequencing itself, become the most costly aspect of the protocol. Improvements to pore chemistry, kits and basecalling software have significantly improved the error rate, often considered the MInION’s Achilles heel. Single read accuracy has improved from 20% error rate of early phase 2014 releases to &amp;lt;5% error rate with R9.4.1 pore chemistry ( <a href="#ref-58">Plesivkova <i>et al.</i>, 2019</a>). The latest version shows further improvements with 1D <sup>2</sup> sequencing technology and error correcting algorithms reaching ~99.3-99.9% consensus accuracy ( <a href="#ref-83">Wick <i>et al.,</i> 2019</a>). A major benefit of nanopore sequencing is the ability to convert raw data to basecalled sequence data in real-time. Although we did not have a GPU powered laptop to take advantage of this function, it offers an even speedier resolution for time-critical sample analysis to inform decision-making. ONT’s MinIT, a portable data processing unit, offers similar functionality but with the flexibility to use with different laptops.</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>Genomic research has the potential to leap-frog LMICs to the forefront of infectious diseases surveillance and control, and facing challenges head-on will hugely benefit the future of public and veterinary health. Most countries with endemic dog-mediated rabies currently have limited local sequencing capacity. These countries also suffer a heavy burden of other neglected endemic diseases, that could benefit from genomic research. The approach that we present is affordable and readily deployable to such settings with limited infrastructure, overcoming issues that typically constrain routine genomics surveillance, including cold chain, electricity and internet. Moreover, the approach is scalable and low-cost (~£60 per sample) for both sequencing large numbers of samples and for rapid turnaround to inform disease management. We predict further improvements to sequencing performance including cost-effectiveness, speed and accuracy, with further optimization underway and protocols available to researchers ( <a href="https://github.com/kirstyn/rabies_minion">https://github.com/kirstyn/rabies_minion</a>; <a href="#ref-14">Brunker, 2019d</a>; <a href="#ref-15">Brunker, 2019e</a>). Nonetheless, improvement to the supply and affordability of reagents for LMICs’ scientists is an area that needs addressing, as is the development of bioinformatics resources and translation of genomic research findings to digestible recommendations for policy-makers. Overall, this in-country application can support capacity building and the sustainability of routine genomic surveillance in LMICs, empowering local scientists to conduct cutting-edge research in their own backyard.</p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Data availability</div> <div class="underlyingdata" title="sec"> <div class="title" tagx="title" title="title">
Underlying data</div> <p>Consensus sequences on Genbank, Accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/MN726802">MN726802</a> - <a href="https://www.ncbi.nlm.nih.gov/nuccore/MN726882">MN726882</a></p> <p>Figshare: rabv_minionSeqConsensus.fasta. <a href="https://doi.org/10.6084/m9.figshare.10282859.v1">https://doi.org/10.6084/m9.figshare.10282859.v1</a> ( <a href="#ref-11">Brunker, 2019a</a>)</p> <p>Figshare: Table 1. Rabies virus samples sequenced on the MinION machine 2016–2019 in East Africa, Philippines and UK. <a href="https://doi.org/10.6084/m9.figshare.11111771.v1">https://doi.org/10.6084/m9.figshare.11111771.v1</a> ( <a href="#ref-12">Brunker, 2019b</a>)</p> <p>Figshare: Breakdown of costs to sequence rabies virus samples on MinION (spreadsheet). <a href="https://doi.org/10.6084/m9.figshare.11120405.v1">https://doi.org/10.6084/m9.figshare.11120405.v1</a> ( <a href="#ref-13">Brunker, 2019c</a>)</p> </div> <div class="extendeddata" title="sec"> <div class="title" tagx="title" title="title">
Extended data</div> <p>Figshare: RabiesVirus_sampleToSequence_SOP. <a href="https://doi.org/10.6084/m9.figshare.10742090.v1">https://doi.org/10.6084/m9.figshare.10742090.v1</a> ( <a href="#ref-14">Brunker, 2019d</a>)</p> <p>Figshare: Rabies virus primer schemes. <a href="https://doi.org/10.6084/m9.figshare.11112134.v1">https://doi.org/10.6084/m9.figshare.11112134.v1</a> ( <a href="#ref-15">Brunker, 2019e</a>)</p> <p>Figshare: Extended Figure 6 phylogeny including tip labels: <a href="https://doi.org/10.6084/m9.figshare.12284006">https://doi.org/10.6084/m9.figshare.12284006</a> ( <a href="#ref-130">Brunker, 2020a</a>)</p> <p>Figshare: Extended Figure 7 phylogeny including tip labels: <a href="https://doi.org/10.6084/m9.figshare.12284009">https://doi.org/10.6084/m9.figshare.12284009</a> ( <a href="#ref-133">Brunker, 2020b</a>)</p> <p>Data are available under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/legalcode">Creative Commons Attribution 4.0 International license</a> (CC-BY 4.0).</p> </div> </div> <div class="consent" title="sec"> <div class="title" tagx="title" title="title">
Consent</div> <p>Written consent to publish the images included in this paper was obtained from the individuals in these images.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We are grateful to the governments of the Philippines, Kenya and the United Republic of Tanzania and local collaborating research institutions for hosting this research. These partners include Ifakara Health Institute, Nelson Mandela African Institution of Science and Technology, the National Institute for Medical Research, the Tanzania Commission for Science and Technology, Sokoine University of Agriculture, and the Tanzanian Veterinary Laboratories Agencies in Tanzania; the University of Nairobi Institute for Tropical Infectious Diseases, the Kenya Zoonotic Disease Unit, Kenya Medical Research Institute, Central Veterinary Laboratory and the regional veterinary investigation laboratories in Kenya; and the Research Institute for Tropical Medicine and the Field Epidemiology Training Programme Alumni Foundation, Inc in the Philippines. We thank the Tanzanian Ministry of Livestock Development and Fisheries, the One Health Coordination Desk under the prime minister's office and all district authorities and veterinary workers who undertook sample collection. Thanks to all workshop participants for taking part in training. Many thanks to Beatus Lyimo for his help to facilitate lab work at NMAIST. This work benefited enormously from the support of the <a href="https://artic.network/">ARTIC network</a> for generously donated time and expertise, in particular Josh Quick. Luke Meredith and Radoslaw Poplawski work as part of the ARTIC Network funded by Wellcome Trust Collaborative Award grant 206298/Z/17/Z. Many thanks to ONT for streamlining the setup of MinION sequencing in East Africa/Philippines and helping with logistical challenges. ONT provided some reagents for training workshops.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <li tag="ref"><a name="ref-1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abela-Ridder</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">Laboratory techniques in rabies, fifth edition. Volume 2</span>. Geneva: World Health Organization; Licence: CC BY-NC-SA 3.0 IGO. (Rupprecht CE, Fooks AR, Abela-Ridder B: Eds.) (5th ed., Vol. 2). Geneva: Geneva: World Health Organization.<span class="year" tagx="year" title="year">2018</span><a href="https://apps.who.int/iris/rest/bitstreams/1207194/retrieve">Reference Source</a></span></li> <li tag="ref"><a name="ref-2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abela-Ridder</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knopf</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">2016: the beginning of the end of rabies?</span><span class="source" tagx="source" title="source"><i>Lancet Glob Health.</i></span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">4</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">e780</span>–<span class="lpage" tagx="lpage" title="lpage">e781</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S2214-109X(16)30245-5">10.1016/S2214-109X(16)30245-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27692777">27692777</a></span></span></li> <li tag="ref"><a name="ref-3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balloux</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brønstad Brynildsrud</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Dorp</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">From Theory to Practice: Translating Whole-Genome Sequencing (WGS) into the Clinic.</span><span class="source" tagx="source" title="source"><i>Trends Microbiol.</i></span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">26</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">1035</span>–<span class="lpage" tagx="lpage" title="lpage">1048</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tim.2018.08.004">10.1016/j.tim.2018.08.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30193960">30193960</a></span></span></li> <li tag="ref"><a name="ref-4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benavides</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Megid</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campos</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">An evaluation of Brazil's surveillance and prophylaxis of canine rabies between 2008 and 2017.</span><span class="source" tagx="source" title="source"><i>PLoS Negl Trop Dis.</i></span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">e0007564</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0007564">10.1371/journal.pntd.0007564</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31381564">31381564</a></span></span></li> <li tag="ref"><a name="ref-5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bitek</span><span class="given-names" tagx="given-names" title="given-names">AO</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osoro</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Munyua</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">A hundred years of rabies in Kenya and the strategy for eliminating dog-mediated rabies by 2030 [version 2; peer review: 4 approved].</span><span class="source" tagx="source" title="source"><i>AAS Open Res.</i></span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">1</span>:<span class="fpage" tagx="fpage" title="fpage">23</span>. <span class="pub-id"><a href="https://dx.doi.org/10.12688/aasopenres.12872.2">10.12688/aasopenres.12872.2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32259023">32259023</a></span></span></li> <li tag="ref"><a name="ref-7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bourhy</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nakouné</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Revealing the Micro-scale Signature of Endemic Zoonotic Disease Transmission in an African Urban Setting.</span><span class="source" tagx="source" title="source"><i>PLoS Pathogens.</i></span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">e1005525</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1005525">10.1371/journal.ppat.1005525</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27058957">27058957</a></span></span></li> <li tag="ref"><a name="ref-8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boykin</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghalab</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossitto De Marchi</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Real time portable genome sequencing for global food security [version 1; peer review: 1 approved, 1 approved with reservations].</span><span class="source" tagx="source" title="source"><i>F1000Res.</i></span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">1101</span><span class="pub-id"><a href="https://dx.doi.org/10.12688/f1000research.15507.1">10.12688/f1000research.15507.1</a></span></span></li> <li tag="ref"><a name="ref-9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boykin</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sseruwagi</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alicai</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Tree Lab: Portable genomics for early detection of plant viruses and pests in Sub-Saharan Africa.</span><span class="source" tagx="source" title="source"><i>BioRxiv.</i></span><span class="year" tagx="year" title="year">2019</span>;<span class="fpage" tagx="fpage" title="fpage">702613</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/702613">10.1101/702613</a></span></span></li> <li tag="ref"><a name="ref-10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Broban</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tejiokem</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tiembré</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Bolstering human rabies surveillance in Africa is crucial to eliminating canine-mediated rabies.</span><span class="source" tagx="source" title="source"><i>PLoS Negl Trop Dis.</i></span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">e0006367</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0006367">10.1371/journal.pntd.0006367</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30188896">30188896</a></span></span></li> <li tag="ref"><a name="ref-11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunker</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">rabv_minionSeqConsensus.fasta</span>. <span class="source" tagx="source" title="source"><i>figshare</i></span>. Dataset.<span class="year" tagx="year" title="year">2019a</span><span class="pub-id"><a href="https://dx.doi.org/10.6084/m9.figshare.10282859.v1">10.6084/m9.figshare.10282859.v1</a></span></span></li> <li tag="ref"><a name="ref-12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunker</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">Table 1. Rabies virus samples sequenced on the MinION machine 2016-2019 in East Africa, Philippines and UK</span>. <span class="source" tagx="source" title="source"><i>figshare.</i></span>Journal contribution.<span class="year" tagx="year" title="year">2019b</span><span class="pub-id"><a href="https://dx.doi.org/10.6084/m9.figshare.11111771.v1">10.6084/m9.figshare.11111771.v1</a></span></span></li> <li tag="ref"><a name="ref-13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunker</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">Breakdown of costs to sequence rabies virus samples on MinION (spreadsheet)</span>. <span class="source" tagx="source" title="source"><i>figshare.</i></span>Journal contribution.<span class="year" tagx="year" title="year">2019c</span><span class="pub-id"><a href="https://dx.doi.org/10.6084/m9.figshare.11120405.v1">10.6084/m9.figshare.11120405.v1</a></span></span></li> <li tag="ref"><a name="ref-14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunker</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">RabiesVirus_sampleToSequence_SOP</span>. <span class="source" tagx="source" title="source"><i>figshare</i></span>. Online resource.<span class="year" tagx="year" title="year">2019d</span><span class="pub-id"><a href="https://dx.doi.org/10.6084/m9.figshare.10742090.v1">10.6084/m9.figshare.10742090.v1</a></span></span></li> <li tag="ref"><a name="ref-15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunker</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">Rabies virus primer schemes.</span><span class="source" tagx="source" title="source"><i>figshare</i></span>. Journal contribution.<span class="year" tagx="year" title="year">2019e</span><span class="pub-id"><a href="https://dx.doi.org/10.6084/m9.figshare.11112134.v1">10.6084/m9.figshare.11112134.v1</a></span></span></li> <li tag="ref"><a name="ref-130" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunker</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">Figure 6 extended.</span><span class="source" tagx="source" title="source"><i>figshare</i></span>. Figure.<span class="year" tagx="year" title="year">2020a</span><span class="pub-id"><a href="https://dx.doi.org/10.6084/m9.figshare.12284006.v1">10.6084/m9.figshare.12284006.v1</a></span></span></li> <li tag="ref"><a name="ref-133" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunker</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">Figure 7 extended.</span><span class="source" tagx="source" title="source"><i>figshare</i></span>. Figure.<span class="year" tagx="year" title="year">2020b</span><span class="pub-id"><a href="https://dx.doi.org/10.6084/m9.figshare.12284009.v1">10.6084/m9.figshare.12284009.v1</a></span></span></li> <li tag="ref"><a name="ref-16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunker</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marston</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Landscape attributes governing local transmission of an endemic zoonosis: Rabies virus in domestic dogs.</span><span class="source" tagx="source" title="source"><i>Mol Ecol.</i></span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">27</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">773</span>–<span class="lpage" tagx="lpage" title="lpage">788</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.14470">10.1111/mec.14470</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29274171">29274171</a></span></span></li> <li tag="ref"><a name="ref-17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunker</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marston</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horton</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Elucidating the phylodynamics of endemic rabies virus in eastern Africa using whole-genome sequencing.</span><span class="source" tagx="source" title="source"><i>Virus Evol.</i></span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">1</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">vev011</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/ve/vev011">10.1093/ve/vev011</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27774283">27774283</a></span></span></li> <li tag="ref"><a name="ref-18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunker</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mollentze</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">Rabies Virus.</span><span class="source" tagx="source" title="source"><i>Trends Microbiol.</i></span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">26</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">886</span>–<span class="lpage" tagx="lpage" title="lpage">887</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tim.2018.07.001">10.1016/j.tim.2018.07.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30072086">30072086</a></span></span></li> <li tag="ref"><a name="ref-79" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carnieli</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castilho</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fahl Wde</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Molecular characterization of Rabies Virus isolates from dogs and crab-eating foxes in Northeastern Brazil.</span><span class="source" tagx="source" title="source"><i>Virus Res.</i></span><span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">141</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">81</span>–<span class="lpage" tagx="lpage" title="lpage">89</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virusres.2008.12.015">10.1016/j.virusres.2008.12.015</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19185599">19185599</a></span></span></li> <li tag="ref"><a name="ref-19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chawla</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kurani</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wren</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Electricity and generator availability in LMIC hospitals: improving access to safe surgery.</span><span class="source" tagx="source" title="source"><i>J Surg Res.</i></span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">223</span>:<span class="fpage" tagx="fpage" title="fpage">136</span>–<span class="lpage" tagx="lpage" title="lpage">141</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jss.2017.10.016">10.1016/j.jss.2017.10.016</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29433865">29433865</a></span></span></li> <li tag="ref"><a name="ref-20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cleaveland</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharp</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abela-Ridder</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">One Health contributions towards more effective and equitable approaches to health in low- and middle-income countries.</span><span class="source" tagx="source" title="source"><i>Philos Trans R Soc B Biol Sci.</i></span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">372</span>(<span class="issue" tagx="issue" title="issue">1725</span>): pii: 20160168. <span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2016.0168">10.1098/rstb.2016.0168</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28584176">28584176</a></span></span></li> <li tag="ref"><a name="ref-21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cotton</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berriman</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dalén</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Eradication genomics-lessons for parasite control.</span><span class="source" tagx="source" title="source"><i>Science.</i></span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">361</span>(<span class="issue" tagx="issue" title="issue">6398</span>):<span class="fpage" tagx="fpage" title="fpage">130</span>–<span class="lpage" tagx="lpage" title="lpage">131</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30002241">30002241</a></span></span></li> <li tag="ref"><a name="ref-22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Danielsen</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kajura</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mulema</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Reaching for the low hanging fruits: One health benefits of joint crop-livestock services for small-scale farmers.</span><span class="source" tagx="source" title="source"><i>One Health.</i></span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">100082</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.onehlt.2019.100082">10.1016/j.onehlt.2019.100082</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30793019">30793019</a></span></span></li> <li tag="ref"><a name="ref-23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dellicour</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rose</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">NR</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Using Viral Gene Sequences to Compare and Explain the Heterogeneous Spatial Dynamics of Virus Epidemics.</span><span class="source" tagx="source" title="source"><i>Mol Biol Evol.</i></span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">34</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">2563</span>–<span class="lpage" tagx="lpage" title="lpage">2571</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msx176">10.1093/molbev/msx176</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28651357">28651357</a></span></span></li> <li tag="ref"><a name="ref-24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dellicour</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Troupin</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jahanbakhsh</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Using phylogeographic approaches to analyse the dispersal history, velocity and direction of viral lineages - Application to rabies virus spread in Iran.</span><span class="source" tagx="source" title="source"><i>Mol Ecol.</i></span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">28</span>(<span class="issue" tagx="issue" title="issue">18</span>):<span class="fpage" tagx="fpage" title="fpage">4335</span>–<span class="lpage" tagx="lpage" title="lpage">4350</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.15222">10.1111/mec.15222</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31535448">31535448</a></span></span></li> <li tag="ref"><a name="ref-80" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Denduangboripant</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wacharapluesadee</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lumlertdacha</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Transmission dynamics of rabies virus in Thailand: implications for disease control.</span><span class="source" tagx="source" title="source"><i>BMC Infect Dis.</i></span><span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">5</span>:<span class="fpage" tagx="fpage" title="fpage">52</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2334-5-52">10.1186/1471-2334-5-52</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15985183">15985183</a></span></span></li> <li tag="ref"><a name="ref-25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudas</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carvalho</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Virus genomes reveal factors that spread and sustained the Ebola epidemic.</span><span class="source" tagx="source" title="source"><i>Nature.</i></span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">544</span>(<span class="issue" tagx="issue" title="issue">7650</span>):<span class="fpage" tagx="fpage" title="fpage">309</span>–<span class="lpage" tagx="lpage" title="lpage">315</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nature22040">10.1038/nature22040</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28405027">28405027</a></span></span></li> <li tag="ref"><a name="ref-90" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">NR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kraemer</span><span class="given-names" tagx="given-names" title="given-names">MU</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hill</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Genomic and epidemiological monitoring of yellow fever virus transmission potential.</span><span class="source" tagx="source" title="source"><i>Science.</i></span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">361</span>(<span class="issue" tagx="issue" title="issue">6405</span>):<span class="fpage" tagx="fpage" title="fpage">894</span>–<span class="lpage" tagx="lpage" title="lpage">899</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.aat7115">10.1126/science.aat7115</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30139911">30139911</a></span></span></li> <li tag="ref"><a name="ref-26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">NR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quick</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Claro</span><span class="given-names" tagx="given-names" title="given-names">IM</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Establishment and cryptic transmission of Zika virus in Brazil and the Americas.</span><span class="source" tagx="source" title="source"><i>Nature.</i></span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">546</span>(<span class="issue" tagx="issue" title="issue">7658</span>):<span class="fpage" tagx="fpage" title="fpage">406</span>–<span class="lpage" tagx="lpage" title="lpage">410</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nature22401">10.1038/nature22401</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28538727">28538727</a></span></span></li> <li tag="ref"><a name="ref-27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">NR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabino</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nunes</span><span class="given-names" tagx="given-names" title="given-names">MR</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Mobile real-time surveillance of Zika virus in Brazil.</span><span class="source" tagx="source" title="source"><i>Genome Med.</i></span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">97</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/s13073-016-0356-2">10.1186/s13073-016-0356-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27683027">27683027</a></span></span></li> <li tag="ref"><a name="ref-28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Folarin</span><span class="given-names" tagx="given-names" title="given-names">OA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Happi</span><span class="given-names" tagx="given-names" title="given-names">AN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Happi</span><span class="given-names" tagx="given-names" title="given-names">CT</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">Empowering African genomics for infectious disease control.</span><span class="source" tagx="source" title="source"><i>Genome Biol.</i></span><span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">15</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">515</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/s13059-014-0515-y">10.1186/s13059-014-0515-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25417758">25417758</a></span></span></li> <li tag="ref"><a name="ref-29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gardy</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loman</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">Towards a genomics-informed, real-time, global pathogen surveillance system.</span><span class="source" tagx="source" title="source"><i>Nat Rev Genet.</i></span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">19</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">9</span>–<span class="lpage" tagx="lpage" title="lpage">20</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nrg.2017.88">10.1038/nrg.2017.88</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29129921">29129921</a></span></span></li> <li tag="ref"><a name="ref-30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goharriz</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marston</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharifzoda</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">First Complete Genomic Sequence of a Rabies Virus from the Republic of Tajikistan Obtained Directly from a Flinders Technology Associates Card.</span><span class="source" tagx="source" title="source"><i>Genome Announc.</i></span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">5</span>(<span class="issue" tagx="issue" title="issue">27</span>): pii: e00515-17. <span class="pub-id"><a href="https://dx.doi.org/10.1128/genomeA.00515-17">10.1128/genomeA.00515-17</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28684566">28684566</a></span></span></li> <li tag="ref"><a name="ref-31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grädel</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Terrazos Miani</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barbani</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Rapid and Cost-Efficient Enterovirus Genotyping from Clinical Samples Using Flongle Flow Cells.</span><span class="source" tagx="source" title="source"><i>Genes (Basel).</i></span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">9</span>): pii: E659. <span class="pub-id"><a href="https://dx.doi.org/10.3390/genes10090659">10.3390/genes10090659</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31470607">31470607</a></span></span></li> <li tag="ref"><a name="ref-32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grubaugh</span><span class="given-names" tagx="given-names" title="given-names">ND</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gangavarapu</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quick</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">An amplicon-based sequencing framework for accurately measuring intrahost virus diversity using PrimalSeq and iVar.</span><span class="source" tagx="source" title="source"><i>Genome Biol.</i></span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">20</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">8</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/s13059-018-1618-7">10.1186/s13059-018-1618-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30621750">30621750</a></span></span></li> <li tag="ref"><a name="ref-33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halliday</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allan</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ekwem</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Endemic zoonoses in the tropics: a public health problem hiding in plain sight.</span><span class="source" tagx="source" title="source"><i>Vet Rec.</i></span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">176</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">220</span>–<span class="lpage" tagx="lpage" title="lpage">225</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1136/vr.h798">10.1136/vr.h798</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25722334">25722334</a></span></span></li> <li tag="ref"><a name="ref-34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Halliday</span><span class="given-names" tagx="given-names" title="given-names">JEB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hampson</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanley</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Driving improvements in emerging disease surveillance through locally relevant capacity strengthening.</span><span class="source" tagx="source" title="source"><i>Science.</i></span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">357</span>(<span class="issue" tagx="issue" title="issue">6347</span>):<span class="fpage" tagx="fpage" title="fpage">146</span>–<span class="lpage" tagx="lpage" title="lpage">148</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.aam8332">10.1126/science.aam8332</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28706036">28706036</a></span></span></li> <li tag="ref"><a name="ref-36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hampson</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abela-ridder</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brunker</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Surveillance to establish elimination of transmission and freedom from Dog-mediated rabies.</span><span class="source" tagx="source" title="source"><i>bioRxiv.</i></span>BioRxiv Preprint,<span class="year" tagx="year" title="year">2016</span>;<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">22</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1101/096883">10.1101/096883</a></span></span></li> <li tag="ref"><a name="ref-37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Henderson</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">The eradication of smallpox--an overview of the past, present, and future.</span><span class="source" tagx="source" title="source"><i>Vaccine.</i></span><span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">29</span>(<span class="issue" tagx="issue" title="issue">Suppl 4</span>):<span class="fpage" tagx="fpage" title="fpage">D7</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.vaccine.2011.06.080">10.1016/j.vaccine.2011.06.080</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22188929">22188929</a></span></span></li> <li tag="ref"><a name="ref-38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaikova</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goerlitz</span><span class="given-names" tagx="given-names" title="given-names">DS</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Real-Time DNA Sequencing in the Antarctic Dry Valleys Using the Oxford Nanopore Sequencer.</span><span class="source" tagx="source" title="source"><i>J Biomol Tech.</i></span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">28</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">2</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28337073">28337073</a></span></span></li> <li tag="ref"><a name="ref-39" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kafetzopoulou</span><span class="given-names" tagx="given-names" title="given-names">LE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Efthymiadis</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewandowski</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Assessment of metagenomic Nanopore and Illumina sequencing for recovering whole genome sequences of chikungunya and dengue viruses directly from clinical samples.</span><span class="source" tagx="source" title="source"><i>Euro Surveill.</i></span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">23</span>(<span class="issue" tagx="issue" title="issue">50</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">13</span>. <span class="pub-id"><a href="https://dx.doi.org/10.2807/1560-7917.ES.2018.23.50.1800228">10.2807/1560-7917.ES.2018.23.50.1800228</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30563591">30563591</a></span></span></li> <li tag="ref"><a name="ref-40" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katoh</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Standley</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability.</span><span class="source" tagx="source" title="source"><i>Mol Biol Evol.</i></span><span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">30</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">772</span>–<span class="lpage" tagx="lpage" title="lpage">80</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/mst010">10.1093/molbev/mst010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23329690">23329690</a></span></span></li> <li tag="ref"><a name="ref-41" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klepac</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Metcalf</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McLean</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Towards the endgame and beyond: complexities and challenges for the elimination of infectious diseases.</span><span class="source" tagx="source" title="source"><i>Philos Trans R Soc Lond B Biol Sci.</i></span><span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">368</span>(<span class="issue" tagx="issue" title="issue">1623</span>):<span class="fpage" tagx="fpage" title="fpage">20120137</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2012.0137">10.1098/rstb.2012.0137</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23798686">23798686</a></span></span></li> <li tag="ref"><a name="ref-42" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kotait</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oliveira</span><span class="given-names" tagx="given-names" title="given-names">RN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carrieri</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Non-human primates as a reservoir for rabies virus in Brazil.</span><span class="source" tagx="source" title="source"><i>Zoonoses Public Health.</i></span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">66</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">47</span>–<span class="lpage" tagx="lpage" title="lpage">59</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/zph.12527">10.1111/zph.12527</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30288933">30288933</a></span></span></li> <li tag="ref"><a name="ref-43" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuzmin</span><span class="given-names" tagx="given-names" title="given-names">IV</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orciari</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Molecular inferences suggest multiple host shifts of rabies viruses from bats to mesocarnivores in Arizona during 2001–2009.</span><span class="source" tagx="source" title="source"><i>PLoS Pathog.</i></span><span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">e1002786</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1002786">10.1371/journal.ppat.1002786</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22737076">22737076</a></span></span></li> <li tag="ref"><a name="ref-44" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lankester</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hampson</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lembo</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Infectious Disease. Implementing Pasteur’s vision for rabies elimination.</span><span class="source" tagx="source" title="source"><i>Science.</i></span><span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">345</span>(<span class="issue" tagx="issue" title="issue">6204</span>):<span class="fpage" tagx="fpage" title="fpage">1562</span>–<span class="lpage" tagx="lpage" title="lpage">1564</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1256306">10.1126/science.1256306</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25258065">25258065</a></span></span></li> <li tag="ref"><a name="ref-45" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Léchenne</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naïssengar</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lepelletier</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Validation of a Rapid Rabies Diagnostic Tool for Field Surveillance in Developing Countries.</span><span class="source" tagx="source" title="source"><i>PLoS Negl Trop Dis.</i></span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">e0005010</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0005010">10.1371/journal.pntd.0005010</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27706156">27706156</a></span></span></li> <li tag="ref"><a name="ref-46" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lembo</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hampson</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haydon</span><span class="given-names" tagx="given-names" title="given-names">DT</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Exploring reservoir dynamics: a case study of rabies in the Serengeti ecosystem.</span><span class="source" tagx="source" title="source"><i>J Appl Ecol.</i></span><span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">45</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">1246</span>–<span class="lpage" tagx="lpage" title="lpage">1257</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2664.2008.01468.x">10.1111/j.1365-2664.2008.01468.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22427710">22427710</a></span></span></li> <li tag="ref"><a name="ref-47" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lembo</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haydon</span><span class="given-names" tagx="given-names" title="given-names">DT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Velasco-Villa</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Molecular epidemiology identifies only a single rabies virus variant circulating in complex carnivore communities of the Serengeti.</span><span class="source" tagx="source" title="source"><i>Proc Biol Sci.</i></span><span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">274</span>(<span class="issue" tagx="issue" title="issue">1622</span>):<span class="fpage" tagx="fpage" title="fpage">2123</span>–<span class="lpage" tagx="lpage" title="lpage">30</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2007.0664">10.1098/rspb.2007.0664</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17609187">17609187</a></span></span></li> <li tag="ref"><a name="ref-48" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durbin</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span><span class="source" tagx="source" title="source"><i>Bioinformatics.</i></span><span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">25</span>(<span class="issue" tagx="issue" title="issue">14</span>):<span class="fpage" tagx="fpage" title="fpage">1754</span>–<span class="lpage" tagx="lpage" title="lpage">60</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btp324">10.1093/bioinformatics/btp324</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168">19451168</a></span></span></li> <li tag="ref"><a name="ref-100" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loman</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quick</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simpson</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">A complete bacterial genome assembled <i>de novo</i> using only nanopore sequencing data.</span><span class="source" tagx="source" title="source"><i>Nat Methods.</i></span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">733</span>–<span class="lpage" tagx="lpage" title="lpage">735</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.3444">10.1038/nmeth.3444</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26076426">26076426</a></span></span></li> <li tag="ref"><a name="ref-49" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marston</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jennings</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MacLaren</span><span class="given-names" tagx="given-names" title="given-names">NC</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Pan-lyssavirus Real Time RT-PCR for Rabies Diagnosis.</span><span class="source" tagx="source" title="source"><i>J Vis Exp.</i></span><span class="year" tagx="year" title="year">2019</span>;<span class="issue" tagx="issue" title="issue">149</span>:<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">10</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3791/59709">10.3791/59709</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31355796">31355796</a></span></span></li> <li tag="ref"><a name="ref-50" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marston</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McElhinney</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ellis</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Next generation sequencing of viral RNA genomes.</span><span class="source" tagx="source" title="source"><i>BMC Genomics.</i></span><span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">14</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">444</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2164-14-444">10.1186/1471-2164-14-444</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23822119">23822119</a></span></span></li> <li tag="ref"><a name="ref-52" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mollentze</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biek</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Streicker</span><span class="given-names" tagx="given-names" title="given-names">DG</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">The role of viral evolution in rabies host shifts and emergence.</span><span class="source" tagx="source" title="source"><i>Curr Opin Virol.</i></span><span class="year" tagx="year" title="year">2014a</span>;<span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">68</span>–<span class="lpage" tagx="lpage" title="lpage">72</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.coviro.2014.07.004">10.1016/j.coviro.2014.07.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25064563">25064563</a></span></span></li> <li tag="ref"><a name="ref-53" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mollentze</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nel</span><span class="given-names" tagx="given-names" title="given-names">LH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Townsend</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">A Bayesian approach for inferring the dynamics of partially observed endemic infectious diseases from space-time-genetic data.</span><span class="source" tagx="source" title="source"><i>Proc Biol Sci.</i></span><span class="year" tagx="year" title="year">2014b</span>;<span class="volume" tagx="volume" title="volume">281</span>(<span class="issue" tagx="issue" title="issue">1782</span>): 20133251. <span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2013.3251">10.1098/rspb.2013.3251</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24619442">24619442</a></span></span></li> <li tag="ref"><a name="ref-51" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mollentze</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weyer</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Markotter</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Dog rabies in southern Africa: regional surveillance and phylogeographical analyses are an important component of control and elimination strategies.</span><span class="source" tagx="source" title="source"><i>Virus Genes.</i></span><span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">47</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">569</span>–<span class="lpage" tagx="lpage" title="lpage">73</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11262-013-0974-3">10.1007/s11262-013-0974-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23996607">23996607</a></span></span></li> <li tag="ref"><a name="ref-54" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Molyneux</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hopkins</span><span class="given-names" tagx="given-names" title="given-names">DR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zagaria</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">Disease eradication, elimination and control: the need for accurate and consistent usage.</span><span class="source" tagx="source" title="source"><i>Trends Parasitol.</i></span><span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">20</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">347</span>–<span class="lpage" tagx="lpage" title="lpage">351</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.pt.2004.06.004">10.1016/j.pt.2004.06.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15246314">15246314</a></span></span></li> <li tag="ref"><a name="ref-55" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mtema</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Changalucha</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cleaveland</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Mobile Phones As Surveillance Tools: Implementing and Evaluating a Large-Scale Intersectoral Surveillance System for Rabies in Tanzania.</span><span class="source" tagx="source" title="source"><i>PLoS Med.</i></span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">13</span>(<span class="issue" tagx="issue" title="issue">4</span>): e1002002. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pmed.1002002">10.1371/journal.pmed.1002002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27070315">27070315</a></span></span></li> <li tag="ref"><a name="ref-56" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nadin-Davis</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colville</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trewby</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Application of high-throughput sequencing to whole rabies viral genome characterisation and its use for phylogenetic re-evaluation of a raccoon strain incursion into the province of Ontario.</span><span class="source" tagx="source" title="source"><i>Virus Res.</i></span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">232</span>:<span class="fpage" tagx="fpage" title="fpage">123</span>–<span class="lpage" tagx="lpage" title="lpage">133</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virusres.2017.02.007">10.1016/j.virusres.2017.02.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28219746">28219746</a></span></span></li> <li tag="ref"><a name="ref-81" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nel</span><span class="given-names" tagx="given-names" title="given-names">LH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabeta</span><span class="given-names" tagx="given-names" title="given-names">CT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">von Teichman</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Mongoose rabies in southern Africa: a re-evaluation based on molecular epidemiology.</span><span class="source" tagx="source" title="source"><i>Virus Res.</i></span><span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">109</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">165</span>–<span class="lpage" tagx="lpage" title="lpage">73</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virusres.2004.12.003">10.1016/j.virusres.2004.12.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15763147">15763147</a></span></span></li> <li tag="ref"><a name="ref-58" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plesivkova</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richards</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harbison</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">A review of the potential of the MinION <sup>TM</sup> single-molecule sequencing system for forensic applications.</span><span class="source" tagx="source" title="source"><i>Wiley Interdiscip Rev: Forensic Sci.</i></span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">1</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">e1323</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/wfs2.1323">10.1002/wfs2.1323</a></span></span></li> <li tag="ref"><a name="ref-57" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poplin</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruano-Rubio</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DePristo</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Scaling accurate genetic variant discovery to tens of thousands of samples.</span><span class="source" tagx="source" title="source"><i>bioRxiv.</i></span><span class="year" tagx="year" title="year">2017</span><span class="pub-id"><a href="https://dx.doi.org/10.1101/201178">10.1101/201178</a></span></span></li> <li tag="ref"><a name="ref-59" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quick</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashton</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Calus</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Rapid draft sequencing and real-time nanopore sequencing in a hospital outbreak of <i>Salmonella</i>.</span><span class="source" tagx="source" title="source"><i>Genome Biol.</i></span><span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">16</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">114</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/s13059-015-0677-2">10.1186/s13059-015-0677-2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26025440">26025440</a></span></span></li> <li tag="ref"><a name="ref-60" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quick</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grubaugh</span><span class="given-names" tagx="given-names" title="given-names">ND</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pullan</span><span class="given-names" tagx="given-names" title="given-names">ST</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Multiplex PCR method for MinION and Illumina sequencing of Zika and other virus genomes directly from clinical samples.</span><span class="source" tagx="source" title="source"><i>Nat Protoc.</i></span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">1261</span>–<span class="lpage" tagx="lpage" title="lpage">1276</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nprot.2017.066">10.1038/nprot.2017.066</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28538739">28538739</a></span></span></li> <li tag="ref"><a name="ref-61" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quick</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loman</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duraffour</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Real-time, portable genome sequencing for Ebola surveillance.</span><span class="source" tagx="source" title="source"><i>Nature.</i></span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">530</span>(<span class="issue" tagx="issue" title="issue">7589</span>):<span class="fpage" tagx="fpage" title="fpage">228</span>–<span class="lpage" tagx="lpage" title="lpage">232</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nature16996">10.1038/nature16996</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26840485">26840485</a></span></span></li> <li tag="ref"><a name="ref-62" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saito</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oshitani</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orbina</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Genetic diversity and geographic distribution of genetically distinct rabies viruses in the Philippines.</span><span class="source" tagx="source" title="source"><i>PLoS Negl Trop Dis.</i></span><span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">e2144</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0002144">10.1371/journal.pntd.0002144</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23593515">23593515</a></span></span></li> <li tag="ref"><a name="ref-63" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shipley</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Selden</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Bats and Viruses: Emergence of Novel Lyssaviruses and Association of Bats with Viral Zoonoses in the EU.</span><span class="source" tagx="source" title="source"><i>Trop Med Infect Dis.</i></span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">4</span>(<span class="issue" tagx="issue" title="issue">1</span>): pii: E31. <span class="pub-id"><a href="https://dx.doi.org/10.3390/tropicalmed4010031">10.3390/tropicalmed4010031</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30736432">30736432</a></span></span></li> <li tag="ref"><a name="ref-64" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singer</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thomson</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mclauchlan</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">GLUE: a flexible software system for virus sequence data.</span><span class="source" tagx="source" title="source"><i>BMC Bioinformatics.</i></span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">19</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">532</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/s12859-018-2459-9">10.1186/s12859-018-2459-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30563445">30563445</a></span></span></li> <li tag="ref"><a name="ref-65" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stamatakis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.</span><span class="source" tagx="source" title="source"><i>Bioinformatics.</i></span><span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">30</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">1312</span>–<span class="lpage" tagx="lpage" title="lpage">1313</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btu033">10.1093/bioinformatics/btu033</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451623">24451623</a></span></span></li> <li tag="ref"><a name="ref-104" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suseno</span><span class="given-names" tagx="given-names" title="given-names">PP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rysava</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brum</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Lessons for rabies control and elimination programmes – a decade of One Health experience from Bali, Indonesia.</span><span class="source" tagx="source" title="source"><i>Rev Sci Tech.</i></span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">38</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">213</span>–<span class="lpage" tagx="lpage" title="lpage">224</span>. <span class="pub-id"><a href="https://dx.doi.org/10.20506/rst.38.1.2954">10.20506/rst.38.1.2954</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31564729">31564729</a></span></span></li> <li tag="ref"><a name="ref-82" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Talbi</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Phylodynamics and human-mediated dispersal of a zoonotic virus.</span><span class="source" tagx="source" title="source"><i>PLoS Pathog.</i></span><span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">6</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">e1001166</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1001166">10.1371/journal.ppat.1001166</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21060816">21060816</a></span></span></li> <li tag="ref"><a name="ref-67" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tohma</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saito</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Demetria</span><span class="given-names" tagx="given-names" title="given-names">CS</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Molecular and mathematical modeling analyses of inter-island transmission of rabies into a previously rabies-free island in the Philippines.</span><span class="source" tagx="source" title="source"><i>Infect Genet Evol.</i></span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">38</span>:<span class="fpage" tagx="fpage" title="fpage">22</span>–<span class="lpage" tagx="lpage" title="lpage">28</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.meegid.2015.12.001">10.1016/j.meegid.2015.12.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26656835">26656835</a></span></span></li> <li tag="ref"><a name="ref-68" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trewby</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nadin-Davis</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Real</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Processes Underlying Rabies Virus Incursions across US-Canada Border as Revealed by Whole-Genome Phylogeography.</span><span class="source" tagx="source" title="source"><i>Emerg Infect Dis.</i></span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">23</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">1454</span>–<span class="lpage" tagx="lpage" title="lpage">1461</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3201/eid2309.170325">10.3201/eid2309.170325</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28820138">28820138</a></span></span></li> <li tag="ref"><a name="ref-69" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Troupin</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dacheux</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanguy</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Large-Scale Phylogenomic Analysis Reveals the Complex Evolutionary History of Rabies Virus in Multiple Carnivore Hosts.</span><span class="source" tagx="source" title="source"><i>PLoS Pathog.</i></span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">e1006041</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1006041">10.1371/journal.ppat.1006041</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27977811">27977811</a></span></span></li> <li tag="ref"><a name="ref-70" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Velasco-Villa</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Escobar</span><span class="given-names" tagx="given-names" title="given-names">LE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanchez</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Successful strategies implemented towards the elimination of canine rabies in the Western Hemisphere.</span><span class="source" tagx="source" title="source"><i>Antiviral Res.</i></span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">143</span>:<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">12</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2017.03.023">10.1016/j.antiviral.2017.03.023</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28385500">28385500</a></span></span></li> <li tag="ref"><a name="ref-71" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Velasco-Villa</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orciari</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Souza</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Molecular epizootiology of rabies associated with terrestrial carnivores in Mexico.</span><span class="source" tagx="source" title="source"><i>Virus Res.</i></span><span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">111</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">13</span>–<span class="lpage" tagx="lpage" title="lpage">27</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virusres.2005.03.007">10.1016/j.virusres.2005.03.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15896399">15896399</a></span></span></li> <li tag="ref"><a name="ref-72" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viana</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shirima</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">John</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Integrating serological and genetic data to quantify cross-species transmission: brucellosis as a case study.</span><span class="source" tagx="source" title="source"><i>Parasitology.</i></span><span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">143</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">821</span>–<span class="lpage" tagx="lpage" title="lpage">834</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1017/S0031182016000044">10.1017/S0031182016000044</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26935267">26935267</a></span></span></li> <li tag="ref"><a name="ref-73" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: WHO</span>: <span class="mixed-article-title" title="mixed-article-title">WHO Expert Consultation on Rabies, third report</span>. World Health Organization technical report series. Geneva: World Health Organization.<span class="year" tagx="year" title="year">2018a</span>;<span class="volume" tagx="volume" title="volume">1012</span><a href="https://apps.who.int/iris/bitstream/handle/10665/272364/9789241210218-eng.pdf?ua=1">Reference Source</a></span></li> <li tag="ref"><a name="ref-74" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="collab'">collab: WHO</span>: <span class="mixed-article-title" title="mixed-article-title">Zero by 30: The global strategic plan to end human deaths from dog-mediated rabies by 2030</span>. Abela-Ridder B, Editor: WHO.<span class="year" tagx="year" title="year">2018b</span><a href="https://apps.who.int/iris/bitstream/handle/10665/272756/9789241513838-eng.pdf?ua=1">Reference Source</a></span></li> <li tag="ref"><a name="ref-83" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wick</span><span class="given-names" tagx="given-names" title="given-names">RR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Judd</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">KE</span></span></span>: <span class="mixed-article-title" title="mixed-article-title">Performance of neural network basecalling tools for Oxford Nanopore sequencing.</span><span class="source" tagx="source" title="source"><i>Genome Biol.</i></span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">20</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">129</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1186/s13059-019-1727-y">10.1186/s13059-019-1727-y</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31234903">31234903</a></span></span></li> <li tag="ref"><a name="ref-75" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewandowski</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lumley</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Detection of viral pathogens with multiplex nanopore MinION sequencing: Be careful with cross-Talk.</span><span class="source" tagx="source" title="source"><i>Front Microbiol.</i></span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">9</span>:<span class="fpage" tagx="fpage" title="fpage">2225</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2018.02225">10.3389/fmicb.2018.02225</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30283430">30283430</a></span></span></li> <li tag="ref"><a name="ref-101" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Zoonotic Source Attribution of <i>Salmonella enterica</i> Serotype Typhimurium Using Genomic Surveillance Data, United States.</span><span class="source" tagx="source" title="source"><i>Emerg Infect Dis.</i></span><span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">25</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">82</span>–<span class="lpage" tagx="lpage" title="lpage">91</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3201/eid2501.180835">10.3201/eid2501.180835</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30561314">30561314</a></span></span></li> <li tag="ref"><a name="ref-76" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zinsstag</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lechenne</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laager</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span>: <span class="mixed-article-title" title="mixed-article-title">Vaccination of dogs in an African city interrupts rabies transmission and reduces human exposure.</span><span class="source" tagx="source" title="source"><i>Sci Transl Med.</i></span><span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">421</span>): pii: eaaf6984. <span class="pub-id"><a href="https://dx.doi.org/10.1126/scitranslmed.aaf6984">10.1126/scitranslmed.aaf6984</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29263230">29263230</a></span></span></li> </ul> </div> </div>  </body></html>