<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/18/2019.12.21.885921.atom" />
<meta name="HW.pisa" content="biorxiv;2019.12.21.885921v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Ecological and evolutionary drivers of hemoplasma infection and bacterial genotype sharing in a Neotropical bat community" />
<meta name="DC.Identifier" content="10.1101/2019.12.21.885921" />
<meta name="DC.Date" content="2020-03-18" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Most emerging pathogens can infect multiple species, underscoring the importance of understanding the ecological and evolutionary factors that allow some hosts to harbor greater infection prevalence and share pathogens with other species. However, our understanding of pathogen jumps is primarily based around viruses, despite bacteria accounting for the greatest proportion of zoonoses. Because bacterial pathogens in bats (Order: Chiroptera) can have conservation and human health consequences, studies that examine the ecological and evolutionary drivers of bacterial prevalence and barriers to pathogen sharing are crucially needed. We here studied hemotropic Mycoplasma spp. (i.e., hemoplasmas) across a species-rich bat community in Belize over two years. Across 469 bats spanning 33 species, half of individuals and two-thirds of species were hemoplasma positive. Infection prevalence was higher for males and for species with larger body mass and colony sizes. Hemoplasmas displayed high genetic diversity (21 novel genotypes) and strong host specificity. Evolutionary patterns supported co-divergence of bats and bacterial genotypes alongside phylogenetically constrained host shifts. Bat species centrality to the network of shared hemoplasma genotypes was phylogenetically clustered and unrelated to prevalence, further suggesting rare—but detectable—bacterial sharing between species. Our study highlights the importance of using fine phylogenetic scales when assessing host specificity and suggests phylogenetic similarity may play a key role in host shifts for not only viruses but also bacteria. Such work more broadly contributes to increasing efforts to understand cross-species transmission and epidemiological consequences of bacterial pathogens." />
<meta name="DC.Contributor" content="Daniel J. Becker" />
<meta name="DC.Contributor" content="Kelly A. Speer" />
<meta name="DC.Contributor" content="Alexis M. Brown" />
<meta name="DC.Contributor" content="M. Brock Fenton" />
<meta name="DC.Contributor" content="Alex D. Washburne" />
<meta name="DC.Contributor" content="Sonia Altizer" />
<meta name="DC.Contributor" content="Daniel G. Streicker" />
<meta name="DC.Contributor" content="Raina K. Plowright" />
<meta name="DC.Contributor" content="Vladimir E. Chizhikov" />
<meta name="DC.Contributor" content="Nancy B. Simmons" />
<meta name="DC.Contributor" content="Dmitriy V. Volokhov" />
<meta name="article:published_time" content="2020-03-18" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Ecological and evolutionary drivers of hemoplasma infection and bacterial genotype sharing in a Neotropical bat community" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Most emerging pathogens can infect multiple species, underscoring the importance of understanding the ecological and evolutionary factors that allow some hosts to harbor greater infection prevalence and share pathogens with other species. However, our understanding of pathogen jumps is primarily based around viruses, despite bacteria accounting for the greatest proportion of zoonoses. Because bacterial pathogens in bats (Order: Chiroptera) can have conservation and human health consequences, studies that examine the ecological and evolutionary drivers of bacterial prevalence and barriers to pathogen sharing are crucially needed. We here studied hemotropic Mycoplasma spp. (i.e., hemoplasmas) across a species-rich bat community in Belize over two years. Across 469 bats spanning 33 species, half of individuals and two-thirds of species were hemoplasma positive. Infection prevalence was higher for males and for species with larger body mass and colony sizes. Hemoplasmas displayed high genetic diversity (21 novel genotypes) and strong host specificity. Evolutionary patterns supported co-divergence of bats and bacterial genotypes alongside phylogenetically constrained host shifts. Bat species centrality to the network of shared hemoplasma genotypes was phylogenetically clustered and unrelated to prevalence, further suggesting rare-but detectable-bacterial sharing between species. Our study highlights the importance of using fine phylogenetic scales when assessing host specificity and suggests phylogenetic similarity may play a key role in host shifts for not only viruses but also bacteria. Such work more broadly contributes to increasing efforts to understand cross-species transmission and epidemiological consequences of bacterial pathogens.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2019.12.21.885921v2" />
<meta name="citation_id" content="2019.12.21.885921v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2019.12.21.885921v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.21.885921v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.21.885921v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921.full.pdf" />
<meta name="citation_doi" content="10.1101/2019.12.21.885921" />
<meta name="citation_num_pages" content="52" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2019.12.21.885921" />
<meta name="citation_author" content="Daniel J. Becker" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University" />
<meta name="citation_author_institution" content="Center for the Ecology of Infectious Disease, University of Georgia" />
<meta name="citation_author_email" content="danbeck@iu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4315-8628" />
<meta name="citation_author" content="Kelly A. Speer" />
<meta name="citation_author_institution" content="Richard Gilder Graduate School, American Museum of Natural History" />
<meta name="citation_author_institution" content="Department of Invertebrate Zoology, National Museum of Natural History, Smithsonian Institution" />
<meta name="citation_author_institution" content="Center for Conservation Genomics, Smithsonian Conservation Biology Institute" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8775-7254" />
<meta name="citation_author" content="Alexis M. Brown" />
<meta name="citation_author_institution" content="Department of Ecology and Evolution, Stony Brook University" />
<meta name="citation_author" content="M. Brock Fenton" />
<meta name="citation_author_institution" content="Department of Biology, Western University" />
<meta name="citation_author" content="Alex D. Washburne" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, Montana State University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4047-3411" />
<meta name="citation_author" content="Sonia Altizer" />
<meta name="citation_author_institution" content="Center for the Ecology of Infectious Disease, University of Georgia" />
<meta name="citation_author_institution" content="Odum School of Ecology, University of Georgia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9966-2773" />
<meta name="citation_author" content="Daniel G. Streicker" />
<meta name="citation_author_institution" content="Odum School of Ecology, University of Georgia" />
<meta name="citation_author_institution" content="MRC–University of Glasgow Centre for Virus Research" />
<meta name="citation_author_institution" content="Institute of Biodiversity, Animal Health and Comparative Medicine, University of Glasgow" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7475-2705" />
<meta name="citation_author" content="Raina K. Plowright" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, Montana State University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3338-6590" />
<meta name="citation_author" content="Vladimir E. Chizhikov" />
<meta name="citation_author_institution" content="Center for Biologics Evaluation and Research, Food and Drug Administration" />
<meta name="citation_author" content="Nancy B. Simmons" />
<meta name="citation_author_institution" content="Richard Gilder Graduate School, American Museum of Natural History" />
<meta name="citation_author_institution" content="Department of Mammalogy, Division of Vertebrate Zoology, American Museum of Natural History" />
<meta name="citation_author" content="Dmitriy V. Volokhov" />
<meta name="citation_author_institution" content="Center for Biologics Evaluation and Research, Food and Drug Administration" />
<meta name="citation_reference" content="Albery, G. F., Eskew, E. A., Ross, N., &amp; Olival, K. J. (2019). Predicting the global mammalian viral sharing network using phylogeography. BioRxiv, 732255." />
<meta name="citation_reference" content="Alcorn, K., Gerrard, J., Cochrane, T., Graham, R., Jennison, A., Irwin, P. J., &amp; Barbosa, A. D. (2020). First report of Candidatus Mycoplasma haemohominis infection in Australia causing persistent fever in an animal carer. Clinical Infectious Diseases: An Official Publication of the Infectious Diseases Society of America. doi: 10.1093/cid/ciaa089" />
<meta name="citation_reference" content="Bai, Y., Kosoy, M., Recuenco, S., Alvarez, D., Moran, D., Turmelle, A., … Rupprecht, C. (2011). Bartonella spp. in Bats, Guatemala. Emerging Infectious Diseases, 17(7), 1269–1272. doi: 10.3201/eid1707.101867" />
<meta name="citation_reference" content="Baillie, J., Hilton-Taylor, C., &amp; Stuart, S. N. (2004). IUCN red list of threatened species: a global species assessment. IUCN." />
<meta name="citation_reference" content="Balbuena, J. A., Míguez-Lozano, R., &amp; Blasco-Costa, I. (2013). PACo: A Novel Procrustes Application to Cophylogenetic Analysis. PLOS ONE, 8(4), e61048. doi: 10.1371/journal.pone.0061048" />
<meta name="citation_reference" content="Becker, D. J., Bergner, L. M., Bentz, A. B., Orton, R. J., Altizer, S., &amp; Streicker, D. G. (2018). Genetic diversity, infection prevalence, and possible transmission routes of Bartonella spp. in vampire bats. PLOS Neglected Tropical Diseases, 12(9), e0006786. doi: 10.1371/journal.pntd.0006786" />
<meta name="citation_reference" content="Becker, D. J., Crowley, D. E., Washburne, A. D., &amp; Plowright, R. K. (2019). Temporal and spatial limitations in global surveillance for bat filoviruses and henipaviruses. Biology Letters, 15(12), 20190423. doi: 10.1098/rsbl.2019.0423" />
<meta name="citation_reference" content="Bell, D. C., Atkinson, J. S., &amp; Carlson, J. W. (1999). Centrality measures for disease transmission networks. Social Networks, 21(1), 1–21. doi: 10.1016/S0378-8733(98)00010-0" />
<meta name="citation_reference" content="Blyton, M. D. J., Banks, S. C., Peakall, R., Lindenmayer, D. B., &amp; Gordon, D. M. (2014). Not all types of host contacts are equal when it comes to E. coli transmission. Ecology Letters, 17(8), 970–978. doi: 10.1111/ele.12300" />
<meta name="citation_reference" content="Bonato, L., Figueiredo, M. A. P., Gonçalves, L. R., Machado, R. Z., &amp; André, M. R. (2015). Occurrence and molecular characterization of Bartonella spp. and hemoplasmas in neotropical primates from Brazilian Amazon. Comparative Immunology, Microbiology and Infectious Diseases, 42, 15–20. doi: 10.1016/j.cimid.2015.09.001" />
<meta name="citation_reference" content="Bonneaud, C., Weinert, L., &amp; Kuijper, B. (2019). Understanding the emergence of bacterial pathogens in novel hosts. Phil. Trans. R. Soc. B. doi: 10.1098/rstb.2018.0328" />
<meta name="citation_reference" content="Botero-Castro, F., Tilak, M., Justy, F., Catzeflis, F., Delsuc, F., &amp; Douzery, E. J. P. (2013). Next-generation sequencing and phylogenetic signal of complete mitochondrial genomes for resolving the evolutionary history of leaf-nosed bats (Phyllostomidae). Molecular Phylogenetics and Evolution, 69(3), 728–739. doi: 10.1016/j.ympev.2013.07.003" />
<meta name="citation_reference" content="Bürkner, P.-C. (2017). brms: An R package for Bayesian multilevel models using Stan. Journal of Statistical Software, 80(1), 1–28." />
<meta name="citation_reference" content="Burnham, K. P., &amp; Anderson, D. R. (2002). Model selection and multimodel inference: a practical information-theoretic approach. Springer Science &amp; Business Media." />
<meta name="citation_reference" content="Butts, C. T. (2008). Social network analysis with sna. Journal of Statistical Software, 24(6), 1–51." />
<meta name="citation_reference" content="Chamberlain, S. A., &amp; Szöcs, E. (2013). taxize: taxonomic search and retrieval in R. F1000Research, 2." />
<meta name="citation_reference" content="Citti, C., &amp; Blanchard, A. (2013). Mycoplasmas and their host: emerging and re-emerging minimal pathogens. Trends in Microbiology, 21(4), 196–203. doi: 10.1016/j.tim.2013.01.003" />
<meta name="citation_reference" content="Clare, E. L., Lim, B. K., Engstrom, M. D., Eger, J. L., &amp; Hebert, P. D. (2007). DNA barcoding of Neotropical bats: species identification and discovery within Guyana. Molecular Ecology Notes, 7(2), 184–190." />
<meta name="citation_reference" content="Cohen, C., Shemesh, M., Garrido, M., Messika, I., Einav, M., Khokhlova, I., … Hawlena, H. (2018). Haemoplasmas in wild rodents: Routes of transmission and infection dynamics. Molecular Ecology, 27(18), 3714–3726. doi: 10.1111/mec.14826" />
<meta name="citation_reference" content="Csardi, G., &amp; Nepusz, T. (2006). The igraph software package for complex network research. InterJournal, Complex Systems, 1695(5), 1–9." />
<meta name="citation_reference" content="Dallas, T. A., Han, B. A., Nunn, C. L., Park, A. W., Stephens, P. R., &amp; Drake, J. M. (2019). Host traits associated with species roles in parasite sharing networks. Oikos, 128(1), 23–32." />
<meta name="citation_reference" content="De Vienne, D. M., Refrégier, G., López-Villavicencio, M., Tellier, A., Hood, M. E., &amp; Giraud, T. (2013). Cospeciation vs host-shift speciation: methods for testing, evidence from natural associations and relation to coevolution. New Phytologist, 198(2), 347–385." />
<meta name="citation_reference" content="Di Cataldo, S., Kamani, J., Cevidanes, A., Msheliza, E. G., &amp; Millán, J. (2020). Hemotropic mycoplasmas in bats captured near human settlements in Nigeria. Comparative Immunology, Microbiology and Infectious Diseases, 70, 101448. doi: 10.1016/j.cimid.2020.101448" />
<meta name="citation_reference" content="Downs, C. J., Schoenle, L. A., Han, B. A., Harrison, J. F., &amp; Martin, L. B. (2019). Scaling of Host Competence. Trends in Parasitology, 0(0). doi: 10.1016/j.pt.2018.12.002" />
<meta name="citation_reference" content="Edgar, R. C., Haas, B. J., Clemente, J. C., Quince, C., &amp; Knight, R. (2011). UCHIME improves sensitivity and speed of chimera detection. Bioinformatics, 27(16), 2194–2200. doi: 10.1093/bioinformatics/btr381" />
<meta name="citation_reference" content="Evans, N. J., Bown, K., Timofte, D., Simpson, V. R., &amp; Birtles, R. J. (2009). Fatal Borreliosis in Bat Caused by Relapsing Fever Spirochete, United Kingdom. Emerging Infectious Diseases, 15(8), 1331–1333. doi: 10.3201/eid1508.090475" />
<meta name="citation_reference" content="Fenton, A., Streicker, D. G., Petchey, O. L., &amp; Pedersen, A. B. (2015). Are All Hosts Created Equal? Partitioning Host Species Contributions to Parasite Persistence in Multihost Communities. The American Naturalist, 186(5), 610–622. doi: 10.1086/683173" />
<meta name="citation_reference" content="Fenton, M. B., Bernard, E., Bouchard, S., Hollis, L., Johnston, D. S., Lausen, C. L., … Zigouris, J. (2001). The bat fauna of Lamanai, Belize: roosts and trophic roles. Journal of Tropical Ecology, 17(4), 511–524." />
<meta name="citation_reference" content="Fountain-Jones, N. M., Pearse, W. D., Escobar, L. E., Alba-Casals, A., Carver, S., Davies, T. J., … Worsley-Tonks, K. (2018). Towards an eco-phylogenetic framework for infectious disease ecology. Biological Reviews, 93(2), 950–970." />
<meta name="citation_reference" content="Frank, H. K., Mendenhall, C. D., Judson, S. D., Daily, G. C., &amp; Hadly, E. A. (2016). Anthropogenic impacts on Costa Rican bat parasitism are sex specific. Ecology and Evolution, 6(14), 4898–4909. doi: 10.1002/ece3.2245" />
<meta name="citation_reference" content="Garamszegi, L. Z. (2006). The evolution of virulence and host specialization in malaria parasites of primates. Ecology Letters, 9(8), 933–940. doi: 10.1111/j.1461-0248.2006.00936.x" />
<meta name="citation_reference" content="Garamszegi, L. Z. (2014). Modern Phylogenetic Comparative Methods and Their Application in Evolutionary Biology: Concepts and Practice. Springer." />
<meta name="citation_reference" content="Gelman, A., Goodrich, B., Gabry, J., &amp; Vehtari, A. (2019). R-squared for Bayesian Regression Models. The American Statistician, 73(3), 307–309. doi: 10.1080/00031305.2018.1549100" />
<meta name="citation_reference" content="Geoghegan, J. L., Duchêne, S., &amp; Holmes, E. C. (2017). Comparative analysis estimates the relative frequencies of co-divergence and cross-species transmission within viral families. PLOS Pathogens, 13(2), e1006215. doi: 10.1371/journal.ppat.1006215" />
<meta name="citation_reference" content="Gómez, J. M., Nunn, C. L., &amp; Verdú, M. (2013). Centrality in primate–parasite networks reveals the potential for the transmission of emerging infectious diseases to humans. Proceedings of the National Academy of Sciences, 110(19), 7738–7741. doi: 10.1073/pnas.1220716110" />
<meta name="citation_reference" content="Gonçalves, L. R., Roque, A. L. R., Matos, C. A., Fernandes, S. de J., Olmos, I. D. F., Machado, R. Z., &amp; André, M. R. (2015). Diversity and molecular characterization of novel hemoplasmas infecting wild rodents from different Brazilian biomes. Comparative Immunology, Microbiology and Infectious Diseases, 43, 50–56. doi: 10.1016/j.cimid.2015.10.006" />
<meta name="citation_reference" content="González-Salazar, C., Martínez-Meyer, E., &amp; López-Santiago, G. (2014). A hierarchical classification of trophic guilds for North American birds and mammals. Revista Mexicana de Biodiversidad, 85(3), 931–941. doi: 10.7550/rmb.38023" />
<meta name="citation_reference" content="Goto, K., Yasuda, M., Hayashimoto, N., &amp; Ebukuro, S. (2010). Morphological and sequence analysis of Mycoplasma sp. isolated from the oral cavity of a house musk shrew (Suncus murinus). The Journal of Veterinary Medical Science, 72(1), 109–111." />
<meta name="citation_reference" content="Graham, C. H., Storch, D., &amp; Machac, A. (2018). Phylogenetic scale in ecology and evolution. Global Ecology and Biogeography, 27(2), 175–187. doi: 10.1111/geb.12686" />
<meta name="citation_reference" content="Gunnell, G. F., &amp; Simmons, N. B. (2012). Evolutionary history of bats: fossils, molecules and morphology. Cambridge University Press." />
<meta name="citation_reference" content="Guy, C., Thiagavel, J., Mideo, N., &amp; Ratcliffe, J. M. (2019). Phylogeny matters: revisiting ‘a comparison of bats and rodents as reservoirs of zoonotic viruses.’ Royal Society Open Science, 6(2), 181182." />
<meta name="citation_reference" content="Han, B. A., Kramer, A. M., &amp; Drake, J. M. (2016). Global Patterns of Zoonotic Disease in Mammals. Trends in Parasitology, 32(7), 565–577." />
<meta name="citation_reference" content="Han, B. A., Schmidt, J. P., Alexander, L. W., Bowden, S. E., Hayman, D. T., &amp; Drake, J. M. (2016). Undiscovered bat hosts of filoviruses. PLOS Negl Trop Dis, 10(7), e0004815." />
<meta name="citation_reference" content="Hattori, N., Kuroda, M., Katano, H., Takuma, T., Ito, T., Arai, N., … Niki, Y. (2020). Candidatus Mycoplasma haemohominis in Human, Japan. Emerging Infectious Diseases, 26(1), 11–19. doi: 10.3201/eid2601.190983" />
<meta name="citation_reference" content="Herrera, J. P., Duncan, N., Clare, E., Fenton, M. B., &amp; Simmons, N. (2018). Disassembly of Fragmented Bat Communities in Orange Walk District, Belize. Acta Chiropterologica, 20(1), 147–159. doi: 10.3161/15081109ACC2018.20.1.011" />
<meta name="citation_reference" content="Hijmans, R. J., Williams, E., Vennes, C., &amp; Hijmans, M. R. J. (2019). Package ‘geosphere.’Spherical Trigonometryv, 1–5." />
<meta name="citation_reference" content="Hochachka, W. M., &amp; Dhondt, A. A. (2000). Density-dependent decline of host abundance resulting from a new infectious disease. Proceedings of the National Academy of Sciences, 97(10), 5303–5306." />
<meta name="citation_reference" content="Hornok, S., Szoke, K., Meli, M. L., Sándor, A. D., Görföl, T., Estók, P., … Hofmann-Lehmann, R. (2019). Molecular detection of vector-borne bacteria in bat ticks (Acari: Ixodidae, Argasidae) from eight countries of the Old and New Worlds. Parasites &amp; Vectors, 12(1), 50. doi: 10.1186/s13071-019-3303-4" />
<meta name="citation_reference" content="Huang, S., Bininda-Emonds, O. R., Stephens, P. R., Gittleman, J. L., &amp; Altizer, S. (2014). Phylogenetically related and ecologically similar carnivores harbour similar parasite assemblages. Journal of Animal Ecology, 83(3), 671–680." />
<meta name="citation_reference" content="Huang, S., Drake, J. M., Gittleman, J. L., &amp; Altizer, S. (2015). Parasite diversity declines with host evolutionary distinctiveness: A global analysis of carnivores. Evolution, 69(3), 621–630. doi: 10.1111/evo.12611" />
<meta name="citation_reference" content="Hutchinson, M. C., Cagua, E. F., Balbuena, J. A., Stouffer, D. B., &amp; Poisot, T. (2017). paco: implementing Procrustean Approach to Cophylogeny in R. Methods in Ecology and Evolution, 8(8), 932–940." />
<meta name="citation_reference" content="Ikeda, P., Seki, M. C., Carrasco, A. O. T., Rudiak, L. V., Miranda, J. M. D., Gonçalves, S. M. M., … others. (2017). Evidence and molecular characterization of Bartonella spp. and hemoplasmas in neotropical bats in Brazil. Epidemiology &amp; Infection, 1–15." />
<meta name="citation_reference" content="Johnson, P. T. J., Rohr, J. R., Hoverman, J. T., Kellermanns, E., Bowerman, J., &amp; Lunde, K. B. (2012). Living fast and dying of infection: host life history drives interspecific variation in infection and disease risk. Ecology Letters, 15(3), 235–242. doi: 10.1111/j.1461-0248.2011.01730.x" />
<meta name="citation_reference" content="Jombart, T., Kendall, M., AlmagrouGarcia, J., &amp; Colijn, C. (2017). treespace: Statistical exploration of landscapes of phylogenetic trees. Molecular Ecology Resources, 17(6), 1385– 1392. doi: 10.1111/1755-0998.12676" />
<meta name="citation_reference" content="Jones, P. L., Hämsch, F., Page, R. A., Kalko, E. K., &amp; O’mara, M. T. (2017). Foraging and roosting behaviour of the fringe-lipped bat, Trachops cirrhosus, on Barro Colorado Island, Panamá. Acta Chiropterologica, 19(2), 337–346." />
<meta name="citation_reference" content="Kämpfer, P., &amp; Glaeser, S. P. (2012). Prokaryotic taxonomy in the sequencing era–the polyphasic approach revisited. Environmental Microbiology, 14(2), 291–317." />
<meta name="citation_reference" content="Kellner, A., Carver, S., Scorza, V., McKee, C. D., Lappin, M., Crooks, K. R., … Antolin, M. F. (2018). Transmission pathways and spillover of an erythrocytic bacterial pathogen from domestic cats to wild felids. Ecology and Evolution, 8(19), 9779–9792. doi: 10.1002/ece3.4451" />
<meta name="citation_reference" content="Kelly, C. D., Stoehr, A. M., Nunn, C., Smyth, K. N., &amp; Prokop, Z. M. (2018). Sexual dimorphism in immunity across animals: a meta-analysis. Ecology Letters, 21(12), 1885– 1894. doi: 10.1111/ele.13164" />
<meta name="citation_reference" content="Kembel, S. W., Cowan, P. D., Helmus, M. R., Cornwell, W. K., Morlon, H., Ackerly, D. D., … Webb, C. O. (2010). Picante: R tools for integrating phylogenies and ecology. Bioinformatics, 26(11), 1463–1464." />
<meta name="citation_reference" content="Kumar, S., Stecher, G., Li, M., Knyaz, C., &amp; Tamura, K. (2018). MEGA X: Molecular Evolutionary Genetics Analysis across Computing Platforms. Molecular Biology and Evolution, 35(6), 1547–1549. doi: 10.1093/molbev/msy096" />
<meta name="citation_reference" content="Loayza, A. P., &amp; Loiselle, B. A. (2008). Preliminary information on the home range and movement patterns of Sturnira lilium (Phyllostomidae) in a naturally fragmented landscape in Bolivia. Biotropica, 40(5), 630–635." />
<meta name="citation_reference" content="Longdon, B., Brockhurst, M. A., Russell, C. A., Welch, J. J., &amp; Jiggins, F. M. (2014). The Evolution and Genetics of Virus Host Shifts. PLOS Pathogens, 10(11), e1004395. doi: 10.1371/journal.ppat.1004395" />
<meta name="citation_reference" content="Luis, A. D., O’Shea, T. J., Hayman, D. T. S., Wood, J. L. N., Cunningham, A. A., Gilbert, A. T., … Webb, C. T. (2015). Network analysis of host–virus communities in bats and rodents reveals determinants of cross-species transmission. Ecology Letters, 18(11), 1153–1162. doi: 10.1111/ele.12491" />
<meta name="citation_reference" content="Maggi, R. G., Compton, S. M., Trull, C. L., Mascarelli, P. E., Mozayeni, B. R., &amp; Breitschwerdt, E. B. (2013). Infection with hemotropic Mycoplasma species in patients with or without extensive arthropod or animal contact. Journal of Clinical Microbiology, 51(10), 3237–3241." />
<meta name="citation_reference" content="Mascarelli, P. E., Keel, M. K., Yabsley, M., Last, L. A., Breitschwerdt, E. B., &amp; Maggi, R. G. (2014). Hemotropic mycoplasmas in little brown bats (Myotis lucifugus). Parasit Vectors, 7, 117." />
<meta name="citation_reference" content="McCallum, H., Barlow, N., &amp; Hone, J. (2001). How should pathogen transmission be modelled? Trends in Ecology &amp; Evolution, 16(6), 295–300. doi: 10.1016/S0169-5347(01)02144-9" />
<meta name="citation_reference" content="McKee, C. D., Krawczyk, A. I., Sándor, A., Görföl, T., Földvári, M., &amp; Földvári, G. (2019). Host phylogeny, geographic overlap, and roost sharing shape parasite communities in European bats. Frontiers in Ecology and Evolution, 7." />
<meta name="citation_reference" content="Messick, J. B. (2004). Hemotrophic mycoplasmas (hemoplasmas): a review and new insights into pathogenic potential. Veterinary Clinical Pathology, 33(1), 2–13. doi: 10.1111/j.1939-165X.2004.tb00342.x" />
<meta name="citation_reference" content="Michonneau, F., Brown, J. W., &amp; Winter, D. J. (2016). rotl : an R package to interact with the Open Tree of Life data. Methods in Ecology and Evolution. doi: 10.1111/2041-210X.12593" />
<meta name="citation_reference" content="Millán, J., Cevidanes, A., Sacristán, I., Alvarado, M., Sepúlveda, G., Ramos-Mella, C. A., &amp; Lisón, F. (2019). Detection and Characterization of Hemotropic Mycoplasmas in Bats in Chile. Journal of Wildlife Diseases. doi: 10.7589/2018-12-290" />
<meta name="citation_reference" content="Millán, J., López-Roig, M., Delicado, V., Serra-Cobo, J., &amp; Esperón, F. (2015). Widespread infection with hemotropic mycoplasmas in bats in Spain, including a hemoplasma closely related to “Candidatus Mycoplasma hemohominis.” Comparative Immunology, Microbiology and Infectious Diseases, 39, 9–12. doi: 10.1016/j.cimid.2015.01.002" />
<meta name="citation_reference" content="Monteiro, L. R., &amp; Nogueira, M. R. (2011). Evolutionary patterns and processes in the radiation of phyllostomid bats. BMC Evolutionary Biology, 11, 137. doi: 10.1186/1471-2148- 11-137" />
<meta name="citation_reference" content="Mühldorfer, K. (2013). Bats and Bacterial Pathogens: A Review. Zoonoses and Public Health, 60(1), 93–103. doi: 10.1111/j.1863-2378.2012.01536.x" />
<meta name="citation_reference" content="Museux, K., Boretti, F. S., Willi, B., Riond, B., Hoelzle, K., Hoelzle, L. E., … Hofmann-Lehmann, R. (2009). In vivo transmission studies of “Candidatus Mycoplasma turicensis” in the domestic cat. Veterinary Research, 40(5), 1, 45–14. doi: 10.1051/vetres/2009028" />
<meta name="citation_reference" content="Myhrvold, N. P., Baldridge, E., Chan, B., Sivam, D., Freeman, D. L., &amp; Ernest, S. K. M. (2015). An amniote life-history database to perform comparative analyses with birds, mammals, and reptiles. Ecology, 96(11), 3109–3109. doi: 10.1890/15-0846R.1" />
<meta name="citation_reference" content="Olival, K. J., Hosseini, P. R., Zambrana-Torrelio, C., Ross, N., Bogich, T. L., &amp; Daszak, P. (2017). Host and viral traits predict zoonotic spillover from mammals. Nature, 546(7660), 646–650. doi: 10.1038/nature22975" />
<meta name="citation_reference" content="Paradis, E., Claude, J., &amp; Strimmer, K. (2004). APE: analyses of phylogenetics and evolution in R language. Bioinformatics, 20(2), 289–290." />
<meta name="citation_reference" content="Pedersen, A. B., Altizer, S., Poss, M., Cunningham, A. A., &amp; Nunn, C. L. (2005). Patterns of host specificity and transmission among parasites of wild primates. International Journal for Parasitology, 35(6), 647–657." />
<meta name="citation_reference" content="Pitcher, D. G., &amp; Nicholas, R. A. J. (2005). Mycoplasma host specificity: Fact or fiction? The Veterinary Journal, 170(3), 300–306. doi: 10.1016/j.tvjl.2004.08.011" />
<meta name="citation_reference" content="Plowright, R. K., Parrish, C. R., McCallum, H., Hudson, P. J., Ko, A. I., Graham, A. L., &amp; Lloyd-Smith, J. O. (2017). Pathways to zoonotic spillover. Nature Reviews Microbiology, 15(8), 502–510." />
<meta name="citation_reference" content="Reid, F. (1997). A Field Guide to the Mammals of Central America and Southeast Mexico. OUP USA." />
<meta name="citation_reference" content="Santana, S. E., Dial, T. O., Eiting, T. P., &amp; Alfaro, M. E. (2011). Roosting Ecology and the Evolution of Pelage Markings in Bats. PLOS ONE, 6(10), e25845. doi: 10.1371/journal.pone.0025845" />
<meta name="citation_reference" content="Sikes, R. S., Care, A., &amp; Mammalogists, U. C. of the A. S. of. (2016). 2016 Guidelines of the American Society of Mammalogists for the use of wild mammals in research and education. Journal of Mammalogy, 97(3), 663–688." />
<meta name="citation_reference" content="Soto, F., Walker, R., Sepulveda, M., Bittencourt, P., Acosta-Jamett, G., &amp; Müller, A. (2017). Occurrence of canine hemotropic mycoplasmas in domestic dogs from urban and rural areas of the Valdivia Province, southern Chile. Comparative Immunology, Microbiology and Infectious Diseases, 50, 70–77. doi: 10.1016/j.cimid.2016.11.013" />
<meta name="citation_reference" content="Streicker, D. G., Recuenco, S., Valderrama, W., Benavides, J. G., Vargas, I., Pacheco, V., … Altizer, S. (2012). Ecological and anthropogenic drivers of rabies exposure in vampire bats: implications for transmission and control. Proc. R. Soc. B, 279(1742), 3384–3392. doi: 10.1098/rspb.2012.0538" />
<meta name="citation_reference" content="Streicker, D. G., Turmelle, A. S., Vonhof, M. J., Kuzmin, I. V., McCracken, G. F., &amp; Rupprecht, C. E. (2010). Host phylogeny constrains cross-species emergence and establishment of rabies virus in bats. Science, 329(5992), 676–679." />
<meta name="citation_reference" content="Ter Hofstede, H. M., Fenton, M. B., &amp; Whitaker, J. O. (2004). Host and host-site specificity of bat flies (Diptera: Streblidae and Nycteribiidae) on Neotropical bats (Chiroptera). Canadian Journal of Zoology, 82(4), 616–626." />
<meta name="citation_reference" content="VanderWaal, K. L., Atwill, E. R., Isbell, L. A., &amp; McCowan, B. (2014). Quantifying microbe transmission networks for wild and domestic ungulates in Kenya. Biological Conservation, 169(Supplement C), 136–146. doi: 10.1016/j.biocon.2013.11.008" />
<meta name="citation_reference" content="VanderWaal, K. L., &amp; Ezenwa, V. O. (2016). Heterogeneity in pathogen transmission: mechanisms and methodology. Functional Ecology, 30(10), 1606–1622." />
<meta name="citation_reference" content="Vehtari, A., Gelman, A., &amp; Gabry, J. (2017). Practical Bayesian model evaluation using leave-one-out cross-validation and WAIC. Statistics and Computing, 27(5), 1413–1432." />
<meta name="citation_reference" content="Viechtbauer, W. (2010). Conducting meta-analyses in R with the metafor package. Journal of Statistical Software, 36(3), 1–48." />
<meta name="citation_reference" content="Vieira, R. F. C., Molento, M. B., dos Santos, L. C., Moraes, W., Cubas, Z. S., Santos, A. P., … Messick, J. B. (2009). Detection of a novel hemoplasma based on 16S rRNA gene DNA in captive and free-ranging capybaras (Hydrochaeris hydrochaeris). Veterinary Microbiology, 139(3–4), 410–413. doi: 10.1016/j.vetmic.2009.06.018" />
<meta name="citation_reference" content="Voigt, C. C., von Helversen, O., Michener, R., &amp; Kunz, T. H. (2001). The economics of harem maintenance in the sac-winged bat, Saccopteryx bilineata (Emballonuridae). Behavioral Ecology and Sociobiology, 50(1), 31–36." />
<meta name="citation_reference" content="Volokhov, D. V., Becker, D. J., Bergner, L. M., Camus, M. S., Orton, R. J., Chizhikov, V. E., … Streicker, D. G. (2017). Novel hemotropic mycoplasmas are widespread and genetically diverse in vampire bats. Epidemiology &amp; Infection, 145(15), 3154–3167." />
<meta name="citation_reference" content="Volokhov, Dmitriy V., Hwang, J., Chizhikov, V. E., Danaceau, H., &amp; Gottdenker, N. L. (2017). Prevalence, Genotype Richness, and Coinfection Patterns of Hemotropic Mycoplasmas in Raccoons (Procyon lotor) on Environmentally Protected and Urbanized Barrier Islands. Applied and Environmental Microbiology, 83(9). doi: 10.1128/AEM.00211-17" />
<meta name="citation_reference" content="Volokhov, Dmitriy V., Kong, H., George, J., Anderson, C., &amp; Chizhikov, V. E. (2008). Biological enrichment of Mycoplasma agents by cocultivation with permissive cell cultures. Applied and Environmental Microbiology, 74(17), 5383–5391. doi: 10.1128/AEM.00720-08" />
<meta name="citation_reference" content="Volokhov, Dmitriy V., Norris, T., Rios, C., Davidson, M. K., Messick, J. B., Gulland, F. M., &amp; Chizhikov, V. E. (2011). Novel hemotrophic mycoplasma identified in naturally infected California sea lions (Zalophus californianus). Veterinary Microbiology, 149(1–2), 262–268. doi: 10.1016/j.vetmic.2010.10.026" />
<meta name="citation_reference" content="Volokhov, Dmitriy V., Simonyan, V., Davidson, M. K., &amp; Chizhikov, V. E. (2012). RNA polymerase beta subunit (rpoB) gene and the 16S–23S rRNA intergenic transcribed spacer region (ITS) as complementary molecular markers in addition to the 16S rRNA gene for phylogenetic analysis and identification of the species of the family Mycoplasmataceae. Molecular Phylogenetics and Evolution, 62(1), 515–528." />
<meta name="citation_reference" content="Walker Vergara, R., Morera Galleguillos, F., Gómez Jaramillo, M., Pereira Almosny, N. R., Arauna Martínez, P., Grob Behne, P., … Müller, A. (2016). Prevalence, risk factor analysis, and hematological findings of hemoplasma infection in domestic cats from Valdivia, Southern Chile. Comparative Immunology, Microbiology and Infectious Diseases, 46, 20–26. doi: 10.1016/j.cimid.2016.03.004" />
<meta name="citation_reference" content="Washburne, A. D., Silverman, J. D., Morton, J. T., Becker, D. J., Crowley, D., Mukherjee, S., … Plowright, R. K. (2019). Phylofactorization: a graph partitioning algorithm to identify phylogenetic scales of ecological data. Ecological Monographs, 0(0), e01353. doi: 10.1002/ecm.1353" />
<meta name="citation_reference" content="Webber, Q. M. R., Fletcher, Q. E., &amp; Willis, C. K. R. (2017). Viral Richness is Positively Related to Group Size, but Not Mating System, in Bats. EcoHealth, 1–10. doi: 10.1007/s10393-017-1276-3" />
<meta name="citation_reference" content="Willi, B., Boretti, F. S., Meli, M. L., Bernasconi, M. V., Casati, S., Hegglin, D., … Hofmann-Lehmann, R. (2007). Real-time PCR investigation of potential vectors, reservoirs, and shedding patterns of feline hemotropic mycoplasmas. Applied and Environmental Microbiology, 73(12), 3798–3802. doi: 10.1128/AEM.02977-06" />
<meta name="citation_reference" content="Willi, B., Boretti, F. S., Tasker, S., Meli, M. L., Wengi, N., Reusch, C. E., … Hofmann-Lehmann, R. (2007). From Haemobartonella to hemoplasma: Molecular methods provide new insights. Veterinary Microbiology, 125(3–4), 197–209. doi: 10.1016/j.vetmic.2007.06.027" />
<meta name="citation_reference" content="Wilman, H., Belmaker, J., Simpson, J., de la Rosa, C., Rivadeneira, M. M., &amp; Jetz, W. (2014). EltonTraits 1.0: Species-level foraging attributes of the world’s birds and mammals. Ecology, 95(7), 2027–2027. doi: 10.1890/13-1917.1" />
<meta name="citation_reference" content="Withenshaw, S. M., Devevey, G., Pedersen, A. B., &amp; Fenton, A. (2016). Multihost Bartonella parasites display covert host specificity even when transmitted by generalist vectors. Journal of Animal Ecology, 85(6), 1442–1452." />
<meta name="citation_reference" content="Woolhouse, M. E. J., &amp; Gowtage-Sequeria, S. (2005). Host Range and Emerging and Reemerging Pathogens. Emerging Infectious Diseases, 11(12), 1842–1847. doi: 10.3201/eid1112.050997" />
<meta name="citation_reference" content="Woolhouse, M. E., Taylor, L. H., &amp; Haydon, D. T. (2001). Population biology of multihost pathogens. Science, 292(5519), 1109–1112." />
<meta name="citation_reference" content="Wright, E. S., Yilmaz, L. S., &amp; Noguera, D. R. (2012). DECIPHER, a Search-Based Approach to Chimera Identification for 16S rRNA Sequences. Applied and Environmental Microbiology, 78(3), 717–725. doi: 10.1128/AEM.06516-11" />
<meta name="twitter:title" content="Ecological and evolutionary drivers of hemoplasma infection and bacterial genotype sharing in a Neotropical bat community" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Most emerging pathogens can infect multiple species, underscoring the importance of understanding the ecological and evolutionary factors that allow some hosts to harbor greater infection prevalence and share pathogens with other species. However, our understanding of pathogen jumps is primarily based around viruses, despite bacteria accounting for the greatest proportion of zoonoses. Because bacterial pathogens in bats (Order: Chiroptera) can have conservation and human health consequences, studies that examine the ecological and evolutionary drivers of bacterial prevalence and barriers to pathogen sharing are crucially needed. We here studied hemotropic Mycoplasma spp. (i.e., hemoplasmas) across a species-rich bat community in Belize over two years. Across 469 bats spanning 33 species, half of individuals and two-thirds of species were hemoplasma positive. Infection prevalence was higher for males and for species with larger body mass and colony sizes. Hemoplasmas displayed high genetic diversity (21 novel genotypes) and strong host specificity. Evolutionary patterns supported co-divergence of bats and bacterial genotypes alongside phylogenetically constrained host shifts. Bat species centrality to the network of shared hemoplasma genotypes was phylogenetically clustered and unrelated to prevalence, further suggesting rare—but detectable—bacterial sharing between species. Our study highlights the importance of using fine phylogenetic scales when assessing host specificity and suggests phylogenetic similarity may play a key role in host shifts for not only viruses but also bacteria. Such work more broadly contributes to increasing efforts to understand cross-species transmission and epidemiological consequences of bacterial pathogens." />
<meta name="og-title" property="og:title" content="Ecological and evolutionary drivers of hemoplasma infection and bacterial genotype sharing in a Neotropical bat community" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2019.12.21.885921v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Most emerging pathogens can infect multiple species, underscoring the importance of understanding the ecological and evolutionary factors that allow some hosts to harbor greater infection prevalence and share pathogens with other species. However, our understanding of pathogen jumps is primarily based around viruses, despite bacteria accounting for the greatest proportion of zoonoses. Because bacterial pathogens in bats (Order: Chiroptera) can have conservation and human health consequences, studies that examine the ecological and evolutionary drivers of bacterial prevalence and barriers to pathogen sharing are crucially needed. We here studied hemotropic Mycoplasma spp. (i.e., hemoplasmas) across a species-rich bat community in Belize over two years. Across 469 bats spanning 33 species, half of individuals and two-thirds of species were hemoplasma positive. Infection prevalence was higher for males and for species with larger body mass and colony sizes. Hemoplasmas displayed high genetic diversity (21 novel genotypes) and strong host specificity. Evolutionary patterns supported co-divergence of bats and bacterial genotypes alongside phylogenetically constrained host shifts. Bat species centrality to the network of shared hemoplasma genotypes was phylogenetically clustered and unrelated to prevalence, further suggesting rare—but detectable—bacterial sharing between species. Our study highlights the importance of using fine phylogenetic scales when assessing host specificity and suggests phylogenetic similarity may play a key role in host shifts for not only viruses but also bacteria. Such work more broadly contributes to increasing efforts to understand cross-species transmission and epidemiological consequences of bacterial pathogens." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-18" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2019.12.21.885921v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2019.12.21.885921v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2019.12.21.885921v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2019.12.21.885921v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1200651" />
    <title>Ecological and evolutionary drivers of hemoplasma infection and bacterial genotype sharing in a Neotropical bat community | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"EdtFfpUDcV57Y--ECmm2IEn3pCDOexEH8SuuVg7GRR8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1200651","apath":"\/biorxiv\/early\/2020\/03\/18\/2019.12.21.885921.atom","pisa":"biorxiv;2019.12.21.885921v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2019.12.21.885921v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2019.12.21.885921v2","title":"Ecological and evolutionary drivers of hemoplasma infection and bacterial genotype sharing in a Neotropical bat community","identifier":"node\/1200651"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AUDPim1alop26i2Su8Yosa9DcjVSaFKOWPWqqSaDu1RnoD6RrDA1K3rb4Q6i7TB2w6imbFNlsrL1wURxeIsXj5waOg5Q9FNakckuaKwHOsjrpH3JLPyy3\\\\\\\/n2ftHhAVfFMsWg3CnKHI7MbcCuOizEIFtkReJGWYTN95VSxUrGwzYwYE6nd6u7IQxH0MC4bUV6UkzrVIxID4LqSuHyyJZkCgYHu5Oz7xHqBizBMhTMrc0cKxiVfjAOc9SFHpwWXTNndZZH9AvtgY3\\\\\\\/jpTLA76GnaXownXrpH1txXObayXPfMt2bh+6\\\\\\\/+ntwoEWyS5tJmPcEc5VOnKQ1JlObpc0J7xmskvYb3KXhMRmfTv34lYcRFfpElrqvmhu8fDPhCLXFlFiADjkV9GBHNtNduzdF1WvUDAfk4LT\\\\\\\/OTbT4ZrJqlX\\\\\\\/it+d6tVFC4AdpKhmuXPYU2rw16J97K2H9KROsC4FtopSG4CuacaSexTZIjhaD\\\\\\\/Ke4rGG4b\\\\\\\/bVOpAyPsBKA\\\\\\\/RnhCsZj0drfkcnOuFMjifpOpKkz3\\\\\\\/PjCed3yc0GeFpElFekgkUccniH8eB9HzY1fqnTUpfEHwV5THp817oaVRRXN1jTWUBdQkEEto1e\\\\\\\/qqlsHiyKrH9b3ZB\\\\\\\/SiZLjzP5KtzVMi57meI2ZixrKag7de7uEmnoOkIqkfwNI6GdPqHKj9D6Pyn6srRpzreCRnwhbccXadR0ZXSrCU27\\\\\\\/iuI6g3AMCAdhUFIud4s3CcQ3DIxtXidNNDKD2UDsIZpIFovaDvHPN+kCR46chsUPZzSLrqprx\\\\\\\/cY3a5h8qheVMbFCHUboA9KaIK0Z5tU8pKe3EJnW2TrNpwnulFg74LErJC0iBzxPstjKkJcKcwDCK2RGWELITDBfCd7smDaTLkQUJ0cCrheW1XxG9ZBWzMlpSlyroqPZ+Int758z2scWjqHVS\\\\\\\/JaKc0rwAuBjlA9ZZp+GSjxCciAfN49uyJqf3jeg9ykx6D3EC0OhGCfPtIj5+dUPMm4oWPr44zNo0djAVyzuIMJ2kcOCtlZmPspJWQpNKTvVzf+UPFxlYJpzQnXCpApAfyRbAVgb49DmeORO0U8wryYxyJQEtPJPcamgIplTYjDQEIKV1GuP\\\\\\\/GU8DmwiBa4a3d4vxDmNO9wMEYGRgAY4mk+uWBw7faOhNjvgyQGIiS\\\\\\\/EneHL6z9jJkeM215TGnUmO6sMrzK+XhcOeJh0MRXOrHnxCjgH7vFVmsS1avsoY14QjRM2\\\\\\\/07WtaX8dLwdAOHpZPD7+El1WjmhwHhRnZ1Rqe\\\\\\\/JcW55tp\\\\\\\/7rTemDgTETbrt+Y208a\\\\\\\/ApXYI88yVKRlt9LfXJOIS7v6Ke1a7EM0aoXFgd\\\u0022,\\\u0022iv\\\u0022:\\\u0022yMtKlLCDtk7NYEVI3lft+A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022be9afc7c0340a645029b6382b3b2f2a4a4b8f8e56c2c12d066b931a64dd29528\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2019.12.21.885921v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1200651 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2019.12.21.885921v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-odzPoQ9MAOola0CBeSIkgNxOOX9N6VHmbgBNdj6ihDc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1200651" id="node1200651" data-pisa="biorxiv;2019.12.21.885921v2" data-pisa-master="biorxiv;2019.12.21.885921" data-apath="/biorxiv/early/2020/03/18/2019.12.21.885921.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Ecological and evolutionary drivers of hemoplasma infection and bacterial genotype sharing in a Neotropical bat community</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4315-8628" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Daniel J.</span> <span class="nlm-surname">Becker</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8775-7254" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Kelly A.</span> <span class="nlm-surname">Speer</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Alexis M.</span> <span class="nlm-surname">Brown</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">M. Brock</span> <span class="nlm-surname">Fenton</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4047-3411" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Alex D.</span> <span class="nlm-surname">Washburne</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-9966-2773" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Sonia</span> <span class="nlm-surname">Altizer</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-7475-2705" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Daniel G.</span> <span class="nlm-surname">Streicker</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-3338-6590" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Raina K.</span> <span class="nlm-surname">Plowright</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Vladimir E.</span> <span class="nlm-surname">Chizhikov</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Nancy B.</span> <span class="nlm-surname">Simmons</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Dmitriy V.</span> <span class="nlm-surname">Volokhov</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2019.12.21.885921 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1200651" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel J. Becker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biology, Indiana University</span>, Bloomington, IN, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Center for the Ecology of Infectious Disease, University of Georgia</span>, Athens, GA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BJ.%2BBecker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Becker%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BJ.%2BBecker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4315-8628" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel J. Becker</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:danbeck@iu.edu" class="" data-icon-position="" data-hide-link-title="0">danbeck@iu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Kelly A. Speer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Richard Gilder Graduate School, American Museum of Natural History</span>, New York, NY, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Invertebrate Zoology, National Museum of Natural History, Smithsonian Institution</span>, Washington, D.C., <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Center for Conservation Genomics, Smithsonian Conservation Biology Institute</span>, National Zoological Park, Washington, D.C., <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kelly%2BA.%2BSpeer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Speer%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKelly%2BA.%2BSpeer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8775-7254" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kelly A. Speer</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alexis M. Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Ecology and Evolution, Stony Brook University</span>, Stony Brook, NY, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexis%2BM.%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexis%2BM.%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">M. Brock Fenton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Department of Biology, Western University</span>, London, Ontario, <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BBrock%2BFenton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fenton%20MB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BBrock%2BFenton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alex D. Washburne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Department of Microbiology and Immunology, Montana State University</span>, Bozeman, MT <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BD.%2BWashburne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Washburne%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlex%2BD.%2BWashburne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4047-3411" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alex D. Washburne</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sonia Altizer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Center for the Ecology of Infectious Disease, University of Georgia</span>, Athens, GA, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>9</span><span class='nlm-institution'>Odum School of Ecology, University of Georgia</span>, Athens, GA <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sonia%2BAltizer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Altizer%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASonia%2BAltizer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9966-2773" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sonia Altizer</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Daniel G. Streicker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>9</span><span class='nlm-institution'>Odum School of Ecology, University of Georgia</span>, Athens, GA <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>10</span><span class='nlm-institution'>MRC–University of Glasgow Centre for Virus Research</span>, Glasgow, <span class='nlm-country'>United Kingdom</span></div><div class='author-affiliation'><span class='nlm-sup'>11</span><span class='nlm-institution'>Institute of Biodiversity, Animal Health and Comparative Medicine, University of Glasgow</span>, <span class='nlm-country'>United Kingdom</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BG.%2BStreicker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Streicker%20DG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BG.%2BStreicker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7475-2705" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel G. Streicker</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Raina K. Plowright </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Department of Microbiology and Immunology, Montana State University</span>, Bozeman, MT <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raina%2BK.%2BPlowright%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Plowright%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARaina%2BK.%2BPlowright%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3338-6590" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Raina K. Plowright</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Vladimir E. Chizhikov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>12</span><span class='nlm-institution'>Center for Biologics Evaluation and Research, Food and Drug Administration</span>, Silver Spring, MD, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vladimir%2BE.%2BChizhikov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chizhikov%20VE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVladimir%2BE.%2BChizhikov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Nancy B. Simmons </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Richard Gilder Graduate School, American Museum of Natural History</span>, New York, NY, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>13</span><span class='nlm-institution'>Department of Mammalogy, Division of Vertebrate Zoology, American Museum of Natural History</span>, New York, NY</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nancy%2BB.%2BSimmons%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simmons%20NB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANancy%2BB.%2BSimmons%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Dmitriy V. Volokhov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>12</span><span class='nlm-institution'>Center for Biologics Evaluation and Research, Food and Drug Administration</span>, Silver Spring, MD, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dmitriy%2BV.%2BVolokhov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Volokhov%20DV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADmitriy%2BV.%2BVolokhov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2019.12.21.885921v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1200651" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1200651/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.21.885921v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1200651" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1200651/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.21.885921v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1200651" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1200651/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.21.885921v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1200651" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1200651/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.21.885921v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1200651" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1200651/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2019.12.21.885921v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1200651" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1200651/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Most emerging pathogens can infect multiple species, underscoring the importance of understanding the ecological and evolutionary factors that allow some hosts to harbor greater infection prevalence and share pathogens with other species. However, our understanding of pathogen jumps is primarily based around viruses, despite bacteria accounting for the greatest proportion of zoonoses. Because bacterial pathogens in bats (Order: Chiroptera) can have conservation and human health consequences, studies that examine the ecological and evolutionary drivers of bacterial prevalence and barriers to pathogen sharing are crucially needed. We here studied hemotropic <em>Mycoplasma</em> spp. (i.e., hemoplasmas) across a species-rich bat community in Belize over two years. Across 469 bats spanning 33 species, half of individuals and two-thirds of species were hemoplasma positive. Infection prevalence was higher for males and for species with larger body mass and colony sizes. Hemoplasmas displayed high genetic diversity (21 novel genotypes) and strong host specificity. Evolutionary patterns supported co-divergence of bats and bacterial genotypes alongside phylogenetically constrained host shifts. Bat species centrality to the network of shared hemoplasma genotypes was phylogenetically clustered and unrelated to prevalence, further suggesting rare—but detectable—bacterial sharing between species. Our study highlights the importance of using fine phylogenetic scales when assessing host specificity and suggests phylogenetic similarity may play a key role in host shifts for not only viruses but also bacteria. Such work more broadly contributes to increasing efforts to understand cross-species transmission and epidemiological consequences of bacterial pathogens.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">Most pathogens that cause disease in humans, domestic animals, and wildlife are capable of infecting multiple host species (<a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">Woolhouse, Taylor, &amp; Haydon, 2001</a>). However, predicting which hosts maintain pathogens and identifying their role in cross-species transmission can be challenging, as many hosts can be infected but not play key roles in the reservoir community (A. <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Fenton, Streicker, Petchey, &amp; Pedersen, 2015</a>). Pathogen jumps between species depends on infection prevalence in the donor host, transmission opportunities between donor and recipient species, and suitability of the recipient host for pathogen replication (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Plowright et al., 2017</a>). Each of these steps can be shaped by ecological and evolutionary factors (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">VanderWaal &amp; Ezenwa, 2016</a>). For example, small-bodied species can have greater competence, the ability to transmit new infections, than larger species (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Downs, Schoenle, Han, Harrison, &amp; Martin, 2019</a>), and host switching is often constrained by phylogeny, owing to similarity in immunological barriers to pathogen replication between closely related species (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Streicker et al., 2010</a>). Identifying the ecological and evolutionary factors that allow some species to harbor greater prevalence and have facilitated pathogen sharing can improve our general understanding of disease emergence (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Fountain-Jones et al., 2018</a>). Examining evolutionary associations between hosts and pathogens can further uncover factors favoring host shifts versus codivergence and assess the frequency of cross-species transmission (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Geoghegan, Duchêne, &amp; Holmes, 2017</a>).</p><p id="p-5">Given the public health and agricultural burdens of many zoonotic pathogens such as SARS coronavirus, avian influenza virus, and rabies virus, many investigations of pathogen prevalence and emergence focus on viruses (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Geoghegan et al., 2017</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Olival et al., 2017</a>). However, more zoonoses are caused by bacteria than other pathogen taxa (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Han, Kramer, &amp; Drake, 2016</a>), and bacterial pathogens can negatively impact newly infected host species (e.g., <em>Mycoplasma galliscepticum</em>, a poultry pathogen, caused rapid population declines in wild house finches; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Hochachka &amp; Dhondt, 2000</a>). More attention to bacteria and their propensity for host specificity versus generalism is accordingly important for understanding whether factors that govern cross-species transmission of viruses can be extended to other pathogens (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bonneaud, Weinert, &amp; Kuijper, 2019</a>). Bacterial pathogens have been especially understudied for bats (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Mühldorfer, 2013</a>), in contrast to intensive studies of zoonotic viruses across the Chiroptera (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Luis et al., 2015</a>). However, many bacterial pathogens are likely important to both bat conservation and human health due to pathogenic effects on bats themselves as well as their zoonotic potential (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Becker et al., 2018</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Evans, Bown, Timofte, Simpson, &amp; Birtles, 2009</a>).</p><p id="p-6">To determine the ecological and evolutionary drivers of bacterial prevalence and barriers to pathogen sharing, we focused on hemotropic <em>Mycoplasma</em> spp. (i.e., hemoplasmas) in a species-rich bat community in Belize (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Fenton et al., 2001</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Herrera, Duncan, Clare, Fenton, &amp; Simmons, 2018</a>). The Neotropics have remarkable bat diversity owing to adaptive radiation in the Phyllostomidae (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Gunnell &amp; Simmons, 2012</a>), producing a range of feeding strategies (e.g., frugivory, carnivory, sanguivory), body sizes, and roost preferences (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Monteiro &amp; Nogueira, 2011</a>). Hemoplasmas are intracellular erythrocytic bacteria transmitted through direct contact (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Cohen et al., 2018</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Museux et al., 2009</a>) and also possibly arthropod vectors (Willi, Boretti, Meli, et al., 2007). Hemoplasmas can cause acute and chronic anemia, especially for immunocompromised hosts; however, many animals develop inapparent infections and are asymptomatic (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Messick, 2004</a>). As <em>Mycoplasma</em> spp. lack many of the metabolic pathways associated with energy production and synthesis of cell components found in other bacteria, they are fully dependent on host cells (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Citti &amp; Blanchard, 2013</a>). Hemoplasmas have therefore been described as mostly host specialists (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Pitcher &amp; Nicholas, 2005</a>), yet interspecies and potentially zoonotic transmission can occur (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Maggi et al., 2013</a>; Willi, Boretti, Tasker, et al., 2007). Hemoplasmas are common and genetically diverse in bats (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Di Cataldo, Kamani, Cevidanes, Msheliza, &amp; Millán, 2020</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Ikeda et al., 2017</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Mascarelli et al., 2014</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Millán et al., 2019</a>, <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">2019</a>; <a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Volokhov et al., 2017</a>), which offers an ideal model system for identifying the ecological and evolutionary factors structuring bacterial infection risks within and between host species.</p><p id="p-7">Many cross-species comparisons of pathogen infection risks and sharing have used less-diverse host communities (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Johnson et al., 2012</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">VanderWaal, Atwill, Isbell, &amp; McCowan, 2014</a>) or global datasets of host–pathogen associations that can be limited by heterogeneous sampling effort and variation in pathogen detection methods (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Dallas et al., 2019</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Huang, Bininda-Emonds, Stephens, Gittleman, &amp; Altizer, 2014</a>). Our focus on a widespread pathogens group in a highly diverse host community allowed us to capitalize on strong host trait variation while controlling for sampling effort and diagnostics methods (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Becker, Crowley, Washburne, &amp; Plowright, 2019</a>; Han, Kramer, et al., 2016). Past work has also used host–pathogen networks to characterize contemporary or historic transmission at often coarse taxonomic scales (e.g., pathogen species complexes or genera; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Blyton, Banks, Peakall, Lindenmayer, &amp; Gordon, 2014</a>; <a id="xref-ref-88-2" class="xref-bibr" href="#ref-88">VanderWaal et al., 2014</a>). However, as bat species can be infected by multiple hemoplasma genotypes, and because genotypes with ≥ 99% sequence identity of their 16S rRNA genes can represent different bacterial species (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Volokhov, Hwang, Chizhikov, Danaceau, &amp; Gottdenker, 2017</a>; <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Volokhov, Simonyan, Davidson, &amp; Chizhikov, 2012</a>), focusing on genotypes provides finer-scale resolution to determine the ecological and evolutionary features of species that facilitate pathogen sharing and to identify likely maintenance hosts of bacterial infections (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Fountain-Jones et al., 2018</a>).</p><p id="p-8">We asked three questions about the relative contribution of ecological traits and evolutionary history to structuring infection patterns and pathogen sharing. First, what are the individual and ecological predictors of hemoplasma infection in a Neotropical bat community? Second, does the distribution of hemoplasma genotypes across the bat community map onto the bat phylogeny, as predicted by host–pathogen codivergence? Lastly, if genotype sharing among host species occurs, which host clades or traits best predict species ability to share pathogens? We predicted that ecological covariates such as ectoparasitism and large host colonies could increase bacteria risk through vector-borne or density-dependent transmission (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">McCallum, Barlow, &amp; Hone, 2001</a>; Willi, Boretti, Meli, et al., 2007). We also expected hemoplasma genotypes would be specific to particular host species but that more closely related bats would share hemoplasma genotypes, indicating phylogenetically restricted host shifts (<a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Pitcher &amp; Nicholas, 2005</a>). However, ecological traits that increase the risk of pathogen exposure between species, such as occupying a greater diversity of roosting habitats, could also facilitate pathogen genotype sharing among less closely related hosts (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">McKee et al., 2019</a>).</p></div><div class="section" id="sec-2"><h2 class="">Methods</h2><div id="sec-3" class="subsection"><h3>Bat capture and sampling</h3><p id="p-9">From April 24 to May 6 2017 and from April 23 to May 5 2018, we sampled 469 bats from 33 species captured in two adjacent areas in the Orange Walk District of Belize: Lamanai Archaeological Reserve (LAR) and Ka’Kabish (KK; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). The LAR is bordered by the New River Lagoon, forest, and agriculture, while KK is a remnant patch of forest surrounded by agriculture located 10 km away. We consider these sites to be independent, as the small home ranges of many Neotropical bat species and lack of continuous forest between LAR and KK has likely restricted most individual bat inter-site movement (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Jones, Hämsch, Page, Kalko, &amp; O’mara, 2017</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Loayza &amp; Loiselle, 2008</a>); however, some species may have greater connectivity (e.g., inter-roost movement of <em>Desmodus rotundus</em> occurs rarely). At least 44 of the 70 bat species in Belize have been recorded in this region (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Herrera et al., 2018</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Reid, 1997</a>). Bats were captured with mist nets primarily along flight paths and occasionally at the exits of roosts from 19:00 until 22:00. Harp traps were also set from 18:00 to 05:00. In KK, <em>Desmodus rotundus</em>, <em>Trachops cirrhosus</em>, and <em>Chrotopterus auritus</em> were captured at their shared roost. In the LAR, <em>Desmodus rotundus</em>, <em>Saccopteryx bilineata</em>, and <em>Glossophaga soricina</em> were sampled from a shared roost, although individuals were also sampled along flight paths across the greater site. More broadly, patterns of roost sharing of bat species in northern Belize remain elusive.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Study sites in northern Belize. The shaded inset shows the location of Orange Walk District. Borders show the boundaries of the LAR (Lamanai Archaeological Reserve) and KK (Ka&#x2019;Kabish). White and tan shading indicates agricultural and urban development, while dark green shading represents intact forest. Satellite imagery was derived from Google Maps. Stacked bar plots display the relative abundance of each sampled bat family per study site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1607181521" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Study sites in northern Belize. The shaded inset shows the location of Orange Walk District. Borders show the boundaries of the LAR (Lamanai Archaeological Reserve) and KK (Ka&#x2019;Kabish). White and tan shading indicates agricultural and urban development, while dark green shading represents intact forest. Satellite imagery was derived from Google Maps. Stacked bar plots display the relative abundance of each sampled bat family per study site.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F1.medium.gif" width="440" height="378"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F1.medium.gif" width="440" height="378"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-10" class="first-child">Study sites in northern Belize. The shaded inset shows the location of Orange Walk District. Borders show the boundaries of the LAR (Lamanai Archaeological Reserve) and KK (Ka’Kabish). White and tan shading indicates agricultural and urban development, while dark green shading represents intact forest. Satellite imagery was derived from Google Maps. Stacked bar plots display the relative abundance of each sampled bat family per study site.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">Bats were placed in cloth bags until processing and were identified to species (and sex) based on morphology (<a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Reid, 1997</a>). Reproductive activity was indicated by the presence of scrotal testes in males and by the evidence of pregnancy or lactation in females; across bat species, 69% of males and 42% of females were in reproductive condition. We also visually screened bats for the presence of ectoparasites (i.e., bat flies, ticks, bat bugs, mites; Ter Hofstede, Fenton, &amp; Whitaker, 2004). We collected 3–30 μL of blood (volumes were dependent on bat mass) by lancing the propatagial vein with a sterile needle. Blood was collected with heparinized capillary tubes and stored on Whatman FTA cards to preserve bacterial DNA. Field procedures followed guidelines for safe and humane handling of bats published by of the American Society of Mammalogists (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Sikes, Care, &amp; Mammalogists, 2016</a>) and were approved by the Institutional Animal Care and Use Committees of the University of Georgia (A2014 04-016-Y3-A5) and American Museum of Natural History (AMNHIACUC-20170403 and AMNHIACUC-20180123). Fieldwork was authorized by the Belize Forest Department under permits WL/2/1/17(16), WL/2/1/17(19), and WL/2/1/18(16). Sample size was similar between years (2017=202, 2018=267) but varied by site (LAR=365, KK=101). More species were sampled for blood at LAR (<em>n</em>=33) than KK (<em>n</em>=17; <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>), reflecting site differences in species richness (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Herrera et al., 2018</a>). We sampled 1–139 individuals per bat species (the maximum being <em>Desmodus rotundus</em>), with a mean of 14 individuals per bat species (Table S1).</p></div><div id="sec-4" class="subsection"><h3>DNA extraction, PCR amplification, and amplicon sequencing</h3><p id="p-12">Genomic DNA was extracted from blood on FTA cards using QIAamp DNA Investigator Kits (Qiagen). We tested DNA for hemoplasmas using PCR with primers and procedures described in prior analyses (<a id="xref-ref-94-2" class="xref-bibr" href="#ref-94">Volokhov et al., 2017</a>). We included blank FTA punches as an extraction control, ultrapure water as a negative control, and <em>Candidatus</em> Mycoplasma haemozalophi DNA as a positive control (<a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Volokhov et al., 2011</a>). Amplicons from PCR-positive samples were purified by electrophoresis and extracted using the QIAquick Gel Extraction Kit (Qiagen).</p><p id="p-13">To determine hemoplasma infection status, all 16S rRNA amplicons were directly sequenced by Macrogen (<a href="https://www.macrogenusa.com/">https://www.macrogenusa.com/</a>). Amplicons were sequenced with the same primers used for PCR amplification and then with internal (walking) primers when needed (<a id="xref-ref-94-3" class="xref-bibr" href="#ref-94">Volokhov et al., 2017</a>). Negative DNA samples were tested for amplification quality using the universal PCR primers targeting the mammal mitochondrial 16S rRNA gene (<a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Volokhov, Kong, George, Anderson, &amp; Chizhikov, 2008</a>) or the mitochondrial cytochrome <em>c</em> oxidase subunit 1 gene (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Clare, Lim, Engstrom, Eger, &amp; Hebert, 2007</a>); all hemoplasma-negative DNA samples gave positive signal in the mitochondrial 16S rRNA gene-and/or the COI-specific PCR assays. All amplified sequences were subjected to chimeric sequence analysis using DECIPHER (<a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">Wright, Yilmaz, &amp; Noguera, 2012</a>) and UCHIME (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Edgar, Haas, Clemente, Quince, &amp; Knight, 2011</a>). All hemoplasma sequences have been deposited in GenBank under accession numbers <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MH245119&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom">MH245119</a>–<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MH245194&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom">MH245194</a> and <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MK353807&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom">MK353807</a>–<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MK353892&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom">MK353892</a>; four positive samples were identified as <em>Bartonella</em> spp. during sequencing and were considered hemoplasma negative in our analyses.</p></div><div id="sec-5" class="subsection"><h3>Bat phylogenetic data</h3><p id="p-14">We used the <em>rotl</em> and <em>ape</em> packages in R to extract a bat phylogeny from the Open Tree of Life and to calculate branch lengths with Grafen’s method (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Michonneau, Brown, &amp; Winter, 2016</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Paradis, Claude, &amp; Strimmer, 2004</a>). To assess hemoplasma genotype sharing as a function of host phylogenetic similarity, we derived pairwise phylogenetic distances between the 33 sampled bat species (Fig. S1). Because more evolutionarily distant and distinct species could display less frequent bacterial genotype sharing owing to ecological and immunological barriers to pathogen exposure and replication (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Huang, Drake, Gittleman, &amp; Altizer, 2015</a>), we used the <em>picante</em> package and our bat phylogeny to derive evolutionary distinctiveness (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Kembel et al., 2010</a>).</p></div><div id="sec-6" class="subsection"><h3>Host species trait data</h3><p id="p-15">We obtained species-level data on host traits relevant to pathogen transmission from previously published sources (Table S2). We obtained fecundity (litter size, litters per year), body mass, and diet from the Amniote Life History and EltonTraits databases (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Myhrvold et al., 2015</a>; <a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">Wilman et al., 2014</a>). For foraging ecology, which could affect bacterial exposure (e.g., trophic interactions; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Kellner et al., 2018</a>), we defined three dietary guilds: frugivory (including nectarivory; <em>n</em>=11), insectivory (<em>n</em>=18), and carnivory (including sanguivory and piscivory, <em>n</em>=4; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">González-Salazar, Martínez-Meyer, &amp; López-Santiago, 2014</a>). We also considered the proportion of plant-based items in diet. We simplified foraging strata into aerial (<em>n</em>=14), arboreal (<em>n</em>=16, including scansorial), and ground-or aquatic-level foraging (<em>n</em>=3). We also expanded prior compilations of wing aspect ratios and roost preferences to serve as proxies for ecological overlap among species (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Fenton et al., 2001</a>; <a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Herrera et al., 2018</a>; <a id="xref-ref-81-3" class="xref-bibr" href="#ref-81">Reid, 1997</a>). Roost type was simplified to open (e.g., only foliage; <em>n</em>=6) or closed (e.g., hollows, caves; <em>n</em>=27), and roost flexibility was simplified to using one (<em>n</em>=16) or multiple roost types (<em>n</em>=17). We classified maximum colony sizes as small-to-medium (e.g., under 100 individuals; <em>n</em>=20) or large (e.g., hundreds to thousands; <em>n</em>=13; <a id="xref-ref-81-4" class="xref-bibr" href="#ref-81">Reid, 1997</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Santana, Dial, Eiting, &amp; Alfaro, 2011</a>), as most values were reported in ranges. We did not record pairwise sympatry (e.g., <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Luis et al., 2015</a>; <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">McKee et al., 2019</a>) given that all species occur in Belize (Fig. S2). Yet because more widely distributed species could have more opportunities for pathogen sharing due to range overlap, we used the <em>geosphere</em> package and data from the International Union for Conservation of Nature to derive geographic range size (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Baillie, Hilton-Taylor, &amp; Stuart, 2004</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Hijmans, Williams, Vennes, &amp; Hijmans, 2019</a>). Missing species-level traits were taken from other databases, primary literature, or closely related species (Table S2).</p></div><div id="sec-7" class="subsection"><h3>Individual-level analysis of bat infection status</h3><p id="p-16">We first used the <em>prevalence</em> package to estimate hemoplasma infection prevalence and its 95% confidence interval (Wilson method). We then used phylogenetic generalized linear mixed models (GLMMs) to test if infection status varied by sex, reproductive status, year, site, and ectoparasite presence while accounting for bat phylogenetic relatedness. We fit candidate GLMMs that considered all fixed effects as well as interactions between sex and reproduction and between site and year. We also considered a model that excluded site to account for possible non-independence of LAR and KK alongside an intercept-only model. As vampire bats were banded for a mark–recapture study (<a id="xref-ref-94-4" class="xref-bibr" href="#ref-94">Volokhov et al., 2017</a>) and some were sampled between and within years (<em>n</em>=14), we randomly selected one of each recapture. After removing recaptures and missing values (<em>n</em>=323), we fit the phylogenetic GLMMs using the <em>brms</em> package, default priors, and infection status as a Bernoulli-distributed response. We included random effects for bat species and phylogeny, the latter of which used the phylogenetic covariance matrix (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Bürkner, 2017</a>). We ran four chains for 20,000 iterations with a burn-in period of 10,000, thinned every 10 steps, for a total 4,000 samples. We compared GLMMs using the leave-one-out cross-validation (LOOIC) and assessed fit with a Bayesian <em>R<sup>2</sup>,</em> including the total modeled variance and that attributed to only the fixed effects (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Gelman, Goodrich, Gabry, &amp; Vehtari, 2019</a>; <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Vehtari, Gelman, &amp; Gabry, 2017</a>). We then estimated fixed effects (means and 95% highest density intervals [HDI]) from the posterior distributions of each predictor from the top GLMM. Lastly, we assessed the sensitivity of individual-level analyses to overrepresentation of <em>Desmodus rotundus</em> by randomly subsampling this species by the maximum <em>n</em> for other bat species.</p></div><div id="sec-8" class="subsection"><h3>Species-level analysis of hemoplasma prevalence</h3><p id="p-17">We next calculated infection prevalence per species, using the <em>metafor</em> package to estimate logit-transformed proportions and sampling variances (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Viechtbauer, 2010</a>). We used the <em>nlme</em> package to estimate phylogenetic signal as Pagel’s λ with a weighted phylogenetic generalized least squares (PGLS) model to account for within-species variance (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Garamszegi, 2014</a>). We next used a graph-partitioning algorithm, phylogenetic factorization, to flexibly identify clades with significantly different prevalence estimates at various taxonomic depths. We used the <em>taxize</em> package to obtain a bat taxonomy from the National Center for Biotechnology Information (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Chamberlain &amp; Szöcs, 2013</a>) and used the <em>phylofactor</em> package to partition prevalence as a Bernoulli-distributed response in a GLM (<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Washburne et al., 2019</a>). We determined the number of significant bat clades using Holm’s sequentially rejective test with a 5% family-wise error rate.</p><p id="p-18">To identify species trait correlates of prevalence, we fit 11 PGLS models (weighted by sampling variance) with body mass, annual fecundity (litters per year * pups per litter), dietary guild, quantitative diet, foraging strata, aspect ratio, roost type, roost flexibility, colony size, geographic range size, and evolutionary distinctiveness as predictors. We also fit PGLS models with only sample size or an intercept. We compared models with Akaike information criterion corrected for small sample sizes (AICc) and estimated <em>R<sup>2</sup></em> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Burnham &amp; Anderson, 2002</a>).</p></div><div id="sec-9" class="subsection"><h3>Hemoplasma phylogenetic analyses and genotype assignment</h3><p id="p-19">We compared our 16S rRNA sequences to those in GenBank (<a id="xref-ref-94-5" class="xref-bibr" href="#ref-94">Volokhov et al., 2017</a>; <a id="xref-ref-97-2" class="xref-bibr" href="#ref-97">Volokhov et al., 2011</a>). Briefly, we aligned sequences using Clustal X, and inter-and intra-species similarity values were generated using BioEdit. Genetic distances were calculated with the Kimura two-parameter and Tamura–Nei models, and the phylogeny was constructed using MEGA X with the minimum evolution algorithm (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Kumar, Stecher, Li, Knyaz, &amp; Tamura, 2018</a>).</p><p id="p-20">We assigned hemoplasma genotypes to positive bats based on analysis of the 16S rRNA partial gene (860–1000 bp) sequences in GenBank and their clustering on the phylogeny. Genotypes were designated as novel if (<em>i</em>) sequences differed from the closest hemoplasma sequences in GenBank by ≥ 1.5% and/or (<em>ii</em>) if sequence similarity was &lt;1.5% but genotype-specific reproducible mutations (at least two per sequence) were observed between hemoplasma sequences from at least two independent bat samples and the nearest GenBank hemoplasma sequences. These genotype-specific mutations were further used to differentiate closely related hemoplasma genotypes from our sample. We caution that genotype is not synonymous with species, as analysis of the 16S rRNA gene alone is insufficient for accurate species identification of <em>Mycoplasma</em> spp. (<a id="xref-ref-98-2" class="xref-bibr" href="#ref-98">Volokhov et al., 2012</a>). Future studies using genomics or housekeeping genes may identify independent but closely related hemoplasma species in our genotypes.</p><p id="p-21">To assess if hemoplasma genotype assignments were associated with site and year, we used χ<sup>2</sup> tests with <em>p</em> values generated through a Monte Carlo procedure. Prior to our phylogenetic and network analyses of genotype distributions across bat species (see below), we used another χ<sup>2</sup> test to assess the association between hemoplasma genotype identify and bat host identity.</p></div><div id="sec-10" class="subsection"><h3>Evolutionary relationships between bats and hemoplasmas</h3><p id="p-22">To determine the degree to which bat hemoplasma genotypes display host specificity and to describe their evolutionary relationships with host species, we used our bat and hemoplasma phylogenies to construct a binary association matrix. To test the dependence of the hemoplasma phylogeny upon the bat phylogeny and thus assess evidence of evolutionary codivergence, we applied the Procrustes Approach to Cophylogeny (PACo) using distance matrices and the <em>paco</em> package (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Hutchinson, Cagua, Balbuena, Stouffer, &amp; Poisot, 2017</a>). We used a jackknife procedure to estimate the degree to which each bat–genotype link supported a hypothesis of phylogenetic congruence; links were supported if their upper 95% confidence interval was below the mean of all squared jackknife residuals (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Balbuena, Míguez-Lozano, &amp; Blasco-Costa, 2013</a>).</p></div><div id="sec-11" class="subsection"><h3>Hemoplasma genotype sharing among bat species</h3><p id="p-23">We used hemoplasma genotype assignments to create a network, with each node representing a bat species and edges representing shared genotypes among bat species pairs. We built an adjacency matrix using the <em>igraph</em> package and used the Louvain method to assess the structure of bat–hemoplasma communities within this network (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Csardi &amp; Nepusz, 2006</a>). To test whether the distribution of hemoplasma genotypes across our Neotropical bat species is shaped by host phylogeny, we used two GLMs to predict counts of shared genotypes (Poisson errors) and the presence of sharing (binomial errors) by phylogenetic distance between bat species. We assessed statistical significance with a quadratic assignment procedure via the <em>sna</em> package (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Butts, 2008</a>).</p><p id="p-24">We calculated two metrics of network centrality to quantify different aspects of how important a node (bat species) is to hemoplasma genotype sharing: degree and eigenvector centrality (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bell, Atkinson, &amp; Carlson, 1999</a>). Whereas degree indicates the number of other species with which a host shares bacterial genotypes (i.e., links per node), eigenvector centrality indicates the tendency for a host to share genotypes with species that also share more genotypes (i.e., connectivity). Eigenvector centrality is thus an extension of degree that can identify hubs of parasite sharing (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Gómez, Nunn, &amp; Verdú, 2013</a>). These two metrics were moderately correlated (ρ=0.59), with many non-zero degree species displaying zero eigenvector centrality. To examine spatial and temporal patterns in host centrality, we built separate adjacency networks per each site and year. We fit separate GLMs to ask how hemoplasma sharing centrality was predicted by site, year, and the two-way interaction. Degree was modeled as a Poisson-distributed response, while eigenvector centrality was logit-transformed and used Gaussian errors. We next applied phylogenetic factorization to both metrics and weighted the algorithms by the square-root sample size per species (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Garamszegi, 2014</a>). We then fit the same PGLS models used in our prevalence analysis to identify the most competitive trait predictors of bat species centrality to hemoplasma sharing. To lastly assess whether network centrality is associated with hemoplasma prevalence, we fit two weighted PGLS models with each centrality metric as a univariate predictor.</p></div></div><div class="section" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Hemoplasma infection status</h3><p id="p-25">We detected sequence-confirmed hemoplasma infection in 239 of 469 individuals (51%; 95% CI: 46–55%), with positive individuals in 23 of the 33 sampled bat species (Table S3). The most parsimonious phylogenetic GLMM explained 22% of the modeled variance in infection status and included only sex, reproductive status, ectoparasites, and year (ΔLOOIC=0.16, <em>w<sub>i</sub></em>=0.27; Table S4), suggesting site to be uninformative. Males had higher odds of infection than females (OR=2.35, 95% HDI: 1.36–4.03), but risk was unrelated to ectoparasitism, reproduction, or year (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a>). This result was weakly sensitive to overrepresentation by <em>Desmodus rotundus</em>, as effect size and the prevalence sex difference was weaker when we randomly subsampled this species (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>). Across all models, fixed effects only explained up to 7% of the modeled variance, suggesting more variation explained by the species and phylogeny random effects.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predictors of individual bat hemoplasma infection status. (A) Odds ratios and 95% HDIs from the most parsimonious phylogenetic GLMM (Table S4). Estimates that do not overlap with 1 (dashed line) are displayed in black. Reference levels for the odds ratios include bats sampled at LAR, females, reproductive bats, absence of ectoparasites, and bats sampled in 2017. (B) Infection prevalence and 95% confidence intervals (Wilson method) stratified by sex. Results are shown for the full dataset and after randomly subsampling Desmodus rotundus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1607181521" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Predictors of individual bat hemoplasma infection status. (A) Odds ratios and 95% HDIs from the most parsimonious phylogenetic GLMM (Table S4). Estimates that do not overlap with 1 (dashed line) are displayed in black. Reference levels for the odds ratios include bats sampled at LAR, females, reproductive bats, absence of ectoparasites, and bats sampled in 2017. (B) Infection prevalence and 95% confidence intervals (Wilson method) stratified by sex. Results are shown for the full dataset and after randomly subsampling &lt;em&gt;Desmodus rotundus&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F2.medium.gif" width="440" height="252"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F2.medium.gif" width="440" height="252"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-26" class="first-child">Predictors of individual bat hemoplasma infection status. (A) Odds ratios and 95% HDIs from the most parsimonious phylogenetic GLMM (Table S4). Estimates that do not overlap with 1 (dashed line) are displayed in black. Reference levels for the odds ratios include bats sampled at LAR, females, reproductive bats, absence of ectoparasites, and bats sampled in 2017. (B) Infection prevalence and 95% confidence intervals (Wilson method) stratified by sex. Results are shown for the full dataset and after randomly subsampling <em>Desmodus rotundus</em>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Inter-species variation in hemoplasma prevalence</h3><p id="p-27">Across bat species, hemoplasma prevalence ranged from 0% to 100% (x =0.37). We estimated Pagel’s λ in logit-transformed prevalence to be 0.39, indicating moderate phylogenetic signal. Similarly, phylogenetic factorization identified one bat clade with significantly lower prevalence compared to the paraphyletic remainder: the Emballonuridae (12% infected; <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a>). Our trait-based analysis showed that relatively larger species (<em>β</em> =1.48, <em>p</em>=0.01, <em>R<sup>2</sup></em>=0.24) and those with larger colonies (<em>β</em>=0.67, <em>p</em>=0.06, <em>R<sup>22</sup></em>=0.20) had higher prevalence (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3B</a>; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Relatively heavier (≥20 g) and larger colony species included <em>Desmodus rotundus</em>, <em>Molossus rufus</em>, and <em>Pteronotus mesoamericanus</em>, for which prevalence was greater than 58%. Although these three species were also heavily sampled, other well-sampled species such as <em>Sturnira parvidens</em> and <em>Carollia sowelli</em> had lower prevalence, and sample size did not predict prevalence (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predictors of species-level hemoplasma prevalence across the Belize bat community. (A) Clades with significantly different prevalence are highlighted. (B) Results from the top PGLS models predicting prevalence as a function of mass and colony size. Model fit and 95% confidence intervals are shown overlaid with data scaled by sample size; species from the clade identified through phylogenetic factorization are colored as in panel A. Species identified by phylogenetic factorization (Saccopteryx bilineata and Rhynchonycteris naso) and with larger body mass, colony size, and hemoplasma prevalence (Desmodus rotundus, Molossus nigricans, and Pteronotus mesoamericanus) are shown to the right (photographs by Brock Fenton)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1607181521" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Predictors of species-level hemoplasma prevalence across the Belize bat community. (A) Clades with significantly different prevalence are highlighted. (B) Results from the top PGLS models predicting prevalence as a function of mass and colony size. Model fit and 95% confidence intervals are shown overlaid with data scaled by sample size; species from the clade identified through phylogenetic factorization are colored as in panel A. Species identified by phylogenetic factorization (&lt;em&gt;Saccopteryx bilineata&lt;/em&gt; and &lt;em&gt;Rhynchonycteris naso&lt;/em&gt;) and with larger body mass, colony size, and hemoplasma prevalence (&lt;em&gt;Desmodus rotundus&lt;/em&gt;, &lt;em&gt;Molossus nigricans&lt;/em&gt;, and &lt;em&gt;Pteronotus mesoamericanus&lt;/em&gt;) are shown to the right (photographs by Brock Fenton).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F3.medium.gif" width="424" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F3.medium.gif" width="424" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-28" class="first-child">Predictors of species-level hemoplasma prevalence across the Belize bat community. (A) Clades with significantly different prevalence are highlighted. (B) Results from the top PGLS models predicting prevalence as a function of mass and colony size. Model fit and 95% confidence intervals are shown overlaid with data scaled by sample size; species from the clade identified through phylogenetic factorization are colored as in panel A. Species identified by phylogenetic factorization (<em>Saccopteryx bilineata</em> and <em>Rhynchonycteris naso</em>) and with larger body mass, colony size, and hemoplasma prevalence (<em>Desmodus rotundus</em>, <em>Molossus nigricans</em>, and <em>Pteronotus mesoamericanus</em>) are shown to the right (photographs by Brock Fenton).</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1204947/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1204947/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1204947" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-29" class="first-child">Competing weighted phylogenetic generalized least squares models predicting hemoplasma infection prevalence (logit-transformed) across the Belize bat community. Models are ranked by ΔAICc with the number of coefficients (<em>k</em>), Akaike weights (<em>w<sub>i</sub></em>), and a likelihood ratio test pseudo-<em>R<sup>2</sup></em>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Hemoplasma genotype diversity</h3><p id="p-30">Our phylogenetic analysis identified 29 <em>Mycoplasma</em> genotypes in the Belize bat community (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), including three previously identified from vampire bats (VBG1–3; <a id="xref-ref-94-6" class="xref-bibr" href="#ref-94">Volokhov et al., 2017</a>). All genotypes demonstrated minor levels of sequence variability (99.5–100%; <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Based on comparisons with sequences from GenBank, 21 of these genotypes represent novel hemoplasmas, five are closely related to non-hemoplasma mycoplasmas, and many are phylogenetically related to previously identified <em>Mycoplasma</em> spp. and hemoplasmas identified from other bat species (e.g., <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Di Cataldo et al., 2020</a>; <a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">Millán et al., 2019</a>, <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">2015</a>), bat ticks (e.g., <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Hornok et al., 2019</a>), primates including humans (e.g., <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alcorn et al., 2020</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bonato, Figueiredo, Gonçalves, Machado, &amp; André, 2015</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Hattori et al., 2020</a>), and rodents (e.g., <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Gonçalves et al., 2015</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Goto, Yasuda, Hayashimoto, &amp; Ebukuro, 2010</a>; <a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Vieira et al., 2009</a>). A more detailed description of these 29 bacterial genotypes is provided in the Supporting Information (Fig. S3).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1204944/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1204944/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-31" class="first-child">Hemoplasma genotypes identified from the Belize bat community. Genotypes are given with their bat host species, representative GenBank numbers, and intra-genotype variability.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">After controlling for multiple comparisons, our 29 bacterial genotypes were associated with site (χ<em><sup>2</sup></em>=47.11, <em>p</em>&lt;0.01) and year (χ<em><sup>2</sup></em>=40.40, <em>p</em>&lt;0.01). Genotype composition was more diverse at LAR (Fig. S4), and KK hemoplasmas were dominated by vampire bat genotypes (VBG1–3). Genotype composition was more idiosyncratic by study year. However, these 29 bacterial genotypes were most strongly associated with bat species (χ<em><sup>2</sup></em>=3532, <em>p</em>&lt;0.01; Fig. S4).</p></div><div id="sec-16" class="subsection"><h3>Bat–hemoplasma evolutionary relationships</h3><p id="p-33">Although some hemoplasma genotypes were shared between bat species (i.e., VBG1, CS2, PPM, EF1, AH1–2, MYE, PLU, SP; <em>n</em>=9), most showed strong host specificity (<em>n</em>=20; <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Our coevolutionary analysis (PACo) supported congruence between the bat and hemoplasma phylogenies (<em>m</em><sup>2</sup><sub>XY</sub>=36.21, <em>p</em>=0.02, <em>n=</em>1000; <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>), suggesting that hemoplasma evolution has mostly tracked bat speciation. However, PACo also demonstrated that only 56% of the 41 unique bat–hemoplasma links displayed significant evidence of coevolution (Fig. S5), and these patterns were almost exclusively found within the Phyllostomidae (with the exception of <em>Saccopteryx bilineata</em> and its <em>Mycoplasma muris</em>–like bacterial genotype). The other 18 bat–hemoplasma links therefore displayed evidence of phylogenetic incongruence and thus likely host shifts.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evolutionary relationships between Belize bats and hemoplasma genotypes. The cophylogeny plot shows the bat phylogeny on the left and the hemoplasma genotype phylogeny on the right. We used the treespace package to collapse our complete hemoplasma phylogeny (Fig. S3) to only the 29 bacterial genotypes (Jombart, Kendall, Almagro Garcia, &amp; Colijn, 2017). Lines display bat&#x2013;hemoplasma associations and are shaded by the inverse of the squared residuals from PACo (i.e., dark lines show small residuals more indicative of coevolution)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1607181521" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Evolutionary relationships between Belize bats and hemoplasma genotypes. The cophylogeny plot shows the bat phylogeny on the left and the hemoplasma genotype phylogeny on the right. We used the &lt;em&gt;treespace&lt;/em&gt; package to collapse our complete hemoplasma phylogeny (Fig. S3) to only the 29 bacterial genotypes (Jombart, Kendall, Almagro Garcia, &amp; Colijn, 2017). Lines display bat&#x2013;hemoplasma associations and are shaded by the inverse of the squared residuals from PACo (i.e., dark lines show small residuals more indicative of coevolution).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F4.medium.gif" width="440" height="340"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F4.medium.gif" width="440" height="340"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-34" class="first-child">Evolutionary relationships between Belize bats and hemoplasma genotypes. The cophylogeny plot shows the bat phylogeny on the left and the hemoplasma genotype phylogeny on the right. We used the <em>treespace</em> package to collapse our complete hemoplasma phylogeny (Fig. S3) to only the 29 bacterial genotypes (Jombart, Kendall, Almagro Garcia, &amp; Colijn, 2017). Lines display bat–hemoplasma associations and are shaded by the inverse of the squared residuals from PACo (i.e., dark lines show small residuals more indicative of coevolution).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Hemoplasma genotype sharing networks</h3><p id="p-35">Within our bat–hemoplasma network, genotype sharing was restricted to five host communities, whereas six genotypes were each restricted to a single bat species (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5A</a>). GLMs showed that both the frequency and presence of genotype sharing declined with phylogenetic distance between bat species (Poisson: <em>p</em>&lt;0.001, <em>R<sup>2</sup></em>=0.08; binomial: <em>p</em>&lt;0.001, <em>R<sup>2</sup></em>=0.51; <a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5B</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Patterns of hemoplasma genotype sharing across the Belize bat community. (A) Nodes in the genotype network represent bat species (abbreviated by Latin binomials), and edges represent a shared genotype. Nodes are colored by communities identified with the Louvain method and are scaled by the number of individuals per species. (B) The matrix shows pairwise hemoplasma genotype sharing, colored by the number of genotypes shared between bat species." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1607181521" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Patterns of hemoplasma genotype sharing across the Belize bat community. (A) Nodes in the genotype network represent bat species (abbreviated by Latin binomials), and edges represent a shared genotype. Nodes are colored by communities identified with the Louvain method and are scaled by the number of individuals per species. (B) The matrix shows pairwise hemoplasma genotype sharing, colored by the number of genotypes shared between bat species.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F5.medium.gif" width="440" height="194"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F5.medium.gif" width="440" height="194"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <p id="p-36" class="first-child">Patterns of hemoplasma genotype sharing across the Belize bat community. (A) Nodes in the genotype network represent bat species (abbreviated by Latin binomials), and edges represent a shared genotype. Nodes are colored by communities identified with the Louvain method and are scaled by the number of individuals per species. (B) The matrix shows pairwise hemoplasma genotype sharing, colored by the number of genotypes shared between bat species.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Bat species shared hemoplasma genotypes with zero to five other species (i.e., degree), and most hosts were not central to the network of genotype sharing (i.e., eigenvector centrality of zero). Six bat species had non-zero eigenvector centrality values that ranged from 37% to 100%, indicating that these hosts generally shared more hemoplasma genotypes with other highly connected hosts. Stratifying our hemoplasma genotype network across sites and years showed that centrality measures varied by space but not time (Fig. S7, Table S5). We observed no hemoplasma genotype sharing at KK, likely reflecting lower host diversity (<a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Herrera et al., 2018</a>).</p><p id="p-38">Phylogenetic factorization identified similar bat clades with significantly different centrality compared to the paraphyletic remainder (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6A–B</a>). For degree, the algorithm only identified <em>Artibeus lituratus</em> as being more central (x=5) than other bats (x=1.14). However, phylogenetic factorization identified three taxa in the subfamily Stenodermatinae that had significantly elevated eigenvector centrality: the genera <em>Artibeus</em> and <em>Dermanura</em> (x=0.67 compared to x=0.02 in all other bats), the species <em>Artibeus lituratus</em> (x=1 compared to x=0.12), and the species <em>Sturnira parvidens</em> (x=0.37 compared to x=0.15). Mirroring these results, phylogenetic signal was absent for degree (λ=0) but high for eigenvector centrality (λ=0.93).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic patterns in hemoplasma genotype networks for Belize bat species (A) degree and (B) eigenvector centrality. Clades showing significantly different centrality metrics are highlighted, and points are scaled by observed values. Results from the top PGLS models predicting both centrality metrics as a function of bat species traits (C&#x2013;D). Model fit and 95% confidence intervals are shown overlaid with data scaled by sample size; species from the clades identified through phylogenetic factorization are colored as in panels A&#x2013;B." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1607181521" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Phylogenetic patterns in hemoplasma genotype networks for Belize bat species (A) degree and (B) eigenvector centrality. Clades showing significantly different centrality metrics are highlighted, and points are scaled by observed values. Results from the top PGLS models predicting both centrality metrics as a function of bat species traits (C&#x2013;D). Model fit and 95% confidence intervals are shown overlaid with data scaled by sample size; species from the clades identified through phylogenetic factorization are colored as in panels A&#x2013;B.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F6.medium.gif" width="416" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F6.medium.gif" width="416" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/18/2019.12.21.885921/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <p id="p-39" class="first-child">Phylogenetic patterns in hemoplasma genotype networks for Belize bat species (A) degree and (B) eigenvector centrality. Clades showing significantly different centrality metrics are highlighted, and points are scaled by observed values. Results from the top PGLS models predicting both centrality metrics as a function of bat species traits (C–D). Model fit and 95% confidence intervals are shown overlaid with data scaled by sample size; species from the clades identified through phylogenetic factorization are colored as in panels A–B.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">Trait-based analyses showed that degree centrality was best predicted by diet (Table S6); bat species feeding more heavily on fruit and nectar shared more bacterial genotypes with other species (<em>β</em>=0.004, <em>p</em>&lt;0.001, <em>R<sup>2</sup></em>=0.20; <a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6C</a>). Similarly, eigenvector centrality was best predicted by bat colony size and diet (Table S7); highly central species had small colonies (<em>β</em><sub>large</sub>=–1.93, <em>p</em>=0.05, <em>R<sup>2</sup></em>=0.13) and fed more on plants (<em>β</em>=0.03, <em>p</em>&lt;0.01, <em>R<sup>2</sup></em>=0.10; <a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. 6D</a>).</p><p id="p-41">As a final analysis, we assessed whether network centrality (i.e., a bat species’ role in hemoplasma genotype sharing) predicted contemporary infection prevalence (Fig. S8). However, we found no associations between species-level infection prevalence and centrality as measured by degree (<em>β</em> =–0.13, <em>R<sup>2</sup></em>=0.03, <em>p=</em>0.42) or eigenvector centrality (<em>β</em>=–0.20, <em>R<sup>2</sup></em>&lt;0.01, <em>p</em>=0.79).</p></div></div><div class="section" id="sec-18"><h2 class="">Discussion</h2><p id="p-42">By examining the prevalence and distribution of a common bacterial pathogen (hemoplasmas) in a diverse bat community, we expanded analysis of the ecological and evolutionary predictors of bat infection and pathogen sharing beyond viruses. Across the bat community, hemoplasma infection risk was weakly higher for males but was better predicted by phylogeny, with large-bodied and large-colony bat species showing greater prevalence. Hemoplasmas showed high diversity and mostly strict host associations, with strong congruence between the bat and hemoplasma phylogenies. Although codivergence was supported by our analyses, we also observed hemoplasma genotype sharing and evidence of historical host shifts between closely related bats. Species most central to this hemoplasma sharing network displayed taxonomic clustering and were disproportionately frugivores and nectarivores. Yet these highly central bat species did not also have the highest hemoplasma prevalence, reinforcing mostly infrequent bacterial sharing between species. Our work reveals phylogenetic patterns in hemoplasma infection in a diverse bat community while contributing to broader efforts to understand the host specificity of bacterial pathogens and their cross-species transmission patterns in wildlife.</p><p id="p-43">Whereas many bacterial pathogens, including hemoplasmas, are common in bats (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bai et al., 2011</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Becker et al., 2018</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Ikeda et al., 2017</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Mascarelli et al., 2014</a>; <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Millán et al., 2015</a>; <a id="xref-ref-94-7" class="xref-bibr" href="#ref-94">Volokhov et al., 2017</a>), the factors that confer high infection probability are poorly understood. In the Belize bat community, the odds of hemoplasma infection tended to be higher in males, suggesting male-biased transmission as detected in feline and canine systems (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Soto et al., 2017</a>; <a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Walker Vergara et al., 2016</a>). Such patterns could stem from males mounting weaker immune responses than females (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Kelly, Stoehr, Nunn, Smyth, &amp; Prokop, 2018</a>) or to male defense of multi-female roosts in many Neotropical bat species (<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Voigt, von Helversen, Michener, &amp; Kunz, 2001</a>). Direct transmission of hemoplasmas has been demonstrated in feline and rodent systems (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Cohen et al., 2018</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Museux et al., 2009</a>) but only inferred in bats from metagenomic studies detecting these bacteria in saliva (<a id="xref-ref-94-8" class="xref-bibr" href="#ref-94">Volokhov et al., 2017</a>). We found weak support for the hypothesis that ectoparasites play a role in infection risks (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Hornok et al., 2019</a>; Willi, Boretti, Meli, et al., 2007). The weak male bias in infection could cast further doubt on vector-borne transmission, as females in some bat species have elevated ectoparasitism (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Frank, Mendenhall, Judson, Daily, &amp; Hadly, 2016</a>); however, a secondary GLMM testing this hypothesis found generally weak support for a female bias in ectoparasitism in our system (Fig. S9). Future work assessing ectoparasite burdens could better elucidate the roles of vectors in hemoplasma risk.</p><p id="p-44">Across Neotropical bats sampled in Belize, we found phylogeny to be a better predictor of hemoplasma risk than individual traits, site, or year. Phylogenetic factorization identified one clade, the Emballonuridae, with significantly lower prevalence than all other bats in the community. This moderate phylogenetic signal mirrors comparable effects of phylogeny for bat viruses (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Guy, Thiagavel, Mideo, &amp; Ratcliffe, 2019</a>), similarly suggesting potential for innate differences in species susceptibility or pathogen exposure. Trait-based analyses revealed that this taxonomic pattern was driven by heavier and large-colony species having greater hemoplasma prevalence. Small-bodied species could have low prevalence due to small blood volumes and low bacterial titers (<a id="xref-ref-94-9" class="xref-bibr" href="#ref-94">Volokhov et al., 2017</a>). Alternatively, the positive, saturating relationship between body mass and bacterial prevalence could be driven by allometric patterns in competence (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Downs et al., 2019</a>), in contrast to weak or opposite relationships between mass and viral richness across bats (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Guy et al., 2019</a>; Han, Schmidt, et al., 2016). As larger-bodied bat species can also be more abundant in Neotropical habitat fragments (<a id="xref-ref-45-6" class="xref-bibr" href="#ref-45">Herrera et al., 2018</a>), these results suggest land conversion could increase the frequency of bat species most capable of maintaining hemoplasma infection. Similarly, positive relationships between colony size and prevalence could support density-dependent transmission of bacteria (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">McCallum et al., 2001</a>), whereas mixed support has been found for bat viruses (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Streicker et al., 2012</a>; <a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Webber, Fletcher, &amp; Willis, 2017</a>). Future work could test how community-wide infection patterns vary across broader habitat gradients and use multiple bacteria to assess the generality of these trends.</p><p id="p-45">Approximately two-thirds of the Neotropical bat species sampled in Belize were infected by hemoplasmas, for which we observed high genetic diversity consistent with other studies of this pathogen in bats (<a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Mascarelli et al., 2014</a>; <a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">Millán et al., 2015</a>; <a id="xref-ref-94-10" class="xref-bibr" href="#ref-94">Volokhov et al., 2017</a>). However, these bacterial genotypes were mostly novel and only weakly related to hemoplasmas described elsewhere in Latin America (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Ikeda et al., 2017</a>; <a id="xref-ref-70-4" class="xref-bibr" href="#ref-70">Millán et al., 2019</a>), with the exception of those previously identified from vampire bats (<a id="xref-ref-94-11" class="xref-bibr" href="#ref-94">Volokhov et al., 2017</a>). When considering the phylogenetic scale of genotypes, most hemoplasmas were host specific. Over half our hemoplasma communities consisted of a single bat–genotype association, matching the degree of host specificity observed more generally for <em>Mycoplasma</em> spp. (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Citti &amp; Blanchard, 2013</a>; <a id="xref-ref-79-3" class="xref-bibr" href="#ref-79">Pitcher &amp; Nicholas, 2005</a>). When we did detect genotype sharing between species, this occurred mostly between closely related hosts (e.g., PPM was detected in <em>Pteronotus mesoamericanus</em> and <em>P. fulvus</em>), indicating bat phylogenetic distance decreased the probability of bacterial transfer.</p><p id="p-46">Analyses to characterize species centrality to the hemoplasma genotype sharing network showed that one species (<em>Artibeus lituratus</em>) and the subfamily Stenodermatinae played key roles. This clade, and especially the genera <em>Artibeus</em> and <em>Dermanura</em> (formerly all classified in <em>Artibeus</em>), was the only taxon with non-zero connectivity, and this pattern was reflected in fruit-and nectar-based diets and small colonies being the primary predictors of centrality. The strictly frugivorous Stenodermatinae represents a recent divergence in the Phyllostomidae (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Botero-Castro et al., 2013</a>), and high centrality of these species may indicate weaker phylogenetic barriers for bacterial transmission between hosts in this clade. Two other analyses reinforced infrequent and conserved hemoplasma sharing between species. First, phylogenetic patterns in prevalence were distinct from those in genotype sharing centrality (e.g., large-colony species had higher prevalence but lower connectivity), and prevalence accordingly did not predict centrality. Second, we found general congruence between bat and hemoplasma phylogenies. Although this shows codivergence is a strong evolutionary force, congruence can also stem from preferential jumps to closely related hosts (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">De Vienne et al., 2013</a>). Though we cannot rule out that some host shifts may be artefacts of the limited resolution of both the phylogenies, our evolutionary analyses and genotype sharing results imply that hemoplasma host shifts are possible yet rare.</p><p id="p-47">By sampling a diverse assemblage of bacterial genotypes in an ecologically and evolutionary rich host community, our work has broader implications for our understanding of disease emergence. Many bacterial pathogens are thought to be generalists and relatively unlikely to specialize in a novel host (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Pedersen, Altizer, Poss, Cunningham, &amp; Nunn, 2005</a>; <a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">Woolhouse &amp; Gowtage-Sequeria, 2005</a>), in contrast to many viruses in which host shifts are more common owing to high mutation rates and short infectious periods (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Geoghegan et al., 2017</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Longdon, Brockhurst, Russell, Welch, &amp; Jiggins, 2014</a>). Recent theoretical work suggests host shift speciation may be less common for bacteria because of higher phenotypic plasticity (e.g., the ability to reside in diverse habitats) and a slower tempo of evolution (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Bonneaud et al., 2019</a>). Obligate reliance of <em>Mycoplasma</em> spp. on host cells and more chronic infections likely explains their propensity to specialize (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Citti &amp; Blanchard, 2013</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Cohen et al., 2018</a>). More broadly, however, using genetics to infer pathogen sharing, rather than coarser phylogenetic scales (e.g., species complexes or genera), is increasingly showing that many bacterial strains may be more host specific (<a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">Withenshaw, Devevey, Pedersen, &amp; Fenton, 2016</a>). The high specialism of bat hemoplasma genotypes thus underscores the importance of using finer phylogenetic scales in the study of infectious disease (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Fountain-Jones et al., 2018</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Graham, Storch, &amp; Machac, 2018</a>).</p><p id="p-48">Comparative analyses of viruses have suggested that phylogenetically conserved pathogen jumps between species may be a broader generality in the study of disease emergence (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albery, Eskew, Ross, &amp; Olival, 2019</a>; <a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Luis et al., 2015</a>; <a id="xref-ref-86-2" class="xref-bibr" href="#ref-86">Streicker et al., 2010</a>). With few exceptions, our results on hemoplasma genotype sharing between Neotropical bat species are generally consistent with this pattern for a bacterial pathogen. Two cases in which hemoplasmas were shared between more distantly related species included the VBG1 genotype in <em>Desmodus rotundus</em> and <em>Pteronotus fulvus</em> (Phyllostomidae and Mormoopidae) and the EF1 genotype in <em>Glossophaga soricina</em> and <em>Saccopteryx bilineata</em> (Phyllostomidae and Emballonuridae). For the latter, both bat species co-roost in the LAR, which suggests an ecological context for pathogen exposure over current timescales. However, other genetic markers (e.g., <em>rpoB, rpoC, gyrB</em>) would be necessary to infer contemporary cross-species transmission (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Kämpfer &amp; Glaeser, 2012</a>; <a id="xref-ref-98-3" class="xref-bibr" href="#ref-98">Volokhov et al., 2012</a>), as analysis of the 16s rRNA gene alone is insufficient for hemoplasma species identification (<a id="xref-ref-98-4" class="xref-bibr" href="#ref-98">Volokhov et al., 2012</a>). If hemoplasmas are more likely to specialize rather than expand their range into new and unrelated species, genotype sharing between unrelated bats could represent more transient spillovers (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Bonneaud et al., 2019</a>). As specialized pathogens could be more transmissible than generalists (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Garamszegi, 2006</a>), species with high infection prevalence of specialist genotypes could be prioritized for bacterial surveillance.</p><p id="p-49">In conclusion, our analysis of a diverse community of bats and their pathogen genotypes identifies several key ecological and evolutionary factors structuring bacterial infection within and between species and provides a starting point for contrasts with such patterns for viruses. Similar to bat viruses, we found moderate phylogenetic signal in hemoplasma prevalence. However, these phylogenetic patterns in prevalence were decoupled from those describing bat species centrality in sharing hemoplasmas, such that genotype sharing was generally restricted by bat phylogeny. These findings imply codivergence of bats and their bacterial pathogens alongside rare and phylogenetically constrained host shifts. Future work more broadly characterizing the ecological and evolutionary determinants of bacterial infections in diverse host communities will improve our understanding of cross-species transmission beyond viruses and contribute to efforts to understand the epidemiological consequences of bacterial pathogens.</p></div><div class="section" id="sec-20"><h2 class="">Competing interests</h2><p id="p-50">We have no competing interests.</p></div><div class="section" id="sec-21"><h2 class="">Data accessibility</h2><p id="p-51">Individual-level data are available in Dryad (<a href="https://datadryad.org/stash/share/7qOGkejyIldAcD9FEZHNNVCc_4k0lmmiABy0EiRwZoo">https://datadryad.org/stash/share/7qOGkejyIldAcD9FEZHNNVCc_4k0lmmiABy0EiRwZoo</a>). Bat species trait data are available in Table S2. Hemoplasma sequences are available in GenBank (<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MH245119&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom">MH245119</a>–<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MH245194&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom">MH245194</a> and <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MK353807&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom">MK353807</a>–<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MK353892&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom">MK353892</a>).</p></div><div class="section" id="sec-22"><h2 class="">Funding</h2><p id="p-52">DJB was funded by the ARCS Foundation, American Society of Mammalogists, Odum School of Ecology, Explorer’s Club, and NSF DEB-1601052. RKP was supported by NSF DEB-1716698, the Defense Advanced Research Projects Agency (DARPA D16AP00113 and the DARPA PREEMPT program Cooperative Agreement D18AC00031), U.S. National Institutes of General Medical Sciences IDeA Program (P20GM103474 and P30GM110732), and the USDA National Institute of Food and Agriculture (Hatch project 1015891). DJB and KAS were both supported by the American Museum of Natural History Theodore Roosevelt Memorial Fund. NBS was supported by the American Museum of Natural History Taxonomic Mammalogy Fund. DGS was supported by a Sir Henry Dale Fellowship, jointly funded by the Wellcome Trust and Royal Society (102507/Z/13/Z). The views, opinions, and/or findings expressed are those of the authors and should not be interpreted as representing the official views or policies of the Department of Defense or the U.S. Government.</p></div><div class="section" id="sec-23"><h2 class="">Statement of authorship</h2><p id="p-54">DJB, KAS, and AMB collected samples; NBS and MBF coordinated fieldwork; RKP, SA, and DGS supported laboratory analyses; DVV and VEC conducted molecular and phylogenetic analyses; and ADW and DGS assisted with statistical analyses. DJB analyzed data, produced figures, and wrote the manuscript. All authors provided critical review of the manuscript.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-53">For assistance with field logistics, bat sampling, and permits, we thank Mark Howells, Neil Duncan, James Herrera, Melissa Ingala, and staff of the Lamanai Field Research Center. We also thank the many colleagues who helped net bats during 2017–2018 bat research in Belize. We thank Seth Walk, Susan Broadway, and Jonathan Martinson for laboratory access and assistance. Lastly, we thank two anonymous reviewers for helpful suggestions to improve this manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">§</span> Currently retired</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Albery</span>,  <span class="cit-name-given-names">G. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eskew</span>,  <span class="cit-name-given-names">E. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ross</span>,  <span class="cit-name-given-names">N.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Olival</span>,  <span class="cit-name-given-names">K. J.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Predicting the global mammalian viral sharing network using phylogeography</span>. <abbr class="cit-jnl-abbrev">BioRxiv</abbr>, <span class="cit-vol">732255</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.2" data-doi="10.1093/cid/ciaa089"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alcorn</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gerrard</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cochrane</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jennison</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Irwin</span>,  <span class="cit-name-given-names">P. J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Barbosa</span>,  <span class="cit-name-given-names">A. D.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">First report of Candidatus Mycoplasma haemohominis infection in Australia causing persistent fever in an animal carer</span>. <abbr class="cit-jnl-abbrev">Clinical Infectious Diseases: An Official Publication of the Infectious Diseases Society of America</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/cid/ciaa089</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BInfectious%2BDiseases%253A%2BAn%2BOfficial%2BPublication%2Bof%2Bthe%2BInfectious%2BDiseases%2BSociety%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcid%252Fciaa089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/cid/ciaa089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.3" data-doi="10.3201/eid1707.101867"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bai</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kosoy</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Recuenco</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alvarez</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moran</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Turmelle</span>,  <span class="cit-name-given-names">A.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Rupprecht</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Bartonella spp. in Bats, Guatemala</span>. <abbr class="cit-jnl-abbrev">Emerging Infectious Diseases</abbr>, <span class="cit-vol">17</span>(<span class="cit-issue">7</span>), <span class="cit-fpage">1269</span>–<span class="cit-lpage">1272</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.3201/eid1707.101867</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DBai%26rft.auinit1%253DY.%26rft.volume%253D17%26rft.issue%253D7%26rft.spage%253D1269%26rft.epage%253D1272%26rft.atitle%253DBartonella%2Bspp.%2Bin%2BBats%252C%2BGuatemala.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1707.101867%26rft_id%253Dinfo%253Apmid%252F21762584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1707.101867&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21762584&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.21.885921v2.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baillie</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hilton-Taylor</span>,  <span class="cit-name-given-names">C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Stuart</span>,  <span class="cit-name-given-names">S. N.</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-source">IUCN red list of threatened species: a global species assessment</span>. <span class="cit-publ-name">IUCN</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.5" data-doi="10.1371/journal.pone.0061048"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Balbuena</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Míguez-Lozano</span>,  <span class="cit-name-given-names">R.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Blasco-Costa</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">PACo: A Novel Procrustes Application to Cophylogenetic Analysis</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr>, <span class="cit-vol">8</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">e61048</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0061048</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D8%26rft.spage%253De61048%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0061048%26rft_id%253Dinfo%253Apmid%252F23580325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0061048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23580325&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.6" data-doi="10.1371/journal.pntd.0006786"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Becker</span>,  <span class="cit-name-given-names">D. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bergner</span>,  <span class="cit-name-given-names">L. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bentz</span>,  <span class="cit-name-given-names">A. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orton</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Altizer</span>,  <span class="cit-name-given-names">S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Streicker</span>,  <span class="cit-name-given-names">D. G.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Genetic diversity, infection prevalence, and possible transmission routes of Bartonella spp. in vampire bats</span>. <abbr class="cit-jnl-abbrev">PLOS Neglected Tropical Diseases</abbr>, <span class="cit-vol">12</span>(<span class="cit-issue">9</span>), <span class="cit-fpage">e0006786</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pntd.0006786</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BNeglected%2BTropical%2BDiseases%26rft.volume%253D12%26rft.spage%253D0006786e%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pntd.0006786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pntd.0006786&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.7" data-doi="10.1098/rsbl.2019.0423"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Becker</span>,  <span class="cit-name-given-names">D. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crowley</span>,  <span class="cit-name-given-names">D. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Washburne</span>,  <span class="cit-name-given-names">A. D.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Plowright</span>,  <span class="cit-name-given-names">R. K.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Temporal and spatial limitations in global surveillance for bat filoviruses and henipaviruses</span>. <abbr class="cit-jnl-abbrev">Biology Letters</abbr>, <span class="cit-vol">15</span>(<span class="cit-issue">12</span>), <span class="cit-fpage">20190423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1098/rsbl.2019.0423</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiology%2BLetters%26rft.volume%253D15%26rft.spage%253D20190423%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2019.0423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsbl.2019.0423&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.8" data-doi="10.1016/S0378-8733(98)00010-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bell</span>,  <span class="cit-name-given-names">D. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atkinson</span>,  <span class="cit-name-given-names">J. S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Carlson</span>,  <span class="cit-name-given-names">J. W.</span></span> (<span class="cit-pub-date">1999</span>). <span class="cit-article-title">Centrality measures for disease transmission networks</span>. <abbr class="cit-jnl-abbrev">Social Networks</abbr>, <span class="cit-vol">21</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0378-8733(98)00010-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSocial%2BNetworks%26rft.volume%253D21%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-8733%252898%252900010-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-8733(98)00010-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.9" data-doi="10.1111/ele.12300"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blyton</span>,  <span class="cit-name-given-names">M. D. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Banks</span>,  <span class="cit-name-given-names">S. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peakall</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lindenmayer</span>,  <span class="cit-name-given-names">D. B.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Gordon</span>,  <span class="cit-name-given-names">D. M.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Not all types of host contacts are equal when it comes to E. coli transmission</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, <span class="cit-vol">17</span>(<span class="cit-issue">8</span>), <span class="cit-fpage">970</span>–<span class="cit-lpage">978</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/ele.12300</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D17%26rft.spage%253D970%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12300%26rft_id%253Dinfo%253Apmid%252F24861219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24861219&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.10" data-doi="10.1016/j.cimid.2015.09.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bonato</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Figueiredo</span>,  <span class="cit-name-given-names">M. A. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonçalves</span>,  <span class="cit-name-given-names">L. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Machado</span>,  <span class="cit-name-given-names">R. Z.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">André</span>,  <span class="cit-name-given-names">M. R.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Occurrence and molecular characterization of Bartonella spp. and hemoplasmas in neotropical primates from Brazilian Amazon</span>. <abbr class="cit-jnl-abbrev">Comparative Immunology, Microbiology and Infectious Diseases</abbr>, <span class="cit-vol">42</span>, <span class="cit-fpage">15</span>–<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cimid.2015.09.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComparative%2BImmunology%252C%2BMicrobiology%2Band%2BInfectious%2BDiseases%26rft.volume%253D42%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cimid.2015.09.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cimid.2015.09.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.11" data-doi="10.1098/rstb.2018.0328"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bonneaud</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weinert</span>,  <span class="cit-name-given-names">L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Kuijper</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Understanding the emergence of bacterial pathogens in novel hosts</span>. <abbr class="cit-jnl-abbrev">Phil. Trans. R. Soc. B</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1098/rstb.2018.0328</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhil.%2BTrans.%2BR.%2BSoc.%2BB%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2018.0328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2018.0328&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.12" data-doi="10.1016/j.ympev.2013.07.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Botero-Castro</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tilak</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Justy</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Catzeflis</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delsuc</span>,  <span class="cit-name-given-names">F.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Douzery</span>,  <span class="cit-name-given-names">E. J. P.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Next-generation sequencing and phylogenetic signal of complete mitochondrial genomes for resolving the evolutionary history of leaf-nosed bats (Phyllostomidae)</span>. <abbr class="cit-jnl-abbrev">Molecular Phylogenetics and Evolution</abbr>, <span class="cit-vol">69</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">728</span>–<span class="cit-lpage">739</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.ympev.2013.07.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BPhylogenetics%2Band%2BEvolution%26rft.volume%253D69%26rft.spage%253D728%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2013.07.003%26rft_id%253Dinfo%253Apmid%252F23850499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2013.07.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23850499&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bürkner</span>,  <span class="cit-name-given-names">P.-C.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">brms: An R package for Bayesian multilevel models using Stan</span>. <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr>, <span class="cit-vol">80</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D80%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.21.885921v2.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Burnham</span>,  <span class="cit-name-given-names">K. P.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">D. R.</span></span> (<span class="cit-pub-date">2002</span>). <span class="cit-source">Model selection and multimodel inference: a practical information-theoretic approach</span>. <span class="cit-publ-name">Springer Science &amp; Business Media</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Butts</span>,  <span class="cit-name-given-names">C. T.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Social network analysis with sna</span>. <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr>, <span class="cit-vol">24</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D24%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>,  <span class="cit-name-given-names">S. A.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Szöcs</span>,  <span class="cit-name-given-names">E.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">taxize: taxonomic search and retrieval in R</span>. <abbr class="cit-jnl-abbrev">F1000Research</abbr>, <span class="cit-vol">2</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.17" data-doi="10.1016/j.tim.2013.01.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Citti</span>,  <span class="cit-name-given-names">C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Blanchard</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Mycoplasmas and their host: emerging and re-emerging minimal pathogens</span>. <abbr class="cit-jnl-abbrev">Trends in Microbiology</abbr>, <span class="cit-vol">21</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">196</span>–<span class="cit-lpage">203</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.tim.2013.01.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BMicrobiology%26rft.volume%253D21%26rft.spage%253D196%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2013.01.003%26rft_id%253Dinfo%253Apmid%252F23419218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2013.01.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23419218&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.18" data-doi="10.1111/j.1471-8286.2006.01657.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Clare</span>,  <span class="cit-name-given-names">E. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lim</span>,  <span class="cit-name-given-names">B. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Engstrom</span>,  <span class="cit-name-given-names">M. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eger</span>,  <span class="cit-name-given-names">J. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Hebert</span>,  <span class="cit-name-given-names">P. D.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">DNA barcoding of Neotropical bats: species identification and discovery within Guyana</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology Notes</abbr>, <span class="cit-vol">7</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">184</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2BNotes%26rft.volume%253D7%26rft.spage%253D184%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2006.01657.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2006.01657.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.19" data-doi="10.1111/mec.14826"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cohen</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shemesh</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garrido</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Messika</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Einav</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khokhlova</span>,  <span class="cit-name-given-names">I.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Hawlena</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Haemoplasmas in wild rodents: Routes of transmission and infection dynamics</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">27</span>(<span class="cit-issue">18</span>), <span class="cit-fpage">3714</span>–<span class="cit-lpage">3726</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/mec.14826</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D27%26rft.spage%253D3714%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14826&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.20" data-doi="10.3724/SP.J.1087.2009.02191"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Csardi</span>,  <span class="cit-name-given-names">G.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Nepusz</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">The igraph software package for complex network research</span>. <abbr class="cit-jnl-abbrev">InterJournal, Complex Systems</abbr>, <span class="cit-vol">1695</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInterJournal%252C%2BComplex%2BSystems%26rft.volume%253D1695%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3724%252FSP.J.1087.2009.02191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3724/SP.J.1087.2009.02191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dallas</span>,  <span class="cit-name-given-names">T. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">B. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nunn</span>,  <span class="cit-name-given-names">C. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">A. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>,  <span class="cit-name-given-names">P. R.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Drake</span>,  <span class="cit-name-given-names">J. M.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Host traits associated with species roles in parasite sharing networks</span>. <abbr class="cit-jnl-abbrev">Oikos</abbr>, <span class="cit-vol">128</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">23</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOikos%26rft.volume%253D128%26rft.spage%253D23%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.22" data-doi="10.1111/nph.12150"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De Vienne</span>,  <span class="cit-name-given-names">D. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Refrégier</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">López-Villavicencio</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tellier</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hood</span>,  <span class="cit-name-given-names">M. E.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Giraud</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Cospeciation vs host-shift speciation: methods for testing, evidence from natural associations and relation to coevolution</span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr>, <span class="cit-vol">198</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">347</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.volume%253D198%26rft.spage%253D347%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12150%26rft_id%253Dinfo%253Apmid%252F23437795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23437795&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316629500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.23" data-doi="10.1016/j.cimid.2020.101448"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Di Cataldo</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kamani</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cevidanes</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Msheliza</span>,  <span class="cit-name-given-names">E. G.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Millán</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Hemotropic mycoplasmas in bats captured near human settlements in Nigeria</span>. <abbr class="cit-jnl-abbrev">Comparative Immunology, Microbiology and Infectious Diseases</abbr>, <span class="cit-vol">70</span>, <span class="cit-fpage">101448</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cimid.2020.101448</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComparative%2BImmunology%252C%2BMicrobiology%2Band%2BInfectious%2BDiseases%26rft.volume%253D70%26rft.spage%253D101448%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cimid.2020.101448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cimid.2020.101448&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.24" data-doi="10.1016/j.pt.2018.12.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Downs</span>,  <span class="cit-name-given-names">C. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schoenle</span>,  <span class="cit-name-given-names">L. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">B. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrison</span>,  <span class="cit-name-given-names">J. F.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">L. B.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Scaling of Host Competence</span>. <abbr class="cit-jnl-abbrev">Trends in Parasitology</abbr>, <span class="cit-vol">0</span>(<span class="cit-fpage">0</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.pt.2018.12.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BParasitology%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pt.2018.12.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pt.2018.12.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.25" data-doi="10.1093/bioinformatics/btr381"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haas</span>,  <span class="cit-name-given-names">B. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clemente</span>,  <span class="cit-name-given-names">J. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quince</span>,  <span class="cit-name-given-names">C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Knight</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">UCHIME improves sensitivity and speed of chimera detection</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">27</span>(<span class="cit-issue">16</span>), <span class="cit-fpage">2194</span>–<span class="cit-lpage">2200</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btr381</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr381%26rft_id%253Dinfo%253Apmid%252F21700674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr381&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21700674&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293620800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.26" data-doi="10.3201/eid1508.090475"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Evans</span>,  <span class="cit-name-given-names">N. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bown</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Timofte</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simpson</span>,  <span class="cit-name-given-names">V. R.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Birtles</span>,  <span class="cit-name-given-names">R. J.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Fatal Borreliosis in Bat Caused by Relapsing Fever Spirochete, United Kingdom</span>. <abbr class="cit-jnl-abbrev">Emerging Infectious Diseases</abbr>, <span class="cit-vol">15</span>(<span class="cit-issue">8</span>), <span class="cit-fpage">1331</span>–<span class="cit-lpage">1333</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.3201/eid1508.090475</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DEvans%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D15%26rft.issue%253D8%26rft.spage%253D1331%26rft.epage%253D1333%26rft.atitle%253DFatal%2Bborreliosis%2Bin%2Bbat%2Bcaused%2Bby%2Brelapsing%2Bfever%2Bspirochete%252C%2BUnited%2BKingdom.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1508.090475%26rft_id%253Dinfo%253Apmid%252F19751613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1508.090475&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19751613&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.27" data-doi="10.1086/683173"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fenton</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Streicker</span>,  <span class="cit-name-given-names">D. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petchey</span>,  <span class="cit-name-given-names">O. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Pedersen</span>,  <span class="cit-name-given-names">A. B.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Are All Hosts Created Equal? Partitioning Host Species Contributions to Parasite Persistence in Multihost Communities</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">186</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">610</span>–<span class="cit-lpage">622</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1086/683173</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D186%26rft.spage%253D610%26rft_id%253Dinfo%253Adoi%252F10.1086%252F683173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/683173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fenton</span>,  <span class="cit-name-given-names">M. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bernard</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bouchard</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hollis</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnston</span>,  <span class="cit-name-given-names">D. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lausen</span>,  <span class="cit-name-given-names">C. L.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Zigouris</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">The bat fauna of Lamanai, Belize: roosts and trophic roles</span>. <abbr class="cit-jnl-abbrev">Journal of Tropical Ecology</abbr>, <span class="cit-vol">17</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">511</span>–<span class="cit-lpage">524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTropical%2BEcology%26rft.volume%253D17%26rft.spage%253D511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.29" data-doi="10.1111/brv.12380"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fountain-Jones</span>,  <span class="cit-name-given-names">N. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pearse</span>,  <span class="cit-name-given-names">W. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Escobar</span>,  <span class="cit-name-given-names">L. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alba-Casals</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carver</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davies</span>,  <span class="cit-name-given-names">T. J.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Worsley-Tonks</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Towards an eco-phylogenetic framework for infectious disease ecology</span>. <abbr class="cit-jnl-abbrev">Biological Reviews</abbr>, <span class="cit-vol">93</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">950</span>–<span class="cit-lpage">970</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BReviews%26rft.volume%253D93%26rft.spage%253D950%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fbrv.12380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/brv.12380&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.30" data-doi="10.1002/ece3.2245"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frank</span>,  <span class="cit-name-given-names">H. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mendenhall</span>,  <span class="cit-name-given-names">C. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Judson</span>,  <span class="cit-name-given-names">S. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daily</span>,  <span class="cit-name-given-names">G. C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Hadly</span>,  <span class="cit-name-given-names">E. A.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Anthropogenic impacts on Costa Rican bat parasitism are sex specific</span>. <abbr class="cit-jnl-abbrev">Ecology and Evolution</abbr>, <span class="cit-vol">6</span>(<span class="cit-issue">14</span>), <span class="cit-fpage">4898</span>–<span class="cit-lpage">4909</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1002/ece3.2245</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2Band%2BEvolution%26rft.volume%253D6%26rft.spage%253D4898%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fece3.2245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ece3.2245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.31" data-doi="10.1111/j.1461-0248.2006.00936.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garamszegi</span>,  <span class="cit-name-given-names">L. Z.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">The evolution of virulence and host specialization in malaria parasites of primates</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, <span class="cit-vol">9</span>(<span class="cit-issue">8</span>), <span class="cit-fpage">933</span>–<span class="cit-lpage">940</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1461-0248.2006.00936.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DGaramszegi%26rft.auinit1%253DL.%2BZ.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D933%26rft.epage%253D940%26rft.atitle%253DThe%2Bevolution%2Bof%2Bvirulence%2Band%2Bhost%2Bspecialization%2Bin%2Bmalaria%2Bparasites%2Bof%2Bprimates.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2006.00936.x%26rft_id%253Dinfo%253Apmid%252F16913936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2006.00936.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16913936&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239510700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.21.885921v2.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garamszegi</span>,  <span class="cit-name-given-names">L. Z.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-source">Modern Phylogenetic Comparative Methods and Their Application in Evolutionary Biology: Concepts and Practice</span>. <span class="cit-publ-name">Springer</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.33" data-doi="10.1080/00031305.2018.1549100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gelman</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goodrich</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gabry</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Vehtari</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">R-squared for Bayesian Regression Models</span>. <abbr class="cit-jnl-abbrev">The American Statistician</abbr>, <span class="cit-vol">73</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">307</span>–<span class="cit-lpage">309</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1080/00031305.2018.1549100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BStatistician%26rft.volume%253D73%26rft.spage%253D307%26rft_id%253Dinfo%253Adoi%252F10.1080%252F00031305.2018.1549100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/00031305.2018.1549100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.34" data-doi="10.1371/journal.ppat.1006215"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Geoghegan</span>,  <span class="cit-name-given-names">J. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duchêne</span>,  <span class="cit-name-given-names">S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E. C.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Comparative analysis estimates the relative frequencies of co-divergence and cross-species transmission within viral families</span>. <abbr class="cit-jnl-abbrev">PLOS Pathogens</abbr>, <span class="cit-vol">13</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">e1006215</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1006215</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BPathogens%26rft.volume%253D13%26rft.spage%253De1006215%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006215%26rft_id%253Dinfo%253Apmid%252F28178344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28178344&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.35" data-doi="10.1073/pnas.1220716110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gómez</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nunn</span>,  <span class="cit-name-given-names">C. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Verdú</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Centrality in primate–parasite networks reveals the potential for the transmission of emerging infectious diseases to humans</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>, <span class="cit-vol">110</span>(<span class="cit-issue">19</span>), <span class="cit-fpage">7738</span>–<span class="cit-lpage">7741</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1220716110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1220716110%26rft_id%253Dinfo%253Apmid%252F23610389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzE5Lzc3MzgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xOC8yMDE5LjEyLjIxLjg4NTkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.36" data-doi="10.1016/j.cimid.2015.10.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gonçalves</span>,  <span class="cit-name-given-names">L. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roque</span>,  <span class="cit-name-given-names">A. L. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matos</span>,  <span class="cit-name-given-names">C. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fernandes</span>,  <span class="cit-name-given-names">S.</span></span> de J., <span class="cit-auth"><span class="cit-name-surname">Olmos</span>,  <span class="cit-name-given-names">I. D. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Machado</span>,  <span class="cit-name-given-names">R. Z.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">André</span>,  <span class="cit-name-given-names">M. R.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Diversity and molecular characterization of novel hemoplasmas infecting wild rodents from different Brazilian biomes</span>. <abbr class="cit-jnl-abbrev">Comparative Immunology, Microbiology and Infectious Diseases</abbr>, <span class="cit-vol">43</span>, <span class="cit-fpage">50</span>–<span class="cit-lpage">56</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cimid.2015.10.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComparative%2BImmunology%252C%2BMicrobiology%2Band%2BInfectious%2BDiseases%26rft.volume%253D43%26rft.spage%253D50%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cimid.2015.10.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cimid.2015.10.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.37" data-doi="10.7550/rmb.38023"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">González-Salazar</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martínez-Meyer</span>,  <span class="cit-name-given-names">E.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">López-Santiago</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">A hierarchical classification of trophic guilds for North American birds and mammals</span>. <abbr class="cit-jnl-abbrev">Revista Mexicana de Biodiversidad</abbr>, <span class="cit-vol">85</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">931</span>–<span class="cit-lpage">941</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.7550/rmb.38023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRevista%2BMexicana%2Bde%2BBiodiversidad%26rft.volume%253D85%26rft.spage%253D931%26rft_id%253Dinfo%253Adoi%252F10.7550%252Frmb.38023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7550/rmb.38023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goto</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yasuda</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayashimoto</span>,  <span class="cit-name-given-names">N.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Ebukuro</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Morphological and sequence analysis of Mycoplasma sp. isolated from the oral cavity of a house musk shrew (Suncus murinus)</span>. <abbr class="cit-jnl-abbrev">The Journal of Veterinary Medical Science</abbr>, <span class="cit-vol">72</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">109</span>–<span class="cit-lpage">111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BVeterinary%2BMedical%2BScience%26rft.volume%253D72%26rft.spage%253D109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.39" data-doi="10.1111/geb.12686"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">C. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Storch</span>,  <span class="cit-name-given-names">D.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Machac</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Phylogenetic scale in ecology and evolution</span>. <abbr class="cit-jnl-abbrev">Global Ecology and Biogeography</abbr>, <span class="cit-vol">27</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">175</span>–<span class="cit-lpage">187</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/geb.12686</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BEcology%2Band%2BBiogeography%26rft.volume%253D27%26rft.spage%253D175%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgeb.12686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/geb.12686&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.21.885921v2.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gunnell</span>,  <span class="cit-name-given-names">G. F.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Simmons</span>,  <span class="cit-name-given-names">N. B.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-source">Evolutionary history of bats: fossils, molecules and morphology</span>. <span class="cit-publ-name">Cambridge University Press</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guy</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thiagavel</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mideo</span>,  <span class="cit-name-given-names">N.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Ratcliffe</span>,  <span class="cit-name-given-names">J. M.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Phylogeny matters: revisiting ‘a comparison of bats and rodents as reservoirs of zoonotic viruses.’</span> <abbr class="cit-jnl-abbrev">Royal Society Open Science</abbr>, <span class="cit-vol">6</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">181182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRoyal%2BSociety%2BOpen%2BScience%26rft.volume%253D6%26rft.spage%253D181182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">B. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kramer</span>,  <span class="cit-name-given-names">A. M.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Drake</span>,  <span class="cit-name-given-names">J. M.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Global Patterns of Zoonotic Disease in Mammals</span>. <abbr class="cit-jnl-abbrev">Trends in Parasitology</abbr>, <span class="cit-vol">32</span>(<span class="cit-issue">7</span>), <span class="cit-fpage">565</span>–<span class="cit-lpage">577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BParasitology%26rft.volume%253D32%26rft.spage%253D565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">43.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.21.885921v2.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">B. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmidt</span>,  <span class="cit-name-given-names">J. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alexander</span>,  <span class="cit-name-given-names">L. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bowden</span>,  <span class="cit-name-given-names">S. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayman</span>,  <span class="cit-name-given-names">D. T.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Drake</span>,  <span class="cit-name-given-names">J. M.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Undiscovered bat hosts of filoviruses</span>. <abbr class="cit-jnl-abbrev">PLOS Negl Trop Dis</abbr>, <span class="cit-vol">10</span>(<span class="cit-issue">7</span>), <span class="cit-fpage">e0004815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BNegl%2BTrop%2BDis%26rft.volume%253D10%26rft.spage%253De0004815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.44" data-doi="10.3201/eid2601.190983"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hattori</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuroda</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Katano</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takuma</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ito</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arai</span>,  <span class="cit-name-given-names">N.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Niki</span>,  <span class="cit-name-given-names">Y.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Candidatus Mycoplasma haemohominis in Human, Japan</span>. <abbr class="cit-jnl-abbrev">Emerging Infectious Diseases</abbr>, <span class="cit-vol">26</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">11</span>–<span class="cit-lpage">19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.3201/eid2601.190983</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2BInfectious%2BDiseases%26rft.volume%253D26%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2601.190983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2601.190983&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.45" data-doi="10.3161/15081109ACC2018.20.1.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Herrera</span>,  <span class="cit-name-given-names">J. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duncan</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clare</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fenton</span>,  <span class="cit-name-given-names">M. B.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Simmons</span>,  <span class="cit-name-given-names">N.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Disassembly of Fragmented Bat Communities in Orange Walk District, Belize</span>. <abbr class="cit-jnl-abbrev">Acta Chiropterologica</abbr>, <span class="cit-vol">20</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">147</span>–<span class="cit-lpage">159</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.3161/15081109ACC2018.20.1.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BChiropterologica%26rft.volume%253D20%26rft.spage%253D147%26rft_id%253Dinfo%253Adoi%252F10.3161%252F15081109ACC2018.20.1.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3161/15081109ACC2018.20.1.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vennes</span>,  <span class="cit-name-given-names">C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">M. R. J.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Package ‘geosphere.’</span><abbr class="cit-jnl-abbrev">Spherical Trigonometryv</abbr>, <span class="cit-fpage">1</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.47" data-doi="10.1073/pnas.080551197"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hochachka</span>,  <span class="cit-name-given-names">W. M.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Dhondt</span>,  <span class="cit-name-given-names">A. A.</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">Density-dependent decline of host abundance resulting from a new infectious disease</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>, <span class="cit-vol">97</span>(<span class="cit-issue">10</span>), <span class="cit-fpage">5303</span>–<span class="cit-lpage">5306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.080551197%26rft_id%253Dinfo%253Apmid%252F10792031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTcvMTAvNTMwMyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzE4LzIwMTkuMTIuMjEuODg1OTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.48" data-doi="10.1186/s13071-019-3303-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hornok</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Szoke</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meli</span>,  <span class="cit-name-given-names">M. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sándor</span>,  <span class="cit-name-given-names">A. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Görföl</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Estók</span>,  <span class="cit-name-given-names">P.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Hofmann-Lehmann</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Molecular detection of vector-borne bacteria in bat ticks (Acari: Ixodidae</span>, <abbr class="cit-jnl-abbrev">Argasidae) from eight countries of the Old and New Worlds. Parasites &amp; Vectors</abbr>, <span class="cit-vol">12</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">50</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1186/s13071-019-3303-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArgasidae%2529%2Bfrom%2Beight%2Bcountries%2Bof%2Bthe%2BOld%2Band%2BNew%2BWorlds.%2BParasites%2B%2526%2BVectors%26rft.volume%253D12%26rft.spage%253D50%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13071-019-3303-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13071-019-3303-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bininda-Emonds</span>,  <span class="cit-name-given-names">O. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>,  <span class="cit-name-given-names">P. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gittleman</span>,  <span class="cit-name-given-names">J. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Altizer</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Phylogenetically related and ecologically similar carnivores harbour similar parasite assemblages</span>. <abbr class="cit-jnl-abbrev">Journal of Animal Ecology</abbr>, <span class="cit-vol">83</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">671</span>–<span class="cit-lpage">680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BEcology%26rft.volume%253D83%26rft.spage%253D671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.50" data-doi="10.1111/evo.12611"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drake</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gittleman</span>,  <span class="cit-name-given-names">J. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Altizer</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Parasite diversity declines with host evolutionary distinctiveness: A global analysis of carnivores</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">69</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">621</span>–<span class="cit-lpage">630</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/evo.12611</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D69%26rft.spage%253D621%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12611&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hutchinson</span>,  <span class="cit-name-given-names">M. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cagua</span>,  <span class="cit-name-given-names">E. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balbuena</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stouffer</span>,  <span class="cit-name-given-names">D. B.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Poisot</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">paco: implementing Procrustean Approach to Cophylogeny in R</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr>, <span class="cit-vol">8</span>(<span class="cit-issue">8</span>), <span class="cit-fpage">932</span>–<span class="cit-lpage">940</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D8%26rft.spage%253D932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ikeda</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seki</span>,  <span class="cit-name-given-names">M. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carrasco</span>,  <span class="cit-name-given-names">A. O. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rudiak</span>,  <span class="cit-name-given-names">L. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miranda</span>,  <span class="cit-name-given-names">J. M. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonçalves</span>,  <span class="cit-name-given-names">S. M. M.</span></span>, … others. (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Evidence and molecular characterization of Bartonella spp</span>. <abbr class="cit-jnl-abbrev">and hemoplasmas in neotropical bats in Brazil. Epidemiology &amp; Infection</abbr>, <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.53" data-doi="10.1111/j.1461-0248.2011.01730.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">P. T. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rohr</span>,  <span class="cit-name-given-names">J. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoverman</span>,  <span class="cit-name-given-names">J. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kellermanns</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bowerman</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Lunde</span>,  <span class="cit-name-given-names">K. B.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Living fast and dying of infection: host life history drives interspecific variation in infection and disease risk</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, <span class="cit-vol">15</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">235</span>–<span class="cit-lpage">242</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1461-0248.2011.01730.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DJohnson%26rft.auinit1%253DP.%2BT.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D235%26rft.epage%253D242%26rft.atitle%253DLiving%2Bfast%2Band%2Bdying%2Bof%2Binfection%253A%2Bhost%2Blife%2Bhistory%2Bdrives%2Binterspecific%2Bvariation%2Bin%2Binfection%2Band%2Bdisease%2Brisk.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2011.01730.x%26rft_id%253Dinfo%253Apmid%252F22221837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2011.01730.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22221837&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">54.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.21.885921v2.54" data-doi="10.1111/1755-0998.12676"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jombart</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kendall</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">AlmagrouGarcia</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Colijn</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">treespace: Statistical exploration of landscapes of phylogenetic trees</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology Resources</abbr>, <span class="cit-vol">17</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">1385</span>– <span class="cit-lpage">1392</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/1755-0998.12676</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2BResources%26rft.volume%253D17%26rft.spage%253D1385%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12676&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.55"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">P. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hämsch</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Page</span>,  <span class="cit-name-given-names">R. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalko</span>,  <span class="cit-name-given-names">E. K.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">O’mara</span>,  <span class="cit-name-given-names">M. T.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Foraging and roosting behaviour of the fringe-lipped bat, Trachops cirrhosus, on Barro Colorado Island, Panamá</span>. <abbr class="cit-jnl-abbrev">Acta Chiropterologica</abbr>, <span class="cit-vol">19</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">337</span>–<span class="cit-lpage">346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BChiropterologica%26rft.volume%253D19%26rft.spage%253D337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.56" data-doi="10.1111/j.1462-2920.2011.02615.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kämpfer</span>,  <span class="cit-name-given-names">P.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Glaeser</span>,  <span class="cit-name-given-names">S. P.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Prokaryotic taxonomy in the sequencing era–the polyphasic approach revisited</span>. <abbr class="cit-jnl-abbrev">Environmental Microbiology</abbr>, <span class="cit-vol">14</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">291</span>–<span class="cit-lpage">317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DKampfer%26rft.auinit1%253DP.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D291%26rft.epage%253D317%26rft.atitle%253DProkaryotic%2Btaxonomy%2Bin%2Bthe%2Bsequencing%2Bera--the%2Bpolyphasic%2Bapproach%2Brevisited.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1462-2920.2011.02615.x%26rft_id%253Dinfo%253Apmid%252F22040009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1462-2920.2011.02615.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22040009&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.57" data-doi="10.1002/ece3.4451"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kellner</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carver</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scorza</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McKee</span>,  <span class="cit-name-given-names">C. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lappin</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crooks</span>,  <span class="cit-name-given-names">K. R.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Antolin</span>,  <span class="cit-name-given-names">M. F.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Transmission pathways and spillover of an erythrocytic bacterial pathogen from domestic cats to wild felids</span>. <abbr class="cit-jnl-abbrev">Ecology and Evolution</abbr>, <span class="cit-vol">8</span>(<span class="cit-issue">19</span>), <span class="cit-fpage">9779</span>–<span class="cit-lpage">9792</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1002/ece3.4451</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2Band%2BEvolution%26rft.volume%253D8%26rft.spage%253D9779%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fece3.4451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ece3.4451&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.58" data-doi="10.1111/ele.13164"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelly</span>,  <span class="cit-name-given-names">C. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stoehr</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nunn</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smyth</span>,  <span class="cit-name-given-names">K. N.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Prokop</span>,  <span class="cit-name-given-names">Z. M.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Sexual dimorphism in immunity across animals: a meta-analysis</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, <span class="cit-vol">21</span>(<span class="cit-issue">12</span>), <span class="cit-fpage">1885</span>– <span class="cit-lpage">1894</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/ele.13164</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D21%26rft.spage%253D1885%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.13164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.13164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.59" data-doi="10.1093/bioinformatics/btq166"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kembel</span>,  <span class="cit-name-given-names">S. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cowan</span>,  <span class="cit-name-given-names">P. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Helmus</span>,  <span class="cit-name-given-names">M. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cornwell</span>,  <span class="cit-name-given-names">W. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morlon</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ackerly</span>,  <span class="cit-name-given-names">D. D.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Webb</span>,  <span class="cit-name-given-names">C. O.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Picante: R tools for integrating phylogenies and ecology</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">26</span>(<span class="cit-issue">11</span>), <span class="cit-fpage">1463</span>–<span class="cit-lpage">1464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq166%26rft_id%253Dinfo%253Apmid%252F20395285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20395285&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277985500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.60" data-doi="10.1093/molbev/msy096"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stecher</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knyaz</span>,  <span class="cit-name-given-names">C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Tamura</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">MEGA X: Molecular Evolutionary Genetics Analysis across Computing Platforms</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>, <span class="cit-vol">35</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">1547</span>–<span class="cit-lpage">1549</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/msy096</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.volume%253D35%26rft.spage%253D1547%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy096%26rft_id%253Dinfo%253Apmid%252F29722887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msy096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29722887&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.61" data-doi="10.1111/j.1744-7429.2008.00422.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Loayza</span>,  <span class="cit-name-given-names">A. P.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Loiselle</span>,  <span class="cit-name-given-names">B. A.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Preliminary information on the home range and movement patterns of Sturnira lilium (Phyllostomidae) in a naturally fragmented landscape in Bolivia</span>. <abbr class="cit-jnl-abbrev">Biotropica</abbr>, <span class="cit-vol">40</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">630</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotropica%26rft.volume%253D40%26rft.spage%253D630%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7429.2008.00422.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7429.2008.00422.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000258581100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.62" data-doi="10.1371/journal.ppat.1004395"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Longdon</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brockhurst</span>,  <span class="cit-name-given-names">M. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Russell</span>,  <span class="cit-name-given-names">C. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Welch</span>,  <span class="cit-name-given-names">J. J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Jiggins</span>,  <span class="cit-name-given-names">F. M.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">The Evolution and Genetics of Virus Host Shifts</span>. <abbr class="cit-jnl-abbrev">PLOS Pathogens</abbr>, <span class="cit-vol">10</span>(<span class="cit-issue">11</span>), <span class="cit-fpage">e1004395</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1004395</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BPathogens%26rft.volume%253D10%26rft.spage%253De1004395%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1004395%26rft_id%253Dinfo%253Apmid%252F25375777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1004395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25375777&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.63" data-doi="10.1111/ele.12491"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luis</span>,  <span class="cit-name-given-names">A. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Shea</span>,  <span class="cit-name-given-names">T. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayman</span>,  <span class="cit-name-given-names">D. T. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wood</span>,  <span class="cit-name-given-names">J. L. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cunningham</span>,  <span class="cit-name-given-names">A. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilbert</span>,  <span class="cit-name-given-names">A. T.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Webb</span>,  <span class="cit-name-given-names">C. T.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Network analysis of host–virus communities in bats and rodents reveals determinants of cross-species transmission</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, <span class="cit-vol">18</span>(<span class="cit-issue">11</span>), <span class="cit-fpage">1153</span>–<span class="cit-lpage">1162</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/ele.12491</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D18%26rft.spage%253D1153%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.64" data-doi="10.1128/JCM.01125-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maggi</span>,  <span class="cit-name-given-names">R. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Compton</span>,  <span class="cit-name-given-names">S. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trull</span>,  <span class="cit-name-given-names">C. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mascarelli</span>,  <span class="cit-name-given-names">P. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mozayeni</span>,  <span class="cit-name-given-names">B. R.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Breitschwerdt</span>,  <span class="cit-name-given-names">E. B.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Infection with hemotropic Mycoplasma species in patients with or without extensive arthropod or animal contact</span>. <abbr class="cit-jnl-abbrev">Journal of Clinical Microbiology</abbr>, <span class="cit-vol">51</span>(<span class="cit-issue">10</span>), <span class="cit-fpage">3237</span>–<span class="cit-lpage">3241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.01125-13%26rft_id%253Dinfo%253Apmid%252F23863574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjEwOiI1MS8xMC8zMjM3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTgvMjAxOS4xMi4yMS44ODU5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.65"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mascarelli</span>,  <span class="cit-name-given-names">P. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keel</span>,  <span class="cit-name-given-names">M. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yabsley</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Last</span>,  <span class="cit-name-given-names">L. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Breitschwerdt</span>,  <span class="cit-name-given-names">E. B.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Maggi</span>,  <span class="cit-name-given-names">R. G.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Hemotropic mycoplasmas in little brown bats (Myotis lucifugus)</span>. <abbr class="cit-jnl-abbrev">Parasit Vectors</abbr>, <span class="cit-vol">7</span>, <span class="cit-fpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DParasit%2BVectors%26rft.volume%253D7%26rft.spage%253D117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.66" data-doi="10.1016/S0169-5347(01)02144-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McCallum</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barlow</span>,  <span class="cit-name-given-names">N.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Hone</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">How should pathogen transmission be modelled?</span> <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr>, <span class="cit-vol">16</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">295</span>–<span class="cit-lpage">300</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0169-5347(01)02144-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DMcCallum%26rft.auinit1%253DH.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D295%26rft.epage%253D300%26rft.atitle%253DHow%2Bshould%2Bpathogen%2Btransmission%2Bbe%2Bmodelled%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252801%252902144-9%26rft_id%253Dinfo%253Apmid%252F11369107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-5347(01)02144-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11369107&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168721800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.67"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McKee</span>,  <span class="cit-name-given-names">C. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krawczyk</span>,  <span class="cit-name-given-names">A. I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sándor</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Görföl</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Földvári</span>,  <span class="cit-name-given-names">M.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Földvári</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Host phylogeny, geographic overlap, and roost sharing shape parasite communities in European bats</span>. <abbr class="cit-jnl-abbrev">Frontiers in Ecology and Evolution</abbr>, <span class="cit-vol">7</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.68" data-doi="10.1111/j.1939-165X.2004.tb00342.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Messick</span>,  <span class="cit-name-given-names">J. B.</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">Hemotrophic mycoplasmas (hemoplasmas): a review and new insights into pathogenic potential</span>. <abbr class="cit-jnl-abbrev">Veterinary Clinical Pathology</abbr>, <span class="cit-vol">33</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">2</span>–<span class="cit-lpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1939-165X.2004.tb00342.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bclinical%2Bpathology%2B%252F%2BAmerican%2BSociety%2Bfor%2BVeterinary%2BClinical%2BPathology%26rft.stitle%253DVet%2BClin%2BPathol%26rft.aulast%253DMessick%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D2%26rft.epage%253D13%26rft.atitle%253DHemotrophic%2Bmycoplasmas%2B%2528hemoplasmas%2529%253A%2Ba%2Breview%2Band%2Bnew%2Binsights%2Binto%2Bpathogenic%2Bpotential.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1939-165X.2004.tb00342.x%26rft_id%253Dinfo%253Apmid%252F15048620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1939-165X.2004.tb00342.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15048620&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220757300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.69" data-doi="10.1111/2041-210X.12593"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Michonneau</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">J. W.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Winter</span>,  <span class="cit-name-given-names">D. J.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">rotl : an R package to interact with the Open Tree of Life data</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/2041-210X.12593</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D7%26rft.spage%253D1476%26rft_id%253Dinfo%253Adoi%252F10.1111%252F2041-210X.12593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/2041-210X.12593&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.70" data-doi="10.7589/2018-12-290"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Millán</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cevidanes</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sacristán</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alvarado</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sepúlveda</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramos-Mella</span>,  <span class="cit-name-given-names">C. A.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Lisón</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Detection and Characterization of Hemotropic Mycoplasmas in Bats in Chile</span>. <abbr class="cit-jnl-abbrev">Journal of Wildlife Diseases</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.7589/2018-12-290</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BWildlife%2BDiseases%26rft_id%253Dinfo%253Adoi%252F10.7589%252F2018-12-290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7589/2018-12-290&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">71.</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.71" data-doi="10.1016/j.cimid.2015.01.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Millán</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">López-Roig</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delicado</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Serra-Cobo</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Esperón</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Widespread infection with hemotropic mycoplasmas in bats in Spain, including a hemoplasma closely related to “Candidatus Mycoplasma hemohominis.”</span> <abbr class="cit-jnl-abbrev">Comparative Immunology, Microbiology and Infectious Diseases</abbr>, <span class="cit-vol">39</span>, <span class="cit-fpage">9</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cimid.2015.01.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComparative%2BImmunology%252C%2BMicrobiology%2Band%2BInfectious%2BDiseases%26rft.volume%253D39%26rft.spage%253D9%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cimid.2015.01.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cimid.2015.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">72.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.72" data-doi="10.1186/1471-2148-"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Monteiro</span>,  <span class="cit-name-given-names">L. R.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Nogueira</span>,  <span class="cit-name-given-names">M. R.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Evolutionary patterns and processes in the radiation of phyllostomid bats</span>. <abbr class="cit-jnl-abbrev">BMC Evolutionary Biology</abbr>, <span class="cit-vol">11</span>, <span class="cit-fpage">137</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1186/1471-2148-</span> 11-137</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvolutionary%2BBiology%26rft.volume%253D11%26rft.spage%253D137%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">73.</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73. in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.73" data-doi="10.1111/j.1863-2378.2012.01536.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mühldorfer</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Bats and Bacterial Pathogens: A Review</span>. <abbr class="cit-jnl-abbrev">Zoonoses and Public Health</abbr>, <span class="cit-vol">60</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">93</span>–<span class="cit-lpage">103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1863-2378.2012.01536.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZoonoses%2Band%2BPublic%2BHealth%26rft.volume%253D60%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1863-2378.2012.01536.x%26rft_id%253Dinfo%253Apmid%252F22862791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1863-2378.2012.01536.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22862791&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000313487000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">74.</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.74" data-doi="10.1051/vetres/2009028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Museux</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boretti</span>,  <span class="cit-name-given-names">F. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willi</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Riond</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoelzle</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoelzle</span>,  <span class="cit-name-given-names">L. E.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Hofmann-Lehmann</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">In vivo transmission studies of “Candidatus Mycoplasma turicensis” in the domestic cat</span>. <abbr class="cit-jnl-abbrev">Veterinary Research</abbr>, <span class="cit-vol">40</span>(<span class="cit-issue">5</span>), 1, <span class="cit-fpage">45</span>–<span class="cit-lpage">14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1051/vetres/2009028</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bresearch%26rft.stitle%253DVet%2BRes%26rft.aulast%253DMuseux%26rft.auinit1%253DK.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D45%26rft.epage%253D45%26rft.atitle%253DIn%2Bvivo%2Btransmission%2Bstudies%2Bof%2B%2527Candidatus%2BMycoplasma%2Bturicensis%2527%2Bin%2Bthe%2Bdomestic%2Bcat.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fvetres%252F2009028%26rft_id%253Dinfo%253Apmid%252F19505421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/vetres/2009028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505421&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">75.</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.75" data-doi="10.1890/15-0846R.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Myhrvold</span>,  <span class="cit-name-given-names">N. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baldridge</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sivam</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Freeman</span>,  <span class="cit-name-given-names">D. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Ernest</span>,  <span class="cit-name-given-names">S. K. M.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">An amniote life-history database to perform comparative analyses with birds, mammals, and reptiles</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr>, <span class="cit-vol">96</span>(<span class="cit-issue">11</span>), <span class="cit-fpage">3109</span>–<span class="cit-lpage">3109</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1890/15-0846R.1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D96%26rft.spage%253D3109%26rft_id%253Dinfo%253Adoi%252F10.1890%252F15-0846R.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/15-0846R.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">76.</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76. in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.76" data-doi="10.1038/nature22975"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Olival</span>,  <span class="cit-name-given-names">K. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hosseini</span>,  <span class="cit-name-given-names">P. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zambrana-Torrelio</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ross</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bogich</span>,  <span class="cit-name-given-names">T. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Daszak</span>,  <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Host and viral traits predict zoonotic spillover from mammals</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">546</span>(<span class="cit-issue">7660</span>), <span class="cit-fpage">646</span>–<span class="cit-lpage">650</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature22975</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D546%26rft.spage%253D646%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22975%26rft_id%253Dinfo%253Apmid%252F28636590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature22975&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28636590&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">77.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.77" data-doi="10.1093/bioinformatics/btg412"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paradis</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Claude</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Strimmer</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">APE: analyses of phylogenetics and evolution in R language</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">20</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">289</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DParadis%26rft.auinit1%253DE.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D289%26rft.epage%253D290%26rft.atitle%253DAPE%253A%2BAnalyses%2Bof%2BPhylogenetics%2Band%2BEvolution%2Bin%2BR%2Blanguage%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg412%26rft_id%253Dinfo%253Apmid%252F14734327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg412&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14734327&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188389700026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">78.</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78. in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.78" data-doi="10.1016/j.ijpara.2005.01.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>,  <span class="cit-name-given-names">A. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Altizer</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poss</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cunningham</span>,  <span class="cit-name-given-names">A. A.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Nunn</span>,  <span class="cit-name-given-names">C. L.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Patterns of host specificity and transmission among parasites of wild primates</span>. <abbr class="cit-jnl-abbrev">International Journal for Parasitology</abbr>, <span class="cit-vol">35</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">647</span>–<span class="cit-lpage">657</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bfor%2Bparasitology%26rft.stitle%253DInt%2BJ%2BParasitol%26rft.aulast%253DPedersen%26rft.auinit1%253DA.%2BB.%26rft.volume%253D35%26rft.issue%253D6%26rft.spage%253D647%26rft.epage%253D657%26rft.atitle%253DPatterns%2Bof%2Bhost%2Bspecificity%2Band%2Btransmission%2Bamong%2Bparasites%2Bof%2Bwild%2Bprimates.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ijpara.2005.01.005%26rft_id%253Dinfo%253Apmid%252F15862578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ijpara.2005.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15862578&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229362100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">79.</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79. in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.79" data-doi="10.1016/j.tvjl.2004.08.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pitcher</span>,  <span class="cit-name-given-names">D. G.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Nicholas</span>,  <span class="cit-name-given-names">R. A. J.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Mycoplasma host specificity: Fact or fiction?</span> <abbr class="cit-jnl-abbrev">The Veterinary Journal</abbr>, <span class="cit-vol">170</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">300</span>–<span class="cit-lpage">306</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.tvjl.2004.08.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bjournal%2B%2528London%252C%2BEngland%2B%253A%2B%2B1997%2529%26rft.stitle%253DVet%2BJ%26rft.aulast%253DPitcher%26rft.auinit1%253DD.%2BG.%26rft.volume%253D170%26rft.issue%253D3%26rft.spage%253D300%26rft.epage%253D306%26rft.atitle%253DMycoplasma%2Bhost%2Bspecificity%253A%2Bfact%2Bor%2Bfiction%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tvjl.2004.08.011%26rft_id%253Dinfo%253Apmid%252F16266844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tvjl.2004.08.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16266844&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233471700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">80.</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80. in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.80" data-doi="10.1038/nrmicro.2017.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plowright</span>,  <span class="cit-name-given-names">R. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parrish</span>,  <span class="cit-name-given-names">C. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCallum</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hudson</span>,  <span class="cit-name-given-names">P. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ko</span>,  <span class="cit-name-given-names">A. I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">A. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Lloyd-Smith</span>,  <span class="cit-name-given-names">J. O.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Pathways to zoonotic spillover</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Microbiology</abbr>, <span class="cit-vol">15</span>(<span class="cit-issue">8</span>), <span class="cit-fpage">502</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BMicrobiology%26rft.volume%253D15%26rft.spage%253D502%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro.2017.45%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro.2017.45&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">81.</span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference 81. in text" id="ref-81">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.21.885921v2.81"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reid</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">1997</span>). <span class="cit-source">A Field Guide to the Mammals of Central America and Southeast Mexico</span>. <span class="cit-publ-name">OUP</span> <span class="cit-publ-loc">USA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">82.</span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference 82. in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.82" data-doi="10.1371/journal.pone.0025845"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Santana</span>,  <span class="cit-name-given-names">S. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dial</span>,  <span class="cit-name-given-names">T. O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eiting</span>,  <span class="cit-name-given-names">T. P.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Alfaro</span>,  <span class="cit-name-given-names">M. E.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Roosting Ecology and the Evolution of Pelage Markings in Bats</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr>, <span class="cit-vol">6</span>(<span class="cit-issue">10</span>), <span class="cit-fpage">e25845</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0025845</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSantana%26rft.auinit1%253DS.%2BE.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253De25845%26rft.epage%253De25845%26rft.atitle%253DRoosting%2Becology%2Band%2Bthe%2Bevolution%2Bof%2Bpelage%2Bmarkings%2Bin%2Bbats.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0025845%26rft_id%253Dinfo%253Apmid%252F21991371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0025845&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21991371&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">83.</span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference 83. in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.83" data-doi="10.1093/jmammal/gyw078"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sikes</span>,  <span class="cit-name-given-names">R. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Care</span>,  <span class="cit-name-given-names">A.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Mammalogists</span></span>, U. C. of the A. S. of. (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">2016 Guidelines of the American Society of Mammalogists for the use of wild mammals in research and education</span>. <abbr class="cit-jnl-abbrev">Journal of Mammalogy</abbr>, <span class="cit-vol">97</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">663</span>–<span class="cit-lpage">688</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMammalogy%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjmammal%252Fgyw078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jmammal/gyw078&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">84.</span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference 84. in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.84" data-doi="10.1016/j.cimid.2016.11.013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Soto</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Walker</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sepulveda</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bittencourt</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Acosta-Jamett</span>,  <span class="cit-name-given-names">G.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Müller</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Occurrence of canine hemotropic mycoplasmas in domestic dogs from urban and rural areas of the Valdivia Province, southern Chile</span>. <abbr class="cit-jnl-abbrev">Comparative Immunology, Microbiology and Infectious Diseases</abbr>, <span class="cit-vol">50</span>, <span class="cit-fpage">70</span>–<span class="cit-lpage">77</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cimid.2016.11.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComparative%2BImmunology%252C%2BMicrobiology%2Band%2BInfectious%2BDiseases%26rft.volume%253D50%26rft.spage%253D70%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cimid.2016.11.013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cimid.2016.11.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">85.</span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference 85. in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.85" data-doi="10.1098/rspb.2012.0538"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Streicker</span>,  <span class="cit-name-given-names">D. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Recuenco</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valderrama</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benavides</span>,  <span class="cit-name-given-names">J. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vargas</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pacheco</span>,  <span class="cit-name-given-names">V.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Altizer</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Ecological and anthropogenic drivers of rabies exposure in vampire bats: implications for transmission and control</span>. <abbr class="cit-jnl-abbrev">Proc. R. Soc. B</abbr>, <span class="cit-vol">279</span>(<span class="cit-issue">1742</span>), <span class="cit-fpage">3384</span>–<span class="cit-lpage">3392</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1098/rspb.2012.0538</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BR.%2BSoc.%2BB%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2012.0538%26rft_id%253Dinfo%253Apmid%252F22696521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2012.0538&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22696521&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">86.</span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference 86. in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.86" data-doi="10.1126/science.1188836"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Streicker</span>,  <span class="cit-name-given-names">D. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Turmelle</span>,  <span class="cit-name-given-names">A. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vonhof</span>,  <span class="cit-name-given-names">M. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuzmin</span>,  <span class="cit-name-given-names">I. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCracken</span>,  <span class="cit-name-given-names">G. F.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Rupprecht</span>,  <span class="cit-name-given-names">C. E.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Host phylogeny constrains cross-species emergence and establishment of rabies virus in bats</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">329</span>(<span class="cit-issue">5992</span>), <span class="cit-fpage">676</span>–<span class="cit-lpage">679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DStreicker%26rft.auinit1%253DD.%2BG.%26rft.volume%253D329%26rft.issue%253D5992%26rft.spage%253D676%26rft.epage%253D679%26rft.atitle%253DHost%2BPhylogeny%2BConstrains%2BCross-Species%2BEmergence%2Band%2BEstablishment%2Bof%2BRabies%2BVirus%2Bin%2BBats%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1188836%26rft_id%253Dinfo%253Apmid%252F20689015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjkvNTk5Mi82NzYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xOC8yMDE5LjEyLjIxLjg4NTkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">87.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.21.885921v2.87" data-doi="10.1139/z04-030"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ter Hofstede</span>,  <span class="cit-name-given-names">H. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fenton</span>,  <span class="cit-name-given-names">M. B.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Whitaker</span>,  <span class="cit-name-given-names">J. O.</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">Host and host-site specificity of bat flies (Diptera: Streblidae and Nycteribiidae) on Neotropical bats (Chiroptera)</span>. <abbr class="cit-jnl-abbrev">Canadian Journal of Zoology</abbr>, <span class="cit-vol">82</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">616</span>–<span class="cit-lpage">626</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCanadian%2BJournal%2Bof%2BZoology%26rft.volume%253D82%26rft.spage%253D616%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fz04-030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/z04-030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000222349900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">88.</span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference 88. in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.88" data-doi="10.1016/j.biocon.2013.11.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">VanderWaal</span>,  <span class="cit-name-given-names">K. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atwill</span>,  <span class="cit-name-given-names">E. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Isbell</span>,  <span class="cit-name-given-names">L. A.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">McCowan</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Quantifying microbe transmission networks for wild and domestic ungulates in Kenya</span>. <abbr class="cit-jnl-abbrev">Biological Conservation</abbr>, <span class="cit-vol">169</span>(<span class="cit-issue">Supplement C</span>), <span class="cit-fpage">136</span>–<span class="cit-lpage">146</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.biocon.2013.11.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BConservation%26rft.volume%253D169%26rft.spage%253D136%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biocon.2013.11.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biocon.2013.11.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000333574400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">89.</span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference 89. in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.89" data-doi="10.1111/1365-2435.12645"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">VanderWaal</span>,  <span class="cit-name-given-names">K. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Ezenwa</span>,  <span class="cit-name-given-names">V. O.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Heterogeneity in pathogen transmission: mechanisms and methodology</span>. <abbr class="cit-jnl-abbrev">Functional Ecology</abbr>, <span class="cit-vol">30</span>(<span class="cit-issue">10</span>), <span class="cit-fpage">1606</span>–<span class="cit-lpage">1622</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunctional%2BEcology%26rft.volume%253D30%26rft.spage%253D1606%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1365-2435.12645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1365-2435.12645&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">90.</span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference 90. in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.90"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vehtari</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gelman</span>,  <span class="cit-name-given-names">A.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Gabry</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Practical Bayesian model evaluation using leave-one-out cross-validation and WAIC</span>. <abbr class="cit-jnl-abbrev">Statistics and Computing</abbr>, <span class="cit-vol">27</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">1413</span>–<span class="cit-lpage">1432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStatistics%2Band%2BComputing%26rft.volume%253D27%26rft.spage%253D1413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">91.</span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference 91. in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.91"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Viechtbauer</span>,  <span class="cit-name-given-names">W.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Conducting meta-analyses in R with the metafor package</span>. <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr>, <span class="cit-vol">36</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D36%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">92.</span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference 92. in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.92" data-doi="10.1016/j.vetmic.2009.06.018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vieira</span>,  <span class="cit-name-given-names">R. F. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Molento</span>,  <span class="cit-name-given-names">M. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">dos Santos</span>,  <span class="cit-name-given-names">L. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moraes</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cubas</span>,  <span class="cit-name-given-names">Z. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Santos</span>,  <span class="cit-name-given-names">A. P.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Messick</span>,  <span class="cit-name-given-names">J. B.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Detection of a novel hemoplasma based on 16S rRNA gene DNA in captive and free-ranging capybaras (Hydrochaeris hydrochaeris)</span>. <abbr class="cit-jnl-abbrev">Veterinary Microbiology</abbr>, <span class="cit-vol">139</span>(<span class="cit-issue">3–4</span>), <span class="cit-fpage">410</span>–<span class="cit-lpage">413</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.vetmic.2009.06.018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bmicrobiology%26rft.stitle%253DVet%2BMicrobiol%26rft.aulast%253DVieira%26rft.auinit1%253DR.%2BF.%26rft.volume%253D139%26rft.issue%253D3-4%26rft.spage%253D410%26rft.epage%253D413%26rft.atitle%253DDetection%2Bof%2Ba%2Bnovel%2Bhemoplasma%2Bbased%2Bon%2B16S%2BrRNA%2Bgene%2BDNA%2Bin%2Bcaptive%2Band%2Bfree-ranging%2Bcapybaras%2B%2528Hydrochaeris%2Bhydrochaeris%2529.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vetmic.2009.06.018%26rft_id%253Dinfo%253Apmid%252F19592180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.vetmic.2009.06.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19592180&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">93.</span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference 93. in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.93" data-doi="10.1007/s002650100337"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Voigt</span>,  <span class="cit-name-given-names">C. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">von Helversen</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Michener</span>,  <span class="cit-name-given-names">R.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Kunz</span>,  <span class="cit-name-given-names">T. H.</span></span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">The economics of harem maintenance in the sac-winged bat, Saccopteryx bilineata (Emballonuridae)</span>. <abbr class="cit-jnl-abbrev">Behavioral Ecology and Sociobiology</abbr>, <span class="cit-vol">50</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">31</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehavioral%2BEcology%2Band%2BSociobiology%26rft.volume%253D50%26rft.spage%253D31%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002650100337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s002650100337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000169536400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">94.</span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference 94. in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.94"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Volokhov</span>,  <span class="cit-name-given-names">D. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becker</span>,  <span class="cit-name-given-names">D. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bergner</span>,  <span class="cit-name-given-names">L. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Camus</span>,  <span class="cit-name-given-names">M. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orton</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chizhikov</span>,  <span class="cit-name-given-names">V. E.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Streicker</span>,  <span class="cit-name-given-names">D. G.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Novel hemotropic mycoplasmas are widespread and genetically diverse in vampire bats</span>. <abbr class="cit-jnl-abbrev">Epidemiology &amp; Infection</abbr>, <span class="cit-vol">145</span>(<span class="cit-issue">15</span>), <span class="cit-fpage">3154</span>–<span class="cit-lpage">3167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEpidemiology%2B%2526%2BInfection%26rft.volume%253D145%26rft.spage%253D3154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">95.</span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference 95. in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.95" data-doi="10.1128/AEM.00211-17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Volokhov</span>,  <span class="cit-name-given-names">Dmitriy V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hwang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chizhikov</span>,  <span class="cit-name-given-names">V. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Danaceau</span>,  <span class="cit-name-given-names">H.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Gottdenker</span>,  <span class="cit-name-given-names">N. L.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Prevalence, Genotype Richness, and Coinfection Patterns of Hemotropic Mycoplasmas in Raccoons (Procyon lotor) on Environmentally Protected and Urbanized Barrier Islands</span>. <abbr class="cit-jnl-abbrev">Applied and Environmental Microbiology</abbr>, <span class="cit-vol">83</span>(<span class="cit-fpage">9</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1128/AEM.00211-17</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.00211-17%26rft_id%253Dinfo%253Apmid%252F28258139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjE0OiI4My85L2UwMDIxMS0xNyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzE4LzIwMTkuMTIuMjEuODg1OTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">96.</span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference 96. in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.96" data-doi="10.1128/AEM.00720-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Volokhov</span>,  <span class="cit-name-given-names">Dmitriy V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kong</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">George</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Chizhikov</span>,  <span class="cit-name-given-names">V. E.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Biological enrichment of Mycoplasma agents by cocultivation with permissive cell cultures</span>. <abbr class="cit-jnl-abbrev">Applied and Environmental Microbiology</abbr>, <span class="cit-vol">74</span>(<span class="cit-issue">17</span>), <span class="cit-fpage">5383</span>–<span class="cit-lpage">5391</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1128/AEM.00720-08</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.00720-08%26rft_id%253Dinfo%253Apmid%252F18606798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3NC8xNy81MzgzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTgvMjAxOS4xMi4yMS44ODU5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">97.</span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference 97. in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.97" data-doi="10.1016/j.vetmic.2010.10.026"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Volokhov</span>,  <span class="cit-name-given-names">Dmitriy V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Norris</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rios</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davidson</span>,  <span class="cit-name-given-names">M. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Messick</span>,  <span class="cit-name-given-names">J. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gulland</span>,  <span class="cit-name-given-names">F. M.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Chizhikov</span>,  <span class="cit-name-given-names">V. E.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Novel hemotrophic mycoplasma identified in naturally infected California sea lions (Zalophus californianus)</span>. <abbr class="cit-jnl-abbrev">Veterinary Microbiology</abbr>, <span class="cit-vol">149</span>(<span class="cit-issue">1–2</span>), <span class="cit-fpage">262</span>–<span class="cit-lpage">268</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.vetmic.2010.10.026</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bmicrobiology%26rft.stitle%253DVet%2BMicrobiol%26rft.aulast%253DVolokhov%26rft.auinit1%253DD.%2BV.%26rft.volume%253D149%26rft.issue%253D1-2%26rft.spage%253D262%26rft.epage%253D268%26rft.atitle%253DNovel%2Bhemotrophic%2Bmycoplasma%2Bidentified%2Bin%2Bnaturally%2Binfected%2BCalifornia%2Bsea%2Blions%2B%2528Zalophus%2Bcalifornianus%2529.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vetmic.2010.10.026%26rft_id%253Dinfo%253Apmid%252F21111543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.vetmic.2010.10.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21111543&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">98.</span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference 98. in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.98" data-doi="10.1016/j.ympev.2011.11.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Volokhov</span>,  <span class="cit-name-given-names">Dmitriy V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simonyan</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davidson</span>,  <span class="cit-name-given-names">M. K.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Chizhikov</span>,  <span class="cit-name-given-names">V. E.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">RNA polymerase beta subunit (rpoB) gene and the 16S–23S rRNA intergenic transcribed spacer region (ITS) as complementary molecular markers in addition to the 16S rRNA gene for phylogenetic analysis and identification of the species of the family Mycoplasmataceae</span>. <abbr class="cit-jnl-abbrev">Molecular Phylogenetics and Evolution</abbr>, <span class="cit-vol">62</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">515</span>–<span class="cit-lpage">528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BPhylogenetics%2Band%2BEvolution%26rft.volume%253D62%26rft.spage%253D515%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2011.11.002%26rft_id%253Dinfo%253Apmid%252F22115576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2011.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22115576&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">99.</span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference 99. in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.99" data-doi="10.1016/j.cimid.2016.03.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walker Vergara</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morera Galleguillos</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gómez Jaramillo</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pereira Almosny</span>,  <span class="cit-name-given-names">N. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arauna Martínez</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grob Behne</span>,  <span class="cit-name-given-names">P.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Müller</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Prevalence, risk factor analysis, and hematological findings of hemoplasma infection in domestic cats from Valdivia, Southern Chile</span>. <abbr class="cit-jnl-abbrev">Comparative Immunology, Microbiology and Infectious Diseases</abbr>, <span class="cit-vol">46</span>, <span class="cit-fpage">20</span>–<span class="cit-lpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cimid.2016.03.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComparative%2BImmunology%252C%2BMicrobiology%2Band%2BInfectious%2BDiseases%26rft.volume%253D46%26rft.spage%253D20%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cimid.2016.03.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cimid.2016.03.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">100.</span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference 100. in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.100" data-doi="10.1002/ecm.1353"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Washburne</span>,  <span class="cit-name-given-names">A. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silverman</span>,  <span class="cit-name-given-names">J. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morton</span>,  <span class="cit-name-given-names">J. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becker</span>,  <span class="cit-name-given-names">D. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crowley</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mukherjee</span>,  <span class="cit-name-given-names">S.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Plowright</span>,  <span class="cit-name-given-names">R. K.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Phylofactorization: a graph partitioning algorithm to identify phylogenetic scales of ecological data</span>. <abbr class="cit-jnl-abbrev">Ecological Monographs</abbr>, <span class="cit-vol">0</span>(<span class="cit-issue">0</span>), <span class="cit-fpage">e01353</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1002/ecm.1353</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BMonographs%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fecm.1353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ecm.1353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">101.</span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference 101. in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.101" data-doi="10.1007/s10393-017-1276-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Webber</span>,  <span class="cit-name-given-names">Q. M. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fletcher</span>,  <span class="cit-name-given-names">Q. E.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Willis</span>,  <span class="cit-name-given-names">C. K. R.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Viral Richness is Positively Related to Group Size, but Not Mating System, in Bats</span>. <abbr class="cit-jnl-abbrev">EcoHealth</abbr>, <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1007/s10393-017-1276-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcoHealth%26rft.volume%253D14%26rft.spage%253D652%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10393-017-1276-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10393-017-1276-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">102.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.21.885921v2.102" data-doi="10.1128/AEM.02977-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Willi</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boretti</span>,  <span class="cit-name-given-names">F. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meli</span>,  <span class="cit-name-given-names">M. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bernasconi</span>,  <span class="cit-name-given-names">M. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Casati</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hegglin</span>,  <span class="cit-name-given-names">D.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Hofmann-Lehmann</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Real-time PCR investigation of potential vectors, reservoirs, and shedding patterns of feline hemotropic mycoplasmas</span>. <abbr class="cit-jnl-abbrev">Applied and Environmental Microbiology</abbr>, <span class="cit-vol">73</span>(<span class="cit-issue">12</span>), <span class="cit-fpage">3798</span>–<span class="cit-lpage">3802</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1128/AEM.02977-06</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.02977-06%26rft_id%253Dinfo%253Apmid%252F17468284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3My8xMi8zNzk4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTgvMjAxOS4xMi4yMS44ODU5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">103.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.21.885921v2.103" data-doi="10.1016/j.vetmic.2007.06.027"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Willi</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boretti</span>,  <span class="cit-name-given-names">F. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tasker</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meli</span>,  <span class="cit-name-given-names">M. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wengi</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reusch</span>,  <span class="cit-name-given-names">C. E.</span></span>, … <span class="cit-auth"><span class="cit-name-surname">Hofmann-Lehmann</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">From Haemobartonella to hemoplasma: Molecular methods provide new insights</span>. <abbr class="cit-jnl-abbrev">Veterinary Microbiology</abbr>, <span class="cit-vol">125</span>(<span class="cit-issue">3–4</span>), <span class="cit-fpage">197</span>–<span class="cit-lpage">209</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.vetmic.2007.06.027</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bmicrobiology%26rft.stitle%253DVet%2BMicrobiol%26rft.aulast%253DWilli%26rft.auinit1%253DB.%26rft.volume%253D125%26rft.issue%253D3-4%26rft.spage%253D197%26rft.epage%253D209%26rft.atitle%253DFrom%2BHaemobartonella%2Bto%2Bhemoplasma%253A%2Bmolecular%2Bmethods%2Bprovide%2Bnew%2Binsights.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vetmic.2007.06.027%26rft_id%253Dinfo%253Apmid%252F17706380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.vetmic.2007.06.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17706380&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">104.</span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference 104. in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.104" data-doi="10.1890/13-1917.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wilman</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belmaker</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simpson</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de la Rosa</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rivadeneira</span>,  <span class="cit-name-given-names">M. M.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Jetz</span>,  <span class="cit-name-given-names">W.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">EltonTraits 1.0: Species-level foraging attributes of the world’s birds and mammals</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr>, <span class="cit-vol">95</span>(<span class="cit-issue">7</span>), <span class="cit-fpage">2027</span>–<span class="cit-lpage">2027</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1890/13-1917.1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D95%26rft.spage%253D2027%26rft_id%253Dinfo%253Adoi%252F10.1890%252F13-1917.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/13-1917.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">105.</span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference 105. in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Withenshaw</span>,  <span class="cit-name-given-names">S. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Devevey</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pedersen</span>,  <span class="cit-name-given-names">A. B.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Fenton</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Multihost Bartonella parasites display covert host specificity even when transmitted by generalist vectors</span>. <abbr class="cit-jnl-abbrev">Journal of Animal Ecology</abbr>, <span class="cit-vol">85</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">1442</span>–<span class="cit-lpage">1452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BEcology%26rft.volume%253D85%26rft.spage%253D1442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">106.</span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference 106. in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.106" data-doi="10.3201/eid1112.050997"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woolhouse</span>,  <span class="cit-name-given-names">M. E. J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Gowtage-Sequeria</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Host Range and Emerging and Reemerging Pathogens</span>. <abbr class="cit-jnl-abbrev">Emerging Infectious Diseases</abbr>, <span class="cit-vol">11</span>(<span class="cit-issue">12</span>), <span class="cit-fpage">1842</span>–<span class="cit-lpage">1847</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.3201/eid1112.050997</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DWoolhouse%26rft.auinit1%253DM.%2BE.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D1842%26rft.epage%253D1847%26rft.atitle%253DHost%2Brange%2Band%2Bemerging%2Band%2Breemerging%2Bpathogens.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1112.050997%26rft_id%253Dinfo%253Apmid%252F16485468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1112.050997&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16485468&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F18%2F2019.12.21.885921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233768300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">107.</span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference 107. in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.107" data-doi="10.1126/science.1059026"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woolhouse</span>,  <span class="cit-name-given-names">M. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taylor</span>,  <span class="cit-name-given-names">L. H.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Haydon</span>,  <span class="cit-name-given-names">D. T.</span></span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">Population biology of multihost pathogens</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">292</span>(<span class="cit-issue">5519</span>), <span class="cit-fpage">1109</span>–<span class="cit-lpage">1112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWoolhouse%26rft.auinit1%253DM.%2BE.%2BJ.%26rft.volume%253D292%26rft.issue%253D5519%26rft.spage%253D1109%26rft.epage%253D1112%26rft.atitle%253DPopulation%2BBiology%2Bof%2BMultihost%2BPathogens%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1059026%26rft_id%253Dinfo%253Apmid%252F11352066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTIvNTUxOS8xMTA5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTgvMjAxOS4xMi4yMS44ODU5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">108.</span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference 108. in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.21.885921v2.108" data-doi="10.1128/AEM.06516-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wright</span>,  <span class="cit-name-given-names">E. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yilmaz</span>,  <span class="cit-name-given-names">L. S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Noguera</span>,  <span class="cit-name-given-names">D. R.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">DECIPHER, a Search-Based Approach to Chimera Identification for 16S rRNA Sequences</span>. <abbr class="cit-jnl-abbrev">Applied and Environmental Microbiology</abbr>, <span class="cit-vol">78</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">717</span>–<span class="cit-lpage">725</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1128/AEM.06516-11</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.06516-11%26rft_id%253Dinfo%253Apmid%252F22101057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjg6Ijc4LzMvNzE3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTgvMjAxOS4xMi4yMS44ODU5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2019.12.21.885921v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2019.12.21.885921v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2019.12.13.875831v2" title="Broccoli: combining phylogenetic and network analyses for orthology assignment" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/858431v3" title="Localizing components of shared transethnic genetic architecture of complex traits from GWAS summary data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 18, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2019.12.21.885921v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2019.12.21.885921v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2019.12.21.885921v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2019.12.21.885921v2" class="active" data-icon-position="" data-hide-link-title="0">Ecological and evolutionary drivers of hemoplasma infection and bacterial genotype sharing in a Neotropical bat community</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1200651" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-yTU1WXHNILRx0bCek-29eEmV7npgOWbhxm8dD20aZj8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1200651" id="node1200651" data-pisa="biorxiv;2019.12.21.885921v2" data-pisa-master="biorxiv;2019.12.21.885921" data-seqnum="1200651" data-apath="/biorxiv/early/2020/03/18/2019.12.21.885921.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Ecological and evolutionary drivers of hemoplasma infection and bacterial genotype sharing in a Neotropical bat community</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Daniel J.</span> <span class="nlm-surname">Becker</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Kelly A.</span> <span class="nlm-surname">Speer</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Alexis M.</span> <span class="nlm-surname">Brown</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">M. Brock</span> <span class="nlm-surname">Fenton</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Alex D.</span> <span class="nlm-surname">Washburne</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Sonia</span> <span class="nlm-surname">Altizer</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Daniel G.</span> <span class="nlm-surname">Streicker</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Raina K.</span> <span class="nlm-surname">Plowright</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Vladimir E.</span> <span class="nlm-surname">Chizhikov</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Nancy B.</span> <span class="nlm-surname">Simmons</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Dmitriy V.</span> <span class="nlm-surname">Volokhov</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.21.885921; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.21.885921 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2019.12.21.885921v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2019.12.21.885921v2&amp;title=Ecological%20and%20evolutionary%20drivers%20of%20hemoplasma%20infection%20and%20bacterial%20genotype%20sharing%20in%20a%20Neotropical%20bat%20community" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.21.885921v2&amp;title=Ecological%20and%20evolutionary%20drivers%20of%20hemoplasma%20infection%20and%20bacterial%20genotype%20sharing%20in%20a%20Neotropical%20bat%20community" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.21.885921v2&amp;text=Ecological%20and%20evolutionary%20drivers%20of%20hemoplasma%20infection%20and%20bacterial%20genotype%20sharing%20in%20a%20Neotropical%20bat%20community" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.21.885921v2&amp;title=Ecological%20and%20evolutionary%20drivers%20of%20hemoplasma%20infection%20and%20bacterial%20genotype%20sharing%20in%20a%20Neotropical%20bat%20community" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2019.12.21.885921v2&amp;t=Ecological%20and%20evolutionary%20drivers%20of%20hemoplasma%20infection%20and%20bacterial%20genotype%20sharing%20in%20a%20Neotropical%20bat%20community" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2019.12.21.885921v2&amp;title=Ecological%20and%20evolutionary%20drivers%20of%20hemoplasma%20infection%20and%20bacterial%20genotype%20sharing%20in%20a%20Neotropical%20bat%20community" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.21.885921v2&amp;title=Ecological%20and%20evolutionary%20drivers%20of%20hemoplasma%20infection%20and%20bacterial%20genotype%20sharing%20in%20a%20Neotropical%20bat%20community" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1200651" id="node1200651--2" data-pisa="biorxiv;2019.12.21.885921v2" data-pisa-master="biorxiv;2019.12.21.885921" data-seqnum="1200651" data-apath="/biorxiv/early/2020/03/18/2019.12.21.885921.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Ecological and evolutionary drivers of hemoplasma infection and bacterial genotype sharing in a Neotropical bat community</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Daniel J.</span> <span class="nlm-surname">Becker</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Kelly A.</span> <span class="nlm-surname">Speer</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Alexis M.</span> <span class="nlm-surname">Brown</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">M. Brock</span> <span class="nlm-surname">Fenton</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Alex D.</span> <span class="nlm-surname">Washburne</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Sonia</span> <span class="nlm-surname">Altizer</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Daniel G.</span> <span class="nlm-surname">Streicker</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Raina K.</span> <span class="nlm-surname">Plowright</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Vladimir E.</span> <span class="nlm-surname">Chizhikov</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Nancy B.</span> <span class="nlm-surname">Simmons</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Dmitriy V.</span> <span class="nlm-surname">Volokhov</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.21.885921; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.21.885921 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Ecological%20and%20evolutionary%20drivers%20of%20hemoplasma%20infection%20and%20bacterial%20genotype%20sharing%20in%20a%20Neotropical%20bat%20community&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Becker%2C+Daniel+J.&amp;rft.au=Speer%2C+Kelly+A.&amp;rft.au=Brown%2C+Alexis+M.&amp;rft.au=Fenton%2C+M.+Brock&amp;rft.au=Washburne%2C+Alex+D.&amp;rft.au=Altizer%2C+Sonia&amp;rft.au=Streicker%2C+Daniel+G.&amp;rft.au=Plowright%2C+Raina+K.&amp;rft.au=Chizhikov%2C+Vladimir+E.&amp;rft.au=Simmons%2C+Nancy+B.&amp;rft.au=Volokhov%2C+Dmitriy+V."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1200651/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1200651/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1200651/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1200651/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1200651/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1200651/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1200651/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1200651/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1200651/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1200651/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1200651/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1200651/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.21.885921v2&amp;count=horizontal&amp;via=&amp;text=Ecological%20and%20evolutionary%20drivers%20of%20hemoplasma%20infection%20and%20bacterial%20genotype%20sharing%20in%20a%20Neotropical%20bat%20community&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2019.12.21.885921v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2019.12.21.885921v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2019.12.21.885921v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/ecology" class="highlight" data-icon-position="" data-hide-link-title="0">Ecology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
