<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Epidemics</span><span class="iso-abbrev" title="iso-abbrev">Epidemics</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Epidemics</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1755-4365</span><span class="issn-epub" tagx="issn" title="issn-epub">1878-0067</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6227250">6227250</a></span><span class="publisher-id" title="publisher-id">S1755-4365(17)30106-8</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1016/j.epidem.2018.05.004">10.1016/j.epidem.2018.05.004</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Tuberculosis outbreak investigation using phylodynamic analysis</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kühnert</span><span class="given-names" tagx="given-names" title="given-names">Denise</span></span><span class="email" tagx="email" title="email">kuehnert@shh.mpg.de</span><a href="#aff0005">a</a><a href="#aff0010">b</a><a href="#aff0015">c</a><a href="#aff0020">d</a><a href="#aff0025">e</a><a href="#cor0005">�?�</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coscolla</span><span class="given-names" tagx="given-names" title="given-names">Mireia</span></span><a href="#aff0030">f</a><a href="#aff0035">g</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brites</span><span class="given-names" tagx="given-names" title="given-names">Daniela</span></span><a href="#aff0030">f</a><a href="#aff0035">g</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stucki</span><span class="given-names" tagx="given-names" title="given-names">David</span></span><a href="#aff0030">f</a><a href="#aff0035">g</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Metcalfe</span><span class="given-names" tagx="given-names" title="given-names">John</span></span><a href="#aff0040">h</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fenner</span><span class="given-names" tagx="given-names" title="given-names">Lukas</span></span><a href="#aff0030">f</a><a href="#aff0035">g</a><a href="#aff0045">i</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gagneux</span><span class="given-names" tagx="given-names" title="given-names">Sebastien</span></span><a href="#aff0030">f</a><a href="#aff0035">g</a><a href="#fn0005">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">Tanja</span></span><a href="#aff0020">d</a><a href="#aff0025">e</a><a href="#fn0005">1</a></span></div><span class="citation_author_institution" id="aff0005">[a], </span><span class="citation_author_institution" id="aff0010">[b], </span><span class="citation_author_institution" id="aff0015">[c], </span><span class="citation_author_institution" id="aff0020">[d], </span><span class="citation_author_institution" id="aff0025">[e], </span><span class="citation_author_institution" id="aff0030">[f], </span><span class="citation_author_institution" id="aff0035">[g], </span><span class="citation_author_institution" id="aff0040">[h], </span><span class="citation_author_institution" id="aff0045">[i], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">�?�</span>Corresponding author at: Max Planck Institute for the Science of Human History, Jena, Germany <span class="email" tagx="email" title="email">kuehnert@shh.mpg.de</span></div> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">1</span><p id="npar0010">Equal contribution.</p> </div> </div><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-12-12</span></span><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2018-12</span></span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">47</span><span class="lpage" tagx="lpage" title="lpage">53</span><span class="history" title="history"><span class="received" title="received">received: 2017-6-19</span><span class="rev-recd" title="rev-recd">rev-recd: 2018-5-7</span><span class="accepted" title="accepted">accepted: 2018-5-13</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article under the CC BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/).</span></span></div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Highlights</div> <p> </p><ul> <li><span class="label" tagx="label" title="label">•</span><p id="par0005">Phylodynamic analysis gives insight into <i>mycobacterium tuberculosis</i> outbreaks.</p> </li> <li><span class="label" tagx="label" title="label">•</span><p id="par0010">Robust estimation of epidemiological parameters in Bern thanks to high sampling rate.</p> </li> <li><span class="label" tagx="label" title="label">•</span><p id="par0015">Infectious period for WTK cases significantly longer than in Bernese outbreak.</p> </li> </ul> <p /> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The fast evolution of pathogenic viruses has allowed for the development of phylodynamic approaches that extract information about the epidemiological characteristics of viral genomes. Thanks to advances in whole genome sequencing, they can be applied to slowly evolving bacterial pathogens like <i>Mycobacterium tuberculosis</i>. In this study, we investigate and compare the epidemiological dynamics underlying two <i>M. tuberculosis</i> outbreaks using phylodynamic methods. Specifically, we (i) test if the outbreak data sets contain enough genetic variation to estimate short-term evolutionary rates and (ii) reconstruct epidemiological parameters such as the effective reproduction number.</p> <p>The first outbreak occurred in the Swiss city of Bern (1987–2012) and was caused by a drug-susceptible strain belonging to the phylogenetic <i>M. tuberculosis</i> Lineage 4. The second outbreak was caused by a multidrug-resistant (MDR) strain of Lineage 2, imported from the Wat Tham Krabok (WTK) refugee camp in Thailand into California.</p> <p>There is little temporal signal in the Bern data set and moderate temporal signal in the WTK data set. Thanks to its high sampling proportion (90%) the Bern outbreak allows robust estimation of epidemiological parameters despite the poor temporal signal. Conversely, there is much uncertainty in the epidemiological estimates concerning the sparsely sampled (9%) WTK outbreak. Our results suggest that both outbreaks peaked around 1990, although they were only recognized as outbreaks in 1993 (Bern) and 2004 (WTK). Furthermore, individuals were infected for a significantly longer period (around 9 years) in the WTK outbreak than in the Bern outbreak (4–5 years).</p> <p>Our work highlights both the limitations and opportunities of phylodynamic analysis of outbreaks involving slowly evolving pathogens: (i) estimation of the evolutionary rate is difficult on outbreak time scales and (ii) a high sampling proportion allows quantification of the age of the outbreak based on the sampling times, and thus allows for robust estimation of epidemiological parameters.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Phylodynamic analysis</div> <div class="kwd" title="kwd">
Tuberculosis outbreak</div> <div class="kwd" title="kwd">
Epidemic dynamics</div> <div class="kwd" title="kwd">
Transmission dynamics</div> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1</span><div class="title" tagx="title" title="title">
Introduction</div> <p id="par0020">Whole genome sequencing (WGS) of clinical <i>M. tuberculosis</i> isolates is performed retrospectively and allows to confirm/refute suspected epidemiological links to identify individuals contributing to transmission, and explore drug-resistance and/or compensatory mechanisms which emerged during anti-tuberculosis treatment (<a href="#bib0155">Walker et al., 2013</a>; <a href="#bib0065">Gardy et al., 2011</a>; <a href="#bib0140">Stucki et al., 2015</a>; <a href="#bib0035">Coscolla et al., 2015a</a>; <a href="#bib0080">Hatherell et al., 2016</a>; <a href="#bib0025">Casali et al., 2016</a>). Although WGS analysis has the potential to reveal more complex epidemiological dynamics such as how long the outbreak was not controlled, the time patients are infectious for, the proportion of sampled cases, and the transmission potential of different strains, these epidemiological parameters are rarely estimated for slowly evolving bacterial pathogens such as <i>M. tuberculosis</i> (<a href="#bib0065">Gardy et al., 2011</a>; <a href="#bib0080">Hatherell et al., 2016</a>; <a href="#bib0145">Tanaka et al., 2006</a>). In the context of tuberculosis disease, answering those questions may help to evaluate and improve treatment strategies and control programs.</p> <p id="par0025">Phylodynamic analysis of real time WGS data can shed light on temporal dynamics of disease outbreaks, for example to determine if there is ongoing transmission (<a href="#bib0080">Hatherell et al., 2016</a>). Here, we employ phylodynamic methods to shed further light on two <i>M. tuberculosis</i> outbreaks. The first outbreak was detected around 1991 in the city of Bern, Switzerland, where twenty-two related cases, mainly homeless individuals and substance abusers, were identified initially (<a href="#bib0075">Genewein et al., 1993</a>). Using a novel combination of strain-specific SNP screening assay and targeted WGS, a tuberculosis cluster spanning 21 years and involving 68 patients was identified (<a href="#bib0140">Stucki et al., 2015</a>; <a href="#bib0145">Tanaka et al., 2006</a>). The genomic analysis revealed that this outbreak was caused by a Lineage 4 strain (Euro-American) of <i>M. tuberculosis</i>, and all but one showed no evidence of antibiotic resistant conferring mutations. The analysis revealed three sub-clusters within the outbreak, one of them associated to HIV coinfection (13 patients were coinfected with HIV).</p> <p id="par0030">The second data set consists of 30 MDR strains imported to California during resettlement of refugees from a refugee camp at Wat Tham Krabok (WTK) (<a href="#bib0035">Coscolla et al., 2015a</a>). Whole genome analysis confirmed that the strains causing the outbreak were multidrug-resistant and belonged to the Lineage 2 (East-Asian, Beijing genotype) of <i>M. tuberculosis</i>. Genomic data supported a single case whose isolate occupied the central node of the transmission network indicating the presence of a super-spreader. Epidemiological data integrated with the transmission chain also demonstrated multiple independent importation events from Thailand with reactivation and transmission within California over a 22-year period.</p> <p id="par0035">In this study, we aim to understand the dynamics of tuberculosis outbreaks by inferring phylogenetic trees together with epidemiological parameters, particularly, transmission and recovery rates, from genome sequence data using phylodynamic methods.</p> </div> <div class="methods" title="sec"><span class="label" tagx="label" title="label">2</span><div class="title" tagx="title" title="title">
Methods</div> <div class="reconstructionoftransmissiondynamics" title="sec"><span class="label" tagx="label" title="label">2.1</span><div class="title" tagx="title" title="title">
Reconstruction of transmission dynamics</div> <p id="par0040">First, we explored the temporal signal in the sequence alignments using TempEst (<a href="#bib0115">Rambaut et al., 2016</a>). The main analysis of both data sets was done within the Bayesian MCMC framework BEAST2 (<a href="#bib0015">Bouckaert et al., 2014</a>). We assume that the phylogeny spanned by the genomic samples is a suitable approximation of the transmission tree, such that we can estimate epidemiological parameters simultaneously with the phylogenetic tree. Since we do not intend to estimate direction of transmission this assumption refers to the branching times of the phylogenetic tree approximating times at which transmissions occurred only. We employ two phylodynamic methods, the birth-death skyline plot (BDSKY) (<a href="#bib0130">Stadler et al., 2013</a>) and the multi-type birth-death model (MTBD) (<a href="#bib0095">Kühnert et al., 2016</a>). Both assume that an infection event can be considered as the "birth�? of a newly infected individual, while a recovery event (successful treatment) is a "death�?. While the BDSKY model assumes that an infected individual is immediately infectious upon infection, the MTBD model allows us to incorporate the fact that <i>M. tuberculosis</i> infections usually start with a latent period in which the infected individual is not yet infectious.</p> <p id="par0045">In both analyses, we employ a general time reversible substitution model with gamma distributed rate heterogeneity and a proportion of invariant sites (GTR+I+Γ). A relaxed lognormal clock is used to model the variation of evolutionary (substitution) rates across branches, such that we estimate a mean clock rate θ (per SNP per year), from which we compute the number of SNPs per genome per year, and standard deviation σ for the lognormal branch rate distribution. In addition to the SNP alignments we specify the proportions of A, C, G and T’s as constant site weights. All parameters are estimated jointly. The prior distributions used are summarized in <a href="#tbl0005">Table 1</a>. <div class="table-wrap_UNKNOWN" id="tbl0005" position="float"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Bayesian prior distributions used for phylodynamic analysis.</p> [alt-text] Table 1 <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" /> <th align="left">Mean substitution rate θ</th> <th align="left">Standard deviation σ</th> <th align="left">Effective reproduction number R<sub>e</sub></th> <th align="left">Recovery rate δ</th> <th align="left">Exposed rate σ</th> <th align="left">Sampling proportion s</th> <th align="left">Origin of sample</th> <th align="left">Removal (upon sampling) probability r</th> </tr> </thead> <tbody> <tr> <td align="left"><b>Bern (BDSKY)</b></td> <td align="left">Unif(0,∞)</td> <td align="left">Exp(0.33)</td> <td align="left">LogN(0,1)</td> <td align="left">LogN<br /><br />(exp(0.5)<a href="#tblfn0005">*</a>,1)</td> <td align="left">–</td> <td align="left">Beta(45,5)</td> <td align="left">Unif(0,40)</td> <td align="left">Unif(0,1)</td> </tr> <tr> <td align="left"><b>Bern (MTBD)</b></td> <td align="left">Unif(0,∞)</td> <td align="left">Exp(0.33)</td> <td align="left">LogN(0,1)</td> <td align="left">Fixed to 2, 4 or 6</td> <td align="left">LogN(exp(m),1)<br /><br />m = 1/(2-1/δ)<a href="#tblfn0005">*</a></td> <td align="left">Beta(45,5)</td> <td align="left">Unif(0,40)</td> <td align="left">Unif(0,1)</td> </tr> <tr> <td align="left"><b>Thailand/</b><br /><br /><b>California</b><br /><br /><b>(BDSKY)</b></td> <td align="left">Unif(0,∞)</td> <td align="left">Exp(0.33)</td> <td align="left">LogN(0,1)</td> <td align="left">LogN(exp(0.5)<a href="#tblfn0005">*</a>,1)</td> <td align="left">–</td> <td align="left">Beta(10,90)</td> <td align="left">Unif(0,100)</td> <td align="left">Unif(0,1)</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">*</span><p id="npar0005">Mean determined to correspond to an infected duration (i.e. the sum of the exposed and infectious periods) of 2 years.</p> </div> </div>  <p /> <div class="phylodynamicanalysiswiththebirth-deathskylinemodel" title="sec"><span class="label" tagx="label" title="label">2.1.1</span><div class="title" tagx="title" title="title">
Phylodynamic analysis with the birth-death skyline model</div> <p id="par0050">The birth-death skyline model (<a href="#bib0130">Stadler et al., 2013</a>) describes a prior distribution for a transmission tree and is based on a stochastic birth-death process, with birth (λ), death (μ) and sampling (ψ) rates. Individuals become non-infectious upon sampling with probability r ∈ [0,1] (<a href="#bib0070">Gavryushkina et al., 2014</a>). Typically, the probability r is close to 1 if sampling is accompanied by successful treatment. To investigate the change of epidemiological dynamics, the period covered by the phylogeny is divided into intervals, and parameters are constant within an interval but may change between intervals. We can estimate the effective reproduction number R<sub>e</sub>, through the alternative parametrization of the model using the effective reproduction number R<sub>e</sub> = λ /(μ + rψ), the rate at which individuals become non-infectious δ = μ + rψ and the sampling proportion s = ψ / (μ + ψ). We employ m = 5 intervals to estimate potential changes in R<sub>e</sub>, and assume that δ is constant through time. The sampling proportion s is set to zero before the first sample, and assumed to be a positive constant thereafter.</p> </div> <div class="phylodynamicanalysiswiththemulti-typebirth-deathmodel–incorporatingthelatentperiod" title="sec"><span class="label" tagx="label" title="label">2.1.2</span><div class="title" tagx="title" title="title">
Phylodynamic analysis with the multi-type birth-death model – incorporating the latent period</div> <p id="par0055">The MTBD model allows us to incorporate and investigate the exposed phase. In the following we use the terms ‘latent’ and ‘exposed’ interchangeably, referring to the time during which individuals are infected but not yet infectious. The multi-type version of the birth-death skyline model (<a href="#bib0095">Kühnert et al., 2016</a>) allows us to distinguish between two types of infected individuals: (i) those who are not yet infectious (typically assigned to a compartment E), and (ii) those who are infectious (compartment I). Previous work has indicated that phylogenetic tools can estimate the overall infected period (including the exposed and infectious phases), but that it is difficult to estimate the exposed and infectious periods separately (<a href="#bib0135">Stadler et al., 2014</a>). Hence, we run three versions of this analysis, with the infectious period fixed to either 6 months (δ=2), 3 months (δ=4) or 2 months (δ=6) and report the results for each of those setups.</p> </div> <div class="modelcomparison" title="sec"><span class="label" tagx="label" title="label">2.1.3</span><div class="title" tagx="title" title="title">
Model comparison</div> <p id="par0060">The goodness of fit of the chosen models was tested using path sampling (<a href="#bib0005">Baele et al., 2012</a>). We tested the goodness of fit of (i) the clock model as well as (ii) the demographic model. As alternative models we used (i) a strict clock model and a relaxed clock model with exponentially distributed branch rates (<a href="#bib0050">Drummond et al., 2006</a>), and (ii) the commonly used constant coalescent model (<a href="#bib0090">Kingman, 1982</a>) and the Bayesian skyline model (<a href="#bib0110">Pybus et al., 2000</a>). The path sampling analyses were run for 30 steps with a chain length of 1.000.000 each and a pre-burnin of 50%.</p> </div> </div> <div class="datasets" title="sec"><span class="label" tagx="label" title="label">2.2</span><div class="title" tagx="title" title="title">
Data sets</div> <p id="par0065">Sampling procedures, strain isolation, sequencing, accession numbers for the sequences, and sequence analysis are described in detail in (<a href="#bib0140">Stucki et al., 2015</a>). In brief, we used the Illumina platform to sequence 68 patients associated with the Bernese outbreak and 46 from the WTK outbreak spanning more than 10 and 36 years, respectively. Only one isolate per patient was included and the isolation dates of the strains were used as sampling times. The isolates obtained from the Bernese outbreak were sequenced using Illumina HiSeq 2000, with single-end reads of approximately 50 bp as described in (<a href="#bib0140">Stucki et al., 2015</a>). The WTK outbreak was sequenced using an Illumina Genome Analyzer IIx, with single-end reads of also approximately 50 bp as described in <a href="#bib0035">Coscolla et al. (2015a)</a>. In both cases, reads were mapped against an inferred common ancestor sequence to all main lineages of the <i>Mycobacterium Tuberculosis</i> complex (<a href="#bib0030">Comas et al., 2010</a>) using both Burrows-Wheeler Aligner v0.5.8c (BWA) (<a href="#bib0100">Li and Durbin, 2009</a>) and SMALT (<a href="http://www.sanger.ac.uk/science/tools/smalt-0">http://www.sanger.ac.uk/science/tools/smalt-0</a>). Only positions consensual to both methods were considered. SNPs were determined with SAMtools (<a href="#bib0105">Li et al., 2009</a>). Only SNPs with a coverage of at least 10 reads and a phred-scaled mapping quality value of 20 were considered. In addition, SNPs falling in genes annotated as PE/PPE/PGRS, "maturase�?, "phage�?, "insertion sequence�? or "13E12 repeat family protein�? were removed due to the uncertainty of mappings.</p> <p id="par0070">An alignment of 133 variable positions among 68 isolates from the Bern outbreak (<a href="#bib0140">Stucki et al., 2015</a>) and an alignment of 150 variable positions among 30 Californian cases from the WTK outbreak were used (<a href="#bib0035">Coscolla et al., 2015a</a>). Possible drug resistance conferring mutations as described in (<a href="#bib0035">Coscolla et al., 2015a</a>) were excluded from the alignment.</p> </div> </div> <div class="results" title="sec"><span class="label" tagx="label" title="label">3</span><div class="title" tagx="title" title="title">
Results</div> <div class="tbinbern" title="sec"><span class="label" tagx="label" title="label">3.1</span><div class="title" tagx="title" title="title">
TB in Bern</div> <p id="par0075">The Lineage 4 Bernese data set shows positive correlation between genetic divergence and sampling time, but there is little temporal signal (TempEst R<sup>2</sup>=0.05, see <a href="#fig0005">Fig. 1</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Plot of phylogenetic Root-to-tip distance relative to sampling time (TempEst). Each dot represents one sample per data set (left: Bern, right: WTK).</p>   </div> <p /> <p id="par0080">The epidemiological parameter estimates we obtain for the Bernese outbreak largely agree among the different model specifications. We estimate that the temporal origin of the Bernese data set was around 1986 with the 95% highest posterior density intervals (HPD) ranging from 1985 to 1988.</p> <p id="par0085">Assuming a model without an exposed phase (BDSKY) we estimate an initial high effective reproduction number R<sub>e</sub> of 4.9 (median, 95% HPD: 2.6–8.1). Around 1991, it declined drastically, staying below the epidemic threshold 1 for the rest of the sampling period (<a href="#fig0010">Fig. 2</a>). The recovery rate δ is estimated to be 0.2 (median), suggesting an infected period of 5 years. The sampling proportion does not deviate from its prior distribution and is hence estimated at 90%. We estimate that the data set contains one sampled ancestor (95% HPD, 0–4), with infected individuals being removed upon sampling with 98% probability (95% HPD, 90–100%). The mean substitution rate is estimated to be 0.55 SNPs per genome per year (95% HPD, 0.32-0.86). <a href="#tbl0010">Table 2</a> summarizes the median posterior estimates and their 95% HPD intervals. <a href="#fig0015">Fig. 3</a> shows the maximum clade credibility tree that was generated from the posterior distribution of trees using TreeAnnotator, which is part of BEAST version 2.4 (<a href="#bib0015">Bouckaert et al., 2014</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>Bern reproduction number through time (BDSKY).</p> <p>The median effective reproduction number (black line) with its 95% highest posterior density (HPD) interval (shaded area). The grey bars display a histogram of the number of cases diagnosed per year.</p>   </div> <div class="table-wrap_UNKNOWN" id="tbl0010" position="float"><span class="label" tagx="label" title="label">Table 2</span> <p>Bayesian Posterior results of phylodynamic analyses of both outbreaks.</p> [alt-text] Table 2 <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" /> <th align="left">Mean number of SNPs per genome per year</th> <th align="left">Standard deviation σ</th> <th align="left">Effective reproduction number R<sub>e</sub><br /><br /><sub>(before 1992)</sub></th> <th align="left">Effective reproduction number R<sub>e</sub><br /><br /><sub>(after 1992)</sub></th> <th align="left">Recovery rate δ</th> <th align="left">Exposed rate σ</th> <th align="left">Sampling proportion s</th> <th align="left">Time of epidemic origin of sample</th> <th align="left">Removal (upon sampling) probability r</th> </tr> </thead> <tbody> <tr> <td align="left"><b>Bern (MTBD)</b><br /><br /><b>δ = 2</b></td> <td align="left">0.72<br /><br />(0.40-1.24)</td> <td align="left">0.95<br /><br />(0.60-1.34)</td> <td align="left">2.28<br /><br />(1.41-3.40)</td> <td align="left">0.24<br /><br />(0.06-0.50)</td> <td align="left">2 (fixed)</td> <td align="left">0.25<br /><br />(0.16-0.38)</td> <td align="left">0.87<br /><br />(0.76-0.96)</td> <td align="left">1986.75<br /><br />(1985.03-1987.47)</td> <td align="left">0.98<br /><br />(0.93-1)</td> </tr> <tr> <td align="left"><b>Bern (MTBD)</b><br /><br /><b>δ = 4</b></td> <td align="left">0.80<br /><br />(0.41-1.50)</td> <td align="left">1.00<br /><br />(0.65-1.44)</td> <td align="left">2.25<br /><br />(1.39-3.33)</td> <td align="left">0.22<br /><br />(0.05-0.47)</td> <td align="left">4 (fixed)</td> <td align="left">0.24<br /><br />(0.16-0.34)</td> <td align="left">0.85<br /><br />(0.73-0.95)</td> <td align="left">1987.08<br /><br />(1985.24-1987.47)</td> <td align="left">0.98<br /><br />(0.93-1)</td> </tr> <tr> <td align="left"><b>Bern (MTBD)</b><br /><br /><b>δ = 6</b></td> <td align="left">0.83<br /><br />(0.42-1.60)</td> <td align="left">1.01<br /><br />(0.65-1.43)</td> <td align="left">2.23<br /><br />(1.39-3.30)</td> <td align="left">0.24<br /><br />(0.06-0.48)</td> <td align="left">6 (fixed)</td> <td align="left">0.24<br /><br />(0.16-0.34)</td> <td align="left">0.83<br /><br />(0.71-0.94)</td> <td align="left">1987.2<br /><br />(1985.07-1987.47)</td> <td align="left">0.97<br /><br />(0.92-1)</td> </tr> <tr> <td align="left"><b>Bern (BDSKY)</b></td> <td align="left">0.55<br /><br />(0.32-0.86)</td> <td align="left">0.90<br /><br />(0.58-1.27)</td> <td colspan="2" align="left">See <a href="#fig0010">Fig. 2</a>.</td> <td align="left">0.20<br /><br />(0.12-0.29)</td> <td align="left">NA</td> <td align="left">0.90<br /><br />(0.81-0.97)</td> <td align="left">1986.39<br /><br />(1985.4-1987.18)</td> <td align="left">0.98<br /><br />(0.90-1)</td> </tr> <tr> <td align="left"><b>Thailand/</b><br /><br /><b>California</b></td> <td align="left">0.36<br /><br />(0.11-0.57)</td> <td align="left">0.27<br /><br />(0.00055-<br /><br />0.63)</td> <td colspan="2" align="left">See <a href="#fig0020">Fig. 4</a>.</td> <td align="left">0.13<br /><br />(0.037-0.27)</td> <td align="left">NA</td> <td align="left">0.08<br /><br />(0.04-0.15)</td> <td align="left">1975.58<br /><br />(1935.85-1993.09)</td> <td align="left">0.49<br /><br />(0.10-1)</td> </tr> </tbody> </table> </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>Bern maximum clade credibility tree.</p>   </div> <p /> <p id="par0090">Explicit incorporation of the exposed period in the MTBD model allows us to distinguish the average duration that infected individuals remain infectious. Due to the computational complexity of the model we only allowed one change in the effective reproduction number R<sub>e</sub> to have occurred in 1992. A more general setup (with more intervals for R<sub>e</sub>) leads to identifiability issues. Before 1992, we estimate median R<sub>e</sub> values around 2.25 and afterwards the median estimates are significantly below the epidemic threshold 1. Under the MTBD model we fixed the rate δ at which infected individuals become non-infectious to 2, 4 or 6 per year, suggesting an infected period of 6, 3 or 2 months. The median rate σ at which infected individuals become infectious is around 0.25, that is, on average infected individuals became infectious after 4 years in each of the three scenarios. Again, we estimate that the data set contains one sampled ancestor, with infected individuals being removed upon sampling with 98% probability. The mean number of SNPs per genome per year is estimated to be 0.72 (95% HPD, 0.40–1.24) when δ=2, 0.8 (95% HPD, 0.41-1.50) when δ=4 and 0.83 (95% HPD, 0.42-1.60) when δ=6 (<a href="#tbl0010">Table 2</a>).</p> </div> <div class="tbinhmongmigrantsfromthailand" title="sec"><span class="label" tagx="label" title="label">3.2</span><div class="title" tagx="title" title="title">
TB in Hmong migrants from Thailand</div> <p id="par0095">The Lineage 2 WTK data set shows positive correlation between genetic divergence and sampling time, and a moderate level of temporal signal (TempEst R<sup>2</sup>=0.35).</p> <p id="par0100">We analysed this data set under the BDSKY model, and allowed m = 4 intervals to estimate changes in the effective reproduction number R<sub>e</sub>. The temporal origin of this data set is estimated around 1976 with the 95% HPD interval ranging from 1935 to 1993. There is much uncertainty in the estimate of the effective reproduction number R<sub>e</sub>, its median and 95% HPD interval are shown in <a href="#fig0020">Fig. 4</a>. The rate δ at which infected individuals become non-infectious is estimated to be 0.13 (median), suggesting an infected period of 8 years. The median sampling proportion estimate is 8% (95% HPD, 4–15%). We estimate that the data set contains no sampled ancestors (95% HPD, 0–2), with the probability to be removed upon sampling r = 64% (95% HPD, 10–100%). The mean number of SNPs per genome per year is estimated to be 0.36 (95% HPD, 0.11–0.57). <a href="#fig0025">Fig. 5</a> shows the maximum clade credibility tree that was generated from the posterior distribution of trees using TreeAnnotator (<a href="#bib0015">Bouckaert et al., 2014</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>WTK reproduction number through time (BDSKY).</p> <p>The median effective reproduction number (black line) with its 95% highest posterior density (HPD) interval (shaded area). The grey bars display a histogram of the number of cases diagnosed per year.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <p>Hmong maximum clade credibility tree.</p>   </div> <p /> <p id="par0105">Due to the large uncertainty in the BDSKY estimates we did not attempt analysis of the WTK data set under the more complex MTBD model.</p> <p id="par0110"><a href="#tbl0015">Table 3</a> summarizes the results of the path sampling analyses. Both data sets are best fit by the BDSKY demographic model combined with a relaxed clock model with lognormally distributed branch rates. <div class="table-wrap_UNKNOWN" id="tbl0015" position="float"><span class="label" tagx="label" title="label">Table 3</span> </div></p><p>Overview of marginal likelihood estimates from Path Sampling analyses.</p> [alt-text] Table 3 <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">Data</th> <th align="left">Demographic Model</th> <th align="left">Clock Model</th> <th align="left">Marginal Likelihood</th> </tr> </thead> <tbody> <tr> <td align="left">WTK</td> <td align="left">BDSKY</td> <td align="left">UCLD</td> <td align="left"><b>−5225011.591</b></td> </tr> <tr> <td align="left">WTK</td> <td align="left">BSP</td> <td align="left">UCLD</td> <td align="left">−5411683.47</td> </tr> <tr> <td align="left">WTK</td> <td align="left">ConstCoal</td> <td align="left">UCLD</td> <td align="left">−5411701.334</td> </tr> <tr> <td align="left">WTK</td> <td align="left">BDSKY</td> <td align="left">SC</td> <td align="left">−5411679.402</td> </tr> <tr> <td align="left">WTK</td> <td align="left">BDSKY</td> <td align="left">UCED</td> <td align="left">−5411681.248</td> </tr> <tr> <td colspan="4" align="left">  </td> </tr> <tr> <td align="left">Bern</td> <td align="left">BDSKY</td> <td align="left">UCLD</td> <td align="left"><b>−5236130.59</b></td> </tr> <tr> <td align="left">Bern</td> <td align="left">MTBD</td> <td align="left">UCLD</td> <td align="left">−5236177.83</td> </tr> <tr> <td align="left">Bern</td> <td align="left">BSP</td> <td align="left">UCLD</td> <td align="left">−5422820.42</td> </tr> <tr> <td align="left">Bern</td> <td align="left">ConstCoal</td> <td align="left">UCLD</td> <td align="left">−5422845.13</td> </tr> <tr> <td align="left">Bern</td> <td align="left">BDSKY</td> <td align="left">SC</td> <td align="left">−5422841.42</td> </tr> <tr> <td align="left">Bern</td> <td align="left">BDSKY</td> <td align="left">UCED</td> <td align="left">−5422827.40</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>The highest marginal likelihood estimate for each data set is shown in bold.</p> </div> </div>  <p /> </div> </div> <div class="discussion" title="sec"><span class="label" tagx="label" title="label">4</span><div class="title" tagx="title" title="title">
Discussion</div> <p id="par0115">In this study, we used Bayesian phylodynamic methods to reconstruct the epidemiological dynamics of two <i>M. tuberculosis</i> WGS data sets corresponding to two unrelated outbreaks. We quantify the time of the start of the outbreaks, and the effective reproductive numbers through time.</p> <p id="par0120">The Bernese outbreak is characterized by (i) the outbreak being restricted to the medium size Swiss city Bern and (ii) a large sampling proportion. Indeed, many cases were sampled shortly after infection and subsequent cases have been recovered by a SNP screening assay and targeted WGS, such that an estimated 90% of secondary (i.e. infectious) cases linked to this outbreak are included in the data set. This high sampling proportion and the geographical containment of the Bernese outbreak are the likely cause of the higher confidence of the estimates obtained for this dataset, because the sampling times of sequences in a densely-sampled outbreak – combined with the genetic variation among them - are very informative for the age of an outbreak, and thus allow to time-calibrate the phylogeny and quantify transmission and recovery rates.</p> <p id="par0125">There is much uncertainty in the epidemiological parameters estimated from the WTK outbreak. The WTK data set has roughly half the number of samples and a much lower sampling proportion of 9% (median estimate). Furthermore, although the outbreak started in Thailand, our WTK data set consists entirely of cases imported into California. This sparse outbreak sample does not contain much information regarding the age of the outbreak, resulting in much uncertainty in the epidemiological estimates. Previous studies focussing on viral phylodynamics have shown that robust phylodynamic reconstruction is possible even when sampling proportions are low (<a href="#bib0010">Boskova et al., 2014</a>). Hence, other factors must play a role, which we discuss below.</p> <p id="par0130">Our analyses were conducted using phylodynamic methods implemented in the Bayesian MCMC framework BEAST version 2.4 (<a href="#bib0015">Bouckaert et al., 2014</a>), which means that we are estimating so-called time-trees using molecular clock models. Before using such models, one should explore the temporal signal in sequence alignments, which can be done using TempEst (<a href="#bib0115">Rambaut et al., 2016</a>). While both data sets exhibit a positive correlation between genetic divergence and sampling time, there is a moderate level of temporal signal only in the WTK data set (R<sup>2</sup> = 0.35). The WTK data set belongs to Lineage 2, for which Duchene et al. (<a href="#bib0055">Duchêne et al., 2016</a>) were unable to reliably determine the evolutionary rate. As outbreak data sets are often not suitable for long-term mutation rate estimation this estimate should be taken with a grain of salt. For a robust long-term estimate, one would want to collect longitudinal data over a longer time period (<a href="#bib0055">Duchêne et al., 2016</a>).</p> <p id="par0135">There is little temporal signal in the Lineage 4 Bernese data set (R<sup>2</sup> = 0.05), which explains the uncertainty in our clock rate estimates of the Bernese outbreak. Our results show that the estimated time of the epidemic origin and the epidemiological parameters are robust to the differing clock rate estimates, see <a href="#tbl0010">Table 2</a>.</p> <p id="par0140">We hypothesize that the two data sets are an example of the time dependency of molecular rate estimates (<a href="#bib0085">Ho et al., 2005</a>): the estimates of the evolutionary rate for the Bernese outbreak represent a high short-term rate of evolution, whereas due to the delayed sampling, the WTK estimate is a lower intermediate-term mutation rate of evolution. Hence, our evolutionary rate estimates are not suitable for comparison between the two <i>M. tuberculosis</i> lineages.</p> <p id="par0145">Our phylodynamic analyses allowed us to estimate the temporal dynamics of the Bernese outbreak. Despite the fact that the sampling dates range from 1987 to 2011, our results support the hypothesis that the epidemic peaked around 1990 (<a href="#bib0140">Stucki et al., 2015</a>). This indicates that the peak of the outbreak occurred several years before it was detected. Indeed, most of the transmission events likely occurred between 1990 and 1991, although the majority of cases was only reported in 1993 (<a href="#bib0140">Stucki et al., 2015</a>). This refutes the previous hypothesis that disease would have occurred shortly after infection, with short latent periods (<a href="#bib0150">W. H. Organization, 2015</a>), due to the population characteristics in the affected population of the Bernese outbreak (homeless, substance abusers, etc.).</p> <p id="par0150">Both models employed for analysis of the Bernese outbreak (BDSKY and MTBD) suggest that the average infected period lasted about 4–5 years. While in BDSKY the infected period is equivalent to the infectious period, the infected period in the MTBD model is the sum of the infectious and exposed periods. In the latter we assume an infectious period of 2, 3 or 6 months (<a href="#bib0125">Sreeramareddy et al., 2009</a>), and in each of those cases the exposed period is robustly estimated around 4 years. While this means that both models agree on the overall infected period to last around 4–5 years, we know that MTBD is the more realistic model. Hence, we conclude that – on average – an infected patient in the Bern outbreak was in the latent/exposed stage of the disease for about 4 years before becoming infectious and consequently being diagnosed and treated shortly after (<a href="#bib0125">Sreeramareddy et al., 2009</a>).</p> <p id="par0155">For the WTK outbreak, we estimated an infectious period of eight years, which is significantly higher than the infectious period estimated for the Bernese outbreak (p-value &amp;lt;2.2 × 10<sup>−16</sup>). This may be due to a delay in sampling and treatment, due to the sampling having taken place in California after immigration from Thailand, such that patients were likely sick and infectious for longer. Furthermore, while the Bernese outbreak was caused by a sensitive strain, the WTK outbreak was caused by an MDR strain. Multidrug resistance is generally associated with poorer treatment success and prolonged infectiousness (<a href="#bib0160">Winston and Mitruka, 2012</a>; <a href="#bib0060">Dye, 2009</a>).</p> <p id="par0160">Hence, this difference in the estimated infectious period for the Bernese outbreak and the WTK outbreak is expected and supports our phylodynamic approach. More generally, a long infectious period is characteristic for a chronic disease like tuberculosis, particularly considering the difficulties in diagnosing the disease and the long delays in health seeking and treatment initiation (<a href="#bib0170">Yuen et al., 2015</a>). Our estimates of the epidemiological parameters, particularly R<sub>e</sub>, are in line with previous estimates obtained from different <i>M. tuberculosis</i> data sets (<a href="#bib0145">Tanaka et al., 2006</a>; <a href="#bib0120">Sanchez and Blower, 1997</a>). However, due to the chronic nature of tuberculosis and the many factors influencing the transmission potential of individual patients (<a href="#bib0020">Brites and Gagneux, 2015</a>; <a href="#bib0040">Coscolla et al., 2015b</a>; <a href="#bib0165">Yruela et al., 2016</a>), more work is needed to understand the complex dynamics influencing these parameters in different epidemiological settings.</p> <p id="par0165">We have employed the commonly used assumption that the infectious and exposed periods are exponentially distributed. Didelot et al. (<a href="#bib0045">Didelot et al. (2014)</a>) have analysed a TB outbreak using a gamma distribution as prior distribution, but found that their posterior distribution had a mode of zero, suggesting that an exponential distribution is indeed suitable.</p> <p id="par0170">Our study shows that phylodynamic analysis of WGS data can shed light on the temporal dynamics of tuberculosis outbreaks. Analysis of the Bernese outbreak has revealed that even when there is little temporal signal, we can robustly estimate epidemiological parameters if the sampling proportion is large. Conversely, in the WTK outbreak there is much uncertainty in the epidemiological parameter estimates despite a moderate temporal signal. This may be due to a difference in transmission dynamics in Thailand versus California as well as the fact that the epidemic peak likely occurred before the first samples were taken.</p> <p id="par0175">Overall, we believe that real time outbreak WGS together with phylodynamic methods will improve future outbreak investigation as phylodynamic analysis can shed light on the timing of the epidemic origin and transmission dynamics through time.</p> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bib0005" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baele</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alekseyenko</span><span class="given-names" tagx="given-names" title="given-names">A.V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Improving the accuracy of demographic and molecular clock model comparison while accommodating phylogenetic uncertainty</span><span class="source" tagx="source" title="source">Mol. Biol Evol.</span><span class="volume" tagx="volume" title="volume">29</span><span class="issue" tagx="issue" title="issue">September (9)</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">2157</span><span class="lpage" tagx="lpage" title="lpage">2167</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22403239">22403239</a></span></span></li> <li tag="ref"><a name="bib0010" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boskova</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonhoeffer</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Inference of epidemiological dynamics based on simulated phylogenies using birth-death and coalescent models</span><span class="source" tagx="source" title="source">PLoS Comput. Biol.</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">11</span><span class="year" tagx="year" title="year">2014</span></span></li> <li tag="ref"><a name="bib0015" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouckaert</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">BEAST 2: a software platform for Bayesian evolutionary analysis</span><span class="source" tagx="source" title="source">PLoS Comput Biol</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">4</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">e1003537</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24722319">24722319</a></span></span></li> <li tag="ref"><a name="bib0020" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brites</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gagneux</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Co-evolution of Mycobacterium tuberculosis and Homo sapiens</span><span class="source" tagx="source" title="source">Immunol. Rev.</span><span class="volume" tagx="volume" title="volume">264</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">6</span><span class="lpage" tagx="lpage" title="lpage">24</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25703549">25703549</a></span></span></li> <li tag="ref"><a name="bib0025" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Casali</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Broda</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">S.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parkhill</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drobniewski</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Whole genome sequence analysis of a large isoniazid-resistant tuberculosis outbreak in London: a retrospective observational study</span><span class="source" tagx="source" title="source">PLoS Med.</span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">October (10)</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">e1002137</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27701423">27701423</a></span></span></li> <li tag="ref"><a name="bib0030" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Comas</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Human T cell epitopes of Mycobacterium tuberculosis are evolutionarily hyperconserved</span><span class="source" tagx="source" title="source">Nat. Genet.</span><span class="volume" tagx="volume" title="volume">42</span><span class="issue" tagx="issue" title="issue">6</span><span class="year" tagx="year" title="year">2010</span><span class="fpage" tagx="fpage" title="fpage">498</span><span class="lpage" tagx="lpage" title="lpage">503</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20495566">20495566</a></span></span></li> <li tag="ref"><a name="bib0035" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coscolla</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genomic epidemiology of multidrug-resistant Mycobacterium tuberculosis during transcontinental spread</span><span class="source" tagx="source" title="source">J. Infect. Dis.</span><span class="volume" tagx="volume" title="volume">212</span><span class="issue" tagx="issue" title="issue">July (2)</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">302</span><span class="lpage" tagx="lpage" title="lpage">310</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25601940">25601940</a></span></span></li> <li tag="ref"><a name="bib0040" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coscolla</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">M. tuberculosis T cell epitope analysis reveals paucity of antigenic variation and identifies rare variable TB antigens</span><span class="source" tagx="source" title="source">Cell. Host Microbe</span><span class="volume" tagx="volume" title="volume">18</span><span class="issue" tagx="issue" title="issue">5</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">538</span><span class="lpage" tagx="lpage" title="lpage">548</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26607161">26607161</a></span></span></li> <li tag="ref"><a name="bib0045" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Didelot</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gardy</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colijn</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bayesian inference of infectious disease transmission from whole-genome sequence data</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="volume" tagx="volume" title="volume">31</span><span class="issue" tagx="issue" title="issue">7</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">1869</span><span class="lpage" tagx="lpage" title="lpage">1879</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24714079">24714079</a></span></span></li> <li tag="ref"><a name="bib0050" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phillips</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span></span></span><span class="mixed-article-title" title="mixed-article-title">A: relaxed phylogenetics and dating with confidence</span><span class="source" tagx="source" title="source">PLoS Biology</span><span class="volume" tagx="volume" title="volume">4</span><span class="year" tagx="year" title="year">2006</span><span class="fpage" tagx="fpage" title="fpage">e88</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16683862">16683862</a></span></span></li> <li tag="ref"><a name="bib0055" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duchêne</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genome-scale rates of evolutionary change in bacteria</span><span class="source" tagx="source" title="source">Microb. Genom.</span><span class="volume" tagx="volume" title="volume">2</span><span class="issue" tagx="issue" title="issue">November (11)</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">e000094</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28348834">28348834</a></span></span></li> <li tag="ref"><a name="bib0060" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dye</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Doomsday postponed? Preventing and reversing epidemics of drug-resistant tuberculosis</span><span class="source" tagx="source" title="source">Nat. Rev. Microbiol.</span><span class="volume" tagx="volume" title="volume">7</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">81</span><span class="lpage" tagx="lpage" title="lpage">87</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19079354">19079354</a></span></span></li> <li tag="ref"><a name="bib0065" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gardy</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Whole-genome sequencing and social-network analysis of a tuberculosis outbreak</span><span class="source" tagx="source" title="source">N. Engl. J. Med.</span><span class="volume" tagx="volume" title="volume">364</span><span class="issue" tagx="issue" title="issue">February (8)</span><span class="year" tagx="year" title="year">2011</span><span class="fpage" tagx="fpage" title="fpage">730</span><span class="lpage" tagx="lpage" title="lpage">739</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21345102">21345102</a></span></span></li> <li tag="ref"><a name="bib0070" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gavryushkina</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Welch</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bayesian inference of sampled ancestor trees for epidemiology and fossil calibration</span><span class="source" tagx="source" title="source">PLoS Comput. Biol.</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">no. 12</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">e1003919</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25474353">25474353</a></span></span></li> <li tag="ref"><a name="bib0075" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Genewein</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular approach to identifying route of transmission of tuberculosis in the community</span><span class="source" tagx="source" title="source">Lancet</span><span class="volume" tagx="volume" title="volume">342</span><span class="issue" tagx="issue" title="issue">October (8875)</span><span class="year" tagx="year" title="year">1993</span><span class="fpage" tagx="fpage" title="fpage">841</span><span class="lpage" tagx="lpage" title="lpage">844</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8104275">8104275</a></span></span></li> <li tag="ref"><a name="bib0080" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hatherell</span><span class="given-names" tagx="given-names" title="given-names">H.-A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Declaring a tuberculosis outbreak over with genomic epidemiology</span><span class="source" tagx="source" title="source">Microb. Genom.</span><span class="volume" tagx="volume" title="volume">2</span><span class="issue" tagx="issue" title="issue">May (5)</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">e000060</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28348853">28348853</a></span></span></li> <li tag="ref"><a name="bib0085" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">S.Y.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phillips</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cooper</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Time dependency of molecular rate estimates and systematic overestimation of recent divergence times</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="volume" tagx="volume" title="volume">22</span><span class="issue" tagx="issue" title="issue">7</span><span class="year" tagx="year" title="year">2005</span><span class="fpage" tagx="fpage" title="fpage">1561</span><span class="lpage" tagx="lpage" title="lpage">1568</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15814826">15814826</a></span></span></li> <li tag="ref"><a name="bib0090" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kingman</span><span class="given-names" tagx="given-names" title="given-names">J.F.C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The coalescent</span><span class="source" tagx="source" title="source">Stochastic Process. Appl.</span><span class="volume" tagx="volume" title="volume">13</span><span class="year" tagx="year" title="year">1982</span><span class="fpage" tagx="fpage" title="fpage">235</span><span class="lpage" tagx="lpage" title="lpage">248</span></span></li> <li tag="ref"><a name="bib0095" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kühnert</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vaughan</span><span class="given-names" tagx="given-names" title="given-names">T.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Phylodynamics with migration: a computational framework to quantify population structure from genomic data</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="volume" tagx="volume" title="volume">33</span><span class="issue" tagx="issue" title="issue">August (8)</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">2102</span><span class="lpage" tagx="lpage" title="lpage">2116</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27189573">27189573</a></span></span></li> <li tag="ref"><a name="bib0100" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durbin</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">25</span><span class="issue" tagx="issue" title="issue">14</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">1754</span><span class="lpage" tagx="lpage" title="lpage">1760</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168">19451168</a></span></span></li> <li tag="ref"><a name="bib0105" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The sequence Alignment/Map format and SAMtools</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">25</span><span class="issue" tagx="issue" title="issue">16</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">2078</span><span class="lpage" tagx="lpage" title="lpage">2079</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943">19505943</a></span></span></li> <li tag="ref"><a name="bib0110" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">O.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harvey</span><span class="given-names" tagx="given-names" title="given-names">P.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">An integrated framework for the inference of viral population history from reconstructed genealogies</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">155</span><span class="issue" tagx="issue" title="issue">3</span><span class="year" tagx="year" title="year">2000</span><span class="fpage" tagx="fpage" title="fpage">1429</span><span class="lpage" tagx="lpage" title="lpage">1437</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10880500">10880500</a></span></span></li> <li tag="ref"><a name="bib0115" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">T.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Max Carvalho</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">O.G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Exploring the temporal structure of heterochronous sequences using TempEst (formerly Path-O-Gen)</span><span class="source" tagx="source" title="source">Virus Evol.</span><span class="volume" tagx="volume" title="volume">2</span><span class="issue" tagx="issue" title="issue">January (1)</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">vew007</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27774300">27774300</a></span></span></li> <li tag="ref"><a name="bib0120" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanchez</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blower</span><span class="given-names" tagx="given-names" title="given-names">S.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Uncertainty and sensitivity analysis of the basic reproductive rate. Tuberculosis as an example</span><span class="source" tagx="source" title="source">Am. J. Epidemiol.</span><span class="volume" tagx="volume" title="volume">145</span><span class="issue" tagx="issue" title="issue">12</span><span class="year" tagx="year" title="year">1997</span><span class="fpage" tagx="fpage" title="fpage">1127</span><span class="lpage" tagx="lpage" title="lpage">1137</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9199543">9199543</a></span></span></li> <li tag="ref"><a name="bib0125" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sreeramareddy</span><span class="given-names" tagx="given-names" title="given-names">C.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panduru</span><span class="given-names" tagx="given-names" title="given-names">K.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Menten</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">den Ende</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Time delays in diagnosis of pulmonary tuberculosis: a systematic review of literature</span><span class="source" tagx="source" title="source">BMC Infect. Dis.</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">June</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">91</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19519917">19519917</a></span></span></li> <li tag="ref"><a name="bib0130" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kühnert</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonhoeffer</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Birth-death skyline plot reveals temporal changes of epidemic spread in HIV and hepatitis C virus (HCV)</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U. S. A.</span><span class="volume" tagx="volume" title="volume">110</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">228</span><span class="lpage" tagx="lpage" title="lpage">233</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23248286">23248286</a></span></span></li> <li tag="ref"><a name="bib0135" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stadler</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kühnert</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rasmussen</span><span class="given-names" tagx="given-names" title="given-names">D.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">du Plessis</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Insights into the early epidemic spread of ebola in sierra leone provided by viral sequence data</span><span class="source" tagx="source" title="source">PLoS Curr.</span><span class="volume" tagx="volume" title="volume">6</span><span class="year" tagx="year" title="year">2014</span></span></li> <li tag="ref"><a name="bib0140" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stucki</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Tracking a tuberculosis outbreak over 21 years: strain-specific single-nucleotide polymorphism typing combined with targeted whole-genome sequencing</span><span class="source" tagx="source" title="source">J. Infect. Dis.</span><span class="volume" tagx="volume" title="volume">211</span><span class="issue" tagx="issue" title="issue">April (8)</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">1306</span><span class="lpage" tagx="lpage" title="lpage">1316</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25362193">25362193</a></span></span></li> <li tag="ref"><a name="bib0145" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanaka</span><span class="given-names" tagx="given-names" title="given-names">M.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Francis</span><span class="given-names" tagx="given-names" title="given-names">A.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luciani</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sisson</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Using approximate Bayesian computation to estimate tuberculosis transmission parameters from genotype data</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">173</span><span class="issue" tagx="issue" title="issue">July (3)</span><span class="year" tagx="year" title="year">2006</span><span class="fpage" tagx="fpage" title="fpage">1511</span><span class="lpage" tagx="lpage" title="lpage">1520</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16624908">16624908</a></span></span></li> <li tag="ref"><a name="bib0150" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">W. H. Organization</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Guidelines on the management of latent tuberculosis infection</span><span class="year" tagx="year" title="year">2015</span></span></li> <li tag="ref"><a name="bib0155" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">T.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Whole-genome sequencing to delineate Mycobacterium tuberculosis outbreaks: a retrospective observational study</span><span class="source" tagx="source" title="source">Lancet Infect. Dis.</span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">February (2)</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">137</span><span class="lpage" tagx="lpage" title="lpage">146</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23158499">23158499</a></span></span></li> <li tag="ref"><a name="bib0160" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Winston</span><span class="given-names" tagx="given-names" title="given-names">C.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitruka</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Treatment duration for patients with drug-resistant tuberculosis, United States</span><span class="source" tagx="source" title="source">Emerg. Infect. Dis.</span><span class="volume" tagx="volume" title="volume">18</span><span class="issue" tagx="issue" title="issue">July (7)</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">1201</span><span class="lpage" tagx="lpage" title="lpage">1202</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22709909">22709909</a></span></span></li> <li tag="ref"><a name="bib0165" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yruela</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Contreras-Moreira</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Magalhã es</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osó Rio</span><span class="given-names" tagx="given-names" title="given-names">N.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gonzalo-Asensio</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mycobacterium tuberculosis complex exhibits lineage-specific variations affecting protein ductility and epitope recognition</span><span class="source" tagx="source" title="source">Genome Biol. Evol.</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">12</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">3751</span><span class="lpage" tagx="lpage" title="lpage">3764</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28062754">28062754</a></span></span></li> <li tag="ref"><a name="bib0170" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuen</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Turning off the tap: stopping tuberculosis transmission through active case-finding and prompt effective treatment</span><span class="source" tagx="source" title="source">Lancet</span><span class="volume" tagx="volume" title="volume">386</span><span class="issue" tagx="issue" title="issue">10010</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">2334</span><span class="lpage" tagx="lpage" title="lpage">2343</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26515675">26515675</a></span></span></li> </ul> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>DK gratefully acknowledges support from the </p><div class="funding-source" title="funding-source">
ETH Zürich Postdoctoral Fellowship Program</div> and the <div class="funding-source" title="funding-source">
Marie Curie Actions for People COFUND</div>, and the <div class="funding-source" title="funding-source">
Swiss National Science Foundation (SNSF)</div> for generously funding her research with a Marie Heim-Vögtlin fellowship. TS is supported in part by the European Research Council under the <div class="funding-source" title="funding-source">
Seventh Framework Programme of the European Commission</div> (PhyPD grant 335529). This work was further supported by the Swiss National Science Foundation (grants 310030_166687, IZRJZ3_164171, IZLSZ3_170834 and CRSII5_177163 to S.G.), the <div class="funding-source" title="funding-source">
European Research Council</div> (309540-EVODRTB to S.G.), and <div class="funding-source" title="funding-source">
SystemsX.ch</div>. The work on the tuberculosis outbreak investigations was supported by a grant from the <div class="funding-source" title="funding-source">
Bernese Lung Association</div> (LF). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.<p /> </div> </div>  </body></html>