<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/02/27/2020.02.22.961268.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.22.961268v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="A simple magnetic nanoparticles-based viral RNA extraction method for efficient detection of SARS-CoV-2"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.22.961268"/>
  <meta name="DC.Date" content="2020-02-27"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The ongoing outbreak of the novel coronavirus disease 2019 (COVID-19) originating from Wuhan, China, draws worldwide concerns due to its long incubation period and strong infectivity. Although RT-PCR-based molecular diagnosis techniques are being widely applied for clinical diagnosis currently, timely and accurate diagnosis are still limited due to labour intensive and time-consuming operations of these techniques. To address the issue, herein we report the synthesis of poly (amino ester) with carboxyl groups (PC)-coated magnetic nanoparticles (pcMNPs), and the development of pcMNPs-based viral RNA extraction method for the sensitive detection of COVID-19 causing virus, the SARS-CoV-2. This method combines the lysis and binding steps into one step, and the pcMNPs-RNA complexes can be directly introduced into subsequent RT-PCR reactions. The simplified process can purify viral RNA from multiple samples within 20 min using a simple manual method or an automated high-throughput approach. By identifying two different regions ( ORFlab and N gene) of viral RNA, a 10-copy sensitivity and a strong linear correlation between 10 and 105 copies of SARS-CoV-2 pseudovirus particles are achieved. Benefitting from the simplicity and excellent performances, this new extraction method can dramatically reduce the turn-around time and operational requirements in current molecular diagnosis of COVID-19, in particular for the early clinical diagnosis."/>
  <meta name="DC.Contributor" content="Zhen Zhao"/>
  <meta name="DC.Contributor" content="Haodong Cui"/>
  <meta name="DC.Contributor" content="Wenxing Song"/>
  <meta name="DC.Contributor" content="Xiaoling Ru"/>
  <meta name="DC.Contributor" content="Wenhua Zhou"/>
  <meta name="DC.Contributor" content="Xuefeng Yu"/>
  <meta name="article:published_time" content="2020-02-27"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="A simple magnetic nanoparticles-based viral RNA extraction method for efficient detection of SARS-CoV-2"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;The ongoing outbreak of the novel coronavirus disease 2019 (COVID-19) originating from Wuhan, China, draws worldwide concerns due to its long incubation period and strong infectivity. Although RT-PCR-based molecular diagnosis techniques are being widely applied for clinical diagnosis currently, timely and accurate diagnosis are still limited due to labour intensive and time-consuming operations of these techniques. To address the issue, herein we report the synthesis of poly (amino ester) with carboxyl groups (PC)-coated magnetic nanoparticles (pcMNPs), and the development of pcMNPs-based viral RNA extraction method for the sensitive detection of COVID-19 causing virus, the SARS-CoV-2. This method combines the lysis and binding steps into one step, and the pcMNPs-RNA complexes can be directly introduced into subsequent RT-PCR reactions. The simplified process can purify viral RNA from multiple samples within 20 min using a simple manual method or an automated high-throughput approach. By identifying two different regions (ORFlab and N gene) of viral RNA, a 10-copy sensitivity and a strong linear correlation between 10 and 100,000 copies of SARS-CoV-2 pseudovirus particles are achieved. Benefitting from the simplicity and excellent performances, this new extraction method can dramatically reduce the turn-around time and operational requirements in current molecular diagnosis of COVID-19, in particular for the early clinical diagnosis.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.22.961268v1"/>
  <meta name="citation_id" content="2020.02.22.961268v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.22.961268v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.22.961268v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.22.961268v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.22.961268"/>
  <meta name="citation_num_pages" content="18"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.22.961268"/>
  <meta name="citation_author" content="Zhen Zhao"/>
  <meta name="citation_author_institution" content="Materials and Interfaces Center, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences"/>
  <meta name="citation_author" content="Haodong Cui"/>
  <meta name="citation_author_institution" content="Materials and Interfaces Center, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences"/>
  <meta name="citation_author_institution" content="University of Chinese Academy of Sciences"/>
  <meta name="citation_author" content="Wenxing Song"/>
  <meta name="citation_author_institution" content="Materials and Interfaces Center, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences"/>
  <meta name="citation_author" content="Xiaoling Ru"/>
  <meta name="citation_author_institution" content="Materials and Interfaces Center, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences"/>
  <meta name="citation_author" content="Wenhua Zhou"/>
  <meta name="citation_author_institution" content="Materials and Interfaces Center, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences"/>
  <meta name="citation_author_email" content="wh.zhou@siat.ac.cn"/>
  <meta name="citation_author_email" content="xf.yu@siat.ac.cn"/>
  <meta name="citation_author" content="Xuefeng Yu"/>
  <meta name="citation_author_institution" content="Materials and Interfaces Center, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences"/>
  <meta name="citation_author_email" content="wh.zhou@siat.ac.cn"/>
  <meta name="citation_author_email" content="xf.yu@siat.ac.cn"/>
  <meta name="citation_reference" content="Zhu, N.; Zhang, D.; Wang, W.; Li, X.; Yang, B.; Song, J.; Zhao, X.; Huang, B.; Shi, W.; Lu, R.; Niu, P.; Zhan, F.; Ma, X.; Wang, D.; Xu, W.; Wu, G.; Gao, G. F.; Tan, W., A Novel Coronavirus from Patients with Pneumonia in China, 2019. N Engl J Med 2020, 382:727–733."/>
  <meta name="citation_reference" content="Espy, M. J.; Uhl, J. R.; Sloan, L. M.; Buckwalter, S. P.; Jones, M. F.; Vetter, E. A.; Yao, J. D. C.; Wengenack, N. L.; Rosenblatt, J. E.; Cockerill, F. R.; Smith, T. F., Real-Time PCR in Clinical Microbiology: Applications for Routine Laboratory Testing. Clin Microbiol Rev 2006, 19 (1), 165–256."/>
  <meta name="citation_reference" content="Ng, E. K. O.; Hui, D. S.; Chan, K. C. A.; Hung, E. C. W.; Chiu, R. W. K.; Lee, N.; Wu, A.; Chim, S. S. C.; Tong, Y. K.; Sung, J. J. Y.; Tam, J. S.; Lo, Y. M. D., Quantitative Analysis and Prognostic Implication of SARS Coronavirus RNA in the Plasma and Serum of Patients with Severe Acute Respiratory Syndrome. Clin. Chem. 2020, 49 (12), 1976–1980."/>
  <meta name="citation_reference" content="Poon, L. L. M.; Chan, K. H.; Wong, O. K.; Yam, W. C.; Yuen, K. Y.; Guan, Y.; Lo, Y. M. D.; Peiris, J. S. M., Early diagnosis of SARS Coronavirus infection by real time RT-PCR. J. Clin. Virol. 2003, 28 (3), 233–238."/>
  <meta name="citation_reference" content="Xu, M.-Y.; Liu, S.-Q.; Deng, C.-L.; Zhang, Q.-Y.; Zhang, B., Detection of Zika virus by SYBR green one-step real-time RT-PCR. J. Virol. Methods 2016, 236, 93–97."/>
  <meta name="citation_reference" content="Shisong, F.; Jianxiong, L.; Xiaowen, C.; Cunyou, Z.; Ting, W.; Xing, L.; Xin, W.; Chunli, W.; Renli, Z.; Jinquan, C.; Hong, X.; Muhua, Y., Simultaneous detection of influenza virus type B and influenza A virus subtypes H1N1, H3N2, and H5N1 using multiplex real-time RT-PCR. Appl. Microbiol. Biotechnol. 2011, 90 (4), 1463–1470."/>
  <meta name="citation_reference" content="Kong, Y. Y.; Thay, C. H.; Tin, T. C.; Devi, S., Rapid detection, serotyping and quantitation of dengue viruses by TaqMan real-time one-step RT-PCR. J. Virol. Methods 2006, 138 (1), 123–130."/>
  <meta name="citation_reference" content="Lu, X.; Whitaker, B.; Sakthivel, S. K. K.; Kamili, S.; Rose, L. E.; Lowe, L.; Mohareb, E.; Elassal, E. M.; Al-sanouri, T.; Haddadin, A.; Erdman, D. D., Real-Time Reverse Transcription-PCR Assay Panel for Middle East Respiratory Syndrome Coronavirus. J Clin Microbiol 2014, 52 (1), 67–75."/>
  <meta name="citation_reference" content="Corman, V. M.; Landt, O.; Kaiser, M.; Molenkamp, R.; Meijer, A.; Chu, D. K.; Bleicker, T.; Brünink, S.; Schneider, J.; Schmidt, M. L.; Mulders, D. G.; Haagmans, B. L.; van der Veer, B.; van den Brink, S.; Wijsman, L.; Goderski, G.; Romette, J.-L.; Ellis, J.; Zambon, M.; Peiris, M.; Goossens, H.; Reusken, C.; Koopmans, M. P.; Drosten, C., Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR. Euro Surveill 2020, 25 (3), 2000045."/>
  <meta name="citation_reference" content="Chu, D. K. W.; Pan, Y.; Cheng, S. M. S.; Hui, K. P. Y.; Krishnan, P.; Liu, Y.; Ng, D. Y. M.; Wan, C. K. C.; Yang, P.; Wang, Q.; Peiris, M.; Poon, L. L. M., Molecular Diagnosis of a Novel Coronavirus (2019-nCoV) Causing an Outbreak of Pneumonia. Clin. Chem. 2020."/>
  <meta name="citation_reference" content="Tang, Y.; Anne Hapip, C.; Liu, B.; Fang, C. T., Highly sensitive TaqMan RT-PCR assay for detection and quantification of both lineages of West Nile virus RNA. J. Clin. Virol. 2006, 36 (3), 177–182."/>
  <meta name="citation_reference" content="Chan, Y. R.; Morris, A., Molecular diagnostic methods in pneumonia. Curr Opin Infect Dis 2007, 20 (2), 157–164."/>
  <meta name="citation_reference" content="Schrader, C.; Schielke, A.; Ellerbroek, L.; Johne, R., PCR inhibitors – occurrence, properties and removal. J Appl Microbiol 2012, 113 (5), 1014–1026."/>
  <meta name="citation_reference" content="citation_title=In PCR Detection of Microbial Pathogens;citation_pages=17-48;citation_year=2013"/>
  <meta name="citation_reference" content="Pichl, L.; Heitmann, A.; Herzog, P.; Oster, J.; Smets, H.; Schottstedt, V., Magnetic bead technology in viral RNA and DNA extraction from plasma minipools. Transfusion 2005, 45 (7), 1106–1110."/>
  <meta name="citation_reference" content="Váradi, C.; Lew, C.; Guttman, A., Rapid Magnetic Bead Based Sample Preparation for Automated and High Throughput N-Glycan Analysis of Therapeutic Antibodies. Anal. Chem. 2014, 86 (12), 5682–5687."/>
  <meta name="citation_reference" content="Riemann, K.; Adamzik, M.; Frauenrath, S.; Egensperger, R.; Schmid, K. W.; Brockmeyer, N. H.; Siffert, W., Comparison of manual and automated nucleic acid extraction from whole-blood samples. J Clin Lab Anal 2007, 21 (4), 244–248."/>
  <meta name="citation_reference" content="Song, W.; Su, X.; Gregory, D. A.; Li, W.; Cai, Z.; Zhao, X., Magnetic Alginate/Chitosan Nanoparticles for Targeted Delivery of Curcumin into Human Breast Cancer Cells. Nanomaterials 2018, 8 (11), 907."/>
  <meta name="citation_reference" content="Sunshine, J.; Green, J. J.; Mahon, K. P.; Yang, F.; Eltoukhy, A. A.; Nguyen, D. N.; Langer, R.; Anderson, D. G., Small-Molecule End-Groups of Linear Polymer Determine Cell-type Gene-Delivery Efficacy. Adv Mater 2009, 21 (48), 4947–4951."/>
  <meta name="twitter:title" content="A simple magnetic nanoparticles-based viral RNA extraction method for efficient detection of SARS-CoV-2"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The ongoing outbreak of the novel coronavirus disease 2019 (COVID-19) originating from Wuhan, China, draws worldwide concerns due to its long incubation period and strong infectivity. Although RT-PCR-based molecular diagnosis techniques are being widely applied for clinical diagnosis currently, timely and accurate diagnosis are still limited due to labour intensive and time-consuming operations of these techniques. To address the issue, herein we report the synthesis of poly (amino ester) with carboxyl groups (PC)-coated magnetic nanoparticles (pcMNPs), and the development of pcMNPs-based viral RNA extraction method for the sensitive detection of COVID-19 causing virus, the SARS-CoV-2. This method combines the lysis and binding steps into one step, and the pcMNPs-RNA complexes can be directly introduced into subsequent RT-PCR reactions. The simplified process can purify viral RNA from multiple samples within 20 min using a simple manual method or an automated high-throughput approach. By identifying two different regions ( ORFlab and N gene) of viral RNA, a 10-copy sensitivity and a strong linear correlation between 10 and 105 copies of SARS-CoV-2 pseudovirus particles are achieved. Benefitting from the simplicity and excellent performances, this new extraction method can dramatically reduce the turn-around time and operational requirements in current molecular diagnosis of COVID-19, in particular for the early clinical diagnosis."/>
  <meta name="og-title" property="og:title" content="A simple magnetic nanoparticles-based viral RNA extraction method for efficient detection of SARS-CoV-2"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.22.961268v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The ongoing outbreak of the novel coronavirus disease 2019 (COVID-19) originating from Wuhan, China, draws worldwide concerns due to its long incubation period and strong infectivity. Although RT-PCR-based molecular diagnosis techniques are being widely applied for clinical diagnosis currently, timely and accurate diagnosis are still limited due to labour intensive and time-consuming operations of these techniques. To address the issue, herein we report the synthesis of poly (amino ester) with carboxyl groups (PC)-coated magnetic nanoparticles (pcMNPs), and the development of pcMNPs-based viral RNA extraction method for the sensitive detection of COVID-19 causing virus, the SARS-CoV-2. This method combines the lysis and binding steps into one step, and the pcMNPs-RNA complexes can be directly introduced into subsequent RT-PCR reactions. The simplified process can purify viral RNA from multiple samples within 20 min using a simple manual method or an automated high-throughput approach. By identifying two different regions ( ORFlab and N gene) of viral RNA, a 10-copy sensitivity and a strong linear correlation between 10 and 105 copies of SARS-CoV-2 pseudovirus particles are achieved. Benefitting from the simplicity and excellent performances, this new extraction method can dramatically reduce the turn-around time and operational requirements in current molecular diagnosis of COVID-19, in particular for the early clinical diagnosis."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-02-27"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>A simple magnetic nanoparticles-based viral RNA extraction method for efficient detection of SARS-CoV-2 | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1170607 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">1 Abstract</h2>
    <p id="p-3">The ongoing outbreak of the novel coronavirus disease 2019 (COVID-19) originating from Wuhan, China, draws worldwide concerns due to its long incubation period and strong infectivity. Although RT-PCR-based molecular diagnosis techniques are being widely applied for clinical diagnosis currently, timely and accurate diagnosis are still limited due to labour intensive and time-consuming operations of these techniques. To address the issue, herein we report the synthesis of poly (amino ester) with carboxyl groups (PC)-coated magnetic nanoparticles (pcMNPs), and the development of pcMNPs-based viral RNA extraction method for the sensitive detection of COVID-19 causing virus, the SARS-CoV-2. This method combines the lysis and binding steps into one step, and the pcMNPs-RNA complexes can be directly introduced into subsequent RT-PCR reactions. The simplified process can purify viral RNA from multiple samples within 20 min using a simple manual method or an automated high-throughput approach. By identifying two different regions (
     <em>ORFlab</em> and 
     <em>N</em> gene) of viral RNA, a 10-copy sensitivity and a strong linear correlation between 10 and 10
     <sup>5</sup> copies of SARS-CoV-2 pseudovirus particles are achieved. Benefitting from the simplicity and excellent performances, this new extraction method can dramatically reduce the turn-around time and operational requirements in current molecular diagnosis of COVID-19, in particular for the early clinical diagnosis.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">2 Introduction</h2>
    <p id="p-4">In December, 2019, an unknown pathogen-mediated pneumonia emerged in Wuhan, China.
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
     </sup> Confirmed cases had been soon diagnosed in other cities in China, as well as in other countries. On 11 February 2020, as reported by the World Health Organization (WHO) and the Chinese Center for Disease Control and Prevention (China CDC), this unknown pneumonia was confirmed to be caused by a novel coronavirus (SARS-CoV-2, previously named as 2019-nCoV). SARS-CoV-2 is etiologically related to the well-known severe acute respiratory syndrome coronavirus (SARS-CoV), belonging to the 
     <em>Coronaviridae</em> family, a type of positive-sense, single-stranded RNA coronavirus with an outer envelope. Although the genome sequences of SARS-CoV-2 have been fully revealed and various RT-PCR-based detection kits have been developed, clinical diagnosis of COVID-19 is still highly challenging.
    </p>
    <p id="p-5">Due to its high sensitivity in exponentially amplifying RNA molecules, reverse transcription polymerase chain reaction (RT-PCR) is identified as a standard and routinely used technique for the analysis and quantification of various pathogenic RNA in laboratories and clinical diagnosis.
     <sup>
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </sup> For example, it has been successfully applied for the detection of SARS-CoV, Middle East respiratory syndrome coronavirus (MERS-CoV), and various other viral pathogens including Zika virus (ZIKV), Influenza A virus, Dengue virus (DENV).
     <sup>
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>-
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
     </sup> After the outbreak of COVID-19, several methods and kits based on RT-PCR for the detection of SARS-CoV-2 genomic RNA have been reported.
     <sup>
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>-
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>
     </sup> While RT-PCR-based methods have been widely used in COVID-19 diagnosis, their application in accurate diagnosis of viral infection and epidemic control is severely hampered by their laborious and time-consuming sample processing steps.
    </p>
    <p id="p-6">Quantitative extraction of nucleic acids with high purity from complex samples are the prerequisite for efficient RT-PCR assays. Low extraction efficiency might give poor signals during exponential amplification and thus result in false negative results.
     <sup>
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>-
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>
     </sup> Low extraction quality, on the other hand, may contain a variety of PCR inhibitors, which gives unreliable readouts during amplification.
     <sup>
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>-
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>
     </sup>
    </p>
    <p id="p-7">In the control and diagnosis towards SARS-CoV-2 currently, silica-based spin column RNA extraction methods are widely used, in which a silica membrane or glass fiber is applied to bind nucleic acids. In these traditional methods, the samples need pre-lysis in an appropriate buffer to release nucleic acids from viral particles before binding to the column membrane and multiple centrifugation steps are required to enable binding, washing and elution of extracted nucleic acids. Additionally, as various corrosive chaotropic salts and toxic organic solvents are involved in the lysis and binding steps, several sequential washing steps are required to eliminate possible PCR inhibitory effects in the eluted products. The whole process comprises multiple centrifuging and column-transferring steps, which is laborious, time-consuming, and vulnerable to contamination or column clogging. More importantly, spin column-based approaches are not suitable for a high-throughput, automated operation. In the monitoring and control of sudden outbreaks, such as SARS-CoV-2, these traditional methods consume a large number of operators, but giving low diagnosis efficiency and high risk of cross infection. Thus, fast, convenient and automated nucleic acids extraction methods are highly desirable not just in the molecular diagnosis of SARS-CoV-2, but also in the monitoring and prevention of other infectious diseases.</p>
    <p id="p-8">As an alternative, magnetic nanoparticles (MNPs)-based extraction methods are centrifuge-free and has proven to be easy to operate and compatible to automation and high-throughput operation.
     <sup>
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>-
      <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>
     </sup> In conventional MNPs-based methods, nucleic acids in the lysed samples can be specifically absorbed on MNPs due to various surface-modified functional groups. In the presence of magnetic fields, nucleic acids are rapidly separated from most impurities in the supernatant. After fast washing steps to eliminate trace impurities, purified nucleic acids can be further released from the surface of MNPs by elution buffer with altered ionic strength. Although much simpler and faster than spin column-based methods, most MNPs-based extraction strategies still contain multiple processing steps such as lysis, binding, washing and elution, which increases operational difficulties in real clinical diagnosis.
    </p>
    <p id="p-9">Herein, a further simplified and updated MNPs-based viral RNA extraction method is described for highly sensitive extraction and RT-PCR detection of viral RNA using SARS-CoV-2 pseudoviruses as models. The MNPs is synthesized via a simple one-pot approach, and functionalized with polymers carrying multi-carboxyl groups by following one-step incubation. Due to the strong interaction between carboxyl groups and nucleic acids, the poly carboxyl-functionalized MNPs (pcMNPs) enables rapid and efficient absorption of RNA molecules. In RNA extraction, one lysis/binding step and one washing step are required for nucleic acids extraction and purification from complex samples. More importantly, extracted RNA can be directed introduced into subsequent RT-PCR together with MB without elution step, which dramatically reduce operating time and risk of contamination. The fast extraction method was verified by both manual operation and automated system. Due to its simplicity, satisfactory performances and robustness, this method provides a promising alternative to decrease the labour-intensity and reduce possibility of false-negative results in current RT-PCR-based SARS-CoV-2 diagnosis.</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">3 Material and methods</h2>
    <div id="sec-3" class="subsection">
     <h3>Chemicals</h3>
     <p id="p-10">Iron (III) chloride hexahydrate, Iron (II) chloride tetrahydrate, ammonium hydroxide, tetraethyl orthosilicate (TEOS), (3-Aminopropyl)triethoxysilane (APTES), and dimethyl sulfoxide (DMSO) were bought from Aladdin Industrial Corporation (Shanghai, China). Ammonium hydroxide solution, isopropanol, ethylenediamine and ethanol were purchased from Sinopharm Chemical Reagent Co., Ltd. (Shanghai, China). 1,4-butanediol diacrylate, 6-amino caproic acid, sodium chloride (NaCl), sodium iodide (NaI), tri(hydroxymethyl)aminomethane (Tris), ethylene diamine tetraacetic acid (EDTA) and polyethylene glycol 8000 were obtained from Sigma-Aldrich (St. Louis, USA).</p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Preparation of amino-modified magnetic nanoparticles (NH
      <sub>2</sub>-MNPs)
     </h3>
     <p id="p-11">Bare magnetic nanoparticles (MNPs) were prepared based on a simple co-precipitation protocol as previously reported.
      <sup>
       <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>
      </sup> Briefly, 3.0 g of Iron (III) chloride hexahydrate and 2.5 g of Iron (II) chloride tetrahydrate were dissolved separately in 100 mL of deionized water and degassed with nitrogen for 20 min to remove the oxygen in the solutions. Both iron solutions were then mixed in a 500 mL round-bottom flask, and 10 mL of ammonium hydroxide was added into the mixture with vigorous stirring under a nitrogen atmosphere. A rapid change of solution colour was observed from orange to black, indicating the formation of co-precipitated bare MNPs. The solution mixture was continuously stirred for another 4 h, and the resulting black products (bare MNPs) were collected with a magnet and dispersed into ethanol after washing several times with deionized water and ethanol. Subsequently, 0.3 g of as-prepared bare MNPs, 15 mL of deionized water and 12 mL of ammonium hydroxide were added into 150 mL of ethanol, followed by a continuous sonication for 30 min at room temperature. 1.2 mL of TEOS was then added dropwise into the solution mixture after sonication, and vigorously stirred for another 4 h at room temperature to allow the formation of silica layers on the surface of MNPs. Afterwards, the silica-coated MNPs were collected with a magnet and the rinsed with deionized water and ethanol for several times to remove residual TEOS. After that, 0.2 g of silica-coated MNPs were dispersed into 50 mL isopropanol, and 0.2 mL of APTES was dropwise mixed with MNPs solution. The mixture was incubated under continuous sonication for 6 h at room temperature, followed by the collection of amino-modified MNPs (NH
      <sub>2</sub>-MNPs) with a magnet and washing with deionized water and ethanol to remove free APTES. The final prepared NH
      <sub>2</sub>-MNPs were preserved in ethanol at 4 °C and the loading of -NH
      <sub>2</sub> group on the surface was verified by Fourier transform infrared (FTIR) spectroscopy (FigureS1A).
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Synthesis and characterization of 
      <span class="underline">p</span>oly (amino ester) with multiple 
      <span class="underline">c</span>arboxyl groups (PC)
     </h3>
     <p id="p-12">As shown in 
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Scheme1 A</a>, polymers were prepared based on the protocol reported previously with minor modification.
      <sup>
       <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>
      </sup> In brief, 3.5 g of 1,4-butanediol diacrylate (17.7 mmol) and 1.5 g of 6-amino caproic acid (11.4 mmol) were firstly mixed in 10 mL of 50% (v/v) DMSO aqueous solution, followed by vigorously stirring for 12 h at 90 °C in a safe dark place. Subsequently, end-capping reaction was performed by the addition of 10% (v/v) amino solution, ethylenediamine, into the polymer/DMSO solution. The as-prepared PC polymers was final preserved at 4 °C with lightproof package. PCs were further characterized by FTIR spectroscopy (FigureS1B).
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Preparation and characterization of PC-coated NH
      <sub>2</sub>-MNPs (pcMNPs)
     </h3>
     <p id="p-13">25 mg of NH
      <sub>2</sub>-MNPs were dispersed into 25 mL of 50% (v/v) DMSO aqueous solution and then mixed with 1.25 g of PC under sonication for 10 min. Subsequently, 2.5 mL of NaOH solution (1 M) was introduced to the mixture and sonicated for another 20 min, followed by vigorously shaking for 4 h at 37 °C and rinsing with DMSO and deionized water for several times. The obtained polymer-coated MNPs (pcMNPs) were stored in deionized water at 4 °C. The size and morphology of the pcMNPs were confirmed by transmission electron microscopy (TEM).
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Common nucleic acid extraction</h3>
     <p id="p-14">The pseudovirus samples were obtained from Zeesan Biotech (Xiamen, China) and the standard samples were freshly prepared by step-wise dilution of pseudovirus in fetal calf serum purchased from Thermofisher (Massachusetts, USA) before nucleic acid extraction experiments. 200 μL of as-prepared standard samples with a known copy number of viral particles (down to 10 copies) were incubated with 400 μL lysis/binding buffer (1 M NaI, 2.5 M NaCl, 10% Triton X-100, 40% polyethylene glycol 8000, 25 mM EDTA) and 40 μg pcMNPs for 10 min at room temperature on a rotating shaker. Then, the pcMNPs-RNA complex was collected magnetically for 1 min and the supernatant was discarded. Afterwards, the complex was washed once or twice with 400 μL washing buffer (75% ethanol v/v). The purified nucleic acids were released from MNPs by incubating the complex in 50 μL of TE buffer (10 mM Tris-HCl, pH 8.0), vortexed at 55 °C bath for 5 min, and 15 μL of supernatant was transferred to subsequent RT-PCR reaction.</p>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Conventional Real-time reverse-transcription PCR</h3>
     <p id="p-15">To quantify the amount of viral RNA captured, Real-time reverse-transcription PCR using VetMAX™-Plus One-Step RT-PCR Kit (Massachusetts, USA) and Bio-Red PCR detection system (California, USA) was applied. For conventional RT-PCR, a 30 μL of reaction solution was set up containing 15 μL of eluted supernatant, 7.5 μL of 4 × reaction buffer, 1.5 μL enzyme mixture, and optimized concentrations of primers (from 0.25 μM to 1 μM)and probe (from 0.25 μM to 1 μM). Subsequent thermal cycling was performed at 55°C for 15 min for reverse transcription, followed by 95°C for 30 s and then 45 cycles of 95°C for 10 s, 60°C for 35 s. Fluorescence readout were taken after each cycle, and the threshold cycle (Ct) was calculated by the Bio-Red analysis system based on plotting against the log10 fluorescence intensity. The oligonucleotide primers and probes were synthesized by GENEWIZ (Suzhou, China), and corresponding sequences are shown in 
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1174706/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1174706/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1174706" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <span class="caption-title">Sequences of primers and probes for RT-PCR.</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Direct RT-PCR amplification and detection of the extracted pcMNP-RNA complex</h3>
     <p id="p-16">To investigate whether the extracted pcMNPs-RNA complexes can be directly used for RT-PCR, an identical extraction was carried out, in which the whole complex was transferred to the PCR tube without the elution step. Specifically, 30 μL of reaction solution composed of 15 μL of TE buffer, 7.5 μL of 4 × reaction buffer, 1.5 μL enzyme mixture, and optimal concentrations of primers (1 μM each) and probe (1 μM) was mixed with pcMNP-RNA complexes by brief vortexing, and directly transferred to the PCR tube for RT-PCR using the same procedure as abovementioned.</p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>Development of an automated protocol for RNA extraction</h3>
     <p id="p-17">A high throughput automated RNA extraction method was adapted using a commercial NP968-C automatic nucleic acid extraction system (TIANLONG, Xi’an, China), which could simultaneously process up to 32 parallel samples in 96-well sample plates. In detail, 400 μL of lysis/binding buffer, 40 μg pcMNPs and 200 μL of the sample containing a specific number of pseudovirus particles was sequentially dispensed into Column 1 of the sample plate. Then the washing buffer and elution buffer was added separately into Column 2 or 4. Finally, the 96-well sample plate was plugged onto the matrix and RNA extraction was performed by following an optimized program (TableS1). Once the program was finished, the 96-well sample plate was removed and the 15 μL of the eluted product was analysed using the conventional RT-PCR protocol as described above.</p>
    </div>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">4 Results and discussion</h2>
    <div id="sec-12" class="subsection">
     <h3>Synthesis and characterization of pcMNPs</h3>
     <p id="p-18">The pcMNPs were prepared by two steps as depicted in 
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Scheme1B</a>. It started with one-pot synthesis of NH
      <sub>2</sub>-MNPs using co-precipitation reaction and hydrolysis of TEOS/APTES. Once PC polymer was successfully synthesized, its reaction with NH
      <sub>2</sub>-MNPs would form pcMNPs by the Michael addition to efficiently give pcMNPs. To investigate the morphology of prepared pcMNPs, transmission electron microscopy (TEM) was used and the representative image was shown in 
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure1A</a>, suggesting a spherical morphology of the prepared pcMNPs. In addition, enlarged TEM image of pcMNPs revealed a thin silica-layer on the particle surfaces with a thickness around 3 nm, implied the successful preparation of silica-coating. By using the dynamic light scattering technique, it is observed that the prepared pcMNPs have an average diameter of 10.22 ± 2.8 nm without apparent aggregation (
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure1B</a>). To verify the successful functionalization of synthesized PC polymers, the average zeta potentials of bare MNPs, silica-coated MNPs and pcMNPs were measured by DLS. As shown in 
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig 1C</a>, the average zeta potential changed from −4.82 mV ± 0.54 of bare MNPs to 20.57 mV ± 0.71 of silica-coated MNPs due to the NH
      <sub>2</sub>-groups introduced during silica-coating. However, functionalization of PC polymers resulted in a decrease of average zeta potential of pcMNPs to −38.67 mV ± 0.84, indicating successful polymer coating and more importantly, good dispersion of pcMNPs in the solution. Subsequently, the magnetic response property of bare MNPs, NH
      <sub>2</sub>-MNPs and pcMNPs were tested (
      <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure1D</a>). The magnetically capture process was completely finished within 30 s, implying excellent paramagnetic property of the prepared pcMNPs.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Schematic illustration for (A) the synthesis of PC polymer, and (B) the preparation of pcMNP." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-546042415" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Schematic illustration for (A) the synthesis of PC polymer, and (B) the preparation of pcMNP.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Scheme1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F1.medium.gif" width="440" height="207"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Scheme1" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F1.medium.gif" width="440" height="207"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Scheme1" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Scheme1</span>
       <p id="p-19" class="first-child">Schematic illustration for (A) the synthesis of PC polymer, and (B) the preparation of pcMNP.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Characterization of pcMNPs. (A) TEM image and (B) size distribution of pcMNPs; The average size of nanoparticles is represented as mean [[#xB1]] SD of over 250 individual particles. (C) Zeta potentials of bare MNPs, NH2-MNPs and pcMNPs in water. (D) The magnetic response property of bare MNPs, NH2-MNPs and MBs in 30 s." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-546042415" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Characterization of pcMNPs.&lt;;/span&gt;; (A) TEM image and (B) size distribution of pcMNPs; The average size of nanoparticles is represented as mean [[#xB1]] SD of over 250 individual particles. (C) Zeta potentials of bare MNPs, NH&lt;;sub&gt;;2&lt;;/sub&gt;;-MNPs and pcMNPs in water. (D) The magnetic response property of bare MNPs, NH&lt;;sub&gt;;2&lt;;/sub&gt;;-MNPs and MBs in 30 s.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F2.medium.gif" width="440" height="335"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure1" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F2.medium.gif" width="440" height="335"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure1" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure1</span>
       <span class="caption-title">Characterization of pcMNPs.</span>
       <p id="p-20" class="first-child">(A) TEM image and (B) size distribution of pcMNPs; The average size of nanoparticles is represented as mean ± SD of over 250 individual particles. (C) Zeta potentials of bare MNPs, NH
        <sub>2</sub>-MNPs and pcMNPs in water. (D) The magnetic response property of bare MNPs, NH
        <sub>2</sub>-MNPs and MBs in 30 s.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-13" class="subsection">
     <h3>RNA binding property of pxMNPs</h3>
     <p id="p-21">To investigate the RNA binding property of prepared pxMNPs, RNA binding assays were performed by incubating 2 μg RNA molecules with 20 μg pcMNPs in 200 μL lysis/binding buffer. As shown in 
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure2A</a>, more than 90% of RNA was absorbed by the pcMNPs within 10 min. In contrast, significant amount RNA molecules have been observed using a commercialized RNA extraction kit under similar conditions. More importantly, no aggregation has been observed when incubating pxMNPs in a high-salt solution (50 mM NaCl), further confirmed their excellent dispersity (
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure2B</a>).
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="RNA binding and dispersion property of pcMNPs. (A) RNA binding affinity of pcMNPs analysed by native PAGE; (B) Dispersion stability of pcMNPs in a high-salt solution (50 mM NaCl)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-546042415" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;RNA binding and dispersion property of pcMNPs.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) RNA binding affinity of pcMNPs analysed by native PAGE; (&lt;;strong&gt;;B&lt;;/strong&gt;;) Dispersion stability of pcMNPs in a high-salt solution (50 mM NaCl).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F3.medium.gif" width="440" height="111"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure2" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F3.medium.gif" width="440" height="111"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure2" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure2</span>
       <span class="caption-title">RNA binding and dispersion property of pcMNPs.</span>
       <p id="p-22" class="first-child">(
        <strong>A</strong>) RNA binding affinity of pcMNPs analysed by native PAGE; (
        <strong>B</strong>) Dispersion stability of pcMNPs in a high-salt solution (50 mM NaCl).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-14" class="subsection">
     <h3>Optimization of SARS-CoV-2 viral RNA extraction and amplification</h3>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="A schematic representation of the pcMNP-based viral RNA extraction method. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-546042415" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;A schematic representation of the pcMNP-based viral RNA extraction method.&lt;;/span&gt;; &lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Scheme2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F4.medium.gif" width="440" height="148"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Scheme2" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F4.medium.gif" width="440" height="148"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Scheme2" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Scheme2</span>
       <span class="caption-title">A schematic representation of the pcMNP-based viral RNA extraction method.</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-23">By following a simple lysis/binding-washing-elution protocol shown in 
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Scheme2</a>, 10
      <sup>5</sup> copies of SARS-CoV-2 pseudovirus in 200 μL serum samples were extracted and subject to RT-PCR analysis. We first carried out a concentration optimization of the primer and probe to maximize the efficiency and sensitivity of SARS-CoV-2 detection. The results showed that the optimal concentrations were 1 μM and 1 μM for the primer pairs and probes respectively (FigureS2). Subsequently, the optimal amount of pcMNPs in viral RNA extraction were evaluated. Considering the high viral load in clinical samples, although 20 μg of pcMNPs were enough to extract 10
      <sup>5</sup> copies of SARS-CoV-2 pseudovirus RNA in 200 μL serum samples (FigureS3), 40 μg pcMNPs was used in subsequent experiments.
     </p>
    </div>
    <div id="sec-15" class="subsection">
     <h3>Applications of pcMNPs in the detection of SARS-CoV-2 viral RNA using Direct RT-PCR</h3>
     <p id="p-24">Due to the excellent water dispersity of pcMNPs, we hypothesized that they can stay dispersed in the first few cycles of RT-PCR reaction, without shielding the extracted RNA molecules in aggregates from primer binding and elongation. Thus, by following the optimized conditions, performances of the pcMNPs-based RNA extraction and Direct RT-PCR was investigated. 10
      <sup>5</sup> copies of pseudoviruses were spiked into 200 μL of serum and the extracted viral RNA were analysed by RT-PCR using eluted products (Conventional RT-PCR) or pcMNPs-RNA complexes without elution (Direct RT-PCR). In these experiments, the serum sample without pseudoviruses was used as a negative control, while the PCR reaction mixture directly spiked with 10
      <sup>5</sup> copies of pseudoviruses was regarded as a positive control. As shown in 
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure3A</a>, in the detection of ORFlab (Open Reading Frame 1ab) region, the direct RT-PCR without elution step exhibited a slightly smaller cycle threshold (Ct) value than that measured using Conventional RT-PCR after elution. This is possibly because all extracted viral RNA was introduced to the amplification in the Direct RT-PCR, while only half of the eluted products was added for the Conventional RT-PCR. This phenomenon suggests that one advantages of the pcMNPs-based Direct RT-PCR protocol is to reduce possible false negative results, since all extracted viral RNA has been directed to the subsequent amplification without potential lost in the elution and transfer steps. Additionally, there is no detectable differences between amplification efficiencies of the positive control and Direct RT-PCR (
      <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure3A</a> red and purple lines). This result suggests that our pcMNPs-based viral RNA extraction protocol not only exhibits nearly 100% RNA extraction efficiency in serum samples, but also provides high-purity products without PCR inhibitors.
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="RT-PCR assays towards viral RNA extracted by different pcMNPs-based methods. RT-PCR assays amplifying (A) the ORF1ab region and (B) the N gene in pseudoviral RNA extracted by a manual protocol." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-546042415" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;RT-PCR assays towards viral RNA extracted by different pcMNPs-based methods.&lt;;/span&gt;; RT-PCR assays amplifying (A) the ORF1ab region and (B) the &lt;;em&gt;;N&lt;;/em&gt;; gene in pseudoviral RNA extracted by a manual protocol.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F5.medium.gif" width="440" height="152"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure3" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F5.medium.gif" width="440" height="152"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure3" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure3</span>
       <span class="caption-title">RT-PCR assays towards viral RNA extracted by different pcMNPs-based methods.</span>
       <p id="p-25" class="first-child">RT-PCR assays amplifying (A) the ORF1ab region and (B) the 
        <em>N</em> gene in pseudoviral RNA extracted by a manual protocol.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-16" class="subsection">
     <h3>Automated viral RNA extraction based on pcMNPs</h3>
     <p id="p-26">As previously described, one of the most serious disadvantages of traditional column-based extraction approaches is the difficulties in automation. Because no centrifugation steps are required, MNPs-based methods allow fully automated nucleic acid purification, which is highly important in current SARS-CoV-2 diagnosis. Therefore, the feasibility of automating viral RNA extraction procedure based on our pcMNPs was subsequently evaluated. A commercialized magnetic rods-based nucleic acid purification system was used. An automated programme was set according to the manual protocol (
      <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure4A</a>). During the extraction process, although the shaking pattern of magnetic rods was set at the most vigorous level, no breakage or leakage of pcMNPs were observed, since the eluted solution are colourless and transparent (FigureS4). As shown in 
      <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure4B</a>, the amplification curve of the automated sample is very close to that of the positive control and manually performed Direct RT-PCR samples, which suggests that our pcMNPs-based method is highly suitable for the automated high throughput viral RNA extraction.
     </p>
     <div id="F6" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Automated viral RNA extraction based on pcMNPs. (A) A schematic diagram of automated extraction protocol. (B) RT-PCR assays amplifying the ORF1ab region in pseudoviral RNA extracted by automated protocol, and (B) RT-PCR assays amplifying the N gene in pseudoviral RNA extracted by an automated protocol." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-546042415" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Automated viral RNA extraction based on pcMNPs.&lt;;/span&gt;; (&lt;;strong&gt;;A&lt;;/strong&gt;;) A schematic diagram of automated extraction protocol. (B) RT-PCR assays amplifying the ORF1ab region in pseudoviral RNA extracted by automated protocol, and (B) RT-PCR assays amplifying the &lt;;em&gt;;N&lt;;/em&gt;; gene in pseudoviral RNA extracted by an automated protocol.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F6.medium.gif" width="429" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure4" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F6.medium.gif" width="429" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure4" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure4</span>
       <span class="caption-title">Automated viral RNA extraction based on pcMNPs.</span>
       <p id="p-27" class="first-child">(
        <strong>A</strong>) A schematic diagram of automated extraction protocol. (B) RT-PCR assays amplifying the ORF1ab region in pseudoviral RNA extracted by automated protocol, and (B) RT-PCR assays amplifying the 
        <em>N</em> gene in pseudoviral RNA extracted by an automated protocol.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-28">Then, the performances of the pcMNPs-based method in extracting and detecting the 
      <em>Nucleocapsid</em> (
      <em>N</em>) gene were evaluated. In agreement with the results of ORFlab region, the 
      <em>N</em> gene assays also confirmed the high extraction efficiency and robustness of pcMNPs-based method in both manually operated Direct RT-PCR and automated protocol (
      <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure4C</a>).
     </p>
    </div>
    <div id="sec-17" class="subsection">
     <h3>Sensitivity and dynamic range of pcMNPs-based viral RNA detection</h3>
     <p id="p-29">Then, the sensitivity and dynamic range of pcMNPs-based viral RNA detection method was evaluated and compared by following the Conventional and Direct RT-PCR protocols, using 
      <em>N</em> gene carrying pseudovirus as a model. A series of standard samples containing 10
      <sup>5</sup> to 10 copies of pseudovirus were freshly prepared by step-wise 10-fold dilution of in serum. A control serum sample (no pseudovirus) was also included as a negative control to monitor the presence of false positive signals. As shown in 
      <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 5A</a>, a detection limit of 10 copies of pseudovirus was achieved using Convectional RT-PCR protocol. A strong linear relationship between the logarithm of pseudovirus particle numbers and the corresponding Ct values was found crossing over 5 orders of magnitude ranging from 10 to 10
      <sup>5</sup> copies with a high correlation coefficient (R
      <sup>2</sup> = 0.999) (
      <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure5B</a>). Parallelly, similar detection limit and linear relationship were observed in experiments following the pcMNPs-based Direct RT-PCR protocol (
      <a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure5C</a> and 
      <a id="xref-fig-7-4" class="xref-fig" href="#F7">D</a>). Meanwhile, the differences of Ct values between neighbouring curves are approximately 3 cycles, which is very close to the theoretical 3.3 cycles for a 10-fold dilution. These results indicated that the RT-PCR amplification is highly efficient both in the presence or absence of pcMNPs. However, it is also important to note that in both cases, the negative controls sometimes gave observable amplification signals. Although their Ct values are lagged too far behind 40 cycles to be regarded as a valid positive result, this phenomenon raise concerns about possible false-positive issues, in which further optimization of primer pairs and probe might be necessary.
     </p>
     <div id="F7" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Sensitivity and dynamics range of pcMNPs-based viral RNA detection. (A) RT-PCR assays amplifying the N gene in pseudoviral RNA following the Conventional RT-PCR protocol, and (B) corresponding calibration curve. The red line is the linear regression fit (R2 = 0.999). (C) RT-PCR assays amplifying the N gene in pseudoviral RNA following the Direct RT-PCR protocol, and (D) corresponding calibration curve. The red line is the linear regression fit (R2 = 0.998)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-546042415" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Sensitivity and dynamics range of pcMNPs-based viral RNA detection.&lt;;/span&gt;; (A) RT-PCR assays amplifying the &lt;;em&gt;;N&lt;;/em&gt;; gene in pseudoviral RNA following the Conventional RT-PCR protocol, and (B) corresponding calibration curve. The red line is the linear regression fit (&lt;;em&gt;;R&lt;;/em&gt;;&lt;;sup&gt;;2&lt;;/sup&gt;; = 0.999). (C) RT-PCR assays amplifying the &lt;;em&gt;;N&lt;;/em&gt;; gene in pseudoviral RNA following the Direct RT-PCR protocol, and (D) corresponding calibration curve. The red line is the linear regression fit (&lt;;em&gt;;R&lt;;/em&gt;;&lt;;sup&gt;;2&lt;;/sup&gt;; = 0.998).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F7.medium.gif" width="440" height="335"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure5" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F7.medium.gif" width="440" height="335"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure5" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.961268/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure5</span>
       <span class="caption-title">Sensitivity and dynamics range of pcMNPs-based viral RNA detection.</span>
       <p id="p-30" class="first-child">(A) RT-PCR assays amplifying the 
        <em>N</em> gene in pseudoviral RNA following the Conventional RT-PCR protocol, and (B) corresponding calibration curve. The red line is the linear regression fit (
        <em>R</em>
        <sup>2</sup> = 0.999). (C) RT-PCR assays amplifying the 
        <em>N</em> gene in pseudoviral RNA following the Direct RT-PCR protocol, and (D) corresponding calibration curve. The red line is the linear regression fit (
        <em>R</em>
        <sup>2</sup> = 0.998).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-31">Overall, the updated MB-based extraction method had highly extraction efficiency and compatibility of PCR amplification in any of the patterns, which dramatically simplified laborious sample processing work and was ideally suitable for RT-PCR assay of SARS-CoV-2 with a sensitivity of 10 copies at least.</p>
    </div>
   </div>
   <div class="section" id="sec-18">
    <h2 class="">5 Conclusions</h2>
    <p id="p-32">Efficient and robust nucleic acids extraction from complex clinical samples is the first and the most important step for subsequent molecular diagnosis, but currently it is still highly labour intensive and time-consuming. For example, although the genome sequences of SARS-CoV2 have been fully revealed and various RT-PCR-based detection kits have been developed, timely diagnosis of COVID-19 is still highly challenging partially due to the lack of satisfactory viral RNA extraction strategy. In this study, a carboxyl polymer-coated MNPs, namely pcMNPs, was developed and a simple but efficient viral RNA extraction system was established for sensitive detection of SARS-CoV-2 RNA via RT-PCR. As compared with traditional column-based nucleic acids extraction methods, our pcMNPs-based method has several advantages (
     <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Firstly, pcMNPs-based method combines the virus lysis and RNA binding steps into one, and the pcMNPs-RNA complexes can be directly introduced into subsequent RT-PCR reactions (Direct RT-PCR), which gives a dramatically simplified RNA extraction protocol. Secondly, pcMNPs have excellent viral RNA binding performances, which results in 10-copy sensitivity and the high linearity over 5 logs of gradient in SARS-CoV-2 viral RNA detection using RT-PCR. Thirdly, this method can be easily adopted in fully automated nucleic acid extraction systems without laborious optimization. Furthermore, the pcMNPs-RNA complexes obtained by this method is also compatible with various isothermal amplification methods, such as RPA and LAMP, and thus could be used in the development of POCT devices. In conclusion, due to its simplicity, robustness, and excellent performances, our pcMNPs-based method may provide a promising alternative to solve the laborious and time-consuming viral RNA extraction operations, and thus exhibits a great potential in the high throughput SARS-CoV-2 molecular diagnosis.
    </p>
    <div id="T2" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1174709/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1174709/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1174709" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table 2.</span>
      <p id="p-33" class="first-child">A comparison of spin column- and pcMNPs-based extraction method in SARS-CoV-2 virus RNA extraction</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">Reference</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.22.961268v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Niu</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhan</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ma</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>, 
          <span class="cit-name-given-names">G. F.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Tan</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>
         <span class="cit-pub-date">2020</span>, 
         <span class="cit-vol">382</span>:
         <span class="cit-fpage">727</span>–
         <span class="cit-lpage">733</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D382%26rft.spage%253D727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.22.961268v1.2" data-doi="10.1128/CMR.19.1.165-256.2006">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Espy</span>, 
          <span class="cit-name-given-names">M. J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Uhl</span>, 
          <span class="cit-name-given-names">J. R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Sloan</span>, 
          <span class="cit-name-given-names">L. M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Buckwalter</span>, 
          <span class="cit-name-given-names">S. P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Jones</span>, 
          <span class="cit-name-given-names">M. F.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Vetter</span>, 
          <span class="cit-name-given-names">E. A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yao</span>, 
          <span class="cit-name-given-names">J. D. C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wengenack</span>, 
          <span class="cit-name-given-names">N. L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Rosenblatt</span>, 
          <span class="cit-name-given-names">J. E.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cockerill</span>, 
          <span class="cit-name-given-names">F. R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Smith</span>, 
          <span class="cit-name-given-names">T. F.</span>
         </span>, 
         <span class="cit-article-title">Real-Time PCR in Clinical Microbiology: Applications for Routine Laboratory Testing</span>. 
         <abbr class="cit-jnl-abbrev">Clin Microbiol Rev</abbr>
         <span class="cit-pub-date">2006</span>, 
         <span class="cit-vol">19</span> (
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">165</span>–
         <span class="cit-lpage">256</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DClinical%2BMicrobiology%2BReviews%26rft.stitle%253DClin.%2BMicrobiol.%2BRev.%26rft.aulast%253DEspy%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D165%26rft.epage%253D256%26rft.atitle%253DReal-Time%2BPCR%2Bin%2BClinical%2BMicrobiology%253A%2BApplications%2Bfor%2BRoutine%2BLaboratory%2BTesting%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCMR.19.1.165-256.2006%26rft_id%253Dinfo%253Apmid%252F16418529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiY21yIjtzOjU6InJlc2lkIjtzOjg6IjE5LzEvMTY1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMjcvMjAyMC4wMi4yMi45NjEyNjguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.22.961268v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ng</span>, 
          <span class="cit-name-given-names">E. K. O.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hui</span>, 
          <span class="cit-name-given-names">D. S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>, 
          <span class="cit-name-given-names">K. C. A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hung</span>, 
          <span class="cit-name-given-names">E. C. W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chiu</span>, 
          <span class="cit-name-given-names">R. W. K.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chim</span>, 
          <span class="cit-name-given-names">S. S. C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Tong</span>, 
          <span class="cit-name-given-names">Y. K.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Sung</span>, 
          <span class="cit-name-given-names">J. J. Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Tam</span>, 
          <span class="cit-name-given-names">J. S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lo</span>, 
          <span class="cit-name-given-names">Y. M. D.</span>
         </span>, 
         <span class="cit-article-title">Quantitative Analysis and Prognostic Implication of SARS Coronavirus RNA in the Plasma and Serum of Patients with Severe Acute Respiratory Syndrome</span>. 
         <abbr class="cit-jnl-abbrev">Clin. Chem</abbr>. 
         <span class="cit-pub-date">2020</span>, 
         <span class="cit-vol">49</span> (
         <span class="cit-issue">12</span>), 
         <span class="cit-fpage">1976</span>–
         <span class="cit-lpage">1980</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DClin.%2BChem%26rft.volume%253D49%26rft.spage%253D1976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.22.961268v1.4" data-doi="10.1016/j.jcv.2003.08.004">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Poon</span>, 
          <span class="cit-name-given-names">L. L. M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>, 
          <span class="cit-name-given-names">K. H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wong</span>, 
          <span class="cit-name-given-names">O. K.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yam</span>, 
          <span class="cit-name-given-names">W. C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yuen</span>, 
          <span class="cit-name-given-names">K. Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Guan</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lo</span>, 
          <span class="cit-name-given-names">Y. M. D.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Peiris</span>, 
          <span class="cit-name-given-names">J. S. M.</span>
         </span>, 
         <span class="cit-article-title">Early diagnosis of SARS Coronavirus infection by real time RT-PCR</span>. 
         <abbr class="cit-jnl-abbrev">J. Clin. Virol</abbr>. 
         <span class="cit-pub-date">2003</span>, 
         <span class="cit-vol">28</span> (
         <span class="cit-issue">3</span>), 
         <span class="cit-fpage">233</span>–
         <span class="cit-lpage">238</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bclinical%2Bvirology%2B%253A%2B%2Bthe%2Bofficial%2Bpublication%2Bof%2Bthe%2BPan%2BAmerican%2BSociety%2Bfor%2BClinical%2BVirology%26rft.stitle%253DJ%2BClin%2BVirol%26rft.aulast%253DPoon%26rft.auinit1%253DL.%2BL.%26rft.volume%253D28%26rft.issue%253D3%26rft.spage%253D233%26rft.epage%253D238%26rft.atitle%253DEarly%2Bdiagnosis%2Bof%2BSARS%2Bcoronavirus%2Binfection%2Bby%2Breal%2Btime%2BRT-PCR.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jcv.2003.08.004%26rft_id%253Dinfo%253Apmid%252F14522060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.jcv.2003.08.004&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=14522060&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.961268.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000187448800002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.22.961268v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">M.-Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">S.-Q.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Deng</span>, 
          <span class="cit-name-given-names">C.-L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Q.-Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-article-title">Detection of Zika virus by SYBR green one-step real-time RT-PCR</span>. 
         <abbr class="cit-jnl-abbrev">J. Virol. Methods</abbr>
         <span class="cit-pub-date">2016</span>, 
         <span class="cit-vol">236</span>, 
         <span class="cit-fpage">93</span>–
         <span class="cit-lpage">97</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%2BMethods%26rft.volume%253D236%26rft.spage%253D93%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.22.961268v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shisong</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Jianxiong</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiaowen</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cunyou</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ting</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xing</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xin</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chunli</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Renli</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Jinquan</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hong</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Muhua</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-article-title">Simultaneous detection of influenza virus type B and influenza A virus subtypes H1N1, H3N2, and H5N1 using multiplex real-time RT-PCR</span>. 
         <abbr class="cit-jnl-abbrev">Appl. Microbiol. Biotechnol</abbr>. 
         <span class="cit-pub-date">2011</span>, 
         <span class="cit-vol">90</span> (
         <span class="cit-issue">4</span>), 
         <span class="cit-fpage">1463</span>–
         <span class="cit-lpage">1470</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.stitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.aulast%253DShisong%26rft.auinit1%253DF.%26rft.volume%253D90%26rft.issue%253D4%26rft.spage%253D1463%26rft.epage%253D1470%26rft.atitle%253DSimultaneous%2Bdetection%2Bof%2Binfluenza%2Bvirus%2Btype%2BB%2Band%2Binfluenza%2BA%2Bvirus%2Bsubtypes%2BH1N1%252C%2BH3N2%252C%2Band%2BH5N1%2Busing%2Bmultiplex%2Breal-time%2BRT-PCR.%26rft_id%253Dinfo%253Apmid%252F21400097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=21400097&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.961268.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.22.961268v1.7" data-doi="10.1016/j.jviromet.2006.08.003">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kong</span>, 
          <span class="cit-name-given-names">Y. Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Thay</span>, 
          <span class="cit-name-given-names">C. H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Tin</span>, 
          <span class="cit-name-given-names">T. C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Devi</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-article-title">Rapid detection, serotyping and quantitation of dengue viruses by TaqMan real-time one-step RT-PCR</span>. 
         <abbr class="cit-jnl-abbrev">J. Virol. Methods</abbr>
         <span class="cit-pub-date">2006</span>, 
         <span class="cit-vol">138</span> (
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">123</span>–
         <span class="cit-lpage">130</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirological%2Bmethods%26rft.stitle%253DJ%2BVirol%2BMethods%26rft.aulast%253DKong%26rft.auinit1%253DY.%2BY.%26rft.volume%253D138%26rft.issue%253D1-2%26rft.spage%253D123%26rft.epage%253D130%26rft.atitle%253DRapid%2Bdetection%252C%2Bserotyping%2Band%2Bquantitation%2Bof%2Bdengue%2Bviruses%2Bby%2BTaqMan%2Breal-time%2Bone-step%2BRT-PCR.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jviromet.2006.08.003%26rft_id%253Dinfo%253Apmid%252F17000012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.jviromet.2006.08.003&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17000012&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.961268.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000242055500016&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.22.961268v1.8" data-doi="10.1128/JCM.02533-13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Whitaker</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Sakthivel</span>, 
          <span class="cit-name-given-names">S. K. K.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kamili</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Rose</span>, 
          <span class="cit-name-given-names">L. E.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lowe</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Mohareb</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Elassal</span>, 
          <span class="cit-name-given-names">E. M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Al-sanouri</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Haddadin</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Erdman</span>, 
          <span class="cit-name-given-names">D. D.</span>
         </span>, 
         <span class="cit-article-title">Real-Time Reverse Transcription-PCR Assay Panel for Middle East Respiratory Syndrome Coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>
         <span class="cit-pub-date">2014</span>, 
         <span class="cit-vol">52</span> (
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">67</span>–
         <span class="cit-lpage">75</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BClin%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.02533-13%26rft_id%253Dinfo%253Apmid%252F24153118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjc6IjUyLzEvNjciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8yNy8yMDIwLjAyLjIyLjk2MTI2OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.22.961268v1.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Corman</span>, 
          <span class="cit-name-given-names">V. M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Landt</span>, 
          <span class="cit-name-given-names">O.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kaiser</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Molenkamp</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Meijer</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chu</span>, 
          <span class="cit-name-given-names">D. K.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bleicker</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Brünink</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Schneider</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Schmidt</span>, 
          <span class="cit-name-given-names">M. L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Mulders</span>, 
          <span class="cit-name-given-names">D. G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Haagmans</span>, 
          <span class="cit-name-given-names">B. L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">van der Veer</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">van den Brink</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wijsman</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Goderski</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Romette</span>, 
          <span class="cit-name-given-names">J.-L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ellis</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zambon</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Peiris</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Goossens</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Reusken</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Koopmans</span>, 
          <span class="cit-name-given-names">M. P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Drosten</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-article-title">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</span>. 
         <abbr class="cit-jnl-abbrev">Euro Surveill</abbr>
         <span class="cit-pub-date">2020</span>, 
         <span class="cit-vol">25</span> (
         <span class="cit-issue">3</span>), 
         <span class="cit-fpage">2000045</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEuro%2BSurveill%26rft.volume%253D25%26rft.spage%253D2000045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.22.961268v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chu</span>, 
          <span class="cit-name-given-names">D. K. W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Pan</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cheng</span>, 
          <span class="cit-name-given-names">S. M. S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hui</span>, 
          <span class="cit-name-given-names">K. P. Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Krishnan</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ng</span>, 
          <span class="cit-name-given-names">D. Y. M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wan</span>, 
          <span class="cit-name-given-names">C. K. C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Peiris</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Poon</span>, 
          <span class="cit-name-given-names">L. L. M.</span>
         </span>, 
         <span class="cit-article-title">Molecular Diagnosis of a Novel Coronavirus (2019-nCoV) Causing an Outbreak of Pneumonia</span>. 
         <abbr class="cit-jnl-abbrev">Clin. Chem</abbr>. 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.22.961268v1.11" data-doi="10.1016/j.jcv.2006.02.008">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Anne Hapip</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Fang</span>, 
          <span class="cit-name-given-names">C. T.</span>
         </span>, 
         <span class="cit-article-title">Highly sensitive TaqMan RT-PCR assay for detection and quantification of both lineages of West Nile virus RNA</span>. 
         <abbr class="cit-jnl-abbrev">J. Clin. Virol</abbr>. 
         <span class="cit-pub-date">2006</span>, 
         <span class="cit-vol">36</span> (
         <span class="cit-issue">3</span>), 
         <span class="cit-fpage">177</span>–
         <span class="cit-lpage">182</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bclinical%2Bvirology%2B%253A%2B%2Bthe%2Bofficial%2Bpublication%2Bof%2Bthe%2BPan%2BAmerican%2BSociety%2Bfor%2BClinical%2BVirology%26rft.stitle%253DJ%2BClin%2BVirol%26rft.aulast%253DTang%26rft.auinit1%253DY.%26rft.volume%253D36%26rft.issue%253D3%26rft.spage%253D177%26rft.epage%253D182%26rft.atitle%253DHighly%2Bsensitive%2BTaqMan%2BRT-PCR%2Bassay%2Bfor%2Bdetection%2Band%2Bquantification%2Bof%2Bboth%2Blineages%2Bof%2BWest%2BNile%2Bvirus%2BRNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jcv.2006.02.008%26rft_id%253Dinfo%253Apmid%252F16675298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.jcv.2006.02.008&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16675298&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.961268.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000238772600003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.22.961268v1.12" data-doi="10.1097/QCO.0b013e32808255f1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>, 
          <span class="cit-name-given-names">Y. R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Morris</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-article-title">Molecular diagnostic methods in pneumonia</span>. 
         <abbr class="cit-jnl-abbrev">Curr Opin Infect Dis</abbr>
         <span class="cit-pub-date">2007</span>, 
         <span class="cit-vol">20</span> (
         <span class="cit-issue">2</span>), 
         <span class="cit-fpage">157</span>–
         <span class="cit-lpage">164</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Binfectious%2Bdiseases%26rft.stitle%253DCurr%2BOpin%2BInfect%2BDis%26rft.aulast%253DChan%26rft.auinit1%253DY.%2BR.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D157%26rft.epage%253D164%26rft.atitle%253DMolecular%2Bdiagnostic%2Bmethods%2Bin%2Bpneumonia.%26rft_id%253Dinfo%253Adoi%252F10.1097%252FQCO.0b013e32808255f1%26rft_id%253Dinfo%253Apmid%252F17496574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1097/QCO.0b013e32808255f1&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17496574&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.961268.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000245527000008&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.22.961268v1.13" data-doi="10.1111/j.1365-2672.2012.05384.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schrader</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Schielke</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ellerbroek</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Johne</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-article-title">PCR inhibitors – occurrence, properties and removal</span>. 
         <abbr class="cit-jnl-abbrev">J Appl Microbiol</abbr>
         <span class="cit-pub-date">2012</span>, 
         <span class="cit-vol">113</span> (
         <span class="cit-issue">5</span>), 
         <span class="cit-fpage">1014</span>–
         <span class="cit-lpage">1026</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DJournal%2Bof%2BApplied%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DSchrader%26rft.auinit1%253DC.%26rft.volume%253D113%26rft.issue%253D5%26rft.spage%253D1014%26rft.epage%253D1026%26rft.atitle%253DPCR%2Binhibitors%2B-%2Boccurrence%252C%2Bproperties%2Band%2Bremoval.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2672.2012.05384.x%26rft_id%253Dinfo%253Apmid%252F22747964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1365-2672.2012.05384.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22747964&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.961268.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000310281100002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-book" id="cit-2020.02.22.961268v1.14">
       <div class="cit-metadata">
        <ol class="duplicate">
         <li>
          <span class="cit-ed">
           <span class="cit-name-surname">Wilks</span>, 
           <span class="cit-name-given-names">M.</span>
          </span>
         </li>
        </ol>
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hedman</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Rådström</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-chapter-title">Overcoming Inhibition in Real-Time Diagnostic PCR</span>. 
         <span class="cit-source">In PCR Detection of Microbial Pathogens</span>, 
         <span class="cit-ed">
          <span class="cit-name-surname">Wilks</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, Ed. 
         <span class="cit-publ-name">Humana Press</span>: 
         <span class="cit-publ-loc">Totowa, NJ</span>, 
         <span class="cit-pub-date">2013</span>; pp 
         <span class="cit-fpage">17</span>–
         <span class="cit-lpage">48</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.22.961268v1.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pichl</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Heitmann</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Herzog</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Oster</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Smets</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Schottstedt</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>, 
         <span class="cit-article-title">Magnetic bead technology in viral RNA and DNA extraction from plasma minipools</span>. 
         <abbr class="cit-jnl-abbrev">Transfusion</abbr>
         <span class="cit-pub-date">2005</span>, 
         <span class="cit-vol">45</span> (
         <span class="cit-issue">7</span>), 
         <span class="cit-fpage">1106</span>–
         <span class="cit-lpage">1110</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTransfusion%26rft.volume%253D45%26rft.spage%253D1106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.22.961268v1.16" data-doi="10.1021/ac501573g">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Váradi</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lew</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Guttman</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-article-title">Rapid Magnetic Bead Based Sample Preparation for Automated and High Throughput N-Glycan Analysis of Therapeutic Antibodies</span>. 
         <abbr class="cit-jnl-abbrev">Anal. Chem</abbr>. 
         <span class="cit-pub-date">2014</span>, 
         <span class="cit-vol">86</span> (
         <span class="cit-issue">12</span>), 
         <span class="cit-fpage">5682</span>–
         <span class="cit-lpage">5687</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnal.%2BChem%26rft.volume%253D86%26rft.spage%253D5682%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fac501573g%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/ac501573g&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.22.961268v1.17" data-doi="10.1002/jcla.20174">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Riemann</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Adamzik</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Frauenrath</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Egensperger</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Schmid</span>, 
          <span class="cit-name-given-names">K. W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Brockmeyer</span>, 
          <span class="cit-name-given-names">N. H.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Siffert</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-article-title">Comparison of manual and automated nucleic acid extraction from whole-blood samples</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Lab Anal</abbr>
         <span class="cit-pub-date">2007</span>, 
         <span class="cit-vol">21</span> (
         <span class="cit-issue">4</span>), 
         <span class="cit-fpage">244</span>–
         <span class="cit-lpage">248</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bclinical%2Blaboratory%2Banalysis%26rft.stitle%253DJ%2BClin%2BLab%2BAnal%26rft.aulast%253DRiemann%26rft.auinit1%253DK.%26rft.volume%253D21%26rft.issue%253D4%26rft.spage%253D244%26rft.epage%253D248%26rft.atitle%253DComparison%2Bof%2Bmanual%2Band%2Bautomated%2Bnucleic%2Bacid%2Bextraction%2Bfrom%2Bwhole-blood%2Bsamples.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcla.20174%26rft_id%253Dinfo%253Apmid%252F17621359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/jcla.20174&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17621359&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.961268.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.22.961268v1.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Su</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gregory</span>, 
          <span class="cit-name-given-names">D. A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cai</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-article-title">Magnetic Alginate/Chitosan Nanoparticles for Targeted Delivery of Curcumin into Human Breast Cancer Cells</span>. 
         <abbr class="cit-jnl-abbrev">Nanomaterials</abbr>
         <span class="cit-pub-date">2018</span>, 
         <span class="cit-vol">8</span> (
         <span class="cit-issue">11</span>), 
         <span class="cit-fpage">907</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNanomaterials%26rft.volume%253D8%26rft.spage%253D907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.22.961268v1.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sunshine</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Green</span>, 
          <span class="cit-name-given-names">J. J.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Mahon</span>, 
          <span class="cit-name-given-names">K. P.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Eltoukhy</span>, 
          <span class="cit-name-given-names">A. A.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Nguyen</span>, 
          <span class="cit-name-given-names">D. N.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Langer</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>; 
         <span class="cit-auth">
          <span class="cit-name-surname">Anderson</span>, 
          <span class="cit-name-given-names">D. G.</span>
         </span>, 
         <span class="cit-article-title">Small-Molecule End-Groups of Linear Polymer Determine Cell-type Gene-Delivery Efficacy</span>. 
         <abbr class="cit-jnl-abbrev">Adv Mater</abbr>
         <span class="cit-pub-date">2009</span>, 
         <span class="cit-vol">21</span> (
         <span class="cit-issue">48</span>), 
         <span class="cit-fpage">4947</span>–
         <span class="cit-lpage">4951</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAdv%2BMater%26rft.volume%253D21%26rft.spage%253D4947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
