<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/04/05/2020.04.01.020594.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.04.01.020594v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="A snapshot of SARS-CoV-2 genome availability up to 30th March, 2020 and its implications"/>
  <meta name="DC.Identifier" content="10.1101/2020.04.01.020594"/>
  <meta name="DC.Date" content="2020-04-05"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The SARS-CoV-2 pandemic has been growing exponentially, affecting nearly 900 thousand people and causing enormous distress to economies and societies worldwide. A plethora of analyses based on viral sequences has already been published, in scientific journals as well as through non-peer reviewed channels, to investigate SARS-CoV-2 genetic heterogeneity and spatiotemporal dissemination. We examined full genome sequences currently available to assess the presence of sufficient information for reliable phylogenetic and phylogeographic studies in countries with the highest toll of confirmed cases. Although number of-available full-genomes is growing daily, and the full dataset contains sufficient phylogenetic information that would allow reliable inference of phylogenetic relationships, country-specific SARS-CoV-2 datasets still present severe limitations. Studies assessing within country spread or transmission clusters should be considered preliminary at best, or hypothesis generating. Hence the need for continuing concerted efforts to increase number and quality of the sequences required for robust tracing of the epidemic. Significance Statement Although genome sequences of SARS-CoV-2 are growing daily and contain sufficient phylogenetic information, country-specific data still present severe limitations and should be interpreted with caution."/>
  <meta name="DC.Contributor" content="Carla Mavian"/>
  <meta name="DC.Contributor" content="Simone Marini"/>
  <meta name="DC.Contributor" content="Mattia Prosperi"/>
  <meta name="DC.Contributor" content="Marco Salemi"/>
  <meta name="article:published_time" content="2020-04-05"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="A snapshot of SARS-CoV-2 genome availability up to 30th March, 2020 and its implications"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;The SARS-CoV-2 pandemic has been growing exponentially, affecting nearly 900 thousand people and causing enormous distress to economies and societies worldwide. A plethora of analyses based on viral sequences has already been published, in scientific journals as well as through non-peer reviewed channels, to investigate SARS-CoV-2 genetic heterogeneity and spatiotemporal dissemination. We examined full genome sequences currently available to assess the presence of sufficient information for reliable phylogenetic and phylogeographic studies in countries with the highest toll of confirmed cases. Although number of-available full-genomes is growing daily, and the full dataset contains sufficient phylogenetic information that would allow reliable inference of phylogenetic relationships, country-specific SARS-CoV-2 datasets still present severe limitations. Studies assessing within country spread or transmission clusters should be considered preliminary at best, or hypothesis generating. Hence the need for continuing concerted efforts to increase number and quality of the sequences required for robust tracing of the epidemic.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.04.01.020594v1"/>
  <meta name="citation_id" content="2020.04.01.020594v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.01.020594v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.01.020594v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.01.020594v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.04.01.020594"/>
  <meta name="citation_num_pages" content="21"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.04.01.020594"/>
  <meta name="citation_author" content="Carla Mavian"/>
  <meta name="citation_author_institution" content="Emerging Pathogens Institute"/>
  <meta name="citation_author_institution" content="Department of Pathology"/>
  <meta name="citation_author_email" content="salemi@pathology.ufl.edu"/>
  <meta name="citation_author_email" content="cmavian@ufl.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3819-5520"/>
  <meta name="citation_author" content="Simone Marini"/>
  <meta name="citation_author_institution" content="Emerging Pathogens Institute"/>
  <meta name="citation_author_institution" content="Department of Epidemiology, University of Florida"/>
  <meta name="citation_author" content="Mattia Prosperi"/>
  <meta name="citation_author_institution" content="Department of Epidemiology, University of Florida"/>
  <meta name="citation_author" content="Marco Salemi"/>
  <meta name="citation_author_institution" content="Emerging Pathogens Institute"/>
  <meta name="citation_author_institution" content="Department of Pathology"/>
  <meta name="citation_author_email" content="salemi@pathology.ufl.edu"/>
  <meta name="citation_author_email" content="cmavian@ufl.edu"/>
  <meta name="citation_reference" content="WHO (2020) World Health Organization. Coronavirus disease 2019 (COVID-19). Situation Report – 43. [online]"/>
  <meta name="citation_reference" content="L. L. M. Poon, M. Peiris, Emergence of a novel human coronavirus threatening human health. Nat Med 10.1038/s41591-020-0796-5 (2020)."/>
  <meta name="citation_reference" content="C. Wang et al., Evolving Epidemiology and Impact of Non-pharmaceutical Interventions on the Outbreak of Coronavirus Disease 2019 in Wuhan, China. medRxiv 10.1101/2020.03.03.20030593, 2020.2003.2003.20030593 (2020)."/>
  <meta name="citation_reference" content="J. Hadfield et al., Nextstrain: real-time tracking of pathogen evolution. Bioinformatics 34, 4121–4123 (2018)."/>
  <meta name="citation_reference" content="X. Tang et al., On the origin and continuing evolution of SARS-CoV-2. National Science Review 10.1093/nsr/nwaa036 (2020)."/>
  <meta name="citation_reference" content="O. A. MacLean, R. Orton, J. B. Singer, D. L. Robertson (2020) Response to “On the origin and continuing evolution of SARS-CoV-2”."/>
  <meta name="citation_reference" content="K. Kupferschmidt (2020) Mutations can reveal how the coronavirus moves— but they’re easy to overinterpret. ed Science."/>
  <meta name="citation_reference" content="M. Salemi, Vandamme, A.-M., &amp;; Lemey, P., The phylogenetic handbook: A practical approach to phylogenetic analysis and hypothesis testing. (Cambridge University Press, Cambridge, UK, 2009)."/>
  <meta name="citation_reference" content="Y. Shu, J. McCauley, GISAID: Global initiative on sharing all influenza data - from vision to reality. Euro Surveill 22 (2017)."/>
  <meta name="citation_reference" content="Anonymous (2020) Sito del Dipartimento della Protezione Civile - Emergenza Coronavirus: la risposta nazionale."/>
  <meta name="citation_reference" content="H. A. Schmidt, K. Strimmer, M. Vingron, A. von Haeseler, TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel computing. Bioinformatics 18, 502–504 (2002)."/>
  <meta name="citation_reference" content="K. Strimmer, A. von Haeseler, Likelihood-mapping: a simple method to visualize phylogenetic content of a sequence alignment. Proc Natl Acad Sci U S A 94, 6815–6819 (1997)."/>
  <meta name="citation_reference" content="A. Rambaut, T. T. Lam, L. Max Carvalho, O. G. Pybus, Exploring the temporal structure of heterochronous sequences using TempEst (formerly Path-O-Gen). Virus Evol 2, vew007 (2016)."/>
  <meta name="citation_reference" content="A. J. Drummond, A. Rambaut, BEAST: Bayesian evolutionary analysis by sampling trees. BMC Evol Biol 7, 214 (2007)."/>
  <meta name="citation_reference" content="A. J. Drummond, S. Y. Ho, M. J. Phillips, A. Rambaut, Relaxed phylogenetics and dating with confidence. PLoS Biol 4, e88 (2006)."/>
  <meta name="citation_reference" content="X. Didelot, C. Fraser, J. Gardy, C. Colijn, Genomic Infectious Disease Epidemiology in Partially Sampled and Ongoing Outbreaks. Mol Biol Evol 34, 997–1007 (2017)."/>
  <meta name="citation_reference" content="C. Jing et al., Indirect Virus Transmission in Cluster of COVID-19 Cases, Wenzhou, China, 2020. Emerging Infectious Disease journal 26 (2020)."/>
  <meta name="citation_reference" content="R. Pung et al., Investigation of three clusters of COVID-19 in Singapore: implications for surveillance and response measures. Lancet 395, 1039–1046 (2020)."/>
  <meta name="citation_reference" content="X. Xia, Z. Xie, DAMBE: software package for data analysis in molecular biology and evolution. J Hered 92, 371–373 (2001)."/>
  <meta name="citation_reference" content="L. T. Nguyen, H. A. Schmidt, A. von Haeseler, B. Q. Minh, IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies. Mol Biol Evol 32, 268–274 (2015)."/>
  <meta name="citation_reference" content="J. Trifinopoulos, L. T. Nguyen, A. von Haeseler, B. Q. Minh, W-IQ-TREE: a fast online phylogenetic tool for maximum likelihood analysis. Nucleic Acids Res 44, W232–235 (2016)."/>
  <meta name="citation_reference" content="H. Shimodaira, M. Hasegawa, Multiple comparisons of log-likelihoods with applications to phylogenetic inference. Molecular Biology and Evolution 16, 1114–1116 (1999)."/>
  <meta name="twitter:title" content="A snapshot of SARS-CoV-2 genome availability up to 30th March, 2020 and its implications"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The SARS-CoV-2 pandemic has been growing exponentially, affecting nearly 900 thousand people and causing enormous distress to economies and societies worldwide. A plethora of analyses based on viral sequences has already been published, in scientific journals as well as through non-peer reviewed channels, to investigate SARS-CoV-2 genetic heterogeneity and spatiotemporal dissemination. We examined full genome sequences currently available to assess the presence of sufficient information for reliable phylogenetic and phylogeographic studies in countries with the highest toll of confirmed cases. Although number of-available full-genomes is growing daily, and the full dataset contains sufficient phylogenetic information that would allow reliable inference of phylogenetic relationships, country-specific SARS-CoV-2 datasets still present severe limitations. Studies assessing within country spread or transmission clusters should be considered preliminary at best, or hypothesis generating. Hence the need for continuing concerted efforts to increase number and quality of the sequences required for robust tracing of the epidemic. Significance Statement Although genome sequences of SARS-CoV-2 are growing daily and contain sufficient phylogenetic information, country-specific data still present severe limitations and should be interpreted with caution."/>
  <meta name="og-title" property="og:title" content="A snapshot of SARS-CoV-2 genome availability up to 30th March, 2020 and its implications"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.01.020594v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The SARS-CoV-2 pandemic has been growing exponentially, affecting nearly 900 thousand people and causing enormous distress to economies and societies worldwide. A plethora of analyses based on viral sequences has already been published, in scientific journals as well as through non-peer reviewed channels, to investigate SARS-CoV-2 genetic heterogeneity and spatiotemporal dissemination. We examined full genome sequences currently available to assess the presence of sufficient information for reliable phylogenetic and phylogeographic studies in countries with the highest toll of confirmed cases. Although number of-available full-genomes is growing daily, and the full dataset contains sufficient phylogenetic information that would allow reliable inference of phylogenetic relationships, country-specific SARS-CoV-2 datasets still present severe limitations. Studies assessing within country spread or transmission clusters should be considered preliminary at best, or hypothesis generating. Hence the need for continuing concerted efforts to increase number and quality of the sequences required for robust tracing of the epidemic. Significance Statement Although genome sequences of SARS-CoV-2 are growing daily and contain sufficient phylogenetic information, country-specific data still present severe limitations and should be interpreted with caution."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-04-05"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>A snapshot of SARS-CoV-2 genome availability up to 30th March, 2020 and its implications | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1229939 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">The SARS-CoV-2 pandemic has been growing exponentially, affecting nearly 900 thousand people and causing enormous distress to economies and societies worldwide. A plethora of analyses based on viral sequences has already been published, in scientific journals as well as through non-peer reviewed channels, to investigate SARS-CoV-2 genetic heterogeneity and spatiotemporal dissemination. We examined full genome sequences currently available to assess the presence of sufficient information for reliable phylogenetic and phylogeographic studies in countries with the highest toll of confirmed cases. Although number of-available full-genomes is growing daily, and the full dataset contains sufficient phylogenetic information that would allow reliable inference of phylogenetic relationships, country-specific SARS-CoV-2 datasets still present severe limitations. Studies assessing within country spread or transmission clusters should be considered preliminary at best, or hypothesis generating. Hence the need for continuing concerted efforts to increase number and quality of the sequences required for robust tracing of the epidemic.</p>
    <div id="sec-1" class="subsection">
     <p id="p-4">
      <strong>Significance Statement</strong> Although genome sequences of SARS-CoV-2 are growing daily and contain sufficient phylogenetic information, country-specific data still present severe limitations and should be interpreted with caution.
     </p>
    </div>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Introduction</h2>
    <p id="p-5">In December 2019, a novel coronavirus, SARS-CoV-2, was identified in Wuhan, China, as the etiologic agent of coronavirus disease 2019 (COVID-19), which by March 2020 has already spread across more than 80 countries (
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>). Common symptoms of infection include fever, cough, and shortness of breath, while severe cases are characterized by advanced respiratory distress and pneumonia, often resulting in death (
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>). It is still unknown what proportion of infected people that only present mild or no symptoms are spreading the virus. A recent study showed that in Wuhan whom roughly 60% of all infections were spread by asymptomatic subjects (
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>). This characteristic significantly thwarts the job of public health officials who are trying to detect transmission clusters. Few transmission clusters have been identified in China recently through epidemiological contact tracing. However, because of the ongoing nature of the outbreak, it is even more complicated if not impossible to detect transmission clusters using genetic data.
    </p>
    <p id="p-6">Soon after the first epidemiological and SARS-CoV-2 genetic sequence data were made available, a glut of phylogeny-based analyses began to circulate discussing, in scientific papers as well as (social) media, countries that might have been fueling the spread. The implications of misunderstanding the real dynamic of the COVID-19 pandemic are extremely dangerous. Ethnic or social discrimination resulting from unsupported assumptions on viral contagion – often amplified by irresponsible, uncontrollable communications – can be highly damaging for people and countries. In particular, the US-based NextStrain (
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>) team has been posting real-time updates on the epidemic tracing by molecular analyses. Despite (social) media are often vehicle for fake news and boast news hype, it is also worth noting the tremendous effort of the scientific community to provide free, up-to-date information on ongoing studies, as well as critical evaluations. Several discussions and evidence-based debates on controversial hypotheses on the epidemic have ensued — such as the number of untraced infections in the US, the putative virus introduction in Italy through Germany, and the alleged lineage diversification in China (
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>) later criticized (
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>). Recently, an editorial published on Science (
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>) has also highlighted how unsupported or misleading claims circulating in forums, social media, and even peer-reviewed articles, have been led by a substantial over interpretation of the available data. Hence, the urgency to reframe the current debate in more rigorous scientific terms, and quantitatively evaluate whether sufficient information for reliable phylogenetic and phylogeographic studies currently exists, or which gaps need to be addressed. We explored the characteristics of few datasets through time and assess phylogenetic signal to understand whether we this data is useful of not.
    </p>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">Results</h2>
    <div id="sec-4" class="subsection">
     <h3>Sampling and phylogeographic uncertainly</h3>
     <p id="p-7">Before carrying out any phylogeny-based analysis of virus evolution and spatiotemporal spread, it is crucial to test the quality of sequence data, since uneven sampling, presence of phylogenetic noise, and absence of temporal signal can affect reliability of the results (e.g. ancestral state reconstructions, molecular clock calibrations) (
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>). SARS-CoV-2 full genome sequences were obatained from GISAID (
      <a href="https://www.gisaid.org/">https://www.gisaid.org/</a>) (
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>) at different timepoints. As of March 30
      <sup>th</sup>, we compared the number of full genomes sampled per country with the number of confirmed cases at the time of sampling, as well as the country’s total population (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). We obtained 2608 full genomes from 55 countries (
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). During the past month, the number of genomes has steeply been increasing: March 3
      <sup>rd</sup>, 169 genome sequences from 22 countries; March 10
      <sup>th</sup>, 331 genome sequences from 29 countries; March 18
      <sup>th</sup>, 794 genome sequences from 35 countries, ρ=0.45; March 25
      <sup>th</sup>, 1662 genome sequences from 42 countries. We found spearman (rank) correlation between confirmed cases and genomes per country to be 0.49 on March 30
      <sup>th</sup>, and we considered it as a proxy for sampling homogeneity. However, correlation could only be investigated with confirmed cases (again as proxy), since not all affected countries have made publicly available the total number of coronavirus testing performed. Moreover, even within the same country, sequenced genomes were usually sampled from few hotspots, not necessarily representative of the whole epidemic in that country. It is worrisome that, as of March 30
      <sup>th</sup> 2020, the two top countries in terms of confirmed cases do not show sufficiently large and representative sampling. SARS-CoV-2 full genome sequences available from patients in the US, the country with the highest number of confirmed cases, have mainly been sampled in Washington state (66%) during the early epidemic, while less than one third (32%) are available from the epicenter of the US epidemic, the state of New York. Italy, the second country per confirmed cases, uploaded 26 genomes, of which one from the Marche region, four from Friuli Venezia Giulia, seven from Abruzzo, nine from Lazio, and only five from Lombardy, which is epicenter of the Italian epidemic (Table S1) (
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>). The top 10 contributors per number of genomes are USA (612), Iceland (343), UK (321), China (300), Netherlands (190), France (119), Japan (83), Canada (80), Australia (64), and Belgium (46). Notably, some countries uploaded a high number of genomes despite having a relatively low number of cases (e.g., Georgia, Iceland, Senegal, DRC).
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Available SARS-CoV-2 genome sequences and confirmed cases per country. Selected countries for number of provided GISAID full genomes (2 or more), or confirmed cases (25K or more) as of March 30th 2020. On the x axis we have number the confirmed cases (log10 scale), on the y axis the number of GISAID full genomes +1 (log10 scale). The number of genomes determines dot color, while the country population determines dot size. Notably, some countries uploaded a high number of genomes despite having a relatively low number of cases (e.g., Georgia, Iceland, Senegal, DRC)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1934261073" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Available SARS-CoV-2 genome sequences and confirmed cases per country. Selected countries for number of provided GISAID full genomes (2 or more), or confirmed cases (25K or more) as of March 30th 2020. On the x axis we have number the confirmed cases (log10 scale), on the y axis the number of GISAID full genomes +1 (log10 scale). The number of genomes determines dot color, while the country population determines dot size. Notably, some countries uploaded a high number of genomes despite having a relatively low number of cases (e.g., Georgia, Iceland, Senegal, DRC).&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F1.medium.gif" width="440" height="432"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F1.medium.gif" width="440" height="432"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <p id="p-8" class="first-child">Available SARS-CoV-2 genome sequences and confirmed cases per country. Selected countries for number of provided GISAID full genomes (2 or more), or confirmed cases (25K or more) as of March 30th 2020. On the x axis we have number the confirmed cases (log10 scale), on the y axis the number of GISAID full genomes +1 (log10 scale). The number of genomes determines dot color, while the country population determines dot size. Notably, some countries uploaded a high number of genomes despite having a relatively low number of cases (e.g., Georgia, Iceland, Senegal, DRC).</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Phylogenetic noise in sequence data</h3>
     <p id="p-9">Lack of resolution and uncertainty in the SARS-CoV-2 phylogenetic tree is to be expected, considering that relatively little genetic diversity can be accumulated during the first three months of an epidemic, even for an exponentially spreading and fast-evolving RNA virus. Assessment of phylogenetic signal in the dataset was carried out using likelihood mapping analysis (
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>), which estimates the likelihood of each of possible tree topology for any group of four sequences (quartet), randomly chosen from an alignment, and reports them inside an equilateral triangle (the likelihood map) where the corners represent distinct tree topologies and the center star-like trees. Quartets are considered “resolved” when the three likelihoods are significantly different (phylogenetic signal, most dots equally distributed in the corners, i.e. data are suitable for robust phylogeny inference), unresolved or partially resolved, when all three likelihood values or two of them are not significantly different (phylogenetic noise, most dots in sides or center areas, i.e. data may not be sufficient for robust phylogeny inference). Extensive simulation studies have shown that, for sequences to be considered robust in terms of phylogenetic signal, side/center areas of the likelihood mapping must include &lt;;40% of the unresolved quartets (
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>). Overall, phylogenetic signal in the present data has been increasing with number of genomes been released. Percentage of unresolved quartets detected in the SARS-CoV-2 full genomes alignment on March 3
      <sup>rd</sup> and 10
      <sup>th</sup> was still too high to allow reliable inferences (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure S1</a>). In other words, such a lack of phylogenetic signal has likely resulted in overall unreliable topology of any SARS-CoV-2 tree obtained using those data, and even clades with high bootstrap values should have been interpreted with extreme caution.
     </p>
     <p id="p-10">The effect of inhomogeneous sampling, lack of phylogenetic signal and missing data on phylogeography reconstructions, like the ones recently rushed through news and (social) media to claim specific dissemination routes of SARS-CoV-2 among countries, can be quite dramatic. An instructive example is the putative introduction of SARS-CoV-2 in Italy from Germany. A preliminary maximum likelihood (ML) tree, inferred from the full genome viral sequences available on March 3
      <sup>rd</sup> 2020, showed a well-supported cluster of European and Asian sequences (reported in 
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure S2</a>), which contained a subclade (Subclade A, 
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2a</a>) including a sequence isolated in Germany that appears to be paraphyletic (with strong bootstrap support) to an Italian sequence clustering, in turn, with sequences from Finland, Mexico, Germany and Switzerland. Based on this observation (which was available on NextStrain), a heated discussion circulated on social media about a transmission event from Germany to Italy followed by further spread from Italy the other countries. However, in a new tree inferred just one week later, when more than 135 new full genome sequences were made available on GISAID (
      <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>), the direct link between Germany and Italy in Subclade A disappeared due to the additional clustering of previously unsampled sequences from Portugal, Brazil, Wales and Netherland (
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2b</a>). In addition, likelihoods of alternative tree topologies generated arbitrarily switching branches in the tree (arrows in 
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2b</a>), implying different dissemination scenarios, were not significantly different (Shimodaira-Hasgawa test, 
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table S2</a>) than the likelihood of the tree inferred from the real data. In other words, it is not possible, with present data, to decide which branching pattern (and, therefore, phylogeographic reconstruction) is the one most likely representing actual dissemination routes among European countries.
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Cladograms of SARS-CoV-2 subclades. Cladograms were extracted from ML phylogenies rooted by enforcing a molecular clock. Colored branches represent country of origin of sampled sequences (tip branches) and ancestral lineages (internal branches). Numbers at nodes indicate ultrafast bootstrap (BB) support (only &gt;;90% values are shown). (a) Cladogram of a monophyletic clade within the SARS-CoV-2 ML tree inferred from sequences available on March 3rd 2020 (Supplementary Figure S1). The subclade including sequences from Italy and Germany, named Subclade A, is highlighted. (b) Cladogram of subclade A of the SARS-CoV-2 ML tree including additional sequences available on March 10th 2020 (Figure S2). Each bi-directional arrow, and corresponding number, connects two tip branches that were switched to generate an alternative tree topology to be tested (see Table 1, Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1934261073" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Cladograms of SARS-CoV-2 subclades. Cladograms were extracted from ML phylogenies rooted by enforcing a molecular clock. Colored branches represent country of origin of sampled sequences (tip branches) and ancestral lineages (internal branches). Numbers at nodes indicate ultrafast bootstrap (BB) support (only &gt;;90% values are shown). (a) Cladogram of a monophyletic clade within the SARS-CoV-2 ML tree inferred from sequences available on March 3&lt;;sup&gt;;rd&lt;;/sup&gt;; 2020 (Supplementary Figure S1). The subclade including sequences from Italy and Germany, named Subclade A, is highlighted. (b) Cladogram of subclade A of the SARS-CoV-2 ML tree including additional sequences available on March 10&lt;;sup&gt;;th&lt;;/sup&gt;; 2020 (Figure S2). Each bi-directional arrow, and corresponding number, connects two tip branches that were switched to generate an alternative tree topology to be tested (see Table 1, Methods).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F2.medium.gif" width="440" height="244"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F2.medium.gif" width="440" height="244"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <p id="p-11" class="first-child">Cladograms of SARS-CoV-2 subclades. Cladograms were extracted from ML phylogenies rooted by enforcing a molecular clock. Colored branches represent country of origin of sampled sequences (tip branches) and ancestral lineages (internal branches). Numbers at nodes indicate ultrafast bootstrap (BB) support (only &gt;;90% values are shown). (a) Cladogram of a monophyletic clade within the SARS-CoV-2 ML tree inferred from sequences available on March 3
        <sup>rd</sup> 2020 (
        <a id="xref-fig-3-2" class="xref-fig" href="#F3">Supplementary Figure S1</a>). The subclade including sequences from Italy and Germany, named Subclade A, is highlighted. (b) Cladogram of subclade A of the SARS-CoV-2 ML tree including additional sequences available on March 10
        <sup>th</sup> 2020 (
        <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure S2</a>). Each bi-directional arrow, and corresponding number, connects two tip branches that were switched to generate an alternative tree topology to be tested (see Table 1, Methods).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-12">Available genome sequences are rapidly growing. SARS-CoV-2 full genome dataset is now showing less than 40% of unresolved quartets in the center: 38.6% unresolved quartets on March 18
      <sup>th</sup> (794 genome sequences) (
      <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure S1c</a>) and 32.3% on March 25
      <sup>th</sup> (1,660 genome sequences) (
      <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure S1d</a>). This indicates that the amount of phylogenetic information is now potentially usable to define phylogenetic relationships among strains. Plotting mean genetic distance of each sequence from the root of a phylogeny versus the sequence sampling time allows to testing for significant linear correlation, which is necessary for the calibration of a reliable molecular clock (
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>) (
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure S3</a>). As expected in genomes obtained over a very short period of time (∼ three months) since the beginning of the outbreak, correlation in the current data is fairly week (Table S3). However, Bayesian analysis (
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>), which infers phylogenetic and phylogeographic patterns from a posterior distribution of trees, might facilitate comparisons about different evolutionary scenarios, help in retrieving the correct topology, and estimate an accurate evolutionary rate using relaxed clock methods (
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>). Reconstructing the phylogenetic relationships of the same European Subclade A discussed above with sequence available on March 18
      <sup>th</sup> 2020 showed a much more complex snapshot of SARS-CoV-2 spreading (
      <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure S4</a>). Taking a closer look at the Sublade A reveals that even with more genomes available, inference is bias by over-sampling of some countries and under-sampling of others (
      <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure S4</a>). Yet, even with more genomes available, inference is bias by over-sampling of some countries and under-sampling of others (
      <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure S4</a>). Recently, methods were developed to estimate, for each pair of viral sequences from two infected individuals, how many intermediates there are in the putative transmission chain connecting them, using a transmission matrix (
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>). The analysis of SARS-CoV-2 genomes shows that numerous links among samples are still missing (
      <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure S5</a>). In such a scenario, it is not advisable to extrapolate conclusions on the origin and dissemination of strains.
     </p>
     <p id="p-13">Phylogenetic signal is increasing in the global alignment; yet, likelihood mapping per country, using data from countries reporting highest number of cases – USA, Italy, Spain, Germany, and France – indicate that some local datasets lack sufficient signal (
      <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure S6</a>). Lack of signal was found in datasets from Italy (26 genomes, 45 variant sites – 0.2% of total sites in the genome - 11 parsimony informative), USA (612 genomes, 675 variant sites - 2.3% of total sites in the genome - 158 parsimony informative) and China (300 genomes, 742 variant sites – 2.5% of total sites in the genome - 98 parsimony informative). The top 5 contributing states in US are Washington (405, 66%), California (45, 7%), Minnesota (33, 5%), Wisconsin (29, 5%), and Utah (22, 4%); 42 genomes are not labeled with a state or city. USA dataset comprised mostly of sequences collected in Washington State (423 genomes, 69.1%). The top 5 contributing provinces in China are Shanghai (96, 32%), Guangdong (80, 27%), Hong Kong (30, 10%), Hubei (31, 10%), Hangzhou (9, 3%), and Shandong (9, 3%); 20 genomes are not labeled with a province or city. Neither China nor US showed phylogenetic signal despite the high number of genome sequences available (
      <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure S6</a>). On the contrary, and unexpectedly, countries with low number of genome sequences did show presence of phylogenetic signal: Germany, Spain and France (
      <a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure S6</a>).
     </p>
     <p id="p-14">Despite the presence of phylogenetic signal in these countries, only genomes form France also show temporal signal that would allow for calibration of a molecular clock and re-framing phylogenetic and phylogeography inferences in spatiotemporal dimension. On the other hand, the transmission matrix for France indicates that considerable links are still missing due to unsampled infected individuals, limiting the reliability of transmission cluster studies based on sequence data (
      <a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure S8</a>).
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Conclusions and Future directions</h3>
     <p id="p-15">As more genome sequences, sampled at different time points and from diverse geographic areas, are daily becoming available, in depth Bayesian phylodynamic and phylogeography analyses of the COVID-19 pandemic may soon be a viable option. As long new data do not increase phylogenetic and temporal signal, results will remain highly questionable. Characterization of transmission events is fundamental to understand the dynamics of any infectious disease. From a public health standpoint, being able to trace transmission at the local level is crucial. Within country identification of active transmission clusters would open the way to more effective public health measures. The most optimal inference of transmission events would have a combination of genetic and epidemiological data for a joint analysis. However, it is not possible, at the moment, to identify transmission clusters within regions, counties, or cities, solely on genetic data, since micro-scale genetic data is not yet available. Indeed, transmission investigations that have been performed so far have been based on contact-tracing, epidemiological and clinical data (
      <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>, 
      <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>).
     </p>
     <p id="p-16">Published scientific data and media are, nowadays, easily accessible to a worldwide audience; properly weighing the information being shared is important more than ever. We deem that current molecular epidemiology data are not solid enough to provide a scientifically sound analysis of SARS-CoV-2 spread. Despite overall increasing of phylogenetic and temporal signal, we suggest that any conclusion drawn, at present, about existing lineages and direction of viral spread, based on phylogenetic analysis of SARS-CoV-2 sequence data, should be considered at the very best preliminary, and hypothesis-generating. The evolutionary dynamics of SARS-CoV-2 spread is unveiling an unprecedented amount of information, essential to make policy decisions. The whole of humanity is threatened by the current pandemic, and policy makers need to adjust their mitigation measures while the pandemic itself is developing. Some of the urgent answers required lie in the timely availability of abundant, high quality genetic data not only from countries experiencing a high number of reported cases, but also from others that seem to be experiencing, at least for now, a lower number of infections.</p>
    </div>
   </div>
   <div class="section" id="sec-7">
    <h2 class="">Methods</h2>
    <div id="sec-8" class="subsection">
     <h3>Data</h3>
     <p id="p-17">GISAID was accessed on March 30
      <sup>th</sup> 2020 (Table S1). After quality control of sequences that were not full genomes or contained extensive stretches of unknown nucleotides, March 30
      <sup>th</sup> March 30
      <sup>th</sup> 2020 of 2608 from 55 countries. Confirmed cases were retrieved from the March 29
      <sup>th</sup> data of the Covid19 website provided by Johns Hopkins University (
      <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>). Top ten countries per confirmed cases were USA (121,146), Italy (97,689), China (82,122), Spain (80,110), Germany (62,095), France (40,708), Iran (38,309), UK (19,778), Switzerland (14,829), and Netherlands (10,930).
     </p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Phylogenetic signal and ML phylogeny inference</h3>
     <p id="p-18">Transition/transversions vs. genetic distance plot were calculated using DAMBE6 (
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>). Evaluation of the presence of phylogenetic signal satisfying resolved phylogenetic relationships among sequences was carried out with IQ-TREE, allowing the software to search for all possible quartets using the best-fitting nucleotide substitution model (
      <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>). ML tree reconstruction was performed in IQ-TREE based on the best-fit model chosen according to Bayesian Information Criterion (BIC) (
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>, 
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>). Exploration of temporal structure, i.e. presence of molecular clock in the data, was assessed by regression of divergence -root-to-tip genetic distance-against sampling time using TempEst (
      <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>). In this case, absence of a linear trend indicates that the data does not contain temporal signal and that the data is not appropriate for phylogenetic inference using molecular clock models. TransPhylo R package was used to infer transmission matrices of SARS-CoV-2 (
      <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>).
     </p>
    </div>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">Supplementary Figures and Tables</h2>
    <div id="F3" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Phylogenetic signal for 169 and 331 full genomes of SARS-CoV-2. Presence of phylogenetic signal was evaluated by likelihood mapping checking for alternative topologies (tips), unresolved quartets (center) and partly resolved quartets (edges) for the (a) 169 genomes available on March 3rd 2020, (b) and 331 genomes on March 10th 2020, (c) 794 genome on March 18th 2020, and (d) 1,660 genome on March 25th 2020." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1934261073" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Phylogenetic signal for 169 and 331 full genomes of SARS-CoV-2. Presence of phylogenetic signal was evaluated by likelihood mapping checking for alternative topologies (tips), unresolved quartets (center) and partly resolved quartets (edges) for the (a) 169 genomes available on March 3&lt;;sup&gt;;rd&lt;;/sup&gt;; 2020, (b) and 331 genomes on March 10&lt;;sup&gt;;th&lt;;/sup&gt;; 2020, (c) 794 genome on March 18&lt;;sup&gt;;th&lt;;/sup&gt;; 2020, and (d) 1,660 genome on March 25&lt;;sup&gt;;th&lt;;/sup&gt;; 2020.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F3.medium.gif" width="440" height="435"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure S1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F3.medium.gif" width="440" height="435"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure S1.</span>
      <p id="p-20" class="first-child">Phylogenetic signal for 169 and 331 full genomes of SARS-CoV-2. Presence of phylogenetic signal was evaluated by likelihood mapping checking for alternative topologies (tips), unresolved quartets (center) and partly resolved quartets (edges) for the (a) 169 genomes available on March 3
       <sup>rd</sup> 2020, (b) and 331 genomes on March 10
       <sup>th</sup> 2020, (c) 794 genome on March 18
       <sup>th</sup> 2020, and (d) 1,660 genome on March 25
       <sup>th</sup> 2020.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F4" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Phylogeographic cladogram of SARS-CoV-2 on March 3rd 2020. Ancestral state reconstruction performed with TreeTime on 169 full genomes of SARS-CoV-2 collected on March 3rd 2020. Branches are colored by countries of origin." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1934261073" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Phylogeographic cladogram of SARS-CoV-2 on March 3&lt;;sup&gt;;rd&lt;;/sup&gt;; 2020. Ancestral state reconstruction performed with TreeTime on 169 full genomes of SARS-CoV-2 collected on March 3&lt;;sup&gt;;rd&lt;;/sup&gt;; 2020. Branches are colored by countries of origin.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F4.medium.gif" width="440" height="285"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure S2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F4.medium.gif" width="440" height="285"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure S2.</span>
      <p id="p-21" class="first-child">Phylogeographic cladogram of SARS-CoV-2 on March 3
       <sup>rd</sup> 2020. Ancestral state reconstruction performed with TreeTime on 169 full genomes of SARS-CoV-2 collected on March 3
       <sup>rd</sup> 2020. Branches are colored by countries of origin.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F5" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Regression analysis of temporal resolution of SARS-CoV-2 genomes on March 18th and March 25th 2020 datasets. The plots represent linear regression of root-to-tip genetic distance within the ML phylogeny against sampling time for each taxa. Temporal resolution was assessed using the slope of the regression, with positive slope indicating sufficient temporal signal for datasets collecet on (a) March 18th and (b) March 25th 2020. Correlation coefficient [[#x201C]]r[[#x201D]] are reported for each genomic fragment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1934261073" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Regression analysis of temporal resolution of SARS-CoV-2 genomes on March 18&lt;;sup&gt;;th&lt;;/sup&gt;; and March 25&lt;;sup&gt;;th&lt;;/sup&gt;; 2020 datasets. The plots represent linear regression of root-to-tip genetic distance within the ML phylogeny against sampling time for each taxa. Temporal resolution was assessed using the slope of the regression, with positive slope indicating sufficient temporal signal for datasets collecet on (a) March 18&lt;;sup&gt;;th&lt;;/sup&gt;; and (b) March 25&lt;;sup&gt;;th&lt;;/sup&gt;; 2020. Correlation coefficient [[#x201C]]r[[#x201D]] are reported for each genomic fragment.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F5.medium.gif" width="440" height="188"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure S3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F5.medium.gif" width="440" height="188"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S3." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure S3.</span>
      <p id="p-22" class="first-child">Regression analysis of temporal resolution of SARS-CoV-2 genomes on March 18
       <sup>th</sup> and March 25
       <sup>th</sup> 2020 datasets. The plots represent linear regression of root-to-tip genetic distance within the ML phylogeny against sampling time for each taxa. Temporal resolution was assessed using the slope of the regression, with positive slope indicating sufficient temporal signal for datasets collecet on (a) March 18
       <sup>th</sup> and (b) March 25
       <sup>th</sup> 2020. Correlation coefficient “r” are reported for each genomic fragment.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F6" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Phylogeographic reconstruction of Subclade A clade as of March 18th. Ancestral state reconstruction performed with TreeTime on Subclade A with genome collected on March 18th 2020. Branches are colored by countries of origin." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1934261073" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Phylogeographic reconstruction of Subclade A clade as of March 18th. Ancestral state reconstruction performed with TreeTime on Subclade A with genome collected on March 18&lt;;sup&gt;;th&lt;;/sup&gt;; 2020. Branches are colored by countries of origin.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure S4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F6.medium.gif" width="440" height="274"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure S4." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F6.medium.gif" width="440" height="274"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure S4.</span>
      <p id="p-23" class="first-child">Phylogeographic reconstruction of Subclade A clade as of March 18th. Ancestral state reconstruction performed with TreeTime on Subclade A with genome collected on March 18
       <sup>th</sup> 2020. Branches are colored by countries of origin.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F7" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Transmission matrix of SARS-CoV-2 of Subclade A as of March 18th 2020. Transmission matrix indicating for each pairs of individuals how many intermediates there are in the transmission chain are given for Subclade A as of March 18th 2020." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1934261073" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Transmission matrix of SARS-CoV-2 of Subclade A as of March 18&lt;;sup&gt;;th&lt;;/sup&gt;; 2020. Transmission matrix indicating for each pairs of individuals how many intermediates there are in the transmission chain are given for Subclade A as of March 18&lt;;sup&gt;;th&lt;;/sup&gt;; 2020.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure S5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F7.medium.gif" width="440" height="313"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure S5." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F7.medium.gif" width="440" height="313"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S5." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure S5.</span>
      <p id="p-24" class="first-child">Transmission matrix of SARS-CoV-2 of Subclade A as of March 18
       <sup>th</sup> 2020. Transmission matrix indicating for each pairs of individuals how many intermediates there are in the transmission chain are given for Subclade A as of March 18
       <sup>th</sup> 2020.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F8" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F8.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Phylogenetic signal for local SARS-CoV-2 datasets. Presence of phylogenetic signal was evaluated by likelihood mapping checking for alternative topologies (tips), unresolved quartets (center) and partly resolved quartets (edges) for genomes available on genome on March 30th from (a) USA (612 genomes, 121,146 confirmed cases), (b) Italy (27 genomes, 97,689 confirmed cases), (c) Spain (40 genomes, 80,110 confirmed cases), (d) China (300 genomes, 82,122 confirmed cases), (e) Germany (27 genomes, 62,095 confirmed cases) and (f) France (119 genomes, 40,708 confirmed cases on March 29th). Presence of phylogenetic signal (" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1934261073" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Phylogenetic signal for local SARS-CoV-2 datasets. Presence of phylogenetic signal was evaluated by likelihood mapping checking for alternative topologies (tips), unresolved quartets (center) and partly resolved quartets (edges) for genomes available on genome on March 30&lt;;sup&gt;;th&lt;;/sup&gt;; from (a) USA (612 genomes, 121,146 confirmed cases), (b) Italy (27 genomes, 97,689 confirmed cases), (c) Spain (40 genomes, 80,110 confirmed cases), (d) China (300 genomes, 82,122 confirmed cases), (e) Germany (27 genomes, 62,095 confirmed cases) and (f) France (119 genomes, 40,708 confirmed cases on March 29&lt;;sup&gt;;th&lt;;/sup&gt;;). Presence of phylogenetic signal (&lt;;40% unresolved quartets in the center) was detected only for: Germany (27 genomes, 34 variant sites [[#x2013]] 0.2% of total sites in the genome - 15 parsimony informative), with D[[#xFC]]sseldorf and North Rhine Westphalia being the most contributing regions, respectively 12 and 11 genomes; Spain (40 genomes, 60 variant sites - 0.2% of total sites in the genome [[#x2013]] 23 parsimony informative) with Madrid and Comunidad Valenciana being the most contributing regions, respectively 18 and 10 genomes; and France (119 genomes, 155 variant sites [[#x2013]] 0.5% of total sites in the genome - 44 parsimony informative) with Auvergne-Rh[[#xF4]]ne-Alpes, Hauts de France, and Bretagne being the most contributing regions, respectively 42, 30, and 13 genomes&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure S6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F8.medium.gif" width="440" height="219"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure S6." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F8.medium.gif" width="440" height="219"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S6." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure S6.</span>
      <p id="p-25" class="first-child">Phylogenetic signal for local SARS-CoV-2 datasets. Presence of phylogenetic signal was evaluated by likelihood mapping checking for alternative topologies (tips), unresolved quartets (center) and partly resolved quartets (edges) for genomes available on genome on March 30
       <sup>th</sup> from (a) USA (612 genomes, 121,146 confirmed cases), (b) Italy (27 genomes, 97,689 confirmed cases), (c) Spain (40 genomes, 80,110 confirmed cases), (d) China (300 genomes, 82,122 confirmed cases), (e) Germany (27 genomes, 62,095 confirmed cases) and (f) France (119 genomes, 40,708 confirmed cases on March 29
       <sup>th</sup>). Presence of phylogenetic signal (&lt;;40% unresolved quartets in the center) was detected only for: Germany (27 genomes, 34 variant sites – 0.2% of total sites in the genome - 15 parsimony informative), with Düsseldorf and North Rhine Westphalia being the most contributing regions, respectively 12 and 11 genomes; Spain (40 genomes, 60 variant sites - 0.2% of total sites in the genome – 23 parsimony informative) with Madrid and Comunidad Valenciana being the most contributing regions, respectively 18 and 10 genomes; and France (119 genomes, 155 variant sites – 0.5% of total sites in the genome - 44 parsimony informative) with Auvergne-Rhône-Alpes, Hauts de France, and Bretagne being the most contributing regions, respectively 42, 30, and 13 genomes
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F9" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F9.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Regression analysis of temporal resolution of SARS-CoV-2 genomes datasets from France, Spain and Germany. The plots represent linear regression of root-to-tip genetic distance within the ML phylogeny against sampling time for each taxa. Temporal resolution was assessed using the slope of the regression, with positive slope indicating sufficient temporal signal for datasets collecet on (a) France, (b) Spain and (c) Germany. Correlation coefficient [[#x201C]]r[[#x201D]] are reported for each genomic fragment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1934261073" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Regression analysis of temporal resolution of SARS-CoV-2 genomes datasets from France, Spain and Germany. The plots represent linear regression of root-to-tip genetic distance within the ML phylogeny against sampling time for each taxa. Temporal resolution was assessed using the slope of the regression, with positive slope indicating sufficient temporal signal for datasets collecet on (a) France, (b) Spain and (c) Germany. Correlation coefficient [[#x201C]]r[[#x201D]] are reported for each genomic fragment.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure S7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F9.medium.gif" width="440" height="127"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure S7." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F9.medium.gif" width="440" height="127"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S7." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure S7.</span>
      <p id="p-26" class="first-child">Regression analysis of temporal resolution of SARS-CoV-2 genomes datasets from France, Spain and Germany. The plots represent linear regression of root-to-tip genetic distance within the ML phylogeny against sampling time for each taxa. Temporal resolution was assessed using the slope of the regression, with positive slope indicating sufficient temporal signal for datasets collecet on (a) France, (b) Spain and (c) Germany. Correlation coefficient “r” are reported for each genomic fragment.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F10" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F10.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Transmission matrix of SARS-CoV-2 in France. Transmission matrix indicates for each pairs of individuals how many intermediates there are in the transmission chain are given for France (119 genomes as of March 30th 2020, 40,708 confirmed cases as of March 29th 2020)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1934261073" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Transmission matrix of SARS-CoV-2 in France. Transmission matrix indicates for each pairs of individuals how many intermediates there are in the transmission chain are given for France (119 genomes as of March 30&lt;;sup&gt;;th&lt;;/sup&gt;; 2020, 40,708 confirmed cases as of March 29&lt;;sup&gt;;th&lt;;/sup&gt;; 2020).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure S8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F10.medium.gif" width="440" height="314"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure S8." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F10.medium.gif" width="440" height="314"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S8." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.01.020594/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure S8.</span>
      <p id="p-27" class="first-child">Transmission matrix of SARS-CoV-2 in France. Transmission matrix indicates for each pairs of individuals how many intermediates there are in the transmission chain are given for France (119 genomes as of March 30
       <sup>th</sup> 2020, 40,708 confirmed cases as of March 29
       <sup>th</sup> 2020).
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-28">
     <strong>Table S1.</strong> Acknowledgment table with full information of genome sequences. Downloaded from GISAID on March 10
     <sup>th</sup> 2020 and provided as Excel file.
    </p>
    <div id="T1" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1234838/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1234838/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1234838" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table S2.</span>
      <p id="p-29" class="first-child">Testing alternative topologies.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgments</h2>
    <p id="p-19">MS is supported in part by the Stephany W. Holloway University Chair in AIDS Research. We thank all those who have contributed SARS-CoV-2 genome sequences to the GISAID database (
     <a href="https://www.gisaid.org/">https://www.gisaid.org/</a>).
    </p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">WHO</span> (
         <span class="cit-pub-date">2020</span>) 
         <abbr class="cit-jnl-abbrev">World Health Organization. Coronavirus disease 2019 (COVID-19). Situation Report – 43. [online]</abbr>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">L. L. M.</span>
          <span class="cit-name-surname">Poon</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Peiris</span>
         </span>, 
         <span class="cit-article-title">Emergence of a novel human coronavirus threatening human health</span>. 
         <abbr class="cit-jnl-abbrev">Nat Med</abbr> 10.1038/s41591-020-0796-5 (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Wang</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Evolving Epidemiology and Impact of Non-pharmaceutical Interventions on the Outbreak of Coronavirus Disease 2019 in Wuhan, China</span>. 
         <abbr class="cit-jnl-abbrev">medRxiv</abbr> 10.1101/2020.03.03.20030593, 2020.2003.2003.20030593 (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Hadfield</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Nextstrain: real-time tracking of pathogen evolution</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">34</span>, 
         <span class="cit-fpage">4121</span>–
         <span class="cit-lpage">4123</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D4121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Tang</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">On the origin and continuing evolution of SARS-CoV-2</span>. 
         <abbr class="cit-jnl-abbrev">National Science Review</abbr> 10.1093/nsr/nwaa036 (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">O. A.</span>
          <span class="cit-name-surname">MacLean</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Orton</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. B.</span>
          <span class="cit-name-surname">Singer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. L.</span>
          <span class="cit-name-surname">Robertson</span>
         </span> (
         <span class="cit-pub-date">2020</span>) 
         <abbr class="cit-jnl-abbrev">Response to “On the origin and continuing evolution of SARS-CoV-2”</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Kupferschmidt</span>
         </span> (
         <span class="cit-pub-date">2020</span>) 
         <abbr class="cit-jnl-abbrev">Mutations can reveal how the coronavirus moves— but they’re easy to overinterpret. ed Science</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-book" id="cit-2020.04.01.020594v1.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Salemi</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vandamme</span>, 
          <span class="cit-name-given-names">A.-M.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lemey</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-source">The phylogenetic handbook: A practical approach to phylogenetic analysis and hypothesis testing</span>. (
         <span class="cit-publ-name">Cambridge University Press</span>, 
         <span class="cit-publ-loc">Cambridge, UK</span>, 
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Shu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">McCauley</span>
         </span>, 
         <span class="cit-article-title">GISAID: Global initiative on sharing all influenza data - from vision to reality</span>. 
         <abbr class="cit-jnl-abbrev">Euro Surveill</abbr>
         <span class="cit-vol">22</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">Anonymous</span> (
         <span class="cit-pub-date">2020</span>) 
         <abbr class="cit-jnl-abbrev">Sito del Dipartimento della Protezione Civile - Emergenza Coronavirus: la risposta nazionale</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.11" data-doi="10.1093/bioinformatics/18.3.502">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">H. A.</span>
          <span class="cit-name-surname">Schmidt</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Strimmer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Vingron</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">von Haeseler</span>
         </span>, 
         <span class="cit-article-title">TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel computing</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">18</span>, 
         <span class="cit-fpage">502</span>–
         <span class="cit-lpage">504</span> (
         <span class="cit-pub-date">2002</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DSchmidt%26rft.auinit1%253DH.%2BA.%26rft.volume%253D18%26rft.issue%253D3%26rft.spage%253D502%26rft.epage%253D504%26rft.atitle%253DTREE-PUZZLE%253A%2Bmaximum%2Blikelihood%2Bphylogenetic%2Banalysis%2Busing%2B%2Bquartets%2Band%2Bparallel%2Bcomputing%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F18.3.502%26rft_id%253Dinfo%253Apmid%252F11934758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/18.3.502&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=11934758&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.01.020594.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000174708500024&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.12" data-doi="10.1073/pnas.94.13.6815">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Strimmer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">von Haeseler</span>
         </span>, 
         <span class="cit-article-title">Likelihood-mapping: a simple method to visualize phylogenetic content of a sequence alignment</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>
         <span class="cit-vol">94</span>, 
         <span class="cit-fpage">6815</span>–
         <span class="cit-lpage">6819</span> (
         <span class="cit-pub-date">1997</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DStrimmer%26rft.auinit1%253DK.%26rft.volume%253D94%26rft.issue%253D13%26rft.spage%253D6815%26rft.epage%253D6819%26rft.atitle%253DLikelihood-mapping%253A%2BA%2Bsimple%2Bmethod%2Bto%2Bvisualize%2Bphylogenetic%2Bcontent%2Bof%2Ba%2Bsequence%2Balignment%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.13.6815%26rft_id%253Dinfo%253Apmid%252F9192648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTQvMTMvNjgxNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA1LzIwMjAuMDQuMDEuMDIwNTk0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.13" data-doi="10.1093/ve/vew007">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Rambaut</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T. T.</span>
          <span class="cit-name-surname">Lam</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L. Max</span>
          <span class="cit-name-surname">Carvalho</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">O. G.</span>
          <span class="cit-name-surname">Pybus</span>
         </span>, 
         <span class="cit-article-title">Exploring the temporal structure of heterochronous sequences using TempEst (formerly Path-O-Gen)</span>. 
         <abbr class="cit-jnl-abbrev">Virus Evol</abbr>
         <span class="cit-vol">2</span>, 
         <span class="cit-fpage">vew007</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirus%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fve%252Fvew007%26rft_id%253Dinfo%253Apmid%252F27774300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/ve/vew007&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27774300&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.01.020594.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.14" data-doi="10.1186/1471-2148-7-214">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A. J.</span>
          <span class="cit-name-surname">Drummond</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Rambaut</span>
         </span>, 
         <span class="cit-article-title">BEAST: Bayesian evolutionary analysis by sampling trees</span>. 
         <abbr class="cit-jnl-abbrev">BMC Evol Biol</abbr>
         <span class="cit-vol">7</span>, 
         <span class="cit-fpage">214</span> (
         <span class="cit-pub-date">2007</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DDrummond%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D7%26rft.spage%253D214%26rft.epage%253D214%26rft.atitle%253DBEAST%253A%2BBayesian%2Bevolutionary%2Banalysis%2Bby%2Bsampling%2Btrees.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-7-214%26rft_id%253Dinfo%253Apmid%252F17996036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2148-7-214&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17996036&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.01.020594.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.15" data-doi="10.1371/journal.pbio.0040088">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A. J.</span>
          <span class="cit-name-surname">Drummond</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. Y.</span>
          <span class="cit-name-surname">Ho</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. J.</span>
          <span class="cit-name-surname">Phillips</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Rambaut</span>
         </span>, 
         <span class="cit-article-title">Relaxed phylogenetics and dating with confidence</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Biol</abbr>
         <span class="cit-vol">4</span>, 
         <span class="cit-fpage">e88</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDrummond%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253De88%26rft.epage%253De88%26rft.atitle%253DRelaxed%2Bphylogenetics%2Band%2Bdating%2Bwith%2Bconfidence.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040088%26rft_id%253Dinfo%253Apmid%252F16683862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040088&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16683862&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.01.020594.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Didelot</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Fraser</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Gardy</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Colijn</span>
         </span>, 
         <span class="cit-article-title">Genomic Infectious Disease Epidemiology in Partially Sampled and Ongoing Outbreaks</span>. 
         <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>
         <span class="cit-vol">34</span>, 
         <span class="cit-fpage">997</span>–
         <span class="cit-lpage">1007</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft.volume%253D34%26rft.spage%253D997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Jing</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Indirect Virus Transmission in Cluster of COVID-19 Cases, Wenzhou, China, 2020</span>. 
         <abbr class="cit-jnl-abbrev">Emerging Infectious Disease journal</abbr>
         <span class="cit-vol">26</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Pung</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Investigation of three clusters of COVID-19 in Singapore: implications for surveillance and response measures</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>
         <span class="cit-vol">395</span>, 
         <span class="cit-fpage">1039</span>–
         <span class="cit-lpage">1046</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D1039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.19" data-doi="10.1093/jhered/92.4.371">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Xia</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Xie</span>
         </span>, 
         <span class="cit-article-title">DAMBE: software package for data analysis in molecular biology and evolution</span>. 
         <abbr class="cit-jnl-abbrev">J Hered</abbr>
         <span class="cit-vol">92</span>, 
         <span class="cit-fpage">371</span>–
         <span class="cit-lpage">373</span> (
         <span class="cit-pub-date">2001</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DXia%26rft.auinit1%253DX.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D371%26rft.epage%253D373%26rft.atitle%253DDAMBE%253A%2BSoftware%2BPackage%2Bfor%2BData%2BAnalysis%2Bin%2BMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F92.4.371%26rft_id%253Dinfo%253Apmid%252F11535656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/jhered/92.4.371&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=11535656&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.01.020594.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000170799400016&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.20" data-doi="10.1093/molbev/msu300">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">L. T.</span>
          <span class="cit-name-surname">Nguyen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">H. A.</span>
          <span class="cit-name-surname">Schmidt</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">von Haeseler</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B. Q.</span>
          <span class="cit-name-surname">Minh</span>
         </span>, 
         <span class="cit-article-title">IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies</span>. 
         <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>
         <span class="cit-vol">32</span>, 
         <span class="cit-fpage">268</span>–
         <span class="cit-lpage">274</span> (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu300%26rft_id%253Dinfo%253Apmid%252F25371430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/msu300&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25371430&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.01.020594.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.01.020594v1.21" data-doi="10.1093/nar/gkw256">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Trifinopoulos</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L. T.</span>
          <span class="cit-name-surname">Nguyen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">von Haeseler</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B. Q.</span>
          <span class="cit-name-surname">Minh</span>
         </span>, 
         <span class="cit-article-title">W-IQ-TREE: a fast online phylogenetic tool for maximum likelihood analysis</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">44</span>, 
         <span class="cit-fpage">W232</span>–
         <span class="cit-lpage">235</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw256%26rft_id%253Dinfo%253Apmid%252F27084950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkw256&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27084950&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.01.020594.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.01.020594v1.22" data-doi="10.1093/oxfordjournals.molbev.a026201">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Shimodaira</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Hasegawa</span>
         </span>, 
         <span class="cit-article-title">Multiple comparisons of log-likelihoods with applications to phylogenetic inference</span>. 
         <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>
         <span class="cit-vol">16</span>, 
         <span class="cit-fpage">1114</span>–
         <span class="cit-lpage">1116</span> (
         <span class="cit-pub-date">1999</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DShimodaira%26rft.auinit1%253DH%26rft.volume%253D16%26rft.issue%253D8%26rft.spage%253D1114%26rft.epage%253D1116%26rft.atitle%253DMultiple%2BComparisons%2Bof%2BLog-Likelihoods%2Bwith%2BApplications%2Bto%2BPhylogenetic%2BInference%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a026201&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000081925400012&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
