<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2017/11/14/219246.atom" />
<meta name="HW.pisa" content="biorxiv;219246v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Large protein complex production using the SmartBac System - Strategies and Applications" />
<meta name="DC.Identifier" content="10.1101/219246" />
<meta name="DC.Date" content="2017-11-14" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="Â© 2017, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Recent revolution of cryo-electron microscopy has opened a new door to solve high-resolution structures of macromolecule complexes without crystallization while how to efficiently obtain homogenous macromolecule complex sample is therefore becoming a bottleneck. Here we report SmartBac, an easy and versatile system for constructing large-sized transfer plasmids used to generate recombinant baculoviruses that express large multiprotein complexes in insect cells. The SmartBac system integrates the univector plasmid-fusion system, Gibson assembly method and polyprotein strategy to construct the final transfer plasmids. The fluorescent proteins are designed to be co-expressed with recombinant proteins to monitor transfection and expression efficiencies. A scheme of screening an optimal tagged subunit for effective purification is provided. Six large multiprotein complexes including the human exocyst complex and dynactin complex were successfully expressed, suggesting a great potential of SmartBac for its wide application in the future structural biology study." />
<meta name="DC.Contributor" content="Yujia Zhai" />
<meta name="DC.Contributor" content="Danyang Zhang" />
<meta name="DC.Contributor" content="Leiye Yu" />
<meta name="DC.Contributor" content="Fang Sun" />
<meta name="DC.Contributor" content="Fei Sun" />
<meta name="article:published_time" content="2017-11-14" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Large protein complex production using the SmartBac System - Strategies and Applications" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;ABSTRACT&lt;/h3&gt;
&lt;p&gt;Recent revolution of cryo-electron microscopy has opened a new door to solve high-resolution structures of macromolecule complexes without crystallization while how to efficiently obtain homogenous macromolecule complex sample is therefore becoming a bottleneck. Here we report SmartBac, an easy and versatile system for constructing large-sized transfer plasmids used to generate recombinant baculoviruses that express large multiprotein complexes in insect cells. The SmartBac system integrates the univector plasmid-fusion system, Gibson assembly method and polyprotein strategy to construct the final transfer plasmids. The fluorescent proteins are designed to be co-expressed with recombinant proteins to monitor transfection and expression efficiencies. A scheme of screening an optimal tagged subunit for effective purification is provided. Six large multiprotein complexes including the human exocyst complex and dynactin complex were successfully expressed, suggesting a great potential of SmartBac for its wide application in the future structural biology study.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2017/01/01" />
<meta name="citation_mjid" content="biorxiv;219246v1" />
<meta name="citation_id" content="219246v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/219246v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/219246v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/219246v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246.full.pdf" />
<meta name="citation_doi" content="10.1101/219246" />
<meta name="citation_num_pages" content="36" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="219246" />
<meta name="citation_author" content="Yujia Zhai" />
<meta name="citation_author_institution" content="National Key Laboratory of Biomacromolecules, CAS Center for Excellence in Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences" />
<meta name="citation_author_institution" content="School of Life Sciences, University of Chinese Academy of Sciences" />
<meta name="citation_author_email" content="yujia@ibp.ac.cn" />
<meta name="citation_author_email" content="feisun@ibp.ac.cn" />
<meta name="citation_author" content="Danyang Zhang" />
<meta name="citation_author_institution" content="National Key Laboratory of Biomacromolecules, CAS Center for Excellence in Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences" />
<meta name="citation_author_institution" content="School of Life Sciences, University of Chinese Academy of Sciences" />
<meta name="citation_author" content="Leiye Yu" />
<meta name="citation_author_institution" content="National Key Laboratory of Biomacromolecules, CAS Center for Excellence in Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences" />
<meta name="citation_author_institution" content="School of Life Sciences, University of Chinese Academy of Sciences" />
<meta name="citation_author" content="Fang Sun" />
<meta name="citation_author_institution" content="National Key Laboratory of Biomacromolecules, CAS Center for Excellence in Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences" />
<meta name="citation_author_institution" content="School of Life Sciences, University of Chinese Academy of Sciences" />
<meta name="citation_author" content="Fei Sun" />
<meta name="citation_author_institution" content="National Key Laboratory of Biomacromolecules, CAS Center for Excellence in Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences" />
<meta name="citation_author_institution" content="School of Life Sciences, University of Chinese Academy of Sciences" />
<meta name="citation_author_institution" content="Center for Biological Imaging, Core Facilities for Protein Science, Institute of Biophysics" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0351-5144" />
<meta name="citation_reference" content="Yan, C. et al. Structure of a yeast spliceosome at 3.6-angstrom resolution. Science (New York, N.Y 349, 1182-1191, doi:10.1126/science.aac7629 (2015)." />
<meta name="citation_reference" content="Nguyen, T. H. D. et al. Cryo-EM structure of the yeast U4/U6.U5 tri-snRNP at 3.7 A resolution. Nature 530, 298-302, doi:10.1038/nature16940 (2016)." />
<meta name="citation_reference" content="Yan, Z. et al. Structure of the rabbit ryanodine receptor RyR1 at near-atomic resolution. Nature 517, 50-55, doi:10.1038/nature14063 (2015)." />
<meta name="citation_reference" content="des Georges, A. et al. Structural Basis for Gating and Activation of RyR1. Cell 167, 145-157 e117, doi:10.1016/j.cell.2016.08.075 (2016)." />
<meta name="citation_reference" content="Wei, R. et al. Structural insights into Ca(2+)-activated long-range allosteric channel gating of RyR1. Cell Res 26, 977-994, doi:10.1038/cr.2016.99 (2016)." />
<meta name="citation_reference" content="Chang, L., Zhang, Z., Yang, J., McLaughlin, S. H. &amp; Barford, D. Atomic structure of the APC/C and its mechanism of protein ubiquitination. Nature 522, 450-454, doi:10.1038/nature14471 (2015)." />
<meta name="citation_reference" content="Wei, X. et al. Structure of spinach photosystem II-LHCII supercomplex at 3.2 A resolution. Nature 534, 69-74, doi:10.1038/nature18020 (2016)." />
<meta name="citation_reference" content="Gu, J. et al. The architecture of the mammalian respirasome. Nature 537, 639-643, doi:10.1038/nature19359 (2016)." />
<meta name="citation_reference" content="Letts, J. A., Fiedorczuk, K. &amp; Sazanov, L. A. The architecture of respiratory supercomplexes. Nature 537, 644-648, doi:10.1038/nature19774 (2016)." />
<meta name="citation_reference" content="Jarvis, D. L. Baculovirus-insect cell expression systems. Methods Enzymol 463, 191-222, doi:10.1016/S0076-6879(09)63014-7 (2009)." />
<meta name="citation_reference" content="Luckow, V. A., Lee, S. C., Barry, G. F. &amp; Olins, P. O. Efficient generation of infectious recombinant baculoviruses by site-specific transposon-mediated insertion of foreign genes into a baculovirus genome propagated in Escherichia coli. J Virol 67, 4566-4579 (1993)." />
<meta name="citation_reference" content="Hitchman, R. B., Possee, R. D. &amp; King, L. A. High-throughput baculovirus expression in insect cells. Methods Mol Biol 824, 609-627, doi:10.1007/978-1-61779-433-9_33 (2012)." />
<meta name="citation_reference" content="Birnbaum, M. E. et al. Molecular architecture of the alphabeta T cell receptor-CD3 complex. Proc Natl Acad Sci U S A 111, 17576-17581, doi:10.1073/pnas.1420936111 (2014)." />
<meta name="citation_reference" content="Chung, Y. C. et al. Expression, purification and characterization of enterovirus-71 virus-like particles. World J Gastroenterol 12, 921-927 (2006)." />
<meta name="citation_reference" content="Hu, Y. C., Hsu, J. T., Huang, J. H., Ho, M. S. &amp; Ho, Y. C. Formation of enterovirus-like particle aggregates by recombinant baculoviruses co-expressing P1 and 3CD in insect cells. Biotechnol Lett 25, 919-925 (2003)." />
<meta name="citation_reference" content="Kee, Y. et al. WDR20 regulates activity of the USP12 Ã UAF1 deubiquitinating enzyme complex. J Biol Chem 285, 11252-11257, doi:10.1074/jbc.M109.095141 (2010)." />
<meta name="citation_reference" content="Wan, L. C. et al. Proteomic analysis of the human KEOPS complex identifies C14ORF142 as a core subunit homologous to yeast Gon7. Nucleic acids research 45, 805-817, doi:10.1093/nar/gkw1181 (2017)." />
<meta name="citation_reference" content="Yoo, H. Y., Kumagai, A., Shevchenko, A., Shevchenko, A. &amp; Dunphy, W. G. The Mre11-Rad50-Nbs1 complex mediates activation of TopBP1 by ATM. Mol Biol Cell 20, 2351-2360, doi:10.1091/mbc.E08-12-1190 (2009)." />
<meta name="citation_reference" content="Belyaev, A. S. &amp; Roy, P. Development of baculovirus triple and quadruple expression vectors: co-expression of three or four bluetongue virus proteins and the synthesis of bluetongue virus-like particles in insect cells. Nucleic acids research 21, 1219-1223 (1993)." />
<meta name="citation_reference" content="Berger, I., Fitzgerald, D. J. &amp; Richmond, T. J. Baculovirus expression system for heterologous multiprotein complexes. Nat Biotechnol 22, 1583-1587, doi:10.1038/nbt1036 (2004)." />
<meta name="citation_reference" content="Bartlam, M., Yang, H. &amp; Rao, Z. Structural insights into SARS coronavirus proteins. Curr Opin Struct Biol 15, 664-672, doi:10.1016/j.sbi.2005.10.004 (2005)." />
<meta name="citation_reference" content="Ziebuhr, J., Snijder, E. J. &amp; Gorbalenya, A. E. Virus-encoded proteinases and proteolytic processing in the Nidovirales. J Gen Virol 81, 853-879, doi:10.1099/0022-1317-81-4-853 (2000)." />
<meta name="citation_reference" content="Sawicki, S. G., Sawicki, D. L. &amp; Siddell, S. G. A contemporary view of coronavirus transcription. J Virol 81, 20-29, doi:10.1128/JVI.01358-06 (2007)." />
<meta name="citation_reference" content="Vijayachandran, L. S. et al. Robots, pipelines, polyproteins: enabling multiprotein expression in prokaryotic and eukaryotic cells. J Struct Biol 175, 198-208, doi:10.1016/j.jsb.2011.03.007 (2011)." />
<meta name="citation_reference" content="Nie, Y., Bellon-Echeverria, I., Trowitzsch, S., Bieniossek, C. &amp; Berger, I. Multiprotein complex production in insect cells by using polyproteins. Methods Mol Biol 1091, 131-141, doi:10.1007/978-1-62703-691-7_8 (2014)." />
<meta name="citation_reference" content="Liu, Q., Li, M. Z., Leibham, D., Cortez, D. &amp; Elledge, S. J. The univector plasmid-fusion system, a method for rapid construction of recombinant DNA without restriction enzymes. Curr Biol 8, 1300-1309 (1998)." />
<meta name="citation_reference" content="Metcalf, W. W., Jiang, W. &amp; Wanner, B. L. Use of the rep technique for allele replacement to construct new Escherichia coli hosts for maintenance of R6K gamma origin plasmids at different copy numbers. Gene 138, 1-7 (1994)." />
<meta name="citation_reference" content="Filutowicz, M. et al. Role of the pi initiation protein and direct nucleotide sequence repeats in the regulation of plasmid R6K replication. Basic Life Sci 30, 125-140 (1985)." />
<meta name="citation_reference" content="Hill-Perkins, M. S. &amp; Possee, R. D. A baculovirus expression vector derived from the basic protein promoter of Autographa californica nuclear polyhedrosis virus. J Gen Virol 71 (Pt 4), 971-976, doi:10.1099/0022-1317-71-4-971 (1990)." />
<meta name="citation_reference" content="van Oers, M. M. Opportunities and challenges for the baculovirus expression system. J Invertebr Pathol 107 Suppl, S3-15, doi:10.1016/j.jip.2011.05.001 (2011)." />
<meta name="citation_reference" content="Ishiyama, S. &amp; Ikeda, M. High-level expression and improved folding of proteins by using the vp39 late promoter enhanced with homologous DNA regions. Biotechnol Lett 32, 1637-1647, doi:10.1007/s10529-010-0340-7 (2010)." />
<meta name="citation_reference" content="Li, S. F., Wang, H. L., Hu, Z. H. &amp; Deng, F. Genetic modification of baculovirus expression vectors. Virol Sin 27, 71-82, doi:10.1007/s12250-012-3236-y (2012)." />
<meta name="citation_reference" content="Gibson, D. G. et al. Enzymatic assembly of DNA molecules up to several hundred kilobases. Nat Methods 6, 343-345, doi:10.1038/nmeth.1318 (2009)." />
<meta name="citation_reference" content="Pijlman, G. P., van Schijndel, J. E. &amp; Vlak, J. M. Spontaneous excision of BAC vector sequences from bacmid-derived baculovirus expression vectors upon passage in insect cells. J Gen Virol 84, 2669-2678, doi:10.1099/vir.0.19438-0 (2003)." />
<meta name="citation_reference" content="Wu, B. &amp; Guo, W. The Exocyst at a Glance. J Cell Sci 128, 2957-2964, doi:10.1242/jcs.156398 (2015)." />
<meta name="citation_reference" content="Heider, M. R. et al. Subunit connectivity, assembly determinants and architecture of the yeast exocyst complex. Nat Struct Mol Biol 23, 59-66, doi:10.1038/nsmb.3146 (2016)." />
<meta name="citation_reference" content="Reck-Peterson, S. L. Dynactin revealed. Nat Struct Mol Biol 22, 359-360, doi:10.1038/nsmb.3022 (2015)." />
<meta name="citation_reference" content="Urnavicius, L. et al. The structure of the dynactin complex and its interaction with dynein. Science (New York, N.Y) 347, 1441-1446, doi:10.1126/science.aaa4080 (2015)." />
<meta name="citation_reference" content="Wang, S. et al. Structural characterization of coatomer in its cytosolic state. Protein Cell 7, 586-600, doi:10.1007/s13238-016-0296-z (2016)." />
<meta name="citation_reference" content="Zhang, K. et al. Cryo-EM Reveals How Human Cytoplasmic Dynein Is Auto-inhibited and Activated. Cell 169, 1303-1314 e1318, doi:10.1016/j.cell.2017.05.025 (2017)." />
<meta name="citation_reference" content="Mosadeghi, R. et al. Structural and kinetic analysis of the COP9-Signalosome activation and the cullin-RING ubiquitin ligase deneddylation cycle. Elife 5, doi:10.7554/eLife.12102 (2016)." />
<meta name="citation_reference" content="Zheng, N. et al. Structure of the Cul1-Rbx1-Skp1-F boxSkp2 SCF ubiquitin ligase complex. Nature 416, 703-709, doi:10.1038/416703a (2002)." />
<meta name="citation_reference" content="Berger, I. et al. The multiBac protein complex production platform at the EMBL. J Vis Exp, e50159, doi:10.3791/50159 (2013)." />
<meta name="citation_reference" content="Chen, X., Pham, E. &amp; Truong, K. TEV protease-facilitated stoichiometric delivery of multiple genes using a single expression vector. Protein Sci 19, 2379-2388, doi:10.1002/pro.518 (2010)." />
<meta name="citation_reference" content="Bieniossek, C., Imasaki, T., Takagi, Y. &amp; Berger, I. MultiBac: expanding the research toolbox for multiprotein complexes. Trends Biochem Sci 37, 49-57, doi:10.1016/j.tibs.2011.10.005 (2012)." />
<meta name="twitter:title" content="Large protein complex production using the SmartBac System - Strategies and Applications" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Recent revolution of cryo-electron microscopy has opened a new door to solve high-resolution structures of macromolecule complexes without crystallization while how to efficiently obtain homogenous macromolecule complex sample is therefore becoming a bottleneck. Here we report SmartBac, an easy and versatile system for constructing large-sized transfer plasmids used to generate recombinant baculoviruses that express large multiprotein complexes in insect cells. The SmartBac system integrates the univector plasmid-fusion system, Gibson assembly method and polyprotein strategy to construct the final transfer plasmids. The fluorescent proteins are designed to be co-expressed with recombinant proteins to monitor transfection and expression efficiencies. A scheme of screening an optimal tagged subunit for effective purification is provided. Six large multiprotein complexes including the human exocyst complex and dynactin complex were successfully expressed, suggesting a great potential of SmartBac for its wide application in the future structural biology study." />
<meta name="og-title" property="og:title" content="Large protein complex production using the SmartBac System - Strategies and Applications" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/219246v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Recent revolution of cryo-electron microscopy has opened a new door to solve high-resolution structures of macromolecule complexes without crystallization while how to efficiently obtain homogenous macromolecule complex sample is therefore becoming a bottleneck. Here we report SmartBac, an easy and versatile system for constructing large-sized transfer plasmids used to generate recombinant baculoviruses that express large multiprotein complexes in insect cells. The SmartBac system integrates the univector plasmid-fusion system, Gibson assembly method and polyprotein strategy to construct the final transfer plasmids. The fluorescent proteins are designed to be co-expressed with recombinant proteins to monitor transfection and expression efficiencies. A scheme of screening an optimal tagged subunit for effective purification is provided. Six large multiprotein complexes including the human exocyst complex and dynactin complex were successfully expressed, suggesting a great potential of SmartBac for its wide application in the future structural biology study." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2017-11-14" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/219246v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/219246v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/219246v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/219246v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/66807" />
    <title>Large protein complex production using the SmartBac System - Strategies and Applications | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"iGEhy_GSB8VOTnK21DqdgYjlxTggBU76ipsLZz1pxzg","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"66807","apath":"\/biorxiv\/early\/2017\/11\/14\/219246.atom","pisa":"biorxiv;219246v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;219246v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/219246v1","title":"Large protein complex production using the SmartBac System - Strategies and Applications","identifier":"node\/66807"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SvnB3ebRY0vdERb4v5zWJQMwlqCJwywtu1JjY93vD33azPyNmhN3MTCn1RW1CxtDi7HXwP2pWwomHf1nVyA3auITylys5s3r7SptVNNYWuUMI92WtOKujisEtXIO6uge2oR5ZqmbtKJCntGPdTj5kW6vaQZbsB5iORXK443Wz65d3HbmGDmdHOUtaVhEqRAkMlaBvTPITOyTjR8\\\\\\\/aQnWKY1QRu4+A8ykM\\\\\\\/vk2Ft5Xm4HcGDlW4h\\\\\\\/3YaSdRlbdRqZYxWMKHz9qaoGLXFmF+C0XqPEOC0tbHJle3NNSf1pD9oSHEXDd3FMu4asc9nlquE4tQfg79oHan0+qPnMVn7vQW8oqtj7wb1if+llNpNe3AUCDCYqUb6adj5sqxMqiH9cCKsq82lsPWxSOcuR9\\\\\\\/s87ySMF5GS91bmRXVqXc19ktnSijT6TNm+NDWx4PSMe\\\\\\\/fJ3m\\\\\\\/+j7gbnSpDYneelh33Te5y7K2hGu0QBeMcsxZ3SH5gpXHEtAxEPu6TjGSmz0CDRLpp8rOkuSpnbIZpWcHEcK5kO12JLEMYZoOnCluHyJjc+OOUTTTGZarCuNsRRNy9NjPQZQ3DF5a6hfgL\\\\\\\/\\\\\\\/DLfjMMcqTrrnVguwPpHmOmdYzhMLF8KNl5WfZYNNAvgv1KzM7nh1Vj9insAjy345rC\\\\\\\/Zq0Rp1bGmrJa3GQr307+6vdXKlGoueRWfn\\\\\\\/DUbCkqOca6sxYX1Ph3i3X9PXHTKL8QuY+Rg8tdG36EGhnsAfopj4DLVHxjM+tCw0gERb+0PR7\\\\\\\/KYJQA5RCORXCtiBc9HOfXIjXOwR83dFl+KY5SSZME+9q6+rZ1UCgAY1VKfynhIxGJz07ZHItt83hV4OD3n50cBZBrf\\\\\\\/c2TYE27s+seqMQSWGutfc5ZMWIVZq5nRCaOC\\\\\\\/8VMeSjAKbv4qEKaj3lGYyuyfLqUB6FthHRnS502GO6QW9L747N4ap66km5ifJMcmA8suvsw6GbeZjuvVO89UBHCnz6qGLbgGk1s7UxZYM8KNewK7Hz1KWWS1iaTBI3lG9p1aHavM10kEQieN8C7f6kO2hd41JXjk52nF5PRYhdmQU\\\\\\\/SCEG5COXu1\\\\\\\/DvfCQJo+GKMf7hgwqzoybSv\\\\\\\/yWL+a8rL7zrO5kNyNFcglrijalu4uoXnmT6G\\\\\\\/\\\\\\\/jrE2+o8gGFBAhqCAGPgwCL12DZvapo4S8jXeqnf5\\\\\\\/mkUx9uHSO8+goWlMn5iD6TPL0j6S78yPSqzntYzV7AujOtwu18hQM3RED2nVAyEB83UOlhxKSfGtSfRrUtSNYqwtn97ZGn8hhS7Ke\\\\\\\/FjDQnmus86i7rOh0uLKF3IyuE4ks6RSEynk=\\\u0022,\\\u0022iv\\\u0022:\\\u0022CMmtCC8388ztYfgRwGcG8Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022ba72374960a1e8ffa79db646f8dd6eb4c9151fc3197b67df5cfea03db0eceab5\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/219246v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-66807 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/219246v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-F2kR2o-JuSuSeiJODybUeiaLzG6bsgyUIEPVAXWfcrw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="66807" id="node66807" data-pisa="biorxiv;219246v1" data-pisa-master="biorxiv;219246" data-apath="/biorxiv/early/2017/11/14/219246.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Large protein complex production using the SmartBac System - Strategies and Applications</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Yujia</span> <span class="nlm-surname">Zhai</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Danyang</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Leiye</span> <span class="nlm-surname">Yu</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Fang</span> <span class="nlm-surname">Sun</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-0351-5144" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Fei</span> <span class="nlm-surname">Sun</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/219246 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node66807" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yujia Zhai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>National Key Laboratory of Biomacromolecules, CAS Center for Excellence in Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences</span>, Beijing 100101, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>School of Life Sciences, University of Chinese Academy of Sciences</span>, Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yujia%2BZhai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhai%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYujia%2BZhai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yujia@ibp.ac.cn" class="" data-icon-position="" data-hide-link-title="0">yujia@ibp.ac.cn</a>
<a href="mailto:feisun@ibp.ac.cn" class="" data-icon-position="" data-hide-link-title="0">feisun@ibp.ac.cn</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Danyang Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>National Key Laboratory of Biomacromolecules, CAS Center for Excellence in Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences</span>, Beijing 100101, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>School of Life Sciences, University of Chinese Academy of Sciences</span>, Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danyang%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanyang%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Leiye Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>National Key Laboratory of Biomacromolecules, CAS Center for Excellence in Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences</span>, Beijing 100101, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>School of Life Sciences, University of Chinese Academy of Sciences</span>, Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leiye%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeiye%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fang Sun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>National Key Laboratory of Biomacromolecules, CAS Center for Excellence in Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences</span>, Beijing 100101, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>School of Life Sciences, University of Chinese Academy of Sciences</span>, Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fang%2BSun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sun%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFang%2BSun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Fei Sun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>National Key Laboratory of Biomacromolecules, CAS Center for Excellence in Biomacromolecules, Institute of Biophysics, Chinese Academy of Sciences</span>, Beijing 100101, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>School of Life Sciences, University of Chinese Academy of Sciences</span>, Beijing, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Center for Biological Imaging, Core Facilities for Protein Science, Institute of Biophysics</span>, CAS, Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fei%2BSun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sun%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFei%2BSun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0351-5144" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fei Sun</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/219246v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:66807" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:66807/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/219246v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:66807" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:66807/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/219246v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:66807" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:66807/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/219246v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:66807" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:66807/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/219246v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:66807" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:66807/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-2">Recent revolution of cryo-electron microscopy has opened a new door to solve high-resolution structures of macromolecule complexes without crystallization while how to efficiently obtain homogenous macromolecule complex sample is therefore becoming a bottleneck. Here we report SmartBac, an easy and versatile system for constructing large-sized transfer plasmids used to generate recombinant baculoviruses that express large multiprotein complexes in insect cells. The SmartBac system integrates the univector plasmid-fusion system, Gibson assembly method and polyprotein strategy to construct the final transfer plasmids. The fluorescent proteins are designed to be co-expressed with recombinant proteins to monitor transfection and expression efficiencies. A scheme of screening an optimal tagged subunit for effective purification is provided. Six large multiprotein complexes including the human exocyst complex and dynactin complex were successfully expressed, suggesting a great potential of SmartBac for its wide application in the future structural biology study.</p></div><div class="section" id="sec-1"><h2 class="">INTRODUCTION</h2><p id="p-3">With the rapid development of single-particle cryo-electron microscopy (cryo-EM), more and more macromolecular machineries structures, e.g. spliceosome<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>, ryanodine receptor<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>-<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup>, anaphase promoting complex<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup>, light harvest complex<sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup> and mitochondrial respirasome<sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup> harvest complex and mitochondrial respirasome, have been solved to near-atomic resolution, which have been waiting for many years. Since cryo-EM does not need crystals, many macromolecular complexes that are difficult to be crystallized are now ready for structural studies; thus, more and more structural biology laboratories are becoming interested in studying the structures of large macromolecular complexes. However, how to obtain enough highly purified specimen suitable for cryo-EM is therefore becoming a new bottleneck, which has restricted the wide application of cryo-EM technology.</p><p id="p-4">One method for obtaining large protein complexes is to extract them from biological tissues. However, this method is not very suitable for low-abundance samples. In addition to consuming large amounts of reagents, protein extraction usually yields samples with low yield and purity. And even for high-abundance samples, it is still difficult to prepare and purify mutant proteins for functional studies. Thus, recombinant expression of protein complexes is more commonly preferred.</p><p id="p-5">The baculovirus expression system (BVES) is a powerful tool for recombinant protein production<sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup> because it is safe, high expression levels can be achieved, and post-translational modifications can be incorporated. The most common baculovirus used for gene expression is AcMNPV (<em>Autographa californica</em> multiple nuclear polyhedrosis virus), which has a large, circular double-stranded DNA genome (about 130kb) that can accommodate very large exogenous DNA fragments <sup><a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a></sup>. However, it is not easy to introduce foreign genes by conventional molecular cloning methods due to the large size of AcMNPV genome. Therefore, researchers have modified the AcMNPV genome to allow effective foreign gene insertion by siteâspecific transposition or homogenous recombination<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>. The widely-used Bac to Bac baculovirus expression system (Invitrogen Co.) is an example of the successful use of this approach. Three strategies are commonly used to overexpress multiprotein complexes in insect cells. In the first strategy insect cells are infected with multiple types of baculoviruses, each of which carries one or two gene expression cassettes (GECs). This strategy, which involves molecular cloning, is relatively simple and has been successfully applied to the Bac to Bac system by many research groups<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>-<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup>. However, when multiple types of baculoviruses are used, the total number of infectious viruses added to the expression culture should be comparable to the total number added when the culture is infected with a single baculovirus. This will inevitably lead to a lower protein yield. In addition, the expression levels of the individual subunits are often imbalanced, which can result in improper complex assembly.</p><p id="p-6">The second strategy used to express multiprotein complexes is to construct a transfer plasmid carrying multiple GECs. The commercial pFastbac-Dual vector features two promoters for expression of two proteins simultaneously. Similar triple or quadruple expression vectors have also been built using traditional molecular cloning methods <sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>. The MultiBac system generates multi-GEC donor and acceptor vectors from junior plasmids carrying individual GECs by homing endonuclease-based multiplication module <sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup>. Then the final transfer plasmid is produced by Cre-mediated recombination between the donor and acceptor vectors <sup><a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a></sup>. Yet the problem of unequal subunit stoichiometry still exists.</p><p id="p-7">The third strategy is the polyprotein strategy that has been used by coronaviruses to produce multiple functional nonstructural proteins (nsps), which are involved in the formation of the replicase-transcription complex (RTC) that mediates viral replication and transcription<sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>. The nsps are encoded in open-reading frame 1a (ORF1a) and ORF1b and are synthesized initially as two large polyproteins, pp1a and pp1ab. During or after synthesis, these poliproteins are cleaved by virus-encoded proteases into 16 nsps<sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>. These nsps, together with other viral proteins and, possibly, cellular proteins, assemble into membrane-bound RTC<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup>. This strategy has been exploited to produce protein complexes in the baculovirus system in recent years<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup>.By this strategy, idividual subunits are separated by protease cleavage sites and expressed as a long polyprotein. In vivo processing of the polyprotein allows the proper assembly of the multi-subunit complex. This method is very good for balancing expression levels and achieving the correct subunit stoichiometry<sup><a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a></sup>. But when expressing high molecular weight multiprotein complexes, the DNA fragment encoding the polyprotein is very long. It is usually not easy to build such large transfer plasmids in the average laboratory. In addition, with increasing gene length, gene synthesis becomes more expensive and time-consuming. Other potential problems include instability of the recombinant baculovirus due to the large foreign gene insertion and inefficient virus amplification.</p><p id="p-8">There are several other considerations that need to be taken into account when using BVES to express large multiprotein complexes. First of all, the transfer vectors carrying genes for multiple subunits need to be designed so that molecular cloning can be easily accomplished. Because the difficulty of molecular cloning increases with the size of the target construct, transfer vectors that allow efficient selection for positive transformants are necessary. Second, one tagged subunit protein is often used to purify entire protein complexes; however, due to a lack of prior knowledge, sometimes we have to screen for the optimal tagged subunit that allows the best purification of the entire complex. A smart experimental scheme is needed to avoid exerting too much effort in building these screening vectors. Third, it is also important to quickly determine whether protein expression is sufficient and virus amplification is successful because insect expression systems are more time-consuming compared with <em>Escherichia coli</em> expression systems. The sooner these problems are identified, the faster they can be solved.</p><p id="p-9">To overcome these problems in expressing recombinant multisubunit proteins, we have developed SmartBac, a simple and versatile vector system, which combines the advantages of the three strategies described above. The SmartBac vectors are optimized for building large constructs, and by using these vectors, large protein complexes with an intricate subunit composition can be conveniently expressed. The addition of a LacZ-alpha cassette allows positive recombinants with large DNA inserts to be easily selected by blue-white screening. A univector plasmid-fusion system (UPS) strategy has been incorporated in our vector system<sup><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>, so that final construction of large transfer plasmids can be realized by Cre-loxP site-specific recombination between donor and acceptor vectors. This approach makes the preparation of large plasmids easier. To simplify vector construction and obtain more homogenous samples, the polyprotein strategy and Gibson assembly were used to construct the transfer plasmid. The co-expression of EGFP and tagRFP in the SmartBac system provides real-time visualization of transfection and expression. In addition, the SmartBac system can be used to conveniently determine the best tagged subunit that provides the best purification of the entire protein complex. Using the SmartBac system, we have expressed many large multiprotein complexes, including the human exocyst complex and dyncatin complex. This system can also be applied to bacteria, yeast, or mammalian cells with minor modifications. We expect the SmartBac system will aid structural and functional studies of large multiprotein complexes in the future.</p></div><div class="section" id="sec-2"><h2 class="">RESUTLS</h2><div id="sec-3" class="subsection"><h3>SmartBac Vectors</h3><p id="p-10">In order to overcome the difficulties in building large plasmids with conventional cloning methods, we used the broadly applicable UPS strategy <sup><a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a></sup> to set up our SmartBac vector system. Briefly, this strategy uses CreâloxP site-specific recombination to catalyze fusion between the univector (donor vector) and host vector (acceptor vector). The kanamycin-resistant donor has a conditional R6K<em>Î³</em> origin of replication that allows its propagation only in bacterial hosts expressing the pir gene, which encodes the essential replication protein <em>Ï</em><sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>,<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup>. Selection for the UPS recombination products is achieved by selecting for kanamycin resistance (Kan<sup>R</sup>) after transformation into a pirâ strain; the Kan<sup>R</sup> gene in the donor vector can be expressed in a pirâ background only when covalently linked to an acceptor vector that has a functional origin of replication (oriColE1) <sup><a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">26</a></sup>. This strategy has been successfully used in the MultiBac system<sup><a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20</a></sup> and is advantageous for the preparation of large plasmids and their mutants due to the relatively small sizes of the donor and acceptor vectors.</p><p id="p-11">We designed four acceptor plasmids (4V1G, 4V1R, 5V1TG and 5V1TR) and two donor plasmids (4V2G and 4V2R) for use in the SmartBac system (<strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a></strong>, see also <strong>Supplementary Materials and Methods S1</strong>). The acceptors can recombine with the donors via Cre-LoxP site-specific recombination. The acceptors harbor a p15A origin of replication that allows propagation in common cloning strains of <em>E. coli</em> at low copy number, which is better for the stability of large plasmids. The acceptors also contain resistance markers for ampicillin and gentamycin and flanking mini-Tn7 elements for the generation of recombinant baculoviruses.</p><div id="F1" class="fig pos-float  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SmartBac vector maps. The SmartBac system includes four acceptor plasmids (4V1G, 4V1R, 5V1TG and 5V1TR) and two donor plasmids (4V2G and 4V2R). Vector maps were produced by SnapGene Software (http://www.snapgene.com/)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1501336387" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;SmartBac vector maps.&lt;/span&gt; The SmartBac system includes four acceptor plasmids (4V1G, 4V1R, 5V1TG and 5V1TR) and two donor plasmids (4V2G and 4V2R). Vector maps were produced by SnapGene Software (http://www.snapgene.com/).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F1.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F1.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">SmartBac vector maps.</span><p id="p-12" class="first-child">The SmartBac system includes four acceptor plasmids (4V1G, 4V1R, 5V1TG and 5V1TR) and two donor plasmids (4V2G and 4V2R). Vector maps were produced by SnapGene Software (<a href="http://www.snapgene.com/">http://www.snapgene.com/</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">Transgene expression in infected insect cells is driven by the baculovirus late p6.9 promoter <sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>. Compared with the routinely used very late polyhedrin promoter, the p6.9 promoter drives expression at earlier stages of infection when cells are more likely to be in good condition and therefore the aggregation of expressed foreign proteins may be avoided<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>-<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>.</p><p id="p-14">The 4V1 acceptor vectors (4V1G and 4V1R) carry an N-terminal HA-tagged TEV protease coding sequence followed by the TEV protease cleavage site (TCS) and a Twin-Strep tag coding sequence followed by a recognition site for enterokinase. Between multiple cloning site (MCS) 1 and 2, there is a LacZ-alpha expression cassette, which allows blue/white selection of recombinant clones. Downstream of MCS2, there is another TCS and an EGFP (4V1G) or TagRFP (4V1R) coding sequence. The fluorescent and target proteins can be expressed as a single ORF. By observing the fluorescence of infected cells, we can easily determine whether the target protein has been expressed.</p><p id="p-15">In the 5V1T acceptor vectors (5V1TG and 5V1TR), different from 4V1 acceptor vectors, the TEV protease and EGFP (5V1TG) or tagRFP (5V1TR) coding sequences are fused and expressed as a GP64 promoter-driven ORF.</p><p id="p-16">The 4V2 donor vectors (4V2G and 4V2R) carry an N-terminal 10ÃHis coding sequence followed by a recognition site for enterokinase. Both vectors contain a kanamycin resistance marker. The screening region is composed of a high-copy PUC origin of replication and a LacZ-alpha expression cassette, flanked by MCS1 and MCS2. Downstream of MCS2, there is a TCS and a fluorescent protein (EGFP in 4V2G and tagRFP in 4V2R) coding sequence. The expression of the target protein is driven by the very late p10 promoter. The 4V2 vectors also contain the conditional origin of replication, R6K<em>Î³</em> replaced by a foreign gene, the donor vector only contains the R6K<em>Î³</em> origin and can only be propagated in <em>E. coli</em> strains with the pir+ genotype.</p><p id="p-17">There are several single restriction sites located on both sides of the p6.9 and p10 promoter regions in the 4V1/5V1 acceptor and 4V2 donor vectors, respectively, so that they can be replaced by other baculovirus promoters, if needed.</p></div><div id="sec-4" class="subsection"><h3>Schemes for the expression of large multiprotein complexes using the SmartBac system</h3><p id="p-18">The SmartBac system was designed for easier and faster expression of large multiprotein complexes in insect cells. A variety of experimental schemes could be applied to produce the final transfer plasmids from the SmartBac vectors. Here we just present two schemes to exemplify how to use the SmartBac vectors. In the example shown in <strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2a</a></strong>, a vector is designed to express a multiprotein complex composed of eight different subunits (e.g. subunits A, B, C, D, E, F, G and H) in insect cells. If the molecular weight of the multiprotein complex is less than 600 kDa, we propose using Scheme 1 (<strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2a</a></strong>). The eight subunits are divided into two groups so that the sum of the lengths of the genes in one group is as similar as possible to the other group. Then for each group of genes, a fusion DNA fragment (ABCD and EFGH) with TCS coding sequences separating the adjacent genes is designed.</p><div id="F2" class="fig pos-float  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schemes for the expression of large multiprotein complexes. (a)The eight-subunit protein complex to be expressed. The eight genes are divided into two groups according to their sizes. Two long polyproteins are designed with TEV cleavage sites separating the adjacent genes. represents the TEV cleavage site. (b) Schematic representation of Scheme 1 for the expression of multiprotein complexes with a molecular weight less than 600 kDa. Here the acceptor vector 4V1R is used, but 5V1TR can also be used. (c) Schematic representation of Scheme 2 for the expression of multiprotein complexes with a molecular weight greater than 600 kDa. The fluorescent protein in the 4V2G/4V2R donor vector is not expressed because a stop codon has been inserted at the end of the fusion gene, which is located at the upstream of the coding sequence of the fluorescent protein. The coding sequences of EGFP and tagRFP can also be removed by restriction enzyme digestion." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1501336387" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Schemes for the expression of large multiprotein complexes.&lt;/span&gt; &lt;strong&gt;(a)&lt;/strong&gt;The eight-subunit protein complex to be expressed. The eight genes are divided into two groups according to their sizes. Two long polyproteins are designed with TEV cleavage sites separating the adjacent genes. represents the TEV cleavage site. &lt;strong&gt;(b)&lt;/strong&gt; Schematic representation of Scheme 1 for the expression of multiprotein complexes with a molecular weight less than 600 kDa. Here the acceptor vector 4V1R is used, but 5V1TR can also be used. &lt;strong&gt;(c)&lt;/strong&gt; Schematic representation of Scheme 2 for the expression of multiprotein complexes with a molecular weight greater than 600 kDa. The fluorescent protein in the 4V2G/4V2R donor vector is not expressed because a stop codon has been inserted at the end of the fusion gene, which is located at the upstream of the coding sequence of the fluorescent protein. The coding sequences of EGFP and tagRFP can also be removed by restriction enzyme digestion.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F2.medium.gif" width="342" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F2.medium.gif" width="342" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Schemes for the expression of large multiprotein complexes.</span><p id="p-19" class="first-child"><strong>(a)</strong>The eight-subunit protein complex to be expressed. The eight genes are divided into two groups according to their sizes. Two long polyproteins are designed with TEV cleavage sites separating the adjacent genes. represents the TEV cleavage site. <strong>(b)</strong> Schematic representation of Scheme 1 for the expression of multiprotein complexes with a molecular weight less than 600 kDa. Here the acceptor vector 4V1R is used, but 5V1TR can also be used. <strong>(c)</strong> Schematic representation of Scheme 2 for the expression of multiprotein complexes with a molecular weight greater than 600 kDa. The fluorescent protein in the 4V2G/4V2R donor vector is not expressed because a stop codon has been inserted at the end of the fusion gene, which is located at the upstream of the coding sequence of the fluorescent protein. The coding sequences of EGFP and tagRFP can also be removed by restriction enzyme digestion.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Next, the long ABCD and EFGH fragments are further divided into two short DNA fragments AB and CD and EF and GH, which can be obtained easily by overlapping PCR (<strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2b</a></strong>). To avoid unnecessary trouble in overlapping PCR, the TCS cleavage sites described above should be coded by multiple degenerate sequences (<strong>Supplementary Materials and Methods S2</strong>). Then, fragments AB and CD are assembled with a linearized SmartBac RFP-expressing acceptor plasmid (we use 4V1R in <strong><a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2b</a></strong>, but 5V1TR can also be used) utilizing a Gibson assembly reaction <sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup>. Fragments EF and GH are also assembled with a linearized SmartBac GFP-expressing donor plasmid (4V2G) using the same method. The positive recombinants can be easily selected by blue-white screening. Finally, the acceptor-4V1R-ABCD and donor-4V2G-EFGH vectors are recombined via Cre-LoxP site-specific recombination to generate the final transfer plasmid ABCD-EFGH. After transforming this plasmid into DH10Bac competent cells, recombinant bacmid will be obtained. This bacmid will be transfected into insect cells to produce high-titer baculovirus BV-ABCD-EFGH used to express the target eight-subunit complex.</p><p id="p-21">If the molecular weight of the multiprotein complex is greater than 600 kDa, the size of the final transfer plasmid constructed using Scheme 1 will be larger than 25kb. It is usually not easy to build such a large plasmid without experience. And even if the construction is successful, the multiprotein complex may fail to be expressed in insect cells. This is because the recombinant bacmid generated from the large transfer plasmid is prone to display an intrinsic genetic instability due to the large foreign gene insertion <sup><a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>,<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>. Spontaneous deletion of the foreign gene insertion may occur during the amplification of P2 virus (our unpublished data). So, in this case Scheme 2 should be used. As shown in <strong><a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2c</a></strong>, Fragments A and B are assembled with linearized 5V1TR, and fragments C and D are fused with linearized 4V2 vector. A stop codon has been inserted upstream of the coding sequences by PCR so that the fluorescent protein in 4V2G and 4V2R will not be expressed. The same method is used to clone fragments E, F, G and H. Next, two different final transfer plasmids ABCD (RFP) and EFGH (GFP) are built. These plasmids will produce two types of recombinant baculoviruses, one expressing subunits A, B, C and D and RFP, and the other expressing subunits E, F, G and H and GFP. Insect cells co-infected with both baculoviruses will produce the entire protein complex, with the appearance of tagRFP and EGFP fluorescence indicating successful expression of the target protein complex.</p><p id="p-22">The SmartBac acceptors carry an optional N-terminal Twin-Step-tag sequence, and the donors carry an optional N-terminal His-Tag sequence. Either tag can be fused to a target subunit and used as a handle to purify the target subunit along with its associated subunits. If the biochemical properties of the protein complex are known, it is easy to determine which subunit is the most suitable to fuse with the affinity tag. However, when previous knowledge is limited, it may be hard to pick the appropriate subunit because different affinity-tagged subunits often differ in their effectiveness in purifying the entire complex. Imagine that we are expressing an eight-subunit complex and are not sure which subunit is suitable for labeling. If we use a classical âTrial and Errorâ approach and construct multiple large final transfer plasmids, the workload will be very high. In addition, the complicated clone scheme is often confusing. To solve this problem, we propose a simple and universal scheme (Scheme 3, see <strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a></strong>). Two large final transfer plasmids ABCD (RFP) and EFGH (GFP) are built according to Scheme 2 but where none of the eight subunits are labeled with affinity tags (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3a</a>). An additional eight smaller transfer plasmids (from V1-TSA to V1-TSH) based on one acceptor vector (4V1G, 4V1R, 5V1TG or 5V1TR) are constructed, each expressing an N-terminal Twin-Strep-tagged subunit. A total of ten recombinant baculoviruses are obtained, including BV-ABCD (RFP), BV-EFGH (GFP) and BV-TSn (where n ranges from A to H) (<strong><a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3b</a></strong>). The insect cells are co-infected with three types of baculoviruses, BV-ABCD (RFP), BV-EFGH (GFP) and one type of BV-TSn. The baculovirus combinations used for screening are shown in <strong><a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3c</a></strong>. After purification, we know affinity-tagged subunit H results in the best purification of the entire complex. To increase yield and obtain a more homogenous sample, a new intermediate vector containing tagged subunit H, EFG-TSH (GFP), is built (<strong><a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3d</a></strong>). The resulting new recombinant baculovirus, BV-EFG-TSH (GFP), along with the existing recombinant baculovirus, BV-ABCD (RFP), are used to express the multiprotein complex, and real-time infection and expression is monitored by observing the fluorescence of co-expressed EGFP and tagRFP.</p><div id="F3" class="fig pos-float  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Screening for the best affinity-tagged subunit through co-infection of insect cells (Scheme 3). (a) Diagrams of the ten types of transfer plasmids. The final transfer plasmids, ABCD (RFP) and EFGH (GFP), are generated using Scheme 2, and each will express four protein subunits without affinity labels. Each of the other eight transfer plasmids will express one subunit with an N-terminal Twin-Strep (TS) tag. Either the 4V1 or 5V1 vector can be used here. (b) Production of ten types of recombinant baculoviruses (RBVs). Transformation of the ten types of plasmids into DH10Bac competent cells generates 10 types of RBVs. (c) Screening baculovirus combinations to find the subunit that results in the best purification. The ten types of RBVs are divided into eight groups, and each group contains BV-ABCD (RFP), BV-EFGH (GFP) and one BV-TSn (where n corresponds to the subunit, A to H). Insect cells are co-infected with eight groups of RBVs and strep-affinity resin is used to pull down proteins bound to the TS-tagged subunit. The tagged subunit that allows the best purification of the whole complex is selected. In this example, subunit H is the best. (d) Production of the multiprotein complex. Based on the screening result in (c), a new final transfer plasmid EFG-TSH (GFP) is constructed, which expresses an N-terminal TS-tagged subunit H. The whole protein complex will be purified from insect cells co-infected with BV-ABCD (RFP) and BV-EFG-TSH (GFP)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1501336387" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Screening for the best affinity-tagged subunit through co-infection of insect cells (Scheme 3).&lt;/span&gt; (&lt;strong&gt;a&lt;/strong&gt;) Diagrams of the ten types of transfer plasmids. The final transfer plasmids, ABCD (RFP) and EFGH (GFP), are generated using Scheme 2, and each will express four protein subunits without affinity labels. Each of the other eight transfer plasmids will express one subunit with an N-terminal Twin-Strep (TS) tag. Either the 4V1 or 5V1 vector can be used here. (&lt;strong&gt;b&lt;/strong&gt;) Production of ten types of recombinant baculoviruses (RBVs). Transformation of the ten types of plasmids into DH10Bac competent cells generates 10 types of RBVs. (&lt;strong&gt;c&lt;/strong&gt;) Screening baculovirus combinations to find the subunit that results in the best purification. The ten types of RBVs are divided into eight groups, and each group contains BV-ABCD (RFP), BV-EFGH (GFP) and one BV-TSn (where n corresponds to the subunit, A to H). Insect cells are co-infected with eight groups of RBVs and strep-affinity resin is used to pull down proteins bound to the TS-tagged subunit. The tagged subunit that allows the best purification of the whole complex is selected. In this example, subunit H is the best. (&lt;strong&gt;d&lt;/strong&gt;) Production of the multiprotein complex. Based on the screening result in (c), a new final transfer plasmid EFG-TSH (GFP) is constructed, which expresses an N-terminal TS-tagged subunit H. The whole protein complex will be purified from insect cells co-infected with BV-ABCD (RFP) and BV-EFG-TSH (GFP).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F3.medium.gif" width="440" height="300"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F3.medium.gif" width="440" height="300"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Screening for the best affinity-tagged subunit through co-infection of insect cells (Scheme 3).</span><p id="p-23" class="first-child">(<strong>a</strong>) Diagrams of the ten types of transfer plasmids. The final transfer plasmids, ABCD (RFP) and EFGH (GFP), are generated using Scheme 2, and each will express four protein subunits without affinity labels. Each of the other eight transfer plasmids will express one subunit with an N-terminal Twin-Strep (TS) tag. Either the 4V1 or 5V1 vector can be used here. (<strong>b</strong>) Production of ten types of recombinant baculoviruses (RBVs). Transformation of the ten types of plasmids into DH10Bac competent cells generates 10 types of RBVs. (<strong>c</strong>) Screening baculovirus combinations to find the subunit that results in the best purification. The ten types of RBVs are divided into eight groups, and each group contains BV-ABCD (RFP), BV-EFGH (GFP) and one BV-TSn (where n corresponds to the subunit, A to H). Insect cells are co-infected with eight groups of RBVs and strep-affinity resin is used to pull down proteins bound to the TS-tagged subunit. The tagged subunit that allows the best purification of the whole complex is selected. In this example, subunit H is the best. (<strong>d</strong>) Production of the multiprotein complex. Based on the screening result in (c), a new final transfer plasmid EFG-TSH (GFP) is constructed, which expresses an N-terminal TS-tagged subunit H. The whole protein complex will be purified from insect cells co-infected with BV-ABCD (RFP) and BV-EFG-TSH (GFP).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Multiprotein complexes expressed using the SmartBac system</h3><p id="p-24">To test the SmartBac system, we expressed the human exocyst complex in insect cells. The exocyst complex is responsible for tethering secretory vesicles to the plasma membrane in preparation for soluble N-ethylmaleimide-sensitive factor (NSF) attachment protein receptor (SNARE) mediated membrane fusion <sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>. The human exocyst complex contains eight evolutionary conserved subunitsâEXOC1 (102 kDa), EXOC2 (104 kDa), EXOC3 (86 kDa), EXOC4 (110 kDa), EXOC5 (82 kDa), EXOC6 (94 kDa), EXOC7 (78 kDa) and EXOC8 (82 kDa). Because the published literature does not provide information about which subunit is the most suitable for complex purification, we used Scheme 3 to screen the target subunits. All of the vectors we built for exocyst expression are shown in <strong><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a></strong> (see also <strong>Supplementary Materials and Methods S3</strong>). First, we constructed two types of recombinant baculoviruses, BV-E1547 and BV-E2863, to express the eight subunits without any tags according to Scheme 2. These baculoviruses also expressed tagRFP and EGFP, respectively, which allowed us to conveniently determine whether virus infection and protein expression was successful (<strong><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4a</a></strong>). We also produced eight additional types of recombinant baculoviruses, each expressing an individual subunit with an N-terminal Twin-Strep tag (BV-SE1 to BV-SE8). Then we co-infected insect cells with BV-E1547, BV-E2863 and a baculovirus expressing a single affinity tagged-subunit (BV-SE1 to BV-SE8). The best purification of the entire exocyst complex was achieved using tagged EXOC5 (BV-SE5) (<strong><a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4b</a></strong>). Then we constructed a new donor vector 4V2-E1S5 which contains EXOC1 and N-terminal Twin Strep-tagged EXOC5. Recombination between 4V2-E1S5 and 5V1TR-E47 (contains EXOC4 and EXOC7) produced a new final transfer plasmid E1S547 (<strong><a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a></strong>), from which recombinant baculovirus BV-E1S547 was obtained. Insect cells were co-infected with BV-E1S547 and BV-2863. After one-step strep-affinity purification, the entire exocyst complex with high purity was obtained (<strong><a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4c</a></strong>). The tethering activity of this purified exocyst complex was determined via In vitro liposome reconstruction assay (data is not shown here). Negative-staining electron microscopy (nsEM) of the sample revealed homogenous rod-like particles (<strong><a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4d</a></strong>). Preliminary 2D classification of nsEM images (<strong><a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4e</a></strong>) and 3D reconstruction (<strong><a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4f</a></strong>) indicate that the human exocyst complex exhibits a similar dimension and shape with the extracted exocyst complex from yeast <sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup>. The detailed information about primer design, molecular cloning, cell transfection, protein expression and purification, and electron microscopy is described in <strong>Supplementary Materials and Methods S4</strong>.</p><div id="F4" class="fig pos-float  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Examples of multiprotein complexes expressed using the SmartBac system. (a) Fluorescence signals for tagRFP (top) and EGFP (bottom) detected from Sf9 cells transfected with BVE1S547 and BV2863 (see Table 1). (b) Coomassie-stained SDS-PAGE gel of human exocyst complex purified using eight different Twin-Strep tagged subunits (BV-SE1 to BV-SE8, see Table 1). (c) Coomassie-stained SDS-PAGE gel of human exocyst complex purified from insect cells co-infected with BV-2863 and BV-E1S547 (see Table 1). The exocyst complex was purified using Twin-Strep-tagged subunit EXOC5. (d) Electron micrograph of negative-stained recombinant human exocyst complex. The bar represents 100 nm. (e) Representative classes from 2D classification of recombinant human exocyst complex particles. (f) 3D reconstruction of recombinant human exocyst complex based nsEM data. (g) Coomassie-stained SDS-PAGE gel of the human dynactin complex purified by one-step strep-affinity purification. (h) Coomassie-stained 3-8% Native-PAGE gel of purified human dynactin complex after glycerol density gradient centrifugation purification. (i) Single-particle nsEM analysis of recombinant human dynactin complex with the representative raw micrograph (top) and 2D class averages (bottom). Scale bar, 50 nm. (j) Coomassie-stained SDS-PAGE gel of purified recombinant human COPI complex, human dynein complex, human CSN complex and human SCF complex." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1501336387" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Examples of multiprotein complexes expressed using the SmartBac system.&lt;/span&gt; &lt;strong&gt;(a)&lt;/strong&gt; Fluorescence signals for tagRFP (top) and EGFP (bottom) detected from Sf9 cells transfected with BVE1S547 and BV2863 (see &lt;strong&gt;Table 1&lt;/strong&gt;). &lt;strong&gt;(b)&lt;/strong&gt; Coomassie-stained SDS-PAGE gel of human exocyst complex purified using eight different Twin-Strep tagged subunits (BV-SE1 to BV-SE8, see &lt;strong&gt;Table 1&lt;/strong&gt;). &lt;strong&gt;(c)&lt;/strong&gt; Coomassie-stained SDS-PAGE gel of human exocyst complex purified from insect cells co-infected with BV-2863 and BV-E1S547 (see &lt;strong&gt;Table 1&lt;/strong&gt;). The exocyst complex was purified using Twin-Strep-tagged subunit EXOC5. &lt;strong&gt;(d)&lt;/strong&gt; Electron micrograph of negative-stained recombinant human exocyst complex. The bar represents 100 nm. &lt;strong&gt;(e)&lt;/strong&gt; Representative classes from 2D classification of recombinant human exocyst complex particles. &lt;strong&gt;(f)&lt;/strong&gt; 3D reconstruction of recombinant human exocyst complex based nsEM data. &lt;strong&gt;(g)&lt;/strong&gt; Coomassie-stained SDS-PAGE gel of the human dynactin complex purified by one-step strep-affinity purification. &lt;strong&gt;(h)&lt;/strong&gt; Coomassie-stained 3-8% Native-PAGE gel of purified human dynactin complex after glycerol density gradient centrifugation purification. &lt;strong&gt;(i)&lt;/strong&gt; Single-particle nsEM analysis of recombinant human dynactin complex with the representative raw micrograph (top) and 2D class averages (bottom). Scale bar, 50 nm. &lt;strong&gt;(j)&lt;/strong&gt; Coomassie-stained SDS-PAGE gel of purified recombinant human COPI complex, human dynein complex, human CSN complex and human SCF complex.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F4.medium.gif" width="299" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F4.medium.gif" width="299" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/11/14/219246/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Examples of multiprotein complexes expressed using the SmartBac system.</span><p id="p-25" class="first-child"><strong>(a)</strong> Fluorescence signals for tagRFP (top) and EGFP (bottom) detected from Sf9 cells transfected with BVE1S547 and BV2863 (see <strong><a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a></strong>). <strong>(b)</strong> Coomassie-stained SDS-PAGE gel of human exocyst complex purified using eight different Twin-Strep tagged subunits (BV-SE1 to BV-SE8, see <strong><a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a></strong>). <strong>(c)</strong> Coomassie-stained SDS-PAGE gel of human exocyst complex purified from insect cells co-infected with BV-2863 and BV-E1S547 (see <strong><a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a></strong>). The exocyst complex was purified using Twin-Strep-tagged subunit EXOC5. <strong>(d)</strong> Electron micrograph of negative-stained recombinant human exocyst complex. The bar represents 100 nm. <strong>(e)</strong> Representative classes from 2D classification of recombinant human exocyst complex particles. <strong>(f)</strong> 3D reconstruction of recombinant human exocyst complex based nsEM data. <strong>(g)</strong> Coomassie-stained SDS-PAGE gel of the human dynactin complex purified by one-step strep-affinity purification. <strong>(h)</strong> Coomassie-stained 3-8% Native-PAGE gel of purified human dynactin complex after glycerol density gradient centrifugation purification. <strong>(i)</strong> Single-particle nsEM analysis of recombinant human dynactin complex with the representative raw micrograph (top) and 2D class averages (bottom). Scale bar, 50 nm. <strong>(j)</strong> Coomassie-stained SDS-PAGE gel of purified recombinant human COPI complex, human dynein complex, human CSN complex and human SCF complex.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/393679/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/393679/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/393679" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <span class="caption-title">Recombination of human Exocyst complex using SmartBac system</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">We also reconstituted the human dynactin complex using the SmartBac system. Dynactin is a multiprotein complex that works with cytoplasmic dynein to transport cargo along microtubules. It is a large complex of approximately 1.2 MDa composed of 23 subunits corresponding to 11 different types of proteins <sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup>. Dynactin is built around a short actin-like filament composed of Arp1 (43 kDa, 8-actin (42 kDa, 1 copy). The barbed end and the pointed end of this filament are capped by the CapZ<em>Î±</em>-CapZ<em>Î²</em> Î²-CapZ\), complex (33 kDa, 31 kDa) and the Arp11-p25-p27-p62 complex (46 kDa, 20kDa, 21 kDa, 52 kDa) respectively. The shoulder complex, which is made up of p50 (45 kDa, 4 copies), p24 (21 kDa, 2 copies) and p150/p135 (142 kDa/127 kDa, 2 copies), is positioned toward the barbed end of the Arp1 filament <sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup>. As shown in <strong><a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a></strong>, three types of vectors were used to express the 11 subunits of the dynactin complex, and the N-terminal Twin-Strep tag on p135 was used to purify the whole complex. The shoulder complex proteins, p135, p50 and p24. were expressed by BV-M5, which was generated from the plasmid 5V1TG-M5. The final transfer plasmid AB was obtained through recombination of the acceptor plasmid 5V1TR-B and donor plasmid 4V2-A. Plasmid AB was then used to produce the recombinant baculovirus BV-AB expressing the other eight dynactin subunits. Insect cells were co-infected with BV-M5 and BV-AB to express the entire dynactin complex. After one-step strep-affinity purification, the dynactin complex was purified well with rational stoichiometry of its subunits (<strong><a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4g</a></strong>). After glycerol density gradient ultracentrifugation, the further purified dynactin complex exhibited a single visible band on a native gel (<strong><a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4h</a></strong>), suggesting a high homogeneity of the specimen. This band was excised and subjected to mass spectrometry and all 11 human dynactin complex subunits were identified (our unpublished data). This recombant human dynactin complex sample was further investigated by nsEM and subseqeuent 2D image classification (<strong><a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4i</a></strong>), showing a rod-like particle with a shoulder at one end, which is similar to that of the endogenous dynactin complex purified from pig brains <sup><a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a></sup>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/393638/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/393638/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/393638" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <span class="caption-title">Recombination of human Dynactin complex using SmartBac system</span><div class="sb-div caption-clear"></div></div></div><p id="p-27">Besides the human exocyst and dynactin complexes, we also successfully expressed many other protein complexes using the SmartBac system (<strong><a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4j</a></strong>). These include the human COPI complex (7 subunits, 558 kDa) <sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup>, cytoplasmic Dynein complex (12 subunits, 1380 kDa) <sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup>, CSN complex (8 subunits, 343 kDa) <sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup> and SCF complex (5 subunits, 180 kDa) <sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup>. The recombinant COPI complex sample has been used to study the structure of coatomer in its soluble form <sup><a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a></sup>. Our results indicate that the SmartBac system can be used to express a wide range of large multiprotein complexes.</p></div></div><div class="section" id="sec-6"><h2 class="">DISCUSSIONS</h2><p id="p-28">Obtaining large multiprotein complexes through recombinant expression has always been challenging for researchers who need a sufficient quantity of high-purity protein for structural or biochemical studies. The key to successful protein production using the baculovirus expression system is the construction of the final transfer plasmid containing multiple protein subunits. The MultiBac system uses polycistronic vectors carrying multiple GECs for the expression of multiprotein complexes <sup><a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">20</a></sup>. This expression strategy requires junior plasmids containing only one GEC to be constructed first. Then several rounds of plasmid construction have to be done to obtain multi-GEC-contained donor and acceptor vectors. The final transfer plasmid carrying all GECs is produced by Cre-LoxP recombination between the donor and acceptor vectors. MultiBac has been proven powerful in generating multiprotein complexes <sup><a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup>, especially when robotic support is available. But in ordinary laboratories without robotics, the first two procedures require a great deal of time. And as more GECs are added to a single vector, plasmid construction becomes more difficult, due to the increasing size of the plasmid and the lack of an effective screening method for large positive recombinants.</p><p id="p-29">To simplify the process for constructing vectors that express multiple protein subunits, we developed the SmartBac system. We mimicked the polyprotein production strategy of coronavirus to realize the expression of multiple subunits. Although this strategy has been discussed and applied in some laboratories <sup><a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>,<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>,<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup>, a specialized vector system and standardized procedures are not available. By generating one GEC expressing a long polyprotein composed of multiple subunits in donor and acceptor vectors, the SmartBac system does not require the construction of numerous junior vectors. Large numbers of gene fragments with overlapping sequences can be produced rapidly by PCR and directly used for Gibson assembly with linearized vectors. Positive recombinants can be easily selected by blue-white screening and then donor and acceptor vectors carrying a long GEC can be combined by Cre-LoxP recombination to produce the final transfer plasmid.</p><p id="p-30">To ensure a high success rate, we fused multiple gene fragments together by overlapping PCR so no more than three DNA fragments (including the linearized vector) were included in a single Gibson assembly reaction. The size of the final transfer plasmid produced using the schemes we have provided is usually less than 25 kb, so either chemical transformation (Trans2blue or XL10-Gold ultracompetent cells) or electroporation can be used.</p><p id="p-31">To increase the stability of the large final transfer plasmid propagated in <em>E. coli</em>, we included a p15A replication origin of low copy number in the acceptor vectors and cultured the bacteria at 30. The restriction sites flanking the promoter â region allow for promoter exchange if needed (eg. To optimize expression levels).</p><p id="p-32">To monitor the expression of target proteins, we added the most commonly used fluorescent proteins EGFP and tagRFP genes to the SmartBac vectors, so that observation can be easily performed with a basic fluorescence microscope. 4V1 and 5V1 vectors provide two different expression modes for these protein markers. 4V1 vectors use a single GEC to express these fluorescent proteins, TEV protease and the target protein subunits. However, for some constructions, the fluorescent protein was insufficiently cleaved from its upstream expressed subunit (our unpublished data). It had a bad effect on the assembly of the entire complex. In this case, 5V1 vectors can be selected because they use one GEC to express the target subunits, and another GEC to express the fluorescent protein and TEV protease. We have provided two easy-to-use schemes to guide the design of large transfer plasmids containing multiple genes as well as a verified scheme to screen an optimal tagged subunit for complex purification.</p><p id="p-33">With the development of cryo-EM, more and more research groups are carrying out structural and functional studies of multiprotein complexes. Recombinant expression is a promising method for obtaining sufficient quantities of high-purity samples, and, we expect that the SmartBac system will allow more researchers to successfully express and purify large multiprotein complexes.</p></div><div class="section COI-statement" id="sec-7"><h2 class="">COMPETING INTERESTS</h2><p id="p-34">Parts of this study (SmartBac system) has been submitted to apply a Chinese patent for invention with the application number of 201610248592.8.</p></div><div class="section" id="sec-8"><h2 class="">AUTHORSâ CONTRIBUTIONS</h2><p id="p-35">Fei Sun initiated and supervised the project. YZ designed all the SmartBac systems including vectors and application strategies. YZ performed all the experiments of molecular cloning and expression constructs production. YZ, DZ, LY and Fang Sun performed protein complex purification and preliminary electron microscopic characterization. YZ and Fei Sun wrote the manuscript.</p></div><div class="section" id="sec-9"><h2 class="">MATERIALS AND METHODS</h2><div class="section" id="sec-10"><h3>Vector construction</h3><p id="p-38">A portion of the 4V1 vectors was derived from pFastbacDUAL (Invitrogen, USA). 4V2 vectors and all other portions of the 4V1 vectors were synthesized by Genewiz, China. The DNA fragments were fused together via Gibson assembly (E2611, NEB, England) to generate the 4V1G, 4V1R, 4V2G and 4V2R vectors. Then, using 4V1 vectors, 5V1 vectors were generated by Gibson assembly and other classical molecular cloning methods. The sequences of the SmartBac vectors are shown in <strong>Supplementary Materials and Methods S1</strong>. We recommend using SnapGene Viewer (<a href="http://www.snapgene.com/">http://www.snapgene.com/</a>) to view the plasmid maps.</p></div><div class="section" id="sec-11"><h3>Gibson assembly reactions</h3><p id="p-39">The linearized plasmid fragments were obtained by restriction enzyme digestion or PCR using Q5 High-fidelity DNA Polymerase (M0492, NEB, England). DNA fragments to be inserted into the SmartBac vectors were amplified by PCR to produce the appropriate overlaps. The overlapping primers were designed according to NEB instruction manual for E2611. Assembly was done in a 15-20 Î¼ reaction volume with 0.2-0.3 pmols each DNA fragment. Samples were incubated in a thermocycler at 50Â°C for 60 minutes.</p></div><div class="section" id="sec-12"><h3>Blue-white selection of positive recombinants</h3><p id="p-40">To perform blue-white selection, 2.5 Î¼l assembled product was added to 100 Î¼ chemically competent cells. The 4V1â and 5V1-based constructs were transformed into chemically competent Mach1â¢-T1R (Invitrogen) or DH5alpha or Trans2-blue (TransGen Biotech, China) cells. After 1 h recovery in SOC medium at 37Â°C, cells were plated onto LB agar plates containing 100 Î¼ Î¼ g/ml Bluo-gal. The 4V2-based constructs were transformed into chemically competent GT115 cells (InvivoGen, USA), and then cells were g/ml kanamycin, 40 Î¼g/ml IPTG and 100 Î¼g/ml Bluo-gal. Single white colonies were picked and grown in 5 ml LB Î¼ medium with the proper antibiotics for further plasmid extraction and PCR analysis. The positive recombinants were sequenced at BioSune, China.</p></div><div class="section" id="sec-13"><h3>Production of the final transfer plasmid by Cre-LoxP Recombination</h3><p id="p-41">The donor and acceptor vectors (0.1 pmols each) were mixed with 1 Î¼l reaction and incubated at 30Â°C for 1 h. Ten Î¼ microliters of the reaction mixture were added to 100 Î¼l chemically competent Trans2-blue cells. After heat-shock at 30Â°C for 30 s, 500 Î¼l SOC medium was added, and the suspension was incubated at 37Â°C for 1 h with shaking (if the size of the recombined vector was larger than 15 kb, the suspension was incubated at 30Â°C for 4 hrs). The cell suspension was plated on LB agar plates containing 50 Î¼g/ml kanamycin and 100 Î¼g/ml ampicillin. The plates were incubated at 37Â°C overnight (or 30Â°C for 24 hrs). Positive colonies were verified by PCR using the primers Loxp-F (5â-CCACTGCGCCGTTACCAC-3â) and Loxp-R (5â-GCCGGTATGTACAGGAAG-3â). A 375 bp PCR product was amplified from positive clones. The final transfer plasmids were extracted from the positive clones.</p></div><div class="section" id="sec-14"><h3>Production of Recombinant Baculovirus</h3><p id="p-42">Chemically competent DH10Bac cells were transformed with the final transfer plasmid according to the Bac to Bac manual instructions (Invitrogen). For transformation of large plasmids, the transformation mixture was incubated at 30Â°C with shaking for 8-12 hrs and plates were incubated at 30Â°C for more than 48 hrs. Single white colonies (3-4) were innoculated into 5 ml LB medium containing 50 Î¼g/ml gentamicin, and 10 Î¼g/ml tetracycline. Recombinant bacmids were extracted and verified by PCR amplification with three pairs of primers (Tn7R:5â-GTTTTCCCAGTCACGAC-3â and 5â-AAGTTTGAGCAGCCGCGTAG-3â; Tn7L:5â-5â-CAGGAAACAGCTATGAC-3â and 5â-ACCTCCCCCTGAACCTGAAA-3â; Empty: 5â-GTTTTCCCAGTCACGAC-3â 5â-CAGGAAACAGCTATGAC-3â). Using the âTn7Râ and âTn7Lâ and M13 Reverse:5â-CAGGAAACAGCTATGA-3â). Using the âTn7Râ and âTn7Lâprimer pairs, PCR products of 661 bp and 521 bp, respectively, are amplified from recombinant bacmids. If the recombinant bacmid is contaminated with wild-type bacmid, a PCR product of 300 bp will produced using the âEmptyâ primer pairs. It is recommended to verify the existence of all of the subunit genes in the recombinant bacmids by PCR if the size of final transfer plasmid is larger than 20kb.</p></div><div class="section" id="sec-15"><h3>Transfection and Virus Production in Insect cells</h3><p id="p-43">Transfection and Baculovirus production were done according to the Bac to Bac manual (Invitrogen, USA). Successful transfection was determined by the expression of EGFP and/or tagRFP fluorescent proteins. P2 virus was used for expression.</p></div></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDMENTS</h2><p id="p-36">We would be grateful to Ping Shan and Ruigang Su (Fei Sunâs lab) for their help on lab maintenance. We would like to thank Xiang Ding and Mengmeing Zhang from Laboratory of Proteomics, Core Facilities for Protein Science, at the Institute of Biophysics (IBP), Chinese Academy of Sciences (CAS), for their help with mass spectrometry analysis. We would like to thank Center for Biological Imaging (CBI, <a href="http://cbi.ibp.ac.cn">http://cbi.ibp.ac.cn</a>), IBP, CAS for the electron microscopy work.</p><p id="p-37">This work was supported by grants from the Strategic Priority Research Program of Chinese Academy of Sciences (XDB08030202) to Fei Sun, the National Basic Research Program (973 Program) of Ministry of Science and Technology of China (2014CB910700) to Fei Sun, and National Natural Science Foundation of China (31771566) to YZ.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list"><li><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">âµ</a><div class="cit ref-cit ref-book" id="cit-219246v1.1" data-doi="10.1126/science.aac7629"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al</span>. <span class="cit-chapter-title">Structure of a yeast spliceosome at 3.6-angstrom resolution</span>. <span class="cit-source">Science</span> (<span class="cit-publ-name">New York</span>, <span class="cit-publ-loc">N.Y</span> <span class="cit-vol">349</span>, <span class="cit-fpage">1182</span>-<span class="cit-lpage">1191</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.aac7629</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aac7629%26rft_id%253Dinfo%253Apmid%252F26292707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDkvNjI1My8xMTgyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMTEvMTQvMjE5MjQ2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.2" data-doi="10.1038/nature16940"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>,  <span class="cit-name-given-names">T. H. D.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">Cryo-EM structure of the yeast U4/U6.U5 tri-snRNP at 3.7 A resolution</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">530</span>, <span class="cit-fpage">298</span>-<span class="cit-lpage">302</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature16940</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D530%26rft.spage%253D298%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature16940%26rft_id%253Dinfo%253Apmid%252F26829225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature16940&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26829225&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.3" data-doi="10.1038/nature14063"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">Z.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">Structure of the rabbit ryanodine receptor RyR1 at near-atomic resolution</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">517</span>, <span class="cit-fpage">50</span>-<span class="cit-lpage">55</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature14063</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D517%26rft.spage%253D50%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14063%26rft_id%253Dinfo%253Apmid%252F25517095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25517095&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000347178400032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-219246v1.4" data-doi="10.1016/j.cell.2016.08.075"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">des Georges</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">Structural Basis for Gating and Activation of RyR1</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">167</span>, <span class="cit-fpage">145</span>-<span class="cit-lpage">157</span> <span class="cit-auth cit-collab">e117</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2016.08.075</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D167%26rft.spage%253D145%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.08.075%26rft_id%253Dinfo%253Apmid%252F27662087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.08.075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27662087&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.5" data-doi="10.1038/cr.2016.99"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">Structural insights into Ca(2+)-activated long-range allosteric channel gating of RyR1</span>. <abbr class="cit-jnl-abbrev">Cell Res</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">977</span>-<span class="cit-lpage">994</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/cr.2016.99</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes%26rft.volume%253D26%26rft.spage%253D977%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2016.99%26rft_id%253Dinfo%253Apmid%252F27573175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2016.99&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27573175&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.6" data-doi="10.1038/nature14471"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McLaughlin</span>,  <span class="cit-name-given-names">S. H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Barford</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Atomic structure of the APC/C and its mechanism of protein ubiquitination</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">522</span>, <span class="cit-fpage">450</span>-<span class="cit-lpage">454</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature14471</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D522%26rft.spage%253D450%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14471%26rft_id%253Dinfo%253Apmid%252F26083744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14471&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26083744&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7 in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.7" data-doi="10.1038/nature18020"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">Structure of spinach photosystem II-LHCII supercomplex at 3.2 A resolution</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">534</span>, <span class="cit-fpage">69</span>-<span class="cit-lpage">74</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature18020</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D534%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature18020%26rft_id%253Dinfo%253Apmid%252F27251276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature18020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27251276&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.8" data-doi="10.1038/nature19359"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gu</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">The architecture of the mammalian respirasome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">537</span>, <span class="cit-fpage">639</span>-<span class="cit-lpage">643</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature19359</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D537%26rft.spage%253D639%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature19359%26rft_id%253Dinfo%253Apmid%252F27654917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature19359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27654917&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.9" data-doi="10.1038/nature19774"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Letts</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fiedorczuk</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sazanov</span>,  <span class="cit-name-given-names">L. A.</span></span> <span class="cit-article-title">The architecture of respiratory supercomplexes</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">537</span>, <span class="cit-fpage">644</span>-<span class="cit-lpage">648</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature19774</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D537%26rft.spage%253D644%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature19774%26rft_id%253Dinfo%253Apmid%252F27654913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature19774&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27654913&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.10" data-doi="10.1016/S0076-6879(09)63014-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jarvis</span>,  <span class="cit-name-given-names">D. L.</span></span> <span class="cit-article-title">Baculovirus-insect cell expression systems</span>. <abbr class="cit-jnl-abbrev">Methods Enzymol</abbr> <span class="cit-vol">463</span>, <span class="cit-fpage">191</span>-<span class="cit-lpage">222</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0076-6879(09)63014-7</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DJarvis%26rft.auinit1%253DD.%2BL.%26rft.volume%253D463%26rft.spage%253D191%26rft.epage%253D222%26rft.atitle%253DBaculovirus-insect%2Bcell%2Bexpression%2Bsystems.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0076-6879%252809%252963014-7%26rft_id%253Dinfo%253Apmid%252F19892174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0076-6879(09)63014-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19892174&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272201000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luckow</span>,  <span class="cit-name-given-names">V. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">S. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barry</span>,  <span class="cit-name-given-names">G. F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Olins</span>,  <span class="cit-name-given-names">P. O.</span></span> <span class="cit-article-title">Efficient generation of infectious recombinant baculoviruses by site-specific transposon-mediated insertion of foreign genes into a baculovirus genome propagated in Escherichia coli</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">67</span>, <span class="cit-fpage">4566</span>-<span class="cit-lpage">4579</span> (<span class="cit-pub-date">1993</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLuckow%26rft.auinit1%253DV%2BA%26rft.volume%253D67%26rft.issue%253D8%26rft.spage%253D4566%26rft.epage%253D4579%26rft.atitle%253DEfficient%2Bgeneration%2Bof%2Binfectious%2Brecombinant%2Bbaculoviruses%2Bby%2Bsite-specific%2Btransposon-mediated%2Binsertion%2Bof%2Bforeign%2Bgenes%2Binto%2Ba%2Bbaculovirus%2Bgenome%2Bpropagated%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Apmid%252F8392598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjY3LzgvNDU2NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzExLzE0LzIxOTI0Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.12" data-doi="10.1007/978-1-61779-433-9_33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hitchman</span>,  <span class="cit-name-given-names">R. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Possee</span>,  <span class="cit-name-given-names">R. D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">King</span>,  <span class="cit-name-given-names">L. A.</span></span> <span class="cit-article-title">High-throughput baculovirus expression in insect cells</span>. <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr> <span class="cit-vol">824</span>, <span class="cit-fpage">609</span>-<span class="cit-lpage">627</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/978-1-61779-433-9_33</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DHitchman%26rft.auinit1%253DR.%2BB.%26rft.volume%253D824%26rft.spage%253D609%26rft.epage%253D627%26rft.atitle%253DHigh-throughput%2Bbaculovirus%2Bexpression%2Bin%2Binsect%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-61779-433-9_33%26rft_id%253Dinfo%253Apmid%252F22160923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-61779-433-9_33&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22160923&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.13" data-doi="10.1073/pnas.1420936111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Birnbaum</span>,  <span class="cit-name-given-names">M. E.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">Molecular architecture of the alphabeta T cell receptor-CD3 complex</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">111</span>, <span class="cit-fpage">17576</span>-<span class="cit-lpage">17581</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1420936111</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1420936111%26rft_id%253Dinfo%253Apmid%252F25422432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzQ5LzE3NTc2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMTEvMTQvMjE5MjQ2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-219246v1.14" data-doi="10.3748/wjg.v12.i6.921"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chung</span>,  <span class="cit-name-given-names">Y. C.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">Expression, purification and characterization of enterovirus-71 virus-like particles</span>. <abbr class="cit-jnl-abbrev">World J Gastroenterol</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">921</span>-<span class="cit-lpage">927</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWorld%2Bjournal%2Bof%2Bgastroenterology%2B%253A%2B%2BWJG%26rft.stitle%253DWorld%2BJ%2BGastroenterol%26rft.aulast%253DChung%26rft.auinit1%253DY.%2BC.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D921%26rft.epage%253D927%26rft.atitle%253DExpression%252C%2Bpurification%2Band%2Bcharacterization%2Bof%2Benterovirus-71%2Bvirus-like%2Bparticles.%26rft_id%253Dinfo%253Adoi%252F10.3748%252Fwjg.v12.i6.921%26rft_id%253Dinfo%253Apmid%252F16521221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3748/wjg.v12.i6.921&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16521221&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-219246v1.15" data-doi="10.1023/A:1024071514438"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Y. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hsu</span>,  <span class="cit-name-given-names">J. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">J. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>,  <span class="cit-name-given-names">M. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ho</span>,  <span class="cit-name-given-names">Y. C.</span></span> <span class="cit-article-title">Formation of enterovirus-like particle aggregates by recombinant baculoviruses co-expressing P1 and 3CD in insect cells</span>. <abbr class="cit-jnl-abbrev">Biotechnol Lett</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">919</span>-<span class="cit-lpage">925</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnology%2Bletters%26rft.stitle%253DBiotechnol%2BLett%26rft.aulast%253DHu%26rft.auinit1%253DY.%2BC.%26rft.volume%253D25%26rft.issue%253D12%26rft.spage%253D919%26rft.epage%253D925%26rft.atitle%253DFormation%2Bof%2Benterovirus-like%2Bparticle%2Baggregates%2Bby%2Brecombinant%2Bbaculoviruses%2Bco-expressing%2BP1%2Band%2B3CD%2Bin%2Binsect%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1024071514438%26rft_id%253Dinfo%253Apmid%252F12889824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1024071514438&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12889824&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-219246v1.16" data-doi="10.1074/jbc.M109.095141"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kee</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">WDR20 regulates activity of the USP12 Ã UAF1 deubiquitinating enzyme complex</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">285</span>, <span class="cit-fpage">11252</span>-<span class="cit-lpage">11257</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1074/jbc.M109.095141</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M109.095141%26rft_id%253Dinfo%253Apmid%252F20147737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvMTUvMTEyNTIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8xMS8xNC8yMTkyNDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-219246v1.17" data-doi="10.1093/nar/gkw1181"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wan</span>,  <span class="cit-name-given-names">L. C.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">Proteomic analysis of the human KEOPS complex identifies C14ORF142 as a core subunit homologous to yeast Gon7</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr> <span class="cit-vol">45</span>, <span class="cit-fpage">805</span>-<span class="cit-lpage">817</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkw1181</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1181%26rft_id%253Dinfo%253Apmid%252F27903914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1181&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27903914&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18 in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.18" data-doi="10.1091/mbc.E08-12-1190"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yoo</span>,  <span class="cit-name-given-names">H. Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumagai</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shevchenko</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shevchenko</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dunphy</span>,  <span class="cit-name-given-names">W. G.</span></span> <span class="cit-article-title">The Mre11-Rad50-Nbs1 complex mediates activation of TopBP1 by ATM</span>. <abbr class="cit-jnl-abbrev">Mol Biol Cell</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">2351</span>-<span class="cit-lpage">2360</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1091/mbc.E08-12-1190</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E08-12-1190%26rft_id%253Dinfo%253Apmid%252F19279141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMjM1MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzExLzE0LzIxOTI0Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19 in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.19" data-doi="10.1093/nar/21.5.1219"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Belyaev</span>,  <span class="cit-name-given-names">A. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Roy</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Development of baculovirus triple and quadruple expression vectors: co-expression of three or four bluetongue virus proteins and the synthesis of bluetongue virus-like particles in insect cells</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr> <span class="cit-vol">21</span>, <span class="cit-fpage">1219</span>-<span class="cit-lpage">1223</span> (<span class="cit-pub-date">1993</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBelyaev%26rft.auinit1%253DA.%2BS.%26rft.volume%253D21%26rft.issue%253D5%26rft.spage%253D1219%26rft.epage%253D1223%26rft.atitle%253DDevelopment%2Bof%2Bbaculovirus%2Btriple%2Band%2Bquadruple%2Bexpression%2Bvectors%253A%2Bco-expression%2Bof%2Bthree%2Bor%2Bfour%2Bbluetongue%2Bvirus%2Bproteins%2Band%2Bthe%2Bsynthesis%2Bof%2Bbluetongue%2Bvirus-like%2Bparticles%2Bin%2Binsect%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F21.5.1219%26rft_id%253Dinfo%253Apmid%252F8385313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/21.5.1219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8385313&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20 in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.20" data-doi="10.1038/nbt1036"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berger</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fitzgerald</span>,  <span class="cit-name-given-names">D. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Richmond</span>,  <span class="cit-name-given-names">T. J.</span></span> <span class="cit-article-title">Baculovirus expression system for heterologous multiprotein complexes</span>. <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">1583</span>-<span class="cit-lpage">1587</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nbt1036</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DBerger%26rft.auinit1%253DI.%26rft.volume%253D22%26rft.issue%253D12%26rft.spage%253D1583%26rft.epage%253D1587%26rft.atitle%253DBaculovirus%2Bexpression%2Bsystem%2Bfor%2Bheterologous%2Bmultiprotein%2Bcomplexes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1036%26rft_id%253Dinfo%253Apmid%252F15568020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15568020&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225638600031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21 in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.21" data-doi="10.1016/j.sbi.2005.10.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bartlam</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rao</span>,  <span class="cit-name-given-names">Z.</span></span> <span class="cit-article-title">Structural insights into SARS coronavirus proteins</span>. <abbr class="cit-jnl-abbrev">Curr Opin Struct Biol</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">664</span>-<span class="cit-lpage">672</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.sbi.2005.10.004</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bstructural%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BStruct%2BBiol%26rft.aulast%253DBartlam%26rft.auinit1%253DM.%26rft.volume%253D15%26rft.issue%253D6%26rft.spage%253D664%26rft.epage%253D672%26rft.atitle%253DStructural%2Binsights%2Binto%2BSARS%2Bcoronavirus%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.sbi.2005.10.004%26rft_id%253Dinfo%253Apmid%252F16263266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.sbi.2005.10.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16263266&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234167500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22 in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.22" data-doi="10.1099/0022-1317-81-4-853"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>,  <span class="cit-name-given-names">E. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>,  <span class="cit-name-given-names">A. E.</span></span> <span class="cit-article-title">Virus-encoded proteinases and proteolytic processing in the Nidovirales</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr> <span class="cit-vol">81</span>, <span class="cit-fpage">853</span>-<span class="cit-lpage">879</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/0022-1317-81-4-853</span> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DZiebuhr%26rft.auinit1%253DJ.%26rft.volume%253D81%26rft.issue%253D4%26rft.spage%253D853%26rft.epage%253D879%26rft.atitle%253DVirus-encoded%2Bproteinases%2Band%2Bproteolytic%2Bprocessing%2Bin%2Bthe%2BNidovirales%26rft_id%253Dinfo%253Adoi%252F10.1099%252F0022-1317-81-4-853%26rft_id%253Dinfo%253Apmid%252F10725411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/0022-1317-81-4-853&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10725411&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086341400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23 in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.23" data-doi="10.1128/JVI.01358-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sawicki</span>,  <span class="cit-name-given-names">S. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sawicki</span>,  <span class="cit-name-given-names">D. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Siddell</span>,  <span class="cit-name-given-names">S. G.</span></span> <span class="cit-article-title">A contemporary view of coronavirus transcription</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">81</span>, <span class="cit-fpage">20</span>-<span class="cit-lpage">29</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.01358-06</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01358-06%26rft_id%253Dinfo%253Apmid%252F16928755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjc6IjgxLzEvMjAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8xMS8xNC8yMTkyNDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24 in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.24" data-doi="10.1016/j.jsb.2011.03.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vijayachandran</span>,  <span class="cit-name-given-names">L. S.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">Robots, pipelines, polyproteins: enabling multiprotein expression in prokaryotic and eukaryotic cells</span>. <abbr class="cit-jnl-abbrev">J Struct Biol</abbr> <span class="cit-vol">175</span>, <span class="cit-fpage">198</span>-<span class="cit-lpage">208</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jsb.2011.03.007</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Bbiology%26rft.stitle%253DJ%2BStruct%2BBiol%26rft.aulast%253DVijayachandran%26rft.auinit1%253DL.%2BS.%26rft.volume%253D175%26rft.issue%253D2%26rft.spage%253D198%26rft.epage%253D208%26rft.atitle%253DRobots%252C%2Bpipelines%252C%2Bpolyproteins%253A%2Benabling%2Bmultiprotein%2Bexpression%2Bin%2Bprokaryotic%2Band%2Beukaryotic%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jsb.2011.03.007%26rft_id%253Dinfo%253Apmid%252F21419851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jsb.2011.03.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21419851&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.25" data-doi="10.1007/978-1-62703-691-7_8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nie</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bellon-Echeverria</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trowitzsch</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bieniossek</span>,  <span class="cit-name-given-names">C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Berger</span>,  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">Multiprotein complex production in insect cells by using polyproteins</span>. <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr> <span class="cit-vol">1091</span>, <span class="cit-fpage">131</span>-<span class="cit-lpage">141</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/978-1-62703-691-7_8</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol%2BBiol%26rft.volume%253D1091%26rft.spage%253D131%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-62703-691-7_8%26rft_id%253Dinfo%253Apmid%252F24203328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-62703-691-7_8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24203328&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26 in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.26" data-doi="10.1016/S0960-9822(07)00560-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">M. Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leibham</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cortez</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Elledge</span>,  <span class="cit-name-given-names">S. J.</span></span> <span class="cit-article-title">The univector plasmid-fusion system, a method for rapid construction of recombinant DNA without restriction enzymes</span>. <abbr class="cit-jnl-abbrev">Curr Biol</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">1300</span>-<span class="cit-lpage">1309</span> (<span class="cit-pub-date">1998</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DLiu%26rft.auinit1%253DQ.%26rft.volume%253D8%26rft.issue%253D24%26rft.spage%253D1300%26rft.epage%253D1309%26rft.atitle%253DThe%2Bunivector%2Bplasmid-fusion%2Bsystem%252C%2Ba%2Bmethod%2Bfor%2Brapid%2Bconstruction%2Bof%2Brecombinant%2BDNA%2Bwithout%2Brestriction%2Benzymes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0960-9822%252807%252900560-X%26rft_id%253Dinfo%253Apmid%252F9843682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0960-9822(07)00560-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9843682&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077341000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27 in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.27" data-doi="10.1016/0378-1119(94)90776-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Metcalf</span>,  <span class="cit-name-given-names">W. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wanner</span>,  <span class="cit-name-given-names">B. L.</span></span> <span class="cit-article-title">Use of the rep technique for allele replacement to construct new Escherichia coli hosts for maintenance of R6K gamma origin plasmids at different copy numbers</span>. <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">138</span>, <span class="cit-fpage">1</span>-<span class="cit-lpage">7</span> (<span class="cit-pub-date">1994</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DMetcalf%26rft.auinit1%253DW.%2BW.%26rft.volume%253D138%26rft.issue%253D1-2%26rft.spage%253D1%26rft.epage%253D7%26rft.atitle%253DUse%2Bof%2Bthe%2Brep%2Btechnique%2Bfor%2Ballele%2Breplacement%2Bto%2Bconstruct%2Bnew%2BEscherichia%2Bcoli%2Bhosts%2Bfor%2Bmaintenance%2Bof%2BR6K%2Bgamma%2Borigin%2Bplasmids%2Bat%2Bdifferent%2Bcopy%2Bnumbers.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252894%252990776-5%26rft_id%253Dinfo%253Apmid%252F8125283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(94)90776-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8125283&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NA66100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28 in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Filutowicz</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">Role of the pi initiation protein and direct nucleotide sequence repeats in the regulation of plasmid R6K replication</span>. <abbr class="cit-jnl-abbrev">Basic Life Sci</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">125</span>-<span class="cit-lpage">140</span> (<span class="cit-pub-date">1985</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBasic%2Blife%2Bsciences%26rft.stitle%253DBasic%2BLife%2BSci%26rft.aulast%253DFilutowicz%26rft.auinit1%253DM.%26rft.volume%253D30%26rft.spage%253D125%26rft.epage%253D140%26rft.atitle%253DRole%2Bof%2Bthe%2Bpi%2Binitiation%2Bprotein%2Band%2Bdirect%2Bnucleotide%2Bsequence%2Brepeats%2Bin%2Bthe%2Bregulation%2Bof%2Bplasmid%2BR6K%2Breplication.%26rft_id%253Dinfo%253Apmid%252F2990404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2990404&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29 in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.29" data-doi="10.1099/0022-1317-71-4-971"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hill-Perkins</span>,  <span class="cit-name-given-names">M. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Possee</span>,  <span class="cit-name-given-names">R. D.</span></span> <span class="cit-article-title">A baculovirus expression vector derived from the basic protein promoter of Autographa californica nuclear polyhedrosis virus</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr> <span class="cit-vol">71</span> (Pt <span class="cit-issue">4</span>), <span class="cit-fpage">971</span>-<span class="cit-lpage">976</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/0022-1317-71-4-971</span> (<span class="cit-pub-date">1990</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DHill-Perkins%26rft.auinit1%253DM.%2BS.%26rft.volume%253D71%26rft.issue%253D4%26rft.spage%253D971%26rft.epage%253D976%26rft.atitle%253DA%2Bbaculovirus%2Bexpression%2Bvector%2Bderived%2Bfrom%2Bthe%2Bbasic%2Bprotein%2Bpromoter%2Bof%2BAutographa%2Bcalifornica%2Bnuclear%2Bpolyhedrosis%2Bvirus%26rft_id%253Dinfo%253Adoi%252F10.1099%252F0022-1317-71-4-971%26rft_id%253Dinfo%253Apmid%252F2109042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/0022-1317-71-4-971&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2109042&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30 in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.30" data-doi="10.1016/j.jip.2011.05.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Oers</span>,  <span class="cit-name-given-names">M. M.</span></span> <span class="cit-article-title">Opportunities and challenges for the baculovirus expression system</span>. <abbr class="cit-jnl-abbrev">J Invertebr Pathol</abbr> <span class="cit-vol">107</span> Suppl, <span class="cit-fpage">S3</span>-<span class="cit-lpage">15</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jip.2011.05.001</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BInvertebr%2BPathol%26rft.volume%253D107%26rft.spage%253DS3%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jip.2011.05.001%26rft_id%253Dinfo%253Apmid%252F21784228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jip.2011.05.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21784228&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-219246v1.31" data-doi="10.1007/s10529-010-0340-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ishiyama</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ikeda</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">High-level expression and improved folding of proteins by using the vp39 late promoter enhanced with homologous DNA regions</span>. <abbr class="cit-jnl-abbrev">Biotechnol Lett</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">1637</span>-<span class="cit-lpage">1647</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s10529-010-0340-7</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnology%2Bletters%26rft.stitle%253DBiotechnol%2BLett%26rft.aulast%253DIshiyama%26rft.auinit1%253DS.%26rft.volume%253D32%26rft.issue%253D11%26rft.spage%253D1637%26rft.epage%253D1647%26rft.atitle%253DHigh-level%2Bexpression%2Band%2Bimproved%2Bfolding%2Bof%2Bproteins%2Bby%2Busing%2Bthe%2Bvp39%2Blate%2Bpromoter%2Benhanced%2Bwith%2Bhomologous%2BDNA%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10529-010-0340-7%26rft_id%253Dinfo%253Apmid%252F20821250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10529-010-0340-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20821250&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282696100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32 in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.32" data-doi="10.1007/s12250-012-3236-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">S. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">H. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Z. H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Deng</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Genetic modification of baculovirus expression vectors</span>. <abbr class="cit-jnl-abbrev">Virol Sin</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">71</span>-<span class="cit-lpage">82</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s12250-012-3236-y</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DVirol%2BSin%26rft.aulast%253DLi%26rft.auinit1%253DS.%2BF.%26rft.volume%253D27%26rft.issue%253D2%26rft.spage%253D71%26rft.epage%253D82%26rft.atitle%253DGenetic%2Bmodification%2Bof%2Bbaculovirus%2Bexpression%2Bvectors.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12250-012-3236-y%26rft_id%253Dinfo%253Apmid%252F22491998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12250-012-3236-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22491998&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33 in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.33" data-doi="10.1038/nmeth.1318"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gibson</span>,  <span class="cit-name-given-names">D. G.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">Enzymatic assembly of DNA molecules up to several hundred kilobases</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">343</span>-<span class="cit-lpage">345</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.1318</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DGibson%26rft.auinit1%253DD.%2BG.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D343%26rft.epage%253D345%26rft.atitle%253DEnzymatic%2Bassembly%2Bof%2BDNA%2Bmolecules%2Bup%2Bto%2Bseveral%2Bhundred%2Bkilobases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1318%26rft_id%253Dinfo%253Apmid%252F19363495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1318&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19363495&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265661600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34 in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.34" data-doi="10.1099/vir.0.19438-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pijlman</span>,  <span class="cit-name-given-names">G. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Schijndel</span>,  <span class="cit-name-given-names">J. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Vlak</span>,  <span class="cit-name-given-names">J. M.</span></span> <span class="cit-article-title">Spontaneous excision of BAC vector sequences from bacmid-derived baculovirus expression vectors upon passage in insect cells</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr> <span class="cit-vol">84</span>, <span class="cit-fpage">2669</span>-<span class="cit-lpage">2678</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/vir.0.19438-0</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DPijlman%26rft.auinit1%253DG.%2BP.%26rft.volume%253D84%26rft.issue%253D10%26rft.spage%253D2669%26rft.epage%253D2678%26rft.atitle%253DSpontaneous%2Bexcision%2Bof%2BBAC%2Bvector%2Bsequences%2Bfrom%2Bbacmid-derived%2Bbaculovirus%2Bexpression%2Bvectors%2Bupon%2Bpassage%2Bin%2Binsect%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.19438-0%26rft_id%253Dinfo%253Apmid%252F13679600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.19438-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=13679600&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35 in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.35" data-doi="10.1242/jcs.156398"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-article-title">The Exocyst at a Glance</span>. <abbr class="cit-jnl-abbrev">J Cell Sci</abbr> <span class="cit-vol">128</span>, <span class="cit-fpage">2957</span>-<span class="cit-lpage">2964</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1242/jcs.156398</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BCell%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjcs.156398%26rft_id%253Dinfo%253Apmid%252F26240175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6MTE6IjEyOC8xNi8yOTU3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMTEvMTQvMjE5MjQ2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36 in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.36" data-doi="10.1038/nsmb.3146"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heider</span>,  <span class="cit-name-given-names">M. R.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">Subunit connectivity, assembly determinants and architecture of the yeast exocyst complex</span>. <abbr class="cit-jnl-abbrev">Nat Struct Mol Biol</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">59</span>-<span class="cit-lpage">66</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nsmb.3146</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BStruct%2BMol%2BBiol%26rft.volume%253D23%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.3146%26rft_id%253Dinfo%253Apmid%252F26656853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.3146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26656853&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37 in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.37" data-doi="10.1038/nsmb.3022"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reck-Peterson</span>,  <span class="cit-name-given-names">S. L.</span></span> <span class="cit-article-title">Dynactin revealed</span>. <abbr class="cit-jnl-abbrev">Nat Struct Mol Biol</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">359</span>-<span class="cit-lpage">360</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nsmb.3022</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BStruct%2BMol%2BBiol%26rft.volume%253D22%26rft.spage%253D359%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.3022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.3022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38 in text" id="ref-38">âµ</a><div class="cit ref-cit ref-book" id="cit-219246v1.38" data-doi="10.1126/science.aaa4080"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Urnavicius</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al</span>. <span class="cit-chapter-title">The structure of the dynactin complex and its interaction with dynein</span>. <span class="cit-source">Science</span> (<span class="cit-publ-name">New York</span>, <span class="cit-publ-loc">N.Y</span>) <span class="cit-vol">347</span>, <span class="cit-fpage">1441</span>-<span class="cit-lpage">1446</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.aaa4080</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaa4080%26rft_id%253Dinfo%253Apmid%252F25814576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDcvNjIyOS8xNDQxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMTEvMTQvMjE5MjQ2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39 in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.39" data-doi="10.1007/s13238-016-0296-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">Structural characterization of coatomer in its cytosolic state</span>. <abbr class="cit-jnl-abbrev">Protein Cell</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">586</span>-<span class="cit-lpage">600</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s13238-016-0296-z</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BCell%26rft.volume%253D7%26rft.spage%253D586%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13238-016-0296-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13238-016-0296-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40 in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.40" data-doi="10.1016/j.cell.2017.05.025"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">Cryo-EM Reveals How Human Cytoplasmic Dynein Is Auto-inhibited and Activated</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">169</span>, <span class="cit-fpage">1303</span>-<span class="cit-lpage">1314</span> e1318<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2017.05.025</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D169%26rft.spage%253D1303%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2017.05.025%26rft_id%253Dinfo%253Apmid%252F28602352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2017.05.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28602352&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41 in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.41" data-doi="10.7554/eLife.12102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mosadeghi</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">Structural and kinetic analysis of the COP9-Signalosome activation and the cullin-RING ubiquitin ligase deneddylation cycle</span>. <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">5</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.7554/eLife.12102</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.12102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.12102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42 in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.42" data-doi="10.1038/416703a"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">Structure of the Cul1-Rbx1-Skp1-F boxSkp2 SCF ubiquitin ligase complex</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">416</span>, <span class="cit-fpage">703</span>-<span class="cit-lpage">709</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/416703a</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DZheng%26rft.auinit1%253DN.%26rft.volume%253D416%26rft.issue%253D6882%26rft.spage%253D703%26rft.epage%253D709%26rft.atitle%253DStructure%2Bof%2Bthe%2BCul1-Rbx1-Skp1-F%2BboxSkp2%2BSCF%2Bubiquitin%2Bligase%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F416703a%26rft_id%253Dinfo%253Apmid%252F11961546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/416703a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11961546&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175033500033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43 in text" id="ref-43">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.43" data-doi="10.3791/50159"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berger</span>,  <span class="cit-name-given-names">I.</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">The multiBac protein complex production platform at the EMBL</span>. <abbr class="cit-jnl-abbrev">J Vis Exp</abbr>, <span class="cit-fpage">e50159</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3791/50159</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVis%2BExp%26rft_id%253Dinfo%253Adoi%252F10.3791%252F50159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3791/50159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44 in text" id="ref-44">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.44" data-doi="10.1002/pro.518"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pham</span>,  <span class="cit-name-given-names">E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Truong</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">TEV protease-facilitated stoichiometric delivery of multiple genes using a single expression vector</span>. <abbr class="cit-jnl-abbrev">Protein Sci</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">2379</span>-<span class="cit-lpage">2388</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/pro.518</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%26rft.stitle%253DProtein%2BSci.%26rft.aulast%253DChen%26rft.auinit1%253DX.%26rft.volume%253D19%26rft.issue%253D12%26rft.spage%253D2379%26rft.epage%253D2388%26rft.atitle%253DTEV%2Bprotease-facilitated%2Bstoichiometric%2Bdelivery%2Bof%2Bmultiple%2Bgenes%2Busing%2Ba%2Bsingle%2Bexpression%2Bvector.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpro.518%26rft_id%253Dinfo%253Apmid%252F20945357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pro.518&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20945357&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45 in text" id="ref-45">âµ</a><div class="cit ref-cit ref-journal" id="cit-219246v1.45" data-doi="10.1016/j.tibs.2011.10.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bieniossek</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Imasaki</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takagi</span>,  <span class="cit-name-given-names">Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Berger</span>,  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">MultiBac: expanding the research toolbox for multiprotein complexes</span>. <abbr class="cit-jnl-abbrev">Trends Biochem Sci</abbr> <span class="cit-vol">37</span>, <span class="cit-fpage">49</span>-<span class="cit-lpage">57</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.tibs.2011.10.005</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiochemical%2Bsciences%26rft.stitle%253DTrends%2BBiochem%2BSci%26rft.aulast%253DBieniossek%26rft.auinit1%253DC.%26rft.volume%253D37%26rft.issue%253D2%26rft.spage%253D49%26rft.epage%253D57%26rft.atitle%253DMultiBac%253A%2Bexpanding%2Bthe%2Bresearch%2Btoolbox%2Bfor%2Bmultiprotein%2Bcomplexes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibs.2011.10.005%26rft_id%253Dinfo%253Apmid%252F22154230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibs.2011.10.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22154230&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F11%2F14%2F219246.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300923200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/219246v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F219246v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/219352v1" title="Independent assessment and improvement of wheat genome assemblies using Fosill jumping libraries" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/218735v1" title="Combined flow cytometry and high throughput image analysis for the study of essential genes in Caenorhabditis elegans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;November 14, 2017.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/219246v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/219246v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/219246v1" class="active" data-icon-position="" data-hide-link-title="0">Large protein complex production using the SmartBac System - Strategies and Applications</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/66807" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-jBVCublabHZLGUGKTVXYGH97L3B4FTKvm4u14Jjj6W0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="66807" id="node66807--2" data-pisa="biorxiv;219246v1" data-pisa-master="biorxiv;219246" data-seqnum="66807" data-apath="/biorxiv/early/2017/11/14/219246.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Large protein complex production using the SmartBac System - Strategies and Applications</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Yujia</span> <span class="nlm-surname">Zhai</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Danyang</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Leiye</span> <span class="nlm-surname">Yu</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Fang</span> <span class="nlm-surname">Sun</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Fei</span> <span class="nlm-surname">Sun</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">219246; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/219246 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/219246v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/219246v1&amp;title=Large%20protein%20complex%20production%20using%20the%20SmartBac%20System%20-%20Strategies%20and%20Applications" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/219246v1&amp;title=Large%20protein%20complex%20production%20using%20the%20SmartBac%20System%20-%20Strategies%20and%20Applications" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/219246v1&amp;text=Large%20protein%20complex%20production%20using%20the%20SmartBac%20System%20-%20Strategies%20and%20Applications" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/219246v1&amp;title=Large%20protein%20complex%20production%20using%20the%20SmartBac%20System%20-%20Strategies%20and%20Applications" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/219246v1&amp;t=Large%20protein%20complex%20production%20using%20the%20SmartBac%20System%20-%20Strategies%20and%20Applications" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/219246v1&amp;title=Large%20protein%20complex%20production%20using%20the%20SmartBac%20System%20-%20Strategies%20and%20Applications" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/219246v1&amp;title=Large%20protein%20complex%20production%20using%20the%20SmartBac%20System%20-%20Strategies%20and%20Applications" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="66807" id="node66807--3" data-pisa="biorxiv;219246v1" data-pisa-master="biorxiv;219246" data-seqnum="66807" data-apath="/biorxiv/early/2017/11/14/219246.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Large protein complex production using the SmartBac System - Strategies and Applications</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Yujia</span> <span class="nlm-surname">Zhai</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Danyang</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Leiye</span> <span class="nlm-surname">Yu</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Fang</span> <span class="nlm-surname">Sun</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Fei</span> <span class="nlm-surname">Sun</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">219246; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/219246 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Large%20protein%20complex%20production%20using%20the%20SmartBac%20System%20-%20Strategies%20and%20Applications&amp;rft.volume&amp;rft.issue&amp;rft.date=2017-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Zhai%2C+Yujia&amp;rft.au=Zhang%2C+Danyang&amp;rft.au=Yu%2C+Leiye&amp;rft.au=Sun%2C+Fang&amp;rft.au=Sun%2C+Fei"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/66807/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/66807/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/66807/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/66807/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/66807/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/66807/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/66807/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/66807/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/66807/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/66807/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/66807/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/66807/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/219246v1&amp;count=horizontal&amp;via=&amp;text=Large%20protein%20complex%20production%20using%20the%20SmartBac%20System%20-%20Strategies%20and%20Applications&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/219246v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/219246v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/219246v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioengineering" class="highlight" data-icon-position="" data-hide-link-title="0">Bioengineering<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
