<?xml version="1.0" encoding="UTF-8"?>
<html xmlns="http://www.w3.org/1999/xhtml">
 <head charset="UTF-8"/>
 <style> * { font-family : helvetica;} .font32 { font-size : 32; } .font12 { font-size : 12; } .font13 { font-size : 13; } .font14 { font-size : 14; } .font15 { font-size : 15; } .font16 { font-size : 16; } .font17 { font-size : 17; } .font18 { font-size : 18; } .font19 { font-size : 19; } .font20 { font-size : 20; } .font21 { font-size : 21; } .font22 { font-size : 22; } .font23 { font-size : 23; } .font24 { font-size : 24; } .font28 { font-size : 28; }</style>
 <body>
  <p>
   <span class="font32">-nCoV </span>
   <span class="font32">pangolin </span>
   <span class="font28">coronavirus </span>
   <span class="font24">pangolins </span>
   <span class="font24">related </span>
   <span class="font23">coronaviruses </span>
   <span class="font23">samples </span>
   <span class="font22">using </span>
   <span class="font21">sequence </span>
   <span class="font20">genome </span>
   <span class="font20">sequences </span>
   <span class="font19">View </span>
   <span class="font19">Guangdong </span>
   <span class="font19">virus </span>
   <span class="font18">Figure </span>
   <span class="font18">bat </span>
   <span class="font17">Table </span>
   <span class="font17">human </span>
   <span class="font17">viral </span>
   <span class="font16">Data </span>
   <span class="font16">Guangxi </span>
   <span class="font16">analysis </span>
   <span class="font16">obtained </span>
   <span class="font16">phylogenetic </span>
   <span class="font16">shown </span>
   <span class="font15">Extended </span>
   <span class="font15">amino </span>
   <span class="font15">animals </span>
   <span class="font15">indicated </span>
   <span class="font15">sequencing </span>
   <span class="font15">similarity </span>
   <span class="font15">-nCoV, </span>
   <span class="font15">China </span>
   <span class="font15">different </span>
   <span class="font15">estimated </span>
   <span class="font15">outbreak </span>
   <span class="font15">recombination </span>
   <span class="font15">red </span>
   <span class="font14">Download </span>
   <span class="font14">OpenUrl </span>
   <span class="font14">SARS-CoV </span>
   <span class="font14">alignment </span>
   <span class="font14">associated </span>
   <span class="font14">data </span>
   <span class="font14">genomes </span>
   <span class="font14">including </span>
   <span class="font14">lineages </span>
   <span class="font14">lung </span>
   <span class="font14">novel </span>
   <span class="font14">performed </span>
   <span class="font14">reads </span>
   <span class="font14">reference </span>
   <span class="font14">used </span>
   <span class="font13">(B) </span>
   <span class="font13">GD/PL </span>
   <span class="font13">RBD </span>
   <span class="font13">RNA </span>
   <span class="font13">S). </span>
   <span class="font13">acid </span>
   <span class="font13">animal </span>
   <span class="font13">anti-smuggling </span>
   <span class="font13">assembly </span>
   <span class="font13">market </span>
   <span class="font13">new </span>
   <span class="font13">qPCR </span>
   <span class="font13">regions </span>
   <span class="font13">(A) </span>
   <span class="font13">-nCoV. </span>
   <span class="font13">GenBank </span>
   <span class="font13">Phylogenetic </span>
   <span class="font13">RaTG </span>
   <span class="font13">Vero </span>
   <span class="font13">accession </span>
   <span class="font13">bootstrap </span>
   <span class="font13">closely </span>
   <span class="font13">collected </span>
   <span class="font13">consensus </span>
   <span class="font13">genomic </span>
   <span class="font13">inline </span>
   <span class="font13">intestine </span>
   <span class="font13">multiple </span>
   <span class="font13">nucleotide </span>
   <span class="font13">popup </span>
   <span class="font13">raw </span>
   <span class="font13">residues </span>
   <span class="font13">scale </span>
   <span class="font13">study. </span>
   <span class="font13">table: </span>
   <span class="font13">thank </span>
   <span class="font13">tissue </span>
   <span class="font13">trees </span>
   <span class="font12">Branch </span>
   <span class="font12">China. </span>
   <span class="font12">CoV </span>
   <span class="font12">CrossRef </span>
   <span class="font12">Customs </span>
   <span class="font12">Dr. </span>
   <span class="font12">GD/PS </span>
   <span class="font12">Guangzhou </span>
   <span class="font12">Malayan </span>
   <span class="font12">Note </span>
   <span class="font12">ORFs </span>
   <span class="font12">able </span>
   <span class="font12">analysis. </span>
   <span class="font12">archived </span>
   <span class="font12">blue </span>
   <span class="font12">boxes, </span>
   <span class="font12">cell </span>
   <span class="font12">coronaviruses, </span>
   <span class="font12">coronaviruses. </span>
   <span class="font12">denoted </span>
   <span class="font12">designed </span>
   <span class="font12">detection </span>
   <span class="font12">effect </span>
   <span class="font12">hosts </span>
   <span class="font12">identified </span>
   <span class="font12">indicate </span>
   <span class="font12">likelihood </span>
   <span class="font12">likely </span>
   <span class="font12">line </span>
   <span class="font12">merged </span>
   <span class="font12">number </span>
   <span class="font12">operations </span>
   <span class="font12">predicted </span>
   <span class="font12">receptor-binding </span>
   <span class="font12">revealed </span>
   <span class="font12">sample </span>
   <span class="font12">source </span>
   <span class="font12">study, </span>
   <span class="font12">suggests </span>
   <span class="font12">viruses </span>
   <span class="font12">wild </span>
   <span class="font12">window </span>
   <span class="font12">zoonotic </span>
  </p>
 </body>
</html>
