<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/04/05/2020.04.03.022723.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.04.03.022723v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Prediction and Evolution of B Cell Epitopes of Surface Protein in SARS-CoV-2"/>
  <meta name="DC.Identifier" content="10.1101/2020.04.03.022723"/>
  <meta name="DC.Date" content="2020-04-05"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The discovery of epitopes is helpful to the development of SARS-CoV-2 vaccine. The sequences of the surface protein of SARS-CoV-2 and its proximal sequences were obtained by BLAST, the sequences of the whole genome of SARS-CoV-2 were obtained from the GenBank. Based on the NCBI Reference Sequence: NC_045512.2, the conformational and linear B cell epitopes of the surface protein were predicted separately by various prediction methods. Furthermore, the conservation of the epitopes, the adaptability and other evolutionary characteristics were also analyzed. 7 epitopes were predicted, including 5 linear epitopes and 2 conformational epitopes, one of the linear and one of the conformational were coincide. The epitope D mutated easily, but the other epitopes were very conservative and the epitope C was the most conservative. It is worth mentioning that all of the 6 dominated epitopes were absolutely conservative in nearly 1000 SARS-CoV-2 genomes, and they deserved further study. The findings would facilitate the vaccine development, had the potential to be directly applied on the treatment in this disease, but also have the potential to prevent the possible threats caused by other types of coronavirus."/>
  <meta name="DC.Contributor" content="Jerome R Lon"/>
  <meta name="DC.Contributor" content="Yunmeng Bai"/>
  <meta name="DC.Contributor" content="Bingxu Zhong"/>
  <meta name="DC.Contributor" content="Fuqaing Cai"/>
  <meta name="DC.Contributor" content="Hongli Du"/>
  <meta name="article:published_time" content="2020-04-05"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Prediction and Evolution of B Cell Epitopes of Surface Protein in SARS-CoV-2"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;The discovery of epitopes is helpful to the development of SARS-CoV-2 vaccine. The sequences of the surface protein of SARS-CoV-2 and its proximal sequences were obtained by BLAST, the sequences of the whole genome of SARS-CoV-2 were obtained from the GenBank. Based on the NCBI Reference Sequence: NC_045512.2, the conformational and linear B cell epitopes of the surface protein were predicted separately by various prediction methods. Furthermore, the conservation of the epitopes, the adaptability and other evolutionary characteristics were also analyzed. 7 epitopes were predicted, including 5 linear epitopes and 2 conformational epitopes, one of the linear and one of the conformational were coincide. The epitope D mutated easily, but the other epitopes were very conservative and the epitope C was the most conservative. It is worth mentioning that all of the 6 dominated epitopes were absolutely conservative in nearly 1000 SARS-CoV-2 genomes, and they deserved further study. The findings would facilitate the vaccine development, had the potential to be directly applied on the treatment in this disease, but also have the potential to prevent the possible threats caused by other types of coronavirus.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.04.03.022723v1"/>
  <meta name="citation_id" content="2020.04.03.022723v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.03.022723v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.03.022723v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.03.022723v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.022723.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.04.03.022723"/>
  <meta name="citation_num_pages" content="26"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.04.03.022723"/>
  <meta name="citation_author" content="Jerome R Lon"/>
  <meta name="citation_author_institution" content="School of Biology and Biological Engineering, South China University of Technology"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4714-6931"/>
  <meta name="citation_author" content="Yunmeng Bai"/>
  <meta name="citation_author_institution" content="School of Biology and Biological Engineering, South China University of Technology"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7915-4501"/>
  <meta name="citation_author" content="Bingxu Zhong"/>
  <meta name="citation_author_institution" content="School of Biology and Biological Engineering, South China University of Technology"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9961-2212"/>
  <meta name="citation_author" content="Fuqaing Cai"/>
  <meta name="citation_author_institution" content="School of Biology and Biological Engineering, South China University of Technology"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1477-8211"/>
  <meta name="citation_author" content="Hongli Du"/>
  <meta name="citation_author_institution" content="School of Biology and Biological Engineering, South China University of Technology"/>
  <meta name="citation_author_email" content="hldu@scut.edu.cn"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6419-0160"/>
  <meta name="citation_reference" content="Han Q, Lin Q, Jin S, You L. Recent insights into 2019-nCoV: a brief but comprehensive review. J Infect 2020. https://doi.org/10.1016/j.jinf.2020.02.010."/>
  <meta name="citation_reference" content="Liu Y, Gayle AA, Wilder-Smith A, Rocklöv J. The reproductive number of COVID-19 is higher compared to SARS coronavirus. J Travel Med 2020. https://doi.org/10.1093/jtm/taaa021."/>
  <meta name="citation_reference" content="El-Manzalawy Y, Honavar V. Recent advances in B-cell epitope prediction methods. Immunome Res 2010;6 Suppl 2:S2. https://doi.org/10.1186/1745-7580-6-S2-S2."/>
  <meta name="citation_reference" content="Sun P, Ju H, Liu Z, Ning Q, Zhang J, Zhao X et al. Bioinformatics resources and tools for conformational B-cell epitope prediction. Comput Math Methods Med 2013;2013:943636. https://doi.org/10.1155/2013/943636."/>
  <meta name="citation_reference" content="Morris AL, MacArthur MW, Hutchinson EG, Thornton JM. Stereochemical quality of protein structure coordinates. Proteins 1992;12(4):345–64. https://doi.org/10.1002/prot.340120407."/>
  <meta name="citation_reference" content="Groot AS de, Sbai H, Aubin CS, McMurry J, Martin W. Immuno-informatics: Mining genomes for vaccine components. Immunol Cell Biol 2002;80(3):255–69. https://doi.org/10.1046/j.1440-1711.2002.01092.x."/>
  <meta name="citation_reference" content="Yang X, Yu X. An introduction to epitope prediction methods and software. Rev Med Virol 2009;19(2):77–96. https://doi.org/10.1002/rmv.602."/>
  <meta name="citation_reference" content="Chen W, Zhong Y, Qin Y, Sun S, Li Z. The evolutionary pattern of glycosylation sites in influenza virus (H5N1) hemagglutinin and neuraminidase. PLoS ONE 2012;7(11):e49224. https://doi.org/10.1371/journal.pone.0049224."/>
  <meta name="citation_reference" content="Grifoni A, Sidney J, Zhang Y, Scheuermann RH, Peters B, Sette A. A Sequence Homology and Bioinformatic Approach Can Predict Candidate Targets for Immune Responses to SARS-CoV-2. Cell Host Microbe 2020. https://doi.org/10.1016/j.chom.2020.03.002."/>
  <meta name="citation_reference" content="Yuan M, Wu NC, Zhu X, Lee C-CD, So RTY, Lv H et al. A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV; 2020."/>
  <meta name="citation_reference" content="Lv H, Wu NC, Tsang OT-Y, Yuan M, Perera RAPM, Leung WS et al. Cross-reactive antibody response between SARS-CoV-2 and SARS-CoV infections; 2020."/>
  <meta name="citation_reference" content="Tian X, Li C, Huang A, Xia S, Lu S, Shi Z et al. Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody; 2020."/>
  <meta name="citation_reference" content="Yan R, Zhang Y, Guo Y, Xia L, Zhou Q. Structural basis for the recognition of the 2019-nCoV by human ACE2; 2020."/>
  <meta name="citation_reference" content="Wang S, Li G, Di WU, Cao Z. Mutation Feature Analysis on Epitope and Receptor Binding Sites of Influenza A H1N1 Hemagglutinin. ACTA BIOPHYSICA SINICA 2012;28(6):486. https://doi.org/10.3724/SP.J.1260.2012.20015."/>
  <meta name="citation_reference" content="Walker JM. The Proteomics Protocols Handbook. Dordrecht: Springer; 2005."/>
  <meta name="citation_reference" content="Blom N, Gammeltoft S, Brunak S. Sequence and structure-based prediction of eukaryotic protein phosphorylation sites. J Mol Biol 1999;294(5):1351–62. https://doi.org/10.1006/jmbi.1999.3310."/>
  <meta name="citation_reference" content="Waterhouse A, Bertoni M, Bienert S, Studer G, Tauriello G, Gumienny R et al. SWISS-MODEL: homology modelling of protein structures and complexes. Nucleic Acids Res 2018;46(W1):W296–W303. https://doi.org/10.1093/nar/gky427."/>
  <meta name="citation_reference" content="Laskowski RA, MacArthur MW, Moss DS, Thornton JM. PROCHECK: a program to check the stereochemical quality of protein structures. J Appl Crystallogr 1993;26(2):283–91. https://doi.org/10.1107/S0021889892009944."/>
  <meta name="citation_reference" content="Guex N, Peitsch MC. SWISS-MODEL and the Swiss-PdbViewer: an environment for comparative protein modeling. Electrophoresis 1997;18(15):2714–23. https://doi.org/10.1002/elps.1150181505."/>
  <meta name="citation_reference" content="Zhou C, Chen Z, Zhang L, Yan D, Mao T, Tang K et al. SEPPA 3.0-enhanced spatial epitope prediction enabling glycoprotein antigens. Nucleic Acids Res 2019;47(W1):W388–W394. https://doi.org/10.1093/nar/gkz413."/>
  <meta name="citation_reference" content="Karplus PA, Schulz GE. Prediction of chain flexibility in proteins. Naturwissenschaften 1985;72(4):212–3. https://doi.org/10.1007/BF01195768."/>
  <meta name="citation_reference" content="Emini EA, Hughes JV, Perlow DS, Boger J. Induction of hepatitis A virus-neutralizing antibody by a virus-specific synthetic peptide. J Virol 1985;55(3):836–9."/>
  <meta name="citation_reference" content="Jameson BA, Wolf H. The antigenic index: a novel algorithm for predicting antigenic determinants. Comput Appl Biosci 1988;4(1):181–6. https://doi.org/10.1093/bioinformatics/4.1.181."/>
  <meta name="citation_reference" content="Saha S, Raghava GPS. Prediction of continuous B-cell epitopes in an antigen using recurrent neural network. Proteins 2006;65(1):40–8. https://doi.org/10.1002/prot.21078."/>
  <meta name="citation_reference" content="Larsen JEP, Lund O, Nielsen M. Improved method for predicting linear B-cell epitopes. Immunome Res 2006;2:2. https://doi.org/10.1186/1745-7580-2-2."/>
  <meta name="citation_reference" content="Gupta R, Jung E, Brunak S. Prediction of N-glycosylation sites in human proteins 2004;46:203–6."/>
  <meta name="citation_reference" content="Ashkenazy H, Abadi S, Martz E, Chay O, Mayrose I, Pupko T et al. ConSurf 2016: an improved methodology to estimate and visualize evolutionary conservation in macromolecules. Nucleic Acids Res 2016;44(W1):W344–50. https://doi.org/10.1093/nar/gkw408."/>
  <meta name="citation_reference" content="Crooks GE, Hon G, Chandonia J-M, Brenner SE. WebLogo: a sequence logo generator. Genome Res 2004;14(6):1188–90. https://doi.org/10.1101/gr.849004."/>
  <meta name="citation_reference" content="Schneider TD, Stephens RM. Sequence logos: a new way to display consensus sequences. Nucleic Acids Res 1990;18(20):6097–100. https://doi.org/10.1093/nar/18.20.6097."/>
  <meta name="twitter:title" content="Prediction and Evolution of B Cell Epitopes of Surface Protein in SARS-CoV-2"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The discovery of epitopes is helpful to the development of SARS-CoV-2 vaccine. The sequences of the surface protein of SARS-CoV-2 and its proximal sequences were obtained by BLAST, the sequences of the whole genome of SARS-CoV-2 were obtained from the GenBank. Based on the NCBI Reference Sequence: NC_045512.2, the conformational and linear B cell epitopes of the surface protein were predicted separately by various prediction methods. Furthermore, the conservation of the epitopes, the adaptability and other evolutionary characteristics were also analyzed. 7 epitopes were predicted, including 5 linear epitopes and 2 conformational epitopes, one of the linear and one of the conformational were coincide. The epitope D mutated easily, but the other epitopes were very conservative and the epitope C was the most conservative. It is worth mentioning that all of the 6 dominated epitopes were absolutely conservative in nearly 1000 SARS-CoV-2 genomes, and they deserved further study. The findings would facilitate the vaccine development, had the potential to be directly applied on the treatment in this disease, but also have the potential to prevent the possible threats caused by other types of coronavirus."/>
  <meta name="og-title" property="og:title" content="Prediction and Evolution of B Cell Epitopes of Surface Protein in SARS-CoV-2"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.03.022723v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The discovery of epitopes is helpful to the development of SARS-CoV-2 vaccine. The sequences of the surface protein of SARS-CoV-2 and its proximal sequences were obtained by BLAST, the sequences of the whole genome of SARS-CoV-2 were obtained from the GenBank. Based on the NCBI Reference Sequence: NC_045512.2, the conformational and linear B cell epitopes of the surface protein were predicted separately by various prediction methods. Furthermore, the conservation of the epitopes, the adaptability and other evolutionary characteristics were also analyzed. 7 epitopes were predicted, including 5 linear epitopes and 2 conformational epitopes, one of the linear and one of the conformational were coincide. The epitope D mutated easily, but the other epitopes were very conservative and the epitope C was the most conservative. It is worth mentioning that all of the 6 dominated epitopes were absolutely conservative in nearly 1000 SARS-CoV-2 genomes, and they deserved further study. The findings would facilitate the vaccine development, had the potential to be directly applied on the treatment in this disease, but also have the potential to prevent the possible threats caused by other types of coronavirus."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-04-05"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Prediction and Evolution of B Cell Epitopes of Surface Protein in SARS-CoV-2 | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1229962 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">The discovery of epitopes is helpful to the development of SARS-CoV-2 vaccine. The sequences of the surface protein of SARS-CoV-2 and its proximal sequences were obtained by BLAST, the sequences of the whole genome of SARS-CoV-2 were obtained from the GenBank. Based on the NCBI Reference Sequence: NC_045512.2, the conformational and linear B cell epitopes of the surface protein were predicted separately by various prediction methods. Furthermore, the conservation of the epitopes, the adaptability and other evolutionary characteristics were also analyzed. 7 epitopes were predicted, including 5 linear epitopes and 2 conformational epitopes, one of the linear and one of the conformational were coincide. The epitope D mutated easily, but the other epitopes were very conservative and the epitope C was the most conservative. It is worth mentioning that all of the 6 dominated epitopes were absolutely conservative in nearly 1000 SARS-CoV-2 genomes, and they deserved further study. The findings would facilitate the vaccine development, had the potential to be directly applied on the treatment in this disease, but also have the potential to prevent the possible threats caused by other types of coronavirus.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-3">In late December 2019, a novel coronavirus was officially named as SARS-CoV-2 by World Health Organization(WHO) and identified as the pathogen causing outbreaks of SARS-like and MERS-like illness in Chinese city of Wuhan, which was a zoonotic disease. As of March 13, 2020, the outbreak of SARS-CoV-2 has been reported in many areas of the world, with more than 130,000 people infected[
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. With an alarmingly human-to-human transmissibility, the reproductive number of SARS-CoV-2 has been computed to around 3.28 [
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. According to the data in NGDC(National Genomics Data Center), at 15:00(GMT+8) on March 13, 2020, 482 genomic variations of SARS-CoV-2 has been reported, which has aroused widespread concern.
    </p>
    <p id="p-4">The B cell epitope of viral surface protein can specifically bind to the host’s B cell antigen receptor and induce the body to produce protective antibody and humoral immune response. The discovery of epitopes is helpful to the development of SARS-CoV-2 vaccine and the understanding of SARS-CoV-2’s pathogenesis[
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. 3 proteins embedded in the virus envelope of SARS-CoV-2 have been identified, Spike(S) protein, Envelope(E) protein, Membrane(M) protein. At present, due to the lack of study of the crystal structure of surface protein of SARS-CoV-2, the study of epitopes is time-consuming, power-consuming, cost and difficult [
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>].
    </p>
    <p id="p-5">In this work, we analyzed the surface protein of SARS-CoV-2, predicted the structures with bioinformatics methods. On the basis, we predicted the linear and conformational B cell epitopes, analyzed the conservation of the epitopes, the adaptability and other evolutionary characteristics of the surface protein, which provided a theoretical basis for the vaccine development and prevention of SARS-CoV-2.</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Results</h2>
    <div id="sec-3" class="subsection">
     <h3>Basic analysis of surface protein of SARS-CoV-2</h3>
     <p id="p-6">The primary structure and physicochemical properties of the S/E/M protein were analyzed. The results revealed that the S protein has an average hydrophilic index of −0.079
      <strong>(Figure S1A)</strong>. On the basis of hydrophilicity, it also showed amphoteric properties. There was an outside-in transmembrane helix in 23 residues from position 1214
      <sup>th</sup> to position 1236
      <sup>th</sup> at the N-terminal(Figure S2A). The protein instability index was 33.01, which revealed the S protein was stable. The E protein has an average hydrophilic index of 1.128(Figure S1B). It was hydrophobic. An inside-out transmembrane helix in 23 residues from position 12
      <sup>th</sup> to position 34
      <sup>th</sup> at the N-terminal was predicted(Figure S2B). The protein instability index was 38.68, which revealed the E protein was stable. The M protein has an average hydrophilic index of 0.446(Figure S1C). On the basis of hydrophobicity, it also showed amphoteric properties. There were two outside-in transmembrane helices, one was in 20 residues from position 20
      <sup>th</sup> to position 39
      <sup>th</sup>, the another one was in 23 residues from position 78
      <sup>th</sup> to position 100
      <sup>th</sup>, and an inside-out transmembrane helix in 20 residues from position 51
      <sup>st</sup> to position 73
      <sup>rd</sup>, at the N-terminal(Figure S2C). The protein instability index was 39.14, which revealed the M protein was stable.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Prediction of the 3D structure of surface protein of SARS-CoV-2</h3>
     <p id="p-7">The optimal template for homology modeling of the S protein of SARS-CoV-2 was the S protein of SARS(PDB ID: 6acc.1), with the sequence identity of 76.47% and the GMQE score of 0.73. According to the evaluation of the structure by Ramachandran plot(
      <strong>
       <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>
      </strong>), 99.3% of the residues were located in the most favoured regions and the allowed regions, 0.7% of the residues were located in the disallowed regions, the high-energy regions(
      <strong>
       <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>
      </strong>), which was possibly due to some energy was spent in the protein processing to make these residues enter the high-energy regions [
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. The result generally showed that the structure was reliable. The structure of S protein(
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>) of SARS-CoV-2 is a trimer, which can be divided into a tightly curled tail and a distributed head. The head is mainly composed of β-sheet, irregular curl and turn, which is exposed to the envelope of the virus, contributes to the formation of epitopes. The tail is mainly composed of several α-helices, part of which is embedded in the envelope, hinders the formation of epitopes.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1236756/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1236756/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1236756" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1</span>
       <span class="caption-title">The plot statistics of the Ramachandran plot</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.022723/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The 3D structure prediction and Ramachandran plot analysis of the S and E protein A. The 3D structure of the S protein predicted by homology modeling. It is a trimer, the head contains RBD (receptor Binding Domain) [24], the tail contains the basic elements required for the membrane fusion, the end of the tail is a transmembrane region and is embedded in the envelope of SARS-CoV-2. B. The 3D structure of the E protein predicted by homology modeling. It is a pentamer with ion channel activity [25]. Its head is short, the middle of the tail is a transmembrane region which help the E protein embed in the envelope of SARS-CoV-2. C. The Ramachandran plot analysis of the 3D structure of the S protein (without Gly and Pro). Most residues located in the red (core) regions, and few in the white regions. D. The Ramachandran plot analysis of the 3D structure of the E protein (without Gly and Pro). All of the residues located on the red(core) region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1112312117" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;The 3D structure prediction and Ramachandran plot analysis of the S and E protein&lt;;/span&gt;; &lt;;strong&gt;;A.&lt;;/strong&gt;; The 3D structure of the S protein predicted by homology modeling. It is a trimer, the head contains RBD (receptor Binding Domain) [24], the tail contains the basic elements required for the membrane fusion, the end of the tail is a transmembrane region and is embedded in the envelope of SARS-CoV-2. &lt;;strong&gt;;B.&lt;;/strong&gt;; The 3D structure of the E protein predicted by homology modeling. It is a pentamer with ion channel activity [25]. Its head is short, the middle of the tail is a transmembrane region which help the E protein embed in the envelope of SARS-CoV-2. &lt;;strong&gt;;C.&lt;;/strong&gt;; The Ramachandran plot analysis of the 3D structure of the S protein (without Gly and Pro). Most residues located in the red (core) regions, and few in the white regions. &lt;;strong&gt;;D.&lt;;/strong&gt;; The Ramachandran plot analysis of the 3D structure of the E protein (without Gly and Pro). All of the residues located on the red(core) region.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.022723/F1.medium.gif" width="440" height="278"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.022723/F1.medium.gif" width="440" height="278"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.022723/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.022723/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1</span>
       <span class="caption-title">The 3D structure prediction and Ramachandran plot analysis of the S and E protein</span>
       <p id="p-8" class="first-child">
        <strong>A.</strong> The 3D structure of the S protein predicted by homology modeling. It is a trimer, the head contains RBD (receptor Binding Domain) [
        <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>], the tail contains the basic elements required for the membrane fusion, the end of the tail is a transmembrane region and is embedded in the envelope of SARS-CoV-2. 
        <strong>B.</strong> The 3D structure of the E protein predicted by homology modeling. It is a pentamer with ion channel activity [
        <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. Its head is short, the middle of the tail is a transmembrane region which help the E protein embed in the envelope of SARS-CoV-2. 
        <strong>C.</strong> The Ramachandran plot analysis of the 3D structure of the S protein (without Gly and Pro). Most residues located in the red (core) regions, and few in the white regions. 
        <strong>D.</strong> The Ramachandran plot analysis of the 3D structure of the E protein (without Gly and Pro). All of the residues located on the red(core) region.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-9">The optimal template for homology modeling of the E protein of SARS-CoV-2 was the E protein of SARS(PDB ID: 5×29.1), with the sequence identity of 91.38% and the GMQE score of 0.73. According to the evaluation of the structure by Ramachandran plot(
      <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>), 100% of the residues were located in the most favoured regions(
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), indicating that the structure was reliable. The E protein of SARS-CoV-2 is a pentamer(
      <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1D</a>), which can be divided into the concentrated transmembrane part and the head located outside the envelope. The head is mainly composed of α-helix, irregular curl and turn, which is exposed to the envelope, contributes to the formation of epitopes. The tail is mainly composed of long α-helix, most of which are embedded in the envelope, hinders the formation of epitopes.
     </p>
     <p id="p-10">The optimal template for homology modeling of the M protein of SARS-CoV-2 was the effector protein Zt-KP6-1(PDB ID: 6qpk. 1. A), with the sequence identity of 20.00% and the GMQE score of 0.06. The sequence identity between the optimal template and the M protein of SARS-CoV-2 and the GMQE score are too low, so that the template is not suitable for homology modeling.</p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Prediction of linear B cell epitopes</h3>
     <p id="p-11">All linear B cell epitopes of the surface protein were filtered according to the following criteria: (1) region with high surface probability(≥0.75), strong antigenicity(≥0) and high flexibility; (2) excluding the region with α-helix, β-sheet and glycosylation site(
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>); (3) in line with the prediction by BepiPred 2.0(cut off to 0.35)
      <strong>(Table S1)</strong> and ABCpred(cut off to 0.51)(Table S2). Based on the results obtained with these methods and artificial optimization, 4 potential linear B cell epitopes of the S protein were predicted(
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, 
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>), including 601-605 aa, 656-660 aa, 676-682 aa, 808-813 aa, and they were named as the epitope A, B, C, D, respectively; 1 epitope of the E protein was selected(60-65 aa) and named as the epitope F(
      <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, 
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3C</a>); 1 epitope of the M protein was selected (211-215 aa) and named as the epitope H(
      <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).
     </p>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1236761/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1236761/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1236761" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 2</span>
       <span class="caption-title">The composition and the antigenic index of the epitopes of SARS-CoV-2</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.022723/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The secondary structures and properties analysis of the S, E and M protein with the Protean tool of DNAStar A. Analysis of the S protein. It contains most [[#x3B1]]-helix and [[#x3B2]]-sheet, some Turn and Coli region, several discontinuous high flexibility fragments, fluctuant surface probablity with a few of positive peak and several antigenicity regions with positive peak. B. Analysis of the E protein. It contains most [[#x3B1]]-helix and [[#x3B2]]-sheet, some Turn and Coli region, three high flexibility fragments, few surface probablity regions and two antigenicity regions with positive peak in the begin and the end of polypeptide chain, respectively. C. Analysis of the M protein. It contains most [[#x3B1]]-helix and [[#x3B2]]-sheet, some Turn and Coli region, several high flexibility fragments, few surface probablity regions, two antigenicity regiona with positive single peak in the begin and middle of peptide chain, respectively, and consecutive positive peaks in the end." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1112312117" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;The secondary structures and properties analysis of the S, E and M protein with the Protean tool of DNAStar&lt;;/span&gt;; &lt;;strong&gt;;A.&lt;;/strong&gt;; Analysis of the S protein. It contains most [[#x3B1]]-helix and [[#x3B2]]-sheet, some Turn and Coli region, several discontinuous high flexibility fragments, fluctuant surface probablity with a few of positive peak and several antigenicity regions with positive peak. &lt;;strong&gt;;B.&lt;;/strong&gt;; Analysis of the E protein. It contains most [[#x3B1]]-helix and [[#x3B2]]-sheet, some Turn and Coli region, three high flexibility fragments, few surface probablity regions and two antigenicity regions with positive peak in the begin and the end of polypeptide chain, respectively. &lt;;strong&gt;;C.&lt;;/strong&gt;; Analysis of the M protein. It contains most [[#x3B1]]-helix and [[#x3B2]]-sheet, some Turn and Coli region, several high flexibility fragments, few surface probablity regions, two antigenicity regiona with positive single peak in the begin and middle of peptide chain, respectively, and consecutive positive peaks in the end.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.022723/F2.medium.gif" width="440" height="235"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.022723/F2.medium.gif" width="440" height="235"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.022723/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.022723/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2</span>
       <span class="caption-title">The secondary structures and properties analysis of the S, E and M protein with the Protean tool of DNAStar</span>
       <p id="p-12" class="first-child">
        <strong>A.</strong> Analysis of the S protein. It contains most α-helix and β-sheet, some Turn and Coli region, several discontinuous high flexibility fragments, fluctuant surface probablity with a few of positive peak and several antigenicity regions with positive peak. 
        <strong>B.</strong> Analysis of the E protein. It contains most α-helix and β-sheet, some Turn and Coli region, three high flexibility fragments, few surface probablity regions and two antigenicity regions with positive peak in the begin and the end of polypeptide chain, respectively. 
        <strong>C.</strong> Analysis of the M protein. It contains most α-helix and β-sheet, some Turn and Coli region, several high flexibility fragments, few surface probablity regions, two antigenicity regiona with positive single peak in the begin and middle of peptide chain, respectively, and consecutive positive peaks in the end.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.022723/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The predicted epitopes of the S and E protein A. The predicted linear B-cell epitopes of the S protein. The epitope A, B, C located in the forepart of the tail, the epitope D located in the back part of the tail and is close to the transmembrane region. B. The predicted conformational B-cell epitope of the S protein. It located in the RBD of the head which is the vital sites binding with ACE2. C. The predicted B-cell epitope of the E protein. The epitope G is the linear epitope and the F is the conformational epitope, which are coincide." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1112312117" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;The predicted epitopes of the S and E protein&lt;;/span&gt;; &lt;;strong&gt;;A.&lt;;/strong&gt;; The predicted linear B-cell epitopes of the S protein. The epitope A, B, C located in the forepart of the tail, the epitope D located in the back part of the tail and is close to the transmembrane region. &lt;;strong&gt;;B.&lt;;/strong&gt;; The predicted conformational B-cell epitope of the S protein. It located in the RBD of the head which is the vital sites binding with ACE2. &lt;;strong&gt;;C.&lt;;/strong&gt;; The predicted B-cell epitope of the E protein. The epitope G is the linear epitope and the F is the conformational epitope, which are coincide.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.022723/F3.medium.gif" width="440" height="263"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.022723/F3.medium.gif" width="440" height="263"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.022723/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/05/2020.04.03.022723/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3</span>
       <span class="caption-title">The predicted epitopes of the S and E protein</span>
       <p id="p-13" class="first-child">
        <strong>A.</strong> The predicted linear B-cell epitopes of the S protein. The epitope A, B, C located in the forepart of the tail, the epitope D located in the back part of the tail and is close to the transmembrane region. 
        <strong>B.</strong> The predicted conformational B-cell epitope of the S protein. It located in the RBD of the head which is the vital sites binding with ACE2. 
        <strong>C.</strong> The predicted B-cell epitope of the E protein. The epitope G is the linear epitope and the F is the conformational epitope, which are coincide.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Prediction of conformational B cell epitopes</h3>
     <p id="p-14">The conformational B cell epitopes of surface protein were predicted with Ellipro(Table S3) and SEPPA 3.0(Table S4) with the threshold of 0.063 and 0.5, respectively. After the artificial optimization, one conformational B-cell epitope (403-405,416,445,446,455,500 aa) of S protein was predicted(
      <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). It is obvious that the region located on the head of the S protein(
      <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>), which is the outside of SARS-CoV-2, making it easy to form an epitope. We selected it as a dominant conformational epitope and named it as the epitope E. Additionally, one conformational B-cell epitope(60-65 aa) of E protein was predicted(
      <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>), which is consistent with the epitope F of the E protein. Similarly, this region located on the outside(
      <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>), we selected it as a dominant conformational epitope and named G. However, the conformational epitope of the M protein could not be predicted due to the failure of credible homology modeling.
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Analysis of epitope conservation</h3>
     <p id="p-15">The Consurf Server was used to predict epitope conservative sites with the structure of surface proteins and the alignment results in different datasets (Table S5). All the epitopes of the S, E, M protein were absolute conservative among all SARS-CoV-2 sequences(
      <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3A</a>, Figure S3A-G). To further calculate the conservation of the epitopes in different coronavirus datasets, the representative sequences from SARS-CoV-2 were selected to participate in the human coronavirus dataset and the coronavirus dataset, due to amino acid sequences of some S or E or M protein were absolute conservative in SARS-CoV-2. The conservation was a little lower in human coronavirus than those of in SARS-CoV-2(
      <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 3B</a>, Figure S4A-G), and the epitope D was easy to mutate. The other epitopes were conservative and the epitope F/G was the most conservative one. As for the coronavirus(
      <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 3C</a>, Figure S5A-G), in the 5 epitopes of S protein, 4 of which obtained the conservative score less than 1, ranging from −0.854 to 0.256. It showed that the epitope C with the minimum score is the most stable and not easy to mutate. Besides, the score of the epitope D was 1.247, showing the relatively high possibility to mutate. The epitopes of the E protein were stable with the conservative score less than 1. The site conservation of the M protein could not be predicted due to the failure of credible homology modeling.
     </p>
     <div id="T3" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1236760/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1236760/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1236760" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 3A</span>
       <span class="caption-title">The conservation of the epitopes in SARS-CoV-2</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T4" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1236759/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1236759/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1236759" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 3B</span>
       <span class="caption-title">The conservation of the epitopes in human coronavirus</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T5" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1236763/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1236763/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1236763" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 3C</span>
       <span class="caption-title">The conservation of the epitopes in coronavirus</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">Discussion</h2>
    <p id="p-16">SARS-CoV-2 caused huge impact to human production, living and even life, has become a major challenge confronting the whole world. Development of vaccine is one of the effective means of prevention and treatment of the virus long-term. Epitope vaccine is the trend of development of vaccine due to the advantages of strong pertinence, less toxic and side effects and easy to transportation and storage [
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]. The determination of epitopes is the basis of the development and application of vaccine, and the clinical diagnosis and treatment. Currently, the methods which were mainly used are X-crystal diffraction method, immune experiment method and bioinformatics method. The first two are time-consuming and laborious, the bioinformatics method is gaining more and more credibility among researchers [
     <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>,
     <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>,
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. There are many factors to be considered in the prediction of epitopes by bioinformatics method, such as the surface probablity and flexibility of the epitopes. At the same time, it is necessary to exclude the structurally stable and non-deformable α-helix, β-sheet, glycosylation sites which may obscure the epitopes or alter the antigenicity, etc [
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. Even so, the predicted epitopes are still inaccurate [
     <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>]. Compared with the current study on SARS-CoV-2, this work adopted various prediction methods and 3D structure databases developed in recent years, which were based on artificial neural network, Hidden Markov Model(HMM), Support Vector Machine(SVM), etc, such as ABCpred, BepiPred2.0, SEPPA 3.0, IEDB, etc. Compared with prediction by a single method [
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>] or on the basis of epitopes of SARS [
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>], these methods and databases greatly improved the accuracy of prediction and had more bioinformatic meaning. We comprehensively analyzed the prediction results from the tools which were widely used, set up screening criteria on the basis of primary structure, secondary structure and tertiary structure, so that the prediction results would more accurate and reliable.
    </p>
    <p id="p-17">The S protein, the E protein and the M protein are surface proteins of SARS-CoV-2, which have the potential as antigenic molecules. However, the current study on the epitopes prediction of SARS-CoV-2 [
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>], due to the S protein has been reported to be the directly binding molecule of SARS-CoV-2 to ACE2[
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>], the prediction of epitopes is mainly focusing on the S protein, with few studies on the E protein and the M protein. In this work, we analyzed the S protein, the E protein and the M protein, predicted their epitopes. On the basis, 7 B cell epitopes were predicted, including 2 conformational and 6 linear B cell epitopes, one of the conformational and one of the linear are coincide. All of the epitope A, B, C, D located on the surface of the tail of the S protein, which is relatively easy to bind. The epitope E is located on the head of the S protein, which is the key area where the S protein recognizes and binds to ACE2 [
     <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>,
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>], has the potential to block the infection process. The epitope F and the epitope G located on the end of the head of the E protein, the two epitopes coincide, this may due to they are all consecutive and the secondary structure avoided the α-helix and the β-sheet. The epitope H is derived from the M protein, the structure and conservation could not be determined due to the inability to predict reliable structure. However, it could be known from the surface probablity scores that the epitope H is more likely to be located on the surface of the M protein.
    </p>
    <p id="p-18">The higher the conservation score calculated by the Consurf Server is, the more likely the site is to be mutated in the evolutionary process. When the score&lt;;1, the site is likely to be a conservative site; when the score is between 1 and 2, the site is a site which is likely to be a relatively easy mutation; when the score&gt;;2, the site is likely to be an easy mutation site [
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. In the 7 epitopes obtained, all the epitopes of the S, E, M protein were absolute conservative among all SARS-CoV-2 sequences. For the human coronavirus dataset and the coronavirus dataset, only the average conservative score of the epitope D is higher than 1, which is prone to mutation. The epitope D should not be used as an epitope of the S protein. The conservation of the epitope H could not be calculated by the PDB file, the application value of the epitope H needed further experimental verification. Although the epitopes could be integrally considered to be conservative, the independent residues of these epitopes could still easy to mutate. Except the epitope E, all of 6 dominate epitopes contain 1-2 residues which has a conservative score higher than 1(
     <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 3C</a>), indicating that these residues were likely to be easy mutation sites. These residues mostly located at the head or the tail of the epitopes, therefore, the mutation of these residues should be paid attention to, and the length of the epitopes should be adjusted according to the actual effect in application. The scores of epitopes in different datasets were different, which could due to the quantity of sequences in the datasets and the structures were analyzed in different situations.
    </p>
    <p id="p-19">In this work, we predicted 6 reliable epitopes: A, B, C, E, F/G and H. The reliability of the epitopes of the S protein was relatively better than that of the epitopes of the E protein and the M protein, indicating that the S protein is still the optimal choice for the prediction of epitopes and the development of vaccine. All of the 6 epitopes were able to achieve absolute conservation in SARS-CoV-2, and to achieve relative conservation in the data set, including SARS, etc. Therefore, the epitopes not only have the potential to be directly applied on the treatment in this disease, but also have the potential to prevent the possible threats caused by other types of coronavirus. In addition, although various factors of prediction were integrated in this work, more experimental data are needed to further verify whether all the 6 epitopes can induce the body to produce corresponding antibodies and generate specific humoral immunity, due to the limited data set and other factors.</p>
   </div>
   <div class="section" id="sec-9">
    <h2 class="">Materials and methods</h2>
    <div id="sec-10" class="subsection">
     <h3>Materials</h3>
     <p id="p-20">All of the analysis was based on the NCBI Reference Sequence: NC_045512.2. We obtained the sequence of S, E and M protein and its proximal sequences by BLAST, which got 420, 334 and 329 sequences in total from NCBI database respectively. We obtained the whole genome sequence of SARS-CoV-2 from Genbank and GISAID(959 in total), which were used to be a dataset after genome annotation. The genome sequences, which performed mistakes of translation, were deleted.</p>
    </div>
    <div id="sec-11" class="subsection">
     <h3>Methods</h3>
     <div id="sec-12" class="subsection">
      <h4>Basic analysis of surface protein of SARS-CoV-2</h4>
      <p id="p-21">The physical and chemical properties of target protein were analyzed by the Port-Param tool in ExPASy [
       <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>], including the primary structure of the target protein, molecular formula, theoretical isoelectric point, the protein instability index(the index&lt;;40 means the protein was stable), 
       <em>etc</em>. Online software, ProtScale, was used to deeply analyze the hydrophilicity and hydrophobicity of target protein and the distribution of hydrophilicity and hydrophobicity of polypeptide chains [
       <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>]. SARS-CoV-2 carried the S/E/M protein through the virus envelope, the transmembrane region of the protein was predicted online by TMHMM 2.0 [
       <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>].
      </p>
     </div>
     <div id="sec-13" class="subsection">
      <h4>Prediction of the 3D structure of target protein</h4>
      <p id="p-22">With the amino acid sequences of the surface protein of SARS-CoV-2 of NC_045512.2 as templates, based on homology modeling method, we predicted the 3D structure through the online server SWISS-MODEL[
       <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>], selected and optimized the optimal structure based on the template identity and GMQE value[
       <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>], the rationality of the structure was evaluated by Ramachandran plot [
       <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>] with PDBsum server. The structures were displayed and analyzed by SWISS-pdb Viewer v4.10 [
       <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>].
      </p>
     </div>
     <div id="sec-14" class="subsection">
      <h4>Prediction of conformational B cell epitopes of target protein of SARS-CoV-2</h4>
      <p id="p-23">Based on the structures, the conformational B cell epitopes were predicted by SEPPA 3.0 [
       <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>] and Ellipro [
       <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>] respectively, and the common predicted conformational B cell epitopes from two methods were selected for the further analysis.
      </p>
     </div>
     <div id="sec-15" class="subsection">
      <h4>Prediction of linear B cell epitopes of target protein of SARS-CoV-2</h4>
      <p id="p-24">The Protean module of DNAStar was used to predict the flexibility[
       <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>], surface probablity [
       <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>] and antigenic index [
       <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>] of the target protein of SARS-CoV-2. The linear B cell epitope was predicted by ABCpred [
       <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>] and BepiPred 2.0 [
       <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>] respectively and the common predicted linear B cell epitopes from two methods were selected for the further analysis. Coupled with the secondary structure, the tertiary structure and the glycosylation sites [
       <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>] 
       <em>etc</em>, the linear B cell epitopes were finally determined.
      </p>
     </div>
     <div id="sec-16" class="subsection">
      <h4>Analysis of epitope conservation</h4>
      <p id="p-25">Based on the PDB model and the multiple alignment result, we used the Consurf Server to analyze the conservation of amino acid sites of the epitopes online[
       <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. The conservation of epitopes on the surface protein of SARS-CoV-2 was analyzed by multiple alignment with MAFFT and Logo was drawn with Weblogo [
       <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>,
       <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>].
      </p>
     </div>
    </div>
   </div>
   <div class="section" id="sec-18">
    <h2 class="">Authors’ contribution</h2>
    <p id="p-26">JL conceived the study and participated in its design and coordination. HD participated in the design of the study and helped draft the manuscript. YB participated in analysis of conservation, sequence alignment and manuscript drafting. BZ participated in antigenic prediction. FC participated in drafting the manuscript. All authors read and approved the final manuscript.</p>
   </div>
   <div class="section" id="sec-19">
    <h2 class="">Competing interests</h2>
    <p id="p-27">The authors have declared no competing interests.</p>
   </div>
   <div class="section" id="sec-20">
    <h2 class="">Supplementary information</h2>
    <p id="p-29">
     <strong>Figure S1</strong> Deep analysis of hydrophilicity and hydrophobicity of surface protein of SARS-CoV-2
    </p>
    <p id="p-30">The online software, ProtScale, was used to predict the hydrophilicity and hydrophobicity of the surface protein deeply. 
     <strong>A.</strong> The S protein has a maximum score of hydrophobicity, 3.222 at the 7
     <sup>th</sup> site, which revealed a strong hydrophobicity; a minimum score of hydrophobicity, −2.589 at the 679
     <sup>th</sup> site, which revealed a strong hydrophilicity. The score of hydrophilicity and hydrophobicity on the polypeptide chain of S protein constantly fluctuates, with most of the scores being negative, which revealed the possibility that the protein had bisexual properties on the basis of hydrophilicity. 
     <strong>B.</strong> The E protein has a maximum score of hydrophobicity, 3.489 at the 21
     <sup>st</sup> and the 25
     <sup>th</sup> site, which revealed a strong hydrophobicity; a minimum score of hydrophobicity, −1.550 at the 65
     <sup>th</sup> site, which revealed a strong hydrophilicity. Most of the scores of the residues being positive, which revealed the possibility that the protein has obvious hydrophobicity. 
     <strong>C.</strong> The M protein has a maximum score of hydrophobicity, 2.978 at the 84
     <sup>th</sup> site, which revealed a strong hydrophobicity; a minimum score of hydrophobicity, −1.956 at the 211
     <sup>th</sup> and the 212
     <sup>th</sup> site, which revealed a strong hydrophilicity. The scores of hydrophilicity and hydrophobicity on the polypeptide chain of M protein showed large fluctuations, and the number of positive scores and negative scores were similar, the positive scores accounted for the majority, which revealed the possibility that the protein had bisexual properties on the basis of hydrophobicity.
    </p>
    <p id="p-31">
     <strong>Figure S2</strong> The transmembrane region of the surface protein of SARS-CoV-2 The S, E and M protein are embedded in the envelope of SARS-CoV-2, the transmembrane helix was predicted by TMHMM 2.0 server. All of three amino acid indexes were higher than 18, indicating the reliability of the prediction. 
     <strong>A.</strong> For the S protein, an outside-in transmembrane helix was predicted in the 23 residues of amino acids from position 1214
     <sup>th</sup> to position 1236
     <sup>th</sup> at the N-terminal. The amino acid index was 23.97303. 
     <strong>B.</strong> For the E protein, an inside-out transmembrane helix was predicted in the 23 residues of amino acids from position 12
     <sup>th</sup> to position 34
     <sup>th</sup> at the N-terminal. The amino acid index was 25.72521. 
     <strong>C.</strong> For the M protein, 2 outside-in transmembrane helices were predicted, which were a helix in the 20 residues of amino acids from position 20
     <sup>th</sup> to position 39
     <sup>th</sup> and a helix in the 23 residues of amino acids from position 78
     <sup>th</sup> to position 100
     <sup>th</sup> at the N-terminal. An inside-out helix was predicted in the 23 residues of amino acids from position 51
     <sup>st</sup> to position 73
     <sup>rd</sup> at the N-terminal. The amino acid index was 64.90522.
    </p>
    <p id="p-32">
     <strong>Figure S3</strong> The antigenic conservation of the surface protein in SARS-CoV-2 
     <strong>A.</strong> The epitope A was absolutely conservative in 756 SARS-CoV-2 genomes. 
     <strong>B.</strong> The epitope B was absolutely conservative in 756 SARS-CoV-2 genomes. 
     <strong>C.</strong> The epitope C was absolutely conservative in 756 SARS-CoV-2 genomes. 
     <strong>D.</strong> The epitope D was absolutely conservative in 756 SARS-CoV-2 genomes. 
     <strong>E.</strong> The epitope E was absolutely conservative in 756 SARS-CoV-2 genomes. 
     <strong>F.</strong> The epitope F/G was absolutely conservative in 939 SARS-CoV-2 genomes. 
     <strong>G.</strong> The epitope H was absolutely conservative in 913 SARS-CoV-2 genomes.
    </p>
    <p id="p-33">
     <strong>Figure S4</strong> The antigenic conservation of the surface protein in human coronavirus 
     <strong>A.</strong> The conservation of the epitope A in 331 human coronavirus genomes. 
     <strong>B.</strong> The conservation of the epitope B in 331 human coronavirus genomes. 
     <strong>C.</strong> The conservation of the epitope C in 331 human coronavirus genomes. 
     <strong>D.</strong> The conservation of the epitope D in 331 human coronavirus genomes. 
     <strong>E.</strong> The conservation of the epitope E in 331 human coronavirus genomes. 
     <strong>F.</strong> The conservation of the epitope F/G in 268 human coronavirus genomes. 
     <strong>G.</strong> The conservation of the epitope H in 268 human coronavirus genomes.
    </p>
    <p id="p-34">
     <strong>Figure S5</strong> The antigenic conservation of the surface protein in coronavirus 
     <strong>A.</strong> The conservation of the epitope A in 403 human coronavirus genomes. 
     <strong>B.</strong> The conservation of the epitope B in 403 human coronavirus genomes. 
     <strong>C.</strong> The conservation of the epitope C in 403 human coronavirus genomes. 
     <strong>D.</strong> The conservation of the epitope D in 403 human coronavirus genomes. 
     <strong>E.</strong> The conservation of the epitope E in 403 human coronavirus genomes. 
     <strong>F.</strong> The conservation of the epitope F/G in 334 human coronavirus genomes. 
     <strong>G.</strong> The conservation of the epitope in 327 human coronavirus genomes.
    </p>
    <p id="p-35">
     <strong>Table S1 Bepipred2.0 linear epitope prediction results</strong>
    </p>
    <p id="p-36">
     <strong>Table S2 ABCpred linear epitope prediction results</strong>
    </p>
    <p id="p-37">
     <strong>Table S3 Prediction results of conformational B cell epitopes of surface protein by Ellipo</strong>
    </p>
    <p id="p-38">
     <strong>Table S4 Prediction results of conformational B cell epitopes of surface protein by SEPPA3.0</strong>
    </p>
    <p id="p-39">
     <strong>Table S5 Conservation analysis of epitopes in different datasets by Consurf</strong>
    </p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-28">This work was supported by the National Key R&amp;;D Program of China (2018YFC0910201), the Key R&amp;;D Program of Guangdong Province (2019B020226001), and the Science and the Technology Planning Project of Guangzhou (201704020176). The Student Entrepreneurship and Innovation Center of the school of biology and biological engineering, South China University of Technology, also provided a lot of help during the preparation of the project.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">[1].</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.04.03.022723v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Han</span>
          <span class="cit-name-given-names">Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>
          <span class="cit-name-given-names">Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jin</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">You</span>
          <span class="cit-name-given-names">L.</span>
         </span>
         <span class="cit-article-title">Recent insights into 2019-nCoV: a brief but comprehensive review</span>. 
         <span class="cit-source">J Infect</span>
         <span class="cit-pub-date">2020</span>. 
         <a href="https://doi.org/10.1016/j.jinf.2020.02.010">https://doi.org/10.1016/j.jinf.2020.02.010</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[2].</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.04.03.022723v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gayle</span>
          <span class="cit-name-given-names">AA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wilder-Smith</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rocklöv</span>
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-article-title">The reproductive number of COVID-19 is higher compared to SARS coronavirus</span>. 
         <span class="cit-source">J Travel Med</span>
         <span class="cit-pub-date">2020</span>. 
         <a href="https://doi.org/10.1093/jtm/taaa021">https://doi.org/10.1093/jtm/taaa021</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[3].</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.3" data-doi="10.1186/1745-7580-6-S2-S2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">El-Manzalawy</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Honavar</span>
          <span class="cit-name-given-names">V.</span>
         </span>
         <span class="cit-article-title">Recent advances in B-cell epitope prediction methods</span>. 
         <abbr class="cit-jnl-abbrev">Immunome Res</abbr>
         <span class="cit-pub-date">2010</span>;
         <span class="cit-vol">6</span>
         <span class="cit-issue">Suppl 2</span>:
         <span class="cit-fpage">S2</span>. 
         <a href="https://doi.org/10.1186/1745-7580-6-S2-S2">https://doi.org/10.1186/1745-7580-6-S2-S2</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DImmunome%2BRes%26rft.volume%253D6%26rft.spage%253DS2%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1745-7580-6-S2-S2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1745-7580-6-S2-S2&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[4].</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sun</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ju</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ning</span>
          <span class="cit-name-given-names">Q</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>
          <span class="cit-name-given-names">X</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Bioinformatics resources and tools for conformational B-cell epitope prediction</span>. 
         <abbr class="cit-jnl-abbrev">Comput Math Methods Med</abbr>
         <span class="cit-pub-date">2013;2013</span>:
         <span class="cit-fpage">943636</span>. 
         <a href="https://doi.org/10.1155/2013/943636">https://doi.org/10.1155/2013/943636</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[5].</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.5" data-doi="10.1002/prot.340120407">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Morris</span>
          <span class="cit-name-given-names">AL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">MacArthur</span>
          <span class="cit-name-given-names">MW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hutchinson</span>
          <span class="cit-name-given-names">EG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Thornton</span>
          <span class="cit-name-given-names">JM</span>
         </span>. 
         <span class="cit-article-title">Stereochemical quality of protein structure coordinates</span>. 
         <abbr class="cit-jnl-abbrev">Proteins</abbr>
         <span class="cit-pub-date">1992</span>;
         <span class="cit-vol">12</span>(
         <span class="cit-issue">4</span>):
         <span class="cit-fpage">345</span>–
         <span class="cit-lpage">64</span>. 
         <a href="https://doi.org/10.1002/prot.340120407">https://doi.org/10.1002/prot.340120407</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProteins%26rft.stitle%253DProteins%26rft.aulast%253DMorris%26rft.auinit1%253DA.%2BL.%26rft.volume%253D12%26rft.issue%253D4%26rft.spage%253D345%26rft.epage%253D364%26rft.atitle%253DStereochemical%2Bquality%2Bof%2Bprotein%2Bstructure%2Bcoordinates.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fprot.340120407%26rft_id%253Dinfo%253Apmid%252F1579569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/prot.340120407&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=1579569&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.022723.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1992HJ85200006&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[6].</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.6" data-doi="10.1046/j.1440-1711.2002.01092.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Groot AS</span>
          <span class="cit-name-surname">de</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sbai</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Aubin</span>
          <span class="cit-name-given-names">CS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McMurry</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Martin</span>
          <span class="cit-name-given-names">W.</span>
         </span>
         <span class="cit-article-title">Immuno-informatics: Mining genomes for vaccine components</span>. 
         <abbr class="cit-jnl-abbrev">Immunol Cell Biol</abbr>
         <span class="cit-pub-date">2002</span>;
         <span class="cit-vol">80</span>(
         <span class="cit-issue">3</span>):
         <span class="cit-fpage">255</span>–
         <span class="cit-lpage">69</span>. 
         <a href="https://doi.org/10.1046/j.1440-1711.2002.01092.x">https://doi.org/10.1046/j.1440-1711.2002.01092.x</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DImmunology%2Band%2BCell%2BBiology%2B%2528Print%2529%26rft.stitle%253DImmunology%2Band%2BCell%2BBiology%2B%2528Print%2529%26rft.aulast%253DDe%2BGroot%26rft.auinit1%253DA.%2BS.%26rft.volume%253D80%26rft.issue%253D3%26rft.spage%253D255%26rft.epage%253D269%26rft.atitle%253DImmuno-informatics%253A%2BMining%2Bgenomes%2Bfor%2Bvaccine%2Bcomponents.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1440-1711.2002.01092.x%26rft_id%253Dinfo%253Apmid%252F12067413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1046/j.1440-1711.2002.01092.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=12067413&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.022723.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[7].</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.7" data-doi="10.1002/rmv.602">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yu</span>
          <span class="cit-name-given-names">X.</span>
         </span>
         <span class="cit-article-title">An introduction to epitope prediction methods and software</span>. 
         <abbr class="cit-jnl-abbrev">Rev Med Virol</abbr>
         <span class="cit-pub-date">2009</span>;
         <span class="cit-vol">19</span>(
         <span class="cit-issue">2</span>):
         <span class="cit-fpage">77</span>–
         <span class="cit-lpage">96</span>. 
         <a href="https://doi.org/10.1002/rmv.602">https://doi.org/10.1002/rmv.602</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DReviews%2Bin%2Bmedical%2Bvirology%26rft.stitle%253DRev%2BMed%2BVirol%26rft.aulast%253DYang%26rft.auinit1%253DX.%26rft.volume%253D19%26rft.issue%253D2%26rft.spage%253D77%26rft.epage%253D96%26rft.atitle%253DAn%2Bintroduction%2Bto%2Bepitope%2Bprediction%2Bmethods%2Band%2Bsoftware.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Frmv.602%26rft_id%253Dinfo%253Apmid%252F19101924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/rmv.602&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19101924&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.022723.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000264423400003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[8].</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.8" data-doi="10.1371/journal.pone.0049224">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhong</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Qin</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sun</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">Z.</span>
         </span>
         <span class="cit-article-title">The evolutionary pattern of glycosylation sites in influenza virus (H5N1) hemagglutinin and neuraminidase</span>. 
         <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>
         <span class="cit-pub-date">2012</span>;
         <span class="cit-vol">7</span>(
         <span class="cit-issue">11</span>):
         <span class="cit-fpage">e49224</span>. 
         <a href="https://doi.org/10.1371/journal.pone.0049224">https://doi.org/10.1371/journal.pone.0049224</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DChen%26rft.auinit1%253DW.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De49224%26rft.epage%253De49224%26rft.atitle%253DThe%2BEvolutionary%2BPattern%2Bof%2BGlycosylation%2BSites%2Bin%2BInfluenza%2BVirus%2B%2528H5N1%2529%2BHemagglutinin%2Band%2BNeuraminidase.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0049224%26rft_id%253Dinfo%253Apmid%252F23133677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0049224&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23133677&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.022723.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[9].</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.04.03.022723v1.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Grifoni</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sidney</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Scheuermann</span>
          <span class="cit-name-given-names">RH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Peters</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sette</span>
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-article-title">A Sequence Homology and Bioinformatic Approach Can Predict Candidate Targets for Immune Responses to SARS-CoV-2</span>. 
         <span class="cit-source">Cell Host Microbe</span>
         <span class="cit-pub-date">2020</span>. 
         <a href="https://doi.org/10.1016/j.chom.2020.03.002">https://doi.org/10.1016/j.chom.2020.03.002</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[10].</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yuan</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">NC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>
          <span class="cit-name-given-names">C-CD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">So</span>
          <span class="cit-name-given-names">RTY</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lv</span>
          <span class="cit-name-given-names">H</span>
         </span>
         <span class="cit-etal">et al.</span>
         <abbr class="cit-jnl-abbrev">A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV</abbr>; 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[11].</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lv</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">NC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tsang</span>
          <span class="cit-name-given-names">OT-Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yuan</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Perera</span>
          <span class="cit-name-given-names">RAPM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leung</span>
          <span class="cit-name-given-names">WS</span>
         </span>
         <span class="cit-etal">et al.</span>
         <abbr class="cit-jnl-abbrev">Cross-reactive antibody response between SARS-CoV-2 and SARS-CoV infections</abbr>; 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[12].</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tian</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xia</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>
          <span class="cit-name-given-names">Z</span>
         </span>
         <span class="cit-etal">et al.</span>
         <abbr class="cit-jnl-abbrev">Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</abbr>; 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[13].</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guo</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xia</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>
          <span class="cit-name-given-names">Q.</span>
         </span>
         <abbr class="cit-jnl-abbrev">Structural basis for the recognition of the 2019-nCoV by human ACE2</abbr>; 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[14].</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Di</span>
          <span class="cit-name-given-names">WU</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cao</span>
          <span class="cit-name-given-names">Z.</span>
         </span>
         <span class="cit-article-title">Mutation Feature Analysis on Epitope and Receptor Binding Sites of Influenza A H1N1 Hemagglutinin</span>. 
         <abbr class="cit-jnl-abbrev">ACTA BIOPHYSICA SINICA</abbr>
         <span class="cit-pub-date">2012</span>;
         <span class="cit-vol">28</span>(
         <span class="cit-issue">6</span>):
         <span class="cit-fpage">486</span>. 
         <a href="https://doi.org/10.3724/SP.J.1260.2012.20015">https://doi.org/10.3724/SP.J.1260.2012.20015</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DACTA%2BBIOPHYSICA%2BSINICA%26rft.volume%253D28%26rft.spage%253D486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[15].</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-book" id="cit-2020.04.03.022723v1.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Walker</span>
          <span class="cit-name-given-names">JM</span>
         </span>. 
         <span class="cit-source">The Proteomics Protocols Handbook</span>. 
         <span class="cit-publ-loc">Dordrecht</span>: 
         <span class="cit-publ-name">Springer</span>; 
         <span class="cit-pub-date">2005</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[16].</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.16" data-doi="10.1006/jmbi.1999.3310">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Blom</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gammeltoft</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brunak</span>
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-article-title">Sequence and structure-based prediction of eukaryotic protein phosphorylation sites</span>. 
         <abbr class="cit-jnl-abbrev">J Mol Biol</abbr>
         <span class="cit-pub-date">1999</span>;
         <span class="cit-vol">294</span>(
         <span class="cit-issue">5</span>):
         <span class="cit-fpage">1351</span>–
         <span class="cit-lpage">62</span>. 
         <a href="https://doi.org/10.1006/jmbi.1999.3310">https://doi.org/10.1006/jmbi.1999.3310</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBlom%26rft.auinit1%253DN.%26rft.volume%253D294%26rft.issue%253D5%26rft.spage%253D1351%26rft.epage%253D1362%26rft.atitle%253DSequence%2Band%2Bstructure-based%2Bprediction%2Bof%2Beukaryotic%2Bprotein%2Bphosphorylation%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1999.3310%26rft_id%253Dinfo%253Apmid%252F10600390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1006/jmbi.1999.3310&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=10600390&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.022723.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000084596800025&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[17].</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.17" data-doi="10.1093/nar/gky427">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Waterhouse</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bertoni</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bienert</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Studer</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tauriello</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gumienny</span>
          <span class="cit-name-given-names">R</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SWISS-MODEL: homology modelling of protein structures and complexes</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">46</span>(
         <span class="cit-issue">W1</span>):
         <span class="cit-fpage">W296</span>–
         <span class="cit-lpage">W303</span>. 
         <a href="https://doi.org/10.1093/nar/gky427">https://doi.org/10.1093/nar/gky427</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DW296%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky427%26rft_id%253Dinfo%253Apmid%252F29788355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gky427&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29788355&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.022723.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[18].</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.18" data-doi="10.1107/S0021889892009944">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Laskowski</span>
          <span class="cit-name-given-names">RA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">MacArthur</span>
          <span class="cit-name-given-names">MW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Moss</span>
          <span class="cit-name-given-names">DS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Thornton</span>
          <span class="cit-name-given-names">JM</span>
         </span>. 
         <span class="cit-article-title">PROCHECK: a program to check the stereochemical quality of protein structures</span>. 
         <abbr class="cit-jnl-abbrev">J Appl Crystallogr</abbr>
         <span class="cit-pub-date">1993</span>;
         <span class="cit-vol">26</span>(
         <span class="cit-issue">2</span>):
         <span class="cit-fpage">283</span>–
         <span class="cit-lpage">91</span>. 
         <a href="https://doi.org/10.1107/S0021889892009944">https://doi.org/10.1107/S0021889892009944</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BAppl%2BCrystallogr%26rft.volume%253D26%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0021889892009944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1107/S0021889892009944&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1993KY85100021&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[19].</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.19" data-doi="10.1002/elps.1150181505">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Guex</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Peitsch</span>
          <span class="cit-name-given-names">MC</span>
         </span>. 
         <span class="cit-article-title">SWISS-MODEL and the Swiss-PdbViewer: an environment for comparative protein modeling</span>. 
         <abbr class="cit-jnl-abbrev">Electrophoresis</abbr>
         <span class="cit-pub-date">1997</span>;
         <span class="cit-vol">18</span>(
         <span class="cit-issue">15</span>):
         <span class="cit-fpage">2714</span>–
         <span class="cit-lpage">23</span>. 
         <a href="https://doi.org/10.1002/elps.1150181505">https://doi.org/10.1002/elps.1150181505</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DElectrophoresis%26rft.stitle%253DElectrophoresis%26rft.aulast%253DGuex%26rft.auinit1%253DN.%26rft.volume%253D18%26rft.issue%253D15%26rft.spage%253D2714%26rft.epage%253D2723%26rft.atitle%253DSWISS-MODEL%2Band%2Bthe%2BSwiss-PdbViewer%253A%2Ban%2Benvironment%2Bfor%2Bcomparative%2Bprotein%2Bmodeling.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Felps.1150181505%26rft_id%253Dinfo%253Apmid%252F9504803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/elps.1150181505&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=9504803&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.022723.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000071948300004&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[20].</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mao</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tang</span>
          <span class="cit-name-given-names">K</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SEPPA 3.0-enhanced spatial epitope prediction enabling glycoprotein antigens</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-pub-date">2019</span>;
         <span class="cit-vol">47</span>(
         <span class="cit-issue">W1</span>):
         <span class="cit-fpage">W388</span>–
         <span class="cit-lpage">W394</span>. 
         <a href="https://doi.org/10.1093/nar/gkz413">https://doi.org/10.1093/nar/gkz413</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253DW388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[21].</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.21" data-doi="10.1007/BF01195768">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Karplus</span>
          <span class="cit-name-given-names">PA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schulz</span>
          <span class="cit-name-given-names">GE</span>
         </span>. 
         <span class="cit-article-title">Prediction of chain flexibility in proteins</span>. 
         <abbr class="cit-jnl-abbrev">Naturwissenschaften</abbr>
         <span class="cit-pub-date">1985</span>;
         <span class="cit-vol">72</span>(
         <span class="cit-issue">4</span>):
         <span class="cit-fpage">212</span>–
         <span class="cit-lpage">3</span>. 
         <a href="https://doi.org/10.1007/BF01195768">https://doi.org/10.1007/BF01195768</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNaturwissenschaften%26rft.volume%253D72%26rft.spage%253D212%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01195768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/BF01195768&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1985AJQ6200010&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[22].</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.22">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Emini</span>
          <span class="cit-name-given-names">EA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hughes</span>
          <span class="cit-name-given-names">JV</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Perlow</span>
          <span class="cit-name-given-names">DS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Boger</span>
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-article-title">Induction of hepatitis A virus-neutralizing antibody by a virus-specific synthetic peptide</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-pub-date">1985</span>;
         <span class="cit-vol">55</span>(
         <span class="cit-issue">3</span>):
         <span class="cit-fpage">836</span>–
         <span class="cit-lpage">9</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DEmini%26rft.auinit1%253DE%2BA%26rft.volume%253D55%26rft.issue%253D3%26rft.spage%253D836%26rft.epage%253D839%26rft.atitle%253DInduction%2Bof%2Bhepatitis%2BA%2Bvirus-neutralizing%2Bantibody%2Bby%2Ba%2Bvirus-specific%2Bsynthetic%2Bpeptide.%26rft_id%253Dinfo%253Apmid%252F2991600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjU1LzMvODM2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDUvMjAyMC4wNC4wMy4wMjI3MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[23].</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.23" data-doi="10.1093/bioinformatics/4.1.181">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jameson</span>
          <span class="cit-name-given-names">BA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wolf</span>
          <span class="cit-name-given-names">H.</span>
         </span>
         <span class="cit-article-title">The antigenic index: a novel algorithm for predicting antigenic determinants</span>. 
         <abbr class="cit-jnl-abbrev">Comput Appl Biosci</abbr>
         <span class="cit-pub-date">1988</span>;
         <span class="cit-vol">4</span>(
         <span class="cit-issue">1</span>):
         <span class="cit-fpage">181</span>–
         <span class="cit-lpage">6</span>. 
         <a href="https://doi.org/10.1093/bioinformatics/4.1.181">https://doi.org/10.1093/bioinformatics/4.1.181</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DJameson%26rft.auinit1%253DB.A.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D181%26rft.epage%253D186%26rft.atitle%253DThe%2Bantigenic%2Bindex%253A%2Ba%2Bnovel%2Balgorithm%2Bfor%2Bpredicting%2Bantigenic%2Bdeterminants%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F4.1.181%26rft_id%253Dinfo%253Apmid%252F2454713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/4.1.181&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=2454713&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.022723.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[24].</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.24" data-doi="10.1002/prot.21078">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Saha</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Raghava</span>
          <span class="cit-name-given-names">GPS</span>
         </span>. 
         <span class="cit-article-title">Prediction of continuous B-cell epitopes in an antigen using recurrent neural network</span>. 
         <abbr class="cit-jnl-abbrev">Proteins</abbr>
         <span class="cit-pub-date">2006</span>;
         <span class="cit-vol">65</span>(
         <span class="cit-issue">1</span>):
         <span class="cit-fpage">40</span>–
         <span class="cit-lpage">8</span>. 
         <a href="https://doi.org/10.1002/prot.21078">https://doi.org/10.1002/prot.21078</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProteins%26rft.stitle%253DProteins%26rft.aulast%253DSaha%26rft.auinit1%253DS.%26rft.volume%253D65%26rft.issue%253D1%26rft.spage%253D40%26rft.epage%253D48%26rft.atitle%253DPrediction%2Bof%2Bcontinuous%2BB-cell%2Bepitopes%2Bin%2Ban%2Bantigen%2Busing%2Brecurrent%2Bneural%2Bnetwork.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fprot.21078%26rft_id%253Dinfo%253Apmid%252F16894596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/prot.21078&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16894596&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.022723.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000240260100005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[25].</span>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.25" data-doi="10.1186/1745-7580-2-2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Larsen</span>
          <span class="cit-name-given-names">JEP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lund</span>
          <span class="cit-name-given-names">O</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nielsen</span>
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-article-title">Improved method for predicting linear B-cell epitopes</span>. 
         <abbr class="cit-jnl-abbrev">Immunome Res</abbr>
         <span class="cit-pub-date">2006</span>;
         <span class="cit-vol">2</span>:
         <span class="cit-fpage">2</span>. 
         <a href="https://doi.org/10.1186/1745-7580-2-2">https://doi.org/10.1186/1745-7580-2-2</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DImmunome%2BRes%26rft.aulast%253DLarsen%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D2%26rft.spage%253D2%26rft.epage%253D2%26rft.atitle%253DImproved%2Bmethod%2Bfor%2Bpredicting%2Blinear%2BB-cell%2Bepitopes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1745-7580-2-2%26rft_id%253Dinfo%253Apmid%252F16635264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1745-7580-2-2&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16635264&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.022723.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[26].</span>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.26">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gupta</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jung</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brunak</span>
          <span class="cit-name-given-names">S.</span>
         </span>
         <abbr class="cit-jnl-abbrev">Prediction of N-glycosylation sites in human proteins</abbr>
         <span class="cit-pub-date">2004</span>;
         <span class="cit-vol">46</span>:
         <span class="cit-fpage">203</span>–
         <span class="cit-lpage">6</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPrediction%2Bof%2BN-glycosylation%2Bsites%2Bin%2Bhuman%2Bproteins%26rft.volume%253D46%26rft.spage%253D203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[27].</span>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.27" data-doi="10.1093/nar/gkw408">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ashkenazy</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Abadi</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Martz</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chay</span>
          <span class="cit-name-given-names">O</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mayrose</span>
          <span class="cit-name-given-names">I</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pupko</span>
          <span class="cit-name-given-names">T</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">ConSurf 2016: an improved methodology to estimate and visualize evolutionary conservation in macromolecules</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-pub-date">2016</span>;
         <span class="cit-vol">44</span>(
         <span class="cit-issue">W1</span>):
         <span class="cit-fpage">W344</span>–
         <span class="cit-lpage">50</span>. 
         <a href="https://doi.org/10.1093/nar/gkw408">https://doi.org/10.1093/nar/gkw408</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw408%26rft_id%253Dinfo%253Apmid%252F27166375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkw408&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27166375&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.022723.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[28].</span>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.28" data-doi="10.1101/gr.849004">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Crooks</span>
          <span class="cit-name-given-names">GE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hon</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chandonia</span>
          <span class="cit-name-given-names">J-M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Brenner</span>
          <span class="cit-name-given-names">SE</span>
         </span>. 
         <span class="cit-article-title">WebLogo: a sequence logo generator</span>. 
         <abbr class="cit-jnl-abbrev">Genome Res</abbr>
         <span class="cit-pub-date">2004</span>;
         <span class="cit-vol">14</span>(
         <span class="cit-issue">6</span>):
         <span class="cit-fpage">1188</span>–
         <span class="cit-lpage">90</span>. 
         <a href="https://doi.org/10.1101/gr.849004">https://doi.org/10.1101/gr.849004</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DCrooks%26rft.auinit1%253DG.%2BE.%26rft.volume%253D14%26rft.issue%253D6%26rft.spage%253D1188%26rft.epage%253D1190%26rft.atitle%253DWebLogo%253A%2BA%2BSequence%2BLogo%2BGenerator%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.849004%26rft_id%253Dinfo%253Apmid%252F15173120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE0LzYvMTE4OCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA1LzIwMjAuMDQuMDMuMDIyNzIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[29].</span>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.03.022723v1.29" data-doi="10.1093/nar/18.20.6097">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schneider</span>
          <span class="cit-name-given-names">TD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stephens</span>
          <span class="cit-name-given-names">RM</span>
         </span>. 
         <span class="cit-article-title">Sequence logos: a new way to display consensus sequences</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-pub-date">1990</span>;
         <span class="cit-vol">18</span>(
         <span class="cit-issue">20</span>):
         <span class="cit-fpage">6097</span>–
         <span class="cit-lpage">100</span>. 
         <a href="https://doi.org/10.1093/nar/18.20.6097">https://doi.org/10.1093/nar/18.20.6097</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSchneider%26rft.auinit1%253DT.%2BD.%26rft.volume%253D18%26rft.issue%253D20%26rft.spage%253D6097%26rft.epage%253D6100%26rft.atitle%253DSequence%2Blogos%253A%2Ba%2Bnew%2Bway%2Bto%2Bdisplay%2Bconsensus%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F18.20.6097%26rft_id%253Dinfo%253Apmid%252F2172928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/18.20.6097&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=2172928&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F05%2F2020.04.03.022723.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1990EF12100024&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
