<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/12/13/2019.12.13.876037.atom" />
<meta name="HW.pisa" content="biorxiv;2019.12.13.876037v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Novel splicing and open reading frames revealed by long-read direct RNA sequencing of adenovirus transcripts" />
<meta name="DC.Identifier" content="10.1101/2019.12.13.876037" />
<meta name="DC.Date" content="2019-12-13" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Adenovirus is a common human pathogen that relies on host cell processes for production and processing of viral RNA. Although adenoviral promoters, splice junctions, and cleavage and polyadenylation sites have been characterized using low-throughput biochemical techniques or short read cDNA-based sequencing, these technologies do not fully capture the complexity of the adenoviral transcriptome. By combining Illumina short-read and nanopore long-read direct RNA sequencing approaches, we mapped transcription start sites and cleavage and polyadenylation sites across the adenovirus genome. The canonical viral early and late RNA cassettes were confirmed, but analysis of splice junctions within long RNA reads revealed an additional 20 novel viral transcripts. These RNAs include seven new splice junctions which lead to expression of canonical open reading frames (ORF), as well as 13 transcripts encoding for messages that potentially alter protein functions through truncations or the fusion of canonical ORFs. In addition, we also detect RNAs that bypass canonical cleavage sites and generate potential chimeric proteins by linking separate gene transcription units. Our work highlights how long-read sequencing technologies can reveal further complexity within viral transcriptomes." />
<meta name="DC.Contributor" content="Alexander M. Price" />
<meta name="DC.Contributor" content="Katharina E. Hayer" />
<meta name="DC.Contributor" content="Daniel P. Depledge" />
<meta name="DC.Contributor" content="Angus C. Wilson" />
<meta name="DC.Contributor" content="Matthew D. Weitzman" />
<meta name="article:published_time" content="2019-12-13" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Novel splicing and open reading frames revealed by long-read direct RNA sequencing of adenovirus transcripts" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Adenovirus is a common human pathogen that relies on host cell processes for production and processing of viral RNA. Although adenoviral promoters, splice junctions, and cleavage and polyadenylation sites have been characterized using low-throughput biochemical techniques or short read cDNA-based sequencing, these technologies do not fully capture the complexity of the adenoviral transcriptome. By combining Illumina short-read and nanopore long-read direct RNA sequencing approaches, we mapped transcription start sites and cleavage and polyadenylation sites across the adenovirus genome. The canonical viral early and late RNA cassettes were confirmed, but analysis of splice junctions within long RNA reads revealed an additional 20 novel viral transcripts. These RNAs include seven new splice junctions which lead to expression of canonical open reading frames (ORF), as well as 13 transcripts encoding for messages that potentially alter protein functions through truncations or the fusion of canonical ORFs. In addition, we also detect RNAs that bypass canonical cleavage sites and generate potential chimeric proteins by linking separate gene transcription units. Our work highlights how long-read sequencing technologies can reveal further complexity within viral transcriptomes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;2019.12.13.876037v1" />
<meta name="citation_id" content="2019.12.13.876037v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2019.12.13.876037v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.13.876037v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.13.876037v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.876037.full.pdf" />
<meta name="citation_doi" content="10.1101/2019.12.13.876037" />
<meta name="citation_num_pages" content="25" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2019.12.13.876037" />
<meta name="citation_author" content="Alexander M. Price" />
<meta name="citation_author_institution" content="Division of Protective Immunity, Department of Pathology and Laboratory Medicine, The Children’s Hospital of Philadelphia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4654-4604" />
<meta name="citation_author" content="Katharina E. Hayer" />
<meta name="citation_author_institution" content="Department of Biomedical and Health Informatics, The Children’s Hospital of Philadelphia" />
<meta name="citation_author" content="Daniel P. Depledge" />
<meta name="citation_author_institution" content="Department of Medicine, New York University School of Medicine" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4292-0599" />
<meta name="citation_author" content="Angus C. Wilson" />
<meta name="citation_author_institution" content="Department of Microbiology, New York University School of Medicine" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5016-4164" />
<meta name="citation_author" content="Matthew D. Weitzman" />
<meta name="citation_author_institution" content="Division of Protective Immunity, Department of Pathology and Laboratory Medicine, The Children’s Hospital of Philadelphia" />
<meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine" />
<meta name="citation_author_email" content="weitzmanm@email.chop.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9713-167X" />
<meta name="citation_reference" content="citation_title=Fields Virology;citation_pages=1704-1731;citation_year=2013" />
<meta name="citation_reference" content="Khanal S, Ghimire P, Dhamoon AS. The Repertoire of Adenovirus in Human Disease: The Innocuous to the Deadly. Biomedicines. 2018; 6. doi:10.3390/biomedicines6010030" />
<meta name="citation_reference" content="Chroboczek J, Bieber F, Jacrot B. The sequence of the genome of adenovirus type 5 and its comparison with the genome of adenovirus type 2. Virology. 1992;186: 280–285. doi:10.1016/0042-6822(92)90082-z" />
<meta name="citation_reference" content="Davison AJ, Benko M, Harrach B. Genetic content and evolution of adenoviruses. J Gen Virol. 2003;84: 2895–2908. doi:10.1099/vir.0.19497-0" />
<meta name="citation_reference" content="Berk AJ. Recent lessons in gene expression, cell cycle control, and cell biology from adenovirus. Oncogene. 2005;24: 7673–85. doi:10.1038/sj.onc.1209040" />
<meta name="citation_reference" content="Chow LT, Gelinas RE, Broker TR, Roberts RJ. An amazing sequence arrangement at the 5’ ends of adenovirus 2 messenger RNA. Cell. 1977;12: 1–8." />
<meta name="citation_reference" content="Berget SM, Moore C, Sharp PA. Spliced segments at the 5’ terminus of adenovirus 2 late mRNA. PNAS. 1977;74: 3171–3175. doi:10.1073/pnas.74.8.3171" />
<meta name="citation_reference" content="Sommer S, Salditt-Georgieff M, Bachenheimer S, Darnell JE, Furuichi Y, Morgan M, et al. The methylation of adenovirus-specific nuclear and cytoplasmic RNA. Nucleic Acids Res. 1976;3: 749–65." />
<meta name="citation_reference" content="Philipson L, Wall R, Glickman G, Darnell JE. Addition of polyadenylate sequences to virus-specific RNA during adenovirus replication. Proc Natl Acad Sci U S A. 1971;68: 2806–9." />
<meta name="citation_reference" content="Berk AJ. Discovery of RNA splicing and genes in pieces. Proc Natl Acad Sci USA. 2016;113: 801–805. doi:10.1073/pnas.1525084113" />
<meta name="citation_reference" content="Montell C, Fisher EF, Caruthers MH, Berk AJ. Resolving the functions of overlapping viral genes by site-specific mutagenesis at a mRNA splice site. Nature. 1982;295: 380–384. doi:10.1038/295380a0" />
<meta name="citation_reference" content="Winberg G, Shenk T. Dissection of overlapping functions within the adenovirus type 5 E1A gene. EMBO J. 1984;3: 1907–1912." />
<meta name="citation_reference" content="Fonseca GJ, Thillainadesan G, Yousef AF, Ablack JN, Mossman KL, Torchia J, et al. Adenovirus evasion of interferon-mediated innate immunity by direct antagonism of a cellular histone posttranslational modification. Cell Host Microbe. 2012; 11: 597–606. doi:10.1016/j.chom.2012.05.005" />
<meta name="citation_reference" content="Zemke NR, Berk AJ. The Adenovirus E1A C Terminus Suppresses a Delayed Antiviral Response and Modulates RAS Signaling. Cell Host &amp; Microbe. 2017;22: 789–800.e5. doi:10.1016/j.chom.2017.11.008" />
<meta name="citation_reference" content="Pelka P, Ablack JNG, Fonseca GJ, Yousef AF, Mymryk JS. Intrinsic Structural Disorder in Adenovirus E1A: a Viral Molecular Hub Linking Multiple Diverse Processes. Journal of Virology. 2008;82: 7252–7263. doi:10.1128/JVI.00104-08" />
<meta name="citation_reference" content="Blackford AN, Grand RJA. Adenovirus E1B 55-Kilodalton Protein: Multiple Roles in Viral Infection and Cell Transformation. Journal of Virology. 2009;83: 4000–4012. doi:10.1128/JVI.02417-08" />
<meta name="citation_reference" content="Han J, Sabbatini P, Perez D, Rao L, Modha D, White E. The E1B 19K protein blocks apoptosis by interacting with and inhibiting the p53-inducible and death-promoting Bax protein. Genes Dev. 1996;10: 461–477. doi:10.1101/gad.10.4.461" />
<meta name="citation_reference" content="Yew PR, Berk AJ. Inhibition of p53 transactivation required for transformation by adenovirus early 1B protein. Nature. 1992;357: 82–85. doi:10.1038/357082a0" />
<meta name="citation_reference" content="Bridge E, Ketner G. Interaction of adenoviral E4 and E1b products in late gene expression. Virology. 1990;174: 345–53." />
<meta name="citation_reference" content="Cathomen T, Weitzman MD. A functional complex of adenovirus proteins E1B-55kDa and E4orf6 is necessary to modulate the expression level of p53 but not its transcriptional activity. J Virol. 2000;74: 11407–12." />
<meta name="citation_reference" content="Harada JN, Shevchenko A, Shevchenko A, Pallas DC, Berk AJ. Analysis of the adenovirus E1B-55K-anchored proteome reveals its link to ubiquitination machinery. J Virol. 2002;76: 9194–206." />
<meta name="citation_reference" content="Querido E, Blanchette P, Yan Q, Kamura T, Morrison M, Boivin D, et al. Degradation of p53 by adenovirus E4orf6 and E1B55K proteins occurs via a novel mechanism involving a Cullin-containing complex. Genes Dev. 2001;15: 3104–17. doi:10.1101/gad.926401" />
<meta name="citation_reference" content="Dybas JM, Herrmann C, Weitzman MD. Ubiquitination at the interface of tumor viruses and DNA damage responses. Curr Opin Virol. 2018;32: 40–47. doi:10.1016/j.coviro.2018.08.017" />
<meta name="citation_reference" content="Winnacker EL. Adenovirus DNA: structure and function of a novel replicon. Cell. 1978;14: 761–773. doi:10.1016/0092-8674(78)90332-x" />
<meta name="citation_reference" content="Webster A, Leith IR, Nicholson J, Hounsell J, Hay RT. Role of preterminal protein processing in adenovirus replication. J Virol. 1997;71: 6381–6389." />
<meta name="citation_reference" content="Brenkman AB, Breure EC, van der Vliet PC. Molecular architecture of adenovirus DNA polymerase and location of the protein primer. J Virol. 2002;76: 8200–8207. doi:10.1128/jvi.76.16.8200-8207.2002" />
<meta name="citation_reference" content="de Jong RN, van der Vliet PC, Brenkman AB. Adenovirus DNA replication: protein priming, jumping back and the role of the DNA binding protein DBP. Curr Top Microbiol Immunol. 2003;272: 187–211. doi:10.1007/978-3-662-05597-7_7" />
<meta name="citation_reference" content="Robinson CM, Rajaiya J, Zhou X, Singh G, Dyer DW, Chodosh J. The E3 CR1-gamma gene in human adenoviruses associated with epidemic keratoconjunctivitis. Virus Res. 2011;160: 120–127. doi:10.1016/j.virusres.2011.05.022" />
<meta name="citation_reference" content="Singh G, Robinson CM, Dehghan S, Jones MS, Dyer DW, Seto D, et al. Homologous recombination in E3 genes of human adenovirus species D. J Virol. 2013;87: 12481–12488. doi:10.1128/JVI.01927-13" />
<meta name="citation_reference" content="citation_title=The Molecular Repertoire of Adenoviruses I: Virion Structure and Infection;citation_author=WSM. Wold;citation_author=AE. Tollefson;citation_author=TW. Hermiston;citation_year=1995;citation_doi=10.1007/978-3-642-79496-4_13" />
<meta name="citation_reference" content="Bridge E, Ketner G. Redundant control of adenovirus late gene expression by early region 4. J Virol. 1989;63: 631–8." />
<meta name="citation_reference" content="Weitzman MD. Functions of the adenovirus E4 proteins and their impact on viral vectors. Front Biosci. 2005;10: 1106–17." />
<meta name="citation_reference" content="Weitzman MD, Ornelles DA. Inactivating intracellular antiviral responses during adenovirus infection. Oncogene. 2005;24: 7686–96. doi:10.1038/sj.onc.1209063" />
<meta name="citation_reference" content="Weinmann R, Raskas HJ, Roeder RG. Role of DNA-dependent RNA polymerases II and III in transcription of the adenovirus genome late in productive infection. Proc Natl Acad Sci USA. 1974;71: 3426–3439. doi:10.1073/pnas.71.9.3426" />
<meta name="citation_reference" content="Vachon VK, Conn GL. Adenovirus VA RNA: An essential pro-viral non-coding RNA. Virus Res. 2016;212: 39–52. doi:10.1016/j.virusres.2015.06.018" />
<meta name="citation_reference" content="Hoeben RC, Uil TG. Adenovirus DNA Replication. Cold Spring Harb Perspect Biol. 2013;5. doi:10.1101/cshperspect.a013003" />
<meta name="citation_reference" content="Shaw AR, Ziff EB. Transcripts from the adenovirus-2 major late promoter yield a single early family of 3’ coterminal mRNAs and five late families. Cell. 1980;22: 905–916. doi:10.1016/0092-8674(80)90568-1" />
<meta name="citation_reference" content="Parks RJ. Adenovirus protein IX: a new look at an old protein. Mol Ther. 2005;11: 19–25. doi:10.1016/j.ymthe.2004.09.018" />
<meta name="citation_reference" content="Zhang W, Imperiale MJ. Requirement of the adenovirus IVa2 protein for virus assembly. J Virol. 2003;77: 3586–3594. doi:10.1128/jvi.77.6.3586-3594.2003" />
<meta name="citation_reference" content="Carter TH, Ginsberg HS. Viral transcription in KB cells infected by temperature-sensitive “early” mutants of adenovirus type 5. J Virol. 1976;18: 156–166." />
<meta name="citation_reference" content="Crossland LD, Raskas HJ. Identification of adenovirus genes that require template replication for expression. J Virol. 1983;46: 737–748." />
<meta name="citation_reference" content="Thomas GP, Mathews MB. DNA replication and the early to late transition in adenovirus infection. Cell. 1980;22: 523–533. doi:10.1016/0092-8674(80)90362-1" />
<meta name="citation_reference" content="Ramke M, Lee JY, Dyer DW, Seto D, Rajaiya J, Chodosh J. The 5’UTR in human adenoviruses: leader diversity in late gene expression. Sci Rep. 2017;7: 618. doi:10.1038/s41598-017-00747-y" />
<meta name="citation_reference" content="Soloway PD, Shenk T. The adenovirus type 5 i-leader open reading frame functions in cis to reduce the half-life of L1 mRNAs. J Virol. 1990;64: 551–558." />
<meta name="citation_reference" content="Morris SJ, Scott GE, Leppard KN. Adenovirus Late-Phase Infection Is Controlled by a Novel L4 Promoter. J Virol. 2010;84: 7096–7104. doi:10.1128/JVI.00107-10" />
<meta name="citation_reference" content="Biasiotto R, Akusjärvi G. Regulation of Human Adenovirus Alternative RNA Splicing by the Adenoviral L4-33K and L4-22K Proteins. Int J Mol Sci. 2015;16: 2893–2912. doi:10.3390/ijms16022893" />
<meta name="citation_reference" content="Tollefson AE, Ying B, Doronin K, Sidor PD, Wold WSM. Identification of a New Human Adenovirus Protein Encoded by a Novel Late l-Strand Transcription Unit. Journal of Virology. 2007;81: 12918–12926. doi:10.1128/JVI.01531-07" />
<meta name="citation_reference" content="Ying B, Tollefson AE, Wold WSM. Identification of a Previously Unrecognized Promoter That Drives Expression of the UXP Transcription Unit in the Human Adenovirus Type 5 Genome. J Virol. 2010;84: 11470–11478. doi:10.1128/JVI.01338-10" />
<meta name="citation_reference" content="Miller DL, Myers CL, Rickards B, Coller HA, Flint SJ. Adenovirus type 5 exerts genome-wide control over cellular programs governing proliferation, quiescence, and survival. Genome Biology. 2007;8: R58. doi:10.1186/gb-2007-8-4-r58" />
<meta name="citation_reference" content="Miller DL, Rickards B, Mashiba M, Huang W, Flint SJ. The adenoviral E1B 55-kilodalton protein controls expression of immune response genes but not p53-dependent transcription. J Virol. 2009;83: 3591–3603. doi:10.1128/JVI.02269-08" />
<meta name="citation_reference" content="Zhao H, Dahlö M, Isaksson A, Syvänen A-C, Pettersson U. The transcriptome of the adenovirus infected cell. Virology. 2012;424: 115–128. doi:10.1016/j.virol.2011.12.006" />
<meta name="citation_reference" content="Zhao H, Chen M, Valdés A, Lind SB, Pettersson U. Transcriptomic and proteomic analyses reveal new insights into the regulation of immune pathways during adenovirus type 2 infection. BMC Microbiol. 2019;19: 15. doi:10.1186/s12866-018-1375-5" />
<meta name="citation_reference" content="Zhao H, Chen M, Pettersson U. A new look at adenovirus splicing. Virology. 2014;456–457: 329–341. doi:10.1016/j.virol.2014.04.006" />
<meta name="citation_reference" content="Crisostomo L, Soriano AM, Mendez M, Graves D, Pelka P. Temporal dynamics of adenovirus 5 gene expression in normal human cells. PLOS ONE. 2019;14: e0211192. doi:10.1371/journal.pone.0211192" />
<meta name="citation_reference" content="Brandes N, Linial M. Gene overlapping and size constraints in the viral world. Biology Direct. 2016;11: 26. doi:10.1186/s13062-016-0128-3" />
<meta name="citation_reference" content="O’Grady T, Wang X, Höner zu Bentrup K, Baddoo M, Concha M, Flemington EK. Global transcript structure resolution of high gene density genomes through multi-platform data integration. Nucleic Acids Res. 2016;44: e145–e145. doi:10.1093/nar/gkw629" />
<meta name="citation_reference" content="Depledge DP, Srinivas KP, Sadaoka T, Bready D, Mori Y, Placantonakis DG, et al. Direct RNA sequencing on nanopore arrays redefines the transcriptional complexity of a viral pathogen. Nat Commun. 2019;10: 1–13. doi:10.1038/s41467-019-08734-9" />
<meta name="citation_reference" content="Tombácz D, Balázs Z, Csabai Z, Moldován N, Szűcs A, Sharon D, et al. Characterization of the Dynamic Transcriptome of a Herpesvirus with Long-read Single Molecule Real-Time Sequencing. Sci Rep. 2017;7: 43751. doi:10.1038/srep43751" />
<meta name="citation_reference" content="Viehweger A, Krautwurst S, Lamkiewicz K, Madhugiri R, Ziebuhr J, Hölzer M, et al. Direct RNA nanopore sequencing of full-length coronavirus genomes provides novel insights into structural variants and enables modification analysis. Genome Res. 2019;29: 1545–1554. doi:10.1101/gr.247064.118" />
<meta name="citation_reference" content="Weirather JL, de Cesare M, Wang Y, Piazza P, Sebastiano V, Wang X-J, et al. Comprehensive comparison of Pacific Biosciences and Oxford Nanopore Technologies and their applications to transcriptome analysis. F1000Res. 2017;6: 100. doi:10.12688/f1000research.10571.2" />
<meta name="citation_reference" content="Danecek P, Auton A, Abecasis G, Albers CA, Banks E, DePristo MA, et al. The variant call format and VCFtools. Bioinformatics. 2011;27: 2156–2158. doi:10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="Narasimhan V, Danecek P, Scally A, Xue Y, Tyler-Smith C, Durbin R. BCFtools/RoH: a hidden Markov model approach for detecting autozygosity from next-generation sequencing data. Bioinformatics. 2016;32: 1749–1751. doi:10.1093/bioinformatics/btw044" />
<meta name="citation_reference" content="Li X, Xiong X, Yi C. Epitranscriptome sequencing technologies: decoding RNA modifications. Nat Methods. 2017;14: 23–31. doi:10.1038/nmeth.4110" />
<meta name="citation_reference" content="Garalde DR, Snell EA, Jachimowicz D, Sipos B, Lloyd JH, Bruce M, et al. Highly parallel direct RNA sequencing on an array of nanopores. Nat Methods. 2018;15: 201–206. doi:10.1038/nmeth.4577" />
<meta name="citation_reference" content="Wu TD, Reeder J, Lawrence M, Becker G, Brauer MJ. GMAP and GSNAP for Genomic Sequence Alignment: Enhancements to Speed, Accuracy, and Functionality. Methods Mol Biol. 2016;1418: 283–334. doi:10.1007/978-1-4939-3578-9_15" />
<meta name="citation_reference" content="Li H. Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics. 2018;34: 3094–3100. doi:10.1093/bioinformatics/bty191" />
<meta name="citation_reference" content="Bonfert T, Friedel CC. Prediction of Poly(A) Sites by Poly(A) Read Mapping. PLoS ONE. 2017;12: e0170914. doi:10.1371/journal.pone.0170914" />
<meta name="citation_reference" content="Borodulina OR, Kramerov DA. Transcripts synthesized by RNA polymerase III can be polyadenylated in an AAUAAA-dependent manner. RNA. 2008;14: 1865–1873. doi:10.1261/rna.1006608" />
<meta name="citation_reference" content="Tang AD, Soulette CM, Baren MJ van, Hart K, Hrabeta-Robinson E, Wu CJ, et al. Full-length transcript characterization of SF3B1 mutation in chronic lymphocytic leukemia reveals downregulation of retained introns. bioRxiv. 2018; 410183. doi:10.1101/410183" />
<meta name="citation_reference" content="Hidalgo P, Anzures L, Hernández-Mendoza A, Guerrero A, Wood CD, Valdés M, et al. Morphological, Biochemical, and Functional Study of Viral Replication Compartments Isolated from Adenovirus-Infected Cells. Journal of Virology. 2016;90: 3411–3427. doi:10.1128/JVI.00033-16" />
<meta name="citation_reference" content="Stern-Ginossar N, Weisburd B, Michalski A, Le VTK, Hein MY, Huang S-X, et al. Decoding human cytomegalovirus. Science. 2012;338: 1088–1093. doi:10.1126/science.1227919" />
<meta name="citation_reference" content="Arvey A, Tempera I, Tsai K, Chen H-S, Tikhmyanova N, Klichinsky M, et al. An Atlas of the Epstein-Barr Virus Transcriptome and Epigenome Reveals Host-Virus Regulatory Interactions. Cell Host Microbe. 2012;12: 233–245. doi:10.1016/j.chom.2012.06.008" />
<meta name="citation_reference" content="Arias C, Weisburd B, Stern-Ginossar N, Mercier A, Madrid AS, Bellare P, et al. KSHV 2.0: A Comprehensive Annotation of the Kaposi’s Sarcoma-Associated Herpesvirus Genome Using Next-Generation Sequencing Reveals Novel Genomic and Functional Features. PLOS Pathogens. 2014;10: e1003847. doi:10.1371/journal.ppat.1003847" />
<meta name="citation_reference" content="Rutkowski AJ, Erhard F, L’Hernault A, Bonfert T, Schilhabel M, Crump C, et al. Widespread disruption of host transcription termination in HSV-1 infection. Nat Commun. 2015;6: 7126. doi:10.1038/ncomms8126" />
<meta name="citation_reference" content="Garren SB, Kondaveeti Y, Duff MO, Carmichael GG. Global Analysis of Mouse Polyomavirus Infection Reveals Dynamic Regulation of Viral and Host Gene Expression and Promiscuous Viral RNA Editing. PLoS Pathog. 2015;11: e1005166. doi:10.1371/journal.ppat.1005166" />
<meta name="citation_reference" content="Evans VC, Barker G, Heesom KJ, Fan J, Bessant C, Matthews DA. De novo derivation of proteomes from transcriptomes for transcript and protein identification. Nat Methods. 2012;9: 1207–1211. doi:10.1038/nmeth.2227" />
<meta name="citation_reference" content="Wright J, Leppard KN. The Human Adenovirus 5 L4 Promoter Is Activated by Cellular Stress Response Protein p53. J Virol. 2013;87: 11617–11625. doi:10.1128/JVI.01924-13" />
<meta name="citation_reference" content="Wright J, Atwan Z, Morris SJ, Leppard KN. The Human Adenovirus Type 5 L4 Promoter Is Negatively Regulated by TFII-I and L4-33K. Journal of Virology. 2015;89: 7053–7063. doi:10.1128/JVI.00683-15" />
<meta name="citation_reference" content="Hennig T, Michalski M, Rutkowski AJ, Djakovic L, Whisnant AW, Friedl MS, et al. HSV-1-induced disruption of transcription termination resembles a cellular stress response but selectively increases chromatin accessibility downstream of genes. PLoS Pathog. 2018;14: e1006954. doi:10.1371/journal.ppat.1006954" />
<meta name="citation_reference" content="Ramírez F, Ryan DP, Grüning B, Bhardwaj V, Kilpert F, Richter AS, et al. deepTools2: a next generation web server for deep-sequencing data analysis. Nucleic Acids Res. 2016;44: W160–165. doi:10.1093/nar/gkw257" />
<meta name="citation_reference" content="Feng Y-Y, Ramu A, Cotto KC, Skidmore ZL, Kunisaki J, Conrad DF, et al. RegTools: Integrated analysis of genomic and transcriptomic data for discovery of splicing variants in cancer. bioRxiv. 2018; 436634. doi:10.1101/436634" />
<meta name="citation_reference" content="Robinson JT, Thorvaldsdóttir H, Winckler W, Guttman M, Lander ES, Getz G, et al. Integrative Genomics Viewer. Nat Biotechnol. 2011;29: 24–26. doi:10.1038/nbt.1754" />
<meta name="citation_reference" content="Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, et al. The Sequence Alignment/Map format and SAMtools. Bioinformatics. 2009;25: 2078–2079. doi:10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="Quinlan AR, Hall IM. BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics. 2010;26: 841–842. doi:10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="Heinz S, Benner C, Spann N, Bertolino E, Lin YC, Laslo P, et al. Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities. Mol Cell. 2010;38: 576–589. doi:10.1016/j.molcel.2010.05.004" />
<meta name="twitter:title" content="Novel splicing and open reading frames revealed by long-read direct RNA sequencing of adenovirus transcripts" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Adenovirus is a common human pathogen that relies on host cell processes for production and processing of viral RNA. Although adenoviral promoters, splice junctions, and cleavage and polyadenylation sites have been characterized using low-throughput biochemical techniques or short read cDNA-based sequencing, these technologies do not fully capture the complexity of the adenoviral transcriptome. By combining Illumina short-read and nanopore long-read direct RNA sequencing approaches, we mapped transcription start sites and cleavage and polyadenylation sites across the adenovirus genome. The canonical viral early and late RNA cassettes were confirmed, but analysis of splice junctions within long RNA reads revealed an additional 20 novel viral transcripts. These RNAs include seven new splice junctions which lead to expression of canonical open reading frames (ORF), as well as 13 transcripts encoding for messages that potentially alter protein functions through truncations or the fusion of canonical ORFs. In addition, we also detect RNAs that bypass canonical cleavage sites and generate potential chimeric proteins by linking separate gene transcription units. Our work highlights how long-read sequencing technologies can reveal further complexity within viral transcriptomes." />
<meta name="og-title" property="og:title" content="Novel splicing and open reading frames revealed by long-read direct RNA sequencing of adenovirus transcripts" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2019.12.13.876037v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Adenovirus is a common human pathogen that relies on host cell processes for production and processing of viral RNA. Although adenoviral promoters, splice junctions, and cleavage and polyadenylation sites have been characterized using low-throughput biochemical techniques or short read cDNA-based sequencing, these technologies do not fully capture the complexity of the adenoviral transcriptome. By combining Illumina short-read and nanopore long-read direct RNA sequencing approaches, we mapped transcription start sites and cleavage and polyadenylation sites across the adenovirus genome. The canonical viral early and late RNA cassettes were confirmed, but analysis of splice junctions within long RNA reads revealed an additional 20 novel viral transcripts. These RNAs include seven new splice junctions which lead to expression of canonical open reading frames (ORF), as well as 13 transcripts encoding for messages that potentially alter protein functions through truncations or the fusion of canonical ORFs. In addition, we also detect RNAs that bypass canonical cleavage sites and generate potential chimeric proteins by linking separate gene transcription units. Our work highlights how long-read sequencing technologies can reveal further complexity within viral transcriptomes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-12-13" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2019.12.13.876037v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2019.12.13.876037v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2019.12.13.876037v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2019.12.13.876037v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1036558" />
    <title>Novel splicing and open reading frames revealed by long-read direct RNA sequencing of adenovirus transcripts | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Gbdxcyh0arN_tXXGhQqVlnP5p91PhMr7q5FuKLQ4pX8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1036558","apath":"\/biorxiv\/early\/2019\/12\/13\/2019.12.13.876037.atom","pisa":"biorxiv;2019.12.13.876037v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2019.12.13.876037v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2019.12.13.876037v1","title":"Novel splicing and open reading frames revealed by long-read direct RNA sequencing of adenovirus transcripts","identifier":"node\/1036558"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nid4ayx6xciGr\\\\\\\/noITMBNB8AIDMsyBzxn85UMzbqzsCCPSyugpk85nm88oVFtmdSJKT\\\\\\\/KsMdncaU4cTGd9+wseAGEQb9W4Ukl\\\\\\\/vcBK7DuhovM5aNhEL9Ig1XoCS7XrCuQu2W9SK1ObEKukywH1krAz2N1Cq9l9k6Dl5EDDll0esomLRqtycFjLZAxBE2Q6CH\\\\\\\/51A1VLRTd7Hsv7yl\\\\\\\/+8p12ISanmI1fMfieVCUv2uF7r0d4utz7ms5hbvMqt2KyxgfuVMcrm9R7THHeAijqJgF0I3m3tN6r1sARV3\\\\\\\/gzoSUceKfej3Y4gG2wXugKxOAbCnDHDq0J2MWK7n5lj+1aWP8AZZsM0uI0K2\\\\\\\/kT+xcZUvXrKjJih\\\\\\\/E+tGnmhoZXaAx7f8d63MBdhGyHOBHyvxfTJvn\\\\\\\/zlyj39axjhtXDOOnyRndMD2KWO+zTkFHrCwt6pLHY2dqpqwF7+nMX9dn3o7eVH8fR21DISsyAzbdnFsKzAWwxLovxHIBHXFUKxpYafgmr1b8Z6bJT375SH7tfnA56iuyEHAIEAtCNIqrDshVGZifSo\\\\\\\/nAeYUfzsv1D1SsgGbXmo0hXdvWVwX5tIMxBY+Fme+itlhUMwmGZY59a\\\\\\\/nDB6w1rG17\\\\\\\/37N\\\\\\\/VLbB5S3cisDw7xWStQXRjVnBCnZHsFRz1OXHTb31LrCzGImOaBty\\\\\\\/9QMhoGieFRqemw0putDh83QFoPgMNUcBHId25x52c2f0CCrZouY7liLlOM8o3JZbmrLVh1GWte7Pja+rQfFn8QfbiVITGIbfgJGXuA+3u2XhhoiLKLjQ1+mrFyYHqYv1U2rtR0oONPSMKly8AzqZghMly2QtAQ3lfSk0veOB8OHwMMJoMqbgD5Cgg2koK6kJ5nAxINMSlIsCBDOrmyZFAT5y3OS6WBFq728Lho\\\\\\\/u2n6sh5sMZNh\\\\\\\/o6xesAFlaOSojtErEI6n4qYrwj0\\\\\\\/9OrZqZWC1NVeUbouPSoJO\\\\\\\/igcyz8aDQDJDrmj5tYVoUz7JENP0MAbGn79C5O8pczqtfsR7uNX92K6wUK4seTXpa3kjpnolJpbQIpuLcot7pTKmMKbtHcld5BpbGxgUL69vfcve2b1iKJN0eLAKNt7xT4zDTOfG9aEkMG79OeUHhAalwG2rWE+Ltczhw1+H3rGPrt5KIMuiadH9u\\\\\\\/RWdVQqxMSQZnZcGJyco6LSgavbNtZdt37EwCJr1P3YCr7WB85FCwg+wrr3z65vwBVn8kQ1P9Q+MaR33r96HT4u7GI\\\\\\\/GOyjidEFwdlpfN57im+vnOBkfuRuu+3RbVMCeUx9PagQettHAxrIQESCzMvjXjodcxAelpxOoV8GozHd512j64LSRp\\\u0022,\\\u0022iv\\\u0022:\\\u00229\\\\\\\/fkyeJIbe5flE+gHpaJ8g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022b0526cf54757086d920108122366c973678899c05556554de0ac8758e944c52a\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2019.12.13.876037v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1036558 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2019.12.13.876037v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ULsbsB92jg6GjWFMzAZRXsL88J-Tcgce9N5PN0A9wTA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1036558" id="node1036558" data-pisa="biorxiv;2019.12.13.876037v1" data-pisa-master="biorxiv;2019.12.13.876037" data-apath="/biorxiv/early/2019/12/13/2019.12.13.876037.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Novel splicing and open reading frames revealed by long-read direct RNA sequencing of adenovirus transcripts</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4654-4604" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Alexander M.</span> <span class="nlm-surname">Price</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Katharina E.</span> <span class="nlm-surname">Hayer</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4292-0599" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Daniel P.</span> <span class="nlm-surname">Depledge</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5016-4164" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Angus C.</span> <span class="nlm-surname">Wilson</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-9713-167X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Matthew D.</span> <span class="nlm-surname">Weitzman</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2019.12.13.876037 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1036558" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alexander M. Price </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Protective Immunity, Department of Pathology and Laboratory Medicine, The Children’s Hospital of Philadelphia</span>, Philadelphia, PA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BM.%2BPrice%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Price%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexander%2BM.%2BPrice%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4654-4604" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alexander M. Price</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katharina E. Hayer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Biomedical and Health Informatics, The Children’s Hospital of Philadelphia</span>, Philadelphia, PA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katharina%2BE.%2BHayer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hayer%20KE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatharina%2BE.%2BHayer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniel P. Depledge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Medicine, New York University School of Medicine</span>, New York, NY</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BP.%2BDepledge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Depledge%20DP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BP.%2BDepledge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4292-0599" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel P. Depledge</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Angus C. Wilson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Microbiology, New York University School of Medicine</span>, New York, NY</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angus%2BC.%2BWilson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilson%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAngus%2BC.%2BWilson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5016-4164" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Angus C. Wilson</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Matthew D. Weitzman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Protective Immunity, Department of Pathology and Laboratory Medicine, The Children’s Hospital of Philadelphia</span>, Philadelphia, PA</div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine</span>, Philadelphia, PA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BD.%2BWeitzman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weitzman%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BD.%2BWeitzman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9713-167X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthew D. Weitzman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:weitzmanm@email.chop.edu" class="" data-icon-position="" data-hide-link-title="0">weitzmanm@email.chop.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2019.12.13.876037v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1036558" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1036558/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.13.876037v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1036558" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1036558/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.13.876037v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1036558" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1036558/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.13.876037v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1036558" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1036558/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.13.876037v1.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:1036558" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:1036558/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2019.12.13.876037v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1036558" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1036558/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Adenovirus is a common human pathogen that relies on host cell processes for production and processing of viral RNA. Although adenoviral promoters, splice junctions, and cleavage and polyadenylation sites have been characterized using low-throughput biochemical techniques or short read cDNA-based sequencing, these technologies do not fully capture the complexity of the adenoviral transcriptome. By combining Illumina short-read and nanopore long-read direct RNA sequencing approaches, we mapped transcription start sites and cleavage and polyadenylation sites across the adenovirus genome. The canonical viral early and late RNA cassettes were confirmed, but analysis of splice junctions within long RNA reads revealed an additional 20 novel viral transcripts. These RNAs include seven new splice junctions which lead to expression of canonical open reading frames (ORF), as well as 13 transcripts encoding for messages that potentially alter protein functions through truncations or the fusion of canonical ORFs. In addition, we also detect RNAs that bypass canonical cleavage sites and generate potential chimeric proteins by linking separate gene transcription units. Our work highlights how long-read sequencing technologies can reveal further complexity within viral transcriptomes.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">Adenoviruses (AdV) are common viral pathogens across multiple species with distinct tissue tropisms including gut, eye, and lung [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. Among the human adenoviruses, serotypes 2 (Ad2) and 5 (Ad5) from subgroup C are the most prevalent within the population, and they cause benign to severe respiratory infections [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. These two serotypes are highly homologous, sharing 94.7% nucleotide identity between their genomes and 69.2-100% amino acid identity amongst conserved open reading frames (ORFs) [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. AdVs readily infect most transformed human cell lines and have proven a valuable tool that has led to seminal discoveries in molecular biology for many decades [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. RNA splicing was discovered by the analysis of adenovirus encoded RNAs [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>,<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>], as well as other important findings in messenger RNA capping and polyadenylation [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. It is now understood that essentially all AdV mRNAs are capped, spliced, polyadenylated, and exported from the nucleus using host cell machinery [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>].</p><p id="p-5">AdV are capable of infecting non-dividing cells and reprogramming cellular processes for productive viral infection. This rewiring involves a highly regulated cascade of viral gene expression over various kinetic classes [<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>]. The first viral gene to be expressed after infection is E1A, a multi-functional transcription factor that activates downstream viral transcription, liberates E2F from RB proteins, as well as alters host transcriptional responses to the virus [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>–<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. While all E1A molecules have identical 5’ and 3’ nucleotide sequences, splicing of differently sized internal introns allows for the production of discrete proteins that lack specific functional domains conserved across serotypes [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. Early after infection, E1A is expressed mainly as large and small isoforms, but later in infection alternative splicing leads to the production of a 9 Svedberg E1A isoform (E1A-9s) as well as low abundance doubly-spliced E1A-11s and E1A-10s. The second viral gene to be activated is E1B, consisting of predominantly two spliced isoforms producing 19-kilodalton and 55-kilodalton proteins, with two less abundant isoforms generating putative ORFs of 156 and 93 residues [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. While E1B-19K acts to block cellular apoptosis [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>], E1B-55K is another multifunctional protein that can cooperate with E1A to alter cellular gene expression downstream of p53 as well as form the targeting component of a viral ubiquitin ligase [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>–<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>]. The remaining early transcription units are all transcriptionally activated by E1A and encode for products of related function. The E2 region on the reverse strand of the AdV genome has both an early and a late promoter, as well as two distinct polyadenylation sites, leading to upstream E2A and downstream E2B transcripts [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. E2A encodes for the viral DNA-binding protein (DBP), while alternative splicing to E2B encodes for the protein-priming terminal protein (pTP) as well as the AdV DNA polymerase (AdPol) [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>–<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. The E3 region encoded on the top strand also has two polyadenylation sites leading to E3A and E3B transcription units, and these gene products are primarily involved in modulating the host innate immune system [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>–<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>]. Like E1A, the E4 region on the reverse strand has identical 5’ and 3’ regions, and encodes up to six ORFs by removal of a first intron of varying length. E4 region transcripts encode for multifunctional proteins that are involved in regulation of transcription, splicing, and translation of viral RNAs, as well as antagonizing intrinsic cellular defenses [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>–<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>]. Additionally, AdV encodes two Pol III-derived virus associated (VA) RNAs involved in the inactivation of Protein Kinase acting on RNA (PKR) [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>,<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>]. Ultimately, the concerted efforts of the AdV early proteins lead to a cellular state that allows for the replication and amplification of the viral DNA genome [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>].</p><p id="p-6">Prior to viral DNA replication, the AdV Major Late Promoter (MLP) is thought to be largely silent with small amounts of RNA being made that terminate at the immediately downstream (L1) polyadenylation site [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>]. At this time, so-called intermediate genes pIX and IVa2 begin to be expressed from promoters within the E1B cassette and antisense to the MLP. Both pIX and IVa2 co-terminate at polyadenylation sites within the early genes they overlap with (E1B and E2B, respectively) and are involved in late gene transcription and packaging [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>,<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>]. Only after viral DNA replication has occurred does the MLP fully activate, supporting the hypothesis that that active replication <em>in cis</em> is a prerequisite for full viral late gene expression [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>–<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>]. The Major Late Transcriptional Unit (MLTU) begins with a series of three constitutive exons spliced together to form the tripartite leader, before downstream splicing to late cassettes defined by one of five alternative polyadenylation sites (termed L1-L5) [<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a>]. Splicing within the tripartite leader to the so-called “i” exon leads to a putative ORF upstream of subsequent late gene splicing events and destabilizes these RNA molecules [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>,<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>]. An additional intermediate promoter has been reported within the L4 region that allows for the early expression of L4-22K and L4-33K proteins important for the splicing of other late genes [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>,<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>]. The MLTU encodes for primarily structural capsid components or proteins involved with packaging of new virions, and their expression ultimately leads to the death of the host cell. Recently, a novel late gene, UXP, was discovered on the reverse strand of the genome [<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>,<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>]. The UXP promoter is located between E4 and E2 on the reverse strand of the genome, and splices downstream to the exons within the E2A region to continue translation of an ORF in an alternate reading frame to that of DBP. This exciting finding suggests that our knowledge of AdV transcripts is incomplete, especially within the complex MLTU region.</p><p id="p-7">The Ad5 genome was fully sequenced in 1991 using Sanger sequencing of viral genome fragments inserted into plasmid DNA and amplified in bacteria [<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>]. This genome sequence was then annotated in 2003 based on homology to similar serotypes of AdV [<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>]. As such, the current reference annotation for Ad5 available on the National Center for Biotechnology Information (<span class="underline">AC_000008</span>) is incomplete, and lacks critical information such as transcription start sites (TSS), cleavage and polyadenylation sites (CPAS), and the resulting 5’ and 3’ untranslated regions (UTR) that the aforementioned information dictates. In recent years, new technologies have allowed for high-throughput investigation of gene expression utilizing various techniques. The effect of AdV infection on host gene expression has been shown for Ad5 by microarray analysis [<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>,<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>], as well as for Ad2 by Illumina-based short-read sequencing [<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>,<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>]. Analyses of both single-end and paired-end short-read RNA-seq data from cells infected with Ad2 revealed both temporal viral gene expression and high-depth splicing information and identified both previously confirmed and novel RNA splice site junctions [<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>]. In addition. temporal analysis of Ad5 viral gene expression was performed using digital PCR to determine expression kinetics of a subset of known viral genes [<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>]. Lastly, the late RNA tripartite leader splicing was analyzed by short-read sequencing across a number of human AdV serotypes [<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a>]. To date, no group has performed a comprehensive analysis of the RNAs generated during Ad5 infection. Furthermore, even though the quality and depth of current short-read sequencing technologies is high, the complex nature of many viral transcriptomes precludes the unambiguous mapping of these short reads to any one particular RNA isoform due to extreme gene density and overlapping transcriptional units [<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>,<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>]. In this regard, the ability of long-read RNA sequencing to map full-length transcripts has the potential to revolutionize detection of divergent isoforms and multiply spliced RNA at the single-molecule level [<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>–<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>].</p><p id="p-8">In this study, we have re-annotated the Ad5 genome and transcriptome using a combination of short-read and long-read RNA sequencing technologies. The high read depth and accuracy of base-calling achieved by Illumina-based short-read sequencing allowed for both the detection of single nucleotide polymorphisms within transcriptionally active regions of the viral DNA genome, as well as error-correction of the inherently noisier base-calling of Nanopore-based long-read direct RNA sequencing (dRNA-seq). dRNA-seq enabled the detection of full-length RNA transcripts and the assignment of TSS and CPAS transcriptome-wide. Furthermore, by combining highly accurate splice site junctions from short-read sequencing and full-length isoform context from long-read sequencing, we were able to reevaluate the splicing complexity of AdV transcriptional units. Using this integrated approach, we have discovered 20 additional viral polyadenylated RNAs for a total of 75 unique mRNAs produced by Ad5. Of these novel isoforms, seven RNAs encode for a canonical ORF with changes in upstream or downstream splicing. The remaining 13 encode new ORFs or alter existing ORFs by internal truncations or in-frame fusion of genes from separate transcription units. Taken together, our data reveal additional transcriptional complexity of AdV and highlight the necessity of revisiting transcriptome annotations following the emergence of appropriate new technologies.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>RNA-seq reveals high-confidence SNPs within the Ad5 genome</h3><p id="p-9">Illumina-based RNA sequencing (RNA-seq) relies on the fractionation of RNA molecules before reverse transcription into complementary DNA, and therefore loses information such as RNA modifications and the context of splice junctions within full length molecules. However, the accuracy of each individual base call is very high [<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>]. Using bcftools, a common variant-calling algorithm designed to assess allele-specific variation within RNA-seq, we were able to detect single nucleotide polymorphisms (SNPs) within the RNA transcriptome that likely emerge from mutation within the DNA genome [<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>,<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>]. While RNA modifications such as inosine can be read as SNPs during the process of reverse transcription, these events should not approach the near 100% read depth stringency we required among our three biological replicates to call a conserved variant [<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>]. While this technique is only applicable for the actively transcribed region of the genome, nearly every nucleotide of the gene-dense AdV genome is transcribed at a sufficient level for this strategy to provide meaningful data.</p><p id="p-10">In total, we discovered 24 SNPs and no insertions or deletions in the Ad5 genome when compared to the original annotation (<strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a></strong>). Of these mutations, exactly half (12) are not predicted to change amino acid coding capacity, with two SNPs occurring within untranslated regions of viral RNA and the remaining ten leading to synonymous amino acid codons within all reading frames annotated to be protein producing. The remaining 12 mutations are predicted to lead to coding sequence variations at the amino acid level, with all examples being missense mutations and no evidence of premature stop codons. Importantly, none of the mutations discovered generated novel RNA splice sites. These data demonstrate the ability to call mutations within the DNA genomes of viruses using solely high-depth RNA sequencing data. Furthermore, detecting only 24 SNPs out of 35,938 nucleotides highlights the overall genomic stability of AdV.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.876037/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="RNA-seq reveals high-confidence SNPs within the Ad5 genome. The 35,938 base pair linear genome of Ad5 is displayed in the traditional left to right format. Major transcriptional units are shown as boxes above or below the genome with arrowheads denoting the orientation of the open reading frames (ORFs) encoded within. Grey boxes denote early gene transcriptional units while black boxes denote late genes. Bcftools was used to analyze short-read RNA seq data to predict single nucleotide polymorphisms (SNPs) and insertions/deletions (InDels) that approach 100% of the RNA reads when compared to the reference Ad5 genome (AC_000008). In total, 23 such SNPs were discovered and their position within the genome is highlighted by a red vertical line. For each SNP, the nucleotide position as well as the top strand reference base and corrected base are shown in black text (nucleotide position, reference base -&gt; corrected base). If indicated SNPs fell within untranslated regions (UTR), or did not change the encoded amino acid of any annotated reading frame potentially impacted by the SNP, these were marked with blue text denoting either UTR or Syn (synonymous mutation), respectively. For any SNP that led to an amino acid change within an annotated ORF, these ORFs as well as the identity of the reference amino acid and corrected amino acid are highlighted in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-5584934" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;RNA-seq reveals high-confidence SNPs within the Ad5 genome.&lt;/span&gt; The 35,938 base pair linear genome of Ad5 is displayed in the traditional left to right format. Major transcriptional units are shown as boxes above or below the genome with arrowheads denoting the orientation of the open reading frames (ORFs) encoded within. Grey boxes denote early gene transcriptional units while black boxes denote late genes. Bcftools was used to analyze short-read RNA seq data to predict single nucleotide polymorphisms (SNPs) and insertions/deletions (InDels) that approach 100% of the RNA reads when compared to the reference Ad5 genome (AC_000008). In total, 23 such SNPs were discovered and their position within the genome is highlighted by a red vertical line. For each SNP, the nucleotide position as well as the top strand reference base and corrected base are shown in black text (nucleotide position, reference base -&gt; corrected base). If indicated SNPs fell within untranslated regions (UTR), or did not change the encoded amino acid of any annotated reading frame potentially impacted by the SNP, these were marked with blue text denoting either UTR or Syn (synonymous mutation), respectively. For any SNP that led to an amino acid change within an annotated ORF, these ORFs as well as the identity of the reference amino acid and corrected amino acid are highlighted in red.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.876037/F1.medium.gif" width="440" height="111"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.876037/F1.medium.gif" width="440" height="111"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.876037/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.876037/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">RNA-seq reveals high-confidence SNPs within the Ad5 genome.</span><p id="p-11" class="first-child">The 35,938 base pair linear genome of Ad5 is displayed in the traditional left to right format. Major transcriptional units are shown as boxes above or below the genome with arrowheads denoting the orientation of the open reading frames (ORFs) encoded within. Grey boxes denote early gene transcriptional units while black boxes denote late genes. Bcftools was used to analyze short-read RNA seq data to predict single nucleotide polymorphisms (SNPs) and insertions/deletions (InDels) that approach 100% of the RNA reads when compared to the reference Ad5 genome (AC_000008). In total, 23 such SNPs were discovered and their position within the genome is highlighted by a red vertical line. For each SNP, the nucleotide position as well as the top strand reference base and corrected base are shown in black text (nucleotide position, reference base -&gt; corrected base). If indicated SNPs fell within untranslated regions (UTR), or did not change the encoded amino acid of any annotated reading frame potentially impacted by the SNP, these were marked with blue text denoting either UTR or Syn (synonymous mutation), respectively. For any SNP that led to an amino acid change within an annotated ORF, these ORFs as well as the identity of the reference amino acid and corrected amino acid are highlighted in red.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Combined short-read and long-read sequencing showcases adenovirus transcriptome complexity</h3><p id="p-12">To compare short-read Illumina sequencing and long-read nanopore sequencing directly, A549 cells were infected with Ad5 for 24 hours and total RNA was harvested in biological triplicate. Fractions of these three samples were prepared into standard strand specific Illumina RNA-seq libraries using the polyadenylated mRNA fraction. The same RNA samples were then poly(A) purified before submitting to direct RNA sequencing (dRNA-seq) on an Oxford Nanopore Technologies MinION MkIb platform [<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>]. Resulting sequence reads were aligned to the Ad5 reference genome using either GSNAP for short-reads [<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>], or MiniMap2 for long-reads [<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>]. Overall sequencing depth for both forward and reverse reads are shown in <strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a></strong>. While Illumina sequencing provided on average three times the read depth when compared to dRNA sequencing, the overall coverage plots were similar.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.876037/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Combined short-read and long-read sequencing showcases adenovirus transcriptome complexity. A549 cells were infected with Ad5 for 24 hours before RNA was extracted and subjected to both short-read and long-read sequencing. Sequence coverage provided by short-read stranded RNA-seq (Illumina, light blue), as well as nanopore long-read direct RNA-seq (Nanopore dRNA, dark blue), is shown along the Ad5 genome. For both tracks, reads aligning to the forward strand are plotted above the genome, while reads aligning to the reverse strand are shown below. For dRNA-seq datasets, reads can be reduced to their 5&#x2019; and 3&#x2019; ends and peak-calling applied to predict individual transcription start sites (TSS, green vertical lines) or cleavage and polyadenylation sites (CPAS, magenta vertical lines), respectively. Similarly, the ContextMap algorithm can predict, albeit at lower sensitivity, CPAS sites from poly(A) containing fragments within Illumina RNA-seq data (ContextMap, light blue vertical lines). Individual RNA transcripts are shown above and below the genome, thin bars denote 5&#x2019; and 3&#x2019; untranslated regions (UTR), thick bars denote open reading frames (ORFs), and thin lines with arrowheads denote both introns and orientation of transcription. Previously characterized early genes are denoted in grey, while previously characterized late genes are denoted in black. RNA isoforms discovered in this study are highlighted in red. Names of transcriptional units are shown under each cluster of transcripts, while the name of the protein derived from the respective ORF is listed after each transcript. The position of Pol III-derived noncoding RNAs virus associated VA-I and VA-II are highlighted in teal boxes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-5584934" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Combined short-read and long-read sequencing showcases adenovirus transcriptome complexity.&lt;/span&gt; A549 cells were infected with Ad5 for 24 hours before RNA was extracted and subjected to both short-read and long-read sequencing. Sequence coverage provided by short-read stranded RNA-seq (Illumina, light blue), as well as nanopore long-read direct RNA-seq (Nanopore dRNA, dark blue), is shown along the Ad5 genome. For both tracks, reads aligning to the forward strand are plotted above the genome, while reads aligning to the reverse strand are shown below. For dRNA-seq datasets, reads can be reduced to their 5&#x2019; and 3&#x2019; ends and peak-calling applied to predict individual transcription start sites (TSS, green vertical lines) or cleavage and polyadenylation sites (CPAS, magenta vertical lines), respectively. Similarly, the ContextMap algorithm can predict, albeit at lower sensitivity, CPAS sites from poly(A) containing fragments within Illumina RNA-seq data (ContextMap, light blue vertical lines). Individual RNA transcripts are shown above and below the genome, thin bars denote 5&#x2019; and 3&#x2019; untranslated regions (UTR), thick bars denote open reading frames (ORFs), and thin lines with arrowheads denote both introns and orientation of transcription. Previously characterized early genes are denoted in grey, while previously characterized late genes are denoted in black. RNA isoforms discovered in this study are highlighted in red. Names of transcriptional units are shown under each cluster of transcripts, while the name of the protein derived from the respective ORF is listed after each transcript. The position of Pol III-derived noncoding RNAs virus associated VA-I and VA-II are highlighted in teal boxes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.876037/F2.medium.gif" width="440" height="390"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.876037/F2.medium.gif" width="440" height="390"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.876037/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.876037/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Combined short-read and long-read sequencing showcases adenovirus transcriptome complexity.</span><p id="p-13" class="first-child">A549 cells were infected with Ad5 for 24 hours before RNA was extracted and subjected to both short-read and long-read sequencing. Sequence coverage provided by short-read stranded RNA-seq (Illumina, light blue), as well as nanopore long-read direct RNA-seq (Nanopore dRNA, dark blue), is shown along the Ad5 genome. For both tracks, reads aligning to the forward strand are plotted above the genome, while reads aligning to the reverse strand are shown below. For dRNA-seq datasets, reads can be reduced to their 5’ and 3’ ends and peak-calling applied to predict individual transcription start sites (TSS, green vertical lines) or cleavage and polyadenylation sites (CPAS, magenta vertical lines), respectively. Similarly, the ContextMap algorithm can predict, albeit at lower sensitivity, CPAS sites from poly(A) containing fragments within Illumina RNA-seq data (ContextMap, light blue vertical lines). Individual RNA transcripts are shown above and below the genome, thin bars denote 5’ and 3’ untranslated regions (UTR), thick bars denote open reading frames (ORFs), and thin lines with arrowheads denote both introns and orientation of transcription. Previously characterized early genes are denoted in grey, while previously characterized late genes are denoted in black. RNA isoforms discovered in this study are highlighted in red. Names of transcriptional units are shown under each cluster of transcripts, while the name of the protein derived from the respective ORF is listed after each transcript. The position of Pol III-derived noncoding RNAs virus associated VA-I and VA-II are highlighted in teal boxes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">dRNA-seq is performed in the 3’ -&gt; 5’ direction and thus allows precise mapping of the 3’ ends of transcripts at which poly(A) tails are added (cleavage and polyadenylation site, CPAS) [<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">64</a>]. Where the quality of input RNA is high, a variable proportion of sequence reads extend all the way to their transcription start site (TSS). By collapsing sequences reads to their 5’ and 3’ ends, we were able to implement a peak-calling approach to predict TSS and CPAS [<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">57</a>], and map their positions along both the forward and reverse strand of the viral genome (<strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a></strong>). In addition, ContextMap2 [<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>] was used to mine Illumina RNA-seq data for short read sequences containing poly(A) stretches that could be aligned against the viral genome for an orthogonal method of CPAS detection (<strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a></strong>). Mapping the TSS on the forward strand revealed the locations of the E1A, E1B, pIX, MLP, and E3 promoters, while the reverse strand revealed the E4, UXP, E2-early, E2-late, and IVa2 promoters. We did not detect any transcripts starting internal to L4 at the proposed L4 promoter [<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a>]. When mapping CPAS loci, we saw great concordance between the dRNA-seq and ContextMap2 performed on short-read sequences. On the forward strand we were able to detect previously mapped CPAS events at the E1A, E1B/pIX, E3A, E3B, and individual L1 through L5 sites. On the reverse strand we detected CPAS at the E4, UXP/DBP, and E2B/IVa2 locations. In addition, we also detected TSS and CPAS around the RNA pol III-derived VA RNA I (<strong><a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a></strong>). While pol III transcripts are generally not polyadenylated, and thus would not be captured by our nanopore sequencing approach, it was previously reported that low levels of polyadenylation can occur on these transcripts [<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>]. Given the high abundance of AdV VA RNAs (up to 10<sup>8</sup> copies per cell during late infection), it remains likely that low level VA RNA polyadenylation events are occurring [<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>].</p><p id="p-15">To generate accurate splicing maps of AdV transcripts we combined the sensitivity of short-read sequencing to identify RNA junctions and then placed them in the context of full-length RNA isoforms using dRNA sequencing. Due to the spurious nature of low-level AdV splicing events [<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">53</a>], we set abundance thresholds for the highly abundant viral late transcripts of 500 reads for short-read junctions, and at least ten events detected in the long-read sequencing when collapsed by FLAIR [<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>]. Using this method, we readily detected other recently discovered viral isoforms, such as multiple splice sites preceding the pVII ORF [<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">53</a>], the so-called X, Y, and Z leaders embedded in E3 and preceding L5-Fiber [<a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">53</a>,<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>], and the newly described UXP [<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">47</a>,<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">48</a>]. Using full-length RNAs, we were able to detect novel splice sites producing canonical ORFs that only differ in UTRs for L4-100K, L4-33K, L4-pVIII, and E4orf6/7. In addition, we discovered canonical ORF isoforms embedded within transcripts generated from non-canonical promoters, such as Fiber driven by the E3 promoter, E3-10K driven by the Major Late Promoter, and DBP driven by the E4 promoter. Within transcriptional units, we discovered the presence of internal splice sites leading to in-frame truncations of existing ORFs, such as L4-22K and four distinct isoforms of truncated L4-100K. We also discovered splicing events predicted to lead to in-frame fusion events within transcriptional units, such as fusions between N-terminal fragments of L4-100K and L4-33K or L4-pVIII or the X-Z-Fiber ORF. Furthermore, gene fusion events were observed that join disparate transcriptional units, such as an N-terminal fragment of E1B-19K and pIX (19K/IX) or E4orf6 and DBP (E4orf6/DBP). Lastly, we detected a splice site leading to a novel ORF of predicted 13 kilodaltons (L2-Unk13K) between the splice sites for L2-V and L2-pX. This novel L2 splice site was conserved in Ad2 [<a id="xref-ref-53-5" class="xref-bibr" href="#ref-53">53</a>]. Overall, we discovered 20 new isoforms for a total of 75 expressed RNA isoforms during Ad5 infection. Of these, many potentially exciting fusions and truncations of existing ORFs remain to be explored.</p></div><div id="sec-5" class="subsection"><h3>Direct RNA Sequencing unambiguously distinguishes early and late transcription</h3><p id="p-16">We next determined if we could provide unambiguous detection of viral transcripts over a time-course of infection that recapitulated early and late viral kinetics. By aligning long reads to the fully re-annotated viral transcriptome (as opposed to the viral genome), and only counting the reads that could be unambiguously assigned to a single transcript, we were able to detect all of the canonical and newly discovered transcripts (<strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a></strong>). At 12 hours post infection (hpi) the majority of viral transcripts detected were early RNAs, particularly E1A-large and E1A-small, E1B-19K and E1B-55K, early promoter DBP, and E4orf3 (<strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a></strong>). However, at this time point we still detected low-level viral late transcripts that progressed beyond the L1 polyadenylation site, corroborating recent work [<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">54</a>]. At 24 hpi, however, viral gene expression shifted to be dominated by late gene expression, as well as early transcripts derived from the E1B locus (<strong><a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a></strong>). At late times post infection we also saw the E3-Fiber transcript, 19K/IX, and E4-DBP transcripts increase dramatically, potentially implicating these messages as novel late transcripts, with expression as abundant as the recently described late UXP transcript [<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">47</a>,<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">48</a>]. Furthermore, while all permutations of the X, Y, and Z leaders preceding Fiber were previously detected by short-read sequencing these could not be phased to full-length transcript isoforms [<a id="xref-ref-53-6" class="xref-bibr" href="#ref-53">53</a>,<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">70</a>]. Our full-length RNA data indicate that all Fiber transcripts can be detected, but MLP-Fiber and Y-Fiber are the most abundant, followed by XY-Fiber, and then all other isoforms. While the previous lack of detection of some of these novel transcripts can be explained by low overall abundance (e.g., L2-Unk13K, L4-100K/VIII) many of the L4-100K truncations and L4-33K fusions are expressed at levels higher than that of the bona fide late transcript UXP. These data demonstrate that the newly discovered viral transcripts can be reproducibly detected over a time-course of infection with Ad5, as well as display differential expression based on the stage of infection.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.876037/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Direct RNA Sequencing (dRNA-seq) unambiguously distinguishes early and late transcription. (A) dRNA-seq was performed on polyadenylated RNA from Ad5-infected A549 cells extracted at 12 hours post infection (hpi). Sequence reads were aligned to the re-annotated transcriptome and filtered to retain only unambiguous primary alignments. Normalized read count indicates the number of RNAs for a particular transcript once normalized to the total number of mappable reads (human plus adenovirus) for the entire sequencing reaction. For all panels, grey bars indicate early genes, black bars indicate late genes, and red bars indicate novel isoforms discovered in this study. Particular transcripts are highlighted if there were less than 10 counts of a particular isoform detected (" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-5584934" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Direct RNA Sequencing (dRNA-seq) unambiguously distinguishes early and late transcription.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; dRNA-seq was performed on polyadenylated RNA from Ad5-infected A549 cells extracted at 12 hours post infection (hpi). Sequence reads were aligned to the re-annotated transcriptome and filtered to retain only unambiguous primary alignments. Normalized read count indicates the number of RNAs for a particular transcript once normalized to the total number of mappable reads (human plus adenovirus) for the entire sequencing reaction. For all panels, grey bars indicate early genes, black bars indicate late genes, and red bars indicate novel isoforms discovered in this study. Particular transcripts are highlighted if there were less than 10 counts of a particular isoform detected (&lt;10), or if the RNA was undetectable at that time point (nd). &lt;strong&gt;(B)&lt;/strong&gt; Same as in Panel A, but with RNA harvested at 24 hpi.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.876037/F3.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.876037/F3.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.876037/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.13.876037/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Direct RNA Sequencing (dRNA-seq) unambiguously distinguishes early and late transcription.</span><p id="p-17" class="first-child"><strong>(A)</strong> dRNA-seq was performed on polyadenylated RNA from Ad5-infected A549 cells extracted at 12 hours post infection (hpi). Sequence reads were aligned to the re-annotated transcriptome and filtered to retain only unambiguous primary alignments. Normalized read count indicates the number of RNAs for a particular transcript once normalized to the total number of mappable reads (human plus adenovirus) for the entire sequencing reaction. For all panels, grey bars indicate early genes, black bars indicate late genes, and red bars indicate novel isoforms discovered in this study. Particular transcripts are highlighted if there were less than 10 counts of a particular isoform detected (&lt;10), or if the RNA was undetectable at that time point (nd). <strong>(B)</strong> Same as in Panel A, but with RNA harvested at 24 hpi.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-6"><h2 class="">Discussion</h2><p id="p-18">DNA viruses encode large amounts of information in compact genomes through alternative splicing, overlapping transcripts, and transcription from both strands of the genome. The complexity of the adenovirus transcriptome has not been fully explored using modern high-throughput technologies. Here we integrate short-read cDNA sequencing and long-read direct RNA sequencing to re-annotate both the Ad5 DNA genome and RNA transcriptome. Using high quality and high depth short-read sequencing, we were able to detect SNPs within the transcribed regions of the genome approaching 100% penetrance, indicating that these sites were likely present in the genome and not due to RNA editing or modifications. We recapitulated the known TSS and CPAS sites throughout the Ad5 genome, and annotated novel splicing events within the viral transcriptome. Of these 20 novel RNAs, 13 are likely to encode for altered ORFs including multiple fusion transcripts that span transcriptional units thought previously to be distinct. Overall, we have provided a more complete annotation of a complex viral transcriptome that highlights potentially new gene products for future study.</p><p id="p-19">Using RNA-seq data to call SNPs in viral DNA genomes is compelling since high quality short-read sequencing data sets already exist for many DNA viruses [<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a>–<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a>]. While half of the SNPs we called were synonymous or in non-coding regions, missense mutations have the potential to change the coding sequence of protein amino acids in meaningful ways. In addition, while SNPs are often tolerated during alignment of RNA-seq data, annotation of the correct primary amino acid sequence is critical for downstream analysis of mass spectrometry data [<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a>]. While the SNPs we detected might be bona fide mutations that have arisen during passage in cell culture, it is also possible that the original reference sequence contains errors introduced by the sequencing technologies employed at the time [<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>]. It will be critical to directly sequence the DNA genomes of Ad5 isolates from multiple laboratories to test this hypothesis.</p><p id="p-20">Previous studies of AdV transcription detected numerous splice sites beyond those employed by known isoforms. However, the constraints of short-read sequencing precluded proper assembly of these sites into full-length transcripts [<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">43</a>,<a id="xref-ref-53-7" class="xref-bibr" href="#ref-53">53</a>,<a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">70</a>]. Furthermore, targeted expression analysis over a time-course of infection was limited to already known transcripts [<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">54</a>]. Using direct RNA sequencing we have been able to confirm that these RNAs exist (e.g., the various x, y, and z leaders preceding some molecules of Fiber transcripts), as well as show regulated expression over a time-course of infection. We have also added ORF predictions to previously detected splice sites, such as L2-Unk13K, X-Z-Fiber, and the pVIII ORF derived from splicing directly from the tripartite leader to the L4-33K splice acceptor. While this last site was previously predicted to lead to the expression of a small 42 amino acid ORF [<a id="xref-ref-53-8" class="xref-bibr" href="#ref-53">53</a>], we propose that this transcript instead primarily encodes for pVIII with a small upstream ORF, as it is over five times as abundant as the canonical pVIII spliced RNA. It should be noted that we did not detect the presence of a putative L4 intermediate promoter TSS at either 12 or 24 hpi [<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">45</a>,<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a>,<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>]. One hypothesis is that the sequence detected in L4-100K that is necessary for early expression of L4-22K and L4-33K might instead encode for a <em>cis</em>-regulatory element that mediates the early accumulation of these two products produced from the major late promoter.</p><p id="p-21">Of the novel transcripts we have so far detected, all of them appear to display delayed late kinetics during infection. Of particular interest is the transcript encoding for a putative fusion event between the E4 transcriptional unit and the E2 transcriptional unit. This transcript, E4orf6/DBP, would have to skip the canonical E4 CPAS for the pre-mRNA to progress downstream to DBP for splicing. The three transcripts displaying this pattern, including E4-promoter driven DBP and frameshifted E4-Unk, are all much more abundant during the late phase of infection even though canonical E4 transcripts are expressed early. It will be very interesting to see if differential polyadenylation is regulated during the life cycle of the virus, as has been previously reported for herpes simplex virus [<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">57</a>,<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a>,<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a>]. Importantly, future research should identify whether the known functions of existing viral ORFs can be explained, at least in part, by the presence of these novel isoforms.</p></div><div class="section" id="sec-7"><h2 class="">Methods</h2><div id="sec-8" class="subsection"><h3>Cell Culture</h3><p id="p-22">A549 cells (ATCC CCL-185) were obtained from American Type Culture Collection (ATCC) and cultured at 37 °C and 5% CO2. Cells were maintained in Ham’s F-12K medium (Gibco, 21127-022) supplemented with 10% v/v FBS (VWR, 89510-186) and 1% v/v Pen/Strep (100 U/ml of penicillin, 100 μg/ml of streptomycin, Gibco, 15140-122). All cell lines tested negative for mycoplasma infection and were routinely tested afterwards using the LookOut Mycoplasma PCR Detection Kit (Sigma-Aldrich).</p></div><div id="sec-9" class="subsection"><h3>Viral infections</h3><p id="p-23">Adenovirus serotype 5 (Ad5) was originally purchased from ATCC. All viruses were expanded on HEK293 cells, purified using two sequential rounds of ultracentrifugation in CsCl gradients, and stored in 40% v/v glycerol at −20 °C (short term) or −80 °C (long term). Viral stock titer was determined on HEK293 cells by plaque assay, and all subsequent infections were performed at a multiplicity of infection (MOI) of 10 PFU/cell. Cells were infected at 80-90% confluent monolayers by incubation with diluted virus in a minimal volume of low serum (2%) F-12K for two hours. After infection viral inoculum was removed by vacuum and full serum growth media was replaced for the duration of the experiment.</p></div><div id="sec-10" class="subsection"><h3>RNA Isolation</h3><p id="p-24">Total RNA was isolated from cells by either TRIzol extraction (Thermo Fisher) or RNeasy Micro kit (Qiagen), following manufacturer protocols. RNA was treated with RNase-free DNase I (Qiagen), either on-column or after ethanol precipitation. To test quality, RNA was converted to complementary DNA (cDNA) using 1 μg of input RNA in the High Capcity RNA-to-cDNA kit (Thermo Fisher). Quantitative PCR was performed using the standard protocol for SYBR Green reagents (Thermo Fisher) in a QuantStudio 7 Flex Real-Time PCR System (Applied Biosystems).</p></div><div id="sec-11" class="subsection"><h3>Illumina Sequencing and Mapping</h3><p id="p-25">Total RNA from three biological replicates of Control knockdown or three biological replicates of METTL3-knockdown A549 cells infected with Ad5 for 24 hours were sent to Genewiz for preparation into strand-specific RNA-Seq libraries. Libraries were then run spread over three lanes of an Illumina HiSeq 2500 using a 150bp paired-end protocol. Raw reads were mapped to the GRCh37/hg19 genome assembly and the Ad5 genome using the RNA-seq aligner GSNAP [<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">65</a>] (version 2019-09-12). The algorithm was given known human gene models provided by GENCODE (release_27_hg19) to achieve higher mapping accuracy. We used R package ggplot2 for visualization. Downstream analysis and visualization was done using deepTools2 [<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a>]. Splice junctions were extracted using regtools [<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a>] and visualized in Integrative Genomics Viewer [<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a>].</p></div><div id="sec-12" class="subsection"><h3>Variant Calling</h3><p id="p-26">Illumina RNA-seq reads were aligned to the Ad5 genome obtained from NCBI (<a href="https://www.ncbi.nlm.nih.gov/nuccore/AC_000008">https://www.ncbi.nlm.nih.gov/nuccore/AC_000008</a>) using GSNAP [<a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">65</a>]. To identify variants such as single nucleotide polymorphisms (SNPs) and insertions/deletions (InDels), we combined mpileup and call from the bcftools (v1.9) package [<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">61</a>,<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">62</a>]. Here we used the following flags “-- redo-BAQ --min-BQ 30 --per-sample-mF” and “--multiallelic-caller --variants-only” respectively. Finally, we only considered variants if they were called significantly in all 3 replicates. We only observed SNPs but no InDels.</p></div><div id="sec-13" class="subsection"><h3>Direct RNA Sequencing on nanopore arrays</h3><p id="p-27">Direct RNA sequencing libraries were generated from 800-900 ng of poly(A) RNA, isolated using the Dynabeads™ mRNA Purification Kit (Invitrogen, 61006). Isolated poly(A) RNA was subsequently spiked with 0.3 μl of a synthetic Enolase 2 (ENO2) calibration RNA (Oxford Nanopore Technologies Ltd.) and prepared for sequencing using standard protocol steps previously described [<a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">57</a>,<a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">64</a>]. Sequencing was carried out on a MinION MkIb with R9.4.1 (rev D) flow cells (Oxford Nanopore Technologies Ltd.) for 20 hours and generated 550,000-770,000 sequence reads per dataset. Raw fast5 datasets were basecalled using Guppy v3.2.2 (-f FLO-MIN106 -k SQK-RNA002) and subsequently aligned against the adenovirus Ad5 reference genome (AC_000008.1) using MiniMap225 (-ax splice -k14 -uf --secondary=no), a splice aware aligner [<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">66</a>]. Resulting SAM files were parsed using SAMtools v1.3 [<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">83</a>].</p></div><div id="sec-14" class="subsection"><h3>Defining TSS and CPAS</h3><p id="p-28">Transcription start sites (TSS) as well as RNA cleavage and polyadenylation sites were identified as follows. Sorted BAM files containing sequence reads aligned to the Ad5 genome were parsed to BED12 files using BEDtools [<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">84</a>], separated by strand, truncated to their 5’ and 3’ termini, and output as BED6 files. Peak regions denoting TSS and CPAS were identified using the HOMER [<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">85</a>] findpeaks module (-o auto -style tss) using a --localSize of 100 and 500 and --size of 15 and 50 for TSS and CPAS, respectively. TSS peaks were compared against Illumina annotated splice sites to identify and remove peak artefacts derived from local alignment errors around splice junctions. To predict CPAS sites on the viral genome, we also used the RNA-seq aligner ContextMap2 (version 2.7.9) [<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">67</a>] which has poly(A) read mapping implemented on our short-read data. To run this tool, we used the following optional flags “-aligner_name bowtie --polyA --strandspecific”. Due to the previously reported errors when using ContextMap2 at very high read depth, we chose to randomly subsample 10 million, 20 million and 30 million and run the tool on each of the subsets. We only report poly(A) sites if they were called in all three replicates and in at least two of the subsample groups.</p></div><div id="sec-15" class="subsection"><h3>Splice junction correction and sequence read collapsing</h3><p id="p-29">Illumina-assisted correction of splice junctions in direct RNA-Seq data was performed using FLAIR v1.3 [<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">69</a>] in a stranded manner. Briefly, Illumina reads aligning to the Ad5 genome were split according to orientation and mapping strand [-f83 &amp; -f163 (forward) and -f99 &amp; -f147 (reverse)] and used to produce strand-specific junction files that were filtered to remove junctions supported by less than 100 Illumina reads. Direct RNA-Seq reads were similarly aligned to the Ad5 genome and separated according to orientation [-F4095 (forward) and -f16 (reverse)] prior to correction using the FLAIR correct module (default parameters). Resulting BED12 files were parsed to extend the termini of each individual sequence read to the nearest TSS and CPAS with BlockStarts and BlockSizes (BED12 cols 11 &amp; 12) corrected to reflect this. BED12 files were subsequently collapsed by identifying all reads sharing the same BlockStarts and BlockSizes and reducing these to a single representative. Resulting data were visualized along with the raw read data using IGV [<a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">82</a>] and low abundance isoforms (supported by less than 500 junctional reads or 10 full-length reads from Illumina or nanopore data, respectively) removed prior to producing the final annotation.</p></div><div id="sec-16" class="subsection"><h3>Isoform counting</h3><p id="p-30">Using our new Ad5 annotation, we generated a transcriptome database by parsing our GFF3 file to a BED12 file using the <em>gff3ToGenePred</em> and <em>genePredtoBED</em> functions within UCSCutils (<a href="https://github.com/itsvenu/UCSC-Utils-Download">https://github.com/itsvenu/UCSC-Utils-Download</a>) and subsequently extracting a fasta sequence for each transcript isoform using the <em>getfasta</em> function within BEDtools [<a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">84</a>]. Direct RNA-Seq reads were then aligned against the transcriptome database using parameters optimized for transcriptome-level alignment (minimap2 -ax map-ont -p 0.99). Isoform counts were generated by filtering only for primary alignments (SAM flag 0) with a mapping quality (MapQ) &gt; 0.</p></div></div><div class="section" id="sec-17"><h2 class="">Data Availability</h2><p id="p-32">Basecalled fast5 (Nanopore) and fastq (Illumina) datasets generated as part of this study can be downloaded from the European Nucleotide Archive (ENA) under the following study accession: PRJEB35667. The authors declare that all other data supporting the findings of this study are available within the article and its Supplementary Information files, or are available from the authors upon request. The newly generated genome and transcriptome annotation can be found at <a href="https://github.com/dandepledge/Ad5-annotation">https://github.com/dandepledge/Ad5-annotation</a>.</p></div><div class="section" id="sec-18"><h2 class="">Author Contributions</h2><p id="p-33">A.M.P. and M.D.W. conceived of the project and designed the experiments; D.P.D. and A.C.W. provided additional input into study design; A.M.P. performed the experiments and Illumina sequencing; D.P.D. performed the nanopore sequencing; K.E.H. and D.P.D. performed computational analyses; A.M.P. and D.P.D analyzed all additional data; A.M.P. and M.D.W. wrote the manuscript; All authors read, edited, and approved the final paper.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-31">We thank members of the Weitzman and Mohr/Wilson Labs for insightful discussions and input. This work was supported through NIH grants R21-AI130618 and R21-AI147163 (ACW), and R01-AI145266, R01-AI121321, and R01-CA097093 (MDW). Additional support came from the NCI T32 Training Grant in Tumor Virology T32-CA115299 (AMP) and Individual National Research Service Award F32-AI138432 (AMP). We extend special thanks to Ian Mohr (New York University School of Medicine) for support of DPD in part through National Institutes of Health (NIH) grants R01-AI073898 and R01-GM056927.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-dataset fn-group-external-links" id="fn-1"><p id="p-3">
<a href="https://github.com/dandepledge/Ad5-annotation">https://github.com/dandepledge/Ad5-annotation</a>
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.13.876037v1.1"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"> <span class="cit-name-given-names">David M.</span> <span class="cit-name-surname">Knipe</span></span>, </li><li><span class="cit-ed"> <span class="cit-name-given-names">Peter M.</span> <span class="cit-name-surname">Howley</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">J. Berk</span></span>. <span class="cit-chapter-title">Adenoviridae</span>. <span class="cit-edition">6th ed.</span> In: <span class="cit-ed"> <span class="cit-name-given-names">David M.</span> <span class="cit-name-surname">Knipe</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">Peter M.</span> <span class="cit-name-surname">Howley</span></span>, editors. <span class="cit-source">Fields Virology</span>. <span class="cit-edition">6th ed.</span> <span class="cit-publ-loc">Philadelphia</span>: <span class="cit-publ-name">Wolters Kluwer Health/Lippincott Williams &amp; Wilkins</span>; <span class="cit-pub-date">2013</span>. pp. <span class="cit-fpage">1704</span>–<span class="cit-lpage">1731</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.2" data-doi="10.3390/biomedicines6010030"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Khanal</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghimire</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dhamoon</span>  <span class="cit-name-given-names">AS</span></span>. <span class="cit-article-title">The Repertoire of Adenovirus in Human Disease: The Innocuous to the Deadly</span>. <abbr class="cit-jnl-abbrev">Biomedicines</abbr>. <span class="cit-pub-date">2018</span>; <span class="cit-vol">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3390/biomedicines6010030</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiomedicines%26rft.volume%253D6%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fbiomedicines6010030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/biomedicines6010030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.3" data-doi="10.1016/0042-6822(92)90082-Z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chroboczek</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bieber</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jacrot</span>  <span class="cit-name-given-names">B</span></span>. <span class="cit-article-title">The sequence of the genome of adenovirus type 5 and its comparison with the genome of adenovirus type 2</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-pub-date">1992</span>;<span class="cit-vol">186</span>: <span class="cit-fpage">280</span>–<span class="cit-lpage">285</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/0042-6822(92)90082-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DChroboczek%26rft.auinit1%253DJ.%26rft.volume%253D186%26rft.issue%253D1%26rft.spage%253D280%26rft.epage%253D285%26rft.atitle%253DThe%2Bsequence%2Bof%2Bthe%2Bgenome%2Bof%2Badenovirus%2Btype%2B5%2Band%2Bits%2Bcomparison%2Bwith%2Bthe%2Bgenome%2Bof%2Badenovirus%2Btype%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252892%252990082-Z%26rft_id%253Dinfo%253Apmid%252F1727603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(92)90082-Z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1727603&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992GU99400028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.4" data-doi="10.1099/vir.0.19497-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Davison</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benko</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrach</span>  <span class="cit-name-given-names">B</span></span>. <span class="cit-article-title">Genetic content and evolution of adenoviruses</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr>. <span class="cit-pub-date">2003</span>;<span class="cit-vol">84</span>: <span class="cit-fpage">2895</span>–<span class="cit-lpage">2908</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/vir.0.19497-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DDavison%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D84%26rft.issue%253D11%26rft.spage%253D2895%26rft.epage%253D2908%26rft.atitle%253DGenetic%2Bcontent%2Band%2Bevolution%2Bof%2Badenoviruses%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.19497-0%26rft_id%253Dinfo%253Apmid%252F14573794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.19497-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14573794&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186146800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.5" data-doi="10.1038/sj.onc.1209040"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berk</span>  <span class="cit-name-given-names">AJ</span></span>. <span class="cit-article-title">Recent lessons in gene expression, cell cycle control, and cell biology from adenovirus</span>. <abbr class="cit-jnl-abbrev">Oncogene</abbr>. <span class="cit-pub-date">2005</span>;<span class="cit-vol">24</span>: <span class="cit-fpage">7673</span>–<span class="cit-lpage">85</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/sj.onc.1209040</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncogene%26rft.stitle%253DOncogene%26rft.aulast%253DBerk%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D24%26rft.issue%253D52%26rft.spage%253D7673%26rft.epage%253D7685%26rft.atitle%253DRecent%2Blessons%2Bin%2Bgene%2Bexpression%252C%2Bcell%2Bcycle%2Bcontrol%252C%2Band%2Bcell%2Bbiology%2Bfrom%2Badenovirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.onc.1209040%26rft_id%253Dinfo%253Apmid%252F16299528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.onc.1209040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16299528&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233372600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.6" data-doi="10.1016/0092-8674(77)90180-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chow</span>  <span class="cit-name-given-names">LT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gelinas</span>  <span class="cit-name-given-names">RE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Broker</span>  <span class="cit-name-given-names">TR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">RJ</span></span>. <span class="cit-article-title">An amazing sequence arrangement at the 5’ ends of adenovirus 2 messenger RNA</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>. <span class="cit-pub-date">1977</span>;<span class="cit-vol">12</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DChow%26rft.auinit1%253DL.%2BT.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D8%26rft.atitle%253DAn%2Bamazing%2Bsequence%2Barrangement%2Bat%2Bthe%2B5%2527%2Bends%2Bof%2Badenovirus%2B2%2Bmessenger%2BRNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252877%252990180-5%26rft_id%253Dinfo%253Apmid%252F902310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(77)90180-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=902310&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1977DV77200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.7" data-doi="10.1073/pnas.74.8.3171"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berget</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">PA</span></span>. <span class="cit-article-title">Spliced segments at the 5’ terminus of adenovirus 2 late mRNA</span>. <abbr class="cit-jnl-abbrev">PNAS</abbr>. <span class="cit-pub-date">1977</span>;<span class="cit-vol">74</span>: <span class="cit-fpage">3171</span>–<span class="cit-lpage">3175</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.74.8.3171</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBerget%26rft.auinit1%253DS.%2BM.%26rft.volume%253D74%26rft.issue%253D8%26rft.spage%253D3171%26rft.epage%253D3175%26rft.atitle%253DSpliced%2Bsegments%2Bat%2Bthe%2B5%2527%2Bterminus%2Bof%2Badenovirus%2B2%2Blate%2BmRNA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.74.8.3171%26rft_id%253Dinfo%253Apmid%252F269380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3NC84LzMxNzEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8xMy8yMDE5LjEyLjEzLjg3NjAzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.8" data-doi="10.1093/nar/3.3.749"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salditt-Georgieff</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bachenheimer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Darnell</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Furuichi</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The methylation of adenovirus-specific nuclear and cytoplasmic RNA</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">1976</span>;<span class="cit-vol">3</span>: <span class="cit-fpage">749</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F3.3.749%26rft_id%253Dinfo%253Apmid%252F1272797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/3.3.749&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1272797&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1976BL19200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.9" data-doi="10.1073/pnas.68.11.2806"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Philipson</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glickman</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Darnell</span>  <span class="cit-name-given-names">JE</span></span>. <span class="cit-article-title">Addition of polyadenylate sequences to virus-specific RNA during adenovirus replication</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>. <span class="cit-pub-date">1971</span>;<span class="cit-vol">68</span>: <span class="cit-fpage">2806</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DPhilipson%26rft.auinit1%253DL.%26rft.volume%253D68%26rft.issue%253D11%26rft.spage%253D2806%26rft.epage%253D2809%26rft.atitle%253DAddition%2Bof%2BPolyadenylate%2BSequences%2Bto%2BVirus-Specific%2BRNA%2Bduring%2BAdenovirus%2BReplication%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.68.11.2806%26rft_id%253Dinfo%253Apmid%252F5315962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNjgvMTEvMjgwNiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzEzLzIwMTkuMTIuMTMuODc2MDM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.10" data-doi="10.1073/pnas.1525084113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berk</span>  <span class="cit-name-given-names">AJ</span></span>. <span class="cit-article-title">Discovery of RNA splicing and genes in pieces</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">113</span>: <span class="cit-fpage">801</span>–<span class="cit-lpage">805</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1525084113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1525084113%26rft_id%253Dinfo%253Apmid%252F26787897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTMvNC84MDEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8xMy8yMDE5LjEyLjEzLjg3NjAzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.11" data-doi="10.1038/295380a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Montell</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">EF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Caruthers</span>  <span class="cit-name-given-names">MH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berk</span>  <span class="cit-name-given-names">AJ</span></span>. <span class="cit-article-title">Resolving the functions of overlapping viral genes by site-specific mutagenesis at a mRNA splice site</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-pub-date">1982</span>;<span class="cit-vol">295</span>: <span class="cit-fpage">380</span>–<span class="cit-lpage">384</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/295380a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMontell%26rft.auinit1%253DC.%26rft.volume%253D295%26rft.issue%253D5848%26rft.spage%253D380%26rft.epage%253D384%26rft.atitle%253DResolving%2Bthe%2Bfunctions%2Bof%2Boverlapping%2Bviral%2Bgenes%2Bby%2Bsite-specific%2Bmutagenesis%2Bat%2Ba%2BmRNA%2Bsplice%2Bsite.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F295380a0%26rft_id%253Dinfo%253Apmid%252F7057903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/295380a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7057903&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.13.876037v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Winberg</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shenk</span>  <span class="cit-name-given-names">T</span></span>. <span class="cit-article-title">Dissection of overlapping functions within the adenovirus type 5 E1A gene</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr>. <span class="cit-pub-date">1984</span>;<span class="cit-vol">3</span>: <span class="cit-fpage">1907</span>–<span class="cit-lpage">1912</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ%26rft.volume%253D3%26rft.spage%253D1907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">13.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.13.876037v1.13" data-doi="10.1016/j.chom.2012.05.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fonseca</span>  <span class="cit-name-given-names">GJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thillainadesan</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yousef</span>  <span class="cit-name-given-names">AF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ablack</span>  <span class="cit-name-given-names">JN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mossman</span>  <span class="cit-name-given-names">KL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Torchia</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Adenovirus evasion of interferon-mediated innate immunity by direct antagonism of a cellular histone posttranslational modification</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr>. <span class="cit-pub-date">2012</span>; <span class="cit-vol">11</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">606</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.chom.2012.05.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BHost%2BMicrobe%26rft.aulast%253DFonseca%26rft.auinit1%253DG.%2BJ.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253D597%26rft.epage%253D606%26rft.atitle%253DAdenovirus%2Bevasion%2Bof%2Binterferon-mediated%2Binnate%2Bimmunity%2Bby%2Bdirect%2Bantagonism%2Bof%2Ba%2Bcellular%2Bhistone%2Bposttranslational%2Bmodification.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2012.05.005%26rft_id%253Dinfo%253Apmid%252F22704620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2012.05.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22704620&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.14" data-doi="10.1016/j.chom.2017.11.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zemke</span>  <span class="cit-name-given-names">NR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berk</span>  <span class="cit-name-given-names">AJ</span></span>. <span class="cit-article-title">The Adenovirus E1A C Terminus Suppresses a Delayed Antiviral Response and Modulates RAS Signaling</span>. <abbr class="cit-jnl-abbrev">Cell Host &amp; Microbe</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">22</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">800.e5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.chom.2017.11.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2B%2526%2BMicrobe%26rft.volume%253D22%26rft.spage%253D789%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2017.11.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2017.11.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.15" data-doi="10.1128/JVI.00104-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pelka</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ablack</span>  <span class="cit-name-given-names">JNG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fonseca</span>  <span class="cit-name-given-names">GJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yousef</span>  <span class="cit-name-given-names">AF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mymryk</span>  <span class="cit-name-given-names">JS</span></span>. <span class="cit-article-title">Intrinsic Structural Disorder in Adenovirus E1A: a Viral Molecular Hub Linking Multiple Diverse Processes</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>. <span class="cit-pub-date">2008</span>;<span class="cit-vol">82</span>: <span class="cit-fpage">7252</span>–<span class="cit-lpage">7263</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.00104-08</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00104-08%26rft_id%253Dinfo%253Apmid%252F18385237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Mi8xNS83MjUyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTMvMjAxOS4xMi4xMy44NzYwMzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.16" data-doi="10.1128/JVI.02417-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blackford</span>  <span class="cit-name-given-names">AN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grand</span>  <span class="cit-name-given-names">RJA</span></span>. <span class="cit-article-title">Adenovirus E1B 55-Kilodalton Protein: Multiple Roles in Viral Infection and Cell Transformation</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">83</span>: <span class="cit-fpage">4000</span>–<span class="cit-lpage">4012</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.02417-08</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02417-08%26rft_id%253Dinfo%253Apmid%252F19211739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgzLzkvNDAwMCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzEzLzIwMTkuMTIuMTMuODc2MDM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.17" data-doi="10.1101/gad.10.4.461"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sabbatini</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Modha</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">E</span></span>. <span class="cit-article-title">The E1B 19K protein blocks apoptosis by interacting with and inhibiting the p53-inducible and death-promoting Bax protein</span>. <abbr class="cit-jnl-abbrev">Genes Dev</abbr>. <span class="cit-pub-date">1996</span>;<span class="cit-vol">10</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">477</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/gad.10.4.461</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DHan%26rft.auinit1%253DJ%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D461%26rft.epage%253D477%26rft.atitle%253DThe%2BE1B%2B19K%2Bprotein%2Bblocks%2Bapoptosis%2Bby%2Binteracting%2Bwith%2Band%2Binhibiting%2Bthe%2Bp53-inducible%2Band%2Bdeath-promoting%2BBax%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.10.4.461%26rft_id%253Dinfo%253Apmid%252F8600029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTAvNC80NjEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8xMy8yMDE5LjEyLjEzLjg3NjAzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.18" data-doi="10.1038/357082a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yew</span>  <span class="cit-name-given-names">PR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berk</span>  <span class="cit-name-given-names">AJ</span></span>. <span class="cit-article-title">Inhibition of p53 transactivation required for transformation by adenovirus early 1B protein</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-pub-date">1992</span>;<span class="cit-vol">357</span>: <span class="cit-fpage">82</span>–<span class="cit-lpage">85</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/357082a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DYew%26rft.auinit1%253DP.%2BR.%26rft.volume%253D357%26rft.issue%253D6373%26rft.spage%253D82%26rft.epage%253D85%26rft.atitle%253DInhibition%2Bof%2Bp53%2Btransactivation%2Brequired%2Bfor%2Btransformation%2Bby%2Badenovirus%2Bearly%2B1B%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F357082a0%26rft_id%253Dinfo%253Apmid%252F1533443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/357082a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1533443&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.13.876037v1.19" data-doi="10.1016/0042-6822(90)90088-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bridge</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ketner</span>  <span class="cit-name-given-names">G</span></span>. <span class="cit-article-title">Interaction of adenoviral E4 and E1b products in late gene expression</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-pub-date">1990</span>;<span class="cit-vol">174</span>: <span class="cit-fpage">345</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DBridge%26rft.auinit1%253DE.%26rft.volume%253D174%26rft.issue%253D2%26rft.spage%253D345%26rft.epage%253D353%26rft.atitle%253DInteraction%2Bof%2Badenoviral%2BE4%2Band%2BE1b%2Bproducts%2Bin%2Blate%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252890%252990088-9%26rft_id%253Dinfo%253Apmid%252F2137659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(90)90088-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2137659&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990CL76500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">20.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.13.876037v1.20" data-doi="10.1128/JVI.74.23.11407-11412.2000"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cathomen</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weitzman</span>  <span class="cit-name-given-names">MD</span></span>. <span class="cit-article-title">A functional complex of adenovirus proteins E1B-55kDa and E4orf6 is necessary to modulate the expression level of p53 but not its transcriptional activity</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2000</span>;<span class="cit-vol">74</span>: <span class="cit-fpage">11407</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DCathomen%26rft.auinit1%253DT.%26rft.volume%253D74%26rft.issue%253D23%26rft.spage%253D11407%26rft.epage%253D11412%26rft.atitle%253DA%2BFunctional%2BComplex%2Bof%2BAdenovirus%2BProteins%2BE1B-55kDa%2Band%2BE4orf6%2BIs%2BNecessary%2BTo%2BModulate%2Bthe%2BExpression%2BLevel%2Bof%2Bp53%2Bbut%2BNot%2BIts%2BTranscriptional%2BActivity%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.74.23.11407-11412.2000%26rft_id%253Dinfo%253Apmid%252F11070042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3NC8yMy8xMTQwNyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzEzLzIwMTkuMTIuMTMuODc2MDM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">21.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.13.876037v1.21" data-doi="10.1128/JVI.76.18.9194-9206.2002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harada</span>  <span class="cit-name-given-names">JN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shevchenko</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shevchenko</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pallas</span>  <span class="cit-name-given-names">DC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berk</span>  <span class="cit-name-given-names">AJ</span></span>. <span class="cit-article-title">Analysis of the adenovirus E1B-55K-anchored proteome reveals its link to ubiquitination machinery</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2002</span>;<span class="cit-vol">76</span>: <span class="cit-fpage">9194</span>–<span class="cit-lpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DHarada%26rft.auinit1%253DJ.%2BN.%26rft.volume%253D76%26rft.issue%253D18%26rft.spage%253D9194%26rft.epage%253D9206%26rft.atitle%253DAnalysis%2Bof%2Bthe%2BAdenovirus%2BE1B-55K-Anchored%2BProteome%2BReveals%2BIts%2BLink%2Bto%2BUbiquitination%2BMachinery%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.76.18.9194-9206.2002%26rft_id%253Dinfo%253Apmid%252F12186903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3Ni8xOC85MTk0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTMvMjAxOS4xMi4xMy44NzYwMzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">22.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.13.876037v1.22" data-doi="10.1101/gad.926401"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Querido</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanchette</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kamura</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morrison</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boivin</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Degradation of p53 by adenovirus E4orf6 and E1B55K proteins occurs via a novel mechanism involving a Cullin-containing complex</span>. <abbr class="cit-jnl-abbrev">Genes Dev</abbr>. <span class="cit-pub-date">2001</span>;<span class="cit-vol">15</span>: <span class="cit-fpage">3104</span>–<span class="cit-lpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/gad.926401</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DQuerido%26rft.auinit1%253DE.%26rft.volume%253D15%26rft.issue%253D23%26rft.spage%253D3104%26rft.epage%253D3117%26rft.atitle%253DDegradation%2Bof%2Bp53%2Bby%2Badenovirus%2BE4orf6%2Band%2BE1B55K%2Bproteins%2Boccurs%2Bvia%2Ba%2Bnovel%2Bmechanism%2Binvolving%2Ba%2BCullin-containing%2Bcomplex%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.926401%26rft_id%253Dinfo%253Apmid%252F11731475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE1LzIzLzMxMDQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8xMy8yMDE5LjEyLjEzLjg3NjAzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.23" data-doi="10.1016/j.coviro.2018.08.017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dybas</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Herrmann</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weitzman</span>  <span class="cit-name-given-names">MD</span></span>. <span class="cit-article-title">Ubiquitination at the interface of tumor viruses and DNA damage responses</span>. <abbr class="cit-jnl-abbrev">Curr Opin Virol</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">32</span>: <span class="cit-fpage">40</span>–<span class="cit-lpage">47</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.coviro.2018.08.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BVirol%26rft.volume%253D32%26rft.spage%253D40%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2018.08.017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2018.08.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.24" data-doi="10.1016/0092-8674(78)90332-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Winnacker</span>  <span class="cit-name-given-names">EL</span></span>. <span class="cit-article-title">Adenovirus DNA: structure and function of a novel replicon</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>. <span class="cit-pub-date">1978</span>;<span class="cit-vol">14</span>: <span class="cit-fpage">761</span>–<span class="cit-lpage">773</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/0092-8674(78)90332-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DWinnacker%26rft.auinit1%253DE.%2BL.%26rft.volume%253D14%26rft.issue%253D4%26rft.spage%253D761%26rft.epage%253D773%26rft.atitle%253DAdenovirus%2BDNA%253A%2Bstructure%2Band%2Bfunction%2Bof%2Ba%2Bnovel%2Breplicon.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252878%252990332-x%26rft_id%253Dinfo%253Apmid%252F357012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(78)90332-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=357012&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Webster</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leith</span>  <span class="cit-name-given-names">IR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nicholson</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hounsell</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hay</span>  <span class="cit-name-given-names">RT</span></span>. <span class="cit-article-title">Role of preterminal protein processing in adenovirus replication</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">1997</span>;<span class="cit-vol">71</span>: <span class="cit-fpage">6381</span>–<span class="cit-lpage">6389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DWebster%26rft.auinit1%253DA.%26rft.volume%253D71%26rft.issue%253D9%26rft.spage%253D6381%26rft.epage%253D6389%26rft.atitle%253DRole%2Bof%2Bpreterminal%2Bprotein%2Bprocessing%2Bin%2Badenovirus%2Breplication%26rft_id%253Dinfo%253Apmid%252F9261355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjcxLzkvNjM4MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzEzLzIwMTkuMTIuMTMuODc2MDM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">26.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.13.876037v1.26" data-doi="10.1128/JVI.76.16.8200-8207.2002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brenkman</span>  <span class="cit-name-given-names">AB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Breure</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Vliet</span>  <span class="cit-name-given-names">PC</span></span>. <span class="cit-article-title">Molecular architecture of adenovirus DNA polymerase and location of the protein primer</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2002</span>;<span class="cit-vol">76</span>: <span class="cit-fpage">8200</span>–<span class="cit-lpage">8207</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.76.16.8200-8207.2002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DBrenkman%26rft.auinit1%253DA.%2BB.%26rft.volume%253D76%26rft.issue%253D16%26rft.spage%253D8200%26rft.epage%253D8207%26rft.atitle%253DMolecular%2BArchitecture%2Bof%2BAdenovirus%2BDNA%2BPolymerase%2Band%2BLocation%2Bof%2Bthe%2BProtein%2BPrimer%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.76.16.8200-8207.2002%26rft_id%253Dinfo%253Apmid%252F12134025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3Ni8xNi84MjAwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTMvMjAxOS4xMi4xMy44NzYwMzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.27" data-doi="10.1007/978-3-662-05597-7_7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Jong</span>  <span class="cit-name-given-names">RN</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Vliet</span>  <span class="cit-name-given-names">PC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brenkman</span>  <span class="cit-name-given-names">AB</span></span>. <span class="cit-article-title">Adenovirus DNA replication: protein priming, jumping back and the role of the DNA binding protein DBP</span>. <abbr class="cit-jnl-abbrev">Curr Top Microbiol Immunol</abbr>. <span class="cit-pub-date">2003</span>;<span class="cit-vol">272</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">211</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/978-3-662-05597-7_7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Btopics%2Bin%2Bmicrobiology%2Band%2Bimmunology%26rft.stitle%253DCurr%2BTop%2BMicrobiol%2BImmunol%26rft.aulast%253Dde%2BJong%26rft.auinit1%253DR.%2BN.%26rft.volume%253D272%26rft.spage%253D187%26rft.epage%253D211%26rft.atitle%253DAdenovirus%2BDNA%2Breplication%253A%2Bprotein%2Bpriming%252C%2Bjumping%2Bback%2Band%2Bthe%2Brole%2Bof%2Bthe%2BDNA%2Bbinding%2Bprotein%2BDBP.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-662-05597-7_7%26rft_id%253Dinfo%253Apmid%252F12747551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-662-05597-7_7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12747551&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184998700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.28" data-doi="10.1016/j.virusres.2011.05.022"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rajaiya</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dyer</span>  <span class="cit-name-given-names">DW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chodosh</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-article-title">The E3 CR1-gamma gene in human adenoviruses associated with epidemic keratoconjunctivitis</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">160</span>: <span class="cit-fpage">120</span>–<span class="cit-lpage">127</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2011.05.022</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DRobinson%26rft.auinit1%253DC.%2BM.%26rft.volume%253D160%26rft.issue%253D1-2%26rft.spage%253D120%26rft.epage%253D127%26rft.atitle%253DThe%2BE3%2BCR1-gamma%2Bgene%2Bin%2Bhuman%2Badenoviruses%2Bassociated%2Bwith%2Bepidemic%2Bkeratoconjunctivitis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2011.05.022%26rft_id%253Dinfo%253Apmid%252F21683743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2011.05.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21683743&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295345600015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.13.876037v1.29" data-doi="10.1128/JVI.01927-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dehghan</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dyer</span>  <span class="cit-name-given-names">DW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seto</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Homologous recombination in E3 genes of human adenovirus species D</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">87</span>: <span class="cit-fpage">12481</span>–<span class="cit-lpage">12488</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.01927-13</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01927-13%26rft_id%253Dinfo%253Apmid%252F24027303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Ny8yMi8xMjQ4MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzEzLzIwMTkuMTIuMTMuODc2MDM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.13.876037v1.30" data-doi="10.1007/978-3-642-79496-4_13"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Doerfler</span>  <span class="cit-name-given-names">W</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Böhm</span>  <span class="cit-name-given-names">P</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">WSM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tollefson</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hermiston</span>  <span class="cit-name-given-names">TW</span></span>. <span class="cit-chapter-title">E3 Transcription Unit of Adenovirus</span>. In: <span class="cit-ed"><span class="cit-name-surname">Doerfler</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-ed"><span class="cit-name-surname">Böhm</span>  <span class="cit-name-given-names">P</span></span>, editors. <span class="cit-source">The Molecular Repertoire of Adenoviruses I: Virion Structure and Infection</span>. <span class="cit-publ-loc">Berlin, Heidelberg</span>: <span class="cit-publ-name">Springer</span>; <span class="cit-pub-date">1995</span>. pp. <span class="cit-fpage">237</span>–<span class="cit-lpage">274</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/978-3-642-79496-4_13</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BMolecular%2BRepertoire%2Bof%2BAdenoviruses%2BI%253A%2BVirion%2BStructure%2Band%2BInfection%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-642-79496-4_13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-642-79496-4_13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bridge</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ketner</span>  <span class="cit-name-given-names">G</span></span>. <span class="cit-article-title">Redundant control of adenovirus late gene expression by early region 4</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">1989</span>;<span class="cit-vol">63</span>: <span class="cit-fpage">631</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DBridge%26rft.auinit1%253DE%26rft.volume%253D63%26rft.issue%253D2%26rft.spage%253D631%26rft.epage%253D638%26rft.atitle%253DRedundant%2Bcontrol%2Bof%2Badenovirus%2Blate%2Bgene%2Bexpression%2Bby%2Bearly%2Bregion%2B4.%26rft_id%253Dinfo%253Apmid%252F2911117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjYzLzIvNjMxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTMvMjAxOS4xMi4xMy44NzYwMzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">32.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.13.876037v1.32" data-doi="10.2741/1604"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weitzman</span>  <span class="cit-name-given-names">MD</span></span>. <span class="cit-article-title">Functions of the adenovirus E4 proteins and their impact on viral vectors</span>. <abbr class="cit-jnl-abbrev">Front Biosci</abbr>. <span class="cit-pub-date">2005</span>;<span class="cit-vol">10</span>: <span class="cit-fpage">1106</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2Bbioscience%2B%253A%2B%2Ba%2Bjournal%2Band%2Bvirtual%2Blibrary%26rft.stitle%253DFront%2BBiosci%26rft.aulast%253DWeitzman%26rft.auinit1%253DM.%2BD.%26rft.volume%253D10%26rft.spage%253D1106%26rft.epage%253D1117%26rft.atitle%253DFunctions%2Bof%2Bthe%2Badenovirus%2BE4%2Bproteins%2Band%2Btheir%2Bimpact%2Bon%2Bviral%2Bvectors.%26rft_id%253Dinfo%253Adoi%252F10.2741%252F1604%26rft_id%253Dinfo%253Apmid%252F15769610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2741/1604&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15769610&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.33" data-doi="10.1038/sj.onc.1209063"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weitzman</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ornelles</span>  <span class="cit-name-given-names">DA</span></span>. <span class="cit-article-title">Inactivating intracellular antiviral responses during adenovirus infection</span>. <abbr class="cit-jnl-abbrev">Oncogene</abbr>. <span class="cit-pub-date">2005</span>;<span class="cit-vol">24</span>: <span class="cit-fpage">7686</span>–<span class="cit-lpage">96</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/sj.onc.1209063</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncogene%26rft.stitle%253DOncogene%26rft.aulast%253DWeitzman%26rft.auinit1%253DM.%2BD.%26rft.volume%253D24%26rft.issue%253D52%26rft.spage%253D7686%26rft.epage%253D7696%26rft.atitle%253DInactivating%2Bintracellular%2Bantiviral%2Bresponses%2Bduring%2Badenovirus%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.onc.1209063%26rft_id%253Dinfo%253Apmid%252F16299529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.onc.1209063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16299529&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233372600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.34" data-doi="10.1073/pnas.71.9.3426"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weinmann</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raskas</span>  <span class="cit-name-given-names">HJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">RG</span></span>. <span class="cit-article-title">Role of DNA-dependent RNA polymerases II and III in transcription of the adenovirus genome late in productive infection</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr>. <span class="cit-pub-date">1974</span>;<span class="cit-vol">71</span>: <span class="cit-fpage">3426</span>–<span class="cit-lpage">3439</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.71.9.3426</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWeinmann%26rft.auinit1%253DR.%26rft.volume%253D71%26rft.issue%253D9%26rft.spage%253D3426%26rft.epage%253D3430%26rft.atitle%253DRole%2Bof%2BDNA-Dependent%2BRNA%2BPolymerases%2BII%2Band%2BIII%2Bin%2BTranscription%2Bof%2Bthe%2BAdenovirus%2BGenome%2BLate%2Bin%2BProductive%2BInfection%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.71.9.3426%26rft_id%253Dinfo%253Apmid%252F4530313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3MS85LzM0MjYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8xMy8yMDE5LjEyLjEzLjg3NjAzNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.35" data-doi="10.1016/j.virusres.2015.06.018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vachon</span>  <span class="cit-name-given-names">VK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Conn</span>  <span class="cit-name-given-names">GL</span></span>. <span class="cit-article-title">Adenovirus VA RNA: An essential pro-viral non-coding RNA</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">212</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2015.06.018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2015.06.018%26rft_id%253Dinfo%253Apmid%252F26116898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2015.06.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26116898&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.36" data-doi="10.1101/cshperspect.a013003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoeben</span>  <span class="cit-name-given-names">RC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uil</span>  <span class="cit-name-given-names">TG</span></span>. <span class="cit-article-title">Adenovirus DNA Replication</span>. <abbr class="cit-jnl-abbrev">Cold Spring Harb Perspect Biol</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/cshperspect.a013003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb%2BPerspect%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a013003%26rft_id%253Dinfo%253Apmid%252F23388625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiI1LzMvYTAxMzAwMyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzEzLzIwMTkuMTIuMTMuODc2MDM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.37" data-doi="10.1016/0092-8674(80)90568-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziff</span>  <span class="cit-name-given-names">EB</span></span>. <span class="cit-article-title">Transcripts from the adenovirus-2 major late promoter yield a single early family of 3’ coterminal mRNAs and five late families</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>. <span class="cit-pub-date">1980</span>;<span class="cit-vol">22</span>: <span class="cit-fpage">905</span>–<span class="cit-lpage">916</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/0092-8674(80)90568-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DShaw%26rft.auinit1%253DA.%2BR.%26rft.volume%253D22%26rft.issue%253D3%26rft.spage%253D905%26rft.epage%253D916%26rft.atitle%253DTranscripts%2Bfrom%2Bthe%2Badenovirus-2%2Bmajor%2Blate%2Bpromoter%2Byield%2Ba%2Bsingle%2Bearly%2Bfamily%2Bof%2B3%2527%2Bcoterminal%2BmRNAs%2Band%2Bfive%2Blate%2Bfamilies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252880%252990568-1%26rft_id%253Dinfo%253Apmid%252F7460018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(80)90568-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7460018&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.38" data-doi="10.1016/j.ymthe.2004.09.018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Parks</span>  <span class="cit-name-given-names">RJ</span></span>. <span class="cit-article-title">Adenovirus protein IX: a new look at an old protein</span>. <abbr class="cit-jnl-abbrev">Mol Ther</abbr>. <span class="cit-pub-date">2005</span>;<span class="cit-vol">11</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.ymthe.2004.09.018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Btherapy%2B%253A%2B%2Bthe%2Bjournal%2Bof%2Bthe%2BAmerican%2BSociety%2Bof%2BGene%2BTherapy%26rft.stitle%253DMol%2BTher%26rft.aulast%253DParks%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D25%26rft.atitle%253DAdenovirus%2Bprotein%2BIX%253A%2Ba%2Bnew%2Blook%2Bat%2Ban%2Bold%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymthe.2004.09.018%26rft_id%253Dinfo%253Apmid%252F15585402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymthe.2004.09.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15585402&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.39" data-doi="10.1128/JVI.77.6.3586-3594.2003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Imperiale</span>  <span class="cit-name-given-names">MJ</span></span>. <span class="cit-article-title">Requirement of the adenovirus IVa2 protein for virus assembly</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2003</span>;<span class="cit-vol">77</span>: <span class="cit-fpage">3586</span>–<span class="cit-lpage">3594</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.77.6.3586-3594.2003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DZhang%26rft.auinit1%253DW.%26rft.volume%253D77%26rft.issue%253D6%26rft.spage%253D3586%26rft.epage%253D3594%26rft.atitle%253DRequirement%2Bof%2Bthe%2BAdenovirus%2BIVa2%2BProtein%2Bfor%2BVirus%2BAssembly%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.77.6.3586-3594.2003%26rft_id%253Dinfo%253Apmid%252F12610134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc3LzYvMzU4NiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzEzLzIwMTkuMTIuMTMuODc2MDM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">TH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ginsberg</span>  <span class="cit-name-given-names">HS</span></span>. <span class="cit-article-title">Viral transcription in KB cells infected by temperature-sensitive “early” mutants of adenovirus type 5</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">1976</span>;<span class="cit-vol">18</span>: <span class="cit-fpage">156</span>–<span class="cit-lpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DCarter%26rft.auinit1%253DT%2BH%26rft.volume%253D18%26rft.issue%253D1%26rft.spage%253D156%26rft.epage%253D166%26rft.atitle%253DViral%2Btranscription%2Bin%2BKB%2Bcells%2Binfected%2Bby%2Btemperature-sensitive%2B%2522early%2522%2Bmutants%2Bof%2Badenovirus%2Btype%2B5.%26rft_id%253Dinfo%253Apmid%252F1255869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjE4LzEvMTU2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTMvMjAxOS4xMi4xMy44NzYwMzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">41.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.13.876037v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crossland</span>  <span class="cit-name-given-names">LD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raskas</span>  <span class="cit-name-given-names">HJ</span></span>. <span class="cit-article-title">Identification of adenovirus genes that require template replication for expression</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">1983</span>;<span class="cit-vol">46</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">748</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DCrossland%26rft.auinit1%253DL%2BD%26rft.volume%253D46%26rft.issue%253D3%26rft.spage%253D737%26rft.epage%253D748%26rft.atitle%253DIdentification%2Bof%2Badenovirus%2Bgenes%2Bthat%2Brequire%2Btemplate%2Breplication%2Bfor%2Bexpression.%26rft_id%253Dinfo%253Apmid%252F6854739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjQ2LzMvNzM3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTMvMjAxOS4xMi4xMy44NzYwMzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.42" data-doi="10.1016/0092-8674(80)90362-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">GP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mathews</span>  <span class="cit-name-given-names">MB</span></span>. <span class="cit-article-title">DNA replication and the early to late transition in adenovirus infection</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>. <span class="cit-pub-date">1980</span>;<span class="cit-vol">22</span>: <span class="cit-fpage">523</span>–<span class="cit-lpage">533</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/0092-8674(80)90362-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DThomas%26rft.auinit1%253DG.%2BP.%26rft.volume%253D22%26rft.issue%253D2%2BPt%2B2%26rft.spage%253D523%26rft.epage%253D533%26rft.atitle%253DDNA%2Breplication%2Band%2Bthe%2Bearly%2Bto%2Blate%2Btransition%2Bin%2Badenovirus%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252880%252990362-1%26rft_id%253Dinfo%253Apmid%252F7448872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(80)90362-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7448872&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980KU63800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.43" data-doi="10.1038/s41598-017-00747-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ramke</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">JY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dyer</span>  <span class="cit-name-given-names">DW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seto</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rajaiya</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chodosh</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-article-title">The 5’UTR in human adenoviruses: leader diversity in late gene expression</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">7</span>: <span class="cit-fpage">618</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41598-017-00747-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D7%26rft.spage%253D618%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-00747-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-017-00747-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Soloway</span>  <span class="cit-name-given-names">PD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shenk</span>  <span class="cit-name-given-names">T</span></span>. <span class="cit-article-title">The adenovirus type 5 i-leader open reading frame functions in cis to reduce the half-life of L1 mRNAs</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">1990</span>;<span class="cit-vol">64</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DSoloway%26rft.auinit1%253DP%2BD%26rft.volume%253D64%26rft.issue%253D2%26rft.spage%253D551%26rft.epage%253D558%26rft.atitle%253DThe%2Badenovirus%2Btype%2B5%2Bi-leader%2Bopen%2Breading%2Bframe%2Bfunctions%2Bin%2Bcis%2Bto%2Breduce%2Bthe%2Bhalf-life%2Bof%2BL1%2BmRNAs.%26rft_id%253Dinfo%253Apmid%252F2296076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjY0LzIvNTUxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTMvMjAxOS4xMi4xMy44NzYwMzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.45" data-doi="10.1128/JVI.00107-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">GE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leppard</span>  <span class="cit-name-given-names">KN</span></span>. <span class="cit-article-title">Adenovirus Late-Phase Infection Is Controlled by a Novel L4 Promoter</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">84</span>: <span class="cit-fpage">7096</span>–<span class="cit-lpage">7104</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.00107-10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00107-10%26rft_id%253Dinfo%253Apmid%252F20444889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4NC8xNC83MDk2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTMvMjAxOS4xMi4xMy44NzYwMzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.46" data-doi="10.3390/ijms16022893"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Biasiotto</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Akusjärvi</span>  <span class="cit-name-given-names">G</span></span>. <span class="cit-article-title">Regulation of Human Adenovirus Alternative RNA Splicing by the Adenoviral L4-33K and L4-22K Proteins</span>. <abbr class="cit-jnl-abbrev">Int J Mol Sci</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">16</span>: <span class="cit-fpage">2893</span>–<span class="cit-lpage">2912</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3390/ijms16022893</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BMol%2BSci%26rft.volume%253D16%26rft.spage%253D2893%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fijms16022893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/ijms16022893&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.47" data-doi="10.1128/JVI.01531-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tollefson</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ying</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doronin</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sidor</span>  <span class="cit-name-given-names">PD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">WSM</span></span>. <span class="cit-article-title">Identification of a New Human Adenovirus Protein Encoded by a Novel Late l-Strand Transcription Unit</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>. <span class="cit-pub-date">2007</span>;<span class="cit-vol">81</span>: <span class="cit-fpage">12918</span>–<span class="cit-lpage">12926</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.01531-07</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01531-07%26rft_id%253Dinfo%253Apmid%252F17881437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MS8yMy8xMjkxOCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzEzLzIwMTkuMTIuMTMuODc2MDM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.48" data-doi="10.1128/JVI.01338-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ying</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tollefson</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">WSM</span></span>. <span class="cit-article-title">Identification of a Previously Unrecognized Promoter That Drives Expression of the UXP Transcription Unit in the Human Adenovirus Type 5 Genome</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">84</span>: <span class="cit-fpage">11470</span>–<span class="cit-lpage">11478</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.01338-10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01338-10%26rft_id%253Dinfo%253Apmid%252F20739525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4NC8yMS8xMTQ3MCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzEzLzIwMTkuMTIuMTMuODc2MDM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.49" data-doi="10.1186/gb-2007-8-4-r58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">DL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">CL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rickards</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coller</span>  <span class="cit-name-given-names">HA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">SJ</span></span>. <span class="cit-article-title">Adenovirus type 5 exerts genome-wide control over cellular programs governing proliferation, quiescence, and survival</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr>. <span class="cit-pub-date">2007</span>;<span class="cit-vol">8</span>: <span class="cit-fpage">R58</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/gb-2007-8-4-r58</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DMiller%26rft.auinit1%253DD.%2BL.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253DR58%26rft.epage%253DR58%26rft.atitle%253DAdenovirus%2Btype%2B5%2Bexerts%2Bgenome-wide%2Bcontrol%2Bover%2Bcellular%2Bprograms%2Bgoverning%2Bproliferation%252C%2Bquiescence%252C%2Band%2Bsurvival.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-4-r58%26rft_id%253Dinfo%253Apmid%252F17430596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-4-r58&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17430596&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.50" data-doi="10.1128/JVI.02269-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">DL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rickards</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mashiba</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">SJ</span></span>. <span class="cit-article-title">The adenoviral E1B 55-kilodalton protein controls expression of immune response genes but not p53-dependent transcription</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">83</span>: <span class="cit-fpage">3591</span>–<span class="cit-lpage">3603</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.02269-08</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02269-08%26rft_id%253Dinfo%253Apmid%252F19211769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgzLzgvMzU5MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzEzLzIwMTkuMTIuMTMuODc2MDM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.51" data-doi="10.1016/j.virol.2011.12.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dahlö</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Isaksson</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Syvänen</span>  <span class="cit-name-given-names">A-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pettersson</span>  <span class="cit-name-given-names">U</span></span>. <span class="cit-article-title">The transcriptome of the adenovirus infected cell</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">424</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">128</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2011.12.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DVirology%26rft.aulast%253DZhao%26rft.auinit1%253DH.%26rft.volume%253D424%26rft.issue%253D2%26rft.spage%253D115%26rft.epage%253D128%26rft.atitle%253DThe%2Btranscriptome%2Bof%2Bthe%2Badenovirus%2Binfected%2Bcell.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2011.12.006%26rft_id%253Dinfo%253Apmid%252F22236370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2011.12.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22236370&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.52" data-doi="10.1186/s12866-018-1375-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valdés</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lind</span>  <span class="cit-name-given-names">SB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pettersson</span>  <span class="cit-name-given-names">U</span></span>. <span class="cit-article-title">Transcriptomic and proteomic analyses reveal new insights into the regulation of immune pathways during adenovirus type 2 infection</span>. <abbr class="cit-jnl-abbrev">BMC Microbiol</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">19</span>: <span class="cit-fpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/s12866-018-1375-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BMicrobiol%26rft.volume%253D19%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12866-018-1375-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12866-018-1375-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.53" data-doi="10.1016/j.virol.2014.04.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pettersson</span>  <span class="cit-name-given-names">U</span></span>. <span class="cit-article-title">A new look at adenovirus splicing</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">456–457</span>: <span class="cit-fpage">329</span>–<span class="cit-lpage">341</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2014.04.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D456457%26rft.spage%253D329%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2014.04.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2014.04.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.54" data-doi="10.1371/journal.pone.0211192"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crisostomo</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soriano</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mendez</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graves</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pelka</span>  <span class="cit-name-given-names">P</span></span>. <span class="cit-article-title">Temporal dynamics of adenovirus 5 gene expression in normal human cells</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">14</span>: <span class="cit-fpage">e0211192</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0211192</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D14%26rft.spage%253De0211192%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0211192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0211192&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.55" data-doi="10.1186/s13062-016-0128-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brandes</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Linial</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-article-title">Gene overlapping and size constraints in the viral world</span>. <abbr class="cit-jnl-abbrev">Biology Direct</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">11</span>: <span class="cit-fpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/s13062-016-0128-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiology%2BDirect%26rft.volume%253D11%26rft.spage%253D26%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13062-016-0128-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13062-016-0128-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.56" data-doi="10.1093/nar/gkw629"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">O’Grady</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Höner zu Bentrup</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baddoo</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Concha</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Flemington</span>  <span class="cit-name-given-names">EK</span></span>. <span class="cit-article-title">Global transcript structure resolution of high gene density genomes through multi-platform data integration</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">44</span>: <span class="cit-fpage">e145</span>–<span class="cit-lpage">e145</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkw629</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw629%26rft_id%253Dinfo%253Apmid%252F27407110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw629&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27407110&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.57" data-doi="10.1038/s41467-019-08734-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Depledge</span>  <span class="cit-name-given-names">DP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Srinivas</span>  <span class="cit-name-given-names">KP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sadaoka</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bready</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mori</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Placantonakis</span>  <span class="cit-name-given-names">DG</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Direct RNA sequencing on nanopore arrays redefines the transcriptional complexity of a viral pathogen</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41467-019-08734-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-019-08734-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-019-08734-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">58.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.13.876037v1.58" data-doi="10.1038/srep43751"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tombácz</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balázs</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Csabai</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moldován</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Szűcs</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharon</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Characterization of the Dynamic Transcriptome of a Herpesvirus with Long-read Single Molecule Real-Time Sequencing</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">7</span>: <span class="cit-fpage">43751</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/srep43751</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep43751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep43751&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.59" data-doi="10.1101/gr.247064.118"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Viehweger</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krautwurst</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lamkiewicz</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madhugiri</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hölzer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Direct RNA nanopore sequencing of full-length coronavirus genomes provides novel insights into structural variants and enables modification analysis</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">29</span>: <span class="cit-fpage">1545</span>–<span class="cit-lpage">1554</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/gr.247064.118</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.247064.118%26rft_id%253Dinfo%253Apmid%252F31439691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI5LzkvMTU0NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzEzLzIwMTkuMTIuMTMuODc2MDM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.60" data-doi="10.12688/f1000research.10571.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weirather</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Cesare</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Piazza</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sebastiano</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X-J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Comprehensive comparison of Pacific Biosciences and Oxford Nanopore Technologies and their applications to transcriptome analysis</span>. <abbr class="cit-jnl-abbrev">F1000Res</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">6</span>: <span class="cit-fpage">100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.12688/f1000research.10571.2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF1000Res%26rft.volume%253D6%26rft.spage%253D100%26rft_id%253Dinfo%253Adoi%252F10.12688%252Ff1000research.10571.2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12688/f1000research.10571.2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.61" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The variant call format and VCFtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btr330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.62" data-doi="10.1093/bioinformatics/btw044"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Narasimhan</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scally</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tyler-Smith</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R</span></span>. <span class="cit-article-title">BCFtools/RoH: a hidden Markov model approach for detecting autozygosity from next-generation sequencing data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">32</span>: <span class="cit-fpage">1749</span>–<span class="cit-lpage">1751</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btw044</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw044%26rft_id%253Dinfo%253Apmid%252F26826718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26826718&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.63" data-doi="10.1038/nmeth.4110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">C</span></span>. <span class="cit-article-title">Epitranscriptome sequencing technologies: decoding RNA modifications</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">14</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">31</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.4110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D14%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.64" data-doi="10.1038/nmeth.4577"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garalde</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snell</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jachimowicz</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sipos</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bruce</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Highly parallel direct RNA sequencing on an array of nanopores</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">15</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">206</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.4577</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4577%26rft_id%253Dinfo%253Apmid%252F29334379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29334379&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.65" data-doi="10.1007/978-1-4939-3578-9_15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">TD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reeder</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brauer</span>  <span class="cit-name-given-names">MJ</span></span>. <span class="cit-article-title">GMAP and GSNAP for Genomic Sequence Alignment: Enhancements to Speed, Accuracy, and Functionality</span>. <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">1418</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">334</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/978-1-4939-3578-9_15</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol%2BBiol%26rft.volume%253D1418%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-3578-9_15%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-3578-9_15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.66" data-doi="10.1093/bioinformatics/bty191"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>. <span class="cit-article-title">Minimap2: pairwise alignment for nucleotide sequences</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">34</span>: <span class="cit-fpage">3094</span>–<span class="cit-lpage">3100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/bty191</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D3094%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty191%26rft_id%253Dinfo%253Apmid%252F29750242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29750242&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.67" data-doi="10.1371/journal.pone.0170914"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bonfert</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Friedel</span>  <span class="cit-name-given-names">CC</span></span>. <span class="cit-article-title">Prediction of Poly(A) Sites by Poly(A) Read Mapping</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">12</span>: <span class="cit-fpage">e0170914</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0170914</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D12%26rft.spage%253De0170914%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0170914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0170914&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.68" data-doi="10.1261/rna.1006608"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Borodulina</span>  <span class="cit-name-given-names">OR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kramerov</span>  <span class="cit-name-given-names">DA</span></span>. <span class="cit-article-title">Transcripts synthesized by RNA polymerase III can be polyadenylated in an AAUAAA-dependent manner</span>. <abbr class="cit-jnl-abbrev">RNA</abbr>. <span class="cit-pub-date">2008</span>;<span class="cit-vol">14</span>: <span class="cit-fpage">1865</span>–<span class="cit-lpage">1873</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1261/rna.1006608</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.1006608%26rft_id%253Dinfo%253Apmid%252F18658125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjE0LzkvMTg2NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzEzLzIwMTkuMTIuMTMuODc2MDM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.69" data-doi="10.1101/410183"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soulette</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Baren MJ</span> <span class="cit-name-surname">van</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hart</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hrabeta-Robinson</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">CJ</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Full-length transcript characterization of SF3B1 mutation in chronic lymphocytic leukemia reveals downregulation of retained introns</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>. <span class="cit-pub-date">2018</span>; <span class="cit-fpage">410183</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/410183</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F410183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI0MTAxODN2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzEzLzIwMTkuMTIuMTMuODc2MDM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.70" data-doi="10.1128/JVI.00033-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hidalgo</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anzures</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hernández-Mendoza</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guerrero</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valdés</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Morphological, Biochemical, and Functional Study of Viral Replication Compartments Isolated from Adenovirus-Infected Cells</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">90</span>: <span class="cit-fpage">3411</span>–<span class="cit-lpage">3427</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.00033-16</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00033-16%26rft_id%253Dinfo%253Apmid%252F26764008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjkwLzcvMzQxMSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzEzLzIwMTkuMTIuMTMuODc2MDM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">71.</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.71" data-doi="10.1126/science.1227919"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stern-Ginossar</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weisburd</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Michalski</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">VTK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hein</span>  <span class="cit-name-given-names">MY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S-X</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Decoding human cytomegalovirus</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">338</span>: <span class="cit-fpage">1088</span>–<span class="cit-lpage">1093</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1227919</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DStern-Ginossar%26rft.auinit1%253DN.%26rft.volume%253D338%26rft.issue%253D6110%26rft.spage%253D1088%26rft.epage%253D1093%26rft.atitle%253DDecoding%2BHuman%2BCytomegalovirus%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1227919%26rft_id%253Dinfo%253Apmid%252F23180859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzgvNjExMC8xMDg4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTMvMjAxOS4xMi4xMy44NzYwMzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">72.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.13.876037v1.72" data-doi="10.1016/j.chom.2012.06.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arvey</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tempera</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H-S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tikhmyanova</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klichinsky</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">An Atlas of the Epstein-Barr Virus Transcriptome and Epigenome Reveals Host-Virus Regulatory Interactions</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">12</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">245</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.chom.2012.06.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BHost%2BMicrobe%26rft.aulast%253DArvey%26rft.auinit1%253DA.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D233%26rft.epage%253D245%26rft.atitle%253DAn%2Batlas%2Bof%2Bthe%2BEpstein-Barr%2Bvirus%2Btranscriptome%2Band%2Bepigenome%2Breveals%2Bhost-virus%2Bregulatory%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2012.06.008%26rft_id%253Dinfo%253Apmid%252F22901543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2012.06.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22901543&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308265900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">73.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.13.876037v1.73" data-doi="10.1371/journal.ppat.1003847"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arias</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weisburd</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stern-Ginossar</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mercier</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madrid</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bellare</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">KSHV 2.0: A Comprehensive Annotation of the Kaposi’s Sarcoma-Associated Herpesvirus Genome Using Next-Generation Sequencing Reveals Novel Genomic and Functional Features</span>. <abbr class="cit-jnl-abbrev">PLOS Pathogens</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">10</span>: <span class="cit-fpage">e1003847</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1003847</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BPathogens%26rft.volume%253D10%26rft.spage%253De1003847%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003847%26rft_id%253Dinfo%253Apmid%252F24453964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24453964&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">74.</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.74" data-doi="10.1038/ncomms8126"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rutkowski</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Erhard</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">L’Hernault</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonfert</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schilhabel</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crump</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Widespread disruption of host transcription termination in HSV-1 infection</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">6</span>: <span class="cit-fpage">7126</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/ncomms8126</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D6%26rft.spage%253D7126%26rft.atitle%253DWIDESPREAD%2BDISRUPTION%2BOF%2BHOST%2BTRANSCRIPTION%2BTERMINATION%2BIN%2BHSV-1%2BINFECTION%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms8126%26rft_id%253Dinfo%253Apmid%252F25989971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms8126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25989971&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">75.</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.75" data-doi="10.1371/journal.ppat.1005166"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garren</span>  <span class="cit-name-given-names">SB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kondaveeti</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duff</span>  <span class="cit-name-given-names">MO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carmichael</span>  <span class="cit-name-given-names">GG</span></span>. <span class="cit-article-title">Global Analysis of Mouse Polyomavirus Infection Reveals Dynamic Regulation of Viral and Host Gene Expression and Promiscuous Viral RNA Editing</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">11</span>: <span class="cit-fpage">e1005166</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1005166</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D11%26rft.spage%253De1005166%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1005166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1005166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">76.</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76. in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.76" data-doi="10.1038/nmeth.2227"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">VC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heesom</span>  <span class="cit-name-given-names">KJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bessant</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matthews</span>  <span class="cit-name-given-names">DA</span></span>. <span class="cit-article-title">De novo derivation of proteomes from transcriptomes for transcript and protein identification</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">9</span>: <span class="cit-fpage">1207</span>–<span class="cit-lpage">1211</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.2227</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DEvans%26rft.volume%253D9%26rft.spage%253D1207%26rft.atitle%253DDe%2Bnovo%2Bderivation%2Bof%2Bproteomes%2Bfrom%2Btranscriptomes%2Bfor%2Btranscript%2Band%2Bprotein%2Bidentification.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2227%26rft_id%253Dinfo%253Apmid%252F23142869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2227&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23142869&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">77.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.77" data-doi="10.1128/JVI.01924-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leppard</span>  <span class="cit-name-given-names">KN</span></span>. <span class="cit-article-title">The Human Adenovirus 5 L4 Promoter Is Activated by Cellular Stress Response Protein p53</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">87</span>: <span class="cit-fpage">11617</span>–<span class="cit-lpage">11625</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.01924-13</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01924-13%26rft_id%253Dinfo%253Apmid%252F23966406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Ny8yMS8xMTYxNyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzEzLzIwMTkuMTIuMTMuODc2MDM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">78.</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78. in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.78" data-doi="10.1128/JVI.00683-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atwan</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leppard</span>  <span class="cit-name-given-names">KN</span></span>. <span class="cit-article-title">The Human Adenovirus Type 5 L4 Promoter Is Negatively Regulated by TFII-I and L4-33K</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">89</span>: <span class="cit-fpage">7053</span>–<span class="cit-lpage">7063</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.00683-15</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00683-15%26rft_id%253Dinfo%253Apmid%252F25926634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OS8xNC83MDUzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTMvMjAxOS4xMi4xMy44NzYwMzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">79.</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79. in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.79" data-doi="10.1371/journal.ppat.1006954"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hennig</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Michalski</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rutkowski</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Djakovic</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whisnant</span>  <span class="cit-name-given-names">AW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Friedl</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">HSV-1-induced disruption of transcription termination resembles a cellular stress response but selectively increases chromatin accessibility downstream of genes</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">14</span>: <span class="cit-fpage">e1006954</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1006954</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D14%26rft.spage%253De1006954%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006954&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">80.</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80. in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.80" data-doi="10.1093/nar/gkw257"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ramírez</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ryan</span>  <span class="cit-name-given-names">DP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grüning</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bhardwaj</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kilpert</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">deepTools2: a next generation web server for deep-sequencing data analysis</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">44</span>: <span class="cit-fpage">W160</span>–<span class="cit-lpage">165</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkw257</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw257%26rft_id%253Dinfo%253Apmid%252F27079975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw257&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27079975&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">81.</span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference 81. in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.81" data-doi="10.1101/436634"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Y-Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramu</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cotto</span>  <span class="cit-name-given-names">KC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skidmore</span>  <span class="cit-name-given-names">ZL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kunisaki</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">DF</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">RegTools: Integrated analysis of genomic and transcriptomic data for discovery of splicing variants in cancer</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>. <span class="cit-pub-date">2018</span>; <span class="cit-fpage">436634</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/436634</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F436634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI0MzY2MzR2MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzEzLzIwMTkuMTIuMTMuODc2MDM3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">82.</span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference 82. in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.82" data-doi="10.1038/nbt.1754"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thorvaldsdóttir</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Winckler</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">ES</span></span>, <span class="cit-auth"><span class="cit-name-surname">Getz</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Integrative Genomics Viewer</span>. <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">29</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nbt.1754</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D26%26rft.atitle%253DIntegrative%2Bgenomics%2Bviewer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1754%26rft_id%253Dinfo%253Apmid%252F21221095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21221095&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286048900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">83.</span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference 83. in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.83" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Homer</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">84.</span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference 84. in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.84" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">IM</span></span>. <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btq033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">85.</span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference 85. in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.13.876037v1.85" data-doi="10.1016/j.molcel.2010.05.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heinz</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benner</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spann</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bertolino</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">YC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laslo</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities</span>. <abbr class="cit-jnl-abbrev">Mol Cell</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">38</span>: <span class="cit-fpage">576</span>–<span class="cit-lpage">589</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.molcel.2010.05.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DHeinz%26rft.auinit1%253DS.%26rft.volume%253D38%26rft.issue%253D4%26rft.spage%253D576%26rft.epage%253D589%26rft.atitle%253DSimple%2Bcombinations%2Bof%2Blineage-determining%2Btranscription%2Bfactors%2Bprime%2Bcis-regulatory%2Belements%2Brequired%2Bfor%2Bmacrophage%2Band%2BB%2Bcell%2Bidentities.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.05.004%26rft_id%253Dinfo%253Apmid%252F20513432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.05.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20513432&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.13.876037.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278448100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2019.12.13.876037v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2019.12.13.876037v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2019.12.13.875971v1" title="Structure and function of a neocortical synapse" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;December 13, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2019.12.13.876037v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2019/12/13/2019.12.13.876037.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2019.12.13.876037v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2019.12.13.876037v1" class="active" data-icon-position="" data-hide-link-title="0">Novel splicing and open reading frames revealed by long-read direct RNA sequencing of adenovirus transcripts</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1036558" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-gQVbCdY-3DBHgXPE4bt113zsr30r6_mdAWgZtiugf8s" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1036558" id="node1036558--2" data-pisa="biorxiv;2019.12.13.876037v1" data-pisa-master="biorxiv;2019.12.13.876037" data-seqnum="1036558" data-apath="/biorxiv/early/2019/12/13/2019.12.13.876037.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Novel splicing and open reading frames revealed by long-read direct RNA sequencing of adenovirus transcripts</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Alexander M.</span> <span class="nlm-surname">Price</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Katharina E.</span> <span class="nlm-surname">Hayer</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Daniel P.</span> <span class="nlm-surname">Depledge</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Angus C.</span> <span class="nlm-surname">Wilson</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Matthew D.</span> <span class="nlm-surname">Weitzman</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.13.876037; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.13.876037 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2019.12.13.876037v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.876037v1&amp;title=Novel%20splicing%20and%20open%20reading%20frames%20revealed%20by%20long-read%20direct%20RNA%20sequencing%20of%20adenovirus%20transcripts" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.876037v1&amp;title=Novel%20splicing%20and%20open%20reading%20frames%20revealed%20by%20long-read%20direct%20RNA%20sequencing%20of%20adenovirus%20transcripts" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.876037v1&amp;text=Novel%20splicing%20and%20open%20reading%20frames%20revealed%20by%20long-read%20direct%20RNA%20sequencing%20of%20adenovirus%20transcripts" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.876037v1&amp;title=Novel%20splicing%20and%20open%20reading%20frames%20revealed%20by%20long-read%20direct%20RNA%20sequencing%20of%20adenovirus%20transcripts" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.876037v1&amp;t=Novel%20splicing%20and%20open%20reading%20frames%20revealed%20by%20long-read%20direct%20RNA%20sequencing%20of%20adenovirus%20transcripts" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.876037v1&amp;title=Novel%20splicing%20and%20open%20reading%20frames%20revealed%20by%20long-read%20direct%20RNA%20sequencing%20of%20adenovirus%20transcripts" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.876037v1&amp;title=Novel%20splicing%20and%20open%20reading%20frames%20revealed%20by%20long-read%20direct%20RNA%20sequencing%20of%20adenovirus%20transcripts" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1036558" id="node1036558--3" data-pisa="biorxiv;2019.12.13.876037v1" data-pisa-master="biorxiv;2019.12.13.876037" data-seqnum="1036558" data-apath="/biorxiv/early/2019/12/13/2019.12.13.876037.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Novel splicing and open reading frames revealed by long-read direct RNA sequencing of adenovirus transcripts</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Alexander M.</span> <span class="nlm-surname">Price</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Katharina E.</span> <span class="nlm-surname">Hayer</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Daniel P.</span> <span class="nlm-surname">Depledge</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Angus C.</span> <span class="nlm-surname">Wilson</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Matthew D.</span> <span class="nlm-surname">Weitzman</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.13.876037; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.13.876037 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Novel%20splicing%20and%20open%20reading%20frames%20revealed%20by%20long-read%20direct%20RNA%20sequencing%20of%20adenovirus%20transcripts&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Price%2C+Alexander+M.&amp;rft.au=Hayer%2C+Katharina+E.&amp;rft.au=Depledge%2C+Daniel+P.&amp;rft.au=Wilson%2C+Angus+C.&amp;rft.au=Weitzman%2C+Matthew+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1036558/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1036558/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1036558/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1036558/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1036558/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1036558/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1036558/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1036558/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1036558/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1036558/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1036558/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1036558/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.876037v1&amp;count=horizontal&amp;via=&amp;text=Novel%20splicing%20and%20open%20reading%20frames%20revealed%20by%20long-read%20direct%20RNA%20sequencing%20of%20adenovirus%20transcripts&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.876037v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2019.12.13.876037v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2019.12.13.876037v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
