<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/31/2020.03.30.015008.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.30.015008v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Evolutionary origins of the SARS-CoV-2 sarbecovirus lineage responsible for the COVID-19 pandemic" />
<meta name="DC.Identifier" content="10.1101/2020.03.30.015008" />
<meta name="DC.Date" content="2020-03-31" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="There are outstanding evolutionary questions on the recent emergence of coronavirus SARS-CoV-2/hCoV-19 in Hubei province that caused the COVID-19 pandemic, including (1) the relationship of the new virus to the SARS-related coronaviruses, (2) the role of bats as a reservoir species, (3) the potential role of other mammals in the emergence event, and (4) the role of recombination in viral emergence. Here, we address these questions and find that the sarbecoviruses – the viral subgenus responsible for the emergence of SARS-CoV and SARS-CoV-2 – exhibit frequent recombination, but the SARS-CoV-2 lineage itself is not a recombinant of any viruses detected to date. In order to employ phylogenetic methods to date the divergence events between SARS-CoV-2 and the bat sarbecovirus reservoir, recombinant regions of a 68-genome sarbecovirus alignment were removed with three independent methods. Bayesian evolutionary rate and divergence date estimates were consistent for all three recombination-free alignments and robust to two different prior specifications based on HCoV-OC43 and MERS-CoV evolutionary rates. Divergence dates between SARS-CoV-2 and the bat sarbecovirus reservoir were estimated as 1948 (95% HPD: 1879-1999), 1969 (95% HPD: 1930-2000), and 1982 (95% HPD: 1948-2009). Despite intensified characterization of sarbecoviruses since SARS, the lineage giving rise to SARS-CoV-2 has been circulating unnoticed for decades in bats and been transmitted to other hosts such as pangolins. The occurrence of a third significant coronavirus emergence in 17 years together with the high prevalence and virus diversity in bats implies that these viruses are likely to cross species boundaries again.

In Brief The Betacoronavirus SARS-CoV-2 is a member of the sarbecovirus subgenus which shows frequent recombination in its evolutionary history. We characterize the extent of this genetic exchange and identify non-recombining regions of the sarbecovirus genome using three independent methods to remove the effects of recombination. Using these non-recombining genome regions and prior information on coronavirus evolutionary rates, we obtain estimates from three approaches that the most likely divergence date of SARS-CoV-2 from its most closely related available bat sequences ranges from 1948 to 1982.

Key Points" />
<meta name="DC.Contributor" content="Maciej F Boni" />
<meta name="DC.Contributor" content="Philippe Lemey" />
<meta name="DC.Contributor" content="Xiaowei Jiang" />
<meta name="DC.Contributor" content="Tommy Tsan-Yuk Lam" />
<meta name="DC.Contributor" content="Blair Perry" />
<meta name="DC.Contributor" content="Todd Castoe" />
<meta name="DC.Contributor" content="Andrew Rambaut" />
<meta name="DC.Contributor" content="David L Robertson" />
<meta name="article:published_time" content="2020-03-31" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Evolutionary origins of the SARS-CoV-2 sarbecovirus lineage responsible for the COVID-19 pandemic" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;There are outstanding evolutionary questions on the recent emergence of coronavirus SARS-CoV-2/hCoV-19 in Hubei province that caused the COVID-19 pandemic, including (1) the relationship of the new virus to the SARS-related coronaviruses, (2) the role of bats as a reservoir species, (3) the potential role of other mammals in the emergence event, and (4) the role of recombination in viral emergence. Here, we address these questions and find that the sarbecoviruses -- the viral subgenus responsible for the emergence of SARS-CoV and SARS-CoV-2 -- exhibit frequent recombination, but the SARS-CoV-2 lineage itself is not a recombinant of any viruses detected to date. In order to employ phylogenetic methods to date the divergence events between SARS-CoV-2 and the bat sarbecovirus reservoir, recombinant regions of a 68-genome sarbecovirus alignment were removed with three independent methods. Bayesian evolutionary rate and divergence date estimates were consistent for all three recombination-free alignments and robust to two different prior specifications based on HCoV-OC43 and MERS-CoV evolutionary rates. Divergence dates between SARS-CoV-2 and the bat sarbecovirus reservoir were estimated as 1948 (95% HPD: 1879-1999), 1969 (95% HPD: 1930-2000), and 1982 (95% HPD: 1948-2009). Despite intensified characterization of sarbecoviruses since SARS, the lineage giving rise to SARS-CoV-2 has been circulating unnoticed for decades in bats and been transmitted to other hosts such as pangolins. The occurrence of a third significant coronavirus emergence in 17 years together with the high prevalence and virus diversity in bats implies that these viruses are likely to cross species boundaries again.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.30.015008v1" />
<meta name="citation_id" content="2020.03.30.015008v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.30.015008v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.30.015008v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.30.015008v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.30.015008" />
<meta name="citation_num_pages" content="25" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.30.015008" />
<meta name="citation_author" content="Maciej F Boni" />
<meta name="citation_author_institution" content="Center for Infectious Disease Dynamics, Department of Biology, Pennsylvania State University" />
<meta name="citation_author_email" content="mfb9@psu.edu" />
<meta name="citation_author_email" content="philippe.lemey@kuleuven.be" />
<meta name="citation_author_email" content="a.rambaut@ed.ac.uk" />
<meta name="citation_author_email" content="David.L.Robertson@glasgow.ac.uk" />
<meta name="citation_author" content="Philippe Lemey" />
<meta name="citation_author_institution" content="Department of Microbiology, Immunology and Transplantation" />
<meta name="citation_author_email" content="mfb9@psu.edu" />
<meta name="citation_author_email" content="philippe.lemey@kuleuven.be" />
<meta name="citation_author_email" content="a.rambaut@ed.ac.uk" />
<meta name="citation_author_email" content="David.L.Robertson@glasgow.ac.uk" />
<meta name="citation_author" content="Xiaowei Jiang" />
<meta name="citation_author_institution" content="Xi’an Jiaotong-Liverpool University (XJTLU)" />
<meta name="citation_author" content="Tommy Tsan-Yuk Lam" />
<meta name="citation_author_institution" content="State Key Laboratory of Emerging Infectious Diseases, School of Public Health, The University of Hong Kong" />
<meta name="citation_author" content="Blair Perry" />
<meta name="citation_author_institution" content="Department of Biology, University of Texas Arlington" />
<meta name="citation_author" content="Todd Castoe" />
<meta name="citation_author_institution" content="Department of Biology, University of Texas Arlington" />
<meta name="citation_author" content="Andrew Rambaut" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh" />
<meta name="citation_author_email" content="mfb9@psu.edu" />
<meta name="citation_author_email" content="philippe.lemey@kuleuven.be" />
<meta name="citation_author_email" content="a.rambaut@ed.ac.uk" />
<meta name="citation_author_email" content="David.L.Robertson@glasgow.ac.uk" />
<meta name="citation_author" content="David L Robertson" />
<meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research (CVR)" />
<meta name="citation_author_email" content="mfb9@psu.edu" />
<meta name="citation_author_email" content="philippe.lemey@kuleuven.be" />
<meta name="citation_author_email" content="a.rambaut@ed.ac.uk" />
<meta name="citation_author_email" content="David.L.Robertson@glasgow.ac.uk" />
<meta name="citation_reference" content="Aiewsakun, P., Katzourakis, A., 2016. Time-Dependent Rate Phenomenon in Viruses. J Virol 90, 7184–7195. https://doi.org/10.1128/JVI.00593-16." />
<meta name="citation_reference" content="Anderson, K.G., 2020. nCoV-2019 codon usage and reservoir (not snakes v2). Virological.org. URL http://virological.org/t/339" />
<meta name="citation_reference" content="Anderson, K.G., Rambaut, A., Lipkin, W.I., Holmes, E.C., Garry, R.F., 2020. The proximal origin of SARS-CoV-2. Nat Med, published online, 2–4." />
<meta name="citation_reference" content="Boni, M.F., de Jong, M.D., van Doorn, H.R., Holmes, E.C., 2010. Guidelines for identifying homologous recombination events in influenza A virus. PLoS One 5, e10434." />
<meta name="citation_reference" content="Boni, M.F., Posada, D., Feldman, M.W., 2007. An exact nonparametric method for inferring mosaic structure in sequence triplets. Genetics 176, 1035–47. https://doi.org/10.1534/genetics.106.068874" />
<meta name="citation_reference" content="Boni, M.F., Zhou, Y., Taubenberger, J.K., Holmes, E.C., 2008. Homologous recombination is very rare or absent in human influenza A virus. J Virol 82, 4807–4811. https://doi.org/10.1128/JVI.02683-07" />
<meta name="citation_reference" content="Bruen, T.C., Philippe, H., Bryant, D., 2006. A simple and robust statistical test for detecting the presence of recombination. Genetics 172, 2665–2681. https://doi.org/10.1534/genetics.105.048975" />
<meta name="citation_reference" content="Bryant, D., Moulton, V., 2004. Neighbor-Net: An Agglomerative Method for the Construction of Phylogenetic Networks. Mol Biol Evol 21, 255–265. https://doi.org/10.1093/molbev/msh018" />
<meta name="citation_reference" content="Chernomor, O., Minh, B.Q., Forest, F., Klaere, S., Ingram, T., Henzinger, M., von Haeseler, A., 2015. Split diversity in constrained conservation prioritization using integer linear programming. Methods Ecol Evol 6, 83–91. https://doi.org/10.1111/2041-210X.12299" />
<meta name="citation_reference" content="Duchene, S., Holmes, E.C., Ho, S.Y.W., 2014. Analyses of evolutionary dynamics in viruses are hindered by a time-dependent bias in rate estimates. Proc R Soc L. B 281, 20140732." />
<meta name="citation_reference" content="Duchene, S., Lemey, P., Stadler, T., Ho, S.Y.W., Duchene, D., Dhanasekaran, V., Baele, G., 2019. Bayesian Evaluation of Temporal Signal in Measurably Evolving Populations. bioRxiv 10.1101/810697." />
<meta name="citation_reference" content="Dudas, G., Carvalho, L.M., Rambaut, A., Bedford, T., 2018. MERS-CoV spillover at the camel-human interface. eLife 7, e31257." />
<meta name="citation_reference" content="Eden, J.-S., Tanaka, M.M., Boni, M.F., Rawlinson, W.D., White, P. A, 2013. Recombination within the pandemic norovirus GII.4 lineage. J Virol 87, 6270–6282. https://doi.org/10.1128/JVI.03464-12" />
<meta name="citation_reference" content="Forni, D., Cagliani, R., Clerici, M., Sironi, M., 2017. Molecular Evolution of Human Coronavirus Genomes. Trends Microbiol 25, 35–48. https://doi.org/10.1016/j.tim.2016.09.001" />
<meta name="citation_reference" content="Ge, X., Li, J., Yang, X., Chmura, A.A., Zhu, G., Epstein, J.H., Mazet, J.K., Hu, B., Zhang, W., Peng, C., Zhang, Y., Luo, C., Tan, B., Wang, N., Zhu, Y., Crameri, G., Zhang, S., Wang, L., Daszak, P., Shi, Z.-L., 2013. Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor. Nature 503, 535–538. https://doi.org/10.1038/nature12711" />
<meta name="citation_reference" content="Gorbalenya, A.E., Baker, S.C., Baric, R.S., Groot, R.J. De, Gulyaeva, A.A., Haagmans, B.L., Lauber, C., Leontovich, A.M., Neuman, B.W., Penzar, D., Perlman, S., Poon, L.L.M., Samborskiy, D., Sidorov, I.A., Sola, I., Ziebuhr, J., 2020. Severe acute respiratory syndrome-related coronavirus: The species and its viruses – a statement of the Coronavirus Study Group. bioRxiv doi.org/10.1101/2020.02.07.937862." />
<meta name="citation_reference" content="Graham, R.L., Baric, R.S., 2010. Recombination, Reservoirs, and the Modular Spike: Mechanisms of Coronavirus Cross-Species Transmission. J Virol 84, 3134–3146. https://doi.org/10.1128/jvi.01394-09" />
<meta name="citation_reference" content="He, B., Zhang, Y., Xu, L., Yang, W., Yang, F., Feng, Yun, Xia, L., Zhou, J., Zhen, W., Feng, Ye, Guo, H., Zhang, H., Tu, C., 2014. Identification of Diverse Alphacoronaviruses and Genomic Characterization of a Novel Severe Acute Respiratory Syndrome-Like Coronavirus from Bats in China. J Virol 88, 7070–7082. https://doi.org/10.1128/JVI.00631-14" />
<meta name="citation_reference" content="Holmes, E.C., 2009. The Evolution and Emergence of RNA Viruses. Oxford University Press, New York, NY." />
<meta name="citation_reference" content="Holmes, E.C., Dudas, G., Rambaut, A., Andersen, K.G., 2016. The evolution of Ebola virus: Insights from the 2013-2016 epidemic. Nature 538, 193–200. https://doi.org/10.1038/nature19790" />
<meta name="citation_reference" content="Holmes, E.C., Rambaut, A., Andersen, K.G., 2018. Pandemics: spend on surveillance, not prediction. Nature 558, 180–182." />
<meta name="citation_reference" content="Hon, C., Lam, T., Shi, Z., Drummond, A.J., Yip, C., Zeng, F., Lam, P., Leung, F.C., 2008. Evidence of the Recombinant Origin of a Bat Severe Acute Respiratory Syndrome (SARS)-Like Coronavirus and Its Implications on the Direct Ancestor of SARS Coronavirus. J Virol 82, 1819–1826. https://doi.org/10.1128/JVI.01926-07" />
<meta name="citation_reference" content="Hu, B., Zeng, L., Yang, X., Ge, X., Zhang, W., Li, B., Xie, Z., Shen, X., Zhang, Y., Wang, N., Luo, D., Zheng, S., Wang, M., Daszak, P., Wang, L., Cui, J., Shi, Z.-L., 2017. Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus. PLoS Pathog 13, e1006698." />
<meta name="citation_reference" content="Ji, W., Wang, W., Zhao, X., Zai, J., Li, X., 2020. Cross-species transmission of the newly identified coronavirus 2019-nCoV. J Med Virol 92, 433–440. https://doi.org/10.1002/jmv.25682" />
<meta name="citation_reference" content="Katoh, K., Asimenos, G., Toh, H., 2009. Multiple Alignment of DNA Sequences with MAFFT, in: Press, H. (Ed.), Bioinformatics for DNA Sequence Analysis. Hertfordshire, UK, pp. 39–64." />
<meta name="citation_reference" content="Kosakovsky Pond, S.L., Posada, D., Gravenor, M.B., Woelk, C.H., Frost, S.D.W., 2006. Automated Phylogenetic Detection of Recombination Using a Genetic Algorithm. Mol Biol Evol 23, 1891–1901. https://doi.org/10.1093/molbev/msl051" />
<meta name="citation_reference" content="Lam, H.M., Ratmann, O., Boni, M.F., 2018. Improved algorithmic complexity for the 3SEQ recombination detection algorithm. Mol Biol Evol 35, 247–251. https://doi.org/10.1093/molbev/msx263" />
<meta name="citation_reference" content="Lam, T.T., Shum, M.H.-H., Zhu, H.-C., Tong, Y.-G., Ni, X.-B., Liao, Y.-S., Wei, W., Cheung, W.Y.-M., Li, W.-J., Li, L.-F., Leung, G.M., Holmes, E.C., Hu, Y.-L., Guan, Y., 2020. Identification of 2019-nCoV related coronaviruses in Malayan pangolins in southern China. Nature doi.org/10.1101/2020.02.13.945485." />
<meta name="citation_reference" content="Lemey, P., Minin, V.N., Bielejec, F., Pond, S.L.K., Suchard, M.A., 2012. A counting renaissance: combining stochastic mapping and empirical Bayes to quickly detect amino acid sites under positive selection. Bioinformatics 28, 3248–3256. https://doi.org/10.1093/bioinformatics/bts580" />
<meta name="citation_reference" content="Li, Q., Guan, X., Wu, P., Wang, X., Zhou, L., Tong, Y., Ren, R., Leung, K.S.M., Lau, E.H.Y., Wong, J.Y., Xing, X., Xiang, N., Wu, Y., Li, C., Chen, Q., Li, D., Liu, T., Zhao, J., Liu, M., Tu, W., Chen, C., Jin, L., Yang, R., Wang, Q., Zhou, S., Wang, R., Liu, H., Luo, Y., Liu, Y., Shao, G., Li, H., Tao, Z., Yang, Y., Deng, Z., Liu, B., Ma, Z., Zhang, Y., Shi, G., Lam, T.T.Y., Wu, J.T., Gao, G.F., Cowling, B.J., Yang, B., Leung, G.M., Feng, Z., 2020a. Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus–Infected Pneumonia. N Engl J Med, published online, doi: 10.1056/NEJMoa2001316. https://doi.org/10.1056/NEJMoa2001316" />
<meta name="citation_reference" content="Li, X., Giorgi, E., Marichann, M.H., Foley, B., Xiao, C., Kong, X., Chen, Y., Korber, B., Gao, F., 2020b. Emergence of SARS-CoV-2 through Recombination and Strong Purifying Selection. bioRxiv 10.1101/2020.03.20.000885" />
<meta name="citation_reference" content="Lie, P., Chen, W., Chen, J.-P., 2019. Viral Metagenomics Revealed Sendai Virus and Coronavirus Infection of Malayan Pangolins. Viruses 11, 979." />
<meta name="citation_reference" content="Lin, X., Wang, W., Hao, Z., Wang, Z., Guo, W., Guan, Q., Wang, M., Wang, H., Zhou, R., Li, M., Tang, P., Wu, J., Holmes, E.C., Zhang, Y., 2017. Extensive diversity of coronaviruses in bats from China. Virology 507, 1–10. https://doi.org/10.1016/j.virol.2017.03.019" />
<meta name="citation_reference" content="Lu, R., Zhao, X., Li, J., Niu, P., Yang, B., Wu, H., Wang, W., Song, H., Huang, B., Zhu, N., Bi, Y., Ma, X., Zhan, F., Wang, L., Hu, T., Zhou, H., Hu, Z., Zhou, W., Zhao, L., Chen, J., Meng, Y., Wang, J., Lin, Y., Yuan, J., Xie, Z., Ma, J., Liu, W.J., Wang, D., Xu, W., Holmes, E.C., Gao, G.F., Wu, G., Chen, W., Shi, W., Tan, W., 2020. Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. Lancet 395, 565–574. https://doi.org/10.1016/S0140-6736(20)30251-8" />
<meta name="citation_reference" content="Martin, D.P., Murrell, B., Golden, M., Khoosal, A., Muhire, B., 2015. RDP4: Detection and analysis of recombination patterns in virus genomes. Virus Evol 1:vev003. https://doi.org/10.1093/ve/vev003" />
<meta name="citation_reference" content="Membrebe, J.V., Suchard, M.A., Rambaut, A., Baele, G., Lemey, P., 2019. Bayesian Inference of Evolutionary Histories under Time-Dependent Substitution Rates. Mol Biol Evol 36, 1793–1803. https://doi.org/10.1093/molbev/msz094" />
<meta name="citation_reference" content="Menachery, V.D., Jr, B.L.Y., Debbink, K., Agnihothram, S., Gralinski, L.E., Plante, J.A., Graham, R.L., Scobey, T., Ge, X., Donaldson, E.F., Randell, S.H., Lanzavecchia, A., Marasco, W.A., Shi, Z., Baric, R.S., 2015. A SARS-like cluster of circulating bat coronaviruses shows potential for human emergence. Nat Med 21, 1508–1514. https://doi.org/10.1038/nm.3985" />
<meta name="citation_reference" content="Menachery, V.D., Yount, B.L., Sims, A.C., Debbink, K., Agnihothram, S.S., Gralinski, L.E., Pickles, R.J., Corti, D., Randell, S.H., Lanzavecchia, A., Marasco, W.A., Baric, R.S., 2016. SARS-like WIV1-CoV poised for human emergence. Proc Natl Acad Sci USA 113, 3048–3053. https://doi.org/10.1073/pnas.1517719113" />
<meta name="citation_reference" content="Nguyen, L., Schmidt, H.A., Haeseler, A. Von, Minh, B.Q., 2014. IQ-TREE: A Fast and Effective Stochastic Algorithm for Estimating Maximum-Likelihood Phylogenies. Mol Biol Evol 32, 268–274. https://doi.org/10.1093/molbev/msu300" />
<meta name="citation_reference" content="Posada, D., Crandall, K.A., Holmes, E.C., 2002. Recombination in evolutionary genomics. Annu Rev Genet 36, 75–97. https://doi.org/10.1146/annurev.genet.36.040202.111115" />
<meta name="citation_reference" content="Rambaut, A., Lam, T.T., Carvalho, L.M., Pybus, O.G., 2016. Exploring the temporal structure of heterochronous sequences using TempEst (formerly Path-O-Gen). Virus Evol 2, vew007. https://doi.org/10.1093/ve/vew007" />
<meta name="citation_reference" content="Robertson, D., 2020. nCoV’s relationship to bat coronaviruses – recombination signals (no snakes) - no evidence the 2019-nCoV lineage is recombinant. Virological.org. URL http://virological.org/t/331" />
<meta name="citation_reference" content="Schierup, M.H., Hein, J., 1999. Recombination and the Molecular Clock. Mol Biol Evol 17, 1578–1579." />
<meta name="citation_reference" content="Stamatakis, A., 2014. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics 30, 1312–1313. https://doi.org/10.1093/bioinformatics/btu033" />
<meta name="citation_reference" content="Stamatakis, A., 2006. RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models. Bioinformatics 22, 2688–90. https://doi.org/10.1093/bioinformatics/btl446" />
<meta name="citation_reference" content="Stegeman, A., Bouma, A., Elbers, A.R.W., Jong, M.C.M. De, Nodelijk, G., Klerk, F. De, Koch, G., van Boven, M., 2004. Avian Influenza A Virus (H7N7) Epidemic in The Netherlands in 2003: Course of the Epidemic and Effectiveness of Control Measures. J Infect Dis 190, 2088–2095." />
<meta name="citation_reference" content="Su, S., Wong, G., Shi, W., Liu, J., Lai, A.C.K., Zhou, J., Liu, W., Bi, Y., Gao, G.F., 2016. Epidemiology, Genetic Recombination, and Pathogenesis of Coronaviruses. Trends Microbiol 24, 490–502. https://doi.org/10.1016/j.tim.2016.03.003" />
<meta name="citation_reference" content="Suchard, M.A., Lemey, P., Baele, G., Ayres, D.L., Drummond, A.J., Rambaut, A., 2018. Bayesian phylogenetic and phylodynamic data integration using BEAST 1.10. Virul Evol 4, vey016. https://doi.org/10.1093/ve/vey016" />
<meta name="citation_reference" content="Trova, S., Baele, G., Vrancken, B., Bielejec, F., Suchard, M.A., Fargette, D., Lemey, P., 2015. Host ecology determines the dispersal patterns of a plant virus. Virus Evol 1:vev016. https://doi.org/10.1093/ve/vev016" />
<meta name="citation_reference" content="Wan, Y., Shang, J., Graham, R., Baric, R., Li, F., 2020. Receptor Recognition by the Novel Coronavirus from Wuhan: an Analysis Based on Decade-Long Structural Studies of SARS Coronavirus. J Virol 94, e00127–20." />
<meta name="citation_reference" content="Wang, L., Fu, S., Cao, Y., Zhang, H., Feng, Y., Yang, W., Nie, K., Ma, X., Liang, G., 2017. Discovery and genetic analysis of novel coronaviruses in least horseshoe bats in southwestern China. Emerg Microb Infect 6, e14. https://doi.org/10.1038/emi.2016.140" />
<meta name="citation_reference" content="Webster, R.G., Bean, W.J., Gorman, O.T., Chambers, T.M., Kawaoka, Y., 1992. Evolution and ecology of influenza A viruses. Microbiol Rev 56, 152–79." />
<meta name="citation_reference" content="Wong, A.C.P., Li, X., Lau, S.K.P., Woo, P.C.Y., 2019. Global Epidemiology of Bat Coronaviruses. Viruses 11, 174. https://doi.org/10.3390/v11020174" />
<meta name="citation_reference" content="World Health Organization, 2020a. Novel Coronavirus (2019-nCoV) Situation Report 1, 21 January 2020. Geneva, Switzerland. World Health Organization, 2020b. Coronavirus disease 2019 (COVID-19) Situation Report – 51, 11 March 2020. Geneva, Switzerland." />
<meta name="citation_reference" content="Wu, F., Zhao, S., Yu, B., Chen, Y.-M., Wang, W., Song, Z.-G., Hu, Y., Tao, Z.-W., Tian, J., Pei, Y., Yuan, M., Zhang, Y., Dai, F.-H., Liu, Y., Wang, Q.-M., Zheng, J.-J., Xu, L., Holmes, E.C., Zhang, Y.-Z., 2020a. A new coronavirus associated with human respiratory disease in China. Nature published, doi.org/10.1038/s41586-020-2008-3. https://doi.org/10.1038/s41586-020-2008-3" />
<meta name="citation_reference" content="Wu, Y., Ho, W., Huang, Y., Jin, D., Li, S., Liu, S., Liu, X., Qiu, J., Sang, Y., Wang, Q., Yuen, K., Zheng, Z., 2020b. SARS-CoV-2 is an appropriate name for the new coronavirus A distinct name is needed for the new coronavirus. Lancet 395, 949–950. https://doi.org/10.1016/S0140-6736(20)30557-2" />
<meta name="citation_reference" content="Xiao, K., Zhai, J., Feng, Y., Zhou, N., Zhang, X., Zou, J.-J., Li, N., Guo, Y., Li, X., Shen, X., Zhang, Z., Shu, F., Huang, W., Li, Y., Zhang, Z., Chen, R.-A., Wu, Y.-J., Peng, S.-M., Huang, M., Xie, W.-J., Cai, Q.-H., Hou, F.-H., Liu, Y., Chen, W., Xiao, L., Shen, Y., 2020. Isolation and Characterization of 2019-nCoV-like Coronavirus from Malayan Pangolins. bioRxiv 10.1101/2020.02.17.951335v1." />
<meta name="citation_reference" content="Yres, D.L., Cummings, M.P., Baele, G.B., Darling, A.E., Lewis, P.O., Swofford, D.L., Huelsenbeck, J.P., Lemey, P., Rambaut, A., Suchard, M.A., 2019. BEAGLE 3: Improved Performance, Scaling, and Usability for a High-Performance Computing Library for Statistical Phylogenetics. Softw. Syst Evol 68, 1052–1061. https://doi.org/10.1093/sysbio/syz020" />
<meta name="citation_reference" content="Yu, H., Wu, J.T., Cowling, B.J., Liao, Q., Fang, V.J., Zhou, S., Wu, P., Zhou, H., Lau, E.H.Y., Guo, D., Ni, M.Y., Peng, Z., Feng, L., Jiang, H., Luo, H., Li, Q., Feng, Z., Wang, Y., Yang, W., Leung, G.M., 2013. Effect of closure of live poultry markets on poultry-to-person transmission of avian influenza A H7N9 virus: an ecological study. Lancet 383, 541–548. https://doi.org/10.1016/S0140-6736(13)61904-2" />
<meta name="citation_reference" content="Yuan, J., Hon, C., Li, Y., Wang, D., Xu, G., Zhang, H., Zhou, P., Poon, L.L.M., Lam, T.T.-Y., Leung, F.C., Shi, Z., 2010. Communication Intraspecies diversity of SARS-like coronaviruses in Rhinolophus sinicus and its implications for the origin of SARS coronaviruses in humans. J Gen Virol 91, 1058–1062. https://doi.org/10.1099/vir.0.016378-0" />
<meta name="citation_reference" content="Zhang, Y.-Z., Holmes, E.C., 2020. Commentary A Genomic Perspective on the Origin and Emergence of SARS-CoV-2. Cell 181, doi.org/10.1016/j.cell.2020.03.035. https://doi.org/10.1016/j.cell.2020.03.035" />
<meta name="citation_reference" content="Zhou, H., Chen, X., Hu, T., Li, J., Song, H., Liu, Y., Wang, P., Liu, D., Yang, J., Holmes, E.C., Hughes, A.C., Bi, Y., Shi, W., 2020a. A novel bat coronavirus reveals natural insertions at the S1/S2 cleavage site of the Spike protein and a possible recombinant origin of HCoV-19. bioRxiv 10.1101/2020.03.02.974139." />
<meta name="citation_reference" content="Zhou, P., Yang, X.-L., Wang, X.-G., Hu, B., Zhang, L., Zhang, W., Si, H.-R., Zhu, Y., Li, B., Huang, C.-L., Chen, H.-D., Chen, J., Luo, Y., Guo, H., Jiang, R.-D., Liu, M.-Q., Chen, Y., Shen, X.-R., Wang, X., Zheng, X.-S., Zhao, K., Chen, Q.-J., Deng, F., Liu, L.-L., Yan, B., Zhan, F.-X., Wang, Y.-Y., Geng-Fu, X., Shi, Z.-L., 2020b. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature, published online, doi.org/10.1038/s41586-020-2012-7." />
<meta name="twitter:title" content="Evolutionary origins of the SARS-CoV-2 sarbecovirus lineage responsible for the COVID-19 pandemic" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="There are outstanding evolutionary questions on the recent emergence of coronavirus SARS-CoV-2/hCoV-19 in Hubei province that caused the COVID-19 pandemic, including (1) the relationship of the new virus to the SARS-related coronaviruses, (2) the role of bats as a reservoir species, (3) the potential role of other mammals in the emergence event, and (4) the role of recombination in viral emergence. Here, we address these questions and find that the sarbecoviruses – the viral subgenus responsible for the emergence of SARS-CoV and SARS-CoV-2 – exhibit frequent recombination, but the SARS-CoV-2 lineage itself is not a recombinant of any viruses detected to date. In order to employ phylogenetic methods to date the divergence events between SARS-CoV-2 and the bat sarbecovirus reservoir, recombinant regions of a 68-genome sarbecovirus alignment were removed with three independent methods. Bayesian evolutionary rate and divergence date estimates were consistent for all three recombination-free alignments and robust to two different prior specifications based on HCoV-OC43 and MERS-CoV evolutionary rates. Divergence dates between SARS-CoV-2 and the bat sarbecovirus reservoir were estimated as 1948 (95% HPD: 1879-1999), 1969 (95% HPD: 1930-2000), and 1982 (95% HPD: 1948-2009). Despite intensified characterization of sarbecoviruses since SARS, the lineage giving rise to SARS-CoV-2 has been circulating unnoticed for decades in bats and been transmitted to other hosts such as pangolins. The occurrence of a third significant coronavirus emergence in 17 years together with the high prevalence and virus diversity in bats implies that these viruses are likely to cross species boundaries again.

In Brief The Betacoronavirus SARS-CoV-2 is a member of the sarbecovirus subgenus which shows frequent recombination in its evolutionary history. We characterize the extent of this genetic exchange and identify non-recombining regions of the sarbecovirus genome using three independent methods to remove the effects of recombination. Using these non-recombining genome regions and prior information on coronavirus evolutionary rates, we obtain estimates from three approaches that the most likely divergence date of SARS-CoV-2 from its most closely related available bat sequences ranges from 1948 to 1982.

Key Points" />
<meta name="og-title" property="og:title" content="Evolutionary origins of the SARS-CoV-2 sarbecovirus lineage responsible for the COVID-19 pandemic" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.30.015008v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="There are outstanding evolutionary questions on the recent emergence of coronavirus SARS-CoV-2/hCoV-19 in Hubei province that caused the COVID-19 pandemic, including (1) the relationship of the new virus to the SARS-related coronaviruses, (2) the role of bats as a reservoir species, (3) the potential role of other mammals in the emergence event, and (4) the role of recombination in viral emergence. Here, we address these questions and find that the sarbecoviruses – the viral subgenus responsible for the emergence of SARS-CoV and SARS-CoV-2 – exhibit frequent recombination, but the SARS-CoV-2 lineage itself is not a recombinant of any viruses detected to date. In order to employ phylogenetic methods to date the divergence events between SARS-CoV-2 and the bat sarbecovirus reservoir, recombinant regions of a 68-genome sarbecovirus alignment were removed with three independent methods. Bayesian evolutionary rate and divergence date estimates were consistent for all three recombination-free alignments and robust to two different prior specifications based on HCoV-OC43 and MERS-CoV evolutionary rates. Divergence dates between SARS-CoV-2 and the bat sarbecovirus reservoir were estimated as 1948 (95% HPD: 1879-1999), 1969 (95% HPD: 1930-2000), and 1982 (95% HPD: 1948-2009). Despite intensified characterization of sarbecoviruses since SARS, the lineage giving rise to SARS-CoV-2 has been circulating unnoticed for decades in bats and been transmitted to other hosts such as pangolins. The occurrence of a third significant coronavirus emergence in 17 years together with the high prevalence and virus diversity in bats implies that these viruses are likely to cross species boundaries again.

In Brief The Betacoronavirus SARS-CoV-2 is a member of the sarbecovirus subgenus which shows frequent recombination in its evolutionary history. We characterize the extent of this genetic exchange and identify non-recombining regions of the sarbecovirus genome using three independent methods to remove the effects of recombination. Using these non-recombining genome regions and prior information on coronavirus evolutionary rates, we obtain estimates from three approaches that the most likely divergence date of SARS-CoV-2 from its most closely related available bat sequences ranges from 1948 to 1982.

Key Points" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-31" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.30.015008v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.30.015008v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.30.015008v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.30.015008v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1218990" />
    <title>Evolutionary origins of the SARS-CoV-2 sarbecovirus lineage responsible for the COVID-19 pandemic | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Zu1s855Wq8nyOB_bWCj79PDL4fAdmzJnBn8pYg7pZLc","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1218990","apath":"\/biorxiv\/early\/2020\/03\/31\/2020.03.30.015008.atom","pisa":"biorxiv;2020.03.30.015008v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.30.015008v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.30.015008v1","title":"Evolutionary origins of the SARS-CoV-2 sarbecovirus lineage responsible for the COVID-19 pandemic","identifier":"node\/1218990"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221s69l2qQNs0j2C7GNhDXlWsO02+eYfiuoW91aehnwdAptq5XEDDE\\\\\\\/ceUR53lJSocvZUoeSOQ3nHIDu1qL8IEdQzNX57t6GNurpGQxJoJCsOn\\\\\\\/VcysKMEExHfaKrUL7FIs2dDzS\\\\\\\/cHgLkjz3dpvy\\\\\\\/t6FnRYEzzWa7ulfVdg6OeJUJxvqaheFT3vSaP2ol1XvjXNtT4XwnJ8EAtoW95mClgHIdFomwMhuJXYlBltUWAoTD6cfGBWx2sKeSCLCZl1g9L6pJ1eprH+33PjheRMGtEgL95QC+iNCxD+hNmFMf8V4X7y86jw5L0+SP0Gsulaase6IKX51dv\\\\\\\/yMjR29Fom1rdi3jQz8JROYtyf5F15A4hRwHEpy\\\\\\\/nuA1UQB+FHVU2T5BpIWrt0p54eQpiLwKeivs3uKijmAnlUryZWdFfG+BS6QXYoXZCPr4cMrwCzEoGd5luxJshRJuJubSLfavdAaxrrEZjs0xRmMD3T4eEGm15tfmmuQwGIaB6ab5pIe8wFkJ4Ay32u3jaAQnZJlQTgtrJfUK3eeN9x8AOgjUivxCsEw67ReR2RZYI\\\\\\\/VlEmwfJ8SZ0jgKflppz3ejekrWSdV5Mh9zcH3r0fPV1Is0IBkA8P+0NV97L4tLYf2suraBsB65UYvVWLDp9277jHxT\\\\\\\/P\\\\\\\/PWA9b4amd7mwbbPAt\\\\\\\/oig9CV+o\\\\\\\/aP1e2rscXAGRhXLrx\\\\\\\/QZe1WoI\\\\\\\/mjNHztzQU85ajIegz92GnuFNxNFBnW5k+NnwG1rfntEiv8SzCdji2iO0jH18d5+YRHG4C8ifx9Jz+ed4gTrgPvzaKe\\\\\\\/62avE17RIFqJn6jajXyzenuz\\\\\\\/uia1uD0pvWSnuj\\\\\\\/fx8B3K94q5qB9L43zx7pcbgAZYWGJFY0+tWQ9Wkhd1OCrkhr7T6gN7mQlDx9szGJethVPhXp\\\\\\\/WlncsC\\\\\\\/pToMTbZyf6zZD0lhff4eKHpeY\\\\\\\/QfC4rO4aoPH6vc7iUf+prtVo0t+1FyCaq7PW4+77PxKSjhfMY5WgzbejrHHG+qDbQNWWtJ61HDJJ73PfLchRWFHO4eKQ77qgAEsCX58b0o9ek3X+2\\\\\\\/xhshMBWqS6VNExhx8HJ5nRXs+IGLhQbLdYHsnNK7tIM9JqbWo+XvhlVuok436bEgM+HMWYgXytVtfn34lMZH9W8ARxvLbPkqXBDOlW3dflOGNo4h04UcdX9K3PKzE13eK1IHKxIofQGFSTBzbuIA+CgxPXQTKKN1bZ15CXmf5aBSVl+bUjHXZHeg2XvwAvzLTrNuo6hjC4AOpi3pWWFUaH7MrGhw1\\\\\\\/5hZ5MmfAs0wFSJJoWJkUbGjnp+1gcX1YFVpQvMRQlpJ1Yykx5sEEyGot4+\\\u0022,\\\u0022iv\\\u0022:\\\u00225crZPc97F\\\\\\\/6qDYhCrC1esQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00221adc10546265bbcff409baeb6285380948a0fb92be5919ee44eaebed6f304e51\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.30.015008v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1218990 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.30.015008v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bXOto2L6ptlpBO-SOMNWoy98fyHMepH5FGcgb5ugnQA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1218990" id="node1218990" data-pisa="biorxiv;2020.03.30.015008v1" data-pisa-master="biorxiv;2020.03.30.015008" data-apath="/biorxiv/early/2020/03/31/2020.03.30.015008.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Evolutionary origins of the SARS-CoV-2 sarbecovirus lineage responsible for the COVID-19 pandemic</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Maciej F</span> <span class="nlm-surname">Boni</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Philippe</span> <span class="nlm-surname">Lemey</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Xiaowei</span> <span class="nlm-surname">Jiang</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Tommy Tsan-Yuk</span> <span class="nlm-surname">Lam</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Blair</span> <span class="nlm-surname">Perry</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Todd</span> <span class="nlm-surname">Castoe</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Andrew</span> <span class="nlm-surname">Rambaut</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">David L</span> <span class="nlm-surname">Robertson</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.30.015008 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1218990" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Maciej F Boni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Infectious Disease Dynamics, Department of Biology, Pennsylvania State University</span>, University Park, PA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maciej%2BF%2BBoni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boni%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaciej%2BF%2BBoni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mfb9@psu.edu" class="" data-icon-position="" data-hide-link-title="0">mfb9@psu.edu</a>
<a href="mailto:philippe.lemey@kuleuven.be" class="" data-icon-position="" data-hide-link-title="0">philippe.lemey@kuleuven.be</a>
<a href="mailto:a.rambaut@ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">a.rambaut@ed.ac.uk</a>
<a href="mailto:David.L.Robertson@glasgow.ac.uk" class="" data-icon-position="" data-hide-link-title="0">David.L.Robertson@glasgow.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Philippe Lemey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Microbiology, Immunology and Transplantation</span>, KU Leuven, Leuven, <span class='nlm-country'>Belgium</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philippe%2BLemey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lemey%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APhilippe%2BLemey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mfb9@psu.edu" class="" data-icon-position="" data-hide-link-title="0">mfb9@psu.edu</a>
<a href="mailto:philippe.lemey@kuleuven.be" class="" data-icon-position="" data-hide-link-title="0">philippe.lemey@kuleuven.be</a>
<a href="mailto:a.rambaut@ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">a.rambaut@ed.ac.uk</a>
<a href="mailto:David.L.Robertson@glasgow.ac.uk" class="" data-icon-position="" data-hide-link-title="0">David.L.Robertson@glasgow.ac.uk</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xiaowei Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Xi’an Jiaotong-Liverpool University (XJTLU)</span>, Suzhou, Jiangsu, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaowei%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaowei%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tommy Tsan-Yuk Lam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>State Key Laboratory of Emerging Infectious Diseases, School of Public Health, The University of Hong Kong</span>, Hong Kong SAR, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tommy%2BTsan-Yuk%2BLam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lam%20TT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATommy%2BTsan-Yuk%2BLam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Blair Perry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Biology, University of Texas Arlington</span>, Arlington, TX, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Blair%2BPerry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Perry%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABlair%2BPerry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Todd Castoe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Biology, University of Texas Arlington</span>, Arlington, TX, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Todd%2BCastoe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Castoe%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATodd%2BCastoe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Andrew Rambaut </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Institute of Evolutionary Biology, University of Edinburgh</span>, Edinburgh, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BRambaut%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rambaut%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BRambaut%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mfb9@psu.edu" class="" data-icon-position="" data-hide-link-title="0">mfb9@psu.edu</a>
<a href="mailto:philippe.lemey@kuleuven.be" class="" data-icon-position="" data-hide-link-title="0">philippe.lemey@kuleuven.be</a>
<a href="mailto:a.rambaut@ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">a.rambaut@ed.ac.uk</a>
<a href="mailto:David.L.Robertson@glasgow.ac.uk" class="" data-icon-position="" data-hide-link-title="0">David.L.Robertson@glasgow.ac.uk</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">David L Robertson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>MRC-University of Glasgow Centre for Virus Research (CVR)</span>, Glasgow, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BL%2BRobertson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Robertson%20DL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BL%2BRobertson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mfb9@psu.edu" class="" data-icon-position="" data-hide-link-title="0">mfb9@psu.edu</a>
<a href="mailto:philippe.lemey@kuleuven.be" class="" data-icon-position="" data-hide-link-title="0">philippe.lemey@kuleuven.be</a>
<a href="mailto:a.rambaut@ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">a.rambaut@ed.ac.uk</a>
<a href="mailto:David.L.Robertson@glasgow.ac.uk" class="" data-icon-position="" data-hide-link-title="0">David.L.Robertson@glasgow.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.30.015008v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1218990" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1218990/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.30.015008v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1218990" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1218990/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.30.015008v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1218990" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1218990/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.30.015008v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1218990" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1218990/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.30.015008v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1218990" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1218990/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.30.015008v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1218990" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1218990/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">There are outstanding evolutionary questions on the recent emergence of coronavirus SARS-CoV-2/hCoV-19 in Hubei province that caused the COVID-19 pandemic, including (1) the relationship of the new virus to the SARS-related coronaviruses, (2) the role of bats as a reservoir species, (3) the potential role of other mammals in the emergence event, and (4) the role of recombination in viral emergence. Here, we address these questions and find that the sarbecoviruses – the viral subgenus responsible for the emergence of SARS-CoV and SARS-CoV-2 – exhibit frequent recombination, but the SARS-CoV-2 lineage itself is not a recombinant of any viruses detected to date. In order to employ phylogenetic methods to date the divergence events between SARS-CoV-2 and the bat sarbecovirus reservoir, recombinant regions of a 68-genome sarbecovirus alignment were removed with three independent methods. Bayesian evolutionary rate and divergence date estimates were consistent for all three recombination-free alignments and robust to two different prior specifications based on HCoV-OC43 and MERS-CoV evolutionary rates. Divergence dates between SARS-CoV-2 and the bat sarbecovirus reservoir were estimated as 1948 (95% HPD: 1879-1999), 1969 (95% HPD: 1930-2000), and 1982 (95% HPD: 1948-2009). Despite intensified characterization of sarbecoviruses since SARS, the lineage giving rise to SARS-CoV-2 has been circulating unnoticed for decades in bats and been transmitted to other hosts such as pangolins. The occurrence of a third significant coronavirus emergence in 17 years together with the high prevalence and virus diversity in bats implies that these viruses are likely to cross species boundaries again.</p><div id="sec-1" class="subsection"><p id="p-4"><strong>In Brief</strong> The Betacoronavirus SARS-CoV-2 is a member of the sarbecovirus subgenus which shows frequent recombination in its evolutionary history. We characterize the extent of this genetic exchange and identify non-recombining regions of the sarbecovirus genome using three independent methods to remove the effects of recombination. Using these non-recombining genome regions and prior information on coronavirus evolutionary rates, we obtain estimates from three approaches that the most likely divergence date of SARS-CoV-2 from its most closely related available bat sequences ranges from 1948 to 1982.</p></div><div id="sec-2" class="subsection"><p><strong>Key Points</strong></p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-5">RaTG13 is the closest available bat virus to SARS-CoV-2; a sub-lineage of these bat viruses is able to infect humans. Two sister lineages of the RaTG13/SARS-CoV-2 lineage infect Malayan pangolins.</p></li><li id="list-item-2"><p id="p-6">The sarbecoviruses show a pattern of deep recombination events, indicating that there are high levels of co-infection in horseshoe bats and that the viral pool can generate novel allele combinations and substantial genetic diversity; the sarbecoviruses are efficient ‘explorers’ of phenotype space.</p></li><li id="list-item-3"><p id="p-7">The SARS-CoV-2 lineage is not a recent recombinant, at least not involving any of the bat or pangolin viruses sampled to date.</p></li><li id="list-item-4"><p id="p-8">Non-recombinant regions of the sarbecoviruses can be identified, allowing for phylogenetic inference and dating to be performed. We constructed three such regions using different methods.</p></li><li id="list-item-5"><p id="p-9">We estimate that RaTG13 and SARS-CoV-2 diverged 40 to 70 years ago. There is a diverse unsampled reservoir of generalist viruses established in horseshoe bats.</p></li><li id="list-item-6"><p id="p-10">While an intermediate host responsible for the zoonotic event cannot be ruled out, the relevant evolution for spillover to humans very likely occurred in horseshoe bats.</p></li></ul></div></div><div class="section" id="sec-3"><h2 class="">Introduction</h2><p id="p-11">In December 2019, a cluster of pneumonia cases epidemiologically linked to an open-air wet market in city of Wuhan (Hubei Province), China (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li et al., 2020a</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Zhou et al., 2020b</a>) led local health officials to issue an epidemiological alert to the Chinese Center for Disease Control and Prevention (China CDC) and the World Health Organization’s (WHO) China Country Office. In early January, the etiological agent of the pneumonia cases was found to be a coronavirus (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">World Health Organization, 2020a</a>), subsequently named SARS-CoV-2 by an International Committee on Taxonomy of Viruses (ICTV) Study Group (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gorbalenya et al., 2020</a>), (also named hCoV-19 by (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wu et al., 2020b</a>)). The first available sequence data (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wu et al., 2020a</a>) placed this novel human pathogen in the <em>Sarbecovirus</em> subgenus of <em>Coronaviridae</em> (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lu et al., 2020</a>), the same sub-genus as the SARS virus that caused a global outbreak of nearly 8000 cases in 2002-2003. By mid-January, the virus was spreading widely inside Hubei province and by early March SARS-CoV-2 reached pandemic status (World Health Organization, 2020b).</p><p id="p-12">In outbreaks of zoonotic pathogens, identification of the infection source is crucial, as this may allow health authorities to separate human populations from the wild-life or domestic-animal reservoirs posing the zoonotic risk (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Stegeman et al., 2004</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Yu et al., 2013</a>). If controlling an outbreak in its early stages is not possible – as was the case for the COVID-19 epidemic in Hubei province – identification of origins and point sources is nevertheless important for containment purposes in other provinces and prevention of future outbreaks. When the first genome sequence of SARS-CoV-2, Wuhan-Hu-1, was released on January 10 2020 on Virological.org by a consortium led by Yong-Zhen Zhang (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Wu et al., 2020a</a>) it enabled the immediate analyses of its ancestry. Across a large region of its genome, corresponding approximately to ORF1b, it did not cluster with any of the known bat coronaviruses indicating that recombination likely played a role in the evolutionary history of these viruses (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Lu et al., 2020</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Wu et al. 2020b</a>). Subsequently, a bat sarbecovirus – RaTG13 sampled from a <em>Rhinolophus affinis</em> horseshoe bat in 2013 in Yunnan province – was reported that clusters with SARS-CoV-2 in almost all genomic regions with approximately 96% genome sequence identity (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Zhou et al., 2020b</a>). <a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Zhou et al. (2020b)</a> concluded from the genetic proximity of SARS-CoV-2 to a bat virus that a bat origin for the current COVID-19 outbreak is probable. Recent evidence has also identified pangolins as potential intermediate species for SARS-CoV-2 emergence or as a potential reservoir species themselves (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lam et al., 2020</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Xiao et al., 2020</a>).</p><p id="p-13">Unlike other viruses that have emerged in the past two decades, coronaviruses are highly recombinogenic (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Forni et al., 2017</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hon et al., 2008</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lam et al., 2018</a>). Influenza viruses reassort (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Webster et al., 1992</a>) but they do not undergo homologous recombination within RNA segments (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Boni et al., 2008</a>, 2010), meaning that ‘origins’ questions for influenza outbreaks can always be reduced to origins questions for each of influenza’s eight RNA segments. For coronaviruses however, recombination means that small genomic sub-regions can have independent origins, identifiable if sufficient sampling has been done in the animal reservoirs that support the endemic circulation, coinfection, and recombination that appear to be common. Here, we analyse the evolutionary history of SARS-CoV-2, using available genomic data on sarbecoviruses. We demonstrate that the sarbecoviruses circulating in horseshoe bats have complex recombination histories permitting generation of novel viral variants, as reported by others (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">He et al., 2014</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Hon et al., 2008</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hu et al., 2017</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li et al., 2020b</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lin et al., 2017</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wang et al., 2017</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Yuan et al., 2010</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Zhou et al., 2020a</a>). Interestingly, despite the lineage leading to SARS-CoV-2 acquiring residues in its Spike (S) protein’s receptor-binding domain (RBD) permitting use of human ACE2 (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wan et al., 2020</a>) and being closer to the pangolin virus than RaTG13 in this region (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Lam et al., 2020</a>) – a signal that suggests recombination – the divergence patterns in the S protein show no evidence of recombination between SARS-CoV-2 and known sarbecoviruses. Our results indicate the presence of a single lineage circulating in bats with properties that allowed it to infect human cells, as previously described for the bat sarbecoviruses related to the first SARS-CoV lineage (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ge et al., 2013</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Menachery et al., 2015</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">2016</a>).</p><p id="p-14">To gauge the length of time this lineage has circulated in bats we estimate the time to most recent common ancestor (tMRCA) of SARS-CoV-2 and RaTG13. We use three bioinformatic approaches to remove the effects of recombination, and we combine these approaches to identify putative non-recombinant regions that can be used for reliable phylogenetic reconstruction and dating. Collectively our analyses point to bats being the primary reservoir for the SARS-CoV-2 lineage. While it is possible that pangolins may have acted as an intermediate species facilitating transmission to humans, the evidence is consistent with the virus having evolved in bats resulting in bat sarbecoviruses that can replicate in the upper respiratory tract of both humans and pangolins (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Zhang and Holmes, 2020</a>; <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Zhou et al., 2020a</a>).</p></div><div class="section" id="sec-4"><h2 class="">Results</h2><div id="sec-5" class="subsection"><h3>Recombination analysis and identification of breakpoint-free genome regions</h3><p id="p-15">Of the 68 sequences in the aligned sarbecovirus sequence set, 67 show evidence of mosaicism, indicating involvement in homologous recombination either directly with identifiable parentals or in their deeper shared evolutionary history, i.e., due to shared ancestral recombination events (all Dunn-Sidak corrected <em>p</em> &lt; 4 × 10<sup>−4</sup>, 3SEQ (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Lam et al., 2018</a>)). This is evidence for numerous recombination events occurring in the evolutionary history of the sarbecoviruses (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Li et al., 2020b</a>); however, identifying all past events and their temporal order (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Eden et al., 2013</a>) is challenging. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1 (top)</a> shows the distribution of all identified breakpoints (using 3SEQ’s exhaustive triplet search) by the number of candidate recombinant sequences supporting them. The histogram allows for the identification of non-recombining regions by revealing regions with no breakpoints. Sorting these breakpoint-free regions (BFRs) by length results in two segments longer than 5kb: an ORF1a sub-region spanning nucleotides 3625-9150 and the first half of ORF1b spanning nucleotides 13291-19628. Eight other BFRs longer than 500nt were identified. Of the nine breakpoints (defining these ten BFRs), four showed phylogenetic incongruence (PI) signals with bootstrap support higher than 80%, adopting previously published criteria on using a combination of mosaic and PI-signals to show evidence of past recombination events (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Boni et al., 2010</a>). All four of these breakpoints were also identified with the tree-based recombination detection method GARD (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kosakovsky Pond et al., 2006</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Top: Breakpoints identified by 3SEQ shown by the percentage of sequences (out of 68) that support a particular breakpoint position. Note that breakpoints can be shared between sequences if they are descendants of the same recombination events. The pink, green, and orange bars show breakpoint-free regions (BFRs), with Region A (nt 13291-19628) showing two trimmed segments to yield Region A&#x2019; (nt 13291-14932, 15405-17162, 18009-19628). Region B spans nucleotides 3625-9150, and region C spans 9261-11795. Concatenated region A&#x2019;BC is non-recombining region 1 (NRR1). Open reading frames are shown above the breakpoint plot, with the variable loop region indicated in the S protein. Center: Similarity plot between SARS-CoV-2 and several selected sequences including RaTG13 (black), SARS 2002-2003 (pink), and two pangolin sequences (orange). Shaded region corresponds to the S protein. Bottom: Maximum likelihood phylogenetic trees rooted on a 2007 virus sampled in Kenya (BtKy72; root truncated from images), shown for five breakpoint-free regions of the sarbecovirus alignment. Nucleotide positions for phylogenetic inference are 147-695, 962-1686 (first tree), 3625-9150 (second tree, also BFR B), 9261-11795 (third tree, also BFR C), 12443-19638 (fourth tree), and 23631-24633, 24795-25847, 27702-28843, 29574-30650 (fifth tree). Relevant bootstrap values are shown on branches, and gray-shaded regions show sequences that exhibit phylogenetic incongruence along the genome. SC China corresponds to provinces in southcentral China, specifically Yunnan, Guizhou, and Guangxi provinces. NE China corresponds to provinces in northeast China: Jilin, Shaanxi, Shanxi, Hebei, and Henan provinces." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1968036515" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Top: Breakpoints identified by 3SEQ shown by the percentage of sequences (out of 68) that support a particular breakpoint position. Note that breakpoints can be shared between sequences if they are descendants of the same recombination events. The pink, green, and orange bars show breakpoint-free regions (BFRs), with Region &lt;strong&gt;A&lt;/strong&gt; (nt 13291-19628) showing two trimmed segments to yield Region &lt;strong&gt;A&#x2019;&lt;/strong&gt; (nt 13291-14932, 15405-17162, 18009-19628). Region &lt;strong&gt;B&lt;/strong&gt; spans nucleotides 3625-9150, and region &lt;strong&gt;C&lt;/strong&gt; spans 9261-11795. Concatenated region &lt;strong&gt;A&#x2019;BC&lt;/strong&gt; is non-recombining region 1 (&lt;strong&gt;NRR1&lt;/strong&gt;). Open reading frames are shown above the breakpoint plot, with the variable loop region indicated in the S protein. Center: Similarity plot between SARS-CoV-2 and several selected sequences including RaTG13 (black), SARS 2002-2003 (pink), and two pangolin sequences (orange). Shaded region corresponds to the S protein. Bottom: Maximum likelihood phylogenetic trees rooted on a 2007 virus sampled in Kenya (BtKy72; root truncated from images), shown for five breakpoint-free regions of the sarbecovirus alignment. Nucleotide positions for phylogenetic inference are 147-695, 962-1686 (first tree), 3625-9150 (second tree, also BFR &lt;strong&gt;B&lt;/strong&gt;), 9261-11795 (third tree, also BFR &lt;strong&gt;C&lt;/strong&gt;), 12443-19638 (fourth tree), and 23631-24633, 24795-25847, 27702-28843, 29574-30650 (fifth tree). Relevant bootstrap values are shown on branches, and gray-shaded regions show sequences that exhibit phylogenetic incongruence along the genome. SC China corresponds to provinces in southcentral China, specifically Yunnan, Guizhou, and Guangxi provinces. NE China corresponds to provinces in northeast China: Jilin, Shaanxi, Shanxi, Hebei, and Henan provinces.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F1.medium.gif" width="440" height="312"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F1.medium.gif" width="440" height="312"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-16" class="first-child"><span class="underline">Top:</span> Breakpoints identified by 3SEQ shown by the percentage of sequences (out of 68) that support a particular breakpoint position. Note that breakpoints can be shared between sequences if they are descendants of the same recombination events. The pink, green, and orange bars show breakpoint-free regions (BFRs), with Region <strong>A</strong> (nt 13291-19628) showing two trimmed segments to yield Region <strong>A’</strong> (nt 13291-14932, 15405-17162, 18009-19628). Region <strong>B</strong> spans nucleotides 3625-9150, and region <strong>C</strong> spans 9261-11795. Concatenated region <strong>A’BC</strong> is non-recombining region 1 (<strong>NRR1</strong>). Open reading frames are shown above the breakpoint plot, with the variable loop region indicated in the S protein. <span class="underline">Center:</span> Similarity plot between SARS-CoV-2 and several selected sequences including RaTG13 (black), SARS 2002-2003 (pink), and two pangolin sequences (orange). Shaded region corresponds to the S protein. <span class="underline">Bottom:</span> Maximum likelihood phylogenetic trees rooted on a 2007 virus sampled in Kenya (BtKy72; root truncated from images), shown for five breakpoint-free regions of the sarbecovirus alignment. Nucleotide positions for phylogenetic inference are 147-695, 962-1686 (first tree), 3625-9150 (second tree, also BFR <strong>B</strong>), 9261-11795 (third tree, also BFR <strong>C</strong>), 12443-19638 (fourth tree), and 23631-24633, 24795-25847, 27702-28843, 29574-30650 (fifth tree). Relevant bootstrap values are shown on branches, and gray-shaded regions show sequences that exhibit phylogenetic incongruence along the genome. SC China corresponds to provinces in southcentral China, specifically Yunnan, Guizhou, and Guangxi provinces. NE China corresponds to provinces in northeast China: Jilin, Shaanxi, Shanxi, Hebei, and Henan provinces.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">The extent of the sarbecoviruses’ recombination history can be demonstrated by five phylogenetic trees inferred from BFRs or concatenated adjacent BFRs (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, bottom</a>). BFRs were concatenated if no phylogenetic incongruence signal could be identified between them. When viewing the last 7kb of the genome a clade of viruses from northeast (NE) China appears to cluster with sequences from south-central (SC) Chinese provinces, but when inspecting trees from different parts of ORF1ab the NE China clade is phylogenetically separated from the SC China clade. Individual sequences such as RpShaanxi2011, Guangxi GX2013, and two sequences from Zhejiang province (CoVZXC21/CoVZC45), as previously shown (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Li et al., 2020b</a>; <a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Zhou et al., 2020a</a>), show strong phylogenetic recombination signals, as they fall on different evolutionary lineages (with bootstrap support &gt; 80%) depending what region of the genome is being examined.</p><p id="p-18">Despite this high frequency of recombination among bat viruses, the ‘block-like’ nature of the genome permits retrieval of a clean sub-alignment for phylogenetic analysis. Conservatively, we can combine three &gt;2kb BFRs identified above into a putative non-recombining region 1 (<strong>NRR1</strong>), after removing five sequences that appear to be recombinants and two small sub-regions of the longer region (nucleotides 13291-19628). Alternatively, combining 3SEQ-inferred breakpoints with GARD-inferred breakpoints and the necessity of PI-signals for inferring recombination, we can use the 9.9kb region spanning nucleotides 11885-21753 (<strong>NRR2</strong>) as a putative non-recombining region; this approach is conservative in identifying breakpoints but not conservative in identifying non-recombining regions. Using a third consensus-based approach for identifying recombinant regions in individual sequences – with six different recombination detection methods in RDP5 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Martin et al., 2015</a>) – gives a putative recombination-free alignment that we call non-recombinant alignment 3 (<strong>NRA3</strong>) (see <em>Methods</em>).</p><p id="p-19">All three approaches to removing recombinant genomic segments point to a single ancestral lineage for SARS-CoV-2 and RaTG13. Two other bat viruses (CoVZXC21 and CoVZC45) from Zhejiang province fall on this lineage as recombinants of the RaTG13/SARS-CoV-2 lineage and the clade of Hong Kong bat viruses sampled between 2005 and 2007 (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, bottom</a>). Specifically, progenitors of the RaTG13/SARS-CoV-2 lineage appear to have recombined with the Hong Kong clade (with inferred breakpoints at 11.9kb and 20.8kb) to form the CoVZXC21/CoVZC45-lineage. Ancestors to the RaTG13/SARS-CoV-2 lineage also include a pangolin sequence sampled in Guangdong province in March 2019 and a clade of pangolin sequences from Guangxi province sampled in 2017. Major phylogenetic and phylogeographic relationships are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, with phylogenies reconstructed for two major sub-regions of <strong>NRR1</strong>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Maximum likelihood trees of the sarbecoviruses using the two longest breakpoint-free regions (BFRs), rooted on the Kenya/Bulgaria lineage. Region A has been shortened to A&#x2019; (5017nt) based on potential recombination signals within the region. Region B is 5525nt long. Sequences are color-coded by province according to the map. Five example sequences with incongruent phylogenetic positions in the two trees are indicated by dashed lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1968036515" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Maximum likelihood trees of the sarbecoviruses using the two longest breakpoint-free regions (BFRs), rooted on the Kenya/Bulgaria lineage. Region &lt;strong&gt;A&lt;/strong&gt; has been shortened to &lt;strong&gt;A&#x2019;&lt;/strong&gt; (5017nt) based on potential recombination signals within the region. Region &lt;strong&gt;B&lt;/strong&gt; is 5525nt long. Sequences are color-coded by province according to the map. Five example sequences with incongruent phylogenetic positions in the two trees are indicated by dashed lines.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F2.medium.gif" width="440" height="250"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F2.medium.gif" width="440" height="250"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-20" class="first-child">Maximum likelihood trees of the sarbecoviruses using the two longest breakpoint-free regions (BFRs), rooted on the Kenya/Bulgaria lineage. Region <strong>A</strong> has been shortened to <strong>A’</strong> (5017nt) based on potential recombination signals within the region. Region <strong>B</strong> is 5525nt long. Sequences are color-coded by province according to the map. Five example sequences with incongruent phylogenetic positions in the two trees are indicated by dashed lines.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">As the SARS-CoV-2 S protein has been implicated in past recombination events or possibly convergent evolution (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Lam et al., 2020</a>), we specifically investigated several sub-regions of the S protein – the N-terminal domain of S1, the C-terminal domain of S1, the variable loop region of the C-terminal domain, and S2. The variable loop region in SARS-CoV-2 shows closer identity to the 2019 pangolin coronavirus sequence than to the RaTG13 bat virus, supported by phylogenetic inference (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). On first examination this would suggest that that SARS-CoV-2 is a recombinant of an ancestor of Pangolin-2019 and RaTG13. However, on closer inspection, the relative divergences in the phylogenetic tree (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, bottom</a>) show that SARS-CoV-2 is unlikely to have acquired the variable loop from an ancestor of Pangolin-2019 as these two sequences are approximately 10% divergent throughout the entire S protein (excluding the NTD). It is RaTG13 that is divergent in the variable loop region and is the likely product of recombination, acquiring a divergent variable loop from an as yet unsampled bat sarbecovirus. This is notable because the variable loop region contains the six key contact residues in the receptor binding domain (RBD) that give SARS-CoV-2 its ACE2 binding specificity (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Anderson et al., 2020</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Wan et al., 2020</a>). These residues are also in the 2019 pangolin coronavirus sequence. The most parsimonious explanation for these shared residues is a single bat virus lineage (including the common ancestor of SARS-CoV-2, RaTG13, and Pangolin Guangdong 2019) has the ACE2 specific residues, rather than the SARS-CoV-2 being recombinant. This provides compelling support for the SARS-CoV-2 lineage being the consequence of a direct or nearly-direct zoonotic jump from bats because the key ACE2 binding residues were present in viruses circulating in bats.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic relationships among SARS-CoV-2 and closely related sequences for sub-regions of the S protein. SARS-CoV-2 and RaTG13 are the most closely related (their most recent common ancestor node denoted by a green circle), except in the 220nt variable loop region of the C-terminal domain (bar graphs at bottom). In the variable loop region, RaTG13 diverges considerably with the tMRCA now outside that of SARS-CoV-2 and Pangolin Guangdong 2019 ancestor, suggesting that RaTG13 has acquired this region from a more divergent and undetected bat lineage. The genetic distances between SARS-CoV-2 and RaTG13 (bottom panel) demonstrate their relationship is consistent across all regions except for the variable loop. The genetic distances between SARS-CoV-2 and Pangolin Guangdong 2019 are consistent across all regions except the NTD." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1968036515" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Phylogenetic relationships among SARS-CoV-2 and closely related sequences for sub-regions of the S protein. SARS-CoV-2 and RaTG13 are the most closely related (their most recent common ancestor node denoted by a green circle), except in the 220nt variable loop region of the C-terminal domain (bar graphs at bottom). In the variable loop region, RaTG13 diverges considerably with the tMRCA now outside that of SARS-CoV-2 and Pangolin Guangdong 2019 ancestor, suggesting that RaTG13 has acquired this region from a more divergent and undetected bat lineage. The genetic distances between SARS-CoV-2 and RaTG13 (bottom panel) demonstrate their relationship is consistent across all regions except for the variable loop. The genetic distances between SARS-CoV-2 and Pangolin Guangdong 2019 are consistent across all regions except the NTD.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F3.medium.gif" width="421" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F3.medium.gif" width="421" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-22" class="first-child">Phylogenetic relationships among SARS-CoV-2 and closely related sequences for sub-regions of the S protein. SARS-CoV-2 and RaTG13 are the most closely related (their most recent common ancestor node denoted by a green circle), except in the 220nt variable loop region of the C-terminal domain (bar graphs at bottom). In the variable loop region, RaTG13 diverges considerably with the tMRCA now outside that of SARS-CoV-2 and Pangolin Guangdong 2019 ancestor, suggesting that RaTG13 has acquired this region from a more divergent and undetected bat lineage. The genetic distances between SARS-CoV-2 and RaTG13 (bottom panel) demonstrate their relationship is consistent across all regions except for the variable loop. The genetic distances between SARS-CoV-2 and Pangolin Guangdong 2019 are consistent across all regions except the NTD.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Ancestry in non-recombinant regions</h3><p id="p-23">Using the most conservative approach to identifying a non-recombinant genomic region (<strong>NRR1</strong>), SARS-CoV-2 forms a sister lineage with RaTG13, with genetically related cousin lineages of coronavirus sampled in pangolins in Guangdong and Guangxi provinces (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Given that these pangolin viruses are ancestral to the progenitor of the RaTG13/SARS-CoV-2 lineage, it is more likely that they are also picking up viruses from bats. While pangolins could be acting as intermediate hosts for bat viruses to get into humans — they develop severe respiratory disease (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lie et al., 2019</a>) and commonly come into contact with people as they are trafficked in large numbers for consumption and use in Chinese medicine — pangolin infection is not a requirement for bat viruses to cross into humans.</p><p id="p-24">Phylogenies of sub-regions of <strong>NRR1</strong> depict an appreciable degree of spatial structuring of the sarbecoviruses population across different regions (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). One cluster consists of viruses from provinces in southcentral (SC) China (Guangxi, Yunnan, Guizhou) as well as one sequence from Sichuan province. The major cluster in its sister lineage is almost exclusively occupied by viruses from provinces in northeast (NE) and central China (Hubei, Shaanxi, Shanxi, Henan, Hebei, Zheijang and Jilin).</p></div><div id="sec-7" class="subsection"><h3>tMRCA for non-recombinant regions of SARS-CoV-2 lineage</h3><p id="p-25">To avoid artefacts due to recombination, we focused on the non-recombining regions <strong>NRR1</strong>, <strong>NRR2</strong>, and the recombination-masked alignment <strong>NRA3</strong> for inferring time-measured evolutionary histories. Visual exploration using TempEst (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Rambaut et al., 2016</a>) indicates there is no evidence for temporal signal in these data sets (Figure S1). This is not surprising for diverse viral populations with relatively deep evolutionary histories. In such cases, even moderate rate variation among long deep phylogenetic branches will significantly impact expected root-to-tip divergences over a sampling time range that represents only a small fraction of the evolutionary history (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Trova et al., 2015</a>). However, formal testing using marginal likelihood estimation (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Duchene et al., 2019</a>) does not reject the absence of temporal signal in all three data sets (Table S1), albeit without strong support in favor of temporal signal (log Bayes factor support of 3, 10, and 3 for <strong>NRR1</strong>, <strong>NRR2</strong>, and <strong>NRA3</strong> respectively).</p><p id="p-26">In the absence of a strong temporal signal, we sought to identify a suitable prior rate distribution to calibrate the time-measured trees by examining several coronaviruses sampled through time, including HCoV-OC43, MERS-CoV, and SARS-CoV virus genomes. These data sets were subjected to the same recombination masking approach as <strong>NRA3</strong> and were characterized by a strong temporal signal (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), but also by markedly different evolutionary rates. Specifically, using a formal Bayesian approach (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Suchard et al., 2018</a>) (see <em>Methods</em>), we estimate a fast evolutionary rate (0.00169 subst/site/yr, 95% highest posterior density (HPD) interval [0.00131,0.00205]) for SARS viruses sampled over a limited time scale (1 year), a slower rate (0.00078 [0.00063,0.00092] subst/site/yr) for MERS-CoV on a time scale of about 4 years, and the slowest rate (0.00024 [0.00019,0.00029] subst/site/yr) for HCoV-OC43 over almost five decades. These differences reflect the fact that rate estimates can vary considerably with the time scale of measurement, a frequently observed phenomenon in viruses known as time-dependent evolutionary rates (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aiewsakun and Katzourakis, 2016</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Duchene et al., 2014</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Membrebe et al., 2019</a>). Over relatively shallow time-scales, such differences can primarily be explained by varying selective pressure with mildly deleterious variants being eliminated more strongly by purifying selection over longer time scales (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Holmes, 2009</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Holmes et al., 2016</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Membrebe et al., 2019</a>). In line with this, we estimate a concomitantly decreasing nonsynonymous-to-synonymous substitution rate ratio (dN/dS) over longer evolutionary time-scales: 1.41 [1.20,1.68], 0.35 [0.30,0.41], and 0.133 [0.129,0.136] for SARS, MERS-CoV, and HCoV-OC43 respectively. In the light of these time-dependent evolutionary rate dynamics, a rate in the slow range is appropriate for calibrating the sarbecovirus evolutionary history, but we compare both MERS-CoV and HCoV-OC43 centred prior distributions (Figure S2) with relatively large variances in our subsequent analyses in order to examine the sensitivity of the date estimates to this prior specification.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Temporal signal and mean evolutionary rate estimates for HCoV-OC43, MERS, and SARS coronaviruses. (A-C) Root-to-tip divergence as a function of sampling time for the three coronavirus evolutionary histories unfolding over different time scales. (D) Mean evolutionary rate estimates plotted against sampling time range for the same three data sets (represented by the same color as the data points in their respective root-to-tip divergence plots), as well as for the comparable recombination-masked sarbecovirus alignment (NRA3) using the two different priors for the rate in the Bayesian inference (red points)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1968036515" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Temporal signal and mean evolutionary rate estimates for HCoV-OC43, MERS, and SARS coronaviruses. (&lt;strong&gt;A-C&lt;/strong&gt;) Root-to-tip divergence as a function of sampling time for the three coronavirus evolutionary histories unfolding over different time scales. (&lt;strong&gt;D&lt;/strong&gt;) Mean evolutionary rate estimates plotted against sampling time range for the same three data sets (represented by the same color as the data points in their respective root-to-tip divergence plots), as well as for the comparable recombination-masked sarbecovirus alignment (&lt;strong&gt;NRA3&lt;/strong&gt;) using the two different priors for the rate in the Bayesian inference (red points).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F4.medium.gif" width="440" height="338"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F4.medium.gif" width="440" height="338"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-27" class="first-child">Temporal signal and mean evolutionary rate estimates for HCoV-OC43, MERS, and SARS coronaviruses. (<strong>A-C</strong>) Root-to-tip divergence as a function of sampling time for the three coronavirus evolutionary histories unfolding over different time scales. (<strong>D</strong>) Mean evolutionary rate estimates plotted against sampling time range for the same three data sets (represented by the same color as the data points in their respective root-to-tip divergence plots), as well as for the comparable recombination-masked sarbecovirus alignment (<strong>NRA3</strong>) using the two different priors for the rate in the Bayesian inference (red points).</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">We infer time-measured evolutionary histories using a Bayesian phylogenetic approach while incorporating rate priors based on the mean MERS-CoV and HCoV-OC43 rates and with standard deviations that allow for more uncertainty than the empirical estimates for both viruses (see <em>Methods</em>). Using both prior distributions, this results in six highly similar posterior rate estimates for <strong>NRR1</strong>, <strong>NRR2</strong>, and <strong>NRA3</strong>, centered around 0.00055 subst/site/yr. The fact that these estimates lie in between the MERS-CoV and HCoV-OC43 rate is consistent with the intermediate sampling time range of about 18 years (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). The consistency of the posterior rates for the different prior means also implies that the data do contribute to the evolutionary rate estimate despite the fact that a temporal signal was visually not apparent (Figure S1). Below, we report divergence time estimates based on the HCoV-OC43-centred rate priors, but also summarize similar corresponding estimates for the MERS-CoV-centred rate priors in Supplementary Figure S3.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Time-measured phylogenetic estimates and divergence times for sarbecovirus lineages using an HCoV-OC43 centred rate prior. The time-calibrated phylogeny represents a maximum clade credibility tree inferred for NRR1. Grey tips correspond to bat viruses, green to pangolin, blue to the SARS-CoV virus, and red to SARS-CoV-2. The size of the black internal node circles are proportional to the posterior node support. 95% credible intervals bars are shown for all internal node ages. The inset represents divergence time estimates based on NRR1, NRR2 and NRA3. The boxplots represent the divergence time estimates for SARS-CoV-2 (red boxplot) and the 2002-2003 SARS-CoV virus (blue boxplot) from their most closely related bat virus. Green boxplots show the TMRCA estimate for the RaTG13/SARS-CoV-2 lineage and its most closely related pangolin lineage (Guangdong 2020). Transparent boxes of the interquartile range width and with the same colours are superimposed to highlight the overlap between the estimates. In Supplementary Figure S3, we compare these divergence time estimates to those obtained using the MERS-CoV-centred rate priors for NRR1, NRR2 and NRA3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1968036515" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Time-measured phylogenetic estimates and divergence times for sarbecovirus lineages using an HCoV-OC43 centred rate prior. The time-calibrated phylogeny represents a maximum clade credibility tree inferred for &lt;strong&gt;NRR1&lt;/strong&gt;. Grey tips correspond to bat viruses, green to pangolin, blue to the SARS-CoV virus, and red to SARS-CoV-2. The size of the black internal node circles are proportional to the posterior node support. 95% credible intervals bars are shown for all internal node ages. The inset represents divergence time estimates based on &lt;strong&gt;NRR1&lt;/strong&gt;, &lt;strong&gt;NRR2&lt;/strong&gt; and &lt;strong&gt;NRA3&lt;/strong&gt;. The boxplots represent the divergence time estimates for SARS-CoV-2 (red boxplot) and the 2002-2003 SARS-CoV virus (blue boxplot) from their most closely related bat virus. Green boxplots show the TMRCA estimate for the RaTG13/SARS-CoV-2 lineage and its most closely related pangolin lineage (Guangdong 2020). Transparent boxes of the interquartile range width and with the same colours are superimposed to highlight the overlap between the estimates. In Supplementary Figure S3, we compare these divergence time estimates to those obtained using the MERS-CoV-centred rate priors for &lt;strong&gt;NRR1&lt;/strong&gt;, &lt;strong&gt;NRR2&lt;/strong&gt; and &lt;strong&gt;NRA3&lt;/strong&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F5.medium.gif" width="440" height="332"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F5.medium.gif" width="440" height="332"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015008/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <p id="p-29" class="first-child">Time-measured phylogenetic estimates and divergence times for sarbecovirus lineages using an HCoV-OC43 centred rate prior. The time-calibrated phylogeny represents a maximum clade credibility tree inferred for <strong>NRR1</strong>. Grey tips correspond to bat viruses, green to pangolin, blue to the SARS-CoV virus, and red to SARS-CoV-2. The size of the black internal node circles are proportional to the posterior node support. 95% credible intervals bars are shown for all internal node ages. The inset represents divergence time estimates based on <strong>NRR1</strong>, <strong>NRR2</strong> and <strong>NRA3</strong>. The boxplots represent the divergence time estimates for SARS-CoV-2 (red boxplot) and the 2002-2003 SARS-CoV virus (blue boxplot) from their most closely related bat virus. Green boxplots show the TMRCA estimate for the RaTG13/SARS-CoV-2 lineage and its most closely related pangolin lineage (Guangdong 2020). Transparent boxes of the interquartile range width and with the same colours are superimposed to highlight the overlap between the estimates. In Supplementary Figure S3, we compare these divergence time estimates to those obtained using the MERS-CoV-centred rate priors for <strong>NRR1</strong>, <strong>NRR2</strong> and <strong>NRA3</strong>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">The divergence time estimates for SARS-CoV-2 and SARS-CoV from their respective most closely related bat lineages are reasonably consistent among the three approaches we use to eliminate the effects of recombination in the alignment. Using the most conservative approach (<strong>NRR1</strong>), the divergence time estimate for SARS-CoV-2 and RaTG13 is 1969 (95% HPD: 1930-2000) while the divergence time estimate between SARS-CoV and its most closely related bat sequence is 1962 (95% HPD: 1932-1988); see <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>. These are in general agreement with estimates using <strong>NRR2</strong> and <strong>NRA3</strong>, which result in divergence times of 1982 [1948,2009] and 1948 [1879-1999], respectively, for SARS-CoV-2, and estimates of 1952 [1906, 1989] and 1970 [1932-1996], respectively, for the divergence time of SARS-CoV from its closest known bat ancestor. The SARS-CoV divergence times are somewhat earlier than previously estimated dates (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Hon et al., 2008</a>) as previous estimates were obtained using a collection of SARS-CoV genomes from human and civet hosts (as well as a few closely related bat genomes) which implies that the evolutionary rates were predominantly informed by the short-term SARS outbreak scale and likely biased upwards for the time scale under investigation. Indeed, the rates reported by these studies are in line with the short-term SARS rates we estimate (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). The estimated divergence times for the pangolin virus most closely related to SARS-CoV-2/RaTG13 range from 1851 [1730,1958] to 1877 [1746,1986], indicating that these pangolin lineages were acquired from bat viruses divergent to those that gave rise to SARS-CoV-2. Current sampling of pangolins does not implicate them as an intermediate host.</p></div></div><div class="section" id="sec-8"><h2 class="">Discussion</h2><p id="p-31">Identifying the origins of an emerging pathogen can be critical during the early stages of an outbreak as it may allow for containment measures to be precisely targeted at a stage when the number of daily new infections is still low. Early detection via genomics and sequencing was not possible during SE Asia’s initial outbreaks of avian influenza H5N1 (1997, 2003-2004) or the first SARS outbreak (2002-2003). By 2009 however rapid genomic analysis was a routine component of outbreak response. The 2009 influenza pandemic and subsequent outbreaks of MERS-CoV (2012), H7N9 avian influenza (2013), Ebola virus (2014), and Zika virus (2015) were met with rapid sequencing and genomic characterization. For the current pandemic, the ‘novel pathogen identification’ component of outbreak response delivered on its promise, with viral identification and rapid genomic analysis providing confirmation and a genome sequence, within weeks, that the December 2019 outbreak in Wuhan was caused by a coronavirus (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">World Health Organization, 2020a</a>). Unfortunately, a response that would achieve containment was not possible. Given what was known about the origins of SARS as well as identification of SARS-like viruses circulating in bats that had binding sites adapted to human receptors (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Ge et al., 2013</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Menachery et al., 2015</a>, <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">2016</a>), appropriate measures should have been in place for outbreaks of novel coronaviruses. The key to successful surveillance is that we know which human-adapted viral phenotypes to look for (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Holmes et al., 2018</a>).</p><p id="p-32">The key difficulty in inferring reliable evolutionary histories for coronaviruses is that their high recombination rate (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Graham and Baric, 2010</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Su et al., 2016</a>) violates the assumption of standard phylogenetic approaches because different parts of the genome will have different origins. In order to begin characterizing any ancestral relationships for SARS-CoV-2, non-recombinant regions of the genome must be identified, so that reliable phylogenetic reconstruction and dating can be performed. Evolutionary rate estimation can be profoundly affected by the presence of recombination (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Schierup and Hein, 1999</a>). As there is no single accepted method of inferring breakpoints and identifying clean sub-regions with high certainty, we implemented several approaches to identifying three classic statistical signals of recombination: mosaicism, phylogenetic incongruence and excessive homoplasy (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Posada et al., 2002</a>). Our most conservative approach attempted to ensure that putative non-recombining regions had no mosaic or phylogenetic incongruence signals. A second approach was conservative with respect to breakpoint identification, while a third approach attempted to minimize the number of regions removed while still minimizing signals of mosaicism and homoplasy. The origins we present in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a> (<strong>NRR1</strong>) are conservative in the sense that <strong>NRR1</strong> is more likely to be non-recombinant than <strong>NRR2</strong> or <strong>NRA3</strong>. As the estimated rates and divergence dates were highly similar in the three data sets analyzed, we conclude that our estimates are robust to the method of identifying the genome’s non-recombinant regions.</p><p id="p-33">Due to the absence of temporal signal in the sarbecovirus data sets, we used informative prior distributions on the evolutionary rate to estimate divergence dates. We show that such prior calibration information can be obtained from other coronaviruses (SARS-CoV, MERS-CoV, and HCoV-OC43), but their rates vary with the time scale of sample collection. In the presence of time-dependent rate variation, a widely observed phenomenon for viruses (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Duchene et al., 2014</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Aiewsakun and Katzourakis, 2016</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Membrebe et al., 2019</a>), the slower rates appear more appropriate for sarbecoviruses that currently encompass a sampling time range of about 18 years. We therefore centered the prior rate distributions on the mean HCoV-OC43 and MERS-CoV rate, but permitted the distributions a greater variance than the empirical rate estimates for these lineages. This approach resulted in similar posterior rates despite the different prior means, implying that the sarbecovirus data do inform the rate estimate even though root-to-top temporal signal was not apparent.</p><p id="p-34">The relatively fast evolutionary rate means that it is most appropriate to estimate shallow nodes in the sarbecovirus evolutionary history including the divergence times of SARS-CoV and SARS-CoV-2 from their most closely related bat viruses. Accurately estimating deeper nodes would require adequately accommodating time-dependent rate variation. While such models have recently been made available, we lack the information to calibrate the rate decline through time (e.g. through internal node calibrations (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Membrebe et al., 2019</a>)). As a proxy, it would be possible to model the long-term purifying selection dynamics as a major source of time-dependent rates rates (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Duchene et al., 2014</a>; <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Aiewsakun and Katzourakis, 2016</a>; <a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Membrebe et al., 2019</a>), but this is beyond the scope of the current study. The assumption of long-term purifying selection would imply that coronaviruses are at endemic equilibrium with their natural host species, horseshoe bats, to which they are presumably well-adapted. Although there is currently little evidence supporting or contradicting strong positive selection in the sarbecovirus lineage, if multiple host species were to be identified that are able to support endemic viral transmission, then the positive selection that would be associated with new species adaptations would need to be considered when inferring evolutionary rate variation through time.</p><p id="p-35">Of importance for future emergence events is the appreciation that SARS-CoV-2 has emerged from the same horseshoe bat subgenus that harbours SARS-like coronaviruses. Another similarity between SARS-CoV and SARS-CoV-2 is their divergence time (40-70 years ago) from currently known extant bat-virus lineages (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). This long divergence period indicates there are unsampled viruses circulating in horseshoe bats that have zoonotic potential (<a id="xref-ref-62-4" class="xref-bibr" href="#ref-62">Zhou et al., 2020a</a>). While there is clear involvement of other mammalian species – specifically pangolins for SARS-CoV-2 – as a plausible conduit for transmission to humans, there is no evidence pangolins are facilitating adaptation to humans. A hypothesis of snakes as intermediate hosts of SARS-CoV-2 was posited during the early epidemic phase (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ji et al., 2020</a>), but we found no evidence of this (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anderson, 2020</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Robertson, 2020</a>); see Supplementary Section 3.</p><p id="p-36">With horseshoe bats currently the most plausible source for SARS-CoV-2, it is important to consider that sarbecoviruses circulate in a variety of horseshoe bat species with often widely overlapping species ranges (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wong et al., 2019</a>). Yet, the viral population is largely spatially structured according to provinces in the south and southeast on one lineage, and provinces in the center, east, and northeast on another (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). This boundary appears to be rarely crossed. Two exceptions can be seen in the relatively close relationship of viruses from Hong Kong bats to viruses from Zheijang (with two of the latter, CoVZC45 and CoVZXC21, identified as recombinants) and a recombinant virus from Sichuan for which part of the genome (region B of SC2018 in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>) clusters with viruses from provinces in the center, east, and northeast of China. SARS-CoV-2 and RaTG13 are also exceptions as they were sampled from Hubei and Yunnan, respectively. The fact that they are geographically relatively distant is in agreement with their somewhat distant tMRCA because the spatial structure suggests that migration between their locations may be uncommon. From this perspective, it may be useful to perform surveillance for more closely related virus to SARS-CoV-2 along the gradient from Yunnan to Hubei.</p><p id="p-37">It is clear from our analysis that viruses closely related to SARS-CoV-2 have been circulating in horseshoe bats for many decades. The substantial unsampled diversity on the SARS-CoV-2/RaTG13 lineage suggests that there is a major clade of bat sarbecoviruses with generalist properties – with respect to their ability to infect a range of mammalian cells – that facilitated its jump to humans and may do so again. Although the human ACE2-compatible receptor binding domain was very likely to have been present in a bat sarbecovirus lineage that ultimately lead to SARS-CoV-2, this RBD sequence has thus far only been found in a few pangolin viruses. Furthermore, the other key feature thought to be instrumental to SARS-CoV-2’s ability to infect humans – a polybasic cleavage site insertion in the Spike protein – has been seen in another close bat relative of the SARS-CoV-2 virus (<a id="xref-ref-62-5" class="xref-bibr" href="#ref-62">Zhou et al., 2020a</a>); however, the sequence is different and it is likely an independent event.</p><p id="p-38">The existing diversity and dynamic process of recombination amongst lineages in the bat reservoir demonstrate how difficult it will be to identify viruses with potential to cause significant human outbreaks before they emerge. This underscores the need for a global network of real-time human disease surveillance systems like that which identified the unusual cluster of pneumonia in Wuhan in December 2019 and the rapid deployment of functional studies and genomic tools for pathogen identification and characterization.</p></div><div class="section" id="sec-9"><h2 class="">Methods</h2><div id="sec-10" class="subsection"><h3>Data set compilation</h3><p id="p-39">Sarbecovirus data. Complete genome sequence data were downloaded from GenBank and ViPIR; accession numbers available in Supplementary Section 4. Sequences were aligned by MAFTT (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Katoh et al., 2009</a>), with a final alignment length of 30927, and used in the analyses below.</p><p id="p-40">HCoV-OC43. We compiled a data set including 27 human coronavirus OC43 virus genomes and 10 related animal virus genomes (6 bovine, 3 white-tailed deer and one canine virus). The canine viral genome was excluded from the Bayesian phylogenetic analyses because temporal signal analyses (see below) indicated that it was an outlier.</p><p id="p-41">MERS-CoV. We extracted a similar number (<em>n</em> = 35) of genomes from a MERS-CoV data set analysed by <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dudas et al (2018)</a> using the phylogenetic diversity analyser tool (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chernomor et al., 2015</a>).</p><p id="p-42">SARS-CoV. We compiled a set of 69 SARS-CoV genomes including 58 genomes sampled from human and 11 sampled from civets and raccoon dogs. This data set comprises an updated version of the one used in <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Hon et al (2008)</a>.</p></div><div id="sec-11" class="subsection"><h3>Recombination analysis</h3><p id="p-43">As coronaviruses are known to be highly recombinant, we used three different approaches to identify non-recombinant regions to be used in our Bayesian time-calibrated phylogenetic inference.</p><p id="p-44">First, we took an approach that relies on identifying mosaic regions (via 3SEQ) that are also supported by phylogenetic incongruence (PI) signals (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Boni et al., 2010</a>). As 3SEQ is the most powerful the mosaic methods (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Boni et al., 2007</a>), we used 3SEQ to identify the best supported breakpoint history for each potential child (recombinant) sequence in the data set. A single 3SEQ run on the genome alignment resulted in 67 out of 68 sequences supporting some recombination in the past, with multiple candidate breakpoint ranges listed for each putative recombinant. Then, we (a) collected all breakpoints into a single set, (b) complemented this set to generate a set of non-breakpoints, (c) grouped the non-breakpoints into contiguous breakpoint-free regions (BFRs), and (d) sorted these regions by length. A phylogenetic tree – using RAxML v8.2.8 (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Stamatakis, 2014</a>, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">2006</a>), GTR+Γ model, 100 bootstrap replicates – was inferred for each BFR longer than 500nt.</p><p id="p-45">We considered both (1) the possibility that BFRs could be combined into larger non-recombinant regions, and (2) the possibility of further recombination within each BFR.</p><p id="p-46">We named the length-sorted BFRs as: region <strong>A</strong> (nt positions 13291-19628, length=6338nt), region <strong>B</strong> (nt positions 3625-9150, length=5526nt), region <strong>C</strong> (nt positions 9261-11795, length=2535nt), region <strong>D</strong> (nt positions 27702-28843, length=1142nt), and six more through region <strong>J</strong>. Phylogenetic trees for all ten BFRs are shown in Supplementary Figures S5-S14. Regions <strong>A</strong>, <strong>B</strong>, and <strong>C</strong> had similar phylogenetic relationships among the southcentral China bat viruses (Yunnan, Guangxi, Guizhou provinces), the Hong Kong viruses, northeastern Chinese viruses (Jilin, Shaanxi, Shanxi, Hebei, Henan provinces), pangolin viruses, and the SARS-CoV-2 lineage. As these subclades had different phylogenetic relationships in region <strong>D</strong> (Supplementary Section 5), region <strong>D</strong> and shorter BFRs were not further included into a combined putative non-recombinant regions.</p><p id="p-47">Regions <strong>A</strong>, <strong>B</strong>, and <strong>C</strong> were further examined for mosaic signals by 3SEQ; all showed signs of mosaicism. In region <strong>A</strong>, we removed sub-region <strong>A1</strong> (nt positions 3872-4716 within region <strong>A</strong>) and sub-region <strong>A4</strong> (nt 1642-2113) as both of these showed PI-signals with other sub-regions of region <strong>A</strong>. After removal of <strong>A1</strong> and <strong>A4</strong>, we named the new region <strong>A′</strong>. In addition, sequences NC_014470 (Bulgaria 2008), CoVZXC21, CoVZC45, and DQ412042 (Hubei-Yichang) needed to be removed to maintain a clean non-recombinant signal in <strong>A′</strong>. Region <strong>B</strong> showed no PI-signals within the region, except one including sequence SC2018 (Sichuan), thus this sequence was also removed from the set. Region <strong>C</strong> showed no PI-signals within it. Combining regions <strong>A′</strong>, <strong>B</strong>, and <strong>C</strong>, and removing the five mentioned sequences gives us putative non-recombining region 1, or <strong>NRR1</strong>, as an alignment of 63 sequences.</p><p id="p-48">Second, we wanted to construct non-recombinant regions where our approach to breakpoint identification was as conservative as possible. In this approach, we considered a breakpoint supported only if it had three types of statistical support from (1) mosaic signals identified by 3SEQ, (2) PI-signals identified by building trees around 3SEQ’s breakpoints, (3) the GARD algorithm (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kosakovsky Pond et al., 2006</a>) which identifies breakpoints by identifying PI-signals across proposed breakpoints. Since 3SEQ identified 10 BFRs longer than 500nt, we used GARD’s inference on 10, 11, and 12 breakpoints. A reduced sequence set of 25 sequences chosen to capture the breadth of diversity in the sarbecoviruses (obvious recombinants not involving the SARS-CoV-2 lineage were also excluded) was used as GARD is computationally intensive. GARD identified eight breakpoints that were also within 50nt of breakpoints identified by 3SEQ. PI-signals were identified (with bootstrap support &gt;80%) for seven of these eight breakpoints: positions 1684, 3046, 9237, 11885, 21753, 22773, and 24628. Using these breakpoints, the longest putative non-recombining segment (nt 11885-21753) is 9.9kb long, and we call this region <strong>NRR2</strong>. Because this approach to breakpoint identification is conservative, the approach to identifying non-recombinant regions is not conservative.</p><p id="p-49">Our third approach involved identifying breakpoints and masking minor recombinant regions (with gaps, which are treated as unobserved characters in probabilistic phylogenetic approaches). Specifically, we used a combination of six methods implemented in RDP5 (RDP, GENECONV, MaxChi, Bootscan SisScan and 3SEQ) and considered recombination signals detected by more than two methods for breakpoint identification. Except for specifying that sequences are linear, all settings were kept to their defaults. Based on the identified breakpoints in each genome, only the major non-recombinant region is kept in each genome while other regions are masked. To evaluate the performance procedure, we confirmed that the recombination masking resulted in (<em>i</em>) a significantly different outcome of the PHI-test (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bruen et al., 2006</a>), (<em>ii</em>) a removal of well-supported (bootstrap value &gt;95%) incompatible splits in Neighbor-Nets (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bryant and Moulton, 2004</a>), and (<em>iii</em>) a near-complete reduction of mosaic signal as identified by 3SEQ. If the latter still identified non-negligible recombination signal, we removed additional genomes that were identified as the major contributors to the remaining signal. This produced non-recombining alignment <strong>NRA3</strong>, which included 63 of the 68 genomes.</p></div><div id="sec-12" class="subsection"><h3>Bayesian divergence time estimation</h3><p id="p-50">We focused on these three non-recombining regions/alignments for divergence time estimation. This avoids inappropriately modelling evolutionary processes with recombination on strictly bifurcating trees, which can result in different artefacts such as homoplasies that inflate branch lengths and result in apparently longer evolutionary divergence times. In order to examine temporal signal in the sequenced data, we plotted root-to-tip divergence against sampling time using TempEst (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Rambaut et al., 2016</a>) based on a maximum likelihood tree. The latter was reconstructed using IQTREE (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Nguyen et al., 2014</a>) under a General-Time Reversible (GTR) model with a discrete gamma distribution to model among-site rate variation.</p><p id="p-51">Time-measured phylogenetic reconstruction was performed using a Bayesian approach implemented in BEAST (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Suchard et al., 2018</a>). When the genomic data included both coding and noncoding regions we used a single GTR+Γ substitution model; for concatenated coding genes we partitioned the alignment by codon positions and specified an independent GTR+Γ model for each partition with a separate gamma model to accommodate among-site rate variation. We used an uncorrelated relaxed clock model with a lognormal distribution for all data sets, except for the low-diversity SARS data for which we specified a strict molecular clock model. For the HCoV-OC43, MERS-CoV, and SARS data sets we specified flexible skygrid coalescent tree priors. In the absence of any reasonable prior knowledge on the tMRCA of the sarbecovirus data sets (which is required for the grid specification in a skygrid model), we specified a simpler constant size population prior. As informative rate priors for the analysis of the sarbecovirus data sets, we used two different normal prior distributions: one with a mean of 0.00078 and a standard deviation of 0.0003 and one with a mean of 0.00024 and standard deviation of 0.0001. These means are based on the mean rates estimated for MERS-CoV and HCoV-OC43 respectively, while the standard deviations are set ten times larger than the empirical standard deviations to allow more prior uncertainty and avoid strong bias (Supplementary Fig. 2). In our analyses of the sarbecovirus data sets, we incorporated the uncertainty of the sampling dates when exact dates were not available. To estimate nonsynonymous over synonymous rate ratios for the concatenated coding genes, we used the empirical Bayes ‘Renaissance counting’ procedure (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lemey et al., 2012</a>). Temporal signal was tested using a recently developed marginal likelihood estimation procedure (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Duchene et al., 2019</a>) (Table S1).</p><p id="p-52">Posterior distributions were approximated through Markov chain Monte Carlo sampling, which were run sufficiently long to ensure effective sampling sizes &gt; 100. BEAST inferences made use of the BEAGLEv3 library (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Yres et al., 2019</a>) for efficient likelihood computations. We used TreeAnnotator to summarize the posterior tree distributions and annotated the estimated to a maximum clade credibility tree, which was visualized using FigTree.</p></div></div><div class="section" id="sec-14"><h2 class="">Author Contributions</h2><p id="p-53">All authors contributed to analyses and interpretations. DLR and XJ performed recombination and phylogenetic analysis, and annotated the virus names with geographical and sampling dates. AR performed the S recombination analysis. PB and TC performed the codon-usage analysis. MFB performed the recombination analysis for non-recombining regions 1 and 2, breakpoint analysis, and phylogenetic inference on recombinant segments. PL performed recombination analysis for non-recombining alignment 3, calibration of rate of evolution, and phylogenetic reconstruction and dating. TT-YL collected SARS-CoV data. PL, MFB, and DLR wrote the first draft of the manuscript, and all authors contributed to manuscript editing.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-54">We would like to thank all the authors who have kindly deposited and shared genome data on GISAID. Thanks to Trevor Bedford for providing MFB an alignment on which an initial recombination analysis was done. The research leading to these results has received funding from the European Research Council under the European Union’s Horizon 2020 research and innovation programme (grant agreement no. 725422-ReservoirDOCS). DLR is funded by the MRC (MC UU 1201412). The Artic Network receives funding from the Wellcome Trust through project 206298/Z/17/Z. PL acknowledges support by the Research Foundation -- Flanders (‘Fonds voor Wetenschappelijk Onderzoek -- Vlaanderen’, G066215N, G0D5117N and G0B9317N). TL is funded by The National Natural Science Foundation of China (NSFC) Excellent Young Scientists Fund (Hong Kong and Macau) (31922087).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">*</span> joint first authors</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.1" data-doi="10.1128/JVI.00593-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aiewsakun</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Katzourakis</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Time-Dependent Rate Phenomenon in Viruses</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">90</span>, <span class="cit-fpage">7184</span>–<span class="cit-lpage">7195</span>. <a href="https://doi.org/10.1128/JVI.00593-16">https://doi.org/10.1128/JVI.00593-16</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00593-16%26rft_id%253Dinfo%253Apmid%252F27252529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI5MC8xNi83MTg0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMzEvMjAyMC4wMy4zMC4wMTUwMDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.30.015008v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">K.G.</span></span>, <span class="cit-pub-date">2020</span>. <span class="cit-article-title">nCoV-2019 codon usage and reservoir (not snakes v2). Virological.org</span>. URL <a href="http://virological.org/t/339">http://virological.org/t/339</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.30.015008v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">K.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lipkin</span>,  <span class="cit-name-given-names">W.I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garry</span>,  <span class="cit-name-given-names">R.F.</span></span>, <span class="cit-pub-date">2020</span>. <span class="cit-chapter-title">The proximal origin of SARS-CoV-2</span>. <span class="cit-source">Nat Med</span>, <span class="cit-publ-name">published online</span>, <span class="cit-fpage">2</span>–<span class="cit-lpage">4</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.4" data-doi="10.1371/journal.pone.0010434"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boni</span>,  <span class="cit-name-given-names">M.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Jong</span>,  <span class="cit-name-given-names">M.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Doorn</span>,  <span class="cit-name-given-names">H.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E.C.</span></span>, <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Guidelines for identifying homologous recombination events in influenza A virus</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">e10434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBoni%26rft.auinit1%253DM.%2BF.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253De10434%26rft.epage%253De10434%26rft.atitle%253DGuidelines%2Bfor%2Bidentifying%2Bhomologous%2Brecombination%2Bevents%2Bin%2Binfluenza%2BA%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0010434%26rft_id%253Dinfo%253Apmid%252F20454662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0010434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20454662&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.5" data-doi="10.1534/genetics.106.068874"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boni</span>,  <span class="cit-name-given-names">M.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Posada</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feldman</span>,  <span class="cit-name-given-names">M.W.</span></span>, <span class="cit-pub-date">2007</span>. <span class="cit-article-title">An exact nonparametric method for inferring mosaic structure in sequence triplets</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>, <span class="cit-fpage">1035</span>–<span class="cit-lpage">47</span>. <a href="https://doi.org/10.1534/genetics.106.068874">https://doi.org/10.1534/genetics.106.068874</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.068874%26rft_id%253Dinfo%253Apmid%252F17409078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni8yLzEwMzUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8zMS8yMDIwLjAzLjMwLjAxNTAwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.6" data-doi="10.1128/JVI.02683-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boni</span>,  <span class="cit-name-given-names">M.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taubenberger</span>,  <span class="cit-name-given-names">J.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E.C.</span></span>, <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Homologous recombination is very rare or absent in human influenza A virus</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">82</span>, <span class="cit-fpage">4807</span>–<span class="cit-lpage">4811</span>. <a href="https://doi.org/10.1128/JVI.02683-07">https://doi.org/10.1128/JVI.02683-07</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02683-07%26rft_id%253Dinfo%253Apmid%252F18353939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Mi8xMC80ODA3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMzEvMjAyMC4wMy4zMC4wMTUwMDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.7" data-doi="10.1534/genetics.105.048975"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bruen</span>,  <span class="cit-name-given-names">T.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Philippe</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bryant</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-pub-date">2006</span>. <span class="cit-article-title">A simple and robust statistical test for detecting the presence of recombination</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>, <span class="cit-fpage">2665</span>–<span class="cit-lpage">2681</span>. <a href="https://doi.org/10.1534/genetics.105.048975">https://doi.org/10.1534/genetics.105.048975</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.048975%26rft_id%253Dinfo%253Apmid%252F16489234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi80LzI2NjUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8zMS8yMDIwLjAzLjMwLjAxNTAwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.8" data-doi="10.1093/molbev/msh018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bryant</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moulton</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-pub-date">2004</span>. <span class="cit-article-title">Neighbor-Net: An Agglomerative Method for the Construction of Phylogenetic Networks</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">21</span>, <span class="cit-fpage">255</span>–<span class="cit-lpage">265</span>. <a href="https://doi.org/10.1093/molbev/msh018">https://doi.org/10.1093/molbev/msh018</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsh018%26rft_id%253Dinfo%253Apmid%252F14660700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msh018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14660700&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220083300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chernomor</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Minh</span>,  <span class="cit-name-given-names">B.Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Forest</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klaere</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ingram</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Henzinger</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">von Haeseler</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Split diversity in constrained conservation prioritization using integer linear programming</span>. <abbr class="cit-jnl-abbrev">Methods Ecol Evol</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">83</span>–<span class="cit-lpage">91</span>. <a href="https://doi.org/10.1111/2041-210X.12299">https://doi.org/10.1111/2041-210X.12299</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol%2BEvol%26rft.volume%253D6%26rft.spage%253D83%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Duchene</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>,  <span class="cit-name-given-names">S.Y.W.</span></span>, <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Analyses of evolutionary dynamics in viruses are hindered by a time-dependent bias in rate estimates</span>. <abbr class="cit-jnl-abbrev">Proc R Soc L. B</abbr> <span class="cit-vol">281</span>, <span class="cit-fpage">20140732</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BR%2BSoc%2BL.%2BB%26rft.volume%253D281%26rft.spage%253D20140732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.11" data-doi="10.1101/810697"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Duchene</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lemey</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stadler</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>,  <span class="cit-name-given-names">S.Y.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duchene</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dhanasekaran</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baele</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Bayesian Evaluation of Temporal Signal in Measurably Evolving Populations</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/810697</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F810697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI4MTA2OTd2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzMxLzIwMjAuMDMuMzAuMDE1MDA4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dudas</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carvalho</span>,  <span class="cit-name-given-names">L.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bedford</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-pub-date">2018</span>. <span class="cit-article-title">MERS-CoV spillover at the camel-human interface</span>. <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">e31257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D7%26rft.spage%253De31257%26rft_id%253Dinfo%253Apmid%252F29336306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=29336306&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.13" data-doi="10.1128/JVI.03464-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eden</span>,  <span class="cit-name-given-names">J.-S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tanaka</span>,  <span class="cit-name-given-names">M.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boni</span>,  <span class="cit-name-given-names">M.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rawlinson</span>,  <span class="cit-name-given-names">W.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">P. A</span></span>, <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Recombination within the pandemic norovirus GII.4 lineage</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">87</span>, <span class="cit-fpage">6270</span>–<span class="cit-lpage">6282</span>. <a href="https://doi.org/10.1128/JVI.03464-12">https://doi.org/10.1128/JVI.03464-12</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03464-12%26rft_id%253Dinfo%253Apmid%252F23536665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ny8xMS82MjcwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMzEvMjAyMC4wMy4zMC4wMTUwMDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.14" data-doi="10.1016/j.tim.2016.09.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Forni</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cagliani</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clerici</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sironi</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Molecular Evolution of Human Coronavirus Genomes</span>. <abbr class="cit-jnl-abbrev">Trends Microbiol</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">35</span>–<span class="cit-lpage">48</span>. <a href="https://doi.org/10.1016/j.tim.2016.09.001">https://doi.org/10.1016/j.tim.2016.09.001</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BMicrobiol%26rft.volume%253D25%26rft.spage%253D35%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2016.09.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2016.09.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.15" data-doi="10.1038/nature12711"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ge</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chmura</span>,  <span class="cit-name-given-names">A.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Epstein</span>,  <span class="cit-name-given-names">J.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mazet</span>,  <span class="cit-name-given-names">J.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crameri</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daszak</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Z.-L.</span></span>, <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">503</span>, <span class="cit-fpage">535</span>–<span class="cit-lpage">538</span>. <a href="https://doi.org/10.1038/nature12711">https://doi.org/10.1038/nature12711</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D503%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12711%26rft_id%253Dinfo%253Apmid%252F24172901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24172901&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327464200045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.16" data-doi="org/10.1101/2020.02.07.937862"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>,  <span class="cit-name-given-names">A.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>,  <span class="cit-name-given-names">S.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Groot</span>,  <span class="cit-name-given-names">R.J. De</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gulyaeva</span>,  <span class="cit-name-given-names">A.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haagmans</span>,  <span class="cit-name-given-names">B.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lauber</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leontovich</span>,  <span class="cit-name-given-names">A.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neuman</span>,  <span class="cit-name-given-names">B.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Penzar</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perlman</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poon</span>,  <span class="cit-name-given-names">L.L.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Samborskiy</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sidorov</span>,  <span class="cit-name-given-names">I.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sola</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-pub-date">2020</span>. <span class="cit-article-title">Severe acute respiratory syndrome-related coronavirus: The species and its viruses – a statement of the Coronavirus Study Group</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> doi.<span class="cit-pub-id cit-pub-id-doi">org/10.1101/2020.02.07.937862</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252Forg%252F10.1101%252F2020.02.07.937862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=org/10.1101/2020.02.07.937862&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.17" data-doi="10.1128/JVI.01394-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">R.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R.S.</span></span>, <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Recombination, Reservoirs, and the Modular Spike: Mechanisms of Coronavirus Cross-Species Transmission</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">84</span>, <span class="cit-fpage">3134</span>–<span class="cit-lpage">3146</span>. <a href="https://doi.org/10.1128/jvi.01394-09">https://doi.org/10.1128/jvi.01394-09</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01394-09%26rft_id%253Dinfo%253Apmid%252F19906932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg0LzcvMzEzNCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzMxLzIwMjAuMDMuMzAuMDE1MDA4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.18" data-doi="10.1128/JVI.00631-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">He</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>,  <span class="cit-name-given-names">Yun</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhen</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>,  <span class="cit-name-given-names">Ye</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tu</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Identification of Diverse Alphacoronaviruses and Genomic Characterization of a Novel Severe Acute Respiratory Syndrome-Like Coronavirus from Bats in China</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">88</span>, <span class="cit-fpage">7070</span>–<span class="cit-lpage">7082</span>. <a href="https://doi.org/10.1128/JVI.00631-14">https://doi.org/10.1128/JVI.00631-14</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00631-14%26rft_id%253Dinfo%253Apmid%252F24719429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OC8xMi83MDcwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMzEvMjAyMC4wMy4zMC4wMTUwMDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.30.015008v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E.C.</span></span>, <span class="cit-pub-date">2009</span>. <span class="cit-chapter-title">The Evolution and Emergence of RNA Viruses</span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.20" data-doi="10.1038/nature19790"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dudas</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andersen</span>,  <span class="cit-name-given-names">K.G.</span></span>, <span class="cit-pub-date">2016</span>. <span class="cit-article-title">The evolution of Ebola virus: Insights from the 2013-2016 epidemic</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">538</span>, <span class="cit-fpage">193</span>–<span class="cit-lpage">200</span>. <a href="https://doi.org/10.1038/nature19790">https://doi.org/10.1038/nature19790</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D538%26rft.spage%253D193%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature19790%26rft_id%253Dinfo%253Apmid%252F27734858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature19790&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27734858&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.21" data-doi="10.1038/d41586-018-05373-w"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andersen</span>,  <span class="cit-name-given-names">K.G.</span></span>, <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Pandemics: spend on surveillance, not prediction</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">558</span>, <span class="cit-fpage">180</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D558%26rft.spage%253D180%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fd41586-018-05373-w%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/d41586-018-05373-w&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.22" data-doi="10.1128/JVI.01926-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hon</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drummond</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yip</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeng</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leung</span>,  <span class="cit-name-given-names">F.C.</span></span>, <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Evidence of the Recombinant Origin of a Bat Severe Acute Respiratory Syndrome (SARS)-Like Coronavirus and Its Implications on the Direct Ancestor of SARS Coronavirus</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">82</span>, <span class="cit-fpage">1819</span>–<span class="cit-lpage">1826</span>. <a href="https://doi.org/10.1128/JVI.01926-07">https://doi.org/10.1128/JVI.01926-07</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01926-07%26rft_id%253Dinfo%253Apmid%252F18057240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgyLzQvMTgxOSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzMxLzIwMjAuMDMuMzAuMDE1MDA4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.23" data-doi="10.1371/journal.ppat.1006698"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeng</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ge</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daszak</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cui</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Z.-L.</span></span>, <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">e1006698</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D13%26rft.spage%253De1006698%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006698%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ji</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zai</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-pub-date">2020</span>. <span class="cit-article-title">Cross-species transmission of the newly identified coronavirus 2019-nCoV</span>. <abbr class="cit-jnl-abbrev">J Med Virol</abbr> <span class="cit-vol">92</span>, <span class="cit-fpage">433</span>–<span class="cit-lpage">440</span>. <a href="https://doi.org/10.1002/jmv.25682">https://doi.org/10.1002/jmv.25682</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BMed%2BVirol%26rft.volume%253D92%26rft.spage%253D433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.30.015008v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Katoh</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Asimenos</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Toh</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-pub-date">2009</span>. <span class="cit-chapter-title">Multiple Alignment of DNA Sequences with MAFFT</span>, in: <span class="cit-publ-name">Press, H</span>. (Ed.), <span class="cit-source">Bioinformatics for DNA Sequence Analysis</span>. <span class="cit-publ-loc">Hertfordshire, UK</span>, pp. <span class="cit-fpage">39</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.26" data-doi="10.1093/molbev/msl051"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kosakovsky Pond</span>,  <span class="cit-name-given-names">S.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Posada</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gravenor</span>,  <span class="cit-name-given-names">M.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woelk</span>,  <span class="cit-name-given-names">C.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frost</span>,  <span class="cit-name-given-names">S.D.W.</span></span>, <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Automated Phylogenetic Detection of Recombination Using a Genetic Algorithm</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">1891</span>–<span class="cit-lpage">1901</span>. <a href="https://doi.org/10.1093/molbev/msl051">https://doi.org/10.1093/molbev/msl051</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl051%26rft_id%253Dinfo%253Apmid%252F16818476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msl051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16818476&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240586600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.27" data-doi="10.1093/molbev/msx263"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lam</span>,  <span class="cit-name-given-names">H.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ratmann</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boni</span>,  <span class="cit-name-given-names">M.F.</span></span>, <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Improved algorithmic complexity for the 3SEQ recombination detection algorithm</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">35</span>, <span class="cit-fpage">247</span>–<span class="cit-lpage">251</span>. <a href="https://doi.org/10.1093/molbev/msx263">https://doi.org/10.1093/molbev/msx263</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft.volume%253D35%26rft.spage%253D247%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx263&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.28" data-doi="org/10.1101/2020.02.13.945485"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lam</span>,  <span class="cit-name-given-names">T.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shum</span>,  <span class="cit-name-given-names">M.H.-H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">H.-C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tong</span>,  <span class="cit-name-given-names">Y.-G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ni</span>,  <span class="cit-name-given-names">X.-B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liao</span>,  <span class="cit-name-given-names">Y.-S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheung</span>,  <span class="cit-name-given-names">W.Y.</span></span>-M., <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.-J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">L.-F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leung</span>,  <span class="cit-name-given-names">G.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Y.-L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-pub-date">2020</span>. <span class="cit-article-title">Identification of 2019-nCoV related coronaviruses in Malayan pangolins in southern China</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> doi.<span class="cit-pub-id cit-pub-id-doi">org/10.1101/2020.02.13.945485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252Forg%252F10.1101%252F2020.02.13.945485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=org/10.1101/2020.02.13.945485&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.29" data-doi="10.1093/bioinformatics/bts580"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lemey</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Minin</span>,  <span class="cit-name-given-names">V.N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bielejec</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pond</span>,  <span class="cit-name-given-names">S.L.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-pub-date">2012</span>. <span class="cit-article-title">A counting renaissance: combining stochastic mapping and empirical Bayes to quickly detect amino acid sites under positive selection</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>, <span class="cit-fpage">3248</span>–<span class="cit-lpage">3256</span>. <a href="https://doi.org/10.1093/bioinformatics/bts580">https://doi.org/10.1093/bioinformatics/bts580</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts580%26rft_id%253Dinfo%253Apmid%252F23064000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts580&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23064000&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312105300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.30" data-doi="10.1056/NEJMoa2001316"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guan</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tong</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ren</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leung</span>,  <span class="cit-name-given-names">K.S.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>,  <span class="cit-name-given-names">E.H.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">J.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xing</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiang</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tu</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jin</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shao</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tao</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deng</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>,  <span class="cit-name-given-names">T.T.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">J.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">G.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cowling</span>,  <span class="cit-name-given-names">B.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leung</span>,  <span class="cit-name-given-names">G.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-pub-date">2020a</span>. <span class="cit-article-title">Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus–Infected Pneumonia</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>, published online<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa2001316</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> <a href="https://doi.org/10.1056/NEJMoa2001316">https://doi.org/10.1056/NEJMoa2001316</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001316%26rft_id%253Dinfo%253Apmid%252F31995857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001316&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31995857&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.31" data-doi="10.1101/2020.03.20.000885"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giorgi</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marichann</span>,  <span class="cit-name-given-names">M.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foley</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiao</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kong</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korber</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-pub-date">2020b</span>. <span class="cit-article-title">Emergence of SARS-CoV-2 through Recombination and Strong Purifying Selection</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/2020.03.20.000885</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.03.20.000885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMy4yMC4wMDA4ODV2MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzMxLzIwMjAuMDMuMzAuMDE1MDA4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lie</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">J.-P.</span></span>, <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Viral Metagenomics Revealed Sendai Virus and Coronavirus Infection of Malayan Pangolins</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">979</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D11%26rft.spage%253D979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hao</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guan</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Extensive diversity of coronaviruses in bats from China</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">507</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>. <a href="https://doi.org/10.1016/j.virol.2017.03.019">https://doi.org/10.1016/j.virol.2017.03.019</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D507%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.34" data-doi="10.1016/S0140-6736(20)302518"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niu</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bi</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhan</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meng</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">W.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">G.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-pub-date">2020</span>. <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">395</span>, <span class="cit-fpage">565</span>–<span class="cit-lpage">574</span>. <a href="https://doi.org/10.1016/S0140-6736(20)30251-8">https://doi.org/10.1016/S0140-6736(20)30251-8</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D565%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%2529302518%26rft_id%253Dinfo%253Apmid%252F32007145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)302518&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32007145&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.35" data-doi="10.1093/ve/vev003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">D.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murrell</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Golden</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khoosal</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muhire</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-pub-date">2015</span>. <span class="cit-article-title">RDP4: Detection and analysis of recombination patterns in virus genomes</span>. <abbr class="cit-jnl-abbrev">Virus Evol</abbr> <span class="cit-vol">1</span>:<span class="cit-fpage">vev003</span>. <a href="https://doi.org/10.1093/ve/vev003">https://doi.org/10.1093/ve/vev003</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fve%252Fvev003%26rft_id%253Dinfo%253Apmid%252F27774277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/ve/vev003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27774277&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Membrebe</span>,  <span class="cit-name-given-names">J.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baele</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lemey</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Bayesian Inference of Evolutionary Histories under Time-Dependent Substitution Rates</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">36</span>, <span class="cit-fpage">1793</span>–<span class="cit-lpage">1803</span>. <a href="https://doi.org/10.1093/molbev/msz094">https://doi.org/10.1093/molbev/msz094</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft.volume%253D36%26rft.spage%253D1793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.37" data-doi="10.1038/nm.3985"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Menachery</span>,  <span class="cit-name-given-names">V.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jr</span>,  <span class="cit-name-given-names">B.L.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Debbink</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Agnihothram</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gralinski</span>,  <span class="cit-name-given-names">L.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plante</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">R.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scobey</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ge</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donaldson</span>,  <span class="cit-name-given-names">E.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Randell</span>,  <span class="cit-name-given-names">S.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lanzavecchia</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marasco</span>,  <span class="cit-name-given-names">W.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R.S.</span></span>, <span class="cit-pub-date">2015</span>. <span class="cit-article-title">A SARS-like cluster of circulating bat coronaviruses shows potential for human emergence</span>. <abbr class="cit-jnl-abbrev">Nat Med</abbr> <span class="cit-vol">21</span>, <span class="cit-fpage">1508</span>–<span class="cit-lpage">1514</span>. <a href="https://doi.org/10.1038/nm.3985">https://doi.org/10.1038/nm.3985</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMed%26rft.volume%253D21%26rft.spage%253D1508%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm.3985%26rft_id%253Dinfo%253Apmid%252F26552008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm.3985&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26552008&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.38" data-doi="10.1073/pnas.1517719113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Menachery</span>,  <span class="cit-name-given-names">V.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yount</span>,  <span class="cit-name-given-names">B.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sims</span>,  <span class="cit-name-given-names">A.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Debbink</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Agnihothram</span>,  <span class="cit-name-given-names">S.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gralinski</span>,  <span class="cit-name-given-names">L.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pickles</span>,  <span class="cit-name-given-names">R.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corti</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Randell</span>,  <span class="cit-name-given-names">S.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lanzavecchia</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marasco</span>,  <span class="cit-name-given-names">W.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R.S.</span></span>, <span class="cit-pub-date">2016</span>. <span class="cit-article-title">SARS-like WIV1-CoV poised for human emergence</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">113</span>, <span class="cit-fpage">3048</span>–<span class="cit-lpage">3053</span>. <a href="https://doi.org/10.1073/pnas.1517719113">https://doi.org/10.1073/pnas.1517719113</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1517719113%26rft_id%253Dinfo%253Apmid%252F26976607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzExLzMwNDgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8zMS8yMDIwLjAzLjMwLjAxNTAwOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.39" data-doi="10.1093/molbev/msu300."><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmidt</span>,  <span class="cit-name-given-names">H.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haeseler</span>,  <span class="cit-name-given-names">A. Von</span></span>, <span class="cit-auth"><span class="cit-name-surname">Minh</span>,  <span class="cit-name-given-names">B.Q.</span></span>, <span class="cit-pub-date">2014</span>. <span class="cit-article-title">IQ-TREE: A Fast and Effective Stochastic Algorithm for Estimating Maximum-Likelihood Phylogenies</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">268</span>–<span class="cit-lpage">274</span>. <a href="https://doi.org/10.1093/molbev/msu300">https://doi.org/10.1093/molbev/msu300</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft.volume%253D32%26rft.spage%253D268%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu300.%26rft_id%253Dinfo%253Apmid%252F25371430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu300.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25371430&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.40" data-doi="10.1146/annurev.genet.36.040202.111115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Posada</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crandall</span>,  <span class="cit-name-given-names">K.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E.C.</span></span>, <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Recombination in evolutionary genomics</span>. <abbr class="cit-jnl-abbrev">Annu Rev Genet</abbr> <span class="cit-vol">36</span>, <span class="cit-fpage">75</span>–<span class="cit-lpage">97</span>. <a href="https://doi.org/10.1146/annurev.genet.36.040202.111115">https://doi.org/10.1146/annurev.genet.36.040202.111115</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BGenet%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.36.040202.111115%26rft_id%253Dinfo%253Apmid%252F12429687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.36.040202.111115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12429687&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180365100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.41" data-doi="10.1093/ve/vew007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rambaut</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>,  <span class="cit-name-given-names">T.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carvalho</span>,  <span class="cit-name-given-names">L.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pybus</span>,  <span class="cit-name-given-names">O.G.</span></span>, <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Exploring the temporal structure of heterochronous sequences using TempEst (formerly Path-O-Gen)</span>. <abbr class="cit-jnl-abbrev">Virus Evol</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">vew007</span>. <a href="https://doi.org/10.1093/ve/vew007">https://doi.org/10.1093/ve/vew007</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fve%252Fvew007%26rft_id%253Dinfo%253Apmid%252F27774300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/ve/vew007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27774300&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.30.015008v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robertson</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-pub-date">2020</span>. <span class="cit-article-title">nCoV’s relationship to bat coronaviruses – recombination signals (no snakes) - no evidence the 2019-nCoV lineage is recombinant. Virological.org</span>. URL <a href="http://virological.org/t/331">http://virological.org/t/331</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schierup</span>,  <span class="cit-name-given-names">M.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hein</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-pub-date">1999</span>. <span class="cit-article-title">Recombination and the Molecular Clock</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">1578</span>–<span class="cit-lpage">1579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft.volume%253D17%26rft.spage%253D1578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.44" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-pub-date">2014</span>. <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>. <a href="https://doi.org/10.1093/bioinformatics/btu033">https://doi.org/10.1093/bioinformatics/btu033</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451623&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.45" data-doi="10.1093/bioinformatics/btl446"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-pub-date">2006</span>. <span class="cit-article-title">RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">2688</span>–<span class="cit-lpage">90</span>. <a href="https://doi.org/10.1093/bioinformatics/btl446">https://doi.org/10.1093/bioinformatics/btl446</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl446%26rft_id%253Dinfo%253Apmid%252F16928733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16928733&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241629600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.46" data-doi="10.1086/425583"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stegeman</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bouma</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elbers</span>,  <span class="cit-name-given-names">A.R.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jong</span>,  <span class="cit-name-given-names">M.C.M. De</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nodelijk</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klerk</span>,  <span class="cit-name-given-names">F. De</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koch</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Boven</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-pub-date">2004</span>. <span class="cit-article-title">Avian Influenza A Virus (H7N7) Epidemic in The Netherlands in 2003: Course of the Epidemic and Effectiveness of Control Measures</span>. <abbr class="cit-jnl-abbrev">J Infect Dis</abbr> <span class="cit-vol">190</span>, <span class="cit-fpage">2088</span>–<span class="cit-lpage">2095</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BInfect%2BDis%26rft_id%253Dinfo%253Adoi%252F10.1086%252F425583%26rft_id%253Dinfo%253Apmid%252F15551206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/425583&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15551206&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225281600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.47" data-doi="10.1016/j.tim.2016.03.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Su</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lai</span>,  <span class="cit-name-given-names">A.C.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bi</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">G.F.</span></span>, <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Epidemiology, Genetic Recombination, and Pathogenesis of Coronaviruses</span>. <abbr class="cit-jnl-abbrev">Trends Microbiol</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">490</span>–<span class="cit-lpage">502</span>. <a href="https://doi.org/10.1016/j.tim.2016.03.003">https://doi.org/10.1016/j.tim.2016.03.003</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BMicrobiol%26rft.volume%253D24%26rft.spage%253D490%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2016.03.003%26rft_id%253Dinfo%253Apmid%252F27012512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2016.03.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27012512&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Suchard</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lemey</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baele</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ayres</span>,  <span class="cit-name-given-names">D.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drummond</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Bayesian phylogenetic and phylodynamic data integration using BEAST 1.10</span>. <abbr class="cit-jnl-abbrev">Virul Evol</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">vey016</span>. <a href="https://doi.org/10.1093/ve/vey016">https://doi.org/10.1093/ve/vey016</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirul%2BEvol%26rft.volume%253D4%26rft.spage%253Dvey016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.49" data-doi="10.1093/ve/vev016"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Trova</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baele</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vrancken</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bielejec</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fargette</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lemey</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Host ecology determines the dispersal patterns of a plant virus</span>. <abbr class="cit-jnl-abbrev">Virus Evol</abbr> <span class="cit-vol">1</span>:<span class="cit-fpage">vev016</span>. <a href="https://doi.org/10.1093/ve/vev016">https://doi.org/10.1093/ve/vev016</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fve%252Fvev016%26rft_id%253Dinfo%253Apmid%252F27774287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/ve/vev016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27774287&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-pub-date">2020</span>. <span class="cit-article-title">Receptor Recognition by the Novel Coronavirus from Wuhan: an Analysis Based on Decade-Long Structural Studies of SARS Coronavirus</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">94</span>, <span class="cit-fpage">e00127</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft.volume%253D94%26rft.spage%253De00127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fu</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cao</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nie</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Discovery and genetic analysis of novel coronaviruses in least horseshoe bats in southwestern China</span>. <abbr class="cit-jnl-abbrev">Emerg Microb Infect</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">e14</span>. <a href="https://doi.org/10.1038/emi.2016.140">https://doi.org/10.1038/emi.2016.140</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BMicrob%2BInfect%26rft.volume%253D6%26rft.spage%253D14e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.52" data-doi="10.1093/infdis/135.4.681"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Webster</span>,  <span class="cit-name-given-names">R.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bean</span>,  <span class="cit-name-given-names">W.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorman</span>,  <span class="cit-name-given-names">O.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chambers</span>,  <span class="cit-name-given-names">T.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kawaoka</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-pub-date">1992</span>. <span class="cit-article-title">Evolution and ecology of influenza A viruses</span>. <abbr class="cit-jnl-abbrev">Microbiol Rev</abbr> <span class="cit-vol">56</span>, <span class="cit-fpage">152</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DWebster%26rft.auinit1%253DR%2BG%26rft.volume%253D56%26rft.issue%253D1%26rft.spage%253D152%26rft.epage%253D179%26rft.atitle%253DEvolution%2Band%2Becology%2Bof%2Binfluenza%2BA%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Finfdis%252F135.4.681%26rft_id%253Dinfo%253Apmid%252F1579108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI1Ni8xLzE1MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzMxLzIwMjAuMDMuMzAuMDE1MDA4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">A.C.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>,  <span class="cit-name-given-names">S.K.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woo</span>,  <span class="cit-name-given-names">P.C.Y.</span></span>, <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Global Epidemiology of Bat Coronaviruses</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">174</span>. <a href="https://doi.org/10.3390/v11020174">https://doi.org/10.3390/v11020174</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D11%26rft.spage%253D174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.30.015008v1.54"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">World Health Organization</span>, <span class="cit-pub-date">2020a</span>. <span class="cit-chapter-title">Novel Coronavirus (2019-nCoV) Situation Report 1, 21 January 2020. Geneva, Switzerland. World Health Organization, 2020b</span>. <span class="cit-publ-name">Coronavirus disease 2019 (COVID-19) Situation Report – 51, 11 March 2020</span>. <span class="cit-publ-loc">Geneva, Switzerland</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.55" data-doi="org/10.1038/s41586-020-2008-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Z.-G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tao</span>,  <span class="cit-name-given-names">Z.-W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pei</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dai</span>,  <span class="cit-name-given-names">F.-H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Q.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">J.-J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.-Z.</span></span>, <span class="cit-pub-date">2020a</span>. <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. <abbr class="cit-jnl-abbrev">Nature published</abbr>, doi.<span class="cit-pub-id cit-pub-id-doi">org/10.1038/s41586-020-2008-3</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> <a href="https://doi.org/10.1038/s41586-020-2008-3">https://doi.org/10.1038/s41586-020-2008-3</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bpublished%26rft_id%253Dinfo%253Adoi%252Forg%252F10.1038%252Fs41586-020-2008-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=org/10.1038/s41586-020-2008-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jin</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qiu</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuen</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-pub-date">2020b</span>. <span class="cit-article-title">SARS-CoV-2 is an appropriate name for the new coronavirus A distinct name is needed for the new coronavirus</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">395</span>, <span class="cit-fpage">949</span>–<span class="cit-lpage">950</span>. <a href="https://doi.org/10.1016/S0140-6736(20)30557-2">https://doi.org/10.1016/S0140-6736(20)30557-2</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.57" data-doi="10.1101/2020.02.17.951335v1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xiao</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhai</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zou</span>,  <span class="cit-name-given-names">J.-J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shu</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">R.-A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y.-J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">S.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">W.-J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cai</span>,  <span class="cit-name-given-names">Q.-H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hou</span>,  <span class="cit-name-given-names">F.-H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiao</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-pub-date">2020</span>. <span class="cit-article-title">Isolation and Characterization of 2019-nCoV-like Coronavirus from Malayan Pangolins</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/2020.02.17.951335v1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.02.17.951335v1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/2020.02.17.951335v1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yres</span>,  <span class="cit-name-given-names">D.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cummings</span>,  <span class="cit-name-given-names">M.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baele</span>,  <span class="cit-name-given-names">G.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Darling</span>,  <span class="cit-name-given-names">A.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lewis</span>,  <span class="cit-name-given-names">P.O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Swofford</span>,  <span class="cit-name-given-names">D.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huelsenbeck</span>,  <span class="cit-name-given-names">J.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lemey</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-pub-date">2019</span>. <span class="cit-article-title">BEAGLE 3: Improved Performance, Scaling, and Usability for a High-Performance Computing Library for Statistical Phylogenetics</span>. <abbr class="cit-jnl-abbrev">Softw. Syst Evol</abbr> <span class="cit-vol">68</span>, <span class="cit-fpage">1052</span>–<span class="cit-lpage">1061</span>. <a href="https://doi.org/10.1093/sysbio/syz020">https://doi.org/10.1093/sysbio/syz020</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoftw.%2BSyst%2BEvol%26rft.volume%253D68%26rft.spage%253D1052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">J.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cowling</span>,  <span class="cit-name-given-names">B.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liao</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fang</span>,  <span class="cit-name-given-names">V.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>,  <span class="cit-name-given-names">E.H.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ni</span>,  <span class="cit-name-given-names">M.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leung</span>,  <span class="cit-name-given-names">G.M.</span></span>, <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Effect of closure of live poultry markets on poultry-to-person transmission of avian influenza A H7N9 virus: an ecological study</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">383</span>, <span class="cit-fpage">541</span>–<span class="cit-lpage">548</span>. <a href="https://doi.org/10.1016/S0140-6736(13)61904-2">https://doi.org/10.1016/S0140-6736(13)61904-2</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D383%26rft.spage%253D541%26rft_id%253Dinfo%253Apmid%252F24183056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24183056&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331136400030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.60" data-doi="10.1099/vir.0.016378-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hon</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poon</span>,  <span class="cit-name-given-names">L.L.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>,  <span class="cit-name-given-names">T.T.</span></span>-Y., <span class="cit-auth"><span class="cit-name-surname">Leung</span>,  <span class="cit-name-given-names">F.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Communication Intraspecies diversity of SARS-like coronaviruses in Rhinolophus sinicus and its implications for the origin of SARS coronaviruses in humans</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr> <span class="cit-vol">91</span>, <span class="cit-fpage">1058</span>–<span class="cit-lpage">1062</span>. <a href="https://doi.org/10.1099/vir.0.016378-0">https://doi.org/10.1099/vir.0.016378-0</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.016378-0%26rft_id%253Dinfo%253Apmid%252F20016037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.016378-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20016037&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015008.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276714300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.61" data-doi="org/10.1016/j.cell.2020.03.035"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.-Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E.C.</span></span>, <span class="cit-pub-date">2020</span>. <span class="cit-article-title">Commentary A Genomic Perspective on the Origin and Emergence of SARS-CoV-2</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">181</span>, doi.<span class="cit-pub-id cit-pub-id-doi">org/10.1016/j.cell.2020.03.035</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span> <a href="https://doi.org/10.1016/j.cell.2020.03.035">https://doi.org/10.1016/j.cell.2020.03.035</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft_id%253Dinfo%253Adoi%252Forg%252F10.1016%252Fj.cell.2020.03.035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=org/10.1016/j.cell.2020.03.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.62" data-doi="10.1101/2020.03.02.974139"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hughes</span>,  <span class="cit-name-given-names">A.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bi</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-pub-date">2020a</span>. <span class="cit-article-title">A novel bat coronavirus reveals natural insertions at the S1/S2 cleavage site of the Spike protein and a possible recombinant origin of HCoV-19</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/2020.03.02.974139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.03.02.974139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMy4wMi45NzQxMzl2MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzMxLzIwMjAuMDMuMzAuMDE1MDA4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015008v1.63" data-doi="org/10.1038/s41586-020-2012-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">X.-L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.-G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Si</span>,  <span class="cit-name-given-names">H.-R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.-L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">H.-D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">R.-D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">M.-Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">X.-R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">X.-S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Q.-J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deng</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">L.-L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhan</span>,  <span class="cit-name-given-names">F.-X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.-Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geng-Fu</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Z.-L.</span></span>, <span class="cit-pub-date">2020b</span>. <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, published online, doi.<span class="cit-pub-id cit-pub-id-doi">org/10.1038/s41586-020-2012-7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252Forg%252F10.1038%252Fs41586-020-2012-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=org/10.1038/s41586-020-2012-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.30.015008v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.30.015008v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.30.016766v1" title="Minimal structural elements required for midline repulsive signaling and regulation of Drosophila Robo1" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.29.008631v1" title="Azithromycin and ciprofloxacin have a chloroquine-like effect on respiratory epithelial cells" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 31, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.30.015008v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.30.015008v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.30.015008v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.30.015008v1" class="active" data-icon-position="" data-hide-link-title="0">Evolutionary origins of the SARS-CoV-2 sarbecovirus lineage responsible for the COVID-19 pandemic</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1218990" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-MHYTdIuCHh8NaJRzOOaZc76BatDasX-X4ptRa4q3zDY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1218990" id="node1218990" data-pisa="biorxiv;2020.03.30.015008v1" data-pisa-master="biorxiv;2020.03.30.015008" data-seqnum="1218990" data-apath="/biorxiv/early/2020/03/31/2020.03.30.015008.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Evolutionary origins of the SARS-CoV-2 sarbecovirus lineage responsible for the COVID-19 pandemic</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Maciej F</span> <span class="nlm-surname">Boni</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Philippe</span> <span class="nlm-surname">Lemey</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Xiaowei</span> <span class="nlm-surname">Jiang</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Tommy Tsan-Yuk</span> <span class="nlm-surname">Lam</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Blair</span> <span class="nlm-surname">Perry</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Todd</span> <span class="nlm-surname">Castoe</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Andrew</span> <span class="nlm-surname">Rambaut</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">David L</span> <span class="nlm-surname">Robertson</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.30.015008; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.30.015008 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.30.015008v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015008v1&amp;title=Evolutionary%20origins%20of%20the%20SARS-CoV-2%20sarbecovirus%20lineage%20responsible%20for%20the%20COVID-19%20pandemic" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015008v1&amp;title=Evolutionary%20origins%20of%20the%20SARS-CoV-2%20sarbecovirus%20lineage%20responsible%20for%20the%20COVID-19%20pandemic" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015008v1&amp;text=Evolutionary%20origins%20of%20the%20SARS-CoV-2%20sarbecovirus%20lineage%20responsible%20for%20the%20COVID-19%20pandemic" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015008v1&amp;title=Evolutionary%20origins%20of%20the%20SARS-CoV-2%20sarbecovirus%20lineage%20responsible%20for%20the%20COVID-19%20pandemic" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015008v1&amp;t=Evolutionary%20origins%20of%20the%20SARS-CoV-2%20sarbecovirus%20lineage%20responsible%20for%20the%20COVID-19%20pandemic" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015008v1&amp;title=Evolutionary%20origins%20of%20the%20SARS-CoV-2%20sarbecovirus%20lineage%20responsible%20for%20the%20COVID-19%20pandemic" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015008v1&amp;title=Evolutionary%20origins%20of%20the%20SARS-CoV-2%20sarbecovirus%20lineage%20responsible%20for%20the%20COVID-19%20pandemic" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1218990" id="node1218990--2" data-pisa="biorxiv;2020.03.30.015008v1" data-pisa-master="biorxiv;2020.03.30.015008" data-seqnum="1218990" data-apath="/biorxiv/early/2020/03/31/2020.03.30.015008.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Evolutionary origins of the SARS-CoV-2 sarbecovirus lineage responsible for the COVID-19 pandemic</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Maciej F</span> <span class="nlm-surname">Boni</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Philippe</span> <span class="nlm-surname">Lemey</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Xiaowei</span> <span class="nlm-surname">Jiang</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Tommy Tsan-Yuk</span> <span class="nlm-surname">Lam</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Blair</span> <span class="nlm-surname">Perry</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Todd</span> <span class="nlm-surname">Castoe</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Andrew</span> <span class="nlm-surname">Rambaut</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">David L</span> <span class="nlm-surname">Robertson</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.30.015008; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.30.015008 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Evolutionary%20origins%20of%20the%20SARS-CoV-2%20sarbecovirus%20lineage%20responsible%20for%20the%20COVID-19%20pandemic&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Boni%2C+Maciej+F&amp;rft.au=Lemey%2C+Philippe&amp;rft.au=Jiang%2C+Xiaowei&amp;rft.au=Lam%2C+Tommy+Tsan-Yuk&amp;rft.au=Perry%2C+Blair&amp;rft.au=Castoe%2C+Todd&amp;rft.au=Rambaut%2C+Andrew&amp;rft.au=Robertson%2C+David+L"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1218990/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1218990/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1218990/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1218990/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1218990/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1218990/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1218990/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1218990/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1218990/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1218990/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1218990/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1218990/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015008v1&amp;count=horizontal&amp;via=&amp;text=Evolutionary%20origins%20of%20the%20SARS-CoV-2%20sarbecovirus%20lineage%20responsible%20for%20the%20COVID-19%20pandemic&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015008v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015008v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.30.015008v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
