<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/02/2020.04.02.021576.atom" />
<meta name="HW.pisa" content="biorxiv;2020.04.02.021576v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Prospects for detecting early warning signals in discrete event sequence data: application to epidemiological incidence data" />
<meta name="DC.Identifier" content="10.1101/2020.04.02.021576" />
<meta name="DC.Date" content="2020-04-02" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Early warning signals (EWS) identify systems approaching a critical transition, where the system undergoes a sudden change in state. For example, monitoring changes in variance or autocorrelation offers a computationally inexpensive method which can be used in real-time to assess when an infectious disease transitions to elimination. EWS have a promising potential to not only be used to monitor infectious diseases, but also to inform control policies to aid disease elimination. Previously, potential EWS have been identified for prevalence data, however the prevalence of a disease is often not known directly. In this work we identify EWS for incidence data, the standard data type collected by the Centers for Disease Control and Prevention (CDC) or World Health Organization (WHO). We show, through several examples, that EWS calculated on simulated incidence time series data exhibit vastly different behaviours to those previously studied on prevalence data. In particular, the variance displays a decreasing trend on the approach to disease elimination, contrary to that expected from critical slowing down theory; this could lead to unreliable indicators of elimination when calculated on real-world data. We derive analytical predictions which can be generalised for many epidemiological systems, and we support our theory with simulated studies of disease incidence. Additionally, we explore EWS calculated on the rate of incidence over time, a property which can be extracted directly from incidence data. We find that although incidence might not exhibit typical critical slowing down properties before a critical transition, the rate of incidence does, presenting a promising new data type for the application of statistical indicators.

Author summary The threat posed by infectious diseases has a huge impact on our global society. It is therefore critical to monitor infectious diseases as new data become available during control campaigns. One obstacle in observing disease emergence or elimination is understanding what influences noise in the data and how this fluctuates when near to zero cases. The standard data type collected is the number of new cases per day/month/year but mathematical modellers often focus on data such as the total number of infectious people, due to its analytical properties. We have developed a methodology to monitor the standard type of data to inform whether a disease is approaching emergence or disease elimination. We have shown computationally how fluctuations change as disease data get closer towards a tipping point and our insights highlight how these observed changes can be strikingly different when calculated on different types of data." />
<meta name="DC.Contributor" content="Emma Southall" />
<meta name="DC.Contributor" content="Michael J. Tildesley" />
<meta name="DC.Contributor" content="Louise Dyson" />
<meta name="article:published_time" content="2020-04-02" />
<meta name="article:section" content="Contradictory Results" />
<meta name="citation_title" content="Prospects for detecting early warning signals in discrete event sequence data: application to epidemiological incidence data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Early warning signals (EWS) identify systems approaching a critical transition, where the system undergoes a sudden change in state. For example, monitoring changes in variance or autocorrelation offers a computationally inexpensive method which can be used in real-time to assess when an infectious disease transitions to elimination.

EWS have a promising potential to not only be used to monitor infectious diseases, but also to inform control policies to aid disease elimination. Previously, potential EWS have been identified for prevalence data, however the prevalence of a disease is often not known directly. In this work we identify EWS for incidence data, the standard data type collected by the Centers for Disease Control and Prevention (CDC) or World Health Organization (WHO). We show, through several examples, that EWS calculated on simulated incidence time series data exhibit vastly different behaviours to those previously studied on prevalence data. In particular, the variance displays a decreasing trend on the approach to disease elimination, contrary to that expected from critical slowing down theory; this could lead to unreliable indicators of elimination when calculated on real-world data. We derive analytical predictions which can be generalised for many epidemiological systems, and we support our theory with simulated studies of disease incidence. Additionally, we explore EWS calculated on the rate of incidence over time, a property which can be extracted directly from incidence data. We find that although incidence might not exhibit typical critical slowing down properties before a critical transition, the rate of incidence does, presenting a promising new data type for the application of statistical indicators.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.04.02.021576v1" />
<meta name="citation_id" content="2020.04.02.021576v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.02.021576v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.02.021576v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.02.021576v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.04.02.021576.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.04.02.021576" />
<meta name="citation_num_pages" content="24" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="Contradictory Results" />
<meta name="citation_firstpage" content="2020.04.02.021576" />
<meta name="citation_author" content="Emma Southall" />
<meta name="citation_author_institution" content="EPSRC &amp; MRC Centre for Doctoral Training in Mathematics for Real-World Systems, University of Warwick" />
<meta name="citation_author_email" content="e.r.southall@warwick.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7319-3080" />
<meta name="citation_author" content="Michael J. Tildesley" />
<meta name="citation_author_institution" content="Zeeman Institute for Systems Biology and Infectious Disease Epidemiology Research, Mathematics Institute and School of Life Sciences, University of Warwick" />
<meta name="citation_author" content="Louise Dyson" />
<meta name="citation_author_institution" content="Zeeman Institute for Systems Biology and Infectious Disease Epidemiology Research, Mathematics Institute and School of Life Sciences, University of Warwick" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9788-4858" />
<meta name="citation_reference" content="Dakos V, Scheffer M, van Nes EH, Brovkin V, Petoukhov V, Held H. Slowing down as an early warning signal for abrupt climate change. Proceedings of the National Academy of Sciences. 2008 Sep 23;105(38):14308–12." />
<meta name="citation_reference" content="Scheffer M, Bascompte J, Brock WA, Brovkin V, Carpenter SR, Dakos V, Held H, Van Nes EH, Rietkerk M, Sugihara G. Early-warning signals for critical transitions. Nature. 2009 Sep;461(7260):53." />
<meta name="citation_reference" content="Brett TS, O’Dea EB, Marty É, Miller PB, Park AW, Drake JM, Rohani P. Anticipating epidemic transitions with imperfect data. PLoS computational biology. 2018 Jun 8;14(6):e1006204." />
<meta name="citation_reference" content="O’Dea EB, Drake JM. Disentangling reporting and disease transmission. Theoretical Ecology. 2019 Mar 1;12(1):89–98." />
<meta name="citation_reference" content="O’Regan SM, Drake JM. Theory of early warning signals of disease emergenceand leading indicators of elimination. Theoretical Ecology. 2013 Aug 1;6(3):333–57." />
<meta name="citation_reference" content="Miller PB, O’Dea EB, Rohani P, Drake JM. Forecasting infectious disease emergence subject to seasonal forcing. Theoretical Biology and Medical Modelling. 2017 Dec;14(1):17." />
<meta name="citation_reference" content="O’Regan SM, Lillie JW, Drake JM. Leading indicators of mosquito-borne disease elimination. Theoretical ecology. 2016 Sep 1;9(3):269–86." />
<meta name="citation_reference" content="Dessavre AG, Southall E, Tildesley MJ, Dyson L. The problem of detrending when analysing potential indicators of disease elimination. Journal of theoretical biology. 2019 Apr 11." />
<meta name="citation_reference" content="Carpenter SR, Brock WA. Rising variance: a leading indicator of ecological transition. Ecology letters. 2006 Mar;9(3):311–8." />
<meta name="citation_reference" content="Carpenter SR, Cole JJ, Pace ML, Batt R, Brock WA, Cline T, Coloso J, Hodgson JR, Kitchell JF, Seekell DA, Smith L. Early warnings of regime shifts: a whole-ecosystem experiment. Science. 2011 May 27;332(6033):1079–82." />
<meta name="citation_reference" content="Lenton TM, Livina VN, Dakos V, Van Nes EH, Scheffer M. Early warning of climate tipping points from critical slowing down: comparing methods to improve robustness. Philosophical Transactions of the Royal Society A: Mathematical, Physical and Engineering Sciences. 2012 Mar 13;370(1962):1185–204." />
<meta name="citation_reference" content="Clements CF, Ozgul A. Including trait-based early warning signals helps predict population collapse. Nature communications. 2016 Mar 24;7:10984." />
<meta name="citation_reference" content="Drake JM, Griffen BD. Early warning signals of extinction in deteriorating environments. Nature. 2010 Sep;467(7314):456." />
<meta name="citation_reference" content="Hefley TJ, Tyre AJ, Blankenship EE. Statistical indicators and state–space population models predict extinction in a population of bobwhite quail. Theoretical Ecology. 2013 Aug 1;6(3):319–31." />
<meta name="citation_reference" content="Dakos V, Carpenter SR, Brock WA, Ellison AM, Guttal V, Ives AR, Kefi S, Livina V, Seekell DA, van Nes EH, Scheffer M. Methods for detecting early warnings of critical transitions in time series illustrated using simulated ecological data. PloS one. 2012 Jul 17;7(7):e41010." />
<meta name="citation_reference" content="Dakos V, Van Nes EH, D’Odorico P, Scheffer M. Robustness of variance and autocorrelation as indicators of critical slowing down. Ecology. 2012 Feb;93(2):264–71." />
<meta name="citation_reference" content="Chen S, O’Dea EB, Drake JM, Epureanu BI. Eigenvalues of the covariance matrix as early warning signals for critical transitions in ecological systems. Scientific reports. 2019 Feb 22;9(1):2572." />
<meta name="citation_reference" content="Kefi S, Guttal V, Brock WA, Carpenter SR, Ellison AM, Livina VN, Seekell DA, Scheffer M, van Nes EH, Dakos V. Early warning signals of ecological transitions: methods for spatial patterns. PloS one. 2014 Mar 21;9(3):e92097." />
<meta name="citation_reference" content="Scheffer M, Carpenter SR. Catastrophic regime shifts in ecosystems: linking theory to observation. Trends in ecology &amp; evolution. 2003 Dec 1;18(12):648–56." />
<meta name="citation_reference" content="Clements CF, Drake JM, Griffiths JI, Ozgul A. Factors influencing the detectability of early warning signals of population collapse. The American Naturalist. 2015 May 7;186(1):50–8." />
<meta name="citation_reference" content="Wen, H., Ciamarra, M. P., &amp; Cheong, S. A. How one might miss early warning signals of critical transitions in time series data: A systematic study of two major currency pairs. PloS one. 2018 Mar 14;13(3):e0191439." />
<meta name="citation_reference" content="Diks, C., Hommes, C. and Wang, J. Critical slowing down as an early warning signal for financial crises?. Empirical Economics. 2019 Oct; 57(4), pp.1201–1228." />
<meta name="citation_reference" content="Carpenter, S.R., Cole, J.J., Pace, M.L., Batt, R., Brock, W.A., Cline, T., Coloso, J., Hodgson, J.R., Kitchell, J.F., Seekell, D.A. and Smith, L. Early warnings of regime shifts: a whole-ecosystem experiment. Science. 2011; 332(6033), pp.1079–1082." />
<meta name="citation_reference" content="Keeling, M.J. and Rohani, P. M Modeling infectious diseases in humans and animals. Princeton University Press. 2011 Sep." />
<meta name="citation_reference" content="Kendall, M. A New Measure of Rank Correlation. Oxford University Press. 1938 Jun." />
<meta name="citation_reference" content="Nájera, J.A., González-Silva, M. and Alonso, P.L. Some lessons for the future from the Global Malaria Eradication Programme (1955–1969). PLoS medicine, 8(1). 2011." />
<meta name="citation_reference" content="Cohen, J.M., Smith, D.L., Cotter, C., Ward, A., Yamey, G., Sabot, O.J. and Moonen, B. Malaria resurgence: a systematic review and assessment of its causes. Malaria journal, 11(1), p.122. 2012." />
<meta name="citation_reference" content="Van Kampen NG. Stochastic processes in physics and chemistry. Elsevier; 1992 Nov 20." />
<meta name="citation_reference" content="Guttal V, Raghavendra S, Goel N, Hoarau Q. Lack of critical slowing down suggests that financial meltdowns are not critical transitions, yet rising variability could signal systemic risk. PloS one, 11(1). 2016." />
<meta name="citation_reference" content="Kéfi S, Dakos V, Scheffer M, Van Nes EH, Rietkerk M. Early warning signals also precede non-catastrophic transitions. Oikos. 122(5):641–8. 2013 May." />
<meta name="citation_reference" content="Holloway B, Goodman R. Recommendations of the international task force for disease eradication. Morbidity and mortality weekly report. 42(16):1–38. 1993." />
<meta name="twitter:title" content="Prospects for detecting early warning signals in discrete event sequence data: application to epidemiological incidence data" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Early warning signals (EWS) identify systems approaching a critical transition, where the system undergoes a sudden change in state. For example, monitoring changes in variance or autocorrelation offers a computationally inexpensive method which can be used in real-time to assess when an infectious disease transitions to elimination. EWS have a promising potential to not only be used to monitor infectious diseases, but also to inform control policies to aid disease elimination. Previously, potential EWS have been identified for prevalence data, however the prevalence of a disease is often not known directly. In this work we identify EWS for incidence data, the standard data type collected by the Centers for Disease Control and Prevention (CDC) or World Health Organization (WHO). We show, through several examples, that EWS calculated on simulated incidence time series data exhibit vastly different behaviours to those previously studied on prevalence data. In particular, the variance displays a decreasing trend on the approach to disease elimination, contrary to that expected from critical slowing down theory; this could lead to unreliable indicators of elimination when calculated on real-world data. We derive analytical predictions which can be generalised for many epidemiological systems, and we support our theory with simulated studies of disease incidence. Additionally, we explore EWS calculated on the rate of incidence over time, a property which can be extracted directly from incidence data. We find that although incidence might not exhibit typical critical slowing down properties before a critical transition, the rate of incidence does, presenting a promising new data type for the application of statistical indicators.

Author summary The threat posed by infectious diseases has a huge impact on our global society. It is therefore critical to monitor infectious diseases as new data become available during control campaigns. One obstacle in observing disease emergence or elimination is understanding what influences noise in the data and how this fluctuates when near to zero cases. The standard data type collected is the number of new cases per day/month/year but mathematical modellers often focus on data such as the total number of infectious people, due to its analytical properties. We have developed a methodology to monitor the standard type of data to inform whether a disease is approaching emergence or disease elimination. We have shown computationally how fluctuations change as disease data get closer towards a tipping point and our insights highlight how these observed changes can be strikingly different when calculated on different types of data." />
<meta name="og-title" property="og:title" content="Prospects for detecting early warning signals in discrete event sequence data: application to epidemiological incidence data" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.02.021576v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Early warning signals (EWS) identify systems approaching a critical transition, where the system undergoes a sudden change in state. For example, monitoring changes in variance or autocorrelation offers a computationally inexpensive method which can be used in real-time to assess when an infectious disease transitions to elimination. EWS have a promising potential to not only be used to monitor infectious diseases, but also to inform control policies to aid disease elimination. Previously, potential EWS have been identified for prevalence data, however the prevalence of a disease is often not known directly. In this work we identify EWS for incidence data, the standard data type collected by the Centers for Disease Control and Prevention (CDC) or World Health Organization (WHO). We show, through several examples, that EWS calculated on simulated incidence time series data exhibit vastly different behaviours to those previously studied on prevalence data. In particular, the variance displays a decreasing trend on the approach to disease elimination, contrary to that expected from critical slowing down theory; this could lead to unreliable indicators of elimination when calculated on real-world data. We derive analytical predictions which can be generalised for many epidemiological systems, and we support our theory with simulated studies of disease incidence. Additionally, we explore EWS calculated on the rate of incidence over time, a property which can be extracted directly from incidence data. We find that although incidence might not exhibit typical critical slowing down properties before a critical transition, the rate of incidence does, presenting a promising new data type for the application of statistical indicators.

Author summary The threat posed by infectious diseases has a huge impact on our global society. It is therefore critical to monitor infectious diseases as new data become available during control campaigns. One obstacle in observing disease emergence or elimination is understanding what influences noise in the data and how this fluctuates when near to zero cases. The standard data type collected is the number of new cases per day/month/year but mathematical modellers often focus on data such as the total number of infectious people, due to its analytical properties. We have developed a methodology to monitor the standard type of data to inform whether a disease is approaching emergence or disease elimination. We have shown computationally how fluctuations change as disease data get closer towards a tipping point and our insights highlight how these observed changes can be strikingly different when calculated on different types of data." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-02" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.04.02.021576v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.04.02.021576v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.04.02.021576v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.04.02.021576v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1224277" />
    <title>Prospects for detecting early warning signals in discrete event sequence data: application to epidemiological incidence data | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"2pQhqT1bGHasXwkRk_lk54PxnHcKAU5qW9SPGgFotCQ","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1224277","apath":"\/biorxiv\/early\/2020\/04\/02\/2020.04.02.021576.atom","pisa":"biorxiv;2020.04.02.021576v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.04.02.021576v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.04.02.021576v1","title":"Prospects for detecting early warning signals in discrete event sequence data: application to epidemiological incidence data","identifier":"node\/1224277"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PSm0ErM4mfVtJWxRDYJeNN2uPIsy8Uh2VO3D0KG4+8LskSKwVKQCmTHKcB+v68Mtywdzry1Fg3DW0UCOh8fYwZ6uz0BWX4Uta8EdOx8RcYKtBRFVjDmvvEoYGMw\\\\\\\/7sesyNvpl8+9ds627NsNAVr+9M5fV\\\\\\\/zR3cDgnvW6jD6JVRb+Gv6j0EYO4WH93472PvhvpcwXSj7yy9qPYEDjssam68mWeClzRnHNxhHR0FC7vtxaIdXIa\\\\\\\/gudi7cZAqzETNXFAfeQW\\\\\\\/sKGC+wIaOaPKt443\\\\\\\/QuujgXUuLIe0+w+flp69LntLzFT4Pa9zwh4BLV6XGB5lx7BrVJYMWljJfxs\\\\\\\/GIquhux7GmdQI\\\\\\\/QVkiE0wRixLF2q2ueEwhMcXcg8L3TA\\\\\\\/RgebHNL\\\\\\\/jYpcVDVURq9t1vmPwGQ1fp1F0G8Z+\\\\\\\/eYUYmwI5S+XrWT3cIakkzzyGAxOp7MH6ek4R0XsaiVvnwb43\\\\\\\/4yb3\\\\\\\/s98bpGxfqChom5hVeHKmPcC3vZGPyjrc4Qk1mqg0hMjcd0MiT8m+ei5zspGwW4+NIAaZ5M7Dct6JA3y6MKe99xCosGf6\\\\\\\/CrM+kZyHC097VSNVFR15DkjaxnJsHzfVRhjzhBsyyw+WJFcIQQe6XYcU3KPlKbNMT1P\\\\\\\/VgzZ2+sj1s\\\\\\\/dTxNUJr1WyMaNQkMXvstoSU\\\\\\\/JZdrPcwDk1UV6yne57x3I147QoxOhNV7BNQBQvJwj4lnILRO9ccL5w2qQ4lX5v\\\\\\\/CjXzAMTx4JdFbU2G2+mV\\\\\\\/mQ9fse\\\\\\\/4RJ3Rgsdbtaj71hrda0b3znK4Cv\\\\\\\/PRXqdNRTviNeaPXNgpYSEKtH0kWXHqECAaX453NJafVuRmYFK9oNk0plx4ymBwWC3jxg2pOiWA6a0rLXMzBz3cM4razOJTa\\\\\\\/T9KKSsXbU9KuR88faI9wCvtgsVRydXGNQul5eP+kI\\\\\\\/HTQN6ECRmy6yhFguL9JgO99UwWkkn9JH4j9XVqdEFroaPV28W0Zg+PgBlKEJBJdYykCSby51Q9WKKX21B7GqgkKMILiTFpHeB\\\\\\\/HKLAcXl3zwrLr1pFVudr0Jw0p8CTgwPaHc6kxlwGnRaXpwNjdWJSn6STo2\\\\\\\/0ub9V5L6pWcbxp8BwUkHe+POXdUPLZGD2r6wNypTlnSksFxFRIQPfp9Z6\\\\\\\/JxdRMeQDcH11nit+Oy+yvi3xlavRYlHZE3NIDYs34qbV8SvOZLOq0fQt99CZjpQxNWaZbJ2bs+cJn3B1nFKPmqKuLynsPYb9MuzAnIFKWumWpw+KLCFzFZ3upqWmDGYrF+unQmV+u8Se\\\\\\\/d2vNXUwBmPjf6Pif2Rw584cH6mQXVs2Yiqal0JP7VZby6Ycmi68ytariYN\\\u0022,\\\u0022iv\\\u0022:\\\u0022L5XJGmALzWfvB0shwOmUPQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00225d76195029de76ff0378bf181b82bebc0223ad4d901fe15aab8a870f9af08b9c\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.04.02.021576v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1224277 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-contradictory-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.04.02.021576v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-84oyTUImSirQFuW8D9qtQgIs40ARID4PfkukNz60qzQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1224277" id="node1224277" data-pisa="biorxiv;2020.04.02.021576v1" data-pisa-master="biorxiv;2020.04.02.021576" data-apath="/biorxiv/early/2020/04/02/2020.04.02.021576.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        Contradictory Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Prospects for detecting early warning signals in discrete event sequence data: application to epidemiological incidence data</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7319-3080" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Emma</span> <span class="nlm-surname">Southall</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Michael J.</span> <span class="nlm-surname">Tildesley</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9788-4858" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Louise</span> <span class="nlm-surname">Dyson</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.04.02.021576 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1224277" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Emma Southall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>EPSRC &#38; MRC Centre for Doctoral Training in Mathematics for Real-World Systems, University of Warwick</span>, Coventry, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emma%2BSouthall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Southall%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEmma%2BSouthall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7319-3080" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Emma Southall</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:e.r.southall@warwick.ac.uk" class="" data-icon-position="" data-hide-link-title="0">e.r.southall@warwick.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael J. Tildesley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Zeeman Institute for Systems Biology and Infectious Disease Epidemiology Research, Mathematics Institute and School of Life Sciences, University of Warwick</span>, Coventry, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ.%2BTildesley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tildesley%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BJ.%2BTildesley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Louise Dyson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Zeeman Institute for Systems Biology and Infectious Disease Epidemiology Research, Mathematics Institute and School of Life Sciences, University of Warwick</span>, Coventry, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Louise%2BDyson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dyson%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALouise%2BDyson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9788-4858" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Louise Dyson</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.04.02.021576v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1224277" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1224277/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.02.021576v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1224277" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1224277/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.02.021576v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1224277" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1224277/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.02.021576v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1224277" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1224277/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.04.02.021576v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1224277" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1224277/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Early warning signals (EWS) identify systems approaching a critical transition, where the system undergoes a sudden change in state. For example, monitoring changes in variance or autocorrelation offers a computationally inexpensive method which can be used in real-time to assess when an infectious disease transitions to elimination. EWS have a promising potential to not only be used to monitor infectious diseases, but also to inform control policies to aid disease elimination. Previously, potential EWS have been identified for prevalence data, however the prevalence of a disease is often not known directly. In this work we identify EWS for incidence data, the standard data type collected by the Centers for Disease Control and Prevention (CDC) or World Health Organization (WHO). We show, through several examples, that EWS calculated on simulated incidence time series data exhibit vastly different behaviours to those previously studied on prevalence data. In particular, the variance displays a decreasing trend on the approach to disease elimination, contrary to that expected from critical slowing down theory; this could lead to unreliable indicators of elimination when calculated on real-world data. We derive analytical predictions which can be generalised for many epidemiological systems, and we support our theory with simulated studies of disease incidence. Additionally, we explore EWS calculated on the rate of incidence over time, a property which can be extracted directly from incidence data. We find that although incidence might not exhibit typical critical slowing down properties before a critical transition, the rate of incidence does, presenting a promising new data type for the application of statistical indicators.</p><div id="sec-1" class="subsection"><p id="p-3"><strong>Author summary</strong> The threat posed by infectious diseases has a huge impact on our global society. It is therefore critical to monitor infectious diseases as new data become available during control campaigns. One obstacle in observing disease emergence or elimination is understanding what influences noise in the data and how this fluctuates when near to zero cases. The standard data type collected is the number of new cases per day/month/year but mathematical modellers often focus on data such as the total number of infectious people, due to its analytical properties. We have developed a methodology to monitor the standard type of data to inform whether a disease is approaching emergence or disease elimination. We have shown computationally how fluctuations change as disease data get closer towards a tipping point and our insights highlight how these observed changes can be strikingly different when calculated on different types of data.</p></div></div><div class="section" id="sec-2"><h2 class="">Introduction</h2><p id="p-4">One of the greatest challenges in society today is the burden of infectious diseases, affecting public health and economic stability all over the world. Infectious diseases disproportionately affect individuals in poverty, with millions of those suffering daily from diseases that are considered eradicable. The potential for eradicating diseases such as polio, guinea worm, measles, mumps or rubella is immense (International Task Force for Disease Elimination, [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]). Even where effective vaccines or treatments exist, disease elimination presents an ongoing challenge. For example, after the establishment of the Global Malaria Eradication Program in 1955 by the World Health Organisation (WHO) it was later abandoned in 1969 due to funding shortages and drug resistance [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>], leading to re-emergence of disease in Europe [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. Assessing when a disease is close enough to elimination to die out without further intervention, thus prompting the end of a control campaign, is a problem of global economic importance. If campaigns are stopped prematurely it can result in disease resurgence and subsequently put control efforts back by decades. Conversely, the threat posed by newly emerging diseases such as SARs, Ebola or the recent corona-virus outbreak COVID-2019 strains available resources, places restrictions on global movement and disrupts the worlds most vulnerable societies. Identifying which newly-emerging diseases will present a global threat, and which will never cause a widespread epidemic is of critical importance.</p><p id="p-5">To overcome the challenges identifying disease elimination or emergence, numerous studies have suggested the use of early warning signals (EWS) [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>–<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. EWS are statistics that may be derived from data that change in a predictable way on the approach to a critical threshold. In epidemiology this threshold is commonly described as the point at which the basic reproduction number, <em>R</em><sub>0</sub>, passes through <em>R</em><sub>0</sub> = 1. A system with <em>R</em><sub>0</sub> increasing through 1 describes an emerging or endemic disease whereas <em>R</em><sub>0</sub> decreasing through 1 results in disease elimination. We seek to find EWS to identify when a disease is approaching such a transition. We may identify such statistics using critical slowing down (CSD) theory, which indicates the imminent approach of a threshold, arising from increasing recovery times of perturbations as a system approaches a critical transition [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. This increase in recovery time occurs because, as the stability of a steady state changes, such as from a disease free state to emergence or from endemic state to elimination, the dominant eigenvalue of the steady state passes through zero. Since the eigenvalue also determines the relaxation time of the system, this recovery time therefore increases as we approach a critical transition.</p><p id="p-6">EWS offer the ability to anticipate a critical transition indirectly in real world noisy time series data, by observing, for example, increasing variance or autocorrelation in the fluctuations around the steady-state [<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. Statistical indicators offer a computationally inexpensive and efficient method for assessing the status of an infectious disease, presenting a simple mechanism for disease surveillance and monitoring of control policies.</p><p id="p-7">The development of EWS is an active area of research in many fields, identifying the statistical signatures of abrupt shifts in many dynamical systems. Studies have applied EWS to historical data or laboratory experiments where a tipping point is known [<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>]; developed methods for using spatial variation [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>], explored the effects of detrending [<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>] using the ensemble of multiple EWS [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>]; and developed understanding of the limitations of EWS [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>].</p><p id="p-8">Discrepancies in statistical signatures have been discovered in a variety of historical datasets known to be going through a critical transition: from climate systems to stock markets, to applications with ecological field data [<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>]. These studies observed unexpected characteristic traits of common EWS, such as identifying a decreasing trend in variance or standard deviation, leading to a discussion on the robustness of indicators. It is therefore highly important to understand analytically how EWS are expected to change on the approach to a critical transition for different data types to avoid any misleading results.</p><p id="p-9">The initial development of EWS in epidemiology focused on prevalence data, producing analytical solutions and numerically testing the capabilities for statistical indicators of emergence and elimination of infectious diseases [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>–<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">8</a>]. Analysis of computer simulations of well-studied epidemiological systems have highlighted challenges such as seasonality [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>] or detrending of epidemiological time series data [<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">8</a>]. However epidemiological data is typically collected in the form of the number of new infectious cases (incidence data) over a certain period of time (weekly/monthly/yearly). Generally, the exact date of infection or recovery of an individual is not known and therefore the exact number of infectious individuals at each point in time (the prevalence data that has been analysed) is unknown.</p><p id="p-10">Simulation-based studies exploring incidence-type data have suggested that the potential for emergence of an infectious disease can be informed by statistical signatures [<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. These studies represent the first attempts to understand the robustness of some indicators when used with disease emergence incidence data, subject to underreporting and time aggregation. Both studies find that EWS do precede disease emergence even when reporting is low. When the numerical performance of 10 EWS are compared, Brett <em>et al.</em> find that the mean and variance perform well unless incidence is subject to a highly overdispersed reporting error and they compare these results with previously studied prevalence results. Theoretical predictions are given for prevalence data, however the analytical behaviour of incidence is not explored.</p><p id="p-11">O’Dea <em>et al.</em> [<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>] incorporate an observation model into a Birth-Death-Immigration (BDI) process to present an analytical study of EWS of disease emergence. This model allows prediction of the behaviour of EWS for dynamics captured by a BDI process but is not suitable for diseases with population-level immunity. O’Dea <em>et al.</em> additionally conducted an investigation into reporting errors in incidence-type data by recording the removal of individuals (“death” component in the BDI process). They describe the probability of a case being reported with either a Binomial or Negative Binomial distribution, allowing for over and under-reporting. In contrast to Brett <em>et al.</em>, they conclude that the mean, variance and coefficient of variation (CV) are poor indicators since they are sensitive to reporting errors and insensitive to differences between transmission and recovery rates.</p><p id="p-12">In this paper, we advance the current literature to describe generalised signatures of statistical indicators for incidence data, on the approach to a threshold, highlighting the differences between EWS descriptors of incidence and prevalence. Strikingly, our results demonstrate that although EWS of emergence exhibit an increasing variance, a trait associated with CSD and supporting results from Brett <em>et al.</em> and O’Dea <em>et al.</em>, we demonstrate that variance instead <em>decreases</em> on the approach to a disease elimination transition. We find that although incidence data does not undergo the transcritical bifurcation traditionally considered by CSD theory, nevertheless time series trends are still a valuable tool to predict disease elimination. The discrepancy between prevalence and incidence on the one hand, and elimination and emergence on the other, could lead to potential problems in detecting thresholds if the differences are not clearly understood.</p><p id="p-13">We introduce an analytical theory from stochastic processes to address why variance in incidence decreases for disease elimination. We study multiple other indicators of disease elimination predicted by this theory, and compare their responses with stochastic simulations. We also consider the rate of incidence as a measurement that can be extracted from incidence data. Notably, we find that on the approach to a critical transition the rate of incidence exhibits typical CSD signatures which correspond with prevalence data, such as an increasing variance. We present a broad analytical framework for EWS of incidence and rate of incidence for a variety of different disease systems. We explore more intricate systems where elimination is driven by different factors to understand the robustness of this theory. This simple generalised result can be applied to many infectious diseases undergoing emergence or elimination, a promising development for EWS of infectious diseases.</p></div><div class="section" id="sec-3"><h2 class="">Methods &amp; Mathematical Theory</h2><p id="p-14">In this paper we focus on the application of EWS to disease elimination where there is a limited understanding on how time series statistics of incidence data behaves on the approach towards this threshold. We consider two simple models, where disease elimination is forced with different mechanisms, to explore how EWS of disease elimination behave for prevalence and incidence data. To demonstrate the broadness of our results, we additionally present a comparative case study to the analytical results for emerging diseases by O’Dea <em>et al.</em></p><p id="p-15">In this section we review the following models: SIS model (Susceptible-Infected-Susceptible model, see for example Keeling &amp; Rohani [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]); SIS model with vaccination and SIS model with external force of infection. For each model that we have chosen to investigate, we derive the stochastic differential equations (SDEs) that describe the analytical behaviour of prevalence in these systems. Derivations of the analytical results and calculations of each statistic can be found in the supporting text (S1 Appendix). We present our analysis for incidence data and derive the corresponding statistical indicators. We exploit the well known fact that a counting process can be described by a Poisson process. We apply this result to the field of EWS to incorporate statistical signatures of a Poisson distributed variable to describe the behaviour of the number of new infectious cases in epidemiology.</p><p id="p-16">We verify our analytical results for prevalence and incidence with simulated studies, and compare the contrasting results between prevalence and incidence. We measure the change in trend of multiple statistical indicators using the Kendall’s Tau score which gives an indication of an increasing or decreasing trend.</p></div><div class="section" id="sec-4"><h2 class="">Prevalence Theory</h2><div id="sec-5" class="subsection"><h3>Model 1: SIS with reducing transmission</h3><p id="p-17">We begin with a simple example of a system that is approaching elimination from an existing endemic state of <em>I</em>. We consider an SIS model where the effective contact rate <em>β</em> acts as the control parameter. Effective reduction of <em>β</em> can be induced by public health campaigns (such as washing hands or improving food hygiene) and through social distancing (such as school closure). By decreasing <em>β</em>(<em>t</em>) in time, it slowly forces <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-1.gif"/></noscript></span></span> through the critical transition at <em>R</em><sub>0</sub> = 1. The model transitions are shown in the following schematic with transition probabilities given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>,</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1230399/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1230399/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1230399" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <span class="caption-title">Transition probabilities in prevalence theory for all models</span><div class="sb-div caption-clear"></div></div></div><p id="p-18">where <em>β</em>(<em>t</em>) changes slowly in time, given by,
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-2.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-3.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-4.gif"/></noscript></span>

</span>
and we fix the population such that <em>N</em> = <em>S</em> + <em>I</em>. Previously work has shown that the fluctuations, <em>ζ</em>, about the prevalence steady state, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-2.gif"/></noscript></span></span> can be separated using the linear noise approximation [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]. The corresponding SDE for <em>ζ</em> defined for the SIS is given by [<a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">8</a>],
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-5.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-6.gif"/></noscript></span>

</span></p></div><div id="sec-6" class="subsection"><h3>Model 2: SIS with increasing vaccination coverage (births and deaths)</h3><p id="p-19">We consider an SIS model where a proportion of susceptible individuals are vaccinated and gain immunity to the disease. By increasing the proportion of individuals vaccinated <em>p</em>(<em>t</em>), this control will reduce the effective reproduction number as the susceptible populations is depleting. Births and deaths are considered to allow for a non-zero steady state of <em>I</em> initially, and to ensure that the susceptible population does not decrease to zero. By increasing the proportion of individuals vaccinated <em>p</em>(<em>t</em>), the system is pushed away from this steady state. The remaining (unvaccinated) individuals enter the susceptible component as shown below,
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-7.gif"/></noscript></span>

</span>
and we gradually increase the proportion of vaccinated individuals over time by,
<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-8.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-9.gif"/></noscript></span>

</span>
to push the system through the critical transition at <em>R</em><sub>0</sub> = 1. We interpret the dynamics of the fluctuations of these SDEs with a two-dimensional Fokker-Planck Equation (see supplementary text S1 Appendix and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> for transition rates),
<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-10.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-11.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-12.gif"/></noscript></span>

</span>
where <em>ζ</em><sub>1</sub> defines the fluctuations about the susceptibles <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-3.gif"/></noscript></span></span> and <em>ζ</em><sub>2</sub> defines the fluctuations about the infecteds <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-4.gif"/></noscript></span></span>, and,
<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-13.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-14.gif"/></noscript></span>

</span></p><p id="p-20">In both cases it is assumed that the fluctuations can be separated linearly from the steady state. Many statistics of this system can be described by the covariance matrix Θ, given by Θ<sub><em>ij</em></sub>⟨⟨<em>ζ</em><sub><em>i</em></sub><em>ζ</em><sub><em>j</em></sub>⟩⟩ = ⟨<em>ζ</em><sub><em>i</em></sub><em>ζ</em><sub><em>j</em></sub>⟩ − ⟨<em>ζ</em><sub><em>i</em></sub>⟩⟨<em>ζ</em><sub><em>j</em></sub>⟩ which satisfies,
<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-15.gif"/></noscript></span>

</span></p><p id="p-21">In particular, the variance of the fluctuations about the infectious steady state is given by Θ<sub>22</sub>.</p></div><div id="sec-7" class="subsection"><h3>Model 3: SIS with external force of infection and increasing transmission</h3><p id="p-22">Finally we consider the SIS model with external infection which has been used to investigate EWS in prevalence and in incidence [<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>]. We demonstrate how our analytical results compare for this system, and illustrate differences when applied to disease elimination.</p><p id="p-23">In this model, in addition to the underlying SIS dynamics, susceptible individuals can be infected by an external force of infection (governed by parameter <em>ν</em>) that does not depend on the level of infection, shown in the schematic below,
<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-16.gif"/></noscript></span>

</span>
and we consider the model in a stochastic formulation, with transition probabilities given in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>. Disease emergence is driven by increasing the effective contact rate <em>β</em>(<em>t</em>) over time, that slowly increases <em>R</em><sub>0</sub> through the critical transition at <em>R</em><sub>0</sub> = 1,
<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-17.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-18.gif"/></noscript></span>

</span></p><p id="p-24">The fluctuations, <em>ζ</em> about the steady state are an extension of those in Model 1 to include the external force of infection and satisfy,
<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-19.gif"/></noscript></span>

</span></p><p id="p-25">The study by O’Dea <em>et al.</em> approximated an SIS with a Birth-Death-Immigration process, where an immigration event approximates the external force of infection; birth events give new infections and the death component is analogous with recovery events. O’Dea <em>et al.</em> derive statistics for incidence data by monitoring the number of individuals recovering (e.g. the transition rate <em>T</em> (<em>I −</em>1<em>|I</em>) = <em>γI</em>). Results from this study can be found in the supplementary text (S1 Appendix).</p><p id="p-26">One limitation of this methodology is its inability to extend to other systems. This derivation would need to be computed again for each specific example. In particular, if one wanted to consider a simpler system with no external forcing, by setting <em>ν</em> = 0, it would make this result redundant - prompting our study for generic EWS that can describe all systems.</p></div></div><div class="section" id="sec-8"><h2 class="">Incidence Theory</h2><p id="p-27">A counting process can be used as a generalised theory to understand the dynamics of the number of new events over a period of time. In particular, a diverse range of data types can be described by a counting process and this motivates us to characterise how statistics of such processes behave on the approach to a critical transition. Incidence (the number of new cases, <em>N</em><sub><em>t</em></sub>) is a counting process, which is known to be described by a non-homogeneous Poisson process {<em>N</em> (<em>t</em>) : <em>t ∈</em> [0, <em>∞</em>)} with time dependent rate <em>λ</em>(<em>t</em>),
<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-20.gif"/></noscript></span>

</span>
where the integral approximation holds for Δ<em>t</em> sufficiently small. In the supporting text (S2 Fig) we demonstrate that for our parameters, this approximation works well for Δ<em>t</em> up to 3. We can derive EWS in disease incidence aggregated over a time interval Δ<em>t</em> (e.g. daily, weekly, biweekly cases) using the well-known central moments of the Poisson distribution:
<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-21.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-22.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-23.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-24.gif"/></noscript></span>

</span></p><p id="p-28">Prior work from O’Dea <em>et al.</em> [<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">4</a>] has also incorporated under-reporting using a negative binomial distribution; this can be included in this model when the rate <em>λ</em>(<em>t</em>) is gamma distributed.</p><p id="p-29">Without under-reporting the rate of new cases is given by the incoming transition probabilities to the infectious state,
<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-25.gif"/></noscript></span>

</span></p><p id="p-30">A common form of this force of infection is,
<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-26.gif"/></noscript></span>

</span>
so that <em>λ</em>(<em>t</em>) depends on the prevalence of infection, <em>I</em>(<em>t</em>). For Model 1, <em>β</em>(<em>t</em>) is a function of time whereas in Model 2 <em>β</em>(<em>t</em>) = <em>β</em><sub>0</sub> is fixed. Infection can also be increased in other ways such as an external force of infection (Model 3), <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-5.gif"/></noscript></span></span>, that is typically used to describe zoonotic spillover events or as an approximation for human migration.</p></div><div class="section" id="sec-9"><h2 class="">Rate of Incidence Theory</h2><p id="p-31">We consider the rate of incidence (or the rate of the Poisson process) <em>λ</em>(<em>t</em>) = <em>T</em> (<em>I</em> + 1<em>|I</em>), which can be described dynamically with an SDE. Our analyses shows that the critical transition of the rate of the Poisson process corresponds to prevalence models (e.g. at <em>R</em><sub>0</sub> = 1) and importantly exhibits behaviours predicted by CSD.</p><p id="p-32">We investigate here calculating statistics on the rate of incidence (RoI) and its potential to be used as an EWS for disease transitions.</p><p id="p-33">Below we present our analytical results describing statistical indicators for each model. These theoretical solutions can be used to derive time-varying indicators for the fluctuations of the rate of incidence. Full derivations of the analytical work are given in the supporting text: S1 Appendix.</p><div id="sec-10" class="subsection"><h3>Model 1</h3><p id="p-34">When considering Model 1 (SIS with decreasing transmission rate), we describe the change in the rate of incidence as <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-6.gif"/></noscript></span></span>. In particular, by considering the time derivative of <em>λ</em><sub><em>t</em></sub> we can conclude that the fixed points of the rate of incidence can be described by the transcritical bifurcation at <em>R</em><sub>0</sub> = 1. We find that the stability of the fixed points of <em>λ</em><sub><em>t</em></sub> also correspond to those of <em>I</em>, as expected.</p><p id="p-35">We describe the fluctuations, <em>ω</em>, about the steady state of <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-7.gif"/></noscript></span></span> using the linear noise approximation (LNA). We are interested in statistics calculated on the fluctuations about the rate of incidence, to develop new indicators of disease elimination. We derive the resulting analytical solution for <em>ω</em> using Ito’s Change of variable formulae (details in supporting text: S1 Appendix) to approximate <em>ω</em> with the following Gaussian process:
<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-27.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-28.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-29.gif"/></noscript></span>

</span></p><p id="p-36">In particular, the changing behaviour of the variance of the rate of incidence as the system approaches disease elimination can be calculated from the SDE <a id="xref-disp-formula-28-1" class="xref-disp-formula" href="#disp-formula-28">Eqn, 25</a>,
<span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-30.gif"/></noscript></span>

</span></p></div><div id="sec-11" class="subsection"><h3>Model 2</h3><p id="p-37">If we consider models where there is population-level immunity, then <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-8.gif"/></noscript></span></span> and we can no longer reduce the dimension of incoming transitions using <em>S</em> = <em>N−I</em>. This can be seen in Model 2 (SIS with increasing rate of vaccination), in particular the prevalence analysis of these systems presented in the Methods Section results in a multivariable Fokker-Plank Equation.</p><p id="p-38">However, we can similarly describe the fluctuations, <em>ω</em>, about the steady state of <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-9.gif"/></noscript></span></span> using the linear noise approximation (LNA) as in Model 1.</p><p id="p-39">We again use Ito’s Change of variable formulae for the multivariable system (which will depend on <em>ζ</em><sub>1</sub> and <em>ζ</em><sub>2</sub>) to approximate <em>ω</em>. This leads to an SDE equation which depends on the description of <em>ζ</em><sub>1</sub> and <em>ζ</em><sub>2</sub> (<a id="xref-disp-formula-11-1" class="xref-disp-formula" href="#disp-formula-11">eqn. 9</a>). In particular, we are interested in statistics of the rate of incidence, such as the variance, which can be simplified in terms of the original covariance matrix Θ (<a id="xref-disp-formula-14-1" class="xref-disp-formula" href="#disp-formula-14">eqn. 12</a>) and mean-field equations of infectious (<em>φ</em>) and susceptible (<em>ψ</em>) individuals,
<span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-31.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-32.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-33.gif"/></noscript></span>

</span></p></div><div id="sec-12" class="subsection"><h3>Model 3</h3><p id="p-40">Incorporating an external force of infection into the SIS model is a direct extension of the Model 1 analysis on the rate of incidence. Incoming transitions into the infectious class increases from <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-10.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-11.gif"/></noscript></span></span>, the case which is observed in Model 3 (SIS with external force of infection and increasing transmission).</p><p id="p-41">Following, we can describe the fluctuations about the rate of incidence (<em>ω</em>) using the LNA. In particular,
<span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-34.gif"/></noscript></span>

</span></p><p id="p-42">As previously, we can derive statistics of <em>ω</em> from the solution of this SDE. In particular, since the SDE is linear in <em>ω</em> then we can describe <em>ω</em> as a Gaussian variable with mean zero and variance given by the solution to the following ODE,
<span class="disp-formula" id="disp-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-35.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-36.gif"/></noscript></span>

</span></p></div></div><div class="section" id="sec-13"><h2 class="">Simulated Study</h2><p id="p-43">We use the Gillespie algorithm to simulate each model, using time varying parameters (<em>β</em>(<em>t</em>) for Model 1 &amp; 3 and <em>p</em>(<em>t</em>) for Model 2) to drive the model either to extinction (Model 1 &amp; 2) or emergence (Model 3). We record prevalence outputs at time steps of 0.1 per day and we aggregate incidence outputs to daily time steps Δ<em>t</em> = 1. Parameters common to each model are given in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">table 2</a>. In Model 1, the transmission parameter <em>β</em> was reduced from <em>β</em><sub>0</sub> = 1 to 0, slowly forcing <em>R</em><sub>0</sub> = 5 to 0. In Model 2, the rate of vaccination was increased from <em>p</em><sub>0</sub> = 0 to 1, slowly forcing <em>R</em><sub>0</sub> = 5 to 0. In Model 3, the transmission parameter <em>β</em> was increased from <em>β</em><sub>0</sub> = 0.12 to 0.24 so that the basic reproduction number increases from <em>R</em><sub>0</sub><em>≈</em>0.6 to<em>≈</em>1.2.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1230396/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1230396/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1230396" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <span class="caption-title">Parameter values shared among all models.</span><div class="sb-div caption-clear"></div></div></div><p id="p-44">A drawback of using the rate of incidence (RoI) as a measure of disease elimination, is the need to develop methods to extract this rate from incidence data. In our simulation study, we calculate the RoI in two ways. Firstly, using simulations of prevalence and taking the product <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-12.gif"/></noscript></span></span>. This method, although unrealistic as it requires knowlege of prevalence (<em>I</em>), demonstrates the accuracy of the analytical results, as it is the “true” definition of RoI. An alternative method uses the Poisson property of incidence, illustrating that the rate of incidence <em>λ</em>(<em>t</em>) also corresponds to the mean incidence over time. Therefore, our second method evaluates RoI by linearly smoothing the Gillespie output of incidence (<em>N</em><sub><em>t</em></sub>) on a rolling window, giving the approximated mean number of new cases for each realisation (we refer to this method as “approximated” RoI). We can then calculate statistics of the fluctuations of the rate of incidence over multiple realisations of this smoothed time series.</p><p id="p-45">For each model, we also perform simulations where the disease has not fully gone through a critical transition (null model) which we refer to as Fix simulations. This null model has no control mechanism and the disease fluctuates about the fixed endemic steady state, at <em>R</em><sub>0</sub> = 5 (Model 1 &amp; 2) and <em>R</em><sub>0</sub> = 0.5 (Model 3).</p><p id="p-46">Before calculating the time changing statistics, we detrend each simulation by removing the mean over all realisations of that setting (Ext, Emg or Fix). We are interested in five common statistical indicators: variance (V), coefficient of variation (CV), skewness (SK), kurtosis (K) and autocorrelation lag-1 (AC(1)). We illustrate how EWS change over time, and how accurate the theory is to predicting these trends. In particular, we are interested in the trends over time and whether these time series properties of EWS are the same for prevalence and incidence data.</p><p id="p-47">The Kendall-tau score gives a measure of an increasing or decreasing trend of each statistic over the time series (where <em>N</em> is the number of time points). We use the measure to evaluate whether a statistic corresponds to an increasing or decreasing trend and compare this for different data types (prevalence, incidence and RoI). The Kendall-tau score is defined as [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>],
<span class="disp-formula" id="disp-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/graphic-38.gif"/></noscript></span>

</span>
where two points in the time series <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-13.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-14.gif"/></noscript></span></span> with <em>t</em><sub>1</sub> <em>&lt; t</em><sub>2</sub> are said to be a concordant pair if <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-15.gif"/></noscript></span></span>, and a discordant pair if <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-16.gif"/></noscript></span></span>. If the two points are equal <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-17.gif"/></noscript></span></span> then the pair is neither concordant or discordant.</p><p id="p-48">We calculate each statistic on a moving window (size 50) for each detrended simulation, and then evaluate the Kendall-tau score. We compare the Kendall-tau scores calculated on simulations going through a critical transition with null simulations, and we then calculate receiver operating characteristic (ROC) curves by considering the null model to be negative and the other models to be positive. We compare the performance of each model statistic using the area under the curve (AUC). Good statistics have an AUC close to 1 or 0 since this indicates the statistic is far from picking by chance.</p></div><div class="section" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>Variance</h3><p id="p-49">Variance is one of the most intuitive statistical indicators. As a system approaches a critical transition the time taken to recover from small perturbations increases, as described by Critical Slowing Down theory. This can be observed in the fluctuations about the steady state, which on the approach to a critical transition take longer to return and consequently vary far more, defining the increasing nature of variance as an early warning signal.</p><p id="p-50">We evaluate analytical solutions of the variance in prevalence using the derived SDE for each model (Model 1: <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">Eqn.4</a>, Model 2: <a id="xref-disp-formula-11-2" class="xref-disp-formula" href="#disp-formula-11">Eqn.9</a>, Model 3: <a id="xref-disp-formula-18-1" class="xref-disp-formula" href="#disp-formula-18">Eqn.15</a>). We compare this to theoretical solutions of the variance in incidence, using the transition rates for each model (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) to compute the rate of the Poisson process <em>λ</em>. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> presents the simulated statistics for both prevalence and incidence theories of elimination and emergence, where we have plotted the variance between multiple homogeneous simulations at each time point (described in supporting text: S3 Fig).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.04.02.021576/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparing predictions to simulations for variance. For each model (Model 1: SIS decreasing transmission; Model 2: SIS increasing vaccination; Model 3: SIS increasing transmission, emergence) we calculate the variance between 500 homogeneous realisations at every time step (daily). Each figure shows: Poisson process distribution (green line); dynamic predictions (red line) and Gillespie simulations (Ext: Model 1 and 2 and Emg: Model 3, blue line). The last model also shows the dynamical prediction from O&#x2019;Dea et al. which was derived for this specific system (orange line)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1520520622" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Comparing predictions to simulations for variance.&lt;/span&gt; For each model (Model 1: SIS decreasing transmission; Model 2: SIS increasing vaccination; Model 3: SIS increasing transmission, emergence) we calculate the variance between 500 homogeneous realisations at every time step (daily). Each figure shows: Poisson process distribution (green line); dynamic predictions (red line) and Gillespie simulations (Ext: Model 1 and 2 and Emg: Model 3, blue line). The last model also shows the dynamical prediction from O&#x2019;Dea &lt;em&gt;et al.&lt;/em&gt; which was derived for this specific system (orange line).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.04.02.021576/F1.medium.gif" width="323" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.04.02.021576/F1.medium.gif" width="323" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.04.02.021576/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.04.02.021576/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig 1.</span> <span class="caption-title">Comparing predictions to simulations for variance.</span><p id="p-51" class="first-child">For each model (Model 1: SIS decreasing transmission; Model 2: SIS increasing vaccination; Model 3: SIS increasing transmission, emergence) we calculate the variance between 500 homogeneous realisations at every time step (daily). Each figure shows: Poisson process distribution (green line); dynamic predictions (red line) and Gillespie simulations (Ext: Model 1 and 2 and Emg: Model 3, blue line). The last model also shows the dynamical prediction from O’Dea <em>et al.</em> which was derived for this specific system (orange line).</p><div class="sb-div caption-clear"></div></div></div><p id="p-52">Our prediction for the variance is similar to the stochastic simulations with a slight underdispersion in the incidence simulations (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1 a(i), b(i)</a> and <a id="xref-fig-1-3" class="xref-fig" href="#F1">c(i)</a>), since the theory (rate of Poisson process, green line) is higher than that of the variance in the fluctuations of the simulations (blue line). We note that our solution for Model 3 (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1</a>) follows the gradient of the stochastic simulations more closely. Since the analytical solution by O’Dea <em>et al.</em> (orange line) is evaluated at the steady state then the result diverges at the critical transition. It can be shown that for larger values of <em>β</em><sub>0</sub>, O’Dea <em>et al.</em> results fit closer to the stochastic simulation, although the general trend of variance for both approaches follows the simulations.</p><p id="p-53">We observe that variance in prevalence simulations (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1 a(ii), b(ii)</a> and <a id="xref-fig-1-6" class="xref-fig" href="#F1">c(ii)</a>) increases on the approach to the critical transition, as predicted by critical slowing down. In comparison the variance in incidence decreases before the critical transition for all disease elimination models (Model 1 <a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1 a(i)</a> and Model 2 <a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig. 1 b(i)</a>) and increases similarly to prevalence for the disease emergence model (Model 4 <a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig. 1 c(i)</a>). This is contradictory to the theory of critical slowing down theory which predicts that derivations from the steady state values return increasingly slowly on the approach to a transition.</p><p id="p-54">As expected by our Poisson process analysis, the variance of this system should be the same as the mean of the system. Therefore for disease elimination models, we should expect a decreasing variance (along with a decreasing mean) when calculated on incidence data, in contrast to an increasing variance with prevalence data. Likewise with disease emergence models we expect an increasing variance to correspond to the increasing mean. This demonstrates that our analysis of incidence has successfully predicted the time-varying variance for these different systems.</p></div></div><div class="section" id="sec-16"><h2 class="">Rate of incidence</h2><p id="p-55">We have observed that incidence data does not approach a critical transition as described by critical slowing down theory. Consequently we demonstrated in <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1</a> that the variance of incidence does not necessarily increase on the approach to a critical transition. A new approach for working with incidence-type data is to consider the rate of incidence, <em>λ</em>(<em>t</em>) = <em>T</em> (<em>I</em> + 1<em>|I</em>), which for each model we have derived the dynamical SDE (see Methods).</p><div id="sec-17" class="subsection"><h3>Variance (RoI)</h3><p id="p-56">The analytical variance in the rate of incidence is presented in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a> (orange line). We find that the theoretical analysis supports the simulation studies and provides us with an understanding of how statistical indicators calculated in RoI data change on the approach to a critical transition. We observe an increasing variance in the rate of incidence before a critical transition; a time series trend which if exhibited in real-world data could be used to anticipate disease tipping points.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.04.02.021576/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variance calculated on the rate of incidence For each model (Model 1: SIS decreasing transmission; Model 2: SIS increasing vaccination; Model 3: SIS increasing transmission, emergence) we calculate the variance on the rate of incidence. between 50 homogeneous realisations at every time step (daily). Each figure shows: dynamic solution (orange line); rate calculated from Gillespie simulations of incidence after smoothing (light blue line) and Gillespie simulations prevalence multiplied by time-varying susceptible and effective contact rate (, purple line)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1520520622" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Variance calculated on the rate of incidence&lt;/span&gt; For each model (Model 1: SIS decreasing transmission; Model 2: SIS increasing vaccination; Model 3: SIS increasing transmission, emergence) we calculate the variance on the rate of incidence. between 50 homogeneous realisations at every time step (daily). Each figure shows: dynamic solution (orange line); rate calculated from Gillespie simulations of incidence after smoothing (light blue line) and Gillespie simulations prevalence multiplied by time-varying susceptible and effective contact rate (, purple line).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.04.02.021576/F2.medium.gif" width="440" height="396"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.04.02.021576/F2.medium.gif" width="440" height="396"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.04.02.021576/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.04.02.021576/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 2.</span> <span class="caption-title">Variance calculated on the rate of incidence</span><p id="p-57" class="first-child">For each model (Model 1: SIS decreasing transmission; Model 2: SIS increasing vaccination; Model 3: SIS increasing transmission, emergence) we calculate the variance on the rate of incidence. between 50 homogeneous realisations at every time step (daily). Each figure shows: dynamic solution (orange line); rate calculated from Gillespie simulations of incidence after smoothing (light blue line) and Gillespie simulations prevalence multiplied by time-varying susceptible and effective contact rate (<span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/F2/embed/inline-graphic-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/F2/embed/inline-graphic-18.gif"/></noscript></span></span>, purple line).</p><div class="sb-div caption-clear"></div></div></div><p id="p-58">We present results calculated in RoI simulations using the two methods: “true” and “approximated” RoI, in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>. The first method uses prevalence data (“true”, purple line) and corresponds well with the analytical solution (orange line) for all models and the latter method (smoothing incidence data “approximated”, blue line) fits particularly well for Model 3 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2(c)</a>). However it does not follow as closely to some time-varying properties of the variance for Model 1 &amp; 2 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2(a)</a> and <a id="xref-fig-2-5" class="xref-fig" href="#F2">(b)</a>) respectively. Although the early dynamics are misrepresented for Models 1 and 2, all time series indicate an increasing variance on the approach to the critical transition. In particular, “approximated” RoI can be implemented in practice from incidence type data (blue line), captures this property in all models.</p><p id="p-59">In <a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2(a)</a> and <a id="xref-fig-2-7" class="xref-fig" href="#F2">(b)</a> we observe that the analytical prediction fits well with the stochastic simulations of <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-19.gif"/></noscript></span></span> (purple line) for Model 1 and Model 2 respectively. This demonstrates that this theory approximates the behaviour of the system well. Indeed, we observe that approximating the rate of incidence by smoothing Gillespie simulations of new cases and then calculating the variance of this quantity (“approximated” RoI) predicts a similar increasing behaviour before the critical transition, shown in <a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2(a)</a> (blue line). This corresponds to the same peak as the analytical prediction and “true” simulations. However, it fails to capture the magnitude of the behaviour earlier on in the dynamics.</p><p id="p-60">An area that still needs to be addressed with this methodology of smoothing new case data is determining a suitable window size. This could result to misleading EWS when used in practice. In the supporting text S1 Fig, we demonstrate that if the disease is approaching elimination at a slower rate, both methods (“true” and “approximated”) converge to the analytical solution. We chose parameters such that Model 1 approaches disease elimination at the same rate as Model 3 approaches disease emergence (<em>R</em><sub>0</sub> changes from 1.2 to 0, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/02/2020.04.02.021576/embed/inline-graphic-20.gif"/></noscript></span></span>). As the system changes slowly enough then the system will be approximately ergodic, such that the moving average resembles the mean incidence. Thus the “approximated” method will be closer to the “true” solution. In comparison, the faster a system changes over time, will correspond to a wider range in incidence cases across the moving window. Resulting in a lower mean over the window which can be seen in <a id="xref-fig-2-9" class="xref-fig" href="#F2">Fig. 2(a),(b)</a>; although the statistic will be more pronounced at the threshold.</p><p id="p-61">We find that for Model 2 (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Fig. 2(b)</a>) the general trend of the variance is less pronounced at the critical transition than observed for Model 1. We observe that the analytical solution (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Fig. 2(b)</a> orange line) and true stochastic simulations (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Fig. 2(b)</a> purple) only slightly increase before the critical transition, implying this trend would be difficult to detect in real-world data. In particular, the Kendall-tau score which can be an indication of an increasing trend, is negative (decreasing, <em>τ</em> =<em>−</em>1) for Model 2, whilst for Model 1 and 3 we find that <em>τ</em> = 0.987 and <em>τ</em> = 1 respectively. Although, we observe that the “approximated” simulations of the rate of incidence (<a id="xref-fig-2-13" class="xref-fig" href="#F2">Fig. 2(b)</a> blue line) exhibit similar properties as Model 1. We observe that the early stage dynamics of this method have not predicted the expected behaviour of the analytical solution. It can be noted that <em>R</em><sub>0</sub> decreases at the same rate as Model 1, suggesting that this can be a result of the approximation when <em>R</em><sub>0</sub> is not slowly changing. Due to this approximation, it can be observed that the variance of new cases does therefore increase before the critical transition (blue line).</p><p id="p-62">In <a id="xref-fig-2-14" class="xref-fig" href="#F2">Fig. 2(c)</a> we observe that both measurements of the variance of <em>λ</em><sub><em>t</em></sub> calculated on stochastic simulations of Model 3 have closely followed the analytical solution of variance. As expected the true stochastic simulations (<a id="xref-fig-2-15" class="xref-fig" href="#F2">Fig. 2(c)</a> purple line) follow closely to the theory, supporting that this derivation of <em>ω</em> is correct. More interestingly, calculating the variance of the rate of incidence directly from simulations of new cases (<em>N</em><sub><em>t</em></sub>, <a id="xref-fig-2-16" class="xref-fig" href="#F2">Fig. 2(c)</a> blue line) has performed far better than when presented in Model 1 (<a id="xref-fig-2-17" class="xref-fig" href="#F2">Fig. 2(a)</a>). For Model 3, we observe that the variance of the rate of incidence increases before the critical threshold, where the infectious disease emerges with outbreaking dynamics similar to prevalence for this model. We further found that the early dynamics of the “approximated” RoI simulations represent the true behaviour of the variance. This result may be due to <em>R</em><sub>0</sub> increasing more slowly in Model 3 than the rate it decreases at in Model 1, satisfying the ergodic condition.</p></div></div><div class="section" id="sec-18"><h2 class="">Other statistical indicators</h2><p id="p-63">In this section, we investigate the potential of identifying an epidemiological transition using five commonly implemented early-warning signals: variance, coefficient of variation (CV), skewness, kurtosis and lag-1 autocorrelation (AC(1)). Exploration of each EWS follows similarly to variance, as analysed above theoretically and numerically for prevalence, incidence and rate of incidence. In the supporting text, time series trends for each indicator are presented for each dataset and model (S3 Fig: variance, S4 Fig: CV, S5 Fig: Skewness, S6 Fig: kurtosis, S7 Fig: AC(1)), along with analytical analyses for these indicators (S1 Appendix).</p><p id="p-64">Here, we quantify these time series trends for each statistical indicator using the Kendall-Tau score as a measure of an overall increasing or decreasing trend. We present in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a> the predictive power of each statistical indicator using its time-changing trend to classify simulations as either extinct (Ext simulations), emerging (Emg simulations) or null simulations (Fix simulations). We calculate the area under the ROC curve (AUC) by comparing the Kendall-tau score calculated over each time series up to two end points: before the critical transition (<em>t</em><sub>1</sub>) and after the critical transition (<em>t</em><sub>2</sub>) which gives an overall score of the true/false positive rate.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.04.02.021576/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="AUC scores for different EWS. We compare the performance of 5 common statistical indicators for Model 1 (a) for disease elimination and Model 3 (b) for disease emergence. For each ROC curve, we measured the AUC which is an indication of how predictive each indicator is by its ability to distinguish between elimination simulations and the null model. A score closer to 0.5 signifies the worst performance (random diagnosis). We evaluate the Kendall-tau score up to before the critical transition (t1 = 390) and after the critical transition (t2 = 450), which gives an indication if the EWS is increasing or decreasing. A score of 1 demonstrates that it is possible to identify all Ext simulations when compared to null simulations by its increasing trend (i.e. perfect sensitivity, true positive rate). A score of 0 means that there is zero sensitivity and instead the simulations are decreasing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1520520622" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;AUC scores for different EWS.&lt;/span&gt; We compare the performance of 5 common statistical indicators for Model 1 (a) for disease elimination and Model 3 (b) for disease emergence. For each ROC curve, we measured the AUC which is an indication of how predictive each indicator is by its ability to distinguish between elimination simulations and the null model. A score closer to 0.5 signifies the worst performance (random diagnosis). We evaluate the Kendall-tau score up to before the critical transition (&lt;em&gt;t&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; = 390) and after the critical transition (&lt;em&gt;t&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; = 450), which gives an indication if the EWS is increasing or decreasing. A score of 1 demonstrates that it is possible to identify all Ext simulations when compared to null simulations by its increasing trend (i.e. perfect sensitivity, true positive rate). A score of 0 means that there is zero sensitivity and instead the simulations are decreasing.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.04.02.021576/F3.medium.gif" width="228" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.04.02.021576/F3.medium.gif" width="228" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.04.02.021576/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/02/2020.04.02.021576/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig 3.</span> <span class="caption-title">AUC scores for different EWS.</span><p id="p-65" class="first-child">We compare the performance of 5 common statistical indicators for Model 1 (a) for disease elimination and Model 3 (b) for disease emergence. For each ROC curve, we measured the AUC which is an indication of how predictive each indicator is by its ability to distinguish between elimination simulations and the null model. A score closer to 0.5 signifies the worst performance (random diagnosis). We evaluate the Kendall-tau score up to before the critical transition (<em>t</em><sub>1</sub> = 390) and after the critical transition (<em>t</em><sub>2</sub> = 450), which gives an indication if the EWS is increasing or decreasing. A score of 1 demonstrates that it is possible to identify all Ext simulations when compared to null simulations by its increasing trend (i.e. perfect sensitivity, true positive rate). A score of 0 means that there is zero sensitivity and instead the simulations are decreasing.</p><div class="sb-div caption-clear"></div></div></div><p id="p-66">The AUC score gives a predictive measure between different indicators, which we use to assess their performances. The closer the AUC is to 0.5 signifies the worst the statistic’s performance at anticipating a critical transition. This is analogous with randomly selecting simulations that are the null and disease elimination (Model 1, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3(a)</a>) or disease emergence (Model 3, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3(b)</a>). In particular, skewness is a poor indicator because of its inability to identify disease elimination with any type of disease data it is applied to (rate of incidence, incidence and prevalence). Identifying emergence with skewness in prevalence or RoI data (red and orange bars respectively) is also very poor and its predictive ability is only slightly increased with incidence (green bars).</p><p id="p-67">A score close to 1 indicates nearly perfect sensitivity and specificity. For each EWS, we assume that an increasing trend represents a disease going through a critical transition. As a result a AUC score of 1 informs us that the indicator is increasing and that it is possible to identify all Ext/Emg simulations when compared to the null simulations by its increasing trend. Notably, in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3</a> coefficient of variation calculated on all types of disease data (rate of incidence, incidence and prevalence) and for both Model 1 &amp; 2, exhibits a near perfect ability to identify the increasing trend.</p><p id="p-68">An AUC score of 0 demonstrates that the time series trend is instead decreasing and as such it doesn’t correspond to the predetermined prediction. A perfectly diagnosed decreasing indicator when compared to the null model will result in zero sensitivity under these conditions and an AUC score of 0. <a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3</a> highlights which indicators are in some cases increasing (AUC close to one), decreasing (AUC close to zero) or are poor indicators (AUC close to 0.5). In particular, as discussed in the previous section, variance always increases prior to disease emergence (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3(b)</a>). However, for disease elimination (Model 1: <a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3(a)</a> and Model 2: S11 Fig) results are substantially different when we compare variance calculated in rate of incidence and prevalence (orange and red bars respectively) with incidence (green bars). For RoI and prevalence data types, the statistical signature is an increasing variance with an AUC near 1. This is in contrast to the latter where the trend is decreasing with an AUC near 0. However, the results for variance (both increasing and decreasing) are highly predictive (<em>|AUC −</em> 0.5<em>| ≈</em> 0.5). Thus, if a system is not known or there is difficultly in determining the type of data, incorrect conclusions could be drawn when interpreting the time series trend.</p></div><div class="section" id="sec-19"><h2 class="">Discussion</h2><p id="p-69">While studies for EWS on incidence-type data have been growing in recent years, theoretical exploration of how these indicators change on the approach to a critical transition have been neglected. In this paper, we have shown that the typical trends of EWS that precede a critical transition are exhibited in prevalence-type data but do not always exist in incidence-type data. In particular, we have focused our investigation on the trend of variance over time as an infectious disease system approaches a tipping point.</p><p id="p-70">Prior work has shown that variance in incidence increases on the approach towards disease emergence. However, our work highlights that this property is not a result from critical slowing down theory as first expected. We have shown it is a consequence of the counting process that can approximate incidence-type data. As such, we demonstrated that the variance in incidence is expected to follow the mean in incidence. In particular, the variance will increase on the approach to disease emergence, but will notably decrease before a disease elimination threshold. We applied these findings to two systems of disease elimination and verified that variance of incidence exhibits a decreasing trend on the approach, following the behaviour of the mean and contradicting critical slowing down theory.</p><p id="p-71">Therefore, it is highly recommended to understand analytically how EWS change on the approach to a critical transition in order to avoid misleading results. The generalised theory of a counting process can be applied to many other systems outside of the scope of epidemiology where we would expect a decreasing variance preceding a critical transition. Potential applications include the observation of animals through camera traps, disease surveillance sampling in wildlife or movements in stock prices, which are all examples of incidence-type data. Notably, a substantial number of studies on ecosystem data, climate data and financial data have observed inconsistencies in statistical indicators [<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]. Although we found the Poisson process to be overdispersed in the context of epidemiology, it provides a broad framework which can be extended to many other infectious disease systems using the incoming transition probabilities into the infectious class.</p><p id="p-72">We proposed extracting the rate of incidence (RoI) or intensity of Poisson process from incidence-type data to illustrate that to utilising CSD, such as observing an increasing variance, requires suitable data which undergoes a bifurcation. In particular, we have shown that the critical threshold in the RoI corresponds with that of prevalence; and as expected we demonstrated that the trend in variance in RoI does increase before an imminent epidemiological transition.</p><p id="p-73">We applied five early warning signals to simulated datasets comprising of the three discussed types: prevalence, incidence and rate of incidence. The simulated data we have investigated represents perfect reporting or the “best case scenario”. Often is the case that there is underreporting that may reduce the detectability of signals in real-world data. The work we have presented here can be extended to include a gamma distributed intensity <em>λ</em>. Using a gamma distributed rate of incidence will account for reporting errors as described by O’Dea <em>et al.</em></p><p id="p-74">Overall, our study suggests that a robust indicator is one that shares a highly predictive time series trait (<em>|AUC −</em> 0.5<em>| ≈</em> 0.5). amongst all three data types, even with inconsistent trends (increasing or decreasing). Therefore, we suggest that variance and coefficient of variation are overall good indicators due to their high predictive power in all cases. Coefficient of variation is a robust indicator for disease elimination since the trend is similar between different types of data (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig. 3 (a)</a>) and S11 Fig. However discrepancies are demonstrated when considering opposite disease thresholds as shown with disease emergence (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Fig. 3(b)</a>) which has a decreasing trend for CV and performs less well with disease prevalence data.</p><p id="p-75">However, we found that kurtosis and AC(1) are not robust indicators. Although kurtosis and AC(1) have a predictive trend with prevalence data, this is not typically the data which is readily available. In particular, kurtosis is highly predictive (with a decreasing trend) in prevalence data on the approach to disease elimination (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Fig. 3(a)</a>) and fairly predictive with an decreasing trend in the case of prevalence with emergence (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Fig. 3(b)</a>); it is a poor indicator for all other types of data. Likewise, although AC(1) has a clear increasing trend for prevalence data elimination systems (<a id="xref-fig-3-12" class="xref-fig" href="#F3">Fig. 3(a)</a>, S11 Fig), it is less predictive trend for incidence and RoI data. Additionally, the trend is not distinct for any datasets when considering an emergence transition, therefore there is a potential for this indicator to be used incorrectly. In the cases where an EWS is poor in some types of data but good for others could lead to misleading judgements of systems, and therefore are not robust.</p><p id="p-76">These findings support prior work on prevalence and initial work from O’Dea <em>et al.</em> and Brett <em>et al.</em> with incidence-type data. Our analytical exploration of incidence has indicated a new data source, RoI, which can be extracted from incidence timeseries. A potential powerful tool would be to compute variance and CV indicators with different types of data (incidence, rate of incidence and prevalence) and ensemble these. An ensemble or combination of multiple statistical indicators was suggested by Drake &amp; Griffen [<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a>] and has been applied to case studies with the same data-type and a combination of EWS by Kefi <em>et al.</em> [<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>] to help interpret between different critical transitions and also has successfully detected transitions using an ensemble of different time series data [<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>]. This suggests a potential approach to achieve a single metric from a combination of indicators calculated on multiple timeseries data with different trends, such as we have observed with incidence and RoI, to achieve a more pronounced indication of disease transitions.</p><p id="p-77">Additionally, further work would be to include a heterogeneous ensemble as suggested by O’Dea <em>et al.</em> [<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">4</a>], whereby all parameters are sampled randomly for each realisation rather than being equal. This will lead to more realistic results, as each parameter sample represents time series data from different locations, as suggested by studies on spatial statistics, a promising method for addressing limited data [<a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>]. Comparatively, we have shown here that computing the statistics on a homogeneous ensemble although unrealistic, it returns exact stochastic behaviours of the system and we used this to verify the simulated study with the theory.</p><p id="p-78">In conclusion, there is a tremendous potential for using early warning signals to provide evidence on our progress towards elimination and inform public health policies. We have indicated that by monitoring simple statistics over time it is possible to observe disease emergence and elimination, which with further development offers a promising solution for an automated system that can update time series statistics in real-time as new data becomes available. This would be particularly useful for emerging diseases where EWS could be used to prompt early detection and help aid rapid responses. The focus of our paper has provided insight on how statistics behave for different types of infectious disease data, where we considered suitable data which could be incorporated into such monitoring system. We have researched the resemblance of observed time series results between different data types, a necessary exploration for the development of EWS before they can impact decision making. We reported that some indicators traits are inconsistent across all data types and some EWS differ significantly between disease thresholds: elimination and emergence. Knowledge of the type of data which has been collected is imperative to avoid misleading judgements in response to time series trends. Our work has provided analytical evidence to understand why results differ, improving our ability to monitor EWS for infectious disease transitions.</p></div><div class="section" id="sec-20"><h2 class="">Supporting information</h2><p id="p-79"><strong>S1 AppendixAnalytical Derivations</strong></p><p id="p-80"><strong>S1 Fig.Slower rate reduces discrepancies between theory and simulated variance in “aproximated” RoI.</strong></p><p id="p-81"><strong>S2 Fig.Sensitivity to time aggregation (</strong>Δ<em>t</em><strong>) when approximating non-homogeneous Poisson process</strong></p><p id="p-82"><strong>S3 Fig.Comparing predictions to simulations for variance.</strong></p><p id="p-83"><strong>S4 Fig.Comparing predictions to simulations for coefficient of variation. S5 Fig.Comparing predictions to simulations for skewness.</strong></p><p id="p-84"><strong>S6 Fig.Comparing predictions to simulations for kurtosis.</strong></p><p id="p-85"><strong>S7 Fig.Comparing predictions to simulations for autocorrelation lag 1.</strong></p><p id="p-86"><strong>S8 Fig.Evaluating the MSE between theoretical prediction and simulations for different window sizes.</strong></p><p id="p-87"><strong>S9 Fig.ROC curves for indicators V, CV and AC(1).</strong></p><p id="p-88"><strong>S10 Fig.ROC curves for indicators Skewness and Kurtosis.</strong></p><p id="p-89"><strong>S11 Fig.AUC scores for different EWS (Model 2).</strong></p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.1" data-doi="10.1073/pnas.0802430105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dakos</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scheffer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Nes</span>  <span class="cit-name-given-names">EH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brovkin</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petoukhov</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Held</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">Slowing down as an early warning signal for abrupt climate change</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>. <span class="cit-pub-date">2008</span> <span class="cit-month">Sep</span> 23;<span class="cit-vol">105</span>(<span class="cit-issue">38</span>):<span class="cit-fpage">14308</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0802430105%26rft_id%253Dinfo%253Apmid%252F18787119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzM4LzE0MzA4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDIvMjAyMC4wNC4wMi4wMjE1NzYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.2" data-doi="10.1038/nature08227"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Scheffer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bascompte</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brock</span>  <span class="cit-name-given-names">WA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brovkin</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dakos</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Held</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Nes</span>  <span class="cit-name-given-names">EH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rietkerk</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sugihara</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Early-warning signals for critical transitions</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-pub-date">2009</span> <span class="cit-month">Sep</span>;<span class="cit-vol">461</span>(<span class="cit-issue">7260</span>):<span class="cit-fpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DScheffer%26rft.auinit1%253DM.%26rft.volume%253D461%26rft.issue%253D7260%26rft.spage%253D53%26rft.epage%253D59%26rft.atitle%253DEarly-warning%2Bsignals%2Bfor%2Bcritical%2Btransitions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08227%26rft_id%253Dinfo%253Apmid%252F19727193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08227&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19727193&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.04.02.021576.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269478800026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brett</span>  <span class="cit-name-given-names">TS</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Dea</span>  <span class="cit-name-given-names">EB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marty</span>  <span class="cit-name-given-names">É</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">PB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">AW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rohani</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Anticipating epidemic transitions with imperfect data</span>. <abbr class="cit-jnl-abbrev">PLoS computational biology</abbr>. <span class="cit-pub-date">2018</span> <span class="cit-month">Jun</span> 8;<span class="cit-vol">14</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">e1006204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bcomputational%2Bbiology%26rft.volume%253D14%26rft.spage%253D1006204e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">O’Dea</span>  <span class="cit-name-given-names">EB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">JM</span></span>. <span class="cit-article-title">Disentangling reporting and disease transmission</span>. <abbr class="cit-jnl-abbrev">Theoretical Ecology</abbr>. <span class="cit-pub-date">2019</span> <span class="cit-month">Mar</span> 1;<span class="cit-vol">12</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">89</span>–<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2BEcology%26rft.volume%253D12%26rft.spage%253D89%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">O’Regan</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">JM</span></span>. <span class="cit-article-title">Theory of early warning signals of disease emergenceand leading indicators of elimination</span>. <abbr class="cit-jnl-abbrev">Theoretical Ecology</abbr>. <span class="cit-pub-date">2013</span> <span class="cit-month">Aug</span> 1;<span class="cit-vol">6</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">333</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2BEcology%26rft.volume%253D6%26rft.spage%253D333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">PB</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Dea</span>  <span class="cit-name-given-names">EB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rohani</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">JM</span></span>. <span class="cit-article-title">Forecasting infectious disease emergence subject to seasonal forcing</span>. <abbr class="cit-jnl-abbrev">Theoretical Biology and Medical Modelling</abbr>. <span class="cit-pub-date">2017</span> <span class="cit-month">Dec</span>;<span class="cit-vol">14</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2BBiology%2Band%2BMedical%2BModelling%26rft.volume%253D14%26rft.spage%253D17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.021576v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">O’Regan</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lillie</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">JM</span></span>. <span class="cit-article-title">Leading indicators of mosquito-borne disease elimination</span>. <abbr class="cit-jnl-abbrev">Theoretical ecology</abbr>. <span class="cit-pub-date">2016</span> <span class="cit-month">Sep</span> 1;<span class="cit-vol">9</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">269</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Becology%26rft.volume%253D9%26rft.spage%253D269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dessavre</span>  <span class="cit-name-given-names">AG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Southall</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tildesley</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dyson</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">The problem of detrending when analysing potential indicators of disease elimination</span>. <abbr class="cit-jnl-abbrev">Journal of theoretical biology</abbr>. <span class="cit-pub-date">2019</span> <span class="cit-month">Apr</span> 11.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.9" data-doi="10.1111/j.1461-0248.2005.00877.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brock</span>  <span class="cit-name-given-names">WA</span></span>. <span class="cit-article-title">Rising variance: a leading indicator of ecological transition</span>. <abbr class="cit-jnl-abbrev">Ecology letters</abbr>. <span class="cit-pub-date">2006</span> <span class="cit-month">Mar</span>;<span class="cit-vol">9</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">311</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DCarpenter%26rft.auinit1%253DS.%2BR.%26rft.volume%253D9%26rft.issue%253D3%26rft.spage%253D311%26rft.epage%253D318%26rft.atitle%253DRising%2Bvariance%253A%2Ba%2Bleading%2Bindicator%2Bof%2Becological%2Btransition.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2005.00877.x%26rft_id%253Dinfo%253Apmid%252F16958897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2005.00877.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16958897&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.04.02.021576.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.021576v1.10" data-doi="10.1126/science.1203672"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cole</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pace</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Batt</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brock</span>  <span class="cit-name-given-names">WA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cline</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coloso</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hodgson</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kitchell</span>  <span class="cit-name-given-names">JF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seekell</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Early warnings of regime shifts: a whole-ecosystem experiment</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2011</span> <span class="cit-month">May</span> 27;<span class="cit-vol">332</span>(<span class="cit-issue">6033</span>):<span class="cit-fpage">1079</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1203672%26rft_id%253Dinfo%253Apmid%252F21527677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzIvNjAzMy8xMDc5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDIvMjAyMC4wNC4wMi4wMjE1NzYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.021576v1.11" data-doi="10.1098/rsta.2011.0304"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lenton</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Livina</span>  <span class="cit-name-given-names">VN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dakos</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Nes</span>  <span class="cit-name-given-names">EH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scheffer</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Early warning of climate tipping points from critical slowing down: comparing methods to improve robustness</span>. <abbr class="cit-jnl-abbrev">Philosophical Transactions of the Royal Society A: Mathematical, Physical and Engineering Sciences</abbr>. <span class="cit-pub-date">2012</span> <span class="cit-month">Mar</span> 13;<span class="cit-vol">370</span>(<span class="cit-issue">1962</span>):<span class="cit-fpage">1185</span>–<span class="cit-lpage">204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BA%253A%2BMathematical%252C%2BPhysical%2Band%2BEngineering%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsta.2011.0304%26rft_id%253Dinfo%253Apmid%252F22291229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsta.2011.0304&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2014086333&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=22291229&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.04.02.021576.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Clements</span>  <span class="cit-name-given-names">CF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ozgul</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Including trait-based early warning signals helps predict population collapse</span>. <abbr class="cit-jnl-abbrev">Nature communications</abbr>. <span class="cit-pub-date">2016</span> <span class="cit-month">Mar</span> 24;<span class="cit-vol">7</span>:<span class="cit-fpage">10984</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D7%26rft.spage%253D10984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.13" data-doi="10.1038/nature09389"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Griffen</span>  <span class="cit-name-given-names">BD</span></span>. <span class="cit-article-title">Early warning signals of extinction in deteriorating environments</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-pub-date">2010</span> <span class="cit-month">Sep</span>;<span class="cit-vol">467</span>(<span class="cit-issue">7314</span>):<span class="cit-fpage">456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDrake%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D467%26rft.issue%253D7314%26rft.spage%253D456%26rft.epage%253D459%26rft.atitle%253DEarly%2Bwarning%2Bsignals%2Bof%2Bextinction%2Bin%2Bdeteriorating%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09389%26rft_id%253Dinfo%253Apmid%252F20827269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20827269&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.04.02.021576.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282090200042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.021576v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hefley</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tyre</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blankenship</span>  <span class="cit-name-given-names">EE</span></span>. <span class="cit-article-title">Statistical indicators and state–space population models predict extinction in a population of bobwhite quail</span>. <abbr class="cit-jnl-abbrev">Theoretical Ecology</abbr>. <span class="cit-pub-date">2013</span> <span class="cit-month">Aug</span> 1;<span class="cit-vol">6</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">319</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2BEcology%26rft.volume%253D6%26rft.spage%253D319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.15" data-doi="10.1371/journal.pone.0041010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dakos</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brock</span>  <span class="cit-name-given-names">WA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ellison</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guttal</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ives</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kefi</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Livina</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seekell</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Nes</span>  <span class="cit-name-given-names">EH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scheffer</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Methods for detecting early warnings of critical transitions in time series illustrated using simulated ecological data</span>. <abbr class="cit-jnl-abbrev">PloS one</abbr>. <span class="cit-pub-date">2012</span> <span class="cit-month">Jul</span> 17;<span class="cit-vol">7</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">e41010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDakos%26rft.auinit1%253DV.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De41010%26rft.epage%253De41010%26rft.atitle%253DMethods%2Bfor%2Bdetecting%2Bearly%2Bwarnings%2Bof%2Bcritical%2Btransitions%2Bin%2Btime%2Bseries%2Billustrated%2Busing%2Bsimulated%2Becological%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0041010%26rft_id%253Dinfo%253Apmid%252F22815897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0041010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22815897&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.04.02.021576.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.16" data-doi="10.1890/11-0889.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dakos</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Nes</span>  <span class="cit-name-given-names">EH</span></span>, <span class="cit-auth"><span class="cit-name-surname">D’Odorico</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scheffer</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Robustness of variance and autocorrelation as indicators of critical slowing down</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr>. <span class="cit-pub-date">2012</span> <span class="cit-month">Feb</span>;<span class="cit-vol">93</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">264</span>–<span class="cit-lpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DDakos%26rft.auinit1%253DV.%26rft.volume%253D93%26rft.issue%253D2%26rft.spage%253D264%26rft.epage%253D271%26rft.atitle%253DRobustness%2Bof%2Bvariance%2Band%2Bautocorrelation%2Bas%2Bindicators%2Bof%2Bcritical%2Bslowing%2Bdown.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F11-0889.1%26rft_id%253Dinfo%253Apmid%252F22624308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/11-0889.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22624308&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.04.02.021576.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302794100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Dea</span>  <span class="cit-name-given-names">EB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Epureanu</span>  <span class="cit-name-given-names">BI</span></span>. <span class="cit-article-title">Eigenvalues of the covariance matrix as early warning signals for critical transitions in ecological systems</span>. <abbr class="cit-jnl-abbrev">Scientific reports</abbr>. <span class="cit-pub-date">2019</span> <span class="cit-month">Feb</span> 22;<span class="cit-vol">9</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">2572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2Breports%26rft.volume%253D9%26rft.spage%253D2572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.18" data-doi="10.1371/journal.pone.0092097"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kefi</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guttal</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brock</span>  <span class="cit-name-given-names">WA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ellison</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Livina</span>  <span class="cit-name-given-names">VN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seekell</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scheffer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Nes</span>  <span class="cit-name-given-names">EH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dakos</span>  <span class="cit-name-given-names">V.</span></span> <span class="cit-article-title">Early warning signals of ecological transitions: methods for spatial patterns</span>. <abbr class="cit-jnl-abbrev">PloS one</abbr>. <span class="cit-pub-date">2014</span> <span class="cit-month">Mar</span> 21;<span class="cit-vol">9</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">e92097</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPloS%2Bone%26rft.volume%253D9%26rft.spage%253De92097%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0092097%26rft_id%253Dinfo%253Apmid%252F24658137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0092097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24658137&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F02%2F2020.04.02.021576.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Scheffer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">SR</span></span>. <span class="cit-article-title">Catastrophic regime shifts in ecosystems: linking theory to observation</span>. <abbr class="cit-jnl-abbrev">Trends in ecology &amp; evolution</abbr>. <span class="cit-pub-date">2003</span> <span class="cit-month">Dec</span> 1;<span class="cit-vol">18</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">648</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Becology%2B%2526%2Bevolution%26rft.volume%253D18%26rft.spage%253D648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.20" data-doi="10.1086/681573"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Clements</span>  <span class="cit-name-given-names">CF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">JI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ozgul</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Factors influencing the detectability of early warning signals of population collapse</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>. <span class="cit-pub-date">2015</span> <span class="cit-month">May</span> 7;<span class="cit-vol">186</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">50</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D186%26rft.spage%253D50%26rft_id%253Dinfo%253Adoi%252F10.1086%252F681573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/681573&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">21.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.02.021576v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wen</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ciamarra</span>,  <span class="cit-name-given-names">M. P.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Cheong</span>,  <span class="cit-name-given-names">S. A.</span></span> <span class="cit-article-title">How one might miss early warning signals of critical transitions in time series data: A systematic study of two major currency pairs</span>. <abbr class="cit-jnl-abbrev">PloS one</abbr>. <span class="cit-pub-date">2018</span> <span class="cit-month">Mar</span> 14;<span class="cit-vol">13</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">e0191439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPloS%2Bone%26rft.volume%253D13%26rft.spage%253De0191439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Diks</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hommes</span>,  <span class="cit-name-given-names">C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Critical slowing down as an early warning signal for financial crises?</span>. <abbr class="cit-jnl-abbrev">Empirical Economics</abbr>. <span class="cit-pub-date">2019</span> <span class="cit-month">Oct</span>; <span class="cit-vol">57</span>(<span class="cit-issue">4</span>), pp.<span class="cit-fpage">1201</span>–<span class="cit-lpage">1228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmpirical%2BEconomics%26rft.volume%253D57%26rft.spage%253D1201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.23" data-doi="10.1126/science.1203672"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>,  <span class="cit-name-given-names">S.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cole</span>,  <span class="cit-name-given-names">J.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pace</span>,  <span class="cit-name-given-names">M.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Batt</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brock</span>,  <span class="cit-name-given-names">W.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cline</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coloso</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hodgson</span>,  <span class="cit-name-given-names">J.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kitchell</span>,  <span class="cit-name-given-names">J.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seekell</span>,  <span class="cit-name-given-names">D.A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Early warnings of regime shifts: a whole-ecosystem experiment</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2011</span>; <span class="cit-vol">332</span>(<span class="cit-issue">6033</span>), pp.<span class="cit-fpage">1079</span>–<span class="cit-lpage">1082</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1203672%26rft_id%253Dinfo%253Apmid%252F21527677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzIvNjAzMy8xMDc5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDIvMjAyMC4wNC4wMi4wMjE1NzYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-2020.04.02.021576v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keeling</span>,  <span class="cit-name-given-names">M.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Rohani</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-source">M Modeling infectious diseases in humans and animals</span>. <span class="cit-publ-name">Princeton University Press</span>. <span class="cit-pub-date">2011</span> <span class="cit-month">Sep</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-2020.04.02.021576v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kendall</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-source">A New Measure of Rank Correlation</span>. <span class="cit-publ-name">Oxford University Press</span>. <span class="cit-pub-date">1938</span> <span class="cit-month">Jun</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nájera</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">González-Silva</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Alonso</span>,  <span class="cit-name-given-names">P.L.</span></span> <span class="cit-article-title">Some lessons for the future from the Global Malaria Eradication Programme (1955–1969)</span>. <abbr class="cit-jnl-abbrev">PLoS medicine</abbr>, <span class="cit-vol">8</span>(<span class="cit-issue">1</span>). <span class="cit-pub-date">2011</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cohen</span>,  <span class="cit-name-given-names">J.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">D.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cotter</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ward</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamey</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sabot</span>,  <span class="cit-name-given-names">O.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Moonen</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Malaria resurgence: a systematic review and assessment of its causes</span>. <abbr class="cit-jnl-abbrev">Malaria journal</abbr>, <span class="cit-vol">11</span>(<span class="cit-issue">1</span>), p.<span class="cit-fpage">122</span>. <span class="cit-pub-date">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMalaria%2Bjournal%26rft.volume%253D11%26rft.spage%253D122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-2020.04.02.021576v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van Kampen</span>  <span class="cit-name-given-names">NG</span></span>. <span class="cit-source">Stochastic processes in physics and chemistry</span>. <span class="cit-publ-name">Elsevier</span>; <span class="cit-pub-date">1992</span> <span class="cit-month">Nov</span> 20.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guttal</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raghavendra</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goel</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoarau</span>  <span class="cit-name-given-names">Q.</span></span> <span class="cit-article-title">Lack of critical slowing down suggests that financial meltdowns are not critical transitions, yet rising variability could signal systemic risk</span>. <abbr class="cit-jnl-abbrev">PloS one</abbr>, <span class="cit-vol">11</span>(<span class="cit-issue">1</span>). <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kéfi</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dakos</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scheffer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Nes</span>  <span class="cit-name-given-names">EH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rietkerk</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Early warning signals also precede non-catastrophic transitions</span>. <abbr class="cit-jnl-abbrev">Oikos</abbr>. <span class="cit-vol">122</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">641</span>–<span class="cit-lpage">8</span>. <span class="cit-pub-date">2013</span> <span class="cit-month">May</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOikos%26rft.volume%253D122%26rft.spage%253D641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.02.021576v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holloway</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Recommendations of the international task force for disease eradication</span>. <abbr class="cit-jnl-abbrev">Morbidity and mortality weekly report</abbr>. <span class="cit-vol">42</span>(<span class="cit-issue">16</span>):<span class="cit-fpage">1</span>–<span class="cit-lpage">38</span>. <span class="cit-pub-date">1993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMorbidity%2Band%2Bmortality%2Bweekly%2Breport%26rft.volume%253D42%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.04.02.021576v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.04.02.021576v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.04.02.021592v1" title="Expression of a transcript encoding for cytosolic renin in the diseased human heart" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.04.02.021675v1" title="C. elegans CLASP/CLS-2 negatively regulates membrane ingression throughout the oocyte cortex and is required for polar body extrusion" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 02, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.04.02.021576v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.04.02.021576v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.04.02.021576v1" class="active" data-icon-position="" data-hide-link-title="0">Prospects for detecting early warning signals in discrete event sequence data: application to epidemiological incidence data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1224277" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Yh8WigM3h7bkygA8hv3X3GAUgxQj5aiRS-vg9ex8Mno" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1224277" id="node1224277--2" data-pisa="biorxiv;2020.04.02.021576v1" data-pisa-master="biorxiv;2020.04.02.021576" data-seqnum="1224277" data-apath="/biorxiv/early/2020/04/02/2020.04.02.021576.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Prospects for detecting early warning signals in discrete event sequence data: application to epidemiological incidence data</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Emma</span> <span class="nlm-surname">Southall</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Michael J.</span> <span class="nlm-surname">Tildesley</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Louise</span> <span class="nlm-surname">Dyson</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.02.021576; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.02.021576 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.04.02.021576v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.021576v1&amp;title=Prospects%20for%20detecting%20early%20warning%20signals%20in%20discrete%20event%20sequence%20data%3A%20application%20to%20epidemiological%20incidence%20data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.021576v1&amp;title=Prospects%20for%20detecting%20early%20warning%20signals%20in%20discrete%20event%20sequence%20data%3A%20application%20to%20epidemiological%20incidence%20data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.021576v1&amp;text=Prospects%20for%20detecting%20early%20warning%20signals%20in%20discrete%20event%20sequence%20data%3A%20application%20to%20epidemiological%20incidence%20data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.021576v1&amp;title=Prospects%20for%20detecting%20early%20warning%20signals%20in%20discrete%20event%20sequence%20data%3A%20application%20to%20epidemiological%20incidence%20data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.021576v1&amp;t=Prospects%20for%20detecting%20early%20warning%20signals%20in%20discrete%20event%20sequence%20data%3A%20application%20to%20epidemiological%20incidence%20data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.021576v1&amp;title=Prospects%20for%20detecting%20early%20warning%20signals%20in%20discrete%20event%20sequence%20data%3A%20application%20to%20epidemiological%20incidence%20data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.021576v1&amp;title=Prospects%20for%20detecting%20early%20warning%20signals%20in%20discrete%20event%20sequence%20data%3A%20application%20to%20epidemiological%20incidence%20data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1224277" id="node1224277--3" data-pisa="biorxiv;2020.04.02.021576v1" data-pisa-master="biorxiv;2020.04.02.021576" data-seqnum="1224277" data-apath="/biorxiv/early/2020/04/02/2020.04.02.021576.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Prospects for detecting early warning signals in discrete event sequence data: application to epidemiological incidence data</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Emma</span> <span class="nlm-surname">Southall</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Michael J.</span> <span class="nlm-surname">Tildesley</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Louise</span> <span class="nlm-surname">Dyson</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.02.021576; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.02.021576 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Prospects%20for%20detecting%20early%20warning%20signals%20in%20discrete%20event%20sequence%20data%3A%20application%20to%20epidemiological%20incidence%20data&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Southall%2C+Emma&amp;rft.au=Tildesley%2C+Michael+J.&amp;rft.au=Dyson%2C+Louise"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1224277/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1224277/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1224277/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1224277/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1224277/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1224277/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1224277/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1224277/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1224277/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1224277/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1224277/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1224277/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.021576v1&amp;count=horizontal&amp;via=&amp;text=Prospects%20for%20detecting%20early%20warning%20signals%20in%20discrete%20event%20sequence%20data%3A%20application%20to%20epidemiological%20incidence%20data&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.021576v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.04.02.021576v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.04.02.021576v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/ecology" class="highlight" data-icon-position="" data-hide-link-title="0">Ecology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
