<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/07/12/539239/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/07/12/539239.atom" />
<meta name="HW.pisa" content="biorxiv;539239v3" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Viral long-term evolutionary strategies favor stability over proliferation" />
<meta name="DC.Identifier" content="10.1101/539239" />
<meta name="DC.Date" content="2019-07-12" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Viruses are known to have some of the highest and most diverse mutation rates found in any biological replicator, with single-stranded (ss) RNA viruses evolving the fastest, and double-stranded (ds) DNA viruses having rates approaching those of bacteria. As mutation rates are tightly and negatively correlated with genome size, selection is a clear driver of viral evolution. However, the role of intragenomic interactions as drivers of viral evolution is still unclear. To understand how these two processes affect the long-term evolution of viruses infecting humans, we comprehensively analyzed ssRNA, ssDNA, dsRNA, and dsDNA viruses, to find which virus types and which functions show evidence for episodic diversifying selection and correlated evolution. We show that selection mostly affects single stranded viruses, that correlated evolution is more prevalent in DNA viruses, and that both processes, taken independently, mostly affect viral replication. However, the genes that are jointly affected by both processes are involved in key aspects of their life cycle, favoring viral stability over proliferation. We further show that both evolutionary processes are intimately linked at the amino acid level, which suggests that it is the joint action of selection and correlated evolution, and not just selection, that shapes the evolutionary trajectories of viruses – and possibly of their epidemiological potential." />
<meta name="DC.Contributor" content="Stéphane Aris-Brosou" />
<meta name="DC.Contributor" content="Louis Parent" />
<meta name="DC.Contributor" content="Neke Ibeh" />
<meta name="article:published_time" content="2019-07-12" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Viral long-term evolutionary strategies favor stability over proliferation" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Viruses are known to have some of the highest and most diverse mutation rates found in any biological replicator, with single-stranded (ss) RNA viruses evolving the fastest, and double-stranded (ds) DNA viruses having rates approaching those of bacteria. As mutation rates are tightly and negatively correlated with genome size, selection is a clear driver of viral evolution. However, the role of intragenomic interactions as drivers of viral evolution is still unclear. To understand how these two processes affect the long-term evolution of viruses infecting humans, we comprehensively analyzed ssRNA, ssDNA, dsRNA, and dsDNA viruses, to find which virus types and which functions show evidence for episodic diversifying selection and correlated evolution. We show that selection mostly affects single stranded viruses, that correlated evolution is more prevalent in DNA viruses, and that both processes, taken independently, mostly affect viral replication. However, the genes that are jointly affected by both processes are involved in key aspects of their life cycle, favoring viral stability over proliferation. We further show that both evolutionary processes are intimately linked at the amino acid level, which suggests that it is the joint action of selection and correlated evolution, and not just selection, that shapes the evolutionary trajectories of viruses – and possibly of their epidemiological potential.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;539239v3" />
<meta name="citation_id" content="539239v3" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/539239v3" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/539239v3.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/539239v3.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/07/12/539239.full.pdf" />
<meta name="citation_doi" content="10.1101/539239" />
<meta name="citation_num_pages" content="21" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="539239" />
<meta name="citation_author" content="Stéphane Aris-Brosou" />
<meta name="citation_author_institution" content="Department of Biology, University of Ottawa" />
<meta name="citation_author_institution" content="Department of Mathematics and Statistics, University of Ottawa" />
<meta name="citation_author_email" content="sarisbro@uottawa.ca" />
<meta name="citation_author" content="Louis Parent" />
<meta name="citation_author_institution" content="Department of Biology, University of Ottawa" />
<meta name="citation_author" content="Neke Ibeh" />
<meta name="citation_author_institution" content="Department of Biology, University of Ottawa" />
<meta name="citation_reference" content="Abascal, F., R. Zardoya, and M. J. Telford, 2010 TranslatorX: multiple alignment of nucleotide sequences guided by amino acid translations. Nucleic Acids Res 38: W7–13." />
<meta name="citation_reference" content="Alexa, A. and J. Rahnenführer, 2009 Gene set enrichment analysis with topGO. Bioconductor Improv 27." />
<meta name="citation_reference" content="Aris-Brosou, S., N. Ibeh, and J. Noël, 2017 Viral outbreaks involve destabilized evolutionary networks: evidence from Ebola, Influenza and Zika. Sci Rep 7: 11881." />
<meta name="citation_reference" content="Aris-Brosou, S. and N. Rodrigue, 2012 The essentials of computational molecular evolution. Methods Mol Biol 855: 111–52." />
<meta name="citation_reference" content="Ashenberg, O., J. Padmakumar, M. B. Doud, and J. D. Bloom, 2017 Deep mutational scanning identifies sites in influenza nucleoprotein that affect viral inhibition by MxA. PLoS Pathog 13: e1006288." />
<meta name="citation_reference" content="Ben-Nun, M., P. Riley, J. Turtle, D. P. Bacon, and S. Riley, 2019 Forecasting national and regional influenza-like illness for the USA. PLoS Comput Biol 15: e1007013." />
<meta name="citation_reference" content="Castresana, J., 2000 Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Molecular biology and evolution 17: 540–552." />
<meta name="citation_reference" content="Conesa, A., S. Götz, J. M. García-Gómez, J. Terol, M. Talón, et al., 2005 Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research. Bioinformatics 21: 3674–6." />
<meta name="citation_reference" content="De Paepe, M. and F. Taddei, 2006 Viruses’ life history: towards a mechanistic basis of a trade-off between survival and reproduction among phages. PLoS Biol 4: e193." />
<meta name="citation_reference" content="Dench, J., A. Hinz, S. Aris-Brosou, and R. Kassen, 2019 The idiosyncratic drivers of correlated evolution. bioRxiv p. 474536." />
<meta name="citation_reference" content="Duffy, S., 2018 Why are RNA virus mutation rates so damn high? PLoS Biol 16: e3000003." />
<meta name="citation_reference" content="Edgar, R. C., 2004 Muscle: multiple sequence alignment with high accuracy and high throughput. Nucleic acids research 32: 1792–1797." />
<meta name="citation_reference" content="Faria, N. R., R. d. S. d. S. Azevedo, M. U. G. Kraemer, R. Souza, M. S. Cunha, et al., 2016a Zika virus in the americas: Early epidemiological and genetic findings. Science 352: 345–349." />
<meta name="citation_reference" content="Faria, N. R., E. C. Sabino, M. R. T. Nunes, L. C. J. Alcantara, N. J. Loman, et al., 2016b Mobile real-time surveillance of Zika virus in Brazil. Genome Medicine 8." />
<meta name="citation_reference" content="Frederico, L. A., T. A. Kunkel, and B. R. Shaw, 1990 A sensitive genetic assay for the detection of cytosine deamination: determination of rate constants and the activation energy. Biochemistry 29: 2532–7." />
<meta name="citation_reference" content="Gao, Y., H. Zhao, Y. Jin, X. Xu, and G.-Z. Han, 2017 Extent and evolution of gene duplication in DNA viruses. Virus research 240: 161–165." />
<meta name="citation_reference" content="García-Villada, L. and J. W. Drake, 2013 Experimental selection reveals a trade-off between fecundity and lifespan in the coliphage Qß. Open Biol 3: 130043." />
<meta name="citation_reference" content="Gilbert, D., 2003 Sequence file format conversion with command-line readseq. Curr Protoc Bioinformatics Appendix 1: Appendix 1E." />
<meta name="citation_reference" content="Gire, S. K., A. Goba, K. G. Andersen, R. S. G. Sealfon, D. J. Park, et al., 2014 Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak. Science 345: 1369–72." />
<meta name="citation_reference" content="Gong, L. I., M. A. Suchard, and J. D. Bloom, 2013 Stability-mediated epistasis constrains the evolution of an influenza protein. Elife 2: e00631." />
<meta name="citation_reference" content="Grenfell, B. T., O. G. Pybus, J. R. Gog, J. L. N. Wood, J. M. Daly, et al., 2004 Unifying the epidemiological and evolutionary dynamics of pathogens. Science 303: 327–32." />
<meta name="citation_reference" content="Holmes, E. C., 2009 The evolution and emergence of RNA viruses. Oxford series in ecology and evolution, Oxford University Press, Oxford." />
<meta name="citation_reference" content="Holmes, E. C., 2011 What does virus evolution tell us about virus origins? J Virol 85: 5247–51." />
<meta name="citation_reference" content="Ibeh, N., J. C. Nshogozabahizi, and S. Aris-Brosou, 2016 Both epistasis and diversifying selection drive the structural evolution of the Ebola virus glycoprotein mucin-like domain. J Virol 90: 5475–5484." />
<meta name="citation_reference" content="Kosakovsky Pond, S. L. and S. D. W. Frost, 2005 Not so different after all: a comparison of methods for detecting amino acid sites under selection. Mol Biol Evol 22: 1208–22." />
<meta name="citation_reference" content="Kryazhimskiy, S., J. Dushoff, G. A. Bazykin, and J. B. Plotkin, 2011 Prevalence of epistasis in the evolution of influenza A surface proteins. PLoS Genet 7: e1001301." />
<meta name="citation_reference" content="Li, C., W. Qian, C. J. Maclean, and J. Zhang, 2016 The fitness landscape of a tRNA gene. Science 352: 837–40." />
<meta name="citation_reference" content="Lynch, M., 2007 The origins of genome architecture. Sinauer Associates, Sunderland, Mass." />
<meta name="citation_reference" content="Lyons, D. M. and A. S. Lauring, 2018 Mutation and epistasis in influenza virus evolution. Viruses 10." />
<meta name="citation_reference" content="Maddison, W. P. and R. G. FitzJohn, 2015 The unsolved challenge to phylogenetic correlation tests for categorical characters. Syst Biol 64: 127–36." />
<meta name="citation_reference" content="Meer, M. V., A. S. Kondrashov, Y. Artzy-Randrup, and F. A. Kondrashov, 2010 Compensatory evolution in mitochondrial tRNAs navigates valleys of low fitness. Nature 464: 279–82." />
<meta name="citation_reference" content="Murrell, B., J. O. Wertheim, S. Moola, T. Weighill, K. Scheffler, et al., 2012 Detecting individual sites subject to episodic diversifying selection. PLoS Genet 8: e1002764." />
<meta name="citation_reference" content="Neverov, A. D., S. Kryazhimskiy, J. B. Plotkin, and G. A. Bazykin, 2015 Coordinated evolution of influenza A surface proteins. PLoS Genet 11: e1005404." />
<meta name="citation_reference" content="Nshogozabahizi, J. C., J. Dench, and S. Aris-Brosou, 2017 Widespread historical contingency in influenza viruses. Genetics 205: 409–420." />
<meta name="citation_reference" content="Pedruzzi, G., A. Barlukova, and I. M. Rouzine, 2018 Evolutionary footprint of epistasis. PLoS Comput Biol 14: e1006426." />
<meta name="citation_reference" content="Pond, S. L. K., S. D. W. Frost, and S. V. Muse, 2005 HyPhy: hypothesis testing using phylogenies. Bioinformatics 21: 676–9." />
<meta name="citation_reference" content="Poon, A. F. Y., F. I. Lewis, S. D. W. Frost, and S. L. Kosakovsky Pond, 2008 Spidermonkey: rapid detection of co-evolving sites using Bayesian graphical models. Bioinformatics 24: 1949–50." />
<meta name="citation_reference" content="Price, M. N., P. S Dehal, and A. P. Arkin, 2010 FastTree 2–approximately maximum- likelihood trees for large alignments. PLoS One 5: e9490." />
<meta name="citation_reference" content="R Core Team, 2016 R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria." />
<meta name="citation_reference" content="Revell, L. J., 2012 phytools: an R package for phylogenetic comparative biology (and other things). Methods in Ecology and Evolution 3: 217–223." />
<meta name="citation_reference" content="Robinson, C. M., D. Seto, M. S. Jones, D. W. Dyer, and J. Chodosh, 2011 Molecular evolution of human species D adenoviruses. Infect Genet Evol 11: 1208–17." />
<meta name="citation_reference" content="Sandie, R. and S. Aris-Brosou, 2014 Predicting the emergence of H3N2 influenza viruses reveals contrasted modes of evolution of HA and NA antigens. J Mol Evol 78: 1–12." />
<meta name="citation_reference" content="Sanjuán, R., 2012 From molecular genetics to phylodynamics: evolutionary relevance of mutation rates across viruses. PLoS Pathog 8: e1002685." />
<meta name="citation_reference" content="Simon-Loriere, E. and E. C. Holmes, 2011 Why do RNA viruses recombine? Nat Rev Microbiol 9: 617–26." />
<meta name="citation_reference" content="Smith, G. J. D., D. Vijaykrishna, J. Bahl, S. J. Lycett, M. Worobey, et al., 2009 Origins and evolutionary genomics of the 2009 swine-origin H1N1 influenza a epidemic. Nature 459: 1122–5." />
<meta name="citation_reference" content="Stano, M., G. Beke, and L. Klucar, 2016 viruSITE-integrated database for viral genomics. Database (Oxford) 2016." />
<meta name="citation_reference" content="van Boheemen, S., M. de Graaf, C. Lauber, T. M. Bestebroer, V. S. Raj, et al., 2012 Genomic characterization of a newly discovered coronavirus associated with acute respiratory distress syndrome in humans. MBio 3." />
<meta name="citation_reference" content="Weinreich, D. M., N. F. Delaney, M. A. Depristo, and D. L. Hartl, 2006 Darwinian evolution can follow only very few mutational paths to fitter proteins. Science 312: 111–4." />
<meta name="citation_reference" content="Woo, J., D. L. Robertson, and S. C. Lovell, 2014 Constraints from protein structure and intra-molecular coevolution influence the fitness of HIV-1 recombinants. Virology 454-455: 34–9." />
<meta name="citation_reference" content="Yohai, V. J., W. A. Stahel, and R. H. Zamar, 1991 A procedure for robust estimation and inference in linear regression. In Directions in robust statistics and diagnostics, pp. 365–374, Springer." />
<meta name="citation_reference" content="Zaki, A. M., S. van Boheemen, T. M. Bestebroer, A. D. M. E. Osterhaus, and R. A. M. Fouchier, 2012 Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia. N Engl J Med 367: 1814–20." />
<meta name="twitter:title" content="Viral long-term evolutionary strategies favor stability over proliferation" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Viruses are known to have some of the highest and most diverse mutation rates found in any biological replicator, with single-stranded (ss) RNA viruses evolving the fastest, and double-stranded (ds) DNA viruses having rates approaching those of bacteria. As mutation rates are tightly and negatively correlated with genome size, selection is a clear driver of viral evolution. However, the role of intragenomic interactions as drivers of viral evolution is still unclear. To understand how these two processes affect the long-term evolution of viruses infecting humans, we comprehensively analyzed ssRNA, ssDNA, dsRNA, and dsDNA viruses, to find which virus types and which functions show evidence for episodic diversifying selection and correlated evolution. We show that selection mostly affects single stranded viruses, that correlated evolution is more prevalent in DNA viruses, and that both processes, taken independently, mostly affect viral replication. However, the genes that are jointly affected by both processes are involved in key aspects of their life cycle, favoring viral stability over proliferation. We further show that both evolutionary processes are intimately linked at the amino acid level, which suggests that it is the joint action of selection and correlated evolution, and not just selection, that shapes the evolutionary trajectories of viruses – and possibly of their epidemiological potential." />
<meta name="og-title" property="og:title" content="Viral long-term evolutionary strategies favor stability over proliferation" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/539239v3" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Viruses are known to have some of the highest and most diverse mutation rates found in any biological replicator, with single-stranded (ss) RNA viruses evolving the fastest, and double-stranded (ds) DNA viruses having rates approaching those of bacteria. As mutation rates are tightly and negatively correlated with genome size, selection is a clear driver of viral evolution. However, the role of intragenomic interactions as drivers of viral evolution is still unclear. To understand how these two processes affect the long-term evolution of viruses infecting humans, we comprehensively analyzed ssRNA, ssDNA, dsRNA, and dsDNA viruses, to find which virus types and which functions show evidence for episodic diversifying selection and correlated evolution. We show that selection mostly affects single stranded viruses, that correlated evolution is more prevalent in DNA viruses, and that both processes, taken independently, mostly affect viral replication. However, the genes that are jointly affected by both processes are involved in key aspects of their life cycle, favoring viral stability over proliferation. We further show that both evolutionary processes are intimately linked at the amino acid level, which suggests that it is the joint action of selection and correlated evolution, and not just selection, that shapes the evolutionary trajectories of viruses – and possibly of their epidemiological potential." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-07-12" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/539239v3.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/539239v3.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/539239v3.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/539239v3" />
<link rel="shortlink" href="https://www.biorxiv.org/node/814925" />
    <title>Viral long-term evolutionary strategies favor stability over proliferation | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"gEGYU5nJlQPfcrhravNXUM1Lps7CtzK_d7MnvwOiWo4","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"814925","apath":"\/biorxiv\/early\/2019\/07\/12\/539239.atom","pisa":"biorxiv;539239v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;539239v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/539239v3","title":"Viral long-term evolutionary strategies favor stability over proliferation","identifier":"node\/814925"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002298rl9Jtl7PcmlmMBCAZow80IaEkyyTcgbD9ig1uoOhRO8fGmBhoy\\\\\\\/G\\\\\\\/utus2fo85Z4WbyuMFi7828lja5DZBgXRH1+bNt+O01Utn8B4DOYu9RalJ4Al+lovPKiSY+ZiMOTeOjEHr4BrUOk23SX\\\\\\\/Pm0SHGTltps5ecrmG9nbCY4xtOJOiWcxgKe3E9emVsfBjKvZ2EPmJ5I5eV8tqn1GqpFLnOMLAtoqv9XRpQKLHqbbSfgDWB2bOSOin\\\\\\\/Ll\\\\\\\/xrDKx1WgADzropZxWTDDBqkixTUQcPhFgG1e\\\\\\\/BMlnPnOmvuOPO7pN8DkCQECoO6KmzskdbwiTZS4E6\\\\\\\/+LNaEYxkSH09G8W0yMYzrEeH89MS3z57z81J3hnIPny5aercMT91uRkxBAghPU6zz2Riiy2D0BUZTX1kgnQqmPLXs\\\\\\\/06c6soHkugKc29efN4gec8NDuhRaTTJj0n1wjCITer0ro+pYDkEw6WgRH6p+qTHnP6ViIVMuXGoJNoELf+w\\\\\\\/Bpe7YiDTeMC8Zj33sQSTvXCJTFQPKj8Afzep+BG59qNnwXgPqpbnZ37\\\\\\\/TsPULeasxCfrZFefD1nGhGLPZwgI0vO3oVe+dEJSlrpQemx4Be2FLvP\\\\\\\/x9tc4vtPPuz8Ds0r0m7hIKLJVp2ayn1aEKd9\\\\\\\/YAOQcsKp0nN3\\\\\\\/VRsdEQ0ghuWaGFPlEvTWbCpQ1\\\\\\\/U3l5WIzM0cRl6GN9+a39fSF7by2xOi6sO5I9cg0ArZWIUiuka191Rg46tWKSRbP8pcuKyUueqeo7SAHVkc8SI+il+4lOLnZ4HHkXO0lDNsiKQmIQ0e2NqwiHou6ChRcj7bG96mJ7NuQDe2WX3OI+S8pgRI3EM4VHspv3y\\\\\\\/hIxHtnuPRbfGIRtula+Ig5iJizazLaaLQTZJBAVhqkQX2+Rlyt\\\\\\\/ND3PFRrTXew43J6762NVj+QmImeyqfWDmFTlK0SrLUrYQsGx8kH891NV8cBzYdvwWQjiFXt4BA1Zd7ZUnV6xn2nsAAil0THPZbYWRqNEq9ALxA+jiPlF6KPIMoBhHSoO0J4l8vFKQx9ADHxpdz0xpOABpX8E7+qHNwiWYpZsnExIQ4jMnSM2X02s2DRVCsGGP5LOFz0PyS7gr0TzP7n2DnL1kQsBq5bSzvi5RU5pAQHY5Arl6wJ19mhPOTdEa5fQOjwD5uqZMYbiT7ZwATdQyIWdHk7bYJTAYtJFeS0\\\\\\\/OXt\\\\\\\/BhBYPxyRJAOuUALYnbXW9u\\\\\\\/6aDlcHS+i4rocbbOofo74efDlfUE\\\\\\\/fn\\\\\\\/Qo+Niw0Qg8cH7AYqo8zLOK4eLg0ZuQtg9P7owPoFGlHrh\\\\\\\/JED5mo30h8Bw=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Jwe9pqY9xuOTEpfKPbE5Yw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022549e6e933985be4883cbde76e506fad50572413ad52d143b47dc195c1dee994e\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/539239v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-814925 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/539239v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-98ReAHHEVVt2s1D2ybeWbslNdIN6tlIfk02h3f9vGF4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="814925" id="node814925" data-pisa="biorxiv;539239v3" data-pisa-master="biorxiv;539239" data-apath="/biorxiv/early/2019/07/12/539239.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Viral long-term evolutionary strategies favor stability over proliferation</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Stéphane</span> <span class="nlm-surname">Aris-Brosou</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Louis</span> <span class="nlm-surname">Parent</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Neke</span> <span class="nlm-surname">Ibeh</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/539239 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node814925" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Stéphane Aris-Brosou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biology, University of Ottawa</span>, Ottawa, ON K1N 6N5, <span class='nlm-country'>Canada</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Mathematics and Statistics, University of Ottawa</span>, Ottawa, ON K1N 6N5, <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=St%C3%A9phane%2BAris-Brosou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aris-Brosou%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASt%25C3%25A9phane%2BAris-Brosou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sarisbro@uottawa.ca" class="" data-icon-position="" data-hide-link-title="0">sarisbro@uottawa.ca</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Louis Parent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biology, University of Ottawa</span>, Ottawa, ON K1N 6N5, <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Louis%2BParent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Parent%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALouis%2BParent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Neke Ibeh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biology, University of Ottawa</span>, Ottawa, ON K1N 6N5, <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Neke%2BIbeh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ibeh%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANeke%2BIbeh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/539239v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:814925" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:814925/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/539239v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:814925" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:814925/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/539239v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:814925" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:814925/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/539239v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:814925" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:814925/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/539239v3.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:814925" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:814925/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/539239v3.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:814925" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:814925/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/539239v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:814925" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:814925/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Viruses are known to have some of the highest and most diverse mutation rates found in any biological replicator, with single-stranded (ss) RNA viruses evolving the fastest, and double-stranded (ds) DNA viruses having rates approaching those of bacteria. As mutation rates are tightly and negatively correlated with genome size, selection is a clear driver of viral evolution. However, the role of intragenomic interactions as drivers of viral evolution is still unclear. To understand how these two processes affect the long-term evolution of viruses infecting humans, we comprehensively analyzed ssRNA, ssDNA, dsRNA, and dsDNA viruses, to find which virus types and which functions show evidence for episodic diversifying selection and correlated evolution. We show that selection mostly affects single stranded viruses, that correlated evolution is more prevalent in DNA viruses, and that both processes, taken independently, mostly affect viral replication. However, the genes that are jointly affected by both processes are involved in key aspects of their life cycle, favoring viral stability over proliferation. We further show that both evolutionary processes are intimately linked at the amino acid level, which suggests that it is the joint action of selection and correlated evolution, and not just selection, that shapes the evolutionary trajectories of viruses – and possibly of their epidemiological potential.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-5">Humanity is regularly reminded of the epidemiological toll of viruses, in part due to recent and ongoing viral outbreaks of influenza <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Smith <em>et al.</em> (2009)</a>, Ebola <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gire <em>et al.</em> (2014)</a>, and Zika <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Faria <em>et al.</em> (2016a)</a>. Thanks to recent technological and analytical developments, it is now possible to elucidate, almost in real time <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Faria <em>et al.</em> (2016b)</a>, their epidemiological dynamics, and unravel their evolutionary dynamics <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Grenfell <em>et al.</em> (2004)</a>. However, while the evolutionary dynamics of RNA viruses are well documented <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Holmes (2009)</a>, those of other viruses are not so well known, in particular in a unifying context including major viral types such as double-stranded (ds) and single-stranded (ss) DNA and RNA viruses.</p><p id="p-6">To date, one of the most salient evolutionary feature shared by all viruses is the existence of a negative correlation between mutation rate and genome size <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Holmes (2009)</a>. This is a critical result as it suggests that selection is driving the evolution of mutation rates, establishing a tradeoff between mutational load and availability of adaptive mutations <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Holmes (2011)</a>. But the processes driving the evolution of different types of viruses are multiple. As already argued, both ssDNA and ssRNA viruses share small genome sizes, high mutation rates, but also large effective population sizes, little to no gene duplication or recombination, and overlapping reading frames <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Holmes (2009)</a>. This last point suggests that a less frequently explored evolutionary process in viral studies, correlated evolution, could be as critical as positive selection. Correlated evolution happens when mutations at two locations in a genome occur one after the other, in a quick succession <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kryazhimskiy <em>et al.</em> (2011)</a>, repeatedly <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Maddison and FitzJohn (2015)</a>. Typical examples include drug resistance mutations that have a fitness cost, and that require a second mutation to compensate for the first one <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gong <em>et al.</em> (2013)</a>, or tRNAs that require a specific base-pairing to maintain their secondary and tertiary structures, so that a mutation in the stem region necessitates a second mutation to restore the correct, functional, structure <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Meer <em>et al.</em> (2010)</a>. This process is of particular interest as correlated evolution can be underlain by epistasis, which occurs when the fitness effects of these two mutations are non-additive <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dench <em>et al.</em> (2019)</a>, as in the two examples above. As such, correlated evolution can help us understand the relationship between genotype and fitness, which is a key determinant of evolutionary trajectories <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li <em>et al.</em> (2016)</a>. To date however, correlated evolution has only sporadically been investigated in viral evolution, and these rare instances only focused on ssRNA viruses. Indeed, recent work uncovered pervasive evidence for correlated evolution in influenza viruses <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Nshogozabahizi <em>et al.</em> (2017)</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lyons and Lauring (2018)</a>, and both the Zika <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Aris-Brosou <em>et al.</em> (2017)</a> and the Ebola viruses <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ibeh <em>et al.</em> (2016)</a>. Intriguingly, in this latter case (Ebola), evidence was found that sites evolving in a correlated manner could also be under positive selection – bearing the question as to how frequently these two processes, correlated evolution and positive selection, occur, possibly jointly, and if this co-occurrence is limited to ssRNA viruses, or can be generalized to all viruses.</p><p id="p-7">To better understand the role of correlated evolution and positive selection in the evolutionary dynamics of viruses infecting humans, we constructed a nearly exhaustive viral data set spanning all dsDNA, dsRNA, ssRNA, and ssDNA viruses deposited in GenBank (as of August 2017), and conducted an extensive survey of correlated evolution and diversifying selection in these viruses, asking more specifically about the prevalence of these two processes in each viral type, independently or jointly, with the specific hypothesis that the genes affected by both processes encode functions that are most critical to each viral life cycle.</p></div><div class="section" id="sec-2"><h2 class="">1 Materials and Methods</h2><div id="sec-3" class="subsection"><h3>1.1 Data retrieval</h3><p id="p-8">Lists of dsDNA, dsRNA, ssRNA, and ssDNA viruses infecting humans were retrieved from the viruSITE database <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Stano <em>et al.</em> (2016)</a> in August 2017 (Tables S1-S4); subtypes / genotypes / clades were treated as independent data sets. Although some of these viruses could be segmented or not, with circular or linear genome, with positive or negative strands, and with or without overlapping reading frames, accounting for these structural features would have led to smaller and smaller data sets, precluding any statistical analysis, so that the data were not split beyond viral type. Each list contained the virus names, the length of their genome, their number of protein-coding genes (CDS’s), and was associated with a reference coding sequence (see <kbd>query sequences.zip</kbd> at <a href="https://github.com/sarisbro">https://github.com/sarisbro</a>). In order to obtain corresponding sequence alignments of orthologous genes, BLASTn searches were performed on a custom database limited to viral genes present in NCBI nucleotide database with blast-2.6.0+. For this, all gbvrl*.seq.gz files were downloaded from <a href="https://ftp.ncbi.nih.gov/genbank">ftp.ncbi.nih.gov/genbank</a> while querying viruSITE, and were concatenated into a single GenBank file, then converted into a FASTA file with <kbd>readseq</kbd> to specifically extract CDS’s <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gilbert (2003)</a>. This was done to avoid retrieving 5’ and 3’ untranslated regions that would cause problems to the downstream codon analyses. BLASTn searches were performed for each viruSITE viral sequence with a stringent E-value threshold of 10<sup>−100</sup>, keeping a maximum of 100 sequences with at least 80% coverage with each query; this ensured that subtype / genotype / clade boundaries were not crossed. As the viruses retrieved from the viruSITE also included viruses that require a vector (<em>e.g.</em>, arboviruses such as the Dengue and Yellow fever viruses), or viruses that circulate in non-human hosts but that can lead to zoonoses (<em>e.g.</em>, the Camel alphacoronavirus leading to MERS van <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Boheemen <em>et al.</em> (2012)</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zaki <em>et al.</em> (2012)</a>), these stringent thresholds also ensured that sequences contained in each alignment mostly came from a single host. Only data sets with at least 20 hits were kept for downstream analyses. Sequences corresponding to each accession number were retrieved from the FASTA file obtained with <kbd>readseq</kbd> <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Gilbert (2003)</a>.</p><p id="p-9">Because this file contained partial sequences, each set of retrieved sequences was first aligned with <kbd>Muscle</kbd> 3.8.31 <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Edgar (2004)</a>. Each alignment was then quality checked ensuring that (i) its length is a multiple of three, (ii) it starts with an ATG and stops with a stop codon. Alignments failing at least one condition were discarded. Within each alignment, mean numbers of indels <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/07/12/539239/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/07/12/539239/embed/inline-graphic-1.gif"/></noscript></span></span> were computed for each sequence, and those containing number of indels <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/07/12/539239/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/07/12/539239/embed/inline-graphic-2.gif"/></noscript></span></span> (standard error of the mean) were eliminated. The remaining nucleotide sequences were then re-aligned with TranslatorX <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abascal <em>et al.</em> (2010)</a>, at the amino acid level, using the <kbd>Muscle</kbd> aligner and their heuristics to determine the correct reading frame. Alignment files were then cleaned-up with <kbd>Gblocks</kbd> 0.91b at the codon level using the stringent default settings <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Castresana (2000)</a>. Both trees and alignments are available from <a href="https://github.com/sarisbro">https://github.com/sarisbro</a>.</p><p id="p-10">To obtain gene annotations, Gene Ontology (GO) terms were retrieved from gene sequences with HMMER2GO (<a href="https://github.com/sestaton/HMMER2GO">github.com/sestaton/HMMER2GO</a>), relying on the Hidden Markov Models in ftp.ebi.ac.uk/pub/databases/Pfam/current_release/Pfam-A.hmm.gz (ver. Feb 23, 2017). This is equivalent to performing a Blast2GO search <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Conesa <em>et al.</em> (2005)</a>, but without the limitations of proprietary software. Individual mapping files coming from our four viral types (ssRNA, dsRNA, ssDNA, and dsDNA viruses) were then merged to create a custom annotation file, used for GO term enrichment testing with topGO <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alexa and Rahnenführer (2009)</a>, based on Fisher’s exact test. The reference gene list was always the entire set of genes within each viral type.</p></div><div id="sec-4" class="subsection"><h3>1.2 Phylogenetic analyses</h3><p id="p-11">Phylogenetic trees were reconstructed with <kbd>FastTree</kbd> 2.1.7 <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Price <em>et al.</em> (2010)</a> under the GTR+Γ model of evolution <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Aris-Brosou and Rodrigue (2012)</a>; note that <kbd>FastTree</kbd> was recompiled locally to use double-precision arithmetics, as recommended by its authors to estimate very short branch lengths accurately. Those with a nonzero tree length were midpoint rerooted using the phytools package ver. 0.4-60 <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Revell (2012)</a> in R ver. 3.2.3 <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">R Core Team (2016)</a>.</p><p id="p-12">Patterns of correlated evolution among sites were identified with the Bayesian graphical model (BGM) implemented in SpiderMonkey <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Poon <em>et al.</em> (2008)</a>, which is part of HyPhy ver. 2.3.3 <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Pond <em>et al.</em> (2005)</a>. Default HyPhy scripts were slightly modified to read in codon data, which were used to reconstruct mutational paths under the MG94×HKY85 substitution model <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kosakovsky Pond and Frost (2005)</a> at nonsynonymous sites along each branch of the estimated trees. Ambiguous reconstructions were resolved by considering all possible resolutions and averaging them. These reconstructed mutational paths were then recoded as a binary matrix, with rows corresponding to branches and columns to a site of the alignment. The BGM was then used to identify the pairs of sites that exhibit correlated patterns of nonsynonymous substitutions according to their posterior probability, estimated with a Markov chain Monte Carlo sampler that was run for 10<sup>5</sup> steps, with a burn-in period of 10,000 steps sampling every 1,000 steps for inference <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Aris-Brosou <em>et al.</em> (2017)</a>.</p><p id="p-13">Patterns of episodic selection were identified based on the Mixed Effects Model of Evolution <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Murrell <em>et al.</em> (2012)</a>, also as implemented in HyPhy. The default script from ver. 2.2.6 was used, still with the 2.3.3 HyPhy engine, to infer nonsynonymous to synonymous rate ratios <em>ω</em> assuming that these rates can vary across lineages and among sites. In this implementation, two categories of sites were assumed, those for which <em>ω</em><sub><em>neg</em></sub> ≤ 1, in proportion <em>p</em>, and those for which <em>ω<sub>pos</sub></em> &gt; 1, that are under positive selection, in proportion 1 − <em>p</em>. Evidence for selection was derived by means of a likelihood ratio test between this model, and a null model where <em>ω<sub>pos</sub></em> was constrained to take its value between 0 and 1. Linear models were fitted through robust regressions <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Yohai <em>et al.</em> (1991)</a>. All R scripts and HyPhy source files are available from <a href="https://github.com/sarisbro">https://github.com/sarisbro</a> (file “API_scripts.zip,” in “data”).</p></div></div><div class="section" id="sec-5"><h2 class="">2 Results and Discussion</h2><p id="p-14">First, in order to better understand the genomic characteristics of the four types of viruses known to infect humans, dsDNA (<em>n</em> = 94), dsRNA (<em>n</em> = 15), ssRNA (<em>n</em> = 354), and ssDNA (<em>n</em> = 84), and understand how these characteristics can impact the evolutionary dynamics of these viruses (Methods; Figure S1), we examined the distribution of four of their genomic features. While dsDNA viruses had the largest and the most variable genome lengths, ssDNA were the smallest genomes by at least an order of magnitude, with both dsRNA and ssRNA exhibiting intermediate sizes (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1a</a>). Because of the compact structure of these genomes, these differences were also reflected in the number of genes, protein-coding genes, and RNAs encoded by the genomes of these viruses (Figure S2). In turn, these characteristics suggest that with their large genomes, dsDNA viruses potentially have a higher degree of functional redundancy than ssDNA or even ssRNA viruses, possibly due to duplication events <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gao <em>et al.</em> (2017)</a>, and thereby be under less stringent selective pressures than viruses with smaller genomes. On the other hand, the extent of correlated evolution and its interaction with selection is more difficult to predict.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/12/539239/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Modes of evolution of viruses. Modes of evolution of viruses. (a) Distribution of genome size, on a log10 scale ssRNA (blue), ssDNA (purple), dsRNA (orange), and dsDNA (red). (b) Proportion of genes detected to be under diversifying selection (top, dark hues); actual numbers of genes are shown within each column. (c) Proportion of genes detected to be evolving in a correlated manner (top, dark hues); actual numbers of genes are shown within each column. (d) Venn diagrams showing the Genes of Interest (GOIs) either under selection (MEME GOIs), or evolving in a correlated manner (SM GOIs), or both (intersect) for each virus type. (e) Gene Ontology (GO) enrichment tests for the genes that are both under selection and evolving in a correlated manner for their Molecular Function (top), Biological Processes (middle), and Cellular Component (bottom), all at level 2 of the ontology." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-923146102" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Modes of evolution of viruses. Modes of evolution of viruses. (&lt;strong&gt;a&lt;/strong&gt;) Distribution of genome size, on a log&lt;sub&gt;10&lt;/sub&gt; scale ssRNA (blue), ssDNA (purple), dsRNA (orange), and dsDNA (red). (&lt;strong&gt;b&lt;/strong&gt;) Proportion of genes detected to be under diversifying selection (top, dark hues); actual numbers of genes are shown within each column. (&lt;strong&gt;c&lt;/strong&gt;) Proportion of genes detected to be evolving in a correlated manner (top, dark hues); actual numbers of genes are shown within each column. (&lt;strong&gt;d&lt;/strong&gt;) Venn diagrams showing the Genes of Interest (GOIs) either under selection (MEME GOIs), or evolving in a correlated manner (SM GOIs), or both (intersect) for each virus type. (&lt;strong&gt;e&lt;/strong&gt;) Gene Ontology (GO) enrichment tests for the genes that are both under selection and evolving in a correlated manner for their Molecular Function (top), Biological Processes (middle), and Cellular Component (bottom), all at level 2 of the ontology.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/12/539239/F1.medium.gif" width="440" height="265"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/12/539239/F1.medium.gif" width="440" height="265"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/12/539239/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/12/539239/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-15" class="first-child">Modes of evolution of viruses. Modes of evolution of viruses. (<strong>a</strong>) Distribution of genome size, on a log<sub>10</sub> scale ssRNA (blue), ssDNA (purple), dsRNA (orange), and dsDNA (red). (<strong>b</strong>) Proportion of genes detected to be under diversifying selection (top, dark hues); actual numbers of genes are shown within each column. (<strong>c</strong>) Proportion of genes detected to be evolving in a correlated manner (top, dark hues); actual numbers of genes are shown within each column. (<strong>d</strong>) Venn diagrams showing the Genes of Interest (GOIs) either under selection (MEME GOIs), or evolving in a correlated manner (SM GOIs), or both (intersect) for each virus type. (<strong>e</strong>) Gene Ontology (GO) enrichment tests for the genes that are both under selection and evolving in a correlated manner for their Molecular Function (top), Biological Processes (middle), and Cellular Component (bottom), all at level 2 of the ontology.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">To address these questions, we first tested for the presence of episodic diversifying selection in each viral type. Altogether, we found extensive differences among all four viral types in terms of the number of genes under selection (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1b</a>; <em>X</em><sup>2</sup> = 99.61, <em>df</em> = 3, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>), and that single stranded viruses were more subject to selection than double stranded viruses (<em>X</em><sup>2</sup> = 95.14, <em>df</em> = 1, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>). Indeed, and <em>contra</em> our original hypothesis, there were no differences between dsDNA and dsRNA viruses (<em>X</em><sup>2</sup> = 0.26, <em>df</em> = 1, <em>P</em> = 0.6110), or between ssDNA and ssRNA viruses in terms of prevalence of diversifying selection (<em>X</em><sup>2</sup> = 2.07, <em>df</em> = 1, <em>P</em> = 0.1503). Note that these differences cannot be attributed to genetic diversity, as dsDNA and dsRNA, which have similar levels of selection, have however different levels of diversity (Figure S3). However, it is unlikely that “strandedness” (single <em>vs.</em> double stranded genetic material) alone drives selection, even if greater instability can be postulated in single-stranded nucleic acids <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Frederico <em>et al.</em> (1990)</a>. Indeed, strandedness is negatively correlated with genome size (<em>t</em> = −4.98, <em>df</em> = 108.35, <em>P</em> = 2.43 × 10<sup>−6</sup>), which is itself correlated with mutation rates <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lynch (2007)</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Duffy (2018)</a>. As a result, episodic diversifying selection is mostly driven by structural aspects of viral genomes, which condition mutation rates across all virus types <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Sanjuán (2012)</a>.</p><p id="p-17">To understand if similar aspects drive intragenic correlated evolution, we counted in the same way the number of genes for which we could find evidence for interactions. Here again, we found extensive differences among all four viral types (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1c</a>; <em>X</em><sup>2</sup> = 29.79, <em>df</em> = 3, <em>P</em> &lt; 1.5 × 10<sup>−16</sup>), with no difference between dsDNA and ssDNA viruses (<em>X</em><sup>2</sup> = 0.0005, <em>df</em> = 1, <em>P</em> = 0.9827), or between dsRNA and ssRNA viruses (<em>X</em><sup>2</sup> = 0.0001, <em>df</em> = 3, <em>P</em> = 0.9903). Again, diversity is not driving these differences, as both dsDNA / ssDNA and dsRNA / ssRNA have significantly different levels of diversity (Figure S3). As a result, the prevalence of correlated evolution seems to be mostly driven by the nature of viral genetic material. At least two processes can underpin correlated evolution: linkage and epistasis <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Dench <em>et al.</em> (2019)</a>. As recombination is pervasive in some dsDNA viruses <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Robinson <em>et al.</em> (2011)</a>, linkage alone may not explain the high prevalence of correlated evolution in DNA viruses (close to 40%: <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1c</a>). Rather, this pattern suggests that intragenic constraints are higher in DNA viruses for unknown structural reasons, maybe due to protein structure <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Woo <em>et al.</em> (2014)</a>, or in the same way that recombination in RNA viruses may be driven by mechanistic constraints associated with genome structures and viral life cycles <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Simon-Loriere and Holmes (2011)</a>.</p><p id="p-18">While previous work showed that both diversifying selection and correlated evolution can affect the same gene and even the same site in a viral genome such as Ebola’s <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Ibeh <em>et al.</em> (2016)</a>, a ssRNA virus, the generality of this association is still unknown. At the gene level, we found strong heterogeneity among all four viral types for gene numbers showing evidence for selection, correlated evolution or both (<em>X</em><sup>2</sup> = 206.54, <em>df</em> = 6, <em>P</em> &lt; 2.2×10<sup>−16</sup>). Surprisingly, ssRNA viruses are those that show the least overlap between selection and correlated evolution, with only 11% of the genes evolving under both mechanisms (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1d</a>). Patterns are however much more difficult to extract here, mostly because the number of genes involved becomes quite small, in particular for the virus type with most overlap, the dsRNA viruses (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1d</a>: 29.4%, <em>i.e.</em> five genes).</p><p id="p-19">To assess the extent to which some of these differences at the gene level are functionally driven, we extracted the Gene Ontology (GO) annotations, or GO terms, associated with the genes analyzed, as well as those under selection, correlated evolution, or both – focusing exclusively on the virus types for which we had the largest samples sizes, dsDNA and ssRNA viruses (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1e</a>). Each of the three parts of the ontology (Molecular Function [MF], Biological Process [BP], and Cellular Component [CC]) were first limited to the second level of GO in order to derive a high-level interpretation (low-level descriptions are shown in Tables S1-S3). <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1e</a> shows that GO terms related to catalytic activity (MF), involved in the establishment or maintenance of a certain location (BP) at the membrane level (CC) are predominantly present in both dsDNA and ssRNA genomes. However, only a subset of these GO terms is mostly under episodic diversifying selection: helicase activity / binding (MF) and replication (BP) in the host cell nucleus (CC) in dsDNA, while it is mostly peptidase activity and methylation on the viral envelope in ssRNA viruses (Table S1; <em>P</em> &lt; 0.01). In spite of these differences, we note that episodic diversifying selection mostly affects genes involved in viral replication (Table S1). Genes that are affected by correlated evolution include: helicase activity and binding at the interface of multiple compartments in dsDNA viruses, or transferase activity and protein modifications on the envelope in ssRNA viruses (Table S2). Again, most of these functions and processes are involved in viral replication (Table S2). At the intersect of these evolutionary processes however, the genes that are jointly affected by selection and correlated evolution are involved in structural integrity and assembly within or outside a cell at the capsid level for dsDNA viruses, or in interacting with host cell surface via antigen activity on the viral envelope for ssRNA viruses – functions and processes that are mostly involved in “viral stability” (cell entry, integrity, assembly, immune escape; Table S3). This suggests that despite key differences in life history strategies adopted by dsDNA and ssRNA, there is a certain unity across viral types, where genes involved in replication are mostly under either selection or correlated evolution, while those involved in viral stability are mostly affected by both evolutionary processes – as has been shown for the influenza <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Gong <em>et al.</em> (2013)</a> and the Ebola <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Ibeh <em>et al.</em> (2016)</a> viruses (both cases involved a glycoprotein required for cell entry). Because these two evolutionary processes are required by epistasis, it is possible that the genes involved in viral stability are the most likely to show evidence for non-additive fitness effects – and hence, shape viral fitness landscapes. This tension between replication and stability is evocative of the existence of tradeoffs between capsid stability and proliferation within a host <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">De Paepe and Taddei (2006)</a>, or fecundity and lifespan <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">García-Villada and Drake (2013)</a> in dsDNA viruses, which supports the idea that it is the joint action of selection and of correlated evolution that shapes viral life histories.</p><p id="p-20">The previous analyses were at the gene level. One outstanding question is whether this relationship between selection and correlated evolution also holds at the amino acid level, that is, if the amino acids under selection are also involved in correlated evolution. This question was addressed in two different ways, all virus types confounded (in order to have larger sample sizes), by focusing on one process at a time, and finding at what point evidence supporting the second process becomes significant. First, we searched the list of pairs of sites evolving in a correlated manner (the SM sites) to see if at least one pair member was potentially evolving under episodic diversifying selection (the MEME sites), irrespective of its probability of being a MEME site. For this, we identified pairs of SM sites, that is those with a posterior probability ≥ 0.95, and plotted their posterior probability as a function of the − log<sub>10</sub> probability of each pair member to be under selection (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2a</a>). Both the least-square (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/07/12/539239/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/07/12/539239/embed/inline-graphic-3.gif"/></noscript></span></span>, <em>t</em> = 3.90, <em>P</em> = 0.0001) and the robust regressions (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/07/12/539239/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/07/12/539239/embed/inline-graphic-4.gif"/></noscript></span></span>, <em>t</em> = 2.62, <em>P</em> = 0.0089) have positive and significant slopes, hereby demonstrating the existence of a relationship between correlated evolution and episodic diversifying selection at the amino acid level – even if most of the SM sites show weak evidence of selection (at <em>P</em> ≤ 0.01, gray broken line in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2a</a>). So, to validate the existence of this relationship, we then took the list of MEME sites, and searched the list of SM sites to see if at least one pair member was potentially a MEME site, irrespective of its posterior probability of being in an SM site pair. For this, we identified the MEME sites, that is those with a <em>P</em>-value ≤ 0.01 (≥ 2 on a − log<sub>10</sub> scale), and plotted this (on the <em>x</em>-scale) <em>vs.</em> their posterior probability of being in an SM pair (on the <em>y</em>-scale; <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2b</a>). Both the least-square (<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/07/12/539239/embed/inline-graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/07/12/539239/embed/inline-graphic-5.gif"/></noscript></span></span>, <em>t</em> = 3.64, <em>P</em> = 0.0003) and the robust regressions (<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/07/12/539239/embed/inline-graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/07/12/539239/embed/inline-graphic-6.gif"/></noscript></span></span>, <em>t</em> = 3.37, <em>P</em> = 0.0008) have positive and significant slopes, further confirming the existence of a relationship between correlated evolution and episodic diversifying selection at the amino acid site level. Note however that this latter analysis is more informative than the former, as the density also shows that most of the sites under selection are involved in weak interactions. This is intriguingly reminiscent of the involvement of weakly interacting pairs of sites in severe outbreaks or pandemics <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Aris-Brosou <em>et al.</em> (2017)</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/12/539239/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Strength of evidence for sites under both selection (MEME) and correlated evolution (SM). (a) SM sites that are also under episodic diversifying selection. (b) MEME sites that are also evolving in a correlated manner. In both cases, the posterior probability of a site evolving in a correlated manner is plotted as a function of the P-value of a site being under selection (on a &#x2212; log10 scale). Regular least-square regressions are shown in blue; robust regressions are shown in red; 95% confidence envelopes are shaded in grey. See text for statistical details. Density scale: from low (cool colors) to high (warm colors)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-923146102" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Strength of evidence for sites under both selection (MEME) and correlated evolution (SM). (&lt;strong&gt;a&lt;/strong&gt;) SM sites that are also under episodic diversifying selection. (&lt;strong&gt;b&lt;/strong&gt;) MEME sites that are also evolving in a correlated manner. In both cases, the posterior probability of a site evolving in a correlated manner is plotted as a function of the &lt;em&gt;P&lt;/em&gt;-value of a site being under selection (on a &#x2212; log&lt;sub&gt;10&lt;/sub&gt; scale). Regular least-square regressions are shown in blue; robust regressions are shown in red; 95% confidence envelopes are shaded in grey. See text for statistical details. Density scale: from low (cool colors) to high (warm colors).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/12/539239/F2.medium.gif" width="440" height="134"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/12/539239/F2.medium.gif" width="440" height="134"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/12/539239/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/12/539239/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-21" class="first-child">Strength of evidence for sites under both selection (MEME) and correlated evolution (SM). (<strong>a</strong>) SM sites that are also under episodic diversifying selection. (<strong>b</strong>) MEME sites that are also evolving in a correlated manner. In both cases, the posterior probability of a site evolving in a correlated manner is plotted as a function of the <em>P</em>-value of a site being under selection (on a − log<sub>10</sub> scale). Regular least-square regressions are shown in blue; robust regressions are shown in red; 95% confidence envelopes are shaded in grey. See text for statistical details. Density scale: from low (cool colors) to high (warm colors).</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-6"><h2 class="">3 Conclusions</h2><p id="p-22">Altogether, we showed that episodic diversifying selection is mostly found in single stranded viruses, while correlated evolution is more prevalent in DNA viruses. More critically, we also showed that the genes affected by each process, when acting independently, are involved in viral replication. However, the genes that are jointly affected by both processes are mostly involved in viral stability (cell entry, integrity, assembly, immune escape), and that the same amino acid sites tend to be affected by both processes. In retrospect, this tight relationship between selection and correlated evolution may not be surprising, as correlated evolution can be underlain by epistasis <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Nshogozabahizi <em>et al.</em> (2017)</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Dench <em>et al.</em> (2019)</a>, which directly involves selection (in a non-additive way). Epistasis being a key determinant of fitness landscapes and hence, of evolutionary trajectories <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Li <em>et al.</em> (2016)</a>, our results suggest that in the long-term, both processes jointly shape the life history of viruses, favoring stability over proliferation. If so, analyzing viral evolution in the joint light of selection and correlated evolution might help us better predict how viruses that affect humans might evolve <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Weinreich <em>et al.</em> (2006)</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Pedruzzi <em>et al.</em> (2018)</a> – as predicting their evolution <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Sandie and Aris-Brosou (2014)</a> and epidemiology <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Ben-Nun <em>et al.</em> (2019)</a> has a long history fraught with mixed success.</p><p id="p-23">We note however that we neglected some aspects of viral structure: indeed, viruses can be segmented or not, with a circular or linear genome, with positive or negative strands, overlapping reading frames, complications that we could not consider here due to the resulting small sample sizes, even if these factors can impact the mode of evolution of viruses <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Holmes (2009)</a>. Future work should however strive to address these limitations. We also neglected the population genetics context in which different viruses evolve, context that can often be correlated to structural constraints <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Lynch (2007)</a>; <a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Holmes (2009)</a>. Furthermore, as we solely focused on intragenic interactions, and not intergenic or higher order correlations, it is not impossible that we missed higher-level constraints affecting viral evolution. In particular, it is possible that intergenic correlations reveal the nature of tradeoffs shaping the life history strategies in viruses <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">De Paepe and Taddei (2006)</a>. Future modeling <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Neverov <em>et al.</em> (2015)</a> and empirical <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Ashenberg <em>et al.</em> (2017)</a> work should probably focus on elucidating not only these constraints, but also the theoretical basis connecting correlated evolution, if not epistasis, to selection in shaping the evolutionary strategies of biological replicators, as current evidence linking these processes is currently limited to the influenza <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Gong <em>et al.</em> (2013)</a> and the Ebola <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Ibeh <em>et al.</em> (2016)</a> viruses.</p></div><div class="section ack" id="ack-1"><h2 class="">4 Acknowledgements</h2><p id="p-24">We thank Compute Canada and Ontario’s Centre for Advanced Computing for giving us access to their servers. This work was supported by the Natural Sciences Research Council of Canada (SAB), and by the University of Ottawa (LP, NI).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-dataset fn-group-external-links" id="fn-2"><p id="p-4">
<a href="https://github.com/sarisbro">https://github.com/sarisbro</a>
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.1" data-doi="10.1093/nar/gkq291"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Abascal</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Zardoya</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Telford</span></span>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">TranslatorX: multiple alignment of nucleotide sequences guided by amino acid translations</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">W7</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq291%26rft_id%253Dinfo%253Apmid%252F20435676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq291&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20435676&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284148900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alexa</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Rahnenführer</span></span>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gene set enrichment analysis with topGO</span>. <abbr class="cit-jnl-abbrev">Bioconductor Improv</abbr> <span class="cit-fpage">27</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aris-Brosou</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Ibeh</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Noël</span></span>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Viral outbreaks involve destabilized evolutionary networks: evidence from Ebola, Influenza and Zika</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">11881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D7%26rft.spage%253D11881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.4" data-doi="10.1007/978-1-61779-582-4_4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aris-Brosou</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Rodrigue</span></span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The essentials of computational molecular evolution</span>. <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr> <span class="cit-vol">855</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DAris-Brosou%26rft.auinit1%253DS.%26rft.volume%253D855%26rft.spage%253D111%26rft.epage%253D152%26rft.atitle%253DThe%2Bessentials%2Bof%2Bcomputational%2Bmolecular%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-61779-582-4_4%26rft_id%253Dinfo%253Apmid%252F22407707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-61779-582-4_4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22407707&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.5" data-doi="10.1371/journal.ppat.1006288"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ashenberg</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Padmakumar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Doud</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Bloom</span></span>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Deep mutational scanning identifies sites in influenza nucleoprotein that affect viral inhibition by MxA</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e1006288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D13%26rft.spage%253De1006288%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006288&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ben-Nun</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Riley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Turtle</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. P.</span> <span class="cit-name-surname">Bacon</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Riley</span></span>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Forecasting national and regional influenza-like illness for the USA</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">e1007013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol%26rft.volume%253D15%26rft.spage%253De1007013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.7" data-doi="10.1093/oxfordjournals.molbev.a026334"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Castresana</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis</span>. <abbr class="cit-jnl-abbrev">Molecular biology and evolution</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">540</span>–<span class="cit-lpage">552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DCastresana%26rft.auinit1%253DJ.%26rft.volume%253D17%26rft.issue%253D4%26rft.spage%253D540%26rft.epage%253D552%26rft.atitle%253DSelection%2Bof%2BConserved%2BBlocks%2Bfrom%2BMultiple%2BAlignments%2Bfor%2BTheir%2BUse%2Bin%2BPhylogenetic%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026334%26rft_id%253Dinfo%253Apmid%252F10742046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a026334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10742046&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086337700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.8" data-doi="10.1093/bioinformatics/bti610"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Conesa</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Götz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">García-Gómez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Terol</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Talón</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3674</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti610%26rft_id%253Dinfo%253Apmid%252F16081474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti610&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16081474&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231694600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.9" data-doi="10.1371/journal.pbio.0040193"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De Paepe</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Taddei</span></span>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Viruses’ life history: towards a mechanistic basis of a trade-off between survival and reproduction among phages</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDe%2BPaepe%26rft.auinit1%253DM.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253De193%26rft.epage%253De193%26rft.atitle%253DViruses%2527%2Blife%2Bhistory%253A%2Btowards%2Ba%2Bmechanistic%2Bbasis%2Bof%2Ba%2Btrade-off%2Bbetween%2Bsurvival%2Band%2Breproduction%2Bamong%2Bphages.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040193%26rft_id%253Dinfo%253Apmid%252F16756387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16756387&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dench</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Hinz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Aris-Brosou</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kassen</span></span>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">The idiosyncratic drivers of correlated evolution</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> p. <span class="cit-fpage">474536</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.11" data-doi="10.1371/journal.pbio.3000003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Duffy</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Why are RNA virus mutation rates so damn high?</span> <abbr class="cit-jnl-abbrev">PLoS Biol</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">e3000003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol%26rft.volume%253D16%26rft.spage%253De3000003%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.3000003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.3000003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.12" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R. C.</span></span>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Muscle: multiple sequence alignment with high accuracy and high throughput</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034147&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220487200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.13" data-doi="10.1126/science.aaf5036"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Faria</span>,  <span class="cit-name-given-names">N. R.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. d. S. d. S.</span> <span class="cit-name-surname">Azevedo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. U. G.</span> <span class="cit-name-surname">Kraemer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Souza</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Cunha</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2016a</span> <span class="cit-article-title">Zika virus in the americas: Early epidemiological and genetic findings</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">352</span>: <span class="cit-fpage">345</span>–<span class="cit-lpage">349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf5036%26rft_id%253Dinfo%253Apmid%252F27013429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTIvNjI4My8zNDUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNy8xMi81MzkyMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Faria</span>,  <span class="cit-name-given-names">N. R.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Sabino</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. R. T.</span> <span class="cit-name-surname">Nunes</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. C. J.</span> <span class="cit-name-surname">Alcantara</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Loman</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2016b</span> <span class="cit-article-title">Mobile real-time surveillance of Zika virus in Brazil</span>. <abbr class="cit-jnl-abbrev">Genome Medicine</abbr> <span class="cit-fpage">8</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.15" data-doi="10.1021/bi00462a015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frederico</span>,  <span class="cit-name-given-names">L. A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. A.</span> <span class="cit-name-surname">Kunkel</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B. R.</span> <span class="cit-name-surname">Shaw</span></span>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">A sensitive genetic assay for the detection of cytosine deamination: determination of rate constants and the activation energy</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">2532</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D29%26rft.spage%253D2532%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi00462a015%26rft_id%253Dinfo%253Apmid%252F2185829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi00462a015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2185829&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990CU24500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.16" data-doi="10.1016/j.virusres.2017.08.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Jin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Xu</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">G.-Z.</span> <span class="cit-name-surname">Han</span></span>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Extent and evolution of gene duplication in DNA viruses</span>. <abbr class="cit-jnl-abbrev">Virus research</abbr> <span class="cit-vol">240</span>: <span class="cit-fpage">161</span>–<span class="cit-lpage">165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.volume%253D240%26rft.spage%253D161%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2017.08.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2017.08.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.17" data-doi="10.1098/rsob.130043"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">García-Villada</span>,  <span class="cit-name-given-names">L.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Drake</span></span>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Experimental selection reveals a trade-off between fecundity and lifespan in the coliphage Qß</span>. <abbr class="cit-jnl-abbrev">Open Biol</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">130043</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOpen%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsob.130043%26rft_id%253Dinfo%253Apmid%252F23760365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsob.130043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23760365&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Sequence file format conversion with command-line readseq</span>. <abbr class="cit-jnl-abbrev">Curr Protoc Bioinformatics</abbr> <span class="cit-vol">Appendix 1</span>: Appendix 1E.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.19" data-doi="10.1126/science.1259657"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gire</span>,  <span class="cit-name-given-names">S. K.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Goba</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. G.</span> <span class="cit-name-surname">Andersen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. S. G.</span> <span class="cit-name-surname">Sealfon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Park</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">345</span>: <span class="cit-fpage">1369</span>–<span class="cit-lpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1259657%26rft_id%253Dinfo%253Apmid%252F25214632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDUvNjIwMi8xMzY5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDcvMTIvNTM5MjM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.20" data-doi="10.7554/eLife.00631"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gong</span>,  <span class="cit-name-given-names">L. I.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Suchard</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Bloom</span></span>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Stability-mediated epistasis constrains the evolution of an influenza protein</span>. <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e00631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.stitle%253Delife%26rft.volume%253D2%26rft.issue%253D0%26rft.spage%253De00631%26rft.epage%253De00631%26rft.atitle%253DStability-mediated%2Bepistasis%2Bconstrains%2Bthe%2Bevolution%2Bof%2Ban%2Binfluenza%2Bprotein%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.00631%26rft_id%253Dinfo%253Apmid%252F23682315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.00631&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23682315&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.21" data-doi="10.1126/science.1090727"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grenfell</span>,  <span class="cit-name-given-names">B. T.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O. G.</span> <span class="cit-name-surname">Pybus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Gog</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. L. N.</span> <span class="cit-name-surname">Wood</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Daly</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Unifying the epidemiological and evolutionary dynamics of pathogens</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">303</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGrenfell%26rft.auinit1%253DB.%2BT.%26rft.volume%253D303%26rft.issue%253D5656%26rft.spage%253D327%26rft.epage%253D332%26rft.atitle%253DUnifying%2Bthe%2BEpidemiological%2Band%2BEvolutionary%2BDynamics%2Bof%2BPathogens%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1090727%26rft_id%253Dinfo%253Apmid%252F14726583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDMvNTY1Ni8zMjciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNy8xMi81MzkyMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-539239v3.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E. C.</span></span>, <span class="cit-pub-date">2009</span> <span class="cit-chapter-title">The evolution and emergence of RNA viruses</span>. <span class="cit-source">Oxford series in ecology and evolution</span>, <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.23" data-doi="10.1128/JVI.02203-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E. C.</span></span>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">What does virus evolution tell us about virus origins?</span> <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">5247</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02203-10%26rft_id%253Dinfo%253Apmid%252F21450811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4NS8xMS81MjQ3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDcvMTIvNTM5MjM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.24" data-doi="10.1128/JVI.00322-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ibeh</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Nshogozabahizi</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Aris-Brosou</span></span>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Both epistasis and diversifying selection drive the structural evolution of the Ebola virus glycoprotein mucin-like domain</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">5475</span>–<span class="cit-lpage">5484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00322-16%26rft_id%253Dinfo%253Apmid%252F27009964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI5MC8xMS81NDc1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDcvMTIvNTM5MjM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.25" data-doi="10.1093/molbev/msi105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kosakovsky Pond</span>,  <span class="cit-name-given-names">S. L.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">S. D. W.</span> <span class="cit-name-surname">Frost</span></span>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Not so different after all: a comparison of methods for detecting amino acid sites under selection</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1208</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi105%26rft_id%253Dinfo%253Apmid%252F15703242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msi105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15703242&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228700200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.26" data-doi="10.1371/journal.pgen.1001301"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kryazhimskiy</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dushoff</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. A.</span> <span class="cit-name-surname">Bazykin</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Plotkin</span></span>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Prevalence of epistasis in the evolution of influenza A surface proteins</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKryazhimskiy%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De1001301%26rft.epage%253De1001301%26rft.atitle%253DPrevalence%2Bof%2Bepistasis%2Bin%2Bthe%2Bevolution%2Bof%2Binfluenza%2BA%2Bsurface%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001301%26rft_id%253Dinfo%253Apmid%252F21390205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21390205&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.27" data-doi="10.1126/science.aae0568"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Qian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Maclean</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The fitness landscape of a tRNA gene</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">352</span>: <span class="cit-fpage">837</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aae0568%26rft_id%253Dinfo%253Apmid%252F27080104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTIvNjI4Ny84MzciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNy8xMi81MzkyMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-539239v3.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lynch</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-pub-date">2007</span> <span class="cit-source">The origins of genome architecture</span>. <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, Mass</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lyons</span>,  <span class="cit-name-given-names">D. M.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">A. S.</span> <span class="cit-name-surname">Lauring</span></span>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Mutation and epistasis in influenza virus evolution</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">10</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.30" data-doi="10.1093/sysbio/syu070"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maddison</span>,  <span class="cit-name-given-names">W. P.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">FitzJohn</span></span>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The unsolved challenge to phylogenetic correlation tests for categorical characters</span>. <abbr class="cit-jnl-abbrev">Syst Biol</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyu070%26rft_id%253Dinfo%253Apmid%252F25209222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syu070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25209222&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.31" data-doi="10.1038/nature08691"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meer</span>,  <span class="cit-name-given-names">M. V.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. S.</span> <span class="cit-name-surname">Kondrashov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Artzy-Randrup</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">F. A.</span> <span class="cit-name-surname">Kondrashov</span></span>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Compensatory evolution in mitochondrial tRNAs navigates valleys of low fitness</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMeer%26rft.auinit1%253DM.%2BV.%26rft.volume%253D464%26rft.issue%253D7286%26rft.spage%253D279%26rft.epage%253D282%26rft.atitle%253DCompensatory%2Bevolution%2Bin%2Bmitochondrial%2BtRNAs%2Bnavigates%2Bvalleys%2Bof%2Blow%2Bfitness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08691%26rft_id%253Dinfo%253Apmid%252F20182427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08691&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20182427&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275366100047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.32" data-doi="10.1371/journal.pgen.1002764"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Murrell</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. O.</span> <span class="cit-name-surname">Wertheim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Moola</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Weighill</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Scheffler</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Detecting individual sites subject to episodic diversifying selection</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002764</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMurrell%26rft.auinit1%253DB.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253De1002764%26rft.epage%253De1002764%26rft.atitle%253DDetecting%2Bindividual%2Bsites%2Bsubject%2Bto%2Bepisodic%2Bdiversifying%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002764%26rft_id%253Dinfo%253Apmid%252F22807683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002764&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22807683&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.33" data-doi="10.1371/journal.pgen.1005404"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Neverov</span>,  <span class="cit-name-given-names">A. D.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kryazhimskiy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Plotkin</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">G. A.</span> <span class="cit-name-surname">Bazykin</span></span>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Coordinated evolution of influenza A surface proteins</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D11%26rft.spage%253De1005404%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005404%26rft_id%253Dinfo%253Apmid%252F26247472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005404&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26247472&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.34" data-doi="10.1534/genetics.116.193979"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nshogozabahizi</span>,  <span class="cit-name-given-names">J. C.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dench</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Aris-Brosou</span></span>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Widespread historical contingency in influenza viruses</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.193979%26rft_id%253Dinfo%253Apmid%252F28049709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA1LzEvNDA5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDcvMTIvNTM5MjM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.35" data-doi="10.1371/journal.pcbi.1006426"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pedruzzi</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Barlukova</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Rouzine</span></span>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Evolutionary footprint of epistasis</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">e1006426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol%26rft.volume%253D14%26rft.spage%253De1006426%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1006426%26rft_id%253Dinfo%253Apmid%252F30222748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1006426&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30222748&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.36" data-doi="10.1093/bioinformatics/bti079"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pond</span>,  <span class="cit-name-given-names">S. L. K.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. D. W.</span> <span class="cit-name-surname">Frost</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. V.</span> <span class="cit-name-surname">Muse</span></span>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">HyPhy: hypothesis testing using phylogenies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">676</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti079%26rft_id%253Dinfo%253Apmid%252F15509596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15509596&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227241200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.37" data-doi="10.1093/bioinformatics/btn313"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Poon</span>,  <span class="cit-name-given-names">A. F. Y.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. I.</span> <span class="cit-name-surname">Lewis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. D. W.</span> <span class="cit-name-surname">Frost</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Kosakovsky Pond</span></span>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Spidermonkey: rapid detection of co-evolving sites using Bayesian graphical models</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1949</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn313%26rft_id%253Dinfo%253Apmid%252F18562270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18562270&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258860700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.38" data-doi="10.1371/journal.pone.0009490"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Price</span>,  <span class="cit-name-given-names">M. N.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. S</span> <span class="cit-name-surname">Dehal</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Arkin</span></span>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">FastTree 2–approximately maximum- likelihood trees for large alignments</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e9490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPrice%26rft.auinit1%253DM.%2BN.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De9490%26rft.epage%253De9490%26rft.atitle%253DFastTree%2B2--approximately%2Bmaximum-likelihood%2Btrees%2Bfor%2Blarge%2Balignments.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0009490%26rft_id%253Dinfo%253Apmid%252F20224823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0009490&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20224823&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-539239v3.39"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Core Team</span>, <span class="cit-pub-date">2016</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Revell</span>,  <span class="cit-name-given-names">L. J.</span></span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">phytools: an R package for phylogenetic comparative biology (and other things)</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D3%26rft.spage%253D217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.41" data-doi="10.1016/j.meegid.2011.04.031"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robinson</span>,  <span class="cit-name-given-names">C. M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Seto</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Jones</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Dyer</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chodosh</span></span>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Molecular evolution of human species D adenoviruses</span>. <abbr class="cit-jnl-abbrev">Infect Genet Evol</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1208</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%252C%2Bgenetics%2Band%2Bevolution%2B%253A%2B%2Bjournal%2Bof%2Bmolecular%2Bepidemiology%2Band%2Bevolutionary%2Bgenetics%2Bin%2Binfectious%2Bdiseases%26rft.stitle%253DInfect%2BGenet%2BEvol%26rft.aulast%253DRobinson%26rft.auinit1%253DC.%2BM.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253D1208%26rft.epage%253D1217%26rft.atitle%253DMolecular%2Bevolution%2Bof%2Bhuman%2Bspecies%2BD%2Badenoviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.meegid.2011.04.031%26rft_id%253Dinfo%253Apmid%252F21570490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.meegid.2011.04.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21570490&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sandie</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Aris-Brosou</span></span>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Predicting the emergence of H3N2 influenza viruses reveals contrasted modes of evolution of HA and NA antigens</span>. <abbr class="cit-jnl-abbrev">J Mol Evol</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BMol%2BEvol%26rft.volume%253D78%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.43" data-doi="10.1371/journal.ppat.1002685"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanjuán</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">From molecular genetics to phylodynamics: evolutionary relevance of mutation rates across viruses</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002685</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DSanjuan%26rft.auinit1%253DR.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253De1002685%26rft.epage%253De1002685%26rft.atitle%253DFrom%2Bmolecular%2Bgenetics%2Bto%2Bphylodynamics%253A%2Bevolutionary%2Brelevance%2Bof%2Bmutation%2Brates%2Bacross%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002685%26rft_id%253Dinfo%253Apmid%252F22570614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002685&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22570614&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.44" data-doi="10.1038/nrmicro2614"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Simon-Loriere</span>,  <span class="cit-name-given-names">E.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Holmes</span></span>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Why do RNA viruses recombine?</span> <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DSimon-Loriere%26rft.auinit1%253DE.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D617%26rft.epage%253D626%26rft.atitle%253DWhy%2Bdo%2BRNA%2Bviruses%2Brecombine%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2614%26rft_id%253Dinfo%253Apmid%252F21725337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2614&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21725337&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.45" data-doi="10.1038/nature08182"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">G. J. D.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Vijaykrishna</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bahl</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Lycett</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Worobey</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Origins and evolutionary genomics of the 2009 swine-origin H1N1 influenza a epidemic</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">459</span>: <span class="cit-fpage">1122</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSmith%26rft.auinit1%253DG.%2BJ.%26rft.volume%253D459%26rft.issue%253D7250%26rft.spage%253D1122%26rft.epage%253D1125%26rft.atitle%253DOrigins%2Band%2Bevolutionary%2Bgenomics%2Bof%2Bthe%2B2009%2Bswine-origin%2BH1N1%2Binfluenza%2BA%2Bepidemic.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08182%26rft_id%253Dinfo%253Apmid%252F19516283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08182&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19516283&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267636700042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stano</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Beke</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Klucar</span></span>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">viruSITE-integrated database for viral genomics</span>. <abbr class="cit-jnl-abbrev">Database (Oxford)</abbr> <span class="cit-vol">2016</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Boheemen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">de Graaf</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lauber</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Bestebroer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V. S.</span> <span class="cit-name-surname">Raj</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic characterization of a newly discovered coronavirus associated with acute respiratory distress syndrome in humans</span>. <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">3</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.48" data-doi="10.1126/science.1123539"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weinreich</span>,  <span class="cit-name-given-names">D. M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. F.</span> <span class="cit-name-surname">Delaney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Depristo</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Hartl</span></span>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Darwinian evolution can follow only very few mutational paths to fitter proteins</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">312</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWeinreich%26rft.auinit1%253DD.%2BM.%26rft.volume%253D312%26rft.issue%253D5770%26rft.spage%253D111%26rft.epage%253D114%26rft.atitle%253DDarwinian%2Bevolution%2Bcan%2Bfollow%2Bonly%2Bvery%2Bfew%2Bmutational%2Bpaths%2Bto%2Bfitter%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1123539%26rft_id%253Dinfo%253Apmid%252F16601193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTIvNTc3MC8xMTEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNy8xMi81MzkyMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woo</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Robertson</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Lovell</span></span>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Constraints from protein structure and intra-molecular coevolution influence the fitness of HIV-1 recombinants</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">454-455</span>: <span class="cit-fpage">34</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D454455%26rft.spage%253D34%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-book" id="cit-539239v3.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yohai</span>,  <span class="cit-name-given-names">V. J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Stahel</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R. H.</span> <span class="cit-name-surname">Zamar</span></span>, <span class="cit-pub-date">1991</span> <span class="cit-chapter-title">A procedure for robust estimation and inference in linear regression</span>. In <span class="cit-source">Directions in robust statistics and diagnostics</span>, pp. <span class="cit-fpage">365</span>–<span class="cit-lpage">374</span>, <span class="cit-publ-name">Springer</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-539239v3.51" data-doi="10.1056/NEJMoa1211721"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zaki</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">van Boheemen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Bestebroer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. D. M. E.</span> <span class="cit-name-surname">Osterhaus</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R. A. M.</span> <span class="cit-name-surname">Fouchier</span></span>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr> <span class="cit-vol">367</span>: <span class="cit-fpage">1814</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DZaki%26rft.auinit1%253DA.%2BM.%26rft.volume%253D367%26rft.issue%253D19%26rft.spage%253D1814%26rft.epage%253D1820%26rft.atitle%253DIsolation%2Bof%2Ba%2Bnovel%2Bcoronavirus%2Bfrom%2Ba%2Bman%2Bwith%2Bpneumonia%2Bin%2BSaudi%2BArabia.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1211721%26rft_id%253Dinfo%253Apmid%252F23075143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1211721&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23075143&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F12%2F539239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310773200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/539239v3.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F539239v3" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/462184v2" title="A curative combination therapy for lymphomas achieves high fractional cell killing through low cross-resistance and drug additivity but not synergy" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;July 12, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/539239v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/539239v3.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2019/07/12/539239.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/539239v3.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/539239v3" class="active" data-icon-position="" data-hide-link-title="0">Viral long-term evolutionary strategies favor stability over proliferation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/814925" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-7boXeR6QVmIoVdssQWd7BQh32XQyZxlA0b_dN7NasgY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="814925" id="node814925--2" data-pisa="biorxiv;539239v3" data-pisa-master="biorxiv;539239" data-seqnum="814925" data-apath="/biorxiv/early/2019/07/12/539239.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Viral long-term evolutionary strategies favor stability over proliferation</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Stéphane</span> <span class="nlm-surname">Aris-Brosou</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Louis</span> <span class="nlm-surname">Parent</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Neke</span> <span class="nlm-surname">Ibeh</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">539239; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/539239 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/539239v3" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/539239v3&amp;title=Viral%20long-term%20evolutionary%20strategies%20favor%20stability%20over%20proliferation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/539239v3&amp;title=Viral%20long-term%20evolutionary%20strategies%20favor%20stability%20over%20proliferation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/539239v3&amp;text=Viral%20long-term%20evolutionary%20strategies%20favor%20stability%20over%20proliferation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/539239v3&amp;title=Viral%20long-term%20evolutionary%20strategies%20favor%20stability%20over%20proliferation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/539239v3&amp;t=Viral%20long-term%20evolutionary%20strategies%20favor%20stability%20over%20proliferation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/539239v3&amp;title=Viral%20long-term%20evolutionary%20strategies%20favor%20stability%20over%20proliferation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/539239v3&amp;title=Viral%20long-term%20evolutionary%20strategies%20favor%20stability%20over%20proliferation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="814925" id="node814925--3" data-pisa="biorxiv;539239v3" data-pisa-master="biorxiv;539239" data-seqnum="814925" data-apath="/biorxiv/early/2019/07/12/539239.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Viral long-term evolutionary strategies favor stability over proliferation</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Stéphane</span> <span class="nlm-surname">Aris-Brosou</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Louis</span> <span class="nlm-surname">Parent</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Neke</span> <span class="nlm-surname">Ibeh</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">539239; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/539239 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Viral%20long-term%20evolutionary%20strategies%20favor%20stability%20over%20proliferation&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Aris-Brosou%2C+St%C3%A9phane&amp;rft.au=Parent%2C+Louis&amp;rft.au=Ibeh%2C+Neke"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/814925/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/814925/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/814925/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/814925/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/814925/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/814925/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/814925/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/814925/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/814925/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/814925/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/814925/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/814925/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/539239v3&amp;count=horizontal&amp;via=&amp;text=Viral%20long-term%20evolutionary%20strategies%20favor%20stability%20over%20proliferation&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/539239v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/539239v3&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/539239v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
