<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/10/24/812313.atom" />
<meta name="HW.pisa" content="biorxiv;812313v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="An iterative and automated computational pipeline for untargeted strain-level identification using MS/MS spectra from pathogenic samples" />
<meta name="DC.Identifier" content="10.1101/812313" />
<meta name="DC.Date" content="2019-10-24" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Untargeted accurate strain-level classification of a priori unidentified organisms using tandem mass spectrometry is a challenging task. Reference databases often lack taxonomic depth, limiting peptide assignments to the species level. However, the extension with detailed strain information increases runtime and decreases statistical power. In addition, larger databases contain a higher number of similar proteomes.

We present TaxIt, an iterative workflow to address the increasing search space required for MS/MS-based strain-level classification of samples with unknown taxonomic origin. TaxIt first applies reference sequence data for initial identification of species candidates, followed by automated acquisition of relevant strain sequences for low level classification. Furthermore, proteome similarities resulting in ambiguous taxonomic assignments are addressed with an abundance weighting strategy to improve candidate confidence.

We apply our iterative workflow on several samples of bacterial and viral origin. In comparison to non-iterative approaches using unique peptides or advanced abundance correction, TaxIt identifies microbial strains correctly in all examples presented (with one tie), thereby demonstrating the potential for untargeted and deeper taxonomic classification. TaxIt makes extensive use of public, unrestricted and continuously growing sequence resources such as the NCBI databases and is available under open-source license at &lt;https://gitlab.com/rki_bioinformatics&gt;.

*   FDR
    :   false discovery rate
    LC
    :   liquid chromatography
    MALDI
    :   matrix-assisted laser desorption/ionization
    MS
    :   mass spectrometry
    PSM
    :   peptide spectrum match
    RSS
    :   resident set size
    TOF
    :   time-of-flight" />
<meta name="DC.Contributor" content="Mathias Kuhring" />
<meta name="DC.Contributor" content="Joerg Doellinger" />
<meta name="DC.Contributor" content="Andreas Nitsche" />
<meta name="DC.Contributor" content="Thilo Muth" />
<meta name="DC.Contributor" content="Bernhard Y. Renard" />
<meta name="article:published_time" content="2019-10-24" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="An iterative and automated computational pipeline for untargeted strain-level identification using MS/MS spectra from pathogenic samples" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Untargeted accurate strain-level classification of a priori unidentified organisms using tandem mass spectrometry is a challenging task. Reference databases often lack taxonomic depth, limiting peptide assignments to the species level. However, the extension with detailed strain information increases runtime and decreases statistical power. In addition, larger databases contain a higher number of similar proteomes.&lt;/p&gt;&lt;p&gt;We present TaxIt, an iterative workflow to address the increasing search space required for MS/MS-based strain-level classification of samples with unknown taxonomic origin. TaxIt first applies reference sequence data for initial identification of species candidates, followed by automated acquisition of relevant strain sequences for low level classification. Furthermore, proteome similarities resulting in ambiguous taxonomic assignments are addressed with an abundance weighting strategy to improve candidate confidence.&lt;/p&gt;&lt;p&gt;We apply our iterative workflow on several samples of bacterial and viral origin. In comparison to non-iterative approaches using unique peptides or advanced abundance correction, TaxIt identifies microbial strains correctly in all examples presented (with one tie), thereby demonstrating the potential for untargeted and deeper taxonomic classification. TaxIt makes extensive use of public, unrestricted and continuously growing sequence resources such as the NCBI databases and is available under open-source license at https://gitlab.com/rki_bioinformatics.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;812313v1" />
<meta name="citation_id" content="812313v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/812313v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/812313v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/812313v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313.full.pdf" />
<meta name="citation_doi" content="10.1101/812313" />
<meta name="citation_num_pages" content="21" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="812313" />
<meta name="citation_author" content="Mathias Kuhring" />
<meta name="citation_author_institution" content="Bioinformatics Unit (MF 1), Department for Methods Development and Research Infrastructure, Robert Koch Institute" />
<meta name="citation_author_institution" content="Core Unit Bioinformatics, Berlin Institute of Health (BIH)" />
<meta name="citation_author_institution" content="Berlin Institute of Health Metabolomics Platform, Berlin Institute of Health (BIH)" />
<meta name="citation_author_institution" content="Max Delbrück Center (MDC) for Molecular Medicine" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3287-0313" />
<meta name="citation_author" content="Joerg Doellinger" />
<meta name="citation_author_institution" content="Centre for Biological Threats and Special Pathogens, Proteomics and Spectroscopy (ZBS 6), Robert Koch Institute" />
<meta name="citation_author_institution" content="Centre for Biological Threats and Special Pathogens, Highly Pathogenic Viruses (ZBS1), Robert Koch Institute" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8309-082X" />
<meta name="citation_author" content="Andreas Nitsche" />
<meta name="citation_author_institution" content="Centre for Biological Threats and Special Pathogens, Highly Pathogenic Viruses (ZBS1), Robert Koch Institute" />
<meta name="citation_author" content="Thilo Muth" />
<meta name="citation_author_institution" content="Bioinformatics Unit (MF 1), Department for Methods Development and Research Infrastructure, Robert Koch Institute" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8304-2684" />
<meta name="citation_author" content="Bernhard Y. Renard" />
<meta name="citation_author_institution" content="Bioinformatics Unit (MF 1), Department for Methods Development and Research Infrastructure, Robert Koch Institute" />
<meta name="citation_author_email" content="renardb@rki.de" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4589-9809" />
<meta name="citation_reference" content="Bengali, Z.; Townsley, A. C.; Moss, B. Vaccinia Virus Strain Differences in Cell Attachment and Entry. Virology 2009, 389 (1), 132–140. https://doi.org/10.1016/j.virol.2009.04.012." />
<meta name="citation_reference" content="Bengali, Z.; Satheshkumar, P. S.; Moss, B. Orthopoxvirus Species and Strain Differences in Cell Entry. Virology 2012, 433 (2), 506–512. https://doi.org/10.1016/j.virol.2012.08.044." />
<meta name="citation_reference" content="Doellinger, J.; Schaade, L.; Nitsche, A. Comparison of the Cowpox Virus and Vaccinia Virus Mature Virion Proteome: Analysis of the Species- and Strain-Specific Proteome. PLOS ONE 2015, 10 (11), e0141527. https://doi.org/10.1371/journal.pone.0141527." />
<meta name="citation_reference" content="Choi, J. Y.; Sifri, C. D.; Goumnerov, B. C.; Rahme, L. G.; Ausubel, F. M.; Calderwood, S. B. Identification of Virulence Genes in a Pathogenic Strain of Pseudomonas Aeruginosa by Representational Difference Analysis. J. Bacteriol. 2002, 184 (4), 952–961. https://doi.org/10.1128/jb.184.4.952-961.2002." />
<meta name="citation_reference" content="Genersch, E.; Ashiralieva, A.; Fries, I. Strain- and Genotype-Specific Differences in Virulence of Paenibacillus Larvae Subsp. Larvae, a Bacterial Pathogen Causing American Foulbrood Disease in Honeybees. Appl. Environ. Microbiol. 2005, 71 (11), 7551–7555. https://doi.org/10.1128/AEM.71.11.7551-7555.2005." />
<meta name="citation_reference" content="Boulund, F.; Karlsson, R.; Gonzales-Siles, L.; Johnning, A.; Karami, N.; Al-Bayati, O.; Åhrén, C.; Moore, E. R. B.; Kristiansson, E. Typing and Characterization of Bacteria Using Bottom-up Tandem Mass Spectrometry Proteomics. Mol. Cell. Proteomics MCP 2017, 16 (6), 1052–1063. https://doi.org/10.1074/mcp.M116.061721." />
<meta name="citation_reference" content="Singhal, N.; Kumar, M.; Kanaujia, P. K.; Virdi, J. S. MALDI-TOF Mass Spectrometry: An Emerging Technology for Microbial Identification and Diagnosis. Front. Microbiol. 2015, 6. https://doi.org/10.3389/fmicb.2015.00791." />
<meta name="citation_reference" content="MALDI Biotyper Systems | Bruker https://www.bruker.com/products/mass-spectrometry-and-separations/maldi-biotyper-systems.html (accessed Sep 29, 2017)." />
<meta name="citation_reference" content="Strain Typing with IR Biotyper Overview | Bruker https://www.bruker.com/applications/microbiology/strain-typing-with-ir-biotyper/overview.html (accessed Sep 29, 2017)." />
<meta name="citation_reference" content="Ecker, D. J.; Drader, J. J.; Gutierrez, J.; Gutierrez, A.; Hannis, J. C.; Schink, A.; Sampath, R.; Blyn, L. B.; Eshoo, M. W.; Hall, T. A.; et al. The Ibis T5000 Universal Biosensor: An Automated Platform for Pathogen Identification and Strain Typing. JALA J. Assoc. Lab. Autom. 2006, 11 (6), 341–351. https://doi.org/10.1016/j.jala.2006.09.001." />
<meta name="citation_reference" content="Gekenidis, M.-T.; Studer, P.; Wüthrich, S.; Brunisholz, R.; Drissner, D. Beyond the Matrix-Assisted Laser Desorption Ionization (MALDI) Biotyping Workflow: In Search of Microorganism-Specific Tryptic Peptides Enabling Discrimination of Subspecies. Appl. Environ. Microbiol. 2014, 80 (14), 4234–4241. https://doi.org/10.1128/AEM.00740-14." />
<meta name="citation_reference" content="Pfrunder, S.; Grossmann, J.; Hunziker, P.; Brunisholz, R.; Gekenidis, M.-T.; Drissner, D. Bacillus Cereus Group-Type Strain-Specific Diagnostic Peptides. J. Proteome Res. 2016, 15 (9), 3098–3107. https://doi.org/10.1021/acs.jproteome.6b00216." />
<meta name="citation_reference" content="Alves, G.; Wang, G.; Ogurtsov, A. Y.; Drake, S. K.; Gucek, M.; Suffredini, A. F.; Sacks, D. B.; Yu, Y.-K. Identification of Microorganisms by High Resolution Tandem Mass Spectrometry with Accurate Statistical Significance. J. Am. Soc. Mass Spectrom. 2016, 27 (2), 194–210. https://doi.org/10.1007/s13361-015-1271-2." />
<meta name="citation_reference" content="Aebersold, R.; Mann, M. Mass Spectrometry-Based Proteomics. Nature 2003, 422 (6928), 198–207. https://doi.org/10.1038/nature01511." />
<meta name="citation_reference" content="Domon, B.; Aebersold, R. Mass Spectrometry and Protein Analysis. Science 2006, 312 (5771), 212–217. https://doi.org/10.1126/science.1124619." />
<meta name="citation_reference" content="McHugh, L.; Arthur, J. W. Computational Methods for Protein Identification from Mass Spectrometry Data. PLoS Comput Biol 2008, 4 (2), e12. https://doi.org/10.1371/journal.pcbi.0040012." />
<meta name="citation_reference" content="Liu, T.; Belov, M. E.; Jaitly, N.; Qian, W.-J.; Smith, R. D. Accurate Mass Measurements in Proteomics. Chem. Rev. 2007, 107 (8), 3621–3653. https://doi.org/10.1021/cr068288j." />
<meta name="citation_reference" content="Mann, M.; Kelleher, N. L. Precision Proteomics: The Case for High Resolution and High Mass Accuracy. Proc. Natl. Acad. Sci. 2008, 105 (47), 18132–18138. https://doi.org/10.1073/pnas.0800788105." />
<meta name="citation_reference" content="Kuhring, M.; Renard, B. Y. Estimating the Computational Limits of Detection of Microbial Non-Model Organisms. PROTEOMICS 2015, 15 (20), 3580–3584. https://doi.org/10.1002/pmic.201400598." />
<meta name="citation_reference" content="Jeong, K.; Kim, S.; Bandeira, N. False Discovery Rates in Spectral Identification. BMC Bioinformatics 2012, 13 (16), S2. https://doi.org/10.1186/1471-2105-13-S16-S2." />
<meta name="citation_reference" content="Muth, T.; Kolmeder, C. A.; Salojärvi, J.; Keskitalo, S.; Varjosalo, M.; Verdam, F. J.; Rensen, S. S.; Reichl, U.; de Vos, W. M.; Rapp, E.; et al. Navigating through Metaproteomics Data: A Logbook of Database Searching. Proteomics 2015. https://doi.org/10.1002/pmic.201400560." />
<meta name="citation_reference" content="Pible, O.; Hartmann, E. M.; Imbert, G.; Armengaud, J. The Importance of Recognizing and Reporting Sequence Database Contamination for Proteomics. EuPA Open Proteomics 2014, 3, 246–249. https://doi.org/10.1016/j.euprot.2014.04.001." />
<meta name="citation_reference" content="Creasy, D. M.; Cottrell, J. S. Error Tolerant Searching of Uninterpreted Tandem Mass Spectrometry Data. PROTEOMICS 2002, 2 (10), 1426–1434. https://doi.org/10.1002/1615-9861(200210)2:10&lt;1426::AID-PROT1426&gt;3.0.CO;2-5." />
<meta name="citation_reference" content="Craig, R.; Beavis, R. C. A Method for Reducing the Time Required to Match Protein Sequences with Tandem Mass Spectra. Rapid Commun. Mass Spectrom. 2003, 17 (20), 2310–2316. https://doi.org/10.1002/rcm.1198." />
<meta name="citation_reference" content="Nesvizhskii, A. I.; Roos, F. F.; Grossmann, J.; Vogelzang, M.; Eddes, J. S.; Gruissem, W.; Baginsky, S.; Aebersold, R. Dynamic Spectrum Quality Assessment and Iterative Computational Analysis of Shotgun Proteomic Data Toward More Efficient Identification of Post-Translational Modifications, Sequence Polymorphisms, and Novel Peptides. Mol. Cell. Proteomics 2006, 5 (4), 652–670. https://doi.org/10.1074/mcp.M500319-MCP200." />
<meta name="citation_reference" content="Rooijers, K.; Kolmeder, C.; Juste, C.; Doré, J.; Been, M. de; Boeren, S.; Galan, P.; Beauvallet, C.; Vos, W. M. de; Schaap, P. J. An Iterative Workflow for Mining the Human Intestinal Metaproteome. BMC Genomics 2011, 12 (1), 6. https://doi.org/10.1186/1471-2164-12-6." />
<meta name="citation_reference" content="Ning, K.; Fermin, D.; Nesvizhskii, A. I. Computational Analysis of Unassigned High-Quality MS/MS Spectra in Proteomic Data Sets. PROTEOMICS 2010, 10 (14), 2712–2718. https://doi.org/10.1002/pmic.200900473." />
<meta name="citation_reference" content="Jagtap, P.; McGowan, T.; Bandhakavi, S.; Tu, Z. J.; Seymour, S.; Griffin, T. J.; Rudney, J. D. Deep Metaproteomic Analysis of Human Salivary Supernatant. PROTEOMICS 2012, 12 (7), 992–1001. https://doi.org/10.1002/pmic.201100503." />
<meta name="citation_reference" content="Jagtap, P.; Goslinga, J.; Kooren, J. A.; McGowan, T.; Wroblewski, M. S.; Seymour, S. L.; Griffin, T. J. A Two-Step Database Search Method Improves Sensitivity in Peptide Sequence Matches for Metaproteomics and Proteogenomics Studies. PROTEOMICS 2013, 13 (8), 1352–1357. https://doi.org/10.1002/pmic.201200352." />
<meta name="citation_reference" content="Jagtap, P. D.; Johnson, J. E.; Onsongo, G.; Sadler, F. W.; Murray, K.; Wang, Y.; Shenykman, G. M.; Bandhakavi, S.; Smith, L. M.; Griffin, T. J. Flexible and Accessible Workflows for Improved Proteogenomic Analysis Using the Galaxy Framework. J. Proteome Res. 2014, 13 (12), 5898–5908. https://doi.org/10.1021/pr500812t." />
<meta name="citation_reference" content="Kertesz-Farkas, A.; Keich, U.; Noble, W. S. Tandem Mass Spectrum Identification via Cascaded Search. J. Proteome Res. 2015, 14 (8), 3027–3038. https://doi.org/10.1021/pr501173s." />
<meta name="citation_reference" content="Chapman, B.; Bellgard, M. High-Throughput Parallel Proteogenomics: A Bacterial Case Study. PROTEOMICS 2014, 14 (23-24), 2780–2789. https://doi.org/10.1002/pmic.201400185." />
<meta name="citation_reference" content="Dworzanski, J. P.; Snyder, A. P.; Chen, R.; Zhang, H.; Wishart, D.; Li, L. Identification of Bacteria Using Tandem Mass Spectrometry Combined with a Proteome Database and Statistical Scoring. Anal. Chem. 2004, 76 (8), 2355–2366. https://doi.org/10.1021/ac0349781." />
<meta name="citation_reference" content="Dworzanski, J. P.; Snyder, A. P. Classification and Identification of Bacteria Using Mass Spectrometry-Based Proteomics. Expert Rev. Proteomics 2005, 2 (6), 863–878. https://doi.org/10.1586/14789450.2.6.863." />
<meta name="citation_reference" content="Dworzanski, J. P.; Deshpande, S. V.; Chen, R.; Jabbour, R. E.; Snyder, A. P.; Wick, C. H.; Li, L. Mass Spectrometry-Based Proteomics Combined with Bioinformatic Tools for Bacterial Classification. J. Proteome Res. 2006, 5 (1), 76–87. https://doi.org/10.1021/pr050294t." />
<meta name="citation_reference" content="Jabbour, R. E.; Deshpande, S. V.; Wade, M. M.; Stanford, M. F.; Wick, C. H.; Zulich, A. W.; Skowronski, E. W.; Snyder, A. P. Double-Blind Characterization of Non-Genome-Sequenced Bacteria by Mass Spectrometry-Based Proteomics. Appl Env. Microbiol 2010, 76 (11), 3637–3644. https://doi.org/10.1128/AEM.00055-10." />
<meta name="citation_reference" content="Jabbour, R. E.; Deshpande, S. V.; Stanford, M. F.; Wick, C. H.; Zulich, A. W.; Snyder, A. P. A Protein Processing Filter Method for Bacterial Identification by Mass Spectrometry-Based Proteomics. J. Proteome Res. 2011, 10 (2), 907–912. https://doi.org/10.1021/pr101086a." />
<meta name="citation_reference" content="Huson, D. H.; Auch, A. F.; Qi, J.; Schuster, S. C. MEGAN Analysis of Metagenomic Data. Genome Res. 2007, 17 (3), 377–386. https://doi.org/10.1101/gr.5969107." />
<meta name="citation_reference" content="Mesuere, B.; Devreese, B.; Debyser, G.; Aerts, M.; Vandamme, P.; Dawyndt, P. Unipept: Tryptic Peptide-Based Biodiversity Analysis of Metaproteome Samples. J. Proteome Res. 2012, 11 (12), 5773–5780. https://doi.org/10.1021/pr300576s." />
<meta name="citation_reference" content="Muth, T.; Behne, A.; Heyer, R.; Kohrs, F.; Benndorf, D.; Hoffmann, M.; Lehtevä, M.; Reichl, U.; Martens, L.; Rapp, E. The MetaProteomeAnalyzer: A Powerful Open-Source Software Suite for Metaproteomics Data Analysis and Interpretation. J. Proteome Res. 2015, 14 (3), 1557–1565. https://doi.org/10.1021/pr501246w." />
<meta name="citation_reference" content="Muth, T.; Kohrs, F.; Heyer, R.; Benndorf, D.; Rapp, E.; Reichl, U.; Martens, L.; Renard, B. Y. MPA Portable: A Stand-Alone Software Package for Analyzing Metaproteome Samples on the Go. Anal. Chem. 2018, 90 (1), 685–689. https://doi.org/10.1021/acs.analchem.7b03544." />
<meta name="citation_reference" content="Alves, G.; Wang, G.; Ogurtsov, A. Y.; Drake, S. K.; Gucek, M.; Sacks, D. B.; Yu, Y.-K. Rapid Classification and Identification of Multiple Microorganisms with Accurate Statistical Significance via High-Resolution Tandem Mass Spectrometry. J. Am. Soc. Mass Spectrom. 2018, 29 (8), 1721–1737. https://doi.org/10.1007/s13361-018-1986-y." />
<meta name="citation_reference" content="Tracz, D. M.; McCorrister, S. J.; Chong, P. M.; Lee, D. M.; Corbett, C. R.; Westmacott, G. R. A Simple Shotgun Proteomics Method for Rapid Bacterial Identification. J. Microbiol. Methods 2013, 94 (1), 54–57. https://doi.org/10.1016/j.mimet.2013.04.008." />
<meta name="citation_reference" content="Penzlin, A.; Lindner, M. S.; Doellinger, J.; Dabrowski, P. W.; Nitsche, A.; Renard, B. Y. Pipasic: Similarity and Expression Correction for Strain-Level Identification and Quantification in Metaproteomics. Bioinformatics 2014, 30 (12), i149–i156. https://doi.org/10.1093/bioinformatics/btu267." />
<meta name="citation_reference" content="Nesvizhskii, A. I.; Aebersold, R. Interpretation of Shotgun Proteomic Data: The Protein Inference Problem. Mol. Cell. Proteomics 2005, 4 (10), 1419–1440. https://doi.org/10.1074/mcp.R500012-MCP200." />
<meta name="citation_reference" content="Köster, J.; Rahmann, S. Snakemake—a Scalable Bioinformatics Workflow Engine. Bioinformatics 2012, 28 (19), 2520–2522. https://doi.org/10.1093/bioinformatics/bts480." />
<meta name="citation_reference" content="Craig, R.; Beavis, R. C. TANDEM: Matching Proteins with Tandem Mass Spectra. Bioinformatics 2004, 20 (9), 1466–1467. https://doi.org/10.1093/bioinformatics/bth092." />
<meta name="citation_reference" content="Muth, T.; Vaudel, M.; Barsnes, H.; Martens, L.; Sickmann, A. XTandem Parser: An Open-Source Library to Parse and Analyse X!Tandem MS/MS Search Results. PROTEOMICS 2010, 10 (7), 1522–1524. https://doi.org/10.1002/pmic.200900759." />
<meta name="citation_reference" content="Kim, S.; Pevzner, P. A. MS-GF+ Makes Progress towards a Universal Database Search Tool for Proteomics. Nat. Commun. 2014, 5, 5277. https://doi.org/10.1038/ncomms6277." />
<meta name="citation_reference" content="Elias, J. E.; Gygi, S. P. Target-Decoy Search Strategy for Increased Confidence in Large-Scale Protein Identifications by Mass Spectrometry. Nat. Methods 2007, 4 (3), 207. https://doi.org/10.1038/nmeth1019." />
<meta name="citation_reference" content="Alexandre Masselot / InSilicoSpectro-Databanks-0.0.43 - search.cpan.org http://search.cpan.org/~alexmass/InSilicoSpectro-Databanks-0.0.43/ (accessed Sep 29, 2017)." />
<meta name="citation_reference" content="cRAP protein sequences http://www.thegpm.org/crap/ (accessed Sep 29, 2017)." />
<meta name="citation_reference" content="Nesvizhskii, A. I. Proteogenomics: Concepts, Applications and Computational Strategies. Nat. Methods 2014, 11 (11), 1114–1125. https://doi.org/10.1038/nmeth.3144." />
<meta name="citation_reference" content="Wheeler, D. L.; Barrett, T.; Benson, D. A.; Bryant, S. H.; Canese, K.; Chetvernin, V.; Church, D. M.; Dicuccio, M.; Edgar, R.; Federhen, S.; et al. Database Resources of the National Center for Biotechnology Information. Nucleic Acids Res. 2008, 36 (Database issue), D13–21. https://doi.org/10.1093/nar/gkm1000." />
<meta name="citation_reference" content="jsoup Java HTML Parser, with best of DOM, CSS, and jquery https://jsoup.org/ (accessed Sep 29, 2017)." />
<meta name="citation_reference" content="Shen, W.; Le, S.; Li, Y.; Hu, F. SeqKit: A Cross-Platform and Ultrafast Toolkit for FASTA/Q File Manipulation. PLOS ONE 2016, 11 (10), e0163962. https://doi.org/10.1371/journal.pone.0163962." />
<meta name="citation_reference" content="citation_title=Electron Microscopy of Model Systems;citation_author=M. Laue;citation_pages=1-20;citation_volume=96;citation_year=2010" />
<meta name="citation_reference" content="yafeng. An in silico trypsin digestion tool https://github.com/yafeng/trypsin (accessed Sep 29, 2017)." />
<meta name="citation_reference" content="Dent, S. D.; Xia, D.; Wastling, J. M.; Neuman, B. W.; Britton, P.; Maier, H. J. The Proteome of the Infectious Bronchitis Virus Beau-R Virion. J. Gen. Virol. 2015, 96 (12), 3499–3506. https://doi.org/10.1099/jgv.0.000304." />
<meta name="citation_reference" content="Trappe, K.; Wulf, B.; Doellinger, J.; Halbedel, S.; Muth, T.; Renard, B. Y. Hortense: Horizontal Gene Transfer Detection Directly from Proteomic MS/MS Data. 2017. https://doi.org/10.7287/peerj.preprints.3248v1." />
<meta name="twitter:title" content="An iterative and automated computational pipeline for untargeted strain-level identification using MS/MS spectra from pathogenic samples" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Untargeted accurate strain-level classification of a priori unidentified organisms using tandem mass spectrometry is a challenging task. Reference databases often lack taxonomic depth, limiting peptide assignments to the species level. However, the extension with detailed strain information increases runtime and decreases statistical power. In addition, larger databases contain a higher number of similar proteomes.

We present TaxIt, an iterative workflow to address the increasing search space required for MS/MS-based strain-level classification of samples with unknown taxonomic origin. TaxIt first applies reference sequence data for initial identification of species candidates, followed by automated acquisition of relevant strain sequences for low level classification. Furthermore, proteome similarities resulting in ambiguous taxonomic assignments are addressed with an abundance weighting strategy to improve candidate confidence.

We apply our iterative workflow on several samples of bacterial and viral origin. In comparison to non-iterative approaches using unique peptides or advanced abundance correction, TaxIt identifies microbial strains correctly in all examples presented (with one tie), thereby demonstrating the potential for untargeted and deeper taxonomic classification. TaxIt makes extensive use of public, unrestricted and continuously growing sequence resources such as the NCBI databases and is available under open-source license at &lt;https://gitlab.com/rki_bioinformatics&gt;.

*   FDR
    :   false discovery rate
    LC
    :   liquid chromatography
    MALDI
    :   matrix-assisted laser desorption/ionization
    MS
    :   mass spectrometry
    PSM
    :   peptide spectrum match
    RSS
    :   resident set size
    TOF
    :   time-of-flight" />
<meta name="og-title" property="og:title" content="An iterative and automated computational pipeline for untargeted strain-level identification using MS/MS spectra from pathogenic samples" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/812313v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Untargeted accurate strain-level classification of a priori unidentified organisms using tandem mass spectrometry is a challenging task. Reference databases often lack taxonomic depth, limiting peptide assignments to the species level. However, the extension with detailed strain information increases runtime and decreases statistical power. In addition, larger databases contain a higher number of similar proteomes.

We present TaxIt, an iterative workflow to address the increasing search space required for MS/MS-based strain-level classification of samples with unknown taxonomic origin. TaxIt first applies reference sequence data for initial identification of species candidates, followed by automated acquisition of relevant strain sequences for low level classification. Furthermore, proteome similarities resulting in ambiguous taxonomic assignments are addressed with an abundance weighting strategy to improve candidate confidence.

We apply our iterative workflow on several samples of bacterial and viral origin. In comparison to non-iterative approaches using unique peptides or advanced abundance correction, TaxIt identifies microbial strains correctly in all examples presented (with one tie), thereby demonstrating the potential for untargeted and deeper taxonomic classification. TaxIt makes extensive use of public, unrestricted and continuously growing sequence resources such as the NCBI databases and is available under open-source license at &lt;https://gitlab.com/rki_bioinformatics&gt;.

*   FDR
    :   false discovery rate
    LC
    :   liquid chromatography
    MALDI
    :   matrix-assisted laser desorption/ionization
    MS
    :   mass spectrometry
    PSM
    :   peptide spectrum match
    RSS
    :   resident set size
    TOF
    :   time-of-flight" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-10-24" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/812313v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/812313v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/812313v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/812313v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/964570" />
    <title>An iterative and automated computational pipeline for untargeted strain-level identification using MS/MS spectra from pathogenic samples | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"bb4tvY_G0u22SWc90g9EjI0XNH8w7U54OYzrzf-_BmY","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"964570","apath":"\/biorxiv\/early\/2019\/10\/24\/812313.atom","pisa":"biorxiv;812313v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;812313v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/812313v1","title":"An iterative and automated computational pipeline for untargeted strain-level identification using MS\/MS spectra from pathogenic samples","identifier":"node\/964570"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022t1j4l1CrZFxw8c7e5+xzkwm12geGznatNsgDTvGY4oLod1xbEPP+CtAafMVu4urfeCOVvEf4M38eSJWYwTMwcJaQOKG7o9cJzeJ56MPHThcW4hKr57mzbkzlLOWFTT3bst5r23hEBygBkNljPBFKXUMrZHUZJbLl1zCmO3+Li47rPxFn7PvMfhh82xPy6c75qSfIUXEpJTKzz7c1HattCG7lkPy\\\\\\\/ySNLSXdR4yi9juBb6ptnwAQxsyZPe6rgjTHC1zbwP7C7gHvtsyqQCCnKpqJGk62\\\\\\\/+x5FeuYMlsiFe5ixy6FN48F7NrOyOwkhaliC6JMa+QNYL3NuaOf\\\\\\\/UG6L37s7EvrpvKM\\\\\\\/bZaiDLP9mmuYOLozta2qumxB4tys8glT9Etzywyh5Q8ACffptoxaKaQu6F7HFx7uG1OvFr\\\\\\\/bSu\\\\\\\/ZM4+coKUM8Sp9aQMEvyExboGIcv3tjwE6rjCBT2oa7\\\\\\\/uV\\\\\\\/4fSZblNsX2XS4nnj\\\\\\\/vRxjhykGQuZs7g6REDwvGFSxGuHufM0LZ7C7setxTenOCJHbND4mZ+hmMIAL5fsgftDwBTXNVsRvLGGOtOWucJ\\\\\\\/ViIc4dfmKKVbUoGUXeAtECiYobQ5th5jvVWgNkfUi6EaDqAln18iUafwNb8zBdPXSbv5C0wo2ijUHcxJBgd8b3tAZilE\\\\\\\/P+TftyYykTNV7MQu5oxIfXSqrSfg1zFEvdH1KmSXWwced8tutOF4JQ7cjl1oizhr9tY\\\\\\\/gn8oz2XOlL4bQTtZe2AXTPjWmmLCIwjrWpWHI3cqL\\\\\\\/OBlKw8oHp+C+BvgSxec1aY66zCkXUavhI5IfbrP3aIUx1WldaOHSfu5Vqjp2\\\\\\\/Hgow6wi3nwW93yMmZKLLNa\\\\\\\/eMwgA30Qzqipq61Iu\\\\\\\/Ia\\\\\\\/Bwh264+VzcJL3SoNWtBrPB4saXaow8GMXunrSohnPH5pFySmXyT\\\\\\\/g9dQIGeZ4VaJGAcOb+0qg\\\\\\\/TRUTJWK4PulgFWNPLRID9A+wH6zyqunWEhoHod8TZeaP3HeiIfKa2\\\\\\\/df7kBknbk+oqYSLi8XqtIZ3bnLxKXjtwXia8N8SFi3ThqjxWo5D+U8HkDx73j2hBxIsDSCQhU7Q5409pxa1G2qAy\\\\\\\/NCwkL50pePABoawSuqmRXxzdKQMFIGYwQ4\\\\\\\/Ft2B8foQsLwEFF7dIOkTrRhZBrVDKc1kzVHzg5r+zdIuDdvHCoWAVW0vnYD\\\\\\\/GHxTekrenWT1UkJCAFpO4YnKcKiyvaMhRCNUSxVwTCqTw381\\\\\\\/wRATbE5J8nhUUSUvuJ1YMH31LqH\\\\\\\/pM0e0UKREfGQ4wn1NRFq1ZFxleIK9KS2DU2pUdwRIysHDGJEU=\\\u0022,\\\u0022iv\\\u0022:\\\u0022IOLKySTQrSsh+FRcWzWDVw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002231c92e338b219ca325951834817a3bdb0bbe18dc8499d6f8efde8b59d9738c7b\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/812313v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-964570 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/812313v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NyEDZ2c6IWAjJ_fL8emLZGKuiNCu2145HIh7iO59ttI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="964570" id="node964570" data-pisa="biorxiv;812313v1" data-pisa-master="biorxiv;812313" data-apath="/biorxiv/early/2019/10/24/812313.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">An iterative and automated computational pipeline for untargeted strain-level identification using MS/MS spectra from pathogenic samples</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3287-0313" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Mathias</span> <span class="nlm-surname">Kuhring</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8309-082X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Joerg</span> <span class="nlm-surname">Doellinger</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Andreas</span> <span class="nlm-surname">Nitsche</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-8304-2684" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Thilo</span> <span class="nlm-surname">Muth</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4589-9809" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Bernhard Y.</span> <span class="nlm-surname">Renard</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/812313 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node964570" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mathias Kuhring </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Bioinformatics Unit (MF 1), Department for Methods Development and Research Infrastructure, Robert Koch Institute</span>, Berlin, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Core Unit Bioinformatics, Berlin Institute of Health (BIH)</span>, Berlin, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Berlin Institute of Health Metabolomics Platform, Berlin Institute of Health (BIH)</span>, Berlin, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Max Delbrück Center (MDC) for Molecular Medicine</span>, Berlin, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathias%2BKuhring%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kuhring%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMathias%2BKuhring%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3287-0313" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mathias Kuhring</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joerg Doellinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Centre for Biological Threats and Special Pathogens, Proteomics and Spectroscopy (ZBS 6), Robert Koch Institute</span>, Berlin, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Centre for Biological Threats and Special Pathogens, Highly Pathogenic Viruses (ZBS1), Robert Koch Institute</span>, Berlin, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joerg%2BDoellinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Doellinger%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoerg%2BDoellinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8309-082X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joerg Doellinger</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andreas Nitsche </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Centre for Biological Threats and Special Pathogens, Highly Pathogenic Viruses (ZBS1), Robert Koch Institute</span>, Berlin, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andreas%2BNitsche%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nitsche%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndreas%2BNitsche%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Thilo Muth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Bioinformatics Unit (MF 1), Department for Methods Development and Research Infrastructure, Robert Koch Institute</span>, Berlin, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thilo%2BMuth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Muth%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThilo%2BMuth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8304-2684" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thilo Muth</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Bernhard Y. Renard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Bioinformatics Unit (MF 1), Department for Methods Development and Research Infrastructure, Robert Koch Institute</span>, Berlin, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernhard%2BY.%2BRenard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Renard%20BY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABernhard%2BY.%2BRenard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4589-9809" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bernhard Y. Renard</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:renardb@rki.de" class="" data-icon-position="" data-hide-link-title="0">renardb@rki.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/812313v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:964570" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:964570/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/812313v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:964570" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:964570/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/812313v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:964570" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:964570/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/812313v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:964570" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:964570/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/812313v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:964570" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:964570/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/812313v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:964570" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:964570/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Untargeted accurate strain-level classification of a priori unidentified organisms using tandem mass spectrometry is a challenging task. Reference databases often lack taxonomic depth, limiting peptide assignments to the species level. However, the extension with detailed strain information increases runtime and decreases statistical power. In addition, larger databases contain a higher number of similar proteomes.</p><p id="p-3">We present TaxIt, an iterative workflow to address the increasing search space required for MS/MS-based strain-level classification of samples with unknown taxonomic origin. TaxIt first applies reference sequence data for initial identification of species candidates, followed by automated acquisition of relevant strain sequences for low level classification. Furthermore, proteome similarities resulting in ambiguous taxonomic assignments are addressed with an abundance weighting strategy to improve candidate confidence.</p><p id="p-4">We apply our iterative workflow on several samples of bacterial and viral origin. In comparison to non-iterative approaches using unique peptides or advanced abundance correction, TaxIt identifies microbial strains correctly in all examples presented (with one tie), thereby demonstrating the potential for untargeted and deeper taxonomic classification. TaxIt makes extensive use of public, unrestricted and continuously growing sequence resources such as the NCBI databases and is available under open-source license at <a href="https://gitlab.com/rki_bioinformatics">https://gitlab.com/rki_bioinformatics</a>.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-5">Pathogenic strain identification using LC-MS/MS-based proteomics presents a crucial, yet highly challenging task. Many pathogenic strains feature significant phenotypic differences within a species with respect to pathogenicity, zoonotic potential, cell attachment and entry, host-virus interaction and clinical symptoms <sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup>. In a diagnostic context, strain-level knowledge is important to infer virulence <sup><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup> and drug resistance <sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup> for appropriate therapy. However, inferring exact strain information from proteomic samples remains a difficult task, in particular when the taxonomic origin of a sample is unknown and when related strains feature high sequence similarity.</p><p id="p-6">In recent years, MALDI-TOF mass spectrometry has gained popularity as fast, sensitive and economical method for microbial biotyping. However, identifying strains using MALDI-TOF workflows is still very challenging and requires curated, often proprietary spectral databases <sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. Several commercial platforms for microbial biotyping down to the species or strain level are available based on MALDI and other technologies such as the Bruker MALDI Biotyper Systems <sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup>, the Bruker Strain typing with IR Biotyper <sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup> and the Ibis T5000 Universal Biosensor <sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>.</p><p id="p-7">Several studies report on limitations of MALDI-TOF biotyping for strain-level identifications and advocate advancements towards MS/MS marker peptide detection and, consequently, the analysis was shifted to the MS/MS level <sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>. In these studies, however, the MS/MS-based protein identification was established using sequence databases that were already targeted or restricted to particular species or limited sets. In contrast, untargeted MS/MS typing approaches are limited to species level identification <sup><a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>,<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>. However, in general MS/MS is preferred for the analysis of complex unpurified peptide mixtures as it is considered to provide more distinct and unambiguous peptide and protein identifications <sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup> and thus increased proteome resolution <sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup> as well as higher statistical confidence <sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>. In particular, organisms with unknown taxonomic origin benefit from peptide sequence-based analysis, as MALDI-based biotyping is in comparison too prone to ambiguous identifications <sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>. Furthermore, advances in instrumentation including higher resolution, mass accuracy and dynamic range increasingly allow for identification of the majority of all fragmented peptides <sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup> resulting in higher sensitivity, higher coverage of target proteomes and thus higher availability of distinctive features.</p><p id="p-8">Taking advantage of the vast amount of available protein sequences for MS/MS strain-level identification is challenging. On the one hand, constraining the search space may result in unidentified strains or incorrectly assigned taxa, in particular for non-model organisms <sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>. On the other hand, applying large databases is not recommended either since it decreases peptide identification rates <sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup> and thus eventually impedes taxonomic inference <sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>. Furthermore, with increasing database size sequence quality often decreases (e.g. when using the complete NCBI Protein in comparison to the NCBI RefSeq database) and contaminant sequences may occur more often <sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>. Therefore, extended databases should only be used when necessary. However, strain-level identification of MS/MS spectra from samples with unclear taxonomic status requires an untargeted search against comprehensive databases holding as many strains as possible.</p><p id="p-9">A common and popular concept to handle increased search spaces is the application of multiple identification steps in general, independently of target application such as strain-level identification. These multi-stage identification search strategies are described by several different terms such as serial search <sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup>, multi-step <sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup>, iterative <sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>,<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>, multi-stage <sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup>, two-step <sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>–<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup> as well as cascade search <sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup> and they find application in proteomics, metaproteomics <sup><a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>,<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a></sup> and proteogenomics <sup><a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>,<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>. Most of these strategies do not only overlap in their objective of increasing the identification rate or identification confidence, but share methodological principles as well. This includes the concept of identifying primarily unassigned spectra using databases of increasing complexity (for instance, by employing altered digestion parameters, additional post-translational modifications or additional spectral and genomic databases) <sup><a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>–<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>,<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a></sup> as well as the recurring theme of database size reduction <sup><a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">26</a>,<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">28</a>–<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">30</a>,<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a></sup>. In addition, some methods rely on spectral quality assessment to enhance subsequent identification steps <sup><a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>,<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">32</a></sup> or exhibit a focus on algorithmic runtime reduction <sup><a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">24</a></sup>.</p><p id="p-10">Apart from database size, multi-proteome databases present an additional challenge for taxonomic assignment in the form of high sequence similarity between proteomes, in particular between related species and strains. These similarities give rise to taxonomically ambiguous, widespread and false assignments and thus need to be accounted for. Several methods exist that provide various strategies to account for ambiguous peptide spectrum matches due to sequence similarity. Dworzanski et al. apply a proteogenomic mapping approach in combination with discriminant analysis to infer most likely bacterial assignments <sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup>. The proteogenomic approach was further developed in BACid that applies several statistical measures to account for similarity, including the comparison of ratios of taxonomic differences and of unique peptides to known error rates, i.e. the noise levels <sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>,<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>. BACid has since been applied to and extended for samples of unknown origin <sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup> as well as mixtures of bacteria <sup><a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>,<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup>. With the focus on metaproteomic analysis, MEGAN <sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup>, UniPept <sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup>, MetaProteomeAnalyzer <sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>,<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup> and TCUP <sup><a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a></sup> rely on lowest common ancestor or most specific taxonomy approaches to assign peptides spectrum matches to taxonomic levels. MiCId uses a clustering approach based on peptidome similarity of taxa on different levels in combination with unified E-values to infer statistically significant representatives of clusters as identified or classified microorganisms <sup><a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup>. Tracz <em>et al.</em> use a direct assignment strategy by concatenating all proteins of a proteome into one pseudo-polyprotein and considering only the top-scored spectrum matches to a peptide as counts for bacterial candidates <sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup>. In contrast, Pipasic explicitly makes use of ambiguous peptide spectrum matches (PSMs) by applying abundance correction based on intensity-weighted proteome similarities of organisms in metaproteomic samples <sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup>. While these methods share the objective of reducing uncertainty in taxonomic identification due to ambiguous assignments in the context of the protein inference issue <sup><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup>, their application for untargeted strain-level identification is not straightforward for several reasons. For example, unique peptides are not necessarily available for all candidate strains in the sequence databases for which the information given is often limited to species or higher taxonomic levels. In addition, the approaches rely on fully sequenced genomes (and thus uniform proteome coverage) or other forms of targeted, curated, inhouse or validated databases and are thus restricted to a predefined limited set of organisms, often with definite focus on bacteria. Furthermore, some methods do not scale well with immoderate large databases (i.e. high number of proteomes) with respect to computational or analytical performance.</p><p id="p-11">Previous multi-step procedures illustrate the effect of database size on identification confidence and the advantage of applying concise, prefiltered or specialized databases. Thus, we transfer the general concept of multi-step procedures to approach the increased search space necessary for untargeted and detailed taxonomic classification. We present TaxIt, an iterative workflow for untargeted strain-level identification of protein samples. By applying two separate identification steps for species- and strain-level classification, we circumvent the immediate need for a comprehensive database containing strain proteome sequences. Thereby, a first untargeted search allows the selection of a relevant species and enables to focus a second search on a highly reduced but adequate choice of automatically downloaded strain proteomes, resulting in increased identification confidence and reduced taxonomic ambiguity. Moreover, strain-level identification relies on recent data, without the need of keeping a local comprehensive strain database up-to-date. The workflow takes advantage of a free, publicly available and continuously growing protein sequence resource (NCBI Protein) and is thus suitable for most common established MS experimental workflows.</p></div><div class="section" id="sec-2"><h2 class="">Material and Methods</h2><p id="p-12">The TaxIt workflow for strain identification consists of several recurring modules interconnected, controlled (in terms of input and output) and automatically executed by the Snakemake computational workflow management system <sup><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup>. The designed workflow executes up to three stages including (i) an optional host filter, (ii) species identification (primary iteration) and (iii) strain identification (secondary iteration). A concise overview of the workflow is illustrated in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. The main steps of each iteration are the execution of a peptide search engine, false discovery rate (FDR) control, taxonomic classification, taxa counting and adjustment as well as candidate selection and visualization. The automated download of strain proteomes bridges the primary and secondary iteration. The procedures of the main modules are described in detail below.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of the TaxIt workflow. In up to three stages, MS/MS spectra are searched against a host proteome to pre-filter host proteins (optional), against an untargeted database for global species identification (primary iteration) and against a targeted, species-based and automatically fetched database for strain level identification (secondary iteration)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1555425841" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Overview of the TaxIt workflow.&lt;/span&gt; In up to three stages, MS/MS spectra are searched against a host proteome to pre-filter host proteins (optional), against an untargeted database for global species identification (primary iteration) and against a targeted, species-based and automatically fetched database for strain level identification (secondary iteration).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F1.medium.gif" width="440" height="116"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F1.medium.gif" width="440" height="116"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Overview of the TaxIt workflow.</span><p id="p-13" class="first-child">In up to three stages, MS/MS spectra are searched against a host proteome to pre-filter host proteins (optional), against an untargeted database for global species identification (primary iteration) and against a targeted, species-based and automatically fetched database for strain level identification (secondary iteration).</p><div class="sb-div caption-clear"></div></div></div><div id="sec-3" class="subsection"><h3>Peptide Search</h3><p id="p-14">The central step in the analysis of MS/MS spectra is an efficient peptide search for comprehensive protein identification. Therefore, we rely on established and reliable open-source database search engines such as X!Tandem <sup><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></sup> in combination with the XTandem Parser <sup><a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a></sup> and MS-GF+ <sup><a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a></sup>. However, any command-line search engine including proprietary ones could be implemented via additional Snakemake rules. We apply a classic target-decoy approach for false discovery rate (FDR) control <sup><a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a></sup>. Decoy sequences are created upfront and independently of the search engine with fasta-decoy.pl <sup><a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a></sup> by reversing the target sequences (including contaminant proteins, for instance cRAP <sup><a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a></sup>) and both target and decoy databases are concatenated as suggested by Jeong et al. <sup><a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a></sup>. PSMs are subjected to an FDR cutoff based on a per-match FDR calculated as N<sub>decoy</sub>/N<sub>target</sub>, with N<sub>decoy</sub> being the number of decoys in between targets (N<sub>target</sub>) in a list of matches sorted by e-value <sup><a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a></sup>. To acknowledge established false positive hits of previous iterations, decoy sequences with an e-value above the FDR-cutoff are passed on and concatenated to the database of the next iteration.</p></div><div id="sec-4" class="subsection"><h3>Taxonomic Classification</h3><p id="p-15">We make use of the NCBI Taxonomy to assign PSMs to corresponding taxa and redistribute shared hits introduced by proteins associated with higher/upper taxonomic levels such as genus. First, NCBI protein accessions are mapped to NCBI taxids using the NCBI protein id mapping file (<a href="ftp://ftp.ncbi.nih.gov/pub/taxonomy/accession2taxid/prot.accession2taxid.gz">ftp.ncbi.nih.gov/pub/taxonomy/accession2taxid/prot.accession2taxid.gz</a>). Next, taxonomic relations are inferred from the NCBI Taxonomy nodes dump file (<a href="ftp://ftp.ncbi.nih.gov/pub/taxonomy/taxdump.tar.gz">ftp.ncbi.nih.gov/pub/taxonomy/taxdump.tar.gz</a>) and PSMs are reassigned to a taxonomic level based on the objective of the current iteration. In general, PSMs assigned to higher taxa such as genus are propagated as shared hits to corresponding leaf taxa as long as the target leaves already exhibit matches on their own. In addition, in the first iteration, PSMs are summarized at the species level.</p></div><div id="sec-5" class="subsection"><h3>Count Adjustment</h3><p id="p-16">For each candidate taxon (i.e. species in primary and strains in secondary iteration, respectively), raw counts are calculated by summing over all assigned PSMs including non-unique matches. To account for taxonomic biases caused by shared matches, we integrate a weighting scheme based on the level of uniqueness using the global frequency of a PSM. Here, a PSM count per taxon is adjusted by the number of occurrences in all candidate taxa. Thereby, unique PSMs gain in value for the final taxa selection without fully neglecting the importance of high numbers of non-unique matches that are often highly present within closely related taxa such as strains.</p></div><div id="sec-6" class="subsection"><h3>Selection and Downloads</h3><p id="p-17">After count adjustment, the most dominant candidate taxon is selected as the most likely species or strain, respectively. In the final step of the first iteration, the selected species is used to retrieve strain-level data for the strain identification in the second iteration. Once more, we rely on the NCBI Taxonomy and infer all available strains for the candidate species via the nodes dump file. Next, strain proteins are automatically downloaded from NCBI Protein using the NCBI Entrez API <sup><a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a></sup> in combination with the jsoup: Java HTML Parser <sup><a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a></sup>. This includes all available RefSeq as well as non-RefSeq sequences since the availability of curated strain material is often limited. Finally, the obtained protein sequences are merged into a single database and redundant entries are removed using seqkit rmdup <sup><a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a></sup>.</p></div><div id="sec-7" class="subsection"><h3>Benchmarking Experiments</h3><p id="p-18">For benchmarking, we compare TaxIt against classic comprehensive search strategies based on straight non-iterative taxonomic identification supported by unique PSMs or abundance similarity correction as provided by Pipasic <sup><a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a></sup>.</p><p id="p-19">TaxIt uses NCBI RefSeq proteins of selected kingdoms as reference databases for initial species identification followed by automated and selective strain protein incorporation. Unique-PSMs- and Pipasic-based strategies, however, apply comprehensive databases integrating as many strain-level sequences as possible at once, including all protein sequences from the NCBI Protein database for selected kingdoms. In general, a preselection of kingdoms may be justified by clinical findings based on, for instance, symptoms or microscopic examination <sup><a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a></sup>. Both, unique-PSMs- and Pipasic-based strategies use the same procedures for peptide search, FDR control and taxonomic classification as described in the iterative workflow. However, PSMs are not summarized at species level and counts are directly inferred at the lowest possible taxonomic level. For the unique-PSMs-based strategy, adjusted counts are based on PSMs that occur only once. In other words, only spectra assigned unambiguously to only one peptide sequence and thus organism are taken into account. The abundance similarity correction of Pipasic uses the similarity of expressed proteomes between taxa to account for attribution biases. Originally intended for metaproteomic abundance correction, it is here applied to highlight the most likely strain within a provided sample.</p><p id="p-20">Since Pipasic is sensitive to a high number of taxa, we limit the input to taxa with a minimum of two hits as well as to the 100 most abundant taxa. Expressed proteins per taxa are extracted according to the taxonomic classification and digested peptides with a minimum length of six amino acids are prepared using trypsin digestion <sup><a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a></sup>. PSMs and tryptic peptides are then passed to Pipasic to obtain corrected relative counts.</p><p id="p-21">We perform all three strategies on several viral and bacterial MS/MS spectra samples with available strain-level knowledge. This includes a <em>Cowpox virus (Brighton Red)</em> strain (acquired in-house), an <em>Avian infectious bronchitis virus (strain Beaudette CK)</em> <sup><a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a></sup> and a <em>Bacillus subtilis BSN238</em> <sup><a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a></sup> that we will refer to as cowpox, bronchitis and bacillus sample, respectively. <em>B. subtilis BSN238</em> is a transgenic organism resulting from horizontal gene transfer (HGT) of the DivIVA protein from <em>Listeria monocytogenes strain EGD-e</em> to <em>Bacillus subtilis</em> subsp<em>. subtilis str. 168</em>. Since <em>B. subtilis BSN238</em> is neither yet present in the NCBI Taxonomy nor in the NCBI Protein database and only one protein is modified, we expect <em>B. subtilis</em> subsp<em>. subtilis str. 168</em> to be selected as final strain candidate. Bacillus samples are examined twice, once completely with 28902 spectra (bacillus all) and once randomly reduced to 1000 spectra (bacillus 1k) to improve performance with respect to the vast bacterial search space. A detailed description of the cowpox sample acquisition and the search parameters for all samples is provided in Supplementary item 1.</p><p id="p-22">For the viral samples we used all viral NCBI RefSeq proteins (314156 entries, via Entrez on July 10, 2017) in the iterative workflow and additionally all viral non-RefSeq proteins (1987275 entries in total, via Entrez on July 11, 2017) for the unique-PSMs- and Pipasic-based strategies. Viral spectra were filtered beforehand using corresponding host proteomes (all isoforms) including the UniProt <em>Homo sapiens</em> reference proteome (93588 entries, UP000005640, May 23, 2017) for the cowpox sample and <em>Gallus gallus Red jungle fowl</em> reference proteome (29732 entries, UP000000539, May 16, 2017) for the bronchitis sample. For the iterative examination of the bacillus samples, all bacterial NCBI RefSeq proteins (65996005 entries, release 82) were downloaded via FTP (<a href="ftp://ftp.ncbi.nlm.nih.gov/refseq/release/bacteria/">ftp.ncbi.nlm.nih.gov/refseq/release/bacteria/</a>). Since downloading all NCBI Protein entries for bacteria (taxid 2) is impractical and requires too much time using the Entrez API, we used the NCBI Blast NR database as most comprehensive, common and readily available protein resource. This database can be obtained in fasta format via FTP (<a href="ftp://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/">ftp.ncbi.nlm.nih.gov/blast/db/FASTA/</a>, release 19.07.2017) and the bacteria subset (93760438 entries) was extracted with in-house scripts to create the databases for the unique-PSMs- and Pipasic-based strategies.</p></div></div><div class="section" id="sec-8"><h2 class="">Results</h2><p id="p-23">To demonstrate the potential of iterative strain-level identification, we compared TaxIt against classic comprehensive search strategies based on non-iterative taxonomic identification supported by either unique PSMs or the abundance similarity correction of Pipasic <sup><a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">44</a></sup>. The final selections of the top taxa candidates for all samples and all three compared identification strategies are summarized in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. For the cowpox sample, identification results agree the most. TaxIt (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) and Pipasic (<strong>Supplementary item 1 - Figure S1</strong>) are both able to identify the expected <em>Cowpox virus (Brighton Red)</em> strain. However, unique PSMs are limited to the parent <em>Cowpox virus</em> species and not available at the strain level. For this reason, an incorrect identification of <em>Bat astrovirus Hil GX bszt12</em> could occur based on a single unique PSM (<strong>Supplementary item 1 - Figure S1</strong>). The results also show that the original TaxIt counts are not an appropriate measure to distinguish the expected strain from competing candidates (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). However, the weight-based correction method implemented in TaxIt resolves the present tie and emphasizes the correct <em>Cowpox virus</em> strain (<em>Brighton Red)</em>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/967366/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/967366/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-24" class="first-child">Expected and identified taxa per sample and strategy.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative counts of cowpox sample analysis. Relative counts are illustrated as result of the TaxIt workflow. Original and weighted relative counts are summarized by means of one vertical stacked bar each. Candidate strains are labeled and color-coded and ratios highlighted as percentages within bars." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1555425841" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Relative counts of cowpox sample analysis.&lt;/span&gt; Relative counts are illustrated as result of the TaxIt workflow. Original and weighted relative counts are summarized by means of one vertical stacked bar each. Candidate strains are labeled and color-coded and ratios highlighted as percentages within bars.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F2.medium.gif" width="440" height="56"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F2.medium.gif" width="440" height="56"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Relative counts of cowpox sample analysis.</span><p id="p-25" class="first-child">Relative counts are illustrated as result of the TaxIt workflow. Original and weighted relative counts are summarized by means of one vertical stacked bar each. Candidate strains are labeled and color-coded and ratios highlighted as percentages within bars.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">In comparison, bronchitis and bacillus samples feature notable variability in proposed taxa candidates. For bronchitis, TaxIt could identify the expected <em>Avian infectious bronchitis virus (strain Beaudette CK)</em> strain. Despite applying a small correction based on weighting, however, final candidate selection is not limited to a single strain but additionally includes a closely related strain, namely <em>Avian infectious bronchitis virus (strain Beaudette US)</em> (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). Nevertheless, TaxIt provides a highly constrained selection of candidates in the first place. In contrast, the Pipasic-based strategy results in considerably more initial candidates and eventually promotes an incorrect strain, namely <em>Avian infectious bronchitis virus (strain 6/82)</em> (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). Since no unique PSMs are available for the bronchitis sample, strain-level identification was not possible with this strategy.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative counts of bronchitis sample analysis. Relative counts are illustrated as result of the TaxIt (A) or Pipasic approach (B), respectively. TaxIt&#x2019;s original and weighted relative counts as well as Pipasic&#x2019;s original, filtered (a minimum of two hits and only the most 100 abundant taxa) and corrected relative counts are summarized by means of one vertical stacked bar each. Candidate strains are labeled and color-coded and ratios highlighted as percentages within bars." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1555425841" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Relative counts of bronchitis sample analysis.&lt;/span&gt; Relative counts are illustrated as result of the TaxIt (A) or Pipasic approach (B), respectively. TaxIt&#x2019;s original and weighted relative counts as well as Pipasic&#x2019;s original, filtered (a minimum of two hits and only the most 100 abundant taxa) and corrected relative counts are summarized by means of one vertical stacked bar each. Candidate strains are labeled and color-coded and ratios highlighted as percentages within bars.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F3.medium.gif" width="440" height="188"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F3.medium.gif" width="440" height="188"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Relative counts of bronchitis sample analysis.</span><p id="p-27" class="first-child">Relative counts are illustrated as result of the TaxIt (A) or Pipasic approach (B), respectively. TaxIt’s original and weighted relative counts as well as Pipasic’s original, filtered (a minimum of two hits and only the most 100 abundant taxa) and corrected relative counts are summarized by means of one vertical stacked bar each. Candidate strains are labeled and color-coded and ratios highlighted as percentages within bars.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Runtime and memory benchmarks. Total runtime in minutes and maximum memory usage in gigabyte is represented per sample and color-coded identification strategy." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1555425841" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Runtime and memory benchmarks.&lt;/span&gt; Total runtime in minutes and maximum memory usage in gigabyte is represented per sample and color-coded identification strategy.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F4.medium.gif" width="440" height="306"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F4.medium.gif" width="440" height="306"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/812313/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Runtime and memory benchmarks.</span><p id="p-28" class="first-child">Total runtime in minutes and maximum memory usage in gigabyte is represented per sample and color-coded identification strategy.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Using TaxIt, the expected <em>B. subtilis str. 168</em> strain is identified correctly in the reduced as well as in the complete sample (<strong>Supplementary item 1 - Figure S3 and S4</strong>). Pipasic consistently misses the correct strain and species in favor of incorrect strains such as <em>Bacillus cereus SJ1</em> and <em>Bacillus cereus B4264</em>. The unique-PSMs-based strategy includes the correct strain <em>B. subtilis str. 168</em> into the final candidate list of the bacillus 1k sample. However, it fails to separate the strain from several distinct species due to equal amounts of unique PSMs (<strong>Supplementary item 1 - Figure S3</strong>). Furthermore, the data analysis on the complete bacillus sample results in the incorrect species <em>Paenisporosarcina quisquiliarum</em> being predominantly present in terms of uniquely assigned PSMs.</p><p id="p-30">Besides identification efficiency, computational performance may delimit the potential application as routine or expeditious method in research or clinics. Therefore, we compared runtime and memory consumption for all samples and strategies (as illustrated in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <strong>Figure 5</strong>). Analysis was performed with X!Tandem as database search engine and limited to 24 threads on a server with Debian GNU/Linux 8.9 (jessie), 64 cores (128 threads) of type Intel(R) Xeon(R) CPU E5-4667 v4 @ 2.20 GHz, 512 GB of RAM and SSD storage. Applying the iterative approach reduces memory usage down to one third for viral strain identification and two third for bacterial strain identification. While the runtime of Pipasic is comparably high, TaxIt shows no substantial change in runtime in comparison to the unique-PSMs-based strategy for small databases such as the collective viral sequences or small sample sizes. However, analyzing the full bacillus sample reveals an increased runtime when using NCBI RefSeq proteins plus selected strain proteomes instead of the extensive NCBI Blast NR database.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/967363/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/967363/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/967363" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-31" class="first-child">Runtime and maximal memory consumption (resident set size, RSS) per identification strategy and sample.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-9"><h2 class="">Discussion</h2><p id="p-32">TaxIt demonstrates superior performance with respect to strain-level identification and computational expense. In summary, our approach can be used to unambiguously identify candidate organisms of all samples down to a low taxonomic level, with the minor exception of a tie for the bronchitis sample. In contrast, the unique-PSMs-based identification strategy is repeatedly deficient at the strain level or features highly ambivalent results. Furthermore, Pipasic frequently favors an incorrect strain or even an incorrect species. For some samples, correct strains are observed as a top candidate even in the original counts, independently of database choice and prior to count adjustments. Nevertheless, all samples benefit either from the iterative and focused database usage, the count adjustment procedure or the reduced resource consumption while the outcome remains legitimate.</p><p id="p-33">It was shown that TaxIt improves taxonomic assignment already in the count-based ranking, independently of count adjustments, by limiting candidates to strains of a single species early on (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). For instance, the search of bronchitis samples against the NCBI Blast NR bacteria subspace results in a rather uniform distribution of original counts for <em>Infectious bronchitis virus</em> strains: the strains Beaudette, Beaudette CK and Beaudette UK are only slightly increased in comparison to other strains (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>, <strong>Original</strong>). In contrast, the iterative approach results in less strain candidates of the same species focusing solely on Beaudette strains when selectively searching against <em>Avian coronavirus</em> strains in the secondary iteration (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a>, <strong>Original</strong>). This is a consequence of how and whether parental or multispecies proteins are associated with strains or taxa in general within the NCBI taxonomy. In case of the iterative approach, fewer mutual species or genus proteins are consulted in the strain identification iteration. However, the extent of this effect varies between samples, candidate strains or target databases and taxonomies, respectively. For instance, bacterial strain proteomes such as the <em>Bacillus subtilis</em> strains feature numerous directly assigned mutual protein sequences and thus result in an extended range of candidate strains of the same species (<strong>Supplementary item 1 - Figure S3 and S4</strong>).</p><p id="p-34">Nevertheless, the restriction to a specific set of strain proteomes prevents manifold primary mis-assignments to distant species, genera or even phyla as can be observed for Pipasic- and unique-PSMs-based original counts, respectively, that cannot be sufficiently resolved after correction (<strong>Supplementary item 1 - Figure S3 and S4</strong>). In general, the iterative and selective database usage ensures that final strain selection is limited to strain candidates of an appropriate species. Thus, it prevents false positive hits on distinct strains of other taxa including species, genera and phyla and allows for a more confident final strain candidate selection.</p><p id="p-35">Furthermore, uniform distributions and even consensus in original counts of strain candidates demonstrate the need and benefit of count adjustment methods. The implemented weighting procedure can resolve ties between strains such as in the TaxIt cowpox sample analysis (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>) or at least amplifies the correct strain and increases the distance to competing candidates.</p><p id="p-36">TaxIt infers exactly one strain for the presented samples each expect for the bronchitis sample where the strains Beaudette CK and Beaudette UK cannot be differentiated. We observed that the corresponding PSMs are fully shared between the two proteomes. Although different proteins are available for each strain in general, peptide hits are either assigned to shared proteins of the parent strain <em>Infectious bronchitis virus</em> or to homologous proteins that differ only in identifier but not in sequence. Though a more granular taxonomic relation cannot be ascertained from the NCBI taxonomy, we expect the Beaudette strains to feature a considerably closer relationship as compared to other <em>Infectious bronchitis virus</em> strains. Therefore, we consider the draw between Beaudette strains as sufficiently appropriate strain identification.</p><p id="p-37">For the unique-PSMs-based strategy, we observed a poor availability of unique PSMs at the strain level. While the exploitation of purely unique features is a common theme for species-level identification, the low amount of unique PSMs in strains is insufficient for strain-level inference. However, the frequency of spectra matching to distinct proteins and proteomes remains a valuable parameter for strain differentiation when considering and weighing both, unique and the plethora of non-unique matches.</p><p id="p-38">TaxIt has a comparable computational runtime for small samples and databases (such as the viral data) despite using a constrained search space. This is primarily a result of the additional strain proteome downloads, since the NCBI Entrez API is not designed and optimized for large scale downloads and proteins need to be fetched in numerous iterations of small chunks. However, the download overhead fades into the background when considering full bacterial samples such as bacillus all (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> and <strong>Figure 5</strong>) and gives place to a runtime improvement of three quarters when compared to NCBI Blast NR database searches. In contrast, the runtime of Pipasic is afflicted with additional sequence comparisons necessary for constructing the similarity matrix that is highly influenced by increasing numbers of PSMs and taxa to compare. Finally, the memory footprint of TaxIt in comparison to the unique-PSMs- and Pipasic-based strategies remains constantly less for all samples, as would be expected when using substantially less proteins in the search databases.</p><p id="p-39">On a final note, strain-level identification performance is generally limited by the availability and integrity of taxa and proteomes in used databases. However, constantly increasing quality and quantity of the NCBI Taxonomy and Protein databases will induce constant improvement of strain-level identification strategies such as the presented iterative workflow.</p></div><div class="section" id="sec-10"><h2 class="">Conclusion</h2><p id="p-40">Untargeted strain level identification via MS/MS spectra is a challenging task with respect to the excessive quantity of strains that need to be considered competitively. For solving this problem, we present TaxIt, an iterative approach that first focuses on species identification and thus limits strain identification to concise selected target databases. In our method, both iteration steps take advantage of publicly available data from the NCBI Taxonomy and Protein databases. Thereby, TaxIt enables final strain identification from recent and relevant sequence data, without the need of heavily curated, tailored or taxonomically constrained databases. TaxIt supports any MS/MS peptide data suitable for classic database search methods. With further ongoing developments, our tool might enable a wide range of applications in public health, research and potentially also clinics in the future: based on further validation, TaxIt might support diagnosis and treatment of pathogen-based infectious diseases. TaxIt is available for download under open-source license at <a href="https://gitlab.com/rki_bioinformatics">https://gitlab.com/rki_bioinformatics</a>.</p></div><div class="section" id="sec-12"><h2 class="">Author Contributions</h2><p id="p-41">MK carried out the implementation and evaluation, participated in the software design and experimental design and drafted the manuscript. JD participated in the experimental design and data acquisition. AN participated in the experimental design. TM participated in the software design and drafted the manuscript. BYR participated in the software design and experimental design and drafted the manuscript. All authors read and approved the final manuscript.</p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENT</h2><p id="p-42">The authors would like to thank Henning Schiebenhoefer for constructive criticism of the manuscript.</p><p id="p-43">This work was supported by the German Research Foundation (DFG) (RE3474/2-2 to B.Y.R.).</p></div><div class="section glossary"><h2 class="">ABBREVIATIONS</h2><dl id="def-list-1"><dt id="def-item-1">FDR<span class="def-item-dt-sep"></span></dt><dd>false discovery rate<span class="def-item-dd-sep"></span></dd><dt id="def-item-2">LC<span class="def-item-dt-sep"></span></dt><dd>liquid chromatography<span class="def-item-dd-sep"></span></dd><dt id="def-item-3">MALDI<span class="def-item-dt-sep"></span></dt><dd>matrix-assisted laser desorption/ionization<span class="def-item-dd-sep"></span></dd><dt id="def-item-4">MS<span class="def-item-dt-sep"></span></dt><dd>mass spectrometry<span class="def-item-dd-sep"></span></dd><dt id="def-item-5">PSM<span class="def-item-dt-sep"></span></dt><dd>peptide spectrum match<span class="def-item-dd-sep"></span></dd><dt id="def-item-6">RSS<span class="def-item-dt-sep"></span></dt><dd>resident set size<span class="def-item-dd-sep"></span></dd><dt id="def-item-7">TOF<span class="def-item-dt-sep"></span></dt><dd>time-of-flight</dd></dl></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">(1).</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference (1) in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.1" data-doi="10.1016/j.virol.2009.04.012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bengali</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Townsley</span>,  <span class="cit-name-given-names">A. C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Moss</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Vaccinia Virus Strain Differences in Cell Attachment and Entry</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-pub-date">2009</span>, <span class="cit-vol">389</span> (<span class="cit-issue">1</span>), <span class="cit-fpage">132</span>–<span class="cit-lpage">140</span>. <a href="https://doi.org/10.1016/j.virol.2009.04.012">https://doi.org/10.1016/j.virol.2009.04.012</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D389%26rft.spage%253D132%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2009.04.012%26rft_id%253Dinfo%253Apmid%252F19428041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2009.04.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19428041&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266948800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">(2).</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-812313v1.2" data-doi="10.1016/j.virol.2012.08.044"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bengali</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Satheshkumar</span>,  <span class="cit-name-given-names">P. S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Moss</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Orthopoxvirus Species and Strain Differences in Cell Entry</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-pub-date">2012</span>, <span class="cit-vol">433</span> (<span class="cit-issue">2</span>), <span class="cit-fpage">506</span>–<span class="cit-lpage">512</span>. <a href="https://doi.org/10.1016/j.virol.2012.08.044">https://doi.org/10.1016/j.virol.2012.08.044</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D433%26rft.spage%253D506%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2012.08.044%26rft_id%253Dinfo%253Apmid%252F22999097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2012.08.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22999097&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(3).</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference (3) in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Doellinger</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Schaade</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Nitsche</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Comparison of the Cowpox Virus and Vaccinia Virus Mature Virion Proteome: Analysis of the Species- and Strain-Specific Proteome</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">10</span> (<span class="cit-issue">11</span>), <span class="cit-fpage">e0141527</span>. <a href="https://doi.org/10.1371/journal.pone.0141527">https://doi.org/10.1371/journal.pone.0141527</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D10%26rft.spage%253De0141527%26rft_id%253Dinfo%253Apmid%252F26556597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26556597&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(4).</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference (4) in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.4" data-doi="10.1128/jb.184.4.952-961.2002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Choi</span>,  <span class="cit-name-given-names">J. Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sifri</span>,  <span class="cit-name-given-names">C. D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Goumnerov</span>,  <span class="cit-name-given-names">B. C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rahme</span>,  <span class="cit-name-given-names">L. G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ausubel</span>,  <span class="cit-name-given-names">F. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Calderwood</span>,  <span class="cit-name-given-names">S. B.</span></span> <span class="cit-article-title">Identification of Virulence Genes in a Pathogenic Strain of Pseudomonas Aeruginosa by Representational Difference Analysis</span>. <abbr class="cit-jnl-abbrev">J. Bacteriol</abbr>. <span class="cit-pub-date">2002</span>, <span class="cit-vol">184</span> (<span class="cit-issue">4</span>), <span class="cit-fpage">952</span>–<span class="cit-lpage">961</span>. <a href="https://doi.org/10.1128/jb.184.4.952-961.2002">https://doi.org/10.1128/jb.184.4.952-961.2002</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DChoi%26rft.auinit1%253DJ.%2BY.%26rft.volume%253D184%26rft.issue%253D4%26rft.spage%253D952%26rft.epage%253D961%26rft.atitle%253DIdentification%2Bof%2BVirulence%2BGenes%2Bin%2Ba%2BPathogenic%2BStrain%2Bof%2BPseudomonas%2Baeruginosa%2Bby%2BRepresentational%2BDifference%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.184.4.952-961.2002%26rft_id%253Dinfo%253Apmid%252F11807055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTg0LzQvOTUyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODEyMzEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">(5).</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference (5) in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.5" data-doi="10.1128/AEM.71.11.7551-7555.2005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Genersch</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ashiralieva</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fries</span>,  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">Strain- and Genotype-Specific Differences in Virulence of Paenibacillus Larvae Subsp. Larvae, a Bacterial Pathogen Causing American Foulbrood Disease in Honeybees</span>. <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol</abbr>. <span class="cit-pub-date">2005</span>, <span class="cit-vol">71</span> (<span class="cit-issue">11</span>), <span class="cit-fpage">7551</span>–<span class="cit-lpage">7555</span>. <a href="https://doi.org/10.1128/AEM.71.11.7551-7555.2005">https://doi.org/10.1128/AEM.71.11.7551-7555.2005</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DGenersch%26rft.auinit1%253DE.%26rft.volume%253D71%26rft.issue%253D11%26rft.spage%253D7551%26rft.epage%253D7555%26rft.atitle%253DStrain-%2Band%2BGenotype-Specific%2BDifferences%2Bin%2BVirulence%2Bof%2BPaenibacillus%2Blarvae%2Bsubsp.%2Blarvae%252C%2Ba%2BBacterial%2BPathogen%2BCausing%2BAmerican%2BFoulbrood%2BDisease%2Bin%2BHoneybees%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.71.11.7551-7555.2005%26rft_id%253Dinfo%253Apmid%252F16269801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3MS8xMS83NTUxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODEyMzEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">(6).</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference (6) in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boulund</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Karlsson</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gonzales-Siles</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Johnning</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Karami</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Al-Bayati</span>,  <span class="cit-name-given-names">O.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Åhrén</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Moore</span>,  <span class="cit-name-given-names">E. R. B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kristiansson</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">Typing and Characterization of Bacteria Using Bottom-up Tandem Mass Spectrometry Proteomics</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Proteomics MCP</abbr> <span class="cit-pub-date">2017</span>, <span class="cit-vol">16</span> (<span class="cit-issue">6</span>), <span class="cit-fpage">1052</span>–<span class="cit-lpage">1063</span>. <a href="https://doi.org/10.1074/mcp.M116.061721">https://doi.org/10.1074/mcp.M116.061721</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BProteomics%2BMCP%26rft.volume%253D16%26rft.spage%253D1052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(7).</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference (7) in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Singhal</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kanaujia</span>,  <span class="cit-name-given-names">P. K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Virdi</span>,  <span class="cit-name-given-names">J. S.</span></span> <span class="cit-article-title">MALDI-TOF Mass Spectrometry: An Emerging Technology for Microbial Identification and Diagnosis</span>. <abbr class="cit-jnl-abbrev">Front. Microbiol</abbr>. <span class="cit-pub-date">2015</span>, <span class="cit-vol">6</span>. <a href="https://doi.org/10.3389/fmicb.2015.00791">https://doi.org/10.3389/fmicb.2015.00791</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">(8).</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference (8) in text" id="ref-8">↵</a><div class="cit ref-cit ref-web" id="cit-812313v1.8"><div class="cit-metadata"><cite><span class="cit-article-title">MALDI Biotyper Systems | Bruker</span> <a href="https://www.bruker.com/products/mass-spectrometry-and-separations/maldi-biotyper-systems.html">https://www.bruker.com/products/mass-spectrometry-and-separations/maldi-biotyper-systems.html</a> (accessed <span class="cit-date-in-citation">Sep 29, 2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">(9).</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference (9) in text" id="ref-9">↵</a><div class="cit ref-cit ref-web" id="cit-812313v1.9"><div class="cit-metadata"><cite><span class="cit-article-title">Strain Typing with IR Biotyper Overview | Bruker</span> <a href="https://www.bruker.com/applications/microbiology/strain-typing-with-ir-biotyper/overview.html">https://www.bruker.com/applications/microbiology/strain-typing-with-ir-biotyper/overview.html</a> (accessed <span class="cit-date-in-citation">Sep 29, 2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">(10).</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference (10) in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ecker</span>,  <span class="cit-name-given-names">D. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Drader</span>,  <span class="cit-name-given-names">J. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gutierrez</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gutierrez</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hannis</span>,  <span class="cit-name-given-names">J. C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Schink</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sampath</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Blyn</span>,  <span class="cit-name-given-names">L. B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Eshoo</span>,  <span class="cit-name-given-names">M. W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hall</span>,  <span class="cit-name-given-names">T. A.</span></span>; <span class="cit-etal">et al.</span> <span class="cit-article-title">The Ibis T5000 Universal Biosensor: An Automated Platform for Pathogen Identification and Strain Typing</span>. <abbr class="cit-jnl-abbrev">JALA J. Assoc. Lab. Autom</abbr>. <span class="cit-pub-date">2006</span>, <span class="cit-vol">11</span> (<span class="cit-issue">6</span>), <span class="cit-fpage">341</span>–<span class="cit-lpage">351</span>. <a href="https://doi.org/10.1016/j.jala.2006.09.001">https://doi.org/10.1016/j.jala.2006.09.001</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJALA%2BJ.%2BAssoc.%2BLab.%2BAutom%26rft.volume%253D11%26rft.spage%253D341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(11).</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference (11) in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.11" data-doi="10.1128/AEM.00740-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gekenidis</span>,  <span class="cit-name-given-names">M.-T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Studer</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wüthrich</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Brunisholz</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Drissner</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Beyond the Matrix-Assisted Laser Desorption Ionization (MALDI) Biotyping Workflow: In Search of Microorganism-Specific Tryptic Peptides Enabling Discrimination of Subspecies</span>. <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol</abbr>. <span class="cit-pub-date">2014</span>, <span class="cit-vol">80</span> (<span class="cit-issue">14</span>), <span class="cit-fpage">4234</span>–<span class="cit-lpage">4241</span>. <a href="https://doi.org/10.1128/AEM.00740-14">https://doi.org/10.1128/AEM.00740-14</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.00740-14%26rft_id%253Dinfo%253Apmid%252F24795381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI4MC8xNC80MjM0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODEyMzEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">(12).</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference (12) in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.12" data-doi="10.1021/acs.jproteome.6b00216"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pfrunder</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Grossmann</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hunziker</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Brunisholz</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gekenidis</span>,  <span class="cit-name-given-names">M.-T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Drissner</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Bacillus Cereus Group-Type Strain-Specific Diagnostic Peptides</span>. <abbr class="cit-jnl-abbrev">J. Proteome Res</abbr>. <span class="cit-pub-date">2016</span>, <span class="cit-vol">15</span> (<span class="cit-issue">9</span>), <span class="cit-fpage">3098</span>–<span class="cit-lpage">3107</span>. <a href="https://doi.org/10.1021/acs.jproteome.6b00216">https://doi.org/10.1021/acs.jproteome.6b00216</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BProteome%2BRes%26rft.volume%253D15%26rft.spage%253D3098%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facs.jproteome.6b00216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acs.jproteome.6b00216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">(13).</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference (13) in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alves</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ogurtsov</span>,  <span class="cit-name-given-names">A. Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Drake</span>,  <span class="cit-name-given-names">S. K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gucek</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Suffredini</span>,  <span class="cit-name-given-names">A. F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sacks</span>,  <span class="cit-name-given-names">D. B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">Y.-K.</span></span> <span class="cit-article-title">Identification of Microorganisms by High Resolution Tandem Mass Spectrometry with Accurate Statistical Significance</span>. <abbr class="cit-jnl-abbrev">J. Am. Soc. Mass Spectrom</abbr>. <span class="cit-pub-date">2016</span>, <span class="cit-vol">27</span> (<span class="cit-issue">2</span>), <span class="cit-fpage">194</span>–<span class="cit-lpage">210</span>. <a href="https://doi.org/10.1007/s13361-015-1271-2">https://doi.org/10.1007/s13361-015-1271-2</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BSoc.%2BMass%2BSpectrom%26rft.volume%253D27%26rft.spage%253D194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(14).</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference (14) in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.14" data-doi="10.1038/nature01511"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aebersold</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mann</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Mass Spectrometry-Based Proteomics</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2003</span>, <span class="cit-vol">422</span> (<span class="cit-issue">6928</span>), <span class="cit-fpage">198</span>–<span class="cit-lpage">207</span>. <a href="https://doi.org/10.1038/nature01511">https://doi.org/10.1038/nature01511</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAebersold%26rft.auinit1%253DR.%26rft.volume%253D422%26rft.issue%253D6928%26rft.spage%253D198%26rft.epage%253D207%26rft.atitle%253DMass%2Bspectrometry-based%2Bproteomics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01511%26rft_id%253Dinfo%253Apmid%252F12634793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01511&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12634793&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181488900054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">(15).</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference (15) in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.15" data-doi="10.1126/science.1124619"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Domon</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Aebersold</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Mass Spectrometry and Protein Analysis</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">312</span> (<span class="cit-issue">5771</span>), <span class="cit-fpage">212</span>–<span class="cit-lpage">217</span>. <a href="https://doi.org/10.1126/science.1124619">https://doi.org/10.1126/science.1124619</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDomon%26rft.auinit1%253DB.%26rft.volume%253D312%26rft.issue%253D5771%26rft.spage%253D212%26rft.epage%253D217%26rft.atitle%253DMass%2Bspectrometry%2Band%2Bprotein%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1124619%26rft_id%253Dinfo%253Apmid%252F16614208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTIvNTc3MS8yMTIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8yNC84MTIzMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">(16).</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference (16) in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.16" data-doi="10.1371/journal.pcbi.0040012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McHugh</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Arthur</span>,  <span class="cit-name-given-names">J. W.</span></span> <span class="cit-article-title">Computational Methods for Protein Identification from Mass Spectrometry Data</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr> <span class="cit-pub-date">2008</span>, <span class="cit-vol">4</span> (<span class="cit-issue">2</span>), <span class="cit-fpage">e12</span>. <a href="https://doi.org/10.1371/journal.pcbi.0040012">https://doi.org/10.1371/journal.pcbi.0040012</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DMcHugh%26rft.auinit1%253DL.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De12%26rft.epage%253De12%26rft.atitle%253DComputational%2Bmethods%2Bfor%2Bprotein%2Bidentification%2Bfrom%2Bmass%2Bspectrometry%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0040012%26rft_id%253Dinfo%253Apmid%252F18463710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0040012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18463710&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(17).</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference (17) in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.17" data-doi="10.1021/cr068288j"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Belov</span>,  <span class="cit-name-given-names">M. E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jaitly</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Qian</span>,  <span class="cit-name-given-names">W.-J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">R. D.</span></span> <span class="cit-article-title">Accurate Mass Measurements in Proteomics</span>. <abbr class="cit-jnl-abbrev">Chem. Rev</abbr>. <span class="cit-pub-date">2007</span>, <span class="cit-vol">107</span> (<span class="cit-issue">8</span>), <span class="cit-fpage">3621</span>–<span class="cit-lpage">3653</span>. <a href="https://doi.org/10.1021/cr068288j">https://doi.org/10.1021/cr068288j</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChemical%2BReviews%26rft.stitle%253DChemical%2BReviews%26rft.aulast%253DLiu%26rft.auinit1%253DT.%26rft.volume%253D107%26rft.issue%253D8%26rft.spage%253D3621%26rft.epage%253D3653%26rft.atitle%253DAccurate%2Bmass%2Bmeasurements%2Bin%2Bproteomics.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fcr068288j%26rft_id%253Dinfo%253Apmid%252F17649984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/cr068288j&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17649984&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248657400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">(18).</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference (18) in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.18" data-doi="10.1073/pnas.0800788105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mann</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kelleher</span>,  <span class="cit-name-given-names">N. L.</span></span> <span class="cit-article-title">Precision Proteomics: The Case for High Resolution and High Mass Accuracy</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci</abbr>. <span class="cit-pub-date">2008</span>, <span class="cit-vol">105</span> (<span class="cit-issue">47</span>), <span class="cit-fpage">18132</span>–<span class="cit-lpage">18138</span>. <a href="https://doi.org/10.1073/pnas.0800788105">https://doi.org/10.1073/pnas.0800788105</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0800788105%26rft_id%253Dinfo%253Apmid%252F18818311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzQ3LzE4MTMyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODEyMzEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">(19).</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference (19) in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuhring</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Renard</span>,  <span class="cit-name-given-names">B. Y.</span></span> <span class="cit-article-title">Estimating the Computational Limits of Detection of Microbial Non-Model Organisms</span>. <abbr class="cit-jnl-abbrev">PROTEOMICS</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">15</span> (<span class="cit-issue">20</span>), <span class="cit-fpage">3580</span>–<span class="cit-lpage">3584</span>. <a href="https://doi.org/10.1002/pmic.201400598">https://doi.org/10.1002/pmic.201400598</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPROTEOMICS%26rft.volume%253D15%26rft.spage%253D3580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(20).</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference (20) in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.20" data-doi="10.1186/1471&amp;ndash;2105&amp;ndash;13-S1-S2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jeong</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bandeira</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-article-title">False Discovery Rates in Spectral Identification</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-pub-date">2012</span>, <span class="cit-vol">13</span> (<span class="cit-issue">16</span>), <span class="cit-fpage">S2</span>. <a href="https://doi.org/10.1186/1471-2105-13-S16-S2">https://doi.org/10.1186/1471-2105-13-S16-S2</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D13%26rft.spage%253DS2%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471%2526ndash%253B2105%2526ndash%253B13-S1-S2%26rft_id%253Dinfo%253Apmid%252F23095127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471&amp;ndash;2105&amp;ndash;13-S1-S2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23095127&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(21).</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference (21) in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Muth</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kolmeder</span>,  <span class="cit-name-given-names">C. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Salojärvi</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Keskitalo</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Varjosalo</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Verdam</span>,  <span class="cit-name-given-names">F. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rensen</span>,  <span class="cit-name-given-names">S. S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Reichl</span>,  <span class="cit-name-given-names">U.</span></span>; <span class="cit-auth"><span class="cit-name-surname">de Vos</span>,  <span class="cit-name-given-names">W. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rapp</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-etal">et al.</span> <span class="cit-article-title">Navigating through Metaproteomics Data: A Logbook of Database Searching</span>. <abbr class="cit-jnl-abbrev">Proteomics</abbr> <span class="cit-pub-date">2015</span>. <a href="https://doi.org/10.1002/pmic.201400560">https://doi.org/10.1002/pmic.201400560</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">(22).</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference (22) in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pible</span>,  <span class="cit-name-given-names">O.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hartmann</span>,  <span class="cit-name-given-names">E. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Imbert</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Armengaud</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">The Importance of Recognizing and Reporting Sequence Database Contamination for Proteomics</span>. <abbr class="cit-jnl-abbrev">EuPA Open Proteomics</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">3</span>, <span class="cit-fpage">246</span>–<span class="cit-lpage">249</span>. <a href="https://doi.org/10.1016/j.euprot.2014.04.001">https://doi.org/10.1016/j.euprot.2014.04.001</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuPA%2BOpen%2BProteomics%26rft.volume%253D3%26rft.spage%253D246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(23).</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference (23) in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.23" data-doi="10.1002/1615-9861(200210)2:10&lt;1426::AID-PROT1426&gt;3.0.CO;2-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Creasy</span>,  <span class="cit-name-given-names">D. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cottrell</span>,  <span class="cit-name-given-names">J. S.</span></span> <span class="cit-article-title">Error Tolerant Searching of Uninterpreted Tandem Mass Spectrometry Data</span>. <abbr class="cit-jnl-abbrev">PROTEOMICS</abbr> <span class="cit-pub-date">2002</span>, <span class="cit-vol">2</span> (<span class="cit-issue">10</span>), <span class="cit-fpage">1426</span>–<span class="cit-lpage">1434</span>. <a href="https://doi.org/10.1002/1615-9861(200210)2:10&lt;1426::AID-PROT1426&gt;3.0.CO;2-5">https://doi.org/10.1002/1615-9861(200210)2:10&lt;1426::AID-PROT1426&gt;3.0.CO;2-5</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteomics%26rft.stitle%253DProteomics%26rft.aulast%253DCreasy%26rft.auinit1%253DD.%2BM.%26rft.volume%253D2%26rft.issue%253D10%26rft.spage%253D1426%26rft.epage%253D1434%26rft.atitle%253DError%2Btolerant%2Bsearching%2Bof%2Buninterpreted%2Btandem%2Bmass%2Bspectrometry%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F1615-9861%2528200210%25292%253A10%253C1426%253A%253AAID-PROT1426%253E3.0.CO%253B2-5%26rft_id%253Dinfo%253Apmid%252F12422359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/1615-9861(200210)2:10&lt;1426::AID-PROT1426&gt;3.0.CO;2-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12422359&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179256000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">(24).</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference (24) in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.24" data-doi="10.1002/rcm.1198"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Craig</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Beavis</span>,  <span class="cit-name-given-names">R. C.</span></span> <span class="cit-article-title">A Method for Reducing the Time Required to Match Protein Sequences with Tandem Mass Spectra</span>. <abbr class="cit-jnl-abbrev">Rapid Commun. Mass Spectrom</abbr>. <span class="cit-pub-date">2003</span>, <span class="cit-vol">17</span> (<span class="cit-issue">20</span>), <span class="cit-fpage">2310</span>–<span class="cit-lpage">2316</span>. <a href="https://doi.org/10.1002/rcm.1198">https://doi.org/10.1002/rcm.1198</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRapid%2BCommunications%2Bin%2BMass%2BSpectrometry%26rft.stitle%253DRapid%2BCommunications%2Bin%2BMass%2BSpectrometry%26rft.aulast%253DCraig%26rft.auinit1%253DR.%26rft.volume%253D17%26rft.issue%253D20%26rft.spage%253D2310%26rft.epage%253D2316%26rft.atitle%253DA%2Bmethod%2Bfor%2Breducing%2Bthe%2Btime%2Brequired%2Bto%2Bmatch%2Bprotein%2Bsequences%2Bwith%2Btandem%2Bmass%2Bspectra.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Frcm.1198%26rft_id%253Dinfo%253Apmid%252F14558131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/rcm.1198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14558131&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185867500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">(25).</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference (25) in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.25" data-doi="10.1074/mcp.M500319-MCP200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nesvizhskii</span>,  <span class="cit-name-given-names">A. I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Roos</span>,  <span class="cit-name-given-names">F. F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Grossmann</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Vogelzang</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Eddes</span>,  <span class="cit-name-given-names">J. S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gruissem</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Baginsky</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Aebersold</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Dynamic Spectrum Quality Assessment and Iterative Computational Analysis of Shotgun Proteomic Data Toward More Efficient Identification of Post-Translational Modifications, Sequence Polymorphisms, and Novel Peptides</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Proteomics</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">5</span> (<span class="cit-issue">4</span>), <span class="cit-fpage">652</span>–<span class="cit-lpage">670</span>. <a href="https://doi.org/10.1074/mcp.M500319-MCP200">https://doi.org/10.1074/mcp.M500319-MCP200</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fmcp.M500319-MCP200%26rft_id%253Dinfo%253Apmid%252F16352522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoibWNwcm90IjtzOjU6InJlc2lkIjtzOjc6IjUvNC82NTIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8yNC84MTIzMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">(26).</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference (26) in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.26" data-doi="10.1186/1471-2164-12-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rooijers</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kolmeder</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Juste</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Doré</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Been</span>,  <span class="cit-name-given-names">M. de</span></span>; <span class="cit-auth"><span class="cit-name-surname">Boeren</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Galan</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Beauvallet</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Vos</span>,  <span class="cit-name-given-names">W. M. de</span></span>; <span class="cit-auth"><span class="cit-name-surname">Schaap</span>,  <span class="cit-name-given-names">P. J.</span></span> <span class="cit-article-title">An Iterative Workflow for Mining the Human Intestinal Metaproteome</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-pub-date">2011</span>, <span class="cit-vol">12</span> (<span class="cit-issue">1</span>), <span class="cit-fpage">6</span>. <a href="https://doi.org/10.1186/1471-2164-12-6">https://doi.org/10.1186/1471-2164-12-6</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DRooijers%26rft.auinit1%253DK.%26rft.volume%253D12%26rft.spage%253D6%26rft.epage%253D6%26rft.atitle%253DAn%2Biterative%2Bworkflow%2Bfor%2Bmining%2Bthe%2Bhuman%2Bintestinal%2Bmetaproteome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-6%26rft_id%253Dinfo%253Apmid%252F21208423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21208423&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(27).</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference (27) in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.27" data-doi="10.1002/pmic.200900473"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ning</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fermin</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Nesvizhskii</span>,  <span class="cit-name-given-names">A. I.</span></span> <span class="cit-article-title">Computational Analysis of Unassigned High-Quality MS/MS Spectra in Proteomic Data Sets</span>. <abbr class="cit-jnl-abbrev">PROTEOMICS</abbr> <span class="cit-pub-date">2010</span>, <span class="cit-vol">10</span> (<span class="cit-issue">14</span>), <span class="cit-fpage">2712</span>–<span class="cit-lpage">2718</span>. <a href="https://doi.org/10.1002/pmic.200900473">https://doi.org/10.1002/pmic.200900473</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteomics%26rft.stitle%253DProteomics%26rft.aulast%253DNing%26rft.auinit1%253DK.%26rft.volume%253D10%26rft.issue%253D14%26rft.spage%253D2712%26rft.epage%253D2718%26rft.atitle%253DComputational%2Banalysis%2Bof%2Bunassigned%2Bhigh-quality%2BMS%252FMS%2Bspectra%2Bin%2Bproteomic%2Bdata%2Bsets.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpmic.200900473%26rft_id%253Dinfo%253Apmid%252F20455209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pmic.200900473&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20455209&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(28).</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference (28) in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.28" data-doi="10.1002/pmic.201100503"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jagtap</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">McGowan</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bandhakavi</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tu</span>,  <span class="cit-name-given-names">Z. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Seymour</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Griffin</span>,  <span class="cit-name-given-names">T. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rudney</span>,  <span class="cit-name-given-names">J. D.</span></span> <span class="cit-article-title">Deep Metaproteomic Analysis of Human Salivary Supernatant</span>. <abbr class="cit-jnl-abbrev">PROTEOMICS</abbr> <span class="cit-pub-date">2012</span>, <span class="cit-vol">12</span> (<span class="cit-issue">7</span>), <span class="cit-fpage">992</span>–<span class="cit-lpage">1001</span>. <a href="https://doi.org/10.1002/pmic.201100503">https://doi.org/10.1002/pmic.201100503</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteomics%26rft.stitle%253DProteomics%26rft.aulast%253DJagtap%26rft.auinit1%253DP.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D992%26rft.epage%253D1001%26rft.atitle%253DDeep%2Bmetaproteomic%2Banalysis%2Bof%2Bhuman%2Bsalivary%2Bsupernatant.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpmic.201100503%26rft_id%253Dinfo%253Apmid%252F22522805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pmic.201100503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22522805&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303053300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">(29).</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-812313v1.29" data-doi="10.1002/pmic.201200352"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jagtap</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Goslinga</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kooren</span>,  <span class="cit-name-given-names">J. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">McGowan</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wroblewski</span>,  <span class="cit-name-given-names">M. S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Seymour</span>,  <span class="cit-name-given-names">S. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Griffin</span>,  <span class="cit-name-given-names">T. J.</span></span> <span class="cit-article-title">A Two-Step Database Search Method Improves Sensitivity in Peptide Sequence Matches for Metaproteomics and Proteogenomics Studies</span>. <abbr class="cit-jnl-abbrev">PROTEOMICS</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">13</span> (<span class="cit-issue">8</span>), <span class="cit-fpage">1352</span>–<span class="cit-lpage">1357</span>. <a href="https://doi.org/10.1002/pmic.201200352">https://doi.org/10.1002/pmic.201200352</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPROTEOMICS%26rft.volume%253D13%26rft.spage%253D1352%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpmic.201200352%26rft_id%253Dinfo%253Apmid%252F23412978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pmic.201200352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23412978&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317684800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">(30).</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference (30) in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.30" data-doi="10.1021/pr500812t"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jagtap</span>,  <span class="cit-name-given-names">P. D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">J. E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Onsongo</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sadler</span>,  <span class="cit-name-given-names">F. W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Murray</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Shenykman</span>,  <span class="cit-name-given-names">G. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bandhakavi</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">L. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Griffin</span>,  <span class="cit-name-given-names">T. J.</span></span> <span class="cit-article-title">Flexible and Accessible Workflows for Improved Proteogenomic Analysis Using the Galaxy Framework</span>. <abbr class="cit-jnl-abbrev">J. Proteome Res</abbr>. <span class="cit-pub-date">2014</span>, <span class="cit-vol">13</span> (<span class="cit-issue">12</span>), <span class="cit-fpage">5898</span>–<span class="cit-lpage">5908</span>. <a href="https://doi.org/10.1021/pr500812t">https://doi.org/10.1021/pr500812t</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BProteome%2BRes%26rft.volume%253D13%26rft.spage%253D5898%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fpr500812t%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/pr500812t&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">(31).</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference (31) in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.31" data-doi="10.1021/pr501173s."><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kertesz-Farkas</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Keich</span>,  <span class="cit-name-given-names">U.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Noble</span>,  <span class="cit-name-given-names">W. S.</span></span> <span class="cit-article-title">Tandem Mass Spectrum Identification via Cascaded Search</span>. <abbr class="cit-jnl-abbrev">J. Proteome Res</abbr>. <span class="cit-pub-date">2015</span>, <span class="cit-vol">14</span> (<span class="cit-issue">8</span>), <span class="cit-fpage">3027</span>–<span class="cit-lpage">3038</span>. <a href="https://doi.org/10.1021/pr501173s">https://doi.org/10.1021/pr501173s</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BProteome%2BRes%26rft.volume%253D14%26rft.spage%253D3027%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fpr501173s.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/pr501173s.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">(32).</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference (32) in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.32" data-doi="10.1002/pmic.201400185"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chapman</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bellgard</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">High-Throughput Parallel Proteogenomics: A Bacterial Case Study</span>. <abbr class="cit-jnl-abbrev">PROTEOMICS</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">14</span> (<span class="cit-issue">23-24</span>), <span class="cit-fpage">2780</span>–<span class="cit-lpage">2789</span>. <a href="https://doi.org/10.1002/pmic.201400185">https://doi.org/10.1002/pmic.201400185</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPROTEOMICS%26rft.volume%253D14%26rft.spage%253D2780%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpmic.201400185%26rft_id%253Dinfo%253Apmid%252F25354770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pmic.201400185&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25354770&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(33).</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference (33) in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dworzanski</span>,  <span class="cit-name-given-names">J. P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Snyder</span>,  <span class="cit-name-given-names">A. P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wishart</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Identification of Bacteria Using Tandem Mass Spectrometry Combined with a Proteome Database and Statistical Scoring</span>. <abbr class="cit-jnl-abbrev">Anal. Chem</abbr>. <span class="cit-pub-date">2004</span>, <span class="cit-vol">76</span> (<span class="cit-issue">8</span>), <span class="cit-fpage">2355</span>–<span class="cit-lpage">2366</span>. <a href="https://doi.org/10.1021/ac0349781">https://doi.org/10.1021/ac0349781</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.stitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.aulast%253DDworzanski%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D76%26rft.issue%253D8%26rft.spage%253D2355%26rft.epage%253D2366%26rft.atitle%253DIdentification%2Bof%2Bbacteria%2Busing%2Btandem%2Bmass%2Bspectrometry%2Bcombined%2Bwith%2Ba%2Bproteome%2Bdatabase%2Band%2Bstatistical%2Bscoring.%26rft_id%253Dinfo%253Apmid%252F15080748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15080748&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(34).</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference (34) in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.34" data-doi="10.1586/14789450.2.6.863"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dworzanski</span>,  <span class="cit-name-given-names">J. P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Snyder</span>,  <span class="cit-name-given-names">A. P.</span></span> <span class="cit-article-title">Classification and Identification of Bacteria Using Mass Spectrometry-Based Proteomics</span>. <abbr class="cit-jnl-abbrev">Expert Rev. Proteomics</abbr> <span class="cit-pub-date">2005</span>, <span class="cit-vol">2</span> (<span class="cit-issue">6</span>), <span class="cit-fpage">863</span>–<span class="cit-lpage">878</span>. <a href="https://doi.org/10.1586/14789450.2.6.863">https://doi.org/10.1586/14789450.2.6.863</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DExpert%2BRev%2BProteomics%26rft.aulast%253DDworzanski%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D2%26rft.issue%253D6%26rft.spage%253D863%26rft.epage%253D878%26rft.atitle%253DClassification%2Band%2Bidentification%2Bof%2Bbacteria%2Busing%2Bmass%2Bspectrometry-based%2Bproteomics.%26rft_id%253Dinfo%253Adoi%252F10.1586%252F14789450.2.6.863%26rft_id%253Dinfo%253Apmid%252F16307516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1586/14789450.2.6.863&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16307516&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(35).</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference (35) in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.35" data-doi="10.1021/pr050294t"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dworzanski</span>,  <span class="cit-name-given-names">J. P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Deshpande</span>,  <span class="cit-name-given-names">S. V.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jabbour</span>,  <span class="cit-name-given-names">R. E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Snyder</span>,  <span class="cit-name-given-names">A. P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wick</span>,  <span class="cit-name-given-names">C. H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Mass Spectrometry-Based Proteomics Combined with Bioinformatic Tools for Bacterial Classification</span>. <abbr class="cit-jnl-abbrev">J. Proteome Res</abbr>. <span class="cit-pub-date">2006</span>, <span class="cit-vol">5</span> (<span class="cit-issue">1</span>), <span class="cit-fpage">76</span>–<span class="cit-lpage">87</span>. <a href="https://doi.org/10.1021/pr050294t">https://doi.org/10.1021/pr050294t</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bproteome%2Bresearch%26rft.stitle%253DJ%2BProteome%2BRes%26rft.aulast%253DMcIntyre%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253D76%26rft.epage%253D87%26rft.atitle%253DMass%2Bspectrometry-based%2Bproteomics%2Bcombined%2Bwith%2Bbioinformatic%2Btools%2Bfor%2Bbacterial%2Bclassification.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fpr050294t%26rft_id%253Dinfo%253Apmid%252F16396497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/pr050294t&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16396497&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(36).</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference (36) in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.36" data-doi="10.1128/AEM.00055-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jabbour</span>,  <span class="cit-name-given-names">R. E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Deshpande</span>,  <span class="cit-name-given-names">S. V.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wade</span>,  <span class="cit-name-given-names">M. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Stanford</span>,  <span class="cit-name-given-names">M. F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wick</span>,  <span class="cit-name-given-names">C. H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zulich</span>,  <span class="cit-name-given-names">A. W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Skowronski</span>,  <span class="cit-name-given-names">E. W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Snyder</span>,  <span class="cit-name-given-names">A. P.</span></span> <span class="cit-article-title">Double-Blind Characterization of Non-Genome-Sequenced Bacteria by Mass Spectrometry-Based Proteomics</span>. <abbr class="cit-jnl-abbrev">Appl Env. Microbiol</abbr> <span class="cit-pub-date">2010</span>, <span class="cit-vol">76</span> (<span class="cit-issue">11</span>), <span class="cit-fpage">3637</span>–<span class="cit-lpage">3644</span>. <a href="https://doi.org/10.1128/AEM.00055-10">https://doi.org/10.1128/AEM.00055-10</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl%2BEnv.%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.00055-10%26rft_id%253Dinfo%253Apmid%252F20363779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3Ni8xMS8zNjM3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODEyMzEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">(37).</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference (37) in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jabbour</span>,  <span class="cit-name-given-names">R. E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Deshpande</span>,  <span class="cit-name-given-names">S. V.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Stanford</span>,  <span class="cit-name-given-names">M. F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wick</span>,  <span class="cit-name-given-names">C. H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zulich</span>,  <span class="cit-name-given-names">A. W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Snyder</span>,  <span class="cit-name-given-names">A. P.</span></span> <span class="cit-article-title">A Protein Processing Filter Method for Bacterial Identification by Mass Spectrometry-Based Proteomics</span>. <abbr class="cit-jnl-abbrev">J. Proteome Res</abbr>. <span class="cit-pub-date">2011</span>, <span class="cit-vol">10</span> (<span class="cit-issue">2</span>), <span class="cit-fpage">907</span>–<span class="cit-lpage">912</span>. <a href="https://doi.org/10.1021/pr101086a">https://doi.org/10.1021/pr101086a</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bproteome%2Bresearch%26rft.stitle%253DJ%2BProteome%2BRes%26rft.aulast%253DJabbour%26rft.auinit1%253DR.%2BE.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D907%26rft.epage%253D912%26rft.atitle%253DA%2Bprotein%2Bprocessing%2Bfilter%2Bmethod%2Bfor%2Bbacterial%2Bidentification%2Bby%2Bmass%2Bspectrometry-based%2Bproteomics.%26rft_id%253Dinfo%253Apmid%252F21126090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21126090&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(38).</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference (38) in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.38" data-doi="10.1101/gr.5969107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huson</span>,  <span class="cit-name-given-names">D. H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Auch</span>,  <span class="cit-name-given-names">A. F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Qi</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Schuster</span>,  <span class="cit-name-given-names">S. C.</span></span> <span class="cit-article-title">MEGAN Analysis of Metagenomic Data</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>. <span class="cit-pub-date">2007</span>, <span class="cit-vol">17</span> (<span class="cit-issue">3</span>), <span class="cit-fpage">377</span>–<span class="cit-lpage">386</span>. <a href="https://doi.org/10.1101/gr.5969107">https://doi.org/10.1101/gr.5969107</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5969107%26rft_id%253Dinfo%253Apmid%252F17255551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzMvMzc3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODEyMzEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">(39).</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference (39) in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mesuere</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Devreese</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Debyser</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Aerts</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Vandamme</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dawyndt</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Unipept: Tryptic Peptide-Based Biodiversity Analysis of Metaproteome Samples</span>. <abbr class="cit-jnl-abbrev">J. Proteome Res</abbr>. <span class="cit-pub-date">2012</span>, <span class="cit-vol">11</span> (<span class="cit-issue">12</span>), <span class="cit-fpage">5773</span>–<span class="cit-lpage">5780</span>. <a href="https://doi.org/10.1021/pr300576s">https://doi.org/10.1021/pr300576s</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bproteome%2Bresearch%26rft.stitle%253DJ%2BProteome%2BRes%26rft.aulast%253DMesuere%26rft.volume%253D11%26rft.spage%253D5773%26rft.atitle%253DUnipept%253A%2Btryptic%2Bpeptide-based%2Bbiodiversity%2Banalysis%2Bof%2Bmetaproteome%2Bsamples.%26rft_id%253Dinfo%253Apmid%252F23153116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23153116&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(40).</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference (40) in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.40" data-doi="10.1021/pr501246w"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Muth</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Behne</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Heyer</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kohrs</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Benndorf</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lehtevä</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Reichl</span>,  <span class="cit-name-given-names">U.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Martens</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rapp</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">The MetaProteomeAnalyzer: A Powerful Open-Source Software Suite for Metaproteomics Data Analysis and Interpretation</span>. <abbr class="cit-jnl-abbrev">J. Proteome Res</abbr>. <span class="cit-pub-date">2015</span>, <span class="cit-vol">14</span> (<span class="cit-issue">3</span>), <span class="cit-fpage">1557</span>–<span class="cit-lpage">1565</span>. <a href="https://doi.org/10.1021/pr501246w">https://doi.org/10.1021/pr501246w</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BProteome%2BRes%26rft.volume%253D14%26rft.spage%253D1557%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fpr501246w%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/pr501246w&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">(41).</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference (41) in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.41" data-doi="10.1021/acs.analchem.7b03544"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Muth</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kohrs</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Heyer</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Benndorf</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rapp</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Reichl</span>,  <span class="cit-name-given-names">U.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Martens</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Renard</span>,  <span class="cit-name-given-names">B. Y.</span></span> <span class="cit-article-title">MPA Portable: A Stand-Alone Software Package for Analyzing Metaproteome Samples on the Go</span>. <abbr class="cit-jnl-abbrev">Anal. Chem</abbr>. <span class="cit-pub-date">2018</span>, <span class="cit-vol">90</span> (<span class="cit-issue">1</span>), <span class="cit-fpage">685</span>–<span class="cit-lpage">689</span>. <a href="https://doi.org/10.1021/acs.analchem.7b03544">https://doi.org/10.1021/acs.analchem.7b03544</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnal.%2BChem%26rft.volume%253D90%26rft.spage%253D685%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facs.analchem.7b03544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acs.analchem.7b03544&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">(42).</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference (42) in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alves</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ogurtsov</span>,  <span class="cit-name-given-names">A. Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Drake</span>,  <span class="cit-name-given-names">S. K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gucek</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sacks</span>,  <span class="cit-name-given-names">D. B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">Y.-K.</span></span> <span class="cit-article-title">Rapid Classification and Identification of Multiple Microorganisms with Accurate Statistical Significance via High-Resolution Tandem Mass Spectrometry</span>. <abbr class="cit-jnl-abbrev">J. Am. Soc. Mass Spectrom</abbr>. <span class="cit-pub-date">2018</span>, <span class="cit-vol">29</span> (<span class="cit-issue">8</span>), <span class="cit-fpage">1721</span>–<span class="cit-lpage">1737</span>. <a href="https://doi.org/10.1007/s13361-018-1986-y">https://doi.org/10.1007/s13361-018-1986-y</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BSoc.%2BMass%2BSpectrom%26rft.volume%253D29%26rft.spage%253D1721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(43).</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference (43) in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tracz</span>,  <span class="cit-name-given-names">D. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">McCorrister</span>,  <span class="cit-name-given-names">S. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chong</span>,  <span class="cit-name-given-names">P. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">D. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Corbett</span>,  <span class="cit-name-given-names">C. R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Westmacott</span>,  <span class="cit-name-given-names">G. R.</span></span> <span class="cit-article-title">A Simple Shotgun Proteomics Method for Rapid Bacterial Identification</span>. <abbr class="cit-jnl-abbrev">J. Microbiol. Methods</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">94</span> (<span class="cit-issue">1</span>), <span class="cit-fpage">54</span>–<span class="cit-lpage">57</span>. <a href="https://doi.org/10.1016/j.mimet.2013.04.008">https://doi.org/10.1016/j.mimet.2013.04.008</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMicrobiol.%2BMethods%26rft.volume%253D94%26rft.spage%253D54%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(44).</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference (44) in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.44" data-doi="10.1093/bioinformatics/btu267"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Penzlin</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lindner</span>,  <span class="cit-name-given-names">M. S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Doellinger</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dabrowski</span>,  <span class="cit-name-given-names">P. W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Nitsche</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Renard</span>,  <span class="cit-name-given-names">B. Y.</span></span> <span class="cit-article-title">Pipasic: Similarity and Expression Correction for Strain-Level Identification and Quantification in Metaproteomics</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">30</span> (<span class="cit-issue">12</span>), <span class="cit-fpage">i149</span>–<span class="cit-lpage">i156</span>. <a href="https://doi.org/10.1093/bioinformatics/btu267">https://doi.org/10.1093/bioinformatics/btu267</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu267%26rft_id%253Dinfo%253Apmid%252F24931978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24931978&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(45).</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference (45) in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.45" data-doi="10.1074/mcp.R500012-MCP200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nesvizhskii</span>,  <span class="cit-name-given-names">A. I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Aebersold</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Interpretation of Shotgun Proteomic Data: The Protein Inference Problem</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Proteomics</abbr> <span class="cit-pub-date">2005</span>, <span class="cit-vol">4</span> (<span class="cit-issue">10</span>), <span class="cit-fpage">1419</span>–<span class="cit-lpage">1440</span>. <a href="https://doi.org/10.1074/mcp.R500012-MCP200">https://doi.org/10.1074/mcp.R500012-MCP200</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fmcp.R500012-MCP200%26rft_id%253Dinfo%253Apmid%252F16009968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoibWNwcm90IjtzOjU6InJlc2lkIjtzOjk6IjQvMTAvMTQxOSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEwLzI0LzgxMjMxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">(46).</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference (46) in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.46" data-doi="10.1093/bioinformatics/bts480"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Köster</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rahmann</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Snakemake—a Scalable Bioinformatics Workflow Engine</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2012</span>, <span class="cit-vol">28</span> (<span class="cit-issue">19</span>), <span class="cit-fpage">2520</span>–<span class="cit-lpage">2522</span>. <a href="https://doi.org/10.1093/bioinformatics/bts480">https://doi.org/10.1093/bioinformatics/bts480</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts480%26rft_id%253Dinfo%253Apmid%252F22908215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts480&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22908215&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309687500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">(47).</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference (47) in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.47" data-doi="10.1093/bioinformatics/bth092"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Craig</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Beavis</span>,  <span class="cit-name-given-names">R. C.</span></span> <span class="cit-article-title">TANDEM: Matching Proteins with Tandem Mass Spectra</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2004</span>, <span class="cit-vol">20</span> (<span class="cit-issue">9</span>), <span class="cit-fpage">1466</span>–<span class="cit-lpage">1467</span>. <a href="https://doi.org/10.1093/bioinformatics/bth092">https://doi.org/10.1093/bioinformatics/bth092</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth092%26rft_id%253Dinfo%253Apmid%252F14976030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bth092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14976030&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222125600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">(48).</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference (48) in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.48" data-doi="10.1002/pmic.200900759"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Muth</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Vaudel</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Barsnes</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Martens</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sickmann</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">XTandem Parser: An Open-Source Library to Parse and Analyse X!Tandem MS/MS Search Results</span>. <abbr class="cit-jnl-abbrev">PROTEOMICS</abbr> <span class="cit-pub-date">2010</span>, <span class="cit-vol">10</span> (<span class="cit-issue">7</span>), <span class="cit-fpage">1522</span>–<span class="cit-lpage">1524</span>. <a href="https://doi.org/10.1002/pmic.200900759">https://doi.org/10.1002/pmic.200900759</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteomics%26rft.stitle%253DProteomics%26rft.aulast%253DMuth%26rft.auinit1%253DT.%26rft.volume%253D10%26rft.issue%253D7%26rft.spage%253D1522%26rft.epage%253D1524%26rft.atitle%253DXTandem%2BParser%253A%2Ban%2Bopen-source%2Blibrary%2Bto%2Bparse%2Band%2Banalyse%2BX%2521Tandem%2BMS%252FMS%2Bsearch%2Bresults.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpmic.200900759%26rft_id%253Dinfo%253Apmid%252F20140905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pmic.200900759&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20140905&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277253200019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">(49).</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference (49) in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.49" data-doi="10.1038/ncomms6277"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Pevzner</span>,  <span class="cit-name-given-names">P. A.</span></span> <span class="cit-article-title">MS-GF+ Makes Progress towards a Universal Database Search Tool for Proteomics</span>. <abbr class="cit-jnl-abbrev">Nat. Commun</abbr>. <span class="cit-pub-date">2014</span>, <span class="cit-vol">5</span>, <span class="cit-fpage">5277</span>. <a href="https://doi.org/10.1038/ncomms6277">https://doi.org/10.1038/ncomms6277</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D5%26rft.spage%253D5277%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms6277%26rft_id%253Dinfo%253Apmid%252F25358478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms6277&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25358478&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(50).</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference (50) in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.50" data-doi="10.1038/nmeth1019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elias</span>,  <span class="cit-name-given-names">J. E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gygi</span>,  <span class="cit-name-given-names">S. P.</span></span> <span class="cit-article-title">Target-Decoy Search Strategy for Increased Confidence in Large-Scale Protein Identifications by Mass Spectrometry</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-pub-date">2007</span>, <span class="cit-vol">4</span> (<span class="cit-issue">3</span>), <span class="cit-fpage">207</span>. <a href="https://doi.org/10.1038/nmeth1019">https://doi.org/10.1038/nmeth1019</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DElias%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D207%26rft.epage%253D214%26rft.atitle%253DTarget-decoy%2Bsearch%2Bstrategy%2Bfor%2Bincreased%2Bconfidence%2Bin%2Blarge-scale%2Bprotein%2Bidentifications%2Bby%2Bmass%2Bspectrometry.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth1019%26rft_id%253Dinfo%253Apmid%252F17327847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth1019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17327847&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244715100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">(51).</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference (51) in text" id="ref-51">↵</a><div class="cit ref-cit ref-web" id="cit-812313v1.51"><div class="cit-metadata"><cite><span class="cit-article-title">Alexandre Masselot / InSilicoSpectro-Databanks-0.0.43 - search.cpan.org</span> <a href="http://search.cpan.org/~alexmass/InSilicoSpectro-Databanks-0.0.43/">http://search.cpan.org/~alexmass/InSilicoSpectro-Databanks-0.0.43/</a> (accessed <span class="cit-date-in-citation">Sep 29, 2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">(52).</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference (52) in text" id="ref-52">↵</a><div class="cit ref-cit ref-web" id="cit-812313v1.52"><div class="cit-metadata"><cite><span class="cit-article-title">cRAP protein sequences</span> <a href="http://www.thegpm.org/crap/">http://www.thegpm.org/crap/</a> (accessed <span class="cit-date-in-citation">Sep 29, 2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">(53).</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference (53) in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.53" data-doi="10.1038/nmeth.3144"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nesvizhskii</span>,  <span class="cit-name-given-names">A. I.</span></span> <span class="cit-article-title">Proteogenomics: Concepts, Applications and Computational Strategies</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">11</span> (<span class="cit-issue">11</span>), <span class="cit-fpage">1114</span>–<span class="cit-lpage">1125</span>. <a href="https://doi.org/10.1038/nmeth.3144">https://doi.org/10.1038/nmeth.3144</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D11%26rft.spage%253D1114%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3144%26rft_id%253Dinfo%253Apmid%252F25357241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25357241&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">(54).</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference (54) in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.54" data-doi="10.1093/nar/gkm1000"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>,  <span class="cit-name-given-names">D. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Barrett</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Benson</span>,  <span class="cit-name-given-names">D. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bryant</span>,  <span class="cit-name-given-names">S. H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Canese</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chetvernin</span>,  <span class="cit-name-given-names">V.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Church</span>,  <span class="cit-name-given-names">D. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dicuccio</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Federhen</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-etal">et al.</span> <span class="cit-article-title">Database Resources of the National Center for Biotechnology Information</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2008</span>, <span class="cit-vol">36</span> (Database issue), <span class="cit-fpage">D13</span>–<span class="cit-lpage">21</span>. <a href="https://doi.org/10.1093/nar/gkm1000">https://doi.org/10.1093/nar/gkm1000</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm1000%26rft_id%253Dinfo%253Apmid%252F18045790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkm1000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18045790&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F812313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252545400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">(55).</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference (55) in text" id="ref-55">↵</a><div class="cit ref-cit ref-web" id="cit-812313v1.55"><div class="cit-metadata"><cite><span class="cit-article-title">jsoup Java HTML Parser, with best of DOM, CSS, and jquery</span> <a href="https://jsoup.org/">https://jsoup.org/</a> (accessed <span class="cit-date-in-citation">Sep 29, 2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">(56).</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference (56) in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.56" data-doi="10.1371/journal.pone.0163962"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Le</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">SeqKit: A Cross-Platform and Ultrafast Toolkit for FASTA/Q File Manipulation</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">11</span> (<span class="cit-issue">10</span>), <span class="cit-fpage">e0163962</span>. <a href="https://doi.org/10.1371/journal.pone.0163962">https://doi.org/10.1371/journal.pone.0163962</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D11%26rft.spage%253De0163962%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0163962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0163962&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">(57).</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference (57) in text" id="ref-57">↵</a><div class="cit ref-cit ref-book" id="cit-812313v1.57"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Müller-Reichert</span>,  <span class="cit-name-given-names">T.</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Laue</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-chapter-title">Chapter 1 - Electron Microscopy of Viruses. In Methods in Cell Biology</span>; <span class="cit-ed"><span class="cit-name-surname">Müller-Reichert</span>,  <span class="cit-name-given-names">T.</span></span>, Ed.; <span class="cit-source">Electron Microscopy of Model Systems</span>; <span class="cit-publ-name">Academic Press</span>, <span class="cit-pub-date">2010</span>; Vol. <span class="cit-vol">96</span>, pp <span class="cit-fpage">1</span>–<span class="cit-lpage">20</span>. <a href="https://doi.org/10.1016/S0091-679X(10)96001-9">https://doi.org/10.1016/S0091-679X(10)96001-9</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElectron%2BMicroscopy%2Bof%2BModel%2BSystems%26rft.volume%253D96%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(58).</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference (58) in text" id="ref-58">↵</a><div class="cit ref-cit ref-web" id="cit-812313v1.58"><div class="cit-metadata"><cite><span class="cit-article-title">yafeng. An in silico trypsin digestion tool</span> <a href="https://github.com/yafeng/trypsin">https://github.com/yafeng/trypsin</a> (accessed <span class="cit-date-in-citation">Sep 29, 2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">(59).</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference (59) in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-812313v1.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dent</span>,  <span class="cit-name-given-names">S. D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wastling</span>,  <span class="cit-name-given-names">J. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Neuman</span>,  <span class="cit-name-given-names">B. W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Britton</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Maier</span>,  <span class="cit-name-given-names">H. J.</span></span> <span class="cit-article-title">The Proteome of the Infectious Bronchitis Virus Beau-R Virion</span>. <abbr class="cit-jnl-abbrev">J. Gen. Virol</abbr>. <span class="cit-pub-date">2015</span>, <span class="cit-vol">96</span> (<span class="cit-issue">12</span>), <span class="cit-fpage">3499</span>–<span class="cit-lpage">3506</span>. <a href="https://doi.org/10.1099/jgv.0.000304">https://doi.org/10.1099/jgv.0.000304</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGen.%2BVirol%26rft.volume%253D96%26rft.spage%253D3499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">(60).</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference (60) in text" id="ref-60">↵</a><div class="cit ref-cit ref-web" id="cit-812313v1.60"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Trappe</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wulf</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Doellinger</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Halbedel</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Muth</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Renard</span>,  <span class="cit-name-given-names">B. Y.</span></span> <span class="cit-article-title">Hortense: Horizontal Gene Transfer Detection Directly from Proteomic MS/MS Data</span>. <span class="cit-pub-date">2017</span>. <a href="https://doi.org/10.7287/peerj.preprints.3248v1">https://doi.org/10.7287/peerj.preprints.3248v1</a>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/812313v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F812313v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/812529v1" title="Reversed Signaling Flow of a Bacterial Pseudokinase" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;October 24, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/812313v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/812313v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/812313v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/812313v1" class="active" data-icon-position="" data-hide-link-title="0">An iterative and automated computational pipeline for untargeted strain-level identification using MS/MS spectra from pathogenic samples</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/964570" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-nPxu0hXab5S3MXOfq4hFgsvz-Wxrzi4FGC_XAPQu2T8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="964570" id="node964570--2" data-pisa="biorxiv;812313v1" data-pisa-master="biorxiv;812313" data-seqnum="964570" data-apath="/biorxiv/early/2019/10/24/812313.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">An iterative and automated computational pipeline for untargeted strain-level identification using MS/MS spectra from pathogenic samples</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Mathias</span> <span class="nlm-surname">Kuhring</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Joerg</span> <span class="nlm-surname">Doellinger</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Andreas</span> <span class="nlm-surname">Nitsche</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Thilo</span> <span class="nlm-surname">Muth</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Bernhard Y.</span> <span class="nlm-surname">Renard</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">812313; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/812313 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/812313v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/812313v1&amp;title=An%20iterative%20and%20automated%20computational%20pipeline%20for%20untargeted%20strain-level%20identification%20using%20MS/MS%20spectra%20from%20pathogenic%20samples" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/812313v1&amp;title=An%20iterative%20and%20automated%20computational%20pipeline%20for%20untargeted%20strain-level%20identification%20using%20MS/MS%20spectra%20from%20pathogenic%20samples" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/812313v1&amp;text=An%20iterative%20and%20automated%20computational%20pipeline%20for%20untargeted%20strain-level%20identification%20using%20MS/MS%20spectra%20from%20pathogenic%20samples" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/812313v1&amp;title=An%20iterative%20and%20automated%20computational%20pipeline%20for%20untargeted%20strain-level%20identification%20using%20MS/MS%20spectra%20from%20pathogenic%20samples" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/812313v1&amp;t=An%20iterative%20and%20automated%20computational%20pipeline%20for%20untargeted%20strain-level%20identification%20using%20MS/MS%20spectra%20from%20pathogenic%20samples" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/812313v1&amp;title=An%20iterative%20and%20automated%20computational%20pipeline%20for%20untargeted%20strain-level%20identification%20using%20MS/MS%20spectra%20from%20pathogenic%20samples" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/812313v1&amp;title=An%20iterative%20and%20automated%20computational%20pipeline%20for%20untargeted%20strain-level%20identification%20using%20MS/MS%20spectra%20from%20pathogenic%20samples" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="964570" id="node964570--3" data-pisa="biorxiv;812313v1" data-pisa-master="biorxiv;812313" data-seqnum="964570" data-apath="/biorxiv/early/2019/10/24/812313.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">An iterative and automated computational pipeline for untargeted strain-level identification using MS/MS spectra from pathogenic samples</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Mathias</span> <span class="nlm-surname">Kuhring</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Joerg</span> <span class="nlm-surname">Doellinger</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Andreas</span> <span class="nlm-surname">Nitsche</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Thilo</span> <span class="nlm-surname">Muth</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Bernhard Y.</span> <span class="nlm-surname">Renard</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">812313; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/812313 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=An%20iterative%20and%20automated%20computational%20pipeline%20for%20untargeted%20strain-level%20identification%20using%20MS/MS%20spectra%20from%20pathogenic%20samples&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Kuhring%2C+Mathias&amp;rft.au=Doellinger%2C+Joerg&amp;rft.au=Nitsche%2C+Andreas&amp;rft.au=Muth%2C+Thilo&amp;rft.au=Renard%2C+Bernhard+Y."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/964570/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/964570/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/964570/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/964570/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/964570/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/964570/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/964570/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/964570/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/964570/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/964570/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/964570/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/964570/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/812313v1&amp;count=horizontal&amp;via=&amp;text=An%20iterative%20and%20automated%20computational%20pipeline%20for%20untargeted%20strain-level%20identification%20using%20MS/MS%20spectra%20from%20pathogenic%20samples&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/812313v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/812313v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/812313v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
