<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/12/16/460832.atom" />
<meta name="HW.pisa" content="biorxiv;460832v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Homologous Recombination as an Evolutionary Force in African Swine Fever Viruses" />
<meta name="DC.Identifier" content="10.1101/460832" />
<meta name="DC.Date" content="2018-12-16" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Recent outbreaks of African swine fever virus (ASFV) in China severely influenced the swine industry of the country. Currently, there is no effective vaccine or drugs against ASFVs. How to effectively control the virus is challenging. In this study, we have analyzed all the publicly available ASFV genomes and demonstrated that there was a large genetic diversity of ASFV genomes. Interestingly, the genetic diversity was mainly caused by extensive genomic insertions and/or deletions (indels) instead of the point mutations. The genomic diversity of the virus resulted in proteome diversity. Over 250 types of proteins were inferred from the ASFV genomes, among which only 144 were observed in all analyzed viruses. Further analyses showed that the homologous recombination may contribute much to the indels, as supported by significant associations between the occurrence of extensive recombination events and the indels in the ASFV genomes. Repeated elements of dozens of nucleotides in length were observed to widely distribute and cluster in the adjacent positions of ASFV genomes, which may facilitate the occurrence of homologous recombination. Moreover, two enzymes, which were possibly related to the homologous recombination, i.e., a Lambda-like exonuclease with a YqaJ-like viral recombinase domain, and a DNA topoisomerase II, were found to be conservative in all the analyzed ASFVs. This work highlighted the importance of the homologous recombination in the evolution of the ASFVs, and helped with the strategy development of the prevention and control of the virus." />
<meta name="DC.Contributor" content="Zhaozhong Zhu" />
<meta name="DC.Contributor" content="Chao-Ting Xiao" />
<meta name="DC.Contributor" content="Yunshi Fan" />
<meta name="DC.Contributor" content="Zena Cai" />
<meta name="DC.Contributor" content="Congyu Lu" />
<meta name="DC.Contributor" content="Gaihua Zhang" />
<meta name="DC.Contributor" content="Taijiao Jiang" />
<meta name="DC.Contributor" content="Yongjun Tan" />
<meta name="DC.Contributor" content="Yousong Peng" />
<meta name="article:published_time" content="2018-12-16" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Homologous Recombination as an Evolutionary Force in African Swine Fever Viruses" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Recent outbreaks of African swine fever virus (ASFV) in China severely influenced the swine industry of the country. Currently, there is no effective vaccine or drugs against ASFVs. How to effectively control the virus is challenging. In this study, we have analyzed all the publicly available ASFV genomes and demonstrated that there was a large genetic diversity of ASFV genomes. Interestingly, the genetic diversity was mainly caused by extensive genomic insertions and/or deletions (indels) instead of the point mutations. The genomic diversity of the virus resulted in proteome diversity. Over 250 types of proteins were inferred from the ASFV genomes, among which only 144 were observed in all analyzed viruses. Further analyses showed that the homologous recombination may contribute much to the indels, as supported by significant associations between the occurrence of extensive recombination events and the indels in the ASFV genomes. Repeated elements of dozens of nucleotides in length were observed to widely distribute and cluster in the adjacent positions of ASFV genomes, which may facilitate the occurrence of homologous recombination. Moreover, two enzymes, which were possibly related to the homologous recombination, i.e., a Lambda-like exonuclease with a YqaJ-like viral recombinase domain, and a DNA topoisomerase II, were found to be conservative in all the analyzed ASFVs. This work highlighted the importance of the homologous recombination in the evolution of the ASFVs, and helped with the strategy development of the prevention and control of the virus.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;460832v2" />
<meta name="citation_id" content="460832v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/460832v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/460832v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/460832v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832.full.pdf" />
<meta name="citation_doi" content="10.1101/460832" />
<meta name="citation_num_pages" content="35" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="460832" />
<meta name="citation_author" content="Zhaozhong Zhu" />
<meta name="citation_author_institution" content="College of Biology, Hunan University" />
<meta name="citation_author" content="Chao-Ting Xiao" />
<meta name="citation_author_institution" content="College of Biology, Hunan University" />
<meta name="citation_author" content="Yunshi Fan" />
<meta name="citation_author_institution" content="College of Biology, Hunan University" />
<meta name="citation_author" content="Zena Cai" />
<meta name="citation_author_institution" content="College of Biology, Hunan University" />
<meta name="citation_author" content="Congyu Lu" />
<meta name="citation_author_institution" content="College of Biology, Hunan University" />
<meta name="citation_author" content="Gaihua Zhang" />
<meta name="citation_author_institution" content="College of Life Sciences, Hunan Normal University" />
<meta name="citation_author" content="Taijiao Jiang" />
<meta name="citation_author_institution" content="Center of System Medicine, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences &amp; Peking Union Medical College" />
<meta name="citation_author_institution" content="Suzhou Institute of Systems Medicine" />
<meta name="citation_author" content="Yongjun Tan" />
<meta name="citation_author_institution" content="College of Biology, Hunan University" />
<meta name="citation_author" content="Yousong Peng" />
<meta name="citation_author_institution" content="College of Biology, Hunan University" />
<meta name="citation_author_email" content="pys2013@hnu.edu.cn" />
<meta name="citation_reference" content="Arias, M., Jurado, C., Gallardo, C., Fernandez-Pinero, J. &amp; Sanchez-Vizcaino, J. M. Gaps in African swine fever: Analysis and priorities. Transboundary and emerging diseases 65, 235–247, doi:10.1111/tbed.12695 (2018)." />
<meta name="citation_reference" content="Galindo, I. &amp; Alonso, C. African Swine Fever Virus: A Review. Viruses 9, doi:10.3390/v9050103 (2017)." />
<meta name="citation_reference" content="Dixon, L. K., Chapman, D. A. G., Netherton, C. L. &amp; Upton, C. African swine fever virus replication and genomics. Virus research 173, 3–14, doi:10.1016/j.virusres.2012.10.020 (2013)." />
<meta name="citation_reference" content="Costard, S., Mur, L., Lubroth, J., Sanchez-Vizcaino, J. M. &amp; Pfeiffer, D. U. Epidemiology of African swine fever virus. Virus research 173, 191–197, doi:10.1016/j.virusres.2012.10.030 (2013)." />
<meta name="citation_reference" content="Sanchez-Cordon, P. J., Montoya, M., Reis, A. L. &amp; Dixon, L. K. African swine fever: A re-emerging viral disease threatening the global pig industry. Vet J 233, 41–48, doi:10.1016/j.tvjl.2017.12.025 (2018)." />
<meta name="citation_reference" content="Arzt, J., White, W. R., Thomsen, B. V. &amp; Brown, C. C. Agricultural Diseases on the Move Early in the Third Millennium. Veterinary pathology 47, 15–27, doi:10.1177/0300985809354350 (2010)." />
<meta name="citation_reference" content="World Organization for animal health. African Swine Fever (ASF) Report N°4: October 5 - 18, 2018, &lt;http://www.oie.int/en/animal-health-in-the-world/information-on-aquatic-and-terrestrial-animal-diseases/african-swine-fever/reports-on-asf/&gt; (2018)." />
<meta name="citation_reference" content="Ge, S. Q. et al. Molecular Characterization of African Swine Fever Virus, China, 2018. Emerg Infect Dis 24, 2131–2133, doi:10.3201/eid2411.181274 (2018)." />
<meta name="citation_reference" content="SS, S. &amp; WR, H. Antibody response to inactivated preparations of African swine fever virus in pigs. American journal of veterinary research 28, 6 (1967)." />
<meta name="citation_reference" content="King, K. et al. Protection of European domestic pigs from virulent African isolates of African swine fever virus by experimental immunisation. Vaccine 29, 4593–4600, doi:10.1016/j.vaccine.2011.04.052 (2011)." />
<meta name="citation_reference" content="Reis, A. L. et al. Deletion of the African Swine Fever Virus Gene DP148R Does Not Reduce Virus Replication in Culture but Reduces Virus Virulence in Pigs and Induces High Levels of Protection against Challenge. Journal of virology 91, doi:UNSP e01428-17 10.1128/JVI.01428-17 (2017)." />
<meta name="citation_reference" content="Escribano, J. M., Galindo, I. &amp; Alonso, C. Antibody-mediated neutralization of African swine fever virus: Myths and facts. Virus research 173, 101–109, doi:10.1016/j.virusres.2012.10.012 (2013)." />
<meta name="citation_reference" content="P, G.-P. et al. Neutralizing antibodies to different proteins of African swine fever virus inhibit both virus attachment and internalization. Journal of virology 70, 6 (1996)." />
<meta name="citation_reference" content="Kessler, C. et al. The intracellular proteome of African swine fever virus. Scientific reports 8, doi:Artn 14714 10.1038/S41598-018-32985-Z (2018)." />
<meta name="citation_reference" content="A, A., T, M., M, G. &amp; G, A. A proteomic atlas of the African swine fever virus particle. Journal of virology, doi:10.1128/JVI.01293-18 (2018)." />
<meta name="citation_reference" content="Chapman, D. A., Tcherepanov, V., Upton, C. &amp; Dixon, L. K. Comparison of the genome sequences of non-pathogenic and pathogenic African swine fever virus isolates. The Journal of general virology 89, 397–408, doi:10.1099/vir.0.83343-0 (2008)." />
<meta name="citation_reference" content="de Villiers, E. P. et al. Phylogenomic analysis of 11 complete African swine fever virus genome sequences. Virology 400, 128–136, doi:10.1016/j.virol.2010.01.019 (2010)." />
<meta name="citation_reference" content="Fraczyk, M. et al. Evolution of African swine fever virus genes related to evasion of host immune response. Veterinary microbiology 193, 133–144, doi:10.1016/j.vetmic.2016.08.018 (2016)." />
<meta name="citation_reference" content="Michaud, V., Randriamparany, T. &amp; Albina, E. Comprehensive phylogenetic reconstructions of African swine fever virus: proposal for a new classification and molecular dating of the virus. PloS one 8, e69662, doi:10.1371/journal.pone.0069662 (2013)." />
<meta name="citation_reference" content="Wang, Y. et al. Origin and Possible Genetic Recombination of the Middle East Respiratory Syndrome Coronavirus from the First Imported Case in China: Phylogenetics and Coalescence Analysis. mBio 6, e01280–01215, doi:10.1128/mBio.01280-15 (2015)." />
<meta name="citation_reference" content="Nagy, P. D. &amp; Bujarski, J. J. Homologous RNA recombination in brome mosaic virus: AU-rich sequences decrease the accuracy of crossovers. Journal of virology 70, 415–426 (1996)." />
<meta name="citation_reference" content="Roossinck, M. J. Mechanisms of plant virus evolution. Annual review of phytopathology 35, 191–209, doi:10.1146/annurev.phyto.35.1.191 (1997)." />
<meta name="citation_reference" content="Wikipedia. Homologous recombination, &lt;https://en.wikipedia.org/wiki/Homologous_recombination#In_viruses&gt; (2018)." />
<meta name="citation_reference" content="YqaJ protein domain, &lt;https://en.wikipedia.org/wiki/YqaJ_protein_domain&gt; (2018)." />
<meta name="citation_reference" content="Bajrai, L. H. et al. Kaumoebavirus, a New Virus That Clusters with Faustoviruses and Asfarviridae. Viruses-Basel 8, doi:Artn 278 10.3390/V8110278 (2016)." />
<meta name="citation_reference" content="Andreani, J. et al. Pacmanvirus, a New Giant Icosahedral Virus at the Crossroads between Asfarviridae and Faustoviruses. Journal of virology 91, doi:UNSP e00212 10.1128/JVI.00212-17 (2017)." />
<meta name="citation_reference" content="de Oliveira, V. L. et al. A novel TLR3 inhibitor encoded by African swine fever virus (ASFV). Archives of virology 156, 597–609, doi:10.1007/s00705-010-0894-7 (2011)." />
<meta name="citation_reference" content="Ruiz-Gonzalvo, F., Rodriguez, F. &amp; Escribano, J. M. Functional and immunological properties of the baculovirus-expressed hemagglutinin of African swine fever virus. Virology 218, 285–289, doi:10.1006/viro.1996.0193 (1996)." />
<meta name="citation_reference" content="Elde, N. C. et al. Poxviruses Deploy Genomic Accordions to Adapt Rapidly against Host Antiviral Defenses. Cell 150, 831–841, doi:10.1016/j.cell.2012.05.049 (2012)." />
<meta name="citation_reference" content="TG, B., Z, L., JG, N., DL, R. &amp; L, Z. African swine fever virus multigene family 360 genes affect virus replication and generalization of infection in Ornithodoros porcinus ticks. Journal of virology 78, 9 (2004)." />
<meta name="citation_reference" content="Zhang, J. Z. Evolution by gene duplication: an update. Trends in Ecology &amp; Evolution 18, 292–298, doi:10.1016/S0169-5347(03)00033-8 (2003)." />
<meta name="citation_reference" content="Freitas-Junior, L. H. et al. Frequent ectopic recombination of virulence factor genes in telomeric chromosome clusters of P-falciparum. Nature 407, 1018–1022, doi: 10.1038/35039531 (2000)." />
<meta name="citation_reference" content="Agarwala, R. et al. Database resources of the National Center for Biotechnology Information. Nucleic Acids Res 44, D7–D19, doi:10.1093/nar/gkv1290 (2016)." />
<meta name="citation_reference" content="Katoh, K. &amp; Standley, D. M. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Molecular biology and evolution 30, 772–780, doi:10.1093/molbev/mst010 (2013)." />
<meta name="citation_reference" content="Larkin, M. A. et al. Clustal W and Clustal X version 2.0. Bioinformatics 23, 2947–2948, doi:10.1093/bioinformatics/btm404 (2007)." />
<meta name="citation_reference" content="Altschul, S. F., Gish, W., Miller, W., Myers, E. W. &amp; Lipman, D. J. Basic local alignment search tool. Journal of molecular biology 215, 403–410 (1990)." />
<meta name="citation_reference" content="Emms, D. M. &amp; Kelly, S. OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy. Genome biology 16, 157, doi:10.1186/s13059-015-0721-2 (2015)." />
<meta name="citation_reference" content="Yang, Z. H. PAML 4: Phylogenetic analysis by maximum likelihood. Molecular biology and evolution 24, 1586–1591, doi:10.1093/molbev/msm088 (2007)." />
<meta name="citation_reference" content="Quevillon, E. et al. InterProScan: protein domains identifier. Nucleic Acids Res 33, W116–120, doi:10.1093/nar/gki442 (2005)." />
<meta name="citation_reference" content="Krogh, A., Larsson, B., von Heijne, G. &amp; Sonnhammer, E. L. Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes. Journal of molecular biology 305, 567–580, doi:10.1006/jmbi.2000.4315 (2001)." />
<meta name="citation_reference" content="Martin, D. &amp; Rybicki, E. RDP: detection of recombination amongst aligned sequences. Bioinformatics 16, 562–563 (2000)." />
<meta name="citation_reference" content="Finn, R. D. et al. The Pfam protein families database: towards a more sustainable future. Nucleic Acids Res 44, D279–D285, doi:10.1093/nar/gkv1344 (2016)." />
<meta name="citation_reference" content="Repbase, &lt;https://girinst.org/&gt; (2018)." />
<meta name="citation_reference" content="Wicker, T. et al. A unified classification system for eukaryotic transposable elements. Nature Reviews Genetics 8, 973–982, doi:10.1038/nrg2165 (2007)." />
<meta name="citation_reference" content="Tamura, K., Stecher, G., Peterson, D., Filipski, A. &amp; Kumar, S. MEGA6: Molecular Evolutionary Genetics Analysis version 6.0. Molecular biology and evolution 30, 2725–2729, doi:10.1093/molbev/mst197 (2013)." />
<meta name="citation_reference" content="Huson, D. H. et al. Dendroscope: An interactive viewer for large phylogenetic trees. Bmc Bioinformatics 8, 460, doi:10.1186/1471-2105-8-460 (2007)." />
<meta name="citation_reference" content="R Core Team, R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria, &lt; https://www.R-project.org/&gt; (2018)." />
<meta name="twitter:title" content="Homologous Recombination as an Evolutionary Force in African Swine Fever Viruses" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Recent outbreaks of African swine fever virus (ASFV) in China severely influenced the swine industry of the country. Currently, there is no effective vaccine or drugs against ASFVs. How to effectively control the virus is challenging. In this study, we have analyzed all the publicly available ASFV genomes and demonstrated that there was a large genetic diversity of ASFV genomes. Interestingly, the genetic diversity was mainly caused by extensive genomic insertions and/or deletions (indels) instead of the point mutations. The genomic diversity of the virus resulted in proteome diversity. Over 250 types of proteins were inferred from the ASFV genomes, among which only 144 were observed in all analyzed viruses. Further analyses showed that the homologous recombination may contribute much to the indels, as supported by significant associations between the occurrence of extensive recombination events and the indels in the ASFV genomes. Repeated elements of dozens of nucleotides in length were observed to widely distribute and cluster in the adjacent positions of ASFV genomes, which may facilitate the occurrence of homologous recombination. Moreover, two enzymes, which were possibly related to the homologous recombination, i.e., a Lambda-like exonuclease with a YqaJ-like viral recombinase domain, and a DNA topoisomerase II, were found to be conservative in all the analyzed ASFVs. This work highlighted the importance of the homologous recombination in the evolution of the ASFVs, and helped with the strategy development of the prevention and control of the virus." />
<meta name="og-title" property="og:title" content="Homologous Recombination as an Evolutionary Force in African Swine Fever Viruses" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/460832v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Recent outbreaks of African swine fever virus (ASFV) in China severely influenced the swine industry of the country. Currently, there is no effective vaccine or drugs against ASFVs. How to effectively control the virus is challenging. In this study, we have analyzed all the publicly available ASFV genomes and demonstrated that there was a large genetic diversity of ASFV genomes. Interestingly, the genetic diversity was mainly caused by extensive genomic insertions and/or deletions (indels) instead of the point mutations. The genomic diversity of the virus resulted in proteome diversity. Over 250 types of proteins were inferred from the ASFV genomes, among which only 144 were observed in all analyzed viruses. Further analyses showed that the homologous recombination may contribute much to the indels, as supported by significant associations between the occurrence of extensive recombination events and the indels in the ASFV genomes. Repeated elements of dozens of nucleotides in length were observed to widely distribute and cluster in the adjacent positions of ASFV genomes, which may facilitate the occurrence of homologous recombination. Moreover, two enzymes, which were possibly related to the homologous recombination, i.e., a Lambda-like exonuclease with a YqaJ-like viral recombinase domain, and a DNA topoisomerase II, were found to be conservative in all the analyzed ASFVs. This work highlighted the importance of the homologous recombination in the evolution of the ASFVs, and helped with the strategy development of the prevention and control of the virus." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-12-16" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/460832v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/460832v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/460832v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/460832v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/147645" />
    <title>Homologous Recombination as an Evolutionary Force in African Swine Fever Viruses | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"1XU6kItWY0-YLgNktLqw4QoXW0sl1Q83LqF-fQyeb6A","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"147645","apath":"\/biorxiv\/early\/2018\/12\/16\/460832.atom","pisa":"biorxiv;460832v2","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;460832\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;460832\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;460832v2"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;460832\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;460832\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/460832v2","title":"Homologous Recombination as an Evolutionary Force in African Swine Fever Viruses","identifier":"node\/147645"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bkh+uYLdzTDajlrtFGR+EG\\\\\\\/3oM4xFizAc4IJsQcCJeoK5sjzE6WdrBZ1ZBdXP3yi8mNMSGseKR3\\\\\\\/VcXN4jHzkXw7G0+rABZ\\\\\\\/Gfp9CAnQCFD1YM7J5JT0dYMV5da3ZEhcZomeD1Esp4QCSYPjhtSfpPQrn+vWNMsnjrMeEekxQ+lQcXLTiPMJErR\\\\\\\/CHAwKD5Wotm2mOs5NqMXR0EXPTHufODEk2IzvY6pRrwIcGcf8dpGO\\\\\\\/3iE9L4LQgM7frXBIO2t4ZBht2nUg9dADmfbUiFwOrgB9Sp28FgCnqJhl8uqu3kpGMz3zUQzXPONtJpGFtj6ER0hQihePPgZqwty2vxstrWbBp3F43wzIK4HIkT9eX9QOumR+\\\\\\\/3yjmInKgt4Rgl+FOca1ChzIJa4+KZ7mdvroH+hrtlJkBSC8ua6QcY3CG5Fejamj7QK9k4nykNskbEwwPJw\\\\\\\/YmyUDQssNztUytAtJiYVCLmyr1MWaTZvds3aV1CYC+tvwU2pyt+7\\\\\\\/BGP6RyBERvBeOfzIZV6duxwVDQXEhaGnCgJT9S+Ld45YiAAw2iJgW1zUuAaCKShpanYdIXXH3w5npCO1UFYI0I3v+MvkkH34SnM+oaiFUKjmw4nhArFZFA0RaUsSJHQJoOiFXEBlWKXZrNx4ce2E6Xa7qye3zSr7go0xjzyhC3VCxkserGv0FD3NxLcOiBvmLAlOUChYRgAEbDqVNU4t6nAZ79rE8mh7kLG0ekDSKJMx2FKXX\\\\\\\/DFXps8DDy+0I06fQQmiJMLhFxX2rvdTozJ9KM5NtkszNJMGZ68X6liFIxvRB2pwpvy1hYmmjTs24ARSEoSJSMH4MVs0Vtd972uIg9YY2qrSlJhhB9Kdp0w6XOQk5FwV\\\\\\\/AVwj69igD581xDlaf0UtXSAZezRfpWUxFBkzDidDbJwv\\\\\\\/nJrTxwl1mPdldlnyk5tZVywns\\\\\\\/ROTwO9O+cg1ujIDuX2fSJu32CJvdLqQFYNgfvdBzuTTVPE7QPFa05wwOC71UaS3WKUpOQ7Znu1FFAB6kSoRr9XgFJ9E0GrES3BnTqfVomPg32R\\\\\\\/FLSjRlbwoVMELigzv\\\\\\\/EGN0q3mkHffYN22n+K1RPLmWoP5vht2ERo8pnMqMyP3LJ3vdH5fWGCMtVwTjM\\\\\\\/ormJVkZFSJkA8lamYggzWH\\\\\\\/bvwSbfzRMyiQFMJJYO\\\\\\\/yw37KWK66sdYgOLqbkSp5J2s4kI2g\\\\\\\/1sojC6Ol4cHLC3HqudwOH7sZmt5mdeJ31MF8znPaVT9QpiTJRcf1s8CiOMEbz6df7\\\\\\\/1uq7Am7uoRkcJD2UF5so0YHNmtnNvtt\\\\\\\/R3ZLxWEX1NIliY=\\\u0022,\\\u0022iv\\\u0022:\\\u0022CgbWPmd0pZDgXykWD3DhoA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022338a2208d24de0269b08ff5e2c408c827c283abd6ec458e32a9f895ab83424e0\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/460832v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-147645 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/460832v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-efIrBxEd5vuGghhLEXqaaVp9_Qq1PoIsRqwTQCiDaLU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="147645" id="node147645" data-pisa="biorxiv;460832v2" data-pisa-master="biorxiv;460832" data-apath="/biorxiv/early/2018/12/16/460832.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Homologous Recombination as an Evolutionary Force in African Swine Fever Viruses</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Zhaozhong</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Chao-Ting</span> <span class="nlm-surname">Xiao</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Yunshi</span> <span class="nlm-surname">Fan</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Zena</span> <span class="nlm-surname">Cai</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Congyu</span> <span class="nlm-surname">Lu</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Gaihua</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Taijiao</span> <span class="nlm-surname">Jiang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Yongjun</span> <span class="nlm-surname">Tan</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Yousong</span> <span class="nlm-surname">Peng</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/460832 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node147645" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhaozhong Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Biology, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhaozhong%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhaozhong%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Chao-Ting Xiao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Biology, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chao-Ting%2BXiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiao%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChao-Ting%2BXiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yunshi Fan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Biology, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yunshi%2BFan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fan%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYunshi%2BFan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zena Cai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Biology, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zena%2BCai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cai%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZena%2BCai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Congyu Lu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Biology, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Congyu%2BLu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lu%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACongyu%2BLu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gaihua Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>College of Life Sciences, Hunan Normal University</span>, Changsha 410081, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gaihua%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGaihua%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Taijiao Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Center of System Medicine, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences &#38; Peking Union Medical College</span>, Beijing, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Suzhou Institute of Systems Medicine</span>, Suzhou, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Taijiao%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATaijiao%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Yongjun Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Biology, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yongjun%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYongjun%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Yousong Peng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Biology, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yousong%2BPeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peng%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYousong%2BPeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:pys2013@hnu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">pys2013@hnu.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/460832v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:147645" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:147645/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/460832v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:147645" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:147645/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/460832v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:147645" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:147645/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/460832v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:147645" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:147645/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/460832v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:147645" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:147645/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/460832v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:147645" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:147645/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Recent outbreaks of African swine fever virus (ASFV) in China severely influenced the swine industry of the country. Currently, there is no effective vaccine or drugs against ASFVs. How to effectively control the virus is challenging. In this study, we have analyzed all the publicly available ASFV genomes and demonstrated that there was a large genetic diversity of ASFV genomes. Interestingly, the genetic diversity was mainly caused by extensive genomic insertions and/or deletions (indels) instead of the point mutations. The genomic diversity of the virus resulted in proteome diversity. Over 250 types of proteins were inferred from the ASFV genomes, among which only 144 were observed in all analyzed viruses. Further analyses showed that the homologous recombination may contribute much to the indels, as supported by significant associations between the occurrence of extensive recombination events and the indels in the ASFV genomes. Repeated elements of dozens of nucleotides in length were observed to widely distribute and cluster in the adjacent positions of ASFV genomes, which may facilitate the occurrence of homologous recombination. Moreover, two enzymes, which were possibly related to the homologous recombination, i.e., a Lambda-like exonuclease with a YqaJ-like viral recombinase domain, and a DNA topoisomerase II, were found to be conservative in all the analyzed ASFVs. This work highlighted the importance of the homologous recombination in the evolution of the ASFVs, and helped with the strategy development of the prevention and control of the virus.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-5">African swine fever virus (ASFV), the causative agent of African swine fever (ASF), is a complex, large, icosahedral multi-enveloped DNA virus. It is classified as the only member in the family <em>Asfarviridae</em> <sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>. The genome of the virus belongs to double-stranded DNA, with the size ranging from 170 kb to 190 kb <sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup>. ASFV mainly infect suids and soft ticks. The suids include domestic pigs and wild boars, and were reported as the natural hosts of the virus <sup><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup>. ASFV was firstly discovered in Kenya in 1921 <sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup>. It remained restricted in Africa till 1957, when it was reported in Spain and Portugal. Up to now, the virus has caused ASF outbreaks in more than fifty countries in Africa, Europe, Asia, and South America <sup><a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a></sup>. The latest reports showed that the virus has caused outbreaks in more than fifteen provinces in China <sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup>. Because of the high lethality of ASFV in domestic pigs, the most commonly used strategies to control the virus were the massive culling campaigns and the restriction of pig movement <sup><a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a></sup>. Both strategies have resulted in a huge economic loss for pig industry and affected people’s livelihoods. Unfortunately, currently there is no available effective vaccine against ASFVs.</p><p id="p-6">Many efforts have been devoted to developing the vaccine for the ASFV <sup><a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>-<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup>, however, most of these attempts failed. One of the most important reasons was the complex composition of the antigenic proteins <sup><a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>. Previous reports showed that p72, p30, and p54 were the three important antigenic proteins during the infection of ASFVs, but the immunity against them could only provide a partial protection <sup><a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>. Many other proteins or other factors such as phospholipid composition may also influence the antigen of the virus <sup><a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a></sup>. Therefore, it is necessary to understand the mechanisms of the antigen diversity of the ASFV virus <sup><a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a></sup>.</p><p id="p-7">The genetic diversity of ASFVs has been investigated in many studies. The ASFV genome encodes over 150 proteins, including viral enzymes, viral transcription and replication-related proteins, structural proteins, other proteins involved in the virus assembly, the evading of host defense systems and the modulation of host cell function, etc <sup><a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>. For example, the transcription of the virus is independent on the host RNA polymerase because the virus contains relevant enzymes and factors <sup><a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a></sup>. The viral genome contains a conservative central region of about 125 kb and two variable ends, which results in the variable size of the genome <sup><a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>. There are significant variations among the ASFV genomes due to the genomic insertion or deletion, such as the deletion of the multigene family (MGF) members <sup><a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">3</a></sup>. Although much progress have been made on genetic diversity of the virus, the extent and mechanisms are still not clear. Besides, most of these studies either only investigated the genetic diversity of some common genes, such as p72 and p54 <sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>, or only used one or several isolate genomes <sup><a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a></sup>. The number of discovered viral genomes has increased rapidly as the development of DNA sequencing technology. Therefore, a comprehensive study on the genetic diversity of ASFVs is necessary.</p><p id="p-8">Homologous recombination, which has been reported to occur in several groups of viruses <sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>-<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup>, such as herpesvirus, retroviruses, and coronaviruses, has played an important role in viral evolution <sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>. A few studies on several ASFV genes have suggested the occurrence of homologous recombination in the evolution of ASFVs <sup><a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a></sup>. However, a comprehensive study on the homologous recombination in ASFV at the genomic scale is lacking, and the role of the recombination on the genetic diversity and the evolution of the virus is still unknown. In this study, we have systematically investigated the genomic diversity and the homologous recombination of ASFVs based on the analysis on all the publicly available ASFV genomes. The results demonstrated that the homologous recombination contributed much to the genetic diversity of ASFVs. This work would help to understand the evolution of the ASFV and thus facilitate the prevention and control of the virus.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>1 ASFV genomes</h3><p id="p-9">A total of 36 genome sequences of ASFVs were obtained from the NCBI GenBank database, which were listed in Table S1. They were mainly isolated from Africa and Europe during the years from 1950 to 2017. The size of the ASFV genomes ranged from 170,101 bp to 193,886 bp, averaged at 185,800 bp. The viral isolate Kenya50 had the largest size, while the isolate BA71V had the smallest size. No increasing or decreasing trend in the genome size was observed from 1950 to 2017 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>), suggesting the dynamic changes of the viral genomes.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic differences between ASFV genomes. (A) The variation of genome size along the isolation time of the virus. (B) Genomic differences caused by point mutations and indels. (C) The size and location of indels along the aligned genomes. For clarity, the natural logarithm of the indel size was used. Position for an indel is defined as the middle position of the indel. The average size was used if more than one indel was found in the position. The blue line refers to indel size of 50 bp. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-492202286" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Genomic differences between ASFV genomes. (A) The variation of genome size along the isolation time of the virus. (B) Genomic differences caused by point mutations and indels. (C) The size and location of indels along the aligned genomes. For clarity, the natural logarithm of the indel size was used. Position for an indel is defined as the middle position of the indel. The average size was used if more than one indel was found in the position. The blue line refers to indel size of 50 bp.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F1.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F1.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Genomic differences between ASFV genomes. (A) The variation of genome size along the isolation time of the virus. (B) Genomic differences caused by point mutations and indels. (C) The size and location of indels along the aligned genomes. For clarity, the natural logarithm of the indel size was used. Position for an indel is defined as the middle position of the indel. The average size was used if more than one indel was found in the position. The blue line refers to indel size of 50 bp.</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>2 Genomic diversity of ASFVs</h3><p id="p-10">Pairwise comparisons between ASFV genomes were conducted after the genome alignment. The average genomic difference between viruses was 24,570 bp, which accounts for more than 10% of the genome. Interestingly, the genomic differences caused by the insertions and deletions (indels) were much more significant than those caused by the point mutations (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a> &amp; Figure S1) in most cases. For example, there were 31,833 bp differences between virus Mkuzi79 and BA71V, 78% of which were caused by indels.</p><p id="p-11">The size and position of indels in ASFV genomes were also analyzed. 70% of indels were no longer than 10 bp, and about 10% of indels were 50 bp or longer (Figure S2). The occurrence of indels was much more frequent in both ends of the genome, especially in the 5’ end (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). Besides, the size of indels in both ends was also much larger than that in the middle region. Large indels with over 50 bp (above the blue line in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>) were mostly observed in both ends. It should be noted that the variation to some extent was observed in the middle region (marked in black arrow), which were considered to be conservative in previous reports.</p></div><div id="sec-5" class="subsection"><h3>3 Proteome diversity of ASFV</h3><p id="p-12">Genomic diversity could lead to proteome diversity. Therefore, the proteome diversity of ASFVs was further analyzed. Firstly, the candidate proteins encoded by ASFV genomes were inferred (Materials and Methods) (Table S2). The plus strand encoded 95-126 proteins, with an average of 109 proteins; the minus strand encoded 106-128 proteins, with an average of 118 proteins. Considering both the plus and minus strands, the ASFV genome encoded 205-254 proteins, with an average of 227 proteins. The viral isolate Russia14 encoded most proteins, although the size of this genome was not the largest. The ratio of coding region in each genome ranged from 88% to 91%.</p><p id="p-13">Furthermore, the ortholog or paralog groups based on sequence homology were identified. A total of 252 protein groups plus 28 singletons were obtained (Table S3), each of which stood for one type of protein encoded by ASFV genomes. The obtained proteins contained almost all the proteins identified in previous experiments (Table S3). Each protein group included 2-99 proteins. Only 144 protein groups were observed in all 36 ASFV viruses, which could be considered as core protein sets of the virus, and were mainly encoded by both plus and minus strands in the middle regions (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The protein groups could be further separated into seven classes by function based on previous studies (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figures 2</a> &amp; S3). Only about 30% of protein groups were observed to have the known functions, including replication and transcription (in red), host cell interactions (in magenta), structure and morphogenesis (in blue), and enzymes (in yellow). Most of the above-described protein groups with the known functions belonged to the core proteins of the virus. In addition, forty protein groups belonged to “Multigene Families (MGF)” (in cyan), most of which had unknown functions. The MGFs were encoded by both ends of the genome. Besides, the remaining 146 protein groups belonged to either the class of “Proteins with unknown function” (in gray) or “Hypothetical proteins” (in black).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The phylogenetic tree of ASFVs and the alignment of their proteomes in plus (left side) and minus strand (right side). Each row refers to the proteome of the virus in the phylogenetic tree; each column refers to one protein group. Protein groups were colored according to their functions. &#x201C;White&#x201D; refers to no protein group in the virus. Asterisks in the bottom refer to membrane proteins. For clarity, the singletons were ignored in the alignment. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-492202286" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The phylogenetic tree of ASFVs and the alignment of their proteomes in plus (left side) and minus strand (right side). Each row refers to the proteome of the virus in the phylogenetic tree; each column refers to one protein group. Protein groups were colored according to their functions. &#x201C;White&#x201D; refers to no protein group in the virus. Asterisks in the bottom refer to membrane proteins. For clarity, the singletons were ignored in the alignment.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F2.medium.gif" width="440" height="210"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F2.medium.gif" width="440" height="210"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">The phylogenetic tree of ASFVs and the alignment of their proteomes in plus (left side) and minus strand (right side). Each row refers to the proteome of the virus in the phylogenetic tree; each column refers to one protein group. Protein groups were colored according to their functions. “White” refers to no protein group in the virus. Asterisks in the bottom refer to membrane proteins. For clarity, the singletons were ignored in the alignment.</span><div class="sb-div caption-clear"></div></div></div><p id="p-14">Analysis of the protein conservation showed that except the functional class of MGFs, the proteins in other functional classes had an average of pairwise sequence identities greater than 90% (Figure S4). The proteins in functional classes of “Other enzymes” and “Replication &amp; transcription” were most conservative, with average pairwise sequence identities larger than 95%. Proteins of these two functional classes also had the smallest ratios of dN/dS (Figure S5), suggesting strong negative selection on them. While the proteins in the functional class of MGF and “Hypothetical proteins” had the largest ratio of dN/dS. The hypothetical proteins had a median dN/dS ratio of 0.82, suggesting strong positive selection on these proteins.</p><p id="p-15">Membrane proteins, which may be located in the inner or outer envelope, were observed to be distributed widely in the proteome of ASFVs (marked with asterisks in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). A total of 67 protein groups belonged to membrane proteins, including 35 in the core protein groups, such as p54 and EP402R. Among the membrane proteins, only 11 protein groups had the known functions, including 8 protein groups in the functional class of “Structural and morphogenesis”, and 1 protein group in each functional class of “Host cell interactions”, “Replication &amp; transcription” and “Other enzymes”.</p><p id="p-16">Thirty-one protein groups were observed to have paralogs (duplicated proteins) in at least one virus (colored in Figure S6). They were mostly located in both ends of the genome. Thirteen of them belonged to MGFs. In addition, two protein groups, “DP71L” and “DP96R”, belonged to the class of “Host cell interactions”. The rest protein groups belonged to either the class of “Proteins with unknown function” or “Hypothetical proteins”. Most of the paralogs were clustered in adjacent positions. Exceptions were observed for some protein groups which were encoded by the first one to three thousands nucleotides in the plus and minus strands, such as the protein group “p01990-3L” (marked with black arrows in Figure S6). Further analysis showed that a segment of 200-3000 bp was exactly the same in the beginning of the plus and minus strands in most viral genomes (Table S4).</p><p id="p-17">Extensive insertion and deletions of proteins were observed in the proteome of ASFVs after alignment. Viruses in the adjacent positions in the phylogenetic tree tended to have similar proteomes. The number of different proteins between different viruses ranged from 1 to 84, with an average of 43, which was about one-fifth of the viral proteome. The differences of the proteome among the viruses were mainly caused by proteins of the class of “Hypothetical proteins”, “Proteins with unknown function” and “MGF” (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>).</p></div><div id="sec-6" class="subsection"><h3>4 Extensive homologous recombination in ASFV genomes</h3><p id="p-18">As numerous indels have been revealed in the ASFV genomes, then, we investigated the mechanism of generating indels. According to the results in previous studies, three factors may contribute to the extensive indels in ASFVs: replication slippage, retrotransposition and recombination <sup><a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a></sup>. Replication slippage mainly produced duplications of short genetic sequences and may cause short indels, but it is unlikely to generate large indels observed in ASFVs. Retrotransposition can result in duplication of large genetic sequences or genes, but the location of duplicates would be randomly distributed in the genome. However, most paralogs shown in Figure S6 were clustered in adjacent positions, thus these paralogs may be not caused by retrotransposition. Besides, no retrotransposons were observed in the analyzed ASFV genomes (as described in Materials and Methods).</p><p id="p-19">Finally, we investigated the role of recombination in the generation of indels in the ASFV genomes. The analyses on the recombination showed that there were a total of 103 recombination events, and each ASFV genome had 3-22 recombination events (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> &amp; Table S5). The virus isolate SouthAfrica79 experienced the largest number of recombination events. On average, each virus experienced 11 recombination events. The sizes of recombination region ranged from 174 to 22,628 bp. The ratio of recombination region in each genome ranged from 2% to 27%. In total, the regions in the ASFV genomes involved in all recombination events covered a total of 101,569 nucleotide positions, accounting for 47% of the aligned genome. Most recombination events happened at both ends, especially at the 5’ end. Interestingly, the recombination event in the aligned genomes was observed to be consistent with the ratio of the gap in the genome (the bottom of <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Almost all the recombination events happened in or close to the gap-rich regions where the indels were observed. The ratios of the gaps in the recombination regions were found to be much larger than those in other regions (Figure S7). Further comparison of the number of indels in the recombination regions and other regions showed that for indels of varing length, such as those greater than 5, 10, or 50 bp, the number of indels in the recombination regions was much larger than those in other regions (Figure S8).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Recombination of ASFV genomes. The black areas indicate the recombination region for each genome. The bottom panel shows the ratio of gap in each position of the aligned genome. The panel uses the grayscale color bar at the bottom-left. The red and blue rectangles in the top-right indicate the coding region of pD345L (Lambda-like exonuclease) and P1192R (DNA topoisomerase II), respectively. The black arrow refers to the recombination event displayed in Figure 4. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-492202286" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Recombination of ASFV genomes. The black areas indicate the recombination region for each genome. The bottom panel shows the ratio of gap in each position of the aligned genome. The panel uses the grayscale color bar at the bottom-left. The red and blue rectangles in the top-right indicate the coding region of pD345L (Lambda-like exonuclease) and P1192R (DNA topoisomerase II), respectively. The black arrow refers to the recombination event displayed in Figure 4.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F3.medium.gif" width="410" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F3.medium.gif" width="410" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Recombination of ASFV genomes. The black areas indicate the recombination region for each genome. The bottom panel shows the ratio of gap in each position of the aligned genome. The panel uses the grayscale color bar at the bottom-left. The red and blue rectangles in the top-right indicate the coding region of pD345L (Lambda-like exonuclease) and P1192R (DNA topoisomerase II), respectively. The black arrow refers to the recombination event displayed in Figure 4.</span><div class="sb-div caption-clear"></div></div></div><p id="p-20"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> illustrates the recombination event in 11 viral isolates (colored in red), including two viral isolates from Africa (SouthAfrica79 and Benin97) and nine viral isolates from Europe. These 11 viral isolates formed a separate lineage in the phylogenetic tree. The recombination region ranged from 133,683 to 142,222 bp, located in the central conservative region of the genome (shown by the black arrow in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). In the phylogenetic tree built with genomic sequences without the recombination regions, the recombinants are the neighbors of a clade containing viruses from Eastern Europe countries (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>); while in the tree built with genomic sequences of the recombination regions, the recombinants are the descendants of Malawi62 from Africa (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="An example of recombination events in 11 ASFVs (colored in red). Figure (A) refers to the maximum-likelihood phylogenetic tree built with genome sequences without the recombination region (133,683-142,222 in the aligned genome). Figure (B) refers to the phylogenetic tree built with genome sequences of the recombination region. The numbers refer to the bootstrap values of nodes in the bootstrapping test with 100 replicates. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-492202286" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;An example of recombination events in 11 ASFVs (colored in red). Figure (A) refers to the maximum-likelihood phylogenetic tree built with genome sequences without the recombination region (133,683-142,222 in the aligned genome). Figure (B) refers to the phylogenetic tree built with genome sequences of the recombination region. The numbers refer to the bootstrap values of nodes in the bootstrapping test with 100 replicates.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F4.medium.gif" width="440" height="338"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F4.medium.gif" width="440" height="338"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">An example of recombination events in 11 ASFVs (colored in red). Figure (A) refers to the maximum-likelihood phylogenetic tree built with genome sequences without the recombination region (133,683-142,222 in the aligned genome). Figure (B) refers to the phylogenetic tree built with genome sequences of the recombination region. The numbers refer to the bootstrap values of nodes in the bootstrapping test with 100 replicates.</span><div class="sb-div caption-clear"></div></div></div><p id="p-21">In addition, the indels introduced directly by the recombination were investigated. The results of comparing the sequence in the recombination regions of recombinants to that in the major parent viruses showed that an average of 37% of differences were caused by the indels in all the recombination events. Further comparison on the proteins showed that in 58 of 103 recombination events, there was at least one different protein encoded by the recombination region of the recombinants from that encoded by the major parent viruses.</p><p id="p-22">Furthermore, the proteins involved in the recombination events were further analyzed. A total of 110 protein groups were involved in the recombination events, including 47 core protein groups and 63 variable protein groups (Table S3). 34 of 110 protein groups belonged to membrane proteins. Four protein groups of “Host cell interactions” (EP153R, A238L, DP96R and DP71L), eight protein groups of “Structure &amp; morphogenesis” and nine protein groups of “Replication &amp; transcription” were involved in the recombination events.</p></div><div id="sec-7" class="subsection"><h3>5 Identification of possible recombinase and DNA topoisomerase in ASFVs</h3><p id="p-23">Interestingly, we found a protein, named pD345L, denoted as the Lambda-like exonuclease, was possibly involved in the recombination because it contained the YqaJ-like viral recombinase domain. The protein pD345L has 345 amino acids and is encoded by the minus strand. It was highly conservative in ASFVs, with an average sequence identity of 96.7% between ASFVs. Even higher level of conservation was observed in the recombinase domain of pD345L, with an average sequence identify of 97.2%. Although the YqaJ-like recombinase domain was extensively distributed in Bacteria, Virus and Eukaryota, it was considered as the viral origin <sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup>. The recombinase domain in ASFV was most similar to that in two giant viruses, Pacmanvirus and Kaumoebavirus (see Materials and Methods) which were possibly distant relatives of ASFVs <sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>,<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>.</p><p id="p-24">Besides, topoisomerase was also reported to be related to homologous recombination. We found that a type II DNA topoisomerase, i.e., P1192R, exists in all analyzed ASFV isolates. P1192R was a protein including 1192 amino acids, and was encoded by the plus strand. P1192R is also conservative in these viruses, with an average sequence identity of 98.3% between ASFVs. Although pD345L and P1192R were encoded by different strands, they were encoded by the genomic sequences in adjacent regions: the former was encoded by the sequences in the positions of 162,236-163,273 (colored in red in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>), while the latter was encoded by the sequences in the positions of 164,743-168,319 (colored in blue in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>). Both enzymes were not involved in any recombination events. The phylogenetic trees for proteins pD345L and P1192R were similar to the tree built with the whole genome (Figure S9).</p></div><div id="sec-8" class="subsection"><h3>6 An abundance of repeated elements in ASFV genomes</h3><p id="p-25">Repeated elements could facilitate the homologous recombination. In this study, lots of repeated elements ranging from 5-100 bp were identified, and then the distribution of the repeated elements in the ASFV genomes was analyzed. As shown in Figure S10, the number of repeated elements in ASFV genomes decreased monotonously as the size of elements increased. Then, the distances between adjacent elements for a given repeated element was investigated (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). As the size of the elements increased from 5 to 10, the average distance between the adjacent elements also increased because the number of repeated elements in the genome decreased. Interestingly, the average distance decreased as the size of the elements increased from 11 to 23; it reached to the minimum (136 bp) when the size was 23; then the distance kept unchanged as the size increased from 23 to 46; finally, it increased as the size of repeated element increased from 47 to 100. It should be noted that the average distance was still less than 400 bp even for the repeated elements of 100 bp. These phenomena suggested that the repeated elements of 11 bp or larger tended to cluster in the genome, especially for those of 23-46 bp.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of the repeated elements. (A)The median distance between adjacent repeated elements versus the size of repeated elements. (B) Number of the repeated elements with the size of 30 bp observed in each genomic position. (C) Comparison of the number of repeated elements (15 bp in length) in the window of 10,000 bp with and without the recombination in viral genomes. For clarity, the natural logarithm of the number of repeated elements was used. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-492202286" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Distribution of the repeated elements. (A)The median distance between adjacent repeated elements versus the size of repeated elements. (B) Number of the repeated elements with the size of 30 bp observed in each genomic position. (C) Comparison of the number of repeated elements (15 bp in length) in the window of 10,000 bp with and without the recombination in viral genomes. For clarity, the natural logarithm of the number of repeated elements was used.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F5.medium.gif" width="440" height="433"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F5.medium.gif" width="440" height="433"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">Distribution of the repeated elements. (A)The median distance between adjacent repeated elements versus the size of repeated elements. (B) Number of the repeated elements with the size of 30 bp observed in each genomic position. (C) Comparison of the number of repeated elements (15 bp in length) in the window of 10,000 bp with and without the recombination in viral genomes. For clarity, the natural logarithm of the number of repeated elements was used.</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">For example, when the size of elements was 30 bp, each genome had a median of 427 types of elements which repeated at least two times in the genome. Some elements appeared for over ten times in the genome, such as the element “AGGCGTTAAACATTAAAATTATTACTACTG” in the viral strain BA71V. The region covered by repeated elements accounted for 1%-3% of the genome in ASFVs. The distance between repeated elements was analyzed and demonstrated to have a median distance of 136 bp, suggesting they tend to cluster in adjacent regions. <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a> shows the distribution of repeated elements in the aligned genome. Most repeated elements were located at both ends of the genome. Besides, there were two clusters of repeated elements in the positions of around 55,000 bp and 120,000 bp (marked by black arrows), respectively.</p><p id="p-27">Finally, the contribution of repeated elements to the recombination was investigated. For elements of 10 or more nucleotides, the number of repeated elements in the windows (2000-10,000bp in length) including the recombination was significantly larger than those without the recombination (Table S6). <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a> shows the comparison of the number of repeated elements (15 bp in length) in the windows of 10,000 bp with and without the recombination in viral genomes. The windows including the recombination had a mean of 194 repeated elements, which was twice of that in the windows without the recombination.</p></div></div><div class="section" id="sec-9"><h2 class="">Discussion</h2><p id="p-28">This work systematically analyzed the genetic diversity of ASFVs. The large genome of the virus enabled the encoding of an abundance of proteins. Each of the functions of the virus in its life circle could be accomplished by multiple proteins. For example, 35, 19, and 7 protein groups were involved in DNA replication and transcription, structure and morphogenesis, and host cell interactions, respectively. On one hand, this multiple protein mechanism could facilitate the efficient control of the host cell by protein-protein interactions, such as inhibiting the transcriptional activation of host immunomodulatory by A238L <sup><a id="xref-ref-3-8" class="xref-bibr" href="#ref-3">3</a></sup> and inhibiting Toll-like receptor 3 signaling pathways by I329L <sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup>; on the other hand, this mechanism could facilitate the precise regulation of the viral activities. For example, the ASFV virus was considered to contain all the enzymes and factors which were required for the transcription and post-treatment of mRNAs <sup><a id="xref-ref-3-9" class="xref-bibr" href="#ref-3">3</a></sup>.</p><p id="p-29">Significant differences were observed among the different proteomes of ASFVs, which may be caused by the following two reasons: i) over 40% of the proteins were non-essential among ASFVs, and ASFVs may have variable number of these proteins; ii) there were 31 genes with replications in ASFV genomes. Diverse proteome among ASFVs may lead to diverse phenotype, such as the diversity in antigen and virulence. The diversity may result in a great challenge for the prevention and control of the virus. For example, the viruses with diverse antigens may need multiple types of vaccines because the effectivity of the cross-protection on viruses may be limited.</p><p id="p-30">Although lots of efforts have been devoted to developing vaccines against ASFVs <sup><a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>-<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">12</a></sup>, unfortunately, most of the attempts have been unsuccessful. The failure could be caused by many factors <sup><a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">12</a></sup>, including the absence of neutralizing antibodies, the diverse antigen-related proteins, the complexity of neutralization, etc. In this study, a total of 65 membrane proteins have been identified. Over eighty percent (80%) of the membrane proteins had unknown functions, many of which may contribute to the antigenic diversity of the virus. Previous studies showed that immunized pigs with the baculovirus expressed hemagglutinin of ASFV were protected against the viral lethal infection <sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup>. The results suggested that incorporation of multiple antigens in the vaccine may provide better protection. Therefore, much more efforts are needed to determine the role of membrane proteins in stimulating neutralizing antibodies, and to investigate the neutralization mechanisms and efficiency of the antibodies.</p><p id="p-31">Indels were found to have larger contribution to the genetic diversity of ASFVs than the point mutations. Compared to point mutations, indels could introduce a larger variation to the genome, and cause a more severe damage to the genome structures, which may lead to the death of viruses. Therefore, only few indels were observed in viruses with small genomes, such as influenza viruses and hepatitis B viruses (HBV). However, it was more robust for the indels to occur inside the viruses with large genomes, such as ASFVs and poxviruses <sup><a id="xref-ref-3-10" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>, because the viruses with large genomes had lots of repeated elements and duplicated proteins (paralogs). Moreover, indels may provide a more efficient way of survival than the point mutations under the natural selection pressure, since the virus with indels could rapidly change its phenotype <sup><a id="xref-ref-3-11" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a></sup>, such as antigen, virulence, or ability of replication and transcription. For example, the deletion of some MGF genes in ASFV could reduce the viral replication or virulence, which may help with the viral infection of soft ticks <sup><a id="xref-ref-3-12" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup>.</p><p id="p-32">Several factors could contribute to the indels and the gene duplications, including replication slippage, retrotransposition, recombination, aneuploidy, polyploidy, etc <sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup>. The replication slippage may introduce short indels which were widely observed in ASFV genomes, but it is unlikely to cause large indels. This study has demonstrated that the ectopic homologous recombination <sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>, during which the segments with unequal length were exchanged (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>), may contribute much to the extensive indels observed in ASFV genomes. As a proof, significant associations were observed between the occurrence of extensive recombination events and the indels. Two factors may facilitate the homologous recombination in ASFVs: firstly, a large amount of clustered repeated elements were observed in ASFV genomes (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>); secondly, all the analyzed ASFVs in this study contained a possible recombinase and DNA topoisomerase, both of which were commonly observed enzymes responsible for homologous recombination. Both of enzymes were very conservative and experienced no recombination, suggesting their important roles in ASFVs. Taken together, the homologous recombination should be the effective strategy of ASFVs to generate the genetic diversity, which further leads to the diverse phenotypes, including antigen, virulence, replication and transcription ability, and the “weapons” of escaping from the host immunity (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Homologous recombination leads to (A) the indels, and (B) the genetic diversity of ASFVs. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-492202286" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Homologous recombination leads to (A) the indels, and (B) the genetic diversity of ASFVs.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F6.medium.gif" width="436" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F6.medium.gif" width="436" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/16/460832/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <span class="caption-title">Homologous recombination leads to (A) the indels, and (B) the genetic diversity of ASFVs.</span><div class="sb-div caption-clear"></div></div></div><p id="p-33">There were some limitations to this study. Firstly, the number of ASFV genomes was limited, which hindered a comprehensive analysis on the evolution of ASFV genomes. Fortunately, the isolates included in this study covered a long time period from 1950 to 2017, and also covered a large area including Africa and Europe, which were the two major areas of the ASFV circulation. Thus the results based on these isolates reflect the genetic diversity of the ASFVs to a large extent. Secondly, the location and size of the indels observed in ASFV genomes may be affected by the sequence alignment algorithm. Two common methods for the alignment of ASFV genomes were used in this study. In both methods, frequent indels were observed, and the indels were demonstrated to be more responsible for the genetic diversity than the point mutations. Thirdly, the proteome of each ASFV was inferred by computational methods. All of the obtained proteins had significant homology to the proteins in the NCBI Protein database, however, most of the proteins in the NCBI Protein database were only predicted without experimental validations. Besides, functions of nearly 70% of ASFV proteins were unknown. Further experimental studies were needed to determine the proteome and the functions in ASFVs <sup><a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a></sup>. Lastly, the extensively repeated elements in ASFV genomes could facilitate the frequent occurrence of recombination events. However, some of recombination events cannot be detected by the recombination detection method because of the exchange between the genomic segments with small indels. Such kinds of recombination events are difficult to detect. Increasing the sensitivity of the recombination detection method can help detect them, but may also bring false positives. Therefore, the sensitivity and specificity should be balanced in the recombination detection methods.</p><p id="p-34">Overall, this work provided a systematic view of the genetic diversity of ASFVs. Extensive homologous recombination detected in this study may contribute much to the widespread indels observed in ASFV genomes, which further lead to the large genetic diversity of ASFVs. The results on the causes of the diversity of ASFVs would help with the understanding of the evolution of the virus and thus facilitate the prevention and control of ASFVs.</p></div><div class="section" id="sec-10"><h2 class="">Materials and Methods</h2><div id="sec-11" class="subsection"><h3>1 ASFV genome and alignment</h3><p id="p-35">All the ASFV genomic sequences with over 160, 000 bp were obtained from NCBI GenBank database on October 7, 2018 <sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup>. After removing the genomic sequences derived from a patent, a total of 36 ASFV genomes were kept in the analysis. The genomic sequences were aligned by MAFFT (version 7.127b) <sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>. To ensure the robustness of the alignment, the traditional tool of CLUSTAL (version 2.1) <sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup> was also used to align these genome sequences.</p></div><div id="sec-12" class="subsection"><h3>2 ORF prediction</h3><p id="p-36">To obtain the proteins encoded by the ASFV genomes, each genome sequence was searched against all the ASFV protein sequences obtained from the NCBI protein database on October 7, 2018, with the help of blastx <sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup>. All genomic regions with significant hits (e-value &lt; 0.001) were checked using a Perl script: overlapping regions in the same coding frame were merged to obtain open reading frames (ORFs) as long as possible; regions without start codon or stop codon were extended upstream or downstream to search for the start or stop codon. Then, the genomic regions which had i) significant hit, ii) both sequence identity and query coverage percentage greater than 60%, iii) both start and stop codons, and iv) over 120 bps, were defined as the candidate ORFs. The candidate ORFs were then translated into proteins using a Perl script. The proteins, which were either completely embedded within another protein, or contained less than 40 amino acids due to early termination of translation, were removed.</p></div><div id="sec-13" class="subsection"><h3>3 Protein grouping</h3><p id="p-37">All the inferred proteins of ASFVs were grouped based on sequence homology using OrthoFinder (version 2.2.7) <sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup> with the default parameters. Manual check was conducted to ensure that each protein group contains one type of protein.</p></div><div id="sec-14" class="subsection"><h3>4 Calculation of the ratio of dN/dS for proteins</h3><p id="p-38">The coding sequences of proteins in each protein group were aligned by codon according to the protein sequence alignment using a Perl script. The ratios of dN/dS between pairwise coding sequences were calculated by yn00 in PAML (version 4.1) <sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup>. The average of pairwise dN/dS ratios was calculated as the ratio of dN/dS for the protein.</p></div><div id="sec-15" class="subsection"><h3>5 Alignment of ASFV proteome</h3><p id="p-39">An ASFV proteome was defined as all the proteins encoded by the ASFV genome. Because both the plus and minus strands could encode proteins, the proteins in a proteome were separated into plus and minus proteome based on the coding strands. Proteome alignment was conducted separately for the plus and the minus proteomes. Firstly, proteins in each proteome were sorted with the order from the 5’ end to the 3’ end of the genome, based on the coding regions of the proteins. Then, the proteomes were aligned using a dynamic programming algorithm. Manual check was conducted to ensure that there was no mismatch of proteins in the alignment.</p></div><div id="sec-16" class="subsection"><h3>6 Function inference and classification of ASFV proteins</h3><p id="p-40">The name of each protein group was obtained from the names of BLAST best hit of proteins included in the protein group. To infer the function of each protein group, the longest protein sequence in each protein group was selected as the representative of the protein group. InterproScan (version 5) <sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup> was used to infer the function of the representative protein sequence. The TMHMM Server (version 2.0) <sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup> was used to predict whether the representative protein had a trans-membrane helix. Membrane proteins were defined as those who had at least one trans-membrane helixes. The functional classification of the proteins was adapted from Dixon’s <sup><a id="xref-ref-3-13" class="xref-bibr" href="#ref-3">3</a></sup> and Alejo’s <sup><a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a></sup> work.</p></div><div id="sec-17" class="subsection"><h3>7 Detection of homologous recombination events</h3><p id="p-41">RDP (version 4) <sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup> was used to detect the recombination events in the aligned ASFV genomes. Multiple methods in RDP were used. Only the recombination events which were detected by at least two methods were used for further analysis.</p></div><div id="sec-18" class="subsection"><h3>8 Evolutionary analysis of YqaJ-like viral recombinase domain</h3><p id="p-42">All viral protein sequences of the family of Yqaj (YqaJ-like viral recombinase domain, ID: PF09588) were downloaded from the Pfam database <sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup> on November 21, 2018. With the help of blastp, the recombinase domain in ASFV was found to be most similar to that in two giant viruses, Pacmanvirus and Kaumoebavirus, with sequence identities equal to 0.35 and 0.30, respectively.</p></div><div id="sec-19" class="subsection"><h3>9 Searching for retrotransposon in ASFV genomes</h3><p id="p-43">All retrotransposons in the databases of RepBase (Version 23.10) <sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup> and TREP <sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup> were downloaded on November 11, 2018. All ASFV genomes were searched against these retrotransposons using blastn. No hits were obtained under the e-value cutoff of 0.001.</p></div><div id="sec-20" class="subsection"><h3>10 Phylogenetic tree inference and visualization</h3><p id="p-44">Maximum-likelihood phylogenetic trees were inferred using MEGA (version 5.0) <sup><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup> with the default values of parameters. Bootstrap analysis was conducted with 100 replicates. The phylogenetic tree was visualized using Denscrope (version 2.4) <sup><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup>.</p></div><div id="sec-21" class="subsection"><h3>11 Statistics analysis</h3><p id="p-45">All the statistical analyses were conducted in R (version 3.2.5) <sup><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></sup>.</p></div></div><div class="section" id="sec-23"><h2 class="">Competing interests</h2><p id="p-47">The authors have declared that no competing interests exist.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-46">This work was supported by the National Key Plan for Scientific Research and Development of China (2016YFD0500300, 2016YFC1200200 and 2017YFD0500104), the National Natural Science Foundation of China (31500126 and 31671371) and the Chinese Academy of Medical Sciences (2016-I2M-1-005).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-2"><p id="p-4">The revised version of the manuscript redefined the proteome of ASFVs,and added the analysis of protein conservation and selective pressure by functional group. More importantly, the association between homologous recombination and indels, and the association between repeated elements and homologous recombination were added.
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list"><li><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.1" data-doi="10.1111/tbed.12695"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arias</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jurado</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gallardo</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fernandez-Pinero</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sanchez-Vizcaino</span>,  <span class="cit-name-given-names">J. M.</span></span> <span class="cit-article-title">Gaps in African swine fever: Analysis and priorities</span>. <abbr class="cit-jnl-abbrev">Transboundary and emerging diseases</abbr> <span class="cit-vol">65</span>, <span class="cit-fpage">235</span>–<span class="cit-lpage">247</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/tbed.12695</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTransboundary%2Band%2Bemerging%2Bdiseases%26rft.volume%253D65%26rft.spage%253D235%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftbed.12695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tbed.12695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.2" data-doi="10.3390/v9050103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Galindo</span>,  <span class="cit-name-given-names">I.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Alonso</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">African Swine Fever Virus: A Review</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">9</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3390/v9050103</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D9%26rft.spage%253DE103%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv9050103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v9050103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.3" data-doi="10.1016/j.virusres.2012.10.020"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dixon</span>,  <span class="cit-name-given-names">L. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chapman</span>,  <span class="cit-name-given-names">D. A. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Netherton</span>,  <span class="cit-name-given-names">C. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Upton</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">African swine fever virus replication and genomics</span>. <abbr class="cit-jnl-abbrev">Virus research</abbr> <span class="cit-vol">173</span>, <span class="cit-fpage">3</span>–<span class="cit-lpage">14</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2012.10.020</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.volume%253D173%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2012.10.020%26rft_id%253Dinfo%253Apmid%252F23142553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2012.10.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23142553&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4 in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.4" data-doi="10.1016/j.virusres.2012.10.030"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Costard</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mur</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lubroth</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanchez-Vizcaino</span>,  <span class="cit-name-given-names">J. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pfeiffer</span>,  <span class="cit-name-given-names">D. U.</span></span> <span class="cit-article-title">Epidemiology of African swine fever virus</span>. <abbr class="cit-jnl-abbrev">Virus research</abbr> <span class="cit-vol">173</span>, <span class="cit-fpage">191</span>–<span class="cit-lpage">197</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2012.10.030</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.volume%253D173%26rft.spage%253D191%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2012.10.030%26rft_id%253Dinfo%253Apmid%252F23123296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2012.10.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23123296&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.5" data-doi="10.1016/j.tvjl.2017.12.025"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanchez-Cordon</span>,  <span class="cit-name-given-names">P. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Montoya</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reis</span>,  <span class="cit-name-given-names">A. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dixon</span>,  <span class="cit-name-given-names">L. K.</span></span> <span class="cit-article-title">African swine fever: A re-emerging viral disease threatening the global pig industry</span>. <abbr class="cit-jnl-abbrev">Vet J</abbr> <span class="cit-vol">233</span>, <span class="cit-fpage">41</span>–<span class="cit-lpage">48</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.tvjl.2017.12.025</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVet%2BJ%26rft.volume%253D233%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tvjl.2017.12.025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tvjl.2017.12.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.6" data-doi="10.1177/0300985809354350"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arzt</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">W. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomsen</span>,  <span class="cit-name-given-names">B. V.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">C. C.</span></span> <span class="cit-article-title">Agricultural Diseases on the Move Early in the Third Millennium</span>. <abbr class="cit-jnl-abbrev">Veterinary pathology</abbr> <span class="cit-vol">47</span>, <span class="cit-fpage">15</span>–<span class="cit-lpage">27</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1177/0300985809354350</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2BPathology%26rft.stitle%253DVeterinary%2BPathology%26rft.aulast%253DArzt%26rft.auinit1%253DJ.%26rft.volume%253D47%26rft.issue%253D1%26rft.spage%253D15%26rft.epage%253D27%26rft.atitle%253DAgricultural%2BDiseases%2Bon%2Bthe%2BMove%2BEarly%2Bin%2Bthe%2BThird%2BMillennium%26rft_id%253Dinfo%253Adoi%252F10.1177%252F0300985809354350%26rft_id%253Dinfo%253Apmid%252F20080480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/0300985809354350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20080480&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274218600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7 in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.7"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">World Organization for animal health</span>. <abbr class="cit-jnl-abbrev">African Swine Fever (ASF) Report N°4: October 5 - 18, 2018</abbr>, &lt;<a href="http://www.oie.int/en/animal-health-in-the-world/information-on-aquatic-and-terrestrial-ani">http://www.oie.int/en/animal-health-in-the-world/information-on-aquatic-and-terrestrial-animal-diseases/african-swine-fever/reports-on-asf/</a>&gt; (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.8" data-doi="10.3201/eid2411.181274"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ge</span>,  <span class="cit-name-given-names">S. Q.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Molecular Characterization of African Swine Fever Virus, China, 2018</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">2131</span>–<span class="cit-lpage">2133</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3201/eid2411.181274</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BInfect%2BDis%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2411.181274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2411.181274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.9"><div class="cit-metadata"><cite>SS, S. &amp; WR, H. <span class="cit-article-title">Antibody response to inactivated preparations of African swine fever virus in pigs</span>. <abbr class="cit-jnl-abbrev">American journal of veterinary research</abbr> <span class="cit-vol">28</span>, <span class="cit-fpage">6</span> (<span class="cit-pub-date">1967</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bveterinary%2Bresearch%26rft.volume%253D28%26rft.spage%253D6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.10" data-doi="10.1016/j.vaccine.2011.04.052"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">King</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Protection of European domestic pigs from virulent African isolates of African swine fever virus by experimental immunisation</span>. <abbr class="cit-jnl-abbrev">Vaccine</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">4593</span>–<span class="cit-lpage">4600</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.vaccine.2011.04.052</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DVaccine%26rft.aulast%253DKing%26rft.auinit1%253DK.%26rft.volume%253D29%26rft.issue%253D28%26rft.spage%253D4593%26rft.epage%253D4600%26rft.atitle%253DProtection%2Bof%2BEuropean%2Bdomestic%2Bpigs%2Bfrom%2Bvirulent%2BAfrican%2Bisolates%2Bof%2BAfrican%2Bswine%2Bfever%2Bvirus%2Bby%2Bexperimental%2Bimmunisation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vaccine.2011.04.052%26rft_id%253Dinfo%253Apmid%252F21549789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.vaccine.2011.04.052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21549789&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.11" data-doi="UNSP e01428-1710.1128/JVI.01428-17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reis</span>,  <span class="cit-name-given-names">A. L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Deletion of the African Swine Fever Virus Gene DP148R Does Not Reduce Virus Replication in Culture but Reduces Virus Virulence in Pigs and Induces High Levels of Protection against Challenge</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">91</span>, doi:UNSP e01428-17 <span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.01428-17</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252FUNSP%2Be01428-1710.1128%252FJVI.01428-17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=UNSP e01428-1710.1128/JVI.01428-17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.12" data-doi="10.1016/j.virusres.2012.10.012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Escribano</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Galindo</span>,  <span class="cit-name-given-names">I.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Alonso</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">Antibody-mediated neutralization of African swine fever virus: Myths and facts</span>. <abbr class="cit-jnl-abbrev">Virus research</abbr> <span class="cit-vol">173</span>, <span class="cit-fpage">101</span>–<span class="cit-lpage">109</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virusres.2012.10.012</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.volume%253D173%26rft.spage%253D101%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2012.10.012%26rft_id%253Dinfo%253Apmid%252F23159730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2012.10.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23159730&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.13"><div class="cit-metadata"><cite>P, G.-P. <span class="cit-etal">et al.</span> <span class="cit-article-title">Neutralizing antibodies to different proteins of African swine fever virus inhibit both virus attachment and internalization</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">70</span>, <span class="cit-fpage">6</span> (<span class="cit-pub-date">1996</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft.volume%253D70%26rft.spage%253D6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.14" data-doi="Artn 1471410.1038/S41598-018-32985-Z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kessler</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The intracellular proteome of African swine fever virus</span>. <abbr class="cit-jnl-abbrev">Scientific reports</abbr> <span class="cit-vol">8</span>, doi:Artn 14714 <span class="cit-pub-id cit-pub-id-doi">10.1038/S41598-018-32985-Z</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2Breports%26rft_id%253Dinfo%253Adoi%252FArtn%2B1471410.1038%252FS41598-018-32985-Z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=Artn 1471410.1038/S41598-018-32985-Z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.15" data-doi="10.1128/JVI.01293-18"><div class="cit-metadata"><cite>A, A., T, M., M, G. &amp; G, A. <span class="cit-article-title">A proteomic atlas of the African swine fever virus particle</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.01293-18</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01293-18%26rft_id%253Dinfo%253Apmid%252F30185597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjE1OiI5Mi8yMy9lMDEyOTMtMTgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8xMi8xNi80NjA4MzIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16 in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.16" data-doi="10.1099/vir.0.83343-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chapman</span>,  <span class="cit-name-given-names">D. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tcherepanov</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Upton</span>,  <span class="cit-name-given-names">C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dixon</span>,  <span class="cit-name-given-names">L. K.</span></span> <span class="cit-article-title">Comparison of the genome sequences of non-pathogenic and pathogenic African swine fever virus isolates</span>. <abbr class="cit-jnl-abbrev">The Journal of general virology</abbr> <span class="cit-vol">89</span>, <span class="cit-fpage">397</span>–<span class="cit-lpage">408</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1099/vir.0.83343-0</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DChapman%26rft.auinit1%253DD.%2BA.%2BG.%26rft.volume%253D89%26rft.issue%253D2%26rft.spage%253D397%26rft.epage%253D408%26rft.atitle%253DComparison%2Bof%2Bthe%2Bgenome%2Bsequences%2Bof%2Bnon-pathogenic%2Band%2Bpathogenic%2BAfrican%2Bswine%2Bfever%2Bvirus%2Bisolates%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.83343-0%26rft_id%253Dinfo%253Apmid%252F18198370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.83343-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18198370&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.17" data-doi="10.1016/j.virol.2010.01.019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Villiers</span>,  <span class="cit-name-given-names">E. P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Phylogenomic analysis of 11 complete African swine fever virus genome sequences</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">400</span>, <span class="cit-fpage">128</span>–<span class="cit-lpage">136</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2010.01.019</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DVirology%26rft.aulast%253Dde%2BVilliers%26rft.auinit1%253DE.%2BP.%26rft.volume%253D400%26rft.issue%253D1%26rft.spage%253D128%26rft.epage%253D136%26rft.atitle%253DPhylogenomic%2Banalysis%2Bof%2B11%2Bcomplete%2BAfrican%2Bswine%2Bfever%2Bvirus%2Bgenome%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2010.01.019%26rft_id%253Dinfo%253Apmid%252F20171711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2010.01.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20171711&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18 in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.18" data-doi="10.1016/j. vetmic.2016.08.018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fraczyk</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Evolution of African swine fever virus genes related to evasion of host immune response</span>. <abbr class="cit-jnl-abbrev">Veterinary microbiology</abbr> <span class="cit-vol">193</span>, <span class="cit-fpage">133</span>–<span class="cit-lpage">144</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.vetmic.2016.08.018</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bmicrobiology%26rft.volume%253D193%26rft.spage%253D133%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.%2Bvetmic.2016.08.018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j. vetmic.2016.08.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19 in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.19" data-doi="10.1371/journal.pone.0069662"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Michaud</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Randriamparany</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Albina</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">Comprehensive phylogenetic reconstructions of African swine fever virus: proposal for a new classification and molecular dating of the virus</span>. <abbr class="cit-jnl-abbrev">PloS one</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">e69662</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0069662</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPloS%2Bone%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0069662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0069662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20 in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.20" data-doi="10.1128/mBio.01280&amp;ndash;15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Origin and Possible Genetic Recombination of the Middle East Respiratory Syndrome Coronavirus from the First Imported Case in China: Phylogenetics and Coalescence Analysis</span>. <abbr class="cit-jnl-abbrev">mBio</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">e01280</span>–<span class="cit-lpage">01215</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/mBio.01280-15</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmBio%26rft.volume%253D6%26rft.spage%253De01280%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.01280%2526ndash%253B15%26rft_id%253Dinfo%253Apmid%252F26350969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.01280&amp;ndash;15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26350969&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21 in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nagy</span>,  <span class="cit-name-given-names">P. D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Bujarski</span>,  <span class="cit-name-given-names">J. J.</span></span> <span class="cit-article-title">Homologous RNA recombination in brome mosaic virus: AU-rich sequences decrease the accuracy of crossovers</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">70</span>, <span class="cit-fpage">415</span>–<span class="cit-lpage">426</span> (<span class="cit-pub-date">1996</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DNagy%26rft.auinit1%253DP.%2BD.%26rft.volume%253D70%26rft.issue%253D1%26rft.spage%253D415%26rft.epage%253D426%26rft.atitle%253DHomologous%2BRNA%2Brecombination%2Bin%2Bbrome%2Bmosaic%2Bvirus%253A%2BAU-rich%2Bsequences%2Bdecrease%2Bthe%2Baccuracy%2Bof%2Bcrossovers%26rft_id%253Dinfo%253Apmid%252F8523555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjcwLzEvNDE1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTIvMTYvNDYwODMyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-460832v2.22" data-doi="10.1146/annurev.phyto.35.1.191"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roossinck</span>,  <span class="cit-name-given-names">M. J.</span></span> <span class="cit-article-title">Mechanisms of plant virus evolution</span>. <abbr class="cit-jnl-abbrev">Annual review of phytopathology</abbr> <span class="cit-vol">35</span>, <span class="cit-fpage">191</span>–<span class="cit-lpage">209</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1146/annurev.phyto.35.1.191</span> (<span class="cit-pub-date">1997</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253DRoossinck%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D191%26rft.epage%253D209%26rft.atitle%253DMECHANISMS%2BOF%2BPLANT%2BVIRUS%2BEVOLUTION%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.phyto.35.1.191%26rft_id%253Dinfo%253Apmid%252F15012521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.phyto.35.1.191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15012521&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XV73800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23 in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.23"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Wikipedia</span>. <abbr class="cit-jnl-abbrev">Homologous recombination</abbr>, &lt;<a href="https://en.wikipedia.org/wiki/Homologous_recombination#In_viruses">https://en.wikipedia.org/wiki/Homologous_recombination#In_viruses</a>&gt; (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24 in text" id="ref-24">↵</a><div class="cit ref-cit ref-web" id="cit-460832v2.24"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">YqaJ protein domain</span>, &lt;<a href="https://en.wikipedia.org/wiki/YqaJ_protein_domain">https://en.wikipedia.org/wiki/YqaJ_protein_domain</a>&gt; (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.25" data-doi="Artn 27810.3390/V8110278"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bajrai</span>,  <span class="cit-name-given-names">L. H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Kaumoebavirus, a New Virus That Clusters with Faustoviruses and Asfarviridae</span>. <abbr class="cit-jnl-abbrev">Viruses-Basel</abbr> <span class="cit-vol">8</span>, doi:Artn 278 <span class="cit-pub-id cit-pub-id-doi">10.3390/V8110278</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses-Basel%26rft_id%253Dinfo%253Adoi%252FArtn%2B27810.3390%252FV8110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=Artn 27810.3390/V8110278&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26 in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.26" data-doi="UNSP e0021210.1128/JVI.00212-17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andreani</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Pacmanvirus, a New Giant Icosahedral Virus at the Crossroads between Asfarviridae and Faustoviruses</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">91</span>, doi:UNSP e00212 <span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.00212-17</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252FUNSP%2Be0021210.1128%252FJVI.00212-17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=UNSP e0021210.1128/JVI.00212-17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27 in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.27" data-doi="10.1007/s00705-010-0894-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Oliveira</span>,  <span class="cit-name-given-names">V. L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A novel TLR3 inhibitor encoded by African swine fever virus (ASFV)</span>. <abbr class="cit-jnl-abbrev">Archives of virology</abbr> <span class="cit-vol">156</span>, <span class="cit-fpage">597</span>–<span class="cit-lpage">609</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s00705-010-0894-7</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Bvirology%26rft.stitle%253DArch%2BVirol%26rft.aulast%253Dde%2BOliveira%26rft.auinit1%253DV.%2BL.%26rft.volume%253D156%26rft.issue%253D4%26rft.spage%253D597%26rft.epage%253D609%26rft.atitle%253DA%2Bnovel%2BTLR3%2Binhibitor%2Bencoded%2Bby%2BAfrican%2Bswine%2Bfever%2Bvirus%2B%2528ASFV%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-010-0894-7%26rft_id%253Dinfo%253Apmid%252F21203785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-010-0894-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21203785&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28 in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.28" data-doi="10.1006/viro.1996.0193"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ruiz-Gonzalvo</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>,  <span class="cit-name-given-names">F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Escribano</span>,  <span class="cit-name-given-names">J. M.</span></span> <span class="cit-article-title">Functional and immunological properties of the baculovirus-expressed hemagglutinin of African swine fever virus</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">218</span>, <span class="cit-fpage">285</span>–<span class="cit-lpage">289</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1006/viro.1996.0193</span> (<span class="cit-pub-date">1996</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DRuiz-Gonzalvo%26rft.auinit1%253DF.%26rft.volume%253D218%26rft.issue%253D1%26rft.spage%253D285%26rft.epage%253D289%26rft.atitle%253DFunctional%2Band%2Bimmunological%2Bproperties%2Bof%2Bthe%2Bbaculovirus-expressed%2Bhemagglutinin%2Bof%2BAfrican%2Bswine%2Bfever%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.1996.0193%26rft_id%253Dinfo%253Apmid%252F8615037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/viro.1996.0193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8615037&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29 in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.29" data-doi="10.1016/j.cell.2012.05.049"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elde</span>,  <span class="cit-name-given-names">N. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Poxviruses Deploy Genomic Accordions to Adapt Rapidly against Host Antiviral Defenses</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">150</span>, <span class="cit-fpage">831</span>–<span class="cit-lpage">841</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2012.05.049</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DElde%26rft.auinit1%253DN.%2BC.%26rft.volume%253D150%26rft.issue%253D4%26rft.spage%253D831%26rft.epage%253D841%26rft.atitle%253DPoxviruses%2Bdeploy%2Bgenomic%2Baccordions%2Bto%2Badapt%2Brapidly%2Bagainst%2Bhost%2Bantiviral%2Bdefenses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.05.049%26rft_id%253Dinfo%253Apmid%252F22901812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.05.049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22901812&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308002300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30 in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.30" data-doi="10.1128/JVI.78.1.9-22.2004"><div class="cit-metadata"><cite>TG, B., Z, L., JG, N., DL, R. &amp; L, Z. <span class="cit-article-title">African swine fever virus multigene family 360 genes affect virus replication and generalization of infection in Ornithodoros porcinus ticks</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr> <span class="cit-vol">78</span>, <span class="cit-fpage">9</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DKreppel%26rft.auinit1%253DF.%26rft.volume%253D78%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D22%26rft.atitle%253DLong-Term%2BTransgene%2BExpression%2Bin%2BProliferating%2BCells%2BMediated%2Bby%2BEpisomally%2BMaintained%2BHigh-Capacity%2BAdenovirus%2BVectors%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.1.9-22.2004%26rft_id%253Dinfo%253Apmid%252F14671083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjY6Ijc4LzEvOSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzEyLzE2LzQ2MDgzMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31 in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.31" data-doi="10.1016/S0169-5347(03)00033-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">J. Z.</span></span> <span class="cit-article-title">Evolution by gene duplication: an update</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">292</span>–<span class="cit-lpage">298</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0169-5347(03)00033-8</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D18%26rft.spage%253D292%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252803%252900033-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-5347(03)00033-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32 in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.32" data-doi="10.1038/35039531"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Freitas-Junior</span>,  <span class="cit-name-given-names">L. H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Frequent ectopic recombination of virulence factor genes in telomeric chromosome clusters of P-falciparum</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">407</span>, <span class="cit-fpage">1018</span>–<span class="cit-lpage">1022</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/35039531</span> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFreitas-Junior%26rft.auinit1%253DL.%2BH.%26rft.volume%253D407%26rft.issue%253D6807%26rft.spage%253D1018%26rft.epage%253D1022%26rft.atitle%253DFrequent%2Bectopic%2Brecombination%2Bof%2Bvirulence%2Bfactor%2Bgenes%2Bin%2Btelomeric%2Bchromosome%2Bclusters%2Bof%2BP.%2Bfalciparum.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35039531%26rft_id%253Dinfo%253Apmid%252F11069183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35039531&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11069183&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000090032500047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33 in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.33" data-doi="10.1093/nar/gkv1290"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Agarwala</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Database resources of the National Center for Biotechnology Information</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">44</span>, <span class="cit-fpage">D7</span>–<span class="cit-lpage">D19</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkv1290</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1290%26rft_id%253Dinfo%253Apmid%252F26615191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1290&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26615191&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34 in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.34" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Katoh</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Standley</span>,  <span class="cit-name-given-names">D. M.</span></span> <span class="cit-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability</span>. <abbr class="cit-jnl-abbrev">Molecular biology and evolution</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">772</span>–<span class="cit-lpage">780</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/mst010</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Band%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23329690&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317002300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35 in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.35" data-doi="10.1093/bioinformatics/btm404"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Larkin</span>,  <span class="cit-name-given-names">M. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Clustal W and Clustal X version 2.0</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">2947</span>–<span class="cit-lpage">2948</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btm404</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm404%26rft_id%253Dinfo%253Apmid%252F17846036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm404&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17846036&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251197700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36 in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.36" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altschul</span>,  <span class="cit-name-given-names">S. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gish</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Myers</span>,  <span class="cit-name-given-names">E. W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lipman</span>,  <span class="cit-name-given-names">D. J.</span></span> <span class="cit-article-title">Basic local alignment search tool</span>. <abbr class="cit-jnl-abbrev">Journal of molecular biology</abbr> <span class="cit-vol">215</span>, <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span> (<span class="cit-pub-date">1990</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37 in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.37" data-doi="10.1186/s13059-015-0721-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Emms</span>,  <span class="cit-name-given-names">D. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kelly</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy</span>. <abbr class="cit-jnl-abbrev">Genome biology</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">157</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/s13059-015-0721-2</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.volume%253D16%26rft.spage%253D157%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0721-2%26rft_id%253Dinfo%253Apmid%252F26243257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0721-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26243257&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38 in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.38" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z. H.</span></span> <span class="cit-article-title">PAML 4: Phylogenetic analysis by maximum likelihood</span>. <abbr class="cit-jnl-abbrev">Molecular biology and evolution</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">1586</span>–<span class="cit-lpage">1591</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/msm088</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Band%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17483113&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248848400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39 in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.39" data-doi="10.1093/nar/gki442"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Quevillon</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">InterProScan: protein domains identifier</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">33</span>, <span class="cit-fpage">W116</span>–<span class="cit-lpage">120</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gki442</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DQuevillon%26rft.auinit1%253DE.%26rft.volume%253D33%26rft.issue%253Dsuppl_2%26rft.spage%253DW116%26rft.epage%253DW120%26rft.atitle%253DInterProScan%253A%2Bprotein%2Bdomains%2Bidentifier%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki442%26rft_id%253Dinfo%253Apmid%252F15980438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki442&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15980438&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230271400020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40 in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.40" data-doi="10.1006/jmbi.2000.4315"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krogh</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Larsson</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">von Heijne</span>,  <span class="cit-name-given-names">G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sonnhammer</span>,  <span class="cit-name-given-names">E. L.</span></span> <span class="cit-article-title">Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes</span>. <abbr class="cit-jnl-abbrev">Journal of molecular biology</abbr> <span class="cit-vol">305</span>, <span class="cit-fpage">567</span>–<span class="cit-lpage">580</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1006/jmbi.2000.4315</span> (<span class="cit-pub-date">2001</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DKrogh%26rft.auinit1%253DA.%26rft.volume%253D305%26rft.issue%253D3%26rft.spage%253D567%26rft.epage%253D580%26rft.atitle%253DPredicting%2Btransmembrane%2Bprotein%2Btopology%2Bwith%2Ba%2Bhidden%2BMarkov%2Bmodel%253A%2Bapplication%2Bto%2Bcomplete%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.2000.4315%26rft_id%253Dinfo%253Apmid%252F11152613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.2000.4315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11152613&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167760800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41 in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.41" data-doi="10.1093/bioinformatics/16.6.562"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rybicki</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">RDP: detection of recombination amongst aligned sequences</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">562</span>–<span class="cit-lpage">563</span> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DMartin%26rft.auinit1%253DD.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D562%26rft.epage%253D563%26rft.atitle%253DRDP%253A%2Bdetection%2Bof%2Brecombination%2Bamongst%2Baligned%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F16.6.562%26rft_id%253Dinfo%253Apmid%252F10980155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/16.6.562&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10980155&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089451500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42 in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.42" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Finn</span>,  <span class="cit-name-given-names">R. D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The Pfam protein families database: towards a more sustainable future</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">44</span>, <span class="cit-fpage">D279</span>–<span class="cit-lpage">D285</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkv1344</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F26673716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26673716&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43 in text" id="ref-43">↵</a><div class="cit ref-cit ref-web" id="cit-460832v2.43"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Repbase</span>, &lt;<a href="https://girinst.org/">https://girinst.org/</a>&gt; (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44 in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.44" data-doi="10.1038/nrg2165"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wicker</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A unified classification system for eukaryotic transposable elements</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Genetics</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">973</span>–<span class="cit-lpage">982</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nrg2165</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMathew%26rft.auinit1%253DC.%2BG.%26rft.volume%253D8%26rft.issue%253D12%26rft.spage%253D973%26rft.epage%253D982%26rft.atitle%253DA%2Bunified%2Bclassification%2Bsystem%2Bfor%2Beukaryotic%2Btransposable%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2165%26rft_id%253Dinfo%253Apmid%252F17984973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2165&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17984973&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45 in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.45" data-doi="10.1093/molbev/mst197"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tamura</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stecher</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peterson</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Filipski</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">MEGA6: Molecular Evolutionary Genetics Analysis version 6.0</span>. <abbr class="cit-jnl-abbrev">Molecular biology and evolution</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">2725</span>–<span class="cit-lpage">2729</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/mst197</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Band%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst197%26rft_id%253Dinfo%253Apmid%252F24132122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24132122&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327793000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46 in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-460832v2.46" data-doi="10.1186/1471-2105-8-460"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huson</span>,  <span class="cit-name-given-names">D. H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Dendroscope: An interactive viewer for large phylogenetic trees</span>. <abbr class="cit-jnl-abbrev">Bmc Bioinformatics</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">460</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/1471-2105-8-460</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHuson%26rft.auinit1%253DD.%2BH.%26rft.volume%253D8%26rft.spage%253D460%26rft.epage%253D460%26rft.atitle%253DDendroscope%253A%2BAn%2Binteractive%2Bviewer%2Bfor%2Blarge%2Bphylogenetic%2Btrees.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-460%26rft_id%253Dinfo%253Apmid%252F18034891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18034891&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F16%2F460832.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47 in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-460832v2.47"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Core Team</span>, <span class="cit-source">R: A language and environment for statistical computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>, &lt; <a href="https://www.R-project.org/">https://www.R-project.org/</a>&gt; (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/460832v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F460832v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/452284v2" title="Resistance from Afar: Distal Mutation V36M Allosterically Modulates the Active Site to Accentuate Drug Resistance in HCV NS3/4A Protease" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;December 16, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/460832v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/460832v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/460832v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/460832v2" class="active" data-icon-position="" data-hide-link-title="0">Homologous Recombination as an Evolutionary Force in African Swine Fever Viruses</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/147645" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-GuMPzI4lu6fu2l36vIM_OHIk6PGHBfrFe9TzYN_nXvU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="147645" id="node147645--2" data-pisa="biorxiv;460832v2" data-pisa-master="biorxiv;460832" data-seqnum="147645" data-apath="/biorxiv/early/2018/12/16/460832.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Homologous Recombination as an Evolutionary Force in African Swine Fever Viruses</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Zhaozhong</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Chao-Ting</span> <span class="nlm-surname">Xiao</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Yunshi</span> <span class="nlm-surname">Fan</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Zena</span> <span class="nlm-surname">Cai</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Congyu</span> <span class="nlm-surname">Lu</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Gaihua</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Taijiao</span> <span class="nlm-surname">Jiang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Yongjun</span> <span class="nlm-surname">Tan</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Yousong</span> <span class="nlm-surname">Peng</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">460832; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/460832 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/460832v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/460832v2&amp;title=Homologous%20Recombination%20as%20an%20Evolutionary%20Force%20in%20African%20Swine%20Fever%20Viruses" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/460832v2&amp;title=Homologous%20Recombination%20as%20an%20Evolutionary%20Force%20in%20African%20Swine%20Fever%20Viruses" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/460832v2&amp;text=Homologous%20Recombination%20as%20an%20Evolutionary%20Force%20in%20African%20Swine%20Fever%20Viruses" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/460832v2&amp;title=Homologous%20Recombination%20as%20an%20Evolutionary%20Force%20in%20African%20Swine%20Fever%20Viruses" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/460832v2&amp;t=Homologous%20Recombination%20as%20an%20Evolutionary%20Force%20in%20African%20Swine%20Fever%20Viruses" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/460832v2&amp;title=Homologous%20Recombination%20as%20an%20Evolutionary%20Force%20in%20African%20Swine%20Fever%20Viruses" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/460832v2&amp;title=Homologous%20Recombination%20as%20an%20Evolutionary%20Force%20in%20African%20Swine%20Fever%20Viruses" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="147645" id="node147645--2" data-pisa="biorxiv;460832v2" data-pisa-master="biorxiv;460832" data-seqnum="147645" data-apath="/biorxiv/early/2018/12/16/460832.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Homologous Recombination as an Evolutionary Force in African Swine Fever Viruses</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Zhaozhong</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Chao-Ting</span> <span class="nlm-surname">Xiao</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Yunshi</span> <span class="nlm-surname">Fan</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Zena</span> <span class="nlm-surname">Cai</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Congyu</span> <span class="nlm-surname">Lu</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Gaihua</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Taijiao</span> <span class="nlm-surname">Jiang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Yongjun</span> <span class="nlm-surname">Tan</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Yousong</span> <span class="nlm-surname">Peng</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">460832; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/460832 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Homologous%20Recombination%20as%20an%20Evolutionary%20Force%20in%20African%20Swine%20Fever%20Viruses&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Zhu%2C+Zhaozhong&amp;rft.au=Xiao%2C+Chao-Ting&amp;rft.au=Fan%2C+Yunshi&amp;rft.au=Cai%2C+Zena&amp;rft.au=Lu%2C+Congyu&amp;rft.au=Zhang%2C+Gaihua&amp;rft.au=Jiang%2C+Taijiao&amp;rft.au=Tan%2C+Yongjun&amp;rft.au=Peng%2C+Yousong"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/147645/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/147645/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/147645/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/147645/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/147645/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/147645/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/147645/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/147645/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/147645/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/147645/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/147645/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/147645/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/460832v2&amp;count=horizontal&amp;via=&amp;text=Homologous%20Recombination%20as%20an%20Evolutionary%20Force%20in%20African%20Swine%20Fever%20Viruses&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/460832v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/460832v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/460832v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
