<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS Comput Biol</span><span class="iso-abbrev" title="iso-abbrev">PLoS Comput. Biol</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">ploscomp</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS Computational Biology</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1553-734X</span><span class="issn-epub" tagx="issn" title="issn-epub">1553-7358</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7316228">7316228</a></span><span class="publisher-id" title="publisher-id">PCOMPBIOL-D-19-01822</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pcbi.1007892">10.1371/journal.pcbi.1007892</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Orthomyxoviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Influenza viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Influenza A virus</span>: <span class="subject" title="subject">H1N1</span></div> </div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Orthomyxoviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Influenza viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Influenza A virus</span>: <span class="subject" title="subject">H1N1</span></div> </div> </div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and health sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and laboratory medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Orthomyxoviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Influenza viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Influenza A virus</span>: <span class="subject" title="subject">H1N1</span></div> </div> </div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Orthomyxoviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Influenza viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Influenza A virus</span>: <span class="subject" title="subject">H1N1</span></div> </div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span>: <span class="subject" title="subject">Evolutionary Immunology</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Evolution</span>: <span class="subject" title="subject">Viral Evolution</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organismal Evolution</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Evolution</span>: <span class="subject" title="subject">Viral Evolution</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Virology</span>: <span class="subject" title="subject">Viral Evolution</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Orthomyxoviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Influenza viruses</span>: <span class="subject" title="subject">Influenza A virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Orthomyxoviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Influenza viruses</span>: <span class="subject" title="subject">Influenza A virus</span></div> </div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and health sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and laboratory medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Orthomyxoviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Influenza viruses</span>: <span class="subject" title="subject">Influenza A virus</span></div> </div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Orthomyxoviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Influenza viruses</span>: <span class="subject" title="subject">Influenza A virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Processes</span>: <span class="subject" title="subject">Evolutionary Rate</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Physical Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Physics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Electricity</span>: <span class="subject" title="subject">Electrostatics</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Gene Identification and Analysis</span>: <span class="subject" title="subject">Mutation Detection</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Glycobiology</span>: <span class="subject" title="subject">Glycosylation</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Proteins</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Post-Translational Modification</span>: <span class="subject" title="subject">Glycosylation</span></div> </div> </div> </div> </div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Implications of localized charge for human influenza A H1N1 hemagglutinin evolution: Insights from deep mutational scans</div> <div class="alt-title" title="alt-title">
Implications of localized charge for human influenza A H1N1 hemagglutinin evolution</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-2217-3071</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saad-Roy</span><span class="given-names" tagx="given-names" title="given-names">Chadi M.</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Writing â€" original draft</span><span class="role" tagx="role" title="role">Writing â€" review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-0412-150X</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arinaminpathy</span><span class="given-names" tagx="given-names" title="given-names">Nimalan</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Writing â€" review &amp;amp; editing</span><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0001-7384-2821</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wingreen</span><span class="given-names" tagx="given-names" title="given-names">Ned S.</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Writing â€" review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levin</span><span class="given-names" tagx="given-names" title="given-names">Simon A.</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Writing â€" review &amp;amp; editing</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-4411-1330</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akey</span><span class="given-names" tagx="given-names" title="given-names">Joshua M.</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Writing â€" review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">Bryan T.</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Supervision</span><span class="role" tagx="role" title="role">Writing â€" original draft</span><span class="role" tagx="role" title="role">Writing â€" review &amp;amp; editing</span><a href="#aff004"><sup>4</sup></a><a href="#aff005"><sup>5</sup></a><a href="#aff006"><sup>6</sup></a><a href="#cor001">*</a></span></div><span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America</span></span><span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">MRC Centre for Global Infectious Disease Analysis, Department of Infectious Disease Epidemiology, School of Public Health, Imperial College London, London, United Kingdom</span></span><span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">Department of Molecular Biology, Princeton University, Princeton, New Jersey, United States of America</span></span><span class="citation_author_institution" id="aff004">[4], <span class="addr-line" title="addr-line">Department of Ecology and Evolutionary Biology, Princeton University, Princeton, New Jersey, United States of America</span></span><span class="citation_author_institution" id="aff005">[5], <span class="addr-line" title="addr-line">Woodrow Wilson School of Public and International Affairs, Princeton University, Princeton, New Jersey, United States of America</span></span><span class="citation_author_institution" id="aff006">[6], <span class="addr-line" title="addr-line">Division of International Epidemiology and Population Studies, Fogarty International Center, National Institutes of Health, Bethesda, Maryland, United States of America</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antia</span><span class="given-names" tagx="given-names" title="given-names">Rustom</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Emory University, UNITED STATES</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors have declared that no competing interests exist.</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">csaadroy@princeton.edu</span> (CMSR); <span class="email" tagx="email" title="email">grenfell@princeton.edu</span> (BTG)</div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-6</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-6-6</span></span><span class="volume" tagx="volume" title="volume">16</span><span class="issue" tagx="issue" title="issue">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1007892</span><span class="history" title="history"><span class="received" title="received">received: 2019-10-18</span><span class="accepted" title="accepted">accepted: 2020-4-21</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Saad-Roy et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></div><a class="self-uri" href="pcbi.1007892.pdf" title="self-uri">pcbi.1007892.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Seasonal influenza A viruses of humans evolve rapidly due to strong selection pressures from host immune responses, principally on the hemagglutinin (HA) viral surface protein. Based on mouse transmission experiments, a proposed mechanism for immune evasion consists of increased avidity to host cellular receptors, mediated by electrostatic charge interactions with negatively charged cell surfaces. In support of this, the HA charge of the globally circulating H3N2 has increased over time since its pandemic. However, the same trend was not seen in H1N1 HA sequences. This is counter-intuitive, since immune escape due to increased avidity (due itself to an increase in charge) was determined experimentally. Here, we explore whether patterns of local charge of H1N1 HA can explain this discrepancy and thus further associate electrostatic charge with immune escape and viral evolutionary dynamics. Measures of site-wise functional selection and expected charge computed from deep mutational scan data on an early H1N1 HA yield a striking division of residues into three groups, separated by charge. We then explored evolutionary dynamics of these groups from 1918 to 2008. In particular, one group increases in net charge over time and consists of sites that are evolving the fastest, that are closest to the receptor binding site (RBS), and that are exposed to solvent (i.e., on the surface). By contrast, another group decreases in net charge and consists of sites that are further away from the RBS and evolving slower, but also exposed to solvent. The last group consists of those sites in the HA core, with no change in net charge and that evolve very slowly. Thus, there is a group of residues that follows the same trend as seen for the entire H3N2 HA. It is possible that the H1N1 HA is under other biophysical constraints that result in compensatory decreases in charge elsewhere on the protein. Our results implicate localized charge in HA interactions with host cells, and highlight how deep mutational scan data can inform evolutionary hypotheses.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Author summary</div> <p>The hemagglutinin (HA) surface protein of influenza A viruses evolves rapidly to evade host immunity, leading to sizable yearly epidemics in human populations. Previous transmission experiments with H1N1 in mice have tied immune escape to an increase in HA avidity for cellular receptors, mediated by electrostatic charge. Furthermore, retrospective sequence analyses from a previous study confirmed that the HA of circulating global H3N2 has increased in net charge, yet surprisingly, that of H1N1 has not varied significantly. How is a stable net charge related to local patterns of H1N1 HA charge in response to selection? To elucidate the role of local electrostatic charge in host-virus interactions, we investigate characteristics of local charge on the H1N1 HA using functional data from deep mutational scan experiments. Combining measures of functional selection and expected charge at each site on DMS data from a 1933 H1N1 HA yields a striking visual pattern that identifies three groups of sites that have different biophysical properties and that prove to have distinct evolutionary patterns in natural human sequences. Essentially, we find evidence for an increase in charge near the receptor binding site and for compensatory changes elsewhere on the protein. Thus, our findings may reconcile disparate results from transmission experiments and natural sequence analyses, and highlight the importance of local properties of the HA protein. Overall, our findings further support the hypothesis of immune escape due to increased HA avidity to cellular receptors mediated by electrostatic charge. More generally, our work illustrates a novel method of leveraging deep mutational scans in conjunction with natural sequences to shed light on virus-host interactions.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000865</span><span class="institution" title="institution">Bill and Melinda Gates Foundation</span></div> </div><span class="award-id" tagx="award-id" title="award-id">OPP1091919</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">Bryan T.</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000913</span><span class="institution" title="institution">James S. McDonnell Foundation</span></div> </div><span class="award-id" tagx="award-id" title="award-id">21st Century Science Initiative Collaborative Award in Understanding Dynamic and Multi-scale Systems</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levin</span><span class="given-names" tagx="given-names" title="given-names">Simon A.</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100000038</span><span class="institution" title="institution">Natural Sciences and Engineering Research Council of Canada</span></div> </div><span class="award-id" tagx="award-id" title="award-id">Postgraduate Scholarship-Doctoral</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-2217-3071</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saad-Roy</span><span class="given-names" tagx="given-names" title="given-names">Chadi M.</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/100000002</span><span class="institution" title="institution">National Institutes of Health</span></div> </div><span class="award-id" tagx="award-id" title="award-id">RAPIDD program of the Science and Technology Directorate Department of Homeland Security and the Fogarty International Center</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">Bryan T.</span></span></span></div> <div class="funding-statement" title="funding-statement">
Bill and Melinda Gates Foundation (<a href="https://www.gatesfoundation.org/">https://www.gatesfoundation.org/</a>) Grant OPP1091919 (BTG); the RAPIDD program of the Science and Technology Directorate Department of Homeland Security and the Fogarty International Center (<a href="https://www.fic.nih.gov/Pages/Default.aspx">https://www.fic.nih.gov/Pages/Default.aspx</a>), National Institutes of Health (BTG); the National Science Foundation (<a href="https://www.nsf.gov/">https://www.nsf.gov/</a>) Grant CCF 1917819 (SAL); the James S. McDonnell Foundation (<a href="https://www.jsmf.org/">https://www.jsmf.org/</a>) 21st Century Science Initiative Collaborative Award in Understanding Dynamic and Multi-scale Systems (SAL), and the Natural Sciences and Engineering Research Council (NSERC) of Canada (<a href="http://www.nserc-crsng.gc.ca/index_eng.asp">http://www.nserc-crsng.gc.ca/index_eng.asp</a>) through a Postgraduate Scholarship-Doctoral to C.M.S.-R. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="All data used are publicly available. The Supporting Information file Analyses.zip contains codes and data for analyses, and also indicates where data can be found." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>All data used are publicly available. The Supporting Information file Analyses.zip contains codes and data for analyses, and also indicates where data can be found.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Influenza A viruses (IAVs) are responsible for a major burden of disease in human populations [<a href="#pcbi.1007892.ref003">3</a>]. IAVs are segmented RNA viruses, generally characterized by the two surface glycoproteins hemagglutinin (HA) and neuraminidase (NA) [<a href="#pcbi.1007892.ref004">4</a>]. Much of the acquired humoral host immune response to IAVs acts against the HA, which is responsible for cell entry. The HA consists of two subunits, HA1 and HA2, the former containing the receptor binding site (RBS) and exhibiting greater variation, while the latter is more antigenically conserved. Phylogenetically, HAs of IAVs form two monophyletic clades, known as group 1 and 2, and each group contains multiple subtypes, including the HA of H1N1 and H3N2, respectively [<a href="#pcbi.1007892.ref005">5</a>]. These viruses evolve via two mechanisms. Strong selective pressures from the host immune system, combined with error-prone RNA polymerases, lead to seasonal variants of a given subtype emerging through â€œdriftâ€� [<a href="#pcbi.1007892.ref006">6</a>]. These yearly epidemics contrast to pandemics caused by a â€œshiftâ€� that occurs when IAVs of different animal species reassort, resulting in viruses with antigens to which the human population has little prior immunity [<a href="#pcbi.1007892.ref007">7</a>].</p> <p>As antigenic drift is driven by selection pressures exerted by previously infected hosts, characterizing HA-antibody interactions is crucial to understand successful immune escape. Caton and coauthors [<a href="#pcbi.1007892.ref008">8</a>] experimentally determined the regions on the H1N1 HA onto which antibodies bind, and grouped these residues into the Sa, Sb, Ca1, Ca2, and Cb antigenic sites (<a href="#pcbi.1007892.g001">Fig 1A</a>). Individual host immune responses to IAV infections are guided by infection history [<a href="#pcbi.1007892.ref009">9</a>]. A key question is to determine the regions on the HA that are most targeted by antibodies. Hemagglutination inhibition (HI) and enzyme-linked immunosorbent (ELISA) assays can establish an immunodominance hierarchy, indicating which sites are most strongly targeted by host antibodies. With the PR8 H1N1 strain in mice, Angeletti and colleagues [<a href="#pcbi.1007892.ref010">10</a>] found this hierarchy to be Sb (highest), Sa, Cb, Ca2, and Ca1 (lowest). In humans and with the A/Michigan/45/2015 H1N1 strain, Liu and coauthors [<a href="#pcbi.1007892.ref011">11</a>] concluded that Sb was the most immunodominant, followed by Sa, Ca1, Ca2, and Cb; Sb and Sa were significantly immunodominant in contrast to Ca1, Ca2, and Cb. In yet another study, Koel and colleagues [<a href="#pcbi.1007892.ref012">12</a>] used viruses with HA mutants to determine if single amino acid changes could lead to immune escape. These experiments identified sites near the RBS responsible for the strongest reduction in HI titers for H1N1 (before 2009). Furthermore, these results tie to statistical findings with H3N2 data which showed that, in contrast to known epitope designations, two residue-specific covariates were most important to explain dN/dS scores: These factors are a residueâ€™s distance to the RBS, in addition to its relative solvent accessibility which distinguishes surface versus internal residues [<a href="#pcbi.1007892.ref013">13</a>].</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1007892.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
Schematic of host-virus interactions at the hemagglutinin (HA) viral surface protein.</div> <p>(<i>A</i>) The structure of the H1N1 HA. The immunodominant HA head (HA1) contains the antigenic sites [<a href="#pcbi.1007892.ref008">8</a>] onto which antibodies bind to neutralize the virus, whereas the HA stalk (HA2) is more conserved. (<i>B</i>) Immune escape due to increased avidity mediated by electrostatic charge, according to transmission experiments by Hensley and coauthors [<a href="#pcbi.1007892.ref001">1</a>]. To infect host cells, virions must first successfully enter cells. To accomplish this, the HA protein binds to a cellular receptor, <i>i.e.</i> sialic acid, on the surface of the cell. An increase in charge is posited to increase HA avidity and therefore facilitate HA binding, leading to successful immune escape. (<i>C</i>) Patterns of charge over time in natural human sequences. Arinaminpathy and Grenfell [<a href="#pcbi.1007892.ref002">2</a>] showed that the H3 increases over time, but that the H1 is constant. (<i>Inset</i>) Schematic of the net charge trends over time for the HAs of H1N1 and H3N2 viruses in natural human sequences.</p>   </div> <p>With an aim toward disease control and pandemic preparedness, there have been considerable efforts to identify and characterize IAV dynamics across scales from molecular to global by confronting theory with data [<a href="#pcbi.1007892.ref014">14</a>]. However, the role of molecular characteristics such as glycosylation and electrostatic charge in evolutionary dynamics have yet to be fully resolved [<a href="#pcbi.1007892.ref015">15</a>]. Glycosylation is known to be involved in IAV evolution [<a href="#pcbi.1007892.ref016">16</a>], and Kobayashi and Suzuki [<a href="#pcbi.1007892.ref017">17</a>] examined net charge in the context of N-linked glycosylation patterns over time. Mechanistically, electrostatic charge may play an important role in biomolecular interactions between virions, host cells, and antibodies. Local charge could also be involved in HA interactions with host immunity. As the cell surface is negatively charged, an increase in net charge is posited to aid the HA in binding to the cell surface, and thereby lead to successful immune escape due to more efficient viral entry [<a href="#pcbi.1007892.ref018">18</a>]. Hensley and colleagues [<a href="#pcbi.1007892.ref001">1</a>] illustrated the importance of HA avidity to the RBS in immune escape, and tied this to electrostatic charge by examining mutations leading to escape (<a href="#pcbi.1007892.g001">Fig 1B</a>). Furthermore, the actual number of charged amino acids in H3 was shown to be greatest in the dominant epitope [<a href="#pcbi.1007892.ref019">19</a>]. By considering the five charged amino acids and summing across IAV sequences, Arinaminpathy and Grenfell [<a href="#pcbi.1007892.ref002">2</a>] found that the net charge of the H3N2 HA increased over time (<a href="#pcbi.1007892.g001">Fig 1C</a>). However, no such increase was found for the H1N1 HA [<a href="#pcbi.1007892.ref002">2</a>] (<a href="#pcbi.1007892.g001">Fig 1C</a>). In addition to starkly contrasting with the results from natural sequence analyses with H3N2 HAs, these findings are particularly puzzling since the transmission experiments in mice by Hensley and coauthors [<a href="#pcbi.1007892.ref001">1</a>] were with H1N1 viruses. Arinaminpathy and Grenfell [<a href="#pcbi.1007892.ref002">2</a>] proposed other explanations to account for their findings: in particular, they highlighted the possibility that the observed trends are covariates of other, more important factors, or that immune escape could also be the result of other mechanisms.</p> <p>Could patterns of localized electrostatic charge within the HA reconcile these findings? Perhaps the simplest explanation for the discrepancy between H1N1 and H3N2 HA net charge over time is that the H1N1 HA could be less tolerant to an overly positive net charge, and thus an increase in charge must be accompanied by a decrease elsewhere to keep the net charge relatively constant and preserve proper protein folding and function (more generally, for a discussion of charge and protein stability, see, e.g. [<a href="#pcbi.1007892.ref020">20</a>]). Here, â€œfunctionâ€� refers to cell entry and viral replication in the absence of immune selection. Therefore, this hypothesis would posit that while certain regions are under strong immune selection to become more positive (in agreement with the immune escape model of Hensley and coauthors [<a href="#pcbi.1007892.ref001">1</a>]), an additional selective pressure acts elsewhere to keep the net H1N1 HA charge constant (thus reflecting the pattern observed by Arinaminpathy and Grenfell [<a href="#pcbi.1007892.ref002">2</a>]). To investigate this hypothesis, local patterns in the HA charge must be examined. Yet, selection for immune escape may modulate residues independent of charge due to function. Thus, while it would be possible to simply group residues based on individual distance to the receptor binding site and to examine their local charge, such an approach would not appropriately account for the functional constraints (on individual residues) that were present from early on in H1N1 HA evolution. Here, we aim to separate local changes in charge due to protein function versus evolutionary pressures from host immune systems, and this requires estimates of local functional selection on the HA protein.</p> <p>Recent developments in sequencing technologies have led to data-rich experimental studies examining protein evolution. To uncover the fitness landscapes of HA proteins in the absence of immune selection, deep mutational scans (<i>e.g.</i>, Araya and colleagues [<a href="#pcbi.1007892.ref021">21</a>], Doud and Bloom [<a href="#pcbi.1007892.ref022">22</a>]) can interrogate HA functional constraints in a site-wise fashion, yielding probability distributions for residue preference at each site, <i>e.g.</i>, for the HA of a particular H1N1 strain. Essentially, all possible âˆ¼10,000 point-wise mutations of the HA protein of a particular background strain are constructed, and viruses with these HAs are then passaged in MDCK-SIAT1 cells at low multiplicity of infection [<a href="#pcbi.1007892.ref022">22</a>]. Subsequently, the resulting samples are sequenced, and those viruses that were capable of cell entry and replication are over-enriched, whereas others that were not are depleted. Essentially, these data reveal mutational constraints of HA in a specific background strain, and can aid in discovering regions that are conserved in natural sequences due to viral function, instead of apparent conservation simply due to evolutionary history.</p> <p><i>What is the role of localized electrostatic charge on the evolutionary dynamics of H1N1 HA? Furthermore, can the characteristics of local charge explain the lack of increase in the HA net charge and thus give further evidence for immune escape due to increased HA avidity to cellular receptors?</i> In this paper, we make use of deep mutational scan data on H1N1 1933/WSN to identify three groups of sites on the HA protein of H1N1: this classification reflects functional constraints (<i>i.e.</i>, purifying selection) in conjunction with the expected charge at each residue. While functional constraints on the H1N1 HA may shift over time, we identify groups of residues based on measures of per-site functional selection and expected charge on an early H1N1 strain, and examine how the charge of these groups has changed over time. We show that in natural sequences from 1918 to 2008, these three groups present different trends of net charge, with the notable finding that the group that is generally closer to the RBS, more accessible to solvent, and evolves the fastest, is increasing in charge. These results possibly implicate a role for the net charge of specific regions on the HA of H1N1 IAVs in evolutionary dynamics, particularly in the context of viral binding to the host cell surface.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="functionalhabranchesandnetchargeevolution" title="sec"> <div class="title" tagx="title" title="title">
Functional HA branches and net charge evolution</div> <p><a href="#pcbi.1007892.g002">Fig 2</a> explores how functional selection and expected charge are related, and explores whether these measures can identify distinct groups of residues. By examining the relationship between functional selection and expected charge across the whole protein (<a href="#sec008">Materials and methods</a>), we identified three groups of sites (<a href="#pcbi.1007892.g002">Fig 2A and 2B</a>). The first set, hereafter named the â€œnegative branchâ€�, exhibits a negative linear relationship between expected charge and functional selection (yellow in <a href="#pcbi.1007892.g002">Fig 2A</a>). The second set, hereafter named the â€œpositive branchâ€�, mirrors the negative branch by having a positive linear relationship between expected charge and functional selection (light blue in <a href="#pcbi.1007892.g002">Fig 2A</a>). The last set, hereafter named the â€œzero branchâ€�, has functional selection independent of expected charge which is clustered around zero charge (dark blue in <a href="#pcbi.1007892.g002">Fig 2A</a>). Thus, we have now grouped residues on the HA protein based on their functional constraints and charge implications, and <a href="#pcbi.1007892.g002">Fig 2</a> illustrates that the distribution of these groups are elaborate. These groupings thus serve as the basis for our further investigations into other properties of the residues in each group (<i>e.g.</i><a href="#pcbi.1007892.g003">Fig 3</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1007892.g002"><span class="label" tagx="label" title="label">Fig 2</span></a></div>   <div class="title" tagx="title" title="title">
Identification of negative, zero, and positive functional branches.</div> <p>(<i>A</i>) Scatter-plot of site-wise functional selection and expected charge, based on deep mutational scanning (DMS). For a given residue using DMS preference data, â€˜functional selectionâ€™ is a measure of functional constraints at this site (a transformation of Shannonâ€™s evenness) and â€˜expected chargeâ€™ is the mean charge from DMS preferences. The colours are to facilitate the identification of the negative (yellow), zero (dark blue), and positive (light blue) branches. (<i>B</i>) The location of the residues belonging to each of the three branches identified in <i>A</i>, on the HA monomer, visualized with PyMOL [<a href="#pcbi.1007892.ref024">24</a>] and PDB 1RVX [<a href="#pcbi.1007892.ref023">23</a>]. <i>(C)-(E)</i> The net charge of the positive (<i>C</i>), negative (<i>D</i>), and zero (<i>E</i>) branches over time in 1741 H1N1 sequences from human hosts during 1918 to 2008.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1007892.g003"><span class="label" tagx="label" title="label">Fig 3</span></a></div>   <div class="title" tagx="title" title="title">
Evolutionary rates, relative solvent accessibility (RSA), and distances to receptor binding site (RBS) on functional branches.</div> <p>(<i>A</i>) Normalized conservation scores (labelled rate of evolution) per site overlaid on the functional selection-expected charge plot. These conservation scores were obtained from Rate4site [<a href="#pcbi.1007892.ref027">27</a>] with H1N1 sequences from the fludb database of the IRD [<a href="#pcbi.1007892.ref028">28</a>]. A higher score indicates lower conservation and faster evolution. (<i>B</i>) Empirical cumulative distributions of conservation scores for each of the three branches. (<i>C</i>) Distances to the RBS overlaid on the functional selection-expected charge plot. These distances were calculated using PyMOL [<a href="#pcbi.1007892.ref023">23</a>] (see <a href="#sec008">Materials and methods</a>) (<i>D</i>) Empirical cumulative distributions of the distance to the RBS for each of the three branches. (<i>E</i>) Relative solvent accessibility overlaid on the functional selection-expected charge plot. These scores were computed using dmstools2 [<a href="#pcbi.1007892.ref029">29</a>] (see <a href="#sec008">Materials and methods</a>) (<i>F</i>) Empirical cumulative distributions of relative solvent accessibility for each of the three branches.</p>   </div> <p>Having identified these groups, we then explored how the charges of these branches individually evolve over time. In particular, are there any distinct evolutionary patterns that could signal pressures exerted by host immune systems? Using available H1N1 sequences, we computed the net charge of these groups and examined their patterns over time (<a href="#sec008">Materials and methods</a>). From 1918 to 2008, the net charge of the negative branch has significantly increased over time (<i>P</i> &amp;lt; 2.2 Ã— 10<sup>âˆ’16</sup>). In contrast, the net charge of the positive branch has significantly decreased over time (<i>P</i> &amp;lt; 2.2 Ã— 10<sup>âˆ’16</sup>). The charge of zero branch does not appear to be changing substantially, though the very slight increase is significant (<i>P</i> = 0.0162) (<a href="#pcbi.1007892.g002">Fig 2C</a>). This positive trend in the zero branch is likely due to two factors that are probably heavily tilting the linear regressions. First, there is an abundance of sequences in certain years compared to others, and second, the net charge on the zero branch is very close to zero and thus heavily influenced by single charged residues. In an effort to remove this source of bias, we performed the same analyses with yearly means, <i>i.e.</i>, net charge averaged for each year, and we found that the change in charge of the zero branch was not significant with yearly means (<i>P</i> = 0.0976) (<a href="#pcbi.1007892.s002">S1C Fig</a>). Furthermore, we obtained analogous results for the positive and negative branches with yearly means (<a href="#pcbi.1007892.s002">S1A and S1B Fig</a>). Therefore, the evolution of net charge on the functional branches exhibits stark differences.</p> <p>The sequence of the HA1 â€œheadâ€� subunit of the HA varies more than the conserved HA2 â€œstemâ€�. In fact, the HA head is immunodominant and under stronger natural selection due to host immune pressures [<a href="#pcbi.1007892.ref025">25</a>, <a href="#pcbi.1007892.ref026">26</a>]. Examining per residue functional selection and expected charge for sites in each HA1 and HA2 reveal similar patterns, with HA1 exhibiting more pronounced division into functional branches (<a href="#pcbi.1007892.g004">Fig 4A and 4C</a>). Thus, we investigated the analogous patterns for these functional branches on the immunodominant HA head. Restricting our analysis to sites that belong to the HA1 subunit, the net charges of the negative and positive branches have significantly increased and decreased over time, respectively (<a href="#pcbi.1007892.s003">S2A and S2B Fig</a>), just as when considering all HA residues. Also, a linear regression reveals a significant increase in the net charge over time for the HA1 sites in the zero branch, which is clearly increasing from a net charge of about 0 to about 2 (<a href="#pcbi.1007892.s003">S2C Fig</a>). As before, we accounted for factors that are possibly affecting the regression by averaging sequences within each year. For these yearly means, the net charge of the HA1 zero branch does not vary over time (<a href="#pcbi.1007892.s004">S3C Fig</a>), whereas those for the HA1 negative and positive branches have significant trends matching our previous results (<a href="#pcbi.1007892.s004">S3A and S3B Fig</a>). As our sequences date from 1918 (compared to 1977 in Arinaminpathy and Grenfell [<a href="#pcbi.1007892.ref002">2</a>]), we verified that there is no positive trend in net charge over time (<a href="#pcbi.1007892.s004">S4A Fig</a>) and in fact found a slightly negative trend, but this trend is likely due to uneven sampling over time as it disappeared once yearly means were considered (<a href="#pcbi.1007892.s004">S4B Fig</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1007892.g004"><span class="label" tagx="label" title="label">Fig 4</span></a></div>   <div class="title" tagx="title" title="title">
Characteristics of conservation and distance to RBS in each HA subunit.</div> <p>(<i>A</i>) Conservation scores (labelled rate of evolution) per HA subunit, overlaid on the functional selection-expected charge plot. These scores were obtained from the Rate4Site algorithm [<a href="#pcbi.1007892.ref027">27</a>] using H1N1 sequences obtained from the IRD fludb database [<a href="#pcbi.1007892.ref028">28</a>] (see <a href="#sec008">Materials and methods</a>). As in <a href="#pcbi.1007892.g003">Fig 3A</a>, note that higher scores indicate higher rate of evolution. (<i>B</i>) Distances to the RBS per HA subunit, overlaid on the functional selection-expected charge plot. These distances were calculated using RBS annotations according to Gamblin and colleagues [<a href="#pcbi.1007892.ref024">24</a>] using PyMOL [<a href="#pcbi.1007892.ref023">23</a>] (see <a href="#sec008">Materials and methods</a>). (<i>C) and (D) are as in (A) and (B), respectively, but for the HA2 subunit.</i> (<i>E</i>) Empirical cumulative distribution of evolutionary rates of each branch separated by domain. (<i>F</i>) Empirical cumulative distribution of distance to RBS, separated by both branch and domain.</p>   </div> </div> <div class="evolutionaryratesalongfunctionalbranches" title="sec"> <div class="title" tagx="title" title="title">
Evolutionary rates along functional branches</div> <p>Rates of evolution, quantified as Rate4Site conservation scores, clearly show that the negative branch is evolving faster than the positive branch (<a href="#pcbi.1007892.g003">Fig 3A and 3B</a>, where a higher score means a higher rate of evolution). Note that these scores are normalized across the whole protein, giving a mean of zero and standard deviation of one. Pairwise Kolmogorov-Smirnov tests reveal that the distributions of conservation scores among the three branches are all statistically different (<i>D</i> = 0.21221 and <i>P</i> = 0.005356, <i>D</i> = 0.27625 and <i>P</i> = 9.274 Ã— 10<sup>âˆ’7</sup>, <i>D</i> = 0.36237 and <i>P</i> = 1.926 Ã— 10<sup>âˆ’10</sup> for comparisons between negative-positive, positive-zero, and negative-zero branches, respectively). Moreover, most of these rapidly evolving sites in the negative branch are in the HA1 subunit (<a href="#pcbi.1007892.g004">Fig 4A</a>) instead of the HA2 subunit (<a href="#pcbi.1007892.g004">Fig 4C</a>). Analyses with only HA1 sites reveal the same patterns (<a href="#pcbi.1007892.g004">Fig 4E</a>).</p> <p>These functional branches we identified may be related to other biochemical properties of HA residues. Meyer and Wilke [<a href="#pcbi.1007892.ref013">13</a>] used statistical models of H3N2 HA data to show that distances to the RBS and relative solvent accessibilities (RSAs) of residues explained a significant fraction of the variance seen in evolutionary rates. On a coarse scale, our H1N1 HA analyses largely agree with these findings. With respect to distances of each residueâ€™s centroid to the centroid of the RBS, the negative branch has a significantly different distribution than either the positive or the zero branch (<i>D</i> = 0.28228 and <i>P</i> = 0.0001638, <i>D</i> = 0.24011 and <i>P</i> = 0.000219, respectively), but the zero and positive branch are not significantly different (<i>D</i> = 0.10263 and <i>P</i> = 0.3347) (<a href="#pcbi.1007892.g003">Fig 3C and 3D</a>), and these results also hold when only HA1 sites are considered (<a href="#pcbi.1007892.g004">Fig 4B and 4F</a>). For RSAs, the distribution of the zero branch differs significantly from the other two (<i>D</i> = 0.5672 for negative branch and <i>D</i> = 0.52792 for positive branch, giving <i>P</i> &amp;lt; 2.2 Ã— 10<sup>âˆ’16</sup> in both cases), and the distributions of the positive and negative branches are also significantly different (<a href="#pcbi.1007892.g003">Fig 3E and 4F</a>, <i>D</i> = 0.18629 and <i>P</i> = 0.03846). These results are largely the same for analogous comparisons with sites on HA1 (<a href="#pcbi.1007892.g004">Fig 4</a>, <a href="#pcbi.1007892.s006">S5 Fig</a>). Furthermore, rates of evolution and distances to RBS generally have greater ranges for residues in HA1 than for those in HA2 (<a href="#pcbi.1007892.g004">Fig 4</a>).</p> <p>In summary, the sites that are on the negative branch evolve faster, are generally closer to the RBS, and are on the protein surface. The sites on the positive branch are generally further away from the RBS, but are also on the protein surface. Lastly, the sites on the zero branch are further away from the RBS, and are more likely to be internal residues. As a form of control to our study, we performed the same analyses with TEM-1 beta-lactamase (<i>i.e.</i>, not under similar immune selection), to see if the three branches due to functional selection and expected charge are conserved on such a different protein. Importantly, the underlying shape of the data mapped on the functional selection and expected charge axes is quite different from that of the H1N1 HA. Indeed, beta-lactamase appears to possess a very different pattern from that of H1N1 HA, with a clear absence of positive or negative branches. Through further visual inspection, we found no patterns for the distributions of RSA or evolutionary rates as a function of functional selection and expected charge (<a href="#pcbi.1007892.s007">S6 Fig</a>).</p> </div> <div class="localchargedistribution" title="sec"> <div class="title" tagx="title" title="title">
Local charge distribution</div> <p>The previous results on the different patterns of net charge for the identified branches have relied upon the aggregation, through summing, of the charges of all sites in each branch. While the net charge on these branches have distinct patterns over time, individual charges per site over time may themselves reveal notable features. To explore this, we fit multinomial logistic regression models at each site in the HA protein to obtain predicted probabilities of given charges over time, using collected natural sequences of H1N1 in humans (<a href="#sec008">Materials and methods</a>). The computed fitted observed charge over time at each site shows that the zero branch remains largely neutral (<a href="#pcbi.1007892.g005">Fig 5</a>). Moreover, as expected, predicted observed charges on the negative and positive branches tend to be negative and positive, respectively (<a href="#pcbi.1007892.g005">Fig 5</a>). However, over time, while certain sites in the negative branch do tend to become more negative as expected due to their functional constraints, others appear to transition away from a negative charge (see arrows in <a href="#pcbi.1007892.g005">Fig 5</a>). This results in the increase in net charge seen for the negative branch (<a href="#pcbi.1007892.g002">Fig 2C</a>). A similar, but opposite, process appears to occur on the positive branch, where certain residues transition away from a positive charge.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1007892.g005"><span class="label" tagx="label" title="label">Fig 5</span></a></div>   <div class="title" tagx="title" title="title">
Fitted observed charge in 1918, 1948, 1978, and 2008 computed from fitted per-site multinomial logistic regression models.</div> <p>These models were fitted using nnet [<a href="#pcbi.1007892.ref030">30</a>] in R [<a href="#pcbi.1007892.ref031">31</a>] (see <a href="#sec008">Materials and methods</a>). All 1744 human HA H1N1 sequences between 1918 and 2008 from fludb [<a href="#pcbi.1007892.ref028">28</a>] were used to fit these statistical models. The arrow is used to highlight sites in the negative branch that are transitioning to positive values.</p>   </div> <p>We then investigated how other immunologically relevant biochemical properties such as glycosylation and receptor binding avidity tie in to our results. As glycosylation has been implicated in IAV dynamics (<i>e.g.</i> Altman and colleagues [<a href="#pcbi.1007892.ref016">16</a>]), we computed the probabilities of glycosylation at each site from the deep mutational scan data assuming site independence (<a href="#sec008">Materials and methods</a>), and no striking patterns emerged with respect to their distributions amongst the 565 sites (<a href="#pcbi.1007892.s008">S7 Fig</a>). The assumption of site independence follows from DMS experiments which study functional fitness of single amino acid variants on a specific background strain. To relax this assumption, DMS data examining covariation of multiple residues would be required. Yet, glycosylation for the HA H1N1 is less pronounced than for H3N2 [<a href="#pcbi.1007892.ref016">16</a>], and perhaps plays a less substantial role in H1N1 HA evolution. Hensley and coauthors [<a href="#pcbi.1007892.ref001">1</a>] found certain sites that influence binding by changing the HA avidity to the receptor binding site. Examining the locations of these sites on the functional selection-expected charge plot, it is clear that they are found preferentially on the zero and negative branches (<a href="#pcbi.1007892.s009">S8 Fig</a>).</p> </div> <div class="testingclusterdistributionswithhierarchicalclustering" title="sec"> <div class="title" tagx="title" title="title">
Testing cluster distributions with hierarchical clustering</div> <p>So far, our results have relied upon the designations of residues belonging to the negative, positive, and zero branches through visual inspection generally supported by RSA values, and this method imposes a hard separation of negative and positive sites near the axis of symmetry. It is important to test if our results are robust to another, more formal, clustering scheme. To further investigate this, we used hierarchical clustering in R [<a href="#pcbi.1007892.ref031">31</a>]. Using unscaled selection coefficients and expected charge values at each residue, we used Wardâ€™s method [<a href="#pcbi.1007892.ref032">32</a>] in R [<a href="#pcbi.1007892.ref031">31</a>] but with Euclidean distance (between residues on the selectionâ€"charge plane) instead of its square as a distance measure. Among all possibilities, we used this clustering method since it accurately discriminates the three branches that are visually striking. Murtagh and Legendre [<a href="#pcbi.1007892.ref033">33</a>] show that this distance metric is distinct from that described by Ward [<a href="#pcbi.1007892.ref032">32</a>], but nevertheless it is better at detecting the three branches here, and so we used it to investigate the characteristics on the branches and to show robustness of our results with an alternative to visual grouping. The separation into the three branches is largely consistent with our initial classification. However, the positive branch contains most of the sites that are at the base of the three branches (<a href="#pcbi.1007892.s010">S9 Fig</a>). Detailed analyses are shown in <a href="#pcbi.1007892.s001">S1 Appendix</a>, and <a href="#pcbi.1007892.s010">S9</a>, <a href="#pcbi.1007892.s011">S10</a> and <a href="#pcbi.1007892.s012">S11</a> Figs. In essence, we found a similar clustering to our more subjective methods. There are subtle differences, but these do not affect the qualitative results.</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Transmission studies [<a href="#pcbi.1007892.ref001">1</a>] and natural sequence analyses [<a href="#pcbi.1007892.ref002">2</a>] for H1N1 HA give seemingly contrasting evidence on the role of electrostatic charge on IAV evolution. Indeed, Hensley and colleagues [<a href="#pcbi.1007892.ref001">1</a>] demonstrated that successful immune escape was driven by increased HA H1N1 avidity and mediated by increases in electrostatic charge. Yet, Arinaminpathy and Grenfell [<a href="#pcbi.1007892.ref002">2</a>] found that the H3N2 HA has increased in net charge since 1968, whereas the H1N1 HA has had no appreciable change. To reconcile the very different H1N1 and H3N2 charge trajectories, we investigated the hypothesis that local charge is more germane than global charge for evolutionary dynamics. Moreover, this hypothesis would imply that differences in global patterns of H1N1 and H3N2 HA simply reflect disparities in net charge constraints, instead of contrasting evolutionary pressures. Local charge in natural sequences reflects a combination of functional constraints and evolutionary pressures from host immunity. To isolate these confounding factors and remove bias due to functional constraints, partitioning of residues into functional patches requires per-residue selection experiments in the absence of immune selection on an early H1N1 strain. Constraints of functional relevance for the H1N1 HA are quantified in DMS data for an H1N1 HA from 1933 [<a href="#pcbi.1007892.ref022">22</a>]. By leveraging these data, we found patches on the HA with distinct evolutionary patterns in natural sequences. It should be noted that, due to evolution, similar functional experiments with more recent H1N1 HAs could reveal different groups of residues. However, our analyses require an early H1N1 HA to properly characterize the starting landscape and enable us to appropriately determine the net charge patterns of these patches afterwards.</p> <p>What are the characteristics of localized electrostatic charge in the H1N1 HA protein evolutionary patterns? Through measures of site-wise functional selection and expected charge obtained from DMS data, we identified three distinct branches of residues. On these â€œnegativeâ€�, â€œpositiveâ€�, and â€œzeroâ€� branches, functional selection and expected charge are positively correlated, negatively correlated, and independent of each other, respectively. The negative branch consists of residues that are the closest to the RBS, have high RSAs, and evolve the fastest. The positive branch has residues that are similarly surface exposed, but are further from the RBS and evolve slower. Lastly, the residues in the zero branch are those that are buried in the core of the protein, are also far from the RBS, and evolve the slowest. Intriguingly, the net charge of the negative branch has been increasing over time, whereas that of the positive and zero branches have been decreasing and constant, respectively. Importantly, multinomial logistic regression models for the dynamics of individual residue charge reveal that individual sites in the positive and negative branches are not each evolving toward an intermediate â€œzeroâ€� charge.</p> <p>Why are there contrasting net charge patterns for each branch over time, in conjunction with differing rates of evolution? These results could imply the following: as the positive branch evolves more slowly than its negative counterpart, it is possible that the decrease in net charge over time seen in the positive branch is in response to an increase in net charge over time of the negative branch. Our results also indicate that the sites in the zero branch are highly conserved on average, that their net charge does not vary significantly over time, and that their site-wise fitted observed charge is generally zero.</p> <p>In the context that the overall net charge of the HA for H1N1 viruses in humans has not increased over time [<a href="#pcbi.1007892.ref002">2</a>], these results indicate that more localized charge may also be important in viral evolution. In fact, by considering those sites in the negative branch and thus closer to the RBS, the increase in net charge over time is likely due in part to the negative charge of the cell surface. An increase in net charge for this group that is nearest to the RBS could facilitate HA interactions with the negatively charged host cell surface. Furthermore, since the negative branch contains more sites that are evolving rapidly, this branch is probably immunodominant in comparison with the other two branches. Therefore, there is an apparent asymmetry between the positive and negative branches that is not revealed by functional selection experiments. This asymmetry could be possibly explained either by an intrinsic molecular property of the HA, or could result from host interactions. Indeed, it is thus possible that charge imposes constraints on these evolving viruses, or immune selection could impose such an asymmetry.</p> <p>Perhaps the discrepancy between H1N1 and H3N2 viruses in the net charge patterns of HAs is simply a reflection of different constraints on HA biophysical factors such as stability, instead of a fundamental difference in the role of electrostatic charge in virion-host cell interactions between these two subtypes. In particular, it may be that the H1N1 HA is less tolerant to charge imbalances than the H3N2 HA, and therefore if a set of sites in the H1N1 HA is under pressure to become more positively charged, other sites must become more negatively charged in counterbalance. Deep mutational scans on H1N1 [<a href="#pcbi.1007892.ref022">22</a>] and H3N2 [<a href="#pcbi.1007892.ref034">34</a>] indicate that the H3N2 HA has a different mutational tolerance profile in comparison to the H1N1 HA. Further experiments should examine and quantify differences in tolerance to charge imbalance for H1N1 and H3N2 HAs to investigate the hypothesis of different tolerances to charge imbalance in these HAs. Furthermore, in natural sequences, H1N1 antigenic evolution is not as rapid as H3N2 antigenic evolution [<a href="#pcbi.1007892.ref035">35</a>], and could indicate some differing constraints.</p> <p>Another possible explanation for the observed differences in H1N1 and H3N2 HAs is that H3N2 and H1N1 virions have different NAs. Specifically, it is possible that having an increasingly positive HA could make it more difficult for NA to cleave new viruses from the cell, and it could be that the H3N2 NA protein is more tolerant of this than the H1N1 NA. To investigate this possibility and quantify the co-evolution of HA and NA charge, a combination of experiments and analyses of natural sequences are required.</p> <p>However, it is possible that the patterns we see are due to other processes, or are simply covariates of other variables that we did not examine. Our inferences from the patterns of charge over time is limited, due to the fact that our study is retrospective and relies upon collected natural sequences of H1N1. The remarkable separation of residues into three branches, based on functional selection and expected charge, should be examined further. With H3N2 data, the separation into three branches is not as clear, and could be due to different relative mutational tolerances in the HA [<a href="#pcbi.1007892.ref034">34</a>]. However, explorations of HA localized charge for this subtype are not as crucial, as the net charge of the whole H3N2 HA was found to increase over time [<a href="#pcbi.1007892.ref002">2</a>]. Echoing Arinaminpathy and Grenfell [<a href="#pcbi.1007892.ref002">2</a>], further experiments will be needed to elucidate the exact mechanisms involved in these processes, and to determine the role of electrostatic charge. Beyond H1N1 and H3N2 viruses, numerous subtypes of influenza circulate or have circulated among a variety of hosts. Analyses that examine the role of local charge could reveal unifying themes, with potential relevance for zoonosis and pandemic preparedness. Such analyses would require subtype-specific DMS data, in conjunction with temporally detailed natural sequences for specific hosts, and these analyses may become feasible with growing data availability.</p> <p>Experiments to probe the mechanisms that give rise to patterns of charge could then guide informed decisions about vaccine design. The importance of these decisions, based on electrostatic interactions at the molecular level, would be especially relevant in the quest for broadly protective vaccines [<a href="#pcbi.1007892.ref015">15</a>]. Preliminarily, our results might indicate that possible mechanisms for disease control could be to either limit the possibility of the negative branch increasing in charge, or, alternatively, limit the possibility of the positive branch decreasing in charge. Either of these options would require rational vaccine design, hopefully eliciting broadly cross-reactive antibodies.</p> <p>Furthermore, understanding how functional selection and expected charge co-vary for different pathogensâ€™ proteins under immune selection could help characterize, quantify, and generalize the role of charge in evolutionary dynamics of pathogens. Quantifying these properties for other proteins is also important, and we showed that beta-lactamase protein gives rise to different patterns than the H1N1 HA. Given this stark contrast, it is possible that the pattern seen for the H1N1 HA is the result of evolution due to immune selection. If this is the case, signatures of evolutionary pressures on other proteins could be detected by functional selection and charge. Thus, independent of the specifics of a particular protein, a â€œnullâ€� distribution of sites according to functional selection and expected charge would be quite valuable in principle. However, such a distribution would be very difficult to obtain. This would require understanding which random sequences of amino acids can properly fold, and with these, perform DMS experiments to quantify amino acid preferences at each site.</p> <p>It is striking that simple measures of functional selection and expected charge are able to discriminate sites that have higher evolutionary rates, particularly when these measures are independent of antibody interactions and are based solely upon cell entry and viral replication. That these sites then show an increase in net charge over time is even more intriguing. Overall, our results possibly implicate localized charge on the HA protein in evolutionary dynamics, particularly in HA-cell host interactions. We could speculate that certain regions on the HA protein could be under pressure to gain charge in order to facilitate attachment to the cell surface, whereas others would be under pressure to decrease in charge in response to such an increase.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="deepmutationalscan(dms)data" title="sec"> <div class="title" tagx="title" title="title">
Deep mutational scan (DMS) data</div> <p>The DMS generates amino acid preferences for each site, which are normalized enrichment. A preference of 0 denotes complete depletion of that particular amino acid and a preference of 1 denotes a complete depletion of all other amino acids in favor of that particular amino acid. For H1N1 HA data using dmstools2 [<a href="#pcbi.1007892.ref029">29</a>], these preferences can be calculated (see example of Doud and Bloom [<a href="#pcbi.1007892.ref022">22</a>] in dmstools2 documentation) for the HA using a Bayesian approach, in order to account for the possibility of errors in sequencing. We obtained the HA amino acid preferences from the Bloom lab GitHub repository.</p> <p>To quantify purifying selection due to function (â€œfunctional selectionâ€�), we use a transformation of Shannonâ€™s evenness, a statistic often used in ecological systems to quantify diversity [<a href="#pcbi.1007892.ref036">36</a>]. Denoting <i>p</i><sub><i>i</i>,<i>a</i></sub> as the preference for amino acid <i>a</i> âˆˆ <i>A</i> at site <i>i</i>, we defined the functional selection coefficient as </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1007892.e001g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
S</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
-</div> <div class="msqrt_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
-</div> <div class="mfrac_UNKNOWN"> <div class="mn_UNKNOWN">
1</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
ln</div> <div class="mn_UNKNOWN">
20</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
âˆˆ</div> <div class="mi_UNKNOWN">
A</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
a</div> </div> </div> <div class="mo_UNKNOWN">
ln</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
a</div> </div> </div> </div> </div> <div class="mo_UNKNOWN">
.</div> </div> </div> </div> </div></div></div><span class="label" tagx="label" title="label">(1)</span></div><i>S</i><sub><i>i</i></sub> is simply a transformation of Shannonâ€™s evenness, so that <i>S</i><sub><i>i</i></sub> â‰ˆ 0 if there are few functional constraints, and <i>S</i><sub><i>i</i></sub> â‰ˆ 1 if there are strong functional constraints.<p /> <p>In addition, by using the same DMS data, we also computed the expected charge at each site, essentially giving the expected charge <i>C</i><sup>(<i>i</i>)</sup> at each site <i>i</i> due to functional constraints on the HA protein. That is, </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1007892.e002g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msup_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mo_UNKNOWN">
=</div> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
a</div> <div class="mo_UNKNOWN">
âˆˆ</div> <div class="mi_UNKNOWN">
A</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
p</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
a</div> </div> </div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
c</div> <div class="mi_UNKNOWN">
a</div> </div> <div class="mo_UNKNOWN">
,</div> </div> </div> </div> </div></div></div><span class="label" tagx="label" title="label">(2)</span></div> where <i>c</i><sub><i>a</i></sub> âˆˆ {âˆ’1, 0, 1} is the charge of the amino acid <i>a</i> âˆˆ <i>A</i>. Here, we assumed that Arginine, Histidine, Lysine, Aspartic Acid, and Glutamic Acid are the only charged amino acids with a charge magnitude of 1, and with the first three positive and the last two negative, just as was done by Arinaminpathy and Grenfell [<a href="#pcbi.1007892.ref002">2</a>]. Note that for beta-lactamase analyses, we used the amino acid preferences as given in Bloom [<a href="#pcbi.1007892.ref037">37</a>].<p /> <p>The three branches shown in <a href="#pcbi.1007892.g002">Fig 2A</a> were separated by visual inspection, with the negative branch consisting of sites <i>i</i> where <i>C</i><sup>(<i>i</i>)</sup> &amp;lt; 0.04 and <i>S</i><sub><i>i</i></sub> &amp;lt; 0.05 âˆ’ 0.7<i>C</i><sup>(<i>i</i>)</sup>, the positive branch consisting of sites <i>i</i> where <i>C</i><sup>(<i>i</i>)</sup> &amp;gt; 0.041 and <i>S</i><sub><i>i</i></sub> &amp;lt; 0.02 + 0.5<i>C</i><sup>(<i>i</i>)</sup>, and the zero branch containing sites that are in neither other branch.</p> </div> <div class="relativesolventaccessibilities(rsas)anddistancestoreceptorbindingsite(rbs)" title="sec"> <div class="title" tagx="title" title="title">
Relative solvent accessibilities (RSAs) and distances to receptor binding site (RBS)</div> <p>For HA residues, we computed relative solvent accessibilities (RSAs) using dmstools2 [<a href="#pcbi.1007892.ref029">29</a>], and distances from the centroid of each residue to the centroid of the receptor binding site (RBS) were obtained from PyMOL 2.2.0 [<a href="#pcbi.1007892.ref023">23</a>] using the HA monomer. The pdb file for the HA monomer was generated with pdb-tools [<a href="#pcbi.1007892.ref038">38</a>] by selecting chain A of the trimer. Note that RSAs and distances to RBS could not be computed for certain sites in the reference DMS sequence, as these values rely upon the closest available 3-dimensional H1N1 HA protein (used in Doud and Bloom [<a href="#pcbi.1007892.ref022">22</a>]). Values for RSAs and distances to the RBS are also missing since we consider HA1 to contain the signal peptide, and some HA2 sites are not identified on the 3-dimensional H1N1 HA protein. To determine which sites on the HA are in the receptor binding site (RBS), we followed [<a href="#pcbi.1007892.ref024">24</a>] (in H3 numbering) and comprised the RBS of the 130-loop (residues 135 to 138), the 190-helix (residues 190 to 198), the 220-loop (residues 221 to 228), and the RBS base (residues 98, 153, 183, and 195). To convert from H3 numbering to sequential numbering, we followed the conversion scheme given in Doud and Bloom [<a href="#pcbi.1007892.ref022">22</a>]. To compute RSAs for beta-lactamase, we followed Bloom [<a href="#pcbi.1007892.ref037">37</a>] and also used dmstools2 [<a href="#pcbi.1007892.ref029">29</a>].</p> </div> <div class="glycosylation" title="sec"> <div class="title" tagx="title" title="title">
Glycosylation</div> <p>Under no immune selection, the probability of glycosylation at site <i>i</i> was calculated by finding the probability of having the Asnâˆ’<i>X</i>âˆ’Ser/Thr sequon [<a href="#pcbi.1007892.ref039">39</a>] (where <i>X</i> is not proline) assuming site independence. This assumption is necessary and implicit as DMS data only consider single residue changes on a specific background HA protein. In particular, these data do not consider epistatic interactions between residues, including any such interactions that would enrich Asnâˆ’<i>X</i>âˆ’Ser/Thr sequons. Therefore, the estimate of the glycosylation probability at site <i>i</i> is </p><div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1007892.e003g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="msub_UNKNOWN"> <div class="mtext_UNKNOWN">
Glycosylation</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
}</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="msub_UNKNOWN"> <div class="mtext_UNKNOWN">
Asn</div> <div class="mi_UNKNOWN">
i</div> </div> <div class="mo_UNKNOWN">
}</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
-</div> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="msub_UNKNOWN"> <div class="mtext_UNKNOWN">
Pro</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
1</div> </div> </div> <div class="mo_UNKNOWN">
}</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="msub_UNKNOWN"> <div class="mtext_UNKNOWN">
Ser</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
}</div> </div> <div class="mo_UNKNOWN">
+</div> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
{</div> <div class="msub_UNKNOWN"> <div class="mtext_UNKNOWN">
Thr</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
+</div> <div class="mn_UNKNOWN">
2</div> </div> </div> <div class="mo_UNKNOWN">
}</div> </div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
.</div> </div> </div> </div> </div></div></div> </div> <p /> </div> <div class="analyseswithnaturalsequences" title="sec"> <div class="title" tagx="title" title="title">
Analyses with natural sequences</div> <p>For our sequence analyses with H1N1 between 1918 to 2008, we obtained all available 1744 H1 sequences in human hosts from <a href="http://www.fludb.org">http://www.fludb.org</a>, of the Influenza Research Database (IRD) [<a href="#pcbi.1007892.ref028">28</a>], and aligned these using MUSCLE in fludb. For net charge computations, we again followed Arinaminpathy and Grenfell [<a href="#pcbi.1007892.ref002">2</a>]. That is, we summed the individual charges of amino acids across the alignments, where Arginine, Histidine, and Lysine have a charge of 1, Aspartic acid and Glutamic acid have a charge of âˆ’1, and all other amino acids have a charge of 0. Given ambiguous amino acids (<i>i.e.</i>, X, B, Z) in a particular sequence, we assumed that the indicated respective choices of residues at that site were uniformly distributed and computed the expected charge at that site for that sequence. These temporal trends, along with other graphs, were visualized with the ggplot2 package [<a href="#pcbi.1007892.ref040">40</a>] of R [<a href="#pcbi.1007892.ref031">31</a>].</p> </div> <div class="evolutionaryrates" title="sec"> <div class="title" tagx="title" title="title">
Evolutionary rates</div> <p>To compute evolutionary rates of amino acid substitutions, we selected the 958 unique sequences between 1918 and 2008, aligned these within fludb, and used the Rate4Site algorithm [<a href="#pcbi.1007892.ref027">27</a>]. To statistically test differences in distributions of evolutionary rates and other HA characteristics across groups of residues, we used the non-parametric Kolmogorov-Smirnov test with ks.test from R [<a href="#pcbi.1007892.ref031">31</a>] (note that the p-values from these tests are sometimes not exact, due to ties). For beta-lactamase, we used the sequences given in Bloom [<a href="#pcbi.1007892.ref037">37</a>] and computed rates using the Rate4Site algorithm [<a href="#pcbi.1007892.ref027">27</a>].</p> </div> <div class="fittedobservedper-residuecharge" title="sec"> <div class="title" tagx="title" title="title">
Fitted observed per-residue charge</div> <p>Across 1918 to 2008, the charge in a given year, <i>i.e.</i>, â€œyearly chargeâ€�, may vary for individual sites. In order to infer per-residue yearly charge from natural sequence data in the multiple sequence alignment, we fit multinomial statistical models. To compute the site-wise fitted observed charge from natural sequences, we used maximum likelihood estimation to obtain fitted observed charge over time at each site. If more than one charge value was present in the set of sequences at a particular site, we used the nnet package [<a href="#pcbi.1007892.ref030">30</a>] in R [<a href="#pcbi.1007892.ref031">31</a>] to fit a multinomial logistic regression model at each of the 565 sites, in order to obtain the probability <i>f</i><sub><i>i</i>,<i>c</i></sub>(<i>t</i>) of having the charge <i>c</i> âˆˆ {âˆ’1, 0, 1} at site <i>i</i> at time <i>t</i>. Otherwise, we set <i>f</i><sub><i>i</i>,<i>c</i></sub>(<i>t</i>) = 1 and <i>f</i><sub><i>i</i>,<i>c</i></sub>(<i>t</i>) = 0, respectively, for the values of <i>c</i> that occurred in all sequences and for the values of <i>c</i> that occurred in none, respectively. Note that if there was ambiguity in a given sequence about its residue at a certain position, that residue was omitted when fitting multinomial models, in contrast to net charge computations. Then, we calculated the fitted observed charge <span class="inline-formula" title="inline-formula"> </span></p><div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1007892.e004g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> </div></div></div> at site <i>i</i> at time <i>t</i> as <div class="disp-formula" title="disp-formula"> <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi.1007892.e005g" mimetype="image" position="anchor" orientation="portrait" /><div class="math_UNKNOWN"> <div class="mtable_UNKNOWN"> <div class="mtr_UNKNOWN"> <div class="mtd_UNKNOWN"> <div class="mrow_UNKNOWN"> <div class="msubsup_UNKNOWN"> <div class="mi_UNKNOWN">
C</div> <div class="mi_UNKNOWN">
P</div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
)</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
=</div> <div class="munder_UNKNOWN"> <div class="mo_UNKNOWN">
âˆ‘</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
c</div> <div class="mo_UNKNOWN">
âˆˆ</div> <div class="mo_UNKNOWN">
{</div> <div class="mo_UNKNOWN">
-</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
0</div> <div class="mo_UNKNOWN">
,</div> <div class="mn_UNKNOWN">
1</div> <div class="mo_UNKNOWN">
}</div> </div> </div> <div class="mi_UNKNOWN">
c</div> <div class="msub_UNKNOWN"> <div class="mi_UNKNOWN">
f</div> <div class="mrow_UNKNOWN"> <div class="mi_UNKNOWN">
i</div> <div class="mo_UNKNOWN">
,</div> <div class="mi_UNKNOWN">
c</div> </div> </div> <div class="mrow_UNKNOWN"> <div class="mo_UNKNOWN">
(</div> <div class="mi_UNKNOWN">
t</div> <div class="mo_UNKNOWN">
)</div> </div> <div class="mo_UNKNOWN">
.</div> </div> </div> </div> </div></div></div><span class="label" tagx="label" title="label">(3)</span></div> <p /> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Appendix</span> <div class="title" tagx="title" title="title">
Supplementary analyses and results.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1007892.s001.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Fig</span> <div class="title" tagx="title" title="title">
Yearly means of net charge over time in negative (<i>A</i>), positive (<i>B</i>), and zero (<i>C</i>) branches.</div> <p>This figure is just as in <a href="#pcbi.1007892.g002">Fig 2A</a> of the main text, but employing yearly means instead of all sequences. Here, yearly means are averages of the charge of all sequences in a given year, giving a single charge value for each year that at least one sequence is available. This approach minimizes any potential biases introduced due to uneven sampling in different years.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1007892.s002.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Fig</span> <div class="title" tagx="title" title="title">
Net charge over time in (<i>A</i>) negative (<i>P</i> &amp;lt; 2.2 Ã— 10<sup>âˆ’16</sup>), (<i>B</i>) positive (<i>P</i> &amp;lt; 2.2 Ã— 10<sup>âˆ’16</sup>), and (<i>C</i>) zero (<i>P</i> &amp;lt; 2.2 Ã— 10<sup>âˆ’16</sup>) branches of HA1 subunit.</div> <p>This figure is just as in 1A, except only the immunodominant HA1 subunit is being considered.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1007892.s003.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S3 Fig</span> <div class="title" tagx="title" title="title">
Yearly means of net charge over time in negative (<i>A</i>), positive (<i>B</i>), and zero (<i>C</i>) branches of HA1 subunit.</div> <p>This figure compliments <a href="#pcbi.1007892.s003">S2 Fig</a> and is analogous to <a href="#pcbi.1007892.s002">S1 Fig</a> except that only residues in the HA1 subunit are being considered in contrast to the whole HA in <a href="#pcbi.1007892.s002">S1 Fig</a>.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1007892.s004.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S4 Fig</span> <div class="title" tagx="title" title="title">
Net charge of H1N1 HA over time.</div> <p>(<i>A</i>) Net charge from 1918 to 2008 in 1741 H1N1 sequences from human hosts, obtained from the IRD [<a href="#pcbi.1007892.ref028">28</a>] at <a href="http://www.fludb.org">http://www.fludb.org</a>. This figure confirms that the net charge of the HA H1N1 has not varied significantly since its introduction in 1918. (<i>B</i>) as in (A), but with yearly means instead.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1007892.s005.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S5 Fig</span> <div class="title" tagx="title" title="title">
Relative Solvent Accessibilities (RSAs) across branches and HA domains.</div> <p>RSAs for (<i>A</i>) HA1 and (<i>B</i>) HA2, with (<i>C</i>) empirical CDFs for each branch in each domain.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1007892.s006.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S6 Fig</span> <div class="title" tagx="title" title="title">
Beta-lactamase functional selection as a function of expected charge, overlaid with both RSA and evolutionary rates.</div> <p>(<i>A</i>) RSA values for beta-lactamase residues overlaid on the functional selection-expected value plot for these residues. (<i>B</i>) Normalized conservation scores, calculated with the Rate4Site algorithm [<a href="#pcbi.1007892.ref027">27</a>], overlaid on the plot of selection as a function of expected charge, for beta-lactamase.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1007892.s007.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S7 Fig</span> <div class="title" tagx="title" title="title">
Probabilities of glycosylation per site, with non-zero probabilities plotted on log-scale to increase contrast in values.</div> <p>These values are plotted on the functional selection-expected value plot for HA residues on (<i>A</i>) a linear scale, and on (<i>B</i>) a log-scale. The probabilities of glycosylation were computed using DMS data and assuming independence of sites (see <a href="#sec008">Materials and methods</a>).</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1007892.s008.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S8 Fig</span> <div class="title" tagx="title" title="title">
Sites shown to influence binding, according to Hensley and coauthors [<a href="#pcbi.1007892.ref001">1</a>], overlaid on the functional selection-expected value RSA plot for HA residues in HA1.</div> <p>The sites that change binding are denoted with large circles, whereas those other residues that were not so identified are depicted as points. Single mutations are these sites led to mutants denoted as â€œbetter bindersâ€� or â€œworse bindersâ€� by Hensley [<a href="#pcbi.1007892.ref001">1</a>], Table S2. In H3 numbering, the sites affecting binding are 128, 129, 164, 165, 166, 158, 156, 192, 193, 196, 198, 143, 224, 244, 74, 75, 119, 162, 93, 145 [<a href="#pcbi.1007892.ref001">1</a>].</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1007892.s009.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S9 Fig</span> <div class="title" tagx="title" title="title">
Alternative clustering scheme, using hierarchical clustering and Wardâ€™s method with Euclidean distance (instead of distance squared) on unscaled data.</div> <p>The three colours denote the three clusters identified through this alternative scheme.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1007892.s010.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S10 Fig</span> <div class="title" tagx="title" title="title">
Temporal charge trends for the clusters determined by the alternative clustering method, with clusters shown in <a href="#pcbi.1007892.s010">S9 Fig</a>.</div> <p>The <i>red</i>, <i>blue</i>, and <i>green</i> colours depict the positive, negative, and zero branches identified through hierarchical clustering with Wardâ€™s method in <a href="#pcbi.1007892.s009">S8 Fig</a>. (<i>A</i>)â€"(<i>B</i>) Temporal trends by clusters with all sequences for (<i>A</i>) the whole HA and (<i>B</i>) HA1. (<i>C</i>)â€"(<i>D</i>) Yearly means for each cluster, for (<i>A</i>) the whole HA and (<i>B</i>) HA1.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1007892.s011.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S11 Fig</span> <div class="title" tagx="title" title="title">
Empirical cumulative distributions for each branch and each HA domain with the alternative clustering scheme.</div> <p>(<i>A</i>)â€"(<i>C</i>) are for distributions of evolutionary rates for (<i>A</i>) the whole HA, (<i>B</i>) HA1, and (<i>C</i>) HA2. (<i>D</i>)â€"(<i>F</i>) are for distributions of distances to the RBS for (<i>D</i>) the whole HA, (<i>E</i>) HA1, and (<i>F</i>) HA2. (<i>G</i>)â€"(<i>I</i>) are for distributions of RSAs for (<i>G</i>) the whole HA, (<i>H</i>) HA1, and (<i>I</i>) HA2. For the whole HA and RBS distributions, the positive-negative and negative-zero branches are significantly different (<i>D</i> = 0.2207, <i>P</i> = 0.003768 and <i>D</i> = 0.26109, <i>P</i> = 9.992 Ã— 10<sup>âˆ’5</sup>, respectively), whereas the positive and zero branches are not significantly different (<i>D</i> = 0.10194, <i>P</i> = 0.2699). For RSA distributions, the positive-negative, positive-zero, and negative-zero branches are all significantly different (<i>D</i> = 0.17977, <i>P</i> = 0.03591; <i>D</i> = 0.57619, <i>P</i> &amp;lt; 2.2 Ã— 10<sup>âˆ’16</sup>; and <i>D</i> = 0.62497, <i>P</i> &amp;lt; 2.2 Ã— 10<sup>âˆ’16</sup>, respectively). For evolutionary rates, all the differences in branches are also significant (<i>D</i> = 0.17297, <i>P</i> = 0.02768 [positive-negative]; <i>D</i> = 0.32241, <i>P</i> = 2.215 Ã— 10<sup>âˆ’10</sup> [positive-zero]; and <i>D</i> = 0.38207, <i>P</i> = 1.837 Ã— 10<sup>âˆ’10</sup> [negative-zero]). The statistical significance of differences is similar for comparisons of branches with only HA1 sites, at a significance level of <i>Î±</i> = 0.05.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pcbi.1007892.s012.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Analyses</span> <div class="title" tagx="title" title="title">
This file contains files and codes for our analyses.</div> <p>(ZIP)</p>  <div class="media" title="media"><a href="pcbi.1007892.s013.zip">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank Jesse Bloom, Daniel Cooney, Adam Dingens, Scott Hensley, Juhye Lee, Sean Liu, Adrian McDermott, Dylan Morris, Peter Palese, and Cecile Viboud for valuable comments and discussions. We also thank the Bloom lab for the DSSP file and the renumbered PDB file.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pcbi.1007892.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hensley</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Das</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bailey</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmidt</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hickman</span><span class="given-names" tagx="given-names" title="given-names">HD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jayaraman</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Hemagglutinin Receptor Binding Avidity Drives Influenza A Virus Antigenic Drift</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">326</span>(<span class="issue" tagx="issue" title="issue">5953</span>):<span class="fpage" tagx="fpage" title="fpage">734</span>â€"<span class="lpage" tagx="lpage" title="lpage">736</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1178258">10.1126/science.1178258</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19900932">19900932</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arinaminpathy</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>. <span class="mixed-article-title" title="mixed-article-title">Dynamics of Glycoprotein Charge in the Evolutionary History of Human Influenza</span>. <span class="source" tagx="source" title="source">PLOS One</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">5</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">e15674</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0015674">10.1371/journal.pone.0015674</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21209885">21209885</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rolfes</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foppa</span><span class="given-names" tagx="given-names" title="given-names">IM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garg</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Flannery</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brammer</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Singleton</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Annual estimates of the burden of seasonal influenza in the United States: A tool for strengthening influenza surveillance and preparedness</span>. <span class="source" tagx="source" title="source">Influenza and Other Respiratory Viruses</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">132</span>â€"<span class="lpage" tagx="lpage" title="lpage">137</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1111/irv.12486">10.1111/irv.12486</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29446233">29446233</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bean</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gorman</span><span class="given-names" tagx="given-names" title="given-names">OT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chambers</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawaoka</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>. <span class="mixed-article-title" title="mixed-article-title">Evolution and ecology of influenza A viruses</span>. <span class="source" tagx="source" title="source">Microbiological Reviews</span>. <span class="year" tagx="year" title="year">1992</span>;<span class="volume" tagx="volume" title="volume">56</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">152</span>â€"<span class="lpage" tagx="lpage" title="lpage">179</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/MMBR.56.1.152-179.1992">10.1128/MMBR.56.1.152-179.1992</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1579108">1579108</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gamblin</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skehel</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">Influenza Hemagglutinin and Neuraminidase Membrane Glycoproteins</span>. <span class="source" tagx="source" title="source">Journal of Biological Chemistry</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">285</span>(<span class="issue" tagx="issue" title="issue">37</span>):<span class="fpage" tagx="fpage" title="fpage">28403</span>â€"<span class="lpage" tagx="lpage" title="lpage">28409</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.R110.129809">10.1074/jbc.R110.129809</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20538598">20538598</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Earn</span><span class="given-names" tagx="given-names" title="given-names">DJD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dushoff</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levin</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span>. <span class="mixed-article-title" title="mixed-article-title">Ecology and evolution of the flu</span>. <span class="source" tagx="source" title="source">Trends in Ecology &amp;amp; Evolution</span>. <span class="year" tagx="year" title="year">2002</span>;<span class="volume" tagx="volume" title="volume">17</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">334</span>â€"<span class="lpage" tagx="lpage" title="lpage">340</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0169-5347(02)02502-8">10.1016/S0169-5347(02)02502-8</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bouvier</span><span class="given-names" tagx="given-names" title="given-names">NM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palese</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>. <span class="mixed-article-title" title="mixed-article-title">The biology of influenza viruses</span>. <span class="source" tagx="source" title="source">Vaccine</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">26</span>:<span class="fpage" tagx="fpage" title="fpage">D49</span>â€"<span class="lpage" tagx="lpage" title="lpage">D53</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.vaccine.2008.07.039">10.1016/j.vaccine.2008.07.039</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19230160">19230160</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caton</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brownlee</span><span class="given-names" tagx="given-names" title="given-names">GG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yewdell</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerhard</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>. <span class="mixed-article-title" title="mixed-article-title">The antigenic structure of the influenza virus A/PR/8/34 hemagglutinin (H1 subtype)</span>. <span class="source" tagx="source" title="source">Cell</span>. <span class="year" tagx="year" title="year">1982</span>;<span class="volume" tagx="volume" title="volume">31</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">417</span>â€"<span class="lpage" tagx="lpage" title="lpage">427</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/0092-8674(82)90135-0">10.1016/0092-8674(82)90135-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6186384">6186384</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cobey</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hensley</span><span class="given-names" tagx="given-names" title="given-names">SE</span></span>. <span class="mixed-article-title" title="mixed-article-title">Immune history and influenza virus susceptibility</span>. <span class="source" tagx="source" title="source">Current Opinion in Virology</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">22</span>:<span class="fpage" tagx="fpage" title="fpage">105</span>â€"<span class="lpage" tagx="lpage" title="lpage">111</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.coviro.2016.12.004">10.1016/j.coviro.2016.12.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28088686">28088686</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Angeletti</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbs</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Angel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kosik</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hickman</span><span class="given-names" tagx="given-names" title="given-names">HD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">M</span><span class="given-names" tagx="given-names" title="given-names">FG</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Defining B cell immunodominance to viruses</span>. <span class="source" tagx="source" title="source">Nature Immunology</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">18</span>:<span class="fpage" tagx="fpage" title="fpage">456</span>â€"<span class="lpage" tagx="lpage" title="lpage">465</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/ni.3680">10.1038/ni.3680</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28192417">28192417</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">STH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Behzadi</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Freyn</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">WC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Broecker</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Antigenic sites in influenza H1 hemagglutinin display species-specific immunodominance</span>. <span class="source" tagx="source" title="source">Journal of Clinical Investigation</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">128</span>:<span class="fpage" tagx="fpage" title="fpage">4992</span>â€"<span class="lpage" tagx="lpage" title="lpage">4996</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1172/JCI122895">10.1172/JCI122895</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30188868">30188868</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koel</span><span class="given-names" tagx="given-names" title="given-names">BF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">M</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Vliet</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zondag</span><span class="given-names" tagx="given-names" title="given-names">GCM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vervaet</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Substitutions near the receptor binding site determine major antigenic change during influenza virus evolution</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">342</span>:<span class="fpage" tagx="fpage" title="fpage">976</span>â€"<span class="lpage" tagx="lpage" title="lpage">979</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1244730">10.1126/science.1244730</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24264991">24264991</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">AG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilke</span><span class="given-names" tagx="given-names" title="given-names">CO</span></span>. <span class="mixed-article-title" title="mixed-article-title">Geometric constrains dominate the antigenic evolution of influenza H3N2 hemagglutinin</span>. <span class="source" tagx="source" title="source">PLoS Pathogens</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">11</span>:<span class="fpage" tagx="fpage" title="fpage">e1004940</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1004940">10.1371/journal.ppat.1004940</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26020774">26020774</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gog</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">JLN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daly</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mumford</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Unifying the Epidemiological and Evolutionary Dynamics of Pathogens</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">303</span>(<span class="issue" tagx="issue" title="issue">5656</span>):<span class="fpage" tagx="fpage" title="fpage">327</span>â€"<span class="lpage" tagx="lpage" title="lpage">332</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1090727">10.1126/science.1090727</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14726583">14726583</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saad-Roy</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McDermott</span><span class="given-names" tagx="given-names" title="given-names">AB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span>. <span class="mixed-article-title" title="mixed-article-title">Dynamic Perspectives on the Search for a Universal Influenza Vaccine</span>. <span class="source" tagx="source" title="source">The Journal of Infectious Diseases</span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">219</span>:<span class="fpage" tagx="fpage" title="fpage">S46</span>â€"<span class="lpage" tagx="lpage" title="lpage">S56</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jiz044">10.1093/infdis/jiz044</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30715467">30715467</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altman</span><span class="given-names" tagx="given-names" title="given-names">MO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Angel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">KoÅ¡Ã­k</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">TrovÃ£o</span><span class="given-names" tagx="given-names" title="given-names">NS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zost</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbs</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Human Influenza A Virus Hemagglutinin Glycan Evolution Follows a Temporal Pattern to a Glycan Limit</span>. <span class="source" tagx="source" title="source">mBio</span>. <span class="year" tagx="year" title="year">2019</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">2</span>). <span class="pub-id"><a href="https://dx.doi.org/10.1128/mBio.00204-19">10.1128/mBio.00204-19</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30940704">30940704</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kobayashi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>. <span class="mixed-article-title" title="mixed-article-title">Compensatory Evolution of Net-Charge in Influenza A Virus Hemagglutinin</span>. <span class="source" tagx="source" title="source">PLOS ONE</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">11</span>.</span></li> <li tag="ref"><a name="pcbi.1007892.ref018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gambaryan</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mastrosovich</span><span class="given-names" tagx="given-names" title="given-names">MN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bender</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kilbourne</span><span class="given-names" tagx="given-names" title="given-names">ED</span></span>. <span class="mixed-article-title" title="mixed-article-title">Differences in the biological phenotype of low-yielding (L) and high-yielding (H) variants of swine influenza virus A/NJ/11/76 are associated with their different receptor-binding activity</span>. <span class="source" tagx="source" title="source">Virology</span>. <span class="year" tagx="year" title="year">1998</span>;<span class="volume" tagx="volume" title="volume">247</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">223</span>â€"<span class="lpage" tagx="lpage" title="lpage">231</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1006/viro.1998.9274">10.1006/viro.1998.9274</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9705915">9705915</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Long</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tobin</span><span class="given-names" tagx="given-names" title="given-names">GJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nara</span><span class="given-names" tagx="given-names" title="given-names">PL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deem</span><span class="given-names" tagx="given-names" title="given-names">MW</span></span>. <span class="mixed-article-title" title="mixed-article-title">Selective pressure to increase charge in immunodominant epitopes of the H3 hemagglutinin influenza protein</span>. <span class="source" tagx="source" title="source">Journal of Molecular Evolution</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">72</span>:<span class="fpage" tagx="fpage" title="fpage">90</span>â€"<span class="lpage" tagx="lpage" title="lpage">103</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00239-010-9405-4">10.1007/s00239-010-9405-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21086120">21086120</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindman</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">W-F</span></span>,<span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szczepankiewicz</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bauer</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nilsson</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linse</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">Salting the charged surface: pH and salt dependence of protein G B1 stability</span>. <span class="source" tagx="source" title="source">Biophysical Journal</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">90</span>:<span class="fpage" tagx="fpage" title="fpage">2911</span>â€"<span class="lpage" tagx="lpage" title="lpage">2921</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1529/biophysj.105.071050">10.1529/biophysj.105.071050</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16443658">16443658</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Araya</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fowler</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span>. <span class="mixed-article-title" title="mixed-article-title">Deep mutational scanning: assessing protein function on a massive scale</span>. <span class="source" tagx="source" title="source">Trends in Biotechnology</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">29</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">435</span>â€"<span class="lpage" tagx="lpage" title="lpage">442</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tibtech.2011.04.003">10.1016/j.tibtech.2011.04.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21561674">21561674</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doud</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bloom</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span>. <span class="mixed-article-title" title="mixed-article-title">Accurate measurement of the effect of all amino-acid mutations on influenza hemagglutinin</span>. <span class="source" tagx="source" title="source">Viruses</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">155</span>.</span></li> <li tag="ref"><a name="pcbi.1007892.ref023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">SchrÃ¶dinger, LLC. The PyMOL Molecular Graphics System, Version 2.2.0; 2018.</span></li> <li tag="ref"><a name="pcbi.1007892.ref024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gamblin</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haire</span><span class="given-names" tagx="given-names" title="given-names">LF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stevens</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ha</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The Structure and Receptor Binding Properties of the 1918 Influenza Hemagglutinin</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">303</span>(<span class="issue" tagx="issue" title="issue">5665</span>):<span class="fpage" tagx="fpage" title="fpage">1838</span>â€"<span class="lpage" tagx="lpage" title="lpage">1842</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1093155">10.1126/science.1093155</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14764886">14764886</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Altman</span><span class="given-names" tagx="given-names" title="given-names">MO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Angeletti</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yewdell</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span>. <span class="mixed-article-title" title="mixed-article-title">Antibody Immunodominance: The Key to Understanding Influenza Virus Antigenic Drift</span>. <span class="source" tagx="source" title="source">Viral Immunology</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">31</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">142</span>â€"<span class="lpage" tagx="lpage" title="lpage">149</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1089/vim.2017.0129">10.1089/vim.2017.0129</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29356618">29356618</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kirkpatrick</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qiu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bahl</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krammer</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>. <span class="mixed-article-title" title="mixed-article-title">The influenza virus hemagglutinin head evolves faster than the stalk domain</span>. <span class="source" tagx="source" title="source">Scientific Reports</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">10432</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-018-28706-1">10.1038/s41598-018-28706-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29992986">29992986</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mayrose</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Graur</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ben-Tal</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pupko</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>. <span class="mixed-article-title" title="mixed-article-title">Comparison of site-specific rate-inference methods for protein sequences: empirical Bayesian methods are superior</span>. <span class="source" tagx="source" title="source">Molecular Biology and Evolution</span>. <span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">21</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">1781</span>â€"<span class="lpage" tagx="lpage" title="lpage">1791</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msh194">10.1093/molbev/msh194</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15201400">15201400</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aevermann</span><span class="given-names" tagx="given-names" title="given-names">BD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">TK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">DF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dauphin</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Influenza Research Database: an integrated bioinformatics resource for influenza virus research</span>. <span class="source" tagx="source" title="source">Nucleic Acids Research</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">45</span>:<span class="fpage" tagx="fpage" title="fpage">D466</span>â€"<span class="lpage" tagx="lpage" title="lpage">D474</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkw857">10.1093/nar/gkw857</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27679478">27679478</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bloom</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span>. <span class="mixed-article-title" title="mixed-article-title">Software for the analysis and visualization of deep mutational scanning data</span>. <span class="source" tagx="source" title="source">BMC bioinformatics</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">16</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">168</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12859-015-0590-4">10.1186/s12859-015-0590-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25990960">25990960</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Venables</span><span class="given-names" tagx="given-names" title="given-names">WN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ripley</span><span class="given-names" tagx="given-names" title="given-names">BD</span></span>. <span class="source" tagx="source" title="source">Modern Applied Statistics with S</span>. <span class="edition" tagx="edition" title="edition">4th ed</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>; <span class="year" tagx="year" title="year">2002</span> Available from: <a href="http://www.stats.ox.ac.uk/pub/MASS4">http://www.stats.ox.ac.uk/pub/MASS4</a>.</span></li> <li tag="ref"><a name="pcbi.1007892.ref031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">R Core Team. R: A Language and Environment for Statistical Computing; 2018. Available from: <a href="https://www.R-project.org/">https://www.R-project.org/</a>.</span></li> <li tag="ref"><a name="pcbi.1007892.ref032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ward</span><span class="given-names" tagx="given-names" title="given-names">JH</span><span class="suffix" tagx="suffix" title="suffix">Jr</span></span><span class="mixed-article-title" title="mixed-article-title">Hierarchical grouping to optimize an objective function</span>. <span class="source" tagx="source" title="source">Journal of the American Statistical Association</span>. <span class="year" tagx="year" title="year">1963</span>;<span class="volume" tagx="volume" title="volume">58</span>(<span class="issue" tagx="issue" title="issue">301</span>):<span class="fpage" tagx="fpage" title="fpage">236</span>â€"<span class="lpage" tagx="lpage" title="lpage">244</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1080/01621459.1963.10500845">10.1080/01621459.1963.10500845</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murtagh</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Legendre</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="mixed-article-title" title="mixed-article-title">Wardâ€™s herarchical agglomerative clustering method: which algorithms implement Wardâ€™s criterion?</span><span class="source" tagx="source" title="source">Journal of Classification</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">31</span>:<span class="fpage" tagx="fpage" title="fpage">274</span>â€"<span class="lpage" tagx="lpage" title="lpage">295</span>.</span></li> <li tag="ref"><a name="pcbi.1007892.ref034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huddleston</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doud</span><span class="given-names" tagx="given-names" title="given-names">MB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hooper</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">NC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Deep mutational scanning of hemagglutinin helps predict evolutionary fates of human H3N2 influenza variants</span>. <span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences</span>. <span class="year" tagx="year" title="year">2018</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1806133115">10.1073/pnas.1806133115</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riley</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barr</span><span class="given-names" tagx="given-names" title="given-names">IG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Broor</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chadha</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Global circulation patterns of seasonal influenza viruses vary with antigenic drift</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="fpage" tagx="fpage" title="fpage">217</span>â€"<span class="lpage" tagx="lpage" title="lpage">220</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nature14460">10.1038/nature14460</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26053121">26053121</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pielou</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>. <span class="mixed-article-title" title="mixed-article-title">The measurement of diversity in different types of biological collections</span>. <span class="source" tagx="source" title="source">Journal of Theoretical Biology</span>. <span class="year" tagx="year" title="year">1966</span>;<span class="volume" tagx="volume" title="volume">13</span>:<span class="fpage" tagx="fpage" title="fpage">131</span>â€"<span class="lpage" tagx="lpage" title="lpage">144</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/0022-5193(66)90013-0">10.1016/0022-5193(66)90013-0</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bloom</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span>. <span class="mixed-article-title" title="mixed-article-title">An Experimentally Informed Evolutionary Model Improves Phylogenetic Fit to Divergent Lactamase Homologs</span>. <span class="source" tagx="source" title="source">Molecular Biology and Evolution</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">31</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">2753</span>â€"<span class="lpage" tagx="lpage" title="lpage">2769</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msu220">10.1093/molbev/msu220</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25063439">25063439</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodrigues</span><span class="given-names" tagx="given-names" title="given-names">JPGLM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teixeira</span><span class="given-names" tagx="given-names" title="given-names">JMC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trellet</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonvin</span><span class="given-names" tagx="given-names" title="given-names">AMJJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">pdb-tools: a swiss army knife for molecular structures</span>. <span class="source" tagx="source" title="source">F1000Research</span><span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">1961</span><span class="pub-id"><a href="https://dx.doi.org/10.12688/f1000research.17456.1">10.12688/f1000research.17456.1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30705752">30705752</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shakin-Eshleman</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spitalnik</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kasturi</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>. <span class="mixed-article-title" title="mixed-article-title">The Amino Acid at the X Position of an Asn-X-Ser Sequon Is an Important Determinant of N-Linked Core-glycosylation Efficiency</span>. <span class="source" tagx="source" title="source">Journal of Biological Chemistry</span>. <span class="year" tagx="year" title="year">1996</span>;<span class="volume" tagx="volume" title="volume">271</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">6363</span>â€"<span class="lpage" tagx="lpage" title="lpage">6366</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.271.11.6363">10.1074/jbc.271.11.6363</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8626433">8626433</a></span></span></li> <li tag="ref"><a name="pcbi.1007892.ref040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wickham</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>. <span class="source" tagx="source" title="source">ggplot2: Elegant Graphics for Data Analysis</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer-Verlag</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span>; <span class="year" tagx="year" title="year">2009</span> Available from: <a href="http://ggplot2.org">http://ggplot2.org</a>.</span></li> </ul> </div> </div>  </body></html>