<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2019/05/06/628073.atom"/>
  <meta name="HW.pisa" content="biorxiv;628073v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="DEN-IM: Dengue Virus identification from shotgun and targeted metagenomics"/>
  <meta name="DC.Identifier" content="10.1101/628073"/>
  <meta name="DC.Date" content="2019-05-06"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Dengue virus (DENV) represents a public health and economic burden in affected countries. The availability of genomic data is key to understand viral evolution and dynamics, supporting improved control strategies. Currently, the use of High Throughput Sequencing (HTS) technologies, which can be applied both directly to patient samples (shotgun metagenomics) and to PCR amplified viral sequences (targeted metagenomics), is the most informative approach to monitor the viral dissemination and genetic diversity. Despite many advantages, these technologies require bioinformatics expertise and appropriate infrastructure for the analysis and interpretation of the resulting data. In addition, the many software solutions available can hamper reproducibility and comparison of results. Here we present DEN-IM, a one-stop, user-friendly, containerised and reproducible workflow for the analysis of DENV sequencing data, both from shotgun and targeted metagenomics approaches. It is able to infer DENV coding sequence (CDS), identify serotype and genotype, and generate a phylogenetic tree. It can easily be run on any UNIX-like system, from local machines to high-performance computing clusters, performing a comprehensive analysis without the requirement of extensive bioinformatics expertise. Using DEN-IM, we successfully analysed two DENV datasets. The first comprised 25 shotgun metagenomic sequencing samples of varying serotype and genotype, including a spiked sample containing the existing four serotypes. The second dataset consisted of 106 targeted metagenomics samples of DENV 3 genotype III where DEN-IM allowed detection of the intra-genotype diversity. The DEN-IM workflow, parameters and execution configuration files, and documentation are freely available at &lt;;https://github.com/B-UMMI/DEN-IM&gt;;."/>
  <meta name="DC.Contributor" content="C I Mendes"/>
  <meta name="DC.Contributor" content="E Lizarazo"/>
  <meta name="DC.Contributor" content="M P Machado"/>
  <meta name="DC.Contributor" content="D N Silva"/>
  <meta name="DC.Contributor" content="A Tami"/>
  <meta name="DC.Contributor" content="M Ramirez"/>
  <meta name="DC.Contributor" content="N Couto"/>
  <meta name="DC.Contributor" content="J W A Rossen"/>
  <meta name="DC.Contributor" content="J A Carriço"/>
  <meta name="article:published_time" content="2019-05-06"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="DEN-IM: Dengue Virus identification from shotgun and targeted metagenomics"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;Dengue virus (DENV) represents a public health and economic burden in affected countries. The availability of genomic data is key to understand viral evolution and dynamics, supporting improved control strategies. Currently, the use of High Throughput Sequencing (HTS) technologies, which can be applied both directly to patient samples (shotgun metagenomics) and to PCR amplified viral sequences (targeted metagenomics), is the most informative approach to monitor the viral dissemination and genetic diversity.&lt;;/p&gt;;&lt;;p&gt;;Despite many advantages, these technologies require bioinformatics expertise and appropriate infrastructure for the analysis and interpretation of the resulting data. In addition, the many software solutions available can hamper reproducibility and comparison of results.&lt;;/p&gt;;&lt;;p&gt;;Here we present DEN-IM, a one-stop, user-friendly, containerised and reproducible workflow for the analysis of DENV sequencing data, both from shotgun and targeted metagenomics approaches. It is able to infer DENV coding sequence (CDS), identify serotype and genotype, and generate a phylogenetic tree. It can easily be run on any UNIX-like system, from local machines to high-performance computing clusters, performing a comprehensive analysis without the requirement of extensive bioinformatics expertise.&lt;;/p&gt;;&lt;;p&gt;;Using DEN-IM, we successfully analysed two DENV datasets. The first comprised 25 shotgun metagenomic sequencing samples of varying serotype and genotype, including a spiked sample containing the existing four serotypes. The second dataset consisted of 106 targeted metagenomics samples of DENV 3 genotype III where DEN-IM allowed detection of the intra-genotype diversity.&lt;;/p&gt;;&lt;;p&gt;;The DEN-IM workflow, parameters and execution configuration files, and documentation are freely available at https://github.com/B-UMMI/DEN-IM.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2019/01/01"/>
  <meta name="citation_mjid" content="biorxiv;628073v1"/>
  <meta name="citation_id" content="628073v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/628073v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/628073v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/628073v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073.full.pdf"/>
  <meta name="citation_doi" content="10.1101/628073"/>
  <meta name="citation_num_pages" content="23"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="628073"/>
  <meta name="citation_author" content="C I Mendes"/>
  <meta name="citation_author_institution" content="Instituto de Microbiologia, Instituto de Medicina Molecular, Faculdade de Medicina, Universidade de Lisboa"/>
  <meta name="citation_author_institution" content="University of Groningen, University Medical Center Groningen, Department of Medical Microbiology and Infection Prevention"/>
  <meta name="citation_author_email" content="cimendes@medicina.ulisboa.pt"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3090-7426"/>
  <meta name="citation_author" content="E Lizarazo"/>
  <meta name="citation_author_institution" content="University of Groningen, University Medical Center Groningen, Department of Medical Microbiology and Infection Prevention"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2749-9780"/>
  <meta name="citation_author" content="M P Machado"/>
  <meta name="citation_author_institution" content="Instituto de Microbiologia, Instituto de Medicina Molecular, Faculdade de Medicina, Universidade de Lisboa"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1786-8522"/>
  <meta name="citation_author" content="D N Silva"/>
  <meta name="citation_author_institution" content="Instituto de Microbiologia, Instituto de Medicina Molecular, Faculdade de Medicina, Universidade de Lisboa"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6827-2673"/>
  <meta name="citation_author" content="A Tami"/>
  <meta name="citation_author_institution" content="University of Groningen, University Medical Center Groningen, Department of Medical Microbiology and Infection Prevention"/>
  <meta name="citation_author" content="M Ramirez"/>
  <meta name="citation_author_institution" content="Instituto de Microbiologia, Instituto de Medicina Molecular, Faculdade de Medicina, Universidade de Lisboa"/>
  <meta name="citation_author" content="N Couto"/>
  <meta name="citation_author_institution" content="University of Groningen, University Medical Center Groningen, Department of Medical Microbiology and Infection Prevention"/>
  <meta name="citation_author" content="J W A Rossen"/>
  <meta name="citation_author_institution" content="University of Groningen, University Medical Center Groningen, Department of Medical Microbiology and Infection Prevention"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7167-8623"/>
  <meta name="citation_author" content="J A Carriço"/>
  <meta name="citation_author_institution" content="Instituto de Microbiologia, Instituto de Medicina Molecular, Faculdade de Medicina, Universidade de Lisboa"/>
  <meta name="citation_reference" content="World Health Organization, “Dengue: guidelines for diagnosis, treatment, prevention, and control,” Special Programme for Research and Training in Tropical Diseases, pp. x, 147, 2009. [Online]. Available: http://whqlibdoc.who.int/publications/2009/9789241547871{_}eng.pdf"/>
  <meta name="citation_reference" content="M. S. Diamond and T. C. Pierson, “Molecular Insight into Dengue Virus Pathogenesis and Its Implications for Disease Control,” Cell, vol. 162, no. 3, pp. 488–492, 2015. [Online]. Available: http://dx.doi.org/10.1016/j.cell.2015.07.005"/>
  <meta name="citation_reference" content="S. Bhatt, P. W. Gething, O. J. Brady, J. P. Messina, A. W. Farlow, C. L. Moyes, J. M. Drake, J. S. Brownstein, A. G. Hoen, O. Sankoh, M. F. Myers, D. B. George, T. Jaenisch, G. R. William Wint, C. P. Simmons, T. W. Scott, J. J. Farrar, and S. I. Hay, “The global distribution and burden of dengue,” Nature, vol. 496, no. 7446, pp. 504–507, 2013. [Online]. Available: http://dx.doi.org/10.1038/nature12060"/>
  <meta name="citation_reference" content="J. Lourenço, W. Tennant, N. R. Faria, A. Walker, S. Gupta, and M. Recker, “Challenges in dengue research: A computational perspective,” Evolutionary Applications, vol. 11, no. 4, pp. 516–533, apr 2018. [Online]. Available: http://doi.wiley.com/10.1111/eva.12554"/>
  <meta name="citation_reference" content="K. C. Leitmeyer, D. W. Vaughn, D. M. Watts, R. Salas, I. Villalobos, de Chacon, C. Ramos, and R. Rico-Hesse, “Dengue virus structural differences that correlate with pathogenesis.” Journal of virology, vol. 73, no. 6, pp. 4738–47, jun 1999. [Online]. Available: http://www.ncbi.nlm.nih.gov/pubmed/10233934{%}0Ahttp://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC112516http://www.ncbi.nlm.nih.gov/pubmed/10233934http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC112516"/>
  <meta name="citation_reference" content="N. L. Yozwiak, P. Skewes-Cox, M. D. Stenglein, A. Balmaseda, E. Harris, and J. L. DeRisi, “Virus identification in unknown tropical febrile illness cases using deep sequencing,” PLoS Neglected Tropical Diseases, vol. 6, no. 2, 2012."/>
  <meta name="citation_reference" content="C. K. Lee, C. W. Chua, L. Chiu, and E. S.-C. Koay, “Clinical use of targeted high-throughput whole-genome sequencing for a dengue virus variant,” Clinical Chemistry and Laboratory Medicine (CCLM), vol. 55, no. 9, p. e209, jan 2017. [Online]. Available: https://www.degruyter.com/view/j/cclm.2017.55.issue-9/cclm-2016-0660/cclm-2016-0660.xmlhttp://www.degruyter.com/view/j/cclm.2017.55.issue-9/cclm-2016-0660/cclm-2016-0660.xml"/>
  <meta name="citation_reference" content="Z. Fatima, M. Idrees, M. A. Bajwa, Z. Tahir, O. Ullah, M. Q. Zia, A. Hussain, M. Akram, B. Khubaib, S. Afzal, S. Munir, S. Saleem, B. Rauff, S. Badar, M. Naudhani, S. Butt, M. Aftab, L. Ali, and M. Ali, “Serotype and genotype analysis of dengue virus by sequencing followed by phylogenetic analysis using samples from three mini outbreaks-2007-2009 in Pakistan,” BMC Microbiology, vol. 11, no. 1, p. 200, 2011. [Online]. Available: http://www.biomedcentral.com/1471-2180/11/200"/>
  <meta name="citation_reference" content="P. Di Tommaso, M. Chatzou, E. W. Floden, P. P. Barja, E. Palumbo, and C. Notredame, “Nextflow enables reproducible computational workflows,” Nature Biotechnology, vol. 35, p. 316, apr 2017. [Online]. Available: http://dx.doi.org/10.1038/nbt.3820http://10.0.4.14/nbt.3820https://www.nature.com/articles/nbt.3820{#}supplementary-information"/>
  <meta name="citation_reference" content="A. M. Bolger, M. Lohse, and B. Usadel, “Trimmomatic: A flexible trimmer for Illumina sequence data,” Bioinformatics, vol. 30, no. 15, pp. 2114–2120, 2014."/>
  <meta name="citation_reference" content="B. Ewing, L. Hillier, M. C. Wendl, and P. Green, “Base-Calling of Automated Sequencer Traces Using Phred. I. Accuracy Assessment,” Genome Research, vol. 8, no. 3, pp. 175–185, mar 1998. [Online]. Available: http://genome.cshlp.org/lookup/doi/10.1101/gr.8.3.175"/>
  <meta name="citation_reference" content="R. Schmieder and R. Edwards, “Quality control and preprocessing of metagenomic datasets,” Bioinformatics, vol. 27, no. 6, pp. 863–864, 2011."/>
  <meta name="citation_reference" content="B. Langmead and S. L. Salzberg, “Fast gapped-read alignment with Bowtie 2, “Nature Methods, vol. 9, no. 4, pp. 357–359, apr 2012. [Online]. Available: http://www.nature.com/articles/nmeth.1923"/>
  <meta name="citation_reference" content="H. Li, “A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data,” Bioinformatics, vol. 27, no. 21, pp. 2987–2993, 2011."/>
  <meta name="citation_reference" content="A. Bankevich, S. Nurk, D. Antipov, A. A. Gurevich, M. Dvorkin, A. S. Kulikov, V. M. Lesin, S. I. Nikolenko, S. Pham, A. D. Prjibelski, A. V. Pyshkin, A. V. Sirotkin, N. Vyahhi, G. Tesler, M. A. Alekseyev, and P. A. Pevzner, “SPAdes: A New Genome Assembly Algorithm and Its Applications to Single-Cell Sequencing,” Journal of Computational Biology, vol. 19, no. 5, pp. 455–477, may 2012. [Online]. Available: http://online.liebertpub.com/doi/abs/10.1089/cmb.2012.0021"/>
  <meta name="citation_reference" content="D. Li, C. M. Liu, R. Luo, K. Sadakane, and T. W. Lam, “MEGAHIT: An ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph,” Bioinformatics, vol. 31, no. 10, pp. 1674–1676, 2015."/>
  <meta name="citation_reference" content="B. J. Walker, T. Abeel, T. Shea, M. Priest, A. Abouelliel, S. Sakthikumar, C. A. Cuomo, Q. Zeng, J. Wortman, S. K. Young, and A. M. Earl, “Pilon: An integrated tool for comprehensive microbial variant detection and genome assembly improvement,” PLoS ONE, vol. 9, no. 11, 2014."/>
  <meta name="citation_reference" content="M. P. Machado, B. Ribeiro-Gonçalves, M. Silva, M. Ramirez, and J. A. Carriço, “Epidemiological Surveillance and Typing Methods to Track Antibiotic Resistant Strains Using High Throughput Sequencing.” Methods in molecular biology (Clifton, N.J.), vol. 1520, pp. 331–356, 2017. [Online]. Available: http://www.ncbi.nlm.nih.gov/pubmed/27873262"/>
  <meta name="citation_reference" content="S. F. Altschul, T. L. Madden, A. A. Schäffer, J. Zhang, Z. Zhang, W. Miller, and D. J. Lipman, “Gapped BLAST and PSI-BLAST: A new generation of protein database search programs,” Nucleic Acids Research, vol. 25, no. 17, pp. 3389–3402, 1997."/>
  <meta name="citation_reference" content="T. Nakamura, K. D. Yamada, K. Tomii, and K. Katoh, “Parallelization of MAFFT for large-scale multiple sequence alignments,” Bioinformatics, vol. 34, no. March, pp. 2490–2492, 2018. [Online]. Available: https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty121/4916099"/>
  <meta name="citation_reference" content="A. Stamatakis, “RAxML version 8: A tool for phylogenetic analysis and post-analysis of large phylogenies,” Bioinformatics, vol. 30, no. 9, pp. 1312–1313, 2014."/>
  <meta name="citation_reference" content="L. Gerhardt, W. Bhimji, S. Canon, M. Fasel, D. Jacobsen, M. Mustafa, J. Porter, and V. Tsulaia, “Shifter: Containers for HPC,” Journal of Physics: Conference Series, vol. 898, p. 082021, oct 2017. [Online]. Available: http://stacks.iop.org/1742-6596/898/i=8/a=082021?key=crossref.b7268cc937fc3b29093062a6749fbbbf"/>
  <meta name="citation_reference" content="G. M. Kurtzer, V. Sochat, and M. W. Bauer, “Singularity: Scientific containers for mobility of compute,” PLoS ONE, vol. 12, no. 5, pp. 1–20, 2017."/>
  <meta name="citation_reference" content="P. Parameswaran, C. Wang, S. B. Trivedi, M. Eswarappa, M. Montoya, A. Balmaseda, and E. Harris, “Intrahost Selection Pressures Drive Rapid Dengue Virus Microevolution in Acute Human Infections,” Cell Host &amp;; Microbe, vol. 22, no. 3, pp. 400–410.e5, sep 2017. [Online]. Available: https://doi.org/10.1016/j.chom.2017.08.003"/>
  <meta name="citation_reference" content="D. E. Wood and S. L. Salzberg, “Kraken: ultrafast metagenomic sequence classification using exact alignments,” Genome Biology, vol. 15, no. 3, p. R46, mar 2014. [Online]. Available: https://doi.org/10.1186/gb-2014-15-3-r46http://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-3-r46"/>
  <meta name="citation_reference" content="P. E. S. Marinho, D. B. De Oliveira, T. M. S. Candiani, A. P. C. Crispim, P. P. M. Alvarenga, F. C. d. S. Castro, J. S. Abrahao, M. Rios, R. S. Coimbra, and E. G. Kroon, “Meningitis associated with simultaneous infection by multiple dengue virus serotypes in children, Brazil,” Emerging Infectious Diseases, vol. 23, no. 1, pp. 115–118, 2017."/>
  <meta name="citation_reference" content="M. N. Reddy, R. Dungdung, L. Valliyott, and R. Pilankatta, “Occurrence of concurrent infections with multiple serotypes of dengue viruses during 2013-2015 in northern Kerala, India,” PeerJ, vol. 5, p. e2970, 2017."/>
  <meta name="citation_reference" content="L. Cuypers, P. Libin, P. Simmonds, A. Nowé, J. Muñoz-Jordán, L. Alcantara, A.-M. Vandamme, G. Santiago, and K. Theys, “Time to Harmonize Dengue Nomenclature and Classification,” Viruses, vol. 10, no. 10, p. 569, oct 2018. [Online]. Available: http://www.mdpi.com/1999-4915/10/10/569"/>
  <meta name="citation_reference" content="B. E. Pickett, D. S. Greer, Y. Zhang, L. Stewart, L. Zhou, G. Sun, Z. Gu, S. Kumar, S. Zaremba, C. N. Larsen, W. Jen, E. B. Klem, and R. H. Scheuermann, “Virus pathogen Database and Analysis Resource (ViPR): A comprehensive bioinformatics Database and Analysis Resource for the Coronavirus research community,” Viruses, vol. 4, no. 11, pp. 3209–3226, 2012."/>
  <meta name="citation_reference" content="T. Pang, T. K. Mak, and D. J. Gubler, “Prevention and control of dengue—the light at the end of the tunnel,” The Lancet Infectious Diseases, vol. 17, no. 3, pp. e79–e87, 2017."/>
  <meta name="citation_reference" content="C. J. Villabona-Arenas and P. M. d. A. Zanotto, “Worldwide Spread of Dengue Virus Type 1,” PLoS One, vol. 8, no. 5, p. e62649, 2013."/>
  <meta name="citation_reference" content="N. Vasilakis and S. C. Weaver, “Chapter 1 The History and Evolution of Human Dengue Emergence,” in Advances in Virus Research, ser. Advances in Virus Research. Academic Press, 2008, vol. 72, pp. 1–76. [Online]. Available: http://www.sciencedirect.com/science/article/pii/S0065352708004016"/>
  <meta name="citation_reference" content="W. Li and A. Godzik, “Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences,” Bioinformatics, vol. 22, no. 13, pp. 1658–1659, 2006. [Online]. Available: https://academic.oup.com/bioinformatics/article/22/13/1658/194225"/>
  <meta name="citation_reference" content="CDC, “CDC DENV-1-4 Real-Time RT-PCR Assay for Detection and Serotype Identification of Dengue Virus,” pp. 1–53, 2013. [Online]. Available: http://www.cdc.gov/dengue/resources/rt{_}pcr/CDCPackageInsert.pdf."/>
  <meta name="citation_reference" content="R. Lanciotti, C. Calisher, D. Gubler, G. Chang, and A. Vorndam, “Rapid detection and typing of dengue viruses from clinical samples by using reverse transcriptase-polymerase chain reaction.” Journal of Clinical Microbiology, vol. 30, no. 3, pp. 545–551, 1992. [Online]. Available: http://jcm.asm.org/content/30/3/545.short"/>
  <meta name="twitter:title" content="DEN-IM: Dengue Virus identification from shotgun and targeted metagenomics"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Dengue virus (DENV) represents a public health and economic burden in affected countries. The availability of genomic data is key to understand viral evolution and dynamics, supporting improved control strategies. Currently, the use of High Throughput Sequencing (HTS) technologies, which can be applied both directly to patient samples (shotgun metagenomics) and to PCR amplified viral sequences (targeted metagenomics), is the most informative approach to monitor the viral dissemination and genetic diversity. Despite many advantages, these technologies require bioinformatics expertise and appropriate infrastructure for the analysis and interpretation of the resulting data. In addition, the many software solutions available can hamper reproducibility and comparison of results. Here we present DEN-IM, a one-stop, user-friendly, containerised and reproducible workflow for the analysis of DENV sequencing data, both from shotgun and targeted metagenomics approaches. It is able to infer DENV coding sequence (CDS), identify serotype and genotype, and generate a phylogenetic tree. It can easily be run on any UNIX-like system, from local machines to high-performance computing clusters, performing a comprehensive analysis without the requirement of extensive bioinformatics expertise. Using DEN-IM, we successfully analysed two DENV datasets. The first comprised 25 shotgun metagenomic sequencing samples of varying serotype and genotype, including a spiked sample containing the existing four serotypes. The second dataset consisted of 106 targeted metagenomics samples of DENV 3 genotype III where DEN-IM allowed detection of the intra-genotype diversity. The DEN-IM workflow, parameters and execution configuration files, and documentation are freely available at &lt;;https://github.com/B-UMMI/DEN-IM&gt;;."/>
  <meta name="og-title" property="og:title" content="DEN-IM: Dengue Virus identification from shotgun and targeted metagenomics"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/628073v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Dengue virus (DENV) represents a public health and economic burden in affected countries. The availability of genomic data is key to understand viral evolution and dynamics, supporting improved control strategies. Currently, the use of High Throughput Sequencing (HTS) technologies, which can be applied both directly to patient samples (shotgun metagenomics) and to PCR amplified viral sequences (targeted metagenomics), is the most informative approach to monitor the viral dissemination and genetic diversity. Despite many advantages, these technologies require bioinformatics expertise and appropriate infrastructure for the analysis and interpretation of the resulting data. In addition, the many software solutions available can hamper reproducibility and comparison of results. Here we present DEN-IM, a one-stop, user-friendly, containerised and reproducible workflow for the analysis of DENV sequencing data, both from shotgun and targeted metagenomics approaches. It is able to infer DENV coding sequence (CDS), identify serotype and genotype, and generate a phylogenetic tree. It can easily be run on any UNIX-like system, from local machines to high-performance computing clusters, performing a comprehensive analysis without the requirement of extensive bioinformatics expertise. Using DEN-IM, we successfully analysed two DENV datasets. The first comprised 25 shotgun metagenomic sequencing samples of varying serotype and genotype, including a spiked sample containing the existing four serotypes. The second dataset consisted of 106 targeted metagenomics samples of DENV 3 genotype III where DEN-IM allowed detection of the intra-genotype diversity. The DEN-IM workflow, parameters and execution configuration files, and documentation are freely available at &lt;;https://github.com/B-UMMI/DEN-IM&gt;;."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2019-05-06"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>DEN-IM: Dengue Virus identification from shotgun and targeted metagenomics | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-705799 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">Dengue virus (DENV) represents a public health and economic burden in affected countries. The availability of genomic data is key to understand viral evolution and dynamics, supporting improved control strategies. Currently, the use of High Throughput Sequencing (HTS) technologies, which can be applied both directly to patient samples (shotgun metagenomics) and to PCR amplified viral sequences (targeted metagenomics), is the most informative approach to monitor the viral dissemination and genetic diversity.</p>
    <p id="p-4">Despite many advantages, these technologies require bioinformatics expertise and appropriate infrastructure for the analysis and interpretation of the resulting data. In addition, the many software solutions available can hamper reproducibility and comparison of results.</p>
    <p id="p-5">Here we present DEN-IM, a one-stop, user-friendly, containerised and reproducible workflow for the analysis of DENV sequencing data, both from shotgun and targeted metagenomics approaches. It is able to infer DENV coding sequence (CDS), identify serotype and genotype, and generate a phylogenetic tree. It can easily be run on any UNIX-like system, from local machines to high-performance computing clusters, performing a comprehensive analysis without the requirement of extensive bioinformatics expertise.</p>
    <p id="p-6">Using DEN-IM, we successfully analysed two DENV datasets. The first comprised 25 shotgun metagenomic sequencing samples of varying serotype and genotype, including a spiked sample containing the existing four serotypes. The second dataset consisted of 106 targeted metagenomics samples of DENV 3 genotype III where DEN-IM allowed detection of the intra-genotype diversity.</p>
    <p id="p-7">The DEN-IM workflow, parameters and execution configuration files, and documentation are freely available at 
     <kbd>
      <a href="https://github.com/B-UMMI/DEN-IM">https://github.com/B-UMMI/DEN-IM</a>
     </kbd>.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">1 Key points</h2>
    <ul class="list-unord " id="list-1">
     <li id="list-item-1">
      <p id="p-8">Understanding DENV transmission in populations where the infection is endemic through the use of metagenomics is a most promising strategy to get insight into the dissemination of the virus and to support measures to prevent or decrease further spread.</p>
     </li>
     <li id="list-item-2">
      <p id="p-9">So far, the analysis, interpretation and dissemination of results encounters computational challenges, and requires appropriate expertise and infrastructure, which poses a challenge for the implementation of metagenomic approaches.</p>
     </li>
     <li id="list-item-3">
      <p id="p-10">We present DEN-IM, a reproducible, containerised and user-friendly workflow for the identification and characterisation of DENV from shotgun and targeted metagenomics data.</p>
     </li>
     <li id="list-item-4">
      <p id="p-11">The HTML reports obtained with DEN-IM can be easily shared, facilitating comparison of results from across the globe.</p>
     </li>
    </ul>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">2 Background</h2>
    <p id="p-12">The Dengue virus (DENV), a single-stranded positive-sense RNA virus belonging to the 
     <em>Flavivirus</em> genus, is one of the most prevalent arboviruses and is mainly concentrated in tropical and subtropical regions. Infection with DENV results in symptoms ranging from mild fever to haemorrhagic fever and shock syndrome [
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. Transmission to humans occurs through the bite of Aedes mosquitoes namely 
     <em>Aedes aegypti</em> and 
     <em>Aedes albopictus</em> [
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>] In 2010, it was predicted that the burden of dengue disease reached 390 million cases per year worldwide [
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. The high morbidity and mortality of dengue makes it the arbovirus with the highest clinical significance [
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>].
    </p>
    <p id="p-13">The viral genome of ~11,000 nucleotides, consists of a Coding Sequence (CDS) of approximately 10.2 Kb that is translated into a single polyprotein encoding three structural proteins (capsid – C, premembrane – prM, envelope – E) and seven non-structural proteins (NS1, NS2A, NS2B, NS3, NS4A, NS4B and NS5. Additionally, the genome contains two Non-Coding Regions (NCRs) at their 5’ and 3’ ends [
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>].
    </p>
    <p id="p-14">DENV can be classified into four serotypes (1, 2, 3 and 4), differing from each other by 25% to 40% at the amino acid level. They are further classified into genotypes that vary by up to ~3% at the amino acid level [
     <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>]. The DENV-1 serotype comprises five genotypes (I-V), DENV-2 groups six (I-VI, also named American, Cosmopolitan, Asian-American, Asian II, Asian I and Sylvatic), DENV-3 four (I-III and V), and DENV-4 also four (I-IV).
    </p>
    <p id="p-15">DENV is a significant public health challenge in countries where the infection is endemic due to the high health and economic burden. Despite the emergence of novel therapies and ecological strategies to control the mosquito vector, there are still important knowledge gaps in the virus biology and its epidemiology [
     <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a>].
    </p>
    <p id="p-16">The implementation of a surveillance system relying on High Throughput Sequencing (HTS) technologies allows the simultaneous identification and surveillance of DENV cases. Due to the high sensitivity of these technologies, previous studies showed that viral sequences can be directly obtained from patient sera using a shotgun metagenomics approach [
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]. Alternatively, HTS can be used in a targeted metagenomics approach in which a PCR step is used to pre-amplify viral sequences before sequencing. In recent years, HTS has been successfully used as a tool for identification of DENV directly from clinical samples [
     <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>,
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. This also allows the rapid identification of the serotype and genotype important for disease management as the genotype may be associated with disease outcome [
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>].
    </p>
    <p id="p-17">Several initiatives aim to facilitate the identification of the DENV serotype and genotype from HTS data. The Genome Detective project (
     <kbd>
      <a href="https://www.genomedetective.com/">https://www.genomedetective.com/</a>
     </kbd>) offers an online Dengue Typing Tool (
     <kbd>
      <a href="https://www.genomedetective.com/app/typingtool/dengue/">https://www.genomedetective.com/app/typingtool/dengue/</a>
     </kbd>) relying on BLAST and phylogenetic methods in order to identify the closest serotype and genotype, but it requires as input assembled genomes in FASTA format. Alternatively, the same project offers a Genome Detective Typing Tool (
     <kbd>
      <a href="https://www.genomedetective.com/app/typingtool/virus/">https://www.genomedetective.com/app/typingtool/virus/</a>
     </kbd>) identifying viruses present in a sample.
    </p>
    <p id="p-18">We developed DEN-IM as a ready-to-use, one-stop, reproducible bioinformatic analysis workflow for the processing and phylogenetic analysis of DENV using paired-end raw HTS data. DEN-IM is implemented in Nextflow [
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>], a workflow manager software that uses Docker (
     <kbd>
      <a href="https://www.docker.com">https://www.docker.com</a>
     </kbd>) containers with pre-installed software for all the workflow tools. The DEN-IM workflow, as well as parameters and documentation, are available at 
     <kbd>
      <a href="https://github.com/B-UMMI/DEN-IM">https://github.com/B-UMMI/DEN-IM</a>
     </kbd>.
    </p>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">3 The DEN-IM Workflow</h2>
    <p id="p-19">DEN-IM is a user-friendly automated workflow allowing the analysis of shotgun or targeted metagenomics data for the identification, serotyping, genotyping, and phylogenetic analysis of DENV. It is implemented in Nextflow, a workflow management system that allows the effortless deployment and execution of complex distributed computational workflows.</p>
    <p id="p-20">The workflow is composed of five blocks: 1. Quality Control and Trimming, 2. Retrieval of DENV sequences, 3. Assembly, 4. 
     <em>in silico</em> Typing, and 5. Phylogeny, described in more detail in 
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. DEN-IM accepts as input raw paired-end sequencing data (FASTQ files), and informs the user with an interactive HTML report with information on the quality control, mapping, assembly typing and phylogenetic analysis, as well as all the output files of the whole pipeline.
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The DEN-IM workflow separated into five different components. The raw sequencing reads are provided as input to the first block (in blue), responsible for quality control and elimination of low-quality reads and sequences. After successful pre-processing of the reads, these enter the second block (green) for retrieval of the DENV reads using the mapping database of 3830 complete DENV genomes as reference. This block also provides an initial estimate of the sequencing depth. After the de novo assembly and assembly correction block (yellow), the coding sequences (CDSs) are retrieved and are then classified with the reduced complexity DENV typing database containing 161 sequences representing the known diversity of DENV serotypes and genotypes (red).If a complete CDS fails to be assembled, the reads are mapped against the DENV typing database and a consensus sequence is obtained for classification and phylogenetic inference. All CDSs are aligned and compared in a phylogenetic analysis (purple). Lastly, a report is compiled (gray) with the results of all the blocks of the workflow." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1582145413" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;The DEN-IM workflow separated into five different components. The raw sequencing reads are provided as input to the first block (in blue), responsible for quality control and elimination of low-quality reads and sequences. After successful pre-processing of the reads, these enter the second block (green) for retrieval of the DENV reads using the mapping database of 3830 complete DENV genomes as reference. This block also provides an initial estimate of the sequencing depth. After the &lt;;em&gt;;de novo&lt;;/em&gt;; assembly and assembly correction block (yellow), the coding sequences (CDSs) are retrieved and are then classified with the reduced complexity DENV typing database containing 161 sequences representing the known diversity of DENV serotypes and genotypes (red).If a complete CDS fails to be assembled, the reads are mapped against the DENV typing database and a consensus sequence is obtained for classification and phylogenetic inference. All CDSs are aligned and compared in a phylogenetic analysis (purple). Lastly, a report is compiled (gray) with the results of all the blocks of the workflow.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F1.medium.gif" width="440" height="360"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F1.medium.gif" width="440" height="360"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1.</span>
      <p id="p-21" class="first-child">The DEN-IM workflow separated into five different components. The raw sequencing reads are provided as input to the first block (in blue), responsible for quality control and elimination of low-quality reads and sequences. After successful pre-processing of the reads, these enter the second block (green) for retrieval of the DENV reads using the mapping database of 3830 complete DENV genomes as reference. This block also provides an initial estimate of the sequencing depth. After the 
       <em>de novo</em> assembly and assembly correction block (yellow), the coding sequences (CDSs) are retrieved and are then classified with the reduced complexity DENV typing database containing 161 sequences representing the known diversity of DENV serotypes and genotypes (red).If a complete CDS fails to be assembled, the reads are mapped against the DENV typing database and a consensus sequence is obtained for classification and phylogenetic inference. All CDSs are aligned and compared in a phylogenetic analysis (purple). Lastly, a report is compiled (gray) with the results of all the blocks of the workflow.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="sec-4" class="subsection">
     <h3>3.1 1. Quality Control and Trimming</h3>
     <p id="p-22">The Quality Control (QC) and Trimming block starts with a process to verify the integrity of the input data. If the sequencing files are corrupted, the execution of the analysis of that sample is terminated.</p>
     <p id="p-23">The sequences are then processed by FastQC (
      <kbd>
       <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>
      </kbd>, version 0.11.7) to determine the quality of the individual base pairs of the raw data files. The low quality bases and adapter sequences are trimmed by Trimmomatic [
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>] (version 0.36). By default, Trimmomatic removes the Illumina Nextera, TruSeq2 and TruSeq3 adapter sequences that are detected, and relies on Phred scores [
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>] and FastQC analysis data to determine to which extend the 5’ and 3’ ends of the reads need to be trimmed. The crop and headcrop setting is calculated per sample, based on the GC ratio of each half of the read. By default, the window size is set to 5 nucleotides with a minimum average quality of 20. In addition, paired-end reads with a read length shorter than 55 nucleotides after trimming are removed from further analyses.
     </p>
     <p id="p-24">Lastly, the low complexity sequences, containing over 50% of poly-A, poly-N or poly-T nucleotides, are filtered out of the raw data using PrinSeq [
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>] (version 0.10.4).
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>3.2 2. Retrieval of DENV Sequences</h3>
     <p id="p-25">In the second step, DENV sequences are selected from the sample using Bowtie2 [
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>] (version 2.2.9) and Samtools [
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>] (version 1.4.1). As a reference we provide the DENV mapping database, a curated DENV database composed of 3830 complete DENV genomes (see Methods, DENV Reference Database). A permissive approach is followed by allowing for mates to be kept in the sample even when only one read maps to the database in order to keep as many DENV derived reads as possible. The output of this step is a set of processed reads of putative DENV origin.
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>3.3 3. Assembly</h3>
     <p id="p-26">DEN-IM applies a two assembler approach to generate assemblies of the DENV CDS. To obtain a high confidence assembly, the processed reads are first 
      <em>de novo</em> assembled with SPAdes [
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>] (version 3.12.0). If the full CDS fails to be assembled into a single contig, the data is re-assembled with MEGAHIT assembler [
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>] (version 1.1.3), a more permissive assembler developed to retrieve longer sequences from metagenomics data. The resulting assemblies are corrected with Pilon [
      <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>] (version 1.22) after mapping the processed reads to the assemblies with Bowtie2 [
      <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>].
     </p>
     <p id="p-27">If more than one complete CDS is present in a sample, each of the sequences will follow the rest of the DEN-IM workflow independently. If no full CDS is assembled neither with SPAdes nor with MEGAHIT, the processed reads are passed on to the next step for consensus generation by mapping, effectively constituting DEN-IM’s two pronged approach using both assemblers and mapping.</p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>3.4 4. Typing</h3>
     <p id="p-28">For each DENV complete CDS, the serotype and genotype is determined with the Seq_Typing tool (
      <kbd>
       <a href="https://github.com/B-UMMI/seq_typing">https://github.com/B-UMMI/seq_typing</a>
      </kbd>, version 2.0) [
      <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>] using BLAST [
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>] and the custom Typing database of DENV containing 161 complete sequences (see Methods, DENV Reference Database). The BLAST results are first cleaned to get the best hit for each sequence in the database, based on alignment length, similarity, e-value and number of gaps. The tool determines which reference sequence is more closely related to the query based on the identity and length of the sequence covered, returning the serotype and genotype of the reference sequence for the purpose of classifying the query.
     </p>
     <p id="p-29">If a complete CDS fails to be obtained through the assembly process, the processed reads are mapped against the DENV typing database, with Bowtie2 [
      <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a>], using Seq_Typing tool, with similar criteria for coverage and identity to those used with the BLAST approach. If a type is determined, the consensus sequence obtained follows through to the next step in the workflow. Otherwise, the sample is classified as Non-Typable and its process terminated.
     </p>
    </div>
    <div id="sec-8" class="subsection">
     <h3>3.5 5. Phylogeny</h3>
     <p id="p-30">All DENV complete CDSs and consensus sequences analysed in a workflow execution are aligned with MAFFT [
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>] (version 7.402), in auto mode and with orientation being automatically determined and adjusted. With the resulting alignment, a Maximum Likelihood tree is inferred with RaXML [
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>] (version 8.2.11), using as default the GTR-Γ substitution model and a 500 times bootstrap.
     </p>
     <p id="p-31">Optionally, the closest reference sequence in the DENV typing database to each analysed sample can be retrieved and included in the phylogenetic analysis.</p>
    </div>
   </div>
   <div class="section" id="sec-9">
    <h2 class="">4 Workflow Execution</h2>
    <p id="p-32">The DEN-IM workflow can be executed in any UNIX-based system, from local machines to high-performance computing clusters (HPC) facilities with Nextflow and a container engine installation, such as Docker (
     <kbd>
      <a href="https://www.docker.com/">https://www.docker.com/</a>
     </kbd>, Shifter [
     <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>] or Singularity [
     <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>].
    </p>
    <p id="p-33">Due to its Nextflow implementation, DEN-IM allows for out-of-the-box high-level parallelization and offers direct support for distributed computational environments. DEN-IM integrates Docker containerised images for all the tools necessary for its execution, ensuring reproducibility and the tracking of both software code and version, regardless of the operating system used. The Docker images provided are also compatible with other container engines.</p>
    <p id="p-34">Users can customise the workflow execution either by using command line options or by modifying a simple plain-text configuration file (params.config). The version of each of the tools used in DEN-IM can be changed by providing new container tags in the appropriate configuration file (containers.config). The resources for each process can also be changed (resources.config). To make the execution of the workflow as simple as possible, a set of default parameters and directives is provided.</p>
    <p id="p-35">The local installation of the DEN-IM workflow, including the docker containers with all the tools needed and the curated DENV database, requires 15 Gigabytes of free disk space. The minimum requirements to execute the workflow are at least 5 Gigabytes of memory and 4 CPUs, although 7 Gigabytes of memory is advised. The disk space required for execution depends greatly on the size of the input data, but for the datasets used in this article DEN-IM generates approximately 20 Gb data per Gb of input data.</p>
   </div>
   <div class="section" id="sec-10">
    <h2 class="">5 Output and Report</h2>
    <p id="p-36">The output files of all tools are stored in the ‘results’ folder in the directory of the DEN-IM execution. The execution log file for each component, as well as a log file for the execution of the workflow, are also available.</p>
    <p id="p-37">DEN-IM creates an interactive HTML report (
     <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure S1</a>), stored in the ‘pipeline_results’ directory, containing all the information in the results divided into four sections: report overview, tables, charts and phylogenetic tree. The report can be easily exchanged between collaborators by compressing and sharing the “pipeline_report” folder.
    </p>
    <p id="p-38">The report overview contains information about the number of samples in the analysis. It allows for selection, filtering and highlighting of particular samples and tools in the workflow.</p>
    <p id="p-39">The table section contains the results and statistics the quality control, assembly, read mapping and 
     <em>in silico</em> typing results. The 
     <strong>
      <em>in silico</em> typing table
     </strong> contains the results of the serotype and genotype of each CDS analysed, as well as identity and coverage and GenBank ID for the closest reference in the DENV typing database.
    </p>
    <p id="p-40">The 
     <strong>quality control table</strong> shows information regarding the number of raw base pairs and number of reads in the raw input files and the percentage of trimmed reads. The 
     <strong>mapping table</strong> includes the results for the mapping of the trimmed reads to the DENV mapping database, including the overall alignment rate, and an estimation of the sequence depth including only the DENV reads. For the 
     <strong>assembly statistics table</strong>, the number of CDSs in each sample is included, and the number of contigs and the number of assembled base pairs generated by either SPAdes or MEGAHIT assemblers. The number of contigs and assembled base pairs after correction with Pilon is also presented in the table. Warning and fail messages are included in each table, as well as the ranking of the value in the cell in relation to other values in the same column, which are shown with a grey bar (
     <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure S1</a>).
    </p>
    <p id="p-41">The 
     <strong>assembled contig size distribution scatter plot</strong> is available in the chart section, showing the contig size distribution for the Pilon corrected assembled CDSs.
    </p>
    <p id="p-42">Lastly, a 
     <strong>phylogenetic tree</strong> is included, rooted at midpoint for visualisation purposes, and with each tip coloured according to the genotyping results. If the option to retrieve the closest typing reference is selected, these sequences are also included in the tree with respective typing metadata. The tree can be displayed in several conformationsprovided by Phylocanvas JavaScript library (
     <kbd>
      <a href="http://phylocanvas.net">http://phylocanvas.net</a>
     </kbd>, version 2.8.1) and it is possible to zoom in or collapse selected branches. The support bootstrap values of the branches can be displayed and the tree can be exported as a Newick tree file or as a PNG image.
    </p>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">6 Results</h2>
    <p id="p-43">To evaluate the DEN-IM workflow performance, we analysed two datasets, one containing shotgun metagenomics sequencing data of patient samples and another with targeted metagenomics sequencing data obtained from Parameswaran 
     <em>et al</em> [
     <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>].
    </p>
    <div id="sec-12" class="subsection">
     <h3>6.1 The Shotgun Metagenomics Dataset</h3>
     <p id="p-44">We analysed 22 shotgun metagenomics paired-end short-read Illumina sequencing datasets of clinical positive dengue cases (
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>)(identified as such by positive qRT-PCR/RT-PCR test and seroconversion test), positive control (purified from DENV culture) and one negative control (blank), and a spiked sample containing the 4 DENV serotypes.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/717093/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/717093/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/717093" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <p id="p-45" class="first-child">Number of raw base pairs, overall alignment rate against the DENV mapping database, estimated coverage depths and serotype and genotype for 25 shotgun metagenomics sequencing samples.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-46">The workflow was executed using the default parameters and directives for resources, with the option to include the closest typing references in the final tree.</p>
     <p id="p-47">The negative control and the 92-1001 sample has no reads after trimming and filtering of low complexity reads, therefore they were removed from further analysis.</p>
     <p id="p-48">When mapping to the DENV mapping database, the percentage of DENV reads in the 21 clinical samples, positive control and spiked sample passing QC ranged from 0.01% (sample UCUG0186) to 85.38% (sample Positive Control). After coverage depth estimation, the analysis of the samples 91-0115 and UCUG0186 was terminated since they did not meet the threshold criterion of having an estimated depth of coverage of 10x.</p>
     <p id="p-49">In the assembly module, the remaining 19 clinical samples, the spiked sample and the positive control were assembled with DEN-IM’s two assembler approach. Twenty-four full CDS were assembled (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>), despite originally having DENV reads content as low as 0.03% of the total number of reads. Sixteen samples, including the spiked sample and the positive control, were assembled in the first step with the SPAdes assembler, and five in the second with the MEGAHIT assembler. In the spiked sample, all four CDSs were successfully assembled and recovered.
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Contig size distribution for the shotgun metagenomics sequencing dataset. Each dot depicts an assembled DENV contig. Above the 10Kb are full CDS of DENV." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1582145413" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Contig size distribution for the shotgun metagenomics sequencing dataset. Each dot depicts an assembled DENV contig. Above the 10Kb are full CDS of DENV.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F2.medium.gif" width="440" height="151"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F2.medium.gif" width="440" height="151"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <p id="p-50" class="first-child">Contig size distribution for the shotgun metagenomics sequencing dataset. Each dot depicts an assembled DENV contig. Above the 10Kb are full CDS of DENV.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-51">The serotype and genotype was successfully determined for the 24 DENV CDSs by BLAST (
      <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure S1</a>, 
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The most common were serotype 2 genotype III (AsianAmerican) and serotype 4 genotype II, with 8 samples each (33.33%), followed by serotype 3 genotype III (n=5, 20.83%), serotype 1 genotype V (n=2, 8.33%) and serotype 2 genotype V (Asian I) (n=1, 4.17%). All CDSs recovered and the respective closest reference genome in the typing database were aligned and a maximum likelihood phylogenetic tree was obtained to visualise the relationship between the samples (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). There was a perfect concordance between the results of serotyping and genotyping and the major groups in the tree.
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Maximum Likelihood tree in the DEN-IM report for the 24 complete CDSs (n=21 samples) obtained with the metagenomics dataset and the respective closest references in the typing database (identified by their GenBank ID). The tree is midpoint rooted for visualisation purposes. The colours depicts the DENV genotyping results." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1582145413" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Maximum Likelihood tree in the DEN-IM report for the 24 complete CDSs (n=21 samples) obtained with the metagenomics dataset and the respective closest references in the typing database (identified by their GenBank ID). The tree is midpoint rooted for visualisation purposes. The colours depicts the DENV genotyping results.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F3.medium.gif" width="440" height="196"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F3.medium.gif" width="440" height="196"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <p id="p-52" class="first-child">Maximum Likelihood tree in the DEN-IM report for the 24 complete CDSs (n=21 samples) obtained with the metagenomics dataset and the respective closest references in the typing database (identified by their GenBank ID). The tree is midpoint rooted for visualisation purposes. The colours depicts the DENV genotyping results.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-13" class="subsection">
     <h3>6.2 The Targeted Metagenomics Dataset</h3>
     <p id="p-53">To validate DEN-IM’s performance in a targeted metagenomics approach, 106 HTS datasets of PCR products using primers targeting DENV-3 [
      <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>] were analysed. As with the shotgun metagenomics dataset, the workflow was executed using the default parameters and directives for resources. This time not including the closest reference genome present in the typing database in the multiple sequence alignment due to the large number of input samples.
     </p>
     <p id="p-54">No samples failed the quality control block. The proportion of DENV reads ranged from 24.72% (SRR5821236) to 99.81% (SRR5821202) of the total processed reads. The samples with less than 70% DENV DNA were taxonomic profiled with the Kraken2 [
      <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>] with the minikraken2_v2 database (
      <kbd>
       <a href="ftp://ftp.ccb.jhu.edu/pub/data/kraken2_dbs/minikraken2_v2_8GB_201904_UPDATE.tgz">ftp://ftp.ccb.jhu.edu/pub/data/kraken2_dbs/minikraken2_v2_8GB_201904_UPDATE.tgz</a>
      </kbd>) and the source of the contamination was determined to have come largely from Human DNA (Table S2).
     </p>
     <p id="p-55">Of the 106 samples, 43 (40.60%) managed to assemble a complete CDS sequence (Table S1) whereas a mapping approach was used for the remaining 63 samples (59.90%) and a consensus CDS was generated. For the assembled CDSs, all but one were assembled with MEGAHIT after not producing a full CDS with SPAdes. Moreover, pronounced variation on the size of the assembled contigs is evident in the contig size distribution plot (
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).
     </p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Contig size distribution of the targeted metagenomics dataset. Each dot depicts an assembled DENV contig. Above the 10Kb are full CDS of DENV. Contigs belonging from samples that assembled a complete DENV CDS are highlighted in green, whereas the remaining are coloured in grey." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1582145413" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Contig size distribution of the targeted metagenomics dataset. Each dot depicts an assembled DENV contig. Above the 10Kb are full CDS of DENV. Contigs belonging from samples that assembled a complete DENV CDS are highlighted in green, whereas the remaining are coloured in grey.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F4.medium.gif" width="440" height="151"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F4.medium.gif" width="440" height="151"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 4.</span>
       <p id="p-56" class="first-child">Contig size distribution of the targeted metagenomics dataset. Each dot depicts an assembled DENV contig. Above the 10Kb are full CDS of DENV. Contigs belonging from samples that assembled a complete DENV CDS are highlighted in green, whereas the remaining are coloured in grey.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-57">All 106 CDSs recovered belonged to serotype 3 genotype III. Despite the same classification, the maximum likelihood tree indicates that there is detectable genetic diversity within the samples (
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Maximum likelihood circular tree in the DEN-IM report for the 106 complete CDSs obtained with the targeted metagenomics dataset (n=106). All samples belong to serotype 3 genotype III." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1582145413" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Maximum likelihood circular tree in the DEN-IM report for the 106 complete CDSs obtained with the targeted metagenomics dataset (n=106). All samples belong to serotype 3 genotype III.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F5.medium.gif" width="440" height="439"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F5.medium.gif" width="440" height="439"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/06/628073/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 5.</span>
       <p id="p-58" class="first-child">Maximum likelihood circular tree in the DEN-IM report for the 106 complete CDSs obtained with the targeted metagenomics dataset (n=106). All samples belong to serotype 3 genotype III.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-14">
    <h2 class="">7 Discussion</h2>
    <p id="p-59">We have successfully analysed two DENV datasets, one comprising 25 shotgun metagenomics sequencing data and 106 targeted metagenomics data.</p>
    <p id="p-60">In the first dataset, we recovered 24 CDSs from 19 clinical samples, a spiked sample and a positive control that were correctly serotyped and genotyped. Besides the negative control, 3 samples did not return typing information due to failing quality checks. In one case (92-1001), no DENV reads after quality control processing were detected as all the reads contained highly repetitive sequences (AAA; TTTT) and were filtered out. The two others (91-0115 and UCUG0186) had a low proportion of DENV reads (0.05% and 0.01%) and an estimated depth of coverage lower than the 10x threshold criterion (3.17x and 5.65x, respectively). Sequence data of sample CC0186 contained only 960 DENV reads (0.03%), but these were successfully assembled into a CDS with an estimated depth of coverage of 14.71x.</p>
    <p id="p-61">The proportion of DENV reads was the metagenomic samples is very variable. This may reflect the viral load in patients in which DENV was detected by PCR. In the spiked sample, containing 4 distinct DENV serotypes, all four were correctly detected despite not being present in equal concentrations(see Methods, Targeted Metagenomics Sequencing Data). This resulted in different coverage of each serotype CDS (2032.31 times coverage for DENV-2, 229.02 times coverage for DENV-1, 76.47 times coverage for DENV-3 and 29.78 times coverage for DENV-4), in accordance with the ranking order of RT-PCR results. It highlights the potential of the DEN-IM workflow to accurately detect and recover multiple DENV genomes from samples with DENV co-infection, even if the serotypes are present in low abundance. Indeed, recent studies from areas of high endemicity suggest that co-infection with multiple DENV serotypes may frequently occur [
     <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>] [
     <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>] and the co-circulation of different DENV strains of the same serotype, but distinct genotypes, in these areas [
     <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>] raises the possibility of simultaneous infection with more than one genotype.
    </p>
    <p id="p-62">When analysing the targeted metagenomics dataset, only 43 CDS out of 106 samples were 
     <em>de novo</em> assembled. For the remaining 63 samples, consensus sequences were obtained through mapping. In all samples DENV 3-III was correctly identified, demonstrating the success of DEN-IM’s two pronged approach of combining assemblers and mapping. We suggest that the lower assembly success of the targeted metagenomics data may be related to errors during the amplification process resulting in low quality reads ends which are then trimmed by the quality control block potentially affecting the assembly process as the overlapping regions are diminished.
    </p>
    <p id="p-63">DEN-IM is built with modularity and containerisation as keystones, leveraging the parallelization of processes and guaranteeing reproducible analyses across platforms. The modular design allows for new modules to be easily added and tools that become outdated can be easily updated, ensuring DEN-IM’s sustainability. The software versions are described in the Nextflow script and configuration files, and in the dockerfiles for each container, allowing traceability of each step of data processing.</p>
    <p id="p-64">Being developed in Nextflow, DEN-IM runs on any UNIX-like system and provides out-of-the-box support for several job schedulers (e.g., PBS, SGE, SLURM) and integration with containerised software like Docker or Singularity. While it has been developed to be ready to use by non-experts, not requiring any software installation or parameter tuning, it can still be easily customised through the configuration files.</p>
    <p id="p-65">The interactive HTML reports (
     <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure S1</a>) provide an intuitive platform for data exploration, allowing the user to highlight specific samples, filter and re-order the data tables, and export the plots as needed. Together with the workflow and software containers, a database containing 3830 complete DENV genomes for DENV sequence retrieval and a subset database with 161 curated DENV genomes for serotyping and genotyping are provided. While constructing these databases, the obstacles reported by Cuypers 
     <em>et al</em> [
     <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>] were apparent, namely the lack of formal definition of a DENV genotype and the lack of a standardised classification procedure that could assign sequences to a previously defined genotypic/sub-genotypic clade [
     <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a>]. Discrepancies between the phylogenetic relationship and the genotype assignment were frequent and, throughout this study, the classification of some strains within the ViPR database [
     <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>] was updated.
    </p>
    <p id="p-66">As suggested previously [
     <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">28</a>], further evaluation of the DENV classification will benefit future research and investigation into the population dynamics of this virus. Our typing approach was designed to use the currently accepted DENV classification. However, DEN-IM can be easily modified if a new DENV classification system is to be established in the future.
    </p>
    <p id="p-67">In conclusion, we provide a user-friendly workflow that makes it possible to analyse paired-end raw sequencing data from shotgun or targeted metagenomics for the presence, typing and phylogenetic analysis of DENV. The use of containerised workflows, together with shareable reports, will allow an easier comparison of results globally, promoting collaborations that can benefit the populations where DENV is endemic. The DEN-IM source code is freely available in the DEN-IM GitHub repository (
     <kbd>
      <a href="https://github.com/B-UMMI/DEN-IM">https://github.com/B-UMMI/DEN-IM</a>
     </kbd>), which includes a wiki with full documentation and easy to follow instructions.
    </p>
   </div>
   <div class="section" id="sec-15">
    <h2 class="">8 Potential implications</h2>
    <p id="p-68">The burden of DENV disease is already large, but is still increasing as the risk of exposure to the virus is increasing, not only through travel to endemic areas but also due to the expansion of the geographic areas of the mosquito vectors and the disease [
     <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>].
    </p>
    <p id="p-69">The decreasing costs and wider availability of HTS makes it an ideal technology to monitor DENV transmission, including the direct processing of patient samples, either through the use of a more affordable targeted metagenomics approach or through shotgun metagenomics. Either way, a ready to use bioinformatics workflow that enables the reproducible analysis of DENV is particularly relevant.</p>
    <p id="p-70">DEN-IM was designed to perform a comprehensive analysis without the requirement of extensive bioinformatics expertise in order to generate either assemblies or consensus of full DENV CDSs and to identify the serotype and genotype of the DENV present in the sample. Although we did not exhaustively test the capacity of DEN-IM to detect co-infection with multiple DENV serotypes and genotypes, all four genotypes present in the spiked sample were accurately detected. This raises the possibility that DEN-IM can play a role in the identification of these cases whose prevalence is increasingly appreciated in highly endemic areas. Moreover, although being ready-to-use, the DEN-IM workflow can be easily customised to optimise the data analysis.</p>
    <p id="p-71">DEN-IM enables reproducible and collaborative research, benefiting a wide group of researchers regardless of their computational expertise and resources available.</p>
   </div>
   <div class="section" id="sec-16">
    <h2 class="">9 Methods</h2>
    <div id="sec-17" class="subsection">
     <h3>9.1 DENV Reference Database</h3>
     <p id="p-72">We have compiled a database of 3830 complete DENV genomes obtained from the NIAID Virus Pathogen Database and Analysis Resource (ViPR) in January 2019 [
      <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>] (
      <kbd>
       <a href="http://www.viprbrc.org/">http://www.viprbrc.org/</a>
      </kbd>). The sequences were distributed unevenly throughout the four DENV serotypes, with DENV-1 being the most represented with 1636 sequences (42.72%), followed by DENV-2 with 1067 sequences (27.86%), DENV-3 with 807 sequences (21.07%), and DENV-4 with 320 sequences (8.36%). The selection criteria for the search were as follows: a) complete genome sequence only, b) human host only, c) collection year (1950-2018). Data available from all countries was included and duplicated sequences were removed and only the sequences with sub-type data were kept. A representative of DENV serotype 1 genotype III was introduced (EF457905, recovered from monkey) as no representatives were available with the search criteria used. This genotype is Sylvatic and considered extinct [
      <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>] [
      <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>]. Additionally, any sample with IUPAC codes in the sequence provided were excluded.
     </p>
     <p id="p-73">In order to recover the maximum number of DENV reads from the input HTS data in the first mapping step (
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), we maintained the database with the 3830 complete DENV genomes to retain as much diversity as possible. This database is referred as “DENV mapping database”.
     </p>
     <p id="p-74">For typing purposes, overly similar sequences in the collection were removed from the database by clustering the sequences in each serotype at 98% nucleotide similarity with CD-HIT [
      <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>], leaving 161 representative sequences of all described DENV serotypes and genotypes, with 46 DENV-1 sequences (Table S4), 63 DENV-2 (Table S5), 25 DENV-3 (Table S6) and 27 DENV-4 (Table S7). This database is referred as “DENV typing database”. This step is necessary to speed up the classification step for genotyping.
     </p>
     <p id="p-75">Phylogenetic analysis of typing collection was performed by aligning the reference genomes with MAFFT [
      <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>], in auto mode and with automatic sequence orientation adjustment. A phylogenetic tree was inferred with RAxML (version 8.12.11) [
      <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>] using the GTR-Γ substitution model and 500 times bootstrap. The resulting trees are available as supplemental material (
      <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figures S2</a> to 
      <a id="xref-fig-10-1" class="xref-fig" href="#F10">S5</a>).
     </p>
     <p id="p-76">The sequence JF459993 from in the DENV-1 collection, as of April 2019, was annotated in ViPR as belonging to genotype IV, but in our analysis it clustered within genotype I clade (
      <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure S2</a>). The classification of DENV-1 I was also obtained from GenomeDetective Dengue Typing Tool (
      <kbd>
       <a href="https://www.genomedetective.com/app/typingtool/dengue/">https://www.genomedetective.com/app/typingtool/dengue/</a>
      </kbd>), so we proceeded to alter the annotation of this particular sample (Table S4). In order to harmonise dengue nomenclature, the system adopted uses Roman-numeric labels to identify the genotype, with the exception of Serotype 2 (Table S5), which used both Roman-numeric and geographic origin due to the widespread adoption of the latter.
     </p>
    </div>
    <div id="sec-18" class="subsection">
     <h3>9.2 Workflow Parameters</h3>
     <p id="p-77">The short-read paired-end data is passed as input through the “
      <em>–fastq</em>” parameter, that by default is set to match all files in the “fastq” folder that match the pattern “
      <em>*_R1,2*</em>”. In the process to verify the integrity of the paired-end raw sequencing data, the integrity of the input files is assessed by attempting to decompress and read the files. An estimation of the depth of coverage is also performed. By default, the input size (“
      <em>–genomeSize</em>”) is set to 0.012 Mb and the minimum coverage depth (“
      <em>–minCoverage</em>”) is set to 10. If any input file is found to be corrupt, its progression in the workflow is aborted.
     </p>
     <p id="p-78">In the FastQC and Trimmomatic module, FastQC (
      <kbd>
       <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>
      </kbd>) is run with the parameters “
      <em>–extract –nogroup –format fastq</em>”. FastQC will inform Trimmomatic [
      <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>] on how many bases to trim from the 3’and 5’ ends of the raw reads. By default, Trimmomatic uses the default set of Illumina adapters provided with the workflow but this behaviour can be overwritten with the “
      <em>–adapters</em>” parameter. The additional Trimmomatic parameters “
      <em>–trimSlidingWindow</em>”, “
      <em>–trimLeading</em>”, “
      <em>–trimTrailing</em>” and “
      <em>–trimMinLength</em>” can all be set to different values.
     </p>
     <p id="p-79">The removal of low complexity sequences is done with PrinSeq [
      <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>] using a custom parameter (“
      <em>–pattern</em>”), which by default is set to the value “A 50%; T 50%; N 50%”, removing sequences whose content is at least half composed of a polymeric sequence (A, T or N).
     </p>
     <p id="p-80">To retrieve the reads that map to the DENV reference database, Bowtie2 [
      <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">13</a>] is run with default parameters with the DENV mapping database as a reference. The reads and their mates that map to the reference are retrieved with “
      <em>samtools view -buh -F 12</em>” and “
      <em>samtools fastq</em>” commands. The DENV mapping database can be altered with the “
      <em>–reference</em>” parameter, or alternatively, a Bowtie2 index can be provided with the “
      <em>–index</em>” parameter. This allows for the workflow to work with other databases obtained through public and owned DENV genomes. The coverage estimation step is performed on the retrieved DENV reads with the same parameters are the first estimation (“—genomeSize=0.012” and “—minCoverage=10”).
     </p>
     <p id="p-81">In the assembly process, the retrieved DENV reads are firstly assembled with SPAdes Genome Assembler [
      <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>] with the options “
      <em>–careful –only-assembler –cov-cutoff</em> “. The coverage cutoff if dictated by the “
      <em>–spadesMinCoverage</em>” and “
      <em>–spadesMinKmerCoverage</em>” parameters, set to 2 by default. If the assembly with SPAdes fails to produce a contig equal or greater than the value defined in the “
      <em>–minimumContigSize</em>” parameter (default of 10000), the data is re-assembled with the MEGAHIT assembler [
      <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>] with default parameters. By default the k-mers to be used in the assembly in both tools (“
      <em>–spadesKmers</em>” and “
      <em>–megahitKmers</em>”) are automatically determined depending on the read size. If the maximum read length is equal or greater than 175 nucleotides, the assembly is done with the k-mers “55, 77, 99, 113, 127”, otherwise the k-mers “21, 33, 55, 67, 77” are used.
     </p>
     <p id="p-82">To correct the assemblies produced, the Pilon tool [
      <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>] is run after mapping the QC’ed reads back to the assembly with Bowtie2 and “
      <em>samtools sort</em>”. This process also verifies the coverage and the number of contigs produced in the assembly. The behaviour can be altered with the parameters “
      <em>–minAssemblyCoverage</em>”, “
      <em>–AMaxContigs</em>” and “
      <em>–genomeSize</em>”, set to “auto”, 1000 and 0.01 Mb by default. The first parameter, when set to ‘auto’, the minimum assembly coverage for each contig required is set to the 1/3 of the assembly mean coverage or to a minimum of 10x. The ratio of contig number per genome MB is calculated based on the genome size estimation for the samples.
     </p>
     <p id="p-83">The contigs larger than the value defined in the “
      <em>–size</em>” parameter (default of 10000 nucleotides) are considered to be complete CDSs and follow the rest to the workflow independently. If no complete CDS is recovered, the QC’ed read data is passed to the mapping to module that does the DENV typing database and consensus generation.
     </p>
     <p id="p-84">The serotyping and genotyping is performed with the Seq_Typing tool [
      <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>] with the command “
      <em>seq_typing.py assembly</em>”or “
      <em>seq_typing.py reads</em>”, using as reference the provided curated DENV typing database. It is possible to retrieve the genomes of the closest references and include them in the downstream analysis by changing the “–get_reference” option to “true”. By default this is not included in the analysis.
     </p>
     <p id="p-85">The CDSs, and the reference sequences if requested, are aligned with the MAFFT tool [
      <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20</a>] with the options “
      <em>–adjustdirection –auto</em>” and a maximum likelihood phylogenetic tree is obtained with the RaXML tool [
      <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">21</a>] with the options “
      <em>-p 12345 -f -a</em>”. Additionally and by default, the substitution model (“
      <em>–substitutionModel</em>”) is set to “GTRGAMMA”, the bootstrap is set to 500 (“
      <em>–bootstrap</em>”) and the seed to “12345” (“
      <em>–seedNumber</em>”).
     </p>
    </div>
    <div id="sec-19" class="subsection">
     <h3>9.3 Shotgun Metagenomics Sequencing Data</h3>
     <p id="p-86">Samples of plasma (n=9) and serum samples (n=13) from confirmed dengue symptomatic patients were collected in Venezuela between 2010-2015 (Table S3) (see Availability of supporting data and materials). DENV positivity was confirmed by either RT-qPCR [
      <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>] or nested RT-PCR [
      <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>].
     </p>
     <p id="p-87">As a positive control sample, the supernatant of a viral culture containing DENV-2 strain 16681 was used. The negative control sample consisted of DNA- and RNA-free water (Sigma-Aldrich, St. Louis, MO, USA).</p>
     <p id="p-88">A spiked sample was produced consisting of a mixture of four 5 
      <em>μ</em>l of cDNA isolated from clinical samples including all DENV serotypes (DENV-1 to -4). The viral cDNA for these samples was not in equal concentration and the viral copy number in the clinical samples was assessed by RT-PCR [
      <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>]. The results were as follow: DENV-2 with 1070000 copies/
      <em>μ</em>l, DENV-1 with 117830 copies/
      <em>μ</em>l, DENV-3 with 44300 copies/
      <em>μ</em>l and DENV-4 with 6600 copies/
      <em>μ</em>l.
     </p>
     <p id="p-89">The cDNA libraries were generated using either the NEBNext
      <sup>®</sup> RNA First and Second strand modules and the Nextera XT DNA library preparation kit (NXT), or the TruSeq RNA V2 library preparation kit (TS). The libraries were sequenced in MiSeq and NextSeq instruments using 300-cycles v2 paired-end cartridges.
     </p>
     <p id="p-90">The DEN-IM workflow was executed with the raw sequencing data using the default parameters and resources in an HPC cluster with 300 Cores/600 Threads of Processing Power and 3 TB RAM divided through 15 computational nodes, 9 with 254 GB Ram and 6 with 126GB RAM.</p>
    </div>
    <div id="sec-20" class="subsection">
     <h3>9.4 Targeted Metagenomics Sequencing Data</h3>
     <p id="p-91">The accession numbers for the 106 DENV-3 amplicon sequencing paired-end short-read datasets are available under BioProject PRJNA394021. The list of Run Accession IDs were obtained with NCBI’s RunSelector and the raw data was downloaded with the GetSeqENA tool (
      <kbd>
       <a href="https://github.com/B-UMMI/getSeqENA">https://github.com/B-UMMI/getSeqENA</a>
      </kbd>).
     </p>
     <p id="p-92">The DEN-IM workflow was executed with the raw sequencing data with default parameters and resources in the same HPC cluster as the shotgun metagenomics dataset.</p>
    </div>
   </div>
   <div class="section" id="sec-21">
    <h2 class="">10 Availability of source code and requirements</h2>
    <p>Lists the following: </p>
    <ul class="list-unord " id="list-2">
     <li id="list-item-5">
      <p id="p-94">Project name: DEN-IM</p>
     </li>
     <li id="list-item-6">
      <p id="p-95">Project home page: 
       <kbd>
        <a href="https://github.com/B-UMMI/DEN-IM">https://github.com/B-UMMI/DEN-IM</a>
       </kbd>
      </p>
     </li>
     <li id="list-item-7">
      <p id="p-96">Operating system(s): UNIX-like systems.</p>
     </li>
     <li id="list-item-8">
      <p id="p-97">Programming language: Nextflow, Python, Bash</p>
     </li>
     <li id="list-item-9">
      <p id="p-98">Other requirements: Java version 8 or highest. Docker/Singularity/Shifter</p>
     </li>
     <li id="list-item-10">
      <p id="p-99">License: GNU GPL v3</p>
     </li>
     <li id="list-item-11">
      <p id="p-100">Documentation and tutorials: 
       <kbd>
        <a href="https://github.com/B-UMMI/DEN-IM/wiki">https://github.com/B-UMMI/DEN-IM/wiki</a>
       </kbd>
      </p>
     </li>
    </ul>
   </div>
   <div class="section" id="sec-22">
    <h2 class="">11 Availability of supporting data and materials</h2>
    <p id="p-101">The 106 DENV-3 targeted metagenomics sequencing paired-end short-read datasets are available under BioProject PRJNA394021. The 25 shotgun metagenomics dataset is available under Bioproject PRJNA474413 The accession number for all the samples in the shotgun metagenomics dataset are available in the supplemental material (Table S3).</p>
   </div>
   <div class="section" id="sec-23">
    <h2 class="">12 Declarations</h2>
    <div id="sec-24" class="subsection">
     <h3>12.1 List of abbreviations</h3>
     <dl id="def-list-1">
      <dt id="def-item-1">DENV
       <span class="def-item-dt-sep"/>
      </dt>
      <dd>Dengue Virus
       <span class="def-item-dd-sep"/>
      </dd>
      <dt id="def-item-2">CDS
       <span class="def-item-dt-sep"/>
      </dt>
      <dd>Coding Sequence
       <span class="def-item-dd-sep"/>
      </dd>
      <dt id="def-item-3">ORF
       <span class="def-item-dt-sep"/>
      </dt>
      <dd>Open Reading Frame
       <span class="def-item-dd-sep"/>
      </dd>
      <dt id="def-item-4">NCR
       <span class="def-item-dt-sep"/>
      </dt>
      <dd>Non-Coding Region
       <span class="def-item-dd-sep"/>
      </dd>
      <dt id="def-item-5">HPC
       <span class="def-item-dt-sep"/>
      </dt>
      <dd>High-Performance Computing
       <span class="def-item-dd-sep"/>
      </dd>
      <dt id="def-item-6">HTS
       <span class="def-item-dt-sep"/>
      </dt>
      <dd>High Throughput Sequencing
       <span class="def-item-dd-sep"/>
      </dd>
      <dt id="def-item-7">QC
       <span class="def-item-dt-sep"/>
      </dt>
      <dd>Quality Control</dd>
     </dl>
    </div>
    <div id="sec-25" class="subsection">
     <h3>12.2 Ethical Approval (optional)</h3>
     <p id="p-109">This study followed international standards for the ethical conduct of research involving human subjects. Data and sample collection was carried out within the DENVEN and IDAMS (International Research Consortium on Dengue Risk Assessment, Management and Surveillance) projects. The study was approved by the Ethics Review Committee of the Biomedical Research Institute, Carabobo University (Aval Bioetico #CBIIB(UC)-014 and CBIIB-(UC)-2013-1), Maracay, Venezuela; the Ethics, Bioethics and Biodiversity Committee (CEBioBio) of the National Foundation for Science, Technology and Innovation (FONACIT) of the Ministry of Science, Technology and Innovation, Caracas, Venezuela; the regional Health authorities of Aragua state (CORPOSALUD Aragua) and Carabobo State (INSALUD); and by the Ethics Committee of the Medical Faculty of Heidelberg University and the Oxford University Tropical Research Ethics Committee.</p>
    </div>
    <div id="sec-26" class="subsection">
     <h3>12.3 Consent for publication</h3>
     <p id="p-110">Not applicable.</p>
    </div>
    <div id="sec-27" class="subsection">
     <h3>12.4 Competing Interests</h3>
     <p id="p-111">The authors declare that they have no competing interests.</p>
    </div>
    <div id="sec-28" class="subsection">
     <h3>12.5 Funding</h3>
     <p id="p-112">C.I.M. was supported by the Fundação para a Ciência e Tecnologia (grant SFRH/BD/129483/2017). Erley Lizarazo received the Abel Tasman Talent Program grant from the UMCG, University of Groningen, Groningen, The Netherlands. This work was partly supported by the ONEIDA project (LISBOA-01-0145-FEDER-016417) co-funded by FEEI-Fundos Europeus Estruturais e de Investimento from Programa Operacional Regional Lisboa 2020 and by national funds from FCT-Fundação para a Ciência e a Tecnologia and UID/BIM/50005/2019, and by UID/BIM/50005/2019, project funded by Fundação para a Ciência e a Tecnologia (FCT)/ Ministério da Ciência, Tecnologia e Ensino Superior (MCTES) through Fundos do Orçamento de Estado.</p>
    </div>
    <div id="sec-29" class="subsection">
     <h3>12.6 Author’s Contributions</h3>
     <p id="p-113">C.I.M., E.L., N.C., M.R., J.A.C. and J.W.A.R. designed the workflow. C.I.M implemented and optimised the workflow, created the Docker containers, and wrote the manuscript. M.P.M. implemented the DENV genotyping module in the workflow and D.N.S. contributed to the development of DEN-IM’s HTML report. E.L., A. T., and N.C. provided the shotgun metagenomics data used to test and validate the workflow and wrote the manuscript. A.T., N.C., M.R., J.A.C. and J.W.A.R. critically revised the article. All authors read, commented on, and approved the final manuscript.</p>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">13 Acknowledgements</h2>
    <p id="p-114">The authors would like to thank Tiago F. Jesus and Bruno Ribeiro-Goncalves for their invaluable help with the Nextflow implementation. We would also like to thank Erwin C. Raangs from the UMCG for his assistance in the sequencing of the shotgun metagenomics dataset. Additionally, the authors thank Lize Cuypers, Krystof Theys, Pieter Libin and Gilberto Santiago for their assistance in DENV nomenclature and classification. This work was done in collaboration with the ESCMID Study Group on Molecular and Genomic Diagnostics (ESGMD), Basel, Switzerland.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">World Health Organization</span>, “
         <span class="cit-article-title">Dengue: guidelines for diagnosis, treatment, prevention, and control</span>,” 
         <abbr class="cit-jnl-abbrev">Special Programme for Research and Training in Tropical Diseases</abbr>, pp. 
         <span class="cit-fpage">x</span>, 
         <span class="cit-vol">147</span>, 
         <span class="cit-pub-date">2009</span>. [Online]. Available: 
         <a href="http://whqlibdoc.who.int/publications/2009/9789241547871_eng.pdf">http://whqlibdoc.who.int/publications/2009/9789241547871{_}eng.pdf</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSpecial%2BProgramme%2Bfor%2BResearch%2Band%2BTraining%2Bin%2BTropical%2BDiseases%26rft.volume%253D147%26rft.spage%253Dx%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.2" data-doi="10.1016/j.cell.2015.07.005">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">M. S.</span>
          <span class="cit-name-surname">Diamond</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T. C.</span>
          <span class="cit-name-surname">Pierson</span>
         </span>, “
         <span class="cit-article-title">Molecular Insight into Dengue Virus Pathogenesis and Its Implications for Disease Control</span>,” 
         <abbr class="cit-jnl-abbrev">Cell</abbr>, vol. 
         <span class="cit-vol">162</span>, no. 
         <span class="cit-issue">3</span>, pp. 
         <span class="cit-fpage">488</span>–
         <span class="cit-lpage">492</span>, 
         <span class="cit-pub-date">2015</span>. [Online]. Available: 
         <a href="http://dx.doi.org/10.1016/j.cell.2015.07.005">http://dx.doi.org/10.1016/j.cell.2015.07.005</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D162%26rft.spage%253D488%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.07.005%26rft_id%253Dinfo%253Apmid%252F26232221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.07.005&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26232221&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F06%2F628073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.3" data-doi="10.1038/nature12060">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Bhatt</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P. W.</span>
          <span class="cit-name-surname">Gething</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">O. J.</span>
          <span class="cit-name-surname">Brady</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. P.</span>
          <span class="cit-name-surname">Messina</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A. W.</span>
          <span class="cit-name-surname">Farlow</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C. L.</span>
          <span class="cit-name-surname">Moyes</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. M.</span>
          <span class="cit-name-surname">Drake</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. S.</span>
          <span class="cit-name-surname">Brownstein</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A. G.</span>
          <span class="cit-name-surname">Hoen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">O.</span>
          <span class="cit-name-surname">Sankoh</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. F.</span>
          <span class="cit-name-surname">Myers</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. B.</span>
          <span class="cit-name-surname">George</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Jaenisch</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G. R.</span>
          <span class="cit-name-surname">William Wint</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C. P.</span>
          <span class="cit-name-surname">Simmons</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T. W.</span>
          <span class="cit-name-surname">Scott</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. J.</span>
          <span class="cit-name-surname">Farrar</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. I.</span>
          <span class="cit-name-surname">Hay</span>
         </span>, “
         <span class="cit-article-title">The global distribution and burden of dengue</span>,” 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, vol. 
         <span class="cit-vol">496</span>, no. 
         <span class="cit-issue">7446</span>, pp. 
         <span class="cit-fpage">504</span>–
         <span class="cit-lpage">507</span>, 
         <span class="cit-pub-date">2013</span>. [Online]. Available: 
         <a href="http://dx.doi.org/10.1038/nature12060">http://dx.doi.org/10.1038/nature12060</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D496%26rft.spage%253D504%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12060%26rft_id%253Dinfo%253Apmid%252F23563266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature12060&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23563266&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F06%2F628073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000317984400041&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Lourenço</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Tennant</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N. R.</span>
          <span class="cit-name-surname">Faria</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Walker</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Gupta</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Recker</span>
         </span>, “
         <span class="cit-article-title">Challenges in dengue research: A computational perspective</span>,” 
         <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr>, vol. 
         <span class="cit-vol">11</span>, no. 
         <span class="cit-issue">4</span>, pp. 
         <span class="cit-fpage">516</span>–
         <span class="cit-lpage">533</span>, 
         <span class="cit-month">apr</span>
         <span class="cit-pub-date">2018</span>. [Online]. Available: 
         <a href="http://doi.wiley.com/10.1111/eva.12554">http://doi.wiley.com/10.1111/eva.12554</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D11%26rft.spage%253D516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">K. C.</span>
          <span class="cit-name-surname">Leitmeyer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. W.</span>
          <span class="cit-name-surname">Vaughn</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. M.</span>
          <span class="cit-name-surname">Watts</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Salas</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">I.</span>
          <span class="cit-name-surname">Villalobos</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">de</span>
          <span class="cit-name-surname">Chacon</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Ramos</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Rico-Hesse</span>
         </span>, “
         <span class="cit-article-title">Dengue virus structural differences that correlate with pathogenesis</span>.” 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>, vol. 
         <span class="cit-vol">73</span>, no. 
         <span class="cit-issue">6</span>, pp. 
         <span class="cit-fpage">4738</span>–
         <span class="cit-lpage">47</span>, 
         <span class="cit-month">jun</span>
         <span class="cit-pub-date">1999</span>. [Online]. Available: 
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/10233934">http://www.ncbi.nlm.nih.gov/pubmed/10233934{%}0A</a>
         <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC112516">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC112516</a>
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/10233934">http://www.ncbi.nlm.nih.gov/pubmed/10233934</a>
         <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC112516">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC112516</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLeitmeyer%26rft.auinit1%253DK.%2BC.%26rft.volume%253D73%26rft.issue%253D6%26rft.spage%253D4738%26rft.epage%253D4747%26rft.atitle%253DDengue%2BVirus%2BStructural%2BDifferences%2BThat%2BCorrelate%2Bwith%2BPathogenesis%26rft_id%253Dinfo%253Apmid%252F10233934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjczLzYvNDczOCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzA2LzYyODA3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">N. L.</span>
          <span class="cit-name-surname">Yozwiak</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Skewes-Cox</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. D.</span>
          <span class="cit-name-surname">Stenglein</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Balmaseda</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Harris</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. L.</span>
          <span class="cit-name-surname">DeRisi</span>
         </span>, “
         <span class="cit-article-title">Virus identification in unknown tropical febrile illness cases using deep sequencing</span>,” 
         <abbr class="cit-jnl-abbrev">PLoS Neglected Tropical Diseases</abbr>, vol. 
         <span class="cit-vol">6</span>, no. 
         <span class="cit-issue">2</span>, 
         <span class="cit-pub-date">2012</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">C. K.</span>
          <span class="cit-name-surname">Lee</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C. W.</span>
          <span class="cit-name-surname">Chua</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Chiu</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">E. S.-C.</span>
          <span class="cit-name-surname">Koay</span>
         </span>, “
         <span class="cit-article-title">Clinical use of targeted high-throughput whole-genome sequencing for a dengue virus variant</span>,” 
         <abbr class="cit-jnl-abbrev">Clinical Chemistry and Laboratory Medicine (CCLM)</abbr>, vol. 
         <span class="cit-vol">55</span>, no. 
         <span class="cit-issue">9</span>, p. 
         <span class="cit-fpage">e209</span>, 
         <span class="cit-month">jan</span>
         <span class="cit-pub-date">2017</span>. [Online]. Available: 
         <a href="https://www.degruyter.com/view/j/cclm.2017.55.issue-9/cclm-2016-0660/cclm-2016-0660.xml">https://www.degruyter.com/view/j/cclm.2017.55.issue-9/cclm-2016-0660/cclm-2016-0660.xml</a>
         <a href="http://www.degruyter.com/view/j/cclm.2017.55.issue-9/cclm-2016-0660/cclm-2016-0660.xml">http://www.degruyter.com/view/j/cclm.2017.55.issue-9/cclm-2016-0660/cclm-2016-0660.xml</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DClinical%2BChemistry%2Band%2BLaboratory%2BMedicine%2B%2528CCLM%2529%26rft.volume%253D55%26rft.spage%253D209e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.8" data-doi="10.1186/1471-2180-11-200">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Fatima</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Idrees</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. A.</span>
          <span class="cit-name-surname">Bajwa</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Tahir</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">O.</span>
          <span class="cit-name-surname">Ullah</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. Q.</span>
          <span class="cit-name-surname">Zia</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Hussain</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Akram</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Khubaib</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Afzal</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Munir</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Saleem</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Rauff</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Badar</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Naudhani</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Butt</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Aftab</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Ali</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Ali</span>
         </span>, “
         <span class="cit-article-title">Serotype and genotype analysis of dengue virus by sequencing followed by phylogenetic analysis using samples from three mini outbreaks-2007-2009 in Pakistan</span>,” 
         <abbr class="cit-jnl-abbrev">BMC Microbiology</abbr>, vol. 
         <span class="cit-vol">11</span>, no. 
         <span class="cit-issue">1</span>, p. 
         <span class="cit-fpage">200</span>, 
         <span class="cit-pub-date">2011</span>. [Online]. Available: 
         <a href="http://www.biomedcentral.com/1471-2180/11/200">http://www.biomedcentral.com/1471-2180/11/200</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2BMicrobiology%26rft.stitle%253DBMC%2BMicrobiology%26rft.aulast%253DFatima%26rft.auinit1%253DZ.%26rft.volume%253D11%26rft.spage%253D200%26rft.epage%253D200%26rft.atitle%253DSerotype%2Band%2Bgenotype%2Banalysis%2Bof%2Bdengue%2Bvirus%2Bby%2Bsequencing%2Bfollowed%2Bby%2Bphylogenetic%2Banalysis%2Busing%2Bsamples%2Bfrom%2Bthree%2Bmini%2Boutbreaks-2007-2009%2Bin%2BPakistan.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2180-11-200%26rft_id%253Dinfo%253Apmid%252F21906394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2180-11-200&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21906394&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F06%2F628073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.9" data-doi="10.1038/nbt.3820">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Di Tommaso</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Chatzou</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E. W.</span>
          <span class="cit-name-surname">Floden</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P. P.</span>
          <span class="cit-name-surname">Barja</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Palumbo</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Notredame</span>
         </span>, “
         <span class="cit-article-title">Nextflow enables reproducible computational workflows</span>,” 
         <abbr class="cit-jnl-abbrev">Nature Biotechnology</abbr>, vol. 
         <span class="cit-vol">35</span>, p. 
         <span class="cit-fpage">316</span>, 
         <span class="cit-month">apr</span>
         <span class="cit-pub-date">2017</span>. [Online]. Available: 
         <a href="http://dx.doi.org/10.1038/nbt.3820">http://dx.doi.org/10.1038/nbt.3820</a>
         <a href="http://10.0.4.14/nbt.3820">http://10.0.4.14/nbt.3820</a>
         <a href="https://www.nature.com/articles/nbt.3820#supplementary-information">https://www.nature.com/articles/nbt.3820{#}supplementary-information</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BBiotechnology%26rft.volume%253D35%26rft.spage%253D316%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3820%26rft_id%253Dinfo%253Apmid%252F28398311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nbt.3820&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=28398311&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F06%2F628073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.10" data-doi="10.1093/bioinformatics/btu170">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A. M.</span>
          <span class="cit-name-surname">Bolger</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Lohse</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Usadel</span>
         </span>, “
         <span class="cit-article-title">Trimmomatic: A flexible trimmer for Illumina sequence data</span>,” 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, vol. 
         <span class="cit-vol">30</span>, no. 
         <span class="cit-issue">15</span>, pp. 
         <span class="cit-fpage">2114</span>–
         <span class="cit-lpage">2120</span>, 
         <span class="cit-pub-date">2014</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24695404&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F06%2F628073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000340049100004&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.11" data-doi="10.1101/gr.8.3.175">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Ewing</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Hillier</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. C.</span>
          <span class="cit-name-surname">Wendl</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Green</span>
         </span>, “
         <span class="cit-article-title">Base-Calling of Automated Sequencer Traces Using Phred. I. Accuracy Assessment</span>,” 
         <abbr class="cit-jnl-abbrev">Genome Research</abbr>, vol. 
         <span class="cit-vol">8</span>, no. 
         <span class="cit-issue">3</span>, pp. 
         <span class="cit-fpage">175</span>–
         <span class="cit-lpage">185</span>, 
         <span class="cit-month">mar</span>
         <span class="cit-pub-date">1998</span>. [Online]. Available: 
         <a href="http://genome.cshlp.org/lookup/doi/10.1101/gr.8.3.175">http://genome.cshlp.org/lookup/doi/10.1101/gr.8.3.175</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DEwing%26rft.auinit1%253DB.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D175%26rft.epage%253D185%26rft.atitle%253DBase-Calling%2Bof%2BAutomated%2BSequencer%2BTraces%2BUsing%2BPhred.%2B%2BI.%2BAccuracy%2BAssessment%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.8.3.175%26rft_id%253Dinfo%253Apmid%252F9521921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjgvMy8xNzUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8wNi82MjgwNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.12" data-doi="10.1093/bioinformatics/btr026">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Schmieder</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Edwards</span>
         </span>, “
         <span class="cit-article-title">Quality control and preprocessing of metagenomic datasets</span>,” 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, vol. 
         <span class="cit-vol">27</span>, no. 
         <span class="cit-issue">6</span>, pp. 
         <span class="cit-fpage">863</span>–
         <span class="cit-lpage">864</span>, 
         <span class="cit-pub-date">2011</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr026%26rft_id%253Dinfo%253Apmid%252F21278185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr026&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21278185&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F06%2F628073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000288277300018&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Langmead</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. L.</span>
          <span class="cit-name-surname">Salzberg</span>
         </span>, “
         <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2</span>, “
         <abbr class="cit-jnl-abbrev">Nature Methods</abbr>, vol. 
         <span class="cit-vol">9</span>, no. 
         <span class="cit-issue">4</span>, pp. 
         <span class="cit-fpage">357</span>–
         <span class="cit-lpage">359</span>, 
         <span class="cit-month">apr</span>
         <span class="cit-pub-date">2012</span>. [Online]. Available: 
         <a href="http://www.nature.com/articles/nmeth.1923">http://www.nature.com/articles/nmeth.1923</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D9%26rft.spage%253D357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.14" data-doi="10.1093/bioinformatics/btr509">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Li</span>
         </span>, “
         <span class="cit-article-title">A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data</span>,” 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, vol. 
         <span class="cit-vol">27</span>, no. 
         <span class="cit-issue">21</span>, pp. 
         <span class="cit-fpage">2987</span>–
         <span class="cit-lpage">2993</span>, 
         <span class="cit-pub-date">2011</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr509%26rft_id%253Dinfo%253Apmid%252F21903627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr509&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21903627&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F06%2F628073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000296099300009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.15" data-doi="10.1089/cmb.2012.0021">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Bankevich</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Nurk</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Antipov</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A. A.</span>
          <span class="cit-name-surname">Gurevich</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Dvorkin</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A. S.</span>
          <span class="cit-name-surname">Kulikov</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">V. M.</span>
          <span class="cit-name-surname">Lesin</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. I.</span>
          <span class="cit-name-surname">Nikolenko</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Pham</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A. D.</span>
          <span class="cit-name-surname">Prjibelski</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A. V.</span>
          <span class="cit-name-surname">Pyshkin</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A. V.</span>
          <span class="cit-name-surname">Sirotkin</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Vyahhi</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.</span>
          <span class="cit-name-surname">Tesler</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. A.</span>
          <span class="cit-name-surname">Alekseyev</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">P. A.</span>
          <span class="cit-name-surname">Pevzner</span>
         </span>, “
         <span class="cit-article-title">SPAdes: A New Genome Assembly Algorithm and Its Applications to Single-Cell Sequencing</span>,” 
         <abbr class="cit-jnl-abbrev">Journal of Computational Biology</abbr>, vol. 
         <span class="cit-vol">19</span>, no. 
         <span class="cit-issue">5</span>, pp. 
         <span class="cit-fpage">455</span>–
         <span class="cit-lpage">477</span>, 
         <span class="cit-month">may</span>
         <span class="cit-pub-date">2012</span>. [Online]. Available: 
         <a href="http://online.liebertpub.com/doi/abs/10.1089/cmb.2012.0021">http://online.liebertpub.com/doi/abs/10.1089/cmb.2012.0021</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DBankevich%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D455%26rft.epage%253D477%26rft.atitle%253DSPAdes%253A%2Ba%2Bnew%2Bgenome%2Bassembly%2Balgorithm%2Band%2Bits%2Bapplications%2Bto%2Bsingle-cell%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2012.0021%26rft_id%253Dinfo%253Apmid%252F22506599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1089/cmb.2012.0021&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22506599&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F06%2F628073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.16" data-doi="10.1093/bioinformatics/btv033">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Li</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C. M.</span>
          <span class="cit-name-surname">Liu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Luo</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Sadakane</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T. W.</span>
          <span class="cit-name-surname">Lam</span>
         </span>, “
         <span class="cit-article-title">MEGAHIT: An ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph</span>,” 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, vol. 
         <span class="cit-vol">31</span>, no. 
         <span class="cit-issue">10</span>, pp. 
         <span class="cit-fpage">1674</span>–
         <span class="cit-lpage">1676</span>, 
         <span class="cit-pub-date">2015</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv033%26rft_id%253Dinfo%253Apmid%252F25609793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv033&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25609793&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F06%2F628073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">B. J.</span>
          <span class="cit-name-surname">Walker</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Abeel</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Shea</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Priest</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Abouelliel</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Sakthikumar</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C. A.</span>
          <span class="cit-name-surname">Cuomo</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Q.</span>
          <span class="cit-name-surname">Zeng</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Wortman</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. K.</span>
          <span class="cit-name-surname">Young</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">A. M.</span>
          <span class="cit-name-surname">Earl</span>
         </span>, “
         <span class="cit-article-title">Pilon: An integrated tool for comprehensive microbial variant detection and genome assembly improvement</span>,” 
         <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>, vol. 
         <span class="cit-vol">9</span>, no. 
         <span class="cit-issue">11</span>, 
         <span class="cit-pub-date">2014</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.18" data-doi="10.1007/978-1-4939-6634-9_20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">M. P.</span>
          <span class="cit-name-surname">Machado</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Ribeiro-Gonçalves</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Silva</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Ramirez</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. A.</span>
          <span class="cit-name-surname">Carriço</span>
         </span>, “
         <span class="cit-article-title">Epidemiological Surveillance and Typing Methods to Track Antibiotic Resistant Strains Using High Throughput Sequencing</span>.” 
         <abbr class="cit-jnl-abbrev">Methods in molecular biology (Clifton, N.J.)</abbr>, vol. 
         <span class="cit-vol">1520</span>, pp. 
         <span class="cit-fpage">331</span>–
         <span class="cit-lpage">356</span>, 
         <span class="cit-pub-date">2017</span>. [Online]. Available: 
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/27873262">http://www.ncbi.nlm.nih.gov/pubmed/27873262</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.volume%253D1520%26rft.spage%253D331%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-6634-9_20%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/978-1-4939-6634-9_20&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.19" data-doi="10.1093/nar/25.17.3389">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">S. F.</span>
          <span class="cit-name-surname">Altschul</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T. L.</span>
          <span class="cit-name-surname">Madden</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A. A.</span>
          <span class="cit-name-surname">Schäffer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Zhang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Zhang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Miller</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. J.</span>
          <span class="cit-name-surname">Lipman</span>
         </span>, “
         <span class="cit-article-title">Gapped BLAST and PSI-BLAST: A new generation of protein database search programs</span>,” 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>, vol. 
         <span class="cit-vol">25</span>, no. 
         <span class="cit-issue">17</span>, pp. 
         <span class="cit-fpage">3389</span>–
         <span class="cit-lpage">3402</span>, 
         <span class="cit-pub-date">1997</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/25.17.3389&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=9254694&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F06%2F628073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1997XU79300002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Nakamura</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K. D.</span>
          <span class="cit-name-surname">Yamada</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Tomii</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Katoh</span>
         </span>, “
         <span class="cit-article-title">Parallelization of MAFFT for large-scale multiple sequence alignments</span>,” 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, vol. 
         <span class="cit-vol">34</span>, no. 
         <span class="cit-month">March</span>, pp. 
         <span class="cit-fpage">2490</span>–
         <span class="cit-lpage">2492</span>, 
         <span class="cit-pub-date">2018</span>. [Online]. Available: 
         <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty121/4916099">https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty121/4916099</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D2490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.21" data-doi="10.1093/bioinformatics/btu033">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Stamatakis</span>
         </span>, “
         <span class="cit-article-title">RAxML version 8: A tool for phylogenetic analysis and post-analysis of large phylogenies</span>,” 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, vol. 
         <span class="cit-vol">30</span>, no. 
         <span class="cit-issue">9</span>, pp. 
         <span class="cit-fpage">1312</span>–
         <span class="cit-lpage">1313</span>, 
         <span class="cit-pub-date">2014</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24451623&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F06%2F628073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000336095100024&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.22">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Gerhardt</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Bhimji</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Canon</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Fasel</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Jacobsen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Mustafa</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Porter</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">V.</span>
          <span class="cit-name-surname">Tsulaia</span>
         </span>, “
         <span class="cit-article-title">Shifter: Containers for HPC</span>,” 
         <abbr class="cit-jnl-abbrev">Journal of Physics: Conference Series</abbr>, vol. 
         <span class="cit-vol">898</span>, p. 
         <span class="cit-fpage">082021</span>, 
         <span class="cit-month">oct</span>
         <span class="cit-pub-date">2017</span>. [Online]. Available: 
         <a href="http://stacks.iop.org/1742-6596/898/i=8/a=082021?key=crossref.b7268cc937fc3b29093062a6749fbbbf">http://stacks.iop.org/1742-6596/898/i=8/a=082021?key=crossref.b7268cc937fc3b29093062a6749fbbbf</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BPhysics%253A%2BConference%2BSeries%26rft.volume%253D898%26rft.spage%253D082021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.23" data-doi="10.1371/journal.pone.0179159">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">G. M.</span>
          <span class="cit-name-surname">Kurtzer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">V.</span>
          <span class="cit-name-surname">Sochat</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. W.</span>
          <span class="cit-name-surname">Bauer</span>
         </span>, “
         <span class="cit-article-title">Singularity: Scientific containers for mobility of compute</span>,” 
         <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>, vol. 
         <span class="cit-vol">12</span>, no. 
         <span class="cit-issue">5</span>, pp. 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">20</span>, 
         <span class="cit-pub-date">2017</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D12%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0179159%26rft_id%253Dinfo%253Apmid%252F28957441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0179159&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=28957441&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F06%2F628073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.24" data-doi="10.1016/j.chom.2017.08.003">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Parameswaran</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Wang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. B.</span>
          <span class="cit-name-surname">Trivedi</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Eswarappa</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Montoya</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Balmaseda</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Harris</span>
         </span>, “
         <span class="cit-article-title">Intrahost Selection Pressures Drive Rapid Dengue Virus Microevolution in Acute Human Infections</span>,” 
         <abbr class="cit-jnl-abbrev">Cell Host &amp;; Microbe</abbr>, vol. 
         <span class="cit-vol">22</span>, no. 
         <span class="cit-issue">3</span>, pp. 
         <span class="cit-fpage">400</span>–
         <span class="cit-lpage">410</span>.
         <span class="cit-page-range">e5</span>, 
         <span class="cit-month">sep</span>
         <span class="cit-pub-date">2017</span>. [Online]. Available: 
         <a href="https://doi.org/10.1016/j.chom.2017.08.003">https://doi.org/10.1016/j.chom.2017.08.003</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2B%2526%2BMicrobe%26rft.volume%253D22%26rft.spage%253D400%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2017.08.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.chom.2017.08.003&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">25.</span>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.25" data-doi="10.1186/gb-2014-15-3-r46">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">D. E.</span>
          <span class="cit-name-surname">Wood</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. L.</span>
          <span class="cit-name-surname">Salzberg</span>
         </span>, “
         <span class="cit-article-title">Kraken: ultrafast metagenomic sequence classification using exact alignments</span>,” 
         <abbr class="cit-jnl-abbrev">Genome Biology</abbr>, vol. 
         <span class="cit-vol">15</span>, no. 
         <span class="cit-issue">3</span>, p. 
         <span class="cit-fpage">R46</span>, 
         <span class="cit-month">mar</span>
         <span class="cit-pub-date">2014</span>. [Online]. Available: 
         <a href="https://doi.org/10.1186/gb-2014-15-3-r46">https://doi.org/10.1186/gb-2014-15-3-r46</a>
         <a href="http://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-3-r46">http://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-3-r46</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D15%26rft.spage%253DR46%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-3-r46%26rft_id%253Dinfo%253Apmid%252F24580807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-3-r46&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24580807&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F06%2F628073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">26.</span>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.26">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">P. E. S.</span>
          <span class="cit-name-surname">Marinho</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. B.</span>
          <span class="cit-name-surname">De Oliveira</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T. M. S.</span>
          <span class="cit-name-surname">Candiani</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A. P. C.</span>
          <span class="cit-name-surname">Crispim</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P. P. M.</span>
          <span class="cit-name-surname">Alvarenga</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">F. C. d. S.</span>
          <span class="cit-name-surname">Castro</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. S.</span>
          <span class="cit-name-surname">Abrahao</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Rios</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R. S.</span>
          <span class="cit-name-surname">Coimbra</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">E. G.</span>
          <span class="cit-name-surname">Kroon</span>
         </span>, “
         <span class="cit-article-title">Meningitis associated with simultaneous infection by multiple dengue virus serotypes in children, Brazil</span>,” 
         <abbr class="cit-jnl-abbrev">Emerging Infectious Diseases</abbr>, vol. 
         <span class="cit-vol">23</span>, no. 
         <span class="cit-issue">1</span>, pp. 
         <span class="cit-fpage">115</span>–
         <span class="cit-lpage">118</span>, 
         <span class="cit-pub-date">2017</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerging%2BInfectious%2BDiseases%26rft.volume%253D23%26rft.spage%253D115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">27.</span>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.27">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">M. N.</span>
          <span class="cit-name-surname">Reddy</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Dungdung</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Valliyott</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Pilankatta</span>
         </span>, “
         <span class="cit-article-title">Occurrence of concurrent infections with multiple serotypes of dengue viruses during 2013-2015 in northern Kerala, India</span>,” 
         <abbr class="cit-jnl-abbrev">PeerJ</abbr>, vol. 
         <span class="cit-vol">5</span>, p. 
         <span class="cit-fpage">e2970</span>, 
         <span class="cit-pub-date">2017</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D5%26rft.spage%253De2970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">28.</span>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.28">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Cuypers</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Libin</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Simmonds</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Nowé</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Muñoz-Jordán</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Alcantara</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.-M.</span>
          <span class="cit-name-surname">Vandamme</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.</span>
          <span class="cit-name-surname">Santiago</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Theys</span>
         </span>, “
         <span class="cit-article-title">Time to Harmonize Dengue Nomenclature and Classification</span>,” 
         <abbr class="cit-jnl-abbrev">Viruses</abbr>, vol. 
         <span class="cit-vol">10</span>, no. 
         <span class="cit-issue">10</span>, p. 
         <span class="cit-fpage">569</span>, 
         <span class="cit-month">oct</span>
         <span class="cit-pub-date">2018</span>. [Online]. Available: 
         <a href="http://www.mdpi.com/1999-4915/10/10/569">http://www.mdpi.com/1999-4915/10/10/569</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D10%26rft.spage%253D569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">29.</span>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.29" data-doi="10.3390/v4113209">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">B. E.</span>
          <span class="cit-name-surname">Pickett</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. S.</span>
          <span class="cit-name-surname">Greer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Zhang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Stewart</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Zhou</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.</span>
          <span class="cit-name-surname">Sun</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Gu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Kumar</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Zaremba</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C. N.</span>
          <span class="cit-name-surname">Larsen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Jen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E. B.</span>
          <span class="cit-name-surname">Klem</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R. H.</span>
          <span class="cit-name-surname">Scheuermann</span>
         </span>, “
         <span class="cit-article-title">Virus pathogen Database and Analysis Resource (ViPR): A comprehensive bioinformatics Database and Analysis Resource for the Coronavirus research community</span>,” 
         <abbr class="cit-jnl-abbrev">Viruses</abbr>, vol. 
         <span class="cit-vol">4</span>, no. 
         <span class="cit-issue">11</span>, pp. 
         <span class="cit-fpage">3209</span>–
         <span class="cit-lpage">3226</span>, 
         <span class="cit-pub-date">2012</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DPickett%26rft.auinit1%253DB.%2BE.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D3209%26rft.epage%253D3226%26rft.atitle%253DVirus%2BPathogen%2BDatabase%2Band%2BAnalysis%2BResource%2B%2528ViPR%2529%253A%2B%2BA%2BComprehensive%2BBioinformatics%2BDatabase%2Band%2BAnalysis%2BResource%2Bfor%2Bthe%2BCoronavirus%2BResearch%2BCommunity.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv4113209%26rft_id%253Dinfo%253Apmid%252F23202522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3390/v4113209&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23202522&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F06%2F628073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">30.</span>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.30" data-doi="10.1016/S1473-3099(16)30471-6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Pang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T. K.</span>
          <span class="cit-name-surname">Mak</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. J.</span>
          <span class="cit-name-surname">Gubler</span>
         </span>, “
         <span class="cit-article-title">Prevention and control of dengue—the light at the end of the tunnel</span>,” 
         <abbr class="cit-jnl-abbrev">The Lancet Infectious Diseases</abbr>, vol. 
         <span class="cit-vol">17</span>, no. 
         <span class="cit-issue">3</span>, pp. 
         <span class="cit-fpage">e79</span>–
         <span class="cit-lpage">e87</span>, 
         <span class="cit-pub-date">2017</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%2BInfectious%2BDiseases%26rft.volume%253D17%26rft.spage%253De79%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252816%252930471-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S1473-3099(16)30471-6&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">31.</span>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.31" data-doi="10.1371/journal.pone.0062649">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">C. J.</span>
          <span class="cit-name-surname">Villabona-Arenas</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">P. M. d. A.</span>
          <span class="cit-name-surname">Zanotto</span>
         </span>, “
         <span class="cit-article-title">Worldwide Spread of Dengue Virus Type 1</span>,” 
         <abbr class="cit-jnl-abbrev">PLoS One</abbr>, vol. 
         <span class="cit-vol">8</span>, no. 
         <span class="cit-issue">5</span>, p. 
         <span class="cit-fpage">e62649</span>, 
         <span class="cit-pub-date">2013</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De62649%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0062649%26rft_id%253Dinfo%253Apmid%252F23675416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0062649&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23675416&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F06%2F628073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">32.</span>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-book" id="cit-628073v1.32" data-doi="10.1016/S0065-3527(08)00401-6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Vasilakis</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. C.</span>
          <span class="cit-name-surname">Weaver</span>
         </span>, “
         <span class="cit-chapter-title">Chapter 1 The History and Evolution of Human Dengue Emergence</span>,” in 
         <span class="cit-source">Advances in Virus Research</span>, ser. 
         <span class="cit-publ-name">Advances in Virus Research. Academic Press</span>, 
         <span class="cit-pub-date">2008</span>, vol. 
         <span class="cit-vol">72</span>, pp. 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">76</span>. [Online]. Available: 
         <a href="http://www.sciencedirect.com/science/article/pii/S0065352708004016">http://www.sciencedirect.com/science/article/pii/S0065352708004016</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bvirus%2Bresearch%26rft.stitle%253DAdv%2BVirus%2BRes%26rft.aulast%253DVasilakis%26rft.auinit1%253DN.%26rft.volume%253D72%26rft.spage%253D1%26rft.epage%253D76%26rft.atitle%253DThe%2Bhistory%2Band%2Bevolution%2Bof%2Bhuman%2Bdengue%2Bemergence.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-3527%252808%252900401-6%26rft_id%253Dinfo%253Apmid%252F19081488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0065-3527(08)00401-6&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19081488&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F06%2F628073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000261402300001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">33.</span>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.33" data-doi="10.1093/bioinformatics/btl158">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Li</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Godzik</span>
         </span>, “
         <span class="cit-article-title">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences</span>,” 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, vol. 
         <span class="cit-vol">22</span>, no. 
         <span class="cit-issue">13</span>, pp. 
         <span class="cit-fpage">1658</span>–
         <span class="cit-lpage">1659</span>, 
         <span class="cit-pub-date">2006</span>. [Online]. Available: 
         <a href="https://academic.oup.com/bioinformatics/article/22/13/1658/194225">https://academic.oup.com/bioinformatics/article/22/13/1658/194225</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl158%26rft_id%253Dinfo%253Apmid%252F16731699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl158&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16731699&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F06%2F628073.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000238905700017&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">34.</span>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-web" id="cit-628073v1.34">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">CDC</span>, “
         <span class="cit-article-title">CDC DENV-1-4 Real-Time RT-PCR Assay for Detection and Serotype Identification of Dengue Virus</span>,” pp. 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">53</span>, 
         <span class="cit-pub-date">2013</span>. [Online]. Available: 
         <a href="http://www.cdc.gov/dengue/resources/rt_pcr/CDCPackageInsert.pdf">http://www.cdc.gov/dengue/resources/rt{_}pcr/CDCPackageInsert.pdf</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">35.</span>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-628073v1.35">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Lanciotti</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Calisher</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Gubler</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.</span>
          <span class="cit-name-surname">Chang</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Vorndam</span>
         </span>, “
         <span class="cit-article-title">Rapid detection and typing of dengue viruses from clinical samples by using reverse transcriptase-polymerase chain reaction</span>.” 
         <abbr class="cit-jnl-abbrev">Journal of Clinical Microbiology</abbr>, vol. 
         <span class="cit-vol">30</span>, no. 
         <span class="cit-issue">3</span>, pp. 
         <span class="cit-fpage">545</span>–
         <span class="cit-lpage">551</span>, 
         <span class="cit-pub-date">1992</span>. [Online]. Available: 
         <a href="http://jcm.asm.org/content/30/3/545.short">http://jcm.asm.org/content/30/3/545.short</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft.stitle%253DJ.%2BClin.%2BMicrobiol.%26rft.aulast%253DLanciotti%26rft.auinit1%253DR%2BS%26rft.volume%253D30%26rft.issue%253D3%26rft.spage%253D545%26rft.epage%253D551%26rft.atitle%253DRapid%2Bdetection%2Band%2Btyping%2Bof%2Bdengue%2Bviruses%2Bfrom%2Bclinical%2Bsamples%2Bby%2Busing%2Breverse%2Btranscriptase-polymerase%2Bchain%2Breaction.%26rft_id%253Dinfo%253Apmid%252F1372617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjg6IjMwLzMvNTQ1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMDYvNjI4MDczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
