<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2016/09/28/078113.atom" />
<meta name="HW.pisa" content="biorxiv;078113v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Unique properties of Zika NS2B-NS3pro complexes as decoded by experiments and MD simulations" />
<meta name="DC.Identifier" content="10.1101/078113" />
<meta name="DC.Date" content="2016-09-28" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2016, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Zika virus can be passed from a pregnant woman to her fetus, thus leading to birth defects including more than microcephaly. It has been recently estimated that one-third of the world population will be infected by Zika, but unfortunately no vaccine or medicine is available so far. Zika NS2B-NS3pro is essential for its replication and thus represents an attractive target for drug discovery/design. Here we characterized conformation, catalysis, inhibition and dynamics of linked and unlinked Zika NS2B-NS3pro complexes by both experiments and MD simulations. The results unveil the unique properties of Zika NS2B-NS3pro which are very different from Dengue one. Particularly, CD and NMR studies indicate that unlike Dengue, the C-terminal region of Zika NS2B with a significant sequence variation is highly disordered in the open conformation. Indeed, MD simulations reveal that up to 100 ns, the Dengue NS2B C-terminus constantly has close contacts with its NS3pro domain. By a sharp contrast, the Zika NS2B C-terminus loses the contacts with its NS3pro domain after 10 ns, further forming a short β-sheet characteristic of the closed conformation at 30 ns. Furthermore, we found that a small molecule, previously identified as an active site inhibitor for other flaviviral NS2B-NS3pro, inhibited Zika NS2B-NS3pro potently in an allosteric manner. Our study provides the first insight into the dynamics of Zika NS2B-NS3pro and further deciphers that it is susceptible to allosteric inhibition, which thus bears critical implications for the future development of therapeutic allosteric inhibitors." />
<meta name="DC.Contributor" content="Amrita Roy" />
<meta name="DC.Contributor" content="Liangzhong Lim" />
<meta name="DC.Contributor" content="Shagun Srivastava" />
<meta name="DC.Contributor" content="Jianxing Song" />
<meta name="article:published_time" content="2016-09-28" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Unique properties of Zika NS2B-NS3pro complexes as decoded by experiments and MD simulations" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;ABSTRACT&lt;/h3&gt;
&lt;p&gt;Zika virus can be passed from a pregnant woman to her fetus, thus leading to birth defects including more than microcephaly. It has been recently estimated that one-third of the world population will be infected by Zika, but unfortunately no vaccine or medicine is available so far. Zika NS2B-NS3pro is essential for its replication and thus represents an attractive target for drug discovery/design. Here we characterized conformation, catalysis, inhibition and dynamics of linked and unlinked Zika NS2B-NS3pro complexes by both experiments and MD simulations. The results unveil the unique properties of Zika NS2B-NS3pro which are very different from Dengue one. Particularly, CD and NMR studies indicate that unlike Dengue, the C-terminal region of Zika NS2B with a significant sequence variation is highly disordered in the open conformation. Indeed, MD simulations reveal that up to 100 ns, the Dengue NS2B C-terminus constantly has close contacts with its NS3pro domain. By a sharp contrast, the Zika NS2B C-terminus loses the contacts with its NS3pro domain after 10 ns, further forming a short β-sheet characteristic of the closed conformation at 30 ns. Furthermore, we found that a small molecule, previously identified as an active site inhibitor for other flaviviral NS2B-NS3pro, inhibited Zika NS2B-NS3pro potently in an allosteric manner. Our study provides the first insight into the dynamics of Zika NS2B-NS3pro and further deciphers that it is susceptible to allosteric inhibition, which thus bears critical implications for the future development of therapeutic allosteric inhibitors.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2016/01/01" />
<meta name="citation_mjid" content="biorxiv;078113v1" />
<meta name="citation_id" content="078113v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/078113v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/078113v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/078113v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113.full.pdf" />
<meta name="citation_doi" content="10.1101/078113" />
<meta name="citation_num_pages" content="38" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="078113" />
<meta name="citation_author" content="Amrita Roy" />
<meta name="citation_author_institution" content="From the Department of Biological Sciences, Faculty of Science, National University of Singapore" />
<meta name="citation_author" content="Liangzhong Lim" />
<meta name="citation_author_institution" content="From the Department of Biological Sciences, Faculty of Science, National University of Singapore" />
<meta name="citation_author" content="Shagun Srivastava" />
<meta name="citation_author_institution" content="From the Department of Biological Sciences, Faculty of Science, National University of Singapore" />
<meta name="citation_author" content="Jianxing Song" />
<meta name="citation_author_institution" content="From the Department of Biological Sciences, Faculty of Science, National University of Singapore" />
<meta name="citation_author_email" content="dbssjx@nus.edu.sg" />
<meta name="citation_reference" content="Dick GW, Kitchen SF, Haddow AJ. (1952) Zika virus isolations and serological specificity. Trans R Soc Trop Med Hyg. 46, 509–520." />
<meta name="citation_reference" content="Wikan N, Smith DR. (2016) Zika virus: history of a newly emerging arbovirus. Lancet Infect Dis. 16, e119–126." />
<meta name="citation_reference" content="Baronti C, Piorkowski G, Charrel RN, Boubis L, Leparc-Goffart I, de Lamballerie X. (2014) Complete coding sequence of zika virus from a French polynesia outbreak in 2013. Genome Announc. 2, pii: e00500–14." />
<meta name="citation_reference" content="Bogoch II, Brady OJ, Kraemer MU, German M, Creatore MI, Brent S, Watts AG, Hay SI, Kulkarni MA, Brownstein JS, Khan K. (2016) Potential for Zika virus introduction and transmission in resource-limited countries in Africa and the Asia-Pacific region: a modelling study. Lancet Infect Dis. S1473-3099, 30270–30275." />
<meta name="citation_reference" content="Mlakar J, Korva M, Tul N, Popovic M, Poljsak-Prijatelj M, Mraz J, Kolenc M, Resman Rus K, Vesnaver Vipotnik T, Fabjan Vodusek V et al; (2016) Zika virus associated with microcephaly. N Engl J Med. 374, 951–958" />
<meta name="citation_reference" content="Li C, Xu D, Ye Q, Hong S, Jiang Y, Liu X, Zhang N, Shi L, Qin CF, Xu Z. (2016) Zika Virus Disrupts Neural Progenitor Development and Leads to Microcephaly in Mice. Cell Stem Cell. 19, 120–126." />
<meta name="citation_reference" content="Cugola FR, Fernandes IR, Russo FB, Freitas BC, Dias JL, Guimarães KP, Benazzato C, Almeida N, Pignatari GC, Romero S, Polonio CM, Cunha I, Freitas CL, Brandão WN, Rossato C, Andrade DG, Faria Dde P, Garcez AT, Buchpigel CA, Braconi CT, Mendes E, Sall AA, Zanotto PM, Peron JP, Muotri AR, Beltrão-Braga PC. (2016) The Brazilian Zika virus strain causes birth defects in experimental models. Nature. 534, 267–271." />
<meta name="citation_reference" content="Vogel G. (2016) INFECTIOUS DISEASE. Experts fear Zika’s effects may be even worse than thought. Science. 352, 1375–1376." />
<meta name="citation_reference" content="WHO statement on the first meeting of the International Health Regulations (2005) (IHR 2005) Emergency Committee on Zika virus and observed increase in neurological disorders and neonatal malformations. 2016. www.who.int/mediacentre/news/statements/2016/1st-emergency-committee-zika/en/" />
<meta name="citation_reference" content="Cox BD, Stanton RA, Schinazi RF. (2016) Predicting Zika virus structural biology: Challenges and opportunities for intervention. Antivir Chem Chemother. 24, 118–126." />
<meta name="citation_reference" content="Dejnirattisai W, Supasa P, Wongwiwat W, Rouvinski A, Barba-Spaeth G, Duangchinda T, Sakuntabhai A, Cao-Lormeau VM, Malasit P, Rey FA, Mongkolsapaya J, Screaton GR. (2016) Dengue virus sero-cross-reactivity drives antibody-dependent enhancement of infection with zika virus. Nat Immunol. 17, 1102–1108." />
<meta name="citation_reference" content="Gupta G, Lim L, Song J. (2015) NMR and MD Studies Reveal That the Isolated Dengue NS3 Protease Is an Intrinsically Disordered Chymotrypsin Fold Which Absolutely Requests NS2B for Correct Folding and Functional Dynamics. PLoS One. 10, e0134823." />
<meta name="citation_reference" content="Chambers TJ, Hahn CS, Galler R, Rice CM. (1990) Flavivirus genome organization, expression, and replication. Annu Rev Microbiol. 44, 649–688." />
<meta name="citation_reference" content="Leung D, Schroder K, White H, Fang NX, Stoermer MJ, Abbenante G, Martin JL, Young PR, Fairlie DP. (2001) Activity of recombinant dengue 2 virus NS3 protease in the presence of a truncated NS2B co-factor, small peptide substrates, and inhibitors. J Biol Chem. 276, 45762–45771." />
<meta name="citation_reference" content="Nall TA1, Chappell KJ, Stoermer MJ, Fang NX, Tyndall JD, Young PR, Fairlie DP. (2004) Enzymatic characterization and homology model of a catalytically active recombinant West Nile virus NS3 protease. J Biol Chem. 279, 48535–48542." />
<meta name="citation_reference" content="Iempridee T, Thongphung R, Angsuthanasombat C, Katzenmeier G. (2008) A comparative biochemical analysis of the NS2B(H)-NS3pro protease complex from four dengue virus serotypes. Biochim Biophys Acta. 1780, 989–994." />
<meta name="citation_reference" content="Li J, Lim SP, Beer D, Patel V, Wen D, Tumanut C, Tully DC, Williams JA, Jiricek J, Priestle JP, Harris JL, Vasudevan SG. (2005) Functional profiling of recombinant NS3 proteases from all four serotypes of dengue virus using tetrapeptide and octapeptide substrate libraries. J Biol Chem. 280, 28766–28774." />
<meta name="citation_reference" content="Erbel P, Schiering N, D’Arcy A, Renatus M, Kroemer M, Lim SP, Yin Z, Keller TH, Vasudevan SG, Hommel U. (2006) Structural basis for the activation of flaviviral NS3 proteases from dengue and West Nile virus. Nat Struct Mol Biol. 13, 372–373." />
<meta name="citation_reference" content="Ekonomiuk D, Su XC, Ozawa K, Bodenreider C, Lim SP, Otting G, Huang D, Caflisch A (2009) Flaviviral protease inhibitors identified by fragment-based library docking into a structure generated by molecular dynamics. J Med Chem. 52, 4860–4868." />
<meta name="citation_reference" content="de la Cruz L1, Nguyen TH, Ozawa K, Shin J, Graham B, Huber T, Otting G. (2011) Binding of low molecular weight inhibitors promotes large conformational changes in the dengue virus NS2B-NS3 protease: fold analysis by pseudocontact shifts. J Am Chem Soc. 133, 19205–19215." />
<meta name="citation_reference" content="Noble CG, Shi PY. (2012) Structural biology of dengue virus enzymes: towards rational design of therapeutics. Antiviral Res. 96, 115–126." />
<meta name="citation_reference" content="Luo D, Vasudevan SG, Lescar J. (2015) The flavivirus NS2B-NS3 protease-helicase as a target for antiviral drug development. Antiviral Res. 118, 148–158." />
<meta name="citation_reference" content="Kim YM, Gayen S, Kang C, Joy J, Huang Q, Chen AS, Wee JL, Ang MJ, Lim HA, Hung AW, Li R, Noble CG, Lee le T, Yip A, Wang QY, Chia CS, Hill J, Shi PY, Keller TH. (2013) NMR analysis of a novel enzymatically active unlinked dengue NS2B-NS3 protease complex. J Biol Chem. 288, 12891–12900." />
<meta name="citation_reference" content="Lim L, Wei Y, Lu Y, Song J. (2016) ALS-Causing Mutations Significantly Perturb the Self-Assembly and Interaction with Nucleic Acid of the Intrinsically Disordered Prion-Like Domain of TDP-43. PLoS Biol. 14, e1002338." />
<meta name="citation_reference" content="Lei J, Hansen G, Nitsche C, Klein CD, Zhang L, Hilgenfeld R. (2016) Crystal structure of Zika virus NS2B-NS3 protease in complex with a boronate inhibitor. Science. 353, 503–505." />
<meta name="citation_reference" content="Fiser A, Šali A. (2003) Modeller: generation and refinement of homology-based protein structure models. Methods Enzym. 374, 461–491." />
<meta name="citation_reference" content="Anand K, Ziebuhr J, Wadhwani P, Mesters JR, Hilgenfeld R. (2003) Coronavirus main proteinase (3CLpro) structure: basis for design of anti-SARS drugs. Science 300, 1763–1767." />
<meta name="citation_reference" content="Ma B, Nussinov R. (2013) Structured crowding and its effects on enzyme catalysis. Top Curr Chem. 337, 123–137." />
<meta name="citation_reference" content="Shi J, Han N, Lim L, Lua S, Sivaraman J, Wang L, Mu Y, Song J. (2011) Dynamically-driven inactivation of the catalytic machinery of the SARS 3C-like protease by the N214A mutation on the extra domain. PLoS Comput Biol. 7, e1001084." />
<meta name="citation_reference" content="Lim L, Shi J, Mu Y, Song J. (2014) Dynamically-driven enhancement of the catalytic machinery of the SARS 3C-like protease by the S284-T285-I286/A mutations on the extra domain. PLoS One. 9, e101941." />
<meta name="citation_reference" content="Noble CG, Seh CC, Chao AT, Shi PY. (2012) Ligand-bound structures of the dengue virus protease reveal the active conformation. J Virol. 86, 438–46." />
<meta name="citation_reference" content="Gupta G, Qin H, Song J. (2012) Intrinsically unstructured domain 3 of hepatitis C Virus NS5A forms a “fuzzy complex” with VAPB-MSP domain which carries ALS-causing mutations. PLoS One. 7, e39261." />
<meta name="citation_reference" content="Fan X, Xue B, Dolan PT, LaCount DJ, Kurgan L, Uversky VN. (2014) The intrinsic disorder status of the human hepatitis C virus proteome. Mol Biosyst. 10, 1345–1363." />
<meta name="citation_reference" content="Nussinov R, Ma B. (2012) Protein dynamics and conformational selection in bidirectional signal transduction. BMC Biol. 10, 2." />
<meta name="citation_reference" content="Yildiz M, Ghosh S, Bell JA, Sherman W, Hardy JA. (2013) Allosteric inhibition of the NS2B-NS3 protease from dengue virus. ACS Chem Biol. 8, 2744–2752." />
<meta name="citation_reference" content="Song J, Ni F. (1998) NMR for the design of functional mimetics of protein-protein interactions: one key is in the building of bridges. Biochem Cell Biol. 76, 177–188." />
<meta name="citation_reference" content="Shi J, Wei Z, Song J. (2004) Dissection study on the severe acute respiratory syndrome 3C-like protease reveals the critical role of the extra domain in dimerization of the enzyme: defining the extra domain as a new target for design of highly specific protease inhibitors. J Biol Chem. 279, 24765–24773." />
<meta name="citation_reference" content="Qin H, Lim LZ, Song J. (2015) Dynamic principle for designing antagonistic/agonistic molecules for EphA4 receptor, the only known ALS modifier. ACS Chem Biol. 10, 372–378." />
<meta name="citation_reference" content="Harvey Motulsky and Arthur Christopoulos, Fitting Models to Biological Data using Linear and Nonlinear Regression. A Practical Guide to Curve Fitting., Oxford University Press, New York, 2004. ISBN: 0195171802." />
<meta name="citation_reference" content="Hess B, Kutzner C, Van Der Spoel D, Lindahl E. (2008) Gromacs 4: algorithms for highly efficient, load-balanced, and scalable molecular simulation. J Chem Theory Comput 4, 435–447." />
<meta name="citation_reference" content="Lindorff-Larsen K, Piana S, Palmo K, Maragakis P, Klepeis JL, Dror RO, Shaw DE. (2010) Improved side-chain torsion potentials for the amber ff99sb protein force field. Proteins. 78, 1950–1958." />
<meta name="citation_reference" content="Hess B, Bekker H, Berendsen HJC, Fraaije JG. (1997) LINCS: A linear constraint solver for molecular simulations. J Comput Chem. 18, 1463–1472." />
<meta name="twitter:title" content="Unique properties of Zika NS2B-NS3pro complexes as decoded by experiments and MD simulations" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Zika virus can be passed from a pregnant woman to her fetus, thus leading to birth defects including more than microcephaly. It has been recently estimated that one-third of the world population will be infected by Zika, but unfortunately no vaccine or medicine is available so far. Zika NS2B-NS3pro is essential for its replication and thus represents an attractive target for drug discovery/design. Here we characterized conformation, catalysis, inhibition and dynamics of linked and unlinked Zika NS2B-NS3pro complexes by both experiments and MD simulations. The results unveil the unique properties of Zika NS2B-NS3pro which are very different from Dengue one. Particularly, CD and NMR studies indicate that unlike Dengue, the C-terminal region of Zika NS2B with a significant sequence variation is highly disordered in the open conformation. Indeed, MD simulations reveal that up to 100 ns, the Dengue NS2B C-terminus constantly has close contacts with its NS3pro domain. By a sharp contrast, the Zika NS2B C-terminus loses the contacts with its NS3pro domain after 10 ns, further forming a short β-sheet characteristic of the closed conformation at 30 ns. Furthermore, we found that a small molecule, previously identified as an active site inhibitor for other flaviviral NS2B-NS3pro, inhibited Zika NS2B-NS3pro potently in an allosteric manner. Our study provides the first insight into the dynamics of Zika NS2B-NS3pro and further deciphers that it is susceptible to allosteric inhibition, which thus bears critical implications for the future development of therapeutic allosteric inhibitors." />
<meta name="og-title" property="og:title" content="Unique properties of Zika NS2B-NS3pro complexes as decoded by experiments and MD simulations" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/078113v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Zika virus can be passed from a pregnant woman to her fetus, thus leading to birth defects including more than microcephaly. It has been recently estimated that one-third of the world population will be infected by Zika, but unfortunately no vaccine or medicine is available so far. Zika NS2B-NS3pro is essential for its replication and thus represents an attractive target for drug discovery/design. Here we characterized conformation, catalysis, inhibition and dynamics of linked and unlinked Zika NS2B-NS3pro complexes by both experiments and MD simulations. The results unveil the unique properties of Zika NS2B-NS3pro which are very different from Dengue one. Particularly, CD and NMR studies indicate that unlike Dengue, the C-terminal region of Zika NS2B with a significant sequence variation is highly disordered in the open conformation. Indeed, MD simulations reveal that up to 100 ns, the Dengue NS2B C-terminus constantly has close contacts with its NS3pro domain. By a sharp contrast, the Zika NS2B C-terminus loses the contacts with its NS3pro domain after 10 ns, further forming a short β-sheet characteristic of the closed conformation at 30 ns. Furthermore, we found that a small molecule, previously identified as an active site inhibitor for other flaviviral NS2B-NS3pro, inhibited Zika NS2B-NS3pro potently in an allosteric manner. Our study provides the first insight into the dynamics of Zika NS2B-NS3pro and further deciphers that it is susceptible to allosteric inhibition, which thus bears critical implications for the future development of therapeutic allosteric inhibitors." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2016-09-28" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/078113v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/078113v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/078113v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/078113v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/21585" />
    <title>Unique properties of Zika NS2B-NS3pro complexes as decoded by experiments and MD simulations | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"zuSNTQ2jy_7YwLzGFjhk4gr1wUb_U2rm3nyRXffva20","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"21585","apath":"\/biorxiv\/early\/2016\/09\/28\/078113.atom","pisa":"biorxiv;078113v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;078113v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/078113v1","title":"Unique properties of Zika NS2B-NS3pro complexes as decoded by experiments and MD simulations","identifier":"node\/21585"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+hT\\\\\\\/tqvJ848mshZtJvcqoVp7cfCiHeiSekc9H34j6d\\\\\\\/ROJCnFS9A7SrIAmpDbV9+lp9uc2HeE+NuZOjdqQ5wRZONTadusoSulgOqX6Vo7f5He\\\\\\\/o+y58Kir9uLqI3SRv13CWKiWq7brL3JBiCHhl67NskLWo0bQul8KlNArWPTARg55NLpYRKyosXfw0ClIeZ40HvR8L\\\\\\\/wEAp+ZX16k\\\\\\\/JWqfkezqeVTeeuWGDteGGF99j\\\\\\\/E0tSp+V04lzUNa22e6ifWtPkpsjtlZd4FjRbNIQ6hV6XKsrrm+HwspHfPx+HWvpyXpfybj+NIWxYyYV7HsG1+\\\\\\\/ipPpiG8Y3cV0rtBwWXYMxW+joZTAj3K4rzph7lIVpq6qNOyVhLkatWeb\\\\\\\/b5PHox5WzGPafhdg8IAJCMQDIhnL1L8sVToem59kNP\\\\\\\/yZdI2BQzRmNb\\\\\\\/h9r8cAbk4dsrs58y0nOLpd\\\\\\\/44br43\\\\\\\/na9LfmAIHvnlNnYK6vhPJ15YVFjLTQaHUcSI3FzGjzb\\\\\\\/qmtzxIdV8tMvZuZZrWFvwn7\\\\\\\/takSq2FWap5ZDTAtZNv21KeDtcLQm45Dr2b0PtYUS\\\\\\\/1qiFhvbX8kCgZbHmA305wXSJZzwb38u+a9irxiPfot6Ff8V2CGwfhXCW0yuApsU3wicQCiKGCC0hCZXtbv1HgJ1sGPRsTXY\\\\\\\/pDQL7IRuo9IeJEobN\\\\\\\/O7UeeQX0idymsGNVOACH54qJ4pwK1rWb7nRnBs3EJ5eZdp26IdXogbrbwEFAcIrMgjA1f10RTrTR\\\\\\\/45AAD0Q\\\\\\\/BZIpcI0q0MmmuH50ix7v2+O+tpMuQLEYArcHXjg75ohIoDM\\\\\\\/H3GrEeD5mqlY1tnus6qv9L1JXvxRXO6Pu+KVR9Glitg5RE1SwZ+i3lVJDdQUC3nZyudfjM7k0v++aFgjYy22d1S5Q20QEN14idJJbFOW782Wn0Fk3AF\\\\\\\/XosHRz3x9E0e8SEwy+Lr0oeCMJ+kaNh6Md2nek6sBc0SwKQAh2564jUToa4yJpc47IPcPNR+z8EkhP4tRinN6Wy4yJaxQJpmhl2OcDEL2Y+LB8Dp5L1kdAsM8IgEb8VMV5a6KvN02KSXWuxiIdpWCVU5hXueHiCtkNDdUMGx8LXFCyJ7JQri3WcJLZUvzZZsaAmupcYH4vh1JboiFlT8K8Als9ENPJuZH8pOs7MdJRCfCgXt3GJBXUWCNjTVUICu+d2VjHg0cRcw4nXWNfPR4F9tOth\\\\\\\/Ij8vUqxQUNEynSofRGQiinh4OVrXGPP+gc82R\\\\\\\/LJwV1bcIiaqCwFLYHzMEQTnRE\\\\\\\/MuSWsDsVHiAupnOApnOBZOsnBYE2F\\\\\\\/vjrSWg=\\\u0022,\\\u0022iv\\\u0022:\\\u00224YkvIR6RGXIRXPetxuH1Iw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00227dc25d55fa54093fe4f31ed001c5b038e64995255e7313c9667d2f5f53cfc04c\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/078113v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-21585 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/078113v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-w4-y4wClDHs2v129EiF-Sg8j1JLfkb8cNb-yHm3_Dog" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="21585" id="node21585" data-pisa="biorxiv;078113v1" data-pisa-master="biorxiv;078113" data-apath="/biorxiv/early/2016/09/28/078113.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Unique properties of Zika NS2B-NS3pro complexes as decoded by experiments and MD simulations</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Amrita</span> <span class="nlm-surname">Roy</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Liangzhong</span> <span class="nlm-surname">Lim</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Shagun</span> <span class="nlm-surname">Srivastava</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jianxing</span> <span class="nlm-surname">Song</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/078113 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node21585" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amrita Roy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>From the Department of Biological Sciences, Faculty of Science, National University of Singapore</span>; 10 Kent Ridge Crescent, Singapore 119260</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amrita%2BRoy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roy%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmrita%2BRoy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Liangzhong Lim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>From the Department of Biological Sciences, Faculty of Science, National University of Singapore</span>; 10 Kent Ridge Crescent, Singapore 119260</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liangzhong%2BLim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lim%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALiangzhong%2BLim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shagun Srivastava </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>From the Department of Biological Sciences, Faculty of Science, National University of Singapore</span>; 10 Kent Ridge Crescent, Singapore 119260</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shagun%2BSrivastava%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Srivastava%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShagun%2BSrivastava%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jianxing Song </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>From the Department of Biological Sciences, Faculty of Science, National University of Singapore</span>; 10 Kent Ridge Crescent, Singapore 119260</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianxing%2BSong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Song%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJianxing%2BSong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dbssjx@nus.edu.sg" class="" data-icon-position="" data-hide-link-title="0">dbssjx@nus.edu.sg</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/078113v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:21585" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:21585/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/078113v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:21585" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:21585/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/078113v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:21585" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:21585/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/078113v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:21585" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:21585/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/078113v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:21585" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:21585/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-3">Zika virus can be passed from a pregnant woman to her fetus, thus leading to birth defects including more than microcephaly. It has been recently estimated that one-third of the world population will be infected by Zika, but unfortunately no vaccine or medicine is available so far. Zika NS2B-NS3pro is essential for its replication and thus represents an attractive target for drug discovery/design. Here we characterized conformation, catalysis, inhibition and dynamics of linked and unlinked Zika NS2B-NS3pro complexes by both experiments and MD simulations. The results unveil the unique properties of Zika NS2B-NS3pro which are very different from Dengue one. Particularly, CD and NMR studies indicate that unlike Dengue, the C-terminal region of Zika NS2B with a significant sequence variation is highly disordered in the open conformation. Indeed, MD simulations reveal that up to 100 ns, the Dengue NS2B C-terminus constantly has close contacts with its NS3pro domain. By a sharp contrast, the Zika NS2B C-terminus loses the contacts with its NS3pro domain after 10 ns, further forming a short β-sheet characteristic of the closed conformation at 30 ns. Furthermore, we found that a small molecule, previously identified as an active site inhibitor for other flaviviral NS2B-NS3pro, inhibited Zika NS2B-NS3pro potently in an allosteric manner. Our study provides the first insight into the dynamics of Zika NS2B-NS3pro and further deciphers that it is susceptible to allosteric inhibition, which thus bears critical implications for the future development of therapeutic allosteric inhibitors.</p></div><div class="section" id="sec-1"><h2 class="">INTRODUCTION</h2><p id="p-4">Zika virus was originally isolated from a sentinel rhesus monkey in the Zika Forest of Uganda in 1947 (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>), which is transmitted to humans by Aedes species mosquitoes. Since 2007, large epidemics of Asian genotype Zika virus have been reported around the world (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>). Recently it has been estimated that one-third of the world population will be infected (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>). Most seriously, Zika infection has been found to be associated with serious sequelae such as Guillain-Barré syndrome, and microcephaly in newborn infants of mothers infected with Zika virus during pregnancy (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>-<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>), and consequently WHO has declared a public health emergency for Zika virus (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>). Zika virus represents a significant challenge to the public health of the whole world but unfortunately there is no effective vaccine or other therapy available so far.</p><p id="p-5">Zika virus with a single stranded, positive sense RNA genome of 10.7 kb belongs to the flavivirus genus, which also contains Dengue, yellow fever, West Nile, Japanese encephalitis, and tick-borne encephalitis viruses (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>). Zika virus shares a high degree of sequence and structural homology with other flaviviruses particularly Dengue virus, thus resulting in immunological cross-reactivity (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>). Seriously, the current Zika outbreaks are largely localized within dengue-endemic areas, it is thus proposed that preexisting dengue-induced antibodies may enhance Zika infection by antibody-dependent enhancement (ADE), a factor that makes the vaccine approaches extremely challenging (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>).</p><p id="p-6">Zika genome is translated into a single ~3,500-residue polyprotein, which is further cleaved into 3 structural proteins and 7 non-structural proteins (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>). The correct processing of the polyprotein is essential for replication of all flaviviruses, which requires both host proteases and a viral NS2B-NS3 protease (NS2B-NS3pro) (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>-<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>). As a consequence, the flaviviral NS2B-NS3pro has been well-established to be a key target for developing antiviral drugs (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>-<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>). In the present study, we successfully obtained two enzymatically active Zika NS2B-NS3pro: one with NS2B and NS3pro linked by extensively used (Gly)<sub>4</sub>-Ser-(Gly)<sub>4</sub> sequence and another with NS2B and NS3pro unlinked. Subsequently we characterized their conformations, catalysis, inhibition and dynamics by both experiments and molecular dynamics (MD) simulations. The results reveal that despite sharing a similarity to the Dengue one, Zika NS2B-NS3pro does have unique catalytic and inhibitory properties, which appear to result from its high dynamics. Furthermore, we identified the first allosteric inhibitor for Zika NS2B-NS3pro. Taken together, our study provides the first insight into the unique dynamics of the Zika NS2B-NS3pro, which may have key implications in further design of better protease inhibitors for fighting Zika.</p></div><div class="section" id="sec-2"><h2 class="">RESULTS</h2><div id="sec-3" class="subsection"><h3>Cloning, expression and purification of linked and unlinked Zika NS2B-NS3pro</h3><p id="p-7">Based on the sequence alignment with NS2B and NS3pro of the Dengue serotype 2 we previously characterized (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>), the corresponding Zika sequences were identified for the NS2B (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>) and NS3pro (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1B</a>) of the Asian Zika virus. From synthetic genes with <em>E. coli</em> preferred codons, we amplified and subsequently cloned the DNA fragments into His-tagged expression vectors, which encode the isolated NS2B (48-100) with the transmembrane regions deleted (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1A</a>); as well as isolated NS3 (14–185) (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1B</a>). The expression level of the recombinant NS2B was extremely low, and consequently we needed to grow many liters of <em>E. coli</em> cells to obtain sufficient NS2B for refolding with NS3pro by the same protocol we previously utilized for the Dengue one (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">12</a>). The refolded NS2B-NS3pro with His-tag (column 2 of <a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1C</a>) was cleaved by thrombin covalently linked to beads to remove the His-tag, followed by binding to an excess amount of the Ni<sup>2+</sup>-beads to remove His-tag and uncleaved fusion protein, as well as final purification with FPLC gel-filtration chromatography to obtain unlinked NS2B-NS3pro without His-tag (column 3 of <a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1C</a>). We attempted to refold Zika NS3pro without NS2B and found that NS3pro completely precipitated during refolding, thus indicating that Zika NS3pro also absolutely requests its NS2B cofactor for the correct folding, as previously observed on other flavivirus NS3 proteases (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">12</a>-<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Construction, expression and purification of Zika NS2B-NS3pro complexes. (A) Sequence alignment between NS2B (48-100) of the Dengue and Zika viruses with the transmembrane region removed. The red arrow is used to indicate the region with significant sequence variations. (B) Sequence alignment between NS3pro (14-185) of Dengue and Zika viruses. (C) SDS PAGE of the purification of unlinked Zika NS2B-NS3pro: column 1: molecular weight makers; column 2: unlinked Zika NS2B-NS3pro; column 3: unlinked Zika NS2B-NS3pro with the His-tag removed by the thrombin beads followed by binding to an excess amount of Ni2+-beads. (D) SDS PAGE of the purification of linked Zika NS2B-NS3pro: column 1: molecular weight makers; column 2: linked Zika NS2B-NS3pro; column 3: linked Zika NS2B-NS3pro with the His-tag removed by the thrombin beads followed by binding to an excess amount of Ni2+-beads. (E) SDS PAGE of the purification of unlinked Zika NS2B(48-74)-NS3pro: column 1: molecular weight makers; column 2: unlinked Zika NS2B(48-74)-NS3pro. Due to the small sizes of NS2B(48-100) and NS2B(48-74), they diffused and thus could not be seen in SDS PAGE." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2130291438" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Construction, expression and purification of Zika NS2B-NS3pro complexes.&lt;/span&gt; (A) Sequence alignment between NS2B (48-100) of the Dengue and Zika viruses with the transmembrane region removed. The red arrow is used to indicate the region with significant sequence variations. (B) Sequence alignment between NS3pro (14-185) of Dengue and Zika viruses. (C) SDS PAGE of the purification of unlinked Zika NS2B-NS3pro: column 1: molecular weight makers; column 2: unlinked Zika NS2B-NS3pro; column 3: unlinked Zika NS2B-NS3pro with the His-tag removed by the thrombin beads followed by binding to an excess amount of Ni&lt;sup&gt;2+&lt;/sup&gt;-beads. (D) SDS PAGE of the purification of linked Zika NS2B-NS3pro: column 1: molecular weight makers; column 2: linked Zika NS2B-NS3pro; column 3: linked Zika NS2B-NS3pro with the His-tag removed by the thrombin beads followed by binding to an excess amount of Ni&lt;sup&gt;2+&lt;/sup&gt;-beads. (E) SDS PAGE of the purification of unlinked Zika NS2B(48-74)-NS3pro: column 1: molecular weight makers; column 2: unlinked Zika NS2B(48-74)-NS3pro. Due to the small sizes of NS2B(48-100) and NS2B(48-74), they diffused and thus could not be seen in SDS PAGE.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIGURE 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F1.medium.gif" width="440" height="136"/><noscript><img class="highwire-fragment fragment-image" alt="FIGURE 1." src="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F1.medium.gif" width="440" height="136"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">FIGURE 1.</span> <span class="caption-title">Construction, expression and purification of Zika NS2B-NS3pro complexes.</span><p id="p-8" class="first-child">(A) Sequence alignment between NS2B (48-100) of the Dengue and Zika viruses with the transmembrane region removed. The red arrow is used to indicate the region with significant sequence variations. (B) Sequence alignment between NS3pro (14-185) of Dengue and Zika viruses. (C) SDS PAGE of the purification of unlinked Zika NS2B-NS3pro: column 1: molecular weight makers; column 2: unlinked Zika NS2B-NS3pro; column 3: unlinked Zika NS2B-NS3pro with the His-tag removed by the thrombin beads followed by binding to an excess amount of Ni<sup>2+</sup>-beads. (D) SDS PAGE of the purification of linked Zika NS2B-NS3pro: column 1: molecular weight makers; column 2: linked Zika NS2B-NS3pro; column 3: linked Zika NS2B-NS3pro with the His-tag removed by the thrombin beads followed by binding to an excess amount of Ni<sup>2+</sup>-beads. (E) SDS PAGE of the purification of unlinked Zika NS2B(48-74)-NS3pro: column 1: molecular weight makers; column 2: unlinked Zika NS2B(48-74)-NS3pro. Due to the small sizes of NS2B(48-100) and NS2B(48-74), they diffused and thus could not be seen in SDS PAGE.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">We also constructed a Zika protease with NS2B and NS3pro linked by a (Gly)4-Ser-(Gly)<sub>4</sub> sequence extensively used for functional and structural characterization of flavivirus NS2B-NS3pro complexes (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>-<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>). No linked NS2B-NS3pro protein was detected in the supernatant, but we were able to purify a small amount from the inclusion body by the Ni<sup>2+</sup>-affinity chromatography under the denaturing condition, which was confirmed to be the linked NS2B-NS3pro by sequencing with mass spectrometry and western plot with the His-tag antibody. Linked NS2B-NS3pro was successfully refolded by dialysis without any precipitation (column 2 of <a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1D</a>). Subsequently, linked NS2B-NS3pro without His-tag (column 3 of <a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig. 1D</a>) was obtained by the cleavage with thrombin covalently linked to beads, followed by binding to Ni<sup>2+</sup>-beads and final purification with FPLC gel-filtration chromatography as described above.</p></div><div id="sec-4" class="subsection"><h3>Biophysical characterization</h3><p id="p-10">We acquired <sup>1</sup>H NMR one-dimensional spectra for both linked and unlinked NS2B-NS3pro (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a>). Both spectra have similar up-field peaks manifested, suggesting that both of them are folded (<a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>). However, it is interesting to note that the peaks of the linked complex are much more broad than those of the unlinked, implying that the linkage may introduce µs-ms conformational dynamics, as previously observed on the linked Dengue NS2B-NS3pro (<a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>). Indeed, we attempted to acquire <sup>1</sup>H-<sup>15</sup>N NMR HSQC spectrum of the linked protease, but failed to detect any peaks due to significant peak broadening. By contrast, despite having almost the same sequences, <sup>15</sup>N-labeled Zika NS3pro in the unlinked complex has a well-dispersed HSQC spectrum, indicating that the NS3pro is well-folded, although many HSQC peaks were still too broad to be detectable (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2B</a>). Amazingly, <sup>15</sup>N-labeled Zika NS2B in the unlinked complex has a very narrowly-dispersed HSQC spectrum, in which HSQC peaks of 22 residues are too broad to be detected, and in particular only two out of four Gly residues have HSQC peaks detectable (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2C</a>). This is very different from what was previously observed on Dengue NS2B in the complex, which showed a well-dispersed HSQC spectrum with peaks of most residues detectable (<a id="xref-ref-12-8" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a>). This implies that Zika NS2B is much more dynamic than Dengue one, in which a portion of residues undergoes µs-ms dynamics, thus becoming too broad to be detectable; while the rest is disordered and absent of tight packing, thus resulting in narrowly-dispersed HSQC peaks (<a id="xref-ref-12-9" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Biophysical characterization of Zika NS2B-NS3pro complexes. (A) One-dimensional 1H NMR spectra over -1.2-1.8 ppm of linked (blue) and unlinked (red) Zika NS2B-NS3pro, as well as NS2B(48-74)-NS3pro (green) complexes at a protein concentration of 30 &#xB5;M. Cyan arrows are used to indicate the very up-field peaks. (B) 1H-15N HSQC spectrum of 15N-labeled NS3pro in complex with unlabeled NS2B at a protein concentration of 30 &#xB5;M. (C) 1H-15N HSQC spectrum of 15N-labeled NS2B in complex with unlabeled NS3pro at a protein concentration of 30 &#xB5;M. Pink arrows are used to indicate the HSQC peaks of Trp side chains in NS3pro and one in NS2B. (D) Emission spectra of the intrinsic UV fluorescence of linked (blue) and unlinked (red) Zika NS2B-NS3pro, as well as NS2B(48-74)-NS3pro (green) complexes at a protein concentration of 10 &#xB5;M. (E) Far-UV CD spectra of linked (blue) and unlinked (red) Zika NS2B-NS3pro, as well as NS2B(48-74)-NS3pro (green) complexes at a protein concentration of 10 &#xB5;M, together with that of Dengue NS2B-NS3pro complex previously obtained (black) (12)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2130291438" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Biophysical characterization of Zika NS2B-NS3pro complexes.&lt;/span&gt; (A) One-dimensional &lt;sup&gt;1&lt;/sup&gt;H NMR spectra over -1.2-1.8 ppm of linked (blue) and unlinked (red) Zika NS2B-NS3pro, as well as NS2B(48-74)-NS3pro (green) complexes at a protein concentration of 30 &#xB5;M. Cyan arrows are used to indicate the very up-field peaks. (B) &lt;sup&gt;1&lt;/sup&gt;H-&lt;sup&gt;15&lt;/sup&gt;N HSQC spectrum of &lt;sup&gt;15&lt;/sup&gt;N-labeled NS3pro in complex with unlabeled NS2B at a protein concentration of 30 &#xB5;M. (C) &lt;sup&gt;1&lt;/sup&gt;H-&lt;sup&gt;15&lt;/sup&gt;N HSQC spectrum of &lt;sup&gt;15&lt;/sup&gt;N-labeled NS2B in complex with unlabeled NS3pro at a protein concentration of 30 &#xB5;M. Pink arrows are used to indicate the HSQC peaks of Trp side chains in NS3pro and one in NS2B. (D) Emission spectra of the intrinsic UV fluorescence of linked (blue) and unlinked (red) Zika NS2B-NS3pro, as well as NS2B(48-74)-NS3pro (green) complexes at a protein concentration of 10 &#xB5;M. (E) Far-UV CD spectra of linked (blue) and unlinked (red) Zika NS2B-NS3pro, as well as NS2B(48-74)-NS3pro (green) complexes at a protein concentration of 10 &#xB5;M, together with that of Dengue NS2B-NS3pro complex previously obtained (black) (12).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIGURE 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F2.medium.gif" width="440" height="400"/><noscript><img class="highwire-fragment fragment-image" alt="FIGURE 2." src="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F2.medium.gif" width="440" height="400"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIGURE 2.</span> <span class="caption-title">Biophysical characterization of Zika NS2B-NS3pro complexes.</span><p id="p-11" class="first-child">(A) One-dimensional <sup>1</sup>H NMR spectra over -1.2-1.8 ppm of linked (blue) and unlinked (red) Zika NS2B-NS3pro, as well as NS2B(48-74)-NS3pro (green) complexes at a protein concentration of 30 µM. Cyan arrows are used to indicate the very up-field peaks. (B) <sup>1</sup>H-<sup>15</sup>N HSQC spectrum of <sup>15</sup>N-labeled NS3pro in complex with unlabeled NS2B at a protein concentration of 30 µM. (C) <sup>1</sup>H-<sup>15</sup>N HSQC spectrum of <sup>15</sup>N-labeled NS2B in complex with unlabeled NS3pro at a protein concentration of 30 µM. Pink arrows are used to indicate the HSQC peaks of Trp side chains in NS3pro and one in NS2B. (D) Emission spectra of the intrinsic UV fluorescence of linked (blue) and unlinked (red) Zika NS2B-NS3pro, as well as NS2B(48-74)-NS3pro (green) complexes at a protein concentration of 10 µM. (E) Far-UV CD spectra of linked (blue) and unlinked (red) Zika NS2B-NS3pro, as well as NS2B(48-74)-NS3pro (green) complexes at a protein concentration of 10 µM, together with that of Dengue NS2B-NS3pro complex previously obtained (black) (<a id="xref-ref-12-10" class="xref-bibr" href="#ref-12">12</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">We also collected spectra of the intrinsic UV fluorescence from Trp residues for both linked and unlinked NS2B-NS3pro complexes (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2D</a>), both of which have similar spectra with the emission maxima at ~344 nm, very similar to what were observed on the NS2B-NS3pro complexes of all four Dengue serotypes (348 nm) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>). This suggests that in Zika NS2B-NS3pro, all 4 Trp residues are similarly buried as Dengue ones, as Trp residue in the unfolded proteins has an emission maximum wavelength &gt; 352 nm (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">24</a>). Strikingly, however, both linked and unlinked Zika NS2B-NS3pro complexes have far-UV CD spectra with the maximal negative signal at the wavelength of ~201 nm and is lacking of any positive signal below 200 nm, which is very different from that of unlinked Dengue one of the same length (<a id="xref-ref-12-11" class="xref-bibr" href="#ref-12">12</a>), which has the maximal negative signal at 217 nm and large positive signal at 192 nm (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2E</a>). This strongly suggests that Zika NS2B-NS3pro complexes contain much more disordered regions than the Dengue, completely consistent with the NMR HSQC result of the <sup>15</sup>N-labeled NS2B (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2C</a>). Unfortunately, the low solubility and µs-ms conformational dynamics prevented us from conducting NMR assignments of Zika NS2B-NS3pro by triple-resonance NMR experiments.</p></div><div id="sec-5" class="subsection"><h3>Characterization of the enzymatic catalysis</h3><p id="p-13">We subsequently characterized the enzymatic properties of both linked and unlinked Zika NS2B-NS3pro complexes. First, we found that the activity of both complexes showed a very similar dependence on the pH values, with the optimal pH at ~9.5 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a>), very similar to other flavivirus NS2B-NS3pro linked by (Gly<sub>4</sub>)-Ser-(Gly<sub>4</sub>) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>-<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a>). Amazingly, this is very different from Dengue NS2B-NS3pro, for which the optimal pH significantly switched from basic pH to neutral pH upon separating NS2B from NS3pro (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">23</a>). Second, the catalytic activity reduced significantly upon increasing the concentrations of NaCl (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3B</a>). In the presence of 150 mM NaCl, the catalytic activity is only 13.6% and 8.2% respectively of the linked and unlinked enzymes in 50 mM Tris buffer at pH 8.5.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Characterization of Enzymatic catalysis and inhibition. (A) Enzymatic activities of linked (blue) and unlinked Zika NS2B-NS3pro complexes at different pH values. (B) Enzymatic activities of linked (blue) and unlinked (red) Zika NS2B-NS3pro complexes in 50 mM Tris buffer at pH 8.5 with additional addition of NaCl at 0, 20, 40, 60, 80, 100,125,150, 200, 250 mM. (C) The tracings of fluorescence intensity within 3 min for three different substrates cleaved by linked Zika NS2B-NS3pro complex: Bz-nKRR-AMC, Boc-GRR-AMC and Boc-GKR-AMC; as well as three assay buffers without the protease. Fluorescence intensity is reported in arbitrary units. (D) Fitting curve of IC50 for linked Zika NS2B-NS3pro inhibited by a small molecule inhibitor p-Nitrophenyl-p-guanidino benzoate. (E) The Lineweaver-Burk plot for linked Zika NS2B-NS3pro. (F) Fitting curve of IC50 for unlinked Zika NS2B-NS3pro. (E) The Lineweaver-Burk plot for unlinked Zika NS2B-NS3pro. [S] is the substrate concentration; v is the initial reaction rate. Both curves were generated by the program GraphPad Prism 7.0." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2130291438" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Characterization of Enzymatic catalysis and inhibition.&lt;/span&gt; (A) Enzymatic activities of linked (blue) and unlinked Zika NS2B-NS3pro complexes at different pH values. (B) Enzymatic activities of linked (blue) and unlinked (red) Zika NS2B-NS3pro complexes in 50 mM Tris buffer at pH 8.5 with additional addition of NaCl at 0, 20, 40, 60, 80, 100,125,150, 200, 250 mM. (C) The tracings of fluorescence intensity within 3 min for three different substrates cleaved by linked Zika NS2B-NS3pro complex: Bz-nKRR-AMC, Boc-GRR-AMC and Boc-GKR-AMC; as well as three assay buffers without the protease. Fluorescence intensity is reported in arbitrary units. (D) Fitting curve of IC&lt;sub&gt;50&lt;/sub&gt; for linked Zika NS2B-NS3pro inhibited by a small molecule inhibitor p-Nitrophenyl-p-guanidino benzoate. (E) The Lineweaver-Burk plot for linked Zika NS2B-NS3pro. (F) Fitting curve of IC&lt;sub&gt;50&lt;/sub&gt; for unlinked Zika NS2B-NS3pro. (E) The Lineweaver-Burk plot for unlinked Zika NS2B-NS3pro. [S] is the substrate concentration; v is the initial reaction rate. Both curves were generated by the program GraphPad Prism 7.0.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIGURE 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F3.medium.gif" width="341" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="FIGURE 3." src="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F3.medium.gif" width="341" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIGURE 3.</span> <span class="caption-title">Characterization of Enzymatic catalysis and inhibition.</span><p id="p-14" class="first-child">(A) Enzymatic activities of linked (blue) and unlinked Zika NS2B-NS3pro complexes at different pH values. (B) Enzymatic activities of linked (blue) and unlinked (red) Zika NS2B-NS3pro complexes in 50 mM Tris buffer at pH 8.5 with additional addition of NaCl at 0, 20, 40, 60, 80, 100,125,150, 200, 250 mM. (C) The tracings of fluorescence intensity within 3 min for three different substrates cleaved by linked Zika NS2B-NS3pro complex: Bz-nKRR-AMC, Boc-GRR-AMC and Boc-GKR-AMC; as well as three assay buffers without the protease. Fluorescence intensity is reported in arbitrary units. (D) Fitting curve of IC<sub>50</sub> for linked Zika NS2B-NS3pro inhibited by a small molecule inhibitor p-Nitrophenyl-p-guanidino benzoate. (E) The Lineweaver-Burk plot for linked Zika NS2B-NS3pro. (F) Fitting curve of IC<sub>50</sub> for unlinked Zika NS2B-NS3pro. (E) The Lineweaver-Burk plot for unlinked Zika NS2B-NS3pro. [S] is the substrate concentration; v is the initial reaction rate. Both curves were generated by the program GraphPad Prism 7.0.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">Furthermore, to allow comparison with the kinetic results previously published on profiling the substrate specificity for the (Gly)<sub>4</sub>-Ser-(Gly)<sub>4</sub> linked NS2B-NS3 proteases of all four Dengue serotypes (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>), here we selected the same three substrates, which were also extensively used for other flavivirus NS2B-NS3pro, namely Bz-nKRR-AMC; Boc-GRR-AMC and Boc-GKR-AMC. We also measured the proteolytic activities of our linked and unlinked Zika NS2B-NS3pro complexes in the same buffer as previously published (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>). Interestingly, as shown in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3C</a>, the Zika NS2B-NS3pro efficiently cleaved Bz-nKRR-AMC, but unexpectedly showed very weak activity on other two. This implies that in addition to requesting dibasic residues at the P1 and P2 sites characteristic of the flaviviral NS3 proteases, Zika NS2B-NS3pro appears to have the higher request of a basic residue at P3 site than the Dengue complexes (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a>). However, as our current focus was not on profiling substrate specificity, in the present study, we did not test on more substrates but measured kinetic constants of linked and unlinked Zika complexes on Bz-nKRR-AMC.</p><p id="p-16">As shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, linked and unlinked Zika complexes only have slightly different kinetic constants. On the other hand, the linked Zika complex showed much larger differences from those of Dengue complexes (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">17</a>). For example, linked Zika complex has a Km of 43.38 µM, which is ~7 folds of the Dengue 1 (6.2 µM); ~3.6 folds of the Dengue 2 and Dengue 3 (12 µM); and ~6 folds of the Dengue 4 (7.3 µM) on the same substrate. This implies that the Zika complex has the lower affinity to this substrate than all four Dengue ones. However, the linked Zika NS2B-NS3pro has a kcat (48.89 S<sup>-1</sup>) much larger than those of all four Dengue (0.32, 1.4, 0.61 and 2.8 S<sup>-1</sup> respectively). Consequently, the catalytic efficiency as reflected by kcat/Km (1,128,681 M<sup>-1</sup>S<sup>-1</sup>) is ~3 folds of that of Dengue 4 (380,000 M<sup>-1</sup>S<sup>-1</sup>), the largest among the four Dengue complexes (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">17</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/236889/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/236889/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/236889" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Kinetic parameters of catalysis and inhibition of linked and unlinked Zika NS2B-NS3pro complexes</span><div class="sb-div caption-clear"></div></div></div><p id="p-17">Interestingly, the kinetic constants were just published for a smaller Zika NS2B-NS3pro which was also linked by the same (Gly)<sub>4</sub>-Ser-(Gly)<sub>4</sub> but had almost all disordered regions removed (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>). It has a Km of 18.3 µM and kcat of 44.6 S<sup>-1</sup> respectively. While the kcat values are very similar, the Km value is ~2.4-fold less than our current one. This small difference is likely due to: 1) the difference of substrate as Bz-nKKR-AMC was used in the study (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>); or/and 2) the presence of more unstructured regions in our construct, 3) or/and the difference of the buffers: our buffer is 50 mM Tris pH 8.5 while the buffer in the publication (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">25</a>) is only 10 mM Tris pH 8.5. It is well-established that high salt concentrations reduced the activity of the flaviviral proteases (<a id="xref-ref-12-12" class="xref-bibr" href="#ref-12">12</a>-<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">18</a>). Indeed, we measured the activity of our linked Zika protease in 10 mM Tris pH 8.5, and the catalytic activity is 1.36-time higher.</p></div><div id="sec-6" class="subsection"><h3>Characterization of the enzymatic inhibition</h3><p id="p-18">Previously, a small molecule p-Nitrophenyl-p-guanidino benzoate (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3D</a>) has been identified to specifically bind to the active site of West Nile NS2B-NS3pro by molecular dynamics (MD) simulations, followed by NMR binding confirmation (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>). Later, this molecule has been extensively demonstrated to bind Dengue NS2B-NS3pro in a similar manner by NMR (<a id="xref-ref-12-13" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">23</a>). Here we have measured its inhibitory effect on both linked and unlinked Zika NS2B-NS3pro complexes. Remarkably, it showed strong inhibitory activities, with IC<sub>50</sub> values of 1.30 ± 0.01 (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3D</a>) and 0.56 ± 0.03 (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3F</a>) µM; and <em>K</em>i values of 0.57 ± 0.02 (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3E</a>) and 0.48 ± 0.03 (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig. 3G</a>) µM respectively for linked and unlinked Zika complex (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Most strikingly, as clearly evidenced from Lineweaver-Burk plots for both linked (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Fig. 3E</a>) and unlinked (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Fig. 3G</a>) Zika complexes, the small molecule acts as a non-competitive inhibitor for both forms. We have searched all literatures citing the original paper (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>), but only found IC<sub>50</sub> value of this inhibitor on West Nile NS2B-NS3pro complex (34.2 µM) in the original paper (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">19</a>). Therefore, this inhibitor could inhibit the enzymatic activity of Zika NS2B-NS3pro much more potently than that of West Nile complex. While it remains unclear whether this molecule inhibits other flaviviral NS2B-NS3pro complexes in a competitive or non-competitive manner, our present results clearly reveal that it inhibits Zika NS2B-NS3pro complexes in a non-competitive manner, fundamentally different from a just reported peptidomimetic boronic acid inhibitor which competitively inhibited Zika NS2B-NS3pro (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">25</a>). Our results thus decoded for the first time that Zika NS2B-NS3pro in fact has a pocket, to which small molecules can bind to trigger allosteric inhibition of the enzymatic activity.</p></div><div id="sec-7" class="subsection"><h3>Dynamic behaviors as revealed by MD simulations</h3><p id="p-19">Low solubility and high dynamics of our longer Zika NS2B-NS3pro constructs prevented from further determination of its structure by crystallography or NMR. On the other hand, Zika and Dengue viruses have a high sequence identity for the NS2B (35.19%) and NS3pro (56.67%) respectively, which is sufficient to allow to build up a homology model (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>,<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>). Therefore, with MODELLER software suite (9v10) (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>), we generated a homology model of the Zika NS2B-NS3pro in the “open” form by use of a Dengue crystal structure (PDB code: 2FOM) as a template. This Dengue crystal structure represents the “open” form, which is adopted by the flaviviral NS2B-NS3pro complexes in the absence of any inhibitors (<a id="xref-ref-12-14" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>,<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>). The obtained Zika model only has 0.13 Å of root-mean-square-distance (RMSD) of heavy atoms from the Dengue complex.</p><p id="p-20">Molecular dynamics (MD) simulation represents a powerful tool to gain insights into roles of protein dynamics in the enzymatic catalysis (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>), and we have previously utilized it to study the Dengue NS2B-NS3pro in the “open” form (<a id="xref-ref-12-15" class="xref-bibr" href="#ref-12">12</a>), as well as the dynamically-driven allosteric mechanisms of the SARS 3C-like protease, which also shares the chymotrypsin fold to host the catalytic machinery (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>,<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>). Here with the exact same protocols we previously used for the Dengue (<a id="xref-ref-12-16" class="xref-bibr" href="#ref-12">12</a>), we conducted MD simulations up to 100 ns for Zika and Dengue NS2B-NS3pro complexes in both open and closed forms, with each having four independent simulations.</p><p id="p-21"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig 4A</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">4B</a> show the root-mean-square deviations (RMSD) trajectories averaged over four independent MD simulations respectively for NS3pro and NS2B of both Zika and Dengue complexes in the open form. While the Zika NS2B and NS3pro have the averaged RMSD values of 5.33 ± 0.45 and 1.73 ± 0.12 Å respectively, the Dengue NS2B and NS3pro have 4.23 ± 0.46 and 1.61 ± 0.11 Å respectively. As seen in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4A</a>, Zika NS3pro only has a slightly higher RMSD than Dengue one and interestingly, after 60 ns their average RMSD trajectories become similar. By contrast, Zika NS2B has a constantly larger average RMSD trajectory than Dengue one during the whole 100-ns simulations. This indicates that both Zika NS2B and NS3pro have intrinsic dynamics higher than those of the Dengue, and particularly the Zika NS2B is much more dynamic than the Dengue one.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dynamics of Zika and Dengue NS2B-NS3pro complexes in open forms by MD simulations. RMSD trajectories of the C&#x3B1; atoms (from their positions in the energy minimized structures) averaged over four independent MD simulations for NS3pro (A) and NS2B (B) in the Zika (red) and Dengue (blue) protease complexes in the open conformations. RMSF of the C&#x3B1; atoms averaged over four independent MD simulations for NS3pro (C) and NS2B (D) in the Zika (red) and Dengue (blue) protease complexes in the open conformation. Structure snapshots (one structure for 10-ns interval) of MD simulations for Dengue (E) and Zika (F) NS2B-NS3pro complexes in the open conformations. NS2B was specially colored as &#x3B1;-helix (red), &#x3B2;-sheet (yellow) and loop (green)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2130291438" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Dynamics of Zika and Dengue NS2B-NS3pro complexes in open forms by MD simulations.&lt;/span&gt; RMSD trajectories of the C&#x3B1; atoms (from their positions in the energy minimized structures) averaged over four independent MD simulations for NS3pro (A) and NS2B (B) in the Zika (red) and Dengue (blue) protease complexes in the open conformations. RMSF of the C&#x3B1; atoms averaged over four independent MD simulations for NS3pro (C) and NS2B (D) in the Zika (red) and Dengue (blue) protease complexes in the open conformation. Structure snapshots (one structure for 10-ns interval) of MD simulations for Dengue (E) and Zika (F) NS2B-NS3pro complexes in the open conformations. NS2B was specially colored as &#x3B1;-helix (red), &#x3B2;-sheet (yellow) and loop (green).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIGURE 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F4.medium.gif" width="403" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="FIGURE 4." src="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F4.medium.gif" width="403" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIGURE 4.</span> <span class="caption-title">Dynamics of Zika and Dengue NS2B-NS3pro complexes in open forms by MD simulations.</span><p id="p-22" class="first-child">RMSD trajectories of the Cα atoms (from their positions in the energy minimized structures) averaged over four independent MD simulations for NS3pro (A) and NS2B (B) in the Zika (red) and Dengue (blue) protease complexes in the open conformations. RMSF of the Cα atoms averaged over four independent MD simulations for NS3pro (C) and NS2B (D) in the Zika (red) and Dengue (blue) protease complexes in the open conformation. Structure snapshots (one structure for 10-ns interval) of MD simulations for Dengue (E) and Zika (F) NS2B-NS3pro complexes in the open conformations. NS2B was specially colored as α-helix (red), β-sheet (yellow) and loop (green).</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Furthermore, although the overall patterns of the root-mean-square fluctuations (RMSF) are similar for NS3pro (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig 4C</a>) and NS2B (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig 4D</a>) of Zika and Dengue complexes, the RMSF values averaged over four simulations of the Zika ones are all larger than those of the Dengue. In particular, the Zika NS2B has much larger RMSF over the N-terminal 6 residues C-terminal 22 residues (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig 4D</a>). A close inspection of the structure snapshots reveals that although the Dengue NS2B residues Glu80-Glu90 have large conformational fluctuations, the residues Gln93-Thr94-Leu95 constantly have close contacts with the NS3pro residues Leu31-Gly32-Tyr33-Ser34 in the whole 100-ns simulations (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig 4E</a>), and consequently have small RMRF over this short fragment (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig 4D</a>). By a sharp contrast, the Zika residues Pro93-Met94-Arg95 corresponding to Dengue Gln93-Thr94-Leu95 lose the contact with NS3pro after 10 ns and consequently the Zika NS2B C-region becomes highly exposed (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Fig. 4F</a>).</p><p id="p-24"><a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5A</a> shows the structure snapshots of Dengue NS2B, which highlights the very similar conformations of Gln93-Leu95 at different simulation time points. Most strikingly, the structure snapshots of Zika NS2B (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig 5B</a>) reveal that only after 30 ns, a short β-sheet is formed with two β-strands over residues Val76-Asp79 and Gly82-L86, which is further illustrated by the structure snapshots of Zika NS2B-NS3pro complex (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5C</a>). This is a significant finding as in the closed conformation, this β-sheet further get packed to NS3pro, even with residues Ser80-Gly81-Asp82 directly contacting the inhibitor (<a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">25</a>). Unfortunately, the formation of such packing is expected to occur at least over µs-ms time scale, which still remains challenging to be simulated by MD.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Significant difference of dynamics between Zika and Dengue NS2B in the open conformation by MD simulations. (A) Structure snapshots of NS2B in Dengue NS2B-NS3pro complex in the open conformation in MD simulations at 0, 10, 20, 30, 40, 60, 80, 100 ns. (B) Structure snapshots of NS2B in Zika NS2B-NS3pro complex in the open conformation in MD simulations at 0, 10, 20, 30, 40, 60, 80, 100 ns. (C) Structure snapshots of Zika NS2B-NS3pro complex in the open conformation in MD simulations at 0, 10, 20, 30, 40, 60, 80, 100 ns. NS2B was specially colored as &#x3B1;-helix (cyan), &#x3B2;-sheet (red) and loop (pink) for the one at 0 ns and &#x3B1;-helix (red), &#x3B2;-sheet (yellow) and loop (green) for the rest. (D) Sequence alignment of NS2B (48-100) of Zika and four serotype Dengue viruses. The red arrow is used to indicate the region with significant sequence variations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2130291438" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Significant difference of dynamics between Zika and Dengue NS2B in the open conformation by MD simulations.&lt;/span&gt; (A) Structure snapshots of NS2B in Dengue NS2B-NS3pro complex in the open conformation in MD simulations at 0, 10, 20, 30, 40, 60, 80, 100 ns. (B) Structure snapshots of NS2B in Zika NS2B-NS3pro complex in the open conformation in MD simulations at 0, 10, 20, 30, 40, 60, 80, 100 ns. (C) Structure snapshots of Zika NS2B-NS3pro complex in the open conformation in MD simulations at 0, 10, 20, 30, 40, 60, 80, 100 ns. NS2B was specially colored as &#x3B1;-helix (cyan), &#x3B2;-sheet (red) and loop (pink) for the one at 0 ns and &#x3B1;-helix (red), &#x3B2;-sheet (yellow) and loop (green) for the rest. (D) Sequence alignment of NS2B (48-100) of Zika and four serotype Dengue viruses. The red arrow is used to indicate the region with significant sequence variations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIGURE 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F5.medium.gif" width="440" height="279"/><noscript><img class="highwire-fragment fragment-image" alt="FIGURE 5." src="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F5.medium.gif" width="440" height="279"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIGURE 5.</span> <span class="caption-title">Significant difference of dynamics between Zika and Dengue NS2B in the open conformation by MD simulations.</span><p id="p-25" class="first-child">(A) Structure snapshots of NS2B in Dengue NS2B-NS3pro complex in the open conformation in MD simulations at 0, 10, 20, 30, 40, 60, 80, 100 ns. (B) Structure snapshots of NS2B in Zika NS2B-NS3pro complex in the open conformation in MD simulations at 0, 10, 20, 30, 40, 60, 80, 100 ns. (C) Structure snapshots of Zika NS2B-NS3pro complex in the open conformation in MD simulations at 0, 10, 20, 30, 40, 60, 80, 100 ns. NS2B was specially colored as α-helix (cyan), β-sheet (red) and loop (pink) for the one at 0 ns and α-helix (red), β-sheet (yellow) and loop (green) for the rest. (D) Sequence alignment of NS2B (48-100) of Zika and four serotype Dengue viruses. The red arrow is used to indicate the region with significant sequence variations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">So why do Zika and Dengue NS2B have such radical differences in conformations and dynamics? Examination of NS2B sequences reveals that the Zika NS2B region corresponding to the Dengue NS2B Glu91-Thr96 shows no sequence homology at all (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5D</a>), thus implying that the sequence variations over this region may mainly account for the loss of the contact between the Zika NS2B and NS3pro corresponding to the Dengue NS2B Gln93-Leu95 and NS3pro Leu31-Ser34, thus leading to the high dynamics of Zika NS2B C-region, which is successfully detected by CD and NMR spectroscopy, as well as pinpointed by MD simulations. We also attempted to assess the dynamics of Zika and Dengue NS2B-NS3pro complexes in the closed conformation. To achieve this, we performed MD simulations up to 100 ns on the crystal structures of Zika (5LC0) (<a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">25</a>) and Dengue (3U1I) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>) NS2B-NS3pro complexes with the small molecule inhibitors removed. We remove the small molecule inhibitors because: 1) we aimed to find whether the removal of the small molecules would trigger a difference in dynamic instability for Zika and Dengue complexes. 2) Currently the force-field parameters for simulating small molecules are still not universally established.</p><p id="p-27"><a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig 6A</a> and <a id="xref-fig-6-2" class="xref-fig" href="#F6">6B</a> show the root-mean-square deviations (RMSD) trajectories averaged over four independent MD simulations respectively for NS3pro and NS2B of both Zika and Dengue complexes in the closed form. Interestingly, for the closed conformation without inhibitors, Dengue complex is more dynamic than the Zika one. While the Dengue NS3pro has the averaged RMSD values of 2.12 ± 0.16 Å, slightly higher than that of Zika (1.96 ± 0.22 Å), the Dengue NS2B has the averaged RMSD values of 8.01 ± 0.82 Å, much higher than that of Zika (6.82 ± 0.78 Å). Furthermore, although the overall patterns of the root-mean-square fluctuations (RMSF) are similar for NS3pro (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig 6C</a>) and NS2B (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig 6D</a>) of Zika and Dengue complexes, the RMSF values averaged over four simulations of the Dengue ones are all larger than those of the Zika. In particular, the Dengue NS2B has relatively larger RMSF than Zika one over the C-terminal 7 residues (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Fig 6D</a>). It is very interesting to observe here that although the majority of contacts of the small molecule inhibitors are with NS3pro, their removal triggers much more significant changes of dynamics of NS2B than those of NS3pro for both Zika and Dengue complexes. Again, as the current simulations only reached 100 ns, the short β-sheet still persists and remains packed with NS3pro for both Dengue (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Fig. 6E</a>) and Zika (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Fig. 6F</a>) complexes.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dynamics of Zika and Dengue NS2B-NS3pro complexes in the closed form by MD simulations. RMSD trajectories of the C&#x3B1; atoms (from their positions in the energy minimized structures) averaged over four independent MD simulations for NS3pro (A) and NS2B (B) in the Zika (red) and Dengue (blue) protease complexes in the closed conformation. RMSF of the C&#x3B1; atoms averaged over four independent MD simulations for NS3pro (C) and NS2B (D) in the Zika (red) and Dengue (blue) protease complexes in the closed conformation. Structure snapshots (one structure for 10-ns interval) of MD simulations for Dengue (E) and Zika (F) NS2B-NS3pro complexes in the closed conformation. NS2B was specially colored as &#x3B1;-helix (red), &#x3B2;-sheet (yellow) and loop (green). (G) Proposed diagram showing the conformational exchange between the &#x201C;open&#x201D; and &#x201C;closed&#x201D; forms of the NS2B-NS3pro complex, which is separated by energy barriers of different heights for Zika and Dengue. The spheres are used to indicate the inhibitor molecule." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2130291438" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Dynamics of Zika and Dengue NS2B-NS3pro complexes in the closed form by MD simulations.&lt;/span&gt; RMSD trajectories of the C&#x3B1; atoms (from their positions in the energy minimized structures) averaged over four independent MD simulations for NS3pro (A) and NS2B (B) in the Zika (red) and Dengue (blue) protease complexes in the closed conformation. RMSF of the C&#x3B1; atoms averaged over four independent MD simulations for NS3pro (C) and NS2B (D) in the Zika (red) and Dengue (blue) protease complexes in the closed conformation. Structure snapshots (one structure for 10-ns interval) of MD simulations for Dengue (E) and Zika (F) NS2B-NS3pro complexes in the closed conformation. NS2B was specially colored as &#x3B1;-helix (red), &#x3B2;-sheet (yellow) and loop (green). (G) Proposed diagram showing the conformational exchange between the &#x201C;open&#x201D; and &#x201C;closed&#x201D; forms of the NS2B-NS3pro complex, which is separated by energy barriers of different heights for Zika and Dengue. The spheres are used to indicate the inhibitor molecule.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIGURE 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F6.medium.gif" width="274" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="FIGURE 6." src="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F6.medium.gif" width="274" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/09/28/078113/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIGURE 6.</span> <span class="caption-title">Dynamics of Zika and Dengue NS2B-NS3pro complexes in the closed form by MD simulations.</span><p id="p-28" class="first-child">RMSD trajectories of the Cα atoms (from their positions in the energy minimized structures) averaged over four independent MD simulations for NS3pro (A) and NS2B (B) in the Zika (red) and Dengue (blue) protease complexes in the closed conformation. RMSF of the Cα atoms averaged over four independent MD simulations for NS3pro (C) and NS2B (D) in the Zika (red) and Dengue (blue) protease complexes in the closed conformation. Structure snapshots (one structure for 10-ns interval) of MD simulations for Dengue (E) and Zika (F) NS2B-NS3pro complexes in the closed conformation. NS2B was specially colored as α-helix (red), β-sheet (yellow) and loop (green). (G) Proposed diagram showing the conformational exchange between the “open” and “closed” forms of the NS2B-NS3pro complex, which is separated by energy barriers of different heights for Zika and Dengue. The spheres are used to indicate the inhibitor molecule.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Characterization of Zika NS2B(48-74)-NS3pro complex</h3><p id="p-29">To test the simulation results showing that the high dynamics of Zika NS2B-NS3pro complex detected by CD and NMR is mostly resulting from the C-terminal region of Zika NS2B, we cloned and expressed a His-tagged Zika NS2B(48-74) with the C-region deleted. Despite its low expression level and insolubility, by growing many liters of <em>E. coli</em> cells, we managed to purify a sufficient amount of NS2B(48-74) for refolding with NS3pro. Interestingly, NS2B(48-74) is capable of forming a soluble complex with NS3pro (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig. 1E</a>). However, its NMR peaks are much more broad even than those of linked NS2B-NS3pro (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig. 2A</a>), most likely due to µs-ms conformational dynamics or/and dynamic aggregation. On the other hand, its intrinsic UV fluorescence spectrum indicates that its four Trp residues are similarly buried as linked and unlinked Zika complexes with the full-length NS2B (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2D</a>). Most interestingly, as judged from its CD spectrum which has the maximal negative signal shifted to 207 nm and has positive signal at 191 nm (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Fig. 2E</a>), this complex with the C-region of NS2B deleted indeed contains the less amount of disordered region than both linked and unlinked Zika complexes with the full-length NS2B (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Fig. 2E</a>). However, despite being less disordered, this complex showed no detectable enzymatic activity even with the protease concentration up to 20 µM, suggesting that this complex is completely inactive. Previously, we have generated a truncated Dengue NS2B with the C-terminal residues deleted but the short NS2B lost the capacity in forming a soluble complex with its NS3pro. However, we generated another truncated NS2B with only residues 77-84 deleted, which was designated as NS2B(48-100; Δ77–84). Interestingly, NS2B(48-100; Δ77–84) was able to form a soluble complex with its NS3pro, which appeared to be highly disordered as reflected by its CD spectrum, as well as highly dynamic as judged by its NMR spectrum (<a id="xref-ref-12-17" class="xref-bibr" href="#ref-12">12</a>).</p><p id="p-30">Together, these results clearly indicate that unlike Dengue one, Zika NS2B indeed has a dynamic and disordered C-terminal region even in the enzymatically active complexes with its NS3pro domain. Nevertheless, our results suggest that this C-terminal region is absolutely required for implementing the catalytic actions, implying that the closed conformation might be indeed the enzymatically-active conformation, as previously speculated (<a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">20</a>-<a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>), because the deletion of the C-terminal region completely abolishes the potential of Zika NS2B-NS3pro complex to form the closed conformation. Additionally, despite being highly disordered, the C-region might have other functional roles such as to maintain functional dynamics to facilitate HCV replication (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>,<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>).</p></div></div><div class="section" id="sec-9"><h2 class="">DISCUSSION</h2><p id="p-31">Knowledge of catalysis, structures and dynamics of all conformational states is beneficial to design of inhibitors of high affinity and specificity for enzymes including viral proteases (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a>). This is particular relevant to flaviviral NS2B-NS3 proteases which appear to need a transition from the open (inactive) to closed (active) conformations to achieve their catalytic actions (<a id="xref-ref-20-7" class="xref-bibr" href="#ref-20">20</a>-<a id="xref-ref-23-8" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">31</a>). As well demonstrated (<a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>-<a id="xref-ref-23-9" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-25-7" class="xref-bibr" href="#ref-25">25</a>,<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">31</a>), the “open” (inactive) and “closed” (active) forms of the flaviviral NS2B-NS3 proteases have very similar structures of the catalytic domains (NS3pro) but differ mostly in the structures and dynamics of NS2B: the “open” is characteristic of the highly unstructured and dynamic C-half of NS2B, while the “closed” has the C-half adopting a well-folded β-hairpin which become tightly contacted with the NS3pro chymotrypsin fold (<a id="xref-fig-6-8" class="xref-fig" href="#F6">Fig. 6G</a>). As a consequence, better understanding of their conformational dynamics may provide key clues to design inhibitors by blocking the conversion from the open to closed conformations.</p><p id="p-32">Here, we have successfully obtained both linked and unlinked Zika NS2B-NS3pro complexes, which, despite sharing similarity, do have some unique catalytic features including the lower substrate affinity but higher efficiency than those of the complexes of all four Dengue serotypes. Most strikingly, as reported by CD and NMR HSQC spectroscopy, Zika NS2B appears to be much more dynamic and disordered than Dengue NS2B of the same length as we previously studied (<a id="xref-ref-12-18" class="xref-bibr" href="#ref-12">12</a>). Indeed, in preparation of our current manuscript, a crystal structure was published on Zika NS2B-NS3pro which was linked with (Gly)<sub>4</sub>-Ser-(Gly)<sub>4</sub> but had almost all flexible regions of both NS2B and NS3pro deleted (<a id="xref-ref-25-8" class="xref-bibr" href="#ref-25">25</a>). Interestingly, it was reported that unlike Dengue complexes whose high-quality crystals could be obtained for the “open” form, this small Zika complex could only be crystallized in complex with a tight competitive inhibitor, and consequently the structure represents a “closed” form (<a id="xref-fig-6-9" class="xref-fig" href="#F6">Fig. 6</a>), in which the characteristic β-sheet is formed over Arg73-Asp79 and Gly82-Leu86 of the NS2B C-half and further becomes tightly packed against NS3pro; and even has residues Ser80-Gly81-Asp82 directly contacting the inhibitor (<a id="xref-fig-6-10" class="xref-fig" href="#F6">Fig. 6G</a>).</p><p id="p-33">To obtain high resolution insights into the dynamic behaviors, we have performed 100-ns MD simulations for Zika and Dengue NS2B-NS3pro complexes in both open and closed conformations. The results reveal that while the Dengue NS2B C-terminal residues Gln93-Leu95 constantly have close contacts with the NS3pro domain in all four independent trajectories of 100-ns simulations of the open conformation (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Fig. 4E</a>), the Zika NS2B C-terminal residues lose the close contacts with its NS3pro domain and become highly exposed to bulk solvent only after 10 ns. This is completely consistent with NMR HSQC spectrum of <sup>15</sup>N-labeled Zika NS2B in complex with unlabeled NS3pro (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Fig. 2C</a>), indicating that at least half of the Zika NS2B residues are highly dynamic. This unusually high dynamics appear to result mainly from the large sequence difference over the NS2B C-terminal residues 91-96 between Zika and Dengue (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. 5D</a>). Indeed, a short Zika NS2B with residues 75-100 deleted is sufficient to form a complex with its NS3pro, which has the less disordered structure but is enzymatically inactive, thus not only confirming that the Zika NS2B C-terminal residues is highly disordered, but also supporting the previous proposal that the formation of the closed conformation is essential for the enzymatic catalysis.</p><p id="p-34">Amazingly, a short β-sheet characteristic of the closed conformation is formed only at 30 ns of simulations of Zika NS2B-NS3pro in the open conformation (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Fig. 5</a>). Furthermore, the contact between Dengue NS2B Gln93-Leu95 and its NS3pro only existing in the “open” form is completely incompatible with the structure of the NS2B C-half in the “closed” conformation (<a id="xref-fig-6-11" class="xref-fig" href="#F6">Fig. 6E</a>). As such, the loss of this contact in Zika NS2B-NS3pro in the open conformation implies that the energy barrier separating the “open” and “closed” forms of the Zika complex might be lower than that of the Dengue complex (<a id="xref-fig-6-12" class="xref-fig" href="#F6">Fig. 6G</a>). In other words, according to “conformational selection” theory (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>), the lower energy barrier leads to the more profound conformational exchanges between the “open” and “closed” forms of the Zika complex even in the free state.</p><p id="p-35">Our study also reveals that a small molecule previously established as an active site inhibitor is able to inhibit both linked and unlinked Zika NS2B-NS3pro complexes at high affinity. Most importantly, this molecule acts as an allosteric inhibitor for both Zika NS2B-NS3pro complexes, fundamentally different from a recently reported peptidomimetic boronic acid inhibitor which is a competitive inhibitor (<a id="xref-ref-25-9" class="xref-bibr" href="#ref-25">25</a>). This suggests that Zika NS2B-NS3pro is susceptible to allosteric inhibition. Furthermore, as this allosteric inhibitor also has a structure scaffold very different from the peptidomimetic boronic acid inhibitor, our discovery might open up a new avenue for the future development of allosteric inhibitors, which is highly demanded to achieve therapeutic inhibition of flaviviral NS2B-NS3pro complexes (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>). Currently we are focused on optimizing various conditions which would allow atomic resolution studies on the structures, dynamics and inhibitor design for Zika NS2B-NS3pro by NMR spectroscopy (<a id="xref-ref-12-19" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">19</a>,<a id="xref-ref-20-8" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-23-10" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>-<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>).</p></div><div class="section" id="sec-10"><h2 class="">EXPERIMENTAL PROCEDURES</h2><div id="sec-11" class="subsection"><h3>Plasmid construction</h3><p id="p-36">The identified genes encoding NS3pro (1-185) and NS2B (1-130) from the Asian Zika strain 8375 (GenBank ID: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KU501217.1&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom">KU501217.1</a>) were optimized and synthesized by GenScript (Piscataway, NJ). With designed primers, the genes by GenScript were used as templates for amplifying DNA fragments encoding the isolated NS3 (14–185) and NS2B (48–100) with the transmembrane regions removed, as well as NS2B (48–74) with the C-region further deleted. Furthermore, DNA fragments were designed for linking NS2B (48-100) to NS3 (14–185) by (Gly)<sub>4</sub>-Ser-(Gly)<sub>4</sub> using overlap PCR as previously described (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>). Amplified DNA fragments were subsequently cloned into His-tagged pET28a vector (Novagen). DNA sequences of all constructs were verified by automated DNA sequencing.</p></div><div id="sec-12" class="subsection"><h3>Protein expression and purification</h3><p id="p-37">All pET28a vectors containing different Zika genes were transformed into <em>Escherichia coli</em> BL21 (DE3) Star cell, which was cultured in Luria-Bertani broth containing 25 μg/ml kanamycin at 37 °C until the A600 reached 0.6. Protein expression was induced with 1 mM isopropyl β-D-thiogalactopyranoside (IPTG) for 4 h at 37 °C. The cell pellets were resuspended in cold PBS buffer at pH 7.4, containing 10 mM β-mercaptoethanol and 8M urea and the supernatant containing the recombinant proteins were purified by Ni-NTA affinity column under denaturing condition. Eluted fractions was subjected to dialysis against PBS pH 7.4, 10 mM β-mercaptoethanol buffer at 4 °C overnight to allow the refolding of the protease. The refolded protease was subjected to thrombin cleavage using thrombin-agarose beads from Thrombin CleanCleave<sup>TM</sup> Kit (Sigma-aldrich, St. Louis, MO), followed by binding to an excess amount of Ni-NTA beads to remove His-tag and uncleaved fusion protein, as well as a FPLC purification on a gel filtration column (HiLoad 16/60 Superdex 200). Recombinant protease samples were checked by SDS-PAGE, molecular weights verification with ESI-MS and protein sequencing with time-of-flight-mass spectrometer (Applied Biosystems). Protein concentration was determined by the UV spectroscopic method with 8 M urea (<a id="xref-ref-12-20" class="xref-bibr" href="#ref-12">12</a>).</p></div><div id="sec-13" class="subsection"><h3>Fluorescence, CD and NMR experiments</h3><p id="p-38">Intrinsic UV fluorescence spectra were measured with a Cary Eclipse fluorescence spectrophotometer as we previously described (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">24</a>) with the excitation wavelength at 280 nm. Circular dichroism (CD) experiments were performed on a Jasco J-1500 spectropolarimeter and data from five independent scans were added and averaged (<a id="xref-ref-12-21" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">24</a>). All NMR experiments were acquired on an 800 MHz Bruker Avance spectrometer equipped with pulse field gradient units as described previously (<a id="xref-ref-12-22" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>).</p></div><div id="sec-14" class="subsection"><h3>Enzymatic Activity and Kinetics</h3><p id="p-39">To allow comparison with the enzymatic kinetic parameters reported on the NS2B-NS3pro complexes of four Dengue serotypes (<a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">17</a>), we selected three fluorophore-tagged substrates previously used (<a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">17</a>): namely Bz-Nle-Lys-Arg-Arg-AMC, Boc-Gly-Arg-Arg-AMC and Boc-Gly-Lys-Arg-AMC (Bachem AG, Bubendorf), which were dissolved in dimethyl sulfoxide for preparing stock solutions (100 mM). All enzymatic experiments were performed in triplicate and data are presented as mean ± SD, while IC<sub>50</sub>, Km and <em>K</em>i were obtained by fitting with GraphPad Prism 7.0 (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>).</p><p id="p-40">The pH dependence was measured with a protease concentration of 50 nM and substrate (Bz-nKRR-AMC) concentration of 250 μM at 0.5 pH intervals using the following buffers: 50 mM citrate-phosphate buffer for pH 4-5, 50 mM phosphate buffer for pH 5.5-8, 50 mM Tris-HCl buffer for pH 8.5-9.5, and 50 mM Na-bicarbonate buffer for pH 10-10.5.</p><p id="p-41">For steady state kinetics, we used the exactly the same buffer as a previous one on profiling substrate specificity for the NS2B-NS3pro complexes of all four Dengue serotypes (<a id="xref-ref-17-8" class="xref-bibr" href="#ref-17">17</a>): 50 mM Tris-HCl at pH 8.5, except that no glycerol was added as it would lead to significant NMR signal broadening due to high viscosity. Briefly, Zika protease at 50 nM was incubated with substrates ranging from 10 to 1000 µM in 100 μl assay buffer at 37 °C. Progression of enzymatic reaction was monitored as an increase in fluorescence at λ<sub>ex</sub> of 380 nm and λ<sub>em</sub> of 450 nm. Fluorescence intensity is reported in arbitrary units. Initial fluorescence velocities (relative fluorescence units/sec) were calculated and curves were fitted to the Michaelis-Menten equation by nonlinear regression For IC<sub>50</sub> determination, the protease at 50 nM was preincubated at 37 °C for 30 min with p-Nitrophenyl-p-guanidino benzoate at various concentrations and the reaction was initiated by adding Bz-nKRR-AMC to 250 μM. For <em>K</em>i determination, the assay was performed with different final concentrations of the inhibitor and substrate. The protease at 50 nM was incubated with the inhibitor at different concentrations for 30 min at 37 ºC. Subsequently, the reaction was initiated by addition of the corresponding concentration series of the substrate as described. The <em>K</em>i was obtained by fitting in the non-competitive inhibition mode (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">36</a>).</p></div><div id="sec-15" class="subsection"><h3>Homology modeling and molecular dynamics (MD) simulations</h3><p id="p-42">Protein sequences of the Zika NS2B and NS3pro were aligned with the corresponding sequences of the Dengue NS2B-NS3pro complex (<a id="xref-ref-12-23" class="xref-bibr" href="#ref-12">12</a>) and 30 homology models were built up with MODELLER 9v11 software package (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">26</a>), and the selected model has the lowest discrete optimized protein energy (DOPE). Structural parameters of the Zika homology model and the Dengue NS2B-NS3pro complex (<a id="xref-ref-12-24" class="xref-bibr" href="#ref-12">12</a>) were very similar and thus judged to be suitable for MD simulation studies.</p><p id="p-43">The protocol for the MD simulations of Zika and Dengue NS2B-NS3pro complexes in both open and closed conformations is exactly the same as we previously used for the Dengue NS2B-NS3pro complex in the open conformation (<a id="xref-ref-12-25" class="xref-bibr" href="#ref-12">12</a>). MD simulations of the closed conformation were performed on the crystal structures of Zika (5LC0) (<a id="xref-ref-25-10" class="xref-bibr" href="#ref-25">25</a>) and Dengue (3U1I) (<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">31</a>) NS2B-NS3pro complexes with their small molecule inhibitors removed. Four independent 100-ns MD simulations were conducted on Zika and Dengue NS2B-NS3pro complexes in both open and closed conformations.</p><p id="p-44">The simulation cell is a dodecahedron with a periodicity of 12 Å between the protein and the box walls to ensure protein will not interact with its own periodic images. Water molecules using the TIP3P model are used to solvate the MD system. 8 Na<sup>+</sup> ions were added into the Zika NS2B-NS3pro to make the charge of the respective MD system neutral. The solvated Zika homology NS2B-NS3pro complex was first energy minimized with 1000 steps of steepest descent using GROMACS version 4.5.7 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>) using AMBER all-atom force field (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>). The time step was set as 2 fs. Prior to MD production simulation, two stages of equilibration were done independently for each MD simulation set. First equilibration phase was done in NVT ensemble for 100 ps with the following settings: long-range electrostatic interactions calculated with fast particle-mesh Ewald summation method; a cutoff of 9 Å for electrostatic interactions and a cutoff of 14 Å for van der Waals interactions; the temperature was kept constant at 300 K with Berendsen’s coupling. Second equilibration phase was done in NPT ensemble for 100 ps with the same settings as the first equilibration phase and some additional settings: pressure coupling was held at 1 bar with Parrinello-Rahma coupling; isothermal compressibility was set at 4.5*10 bar. Heavy atoms of proteins were position-contrained using LINCS algorithm (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>). The MD production simulation was done with the same settings with NPT equilibration without the position-contraints on heavy atoms of protein and temperature coupling was done with Nose-Hoover.</p></div></div><div class="section" id="sec-16"><h2 class="">CONFLICT OF INTEREST</h2><p id="p-46">The authors declare that they have no conflicts of interest with the contents of this article.</p></div><div class="section" id="sec-17"><h2 class="">AUTHOR CONTRIBUTIONS</h2><p id="p-47">Conceived the idea: JXS; Performed the experiments: AR LZL SS; Prepare figures and wrote the paper: JXS.</p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGEMENT</h2><p id="p-45">This study is supported by Ministry of Education of Singapore (MOE) Tier 3 Grant R-154-002-580-112; Tier 2 Grant MOE2015-T2-1-111 and National Research Foundation of Singapore (NRF) R-154-002-529-281 to Jianxing Song. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.1" data-doi="10.1016/0035-9203(52)90042-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dick</span>  <span class="cit-name-given-names">GW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kitchen</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haddow</span>  <span class="cit-name-given-names">AJ.</span></span> (<span class="cit-pub-date">1952</span>) <span class="cit-article-title">Zika virus isolations and serological specificity</span>. <abbr class="cit-jnl-abbrev">Trans R Soc Trop Med Hyg</abbr>. <span class="cit-vol">46</span>, <span class="cit-fpage">509</span>–<span class="cit-lpage">520</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans%2BR%2BSoc%2BTrop%2BMed%2BHyg%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0035-9203%252852%252990042-4%26rft_id%253Dinfo%253Apmid%252F12995440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0035-9203(52)90042-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12995440&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.2" data-doi="10.1016/S1473-3099(16)30010-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wikan</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">DR.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Zika virus: history of a newly emerging arbovirus</span>. <abbr class="cit-jnl-abbrev">Lancet Infect Dis</abbr>. <span class="cit-vol">16</span>, <span class="cit-fpage">e119</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2BInfect%2BDis%26rft.volume%253D16%26rft.spage%253De119%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252816%252930010-X%26rft_id%253Dinfo%253Apmid%252F27282424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1473-3099(16)30010-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27282424&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baronti</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Piorkowski</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Charrel</span>  <span class="cit-name-given-names">RN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boubis</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leparc-Goffart</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Lamballerie</span>  <span class="cit-name-given-names">X.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Complete coding sequence of zika virus from a French polynesia outbreak in 2013</span>. <abbr class="cit-jnl-abbrev">Genome Announc</abbr>. <span class="cit-vol">2</span>, pii: <span class="cit-fpage">e00500</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc%26rft.volume%253D2%26rft.spage%253D00500e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bogoch</span>  <span class="cit-name-given-names">II</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brady</span>  <span class="cit-name-given-names">OJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kraemer</span>  <span class="cit-name-given-names">MU</span></span>, <span class="cit-auth"><span class="cit-name-surname">German</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Creatore</span>  <span class="cit-name-given-names">MI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brent</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watts</span>  <span class="cit-name-given-names">AG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hay</span>  <span class="cit-name-given-names">SI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kulkarni</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brownstein</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Potential for Zika virus introduction and transmission in resource-limited countries in Africa and the Asia-Pacific region: a modelling study</span>. <abbr class="cit-jnl-abbrev">Lancet Infect Dis</abbr>. <span class="cit-vol">S1473-3099</span>, <span class="cit-fpage">30270</span>–<span class="cit-lpage">30275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2BInfect%2BDis%26rft.volume%253D14733099%26rft.spage%253D30270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.5" data-doi="10.1056/NEJMoa1600651"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mlakar</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korva</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tul</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Popovic</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poljsak-Prijatelj</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mraz</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kolenc</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Resman Rus</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vesnaver Vipotnik</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fabjan Vodusek</span>  <span class="cit-name-given-names">V</span></span> <span class="cit-etal">et al</span>; (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Zika virus associated with microcephaly</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>. <span class="cit-vol">374</span>, <span class="cit-fpage">951</span>–<span class="cit-lpage">958</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D374%26rft.spage%253D951%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1600651%26rft_id%253Dinfo%253Apmid%252F26862926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1600651&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26862926&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-078113v1.6" data-doi="10.1016/j.stem.2016.04.017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">CF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Zika Virus Disrupts Neural Progenitor Development and Leads to Microcephaly in Mice</span>. <abbr class="cit-jnl-abbrev">Cell Stem Cell</abbr>. <span class="cit-vol">19</span>, <span class="cit-fpage">120</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BStem%2BCell%26rft.volume%253D19%26rft.spage%253D120%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.stem.2016.04.017%26rft_id%253Dinfo%253Apmid%252F27179424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.stem.2016.04.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27179424&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-078113v1.7" data-doi="10.1038/nature18296"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cugola</span>  <span class="cit-name-given-names">FR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fernandes</span>  <span class="cit-name-given-names">IR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Russo</span>  <span class="cit-name-given-names">FB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Freitas</span>  <span class="cit-name-given-names">BC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dias</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guimarães</span>  <span class="cit-name-given-names">KP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benazzato</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pignatari</span>  <span class="cit-name-given-names">GC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Polonio</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cunha</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Freitas</span>  <span class="cit-name-given-names">CL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brandão</span>  <span class="cit-name-given-names">WN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rossato</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andrade</span>  <span class="cit-name-given-names">DG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faria Dde</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garcez</span>  <span class="cit-name-given-names">AT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buchpigel</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Braconi</span>  <span class="cit-name-given-names">CT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mendes</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sall</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zanotto</span>  <span class="cit-name-given-names">PM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peron</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muotri</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beltrão-Braga</span>  <span class="cit-name-given-names">PC.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">The Brazilian Zika virus strain causes birth defects in experimental models</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-vol">534</span>, <span class="cit-fpage">267</span>–<span class="cit-lpage">271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D534%26rft.spage%253D267%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature18296%26rft_id%253Dinfo%253Apmid%252F27279226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature18296&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27279226&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.8" data-doi="10.1126/science.352.6292.1375"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">INFECTIOUS DISEASE. Experts fear Zika’s effects may be even worse than thought</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-vol">352</span>, <span class="cit-fpage">1375</span>–<span class="cit-lpage">1376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DVogel%26rft.auinit1%253DG.%26rft.volume%253D352%26rft.issue%253D6292%26rft.spage%253D1375%26rft.epage%253D1376%26rft.atitle%253DExperts%2Bfear%2BZika%2527s%2Beffects%2Bmay%2Bbe%2Beven%2Bworse%2Bthan%2Bthought%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.352.6292.1375%26rft_id%253Dinfo%253Apmid%252F27313013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTIvNjI5Mi8xMzc1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDkvMjgvMDc4MTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.9"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WHO statement on the first meeting of the International Health Regulations</span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">(IHR 2005) Emergency Committee on Zika virus and observed increase in neurological disorders and neonatal malformations</span>. 2016. <a href="http://www.who.int/mediacentre/news/statements/2016/1st-emergency-committee-zika/en/">www.who.int/mediacentre/news/statements/2016/1st-emergency-committee-zika/en/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">BD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stanton</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schinazi</span>  <span class="cit-name-given-names">RF.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Predicting Zika virus structural biology: Challenges and opportunities for intervention</span>. <abbr class="cit-jnl-abbrev">Antivir Chem Chemother</abbr>. <span class="cit-vol">24</span>, <span class="cit-fpage">118</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntivir%2BChem%2BChemother%26rft.volume%253D24%26rft.spage%253D118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.11" data-doi="10.1038/ni.3515"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dejnirattisai</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Supasa</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wongwiwat</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rouvinski</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barba-Spaeth</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duangchinda</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sakuntabhai</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cao-Lormeau</span>  <span class="cit-name-given-names">VM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Malasit</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rey</span>  <span class="cit-name-given-names">FA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mongkolsapaya</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Screaton</span>  <span class="cit-name-given-names">GR.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Dengue virus sero-cross-reactivity drives antibody-dependent enhancement of infection with zika virus</span>. <abbr class="cit-jnl-abbrev">Nat Immunol</abbr>. <span class="cit-vol">17</span>, <span class="cit-fpage">1102</span>–<span class="cit-lpage">1108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BImmunol%26rft.volume%253D17%26rft.spage%253D1102%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fni.3515%26rft_id%253Dinfo%253Apmid%252F27339099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ni.3515&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27339099&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">NMR and MD Studies Reveal That the Isolated Dengue NS3 Protease Is an Intrinsically Disordered Chymotrypsin Fold Which Absolutely Requests NS2B for Correct Folding and Functional Dynamics</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-vol">10</span>, <span class="cit-fpage">e0134823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0134823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.13" data-doi="10.1146/annurev.mi.44.100190.003245"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chambers</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">CS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Galler</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">CM.</span></span> (<span class="cit-pub-date">1990</span>) <span class="cit-article-title">Flavivirus genome organization, expression, and replication</span>. <abbr class="cit-jnl-abbrev">Annu Rev Microbiol</abbr>. <span class="cit-vol">44</span>, <span class="cit-fpage">649</span>–<span class="cit-lpage">688</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DChambers%26rft.auinit1%253DT.%26rft.volume%253D44%26rft.issue%253D1%26rft.spage%253D649%26rft.epage%253D688%26rft.atitle%253DFlavivirus%2BGenome%2BOrganization%252C%2BExpression%252C%2BAnd%2BReplication%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.mi.44.100190.003245%26rft_id%253Dinfo%253Apmid%252F2174669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.mi.44.100190.003245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2174669&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990EA66200026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.14" data-doi="10.1074/jbc.M107360200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schroder</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">NX</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stoermer</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abbenante</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">PR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fairlie</span>  <span class="cit-name-given-names">DP.</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">Activity of recombinant dengue 2 virus NS3 protease in the presence of a truncated NS2B co-factor, small peptide substrates, and inhibitors</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>. <span class="cit-vol">276</span>, <span class="cit-fpage">45762</span>–<span class="cit-lpage">45771</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M107360200%26rft_id%253Dinfo%253Apmid%252F11581268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzYvNDkvNDU3NjIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wOS8yOC8wNzgxMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-078113v1.15" data-doi="10.1074/jbc.M406810200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nall</span>  <span class="cit-name-given-names">TA</span></span>1, <span class="cit-auth"><span class="cit-name-surname">Chappell</span>  <span class="cit-name-given-names">KJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stoermer</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">NX</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tyndall</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">PR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fairlie</span>  <span class="cit-name-given-names">DP.</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Enzymatic characterization and homology model of a catalytically active recombinant West Nile virus NS3 protease</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>. <span class="cit-vol">279</span>, <span class="cit-fpage">48535</span>–<span class="cit-lpage">48542</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M406810200%26rft_id%253Dinfo%253Apmid%252F15322074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzkvNDcvNDg1MzUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wOS8yOC8wNzgxMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Iempridee</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thongphung</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Angsuthanasombat</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Katzenmeier</span>  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">A comparative biochemical analysis of the NS2B(H)-NS3pro protease complex from four dengue virus serotypes</span>. <abbr class="cit-jnl-abbrev">Biochim Biophys Acta</abbr>. <span class="cit-vol">1780</span>, <span class="cit-fpage">989</span>–<span class="cit-lpage">994</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DIempridee%26rft.auinit1%253DT.%26rft.volume%253D1780%26rft.issue%253D7-8%26rft.spage%253D989%26rft.epage%253D994%26rft.atitle%253DA%2Bcomparative%2Bbiochemical%2Banalysis%2Bof%2Bthe%2BNS2B%2528H%2529-NS3pro%2Bprotease%2Bcomplex%2Bfrom%2Bfour%2Bdengue%2Bvirus%2Bserotypes.%26rft_id%253Dinfo%253Apmid%252F18439431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18439431&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.17" data-doi="10.1074/jbc.M500588200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">SP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beer</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tumanut</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tully</span>  <span class="cit-name-given-names">DC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiricek</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Priestle</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vasudevan</span>  <span class="cit-name-given-names">SG.</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Functional profiling of recombinant NS3 proteases from all four serotypes of dengue virus using tetrapeptide and octapeptide substrate libraries</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>. <span class="cit-vol">280</span>, <span class="cit-fpage">28766</span>–<span class="cit-lpage">28774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M500588200%26rft_id%253Dinfo%253Apmid%252F15932883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODAvMzEvMjg3NjYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wOS8yOC8wNzgxMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.18" data-doi="10.1038/nsmb1073"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Erbel</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schiering</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">D’Arcy</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Renatus</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kroemer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">SP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">TH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vasudevan</span>  <span class="cit-name-given-names">SG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hommel</span>  <span class="cit-name-given-names">U.</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Structural basis for the activation of flaviviral NS3 proteases from dengue and West Nile virus</span>. <abbr class="cit-jnl-abbrev">Nat Struct Mol Biol</abbr>. <span class="cit-vol">13</span>, <span class="cit-fpage">372</span>–<span class="cit-lpage">373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DErbel%26rft.auinit1%253DP.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D372%26rft.epage%253D373%26rft.atitle%253DStructural%2Bbasis%2Bfor%2Bthe%2Bactivation%2Bof%2Bflaviviral%2BNS3%2Bproteases%2Bfrom%2Bdengue%2Band%2BWest%2BNile%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb1073%26rft_id%253Dinfo%253Apmid%252F16532006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb1073&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16532006&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236581400020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.19" data-doi="10.1021/jm900448m"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ekonomiuk</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">XC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ozawa</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bodenreider</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">SP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Otting</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Caflisch</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Flaviviral protease inhibitors identified by fragment-based library docking into a structure generated by molecular dynamics</span>. <abbr class="cit-jnl-abbrev">J Med Chem</abbr>. <span class="cit-vol">52</span>, <span class="cit-fpage">4860</span>–<span class="cit-lpage">4868</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedicinal%2Bchemistry%26rft.stitle%253DJ%2BMed%2BChem%26rft.aulast%253DEkonomiuk%26rft.auinit1%253DD.%26rft.volume%253D52%26rft.issue%253D15%26rft.spage%253D4860%26rft.epage%253D4868%26rft.atitle%253DFlaviviral%2Bprotease%2Binhibitors%2Bidentified%2Bby%2Bfragment-based%2Blibrary%2Bdocking%2Binto%2Ba%2Bstructure%2Bgenerated%2Bby%2Bmolecular%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjm900448m%26rft_id%253Dinfo%253Apmid%252F19572550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/jm900448m&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19572550&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.20" data-doi="10.1021/ja208435s"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de la Cruz</span>  <span class="cit-name-given-names">L</span></span>1, <span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">TH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ozawa</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Otting</span>  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Binding of low molecular weight inhibitors promotes large conformational changes in the dengue virus NS2B-NS3 protease: fold analysis by pseudocontact shifts</span>. <abbr class="cit-jnl-abbrev">J Am Chem Soc</abbr>. <span class="cit-vol">133</span>, <span class="cit-fpage">19205</span>–<span class="cit-lpage">19215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.stitle%253DJournal%2Bof%2Bthe%2BAmerican%2BChemical%2BSociety%26rft.aulast%253Dde%2Bla%2BCruz%26rft.auinit1%253DL.%26rft.volume%253D133%26rft.issue%253D47%26rft.spage%253D19205%26rft.epage%253D19215%26rft.atitle%253DBinding%2Bof%2Blow%2Bmolecular%2Bweight%2Binhibitors%2Bpromotes%2Blarge%2Bconformational%2Bchanges%2Bin%2Bthe%2Bdengue%2Bvirus%2BNS2B-NS3%2Bprotease%253A%2Bfold%2Banalysis%2Bby%2Bpseudocontact%2Bshifts.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fja208435s%26rft_id%253Dinfo%253Apmid%252F22007671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ja208435s&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22007671&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297662800032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.21" data-doi="10.1016/j.antiviral.2012.09.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">CG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">PY.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Structural biology of dengue virus enzymes: towards rational design of therapeutics</span>. <abbr class="cit-jnl-abbrev">Antiviral Res</abbr>. <span class="cit-vol">96</span>, <span class="cit-fpage">115</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2Bresearch%26rft.stitle%253DAntiviral%2BRes%26rft.aulast%253DNoble%26rft.auinit1%253DC.%2BG.%26rft.volume%253D96%26rft.issue%253D2%26rft.spage%253D115%26rft.epage%253D126%26rft.atitle%253DStructural%2Bbiology%2Bof%2Bdengue%2Bvirus%2Benzymes%253A%2BTowards%2Brational%2Bdesign%2Bof%2Btherapeutics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2012.09.007%26rft_id%253Dinfo%253Apmid%252F22995600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2012.09.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22995600&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000313393800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.22" data-doi="10.1016/j.antiviral.2015.03.014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vasudevan</span>  <span class="cit-name-given-names">SG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lescar</span>  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">The flavivirus NS2B-NS3 protease-helicase as a target for antiviral drug development</span>. <abbr class="cit-jnl-abbrev">Antiviral Res</abbr>. <span class="cit-vol">118</span>, <span class="cit-fpage">148</span>–<span class="cit-lpage">158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2BRes%26rft.volume%253D118%26rft.spage%253D148%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2015.03.014%26rft_id%253Dinfo%253Apmid%252F25842996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2015.03.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25842996&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.23" data-doi="10.1074/jbc.M112.442723"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">YM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gayen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joy</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wee</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ang</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">HA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hung</span>  <span class="cit-name-given-names">AW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">CG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee le</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yip</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">QY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">CS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">PY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">TH.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">NMR analysis of a novel enzymatically active unlinked dengue NS2B-NS3 protease complex</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>. <span class="cit-vol">288</span>, <span class="cit-fpage">12891</span>–<span class="cit-lpage">12900</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M112.442723%26rft_id%253Dinfo%253Apmid%252F23511634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODgvMTgvMTI4OTEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wOS8yOC8wNzgxMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.24" data-doi="10.1371/journal.pbio.1002338"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">ALS-Causing Mutations Significantly Perturb the Self-Assembly and Interaction with Nucleic Acid of the Intrinsically Disordered Prion-Like Domain of TDP-43</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr>. <span class="cit-vol">14</span>, <span class="cit-fpage">e1002338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol%26rft.volume%253D14%26rft.spage%253De1002338%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1002338%26rft_id%253Dinfo%253Apmid%252F26735904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1002338&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26735904&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.25" data-doi="10.1126/science.aag2419"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lei</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nitsche</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hilgenfeld</span>  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Crystal structure of Zika virus NS2B-NS3 protease in complex with a boronate inhibitor</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-vol">353</span>, <span class="cit-fpage">503</span>–<span class="cit-lpage">505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aag2419%26rft_id%253Dinfo%253Apmid%252F27386922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTMvNjI5OC81MDMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wOS8yOC8wNzgxMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.26" data-doi="10.1016/S0076-6879(03)74020-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fiser</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Šali</span>  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Modeller: generation and refinement of homology-based protein structure models</span>. <abbr class="cit-jnl-abbrev">Methods Enzym</abbr>. <span class="cit-vol">374</span>, <span class="cit-fpage">461</span>–<span class="cit-lpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DFiser%26rft.auinit1%253DA.%26rft.volume%253D374%26rft.spage%253D461%26rft.epage%253D491%26rft.atitle%253DModeller%253A%2Bgeneration%2Band%2Brefinement%2Bof%2Bhomology-based%2Bprotein%2Bstructure%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0076-6879%252803%252974020-8%26rft_id%253Dinfo%253Apmid%252F14696385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0076-6879(03)74020-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14696385&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188419600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.27" data-doi="10.1126/science.1085658"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anand</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wadhwani</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mesters</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hilgenfeld</span>  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Coronavirus main proteinase (3CLpro) structure: basis for design of anti-SARS drugs</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">300</span>, <span class="cit-fpage">1763</span>–<span class="cit-lpage">1767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1085658%26rft_id%253Dinfo%253Apmid%252F12746549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDAvNTYyNi8xNzYzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDkvMjgvMDc4MTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nussinov</span>  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Structured crowding and its effects on enzyme catalysis</span>. <abbr class="cit-jnl-abbrev">Top Curr Chem</abbr>. <span class="cit-vol">337</span>, <span class="cit-fpage">123</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTop%2BCurr%2BChem%26rft.volume%253D337%26rft.spage%253D123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.29" data-doi="10.1371/journal.pcbi.1001084"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lua</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sivaraman</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Dynamically-driven inactivation of the catalytic machinery of the SARS 3C-like protease by the N214A mutation on the extra domain</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr>. <span class="cit-vol">7</span>, <span class="cit-fpage">e1001084</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DShi%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De1001084%26rft.epage%253De1001084%26rft.atitle%253DDynamically-driven%2Binactivation%2Bof%2Bthe%2Bcatalytic%2Bmachinery%2Bof%2Bthe%2BSARS%2B3C-like%2Bprotease%2Bby%2Bthe%2BN214A%2Bmutation%2Bon%2Bthe%2Bextra%2Bdomain.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1001084%26rft_id%253Dinfo%253Apmid%252F21390281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1001084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21390281&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.30" data-doi="10.1371/journal.pone.0101941"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Dynamically-driven enhancement of the catalytic machinery of the SARS 3C-like protease by the S284-T285-I286/A mutations on the extra domain</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-vol">9</span>, <span class="cit-fpage">e101941</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De101941%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0101941%26rft_id%253Dinfo%253Apmid%252F25036652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0101941&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25036652&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.31" data-doi="10.1128/JVI.06225-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">CG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seh</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">AT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">PY.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Ligand-bound structures of the dengue virus protease reveal the active conformation</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-vol">86</span>, <span class="cit-fpage">438</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.06225-11%26rft_id%253Dinfo%253Apmid%252F22031935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6Ijg2LzEvNDM4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDkvMjgvMDc4MTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.32" data-doi="10.1371/journal.pone.0039261"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Intrinsically unstructured domain 3 of hepatitis C Virus NS5A forms a “fuzzy complex” with VAPB-MSP domain which carries ALS-causing mutations</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-vol">7</span>, <span class="cit-fpage">e39261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGupta%26rft.auinit1%253DG.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De39261%26rft.epage%253De39261%26rft.atitle%253DIntrinsically%2Bunstructured%2Bdomain%2B3%2Bof%2Bhepatitis%2BC%2BVirus%2BNS5A%2Bforms%2Ba%2B%2522fuzzy%2Bcomplex%2522%2Bwith%2BVAPB-MSP%2Bdomain%2Bwhich%2Bcarries%2BALS-causing%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0039261%26rft_id%253Dinfo%253Apmid%252F22720086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0039261&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22720086&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.33" data-doi="10.1039/c4mb00027g"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dolan</span>  <span class="cit-name-given-names">PT</span></span>, <span class="cit-auth"><span class="cit-name-surname">LaCount</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kurgan</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uversky</span>  <span class="cit-name-given-names">VN.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">The intrinsic disorder status of the human hepatitis C virus proteome</span>. <abbr class="cit-jnl-abbrev">Mol Biosyst</abbr>. <span class="cit-vol">10</span>, <span class="cit-fpage">1345</span>–<span class="cit-lpage">1363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiosyst%26rft.volume%253D10%26rft.spage%253D1345%26rft_id%253Dinfo%253Adoi%252F10.1039%252Fc4mb00027g%26rft_id%253Dinfo%253Apmid%252F24752801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1039/c4mb00027g&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24752801&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.34" data-doi="10.1186/1741-7007-10-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nussinov</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Protein dynamics and conformational selection in bidirectional signal transduction</span>. <abbr class="cit-jnl-abbrev">BMC Biol</abbr>. <span class="cit-vol">10</span>, <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DNussinov%26rft.auinit1%253DR.%26rft.volume%253D10%26rft.spage%253D2%26rft.epage%253D2%26rft.atitle%253DProtein%2Bdynamics%2Band%2Bconformational%2Bselection%2Bin%2Bbidirectional%2Bsignal%2Btransduction.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-10-2%26rft_id%253Dinfo%253Apmid%252F22277130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-10-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22277130&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yildiz</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hardy</span>  <span class="cit-name-given-names">JA.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Allosteric inhibition of the NS2B-NS3 protease from dengue virus</span>. <abbr class="cit-jnl-abbrev">ACS Chem Biol</abbr>. <span class="cit-vol">8</span>, <span class="cit-fpage">2744</span>–<span class="cit-lpage">2752</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BChem%2BBiol%26rft.volume%253D8%26rft.spage%253D2744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.36" data-doi="10.1139/bcb-76-2-3-177"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">1998</span>) <span class="cit-article-title">NMR for the design of functional mimetics of protein-protein interactions: one key is in the building of bridges</span>. <abbr class="cit-jnl-abbrev">Biochem Cell Biol</abbr>. <span class="cit-vol">76</span>, <span class="cit-fpage">177</span>–<span class="cit-lpage">188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%2Band%2Bcell%2Bbiology%2B%253D%2BBiochimie%2Bet%2Bbiologie%2Bcellulaire%26rft.stitle%253DBiochem%2BCell%2BBiol%26rft.aulast%253DSong%26rft.auinit1%253DJ.%26rft.volume%253D76%26rft.issue%253D2-3%26rft.spage%253D177%26rft.epage%253D188%26rft.atitle%253DNMR%2Bfor%2Bthe%2Bdesign%2Bof%2Bfunctional%2Bmimetics%2Bof%2Bprotein-protein%2Binteractions%253A%2Bone%2Bkey%2Bis%2Bin%2Bthe%2Bbuilding%2Bof%2Bbridges.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fbcb-76-2-3-177%26rft_id%253Dinfo%253Apmid%252F9923687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/bcb-76-2-3-177&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9923687&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078073600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">37.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-078113v1.37" data-doi="10.1074/jbc.M311744200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Dissection study on the severe acute respiratory syndrome 3C-like protease reveals the critical role of the extra domain in dimerization of the enzyme: defining the extra domain as a new target for design of highly specific protease inhibitors</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr>. <span class="cit-vol">279</span>, <span class="cit-fpage">24765</span>–<span class="cit-lpage">24773</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M311744200%26rft_id%253Dinfo%253Apmid%252F15037623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzkvMjMvMjQ3NjUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wOS8yOC8wNzgxMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.38" data-doi="10.1021/cb500413n"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">LZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Dynamic principle for designing antagonistic/agonistic molecules for EphA4 receptor, the only known ALS modifier</span>. <abbr class="cit-jnl-abbrev">ACS Chem Biol</abbr>. <span class="cit-vol">10</span>, <span class="cit-fpage">372</span>–<span class="cit-lpage">378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BChem%2BBiol%26rft.volume%253D10%26rft.spage%253D372%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fcb500413n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/cb500413n&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-078113v1.39"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Harvey</span> <span class="cit-name-surname">Motulsky</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Arthur</span> <span class="cit-name-surname">Christopoulos</span></span>, <span class="cit-chapter-title">Fitting Models to Biological Data using Linear and Nonlinear Regression</span>. <span class="cit-source">A Practical Guide to Curve Fitting</span>., <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York</span>, <span class="cit-pub-date">2004</span>. ISBN: <span class="cit-isbn">0195171802</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.40" data-doi="10.1021/ct700301q"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hess</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kutzner</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Der Spoel</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lindahl</span>  <span class="cit-name-given-names">E.</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Gromacs 4: algorithms for highly efficient, load-balanced, and scalable molecular simulation</span>. <abbr class="cit-jnl-abbrev">J Chem Theory Comput</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">435</span>–<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BChem%2BTheory%2BComput%26rft.volume%253D4%26rft.spage%253D435%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fct700301q%26rft_id%253Dinfo%253Apmid%252F26620784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ct700301q&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26620784&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254277900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.41" data-doi="10.1002/prot.22711"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lindorff-Larsen</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Piana</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palmo</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maragakis</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klepeis</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dror</span>  <span class="cit-name-given-names">RO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">DE.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Improved side-chain torsion potentials for the amber ff99sb protein force field</span>. <abbr class="cit-jnl-abbrev">Proteins</abbr>. <span class="cit-vol">78</span>, <span class="cit-fpage">1950</span>–<span class="cit-lpage">1958</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%26rft.stitle%253DProteins%26rft.aulast%253DLindorff-Larsen%26rft.auinit1%253DK.%26rft.volume%253D78%26rft.issue%253D8%26rft.spage%253D1950%26rft.epage%253D1958%26rft.atitle%253DImproved%2Bside-chain%2Btorsion%2Bpotentials%2Bfor%2Bthe%2BAmber%2Bff99SB%2Bprotein%2Bforce%2Bfield.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fprot.22711%26rft_id%253Dinfo%253Apmid%252F20408171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/prot.22711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20408171&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F09%2F28%2F078113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277767700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-078113v1.42" data-doi="10.1002/(SICI)1096-987X(199709)18:12&lt;1463::AID-JCC4&gt;3.3.CO;2-L"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hess</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bekker</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berendsen</span>  <span class="cit-name-given-names">HJC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fraaije</span>  <span class="cit-name-given-names">JG.</span></span> (<span class="cit-pub-date">1997</span>) <span class="cit-article-title">LINCS: A linear constraint solver for molecular simulations</span>. <abbr class="cit-jnl-abbrev">J Comput Chem</abbr>. <span class="cit-vol">18</span>, <span class="cit-fpage">1463</span>–<span class="cit-lpage">1472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BComput%2BChem%26rft.volume%253D18%26rft.spage%253D1463%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291096-987X%2528199709%252918%253A12%253C1463%253A%253AAID-JCC4%253E3.3.CO%253B2-L%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1096-987X(199709)18:12&lt;1463::AID-JCC4&gt;3.3.CO;2-L&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XT81100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/078113v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F078113v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/078170v1" title="Evolutionary and Functional Implications of Hypervariable Loci Within the Skin Virome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/078162v1" title="A Pif1-dependent threshold separates DNA double-strand breaks and telomeres" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;September 28, 2016.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/078113v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/078113v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/078113v1" class="active" data-icon-position="" data-hide-link-title="0">Unique properties of Zika NS2B-NS3pro complexes as decoded by experiments and MD simulations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/21585" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-BF7qXVh2BQyWixUsMONMyejxWoC0hS9EFg6dHRV2Lo8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="21585" id="node21585--2" data-pisa="biorxiv;078113v1" data-pisa-master="biorxiv;078113" data-seqnum="21585" data-apath="/biorxiv/early/2016/09/28/078113.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Unique properties of Zika NS2B-NS3pro complexes as decoded by experiments and MD simulations</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Amrita</span> <span class="nlm-surname">Roy</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Liangzhong</span> <span class="nlm-surname">Lim</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Shagun</span> <span class="nlm-surname">Srivastava</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jianxing</span> <span class="nlm-surname">Song</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">078113; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/078113 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/078113v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/078113v1&amp;title=Unique%20properties%20of%20Zika%20NS2B-NS3pro%20complexes%20as%20decoded%20by%20experiments%20and%20MD%20simulations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/078113v1&amp;title=Unique%20properties%20of%20Zika%20NS2B-NS3pro%20complexes%20as%20decoded%20by%20experiments%20and%20MD%20simulations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/078113v1&amp;text=Unique%20properties%20of%20Zika%20NS2B-NS3pro%20complexes%20as%20decoded%20by%20experiments%20and%20MD%20simulations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/078113v1&amp;title=Unique%20properties%20of%20Zika%20NS2B-NS3pro%20complexes%20as%20decoded%20by%20experiments%20and%20MD%20simulations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/078113v1&amp;t=Unique%20properties%20of%20Zika%20NS2B-NS3pro%20complexes%20as%20decoded%20by%20experiments%20and%20MD%20simulations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/078113v1&amp;title=Unique%20properties%20of%20Zika%20NS2B-NS3pro%20complexes%20as%20decoded%20by%20experiments%20and%20MD%20simulations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/078113v1&amp;title=Unique%20properties%20of%20Zika%20NS2B-NS3pro%20complexes%20as%20decoded%20by%20experiments%20and%20MD%20simulations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="21585" id="node21585--3" data-pisa="biorxiv;078113v1" data-pisa-master="biorxiv;078113" data-seqnum="21585" data-apath="/biorxiv/early/2016/09/28/078113.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Unique properties of Zika NS2B-NS3pro complexes as decoded by experiments and MD simulations</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Amrita</span> <span class="nlm-surname">Roy</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Liangzhong</span> <span class="nlm-surname">Lim</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Shagun</span> <span class="nlm-surname">Srivastava</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jianxing</span> <span class="nlm-surname">Song</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">078113; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/078113 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Unique%20properties%20of%20Zika%20NS2B-NS3pro%20complexes%20as%20decoded%20by%20experiments%20and%20MD%20simulations&amp;rft.volume&amp;rft.issue&amp;rft.date=2016-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Roy%2C+Amrita&amp;rft.au=Lim%2C+Liangzhong&amp;rft.au=Srivastava%2C+Shagun&amp;rft.au=Song%2C+Jianxing"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/21585/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/21585/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/21585/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/21585/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/21585/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/21585/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/21585/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/21585/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/21585/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/21585/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/21585/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/21585/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/078113v1&amp;count=horizontal&amp;via=&amp;text=Unique%20properties%20of%20Zika%20NS2B-NS3pro%20complexes%20as%20decoded%20by%20experiments%20and%20MD%20simulations&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/078113v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/078113v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/078113v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/biophysics" class="highlight" data-icon-position="" data-hide-link-title="0">Biophysics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20277)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
