<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/embed/graphic-4.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/01/24/732255.atom" />
<meta name="HW.pisa" content="biorxiv;732255v3" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Predicting the global mammalian viral sharing network using phylogeography" />
<meta name="DC.Identifier" content="10.1101/732255" />
<meta name="DC.Date" content="2020-01-24" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Understanding interspecific viral transmission is key to understanding viral ecology and evolution, disease spillover into humans, and the consequences of global change. Prior work has demonstrated that macroecological factors drive viral sharing in some mammalian groups, but analyses have never attempted to predict viral sharing in a pan-mammalian context. Here we show that host phylogenetic similarity and geographic range overlap are strong, nonlinear predictors of viral sharing among species across the entire mammal class. Using these traits, we predict global viral sharing patterns across 4196 mammal species and show that our simulated network successfully predicts viral sharing and reservoir host status using internal validation and an external dataset. We predict high rates of mammalian viral sharing in the tropics, particularly among rodents and bats, and that within- and between-order sharing differs geographically and taxonomically. Our results emphasize the importance of macroecological factors in shaping mammalian viral communities, and provide a robust, general model to predict viral host range and guide pathogen surveillance and conservation efforts." />
<meta name="DC.Contributor" content="Gregory F Albery" />
<meta name="DC.Contributor" content="Evan A Eskew" />
<meta name="DC.Contributor" content="Noam Ross" />
<meta name="DC.Contributor" content="Kevin J Olival" />
<meta name="article:published_time" content="2020-01-24" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Predicting the global mammalian viral sharing network using phylogeography" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Understanding interspecific viral transmission is key to understanding viral ecology and evolution, disease spillover into humans, and the consequences of global change. Prior work has demonstrated that macroecological factors drive viral sharing in some mammalian groups, but analyses have never attempted to predict viral sharing in a pan-mammalian context. Here we show that host phylogenetic similarity and geographic range overlap are strong, nonlinear predictors of viral sharing among species across the entire mammal class. Using these traits, we predict global viral sharing patterns across 4196 mammal species and show that our simulated network successfully predicts viral sharing and reservoir host status using internal validation and an external dataset. We predict high rates of mammalian viral sharing in the tropics, particularly among rodents and bats, and that within- and between-order sharing differs geographically and taxonomically. Our results emphasize the importance of macroecological factors in shaping mammalian viral communities, and provide a robust, general model to predict viral host range and guide pathogen surveillance and conservation efforts.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;732255v3" />
<meta name="citation_id" content="732255v3" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/732255v3" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/732255v3.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/732255v3.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255.full.pdf" />
<meta name="citation_doi" content="10.1101/732255" />
<meta name="citation_num_pages" content="30" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="732255" />
<meta name="citation_author" content="Gregory F Albery" />
<meta name="citation_author_institution" content="EcoHealth Alliance" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh" />
<meta name="citation_author_institution" content="Department of Biology, Georgetown University" />
<meta name="citation_author_email" content="gfalbery@gmail.com" />
<meta name="citation_author_email" content="olival@ecohealthalliance.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6260-2662" />
<meta name="citation_author" content="Evan A Eskew" />
<meta name="citation_author_institution" content="EcoHealth Alliance" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1153-5356" />
<meta name="citation_author" content="Noam Ross" />
<meta name="citation_author_institution" content="EcoHealth Alliance" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2136-0000" />
<meta name="citation_author" content="Kevin J Olival" />
<meta name="citation_author_institution" content="EcoHealth Alliance" />
<meta name="citation_author_email" content="gfalbery@gmail.com" />
<meta name="citation_author_email" content="olival@ecohealthalliance.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3211-1875" />
<meta name="citation_reference" content="Woolhouse, M. E. J. &amp; Gowtage-Sequeria, S. Host range and emerging and reemerging pathogens. Emerg. Infect. Dis. 11, 1842–7 (2005)." />
<meta name="citation_reference" content="Johnson, C. K. et al. Spillover and pandemic properties of zoonotic viruses with high host plasticity. Sci. Rep. 5, 1–8 (2015)." />
<meta name="citation_reference" content="Carroll, D. et al. The Global Virome Project. Science (80-.). 359, 872–874 (2018)." />
<meta name="citation_reference" content="Carlson, C. J., Zipfel, C. M., Garnier, R. &amp; Bansal, S. Global estimates of mammalian viral diversity accounting for host sharing. Nat. Ecol. Evol. 3, 1070–1075 (2019)." />
<meta name="citation_reference" content="Olival, K. J. et al. Host and viral traits predict zoonotic spillover from mammals. Nature 546, 646–650 (2017)." />
<meta name="citation_reference" content="Han, B. A. et al. Undiscovered Bat Hosts of Filoviruses. PLoS Negl. Trop. Dis. 10, e0004815 (2016)." />
<meta name="citation_reference" content="Babayan, S. A., Orton, R. J. &amp; Streicker, D. G. Predicting reservoir hosts and arthropod vectors from evolutionary signatures in RNA virus genomes. Science (80-.). 362, 577–580 (2018)." />
<meta name="citation_reference" content="Luis, A. D. et al. A comparison of bats and rodents as reservoirs of zoonotic viruses: are bats special? Proc. R. Soc. B Biol. Sci. 280, 20122753 (2013)." />
<meta name="citation_reference" content="Han, B. A., Schmidt, J. P., Bowden, S. E. &amp; Drake, J. M. Rodent reservoirs of future zoonotic diseases. Proc. Natl. Acad. Sci. 112, 7039–7044 (2015)." />
<meta name="citation_reference" content="Plourde, B. T. et al. Are disease reservoirs special? Taxonomic and life history characteristics. PLoS One 12, e0180716 (2017)." />
<meta name="citation_reference" content="Dallas, T. A. et al. Host traits associated with species roles in parasite sharing networks. Oikos 128, 23–32 (2019)." />
<meta name="citation_reference" content="Plowright, R. K. et al. Pathways to zoonotic spillover. Nat. Rev. Microbiol. 15, 502–510 (2017)." />
<meta name="citation_reference" content="Ge, X. Y. et al. Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor. Nature 503, 535–538 (2013)." />
<meta name="citation_reference" content="Huang, S., Bininda-Emonds, O. R. P., Stephens, P. R., Gittleman, J. L. &amp; Altizer, S. Phylogenetically related and ecologically similar carnivores harbour similar parasite assemblages. J. Anim. Ecol. 83, 671–680 (2014)." />
<meta name="citation_reference" content="Wells, K. et al. Global spread of helminth parasites at the human-domestic animal-wildlife interface. Glob. Chang. Biol. 24, 3254–3265 (2018)." />
<meta name="citation_reference" content="Stephens, P. R. et al. Parasite sharing in wild ungulates and their predators: Effects of phylogeny, range overlap, and trophic links. J. Anim. Ecol. 88, 1017–1028 (2019)." />
<meta name="citation_reference" content="Streicker, D. G. et al. Host Phylogeny Constrains Cross-Species Emergence and Establishment of Rabies Virus in Bats. Science (80-.). 329, 676–679 (2010)." />
<meta name="citation_reference" content="Willoughby, A. R., Phelps, K. L., Predict Consortium, predict@ucdavis edu &amp; Olival, K. J. A comparative analysis of viral richness and viral sharing in cave-roosting bats. Diversity 9, 1–16 (2017)." />
<meta name="citation_reference" content="Davies, T. J. &amp; Pedersen, A. B. Phylogeny and geography predict pathogen community similarity in wild primates and humans. Proc. R. Soc. B Biol. Sci. 275, 1695–1701 (2008)." />
<meta name="citation_reference" content="Glennon, E. E. et al. Domesticated animals as hosts of henipaviruses and filoviruses: A systematic review. Vet. J. 233, 25–34 (2018)." />
<meta name="citation_reference" content="Chua, K. B. et al. Nipah Virus: A Recently Emergent Deadly Paramyxovirus. Science (80-.). 288, 1432–1435 (2000)." />
<meta name="citation_reference" content="Guy, C., Thiagavel, J., Mideo, N. &amp; Ratcliffe, J. M. Phylogeny matters: Revisiting ‘a comparison of bats and rodents as reservoirs of zoonotic viruses’. R. Soc. Open Sci. 6, 181182 (2019)." />
<meta name="citation_reference" content="Pybus, O. G., Tatem, A. J. &amp; Lemey, P. Virus evolution and transmission in an ever more connected world. Proc. R. Soc. B Biol. Sci. 282, 20142878 (2015)." />
<meta name="citation_reference" content="Sanjuán, R., Nebot, M. R., Chirico, N., Mansky, L. M. &amp; Belshaw, R. Viral mutation rates. J. Virol. 84, 9733–9748 (2010)." />
<meta name="citation_reference" content="Ryan, S. J., Carlson, C. J., Mordecai, E. A. &amp; Johnson, L. R. Global expansion and redistribution of Aedes-borne virus transmission risk with climate change. PLoS Negl. Trop. Dis. 13, e0007213 (2019)." />
<meta name="citation_reference" content="Drake, J. M. &amp; Beier, J. C. Ecological niche and potential distribution of Anopheles arabiensis in Africa in 2050. Malar. J. 13, 213 (2014)." />
<meta name="citation_reference" content="IUCN. The IUCN Red List of Threatened Species. IUCN Red List of Threatened Species. Version 2019–2 (2019). Available at: https://www.iucnredlist.org." />
<meta name="citation_reference" content="Fritz, S. A., Bininda-Emonds, O. R. P. &amp; Purvis, A. Geographical variation in predictors of mammalian extinction risk: big is bad, but only in the tropics. Ecol. Lett. 12, 538–549 (2009)." />
<meta name="citation_reference" content="Wardeh, M., Risley, C., Mcintyre, M. K., Setzkorn, C. &amp; Baylis, M. Database of host-pathogen and related species interactions, and their global distribution. Sci. Data 2, 150049 (2015)." />
<meta name="citation_reference" content="Gómez, J. M., Nunn, C. L. &amp; Verdú, M. Centrality in primate-parasite networks reveals the potential for the transmission of emerging infectious diseases to humans. Proc. Natl. Acad. Sci. 110, 7738–41 (2013)." />
<meta name="citation_reference" content="Wang, Y. X. G. G. et al. Phylogenetic structure of wildlife assemblages shapes patterns of infectious livestock diseases in Africa. Funct. Ecol. 33, 1332–1341 (2019)." />
<meta name="citation_reference" content="Allen, T. et al. Global hotspots and correlates of emerging zoonotic diseases. Nat. Commun. 8, 1124 (2017)." />
<meta name="citation_reference" content="Becker, D. J., Crowley, D. E., Washburne, A. D. &amp; Plowright, R. K. Temporal and spatial limitations in global surveillance for bat filoviruses and henipaviruses. Biol. Lett. 15, 20190423 (2019)." />
<meta name="citation_reference" content="Xie, J. et al. Dampened STING-Dependent Interferon Activation in Bats. Cell Host Microbe 23, 297–301 (2018)." />
<meta name="citation_reference" content="Park, A. W. Phylogenetic aggregation increases zoonotic potential of mammalian viruses. Biol. Lett. 15, 20190668 (2019)." />
<meta name="citation_reference" content="Lloyd-smith, J. O. et al. Epidemic Dynamics at the Human-Animal Interface. Science (80-.). 326, 1362–1368 (2009)." />
<meta name="citation_reference" content="Brose, U., Ostling, A., Harrison, K. &amp; Martinez, N. D. Unified spatial scaling of species and their trophic interactions. Nature 108, 167–171 (2003)." />
<meta name="citation_reference" content="Silk, M. et al. Integrating social behaviour, demography and disease dynamics in network models: applications to disease management in declining wildlife populations. Philos. Trans. R. Soc. B 374, 20180211 (2019)." />
<meta name="citation_reference" content="Carlson, C. J. et al. Parasite biodiversity faces extinction and redistribution in a changing climate. Sci. Adv. 3, e1602422 (2017)." />
<meta name="citation_reference" content="Williams, J. E. &amp; Blois, J. L. Range shifts in response to past and future climate change: Can climate velocities and species’ dispersal capabilities explain variation in mammalian range shifts? J. Biogeogr. 45, 2175–2189 (2018)." />
<meta name="citation_reference" content="Chen, I.-C., Hill, J. K., Ohlemüller, R., Roy, D. B. &amp; Thomas, C. D. Rapid range shifts of species associated with high levels of climate warming. Science (80-.). 333, 1024–6 (2011)." />
<meta name="citation_reference" content="Tompkins, D. M., Sainsbury, A. W., Nettleton, P., Buxton, D. &amp; Gurnell, J. Parapoxvirus causes a deleterious disease in red squirrels associated with UK population declines. Proc. R. Soc. B Biol. Sci. 269, 529–533 (2002)." />
<meta name="citation_reference" content="R Core Team. R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. (2018)." />
<meta name="citation_reference" content="Wood, S. N. Fast stable restricted maximum likelihood and marginal likelihood estimation of semiparametric generalized linear models. J. R. Stat. Soc. Ser. B (Statistical Methodol. 73, 3–36 (2011)." />
<meta name="twitter:title" content="Predicting the global mammalian viral sharing network using phylogeography" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Understanding interspecific viral transmission is key to understanding viral ecology and evolution, disease spillover into humans, and the consequences of global change. Prior work has demonstrated that macroecological factors drive viral sharing in some mammalian groups, but analyses have never attempted to predict viral sharing in a pan-mammalian context. Here we show that host phylogenetic similarity and geographic range overlap are strong, nonlinear predictors of viral sharing among species across the entire mammal class. Using these traits, we predict global viral sharing patterns across 4196 mammal species and show that our simulated network successfully predicts viral sharing and reservoir host status using internal validation and an external dataset. We predict high rates of mammalian viral sharing in the tropics, particularly among rodents and bats, and that within- and between-order sharing differs geographically and taxonomically. Our results emphasize the importance of macroecological factors in shaping mammalian viral communities, and provide a robust, general model to predict viral host range and guide pathogen surveillance and conservation efforts." />
<meta name="og-title" property="og:title" content="Predicting the global mammalian viral sharing network using phylogeography" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/732255v3" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Understanding interspecific viral transmission is key to understanding viral ecology and evolution, disease spillover into humans, and the consequences of global change. Prior work has demonstrated that macroecological factors drive viral sharing in some mammalian groups, but analyses have never attempted to predict viral sharing in a pan-mammalian context. Here we show that host phylogenetic similarity and geographic range overlap are strong, nonlinear predictors of viral sharing among species across the entire mammal class. Using these traits, we predict global viral sharing patterns across 4196 mammal species and show that our simulated network successfully predicts viral sharing and reservoir host status using internal validation and an external dataset. We predict high rates of mammalian viral sharing in the tropics, particularly among rodents and bats, and that within- and between-order sharing differs geographically and taxonomically. Our results emphasize the importance of macroecological factors in shaping mammalian viral communities, and provide a robust, general model to predict viral host range and guide pathogen surveillance and conservation efforts." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-01-24" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/732255v3.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/732255v3.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/732255v3.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/732255v3" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1119825" />
    <title>Predicting the global mammalian viral sharing network using phylogeography | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"vi6pAR6JmupRRErhaVk98zGI9sWaRZZiIc52b5ThQDM","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1119825","apath":"\/biorxiv\/early\/2020\/01\/24\/732255.atom","pisa":"biorxiv;732255v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;732255v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/732255v3","title":"Predicting the global mammalian viral sharing network using phylogeography","identifier":"node\/1119825"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yiLZOVt7c1tZFNkuSgZZRNVtTHpD5mD3JFVpEFyBGacaxoSwIUaidAVIm\\\\\\\/9GHNOWfxpYZhhGc6Y2b6bKvlys7vprbzGzLS9OTUgAmexOeU2p6dX25E615yKO65ixE94N5z7u7FaRb8\\\\\\\/F2NWoVtoJ2WYe3NJvcPPOlRQqcocloKM2Nubp0Q7YQ6ytYS28JYieOEM3xnofJApekgHZolmxONtnw6okUhLBRHxE2wQM9rlBRZ1fZOcWeQC5lQJGI1QuoLw6ifXpAx3y9qtMSw0ArhVTrkYBiPBaIjB1nF2jgei8UDCfoksxCSapW4FKhiNn8mtncx0fRy7AnzmHY8wX9IfnTfhAbvs8rJUW0umDp9DzOgoAtQjMyK4TRKiHtppfzQNf9vKak1e27iy+lPGyyUkfAnA+NWWFouLo7bD0OxjaC2kPfISmexQlxfXJoDwSacVb9LOp4ao9Qn+4HLu7eFf0P4pr\\\\\\\/koollQnukzX7mBy0qhOGbZ9vA02KHSufM2GnRsdBP5vIYOVVGICega2llgRm3QVwst64D1Nlcv1TdKkIDwSOVGoZ6OIVM5GjBgDdykMONxEPkZSJphjcOTBv\\\\\\\/F4eRRyJfv7TIslD7ry7jT4\\\\\\\/AGSi9Ldi58EMyS4vXRLYL7OYjaZZnZSzfv1Ye1clN6d6CwKm9Cu0aYW4bA\\\\\\\/TPKBv0xGbnPyQ4O+kl87sSdKUcPcH5fHIh4NKfrmaPx\\\\\\\/31RvaeBfQFDITyMnqhS15+\\\\\\\/mrUXctvjcqpkkg2CaNOTvYWqbZOQVi79O53zBSjvZ\\\\\\\/gri4UNeKd+7WsMX\\\\\\\/gBwq7mpqgy\\\\\\\/SAnWX8xuhlQHBs9VqAfiEvMCZwRncIJb23SBZZEWKfkwPaIFGZvdsBXWOOMNVR8uox9gGzK6EAHiR3vK3kW5x1kktvCT1AG0oa3Jyphr7auU7VFTesHhyPqWItJA4CqWhy+XIROebuQdk64YfOtpDo7dh6yPya7V6jEOkHuDRG1x6Zu6d6ZkEfX3bovdWC9vTOT28K2AzyKyHnSAJkNWjVd48kf9itYGwriJukxfG3DSIIm9kBqYBQ6gDKiUhCSMUNuOC5b5orLrOPmUG21RHpDeddv0m0Fgrg5OhalPPH1JoMzZO4QpCDBoNPskAtrNrd0yORaKCmbS3HGxCt9pVj4KOVJ1xTDS9PfExa7bJSGv6\\\\\\\/jR8iTceqV27WH97h9Xulpf3vpDxeCKMYiQDB8BlW8eBPFJSpZqYA0CXQ4\\\\\\\/5NPGIm9eI2P7WkSQLfDeN8pW\\\\\\\/TqCAwDRbdyqYxfsfT0Z0vhUpVzdHTN+H0vdj6AWvUkTbtpNKsr8rGx62hQyv0WKT6Jw2bcXnHs6SzvW\\\u0022,\\\u0022iv\\\u0022:\\\u0022V60amlBaX7oZatAfd6A6jQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022680d3ae5af4ac7724ae451efb9af4ad344c192cb1191d13804723b9027a942aa\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/732255v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1119825 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/732255v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OlW2aXaJpOoyXYlhG7WQLtQqFcsSWuOcMbllinQ8juo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1119825" id="node1119825" data-pisa="biorxiv;732255v3" data-pisa-master="biorxiv;732255" data-apath="/biorxiv/early/2020/01/24/732255.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Predicting the global mammalian viral sharing network using phylogeography</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6260-2662" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Gregory F</span> <span class="nlm-surname">Albery</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1153-5356" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Evan A</span> <span class="nlm-surname">Eskew</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2136-0000" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Noam</span> <span class="nlm-surname">Ross</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-3211-1875" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Kevin J</span> <span class="nlm-surname">Olival</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/732255 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1119825" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gregory F Albery </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>EcoHealth Alliance</span>, New York, NY, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Institute of Evolutionary Biology, University of Edinburgh</span>, Edinburgh, Scotland</div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Biology, Georgetown University</span>, Washington, DC, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gregory%2BF%2BAlbery%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Albery%20GF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGregory%2BF%2BAlbery%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6260-2662" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gregory F Albery</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gfalbery@gmail.com" class="" data-icon-position="" data-hide-link-title="0">gfalbery@gmail.com</a>
<a href="mailto:olival@ecohealthalliance.org" class="" data-icon-position="" data-hide-link-title="0">olival@ecohealthalliance.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Evan A Eskew </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>EcoHealth Alliance</span>, New York, NY, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evan%2BA%2BEskew%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskew%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEvan%2BA%2BEskew%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1153-5356" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Evan A Eskew</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Noam Ross </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>EcoHealth Alliance</span>, New York, NY, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Noam%2BRoss%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ross%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANoam%2BRoss%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2136-0000" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Noam Ross</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kevin J Olival </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>EcoHealth Alliance</span>, New York, NY, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BJ%2BOlival%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Olival%20KJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKevin%2BJ%2BOlival%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3211-1875" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kevin J Olival</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gfalbery@gmail.com" class="" data-icon-position="" data-hide-link-title="0">gfalbery@gmail.com</a>
<a href="mailto:olival@ecohealthalliance.org" class="" data-icon-position="" data-hide-link-title="0">olival@ecohealthalliance.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/732255v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1119825" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1119825/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/732255v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1119825" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1119825/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/732255v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1119825" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1119825/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/732255v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1119825" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1119825/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/732255v3.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:1119825" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:1119825/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/732255v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1119825" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1119825/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Understanding interspecific viral transmission is key to understanding viral ecology and evolution, disease spillover into humans, and the consequences of global change. Prior work has demonstrated that macroecological factors drive viral sharing in some mammalian groups, but analyses have never attempted to predict viral sharing in a pan-mammalian context. Here we show that host phylogenetic similarity and geographic range overlap are strong, nonlinear predictors of viral sharing among species across the entire mammal class. Using these traits, we predict global viral sharing patterns across 4196 mammal species and show that our simulated network successfully predicts viral sharing and reservoir host status using internal validation and an external dataset. We predict high rates of mammalian viral sharing in the tropics, particularly among rodents and bats, and that within- and between-order sharing differs geographically and taxonomically. Our results emphasize the importance of macroecological factors in shaping mammalian viral communities, and provide a robust, general model to predict viral host range and guide pathogen surveillance and conservation efforts.</p></div><div id="sec-1" class="subsection"><p id="p-5">Most emerging human viruses originate in wild mammals, so understanding the drivers of interspecific viral transmission in these taxa is an important public health research priority<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>. Despite a rapidly expanding knowledge base, the mammalian viruses known to science remain taxonomically biased and limited in scope, likely comprising less than 1% of the complete mammalian virome<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup>. Furthermore, host range is inadequately characterized even for the best-studied viruses<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>–<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. To help prioritise viral discovery efforts and zoonotic disease surveillance in wildlife, studies have revealed high (zoonotic) parasite diversity in certain host taxa, such as rodents and bats<sup><a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup>, and/or linked parasite diversity with host phenotypic traits such as reproductive output<sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>,<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>. Viral diversity has also been associated with host macroecological traits, including geographic range size<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup> and sympatry with other mammals<sup><a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a></sup>. The rationale for investigating viral diversity is that species with more viruses will generate more opportunities for viral transmission to other species, including humans. However, in order to infect a new host species, a virus must transmit, invade, and potentially replicate within the novel host<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>. Each of these processes becomes less likely if the two hosts differ more in terms of their geographic range, behaviour, and/or biochemistry (i.e., cellular receptors allowing viral attachment and invasion)<sup><a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>. Consequently, the probability that a pair of hosts will share a virus is shaped both by the species’ underlying viral diversity and by species interactions represented by pairwise measures such as spatial overlap, phylogenetic relatedness, and ecological similarity<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>–<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>.</p><p id="p-6">Previous investigations into pairwise determinants of viral sharing have been limited to one or two host orders (e.g., bats<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>,<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup>, primates<sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>, ungulates<sup><a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a></sup>, and carnivores<sup><a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a></sup>), while sometimes lumping together different types of pathogen (e.g., helminths, viruses, and bacteria). Viruses are sometimes shared across large host phylogenetic distances (e.g., Nipah virus in bats and pigs, among many others<sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>), requiring a broader understanding of viral sharing across mammals to predict patterns at different taxonomic and geographic scales. In addition, many mammalian orders have yet to be investigated in these analyses – most notably rodents, which are highly diverse and host important zoonotic viruses<sup><a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a></sup>. In addition, although phylogenetic and geographic viral sharing effects have been empirically demonstrated, the models have not yet been applied to validate viral sharing predictions using external datasets or make inferences about mammals with no known viral associations. If geographic and phylogenetic effects on viral sharing are as ubiquitous as they seem, these variables alone could provide a useful baseline model of viral sharing applicable across the mammal class.</p><p id="p-7">Here, we analyse pairwise viral sharing using a novel, conservative modelling approach designed to partition the contribution of species-level traits from pairwise phylogeographic traits. This method of analysis stands in contrast to previous studies of mammalian viral sharing which have mainly focussed on host-level traits, and importantly buffers against certain inherent biases in the observed viral sharing network, including host sampling bias, when making predictions.</p></div><div class="section" id="sec-2"><h2 class="">Results and Discussion</h2><div id="sec-3" class="subsection"><h3>Predictors of viral sharing</h3><p id="p-8">We fitted a model designed to partition the contribution of species-level effects and pairwise similarity measures to mammalian viral sharing probability. We used a published database of 1920 mammal-virus associations (excluding humans) as a training dataset<sup><a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">5</a></sup>. These data included 591 wild mammal species, equalling 174345 pairwise host species combinations, with 6.4% connectance – that is, 6.4% of species pairs shared at least one virus. We used a generalised additive mixed model (GAMM) framework, including a species-level effect in our model as a multi-membership random effect, capturing variation in each species’ connectedness and underlying viral diversity (see Methods). Overall, our model accounted for 44.8% of the total deviance in pairwise viral sharing, with 51.1% of this explained deviance attributable to the identities of the species involved (i.e., the species-level effect). Our model structure was effective at controlling for species-level variation in our dataset: i.e., the term had a strong impact on the centrality of each species when we simulated networks using just these parameters (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure SI1</a>). This observation suggests that ~50% of the dyadic structure of observed viral sharing networks (in contrast to the true underlying network) is determined by uneven sampling and concentration on specific species, and the remainder by macroecological processes.</p><p id="p-9">As expected, increasing host phylogenetic similarity and geographic overlap were associated with increased probability of viral sharing across mammals, together accounting for the remaining 49% of explained model deviance (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A-C</a>). Geography, phylogeny, and their interaction all showed strong nonlinear effects, with geographic overlap in particular driving a rapid increase in viral sharing that began at ~0-5% range overlap values, peaked at 50% overlap values, and then levelled off (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). This effect closely mirrors previous observations of strong, nonlinear effects of geographic and phylogenetic similarity determining within-order viral sharing<sup><a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">16</a>–<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a></sup>. Although occupying little of the visual space within the model presentation, 93% of mammal pairs had less than 5% spatial overlap (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B,D</a>). The great majority (86%) of mammal pairs in our dataset did not overlap geographically and rarely shared viruses unless phylogenetic similarity exceeded ~0.5 (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>). This phylogenetic distance corresponds roughly to order-level similarity; that is, if two species did not overlap in space, it was highly unlikely that they shared a virus unless they were within the same taxonomic order (8% of pairs). Notably, phylogenetic similarity accounted for more than twice as much model deviance as did spatial overlap (33.8% vs 14.4%). The greater importance of phylogeny relative to geography contrasts with previous analyses concerning viral sharing in primates<sup><a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">19</a></sup> and ungulates<sup><a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">16</a></sup>, likely reflecting the wider phylogenetic range of hosts considered here. This finding supports the important role of mammalian evolutionary history in shaping contemporary patterns of viral sharing and diversity<sup><a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Viral sharing GAMM model outputs and data distribution. A: predicted viral sharing probability increases with increasing phylogenetic relatedness; the different coloured lines represent different geographic overlap values. B: predicted viral sharing probability increases with increasing geographic overlap; the different coloured lines represent different phylogenetic relatedness values. C: the geographic overlap:phylogenetic similarity interaction surface, where the darker colours represent increased probability of viral sharing. White contour lines denote 10% increments of sharing probability. Labels have been removed from some contours to avoid overplotting. D: hexagonal bin chart displaying the data distribution, which was highly aggregated at low values of phylogenetic similarity and especially of geographic overlap." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1692273685" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Viral sharing GAMM model outputs and data distribution. A: predicted viral sharing probability increases with increasing phylogenetic relatedness; the different coloured lines represent different geographic overlap values. B: predicted viral sharing probability increases with increasing geographic overlap; the different coloured lines represent different phylogenetic relatedness values. C: the geographic overlap:phylogenetic similarity interaction surface, where the darker colours represent increased probability of viral sharing. White contour lines denote 10% increments of sharing probability. Labels have been removed from some contours to avoid overplotting. D: hexagonal bin chart displaying the data distribution, which was highly aggregated at low values of phylogenetic similarity and especially of geographic overlap.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F1.medium.gif" width="440" height="437"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F1.medium.gif" width="440" height="437"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-10" class="first-child">Viral sharing GAMM model outputs and data distribution. A: predicted viral sharing probability increases with increasing phylogenetic relatedness; the different coloured lines represent different geographic overlap values. B: predicted viral sharing probability increases with increasing geographic overlap; the different coloured lines represent different phylogenetic relatedness values. C: the geographic overlap:phylogenetic similarity interaction surface, where the darker colours represent increased probability of viral sharing. White contour lines denote 10% increments of sharing probability. Labels have been removed from some contours to avoid overplotting. D: hexagonal bin chart displaying the data distribution, which was highly aggregated at low values of phylogenetic similarity and especially of geographic overlap.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">In contrast to geography and phylogeny, minimum citation count and domestication status accounted for a vanishingly small amount of the deviance in viral sharing probability (0.2% and 0.1%, respectively) even though they have important effects on observed viral diversity in this dataset<sup><a id="xref-ref-5-7" class="xref-bibr" href="#ref-5">5</a></sup>. Their impacts on viral sharing may have been largely accounted for by species-level random effects.</p><p id="p-12">Our use of a pan-mammalian viral sharing dataset with a large sample size allowed us to investigate how geographic overlap and phylogenetic similarity affect viral sharing across different viral subgroups. These subgroups included RNA viruses, vector-borne RNA viruses, non-vector-borne RNA viruses, and DNA viruses. The importance of geographic overlap varied widely across all groups of viruses (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure SI2</a>; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table SI1</a>), while the influence of host phylogenetic relatedness was more consistent (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure SI3</a>; <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table SI1</a>). Generally, host phylogeny was more important in determining sharing of DNA viruses than it was for RNA viruses, while space sharing was more important for vector-borne RNA viruses, and less so for non-vector-borne RNA viruses. These results likely reflect important aspects of viral ecology, transmission, and evolution: for example, RNA viruses are fast-evolving, allowing them to more quickly adapt to novel hosts, such that phylogenetic distances are less important in determining viral sharing patterns<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup>. Conversely, DNA viruses are more evolutionarily constrained, with an evolutionary rate typically &lt;1% that of RNA viruses, such that phylogenetic distance between hosts presents a more significant obstacle for sharing of DNA viruses<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup>. The profound importance of geographic overlap in shaping the viral sharing network for vector-borne RNA viruses (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure SI3</a>) likely emerges from the geographic distributions and ecological constraints placed on vectors, lending further support to efforts to model the global spread of arboviruses by predicting changes in their vectors’ distributions and ecological niches<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>,<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>. Generally, the fact that viral sharing across different viral subgroups was predicted by different macroecological relationships suggests they should be examined separately in future analyses where possible.</p></div><div id="sec-4" class="subsection"><h3>Predicting pan-mammalian viral sharing</h3><p id="p-13">Previous trait-based approaches to predict viral sharing and reservoir hosts have been hindered by incomplete and inconsistent characterization of traits central to those modelling efforts. In contrast, spatial distributions and phylogenetic data are readily available and uniformly quantified for the vast majority of mammals and, as we have shown, are reliable predictors of viral sharing (&gt;20% of total deviance). Thus, we used our GAMM estimates to predict unobserved global viral sharing patterns across 8.8 million mammal-mammal pairs using a database of geographic distributions<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup> and a recent mammalian supertree<sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup> (see Methods). The predicted network included 4196 (non-human) Eutherian mammals with available data, 591 of which were recorded with viral associations in our training data. We calculated each species’ predicted degree centrality, as a simple and interpretable network-derived measure of viral sharing: that is, the number of other mammal species a given mammal species is expected to share at least one virus with. We identified geographic and taxonomic trends in degree centrality, validated our predicted sharing network using an external dataset, and simulated reservoir identification to assess host predictability for focal viruses (see Methods).</p><p id="p-14">We confirmed that our modelled network recapitulated expected patterns of viral sharing using the Enhanced Infectious Diseases Database (EID2) as an external dataset<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>. This dataset was constructed by mining web-based sequence data to identify host-pathogen associations, many of which are mammal-virus interactions<sup><a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a></sup>. Pairs of species that share viruses in EID2, but which were not in our training dataset (see Methods), had a much higher mean sharing probability in our predicted network (20% versus 5%; <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). In addition, more central species in the predicted network were more likely to have been observed with a virus, whether zoonotic (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>) or non-zoonotic (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>), implying that the predicted network accurately captured realised potential for viral sharing and zoonotic spillover. This finding concurs with similar work in primates which demonstrated that high centrality in primate-parasite networks is associated with carriage of zoonoses<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup>. We corroborate these findings considering all mammal-mammal viral sharing links, not just zoonotic links, and show that for each mammalian order, species with higher degree centrality in our predicted network are more likely to have been observed with viruses in the EID2 dataset (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>; <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure SI4</a>). It is possible that species with higher centrality in the global viral sharing network are more important for viral sharing, and thus have been more likely to be observed with a (zoonotic) virus. Species that are more central in our predicted network could therefore be prioritised for zoonotic surveillance or sampling in the event of viral outbreaks with unknown mammalian origins. Given that mammal diversity predicts patterns of livestock disease<sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup> and zoonoses<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>, the geographic patterns of degree centrality predicted here (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure SI5</a>; see below) could also be used as a coarse predictor of viral disease risk to livestock and human health, providing additional insights that emerge from the joint, nonlinear effects of geography and phylogeny as opposed to examination of their effects in isolation. Similarly, where there is limited knowledge of mammalian host range for newly-discovered viruses, our modelled network can be used to prioritise the sampling of additional species for viral surveillance.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The modelled mammalian viral sharing network predicts observed viral sharing trends in an independent dataset. In all figures, points are jittered along the x axis according to a density function; the black points and associated error bars are means +/&#x2212; standard errors. A: species pairs with higher predicted viral sharing probability from our model were more likely to be observed sharing a virus in the independent EID2 dataset. This comparison excludes species pairs that were also present in our training data. B: species that hosted a zoonotic virus in our dataset had more viral sharing links in the predicted all-mammal network than those without zoonotic viruses. C: species that had never been observed with a virus have fewer links in the predicted network than species that hosted viruses in the EID2 dataset only, in our training data only, or in both. The y axis represents viral sharing link number, scaled to have a mean of 0 and a standard deviation of 1 within each order for clarity. Figure SI4 displays these same data without the within-order scaling." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1692273685" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The modelled mammalian viral sharing network predicts observed viral sharing trends in an independent dataset. In all figures, points are jittered along the x axis according to a density function; the black points and associated error bars are means +/&#x2212; standard errors. A: species pairs with higher predicted viral sharing probability from our model were more likely to be observed sharing a virus in the independent EID2 dataset. This comparison excludes species pairs that were also present in our training data. B: species that hosted a zoonotic virus in our dataset had more viral sharing links in the predicted all-mammal network than those without zoonotic viruses. C: species that had never been observed with a virus have fewer links in the predicted network than species that hosted viruses in the EID2 dataset only, in our training data only, or in both. The y axis represents viral sharing link number, scaled to have a mean of 0 and a standard deviation of 1 within each order for clarity. Figure SI4 displays these same data without the within-order scaling.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F2.medium.gif" width="440" height="437"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F2.medium.gif" width="440" height="437"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-15" class="first-child">The modelled mammalian viral sharing network predicts observed viral sharing trends in an independent dataset. In all figures, points are jittered along the x axis according to a density function; the black points and associated error bars are means +/− standard errors. A: species pairs with higher predicted viral sharing probability from our model were more likely to be observed sharing a virus in the independent EID2 dataset. This comparison excludes species pairs that were also present in our training data. B: species that hosted a zoonotic virus in our dataset had more viral sharing links in the predicted all-mammal network than those without zoonotic viruses. C: species that had never been observed with a virus have fewer links in the predicted network than species that hosted viruses in the EID2 dataset only, in our training data only, or in both. The y axis represents viral sharing link number, scaled to have a mean of 0 and a standard deviation of 1 within each order for clarity. <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure SI4</a> displays these same data without the within-order scaling.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Taxonomic and geographic patterns of mean predicted viral sharing link numbers (degree centrality). Top row: all viral sharing links; middle row: viral sharing links with species in the same order; bottom row: viral sharing links with species in another order. A,C,E: average species-level viral sharing link numbers for mammalian orders in our dataset. Bars represent means; error bars represent standard errors. B,D,F: geographic distributions of mean viral sharing link numbers. Distributions were derived by summing the viral sharing link numbers of all species inhabiting a 25km2 grid square and dividing them by the number of species inhabiting the grid square, giving mean degree number at the grid level." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1692273685" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Taxonomic and geographic patterns of mean predicted viral sharing link numbers (degree centrality). Top row: all viral sharing links; middle row: viral sharing links with species in the same order; bottom row: viral sharing links with species in another order. A,C,E: average species-level viral sharing link numbers for mammalian orders in our dataset. Bars represent means; error bars represent standard errors. B,D,F: geographic distributions of mean viral sharing link numbers. Distributions were derived by summing the viral sharing link numbers of all species inhabiting a 25km&lt;sup&gt;2&lt;/sup&gt; grid square and dividing them by the number of species inhabiting the grid square, giving mean degree number at the grid level.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F3.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F3.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-16" class="first-child">Taxonomic and geographic patterns of mean predicted viral sharing link numbers (degree centrality). Top row: all viral sharing links; middle row: viral sharing links with species in the same order; bottom row: viral sharing links with species in another order. A,C,E: average species-level viral sharing link numbers for mammalian orders in our dataset. Bars represent means; error bars represent standard errors. B,D,F: geographic distributions of mean viral sharing link numbers. Distributions were derived by summing the viral sharing link numbers of all species inhabiting a 25km<sup>2</sup> grid square and dividing them by the number of species inhabiting the grid square, giving mean degree number at the grid level.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">The high predicted centrality of known hosts may be due partly to selective sampling (i.e., viral researchers are more likely to sample wide-ranging and common host species that also share viruses with many other species<sup><a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a></sup>). This possibility is supported by the increased degree centrality for species that appear in both EID2 and our dataset rather than in only one of the two, as these species are presumably more well-known (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>). Similarly, while we believe that our model was successful at accounting for variation in host-level diversity and study effort that influences network topology (see above; <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure SI1</a>), there are certain inherent biases in the training data which must be considered when interpreting our findings. Most notably, viral sharing estimates in our dataset may be affected by the fact that zoonotic discovery efforts commonly search limited geographic regions for a specific virus or group of viruses, artificially increasing the likelihood of detecting these viruses in the same region compared to a geographically random sampling regime. Moreover, when a mammal species (e.g., a bat) is found with a focal virus (e.g., an ebolavirus), it is logical for researchers to then investigate similar, closely related species in nearby locales<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup>. These sampling approaches could disproportionately weight the network towards finding phylogeographic effects on viral sharing probability. However, it is highly encouraging that our model predicted patterns in the external EID2 dataset, which was constructed using different data compilation methods but also comprises global data covering several decades of research<sup><a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">29</a></sup>. In sum, we believe that our approach is a conservative method for minimising the biases inherent in the data. The knowledge that the observed mammalian virome is biased ultimately calls for more uniform viral sampling across the mammal class and increased coverage of rarely-sampled groups, lending support to ongoing efforts to systematically catalogue mammalian viral diversity<sup><a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a></sup>.</p></div><div id="sec-5" class="subsection"><h3>Taxonomic and geographic patterns of predicted viral sharing</h3><p id="p-18">Our network predicted strong taxonomic patterns in the probability of viral sharing. Looking across mammalian orders, rodents (Rodentia) and bats (Chiroptera) had the most predicted species-level viral links, while carnivores and artiodactyl ungulates had substantially fewer (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). Examining multiple mammalian orders allowed us to partition the predicted sharing network into within- and between-order links to investigate whether certain orders are better-connected to other orders. Indeed, this partitioning revealed differences in taxonomic and geographic patterns of viral sharing. In bats and rodents, large numbers of within-order links are driven by high within-order species diversity (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). Interestingly, when within-order links were ignored, leaving only out-of-order links, rodents and bats were among the least-connected Eutherian orders (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3E</a>), while even-toed ungulates and carnivores were ranked among the most-connected (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3E</a>). Taken together, these results imply that while bats and rodents are important in viral sharing networks, their sharing is mainly restricted to other bats and rodents, respectively. This distinction only applied to mean link numbers; when link numbers were summed, rodents and bats remained highly connected regardless of which metric was used, as a result of their species richness (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure SI5</a>).</p><p id="p-19">Previous analyses have demonstrated that both bats and rodents are important for hosting zoonotic viruses, with possible explanations including species-level phenotypic traits such as behaviour<sup><a id="xref-ref-5-8" class="xref-bibr" href="#ref-5">5</a></sup>, life history<sup><a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a></sup>, or metabolic idiosyncracies<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>. Our results imply that while both orders potentially host many zoonoses purely as a result of their species richness (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure SI5</a>), the vast majority of their viral sharing occurs within-order even though larger phylogenetic jumps are necessary for spillover. Intriguingly, recent work has shown that infection of an aggregated phylogenetic selection of hosts is an important contributor to viral zoonotic potential<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>. Rodents’ and bats’ tendency towards high viral interconnectedness could encourage viruses to achieve such aggregation, leading to opportunities for spillover into humans. In our analysis, both orders’ high centrality emerged purely as a result of their phylogenetic diversity and geographic distributions, rather than from other phenotypic traits. If well-connected species in our network are more likely to maintain a high diversity of viruses (e.g., via multi-host dynamics leading to an expanded threshold population size<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup>), this may contribute to the high viral diversity documented in bats and rodents<sup><a id="xref-ref-5-9" class="xref-bibr" href="#ref-5">5</a></sup>. Efforts to prioritise viral sampling regimes should consider biogeography and mammal-mammal interactions in addition to searching for species-level traits associated with high viral diversity.</p><p id="p-20">Encouragingly, our network showed predictable scaling laws similar to those of other known ecological networks<sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup>. Viral link numbers in within-order subnetworks (e.g., between different bat species) correlated strongly with species diversity within each order (R<sup>2</sup>=~0.85), following a power law with a Z value of ~0.8 (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure SI6</a>). Similarly, out-of-order links (e.g., between a bat and a rodent) scaled linearly with the product of the species richness of both orders (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure SI7</a>).</p><p id="p-21">To visualize geographic patterns of viral sharing, we projected species-level degree centrality across the species’ ranges then calculated grid cell-level mean degree centrality (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3B</a>), as well as summed degree centrality (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure SI5</a>). Average centrality peaked in tropical areas of South and Central America, Sub-Saharan Africa, and Southeast Asia, especially in the Andes and Himalayas (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3B</a>). These patterns align with previously-reported hotspots of emerging zoonoses and predicted viral diversity<sup><a id="xref-ref-5-10" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a></sup> and imply that areas of high biodiversity are centres of viral sharing not just because of the number of overlapping species (i.e., high species richness), but also because more closely related species create a more connected viral sharing network in these areas. This densely-connected network structure and the increased biomass present in the tropics might have synergistic implications for cross-species maintenance and transmission of viral diversity in these areas. The geographic distributions of mean predicted within- and between-order viral links differed notably from the distribution of interspecific links generally: the relative importance of South America and East Asia was higher for within-order links (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3D</a>), while Sub-Saharan Africa remained a hotspot for out-of-order links (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3F</a>). Geographic patterns of summed link numbers more closely mirrored underlying host species richness, whether for all links, within-order links, or out-of-order links (<a id="xref-fig-8-5" class="xref-fig" href="#F8">Figure SI5</a>).</p><p id="p-22">We acknowledge that our phylogeographic model of viral sharing does not account for complex ecological interactions such as coinfection or coevolution, which could impact how patterns of exposure and host susceptibility translate to realised viral diversity. Future investigations could extend our framework to simulate the dynamic co-speciation of mammals and their viruses in order to account for these processes and/or to explicitly investigate how viral sharing connectivity and viral diversity are correlated across mammal species. Our model may also prove useful for building and parameterising much-needed multi-host network models for conservation purposes, particularly where there is scarce prior information on interspecific pathogen sharing<sup><a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>,<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup>.</p></div><div id="sec-6" class="subsection"><h3>The network as a predictive tool</h3><p id="p-23">Identifying potential hosts for known and novel viruses is an important component of preemptive zoonotic disease surveillance that can speed public health responses. Predictive techniques based on species-level phenotypic and genomic data have been suggested to help prioritise sampling targets<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>,<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">9</a></sup>. Although these approaches represent a promising methodological advance, they may not elucidate the mechanistic underpinnings of viral host range, reducing their potential efficacy for guiding public health interventions. In addition, genomic approaches require viral sequence data, which can be time-consuming and operationally challenging to acquire or share publicly. We therefore interrogated our predicted viral sharing network to investigate whether it could be used to identify potential hosts of known viruses at the species level. Using a leave-one-out prediction process (see Methods), our model showed a surprisingly strong ability to predict observed host species for 250 viruses with at least two known (non-human) mammal hosts.</p><p id="p-24">We investigated the predictive potential of our model by iteratively selecting all but one of the known hosts for a given virus, then using the predicted sharing patterns of the remaining hosts to identify how the focal (removed) host was ranked in terms of its sharing probability. In practical terms, these species-level rankings could set sampling priorities for public health efforts seeking to identify hosts of a novel zoonotic virus, where one or more hosts are already known. Across all 250 viruses, the median ranking of the left-out host was 72 out of a potential 4196 mammals (i.e., in the top 1.7% of potential hosts). To compare this ranking to alternative heuristics, we examined how high the focal host would be ranked using simple ranked phylogenetic relatedness or spatial overlap values alone (i.e., the most closely-related, followed by the second-most-related, etc.). Using this method, the focal host was ranked 288th (for phylogeny) or 283rd (for space), identifying the focal host in the top 7% of potential hosts and demonstrating that sampling prioritization schemes based on our phylogeographic model would require only ¼ as many sampling targets in order to identify the correct sharing host. Our model therefore represents a substantial improvement over search methods that involve only spatial or phylogenetic similarity. Our model performed similarly at identifying focal hosts in the EID2 dataset<sup><a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">29</a></sup>: for the 109 viruses in the EID2 dataset with more than one host, the focal host was identified in the top 63 (1.5%) potential hosts. In contrast, ranked spatial overlap predicted the focal host in the top 560 hosts, and phylogenetic relatedness in the top 174.</p><p id="p-25">We observed substantial variation in our model’s ability to predict known hosts among different viruses. For example, the correct host was predicted first in every iteration for 7 viruses and in the top 10 hosts for 42 viruses. Results for 128 viruses had the focal host falling within the top 100 guesses, and for only 6 viruses were the model-based host searches worse than chance (focal host ranked lower than 50% of all mammals in terms of sharing probability). We used this measure of viral sharing “predictability” to investigate whether certain viral traits affected the ease with which phylogeography predicted their hosts. Viruses with broad host phylogenetic ranges, most notably Ebola virus, challenge reservoir prediction efforts since many more species must often be sampled before identifying the correct host(s). To investigate whether the predictive strength of our model was limited for viruses with broad host ranges and/or other viral traits, we fitted a linear mixed model (LMM) which showed a strong negative association between viruses’ known phylogenetic host breadth and the predictability of focal hosts (model R<sup>2</sup>=0.70; host breadth R<sup>2</sup>=0.67; <a id="xref-fig-12-1" class="xref-fig" href="#F12">Figure SI9</a>). This association demonstrates, unsurprisingly, that predicting the hosts of generalist viruses is intrinsically difficult using our method. This adds a potential limitation to the applicability of our network approach, given that zoonotic viruses commonly exhibit wide host ranges<sup><a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-5-11" class="xref-bibr" href="#ref-5">5</a></sup>. A family-level random effect accounted for little of the apparent variance in predictability among viral families (<a id="xref-fig-11-1" class="xref-fig" href="#F11">Figure SI8</a>).</p><p id="p-26">Once viral host range was accounted for, hosts of vector-borne viruses were slightly easier to predict than non-vector-borne viruses (R<sup>2</sup>=0.1; <a id="xref-fig-12-2" class="xref-fig" href="#F12">Figure SI9</a>) – perhaps because the sharing of vector-borne viruses depends more heavily on host geographic distributions (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure SI3</a>). Despite additional variation in the data, no other viral traits (e.g., RNA vs. DNA, segmented vs. non-segmented) were important in the LMM. This implies that host phylogeographic traits are a good broad-scale indicator of viral sharing, particularly when ecological specifics of the virus itself are unknown.</p></div></div><div class="section" id="sec-7"><h2 class="">Conclusion</h2><p id="p-27">In summary, we present a simple, highly interpretable model that predicted a substantial proportion of viral sharing across mammals and is capable of identifying species-level sampling priorities for viral surveillance and discovery. It is worth noting that the analytical framework and validation we describe were conducted on a global scale, while many zoonotic sampling efforts occur on a national or regional scale. Restricting the focal mammals to a regional pool may improve the applicability of our model in certain sampling contexts, and future studies could leverage higher-resolution phylogenetic and geographic data to fine-tune predictions. In particular, the mammalian supertree<sup><a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a></sup> has relatively poor resolution at the species tips such that relatedness estimates based on alternative molecular evidence (e.g., full host genome data) may allow more precise estimates of the phylogenetic relatedness effect on viral sharing. Alternatively, our model could be augmented with additional host, virus, and pairwise traits, using similar pairwise formulations of viral sharing as a response variable, to identify ecological specificities that are critical for the transmission of certain viruses, to partition viral subtypes, and, ultimately, to increase the accuracy of host prediction. By generalising the spatial and phylogenetic processes that drive viral sharing, our model serves as a useful guide for the prioritization of viral sampling, presenting a baseline for future modelling efforts to compare against and improve upon.</p><p id="p-28">Our ability to model and predict macroecological patterns of viral sharing is important in an era of rapid global change. Under all conceivable global change scenarios, many mammals will shift their geographic ranges, whether of their own volition or through human assistance. Mammalian parasite communities will likely undergo considerable rearrangement as a result, with potentially far-reaching ecological consequences<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>–<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup>. Our findings suggest that novel species encounters will provide opportunities for interspecific viral transmission, which could be facilitated by even relatively small changes in range overlap. These future cross-species transmission events will have profound implications for conservation and public health, potentially devastating populations of host species without evolved resistance to the novel viruses (e.g., red squirrel declines brought about by parapoxvirus infections spread by introduced grey squirrels<sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup>) or increasing zoonotic disease risk by introducing viruses to human-adjacent amplifier hosts (e.g., horses increasing the risk of human infection with Hendra virus<sup><a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20</a></sup>). Thus, our global model of mammalian viral sharing provides a crucial complement to ongoing work modelling the spread of hosts, vectors, and their associated diseases as the result of climate change-induced range expansions<sup><a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>,<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a></sup>.</p></div><div class="section" id="sec-8"><h2 class="">Author contributions</h2><p id="p-30">GFA, EAE, NR, and KJO designed the study together. GFA conducted the analyses under NR’s supervision. GFA wrote the manuscript, while EAE, NR, and KJO offered comments and edits to the manuscript throughout.</p></div><div class="section" id="sec-9"><h2 class="">Methods</h2><div class="section" id="sec-10"><h3>Making the training data network</h3><p id="p-31">Code and data for all analyses are available at <a href="https://github.com/gfalbery/ViralSharingPhylogeography">https://github.com/gfalbery/ViralSharingPhylogeography</a>. Our dataset included 1920 mammal-virus associations obtained from an exhaustive literature search which has been used to investigate how species traits influence mammalian viral diversity<sup><a id="xref-ref-5-12" class="xref-bibr" href="#ref-5">5</a></sup>. We removed humans and rabies virus from the dataset as both were disproportionately well-connected, and we removed 20 non-Eutherian mammals because they were extreme phylogenetic outliers, leaving 591 Eutherian mammals that shared 401 viruses. We made an unweighted bipartite network using the mammal-virus associations and projected the unipartite mammal-mammal network, which we then converted into a sequence of all unique mammal-mammal pairs where 1/0 denoted whether the pair of species shared a virus or not. This comprised only the lower triangle of the adjacency matrix to avoid duplicating associations and to remove self-connections, and only included mammals with at least one sharing link (final N=174345 unique mammal-mammal pairs). 6.4% of these pairs shared at least one virus.</p><p id="p-32">All analyses were performed in R version 3.6.0<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup>. Phylogenetic similarity was calculated using a mammalian supertree<sup><a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">28</a></sup> as previously described<sup><a id="xref-ref-5-13" class="xref-bibr" href="#ref-5">5</a></sup>. Pairwise phylogenetic distances were defined as the cumulative branch length between the two species and were scaled to between 0 and 1, and subtracted from 1 to give a measure of relative phylogenetic similarity (rather than distance). Of the 4716 Eutherian species in the mammalian supertree, 591 had virus association records in our fully-connected network and 4196 had known geographic ranges. We used IUCN species ranges to quantify species’ geographic distributions<sup><a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a></sup>. These range maps are generated based on expert knowledge and only comprise species presence/absence information rather than density. We converted all range polygons to 25 km<sup>2</sup> raster grids. For each species-pair, we quantified range overlap as the number of raster grid squares jointly inhabited by the two species (in the Mollweide projection, which exhibits equal grid size), divided by the total number of grid squares occupied by these species combined, so that each value was scaled from 0-1: overlap<sub>A,B</sub>=grid<sub>A,B</sub>/(grid<sub>A</sub>+grid<sub>B</sub>-grid<sub>A,B</sub>). Disease-related research effort for each host species was quantified as previously described, using counts of studies including species names and disease-related terms such as “virus,” “pathogen”, or “parasite<sup><a id="xref-ref-5-14" class="xref-bibr" href="#ref-5">5</a></sup>. To fit citation number as a pairwise trait, we took the smaller of a pair of species’ respective citations, and log-transformed the value. Domestication status was defined <em>sensu lato</em>, again as previously described<sup><a id="xref-ref-5-15" class="xref-bibr" href="#ref-5">5</a></sup>, based on whether a species was ever seen in a domestic setting. We fit this as a binary pairwise trait where 1=at least one of the species was domesticated and 0=neither species had been domesticated.</p></div><div class="section" id="sec-11"><h3>Model formulation</h3><p id="p-33">We fitted a Generalised Additive Mixed Model (GAMM) to examine which traits influenced viral sharing among mammal pairs using accelerated discretized implementation in the <strong>mgcv</strong> package<sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup>. We fitted viral sharing (0/1) as the response variable, with a binomial family specification. The model had the following structure:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/24/732255/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/24/732255/embed/graphic-4.gif"/></noscript></span>

</span></p><p id="p-34">The first term (“s”) represents a phylogeny effect smooth fitted across species pairs that did not overlap in space (Gz=1), and “t2” represents a phylogeny:geography tensor product smooth fitted to species that had geographic overlap greater than zero (Gz=0). This allowed us to model these two aspects of the data separately, helping us to more effectively model the large number of spatial zeroes (85% of species pairs did not overlap in space). “mm” represents a multi-membership random effect, accounting for the identity of both species in the pair. We implemented this multi-membership effect to control for species-level effects by including a species-level effect for both the row (Species 1) and column (Species 2) of the sharing matrix. Using the paraPen specification in <strong>mgcv</strong>, these random effects were constrained to sample from the same distribution, resulting in a single estimate of the variance associated with each unique species. Most precisely, these effects in our model help capture variation in viral sharing that could likely be explained by species-level factors that are unobserved or otherwise excluded (i.e., differences in underlying viral diversity, which would be expected to positively impact the probability of interspecific sharing). In sum, this model formulation allowed us to estimate the effect of pairwise predictors (geographic overlap, phylogenetic similarity) in determining viral sharing as well as evaluate the influence of species identity.</p><p id="p-35">To investigate whether the effects of geography and phylogeny depended on which subset of viruses we investigated, we fit the model to non-exclusive subnetworks of mammal-mammal pairs based on the types of viruses they were connected by. Viral subtypes included RNA viruses (566 hosts sharing 381 viruses); vector-borne RNA viruses (333 hosts sharing 164 viruses); non-vector-borne RNA viruses (391 hosts sharing 205 viruses); and DNA viruses (151 hosts sharing 205 viruses). There were only 2 vector-borne DNA viruses in our data. We eliminated from each analysis any hosts that were not carrying the focal virus type.</p><p id="p-36">We elected to use a binary model of viral sharing (0/1) rather than an integer count model (0+) for two reasons: first, the data distribution was highly skewed, with few very large values and many zeroes. Under these conditions, we found a count-based model formulation including species-level random effects computationally intractable. Second, the observed viral diversity is likely a considerable underestimate, but the extent of this underestimate is a matter of hot debate<sup><a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a></sup>. As such, the predictions for viral sharing from such a model could be relative and biased, while binary models offer a more appropriate resolution to quantify sharing patterns. We wished to avoid estimating a precise number of viruses shared among pairs of species for this reason.</p></div><div class="section" id="sec-12"><h3>Model validation</h3><p id="p-37">To check the fit of the model, we predicted 0/1 viral sharing values from the model 1000 times and examined how the values compared to the proportions of 0’s and 1’s in the observed data, finding high agreement between the two. We repeated this procedure using a) the full dataset; b) only the fixed effects, with random effects randomised in each iteration; and c) only the random effects, with fixed effects held at the mean. We then used these predicted links to create 1000 unipartite viral sharing networks, estimating link numbers (degree centrality) for species in each replicated network. We took the mean of these values across the 1000 replicated networks to give the predicted values displayed in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure SI1</a>.</p><p id="p-38">We quantified deviance contributions of our explanatory variables by calculating model deviance when dropping each variable, and comparing these against the full model and an intercept-only model deviance. For each of our explanatory variables (geographic overlap, phylogenetic similarity, minimum citation number, domestication status, and species-level random effects) we randomised the observed values 1000 times, then predicted sharing probabilities for these values using our model estimates. This randomisation procedure allowed us to predict while accounting for the uneven data distribution, rather than using mean values.</p></div><div class="section" id="sec-13"><h3>Simulating viral sharing networks</h3><p id="p-39">Following reconstruction of the observed network as part of our model validation, we repeated the prediction process on an exhaustive mammal dataset to estimate viral sharing across all mammals. We set minimum citation number to the data mean, and set domestication status to 0. We repeated the predictions 1000 times, randomising the species-level random effects each time. The full prediction dataset included 4196 Eutherian mammals with known spatial distributions and phylogenetic associations, resulting in 8.8 million unique pairwise combinations. After predicting 1000 binary sharing networks across all mammals, we summarised the average predicted link number (degree centrality) of each species across the 1000 replicates. We then calculated the mean species-level link number within each mammalian order to examine taxonomic patterns. To project the spatial patterns of connectedness, we assigned each species range polygon the link number (degree centrality) of its host species<sup><a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">27</a></sup> and took the mean value for each grid square, thereby correcting for species richness. We then repeated these taxonomic and geographic summaries using within-order and between-order link numbers separately. We also took the summed values, which more closely reflect underlying patterns of species richness.</p><p id="p-40">We validated the predicted network by comparing it to sharing patterns in the Enhanced Infectious Diseases Database (EID2)<sup><a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">29</a></sup>. We eliminated species pairs that were in our training data and identified whether species pairs that shared viruses in EID2 were more likely to share viruses in our predicted network than species pairs that did not. In addition, we investigated whether species that were shown to host zoonoses in our training dataset were more highly-connected in the predicted network. Finally, we investigated whether species that were present in only EID2, in only our training data, or in both were more highly-connected in our predicted network than species that did not appear in either dataset and were therefore taken to have not been observed hosting a virus.</p></div><div class="section" id="sec-14"><h3>Predicting hosts of focal viruses</h3><p id="p-41">To investigate the ability of the model to predict known hosts of viruses in our dataset, we iteratively investigated the sharing patterns of known hosts independently for all viruses with &gt;1 host. For each virus, we removed one host at a time, and then investigated which species the remaining known host species were likely to share viruses with based on the all-mammal predicted network. If the removed host (“focal host”) was on average highly likely to share viruses with the remaining species, our model was taken to be useful for predicting patterns of mammal sharing based on known host distributions. The mean ranking of the focal hosts across each prediction iteration was used as a measure of “predictability” for each virus. We carried out this process for the 250 viruses with more than one known host with associated geographic and phylogenetic data and then on the 109 such viruses in the EID2 data.</p><p id="p-42">Once the predictability of each virus was calculated, we fitted a linear mixed model examining log10(mean focal host rank) as an inverse measure of predictability (higher rank corresponds to decreased predictability) for each virus. We added mean phylogenetic host similarity as a fixed effect and viral family as a random effect to quantify how viral phylogeny affected predictability. We included additional viral traits in the model, including: cytoplasmic replication (0/1); segmentation (0/1); vector-borne transmission (0/1); double- or single-strandedness; DNA or RNA; enveloped or non-enveloped; or zoonotic ability (0/1 for whether the virus was associated with humans in our dataset).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predicted degree centrality of species in our training data network, predicted using our GAMM estimates. Fixed + random effects were very effective at reproducing individual species&#x2019; degree centrality (left); fixed effects were less effective (middle); and random effects alone had a strong but imperfect effect (right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1692273685" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Predicted degree centrality of species in our training data network, predicted using our GAMM estimates. Fixed + random effects were very effective at reproducing individual species&#x2019; degree centrality (left); fixed effects were less effective (middle); and random effects alone had a strong but imperfect effect (right).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure SI1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F4.medium.gif" width="440" height="174"/><noscript><img class="highwire-fragment fragment-image" alt="Figure SI1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F4.medium.gif" width="440" height="174"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure SI1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure SI1:</span> <p id="p-43" class="first-child">Predicted degree centrality of species in our training data network, predicted using our GAMM estimates. Fixed + random effects were very effective at reproducing individual species’ degree centrality (left); fixed effects were less effective (middle); and random effects alone had a strong but imperfect effect (right).</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="GAMM-derived viral sharing estimates for the effect of phylogenetic similarity for four viral subsets (top row: all RNA viruses and DNA viruses; bottom row: vector-borne RNA viruses and non-vector-borne RNA viruses). Each GAMM smooth is displayed at multiple geographic overlap values (different colours)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1692273685" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;GAMM-derived viral sharing estimates for the effect of phylogenetic similarity for four viral subsets (top row: all RNA viruses and DNA viruses; bottom row: vector-borne RNA viruses and non-vector-borne RNA viruses). Each GAMM smooth is displayed at multiple geographic overlap values (different colours).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure SI2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F5.medium.gif" width="433" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure SI2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F5.medium.gif" width="433" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure SI2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure SI2:</span> <p id="p-44" class="first-child">GAMM-derived viral sharing estimates for the effect of phylogenetic similarity for four viral subsets (top row: all RNA viruses and DNA viruses; bottom row: vector-borne RNA viruses and non-vector-borne RNA viruses). Each GAMM smooth is displayed at multiple geographic overlap values (different colours).</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="GAMM-derived viral sharing estimates for the effect of geographic overlap for four viral subsets (top row: all RNA viruses and DNA viruses; bottom row: vector-borne RNA viruses and non-vector-borne RNA viruses). Each GAMM smooth is displayed at multiple geographic overlap values (different colours)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1692273685" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;GAMM-derived viral sharing estimates for the effect of geographic overlap for four viral subsets (top row: all RNA viruses and DNA viruses; bottom row: vector-borne RNA viruses and non-vector-borne RNA viruses). Each GAMM smooth is displayed at multiple geographic overlap values (different colours).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure SI3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F6.medium.gif" width="433" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure SI3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F6.medium.gif" width="433" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure SI3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure SI3:</span> <p id="p-45" class="first-child">GAMM-derived viral sharing estimates for the effect of geographic overlap for four viral subsets (top row: all RNA viruses and DNA viruses; bottom row: vector-borne RNA viruses and non-vector-borne RNA viruses). Each GAMM smooth is displayed at multiple geographic overlap values (different colours).</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1126827/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1126827/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1126827" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table SI1:</span> <p id="p-46" class="first-child">The deviance contributions and sample sizes (number of hosts) for each of our viral sharing GAMMs. The deviance terms are, in order: proportional geographic overlap; binary geographic overlap greater than zero (0/1); phylogenetic similarity; minimum citation number; domestication status; and the species-level random effect.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mammal species that were observed with at least one virus in the training dataset or the EID2 dataset had higher degree centrality (link number) in our predicted network. This figure displays the raw data that are displayed in Figure 2C in the main text, but without being scaled within orders." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1692273685" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Mammal species that were observed with at least one virus in the training dataset or the EID2 dataset had higher degree centrality (link number) in our predicted network. This figure displays the raw data that are displayed in Figure 2C in the main text, but without being scaled within orders.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure SI4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F7.medium.gif" width="440" height="220"/><noscript><img class="highwire-fragment fragment-image" alt="Figure SI4:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F7.medium.gif" width="440" height="220"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure SI4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure SI4:</span> <p id="p-47" class="first-child">Mammal species that were observed with at least one virus in the training dataset or the EID2 dataset had higher degree centrality (link number) in our predicted network. This figure displays the raw data that are displayed in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2C</a> in the main text, but without being scaled within orders.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Taxonomic and geographic patterns of predicted viral link numbers. Top row: all links; middle row: links with species in the same order; bottom row: links with species in another order. A,C,E: summed species-level link numbers for mammalian orders in our dataset. B,D,F: geographic distributions of link numbers. Distributions were derived by summing the link numbers of all species inhabiting a grid square." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1692273685" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Taxonomic and geographic patterns of predicted viral link numbers. Top row: all links; middle row: links with species in the same order; bottom row: links with species in another order. A,C,E: summed species-level link numbers for mammalian orders in our dataset. B,D,F: geographic distributions of link numbers. Distributions were derived by summing the link numbers of all species inhabiting a grid square.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure SI5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F8.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure SI5:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F8.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure SI5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure SI5:</span> <p id="p-48" class="first-child">Taxonomic and geographic patterns of predicted viral link numbers. Top row: all links; middle row: links with species in the same order; bottom row: links with species in another order. A,C,E: summed species-level link numbers for mammalian orders in our dataset. B,D,F: geographic distributions of link numbers. Distributions were derived by summing the link numbers of all species inhabiting a grid square.</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scaling of degree centrality (link numbers) followed a power law when looking within-orders, but not between orders. The trend line and 95% confidence intervals are derived from a linear model fitted to the data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1692273685" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Scaling of degree centrality (link numbers) followed a power law when looking within-orders, but not between orders. The trend line and 95% confidence intervals are derived from a linear model fitted to the data.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure SI6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F9.medium.gif" width="440" height="153"/><noscript><img class="highwire-fragment fragment-image" alt="Figure SI6:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F9.medium.gif" width="440" height="153"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure SI6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure SI6:</span> <p id="p-49" class="first-child">Scaling of degree centrality (link numbers) followed a power law when looking within-orders, but not between orders. The trend line and 95% confidence intervals are derived from a linear model fitted to the data.</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predicted between-order link numbers scales according to the log-product of the number of species in the two orders. Each point represents a pair of orders (N=171)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1692273685" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Predicted between-order link numbers scales according to the log-product of the number of species in the two orders. Each point represents a pair of orders (N=171).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure SI7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F10.medium.gif" width="440" height="369"/><noscript><img class="highwire-fragment fragment-image" alt="Figure SI7:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F10.medium.gif" width="440" height="369"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure SI7:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure SI7:</span> <p id="p-50" class="first-child">Predicted between-order link numbers scales according to the log-product of the number of species in the two orders. Each point represents a pair of orders (N=171).</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The phylogeographic predictability of viruses&#x2019; reservoir hosts varied considerably across viral families, although the family-level random effect did not account for much of the model&#x2019;s variance. Families are ordered along the x axis in order of decreasing predictability. The y axis displays the mean rank of the focal host in our reservoir host prediction simulation, on a reversed log10-scale. Values closer to the top of the figure represent more predictable viruses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1692273685" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The phylogeographic predictability of viruses&#x2019; reservoir hosts varied considerably across viral families, although the family-level random effect did not account for much of the model&#x2019;s variance. Families are ordered along the x axis in order of decreasing predictability. The y axis displays the mean rank of the focal host in our reservoir host prediction simulation, on a reversed log10-scale. Values closer to the top of the figure represent more predictable viruses.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure SI8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F11.medium.gif" width="440" height="285"/><noscript><img class="highwire-fragment fragment-image" alt="Figure SI8:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F11.medium.gif" width="440" height="285"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure SI8:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure SI8:</span> <p id="p-51" class="first-child">The phylogeographic predictability of viruses’ reservoir hosts varied considerably across viral families, although the family-level random effect did not account for much of the model’s variance. Families are ordered along the x axis in order of decreasing predictability. The y axis displays the mean rank of the focal host in our reservoir host prediction simulation, on a reversed log10-scale. Values closer to the top of the figure represent more predictable viruses.</p><div class="sb-div caption-clear"></div></div></div><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Viral host range strongly impacted the predictability of reservoir hosts. The x axis displays the mean phylogenetic similarity of a virus&#x2019;s hosts (i.e., an inverse measurement of viral host range). The y axis displays the mean rank of the focal host in our reservoir host prediction simulation, on a reversed log10-scale. Values closer to the top of the figure represent more predictable viruses. The trend lines and 95% confidence intervals were derived from a linear mixed model fitted to the data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1692273685" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Viral host range strongly impacted the predictability of reservoir hosts. The x axis displays the mean phylogenetic similarity of a virus&#x2019;s hosts (i.e., an inverse measurement of viral host range). The y axis displays the mean rank of the focal host in our reservoir host prediction simulation, on a reversed log10-scale. Values closer to the top of the figure represent more predictable viruses. The trend lines and 95% confidence intervals were derived from a linear mixed model fitted to the data.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure SI9:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F12.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure SI9:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F12.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure SI9:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/24/732255/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure SI9:</span> <p id="p-52" class="first-child">Viral host range strongly impacted the predictability of reservoir hosts. The x axis displays the mean phylogenetic similarity of a virus’s hosts (i.e., an inverse measurement of viral host range). The y axis displays the mean rank of the focal host in our reservoir host prediction simulation, on a reversed log10-scale. Values closer to the top of the figure represent more predictable viruses. The trend lines and 95% confidence intervals were derived from a linear mixed model fitted to the data.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-29">This work was conducted during a placement funded by the National Environmental Research Council (NERC) Overseas Research Fund awarded to GFA. GFA’s PhD studentship was likewise funded by NERC (grant number: NE/L002558/1). EAE, NR, and KJO were funded by the generous support of the American people through the United States Agency for International Development (USAID) Emerging Pandemic Threats PREDICT project. Additional support was provided by the National Institute of Allergy and Infectious Diseases of the National Institutes of Health (Award Number R01AI110964) and the US Department of Defense, Defense Threat Reduction Agency (HDTRA11710064). We thank Colin Carlson, Verity Hill, and members of EcoHealth Alliance for advice and helpful comments on the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-3">Revisions for resubmission</p></li><li class="fn-dataset fn-group-external-links" id="fn-2"><p id="p-4">
<a href="https://github.com/gfalbery/ViralSharingPhylogeography">https://github.com/gfalbery/ViralSharingPhylogeography</a>
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.1" data-doi="10.3201/eid1112.050997"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woolhouse</span>,  <span class="cit-name-given-names">M. E. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gowtage-Sequeria</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Host range and emerging and reemerging pathogens</span>. <abbr class="cit-jnl-abbrev">Emerg. Infect. Dis</abbr>. <span class="cit-vol">11</span>, <span class="cit-fpage">1842</span>–<span class="cit-lpage">7</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DWoolhouse%26rft.auinit1%253DM.%2BE.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D1842%26rft.epage%253D1847%26rft.atitle%253DHost%2Brange%2Band%2Bemerging%2Band%2Breemerging%2Bpathogens.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1112.050997%26rft_id%253Dinfo%253Apmid%252F16485468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1112.050997&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16485468&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F24%2F732255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233768300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.2" data-doi="10.1038/srep08026"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">C. K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Spillover and pandemic properties of zoonotic viruses with high host plasticity</span>. <abbr class="cit-jnl-abbrev">Sci. Rep</abbr>. <span class="cit-vol">5</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep%26rft.volume%253D5%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep08026%26rft_id%253Dinfo%253Apmid%252F26082330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep08026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26082330&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F24%2F732255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.3" data-doi="10.1126/science.aap7463"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carroll</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The Global Virome Project</span>. <abbr class="cit-jnl-abbrev">Science (80-.)</abbr>. <span class="cit-vol">359</span>, <span class="cit-fpage">872</span>–<span class="cit-lpage">874</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCarroll%26rft.auinit1%253DD.%26rft.volume%253D359%26rft.issue%253D6378%26rft.spage%253D872%26rft.epage%253D874%26rft.atitle%253DThe%2BGlobal%2BVirome%2BProject%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aap7463%26rft_id%253Dinfo%253Apmid%252F29472471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTkvNjM3OC84NzIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8yNC83MzIyNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carlson</span>,  <span class="cit-name-given-names">C. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zipfel</span>,  <span class="cit-name-given-names">C. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garnier</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Bansal</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Global estimates of mammalian viral diversity accounting for host sharing</span>. <abbr class="cit-jnl-abbrev">Nat. Ecol. Evol</abbr>. <span class="cit-vol">3</span>, <span class="cit-fpage">1070</span>–<span class="cit-lpage">1075</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BEcol.%2BEvol%26rft.volume%253D3%26rft.spage%253D1070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.5" data-doi="10.1038/nature22975"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Olival</span>,  <span class="cit-name-given-names">K. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Host and viral traits predict zoonotic spillover from mammals</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">546</span>, <span class="cit-fpage">646</span>–<span class="cit-lpage">650</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D546%26rft.spage%253D646%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22975%26rft_id%253Dinfo%253Apmid%252F28636590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature22975&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28636590&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F24%2F732255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">B. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Undiscovered Bat Hosts of Filoviruses</span>. <abbr class="cit-jnl-abbrev">PLoS Negl. Trop. Dis</abbr>. <span class="cit-vol">10</span>, <span class="cit-fpage">e0004815</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BNegl.%2BTrop.%2BDis%26rft.volume%253D10%26rft.spage%253De0004815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.7" data-doi="10.1126/science.aap9072"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Babayan</span>,  <span class="cit-name-given-names">S. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orton</span>,  <span class="cit-name-given-names">R. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Streicker</span>,  <span class="cit-name-given-names">D. G.</span></span> <span class="cit-article-title">Predicting reservoir hosts and arthropod vectors from evolutionary signatures in RNA virus genomes</span>. <abbr class="cit-jnl-abbrev">Science (80-.)</abbr>. <span class="cit-vol">362</span>, <span class="cit-fpage">577</span>–<span class="cit-lpage">580</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBabayan%26rft.auinit1%253DS.%2BA.%26rft.volume%253D362%26rft.issue%253D6414%26rft.spage%253D577%26rft.epage%253D580%26rft.atitle%253DPredicting%2Breservoir%2Bhosts%2Band%2Barthropod%2Bvectors%2Bfrom%2Bevolutionary%2Bsignatures%2Bin%2BRNA%2Bvirus%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aap9072%26rft_id%253Dinfo%253Apmid%252F30385576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjIvNjQxNC81NzciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8yNC83MzIyNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.8" data-doi="10.1098/rspb.2012.2753"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luis</span>,  <span class="cit-name-given-names">A. D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A comparison of bats and rodents as reservoirs of zoonotic viruses: are bats special</span>? <abbr class="cit-jnl-abbrev">Proc. R. Soc. B Biol. Sci</abbr>. <span class="cit-vol">280</span>, <span class="cit-fpage">20122753</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BR.%2BSoc.%2BB%2BBiol.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2012.2753%26rft_id%253Dinfo%253Apmid%252F23378666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2012.2753&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23378666&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F24%2F732255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.9" data-doi="10.1073/pnas.1501598112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">B. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmidt</span>,  <span class="cit-name-given-names">J. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bowden</span>,  <span class="cit-name-given-names">S. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Drake</span>,  <span class="cit-name-given-names">J. M.</span></span> <span class="cit-article-title">Rodent reservoirs of future zoonotic diseases</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci</abbr>. <span class="cit-vol">112</span>, <span class="cit-fpage">7039</span>–<span class="cit-lpage">7044</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1501598112%26rft_id%253Dinfo%253Apmid%252F26038558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzIyLzcwMzkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8yNC83MzIyNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.10" data-doi="10.1371/journal.pone.0180716"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plourde</span>,  <span class="cit-name-given-names">B. T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Are disease reservoirs special? Taxonomic and life history characteristics</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">e0180716</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0180716%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0180716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0180716&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dallas</span>,  <span class="cit-name-given-names">T. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Host traits associated with species roles in parasite sharing networks</span>. <abbr class="cit-jnl-abbrev">Oikos</abbr> <span class="cit-vol">128</span>, <span class="cit-fpage">23</span>–<span class="cit-lpage">32</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOikos%26rft.volume%253D128%26rft.spage%253D23%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.12" data-doi="10.1038/nrmicro.2017.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plowright</span>,  <span class="cit-name-given-names">R. K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Pathways to zoonotic spillover</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol</abbr>. <span class="cit-vol">15</span>, <span class="cit-fpage">502</span>–<span class="cit-lpage">510</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BMicrobiol%26rft.volume%253D15%26rft.spage%253D502%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro.2017.45%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro.2017.45&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.13" data-doi="10.1038/nature12711"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ge</span>,  <span class="cit-name-given-names">X. Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">503</span>, <span class="cit-fpage">535</span>–<span class="cit-lpage">538</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D503%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12711%26rft_id%253Dinfo%253Apmid%252F24172901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24172901&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F24%2F732255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327464200045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bininda-Emonds</span>,  <span class="cit-name-given-names">O. R. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>,  <span class="cit-name-given-names">P. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gittleman</span>,  <span class="cit-name-given-names">J. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Altizer</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Phylogenetically related and ecologically similar carnivores harbour similar parasite assemblages</span>. <abbr class="cit-jnl-abbrev">J. Anim. Ecol</abbr>. <span class="cit-vol">83</span>, <span class="cit-fpage">671</span>–<span class="cit-lpage">680</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BEcol%26rft.volume%253D83%26rft.spage%253D671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">15.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-732255v3.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wells</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Global spread of helminth parasites at the human-domestic animal-wildlife interface</span>. <abbr class="cit-jnl-abbrev">Glob. Chang. Biol</abbr>. <span class="cit-vol">24</span>, <span class="cit-fpage">3254</span>–<span class="cit-lpage">3265</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob.%2BChang.%2BBiol%26rft.volume%253D24%26rft.spage%253D3254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stephens</span>,  <span class="cit-name-given-names">P. R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Parasite sharing in wild ungulates and their predators: Effects of phylogeny, range overlap, and trophic links</span>. <abbr class="cit-jnl-abbrev">J. Anim. Ecol</abbr>. <span class="cit-vol">88</span>, <span class="cit-fpage">1017</span>–<span class="cit-lpage">1028</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BEcol%26rft.volume%253D88%26rft.spage%253D1017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.17" data-doi="10.1126/science.1188836"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Streicker</span>,  <span class="cit-name-given-names">D. G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Host Phylogeny Constrains Cross-Species Emergence and Establishment of Rabies Virus in Bats</span>. <abbr class="cit-jnl-abbrev">Science (80-.)</abbr>. <span class="cit-vol">329</span>, <span class="cit-fpage">676</span>–<span class="cit-lpage">679</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DStreicker%26rft.auinit1%253DD.%2BG.%26rft.volume%253D329%26rft.issue%253D5992%26rft.spage%253D676%26rft.epage%253D679%26rft.atitle%253DHost%2BPhylogeny%2BConstrains%2BCross-Species%2BEmergence%2Band%2BEstablishment%2Bof%2BRabies%2BVirus%2Bin%2BBats%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1188836%26rft_id%253Dinfo%253Apmid%252F20689015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjkvNTk5Mi82NzYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8yNC83MzIyNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Willoughby</span>,  <span class="cit-name-given-names">A. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Phelps</span>,  <span class="cit-name-given-names">K. L.</span></span>, <span class="cit-auth cit-collab">Predict Consortium, predict@ucdavis edu &amp; Olival, K. J.</span> <span class="cit-article-title">A comparative analysis of viral richness and viral sharing in cave-roosting bats</span>. <abbr class="cit-jnl-abbrev">Diversity</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDiversity%26rft.volume%253D9%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.19" data-doi="10.1098/rspb.2008.0284"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Davies</span>,  <span class="cit-name-given-names">T. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pedersen</span>,  <span class="cit-name-given-names">A. B.</span></span> <span class="cit-article-title">Phylogeny and geography predict pathogen community similarity in wild primates and humans</span>. <abbr class="cit-jnl-abbrev">Proc. R. Soc. B Biol. Sci</abbr>. <span class="cit-vol">275</span>, <span class="cit-fpage">1695</span>–<span class="cit-lpage">1701</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DDavies%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D275%26rft.issue%253D1643%26rft.spage%253D1695%26rft.epage%253D1701%26rft.atitle%253DPhylogeny%2Band%2Bgeography%2Bpredict%2Bpathogen%2Bcommunity%2Bsimilarity%2Bin%2Bwild%2Bprimates%2Band%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2008.0284%26rft_id%253Dinfo%253Apmid%252F18445561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2008.0284&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18445561&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F24%2F732255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.20" data-doi="10.1016/j.tvjl.2017.12.024"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Glennon</span>,  <span class="cit-name-given-names">E. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Domesticated animals as hosts of henipaviruses and filoviruses: A systematic review</span>. <abbr class="cit-jnl-abbrev">Vet. J</abbr>. <span class="cit-vol">233</span>, <span class="cit-fpage">25</span>–<span class="cit-lpage">34</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVet.%2BJ%26rft.volume%253D233%26rft.spage%253D25%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tvjl.2017.12.024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tvjl.2017.12.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.21" data-doi="10.1126/science.288.5470.1432"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chua</span>,  <span class="cit-name-given-names">K. B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Nipah Virus: A Recently Emergent Deadly Paramyxovirus</span>. <abbr class="cit-jnl-abbrev">Science (80-.)</abbr>. <span class="cit-vol">288</span>, <span class="cit-fpage">1432</span>–<span class="cit-lpage">1435</span> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DChua%26rft.auinit1%253DK.%2BB.%26rft.volume%253D288%26rft.issue%253D5470%26rft.spage%253D1432%26rft.epage%253D1435%26rft.atitle%253DNipah%2BVirus%253A%2BA%2BRecently%2BEmergent%2BDeadly%2BParamyxovirus%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.288.5470.1432%26rft_id%253Dinfo%253Apmid%252F10827955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODgvNTQ3MC8xNDMyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMjQvNzMyMjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guy</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thiagavel</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mideo</span>,  <span class="cit-name-given-names">N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ratcliffe</span>,  <span class="cit-name-given-names">J. M.</span></span> <span class="cit-article-title">Phylogeny matters: Revisiting ‘a comparison of bats and rodents as reservoirs of zoonotic viruses’</span>. <abbr class="cit-jnl-abbrev">R. Soc. Open Sci</abbr>. <span class="cit-vol">6</span>, <span class="cit-fpage">181182</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR.%2BSoc.%2BOpen%2BSci%26rft.volume%253D6%26rft.spage%253D181182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.23" data-doi="10.1098/rspb.2014.2878"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pybus</span>,  <span class="cit-name-given-names">O. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tatem</span>,  <span class="cit-name-given-names">A. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lemey</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Virus evolution and transmission in an ever more connected world</span>. <abbr class="cit-jnl-abbrev">Proc. R. Soc. B Biol. Sci</abbr>. <span class="cit-vol">282</span>, <span class="cit-fpage">20142878</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BR.%2BSoc.%2BB%2BBiol.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2014.2878%26rft_id%253Dinfo%253Apmid%252F26702033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2014.2878&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26702033&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F24%2F732255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.24" data-doi="10.1128/JVI.00694-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanjuán</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nebot</span>,  <span class="cit-name-given-names">M. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chirico</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mansky</span>,  <span class="cit-name-given-names">L. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Belshaw</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Viral mutation rates</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">84</span>, <span class="cit-fpage">9733</span>–<span class="cit-lpage">9748</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00694-10%26rft_id%253Dinfo%253Apmid%252F20660197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4NC8xOS85NzMzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMjQvNzMyMjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ryan</span>,  <span class="cit-name-given-names">S. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carlson</span>,  <span class="cit-name-given-names">C. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mordecai</span>,  <span class="cit-name-given-names">E. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">L. R.</span></span> <span class="cit-article-title">Global expansion and redistribution of Aedes-borne virus transmission risk with climate change</span>. <abbr class="cit-jnl-abbrev">PLoS Negl. Trop. Dis</abbr>. <span class="cit-vol">13</span>, <span class="cit-fpage">e0007213</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BNegl.%2BTrop.%2BDis%26rft.volume%253D13%26rft.spage%253De0007213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drake</span>,  <span class="cit-name-given-names">J. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Beier</span>,  <span class="cit-name-given-names">J. C.</span></span> <span class="cit-article-title">Ecological niche and potential distribution of <em>Anopheles arabiensis</em> in Africa in 2050</span>. <abbr class="cit-jnl-abbrev">Malar. J</abbr>. <span class="cit-vol">13</span>, <span class="cit-fpage">213</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMalar.%2BJ%26rft.volume%253D13%26rft.spage%253D213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-web" id="cit-732255v3.27"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">IUCN</span>. <span class="cit-article-title">The IUCN Red List of Threatened Species</span>. <span class="cit-source">IUCN Red List of Threatened Species</span>. Version <span class="cit-fpage">2019</span>–<span class="cit-lpage">2</span> (<span class="cit-pub-date">2019</span>). Available at: <a href="https://www.iucnredlist.org">https://www.iucnredlist.org</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.28" data-doi="10.1111/j.1461-0248.2009.01307.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fritz</span>,  <span class="cit-name-given-names">S. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bininda-Emonds</span>,  <span class="cit-name-given-names">O. R. P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Purvis</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Geographical variation in predictors of mammalian extinction risk: big is bad, but only in the tropics</span>. <abbr class="cit-jnl-abbrev">Ecol. Lett</abbr>. <span class="cit-vol">12</span>, <span class="cit-fpage">538</span>–<span class="cit-lpage">549</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DFritz%26rft.auinit1%253DS.%2BA.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D538%26rft.epage%253D549%26rft.atitle%253DGeographical%2Bvariation%2Bin%2Bpredictors%2Bof%2Bmammalian%2Bextinction%2Brisk%253A%2Bbig%2Bis%2Bbad%252C%2Bbut%2Bonly%2Bin%2Bthe%2Btropics.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2009.01307.x%26rft_id%253Dinfo%253Apmid%252F19392714%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2009.01307.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19392714&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F24%2F732255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266111200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wardeh</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Risley</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mcintyre</span>,  <span class="cit-name-given-names">M. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Setzkorn</span>,  <span class="cit-name-given-names">C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Baylis</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Database of host-pathogen and related species interactions, and their global distribution</span>. <abbr class="cit-jnl-abbrev">Sci. Data</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">150049</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BData%26rft.volume%253D2%26rft.spage%253D150049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.30" data-doi="10.1073/pnas.1220716110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gómez</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nunn</span>,  <span class="cit-name-given-names">C. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Verdú</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Centrality in primate-parasite networks reveals the potential for the transmission of emerging infectious diseases to humans</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci</abbr>. <span class="cit-vol">110</span>, <span class="cit-fpage">7738</span>–<span class="cit-lpage">41</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1220716110%26rft_id%253Dinfo%253Apmid%252F23610389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzE5Lzc3MzgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8yNC83MzIyNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y. X. G. G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Phylogenetic structure of wildlife assemblages shapes patterns of infectious livestock diseases in Africa</span>. <abbr class="cit-jnl-abbrev">Funct. Ecol</abbr>. <span class="cit-vol">33</span>, <span class="cit-fpage">1332</span>–<span class="cit-lpage">1341</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BEcol%26rft.volume%253D33%26rft.spage%253D1332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.32" data-doi="10.1038/s41467-017-00923-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Allen</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Global hotspots and correlates of emerging zoonotic diseases</span>. <abbr class="cit-jnl-abbrev">Nat. Commun</abbr>. <span class="cit-vol">8</span>, <span class="cit-fpage">1124</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D8%26rft.spage%253D1124%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-017-00923-8%26rft_id%253Dinfo%253Apmid%252F29066781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-017-00923-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29066781&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F24%2F732255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Becker</span>,  <span class="cit-name-given-names">D. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crowley</span>,  <span class="cit-name-given-names">D. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Washburne</span>,  <span class="cit-name-given-names">A. D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Plowright</span>,  <span class="cit-name-given-names">R. K.</span></span> <span class="cit-article-title">Temporal and spatial limitations in global surveillance for bat filoviruses and henipaviruses</span>. <abbr class="cit-jnl-abbrev">Biol. Lett</abbr>. <span class="cit-vol">15</span>, <span class="cit-fpage">20190423</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BLett%26rft.volume%253D15%26rft.spage%253D20190423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.34" data-doi="10.1016/j.chom.2018.01.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Dampened STING-Dependent Interferon Activation in Bats</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">297</span>–<span class="cit-lpage">301</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft.volume%253D23%26rft.spage%253D297%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2018.01.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2018.01.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">A. W.</span></span> <span class="cit-article-title">Phylogenetic aggregation increases zoonotic potential of mammalian viruses</span>. <abbr class="cit-jnl-abbrev">Biol. Lett</abbr>. <span class="cit-vol">15</span>, <span class="cit-fpage">20190668</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BLett%26rft.volume%253D15%26rft.spage%253D20190668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.36" data-doi="10.1126/science.1177345"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lloyd-smith</span>,  <span class="cit-name-given-names">J. O.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Epidemic Dynamics at the Human-Animal Interface</span>. <abbr class="cit-jnl-abbrev">Science (80-.)</abbr>. <span class="cit-vol">326</span>, <span class="cit-fpage">1362</span>–<span class="cit-lpage">1368</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLloyd-Smith%26rft.auinit1%253DJ.%2BO.%26rft.volume%253D326%26rft.issue%253D5958%26rft.spage%253D1362%26rft.epage%253D1367%26rft.atitle%253DEpidemic%2BDynamics%2Bat%2Bthe%2BHuman-Animal%2BInterface%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1177345%26rft_id%253Dinfo%253Apmid%252F19965751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk1OC8xMzYyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMjQvNzMyMjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brose</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ostling</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrison</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Martinez</span>,  <span class="cit-name-given-names">N. D.</span></span> <span class="cit-article-title">Unified spatial scaling of species and their trophic interactions</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">108</span>, <span class="cit-fpage">167</span>–<span class="cit-lpage">171</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D108%26rft.spage%253D167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.38" data-doi="10.1098/rstb.2018.0211"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Silk</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Integrating social behaviour, demography and disease dynamics in network models: applications to disease management in declining wildlife populations</span>. <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. B</abbr> <span class="cit-vol">374</span>, <span class="cit-fpage">20180211</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BB%26rft.volume%253D374%26rft.spage%253D20180211%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2018.0211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2018.0211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.39" data-doi="10.1126/sciadv.1602422"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carlson</span>,  <span class="cit-name-given-names">C. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Parasite biodiversity faces extinction and redistribution in a changing climate</span>. <abbr class="cit-jnl-abbrev">Sci. Adv</abbr>. <span class="cit-vol">3</span>, <span class="cit-fpage">e1602422</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DCarlson%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D3%26rft.issue%253D9%26rft.spage%253De1602422%26rft.epage%253De1602422%26rft.atitle%253DParasite%2Bbiodiversity%2Bfaces%2Bextinction%2Band%2Bredistribution%2Bin%2Ba%2Bchanging%2Bclimate%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.1602422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiMy85L2UxNjAyNDIyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMjQvNzMyMjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">40.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-732255v3.40" data-doi="10.1111/jbi.13395"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">J. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Blois</span>,  <span class="cit-name-given-names">J. L.</span></span> <span class="cit-article-title">Range shifts in response to past and future climate change: Can climate velocities and species’ dispersal capabilities explain variation in mammalian range shifts</span>? <abbr class="cit-jnl-abbrev">J. Biogeogr</abbr>. <span class="cit-vol">45</span>, <span class="cit-fpage">2175</span>–<span class="cit-lpage">2189</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiogeogr%26rft.volume%253D45%26rft.spage%253D2175%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjbi.13395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jbi.13395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.41" data-doi="10.1126/science.1206432"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">I.-C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">J. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ohlemüller</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roy</span>,  <span class="cit-name-given-names">D. B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Thomas</span>,  <span class="cit-name-given-names">C. D.</span></span> <span class="cit-article-title">Rapid range shifts of species associated with high levels of climate warming</span>. <abbr class="cit-jnl-abbrev">Science (80-.)</abbr>. <span class="cit-vol">333</span>, <span class="cit-fpage">1024</span>–<span class="cit-lpage">6</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DChen%26rft.auinit1%253DI.-C.%26rft.volume%253D333%26rft.issue%253D6045%26rft.spage%253D1024%26rft.epage%253D1026%26rft.atitle%253DRapid%2BRange%2BShifts%2Bof%2BSpecies%2BAssociated%2Bwith%2BHigh%2BLevels%2Bof%2BClimate%2BWarming%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1206432%26rft_id%253Dinfo%253Apmid%252F21852500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzMvNjA0NS8xMDI0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMjQvNzMyMjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.42" data-doi="10.1098/rspb.2001.1897"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tompkins</span>,  <span class="cit-name-given-names">D. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sainsbury</span>,  <span class="cit-name-given-names">A. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nettleton</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buxton</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gurnell</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Parapoxvirus causes a deleterious disease in red squirrels associated with UK population declines</span>. <abbr class="cit-jnl-abbrev">Proc. R. Soc. B Biol. Sci</abbr>. <span class="cit-vol">269</span>, <span class="cit-fpage">529</span>–<span class="cit-lpage">533</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DTompkins%26rft.auinit1%253DD.%2BM.%26rft.volume%253D269%26rft.issue%253D1490%26rft.spage%253D529%26rft.epage%253D533%26rft.atitle%253DParapoxvirus%2Bcauses%2Ba%2Bdeleterious%2Bdisease%2Bin%2Bred%2Bsquirrels%2Bassociated%2Bwith%2BUK%2Bpopulation%2Bdeclines%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2001.1897%26rft_id%253Dinfo%253Apmid%252F11886647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2001.1897&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11886647&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F24%2F732255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174543200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.43"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Core Team</span>. <span class="cit-article-title">R: A language and environment for statistical computing</span>. <abbr class="cit-jnl-abbrev">R Foundation for Statistical Computing, Vienna, Austria</abbr>. (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-732255v3.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wood</span>,  <span class="cit-name-given-names">S. N.</span></span> <span class="cit-article-title">Fast stable restricted maximum likelihood and marginal likelihood estimation of semiparametric generalized linear models</span>. <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Ser. B (Statistical Methodol</abbr>. <span class="cit-vol">73</span>, <span class="cit-fpage">3</span>–<span class="cit-lpage">36</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSer.%2BB%2B%2528Statistical%2BMethodol%26rft.volume%253D73%26rft.spage%253D3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/732255v3.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F732255v3" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/782649v2" title="New insights on the ventral attention network: Active suppression and involuntary recruitment during a bimodal task" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/797183v5" title="Chromatin accessibility and histone acetylation in the regulation of competence in early development" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 24, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/732255v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2020/01/24/732255.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/732255v3.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/732255v3" class="active" data-icon-position="" data-hide-link-title="0">Predicting the global mammalian viral sharing network using phylogeography</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1119825" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ioxphDNukS9UCaAxG-puU878zVZAzAupMN4waBO0QNo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1119825" id="node1119825--2" data-pisa="biorxiv;732255v3" data-pisa-master="biorxiv;732255" data-seqnum="1119825" data-apath="/biorxiv/early/2020/01/24/732255.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Predicting the global mammalian viral sharing network using phylogeography</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Gregory F</span> <span class="nlm-surname">Albery</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Evan A</span> <span class="nlm-surname">Eskew</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Noam</span> <span class="nlm-surname">Ross</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Kevin J</span> <span class="nlm-surname">Olival</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">732255; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/732255 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/732255v3" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/732255v3&amp;title=Predicting%20the%20global%20mammalian%20viral%20sharing%20network%20using%20phylogeography" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/732255v3&amp;title=Predicting%20the%20global%20mammalian%20viral%20sharing%20network%20using%20phylogeography" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/732255v3&amp;text=Predicting%20the%20global%20mammalian%20viral%20sharing%20network%20using%20phylogeography" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/732255v3&amp;title=Predicting%20the%20global%20mammalian%20viral%20sharing%20network%20using%20phylogeography" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/732255v3&amp;t=Predicting%20the%20global%20mammalian%20viral%20sharing%20network%20using%20phylogeography" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/732255v3&amp;title=Predicting%20the%20global%20mammalian%20viral%20sharing%20network%20using%20phylogeography" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/732255v3&amp;title=Predicting%20the%20global%20mammalian%20viral%20sharing%20network%20using%20phylogeography" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1119825" id="node1119825--3" data-pisa="biorxiv;732255v3" data-pisa-master="biorxiv;732255" data-seqnum="1119825" data-apath="/biorxiv/early/2020/01/24/732255.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Predicting the global mammalian viral sharing network using phylogeography</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Gregory F</span> <span class="nlm-surname">Albery</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Evan A</span> <span class="nlm-surname">Eskew</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Noam</span> <span class="nlm-surname">Ross</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Kevin J</span> <span class="nlm-surname">Olival</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">732255; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/732255 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Predicting%20the%20global%20mammalian%20viral%20sharing%20network%20using%20phylogeography&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Albery%2C+Gregory+F&amp;rft.au=Eskew%2C+Evan+A&amp;rft.au=Ross%2C+Noam&amp;rft.au=Olival%2C+Kevin+J"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1119825/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1119825/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1119825/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1119825/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1119825/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1119825/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1119825/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1119825/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1119825/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1119825/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1119825/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1119825/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/732255v3&amp;count=horizontal&amp;via=&amp;text=Predicting%20the%20global%20mammalian%20viral%20sharing%20network%20using%20phylogeography&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/732255v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/732255v3&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/732255v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/ecology" class="highlight" data-icon-position="" data-hide-link-title="0">Ecology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
