<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2017/08/11/175562.atom" />
<meta name="HW.pisa" content="biorxiv;175562v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Proofreading-deficient coronaviruses adapt over long-term passage for increased fidelity and fitness without reversion of exoribonuclease-inactivating mutations" />
<meta name="DC.Identifier" content="10.1101/175562" />
<meta name="DC.Date" content="2017-08-11" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2017, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The coronavirus (CoV) RNA genome is the largest among single-stranded positive sense RNA viruses. CoVs encode a proofreading 3′→5′exoribonuclease within nonstructural protein 14 (nsp14-ExoN) that is responsible for CoV high-fidelity replication. Alanine substitution of ExoN catalytic residues [ExoN(-)] in SARS-CoV and murine hepatitis virus (MHV) disrupts ExoN activity, yielding viable mutant viruses with defective replication, up to 20-fold decreased fidelity, and increased susceptibility to nucleoside analogs. To test the stability of the ExoN(-) genotype and phenotype, we passaged MHV-ExoN(-) 250 times in cultured cells (P250), in parallel with WT-MHV. Compared to MHV-ExoN(-) P3, MHV-ExoN(-) P250 demonstrated enhanced replication, reduced susceptibility to nucleoside analogs, and increased competitive fitness. However, passage did not select for complete or partial reversion at the ExoN-inactivating mutations. We identified novel amino acid changes within the RNA-dependent RNA polymerase (nsp12-RdRp) and nsp14 of MHV-ExoN(-) P250 that partially account for the observed changes in replication, susceptibility to nucleoside analogs, and competitive fitness observed in the passaged virus population, indicating that additional determinants can compensate for the activities of nsp14-ExoN. Our results suggest that while selection favors restoration of replication fidelity in ExoN(-) CoVs, there may be a significant barrier to ExoN(-) reversion. These results also support the hypothesis that high-fidelity replication is linked to CoV fitness and identify additional candidate proteins that may regulate CoV replication fidelity.

IMPORTANCE Unique among RNA viruses, CoVs encode a proofreading exoribonuclease (ExoN) in nsp14 that mediates high-fidelity RNA genome replication. Proofreading-deficient CoVs with disrupted ExoN activity [ExoN(-)] are either non-viable or have significant defects in replication, RNA synthesis, fidelity, fitness, and virulence. In this study, we show that ExoN(-) murine hepatitis virus can adapt over long-term passage for increased replication and fitness without reverting the ExoN-inactivating mutations. Passage-adapted ExoN(-) mutants also demonstrate increasing resistance to nucleoside analogs that is only partially explained by secondary mutations in nsp12 and nsp14. These data suggest that enhanced resistance to nucleoside analogs is mediated by the interplay of multiple replicase proteins and support the proposed link between CoV fidelity and fitness." />
<meta name="DC.Contributor" content="Kevin W. Graepel" />
<meta name="DC.Contributor" content="Xiaotao Lu" />
<meta name="DC.Contributor" content="James Brett Case" />
<meta name="DC.Contributor" content="Nicole R. Sexton" />
<meta name="DC.Contributor" content="Everett Clinton Smith" />
<meta name="DC.Contributor" content="Mark R. Denison" />
<meta name="article:published_time" content="2017-08-11" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Proofreading-deficient coronaviruses adapt over long-term passage for increased fidelity and fitness without reversion of exoribonuclease-inactivating mutations" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;ABSTRACT&lt;/h3&gt;
&lt;p&gt;The coronavirus (CoV) RNA genome is the largest among single-stranded positive sense RNA viruses. CoVs encode a proofreading 3′→5′exoribonuclease within nonstructural protein 14 (nsp14-ExoN) that is responsible for CoV high-fidelity replication. Alanine substitution of ExoN catalytic residues [ExoN(-)] in SARS-CoV and murine hepatitis virus (MHV) disrupts ExoN activity, yielding viable mutant viruses with defective replication, up to 20-fold decreased fidelity, and increased susceptibility to nucleoside analogs. To test the stability of the ExoN(-) genotype and phenotype, we passaged MHV-ExoN(-) 250 times in cultured cells (P250), in parallel with WT-MHV. Compared to MHV-ExoN(-) P3, MHV-ExoN(-) P250 demonstrated enhanced replication, reduced susceptibility to nucleoside analogs, and increased competitive fitness. However, passage did not select for complete or partial reversion at the ExoN-inactivating mutations. We identified novel amino acid changes within the RNA-dependent RNA polymerase (nsp12-RdRp) and nsp14 of MHV-ExoN(-) P250 that partially account for the observed changes in replication, susceptibility to nucleoside analogs, and competitive fitness observed in the passaged virus population, indicating that additional determinants can compensate for the activities of nsp14-ExoN. Our results suggest that while selection favors restoration of replication fidelity in ExoN(-) CoVs, there may be a significant barrier to ExoN(-) reversion. These results also support the hypothesis that high-fidelity replication is linked to CoV fitness and identify additional candidate proteins that may regulate CoV replication fidelity.&lt;/p&gt;&lt;h3&gt;IMPORTANCE&lt;/h3&gt;
&lt;p&gt;Unique among RNA viruses, CoVs encode a proofreading exoribonuclease (ExoN) in nsp14 that mediates high-fidelity RNA genome replication. Proofreading-deficient CoVs with disrupted ExoN activity [ExoN(-)] are either non-viable or have significant defects in replication, RNA synthesis, fidelity, fitness, and virulence. In this study, we show that ExoN(-) murine hepatitis virus can adapt over long-term passage for increased replication and fitness without reverting the ExoN-inactivating mutations. Passage-adapted ExoN(-) mutants also demonstrate increasing resistance to nucleoside analogs that is only partially explained by secondary mutations in nsp12 and nsp14. These data suggest that enhanced resistance to nucleoside analogs is mediated by the interplay of multiple replicase proteins and support the proposed link between CoV fidelity and fitness.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2017/01/01" />
<meta name="citation_mjid" content="biorxiv;175562v1" />
<meta name="citation_id" content="175562v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/175562v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/175562v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/175562v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562.full.pdf" />
<meta name="citation_doi" content="10.1101/175562" />
<meta name="citation_num_pages" content="53" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="175562" />
<meta name="citation_author" content="Kevin W. Graepel" />
<meta name="citation_author_institution" content="Departments of Pathology, Microbiology and Immunology, Vanderbilt University Medical Center" />
<meta name="citation_author_institution" content="Departments of Pathology, Microbiology and Pediatrics, Vanderbilt University Medical Center" />
<meta name="citation_author_institution" content="Departments of Pathology, Microbiology and Elizabeth B. Lamb Center for Pediatric Research, Vanderbilt University Medical Center" />
<meta name="citation_author" content="Xiaotao Lu" />
<meta name="citation_author_institution" content="Departments of Pathology, Microbiology and Pediatrics, Vanderbilt University Medical Center" />
<meta name="citation_author_institution" content="Departments of Pathology, Microbiology and Elizabeth B. Lamb Center for Pediatric Research, Vanderbilt University Medical Center" />
<meta name="citation_author" content="James Brett Case" />
<meta name="citation_author_institution" content="Departments of Pathology, Microbiology and Immunology, Vanderbilt University Medical Center" />
<meta name="citation_author_institution" content="Departments of Pathology, Microbiology and Pediatrics, Vanderbilt University Medical Center" />
<meta name="citation_author_institution" content="Departments of Pathology, Microbiology and Elizabeth B. Lamb Center for Pediatric Research, Vanderbilt University Medical Center" />
<meta name="citation_author" content="Nicole R. Sexton" />
<meta name="citation_author_institution" content="Departments of Pathology, Microbiology and Immunology, Vanderbilt University Medical Center" />
<meta name="citation_author_institution" content="Departments of Pathology, Microbiology and Pediatrics, Vanderbilt University Medical Center" />
<meta name="citation_author_institution" content="Departments of Pathology, Microbiology and Elizabeth B. Lamb Center for Pediatric Research, Vanderbilt University Medical Center" />
<meta name="citation_author" content="Everett Clinton Smith" />
<meta name="citation_author_institution" content="Departments of Pathology, Microbiology and Pediatrics, Vanderbilt University Medical Center" />
<meta name="citation_author_institution" content="Departments of Pathology, Microbiology and Elizabeth B. Lamb Center for Pediatric Research, Vanderbilt University Medical Center" />
<meta name="citation_author_institution" content="Department of Biology, The University of the South" />
<meta name="citation_author_email" content="mark.denison@vanderbilt.edu" />
<meta name="citation_author_email" content="ecsmith@sewanee.edu" />
<meta name="citation_author" content="Mark R. Denison" />
<meta name="citation_author_institution" content="Departments of Pathology, Microbiology and Immunology, Vanderbilt University Medical Center" />
<meta name="citation_author_institution" content="Departments of Pathology, Microbiology and Pediatrics, Vanderbilt University Medical Center" />
<meta name="citation_author_institution" content="Departments of Pathology, Microbiology and Elizabeth B. Lamb Center for Pediatric Research, Vanderbilt University Medical Center" />
<meta name="citation_author_email" content="mark.denison@vanderbilt.edu" />
<meta name="citation_author_email" content="ecsmith@sewanee.edu" />
<meta name="citation_reference" content="Steinhauer DA, Domingo E, Holland JJ. 1992. Lack of evidence for proofreading mechanisms associated with an RNA virus polymerase. Gene 122:281–288" />
<meta name="citation_reference" content="Sanjuán R, Nebot MR, Chirico N, Mansky LM, Belshaw R. 2010. Viral mutation rates. J Virol 84:9733–9748." />
<meta name="citation_reference" content="Drake JW, Holland JJ. 1999. Mutation rates among RNA viruses. Proc Natl Acad Sci USA 96:13910–13913." />
<meta name="citation_reference" content="Bull JJ, Sanjuán R, Wilke CO. 2007. Theory of lethal mutagenesis for viruses. J Virol 81:2930–2939." />
<meta name="citation_reference" content="Domingo E, Sheldon J, Perales C. 2012. Viral quasispecies evolution. Microbiol Mol Biol Rev 76:159–216." />
<meta name="citation_reference" content="Bradwell K, Combe M, Domingo-Calap P, Sanjuán R. 2013. Correlation between mutation rate and genome size in riboviruses: mutation rate of bacteriophage Qß. Genetics 195:243–251." />
<meta name="citation_reference" content="Gago S, Elena SF, Flores R, Sanjuán R. 2009. Extremely high mutation rate of a hammerhead viroid. Science 323:1308–1308." />
<meta name="citation_reference" content="Peris JB, Davis P, Cuevas JM, Nebot MR, Sanjuán R. 2010. Distribution of Fitness Effects Caused by Single-Nucleotide Substitutions in Bacteriophage f1. Genetics 185:603–609." />
<meta name="citation_reference" content="Malpica JM, Fraile A, Moreno I, Obies CI, Drake JW, García-Arenal F. 2002. The rate and character of spontaneous mutation in an RNA virus. Genetics 162:1505–1511." />
<meta name="citation_reference" content="Elena SF, Moya A. 1999. Rate of deleterious mutation and the distribution of its effects on fitness in vesicular stomatitis virus. Journal of Evolutionary Biology 12:1078–1088." />
<meta name="citation_reference" content="Visher E, Whitefield SE, McCrone JT, Fitzsimmons W, Lauring AS. 2016. The Mutational Robustness of Influenza A Virus. PLoS Pathog 12:e1005856–25." />
<meta name="citation_reference" content="Sanjuán R, Moya A, Elena SF. 2004. The distribution of fitness effects caused by single-nucleotide substitutions in an RNA virus. Proc Natl Acad Sci USA 101:8396–8401." />
<meta name="citation_reference" content="Eckerle LD, Becker MM, Halpin RA, Li K, Venter E, Lu X, Scherbakova S, Graham RL, Baric RS, Stockwell TB, Spiro DJ, Denison MR. 2010. Infidelity of SARS-CoV Nsp14-exonuclease mutant virus replication is revealed by complete genome sequencing. PLoS Pathog 6:e1000896." />
<meta name="citation_reference" content="Eckerle LD, Lu X, Sperry SM, Choi L, Denison MR. 2007. High fidelity of murine hepatitis virus replication is decreased in nsp14 exoribonuclease mutants. J Virol 81:12135–12144." />
<meta name="citation_reference" content="Smith EC, Case JB, Blanc H, Isakov O, Shomron N, Vignuzzi M, Denison MR. 2015. Mutations in coronavirus nonstructural protein 10 decrease virus replication fidelity. J Virol 89:6418–6426." />
<meta name="citation_reference" content="Smith EC, Blanc H, Surdel MC, Vignuzzi M, Denison MR. 2013. Coronaviruses lacking exoribonuclease activity are susceptible to lethal mutagenesis: evidence for proofreading and potential therapeutics. PLoS Pathog 9:e1003565." />
<meta name="citation_reference" content="Sexton NR, Smith EC, Blanc H, Vignuzzi M, Peersen OB, Denison MR. 2016. Homology-Based Identification of a Mutation in the Coronavirus RNA-Dependent RNA Polymerase That Confers Resistance to Multiple Mutagens. J Virol 90:7415–7428." />
<meta name="citation_reference" content="Arias A, Arnold JJ, Sierra M, Smidansky ED, Domingo E, Cameron CE. 2008. Determinants of RNA-dependent RNA polymerase (in)fidelity revealed by kinetic analysis of the polymerase encoded by a foot-and-mouth disease virus mutant with reduced sensitivity to ribavirin. J Virol 82:12346–12355." />
<meta name="citation_reference" content="Zeng J, Wang H, Xie X, Yang D, Zhou G, Yu L. 2013. An increased replication fidelity mutant of foot-and-mouth disease virus retains fitness in vitro and virulence in vivo. Antiviral Res 100:1–7." />
<meta name="citation_reference" content="Zeng J, Wang H, Xie X, Li C, Zhou G, Yang D, Yu L. 2014. Ribavirin-resistant variants of foot-and-mouth disease virus: the effect of restricted quasispecies diversity on viral virulence. J Virol 88:4008–4020." />
<meta name="citation_reference" content="Xie X, Wang H, Zeng J, Li C, Zhou G, Yang D, Yu L. 2014. Foot-and-mouth disease virus low-fidelity polymerase mutants are attenuated. Arch Virol 159:2641–2650." />
<meta name="citation_reference" content="Sierra M, Airaksinen A, González-López C, Agudo R, Arias A, Domingo E. 2007. Foot-and-mouth disease virus mutant with decreased sensitivity to ribavirin: implications for error catastrophe. J Virol 81:2012–2024." />
<meta name="citation_reference" content="Vignuzzi M, Stone JK, Arnold JJ, Cameron CE, Andino R. 2006. Quasispecies diversity determines pathogenesis through cooperative interactions in a viral population. Nature 439:344–348." />
<meta name="citation_reference" content="Vignuzzi M, Wendt E, Andino R. 2008. Engineering attenuated virus vaccines by controlling replication fidelity. Nat Med 14:154–161." />
<meta name="citation_reference" content="Pfeiffer JK, Kirkegaard K. 2003. A single mutation in poliovirus RNA-dependent RNA polymerase confers resistance to mutagenic nucleotide analogs via increased fidelity. Proc Natl Acad Sci USA 100:7289–7294." />
<meta name="citation_reference" content="Arnold JJ, Vignuzzi M, Stone JK, Andino R, Cameron CE. 2005. Remote site control of an active site fidelity checkpoint in a viral RNA-dependent RNA polymerase. J Biol Chem 280:25706–25716." />
<meta name="citation_reference" content="Liu X, Yang X, Lee CA, Moustafa IM, Smidansky ED, Lum D, Arnold JJ, Cameron CE, Boehr DD. 2013. Vaccine-derived mutation in motif D of poliovirus RNA-dependent RNA polymerase lowers nucleotide incorporation fidelity. J Biol Chem 288:32753–32765." />
<meta name="citation_reference" content="Weeks SA, Lee CA, Zhao Y, Smidansky ED, August A, Arnold JJ, Cameron CE. 2012. A Polymerase mechanism-based strategy for viral attenuation and vaccine development. J Biol Chem 287:31618–31622." />
<meta name="citation_reference" content="Korboukh VK, Lee CA, Acevedo A, Vignuzzi M, Xiao Y, Arnold JJ, Hemperly S, Graci JD, August A, Andino R, Cameron CE. 2014. RNA virus population diversity, an optimum for maximal fitness and virulence. J Biol Chem 289:29531–29544." />
<meta name="citation_reference" content="Coffey LL, Beeharry Y, Bordería AV, Blanc H, Vignuzzi M. 2011. Arbovirus high fidelity variant loses fitness in mosquitoes and mice. Proc Natl Acad Sci USA 108:16038– 16043." />
<meta name="citation_reference" content="Rozen-Gagnon K, Stapleford KA, Mongelli V, Blanc H, Failloux A-B, Saleh M-C, Vignuzzi M. 2014. Alphavirus mutator variants present host-specific defects and attenuation in mammalian and insect models. PLoS Pathog 10:e1003877." />
<meta name="citation_reference" content="Cheung PPH, Watson SJ, Choy K-T, Fun Sia S, Wong DDY, Poon LLM, Kellam P, Guan Y, Malik Peiris JS, Yen H-L. 2014. Generation and characterization of influenza A viruses with altered polymerase fidelity. Nat Commun 5:4794." />
<meta name="citation_reference" content="Gnädig NF, Beaucourt S, Campagnola G, Bordería AV, Sanz-Ramos M, Gong P, Blanc H, Peersen OB, Vignuzzi M. 2012. Coxsackievirus B3 mutator strains are attenuated in vivo. Proc Natl Acad Sci USA 109:E2294–303." />
<meta name="citation_reference" content="Campagnola G, McDonald S, Beaucourt S, Vignuzzi M, Peersen OB. 2015. Structure-Function Relationships Underlying the Replication Fidelity of Viral RNA-Dependent RNA Polymerases. J Virol 89:275–286." />
<meta name="citation_reference" content="Sadeghipour S, McMinn PC. 2013. A study of the virulence in mice of high copying fidelity variants of human enterovirus 71. Virus Research 176:265–272." />
<meta name="citation_reference" content="Sadeghipour S, Bek EJ, McMinn PC. 2013. Ribavirin-resistant mutants of human enterovirus 71 express a high replication fidelity phenotype during growth in cell culture. J Virol 87:1759–1769." />
<meta name="citation_reference" content="Meng T, Kwang J. 2014. Attenuation of human enterovirus 71 high-replication-fidelity variants in AG129 mice. J Virol 88:5803–5815." />
<meta name="citation_reference" content="Graham RL, Becker MM, Eckerle LD, Bolles M, Denison MR, Baric RS. 2012. A live,impaired-fidelity coronavirus vaccine protects in an aged, immunocompromised mouse model of lethal disease. Nat Med 18:1820–1826." />
<meta name="citation_reference" content="Smith EC, Sexton NR, Denison MR. 2014. Thinking Outside the Triangle: Replication Fidelity of the Largest RNA Viruses. Annual Review of Virology 1:111–132." />
<meta name="citation_reference" content="Gorbalenya AE, Enjuanes L, Ziebuhr J, Snijder EJ. 2006. Nidovirales: evolving the largest RNA virus genome. Virus Research 117:17–37." />
<meta name="citation_reference" content="Brister JR, Ako-Adjei D, Bao Y, Blinkova O. 2014. NCBI Viral Genomes Resource. Nucleic Acids Res 43:gku1207–D577." />
<meta name="citation_reference" content="Stenglein MD, Jacobson ER, Wozniak EJ, Wellehan JFX, Kincaid A, Gordon M, Porter BF, Baumgartner W, Stahl S, Kelley K, Towner JS, DeRisi JL. 2014. Ball Python Nidovirus: a Candidate Etiologic Agent for Severe Respiratory Disease in Python regius. MBio 5:e01484-14–e01484-14." />
<meta name="citation_reference" content="Minskaia E, Hertzig T, Gorbalenya AE, Campanacci V, Cambillau C, Canard B, Ziebuhr J. 2006. Discovery of an RNA virus 3“-&gt;5” exoribonuclease that is critically involved in coronavirus RNA synthesis. Proc Natl Acad Sci USA 103:5108–5113." />
<meta name="citation_reference" content="Snijder EJ, Bredenbeek PJ, Dobbe JC, Thiel V, Ziebuhr J, Poon LLM, Guan Y, Rozanov M, Spaan WJM, Gorbalenya AE. 2003. Unique and conserved features of genome and proteome of SARS-coronavirus, an early split-off from the coronavirus group 2 lineage. J Mol Biol 331:991–1004." />
<meta name="citation_reference" content="Ma Y, Wu L, Shaw N, Gao Y, Wang J, Sun Y, Lou Z, Yan L, Zhang R, Rao Z. 2015. Structural basis and functional analysis of the SARS coronavirus nsp14-nsp10 complex. Proc Natl Acad Sci USA 112:9436–9441." />
<meta name="citation_reference" content="Zuo Y, Deutscher MP. 2001. Exoribonuclease superfamilies: structural analysis and phylogenetic distribution. Nucleic Acids Res 29:1017–1026." />
<meta name="citation_reference" content="Bouvet M, Imbert I, Subissi L, Gluais L, Canard B, Decroly E. 2012. RNA 3’-end mismatch excision by the severe acute respiratory syndrome coronavirus nonstructural protein nsp10/nsp14 exoribonuclease complex. Proc Natl Acad Sci USA 109:9372–9377." />
<meta name="citation_reference" content="Yokomori K, Banner LR, Lai MMC. 1991. Heterogeneity of gene expression of the hemagglutinin-esterase (HE) protein of murine coronaviruses. Virology 183:647–657." />
<meta name="citation_reference" content="Luytjes W, Bredenbeek PJ, Noten AF, Horzinek MC, Spaan WJ. 1988. Sequence of mouse hepatitis virus A59 mRNA 2: indications for RNA recombination between coronaviruses and influenza C virus. Virology 166:415–422." />
<meta name="citation_reference" content="Yokomori K, Lai MM. 1991. Mouse hepatitis virus S RNA sequence reveals that nonstructural proteins ns4 and ns5a are not essential for murine coronavirus replication. J Virol 65:5605–5608." />
<meta name="citation_reference" content="Lissenberg A, Vrolijk MM, van Vliet ALW, Langereis MA, de Groot-Mijnes JDF, Rottier PJM, de Groot RJ. 2005. Luxury at a Cost? Recombinant Mouse Hepatitis Viruses Expressing the Accessory Hemagglutinin Esterase Protein Display Reduced Fitness In Vitro. J Virol 79:15054–15063." />
<meta name="citation_reference" content="Gadlage MJ, Graham RL, Denison MR. 2008. Murine coronaviruses encoding nsp2 at different genomic loci have altered replication, protein expression, and localization. J Virol 82:11964–11969." />
<meta name="citation_reference" content="Zhao L, Jha BK, Wu A, Elliott R, Ziebuhr J, Gorbalenya AE, Silverman RH, Weiss SR. 2012. Antagonism of the Interferon-Induced OAS-RNase L Pathway by Murine Coronavirus ns2 Protein Is Required for Virus Replication and Liver Pathology. Cell Host and Microbe 11:607–616." />
<meta name="citation_reference" content="Li Y, Weiss SR. 2016. Antagonism of RNase L is required for murine coronavirus replication in Kupffer cells and liver sinusoidal endothelial cells but not in hepatocytes. J Virol JVI.01423–16–24." />
<meta name="citation_reference" content="Zhang R, Jha BK, Ogden KM, Dong B, Zhao L, Elliott R, Patton JT, Silverman RH, Weiss SR. 2013. Homologous 2“,5-”phosphodiesterases from disparate RNA viruses antagonize antiviral innate immunity. Proc Natl Acad Sci USA 110:13114–13119." />
<meta name="citation_reference" content="Sui B, Huang J, Jha BK, Yin P, Zhou M, Fu ZF, Silverman RH, Weiss SR, Peng G, Zhao L. 2016. Crystal structure of the mouse hepatitis virus ns2 phosphodiesterase domain that antagonizes RNase L activation. J Gen Virol 97:880–886." />
<meta name="citation_reference" content="Schwarz B, Routledge E, Siddell SG. 1990. Murine coronavirus nonstructural protein ns2 is not essential for virus replication in transformed cells. J Virol 64:4784–4791." />
<meta name="citation_reference" content="Zhao L, Rose KM, Elliott R, Van Rooijen N, Weiss SR. 2011. Cell-type-specific type I interferon antagonism influences organ tropism of murine coronavirus. J Virol 85:10058– 10068." />
<meta name="citation_reference" content="Pereira-Gómez M, Sanjuán R. 2014. Delayed lysis confers resistance to the nucleoside analogue 5-fluorouracil and alleviates mutation accumulation in the single-stranded DNA bacteriophage ?X174. J Virol 88:5042–5049." />
<meta name="citation_reference" content="Ng KKS, Arnold JJ, Cameron CE. 2008. Structure-function relationships among RNA-dependent RNA polymerases. Curr Top Microbiol Immunol 320:137–156." />
<meta name="citation_reference" content="Brockway SM, Clay CT, Lu XT, Denison MR. 2003. Characterization of the expression, intracellular localization, and replication complex association of the putative mouse hepatitis virus RNA-dependent RNA polymerase. J Virol 77:10515–10527." />
<meta name="citation_reference" content="Lehmann KC, Gulyaeva A, Zevenhoven-Dobbe JC, Janssen GMC, Ruben M, Overkleeft HS, van Veelen PA, Samborskiy DV, Kravchenko AA, Leontovich AM, Sidorov IA, Snijder EJ, Posthuma CC, Gorbalenya AE. 2015. Discovery of an essential nucleotidylating activity associated with a newly delineated conserved domain in the RNA polymerase-containing protein of all nidoviruses. Nucleic Acids Res 43:8416– 8434." />
<meta name="citation_reference" content="Kelley LA, Mezulis S, Yates CM, Wass MN, Sternberg MJE. 2015. The Phyre2 web portal for protein modeling, prediction and analysis. Nat Protoc 10:845–858." />
<meta name="citation_reference" content="Bouvet M, Lugari A, Posthuma CC, Zevenhoven JC, Bernard S, Betzi S, Imbert I, Canard B, Guillemot J-C, Lécine P, Pfefferle S, Drosten C, Snijder EJ, Decroly E, Morelli X. 2014. Coronavirus Nsp10, a critical co-factor for activation of multiple replicative enzymes. J Biol Chem 289:25783–25796." />
<meta name="citation_reference" content="Donaldson EF, Sims AC, Graham RL, Denison MR, Baric RS. 2007. Murine hepatitis virus replicase protein nsp10 is a critical regulator of viral RNA synthesis. J Virol 81:6356–6368." />
<meta name="citation_reference" content="Crotty S, Cameron C, Andino R. 2002. Ribavirin’s antiviral mechanism of action: lethal mutagenesis? J Mol Med 80:86–95." />
<meta name="citation_reference" content="Pathak VK, Temin HM. 1992. 5-Azacytidine and RNA secondary structure increase the retrovirus mutation rate. J Virol 66:3093–3100." />
<meta name="citation_reference" content="Carroll SS, Tomassini JE, Bosserman M, Getty K, Stahlhut MW, Eldrup AB, Bhat B, Hall D, Simcoe AL, LaFemina R, Rutkowski CA, Wolanski B, Yang Z, Migliaccio G, De Francesco R, Kuo LC, MacCoss M, Olsen DB. 2003. Inhibition of hepatitis C virus RNA replication by 2’-modified nucleoside analogs. J Biol Chem 278:11979–11984." />
<meta name="citation_reference" content="Lauber C, Goeman JJ, del Carmen Parquet M, Nga PT, Snijder EJ, Morita K, Gorbalenya AE. 2013. The Footprint of Genome Architecture in the Largest Genome Expansion in RNA Viruses. PLoS Pathog 9:e1003500." />
<meta name="citation_reference" content="Derbyshire V, Grindley ND, Joyce CM. 1991. The 3“-5” exonuclease of DNA polymerase I of Escherichia coli: contribution of each amino acid at the active site to the reaction. EMBO J 10:17–24." />
<meta name="citation_reference" content="Becares M, Pascual-Iglesias A, Nogales A, Sola I, Enjuanes L, Zuñiga S. 2016. Mutagenesis of Coronavirus nsp14 Reveals Its Potential Role in Modulation of the Innate Immune Response. J Virol 90:5399–5414." />
<meta name="citation_reference" content="Cameron CE, Moustafa IM, Arnold JJ. 2016. Fidelity of Nucleotide Incorporation by the RNA-Dependent RNA Polymerase from PoliovirusDNA Replication Across Taxa, 1st ed. Elsevier Inc." />
<meta name="citation_reference" content="Zeng J, Wang H, Xie X, Yang D, Zhou G, Yu L. 2013. An increased replication fidelity mutant of foot-and-mouth disease virus retains fitness in vitro and virulence in vivo. Antiviral Res 100:1–7." />
<meta name="citation_reference" content="Subissi L, Posthuma CC, Collet A, Zevenhoven-Dobbe JC, Gorbalenya AE, Decroly E, Snijder EJ, Canard B, Imbert I. 2014. One severe acute respiratory syndrome coronavirus protein complex integrates processive RNA polymerase and exonuclease activities. Proc Natl Acad Sci USA 111:E3900–9." />
<meta name="citation_reference" content="Imbert I, Guillemot J-C, Bourhis J-M, Bussetta C, Coutard B, Egloff M-P, Ferron F, Gorbalenya AE, Canard B. 2006. A second, non-canonical RNA-dependent RNA polymerase in SARS coronavirus. EMBO J 25:4933–4942." />
<meta name="citation_reference" content="Adedeji AO, Marchand B, Velthuis te AJW, Snijder EJ, Weiss S, Eoff RL, Singh K, Sarafianos SG. 2012. Mechanism of Nucleic Acid Unwinding by SARS-CoV Helicase. PLoS ONE 7:e36521." />
<meta name="citation_reference" content="Chaudhuri M, Chaudhuri M, Song L, Parris DS. 2003. The herpes simplex virus type 1 DNA polymerase processivity factor increases fidelity without altering pre-steady-state rate constants for polymerization or excision. J Biol Chem 278:8996–9004." />
<meta name="citation_reference" content="Gu S, Li W, Zhang H, Fleming J, Yang W, Wang S, Wei W, Zhou J, Zhu G, Deng J, Hou J, Zhou Y, Lin S, Zhang X-E, Bi L. 2016. The ß2 clamp in the Mycobacterium tuberculosis DNA polymerase III aß2e replicase promotes polymerization and reduces exonuclease activity. Sci Rep 6:18418." />
<meta name="citation_reference" content="Stapleford KA, Rozen-Gagnon K, Das PK, Saul S, Poirier EZ, Blanc H, Vidalain P-O, Merits A, Vignuzzi M. 2015. Viral Polymerase-Helicase Complexes Regulate Replication Fidelity To Overcome Intracellular Nucleotide Depletion. J Virol 89:11233– 11244." />
<meta name="citation_reference" content="Agudo R, la Higuera de I, Arias A, Grande-Pérez A, Domingo E. 2016. Involvement of a joker mutation in a polymerase-independent lethal mutagenesis escape mechanism. Virology 494:257–266." />
<meta name="citation_reference" content="Egloff M-P, Ferron F, Campanacci V, Longhi S, Rancurel C, Dutartre H, Snijder EJ, Gorbalenya AE, Cambillau C, Canard B. 2004. The severe acute respiratory syndrome-coronavirus replicative protein nsp9 is a single-stranded RNA-binding subunit unique in the RNA virus world. Proc Natl Acad Sci USA 101:3792–3796." />
<meta name="citation_reference" content="Bhardwaj K, Guarino L, Kao CC. 2004. The severe acute respiratory syndrome coronavirus Nsp15 protein is an endoribonuclease that prefers manganese as a cofactor. J Virol 78:12218–12224." />
<meta name="citation_reference" content="Sanjuán R, Cuevas JM, Furió V, Holmes EC, Moya A. 2007. Selection for Robustness in Mutagenized RNA Viruses. PLoS Genet 3:e93." />
<meta name="citation_reference" content="Goldhill D, Lee A, Williams ESCP, Turner PE. 2014. Evolvability and robustness in populations of RNA virus F6. Frontiers in Microbiology 5." />
<meta name="citation_reference" content="Montville R, Froissart R, Remold SK, Tenaillon O, Turner PE. 2005. Evolution of mutational robustness in an RNA virus. PLoS Biol 3:e381." />
<meta name="citation_reference" content="Graci JD, Gnädig NF, Galarraga JE, Castro C, Vignuzzi M, Cameron CE. 2012. Mutational robustness of an RNA virus influences sensitivity to lethal mutagenesis. J Virol 86:2869–2873." />
<meta name="citation_reference" content="Lauring AS, Acevedo A, Cooper SB, Andino R. 2012. Codon usage determines the mutational robustness, evolutionary capacity, and virulence of an RNA virus. Cell Host and Microbe 12:623–632." />
<meta name="citation_reference" content="Lauring AS, Frydman J, Andino R. 2013. The role of mutational robustness in RNA virus evolution. Nat Rev Microbiol 11:327–336." />
<meta name="citation_reference" content="Aguilera ER, Erickson AK, Jesudhasan PR, Robinson CM, Pfeiffer JK. 2017. Plaques Formed by Mutagenized Viral Populations Have Elevated Coinfection Frequencies. MBio 8:e02020-16–12." />
<meta name="citation_reference" content="O’Dea EB, Keller TE, Wilke CO. 2010. Does Mutational Robustness Inhibit Extinction by Lethal Mutagenesis in Viral Populations? PLoS Comput Biol 6:e1000811–12." />
<meta name="citation_reference" content="Yount B, Denison MR, Weiss SR, Baric RS. 2002. Systematic assembly of a full-length infectious cDNA of mouse hepatitis virus strain A59. J Virol 76:11065–11078." />
<meta name="twitter:title" content="Proofreading-deficient coronaviruses adapt over long-term passage for increased fidelity and fitness without reversion of exoribonuclease-inactivating mutations" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The coronavirus (CoV) RNA genome is the largest among single-stranded positive sense RNA viruses. CoVs encode a proofreading 3′→5′exoribonuclease within nonstructural protein 14 (nsp14-ExoN) that is responsible for CoV high-fidelity replication. Alanine substitution of ExoN catalytic residues [ExoN(-)] in SARS-CoV and murine hepatitis virus (MHV) disrupts ExoN activity, yielding viable mutant viruses with defective replication, up to 20-fold decreased fidelity, and increased susceptibility to nucleoside analogs. To test the stability of the ExoN(-) genotype and phenotype, we passaged MHV-ExoN(-) 250 times in cultured cells (P250), in parallel with WT-MHV. Compared to MHV-ExoN(-) P3, MHV-ExoN(-) P250 demonstrated enhanced replication, reduced susceptibility to nucleoside analogs, and increased competitive fitness. However, passage did not select for complete or partial reversion at the ExoN-inactivating mutations. We identified novel amino acid changes within the RNA-dependent RNA polymerase (nsp12-RdRp) and nsp14 of MHV-ExoN(-) P250 that partially account for the observed changes in replication, susceptibility to nucleoside analogs, and competitive fitness observed in the passaged virus population, indicating that additional determinants can compensate for the activities of nsp14-ExoN. Our results suggest that while selection favors restoration of replication fidelity in ExoN(-) CoVs, there may be a significant barrier to ExoN(-) reversion. These results also support the hypothesis that high-fidelity replication is linked to CoV fitness and identify additional candidate proteins that may regulate CoV replication fidelity.

IMPORTANCE Unique among RNA viruses, CoVs encode a proofreading exoribonuclease (ExoN) in nsp14 that mediates high-fidelity RNA genome replication. Proofreading-deficient CoVs with disrupted ExoN activity [ExoN(-)] are either non-viable or have significant defects in replication, RNA synthesis, fidelity, fitness, and virulence. In this study, we show that ExoN(-) murine hepatitis virus can adapt over long-term passage for increased replication and fitness without reverting the ExoN-inactivating mutations. Passage-adapted ExoN(-) mutants also demonstrate increasing resistance to nucleoside analogs that is only partially explained by secondary mutations in nsp12 and nsp14. These data suggest that enhanced resistance to nucleoside analogs is mediated by the interplay of multiple replicase proteins and support the proposed link between CoV fidelity and fitness." />
<meta name="og-title" property="og:title" content="Proofreading-deficient coronaviruses adapt over long-term passage for increased fidelity and fitness without reversion of exoribonuclease-inactivating mutations" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/175562v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The coronavirus (CoV) RNA genome is the largest among single-stranded positive sense RNA viruses. CoVs encode a proofreading 3′→5′exoribonuclease within nonstructural protein 14 (nsp14-ExoN) that is responsible for CoV high-fidelity replication. Alanine substitution of ExoN catalytic residues [ExoN(-)] in SARS-CoV and murine hepatitis virus (MHV) disrupts ExoN activity, yielding viable mutant viruses with defective replication, up to 20-fold decreased fidelity, and increased susceptibility to nucleoside analogs. To test the stability of the ExoN(-) genotype and phenotype, we passaged MHV-ExoN(-) 250 times in cultured cells (P250), in parallel with WT-MHV. Compared to MHV-ExoN(-) P3, MHV-ExoN(-) P250 demonstrated enhanced replication, reduced susceptibility to nucleoside analogs, and increased competitive fitness. However, passage did not select for complete or partial reversion at the ExoN-inactivating mutations. We identified novel amino acid changes within the RNA-dependent RNA polymerase (nsp12-RdRp) and nsp14 of MHV-ExoN(-) P250 that partially account for the observed changes in replication, susceptibility to nucleoside analogs, and competitive fitness observed in the passaged virus population, indicating that additional determinants can compensate for the activities of nsp14-ExoN. Our results suggest that while selection favors restoration of replication fidelity in ExoN(-) CoVs, there may be a significant barrier to ExoN(-) reversion. These results also support the hypothesis that high-fidelity replication is linked to CoV fitness and identify additional candidate proteins that may regulate CoV replication fidelity.

IMPORTANCE Unique among RNA viruses, CoVs encode a proofreading exoribonuclease (ExoN) in nsp14 that mediates high-fidelity RNA genome replication. Proofreading-deficient CoVs with disrupted ExoN activity [ExoN(-)] are either non-viable or have significant defects in replication, RNA synthesis, fidelity, fitness, and virulence. In this study, we show that ExoN(-) murine hepatitis virus can adapt over long-term passage for increased replication and fitness without reverting the ExoN-inactivating mutations. Passage-adapted ExoN(-) mutants also demonstrate increasing resistance to nucleoside analogs that is only partially explained by secondary mutations in nsp12 and nsp14. These data suggest that enhanced resistance to nucleoside analogs is mediated by the interplay of multiple replicase proteins and support the proposed link between CoV fidelity and fitness." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2017-08-11" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/175562v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/175562v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/175562v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/175562v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/52604" />
    <title>Proofreading-deficient coronaviruses adapt over long-term passage for increased fidelity and fitness without reversion of exoribonuclease-inactivating mutations | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"mLoGgajed6UQoH5deALO5lJ_G29o8LA3edz41ECxLfI","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"52604","apath":"\/biorxiv\/early\/2017\/08\/11\/175562.atom","pisa":"biorxiv;175562v1","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;175562\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;175562\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;175562v1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;175562\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;175562\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/175562v1","title":"Proofreading-deficient coronaviruses adapt over long-term passage for increased fidelity and fitness without reversion of exoribonuclease-inactivating mutations","identifier":"node\/52604"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GGPYcKbTLMOHrsF\\\\\\\/cXvJL3myFJhA48CCev9QVP2SAfQaaQiZIkZGKjukKLhq+Bjmp8aimBeAADDI0hnyuNpVPZAXaPttF+i5AXxrAkNBwFz3pKAPfHwsq4yxW62TY6Z+1PCFSQkzLvTxDMhqUBck\\\\\\\/YoTEFSltcvpxGisafpoUqysZwMKEyN4dIuCOCKj5Qnv0U1uPKfBW2LV1JUSQn+TaDPFPaM6RFHR1FTpMcrGOk56bnz5Vc0OL9Lw2OdCMLWnWEoRuOpW4MetebqEeXjYiMG8jx0QN\\\\\\\/OH23kdUWkcybN8Yw+FLpAHZXnL6dtqbluXoiMZ7BRp0gGeUhpgbU++L8AhOvYXpFqM91C8mlGgKgXSqwLdqjrAxkAu10ktuBo6\\\\\\\/z0rPlx0yUe1PfVLNChsFqF78FN55XUqNRAFuYXaGwbyW8p9xd0FkUgrRDTLocLazb\\\\\\\/a2xPk8JY7jhbQF1pkSLCFfaJYqyPHzQaSUA43fjSVb\\\\\\\/GIAX2fsoOcwYoO5SEY\\\\\\\/zrsgWA2v8x5EatQI5GlNr4T6rFEgxM1ni6jdJ4FqH4k4U7O8PxAQSPrlO7O8OQnpTONetoqsylIdn0JF4KQvTC3NDFCDPP+0FX3hsdpfEB\\\\\\\/rW7ycczwPfqS6YIBwRZ+evgE5X+9ZBlQEE6gMnYSiZwYfiX4U9ypaHRlTOxK01l+3JvK0JFu0FiBYjJ80K4Z2i5gCPIzlYXh0CYL7srawL3tnvkcengBTUW47tkNuetW8AljLMogmdZR\\\\\\\/3ut9\\\\\\\/QjNI3piIvW8HzQQ0ZlSxu85ojgIGtipbOGYl25pQs0vBU3+Q4HsWSJuNru1Vnio60WhDBSDPiTPhH3jWi3BthXevEKPclgfyd6l5aYwfFzFKZ\\\\\\\/Ms0x8S6kqxFE66oDVx85HO4js0mfZsoLUmTmXXS1PaOSOvGiH25CIARgNPT5z6Gc2bBXy7Dk29YUmOw1eEdMYY7Q2rDe6+FbCYTdJTUFPd596M\\\\\\\/YCWd8EQxFN3VVxMRdRWTEy+V18YCtnpuUZMxPYlRc5fY99WSmjHsHY2jPGELhDnPZpR2u7VgjwT5vu0AZ92ULyYcoI0ovqDigsnv2azD2HYJdCHpupj51zg3ffwIvx6A1kej3YQNTXn39psguqKX48u4azpTCMCstXTySPOBHVjHyaOb+AzTh1qhG4EsxkQGRxn15qB7bqGjno3hyK2RmEYKB9y\\\\\\\/tD3y7rYXEz0486sf+rUDxnz+egCNY04IpDnpHO5bPzXxNu5YdN\\\\\\\/nEOfAO9JsRmZSUSwUktpTlZxIGgCsQ6jRe5s8MJudoueKwhq\\\\\\\/BvcHGlvSleWDmUFg=\\\u0022,\\\u0022iv\\\u0022:\\\u0022lIKKjOaRHCavpsxeaSBWHw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002278efade3cccece9e1ec9e90ac708ceffba3279ea5cb0d6faafdcc2a26c29e41c\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/175562v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-52604 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/175562v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VTtU7EibHYQnqx1rswLUL_ujLbw6q_q7VqWmwUGJltk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="52604" id="node52604" data-pisa="biorxiv;175562v1" data-pisa-master="biorxiv;175562" data-apath="/biorxiv/early/2017/08/11/175562.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Proofreading-deficient coronaviruses adapt over long-term passage for increased fidelity and fitness without reversion of exoribonuclease-inactivating mutations</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Kevin W.</span> <span class="nlm-surname">Graepel</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Xiaotao</span> <span class="nlm-surname">Lu</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">James Brett</span> <span class="nlm-surname">Case</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Nicole R.</span> <span class="nlm-surname">Sexton</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Everett Clinton</span> <span class="nlm-surname">Smith</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Mark R.</span> <span class="nlm-surname">Denison</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/175562 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node52604" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kevin W. Graepel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Departments of Pathology, Microbiology and Immunology, Vanderbilt University Medical Center</span>, Nashville, Tennessee, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Departments of Pathology, Microbiology and Pediatrics, Vanderbilt University Medical Center</span>, Nashville, Tennessee, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>c</span><span class='nlm-institution'>Departments of Pathology, Microbiology and Elizabeth B. Lamb Center for Pediatric Research, Vanderbilt University Medical Center</span>, Nashville, Tennessee, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BW.%2BGraepel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graepel%20KW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BW.%2BGraepel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xiaotao Lu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Departments of Pathology, Microbiology and Pediatrics, Vanderbilt University Medical Center</span>, Nashville, Tennessee, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>c</span><span class='nlm-institution'>Departments of Pathology, Microbiology and Elizabeth B. Lamb Center for Pediatric Research, Vanderbilt University Medical Center</span>, Nashville, Tennessee, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaotao%2BLu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lu%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaotao%2BLu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">James Brett Case </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Departments of Pathology, Microbiology and Immunology, Vanderbilt University Medical Center</span>, Nashville, Tennessee, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Departments of Pathology, Microbiology and Pediatrics, Vanderbilt University Medical Center</span>, Nashville, Tennessee, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>c</span><span class='nlm-institution'>Departments of Pathology, Microbiology and Elizabeth B. Lamb Center for Pediatric Research, Vanderbilt University Medical Center</span>, Nashville, Tennessee, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BBrett%2BCase%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Case%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BBrett%2BCase%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nicole R. Sexton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Departments of Pathology, Microbiology and Immunology, Vanderbilt University Medical Center</span>, Nashville, Tennessee, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Departments of Pathology, Microbiology and Pediatrics, Vanderbilt University Medical Center</span>, Nashville, Tennessee, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>c</span><span class='nlm-institution'>Departments of Pathology, Microbiology and Elizabeth B. Lamb Center for Pediatric Research, Vanderbilt University Medical Center</span>, Nashville, Tennessee, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicole%2BR.%2BSexton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sexton%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicole%2BR.%2BSexton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Everett Clinton Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Departments of Pathology, Microbiology and Pediatrics, Vanderbilt University Medical Center</span>, Nashville, Tennessee, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>c</span><span class='nlm-institution'>Departments of Pathology, Microbiology and Elizabeth B. Lamb Center for Pediatric Research, Vanderbilt University Medical Center</span>, Nashville, Tennessee, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>d</span><span class='nlm-institution'>Department of Biology, The University of the South</span>, Sewanee, Tennessee, <span class='nlm-country'>USA</span>
</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Everett%2BClinton%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20EC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEverett%2BClinton%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mark.denison@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">mark.denison@vanderbilt.edu</a>
<a href="mailto:ecsmith@sewanee.edu" class="" data-icon-position="" data-hide-link-title="0">ecsmith@sewanee.edu</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mark R. Denison </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Departments of Pathology, Microbiology and Immunology, Vanderbilt University Medical Center</span>, Nashville, Tennessee, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Departments of Pathology, Microbiology and Pediatrics, Vanderbilt University Medical Center</span>, Nashville, Tennessee, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>c</span><span class='nlm-institution'>Departments of Pathology, Microbiology and Elizabeth B. Lamb Center for Pediatric Research, Vanderbilt University Medical Center</span>, Nashville, Tennessee, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BR.%2BDenison%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Denison%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BR.%2BDenison%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mark.denison@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">mark.denison@vanderbilt.edu</a>
<a href="mailto:ecsmith@sewanee.edu" class="" data-icon-position="" data-hide-link-title="0">ecsmith@sewanee.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/175562v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:52604" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:52604/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/175562v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:52604" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:52604/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/175562v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:52604" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:52604/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/175562v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:52604" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:52604/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/175562v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:52604" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:52604/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/175562v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:52604" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:52604/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-2">The coronavirus (CoV) RNA genome is the largest among single-stranded positive sense RNA viruses. CoVs encode a proofreading 3′→5′exoribonuclease within nonstructural protein 14 (nsp14-ExoN) that is responsible for CoV high-fidelity replication. Alanine substitution of ExoN catalytic residues [ExoN(-)] in SARS-CoV and murine hepatitis virus (MHV) disrupts ExoN activity, yielding viable mutant viruses with defective replication, up to 20-fold decreased fidelity, and increased susceptibility to nucleoside analogs. To test the stability of the ExoN(-) genotype and phenotype, we passaged MHV-ExoN(-) 250 times in cultured cells (P250), in parallel with WT-MHV. Compared to MHV-ExoN(-) P3, MHV-ExoN(-) P250 demonstrated enhanced replication, reduced susceptibility to nucleoside analogs, and increased competitive fitness. However, passage did not select for complete or partial reversion at the ExoN-inactivating mutations. We identified novel amino acid changes within the RNA-dependent RNA polymerase (nsp12-RdRp) and nsp14 of MHV-ExoN(-) P250 that partially account for the observed changes in replication, susceptibility to nucleoside analogs, and competitive fitness observed in the passaged virus population, indicating that additional determinants can compensate for the activities of nsp14-ExoN. Our results suggest that while selection favors restoration of replication fidelity in ExoN(-) CoVs, there may be a significant barrier to ExoN(-) reversion. These results also support the hypothesis that high-fidelity replication is linked to CoV fitness and identify additional candidate proteins that may regulate CoV replication fidelity.</p><div id="sec-1" class="subsection"><p id="p-3"><strong>IMPORTANCE</strong> Unique among RNA viruses, CoVs encode a proofreading exoribonuclease (ExoN) in nsp14 that mediates high-fidelity RNA genome replication. Proofreading-deficient CoVs with disrupted ExoN activity [ExoN(-)] are either non-viable or have significant defects in replication, RNA synthesis, fidelity, fitness, and virulence. In this study, we show that ExoN(-) murine hepatitis virus can adapt over long-term passage for increased replication and fitness without reverting the ExoN-inactivating mutations. Passage-adapted ExoN(-) mutants also demonstrate increasing resistance to nucleoside analogs that is only partially explained by secondary mutations in nsp12 and nsp14. These data suggest that enhanced resistance to nucleoside analogs is mediated by the interplay of multiple replicase proteins and support the proposed link between CoV fidelity and fitness.</p></div></div><div class="section" id="sec-2"><h2 class="">INTRODUCTION</h2><p id="p-4">A paradigm of RNA virus biology is error-prone genomic replication due to the lack of proofreading or post-replicative RNA repair mechanisms (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>). Decreased replication fidelity may constrain RNA genome size and complexity and risks the accumulation of deleterious mutations leading to population extinction, a process known as lethal mutagenesis (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>-<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>). While genetic diversity allows viral populations to adapt rapidly under selective pressure, many mutations are neutral or detrimental to viral fitness (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>-<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>). Research performed with many RNA viruses supports the hypothesis that the mutation rate of RNA virus replicases has evolved to balance multiple characteristics of the viral population such as genetic diversity, genomic integrity, and virulence. High-or low-fidelity variants are described for many RNA viruses infecting animals including the coronaviruses (CoVs): murine hepatitis virus (MHV-A59) and Severe Acute Respiratory Syndrome-associated coronavirus (SARS-CoV) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>-<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>), as well as foot-and-mouth disease virus {Arias:2008bp, Zeng:2013dg, Zeng:2014dj, Xie:2014cs, Sierra:2007dg}, poliovirus (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>-<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>), chikungunya virus (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>), influenza virus (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>), coxsackievirus B3 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>), and human enterovirus 71 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>-<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>). Most altered-fidelity variants described to date harbor mutations within the viral RNA-dependent RNA polymerase (RdRp), are attenuated <em>in vivo</em>, and protect against reinfection, highlighting their potential utility as live-attenuated vaccines (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>, <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>). These studies underscore the importance of understanding the molecular mechanisms by which RNA viruses regulate their replication fidelity.</p><p id="p-5">Viruses in the family <em>Coronaviridae</em> have large single-stranded positive-sense RNA genomes [(+)ssRNA] (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>), ranging between 25.4 and 33.5 kilobases in length (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>). CoVs encode a 3′→5′ exoribonuclease (ExoN) in the N-terminal half of nonstructural protein 14 (nsp14-ExoN) (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>). CoV ExoN activity requires the presence of conserved magnesium-coordinating acidic amino acids in three motifs that together constitute the active site (DE-E-D) (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>). The CoV ExoN is grouped with the DE-D-Dh superfamily of exonucleases involved in proofreading during prokaryotic and eukaryotic DNA replication (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>) (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a>-<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>). Alanine substitution of CoV motif I DE residues (DE→AA) reduces biochemical ExoN activity in SARS-CoV (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a>, <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">47</a>) and human coronavirus 229E (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">43</a>). MHV-A59 and SARS-CoV lacking ExoN activity [ExoN(-)] have mutation frequencies 8-to-20-fold greater than WT viruses (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>). Thus, all available data to date support the hypothesis that nsp14-ExoN is the first known proofreading enzyme encoded by an RNA virus.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MHV genome organization and nsp14 exoribonuclease motifs. (Top) The MHV genome is a 31.4kb, capped (dark circle), and polyadenylated positive sense RNA molecule. The first two thirds of the genome encode 16 nonstructural proteins translated as a single polyprotein with a ribosomal frameshift. The final one third encodes the structural proteins. (Inset) Nsp14 encodes an exoribonuclease (solid blue) and N7-methyltransferase (hatched blue) and has 3 zinc fingers (gray boxes) predicted from the solved SARS nsp10/14 crystal structure (PDB 5C8U) (45). Catalytic residues for ExoN are marked with white boxes, and the engineered mutations for MHV-ExoN(-) are shown below the genome. The nsp12 RNA-dependent RNA polymerase is highlighted in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-596209437" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;MHV genome organization and nsp14 exoribonuclease motifs.&lt;/span&gt; (&lt;em&gt;Top&lt;/em&gt;) The MHV genome is a 31.4kb, capped (dark circle), and polyadenylated positive sense RNA molecule. The first two thirds of the genome encode 16 nonstructural proteins translated as a single polyprotein with a ribosomal frameshift. The final one third encodes the structural proteins. (&lt;em&gt;Inset&lt;/em&gt;) Nsp14 encodes an exoribonuclease (solid blue) and N7-methyltransferase (hatched blue) and has 3 zinc fingers (gray boxes) predicted from the solved SARS nsp10/14 crystal structure (PDB 5C8U) (45). Catalytic residues for ExoN are marked with white boxes, and the engineered mutations for MHV-ExoN(-) are shown below the genome. The nsp12 RNA-dependent RNA polymerase is highlighted in red.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F1.medium.gif" width="440" height="322"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F1.medium.gif" width="440" height="322"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">MHV genome organization and nsp14 exoribonuclease motifs.</span><p id="p-6" class="first-child">(<em>Top</em>) The MHV genome is a 31.4kb, capped (dark circle), and polyadenylated positive sense RNA molecule. The first two thirds of the genome encode 16 nonstructural proteins translated as a single polyprotein with a ribosomal frameshift. The final one third encodes the structural proteins. (<em>Inset</em>) Nsp14 encodes an exoribonuclease (solid blue) and N7-methyltransferase (hatched blue) and has 3 zinc fingers (gray boxes) predicted from the solved SARS nsp10/14 crystal structure (PDB 5C8U) (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">45</a>). Catalytic residues for ExoN are marked with white boxes, and the engineered mutations for MHV-ExoN(-) are shown below the genome. The nsp12 RNA-dependent RNA polymerase is highlighted in red.</p><div class="sb-div caption-clear"></div></div></div><p id="p-7">Despite the critical role of ExoN in virus replication, fidelity, fitness, and virulence, ExoN(-) mutants have not reverted the inactivating substitutions (G<span class="underline">AT</span>→G<span class="underline">CA</span> – D89A; G<span class="underline">AA</span>→G<span class="underline">CT</span> – E91A) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>) (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">38</a>). In fact, ExoN-inactivating mutations have been retained during up to 20 passages in culture, eight acute passages of SARS-CoV-ExoN(-) in aged BALB/c mice, and 60 days of persistent SARS-CoV-ExoN(-) infection in immunodeficient Rag-/-mice. In this study, we sought to determine whether long-term passage of MHV-A59-ExoN(-) (250 passages over one year; P250), hereafter MHV-ExoN(-), would result in virus extinction, ExoN(-) reversion, or compensation for the loss of proofreading. We demonstrate that MHV-ExoN(-) did not extinguish during the 250 passages and adapted in a gradual, continuous process for increased replication. MHV-ExoN(-) concurrently evolved reduced susceptibility to nucleoside and base analogs without reversion of the ExoN-inactivating mutations. The evolved mutations in MHV-ExoN(-) nsp14 and nsp12, which encodes the RdRp, accounted for only part of the increased replication and nucleoside analog resistance of MHV-ExoN(-) P250. This study demonstrates a surprising stability of the ExoN-inactivating substitutions, suggests that CoVs can evolve to compensate for the loss of ExoN functions, and indicates that proteins or other determinants outside of nsp12 and nsp14 likely contribute to CoV fidelity regulation.</p></div><div class="section" id="sec-3"><h2 class="">RESULTS</h2><div id="sec-4" class="subsection"><h3>Long-term passage of MHV-ExoN(-)</h3><p id="p-8">We serially passaged WT-MHV and MHV-ExoN(-) in delayed brain tumor (DBT) cells 250 times (P250). Virus from each passage was harvested once 50-100% of the monolayer was involved in syncytia, which occurred between 8 and 24 hours post-infection. Passage conditions varied for WT-MHV and MHV-ExoN(-) due to differences in replication kinetics between the two viruses. We stopped passage at P250 after observing reduced syncytia formation in MHV-ExoN(-)-infected flasks, likely resulting from a mutation in the MHV-ExoN(-) P250 spike protein cleavage site (discussed below).</p></div><div id="sec-5" class="subsection"><h3>MHV-ExoN(-) and WT-MHV replicate with identical kinetics following 250 passages</h3><p id="p-9">MHV-ExoN(-) has a significant replication defect relative to WT-MHV (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a>). We first tested whether replication of MHV-ExoN(-) P250 was affected by long-term passage. MHV-ExoN(-) P3 replication was delayed by ∼2 hours and peak titer reduced by ∼1 log10 relative to WT-MHV P3 during high MOI infection (1 PFU/cell) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a>, solid lines), consistent with our previous studies (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">14</a>). By P250, both viruses replicated with identical kinetics (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2A</a>, dotted lines). This represented a ∼1 log10 increase in peak replication for WT-MHV and a ∼2 log10 increase for MHV-ExoN(-), compared with the respective parental virus. Further, MHV-ExoN(-) P250 titers were maintained throughout the experiment, while WT-MHV P250 titers gradually decreased after 12 hours. We next performed infection at low MOI (0.01 PFU/cell) to examine replication over multiple cycles. Much like at MOI=1, WT-MHV P250 and MHV-ExoN(-) P250 replicated identically during multi-cycle infection, reaching peak titer around 28 hours post-infection (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2B</a>). We also tested replication of MHV-ExoN(-) at P10, P50, P100, and P160. Replication kinetics gradually increased over passage, reaching P250-like levels by P100 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2B</a>). To determine whether the increased replication of MHV-ExoN(-) P250 was affected by the presence of potential defective viral genomes or by some other population-based phenomenon, both WT-MHV P250 and MHV-ExoN(-) P250 were plaque purified three times. The plaque-purified viruses replicated indistinguishably from the parent populations (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2C</a>). Together, these data demonstrate that WT-MHV and MHV-ExoN(-) populations adapted for increased replication and that either individual genomes or those derived from a single virus plaque encoded the adaptive changes required by the total population.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MHV-ExoN(-) evolved increased replicative capacity over long-term passage. Replication curves were performed for the indicated viruses during high MOI (1 PFU/cell) (A) and low MOI (0.01 PFU/cell) (B) infections. (C) WT-MHV P250 and MHV-ExoN(-) P250 were plaque purified three times, and replication curves were performed in parallel with the full population (MOI = 0.01 PFU/cell. Supernatants were collected at the indicated times post infection, and titers were determined by plaque assay. Data for A-C represent mean and standard deviation of n = 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-596209437" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;MHV-ExoN(-) evolved increased replicative capacity over long-term passage.&lt;/span&gt; Replication curves were performed for the indicated viruses during high MOI (1 PFU/cell) (&lt;em&gt;A&lt;/em&gt;) and low MOI (0.01 PFU/cell) (&lt;em&gt;B&lt;/em&gt;) infections. (&lt;em&gt;C&lt;/em&gt;) WT-MHV P250 and MHV-ExoN(-) P250 were plaque purified three times, and replication curves were performed in parallel with the full population (MOI = 0.01 PFU/cell. Supernatants were collected at the indicated times post infection, and titers were determined by plaque assay. Data for &lt;em&gt;A-C&lt;/em&gt; represent mean and standard deviation of n = 3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F2.medium.gif" width="440" height="164"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F2.medium.gif" width="440" height="164"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">MHV-ExoN(-) evolved increased replicative capacity over long-term passage.</span><p id="p-10" class="first-child">Replication curves were performed for the indicated viruses during high MOI (1 PFU/cell) (<em>A</em>) and low MOI (0.01 PFU/cell) (<em>B</em>) infections. (<em>C</em>) WT-MHV P250 and MHV-ExoN(-) P250 were plaque purified three times, and replication curves were performed in parallel with the full population (MOI = 0.01 PFU/cell. Supernatants were collected at the indicated times post infection, and titers were determined by plaque assay. Data for <em>A-C</em> represent mean and standard deviation of n = 3.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>MHV-ExoN(-) accumulated 8-fold more mutations than WT-MHV but did not revert ExoN-inactivating mutations</h3><p id="p-11">To determine whether primary reversion of nsp14-ExoN(-) resulted in enhanced replication of MHV-ExoN(-) P250, we sequenced nsp14 from infected-cell total RNA. MHV-ExoN(-) P250 retained the motif I DE→AA substitutions, and we did not detect any variation in motif I at the level of di-deoxy sequencing at any passage, demonstrating that phenotypic changes were not the result of primary reversion of ExoN(-) motif I. To identify potentially adaptive consensus mutations, we performed full-genome di-deoxy sequencing of MHV-ExoN(-) P250 and WT-MHV P250. Within WT-MHV P250, we identified 23 mutations, of which 17 were nonsynonymous (NS) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a>). In contrast, MHV-ExoN(-) P250 had 171 total mutations (74 NS) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3B</a>). The full-genome sequences have been deposited in GenBank, and the mutations for both viruses are listed in Supplemental Tables 1 and 2. We identified only one mutation shared by both viruses (nsp1 A146T), though it was present in approximately 50% of the WT-MHV P250 population by di-deoxy sequencing. Both viruses deleted most of the hemagglutinin esterase (HE). In MHV-A59, HE mRNA is not transcribed <em>in vitro</em> (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>-<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>), and HE protein expression is detrimental to MHV-A59 fitness in cell culture (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>). WT-MHV P250 also deleted ORF 4a, which is dispensable for MHV replication in cell culture (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>). The C-terminal region of ns2 within MHV-ExoN(-) P250 was truncated and fused to HE with a -1 frameshift. Ns2 is a phosphodiesterase (PDE) that protects viral RNA by degrading 2′→5′ oligoadenylate, the activating factor for cellular RNase L (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>-<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>). The portion of ns2 deleted in MHV-ExoN(-) P250 lies outside the PDE catalytic domain, in a region of unknown function. C-terminally truncated ns2 retains enzymatic activity (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>), but whether this specific deletion and fusion disrupts PDE activity remains to be tested. Nevertheless, ns2 is dispensable for MHV replication in immortalized cells (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>, <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>). Details about the deletion sites are provided in Supplemental Figure 1. Within proteins predicted to be part of the replicase-transcriptase complex (nsp7-16 and nucleocapsid) (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a>), WT-MHV P250 had only one NS change, located in the nsp13-helicase (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3A</a>). In contrast, MHV-ExoN(-) P250 had 17 NS changes within this region (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3B</a> and Supplemental Tables 1, 2).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mutation locations within passaged viruses. Mutations present at &gt;50% by di-deoxy sequencing at passage 250 in WT-MHV (A) and MHV-ExoN(-) (B). Nonsynonymous mutations (red), noncoding mutations (cyan), and deletions (green boxes) are plotted above the schematic, and synonymous mutations (purple) are below." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-596209437" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Mutation locations within passaged viruses.&lt;/span&gt; Mutations present at &gt;50% by di-deoxy sequencing at passage 250 in WT-MHV (&lt;em&gt;A&lt;/em&gt;) and MHV-ExoN(-) (&lt;em&gt;B&lt;/em&gt;). Nonsynonymous mutations (red), noncoding mutations (cyan), and deletions (green boxes) are plotted above the schematic, and synonymous mutations (purple) are below.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F3.medium.gif" width="440" height="400"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F3.medium.gif" width="440" height="400"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Mutation locations within passaged viruses.</span><p id="p-12" class="first-child">Mutations present at &gt;50% by di-deoxy sequencing at passage 250 in WT-MHV (<em>A</em>) and MHV-ExoN(-) (<em>B</em>). Nonsynonymous mutations (red), noncoding mutations (cyan), and deletions (green boxes) are plotted above the schematic, and synonymous mutations (purple) are below.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>MHV-ExoN(-) P250 displays increased genomic RNA accumulation and increased resistance to 5-fluorouracil</h3><p id="p-13">Coronaviruses lacking ExoN consistently display defects in RNA synthesis relative to WT (<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">43</a>). To determine whether the increased replication of MHV-ExoN(-) P250 was associated with restored genomic RNA (gRNA) production, we measured gRNA accumulation over time using two-step real-time quantitative PCR (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>). MHV-ExoN(-) P250 accumulated similar levels of gRNA to WT-MHV P3 and WT-MHV P250 at early time points, while gRNA levels for MHV-ExoN(-) P3 were ∼1 log10 lower (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A</a>). MHV-ExoN(-) P250 gRNA levels fell below those of WT-MHV and WT-MHV P250 after 8 hours and were similar to those of MHV-ExoN(-) P3 at 10 hours post-infection. Normalizing to the gRNA abundance at four hours for each virus demonstrated that the rates of gRNA accumulation were similar for all four viruses (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4B</a>). These data suggest that the increased replication of P250 viruses relative to WT-MHV is not fully accounted for by increased RNA synthesis. In addition to RNA synthesis defects, ExoN(-) CoVs have up-to 20-fold increased mutation frequencies and profoundly increased sensitivity to nucleoside and base analogs relative to WT CoVs (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">38</a>). To determine whether nucleoside analog sensitivity of MHV-ExoN(-) was altered by long-term passage, we treated cells infected with parental and passaged viruses with the base analog, 5-fluorouracil (5-FU). 5-FU is converted intracellularly into a nucleoside analog that incorporates into growing RNA strands and causes A:G and U:C mutations. For simplicity, we hereafter refer to 5-FU as a nucleoside analog. Incorporation of 5-FU is increased in the absence of ExoN activity (<a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">16</a>). All viruses displayed a concentration-dependent decrease in viral titer but differed greatly in their susceptibility to 5-FU (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4C</a>). At 120μM, WT-MHV P3 titers were reduced by ∼1 log10, while MHV-ExoN(-) P3 titers were undetectable (&gt; 5 log10-fold reduction). WT-MHV 5-FU sensitivity was not altered by passage. MHV-ExoN(-) P250 was less susceptible than MHV-ExoN(-) P3 to 5-FU treatment, with only a ∼1.5 log10 decrease in titer at 120 μM. MHV-ExoN(-) P250 remained more sensitive to 5-FU than WT-MHV, suggesting that WT-like resistance requires an intact ExoN. These data demonstrate that MHV-ExoN(-) P3 evolved resistance to 5-FU through mutations outside of ExoN(-) motif I.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MHV-ExoN(-) P250 has increased genomic RNA accumulation and increased resistance to 5-fluorouracil. (A) Cells were infected with the indicated viruses at MOI = 1 PFU/cell, and intracellular RNA was harvested using TRIzol at the indicated times post infection. MHV genomic RNA was detected using SYBR green and primers directed to nsp10, and values were normalized to intracellular GAPDH. (B) Same data as in panel (A) normalized to RNA level for each virus at 4 hours post infection. Data represent mean and standard error for n = 9 (3 triplicate experiments). (C) Sensitivity of passaged viruses to 5-FU during low MOI infection (0.01 PFU/cell). Cells were treated with the indicated concentrations of 5-FU for 30 minutes prior to infection, and supernatants were harvested at 24 hours post-infection and titered by plaque assay. Data represent change in titer relative to untreated control and plotted as mean and standard error of n = 6 (two triplicate experiments). For panel C, statistical significance for change in titer of MHV-ExoN(-) P250 relative to MHV-ExoN(-) P3 was determined Mann-Whitney test (*P" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-596209437" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;MHV-ExoN(-) P250 has increased genomic RNA accumulation and increased resistance to 5-fluorouracil.&lt;/span&gt; (&lt;em&gt;A&lt;/em&gt;) Cells were infected with the indicated viruses at MOI = 1 PFU/cell, and intracellular RNA was harvested using TRIzol at the indicated times post infection. MHV genomic RNA was detected using SYBR green and primers directed to nsp10, and values were normalized to intracellular GAPDH. (&lt;em&gt;B&lt;/em&gt;) Same data as in panel (&lt;em&gt;A&lt;/em&gt;) normalized to RNA level for each virus at 4 hours post infection. Data represent mean and standard error for n = 9 (3 triplicate experiments). (&lt;em&gt;C&lt;/em&gt;) Sensitivity of passaged viruses to 5-FU during low MOI infection (0.01 PFU/cell). Cells were treated with the indicated concentrations of 5-FU for 30 minutes prior to infection, and supernatants were harvested at 24 hours post-infection and titered by plaque assay. Data represent change in titer relative to untreated control and plotted as mean and standard error of n = 6 (two triplicate experiments). For panel C, statistical significance for change in titer of MHV-ExoN(-) P250 relative to MHV-ExoN(-) P3 was determined Mann-Whitney test (*P&lt;0.05, **&lt;0.01, ***P&lt;0.001).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F4.medium.gif" width="440" height="151"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F4.medium.gif" width="440" height="151"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">MHV-ExoN(-) P250 has increased genomic RNA accumulation and increased resistance to 5-fluorouracil.</span><p id="p-14" class="first-child">(<em>A</em>) Cells were infected with the indicated viruses at MOI = 1 PFU/cell, and intracellular RNA was harvested using TRIzol at the indicated times post infection. MHV genomic RNA was detected using SYBR green and primers directed to nsp10, and values were normalized to intracellular GAPDH. (<em>B</em>) Same data as in panel (<em>A</em>) normalized to RNA level for each virus at 4 hours post infection. Data represent mean and standard error for n = 9 (3 triplicate experiments). (<em>C</em>) Sensitivity of passaged viruses to 5-FU during low MOI infection (0.01 PFU/cell). Cells were treated with the indicated concentrations of 5-FU for 30 minutes prior to infection, and supernatants were harvested at 24 hours post-infection and titered by plaque assay. Data represent change in titer relative to untreated control and plotted as mean and standard error of n = 6 (two triplicate experiments). For panel C, statistical significance for change in titer of MHV-ExoN(-) P250 relative to MHV-ExoN(-) P3 was determined Mann-Whitney test (*P&lt;0.05, **&lt;0.01, ***P&lt;0.001).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Spike mutations in MHV-ExoN(-) P250 do not increase resistance to 5-FU</h3><p id="p-15">Bacteriophage ?X174 acquired resistance to 5-FU by delaying cell lysis, thereby reducing the number of replication cycles in which 5-FU can be incorporated (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>). MHV-ExoN(-) P250 had multiple mutations in the spike glycoprotein, including one in the spike furin cleavage site that reduced syncytia formation. To test whether the spike mutations manifested in resistance to 5-FU, we cloned the spike gene from MHV-ExoN(-) P250 into the isogenic MHV-ExoN(-) background. The recombinant virus demonstrated intermediate replication kinetics between MHV-ExoN(-) P3 and MHV-ExoN(-) P250 (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5A</a>) and did not form syncytia. Spike-P250 also increased the specific infectivity of viral particles (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5B</a>). However, the MHV-ExoN(-) P250 spike did not affect sensitivity of the recombinant virus to 5-FU (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5C</a>). Thus, any adaptive increase in 5-FU resistance must be located elsewhere in the genome.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mutations in the spike envelope protein from MHV-ExoN(-) P250 increase replicative capacity but do not affect sensitivity to 5-fluorouracil. (A) Replication kinetics of indicated viruses (MOI = 0.01 PFU/cell) plotted as mean and standard deviation with n = 3. (B) Specific infectivity of indicated viruses 12 hours post-infection (MOI = 1 PFU/cell). Data represent mean and standard error of n = 6 (two triplicate experiments). (C) Sensitivity of indicated viruses to 5-fluorouracil during low MOI infection (0.01 PFU/cell), as described in Fig. 4. Data represent mean and standard error of n = 6 (two triplicate experiments). For panel B, statistical significance was determined using one-way ANOVA. For Panel C, statistical significance for change in titer of MHV-ExoN(-) spike-P250 relative to MHV-ExoN(-) P3 was determined using Mann-Whitney test (*P" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-596209437" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Mutations in the spike envelope protein from MHV-ExoN(-) P250 increase replicative capacity but do not affect sensitivity to 5-fluorouracil.&lt;/span&gt; (&lt;em&gt;A&lt;/em&gt;) Replication kinetics of indicated viruses (MOI = 0.01 PFU/cell) plotted as mean and standard deviation with n = 3. (&lt;em&gt;B&lt;/em&gt;) Specific infectivity of indicated viruses 12 hours post-infection (MOI = 1 PFU/cell). Data represent mean and standard error of n = 6 (two triplicate experiments). (&lt;em&gt;C&lt;/em&gt;) Sensitivity of indicated viruses to 5-fluorouracil during low MOI infection (0.01 PFU/cell), as described in Fig. 4. Data represent mean and standard error of n = 6 (two triplicate experiments). For panel B, statistical significance was determined using one-way ANOVA. For Panel C, statistical significance for change in titer of MHV-ExoN(-) spike-P250 relative to MHV-ExoN(-) P3 was determined using Mann-Whitney test (*P&lt;0.05, **&lt;0.01, ***P&lt;0.001, ns = not significant).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F5.medium.gif" width="334" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F5.medium.gif" width="334" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">Mutations in the spike envelope protein from MHV-ExoN(-) P250 increase replicative capacity but do not affect sensitivity to 5-fluorouracil.</span><p id="p-16" class="first-child">(<em>A</em>) Replication kinetics of indicated viruses (MOI = 0.01 PFU/cell) plotted as mean and standard deviation with n = 3. (<em>B</em>) Specific infectivity of indicated viruses 12 hours post-infection (MOI = 1 PFU/cell). Data represent mean and standard error of n = 6 (two triplicate experiments). (<em>C</em>) Sensitivity of indicated viruses to 5-fluorouracil during low MOI infection (0.01 PFU/cell), as described in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4</a>. Data represent mean and standard error of n = 6 (two triplicate experiments). For panel B, statistical significance was determined using one-way ANOVA. For Panel C, statistical significance for change in titer of MHV-ExoN(-) spike-P250 relative to MHV-ExoN(-) P3 was determined using Mann-Whitney test (*P&lt;0.05, **&lt;0.01, ***P&lt;0.001, ns = not significant).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>MHV-ExoN(-) passage resulted in unique mutations in nsp12 and nsp14</h3><p id="p-17">To date, three proteins have been shown to alter CoV sensitivity to 5-FU: the nsp12-RdRp, nsp14-ExoN, and nsp10, which stimulates ExoN activity (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">39</a>). Neither WT-MHV nor MHV-ExoN(-) P250 contained a NS mutation in nsp10, and WT-MHV P250 had no mutations within either nsp12 or nsp14. In contrast, MHV-ExoN(-) P250 had 7 NS mutations in nsp12 and 6 NS mutations in nsp14 (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figs. 3</a>, <a id="xref-fig-6-1" class="xref-fig" href="#F6">6</a>), none of which have been described previously <em>in vitro</em> or in viable viruses. Within nsp12, six mutations were in the predicted RdRp fingers, palm, and thumb domains (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6A</a>) (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>). Four residues (H709, F766, S776, and M814) can be visualized on a Phyre<sup>2</sup>-modeled structure of the MHV-nsp12 RdRp, while the remainder lie outside the modeled core RdRp (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. 6A</a>) (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">17</a>). One mutation, M288T, lies in the CoV-specific domain, which is conserved among nidoviruses. This domain has been implicated in membrane targeting in MHV-A59 (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>) and performs an essential nucleotidylation activity in the arterivirus, equine arteritis virus (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>). However, M288T is not predicted to catalyze nucleotidylation. Within nsp14, 4 NS mutations were identified in the ExoN domain, and 2 NS mutations were in the C-terminal N7-methyltransferase domain (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig. 6B</a>). We next modeled the structure of MHV nsp14 using Phyre<sup>2</sup> software (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>), resulting in highest-probability similarity to the SARS-CoV nsp14-nsp10 complex (PDB: 5C8S) (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">45</a>) with high-confidence (i.e. the calculated probability of true homology between the structures) of 100% for residues 3-519 of MHV-nsp14. The model predicts that five mutations are located close to surface of the protein (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Fig. 6B</a>). All three modeled zinc finger domains contain one NS mutation (F216Y, Y248H, L473I). Two mutations, D128E and F216Y, are located near the interface between nsp10 and nsp14, though neither site has previously been implicated in nsp10-nsp14 interaction (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>, <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>). One NS mutation resulted in a D272E substitution in ExoN motif III, a metal-coordinating active site residue. We previously reported that alanine substitution of D272 results in an ExoN(-) phenotype (<a id="xref-ref-14-7" class="xref-bibr" href="#ref-14">14</a>), but the viability or phenotype of a D272E substitution was not tested in that study. These data suggest that a network of residues evolved to regulate nsp12 and nsp14 activity or stability in the ExoN(-) background.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The timing of fixation of mutations in nsp12-RdRp and nsp14-ExoN within MHV-ExoN(-) correlates with increased resistance to nucleoside analogs. (A) A schematic of nsp12-RdRp with the CoV-specific region and the canonical fingers, palm, and thumb domains of RdRps is shown. The nsp12-RdRp coding region was sequenced at the indicated passage, and the nonsynonymous changes are plotted; gray boxes indicate consensus changes, and hatched boxes indicate variants present at " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-596209437" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;The timing of fixation of mutations in nsp12-RdRp and nsp14-ExoN within MHV-ExoN(-) correlates with increased resistance to nucleoside analogs.&lt;/span&gt; (&lt;em&gt;A&lt;/em&gt;) A schematic of nsp12-RdRp with the CoV-specific region and the canonical fingers, palm, and thumb domains of RdRps is shown. The nsp12-RdRp coding region was sequenced at the indicated passage, and the nonsynonymous changes are plotted; gray boxes indicate consensus changes, and hatched boxes indicate variants present at &lt;50% of the population by di-deoxy sequencing. At right, mutations are marked in orange on a Phyre&lt;sup&gt;2&lt;/sup&gt;-modeled structure of MHV-nsp12, with the active site residues marked in yellow (17). RdRp domains are colored according to the linear schematic. &lt;em&gt;NB&lt;/em&gt;: M288T, L376P, and D913E lie outside the modeled region and thus are not marked. (&lt;em&gt;B&lt;/em&gt;) A schematic of nsp14 with the ExoN and N7-methyltransferase domains is shown, with mutation plotting as in panel A. The black box denotes a mutation to ExoN motif III. At right, mutations are marked in orange on a Phyre&lt;sup&gt;2&lt;/sup&gt;-modeled structure of the MHV-nsp14/nsp10 complex. Domains are colored according to the linear schematic.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F6.medium.gif" width="440" height="308"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F6.medium.gif" width="440" height="308"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <span class="caption-title">The timing of fixation of mutations in nsp12-RdRp and nsp14-ExoN within MHV-ExoN(-) correlates with increased resistance to nucleoside analogs.</span><p id="p-18" class="first-child">(<em>A</em>) A schematic of nsp12-RdRp with the CoV-specific region and the canonical fingers, palm, and thumb domains of RdRps is shown. The nsp12-RdRp coding region was sequenced at the indicated passage, and the nonsynonymous changes are plotted; gray boxes indicate consensus changes, and hatched boxes indicate variants present at &lt;50% of the population by di-deoxy sequencing. At right, mutations are marked in orange on a Phyre<sup>2</sup>-modeled structure of MHV-nsp12, with the active site residues marked in yellow (<a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">17</a>). RdRp domains are colored according to the linear schematic. <em>NB</em>: M288T, L376P, and D913E lie outside the modeled region and thus are not marked. (<em>B</em>) A schematic of nsp14 with the ExoN and N7-methyltransferase domains is shown, with mutation plotting as in panel A. The black box denotes a mutation to ExoN motif III. At right, mutations are marked in orange on a Phyre<sup>2</sup>-modeled structure of the MHV-nsp14/nsp10 complex. Domains are colored according to the linear schematic.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Fixed mutations in nsp12 and nsp14 in MHV-ExoN(-) P250 directly correlate with increased resistance to multiple nucleoside analogs</h3><p id="p-19">To determine approximately when the mutations in nsp12 and nsp14 arose, we performed di-deoxy sequencing across these protein-coding regions roughly every 20 passages (P10, 31, 50, 72, 90, 100, 120, 140, 160, 180, 200, 220, 240). By this method, we detected consensus NS mutations at P10, P50, and P160 for nsp12, and at P50 and P160 for nsp14 (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Fig. 6</a>). Both nsp12 and nsp14 carried their full complement of P250 consensus mutations by P160, except for a minority variant (D913E) in nsp12 maintained at &lt;50% of the population between P200 and P250. These passage levels correlated with increased replication of MHV-ExoN(-) (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2B</a>) and with decreasing sensitivity to 5-FU (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 7A</a>). Neither replication nor 5-FU sensitivity of MHV-ExoN(-) changed substantially between P160 and P250. To determine whether MHV-ExoN(-) evolved increased resistance to multiple nucleoside analogs, we treated virus-infected cells with three additional analogs that are substrates for viral RdRps: ribavirin (RBV), a guanine analog that inhibits viral replication through multiple mechanisms, including mutagenesis and inhibition of purine biosynthesis (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>); 5-azacytidine (AZC), an RNA mutagen (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>); and 2’-C-methyladenosine (CMeA), which is proposed to incorporate in viral RNA and terminate nascent transcripts (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>). As with 5-FU, we observed dose-dependent sensitivity to RBV, AZC, and CMeA in all MHV-ExoN(-) viruses that decreased with increasing passage number (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Fig. 7B</a>-D). Except against AZC, MHV-ExoN(-) sensitivity did not change between P160 and P250. Together, these data demonstrate that MHV-ExoN(-) evolved increased resistance to multiple nucleoside analogs that correlated with the length of passage and the acquisition of mutations in nsp12 and nsp14. Importantly, this occurred in the absence of specific mutagenic selection and without reversion of ExoN motif I. This increased general selectivity towards all four classes of nucleotide strongly advocates for an overall increase in fidelity in MHV-ExoN(-) P250.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MHV-ExoN(-) evolved increased resistance to nucleoside and base analogs over passage. Sensitivity of isogenic and passaged viruses to the nucleoside analogs, 5-FU (A), ribavirin (B), 5-azacytidine (C), and 2&#x2019;-C-methyladenosine (D) during low MOI infection (0.01 PFU/cell) as described in Fig. 4. Data represent mean and standard error for n = 6 (2 triplicate experiments). NB: Data for WT-MHV, WT-MHV P250, MHV-ExoN(-) P3, MHV-ExoN(-) P250 in panel (A) are the same as shown in Fig. 4C. Statistical significance for change in titer of MHV-ExoN(-) P3 relative to MHV-ExoN(-) P250 was determined using Mann-Whitney test (*P" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-596209437" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;MHV-ExoN(-) evolved increased resistance to nucleoside and base analogs over passage.&lt;/span&gt; Sensitivity of isogenic and passaged viruses to the nucleoside analogs, 5-FU (&lt;em&gt;A&lt;/em&gt;), ribavirin (&lt;em&gt;B&lt;/em&gt;), 5-azacytidine (&lt;em&gt;C&lt;/em&gt;), and 2&#x2019;-C-methyladenosine (&lt;em&gt;D&lt;/em&gt;) during low MOI infection (0.01 PFU/cell) as described in Fig. 4. Data represent mean and standard error for n = 6 (2 triplicate experiments). &lt;em&gt;NB&lt;/em&gt;: Data for WT-MHV, WT-MHV P250, MHV-ExoN(-) P3, MHV-ExoN(-) P250 in panel (&lt;em&gt;A&lt;/em&gt;) are the same as shown in Fig. 4C. Statistical significance for change in titer of MHV-ExoN(-) P3 relative to MHV-ExoN(-) P250 was determined using Mann-Whitney test (*P&lt;0.05, **&lt;0.01, ***P&lt;0.001).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F7.medium.gif" width="171" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F7.medium.gif" width="171" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7.</span> <span class="caption-title">MHV-ExoN(-) evolved increased resistance to nucleoside and base analogs over passage.</span><p id="p-20" class="first-child">Sensitivity of isogenic and passaged viruses to the nucleoside analogs, 5-FU (<em>A</em>), ribavirin (<em>B</em>), 5-azacytidine (<em>C</em>), and 2’-C-methyladenosine (<em>D</em>) during low MOI infection (0.01 PFU/cell) as described in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4</a>. Data represent mean and standard error for n = 6 (2 triplicate experiments). <em>NB</em>: Data for WT-MHV, WT-MHV P250, MHV-ExoN(-) P3, MHV-ExoN(-) P250 in panel (<em>A</em>) are the same as shown in <a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4C</a>. Statistical significance for change in titer of MHV-ExoN(-) P3 relative to MHV-ExoN(-) P250 was determined using Mann-Whitney test (*P&lt;0.05, **&lt;0.01, ***P&lt;0.001).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Mutations in nsp12 partially account for increased resistance of MHV-ExoN(-) P250 to multiple nucleoside analogs</h3><p id="p-21">We hypothesized that mutations in MHV-ExoN(-) P250 nsp12 and nsp14 were most likely to impact replication and nucleoside analog sensitivity based on their enzymatic activities and temporal association with phenotypic changes. To test this hypothesis, we engineered recombinant MHV-ExoN(-) to encode the P250 nsp12 and nsp14, alone and together. Expression of nsp12-P250 and nsp14-P250, alone or in combination, altered replication kinetics of MHV-ExoN(-) without affecting peak titers (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Fig. 8A</a>) and increased gRNA levels above those of MHV-ExoN(-) P3 (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Fig. 8B</a>). Nsp12-P250 had a greater effect than nsp14-P250 on the sensitivity of MHV-ExoN(-) to all analogs tested, and the combination of nsp12-and nsp14-P250 did not increase resistance above nsp12-P250 alone (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Fig. 8C</a>-E). None of the recombinant viruses recapitulated the resistance phenotypes of the MHV-ExoN(-) P250 population. Together, these data demonstrate that nsp12-P250 mutations only partially account for the nucleoside analog resistance of MHV-ExoN(-) P250, and that adaptations in nsp12-P250 mask those in nsp14-P250. We also can conclude that the nsp14-P250 D272E active site mutation does not correct the defect caused by the motif I DE→AA substitutions. Together, the results suggest possible limitations for fidelity adaptation in nsp12 and nsp14, and also the exciting prospect that other CoV proteins participate in fidelity regulation within MHV-ExoN(-) P250.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mutations in nsp12-RdRp and nsp14-ExoN from MHV-ExoN(-) P250 incompletely increase resistance to RNA mutagens and increase fitness of MHV-ExoN(-). (A) Replication kinetics of recombinant P250 viruses (MOI = 0.01 PFU/cell) plotted as mean and standard deviation with n = 3. (B) Genomic RNA accumulation relative to intracellular GAPDH, as described in Fig. 4. Data represent mean and standard error for n = 6-9 (2-3 triplicate experiments). (C-E) Sensitivity of recombinant MHV-ExoN(-) viruses to 5-FU (C), ribavirin (D), and 5-azacytidine (E) during low MOI infection (0.01 PFU/cell) as described in Fig. 4. Data represent mean and standard error of n = 6. (F) Recombinant viruses were competed against a reference MHV-ExoN(-) containing 10 silent mutations within nsp2. The ratio of viral genome copies relative to the MHV-ExoN(-) reference is plotted. Data represent mean and standard error of n = 6. NB: MHV-ExoN(-) P250 data set contains 4 replicates at passage 3 and a single replicate at passage 4 due to undetectable levels of MHV-ExoN(-) silent. For panels C-E, statistical significance for change in titer of swapped viruses relative to MHV-ExoN(-) P3 at highest drug concentration tolerated was determined using Mann-Whitney test (*P" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-596209437" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Mutations in nsp12-RdRp and nsp14-ExoN from MHV-ExoN(-) P250 incompletely increase resistance to RNA mutagens and increase fitness of MHV-ExoN(-).&lt;/span&gt; (&lt;em&gt;A&lt;/em&gt;) Replication kinetics of recombinant P250 viruses (MOI = 0.01 PFU/cell) plotted as mean and standard deviation with n = 3. (&lt;em&gt;B&lt;/em&gt;) Genomic RNA accumulation relative to intracellular GAPDH, as described in Fig. 4. Data represent mean and standard error for n = 6-9 (2-3 triplicate experiments). (&lt;em&gt;C-E&lt;/em&gt;) Sensitivity of recombinant MHV-ExoN(-) viruses to 5-FU (&lt;em&gt;C&lt;/em&gt;), ribavirin (&lt;em&gt;D&lt;/em&gt;), and 5-azacytidine (&lt;em&gt;E&lt;/em&gt;) during low MOI infection (0.01 PFU/cell) as described in Fig. 4. Data represent mean and standard error of n = 6. (&lt;em&gt;F&lt;/em&gt;) Recombinant viruses were competed against a reference MHV-ExoN(-) containing 10 silent mutations within nsp2. The ratio of viral genome copies relative to the MHV-ExoN(-) reference is plotted. Data represent mean and standard error of n = 6. &lt;em&gt;NB&lt;/em&gt;: MHV-ExoN(-) P250 data set contains 4 replicates at passage 3 and a single replicate at passage 4 due to undetectable levels of MHV-ExoN(-) silent. For panels C-E, statistical significance for change in titer of swapped viruses relative to MHV-ExoN(-) P3 at highest drug concentration tolerated was determined using Mann-Whitney test (*P&lt;0.05, **&lt;0.01, ***P&lt;0.001, ns = not significant). For panel F, statistical significance for the indicated comparisons was determined using Mann-Whitney test. Boxed points have the same P value.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F8.medium.gif" width="440" height="278"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8." src="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F8.medium.gif" width="440" height="278"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/08/11/175562/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8.</span> <span class="caption-title">Mutations in nsp12-RdRp and nsp14-ExoN from MHV-ExoN(-) P250 incompletely increase resistance to RNA mutagens and increase fitness of MHV-ExoN(-).</span><p id="p-22" class="first-child">(<em>A</em>) Replication kinetics of recombinant P250 viruses (MOI = 0.01 PFU/cell) plotted as mean and standard deviation with n = 3. (<em>B</em>) Genomic RNA accumulation relative to intracellular GAPDH, as described in <a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig. 4</a>. Data represent mean and standard error for n = 6-9 (2-3 triplicate experiments). (<em>C-E</em>) Sensitivity of recombinant MHV-ExoN(-) viruses to 5-FU (<em>C</em>), ribavirin (<em>D</em>), and 5-azacytidine (<em>E</em>) during low MOI infection (0.01 PFU/cell) as described in <a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig. 4</a>. Data represent mean and standard error of n = 6. (<em>F</em>) Recombinant viruses were competed against a reference MHV-ExoN(-) containing 10 silent mutations within nsp2. The ratio of viral genome copies relative to the MHV-ExoN(-) reference is plotted. Data represent mean and standard error of n = 6. <em>NB</em>: MHV-ExoN(-) P250 data set contains 4 replicates at passage 3 and a single replicate at passage 4 due to undetectable levels of MHV-ExoN(-) silent. For panels C-E, statistical significance for change in titer of swapped viruses relative to MHV-ExoN(-) P3 at highest drug concentration tolerated was determined using Mann-Whitney test (*P&lt;0.05, **&lt;0.01, ***P&lt;0.001, ns = not significant). For panel F, statistical significance for the indicated comparisons was determined using Mann-Whitney test. Boxed points have the same P value.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Resistance to nucleoside analogs correlates with MHV-ExoN(-) fitness</h3><p id="p-23">We hypothesized that mutations in nsp12 and nsp14 provided a fitness advantage to MHV-ExoN(-) P250. We competed the recombinant viruses with a reference MHV-ExoN(-) virus (P1 stock) containing 10 silent mutations in the nsp2 coding region. Mutant and reference viruses were detected in the mixed infection by real-time quantitative PCR using dual-labeled probes specific for each virus. MHV-ExoN(-) P3 showed a modest fitness advantage over the reference P1 MHV-ExoN(-) silent (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Fig. 8F</a>, solid green). MHV-ExoN(-) P250 profoundly outcompeted MHV-ExoN(-) silent, with &gt;1000-fold more MHV-ExoN(-) P250 genomes present at the end of passage 1 (<a id="xref-fig-8-5" class="xref-fig" href="#F8">Fig. 8F</a>, dotted green). MHV-ExoN(-) nsp12-P250 had greater relative fitness than MHV-ExoN(-) nsp14-P250, and MHV-ExoN(-) nsp12/14-P250 was intermediate between the single recombinants, implicating a complex evolutionary interaction between these two proteins. The measured fitness correlated with the patterns of nucleoside analog resistance and RNA synthesis associated with mutations in nsp12 and nsp14, suggesting a link between the evolution of these virus phenotypes.</p><p id="p-24">The result also confirms that nsp12 and nsp14 are important but not sufficient to account for the significantly increased fitness of MHV-ExoN(-) P250 relative to MHV-ExoN(-) P3.</p></div></div><div class="section" id="sec-13"><h2 class="">DISCUSSION</h2><p id="p-25">High-fidelity replication is proposed to be critical for maintaining the large (+)ssRNA genomes of CoVs (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>, <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>). Our previous studies demonstrate that ExoN-mediated proofreading is required for replicative fitness, nucleoside analog resistance, and virulence (<a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">38</a>). Yet no direct test of the requirement for ExoN activity for long-term survival and stability of the CoV genome had been performed. In this study, we describe experimental adaptive evolution of WT-MHV and MHV-ExoN(-) during long-term passage in cell culture. WT-MHV evolved increased replication kinetics over 250 passages, with few consensus mutations arising in the WT-MHV P250 genome. In contrast, MHV-ExoN(-) accumulated 8-fold more mutations than WT-MHV, none of which occurred at the ExoN-inactivating mutations. Nevertheless, MHV-ExoN(-) P250 demonstrated increased replication kinetics and increased resistance to nucleoside analogs as compared to MHV-ExoN(-) P3. Mutations within nsp12-P250 conferred greater resistance to nucleoside analogs than nsp14-P250, suggesting that secondary mutations within nsp14 cannot restore ExoN activity without the full complement of catalytic DE-E-D residues. These results further support our studies and those from other exonucleases that the motif I DE residues are absolutely required for ExoN activity. However, mutations in nsp12-P250 also were not sufficient, alone or in combination with nsp14-P250, to account for the increased nucleoside analog resistance of the MHV-ExoN(-) P250 virus population, indicating that other proteins contribute to resistance or that the virus evolved increased mutational robustness over passage.</p><div id="sec-14" class="subsection"><h3>Why doesn’t MHV-ExoN(-) revert over long-term passage?</h3><p id="p-26">We detected no primary reversion at the DE→AA substitutions MHV-ExoN(-) at any passage tested. These data are consistent with and significantly extend previous studies reporting genotypic stability of ExoN(-) motif I in MHV and SARS-CoV (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-14-8" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-17-8" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">38</a>). Complete reversion within ExoN(-) motif I to DE would require four nucleotide changes. This likely represents a high genetic barrier to reversion, especially given that fitness can be increased by mutations outside of nsp14-ExoN. Single and double nucleotide changes within motif I could restore acidic charge to individual residues (e.g. motif I EA, AD, ED, etc). However, the active site compositions of DEDDh exonucleases, such as the Klenow fragment, are so stringent that even conservative mutations (D-to-E or E-to-D) reduce ExoN activity by &gt;96% (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>). Thus, intermediate amino acid changes may not have a selective advantage compared to motif I AA, limiting the evolutionary pathways to reversion. However, nsp14-P250 increased competitive fitness in the MHV-ExoN(-) background (<a id="xref-fig-8-6" class="xref-fig" href="#F8">Fig. 8F</a>), demonstrating a modest capacity for fitness adaptation in nsp14 outside of the catalytic residues. Whether these mutations resulted from genetic drift or positive selection remains unclear. Nevertheless, our data show that MHV-ExoN(-) can adapt for increased fitness without fully restoring exoribonuclease activity. Understanding the mechanisms by which MHV-ExoN(-) P250 compensated for ExoN activity could allow recovery of ExoN(-) variants of other CoVs, such as transmissible gastroenteritis virus and human CoV 229E, which to date have been unrecoverable as ExoN(-) recombinants (<a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">43</a>, <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a>).</p></div><div id="sec-15" class="subsection"><h3>CoV adaptation to loss of ExoN-mediated proofreading</h3><p id="p-27">High-fidelity replication, made possible in part by the proofreading activity of nsp14-ExoN, is thought to have been essential for the expansion and maintenance of large CoV genomes (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">39</a>, <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">40</a>, <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">69</a>). Compensation for the loss of ExoN could have occurred through the evolution of increased fidelity via other replicase proteins. Mutations conferring increased replication fidelity to RNA viruses have most frequently been mapped to RdRps (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a>). Three findings suggest that mutations within nsp12-P250 confer increased polymerase fidelity. First, nonsynonymous mutations to the RdRp arose in the low-fidelity MHV-ExoN(-) but not in the presence of proofreading (WT-MHV). Second, five of the mutations lie in or near structural motifs important for fidelity regulation in other RdRps.</p><p id="p-28">Amino acid substitutions in the fingers and palm domains have been repeatedly shown to affect viral RdRp fidelity (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>), and we have recently reported a fingers mutation (nsp12-V553I) that likely increases the fidelity of the MHV RdRp (<a id="xref-ref-17-9" class="xref-bibr" href="#ref-17">17</a>). Our modeled structure predicts that nsp12-P250 contains three mutations in the palm domain and one in the fingers domain, with the M814K thumb domain mutation lying near the palm (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Fig. 6A</a>). Third, exchange of nsp12-P250 alone into the background of MHV-ExoN(-) reduced the susceptibility of MHV-ExoN(-) to three different nucleoside analogs (<a id="xref-fig-8-7" class="xref-fig" href="#F8">Fig. 8</a>). Although resistance to a single nucleoside analog can evolve without increasing overall fidelity, resistance to multiple nucleoside analogs suggests a broadly increased capacity to discriminate modified nucleotides {Arias:2008bp, Sierra:2007dg, Zeng:2013dg, Guo:2016fx}. Thus, nsp12-P250 is likely a high-fidelity polymerase. However, nsp12-P250 only partially accounts for the MHV-ExoN(-) P250 nucleoside analog resistance phenotype (<a id="xref-fig-8-8" class="xref-fig" href="#F8">Fig. 8</a>), suggesting a possible limit to the compensation achievable by mutating the RdRp alone. Further, the effects of mutations in nsp12-P250 and nsp14-P250 are not additive and may be antagonistic when isolated from the whole passaged virus (<a id="xref-fig-8-9" class="xref-fig" href="#F8">Fig. 8</a>), indicating that the relationships between nsp12-and nsp14-P250 mutations are likely evolutionarily linked with those in other MHV proteins. In fact, a substantial component of the evolved resistance to nucleoside analogs cannot be explained by nsp12-P250 and nsp14-P250, alone or together. In support of this hypothesis, we identified several nonsynonymous mutations in other replicase proteins, such as nsps 8, 9, 13, and 15. SARS-CoV nsp8 and nsp13 have functional interactions with nsp12, acting as a primase/processivity factor (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a>, <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a>) and a helicase/NTPase, respectively (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a>). Processivity factors in herpes simplex virus and <em>Mycobacterium tuberculosis</em> regulate DNA polymerase fidelity by balancing polymerase extension and exonuclease activity (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a>, <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>), and helicases in chikungunya virus and foot-and-mouth disease virus can evolve to increase fidelity (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a>) and alter the frequency of ribavirin-induced mutations (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a>), respectively. SARS-CoV nsp9 has RNA-binding activities and is proposed to participate in the multi-protein replicase complex (<a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">39</a>, <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a>), and MHV nsp15 is a uridylate-specific endoribonuclease (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a>). Both could plausibly be involved in modifying polymerase activity. Additionally, it remains possible that evolution for increased fidelity could involve proteins outside the canonical replication complex (nsps7-16), including those in the structural and accessory cassette. Thus, while immediate studies will focus on testing whether replicase proteins nsp8, 9, 13, and 15 regulate fidelity, it is exciting to consider the possibility that this virus-directed discovery approach will reveal novel interactions between multiple MHV proteins.</p></div><div id="sec-16" class="subsection"><h3>Has MHV-ExoN(-) P250 evolved robustness as a defense against ExoN(-) infidelity?</h3><p id="p-29">Three of the nucleoside analogs used in this study (5-FU, ribavirin, and 5-azacytidine) reduce virus titers at least in part through mutagenesis (<a id="xref-ref-16-8" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">66</a>, <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">67</a>). Although the resistance of MHV-ExoN(-) P250 to these RNA mutagens may be fully explained by the evolution of a high-fidelity replication complex, an additional possibility is that MHV-ExoN(-) P3 evolved increased mutational robustness. Mutational robustness is the capacity of a virus to buffer the fitness effects of mutations, which are most frequently detrimental or lethal (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>-<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>). Thus, in a low-fidelity virus like MHV-ExoN(-) P3, genomic mutations which reduce the fitness cost of subsequent mutations can provide a selective advantage (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">83</a>-<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">85</a>). The relationship between fidelity and robustness has been demonstrated for poliovirus and coxsackievirus B3: wild-type poliovirus encodes a lower-fidelity polymerase than coxsackievirus and is more mutationally robust (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">86</a>). Selection for increased robustness could explain the ∼90 synonymous changes in MHV-ExoN(-) P250. Synonymous changes can alter codons to reduce the probability of non-conservative amino acid changes (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">87</a>). Alternatively, the increased population size of MHV-ExoN(-) P250 could promote robustness by a “safety-in-numbers” effect, allowing efficient purging of low-fitness mutants while maintaining population fitness (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">88</a>). Large populations also increase the likelihood of co-infection, allowing complementation between viral genomes. We did not observe an effect of increased replication in MHV-ExoN(-) P250 nucleoside analog resistance (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5</a>), but a recent study with poliovirus suggests that mutagenized populations have elevated coinfection frequencies (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">89</a>). Thus, complementation may contribute to MHV-ExoN(-) P250 nucleoside analog resistance. Conflicting evidence exists regarding whether mutational robustness itself affects the sensitivity to RNA mutagens (<a id="xref-ref-86-2" class="xref-bibr" href="#ref-86">86</a>, <a id="xref-ref-87-2" class="xref-bibr" href="#ref-87">87</a>, <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">90</a>); nevertheless, the robustness of MHV-ExoN(-) P250 merits further investigation.</p></div><div id="sec-17" class="subsection"><h3>Conclusions</h3><p id="p-30">One of the defining features of CoV replication is the proofreading activity of the nsp14 exoribonuclease that is a critical determinant of CoV replication, fidelity, and fitness. We here show that CoVs also have the capacity to compensate for loss of ExoN activity through a network of mutations in nsp12, nsp14, and elsewhere in the genome. Thus, while nsp14-ExoN appears to play a dominant role in CoV replication fidelity, its activity is likely closely tied to a highly evolved network of proteins. The demonstrated co-adaptation for replication, fidelity, and competitive fitness within MHV-ExoN(-) supports the hypothesis that these roles are linked functionally and evolutionarily. Genetic and biochemical testing of the rich mutational resource revealed in MHV-ExoN(-) P250 will likely inform the design of countermeasures for endemic and emerging CoVs by defining novel common targets for stable virus attenuation or direct inhibition.</p></div></div><div class="section" id="sec-18"><h2 class="">MATERIALS AND METHODS</h2><div id="sec-19" class="subsection"><h3>Cell culture</h3><p id="p-31">DBT-9 (delayed brain tumor, murine astrocytoma clone 9) cells were maintained as described previously (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">91</a>). Baby hamster kidney (BHK) cells stably expressing the MHV-A59 receptor, CEACAM1, [BHK-R, (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">15</a>)] were maintained under selection with 0.8 mg/ml of G418 (Mediatech) as described previously (<a id="xref-ref-91-2" class="xref-bibr" href="#ref-91">91</a>).</p></div><div id="sec-20" class="subsection"><h3>Long-term passage of virus and stock generation</h3><p id="p-32">The infectious cDNA clone for MHV-A59 and the recovery of MHV-ExoN(-) are described previously (<a id="xref-ref-14-9" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-91-3" class="xref-bibr" href="#ref-91">91</a>). Long-term passage was initiated by infecting subconfluent monolayers of DBT-9 cells in 25 cm2 flasks with either wild-type MHV-A59 or MHV-ExoN(-) at a multiplicity of infection (MOI) of approximately 0.1 particle forming unit per cell (PFU/cell). One lineage of each virus was passaged blindly for a total of 250 passages (P250). Supernatant was harvested at each passage and stored at -80°C. Total RNA was harvested for most passages using 1 ml of TRIzol Reagent (Ambion) per 25 cm2 flask and stored at -80°C. Virus stocks of select intermediate passages were generated by infecting a subconfluent 150 cm2 flask of DBT-9 cells at an MOI of 0.01 PFU/cell.</p><p id="p-33">Approximately 24 hours post-infection the flask was frozen at -80°C and the supernatant was clarified by centrifugation at 4,000 x g (Sorvall RC 3B Plus; HA-6000A rotor) for 10 min at 4°C. The virus titer of each stock was determined by plaque assay using DBT-9 cells as described previously (<a id="xref-ref-14-10" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-91-4" class="xref-bibr" href="#ref-91">91</a>). For plaque assays of viruses containing the spike protein from MHV-ExoN(-) P250, which does not form syncytia, plaques were visualized with neutral red (Sigma #N6264, diluted 1:10 in PBS containing calcium and magnesium). Neutral red was added 24 hours after plating and incubated for an additional 3-8 hours before formaldehyde fixation. Plaque purification was performed by infecting DBT cells with serial dilutions of virus and overlaying the cultures with agar. Single plaques were isolated, resuspended in PBS containing calcium and magnesium, and inoculated onto fresh DBTs. This process was completed 3 times before experimental stocks were generated, as above.</p></div><div id="sec-21" class="subsection"><h3>Sequencing of virus stocks</h3><p id="p-34">Following P250, RNA was purified from the harvested TRIzol samples according to the manufacturer’s protocol and reverse transcribed (RT) using SuperScript III (Invitrogen) as described previously (<a id="xref-ref-14-11" class="xref-bibr" href="#ref-14">14</a>). Full-genome di-deoxy sequencing was performed for both WT-MHV P250 and MHV-ExoN(-) P250 using 12 overlapping amplicons approximately three kilobases in length. All coding regions were sequenced fully, and out of 31,409 nucleotides, &gt; 99% were sequenced for each virus [WT-MHV P250: 21-31,279 and MHV-ExoN(-) P250: 21-31,275]. Two microliters of RT product were used for each PCR reaction (<a id="xref-ref-16-9" class="xref-bibr" href="#ref-16">16</a>). Di-deoxy sequencing was performed by Genhunter Corporation (Nashville, TN) or GENEWIZ (South Plainfield, NJ). Sequence analysis was performed using MacVector version 14 (MacVector, Inc.; Apex, North Carolina) using the MHV-A59 infectious clone reference genome (GenBank accession no. <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AY910861&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom">AY910861</a>). The nucleotide sequences of the amplicon and sequencing primers are available upon request. Sequencing of nsp12 and nsp14 from intermediate passages was performed using TRIzol-purified RNA from infected monolayers and using the primers listed below. Primers 6M1F (5’-TTTTGGCGAGATGGTAGC-3’) and 7M2R (5’-GGTAAGACAGTTTTAGGTGAG-3’) were used to generate a 3,425 nucleotide amplicon containing all of nsp12. Primers 7M3F (5’-ATGCTTACCAACTATGAGC-3’) and 8M3R (5’-CCGATTTGAATGGCGTAG-3’) were used to generate a 2,713 nucleotide amplicon containing all of nsp14. The PCR conditions for these reactions are the same as those used to generate the amplicons used for full-genome sequencing (<a id="xref-ref-16-10" class="xref-bibr" href="#ref-16">16</a>).</p></div><div id="sec-22" class="subsection"><h3>Replication and RNA synthesis kinetics</h3><p id="p-35">Viral replication kinetics in DBT-9 cells were determined at an MOI of 1 PFU/cell or MOI of 0.01 PFU/cell as described previously (<a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">15</a>). Supernatant (300 μL) was harvested at the indicated time points, and the virus titer was determined by plaque assay. The accumulation of genomic RNA at an MOI of 1 PFU/cell was measured by two-step real-time quantitative RT-PCR (RT-qPCR) using intracellular RNA that was TRIzol-extracted at the time points indicated. RNA was reverse-transcribed using SuperScript III (Invitrogen), and cDNA derived from intracellular positive-sense viral RNA was measured using primers directed to nsp10. Values were normalized to levels of the endogenous control glyceraldehyde-3-phosphate dehydrogenase (GAPDH). No mutations within the primer binding sites emerged in either P250 population. The primers and amplification conditions are the same as reported previously (<a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">15</a>), except that the RT product was diluted 1:10 prior to use. Samples were plated in technical duplicate to minimize well-to-well variation. Data are presented as 2-ΔCT, where ΔCT denotes CT (target, nsp10) minus CT (reference, GAPDH).</p></div><div id="sec-23" class="subsection"><h3>Determination of specific infectivity</h3><p id="p-36">Subconfluent monolayers of DBT-9 cells in 24-well plates were infected with the indicated virus at an MOI of 1 PFU/cell, and supernatant was harvested at 12 h.p.i. The levels of genomic RNA in supernatant were measured using one-step real-time quantitative RT-PCR (RT-qPCR) on TRIzol-extracted RNA as described previously (<a id="xref-ref-17-10" class="xref-bibr" href="#ref-17">17</a>). Briefly, genomic RNA was detected with a 5’ 6-carboxyfluorescein (FAM) and 3’ black hole quencher 1 (BHQ-1) labeled probe targeting nsp2 (Biosearch Technologies, Petaluma, CA), and RNA copy number was calculated by reference to an RNA standard derived from the MHV A fragment. Samples were plated in technical duplicate to minimize well-to-well variation. Titers were determined by plaque assay in DBT-9 cells, and specific infectivity was calculated as PFU per supernatant genomic RNA copy.</p></div><div id="sec-24" class="subsection"><h3>Nucleoside and base analog sensitivity assays</h3><p id="p-37">5-azacytidine (AZC), 5-fluorouracil (5-FU), and ribavirin (RBV) were purchased from Sigma (product numbers A2385, F6627, and R9644, respectively). Stock solutions of 5-FU and RBV were prepared in dimethyl sulfoxide (DMSO). 2’-C-methyladenosine (CMeA) was received from Gilead Sciences, Inc (Foster City, CA). Sensitivity assays were performed as described previously (<a id="xref-ref-16-11" class="xref-bibr" href="#ref-16">16</a>), except in 24-well plates at an MOI of 0.01 PFU/cell. Supernatants were harvested at 24 hours post-infection, and titers were determined by plaque assay.</p></div><div id="sec-25" class="subsection"><h3>Phyre<sup>2</sup>-modeling of MHV-nsp14</h3><p id="p-38">The MHV nsp14 structure was modeled with the Phyre<sup>2</sup> online program (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">63</a>) using nsp14 residues 3-519, corresponding to residues 6056-6573 of the ORF1ab polyprotein. The model was analyzed using the Pymol Molecular Graphics System, Version (Schrödinger, LLC).</p></div><div id="sec-26" class="subsection"><h3>Generation of nsp12 and nsp14 swaps</h3><p id="p-39">Viruses containing nsp12, nsp14, or nsp12 and nsp14 from MHV-ExoN(-) P250 were generated using the MHV-A59 reverse genetics system (<a id="xref-ref-91-5" class="xref-bibr" href="#ref-91">91</a>). RNA from the MHV-ExoN(-) P250 virus was reversed transcribed with SuperScript III (Invitrogen) and used to generate amplicons containing either nsp12 or nsp14. Each amplicon was flanked by 15 bp that overlapped with an amplicon generated from the backbone plasmid. Amplicons were inserted into MHV-A59 fragments using the InFusion HD Cloning kit (Takara Bio USA, Inc, Mountain View, CA). Nsp12 is split across MHV E and F, while nsp14 is contained within MHV F. Reaction mixtures contained 50ng of vector, 200ng of insert, and 2μL of enzyme and were incubated for 15 minutes at 50°C. Errors were corrected by site-directed mutagenesis using Pfu Turbo polymerase (Agilent, Santa Clara, CA). The nsp12/14-P250 swap was generated through restriction digestion of the individual swaps using BsmBI and StuI followed by gel purification and assembly using T4 DNA ligase (NEB, Ipswich, MA). Viable viruses were constructed and rescued as described previously (<a id="xref-ref-91-6" class="xref-bibr" href="#ref-91">91</a>).</p></div><div id="sec-27" class="subsection"><h3>Competitive fitness assays</h3><p id="p-40">Competitor viruses were competed with an MHV-ExoN(-) virus harboring 10 silent mutations in the probe-binding region within nsp2. Subconfluent DBT-9 monolayers in 24-well plates were coinfected at a total MOI of 0.01 PFU/cell with competitor and reference viruses at a 1:1 ratio and passaged 4 times. For each passage, supernatants were harvested at 24 hours. RNA was extracted from 100μL of supernatant using 900μL of TRIzol reagent and PureLink RNA Mini Kit columns (Thermo Scientific, Waltham, MA), and 150μL was used to infect fresh cells in a 24-well plate (total MOI estimated at 1 PFU/cell). The proportion of each virus was determined by real-time RT-qPCR from the infection supernatant using two Taqman probes with different fluorescent dyes in separate reactions. Competitor viruses were detected with the same probe used in specific infectivity analyses (<a id="xref-ref-14-12" class="xref-bibr" href="#ref-14">14</a>). Reference viruses were detected by a probe targeting the same region but with 10 silent mutations (5’-TCCGAACTACTGCAACCCCAAGTG-3’) and labeled with 5’ Quasar 670 and 3’ black hole quencher 2 (BHQ-2) (Biosearch Technologies, Petaluma, CA). RNA copy number was calculated by reference to an RNA standard generated by <em>in vitro</em> transcription of the corresponding MHV A fragment, and relative RNA abundance was calculated as the ratio of competitor genomes to reference genomes.</p></div><div id="sec-28" class="subsection"><h3>Statistical analysis</h3><p id="p-41">GraphPad Prism 6 (La Jolla, CA) was used to perform statistical tests. Only the comparisons shown [e.g. ns or asterisk(s)] within the figure or legend were performed. In many cases the data were normalized to untreated controls. This was performed using GraphPad Prism 6. The number of replicate samples is denoted within each figure legend.</p></div><div id="sec-29" class="subsection"><h3>Accession numbers</h3><p id="p-42">Full-length genome sequences for WT-MHV P250 and MHV-ExoN(-) P250 have been deposited in GenBank (accession numbers <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MF618252&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom">MF618252</a> and <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MF618253&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom">MF618253</a>, respectively).</p></div></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGEMENTS</h2><p id="p-43">We thank members of the Denison laboratory for valuable discussions. This work was supported by United States Public Health Service awards R01-AI108197 (M.R.D), T32-GM007347 (K.W.G), F30-AI129229 (K.W.G), T32-HL07751 (J.B.C), T32-AI089554 (N.R.S.), T32-AI095202 (E.C.S.), and F32-AI108102 (E.C.S.), all from the National Institutes of Health. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health. The authors declare no conflicts of interest.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.1" data-doi="10.1016/0378-1119(92)90216-C"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Steinhauer</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domingo</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">JJ</span></span>. <span class="cit-pub-date">1992</span>. <span class="cit-article-title">Lack of evidence for proofreading mechanisms associated with an RNA virus polymerase</span>. <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">122</span>:<span class="cit-fpage">281</span>–<span class="cit-lpage">288</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DSteinhauer%26rft.auinit1%253DD.%2BA.%26rft.volume%253D122%26rft.issue%253D2%26rft.spage%253D281%26rft.epage%253D288%26rft.atitle%253DLack%2Bof%2Bevidence%2Bfor%2Bproofreading%2Bmechanisms%2Bassociated%2Bwith%2Ban%2BRNA%2Bvirus%2Bpolymerase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252892%252990216-C%26rft_id%253Dinfo%253Apmid%252F1336756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(92)90216-C&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1336756&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992KG73200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.2" data-doi="10.1128/JVI.00694-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanjuán</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nebot</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chirico</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mansky</span>  <span class="cit-name-given-names">LM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belshaw</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Viral mutation rates</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">84</span>:<span class="cit-fpage">9733</span>–<span class="cit-lpage">9748</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00694-10%26rft_id%253Dinfo%253Apmid%252F20660197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4NC8xOS85NzMzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDgvMTEvMTc1NTYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.3" data-doi="10.1073/pnas.96.24.13910"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">JJ</span></span>. <span class="cit-pub-date">1999</span>. <span class="cit-article-title">Mutation rates among RNA viruses</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">96</span>:<span class="cit-fpage">13910</span>–<span class="cit-lpage">13913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DDrake%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D96%26rft.issue%253D24%26rft.spage%253D13910%26rft.epage%253D13913%26rft.atitle%253DMutation%2Brates%2Bamong%2BRNA%2Bviruses%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.24.13910%26rft_id%253Dinfo%253Apmid%252F10570172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTYvMjQvMTM5MTAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wOC8xMS8xNzU1NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.4" data-doi="10.1128/JVI.01624-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanjuán</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">CO</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Theory of lethal mutagenesis for viruses</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">81</span>:<span class="cit-fpage">2930</span>–<span class="cit-lpage">2939</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01624-06%26rft_id%253Dinfo%253Apmid%252F17202214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgxLzYvMjkzMCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">5.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.5" data-doi="10.1128/MMBR.05023-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Domingo</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sheldon</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perales</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Viral quasispecies evolution</span>. <abbr class="cit-jnl-abbrev">Microbiol Mol Biol Rev</abbr> <span class="cit-vol">76</span>:<span class="cit-fpage">159</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DDomingo%26rft.auinit1%253DE.%26rft.volume%253D76%26rft.issue%253D2%26rft.spage%253D159%26rft.epage%253D216%26rft.atitle%253DViral%2BQuasispecies%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.05023-11%26rft_id%253Dinfo%253Apmid%252F22688811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI3Ni8yLzE1OSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">6.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.6" data-doi="10.1534/genetics.113.154963"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bradwell</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Combe</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domingo-Calap</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanjuán</span>  <span class="cit-name-given-names">R</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Correlation between mutation rate and genome size in riboviruses: mutation rate of bacteriophage Qß</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>:<span class="cit-fpage">243</span>–<span class="cit-lpage">251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.154963%26rft_id%253Dinfo%253Apmid%252F23852383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzEvMjQzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDgvMTEvMTc1NTYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.7" data-doi="10.1126/science.1169202"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gago</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elena</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Flores</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanjuán</span>  <span class="cit-name-given-names">R</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Extremely high mutation rate of a hammerhead viroid</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>:<span class="cit-fpage">1308</span>–<span class="cit-lpage">1308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGago%26rft.auinit1%253DS.%26rft.volume%253D323%26rft.issue%253D5919%26rft.spage%253D1308%26rft.epage%253D1308%26rft.atitle%253DExtremely%2BHigh%2BMutation%2BRate%2Bof%2Ba%2BHammerhead%2BViroid%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1169202%26rft_id%253Dinfo%253Apmid%252F19265013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjMvNTkxOS8xMzA4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDgvMTEvMTc1NTYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.8" data-doi="10.1534/genetics.110.115162"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peris</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nebot</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanjuán</span>  <span class="cit-name-given-names">R</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Distribution of Fitness Effects Caused by Single-Nucleotide Substitutions in Bacteriophage f1</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>:<span class="cit-fpage">603</span>–<span class="cit-lpage">609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.115162%26rft_id%253Dinfo%253Apmid%252F20382832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzIvNjAzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDgvMTEvMTc1NTYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Malpica</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fraile</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Obies</span>  <span class="cit-name-given-names">CI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">García-Arenal</span>  <span class="cit-name-given-names">F.</span></span> <span class="cit-pub-date">2002</span>. <span class="cit-article-title">The rate and character of spontaneous mutation in an RNA virus</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>:<span class="cit-fpage">1505</span>–<span class="cit-lpage">1511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMalpica%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D162%26rft.issue%253D4%26rft.spage%253D1505%26rft.epage%253D1511%26rft.atitle%253DThe%2BRate%2Band%2BCharacter%2Bof%2BSpontaneous%2BMutation%2Bin%2Ban%2BRNA%2BVirus%26rft_id%253Dinfo%253Apmid%252F12524327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Mi80LzE1MDUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wOC8xMS8xNzU1NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.10" data-doi="10.1046/j.1420-9101.1999.00110.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elena</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moya</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">1999</span>. <span class="cit-article-title">Rate of deleterious mutation and the distribution of its effects on fitness in vesicular stomatitis virus</span>. <abbr class="cit-jnl-abbrev">Journal of Evolutionary Biology</abbr> <span class="cit-vol">12</span>:<span class="cit-fpage">1078</span>–<span class="cit-lpage">1088</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BEvolutionary%2BBiology%26rft.volume%253D12%26rft.spage%253D1078%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1420-9101.1999.00110.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1420-9101.1999.00110.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000083843600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.11" data-doi="10.1371/journal.ppat.1005856"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Visher</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitefield</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCrone</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fitzsimmons</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lauring</span>  <span class="cit-name-given-names">AS</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">The Mutational Robustness of Influenza A Virus</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">12</span>:<span class="cit-fpage">e1005856</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D12%26rft.spage%253De1005856%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1005856%26rft_id%253Dinfo%253Apmid%252F27571422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1005856&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27571422&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.12" data-doi="10.1073/pnas.0400146101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanjuán</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moya</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elena</span>  <span class="cit-name-given-names">SF</span></span>. <span class="cit-pub-date">2004</span>. <span class="cit-article-title">The distribution of fitness effects caused by single-nucleotide substitutions in an RNA virus</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">101</span>:<span class="cit-fpage">8396</span>–<span class="cit-lpage">8401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0400146101%26rft_id%253Dinfo%253Apmid%252F15159545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzIyLzgzOTYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wOC8xMS8xNzU1NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.13" data-doi="10.1371/journal.ppat.1000896"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eckerle</span>  <span class="cit-name-given-names">LD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Halpin</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Venter</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scherbakova</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stockwell</span>  <span class="cit-name-given-names">TB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spiro</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Infidelity of SARS-CoV Nsp14-exonuclease mutant virus replication is revealed by complete genome sequencing</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">e1000896</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DEckerle%26rft.auinit1%253DL.%2BD.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De1000896%26rft.epage%253De1000896%26rft.atitle%253DInfidelity%2Bof%2BSARS-CoV%2BNsp14-exonuclease%2Bmutant%2Bvirus%2Breplication%2Bis%2Brevealed%2Bby%2Bcomplete%2Bgenome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1000896%26rft_id%253Dinfo%253Apmid%252F20463816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1000896&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20463816&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.14" data-doi="10.1128/JVI.01296-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eckerle</span>  <span class="cit-name-given-names">LD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sperry</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-article-title">High fidelity of murine hepatitis virus replication is decreased in nsp14 exoribonuclease mutants</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">81</span>:<span class="cit-fpage">12135</span>–<span class="cit-lpage">12144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01296-07%26rft_id%253Dinfo%253Apmid%252F17804504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MS8yMi8xMjEzNSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.15" data-doi="10.1128/JVI.00110-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Case</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Isakov</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shomron</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Mutations in coronavirus nonstructural protein 10 decrease virus replication fidelity</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">89</span>:<span class="cit-fpage">6418</span>–<span class="cit-lpage">6426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00110-15%26rft_id%253Dinfo%253Apmid%252F25855750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OS8xMi82NDE4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDgvMTEvMTc1NTYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.16" data-doi="10.1371/journal.ppat.1003565"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Surdel</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Coronaviruses lacking exoribonuclease activity are susceptible to lethal mutagenesis: evidence for proofreading and potential therapeutics</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">9</span>:<span class="cit-fpage">e1003565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D9%26rft.spage%253De1003565%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003565%26rft_id%253Dinfo%253Apmid%252F23966862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003565&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23966862&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.17" data-doi="10.1128/JVI.00080-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sexton</span>  <span class="cit-name-given-names">NR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peersen</span>  <span class="cit-name-given-names">OB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Homology-Based Identification of a Mutation in the Coronavirus RNA-Dependent RNA Polymerase That Confers Resistance to Multiple Mutagens</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">90</span>:<span class="cit-fpage">7415</span>–<span class="cit-lpage">7428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00080-16%26rft_id%253Dinfo%253Apmid%252F27279608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI5MC8xNi83NDE1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDgvMTEvMTc1NTYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">18.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.18" data-doi="10.1128/JVI.01297-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arias</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sierra</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smidansky</span>  <span class="cit-name-given-names">ED</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domingo</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Determinants of RNA-dependent RNA polymerase (in)fidelity revealed by kinetic analysis of the polymerase encoded by a foot-and-mouth disease virus mutant with reduced sensitivity to ribavirin</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">82</span>:<span class="cit-fpage">12346</span>–<span class="cit-lpage">12355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01297-08%26rft_id%253Dinfo%253Apmid%252F18829745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Mi8yNC8xMjM0NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.19" data-doi="10.1016/j.antiviral.2013.07.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">An increased replication fidelity mutant of foot-and-mouth disease virus retains fitness in vitro and virulence in vivo</span>. <abbr class="cit-jnl-abbrev">Antiviral Res</abbr> <span class="cit-vol">100</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2BRes%26rft.volume%253D100%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2013.07.008%26rft_id%253Dinfo%253Apmid%252F23880348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2013.07.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23880348&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.20" data-doi="10.1128/JVI.03594-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Ribavirin-resistant variants of foot-and-mouth disease virus: the effect of restricted quasispecies diversity on viral virulence</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">88</span>:<span class="cit-fpage">4008</span>–<span class="cit-lpage">4020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03594-13%26rft_id%253Dinfo%253Apmid%252F24453363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg4LzgvNDAwOCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">21.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.21" data-doi="10.1007/s00705-014-2126-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Foot-and-mouth disease virus low-fidelity polymerase mutants are attenuated</span>. <abbr class="cit-jnl-abbrev">Arch Virol</abbr> <span class="cit-vol">159</span>:<span class="cit-fpage">2641</span>–<span class="cit-lpage">2650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArch%2BVirol%26rft.volume%253D159%26rft.spage%253D2641%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-014-2126-z%26rft_id%253Dinfo%253Apmid%252F24888311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-014-2126-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24888311&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.22" data-doi="10.1128/JVI.01606-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sierra</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Airaksinen</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">González-López</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Agudo</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arias</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domingo</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Foot-and-mouth disease virus mutant with decreased sensitivity to ribavirin: implications for error catastrophe</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">81</span>:<span class="cit-fpage">2012</span>–<span class="cit-lpage">2024</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01606-06%26rft_id%253Dinfo%253Apmid%252F17151116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgxLzQvMjAxMiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.23" data-doi="10.1038/nature04388"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Quasispecies diversity determines pathogenesis through cooperative interactions in a viral population</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">439</span>:<span class="cit-fpage">344</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DVignuzzi%26rft.auinit1%253DM.%26rft.volume%253D439%26rft.issue%253D7074%26rft.spage%253D344%26rft.epage%253D348%26rft.atitle%253DQuasispecies%2Bdiversity%2Bdetermines%2Bpathogenesis%2Bthrough%2Bcooperative%2Binteractions%2Bin%2Ba%2Bviral%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04388%26rft_id%253Dinfo%253Apmid%252F16327776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04388&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16327776&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234682100047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.24" data-doi="10.1038/nm1726"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wendt</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Engineering attenuated virus vaccines by controlling replication fidelity</span>. <abbr class="cit-jnl-abbrev">Nat Med</abbr> <span class="cit-vol">14</span>:<span class="cit-fpage">154</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmedicine%26rft.stitle%253DNat%2BMed%26rft.aulast%253DVignuzzi%26rft.auinit1%253DM.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D154%26rft.epage%253D161%26rft.atitle%253DEngineering%2Battenuated%2Bvirus%2Bvaccines%2Bby%2Bcontrolling%2Breplication%2Bfidelity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm1726%26rft_id%253Dinfo%253Apmid%252F18246077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm1726&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18246077&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252946700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.25" data-doi="10.1073/pnas.1232294100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pfeiffer</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kirkegaard</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">A single mutation in poliovirus RNA-dependent RNA polymerase confers resistance to mutagenic nucleotide analogs via increased fidelity</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">100</span>:<span class="cit-fpage">7289</span>–<span class="cit-lpage">7294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1232294100%26rft_id%253Dinfo%253Apmid%252F12754380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzEyLzcyODkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wOC8xMS8xNzU1NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">26.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.26" data-doi="10.1074/jbc.M503444200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Remote site control of an active site fidelity checkpoint in a viral RNA-dependent RNA polymerase</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">280</span>:<span class="cit-fpage">25706</span>–<span class="cit-lpage">25716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M503444200%26rft_id%253Dinfo%253Apmid%252F15878882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODAvMjcvMjU3MDYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wOC8xMS8xNzU1NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.27" data-doi="10.1074/jbc.M113.484428"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moustafa</span>  <span class="cit-name-given-names">IM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smidansky</span>  <span class="cit-name-given-names">ED</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lum</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boehr</span>  <span class="cit-name-given-names">DD</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Vaccine-derived mutation in motif D of poliovirus RNA-dependent RNA polymerase lowers nucleotide incorporation fidelity</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">288</span>:<span class="cit-fpage">32753</span>–<span class="cit-lpage">32765</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M113.484428%26rft_id%253Dinfo%253Apmid%252F24085299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODgvNDUvMzI3NTMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wOC8xMS8xNzU1NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.28" data-doi="10.1074/jbc.C112.401471"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weeks</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smidansky</span>  <span class="cit-name-given-names">ED</span></span>, <span class="cit-auth"><span class="cit-name-surname">August</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">A Polymerase mechanism-based strategy for viral attenuation and vaccine development</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">287</span>:<span class="cit-fpage">31618</span>–<span class="cit-lpage">31622</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.C112.401471%26rft_id%253Dinfo%253Apmid%252F22854962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODcvMzgvMzE2MTgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wOC8xMS8xNzU1NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.29" data-doi="10.1074/jbc.M114.592303"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Korboukh</span>  <span class="cit-name-given-names">VK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Acevedo</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hemperly</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graci</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">August</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">RNA virus population diversity, an optimum for maximal fitness and virulence</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">289</span>:<span class="cit-fpage">29531</span>–<span class="cit-lpage">29544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M114.592303%26rft_id%253Dinfo%253Apmid%252F25213864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODkvNDMvMjk1MzEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wOC8xMS8xNzU1NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.30" data-doi="10.1073/pnas.1111650108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coffey</span>  <span class="cit-name-given-names">LL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beeharry</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bordería</span>  <span class="cit-name-given-names">AV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Arbovirus high fidelity variant loses fitness in mosquitoes and mice</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">108</span>:<span class="cit-fpage">16038</span>– <span class="cit-lpage">16043</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1111650108%26rft_id%253Dinfo%253Apmid%252F21896755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzM4LzE2MDM4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDgvMTEvMTc1NTYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.31" data-doi="10.1371/journal.ppat.1003877"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rozen-Gagnon</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stapleford</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mongelli</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Failloux</span>  <span class="cit-name-given-names">A-B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saleh</span>  <span class="cit-name-given-names">M-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Alphavirus mutator variants present host-specific defects and attenuation in mammalian and insect models</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">10</span>:<span class="cit-fpage">e1003877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D10%26rft.spage%253De1003877%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003877%26rft_id%253Dinfo%253Apmid%252F24453971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003877&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24453971&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.32" data-doi="10.1038/ncomms5794"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">PPH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choy</span>  <span class="cit-name-given-names">K-T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fun Sia</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">DDY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poon</span>  <span class="cit-name-given-names">LLM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kellam</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Malik Peiris</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yen</span>  <span class="cit-name-given-names">H-L.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Generation and characterization of influenza A viruses with altered polymerase fidelity</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">4794</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D5%26rft.spage%253D4794%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms5794%26rft_id%253Dinfo%253Apmid%252F25183443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms5794&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25183443&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.33" data-doi="10.1073/pnas.1204022109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gnädig</span>  <span class="cit-name-given-names">NF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beaucourt</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campagnola</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bordería</span>  <span class="cit-name-given-names">AV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanz-Ramos</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peersen</span>  <span class="cit-name-given-names">OB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Coxsackievirus B3 mutator strains are attenuated in vivo</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">109</span>:<span class="cit-fpage">E2294</span>–<span class="cit-lpage">303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1204022109%26rft_id%253Dinfo%253Apmid%252F22853955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM0L0UyMjk0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDgvMTEvMTc1NTYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.34" data-doi="10.1128/JVI.01574-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Campagnola</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beaucourt</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peersen</span>  <span class="cit-name-given-names">OB</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Structure-Function Relationships Underlying the Replication Fidelity of Viral RNA-Dependent RNA Polymerases</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">89</span>:<span class="cit-fpage">275</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01574-14%26rft_id%253Dinfo%253Apmid%252F25320316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6Ijg5LzEvMjc1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDgvMTEvMTc1NTYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.35" data-doi="10.1016/j.virusres.2013.06.019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sadeghipour</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">McMinn</span>  <span class="cit-name-given-names">PC</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">A study of the virulence in mice of high copying fidelity variants of human enterovirus 71</span>. <abbr class="cit-jnl-abbrev">Virus Research</abbr> <span class="cit-vol">176</span>:<span class="cit-fpage">265</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BResearch%26rft.volume%253D176%26rft.spage%253D265%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2013.06.019%26rft_id%253Dinfo%253Apmid%252F23856384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2013.06.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23856384&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">36.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.36" data-doi="10.1128/JVI.02139-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sadeghipour</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bek</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">McMinn</span>  <span class="cit-name-given-names">PC</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Ribavirin-resistant mutants of human enterovirus 71 express a high replication fidelity phenotype during growth in cell culture</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">87</span>:<span class="cit-fpage">1759</span>–<span class="cit-lpage">1769</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02139-12%26rft_id%253Dinfo%253Apmid%252F23175376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg3LzMvMTc1OSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.37" data-doi="10.1128/JVI.00289-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kwang</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Attenuation of human enterovirus 71 high-replication-fidelity variants in AG129 mice</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">88</span>:<span class="cit-fpage">5803</span>–<span class="cit-lpage">5815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00289-14%26rft_id%253Dinfo%253Apmid%252F24623423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OC8xMC81ODAzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDgvMTEvMTc1NTYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.38" data-doi="10.1038/nm.2972"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eckerle</span>  <span class="cit-name-given-names">LD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bolles</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">A live,impaired-fidelity coronavirus vaccine protects in an aged, immunocompromised mouse model of lethal disease</span>. <abbr class="cit-jnl-abbrev">Nat Med</abbr> <span class="cit-vol">18</span>:<span class="cit-fpage">1820</span>–<span class="cit-lpage">1826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMed%26rft.volume%253D18%26rft.spage%253D1820%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm.2972%26rft_id%253Dinfo%253Apmid%252F23142821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm.2972&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23142821&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sexton</span>  <span class="cit-name-given-names">NR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Thinking Outside the Triangle: Replication Fidelity of the Largest RNA Viruses</span>. <abbr class="cit-jnl-abbrev">Annual Review of Virology</abbr> <span class="cit-vol">1</span>:<span class="cit-fpage">111</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BVirology%26rft.volume%253D1%26rft.spage%253D111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.40" data-doi="10.1016/j.virusres.2006.01.017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>. <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Nidovirales: evolving the largest RNA virus genome</span>. <abbr class="cit-jnl-abbrev">Virus Research</abbr> <span class="cit-vol">117</span>:<span class="cit-fpage">17</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DGorbalenya%26rft.auinit1%253DA.%2BE.%26rft.volume%253D117%26rft.issue%253D1%26rft.spage%253D17%26rft.epage%253D37%26rft.atitle%253DNidovirales%253A%2Bevolving%2Bthe%2Blargest%2BRNA%2Bvirus%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2006.01.017%26rft_id%253Dinfo%253Apmid%252F16503362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2006.01.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16503362&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236746000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brister</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ako-Adjei</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blinkova</span>  <span class="cit-name-given-names">O.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">NCBI Viral Genomes Resource</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">43</span>:<span class="cit-fpage">gku1207</span>–<span class="cit-lpage">D577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D43%26rft.spage%253Dgku1207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.42" data-doi="10.1128/mBio.01484-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stenglein</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jacobson</span>  <span class="cit-name-given-names">ER</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wozniak</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wellehan</span>  <span class="cit-name-given-names">JFX</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kincaid</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Porter</span>  <span class="cit-name-given-names">BF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baumgartner</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Towner</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeRisi</span>  <span class="cit-name-given-names">JL</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Ball Python Nidovirus: a Candidate Etiologic Agent for Severe Respiratory Disease in Python regius</span>. <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">e01484-14</span>–<span class="cit-lpage">e01484-14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.01484-14%26rft_id%253Dinfo%253Apmid%252F25205093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibWJpbyI7czo1OiJyZXNpZCI7czoxMzoiNS81L2UwMTQ4NC0xNCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.43" data-doi="10.1073/pnas.0508200103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Minskaia</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hertzig</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campanacci</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cambillau</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Discovery of an RNA virus 3“-&gt;5” exoribonuclease that is critically involved in coronavirus RNA synthesis</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">103</span>:<span class="cit-fpage">5108</span>–<span class="cit-lpage">5113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0508200103%26rft_id%253Dinfo%253Apmid%252F16549795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAzLzEzLzUxMDgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wOC8xMS8xNzU1NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.44" data-doi="10.1016/S0022-2836(03)00865-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bredenbeek</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dobbe</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thiel</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poon</span>  <span class="cit-name-given-names">LLM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rozanov</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spaan</span>  <span class="cit-name-given-names">WJM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>. <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Unique and conserved features of genome and proteome of SARS-coronavirus, an early split-off from the coronavirus group 2 lineage</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">331</span>:<span class="cit-fpage">991</span>–<span class="cit-lpage">1004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DSnijder%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D331%26rft.issue%253D5%26rft.spage%253D991%26rft.epage%253D1004%26rft.atitle%253DUnique%2Band%2Bconserved%2Bfeatures%2Bof%2Bgenome%2Band%2Bproteome%2Bof%2BSARS-coronavirus%252C%2Ban%2Bearly%2Bsplit-off%2Bfrom%2Bthe%2Bcoronavirus%2Bgroup%2B2%2Blineage.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252803%252900865-9%26rft_id%253Dinfo%253Apmid%252F12927536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-2836(03)00865-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12927536&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185036400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.45" data-doi="10.1073/pnas.1508686112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lou</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">Z.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Structural basis and functional analysis of the SARS coronavirus nsp14-nsp10 complex</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">112</span>:<span class="cit-fpage">9436</span>–<span class="cit-lpage">9441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1508686112%26rft_id%253Dinfo%253Apmid%252F26159422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzMwLzk0MzYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wOC8xMS8xNzU1NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">46.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.46" data-doi="10.1093/nar/29.5.1017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zuo</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deutscher</span>  <span class="cit-name-given-names">MP</span></span>. <span class="cit-pub-date">2001</span>. <span class="cit-article-title">Exoribonuclease superfamilies: structural analysis and phylogenetic distribution</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">29</span>:<span class="cit-fpage">1017</span>–<span class="cit-lpage">1026</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DZuo%26rft.auinit1%253DY.%26rft.volume%253D29%26rft.issue%253D5%26rft.spage%253D1017%26rft.epage%253D1026%26rft.atitle%253DExoribonuclease%2Bsuperfamilies%253A%2Bstructural%2Banalysis%2Band%2Bphylogenetic%2Bdistribution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F29.5.1017%26rft_id%253Dinfo%253Apmid%252F11222749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/29.5.1017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11222749&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167240500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.47" data-doi="10.1073/pnas.1201130109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bouvet</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Imbert</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Subissi</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gluais</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Decroly</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">RNA 3’-end mismatch excision by the severe acute respiratory syndrome coronavirus nonstructural protein nsp10/nsp14 exoribonuclease complex</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">109</span>:<span class="cit-fpage">9372</span>–<span class="cit-lpage">9377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1201130109%26rft_id%253Dinfo%253Apmid%252F22635272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA5LzI0LzkzNzIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wOC8xMS8xNzU1NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.48" data-doi="10.1016/0042-6822(91)90994-M"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yokomori</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Banner</span>  <span class="cit-name-given-names">LR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">MMC</span></span>. <span class="cit-pub-date">1991</span>. <span class="cit-article-title">Heterogeneity of gene expression of the hemagglutinin-esterase (HE) protein of murine coronaviruses</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">183</span>:<span class="cit-fpage">647</span>–<span class="cit-lpage">657</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DYokomori%26rft.auinit1%253DK.%26rft.volume%253D183%26rft.issue%253D2%26rft.spage%253D647%26rft.epage%253D657%26rft.atitle%253DHeterogeneity%2Bof%2Bgene%2Bexpression%2Bof%2Bthe%2Bhemagglutinin-esterase%2B%2528HE%2529%2Bprotein%2Bof%2Bmurine%2Bcoronaviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252891%252990994-M%26rft_id%253Dinfo%253Apmid%252F1649505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(91)90994-M&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1649505&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">49.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.49" data-doi="10.1016/0042-6822(88)90512-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luytjes</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bredenbeek</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Noten</span>  <span class="cit-name-given-names">AF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horzinek</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spaan</span>  <span class="cit-name-given-names">WJ</span></span>. <span class="cit-pub-date">1988</span>. <span class="cit-article-title">Sequence of mouse hepatitis virus A59 mRNA 2: indications for RNA recombination between coronaviruses and influenza C virus</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">166</span>:<span class="cit-fpage">415</span>–<span class="cit-lpage">422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DLuytjes%26rft.auinit1%253DW.%26rft.volume%253D166%26rft.issue%253D2%26rft.spage%253D415%26rft.epage%253D422%26rft.atitle%253DSequence%2Bof%2Bmouse%2Bhepatitis%2Bvirus%2BA59%2BmRNA%2B2%253A%2Bindications%2Bfor%2BRNA%2Brecombination%2Bbetween%2Bcoronaviruses%2Band%2Binfluenza%2BC%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252888%252990512-0%26rft_id%253Dinfo%253Apmid%252F2845655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(88)90512-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2845655&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988Q520100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yokomori</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">MM</span></span>. <span class="cit-pub-date">1991</span>. <span class="cit-article-title">Mouse hepatitis virus S RNA sequence reveals that nonstructural proteins ns4 and ns5a are not essential for murine coronavirus replication</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">65</span>:<span class="cit-fpage">5605</span>–<span class="cit-lpage">5608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DYokomori%26rft.auinit1%253DK%26rft.volume%253D65%26rft.issue%253D10%26rft.spage%253D5605%26rft.epage%253D5608%26rft.atitle%253DMouse%2Bhepatitis%2Bvirus%2BS%2BRNA%2Bsequence%2Breveals%2Bthat%2Bnonstructural%2Bproteins%2Bns4%2Band%2Bns5a%2Bare%2Bnot%2Bessential%2Bfor%2Bmurine%2Bcoronavirus%2Breplication.%26rft_id%253Dinfo%253Apmid%252F1654456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI2NS8xMC81NjA1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDgvMTEvMTc1NTYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.51" data-doi="10.1128/JVI.79.24.15054-15063.2005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lissenberg</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vrolijk</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Vliet</span>  <span class="cit-name-given-names">ALW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Langereis</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Groot-Mijnes</span>  <span class="cit-name-given-names">JDF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rottier</span>  <span class="cit-name-given-names">PJM</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Groot</span>  <span class="cit-name-given-names">RJ</span></span>. <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Luxury at a Cost? Recombinant Mouse Hepatitis Viruses Expressing the Accessory Hemagglutinin Esterase Protein Display Reduced Fitness In Vitro</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">79</span>:<span class="cit-fpage">15054</span>–<span class="cit-lpage">15063</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLissenberg%26rft.auinit1%253DA.%26rft.volume%253D79%26rft.issue%253D24%26rft.spage%253D15054%26rft.epage%253D15063%26rft.atitle%253DLuxury%2Bat%2Ba%2BCost%253F%2BRecombinant%2BMouse%2BHepatitis%2BViruses%2BExpressing%2Bthe%2BAccessory%2BHemagglutinin%2BEsterase%2BProtein%2BDisplay%2BReduced%2BFitness%2BIn%2BVitro%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.24.15054-15063.2005%26rft_id%253Dinfo%253Apmid%252F16306576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3OS8yNC8xNTA1NCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.52" data-doi="10.1128/JVI.01126-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gadlage</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Murine coronaviruses encoding nsp2 at different genomic loci have altered replication, protein expression, and localization</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">82</span>:<span class="cit-fpage">11964</span>–<span class="cit-lpage">11969</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01126-07%26rft_id%253Dinfo%253Apmid%252F18815297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Mi8yMy8xMTk2NCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jha</span>  <span class="cit-name-given-names">BK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elliott</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silverman</span>  <span class="cit-name-given-names">RH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">SR</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Antagonism of the Interferon-Induced OAS-RNase L Pathway by Murine Coronavirus ns2 Protein Is Required for Virus Replication and Liver Pathology</span>. <abbr class="cit-jnl-abbrev">Cell Host and Microbe</abbr> <span class="cit-vol">11</span>:<span class="cit-fpage">607</span>–<span class="cit-lpage">616</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2Band%2BMicrobe%26rft.volume%253D11%26rft.spage%253D607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">54.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">SR</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Antagonism of RNase L is required for murine coronavirus replication in Kupffer cells and liver sinusoidal endothelial cells but not in hepatocytes</span>. <abbr class="cit-jnl-abbrev">J Virol JVI.01423–</abbr><span class="cit-fpage">16</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.55" data-doi="10.1073/pnas.1306917110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jha</span>  <span class="cit-name-given-names">BK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ogden</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elliott</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patton</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silverman</span>  <span class="cit-name-given-names">RH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">SR</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Homologous 2“,5-”phosphodiesterases from disparate RNA viruses antagonize antiviral innate immunity</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">110</span>:<span class="cit-fpage">13114</span>–<span class="cit-lpage">13119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1306917110%26rft_id%253Dinfo%253Apmid%252F23878220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzMyLzEzMTE0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDgvMTEvMTc1NTYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sui</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jha</span>  <span class="cit-name-given-names">BK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">ZF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silverman</span>  <span class="cit-name-given-names">RH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Crystal structure of the mouse hepatitis virus ns2 phosphodiesterase domain that antagonizes RNase L activation</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr> <span class="cit-vol">97</span>:<span class="cit-fpage">880</span>–<span class="cit-lpage">886</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft.volume%253D97%26rft.spage%253D880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Routledge</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siddell</span>  <span class="cit-name-given-names">SG</span></span>. <span class="cit-pub-date">1990</span>. <span class="cit-article-title">Murine coronavirus nonstructural protein ns2 is not essential for virus replication in transformed cells</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">64</span>:<span class="cit-fpage">4784</span>–<span class="cit-lpage">4791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DSchwarz%26rft.auinit1%253DB%26rft.volume%253D64%26rft.issue%253D10%26rft.spage%253D4784%26rft.epage%253D4791%26rft.atitle%253DMurine%2Bcoronavirus%2Bnonstructural%2Bprotein%2Bns2%2Bis%2Bnot%2Bessential%2Bfor%2Bvirus%2Breplication%2Bin%2Btransformed%2Bcells.%26rft_id%253Dinfo%253Apmid%252F2168966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI2NC8xMC80Nzg0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDgvMTEvMTc1NTYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.58" data-doi="10.1128/JVI.05075-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elliott</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Rooijen</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">SR</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Cell-type-specific type I interferon antagonism influences organ tropism of murine coronavirus</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">85</span>:<span class="cit-fpage">10058</span>– <span class="cit-lpage">10068</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.05075-11%26rft_id%253Dinfo%253Apmid%252F21752905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4NS8xOS8xMDA1OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.59" data-doi="10.1128/JVI.02147-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pereira-Gómez</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanjuán</span>  <span class="cit-name-given-names">R</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Delayed lysis confers resistance to the nucleoside analogue 5-fluorouracil and alleviates mutation accumulation in the single-stranded DNA bacteriophage ?X174</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">88</span>:<span class="cit-fpage">5042</span>–<span class="cit-lpage">5049</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02147-13%26rft_id%253Dinfo%253Apmid%252F24554658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg4LzkvNTA0MiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.60" data-doi="10.1007/978-3-540-75157-1_7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">KKS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Structure-function relationships among RNA-dependent RNA polymerases</span>. <abbr class="cit-jnl-abbrev">Curr Top Microbiol Immunol</abbr> <span class="cit-vol">320</span>:<span class="cit-fpage">137</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Btopics%2Bin%2Bmicrobiology%2Band%2Bimmunology%26rft.stitle%253DCurr%2BTop%2BMicrobiol%2BImmunol%26rft.aulast%253DNg%26rft.auinit1%253DK.%2BK.%26rft.volume%253D320%26rft.spage%253D137%26rft.epage%253D156%26rft.atitle%253DStructure-function%2Brelationships%2Bamong%2BRNA-dependent%2BRNA%2Bpolymerases.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-540-75157-1_7%26rft_id%253Dinfo%253Apmid%252F18268843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-540-75157-1_7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18268843&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253430500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.61" data-doi="10.1128/JVI.77.19.10515-10527.2003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brockway</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clay</span>  <span class="cit-name-given-names">CT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">XT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR</span></span>. <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Characterization of the expression, intracellular localization, and replication complex association of the putative mouse hepatitis virus RNA-dependent RNA polymerase</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">77</span>:<span class="cit-fpage">10515</span>–<span class="cit-lpage">10527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DBrockway%26rft.auinit1%253DS.%2BM.%26rft.volume%253D77%26rft.issue%253D19%26rft.spage%253D10515%26rft.epage%253D10527%26rft.atitle%253DCharacterization%2Bof%2Bthe%2BExpression%252C%2BIntracellular%2BLocalization%252C%2Band%2BReplication%2BComplex%2BAssociation%2Bof%2Bthe%2BPutative%2BMouse%2BHepatitis%2BVirus%2BRNA-Dependent%2BRNA%2BPolymerase%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.77.19.10515-10527.2003%26rft_id%253Dinfo%253Apmid%252F12970436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3Ny8xOS8xMDUxNSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.62" data-doi="10.1093/nar/gkv838"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lehmann</span>  <span class="cit-name-given-names">KC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gulyaeva</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zevenhoven-Dobbe</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Janssen</span>  <span class="cit-name-given-names">GMC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruben</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Overkleeft</span>  <span class="cit-name-given-names">HS</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Veelen</span>  <span class="cit-name-given-names">PA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Samborskiy</span>  <span class="cit-name-given-names">DV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kravchenko</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leontovich</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sidorov</span>  <span class="cit-name-given-names">IA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Posthuma</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Discovery of an essential nucleotidylating activity associated with a newly delineated conserved domain in the RNA polymerase-containing protein of all nidoviruses</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">43</span>:<span class="cit-fpage">8416</span>– <span class="cit-lpage">8434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv838%26rft_id%253Dinfo%253Apmid%252F26304538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv838&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26304538&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.63" data-doi="10.1038/nprot.2015.053"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">LA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mezulis</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yates</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wass</span>  <span class="cit-name-given-names">MN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sternberg</span>  <span class="cit-name-given-names">MJE</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">The Phyre<sup>2</sup> web portal for protein modeling, prediction and analysis</span>. <abbr class="cit-jnl-abbrev">Nat Protoc</abbr> <span class="cit-vol">10</span>:<span class="cit-fpage">845</span>–<span class="cit-lpage">858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BProtoc%26rft.volume%253D10%26rft.spage%253D845%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2015.053%26rft_id%253Dinfo%253Apmid%252F25950237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2015.053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25950237&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.64" data-doi="10.1074/jbc.M114.577353"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bouvet</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lugari</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Posthuma</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zevenhoven</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bernard</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Betzi</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Imbert</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guillemot</span>  <span class="cit-name-given-names">J-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lécine</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pfefferle</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drosten</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Decroly</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morelli</span>  <span class="cit-name-given-names">X.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Coronavirus Nsp10, a critical co-factor for activation of multiple replicative enzymes</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">289</span>:<span class="cit-fpage">25783</span>–<span class="cit-lpage">25796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M114.577353%26rft_id%253Dinfo%253Apmid%252F25074927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODkvMzcvMjU3ODMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wOC8xMS8xNzU1NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.65" data-doi="10.1128/JVI.02805-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Donaldson</span>  <span class="cit-name-given-names">EF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sims</span>  <span class="cit-name-given-names">AC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Murine hepatitis virus replicase protein nsp10 is a critical regulator of viral RNA synthesis</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">81</span>:<span class="cit-fpage">6356</span>–<span class="cit-lpage">6368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02805-06%26rft_id%253Dinfo%253Apmid%252F17392363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MS8xMi82MzU2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDgvMTEvMTc1NTYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.66" data-doi="10.1007/s00109-001-0308-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crotty</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Ribavirin’s antiviral mechanism of action: lethal mutagenesis?</span> <abbr class="cit-jnl-abbrev">J Mol Med</abbr> <span class="cit-vol">80</span>:<span class="cit-fpage">86</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bmedicine%2B%2528Berlin%252C%2BGermany%2529%26rft.stitle%253DJ%2BMol%2BMed%26rft.aulast%253DCrotty%26rft.auinit1%253DS.%26rft.volume%253D80%26rft.issue%253D2%26rft.spage%253D86%26rft.epage%253D95%26rft.atitle%253DRibavirin%2527s%2Bantiviral%2Bmechanism%2Bof%2Baction%253A%2Blethal%2Bmutagenesis%253F%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00109-001-0308-0%26rft_id%253Dinfo%253Apmid%252F11907645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00109-001-0308-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11907645&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174679700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.67"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pathak</span>  <span class="cit-name-given-names">VK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Temin</span>  <span class="cit-name-given-names">HM</span></span>. <span class="cit-pub-date">1992</span>. <span class="cit-article-title">5-Azacytidine and RNA secondary structure increase the retrovirus mutation rate</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">66</span>:<span class="cit-fpage">3093</span>–<span class="cit-lpage">3100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DPathak%26rft.auinit1%253DV%2BK%26rft.volume%253D66%26rft.issue%253D5%26rft.spage%253D3093%26rft.epage%253D3100%26rft.atitle%253D5-Azacytidine%2Band%2BRNA%2Bsecondary%2Bstructure%2Bincrease%2Bthe%2Bretrovirus%2Bmutation%2Brate.%26rft_id%253Dinfo%253Apmid%252F1373201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjY2LzUvMzA5MyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.68" data-doi="10.1074/jbc.M210914200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">SS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tomassini</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bosserman</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Getty</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stahlhut</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eldrup</span>  <span class="cit-name-given-names">AB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simcoe</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">LaFemina</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rutkowski</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wolanski</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Migliaccio</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">De Francesco</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">LC</span></span>, <span class="cit-auth"><span class="cit-name-surname">MacCoss</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">DB</span></span>. <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Inhibition of hepatitis C virus RNA replication by 2’-modified nucleoside analogs</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">278</span>:<span class="cit-fpage">11979</span>–<span class="cit-lpage">11984</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M210914200%26rft_id%253Dinfo%253Apmid%252F12554735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzgvMTQvMTE5NzkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wOC8xMS8xNzU1NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.69" data-doi="10.1371/journal.ppat.1003500"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lauber</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goeman</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">del Carmen Parquet</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nga</span>  <span class="cit-name-given-names">PT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morita</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">The Footprint of Genome Architecture in the Largest Genome Expansion in RNA Viruses</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">9</span>:<span class="cit-fpage">e1003500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D9%26rft.spage%253De1003500%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003500%26rft_id%253Dinfo%253Apmid%252F23874204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874204&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.70"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Derbyshire</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grindley</span>  <span class="cit-name-given-names">ND</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joyce</span>  <span class="cit-name-given-names">CM</span></span>. <span class="cit-pub-date">1991</span>. <span class="cit-article-title">The 3“-5” exonuclease of DNA polymerase I of Escherichia coli: contribution of each amino acid at the active site to the reaction</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr> <span class="cit-vol">10</span>:<span class="cit-fpage">17</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ%26rft.volume%253D10%26rft.spage%253D17%26rft_id%253Dinfo%253Apmid%252F1989882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=1989882&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991EU78700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">71.</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.71" data-doi="10.1128/JVI.03259-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Becares</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pascual-Iglesias</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nogales</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sola</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zuñiga</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Mutagenesis of Coronavirus nsp14 Reveals Its Potential Role in Modulation of the Innate Immune Response</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">90</span>:<span class="cit-fpage">5399</span>–<span class="cit-lpage">5414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03259-15%26rft_id%253Dinfo%253Apmid%252F27009949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI5MC8xMS81Mzk5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDgvMTEvMTc1NTYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">72.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a><div class="cit ref-cit ref-book" id="cit-175562v1.72"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moustafa</span>  <span class="cit-name-given-names">IM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">JJ</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-source">Fidelity of Nucleotide Incorporation by the RNA-Dependent RNA Polymerase from PoliovirusDNA Replication Across Taxa</span>, <span class="cit-edition">1st ed</span>. <span class="cit-publ-name">Elsevier Inc</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">73.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.73" data-doi="10.1016/j.antiviral.2013.07.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">An increased replication fidelity mutant of foot-and-mouth disease virus retains fitness in vitro and virulence in vivo</span>. <abbr class="cit-jnl-abbrev">Antiviral Res</abbr> <span class="cit-vol">100</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2BRes%26rft.volume%253D100%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2013.07.008%26rft_id%253Dinfo%253Apmid%252F23880348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2013.07.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23880348&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">74.</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.74" data-doi="10.1073/pnas.1323705111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Subissi</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Posthuma</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collet</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zevenhoven-Dobbe</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Decroly</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Imbert</span>  <span class="cit-name-given-names">I.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">One severe acute respiratory syndrome coronavirus protein complex integrates processive RNA polymerase and exonuclease activities</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">111</span>:<span class="cit-fpage">E3900</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1323705111%26rft_id%253Dinfo%253Apmid%252F25197083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzM3L0UzOTAwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDgvMTEvMTc1NTYyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">75.</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.75" data-doi="10.1038/sj.emboj.7601368"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Imbert</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guillemot</span>  <span class="cit-name-given-names">J-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bourhis</span>  <span class="cit-name-given-names">J-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bussetta</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coutard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Egloff</span>  <span class="cit-name-given-names">M-P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferron</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canard</span>  <span class="cit-name-given-names">B.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">A second, non-canonical RNA-dependent RNA polymerase in SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr> <span class="cit-vol">25</span>:<span class="cit-fpage">4933</span>–<span class="cit-lpage">4942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DImbert%26rft.auinit1%253DI.%26rft.volume%253D25%26rft.issue%253D20%26rft.spage%253D4933%26rft.epage%253D4942%26rft.atitle%253DA%2Bsecond%252C%2Bnon-canonical%2BRNA-dependent%2BRNA%2Bpolymerase%2Bin%2BSARS%2BCoronavirus%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7601368%26rft_id%253Dinfo%253Apmid%252F17024178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjUvMjAvNDkzMyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">76.</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76. in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.76" data-doi="10.1371/journal.pone.0036521"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Adedeji</span>  <span class="cit-name-given-names">AO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marchand</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Velthuis te</span>  <span class="cit-name-given-names">AJW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eoff</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sarafianos</span>  <span class="cit-name-given-names">SG</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Mechanism of Nucleic Acid Unwinding by SARS-CoV Helicase</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>:<span class="cit-fpage">e36521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DAdedeji%26rft.auinit1%253DA.%2BO.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De36521%26rft.epage%253De36521%26rft.atitle%253DMechanism%2Bof%2Bnucleic%2Bacid%2Bunwinding%2Bby%2BSARS-CoV%2Bhelicase.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0036521%26rft_id%253Dinfo%253Apmid%252F22615777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0036521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22615777&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">77.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.77" data-doi="10.1074/jbc.M210023200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chaudhuri</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chaudhuri</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parris</span>  <span class="cit-name-given-names">DS</span></span>. <span class="cit-pub-date">2003</span>. <span class="cit-article-title">The herpes simplex virus type 1 DNA polymerase processivity factor increases fidelity without altering pre-steady-state rate constants for polymerization or excision</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">278</span>:<span class="cit-fpage">8996</span>–<span class="cit-lpage">9004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M210023200%26rft_id%253Dinfo%253Apmid%252F12519753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNzgvMTEvODk5NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">78.</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78. in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.78" data-doi="10.1038/srep18418"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fleming</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X-E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bi</span>  <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">The ß2 clamp in the Mycobacterium tuberculosis DNA polymerase III aß2e replicase promotes polymerization and reduces exonuclease activity</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">18418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D6%26rft.spage%253D18418%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep18418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep18418&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">79.</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79. in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.79" data-doi="10.1128/JVI.01553-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stapleford</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rozen-Gagnon</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Das</span>  <span class="cit-name-given-names">PK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saul</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poirier</span>  <span class="cit-name-given-names">EZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vidalain</span>  <span class="cit-name-given-names">P-O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Merits</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Viral Polymerase-Helicase Complexes Regulate Replication Fidelity To Overcome Intracellular Nucleotide Depletion</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">89</span>:<span class="cit-fpage">11233</span>– <span class="cit-lpage">11244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01553-15%26rft_id%253Dinfo%253Apmid%252F26311883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4OS8yMi8xMTIzMyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">80.</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80. in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.80" data-doi="10.1016/j.virol.2016.04.023"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Agudo</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">la Higuera</span> <span class="cit-name-surname">de I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arias</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grande-Pérez</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domingo</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Involvement of a joker mutation in a polymerase-independent lethal mutagenesis escape mechanism</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">494</span>:<span class="cit-fpage">257</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D494%26rft.spage%253D257%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2016.04.023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2016.04.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">81.</span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference 81. in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.81" data-doi="10.1073/pnas.0307877101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Egloff</span>  <span class="cit-name-given-names">M-P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferron</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campanacci</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Longhi</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rancurel</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dutartre</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cambillau</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canard</span>  <span class="cit-name-given-names">B.</span></span> <span class="cit-pub-date">2004</span>. <span class="cit-article-title">The severe acute respiratory syndrome-coronavirus replicative protein nsp9 is a single-stranded RNA-binding subunit unique in the RNA virus world</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">101</span>:<span class="cit-fpage">3792</span>–<span class="cit-lpage">3796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0307877101%26rft_id%253Dinfo%253Apmid%252F15007178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzExLzM3OTIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wOC8xMS8xNzU1NjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">82.</span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference 82. in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.82" data-doi="10.1128/JVI.78.22.12218-12224.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bhardwaj</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guarino</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">CC</span></span>. <span class="cit-pub-date">2004</span>. <span class="cit-article-title">The severe acute respiratory syndrome coronavirus Nsp15 protein is an endoribonuclease that prefers manganese as a cofactor</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">78</span>:<span class="cit-fpage">12218</span>–<span class="cit-lpage">12224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DBhardwaj%26rft.auinit1%253DK.%26rft.volume%253D78%26rft.issue%253D22%26rft.spage%253D12218%26rft.epage%253D12224%26rft.atitle%253DThe%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%2BNsp15%2BProtein%2BIs%2Ban%2BEndoribonuclease%2BThat%2BPrefers%2BManganese%2Bas%2Ba%2BCofactor%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.22.12218-12224.2004%26rft_id%253Dinfo%253Apmid%252F15507608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3OC8yMi8xMjIxOCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">83.</span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference 83. in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.83" data-doi="10.1371/journal.pgen.0030093"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanjuán</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Furió</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moya</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Selection for Robustness in Mutagenized RNA Viruses</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">3</span>:<span class="cit-fpage">e93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSanjuan%26rft.auinit1%253DR.%26rft.volume%253D3%26rft.issue%253D6%26rft.spage%253De93%26rft.epage%253De93%26rft.atitle%253DSelection%2Bfor%2Brobustness%2Bin%2Bmutagenized%2BRNA%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030093%26rft_id%253Dinfo%253Apmid%252F17571922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030093&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17571922&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">84.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-175562v1.84"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goldhill</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">ESCP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">PE</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Evolvability and robustness in populations of RNA virus F6</span>. <abbr class="cit-jnl-abbrev">Frontiers in Microbiology</abbr> <span class="cit-vol">5</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">85.</span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference 85. in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.85" data-doi="10.1371/journal.pbio.0030381"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Montville</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Froissart</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Remold</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tenaillon</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">PE</span></span>. <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Evolution of mutational robustness in an RNA virus</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr> <span class="cit-vol">3</span>:<span class="cit-fpage">e381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DMontville%26rft.auinit1%253DR.%26rft.volume%253D3%26rft.issue%253D11%26rft.spage%253De381%26rft.epage%253De381%26rft.atitle%253DEvolution%2Bof%2Bmutational%2Brobustness%2Bin%2Ban%2BRNA%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030381%26rft_id%253Dinfo%253Apmid%252F16248678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030381&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16248678&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">86.</span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference 86. in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.86" data-doi="10.1128/JVI.05712-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Graci</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gnädig</span>  <span class="cit-name-given-names">NF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Galarraga</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Castro</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vignuzzi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CE</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Mutational robustness of an RNA virus influences sensitivity to lethal mutagenesis</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">86</span>:<span class="cit-fpage">2869</span>–<span class="cit-lpage">2873</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.05712-11%26rft_id%253Dinfo%253Apmid%252F22190724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg2LzUvMjg2OSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">87.</span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference 87. in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.87" data-doi="10.1016/j.chom.2012.10.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lauring</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Acevedo</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">SB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Codon usage determines the mutational robustness, evolutionary capacity, and virulence of an RNA virus</span>. <abbr class="cit-jnl-abbrev">Cell Host and Microbe</abbr> <span class="cit-vol">12</span>:<span class="cit-fpage">623</span>–<span class="cit-lpage">632</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2Band%2BMicrobe%26rft.volume%253D12%26rft.spage%253D623%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2012.10.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2012.10.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">88.</span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference 88. in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.88" data-doi="10.1038/nrmicro3003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lauring</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frydman</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andino</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">The role of mutational robustness in RNA virus evolution</span>. <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr> <span class="cit-vol">11</span>:<span class="cit-fpage">327</span>–<span class="cit-lpage">336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BMicrobiol%26rft.volume%253D11%26rft.spage%253D327%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro3003%26rft_id%253Dinfo%253Apmid%252F23524517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro3003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23524517&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">89.</span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference 89. in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.89" data-doi="10.1128/mBio.02020-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aguilera</span>  <span class="cit-name-given-names">ER</span></span>, <span class="cit-auth"><span class="cit-name-surname">Erickson</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jesudhasan</span>  <span class="cit-name-given-names">PR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pfeiffer</span>  <span class="cit-name-given-names">JK</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Plaques Formed by Mutagenized Viral Populations Have Elevated Coinfection Frequencies</span>. <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">8</span>:<span class="cit-fpage">e02020-16</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.02020-16%26rft_id%253Dinfo%253Apmid%252F28292984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibWJpbyI7czo1OiJyZXNpZCI7czoxMzoiOC8yL2UwMjAyMC0xNiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">90.</span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference 90. in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.90" data-doi="10.1371/journal.pcbi.1000811"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">O’Dea</span>  <span class="cit-name-given-names">EB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">TE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">CO</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Does Mutational Robustness Inhibit Extinction by Lethal Mutagenesis in Viral Populations?</span> <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">e1000811</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DO%2527Dea%26rft.auinit1%253DE.%2BB.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253De1000811%26rft.epage%253De1000811%26rft.atitle%253DDoes%2Bmutational%2Brobustness%2Binhibit%2Bextinction%2Bby%2Blethal%2Bmutagenesis%2Bin%2Bviral%2Bpopulations%253F%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000811%26rft_id%253Dinfo%253Apmid%252F20548958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20548958&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F08%2F11%2F175562.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">91.</span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference 91. in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-175562v1.91" data-doi="10.1128/JVI.76.21.11065-11078.2002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yount</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>. <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Systematic assembly of a full-length infectious cDNA of mouse hepatitis virus strain A59</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">76</span>:<span class="cit-fpage">11065</span>–<span class="cit-lpage">11078</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DYount%26rft.auinit1%253DB.%26rft.volume%253D76%26rft.issue%253D21%26rft.spage%253D11065%26rft.epage%253D11078%26rft.atitle%253DSystematic%2BAssembly%2Bof%2Ba%2BFull-Length%2BInfectious%2BcDNA%2Bof%2BMouse%2BHepatitis%2BVirus%2BStrain%2BA59%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.76.21.11065-11078.2002%26rft_id%253Dinfo%253Apmid%252F12368349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3Ni8yMS8xMTA2NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA4LzExLzE3NTU2Mi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/175562v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F175562v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/175265v1" title="The role of a single non-coding nucleotide in the evolution of an epidemic African clade of Salmonella" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/174516v2" title="Computational pipeline for the PGV-001 neoantigen vaccine trial" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;August 11, 2017.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/175562v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/175562v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/175562v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/175562v1" class="active" data-icon-position="" data-hide-link-title="0">Proofreading-deficient coronaviruses adapt over long-term passage for increased fidelity and fitness without reversion of exoribonuclease-inactivating mutations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/52604" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-hJhTCg49Fa3OaWscIJr27tOjPGWzmAHhaWZSdn8Fjuo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="52604" id="node52604--2" data-pisa="biorxiv;175562v1" data-pisa-master="biorxiv;175562" data-seqnum="52604" data-apath="/biorxiv/early/2017/08/11/175562.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Proofreading-deficient coronaviruses adapt over long-term passage for increased fidelity and fitness without reversion of exoribonuclease-inactivating mutations</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Kevin W.</span> <span class="nlm-surname">Graepel</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Xiaotao</span> <span class="nlm-surname">Lu</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">James Brett</span> <span class="nlm-surname">Case</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Nicole R.</span> <span class="nlm-surname">Sexton</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Everett Clinton</span> <span class="nlm-surname">Smith</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Mark R.</span> <span class="nlm-surname">Denison</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">175562; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/175562 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/175562v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/175562v1&amp;title=Proofreading-deficient%20coronaviruses%20adapt%20over%20long-term%20passage%20for%20increased%20fidelity%20and%20fitness%20without%20reversion%20of%20exoribonuclease-inactivating%20mutations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/175562v1&amp;title=Proofreading-deficient%20coronaviruses%20adapt%20over%20long-term%20passage%20for%20increased%20fidelity%20and%20fitness%20without%20reversion%20of%20exoribonuclease-inactivating%20mutations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/175562v1&amp;text=Proofreading-deficient%20coronaviruses%20adapt%20over%20long-term%20passage%20for%20increased%20fidelity%20and%20fitness%20without%20reversion%20of%20exoribonuclease-inactivating%20mutations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/175562v1&amp;title=Proofreading-deficient%20coronaviruses%20adapt%20over%20long-term%20passage%20for%20increased%20fidelity%20and%20fitness%20without%20reversion%20of%20exoribonuclease-inactivating%20mutations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/175562v1&amp;t=Proofreading-deficient%20coronaviruses%20adapt%20over%20long-term%20passage%20for%20increased%20fidelity%20and%20fitness%20without%20reversion%20of%20exoribonuclease-inactivating%20mutations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/175562v1&amp;title=Proofreading-deficient%20coronaviruses%20adapt%20over%20long-term%20passage%20for%20increased%20fidelity%20and%20fitness%20without%20reversion%20of%20exoribonuclease-inactivating%20mutations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/175562v1&amp;title=Proofreading-deficient%20coronaviruses%20adapt%20over%20long-term%20passage%20for%20increased%20fidelity%20and%20fitness%20without%20reversion%20of%20exoribonuclease-inactivating%20mutations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="52604" id="node52604--3" data-pisa="biorxiv;175562v1" data-pisa-master="biorxiv;175562" data-seqnum="52604" data-apath="/biorxiv/early/2017/08/11/175562.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Proofreading-deficient coronaviruses adapt over long-term passage for increased fidelity and fitness without reversion of exoribonuclease-inactivating mutations</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Kevin W.</span> <span class="nlm-surname">Graepel</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Xiaotao</span> <span class="nlm-surname">Lu</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">James Brett</span> <span class="nlm-surname">Case</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Nicole R.</span> <span class="nlm-surname">Sexton</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Everett Clinton</span> <span class="nlm-surname">Smith</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Mark R.</span> <span class="nlm-surname">Denison</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">175562; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/175562 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Proofreading-deficient%20coronaviruses%20adapt%20over%20long-term%20passage%20for%20increased%20fidelity%20and%20fitness%20without%20reversion%20of%20exoribonuclease-inactivating%20mutations&amp;rft.volume&amp;rft.issue&amp;rft.date=2017-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Graepel%2C+Kevin+W.&amp;rft.au=Lu%2C+Xiaotao&amp;rft.au=Case%2C+James+Brett&amp;rft.au=Sexton%2C+Nicole+R.&amp;rft.au=Smith%2C+Everett+Clinton&amp;rft.au=Denison%2C+Mark+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/52604/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/52604/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/52604/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/52604/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/52604/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/52604/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/52604/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/52604/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/52604/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/52604/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/52604/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/52604/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/175562v1&amp;count=horizontal&amp;via=&amp;text=Proofreading-deficient%20coronaviruses%20adapt%20over%20long-term%20passage%20for%20increased%20fidelity%20and%20fitness%20without%20reversion%20of%20exoribonuclease-inactivating%20mutations&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/175562v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/175562v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/175562v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
