<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/embed/graphic-18.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/02/2020.02.28.969782.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.28.969782v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Vorpal: A novel RNA virus feature-extraction algorithm demonstrated through interpretable genotype-to-phenotype linear models"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.28.969782"/>
  <meta name="DC.Date" content="2020-03-02"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="In the analysis of genomic sequence data, so-called “alignment free” approaches are often selected for their relative speed compared to alignment-based approaches, especially in the application of distance comparisons and taxonomic classification[1][1],[2][2],[3][3],[4][4]. These methods are typically reliant on excising K-length substrings of the input sequence, called K-mers[5][5]. In the context of machine learning, K-mer based feature vectors have been used in applications ranging from amplicon sequencing classification to predictive modeling for antimicrobial resistance genes[6][6],[7][7],[8][8]. This can be seen as an analogy of the “bag-of-words” model successfully employed in natural language processing and computer vision for document and image classification[9][9],[10][10]. Feature extraction techniques from natural language processing have previously been analogized to genomics data[11][11]; however, the “bag-of-words” approach is brittle in the RNA virus space due to the high intersequence variance and the exact matching requirement of K-mers. To reconcile the simplicity of “bag-of-words” methods with the complications presented by the intrinsic variance of RNA virus space, a method to resolve the fragility of extracted K-mers in a way that faithfully reflects an underlying biological phenomenon was devised. Our algorithm, Vorpal , allows the construction of interpretable linear models with clustered, representative ‘degenerate’ K-mers as the input vector and, through regularization, sparse predictors of binary phenotypes as the output. Here, we demonstrate the utility of Vorpal by identifying nucleotide-level genomic motif predictors for binary phenotypes in three separate RNA virus clades; human pathogen vs. non-human pathogen in Orthocoronavirinae , hemorrhagic fever causing vs. non-hemorrhagic fever causing in Ebolavirus , and human-host vs. non-human host in Influenza A. The capacity of this approach for in silico identification of hypotheses which can be validated by direct experimentation, as well as identification of genomic targets for preemptive biosurveillance of emerging viruses, is discussed. The code is available for download at &lt;;https://github.com/mriglobal/vorpal&gt;;. [1]: #ref-1 [2]: #ref-2 [3]: #ref-3 [4]: #ref-4 [5]: #ref-5 [6]: #ref-6 [7]: #ref-7 [8]: #ref-8 [9]: #ref-9 [10]: #ref-10 [11]: #ref-11"/>
  <meta name="DC.Contributor" content="Phillip Davis"/>
  <meta name="DC.Contributor" content="John Bagnoli"/>
  <meta name="DC.Contributor" content="David Yarmosh"/>
  <meta name="DC.Contributor" content="Alan Shteyman"/>
  <meta name="DC.Contributor" content="Lance Presser"/>
  <meta name="DC.Contributor" content="Sharon Altmann"/>
  <meta name="DC.Contributor" content="Shelton Bradrick"/>
  <meta name="DC.Contributor" content="Joseph A. Russell"/>
  <meta name="article:published_time" content="2020-03-02"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Vorpal: A novel RNA virus feature-extraction algorithm demonstrated through interpretable genotype-to-phenotype linear models"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;In the analysis of genomic sequence data, so-called alignment free approaches are often selected for their relative speed compared to alignment-based approaches, especially in the application of distance comparisons and taxonomic classification. These methods are typically reliant on excising K-length substrings of the input sequence, called K-mers. In the context of machine learning, K-mer based feature vectors have been used in applications ranging from amplicon sequencing classification to predictive modeling for antimicrobial resistance genes. This can be seen as an analogy of the bag-of-words model successfully employed in natural language processing and computer vision for document and image classification. Feature extraction techniques from natural language processing have previously been analogized to genomics data; however, the bag-of-words approach is brittle in the RNA virus space due to the high intersequence variance and the exact matching requirement of K-mers. To reconcile the simplicity of bag-of-words methods with the complications presented by the intrinsic variance of RNA virus space, a method to resolve the fragility of extracted K-mers in a way that faithfully reflects an underlying biological phenomenon was devised. Our algorithm, Vorpal, allows the construction of interpretable linear models with clustered, representative degenerate K-mers as the input vector and, through regularization, sparse predictors of binary phenotypes as the output. Here, we demonstrate the utility of Vorpal by identifying nucleotide-level genomic motif predictors for binary phenotypes in three separate RNA virus clades; human pathogen vs. non-human pathogen in Orthocoronavirinae, hemorrhagic fever causing vs. non-hemorrhagic fever causing in Ebolavirus, and human-host vs. non-human host in Influenza A. The capacity of this approach for in-silico identification of hypotheses which can be validated by direct experimentation, as well as identification of genomic targets for preemptive biosurveillance of emerging viruses, is discussed. The code is available for download at https://github.com/mriglobal/vorpal.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.28.969782v1"/>
  <meta name="citation_id" content="2020.02.28.969782v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.28.969782v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.28.969782v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.28.969782v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.28.969782"/>
  <meta name="citation_num_pages" content="26"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.28.969782"/>
  <meta name="citation_author" content="Phillip Davis"/>
  <meta name="citation_author_institution" content="MRIGlobal"/>
  <meta name="citation_author_email" content="pdavis@mriglobal.org"/>
  <meta name="citation_author" content="John Bagnoli"/>
  <meta name="citation_author_institution" content="MRIGlobal"/>
  <meta name="citation_author" content="David Yarmosh"/>
  <meta name="citation_author_institution" content="MRIGlobal"/>
  <meta name="citation_author" content="Alan Shteyman"/>
  <meta name="citation_author_institution" content="MRIGlobal"/>
  <meta name="citation_author" content="Lance Presser"/>
  <meta name="citation_author_institution" content="MRIGlobal"/>
  <meta name="citation_author" content="Sharon Altmann"/>
  <meta name="citation_author_institution" content="MRIGlobal"/>
  <meta name="citation_author" content="Shelton Bradrick"/>
  <meta name="citation_author_institution" content="MRIGlobal"/>
  <meta name="citation_author" content="Joseph A. Russell"/>
  <meta name="citation_author_institution" content="MRIGlobal"/>
  <meta name="citation_reference" content="Zielezinski, A., Vinga, S., Almeida, J. &amp;; Karlowski, W. M. Alignment-free sequence comparison: benefits, applications, and tools. Genome Biology 18, (2017)."/>
  <meta name="citation_reference" content="Ondov, B. D. et al. Mash: fast genome and metagenome distance estimation using MinHash. Genome Biology 17, (2016)."/>
  <meta name="citation_reference" content="Koslicki, D. &amp;; Falush, D. MetaPalette: a k-mer Painting Approach for Metagenomic Taxonomic Profiling and Quantification of Novel Strain Variation. mSystems 1, (2016)."/>
  <meta name="citation_reference" content="Déraspe, M. et al. Phenetic Comparison of Prokaryotic Genomes Using k-mers. Molecular Biology and Evolution 34, 2716–2729 (2017)."/>
  <meta name="citation_reference" content="Rizk, G., Lavenier, D. &amp;; Chikhi, R. DSK: k-mer counting with very low memory usage. Bioinformatics 29, 652–653 (2013)."/>
  <meta name="citation_reference" content="Wang, Q., Garrity, G. M., Tiedje, J. M. &amp;; Cole, J. R. Naive Bayesian Classifier for Rapid Assignment of rRNA Sequences into the New Bacterial Taxonomy. Applied and Environmental Microbiology 73, 5261–5267 (2007)."/>
  <meta name="citation_reference" content="Chaudhary, N., Sharma, A. K., Agarwal, P., Gupta, A. &amp;; Sharma, V. K. 16S Classifier: A Tool for Fast and Accurate Taxonomic Classification of 16S rRNA Hypervariable Regions in Metagenomic Datasets. Plos One 10, (2015)."/>
  <meta name="citation_reference" content="Drouin, A. et al. Interpretable genotype-to-phenotype classifiers with performance guarantees. Scientific Reports 9, (2019)."/>
  <meta name="citation_reference" content="Fei-Fei, L. &amp;; Perona, P. A Bayesian Hierarchical Model for Learning Natural Scene Categories. IEEE CVPR. (2005)"/>
  <meta name="citation_reference" content="Asgari, E. &amp;; Mofrad, M. R. K. Continuous Distributed Representation of Biological Sequences for Deep Proteomics and Genomics. Plos One 10, (2015)."/>
  <meta name="citation_reference" content="Domingo, E., Sheldon, J. &amp;; Perales, C. Viral Quasispecies Evolution. Microbiology and Molecular Biology Reviews 76, 159–216 (2012)."/>
  <meta name="citation_reference" content="Sahlgren, M. The Distributional Hypothesis. Rivista di Linguistica (Italian Journal of Linguistics) 20, 33–53 (2008)"/>
  <meta name="citation_reference" content="Martell, H. J., Masterson, S. G., Mcgreig, J. E., Michaelis, M. &amp;; Wass, M. N. Is the Bombali virus pathogenic in humans? Bioinformatics 35, 3553–3558 (2019)."/>
  <meta name="citation_reference" content="Shabman, R. S. et al. An Upstream Open Reading Frame Modulates Ebola Virus Polymerase Translation and Virus Replication. PLoS Pathogens 9, (2013)."/>
  <meta name="citation_reference" content="Ikegami, T. et al. Genome structure of Ebola virus subtype Reston: differences among Ebola subtypes. Archives of Virology 146, 2021–2027 (2001)."/>
  <meta name="citation_reference" content="Sriwilaijaroen, N. &amp;; Suzuki, Y. Molecular basis of the structure and function of H1 hemagglutinin of influenza virus. Proceedings of the Japan Academy, Series B 88, 226–249 (2012)."/>
  <meta name="citation_reference" content="Nakamura, Y. Codon usage tabulated from international DNA sequence databases: status for the year 2000. Nucleic Acids Research 28, 292–292 (2000)."/>
  <meta name="citation_reference" content="Clarke, T. F. &amp;; Clark, P. L. Rare Codons Cluster. PLoS ONE 3, (2008)."/>
  <meta name="citation_reference" content="Cruz, E., Cain, J., Crossett, B. &amp;; Kayser, V. Site-specific glycosylation profile of influenza A (H1N1) hemagglutinin through tandem mass spectrometry. Human Vaccines &amp;; Immunotherapeutics 14, 508–517 (2017)."/>
  <meta name="citation_reference" content="Ebrahimi, K. H., West, G. M. &amp;; Flefil, R. Mass Spectrometry Approach and ELISA Reveal the Effect of Codon Optimization on N-Linked Glycosylation of HIV-1 gp120. Journal of Proteome Research 13, 5801–5811 (2014)."/>
  <meta name="citation_reference" content="Zucchelli, E. et al. Codon Optimization Leads to Functional Impairment of RD114-TR Envelope Glycoprotein. Molecular Therapy - Methods &amp;; Clinical Development 4, 102–114 (2017)."/>
  <meta name="citation_reference" content="Min, J.-Y. et al. Mammalian Adaptation in the PB2 Gene of Avian H5N1 Influenza Virus. Journal of Virology 87, 10884–10888 (2013)."/>
  <meta name="citation_reference" content="Tinoco, Y. O. et al. Transmission dynamics of pandemic influenza A(H1N1)pdm09 virus in humans and swine in backyard farms in Tumbes, Peru. Influenza and Other Respiratory Viruses 10, 47–56 (2015)."/>
  <meta name="citation_reference" content="Li, D. et al. Influenza A(H1N1)pdm09 Virus Infection in Giant Pandas, China. Emerging Infectious Diseases 20, 480–483 (2014)."/>
  <meta name="citation_reference" content="Shinde, V. et al. Triple-Reassortant Swine Influenza A (H1) in Humans in the United States, 2005–2009. New England Journal of Medicine 360, 2616–2625 (2009)."/>
  <meta name="citation_reference" content="Bastien, N. et al. Human Infection with a Triple-Reassortant Swine Influenza A(H1N1) Virus Containing the Hemagglutinin and Neuraminidase Genes of Seasonal Influenza Virus. The Journal of Infectious Diseases 201, 1178–1182 (2010)."/>
  <meta name="citation_reference" content="Jonas, O. &amp;; Seifman, R. Do we need a Global Virome Project? The Lancet Global Health 7, (2019)."/>
  <meta name="citation_reference" content="Carlson, C. J., Zipfel, C. M., Garnier, R. &amp;; Bansal, S. Global estimates of mammalian viral diversity accounting for host sharing. Nature Ecology &amp;; Evolution 3, 1070–1075 (2019)."/>
  <meta name="citation_reference" content="Fan, R.-E., Chang, K.-W., Hsieh, C.-J., Wang, X.-R. &amp;; Lin, C.-J. LIBLINEAR: A Library for Large Linear Classification. Journal of Machine Learning Research 9, 1871–1874 (2008)."/>
  <meta name="citation_reference" content="Bottou, L. Large-Scale Machine Learning with Stochastic Gradient Descent. Proceedings of COMPSTAT2010 177–186 (2010). doi:10.1007/978-3-7908-2604-3_16"/>
  <meta name="citation_reference" content="Zhang, Q., Jun, S.-R., Leuze, M., Ussery, D. &amp;; Nookaew, I. Viral Phylogenomics Using an Alignment-Free Method: A Three-Step Approach to Determine Optimal Length of k-mer. Scientific Reports 7, (2017)."/>
  <meta name="citation_reference" content="Kurtz, S., Narechania, A., Stein, J. C. &amp;; Ware, D. A new method to compute K-mer frequencies and its application to annotate large repetitive plant genomes. BMC Genomics 9, 517 (2008)."/>
  <meta name="citation_reference" content="Hysom, D. A. et al. Skip the Alignment: Degenerate, Multiplex Primer and Probe Design Using K-mer Matching Instead of Alignments. PLoS ONE 7, (2012)."/>
  <meta name="citation_reference" content="Müllner, D. fastcluster: Fast Hierarchical, Agglomerative Clustering Routines forRandPython. Journal of Statistical Software 53, (2013)."/>
  <meta name="citation_reference" content="A revision of the system of nomenclature for influenza viruses: a WHO memorandum. Bulletin of the World Health Organization 58, 585–591 (1980)."/>
  <meta name="citation_reference" content="Wang, M. et al. SARS-CoV Infection in a Restaurant from Palm Civet. Emerging Infectious Diseases 11, 1860–1865 (2005)."/>
  <meta name="citation_reference" content="Younan, M., Bornstein, S. &amp;; Gluecks, I. V. MERS and the dromedary camel trade between Africa and the Middle East. Tropical Animal Health and Production 48, 1277–1282 (2016)."/>
  <meta name="citation_reference" content="Hu, B. et al. Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus. PLOS Pathogens 13, (2017)."/>
  <meta name="citation_reference" content="Goodacre, N., Aljanahi, A., Nandakumar, S., Mikailov, M. &amp;; Khan, A. S. A Reference Viral Database (RVDB) To Enhance Bioinformatics Analysis of High-Throughput Sequencing for Novel Virus Detection. mSphere 3, (2018)."/>
  <meta name="citation_reference" content="Hatcher, E. L. et al. Virus Variation Resource – improved response to emergent viral outbreaks. Nucleic Acids Research 45, (2016)."/>
  <meta name="citation_reference" content="Manning, C. D. in Introduction to Information Retrieval 403–419 (Cambridge University Press, 2008)."/>
  <meta name="citation_reference" content="van der Maaten, L. &amp;; Hinton, G. Visualizing High-Dimensional Data Using t-SNE. Journal of Machine Learning Research 9, 2579–2605 (2008)."/>
  <meta name="citation_reference" content="van der Maaten, L. Accelerating t-SNE using Tree-Based Algorithms. Journal of Machine Learning Research 15, 3221–3245 (2014)."/>
  <meta name="citation_reference" content="Plotly Technologies Inc. Collaborative data science. 2015. https://plot.ly."/>
  <meta name="twitter:title" content="Vorpal: A novel RNA virus feature-extraction algorithm demonstrated through interpretable genotype-to-phenotype linear models"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="In the analysis of genomic sequence data, so-called “alignment free” approaches are often selected for their relative speed compared to alignment-based approaches, especially in the application of distance comparisons and taxonomic classification[1][1],[2][2],[3][3],[4][4]. These methods are typically reliant on excising K-length substrings of the input sequence, called K-mers[5][5]. In the context of machine learning, K-mer based feature vectors have been used in applications ranging from amplicon sequencing classification to predictive modeling for antimicrobial resistance genes[6][6],[7][7],[8][8]. This can be seen as an analogy of the “bag-of-words” model successfully employed in natural language processing and computer vision for document and image classification[9][9],[10][10]. Feature extraction techniques from natural language processing have previously been analogized to genomics data[11][11]; however, the “bag-of-words” approach is brittle in the RNA virus space due to the high intersequence variance and the exact matching requirement of K-mers. To reconcile the simplicity of “bag-of-words” methods with the complications presented by the intrinsic variance of RNA virus space, a method to resolve the fragility of extracted K-mers in a way that faithfully reflects an underlying biological phenomenon was devised. Our algorithm, Vorpal , allows the construction of interpretable linear models with clustered, representative ‘degenerate’ K-mers as the input vector and, through regularization, sparse predictors of binary phenotypes as the output. Here, we demonstrate the utility of Vorpal by identifying nucleotide-level genomic motif predictors for binary phenotypes in three separate RNA virus clades; human pathogen vs. non-human pathogen in Orthocoronavirinae , hemorrhagic fever causing vs. non-hemorrhagic fever causing in Ebolavirus , and human-host vs. non-human host in Influenza A. The capacity of this approach for in silico identification of hypotheses which can be validated by direct experimentation, as well as identification of genomic targets for preemptive biosurveillance of emerging viruses, is discussed. The code is available for download at &lt;;https://github.com/mriglobal/vorpal&gt;;. [1]: #ref-1 [2]: #ref-2 [3]: #ref-3 [4]: #ref-4 [5]: #ref-5 [6]: #ref-6 [7]: #ref-7 [8]: #ref-8 [9]: #ref-9 [10]: #ref-10 [11]: #ref-11"/>
  <meta name="og-title" property="og:title" content="Vorpal: A novel RNA virus feature-extraction algorithm demonstrated through interpretable genotype-to-phenotype linear models"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.28.969782v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="In the analysis of genomic sequence data, so-called “alignment free” approaches are often selected for their relative speed compared to alignment-based approaches, especially in the application of distance comparisons and taxonomic classification[1][1],[2][2],[3][3],[4][4]. These methods are typically reliant on excising K-length substrings of the input sequence, called K-mers[5][5]. In the context of machine learning, K-mer based feature vectors have been used in applications ranging from amplicon sequencing classification to predictive modeling for antimicrobial resistance genes[6][6],[7][7],[8][8]. This can be seen as an analogy of the “bag-of-words” model successfully employed in natural language processing and computer vision for document and image classification[9][9],[10][10]. Feature extraction techniques from natural language processing have previously been analogized to genomics data[11][11]; however, the “bag-of-words” approach is brittle in the RNA virus space due to the high intersequence variance and the exact matching requirement of K-mers. To reconcile the simplicity of “bag-of-words” methods with the complications presented by the intrinsic variance of RNA virus space, a method to resolve the fragility of extracted K-mers in a way that faithfully reflects an underlying biological phenomenon was devised. Our algorithm, Vorpal , allows the construction of interpretable linear models with clustered, representative ‘degenerate’ K-mers as the input vector and, through regularization, sparse predictors of binary phenotypes as the output. Here, we demonstrate the utility of Vorpal by identifying nucleotide-level genomic motif predictors for binary phenotypes in three separate RNA virus clades; human pathogen vs. non-human pathogen in Orthocoronavirinae , hemorrhagic fever causing vs. non-hemorrhagic fever causing in Ebolavirus , and human-host vs. non-human host in Influenza A. The capacity of this approach for in silico identification of hypotheses which can be validated by direct experimentation, as well as identification of genomic targets for preemptive biosurveillance of emerging viruses, is discussed. The code is available for download at &lt;;https://github.com/mriglobal/vorpal&gt;;. [1]: #ref-1 [2]: #ref-2 [3]: #ref-3 [4]: #ref-4 [5]: #ref-5 [6]: #ref-6 [7]: #ref-7 [8]: #ref-8 [9]: #ref-9 [10]: #ref-10 [11]: #ref-11"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-02"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Vorpal: A novel RNA virus feature-extraction algorithm demonstrated through interpretable genotype-to-phenotype linear models | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1177347 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">SUMMARY</h2>
    <p id="p-2">In the analysis of genomic sequence data, so-called “alignment free” approaches are often selected for their relative speed compared to alignment-based approaches, especially in the application of distance comparisons and taxonomic classification
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>,
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>,
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>,
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
     </sup>. These methods are typically reliant on excising K-length substrings of the input sequence, called K-mers
     <sup>
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
     </sup>. In the context of machine learning, K-mer based feature vectors have been used in applications ranging from amplicon sequencing classification to predictive modeling for antimicrobial resistance genes
     <sup>
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>,
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>,
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
     </sup>. This can be seen as an analogy of the “bag-of-words” model successfully employed in natural language processing and computer vision for document and image classification
     <sup>
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>,
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>
     </sup>. Feature extraction techniques from natural language processing have previously been analogized to genomics data
     <sup>
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>
     </sup>; however, the “bag-of-words” approach is brittle in the RNA virus space due to the high intersequence variance and the exact matching requirement of K-mers. To reconcile the simplicity of “bag-of-words” methods with the complications presented by the intrinsic variance of RNA virus space, a method to resolve the fragility of extracted K-mers in a way that faithfully reflects an underlying biological phenomenon was devised. Our algorithm, 
     <em>Vorpal</em>, allows the construction of interpretable linear models with clustered, representative ‘degenerate’ K-mers as the input vector and, through regularization, sparse predictors of binary phenotypes as the output. Here, we demonstrate the utility of 
     <em>Vorpal</em> by identifying nucleotide-level genomic motif predictors for binary phenotypes in three separate RNA virus clades; human pathogen vs. non-human pathogen in 
     <em>Orthocoronavirinae</em>, hemorrhagic fever causing vs. non-hemorrhagic fever causing in 
     <em>Ebolavirus</em>, and human-host vs. non-human host in Influenza A. The capacity of this approach for 
     <em>in silico</em> identification of hypotheses which can be validated by direct experimentation, as well as identification of genomic targets for preemptive biosurveillance of emerging viruses, is discussed. The code is available for download at 
     <a href="https://github.com/mriglobal/vorpal">https://github.com/mriglobal/vorpal</a>.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Feature Extraction Algorithm Overview</h2>
    <p id="p-4">In the quasispecies model, the virus organism is represented by the “cloud” of genotypes that can be maintained by the virus within the allowable fitness parameters
     <sup>
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>
     </sup>. In the method proposed here, the frame of reference for the quasispecies “cloud” is reduced to the level of K-length motifs. In order to estimate the connectedness of these K-mers across the input assemblies, a distance matrix between all of the unique K-mers observed across the designated virus genome assemblies is established using hamming distance. Hierarchical clustering is then performed on the resulting distance matrix using an average linkage function, corresponding to the ultrametric assumption used in Unweighted Pair Group Method with Arithmetic Mean (UPGMA) phylogenies, and flat clusters are extracted using a hyperparameter for the distance cutoff of cluster membership. The constituents of these clusters are then aligned and their positional variants represented using the International Union of Pure and Applied Chemistry (IUPAC) nucleic acid notation with degenerate base symbols. These degenerate motifs are mapped back to their respective assemblies. This approach facilitates interpretation of model features in a functional profiling and hypothesis generating context. To demonstrate the effectiveness of this new feature extraction technique, genotype-to-phenotype linear models were trained on various RNA virus groups. A description of the Python implementation of the algorithm is detailed in Methods and the code is available for download at 
     <a href="https://github.com/mriglobal/vorpal">https://github.com/mriglobal/vorpal</a>, along with persistent versions of the models described here-in. A simplified example of the agglomerative clustering step is depicted in 
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Hierarchical K-mer Clustering. A simplified example of K-mer clustering to produce degenerate motifs. After K-mer counting and filtering on frequency, K-mers are clustered using an average linkage function with hamming distance, or positional agreement, as the metric. The resulting alignments, after tree-cutting at a user specified cutoff, are collapsed into their IUPAC character representation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1790953588" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Hierarchical K-mer Clustering.&lt;;/span&gt;; A simplified example of K-mer clustering to produce degenerate motifs. After K-mer counting and filtering on frequency, K-mers are clustered using an average linkage function with hamming distance, or positional agreement, as the metric. The resulting alignments, after tree-cutting at a user specified cutoff, are collapsed into their IUPAC character representation.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F1.medium.gif" width="440" height="153"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F1.medium.gif" width="440" height="153"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1.</span>
      <span class="caption-title">Hierarchical K-mer Clustering.</span>
      <p id="p-5" class="first-child">A simplified example of K-mer clustering to produce degenerate motifs. After K-mer counting and filtering on frequency, K-mers are clustered using an average linkage function with hamming distance, or positional agreement, as the metric. The resulting alignments, after tree-cutting at a user specified cutoff, are collapsed into their IUPAC character representation.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p>By their nature, feature extraction methods make either explicit or implicit hypotheses about what the learner can discover about the data. For instance, in the Natural Language Processing (NLP) domain, the famous “distributional hypothesis” is what forms the theoretical framework for word embedding algorithms such as Word2Vec
     <sup>
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>,
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>
     </sup>. The hypothesis central to the Vorpal algorithm makes the following predictions about the types of phenomena that could be learned from RNA virus genomics data, if they are relevant to the output label: 
    </p>
    <ol class="list-ord " id="list-1">
     <li id="list-item-1">
      <p id="p-7">The predictive motifs are positionally independent</p>
     </li>
     <li id="list-item-2">
      <p id="p-8">The frequency of occurrence of a motif is predictive</p>
     </li>
     <li id="list-item-3">
      <p id="p-9">There are predictive motifs observable only at the nucleic acid level, i.e. in non-coding regions or not observable in the translated product</p>
     </li>
    </ol>
    <p id="p-10">The strongest predictors for the output phenotypes in the models discussed in this paper demonstrate each of these phenomena.</p>
    <p id="p-11">Three RNA virus groups were chosen to evaluate the methodology, due to their relevance as important human pathogens – Orthocoronavirinae at the sub-family level, Ebolavirus at the genus level, and Influenza A at the species level. The phenotypes for these virus groups were binary output variables corresponding to human pathogen (vs. non-pathogen), human-hemorrhagic-fever-causing (vs. not human-hemorrhagic-fever-causing), and human-host isolate (vs. non-human-host isolate), respectively. The procedure for labeling these phenotypes is detailed in Methods.</p>
    <p id="p-12">This entire algorithm was developed and implemented using Biopython, skbio, and the scipy computing stack contained in the open-source Anaconda Distribution.</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Results</h2>
    <p id="p-13">Logistic regression models were fit, in triplicate, for the binary phenotypes described above, across different degeneracy cutoffs for the Ebolavirus and Orthocoronavirinae groups. Due to the training time for the Influenza A models (around 72 hours), instead of exploring different degeneracy cutoffs to find the sparsest feature vector, all Influenza A segment models, which were fit independently, were evaluated with a 1.5 degeneracy cutoff for clustering. Model parameter selection for degeneracy cutoff is visualized in 
     <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. All models were highly accurate on both the training and test sets. Selected models are summarized in 
     <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Construction of the training and test sets is described in the Methods section.
    </p>
    <div id="F2" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Degeneracy cutoff parameter search. Range of feature vector sizes across different degeneracy cutoff levels. Ebolavirus and Orthocoronavirinae find the least number of non-zero coefficients in the weights vector at 2.0 and 4.0 average degeneracy respectively. They also find very high numerical stability at these cutoffs, with repeated fitting returning almost identical motif set membership. Error bars correspond to standard error of the mean." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1790953588" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Degeneracy cutoff parameter search.&lt;;/span&gt;; Range of feature vector sizes across different degeneracy cutoff levels. Ebolavirus and Orthocoronavirinae find the least number of non-zero coefficients in the weights vector at 2.0 and 4.0 average degeneracy respectively. They also find very high numerical stability at these cutoffs, with repeated fitting returning almost identical motif set membership. Error bars correspond to standard error of the mean.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F2.medium.gif" width="440" height="236"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F2.medium.gif" width="440" height="236"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 2.</span>
      <span class="caption-title">Degeneracy cutoff parameter search.</span>
      <p id="p-14" class="first-child">Range of feature vector sizes across different degeneracy cutoff levels. Ebolavirus and Orthocoronavirinae find the least number of non-zero coefficients in the weights vector at 2.0 and 4.0 average degeneracy respectively. They also find very high numerical stability at these cutoffs, with repeated fitting returning almost identical motif set membership. Error bars correspond to standard error of the mean.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="T1" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1179410/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1179410/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1179410" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table 1.</span>
      <span class="caption-title">Models Summary.</span>
      <p id="p-15" class="first-child">A summary of the attributes for the models built for each RNA virus group that are discussed. NNZs indicate number of non-zero coefficients in the weights vector after regularization.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="sec-3" class="subsection">
     <h3>Explanatory Modeling through Feature Selection</h3>
     <p id="p-16">Tables containing the motif identity and corresponding coefficients for the selected models, along with a list of the accession numbers used for training and test sets, are provided as part of the Supplementary materials. We encourage researchers to explore the contents of these models. Below, we analyze a handful of properties of the models to explain their utility in interpretation.</p>
     <div id="sec-4" class="subsection">
      <h4>Orthocoronavirinae</h4>
      <p id="p-17">The model for the Orthocoronavirinae sub-family was built around the phenotype of human pathogen. The motif with the highest coefficient for the human pathogen phenotype, AKRATGKTGTTAATMAA, is an example of the positional independence phenomena that the Vorpal algorithm could learn if it contains information about the response variable. The motif also appears across both Alphacoronavirus and Betacoronavirus group species that infect humans. Interestingly its pattern of appearance in those groups varies in a way not predicated on this taxonomic organization. In the Alphacoronavirus examples that it appears in, namely 229E and NL63, this motif is located in the same reading frame within the spike S2 glycoprotein protein and encodes a conserved QDVVNQ amino acid sequence. However, when it appears in Severe Acute Respiratory Syndrome (SARS), it remains in the same reading frame, coding for a YNVVNK amino acid sequence, but instead occurs in the polyprotein in the N-terminus of non-structural protein (NSP) 15. The other Betacoronavirus member it appears in, OC43, presents this motif in the same reading frame but it has returned to the spike protein as QDGVNK. This motif serves as a signal for human pathogenicity whose importance is based at least partially on its translation, though the domain itself can appear in completely different protein products. It was also recognized that another positive predictor in the model was a motif related to this one, KGATGTTGTTARWCAAY, offset by a single nucleotide. This related motif sometimes co-occurred at the same position as the one mentioned above, and other times appears at a different position in the genome, which suggests this is part of a larger, repetitive motif.</p>
      <p id="p-18">This is summarized in 
       <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>.
      </p>
      <div id="T2" class="table pos-float">
       <div class="table-inline table-callout-links">
        <div class="callout">
         <span>View this table:</span>
         <ul class="callout-links">
          <li class="view-inline first">
           <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1179404/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
          </li>
          <li class="view-popup">
           <a href="/highwire/markup/1179404/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
          </li>
          <li class="download-ppt last">
           <a href="/highwire/powerpoint/1179404" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
          </li>
         </ul>
        </div>
       </div>
       <div class="table-caption">
        <span class="table-label">Table 2.</span>
        <span class="caption-title">Positive Coefficient Coronavirus (COV) motifs of interest.</span>
        <p id="p-19" class="first-child">Organism, genome locations, and corresponding translated products for selected predictors in the Orthocoronavirinae model. Bolded examples are instances labeled Non-human-pathogens in the training set, all others are members of the Human pathogen class. Note: 2019-nCoV was not part of the training set when these models were developed.</p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
     </div>
     <div id="sec-5" class="subsection">
      <h4>Ebolavirus</h4>
      <p id="p-20">The model for the genus Ebolavirus was specified for a phenotype corresponding to human-hemorrhagic-fever causing, i.e. the African Ebolavirus constituents, and non-human-hemorrhagic-fever causing, i.e. Reston ebolavirus (EBOV). The recently discovered Bombali EBOV, was excluded due to its ambiguity as a human pathogen
       <sup>
        <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>
       </sup>.
      </p>
      <p id="p-21">The Ebola model demonstrates the utility of the assumption in the Vorpal algorithm that the feature vector contains information about the frequency of genomic motifs. The preservation of repeated motifs in the 5’ untranslated region (UTR), especially of those in the overlapping UTRs in the Ebola genome, are the predictors of primary importance in differentiating the phenotypes. These repeating motifs, or “motif blocks”, and their corresponding coefficients in the model, are summarized in 
       <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and visualized in 
       <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. These motifs in the 5’ UTRs, specifically in the leading sequence of the L protein, have been previously established as being functionally important to growth kinetics in cell culture
       <sup>
        <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>
       </sup>. The presence and location of this motif across the Reston and African constituents of the Ebola genus forms an obvious distinguishing factor. The contiguous block of overlapping motifs identified in 
       <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>, appear across all known Ebolavirus genomes. However, in the Reston version, this block appears only in the 5’ UTR of VP40 and L, which is one of the several genome locations in Reston containing overlapping 3’ and 5’ UTRs. When this motif block occurs in the African-derived constituents of the Ebola genus, it appears in the 5’ UTR of VP40, VP30, and L. The VP40 and VP30 5’ UTRs are characterized by overlapping transcriptional units in African ebolaviruses. In 
       <em>Zaire ebolavirus,</em> there is an intergenic region between VP24 and L protein. However, despite the insertion of an intergenic region at this location in 
       <em>Zaire ebolavirus</em>, this representation of the motif block is still preserved. Comparison of the transcriptional start and stop signals between Reston and Zaire ebolavirus has been performed before
       <sup>
        <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>
       </sup>, but the conservation of this motif and this pattern of appearance across the genus has not been established to our knowledge.
      </p>
      <div id="T3" class="table pos-float">
       <div class="table-inline table-callout-links">
        <div class="callout">
         <span>View this table:</span>
         <ul class="callout-links">
          <li class="view-inline first">
           <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1179416/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
          </li>
          <li class="view-popup">
           <a href="/highwire/markup/1179416/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
          </li>
          <li class="download-ppt last">
           <a href="/highwire/powerpoint/1179416" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
          </li>
         </ul>
        </div>
       </div>
       <div class="table-caption">
        <span class="table-label">Table 3.</span>
        <span class="caption-title">Ebolavirus overlapping UTR “ motif block”.</span>
        <p id="p-22" class="first-child">Contiguous motifs that form the 5’UTR overlap conserved at varying frequency across the entire Ebola genus. Identical coefficients represent completely colinear predictors.</p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <div id="F3" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Ebolavirus UTR overlap mapping. Visual comparison of the UTR overlap motifs specified in Table 3. A) Mapping of motifs on the Zaire Ebolavirus genome. The motifs occur three times in the African constituents of Ebolavirus. B) Mapping of motifs on the Reston Ebolavirus genome. The motifs occur only twice in Reston ebolavirus, with the UTR overlap between VP30 and VP24 replaced by an intergenic spacer." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1790953588" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Ebolavirus UTR overlap mapping.&lt;;/span&gt;; Visual comparison of the UTR overlap motifs specified in Table 3. A) Mapping of motifs on the Zaire Ebolavirus genome. The motifs occur three times in the African constituents of Ebolavirus. B) Mapping of motifs on the Reston Ebolavirus genome. The motifs occur only twice in Reston ebolavirus, with the UTR overlap between VP30 and VP24 replaced by an intergenic spacer.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F3.medium.gif" width="440" height="269"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F3.medium.gif" width="440" height="269"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">Figure 3.</span>
        <span class="caption-title">Ebolavirus UTR overlap mapping.</span>
        <p id="p-23" class="first-child">Visual comparison of the UTR overlap motifs specified in 
         <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>. A) Mapping of motifs on the Zaire Ebolavirus genome. The motifs occur three times in the African constituents of Ebolavirus. B) Mapping of motifs on the Reston Ebolavirus genome. The motifs occur only twice in Reston ebolavirus, with the UTR overlap between VP30 and VP24 replaced by an intergenic spacer.
        </p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
     </div>
     <div id="sec-6" class="subsection">
      <h4>Influenza A</h4>
      <p id="p-24">The Influenza A model was trained using isolation host as the output variable. As illustrated in 
       <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> above, an independent model was built around each segment of Influenza A’s genome. Therefore, the model is trying to find signals of host conformational changes on each segment. However, within the constraints of this paper, only results derived from the segment 4 model will be discussed in detail.
      </p>
      <p id="p-25">Influenza A’s fourth segment contains the HA gene from which Influenza A strains derive their H subtype designation. In the corresponding model, a pattern was observed in the motif distributions that was common to all of the Influenza A segment models examined. This pattern aligns with the third assumption associated with the Vorpal feature extraction method – some degenerate predictors encode only silent mutations. In other words, the signal for the output label is observed only at the nucleotide level for many explanatory variables. For example, one of the highest coefficient predictors for the human-isolate phenotype, GTCTCTACARTGTAGAA, appeared to be related to one of the motifs amongst the most negative predictors, GGTCTYTACARTGTAGA. These motifs correspond to a location towards the end of the C terminus of the HA2 protein, at the location of a conserved, H1-subtype, N-linked glycosylation site following the transmembrane region
       <sup>
        <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>
       </sup>. The pattern of appearance for these motifs is described in 
       <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4A</a>.
      </p>
      <div id="T4" class="table pos-float">
       <div class="table-inline table-callout-links">
        <div class="callout">
         <span>View this table:</span>
         <ul class="callout-links">
          <li class="view-inline first">
           <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1179417/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
          </li>
          <li class="view-popup">
           <a href="/highwire/markup/1179417/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
          </li>
          <li class="download-ppt last">
           <a href="/highwire/powerpoint/1179417" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
          </li>
         </ul>
        </div>
       </div>
       <div class="table-caption">
        <span class="table-label">Table 4A.</span>
        <span class="caption-title">Influenza A HA2 motifs.</span>
        <p id="p-26" class="first-child">Shows three overlapping segments where the addition of a degeneracy allowing for the TTA codon for leucine is an important predictor for the non-human conformation for the H1 subtype. The third motif with no coefficient was identified by looking in avian isolates at the same genomic position. This motif was not used by the model but provides additional interpretation of the phenomenon in effect. The only avian flu examples in the model predictors that these motifs appear in are North American Turkey isolates. No other avian examples of any HA gene subtypes contain these motifs utilizing rare leucine codons. This serine at the beginning of this amino acid sequence is the tail constituent of a N-x-S/T glycosylation motif.</p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <p id="p-27">Examination of the constituent K-mers of these motifs demonstrated that the allowance of the negative predictor to map to the TTA leucine codon introduced, almost exclusively, swine isolates. The conservation of the CTA leucine codon in the human-isolate predictor is noteworthy because this codon is one of the rare leucine codons in the human genome, with a relative frequency of 7%. Alternatively, the TTA codon being more predictive for swine isolates is notable because while TTA also only has a 7% relative abundance in humans, its abundance in pigs is 6% while the CTA codon is less rare (13% relative abundance)
       <sup>
        <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>
       </sup>. This mammalian adaptation separates it almost entirely from any avian examples and there appears to be a fitness gradient. When it appears in mammals, there is a higher incidence of the uncommon leucine codon at this location. As previously mentioned, the SLQCR motif is canonical across all H1 subtype examples, including those of chicken and duck. A degenerate motif that mapped to the corresponding position in avian examples was determined to be GGTCWTTGCAATGCAGA. The underlying nucleotide conformations appear to be strictly enforced where the use of the TTG codon for leucine, along with the TGC codon for cysteine, produces 427 avian examples and only 14 human examples. Curiously, the preference for these codons in the avian examples are not correlated with their rarity in those hosts. The TTG codon for leucine has a relative frequency of 13% in mallards, while the CTA and TTA codons are both 6%. A table of the relative codon frequencies by host are noted in 
       <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 4B</a>, and this relationship between motif mapping frequency following a codon rarity gradient in mammals, and the inverse in birds, is visualized in 
       <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>.
      </p>
      <div id="T5" class="table pos-float">
       <div class="table-inline table-callout-links">
        <div class="callout">
         <span>View this table:</span>
         <ul class="callout-links">
          <li class="view-inline first">
           <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1179409/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
          </li>
          <li class="view-popup">
           <a href="/highwire/markup/1179409/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
          </li>
          <li class="download-ppt last">
           <a href="/highwire/powerpoint/1179409" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
          </li>
         </ul>
        </div>
       </div>
       <div class="table-caption">
        <span class="table-label">Table 4B.</span>
        <span class="caption-title">Relative Host Codon Frequencies for HA2 motifs.</span>
        <p id="p-28" class="first-child">Shows three overlapping segments where the addition of a degeneracy allowing for the TTA codon for leucine is an important predictor for the non-human conformation for the H1 subtype. The third motif with no coefficient was identified by looking in avian isolates at the same genomic position.</p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <div id="F4" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Host Codon Optimizations for H1 subtype. The justification for the coefficients assigned by the model for the motifs specified in Table4A are demonstrated by the clear role that the TTA codon for leucine plays in increasing the probability of a Swine isolate classification for the H1 subtype. Arrows indicating the increase or decrease in total number of motif mappings point in a direction along the relative host codon frequency gradient where rightwards movement indicates optimization towards the lower frequency rank for the corresponding amino acid. Table 4B shows the relative frequencies for these codons across these animal clades. Magnitude of arrows expressing change in number of reference mappings are not drawn to scale." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1790953588" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Host Codon Optimizations for H1 subtype.&lt;;/span&gt;; The justification for the coefficients assigned by the model for the motifs specified in Table4A are demonstrated by the clear role that the TTA codon for leucine plays in increasing the probability of a Swine isolate classification for the H1 subtype. Arrows indicating the increase or decrease in total number of motif mappings point in a direction along the relative host codon frequency gradient where rightwards movement indicates optimization towards the lower frequency rank for the corresponding amino acid. Table 4B shows the relative frequencies for these codons across these animal clades. Magnitude of arrows expressing change in number of reference mappings are not drawn to scale.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F4.medium.gif" width="440" height="384"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F4.medium.gif" width="440" height="384"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">Figure 4.</span>
        <span class="caption-title">Host Codon Optimizations for H1 subtype.</span>
        <p id="p-29" class="first-child">The justification for the coefficients assigned by the model for the motifs specified in 
         <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table4A</a> are demonstrated by the clear role that the TTA codon for leucine plays in increasing the probability of a Swine isolate classification for the H1 subtype. Arrows indicating the increase or decrease in total number of motif mappings point in a direction along the relative host codon frequency gradient where rightwards movement indicates optimization towards the lower frequency rank for the corresponding amino acid. 
         <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 4B</a> shows the relative frequencies for these codons across these animal clades. Magnitude of arrows expressing change in number of reference mappings are not drawn to scale.
        </p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <p id="p-30">The predictor variable with the highest coefficient from the segment 4 model is another, more dramatic example, of the phenomenon described above. The identity of the motif, AATGTRACAGTAACACA, and its translated product, NVTVTH, again demonstrate a preference for rare human codons - in this case, valine. Like the example discussed above, this motif is present almost exclusively in human (N=15913) and swine (N=3885) examples of the H1 subtype. The associated NVTVTH amino acid sequence is also completely conserved across all the examples, avian included. The valine codons in the human-isolate versions, almost exclusively GTA, have a relative frequency of only 11% in the human genome. While in the avian examples of segment 4, those codons are switched to GTG, which are the most common Valine codons with a 46% relative frequency in mallards. A motif for the avian version of this was developed using a multiple sequence alignment of the non-human and non-swine isolates of the H1 subtype assemblies in the training set. This motif was established as AAYGTRACYGTGACYCA and mapped back to the training set sequences. When mapped, this new motif was resolved to 480 avian isolates, 33 swine isolates, and nothing else. Unfortunately, unlike the above-mentioned Influenza A motif, the constituent K-mers for this motif were below the quantile cutoff for clustering, and thus, were unable to become a directly observed feature of the model. The use of rare codons, and their tendency to cluster, has been observed across both eukaryotes and prokaryotes
       <sup>
        <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>
       </sup>. This NVTVTH amino acid motif is also, like the SLQCR sequence described above, an experimentally validated N-linked glycosylation site on the HA gene in H1N1
       <sup>
        <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>
       </sup>. Rare-codon clusters in association with N-linked glycosylation sites in human pathogens have previously observed in HIV-1 envelope glycoprotein gp120, where the conservation of the rare-codon RNA sequence conferred increased glycosylation efficiency compared to gp120 mutants
       <sup>
        <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>
       </sup>. Codon optimization efforts for lentivirus envelope protein have also induced non-functional proteins, hypothesized to be related to glycosylation disruption
       <sup>
        <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>
       </sup>. The fidelity of conformational change in mammalian isolates to these rare codon identities is extremely high. The oscillation between these conformational states is suggestive of another dimension of interpretation that these logistic regression models offer, outside of the examination of the genomic motifs themselves.
      </p>
     </div>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Other Dimensions of Interpretation</h3>
     <p id="p-31">The fragility of the phenotype for the Influenza A model resulted in a model with higher complexity than the other RNA viruses studied. However, this provides another avenue for model analysis. Logistic regression classifiers offer not only an output label, but also a probability assignment to the corresponding label. Thus, additional information can be encoded in this output. 
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> presents a graphical representation of the distribution of these class probabilities for the training sets for the segments described.
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Influenza A training set class probability histograms. The class probability distributions for the Influenza A segment 1 and segment 4 models discussed in this section. The frequency is presented in log scale so it can be observed that the vast majority of class predictions belong to the highest and lowest probability bins. We explore the possibility that instances with class probabilities in the middle of the distribution are in transition between host-isolate states as a results of recent zoonosis events." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1790953588" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Influenza A training set class probability histograms.&lt;;/span&gt;; The class probability distributions for the Influenza A segment 1 and segment 4 models discussed in this section. The frequency is presented in log scale so it can be observed that the vast majority of class predictions belong to the highest and lowest probability bins. We explore the possibility that instances with class probabilities in the middle of the distribution are in transition between host-isolate states as a results of recent zoonosis events.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F5.medium.gif" width="387" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F5.medium.gif" width="387" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 5.</span>
       <span class="caption-title">Influenza A training set class probability histograms.</span>
       <p id="p-32" class="first-child">The class probability distributions for the Influenza A segment 1 and segment 4 models discussed in this section. The frequency is presented in log scale so it can be observed that the vast majority of class predictions belong to the highest and lowest probability bins. We explore the possibility that instances with class probabilities in the middle of the distribution are in transition between host-isolate states as a results of recent zoonosis events.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-33">The highest coefficient predictor in the model for Influenza A segment 1, which codes for the PB2 polymerase gene, is a motif which represents a mammalian amino acid substitution experimentally observed in a mouse model
      <sup>
       <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>
      </sup>. This mammalian adaptation was identified as relevant to the temperature sensitivity of the polymerase in H5N1. The reversion of the avian conformation containing the glutamic acid residue, to the mammalian conformation containing lysine, was observed to be approximately six days. By chance, some subset of viral isolates could have been sampled during this window while “in transit” between host-signature genotypes. Thus, the misclassified examples from the training set invite further scrutiny. Of the 79892 instances in the training set, 274 were misclassified, and approximately 10 of these misclassifications were discovered to be mislabeling due to erroneous formatting of the WHO nomenclature. The remainder are examples where the model has, in some cases with a high probability, assigned a classification that disagreed with the class labeling.
     </p>
     <p id="p-34">One particularly interesting example of this can be seen in a pair of swine isolates (KM289087.1, KM289089.1) misclassified by the model as human, which were attributed to human-to-pig H1N1 transmission events in backyard farms in Peru
      <sup>
       <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>
      </sup>. A third isolate from this study (KM289088.1) was classified correctly but also expressed some ambiguity in the class designation from the perspective of the class probability. Fortunately, this study included in the publication the sampling dates for the pigs at a central processing facility, allowing the Vorpal algorithm to detect a trend in the data as demonstrated in 
      <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 5</a>. Transition from the human conformation of the virus (from the perspective of the model) to the non-human conformation follows the progression of the calendar date. The original authors had previously speculated about the simultaneous exposure of two of these swine isolates based on phylogeny.
     </p>
     <div id="T6" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1179418/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1179418/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1179418" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 5.</span>
       <span class="caption-title">Notable H1N1 Swine Isolates.</span>
       <p id="p-35" class="first-child">Transition from the human conformation of the H1N1 virus to the swine conformation from samples in Peru.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-36">A second case where a training sample was misclassified as a human isolate from the model was an Influenza A H1N1 instance (KF277197.1) isolated from a giant panda at the Conservation and Research Center for the Giant Panda in Ya’an City, China. There are several plausible hypotheses that could explain the consistent misclassification of this isolate from the model, including the most obvious, that the Giant Panda conformation of the virus is only represented by this distinct example, and thus, the model could not learn the features that may distinguish it from a human-isolated example. However, this assembly was accompanied by a publication which points to a different explanation for model confusion. The paper’s authors, through phylogenetic analysis, suggest that this case was an example of pandemic H1N1 transmitted directly from humans to the pandas
      <sup>
       <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>
      </sup>. Similar examples are abundant. A pair of misclassified swine isolates were identified in a 2009 publication studying triple-reassortment swine Influenza A infections in people from 2005 – 2009
      <sup>
       <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>
      </sup>. Both of these human infections were linked to direct contact with sick pigs presented at a county fair within a 3 to 4 day window of sampling. The findings regarding these examples are contained in 
      <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Tables 6A</a> and 
      <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">6B</a>. Model prediction probabilities that disagree with the known host source may be useful as a way to infer spill-over events.
     </p>
     <div id="T7" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1179411/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1179411/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1179411" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 6A.</span>
       <span class="caption-title">Midwest, US Influenza A segment four isolates.</span>
       <p id="p-37" class="first-child">The local neighbors of A/Ohio/01/2007 (H1N1) and A/Ohio/02/ 2007 (H1N1) identified in Shinde et. al. 2009
        <sup>
         <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>
        </sup> as swine influenza virus infections of human hosts at a county fair in 2007. The estimated incubation period for these misclassified training examples was 3-4 days.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T8" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1179400/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1179400/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1179400" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 6B.</span>
       <span class="caption-title">Sichuan, CN Influenza A segment four isolates.</span>
       <p id="p-38" class="first-child">The Giant Panda isolate misclassified in the training set and its nearest neighbor in the embedding space.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-39">If the misclassified Giant Panda isolate is observed in context in a two-dimensional embedded space, where the motif feature vectors are used as the input space, then its nearest-neighbor in the lower dimensional representation is a human H1N1 isolate, also from Sichuan, in 2009. In the case of the misclassified swine isolates, they are surrounded in the local neighborhood by H1N1 Swine instances from Ohio and Iowa in 2007 and 2008. This proximity in embedded spaces offers another angle for interpretation, especially in regards to identifying possible spill-over or re-assortment events and is depicted in 
      <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. Neighbors in the local embedding are often temporally and geographically proximal, in addition to sharing host isolate membership. Comingling of class labels in the embedded space potentially offers the opportunity for identification of zoonosis events.
     </p>
     <div id="F6" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Embedding of Influenza A segment four train set data. Two-dimensional t-SNE embedding of the feature vector for the Influenza A segment four (HA gene). Many clusters can be observed to segregate with respect to the human isolate class label (blue) vs. non-human isolates (red). Close inspection of region (A) identifies linkage of H1N1 isolates from swine and humans likely infected from the same swine population, with the swine-conformation shifting towards a human-conformation. Region A corresponds with data in Table 6A. Close inspection of region (B) identifies linkage of human-conformation H1N1 isolates from humans in Sichuan, China with those from pandas believed to have been infected by direct human contact at a conservation center in the same locale. Region B corresponds with data in Table 6B. Note: Axes in t-SNE plots have no intrinsic meaning except to represent pair-wise distances between points." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1790953588" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Embedding of Influenza A segment four train set data.&lt;;/span&gt;; Two-dimensional t-SNE embedding of the feature vector for the Influenza A segment four (HA gene). Many clusters can be observed to segregate with respect to the human isolate class label (blue) vs. non-human isolates (red). Close inspection of region (A) identifies linkage of H1N1 isolates from swine and humans likely infected from the same swine population, with the swine-conformation shifting towards a human-conformation. Region A corresponds with data in Table 6A. Close inspection of region (B) identifies linkage of human-conformation H1N1 isolates from humans in Sichuan, China with those from pandas believed to have been infected by direct human contact at a conservation center in the same locale. Region B corresponds with data in Table 6B. Note: Axes in t-SNE plots have no intrinsic meaning except to represent pair-wise distances between points.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F6.medium.gif" width="440" height="193"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F6.medium.gif" width="440" height="193"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 6.</span>
       <span class="caption-title">Embedding of Influenza A segment four train set data.</span>
       <p id="p-40" class="first-child">Two-dimensional t-SNE embedding of the feature vector for the Influenza A segment four (HA gene). Many clusters can be observed to segregate with respect to the human isolate class label (blue) vs. non-human isolates (red). Close inspection of region (A) identifies linkage of H1N1 isolates from swine and humans likely infected from the same swine population, with the swine-conformation shifting towards a human-conformation. Region A corresponds with data in 
        <a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table 6A</a>. Close inspection of region (B) identifies linkage of human-conformation H1N1 isolates from humans in Sichuan, China with those from pandas believed to have been infected by direct human contact at a conservation center in the same locale. Region B corresponds with data in 
        <a id="xref-table-wrap-8-2" class="xref-table" href="#T8">Table 6B</a>. Note: Axes in t-SNE plots have no intrinsic meaning except to represent pair-wise distances between points.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-41">Inspection of the embedded space makes it possible to identify candidate events, even if the model has not made a classification error. Examples of these are summarized in 
      <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> and 
      <a id="xref-table-wrap-9-1" class="xref-table" href="#T9">Table 7A</a> and 
      <a id="xref-table-wrap-10-1" class="xref-table" href="#T10">7B</a> where Influenza A segment 1 (PB2) sequences are embedded into a two-dimensional field. Further experimentation may also help develop models that incorporate a velocity to the conformational changes of host-predictor motifs and estimate temporal distance from a prospective zoonotic event, in a segment-specific manner.
     </p>
     <div id="T9" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1179399/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1179399/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1179399" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 7A.</span>
       <span class="caption-title">New York Influenza A segment one isolates.</span>
       <p id="p-42" class="first-child">A human isolate collocated amongst avian isolates in a cluster of H7N2 subtype Influenza examples from New York in 2003. The nearest-neighbor for the human isolate (GU186492.2) is an environmental sample from a live-bird market. The model also encodes the ambiguity of the classification in class probability for the human-isolate phenotype.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T10" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1179407/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1179407/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1179407" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 7B.</span>
       <span class="caption-title">Saskatchewan, CA Influenza A segment one isolates.</span>
       <p id="p-43" class="first-child">A Saskatchewan specific sub-cluster that belongs to a larger cluster of PB2 genes isolated from Swine and co-assorted with H1N1, and H3N2 subtypes in circulation in North America. The human isolates represented in this group belong to pig farm workers who all contracted swine influenza virus, it is presumed, through their place of work
        <sup>
         <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>
        </sup>. Interestingly, this distinctive genotype of PB2 seems to be preserved across long time frames (2009 to 2015) and is free to re-assort with different Influenza A subtypes (H1N1 and H3N2). In addition, these same isolates had corresponding HA gene sequences published, but the ambiguities seen in the class probabilities for PB2 segment were not observed in the HA gene (i.e. they were all 99% probability human-isolate)
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F7" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Embedding of Influenza A segment one train set data. Two-dimensional t-SNE embedding of the feature vector for the Influenza A segment one (PB2 gene). Many clusters can be observed to segregate with respect to the human isolate class label (blue) vs. non-human isolates (red). Close inspection of region (A) identifies linkage of H7N2 isolates from birds and a human likely infected from the same avian population at a live bird market within the same locality. Region A corresponds with data in Table 7A. Close inspection of region (B) identifies co-mingling of human-isolated and swine-isolated H1N1 and H3N2 strains from Saskatchewan, Canada. Region B corresponds with data in Table 7B. Note: Axes in t-SNE plots have no intrinsic meaning except to represent pair-wise distances between points." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1790953588" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Embedding of Influenza A segment one train set data.&lt;;/span&gt;; Two-dimensional t-SNE embedding of the feature vector for the Influenza A segment one (PB2 gene). Many clusters can be observed to segregate with respect to the human isolate class label (blue) vs. non-human isolates (red). Close inspection of region (A) identifies linkage of H7N2 isolates from birds and a human likely infected from the same avian population at a live bird market within the same locality. Region A corresponds with data in Table 7A. Close inspection of region (B) identifies co-mingling of human-isolated and swine-isolated H1N1 and H3N2 strains from Saskatchewan, Canada. Region B corresponds with data in Table 7B. Note: Axes in t-SNE plots have no intrinsic meaning except to represent pair-wise distances between points.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F7.medium.gif" width="440" height="195"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F7.medium.gif" width="440" height="195"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 7.</span>
       <span class="caption-title">Embedding of Influenza A segment one train set data.</span>
       <p id="p-44" class="first-child">Two-dimensional t-SNE embedding of the feature vector for the Influenza A segment one (PB2 gene). Many clusters can be observed to segregate with respect to the human isolate class label (blue) vs. non-human isolates (red). Close inspection of region (A) identifies linkage of H7N2 isolates from birds and a human likely infected from the same avian population at a live bird market within the same locality. Region A corresponds with data in 
        <a id="xref-table-wrap-9-2" class="xref-table" href="#T9">Table 7A</a>. Close inspection of region (B) identifies co-mingling of human-isolated and swine-isolated H1N1 and H3N2 strains from Saskatchewan, Canada. Region B corresponds with data in 
        <a id="xref-table-wrap-10-2" class="xref-table" href="#T10">Table 7B</a>. Note: Axes in t-SNE plots have no intrinsic meaning except to represent pair-wise distances between points.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">Discussion</h2>
    <p id="p-45">The observations presented in this paper represent a fraction of the information potentially contained in the developed models using the Vorpal feature extraction algorithm. Efforts to build robust metanalysis tools based on the model outputs is a focus of further development. While we also think the discoveries mentioned herein make a compelling argument for the power of these models in automatically generating hypotheses to direct experiments, we acknowledge the inherent difficulty in leveraging these models for predictive analytics, where, due to the role of evolution, extrapolation to data unsupported at training time is inevitable.</p>
    <p id="p-46">To emphasize the hazard of using these models to predict on new data, the emerging Wuhan pneumonia coronavirus and Bombali ebolavirus provide illustrative examples. The Wuhan COV (MN908947.1) and Bombali ebolavirus (NC_039245.1) assemblies were predicted on using the models denoted in 
     <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>. The model classified Wuhan COV as 0.004% probable for the Human pathogen phenotype and Bombali ebolavirus as 90.2% probable for the Human-hemorrhagic-fever phenotype. Both of these classifications, especially the Wuhan COV designation, are out-of-step with what is known, or in the case of Bombali, suspected, about these viruses. However, it is possible to imagine these functional profiles leading to a more deterministic understanding of function with which to build a predictive frame work. Nonetheless, improvements in data structure and metadata association may yield better abilities to estimate the probability of future events. Certain observations seen in the models thus far may themselves be predictive of the respective phenotype before it is observed, rather than an effect of it already having occurred. The primary example of this is the predictor identified in the Orthocoronavirinae model. As described in the Methods section, certain assumptions were built into labeling for the human-pathogen phenotype that incorporated theories about the zoonotic provenance of SARS and Middle East Respiratory Syndrome-related (MERS) from civets and camels respectively. Observing human-pathogen predictors occurring in SARS and MERS viruses from non-human hosts could suggest the ability to predict the potential of a virus as a human pathogen in advance of a spill-over event. This is observed in the data. The AKRATGKTGTTAATMAA motif appears in all five of the civet SARS assemblies in the dataset. In the case of the camel isolates, the motif KGATGTTGTTARWCAAY, which is also related to the one mentioned above, is another high coefficient predictor for human pathogenicity and it appears in 231 of the 232 Camel-MERS instances in the training set. This motif also appears in the emerging 2019-nCoV as noted in 
     <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>.
    </p>
    <p id="p-47">As for the obstacles for predictive efforts, there are many opportunities for improvements in the collection and annotation of viral genomic data. In 
     <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>, a slight drift can be observed in the Influenza A model accuracies between the training and test sets. Because the test set represented the most recently isolated viruses, it is attractive to explain this drift as real, i.e. due to evolution. However, there are other factors to control for since the underlying process generating the data has changed over the time period of data collection. The use of cell lines and PCR based amplification of signal for genome assembly, as well as the use of different sequencing technologies suggest other variables to account for. To demonstrated this, a search through the Genbank records for the Influenza A training set members for “passage” annotations revealed that 42.3% of the instances in that set contained such annotations for cell passage. In contrast, the Influenza A test set members, which represents more recently generated data, only contained “passage” annotations in 29.0% of those records.
    </p>
    <p id="p-48">Lastly, we hope that this analysis demonstrates that the utility of a Global Virome Project is not ambiguous. Controversy about the value of such a project has been described
     <sup>
      <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>
     </sup> and this thinking has been reflected in policymakers’ decision to end funding to USAID Predict. If recent estimates of mammalian viral diversity hold true
     <sup>
      <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>
     </sup>, then marginal increases in monitoring infrastructure combined with new and developing analysis methods, such as Vorpal, might finally deliver the long sought preemptive strategies for emergent diseases, and enable us to more effectively battle those from which we are already suffering.
    </p>
   </div>
   <div class="section" id="sec-9">
    <h2 class="">Conclusion</h2>
    <p id="p-49">The use of this algorithm for genotype-to-phenotype models is just one of the potential applications. Automated molecular assay design and degenerate-motif based phylogenetics are examples of the downstream uses already being investigated. The ability to make use of the latent data that is accumulating in databases, as well as novel surveillance data, is made more tangible with this algorithm. Well-curated and richly annotated metadata promises to allow machine learning and other data science techniques to unleash a torrent of discovery in genomics at large. The mantra we are positing for the infectious and emergent diseases surveillance community is “
     <em>More data, Better data, Metadata</em>.” The techniques to unlock the potential of data-driven genomic science are gathering momentum.
    </p>
   </div>
   <div class="section" id="sec-10">
    <h2 class="">Methods</h2>
    <div class="section" id="sec-11">
     <h3>Algorithm</h3>
     <p id="p-50">The Vorpal algorithm for feature extraction was developed using the libraries and versions delineated in the requirements.txt document located on the Github. The Vorpal feature extraction algorithm has 3 steps, each corresponding to a script that becomes the Vorpal workflow.</p>
     <ol class="list-ord " id="list-2">
      <li id="list-item-4">
       <p>kmercountouter_sparse.py </p>
       <ol class="list-alphalower " id="list-3">
        <li id="list-item-5">
         <p>Input: </p>
         <ol class="list-romanlower " id="list-4">
          <li id="list-item-6">
           <p id="p-53">a reference genome in FASTA format</p>
          </li>
          <li id="list-item-7">
           <p id="p-54">a folder containing complete assemblies for the viral group of interest</p>
          </li>
          <li id="list-item-8">
           <p id="p-55">a parameter for K-mer size</p>
          </li>
          <li id="list-item-9">
           <p id="p-56">a percent variance argument for filtering out assemblies that are divergent from the reference genome in terms of length</p>
          </li>
         </ol>
        </li>
        <li id="list-item-10">
         <p>Output: </p>
         <ol class="list-romanlower " id="list-5">
          <li id="list-item-11">
           <p id="p-58">a pickled sparse dataframe object containing K-mer counts across every input instance</p>
          </li>
         </ol>
        </li>
       </ol>
      </li>
      <li id="list-item-12">
       <p>hammingclusters_fast.py </p>
       <ol class="list-alphalower " id="list-6">
        <li id="list-item-13">
         <p>Input: </p>
         <ol class="list-romanlower " id="list-7">
          <li id="list-item-14">
           <p id="p-61">A pickled sparse dataframe produced by kmercountouter_sparse.py</p>
          </li>
          <li id="list-item-15">
           <p id="p-62">The average number of allowed degenerate bases for clustering. This is converted to the equivalent hamming distance cutoff by 
            <span class="disp-formula" id="disp-formula-1">
             <span class="highwire-responsive-lazyload">
              <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/02/2020.02.28.969782/embed/graphic-18.gif"/>
              <noscript>
               <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/02/2020.02.28.969782/embed/graphic-18.gif"/>
              </noscript>
             </span>
            </span>
           </p>
          </li>
          <li id="list-item-16">
           <p id="p-63">The quantile cutoff for high frequency K-mer filtering</p>
          </li>
          <li id="list-item-17">
           <p id="p-64">The number of chunks to split the count data into when calculating K-mer frequency. This allows for processing of the K-mer counts table in a memory constrained environment (optional)</p>
          </li>
          <li id="list-item-18">
           <p id="p-65">A temp folder directory to memory map the distance matrix to, again to allow for more memory overhead to be available at the linkage step. (optional)</p>
          </li>
          <li id="list-item-19">
           <p id="p-66">A memory allocation argument for the development of the distance matrix in chunks. This can be used in conjunction with memory mapping or without it. Uses the sci-kit learn 
            <kbd>pairwise_distances_chunked</kbd> function instead of the scipy 
            <kbd>pdist</kbd> function (optional)
           </p>
          </li>
         </ol>
        </li>
        <li id="list-item-20">
         <p>Output: </p>
         <ol class="list-romanlower " id="list-8">
          <li id="list-item-21">
           <p id="p-68">A multi-FASTA file with degenerate motifs of K length.</p>
          </li>
         </ol>
        </li>
       </ol>
      </li>
      <li id="list-item-22">
       <p>referencemapping_mp.py </p>
       <ol class="list-alphalower " id="list-9">
        <li id="list-item-23">
         <p>Input: </p>
         <ol class="list-romanlower " id="list-10">
          <li id="list-item-24">
           <p id="p-71">A multi-FASTA with all of the assemblies to map to</p>
          </li>
          <li id="list-item-25">
           <p id="p-72">The multi-FASTA file of degenerate motifs produced by hammingclusters_fast.py</p>
          </li>
          <li id="list-item-26">
           <p id="p-73">A threads argument for parallel processing</p>
          </li>
         </ol>
        </li>
        <li id="list-item-27">
         <p>Output: </p>
         <ol class="list-romanlower " id="list-11">
          <li id="list-item-28">
           <p id="p-75">A series of BED files with the following column specifications: </p>
           <div id="T11" class="table pos-float">
            <div class="table-inline table-callout-links">
             <div class="callout">
              <span>View this table:</span>
              <ul class="callout-links">
               <li class="view-inline first">
                <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1179402/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
               </li>
               <li class="view-popup">
                <a href="/highwire/markup/1179402/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
               </li>
               <li class="download-ppt last">
                <a href="/highwire/powerpoint/1179402" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
               </li>
              </ul>
             </div>
            </div>
           </div>
          </li>
         </ol>
        </li>
       </ol>
      </li>
     </ol>
     <p id="p-76">Wrapper scripts for reproducing the models with the parameters described below are also provided as binary_vorpal_model.py and binary_vorpal_model_ElasticNet.py.</p>
    </div>
    <div class="section" id="sec-12">
     <h3>Model Parameters</h3>
     <p id="p-77">All models were built around binary output variables using a logistic regression classifier. The models were regularized using either 
      <em>ℓ</em>1 or ElasticNet methods, using the liblinear
      <sup>
       <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>
      </sup> solver or Stochastic Gradient Descent estimators
      <sup>
       <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>,
       <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>,
       <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>
      </sup> in scikit-learn, respectively. The parameters evaluated for optimization for both approaches were kept uniform for every model fit, with the parameter values searched over listed in the 
      <a id="xref-table-wrap-12-1" class="xref-table" href="#T12">Table 8</a>.
     </p>
     <div id="T12" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1179413/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1179413/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1179413" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 8.</span>
       <span class="caption-title">Grid Search Parameters.</span>
       <p id="p-78" class="first-child">Optimization search parameters for regularization methods. Lambda in the LASSO method corresponds to the constraint on the 
        <strong>
         <em>ℓ</em>1
        </strong> norm of the feature vector while alpha in ElasticNet corresponds to the constraint on the vector magnitude as well as the learning rate for Stochastic Gradient Descent.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-79">All of the input parameters for feature extraction and the rationale behind the use and tuning of each parameter, and their relation to the corresponding model discussed above is provided here.</p>
    </div>
    <div class="section" id="sec-13">
     <h3>Feature Extraction Parameters</h3>
     <p>The first parameter, K length, can be a variable input, but in the development of these methods was fixed at 17. The decision to set the k-length at 17 had many facets. The first is that the feature space should be large enough, that the introduction of degenerate positions does not cause a complete collapse of feature structures. Evaluation of optimal K length for specific tasks has been performed in many contexts. For phylogenetic representations of viruses, an optimal range of 9 to 13 has been proposed
      <sup>
       <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>
      </sup>, for the optimal uniqueness ratio in plant genomes a K length of 20 has been identified
      <sup>
       <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>
      </sup>, and in phenetic analysis of bacteria a K length of 31 has been demonstrated to yield the best balance between sensitivity and specificity in intra- and interspecies distance analysis
      <sup>
       <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>
      </sup>. However, defining a subspace that lends itself to genotype-to-phenotype model interpretability should have the following desiderata: 
     </p>
     <ol class="list-ord " id="list-12">
      <li id="list-item-29">
       <p id="p-81">K-size motifs should map to mostly unique genomic loci. In other words, sparsity in the weights vector is influenced by sparsity in the input vector.</p>
      </li>
      <li id="list-item-30">
       <p id="p-82">K-size should be small enough, that the feature space inflation is not catastrophic to memory constraints.</p>
      </li>
     </ol>
     <div id="F8" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F8.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Post-quantile filtering K-mer distribution. K-mer rank-frequency distribution plots for Influenza segments 1 and 4, Ebolavirus, and Orthocoronavirinae at the quantile used in the models discussed. Frequency is calculated as number of instances the ranked K-mer appears in." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1790953588" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Post-quantile filtering K-mer distribution.&lt;;/span&gt;; K-mer rank-frequency distribution plots for Influenza segments 1 and 4, Ebolavirus, and Orthocoronavirinae at the quantile used in the models discussed. Frequency is calculated as number of instances the ranked K-mer appears in.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F8.medium.gif" width="440" height="236"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 8." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F8.medium.gif" width="440" height="236"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 8.</span>
       <span class="caption-title">Post-quantile filtering K-mer distribution.</span>
       <p id="p-83" class="first-child">K-mer rank-frequency distribution plots for Influenza segments 1 and 4, Ebolavirus, and Orthocoronavirinae at the quantile used in the models discussed. Frequency is calculated as number of instances the ranked K-mer appears in.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-84">This method implements canonical K-mer counting, where the reverse complement of a K-mer is counted as the same time as the forward oriented K-mer, because of uncertainty about strand orientation in the input data. It was known that there were example assemblies in GenBank for Lassa virus where different instances had inconsistent strand reporting. This assumption seems to be unwarranted for the viruses selected for this study and could be removed for future implementations. It should be pointed out that, while maintaining this assumption seems wasteful from a memory overhead perspective, certain features could only be revealed through this canonical approach, such as hair-pin complements in RNA secondary structures, where the resolution of this structural motif is only possible when compared to the K-mer produced by the complementary region. Other dimensionality reduction techniques, namely high-frequency K-mer filtering, allowing the feature extraction to remain tractable, given the computing resources available for this study. The effect of this canonical approach, and the information it potentially encodes in the feature space, is demonstrated in 
      <a id="xref-table-wrap-13-1" class="xref-table" href="#T13">Table 9A</a> and 
      <a id="xref-table-wrap-14-1" class="xref-table" href="#T14">9B</a>.
     </p>
     <div id="T13" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1179408/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1179408/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1179408" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 9A.</span>
       <span class="caption-title">Orthocoronavirinae Feature Extraction Summary (0.95 quantile).</span>
       <p id="p-85" class="first-child">Summary statistics for feature extraction for Orthocoronavirinae from 0.0 to 5.0 degeneracy cutoff for clustering. Feature space tracks the dimensionality reduction introduced by degeneracy to motifs that map back to training set. Initially, since no odd-length K-mer can be a reverse complement of itself, canonical K-mers counted compared to those mapped should be half. As degeneracy is introduced, the Motif/Feature ratio is expected to converge to 1.0, which describes a single motif of all “N” symbols. This ratio tracks the amount of previously distinct motifs now represented as a single feature. The final column, shows the phenomena of motifs that are now reverse complements of themselves as a result of degeneracy, contributing to the inflation of the Motif/Feature ratio. Of note in the Orthocoronavirinae features, is while dimensionality reduction continues with the allowance of more degeneracy, the fraction of those resulting features that have corresponding reverse complements in the feature set does not increase past 4.0 degeneracy.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T14" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1179405/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1179405/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1179405" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 9B.</span>
       <span class="caption-title">Ebolavirus Feature Extraction Summary (0.0 quantile).</span>
       <p id="p-86" class="first-child">Summary statistics for feature extraction for Ebolavirus from 0.0 to 5.0 degeneracy cutoff for clustering. A larger fraction of the features at the highest degeneracy allowance produced contain corresponding reverse complement motifs in the feature set in the Ebolavirus data than in the Orthocoronavirinae data. This could be attributable to the high frequency K-mer quantile cutoff utilized in the Coronavirus group, or it could allude to generally higher fraction of the Ebolavirus genomes having self-complementation than Coronavirus genomes.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F9" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F9.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Line plot for three of the selected columns from Table 10A. The plateau reached at 4.0 degeneracy for the ratio of reverse complement motifs is clearly evident." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1790953588" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Line plot for three of the selected columns from Table 10A.&lt;;/span&gt;; The plateau reached at 4.0 degeneracy for the ratio of reverse complement motifs is clearly evident.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 9A." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F9.medium.gif" width="440" height="244"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 9A." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F9.medium.gif" width="440" height="244"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9A." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 9A.</span>
       <span class="caption-title">Line plot for three of the selected columns from Table 10A.</span>
       <p id="p-87" class="first-child">The plateau reached at 4.0 degeneracy for the ratio of reverse complement motifs is clearly evident.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F10" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F10.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Line plot for three of the selected columns from Table 9b. The relationship between mean number of degenerate positions in the motifs and the mean number of degenerate bases represented is very similar between Ebolavirus and Coronavirus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1790953588" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Line plot for three of the selected columns from Table 9b.&lt;;/span&gt;; The relationship between mean number of degenerate positions in the motifs and the mean number of degenerate bases represented is very similar between Ebolavirus and Coronavirus.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 9B." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F10.medium.gif" width="440" height="236"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 9B." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F10.medium.gif" width="440" height="236"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9B." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.28.969782/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 9B.</span>
       <span class="caption-title">Line plot for three of the selected columns from Table 9b.</span>
       <p id="p-88" class="first-child">The relationship between mean number of degenerate positions in the motifs and the mean number of degenerate bases represented is very similar between Ebolavirus and Coronavirus.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-89">Applying a filter to the K-mers that are allowed to proceed to the clustering step has two purposes. The first is to denoise the data by removing low abundance features that could be the result of error or other transient sources of variance. The removal of these K-mers is achieved through a parameter specified at the clustering stage, the K-mer quantile. Singletons, or K-mer that are unique to a single instance, are always removed no matter the quantile specified. It was discovered that allowing the singletons to form motifs through agglomerative clustering introduced instability into the model parameter estimation (data not shown). Contribution to frequency is determined not by cumulative sum of count across every instance but rather frequency of presence across the sample instances. This is identical to the way the “TopN” score is calculated for K-mers in PriMux primer design software
      <sup>
       <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>
      </sup>. Using a K-mer frequency filter selects for a conserved variance signal. This is a reasonable heuristic to introduce, especially for predictive models, where these high-frequency K-mer derived motifs are the those with the presumed highest probability of appearing in a novel example of a related organism in nature. The second function of this feature extraction parameter, made reference to above, is as a dimensionality reduction technique to make K-mer clustering more tractable in the current algorithm implementation, given limitations in computational resources. Memory constraints during the tree building step represents the primary bottleneck with the scipy implementation of the nearest-neighbors chain algorithm for average linkage using 
      <span class="inline-formula" id="inline-formula-1">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/02/2020.02.28.969782/embed/inline-graphic-1.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/02/2020.02.28.969782/embed/inline-graphic-1.gif"/>
        </noscript>
       </span>
      </span> memory
      <sup>
       <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>,
       <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>
      </sup>. The user specifies an average number of degenerate bases to apply when flat clustering. This number is then divided by the K length specified to estimate the corresponding hamming distance to provide as the max distance for flat clustering. After flat clusters are grouped into alignments and a degenerate motif of the alignment is generated by collapsing each position in the K length alignment into the IUPAC symbol matching the bases seen at that position. This clustering of K-mers, and subsequent representation as degenerate motifs, is another layer of dimensionality reduction similar to lemmatization of words in a Natural Language Processing (NLP) feature extraction technique
      <sup>
       <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>
      </sup>. Much of this approach could be described as modifications of equivalent NLP feature extraction and modeling strategies. It should be noted however, that data preparation techniques such as term frequency-inverse document frequency (tf-idv), were considered inappropriate to apply in this circumstance for multiple reasons. First, “document” length was invariant in the sense that complete assemblies were the only instances allowed in the training data, and differences in genome sizes within the taxonomies considered were considered irrelevant. Second, document terms, in this case K-mer motifs, that follow a frequency pattern similar to the word “the” in the English language are not present. Additionally, for this reason, the data was not normalized, however to improve convergence speed this could be a future improvement.
     </p>
    </div>
    <div class="section" id="sec-14">
     <h3>Phenotype Labeling</h3>
     <p id="p-90">Phenotype labels for the different organisms modeled were applied using a variety of strategies with some specific assumptions introduced for labeling of the Orthocoronavirinae group. In the cases of Ebolavirus and Coronavirus, taxonomy was used as a guide for phenotype labels, where knowledge about the phenotype of interest was usually easily delineated along taxonomic boundaries. For Influenza A, the World Health Organization nomenclature for Influenza strain identification, which is encoded in the FASTA header, was parsed for labeling of human isolate
      <sup>
       <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>
      </sup>. For those FASTA headers which contained malformed strain identifiers, an ambiguous labeling was applied and removed from the training set.
     </p>
     <p id="p-91">The following explicit assumptions were applied when labeling viral instances for the human pathogen phenotype in the Orthocoronavirinae model. First, since most transmissions of Middle East Respiratory Syndrome-related (MERS) betacoronavirus to humans have been zoonic events traced to dromedary camels, all camel isolates for MERS coronavirus were labeled in the positive class corresponding to human pathogen. Likewise, in the cases for Severe Acute Respiratory Syndrome-related betacoronavirus, since the initial outbreak had been theorized to begin from a zoonic event from infected palm civets at a market in Guangdong, China, along with a specific civet spill over event documented in a waitress and a customer in a restaurant in Guangzhou
      <sup>
       <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>
      </sup>, all civet SARS-like isolates were also labeled as belonging to the positive class. However, since there is no clear evidence of bat-Coronavirus-to-human transmissions, the assumption was built-in that bat isolates of both MERS-like and SARS-like betacoronaviruses were not part of the human pathogen class. In the instance of MERS-like bat isolates, examples have been found across wide geographic ranges, such as South Africa, while human cases appear to restricted to areas where Saudi Arabian dromedary camels are present
      <sup>
       <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>
      </sup> or hospital acquired infections. The same is true of SARS-like bat isolates discovered in caves in China, where assemblies from these isolates show varying similarities to the strain from the 2003-2004 outbreak but not the sum of them
      <sup>
       <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>
      </sup>.
     </p>
     <p id="p-92">Training sets were developed from the un-clustered Reference Viral Database
      <sup>
       <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>
      </sup> (RVDB) version 14 published October 1
      <sup>st</sup>, 2018. Accessions for designated taxonomic groups were derived from National Center for Biotechnology Virus
      <sup>
       <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>
      </sup> and then used to extract the associated assemblies from RVDB. Test sets were developed from RVDB version 15 published February 6
      <sup>th</sup>, 2019 using the references for the modeled organisms that had been added between version releases.
     </p>
    </div>
    <div class="section" id="sec-15">
     <h3>Embedding Visualization</h3>
     <p id="p-93">The same feature vectors used to produce the logistic regression models were topic modeled similarly to Latent Semantic Analysis
      <sup>
       <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>
      </sup> (LSA) using a truncated Singular Value Decomposition (SVD) to a 500 component subspace, which was then subjected to a t-distributed Stochastic Neighbor Embedding
      <sup>
       <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>,
       <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>
      </sup> (t-SNE) to a two-dimensional space to observe the local structure of the Influenza A viral assemblies. Both of these methods were employed using the associated classes in Scikit-learn. Visualization and exploration of the embedded space was facilitated by Plotly
      <sup>
       <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>
      </sup>.
     </p>
    </div>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-dataset fn-group-external-links" id="fn-1">
      <p id="p-3">
       <a href="https://github.com/mriglobal/vorpal">https://github.com/mriglobal/vorpal</a>
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zielezinski</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vinga</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Almeida</span>, 
          <span class="cit-name-given-names">J.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Karlowski</span>, 
          <span class="cit-name-given-names">W. M.</span>
         </span>
         <span class="cit-article-title">Alignment-free sequence comparison: benefits, applications, and tools</span>. 
         <abbr class="cit-jnl-abbrev">Genome Biology</abbr>
         <span class="cit-vol">18</span>, (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ondov</span>, 
          <span class="cit-name-given-names">B. D.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Mash: fast genome and metagenome distance estimation using MinHash</span>. 
         <abbr class="cit-jnl-abbrev">Genome Biology</abbr>
         <span class="cit-vol">17</span>, (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Koslicki</span>, 
          <span class="cit-name-given-names">D.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Falush</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-article-title">MetaPalette: a k-mer Painting Approach for Metagenomic Taxonomic Profiling and Quantification of Novel Strain Variation</span>. 
         <abbr class="cit-jnl-abbrev">mSystems</abbr>
         <span class="cit-vol">1</span>, (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Déraspe</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Phenetic Comparison of Prokaryotic Genomes Using k-mers</span>. 
         <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>
         <span class="cit-vol">34</span>, 
         <span class="cit-fpage">2716</span>–
         <span class="cit-lpage">2729</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.volume%253D34%26rft.spage%253D2716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.5" data-doi="10.1093/bioinformatics/btt020">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rizk</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lavenier</span>, 
          <span class="cit-name-given-names">D.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Chikhi</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-article-title">DSK: k-mer counting with very low memory usage</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">29</span>, 
         <span class="cit-fpage">652</span>–
         <span class="cit-lpage">653</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt020%26rft_id%253Dinfo%253Apmid%252F23325618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt020&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23325618&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.28.969782.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000315623000017&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.6" data-doi="10.1128/AEM.00062-07">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Garrity</span>, 
          <span class="cit-name-given-names">G. M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tiedje</span>, 
          <span class="cit-name-given-names">J. M.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Cole</span>, 
          <span class="cit-name-given-names">J. R.</span>
         </span>
         <span class="cit-article-title">Naive Bayesian Classifier for Rapid Assignment of rRNA Sequences into the New Bacterial Taxonomy</span>. 
         <abbr class="cit-jnl-abbrev">Applied and Environmental Microbiology</abbr>
         <span class="cit-vol">73</span>, 
         <span class="cit-fpage">5261</span>–
         <span class="cit-lpage">5267</span> (
         <span class="cit-pub-date">2007</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.00062-07%26rft_id%253Dinfo%253Apmid%252F17586664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3My8xNi81MjYxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMDIvMjAyMC4wMi4yOC45Njk3ODIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chaudhary</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sharma</span>, 
          <span class="cit-name-given-names">A. K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Agarwal</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gupta</span>, 
          <span class="cit-name-given-names">A.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Sharma</span>, 
          <span class="cit-name-given-names">V. K.</span>
         </span>
         <span class="cit-article-title">16S Classifier: A Tool for Fast and Accurate Taxonomic Classification of 16S rRNA Hypervariable Regions in Metagenomic Datasets</span>. 
         <abbr class="cit-jnl-abbrev">Plos One</abbr>
         <span class="cit-vol">10</span>, (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Drouin</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Interpretable genotype-to-phenotype classifiers with performance guarantees</span>. 
         <abbr class="cit-jnl-abbrev">Scientific Reports</abbr>
         <span class="cit-vol">9</span>, (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Fei-Fei</span>, 
          <span class="cit-name-given-names">L.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Perona</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-article-title">A Bayesian Hierarchical Model for Learning Natural Scene Categories</span>. 
         <abbr class="cit-jnl-abbrev">IEEE CVPR</abbr>. (
         <span class="cit-pub-date">2005</span>)
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-confproc" id="cit-2020.02.28.969782v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pu</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Local Word Bag Model for Text Categorization</span>. 
         <span class="cit-conf-name">Seventh IEEE International Conference on Data Mining (ICDM 2007)</span> (
         <span class="cit-pub-date">2007</span>).
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1109/icdm.2007.69</span>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Asgari</span>, 
          <span class="cit-name-given-names">E.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Mofrad</span>, 
          <span class="cit-name-given-names">M. R. K.</span>
         </span>
         <span class="cit-article-title">Continuous Distributed Representation of Biological Sequences for Deep Proteomics and Genomics</span>. 
         <abbr class="cit-jnl-abbrev">Plos One</abbr>
         <span class="cit-vol">10</span>, (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.12" data-doi="10.1128/MMBR.05023-11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Domingo</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sheldon</span>, 
          <span class="cit-name-given-names">J.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Perales</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>
         <span class="cit-article-title">Viral Quasispecies Evolution</span>. 
         <abbr class="cit-jnl-abbrev">Microbiology and Molecular Biology Reviews</abbr>
         <span class="cit-vol">76</span>, 
         <span class="cit-fpage">159</span>–
         <span class="cit-lpage">216</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DDomingo%26rft.auinit1%253DE.%26rft.volume%253D76%26rft.issue%253D2%26rft.spage%253D159%26rft.epage%253D216%26rft.atitle%253DViral%2BQuasispecies%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.05023-11%26rft_id%253Dinfo%253Apmid%252F22688811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI3Ni8yLzE1OSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzAyLzIwMjAuMDIuMjguOTY5NzgyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sahlgren</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-article-title">The Distributional Hypothesis</span>. 
         <abbr class="cit-jnl-abbrev">Rivista di Linguistica (Italian Journal of Linguistics)</abbr>
         <span class="cit-vol">20</span>, 
         <span class="cit-fpage">33</span>–
         <span class="cit-lpage">53</span> (
         <span class="cit-pub-date">2008</span>)
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DRivista%2Bdi%2BLinguistica%2B%2528Italian%2BJournal%2Bof%2BLinguistics%2529%26rft.volume%253D20%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Mikolov</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sutskever</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Corrado</span>, 
          <span class="cit-name-given-names">G.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Dean</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-article-title">Distributed Representations of Words and Phrases and their Compositionality</span>.
         <span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span>
         <span class="cit-pub-id-scheme">arXiv:</span>
         <span class="cit-pub-id cit-pub-id-arxiv">1310.4546</span> [cs.CL]. (
         <span class="cit-pub-date">2013</span>)
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Martell</span>, 
          <span class="cit-name-given-names">H. J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Masterson</span>, 
          <span class="cit-name-given-names">S. G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mcgreig</span>, 
          <span class="cit-name-given-names">J. E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Michaelis</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Wass</span>, 
          <span class="cit-name-given-names">M. N.</span>
         </span>
         <span class="cit-article-title">Is the Bombali virus pathogenic in humans?</span>
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">35</span>, 
         <span class="cit-fpage">3553</span>–
         <span class="cit-lpage">3558</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D35%26rft.spage%253D3553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shabman</span>, 
          <span class="cit-name-given-names">R. S.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">An Upstream Open Reading Frame Modulates Ebola Virus Polymerase Translation and Virus Replication</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Pathogens</abbr>
         <span class="cit-vol">9</span>, (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.17" data-doi="10.1007/s007050170049">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ikegami</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Genome structure of Ebola virus subtype Reston: differences among Ebola subtypes</span>. 
         <abbr class="cit-jnl-abbrev">Archives of Virology</abbr>
         <span class="cit-vol">146</span>, 
         <span class="cit-fpage">2021</span>–
         <span class="cit-lpage">2027</span> (
         <span class="cit-pub-date">2001</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Bvirology%26rft.stitle%253DArch%2BVirol%26rft.aulast%253DIkegami%26rft.auinit1%253DT.%26rft.volume%253D146%26rft.issue%253D10%26rft.spage%253D2021%26rft.epage%253D2027%26rft.atitle%253DGenome%2Bstructure%2Bof%2BEbola%2Bvirus%2Bsubtype%2BReston%253A%2Bdifferences%2Bamong%2BEbola%2Bsubtypes.%2BBrief%2Breport.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs007050170049%26rft_id%253Dinfo%253Apmid%252F11722021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/s007050170049&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=11722021&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.28.969782.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000172335900013&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sriwilaijaroen</span>, 
          <span class="cit-name-given-names">N.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Suzuki</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>
         <span class="cit-article-title">Molecular basis of the structure and function of H1 hemagglutinin of influenza virus</span>. 
         <abbr class="cit-jnl-abbrev">Proceedings of the Japan Academy, Series B</abbr>
         <span class="cit-vol">88</span>, 
         <span class="cit-fpage">226</span>–
         <span class="cit-lpage">249</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BJapan%2BAcademy%252C%2BSeries%2BB%26rft.volume%253D88%26rft.spage%253D226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.19" data-doi="10.1093/nar/28.1.292">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Nakamura</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>
         <span class="cit-article-title">Codon usage tabulated from international DNA sequence databases: status for the year 2000</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>
         <span class="cit-vol">28</span>, 
         <span class="cit-fpage">292</span>–
         <span class="cit-lpage">292</span> (
         <span class="cit-pub-date">2000</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DNakamura%26rft.auinit1%253DY.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D292%26rft.epage%253D292%26rft.atitle%253DCodon%2Busage%2Btabulated%2Bfrom%2Binternational%2BDNA%2Bsequence%2Bdatabases%253A%2Bstatus%2Bfor%2Bthe%2Byear%2B2000%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.292%26rft_id%253Dinfo%253Apmid%252F10592250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/28.1.292&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=10592250&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.28.969782.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000084896300084&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Clarke</span>, 
          <span class="cit-name-given-names">T. F.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Clark</span>, 
          <span class="cit-name-given-names">P. L.</span>
         </span>
         <span class="cit-article-title">Rare Codons Cluster</span>. 
         <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>
         <span class="cit-vol">3</span>, (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cruz</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cain</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Crossett</span>, 
          <span class="cit-name-given-names">B.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Kayser</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>
         <span class="cit-article-title">Site-specific glycosylation profile of influenza A (H1N1) hemagglutinin through tandem mass spectrometry</span>. 
         <abbr class="cit-jnl-abbrev">Human Vaccines &amp;; Immunotherapeutics</abbr>
         <span class="cit-vol">14</span>, 
         <span class="cit-fpage">508</span>–
         <span class="cit-lpage">517</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DHuman%2BVaccines%2B%2526%2BImmunotherapeutics%26rft.volume%253D14%26rft.spage%253D508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.22">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ebrahimi</span>, 
          <span class="cit-name-given-names">K. H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">West</span>, 
          <span class="cit-name-given-names">G. M.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Flefil</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-article-title">Mass Spectrometry Approach and ELISA Reveal the Effect of Codon Optimization on N-Linked Glycosylation of HIV-1 gp120</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Proteome Research</abbr>
         <span class="cit-vol">13</span>, 
         <span class="cit-fpage">5801</span>–
         <span class="cit-lpage">5811</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BProteome%2BResearch%26rft.volume%253D13%26rft.spage%253D5801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.23">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zucchelli</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Codon Optimization Leads to Functional Impairment of RD114-TR Envelope Glycoprotein</span>. 
         <abbr class="cit-jnl-abbrev">Molecular Therapy - Methods &amp;; Clinical Development</abbr>
         <span class="cit-vol">4</span>, 
         <span class="cit-fpage">102</span>–
         <span class="cit-lpage">114</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2BTherapy%2B-%2BMethods%2B%2526%2BClinical%2BDevelopment%26rft.volume%253D4%26rft.spage%253D102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.24" data-doi="10.1128/JVI.01016-13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Min</span>, 
          <span class="cit-name-given-names">J.-Y.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Mammalian Adaptation in the PB2 Gene of Avian H5N1 Influenza Virus</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>
         <span class="cit-vol">87</span>, 
         <span class="cit-fpage">10884</span>–
         <span class="cit-lpage">10888</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01016-13%26rft_id%253Dinfo%253Apmid%252F23864613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Ny8xOS8xMDg4NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzAyLzIwMjAuMDIuMjguOTY5NzgyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">25.</span>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.25">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tinoco</span>, 
          <span class="cit-name-given-names">Y. O.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Transmission dynamics of pandemic influenza A(H1N1)pdm09 virus in humans and swine in backyard farms in Tumbes, Peru</span>. 
         <abbr class="cit-jnl-abbrev">Influenza and Other Respiratory Viruses</abbr>
         <span class="cit-vol">10</span>, 
         <span class="cit-fpage">47</span>–
         <span class="cit-lpage">56</span> (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DInfluenza%2Band%2BOther%2BRespiratory%2BViruses%26rft.volume%253D10%26rft.spage%253D47%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">26.</span>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.26">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Influenza A(H1N1)pdm09 Virus Infection in Giant Pandas, China</span>. 
         <abbr class="cit-jnl-abbrev">Emerging Infectious Diseases</abbr>
         <span class="cit-vol">20</span>, 
         <span class="cit-fpage">480</span>–
         <span class="cit-lpage">483</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerging%2BInfectious%2BDiseases%26rft.volume%253D20%26rft.spage%253D480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">27.</span>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.27" data-doi="10.1056/NEJMoa0903812">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shinde</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Triple-Reassortant Swine Influenza A (H1) in Humans in the United States, 2005–2009</span>. 
         <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr>
         <span class="cit-vol">360</span>, 
         <span class="cit-fpage">2616</span>–
         <span class="cit-lpage">2625</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa0903812%26rft_id%253Dinfo%253Apmid%252F19423871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMoa0903812&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19423871&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.28.969782.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000267060000006&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">28.</span>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.28" data-doi="10.1086/651507">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bastien</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Human Infection with a Triple-Reassortant Swine Influenza A(H1N1) Virus Containing the Hemagglutinin and Neuraminidase Genes of Seasonal Influenza Virus</span>. 
         <abbr class="cit-jnl-abbrev">The Journal of Infectious Diseases</abbr>
         <span class="cit-vol">201</span>, 
         <span class="cit-fpage">1178</span>–
         <span class="cit-lpage">1182</span> (
         <span class="cit-pub-date">2010</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BInfectious%2BDiseases%26rft_id%253Dinfo%253Adoi%252F10.1086%252F651507%26rft_id%253Dinfo%253Apmid%252F20199242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1086/651507&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20199242&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.28.969782.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000275493400012&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">29.</span>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.29">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jonas</span>, 
          <span class="cit-name-given-names">O.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Seifman</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-article-title">Do we need a Global Virome Project?</span>
         <abbr class="cit-jnl-abbrev">The Lancet Global Health</abbr>
         <span class="cit-vol">7</span>, (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">30.</span>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.30">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Carlson</span>, 
          <span class="cit-name-given-names">C. J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zipfel</span>, 
          <span class="cit-name-given-names">C. M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Garnier</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bansal</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-article-title">Global estimates of mammalian viral diversity accounting for host sharing</span>. 
         <abbr class="cit-jnl-abbrev">Nature Ecology &amp;; Evolution</abbr>
         <span class="cit-vol">3</span>, 
         <span class="cit-fpage">1070</span>–
         <span class="cit-lpage">1075</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BEcology%2B%2526%2BEvolution%26rft.volume%253D3%26rft.spage%253D1070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <div class="section ref-list" id="ref-list-2">
    <h2 class="">Method References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">31.</span>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.31">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Fan</span>, 
          <span class="cit-name-given-names">R.-E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chang</span>, 
          <span class="cit-name-given-names">K.-W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hsieh</span>, 
          <span class="cit-name-given-names">C.-J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">X.-R.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>, 
          <span class="cit-name-given-names">C.-J.</span>
         </span>
         <span class="cit-article-title">LIBLINEAR: A Library for Large Linear Classification</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Machine Learning Research</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">1871</span>–
         <span class="cit-lpage">1874</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMachine%2BLearning%2BResearch%26rft.volume%253D9%26rft.spage%253D1871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">32.</span>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.32" data-doi="10.1007/978-3-7908-2604-3_16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bottou</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>
         <span class="cit-article-title">Large-Scale Machine Learning with Stochastic Gradient Descent</span>. 
         <abbr class="cit-jnl-abbrev">Proceedings of COMPSTAT2010</abbr>
         <span class="cit-fpage">177</span>–
         <span class="cit-lpage">186</span> (
         <span class="cit-pub-date">2010</span>).
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1007/978-3-7908-2604-3_16</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2BCOMPSTAT2010%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-7908-2604-3_16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/978-3-7908-2604-3_16&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">33.</span>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-confproc" id="cit-2020.02.28.969782v1.33">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shalev-Shwartz</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Singer</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Srebro</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>
         <span class="cit-article-title">Pegasos</span>. 
         <span class="cit-conf-name">Proceedings of the 24th international conference on Machine learning - ICML 07</span> (
         <span class="cit-pub-date">2007</span>).
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1145/1273496.1273598</span>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">34.</span>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-confproc" id="cit-2020.02.28.969782v1.34">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tsuruoka</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tsujii</span>, 
          <span class="cit-name-given-names">J.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ananiadou</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-article-title">Stochastic gradient descent training for L1-regularized log-linear models with cumulative penalty</span>. 
         <span class="cit-conf-name">Proceedings of the Joint Conference of the 47th Annual Meeting of the ACL and the 4th International Joint Conference on Natural Language Processing of the AFNLP: Volume 1 - ACL-IJCNLP 09</span> (
         <span class="cit-pub-date">2009</span>).
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.3115/1687878.1687946</span>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">35.</span>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.35">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jun</span>, 
          <span class="cit-name-given-names">S.-R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leuze</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ussery</span>, 
          <span class="cit-name-given-names">D.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Nookaew</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>
         <span class="cit-article-title">Viral Phylogenomics Using an Alignment-Free Method: A Three-Step Approach to Determine Optimal Length of k-mer</span>. 
         <abbr class="cit-jnl-abbrev">Scientific Reports</abbr>
         <span class="cit-vol">7</span>, (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">36.</span>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.36" data-doi="10.1186/1471-2164-9-517">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kurtz</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Narechania</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stein</span>, 
          <span class="cit-name-given-names">J. C.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ware</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-article-title">A new method to compute K-mer frequencies and its application to annotate large repetitive plant genomes</span>. 
         <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">517</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DKurtz%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.spage%253D517%26rft.epage%253D517%26rft.atitle%253DA%2Bnew%2Bmethod%2Bto%2Bcompute%2BK-mer%2Bfrequencies%2Band%2Bits%2Bapplication%2Bto%2Bannotate%2Blarge%2Brepetitive%2Bplant%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-517%26rft_id%253Dinfo%253Apmid%252F18976482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-517&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18976482&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.28.969782.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">37.</span>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.37">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hysom</span>, 
          <span class="cit-name-given-names">D. A.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Skip the Alignment: Degenerate, Multiplex Primer and Probe Design Using K-mer Matching Instead of Alignments</span>. 
         <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>
         <span class="cit-vol">7</span>, (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">38.</span>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.38">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Müllner</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-article-title">Modern hierarchical, agglomerative clustering algorithms</span>. (
         <span class="cit-pub-date">2011</span>). doi
         <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">: </span>
         <span class="cit-pub-id-scheme">arXiv:</span>
         <span class="cit-pub-id cit-pub-id-arxiv">1109.2378v1</span> [stat.ML]
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">39.</span>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.39">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Müllner</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-article-title">fastcluster: Fast Hierarchical, Agglomerative Clustering Routines forRandPython</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr>
         <span class="cit-vol">53</span>, (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">40.</span>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.40">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">May</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cotterell</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Van Durme</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>
         <span class="cit-article-title">An Analysis of Lemmatization on TopicModels of Morphologically Rich Language</span>. (
         <span class="cit-pub-date">2016</span>). doi
         <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">: </span>
         <span class="cit-pub-id-scheme">arXiv:</span>
         <span class="cit-pub-id cit-pub-id-arxiv">1608.03995v2</span> [cs.CL]
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">41.</span>
      <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.41">
       <div class="cit-metadata">
        <cite>
         <span class="cit-article-title">A revision of the system of nomenclature for influenza viruses: a WHO memorandum</span>. 
         <abbr class="cit-jnl-abbrev">Bulletin of the World Health Organization</abbr>
         <span class="cit-vol">58</span>, 
         <span class="cit-fpage">585</span>–
         <span class="cit-lpage">591</span> (
         <span class="cit-pub-date">1980</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBulletin%2Bof%2Bthe%2BWorld%2BHealth%2BOrganization%26rft.stitle%253DBull%2BWorld%2BHealth%2BOrgan%26rft.volume%253D58%26rft.issue%253D4%26rft.spage%253D585%26rft.epage%253D591%26rft.atitle%253DA%2Brevision%2Bof%2Bthe%2Bsystem%2Bof%2Bnomenclature%2Bfor%2Binfluenza%2Bviruses%253A%2Ba%2BWHO%2Bmemorandum.%26rft_id%253Dinfo%253Apmid%252F6969132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=6969132&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.28.969782.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1980KR70300003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">42.</span>
      <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.42" data-doi="10.3201/eid1112.041293">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SARS-CoV Infection in a Restaurant from Palm Civet</span>. 
         <abbr class="cit-jnl-abbrev">Emerging Infectious Diseases</abbr>
         <span class="cit-vol">11</span>, 
         <span class="cit-fpage">1860</span>–
         <span class="cit-lpage">1865</span> (
         <span class="cit-pub-date">2005</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DWang%26rft.auinit1%253DM.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D1860%26rft.epage%253D1865%26rft.atitle%253DSARS-CoV%2Binfection%2Bin%2Ba%2Brestaurant%2Bfrom%2Bpalm%2Bcivet.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1112.041293%26rft_id%253Dinfo%253Apmid%252F16485471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3201/eid1112.041293&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16485471&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.28.969782.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000233768300009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">43.</span>
      <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.43">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Younan</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bornstein</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Gluecks</span>, 
          <span class="cit-name-given-names">I. V.</span>
         </span>
         <span class="cit-article-title">MERS and the dromedary camel trade between Africa and the Middle East</span>. 
         <abbr class="cit-jnl-abbrev">Tropical Animal Health and Production</abbr>
         <span class="cit-vol">48</span>, 
         <span class="cit-fpage">1277</span>–
         <span class="cit-lpage">1282</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTropical%2BAnimal%2BHealth%2Band%2BProduction%26rft.volume%253D48%26rft.spage%253D1277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">44.</span>
      <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.44">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Discovery of a rich gene pool of bat SARS-related coronaviruses provides new insights into the origin of SARS coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">PLOS Pathogens</abbr>
         <span class="cit-vol">13</span>, (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">45.</span>
      <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.45">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Goodacre</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Aljanahi</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nandakumar</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mikailov</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Khan</span>, 
          <span class="cit-name-given-names">A. S.</span>
         </span>
         <span class="cit-article-title">A Reference Viral Database (RVDB) To Enhance Bioinformatics Analysis of High-Throughput Sequencing for Novel Virus Detection</span>. 
         <abbr class="cit-jnl-abbrev">mSphere</abbr>
         <span class="cit-vol">3</span>, (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">46.</span>
      <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.46">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hatcher</span>, 
          <span class="cit-name-given-names">E. L.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Virus Variation Resource – improved response to emergent viral outbreaks</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>
         <span class="cit-vol">45</span>, (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">47.</span>
      <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a>
      <div class="cit ref-cit ref-book" id="cit-2020.02.28.969782v1.47">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Manning</span>, 
          <span class="cit-name-given-names">C. D.</span>
         </span> in 
         <span class="cit-source">Introduction to Information Retrieval</span>
         <span class="cit-fpage">403</span>–
         <span class="cit-lpage">419</span> (
         <span class="cit-publ-name">Cambridge University Press</span>, 
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">48.</span>
      <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.48">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">van der Maaten</span>, 
          <span class="cit-name-given-names">L.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hinton</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>
         <span class="cit-article-title">Visualizing High-Dimensional Data Using t-SNE</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Machine Learning Research</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">2579</span>–
         <span class="cit-lpage">2605</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMachine%2BLearning%2BResearch%26rft.volume%253D9%26rft.spage%253D2579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">49.</span>
      <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.28.969782v1.49">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">van der Maaten</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>
         <span class="cit-article-title">Accelerating t-SNE using Tree-Based Algorithms</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Machine Learning Research</abbr>
         <span class="cit-vol">15</span>, 
         <span class="cit-fpage">3221</span>–
         <span class="cit-lpage">3245</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMachine%2BLearning%2BResearch%26rft.volume%253D15%26rft.spage%253D3221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">50.</span>
      <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.28.969782v1.50">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">Plotly Technologies Inc</span>. 
         <span class="cit-source">Collaborative data science</span>. 
         <span class="cit-pub-date">2015</span>. 
         <a href="https://plot.ly">https://plot.ly</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
