<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/> 
  <meta name="type" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/04/10/2020.04.07.024752.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.04.07.024752v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Human ACE2 receptor polymorphisms predict SARS-CoV-2 susceptibility"/>
  <meta name="DC.Identifier" content="10.1101/2020.04.07.024752"/>
  <meta name="DC.Date" content="2020-04-10"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="Â© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of coronavirus disease (COVID-19) that has resulted in a global pandemic. It is a highly contagious positive strand RNA virus and its clinical presentation includes severe to critical respiratory disease that appears to be fatal in ~3-5% of the cases. The viral spike (S) coat protein engages the human angiotensin-converting enzyme2 (ACE2) cell surface protein to invade the host cell. The SARS-CoV-2 S-protein has acquired mutations that increase its affinity to human ACE2 by ~10-15-fold compared to SARS-CoV S-protein, making it highly infectious. In this study, we assessed if ACE2 polymorphisms might alter host susceptibility to SARS-CoV-2 by affecting the ACE2 S-protein interaction. Our comprehensive analysis of several large genomic datasets that included over 290,000 samples representing &gt;;400 population groups identified multiple ACE2 protein-altering variants, some of which mapped to the S-protein-interacting ACE2 surface. Using recently reported structural data and a recent S-protein-interacting synthetic mutant map of ACE2, we have identified natural ACE2 variants that are predicted to alter the virus-host interaction and thereby potentially alter host susceptibility. In particular, human ACE2 variants S19P, I21V, E23K, K26R, T27A, N64K, T92I, Q102P and H378R are predicted to increase susceptibility. The T92I variant, part of a consensus NxS/T N-glycosylation motif, confirmed the role of N90 glycosylation in immunity from non-human CoVs. Other ACE2 variants K31R, N33I, H34R, E35K, E37K, D38V, Y50F, N51S, M62V, K68E, F72V, Y83H, G326E, G352V, D355N, Q388L and D509Y are putative protective variants predicted to show decreased binding to SARS-CoV-2 S-protein. Overall, ACE2 variants are rare, consistent with the lack of selection pressure given the recent history of SARS-CoV epidemics, however, are likely to play an important role in altering susceptibility to CoVs. ### Competing Interest Statement"/>
  <meta name="DC.Contributor" content="Eric W Stawiski"/>
  <meta name="DC.Contributor" content="Devan Diwanji"/>
  <meta name="DC.Contributor" content="Kushal Suryamohan"/>
  <meta name="DC.Contributor" content="Ravi Gupta"/>
  <meta name="DC.Contributor" content="Frederic A Fellouse"/>
  <meta name="DC.Contributor" content="Fah Sathirapongsasuti"/>
  <meta name="DC.Contributor" content="Jiang Liu"/>
  <meta name="DC.Contributor" content="Ying-Ping Jiang"/>
  <meta name="DC.Contributor" content="Aakrosh Ratan"/>
  <meta name="DC.Contributor" content="Monika Mis"/>
  <meta name="DC.Contributor" content="Devi Santhosh"/>
  <meta name="DC.Contributor" content="Sneha Somasekar"/>
  <meta name="DC.Contributor" content="Sangeetha Mohan"/>
  <meta name="DC.Contributor" content="Sameer Phalke"/>
  <meta name="DC.Contributor" content="Boney Kuriakose"/>
  <meta name="DC.Contributor" content="Aju Antony"/>
  <meta name="DC.Contributor" content="Jagath R Junutula"/>
  <meta name="DC.Contributor" content="Stephan C Schuster"/>
  <meta name="DC.Contributor" content="Natalia Jura"/>
  <meta name="DC.Contributor" content="Somasekar Seshagiri"/>
  <meta name="article:published_time" content="2020-04-10"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Human ACE2 receptor polymorphisms predict SARS-CoV-2 susceptibility"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of coronavirus disease (COVID-19) that has resulted in a global pandemic. It is a highly contagious positive strand RNA virus and its clinical presentation includes severe to critical respiratory disease that appears to be fatal in ~3-5% of the cases. The viral spike (S) coat protein engages the human angiotensin-converting enzyme2 (ACE2) cell surface protein to invade the host cell. The SARS-CoV-2 S-protein has acquired mutations that increase its affinity to human ACE2 by ~10-15-fold compared to SARS-CoV S-protein, making it highly infectious. In this study, we assessed if ACE2 polymorphisms might alter host susceptibility to SARS-CoV-2 by affecting the ACE2 S-protein interaction. Our comprehensive analysis of several large genomic datasets that included over 290,000 samples representing &amp;;gt;400 population groups identified multiple ACE2 protein-altering variants, some of which mapped to the S-protein-interacting ACE2 surface. Using recently reported structural data and a recent S-protein-interacting synthetic mutant map of ACE2, we have identified natural ACE2 variants that are predicted to alter the virus-host interaction and thereby potentially alter host susceptibility. In particular, human ACE2 variants S19P, I21V, E23K, K26R, T27A, N64K, T92I, Q102P and H378R are predicted to increase susceptibility. The T92I variant, part of a consensus NxS/T N-glycosylation motif, confirmed the role of N90 glycosylation in immunity from non-human CoVs. Other ACE2 variants K31R, N33I, H34R, E35K, E37K, D38V, Y50F, N51S, M62V, K68E, F72V, Y83H, G326E, G352V, D355N, Q388L and D509Y are putative protective variants predicted to show decreased binding to SARS-CoV-2 S-protein. Overall, ACE2 variants are rare, consistent with the lack of selection pressure given the recent history of SARS-CoV epidemics, however, are likely to play an important role in altering susceptibility to CoVs.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.04.07.024752v1"/>
  <meta name="citation_id" content="2020.04.07.024752v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.07.024752v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.07.024752v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.07.024752v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.07.024752.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.04.07.024752"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.04.07.024752"/>
  <meta name="citation_author" content="Eric W Stawiski"/>
  <meta name="citation_author_institution" content="Research and Development Department, MedGenome Inc., Foster City, CA 94404, USA;"/>
  <meta name="citation_author_email" content="eric.s@medgenome.com"/>
  <meta name="citation_author" content="Devan Diwanji"/>
  <meta name="citation_author_institution" content="Cardiovascular Research Institute, University of California San Francisco, San Francisco, CA 94158, USA and Department of Cellular and Molecular Pharmacology University of California San Francisco, San Francisco, CA 94158, USA;"/>
  <meta name="citation_author_email" content="devan.diwanji@ucsf.edu"/>
  <meta name="citation_author" content="Kushal Suryamohan"/>
  <meta name="citation_author_institution" content="Research and Development Department, MedGenome Inc., Foster City, CA 94404, USA;"/>
  <meta name="citation_author_email" content="kushal.s@medgenome.com"/>
  <meta name="citation_author" content="Ravi Gupta"/>
  <meta name="citation_author_institution" content="MedGenome Labs Ltd., 3rd Floor, Narayana Netralaya Building, Narayana Health City, #258/A, Bommasandra, Hosur Road, Bangalore, Karnataka 560099, India;"/>
  <meta name="citation_author_email" content="ravig@medgenome.com"/>
  <meta name="citation_author" content="Frederic A Fellouse"/>
  <meta name="citation_author_institution" content="ModMab Therapeutics, Accelerator for Donnelly Collaboration, University of Toronto, Toronto, Ontario M5S 1A8, Canada;"/>
  <meta name="citation_author_email" content="fred@modmab.com"/>
  <meta name="citation_author" content="Fah Sathirapongsasuti"/>
  <meta name="citation_author_institution" content="Research and Development Department, MedGenome Inc., Foster City, CA 94404, USA;"/>
  <meta name="citation_author_email" content="fah.s@medgenome.com"/>
  <meta name="citation_author" content="Jiang Liu"/>
  <meta name="citation_author_institution" content="ModMab Therapeutics, 384 Hatch Drive, Foster City, CA 94404, USA;"/>
  <meta name="citation_author_email" content="jiangliu@modmab.com"/>
  <meta name="citation_author" content="Ying-Ping Jiang"/>
  <meta name="citation_author_institution" content="ModMab Therapeutics, 384 Hatch Drive, Foster City, CA 94404, USA;"/>
  <meta name="citation_author_email" content="pingj@modmab.com"/>
  <meta name="citation_author" content="Aakrosh Ratan"/>
  <meta name="citation_author_institution" content="Center for Public Health Genomics, University of Virginia, Charlottesville, VA 22908, USA;"/>
  <meta name="citation_author_email" content="ar7jq@virginia.edu"/>
  <meta name="citation_author" content="Monika Mis"/>
  <meta name="citation_author_institution" content="Research and Development Department, MedGenome Inc., Foster City, CA 94404, USA;"/>
  <meta name="citation_author_email" content="monika.m@medgenome.com"/>
  <meta name="citation_author" content="Devi Santhosh"/>
  <meta name="citation_author_institution" content="ModMab Therapeutics, 384 Hatch Drive, Foster City, CA 94404, USA;"/>
  <meta name="citation_author_email" content="devi.santhosh@modmab.com"/>
  <meta name="citation_author" content="Sneha Somasekar"/>
  <meta name="citation_author_institution" content="Midwestern University, Glendale, AZ 85308, USA;"/>
  <meta name="citation_author" content="Sangeetha Mohan"/>
  <meta name="citation_author_institution" content="Department of Molecular Biology, SciGenom Labs Pvt Ltd, Kerala 682037, India;"/>
  <meta name="citation_author" content="Sameer Phalke"/>
  <meta name="citation_author_institution" content="Department of Molecular Biology, SciGenom Labs Pvt Ltd, Kerala 682037, India;"/>
  <meta name="citation_author" content="Boney Kuriakose"/>
  <meta name="citation_author_institution" content="AgriGenome Labs Private Ltd, Kochi, Kerala 682030, India;"/>
  <meta name="citation_author" content="Aju Antony"/>
  <meta name="citation_author_institution" content="Department of Molecular Biology, SciGenom Labs Pvt Ltd, Kerala 682037, India;"/>
  <meta name="citation_author" content="Jagath R Junutula"/>
  <meta name="citation_author_institution" content="ModMab Therapeutics, 348 Hatch Drive, Foster City, CA 94404;"/>
  <meta name="citation_author" content="Stephan C Schuster"/>
  <meta name="citation_author_institution" content="Singapore Centre for Environmental Life Sciences Engineering, Nanyang Technological University, Singapore 637551, Singapore;"/>
  <meta name="citation_author" content="Natalia Jura"/>
  <meta name="citation_author_institution" content="Cardiovascular Research Institute, University of California San Francisco, San Francisco, CA 94158, USA and Department of Cellular and Molecular Pharmacology University of California San Francisco, San Francisco, CA 94158, USA;"/>
  <meta name="citation_author_email" content="natalia.jura@ucsf.edu"/>
  <meta name="citation_author" content="Somasekar Seshagiri"/>
  <meta name="citation_author_institution" content="SciGenom Research Foundation, Chennai, Tamil Nadu 600020, India; ModMab Therapeutics, 384 Hatch Drive, Foster City, CA 94404, USA"/>
  <meta name="citation_author_email" content="sekar@sgrf.org"/>
  <meta name="twitter:title" content="Human ACE2 receptor polymorphisms predict SARS-CoV-2 susceptibility"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of coronavirus disease (COVID-19) that has resulted in a global pandemic. It is a highly contagious positive strand RNA virus and its clinical presentation includes severe to critical respiratory disease that appears to be fatal in ~3-5% of the cases. The viral spike (S) coat protein engages the human angiotensin-converting enzyme2 (ACE2) cell surface protein to invade the host cell. The SARS-CoV-2 S-protein has acquired mutations that increase its affinity to human ACE2 by ~10-15-fold compared to SARS-CoV S-protein, making it highly infectious. In this study, we assessed if ACE2 polymorphisms might alter host susceptibility to SARS-CoV-2 by affecting the ACE2 S-protein interaction. Our comprehensive analysis of several large genomic datasets that included over 290,000 samples representing &gt;;400 population groups identified multiple ACE2 protein-altering variants, some of which mapped to the S-protein-interacting ACE2 surface. Using recently reported structural data and a recent S-protein-interacting synthetic mutant map of ACE2, we have identified natural ACE2 variants that are predicted to alter the virus-host interaction and thereby potentially alter host susceptibility. In particular, human ACE2 variants S19P, I21V, E23K, K26R, T27A, N64K, T92I, Q102P and H378R are predicted to increase susceptibility. The T92I variant, part of a consensus NxS/T N-glycosylation motif, confirmed the role of N90 glycosylation in immunity from non-human CoVs. Other ACE2 variants K31R, N33I, H34R, E35K, E37K, D38V, Y50F, N51S, M62V, K68E, F72V, Y83H, G326E, G352V, D355N, Q388L and D509Y are putative protective variants predicted to show decreased binding to SARS-CoV-2 S-protein. Overall, ACE2 variants are rare, consistent with the lack of selection pressure given the recent history of SARS-CoV epidemics, however, are likely to play an important role in altering susceptibility to CoVs. ### Competing Interest Statement"/>
  <meta name="og-title" property="og:title" content="Human ACE2 receptor polymorphisms predict SARS-CoV-2 susceptibility"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.07.024752v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of coronavirus disease (COVID-19) that has resulted in a global pandemic. It is a highly contagious positive strand RNA virus and its clinical presentation includes severe to critical respiratory disease that appears to be fatal in ~3-5% of the cases. The viral spike (S) coat protein engages the human angiotensin-converting enzyme2 (ACE2) cell surface protein to invade the host cell. The SARS-CoV-2 S-protein has acquired mutations that increase its affinity to human ACE2 by ~10-15-fold compared to SARS-CoV S-protein, making it highly infectious. In this study, we assessed if ACE2 polymorphisms might alter host susceptibility to SARS-CoV-2 by affecting the ACE2 S-protein interaction. Our comprehensive analysis of several large genomic datasets that included over 290,000 samples representing &gt;;400 population groups identified multiple ACE2 protein-altering variants, some of which mapped to the S-protein-interacting ACE2 surface. Using recently reported structural data and a recent S-protein-interacting synthetic mutant map of ACE2, we have identified natural ACE2 variants that are predicted to alter the virus-host interaction and thereby potentially alter host susceptibility. In particular, human ACE2 variants S19P, I21V, E23K, K26R, T27A, N64K, T92I, Q102P and H378R are predicted to increase susceptibility. The T92I variant, part of a consensus NxS/T N-glycosylation motif, confirmed the role of N90 glycosylation in immunity from non-human CoVs. Other ACE2 variants K31R, N33I, H34R, E35K, E37K, D38V, Y50F, N51S, M62V, K68E, F72V, Y83H, G326E, G352V, D355N, Q388L and D509Y are putative protective variants predicted to show decreased binding to SARS-CoV-2 S-protein. Overall, ACE2 variants are rare, consistent with the lack of selection pressure given the recent history of SARS-CoV epidemics, however, are likely to play an important role in altering susceptibility to CoVs. ### Competing Interest Statement"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-04-10"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Human ACE2 receptor polymorphisms predict SARS-CoV-2 susceptibility | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1239194 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-category-new-results"> 
  <div class="article abstract-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2>Abstract</h2>
    <p id="p-2">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of coronavirus disease (COVID-19) that has resulted in a global pandemic. It is a highly contagious positive strand RNA virus and its clinical presentation includes severe to critical respiratory disease that appears to be fatal in ~3-5% of the cases. The viral spike (S) coat protein engages the human angiotensin-converting enzyme2 (ACE2) cell surface protein to invade the host cell. The SARS-CoV-2 S-protein has acquired mutations that increase its affinity to human ACE2 by ~10-15-fold compared to SARS-CoV S-protein, making it highly infectious. In this study, we assessed if ACE2 polymorphisms might alter host susceptibility to SARS-CoV-2 by affecting the ACE2 S-protein interaction. Our comprehensive analysis of several large genomic datasets that included over 290,000 samples representing &gt;;400 population groups identified multiple ACE2 protein-altering variants, some of which mapped to the S-protein-interacting ACE2 surface. Using recently reported structural data and a recent S-protein-interacting synthetic mutant map of ACE2, we have identified natural ACE2 variants that are predicted to alter the virus-host interaction and thereby potentially alter host susceptibility. In particular, human ACE2 variants S19P, I21V, E23K, K26R, T27A, N64K, T92I, Q102P and H378R are predicted to increase susceptibility. The T92I variant, part of a consensus NxS/T N-glycosylation motif, confirmed the role of N90 glycosylation in immunity from non-human CoVs. Other ACE2 variants K31R, N33I, H34R, E35K, E37K, D38V, Y50F, N51S, M62V, K68E, F72V, Y83H, G326E, G352V, D355N, Q388L and D509Y are putative protective variants predicted to show decreased binding to SARS-CoV-2 S-protein. Overall, ACE2 variants are rare, consistent with the lack of selection pressure given the recent history of SARS-CoV epidemics, however, are likely to play an important role in altering susceptibility to CoVs.</p>
   </div>
   <h3>Competing Interest Statement</h3>
   <p id="p-3"/>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
