<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/06/07/341735.atom" />
<meta name="HW.pisa" content="biorxiv;341735v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="SKEMPI 2.0: An updated benchmark of changes in protein-protein binding energy, kinetics and thermodynamics upon mutation" />
<meta name="DC.Identifier" content="10.1101/341735" />
<meta name="DC.Date" content="2018-06-07" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Motivation Understanding the relationship between the sequence, structure, binding energy, binding kinetics and binding thermodynamics of protein-protein interactions is crucial to understanding cellular signaling, the assembly and regulation of molecular complexes, the mechanisms through which mutations lead to disease, and protein engineering.

Results We present SKEMPI 2.0, a major update to our database of binding free energy changes upon mutation for structurally resolved protein-protein interactions. This version now contains manually curated binding data for 7085 mutations, an increase of 133%, including changes in kinetics for 1844 mutations, enthalpy and entropy changes for 443 mutations, and 440 mutations which abolish detectable binding.

Availability The database is available at &lt;https://life.bsc.es/pid/skempi2/&gt;" />
<meta name="DC.Contributor" content="Justina Jankauskaitė" />
<meta name="DC.Contributor" content="Brian Jiménez-García" />
<meta name="DC.Contributor" content="Justas Dapkūnas" />
<meta name="DC.Contributor" content="Juan Fernández-Recio" />
<meta name="DC.Contributor" content="Iain H. Moal" />
<meta name="article:published_time" content="2018-06-07" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="SKEMPI 2.0: An updated benchmark of changes in protein-protein binding energy, kinetics and thermodynamics upon mutation" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;h3&gt;Motivation&lt;/h3&gt;
&lt;p&gt;Understanding the relationship between the sequence, structure, binding energy, binding kinetics and binding thermodynamics of protein-protein interactions is crucial to understanding cellular signaling, the assembly and regulation of molecular complexes, the mechanisms through which mutations lead to disease, and protein engineering.&lt;/p&gt;&lt;h3&gt;Results&lt;/h3&gt;
&lt;p&gt;We present SKEMPI 2.0, a major update to our database of binding free energy changes upon mutation for structurally resolved protein-protein interactions. This version now contains manually curated binding data for 7085 mutations, an increase of 133%, including changes in kinetics for 1844 mutations, enthalpy and entropy changes for 443 mutations, and 440 mutations which abolish detectable binding.&lt;/p&gt;&lt;h3&gt;Availability&lt;/h3&gt;
&lt;p&gt;The database is available at https://life.bsc.es/pid/skempi2/&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;341735v1" />
<meta name="citation_id" content="341735v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/341735v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/341735v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/341735v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/06/07/341735.full.pdf" />
<meta name="citation_doi" content="10.1101/341735" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="341735" />
<meta name="citation_author" content="Justina Jankauskaitė" />
<meta name="citation_author_institution" content="Institute of Biotechnology, Life Sciences Center, Vilnius University" />
<meta name="citation_author" content="Brian Jiménez-García" />
<meta name="citation_author_institution" content="Barcelona Supercomputing Center (BSC)" />
<meta name="citation_author_institution" content="Bijvoet Center for Biomolecular Research, Faculty of Science, Utrecht University" />
<meta name="citation_author" content="Justas Dapkūnas" />
<meta name="citation_author_institution" content="Institute of Biotechnology, Life Sciences Center, Vilnius University" />
<meta name="citation_author" content="Juan Fernández-Recio" />
<meta name="citation_author_institution" content="Barcelona Supercomputing Center (BSC)" />
<meta name="citation_author_institution" content="Institut de Biologia Molecular de Barcelona (IBMB)" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3986-7686" />
<meta name="citation_author" content="Iain H. Moal" />
<meta name="citation_author_institution" content="European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI)" />
<meta name="citation_author_email" content="moal@ebi.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4960-5487" />
<meta name="citation_reference" content="R. Agius, M. Torchala, I. H. Moal, J. Fernandez-Recio, and P. A. Bates. Characterizing changes in the rate of protein-protein dissociation upon interface mutation using hotspot energy and organization. PLoS Comput. Biol., 9(9):e1003216, 2013." />
<meta name="citation_reference" content="Y. Aizner, O. Sharabi, J. Shirian, G. R. Dakwar, M. Risman, O. Avraham, and J. Shifman. Mapping of the binding landscape for a pico-molar protein-protein complex through computation and experiment. Structure, 22(4):636–645, Apr 2014." />
<meta name="citation_reference" content="V. Arkadash, G. Yosef, J. Shirian, I. Cohen, Y. Horev, M. Grossman, I. Sagi, E. S. Radisky, J. M. Shifman, and N. Papo. Development of High Affinity and High Specificity Inhibitors of Matrix Metalloproteinase 14 through Computational Design and Directed Evolution. J. Biol. Chem., 292(8):3481–3495, Feb 2017." />
<meta name="citation_reference" content="H. Aydin, A. Sultana, S. Li, A. Thavalingam, and J. E. Lee. Molecular architecture of the human sperm IZUMO1 and egg JUNO fertilization complex. Nature, 534(7608):562–565, 06 2016." />
<meta name="citation_reference" content="K. A. Barlow, S. O Conchuir, S. Thompson, P. Suresh, J. E. Lucas, M. Heinonen, and T. Kortemme. Flex ddG: Rosetta Ensemble-Based Estimation of Changes in Protein-Protein Binding Affinity upon Mutation. J. Phys. Chem. B, in press, Feb 2018." />
<meta name="citation_reference" content="D. Barradas-Bautista, I. H. Moal, and J. Fernandez-Recio. A systematic analysis of scoring functions in rigid-body protein docking: The delicate balance between the predictive rate improvement and the risk of overtraining. Proteins, 85(7):1287–1297, Jul 2017." />
<meta name="citation_reference" content="N. Berliner, J. Teyra, R. Colak, S. Garcia Lopez, and P. M. Kim. Combining structural modeling with ensemble machine learning to accurately predict protein fold stability and binding affinity effects upon mutation. PLoS ONE, 9(9):e107353, 2014." />
<meta name="citation_reference" content="H. M. Berman, J. Westbrook, Z. Feng, G. Gilliland, T. N. Bhat, H. Weissig, I. N. Shindyalov, and P. E. Bourne. The Protein Data Bank. Nucleic Acids Res., 28(1):235–242, Jan 2000." />
<meta name="citation_reference" content="S. E. Broughton, T. R. Hercus, T. L. Nero, M. Dottore, B. J. McClure, U. Dhagat, H. Taing, M. A. Gorman, J. King-Scott, A. F. Lopez, and M. W. Parker. Conformational Changes in the GM-CSF Receptor Suggest a Molecular Mechanism for Affinity Conversion and Receptor Signaling. Structure, 24(8):1271–1281, 08 2016." />
<meta name="citation_reference" content="N. G. Brown, D. C. Chow, K. E. Ruprecht, and T. Palzkill. Identification of the β-lactamase inhibitor protein-II (BLIP-II)interface residues essential for binding affinity and specificity for class A β-lactamases. J. Biol. Chem., 288(24):17156–17166, Jun 2013." />
<meta name="citation_reference" content="A. J. Clark, T. Gindin, B. Zhang, L. Wang, R. Abel, C. S. Murret, F. Xu, A. Bao, N. J. Lu, T. Zhou, P. D. Kwong, L. Shapiro, B. Honig, and R. A. Friesner. Free Energy Perturbation Calculation of Relative Binding Free Energy between Broadly Neutralizing Antibodies and the gp120 Glycoprotein of HIV-1. J. Mol. Biol., 429(7):930–947, Apr 2017." />
<meta name="citation_reference" content="L. A. Clark, P. A. Boriack-Sjodin, J. Eldredge, C. Fitch, B. Friedman, K. J. Hanf, M. Jarpe, S. F. Liparoto, Y. Li, A. Lugovskoy, S. Miller, M. Rushe, W. Sherman, K. Simon, and H. Van Vlijmen. Affinity enhancement of an in vivo matured therapeutic antibody using structure-based computational design. Protein Sci., 15(5):949–960, May 2006." />
<meta name="citation_reference" content="R. Cohen-Khait and G. Schreiber. Low-stringency selection of TEM1 for BLIP shows interface plasticity and selection for faster binders. Proc. Natl. Acad. Sci. U.S.A., 113(52):14982–14987, Dec 2016." />
<meta name="citation_reference" content="B. C. Cunningham and J. A. Wells. Rational design of receptor-specific variants of human growth hormone. Proc. Natl. Acad. Sci. U.S.A., 88(8):3407–3411, Apr 1991." />
<meta name="citation_reference" content="W. Dall’Acqua, E. R. Goldman, W. Lin, C. Teng, D. Tsuchiya, H. Li, X. Ysern, B. C. Braden, Y. Li, S. J. Smith-Gill, and R. A. Mariuzza. A mutational analysis of binding interactions in an antigen-antibody protein-protein complex. Biochemistry, 37(22):7981–7991, Jun 1998." />
<meta name="citation_reference" content="J. Das, H. R. Lee, A. Sagar, R. Fragoza, J. Liang, X. Wei, X. Wang, M. Mort, P. D. Stenson, D. N. Cooper, and H. Yu. Elucidating common structural features of human pathogenic variations using large-scale atomic-resolution protein networks. Hum. Mutat., 35(5):585–593, May 2014." />
<meta name="citation_reference" content="Y. Dehouck, J. M. Kwasigroch, M. Rooman, and D. Gilis. BeAt-MuSiC: Prediction of changes in protein-protein binding affinity on mutations. Nucleic Acids Res., 41(Web Server issue):W333–339, Jul 2013." />
<meta name="citation_reference" content="D. F. Dourado and S. C. Flores. A multiscale approach to predicting affinity changes in protein-protein interfaces. Proteins, 82(10):2681–2690, Oct 2014." />
<meta name="citation_reference" content="V. C. Epa, O. Dolezal, L. Doughty, X. Xiao, C. Jost, A. Pluckthun, and T. E. Adams. Structural model for the interaction of a designed Ankyrin Repeat Protein with the human epidermal growth factor receptor 2. PLoS ONE, 8(3):e59163, 2013." />
<meta name="citation_reference" content="S. J. Fleishman, T. A. Whitehead, D. C. Ekiert, C. Dreyfus, J. E. Corn, E. M. Strauch, I. A. Wilson, and D. Baker. Computational design of proteins targeting the conserved stem region of influenza hemagglutinin. Science, 332(6031):816–821, May 2011." />
<meta name="citation_reference" content="H. Gardsvoll, B. Gilquin, M. H. Le Du, A. Menez, T. J. J?rgensen, and M. Ploug. Characterization of the functional epitope on the urokinase receptor. Complete alanine scanning mutagenesis supplemented by chemical cross-linking. J. Biol. Chem., 281(28):19260–19272, Jul 2006." />
<meta name="citation_reference" content="C. Geng, A. Vangone, and A. M. J. J. Bonvin. Exploring the interplay between experimental methods and the performance of predictors of binding affinity change upon mutations in protein complexes. Protein Eng. Des. Sel., 29(8):291–299, Aug 2016." />
<meta name="citation_reference" content="L. Gossage, D. E. Pires, A. Olivera-Nappa, J. Asenjo, M. Bycroft, T. L. Blundell, and T. Eisen. An integrated computational approach can classify VHL missense mutations according to risk of clear cell renal carcinoma. Hum. Mol. Genet., 23(22):5976–5988, Nov 2014." />
<meta name="citation_reference" content="G. Guntas, C. Purbeck, and B. Kuhlman. Engineering a protein-protein interface using a computationally designed library. Proc. Natl. Acad. Sci. U.S.A., 107(45):19296–19301, Nov 2010." />
<meta name="citation_reference" content="W. Guo, J. A. Wisniewski, and H. Ji. Hot spot-based design of small-molecule inhibitors for protein-protein interactions. Bioorg. Med. Chem. Lett., 24(11):2546–2554, Jun 2014." />
<meta name="citation_reference" content="A. Horovitz and A. R. Fersht. Strategy for analysing the co-operativity of intramolecular interactions in peptides and proteins. J. Mol. Biol., 214(3):613–617, Aug 1990." />
<meta name="citation_reference" content="J. Howitt, M. C. Bewley, V. Graziano, J. M. Flanagan, and P. Freimuth. Structural basis for variation in adenovirus affinity for the cellular coxsackievirus and adenovirus receptor. J. Biol. Chem., 278(28):26208–26215, Jul 2003." />
<meta name="citation_reference" content="J. Hu, J. Li, N. Chen, and X. Zhang. Conservation of hot regions in protein-protein interaction in evolution. Methods, 110:73–80, Nov 2016." />
<meta name="citation_reference" content="X. Huang and D. L. Brutlag. Dynamic use of multiple parameter sets in sequence alignment. Nucleic Acids Res., 35(2):678–686, 2007." />
<meta name="citation_reference" content="H. Hwang, T. Vreven, and Z. Weng. Binding interface prediction by combining protein-protein docking results. Proteins, 82(1):57–66, Jan 2014." />
<meta name="citation_reference" content="S. Jemimah, K. Yugandhar, and M. Michael Gromiha. PROXiMATE: a database of mutant protein-protein complex thermodynamics and kinetics. Bioinformatics, 33(17):2787–2788, Sep 2017." />
<meta name="citation_reference" content="P. Jemth, X. Mu, A. Engstrom, and J. Dogan. A frustrated binding interface for intrinsically disordered proteins. J. Biol. Chem., 289(9):5528–5533, Feb 2014." />
<meta name="citation_reference" content="P. L. Kastritis, I. H. Moal, H. Hwang, Z. Weng, P. A. Bates, A. M. Bonvin, and J. Janin. A structure-based benchmark for protein-protein binding affinity. Protein Sci., 20(3):482–491, Mar 2011." />
<meta name="citation_reference" content="C. Kiel, L. Serrano, and C. Herrmann. A detailed thermodynamic analysis of ras/effector complex interfaces. J. Mol. Biol., 340(5):1039–1058, Jul 2004." />
<meta name="citation_reference" content="M. V. Kulkarni, M. C. Tettamanzi, J. W. Murphy, C. Keeler, D. G. Myszka, N. E. Chayen, E. J. Lolis, and M. E. Hodsdon. Two independent histidines, one in human prolactin and one in its receptor, are critical for pH-dependent receptor recognition and activation. J. Biol. Chem., 285(49):38524–38533, Dec 2010." />
<meta name="citation_reference" content="M. D. Kumar and M. M. Gromiha. PINT: Protein-protein Interactions Thermodynamic Database. Nucleic Acids Res., 34(Database issue):D195–198, Jan 2006." />
<meta name="citation_reference" content="J. K. Lai, J. Ambia, Y. Wang, and P. Barth. Enhancing Structure Prediction and Design of Soluble and Membrane Proteins with Explicit Solvent-Protein Interactions. Structure, 25(11):1758–1770, Nov 2017." />
<meta name="citation_reference" content="E. D. Levy. A simple definition of structural regions in proteins and its use in analyzing interface evolution. J. Mol. Biol., 403(4):660–670, Nov 2010." />
<meta name="citation_reference" content="M. Li, F. L. Simonetti, A. Goncearenco, and A. R. Panchenko. MutaBind estimates and interprets the effects of sequence variants on protein-protein interactions. Nucleic Acids Res., 44(W1):494–501, Jul 2016." />
<meta name="citation_reference" content="W. Li, C. A. Dennis, G. R. Moore, R. James, and C. Kleanthous. Protein-protein interaction specificity of Im9 for the endonuclease toxin colicin E9 defined by homologue-scanning mutagenesis. J. Biol. Chem., 272(35):22253–22258, Aug 1997." />
<meta name="citation_reference" content="Q. Liu, P. Chen, B. Wang, and J. Li. dbMPIKT: A web resource for the kinetic and thermodynamic database of mutant protein interactions. arXiv:1708.01857, 2017." />
<meta name="citation_reference" content="Q. Liu, J. Ren, J. Song, and J. Li. Co-Occurring Atomic Contacts for the Characterization of Protein Binding Hot Spots. PLoS ONE, 10(12):e0144486, 2015." />
<meta name="citation_reference" content="S. M. Lu, W. Lu, M. A. Qasim, S. Anderson, I. Apostol, W. Ardelt, T. Bigler, Y. W. Chiang, J. Cook, M. N. James, I. Kato, C. Kelly, W. Kohr, T. Komiyama, T. Y. Lin, M. Ogawa, J. Otlewski, S. J. Park, S. Qasim, M. Ranjbar, M. Tashiro, N. Warne, H. Whatley, A. Wieczorek, M. Wieczorek, T. Wilusz, R. Wynn, W. Zhang, and M. Laskowski. Predicting the reactivity of proteins from their sequence alone: Kazal family of protein inhibitors of serine proteinases. Proc. Natl. Acad. Sci. U.S.A., 98(4):1410–1415, Feb 2001." />
<meta name="citation_reference" content="R. Melo, R. Fieldhouse, A. Melo, J. D. Correia, M. N. Cordeiro, Z. H. Gumu?, J. Costa, A. M. Bonvin, and I. S. Moreira. A Machine Learning Approach for Hot-Spot Detection at Protein-Protein Interfaces. Int. J. Mol. Sci., 17(8), Jul 2016." />
<meta name="citation_reference" content="I. H. Moal, D. Barradas-Bautista, B. Jimenez-Garcia, M. Torchala, A. van der Velde, T. Vreven, Z. Weng, P. A. Bates, and J. Fernandez-Recio. IRaPPA: information retrieval based integration of biophysical models for protein assembly selection. Bioinformatics, 33(12):1806–1813, Jun 2017." />
<meta name="citation_reference" content="I. H. Moal, J. Dapk?nas, and J. Fernandez-Recio. Inferring the microscopic surface energy of protein-protein interfaces from mutation data. Proteins, 83(4):640–650, Apr 2015." />
<meta name="citation_reference" content="I. H. Moal and J. Fernandez-Recio. SKEMPI: a Structural Kinetic and Energetic database of Mutant Protein Interactions and its use in empirical models. Bioinformatics, 28(20):2600–2607, Oct 2012." />
<meta name="citation_reference" content="I. H. Moal and J. Fernandez-Recio. Intermolecular Contact Potentials for Protein-Protein Interactions Extracted from Binding Free Energy Changes upon Mutation. J. Chem. Theory Comput., 9(8):3715–3727, Aug 2013." />
<meta name="citation_reference" content="I. H. Moal, B. Jimenez-Garcia, and J. Fernandez-Recio. CCharPPI web server: computational characterization of protein-protein interactions from structure. Bioinformatics, 31(1):123–125, Jan 2015." />
<meta name="citation_reference" content="I. H. Moal, M. Torchala, P. A. Bates, and J. Fernandez-Recio. The scoring of poses in protein-protein docking: current capabilities and future directions. BMC Bioinformatics, 14:286, Oct 2013." />
<meta name="citation_reference" content="R. Moretti, S. J. Fleishman, R. Agius, M. Torchala, P. A. Bates, P. L. Kastritis, J. P. Rodrigues, M. Trellet, A. M. Bonvin, M. Cui, M. Rooman, D. Gillis, Y. Dehouck, I. Moal, M. Romero-Durana, L. Perez-Cano, C. Pallara, B. Jimenez, J. Fernandez-Recio, S. Flores, M. Pacella, K. Praneeth Kilambi, J. J. Gray, P. Popov, S. Grudinin, J. Esquivel-Rodriguez, D. Kihara, N. Zhao, D. Korkin, X. Zhu, O. N. Demerdash, J. C. Mitchell, E. Kanamori, Y. Tsuchiya, H. Nakamura, H. Lee, H. Park, C. Seok, J. Sarmiento, S. Liang, S. Teraguchi, D. M. Standley, H. Shimoyama, G. Terashi, M. Takeda-Shitaka, M. Iwadate, H. Umeyama, D. Beglov, D. R. Hall, D. Kozakov, S. Vajda, B. G. Pierce, H. Hwang, T. Vreven, Z. Weng, Y. Huang, H. Li, X. Yang, X. Ji, S. Liu, Y. Xiao, M. Zacharias, S. Qin, H. X. Zhou, S. Y. Huang, X. Zou, S. Velankar, J. Janin, S. J. Wodak, and D. Baker. Community-wide evaluation of methods for predicting the effect of mutations on protein-protein interactions. Proteins, 81(11):1980–1987, Nov 2013." />
<meta name="citation_reference" content="T. W. Nam, H. I. Jung, Y. J. An, Y. H. Park, S. H. Lee, Y. J. Seok, and S. S. Cha. Analyses of Mlc-IIBGlc interaction and a plausible molecular mechanism of Mlc inactivation by membrane sequestration. Proc. Natl. Acad. Sci. U.S.A., 105(10):3751–3756, Mar 2008." />
<meta name="citation_reference" content="C. Pallara, B. Jimenez-Garcia, L. Perez-Cano, M. Romero-Durana, A. Solernou, S. Grosdidier, C. Pons, I. H. Moal, and J. Fernandez-Recio. Expanding the frontiers of protein-protein modeling: from docking and scoring to binding affinity predictions and other challenges. Proteins, 81(12):2192–2200, Dec 2013." />
<meta name="citation_reference" content="R. J. Pantazes, M. J. Grisewood, T. Li, N. P. Gifford, and C. D. Maranas. The Iterative Protein Redesign and Optimization (IPRO) suite of programs. J. Comput. Chem., 36(4):251–263, Feb 2015." />
<meta name="citation_reference" content="Y. Peng and E. Alexov. Investigating the linkage between disease-causing amino acid variants and their effect on protein stability and binding. Proteins, 84(2):232–239, Feb 2016." />
<meta name="citation_reference" content="M. Petukh, T. G. Kucukkal, and E. Alexov. On human disease-causing amino acid variants: statistical study of sequence and structural patterns. Hum. Mutat., 36(5):524–534, May 2015." />
<meta name="citation_reference" content="M. Petukh, M. Li, and E. Alexov. Predicting Binding Free Energy Change Caused by Point Mutations with Knowledge-Modified MM/PBSA Method. PLoS Comput. Biol., 11(7):e1004276, Jul 2015." />
<meta name="citation_reference" content="E. Pfeiffenberger, R. A. Chaleil, I. H. Moal, and P. A. Bates. A machine learning approach for ranking clusters of docked protein-protein complexes by pairwise cluster comparison. Proteins, 85(3):528–543, Mar 2017." />
<meta name="citation_reference" content="K. H. Piepenbrink, S. J. Blevins, D. R. Scott, and B. M. Baker. The basis for limited specificity and MHC restriction in a T cell receptor interface. Nat Commun, 4:1948, 2013." />
<meta name="citation_reference" content="D. E. Pires, D. B. Ascher, and T. L. Blundell. mCSM: predicting the effects of mutations in proteins using graph-based signatures. Bioinformatics, 30(3):335–342, Feb 2014." />
<meta name="citation_reference" content="N. Plattner, S. Doerr, G. De Fabritiis, and F. Noe. Complete protein-protein association kinetics in atomic detail revealed by molecular dynamics simulations and Markov modelling. Nat Chem, 9(10):1005–1011, Oct 2017." />
<meta name="citation_reference" content="D. Reichmann, M. Cohen, R. Abramovich, O. Dym, D. Lim, N. C. Strynadka, and G. Schreiber. Binding hot spots in the TEM1-BLIP interface in light of its modular architecture. J. Mol. Biol., 365(3):663–679, Jan 2007." />
<meta name="citation_reference" content="L. C. Roisman, J. Piehler, J. Y. Trosset, H. A. Scheraga, and G. Schreiber. Structure of the interferon-receptor complex determined by distance constraints from double-mutant cycles and flexible docking. Proc. Natl. Acad. Sci. U.S.A., 98(23):13231–13236, Nov 2001." />
<meta name="citation_reference" content="R. Rosenfeld, R. Alcalay, A. Mechaly, G. Lapidoth, E. Epstein, C. Kronman, S. J Fleishman, and O. Mazor. Improved antibody-based ricin neutralization by affinity maturation is correlated with slower off-rate values. Protein Eng. Des. Sel., pages 1–7, May 2017." />
<meta name="citation_reference" content="E. Seiradake, H. Lortat-Jacob, O. Billet, E. J. Kremer, and S. Cusack. Structural and mutational analysis of human Ad37 and canine adenovirus 2 fiber heads in complex with the D1 domain of coxsackie and adenovirus receptor. J. Biol. Chem., 281(44):33704–33716, Nov 2006." />
<meta name="citation_reference" content="I. C. Simoes, I. P. Costa, J. T. Coimbra, M. J. Ramos, and P. A. Fernandes. New Parameters for Higher Accuracy in the Computation of Binding Free Energy Differences upon Alanine Scanning Mutagenesis on Protein-Protein Interfaces. J. Chem. Inf. Model., 57(1):60–72, Jan 2017." />
<meta name="citation_reference" content="S. Sirin, J. R. Apgar, E. M. Bennett, and A. E. Keating. AB-Bind: Antibody binding mutational database for computational affinity predictions. Protein Sci., 25(2):393–409, Feb 2016." />
<meta name="citation_reference" content="L. M. Stevers, R. M. de Vries, R. G. Doveston, L. G. Milroy, L. Brunsveld, and C. Ottmann. Structural interface between LRRK2 and 14-3-3 protein. Biochem. J., 474(7):1273–1287, Mar 2017." />
<meta name="citation_reference" content="F. Sumbul, S. E. Acuner-Ozbabacan, and T. Haliloglu. Allosteric Dynamic Control of Binding. Biophys. J., 109(6):1190–1201, Sep 2015." />
<meta name="citation_reference" content="C. Thomas, I. Moraga, D. Levin, P. O. Krutzik, Y. Podoplelova, A. Trejo, C. Lee, G. Yarden, S. E. Vleck, J. S. Glenn, G. P. Nolan, J. Piehler, G. Schreiber, and K. C. Garcia. Structural linkage between ligand discrimination and receptor activation by type I interferons. Cell, 146(4):621–632, Aug 2011." />
<meta name="citation_reference" content="K. S. Thorn and A. A. Bogan. ASEdb: a database of alanine mutations and their effects on the free energy of binding in protein interactions. Bioinformatics, 17(3):284–285, Mar 2001." />
<meta name="citation_reference" content="H. Tidow, D. B. Veprintsev, S. M. Freund, and A. R. Fersht. Effects of oncogenic mutations and DNA response elements on the binding of p53 to p53-binding protein 2 (53BP2). J. Biol. Chem., 281(43):32526–32533, Oct 2006." />
<meta name="citation_reference" content="T. Vreven, I. H. Moal, A. Vangone, B. G. Pierce, P. L. Kastritis, M. Torchala, R. Chaleil, B. Jimenez-Garcia, P. A. Bates, J. Fernandez-Recio, A. M. Bonvin, and Z. Weng. Updates to the Integrated Protein-Protein Interaction Benchmarks: Docking Benchmark Version 5 and Affinity Benchmark Version 2. J. Mol. Biol., 427(19):3031–3041, Sep 2015." />
<meta name="citation_reference" content="M. D. Winn, C. C. Ballard, K. D. Cowtan, E. J. Dodson, P. Emsley, P. R. Evans, R. M. Keegan, E. B. Krissinel, A. G. Leslie, A. McCoy, S. J. McNicholas, G. N. Murshudov, N. S. Pannu, E. A. Potterton, H. R. Powell, R. J. Read, A. Vagin, and K. S. Wilson. Overview of the CCP4 suite and current developments. Acta Crystallogr. D Biol. Crystallogr., 67(Pt 4):235–242, Apr 2011." />
<meta name="citation_reference" content="L. C. Wu, D. S. Tuot, D. S. Lyons, K. C. Garcia, and M. M. Davis. Two-step binding mechanism for T-cell receptor recognition of peptide MHC. Nature, 418(6897):552–556, Aug 2002." />
<meta name="citation_reference" content="P. Xiong, C. Zhang, W. Zheng, and Y. Zhang. BindProfX: Assessing Mutation-Induced Binding Affinity Change by Protein Interface Profiles with Pseudo-Counts. J. Mol. Biol., 429(3):426–434, Feb 2017." />
<meta name="citation_reference" content="Y. Yan, M. Yang, C. G. Ji, and J. Z. H. Zhang. Interaction Entropy for Computational Alanine Scanning. J. Chem. Inf. Model., 57(5):1112–1122, May 2017." />
<meta name="citation_reference" content="N. Zhao, J. G. Han, C. R. Shyu, and D. Korkin. Determining effects of non-synonymous SNPs on protein-protein interactions using supervised and semi-supervised learning. PLoS Comput. Biol., 10(5):e1003592, May 2014." />
<meta name="twitter:title" content="SKEMPI 2.0: An updated benchmark of changes in protein-protein binding energy, kinetics and thermodynamics upon mutation" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Motivation Understanding the relationship between the sequence, structure, binding energy, binding kinetics and binding thermodynamics of protein-protein interactions is crucial to understanding cellular signaling, the assembly and regulation of molecular complexes, the mechanisms through which mutations lead to disease, and protein engineering.

Results We present SKEMPI 2.0, a major update to our database of binding free energy changes upon mutation for structurally resolved protein-protein interactions. This version now contains manually curated binding data for 7085 mutations, an increase of 133%, including changes in kinetics for 1844 mutations, enthalpy and entropy changes for 443 mutations, and 440 mutations which abolish detectable binding.

Availability The database is available at &lt;https://life.bsc.es/pid/skempi2/&gt;" />
<meta name="og-title" property="og:title" content="SKEMPI 2.0: An updated benchmark of changes in protein-protein binding energy, kinetics and thermodynamics upon mutation" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/341735v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Motivation Understanding the relationship between the sequence, structure, binding energy, binding kinetics and binding thermodynamics of protein-protein interactions is crucial to understanding cellular signaling, the assembly and regulation of molecular complexes, the mechanisms through which mutations lead to disease, and protein engineering.

Results We present SKEMPI 2.0, a major update to our database of binding free energy changes upon mutation for structurally resolved protein-protein interactions. This version now contains manually curated binding data for 7085 mutations, an increase of 133%, including changes in kinetics for 1844 mutations, enthalpy and entropy changes for 443 mutations, and 440 mutations which abolish detectable binding.

Availability The database is available at &lt;https://life.bsc.es/pid/skempi2/&gt;" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-06-07" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/341735v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/341735v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/341735v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/341735v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/103964" />
    <title>SKEMPI 2.0: An updated benchmark of changes in protein-protein binding energy, kinetics and thermodynamics upon mutation | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"3xa6S_HthFzQDPEKCHcANHgHlntU_dVLoJOlkJhyQL0","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"103964","apath":"\/biorxiv\/early\/2018\/06\/07\/341735.atom","pisa":"biorxiv;341735v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;341735v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/341735v1","title":"SKEMPI 2.0: An updated benchmark of changes in protein-protein binding energy, kinetics and thermodynamics upon mutation","identifier":"node\/103964"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022iAznjn8lR0bG\\\\\\\/TMg2NNgsgqjcvfvjRTjgvHRdR+Xgi3pgfz\\\\\\\/F4k7kKoe08ZIMGlmd9S0JCQ4l0ldlg29gJR0GADypWavqffNdJOq70KvXxuSd3DvG2Umkeap\\\\\\\/yknlCAQ22M6HAuWqNzKyEZHv2xtyokHZP8Rm6gviB2X2RloP+kasVKYhp9LtpesHyW78BOUC8HAI6fBEflUzatamtaElburShaNij0DNKkA4ZdEDmVd3CxTuPKldHzByxEw6BpZWOicLvQRtQMzb7YVXPp03t+y3W5Q4D8s2T63dSUzKbZ5BMceMWpmQS9yrRAPVRjPRltWDl8pxgbqYRVrukGjTQhRg9nHzwfnytpR7Dz+JhXSj0FPpxvXxZK8WdMlw3+TDT57IbHvuot5c0wGM5Y9ws9XlhD59Uva0ZyISoECNHD7sjpyJ6RPQBKAb97++Zvz\\\\\\\/DwV0w3OPA5j5U2Lpo0NaelwgQkktPAfwb7HokPjkE\\\\\\\/wd8SRkanzp2FYlc5O7D2Y+BRqXWksoG13Mv9IVHMaCW4aRhasAUbfKUHYZ3MuChPnoM3m+t4lWE+q2wKP8bssh4rz24ln1UoEtZii0E1HdKIm3UJeZe9qUrIn0+oT2OIDPdxeguASJNTi3Zg6bxGGoGGp8J5zHNerNzlxSY0VZtD7dnS6tUV96aRNqeLaOeJgiMhjV4KjG9KoimsWX4bd2xrQay4BvZL3WfnnSvnYCEJZkX13vdGfNUOQWNoPK8UT5ZP9GdeKivMebvnTOcuL42357c2NX8sFIWiehF33KtANkEe64ZFlZZkBBoMjGfYM\\\\\\\/4GzbkHKES0eXFvfIm8cQlZkbBKYLyfRM8TvbKxuL\\\\\\\/NN7WUxhZmCcJaTqHHp1HjYHU7lMtuoyjR5S7fxSBciTexOEqpm3O5uhT+IZD48OAiylPpOWxptIOkYazVpb0fSZauM3D6F+sXwHwNcNjjUT1rGOIsyG9mgXfBQlNfwa8cys2SA30514YnF5vOQYoC\\\\\\\/dVzBvU3Fj+p6vCTLUfQRN5nKwN9yw5HHfj0L1KocJlaCufballYAkv+IS4A27uMUTqx3SJ32fr2j7UAFhCszwsz5GTdgj\\\\\\\/7teiSNyr8HhswtCXi9RvLwHHWWG\\\\\\\/uADf\\\\\\\/fWdcoIOlFhDUfbksXAJ1tIYIS8mkTz0HG0b8FhEAifwZBnj3rIcQl9eRR6yifW7zouxQLFj3Ha1gXYZGf\\\\\\\/AiqALd4Fp3nyNoak4cJ9Q5EhY6t0w9Hm723x+ynl4OoOjv1g\\\\\\\/z+gbjHC63pMg0MOPh3Qa5fFbgCaonEGEru8xeKFXTTDMYIZhC5qZyW2q\\\\\\\/fCUg=\\\u0022,\\\u0022iv\\\u0022:\\\u0022S\\\\\\\/vXqSa3qSyN2f9dzzGbDg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002254f4c5ac3ff39ec660b6958884822a2fc27c9e7943dabd0797d9713c14379ddc\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/341735v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-103964 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/341735v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-w0oyvS8A6e7Xal99b_DzDOWiUIgE-ZtasVR996ohxHg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="103964" id="node103964" data-pisa="biorxiv;341735v1" data-pisa-master="biorxiv;341735" data-apath="/biorxiv/early/2018/06/07/341735.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">SKEMPI 2.0: An updated benchmark of changes in protein-protein binding energy, kinetics and thermodynamics upon mutation</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Justina</span> <span class="nlm-surname">Jankauskaitė</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Brian</span> <span class="nlm-surname">Jiménez-García</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Justas</span> <span class="nlm-surname">Dapkūnas</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-3986-7686" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Juan</span> <span class="nlm-surname">Fernández-Recio</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-4960-5487" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Iain H.</span> <span class="nlm-surname">Moal</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/341735 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node103964" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Justina Jankauskaitė </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute of Biotechnology, Life Sciences Center, Vilnius University</span>, Vilnius LT-10257, <span class='nlm-country'>Lithuania</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justina%2BJankauskait%C4%97%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jankauskait%C4%97%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustina%2BJankauskait%25C4%2597%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Brian Jiménez-García </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Barcelona Supercomputing Center (BSC)</span>, Barcelona 08034, <span class='nlm-country'>Spain</span>.</div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Bijvoet Center for Biomolecular Research, Faculty of Science, Utrecht University</span>, Utrecht, <span class='nlm-country'>the Netherlands</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BJim%C3%A9nez-Garc%C3%ADa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jim%C3%A9nez-Garc%C3%ADa%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BJim%25C3%25A9nez-Garc%25C3%25ADa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Justas Dapkūnas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute of Biotechnology, Life Sciences Center, Vilnius University</span>, Vilnius LT-10257, <span class='nlm-country'>Lithuania</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justas%2BDapk%C5%ABnas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dapk%C5%ABnas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustas%2BDapk%25C5%25ABnas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Juan Fernández-Recio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Barcelona Supercomputing Center (BSC)</span>, Barcelona 08034, <span class='nlm-country'>Spain</span>.</div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Institut de Biologia Molecular de Barcelona (IBMB)</span>, CSIC, Barcelona 08028, <span class='nlm-country'>Spain</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BFern%C3%A1ndez-Recio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fern%C3%A1ndez-Recio%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJuan%2BFern%25C3%25A1ndez-Recio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3986-7686" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Juan Fernández-Recio</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Iain H. Moal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI)</span>, Hinxton, Cambridge, CB10 1SD, <span class='nlm-country'>UK</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Iain%2BH.%2BMoal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moal%20IH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIain%2BH.%2BMoal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4960-5487" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Iain H. Moal</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:moal@ebi.ac.uk" class="" data-icon-position="" data-hide-link-title="0">moal@ebi.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/341735v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:103964" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:103964/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/341735v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:103964" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:103964/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/341735v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:103964" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:103964/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/341735v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:103964" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:103964/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/341735v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:103964" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:103964/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><div id="sec-1" class="subsection"><p id="p-2"><strong>Motivation</strong> Understanding the relationship between the sequence, structure, binding energy, binding kinetics and binding thermodynamics of protein-protein interactions is crucial to understanding cellular signaling, the assembly and regulation of molecular complexes, the mechanisms through which mutations lead to disease, and protein engineering.</p></div><div id="sec-2" class="subsection"><p id="p-3"><strong>Results</strong> We present SKEMPI 2.0, a major update to our database of binding free energy changes upon mutation for structurally resolved protein-protein interactions. This version now contains manually curated binding data for 7085 mutations, an increase of 133%, including changes in kinetics for 1844 mutations, enthalpy and entropy changes for 443 mutations, and 440 mutations which abolish detectable binding.</p></div><div id="sec-3" class="subsection"><p id="p-4"><strong>Availability</strong> The database is available at <a href="https://life.bsc.es/pid/skempi2/">https://life.bsc.es/pid/skempi2/</a></p></div></div><div class="section" id="sec-4"><h2 class="">I. INTRODUCTION</h2><p id="p-5">Protein-protein interactions are central to almost all biological processes, from cellular signal transduction and the assembly of mesoscopic structures such as myofilaments, to viral adhesion and the immune response. Consequently the effects of changes in protein sequence on the structure, thermodynamics and kinetics of protein-protein interactions has wide implications for constraining the permissible sub-stitutions that accrue over the course of evolution, and for understanding the molecular etiology of disease. Methods which measure, predict or optimise these changes have applications in designing <em>de novo</em> interactions [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>], enhancing the specificity and affinity of biological therapeutics (e.g. [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]), designing combinatorial protein libraries (e.g. [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]), uncovering the effects of pathological mutations (e.g. [<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a>]), locating druggable binding sites (e.g. [<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>]) and binding hotspots for drug design [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>], altering binding kinetics [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>], [<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>], protein-protein docking (e.g. [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]), and characterising transition states (e.g. [<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a>]), binding pathways [<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>], and sequence-affinity landscapes [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>].</p><p id="p-6">SKEMPI is a manually curated database of mutations in structurally characterised protein-protein interactions and the effect of those mutations on binding affinity and other parameters [<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>]. The first release has been used as a basis for many further studies, including the development of energy functions [<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>], [<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>] which were subsequently implemented in the CCharPPI web server for characterising protein-protein interactions [<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>], as well as being used for ranking docked poses [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>], [<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>], [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>], [<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>]. SKEMPI has also been used to study human disease [<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>], [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>], [<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>], assessing the role of dynamics on binding [<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>], exploring the conservation of binding regions [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>], evaluating experimental affinity measurement methods [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>], as well serving as a data source for models which predict dissociation rate changes upon mutation [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>], pathological mutations [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>], hotspot residues (e.g. [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>], [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>], [<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>], [<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>]) and changes in binding energy (e.g. [<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>], [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>], [<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>], [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>], [<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>], [<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>], [<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>], [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>], [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>], [<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a>], [<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>], [<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a>], [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>], [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]).</p><p id="p-7">Here we present a major update to the benchmark in terms of the number of mutations in the database and the number of different systems included (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table I</a>). We now also include details of the experimental method for all entries, based on the categories of [<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>], as well as mutations which abolished detectable binding or for which only an upper or lower affinity limit could be ascertained for the wild-type or mutant.

</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/513933/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/513933/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/513933" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table I</span> <p id="p-8" class="first-child">COMPARISON WITH PREVIOUS VERSION</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-5"><h2 class="">II. METHODS</h2><div id="sec-6" class="subsection"><h3>A. Data sources</h3><p id="p-9">Just over two fifths of the data come from the previous version of SKEMPI [<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">47</a>], comprised mostly of data found in literature sources which came to the authors’ attention, in some cases during the data collection for the structural affinity benchmark [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>] and following references therein. Some entries in SKEMPI 1.1 were found by checking the references in the ASEdb [<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a>] and PINT [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>] databases, although not all the data passed the checks required for inclusion (see <a id="xref-sec-7-1" class="xref-sec" href="#sec-7">section II-B</a>). Similarly, most of the new data in SKEMPI 2.0 was found by searching the literature, partly in tandem with the literature search for the more recent structural affinity benchmark [<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a>]. During data collection three other relevant databases were published: ABbind [<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>], PROXiMATE [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>] and dbMPIKT [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>]. Their references were checked if they were not already included. Data from these sources comprise 4%, 3% and 6% of SKEMPI 2.0 respectively. As with ASEdb and PINT, none of the data were directly copied into SKEMPI. Moreover, the cited papers were read and data entered using the same checks and procedures as other entries.</p></div><div id="sec-7" class="subsection"><h3>B. Data collection</h3><p id="p-10">Each entry was found in the literature and manually vetted. To ensure quality, a number of stringent checks were applied. Firstly, we ensured that the structure and the paper reporting the affinities refer to the same protein in the same species, and that structural and affinity data matched in terms of cofactors, ancillary chains and post-translational modifications. For instance, we distinguish between Ras<sup>GTP</sup> and Ras<sup>GppNHp</sup>, as the nucleotide modulates the affinity of Ras with its effectors. Where the full length protein was not used, we checked to ensure that the fragment in the crystal structure matched that for which affinities are reported.</p><p id="p-11">Once the checks are passed, the data is collected, including the PDB file, the chains of the interacting subunits, the mutation, the wild-type and mutant affinities (<em>K<sub>D</sub></em>, <em>M</em>), the reference, the names of the proteins, the temperature at which the experiment is performed (<em>T</em>, <em>K</em>), the experimental method used (an extension of the category scheme of [<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a>]), notes on the entry and, when available, the association rate (<em>k</em><sub>on</sub>, <em>M</em><sup>−1</sup><em>s</em><sup>−1</sup>), dissociation rate (<em>k</em><sub>off</sub>, <em>s</em><sup>−1</sup>), enthalpy (Δ<em>H</em>, kcal.mol<sup>−1</sup>) and entropy (Δ<em>S</em>, cal.mol<sup>−1</sup>.K<sup>−1</sup>). For cases where multiple PDB entries are available, the higher resolution structure is chosen. Where affinities or kinetic or thermodynamic parameters are reported in different units, these are converted to the units specified above. In some cases, when not reported directly, <em>K<sub>D</sub></em>, <em>k</em><sub>on</sub>, <em>k</em><sub>off</sub>, Δ<em>H</em> and Δ<em>S</em> were calculated using the relationships Δ<em>G</em> = Δ<em>H</em> – <em>T</em>Δ<em>S</em> = <em>RT</em>ln(<em>K<sub>D</sub></em>) and <em>K<sub>D</sub></em> = 1/<em>K<sub>A</sub></em> = <em>k</em><sub>off</sub>/<em>k</em><sub>on</sub>. To ensure consistency, the residue numbering in SKEMPI is the same as that reported in the PDB file. Thus the numbering is often shifted or altered compared to that in the cited paper such that, for instance, if a crystal structure of an antibody is reported in the Kabat numbering scheme but the mutation data is not, then the mutation data is converted before entry into SKEMPI. For all entries it is the case that the the affinity reported in the “wild-type” column corresponds to that of the PDB file, and the affinity in the “mutant” column is that after applying the specified mutation to the protein in the PDB file. Thus, where there are cases in which the PDB reports a mutant form and the entry corresponds to the reverse mutation back to the wild-type, the affinity of the former appears in the “wild-type” column and the latter in the “mutant” column. Such cases are noted in the database.</p><p id="p-12">In addition to checking new entries, we reappraised the papers cited in SKEMPI 1.1 to collect data that were not collected previously, specifically to find mutants which abolish binding and to classify the experimental method used when not already included in the subset of SKEMPI covered in [<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">22</a>]. It is worth noting that often an author’s decision to report an interaction of affinity below the detection threshold as either non-binding, or as less affine than the weakest affinity presented in the paper, is arbitrary. Thus, those wishing to use the non-binding data as an inequality on the affinity may do so. We also corrected entries for 5 wild-type and 4 mutant affinities identified by [<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">22</a>].</p></div><div id="sec-8" class="subsection"><h3>C. Post-processing and annotation</h3><p id="p-13">In addition to the above data, SKEMPI 2.0 also provides data on the location of the mutated residues, the homology between interactions in the data set, and processed PDB files which can be easily parsed.</p><div id="sec-9" class="subsection"><h4>Residue location</h4><p id="p-14">Each mutated residue is classified according to the scheme proposed by [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>]; residues at the interface are classified as support (mostly buried when unbound and entirely buried upon binding), core (mostly solvent exposed when unbound but buried upon binding) and rim (partly buried upon binding), while residues away from the binding site are classified as interior or surface. Solvent exposed surface area was calculated using CCP4 [<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a>].</p></div><div id="sec-10" class="subsection"><h4>Processed PDB files</h4><p id="p-15">The PDB files for the interactions, as downloaded from the Protein Data Bank [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>], often contain multiple copies of the interacting proteins in the unit cell or other chains irrelevant to the interaction. In one instance, the binding of dimeric myostatin to follistatin-like 3, the myostatin dimer must be created by tessellating the unit cell. Further, some PDB files contain features that are not readily parsed by some software, such as residue insertion codes or negative residue numbers. To help users we provide “cleaned” PDB files which contain only the chains of interest, renumbered from one, as well as waters and other molecules with a non-hydrogen atom within 5 Å of a non-hydrogen atom of any of the chains of interest. Consequently each mutation is reported with both PDB numbering and renumbered.</p></div><div id="sec-11" class="subsection"><h4>Defining homologous interactions</h4><p id="p-16">Each entry also specifies which other entries are mutations to homologous interactions. Two interactions are deemed homologous if they have a shared binding partner or homologous binding partner and at least 70% of the corresponding interface residues are common to both interactions. We determine the homology between proteins using the GAP4 program [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>], and define homologous proteins as those with a similarity score greater than 50 and at least 30% sequence identity. Interface residues are defined as those with a non-hydrogen atom within 10 Å of a non-hydrogen atom on the binding partner. Interactions falling within manually assigned clusters of homologous interactions are designated as pMHC/TCR, antibody/antigen or protease/inhibitor. While the names of these clusters have been chosen to reflect the predominant function of their constituent interactions, they reflect the homologies within the data set and are not functional assignments. Thus, for instance, some nanobodies are classified as antibodies as they bind to the same site as cetuximab, 14.3.d is classified as TCR, even though it is only the <em>β</em> chain, and its binding partner, enterotoxin C3, is classified as a pMHC.</p></div></div></div><div class="section" id="sec-12"><h2 class="">III. RESULTS &amp; DISCUSSION</h2><div id="sec-13" class="subsection"><h3>A. Diversity, bias and interrelationships within SKEMPI</h3><p id="p-17">In total 7085 entries were collected, summarised in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table I</a> and <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>. These data were derived from the literature and consequently, while encompassing a broad range of residues, proteins, interactions and systems, are biased toward the interests and capabilities of the research community. These biases are evident in the composition of the database according to parameters shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>. The ΔΔ<em>G</em> values span a large range, but mostly fall within −3 to 7 kcal.mol<sup>−1</sup> (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>), for both biophysical and technical reasons (see <a id="xref-sec-15-1" class="xref-sec" href="#sec-15">Section III-C</a>). Almost three quarters of the data correspond to single point mutations, and more than half of those are mutations to alanine (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). Charge swap mutations and mutations between aromatic residues are also over-represented. Most single point mutations are located at the binding site, and most of those are at the core of the interface. Similarly, most double mutations are both in the binding site, and most of those are both in the core. By far the most popular methods for measuring binding affinity were surface plasmon resonance and spectroscopic methods such as fluorescence. Large biases towards specific interactions and classes of interaction are also present, such as early studies into protease inhibition and immunological interactions such as antibody-antigen complexes, the recognition of peptides presented on cell surfaces by T-cell receptors, cytokine signalling and the complement system. Indeed, almost half of the data corresponds to protease-inhibitor, antibody-antigen and pMHC/TCR interactions alone. While many interactions within these classes share common binding sites or homologous binding sites (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>), there are also connections between these groups, for instance via inhibitory antibodies which bind to a protease active site, or due to common binding regions of proteins in the immuoglobulin superfamily, such as antibodies, TCRs, MHCs and <em>β</em>-2 microglobulin. Also present in the data are smaller clusters of shared and homologous interactions, such as the Ras-effector cluster. These relationships are noted in the database and may be useful for avoiding overfitting when developing models or for validation and estimating generalisation error, as described previously [<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">47</a>].

</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/06/07/341735/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="An overview of SKEMPI 2.0. (A) Mutations partitioned according to their origin, the number of altered residues, location within the complex, by the availability of additional kinetic and thermodynamic data, according to the experimental method used, and by category. (B) Distribution of &#x394;&#x394;G. (C) Source and target amino acids for single point mutations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1344929061" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;An overview of SKEMPI 2.0. (A) Mutations partitioned according to their origin, the number of altered residues, location within the complex, by the availability of additional kinetic and thermodynamic data, according to the experimental method used, and by category. (B) Distribution of &#x394;&#x394;&lt;em&gt;G&lt;/em&gt;. (C) Source and target amino acids for single point mutations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/06/07/341735/F1.medium.gif" width="440" height="414"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2018/06/07/341735/F1.medium.gif" width="440" height="414"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/06/07/341735/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/06/07/341735/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1.</span> <p id="p-18" class="first-child">An overview of SKEMPI 2.0. (A) Mutations partitioned according to their origin, the number of altered residues, location within the complex, by the availability of additional kinetic and thermodynamic data, according to the experimental method used, and by category. (B) Distribution of ΔΔ<em>G</em>. (C) Source and target amino acids for single point mutations.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/06/07/341735/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of the interactions in SKEMPI. Nodes indicate proteins, scaled by the number of mutations of that protein and coloured according to category. Edges show direct interactions, as well as relationships between proteins that share a common or homologous binding site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1344929061" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Overview of the interactions in SKEMPI. Nodes indicate proteins, scaled by the number of mutations of that protein and coloured according to category. Edges show direct interactions, as well as relationships between proteins that share a common or homologous binding site.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/06/07/341735/F2.medium.gif" width="301" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2018/06/07/341735/F2.medium.gif" width="301" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/06/07/341735/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/06/07/341735/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2.</span> <p id="p-19" class="first-child">Overview of the interactions in SKEMPI. Nodes indicate proteins, scaled by the number of mutations of that protein and coloured according to category. Edges show direct interactions, as well as relationships between proteins that share a common or homologous binding site.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">The entries also vary in the degree of structural order. While most correspond to interactions between folded domains, the database contains entries in which structuring occurs upon binding, such as protein-peptide interactions and, in the extreme case, the ACTR/NCBD interaction in which both binding partners become ordered upon binding [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>]. Indeed, the requirements of having a structure in order to be included in the data set, <em>ipso facto</em> biases the data, and means that there is no representation of “fuzzy” complexes in which a diffuse structural ensemble in the bound state prevents the formation of a resolvable crystal.</p><p id="p-21">Another source of variation is the origins of the interacting proteins. While entries range from viral and bacterial to the higher eukaryotes, biases are evident in the over-representation of model organisms including humans. With the exception of the pMHC-TCR, antibody-antigen and protease-inhibitor classes, most of the interactions are endogenous. Nevertheless, the set also includes exogenous interactions ranging from those between proteins from different individuals within the same species, namely the sex fusion proteins Juno and Izumo1 from human sperm and egg respectively [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>], to host-pathogen interactions such as adenovirus and coronavirus interactions with human receptors during viral entry [<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>], [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>], to the inhibition of acetyl-cholinesterase by the snake venom neurotoxin fasciculin [<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>]. For the pMHC-TCR interactions there are a variety of presented antigens, including exogenous viral peptides and gluten, as well as endogenous autoimmune and cancer peptides. The antibody interactions include pathogen antibodies, as well as antibodies raised and optimised to target extracellular therapeutic targets. The protease interactions are mostly exogenous, arising from their inherent cross-reactivity due to the convergent evolution of their canonical inhibitory loop.</p></div><div id="sec-14" class="subsection"><h3>B. Notable studies comprising SKEMPI 2.0</h3><p id="p-22">The investigations from which SKEMPI data is derived are diverse, spanning many biological processes and reported in 295 publications including systematic scans, alanine and homolog scanning, design studies including computational design and designs derived from phage display, double mutant cycle studies, antibody engineering, biologic drug design and the evaluation of pathological mutations. The largest contribution comes from the group of the late Michael Laskowski Jr., a systematic study of all possible mutations at selected sites in the turkey ovomucoid third domain and its inhibitory interactions with four proteases [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>], as well as studies of interactions of the same domain in other bird species and the design of ultra-high affinity broad-spectrum inhibitors. Substantial data also come from investigations into the inhibitory interactions of class A <em>β</em>-lactamases from the groups of Gideon Schreiber (e.g. [<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>]) and Timothy Palzkill (e.g. [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]), as well as cytokine receptor interactions, in particular studies of type I interferons also from the Schreiber group (e.g. [<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>]) and that of K. Christopher Garcia [<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>], but also the study of the GM-CSF / GMR<em>α</em> interaction from the group of Michael W. Parker [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. Other prominent sources of data are studies into hormone receptor interactions, in particular the human growth hormone receptor from the group of Jim Wells (e.g. [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]) and the prolactin receptor from the group of Michael E. Hodsdon [<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>], as well as studies into antigen recognition including the combined computational and experimental design study to enhance affinity of the AQC2 antibody to integrin <em>α</em>-1 from the group of Herman Van Vlijmen [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>], the dissection of the interactions of broadly neutralising antibodies targeting HIV gp120 [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>] from the group of Richard A. Friesner, and various investigations from the group of Roy A. Mariuzza (e.g. [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]). Also notable is the alanine scanning of the urokinase-type plasminogen activator and its receptor from the group of Michael Ploug [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>], studies of Ras effector interactions from the group of Christian Herrmann (e.g. [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]), investigations of pMHC/TCR interactions from the group of Brian Baker (e.g. [<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>]), and investigations into the congate and non-cognate recognition of <em>E. coli</em> colicin DNase bacteriotoxins by their immunity proteins from the group of Colin Kleanthous (e.g. [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>]).</p></div><div id="sec-15" class="subsection"><h3>C. Range and error</h3><div id="sec-16" class="subsection"><h4>Range</h4><p id="p-23">The changes in binding free energy upon mutation range from −12.4 to 12.4 kcal.mol<sup>−1</sup>, as in SKEMPI 1.1, with Δlog<sub>10</sub> <em>k</em><sub>on</sub> ranging from −3.6 to 2.4, Δlog<sub>10</sub> <em>k</em><sub>off</sub> ranging from −6.0 to 6.8, ΔΔ<em>H</em> ranging from −18.3 to 26.5 kcal.mol<sup>−1</sup>, and ΔΔ<em>S</em> ranging from −61 to 80 cal.mol<sup>−1</sup>.K<sup>−1</sup>. Around 60 mutant are very destabilising, reducing binding energy by 8 kcal.mol<sup>−1</sup> or more. These are all in enzyme/inhibitor complexes such as the inhibition of acetylcholinesterase by the snake venom fasciculin, or the inhibition of enzymes which would be detrimental should they unbind and become active in the wrong location, such as nucleases (barnase / barstar, colicin E9 DNase / Im9, RNase A / angiogenin) and proteases (such as trypsin / BPTI). These interactions tend to be around picomolar affinity and are at the upper limit of what can be detected, due to the time required to reach equilibrium and the low concentrations required by the mass action law to probe informative regions of the binding curve. These very destabilising mutations reduce affinity into the micromolar range, near the lower limit of what can be quantified using standard methods. As a consequence of both mutant and wild-type affinities being near detection thresholds, errors in these entries are typically large. Further, while some mutations may cause changes in affinity larger than seven orders of magnitude, the absence of affinities for such mutations in the benchmark can be explained by the fact that such mutations would involve affinities beyond the upper or lower limit. Indeed, there are new entries in which single or double substitutions reduce binding from tens of picomolar to having no detectable binding. For many of the highly destabilising mutations a crystal structure for the mutant has also been solved, and the 30 most stabilising mutations in the database (ΔΔ<em>G</em> ¡ −5 kcal.mol<sup>−1</sup>) consist of the reverse mutation applied to these structures. These are mostly single or double mutants, but include mutations to up to 27 residues of the non-cognate Colicin E2/Im9 complex, which move it toward the cognate E9/Im9 in sequence space [<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>].</p></div><div id="sec-17" class="subsection"><h4>Errors</h4><p id="p-24">Standard errors in <em>K<sub>D</sub></em> are typically reported in the order of 50%, around 0.25 kcal.mol<sup>−1</sup>. These estimates are derived by repeat measurements using the same equipment, environment and protocol, and thus do not include errors arising from systematic bias. Such biases can, however, be estimated from pairs of entries in which the same mutation is evaluated by different groups or using different techniques. For 84% of 1741 such pairs, both entries give a ΔΔ<em>G</em> value within 1 kcal.mol<sup>−1</sup> of each other. For 704 pairs for which kon is available for both, 80% have Δlog<sub>10</sub> (<em>k</em><sub>on</sub>) within 0.5 of each other. For 702 pairs for which ko <sub>ff</sub> is available, 83% have Δlog<sub>10</sub> (<em>k</em><sub>off</sub>) within 0.5. For the 62 pair with both ΔΔ<em>H</em> and ΔΔ<em>S</em> values, 61% have ΔΔ<em>H</em> within 3.0 kcal.mol<sup>−1</sup> of each other and 58% have ΔΔ<em>S</em> within 10 cal.mol<sup>−1</sup>.K<sup>−1</sup> of each other.</p></div></div><div id="sec-18" class="subsection"><h3>D. Mutant Cycles</h3><p id="p-25">Within SKEMPI some entries can be combined to construct mutant cycles, which quantify the interactions between residues, the dependence of these interactions on other residues, and other higher order effects. The most common instances are double mutant cycles, where affinities are available for the wild type, A, B and AB mutations, of which there are 610 examples. Of these, 53 involve at least one mutant for which binding was not observed, or only an inequality is available, and 235 involve mutations reported in the same reference, and thus the affinities are likely to have been measured using the same technique and conditions. A further 218 double mutant cycles can be constructed in the background of a third mutation (i.e. C, AC, BC and ABC mutations are available), of which 209 are not composed of non-binding mutations or mutations with inequalities, and 131 involve affinities coming from the same reference. Of the 766 double mutant cycles containing neither inequalities nor non-binding mutants, a number of parameters can be calculated, including ΔΔ<em>G<sub>ab</sub></em><sub>→</sub> <em><sub>Ab</sub></em>, ΔΔ<em>G<sub>ab→aB</sub></em> and ΔΔ<em>G<sub>ab</sub></em><sub>→</sub> <em><sub>AB</sub></em>, the binding free energy change of both single and the double mutation respectively, as well as ΔΔ<em>G<sub>aB</sub></em><sub>→</sub> <em><sub>AB</sub></em> and ΔΔ<em>G<sub>Ab</sub></em><sub>→</sub> <em><sub>AB</sub></em>, the energy of a single mutation within the context of the other mutation, and ΔΔ<em>G<sub>int</sub></em> = ΔΔ<em>G<sub>aB</sub></em><sub>→</sub> <em><sub>AB</sub></em> − ΔΔ<em>G<sub>ab</sub></em><sub>→</sub> <em><sub>Ab</sub></em> = ΔΔ<em>G<sub>Ab</sub></em><sub>→</sub> <em><sub>AB</sub></em> − ΔΔ<em>G<sub>ab</sub></em><sub>→</sub> <em><sub>aB</sub></em>, the interaction energy of the two mutations [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>]. From these, it can be deduced that 345 are additive (ΔΔ<em>G<sub>int</sub></em> &lt; 0.5kcal.mol<sup>−1</sup>). Of the non-additive cycles, 293 exhibit tighter binding in the double mutant than the sum of the single mutants (positive epistasis), of which 6 result in even tighter binding than individual effects of two single mutations that strengthen the interaction (synergistic positive), while 273 correspond to double mutants which reduce binding by less than the sum of two single mutants which reduce binding (antagonistic positive). Similarly, 128 cycles have double mutants exhibiting weaker binding than the sum of the two single mutants (negative epistasis), of which 58 contain two destabilising single mutations (synergistic negative) and 26 contain two stabilising single mutations (antagonistic negative). The range of ΔΔ<em>G<sub>int</sub></em> values rarely fall outside of the −5 to 3 kcal.mol<sup>−1</sup> range. Of the 421 non-additive cycles, 151 show noticeable sign epistasis, in which the sign of the effect of either the A or B mutation flips depending on the presence or absence of the background mutation (i.e., for the A mutation, |ΔΔ<em>G<sub>ab</sub></em><sub>→<em>Ab</em></sub>| &gt; 0.2 kcal.mol<sup>−1</sup> and |ΔΔ<em>G<sub>aB</sub></em><sub>→<em>AB</em></sub>| &gt; 0.2 kcal.mol<sup>−1</sup> and |ΔΔ<em>G<sub>ab</sub></em><sub>→<em>Ab</em></sub> − ΔΔ<em>G<sub>aB</sub></em><sub>→<em>AB</em></sub>| &gt; 0.4 kcal.mol<sup>−–1</sup>). Of these, 38 correspond to mutations which destabilise the complex in the presence of the background mutation, but stabilise in its absence (destabilising sign epistasis), while 113 correspond to mutations which stabilise the complex in the mutant background but otherwise destabilise the complex (stabilising sign epistasis). Only 8 cycles exhibit the more extreme reciprocal sign epistasis, which in 6 cases are where both single mutations are stabilising (&lt; –0.2 kcal.mol<sup>−1</sup>), but the double mutant is destabilising (&gt; 0.2 kcal.mol<sup>−1</sup>), and the remaining two correspond to two destabilising mutations (&gt; 0.2 kcal.mol<sup>−1</sup>) for which the double mutation is stabilising (&lt; –0.2 kcal.mol<sup>−1</sup>). The types of substitutions that can give rise to extreme effects such as stabilising reciprocal sign epistasis can be illustrated with the Mlc-IIB<sup>Glc</sup> interaction in <em>E. coli</em> [<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>], in which the removal of the F136 side-chain of MlC creates a large cavity at the binding interface, the addition of a phenylalanine at the A451 position of IIBG<sup>l</sup>c creates a large clash, however the double mutation creates an interaction that is even more stable than the wild-type by creating an anchor residue across the binding interface in which the cavity in MlC is filled by the new side-chain of IIB.</p><p id="p-26">Higher order interaction terms can be garnered from higher cycles, such as triple mutant cubes, constructed from the energies of the wild-type, three single mutants, three corresponding double mutants and the triple mutant [<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>]. In SKEMPI, 45 triple mutant cubes can be made, with 10 coming from the same reference. For these, third order interaction energies fall within the −1 to 1 kcal.mol<sup>−1</sup> range. For fourth order interactions, constructed from energies of the wild-type, four single mutants, six double mutants, four triple mutants and the quadruple mutant, 14 examples exist within SKEMPI. However, care should be taken in ascribing meaning to fourth order residue coupling energies due to the accumulations of errors, which in these cases are exacerbated by the affinities having been reported in different publications. No fifth or higher order interactions are present.</p></div><div id="sec-19" class="subsection"><h3>E. The SKEMPI website</h3><p id="p-27">The database is accessible online at <a href="https://life.bsc.es/pid/skempi2/">https://life.bsc.es/pid/skempi2/</a>, where the raw CSV (comma-separated values) file containing all the data can be downloaded. The data can also be browsed online, ordered and searched by any field, such as the experimental method or the location of the mutation, or searching for a specific protein by its name or PDB code, and structures may be visualised. Other pages on the web site offer a summary of the data, an FAQ and help page, and a page for user contributions which will be evaluated to appear in future releases. Before you begin to format your paper, first write and save the content as a separate text file. Keep your text and graphic files separate until after the text has been formatted and styled. Do not use hard tabs, and limit use of hard returns to only one return at the end of a paragraph. Do not add any kind of pagination anywhere in the paper. Do not number text heads-the template will do that for you.</p><p id="p-28">Finally, complete content and organizational editing before formatting. Please take note of the following items when proofreading spelling and grammar:</p></div><div id="sec-20" class="subsection"><h3>F. Abbreviations and Acronyms</h3><p id="p-29">Define abbreviations and acronyms the first time they are used in the text, even after they have been defined in the abstract. Abbreviations such as IEEE, SI, MKS, CGS, sc, dc, and rms do not have to be defined. Do not use abbreviations in the title or heads unless they are unavoidable.</p></div></div><div class="section" id="sec-21"><h2 class="">FUNDING</h2><p id="p-30">This work has been supported by the European Molecular Biology Laboratory [IHM]; Biotechnology and Biological Sciences Research Council [Future Leader Fellowship BB/N011600/1 to IHM]; Spanish Ministry of Economy and Competitiveness (MINECO) [BIO2016-79930-R to JFR]; Interreg POCTEFA [EFA086/15 to JFR]; European Commission [H2020 grant 676566 (MuG)].</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">[1].</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.1" data-doi="10.1371/journal.pcbi.1003216"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Agius</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Torchala</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. H.</span> <span class="cit-name-surname">Moal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fernandez-Recio</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Bates</span></span>. <span class="cit-article-title">Characterizing changes in the rate of protein-protein dissociation upon interface mutation using hotspot energy and organization</span>. <abbr class="cit-jnl-abbrev">PLoS Comput. Biol.</abbr>, <span class="cit-vol">9</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">e1003216</span>, <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput.%2BBiol.%26rft.volume%253D9%26rft.spage%253De1003216%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003216%26rft_id%253Dinfo%253Apmid%252F24039569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24039569&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[2].</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.2" data-doi="10.1016/j.str.2014.01.012"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Aizner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Sharabi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Shirian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. R.</span> <span class="cit-name-surname">Dakwar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Risman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Avraham</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Shifman</span></span>. <span class="cit-article-title">Mapping of the binding landscape for a pico-molar protein-protein complex through computation and experiment</span>. <abbr class="cit-jnl-abbrev">Structure</abbr>, <span class="cit-vol">22</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">636</span>–<span class="cit-lpage">645</span>, Apr <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%26rft.volume%253D22%26rft.spage%253D636%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2014.01.012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.str.2014.01.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[3].</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.3" data-doi="10.1074/jbc.M116.756718"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Arkadash</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Yosef</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Shirian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Cohen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Horev</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Grossman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Sagi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Radisky</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Shifman</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Papo</span></span>. <span class="cit-article-title">Development of High Affinity and High Specificity Inhibitors of Matrix Metalloproteinase 14 through Computational Design and Directed Evolution</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr>, <span class="cit-vol">292</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">3481</span>–<span class="cit-lpage">3495</span>, Feb <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M116.756718%26rft_id%253Dinfo%253Apmid%252F28087697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyOTIvOC8zNDgxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDYvMDcvMzQxNzM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[4].</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.4" data-doi="10.1038/nature18595"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Aydin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sultana</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Thavalingam</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Lee</span></span>. <span class="cit-article-title">Molecular architecture of the human sperm IZUMO1 and egg JUNO fertilization complex</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">534</span>(<span class="cit-issue">7608</span>):<span class="cit-fpage">562</span>–<span class="cit-lpage">565</span>, 06 <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D534%26rft.spage%253D562%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature18595%26rft_id%253Dinfo%253Apmid%252F27309818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature18595&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27309818&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[5].</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.5"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Barlow</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. O</span> <span class="cit-name-surname">Conchuir</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Thompson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Suresh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Lucas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Heinonen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kortemme</span></span>. <span class="cit-article-title">Flex ddG: Rosetta Ensemble-Based Estimation of Changes in Protein-Protein Binding Affinity upon Mutation</span>. <abbr class="cit-jnl-abbrev">J. Phys. Chem. B</abbr>, in press, Feb <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[6].</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.6"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Barradas-Bautista</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. H.</span> <span class="cit-name-surname">Moal</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fernandez-Recio</span></span>. <span class="cit-article-title">A systematic analysis of scoring functions in rigid-body protein docking: The delicate balance between the predictive rate improvement and the risk of overtraining</span>. <abbr class="cit-jnl-abbrev">Proteins</abbr>, <span class="cit-vol">85</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">1287</span>–<span class="cit-lpage">1297</span>, Jul <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%26rft.volume%253D85%26rft.spage%253D1287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[7].</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.7" data-doi="10.1371/journal.pone.0107353"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Berliner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Teyra</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Colak</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Garcia Lopez</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Kim</span></span>. <span class="cit-article-title">Combining structural modeling with ensemble machine learning to accurately predict protein fold stability and binding affinity effects upon mutation</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>, <span class="cit-vol">9</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">e107353</span>, <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D9%26rft.spage%253De107353%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0107353%26rft_id%253Dinfo%253Apmid%252F25243403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0107353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25243403&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[8].</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.8" data-doi="10.1093/nar/28.1.235"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H. M.</span> <span class="cit-name-surname">Berman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Westbrook</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Feng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Gilliland</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. N.</span> <span class="cit-name-surname">Bhat</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Weissig</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. N.</span> <span class="cit-name-surname">Shindyalov</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P. E.</span> <span class="cit-name-surname">Bourne</span></span>. <span class="cit-article-title">The Protein Data Bank</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, <span class="cit-vol">28</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">235</span>–<span class="cit-lpage">242</span>, Jan <span class="cit-pub-date">2000</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBerman%26rft.auinit1%253DH.%2BM.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D235%26rft.epage%253D242%26rft.atitle%253DThe%2BProtein%2BData%2BBank%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.235%26rft_id%253Dinfo%253Apmid%252F10592235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/28.1.235&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10592235&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084896300069&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[9].</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.9"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Broughton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. R.</span> <span class="cit-name-surname">Hercus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">Nero</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Dottore</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">McClure</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Dhagat</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Taing</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Gorman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">King-Scott</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. F.</span> <span class="cit-name-surname">Lopez</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Parker</span></span>. <span class="cit-article-title">Conformational Changes in the GM-CSF Receptor Suggest a Molecular Mechanism for Affinity Conversion and Receptor Signaling</span>. <abbr class="cit-jnl-abbrev">Structure</abbr>, <span class="cit-vol">24</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">1271</span>–<span class="cit-lpage">1281</span>, 08 <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%26rft.volume%253D24%26rft.spage%253D1271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[10].</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.10" data-doi="10.1074/jbc.M113.463521"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N. G.</span> <span class="cit-name-surname">Brown</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">Chow</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Ruprecht</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Palzkill</span></span>. <span class="cit-article-title">Identification of the <em>β</em>-lactamase inhibitor protein-II (BLIP-II)interface residues essential for binding affinity and specificity for class A <em>β</em>-lactamases</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr>, <span class="cit-vol">288</span>(<span class="cit-issue">24</span>):<span class="cit-fpage">17156</span>–<span class="cit-lpage">17166</span>, Jun <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M113.463521%26rft_id%253Dinfo%253Apmid%252F23625930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODgvMjQvMTcxNTYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNi8wNy8zNDE3MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[11].</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.11" data-doi="10.1016/j.jmb.2016.11.021"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Clark</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Gindin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Abel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. S.</span> <span class="cit-name-surname">Murret</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Xu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Lu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Kwong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Shapiro</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Honig</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Friesner</span></span>. <span class="cit-article-title">Free Energy Perturbation Calculation of Relative Binding Free Energy between Broadly Neutralizing Antibodies and the gp120 Glycoprotein of HIV-1</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr>, <span class="cit-vol">429</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">930</span>–<span class="cit-lpage">947</span>, Apr <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BBiol.%26rft.volume%253D429%26rft.spage%253D930%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2016.11.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2016.11.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[12].</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.12" data-doi="10.1110/ps.052030506"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L. A.</span> <span class="cit-name-surname">Clark</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Boriack-Sjodin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Eldredge</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Fitch</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Friedman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. J.</span> <span class="cit-name-surname">Hanf</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Jarpe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. F.</span> <span class="cit-name-surname">Liparoto</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lugovskoy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Miller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Rushe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Sherman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Simon</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Van Vlijmen</span></span>. <span class="cit-article-title">Affinity enhancement of an in vivo matured therapeutic antibody using structure-based computational design</span>. <abbr class="cit-jnl-abbrev">Protein Sci.</abbr>, <span class="cit-vol">15</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">949</span>–<span class="cit-lpage">960</span>, May <span class="cit-pub-date">2006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1110%252Fps.052030506%26rft_id%253Dinfo%253Apmid%252F16597831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1110/ps.052030506&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16597831&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237237800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[13].</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.13" data-doi="10.1073/pnas.1613122113"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Cohen-Khait</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Schreiber</span></span>. <span class="cit-article-title">Low-stringency selection of TEM1 for BLIP shows interface plasticity and selection for faster binders</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U.S.A.</abbr>, <span class="cit-vol">113</span>(<span class="cit-issue">52</span>):<span class="cit-fpage">14982</span>–<span class="cit-lpage">14987</span>, Dec <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1613122113%26rft_id%253Dinfo%253Apmid%252F27956635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzUyLzE0OTgyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDYvMDcvMzQxNzM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[14].</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.14" data-doi="10.1073/pnas.88.8.3407"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B. C.</span> <span class="cit-name-surname">Cunningham</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Wells</span></span>. <span class="cit-article-title">Rational design of receptor-specific variants of human growth hormone</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U.S.A.</abbr>, <span class="cit-vol">88</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">3407</span>–<span class="cit-lpage">3411</span>, Apr <span class="cit-pub-date">1991</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DCunningham%26rft.auinit1%253DB.%2BC.%26rft.volume%253D88%26rft.issue%253D8%26rft.spage%253D3407%26rft.epage%253D3411%26rft.atitle%253DRational%2Bdesign%2Bof%2Breceptor-specific%2Bvariants%2Bof%2Bhuman%2Bgrowth%2Bhormone.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.8.3407%26rft_id%253Dinfo%253Apmid%252F2014261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4OC84LzM0MDciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNi8wNy8zNDE3MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[15].</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.15" data-doi="10.1021/bi980148j"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Dall’Acqua</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. R.</span> <span class="cit-name-surname">Goldman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Lin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Teng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Tsuchiya</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Ysern</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. C.</span> <span class="cit-name-surname">Braden</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Smith-Gill</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Mariuzza</span></span>. <span class="cit-article-title">A mutational analysis of binding interactions in an antigen-antibody protein-protein complex</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr>, <span class="cit-vol">37</span>(<span class="cit-issue">22</span>):<span class="cit-fpage">7981</span>–<span class="cit-lpage">7991</span>, Jun <span class="cit-pub-date">1998</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D37%26rft.spage%253D7981%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi980148j%26rft_id%253Dinfo%253Apmid%252F9609690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi980148j&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9609690&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074061600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[16].</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.16" data-doi="10.1002/humu.22534"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Das</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. R.</span> <span class="cit-name-surname">Lee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sagar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Fragoza</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Liang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wei</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mort</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Stenson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. N.</span> <span class="cit-name-surname">Cooper</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Yu</span></span>. <span class="cit-article-title">Elucidating common structural features of human pathogenic variations using large-scale atomic-resolution protein networks</span>. <abbr class="cit-jnl-abbrev">Hum. Mutat.</abbr>, <span class="cit-vol">35</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">585</span>–<span class="cit-lpage">593</span>, May <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMutat.%26rft.volume%253D35%26rft.spage%253D585%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fhumu.22534%26rft_id%253Dinfo%253Apmid%252F24599843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/humu.22534&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24599843&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[17].</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.17" data-doi="10.1093/nar/gkt450"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Dehouck</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Kwasigroch</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Rooman</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gilis</span></span>. <span class="cit-article-title">BeAt-MuSiC: Prediction of changes in protein-protein binding affinity on mutations</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, <span class="cit-vol">41</span>(<span class="cit-issue">Web Server issue</span>):<span class="cit-fpage">W333</span>–<span class="cit-lpage">339</span>, Jul <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt450%26rft_id%253Dinfo%253Apmid%252F23723246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt450&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23723246&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323603200053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[18].</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.18" data-doi="10.1002/prot.24634"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Dourado</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Flores</span></span>. <span class="cit-article-title">A multiscale approach to predicting affinity changes in protein-protein interfaces</span>. <abbr class="cit-jnl-abbrev">Proteins</abbr>, <span class="cit-vol">82</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">2681</span>–<span class="cit-lpage">2690</span>, Oct <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%26rft.volume%253D82%26rft.spage%253D2681%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fprot.24634%26rft_id%253Dinfo%253Apmid%252F24975440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/prot.24634&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24975440&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[19].</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.19"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">V. C.</span> <span class="cit-name-surname">Epa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Dolezal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Doughty</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Xiao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Jost</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Pluckthun</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">T. E.</span> <span class="cit-name-surname">Adams</span></span>. <span class="cit-article-title">Structural model for the interaction of a designed Ankyrin Repeat Protein with the human epidermal growth factor receptor 2</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>, <span class="cit-vol">8</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">e59163</span>, <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De59163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[20].</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.20" data-doi="10.1126/science.1202617"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Fleishman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. A.</span> <span class="cit-name-surname">Whitehead</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">Ekiert</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Dreyfus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Corn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Strauch</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. A.</span> <span class="cit-name-surname">Wilson</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Baker</span></span>. <span class="cit-article-title">Computational design of proteins targeting the conserved stem region of influenza hemagglutinin</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">332</span>(<span class="cit-issue">6031</span>):<span class="cit-fpage">816</span>–<span class="cit-lpage">821</span>, May <span class="cit-pub-date">2011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFleishman%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D332%26rft.issue%253D6031%26rft.spage%253D816%26rft.epage%253D821%26rft.atitle%253DComputational%2BDesign%2Bof%2BProteins%2BTargeting%2Bthe%2BConserved%2BStem%2BRegion%2Bof%2BInfluenza%2BHemagglutinin%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1202617%26rft_id%253Dinfo%253Apmid%252F21566186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzIvNjAzMS84MTYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNi8wNy8zNDE3MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[21].</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.21" data-doi="10.1074/jbc.M513583200"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Gardsvoll</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Gilquin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. H.</span> <span class="cit-name-surname">Le Du</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Menez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. J.</span> <span class="cit-name-surname">J?rgensen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ploug</span></span>. <span class="cit-article-title">Characterization of the functional epitope on the urokinase receptor. Complete alanine scanning mutagenesis supplemented by chemical cross-linking</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr>, <span class="cit-vol">281</span>(<span class="cit-issue">28</span>):<span class="cit-fpage">19260</span>–<span class="cit-lpage">19272</span>, Jul <span class="cit-pub-date">2006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M513583200%26rft_id%253Dinfo%253Apmid%252F16672229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvMjgvMTkyNjAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNi8wNy8zNDE3MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[22].</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.22" data-doi="10.1093/protein/gzw020"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Geng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Vangone</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A. M. J. J.</span> <span class="cit-name-surname">Bonvin</span></span>. <span class="cit-article-title">Exploring the interplay between experimental methods and the performance of predictors of binding affinity change upon mutations in protein complexes</span>. <abbr class="cit-jnl-abbrev">Protein Eng. Des. Sel.</abbr>, <span class="cit-vol">29</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">291</span>–<span class="cit-lpage">299</span>, Aug <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BEng.%2BDes.%2BSel.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fprotein%252Fgzw020%26rft_id%253Dinfo%253Apmid%252F27284087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/protein/gzw020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27284087&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[23].</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.23" data-doi="10.1093/hmg/ddu321"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Gossage</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Pires</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Olivera-Nappa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Asenjo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bycroft</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">Blundell</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Eisen</span></span>. <span class="cit-article-title">An integrated computational approach can classify VHL missense mutations according to risk of clear cell renal carcinoma</span>. <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr>, <span class="cit-vol">23</span>(<span class="cit-issue">22</span>):<span class="cit-fpage">5976</span>–<span class="cit-lpage">5988</span>, Nov <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddu321%26rft_id%253Dinfo%253Apmid%252F24969085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/hmg/ddu321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24969085&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[24].</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.24" data-doi="10.1073/pnas.1006528107"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Guntas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Purbeck</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Kuhlman</span></span>. <span class="cit-article-title">Engineering a protein-protein interface using a computationally designed library</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U.S.A.</abbr>, <span class="cit-vol">107</span>(<span class="cit-issue">45</span>):<span class="cit-fpage">19296</span>–<span class="cit-lpage">19301</span>, Nov <span class="cit-pub-date">2010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1006528107%26rft_id%253Dinfo%253Apmid%252F20974935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzQ1LzE5Mjk2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDYvMDcvMzQxNzM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[25].</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.25" data-doi="10.1016/j.bmcl.2014.03.095"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Guo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Wisniewski</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Ji</span></span>. <span class="cit-article-title">Hot spot-based design of small-molecule inhibitors for protein-protein interactions</span>. <abbr class="cit-jnl-abbrev">Bioorg. Med. Chem. Lett.</abbr>, <span class="cit-vol">24</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">2546</span>–<span class="cit-lpage">2554</span>, Jun <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioorg.%2BMed.%2BChem.%2BLett.%26rft.volume%253D24%26rft.spage%253D2546%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bmcl.2014.03.095%26rft_id%253Dinfo%253Apmid%252F24751445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bmcl.2014.03.095&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24751445&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[26].</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.26" data-doi="10.1016/0022-2836(90)90275-Q"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Horovitz</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Fersht</span></span>. <span class="cit-article-title">Strategy for analysing the co-operativity of intramolecular interactions in peptides and proteins</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr>, <span class="cit-vol">214</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">613</span>–<span class="cit-lpage">617</span>, Aug <span class="cit-pub-date">1990</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DHorovitz%26rft.auinit1%253DA.%26rft.volume%253D214%26rft.issue%253D3%26rft.spage%253D613%26rft.epage%253D617%26rft.atitle%253DStrategy%2Bfor%2Banalysing%2Bthe%2Bco-operativity%2Bof%2Bintramolecular%2Binteractions%2Bin%2Bpeptides%2Band%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252890%252990275-Q%26rft_id%253Dinfo%253Apmid%252F2388258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(90)90275-Q&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2388258&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DU27400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[27].</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.27" data-doi="10.1074/jbc.M301492200"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Howitt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Bewley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Graziano</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Flanagan</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Freimuth</span></span>. <span class="cit-article-title">Structural basis for variation in adenovirus affinity for the cellular coxsackievirus and adenovirus receptor</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr>, <span class="cit-vol">278</span>(<span class="cit-issue">28</span>):<span class="cit-fpage">26208</span>–<span class="cit-lpage">26215</span>, Jul <span class="cit-pub-date">2003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M301492200%26rft_id%253Dinfo%253Apmid%252F12716886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzgvMjgvMjYyMDgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNi8wNy8zNDE3MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[28].</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.28"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Chen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhang</span></span>. <span class="cit-article-title">Conservation of hot regions in protein-protein interaction in evolution</span>. <abbr class="cit-jnl-abbrev">Methods</abbr>, <span class="cit-vol">110</span>:<span class="cit-fpage">73</span>–<span class="cit-lpage">80</span>, Nov <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%26rft.volume%253D110%26rft.spage%253D73%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[29].</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.29" data-doi="10.1093/nar/gkl1063"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Huang</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Brutlag</span></span>. <span class="cit-article-title">Dynamic use of multiple parameter sets in sequence alignment</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, <span class="cit-vol">35</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">678</span>–<span class="cit-lpage">686</span>, <span class="cit-pub-date">2007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl1063%26rft_id%253Dinfo%253Apmid%252F17182633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl1063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17182633&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243993600038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[30].</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.30"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Hwang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Vreven</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Weng</span></span>. <span class="cit-article-title">Binding interface prediction by combining protein-protein docking results</span>. <abbr class="cit-jnl-abbrev">Proteins</abbr>, <span class="cit-vol">82</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">57</span>–<span class="cit-lpage">66</span>, Jan <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%26rft.volume%253D82%26rft.spage%253D57%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[31].</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.31"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Jemimah</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Yugandhar</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Michael Gromiha</span></span>. <span class="cit-article-title">PROXiMATE: a database of mutant protein-protein complex thermodynamics and kinetics</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">33</span>(<span class="cit-issue">17</span>):<span class="cit-fpage">2787</span>–<span class="cit-lpage">2788</span>, Sep <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D2787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[32].</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [32] in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.32" data-doi="10.1074/jbc.M113.537068"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Jemth</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Mu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Engstrom</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dogan</span></span>. <span class="cit-article-title">A frustrated binding interface for intrinsically disordered proteins</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr>, <span class="cit-vol">289</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">5528</span>–<span class="cit-lpage">5533</span>, Feb <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M113.537068%26rft_id%253Dinfo%253Apmid%252F24421312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODkvOS81NTI4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDYvMDcvMzQxNzM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[33].</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [33] in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.33" data-doi="10.1002/pro.580"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P. L.</span> <span class="cit-name-surname">Kastritis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. H.</span> <span class="cit-name-surname">Moal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Hwang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Weng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Bates</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Bonvin</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Janin</span></span>. <span class="cit-article-title">A structure-based benchmark for protein-protein binding affinity</span>. <abbr class="cit-jnl-abbrev">Protein Sci.</abbr>, <span class="cit-vol">20</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">482</span>–<span class="cit-lpage">491</span>, Mar <span class="cit-pub-date">2011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%26rft.stitle%253DProtein%2BSci.%26rft.aulast%253DKastritis%26rft.auinit1%253DP.%2BL.%26rft.volume%253D20%26rft.issue%253D3%26rft.spage%253D482%26rft.epage%253D491%26rft.atitle%253DA%2Bstructure-based%2Bbenchmark%2Bfor%2Bprotein-protein%2Bbinding%2Baffinity.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpro.580%26rft_id%253Dinfo%253Apmid%252F21213247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pro.580&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21213247&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288136900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[34].</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.34" data-doi="10.1016/j.jmb.2004.05.050"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kiel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Serrano</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Herrmann</span></span>. <span class="cit-article-title">A detailed thermodynamic analysis of ras/effector complex interfaces</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr>, <span class="cit-vol">340</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">1039</span>–<span class="cit-lpage">1058</span>, Jul <span class="cit-pub-date">2004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DKiel%26rft.auinit1%253DC.%26rft.volume%253D340%26rft.issue%253D5%26rft.spage%253D1039%26rft.epage%253D1058%26rft.atitle%253DA%2Bdetailed%2Bthermodynamic%2Banalysis%2Bof%2Bras%252Feffector%2Bcomplex%2Binterfaces.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2004.05.050%26rft_id%253Dinfo%253Apmid%252F15236966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2004.05.050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15236966&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222760300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[35].</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [35] in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.35" data-doi="10.1074/jbc.M110.172072"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Kulkarni</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Tettamanzi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Murphy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Keeler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. G.</span> <span class="cit-name-surname">Myszka</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. E.</span> <span class="cit-name-surname">Chayen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Lolis</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Hodsdon</span></span>. <span class="cit-article-title">Two independent histidines, one in human prolactin and one in its receptor, are critical for pH-dependent receptor recognition and activation</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr>, <span class="cit-vol">285</span>(<span class="cit-issue">49</span>):<span class="cit-fpage">38524</span>–<span class="cit-lpage">38533</span>, Dec <span class="cit-pub-date">2010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M110.172072%26rft_id%253Dinfo%253Apmid%252F20889499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvNDkvMzg1MjQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNi8wNy8zNDE3MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[36].</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference [36] in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.36" data-doi="10.1093/nar/gkj017"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Kumar</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">M. M.</span> <span class="cit-name-surname">Gromiha</span></span>. <span class="cit-article-title">PINT: Protein-protein Interactions Thermodynamic Database</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, <span class="cit-vol">34</span>(<span class="cit-issue">Database issue</span>):<span class="cit-fpage">D195</span>–<span class="cit-lpage">198</span>, Jan <span class="cit-pub-date">2006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKumar%26rft.auinit1%253DM.%2BD.%2BS.%26rft.volume%253D34%26rft.issue%253Dsuppl_1%26rft.spage%253DD195%26rft.epage%253DD198%26rft.atitle%253DPINT%253A%2BProtein-protein%2BInteractions%2BThermodynamic%2BDatabase%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkj017%26rft_id%253Dinfo%253Apmid%252F16381844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkj017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16381844&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239307700041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[37].</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [37] in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.37"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Lai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ambia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Barth</span></span>. <span class="cit-article-title">Enhancing Structure Prediction and Design of Soluble and Membrane Proteins with Explicit Solvent-Protein Interactions</span>. <abbr class="cit-jnl-abbrev">Structure</abbr>, <span class="cit-vol">25</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">1758</span>–<span class="cit-lpage">1770</span>, Nov <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%26rft.volume%253D25%26rft.spage%253D1758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[38].</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference [38] in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.38" data-doi="10.1016/j.jmb.2010.09.028"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E. D.</span> <span class="cit-name-surname">Levy</span></span>. <span class="cit-article-title">A simple definition of structural regions in proteins and its use in analyzing interface evolution</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr>, <span class="cit-vol">403</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">660</span>–<span class="cit-lpage">670</span>, Nov <span class="cit-pub-date">2010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DLevy%26rft.auinit1%253DE.%2BD.%26rft.volume%253D403%26rft.issue%253D4%26rft.spage%253D660%26rft.epage%253D670%26rft.atitle%253DA%2Bsimple%2Bdefinition%2Bof%2Bstructural%2Bregions%2Bin%2Bproteins%2Band%2Bits%2Buse%2Bin%2Banalyzing%2Binterface%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2010.09.028%26rft_id%253Dinfo%253Apmid%252F20868694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2010.09.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20868694&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284440500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[39].</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference [39] in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.39"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. L.</span> <span class="cit-name-surname">Simonetti</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Goncearenco</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Panchenko</span></span>. <span class="cit-article-title">MutaBind estimates and interprets the effects of sequence variants on protein-protein interactions</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, <span class="cit-vol">44</span>(<span class="cit-issue">W1</span>):<span class="cit-fpage">494</span>–<span class="cit-lpage">501</span>, Jul <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D44%26rft.spage%253D494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[40].</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference [40] in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.40" data-doi="10.1074/jbc.272.35.22253"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Dennis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. R.</span> <span class="cit-name-surname">Moore</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">James</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kleanthous</span></span>. <span class="cit-article-title">Protein-protein interaction specificity of Im9 for the endonuclease toxin colicin E9 defined by homologue-scanning mutagenesis</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr>, <span class="cit-vol">272</span>(<span class="cit-issue">35</span>):<span class="cit-fpage">22253</span>–<span class="cit-lpage">22258</span>, Aug <span class="cit-pub-date">1997</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D272%26rft.issue%253D35%26rft.spage%253D22253%26rft.epage%253D22258%26rft.atitle%253DProtein-Protein%2BInteraction%2BSpecificity%2Bof%2BIm9%2Bfor%2Bthe%2BEndonuclease%2BToxin%2BColicin%2BE9%2BDefined%2Bby%2BHomologue-scanning%2BMutagenesis%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.272.35.22253%26rft_id%253Dinfo%253Apmid%252F9268373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzIvMzUvMjIyNTMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNi8wNy8zNDE3MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[41].</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [41] in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.41"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Wang</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Li</span></span>. <abbr class="cit-jnl-abbrev">dbMPIKT: A web resource for the kinetic and thermodynamic database of mutant protein interactions</abbr>. <em>arXiv:1708.01857</em>, <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[42].</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference [42] in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.42"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ren</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Song</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Li</span></span>. <span class="cit-article-title">Co-Occurring Atomic Contacts for the Characterization of Protein Binding Hot Spots</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>, <span class="cit-vol">10</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">e0144486</span>, <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D10%26rft.spage%253De0144486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[43].</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference [43] in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.43" data-doi="10.1073/pnas.98.4.1410"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Lu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Lu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Qasim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Anderson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Apostol</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Ardelt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Bigler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y. W.</span> <span class="cit-name-surname">Chiang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cook</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. N.</span> <span class="cit-name-surname">James</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Kato</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kelly</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Kohr</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Komiyama</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. Y.</span> <span class="cit-name-surname">Lin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ogawa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Otlewski</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Park</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Qasim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ranjbar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Tashiro</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Warne</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Whatley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wieczorek</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wieczorek</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Wilusz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Wynn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zhang</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Laskowski</span></span>. <span class="cit-article-title">Predicting the reactivity of proteins from their sequence alone: Kazal family of protein inhibitors of serine proteinases</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U.S.A.</abbr>, <span class="cit-vol">98</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">1410</span>–<span class="cit-lpage">1415</span>, Feb <span class="cit-pub-date">2001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLu%26rft.auinit1%253DS.%2BM.%26rft.volume%253D98%26rft.issue%253D4%26rft.spage%253D1410%26rft.epage%253D1415%26rft.atitle%253DPredicting%2Bthe%2Breactivity%2Bof%2Bproteins%2Bfrom%2Btheir%2Bsequence%2Balone%253A%2BKazal%2Bfamily%2Bof%2Bprotein%2Binhibitors%2Bof%2Bserine%2Bproteinases%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.98.4.1410%26rft_id%253Dinfo%253Apmid%252F11171964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OC80LzE0MTAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNi8wNy8zNDE3MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[44].</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [44] in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.44"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Melo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Fieldhouse</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Melo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Correia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. N.</span> <span class="cit-name-surname">Cordeiro</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z. H.</span> <span class="cit-name-surname">Gumu?</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Costa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Bonvin</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">I. S.</span> <span class="cit-name-surname">Moreira</span></span>. <span class="cit-article-title">A Machine Learning Approach for Hot-Spot Detection at Protein-Protein Interfaces</span>. <abbr class="cit-jnl-abbrev">Int. J. Mol. Sci.</abbr>, <span class="cit-vol">17</span>(<span class="cit-issue">8</span>), Jul <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[45].</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference [45] in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.45"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I. H.</span> <span class="cit-name-surname">Moal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Barradas-Bautista</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Jimenez-Garcia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Torchala</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">van der Velde</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Vreven</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Weng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Bates</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fernandez-Recio</span></span>. <span class="cit-article-title">IRaPPA: information retrieval based integration of biophysical models for protein assembly selection</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">33</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">1806</span>–<span class="cit-lpage">1813</span>, Jun <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D1806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[46].</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference [46] in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.46"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I. H.</span> <span class="cit-name-surname">Moal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dapk?nas</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fernandez-Recio</span></span>. <span class="cit-article-title">Inferring the microscopic surface energy of protein-protein interfaces from mutation data</span>. <abbr class="cit-jnl-abbrev">Proteins</abbr>, <span class="cit-vol">83</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">640</span>–<span class="cit-lpage">650</span>, Apr <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%26rft.volume%253D83%26rft.spage%253D640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[47].</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference [47] in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.47" data-doi="10.1093/bioinformatics/bts489"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I. H.</span> <span class="cit-name-surname">Moal</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fernandez-Recio</span></span>. <span class="cit-article-title">SKEMPI: a Structural Kinetic and Energetic database of Mutant Protein Interactions and its use in empirical models</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">28</span>(<span class="cit-issue">20</span>):<span class="cit-fpage">2600</span>–<span class="cit-lpage">2607</span>, Oct <span class="cit-pub-date">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts489%26rft_id%253Dinfo%253Apmid%252F22859501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts489&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22859501&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309881200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[48].</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference [48] in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.48" data-doi="10.1021/ct400295z"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I. H.</span> <span class="cit-name-surname">Moal</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fernandez-Recio</span></span>. <span class="cit-article-title">Intermolecular Contact Potentials for Protein-Protein Interactions Extracted from Binding Free Energy Changes upon Mutation</span>. <abbr class="cit-jnl-abbrev">J. Chem. Theory Comput.</abbr>, <span class="cit-vol">9</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">3715</span>–<span class="cit-lpage">3727</span>, Aug <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BTheory%2BComput.%26rft.volume%253D9%26rft.spage%253D3715%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fct400295z%26rft_id%253Dinfo%253Apmid%252F26584123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ct400295z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26584123&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[49].</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference [49] in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.49" data-doi="10.1093/bioinformatics/btu594"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I. H.</span> <span class="cit-name-surname">Moal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Jimenez-Garcia</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fernandez-Recio</span></span>. <span class="cit-article-title">CCharPPI web server: computational characterization of protein-protein interactions from structure</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">31</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">123</span>–<span class="cit-lpage">125</span>, Jan <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu594%26rft_id%253Dinfo%253Apmid%252F25183488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu594&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25183488&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[50].</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference [50] in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.50" data-doi="10.1186/1471-2105-14-286"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I. H.</span> <span class="cit-name-surname">Moal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Torchala</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Bates</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fernandez-Recio</span></span>. <span class="cit-article-title">The scoring of poses in protein-protein docking: current capabilities and future directions</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>, <span class="cit-vol">14</span>:<span class="cit-fpage">286</span>, Oct <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D14%26rft.spage%253D286%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-14-286%26rft_id%253Dinfo%253Apmid%252F24079540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-14-286&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24079540&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[51].</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference [51] in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.51" data-doi="10.1002/prot.24356"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Moretti</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Fleishman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Agius</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Torchala</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Bates</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. L.</span> <span class="cit-name-surname">Kastritis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Rodrigues</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Trellet</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Bonvin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Rooman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gillis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Dehouck</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Moal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Romero-Durana</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Perez-Cano</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Pallara</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Jimenez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fernandez-Recio</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Flores</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pacella</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Praneeth Kilambi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Gray</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Popov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Grudinin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Esquivel-Rodriguez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Kihara</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Korkin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O. N.</span> <span class="cit-name-surname">Demerdash</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Mitchell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Kanamori</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Tsuchiya</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Nakamura</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Park</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Seok</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sarmiento</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Liang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Teraguchi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Standley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Shimoyama</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Terashi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Takeda-Shitaka</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Iwadate</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Umeyama</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Beglov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Hall</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Kozakov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Vajda</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. G.</span> <span class="cit-name-surname">Pierce</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Hwang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Vreven</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Weng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Ji</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Xiao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Zacharias</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Qin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. X.</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. Y.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Velankar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Janin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Wodak</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Baker</span></span>. <span class="cit-article-title">Community-wide evaluation of methods for predicting the effect of mutations on protein-protein interactions</span>. <abbr class="cit-jnl-abbrev">Proteins</abbr>, <span class="cit-vol">81</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">1980</span>–<span class="cit-lpage">1987</span>, Nov <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%26rft.volume%253D81%26rft.spage%253D1980%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fprot.24356%26rft_id%253Dinfo%253Apmid%252F23843247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/prot.24356&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23843247&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325980300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[52].</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference [52] in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.52" data-doi="10.1073/pnas.0709295105"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T. W.</span> <span class="cit-name-surname">Nam</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. I.</span> <span class="cit-name-surname">Jung</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y. J.</span> <span class="cit-name-surname">An</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y. H.</span> <span class="cit-name-surname">Park</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Lee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y. J.</span> <span class="cit-name-surname">Seok</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. S.</span> <span class="cit-name-surname">Cha</span></span>. <span class="cit-article-title">Analyses of Mlc-IIBGlc interaction and a plausible molecular mechanism of Mlc inactivation by membrane sequestration</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U.S.A.</abbr>, <span class="cit-vol">105</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">3751</span>–<span class="cit-lpage">3756</span>, Mar <span class="cit-pub-date">2008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0709295105%26rft_id%253Dinfo%253Apmid%252F18319344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzEwLzM3NTEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNi8wNy8zNDE3MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[53].</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference [53] in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.53" data-doi="10.1002/prot.24387"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Pallara</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Jimenez-Garcia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Perez-Cano</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Romero-Durana</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Solernou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Grosdidier</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Pons</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. H.</span> <span class="cit-name-surname">Moal</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fernandez-Recio</span></span>. <span class="cit-article-title">Expanding the frontiers of protein-protein modeling: from docking and scoring to binding affinity predictions and other challenges</span>. <abbr class="cit-jnl-abbrev">Proteins</abbr>, <span class="cit-vol">81</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">2192</span>–<span class="cit-lpage">2200</span>, Dec <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%26rft.volume%253D81%26rft.spage%253D2192%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fprot.24387%26rft_id%253Dinfo%253Apmid%252F23934865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/prot.24387&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23934865&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327344300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[54].</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference [54] in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.54"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Pantazes</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Grisewood</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. P.</span> <span class="cit-name-surname">Gifford</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Maranas</span></span>. <span class="cit-article-title">The Iterative Protein Redesign and Optimization (IPRO) suite of programs</span>. <abbr class="cit-jnl-abbrev">J. Comput. Chem.</abbr>, <span class="cit-vol">36</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">251</span>–<span class="cit-lpage">263</span>, Feb <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComput.%2BChem.%26rft.volume%253D36%26rft.spage%253D251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[55].</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference [55] in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.55"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Peng</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Alexov</span></span>. <span class="cit-article-title">Investigating the linkage between disease-causing amino acid variants and their effect on protein stability and binding</span>. <abbr class="cit-jnl-abbrev">Proteins</abbr>, <span class="cit-vol">84</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">232</span>–<span class="cit-lpage">239</span>, Feb <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%26rft.volume%253D84%26rft.spage%253D232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[56].</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference [56] in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.56" data-doi="10.1002/humu.22770"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Petukh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. G.</span> <span class="cit-name-surname">Kucukkal</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Alexov</span></span>. <span class="cit-article-title">On human disease-causing amino acid variants: statistical study of sequence and structural patterns</span>. <abbr class="cit-jnl-abbrev">Hum. Mutat.</abbr>, <span class="cit-vol">36</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">524</span>–<span class="cit-lpage">534</span>, May <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMutat.%26rft.volume%253D36%26rft.spage%253D524%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fhumu.22770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/humu.22770&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[57].</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference [57] in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.57" data-doi="10.1371/journal.pcbi.1004276"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Petukh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Li</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Alexov</span></span>. <span class="cit-article-title">Predicting Binding Free Energy Change Caused by Point Mutations with Knowledge-Modified MM/PBSA Method</span>. <abbr class="cit-jnl-abbrev">PLoS Comput. Biol.</abbr>, <span class="cit-vol">11</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">e1004276</span>, Jul <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput.%2BBiol.%26rft.volume%253D11%26rft.spage%253De1004276%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1004276%26rft_id%253Dinfo%253Apmid%252F26146996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1004276&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26146996&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[58].</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference [58] in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.58"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Pfeiffenberger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Chaleil</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. H.</span> <span class="cit-name-surname">Moal</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Bates</span></span>. <span class="cit-article-title">A machine learning approach for ranking clusters of docked protein-protein complexes by pairwise cluster comparison</span>. <abbr class="cit-jnl-abbrev">Proteins</abbr>, <span class="cit-vol">85</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">528</span>–<span class="cit-lpage">543</span>, Mar <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%26rft.volume%253D85%26rft.spage%253D528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[59].</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference [59] in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.59" data-doi="10.1038/ncomms2948"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K. H.</span> <span class="cit-name-surname">Piepenbrink</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Blevins</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Scott</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B. M.</span> <span class="cit-name-surname">Baker</span></span>. <span class="cit-article-title">The basis for limited specificity and MHC restriction in a T cell receptor interface</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr>, <span class="cit-vol">4</span>:<span class="cit-fpage">1948</span>, <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D4%26rft.spage%253D1948%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms2948%26rft_id%253Dinfo%253Apmid%252F23736024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms2948&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23736024&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[60].</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference [60] in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.60" data-doi="10.1093/bioinformatics/btt691"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Pires</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Ascher</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">Blundell</span></span>. <span class="cit-article-title">mCSM: predicting the effects of mutations in proteins using graph-based signatures</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">30</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">335</span>–<span class="cit-lpage">342</span>, Feb <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt691%26rft_id%253Dinfo%253Apmid%252F24281696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt691&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24281696&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331271100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[61].</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference [61] in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.61"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Plattner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Doerr</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">De Fabritiis</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Noe</span></span>. <span class="cit-article-title">Complete protein-protein association kinetics in atomic detail revealed by molecular dynamics simulations and Markov modelling</span>. <abbr class="cit-jnl-abbrev">Nat Chem</abbr>, <span class="cit-vol">9</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">1005</span>–<span class="cit-lpage">1011</span>, Oct <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BChem%26rft.volume%253D9%26rft.spage%253D1005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[62].</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference [62] in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.62" data-doi="10.1016/j.jmb.2006.09.076"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Reichmann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cohen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Abramovich</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Dym</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Lim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. C.</span> <span class="cit-name-surname">Strynadka</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Schreiber</span></span>. <span class="cit-article-title">Binding hot spots in the TEM1-BLIP interface in light of its modular architecture</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr>, <span class="cit-vol">365</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">663</span>–<span class="cit-lpage">679</span>, Jan <span class="cit-pub-date">2007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DReichmann%26rft.auinit1%253DD.%26rft.volume%253D365%26rft.issue%253D3%26rft.spage%253D663%26rft.epage%253D679%26rft.atitle%253DBinding%2Bhot%2Bspots%2Bin%2Bthe%2BTEM1-BLIP%2Binterface%2Bin%2Blight%2Bof%2Bits%2Bmodular%2Barchitecture.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2006.09.076%26rft_id%253Dinfo%253Apmid%252F17070843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2006.09.076&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17070843&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243561500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[63].</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference [63] in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.63" data-doi="10.1073/pnas.221290398"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L. C.</span> <span class="cit-name-surname">Roisman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Piehler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. Y.</span> <span class="cit-name-surname">Trosset</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. A.</span> <span class="cit-name-surname">Scheraga</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Schreiber</span></span>. <span class="cit-article-title">Structure of the interferon-receptor complex determined by distance constraints from double-mutant cycles and flexible docking</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U.S.A.</abbr>, <span class="cit-vol">98</span>(<span class="cit-issue">23</span>):<span class="cit-fpage">13231</span>–<span class="cit-lpage">13236</span>, Nov <span class="cit-pub-date">2001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DRoisman%26rft.auinit1%253DL.%2BC.%26rft.volume%253D98%26rft.issue%253D23%26rft.spage%253D13231%26rft.epage%253D13236%26rft.atitle%253DStructure%2Bof%2Bthe%2Binterferon-receptor%2Bcomplex%2Bdetermined%2Bby%2Bdistance%2Bconstraints%2Bfrom%2Bdouble-mutant%2Bcycles%2Band%2Bflexible%2Bdocking%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.221290398%26rft_id%253Dinfo%253Apmid%252F11698684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTgvMjMvMTMyMzEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNi8wNy8zNDE3MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[64].</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference [64] in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.64"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Rosenfeld</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Alcalay</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Mechaly</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Lapidoth</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Epstein</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kronman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. J</span> <span class="cit-name-surname">Fleishman</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Mazor</span></span>. <span class="cit-article-title">Improved antibody-based ricin neutralization by affinity maturation is correlated with slower off-rate values</span>. <abbr class="cit-jnl-abbrev">Protein Eng. Des. Sel.</abbr>, pages <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>, May <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[65].</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference [65] in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.65" data-doi="10.1074/jbc.M605316200"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Seiradake</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lortat-Jacob</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Billet</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Kremer</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Cusack</span></span>. <span class="cit-article-title">Structural and mutational analysis of human Ad37 and canine adenovirus 2 fiber heads in complex with the D1 domain of coxsackie and adenovirus receptor</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr>, <span class="cit-vol">281</span>(<span class="cit-issue">44</span>):<span class="cit-fpage">33704</span>–<span class="cit-lpage">33716</span>, Nov <span class="cit-pub-date">2006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M605316200%26rft_id%253Dinfo%253Apmid%252F16923808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvNDQvMzM3MDQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNi8wNy8zNDE3MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[66].</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference [66] in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.66"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I. C.</span> <span class="cit-name-surname">Simoes</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. P.</span> <span class="cit-name-surname">Costa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Coimbra</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Ramos</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Fernandes</span></span>. <span class="cit-article-title">New Parameters for Higher Accuracy in the Computation of Binding Free Energy Differences upon Alanine Scanning Mutagenesis on Protein-Protein Interfaces</span>. <abbr class="cit-jnl-abbrev">J. Chem. Inf. Model.</abbr>, <span class="cit-vol">57</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">60</span>–<span class="cit-lpage">72</span>, Jan <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BInf.%2BModel.%26rft.volume%253D57%26rft.spage%253D60%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[67].</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference [67] in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.67"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sirin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Apgar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Bennett</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Keating</span></span>. <span class="cit-article-title">AB-Bind: Antibody binding mutational database for computational affinity predictions</span>. <abbr class="cit-jnl-abbrev">Protein Sci.</abbr>, <span class="cit-vol">25</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">393</span>–<span class="cit-lpage">409</span>, Feb <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BSci.%26rft.volume%253D25%26rft.spage%253D393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[68].</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference [68] in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.68" data-doi="10.1042/BCJ20161078"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Stevers</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">de Vries</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">Doveston</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. G.</span> <span class="cit-name-surname">Milroy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Brunsveld</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Ottmann</span></span>. <span class="cit-article-title">Structural interface between LRRK2 and 14-3-3 protein</span>. <abbr class="cit-jnl-abbrev">Biochem. J.</abbr>, <span class="cit-vol">474</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">1273</span>–<span class="cit-lpage">1287</span>, Mar <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBCJ20161078%26rft_id%253Dinfo%253Apmid%252F28202711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6MTA6IjQ3NC83LzEyNzMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNi8wNy8zNDE3MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[69].</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference [69] in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.69"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Sumbul</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Acuner-Ozbabacan</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Haliloglu</span></span>. <span class="cit-article-title">Allosteric Dynamic Control of Binding</span>. <abbr class="cit-jnl-abbrev">Biophys. J.</abbr>, <span class="cit-vol">109</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">1190</span>–<span class="cit-lpage">1201</span>, Sep <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophys.%2BJ.%26rft.volume%253D109%26rft.spage%253D1190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[70].</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference [70] in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.70" data-doi="10.1016/j.cell.2011.06.048"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Thomas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Moraga</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Levin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. O.</span> <span class="cit-name-surname">Krutzik</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Podoplelova</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Trejo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Yarden</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Vleck</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Glenn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. P.</span> <span class="cit-name-surname">Nolan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Piehler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Schreiber</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">K. C.</span> <span class="cit-name-surname">Garcia</span></span>. <span class="cit-article-title">Structural linkage between ligand discrimination and receptor activation by type I interferons</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-vol">146</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">621</span>–<span class="cit-lpage">632</span>, Aug <span class="cit-pub-date">2011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DThomas%26rft.auinit1%253DC.%26rft.volume%253D146%26rft.issue%253D4%26rft.spage%253D621%26rft.epage%253D632%26rft.atitle%253DStructural%2Blinkage%2Bbetween%2Bligand%2Bdiscrimination%2Band%2Breceptor%2Bactivation%2Bby%2Btype%2BI%2Binterferons.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.06.048%26rft_id%253Dinfo%253Apmid%252F21854986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.06.048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21854986&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294043600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[71].</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference [71] in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.71" data-doi="10.1093/bioinformatics/17.3.284"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">Thorn</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Bogan</span></span>. <span class="cit-article-title">ASEdb: a database of alanine mutations and their effects on the free energy of binding in protein interactions</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">17</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">284</span>–<span class="cit-lpage">285</span>, Mar <span class="cit-pub-date">2001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DThorn%26rft.auinit1%253DK.%2BS.%26rft.volume%253D17%26rft.issue%253D3%26rft.spage%253D284%26rft.epage%253D285%26rft.atitle%253DASEdb%253A%2Ba%2Bdatabase%2Bof%2Balanine%2Bmutations%2Band%2Btheir%2Beffects%2Bon%2Bthe%2B%2Bfree%2Benergy%2Bof%2Bbinding%2Bin%2Bprotein%2Binteractions%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.3.284%26rft_id%253Dinfo%253Apmid%252F11294795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/17.3.284&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11294795&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168053800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[72].</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference [72] in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.72" data-doi="10.1074/jbc.M604725200"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Tidow</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Veprintsev</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Freund</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Fersht</span></span>. <span class="cit-article-title">Effects of oncogenic mutations and DNA response elements on the binding of p53 to p53-binding protein 2 (53BP2)</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr>, <span class="cit-vol">281</span>(<span class="cit-issue">43</span>):<span class="cit-fpage">32526</span>–<span class="cit-lpage">32533</span>, Oct <span class="cit-pub-date">2006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M604725200%26rft_id%253Dinfo%253Apmid%252F16887812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvNDMvMzI1MjYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNi8wNy8zNDE3MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[73].</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference [73] in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.73" data-doi="10.1016/j.jmb.2015.07.016"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Vreven</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. H.</span> <span class="cit-name-surname">Moal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Vangone</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. G.</span> <span class="cit-name-surname">Pierce</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. L.</span> <span class="cit-name-surname">Kastritis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Torchala</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Chaleil</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Jimenez-Garcia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Bates</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fernandez-Recio</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Bonvin</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Weng</span></span>. <span class="cit-article-title">Updates to the Integrated Protein-Protein Interaction Benchmarks: Docking Benchmark Version 5 and Affinity Benchmark Version 2</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr>, <span class="cit-vol">427</span>(<span class="cit-issue">19</span>):<span class="cit-fpage">3031</span>–<span class="cit-lpage">3041</span>, Sep <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BBiol.%26rft.volume%253D427%26rft.spage%253D3031%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2015.07.016%26rft_id%253Dinfo%253Apmid%252F26231283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2015.07.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26231283&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[74].</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference [74] in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.74" data-doi="10.1107/S0907444910045749"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Winn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Ballard</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. D.</span> <span class="cit-name-surname">Cowtan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Dodson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Emsley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. R.</span> <span class="cit-name-surname">Evans</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Keegan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. B.</span> <span class="cit-name-surname">Krissinel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Leslie</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">McCoy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">McNicholas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. N.</span> <span class="cit-name-surname">Murshudov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. S.</span> <span class="cit-name-surname">Pannu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Potterton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. R.</span> <span class="cit-name-surname">Powell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Read</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Vagin</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">Wilson</span></span>. <span class="cit-article-title">Overview of the CCP4 suite and current developments</span>. <abbr class="cit-jnl-abbrev">Acta Crystallogr. D Biol. Crystallogr.</abbr>, <span class="cit-vol">67</span>(<span class="cit-issue">Pt 4</span>):<span class="cit-fpage">235</span>–<span class="cit-lpage">242</span>, Apr <span class="cit-pub-date">2011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.stitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.aulast%253DWinn%26rft.auinit1%253DM.%2BD.%26rft.volume%253D67%26rft.issue%253DPt%2B4%26rft.spage%253D235%26rft.epage%253D242%26rft.atitle%253DOverview%2Bof%2Bthe%2BCCP4%2Bsuite%2Band%2Bcurrent%2Bdevelopments.%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0907444910045749%26rft_id%253Dinfo%253Apmid%252F21460441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1107/S0907444910045749&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21460441&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288532800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[75].</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference [75] in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.75" data-doi="10.1038/nature00920"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L. C.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Tuot</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Lyons</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. C.</span> <span class="cit-name-surname">Garcia</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M. M.</span> <span class="cit-name-surname">Davis</span></span>. <span class="cit-article-title">Two-step binding mechanism for T-cell receptor recognition of peptide MHC</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">418</span>(<span class="cit-issue">6897</span>):<span class="cit-fpage">552</span>–<span class="cit-lpage">556</span>, Aug <span class="cit-pub-date">2002</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWu%26rft.auinit1%253DL.%2BC.%26rft.volume%253D418%26rft.issue%253D6897%26rft.spage%253D552%26rft.epage%253D556%26rft.atitle%253DTwo-step%2Bbinding%2Bmechanism%2Bfor%2BT-cell%2Breceptor%2Brecognition%2Bof%2Bpeptide%2BMHC.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00920%26rft_id%253Dinfo%253Apmid%252F12152083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00920&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12152083&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177162800045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[76].</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference [76] in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.76"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Xiong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zheng</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>. <span class="cit-article-title">BindProfX: Assessing Mutation-Induced Binding Affinity Change by Protein Interface Profiles with Pseudo-Counts</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr>, <span class="cit-vol">429</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">426</span>–<span class="cit-lpage">434</span>, Feb <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BBiol.%26rft.volume%253D429%26rft.spage%253D426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[77].</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference [77] in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.77"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Yan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. G.</span> <span class="cit-name-surname">Ji</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. Z. H.</span> <span class="cit-name-surname">Zhang</span></span>. <span class="cit-article-title">Interaction Entropy for Computational Alanine Scanning</span>. <abbr class="cit-jnl-abbrev">J. Chem. Inf. Model.</abbr>, <span class="cit-vol">57</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">1112</span>–<span class="cit-lpage">1122</span>, May <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BInf.%2BModel.%26rft.volume%253D57%26rft.spage%253D1112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[78].</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference [78] in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-341735v1.78" data-doi="10.1371/journal.pcbi.1003592"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Han</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. R.</span> <span class="cit-name-surname">Shyu</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Korkin</span></span>. <span class="cit-article-title">Determining effects of non-synonymous SNPs on protein-protein interactions using supervised and semi-supervised learning</span>. <abbr class="cit-jnl-abbrev">PLoS Comput. Biol.</abbr>, <span class="cit-vol">10</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">e1003592</span>, May <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput.%2BBiol.%26rft.volume%253D10%26rft.spage%253De1003592%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003592%26rft_id%253Dinfo%253Apmid%252F24784581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003592&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24784581&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F06%2F07%2F341735.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/341735v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F341735v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/290957v2" title="IFITM3 restricts human metapneumovirus infection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;June 07, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/341735v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/341735v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/341735v1" class="active" data-icon-position="" data-hide-link-title="0">SKEMPI 2.0: An updated benchmark of changes in protein-protein binding energy, kinetics and thermodynamics upon mutation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/103964" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-sds9ovF18fsgD-CuBerYW3Uo03Ihwmz4-q5bRJTL9YY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="103964" id="node103964--2" data-pisa="biorxiv;341735v1" data-pisa-master="biorxiv;341735" data-seqnum="103964" data-apath="/biorxiv/early/2018/06/07/341735.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">SKEMPI 2.0: An updated benchmark of changes in protein-protein binding energy, kinetics and thermodynamics upon mutation</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Justina</span> <span class="nlm-surname">Jankauskaitė</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Brian</span> <span class="nlm-surname">Jiménez-García</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Justas</span> <span class="nlm-surname">Dapkūnas</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Juan</span> <span class="nlm-surname">Fernández-Recio</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Iain H.</span> <span class="nlm-surname">Moal</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">341735; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/341735 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/341735v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/341735v1&amp;title=SKEMPI%202.0%3A%20An%20updated%20benchmark%20of%20changes%20in%20protein-protein%20binding%20energy%2C%20kinetics%20and%20thermodynamics%20upon%20mutation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/341735v1&amp;title=SKEMPI%202.0%3A%20An%20updated%20benchmark%20of%20changes%20in%20protein-protein%20binding%20energy%2C%20kinetics%20and%20thermodynamics%20upon%20mutation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/341735v1&amp;text=SKEMPI%202.0%3A%20An%20updated%20benchmark%20of%20changes%20in%20protein-protein%20binding%20energy%2C%20kinetics%20and%20thermodynamics%20upon%20mutation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/341735v1&amp;title=SKEMPI%202.0%3A%20An%20updated%20benchmark%20of%20changes%20in%20protein-protein%20binding%20energy%2C%20kinetics%20and%20thermodynamics%20upon%20mutation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/341735v1&amp;t=SKEMPI%202.0%3A%20An%20updated%20benchmark%20of%20changes%20in%20protein-protein%20binding%20energy%2C%20kinetics%20and%20thermodynamics%20upon%20mutation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/341735v1&amp;title=SKEMPI%202.0%3A%20An%20updated%20benchmark%20of%20changes%20in%20protein-protein%20binding%20energy%2C%20kinetics%20and%20thermodynamics%20upon%20mutation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/341735v1&amp;title=SKEMPI%202.0%3A%20An%20updated%20benchmark%20of%20changes%20in%20protein-protein%20binding%20energy%2C%20kinetics%20and%20thermodynamics%20upon%20mutation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="103964" id="node103964--3" data-pisa="biorxiv;341735v1" data-pisa-master="biorxiv;341735" data-seqnum="103964" data-apath="/biorxiv/early/2018/06/07/341735.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">SKEMPI 2.0: An updated benchmark of changes in protein-protein binding energy, kinetics and thermodynamics upon mutation</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Justina</span> <span class="nlm-surname">Jankauskaitė</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Brian</span> <span class="nlm-surname">Jiménez-García</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Justas</span> <span class="nlm-surname">Dapkūnas</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Juan</span> <span class="nlm-surname">Fernández-Recio</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Iain H.</span> <span class="nlm-surname">Moal</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">341735; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/341735 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=SKEMPI%202.0%3A%20An%20updated%20benchmark%20of%20changes%20in%20protein-protein%20binding%20energy%2C%20kinetics%20and%20thermodynamics%20upon%20mutation&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Jankauskait%C4%97%2C+Justina&amp;rft.au=Jim%C3%A9nez-Garc%C3%ADa%2C+Brian&amp;rft.au=Dapk%C5%ABnas%2C+Justas&amp;rft.au=Fern%C3%A1ndez-Recio%2C+Juan&amp;rft.au=Moal%2C+Iain+H."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/103964/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/103964/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/103964/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/103964/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/103964/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/103964/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/103964/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/103964/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/103964/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/103964/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/103964/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/103964/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/341735v1&amp;count=horizontal&amp;via=&amp;text=SKEMPI%202.0%3A%20An%20updated%20benchmark%20of%20changes%20in%20protein-protein%20binding%20energy%2C%20kinetics%20and%20thermodynamics%20upon%20mutation&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/341735v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/341735v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/341735v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20277)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
