<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/31/2020.03.30.015164.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.30.015164v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Epitope-based chimeric peptide vaccine design against S, M and E proteins of SARS-CoV-2 etiologic agent of global pandemic COVID-19: an in silico approach" />
<meta name="DC.Identifier" content="10.1101/2020.03.30.015164" />
<meta name="DC.Date" content="2020-03-31" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of the ongoing pandemic of coronavirus disease 2019 (COVID-19), a public health emergency of international concern declared by the World Health Organization (WHO). An immuno-informatics approach along with comparative genomic was applied to design a multi-epitope-based peptide vaccine against SARS-CoV-2 combining the antigenic epitopes of the S, M and E proteins. The tertiary structure was predicted, refined and validated using advanced bioinformatics tools. The candidate vaccine showed an average of ≥ 90.0% world population coverage for different ethnic groups. Molecular docking of the chimeric vaccine peptide with the immune receptors (TLR3 and TLR4) predicted efficient binding. Immune simulation predicted significant primary immune response with increased IgM and secondary immune response with high levels of both IgG1 and IgG2. It also increased the proliferation of T-helper cells and cytotoxic T-cells along with the increased INF-γ and IL-2 cytokines. The codon optimization and mRNA secondary structure prediction revealed the chimera is suitable for high-level expression and cloning. Overall, the constructed recombinant chimeric vaccine candidate demonstrated significant potential and can be considered for clinical validation to fight against this global threat, COVID-19." />
<meta name="DC.Contributor" content="M. Shaminur Rahman" />
<meta name="DC.Contributor" content="M. Nazmul Hoque" />
<meta name="DC.Contributor" content="M. Rafiul Islam" />
<meta name="DC.Contributor" content="Salma Akter" />
<meta name="DC.Contributor" content="A. S. M. Rubayet-Ul-Alam" />
<meta name="DC.Contributor" content="Mohammad Anwar Siddique" />
<meta name="DC.Contributor" content="Otun Saha" />
<meta name="DC.Contributor" content="Md. Mizanur Rahaman" />
<meta name="DC.Contributor" content="Munawar Sultana" />
<meta name="DC.Contributor" content="M. Anwar Hossain" />
<meta name="article:published_time" content="2020-03-31" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Epitope-based chimeric peptide vaccine design against S, M and E proteins of SARS-CoV-2 etiologic agent of global pandemic COVID-19: an in silico approach" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of the ongoing pandemic of coronavirus disease 2019 (COVID-19), a public health emergency of international concern declared by the World Health Organization (WHO). An immuno-informatics approach along with comparative genomic was applied to design a multi-epitope-based peptide vaccine against SARS-CoV-2 combining the antigenic epitopes of the S, M and E proteins. The tertiary structure was predicted, refined and validated using advanced bioinformatics tools. The candidate vaccine showed an average of ≥ 90.0% world population coverage for different ethnic groups. Molecular docking of the chimeric vaccine peptide with the immune receptors (TLR3 and TLR4) predicted efficient binding. Immune simulation predicted significant primary immune response with increased IgM and secondary immune response with high levels of both IgG1 and IgG2. It also increased the proliferation of T-helper cells and cytotoxic T-cells along with the increased INF-γ and IL-2 cytokines. The codon optimization and mRNA secondary structure prediction revealed the chimera is suitable for high-level expression and cloning. Overall, the constructed recombinant chimeric vaccine candidate demonstrated significant potential and can be considered for clinical validation to fight against this global threat, COVID-19.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.30.015164v1" />
<meta name="citation_id" content="2020.03.30.015164v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.30.015164v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.30.015164v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.30.015164v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.30.015164" />
<meta name="citation_num_pages" content="66" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.30.015164" />
<meta name="citation_author" content="M. Shaminur Rahman" />
<meta name="citation_author_institution" content="Department of Microbiology, University of Dhaka" />
<meta name="citation_author" content="M. Nazmul Hoque" />
<meta name="citation_author_institution" content="Department of Microbiology, University of Dhaka" />
<meta name="citation_author_institution" content="Department of Gynecology, Obstetrics and Reproductive Health, Bangabandhu Sheikh Mujibur Rahman Agricultural University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4861-0030" />
<meta name="citation_author" content="M. Rafiul Islam" />
<meta name="citation_author_institution" content="Department of Microbiology, University of Dhaka" />
<meta name="citation_author" content="Salma Akter" />
<meta name="citation_author_institution" content="Department of Microbiology, University of Dhaka" />
<meta name="citation_author_institution" content="Department of Microbiology, Jahangirnagar University, Savar" />
<meta name="citation_author" content="A. S. M. Rubayet-Ul-Alam" />
<meta name="citation_author_institution" content="Department of Microbiology, University of Dhaka" />
<meta name="citation_author_institution" content="Department of Microbiology, Jashore University of Science and Technology" />
<meta name="citation_author" content="Mohammad Anwar Siddique" />
<meta name="citation_author_institution" content="Department of Microbiology, University of Dhaka" />
<meta name="citation_author" content="Otun Saha" />
<meta name="citation_author_institution" content="Department of Microbiology, University of Dhaka" />
<meta name="citation_author" content="Md. Mizanur Rahaman" />
<meta name="citation_author_institution" content="Department of Microbiology, University of Dhaka" />
<meta name="citation_author_email" content="hossaina@du.ac.bd" />
<meta name="citation_author_email" content="razu002@du.ac.bd" />
<meta name="citation_author" content="Munawar Sultana" />
<meta name="citation_author_institution" content="Department of Microbiology, University of Dhaka" />
<meta name="citation_author" content="M. Anwar Hossain" />
<meta name="citation_author_institution" content="Department of Microbiology, University of Dhaka" />
<meta name="citation_author_institution" content="Vice-Chancellor, Jashore University of Science and Technology" />
<meta name="citation_author_email" content="hossaina@du.ac.bd" />
<meta name="citation_author_email" content="razu002@du.ac.bd" />
<meta name="citation_reference" content="WHO, 2020. Novel Coronavirus (2019-nCoV) situation reports - World Health Organization (WHO)" />
<meta name="citation_reference" content="Almofti, Y. A., Abd-elrahman, K. A., Gassmallah, S. A. E. &amp; Salih, M. A. Multi Epitopes Vaccine Prediction against Severe Acute Respiratory Syndrome (SARS) Coronavirus Using Immunoinformatics Approaches. American J. Microbiol. Res. 6(3), 94–114 (2018)." />
<meta name="citation_reference" content="Wu, F. et al. “A new coronavirus associated with human respiratory disease in China.” Nat. 1–5 (2020)" />
<meta name="citation_reference" content="Badawi, M. M. et al. In Silico Prediction of a Novel Universal Multi-epitope Peptide Vaccine in the Whole Spike Glycoprotein of MERS CoV. Am. J. Microbiol Res. 4(4), 101–21 (2016)." />
<meta name="citation_reference" content="Pallesen, J. et al. Immunogenicity and structures of a rationally designed prefusion MERS-CoV spike antigen. Proc. Nat. Aca. Sci. 114(35), E7348–E7357 (2017)." />
<meta name="citation_reference" content="ul Qamar, M. T., Saleem, S., Ashfaq, U. A., Bari, A., Anwar, F. &amp; Alqahtani, S. Epitope based peptide vaccine design and target site depiction against Middle East Respiratory Syndrome Coronavirus: an immune-informatics study. J. Translational Med. 17(1), 362 (2019)." />
<meta name="citation_reference" content="Ahmed, S. F., Quadeer, A. A. &amp; McKay, M. R. Preliminary identification of potential vaccine targets for the COVID-19 coronavirus (SARS-CoV-2) based on SARS-CoV immunological studies. Viruses. 12(3), 254 (2020)." />
<meta name="citation_reference" content="Abdelmageed, M. I. et al. Design of multi epitope-based peptide vaccine against E protein of human 2019-nCoV: An immunoinformatics approach. BioRxiv. (2020)." />
<meta name="citation_reference" content="Song, W., Gui, M., Wang, X. &amp; Xiang, Y. Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2. PLoS Pathog. 14(8), e1007236 (2018)." />
<meta name="citation_reference" content="Wrapp, D. et al. Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science 2020." />
<meta name="citation_reference" content="Shang, W. et al. The outbreak of SARS-CoV-2 pneumonia calls for viral vaccines. npj Vaccines 5, 18 (2020)." />
<meta name="citation_reference" content="Gralinski, L. E. &amp; Menachery, V. D. Return of the coronavirus: 2019-nCoV. Viruses 12, 135 (2020)." />
<meta name="citation_reference" content="Zhou, H. et al. Structural definition of a neutralization epitope on the N-terminal domain of MERS-CoV spike glycoprotein. Nat. Commun. 10, 3068 (2019)." />
<meta name="citation_reference" content="Rabi, F. A., Al Zoubi, M. S., Kasasbeh, G. A., Salameh, D. M. &amp; Al-Nasser, A. D. SARS-CoV-2 and Coronavirus Disease 2019: What We Know So Far. Pathogens 9(3), 231 (2020)." />
<meta name="citation_reference" content="Wang, N. et al. Structural Definition of a Neutralization-sensitive Epitope on the MERS-CoV S1-NTD. Cell Rep. 28(13), 3395–405 (2019)." />
<meta name="citation_reference" content="Shi, S. Q. et al. The expression of membrane protein augments the specific responses induced by SARS-CoV nucleocapsid DNA immunization. Mol. Immunol. 43, 1791–1798 (2006)." />
<meta name="citation_reference" content="Schoeman, D. &amp; Fielding, B. C. Coronavirus envelope protein: current knowledge. Virol J. 16, 69 (2019)." />
<meta name="citation_reference" content="Chan, J. F. et al. Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan. Emerg. Microbes Infect. 9, 221–236 (2020)." />
<meta name="citation_reference" content="Li, G., et al. Coronavirus infections and immune responses. J. Med. Virology 92(4), 424–432 (2020)." />
<meta name="citation_reference" content="Shi, J. et al. Epitope-Based Vaccine Target Screening against Highly Pathogenic MERS-CoV: An In Silico Approach Applied to Emerging Infectious Diseases. PLoS One 10(12), e0144475 (2015)." />
<meta name="citation_reference" content="Shey, R. A. et al. In-silico design of a multi-epitope vaccine candidate against onchocerciasis and related filarial diseases. Sci Rep. 9(1), 1–18 (2019)." />
<meta name="citation_reference" content="Yong, et al. Recent Advances in the Vaccine Development Against Middle East Respiratory Syndrome-Coronavirus. Front. Microbiol. 10, 1781 (2019)." />
<meta name="citation_reference" content="Srivastava, S. et al. Design of novel multi-epitope vaccines against severe acute respiratory syndrome validated through multistage molecular interaction and dynamics. J. Biomol. Struc. Dyn. 37(16), 4345–4360 (2019)." />
<meta name="citation_reference" content="Sakib, M. S., Islam, M., Hasan, A. K. M. &amp; Nabi, A. H. M. Prediction of epitope-based peptides for the utility of vaccine development from fusion and glycoprotein of nipah virus using in silico approach. Adv. Bioinform. (2014)." />
<meta name="citation_reference" content="Adhikari, U. K., Tayebi, M. &amp; Rahman, M. M. Immunoinformatics approach for epitope-based peptide vaccine design and active site prediction against polyprotein of emerging oropouche virus. J. Immunol. Res. (2018)." />
<meta name="citation_reference" content="Hebditch, M., M. A. Carballo-Amador, S. Charonis, R. Curtis &amp; J. Warwicker. Protein– Sol: a web tool for predicting protein solubility from sequence. Bioinform. 33(19), 3098–3100 (2017)." />
<meta name="citation_reference" content="Van Gunsteren, W., S., et al. The GROMOS96 manual and user guide. (1996)." />
<meta name="citation_reference" content="Rapin, N., Lund, O, Bernaschi, M &amp; Castiglione, M. Computational immunology meets bioinformatics: the use of prediction tools for molecular binding in the simulation of the immune system. PLoS One 5(4) (2010)." />
<meta name="citation_reference" content="Biswal, J. K., Bisht, P., Mohapatra, J. K., Ranjan, R., Sanyal, A. &amp; Pattnaik, B. “Application of a recombinant capsid polyprotein (P1) expressed in a prokaryotic system to detect antibodies against foot-and-mouth disease virus serotype O.” J. Virol. Methods 215, 45–51 (2015)." />
<meta name="citation_reference" content="Tai, W. et al. Characterization of the receptor-binding domain (RBD) of 2019 novel coronavirus: implication for development of RBD protein as a viral attachment inhibitor and vaccine. Cell. Mol. Immunol. (2020)." />
<meta name="citation_reference" content="Zhou, P. et al. Discovery of a novel coronavirus associated with the recent pneumonia outbreak in humans and its potential bat origin. bioRxiv. (2020)." />
<meta name="citation_reference" content="Hui, D. S., et al. The continuing 2019-nCoV epidemic threat of novel coronaviruses to global health—The latest 2019 novel coronavirus outbreak in Wuhan, China. Int. J. Inft. Dis. 91, 264–266 (2020)." />
<meta name="citation_reference" content="Pei, H. et al. Expression of SARS-coronavirus nucleocapsid protein in Escherichia coli and Lactococcus lactis for serodiagnosis and mucosal vaccination. Applied Microbiol. Biotech. 68(2), 220–227 (2005)." />
<meta name="citation_reference" content="Pang, J. et al. Potential rapid diagnostics, vaccine and therapeutics for 2019 novel Coronavirus (2019-ncoV): a systematic review. J. Clinical Med. 9(3), 623 (2020)." />
<meta name="citation_reference" content="Frieman, M. et al. “SARS coronavirus and innate immunity.” Virus Research. 133(1), 101–112 (2018)." />
<meta name="citation_reference" content="Millet, J. K. &amp; Whittaker, G. R. “Host cell proteases: Critical determinants of coronavirus tropism and pathogenesis.” Virus Res. 202, 120–134 (2015)." />
<meta name="citation_reference" content="Jaimes, J. A., Andre, N. M., Millet, J. K. &amp; Whittaker, G. R. Structural modeling of 2019-novel coronavirus (nCoV) spike protein reveals a proteolytically-sensitive activation loop as a distinguishing feature compared to SARS-CoV and related SARS-like coronaviruses. arXiv. (2020)." />
<meta name="citation_reference" content="Huang, H. et al. CD4(+) Th1 cells promote CD8(+) Tc1 cell survival, memory response, tumor localization and therapy by targeted delivery of interleukin 2 via acquired pMHC I complexes. Immunology 120, 148–159 (2007)." />
<meta name="citation_reference" content="Mohammad, N. et al. In silico design of a chimeric protein containing antigenic fragments of Helicobacter pylori; a bioinformatic approach. The Open Microbiol. J. 10, 97 (2016)." />
<meta name="citation_reference" content="Meza, B., Ascencio, F., Sierra-Beltrán, A. P., Torres, J. &amp; Angulo, C. A novel design of a multi-antigenic, multistage and multi-epitope vaccine against Helicobacter pylori: an in silico approach. Infect. Genet. Evol. 49, 309–317 (2017)." />
<meta name="citation_reference" content="Totura, A. L. et al. Toll-like receptor 3 signaling via TRIF contributes to a protective innate immune response to severe acute respiratory syndrome coronavirus infection. MBio 6(3), e00638–15 (2015)." />
<meta name="citation_reference" content="Mosaddeghi, P., et al. Therapeutic approaches for COVID-19 based on the dynamics of interferon-mediated immune responses. (2020)." />
<meta name="citation_reference" content="Zander, R. A. et al. Th1-like plasmodium-specific memory CD4+ T cells support humoral immunity. Cell Rep. 21(7), 1839–1852 (2017)." />
<meta name="citation_reference" content="Carvalho, L. H. et al. IL-4-secreting CD4+ T cells are crucial to the development of CD8+ T-cell responses against malaria liver stages. Nat. Med. 8, 166–70 (2002)." />
<meta name="citation_reference" content="Hoque, M. N. et al. Metagenomic deep sequencing reveals association of microbiome signature with functional biases in bovine mastitis. Sci. Rep. 9, 13536 (2019)." />
<meta name="citation_reference" content="Waterhouse, A. et al. “SWISS-MODEL: homology modelling of protein structures and complexes.” Nucleic Acids Res. 46(W1), W296–W303 (2018)." />
<meta name="citation_reference" content="Guex, N. &amp; Peitsch, M. C. “SWISS-MODEL and the Swiss-Pdb Viewer: an environment for comparative protein modeling.” Electrophoresis 18(15), 2714–2723 (1997)." />
<meta name="citation_reference" content="Laskowski, R. A., MacArthur, M. W., Moss, D. S. &amp; Thornton, J. M. “PROCHECK: a program to check the stereochemical quality of protein structures.” J. Applied Crystal 26(2), 283–291 (1993)." />
<meta name="citation_reference" content="Faure, G. et al. iPBAvizu: a PyMOL plugin for an efficient 3D protein structure superimposition approach. Source Code Biol. Med. 14, 5 (2019)." />
<meta name="citation_reference" content="Gasteiger, E. et al. Protein identification and analysis tools on the ExPASy server. The Proteomics Protocols Handbook, Springer: 571–607 (2005)." />
<meta name="citation_reference" content="Kringelum, J. V., Lundegaard, C., Lund, O. &amp; Nielsen, M. Reliable B cell epitope predictions: Impacts of method development and improved benchmarking. PLoS Comput. Biol. 8(12), e1002829 (2012)." />
<meta name="citation_reference" content="Doytchinova, I. A. &amp; Flower, D. R. VaxiJen: a server for prediction of protective antigens, tumour antigens and subunit vaccines. BMC Bioinformatics 8(1), 4 (2007)." />
<meta name="citation_reference" content="Larsen, J. E. P., Lund, O. &amp; Nielsen, M. Improved method for predicting linear B-cell epitopes. Immunome Res. 2(1), 2 (2006)." />
<meta name="citation_reference" content="Saha, S. &amp; Raghava G. P. S. “Prediction of continuous BLJ using recurrent neural network.” Proteins: Structure, Function, Bioinformatics 65(1), 40–48 (2006)." />
<meta name="citation_reference" content="Kolaskar, A. &amp; Tongaonkar, P. C. “A semi empirical method for prediction of antigenic determinants on protein antigens.” FEBS Letters 276(1-2), 172–174 (1990)." />
<meta name="citation_reference" content="Xue, L. C., Rodrigues, J. P., Kastritis, P. L., Bonvin, A. M. &amp; Vangone, A. “PRODIGY: a web server for predicting the binding affinity of protein–protein complexes.” Bioinformatics 32(23), 3676–3678 (2016)." />
<meta name="citation_reference" content="Bui, H. H., Sidney, J., Dinh, K., Southwood, S., Newman, M. J &amp; Sette, A. Predicting population coverage of T-cell epitope-based diagnostics and vaccines. BMC Bioinformatics 7(1),153 (2006)." />
<meta name="citation_reference" content="Hoque, M. N. et al. Resistome diversity in bovine clinical mastitis microbiome, a signature concurrence. bioRxiv, 829283 (2019)." />
<meta name="citation_reference" content="Dhanda, S. K., Vir, P. &amp; Raghava, G. P. “Designing of interferon-gamma inducing MHC class-II binders.” Biology Direct 8(1), 30 (2013)." />
<meta name="citation_reference" content="Agadjanyan, M. G. et al. “Prototype Alzheimer’s disease vaccine using the β-amyloid and promiscuous T cell epitope pan HLA DR-binding peptide.” The J. Immunology 174(3), 1580–1586 (2005)." />
<meta name="citation_reference" content="Li, H. et al. Co-expression of the C-terminal domain of Yersinia enterocolitica invasin enhances the efficacy of classical swine-fever-vectored vaccine based on human adenovirus. J. Biosci. 40, 79–90 (2015)." />
<meta name="citation_reference" content="Källberg, M., Margaryan, G., Wang, S., Ma, J. &amp; Xu, J. RaptorX server: a resource for template-based protein structure modeling. In Protein Structure Prediction (pp. 17–27), Humana Press, New York, NY. 2014." />
<meta name="citation_reference" content="Xu, D. &amp; Zhang, Y. “Improving the physical realism and structural accuracy of protein models by a two-step atomic-level energy minimization.” Biophysical J. 101(10), 2525–2534 (2011)." />
<meta name="citation_reference" content="Lovell, S. C. et al. “Structure validation by Cα geometry □, ψ and Cβ deviation.” Proteins: Structure, Function, Bioinformatics 50(3), 437–450 (2003)." />
<meta name="citation_reference" content="De Vries, S. J., Van Dijk, M. &amp; Bonvin, A. M. The HADDOCK web server for data-driven biomolecular docking. Nat. Protoc. 5(5), 883 (2010)." />
<meta name="citation_reference" content="Morla, S., Makhija, A. &amp; Kumar, S. “Synonymous codon usage pattern in glycoprotein gene of rabies virus.” Gene 584(1), 1–6 (2016)." />
<meta name="twitter:title" content="Epitope-based chimeric peptide vaccine design against S, M and E proteins of SARS-CoV-2 etiologic agent of global pandemic COVID-19: an in silico approach" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of the ongoing pandemic of coronavirus disease 2019 (COVID-19), a public health emergency of international concern declared by the World Health Organization (WHO). An immuno-informatics approach along with comparative genomic was applied to design a multi-epitope-based peptide vaccine against SARS-CoV-2 combining the antigenic epitopes of the S, M and E proteins. The tertiary structure was predicted, refined and validated using advanced bioinformatics tools. The candidate vaccine showed an average of ≥ 90.0% world population coverage for different ethnic groups. Molecular docking of the chimeric vaccine peptide with the immune receptors (TLR3 and TLR4) predicted efficient binding. Immune simulation predicted significant primary immune response with increased IgM and secondary immune response with high levels of both IgG1 and IgG2. It also increased the proliferation of T-helper cells and cytotoxic T-cells along with the increased INF-γ and IL-2 cytokines. The codon optimization and mRNA secondary structure prediction revealed the chimera is suitable for high-level expression and cloning. Overall, the constructed recombinant chimeric vaccine candidate demonstrated significant potential and can be considered for clinical validation to fight against this global threat, COVID-19." />
<meta name="og-title" property="og:title" content="Epitope-based chimeric peptide vaccine design against S, M and E proteins of SARS-CoV-2 etiologic agent of global pandemic COVID-19: an in silico approach" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.30.015164v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of the ongoing pandemic of coronavirus disease 2019 (COVID-19), a public health emergency of international concern declared by the World Health Organization (WHO). An immuno-informatics approach along with comparative genomic was applied to design a multi-epitope-based peptide vaccine against SARS-CoV-2 combining the antigenic epitopes of the S, M and E proteins. The tertiary structure was predicted, refined and validated using advanced bioinformatics tools. The candidate vaccine showed an average of ≥ 90.0% world population coverage for different ethnic groups. Molecular docking of the chimeric vaccine peptide with the immune receptors (TLR3 and TLR4) predicted efficient binding. Immune simulation predicted significant primary immune response with increased IgM and secondary immune response with high levels of both IgG1 and IgG2. It also increased the proliferation of T-helper cells and cytotoxic T-cells along with the increased INF-γ and IL-2 cytokines. The codon optimization and mRNA secondary structure prediction revealed the chimera is suitable for high-level expression and cloning. Overall, the constructed recombinant chimeric vaccine candidate demonstrated significant potential and can be considered for clinical validation to fight against this global threat, COVID-19." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-31" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.30.015164v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.30.015164v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.30.015164v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.30.015164v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1219018" />
    <title>Epitope-based chimeric peptide vaccine design against S, M and E proteins of SARS-CoV-2 etiologic agent of global pandemic COVID-19: an in silico approach | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"woMi2ZfRQiYKJaV7GbUJldoRqF4ktO1-WpAX1nAmAyU","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1219018","apath":"\/biorxiv\/early\/2020\/03\/31\/2020.03.30.015164.atom","pisa":"biorxiv;2020.03.30.015164v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.30.015164v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.30.015164v1","title":"Epitope-based chimeric peptide vaccine design against S, M and E proteins of SARS-CoV-2 etiologic agent of global pandemic COVID-19: an in silico approach","identifier":"node\/1219018"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022N13vCpQM8uIDubY51C2mV3SQH6Hi91H4t6GR2BIGVxkG5bwTs+6yprqfWKSn2cZ7YdXIdeHVFAAjDHRhRJgHDhEYIflKb8LBYaNiyjNhv1IONaDVIy31Ff1eeaVdW6gKIty8DEgFuzXk\\\\\\\/nSZiG+5nV4PpDQe9hQP2h2EzciDO6ccaUui7HqpjkzIPfhLwwyNAAj2yPn3TQtG5HLKWOJsASQHzLq34ePwAtfV1KxffeldSM2rvzQZIrT6htCjSkJdQ8eAbMObYhdtxRA+cjxoBDpetoW16imZl7arYM5DP8kRDOwV7I70av3VgWjMDVGbJC3T7l44VJ4P5Bs+DSBBYtl5feCQWN9herY\\\\\\\/6y7kF8cjfTsQmSXr9qjX\\\\\\\/J3FOBsnzKLDDkpnZzcfPLqjmyWE+5uc5gY4Xq3xoYoBaj0V8isQ3DzPHA6gbLydVQ014nQF8Ig95N\\\\\\\/RzQjgjAUQmozOSHF2wWV\\\\\\\/mn6Jv0QQnvPeqRrEjA2vivrZyrtgm5G1xzei7oFdVHYam\\\\\\\/bs2M9iAalJnQLR8QiH5vtVPjPfxm5DaPoKgUq\\\\\\\/vZSBcTLAUkcXOU\\\\\\\/t19AbA1oyH5LSBiredRs0V8B+1YMFl4snJ20VFqjfRXED3ArAWDFmmFSwW1Y+b+RwccqqiMx0I9csvbsS\\\\\\\/ddFgyWv7GNMQ0mw7XOYqxmW4uF3OcnGd+OnbUfgQ24DYWgVTaGNKSHDiH53gutXwPGksXCydOSVDc+oe087a\\\\\\\/TiZ3JTIvNI4a1Al6aBNtjBSvUEvWVb8j22XYberrVEhsOaXlcOgyLmZPG1Oc6GP5GxPwdGMkIqVE2oWtNj1qcDjbldPKbt203dnh+uGZrFGYY7Fga+xIqyFzHKTcv7q\\\\\\\/vu+FxH6f3qJpOlDCZbimkwUHeK5AbGtA+5CsWQEWppTX0uEo+TZPMaiwZuDNw56OcXXxc7L2gP6kSbqWhaBQg2AgB2qxn3i+f7ohBnNQnOlWLjG\\\\\\\/oDzcFei50GP5VvVHBDq7kzt+HcjIljCkBh\\\\\\\/a0wzMlqgLnUz2N2NVPqsMJV8tGUlFr\\\\\\\/f3Xfjs+dX5XGj5nNVNv6uVGGW05S7tSKxW1BFwagZ7Zv1jJUm5bAmJqA3U3+lyADhBqoqNPdXJuzYsT9AKy7m5QVR8G\\\\\\\/y7Z0HRmnpjUDUKUwSjERk0NUf0wJc\\\\\\\/ZfxPG9QptQT0ZrDAm6PQDcMW6MhS+vUh1GZyNvHK1GqR5EuH\\\\\\\/16phgK7zFnptPPqz2F1yUA1Xa\\\\\\\/KHvFZfGqL347gJusLtL96iB0wozpWF6CK7+2H8gED\\\\\\\/09qn7BsDmfGM4mMm91CzEv7scg\\\\\\\/aqSnYcNnlPGQ+8+e5mGo8HNh70z\\\\\\\/xA\\\u0022,\\\u0022iv\\\u0022:\\\u00224MTMY24tw7fMgut7UVpvxg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002272d91714c5fcc420b647c1d2ea6913bfa1e2b69756c1cd3f3213dea0e77e6ed0\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.30.015164v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1219018 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.30.015164v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YaGq9elbKxe1vcYxyxqDlUrjeEZT4SYQZWrhlDsXBWo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1219018" id="node1219018" data-pisa="biorxiv;2020.03.30.015164v1" data-pisa-master="biorxiv;2020.03.30.015164" data-apath="/biorxiv/early/2020/03/31/2020.03.30.015164.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Epitope-based chimeric peptide vaccine design against S, M and E proteins of SARS-CoV-2 etiologic agent of global pandemic COVID-19: an <em>in silico</em> approach</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">M. Shaminur</span> <span class="nlm-surname">Rahman</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4861-0030" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">M. Nazmul</span> <span class="nlm-surname">Hoque</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">M. Rafiul</span> <span class="nlm-surname">Islam</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Salma</span> <span class="nlm-surname">Akter</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">A. S. M.</span> <span class="nlm-surname">Rubayet-Ul-Alam</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Mohammad Anwar</span> <span class="nlm-surname">Siddique</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Otun</span> <span class="nlm-surname">Saha</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Md. Mizanur</span> <span class="nlm-surname">Rahaman</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Munawar</span> <span class="nlm-surname">Sultana</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">M. Anwar</span> <span class="nlm-surname">Hossain</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.30.015164 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1219018" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">M. Shaminur Rahman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology, University of Dhaka</span>, Dhaka 1000, <span class='nlm-country'>Bangladesh</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BShaminur%2BRahman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rahman%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BShaminur%2BRahman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">M. Nazmul Hoque </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology, University of Dhaka</span>, Dhaka 1000, <span class='nlm-country'>Bangladesh</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Gynecology, Obstetrics and Reproductive Health, Bangabandhu Sheikh Mujibur Rahman Agricultural University</span>, Gazipur-1706, <span class='nlm-country'>Bangladesh</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BNazmul%2BHoque%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoque%20MN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BNazmul%2BHoque%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4861-0030" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for M. Nazmul Hoque</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">M. Rafiul Islam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology, University of Dhaka</span>, Dhaka 1000, <span class='nlm-country'>Bangladesh</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BRafiul%2BIslam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Islam%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BRafiul%2BIslam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Salma Akter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology, University of Dhaka</span>, Dhaka 1000, <span class='nlm-country'>Bangladesh</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Microbiology, Jahangirnagar University, Savar</span>, Dhaka-1342, <span class='nlm-country'>Bangladesh</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Salma%2BAkter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Akter%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASalma%2BAkter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">A. S. M. Rubayet-Ul-Alam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology, University of Dhaka</span>, Dhaka 1000, <span class='nlm-country'>Bangladesh</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Microbiology, Jashore University of Science and Technology</span>, Jashore 7408, <span class='nlm-country'>Bangladesh</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=A.%2BS.%2BM.%2BRubayet-Ul-Alam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rubayet-Ul-Alam%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AA.%2BS.%2BM.%2BRubayet-Ul-Alam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Mohammad Anwar Siddique </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology, University of Dhaka</span>, Dhaka 1000, <span class='nlm-country'>Bangladesh</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mohammad%2BAnwar%2BSiddique%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Siddique%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMohammad%2BAnwar%2BSiddique%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Otun Saha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology, University of Dhaka</span>, Dhaka 1000, <span class='nlm-country'>Bangladesh</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Otun%2BSaha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saha%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOtun%2BSaha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Md. Mizanur Rahaman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology, University of Dhaka</span>, Dhaka 1000, <span class='nlm-country'>Bangladesh</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Md.%2BMizanur%2BRahaman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rahaman%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMd.%2BMizanur%2BRahaman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hossaina@du.ac.bd" class="" data-icon-position="" data-hide-link-title="0">hossaina@du.ac.bd</a>
<a href="mailto:razu002@du.ac.bd" class="" data-icon-position="" data-hide-link-title="0">razu002@du.ac.bd</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Munawar Sultana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology, University of Dhaka</span>, Dhaka 1000, <span class='nlm-country'>Bangladesh</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Munawar%2BSultana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sultana%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMunawar%2BSultana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">M. Anwar Hossain </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology, University of Dhaka</span>, Dhaka 1000, <span class='nlm-country'>Bangladesh</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Vice-Chancellor, Jashore University of Science and Technology</span>, Jashore 7408, <span class='nlm-country'>Bangladesh</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BAnwar%2BHossain%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hossain%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BAnwar%2BHossain%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hossaina@du.ac.bd" class="" data-icon-position="" data-hide-link-title="0">hossaina@du.ac.bd</a>
<a href="mailto:razu002@du.ac.bd" class="" data-icon-position="" data-hide-link-title="0">razu002@du.ac.bd</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.30.015164v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1219018" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1219018/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.30.015164v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1219018" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1219018/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.30.015164v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1219018" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1219018/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.30.015164v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1219018" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1219018/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.30.015164v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1219018" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1219018/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.30.015164v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1219018" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1219018/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the cause of the ongoing pandemic of coronavirus disease 2019 (COVID-19), a public health emergency of international concern declared by the World Health Organization (WHO). An immuno-informatics approach along with comparative genomic was applied to design a multi-epitope-based peptide vaccine against SARS-CoV-2 combining the antigenic epitopes of the S, M and E proteins. The tertiary structure was predicted, refined and validated using advanced bioinformatics tools. The candidate vaccine showed an average of ≥ 90.0% world population coverage for different ethnic groups. Molecular docking of the chimeric vaccine peptide with the immune receptors (TLR3 and TLR4) predicted efficient binding. Immune simulation predicted significant primary immune response with increased IgM and secondary immune response with high levels of both IgG1 and IgG2. It also increased the proliferation of T-helper cells and cytotoxic T-cells along with the increased INF-γ and IL-2 cytokines. The codon optimization and mRNA secondary structure prediction revealed the chimera is suitable for high-level expression and cloning. Overall, the constructed recombinant chimeric vaccine candidate demonstrated significant potential and can be considered for clinical validation to fight against this global threat, COVID-19.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">Emergence of novel coronavirus, known as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which was first reported in Wuhan, Hubei Province, China in December 2019 is responsible for the ongoing global pandemic of coronavirus disease 2019 (COVID-19). The ongoing outbreak of COVID-19 has already made its alarmingly quick transmission across the globe in 199 countries and regions with a total of 723,077 active infection cases and 33,983 deaths until March 29, 2020<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup>. World Health Organization (WHO) has declared a global health emergency, and scientists are racing over the clock to develop effective interventions for controlling and preventing COVID-19 by analyzing novel SARS-CoV-2 genomics, functional structures, and its host-pathogen interactions. Efforts to contain the virus are ongoing; however, given the many uncertainties regarding pathogen transmissibility and virulence, the effectiveness of these efforts is unknown.</p><p id="p-5">This SARS-CoV-2 is the third coronovairus (CoV) that can infect human after the two previously reported coronavirus-severe acute respiratory syndrome (SARS-CoV)<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup> and Middle East respiratory syndrome (MERS-CoV)<sup><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>–<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup>. Alike SARS-CoV and MERS-CoV, the recent SARS-CoV-2 is a positive-sense single-stranded RNA virus (+ssRNA) belonging to Genus <em>Betacoronavirus</em> within the family Coronaviridae<sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. The genome of SARS-CoV-2 is approximately 30 kilobases (between 26,000 and 32,000 bases) in length<sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup>, and encodes for multiple structural and non-structural proteins<sup><a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a></sup>. The four major structural proteins of CoVs are the spike (S) glycoprotein, small envelope protein (E), membrane protein (M), and nucleocapsid protein (N)<sup><a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a></sup>. Among these proteins, the S glycoprotein is composed of two main domains, the receptor-binding domain (RBD) and N-terminal domain (NTD),<sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup> which play essential role for the attachment and entry of viral particles into the host cells<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup> and characterized by high antigenicity and surface exposure<sup><a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>.</p><p id="p-6">To combat the devastating effects of SARS-CoV-2, Scientists applied different vaccine strategies globally. In addition, a curative approach using different drug candidates have been tried so far to combat the current pandemic COVID-19. However, COVID-19 is marching too fast and reached almost every territory in earth. Hence, apart from developing novel drugs against this viral infection, a very potent vaccine candidate is very much desirable. Being an RNA virus, the virus had undergone mutations several times already in the last 4 months<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>. As a result, instead of single epitope, multi-epitope based vaccines can play a great role in fighting against this novel CoV.</p><p id="p-7">On coronaviruses, the component primarily targeted by monoclonal antibodies (mAbs) is the trimeric S glycoprotein of the virion consisting of three homologous chains (A, B and C), which mediate receptor recognition and membrane fusion during viral entry into sensitive cells through the host angiotensin-converting enzyme 2 (ACE2) receptor<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a></sup>. However, with the rapid evolution and high genomic variation of RNA viruses, mutations on the RBD may enable the new strains to escape neutralization by currently known RBD-targeting antibodies. Therefore, other functional regions of the S glycoprotein can be important for developing effective prophylactic and therapeutic interventions against SARS-CoV-2 infection. Several mAbs targeting non-RBD regions, especially the NTD has recently been reported<sup><a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>. The NTD is located on the side of the spike trimer and has not been observed to undergo any dynamic conformational changes<sup><a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">11</a></sup>, and therefore, might play role in viral attachment, inducing neutralizing antibody responses and stimulates a protective cellular immunity against SARS-CoV<sup><a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">11</a></sup>. Besides, administration of RBD and NTD proteins also induced highly potent neutralizing antibodies and long-term protective immunity in animal models<sup><a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">9</a></sup>. Furthermore, the E and M proteins also have important functions in the viral assembly of a coronavirus and can augment the immune response against SARS-CoV<sup><a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>. Monoclonal antibodies with potent neutralizing activity have become promising candidates for both prophylactic and therapeutic interventions against SARS-CoV-2 infections<sup><a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">13</a></sup>. Therefore, the generation of antibodies targeting the RBD and/or NTD of the S glycoprotein, M and E proteins of SARS-CoV-2 would be an important preventive and treatment strategy that can be tested further in suitable models before clinical trials<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup>. The nucleocapsid (N) protein of SARS-coronavirus (SARS-CoV), buried inside phospholipid bilayer, is the major protein in the helical nucleocapsid of virion. This protein is reported to be more conserved than other structural proteins, an important B cell immunogen, as well as, can elicit long lasting and persistent cellular immune responses. Nevertheless, we did not consider this protein in the chimeric vaccine formation because of its initial unavailability outside of the host cell during infection<sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>.</p><p id="p-8">Most previous studies of the CoVs have focused on the use of single recombinant antigens, however, the immune responses generated have been inadequate to warrant their use in the development of an effectual protective tool<sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>. Alternatively, we propose the development of a multi-epitope vaccine candidate which could lead to the generation of a more potent protective immune response. Multi-epitope vaccine candidates have already been designed for several diseases, including FMD (in our laboratory; unpublished data), MERS and SARS, and their efficacies have been further reported in MERS-CoV and SARS-CoV<sup><a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>. Hence, we can assume that chimeric vaccine targeting multiple epitopes on the RBD and NTD segments of the S protein, M and E proteins would be a potentially effective vaccine candidate in combatting SARS-CoV-2, and therefore, could be used for vaccine design. Combining subunit vaccines with established or new adjuvants may represent a faster and safer strategy to move through early clinical development and analyze the efficacy. This study was therefore aimed to <em>in-silico</em> design and construction of a multi-epitope-based chimeric vaccine candidate against the globally pandemic highly pathogenic SARS-CoV-2.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Retrieving protein sequences</h3><p id="p-9">We retrieved the spike (S), envelop (E) and membrane (M) protein sequences of the SARS-CoV-2, and the S protein sequences of the SARS-CoV and MERS-CoV from the whole genome reference sequences of the respective three viruses from the NCBI database. To map the structural variability of the S protein of SARS-CoV-2, SARS-CoV and MERS-CoV, we applied homology search on the respective S protein sequences, and finally selected three protein data bank (PDB) templates such as 6vsb, 6acd and 5w9h, respectively for predicting the three dimensional (3D) conformation of the S proteins. Finally, the 3D conformation of the S proteins of the respective three viruses were validated using the Ramachandran plot analysis (Supplementary Fig. 1).</p></div><div id="sec-4" class="subsection"><h3>Physicochemical properties and trimeric structure of spike (S) protein</h3><p id="p-10">The physiochemical properties of the spike (S) glycoproteins of the SARS-CoV-2, SARS-CoV and MERS-CoV computed using the Protparam on the ExPASy server demonstrates that it varies significantly among the three viruses. The theoretical isoelectric points (PI) of the subject proteins were 6.24, 5.56 and 5.70, respectively. Protparam computed instability-index (II) of the S protein of SARS-CoV-2, SARS-CoV and MERS-CoV were 33.01, 32.42 and 36.60 indicating that these proteins are quite stable<sup><a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup>.</p><p id="p-11">The S protein of the SARS-CoV-2, SARS-CoV and MERS-CoV is a trimeric conformation consisting of three homologous chains naming as chain A, B and C<sup><a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a></sup> (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). The individual structural domain of coronavirus S proteins reflects high degree of structural divergences in the receptor-binding domain (RBD) and N-terminal domain (NTD) of the chain A and chain C compared to that of chain B. Multiple sequence alignment revealed that the S protein of SARS-CoV-2 shares 77.38% and 31.93% sequence identity with the S proteins of the SARS-CoV and MERS-CoV, respectively, and these findings are in line with many earlier studies<sup><a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>–<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">7</a></sup> (Supplementary Fig. 2). Due to structural divergence in RBD and NTD of S proteins in SARS-CoV-2, SARS-CoV and MERS-CoV, these regions are the main focus of structural studies including epitope-based chimeric vaccine candidate designing, selection, and development.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The trimeric conformation spike (S) proteins. The S proteins of the SARS-CoV-2, SARS-CoV and MERS-CoV is a trimeric conformation consisting of three homologous chains named as chain A, B and C. Respective sequences of these three chains were aligned and visualized using PyMOL which revealed high degree of structural divergences in the N-terminal domains (NTDs) and receptor binding domains (RBDs) of the chain A and C compared to that of chain B." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1062348503" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The trimeric conformation spike (S) proteins.&lt;/span&gt; The S proteins of the SARS-CoV-2, SARS-CoV and MERS-CoV is a trimeric conformation consisting of three homologous chains named as chain A, B and C. Respective sequences of these three chains were aligned and visualized using PyMOL which revealed high degree of structural divergences in the N-terminal domains (NTDs) and receptor binding domains (RBDs) of the chain A and C compared to that of chain B.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F1.medium.gif" width="440" height="230"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F1.medium.gif" width="440" height="230"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1</span> <span class="caption-title">The trimeric conformation spike (S) proteins.</span><p id="p-12" class="first-child">The S proteins of the SARS-CoV-2, SARS-CoV and MERS-CoV is a trimeric conformation consisting of three homologous chains named as chain A, B and C. Respective sequences of these three chains were aligned and visualized using PyMOL which revealed high degree of structural divergences in the N-terminal domains (NTDs) and receptor binding domains (RBDs) of the chain A and C compared to that of chain B.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Structure-based B-cell epitopes prediction</h3><p id="p-13">Linear epitopes prediction based on solvent-accessibility and flexibility revealed that chain A of S protein possessed 15 epitopes, of which three epitopes having different residues positions (395-514, 58-194, 1067-1146) were highly antigenic (score &gt; 0.8). However, the B and C chains had 18 and 19 epitopes, respectively, and of them three epitopes in chain B (residues position 1067-1146, 89-194, 58-87) and two epitopes in chain C (residues position 56-194, 1067-1146) seem to be highly antigenic (score &gt; 0.8) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The amino acid residues 395-514 and 56-194 of the detected epitopes belonged to RBD and NTD regions of the S protein, respectively. These regions were considered as the potential epitope candidates in the IEDB analysis resource Elipro analysis. However, the epitopes with amino-acids position of 1067-1146 were not selected as the potential epitope candidate because of their presence in viral transmembrane domains (Supplementary Fig. 3). The tertiary structures of the RBD and NTD illustrate their surface-exposed position on the S protein (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The three-dimensional (3D) structure of the N-terminal domains (NTDs) and receptor binding domains (RBDs) of the spike (S) proteins of SARS-CoV-2 (surface view). The red, cyan, and yellow colored regions represent the potential antigenic domains predicted by the IEDB analysis resource Elipro analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1062348503" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The three-dimensional (3D) structure of the N-terminal domains (NTDs) and receptor binding domains (RBDs) of the spike (S) proteins of SARS-CoV-2 (surface view).&lt;/span&gt; The red, cyan, and yellow colored regions represent the potential antigenic domains predicted by the IEDB analysis resource Elipro analysis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F2.medium.gif" width="440" height="248"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F2.medium.gif" width="440" height="248"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2</span> <span class="caption-title">The three-dimensional (3D) structure of the N-terminal domains (NTDs) and receptor binding domains (RBDs) of the spike (S) proteins of SARS-CoV-2 (surface view).</span><p id="p-14" class="first-child">The red, cyan, and yellow colored regions represent the potential antigenic domains predicted by the IEDB analysis resource Elipro analysis.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1228444/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1228444/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-15" class="first-child">Linear epitopes present on spike (S) glycoprotein surface predicted through ElliPro in IEDB-analysis resource based upon solvent-accessibility and flexibility are shown with their antigenicity scores. The highlighted green coloured regions were the potential antigenic domains while the yellow coloured region represents the trans-membrane domain of the S protein.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Sequence-based B-cell epitopes</h3><p id="p-16">Antigenicity analysis of full-length S glycoprotein, M (membrane) and E (envelope) proteins using VaxiJen v2.0 showed antigenicity scores of 0.465, 0.510 and 0.603, respectively exhibiting them as potential antigens. Therefore, they were considered to be promising epitopes for vaccine candidate selection against globally pandemic SARS-CoV-2 infections. A total of 22 B-cell epitopes were predicted in S protein of the SARS-CoV-2 using IEDB analysis resource and Bepipred linear epitope prediction 2.0 tools. Of the detected epitopes, eight and six epitopes were respectively found in RBD (aa position 328-528) and NTD (aa position 56-194) regions while the E and M proteins had 2 and 6 epitopes, respectively (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>, Supplementary Table 1). However, only 5 epitopes were exposed on the surface of the virion, and had a high antigenicity score (&gt; 0.4), indicating their potentials in initiating immune responses (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). To find out the most probable peptide-based epitopes with better confidence level, selected peptides were further tested using VaxiJen 2.0 tool, and five peptides having antigenicity score of ≥ 0.5 were annotated. Among the 5 annotated epitopes, RBD region had two highly antigenic epitopes (75-IRGDEVRQIAPGQTGKIADYNYKLPD-100, antigenicity score = 0.932 and 166-QSYGFQPTNGVGYQ-189, antigenicity score = 0.668). Similarly, the NTD region harbored two highly antigenic epitopes (117-SQPFLMDLEGKQGNFKNLRE-136, antigenicity score = 0.749 and 17-GTNGTKRFDN-26, antigenicity score = 0.667). In addition, the envelop (E) protein possessed only one highly antigenic epitope (57-YVYSRVKNLNSSRVP-71, antigenicity score = 0.449) which might be potentially functional in host cell binding (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Furthermore, the Kolaskar and Tongaonkar antigenicity profiling found five highly antigenic epitopes in RBD region with an average (antigenicity) score of 1.042 (minimum = 0.907, maximum = 1.214), and seven highly antigenic epitopes in NTD with an average (antigenicity) score of 1.023 (minimum = 0.866, maximum = 1.213) (Supplementary Fig. 4, Supplementary Table 2). The average Kolaskar scores for envelop protein B-cell epitope (EBE) and membrane protein B-cell epitope (MBE) were 0.980 and 1.032, respectively (Supplementary Table 2). However, through ABCPred analysis, we identified 18 and 11 B-cell epitopes in RBD and NTD regions with average antigenicity score of 0.775 and 0.773 in the associated domains, respectively (Supplementary Table 3).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predicted B-cell epitopes using BepiPred-2.0 epitope predictor in IEDB-analysis resource web-based repository. Yellow areas above threshold (red line) are proposed to be a part of B cell epitopes in (a) RBD and (b) NTD regions of S protein, (c) envelop (E) and (d) membrane (M) proteins of SARS-CoV-2. While green areas are not." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1062348503" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Predicted B-cell epitopes using BepiPred-2.0 epitope predictor in IEDB-analysis resource web-based repository.&lt;/span&gt; Yellow areas above threshold (red line) are proposed to be a part of B cell epitopes in (a) RBD and (b) NTD regions of S protein, (c) envelop (E) and (d) membrane (M) proteins of SARS-CoV-2. While green areas are not.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F3.medium.gif" width="440" height="269"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F3.medium.gif" width="440" height="269"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3</span> <span class="caption-title">Predicted B-cell epitopes using BepiPred-2.0 epitope predictor in IEDB-analysis resource web-based repository.</span><p id="p-17" class="first-child">Yellow areas above threshold (red line) are proposed to be a part of B cell epitopes in (a) RBD and (b) NTD regions of S protein, (c) envelop (E) and (d) membrane (M) proteins of SARS-CoV-2. While green areas are not.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1228445/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1228445/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1228445" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <p id="p-18" class="first-child">B-cell epitopes predicted using Bepipred linear epitope prediction 2.0 in IEDB analysis resource web-server along with their start and end positions, average score, and VaxiJen 2.0 determined antigenicity scores.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Recognition of T-cell epitopes</h3><p id="p-19">The IEDB analysis resource tool was employed to identify T-cell epitopes in RBD and NTD of the S protein of SARS-CoV-2. The prediction method included the major histocompatibility complex class I and (MHC-I, MHC-II), proteasomal cleavage and TAP transport. The RBD and NTD regions were analyzed using IEDB MHC-I binding prediction tool to predict T lymphocytes epitopes that have binding affinity with MHC-I alleles based on Artificial Neural Network (ANN) with half-maximal inhibitory concentration (IC<sub>50</sub>) ≤ 50 nM. A lower IC<sub>50</sub> value indicates higher binding affinity of the epitopes with the MHC class I and II molecules. While most of the studies<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup> reported that a binding affinity (IC<sub>50</sub>) threshold of 250 nM identifies peptide binders recognized by T-cells, and this threshold can be used to select peptides, we kept binding affinity within 50 nM to get better confidence level in predicting epitopes for MHC-I and MHC-II alleles. The IEDB MHC-I prediction tool retrieved 77 T-cell epitopes in RBD that interacted with 21 possible MHC-I alleles whereas the NTD domain possessed 35 T-cell epitopes with 17 possible MHC-I alleles (Supplementary Data 1). Similarly, the IEDB MHC-II prediction tool generated 13-mer124 peptides from the RBD, and 10-mer 73 peptides in the NTD segments of the S protein that showed interaction with many different and/or common MHC-II alleles with an IC<sub>50</sub> value ranging from 1.4 to 49.9 nM (Supplementary Data 1). Furthermore, for MHC-I and MHC-II processing, the analysis tool of the IEDB generates an overall score for each epitope’s intrinsic potential of being a T-cell epitope based on proteasomal processing, TAP transport, and MHC-binding efficiency (Supplementary Data 1). The outcomes of these tools are quite substantial because they utilize vast number of alleles of HLAs (human-leukocyte-antigens) during computation.</p></div><div id="sec-8" class="subsection"><h3>Molecular docking analysis of T-cell epitopes with HLA alleles</h3><p id="p-20">From the selected epitopes from the RBD and NTD segments, top five based on IC<sub>50</sub> score were used in molecular docking analysis using the GalaxyWeb server with their respective HLA allele binders, in which they revealed significantly favorable molecular interaction for binding affinity. Docking complexes thus formed have significantly negative binding energy, and most of the aa residues of the epitopes were involved in molecular interactions with their respective HLA alleles (Supplementary data 1). The epitope-HLA docking complexes were further refined with GalaxyRefineComplex, and their binding affinity was analyzed through PRODIGY web-server. All of the selected epitopes showed significantly negative binding affinity (ΔG always remained ≤ −8.2 kcal mol<sup>-1</sup>, average = −9.94 kcal mol<sup>-1</sup>, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>, Supplementary data 1).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Molecular docking of top five MHC-&#x25A1; and MHC-&#x25A1; epitopes of RBD and NTD domains with respect to HLA allele binders. The protein-peptide docking was performed in GalaxyWEB-GalaxyPepDock-server followed by the refinement using GalaxyRefineComplex and free energy (&#x394;G) of each complex was determined in PRODIGY server. Ribbon structures represent HLA alleles and stick structures represent the respective epitopes. Light color represents the templates to which the alleles and epitopes structures were built. Further information on molecular docking analysis is also available in Supplementary Data 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1062348503" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Molecular docking of top five MHC-&#x25A1; and MHC-&#x25A1; epitopes of RBD and NTD domains with respect to HLA allele binders.&lt;/span&gt; The protein-peptide docking was performed in GalaxyWEB-GalaxyPepDock-server followed by the refinement using GalaxyRefineComplex and free energy (&#x394;G) of each complex was determined in PRODIGY server. Ribbon structures represent HLA alleles and stick structures represent the respective epitopes. Light color represents the templates to which the alleles and epitopes structures were built. Further information on molecular docking analysis is also available in Supplementary Data 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F4.medium.gif" width="440" height="405"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F4.medium.gif" width="440" height="405"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4</span> <span class="caption-title">Molecular docking of top five MHC-□ and MHC-□ epitopes of RBD and NTD domains with respect to HLA allele binders.</span><p id="p-21" class="first-child">The protein-peptide docking was performed in GalaxyWEB-GalaxyPepDock-server followed by the refinement using GalaxyRefineComplex and free energy (ΔG) of each complex was determined in PRODIGY server. Ribbon structures represent HLA alleles and stick structures represent the respective epitopes. Light color represents the templates to which the alleles and epitopes structures were built. Further information on molecular docking analysis is also available in Supplementary Data 1.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>IFN-<strong>γ</strong> inducing epitope prediction</h3><p id="p-22">The findings of IFNepitope program suggests that, both the target RBD and NTD regions of S protein, and B-cell linear epitope (MBE) had great probability to release of IFN-γ with a positive score. A total of 56 potential positive IFN-γ inducing epitopes (15-mer) were predicted for the RBD domain with an average epitope prediction score of 0.255 and the maximum SVM score of 0.625. On the other hand, a total of 33 potential positive epitopes were predicted for the NTD domain with an average epitope prediction score of 0.312 and the maximum SVM score of 0.811. Moreover, the M protein also possessed several IFN-γ inducing epitopes having an average epitope prediction score of 0.980 (Supplementary Table 4).</p></div><div id="sec-10" class="subsection"><h3>Population coverage analysis</h3><p id="p-23">The distribution of HLA allele varies between different geographical regions and ethnic groups around the globe. Therefore, population coverage during the development of an efficient vaccine must be taken into account. In this study, epitopes having IC<sub>50</sub> values ≤ 50 nM and their respective alleles (CTL and HTL) were considered for population coverage analysis both individually (MHC-I and MHC-II) and in combination (Supplementary Data 2). Our selected CTL and HTL epitopes were found to cover 94.9% and 73.11% of the world population, respectively. Importantly, CTL and HTL epitopes showed 98.63% population coverage worldwide when used in combination. The highest population coverage was found to be 99.99% in the Latin American country, Peru (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5</a>, Supplementary Data 2). In China, where the viral strain (SARS-CoV-2) first appeared and had more devastating outbreaks, the population coverage for CTL and HTL epitopes was 92.67% and 53.44%, respectively with a combined coverage of 96.59%. SARS-CoV-2 is currently causing serious pandemics in different continents of the globe including Italy, England, Spain, Iran, South Korea and United States of America where the combined population coverage was found to be 98.8%, 99.44%, 95.35%, 98.48%, 99.19% and 99.35%, respectively (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5a</a>, Supplementary Data 2). In addition to geographical distribution, the ethnic groups also found to be an important determinant for good coverage of the CTL and HTL epitopes (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5b</a>). Of the studied 147 ethnic groups, the Peru Amerindian had highest population coverage for CTL (99.98%) while the HTL epitopes had highest population coverage for Austria Caucasoid (88.44%) (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5b</a>, Supplementary Data 2). Furthermore, 53.06% of the ethnic groups had a combined population coverage of more than 90.0% for both CTL and HTL epitopes.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population coverage of the selected T-cell epitopes and their respective HLA alleles. The circular plot illustrates the relative abundance of the top 70 geographic regions and ethnic groups for selected CTL and HTL epitopes, which were used to construct the vaccine and their corresponding MHC HLA alleles were obtained for population coverage analysis both individually (either MHC-&#x25A1; or MHC-&#x25A1;) and in combination (MHC-&#x25A1; and MHC-&#x25A1;). (a) Population coverage of top seventy geographical regions out of 123 regions. (b) Population coverage of top seventy ethnic groups selected from 146 ethnic groups. Regions and ethnic groups in the respective MHC-&#x25A1; and MHC-&#x25A1; epitopes are represented by different colored ribbons, and the inner blue bars indicate their respective relative coverages. Further information on population coverage analysis is also available in Supplementary Data 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1062348503" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Population coverage of the selected T-cell epitopes and their respective HLA alleles.&lt;/span&gt; The circular plot illustrates the relative abundance of the top 70 geographic regions and ethnic groups for selected CTL and HTL epitopes, which were used to construct the vaccine and their corresponding MHC HLA alleles were obtained for population coverage analysis both individually (either MHC-&#x25A1; or MHC-&#x25A1;) and in combination (MHC-&#x25A1; and MHC-&#x25A1;). (a) Population coverage of top seventy geographical regions out of 123 regions. (b) Population coverage of top seventy ethnic groups selected from 146 ethnic groups. Regions and ethnic groups in the respective MHC-&#x25A1; and MHC-&#x25A1; epitopes are represented by different colored ribbons, and the inner blue bars indicate their respective relative coverages. Further information on population coverage analysis is also available in Supplementary Data 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F5.medium.gif" width="440" height="249"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 5" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F5.medium.gif" width="440" height="249"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 5</span> <span class="caption-title">Population coverage of the selected T-cell epitopes and their respective HLA alleles.</span><p id="p-24" class="first-child">The circular plot illustrates the relative abundance of the top 70 geographic regions and ethnic groups for selected CTL and HTL epitopes, which were used to construct the vaccine and their corresponding MHC HLA alleles were obtained for population coverage analysis both individually (either MHC-□ or MHC-□) and in combination (MHC-□ and MHC-□). (a) Population coverage of top seventy geographical regions out of 123 regions. (b) Population coverage of top seventy ethnic groups selected from 146 ethnic groups. Regions and ethnic groups in the respective MHC-□ and MHC-□ epitopes are represented by different colored ribbons, and the inner blue bars indicate their respective relative coverages. Further information on population coverage analysis is also available in Supplementary Data 1.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Design and construction of chimeric vaccine candidate</h3><p id="p-25">The selected sequences for designing of chimeric construct were PADRE (13 aa), MBE (20 aa), NTD (139 aa), RBD (200 aa), EBE (15 aa), and Invasin (16 aa), and the construct was named as CoV-RMEN. The schematic diagram of protein domain structures, their junctions and linker sites in CoV-RMEN are shown in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6a</a>. These fragments were fused together using a repeat of hydrophobic (glycine; G) and acidic aa (glutamic acid; E) linkers. The EGGE, GG, GG, GG and EGGE sequences were introduced between different domains, and epitopes for more flexibility and efficient separation with balancing of acidic and basic amino acids.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Design, construction and structural validation of multi-epitope vaccine candidate (CoV-RMEN) for SARS-CoV-2. (a) Structural domains and epitopes rearrangement of CoV-RMEN, (b) secondary structure of CoV-RMEN as analyzed through CFSSP:Chou and Fasman secondary structure prediction server, (c) final tertiary structure of CoV-RMEN (surface view) obtained from homology modelling on Phyre2 in which domains and epitopes are represented in different colors (PADRE-green; membrane B-cell epitope, MBE-magenta; N-terminal domain, NTD-orange; receptor-binding domain, RBD-cyan; envelop B-cell epitope, EBE-blue; invasin-yellow), (d) validation of the refined model with Ramachandran plot analysis showing 94.7%, 4.8% and 0.5% of protein residues in favored, allowed, and disallowed (outlier) regions respectively, (e) ProSA-web, giving a Z-score of &#x2212;6.17, and (f) the finally predicted primary structure of the CoV-RMEN." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1062348503" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Design, construction and structural validation of multi-epitope vaccine candidate (CoV-RMEN) for SARS-CoV-2.&lt;/span&gt; (a) Structural domains and epitopes rearrangement of CoV-RMEN, (b) secondary structure of CoV-RMEN as analyzed through CFSSP:Chou and Fasman secondary structure prediction server, (c) final tertiary structure of CoV-RMEN (surface view) obtained from homology modelling on Phyre2 in which domains and epitopes are represented in different colors (PADRE-green; membrane B-cell epitope, MBE-magenta; N-terminal domain, NTD-orange; receptor-binding domain, RBD-cyan; envelop B-cell epitope, EBE-blue; invasin-yellow), (d) validation of the refined model with Ramachandran plot analysis showing 94.7%, 4.8% and 0.5% of protein residues in favored, allowed, and disallowed (outlier) regions respectively, (e) ProSA-web, giving a Z-score of &#x2212;6.17, and (f) the finally predicted primary structure of the CoV-RMEN.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F6.medium.gif" width="440" height="242"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 6" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F6.medium.gif" width="440" height="242"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 6</span> <span class="caption-title">Design, construction and structural validation of multi-epitope vaccine candidate (CoV-RMEN) for SARS-CoV-2.</span><p id="p-26" class="first-child">(a) Structural domains and epitopes rearrangement of CoV-RMEN, (b) secondary structure of CoV-RMEN as analyzed through CFSSP:Chou and Fasman secondary structure prediction server, (c) final tertiary structure of CoV-RMEN (surface view) obtained from homology modelling on Phyre2 in which domains and epitopes are represented in different colors (PADRE-green; membrane B-cell epitope, MBE-magenta; N-terminal domain, NTD-orange; receptor-binding domain, RBD-cyan; envelop B-cell epitope, EBE-blue; invasin-yellow), (d) validation of the refined model with Ramachandran plot analysis showing 94.7%, 4.8% and 0.5% of protein residues in favored, allowed, and disallowed (outlier) regions respectively, (e) ProSA-web, giving a Z-score of −6.17, and (f) the finally predicted primary structure of the CoV-RMEN.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Prediction of the antigenicity and allergenicity of the CoV-RMEN</h3><p id="p-27">The antigenicity of the final chimeric protein sequence was predicted by the VaxiJen 2.0 server to be 0.450 with a virus model at a threshold of 0.4 and 0.875 with ANTIGENpro. The results indicate that the generated CoV-RMEN sequences are both antigenic in nature. The vaccine sequence without the adjuvant was also predicted to be non-allergenic on both the AllerTOP v.2 and AllergenFP servers<sup><a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a></sup>.</p></div><div id="sec-13" class="subsection"><h3>Physiochemical properties and solubility prediction of the CoV-RMEN</h3><p id="p-28">The molecular weight (MW) of the finally constructed vaccine candidate was predicted to be 46.8 kDa with a theoretical isoelectric point value (pI) of 8.71. The protein is predicted to be slightly basic in nature based on the pI. The half-life was assessed to be 4.4 hours in mammalian reticulocytes <em>in vitro</em>, and &gt;20 hours in yeast and &gt;10 hours in <em>E. coli in vivo</em>. The protein was predicted to be less soluble upon expression with a solubility score of 0.330 corroborating the findings of many previous studies<sup><a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">21</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>. An instability index (II) of 29.74 was predicted, classifying the protein as stable (II of &gt; 40 indicates instability). The estimated aliphatic index was predicted to be 66.59, indicating thermostability. The predicted Grand average of hydropathicity (GRAVY) was −0.300. The negative value indicates the protein is hydrophilic in nature and can interact with water molecules<sup><a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">21</a></sup>.</p></div><div id="sec-14" class="subsection"><h3>Secondary and tertiary structures prediction of the CoV-RMEN</h3><p id="p-29">The CoV-RMEN peptide was predicted to contain 43.2% alpha helix, 67.4% beta sheet, and 12% turns (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6b</a>, Supplementary Fig. 5) using CFSSP:Chou and Fasman secondary structure prediction server. In addition, with regards to solvent accessibility of aa residues, 34% were predicted to be exposed, 30% medium exposed, and 34% were predicted to be buried. Only 2 aa residues (0.0%) were predicted to be located in disordered domains by the RaptorX Property server (Supplementary Fig. 6). The Phyre2 server predicted the tertiary structure model of the designed chimeric protein in 5 templates (c5x5bB, c2mm4A, c6vsbB, c5x29B and c6vybB) based on heuristics to maximize confidence, percent identity and alignment coverage. The final model of the CoV-RMEN peptide modelled at 82% with more than 90% confidence (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. 6c</a>). Moreover, 65 residues were modelled by <em>ab initio</em>.</p></div><div id="sec-15" class="subsection"><h3>Tertiary structure refinement and validation of the CoV-RMEN</h3><p id="p-30">After energy minimization of the CoV-RMEN using GROMOS96 program<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup>, refinement of the vaccine model on the ModRefiner server followed by further refinement on the GalaxyRefine server yielded five models. The selected model has parameters scores of GDT-HA (0.9538), RMSD (0.414), and MolProbity (2.035). The Ramachandran plot analysis of the finally modelled protein revealed that 94.7% of residues in the protein are in favored regions (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig. 6d</a>), consistent with the 94.0% score predicted by the GalaxyRefine analysis. Additionally, 4.8% of the residues were predicted to be in allowed regions, and only 0.5% in disallowed region (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Fig. 6d</a>). The chosen model after refinement had an overall quality factor of 74.45% with ERRAT (Supplementary Fig. 7) while ProSA-web gave a Z-score of −6.17 (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Fig. 6e</a>) for the CoV-RMEN protein model.</p></div><div id="sec-16" class="subsection"><h3>Immune simulation</h3><p id="p-31">The immune-stimulatory ability of the predicted vaccine CoV-RMEN was conducted through the C-ImmSimm server. The analysis predicts the generation of adaptive immunity in target host species (human) using position-specific scoring matrix (PSSM), and machine learning techniques for the prediction of epitopes and immune interactions<sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup>. The cumulative results of immune responses after three times antigen exposure with four weeks interval each time revealed that the primary immune response against the antigenic fragments was elevated indicated by gradual increase of IgM level after each antigen exposure (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 7a</a>). Besides, the secondary immune response, crucial for immune stability, have been shown as increased with adequate generation of both IgG1 and IgG2. Also, the elevated level of all circulating immunoglobulins indicates the accuracy of relevant clonal proliferation of B-cell and T-cell population. The level of cytokines after antigen exposure increased concomitantly reflected by escalation of IFN-γ and IL-2 which are most significant cytokines for anti-viral immune response and clonal selection (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Fig. 7b</a>). The abundance of different types of B-cells and T-cells, like antigen processing B-cells, resting memory B- and T- cells, B-cells with IgM and IgG remains significantly higher indicating development of immune memory and consequently increased clearance of antigen after exposure (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Fig. 7c,d</a>). Additionally, T-helper cells and cytotoxic T-cells were found with a drastic up-regulation of Th1 concentration enhancing the B-cell proliferation and immune memory development (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Fig. 7e,f</a>). The high level of immunoglobulin IgG1 + IgG2, active B-cell and T-helper cell population reflected the development of strong immune response reinforcing the indelible and peerless antigenicity of the CoV-RMEN vaccine candidate.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="C-ImmSim presentation of an in silico immune simulation with the chimeric peptide. (a) The immunoglobulins and the immunocomplex response to antigen (CoV-RMEN) inoculations (black vertical lines); specific subclasses are indicated as colored peaks, (b) concentration of cytokines and interleukins, and inset plot shows danger signal together with leukocyte growth factor IL-2, (c) B-cell populations after three injections, (d) evolution of B cell, (e) T-helper cell populations per state after injections, and (f) evolution of T-helper cell classes with the course of vaccination." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1062348503" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;C-ImmSim presentation of an &lt;em&gt;in silico&lt;/em&gt; immune simulation with the chimeric peptide.&lt;/span&gt; (a) The immunoglobulins and the immunocomplex response to antigen (CoV-RMEN) inoculations (black vertical lines); specific subclasses are indicated as colored peaks, (b) concentration of cytokines and interleukins, and inset plot shows danger signal together with leukocyte growth factor IL-2, (c) B-cell populations after three injections, (d) evolution of B cell, (e) T-helper cell populations per state after injections, and (f) evolution of T-helper cell classes with the course of vaccination.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F7.medium.gif" width="440" height="243"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 7" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F7.medium.gif" width="440" height="243"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 7</span> <span class="caption-title">C-ImmSim presentation of an <em>in silico</em> immune simulation with the chimeric peptide.</span><p id="p-32" class="first-child">(a) The immunoglobulins and the immunocomplex response to antigen (CoV-RMEN) inoculations (black vertical lines); specific subclasses are indicated as colored peaks, (b) concentration of cytokines and interleukins, and inset plot shows danger signal together with leukocyte growth factor IL-2, (c) B-cell populations after three injections, (d) evolution of B cell, (e) T-helper cell populations per state after injections, and (f) evolution of T-helper cell classes with the course of vaccination.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Molecular docking of CoV-RMEN with immune receptors (TLR3 and TLR4)</h3><p id="p-33">The ClusPro server was used to determine the protein binding and hydrophobic interaction sites on the protein surface. The immune responses of TLR3 and TLR4 against vaccine construct (CoV-RMEN) were estimated by analyzing the overall conformational stability of vaccine protein-TLRs docked complexes. The active interface aa residues of refined complexes of CoV-RMEN and TLRs were predicted (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Fig. 8</a>, <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The relative binding free energies (ΔG) of the protein-TLRs complexes were significantly negative (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>) which suggest that the interaction of the chimeric protein might favor stimulation of the TLR receptors. Interface contacts (IC) per property (ICs charged-charged: 16, ICs charged-polar: 22, ICs charged-apolar: 26, polar-polar: 6, ICs polar-apolar: 25 and apolar-apolar: 29) were for the vaccine protein-TLR3 complex. Also, vaccine protein-TLR4 complex showed similar (ICs) per property (ICs charged-charged: 5, ICs charged-polar: 11, ICs charged-apolar: 30, polar-polar: 4, ICs polar-apolar: 31 and apolar-apolar: 39). These data validate the selected docked complexes that may result high binding among TLRs and the chimeric protein (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Fig. 8</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Molecular docking of CoV-RMEN vaccine with immune receptors (TLR2, TLR3 and TLR4). Docked complexes for (a) CoV-RMEN and TLR2, (b) CoV-RMEN and TLR3, and (c) CoV-RMEN and TLR4. Magnified interfaces of the complexes are figured to (d), (e) and (f) respectively. Active residues of CoV-RMEN colored magenta, and of TLRs colored orange with stick view. &#x394;G represents the binding affinity of the complexes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1062348503" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Molecular docking of CoV-RMEN vaccine with immune receptors (TLR2, TLR3 and TLR4).&lt;/span&gt; Docked complexes for (a) CoV-RMEN and TLR2, (b) CoV-RMEN and TLR3, and (c) CoV-RMEN and TLR4. Magnified interfaces of the complexes are figured to (d), (e) and (f) respectively. Active residues of CoV-RMEN colored magenta, and of TLRs colored orange with stick view. &#x394;G represents the binding affinity of the complexes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F8.medium.gif" width="440" height="248"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 8" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F8.medium.gif" width="440" height="248"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 8</span> <span class="caption-title">Molecular docking of CoV-RMEN vaccine with immune receptors (TLR2, TLR3 and TLR4).</span><p id="p-34" class="first-child">Docked complexes for (a) CoV-RMEN and TLR2, (b) CoV-RMEN and TLR3, and (c) CoV-RMEN and TLR4. Magnified interfaces of the complexes are figured to (d), (e) and (f) respectively. Active residues of CoV-RMEN colored magenta, and of TLRs colored orange with stick view. ΔG represents the binding affinity of the complexes.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1228461/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1228461/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1228461" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3:</span> <p id="p-35" class="first-child">Active interface amino acid residues and binding scores among Toll Like Receptors (TLRs) and the constructed vaccine CoV-RMEN.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Codon optimization of the CoV-RMEN</h3><p id="p-36">The high yield expression of a recombinant protein is always remained as a challenge, whereas heterologous prokaryotic expression vector can potentiate a robust, low cost and user-friendly technology for large scale vaccine production. For this reason, the recombinant protein encoding sequence need to be optimized for escalation of protein expression and downstream processing for a final product development. In order to optimize codon usage of the vaccine construct CoV-RMEN in <em>E. coli</em> (strain K12) for maximal protein expression, Rare Codon Analysis Tool (GenScript) was used. The length of the optimized codon sequence was 1,251 nucleotides. The Codon Adaptation Index (CAI) of the optimized nucleotide sequence was 0.87, and the average GC content of the adapted sequence was 50.26% showing the possibility of good expression of the vaccine candidate in the <em>E. coli</em> host. The optimal percentage range of GC content is between 30% and 70% (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Fig. 9 a,b,c</a>).</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Codon optimization and mRNA structure of CoV-RMEN gene for expression in E. coli (a) GC curve (average GC content: 50.26%) of the optimized CoV-RMEN gene, (b) percentage distribution of codons in computed codon quality groups, (c) relative distribution of codon usage frequency along the gene sequence to be expressed in E. coli, and codon adaptation index (CAI) was found to be 0.87 for the desired gene, (d) secondary structure and stability of corresponding mRNA, and (e) resolved view of the start region in the mRNA structure of CoV-RMEN." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1062348503" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Codon optimization and mRNA structure of CoV-RMEN gene for expression in &lt;em&gt;E. coli&lt;/em&gt;&lt;/span&gt; (a) GC curve (average GC content: 50.26%) of the optimized CoV-RMEN gene, (b) percentage distribution of codons in computed codon quality groups, (c) relative distribution of codon usage frequency along the gene sequence to be expressed in &lt;em&gt;E. coli&lt;/em&gt;, and codon adaptation index (CAI) was found to be 0.87 for the desired gene, (d) secondary structure and stability of corresponding mRNA, and (e) resolved view of the start region in the mRNA structure of CoV-RMEN.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F9.medium.gif" width="440" height="254"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 9" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F9.medium.gif" width="440" height="254"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 9</span> <span class="caption-title">Codon optimization and mRNA structure of CoV-RMEN gene for expression in <em>E. coli</em></span><p id="p-37" class="first-child">(a) GC curve (average GC content: 50.26%) of the optimized CoV-RMEN gene, (b) percentage distribution of codons in computed codon quality groups, (c) relative distribution of codon usage frequency along the gene sequence to be expressed in <em>E. coli</em>, and codon adaptation index (CAI) was found to be 0.87 for the desired gene, (d) secondary structure and stability of corresponding mRNA, and (e) resolved view of the start region in the mRNA structure of CoV-RMEN.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Prediction of mRNA secondary structure of the CoV-RMEN</h3><p id="p-38">The evaluation of minimum free energy for 25 structures of chimeric mRNA, the optimized sequences carried out by the ‘Mfold’server. The results showed that ΔG of the best predicted structure for the optimized construct was ΔG = −386.50 kcal/mol. The first nucleotides at 5’ did not have a long stable hairpin or pseudoknot. Therefore, the binding of ribosomes to the translation initiation site, and the following translation process can be readily accomplished in the target host. These outcomes were in the agreement with data obtained from the ‘RNAfold’web server (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Fig. 9</a> d,e) where the free energy was −391.37 kcal/mol.</p></div><div id="sec-20" class="subsection"><h3>Expression of CoV-RMEN with SUMO-fusion</h3><p id="p-39">After codon optimization and mRNA secondary structure analysis, the sequence of the recombinant plasmid was designed by inserting the adapted codon sequences into pETite vector (Lucigen, USA) using SnapGene software (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Fig. 10</a>). The utilization of pETite vector containing SUMO (Small Ubiquitin-like Modifier) tag and 6x-His tag will facilitate both the solubilization and affinity purification of the recombinant protein<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>. After successful cloning of the gene, the recombinant plasmid can be propagated efficiently using <em>E. coli</em> cells, and subsequent protein expression can be performed in <em>E. coli</em> K-12 strain using IPTG (Isopropyl β-d-1-thiogalactopyranoside) induction and cultivation at 28 °C as also reported earlier<sup><a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a></sup>.</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="In silico fusion cloning of the CoV-RMEN. The final vaccine candidate sequence was inserted into the pETite expression vector where the red part represents the gene coding for the predicted vaccine, and the black circle represents the vector backbone. The six His-tag and SUMU-tag are located at the Carboxy-terminal end." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1062348503" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;&lt;em&gt;In silico&lt;/em&gt; fusion cloning of the CoV-RMEN.&lt;/span&gt; The final vaccine candidate sequence was inserted into the pETite expression vector where the red part represents the gene coding for the predicted vaccine, and the black circle represents the vector backbone. The six His-tag and SUMU-tag are located at the Carboxy-terminal end.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F10.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 10" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F10.medium.gif" width="440" height="292"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 10</span> <span class="caption-title"><em>In silico</em> fusion cloning of the CoV-RMEN.</span><p id="p-40" class="first-child">The final vaccine candidate sequence was inserted into the pETite expression vector where the red part represents the gene coding for the predicted vaccine, and the black circle represents the vector backbone. The six His-tag and SUMU-tag are located at the Carboxy-terminal end.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-21"><h2 class="">Discussion</h2><p id="p-41">The causative agent of the ongoing COVID-19 pandemic caused by SARS-CoV-2, the seventh coronavirus with high zoonotic importance and transmission rate, has proved to be transmitted through direct and indirect contact along with airborne transmission which contribute very potentially for community transmission<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>–<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>. Since the emergence of SARS-CoV-2 in 2019 in Wuhan, China<sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>, <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a></sup>, it has exceeded both SARS-CoV and MERS-CoV in its rate of transmission among humans<sup><a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a></sup>, and therefore, effective measures counteracting its infection have become a major research focus. Scientific efforts have been going on extensively to learn about the virus, its biological properties, epidemiology, and pathophysiology<sup><a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">7</a></sup>. At this early stage, different vaccine development efforts have been going on globally along with the novel drug development strategies<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup>.</p><p id="p-42">Development of an effective live attenuated vaccine for viral pathogens has been the most practiced strategy for prevention. But due to its extremely high contagious nature and ability for community transmission, this novel SARS-CoV-2 attenuated vaccine research essentially requires high biosafety level for researchers and work stations, which may eventually make it difficult to avail the vaccine for mass population of all the countries. Although live attenuated vaccine is often very immunogenic, however, the risk of reversion to a virulent virus has limited its usage as SARS-CoV and MERS-CoV vaccine<sup><a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a></sup>. Unlike other types of vaccines, chimeric peptide vaccine production does not involve virus replication, therefore reduce the cost of production. Hence, a low cost strategy should be adopted for developing a highly demanded vaccine for the mankind. Heterologous expression of any vaccine candidate protein has very promising scopes for developing such low cost vaccine, providing that all essential properties for antigenicity, immunogenicity and functional configuration are being conserved to mimic the structural and functional property of the actual antigen<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>. Construction of a vaccine candidate with multiple potential epitopes can obviously potentiate the multi-valency of the antigen to develop immune response against a number of epitopes of any pathogen. Also, rational engineering of epitopes for increased potency and magnitude, ability to enhance immune response in conserved epitopes, increased safety and absence of unnecessary viral materials and cost effectiveness all these cumulatively include potential benefit to multi-epitope recombinant protein based vaccine<sup><a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a></sup>. This study was designed to assist with the initial phase of multi-epitope vaccine candidate selection. Thereby, safe and effective vaccine development by providing recommendations of epitopes that may potentially be considered for incorporation in vaccine design for SARS-CoV-2.</p><p id="p-43">Pathophysiology and virulence mechanisms of coronaviruses (CoVs), and therefore similarly of SARS-CoV-2 have links to the function of the structural proteins such as spike (S) glycoprotein, envelop (E), and membrane (M) proteins<sup><a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>. Across the CoV families, the antibodies against the S glycoprotein, M and E proteins of SARS-CoV-2 would provide immunity to the infection<sup><a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a></sup>. Therefore, this work focused on the <em>in silico</em> design and development of a potential multi-epitope vaccine for SARS-CoV-2 selecting epitopes and domains from the S, E and M proteins.</p><p id="p-44">Among the structural elements of CoVs, the spike S glycoproteins are composed of two major domains, the RBD and NTD. Furthermore, the homotrimeric S proteins of the SARS-CoV-2, SARS-CoV<sup><a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a></sup> and MERS-CoV<sup><a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a></sup> comprises three chains: the chain A, B and C on the viral surface, guiding the link to host receptors. Unlike the full-length S protein of the CoVs, the RBD and NTD segments possessed the critical neutralizing domains but lacking the non-neutralizing immunodominant region<sup><a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">22</a></sup>. Therefore, considering the safety and effectiveness perspectives, the RBD and NTD are more promising candidates in the development of SARS-CoV-2 vaccines over the full-length S protein. The presence of E and M proteins on the envelope can augment the immune response against SARS-CoV<sup><a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup> and thus, considered for suitable candidate for vaccine development<sup><a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">22</a></sup>. Therefore, antibodies against the S, M and E proteins of SARS-CoV-2 would provide protective immunity to the infection<sup><a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">22</a></sup>.</p><p id="p-45">Immuno-informatics analyses of the selected RBD and NTD regions indicated that these contain ample amount of high-affinity B-cell, MHC Class I, MHC Class II and interferon-γ (IFN-γ) epitopes with high antigenicity scores. Moreover, membrane B-cell epitope (MBE) and envelop B-cell epitope (EBE) were selected from the M and E proteins, respectively for the final vaccine construct to enhance the overall stability, immunogenicity and antigenicity of the chimeric vaccine candidate, the CoV-RMEN<sup><a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">21</a></sup>. The HLA alleles maintain the response to T-cell epitopes, and in different ethnic communities and geographical regions, these alleles are highly polymorphic. T-cell epitopes from RBD and NTD regions having high interaction with more HLA alleles were found to cover more than 98% of the world population with different ethnic groups<sup><a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup>. Moreover, the molecular docking between T-cell epitopes and their respective HLA alleles revealed their highly significant binding affinity reflecting the immune activation of B- and T-cells<sup><a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a></sup>.</p><p id="p-46">Vaccine design is improved through the use of specialized spacer sequences<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup>. To designing the CoV-RMEN (vaccine candidate) GG and EGGE linkers were incorporated between the predicted epitopes to produce sequences with minimized junctional immunogenicity, thereby, allowing the rational design construction of a potent multi-epitope vaccine<sup><a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">21</a>, <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a></sup>. The molecular weight of our vaccine candidate, the CoV-RMEN is 46.8 kDa with a predicted theoretical pI of 8.71, indicating that the protein is basic in nature. Also, the predicted instability index indicates that the protein will be stable upon expression, thus further strengthening its potential for use. The aliphatic index showed that the protein contains aliphatic side chains, indicating potential hydrophobicity. All these parameters indicate that the recombinant protein is thermally stable, hence would be best suited for use in different endemic areas worldwide<sup><a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-21-7" class="xref-bibr" href="#ref-21">21</a></sup>.</p><p id="p-47">The knowledge of secondary and tertiary structures of the target protein is essential in vaccine design<sup><a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup>. Secondary structure analysis of the CoV-RMEN indicated that the protein consisted of 43.2% alpha helix, 67.4% beta sheet, and 12% turns with only 2 residues disordered. Natively unfolded protein regions and alpha-helical and beta sheet peptides have been reported to be important forms of structural antigens<sup><a id="xref-ref-21-8" class="xref-bibr" href="#ref-21">21</a>, <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a></sup>. These two structural forms (secondary and tertiary), when tested as the synthetic peptides, have the ability to fold into their native structure,hence, could be recognized by naturally induced antibodies in response to infection<sup><a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">2</a></sup>. The tertiary (3D) structure of the vaccine candidate improved markedly after the refinement, and showed desirable properties based on Ramachandran plot predictions. The Ramachandran plot shows that most of the residues are found in the favoured and allowed regions (94.7%) with very few residues in the outlier region; this indicates that the quality of the overall model is satisfactory<sup><a id="xref-ref-21-9" class="xref-bibr" href="#ref-21">21</a>, <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a></sup>. The lack of allergenic properties of the CoV-RMEN further strengthens its potential as a vaccine candidate. Strikingly, the multi-epitope peptide we designed and constructed (CoV-RMEN), showed higher antigenicity scores on Vaxijen v2.0 (0.450 with a virus model at a threshold of 0.4) and ANTIGENpro (0.875) gave an antigenic score almost twice that of Ov-103 on ANTIGENpro (0.93 for the chimeric peptide and 0.59 for Ov-103) supporting that this multiple-epitopes carrying vaccine candidate would be poorly immunogenic and require coupling to adjuvants. However, the designed protein must show similar antigenicity scores with and without the addition of an adjuvant sequence<sup><a id="xref-ref-21-10" class="xref-bibr" href="#ref-21">21</a></sup>.</p><p id="p-48">It has been reported that immunity to viral infections is dependent on both B- and T- cells<sup><a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-21-11" class="xref-bibr" href="#ref-21">21</a></sup>. Toll-Like Receptors (TLRs) recognize Pathogen Associated Molecular Patterns (PAMPs) from viral components or replication intermediates, resulting in signaling cascades that initiate an antiviral state in cells as a result of infection<sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup>. However, it is likely that TLRs might play an important role in the innate immune response to SARS-CoV-2 infection<sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup>. Though, we did not use any adjuvants, it is reported that TLRs (TLR3 and TLR4) can effectively bind with spike protein of the CoV<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup>. The molecular docking analysis of the CoV-RMEN through HADDOCK showed that the chimeric protein can establish stable protein-protein interactions with TLRs (TLR-3,TLR-4)<sup><a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a></sup> indicating efficient activation of these surface molecules which is very crucial for immune activation of dendritic cells for subsequent antigen processing and presentation to CD4+ and CD8+ T-cells via MHC-II and MHC-1, respectively corroborating the findings of different earlier studies<sup><a id="xref-ref-11-9" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-21-12" class="xref-bibr" href="#ref-21">21</a></sup>. TLR-4 might bind to S protein leading to the activation of the MyD88 dependent signaling pathway, which ultimately releases proinflammatory cytokines<sup><a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a></sup>. Immune simulation of the designed CoV-RMEN showed results consistent with typical immune responses, and there was a general increase in the generated immune responses after repeated antigen exposures. The comprehensive immune response developed after repeated antigen exposure reflected a perfect host defence development with the uplifted primary immune response and subsequent secondary immune response resulting in immune memory development<sup><a id="xref-ref-2-8" class="xref-bibr" href="#ref-2">2</a></sup>. In the present study, the development of memory B-cells and T-cells was evident, with memory in B-cells lasting for several months. Helper T-cells were particularly stimulated. The engrossing findings of the study is the development of Th1 response which enhance the growth and proliferation of B-cells augmenting the adaptive immunity<sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup>. The antiviral cytokine IFN-γ and cell stimulatory IL-2 level significantly increased which also contribute to the subsequent immune response after vaccination in host<sup><a id="xref-ref-2-9" class="xref-bibr" href="#ref-2">2</a></sup>. This indicates high levels of helper T-cells and consequently efficient antibody production, supporting a humoral response<sup><a id="xref-ref-21-13" class="xref-bibr" href="#ref-21">21</a>, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup>. The Simpson index, D for investigation of clonal specificity suggests a possible diverse immune response. This is plausible considering the generated chimeric peptide is composed of sufficient B- and T-cell epitopes.</p><p id="p-49">One of the first steps in validating a candidate vaccine is to screen for immunoreactivity through serological analysis. This requires the expression of the recombinant protein in a suitable host. <em>Escherichia coli</em> expression systems are the preferred choice for the production of recombinant proteins<sup><a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a></sup>. In order to achieve high-level expression of our recombinant vaccine protein in <em>E. coli</em> (strain K12), codon optimization was carried out. Stable mRNA structure, codon adaptability index (0.87), and the GC content (50.26%) were favourable for high-level expression of the protein in bacteria. The next step projected at the moment, is to express this peptide in a bacterial system and perform the various immunological assays needed to validate the results obtained here through immuno-informatics analyses.</p></div><div class="section" id="sec-22"><h2 class="">Conclusions</h2><p id="p-50">COVID-19 is reaching entire globe and emergence of the causative agent SARS-CoV-2 kept us nowhere as we are racing for treatment and prevention. This multi-epitope chimera that we named as CoV-RMEN possess domains of RBD and NTD segments of S, M and E proteins all of which showed significant antigenic properties compared to any other viral proteins. This chimera also includes potential CTL, HTL and B-cell epitopes to ensure humoral as well as cellular immune response and the optimal expression and stability of the chimera was validated. With multiple limitations and high cost requirements for the attenuated vaccine preparation for contagious agents like SARS-CoV-2, this chimeric peptide vaccine candidate gives us the hope to ensure it’s availability and relatively cheap option to reach entire world. This CoV-RMEN can be very effective measure against COVID-19 to reach globally. Hence, this could be cloned, expressed and tried for <em>in vivo</em> validations and animal trials at the laboratory level.</p></div><div class="section" id="sec-23"><h2 class="">Methods</h2><div id="sec-24" class="subsection"><h3>Sequence retrieval and structure generation</h3><p id="p-51">A total of 250 partial and complete genome sequences of SARS-CoV-2 were retrieved from NCBI (Supplementary Table 5). We aligned these sequences through MAFFT online server (<a href="https://mafft.cbrc.jp/alignment/server/">https://mafft.cbrc.jp/alignment/server/</a>) using default parameters, and Wu-Kabat protein variability was analyzed in Protein variability server (<a href="http://imed.med.ucm.es/PVS/">http://imed.med.ucm.es/PVS/</a>) with respect to NCBI (National Center for Biotechnology Information, <a href="https://www.ncbi.nlm.nih.gov/protein">https://www.ncbi.nlm.nih.gov/protein</a>) reference genome (Accession no : <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_045512.2&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015164.atom">NC_045512.2</a>). Despite having minor heterogeneity in spike glycoprotein (S), membrane (M) and envelop (E) proteins, the NCBI reference genome (Accession no: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_045512.2&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015164.atom">NC_045512.2</a>) of SARS-CoV-2 from Wuhan, China was finally selected for domains and epitopes selection, secondary and tertiary (3D) structure prediction, antigenicity and allergenicity assessment, refinement and validation, molecular docking, cDNA and mRNA analysis for cloning and expression<sup><a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-21-14" class="xref-bibr" href="#ref-21">21</a></sup>. Moreover, two reference genome sequences of SARS-CoV (NC_004718.3) and MERS-CoV (NC_019843.3) were also retrieved from the NCBI database to reveal the structural heterogeneity of S protein. Multiple sequence alignment of the S proteins of the viruses was performed by CLUSTALW<sup><a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a></sup>. Since the S glycoprotein of coronaviruses is important for viral attachment to host cell, antibody against this protein acts as inhibitor<sup><a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">30</a></sup>, therefore, S proteins of SARS-CoV, MERS-CoV and SARS-CoV-2 were structurally compared using SWISS homology modeling<sup><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup> based on the protein databank (PDB) templates 6acd, 5w9h and 6vsb, respectively. The models were then optimized by energy minimization using the GROMOS96 program<sup><a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a></sup> implemented within the Swiss-PdbViewer, version 4.1.0<sup><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></sup>. The Ramachandran plots of the derived models were evaluated using a PROCHECK (version 3.5)-server to check the stereochemical properties of the modeled structures<sup><a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a></sup>. Finally, the homology models were structurally aligned using PyMOL<sup><a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a></sup>, and observed for heterogeneity in the conformations. The physiochemical properties of the S glycoprotein of the SARS-CoV-2, SARS-CoV and MERS-CoV was computed using the Protparam ExPasy-server<sup><a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a></sup>.</p></div><div id="sec-25" class="subsection"><h3>Linear B-cell epitopes prediction</h3><p id="p-52">We employed both structure and sequence-based methods for B-cell epitopes prediction. Conformational B-cell epitopes on the S protein were predicted by Ellipro (Antibody Epitope Prediction tool; <a href="http://tools.iedb.org/ellipro/">http://tools.iedb.org/ellipro/</a>) available in IEDB analysis resource<sup><a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a></sup> with the minimum score value set at 0.4 while the maximum distance selected as 6 Å. The ElliPro allows the prediction and visualization of B-cell epitopes in a given protein sequence or structure. The ElliPro method is based on the location of the residue in the protein’s three-dimensional (3D) structure. ElliPro implements three algorithms to approximate the protein shape as an ellipsoid, calculate the residue protrusion index (PI), and cluster neighboring residues based on their protrusion index (PI) value. The residues lying outside of the ellipsoid covering 90% of the inner core residues of the protein score highest PI of 0.9<sup><a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">23</a></sup>. Antigenicity of full-length S (spike glycoprotein), M (membrane protein) and E (envelope protein) proteins was predicted using VaxiJen v2.0 (<a href="http://www.ddg-pharmfac.net/vaxijen/VaxiJen/VaxiJen.html">http://www.ddg-pharmfac.net/vaxijen/VaxiJen/VaxiJen.html</a>)<sup><a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a></sup>. The linear B-cell epitopes of RBD and NTD regions of S protein, full length E and M proteins were predicted by “BepiPred Linear Epitope Prediction” (propensity scale method such as hidden Markov model)<sup><a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a></sup> and ABCPred under default parameters<sup><a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a></sup>. To find out the most probable peptide-based epitopes with better confidence level, selected peptides were further tested using VaxiJen antigenicity scores<sup><a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">51</a></sup>. The Kolaskar and Tongaonkar antigenicity profiling from IEDB analysis resource was also used for RBD and NTD segments, E and M proteins<sup><a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a></sup>.</p></div><div id="sec-26" class="subsection"><h3>Screening for T-cell epitopes</h3><p id="p-53">Immune Epitope Database (IEDB) tool peptide binding to MHC class I molecules and Proteasomal cleavage/TAP transport/MHC class I combined predictor (<a href="http://tools.iedb.org/main/tcell/">http://tools.iedb.org/main/tcell/</a>) were used to screen CTL epitopes, proteasomal cleavage and transporter associated with antigen processing (TAP) with all parameters set to default. Threshold for strong binding peptides (IC<sub>50</sub>) was set at 50 nM to determine the binding and interaction potentials of helper T-cell epitope peptide and major histocompatibility complex (MHC) class I allele<sup><a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">23</a></sup>. The total score generated by the tool is a combined score of proteasome, major histocompatibility complex (MHC), TAP (N-terminal interaction), processing analysis scores. The HTL epitopes were screened using the IEDB tool “MHC-II Binding Predictions” (<a href="http://tools.iedb.org/mhcii/">http://tools.iedb.org/mhcii/</a>). The tool generated the median percentile rank for each predicted epitope through combination of three methods (Combinatorial Library, SMM-align, and Sturniolo) by comparing the score of peptide against the scores of other random five million 15-mer peptides from the SwissProt database<sup><a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a></sup>. Top five HLA epitopes for each RBD and NTD segments were docked against the respective HLA (MHC-□ and MHC-□) allele binders by interaction similarity-based protein-peptide docking system GalaxyPepDock of the GalaxyWeb, docked HLA-epitope complexes were refined in GalaxyRefineComplex and binding affinity (ΔG) was determined PROtein binDIng enerGY prediction (PRODIGY) tool<sup><a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a></sup>.</p></div><div id="sec-27" class="subsection"><h3>Population coverage by CTL and HTL epitopes</h3><p id="p-54">Due to the MHC restriction of T-cell response, the peptides with more different HLA binding specificities mean more population coverage in defined geographical regions and ethnic groups where-and-to-whom the peptide-based vaccine might be employed. The predicted T-cell epitopes were shortlisted based on the aligned Artificial Neural Network (ANN) with half-maximal inhibitory concentration (annIC<sub>50</sub>) values up to 50 nM. The IEDB “Population Coverage” tool (<a href="http://tools.iedb.org/population/">http://tools.iedb.org/population/</a>)<sup><a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a></sup> was used to determine the world human population coverage by the shortlisted CTL and HTL epitopes. We used multi-epitopes involving both CTL and HTL epitopes to have the higher probability of larger human population coverage worldwide. We used OmicsCircos to visualize the association between world population and different ethnic groups<sup><a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a></sup>.</p></div><div id="sec-28" class="subsection"><h3>IFN-<strong>γ</strong>-inducing epitope prediction</h3><p id="p-55">Potential IFN-γ epitopes of all the selected antigenic sites of RBD, NTD, envelop protein B-cell epitope (EBE), and membrane protein B-cell epitope (MBE) were predicted by “IFNepitope” server (<a href="http://crdd.osdd.net/raghava/">http://crdd.osdd.net/raghava/</a> ifnepitope/scan.php). To identify the set of epitopes associated with MHC alleles that would maximize the population coverage, we adopted the “Motif and SVM hybrid” (MERCI: Motif-EmeRging and with Classes-Identification, and SVM) approach. The tool generates overlapping sequences from the query antigens for IFN-γ-inducing epitope prediction. The estimated population coverage represents the percentage of individuals within the population that are likely to elicit an immune response to at least one T cell epitope from the set, and finally the coverage was observed by adding epitopes associated with any of the remaining MHC alleles<sup><a id="xref-ref-2-10" class="xref-bibr" href="#ref-2">2</a></sup>. The prediction is based on a dataset of IFN-γ-inducing and IFN-γ-noninducing MHC allele binders<sup><a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a></sup>.</p></div><div id="sec-29" class="subsection"><h3>Design and construction of multi-epitope vaccine candidate</h3><p id="p-56">The candidate vaccine design and construction method follows previously developed FMD peptide vaccine development protocol in Professor M Anwar Hossain’s laboratory (unpublished data, and under patent application in Bangladesh Application No. P/BD/2018/000280, date: October 1, 2018; India Patent Number: 201924036919, Date: September 13, 2019). The multi-epitope protein was constructed by positioning the selected RBD, NTD, MBE and EBE amino acid sequences linked with short, rigid and flexible linkers GG. To develop highly immunogenic recombinant proteins, two universal T-cell epitopes were used, namely, a pan-human leukocyte antigen DR-binding peptide (PADRE)<sup><a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a></sup>, and an invasin immunostimulatory sequence taken from <em>Yersinia</em> (Invasin)<sup><a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a></sup> were used to the N and C terminal of the vaccine construct respectively, linked with EGGE. Here, acidic aa, Glutamate (E) was added to balance the ratio of acidic and basic amino acids. We denoted the vaccine candidate as CoV-RMEN. VaxiJen 2.0<sup><a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">52</a></sup> and ANTIGENpro (<a href="http://scratch.proteomics.ics.uci.edu/">http://scratch.proteomics.ics.uci.edu/</a>) web-servers were used to predict the antigenicity of the CoV-RMEN while the AllerTOP 2.0 (<a href="http://www.ddg-pharmfac.net/AllerTOP">http://www.ddg-pharmfac.net/AllerTOP</a>) and AllergenFP (<a href="http://ddg-pharmfac.net/AllergenFP/">http://ddg-pharmfac.net/AllergenFP/</a>) web-servers were used to predict vaccine allergenicity.</p></div><div id="sec-30" class="subsection"><h3>Physiochemical properties and solubility prediction of the CoV-RMEN</h3><p id="p-57">Various physiochemical parameters of the CoV-RMEN were assessed using the online web-server ProtParam<sup><a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">50</a></sup>. These parameters included aa residue composition, theoretical pI, instability index, <em>in vitro</em> and <em>in vivo</em> half-life, aliphatic index, molecular weight, and grand average of hydropathicity (GRAVY). The solubility of the multi-epitope vaccine peptide was evaluated using the Protein-Sol server (<a href="https://protein-sol.manchester.ac.uk/">https://protein-sol.manchester.ac.uk/</a>).</p></div><div id="sec-31" class="subsection"><h3>Secondary and tertiary structure prediction</h3><p id="p-58">Chou and Fasman secondary structure prediction server (CFSSP: <a href="https://www.biogem.org/tool/chou-fasman/">https://www.biogem.org/tool/chou-fasman/</a>) and RaptorX Property (<a href="http://raptorx.uchicago.edu/StructurePropertyPred/predict/">http://raptorx.uchicago.edu/StructurePropertyPred/predict/</a>)<sup><a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a></sup>, web-servers were used for secondary structure predictions. The RaptorX web-server (<a href="http://raptorx.uchicago.edu/">http://raptorx.uchicago.edu/</a>)<sup><a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">62</a></sup> was used to predict the three-dimensional (3D) structure and binding residues of the chosen protein. Homology modelling of the CoV-RMEN was carried out using the protein homology/analogy recognition engine (Phyre2) (<a href="http://www.sbg.bio.ic.ac.uk/~phyre2/html/page.cgi?id=index">http://www.sbg.bio.ic.ac.uk/~phyre2/html/page.cgi?id=index</a>) web-server.</p></div><div id="sec-32" class="subsection"><h3>Refinement and validation of the tertiary structure of the CoV-RMEN</h3><p id="p-59">The 3D model obtained for the CoV-RMEN was refined in a three-step process, initially energy minimization using the GROMOS96 program<sup><a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">27</a></sup> implemented within the Swiss-PdbViewer (version 4.1.0)<sup><a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">47</a></sup>. After energy minimization, the model was refined using ModRefiner (<a href="https://zhanglab.ccmb.med.umich.edu/ModRefiner/">https://zhanglab.ccmb.med.umich.edu/ModRefiner/</a>) and then GalaxyRefine server (<a href="http://galaxy.seoklab.org/cgi-bin/submit.cgi?type=REFINE">http://galaxy.seoklab.org/cgi-bin/submit.cgi?type=REFINE</a>). The construction and refinement of the CoV-RMEN was further assessed through ModRefiner server for atomic-level energy minimization. This results in improvements in both global and local structures, with more accurate side chain positions, better hydrogen-bonding networks, and fewer atomic overlaps<sup><a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a></sup>. The GalaxyRefine server was further used to improving the best local structural quality of the CoV-RMEN according to the CASP10 assessment, and ProSA-web (<a href="https://prosa.services.came.sbg.ac.at/prosa.php">https://prosa.services.came.sbg.ac.at/prosa.php</a>) was used to calculate overall quality score for a specific input structure, and this is displayed in the context of all known protein structures. The ERRAT server (<a href="http://services.mbi.ucla.edu/ERRAT/">http://services.mbi.ucla.edu/ERRAT/</a>) was also used to analyze non-bonded atom-atom interactions compared to reliable high-resolution crystallography structures. A Ramachandran plot was obtained through the RAMPAGE server (<a href="http://mordred.bioc.cam.ac.uk/~rapper/rampage.php">http://mordred.bioc.cam.ac.uk/~rapper/rampage.php</a>). The server uses the PROCHECK principle to validate a protein structure by using a Ramachandran plot and separates plots for Glycine and Proline residues<sup><a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a></sup>.</p></div><div id="sec-33" class="subsection"><h3>Immune simulation</h3><p id="p-60">To further characterize the immunogenicity and immune response profile of the CoV-RMEN, <em>in silico</em> immune simulations were conducted using the C-ImmSim server (<a href="http://150.146.2.1/C-IMMSIM/index.php">http://150.146.2.1/C-IMMSIM/index.php</a>)<sup><a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">28</a></sup>. All simulation parameters were set at default with time steps set at 1, 84, and 170 (each time step is 8 hours and time step 1 is injection at time = 0). Therefore, three injections were given at four weeks apart. The Simpson index, D (a measure of diversity) was interpreted from the plot.</p></div><div id="sec-34" class="subsection"><h3>Molecular docking of the CoV-RMEN with TLRs</h3><p id="p-61">The generation of an appropriate immune response is dependent on the interaction between an antigenic molecule and a specific immune receptor like Toll Like Receptors (TLRs)<sup><a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">33</a></sup>. Molecular docking of the CoV-RMEN with the TLR3 (PDB ID: 1ZIW) and TLR4 (PDB ID: 4G8A) receptor was performed using the High Ambiguity Driven DOCKing (HADDOCK, version 2.4)<sup><a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a></sup> web-server in order to evaluate the interaction between ligand and receptor and consequently the development of an immune response. In order to predict such interaction, data-driven docking of designed chimeric protein and TLR3 and TLR4 complexes were performed<sup><a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">33</a></sup>. In this regard, CPORT (<a href="https://milou.science.uu.nl/services/CPORT/">https://milou.science.uu.nl/services/CPORT/</a>) was used to predict active interface residues of the CoV-RMEN and TLRs. The HADDOCK server was then employed to perform docking simulations of the CoV-RMEN-TLRs complexes due to perform short MD simulations of docked complexes<sup><a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">56</a></sup>. Finally, the binding affinities of the top-ranking docking pose of chimeric protein-TLRs complexes was predicted using the PRODIGY (PROtein binDIng enerGY prediction) (<a href="https://nestor.science.uu.nl/prodigy/">https://nestor.science.uu.nl/prodigy/</a>) web-server<sup><a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">56</a></sup>.</p></div><div id="sec-35" class="subsection"><h3>Analysis of cDNA and mRNA for cloning and expression</h3><p id="p-62">Reverse translation and codon optimization were performed using the GenScript Rare Codon Analysis Tool (<a href="https://www.genscript.com/tools/rare-codon-analysis">https://www.genscript.com/tools/rare-codon-analysis</a>) in order to express the CoV-RMEN in a selected expression vector. Codon optimization was performed in order to express the final vaccine construct in the <em>E. coli</em> (strain K12). The analysis output includes the codon adaptation index (CAI) and percentage GC content, which can be used to assess protein expression levels. CAI provides information on codon usage biases; the ideal CAI score is 1.0 but &gt;0.8 is considered a good score<sup><a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a></sup>. The GC content of a sequence should range between 30–70%. GC content values outside this range suggest unfavorable effects on translational and transcriptional efficiencies<sup><a id="xref-ref-22-7" class="xref-bibr" href="#ref-22">22</a></sup>. Stability of the mRNA was measured using two different tools namely RNAfold (<a href="http://rna.tbi.univie.ac.at/cgi-bin/RNAWebSuite/RNAfold.cgi">http://rna.tbi.univie.ac.at/cgi-bin/RNAWebSuite/RNAfold.cgi</a>) and the mfold (<a href="http://unafold.rna.albany.edu/?q=mfold">http://unafold.rna.albany.edu/?q=mfold</a>) web-servers. To clone the optimized gene sequence of the final vaccine construct in <em>E. coli</em> into pETite vector (Lucigen, USA) through enzyme-free method. The primers for amplification of the DNA fragment must contain 18-nucletide short DNA sequence (5’-CGCGAACA-GATTGGAGGT-3’ in upstream of forward primer and 5’-GTGGCGGCCGCTCTATTA-3’ in upstream of reverse primer) to facilitate the enzyme-free cloning of the gene at the insertion site. Finally, the sequence of the recombinant plasmid was designed by inserting the adapted codon sequences into pETite vector using SnapGene software (from Insightful Science; available at snapgene.com).</p></div></div><div class="section" id="sec-37"><h2 class="">Conflict of interest statement</h2><p id="p-63">The authors declare no competing interests.</p></div><div class="section" id="sec-38"><h2 class="">Author contributions</h2><p id="p-64">MSR, MNH, and MRI done the overall study and also draft the manuscript. SA drafted some parts of results and discussion. MNH finally compiled the manuscript. OS, and MAS helped in final discussion and reference indexing. ASMRUA, MMR, MS and MAH contributed intellectually to the interpretation and presentation of the results. MS and MAH developed the concept of peptide vaccine development. Finally, all authors have approved the manuscript for submission.</p></div><div class="section" id="sec-39"><h2 class="">Supplementary Material</h2><p id="p-66">Supplementary information supporting the findings of the study are available in this article as Supplementary Data files, or from the corresponding author on request.</p></div><div id="sec-40" class="subsection"><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1062348503" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure11" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F11.medium.gif" width="328" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure11" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F11.medium.gif" width="328" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure11" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/31/2020.03.30.015164/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div><p id="p-67"></p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-65">The authors thank Joynob Akter Puspo, Masuda Akter and Israt Islam (MS student), and DR. Kazi Alamgir Hossain (PhD Fellow) of the Microbial Genetics and Bioinformatics Laboratory, Department of Microbiology, University of Dhaka for their cooperation, suggestions and overall encouragement during the preparation of the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">*</span> Equal contribution</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.1"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WHO</span>, <span class="cit-pub-date">2020</span>. <abbr class="cit-jnl-abbrev">Novel Coronavirus (2019-nCoV) situation reports - World Health Organization (WHO)</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Almofti</span>,  <span class="cit-name-given-names">Y. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abd-elrahman</span>,  <span class="cit-name-given-names">K. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gassmallah</span>,  <span class="cit-name-given-names">S. A. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Salih</span>,  <span class="cit-name-given-names">M. A</span></span>. <span class="cit-article-title">Multi Epitopes Vaccine Prediction against Severe Acute Respiratory Syndrome (SARS) Coronavirus Using Immunoinformatics Approaches. <em>American J</em></span>. <abbr class="cit-jnl-abbrev">Microbiol. Res</abbr>. <span class="cit-vol">6</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">94</span>–<span class="cit-lpage">114</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiol.%2BRes%26rft.volume%253D6%26rft.spage%253D94%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-etal">et al.</span> “<span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>.” <abbr class="cit-jnl-abbrev">Nat</abbr>. <span class="cit-fpage">1</span>–<span class="cit-lpage">5</span> (<span class="cit-pub-date">2020</span>)</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Badawi</span>,  <span class="cit-name-given-names">M. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title"><em>In Silico</em> Prediction of a Novel Universal Multi-epitope Peptide Vaccine in the Whole Spike Glycoprotein of MERS CoV</span>. <abbr class="cit-jnl-abbrev">Am. J. Microbiol Res</abbr>. <span class="cit-vol">4</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">101</span>–<span class="cit-lpage">21</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BMicrobiol%2BRes%26rft.volume%253D4%26rft.spage%253D101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pallesen</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Immunogenicity and structures of a rationally designed prefusion MERS-CoV spike antigen</span>. <abbr class="cit-jnl-abbrev">Proc. Nat. Aca. Sci</abbr>. <span class="cit-vol">114</span>(<span class="cit-issue">35</span>), <span class="cit-fpage">E7348</span>–<span class="cit-lpage">E7357</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNat.%2BAca.%2BSci%26rft.volume%253D114%26rft.spage%253D7348E%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">ul Qamar</span>,  <span class="cit-name-given-names">M. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saleem</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ashfaq</span>,  <span class="cit-name-given-names">U. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bari</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anwar</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-auth"><span class="cit-name-surname">&amp; Alqahtani</span></span>, S. <span class="cit-article-title">Epitope based peptide vaccine design and target site depiction against Middle East Respiratory Syndrome Coronavirus: an immune-informatics study</span>. <abbr class="cit-jnl-abbrev">J. Translational Med</abbr>. <span class="cit-vol">17</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">362</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BTranslational%2BMed%26rft.volume%253D17%26rft.spage%253D362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ahmed</span>,  <span class="cit-name-given-names">S. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quadeer</span>,  <span class="cit-name-given-names">A. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">McKay</span>,  <span class="cit-name-given-names">M. R</span></span>. <span class="cit-article-title">Preliminary identification of potential vaccine targets for the COVID-19 coronavirus (SARS-CoV-2) based on SARS-CoV immunological studies</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr>. <span class="cit-vol">12</span>(<span class="cit-issue">3</span>), <span class="cit-issue">254</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Abdelmageed</span>,  <span class="cit-name-given-names">M. I.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Design of multi epitope-based peptide vaccine against E protein of human 2019-nCoV: An immunoinformatics approach</span>. <abbr class="cit-jnl-abbrev">BioRxiv</abbr>. (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gui</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Xiang</span>,  <span class="cit-name-given-names">Y</span></span>. <span class="cit-article-title">Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>. <span class="cit-vol">14</span>(<span class="cit-issue">8</span>), <span class="cit-issue">e1007236</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wrapp</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shang</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The outbreak of SARS-CoV-2 pneumonia calls for viral vaccines</span>. <abbr class="cit-jnl-abbrev">npj Vaccines</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">18</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253Dnpj%2BVaccines%26rft.volume%253D5%26rft.spage%253D18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gralinski</span>,  <span class="cit-name-given-names">L. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Menachery</span>,  <span class="cit-name-given-names">V. D</span></span>. <span class="cit-article-title">Return of the coronavirus: 2019-nCoV</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">12</span>, <span class="cit-issue">135</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.13" data-doi="10.1038/s41467-019-10897-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structural definition of a neutralization epitope on the N-terminal domain of MERS-CoV spike glycoprotein</span>. <abbr class="cit-jnl-abbrev">Nat. Commun</abbr>. <span class="cit-vol">10</span>, <span class="cit-fpage">3068</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D10%26rft.spage%253D3068%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-019-10897-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-019-10897-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rabi</span>,  <span class="cit-name-given-names">F. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al Zoubi</span>,  <span class="cit-name-given-names">M. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kasasbeh</span>,  <span class="cit-name-given-names">G. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salameh</span>,  <span class="cit-name-given-names">D. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Al-Nasser</span>,  <span class="cit-name-given-names">A. D</span></span>. <span class="cit-article-title">SARS-CoV-2 and Coronavirus Disease 2019: What We Know So Far</span>. <abbr class="cit-jnl-abbrev">Pathogens</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">3</span>), <span class="cit-issue">231</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structural Definition of a Neutralization-sensitive Epitope on the MERS-CoV S1-NTD</span>. <abbr class="cit-jnl-abbrev">Cell Rep</abbr>. <span class="cit-vol">28</span>(<span class="cit-issue">13</span>), <span class="cit-fpage">3395</span>–<span class="cit-lpage">405</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep%26rft.volume%253D28%26rft.spage%253D3395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.16" data-doi="10.1016/j.molimm.2005.11.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">S. Q.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The expression of membrane protein augments the specific responses induced by SARS-CoV nucleocapsid DNA immunization</span>. <abbr class="cit-jnl-abbrev">Mol. Immunol</abbr>. <span class="cit-vol">43</span>, <span class="cit-fpage">1791</span>–<span class="cit-lpage">1798</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bimmunology%26rft.stitle%253DMol%2BImmunol%26rft.aulast%253DShi%26rft.auinit1%253DS.%2BQ.%26rft.volume%253D43%26rft.issue%253D11%26rft.spage%253D1791%26rft.epage%253D1798%26rft.atitle%253DThe%2Bexpression%2Bof%2Bmembrane%2Bprotein%2Baugments%2Bthe%2Bspecific%2Bresponses%2Binduced%2Bby%2BSARS-CoV%2Bnucleocapsid%2BDNA%2Bimmunization.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molimm.2005.11.005%26rft_id%253Dinfo%253Apmid%252F16423399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molimm.2005.11.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16423399&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015164.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schoeman</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fielding</span>,  <span class="cit-name-given-names">B. C</span></span>. <span class="cit-article-title">Coronavirus envelope protein: current knowledge</span>. <abbr class="cit-jnl-abbrev">Virol J</abbr>. <span class="cit-vol">16</span>, <span class="cit-issue">69</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.18" data-doi="10.1080/22221751.2020.1719902"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chan</span>,  <span class="cit-name-given-names">J. F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan</span>. <abbr class="cit-jnl-abbrev">Emerg. Microbes Infect</abbr>. <span class="cit-vol">9</span>, <span class="cit-fpage">221</span>–<span class="cit-lpage">236</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg.%2BMicrobes%2BInfect%26rft.volume%253D9%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.1080%252F22221751.2020.1719902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/22221751.2020.1719902&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.19" data-doi="10.1002/jmv.25685"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Coronavirus infections and immune responses</span>. <abbr class="cit-jnl-abbrev">J. Med. Virology</abbr> <span class="cit-vol">92</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">424</span>–<span class="cit-lpage">432</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMed.%2BVirology%26rft.volume%253D92%26rft.spage%253D424%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjmv.25685%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jmv.25685&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015164.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Epitope-Based Vaccine Target Screening against Highly Pathogenic MERS-CoV: An <em>In Silico</em> Approach Applied to Emerging Infectious Diseases</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">12</span>), <span class="cit-fpage">e0144475</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0144475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.21" data-doi="10.1038/s41598-019-40055-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shey</span>,  <span class="cit-name-given-names">R. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title"><em>In-silico</em> design of a multi-epitope vaccine candidate against onchocerciasis and related filarial diseases</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr>. <span class="cit-vol">9</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-019-40055-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-019-40055-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.22" data-doi="10.3389/fmicb.2019.01781"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yong</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Recent Advances in the Vaccine Development Against Middle East Respiratory Syndrome-Coronavirus</span>. <abbr class="cit-jnl-abbrev">Front. Microbiol</abbr>. <span class="cit-vol">10</span>, <span class="cit-fpage">1781</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMicrobiol%26rft.volume%253D10%26rft.spage%253D1781%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffmicb.2019.01781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fmicb.2019.01781&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Design of novel multi-epitope vaccines against severe acute respiratory syndrome validated through multistage molecular interaction and dynamics</span>. <abbr class="cit-jnl-abbrev">J. Biomol. Struc. Dyn</abbr>. <span class="cit-vol">37</span>(<span class="cit-issue">16</span>), <span class="cit-fpage">4345</span>–<span class="cit-lpage">4360</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiomol.%2BStruc.%2BDyn%26rft.volume%253D37%26rft.spage%253D4345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sakib</span>,  <span class="cit-name-given-names">M. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Islam</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hasan</span>,  <span class="cit-name-given-names">A. K. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Nabi</span>,  <span class="cit-name-given-names">A. H. M</span></span>. <span class="cit-article-title">Prediction of epitope-based peptides for the utility of vaccine development from fusion and glycoprotein of nipah virus using in silico approach</span>. <abbr class="cit-jnl-abbrev">Adv. Bioinform</abbr>. (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Adhikari</span>,  <span class="cit-name-given-names">U. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tayebi</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rahman</span>,  <span class="cit-name-given-names">M. M</span></span>. <span class="cit-article-title">Immunoinformatics approach for epitope-based peptide vaccine design and active site prediction against polyprotein of emerging oropouche virus</span>. <abbr class="cit-jnl-abbrev">J. Immunol. Res</abbr>. (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hebditch</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Carballo-Amador</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Charonis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Curtis</span></span> &amp; <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Warwicker</span></span>. <span class="cit-article-title">Protein– Sol: a web tool for predicting protein solubility from sequence</span>. <abbr class="cit-jnl-abbrev">Bioinform</abbr>. <span class="cit-vol">33</span>(<span class="cit-issue">19</span>), <span class="cit-fpage">3098</span>–<span class="cit-lpage">3100</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinform%26rft.volume%253D33%26rft.spage%253D3098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van Gunsteren</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span>, <span class="cit-name-surname">et al</span></span>. <abbr class="cit-jnl-abbrev">The GROMOS96 manual and user guide</abbr>. (<span class="cit-pub-date">1996</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rapin</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lund</span>,  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bernaschi</span>,  <span class="cit-name-given-names">M</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Castiglione</span>,  <span class="cit-name-given-names">M</span></span>. <span class="cit-article-title">Computational immunology meets bioinformatics: the use of prediction tools for molecular binding in the simulation of the immune system</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">4</span>) (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Biswal</span>,  <span class="cit-name-given-names">J. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bisht</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mohapatra</span>,  <span class="cit-name-given-names">J. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ranjan</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanyal</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pattnaik</span>,  <span class="cit-name-given-names">B</span></span>. “<span class="cit-article-title">Application of a recombinant capsid polyprotein (P1) expressed in a prokaryotic system to detect antibodies against foot-and-mouth disease virus serotype O</span>.” <abbr class="cit-jnl-abbrev">J. Virol. Methods</abbr> <span class="cit-vol">215</span>, <span class="cit-fpage">45</span>–<span class="cit-lpage">51</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%2BMethods%26rft.volume%253D215%26rft.spage%253D45%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tai</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Characterization of the receptor-binding domain (RBD) of 2019 novel coronavirus: implication for development of RBD protein as a viral attachment inhibitor and vaccine</span>. <abbr class="cit-jnl-abbrev">Cell. Mol. Immunol</abbr>. (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Discovery of a novel coronavirus associated with the recent pneumonia outbreak in humans and its potential bat origin</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>. (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hui</span>,  <span class="cit-name-given-names">D. S.</span></span>, et al. <span class="cit-article-title">The continuing 2019-nCoV epidemic threat of novel coronaviruses to global health—The latest 2019 novel coronavirus outbreak in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">Int. J. Inft. Dis</abbr>. <span class="cit-vol">91</span>, <span class="cit-fpage">264</span>–<span class="cit-lpage">266</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BInft.%2BDis%26rft.volume%253D91%26rft.spage%253D264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pei</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Expression of SARS-coronavirus nucleocapsid protein in <em>Escherichia coli</em> and <em>Lactococcus lactis</em> for serodiagnosis and mucosal vaccination</span>. <abbr class="cit-jnl-abbrev">Applied Microbiol. Biotech</abbr>. <span class="cit-vol">68</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">220</span>–<span class="cit-lpage">227</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2BMicrobiol.%2BBiotech%26rft.volume%253D68%26rft.spage%253D220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pang</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Potential rapid diagnostics, vaccine and therapeutics for 2019 novel Coronavirus (2019-ncoV): a systematic review</span>. <abbr class="cit-jnl-abbrev">J. Clinical Med</abbr>. <span class="cit-vol">9</span>(<span class="cit-issue">3</span>), <span class="cit-issue">623</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frieman</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> “<span class="cit-article-title">SARS coronavirus and innate immunity</span>.” <abbr class="cit-jnl-abbrev">Virus Research</abbr>. <span class="cit-vol">133</span>(<span class="cit-issue"><em>1</em></span>), <span class="cit-fpage">101</span>–<span class="cit-lpage">112</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BResearch%26rft.volume%253D133%26rft.spage%253D101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.36" data-doi="10.1016/j.virusres.2014.11.021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Millet</span>,  <span class="cit-name-given-names">J. K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Whittaker</span>,  <span class="cit-name-given-names">G. R</span></span>. “<span class="cit-article-title">Host cell proteases: Critical determinants of coronavirus tropism and pathogenesis</span>.” <abbr class="cit-jnl-abbrev">Virus Res</abbr>. <span class="cit-vol">202</span>, <span class="cit-fpage">120</span>–<span class="cit-lpage">134</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes%26rft.volume%253D202%26rft.spage%253D120%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2014.11.021%26rft_id%253Dinfo%253Apmid%252F25445340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2014.11.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25445340&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015164.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jaimes</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andre</span>,  <span class="cit-name-given-names">N. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Millet</span>,  <span class="cit-name-given-names">J. K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Whittaker</span>,  <span class="cit-name-given-names">G. R</span></span>. <span class="cit-article-title">Structural modeling of 2019-novel coronavirus (nCoV) spike protein reveals a proteolytically-sensitive activation loop as a distinguishing feature compared to SARS-CoV and related SARS-like coronaviruses</span>. <abbr class="cit-jnl-abbrev">arXiv</abbr>. (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.38" data-doi="10.1111/j.1365-2567.2006.02452.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">CD4(+) Th1 cells promote CD8(+) Tc1 cell survival, memory response, tumor localization and therapy by targeted delivery of interleukin 2 via acquired pMHC I complexes</span>. <abbr class="cit-jnl-abbrev">Immunology</abbr> <span class="cit-vol">120</span>, <span class="cit-fpage">148</span>–<span class="cit-lpage">159</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BImmunology%26rft.stitle%253DSci.%2BImmunol.%26rft.aulast%253DHuang%26rft.auinit1%253DH.%26rft.volume%253D120%26rft.issue%253D2%26rft.spage%253D148%26rft.epage%253D159%26rft.atitle%253DCD4%252B%2BTh1%2Bcells%2Bpromote%2BCD8%252B%2BTc1%2Bcell%2Bsurvival%252C%2Bmemory%2Bresponse%252C%2Btumor%2Blocalization%2Band%2Btherapy%2Bby%2Btargeted%2Bdelivery%2Bof%2Binterleukin%2B2%2Bvia%2Bacquired%2BpMHC%2BI%2Bcomplexes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2567.2006.02452.x%26rft_id%253Dinfo%253Apmid%252F17274112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2567.2006.02452.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17274112&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015164.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mohammad</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title"><em>In silico</em> design of a chimeric protein containing antigenic fragments of Helicobacter pylori; a bioinformatic approach</span>. <abbr class="cit-jnl-abbrev">The Open Microbiol. J</abbr>. <span class="cit-vol">10</span>, <span class="cit-issue">97</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meza</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ascencio</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sierra-Beltrán</span>,  <span class="cit-name-given-names">A. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Torres</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Angulo</span>,  <span class="cit-name-given-names">C</span></span>. <span class="cit-article-title">A novel design of a multi-antigenic, multistage and multi-epitope vaccine against <em>Helicobacter pylori</em>: an <em>in silico</em> approach</span>. <abbr class="cit-jnl-abbrev">Infect. Genet. Evol</abbr>. <span class="cit-vol">49</span>, <span class="cit-fpage">309</span>–<span class="cit-lpage">317</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect.%2BGenet.%2BEvol%26rft.volume%253D49%26rft.spage%253D309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.41" data-doi="10.1128/mBio.00638-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Totura</span>,  <span class="cit-name-given-names">A. L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Toll-like receptor 3 signaling via TRIF contributes to a protective innate immune response to severe acute respiratory syndrome coronavirus infection</span>. <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">e00638</span>–<span class="cit-lpage">15</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D6%26rft.spage%253De00638%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00638-15%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.00638-15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mosaddeghi</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">Therapeutic approaches for COVID-19 based on the dynamics of interferon-mediated immune responses</abbr>. (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zander</span>,  <span class="cit-name-given-names">R. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Th1-like plasmodium-specific memory CD4+ T cells support humoral immunity</span>. <abbr class="cit-jnl-abbrev">Cell Rep</abbr>. <span class="cit-vol">21</span>(<span class="cit-issue">7</span>), <span class="cit-fpage">1839</span>–<span class="cit-lpage">1852</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep%26rft.volume%253D21%26rft.spage%253D1839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.44" data-doi="10.1038/nm0202-166"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carvalho</span>,  <span class="cit-name-given-names">L. H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">IL-4-secreting CD4+ T cells are crucial to the development of CD8+ T-cell responses against malaria liver stages</span>. <abbr class="cit-jnl-abbrev">Nat. Med</abbr>. <span class="cit-vol">8</span>, <span class="cit-fpage">166</span>–<span class="cit-lpage">70</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmedicine%26rft.stitle%253DNat%2BMed%26rft.aulast%253DCarvalho%26rft.auinit1%253DL.%2BH.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D166%26rft.epage%253D170%26rft.atitle%253DIL-4-secreting%2BCD4%252B%2BT%2Bcells%2Bare%2Bcrucial%2Bto%2Bthe%2Bdevelopment%2Bof%2BCD8%252B%2BT-cell%2Bresponses%2Bagainst%2Bmalaria%2Bliver%2Bstages.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm0202-166%26rft_id%253Dinfo%253Apmid%252F11821901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm0202-166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11821901&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015164.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173600600026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoque</span>,  <span class="cit-name-given-names">M. N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Metagenomic deep sequencing reveals association of microbiome signature with functional biases in bovine mastitis</span>. <abbr class="cit-jnl-abbrev">Sci. Rep</abbr>. <span class="cit-vol">9</span>, <span class="cit-issue">13536</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.46" data-doi="10.1093/nar/gky427"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> “<span class="cit-article-title">SWISS-MODEL: homology modelling of protein structures and complexes</span>.” <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">46</span>(<span class="cit-issue">W1</span>), <span class="cit-fpage">W296</span>–<span class="cit-lpage">W303</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DW296%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky427%26rft_id%253Dinfo%253Apmid%252F29788355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29788355&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015164.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.47" data-doi="10.1002/elps.1150181505"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guex</span>,  <span class="cit-name-given-names">N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Peitsch</span>,  <span class="cit-name-given-names">M. C</span></span>. “<span class="cit-article-title">SWISS-MODEL and the Swiss-Pdb Viewer: an environment for comparative protein modeling</span>.” <abbr class="cit-jnl-abbrev">Electrophoresis</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">15</span>), <span class="cit-fpage">2714</span>–<span class="cit-lpage">2723</span> (<span class="cit-pub-date">1997</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElectrophoresis%26rft.stitle%253DElectrophoresis%26rft.aulast%253DGuex%26rft.auinit1%253DN.%26rft.volume%253D18%26rft.issue%253D15%26rft.spage%253D2714%26rft.epage%253D2723%26rft.atitle%253DSWISS-MODEL%2Band%2Bthe%2BSwiss-PdbViewer%253A%2Ban%2Benvironment%2Bfor%2Bcomparative%2Bprotein%2Bmodeling.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Felps.1150181505%26rft_id%253Dinfo%253Apmid%252F9504803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/elps.1150181505&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9504803&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015164.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071948300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Laskowski</span>,  <span class="cit-name-given-names">R. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">MacArthur</span>,  <span class="cit-name-given-names">M. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moss</span>,  <span class="cit-name-given-names">D. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Thornton</span>,  <span class="cit-name-given-names">J. M</span></span>. “<span class="cit-article-title">PROCHECK: a program to check the stereochemical quality of protein structures</span>.” <abbr class="cit-jnl-abbrev">J. Applied Crystal</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">283</span>–<span class="cit-lpage">291</span> (<span class="cit-pub-date">1993</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BApplied%2BCrystal%26rft.volume%253D26%26rft.spage%253D283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Faure</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">iPBAvizu: a PyMOL plugin for an efficient 3D protein structure superimposition approach</span>. <abbr class="cit-jnl-abbrev">Source Code Biol. Med</abbr>. <span class="cit-vol">14</span>, <span class="cit-issue">5</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gasteiger</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Protein identification and analysis tools on the ExPASy server. <em>The Proteomics Protocols Handbook</em></span>, <abbr class="cit-jnl-abbrev">Springer</abbr>: <span class="cit-fpage">571</span>–<span class="cit-lpage">607</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kringelum</span>,  <span class="cit-name-given-names">J. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lundegaard</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lund</span>,  <span class="cit-name-given-names">O.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">M</span></span>. <span class="cit-article-title">Reliable B cell epitope predictions: Impacts of method development and improved benchmarking</span>. <abbr class="cit-jnl-abbrev">PLoS Comput. Biol</abbr>. <span class="cit-vol">8</span>(<span class="cit-issue">12</span>), <span class="cit-issue">e1002829</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Doytchinova</span>,  <span class="cit-name-given-names">I. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Flower</span>,  <span class="cit-name-given-names">D. R</span></span>. <span class="cit-article-title">VaxiJen: a server for prediction of protective antigens, tumour antigens and subunit vaccines</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>), <span class="cit-issue">4</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Larsen</span>,  <span class="cit-name-given-names">J. E. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lund</span>,  <span class="cit-name-given-names">O.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">M</span></span>. <span class="cit-article-title">Improved method for predicting linear B-cell epitopes</span>. <abbr class="cit-jnl-abbrev">Immunome Res</abbr>. <span class="cit-vol">2</span>(<span class="cit-issue">1</span>), <span class="cit-issue">2</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saha</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Raghava</span>  <span class="cit-name-given-names">G. P. S</span></span>. “<span class="cit-article-title">Prediction of continuous BLJ using recurrent neural network</span>.” <abbr class="cit-jnl-abbrev">Proteins: Structure, Function, Bioinformatics</abbr> <span class="cit-vol">65</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">40</span>–<span class="cit-lpage">48</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%253A%2BStructure%252C%2BFunction%252C%2BBioinformatics%26rft.volume%253D65%26rft.spage%253D40%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.55" data-doi="10.1016/0014-5793(90)80535-Q"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kolaskar</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Tongaonkar</span>,  <span class="cit-name-given-names">P. C</span></span>. “<span class="cit-article-title">A semi empirical method for prediction of antigenic determinants on protein antigens</span>.” <abbr class="cit-jnl-abbrev">FEBS Letters</abbr> <span class="cit-vol">276</span>(<span class="cit-issue">1-2</span>), <span class="cit-fpage">172</span>–<span class="cit-lpage">174</span> (<span class="cit-pub-date">1990</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DKolaskar%26rft.auinit1%253DA.%2BS.%26rft.volume%253D276%26rft.issue%253D1-2%26rft.spage%253D172%26rft.epage%253D174%26rft.atitle%253DA%2Bsemi-empirical%2Bmethod%2Bfor%2Bprediction%2Bof%2Bantigenic%2Bdeterminants%2Bon%2Bprotein%2Bantigens.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0014-5793%252890%252980535-Q%26rft_id%253Dinfo%253Apmid%252F1702393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0014-5793(90)80535-Q&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1702393&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015164.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990EP34200044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.56" data-doi="10.1093/bioinformatics/btw514"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xue</span>,  <span class="cit-name-given-names">L. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodrigues</span>,  <span class="cit-name-given-names">J. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kastritis</span>,  <span class="cit-name-given-names">P. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonvin</span>,  <span class="cit-name-given-names">A. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Vangone</span>,  <span class="cit-name-given-names">A</span></span>. “<span class="cit-article-title">PRODIGY: a web server for predicting the binding affinity of protein–protein complexes</span>.” <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>(<span class="cit-issue">23</span>), <span class="cit-fpage">3676</span>–<span class="cit-lpage">3678</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw514%26rft_id%253Dinfo%253Apmid%252F27503228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw514&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27503228&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015164.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bui</span>,  <span class="cit-name-given-names">H. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sidney</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dinh</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Southwood</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Newman</span>,  <span class="cit-name-given-names">M. J</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sette</span>,  <span class="cit-name-given-names">A</span></span>. <span class="cit-article-title">Predicting population coverage of T-cell epitope-based diagnostics and vaccines</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">1</span>),<span class="cit-issue">153</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoque</span>,  <span class="cit-name-given-names">M. N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Resistome diversity in bovine clinical mastitis microbiome, a signature concurrence</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, <span class="cit-vol">829283</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dhanda</span>,  <span class="cit-name-given-names">S. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vir</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Raghava</span>,  <span class="cit-name-given-names">G. P</span></span>. “<span class="cit-article-title">Designing of interferon-gamma inducing MHC class-II binders</span>.” <abbr class="cit-jnl-abbrev">Biology Direct</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>), <span class="cit-issue">30</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.60"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Agadjanyan</span>,  <span class="cit-name-given-names">M. G.</span></span> <span class="cit-etal">et al.</span> “<span class="cit-article-title">Prototype Alzheimer’s disease vaccine using the β-amyloid and promiscuous T cell epitope pan HLA DR-binding peptide</span>.” <abbr class="cit-jnl-abbrev">The J. Immunology</abbr> <span class="cit-vol">174</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">1580</span>–<span class="cit-lpage">1586</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJ.%2BImmunology%26rft.volume%253D174%26rft.spage%253D1580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Co-expression of the C-terminal domain of <em>Yersinia enterocolitica</em> invasin enhances the efficacy of classical swine-fever-vectored vaccine based on human adenovirus</span>. <abbr class="cit-jnl-abbrev">J. Biosci</abbr>. <span class="cit-vol">40</span>, <span class="cit-fpage">79</span>–<span class="cit-lpage">90</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiosci%26rft.volume%253D40%26rft.spage%253D79%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.30.015164v1.62"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Källberg</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Margaryan</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">J</span></span>. <span class="cit-chapter-title">RaptorX server: a resource for template-based protein structure modeling</span>. In <span class="cit-source">Protein Structure Prediction</span> (pp. <span class="cit-fpage">17</span>–<span class="cit-lpage">27</span>), <span class="cit-publ-name">Humana Press</span>, <span class="cit-publ-loc">New York, NY</span>. <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.63" data-doi="10.1016/j.bpj.2011.10.024"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y</span></span>. “<span class="cit-article-title">Improving the physical realism and structural accuracy of protein models by a two-step atomic-level energy minimization</span>.” <abbr class="cit-jnl-abbrev">Biophysical J</abbr>. <span class="cit-vol">101</span>(<span class="cit-issue">10</span>), <span class="cit-fpage">2525</span>–<span class="cit-lpage">2534</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft.stitle%253DBiophys.%2BJ%26rft.aulast%253DXu%26rft.auinit1%253DD.%26rft.volume%253D101%26rft.issue%253D10%26rft.spage%253D2525%26rft.epage%253D2534%26rft.atitle%253DImproving%2Bthe%2Bphysical%2Brealism%2Band%2Bstructural%2Baccuracy%2Bof%2Bprotein%2Bmodels%2Bby%2Ba%2Btwo-step%2Batomic-level%2Benergy%2Bminimization.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bpj.2011.10.024%26rft_id%253Dinfo%253Apmid%252F22098752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bpj.2011.10.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22098752&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F31%2F2020.03.30.015164.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297143500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.64"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lovell</span>,  <span class="cit-name-given-names">S. C.</span></span> <span class="cit-etal">et al.</span> “<span class="cit-article-title">Structure validation by Cα geometry □, ψ and Cβ deviation</span>.” <abbr class="cit-jnl-abbrev">Proteins: Structure, Function, Bioinformatics</abbr> <span class="cit-vol">50</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">437</span>–<span class="cit-lpage">450</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%253A%2BStructure%252C%2BFunction%252C%2BBioinformatics%26rft.volume%253D50%26rft.spage%253D437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.65"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De Vries</span>,  <span class="cit-name-given-names">S. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Dijk</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Bonvin</span>,  <span class="cit-name-given-names">A. M</span></span>. <span class="cit-article-title">The HADDOCK web server for data-driven biomolecular docking</span>. <abbr class="cit-jnl-abbrev">Nat. Protoc</abbr>. <span class="cit-vol">5</span>(<span class="cit-issue">5</span>), <span class="cit-issue">883</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.30.015164v1.66"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morla</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makhija</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">S</span></span>. “<span class="cit-article-title">Synonymous codon usage pattern in glycoprotein gene of rabies virus</span>.” <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">584</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">6</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D584%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.30.015164v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.30.015164v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.30.016832v1" title="Sequence analysis of SARS-CoV-2 genome reveals features important for vaccine design" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.30.015461v1" title="Potent neutralizing antibodies in the sera of convalescent COVID-19 patients are directed against conserved linear epitopes on the SARS-CoV-2 spike protein" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 31, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.30.015164v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.30.015164v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.30.015164v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.30.015164v1" class="active" data-icon-position="" data-hide-link-title="0">Epitope-based chimeric peptide vaccine design against S, M and E proteins of SARS-CoV-2 etiologic agent of global pandemic COVID-19: an in silico approach</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1219018" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-N8QZHrS2Uut2AGi8WcoQhH4bc2fDhnk_FpzLyQ2ntq0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1219018" id="node1219018--2" data-pisa="biorxiv;2020.03.30.015164v1" data-pisa-master="biorxiv;2020.03.30.015164" data-seqnum="1219018" data-apath="/biorxiv/early/2020/03/31/2020.03.30.015164.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Epitope-based chimeric peptide vaccine design against S, M and E proteins of SARS-CoV-2 etiologic agent of global pandemic COVID-19: an <em>in silico</em> approach</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">M. Shaminur</span> <span class="nlm-surname">Rahman</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">M. Nazmul</span> <span class="nlm-surname">Hoque</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">M. Rafiul</span> <span class="nlm-surname">Islam</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Salma</span> <span class="nlm-surname">Akter</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">A. S. M.</span> <span class="nlm-surname">Rubayet-Ul-Alam</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Mohammad Anwar</span> <span class="nlm-surname">Siddique</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Otun</span> <span class="nlm-surname">Saha</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Md. Mizanur</span> <span class="nlm-surname">Rahaman</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Munawar</span> <span class="nlm-surname">Sultana</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">M. Anwar</span> <span class="nlm-surname">Hossain</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.30.015164; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.30.015164 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.30.015164v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015164v1&amp;title=Epitope-based%20chimeric%20peptide%20vaccine%20design%20against%20S%2C%20M%20and%20E%20proteins%20of%20SARS-CoV-2%20etiologic%20agent%20of%20global%20pandemic%20COVID-19%3A%20an%20in%20silico%20approach" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015164v1&amp;title=Epitope-based%20chimeric%20peptide%20vaccine%20design%20against%20S%2C%20M%20and%20E%20proteins%20of%20SARS-CoV-2%20etiologic%20agent%20of%20global%20pandemic%20COVID-19%3A%20an%20in%20silico%20approach" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015164v1&amp;text=Epitope-based%20chimeric%20peptide%20vaccine%20design%20against%20S%2C%20M%20and%20E%20proteins%20of%20SARS-CoV-2%20etiologic%20agent%20of%20global%20pandemic%20COVID-19%3A%20an%20in%20silico%20approach" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015164v1&amp;title=Epitope-based%20chimeric%20peptide%20vaccine%20design%20against%20S%2C%20M%20and%20E%20proteins%20of%20SARS-CoV-2%20etiologic%20agent%20of%20global%20pandemic%20COVID-19%3A%20an%20in%20silico%20approach" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015164v1&amp;t=Epitope-based%20chimeric%20peptide%20vaccine%20design%20against%20S%2C%20M%20and%20E%20proteins%20of%20SARS-CoV-2%20etiologic%20agent%20of%20global%20pandemic%20COVID-19%3A%20an%20in%20silico%20approach" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015164v1&amp;title=Epitope-based%20chimeric%20peptide%20vaccine%20design%20against%20S%2C%20M%20and%20E%20proteins%20of%20SARS-CoV-2%20etiologic%20agent%20of%20global%20pandemic%20COVID-19%3A%20an%20in%20silico%20approach" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015164v1&amp;title=Epitope-based%20chimeric%20peptide%20vaccine%20design%20against%20S%2C%20M%20and%20E%20proteins%20of%20SARS-CoV-2%20etiologic%20agent%20of%20global%20pandemic%20COVID-19%3A%20an%20in%20silico%20approach" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1219018" id="node1219018--3" data-pisa="biorxiv;2020.03.30.015164v1" data-pisa-master="biorxiv;2020.03.30.015164" data-seqnum="1219018" data-apath="/biorxiv/early/2020/03/31/2020.03.30.015164.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Epitope-based chimeric peptide vaccine design against S, M and E proteins of SARS-CoV-2 etiologic agent of global pandemic COVID-19: an <em>in silico</em> approach</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">M. Shaminur</span> <span class="nlm-surname">Rahman</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">M. Nazmul</span> <span class="nlm-surname">Hoque</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">M. Rafiul</span> <span class="nlm-surname">Islam</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Salma</span> <span class="nlm-surname">Akter</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">A. S. M.</span> <span class="nlm-surname">Rubayet-Ul-Alam</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Mohammad Anwar</span> <span class="nlm-surname">Siddique</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Otun</span> <span class="nlm-surname">Saha</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Md. Mizanur</span> <span class="nlm-surname">Rahaman</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Munawar</span> <span class="nlm-surname">Sultana</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">M. Anwar</span> <span class="nlm-surname">Hossain</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.30.015164; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.30.015164 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Epitope-based%20chimeric%20peptide%20vaccine%20design%20against%20S%2C%20M%20and%20E%20proteins%20of%20SARS-CoV-2%20etiologic%20agent%20of%20global%20pandemic%20COVID-19%3A%20an%20in%20silico%20approach&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Rahman%2C+M.+Shaminur&amp;rft.au=Hoque%2C+M.+Nazmul&amp;rft.au=Islam%2C+M.+Rafiul&amp;rft.au=Akter%2C+Salma&amp;rft.au=Rubayet-Ul-Alam%2C+A.+S.+M.&amp;rft.au=Siddique%2C+Mohammad+Anwar&amp;rft.au=Saha%2C+Otun&amp;rft.au=Rahaman%2C+Md.+Mizanur&amp;rft.au=Sultana%2C+Munawar&amp;rft.au=Hossain%2C+M.+Anwar"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1219018/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1219018/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1219018/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1219018/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1219018/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1219018/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1219018/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1219018/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1219018/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1219018/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1219018/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1219018/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015164v1&amp;count=horizontal&amp;via=&amp;text=Epitope-based%20chimeric%20peptide%20vaccine%20design%20against%20S%2C%20M%20and%20E%20proteins%20of%20SARS-CoV-2%20etiologic%20agent%20of%20global%20pandemic%20COVID-19%3A%20an%20in%20silico%20approach&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015164v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.30.015164v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.30.015164v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
