<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2015/12/23/020834.atom" />
<meta name="HW.pisa" content="biorxiv;020834v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="MERS-CoV recombination: implications about the reservoir and potential for adaptation" />
<meta name="DC.Identifier" content="10.1101/020834" />
<meta name="DC.Date" content="2015-12-23" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2015, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Recombination is a process that unlinks neighbouring loci allowing for independent evolutionary trajectories within genomes of many organisms. If not properly accounted for, recombination can compromise many evolutionary analyses. In addition, when dealing with organisms that are not obligately sexually reproducing, recombination gives insight into the rate at which distinct genetic lineages come into contact. Since June, 2012, Middle East respiratory syndrome coronavirus (MERS-CoV) has caused 1106 laboratory-confirmed infections, with 421 MERS-CoV associated deaths as of April 16, 2015. Although bats are considered as the likely ultimate source of zoonotic betacoronaviruses, dromedary camels have been consistently implicated as the source of current human infections in the Middle East. In this paper we use phylogenetic methods and simulations to show that MERS-CoV genome has likely undergone numerous recombinations recently. Recombination in MERS-CoV implies frequent co-infection with distinct lineages of MERS-CoV, probably in camels given the current understanding of MERS-CoV epidemiology." />
<meta name="DC.Contributor" content="Gytis Dudas" />
<meta name="DC.Contributor" content="Andrew Rambaut" />
<meta name="article:published_time" content="2015-12-23" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="MERS-CoV recombination: implications about the reservoir and potential for adaptation" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Recombination is a process that unlinks neighbouring loci allowing for independent evolutionary trajectories within genomes of many organisms. If not properly accounted for, recombination can compromise many evolutionary analyses. In addition, when dealing with organisms that are not obligately sexually reproducing, recombination gives insight into the rate at which distinct genetic lineages come into contact. Since June, 2012, Middle East respiratory syndrome coronavirus (MERS-CoV) has caused 1106 laboratory-confirmed infections, with 421 MERS-CoV associated deaths as of April 16, 2015. Although bats are considered as the likely ultimate source of zoonotic betacoronaviruses, dromedary camels have been consistently implicated as the source of current human infections in the Middle East. In this paper we use phylogenetic methods and simulations to show that MERS-CoV genome has likely undergone numerous recombinations recently. Recombination in MERS-CoV implies frequent co-infection with distinct lineages of MERS-CoV, probably in camels given the current understanding of MERS-CoV epidemiology.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2015/01/01" />
<meta name="citation_mjid" content="biorxiv;020834v2" />
<meta name="citation_id" content="020834v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/020834v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/020834v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/020834v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834.full.pdf" />
<meta name="citation_doi" content="10.1101/020834" />
<meta name="citation_num_pages" content="23" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="020834" />
<meta name="citation_author" content="Gytis Dudas" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh" />
<meta name="citation_author_email" content="g.dudas@sms.ed.ac.uk" />
<meta name="citation_author" content="Andrew Rambaut" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh" />
<meta name="citation_author_institution" content="Centre for Immunology, Infection and Evolution at the University of Edinburgh" />
<meta name="citation_author_institution" content="Fogarty International Center, National Institutes of Health" />
<meta name="citation_reference" content="Aburizaiza AS, Mattes FM, Azhar EI, Hassan AM, Memish ZA, Muth D, Meyer B, Lattwein E, Müller M, Drosten C. 2013. Investigation of anti-MERS-coronavirus antibodies in blood donors and abbatoir workers in jeddah and makkah, kingdom of saudi arabia, fall 2012. Journal of Infectious Diseases. p. jit589." />
<meta name="citation_reference" content="Adney DR, van Doremalen N, Brown VR, Bushmaker T, Scott D, de Wit E, Bowen RA, Munster VJ. 2014. Replication and shedding of MERS-CoV in upper respiratory tract of inoculated dromedary camels. Emerging Infectious Diseases. 20: 1999–2005." />
<meta name="citation_reference" content="Anisimova M, Nielsen R, Yang Z. 2003. Effect of recombination on the accuracy of the likelihood method for detecting positive selection at amino acid sites. Genetics. 164: 1229–1236." />
<meta name="citation_reference" content="Assiri A, Al-Tawfiq JA, Al-Rabeeah AA, et al. (13 co-authors). 2013. Epidemiological, demographic, and clinical characteristics of 47 cases of middle east respiratory syndrome coronavirus disease from saudi arabia: a descriptive study. The Lancet Infectious Diseases. 13: 752–761." />
<meta name="citation_reference" content="Baele G, Lemey P, Bedford T, Rambaut A, Suchard MA, Alekseyenko AV. 2012. Improving the accuracy of demographic and molecular clock model comparison while accommodating phylogenetic uncertainty. Molecular Biology and Evolution. 29: 2157–2167." />
<meta name="citation_reference" content="Baric RS, Shieh CK, Stohlman SA, Lai MMC. 1987. Analysis of intracellular small RNAs of mouse hepatitis virus: evidence for discontinuous transcription. Virology. 156: 342–354." />
<meta name="citation_reference" content="Bielejec F, Lemey P, Carvalho LM, Baele G, Rambaut A, Suchard MA. 2014. πBUSS: a parallel BEAST/BEAGLE utility for sequence simulation under complex evolutionary scenarios. BMC Bioinformatics. 15: 133." />
<meta name="citation_reference" content="Boni MF, de Jong MD, van Doorn HR, Holmes EC. 2010. Guidelines for Identifying Homologous Recombination Events in Influenza A Virus. PLoS ONE. 5:e10434." />
<meta name="citation_reference" content="Boni MF, Posada D, Feldman MW. 2007. An Exact Nonparametric Method for Inferring Mosaic Structure in Sequence Triplets. Genetics. 176: 1035–1047." />
<meta name="citation_reference" content="Boucher CA, Cammack N, Schipper P, Schuurman R, Rouse P, Wainberg MA, Cameron JM. 1993. High-level resistance to (-) enantiomeric 2’-deoxy-3’-thiacytidine in vitro is due to one amino acid substitution in the catalytic site of human immunodeficiency virus type 1 reverse transcriptase. Antimicrobial Agents and Chemotherapy. 37: 2231–2234." />
<meta name="citation_reference" content="Bruen TC, Philippe H, Bryant D. 2006. A Simple and Robust Statistical Test for Detecting the Presence of Recombination. Genetics. 172: 2665–2681." />
<meta name="citation_reference" content="Cauchemez S, Fraser C, Van Kerkhove MD, Donnelly CA, Riley S, Rambaut A, Enouf V, van der Werf S, Ferguson NM. 2014. Middle east respiratory syndrome coronavirus: quantification of the extent of the epidemic, surveillance biases, and transmissibility. The Lancet Infectious Diseases. 14: 50–56." />
<meta name="citation_reference" content="Chang SF, Sgro JY, Parrish CR. 1992. Multiple amino acids in the capsid structure of canine parvovirus coordinately determine the canine host range and specific antigenic and hemagglutination properties. Journal of Virology. 66: 6858–6867." />
<meta name="citation_reference" content="Chare ER, Gould EA, Holmes EC. 2003. Phylogenetic analysis reveals a low rate of homologous recombination in negative-sense RNA viruses. Journal of General Virology. 84: 2691–2703." />
<meta name="citation_reference" content="Chu DK, Poon LL, Gomaa MM, et al. (13 co-authors). 2014 MERS coronaviruses in dromedary camels, egypt. Emerging Infectious Diseases. 20: 1049–1053." />
<meta name="citation_reference" content="Corman VM, Ithete NL, Richards LR, Schoeman MC, Preiser W, Drosten C, Drexler JF. 2014a. Rooting the phylogenetic tree of middle east respiratory syndrome coronavirus by characterization of a conspecific virus from an african bat. Journal of Virology. 88: 11297–11303." />
<meta name="citation_reference" content="Corman VM, Jores J, Meyer B, et al. (13 co-authors). 2014b. Antibodies against MERS coronavirus in dromedary camels, kenya, 1992–2013. Emerging Infectious Diseases. 20." />
<meta name="citation_reference" content="Cotten M, Watson SJ, Kellam P, et al. (22 co-authors).2013. Transmission and evolution of the middle east respiratory syndrome coronavirus in saudi arabia: a descriptive genomic study. The Lancet. 382: 1993–2002." />
<meta name="citation_reference" content="Cotten M, Watson SJ, Zumla AI, et al. (20 co-authors).2014. Spread, circulation, and evolution of the middle east respiratory syndrome coronavirus. mBio. 5:e01062–13." />
<meta name="citation_reference" content="Didelot X, Falush D. 2007. Inference of bacterial microevolution using multilocus sequence data. Genetics. 175: 1251–1266." />
<meta name="citation_reference" content="Dong G, Luo J, Zhang H, Wang C, Duan M, Deliberto TJ, Nolte DL, Ji G, He H. 2011. Phylogenetic diversity and genotypical complexity of h9n2 influenza a viruses revealed by genomic sequence analysis. PLoS ONE. 6:e17212." />
<meta name="citation_reference" content="Drosten C, Muth D, Corman V, et al. (15 co-authors).2014. An observational, laboratory-based study of outbreaks of MERS-coronavirus in jeddah and riyadh, kingdom of saudi arabia, 2014. Clinical Infectious Diseases. p. ciu812." />
<meta name="citation_reference" content="Drosten C, Seilmaier M, Corman VM, et al. (22 co-authors).2013. Clinical features and virological analysis of a case of middle east respiratory syndrome coronavirus infection. The Lancet Infectious Diseases. 13: 745–751." />
<meta name="citation_reference" content="Drummond AJ, Ho SYW, Phillips MJ, Rambaut A. 2006. Relaxed phylogenetics and dating with confidence. PLoS Biol. 4:e88." />
<meta name="citation_reference" content="Drummond AJ, Suchard MA, Xie D, Rambaut A. 2012. Bayesian phylogenetics with BEAUti and the BEAST 1.7. Molecular Biology and Evolution. 29." />
<meta name="citation_reference" content="Excoffier L, Dupanloup I, Huerta-Sánchez E, Sousa VC, Foll M. 2013. Robust demographic inference from genomic and SNP data. PLoS Genet. 9:e1003905." />
<meta name="citation_reference" content="Ferreira MAR, Suchard MA. 2008. Bayesian analysis of elapsed times in continuous-time markov chains. Canadian Journal of Statistics. 36: 355–368." />
<meta name="citation_reference" content="Gierer S, Hofmann-Winkler H, Albuali WH, et al. (11 co-authors). 2013. Lack of MERS coronavirus neutralizing antibodies in humans, eastern province, saudi arabia. Emerging Infectious Diseases. 19: 2034–2036." />
<meta name="citation_reference" content="Gill MS, Lemey P, Faria NR, Rambaut A, Shapiro B, Suchard MA. 2013. Improving bayesian population dynamics inference: A coalescent-based model for multiple loci. Molecular Biology and Evolution. 30: 713–724." />
<meta name="citation_reference" content="Gire SK, Goba A, Andersen KG, et al. (58 co-authors).2014. Genomic surveillance elucidates ebola virus origin and transmission during the 2014 outbreak. Science. 345: 1369–1372." />
<meta name="citation_reference" content="Goldman N, Yang Z. 1994. A codon-based model of nucleotide substitution for protein-coding DNA sequences. Molecular Biology and Evolution. 11: 725–736." />
<meta name="citation_reference" content="Gubareva LV, Kaiser L, Matrosovich MN, Soo-Hoo Y, Hayden FG. 2001. Selection of influenza virus mutants in experimentally infected volunteers treated with oseltamivir. Journal of Infectious Diseases. 183: 523–531." />
<meta name="citation_reference" content="Guindon S, Gascuel O. 2003. A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Systematic Biology. 52: 696–704." />
<meta name="citation_reference" content="Hasegawa M, Kishino H, Yano Ta. 1985. Dating of the human-ape splitting by a molecular clock of mitochondrial DNA. Journal of Molecular Evolution. 22: 160–174." />
<meta name="citation_reference" content="Hedrick PW, Thomson G. 1986. A two-locus neutrality test: Applications to humans, E. coli and lodgepole pine. Genetics. 112:135–156. PMID: 3510942." />
<meta name="citation_reference" content="Herrewegh AAPM, Smeenk I, Horzinek MC, Rottier PJM, Groot RJd. 1998. Feline coronavirus type II strains 79–1683 and 79–1146 originate from a double recombination between feline coronavirus type i and canine coronavirus. Journal of Virology. 72: 4508–4514." />
<meta name="citation_reference" content="Hill WG, Robertson A. 1968. Linkage disequilibrium in finite populations. TAG. Theoretical and applied genetics. Theoretische und angewandte Genetik. 38: 226–231." />
<meta name="citation_reference" content="Holmes EC, Urwin R, Maiden MC. 1999. The influence of recombination on the population structure and evolution of the human pathogen neisseria meningitidis. Molecular Biology and Evolution. 16: 741–749." />
<meta name="citation_reference" content="Hon CC, Lam TY, Shi ZL, Drummond AJ, Yip CW, Zeng F, Lam PY, Leung FCC. 2008. Evidence of the Recombinant Origin of a Bat Severe Acute Respiratory Syndrome (SARS)-Like Coronavirus and Its Implications on the Direct Ancestor of SARS Coronavirus. Journal of Virology. 82: 1819–1826." />
<meta name="citation_reference" content="Keck JG, Matsushima GK, Makino S, Fleming JO, Vannier DM, Stohlman SA, Lai MM. 1988. In vivo RNA-RNA recombination of coronavirus in mouse brain. Journal of Virology. 62: 1810–1813." />
<meta name="citation_reference" content="Kirkegaard K, Baltimore D. 1986. The mechanism of RNA recombination in poliovirus. Cell. 47: 433–443." />
<meta name="citation_reference" content="Kosakovsky Pond SL, Posada D, Gravenor MB, Woelk CH, Frost SDW. 2006. Automated phylogenetic detection of recombination using a genetic algorithm. Molecular Biology and Evolution. 23: 1891–1901." />
<meta name="citation_reference" content="Kottier SA, Cavanagh D, Britton P. 1995. Experimental evidence of recombination in coronavirus infectious bronchitis virus. Virology. 213: 569–580." />
<meta name="citation_reference" content="Lai MM, Baric RS, Makino S, Keck JG, Egbert J, Leibowitz JL, Stohlman SA. 1985. Recombination between nonsegmented RNA genomes of murine coronaviruses. Journal of Virology. 56: 449–456." />
<meta name="citation_reference" content="Lau SKP, Li KSM, Huang Y, et al. (14 co-authors).2010. Ecoepidemiology and Complete Genome Comparison of Different Strains of Severe Acute Respiratory Syndrome-Related Rhinolophus Bat Coronavirus in China Reveal Bats as a Reservoir for Acute, Self-Limiting Infection That Allows Recombination Events. Journal of Virology. 84: 2808–2819." />
<meta name="citation_reference" content="Lemey P, Suchard M, Rambaut A. 2009. Reconstructing the initial global spread of a human influenza pandemic. PLoS Currents. 1." />
<meta name="citation_reference" content="Lewontin RC. 1964. The interaction of selection and linkage. i. general considerations; heterotic models. Genetics. 49:49–67. PMID: 17248194 PMCID: PMC1210557." />
<meta name="citation_reference" content="Li KS, Guan Y, Wang J, et al. (22 co-authors).2004. Genesis of a highly pathogenic and potentially pandemic h5n1 influenza virus in eastern asia. Nature. 430: 209–213." />
<meta name="citation_reference" content="Lu L, Lycett SJ, Brown AJL. 2014. Reassortment patterns of avian influenza virus internal segments among different subtypes. BMC Evolutionary Biology. 14: 16." />
<meta name="citation_reference" content="Makino S, Keck JG, Stohlman SA, Lai MM. 1986. High-frequency RNA recombination of murine coronaviruses. Journal of Virology. 57: 729–737." />
<meta name="citation_reference" content="Maynard Smith J, Smith NH. 1998. Detecting recombination from gene trees. Molecular Biology and Evolution. 15: 590–599." />
<meta name="citation_reference" content="McVean G, Awadalla P, Fearnhead P. 2002. A coalescent-based method for detecting and estimating recombination from gene sequences. Genetics. 160: 1231–1241." />
<meta name="citation_reference" content="Memish ZA, Zumla AI, Al-Hakeem RF, Al-Rabeeah AA, Stephens GM. 2013. Family cluster of middle east respiratory syndrome coronavirus infections. New England Journal of Medicine. 368: 2487–2494." />
<meta name="citation_reference" content="Meunier J, Eyre-Walker A. 2001. The correlation between linkage disequilibrium and distance: Implications for recombination in hominid mitochondria. Molecular Biology and Evolution. 18: 2132–2135." />
<meta name="citation_reference" content="Milkman R, Bridges MM. 1990. Molecular evolution of the Escherichia coli chromosome. III. Clonal frames. Genetics. 126: 505–517." />
<meta name="citation_reference" content="Mitnaul LJ, Matrosovich MN, Castrucci MR, Tuzikov AB, Bovin NV, Kobasa D, Kawaoka Y. 2000. Balanced hemagglutinin and neuraminidase activities are critical for efficient replication of influenza a virus. Journal of Virology. 74: 6015–6020." />
<meta name="citation_reference" content="Muller HJ. 1932. Some genetic aspects of sex. The American Naturalist. 66: 118–138." />
<meta name="citation_reference" content="Müller MA, Corman VM, Jores J, et al. (12 co-authors).2014. MERS coronavirus neutralizing antibodies in camels, eastern africa, 1983–1997. Emerging Infectious Diseases. 20." />
<meta name="citation_reference" content="Müller MA, Meyer B, Corman VM, et al. (19 co-authors).2015. Presence of Middle East respiratory syndrome coronavirus antibodies in Saudi Arabia: a nationwide, crosssectional, serological study. The Lancet Infectious Diseases. 15: 559–564." />
<meta name="citation_reference" content="O’Brien JD, Minin VN, Suchard MA. 2009. Learning to count: Robust estimates for labeled distances between molecular sequences. Molecular Biology and Evolution. 26:801–814. PMID: 19131426." />
<meta name="citation_reference" content="Pond SLK, Frost SDW, Muse SV. 2005. HyPhy: hypothesis testing using phylogenies. Bioinformatics. 21: 676–679." />
<meta name="citation_reference" content="Posada D, Crandall KA. 2001. Evaluation of methods for detecting recombination from DNA sequences: Computer simulations. Proceedings of the National Academy of Sciences. 98: 13757–13762." />
<meta name="citation_reference" content="Posada D, Crandall KA. 2002. The effect of recombination on the accuracy of phylogeny estimation. Journal of Molecular Evolution. 54: 396–402." />
<meta name="citation_reference" content="Posada D, Crandall KA, Holmes EC. 2002. Recombination in Evolutionary Genomics. Annual Review of Genetics. 36: 75–97." />
<meta name="citation_reference" content="Rambaut A, Holmes E. 2009. The early molecular epidemiology of the swine-origin a/h1n1 human influenza pandemic. PLoS Currents. 1." />
<meta name="citation_reference" content="Reusken CB, Haagmans BL, Müller MA, et al. (24 co-authors).2013. Middle east respiratory syndrome coronavirus neutralising serum antibodies in dromedary camels: a comparative serological study. The Lancet Infectious Diseases. 13: 859–866." />
<meta name="citation_reference" content="Reusken CB, Messadi L, Feyisa A, et al. (17 co-authors).2014. Geographic distribution of MERS coronavirus among dromedary camels, africa. Emerging Infectious Diseases. 20: 1370–1374." />
<meta name="citation_reference" content="Robertson DL, Hahn BH, Sharp PM. 1995a. Recombination in AIDS viruses. Journal of Molecular Evolution. 40: 249–259." />
<meta name="citation_reference" content="Robertson DL, Sharp PM, McCutchan FE, Hahn BH. 1995b. Recombination in HIV-1. Nature. 374: 124–126." />
<meta name="citation_reference" content="Schierup MH, Hein J. 2000. Consequences of recombination on traditional phylogenetic analysis. Genetics. 156: 879–891." />
<meta name="citation_reference" content="Shriner D, Nickle DC, Jensen MA, Mullins JI. 2003. Potential impact of recombination on sitewise approaches for detecting positive natural selection. Genetics Research. 81: 115–121." />
<meta name="citation_reference" content="Smith GJD, Vijaykrishna D, Bahl J, et al. (13 co-authors).2009. Origins and evolutionary genomics of the 2009 swine-origin h1n1 influenza a epidemic. Nature. 459: 1122–1125." />
<meta name="citation_reference" content="Subbarao EK, London W, Murphy BR. 1993. A single amino acid in the PB2 gene of influenza a virus is a determinant of host range. Journal of Virology. 67: 1761–1764." />
<meta name="citation_reference" content="Tavaré S. 1986. Some Mathematical Questions in Biology: DNA Sequence Analysis. Lectures on Mathematics in the Life Sciences, volume 17, 57–86. American Mathematical Society." />
<meta name="citation_reference" content="Tisdale M, Kemp SD, Parry NR, Larder BA. 1993. Rapid in vitro selection of human immunodeficiency virus type 1 resistant to 3’-thiacytidine inhibitors due to a mutation in the YMDD region of reverse transcriptase. Proceedings of the National Academy of Sciences of the United States of America. 90: 5653–5656." />
<meta name="citation_reference" content="Yang Z. 1994. Maximum likelihood phylogenetic estimation from DNA sequences with variable rates over sites: Approximate methods. Journal of Molecular Evolution. 39: 306–314." />
<meta name="citation_reference" content="Zaki AM, van Boheemen S, Bestebroer TM, Osterhaus AD, Fouchier RA. 2012. Isolation of a novel coronavirus from a man with pneumonia in saudi arabia. New England Journal of Medicine. 367: 1814–1820." />
<meta name="twitter:title" content="MERS-CoV recombination: implications about the reservoir and potential for adaptation" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Recombination is a process that unlinks neighbouring loci allowing for independent evolutionary trajectories within genomes of many organisms. If not properly accounted for, recombination can compromise many evolutionary analyses. In addition, when dealing with organisms that are not obligately sexually reproducing, recombination gives insight into the rate at which distinct genetic lineages come into contact. Since June, 2012, Middle East respiratory syndrome coronavirus (MERS-CoV) has caused 1106 laboratory-confirmed infections, with 421 MERS-CoV associated deaths as of April 16, 2015. Although bats are considered as the likely ultimate source of zoonotic betacoronaviruses, dromedary camels have been consistently implicated as the source of current human infections in the Middle East. In this paper we use phylogenetic methods and simulations to show that MERS-CoV genome has likely undergone numerous recombinations recently. Recombination in MERS-CoV implies frequent co-infection with distinct lineages of MERS-CoV, probably in camels given the current understanding of MERS-CoV epidemiology." />
<meta name="og-title" property="og:title" content="MERS-CoV recombination: implications about the reservoir and potential for adaptation" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/020834v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Recombination is a process that unlinks neighbouring loci allowing for independent evolutionary trajectories within genomes of many organisms. If not properly accounted for, recombination can compromise many evolutionary analyses. In addition, when dealing with organisms that are not obligately sexually reproducing, recombination gives insight into the rate at which distinct genetic lineages come into contact. Since June, 2012, Middle East respiratory syndrome coronavirus (MERS-CoV) has caused 1106 laboratory-confirmed infections, with 421 MERS-CoV associated deaths as of April 16, 2015. Although bats are considered as the likely ultimate source of zoonotic betacoronaviruses, dromedary camels have been consistently implicated as the source of current human infections in the Middle East. In this paper we use phylogenetic methods and simulations to show that MERS-CoV genome has likely undergone numerous recombinations recently. Recombination in MERS-CoV implies frequent co-infection with distinct lineages of MERS-CoV, probably in camels given the current understanding of MERS-CoV epidemiology." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2015-12-23" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/020834v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/020834v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/020834v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/020834v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/9411" />
    <title>MERS-CoV recombination: implications about the reservoir and potential for adaptation | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"X8UqtmF3Jby-DO43etgrzHDVE7H1VWUV6Kk_6RBa0bU","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"9411","apath":"\/biorxiv\/early\/2015\/12\/23\/020834.atom","pisa":"biorxiv;020834v2","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;020834\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;020834\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;020834v2"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;020834\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;020834\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/020834v2","title":"MERS-CoV recombination: implications about the reservoir and potential for adaptation","identifier":"node\/9411"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Un3UFjFQt0Ey0dTsQ1SCzNxRM5WMgx0raYkQM2Q10UUmp882MA4J54XVzJ31IpuQdzAlsIGHgi6p1LnkOneLtoC90+6c9GgcSWFgjvwC1AK+n0vMFNp5W4xUPRCqo1goMjIvuaznWUEtWoX+nAGjmnKU0RCB1RAicfsaGC59XEuE08dLieaqrS6GpzLLbRzAtSwl1RlqvtS1esxqd0+TD3vFtv3gcecQswaQKlpblwydzKGZWDhb9RNzb5bRqGZNQ4aXAIg3l1aGhvlTQhmCEytEQomBj\\\\\\\/dmfSGW7pNp7NgbDamP71zAKC09SR56RX2SOKTpZgvIjfTBTcA0uZvGkb9A93B74ed8Ii+A3+7XMSRTvio2kiNbAKkID96exuAcA4j8dndS77fCS9uoki\\\\\\\/P87+SqE9\\\\\\\/T8\\\\\\\/QEwN3vYIaSXBkk9hs1xa80YeJDkDpyBm99O6Jj64N4q\\\\\\\/aQ6cuwhWiC0RBLsP8Xl7nkx4SFyodY\\\\\\\/ermM128yXx8XMlwU9yO0ozbCDXPpMyHGdHVGhdItlElzAN+vhmFdy536rEzeccGhMl4U72O40Z9OVpzxgiXZ+IVBQ2iRHiLxRytLRyR8Ryn3VPVj+0Az\\\\\\\/AX4yjZq6fPas3w0\\\\\\\/P0\\\\\\\/lseL+Y64aX7wyByPexhPc6C+UYUb1\\\\\\\/PpPQRe7E20datIrRAO4grCeajZaz16n25HIk20TWAHpWfcXW09Pf+V8EFuwA6Ovw3SsScGsvDh\\\\\\\/zl2K9oKnbsgpqF2pd\\\\\\\/5THGq9wrGRy2e\\\\\\\/tz6AVrxKjwBEGiz7G2shOdqm1iqTtJfvlLEiIp2GLobvNWNdTIR7f15rMXwuMn7g3FRVI\\\\\\\/F9doqVOjSPGQ4SYQ5EKdl4YL8SaqkqWQJ\\\\\\\/WnOAIqd\\\\\\\/tkjNUxoy9i4iwTtFTgfHa5qhQXAMGzwzM95XySlDUfEVldRc9vlhEV9Ac493Zl00PvFLt3GBnCyBx3p\\\\\\\/L4umAdUrni7qC2bTsm8QcRP4UMosuPfNxKnRz2lIKRka3D7gsM0HTN2yVqr2dV0wEvIWmKxtw8Gr9Z1gORCBM+\\\\\\\/GgjfVYnw9OzqlaU1EZegBGolv7a1ggqYhAMEPfrirq5WSyc8S4J+X13erxMYuDDlOuRmTxCQddxTRxxFKMVmaSzEOy0LtoGFq7wljwcvHYtCcyqu38Ddhz2axFOKzk+8hfnrk4Ki6Qs8J24+j5lx632roF\\\\\\\/CLmMPhlLBYbXq6fem3lRUYJAnAHXUw386zY4lyoSFojIYxm4Mz0jFfuungp5WRn+JoyqGylgT7lzzPfO804STiGibKMCnBDASpVsw\\\\\\\/HP4ZPTAsaj9ekTkRU8Ua9yUM=\\\u0022,\\\u0022iv\\\u0022:\\\u00227Uc2oOtcHTe7CO9Hw0PrCw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00228e087450b2f76dd864f8a5c6863991c167e335d0f66cd1f74603566730e3b35c\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/020834v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-9411 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/020834v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8ppyJRnacIkRqbLTKmM0W5jDGRMZ2QrECBHBS9tXCmI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="9411" id="node9411" data-pisa="biorxiv;020834v2" data-pisa-master="biorxiv;020834" data-apath="/biorxiv/early/2015/12/23/020834.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">MERS-CoV recombination: implications about the reservoir and potential for adaptation</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Gytis</span> <span class="nlm-surname">Dudas</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Andrew</span> <span class="nlm-surname">Rambaut</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/020834 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node9411" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gytis Dudas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute of Evolutionary Biology, University of Edinburgh</span>, Edinburgh, <span class='nlm-country'>UK</span>
</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gytis%2BDudas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dudas%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGytis%2BDudas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:g.dudas@sms.ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">g.dudas@sms.ed.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrew Rambaut </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Institute of Evolutionary Biology, University of Edinburgh</span>, Edinburgh, <span class='nlm-country'>UK</span>
</div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Centre for Immunology, Infection and Evolution at the University of Edinburgh</span>, Edinburgh, <span class='nlm-country'>UK</span>
</div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Fogarty International Center, National Institutes of Health</span>, Bethesda, MD, <span class='nlm-country'>USA</span>
</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BRambaut%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rambaut%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BRambaut%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/020834v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:9411" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:9411/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/020834v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:9411" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:9411/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/020834v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:9411" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:9411/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/020834v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:9411" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:9411/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/020834v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:9411" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:9411/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/020834v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:9411" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:9411/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Recombination is a process that unlinks neighbouring loci allowing for independent evolutionary trajectories within genomes of many organisms. If not properly accounted for, recombination can compromise many evolutionary analyses. In addition, when dealing with organisms that are not obligately sexually reproducing, recombination gives insight into the rate at which distinct genetic lineages come into contact. Since June, 2012, Middle East respiratory syndrome coronavirus (MERS-CoV) has caused 1106 laboratory-confirmed infections, with 421 MERS-CoV associated deaths as of April 16, 2015. Although bats are considered as the likely ultimate source of zoonotic betacoronaviruses, dromedary camels have been consistently implicated as the source of current human infections in the Middle East. In this paper we use phylogenetic methods and simulations to show that MERS-CoV genome has likely undergone numerous recombinations recently. Recombination in MERS-CoV implies frequent co-infection with distinct lineages of MERS-CoV, probably in camels given the current understanding of MERS-CoV epidemiology.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Recombination is an important process which expedites selection in many organisms (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Muller, 1932</a>) by unlinking loci. It also leads to different parts of recombining genomes to have different histories which, if not properly accounted for, can interfere with many genetic analyses, of which phylogenetic methods are amongst the most sensitive. Not accounting for recombination in phylogenetic analyses leads to incorrect (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Schierup and Hein, 2000</a>) and poorly supported genealogies (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Posada and Crandall, 2002</a>) and false inference of selection (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Anisimova et al., 2003</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Shriner et al., 2003</a>).</p><p id="p-4">With rising sequence availability during outbreaks of viral infectious disease, phylogenetic methods have been used to supplement our knowledge of epidemics in real time (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Smith et al., 2009</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Rambaut and Holmes, 2009</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lemey et al., 2009</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Drosten et al., 2013</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Cotten et al., 2013</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">2014</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Drosten et al., 2014</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Gire et al., 2014</a>). For some outbreaks there is little reason to suspect recombination, <em>e.g.</em> negative sense single stranded RNA viruses are thought to recombine over evolutionary, not population-level, time scales (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chare et al., 2003</a>). Observable recombination in RNA viruses requires that two conditions are met: that viruses from distinct lineages co-infect a host and that a mechanism for recombination exists. For example, even though influenza A virus co-infection is extremely common in birds based on genome segment reassortment patterns (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Li et al., 2004</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Dong et al., 2011</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Lu et al., 2014</a>), recombination is extremely rare or absent (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Chare et al., 2003</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Boni et al., 2010</a>). This is thought to be because template switching (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Kirkegaard and Baltimore, 1986</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Baric et al., 1987</a>), the main mechanism of recombination in RNA viruses, is mechanistically difficult for single stranded negative sense RNA viruses (see <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Chare et al. 2003</a>), and for influenza A viruses has only been convincingly shown in cell culture under extreme conditions (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Mitnaul et al., 2000</a>). When the genomic architecture of a virus is permissive to recombination, <em>i.e.</em> template switching occurs and is detectable, the extent of recombination is informative of co-infection and/or duration of infection.</p><p id="p-5">Here we focus our attention on the Middle East respiratory syndrome coronavirus (MERS-CoV) (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Zaki et al., 2012</a>), a recent zoonotic infection with a relatively high case fatality ratio (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cauchemez et al., 2014</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Memish et al., 2013</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Assiri et al., 2013</a>). Most human infections with MERS-CoV are thought to be the result of contact with <em>Camelus dromedarius</em> L., the dromedary camel, which is the presumed host of the virus. MERS-CoV, much like Severe acute respiratory syndrome coronavirus (SARS-CoV), is likely ultimately derived from bats (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Corman et al., 2014a</a>). MERS-CoV, along with Murine hepatitis virus and SARS-CoV, belongs to the Betacoronavirus genus. Betacoronavirus, as well as two other genera (Alpha- and Gammacoronavirus) out of four within the subfamily <em>Coronavirinae</em> have been shown to recombine in cell culture, <em>in vivo</em> and in eggs (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lai et al., 1985</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Makino et al., 1986</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Keck et al., 1988</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Kottier et al., 1995</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Herrewegh et al., 1998</a>). Additionally, a coronavirus lineage related to MERS-CoV which was isolated from bats appears to have recombined around the spike (S) protein (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Corman et al., 2014a</a>). In this paper we show that although the genome of MERS-CoV contains considerable amounts of rate heterogeneity between genomic regions that can interfere with detection of recombination, we do nonetheless find evidence of sustained recombination that cannot be explained by rate heterogeneity alone. This has two important consequences: one is that care has to be taken when constructing phylogenetic trees of MERS-CoV as a single tree cannot accurately describe the complete history of all loci within a recombining genome. Secondly and more importantly, the observed rates of recombination in the MERS-CoV genome is evidence of a large number of MERS-CoV co-infections in some hosts which has implications for understanding the dynamics of the virus in the animal reservoir.</p></div><div class="section" id="sec-2"><h2 class="">Methods</h2><div id="sec-3" class="subsection"><h3>Overview</h3><p id="p-6">Recombination leaves several characteristic clues in genomes:</p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-7">Alternative topologies (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Robertson et al., 1995a</a>,<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">b</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Holmes et al., 1999</a>). In some scenarios, for example if there has been a single large-scale recombination event, it is possible to clearly identify recombining fragments based on phylogenetic incongruity. Recombination can be inferred by reconstructing two or more phylogenetic trees from a partitioned alignment and looking for topological incongruity between them. Strong support for at least 2 incompatible phylogenetic trees across well-defined breakpoints is usually the most convincing evidence of recombination.</p></li><li id="list-item-2"><p id="p-8">Excessive homoplasies (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Maynard Smith and Smith, 1998</a>). The transfer of genetic material from one genetic background to another will result in apparent repeat mutations in different parts of a phylogenetic tree. However, it is possible for the same locus to undergo mutation independently, especially if the locus in question is under Darwinian selection. Detecting homoplasies alone is not sufficient to infer recombination, but should be demonstrated to occur in excess of expectation.</p></li><li id="list-item-3"><p id="p-9">Linkage disequilibrium (LD) decay (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Meunier and Eyre-Walker, 2001</a>). Linkage disequilibrium or LD is the non-random association of alleles at different loci. This is a statistic often reported for contemporaneous sequence data. In clonally (<em>i.e.</em> non-recombining) evolving organisms every allele is linked to every other allele in the genome and requires mutation to break linkage. In recombining organisms there is an expectation that LD will decay with distance between the loci, <em>i.e.</em> that loci further away from each other are more likely to be unlinked via recombination.</p></li></ul><p id="p-10">We test for each of these hallmarks of recombination in the MERS coronavirus genome using a combination of phylogenetic and linkage disequilibrium metrics. For a more detailed review of recombination detection methods see <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Posada et al. (2002)</a>.</p></div><div id="sec-4" class="subsection"><h3>Alternative topologies</h3><p id="p-11">We use the Genetic Algorithm for Recombination Detection (GARD) method (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kosakovsky Pond et al., 2006</a>), as implemented in the software package HyPhy (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Pond et al., 2005</a>), to look for alternative tree topologies in sequence data. Briefly, the method compares a model where a single tree is derived from the whole alignment and alternative models where breakpoints are introduced into the alignment and phylogenetic trees are derived independently from the resulting fragments. The presence of recombination, especially if it is recent and concentrated in some parts of the alignment, will result in two or more phylogenetic trees fitting the data better than a single tree model. It is important to note that likelihood estimation also involves other parameters, such as branch lengths, not just topology. We use GARD under a GTR (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Tavaré, 1986</a>) substitution model with Γ<sub>4</sub>-distributed rate heterogeneity amongst sites (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Yang, 1994</a>) on a dataset of 85 MERS-CoV sequences. GARD was run repeatedly until no more breakpoints could be identified in the resulting fragments.</p><p id="p-12">In addition to this test, we run BEAST (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Drummond et al., 2012</a>) on partitioned coding sequences derived from the first well-supported breakpoint inferred by GARD. We extracted the coding sequences from nucleotide positions 1-23722 and 23723-30126 (referred to as fragment 1 and 2, respectively) of MERS-CoV genomes. Independent HKY+Γ<sub>4</sub> (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hasegawa et al., 1985</a>; <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Yang, 1994</a>) nucleotide substitution models were specified for codon positions 1+2 and 3 and the analyses were run under an uncorrelated relaxed lognormal clock with an uninformative CTMC reference prior (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ferreira and Suchard, 2008</a>) on the substitution rate for 100 million states, subsampling every 10000 states. The molecular clocks and trees of each genomic partition were either linked or unlinked, giving a total of 4 models. We used the multi-locus skygrid (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gill et al., 2013</a>) as the demographic model for all analyses. Path-sampling and stepping stone sampling (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Baele et al., 2012</a>) were used to calculate marginal likelihoods and test the fit of each of the 4 models, under default parameters. In addition, 4 similar analyses were set up, but with strict molecular clocks, in order to contrast the performance of relaxed molecular clocks.</p></div><div id="sec-5" class="subsection"><h3>Excessive homoplasies</h3><p id="p-13">Testing for recombination by looking for homoplasic mutations in phylogenetic trees requires that two conditions are met. One, that recombination is rare enough, so that there is sufficient phylogenetic signal to reconstruct the “correct” phylogeny otherwise known as the clonal frame (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Milkman and Bridges, 1990</a>). Two, that alternative explanations for homoplasic mutations can be dismissed with some certainty. There is no straightforward way of testing for the former, but the latter is usually dictated by the underlying biology. For example, repeat amino acid substitutions are a well documented response of influenza viruses and HIV to drug treatment (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Gubareva et al., 2001</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Tisdale et al., 1993</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Boucher et al., 1993</a>).</p><p id="p-14">We employ two methods to test for excessive homoplasies. First, we use a maximum likelihood phylogeny inferred using PhyML (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Guindon and Gascuel, 2003</a>) under a GTR+Γ<sub>4</sub> (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Tavaré, 1986</a>; <a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Yang, 1994</a>) nucleotide substitution model to recover a single tree using a MERS-CoV dataset comprised of 85 sequences. We then reconstruct ancestral sequences at each internal node and identify the mutations that have taken place along each branch using ClonalFrameML (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Didelot and Falush, 2007</a>). Mutations are then classified as either synapomorphies, shared variation derived via common descent or apparent homoplasies, shared variation derived from convergence, depending on how many times a given mutation has arisen in the phylogeny. The drawback of this method is that it necessarily conditions on a single tree with the highest likelihood.</p><p id="p-15">We also employ BEAST (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Drummond et al., 2012</a>) to circumvent the limitation of conditioning the ancestral state reconstruction on a single tree. In addition to sampling various phylogenetic parameters from the posterior distribution BEAST is also able to map substitutions onto the branches of each MCMC-sampled phylogeny (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">O’Brien et al., 2009</a>). This method is thus capable of estimating the posterior probability of a given mutation being synapomorphic or homoplasic by integrating over different tree topologies. Homo-plasy analyses were performed on the concatenated coding sequences of MERS-CoV after partitioning the alignment into all 3 codon positions, each with an HKY nucleotide substitution model (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Hasegawa et al., 1985</a>) and no Γ-distributed rate heterogeneity amongst sites. A relaxed uncorrelated molecular clock with lognormally distributed rates (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Drummond et al., 2006</a>) under a CTMC reference prior (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Ferreira and Suchard, 2008</a>) and the flexible multi-locus skygrid as the demographic model (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Gill et al., 2013</a>) were used. The MCMC chain was run for 100 million steps, sampling every 10000 steps.</p><p id="p-16">Throughout the paper we will refer to the number of branches that have experienced a given mutation as homoplasy degree. We define the homoplasy degree to be the number of times a given mutation has originated independently minus one. For example a homoplasy degree of 1 indicates that a mutation has occured on 2 different branches in the phylogeny. That is, we assume that one of the mutations has arisen through replication error, whereas the other has potential to have been introduced via recombination and thus can be thought of as excessive. Synapomorphies, on the other hand, are states that are shared by two or more taxa through common descent and thus necessarily are those mutations that have occured exactly once in the phylogeny. They have a homoplasy degree of 0 in all figures.</p><p id="p-17">Additional tests for recombination were also performed, namely the estimation of the pairwise homoplasy index (PHI) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bruen et al., 2006</a>) and the triplet test implemented in 3Seq (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Boni et al., 2007</a>).</p></div><div id="sec-6" class="subsection"><h3>Linkage disequilibrium decay</h3><p id="p-18">In the absence of recombination every allele should exhibit a high degree of linkage with other alleles in the genome. Under two extremes - clonal reproduction without recombination and free recombination - there is no correlation between LD and genomic distance and loci should be interchangeable. This is the basis of several non-parametric permutation tests for recombination that are implemented in the software package LDhat (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">McVean et al., 2002</a>), which we used in combination with a dataset of 109 MERS-CoV genomes. Other, more complicated tests, such as composite likelihood methods, are also available but in our experience were incompatible with temporal sampling and rate heterogeneity.</p></div><div id="sec-7" class="subsection"><h3>Sequence simulations</h3><p id="p-19">To test the performance of some of the methods we simulated two sets of sequences. We use fastsimcoal2 (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Excoffier et al., 2013</a>) to simulate 10 replicate datasets that have the same dates of isolation and similar diversity to the MERS dataset with 85 sequences under no recombination.</p><p id="p-20">Additionally, we use πBUSS (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bielejec et al., 2014</a>) to simulate sequences down an MCMC-sampled phylogeny drawn at random from a linked-tree unlinked-clocks BEAST analysis described above. We modelled region-specific rate heterogeneity by simulating a 30kb “genome” and setting the molecular clock rate for the first 20kb to be 9.5 × 10<sup>−4</sup> substitutions site<sup>−1</sup>year<sup>−1</sup> and the last 10kb to be 2.85×10<sup>−3</sup>, 1.9×10<sup>−3</sup> or 1.3×10<sup>−3</sup> substitutions site<sup>−1</sup>year<sup>−1</sup>, corresponding to roughly 3-, 2- or 1.3-fold rate heterogeneity between the two parts of the simulated genome. Two replicate datasets were generated for each category of rate heterogeneity. Other than that all simulations were run under a relaxed lognormal molecular clock (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Drummond et al., 2006</a>) with standard deviation set to 7.42×10<sup>−7</sup>, HKY substitution model (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Hasegawa et al., 1985</a>) with the transition/transversion ratio parameter (<em>κ</em>) set to 6.0 and Γ-distributed rate heterogeneity with 4 categories and shape parameter 0.04 and empirical nucleotide frequencies, all derived from the results of the marginal likelihood analyses described earlier. A MERS-CoV sequence isolated from a camel (NRCE-HKU270) was provided as the starting state at the root. To include the effects of site-specific constraint we additionally carried out simulations under a Goldman-Yang codon model (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Goldman and Yang, 1994</a>) in πBUSS with empirical codon frequencies, <em>κ</em>=6.0 and dN/dS (<em>ω</em>) set to 0.1 (<em>i.e.</em> purifying selection) under same levels of rate heterogeneity and on the same phylogeny as the simulations described above. As these sequences were simulated on a tree of MERS-CoV, we refer to these datasets as being empirically simulated. We also reconstructed ancestral states for these sequences using ClonalFrameML, as described above, to arrive at a null expectation for the number of homoplasies we expect to observe under rate heterogeneity but without recombination.</p></div><div id="sec-8" class="subsection"><h3>Investigating the effects of temporal sampling and rate heterogeneity</h3><p id="p-21">All 10 sequence datasets simulated with fastsimcoal2 and 12 sequences empirically simulated in πBUSS were analyzed using LDhat (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">McVean et al., 2002</a>) to ascertain the effects of temporal sampling, and in the case of πBUSS-simulated sequences, the effects of rate heterogeneity in the presence of absence of position-specific constraint. Additionally, empirically simulated sequence datasets were run through GARD (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Kosakovsky Pond et al., 2006</a>), since the method considers both differences in tree topology and branch lengths when calculating the likelihoods of trees. Stark rate heterogeneity could thus easily be mis-interpreted as evidence for recombination by GARD.</p></div><div id="sec-9" class="subsection"><h3>Host-association alleles</h3><p id="p-22">In order to test for the presence of alleles associated with host shifts (presumably camel to human) we adapt the <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2015/12/23/020834/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2015/12/23/020834/embed/inline-graphic-1.gif"/></noscript></span></span> (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hedrick and Thomson, 1986</a>) statistic of LD to estimate the association between host (camel or human) and alleles at polymorphic loci. Briefly, we consider the host to act as a polymorphic site (encoded as H or C, for human and camel, respectively) and compare the association between the “allele” or host and alleles at polymorphic sites. A perfect association of 1.0 could mean, for example, that a biallelic site has one allele that is only found in camel viruses and the other allele only in human viruses.</p></div></div><div class="section" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>MERS-CoV genome shows evidence of alternative tree topologies</h3><p id="p-23">GARD identified a breakpoint at nucleotide position 23722 (corrected ΔAIC=103.6 between single versus two tree model), roughly in the middle of the coding sequence for the S (spike) protein. The 2 phylogenies recovered from this breakpoint were incongruent (see figure S1). Running the resulting fragment 1 (positions 1-23722) and fragment 2 (positions 23723-30126) through GARD again yielded a further breakpoint in fragment 1 at position 12257 (corrected ΔAIC=33.7), near the boundary between ORF1a and ORF1b genes. No more breakpoints could be identified by GARD in the resulting fragments 1.1 (positions 1-12257), 1.2 (positions 12258-23722) and 2 (positions 23723-30126).</p><p id="p-24">5 out of 6 empirical simulation alignments under a nucleotide model simulated without recombination, were identified by GARD as having breakpoints around position 20000, where the clock rate for the rest of the “genome” was increased to be 1.3, 2 or 3 times higher than the first 20kb. Similarly, 2/6 alignments empirically simulated under a codon model were identified as having a breakpoint. Corrected ΔAIC values decreased with decreasing rate heterogeneity, indicating loss of statistical power to detect differences between genomic regions. Analyses in BEAST where the MERS-CoV genome is partitioned into positions 1-23722 and positions 23723-30126 (corresponding to the first GARD-inferred breakpoint) with each partition having an independent molecular clock rate but the same tree or both independent molecular clock rates and independent trees, showed that rate heterogeneity as expressed by the ratio of second fragment rate to first fragment rate to be on the order of 1.513 (95% highest posterior density 1.275, 1.769) for unlinked clocks and 1.375 (95% HPDs 1.079, 1.707) for unlinked clocks and trees (see figure S2). As such, empirically simulated sequence data under 2-fold and 3-fold rate heterogeneity should be considered as caricature examples of a MERS-like organism, which we use to test the sensitivity of the methods we employ.</p></div><div id="sec-12" class="subsection"><h3>MERS-CoV genome exhibits linkage disequilibrium decay</h3><p id="p-25">Permutation tests as implemented in LDhat work under the assumption that loci are interchangeable only when there is free recombination or no recombination at all. The tests compare 4 statistics estimated from the actual data to 1000 permutations of the data where site numbers for each locus are reshuffled. Correlation coefficient between two measures of LD, <em>r</em><sup>2</sup> (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Hill and Robertson, 1968</a>) and <em>D'</em> (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lewontin, 1964</a>), are expected to show a negative correlation with increasing distance between loci if there is recombination. Permutation of recombining loci will produce a distribution skewed towards more positive values for these two LD statistics and the percentile of the actual observed value can then be used to assess significance.</p><p id="p-26">G4 is the sum of distances between pairs of loci with 4 observed haplotypes, which can only occur if there is repeat mutation or recombination at one of the loci. Under recombination the observed G4 statistic should take a statistically higher value in a distribution of G4 values derived from permuted data. Lkmax is the composite likelihood of pairs of loci under an estimated recombination rate and a given level of sequence diversity. Like the G4 statistic, this statistic is expected to fall in the upper tail of the distribution derived from permuted data in the presence of recombination.</p><p id="p-27">All 4 permutation tests show a consistent signal of recombination in the MERS-CoV genome (see <a id="xref-fig-2-1" class="xref-fig" href="#F2">figures 2</a> and S3). Data from fastsimcoal2 simulations, which did not have rate heterogeneity, produced values for these statistics which mostly fell inside the range of values generated by permuting the simulated data, as expected (<a id="xref-fig-2-2" class="xref-fig" href="#F2">figure 2</a>). On 1 occassion this is not the case – simulation 9 passed the Lkmax test and failed the other three. Empirically simulated data, on the other hand, tended to exhibit extreme values, that is the observed value fell below the 2.5th or above the 97.5th percentile of the permuted data, but in ways which were not consistent with recombination. For example, replicate 1 of simulation with 3-fold rate heterogeneity under a nucleotide substitution model exhibits extreme values for all 4 tests, but only one of these – corr(r<sup>2</sup>, d) is consistent with recombination.</p></div><div id="sec-13" class="subsection"><h3>Composite likelihood methods are susceptible to rate heterogeneity</h3><p id="p-28">The composite likelihood method, which finds the composite likelihood surface of recombination rate, inferred non-zero recombination rates for all simulated datasets (see figure S4), revealing some degree of susceptibility to both temporal sampling and rate heterogeneity. A window-based approach of this test shows a sharp increase in the recombination rate estimated within 300 nucleotide windows around nucleotide 21000, close to the breakpoint inferred by GARD (see figure S5). We recovered a qualitatively similar pattern when analyzing empirically simulated sequences. It is important to note, however, that none of the simulated data, even under extreme heterogeneity, reproduced the same scale of the estimated recombination rate. Whereas in MERS-CoV data the vast majority of 300 nucleotide windows after position 23000 have a recombination rate per base consistently higher than 0.005, only data simulated under extreme rate heterogeneity approach values as high as that.</p><p id="p-29">In addition to the apparently higher recombination rate in regions with higher rates we expect rate heterogeneity to produce a higher density of polymorphic sites in regions that are evolving faster. This is quite obvious in empirically simulated data with 3-fold rate heterogeneity – the region with higher rate also contains, on average, more polymorphic loci per window in the last third of the “genome” than the first 20kb (see figure S6). We only see hints of this in the actual MERS-CoV genome, with an apparent decline in polymorphism density from position 5000 to 15000 which resembles that of the simulated data with 1.3-fold rate heterogeneity.</p></div><div id="sec-14" class="subsection"><h3>Homoplasies in MERS-CoV genomes are ubiquitous</h3><p id="p-30">Homoplasy analyses suggests that the MERS-CoV genome is rife with apparent homoplasies. Both maximum likelihood and Bayesian approaches to ancestral sequence reconstruction converge on similar patterns of homoplasy density (<a id="xref-fig-3-1" class="xref-fig" href="#F3">figures 3</a> and S7). Both methods identify the region around the S (spike) protein as having a high density of synonymous homoplasies.</p><p id="p-31">Empirically simulated sequences showed that homoplasies are not that unlikely in the absence of recombination. All sequences empirically simulated in πBUSS under a nucleotide substitution model had 2-fold homoplasies ranging in frequency from 0.0222 to 0.0550 of all polymorphic sites, with sequences simulated under higher levels of rate heterogeneity having more homoplasies and higher homoplasy degrees (<a id="xref-fig-4-1" class="xref-fig" href="#F4">figure 4</a>). However, even under a caricature model of rate heterogeneity we did not reach the same degree of homoplasy as that observed in MERS-CoV, where homoplasic sites comprise as much as 0.1447 of all polymorphic sites and reach homoplasy degrees as high as 4. We were able to recover similar proportions of homoplasic sites from empirical simulations under a codon model (ranging from 0.0821 to 0.1518 of all polymorphic sites), but only under unrealistic values of rate heterogeneity. Even then, the distributions of homoplasy degrees for all simulated datasets are heavily skewed towards low homoplasy degrees, whereas the homoplasy degree distribution for MERS-CoV has a longer tail, indicating that homoplasies are more repeatable across the phylogeny.</p><p id="p-32">Additional tests for excessive homoplasies, PHI and 3Seq also identify MERS-CoV sequences as being recombinant, albeit both spuriously identify some of the simulated datasets as recombinant (see figure S10). Similarly to LDhat results, however, there were no cases where both methods falsely inferred the presence of recombination for the same dataset.</p></div><div id="sec-15" class="subsection"><h3>Model testing supports a model including rate heterogeneity, but not alternative tree topologies</h3><p id="p-33">A model including rate heterogeneity alone across breakpoints inferred by the GARD method (<em>i.e.</em> linked trees, unlinked relaxed clocks) performs best when applied to MERS-CoV data (<a id="xref-fig-5-1" class="xref-fig" href="#F5">figure 5</a> log marginal likelihoods: -48137.86 and -48138.91, using path and stepping stone sampling, respectively). The next best-performing model (log Bayes factor ≈18) is linked trees and relaxed clocks. Overall, unlinking molecular clock rates between the two genomic partitions appears sufficient to dramatically improve model fit. Additionally, relaxed molecular clocks are preferred over strict molecular clocks (log BF&gt;15, <a id="xref-fig-5-2" class="xref-fig" href="#F5">figure 5</a>).</p></div></div><div class="section" id="sec-16"><h2 class="">Discussion</h2><div id="sec-17" class="subsection"><h3>Recombination tests consistently point to recombination in MERS-CoV</h3><p id="p-34">The majority of methods we used (with the exception of marginal likelihood model testing) point to the combined effects of recombination and rate heterogeneity in the genome of MERS coronavirus. GARD (<a id="xref-fig-1-1" class="xref-fig" href="#F1">figure 1</a>) identified 2 breakpoints in the genome with high support. It is important to note that calculating the likelihood of a phylogenetic tree involves the estimation of multiple parameters, which at the very least must include a particular tree topology and a set of branch lengths. As such, we considered that the inference of a breakpoint in the MERS-CoV genome could be caused by systematic rate heterogeneity, which we address by evaluating rate heterogeneity across the first breakpoint in BEAST. We estimate an empirical rate heterogeneity ratio between MERS-CoV genome positions 23723-30126 and 1-23722 to be on the order between 1.3 and 1.5 (see figure S2). However, the support for this first breakpoint in MERS-CoV is comparable to support for empirically simulated sequences with 2-fold rate heterogeneity, and breakpoints under MERS-like levels of rate heterogeneity are difficult to detect. Overall, this suggests that evidence for differences in likelihoods between MERS fragments 1 and 2 are beyond what would be expected from rate heterogeneity alone.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of GARD results. Coloured boxes indicate fragments resulting from GARD-inferred breakpoints with corrected &#x394;-AIC values shown on the right. Dashed line indicates the actual position where the evolutionary model for simulated sequences under 3 levels of rate heterogeneity is changed. Arrows at the top indicate the positions and names of coding sequences within the MERS-CoV genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-565555544" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Summary of GARD results.&lt;/span&gt; Coloured boxes indicate fragments resulting from GARD-inferred breakpoints with corrected &#x394;-AIC values shown on the right. Dashed line indicates the actual position where the evolutionary model for simulated sequences under 3 levels of rate heterogeneity is changed. Arrows at the top indicate the positions and names of coding sequences within the MERS-CoV genome.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F1.medium.gif" width="440" height="221"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F1.medium.gif" width="440" height="221"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Summary of GARD results.</span><p id="p-35" class="first-child">Coloured boxes indicate fragments resulting from GARD-inferred breakpoints with corrected Δ-AIC values shown on the right. Dashed line indicates the actual position where the evolutionary model for simulated sequences under 3 levels of rate heterogeneity is changed. Arrows at the top indicate the positions and names of coding sequences within the MERS-CoV genome.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of non-parametric tests for recombination. The percentile of the observed value for 4 statistics of LD decay (y axis) in the distribution of permuted datasets is indicated by colour. Sequence datasets are shown on the x axis, starting with MERS-CoV sequences, followed by 10 fastsimcoal2-simulated datasets and 12 empirically simulated datasets with different degrees of rate heterogeneity. Expected values for ideal datasets are shown in the last two columns, an ideal positive corresponds to the presence of recombination. Values falling between the 2.5th and 97.5th percentile are shown in green, values falling below the 2.5th percentile are in blue, those that are above the 97.5th percentile in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-565555544" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Summary of non-parametric tests for recombination.&lt;/span&gt; The percentile of the observed value for 4 statistics of LD decay (y axis) in the distribution of permuted datasets is indicated by colour. Sequence datasets are shown on the x axis, starting with MERS-CoV sequences, followed by 10 fastsimcoal2-simulated datasets and 12 empirically simulated datasets with different degrees of rate heterogeneity. Expected values for ideal datasets are shown in the last two columns, an ideal positive corresponds to the presence of recombination. Values falling between the 2.5th and 97.5th percentile are shown in green, values falling below the 2.5th percentile are in blue, those that are above the 97.5th percentile in red.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F2.medium.gif" width="440" height="200"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F2.medium.gif" width="440" height="200"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Summary of non-parametric tests for recombination.</span><p id="p-36" class="first-child">The percentile of the observed value for 4 statistics of LD decay (y axis) in the distribution of permuted datasets is indicated by colour. Sequence datasets are shown on the x axis, starting with MERS-CoV sequences, followed by 10 fastsimcoal2-simulated datasets and 12 empirically simulated datasets with different degrees of rate heterogeneity. Expected values for ideal datasets are shown in the last two columns, an ideal positive corresponds to the presence of recombination. Values falling between the 2.5th and 97.5th percentile are shown in green, values falling below the 2.5th percentile are in blue, those that are above the 97.5th percentile in red.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Permutation tests show that statistics related to LD decay derived from MERS-CoV sequence data are outliers compared to permuted data (<a id="xref-fig-2-3" class="xref-fig" href="#F2">figures 2</a> and S3). Sequences simulated empirically with varying levels of rate heterogeneity, under nucleotide or codon models of substitution, also have a tendency to exhibit extreme values for these statistics. However, only MERS-CoV data has values for all 4 tests that are in the direction consistent with recombination. For simulated datasets, especially those simulated under extreme levels of rate heterogeneity, values deviating significantly from the permuted data were recovered, but often indicated internally contradictory scenarios.</p><p id="p-38">Homoplasy analyses regardless of inference method show that MERS-CoV sequences contain a large number of homoplasic sites with high homoplasy degrees (<a id="xref-fig-3-2" class="xref-fig" href="#F3">figures 3</a> and S7). Through sequence simulation we also confirmed that both the numbers of homoplasic sites and their homoplasy degrees in MERS-CoV genomes are excessive, even when compared to unrealistic scenarios (<em>e.g.</em> 3-fold rate heterogeneity, see <a id="xref-fig-4-2" class="xref-fig" href="#F4">figure 4</a>). Homoplasies become increasingly more prevalent when a more realistic codon model is used, due to differences in codon position constraint. Even then, MERS-CoV genomes possess mutations with much higher homoplasy degrees, surpassing simulated datasets with caricature levels of rate heterogeneity. This makes sense under a recombination scenario, as alleles persist in a diverse population and get recombined into novel backgrounds repeatedly, giving an appearance of highly repeatable mutations. Nevertheless, substitution patterns in real genomes are often highly complex and homoplasy-based methods have been shown to be susceptible to rate heterogeneity across sites, especially under higher levels of sequence divergence (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Posada and Crandall, 2001</a>). Although rate heterogeneity certainly exists in MERS-CoV data, the divergence levels are still quite low (<em>θ</em>/site = 0.0047), giving us some degree of certainty in our inference of homoplasies. It is also reassuring that both maximum likelihood and Bayesian sequence reconstruction converged on similar patterns of homoplasy and synapomorphy across the genome (<a id="xref-fig-3-3" class="xref-fig" href="#F3">figures 3</a> and S7). This is important, since homoplasies inferred using BEAST are integrated over all possible tree topologies, whereas homoplasies inferred by maximum likelihood were conditioned on a single tree. The convergence between these two methods suggests that the data contain enough phylogenetic signal to recover what could be called a “true” tree and that homoplasies, for the most part, can be correctly identified as such.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of apparent homoplasies. Position along the genome is shown on the x axis and homoplasy degree, the number of times a particular mutation has occured in excess in the tree as inferred by maximum likelihood, is shown on the y axis (left). Individual mutations are marked by vertical lines, synonymous ones in green and non-synonymous in red. The ratio of apparent homoplasy over synapomorphy kernel density estimates (bandwidth=0.1) is shown in blue for synonymous (dashed) and non-synonymous (solid) sites separately. Arrows at the top indicate the positions and names of coding sequences within the MERS-CoV genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-565555544" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Distribution of apparent homoplasies.&lt;/span&gt; Position along the genome is shown on the x axis and homoplasy degree, the number of times a particular mutation has occured in excess in the tree as inferred by maximum likelihood, is shown on the y axis (left). Individual mutations are marked by vertical lines, synonymous ones in green and non-synonymous in red. The ratio of apparent homoplasy over synapomorphy kernel density estimates (bandwidth=0.1) is shown in blue for synonymous (dashed) and non-synonymous (solid) sites separately. Arrows at the top indicate the positions and names of coding sequences within the MERS-CoV genome.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F3.medium.gif" width="440" height="261"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F3.medium.gif" width="440" height="261"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Distribution of apparent homoplasies.</span><p id="p-39" class="first-child">Position along the genome is shown on the x axis and homoplasy degree, the number of times a particular mutation has occured in excess in the tree as inferred by maximum likelihood, is shown on the y axis (left). Individual mutations are marked by vertical lines, synonymous ones in green and non-synonymous in red. The ratio of apparent homoplasy over synapomorphy kernel density estimates (bandwidth=0.1) is shown in blue for synonymous (dashed) and non-synonymous (solid) sites separately. Arrows at the top indicate the positions and names of coding sequences within the MERS-CoV genome.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Homoplasy prevalence in MERS-CoV and simulated datasets. Bars show the proportion of all polymorphic sites that are homoplasic, split by homoplasy degree as inferred by maximum likelihood, in MERS-CoV and datasets simulated with different degrees of rate heterogeneity in the presence or absence of site-specific constraint in the form of a codon model. Homoplasy degree indicates how many times a given mutation has occured in excess in the phylogenetic tree." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-565555544" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Homoplasy prevalence in MERS-CoV and simulated datasets.&lt;/span&gt; Bars show the proportion of all polymorphic sites that are homoplasic, split by homoplasy degree as inferred by maximum likelihood, in MERS-CoV and datasets simulated with different degrees of rate heterogeneity in the presence or absence of site-specific constraint in the form of a codon model. Homoplasy degree indicates how many times a given mutation has occured in excess in the phylogenetic tree.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F4.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F4.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Homoplasy prevalence in MERS-CoV and simulated datasets.</span><p id="p-40" class="first-child">Bars show the proportion of all polymorphic sites that are homoplasic, split by homoplasy degree as inferred by maximum likelihood, in MERS-CoV and datasets simulated with different degrees of rate heterogeneity in the presence or absence of site-specific constraint in the form of a codon model. Homoplasy degree indicates how many times a given mutation has occured in excess in the phylogenetic tree.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">One major concern surrounding the inference of homoplasies is host adaptation. There are a number of canonical mutations associated with host shifts, <em>e.g.</em> the glutamic acid to lysine amino acid substitution at position 627 in the PB2 protein of avian influenza A viruses confers the ability of the virus to replicate in mammals (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Subbarao et al., 1993</a>) and a small number of amino acid substitutions in Parvoviruses are associated with adaptations to different hosts (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chang et al., 1992</a>). If MERS-CoV is repeatedly emerging in humans convergent mutations would be expected to arise that might allow the virus to adapt to humans.</p><p id="p-42">However, we expect most host-adaptation mutations to be non-synonymous, whereas we detect both non-synonymous and synonymous homoplasies. This implies the action of recombination, rather than repeated selection for the same host-specific mutations. Furthermore, we do not detect any strong associations between host and particular alleles (figure S8), although we do not believe that there is a sufficient number of sequences from camels to have much confidence in this result.</p><p id="p-43">The overall phylogenetic and genomic patterns of homoplasies are consistent with fairly frequent recombination through time (<a id="xref-fig-6-1" class="xref-fig" href="#F6">figure 6</a>). Recent recombination should result in long homoplasy tracts shared across branches in the phylogenetic tree. At most we observe 2 stretches of adjacent homoplasies, one encompassing 3 homoplasies and another encompassing 2 homoplasies, that are shared between taxa, and likely to be caused by recent recombination. The vast majority of homoplasies that we observe, however, occur on their own. Recombination tracts, rather than single template switches are not uncommon in other coronaviruses (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Keck et al., 1988</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Kottier et al., 1995</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Herrewegh et al., 1998</a>). Thus in MERS-CoV we interpret extremely short homoplasy tracts as evidence of relatively frequent recombination. Alternatively, recombination tracts might be short and thus unable to transfer multiple informative sites across lineages.</p><p id="p-44">Unlike all other tests we performed model testing through marginal likelihoods indicates that models including rate heterogeneity explain MERS-CoV data partitioned across a well-supported breakpoint better than models including independent trees. At first this may seem paradoxical, but we believe this result is due to the combined effects of the way homoplasious sites are distributed across the genome and phylogenetic tree of MERS-CoV (<a id="xref-fig-6-2" class="xref-fig" href="#F6">figure 6</a>) and the number of parameters involved. A speckled pattern of homoplasious sites without phylogenetic signal could easily be overwhelmed by the signal coming from the sites that support what could be called “the one true tree”, <em>i.e.</em> the clonal frame, in the data. Secondly, each phylogenetic tree contains at least n-1 free parameters, so it is not surprising then that models attempting to recover 2 independent trees for both genomic fragments resulting from alternative tree topology analysis of MERS-CoV with highly correlated genealogies are penalized for the extra parameters introduced by a second tree. We would additionally like to point out that the fit of models including relaxed molecular clocks result in dramatic improvements to model fit compared to models with strict molecular clocks (log BF&gt;15, <a id="xref-fig-5-3" class="xref-fig" href="#F5">figure 5</a>). Although this could be interpreted as evidence for a considerable degree of lineage rate heterogeneity, the more parsimonious explanation is the ability of a relaxed molecular clock to accommodate homoplasies of recombinant origin, which do not necessarily accumulate at a relatively constant rate like genuine <em>de novo</em> mutations do.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of model comparisons. Difference in marginal likelihoods (Bayes factor) estimated by path-sampling between the worst model (linked strict molecular clock, unlinked trees) and all others. Asterisks indicate the best-performing model (unlinked relaxed clocks, linked trees, run 2) for MERS-CoV data. Analyses employing a relaxed molecular clock were run independently 3 times, those with a strict molecular clock 2 times. Marginal likelihoods estimated using stepping stone sampling gave identical results." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-565555544" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Summary of model comparisons.&lt;/span&gt; Difference in marginal likelihoods (Bayes factor) estimated by path-sampling between the worst model (linked strict molecular clock, unlinked trees) and all others. Asterisks indicate the best-performing model (unlinked relaxed clocks, linked trees, run 2) for MERS-CoV data. Analyses employing a relaxed molecular clock were run independently 3 times, those with a strict molecular clock 2 times. Marginal likelihoods estimated using stepping stone sampling gave identical results.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F5.medium.gif" width="440" height="377"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F5.medium.gif" width="440" height="377"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">Summary of model comparisons.</span><p id="p-45" class="first-child">Difference in marginal likelihoods (Bayes factor) estimated by path-sampling between the worst model (linked strict molecular clock, unlinked trees) and all others. Asterisks indicate the best-performing model (unlinked relaxed clocks, linked trees, run 2) for MERS-CoV data. Analyses employing a relaxed molecular clock were run independently 3 times, those with a strict molecular clock 2 times. Marginal likelihoods estimated using stepping stone sampling gave identical results.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mutations mapped onto a ML phylogeny. A maximum likelihood phylogeny of 85 MERS-CoV sequences with maximum likelihood-mapped mutations. Synapomorphies are shown as coloured ticks (coloured by coding sequence in which they occur) on branches where they occur. Homoplasies are shown as circles connected with coloured lines, colour corresponds with the coding sequence in which the mutation has occured. Mutations are positioned on the branches in proportion to where the mutation occurs in the genome, e.g. mutations shown towards the end of a branch correspond to mutations near the 3&#x2019; terminus of the genome. Arrows at the top indicate the order, relative length and names of coding sequences within the MERS-CoV genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-565555544" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Mutations mapped onto a ML phylogeny.&lt;/span&gt; A maximum likelihood phylogeny of 85 MERS-CoV sequences with maximum likelihood-mapped mutations. Synapomorphies are shown as coloured ticks (coloured by coding sequence in which they occur) on branches where they occur. Homoplasies are shown as circles connected with coloured lines, colour corresponds with the coding sequence in which the mutation has occured. Mutations are positioned on the branches in proportion to where the mutation occurs in the genome, &lt;em&gt;e.g.&lt;/em&gt; mutations shown towards the end of a branch correspond to mutations near the 3&#x2019; terminus of the genome. Arrows at the top indicate the order, relative length and names of coding sequences within the MERS-CoV genome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F6.medium.gif" width="415" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F6.medium.gif" width="415" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/12/23/020834/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <span class="caption-title">Mutations mapped onto a ML phylogeny.</span><p id="p-46" class="first-child">A maximum likelihood phylogeny of 85 MERS-CoV sequences with maximum likelihood-mapped mutations. Synapomorphies are shown as coloured ticks (coloured by coding sequence in which they occur) on branches where they occur. Homoplasies are shown as circles connected with coloured lines, colour corresponds with the coding sequence in which the mutation has occured. Mutations are positioned on the branches in proportion to where the mutation occurs in the genome, <em>e.g.</em> mutations shown towards the end of a branch correspond to mutations near the 3’ terminus of the genome. Arrows at the top indicate the order, relative length and names of coding sequences within the MERS-CoV genome.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Implications for future analyses</h3><p id="p-47">Recombination aside, MERS-CoV genomes exhibit a significant degree of rate heterogeneity amongst sites. Marginal likelihood analyses indicate that estimating independent molecular clocks after partitioning the MERS-CoV genome into 2 fragments alone substantially increases model fit over a completely linked (trees and clocks) model (log Bayes factor ≈18). This highlights the advantage of employing relaxed molecular clocks, as in our case the method is clearly capable of accomodating recombination in an otherwise entirely clonal analysis framework. In addition, previous studies of SARS-like coronaviruses in bats have identified recombination breakpoints in small numbers of isolates falling close to the “transition zone” around site 22000 (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Hon et al., 2008</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lau et al., 2010</a>) which in our analysis of MERS-CoV is where GARD, LDhat and BEAST identify changes in the underlying model of evolution (<a id="xref-fig-1-2" class="xref-fig" href="#F1">figures 1</a>, S5 and 3). Overall, a more detailed investigation will need to be done to determine if empirical patterns of rate variation in MERS-CoV have the potential to generate apparent recombination signals.</p><p id="p-48">We also expect that as more sequences of MERS-CoV become available more homoplasies will be detected, some contributing to the homoplasy degree of the homoplasies already reported here, some previously unknown and some turning mutations currently thought of as synapomorphies into homoplasies. Although new sequences are likely to come from human cases, we think that sequencing MERS-CoV circulating in dromedary camels is of extreme importance from both surveillance and epidemiological points of view.</p></div><div id="sec-19" class="subsection"><h3>Implications about the virus population structure and infection dynamics</h3><p id="p-49">Our results point towards frequent recombination in MERS-CoV in the recent history of the MERS-CoV outbreak. For this to occur different lineages of the virus must encounter each other often and implies frequent co-infection with MERS-CoV. To date it is difficult to ascertain whether the human infections with MERS-CoV are a result of substantial asymptomatic transmission amongst humans, or repeated zoonosis of the virus from camels to humans or a combination thereof. Given the severity of MERS we find it unlikely that humans could be sufficiently frequently co-infected with two or more different lineages of the virus. Previous serological studies have failed to find evidence of prevalent past MERS-CoV infections of humans (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gierer et al., 2013</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aburizaiza et al., 2013</a>), although a recent nation-wide study in Saudi Arabia has detected non-negligible numbers of individuals with antibodies against MERS-CoV, especially amongst shepherds and slaughterhouse workers (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Müller et al., 2015</a>). We thus propose that MERS-CoV mostly infects, and recombines, in camels. A study by <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Adney et al. (2014)</a> has shown that camels only suffer mild symptoms from MERS-CoV infection and numerous other studies indicate an extremely high prevalence of antibodies specific against MERS-CoV in camels (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Müller et al., 2014</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Corman et al., 2014b</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Chu et al., 2014</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Reusken et al., 2013</a>, <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">2014</a>). At the same time, however, sequencing has not indicated the presence of multiple infection in camels, or any other animal. We believe that individual MERS-CoV co-infections are rare, but given the size of the epidemic in camels, as inferred from serology, the total number of co-infections is high. In addition, MERS-CoV infection is transient in camels <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Adney et al. (2014)</a> and thus sequencing efforts, which have been insufficient and very limited in camels, are highly unlikely to capture a co-infection.</p><p id="p-50">Another point worth considering is that alleles that have arisen through mutation in MERS-CoV can be recombined, increasing the genetic variation of the virus (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Muller, 1932</a>). Whether this is of epidemiological importance for humans depends entirely on what alleles are circulating in the reservoir, although there is no evidence that MERS-CoV is particularly likely to become as transmissible as common human pathogens or even SARS-CoV.</p></div><div id="sec-20" class="subsection"><h3>Data availability</h3><p id="p-51">Python scripts used to process trees and sequences are available at:</p><p id="p-52"><a href="https://github.com/evogytis/MERS_recombination/tree/master/scripts">https://github.com/evogytis/MERS_recombination/tree/master/scripts</a>.</p><p id="p-53">Input and output files for programs used are publicly available at:</p><p id="p-54"><a href="https://github.com/evogytis/MERS_recombination">https://github.com/evogytis/MERS_recombination</a>.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-55">GD was supported by a Natural Environment Research Council studentship D76739X. The research leading to these results has received funding from the European Research Council under the European Community’s Seventh Framework Programme (FP7/2007-2013) under Grant Agreement no. 278433-PREDEMICS and ERC Grant agreement no. 260864.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aburizaiza</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mattes</span>  <span class="cit-name-given-names">FM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Azhar</span>  <span class="cit-name-given-names">EI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hassan</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Memish</span>  <span class="cit-name-given-names">ZA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muth</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lattwein</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drosten</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Investigation of anti-MERS-coronavirus antibodies in blood donors and abbatoir workers in jeddah and makkah, kingdom of saudi arabia, fall 2012</span>. <abbr class="cit-jnl-abbrev">Journal of Infectious Diseases</abbr>. p. <span class="cit-fpage">jit589</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.2" data-doi="10.3201/eid2012.141280"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Adney</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Doremalen</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">VR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bushmaker</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Wit</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munster</span>  <span class="cit-name-given-names">VJ.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Replication and shedding of MERS-CoV in upper respiratory tract of inoculated dromedary camels</span>. <abbr class="cit-jnl-abbrev">Emerging Infectious Diseases</abbr>. <span class="cit-vol">20</span>: <span class="cit-fpage">1999</span>–<span class="cit-lpage">2005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2BInfectious%2BDiseases%26rft.volume%253D20%26rft.spage%253D1999%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2012.141280%26rft_id%253Dinfo%253Apmid%252F25418529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2012.141280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25418529&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anisimova</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Effect of recombination on the accuracy of the likelihood method for detecting positive selection at amino acid sites</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>. <span class="cit-vol">164</span>: <span class="cit-fpage">1229</span>–<span class="cit-lpage">1236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAnisimova%26rft.auinit1%253DM.%26rft.volume%253D164%26rft.issue%253D3%26rft.spage%253D1229%26rft.epage%253D1236%26rft.atitle%253DEffect%2Bof%2BRecombination%2Bon%2Bthe%2BAccuracy%2Bof%2Bthe%2BLikelihood%2BMethod%2Bfor%2BDetecting%2BPositive%2BSelection%2Bat%2BAmino%2BAcid%2BSites%26rft_id%253Dinfo%253Apmid%252F12871927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC8zLzEyMjkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNS8xMi8yMy8wMjA4MzQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.4" data-doi="10.1016/S1473-3099(13)70204-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Assiri</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al-Tawfiq</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al-Rabeeah</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-etal">et al.</span> (13 co-authors). <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Epidemiological, demographic, and clinical characteristics of 47 cases of middle east respiratory syndrome coronavirus disease from saudi arabia: a descriptive study</span>. <abbr class="cit-jnl-abbrev">The Lancet Infectious Diseases</abbr>. <span class="cit-vol">13</span>: <span class="cit-fpage">752</span>–<span class="cit-lpage">761</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%2BInfectious%2BDiseases%26rft.volume%253D13%26rft.spage%253D752%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252813%252970204-4%26rft_id%253Dinfo%253Apmid%252F23891402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1473-3099(13)70204-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23891402&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323465900020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.5" data-doi="10.1093/molbev/mss084"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baele</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lemey</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bedford</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alekseyenko</span>  <span class="cit-name-given-names">AV.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Improving the accuracy of demographic and molecular clock model comparison while accommodating phylogenetic uncertainty</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>. <span class="cit-vol">29</span>: <span class="cit-fpage">2157</span>–<span class="cit-lpage">2167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss084%26rft_id%253Dinfo%253Apmid%252F22403239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22403239&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308851600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.6" data-doi="10.1016/0042-6822(87)90414-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shieh</span>  <span class="cit-name-given-names">CK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stohlman</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lai MM</span> <span class="cit-name-given-names">C.</span></span> <span class="cit-pub-date">1987</span>. <span class="cit-article-title">Analysis of intracellular small RNAs of mouse hepatitis virus: evidence for discontinuous transcription</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-vol">156</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DBaric%26rft.auinit1%253DR.%2BS.%26rft.volume%253D156%26rft.issue%253D2%26rft.spage%253D342%26rft.epage%253D354%26rft.atitle%253DAnalysis%2Bof%2Bintracellular%2Bsmall%2BRNAs%2Bof%2Bmouse%2Bhepatitis%2Bvirus%253A%2Bevidence%2Bfor%2Bdiscontinuous%2Btranscription.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252887%252990414-4%26rft_id%253Dinfo%253Apmid%252F3027983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(87)90414-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3027983&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.7" data-doi="10.1186/1471-2105-15-133"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bielejec</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lemey</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carvalho</span>  <span class="cit-name-given-names">LM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baele</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">πBUSS: a parallel BEAST/BEAGLE utility for sequence simulation under complex evolutionary scenarios</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>. <span class="cit-vol">15</span>: <span class="cit-fpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D133%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-133%26rft_id%253Dinfo%253Apmid%252F24885610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24885610&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.8" data-doi="10.1371/journal.pone.0010434"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boni</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Jong</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Doorn</span>  <span class="cit-name-given-names">HR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Guidelines for Identifying Homologous Recombination Events in Influenza A Virus</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>. <span class="cit-vol">5</span>:<span class="cit-fpage">e10434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBoni%26rft.auinit1%253DM.%2BF.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253De10434%26rft.epage%253De10434%26rft.atitle%253DGuidelines%2Bfor%2Bidentifying%2Bhomologous%2Brecombination%2Bevents%2Bin%2Binfluenza%2BA%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0010434%26rft_id%253Dinfo%253Apmid%252F20454662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0010434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20454662&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.9" data-doi="10.1534/genetics.106.068874"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boni</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">MW.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">An Exact Nonparametric Method for Inferring Mosaic Structure in Sequence Triplets</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>. <span class="cit-vol">176</span>: <span class="cit-fpage">1035</span>–<span class="cit-lpage">1047</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.068874%26rft_id%253Dinfo%253Apmid%252F17409078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni8yLzEwMzUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNS8xMi8yMy8wMjA4MzQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.10" data-doi="10.1128/AAC.37.10.2231"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boucher</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cammack</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schipper</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schuurman</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rouse</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wainberg</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">JM.</span></span> <span class="cit-pub-date">1993</span>. <span class="cit-article-title">High-level resistance to (-) enantiomeric 2’-deoxy-3’-thiacytidine in vitro is due to one amino acid substitution in the catalytic site of human immunodeficiency virus type 1 reverse transcriptase</span>. <abbr class="cit-jnl-abbrev">Antimicrobial Agents and Chemotherapy</abbr>. <span class="cit-vol">37</span>: <span class="cit-fpage">2231</span>–<span class="cit-lpage">2234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrobial%2BAgents%2Band%2BChemotherapy%26rft.stitle%253DAntimicrob.%2BAgents%2BChemother.%26rft.aulast%253DBoucher%26rft.auinit1%253DC%2BA%26rft.volume%253D37%26rft.issue%253D10%26rft.spage%253D2231%26rft.epage%253D2234%26rft.atitle%253DHigh-level%2Bresistance%2Bto%2B%2528-%2529%2Benantiomeric%2B2%2527-deoxy-3%2527-thiacytidine%2Bin%2Bvitro%2Bis%2Bdue%2Bto%2Bone%2Bamino%2Bacid%2Bsubstitution%2Bin%2Bthe%2Bcatalytic%2Bsite%2Bof%2Bhuman%2Bimmunodeficiency%2Bvirus%2Btype%2B1%2Breverse%2Btranscriptase.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAAC.37.10.2231%26rft_id%253Dinfo%253Apmid%252F7504909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjEwOiIzNy8xMC8yMjMxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMTIvMjMvMDIwODM0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.11" data-doi="10.1534/genetics.105.048975"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bruen</span>  <span class="cit-name-given-names">TC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Philippe</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bryant</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">A Simple and Robust Statistical Test for Detecting the Presence of Recombination</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>. <span class="cit-vol">172</span>: <span class="cit-fpage">2665</span>–<span class="cit-lpage">2681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.048975%26rft_id%253Dinfo%253Apmid%252F16489234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi80LzI2NjUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNS8xMi8yMy8wMjA4MzQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.12" data-doi="10.1016/S1473-3099(13)70304-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cauchemez</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Kerkhove</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Riley</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Enouf</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">NM.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Middle east respiratory syndrome coronavirus: quantification of the extent of the epidemic, surveillance biases, and transmissibility</span>. <abbr class="cit-jnl-abbrev">The Lancet Infectious Diseases</abbr>. <span class="cit-vol">14</span>: <span class="cit-fpage">50</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%2BInfectious%2BDiseases%26rft.volume%253D14%26rft.spage%253D50%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252813%252970304-9%26rft_id%253Dinfo%253Apmid%252F24239323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1473-3099(13)70304-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24239323&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329079100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sgro</span>  <span class="cit-name-given-names">JY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parrish</span>  <span class="cit-name-given-names">CR.</span></span> <span class="cit-pub-date">1992</span>. <span class="cit-article-title">Multiple amino acids in the capsid structure of canine parvovirus coordinately determine the canine host range and specific antigenic and hemagglutination properties</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>. <span class="cit-vol">66</span>: <span class="cit-fpage">6858</span>–<span class="cit-lpage">6867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DChang%26rft.auinit1%253DS%2BF%26rft.volume%253D66%26rft.issue%253D12%26rft.spage%253D6858%26rft.epage%253D6867%26rft.atitle%253DMultiple%2Bamino%2Bacids%2Bin%2Bthe%2Bcapsid%2Bstructure%2Bof%2Bcanine%2Bparvovirus%2Bcoordinately%2Bdetermine%2Bthe%2Bcanine%2Bhost%2Brange%2Band%2Bspecific%2Bantigenic%2Band%2Bhemagglutination%2Bproperties.%26rft_id%253Dinfo%253Apmid%252F1331498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI2Ni8xMi82ODU4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMTIvMjMvMDIwODM0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.14" data-doi="10.1099/vir.0.19277-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chare</span>  <span class="cit-name-given-names">ER</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gould</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Phylogenetic analysis reveals a low rate of homologous recombination in negative-sense RNA viruses</span>. <abbr class="cit-jnl-abbrev">Journal of General Virology</abbr>. <span class="cit-vol">84</span>: <span class="cit-fpage">2691</span>–<span class="cit-lpage">2703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DChare%26rft.auinit1%253DE.%2BR.%26rft.volume%253D84%26rft.issue%253D10%26rft.spage%253D2691%26rft.epage%253D2703%26rft.atitle%253DPhylogenetic%2Banalysis%2Breveals%2Ba%2Blow%2Brate%2Bof%2Bhomologous%2Brecombination%2Bin%2Bnegative-sense%2BRNA%2Bviruses%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.19277-0%26rft_id%253Dinfo%253Apmid%252F13679603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.19277-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=13679603&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000189155700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.15" data-doi="10.3201/eid2006.140299"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">DK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poon</span>  <span class="cit-name-given-names">LL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gomaa</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-etal">et al.</span> (13 co-authors). <span class="cit-pub-date">2014</span> <span class="cit-article-title">MERS coronaviruses in dromedary camels, egypt</span>. <abbr class="cit-jnl-abbrev">Emerging Infectious Diseases</abbr>. <span class="cit-vol">20</span>: <span class="cit-fpage">1049</span>–<span class="cit-lpage">1053</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2BInfectious%2BDiseases%26rft.volume%253D20%26rft.spage%253D1049%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2006.140299%26rft_id%253Dinfo%253Apmid%252F24856660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2006.140299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24856660&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.16" data-doi="10.1128/JVI.01498-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Corman</span>  <span class="cit-name-given-names">VM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ithete</span>  <span class="cit-name-given-names">NL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">LR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schoeman</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Preiser</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drosten</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drexler</span>  <span class="cit-name-given-names">JF.</span></span> <span class="cit-pub-date">2014a</span>. <span class="cit-article-title">Rooting the phylogenetic tree of middle east respiratory syndrome coronavirus by characterization of a conspecific virus from an african bat</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>. <span class="cit-vol">88</span>: <span class="cit-fpage">11297</span>–<span class="cit-lpage">11303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01498-14%26rft_id%253Dinfo%253Apmid%252F25031349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4OC8xOS8xMTI5NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE1LzEyLzIzLzAyMDgzNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Corman</span>  <span class="cit-name-given-names">VM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jores</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-etal">et al.</span> (13 co-authors). <span class="cit-pub-date">2014b</span>. <span class="cit-article-title">Antibodies against MERS coronavirus in dromedary camels, kenya, 1992–2013</span>. <abbr class="cit-jnl-abbrev">Emerging Infectious Diseases</abbr>. <span class="cit-fpage">20</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cotten</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kellam</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> (22 co-authors).<span class="cit-pub-date">2013</span>. <span class="cit-article-title">Transmission and evolution of the middle east respiratory syndrome coronavirus in saudi arabia: a descriptive genomic study</span>. <abbr class="cit-jnl-abbrev">The Lancet</abbr>. <span class="cit-vol">382</span>: <span class="cit-fpage">1993</span>–<span class="cit-lpage">2002</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%26rft.volume%253D382%26rft.spage%253D1993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.19" data-doi="10.1128/mBio.01062-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cotten</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zumla</span>  <span class="cit-name-given-names">AI</span></span>, <span class="cit-etal">et al.</span> (20 co-authors).<span class="cit-pub-date">2014</span>. <span class="cit-article-title">Spread, circulation, and evolution of the middle east respiratory syndrome coronavirus</span>. <abbr class="cit-jnl-abbrev">mBio</abbr>. <span class="cit-vol">5</span>:<span class="cit-fpage">e01062</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmBio%26rft.volume%253D5%26rft.spage%253De01062%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.01062-13%26rft_id%253Dinfo%253Apmid%252F24549846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.01062-13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24549846&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.20" data-doi="10.1534/genetics.106.063305"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Didelot</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Inference of bacterial microevolution using multilocus sequence data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>. <span class="cit-vol">175</span>: <span class="cit-fpage">1251</span>–<span class="cit-lpage">1266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.063305%26rft_id%253Dinfo%253Apmid%252F17151252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS8zLzEyNTEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNS8xMi8yMy8wMjA4MzQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.21" data-doi="10.1371/journal.pone.0017212"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duan</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deliberto</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">DL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Phylogenetic diversity and genotypical complexity of h9n2 influenza a viruses revealed by genomic sequence analysis</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>. <span class="cit-vol">6</span>:<span class="cit-fpage">e17212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDong%26rft.auinit1%253DG.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De17212%26rft.epage%253De17212%26rft.atitle%253DPhylogenetic%2Bdiversity%2Band%2Bgenotypical%2Bcomplexity%2Bof%2BH9N2%2Binfluenza%2BA%2Bviruses%2Brevealed%2Bby%2Bgenomic%2Bsequence%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0017212%26rft_id%253Dinfo%253Apmid%252F21386964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0017212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21386964&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drosten</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muth</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corman</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-etal">et al.</span> (15 co-authors).<span class="cit-pub-date">2014</span>. <span class="cit-article-title">An observational, laboratory-based study of outbreaks of MERS-coronavirus in jeddah and riyadh, kingdom of saudi arabia, 2014</span>. <abbr class="cit-jnl-abbrev">Clinical Infectious Diseases</abbr>. p. <span class="cit-fpage">ciu812</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.23" data-doi="10.1016/S1473-3099(13)70154-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drosten</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seilmaier</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corman</span>  <span class="cit-name-given-names">VM</span></span>, <span class="cit-etal">et al.</span> (22 co-authors).<span class="cit-pub-date">2013</span>. <span class="cit-article-title">Clinical features and virological analysis of a case of middle east respiratory syndrome coronavirus infection</span>. <abbr class="cit-jnl-abbrev">The Lancet Infectious Diseases</abbr>. <span class="cit-vol">13</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">751</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%2BInfectious%2BDiseases%26rft.volume%253D13%26rft.spage%253D745%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252813%252970154-3%26rft_id%253Dinfo%253Apmid%252F23782859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1473-3099(13)70154-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23782859&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323465900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.24" data-doi="10.1371/journal.pbio.0040088"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho SY</span> <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Relaxed phylogenetics and dating with confidence</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr>. <span class="cit-vol">4</span>:<span class="cit-fpage">e88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDrummond%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253De88%26rft.epage%253De88%26rft.atitle%253DRelaxed%2Bphylogenetics%2Band%2Bdating%2Bwith%2Bconfidence.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040088%26rft_id%253Dinfo%253Apmid%252F16683862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16683862&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Bayesian phylogenetics with BEAUti and the BEAST 1.7</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>. <span class="cit-vol">29</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.26" data-doi="10.1371/journal.pgen.1003905"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dupanloup</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huerta-Sánchez</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sousa</span>  <span class="cit-name-given-names">VC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Robust demographic inference from genomic and SNP data</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr>. <span class="cit-vol">9</span>:<span class="cit-fpage">e1003905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D9%26rft.spage%253De1003905%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003905%26rft_id%253Dinfo%253Apmid%252F24204310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003905&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24204310&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.27" data-doi="10.1002/cjs.5550360302"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">MAR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Bayesian analysis of elapsed times in continuous-time markov chains</span>. <abbr class="cit-jnl-abbrev">Canadian Journal of Statistics</abbr>. <span class="cit-vol">36</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCanadian%2BJournal%2Bof%2BStatistics%26rft.volume%253D36%26rft.spage%253D355%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fcjs.5550360302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/cjs.5550360302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.28" data-doi="10.3201/eid1912.130701"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gierer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hofmann-Winkler</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Albuali</span>  <span class="cit-name-given-names">WH</span></span>, <span class="cit-etal">et al.</span> (11 co-authors). <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Lack of MERS coronavirus neutralizing antibodies in humans, eastern province, saudi arabia</span>. <abbr class="cit-jnl-abbrev">Emerging Infectious Diseases</abbr>. <span class="cit-vol">19</span>: <span class="cit-fpage">2034</span>–<span class="cit-lpage">2036</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2BInfectious%2BDiseases%26rft.volume%253D19%26rft.spage%253D2034%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1912.130701%26rft_id%253Dinfo%253Apmid%252F24274664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1912.130701&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24274664&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.29" data-doi="10.1093/molbev/mss265"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lemey</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faria</span>  <span class="cit-name-given-names">NR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Improving bayesian population dynamics inference: A coalescent-based model for multiple loci</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>. <span class="cit-vol">30</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss265%26rft_id%253Dinfo%253Apmid%252F23180580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss265&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23180580&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314891800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.30" data-doi="10.1126/science.1259657"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gire</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goba</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">KG</span></span>, <span class="cit-etal">et al.</span> (58 co-authors).<span class="cit-pub-date">2014</span>. <span class="cit-article-title">Genomic surveillance elucidates ebola virus origin and transmission during the 2014 outbreak</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-vol">345</span>: <span class="cit-fpage">1369</span>–<span class="cit-lpage">1372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1259657%26rft_id%253Dinfo%253Apmid%252F25214632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDUvNjIwMi8xMzY5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMTIvMjMvMDIwODM0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.31" data-doi="10.1093/oxfordjournals.molbev.a040153"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goldman</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span> <span class="cit-pub-date">1994</span>. <span class="cit-article-title">A codon-based model of nucleotide substitution for protein-coding DNA sequences</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>. <span class="cit-vol">11</span>: <span class="cit-fpage">725</span>–<span class="cit-lpage">736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DGoldman%26rft.auinit1%253DN.%26rft.volume%253D11%26rft.issue%253D5%26rft.spage%253D725%26rft.epage%253D736%26rft.atitle%253DA%2Bcodon-based%2Bmodel%2Bof%2Bnucleotide%2Bsubstitution%2Bfor%2Bprotein-coding%2BDNA%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a040153%26rft_id%253Dinfo%253Apmid%252F7968486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a040153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7968486&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PG15600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.32" data-doi="10.1086/318537"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gubareva</span>  <span class="cit-name-given-names">LV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaiser</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matrosovich</span>  <span class="cit-name-given-names">MN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soo-Hoo</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayden</span>  <span class="cit-name-given-names">FG.</span></span> <span class="cit-pub-date">2001</span>. <span class="cit-article-title">Selection of influenza virus mutants in experimentally infected volunteers treated with oseltamivir</span>. <abbr class="cit-jnl-abbrev">Journal of Infectious Diseases</abbr>. <span class="cit-vol">183</span>: <span class="cit-fpage">523</span>–<span class="cit-lpage">531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BInfectious%2BDiseases%26rft_id%253Dinfo%253Adoi%252F10.1086%252F318537%26rft_id%253Dinfo%253Apmid%252F11170976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/318537&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11170976&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166586000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.33" data-doi="10.1080/10635150390235520"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood</span>. <abbr class="cit-jnl-abbrev">Systematic Biology</abbr>. <span class="cit-vol">52</span>: <span class="cit-fpage">696</span>–<span class="cit-lpage">704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150390235520%26rft_id%253Dinfo%253Apmid%252F14530136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150390235520&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14530136&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185732500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.34" data-doi="10.1007/BF02101694"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hasegawa</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kishino</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">Ta.</span></span> <span class="cit-pub-date">1985</span>. <span class="cit-article-title">Dating of the human-ape splitting by a molecular clock of mitochondrial DNA</span>. <abbr class="cit-jnl-abbrev">Journal of Molecular Evolution</abbr>. <span class="cit-vol">22</span>: <span class="cit-fpage">160</span>–<span class="cit-lpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DHasegawa%26rft.auinit1%253DM.%26rft.volume%253D22%26rft.issue%253D2%26rft.spage%253D160%26rft.epage%253D174%26rft.atitle%253DDating%2Bof%2Bthe%2Bhuman-ape%2Bsplitting%2Bby%2Ba%2Bmolecular%2Bclock%2Bof%2Bmitochondrial%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02101694%26rft_id%253Dinfo%253Apmid%252F3934395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02101694&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3934395&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985AWB1600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hedrick</span>  <span class="cit-name-given-names">PW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-pub-date">1986</span>. <span class="cit-article-title">A two-locus neutrality test: Applications to humans, E. coli and lodgepole pine</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>. <span class="cit-vol">112</span>:<span class="cit-fpage">135</span>–<span class="cit-lpage">156</span>. PMID: 3510942.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHedrick%26rft.auinit1%253DP.%2BW.%26rft.volume%253D112%26rft.issue%253D1%26rft.spage%253D135%26rft.epage%253D156%26rft.atitle%253DA%2BTWO-LOCUS%2BNEUTRALITY%2BTEST%253A%2BAPPLICATIONS%2BTO%2BHUMANS%252C%2BE.%2BCOLI%2BAND%2BLODGEPOLE%2BPINE%26rft_id%253Dinfo%253Apmid%252F3510942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTEyLzEvMTM1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMTIvMjMvMDIwODM0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Herrewegh</span>  <span class="cit-name-given-names">AAPM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smeenk</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horzinek</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rottier PJ</span> <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Groot</span>  <span class="cit-name-given-names">RJd.</span></span> <span class="cit-pub-date">1998</span>. <span class="cit-article-title">Feline coronavirus type II strains 79–1683 and 79–1146 originate from a double recombination between feline coronavirus type i and canine coronavirus</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>. <span class="cit-vol">72</span>: <span class="cit-fpage">4508</span>–<span class="cit-lpage">4514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DHerrewegh%26rft.auinit1%253DA.%2BA.%25A0P.%25A0M.%26rft.volume%253D72%26rft.issue%253D5%26rft.spage%253D4508%26rft.epage%253D4514%26rft.atitle%253DFeline%2BCoronavirus%2BType%2BII%2BStrains%2B79-1683%25A0and%2B79-1146%25A0Originate%2Bfrom%2Ba%2BDouble%2BRecombination%2Bbetween%2BFeline%2BCoronavirus%2BType%2BI%2Band%2BCanine%2BCoronavirus%26rft_id%253Dinfo%253Apmid%252F9557750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjcyLzUvNDUwOCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE1LzEyLzIzLzAyMDgzNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">WG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">1968</span>. <span class="cit-article-title">Linkage disequilibrium in finite populations</span>. <abbr class="cit-jnl-abbrev">TAG. Theoretical and applied genetics. Theoretische und angewandte Genetik</abbr>. <span class="cit-vol">38</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.volume%253D38%26rft.spage%253D226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.38" data-doi="10.1093/oxfordjournals.molbev.a026159"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Urwin</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maiden</span>  <span class="cit-name-given-names">MC.</span></span> <span class="cit-pub-date">1999</span>. <span class="cit-article-title">The influence of recombination on the population structure and evolution of the human pathogen neisseria meningitidis</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>. <span class="cit-vol">16</span>: <span class="cit-fpage">741</span>–<span class="cit-lpage">749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DHolmes%26rft.auinit1%253DE.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D741%26rft.epage%253D749%26rft.atitle%253DThe%2Binfluence%2Bof%2Brecombination%2Bon%2Bthe%2Bpopulation%2Bstructure%2Band%2Bevolution%2Bof%2Bthe%2Bhuman%2Bpathogen%2BNeisseria%2Bmeningitidis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026159%26rft_id%253Dinfo%253Apmid%252F10368953%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a026159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10368953&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080738400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.39" data-doi="10.1128/JVI.01926-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hon</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">TY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">ZL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yip</span>  <span class="cit-name-given-names">CW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">PY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">FCC.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Evidence of the Recombinant Origin of a Bat Severe Acute Respiratory Syndrome (SARS)-Like Coronavirus and Its Implications on the Direct Ancestor of SARS Coronavirus</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>. <span class="cit-vol">82</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01926-07%26rft_id%253Dinfo%253Apmid%252F18057240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgyLzQvMTgxOSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE1LzEyLzIzLzAyMDgzNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keck</span>  <span class="cit-name-given-names">JG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matsushima</span>  <span class="cit-name-given-names">GK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makino</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fleming</span>  <span class="cit-name-given-names">JO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vannier</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stohlman</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">MM.</span></span> <span class="cit-pub-date">1988</span>. <span class="cit-article-title">In vivo RNA-RNA recombination of coronavirus in mouse brain</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>. <span class="cit-vol">62</span>: <span class="cit-fpage">1810</span>–<span class="cit-lpage">1813</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DKeck%26rft.auinit1%253DJ%2BG%26rft.volume%253D62%26rft.issue%253D5%26rft.spage%253D1810%26rft.epage%253D1813%26rft.atitle%253DIn%2Bvivo%2BRNA-RNA%2Brecombination%2Bof%2Bcoronavirus%2Bin%2Bmouse%2Bbrain.%26rft_id%253Dinfo%253Apmid%252F2833625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjYyLzUvMTgxMCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE1LzEyLzIzLzAyMDgzNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.41" data-doi="10.1016/0092-8674(86)90600-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kirkegaard</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baltimore</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">1986</span>. <span class="cit-article-title">The mechanism of RNA recombination in poliovirus</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>. <span class="cit-vol">47</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKirkegaard%26rft.auinit1%253DK.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DThe%2Bmechanism%2Bof%2BRNA%2Brecombination%2Bin%2Bpoliovirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252886%252990600-8%26rft_id%253Dinfo%253Apmid%252F3021340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(86)90600-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3021340&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986E805900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.42" data-doi="10.1093/molbev/msl051"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kosakovsky</span>  <span class="cit-name-given-names">Pond SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gravenor</span>  <span class="cit-name-given-names">MB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woelk</span>  <span class="cit-name-given-names">CH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frost</span>  <span class="cit-name-given-names">SDW.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Automated phylogenetic detection of recombination using a genetic algorithm</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>. <span class="cit-vol">23</span>: <span class="cit-fpage">1891</span>–<span class="cit-lpage">1901</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl051%26rft_id%253Dinfo%253Apmid%252F16818476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msl051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16818476&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240586600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.43" data-doi="10.1006/viro.1995.0029"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kottier</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Britton</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-pub-date">1995</span>. <span class="cit-article-title">Experimental evidence of recombination in coronavirus infectious bronchitis virus</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>. <span class="cit-vol">213</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DKottier%26rft.auinit1%253DS.%2BA.%26rft.volume%253D213%26rft.issue%253D2%26rft.spage%253D569%26rft.epage%253D580%26rft.atitle%253DExperimental%2Bevidence%2Bof%2Brecombination%2Bin%2Bcoronavirus%2Binfectious%2Bbronchitis%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.1995.0029%26rft_id%253Dinfo%253Apmid%252F7491781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/viro.1995.0029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7491781&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TE73200029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makino</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keck</span>  <span class="cit-name-given-names">JG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Egbert</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leibowitz</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stohlman</span>  <span class="cit-name-given-names">SA.</span></span> <span class="cit-pub-date">1985</span>. <span class="cit-article-title">Recombination between nonsegmented RNA genomes of murine coronaviruses</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>. <span class="cit-vol">56</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLai%26rft.auinit1%253DM%2BM%26rft.volume%253D56%26rft.issue%253D2%26rft.spage%253D449%26rft.epage%253D456%26rft.atitle%253DRecombination%2Bbetween%2Bnonsegmented%2BRNA%2Bgenomes%2Bof%2Bmurine%2Bcoronaviruses.%26rft_id%253Dinfo%253Apmid%252F2997467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjU2LzIvNDQ5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMTIvMjMvMDIwODM0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.45" data-doi="10.1128/JVI.02219-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">SKP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">KSM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> (14 co-authors).<span class="cit-pub-date">2010</span>. <span class="cit-article-title">Ecoepidemiology and Complete Genome Comparison of Different Strains of Severe Acute Respiratory Syndrome-Related Rhinolophus Bat Coronavirus in China Reveal Bats as a Reservoir for Acute, Self-Limiting Infection That Allows Recombination Events</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>. <span class="cit-vol">84</span>: <span class="cit-fpage">2808</span>–<span class="cit-lpage">2819</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02219-09%26rft_id%253Dinfo%253Apmid%252F20071579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg0LzYvMjgwOCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE1LzEyLzIzLzAyMDgzNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lemey</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Reconstructing the initial global spread of a human influenza pandemic</span>. <abbr class="cit-jnl-abbrev">PLoS Currents</abbr>. <span class="cit-vol">1</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">RC.</span></span> <span class="cit-pub-date">1964</span>. <span class="cit-article-title">The interaction of selection and linkage. i. general considerations; heterotic models</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>. <span class="cit-vol">49</span>:<span class="cit-fpage">49</span>–<span class="cit-lpage">67</span>. PMID: 17248194 PMCID: PMC1210557.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLewontin%26rft.auinit1%253DR.%2BC.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D49%26rft.epage%253D67%26rft.atitle%253DTHE%2BINTERACTION%2BOF%2BSELECTION%2BAND%2BLINKAGE.%2BI.%2BGENERAL%2BCONSIDERATIONS%253B%2BHETEROTIC%2BMODELS%26rft_id%253Dinfo%253Apmid%252F17248194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiI0OS8xLzQ5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMTIvMjMvMDIwODM0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.48" data-doi="10.1038/nature02746"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">KS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> (22 co-authors).<span class="cit-pub-date">2004</span>. <span class="cit-article-title">Genesis of a highly pathogenic and potentially pandemic h5n1 influenza virus in eastern asia</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-vol">430</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DK.%2BS.%26rft.volume%253D430%26rft.issue%253D6996%26rft.spage%253D209%26rft.epage%253D213%26rft.atitle%253DGenesis%2Bof%2Ba%2Bhighly%2Bpathogenic%2Band%2Bpotentially%2Bpandemic%2BH5N1%2Binfluenza%2Bvirus%2Bin%2Beastern%2BAsia.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02746%26rft_id%253Dinfo%253Apmid%252F15241415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02746&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15241415&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222470600044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.49" data-doi="10.1186/1471-2148-14-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu L</span>,  <span class="cit-name-given-names">Lycett SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown AJ</span> <span class="cit-name-given-names">L.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Reassortment patterns of avian influenza virus internal segments among different subtypes</span>. <abbr class="cit-jnl-abbrev">BMC Evolutionary Biology</abbr>. <span class="cit-vol">14</span>: <span class="cit-fpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvolutionary%2BBiology%26rft.volume%253D14%26rft.spage%253D16%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-14-16%26rft_id%253Dinfo%253Apmid%252F24456010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-14-16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24456010&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Makino</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keck</span>  <span class="cit-name-given-names">JG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stohlman</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">MM.</span></span> <span class="cit-pub-date">1986</span>. <span class="cit-article-title">High-frequency RNA recombination of murine coronaviruses</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>. <span class="cit-vol">57</span>: <span class="cit-fpage">729</span>–<span class="cit-lpage">737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DMakino%26rft.auinit1%253DS%26rft.volume%253D57%26rft.issue%253D3%26rft.spage%253D729%26rft.epage%253D737%26rft.atitle%253DHigh-frequency%2BRNA%2Brecombination%2Bof%2Bmurine%2Bcoronaviruses.%26rft_id%253Dinfo%253Apmid%252F3005623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjU3LzMvNzI5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMTIvMjMvMDIwODM0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.51" data-doi="10.1093/oxfordjournals.molbev.a025960"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maynard</span>  <span class="cit-name-given-names">Smith J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">NH.</span></span> <span class="cit-pub-date">1998</span>. <span class="cit-article-title">Detecting recombination from gene trees</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>. <span class="cit-vol">15</span>: <span class="cit-fpage">590</span>–<span class="cit-lpage">599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DMaynard%2BSmith%26rft.auinit1%253DJ%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D590%26rft.epage%253D599%26rft.atitle%253DDetecting%2Brecombination%2Bfrom%2Bgene%2Btrees%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025960%26rft_id%253Dinfo%253Apmid%252F9580989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a025960&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9580989&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073343800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Awadalla</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fearnhead</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-pub-date">2002</span>. <span class="cit-article-title">A coalescent-based method for detecting and estimating recombination from gene sequences</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>. <span class="cit-vol">160</span>: <span class="cit-fpage">1231</span>–<span class="cit-lpage">1241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%26rft.volume%253D160%26rft.issue%253D3%26rft.spage%253D1231%26rft.epage%253D1241%26rft.atitle%253DA%2BCoalescent-Based%2BMethod%2Bfor%2BDetecting%2Band%2BEstimating%2BRecombination%2BFrom%2BGene%2BSequences%26rft_id%253Dinfo%253Apmid%252F11901136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MC8zLzEyMzEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNS8xMi8yMy8wMjA4MzQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.53" data-doi="10.1056/NEJMoa1303729"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Memish</span>  <span class="cit-name-given-names">ZA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zumla</span>  <span class="cit-name-given-names">AI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al-Hakeem</span>  <span class="cit-name-given-names">RF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al-Rabeeah</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">GM.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Family cluster of middle east respiratory syndrome coronavirus infections</span>. <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr>. <span class="cit-vol">368</span>: <span class="cit-fpage">2487</span>–<span class="cit-lpage">2494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.volume%253D368%26rft.spage%253D2487%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1303729%26rft_id%253Dinfo%253Apmid%252F23718156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1303729&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23718156&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000320926900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.54" data-doi="10.1093/oxfordjournals.molbev.a003756"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meunier</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2001</span>. <span class="cit-article-title">The correlation between linkage disequilibrium and distance: Implications for recombination in hominid mitochondria</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>. <span class="cit-vol">18</span>: <span class="cit-fpage">2132</span>–<span class="cit-lpage">2135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DMeunier%26rft.auinit1%253DJ.%26rft.volume%253D18%26rft.issue%253D11%26rft.spage%253D2132%26rft.epage%253D2135%26rft.atitle%253DThe%2BCorrelation%2BBetween%2BLinkage%2BDisequilibrium%2Band%2BDistance%253A%2BImplications%2Bfor%2BRecombination%2Bin%2BHominid%2BMitochondria%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a003756%26rft_id%253Dinfo%253Apmid%252F11606711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a003756&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11606711&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171998900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.55"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Milkman</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bridges</span>  <span class="cit-name-given-names">MM.</span></span> <span class="cit-pub-date">1990</span>. <span class="cit-article-title">Molecular evolution of the Escherichia coli chromosome. III. Clonal frames</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>. <span class="cit-vol">126</span>: <span class="cit-fpage">505</span>–<span class="cit-lpage">517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMilkman%26rft.auinit1%253DR.%26rft.volume%253D126%26rft.issue%253D3%26rft.spage%253D505%26rft.epage%253D517%26rft.atitle%253DMolecular%2Bevolution%2Bof%2Bthe%2BEscherichia%2Bcoli%2Bchromosome.%2BIII.%2BClonal%2Bframes.%26rft_id%253Dinfo%253Apmid%252F1979037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI2LzMvNTA1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMTIvMjMvMDIwODM0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.56" data-doi="10.1128/JVI.74.13.6015-6020.2000"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mitnaul</span>  <span class="cit-name-given-names">LJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matrosovich</span>  <span class="cit-name-given-names">MN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Castrucci</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tuzikov</span>  <span class="cit-name-given-names">AB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bovin</span>  <span class="cit-name-given-names">NV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kobasa</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kawaoka</span>  <span class="cit-name-given-names">Y.</span></span> <span class="cit-pub-date">2000</span>. <span class="cit-article-title">Balanced hemagglutinin and neuraminidase activities are critical for efficient replication of influenza a virus</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>. <span class="cit-vol">74</span>: <span class="cit-fpage">6015</span>–<span class="cit-lpage">6020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DMitnaul%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D74%26rft.issue%253D13%26rft.spage%253D6015%26rft.epage%253D6020%26rft.atitle%253DBalanced%2BHemagglutinin%2Band%2BNeuraminidase%2BActivities%2BAre%2BCritical%2Bfor%2BEfficient%2BReplication%2Bof%2BInfluenza%2BA%2BVirus%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.74.13.6015-6020.2000%26rft_id%253Dinfo%253Apmid%252F10846083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3NC8xMy82MDE1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMTIvMjMvMDIwODM0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.57" data-doi="10.1086/280418"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">HJ.</span></span> <span class="cit-pub-date">1932</span>. <span class="cit-article-title">Some genetic aspects of sex</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>. <span class="cit-vol">66</span>: <span class="cit-fpage">118</span>–<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D66%26rft.spage%253D118%26rft_id%253Dinfo%253Adoi%252F10.1086%252F280418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/280418&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200906900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corman</span>  <span class="cit-name-given-names">VM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jores</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> (12 co-authors).<span class="cit-pub-date">2014</span>. <span class="cit-article-title">MERS coronavirus neutralizing antibodies in camels, eastern africa, 1983–1997</span>. <abbr class="cit-jnl-abbrev">Emerging Infectious Diseases</abbr>. <span class="cit-vol">20</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.59" data-doi="10.1016/S1473-3099(15)70090-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corman</span>  <span class="cit-name-given-names">VM</span></span>, <span class="cit-etal">et al.</span> (19 co-authors).<span class="cit-pub-date">2015</span>. <span class="cit-article-title">Presence of Middle East respiratory syndrome coronavirus antibodies in Saudi Arabia: a nationwide, crosssectional, serological study</span>. <abbr class="cit-jnl-abbrev">The Lancet Infectious Diseases</abbr>. <span class="cit-vol">15</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">564</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%2BInfectious%2BDiseases%26rft.volume%253D15%26rft.spage%253D559%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252815%252970090-3%26rft_id%253Dinfo%253Apmid%252F25863564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1473-3099(15)70090-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25863564&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.60" data-doi="10.1093/molbev/msp003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">O’Brien JD</span>,  <span class="cit-name-given-names">Minin VN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Learning to count: Robust estimates for labeled distances between molecular sequences</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>. <span class="cit-vol">26</span>:<span class="cit-fpage">801</span>–<span class="cit-lpage">814</span>. PMID: 19131426.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp003%26rft_id%253Dinfo%253Apmid%252F19131426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msp003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19131426&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264188700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.61" data-doi="10.1093/bioinformatics/bti079"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pond</span>  <span class="cit-name-given-names">SLK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frost</span>  <span class="cit-name-given-names">SDW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muse</span>  <span class="cit-name-given-names">SV.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">HyPhy: hypothesis testing using phylogenies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-vol">21</span>: <span class="cit-fpage">676</span>–<span class="cit-lpage">679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti079%26rft_id%253Dinfo%253Apmid%252F15509596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15509596&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227241200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.62" data-doi="10.1073/pnas.241370698"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crandall</span>  <span class="cit-name-given-names">KA.</span></span> <span class="cit-pub-date">2001</span>. <span class="cit-article-title">Evaluation of methods for detecting recombination from DNA sequences: Computer simulations</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>. <span class="cit-vol">98</span>: <span class="cit-fpage">13757</span>–<span class="cit-lpage">13762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DPosada%26rft.auinit1%253DD.%26rft.volume%253D98%26rft.issue%253D24%26rft.spage%253D13757%26rft.epage%253D13762%26rft.atitle%253DEvaluation%2Bof%2Bmethods%2Bfor%2Bdetecting%2Brecombination%2Bfrom%2BDNA%2Bsequences%253A%2BComputer%2Bsimulations%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.241370698%26rft_id%253Dinfo%253Apmid%252F11717435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTgvMjQvMTM3NTciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNS8xMi8yMy8wMjA4MzQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.63" data-doi="10.1007/s00239-001-0034-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crandall</span>  <span class="cit-name-given-names">KA.</span></span> <span class="cit-pub-date">2002</span>. <span class="cit-article-title">The effect of recombination on the accuracy of phylogeny estimation</span>. <abbr class="cit-jnl-abbrev">Journal of Molecular Evolution</abbr>. <span class="cit-vol">54</span>: <span class="cit-fpage">396</span>–<span class="cit-lpage">402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DPosada%26rft.auinit1%253DD.%26rft.volume%253D54%26rft.issue%253D3%26rft.spage%253D396%26rft.epage%253D402%26rft.atitle%253DThe%2Beffect%2Bof%2Brecombination%2Bon%2Bthe%2Baccuracy%2Bof%2Bphylogeny%2Bestimation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-001-0034-9%26rft_id%253Dinfo%253Apmid%252F11847565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-001-0034-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11847565&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173928400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.64" data-doi="10.1146/annurev.genet.36.040202.111115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crandall</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC.</span></span> <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Recombination in Evolutionary Genomics</span>. <abbr class="cit-jnl-abbrev">Annual Review of Genetics</abbr>. <span class="cit-vol">36</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">97</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BGenetics%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.36.040202.111115%26rft_id%253Dinfo%253Apmid%252F12429687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.36.040202.111115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12429687&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180365100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.65"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">The early molecular epidemiology of the swine-origin a/h1n1 human influenza pandemic</span>. <abbr class="cit-jnl-abbrev">PLoS Currents</abbr>. <span class="cit-vol">1</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.66" data-doi="10.1016/S1473-3099(13)70164-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reusken</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haagmans</span>  <span class="cit-name-given-names">BL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-etal">et al.</span> (24 co-authors).<span class="cit-pub-date">2013</span>. <span class="cit-article-title">Middle east respiratory syndrome coronavirus neutralising serum antibodies in dromedary camels: a comparative serological study</span>. <abbr class="cit-jnl-abbrev">The Lancet Infectious Diseases</abbr>. <span class="cit-vol">13</span>: <span class="cit-fpage">859</span>–<span class="cit-lpage">866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%2BInfectious%2BDiseases%26rft.volume%253D13%26rft.spage%253D859%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252813%252970164-6%26rft_id%253Dinfo%253Apmid%252F23933067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1473-3099(13)70164-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23933067&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000324899700020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.67" data-doi="10.3201/eid2008.140590"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reusken</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Messadi</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feyisa</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> (17 co-authors).<span class="cit-pub-date">2014</span>. <span class="cit-article-title">Geographic distribution of MERS coronavirus among dromedary camels, africa</span>. <abbr class="cit-jnl-abbrev">Emerging Infectious Diseases</abbr>. <span class="cit-vol">20</span>: <span class="cit-fpage">1370</span>–<span class="cit-lpage">1374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2BInfectious%2BDiseases%26rft.volume%253D20%26rft.spage%253D1370%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2008.140590%26rft_id%253Dinfo%253Apmid%252F25062254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2008.140590&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25062254&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.68" data-doi="10.1007/BF00163230"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">DL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">BH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">PM.</span></span> <span class="cit-pub-date">1995a</span>. <span class="cit-article-title">Recombination in AIDS viruses</span>. <abbr class="cit-jnl-abbrev">Journal of Molecular Evolution</abbr>. <span class="cit-vol">40</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DRobertson%26rft.auinit1%253DD.%2BL.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D249%26rft.epage%253D259%26rft.atitle%253DRecombination%2Bin%2BAIDS%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00163230%26rft_id%253Dinfo%253Apmid%252F7723052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00163230&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7723052&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QN47900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.69" data-doi="10.1038/374124b0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">DL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">PM</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCutchan</span>  <span class="cit-name-given-names">FE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">BH.</span></span> <span class="cit-pub-date">1995b</span>. <span class="cit-article-title">Recombination in HIV-1</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-vol">374</span>: <span class="cit-fpage">124</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNakamura%26rft.auinit1%253DM.%2BT.%26rft.volume%253D374%26rft.issue%253D6518%26rft.spage%253D124%26rft.epage%253D124%26rft.atitle%253DAnother%2Bobese%2Bgene%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F374124b0%26rft_id%253Dinfo%253Apmid%252F7877681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/374124b0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7877681&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.70"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schierup</span>  <span class="cit-name-given-names">MH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hein</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2000</span>. <span class="cit-article-title">Consequences of recombination on traditional phylogenetic analysis</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>. <span class="cit-vol">156</span>: <span class="cit-fpage">879</span>–<span class="cit-lpage">891</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSchierup%26rft.auinit1%253DM.%2BH.%26rft.volume%253D156%26rft.issue%253D2%26rft.spage%253D879%26rft.epage%253D891%26rft.atitle%253DConsequences%2Bof%2BRecombination%2Bon%2BTraditional%2BPhylogenetic%2BAnalysis%26rft_id%253Dinfo%253Apmid%252F11014833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU2LzIvODc5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMTIvMjMvMDIwODM0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.71" data-doi="10.1017/S0016672303006128"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shriner</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nickle</span>  <span class="cit-name-given-names">DC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mullins</span>  <span class="cit-name-given-names">JI.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Potential impact of recombination on sitewise approaches for detecting positive natural selection</span>. <abbr class="cit-jnl-abbrev">Genetics Research</abbr>. <span class="cit-vol">81</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%2BResearch%26rft.volume%253D81%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672303006128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672303006128&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.72" data-doi="10.1038/nature08182"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">GJD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vijaykrishna</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bahl</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> (13 co-authors).<span class="cit-pub-date">2009</span>. <span class="cit-article-title">Origins and evolutionary genomics of the 2009 swine-origin h1n1 influenza a epidemic</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-vol">459</span>: <span class="cit-fpage">1122</span>–<span class="cit-lpage">1125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSmith%26rft.auinit1%253DG.%2BJ.%26rft.volume%253D459%26rft.issue%253D7250%26rft.spage%253D1122%26rft.epage%253D1125%26rft.atitle%253DOrigins%2Band%2Bevolutionary%2Bgenomics%2Bof%2Bthe%2B2009%2Bswine-origin%2BH1N1%2Binfluenza%2BA%2Bepidemic.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08182%26rft_id%253Dinfo%253Apmid%252F19516283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08182&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19516283&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267636700042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.73"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Subbarao</span>  <span class="cit-name-given-names">EK</span></span>, <span class="cit-auth"><span class="cit-name-surname">London</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">BR.</span></span> <span class="cit-pub-date">1993</span>. <span class="cit-article-title">A single amino acid in the PB2 gene of influenza a virus is a determinant of host range</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>. <span class="cit-vol">67</span>: <span class="cit-fpage">1761</span>–<span class="cit-lpage">1764</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DSubbarao%26rft.auinit1%253DE%2BK%26rft.volume%253D67%26rft.issue%253D4%26rft.spage%253D1761%26rft.epage%253D1764%26rft.atitle%253DA%2Bsingle%2Bamino%2Bacid%2Bin%2Bthe%2BPB2%2Bgene%2Bof%2Binfluenza%2BA%2Bvirus%2Bis%2Ba%2Bdeterminant%2Bof%2Bhost%2Brange.%26rft_id%253Dinfo%253Apmid%252F8445709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjY3LzQvMTc2MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE1LzEyLzIzLzAyMDgzNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-book" id="cit-020834v2.74"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tavaré</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">1986</span>. <span class="cit-chapter-title">Some Mathematical Questions in Biology: DNA Sequence Analysis</span>. <span class="cit-source">Lectures on Mathematics in the Life Sciences</span>, volume <span class="cit-vol">17</span>, <span class="cit-fpage">57</span>–<span class="cit-lpage">86</span>. <span class="cit-publ-loc">American Mathematical Society</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLectures%2Bon%2BMathematics%2Bin%2Bthe%2BLife%2BSciences%26rft.volume%253D17%26rft.spage%253D57%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.75" data-doi="10.1073/pnas.90.12.5653"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tisdale</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kemp</span>  <span class="cit-name-given-names">SD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parry</span>  <span class="cit-name-given-names">NR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Larder</span>  <span class="cit-name-given-names">BA.</span></span> <span class="cit-pub-date">1993</span>. <span class="cit-article-title">Rapid in vitro selection of human immunodeficiency virus type 1 resistant to 3’-thiacytidine inhibitors due to a mutation in the YMDD region of reverse transcriptase</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr>. <span class="cit-vol">90</span>: <span class="cit-fpage">5653</span>–<span class="cit-lpage">5656</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DTisdale%26rft.auinit1%253DM.%26rft.volume%253D90%26rft.issue%253D12%26rft.spage%253D5653%26rft.epage%253D5656%26rft.atitle%253DRapid%2Bin%2Bvitro%2Bselection%2Bof%2Bhuman%2Bimmunodeficiency%2Bvirus%2Btype%2B1%2Bresistant%2Bto%2B3%2527-thiacytidine%2Binhibitors%2Bdue%2Bto%2Ba%2Bmutation%2Bin%2Bthe%2BYMDD%2Bregion%2Bof%2Breverse%2Btranscriptase.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.90.12.5653%26rft_id%253Dinfo%253Apmid%252F7685907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTAvMTIvNTY1MyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE1LzEyLzIzLzAyMDgzNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.76" data-doi="10.1007/BF00160154"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span> <span class="cit-pub-date">1994</span>. <span class="cit-article-title">Maximum likelihood phylogenetic estimation from DNA sequences with variable rates over sites: Approximate methods</span>. <abbr class="cit-jnl-abbrev">Journal of Molecular Evolution</abbr>. <span class="cit-vol">39</span>: <span class="cit-fpage">306</span>–<span class="cit-lpage">314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DYang%26rft.auinit1%253DZ.%26rft.volume%253D39%26rft.issue%253D3%26rft.spage%253D306%26rft.epage%253D314%26rft.atitle%253DMaximum%2Blikelihood%2Bphylogenetic%2Bestimation%2Bfrom%2BDNA%2Bsequences%2Bwith%2Bvariable%2Brates%2Bover%2Bsites%253A%2Bapproximate%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00160154%26rft_id%253Dinfo%253Apmid%252F7932792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00160154&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7932792&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PB53900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-020834v2.77" data-doi="10.1056/NEJMoa1211721"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zaki</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Boheemen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bestebroer</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osterhaus</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fouchier</span>  <span class="cit-name-given-names">RA.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Isolation of a novel coronavirus from a man with pneumonia in saudi arabia</span>. <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr>. <span class="cit-vol">367</span>: <span class="cit-fpage">1814</span>–<span class="cit-lpage">1820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DZaki%26rft.auinit1%253DA.%2BM.%26rft.volume%253D367%26rft.issue%253D19%26rft.spage%253D1814%26rft.epage%253D1820%26rft.atitle%253DIsolation%2Bof%2Ba%2Bnovel%2Bcoronavirus%2Bfrom%2Ba%2Bman%2Bwith%2Bpneumonia%2Bin%2BSaudi%2BArabia.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1211721%26rft_id%253Dinfo%253Apmid%252F23075143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1211721&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23075143&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F12%2F23%2F020834.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310773200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/020834v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F020834v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/030601v2" title="Hydroxide Once or Twice? A combined Neutron Crystallographic and Quantum Chemical Study of the Hydride Shift in D-Xylose Isomerase" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/021030v3" title="Environmental fluctuations do not select for increased variation or population-based resistance in Escherichia coli" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;December 23, 2015.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/020834v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/020834v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/020834v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/020834v2" class="active" data-icon-position="" data-hide-link-title="0">MERS-CoV recombination: implications about the reservoir and potential for adaptation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/9411" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-y2ZXoQBmVbwGciBk9SD6gkz9ZazF5KPIUZSTmcG_3Vw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="9411" id="node9411--2" data-pisa="biorxiv;020834v2" data-pisa-master="biorxiv;020834" data-seqnum="9411" data-apath="/biorxiv/early/2015/12/23/020834.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">MERS-CoV recombination: implications about the reservoir and potential for adaptation</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Gytis</span> <span class="nlm-surname">Dudas</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Andrew</span> <span class="nlm-surname">Rambaut</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">020834; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/020834 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/020834v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/020834v2&amp;title=MERS-CoV%20recombination%3A%20implications%20about%20the%20reservoir%20and%20potential%20for%20adaptation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/020834v2&amp;title=MERS-CoV%20recombination%3A%20implications%20about%20the%20reservoir%20and%20potential%20for%20adaptation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/020834v2&amp;text=MERS-CoV%20recombination%3A%20implications%20about%20the%20reservoir%20and%20potential%20for%20adaptation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/020834v2&amp;title=MERS-CoV%20recombination%3A%20implications%20about%20the%20reservoir%20and%20potential%20for%20adaptation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/020834v2&amp;t=MERS-CoV%20recombination%3A%20implications%20about%20the%20reservoir%20and%20potential%20for%20adaptation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/020834v2&amp;title=MERS-CoV%20recombination%3A%20implications%20about%20the%20reservoir%20and%20potential%20for%20adaptation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/020834v2&amp;title=MERS-CoV%20recombination%3A%20implications%20about%20the%20reservoir%20and%20potential%20for%20adaptation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="9411" id="node9411--3" data-pisa="biorxiv;020834v2" data-pisa-master="biorxiv;020834" data-seqnum="9411" data-apath="/biorxiv/early/2015/12/23/020834.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">MERS-CoV recombination: implications about the reservoir and potential for adaptation</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Gytis</span> <span class="nlm-surname">Dudas</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Andrew</span> <span class="nlm-surname">Rambaut</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">020834; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/020834 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=MERS-CoV%20recombination%3A%20implications%20about%20the%20reservoir%20and%20potential%20for%20adaptation&amp;rft.volume&amp;rft.issue&amp;rft.date=2015-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Dudas%2C+Gytis&amp;rft.au=Rambaut%2C+Andrew"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/9411/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/9411/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/9411/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/9411/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/9411/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/9411/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/9411/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/9411/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/9411/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/9411/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/9411/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/9411/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/020834v2&amp;count=horizontal&amp;via=&amp;text=MERS-CoV%20recombination%3A%20implications%20about%20the%20reservoir%20and%20potential%20for%20adaptation&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/020834v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/020834v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/020834v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
