<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS Pathog</span><span class="iso-abbrev" title="iso-abbrev">PLoS Pathog</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plospath</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS Pathogens</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1553-7366</span><span class="issn-epub" tagx="issn" title="issn-epub">1553-7374</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6093697">6093697</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/30075025">30075025</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.ppat.1007190">10.1371/journal.ppat.1007190</a></span><span class="publisher-id" title="publisher-id">PPATHOGENS-D-17-02324</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Virology</span>: <span class="subject" title="subject">Viral Replication</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pulmonology</span>: <span class="subject" title="subject">Respiratory Infections</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Anatomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Digestive System</span>: <span class="subject" title="subject">Gastrointestinal Tract</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Anatomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Digestive System</span>: <span class="subject" title="subject">Gastrointestinal Tract</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Physiology</span>: <span class="subject" title="subject">Cell Binding</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cellular Types</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Animal Cells</span>: <span class="subject" title="subject">Neurons</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Neuroscience</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cellular Neuroscience</span>: <span class="subject" title="subject">Neurons</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biological Cultures</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Lines</span>: <span class="subject" title="subject">Caco-2 Cells</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Chemical Characterization</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Binding Analysis</span>: <span class="subject" title="subject">Cell Binding Assay</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Virology</span>: <span class="subject" title="subject">Viral Tropism</span></div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
A VP1 mutation acquired during an enterovirus 71 disseminated infection confers heparan sulfate binding ability and modulates <i>ex vivo</i> tropism</div> <div class="alt-title" title="alt-title">
Heparan sulfate binding as a determinant of enterovirus 71 tropism</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tseligka</span><span class="given-names" tagx="given-names" title="given-names">Eirini D.</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Validation</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sobo</span><span class="given-names" tagx="given-names" title="given-names">Komla</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stoppini</span><span class="given-names" tagx="given-names" title="given-names">Luc</span></span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-5597-334X</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cagno</span><span class="given-names" tagx="given-names" title="given-names">Valeria</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abdul</span><span class="given-names" tagx="given-names" title="given-names">Fabien</span></span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Supervision</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piuz</span><span class="given-names" tagx="given-names" title="given-names">Isabelle</span></span><span class="role" tagx="role" title="role">Methodology</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meylan</span><span class="given-names" tagx="given-names" title="given-names">Pascal</span></span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Song</span></span><span class="role" tagx="role" title="role">Resources</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Constant</span><span class="given-names" tagx="given-names" title="given-names">Samuel</span></span><span class="role" tagx="role" title="role">Resources</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-0411-6567</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tapparel</span><span class="given-names" tagx="given-names" title="given-names">Caroline</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Funding acquisition</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Supervision</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#cor001">*</a></span></div><span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">Department of Microbiology and Molecular Medicine, University of Geneva Medical School, Geneva, Switzerland</span></span><span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Tissue Engineering Laboratory, HES-SO/University of Applied Sciences, Geneva, Western Switzerland</span></span><span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">Institute of Microbiology, Centre Hospitalier Universitaire Vaudois, Lausanne, Switzerland</span></span><span class="citation_author_institution" id="aff004">[4], <span class="addr-line" title="addr-line">Epithelix Sàrl, Geneva, Switzerland</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shih</span><span class="given-names" tagx="given-names" title="given-names">Shin-Ru</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Chang Gung University, TAIWAN</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p>SH and SC are employees of Epithelix Sàrl. There is no other relevant information linked to this affiliation that presents any conflict of interest. Moreover, we confirm that this affiliation does not alter our adherence to all PLOS Pathogens policies on sharing data and materials.</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">Caroline.Tapparel@unige.ch</span></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-8-8</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2018-8</span></span><span class="volume" tagx="volume" title="volume">14</span><span class="issue" tagx="issue" title="issue">8</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1007190</span><span class="history" title="history"><span class="received" title="received">received: 2017-10-23</span><span class="accepted" title="accepted">accepted: 2018-6-29</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Tseligka et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></div><a class="self-uri" href="ppat.1007190.pdf" title="self-uri">ppat.1007190.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Enterovirus 71 (EV71) causes hand, foot and mouth disease, a mild and self-limited illness that is sometimes associated with severe neurological complications. EV71 neurotropic determinants remain ill-defined to date. We previously identified a mutation in the VP1 capsid protein (L97R) that was acquired over the course of a disseminated infection in an immunocompromised host. The mutation was absent in the respiratory tract but was present in the gut (as a mixed population) and in blood and cerebrospinal fluid (as a dominant species). In this study, we demonstrated that this mutation does not alter the dependence of EV71 on the human scavenger receptor class B2 (SCARB2), while it enables the virus to bind to the heparan sulfate (HS) attachment receptor and modifies viral tropism in cell lines and in respiratory, intestinal and neural tissues. Variants with VP1<sub>97L</sub> or VP1<sub>97R</sub> were able to replicate to high levels in intestinal and neural tissues and, to a lesser extent, in respiratory tissues, but their preferred entry site (from the luminal or basal tissue side) differed in respiratory and intestinal tissues and correlated with HS expression levels. These data account for the viral populations sequenced from the patient’s respiratory and intestinal samples and suggest that improved dissemination, resulting from an acquired ability to bind HS, rather than specific neurotropism determinants, enabled the virus to reach and infect the central nervous system. Finally, we showed that iota-carrageenan, a highly sulfated polysaccharide, efficiently blocks the replication of HS-dependent variants in cells and 2D neural cultures. Overall, the results of this study emphasize the importance of HS binding in EV71 pathogenesis and open new avenues for the development of antiviral molecules that may prevent this virus’s dissemination.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Author summary</div> <p>Enterovirus 71 (EV71) has been the cause of major hand-foot-and-mouth disease outbreaks, particularly in the Asia-Pacific region. However, the recent emergence of severe neurological cases associated with EV71 infection in Europe and the lack of an efficient vaccine or antiviral agent to fight EV71 infections highlight two critical needs: (A) the identification of ill-defined viral factors that contribute to viral dissemination and pathogenesis in humans and (B) the development of effective antiviral strategies. Herein, based on clinical observation in an immunocompromised host, we have demonstrated that heparan sulfate attachment receptor played a critical role in EV71 virulence and that "in host�? EV71 adaptation to a heparan sulfate-dependent virus was likely responsible for its dissemination. To our knowledge, this is the first study highlighting the key determinants of EV71 dissemination based on a clinical case and proposing a new therapeutic approach against EV71 neurological diseases.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Swiss National Science Foundation</span></div><span class="award-id" tagx="award-id" title="award-id">310030_166218</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-0411-6567</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tapparel</span><span class="given-names" tagx="given-names" title="given-names">Caroline</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100006387</span><span class="institution" title="institution">Fondation Leenaards</span></div> </div><span class="award-id" tagx="award-id" title="award-id">4390</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-0411-6567</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tapparel</span><span class="given-names" tagx="given-names" title="given-names">Caroline</span></span></span></div> <div class="funding-statement" title="funding-statement">
This work is supported by the Swiss National Science Foundation (grant # 310030_166218 to CT) and the Leenaards Foundation (grant 4390 to CT). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="PLOS Publication Stage" value="vor-update-to-uncorrected-proof" /> <meta name="Publication Update" value="2018-08-15" /> <meta name="Data Availability" value="All relevant data are available within the manuscript." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>All relevant data are available within the manuscript.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Enterovirus 71 (EV71) is one of a limited number of enterovirus genotypes that have the ability to infect the central nervous system (CNS) [<a href="#ppat.1007190.ref001">1</a>–<a href="#ppat.1007190.ref003">3</a>] and has emerged as a major health-care threat across the Asia-Pacific region [<a href="#ppat.1007190.ref004">4</a>–<a href="#ppat.1007190.ref007">7</a>]. Although this virus is typically associated with mild hand, foot and mouth disease (HFMD) epidemics, EV71 has been increasingly associated with neurological disorders that range from aseptic meningitis with or without pulmonary edema to brain stem encephalitis and poliomyelitis-like acute flaccid paralysis, particularly among children less than 6 years old [<a href="#ppat.1007190.ref008">8</a>–<a href="#ppat.1007190.ref010">10</a>]. EV71 transmission depends on hygiene, water quality, and the extent of crowding [<a href="#ppat.1007190.ref011">11</a>]. EV71 is typically transmitted through fecal-oral routes, although transmission via respiratory secretions also frequently occurs, particularly in countries with high standard sanitation [<a href="#ppat.1007190.ref012">12</a>, <a href="#ppat.1007190.ref013">13</a>].</p> <p>Following infection, EV71 predominantly replicates in the intestinal mucosa and, to a lesser extent, in the respiratory mucosa [<a href="#ppat.1007190.ref005">5</a>, <a href="#ppat.1007190.ref014">14</a>]. Subsequently, the virus can enter the bloodstream and disseminate to a variety of organs, including the central nervous system (CNS) [<a href="#ppat.1007190.ref015">15</a>]. EV71 dissemination to the CNS remains a rare event and the viral neurotropic determinants remain ill-defined, despite extensive epidemiological studies and experimentation in animal models.</p> <p>EV71 tissue tropism may be modulated by its ability to bind to a variety of receptors [<a href="#ppat.1007190.ref016">16</a>]. The human scavenger receptor class B2 (SCARB2), a major transmembrane lysosomal protein, is ubiquitously expressed and is the best studied receptor [<a href="#ppat.1007190.ref017">17</a>]. In the human CNS, SCARB2 is expressed on neurons and glial cells [<a href="#ppat.1007190.ref018">18</a>], and a transgenic mouse model expressing human SCARB2 has exhibited susceptibility to EV71 infection and the development of ataxia, paralysis and death [<a href="#ppat.1007190.ref019">19</a>]. The second EV71 entry receptor, P-selectin glycoprotein ligand-1 (PSGL-1), is expressed exclusively on leukocytes and is only bound by certain EV71 strains [<a href="#ppat.1007190.ref020">20</a>, <a href="#ppat.1007190.ref021">21</a>]. Finally, attachment molecules, such as heparan sulfate glycosaminoglycan (HS) [<a href="#ppat.1007190.ref022">22</a>], sialic acids [<a href="#ppat.1007190.ref023">23</a>], nucleolin [<a href="#ppat.1007190.ref024">24</a>], vimentin [<a href="#ppat.1007190.ref025">25</a>] and annexin II [<a href="#ppat.1007190.ref026">26</a>] also enhance EV71 infectivity and may contribute to viral dissemination and neurotropism. Residues that are crucial for the binding of EV71 to certain of these receptors have been mapped on the VP1 viral capsid protein. For instance, 172Q and adjacent amino acids in the EF loop are required for binding to human SCARB2 and for efficient infection of cells expressing this receptor [<a href="#ppat.1007190.ref027">27</a>] while K98E, E145A and L169F substitutions confer binding ability to murine SCARB2 [<a href="#ppat.1007190.ref028">28</a>]. Similarly, EV71 strains with a glycine or a glutamine at position 145 of VP1 are able to bind PSGL-1, while strains with a glutamate at this position are not [<a href="#ppat.1007190.ref021">21</a>]. Binding of EV71 to annexin II has been mapped to the BC loop region of VP1 (residues 40–100) [<a href="#ppat.1007190.ref026">26</a>]. Finally, positively charged residues at the 5-fold axis of EV71 capsids (VP1 162K, 242K, and 244K) appear to be essential for attachment to HS. When these residues are mutated, compensatory mutations (E98A, T100K and Q145R-T237N) can restore binding. Similarly, strains with 98E and 145E are poor HS binders, while E98K restores binding [<a href="#ppat.1007190.ref029">29</a>]. These observations are based on <i>in vitro</i> adapted EV71 isolates and do not address the <i>in vivo</i> significance of dissemination and neurotropism. In addition, the implication of immune escape mutations may add a level of complexity. VP1 residues 98 and 145 have been shown to be under positive selection and may be targeted by neutralizing antibodies [<a href="#ppat.1007190.ref030">30</a>, <a href="#ppat.1007190.ref031">31</a>]. Finally, EV71 virulence determinants that have been identified in animal models contradict epidemiological studies in humans. VP1 residue 145E was shown to be a virulence factor for both cynomolgus monkeys [<a href="#ppat.1007190.ref031">31</a>, <a href="#ppat.1007190.ref032">32</a>] and mice [<a href="#ppat.1007190.ref033">33</a>, <a href="#ppat.1007190.ref034">34</a>], while isolates with 145G/Q are also frequently associated with severe neurological disease in humans [<a href="#ppat.1007190.ref035">35</a>–<a href="#ppat.1007190.ref038">38</a>]. Mutations in other genomic regions may explain these apparent contradictions.</p> <p>We previously isolated EV71 from an immunocompromised patient with disseminated disease. Genomic analysis revealed that this clinical strain clusters with EV71 in the human EV-A species and is related to the subgenogroup C1. Comparison of five full-length genomes sequenced directly from respiratory, gastrointestinal, nervous system and blood specimens highlighted a critical non-synonymous single amino acid change within the EV71 VP1 BC loop (L97R) that could potentially lead to dissemination in natural infections. EV71-VP1<sub>97R</sub>, which was absent in the respiratory sample, was present as a dominant population in blood and cerebrospinal fluid samples and as a mixed population (EV71-VP1<sub>97R/L</sub>) in the stool sample [<a href="#ppat.1007190.ref039">39</a>]. However, the mechanisms by which the mutant virus could disseminate are not clear. In this study, we investigated the capacity of EV71-VP1<sub>97L</sub> and EV71-VP1<sub>97R</sub> to bind and mediate infection in different cell lines and in 3D reconstituted respiratory, intestinal and neural tissues. We demonstrated that both derivatives are dependent on SCARB2 for effective infection, while their specific tropism is linked to a different ability to bind HS and further correlates with the expression level of this attachment receptor in the tested tissues. This study provides new insight regarding HS binding as a critical determinant of EV71 dissemination in humans and opens the door to antiviral strategies aimed at preventing EV71 in-host adaptation and dissemination.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="ev71-vp197randev71-vp197lvariantsexhibitdifferentcelltropism" title="sec"> <div class="title" tagx="title" title="title">
EV71-VP1<sub>97R</sub> and EV71-VP1<sub>97L</sub> variants exhibit different cell tropism</div> <p>We previously showed that the VP1 L97R mutation conferred a binding advantage in RD and Vero cells. In addition, we showed <i>in vitro</i> that this mutation was associated with a second non-conservative mutation (E167G/A) in the VP1 EF loop [<a href="#ppat.1007190.ref039">39</a>]. To define the role of these mutations more precisely, we generated 4 infectious clones with distinct combinations of each of the mutations (<a href="#ppat.1007190.t001">Table 1</a>).</p> <div class="table-wrap_UNKNOWN" id="ppat.1007190.t001" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.ppat.1007190.t001">10.1371/journal.ppat.1007190.t001</a><span class="label" tagx="label" title="label">Table 1</span> <div class="title" tagx="title" title="title">
Schematic representation of the 4 infectious clone constructs and spontaneous mutations arising in culture.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="ppat.1007190.t001g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="justify" style="border-right:dotted" rowspan="1" colspan="1">Construct name</th> <th align="justify" style="border-left:dotted" rowspan="1" colspan="1">VP1 changes in construct</th> <th align="justify" rowspan="1" colspan="1">Spontaneous mutations</th> </tr> </thead> <tbody> <tr> <td align="justify" style="border-right:dotted" rowspan="1" colspan="1"><a href="#t001fn001">*</a><b>EV71-VP1<sub>97L167E</sub></b></td> <td align="justify" style="border-left:dotted" rowspan="1" colspan="1">L<sup>0</sup><sub>97</sub> E<sup>−</sup><sub>167</sub></td> <td align="justify" rowspan="1" colspan="1">No</td> </tr> <tr> <td align="justify" style="border-right:dotted" rowspan="1" colspan="1"><b>EV71-VP1<sub>97L167E</sub></b></td> <td align="justify" style="border-left:dotted" rowspan="1" colspan="1">L<sup>0</sup><sub>97</sub> G<sup>0</sup><sub>167</sub></td> <td align="justify" rowspan="1" colspan="1">97 L to R</td> </tr> <tr> <td align="justify" style="border-right:dotted" rowspan="1" colspan="1"><a href="#t001fn002">**</a><b>EV71-VP1<sub>97R167E</sub></b></td> <td align="justify" style="border-left:dotted" rowspan="1" colspan="1">R<sup>+</sup><sub>97</sub> E<sup>−</sup><sub>167</sub></td> <td align="justify" rowspan="1" colspan="1">167 E to G/A</td> </tr> <tr> <td align="justify" style="border-right:dotted" rowspan="1" colspan="1"><b>EV71-VP1<sub>97R167G</sub></b></td> <td align="justify" style="border-left:dotted" rowspan="1" colspan="1">R<sup>+</sup><sub>97</sub> G<sup>0</sup><sub>167</sub></td> <td align="justify" rowspan="1" colspan="1">No</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>*Virus sequenced from the bronchoalveolar lavage of a patient with disseminated EV71 infection.</p> </div> <div class="fn-type-" title=""> <p>**Virus sequenced from the stool, blood and cerebrospinal fluid of a patient with disseminated EV71 infection [<a href="#ppat.1007190.ref039">39</a>]. +, positive charge; -, negative charge; o, neutral charge.</p> </div> </div> </div> <p>Viral stocks were prepared in monkey kidney cells (Vero) and titrated in Vero, colorectal carcinoma (Caco-2), rhabdomyosarcoma (RD) and neuroblastoma cells (SH-SY5Y). After 4 days of infection, viruses were sequenced from each cell line. EV71-VP1<sub>97L167G</sub> and EV71-VP1<sub>97R167E</sub> viral stocks were unstable independent on the cell line (<a href="#ppat.1007190.t001">Table 1</a>) and thus could not be amplified and purified in sufficient quantities to be used in functional assays. EV71-VP1<sub>97L167E</sub> (originally isolated from the lower respiratory tract sample of the patient with a disseminated infection) and EV71-VP1<sub>97R167G</sub> (<i>in vitro</i> adapted from EV71-VP1<sub>97R167E</sub>, which was originally isolated from the stool, cerebrospinal fluid and blood of the patient) were thus used for all further investigations. Titration showed that EV71-VP1<sub>97L167E</sub> and EV71-VP1<sub>97R167G</sub> exhibited differential cell tropism, with significantly enhanced replication being observed for EV71-VP1<sub>97R167G</sub> in RD and SH-SY5Y cells (<a href="#ppat.1007190.g001">Fig 1</a>). Notably, the RNA load measured in the stocks prepared in Vero cells was comparable for the two viruses (12.38 log RNA copies/ml for EV71-VP1<sub>97R167G</sub> and 12.30 log RNA copies/ml for EV71-VP1<sub>97L167E</sub>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007190.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
Differential cell tropism EV71-VP1<sub>97R167G</sub> and EV71-VP1<sub>97L167E</sub>.</div> <p>The mean (N = 2) viral titers (±SEM) expressed as log TCID<sub>50</sub>/ml (median tissue culture infective dose) obtained 4 dpi are indicated. *P&amp;lt;0.05.</p>   </div> </div> <div class="ev71-vp197r167gandev71-vp197l167einfectionsaredependentonscarb2" title="sec"> <div class="title" tagx="title" title="title">
EV71-VP1<sub>97R167G</sub> and EV71-VP1<sub>97L167E</sub> infections are dependent on SCARB2</div> <p>As SCARB2 plays a critical role for EV71 infection [<a href="#ppat.1007190.ref017">17</a>], we investigated whether the differential cell tropism of the two variants is linked to a different dependence on this receptor. We used the CRISPR/Cas9 system to knockout SCARB2 in Caco-2 and RD cells, two cell lines presenting different susceptibilities to the two variants (<a href="#ppat.1007190.g001">Fig 1</a>). A striking reduction in viral titers was observed in SCARB2 knockout RD and Caco-2 cells for EV71-VP1<sub>97L167E</sub> and EV71-VP1<sub>97R167G</sub> (<a href="#ppat.1007190.g002">Fig 2</a>) suggesting that both variants depend on this receptor for effective infection regardless of the cell line. This was also shown by transfection of human SCARB2 in mouse L929 cells that are non-permissive for infection by EV71 [<a href="#ppat.1007190.ref017">17</a>]. Exogenous expression of human SCARB2 conferred susceptibility to both EV71-VP1<sub>97L167E</sub> and EV71-VP1<sub>97R167G</sub> (<a href="#ppat.1007190.s001">S1 Fig</a>) confirming that the two viruses are dependent on SCARB2.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007190.g002"><span class="label" tagx="label" title="label">Fig 2</span></a></div>   <div class="title" tagx="title" title="title">
SCARB2 is necessary for effective infection by EV71-VP1<sub>97R167G</sub> and EV71-VP1<sub>97L167E</sub>.</div> <p><b>A)</b> Mean (N = 2) viral titers (±SEM) expressed as log TCID<sub>50</sub>/ml (median tissue culture infective dose) obtained 5 dpi in cells expressing (control) or not (ΔSCARB2) SCARB2. Wells with less than 50 infected cells were considered as negative. <b>B)</b> Western blot showing SCARB2 expression in ΔSCARB2 cells and control cells. ****P&amp;lt;0.0001, **P&amp;lt;0.01, *P&amp;lt;0.05.</p>   </div> </div> <div class="differentialcelltropismofev71-vp197r167gandev71-vp197l167eisrelatedtodifferentialuseoftheheparansulfateattachmentreceptor" title="sec"> <div class="title" tagx="title" title="title">
Differential cell tropism of EV71-VP1<sub>97R167G</sub> and EV71-VP1<sub>97L167E</sub> is related to differential use of the heparan sulfate attachment receptor</div> <p>We then investigated the implication of another ubiquitously expressed receptor, the HS attachment receptor, in this observed differential cell tropism. We measured the amount of EV71-VP1<sub>97R167G</sub> and EV71-VP1<sub>97L167E</sub> viruses bound to untreated cells or cells pretreated with heparinase III (<a href="#ppat.1007190.g003">Fig 3A</a>). EV71-VP1<sub>97R167G</sub> exhibited a binding advantage over EV71-VP1<sub>97L167E</sub> in RD, Vero and SH-SY5Y cells but not in Caco-2 cells. This advantage was suppressed after cleavage of HS from the cell surface (<a href="#ppat.1007190.g003">Fig 3A</a>, left panel). In contrast, EV71-VP1<sub>97L167E</sub> did not show a significant difference in any cell line (<a href="#ppat.1007190.g003">Fig 3A</a>, right panel).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007190.g003"><span class="label" tagx="label" title="label">Fig 3</span></a></div>   <div class="title" tagx="title" title="title">
Differential cell tropism of EV71-VP1<sub>97R167G</sub> and EV71-VP1<sub>97L167E</sub> is linked to HS binding and correlates with its expression level at the cell surface.</div> <p><b>(A).</b> Binding of EV71-VP1<sub>97R167G</sub> (left panel) and EV71-VP1<sub>97L167E</sub> (right panel) in the presence or absence of HS. Binding <b>(B)</b> and replication <b>(C)</b> of a viral stock composed of equivalent amounts of EV71-VP1<sub>97R167G</sub> and EV71-VP1<sub>97L167E</sub> in different cell lines and under conditions preventing attachment to HS. Viral loads were measured by RT-qPCR (real-time quantitative reverse transcription PCR) in whole cell extracts for binding assay and both cell extracts and supernatants for replication assay, and expressed as the mean (±SEM, N≥2) relative to the EV71-VP1<sub>97R167G</sub> load in non-treated RD cells (100%). <b>(D).</b> Quantification of HS-expressing (positive) cells was achieved through an analysis using MetaMorph software (left panel) based on the immunofluorescence labeling (right panel) of HS attachment receptors at the surface of Vero, RD, SH-SY5Y cells and Caco-2 cells. More than 500 cells were included in the analysis. Scale bar = 20 μm. <b>(E)</b> Binding of EV71-VP1<sub>97R/L167G/E</sub> to heparin sepharose beads. Viral population present in the input, flow-through and elute population was characterized by sequencing. ****P&amp;lt;0.0001, ***P&amp;lt;0.001, **P&amp;lt;0.01, *P&amp;lt;0.05. For B, C and E, the frequency plot [<a href="http://weblogo.berkeley.edu/">http://weblogo.berkeley.edu/</a>] of codons encoding VP1 aa 97, 98 and 167 are shown and are representative of biological replicates (N≥2). Arginine (R) at position 97 is encoded by CGC, Leucine (L) by CTC; Glutamate (E) at position 98 is encoded by GAA, Lysine (K) by AAA; Glycine (G) at position 167 is encoded by GGG, Glutamate (E) by CTC.</p>   </div> <p>To confirm this observation, competition experiments were performed with a viral stock (EV71-VP1<sub>97R/L-167G/E</sub>) containing equivalent amounts of each derivative. The competition was run under 3 conditions: using untreated cells, using cells having surface HS digested with heparinase, or using a viral population that was pretreated with heparin, a soluble HS analogue. Both binding (1 hour post-infection [hpi]) and replication (24 hpi) efficiencies were monitored by real time-quantitative polymerase chain reaction (RT-qPCR), and the dominant species was characterized by Sanger sequencing for each binding (<a href="#ppat.1007190.g003">Fig 3B</a>) and replication condition (<a href="#ppat.1007190.g003">Fig 3C</a>). The overall binding was lower for the mixed population than for the EV71-VP1<sub>97R167G</sub> stock (<a href="#ppat.1007190.g003">Fig 3B</a> versus <a href="#ppat.1007190.g003">3A</a>). Accordingly, the pretreatment of cells with heparinase reduced viral binding less significantly, while viral replication was affected in SH-SY5Y, Vero and Caco-2 cells. Sanger sequencing of the entire VP1 region after binding showed that in Vero, SH-SY5Y and RD cells, EV71-VP1<sub>97R167G</sub> (with R at position 97 encoded by CGC and G at position 167 encoded by GGG) was the dominant variant in non-treated conditions, which was not the case in Caco-2 cells, where EV71-VP1<sub>97L167E</sub> (with L at position 97 encoded by CTC and E at position 167 encoded by GAG) and EV71-VP1<sub>97L167G</sub> were dominant (<a href="#ppat.1007190.g003">Fig 3C</a>). Upon heparinase treatment, the dominant consensus sequence shifted towards EV71-VP1<sub>97R/L167G/E</sub>, EV71-VP1<sub>97L167G/E</sub> or EV71-VP1<sub>97L167E</sub> in binding and replication assays. Notably, viral species with a positive charge at VP1 position 98 (K encoded by GAA instead of E encoded by AAA) emerged after treatment of SH-SY5Y, RD and Caco-2 cells with heparinase (<a href="#ppat.1007190.g003">Fig 3C and 3D</a>). Pre-incubation of viruses with heparin also resulted, although in a less stringent manner, in a shift of the dominant population from EV71-VP1<sub>97R167G</sub> to a mixed EV71-VP1<sub>97R/L167G/E</sub> population, except after replication in SH-SY5Y and RD cells.</p> <p>To determine if the improved binding and replication of EV71-VP1<sub>97R167G</sub> in cells correlated with HS expression levels, quantification of HS expression was assessed by immunofluorescence (<a href="#ppat.1007190.g003">Fig 3D and 3E</a>). SH-SY5Y, RD and Vero cell lines strongly express HS, while Caco-2 showed low HS expression.</p> <p>A recent publication showed that variants with VP1 E167G do not bind heparin [<a href="#ppat.1007190.ref029">29</a>], and the differential binding and replication of EV71-VP1<sub>97R167G</sub> confirmed our previous observations with EV71-VP1<sub>97R167E</sub> in SH-SY5Y and Vero cells [<a href="#ppat.1007190.ref039">39</a>], indicating that the acquired HS binding ability is due only to the L97R substitution.</p> <p>Finally, we confirmed the different affinity of EV71-VP1<sub>97R167G</sub> and EV71-VP1<sub>97L167E</sub> for HS using a heparin sepharose binding assay. After incubation of a mixed viral stock (EV71-VP1<sub>97R/L-167G/E</sub>) composed of 50% of each derivative with sepharose beads, we sequenced the viral populations present in the non-heparin-binding population (flow-through) or in the heparin-binding population (eluate) (<a href="#ppat.1007190.g003">Fig 3E</a>). As expected, EV71-VP1<sub>97L167E</sub> was highlighted in the non-heparin-binding population, while EV71-VP1<sub>97R167G</sub> was detected in the heparin-binding population.</p> </div> <div class="differentialtissuetropismofev71-vp197r167gandev71-vp197l167evariantsisrelatedtodifferentialuseoftheheparansulfateattachmentreceptor" title="sec"> <div class="title" tagx="title" title="title">
Differential tissue tropism of EV71-VP1<sub>97R167G</sub> and EV71-VP1<sub>97L167E</sub> variants is related to differential use of the heparan sulfate attachment receptor</div> <p>To better replicate the viral tropism observed in the patient, infection assays were performed with EV71-VP1<sub>97R167G</sub>, EV71-VP1<sub>97L167E</sub> and EV71-VP1<sub>97R/L167G/E</sub> in complex tissue models, <i>in vitro</i> reconstituted from human primary cells, that mimic the upper respiratory tract, small intestine and neural tissues composed of neurons and glial cells.</p> <div class="respiratorytissues" title="sec"> <div class="title" tagx="title" title="title">
Respiratory tissues</div> <p>As only viruses with VP1 97L were detected in the patient bronchoalveolar lavage, the replication of EV71-VP1<sub>97R167G</sub>, EV71-VP1<sub>97L167E</sub> and EV71-VP1<sub>97R/L167G/E</sub> was compared in 3D reconstituted human upper (<a href="#ppat.1007190.g004">Fig 4A</a> MucilAir) and lower (<a href="#ppat.1007190.g004">Fig 4B</a> SmallAir) airway epithelia cultured at an air-liquid interface. These tissues faithfully reproduce the pseudostratified architecture of the upper and lower airway epithelia composed of ciliated cells, mucus secreting cells (as well as club cells in the lower airway) and basal cells (<a href="#ppat.1007190.g004">Fig 4C and 4D</a>, upper panels) [<a href="#ppat.1007190.ref040">40</a>, <a href="#ppat.1007190.ref041">41</a>]. Viral suspensions were applied apically (air exposed side) or basally (liquid exposed side) for 4 hours before removal of unbound viruses by extensive washing. Each day afterwards, apically released viruses were collected and basal medium was changed. Viruses released apically at day 5 pi were quantified by RT-qPCR (<a href="#ppat.1007190.g004">Fig 4A and 4B</a>). Overall replication was low in both tissue types compared to typical respiratory EVs, such as EV-D68 or rhinoviruses [<a href="#ppat.1007190.ref040">40</a>, <a href="#ppat.1007190.ref042">42</a>], however the infectivity of viruses collected at day 5 pi was confirmed by productive infection in RD cells. No significant difference was observed in the level of replication of EV71-VP1<sub>97R167G</sub> and EV71-VP1<sub>97L167E</sub> after apical inoculation of MucilAir (<a href="#ppat.1007190.g004">Fig 4A</a>) or SmallAir (<a href="#ppat.1007190.g004">Fig 4B</a>). Nevertheless, sequencing of the viral population collected at 5 dpi highlighted that, when inoculated together, EV71-VP1<sub>97L167E</sub> outcompeted EV71-VP1<sub>97R167G</sub> (<a href="#ppat.1007190.g004">Fig 4A and 4B</a>). In contrast, after basal infection, EV71-VP1<sub>97R167G</sub> showed a net replication advantage in both individual and competition replication assays.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007190.g004"><span class="label" tagx="label" title="label">Fig 4</span></a></div>   <p><b>Replication of EV71-VP1</b><sub><b>97R167G</b></sub><b>, EV71-VP1</b><sub><b>97L167E</b></sub><b>and EV71-VP1</b><sub><b>97R/L167G/E</b></sub><b>in reconstituted airway epithelia developed from the upper (A) and lower (B) respiratory tract and HS/SCARB2 expression in the same tissues (C -D)</b>. <b>(A-B)</b> Viral loads were measured by RT-qPCR 5 dpi in the apically collected washes of the apically or basally inoculated tissues and are expressed as the mean ±SEM (N≥2). ***P&amp;lt;0.001, **P&amp;lt;0.01, *P&amp;lt;0.05. Frequency plots are shown on the right (see <a href="#ppat.1007190.g003">Fig 3</a> for details) <b>(C-D)</b> Tissue sections were colored with hematoxylin and eosin (upper panel) or labeled by immunofluorescence. SCARB2 is stained in green, HS in red and cell nuclei in blue. Scale bar = 20 μm.</p>   </div> <p>These results correlate with high expression of HS at the basal tissue side and notably low HS expression at the apical side (<a href="#ppat.1007190.g004">Fig 4C and 4D</a>). Interestingly, the expression of SCARB2 showed the opposite pattern with high expression at the apical tissue side and low expression at the basal side (<a href="#ppat.1007190.g004">Fig 4C and 4D</a>).</p> </div> <div class="intestinaltissues" title="sec"> <div class="title" tagx="title" title="title">
Intestinal tissues</div> <p>As a mixed population of viruses with VP1 97L and VP1 97R was highlighted in the patient’s stool, the experimental approach used for respiratory tissues was applied to assess the replication abilities of EV71-VP1<sub>97R167G</sub> and EV71-VP1<sub>97L167E</sub> in 3D reconstituted human small intestine tissues (EpiIntestinal, Mattek) cultured at the air-liquid interface (<a href="#ppat.1007190.g005">Fig 5</a>). In this tissue culture model, that reproduces the cellular composition of the small intestine with enterocytes, paneth cells, M cells, tuft cells and intestinal stem cells, viral loads were more than 2 log higher than those measured in respiratory tissues (<a href="#ppat.1007190.g005">Fig 5A</a> versus <a href="#ppat.1007190.g004">Fig 4A and 4B</a>). For small intestine tissues as for respiratory tissues, there was no significant difference between the replication of EV71-VP1<sub>97R167G</sub> and EV71-VP1<sub>97L167E</sub> 5 dpi for the apical inoculation (<a href="#ppat.1007190.g005">Fig 5A</a>), while for the basal inoculation, EV71-VP1<sub>97R167G</sub> replicated significantly better than EV71-VP1<sub>97L167E</sub> and systematically outcompeted the latter variant in competition. However, in contrast to observations made in respiratory tissues, EV71-VP1<sub>97L167E</sub> did not outcompete EV71-VP1<sub>97R167G</sub> after the apical infection, and both populations were present 5 dpi in a mixed viral population. These data are in agreement with the expression of HS in the tissue, with the attachment receptor being expressed both at the apical and basal side of the tissue, although with higher concentration at the basal tissue side (<a href="#ppat.1007190.g005">Fig 5B and 5C</a>). Again, for these tissues as for respiratory tissues, SCARB2 expression is more abundant at the apical tissue side (<a href="#ppat.1007190.g005">Fig 5B</a>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007190.g005"><span class="label" tagx="label" title="label">Fig 5</span></a></div>   <p><b>Replication of EV71-VP1</b><sub><b>97R167G</b></sub><b>, EV71-VP1</b><sub><b>97L167E</b></sub><b>and EV71-VP1</b><sub><b>97R/L167G/E</b></sub><b>in reconstituted small intestine tissues (A) and HS/SCARB2 expression in the same tissues (B and C)</b>. <b>(A)</b> Viral loads were measured by RT-qPCR 5 dpi in the apically collected washes of the apically or basally inoculated tissues and are expressed as the mean (±SEM) N = 2. **P&amp;lt;0.01, *P&amp;lt;0.05. The frequency plots are shown on the right (see <a href="#ppat.1007190.g003">Fig 3</a> for details) <b>(B)</b> Tissue sections were colored with hematoxylin and eosin (upper panel) or labeled by immunofluorescence. SCARB2 is stained in green, HS in red and cell nuclei in blue. Scale bar = 20 μm. <b>(C)</b> Quantification of HS expression at the apical and basal surface of intestinal tissues was achieved through an analysis using MetaMorph software based on the immunofluorescence labeling of HS attachment receptor in sectioned tissues.</p>   </div> </div> <div class="2dand3dengineeredneuralcultures" title="sec"> <div class="title" tagx="title" title="title">
2D and 3D engineered neural cultures</div> <p>Since only EV71 with VP1 97R was sequenced from the patient’s CSF, the replication of the different variants was also compared in engineered 2D (<a href="#ppat.1007190.g006">Fig 6A</a>) and 3D (<a href="#ppat.1007190.g006">Fig 6B</a>) neural tissues containing a mixed population of mature, functional neurons and glial cells. Viral loads were measured in 2D culture lysates at 1 hpi and 5 dpi and in 3D tissue lysates at 4 hpi and 5 dpi. EV71-VP1<sub>97R167G</sub>, EV71-VP1<sub>97L167E</sub> and EV71-VP1<sub>97R/L167G/E</sub> replicated efficiently in both types of cultures and without significant difference 5 dpi. Notably, EV71-VP1<sub>97R167G</sub> viral loads were significantly higher 4 hpi in 3D tissues. Upon infection with a mixed population, this variant was also dominant at 1 hpi and 4 hpi in 2D and 3D tissues, respectively (<a href="#ppat.1007190.g006">Fig 6A and 6B</a>), as well as at 1 dpi in 2D cultures (<a href="#ppat.1007190.g007">Fig 7B</a>). However, the replication fitness of EV71-VP1<sub>97L167E</sub> was greater in competition experiments at 5 dpi. HS expression is abundant in both models, supporting the selection of EV71-VP1<sub>97R167G</sub> at early time points but not the dominance of EV71-VP1<sub>97L167E</sub> 5 dpi (<a href="#ppat.1007190.g006">Fig 6C</a>). Of note both HS (<a href="#ppat.1007190.s002">S2A and S2B Fig</a>) and SCARB2 (<a href="#ppat.1007190.s002">S2C and S2D Fig</a>) are highly expressed in neurons and glial cells.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007190.g006"><span class="label" tagx="label" title="label">Fig 6</span></a></div>   <p><b>Replication of EV71-VP1</b><sub><b>97R167G</b></sub><b>, EV71-VP1</b><sub><b>97L167E</b></sub><b>and EV71-VP197</b><sub><b>R/L167G/E</b></sub><b>in 2D (A) and 3D (B) neural tissues and HS/SCARB2 expression in the same 2D tissues (C). (A-B)</b> Viral load was measured by RT-qPCR and expressed as the mean (±SEM). (N = 2). ****P&amp;lt;0.0001, <sup>++</sup>P&amp;lt;0.01 or <sup>#</sup>P&amp;lt;0.05. (<sup>+</sup>relative to EV71-VP1<sub>97L167E</sub> and <sup>#</sup> relative to EV71-VP1<sub>97R/L167G/E</sub> at the same time point). The frequency plots are shown on the right (see <a href="#ppat.1007190.g003">Fig 3</a> for details) <b>(C)</b> 2D neural cultures were labeled by immunofluorescence. SCARB2 is stained in green, HS in red and cell nuclei in blue. Scale bar = 20 μm.</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007190.g007"><span class="label" tagx="label" title="label">Fig 7</span></a></div>   <p><b>Viral infectivity of EV71-VP1</b><sub><b>97R167G</b></sub><b>and EV71-VP1</b><sub><b>97L167E</b></sub><b>in Vero cells (A) and of EV71-VP1</b><sub><b>97R167G</b></sub><b>, EV71-VP1</b><sub><b>97L167E</b></sub><b>and EV71-VP1</b><sub><b>97L/R167E/G</b></sub><b>in neural cells (B) in the presence of increasing doses of ı-carrageenan.</b> The percent of infection was measured by an immunocytochemistry assay in Vero cells (A) and by RT-qPCR on cell lysate 1 dpi in neural cells (B) and expressed as the mean (±SEM) compared to non-treated viruses. (N = 2). **P&amp;lt;0.005. The frequency plots are shown on the right (see <a href="#ppat.1007190.g003">Fig 3</a> for details).</p>   </div> </div> </div> <div class="iota-carrageenanspecificallyinhibitsinfectionbyev71-vp197r167g" title="sec"> <div class="title" tagx="title" title="title">
Iota-carrageenan specifically inhibits infection by EV71-VP1<sub>97R167G</sub></div> <p>Because of the acquired ability of EV71-VP1<sub>97R167G</sub> to bind HS, its sensitivity to iota-carrageenan (ı-carrageenan), a highly sulfated polysaccharide, was investigated and compared to the sensitivity of EV71-VP1<sub>97L167E</sub> in Vero and 2D neural cultures. As expected, EV71-VP1<sub>97R167G</sub> infection was strongly inhibited by ı-carrageenan in Vero cells (IC<sub>50</sub> = 17.8 μg/ml), whereas EV71-VP1<sub>97L167E</sub> infection was only slightly affected by the treatment (IC<sub>50</sub>&amp;gt;200 μg/ml) (<a href="#ppat.1007190.g007">Fig 7A</a>). Similar results were obtained in neural cultures, where the infection of EV71-VP1<sub>97R167G</sub> and EV71-VP1<sub>97R/L167G/E</sub> was inhibited up to 90% in a dose-dependent manner. EV71-VP1<sub>97L167E</sub> infectivity was also inhibited but to a lesser extent (up to 50%). Treatment of the mixed viral population altered the dominant sequence from EV71-VP1<sub>97R167G</sub> at 1 dpi towards EV71-VP1<sub>97R/L167G/E</sub> (<a href="#ppat.1007190.g007">Fig 7B</a>).</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>In 2012, we reported an EV71 genogroup C1 disseminated infection in an immunocompromised patient treated with rituximab who was hospitalized with respiratory and neurological symptoms [<a href="#ppat.1007190.ref039">39</a>]. Although EV71 genogroup C1 infections have not been associated with severe diseases in the past, several complicated cases have been recently reported in Europe, underscoring the importance of defining EV71 virulence determinants [<a href="#ppat.1007190.ref043">43</a>, <a href="#ppat.1007190.ref044">44</a>]. By sequencing the full genome of the virus from different samples, we identified the acquisition of a non-conservative mutation in VP1 (L97R), which was absent at the beginning of the infection in the respiratory tract but was present in a mixed population in the gastrointestinal tract and became dominant in the blood and CSF at later time points. We showed that this mutation conferred a binding advantage to the virus in certain cell lines. In this study, we extended these findings and showed that the VP1 L97R substitution, as well as an associated mutation (VP1 E167G) observed after <i>in vitro</i> culturing, confers an ability of the virus to use HS as an attachment receptor while it does not alter the dependence of the virus on the ubiquitously expressed SCARB2 entry receptor. Tan and colleagues recently showed that variants with VP1 E167G do not bind heparin [<a href="#ppat.1007190.ref029">29</a>]. In addition, in our previous publication, we observed the same binding advantage of the VP1<sub>97R</sub> mutant in neuroblastoma and Vero cells that was observed in this study with the VP1<sub>97R167G</sub> mutant [<a href="#ppat.1007190.ref039">39</a>]. This strongly suggests that the VP1 L97R mutation confers an HS binding ability, while the VP1 E167G mutation has a stabilizing function as proposed previously based on VP1 3D structure [<a href="#ppat.1007190.ref039">39</a>]. Interestingly, after competition between EV71-VP1<sub>97L167E</sub> and EV71-VP1<sub>97R167G</sub> for binding and replication in cells pretreated with heparinase, EV71-VP1<sub>97R</sub> was excluded from the viral population, while a new variant (EV71-VP1<sub>98K</sub>) which is also known to promote HS binding [<a href="#ppat.1007190.ref029">29</a>], emerged. Thus, it is very likely that after cleavage with heparinase at the 1→4 linkages between hexosamine and glucuronic acid residues, HS is still bound by EV71-VP1<sub>98K</sub> but not by EV71-VP1<sub>97R</sub>, suggesting that these two variants present different HS binding specificities.</p> <p>To assess EV71 replication <i>in vivo</i> in the gastrointestinal and respiratory tracts and in the CNS, we analyzed the replication of HS-dependent (EV71-VP1<sub>97R167G</sub>) and independent (EV71-VP1<sub>97L167E</sub>) variants in relevant tissue culture models that mimicked the upper and lower respiratory airway epithelia, intestinal and neural tissues. To reproduce natural infection routes, we inoculated intestinal and respiratory tissue cultures that grow at an air-liquid interface from the apical (corresponding to the lumen) or basal (corresponding to the blood-borne) tissue sides. A comparison of viral growth of HS-dependent and HS-independent variants and competition experiments highlighted that HS-dependent variants had improved access to respiratory and intestinal tissues from the basal tissue side, where HS is highly expressed, while HS-independent variants show improved replication after infection from the air-exposed side of respiratory tissues, where HS is only slightly present, in line with previous publications [<a href="#ppat.1007190.ref045">45</a>, <a href="#ppat.1007190.ref046">46</a>]. Infection from the luminal side of intestinal tissues sustained efficient replication of both variants and correlated with intermediate expression of HS at this side. We also highlighted high expression of SCARB2 at the apical side of respiratory and intestinal tissues and low expression at the basal side. HS may thus be necessary in tissues with low SCARB2 expression to concentrate the virus at the cell surface and promote subsequent interaction with SCARB2. This would explain why HS-dependent variants infect the basal side of respiratory and intestinal tissues better than HS-independent variants.</p> <p>Our findings in respiratory and intestinal tissues correlate with the viral populations isolated from the different specimens of the patient. We only observed an HS-independent variant in the bronchoalveolar lavage of the patient, a sample collected from the luminal side of the respiratory tissue. Importantly, sequencing of EV71 that was detected in bronchoalveolar lavages from two other immunosuppressed patients who presented neurological complications in Switzerland also identified viruses from the same genogroup and with the same VP1 sequence (GenBank accession numbers MH256663 and MH256664). Of note in these 2 cases as in our case, the infecting virus had a glutamate at position 145, a residue known to confer poor HS binding ability [<a href="#ppat.1007190.ref029">29</a>]. This suggests that HS-independent variants may be preferentially transmitted via the respiratory route.</p> <p>We observed a mixed viral population in the stool of the patient, a sample that reflects viral population isolated from the luminal side of the gastrointestinal tract. A mixed viral population was also obtained after apical inoculation of intestinal tissue cultures with a viral population composed of EV71-VP1<sub>97L167E</sub> and EV71-VP1<sub>97R167G</sub>. Since both <i>ex vivo</i> and <i>in vivo</i> intestinal tissues are the favored site of EV71 replication, and since these tissues express HS at the basal and apical surface, it is very likely that the emergence of an HS-dependent virus arose at this site. Whether this site was reached after replication in the respiratory tract and propagation via shedding in oral secretions and swallowing or whether infection occurred simultaneously in the two sites remains an open question.</p> <p>Concerning neurotropism, the viral population sequenced from the CSF of the patient was HS-dependent while results obtained in 2D and 3D neural cultures were more intricate. In single infections, HS-dependent and HS-independent variants revealed similar neurotropic potential as both showed high replication levels in primary neural cells without significant difference 5 dpi. However, in competition experiments, HS-independent variants outcompeted HS-dependent variants 5 dpi while the opposite occurred at earlier time points (1 hpi and 1 dpi in 2D and 4 hpi in 3D cultures). The higher amount of cell-associated HS-dependent viruses shortly after inoculation is easily explicable by the high expression of HS in neural tissues. However, the fact that the improved binding is not reflected by an improved replicative fitness 5 dpi is less obvious. A similar situation was observed in Vero cells where the binding advantage of EV71-VP1<sub>97R167G</sub> did not result in improved replication relative to EV71-VP1<sub>97L167E</sub>. Whether another step of the viral life is affected by the VP1 L97R mutation in Vero, 2D and 3D neural cultures is currently under investigation. Of note in this study and in our previous publication [<a href="#ppat.1007190.ref039">39</a>], the HS-dependent variant was fitter in SH-SY5Y neuroblastoma cells at any time post infection. This difference between primary neural cultures and immortalized cells may rely on the nature of these culture models (cancer neuron precursors for neuroblastoma cells versus mature neurons and glial cells for both 2D and 3D neural cultures) and emphasizes the need to use relevant primary tissue culture models to study viral pathogenesis.</p> <p>Two explanations may account for the detection of HS-dependent virus only in the patient’s CSF and our experimental observations that the HS-independent virus outcompetes the HS-dependent virus in neural tissues 5 dpi (thought both variants replicate to high level in independent infections). First, the patient’s CSF may have been sampled shortly after neural invasion, at a time where the HS-dependent variant was also fitter in neural cultures. However, this would imply that both variants were present together in the CNS where the HS-dependent variants outcompeted the HS-independent variants. The fact that only HS-dependent variants were found in the patient’s blood does not support this hypothesis. More probably, the ability to bind HS promoted EV71-VP1<sub>97R167G</sub> replication in HS-enriched tissues (such as the basal layers of the gastrointestinal mucosa, muscle or endothelial cells), resulting in sustained viremia and subsequent dissemination to neural tissues. In this context, EV71 neurotropism may not rely on an improved growth in neural tissues but rather on an improved dissemination ability selected upstream of CNS contamination over the course of an infection.</p> <p>The implication of HS binding in dissemination and neurotropism has been investigated for several viruses and remains controversial. Studies on aphtoviruses [<a href="#ppat.1007190.ref047">47</a>], flaviviruses [<a href="#ppat.1007190.ref048">48</a>], togaviruses [<a href="#ppat.1007190.ref049">49</a>], alphaviruses [<a href="#ppat.1007190.ref050">50</a>] and even enteroviruses such as coxsackievirus B3 [<a href="#ppat.1007190.ref051">51</a>] suggest that binding to HS may lower viremia due to virus trapping in tissues expressing high HS levels, while it may improve neurotropism and ability to cross the blood-brain barrier [<a href="#ppat.1007190.ref052">52</a>]. Of note, two recent papers reported that infection by EV71 with VP1 E145Q substitution was associated with an attenuated phenotype in mice and cynomolgus monkeys and that this attenuation was linked to the ability of VP1 145Q to bind HS [<a href="#ppat.1007190.ref031">31</a>, <a href="#ppat.1007190.ref034">34</a>]. The authors conclude that HS binding and <i>in vivo</i> virulence are negatively correlated due to trapping and abortive infection of HS-dependent viruses in tissues expressing high levels of HS but low levels of SCARB2. In the clinical case investigated here, the HS binding variant was the only variant sequenced from the blood of the patient suggesting that HS binding did not prevent viremia and dissemination but rather promoted it. Several hypotheses could explain these different observations: first, mice and monkeys may not faithfully reproduce EV71 infection in human as suggested by studies showing that isolates with VP1 145G/Q are frequently associated with severe neurological disease in humans [<a href="#ppat.1007190.ref035">35</a>–<a href="#ppat.1007190.ref038">38</a>]; second, the inoculation site may modulate the infection outcome. Huang and colleagues [<a href="#ppat.1007190.ref053">53</a>] showed that broad mutant spectra with divergent mutations are observed at the initial infection sites (the respiratory and digestive systems) and that a selection bottleneck occurs afterwards with subsequent enrichment of advantageous mutations in the viral population. Intraperitoneal or intravenous injections may thus prevent the natural evolution of the viral population observed after fecal-oral or respiratory transmission; third, the binding affinity for HS may differ according to the number and/or location of positive charges on the viral capsid, and different mutations (VP1 E145Q versus VP1 L97R) may result in different binding intensities and different trapping strengths; finally, the host immunity certainly plays a key role. In the clinical case presented here, the patient underwent immunosuppressive therapy with rituximab, an anti-CD20 antibody that depletes the peripheral B�?cell pool. Seroneutralisation experiments performed with the patients’ serum at a time when the VP1 97R mutation was already present in the plasma failed to highlight the presence of an antibody-mediated selective immune pressure against HS-dependent and HS-independent variants [<a href="#ppat.1007190.ref039">39</a>]. This absence of neutralizing antibodies was correlated with a negative complement fixation assay confirming a poor antibody response against enterovirus linked to the immunosuppressed state of the patient. Fujii et al showed that viruses with VP1 145E are more resistant to neutralizing antibodies than viruses with VP1 145G [<a href="#ppat.1007190.ref031">31</a>]. The absence of immune pressure may thus be a prerequisite for the emergence of HS-dependent variants <i>in vivo</i>, since critical residues, such as VP1 98 and 145, seem to play an important role both in immune escape [<a href="#ppat.1007190.ref030">30</a>, <a href="#ppat.1007190.ref031">31</a>] and HS binding [<a href="#ppat.1007190.ref029">29</a>, <a href="#ppat.1007190.ref034">34</a>, <a href="#ppat.1007190.ref054">54</a>, <a href="#ppat.1007190.ref055">55</a>]. The emergence of HS-dependent mutants and their dissemination could thus be favored in the presence of a poor antibody response. Additional clinical investigations are needed to further assess the implication of HS binding in dissemination and disease severity in humans.</p> <p>The recent emergence of severe cases associated with EV71 genogroup C1 infections and the lack of an efficient vaccine or antiviral to fight EV71 infections [<a href="#ppat.1007190.ref056">56</a>, <a href="#ppat.1007190.ref057">57</a>] highlights an important need for the development of effective antiviral strategies [<a href="#ppat.1007190.ref058">58</a>]. In this study, we demonstrated the key role of HS attachment receptor binding in EV71 dissemination in an immunocompromised host, and our preliminary results in cells and neural cultures highlighted the inhibition of HS-dependent variants by soluble HS mimetic compounds. Soluble heparan sulfate analogs may thus be a useful tool to limit enterovirus replication and dissemination. In conclusion, our data may help to unravel the key determinants of EV71 dissemination and neurotropism and bridge the gap between EV71 neurological diseases and the lack of a therapeutic approach.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="ethicsstatement" title="sec"> <div class="title" tagx="title" title="title">
Ethics statement</div> <p>Respiratory (<a href="http://www.epithelix.com/products/mucilair">http://www.epithelix.com/products/mucilair</a>) and EpiIntestinal (<a href="https://www.mattek.com/products/epiintestinal/">https://www.mattek.com/products/epiintestinal/</a> tissues) were ordered from Epithelix and MatTek biotechnology companies. There, the tissues are developed from anonymized samples and after Ethical approval. The study was conducted according to the Declaration of Helsinki on biomedical research (Hong Kong amendment, 1989), and the research protocol was approved by our local ethics committee.</p> </div> <div class="cells" title="sec"> <div class="title" tagx="title" title="title">
Cells</div> <p>RD (human rhabdomyosarcoma, ATCC CCL-136) and Vero (monkey kidney, ATCC CRL-1587) cells were cultured in Dulbecco's Modified Eagle Medium (DMEM) and GlutaMAX (31966021, Gibco, Thermo Fisher Scientific, Switzerland) supplemented with 10% (v/v) fetal bovine serum (FBS) (P40-37500, Pan Biotech, Chemie Brunschwig, Switzerland) and 100 μg/ ml of penicillin and streptomycin (15140–122, Gibco, Thermo Fisher Scientific, Switzerland). Caco-2 (human colorectal adenocarcinoma, ATCC HTB 37) and SH-SY5Y cells (human neuroblastoma, ATCC CRL- 2266) were grown in Eagle’s minimum essential medium (EMEM) (BE12-125F, Lonza, Switzerland) and DMEM/F12 medium (31331093, Gibco, Thermo Fisher Scientific, Switzerland), respectively, supplemented with 2 mM L-glutamine (25030–024, Gibco, Thermo Fisher Scientific, Switzerland), 1% non-essential amino acids (11140–035, Gibco, Thermo Fisher Scientific, Switzerland) and 10% (v/v) FBS. HEK 293T/17 (human kidney, ATC C#CRL-11268) cells were grown in DMEM (41965039, Gibco, Thermo Fisher Scientific, Switzerland) supplemented with 10% (v/v) fetal bovine serum (FBS), 100 μg/ ml of penicillin and streptomycin, 2 mM l-glutamine, 1 mM sodium pyruvate (11360–039, Gibco, Thermo Fisher Scientific, Switzerland), and 1% non-essential amino acids.</p> <p>Cultures were maintained at 37°C in a 5% CO<sub>2</sub> atmosphere. Infection medium was similar but with reduced FBS concentrations (2.5% for Vero and RD cells and 5% for Caco-2 and SH-SY5Y cells).</p> </div> <div class="viralstocks" title="sec"> <div class="title" tagx="title" title="title">
Viral stocks</div> <div class="ev71-vp197l167eandev71-vp197r167g" title="sec"> <div class="title" tagx="title" title="title">
EV71-VP1<sub>97L167E</sub> and EV71-VP1<sub>97R167G</sub></div> <p>The pClVP1<sub>97L167E</sub> and pClVP1<sub>97R167E</sub> infectious clones were ordered from Biomatik (Ontario, Canada) as previously described [<a href="#ppat.1007190.ref039">39</a>]. The VP1 E167G mutation was added by site directed mutagenesis as previously described [<a href="#ppat.1007190.ref059">59</a>] using 5’GCCAGATTCCAGAGGGTCTCTCGCATGGC3’ forward and 5’GCCATGCGAGAGACCCTCTGGAATCTGGC3’ reverse primers. Constructs were checked by sequencing, linearized with <i>Bam</i>HI, <i>in vitro</i> transcribed and transfected into Vero or Caco-2 cells, as previously described [<a href="#ppat.1007190.ref039">39</a>].</p> <p>Seven days after transfection, supernatants and cells were collected, subjected to 3 freeze-thaw cycles, clarified by centrifugation (5 min at 1000 rpm) and re-passaged 3 times in 80% confluent Vero cells. Alternatively, confluent Vero cells were inoculated with supernatant from infected Caco-2 cells. For the last passage, and when cytopathic effect (CPE) was strong enough, the infection medium was changed 24 h before collection to obtain only freshly produced viruses. Supernatants were collected, clarified by centrifugation (5 min at 1000 rpm) and concentrated by ultracentrifugation as previously described [<a href="#ppat.1007190.ref030">30</a>]. Ultracentrifuged viral stocks were aliquoted and stored at -80°C. VP1 was PCR amplified and sequenced to confirm the absence of unexpected mutations (see below).</p> </div> <div class="ev71-vp197r/l167g/e" title="sec"> <div class="title" tagx="title" title="title">
EV71-VP1<sub>97R/L167G/E</sub></div> <p>Two viral stocks that contained equimolar amounts of EV71-VP1<sub>97R167G</sub> and EV71-VP1<sub>97L167E</sub> were used in this study. The first stock was naturally generated by the emergence of spontaneous mutations during viral stock preparation. The presence of equimolar amounts of EV71-VP1<sub>97R167G</sub> and EV71-VP1<sub>97L167E</sub> was assessed by Sanger sequencing and was confirmed by subcloning (pCR2.1-TOPO cloning kit, 45–0641, Invitrogen, Thermo Fisher Scientific, Switzerland) and sequencing of VP1 from individual clones. The second stock was created by mixing equimolar amounts of EV71-VP1<sub>97R167G</sub> and EV71-VP1<sub>97L167E</sub>. Again, equal representation of the two derivatives was assessed by sequencing with the same primers. Both stocks were used in <i>ex vivo</i> experiments (in MucilAir, SmallAir, EpiIntestinal and neural tissues) while the naturally generated stock was used for binding and infection assays <i>in vitro</i>.</p> <p>All viral stocks were quantified by RT-qPCR and titration, according to the Reed and Munch method [<a href="#ppat.1007190.ref060">60</a>] and were passaged less than 3 times to exclude any bias linked to cell adaptation.</p> </div> </div> <div class="heparinaseassay" title="sec"> <div class="title" tagx="title" title="title">
Heparinase assay</div> <p>Confluent Vero, Caco-2, RD or SH-SY5Y cells pre-plated in a 96 well plate, were washed twice with heparinase III digestion buffer (0.1 M sodium acetate pH 7.0, 1 mM calcium acetate and 0.2% BSA) and incubated for 1 h at 37°C with 3.5 mIU/ml of heparinase III (AMS.HEP_ENZ III_S, Amsbio, Switzerland) (50 μl/well). Control samples were incubated with digestion buffer alone. Heparinase III efficient cleavage was assessed by immunofluorescence with a mouse anti-Δ-heparan sulfate F69-3G10 antibody (370260-S, Amsbio, Switzerland, diluted 1:500) specific for a heparan sulfate neo-epitope generated after digestion.</p> </div> <div class="heparininhibitionassay" title="sec"> <div class="title" tagx="title" title="title">
Heparin inhibition assay</div> <p>EV71-VP1<sub>97R/L167G/E</sub> mixed viral population was pre-incubated with 300 μg/ml of soluble heparin sodium salt from porcine intestinal mucosa (H4784, Sigma-Aldrich, Merck, Switzerland) diluted in culture medium or with culture medium alone as control for 1 h at 37°C. Confluent monolayers of Vero, Caco-2, RD or SH-SY5Y cells were challenged with 50 μl of the pre-incubated virus-heparin or virus-alone mixture and virus binding and replication assays were performed.</p> </div> <div class="virusbindingandreplicationassay" title="sec"> <div class="title" tagx="title" title="title">
Virus binding and replication assay</div> <p>Confluent cells were first seeded in 96-well plates and binding and replication assays were performed 24h later as previously described [<a href="#ppat.1007190.ref039">39</a>]. Briefly, the medium was removed, cells were washed with cold binding buffer (Hanks' Balanced Salt Solution [HBSS, 14175053, Gibco, Thermo Fisher Scientific, Switzerland] containing 1% BSA [wt/vol] and 0.1% sodium azide [wt/vol]) and incubated for 1h on ice with 50 μl of viral suspension containing 5*10<sup>8</sup> viral RNA copies. After removal of unbound virus with 3 washes with 200 μl of cold binding buffer, cells were either lysed directly in 200 μl of easyMAG lysis buffer to quantify bound viruses by RT-qPCR or overlaid with fresh culture medium and left an additional day at 37°C before lysis and viral load quantification by RT-qPCR. To determine the dominant population in binding and replication experiment run with EV71-VP1<sub>97R/L167G/E</sub>, the VP1 region was PCR-amplified and sequenced.</p> </div> <div class="virusbindingtoimmobilizedheparinsepharosebeads" title="sec"> <div class="title" tagx="title" title="title">
Virus binding to immobilized heparin sepharose beads</div> <p>Aliquots of 200 μl heparin sepharose beads (ab193268, Abcam, UK) were immobilized to Pierce cellulose acetate filter spin cups (69702, Thermo Fisher Scientific, Switzerland) by centrifugation at 1500 rpm for 5 min. The beads were then equilibrated with 600 μl of binding buffer (0.02M Tris-HCl, 0.14M NaCl, pH 7.4) and centrifuged for 5 min at 1500 rpm. Subsequently, 100, 200 or 600 μl of viral suspension of EV71-VP1<sub>97R/L167G/E</sub> containing respectively 10<sup>9</sup>, 2*10<sup>9</sup> and 6*10<sup>9</sup> viral RNA copies were added and incubated with the heparin sepharose beads for 1h at 4°C with gentle mixing. The flow-through fraction was collected by centrifugation (5 min at 1500 rpm) and the heparin sepharose beads were washed 5 times with 200 μl of binding buffer. Bound viral particles were then eluted twice by sequential incubation of 5 min with 200 μl of elution buffer (0.02M Tri-HCl, 2M NaCl, pH 7.4). The collected fractions (input, flow-through and eluate) were analyzed by RT-qPCR and VP1 was PCR amplified and sequenced.</p> </div> <div class="rnaextractionandrealtime-quantitativepolymerasechainreaction" title="sec"> <div class="title" tagx="title" title="title">
RNA extraction and real time-quantitative polymerase chain reaction</div> <p>RNA was extracted using the NucliSens easyMAG magnetic beads system (BioMérieux, France) according to the manufacturer’s instructions. RT-qPCR was performed using the quantitative Entero/Ge/08 assay as previously described [<a href="#ppat.1007190.ref061">61</a>] in a one-step format using the QuantiTect Probe RT-PCR Kit (Qiagen, Switzerland) according to the manufacturer's instructions in a StepOne Applied Biosystems thermocycler. As a quantitative reference standard for each run, 10-fold dilution series (from 2.5*10<sup>8</sup> to 2.5*10<sup>5</sup> copies/ml) of the <i>in vitro</i> transcribed full-length pBMH-EV-D68 was used. The RNAse P housekeeping gene (4316831, Thermo Fisher Scientific, Switzerland) was quantified by qPCR in binding and replication experiments to confirm homogeneous cell number.</p> </div> <div class="rt-pcrandsequencing" title="sec"> <div class="title" tagx="title" title="title">
RT-PCR and sequencing</div> <p>Retro-transcription was performed using Superscript II (Invitrogen) and either random hexamer primers (Roche) or a specific primer (2A-3408-R: 5’CTGGGTTTTGAAAAGCTGACC3’) as previously described [<a href="#ppat.1007190.ref039">39</a>]. The VP1 region was amplified by PCR (Fwd 5’TGCTCGAGATGGAGTATTCG3’ and Rev 5’CTGGGTTTTGAAAAGCTGACC3’) and nested PCR (Fwd 5’CGACTACTACACTACAGGCTTGGTTAG3’ and Rev 5’CATTGGGCGAGGTATCCAC3’) with platinum Taq DNA polymerase (P/N 10966026, Invitrogen, Thermo Fisher Scientific, Switzerland). PCR products were purified as previously described [<a href="#ppat.1007190.ref039">39</a>] and sequenced (Fasteris-DNA sequencing service, Switzerland).</p> </div> <div class="generationofkocelllinesusingcrispr/cas9" title="sec"> <div class="title" tagx="title" title="title">
Generation of KO cell lines using CRISPR/Cas9</div> <p>Two SCARB2 targeting guide RNAs (sgRNA1 5’-CACCGCGATGCTGCTTCTACACGGC-3’ &amp;amp; 3’-CGCTACGACGAAGATGTGCCGCAAA-5’, and sgRNA2 5’-CACCGCCGGCATTGTCTGACGTAT-3’ &amp;amp; 3’-CGGCCGTAACAGACTGCATACAAA-5’ for the sgRNA2) were subcloned in the pLentiCRISPRV2 vector (Addgene # 52961). Recombinant lentiviruses (plentiCRISPRV2, pLentiCRISPRV2-SCARB2-sg1 or pLentiCRISPRV2-SCARB2-sg2) were produced by transient transfection of HEK 293T/17 cells as previously described [<a href="#ppat.1007190.ref062">62</a>]. Caco-2 and RD cells (70–80% confluency in 6-well plates) were transduced with 1 ml of viral supernatants. Five days post transduction the cells were trypsinised and subjected to puromycin selection (58-58-2, Invivogen, LABFORCE, Switzerland; 10 μg/ml for Caco-2 cells and 6 μg/ml for RD cells). Cells were maintained in the selection medium and the expression of SCARB2 was assessed by Western Blot.</p> </div> <div class="infectionofinvitrohumanreconstitutedrespiratory(mucilair,smallair)andsmallintestinetissues(epiintestinal)" title="sec"> <div class="title" tagx="title" title="title">
Infection of <i>in vitro</i> human reconstituted respiratory (MucilAir, SmallAir) and small intestine tissues (EpiIntestinal)</div> <p>Human upper (MucilAir) and lower (SmallAir) airway epithelia (Epithelix, Geneva, Switzerland) were developed from isolated nasal polyp and distal lung epithelial cells originating from surgeries as previously described [<a href="#ppat.1007190.ref040">40</a>, <a href="#ppat.1007190.ref041">41</a>]. Human small intestine tissues (EpiIntestinal) were purchased from MatTek (Ashland, USA) and are reconstituted from primary human small intestinal epithelial cells isolated from the ileum of healthy adult donors. After full differentiation, the EpiIntestinal tissues faithfully reproduce the pseudo-stratified architecture of the human small intestine composed of enterocytes, paneth cells, M cells, tuft cells and intestinal stem cells [<a href="https://www.mattek.com/products/epiintestinal/">https://www.mattek.com/products/epiintestinal/</a>].</p> <p>The tissues, cultured at an air-liquid interface (at 33°C for MucilAir and at 37°C for the other tissues), were infected apically or basally as previously described [<a href="#ppat.1007190.ref063">63</a>]. Briefly, for apical infection, 10<sup>8</sup> viral RNA copies (in 100 μl of culture medium) culture medium were applied at the tissue/air interface while for basal infection, 6*10<sup>8</sup> viral RNA copies (in 600 μl of culture medium) were applied at the tissue/liquid interface. After 4 hours, residual viruses were removed by extensive apical or basal washes and then, every day, 200 μl of culture medium was applied apically for 20 min to collect apically released viruses. Basal medium was also collected daily and replaced with 500 μl of fresh medium. RNA was extracted from apical samples collected 5 dpi.</p> </div> <div class="infectionof2dand3dengineeredneuraltissues" title="sec"> <div class="title" tagx="title" title="title">
Infection of 2D and 3D engineered neural tissues</div> <p>2D and 3D neural tissues were engineered from human induced pluripotent neural stem cells (GSC-4311, MTI-GlobalStem, Thermo Fisher Scientific, Switzerland) and contained a mixed population of mature neurons and glial cells [<a href="#ppat.1007190.ref064">64</a>]. 2D cultures were inoculated with 10<sup>7</sup> viral RNA copies (in 100 μl of culture medium). Residual inoculum was removed after 1 h and tissues were lysed 1 hpi and 5 dpi for RNA extraction. 3D cultures were inoculated with 4*10<sup>7</sup> viral RNA copies (in 40 μl of medium) and tissues were lysed 4 hpi or 5 dpi for RNA extraction.</p> <p>For all tissues, replication was quantified by RT-qPCR and VP1 was PCR amplified and sequenced in both single infections or competition assays.</p> </div> <div class="westernblotandimmunofluorescence(if)" title="sec"> <div class="title" tagx="title" title="title">
Western blot and immunofluorescence (IF)</div> <div class="antibodies" title="sec"> <div class="title" tagx="title" title="title">
Antibodies</div> <p>Mouse anti-HS (F58-10E4, 370255–1, Amsbio, Switzerland, diluted 1:100 in PBS + 1% BSA), rabbit anti-LIMPII/Igp85 (SCARB2) (PA3-1682, Thermo Fisher Scientific, Switzerland, diluted 1:1000), rabbit anti-GFAP (Z0334, Dako, Agilent, Switzerland, diluted 1:100), mouse anti-GFAP (MAB360, Millipore, Merck, Switzerland, diluted 1:50), rabbit anti-tubulin β-III (Previously Covance Catalog# PRB-435P, Biolegend, Switzerland, diluted 1:1000) and mouse anti-tubulin β-III (T8660, Sigma-Aldrich, Merck, Switzerland, diluted 1:200) were used as primary antibodies for IF staining of respectively HS, SCARB2, glial cells and neurons. Goat anti-mouse Alexa Fluor 594 (A21044, Invitrogen, Thermo Fisher Scientific, Switzerland, diluted 1:2000), goat anti-mouse Alexa Fluor 594 (A11032, Invitrogen, Thermo Fisher Scientific, Switzerland, diluted 1:2000) and goat anti-rabbit Alexa Fluor 488 (A11008, Invitrogen, Thermo Fisher Scientific, Switzerland, diluted 1:2000) were used as secondary antibodies. Rabbit anti- LIMPII/Igp85 (SCARB2) Ab (diluted 1:1000 in 5% milk/TTBS 0.05%), mouse anti-EV71 VP2 mAb (MAB979, Millipore, Merck, Switzerland, diluted 1:1000 in 5% milk/TTBS 0.05%) and mouse anti-GAPDH mAb (6C5, sc-32233, Santa Cruz, Switzerland, diluted 1:1000 in 5% milk/TTBS 0.05%) were used as primary antibodies for western blot while anti-rabbit HRP-labelled Ab (7074, Cell Signaling Technology; diluted 1:1000 in 5% milk/TTBS 0.05%) or anti-mouse HRP-labelled Ab (7076, Cell Signaling Technology; diluted 1:1000 in 5% milk/TTBS 0.05%) were used as secondary antibodies.</p> </div> </div> <div class="westernblot" title="sec"> <div class="title" tagx="title" title="title">
Western blot</div> <p>Proteins were loaded on a 10% SDS-PAGE gel, transferred on a PVDF membrane (162–0177, BIO-RAD, Switzerland) that was hybridized with the primary Ab overnight at 4°C. The membranes were washed twice with TTBS 0.05% [TBS buffer (10 mM Tris HCl, pH 7.5, 500 mM NaCl) with 0.05% Tween] for 10 min and incubated for 1 h at 37°C with the secondary Ab. The membranes were washed twice with TTBS for 10 min and developed with the ECL system (34080, Thermo Fisher Scientific, Switzerland) according to the manufacturer’s protocol. Images were acquired with the Fujifilm LAS 4000 luminescence imager.</p> <div class="ifoncells" title="sec"> <div class="title" tagx="title" title="title">
IF on cells</div> <p>Cells plated on coverslips were washed twice with phosphate-buffered saline (PBS) and fixed for 20 min at room temperature (RT) in a 4% paraformaldehyde solution. Fixed cells were washed with PBS before incubation for 1 h at 37°C with the primary Ab. Intensive PBS washings were performed prior to incubation for 45 min at 37°C with the secondary Ab. Cells were then washed 3 times with PBS, stained with 4,6-diamidino-2-phenylindole (DAPI) for 7 min at RT and washed with PBS a final time. Coverslips were mounted in Fluoroprep mounting medium (BioMérieux, France)</p> </div> <div class="ifonparaffin-embeddedtissuesections" title="sec"> <div class="title" tagx="title" title="title">
IF on paraffin-embedded tissue sections</div> <p>Paraffin was removed from MucilAir, SmallAir and EpiIntestinal tissue sections by embedding the sections twice in UltraClear solution (3905.5000PE, Biosystems, Switzerland) for 3 min and twice in EtOH 96% (E/0600DF/15, Fisher Chemical, Thermo Fisher Scientific, Switzerland) for 3 min. Sections were then air-dried for 5 min and quickly immersed in water. Tissues were rinsed with PBS, permeabilized with 1% Triton (Fluka, Switzerland) for 15 min at RT and incubated with primary antibodies at 4°C overnight. Intensive PBS washes were performed prior to incubation with the secondary antibody for 45 min at 37°C. Next, tissues were washed 3 times with PBS, stained with DAPI at RT for 7 min, and washed with PBS a final time. Coverslips were mounted in Fluoroprep mounting medium (BioMérieux, France). Alternatively, tissue sections were counterstained with Mayer’s Hematoxylin Solution (5 min), eosin (7 min) and mounted with Néo Mount medium (1.09016.0500, Merck, Switzerland). Images were acquired with a Zeiss LSM 700 Meta confocal microscope with the "Plan-Neofluar" 63x/1.4 oil objective and were processed with the ZEN Blue Advanced Processing and Analysis module.</p> </div> <div class="ifon2dneuralcultures" title="sec"> <div class="title" tagx="title" title="title">
IF on 2D neural cultures</div> <p>2D neural cultures were fixed at RT in 4% PFA for 20 min, rinsed with PBS permeabilized with 0.1% Triton for 15 min at RT, and embedded in blocking buffer (PBS-BSA 10%) before being incubated with primary antibodies at 4°C overnight. The cultures were washed with PBS prior to incubation with the secondary antibody for 1 hour at RT. Goat anti-mouse Alexa Fluor 594 and goat anti-rabbit Alexa Fluor 488 were used as secondary Ab. After PBS washes, the nuclei were stained with DAPI at RT for 7 min and washed again with PBS. Coverslips were mounted in Fluoroprep mounting medium (BioMérieux, France). All IF Images were acquired with a Zeiss LSM 700 Meta confocal microscope and were processed with the ZEN Blue Advanced Processing and Analysis module.</p> </div> </div> <div class="metamorphanalysisincelllinesandintestinaltissues" title="sec"> <div class="title" tagx="title" title="title">
MetaMorph analysis in cell lines and intestinal tissues</div> <p>Images were acquired on a Zeiss LSM700 confocal microscope with a 63x/1.4 oil objective leading to the calibration of 0.396 microns per pixel. HS expression was measured with MetaMorph software version 7.7.6 (Molecular Devices, Sunnyvale, CA).</p> <div class="celllines" title="sec"> <div class="title" tagx="title" title="title">
Cell lines</div> <p>Blue (DAPI) and red (HS) or green (SCARB2) color channels were separated with the ColorSeparate function, and both images were converted to compatible files. A cell scoring step followed, in which the blue channel was used in "All nuclei�? section and the red or green channel was used as a positive marker with the following respective parameters: "Approximate min width�? (7 and 1 microns), "Approximate max width�? (17 and 12 microns), "Parameter Intensity above local background�? (8 and 40 levels). The "Algorithm�? parameter was set to Standard. The cell scoring step reported the "Total Cells�? number to ensure the statistical reliability of measurements, and the parameter "%Positive Cells�? to reflect the expression of the receptor.</p> </div> <div class="intestinaltissues" title="sec"> <div class="title" tagx="title" title="title">
Intestinal tissues</div> <p>Images were acquired as stacks in which four channels/wavelengths were present (blue: 440 nm; red: 645 nm; green: 510 nm; and transmission light) and four to six z positions were acquired. Red channel planes were first selected with the Select Plane function. Two regions of exactly the same size were manually drawn in the image that corresponded to apical and basal areas of the tissue and both areas were measured and stored. Next, we looped through z planes of the red channel and performed the following actions for each of the two regions: "Area of the intensity above level 3�? was measured as Total Tissue Area; "Area of the intensity above level 15�? was measured as High Intensity Tissue Area; and values of the "High Intensity Tissue Area�? across all z planes were averaged, with values being obtained for the apical and basal areas.</p> </div> </div> <div class="i-carrageenaninhibitionassay" title="sec"> <div class="title" tagx="title" title="title">
I-carrageenan inhibition assay</div> <p>I-carrageenan (C1138, Sigma-Aldrich, Merck, Switzerland) was serially diluted and added for 1h at 37°C to EV71-VP1<sub>97R167G</sub>, EV71-VP1<sub>97L167E</sub> or EV71-VP1<sub>97R/L167E/G</sub> (multiplicity of infection [MOI], 0.01 PFU/cell). The mixture was then added to confluent 2D neural cells or Vero cells grown in a 96-well plate at a density of 13 × 10<sup>4</sup>/well. After 1 h at 37°C, monolayers were washed and overlaid with fresh medium. One dpi, viral replication was monitored by immunocytochemistry (ICC) in Vero cells and by RT-qPCR in neural cells.</p> <p>For ICC, cells were washed with PBS, fixed with cold methanol:acetone (1:1) for 1 min and permeabilised with PBS-Triton 0.1% for 5 min on ice. Cells were then incubated 1h at 37°C with mouse anti-EV71 VP2 monoclonal antibody (MAB979, Millipore, Merck, Switzerland; diluted 1:500 in PBS-BSA 1%). After 3 washes with PBS-Triton 0.1%, cells were incubated 1 h at 37°C with the anti-mouse HRP-labelled secondary antibody (7076, Cell Signaling Technology; diluted 1:1000 in PBS-BSA 1%). After extensive washing, DAB substrate solution (D4293-50SET, Sigma-Aldrich, Merck, Switzerland) was added for 15 min before final washing with PBS. More than 400 cells per well were scored under a light microscope. The percent of inhibition of virus infectivity was determined by comparing the percentage of infected cells in presence of increasing concentrations of ı-carrageenan relative to the percent of infected cells in absence of ı-carrageenan (set as reference at 100%). All data were generated from duplicate wells in three independent experiments.</p> <p>For quantification by RT-qPCR, neural cultures were lysed in 200 μl of easyMAG lysis buffer. RNA was extracted, viral replication was quantified by RT-qPCR and VP1 was PCR amplified and sequenced. The percent of inhibition of virus infectivity was determined by comparing the viral load measured in neural cells in presence of increasing concentrations of ı-carrageenan relative to the viral load measured in absence of ı-carrageenan (set as reference at 100%).</p> </div> <div class="statistics" title="sec"> <div class="title" tagx="title" title="title">
Statistics</div> <p>Values are expressed as mean (± SEM). Experiments were performed at least in biological duplicates, with N in the figure legend indicating the number of replicates. Two-way ANOVA, Fisher’s exact tests and determination of the fifty percent inhibitory concentration (IC<sub>50</sub>) was determined using GraphPad Prism 7.02 software.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Fig</span> <div class="title" tagx="title" title="title">
Human SCARB2 is necessary for successful infection by EV71-VP1<sub>97R167G</sub>, EV71-VP1<sub>97L167E</sub> variants in mouse cells.</div> <p>Mouse L929 cells were transfected with a control plasmid (pCTL) or a plasmid expressing human SCARB2 (pSCARB2). 24h post transfection, cells were infected with EV71-VP1<sub>97R167G</sub> or EV71-VP1<sub>97L167E</sub> variants at a MOI = 0.2. 24 h later, cells were lysed and the expression of SCARB2 and of EV71 VP0 and VP2 proteins were quantified by western blot. Of note, the antibody used to detect SCARB2 is not specific for human SCARB2 and recognises endogenous mouse SCARB2.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="ppat.1007190.s001.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Fig</span> <div class="title" tagx="title" title="title">
Glial cells and mature neurons express HS and SCARB2.</div> <p>Co-localisation of HS expressing cells (stained in red) with <b>(A)</b> glial cells (stained in green) and <b>(B)</b> neurons (stained in green). Co-localisation of SCARB2 expressing cells (stained in green) with <b>(C)</b> glial cells (stained in red) or <b>(D)</b> neurons (stained in red). For each panel subpanel showing cell nuclei and merged images are also shown. Scale bar = 20 μm.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="ppat.1007190.s002.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Text</span> <div class="title" tagx="title" title="title">
Exogenous expression of hSCARB2 in mouse cells (L929).</div> <p>The pCWX-UBI-SCARB2-PGK-GFP SCARB2 expressing vector was constructed with the Gateway cloning technology according to the manufacturer’s instruction (Gateway LR Clonase II Enzyme mix, 11791020, Invitrogen, Thermo Fisher Scientific, Switzerland). An LR recombination reaction was performed between the entry clone containing the SCARB2 coding sequence (pENTR-L1-SCARB2-L2, Clone ID IOH9776, Invitrogen, Thermo Fisher Scientific, Switzerland), the entry clone containing the ubiquitin promoter (pENTR-L4-UBI-L1R, gift from Patrick Salmon, Addgene plasmid # 45959) and the destination vector containing attR sites (pCWX-R4-DEST-R2-PGK-GFP, kindly provided by Prof. Karl-Heinz Krause, University of Geneva). Sub confluent mouse L929 cells, were transfected with pCWX-UBI-SCARB2-PGK-GFP or pCLX-UBI-GFP (gift from Patrick Salmon, Addgene # 27245). After 24 h, the cells were infected with EV71-VP1<sub>97R167G</sub> and EV71-VP1<sub>97L167E</sub> at an MOI of 0.2. 24 h post infection cells were lysed with RIPA (Tris 50 mM- pH 7, NaCl 150 mM, 0.1% SDS, 0.5% Sodium deoxycholate, 1% Triton X-100) and analysed by Western Blot both for quantification of SCARB2 expression and to highlight expression of viral protein. To this end, protein were loaded on a 10% SDS-PAGE gel, transferred on a PVDF membrane (162–0177, BIO-RAD, Switzerland) that was hybridized with the primary rabbit anti- LIMPII/Igp85 (SCARB2) Ab (PA3-1682, Thermo Fisher Scientific, Switzerland, diluted 1:1000 in 5% milk/TTBS 0.05%), mouse anti-EV71 VP2 mAb (MAB979, Millipore, Merck, Switzerland, diluted 1:1000 in 5% milk/TTBS 0.05%) and mouse anti-GAPDH mAb (6C5, sc-32233, Santa Cruz, Switzerland) overnight at 4°C. The membranes were incubated for 1 h at 37°C with the anti-rabbit HRP-labelled secondary antibody (7074, Cell Signaling Technology; diluted 1:1000 in 5% milk/TTBS 0.05%) or the anti-mouse HRP-labelled secondary antibody (7076, Cell Signaling Technology; diluted 1:1000 in 5% milk/TTBS 0.05%).</p> <p>(DOCX)</p>  <div class="media" title="media"><a href="ppat.1007190.s003.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank Sergei Startchik for image analysis (Bioimaging Core Facility, University of Geneva), Marie Ebrahim Malek for tissue sectioning and staining with H-E (Histology Core Facility, University of Geneva), Ophélie Cherpin (Department of Pathology and Immunology, University of Geneva), Laetitia Nikles (Tissue Engineering Laboratory, University of Applied Sciences, Western Switzerland), Laura Batti (Microscopy Facility Manager, WYSS Center) and Liudmila Efremova (Neurix SA, Geneva, Switzerland) for their technical input.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="ppat.1007190.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pallansch</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>. <span class="mixed-article-title" title="mixed-article-title">Complete nucleotide sequence of enterovirus 71 is distinct from poliovirus</span>. <span class="source" tagx="source" title="source">Virus Res</span>. <span class="year" tagx="year" title="year">1995</span>;<span class="volume" tagx="volume" title="volume">39</span>(<span class="issue" tagx="issue" title="issue">2–3</span>):<span class="fpage" tagx="fpage" title="fpage">195</span>–<span class="lpage" tagx="lpage" title="lpage">205</span>. .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8837884">8837884</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Melnick</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>. <span class="mixed-article-title" title="mixed-article-title">Enterovirus type 71 infections: a varied clinical pattern sometimes mimicking paralytic poliomyelitis</span>. <span class="source" tagx="source" title="source">Rev Infect Dis</span>. <span class="year" tagx="year" title="year">1984</span>;<span class="volume" tagx="volume" title="volume">6</span> Suppl <span class="issue" tagx="issue" title="issue">2</span>:<span class="fpage" tagx="fpage" title="fpage">S387</span>–<span class="lpage" tagx="lpage" title="lpage">90</span>. .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6330839">6330839</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perez-Velez</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McFarland</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nix</span><span class="given-names" tagx="given-names" title="given-names">WA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pallansch</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Outbreak of neurologic enterovirus type 71 disease: a diagnostic challenge</span>. <span class="source" tagx="source" title="source">Clin Infect Dis</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">45</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">950</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1086/521895">10.1086/521895</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17879907">17879907</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">LG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parashar</span><span class="given-names" tagx="given-names" title="given-names">UD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lye</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ong</span><span class="given-names" tagx="given-names" title="given-names">FG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaki</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alexander</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Deaths of children during an outbreak of hand, foot, and mouth disease in sarawak, malaysia: clinical and pathological characteristics of the disease. For the Outbreak Study Group</span>. <span class="source" tagx="source" title="source">Clin Infect Dis</span>. <span class="year" tagx="year" title="year">2000</span>;<span class="volume" tagx="volume" title="volume">31</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">678</span>–<span class="lpage" tagx="lpage" title="lpage">83</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1086/314032">10.1086/314032</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11017815">11017815</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">LY</span></span>. <span class="mixed-article-title" title="mixed-article-title">Enterovirus 71 in Taiwan</span>. <span class="source" tagx="source" title="source">Pediatr Neonatol</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">49</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">103</span>–<span class="lpage" tagx="lpage" title="lpage">12</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S1875-9572(08)60023-6">10.1016/S1875-9572(08)60023-6</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19054914">19054914</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsu</span><span class="given-names" tagx="given-names" title="given-names">KH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Twu</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">KT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsai</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">An epidemic of enterovirus 71 infection in Taiwan. Taiwan Enterovirus Epidemic Working Group</span>. <span class="source" tagx="source" title="source">N Engl J Med</span>. <span class="year" tagx="year" title="year">1999</span>;<span class="volume" tagx="volume" title="volume">341</span>(<span class="issue" tagx="issue" title="issue">13</span>):<span class="fpage" tagx="fpage" title="fpage">929</span>–<span class="lpage" tagx="lpage" title="lpage">35</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJM199909233411301">10.1056/NEJM199909233411301</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10498487">10498487</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qiu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="mixed-article-title" title="mixed-article-title">Enterovirus 71 infection: a new threat to global public health?</span><span class="source" tagx="source" title="source">Lancet neurology</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">868</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S1474-4422(08)70207-2">10.1016/S1474-4422(08)70207-2</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18848307">18848307</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ooi</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewthwaite</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cardosa</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Solomon</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>. <span class="mixed-article-title" title="mixed-article-title">Clinical features, diagnosis, and management of enterovirus 71</span>. <span class="source" tagx="source" title="source">Lancet neurology</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">1097</span>–<span class="lpage" tagx="lpage" title="lpage">105</span>. Epub 2010/10/23. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S1474-4422(10)70209-X">10.1016/S1474-4422(10)70209-X</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20965438">20965438</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Solomon</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lewthwaite</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perera</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cardosa</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMinn</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ooi</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span>. <span class="mixed-article-title" title="mixed-article-title">Virology, epidemiology, pathogenesis, and control of enterovirus 71</span>. <span class="source" tagx="source" title="source">Lancet Infect Dis</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">778</span>–<span class="lpage" tagx="lpage" title="lpage">90</span>. Epub 2010/10/22. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S1473-3099(10)70194-8">10.1016/S1473-3099(10)70194-8</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20961813">20961813</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMinn</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span>. <span class="mixed-article-title" title="mixed-article-title">An overview of the evolution of enterovirus 71 and its clinical and public health significance</span>. <span class="source" tagx="source" title="source">FEMS Microbiol Rev</span>. <span class="year" tagx="year" title="year">2002</span>;<span class="volume" tagx="volume" title="volume">26</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">91</span>–<span class="lpage" tagx="lpage" title="lpage">107</span>. .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12007645">12007645</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ooi</span><span class="given-names" tagx="given-names" title="given-names">EE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phoon</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ishak</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span>. <span class="mixed-article-title" title="mixed-article-title">Seroepidemiology of human enterovirus 71, Singapore</span>. <span class="source" tagx="source" title="source">Emerg Infect Dis</span>. <span class="year" tagx="year" title="year">2002</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">995</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3201/eid0809.10.3201/eid0809.010397">10.3201/eid0809.10.3201/eid0809.010397</a></span> ; PubMed Central PMCID: PMCPMC2732542.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12194783">12194783</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pallansch MA</span><span class="given-names" tagx="given-names" title="given-names">RR</span></span>. <span class="chapter-title" tagx="chapter-title" title="chapter-title">Enteroviruses: poliovirus, coxsackieviruses, echoviruses, and newer enteroviruses</span><span class="source" tagx="source" title="source">Field Virology</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Philadelphia</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">LippincottWilliams &amp;amp; Wilkins</span>; <span class="year" tagx="year" title="year">2001</span> p. pp. <span class="fpage" tagx="fpage" title="fpage">723</span>–<span class="lpage" tagx="lpage" title="lpage">75</span>.</span></li> <li tag="ref"><a name="ppat.1007190.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">WHO. A Guide to Clinical Management and Public Health Response for Hand, Foot and Mouth Disease (HFMD). HFMD Guidance2011. p. 71.</span></li> <li tag="ref"><a name="ppat.1007190.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ong</span><span class="given-names" tagx="given-names" title="given-names">KC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">KT</span></span>. <span class="mixed-article-title" title="mixed-article-title">Understanding Enterovirus 71 Neuropathogenesis and Its Impact on Other Neurotropic Enteroviruses</span>. <span class="source" tagx="source" title="source">Brain Pathol</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">25</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">614</span>–<span class="lpage" tagx="lpage" title="lpage">24</span>. Epub 2015/08/16. <span class="pub-id"><a href="https://dx.doi.org/10.1111/bpa.12279">10.1111/bpa.12279</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26276025">26276025</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anastasina</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Domanska</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palm</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Butcher</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">Human picornaviruses associated with neurological diseases and their neutralization by antibodies</span>. <span class="source" tagx="source" title="source">J Gen Virol</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">98</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">1145</span>–<span class="lpage" tagx="lpage" title="lpage">58</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1099/jgv.0.000780">10.1099/jgv.0.000780</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28631594">28631594</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamayoshi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fujii</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koike</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">Receptors for enterovirus 71</span>. <span class="source" tagx="source" title="source">Emerg Microbes Infect</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">3</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">e53</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/emi.2014.49">10.1038/emi.2014.49</a></span> ; PubMed Central PMCID: PMCPMC4126179.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26038749">26038749</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamayoshi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamashita</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanagata</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minowa</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takemura</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Scavenger receptor B2 is a cellular receptor for enterovirus 71</span>. <span class="source" tagx="source" title="source">Nat Med</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">15</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">798</span>–<span class="lpage" tagx="lpage" title="lpage">801</span>. Epub 2009/06/23. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nm.1992">10.1038/nm.1992</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19543282">19543282</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiao</span><span class="given-names" tagx="given-names" title="given-names">XY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">DY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">XL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qiu</span><span class="given-names" tagx="given-names" title="given-names">QC</span></span>. <span class="mixed-article-title" title="mixed-article-title">Distribution of EV71 receptors SCARB2 and PSGL-1 in human tissues</span>. <span class="source" tagx="source" title="source">Virus Res</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">190</span>:<span class="fpage" tagx="fpage" title="fpage">40</span>–<span class="lpage" tagx="lpage" title="lpage">52</span>. Epub 2014/07/06. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virusres.2014.05.007">10.1016/j.virusres.2014.05.007</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24997419">24997419</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fujii</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagata</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sato</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ong</span><span class="given-names" tagx="given-names" title="given-names">KC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">KT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamayoshi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Transgenic mouse model for the study of enterovirus 71 neuropathogenesis</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">110</span>(<span class="issue" tagx="issue" title="issue">36</span>):<span class="fpage" tagx="fpage" title="fpage">14753</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1217563110">10.1073/pnas.1217563110</a></span> ; PubMed Central PMCID: PMCPMC3767555.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23959904">23959904</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishimura</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shimojima</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tano</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miyamura</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wakita</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shimizu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>. <span class="mixed-article-title" title="mixed-article-title">Human P-selectin glycoprotein ligand-1 is a functional receptor for enterovirus 71</span>. <span class="source" tagx="source" title="source">Nat Med</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">15</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">794</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nm.1961">10.1038/nm.1961</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19543284">19543284</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nishimura</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hafenstein</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kataoka</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wakita</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bergelson</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Enterovirus 71 binding to PSGL-1 on leukocytes: VP1-145 acts as a molecular switch to control receptor interaction</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">e1003511</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1003511">10.1371/journal.ppat.1003511</a></span> ; PubMed Central PMCID: PMCPMC3723564.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23935488">23935488</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poh</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sam</span><span class="given-names" tagx="given-names" title="given-names">IC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">YF</span></span>. <span class="mixed-article-title" title="mixed-article-title">Enterovirus 71 uses cell surface heparan sulfate glycosaminoglycan as an attachment receptor</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">87</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">611</span>–<span class="lpage" tagx="lpage" title="lpage">20</span>. Epub 2012/10/26. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02226-12">10.1128/JVI.02226-12</a></span> ; PubMed Central PMCID: PMC3536405.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23097443">23097443</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chuang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span>. <span class="mixed-article-title" title="mixed-article-title">Sialylated glycans as receptor and inhibitor of enterovirus 71 infection to DLD-1 intestinal cells</span>. <span class="source" tagx="source" title="source">Virol J</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">141</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1743-422X-6-141">10.1186/1743-422X-6-141</a></span> ; PubMed Central PMCID: PMC2751754.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19751532">19751532</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">PY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">YF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">SW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lo</span><span class="given-names" tagx="given-names" title="given-names">YC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">YH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Cell surface nucleolin facilitates enterovirus 71 binding and infection</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">89</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">4527</span>–<span class="lpage" tagx="lpage" title="lpage">38</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.03498-14">10.1128/JVI.03498-14</a></span> ; PubMed Central PMCID: PMC4442404.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25673703">25673703</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cong</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Cell surface vimentin is an attachment receptor for enterovirus 71</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">88</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">5816</span>–<span class="lpage" tagx="lpage" title="lpage">33</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.03826-13">10.1128/JVI.03826-13</a></span> ; PubMed Central PMCID: PMCPMC4019121.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24623428">24623428</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chou</span><span class="given-names" tagx="given-names" title="given-names">YT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">CN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>. <span class="mixed-article-title" title="mixed-article-title">Annexin II binds to capsid protein VP1 of enterovirus 71 and enhances viral infectivity</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">85</span>(<span class="issue" tagx="issue" title="issue">22</span>):<span class="fpage" tagx="fpage" title="fpage">11809</span>–<span class="lpage" tagx="lpage" title="lpage">20</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00297-11">10.1128/JVI.00297-11</a></span> ; PubMed Central PMCID: PMCPMC3209289.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21900167">21900167</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Molecular determinants of enterovirus 71 viral entry: cleft around GLN-172 on VP1 protein interacts with variable region on scavenge receptor B 2</span>. <span class="source" tagx="source" title="source">J Biol Chem</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">287</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">6406</span>–<span class="lpage" tagx="lpage" title="lpage">20</span>. Epub 2012/01/06. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M111.301622">10.1074/jbc.M111.301622</a></span> ; PubMed Central PMCID: PMC3307280.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22219187">22219187</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Victorio</span><span class="given-names" tagx="given-names" title="given-names">CB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chow</span><span class="given-names" tagx="given-names" title="given-names">VT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chua</span><span class="given-names" tagx="given-names" title="given-names">KB</span></span>. <span class="mixed-article-title" title="mixed-article-title">Cooperative effect of the VP1 amino acids 98E, 145A and 169F in the productive infection of mouse cell lines by enterovirus 71 (BS strain)</span>. <span class="source" tagx="source" title="source">Emerg Microbes Infect</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">5</span>:<span class="fpage" tagx="fpage" title="fpage">e60</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/emi.2016.56">10.1038/emi.2016.56</a></span> ; PubMed Central PMCID: PMCPMC4932649.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27329847">27329847</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sam</span><span class="given-names" tagx="given-names" title="given-names">IC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">VS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">HV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">YF</span></span>. <span class="mixed-article-title" title="mixed-article-title">VP1 residues around the five-fold axis of enterovirus A71 mediate heparan sulfate interaction</span>. <span class="source" tagx="source" title="source">Virology</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">501</span>:<span class="fpage" tagx="fpage" title="fpage">79</span>–<span class="lpage" tagx="lpage" title="lpage">87</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2016.11.009">10.1016/j.virol.2016.11.009</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27875780">27875780</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cifuente</span><span class="given-names" tagx="given-names" title="given-names">JO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashley</span><span class="given-names" tagx="given-names" title="given-names">RE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Conway</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Makhov</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tano</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">A strain-specific epitope of enterovirus 71 identified by cryo-electron microscopy of the complex with fab from neutralizing antibody</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">87</span>(<span class="issue" tagx="issue" title="issue">21</span>):<span class="fpage" tagx="fpage" title="fpage">11363</span>–<span class="lpage" tagx="lpage" title="lpage">70</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01926-13">10.1128/JVI.01926-13</a></span> ; PubMed Central PMCID: PMCPMC3807339.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23946455">23946455</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fujii</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sudaka</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takashino</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kobayashi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kataoka</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">VP1 amino acid residue 145 of enterovirus 71 is a key residue for its receptor attachment and resistance to neutralizing antibody during cynomolgus monkey infection</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2018</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00682-18">10.1128/JVI.00682-18</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29848582">29848582</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kataoka</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suzuki</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kotani</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iwata-Yoshikawa</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagata</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ami</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The Role of VP1 Amino Acid Residue 145 of Enterovirus 71 in Viral Fitness and Pathogenesis in a Cynomolgus Monkey Model</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">e1005033</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1005033">10.1371/journal.ppat.1005033</a></span> ; PubMed Central PMCID: PMCPMC4504482.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26181772">26181772</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaini</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMinn</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>. <span class="mixed-article-title" title="mixed-article-title">A single mutation in capsid protein VP1 (Q145E) of a genogroup C4 strain of human enterovirus 71 generates a mouse-virulent phenotype</span>. <span class="source" tagx="source" title="source">J Gen Virol</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">93</span>(<span class="issue" tagx="issue" title="issue">Pt 9</span>):<span class="fpage" tagx="fpage" title="fpage">1935</span>–<span class="lpage" tagx="lpage" title="lpage">40</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1099/vir.0.043893-0">10.1099/vir.0.043893-0</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22647370">22647370</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kobayashi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sudaka</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takashino</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imura</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fujii</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koike</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">Amino acid variation at VP1-145 of enterovirus 71 determines attachment receptor usage and neurovirulence in human scavenger receptor B2 transgenic mice</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2018</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00681-18">10.1128/JVI.00681-18</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29848584">29848584</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">SC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">WC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">GW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsao</span><span class="given-names" tagx="given-names" title="given-names">KC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">CG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">YC</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Genetic characterization of enterovirus 71 isolated from patients with severe disease by comparative analysis of complete genomes</span>. <span class="source" tagx="source" title="source">J Med Virol</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">84</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">931</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jmv.23287">10.1002/jmv.23287</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22499017">22499017</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wan</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The variations of VP1 protein might be associated with nervous system symptoms caused by enterovirus 71 infection</span>. <span class="source" tagx="source" title="source">BMC Infect Dis</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">14</span>:<span class="fpage" tagx="fpage" title="fpage">243</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2334-14-243">10.1186/1471-2334-14-243</a></span> ; PubMed Central PMCID: PMCPMC4101859.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24886383">24886383</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">A novel finding for enterovirus virulence from the capsid protein VP1 of EV71 circulating in mainland China</span>. <span class="source" tagx="source" title="source">Virus Genes</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">48</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">260</span>–<span class="lpage" tagx="lpage" title="lpage">72</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s11262-014-1035-2">10.1007/s11262-014-1035-2</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24442718">24442718</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zou</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>. <span class="mixed-article-title" title="mixed-article-title">Molecular analysis of virulent determinants of enterovirus 71</span>. <span class="source" tagx="source" title="source">PLoS ONE</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">6</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">e26237</span> Epub 2011/11/01. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0026237">10.1371/journal.pone.0026237</a></span> ; PubMed Central PMCID: PMC3198388.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22039449">22039449</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cordey</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Petty</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schibler</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martinez</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerlach</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Belle</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Identification of site-specific adaptations conferring increased neural cell tropism during human enterovirus 71 infection</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">e1002826</span> Epub 2012/08/23. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1002826">10.1371/journal.ppat.1002826</a></span> ; PubMed Central PMCID: PMC3406088.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22910880">22910880</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Essaidi-Laziosi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brito</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benaoudia</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Royston</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cagno</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernandes-Rocha</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Propagation of respiratory viruses in human airway epithelia reveals persistent virus-specific signatures</span>. <span class="source" tagx="source" title="source">J Allergy Clin Immunol</span>. <span class="year" tagx="year" title="year">2017</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jaci.2017.07.018">10.1016/j.jaci.2017.07.018</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28797733">28797733</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref041" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boda</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vernaz</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferreira</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wiszniewski</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Constant</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">Establishment and characterization of an in vitro human small airway model (SmallAir)</span>. <span class="source" tagx="source" title="source">Eur J Pharm Biopharm</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">118</span>:<span class="fpage" tagx="fpage" title="fpage">68</span>–<span class="lpage" tagx="lpage" title="lpage">72</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.ejpb.2016.12.006">10.1016/j.ejpb.2016.12.006</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28040470">28040470</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref042" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Royston</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Essaidi-Laziosi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perez-Rodriguez</span><span class="given-names" tagx="given-names" title="given-names">FJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Piuz</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geiser</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krause</span><span class="given-names" tagx="given-names" title="given-names">KH</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Viral chimeras decrypt the role of enterovirus capsid proteins in viral tropism, acid sensitivity and optimal growth temperature</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">14</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">e1006962</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1006962">10.1371/journal.ppat.1006962</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29630666">29630666</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref043" /><span class="label" tagx="label" title="label">43</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antona</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kossorotoff</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schuffenecker</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mirand</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leruez-Ville</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bassi</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Severe paediatric conditions linked with EV-A71 and EV-D68, France, May to October 2016</span>. <span class="source" tagx="source" title="source">Euro surveillance: bulletin europeen sur les maladies transmissibles = European communicable disease bulletin</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">21</span>(<span class="issue" tagx="issue" title="issue">46</span>). <span class="pub-id"><a href="https://dx.doi.org/10.2807/1560-7917.ES.2016.21.46.30402">10.2807/1560-7917.ES.2016.21.46.30402</a></span> ; PubMed Central PMCID: PMCPMC5144948.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27918268">27918268</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref044" /><span class="label" tagx="label" title="label">44</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karrasch</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fischer</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scholten</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sauerbrei</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Henke</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Renz</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">A severe pediatric infection with a novel enterovirus A71 strain, Thuringia, Germany</span>. <span class="source" tagx="source" title="source">J Clin Virol</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">84</span>:<span class="fpage" tagx="fpage" title="fpage">90</span>–<span class="lpage" tagx="lpage" title="lpage">5</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jcv.2016.09.007">10.1016/j.jcv.2016.09.007</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27771495">27771495</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref045" /><span class="label" tagx="label" title="label">45</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kong</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohapatra</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">San Juan-Vergara</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hellermann</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Inhibition of respiratory syncytial virus infection with intranasal siRNA nanoparticles targeting the viral NS1 gene</span>. <span class="source" tagx="source" title="source">Nat Med</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">56</span>–<span class="lpage" tagx="lpage" title="lpage">62</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nm1174">10.1038/nm1174</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15619625">15619625</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref046" /><span class="label" tagx="label" title="label">46</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bode</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salvestrini</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">PW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Esko</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamaguchi</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Heparan sulfate and syndecan-1 are essential in maintaining murine and human intestinal epithelial barrier function</span>. <span class="source" tagx="source" title="source">J Clin Invest</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">118</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">229</span>–<span class="lpage" tagx="lpage" title="lpage">38</span>. Epub 2007/12/08. <span class="pub-id"><a href="https://dx.doi.org/10.1172/JCI32335">10.1172/JCI32335</a></span> ; PubMed Central PMCID: PMCPMC2117765.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18064305">18064305</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref047" /><span class="label" tagx="label" title="label">47</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sa-Carvalho</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rieder</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baxt</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodarte</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanuri</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mason</span><span class="given-names" tagx="given-names" title="given-names">PW</span></span>. <span class="mixed-article-title" title="mixed-article-title">Tissue culture adaptation of foot-and-mouth disease virus selects viruses that bind to heparin and are attenuated in cattle</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">1997</span>;<span class="volume" tagx="volume" title="volume">71</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">5115</span>–<span class="lpage" tagx="lpage" title="lpage">23</span>. ; PubMed Central PMCID: PMCPMC191746.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9188578">9188578</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref048" /><span class="label" tagx="label" title="label">48</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prestwood</span><span class="given-names" tagx="given-names" title="given-names">TR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prigozhin</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharar</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zellweger</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shresta</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">A mouse-passaged dengue virus strain with reduced affinity for heparan sulfate causes severe disease in mice by establishing increased systemic viral loads</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">82</span>(<span class="issue" tagx="issue" title="issue">17</span>):<span class="fpage" tagx="fpage" title="fpage">8411</span>–<span class="lpage" tagx="lpage" title="lpage">21</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00611-08">10.1128/JVI.00611-08</a></span> ; PubMed Central PMCID: PMCPMC2519668.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18562532">18562532</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref049" /><span class="label" tagx="label" title="label">49</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferguson</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saul</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fragkoudis</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weisheit</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patabendige</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Ability of the Encephalitic Arbovirus Semliki Forest Virus To Cross the Blood-Brain Barrier Is Determined by the Charge of the E2 Glycoprotein</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">89</span>(<span class="issue" tagx="issue" title="issue">15</span>):<span class="fpage" tagx="fpage" title="fpage">7536</span>–<span class="lpage" tagx="lpage" title="lpage">49</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.03645-14">10.1128/JVI.03645-14</a></span> ; PubMed Central PMCID: PMCPMC4505677.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25972559">25972559</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref050" /><span class="label" tagx="label" title="label">50</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silva</span><span class="given-names" tagx="given-names" title="given-names">LA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khomandiak</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashbrook</span><span class="given-names" tagx="given-names" title="given-names">AW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weller</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heise</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morrison</span><span class="given-names" tagx="given-names" title="given-names">TE</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">A single-amino-acid polymorphism in Chikungunya virus E2 glycoprotein influences glycosaminoglycan utilization</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">88</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">2385</span>–<span class="lpage" tagx="lpage" title="lpage">97</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.03116-13">10.1128/JVI.03116-13</a></span> ; PubMed Central PMCID: PMCPMC3958064.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24371059">24371059</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref051" /><span class="label" tagx="label" title="label">51</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pfeiffer</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span>. <span class="mixed-article-title" title="mixed-article-title">Emergence of a Large-Plaque Variant in Mice Infected with Coxsackievirus B3</span>. <span class="source" tagx="source" title="source">MBio</span>. <span class="year" tagx="year" title="year">2016</span>;<span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">e00119</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/mBio.00119-16">10.1128/mBio.00119-16</a></span> ; PubMed Central PMCID: PMCPMC4817249.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27025249">27025249</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref052" /><span class="label" tagx="label" title="label">52</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryman</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gardner</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">CW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meier</span><span class="given-names" tagx="given-names" title="given-names">KC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klimstra</span><span class="given-names" tagx="given-names" title="given-names">WB</span></span>. <span class="mixed-article-title" title="mixed-article-title">Heparan sulfate binding can contribute to the neurovirulence of neuroadapted and nonneuroadapted Sindbis viruses</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">81</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">3563</span>–<span class="lpage" tagx="lpage" title="lpage">73</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02494-06">10.1128/JVI.02494-06</a></span> ; PubMed Central PMCID: PMCPMC1866052.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17215278">17215278</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref053" /><span class="label" tagx="label" title="label">53</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">SW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">YH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsai</span><span class="given-names" tagx="given-names" title="given-names">HP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuo</span><span class="given-names" tagx="given-names" title="given-names">PH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">A Selective Bottleneck Shapes the Evolutionary Mutant Spectra of Enterovirus A71 during Viral Dissemination in Humans</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="fpage" tagx="fpage" title="fpage">91</span>(23). <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01062-17">10.1128/JVI.01062-17</a></span> ; PubMed Central PMCID: PMCPMC5686718.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28931688">28931688</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref054" /><span class="label" tagx="label" title="label">54</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arthur Huang</span><span class="given-names" tagx="given-names" title="given-names">KY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">MF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">YC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shih</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiu</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Epitope-associated and specificity-focused features of EV71-neutralizing antibody repertoires from plasmablasts of infected children</span>. <span class="source" tagx="source" title="source">Nat Commun</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">762</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-017-00736-9">10.1038/s41467-017-00736-9</a></span> ; PubMed Central PMCID: PMCPMC5624920.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28970483">28970483</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref055" /><span class="label" tagx="label" title="label">55</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">SW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tai</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fonville</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">CC</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Mapping Enterovirus A71 Antigenic Determinants from Viral Evolution</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">89</span>(<span class="issue" tagx="issue" title="issue">22</span>):<span class="fpage" tagx="fpage" title="fpage">11500</span>–<span class="lpage" tagx="lpage" title="lpage">6</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02035-15">10.1128/JVI.02035-15</a></span> ; PubMed Central PMCID: PMCPMC4645677.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26339057">26339057</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref056" /><span class="label" tagx="label" title="label">56</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yi</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kung</span><span class="given-names" tagx="given-names" title="given-names">HF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span>. <span class="mixed-article-title" title="mixed-article-title">The virology and developments toward control of human enterovirus 71</span>. <span class="source" tagx="source" title="source">Critical reviews in microbiology</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">37</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">313</span>–<span class="lpage" tagx="lpage" title="lpage">27</span>. Epub 2011/06/10. <span class="pub-id"><a href="https://dx.doi.org/10.3109/1040841X.2011.580723">10.3109/1040841X.2011.580723</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21651436">21651436</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref057" /><span class="label" tagx="label" title="label">57</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaoutis</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klein</span><span class="given-names" tagx="given-names" title="given-names">JD</span></span>. <span class="mixed-article-title" title="mixed-article-title">Enterovirus infections</span>. <span class="source" tagx="source" title="source">Pediatr Rev</span>. <span class="year" tagx="year" title="year">1998</span>;<span class="volume" tagx="volume" title="volume">19</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">183</span>–<span class="lpage" tagx="lpage" title="lpage">91</span>. .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9613170">9613170</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref058" /><span class="label" tagx="label" title="label">58</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abzug</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">The enteroviruses: problems in need of treatments</span>. <span class="source" tagx="source" title="source">The Journal of infection</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">68</span> Suppl <span class="issue" tagx="issue" title="issue">1</span>:<span class="fpage" tagx="fpage" title="fpage">S108</span>–<span class="lpage" tagx="lpage" title="lpage">14</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jinf.2013.09.020">10.1016/j.jinf.2013.09.020</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24119825">24119825</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref059" /><span class="label" tagx="label" title="label">59</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schibler</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerlach</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martinez</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belle</span><span class="given-names" tagx="given-names" title="given-names">SV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turin</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaiser</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Experimental human rhinovirus and enterovirus interspecies recombination</span>. <span class="source" tagx="source" title="source">J Gen Virol</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">93</span>(<span class="issue" tagx="issue" title="issue">Pt 1</span>):<span class="fpage" tagx="fpage" title="fpage">93</span>–<span class="lpage" tagx="lpage" title="lpage">101</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1099/vir.0.035808-0">10.1099/vir.0.035808-0</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21940413">21940413</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref060" /><span class="label" tagx="label" title="label">60</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reed</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muench</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>. <span class="mixed-article-title" title="mixed-article-title">A simple method of estimating fifty percent endpoints</span>. <span class="source" tagx="source" title="source">Am J Hyg</span>. <span class="year" tagx="year" title="year">1938</span>;(<span class="issue" tagx="issue" title="issue">27</span>):<span class="fpage" tagx="fpage" title="fpage">493</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>.</span></li> <li tag="ref"><a name="ppat.1007190.ref061" /><span class="label" tagx="label" title="label">61</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schibler</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yerly</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vieille</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Docquier</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turin</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaiser</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Critical analysis of rhinovirus RNA load quantification by real-time reverse transcription-PCR</span>. <span class="source" tagx="source" title="source">J Clin Microbiol</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">50</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">2868</span>–<span class="lpage" tagx="lpage" title="lpage">72</span>. Epub 2012/06/22. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JCM.06752-11">10.1128/JCM.06752-11</a></span> ; PubMed Central PMCID: PMC3421824.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22718934">22718934</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref062" /><span class="label" tagx="label" title="label">62</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zufferey</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagy</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mandel</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naldini</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trono</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>. <span class="mixed-article-title" title="mixed-article-title">Multiply attenuated lentiviral vector achieves efficient gene delivery in vivo</span>. <span class="source" tagx="source" title="source">Nat Biotechnol</span>. <span class="year" tagx="year" title="year">1997</span>;<span class="volume" tagx="volume" title="volume">15</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">871</span>–<span class="lpage" tagx="lpage" title="lpage">5</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nbt0997-871">10.1038/nbt0997-871</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9306402">9306402</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref063" /><span class="label" tagx="label" title="label">63</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tapparel</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sobo</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Constant</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Belle</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaiser</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>. <span class="mixed-article-title" title="mixed-article-title">Growth and characterization of different human rhinovirus C types in three-dimensional human airway epithelia reconstituted in vitro</span>. <span class="source" tagx="source" title="source">Virology</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">446</span>(<span class="issue" tagx="issue" title="issue">1–2</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2013.06.031">10.1016/j.virol.2013.06.031</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24074561">24074561</a></span></span></li> <li tag="ref"><a name="ppat.1007190.ref064" /><span class="label" tagx="label" title="label">64</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Richard Josephson and Jonathan M. Auerbach MT, Inc.—GlobalStem, Gaithersburg, MD USA 20877. Human Induced Pluripotent Stem cell-derived neurons as a reproducible system for drug discovery. Stem Cell Symposium; Maryland2014.</span></li> </ul> </div> </div>  </body></html>