<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/embed/graphic-5.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/09/2020.03.28.013789.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.28.013789v3" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Structural analysis of SARS-CoV-2 and predictions of the human interactome" />
<meta name="DC.Identifier" content="10.1101/2020.03.28.013789" />
<meta name="DC.Date" content="2020-04-09" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="We calculated the structural properties of &gt;2500 coronaviruses and computed &gt;100000 human protein interactions with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Using the CROSS method, we found that the SARS-CoV-2 region encompassing nucleotides 23000 - 24000 is highly conserved at the structural level, while the region upstream varies significantly. These two sequences are important for viral infection as they code for a domain of the viral protein Spike S interacting with the human receptor angiotensin-converting enzyme 2 (ACE2) and, in the close homologue from Middle East respiratory syndrome coronavirus (MERS-CoV), sialic acids. We predict highly structured regions at the 5’ and 3’ where our calculations indicate strong propensity to bind to human proteins involved in viral replication. Using the cat RAPID method, we identified that the 5’ interacts with double-stranded RNA-specific editase 1 ADARB1, 2-5A-dependent ribonuclease RNASEL, ATP-dependent RNA helicase DDX1 and A-kinase anchor protein 8-like AKAP8L, in addition to &gt;10 high-confidence candidate partners. These interactions, also implicated in HIV replication, should be further investigated for a better understanding of host-virus interaction mechanisms.

### Competing Interest Statement

The authors have declared no competing interest." />
<meta name="DC.Contributor" content="Andrea Vandelli" />
<meta name="DC.Contributor" content="Michele Monti" />
<meta name="DC.Contributor" content="Edoardo Milanetti" />
<meta name="DC.Contributor" content="Riccardo Delli Ponti" />
<meta name="DC.Contributor" content="Gian Gaetano Tartaglia" />
<meta name="article:published_time" content="2020-04-09" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Structural analysis of SARS-CoV-2 and predictions of the human interactome" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We calculated the structural properties of &amp;gt;2500 coronaviruses and computed &amp;gt;100000 human protein interactions with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Using the CROSS method, we found that the SARS-CoV-2 region encompassing nucleotides 23000 - 24000 is highly conserved at the structural level, while the region upstream varies significantly. These two sequences are important for viral infection as they code for a domain of the viral protein Spike S interacting with the human receptor angiotensin-converting enzyme 2 (ACE2) and, in the close homologue from Middle East respiratory syndrome coronavirus (MERS-CoV), sialic acids. We predict highly structured regions at the 5 and 3 prime ends where our calculations indicate strong propensity to bind to human proteins involved in viral replication. Using the catRAPID method, we identified that the 5 prime end interacts with double-stranded RNA-specific editase 1 ADARB1, 2-5A-dependent ribonuclease RNASEL, ATP-dependent RNA helicase DDX1 and A-kinase anchor protein 8-like AKAP8L, in addition to &amp;gt;10 high-confidence candidate partners. These interactions, also implicated in HIV replication, should be further investigated for a better understanding of host-virus interaction mechanisms.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.28.013789v3" />
<meta name="citation_id" content="2020.03.28.013789v3" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.28.013789v3" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.28.013789v3.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.28.013789v3.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.28.013789" />
<meta name="citation_num_pages" content="28" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.28.013789" />
<meta name="citation_author" content="Andrea Vandelli" />
<meta name="citation_author_institution" content="Centre for Genomic Regulation (CRG), The Barcelona Institute for Science and Technology, Dr. Aiguader 88, 08003 Barcelona, Spain and Universitat Pompeu Fabra (UPF)" />
<meta name="citation_author_institution" content="Systems Biology of Infection Lab, Department of Biochemistry and Molecular Biology, Biosciences Faculty, Universitat Autònoma de Barcelona" />
<meta name="citation_author" content="Michele Monti" />
<meta name="citation_author_institution" content="Centre for Genomic Regulation (CRG), The Barcelona Institute for Science and Technology, Dr. Aiguader 88, 08003 Barcelona, Spain and Universitat Pompeu Fabra (UPF)" />
<meta name="citation_author_institution" content="RNA System Biology Lab, department of Neuroscience and Brain Technologies, Istituto Italiano di Tecnologia" />
<meta name="citation_author" content="Edoardo Milanetti" />
<meta name="citation_author_institution" content="Department of Physics, Sapienza University" />
<meta name="citation_author_institution" content="Center for Life Nanoscience, Istituto Italiano di Tecnologia" />
<meta name="citation_author" content="Riccardo Delli Ponti" />
<meta name="citation_author_institution" content="School of Biological Sciences, Nanyang Technological University" />
<meta name="citation_author_email" content="riccardo.ponti@ntu.edu.sg" />
<meta name="citation_author_email" content="giangaetano.tartaglia@uniroma1.it" />
<meta name="citation_author_email" content="gian.tartaglia@iit.it" />
<meta name="citation_author" content="Gian Gaetano Tartaglia" />
<meta name="citation_author_institution" content="Centre for Genomic Regulation (CRG), The Barcelona Institute for Science and Technology, Dr. Aiguader 88, 08003 Barcelona, Spain and Universitat Pompeu Fabra (UPF)" />
<meta name="citation_author_institution" content="RNA System Biology Lab, department of Neuroscience and Brain Technologies, Istituto Italiano di Tecnologia" />
<meta name="citation_author_institution" content="Center for Life Nanoscience, Istituto Italiano di Tecnologia" />
<meta name="citation_author_institution" content="Institucio Catalana de Recerca i Estudis Avançats (ICREA)" />
<meta name="citation_author_email" content="riccardo.ponti@ntu.edu.sg" />
<meta name="citation_author_email" content="giangaetano.tartaglia@uniroma1.it" />
<meta name="citation_author_email" content="gian.tartaglia@iit.it" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7524-6310" />
<meta name="citation_reference" content="Zhu, N. et al. A Novel Coronavirus from Patients with Pneumonia in China, 2019. N. Engl. J. Med. 382, 727–733 (2020)." />
<meta name="citation_reference" content="D’Antiga, L. Coronaviruses and immunosuppressed patients. The facts during the third epidemic. Liver Transplant. Off. Publ. Am. Assoc. Study Liver Dis. Int. Liver Transplant. Soc. (2020) doi:10.1002/lt.25756." />
<meta name="citation_reference" content="Cascella, M., Rajnik, M., Cuomo, A., Dulebohn, S. C. &amp; Di Napoli, R. Features, Evaluation and Treatment Coronavirus (COVID-19). in StatPearls (StatPearls Publishing, 2020)." />
<meta name="citation_reference" content="Ge, X.-Y. et al. Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor. Nature 503, 535–538 (2013)." />
<meta name="citation_reference" content="Follis, K. E., York, J. &amp; Nunberg, J. H. Furin cleavage of the SARS coronavirus spike glycoprotein enhances cell-cell fusion but does not affect virion entry. Virology 350, 358–369 (2006)." />
<meta name="citation_reference" content="Xiao, K. et al. Isolation and Characterization of 2019-nCoV-like Coronavirus from Malayan Pangolins. bioRxiv 2020.02.17.951335 (2020) doi:10.1101/2020.02.17.951335." />
<meta name="citation_reference" content="Park, Y.-J. et al. Structures of MERS-CoV spike glycoprotein in complex with sialoside attachment receptors. Nat. Struct. Mol. Biol. 26, 1151–1157 (2019)." />
<meta name="citation_reference" content="Walls, A. C. et al. Cryo-electron microscopy structure of a coronavirus spike glycoprotein trimer. Nature 531, 114–117 (2016)." />
<meta name="citation_reference" content="Li, W. et al. Identification of sialic acid-binding function for the Middle East respiratory syndrome coronavirus spike glycoprotein. Proc. Natl. Acad. Sci. U. S. A. 114, E8508–E8517 (2017)." />
<meta name="citation_reference" content="Yang, D. &amp; Leibowitz, J. L. The Structure and Functions of Coronavirus Genomic 3’ and 5’ Ends. Virus Res. 206, 120–133 (2015)." />
<meta name="citation_reference" content="Delli Ponti, R., Marti, S., Armaos, A. &amp; Tartaglia, G. G. A high-throughput approach to profile RNA structure. Nucleic Acids Res. 45, e35–e35 (2017)." />
<meta name="citation_reference" content="Delli Ponti, R., Armaos, A., Marti, S. &amp; Gian Gaetano Tartaglia. A Method for RNA Structure Prediction Shows Evidence for Structure in lncRNAs. Front. Mol. Biosci. 5, 111 (2018)." />
<meta name="citation_reference" content="Sanchez de Groot, N. et al. RNA structure drives interaction with proteins. Nat. Commun. 10, 3246 (2019)." />
<meta name="citation_reference" content="Cid-Samper, F. et al. An Integrative Study of Protein-RNA Condensates Identifies Scaffolding RNAs and Reveals Players in Fragile X-Associated Tremor/Ataxia Syndrome. Cell Rep. 25, 3422–3434.e7 (2018)." />
<meta name="citation_reference" content="Cerase, A. et al. Phase separation drives X-chromosome inactivation: a hypothesis. Nat. Struct. Mol. Biol. 26, 331 (2019)." />
<meta name="citation_reference" content="Agostini, F. et al. catRAPID omics: a web server for large-scale prediction of protein-RNA interactions. Bioinforma. Oxf. Engl. 29, 2928–2930 (2013)." />
<meta name="citation_reference" content="Cirillo, D. et al. Quantitative predictions of protein interactions with long noncoding RNAs. Nat. Methods 14, 5–6 (2017)." />
<meta name="citation_reference" content="Bellucci, M., Agostini, F., Masin, M. &amp; Tartaglia, G. G. Predicting protein associations with long noncoding RNAs. Nat. Methods 8, 444–445 (2011)." />
<meta name="citation_reference" content="Lang, B., Armaos, A. &amp; Tartaglia, G. G. RNAct: Protein–RNA interaction predictions for model organisms with supporting experimental data. Nucleic Acids Res. doi:10.1093/nar/gky967." />
<meta name="citation_reference" content="Kliger, Y. &amp; Levanon, E. Y. Cloaked similarity between HIV-1 and SARS-CoV suggests an anti-SARS strategy. BMC Microbiol. 3, 20 (2003)." />
<meta name="citation_reference" content="Hallenberger, S. et al. Inhibition of furin-mediated cleavage activation of HIV-1 glycoprotein gp160. Nature 360, 358–361 (1992)." />
<meta name="citation_reference" content="Glowacka, I. et al. Differential downregulation of ACE2 by the spike proteins of severe acute respiratory syndrome coronavirus and human coronavirus NL63. J. Virol. 84, 1198–1205 (2010)." />
<meta name="citation_reference" content="Gultyaev, A. P., Richard, M., Spronken, M. I., Olsthoorn, R. C. L. &amp; Fouchier, R. A. M. Conserved structural RNA domains in regions coding for cleavage site motifs in hemagglutinin genes of influenza viruses. Virus Evol. 5, (2019)." />
<meta name="citation_reference" content="Wu, A. et al. Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China. Cell Host Microbe 27, 325–328 (2020)." />
<meta name="citation_reference" content="Lorenz, R. et al. ViennaRNA Package 2.0. Algorithms Mol. Biol. 6, 26 (2011)." />
<meta name="citation_reference" content="Madeira, F. et al. The EMBL-EBI search and sequence analysis tools APIs in 2019. Nucleic Acids Res. 47, W636–W641 (2019)." />
<meta name="citation_reference" content="Andersen, K. G., Rambaut, A., Lipkin, W. I., Holmes, E. C. &amp; Garry, R. F. The proximal origin of SARS-CoV-2. Nat. Med. 1–3 (2020) doi:10.1038/s41591-020-0820-9." />
<meta name="citation_reference" content="Zhou, P. et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 579, 270–273 (2020)." />
<meta name="citation_reference" content="Di Tommaso, P. et al. T-Coffee: a web server for the multiple sequence alignment of protein and RNA sequences using structural information and homology extension. Nucleic Acids Res. 39, W13–W17 (2011)." />
<meta name="citation_reference" content="Qing, E., Hantak, M., Perlman, S. &amp; Gallagher, T. Distinct Roles for Sialoside and Protein Receptors in Coronavirus Infection. mBio 11, (2020)." />
<meta name="citation_reference" content="Milanetti, E. et al. In-Silico evidence for two receptors based strategy of SARS-CoV-2. ArXiv200311107Phys.Q-Bio (2020)." />
<meta name="citation_reference" content="Galán, C. et al. Host cell proteins interacting with the 3’ end of TGEV coronavirus genome influence virus replication. Virology 391, 304–314 (2009)." />
<meta name="citation_reference" content="Lu, K., Heng, X. &amp; Summers, M. F. Structural determinants and mechanism of HIV-1 genome packaging. J. Mol. Biol. 410, 609–633 (2011)." />
<meta name="citation_reference" content="Fehr, A. R. &amp; Perlman, S. Coronaviruses: An Overview of Their Replication and Pathogenesis. Methods Mol. Biol. Clifton NJ 1282, 1–23 (2015)." />
<meta name="citation_reference" content="Klus, P., Ponti, R. D., Livi, C. M. &amp; Tartaglia, G. G. Protein aggregation, structural disorder and RNA-binding ability: a new approach for physico-chemical and gene ontology classification of multiple datasets. BMC Genomics 16, 1071 (2015)." />
<meta name="citation_reference" content="UniProt: a worldwide hub of protein knowledge. Nucleic Acids Res. 47, D506–D515 (2019)." />
<meta name="citation_reference" content="Castello, A. et al. Insights into RNA biology from an atlas of mammalian mRNA-binding proteins. Cell 149, 1393–1406 (2012)." />
<meta name="citation_reference" content="Bell, N. M. &amp; Lever, A. M. L. HIV Gag polyprotein: processing and early viral particle assembly. Trends Microbiol. 21, 136–144 (2013)." />
<meta name="citation_reference" content="Warde-Farley, D. et al. The GeneMANIA prediction server: biological network integration for gene prioritization and predicting gene function. Nucleic Acids Res. 38, W214–W220 (2010)." />
<meta name="citation_reference" content="Hayward, S. D. Viral interactions with the Notch pathway. Semin. Cancer Biol. 14, 387–396 (2004)." />
<meta name="citation_reference" content="Dudley, J. P., Mertz, J. A., Rajan, L., Lozano, M. &amp; Broussard, D. R. What retroviruses teach us about the involvement of c-Myc in leukemias and lymphomas. Leukemia 16, 1086–1098 (2002)." />
<meta name="citation_reference" content="Kim, D. et al. The architecture of SARS-CoV-2 transcriptome. bioRxiv 2020.03.12.988865 (2020) doi:10.1101/2020.03.12.988865." />
<meta name="citation_reference" content="Fang, J. et al. A DEAD box protein facilitates HIV-1 replication as a cellular co-factor of Rev. Virology 330, 471–480 (2004)." />
<meta name="citation_reference" content="Xu, L. et al. The cellular RNA helicase DDX1 interacts with coronavirus nonstructural protein 14 and enhances viral replication. J. Virol. 84, 8571–8583 (2010)." />
<meta name="citation_reference" content="Carlson, K. A. et al. Molecular characterization of a putative antiretroviral transcriptional factor, OTK18. J. Immunol. Baltim. Md 1950 172, 381–391 (2004)." />
<meta name="citation_reference" content="Ivanov, D. et al. Cyclin T1 domains involved in complex formation with Tat and TAR RNA are critical for tat-activation. J. Mol. Biol. 288, 41–56 (1999)." />
<meta name="citation_reference" content="Kwak, Y. T., Ivanov, D., Guo, J., Nee, E. &amp; Gaynor, R. B. Role of the human and murine cyclin T proteins in regulating HIV-1 tat-activation. J. Mol. Biol. 288, 57–69 (1999)." />
<meta name="citation_reference" content="Locke, M., Tinsley, C. L., Benson, M. A. &amp; Blake, D. J. TRIM32 is an E3 ubiquitin ligase for dysbindin. Hum. Mol. Genet. 18, 2344–2358 (2009)." />
<meta name="citation_reference" content="Gordon, D. E. et al. A SARS-CoV-2-Human Protein-Protein Interaction Map Reveals Drug Targets and Potential Drug-Repurposing. bioRxiv 2020.03.22.002386 (2020) doi:10.1101/2020.03.22.002386." />
<meta name="citation_reference" content="Xing, L., Zhao, X., Guo, F. &amp; Kleiman, L. The role of A-kinase anchoring protein 95-like protein in annealing of tRNALys3 to HIV-1 RNA. Retrovirology 11, 58 (2014)." />
<meta name="citation_reference" content="Markert, A. et al. The La-related protein LARP7 is a component of the 7SK ribonucleoprotein and affects transcription of cellular and viral polymerase II genes. EMBO Rep. 9, 569–575 (2008)." />
<meta name="citation_reference" content="C, J. et al. Systematic Analysis of the Protein Interaction Network for the Human Transcription Machinery Reveals the Identity of the 7SK Capping Enzyme. Molecular cell vol. 27 https://pubmed.ncbi.nlm.nih.gov/17643375/ (2007)." />
<meta name="citation_reference" content="Mendez, D. et al. ChEMBL: towards direct deposition of bioassay data. Nucleic Acids Res. 47, D930–D940 (2019)." />
<meta name="citation_reference" content="Delli Ponti, R., Marti, S., Armaos, A. &amp; Tartaglia, G. G. A high-throughput approach to profile RNA structure. Nucleic Acids Res. 45, e35–e35 (2017)." />
<meta name="citation_reference" content="Zhang, S., Schlott, B., Görlach, M. &amp; Grosse, F. DNA-dependent protein kinase (DNA-PK) phosphorylates nuclear DNA helicase II/RNA helicase A and hnRNP proteins in an RNA-dependent manner. Nucleic Acids Res. 32, 1–10 (2004)." />
<meta name="citation_reference" content="Siddiqui, M. A., Mukherjee, S., Manivannan, P. &amp; Malathi, K. RNase L Cleavage Products Promote Switch from Autophagy to Apoptosis by Caspase-Mediated Cleavage of Beclin-1. Int. J. Mol. Sci. 16, 17611–17636 (2015)." />
<meta name="twitter:title" content="Structural analysis of SARS-CoV-2 and predictions of the human interactome" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="We calculated the structural properties of &gt;2500 coronaviruses and computed &gt;100000 human protein interactions with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Using the CROSS method, we found that the SARS-CoV-2 region encompassing nucleotides 23000 - 24000 is highly conserved at the structural level, while the region upstream varies significantly. These two sequences are important for viral infection as they code for a domain of the viral protein Spike S interacting with the human receptor angiotensin-converting enzyme 2 (ACE2) and, in the close homologue from Middle East respiratory syndrome coronavirus (MERS-CoV), sialic acids. We predict highly structured regions at the 5’ and 3’ where our calculations indicate strong propensity to bind to human proteins involved in viral replication. Using the cat RAPID method, we identified that the 5’ interacts with double-stranded RNA-specific editase 1 ADARB1, 2-5A-dependent ribonuclease RNASEL, ATP-dependent RNA helicase DDX1 and A-kinase anchor protein 8-like AKAP8L, in addition to &gt;10 high-confidence candidate partners. These interactions, also implicated in HIV replication, should be further investigated for a better understanding of host-virus interaction mechanisms.

### Competing Interest Statement

The authors have declared no competing interest." />
<meta name="og-title" property="og:title" content="Structural analysis of SARS-CoV-2 and predictions of the human interactome" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.28.013789v3" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="We calculated the structural properties of &gt;2500 coronaviruses and computed &gt;100000 human protein interactions with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Using the CROSS method, we found that the SARS-CoV-2 region encompassing nucleotides 23000 - 24000 is highly conserved at the structural level, while the region upstream varies significantly. These two sequences are important for viral infection as they code for a domain of the viral protein Spike S interacting with the human receptor angiotensin-converting enzyme 2 (ACE2) and, in the close homologue from Middle East respiratory syndrome coronavirus (MERS-CoV), sialic acids. We predict highly structured regions at the 5’ and 3’ where our calculations indicate strong propensity to bind to human proteins involved in viral replication. Using the cat RAPID method, we identified that the 5’ interacts with double-stranded RNA-specific editase 1 ADARB1, 2-5A-dependent ribonuclease RNASEL, ATP-dependent RNA helicase DDX1 and A-kinase anchor protein 8-like AKAP8L, in addition to &gt;10 high-confidence candidate partners. These interactions, also implicated in HIV replication, should be further investigated for a better understanding of host-virus interaction mechanisms.

### Competing Interest Statement

The authors have declared no competing interest." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-09" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.28.013789v3.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.28.013789v3.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.28.013789v3.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.28.013789v3" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1237309" />
    <title>Structural analysis of SARS-CoV-2 and predictions of the human interactome | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"mmyYOOXHlFyyM0wHaQTPzUPRa_e1lI8kxbFqKy58hSE","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1237309","apath":"\/biorxiv\/early\/2020\/04\/09\/2020.03.28.013789.atom","pisa":"biorxiv;2020.03.28.013789v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.28.013789v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.28.013789v3","title":"Structural analysis of SARS-CoV-2 and predictions of the human interactome","identifier":"node\/1237309"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YeRoX6M7jGAHXW5RW3LWVKhHv8BM5PAJIBzR8GZ7HuEfYWY13xFmtLVWElk6GNjRmokjw+UMx0bCa59UQsntj32pd+nWtH0isUw7niG8oCmARYZ58WkgKilPs+FIT+B+hiiDtjSBqlYEJtG3I5Rwag5ILxq3lYa8z0P0ZEmzNUcVH0GY44jzs9KHMOctZkeZ8Z7+0ciVfkSnavlYeUCgcr0sswkwCKUL1p2qi4242BspdG1aUSWzU6sicqKN2hqIZ5LR0G6U+dE+R0+sLBFANCBWS3afez\\\\\\\/cRYm7xclo9OhiQUUGzXs6nfFed2KfPJYmtUNi9COVgupFuWBUh4zs3L68KQ6cCnSSmnhZXqTISovyrBi2k34ohzjPa3Wl\\\\\\\/gjHMRs9KNsiEWgHGfzfuG0twToySKqLD0Rs3KjsYkyymGWZGS9UfW27VLLtA82BYC\\\\\\\/mP2DbUCK5nWOU4HXO+UXm9M8Jvodsz1sqgiBkkOFTPDvpp4oQXgH1HtcDletKKjYxP2M2BIUfpAGIOQAR4xrChw0EVsdyiAiWy2wISTEUL9J9f8dNakac\\\\\\\/Y4D8DK0SY4s+A14OWzZrb8kXWAFpIAo2k9pGob1zuzQz4dcZIYqQmc4zzAplRpoiaRfUSZcnzXhEHuEwmg8dekpge3JB2BFuEsWKHY8UzS7r0WBJIIEX\\\\\\\/RBOUdrSezdj1\\\\\\\/0I1q36hz3csF4nOH93i3HZxVRgEv2Uh0jIiRs37QqIoWCcP5T2fDHt3VMmvEsCBBsLNkwzmfWMHUPGzLrBhmiA8OfMN1AUwJtapAPM5azZp18yFTXOEg\\\\\\\/AvSJ2HD67fks77Kjfq9srrMxs+zdaa\\\\\\\/yDxxFXoZBx3puVmwiv74Dc63uyN62hyYFjy40eJ3gVz6wsIc8FzuEQzUmEmZvhLwpXCws9XbqMv0HLJTDhH64QFI5akT87q9Zg+BShZ2Z+bZKG4WCpJZ8ZJ+R92SzTErNhsA9HJgiGvVGDXcuSLAoCcsMCehQWlrDlnzuAvdnfi8Tnh8yC8z1xo2KnhHNgtqz+v1RiB1tjHGmC3hd3cqtStWpCYi9bAPRTqfZv2RmUEgTQXm66qTy8pSIQb+WUr0eMN\\\\\\\/l6s\\\\\\\/Ud6nMMRj6nu4hkQ3Aial38ygeL1p3XZlOvsQ9HmbRrAhFYQd9F286kZ\\\\\\\/KTARHXKA9EzyjcGnjJj\\\\\\\/VVtp+0by52ReTIL0CE3\\\\\\\/G5ifA9k5gFZHbapOl9XCKfmqCmlN+A71+OsKM+zFpn4R0PVV5GkwIju6k5\\\\\\\/kdhhQkjEEea58TtiVVqRdbhs\\\\\\\/QUsCrh5J2Cre4rlxB0rfi23hLzlhJWsjJ9Qd9nfn6U4Uksh2QwNA4NDoz\\\u0022,\\\u0022iv\\\u0022:\\\u00224vIdFqJJ4BtHztAjFw1nNA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00228e02afc5b9b6c8e3ba795ebc5ea6858c98211d073fd4a5fe87adf7a08da2bd11\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.28.013789v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1237309 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.28.013789v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-C2dkttTadYhcDASTKoyND4J4T1WjLK8WpK2F-xntTJw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1237309" id="node1237309" data-pisa="biorxiv;2020.03.28.013789v3" data-pisa-master="biorxiv;2020.03.28.013789" data-apath="/biorxiv/early/2020/04/09/2020.03.28.013789.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Structural analysis of SARS-CoV-2 and predictions of the human interactome</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Andrea</span> <span class="nlm-surname">Vandelli</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Michele</span> <span class="nlm-surname">Monti</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Edoardo</span> <span class="nlm-surname">Milanetti</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Riccardo</span> <span class="nlm-surname">Delli Ponti</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-7524-6310" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Gian</span> <span class="nlm-surname">Gaetano Tartaglia</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.28.013789 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1237309" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrea Vandelli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Centre for Genomic Regulation (CRG), The Barcelona Institute for Science and Technology, Dr. Aiguader 88, 08003 Barcelona, Spain and Universitat Pompeu Fabra (UPF)</span>, 08003 Barcelona, <span class='nlm-country'>Spain</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Systems Biology of Infection Lab, Department of Biochemistry and Molecular Biology, Biosciences Faculty, Universitat Autònoma de Barcelona</span>, 08193 Cerdanyola del Vallès, <span class='nlm-country'>Spain</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BVandelli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vandelli%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrea%2BVandelli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michele Monti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Centre for Genomic Regulation (CRG), The Barcelona Institute for Science and Technology, Dr. Aiguader 88, 08003 Barcelona, Spain and Universitat Pompeu Fabra (UPF)</span>, 08003 Barcelona, <span class='nlm-country'>Spain</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>RNA System Biology Lab, department of Neuroscience and Brain Technologies, Istituto Italiano di Tecnologia</span>, Via Morego 30, 16163, Genoa, <span class='nlm-country'>Italy</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michele%2BMonti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Monti%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichele%2BMonti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Edoardo Milanetti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Physics, Sapienza University</span>, Piazzale Aldo Moro 5, 00185, Rome, <span class='nlm-country'>Italy</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Center for Life Nanoscience, Istituto Italiano di Tecnologia</span>, Viale Regina Elena 291, 00161, Rome, <span class='nlm-country'>Italy</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edoardo%2BMilanetti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Milanetti%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdoardo%2BMilanetti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Riccardo Delli Ponti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>School of Biological Sciences, Nanyang Technological University</span>, 60 Nanyang Drive, Singapore, 637551, <span class='nlm-country'>Singapore</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Riccardo%2BDelli%2BPonti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Delli%20Ponti%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARiccardo%2BDelli%2BPonti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:riccardo.ponti@ntu.edu.sg" class="" data-icon-position="" data-hide-link-title="0">riccardo.ponti@ntu.edu.sg</a>
<a href="mailto:giangaetano.tartaglia@uniroma1.it" class="" data-icon-position="" data-hide-link-title="0">giangaetano.tartaglia@uniroma1.it</a>
<a href="mailto:gian.tartaglia@iit.it" class="" data-icon-position="" data-hide-link-title="0">gian.tartaglia@iit.it</a></span></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gian Gaetano Tartaglia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Centre for Genomic Regulation (CRG), The Barcelona Institute for Science and Technology, Dr. Aiguader 88, 08003 Barcelona, Spain and Universitat Pompeu Fabra (UPF)</span>, 08003 Barcelona, <span class='nlm-country'>Spain</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>RNA System Biology Lab, department of Neuroscience and Brain Technologies, Istituto Italiano di Tecnologia</span>, Via Morego 30, 16163, Genoa, <span class='nlm-country'>Italy</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Center for Life Nanoscience, Istituto Italiano di Tecnologia</span>, Viale Regina Elena 291, 00161, Rome, <span class='nlm-country'>Italy</span></div><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Institucio Catalana de Recerca i Estudis Avançats (ICREA)</span>, 23 Passeig Lluis Companys, 08010 Barcelona, <span class='nlm-country'>Spain</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gian%2BGaetano%2BTartaglia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gaetano%20Tartaglia%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGian%2BGaetano%2BTartaglia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7524-6310" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gian Gaetano Tartaglia</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:riccardo.ponti@ntu.edu.sg" class="" data-icon-position="" data-hide-link-title="0">riccardo.ponti@ntu.edu.sg</a>
<a href="mailto:giangaetano.tartaglia@uniroma1.it" class="" data-icon-position="" data-hide-link-title="0">giangaetano.tartaglia@uniroma1.it</a>
<a href="mailto:gian.tartaglia@iit.it" class="" data-icon-position="" data-hide-link-title="0">gian.tartaglia@iit.it</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.28.013789v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1237309" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1237309/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.28.013789v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1237309" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1237309/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.28.013789v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1237309" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1237309/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.28.013789v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1237309" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1237309/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.28.013789v3.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1237309" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1237309/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.28.013789v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1237309" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1237309/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-2">We calculated the structural properties of &gt;2500 coronaviruses and computed &gt;100000 human protein interactions with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Using the <em>CROSS</em> method, we found that the SARS-CoV-2 region encompassing nucleotides 23000 - 24000 is highly conserved at the structural level, while the region upstream varies significantly. These two sequences are important for viral infection as they code for a domain of the viral protein Spike S interacting with the human receptor angiotensin-converting enzyme 2 (ACE2) and, in the close homologue from Middle East respiratory syndrome coronavirus (MERS-CoV), sialic acids. We predict highly structured regions at the 5’ and 3’ where our calculations indicate strong propensity to bind to human proteins involved in viral replication. Using the <em>cat</em>RAPID method, we identified that the 5’ interacts with double-stranded RNA-specific editase 1 ADARB1, 2-5A-dependent ribonuclease RNASEL, ATP-dependent RNA helicase DDX1 and A-kinase anchor protein 8-like AKAP8L, in addition to &gt;10 high-confidence candidate partners. These interactions, also implicated in HIV replication, should be further investigated for a better understanding of host-virus interaction mechanisms.</p></div><div class="section" id="sec-1"><h2 class="">INTRODUCTION</h2><p id="p-5">A novel disease named Covid-19 by the World Health Organization and caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has been recognized as responsible for the pneumonia outbreak that started in December, 2019 in Wuhan City, Hubei, China <sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup> and spread in February to Milan, Lombardy, Italy <sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup> becoming pandemic. As of April 2020, the virus infected &gt;1’000’000 people in more than 200 countries.</p><p id="p-6">SARS-CoV-2 is a positive-sense single-stranded RNA virus that shares similarities with other beta-coronavirus such as severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>. Bats have been identified as the primary host for SARS-CoV and SARS-CoV-2 <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a> but the intermediate host linking SARS-CoV-2 to humans is still unknown, although a recent report indicates that pangolins could be involved <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>.</p><p id="p-7">Coronaviruses use species-specific regions to mediate the entry in the host cell and SARS-CoV, MERS-CoV and SARS-CoV-2, the spike S protein activates the infection in human respiratory epithelial cells <sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. Spike S is assembled as a trimer and contains around 1,300 amino acids within each unit <sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup>. In the S’ region of the protein, the receptor binding domain (RBD), which contains around 300 amino acids, mediates the binding with angiotensin-converting enzyme, (ACE2) attacking respiratory cells. Another region upstream of the RBD, present in MERS-CoV but not in SARS-CoV, is involved in the adhesion to sialic acid and could play a key role in regulating viral infection <sup><a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup>.</p><p id="p-8">At present, very few molecular details are available on SARS-CoV-2 and its interactions with human host, which are mediated by specific RNA elements <sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>. To study the RNA structural content, we used <em>CROSS</em> <sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup> that was previously developed to investigate large transcripts such as the human immunodeficiency virus HIV-1 <sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>. <em>CROSS</em> predicts the structural profile (single- and double-stranded state) at single-nucleotide resolution using sequence information only. We performed sequence and structural alignments among 62 SARS-CoV-2 strains and identified the conservation of specific elements in the spike S region, which provide clues on the evolution of domains involved in the binding to ACE2 and sialic acid.</p><p id="p-9">As highly structured regions of RNA molecules have strong propensity to form stable contacts with proteins <sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup> and promote assembly of specific complexes <sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>, SARS-CoV-2 domains enriched in double-stranded content are expected to establish interactions within host cells that are important to replicate the virus. To investigate the interactions potential of SARS-CoV-2 RNA with human proteins, we employed <em>cat</em>RAPID <sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>. <em>cat</em>RAPID <sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup> estimates the binding potential of protein and RNA molecules through van der Waals, hydrogen bonding and secondary structure propensities of allowing identification of interaction partners with high confidence <sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>. The unbiased analysis of more than 100000 protein interactions with SARS-CoV-2 RNA reveals that the 5’ of SARS-CoV-2 has strong propensity to bind to human proteins involved in viral infection and especially reported to be associated with HIV infection. A comparison between SARS-CoV and HIV reveals indeed similarities <sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup>, but the relationship between SARS-CoV-2 and HIV is still unexplored. Interestingly, HIV and SARS-CoV-2, but not SARS-CoV nor MERS-CoV, have a furin-cleavage site occurs in the spike S protein, which could explain the spread velocity of SARS-CoV-2 compared to SARS-CoV and MERS-CoV <sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>,<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>. Yet, many processes related to SARS-CoV-2 replication are unknown and our study aims to suggest relevant protein interactions for further investigation.</p><p id="p-10">We hope that our large-scale calculations of structural properties and binding partners of SARS-CoV-2 will be useful to identify the mechanisms of virus replication within the human host.</p></div><div class="section" id="sec-2"><h2 class="">RESULTS</h2><div id="sec-3" class="subsection"><h3>SARS-CoV-2 contains highly structured elements</h3><p id="p-11">Structural elements within RNA molecules attract proteins <sup><a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a></sup> and reveal regions important for interactions with the host <sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup>.</p><p id="p-12">To analyze SARS-CoV-2 (reference Wuhan strain MN908947), we employed <em>CROSS</em> <sup><a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a></sup> that predicts the double- and single-stranded content of large transcripts such as <em>Xist</em> and HIV-1 <sup><a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a></sup>. We found the highest density of double-stranded regions in the 5’ (nucleotides 1-253), membrane M protein (nucleotides 26523-27191), spike S protein (nucleotides 23000-24000), and nucleocapsid N protein (nucleotides 2874-29533; <strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a></strong>) <sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup>. The lowest density of double-stranded regions were observed at nucleotides 6000-6250 and 20000-21500 and correspond to the regions between the non-structural proteins nsp14 and nsp15 and the upstream region of the spike surface protein S (<strong><a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a></strong>) <sup><a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a></sup>. In addition to the maximum corresponding to nucleotides 23000-24000, the structural content of spike S protein shows minima at around nucleotides 20500 and 24500 (<strong><a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1</a></strong>). We used the <em>Vienna</em> method <sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup> to further investigate the RNA secondary structure of specific regions identified with <em>CROSS</em> <sup><a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a></sup>. Employing a 100 nucleotide window centered around <em>CROSS</em> maxima and minima, we found good match between <em>CROSS</em> scores and Vienna free energies (<strong><a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1</a>).</strong>Strong agreement is also observed between <em>CROSS</em> and <em>Vienna</em> positional entropy, indicating that regions with the highest structural content have also the lowest structural diversity.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Using the CROSS approach 12,54, we studied the structural content of SARS-CoV-2. We found the highest density of double-stranded regions in the 5&#x2019; (nucleotides 1-253), membrane M protein (nucleotides 26523-27191), and the spike S protein (nucleotides 23000-24000). Strong match is observed between CROSS and Vienna analyses (centroid structures shown, indicating that regions with the highest structural content have the lowest free energies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1501986072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Using the CROSS approach&lt;/em&gt; &lt;sup&gt;12,54&lt;/sup&gt;&lt;em&gt;, we studied the structural content of SARS-CoV-2. We found the highest density of double-stranded regions in the 5&#x2019; (nucleotides 1-253), membrane M protein (nucleotides 26523-27191), and the spike S protein (nucleotides 23000-24000). Strong match is observed between CROSS and Vienna analyses (centroid structures shown, indicating that regions with the highest structural content have the lowest free energies&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F1.medium.gif" width="440" height="337"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F1.medium.gif" width="440" height="337"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1.</span> <p id="p-13" class="first-child"><em>Using the CROSS approach</em> <sup><a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a></sup><em>, we studied the structural content of SARS-CoV-2. We found the highest density of double-stranded regions in the 5’ (nucleotides 1-253), membrane M protein (nucleotides 26523-27191), and the spike S protein (nucleotides 23000-24000). Strong match is observed between CROSS and Vienna analyses (centroid structures shown, indicating that regions with the highest structural content have the lowest free energies</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">Our analysis suggests presence of structural elements in SARS-CoV-2 that have evolved to interact with specific human proteins <sup><a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a></sup>. Our observation is based on the assumption that structured regions have an intrinsic propensity to recruit proteins <sup><a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a></sup>, which is supported by the fact that structured transcripts act as scaffolds for protein assembly <sup><a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a></sup>.</p></div><div id="sec-4" class="subsection"><h3>Structural comparisons reveal that the spike S region of SARS-CoV-2 is conserved among coronaviruses</h3><p id="p-15">We employed <em>CROSS</em>align <sup><a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">12</a></sup> to study the structural conservation of SARS-CoV-2 in different strains (<strong>Materials and Methods</strong>).</p><p id="p-16">In this analysis, we compared the Wuhan strain MN908947 with around 2800 other coronaviruses (data from NCBI) having as host human (<strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a></strong>) or other species (<strong>Supp. <a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1</a></strong>). When comparing SARS-CoV-2 with human coronaviruses (1387 strains, including SARS-CoV and MERS-CoV), we found that the most conserved region falls inside the spike S genomic locus (<strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a></strong>). More precisely, the conserved region is between nucleotides 23000 - 24000 and exhibits an intricate and stable secondary structure (RNA<em>fold</em> minimum free energy= −269 kcal/mol)<sup><a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a></sup>. High conservation of a structured regions suggests a functional activity that might be relevant for host infection.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="We employed the CROSSalign approach 12,54 to compare the Wuhan strain MN908947 with other coronaviruses (1387 strains, including SARS-CoV and MERS-CoV) indicates that the most conserved region falls inside the spike S genomic locus. The inset shows thermodynamic structural variability (positional entropy) within regions encompassing nucleotides 23000-24000 along with the centroid structure and free energy." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1501986072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;We employed the CROSSalign approach&lt;/em&gt; &lt;sup&gt;12,54&lt;/sup&gt; &lt;em&gt;to compare the Wuhan strain MN908947 with other coronaviruses (1387 strains, including SARS-CoV and MERS-CoV) indicates that the most conserved region falls inside the spike S genomic locus. The inset shows thermodynamic structural variability (positional entropy) within regions encompassing nucleotides 23000-24000 along with the centroid structure and free energy&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F2.medium.gif" width="440" height="408"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F2.medium.gif" width="440" height="408"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2.</span> <p id="p-17" class="first-child"><em>We employed the CROSSalign approach</em> <sup><a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">54</a></sup> <em>to compare the Wuhan strain MN908947 with other coronaviruses (1387 strains, including SARS-CoV and MERS-CoV) indicates that the most conserved region falls inside the spike S genomic locus. The inset shows thermodynamic structural variability (positional entropy) within regions encompassing nucleotides 23000-24000 along with the centroid structure and free energy</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">While the 3’ and 5’ of SARS-CoV-2 were shown to be relatively conserved in some beta-coronavirus <sup><a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a></sup>, they are highly variable in the entire set. However, the 3’ and 5’ are more structured in SARS-CoV-2 than other coronaviruses (average structural content for SARS-CoV-2 = 0.56 in the 5’ and 0.49 in the 3’; other coronaviruses 0.49 in the 5’ and 0.42 in the 3’).</p></div><div id="sec-5" class="subsection"><h3>Sequence and structural comparisons among SARS-CoV-2 strains indicate conservation of the ACE2 binding site and high variability in a region potentially interacting with sialic acids</h3><p id="p-19">To better investigate the sequence conservation of SARS-CoV-2, we compared 62 strains isolated form different countries during the pandemic (including China, USA, Japan, Taiwan, India, Brazil, Sweden, and Australia; data from NCBI and in VIPR <a href="http://www.viprbrc.org">www.viprbrc.org</a>; <strong>Materials and Methods</strong>). Our analysis aims to determine the relationship between structural content and sequence conservation.</p><p id="p-20">Using <em>Clustal W</em> for multiple sequence alignments <sup><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>, we observed general conservation of the coding regions with several <em>minima</em> in correspondence to areas between genes (<strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a></strong>). One highly conserved region is between nucleotides 23000 - 24000 in the spike S genomic locus, while sequences up- and down-stream are variable (<strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3A</a></strong>). We then used CROSS<em>align</em> <sup><a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">12</a></sup> to compare the structural content (<strong>Materials and Methods</strong>). High variability of structure is observed for both the 5’ and 3’ and for nucleotides between 21000 - 22000 as well as 24000 - 25000, associated with the S region (red bars in <strong><a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3A</a></strong>). The rest of the regions are significantly conserved at a structural level (p-value &lt; 0.0001; Fisher’s test).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sequence and structural comparison of human SARS-CoV-2 strains. (A) Strong sequence conservation (Clustal W multiple sequence alignments 35) is observed in coding regions, including the region between nucleotides 23000 and 24000 of spike S protein. High structural variability (red bars on top) is observed for both the UTRs and for nucleotides between 21000 and 22000 as well as 24000 and 25000, associated with the S region. The rest of the regions are significantly conserved at a structural level. (B) The sequence variability (Shannon entropy computed on Tcoffee multiple sequence alignments 29) in the spike S protein indicate conservation between amino-acids 460 and 520 (blue box) binding to the host receptor angiotensin-converting enzyme 2 ACE2. The region encompassing amino-acids 243 and 302 is highly variable and is implicated in sialic acids in MERS-CoV (red box). The S1 and S2 domains of Spike S protein are displayed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1501986072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Sequence and structural comparison of human SARS-CoV-2 strains. (&lt;strong&gt;A&lt;/strong&gt;) Strong sequence conservation (Clustal W multiple sequence alignments&lt;/em&gt; &lt;sup&gt;35&lt;/sup&gt;&lt;em&gt;) is observed in coding regions, including the region between nucleotides 23000 and 24000 of spike S protein. High structural variability (red bars on top) is observed for both the UTRs and for nucleotides between 21000 and 22000 as well as 24000 and 25000, associated with the S region. The rest of the regions are significantly conserved at a structural level. (&lt;strong&gt;B&lt;/strong&gt;) The sequence variability (Shannon entropy computed on Tcoffee multiple sequence alignments&lt;/em&gt; &lt;sup&gt;29&lt;/sup&gt;&lt;em&gt;) in the spike S protein indicate conservation between amino-acids 460 and 520 (blue box) binding to the host receptor angiotensin-converting enzyme 2 ACE2. The region encompassing amino-acids 243 and 302 is highly variable and is implicated in sialic acids in MERS-CoV (red box). The S1 and S2 domains of Spike S protein are displayed&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F3.medium.gif" width="440" height="385"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F3.medium.gif" width="440" height="385"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3.</span> <p id="p-21" class="first-child"><em>Sequence and structural comparison of human SARS-CoV-2 strains. (<strong>A</strong>) Strong sequence conservation (Clustal W multiple sequence alignments</em> <sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup><em>) is observed in coding regions, including the region between nucleotides 23000 and 24000 of spike S protein. High structural variability (red bars on top) is observed for both the UTRs and for nucleotides between 21000 and 22000 as well as 24000 and 25000, associated with the S region. The rest of the regions are significantly conserved at a structural level. (<strong>B</strong>) The sequence variability (Shannon entropy computed on Tcoffee multiple sequence alignments</em> <sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup><em>) in the spike S protein indicate conservation between amino-acids 460 and 520 (blue box) binding to the host receptor angiotensin-converting enzyme 2 ACE2. The region encompassing amino-acids 243 and 302 is highly variable and is implicated in sialic acids in MERS-CoV (red box). The S1 and S2 domains of Spike S protein are displayed</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">We then compared protein sequences coded by the spike S genomic locus (NCBI reference QHD43416) and found that both sequence (<strong><a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3A</a></strong>) and structure (<strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2</a></strong>) of nucleotides 23000 - 24000 are highly conserved. The region corresponds to amino acids 330-500 that contact the host receptor angiotensin-converting enzyme 2 (ACE2) <sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup> promoting infection and provoking lung injury <sup><a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>,<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup>. By contrast, the region upstream of the binding site receptor ACE2 and located in correspondence to the minimum of the structural profile at around nucleotides 22500-23000 (<strong><a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1</a></strong>) is highly variable <sup><a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a></sup>, as calculated with <em>Tcoffee</em> multiple sequence alignments <sup><a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">29</a></sup> (<strong><a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3A</a></strong>). This part of the spike S region corresponds to amino acids 243-302 that in MERS-CoV binds to sialic acids regulating infection through cell-cell membrane fusion (<strong><a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3B</a>;</strong>see related manuscript by E. Milanetti <em>et al</em>. “In-Silico evidence for two receptors based strategy of SARS-CoV-2”) <sup><a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>,<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>,<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup>. Our analysis suggests that the structural region between nucleotides 23000 and 24000 of Spike S region is conserved among coronaviruses (<strong><a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2</a></strong>) and the binding site for ACE2 has poor variation in human SARS-CoV-2 strains (<strong><a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3B</a></strong>). By contrast, the region upstream, potentially involved in adhesion to sialic acids, has almost poor structural content and varies significantly in the human population (<strong><a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig. 3B</a></strong>).</p></div><div id="sec-6" class="subsection"><h3>Analysis of human interactions with SARS-CoV-2 identifies proteins involved in viral replication and HIV infection</h3><p id="p-23">In order to obtain insights on how the virus is replicated in human cells, we predicted SARS-CoV-2 interactions with the whole RNA-binding human proteome. Following a protocol to study structural conservation in viruses <sup><a id="xref-ref-12-8" class="xref-bibr" href="#ref-12">12</a></sup>, we first divided the Wuhan sequence in 30 fragments of 1000 nucleotides each moving from the 5’ to 3’ and then calculated the protein-RNA interactions of each fragment with <em>cat</em>RAPID <em>omics</em> (3340 canonical and non-canonical RNA-binding proteins, or RBPs, for a total 102000 interactions) <sup><a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a></sup>. Proteins such as PTBP1 showed the highest interaction propensity (or Z-score; <strong>Materials and Methods</strong>) at the 5’ while others such as HNRNPQ showed the highest interaction propensity at the 3’, in agreement with previous studies on coronaviruses <sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>.</p><p id="p-24">For each fragment, we predicted the most significant interactions by filtering according to the Z score. We used three different thresholds in ascending order of stringency: Z ≥ 1.50, 1.75 and 2 respectively. Importantly, we removed from the list proteins that were predicted to interact promiscuously with different fragments. Fragment 1 corresponds to the 5’ and is the most contacted by RBPs (around 120 with Z≥2 high-confidence interactions; <strong><a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A</a></strong>), which is in agreement with the observation that highly structured regions attract a large number of proteins <sup><a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">13</a></sup>. Indeed, the 5’ contains a leader sequence and the untranslated region with multiple stem loop structures that control RNA replication and transcription <sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>,<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Characterization of protein interactions with SARS-CoV-2 RNA, (A) Number of RBP interactions for different SARS-CoV-2 regions (colours indicate different catRAPID 16,17 confidence levels: Z=1.5 or low Z=1.75 or medium and Z=2.0 or high; regions with scores lower than Z=1.5 are omitted); (B) Enrichment of viral processes in the 5&#x2019; of SARS-CoV-2 (precision = term precision calculated from the GO graph structure lvl = depth of the term; go_term = GO term identifier, with link to term description at AmiGO website; description = Textual label for the term; e/d = e signifies enrichment of the term, d signifies depletion compared to the population; %_set = coverage on the provided set - how much of the set is annotated with the GO?; %_pop = coverage of the same term on the population; p_bonf = p-value of the enrichment. To correct for multiple testing bias, we are applying Bonferroni correction) 35; (C) Viral processes are the third largest cluster identified in our analysis; (D) Protein interactions with the 5&#x2019; of SARS-CoV-2 RNA (inner circle) and associations with other human genes retrieved from literature (green: genetic associations; pink: physical associations); (E) Number of RBP interactions identified by Gordon et al. 49 for different SARS-CoV-2 regions (see panel A for reference)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1501986072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Characterization of protein interactions with SARS-CoV-2 RNA, (&lt;strong&gt;A&lt;/strong&gt;) Number of RBP interactions for different SARS-CoV-2 regions (colours indicate different catRAPID&lt;/em&gt; &lt;sup&gt;16,17&lt;/sup&gt; &lt;em&gt;confidence levels: Z=1.5 or low Z=1.75 or medium and Z=2.0 or high; regions with scores lower than Z=1.5 are omitted); (&lt;strong&gt;B&lt;/strong&gt;) Enrichment of viral processes in the 5&#x2019; of SARS-CoV-2 (precision = term precision calculated from the GO graph structure lvl = depth of the term; go_term = GO term identifier, with link to term description at AmiGO website; description = Textual label for the term; e/d =&lt;/em&gt; e &lt;em&gt;signifies enrichment of the term,&lt;/em&gt; d &lt;em&gt;signifies depletion compared to the population; %_set = coverage on the provided set - how much of the set is annotated with the GO?; %_pop = coverage of the same term on the population; p_bonf = p-value of the enrichment. To correct for multiple testing bias, we are applying Bonferroni correction)&lt;/em&gt; &lt;sup&gt;35&lt;/sup&gt;&lt;em&gt;; (&lt;strong&gt;C&lt;/strong&gt;) Viral processes are the third largest cluster identified in our analysis; (&lt;strong&gt;D&lt;/strong&gt;) Protein interactions with the 5&#x2019; of SARS-CoV-2 RNA (inner circle) and associations with other human genes retrieved from literature (green: genetic associations; pink: physical associations); (&lt;strong&gt;E&lt;/strong&gt;) Number of RBP interactions identified by Gordon et al.&lt;/em&gt; &lt;sup&gt;49&lt;/sup&gt; &lt;em&gt;for different SARS-CoV-2 regions (see panel A for reference)&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F4.medium.gif" width="341" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F4.medium.gif" width="341" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.03.28.013789/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4.</span> <p id="p-25" class="first-child"><em>Characterization of protein interactions with SARS-CoV-2 RNA, (<strong>A</strong>) Number of RBP interactions for different SARS-CoV-2 regions (colours indicate different catRAPID</em> <sup><a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a></sup> <em>confidence levels: Z=1.5 or low Z=1.75 or medium and Z=2.0 or high; regions with scores lower than Z=1.5 are omitted); (<strong>B</strong>) Enrichment of viral processes in the 5’ of SARS-CoV-2 (precision = term precision calculated from the GO graph structure lvl = depth of the term; go_term = GO term identifier, with link to term description at AmiGO website; description = Textual label for the term; e/d =</em> e <em>signifies enrichment of the term,</em> d <em>signifies depletion compared to the population; %_set = coverage on the provided set - how much of the set is annotated with the GO?; %_pop = coverage of the same term on the population; p_bonf = p-value of the enrichment. To correct for multiple testing bias, we are applying Bonferroni correction)</em> <sup><a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a></sup><em>; (<strong>C</strong>) Viral processes are the third largest cluster identified in our analysis; (<strong>D</strong>) Protein interactions with the 5’ of SARS-CoV-2 RNA (inner circle) and associations with other human genes retrieved from literature (green: genetic associations; pink: physical associations); (<strong>E</strong>) Number of RBP interactions identified by Gordon et al.</em> <sup><a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a></sup> <em>for different SARS-CoV-2 regions (see panel A for reference)</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The interactome of each fragment was then analysed using <em>clever</em>GO, a tool for GO enrichment analysis <sup><a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">35</a></sup>. Proteins interacting with fragments 1, 2 and 29 were associated with annotations related to viral processes (<strong><a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4B</a>; Supp. Table 1</strong>). Considering the three thresholds applied (<strong>Materials and Methods</strong>), we found 22 viral proteins for fragment 1, 2 proteins for fragment 2 and 11 proteins for fragment 29 (<strong><a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4C</a></strong>). Among the high-confidence interactors of fragment 1, we discovered RBPs involved in positive regulation of viral processes and viral genome replication, such as double-stranded RNA-specific editase 1 ADARB1 (Uniprot P78563 <sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup>) and 2-5A-dependent ribonuclease RNASEL (Q05823). We also identified proteins related to the establishment of integrated proviral latency, including X-ray repair cross-complementing protein 5 XRCC5 (P13010) and X-ray repair cross-complementing protein 6 XRCC6 (P12956; <strong><a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4D</a></strong>).</p><p id="p-27">Importantly, we found proteins related to defence response to viruses, such as ATP-dependent RNA helicase DDX1 (Q92499), are involved in the negative regulation of viral genome replication. Some proteins are listed as DNA binding proteins such as Cyclin-T1 CCNT1 (Uniprot code O60563 <sup><a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a></sup>), Zinc finger protein 175 ZNF175 (Q9Y473), while Prospero homeobox protein 1 PROX1 (Q92786) were included because they could have potential RNA-binding ability (<strong><a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4D</a></strong>) <sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup>. As for fragment 2, we found two canonical RBPs: E3 ubiquitin-protein ligase TRIM32 (Q13049) and E3 ubiquitin-protein ligase TRIM21 (P19474), which are listed as negative regulators of viral release from host cell, negative regulators of viral transcription and positive regulators of viral entry into host cells. Finally, for fragment 29, 10 of the 11 viral proteins found are members of the <em>Gag polyprotein family</em>, that perform different tasks during HIV assembly, budding, maturation. More than a simple scaffold protein forming the viral core, Gag proteins are recognized as elements able to select viral and host proteins as they traffic to the cell membrane (<strong>Supp. Table 1</strong>) <sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup>.</p><p id="p-28">Analysis of functional annotations carried out with <em>GeneMania</em> <sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup> reveals that proteins interacting with the 5’ of SARS-CoV-2 RNA are associated with regulatory pathways involving NOTCH2, MYC and MAX that have been previously connected to viral infection processes (<strong><a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4B</a></strong>) <sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>,<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup>. Interestingly, some of the proteins, including DDX1, ZNF175 and CCNT1 for fragment 1 and TRIM32 for fragment 2, are reported to be necessary for HIV functions and replication inside the cell. The roles of these proteins in the replication of a retrovirus such as HIV are expected to be different from those associated with SARS-CoV-2, yet it has been reported that SARS-CoV-2 represses host gene expression through a number of unknown mechanisms, which could also involve sequestration of transcriptional elements such as Cyclin-T1 CCNT1 <sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup>. DDX1 is required for HIV-1 Rev function as well as for HIV-1 and coronavirus IBV replication and it binds to the RRE sequence of HIV-1 RNAs <sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>,<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup>. ZNF175 is relatively uncharacterized reported to interfere with HIV-1 replication by suppressing Tat-induced viral LTR promoter activity <sup><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup>. Finally, CCNT1 is 7SK snRNA binding and regulates transactivation domain of the viral nuclear transcriptional activator, Tat <sup><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>,<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></sup>. In addition, TRIM32 (fragment 2) is a well-defined Tat binding protein and, more specifically, it binds to the activation domain of HIV-1 Tat and can also interact with the HIV-2 and EIAV Tat proteins <em>in vivo</em> <sup><a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a></sup>.</p></div><div id="sec-7" class="subsection"><h3>Analysis of interactions with SARS-CoV-2 Open Reading Frames identifies additional interactions involved in HIV infection</h3><p id="p-29">Recently, Gordon <em>et al</em>. reported a list of human proteins binding to Open Reading Frames (ORFs) translated from SARS-CoV-2 <sup><a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">49</a></sup>. Identified through affinity purification followed by mass spectrometry quantification, 332 proteins from HEK-293T cells interact with viral ORF peptides. By selecting 274 proteins binding at the 5’ with Z score ≥1.5 (<strong>Supp. Table 1</strong>), of which 140 are exclusively interacting with fragment 1 (<strong><a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig. 4B</a></strong>), we found that 8 are also reported in the list by Gordon <em>et al</em>. <sup><a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">49</a></sup>, which indicates significant enrichment (representation factor of 2.5; p-value of 0.02; hypergeometric test with human proteome in background). The fact that our list of protein-RNA binding partners contains elements identified also in the protein-protein network analysis is not surprising, as ribonucleoprotein complexes evolve together <sup><a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">13</a></sup> and their components sustain each other activities through different types of interactions <sup><a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a></sup>.</p><p id="p-30">We note that out of 332 interactions, 60 are RBPs (as reported in Uniprot <sup><a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">36</a></sup>), which represents a considerable fraction (20%), considering that there are around 1500 RBPs in the human proteome (6%) and fully justified by the fact that they involve association with viral RNAs. Comparing the RBPs present in Gordon <em>et al.</em> <sup><a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">49</a></sup> and those present in our list (79 as reported in Uniprot), we found an overlap of 6 proteins (representation factor = 26.5; p-value &lt; 10<sup>−8</sup>; hypergeometric test), including: Janus kinase and microtubule-interacting protein 1 JAKMIP1 (Q96N16), A-kinase anchor protein 8 AKAP8 (O43823) and A-kinase anchor protein 8-like AKAP8L (Q9ULX6), which in case of HIV-1 infection is involved as a DEAD/H-box RNA helicase binding <sup><a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a></sup>, signal recognition particle subunit SRP72 (O76094), binding to the 7S RNA in presence of SRP68, La-related protein 7, LARP7 (Q4G0J3) and La-related protein 4B LARP4B (Q92615), which are part of a system for transcriptional regulation acting by means of the 7SK RNP system <sup><a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a></sup> (<strong><a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig. 4E</a>; Supp. Table 2</strong>). We speculate that sequestration of elements binding to the 7S RNA is part of a viral program aiming to host genes <sup><a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">42</a></sup>.</p><p id="p-31">Moreover, by analysing the RNA interaction potential of all the 332 proteins by Gordon <em>et al.</em> <sup><a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">49</a></sup>, <em>cat</em>RAPID identified 38 putative binders at the 5’ (Z score ≥ 1.5; 27 occurring exclusively in the 5’ and not in other regions of the RNA) <sup><a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">16</a></sup>, including Serine/threonine-protein kinase TBK1 (Q9UHD2), among which 10 RBPs (as reported in Uniprot) such as: Splicing elements U3 small nucleolar ribonucleoprotein protein MPP10 (O00566) and Pre-mRNA-splicing factor SLU7 (O95391), snRNA methylphosphate capping enzyme MEPCE involved in negative regulation of transcription by RNA polymerase II 7SK (Q7L2J0) <sup><a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a></sup>, Nucleolar protein 10 NOL10 (Q9BSC4) and protein kinase A Radixin RDX (P35241; in addition to those mentioned above; <strong>Supp. Table 2</strong>).</p></div><div id="sec-8" class="subsection"><h3>HIV-related RBPs are significantly enriched in the 5’ interactions</h3><p id="p-32">In the list of 274 proteins predicted to bind at the 5’ (fragment 1) with Z score ≥1.5, we found 10 hits reported to be involved in HIV (<strong>Supp. Table 3</strong>), which is a highly significant enrichment (p-value=0.0004; Fisher’s exact test), considering that the total number of HIV-related proteins is 35 in the whole <em>cat</em>RAPID library (3340 elements). The complete list of proteins includes ATP-dependent RNA helicase DDX1 (Q92499 also involved in Coronavirus replication <sup><a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a>,<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a></sup>), ATP-dependent RNA helicase DDX3X (O00571 also involved in Dengue and Zika Viruses), Tyrosine-protein kinase HCK (P08631, nucleotide binding), Arf-GAP domain and FG repeat-containing protein 1 (P52594), Double-stranded RNA-specific editase 1 ADARB1 (P78563), Insulin-like growth factor 2 mRNA-binding protein 1 IGF2BP1 (Q9NZI8), A-kinase anchor protein 8-like AKAP8L (Q9ULX6; its partner AKAP8 is also found in Gordon <em>et al.</em> <sup><a id="xref-ref-49-6" class="xref-bibr" href="#ref-49">49</a></sup>) Cyclin-T1 CCNT1 (O60563; DNA-binding) and Forkhead box protein K2 FOXK2 (Q01167; DNA-binding; <strong>Supp. Table 3</strong>).</p><p id="p-33">Smaller enrichments were found for proteins related to Hepatitis B virus (HBV; p-value=0.01; 3 hits out of 7 in the whole <em>cat</em>RAPID library; Fisher’s exact test), Nuclear receptor subfamily 5 group A member 2 NR5A2 (DNA-binding; O00482), Interferon-induced, double-stranded RNA-activated protein kinase EIF2AK2 (P19525), and SRSF protein kinase 1 SRPK1 (Q96SB4) as well as Influenza (p-value=0.03; 2 hits out of 4; Fisher’s exact test), Synaptic functional regulator FMR1 (Q06787) and RNA polymerase-associated protein RTF1 homologue (Q92541; <strong>Supp. Table 3</strong>). By contrast, no significant enrichments were found for other viruses such as for instance Ebola.</p><p id="p-34">Interestingly, specific drugs are reported in ChEMBL<sup><a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a></sup> for HIV-related proteins ATP-dependent RNA helicase DDX1 (CHEMBL2011807), ATP-dependent RNA helicase DDX3X (CHEMBL2011808), Cyclin-T1 CCNT1 (CHEMBL2348842), and Tyrosine-protein kinase HCK (CHEMBL2408778)<sup><a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">53</a></sup>, as well as HVB-related proteins Nuclear receptor subfamily 5 group A member 2 NR5A2 (CHEMBL3544), Interferon-induced, double-stranded RNA-activated protein kinase EIF2AK2 (CHEMBL5785) and SRSF protein kinase 1 SRPK1 (CHEMBL4375), which could be a starting point for further investigations.</p></div></div><div class="section" id="sec-9"><h2 class="">CONCLUSIONS</h2><p id="p-35">Our study is motivated by the need to identify molecular mechanisms involved in Covid-19 spreading. Using advanced computational approaches, we investigated the structural content of SARS-CoV-2 RNA and predicted human proteins that bind it.</p><p id="p-36">We employed <em>CROSS</em> <sup><a id="xref-ref-12-9" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">54</a></sup> to compare the structural properties of 2800 coronaviruses and identified elements conserved in SARS-CoV-2 strains. The regions containing the highest amount of structure are the 5’ as well as glycoproteins spike S and membrane M.</p><p id="p-37">We found that the spike S protein domain encompassing amino acids 330-500 is highly conserved across SARS-CoV-2 strains. This result suggests that spike S must have evolved to specifically interact with its host partner ACE2 <sup><a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a></sup> and mutations increasing the binding affinity are highly infrequent. As the nucleic acids encoding for this region are enriched in double-stranded content, we speculate that the structure might attract host regulatory elements, which further constrains its variability. The fact that the ACE2 receptor binding site is conserved among the SARS-CoV-2 strains suggests that a specific drug can be designed to prevent host interaction and thus infection, which could work for a large number of coronaviruses.</p><p id="p-38">By contrast, the highly variable region at amino acids 243-302 in spike S protein corresponds to the binding site of sialic acids in MERS-CoV (see manuscript by E. Milanetti <em>et al</em>. “In-Silico evidence for two receptors based strategy of SARS-CoV-2”) <sup><a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">9</a>,<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a></sup> and could play a role in infection <sup><a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a></sup>. The fact that the binding region changes in the different strains might indicate a variety of binding affinities for sialic acids, which could provide clues on the specific responses in the human population. Interestingly, the sialic acid binding site is absent in SARS-CoV but present in MERS-CoV, which indicates that it must have evolved recently.</p><p id="p-39">Both our sequence and structural analyses of spike S protein indicate that human engineering of SARS-CoV-2 is highly unlikely.</p><p id="p-40">Using <em>cat</em>RAPID <sup><a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a></sup> we computed &gt;100000 protein interactions with SARS-CoV-2 and found that the highly structured region at the 5’ has the largest number of protein partners including ATP-dependent RNA helicase DDX1, which has been previously reported to be essential for HIV-1 and coronavirus IBV replication <sup><a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">43</a>,<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">44</a></sup>, double-stranded RNA-specific editase 1 ADARB1, which catalyses the hydrolytic deamination of adenosine to inosine and might take part in the chemical modification of SARS-CoV-2 RNA <sup><a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">42</a></sup>. Other relevant interactions are XRCC5 and XRCC6 members of the HDP-RNP complex interacting with ATP-dependent RNA helicase DHX9 <sup><a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a></sup> and 2-5A-dependent ribonuclease RNASEL that has antiviral effects through a combination of cleavage of single-stranded viral RNAs, inhibition of protein synthesis, induction of apoptosis, and induction of antiviral genes <sup><a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a></sup>.</p><p id="p-41">A significant overlap exists with the list of protein interactions reported by Gordon <em>et al</em>. <sup><a id="xref-ref-49-7" class="xref-bibr" href="#ref-49">49</a></sup>, and among the candidate partners we identified AKAP8L, involved as a DEAD/H-box RNA helicase binding in HIV infection <sup><a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">50</a></sup>. In general, proteins involved in the replication of retroviruses such as HIV are expected to play a different role in mechanisms related to SARS-CoV-2 that uses its own RNA-dependent RNA polymerase, yet it must be considered that SARS-CoV-2 represses host gene expression through a number of unknown mechanisms, which could imply sequestration of transcriptional components, such as specific polymerase II genes and splicing factors <sup><a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">42</a></sup>. Thus, the link to HIV and other viruses such as HBV and Influenza could be key to identify targets for the repurposing of drugs for treatment of SARS-CoV-2 infection <sup><a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">53</a></sup>.</p><p id="p-42">In conclusion, we hope that our analysis would be useful to the scientific community to identify virus-host interactions and block SARS-CoV-2 spreading.</p></div><div class="section" id="sec-11"><h2 class="">Contributions</h2><p id="p-45">GGT and RDP conceived the study. AV carried out <em>cat</em>RAPID analysis of protein interactions, RDP calculated <em>CROSS</em> structures of coronaviruses, GGT, MM and EM performed and analysed sequence alignments. AV, RDP and GGT wrote the paper.</p></div><div class="section" id="sec-12"><h2 class="">MATERIALS AND METHODS</h2><div class="section" id="sec-13"><h3>Structure prediction</h3><p id="p-46">We predicted the secondary structure of transcripts using <em>CROSS</em> (Computational Recognition of Secondary Structure <sup><a id="xref-ref-12-10" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">54</a></sup>. <em>CROSS</em> was developed to perform high-throughput RNA profiling. The algorithm predicts the structural profile (single- and double-stranded state) at single-nucleotide resolution using sequence information only and without sequence length restrictions (scores &gt; 0 indicate double stranded regions). We used the <em>Vienna</em> method <sup><a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">25</a></sup> to further investigate the RNA secondary structure of minima and maxima identified with <em>CROSS</em> <sup><a id="xref-ref-12-11" class="xref-bibr" href="#ref-12">12</a></sup>.</p></div><div class="section" id="sec-14"><h3>Structural conservation</h3><p id="p-47">We used <em>CROSS</em>align <sup><a id="xref-ref-12-12" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-54-5" class="xref-bibr" href="#ref-54">54</a></sup> an algorithm based on Dynamic Time Warping (DTW), to check and evaluate the structural conservation between different viral genomes <sup><a id="xref-ref-12-13" class="xref-bibr" href="#ref-12">12</a></sup>. CROSS<em>align</em> was previously employed to study the structural conservation of ~5000 HIV genomes. SARS-CoV-2 fragments (1000 nt, not overlapping) were searched inside other complete genomes using the OBE (open begin and end) module, in order to search a small profile inside a larger one. The lower the structural distance, the higher the structural similarities (with a minimum of 0 for almost identical secondary structure profiles). The significance is assessed as in the original publication <sup><a id="xref-ref-12-14" class="xref-bibr" href="#ref-12">12</a></sup>.</p></div><div class="section" id="sec-15"><h3>Sequence collection</h3><p id="p-48">The FASTA sequences of the complete genomes of SARS-CoV-2 were downloaded from Virus Pathogen Resource (VIPR; <a href="http://www.viprbrc.org">www.viprbrc.org</a>), for a total of 62 strains. Regarding the overall coronaviruses, the sequences were downloaded from NCBI selecting only complete genomes, for a total of 2862 genomes. The reference Wuhan sequence with available annotation (EPI_ISL_402119) was downloaded from Global Initiative on Sharing All Influenza Data. (GISAID <a href="https://www.gisaid.org/">https://www.gisaid.org/</a>).</p></div><div class="section" id="sec-16"><h3>Protein-RNA interaction prediction</h3><p id="p-49">Interactions between each fragment of target sequence and the human proteome were predicted using <em>cat</em>RAPID <em>omics</em> <sup><a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">17</a></sup>, an algorithm that estimates the binding propensity of protein-RNA pairs by combining secondary structure, hydrogen bonding and van der Waals contributions. As reported in a recent analysis of about half a million of experimentally validated interactions <sup><a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">31</a></sup>, the algorithm is able to separate interacting vs non-interacting pairs with an area under the ROC curve of 0.78.</p><p id="p-50">The complete list of interactions between the 30 fragments and the human proteome is available at <a href="http://crg-webservice.s3.amazonaws.com/submissions/2020-03/252523/output/index.html?unlock=f6ca306af0">http://crg-webservice.s3.amazonaws.com/submissions/2020-03/252523/output/index.html?unlock=f6ca306af0</a>. The output then is filtered according to the Z-score column, which is the interaction propensity normalised by the mean and standard deviation calculated over the reference RBP set (<a href="http://s.tartaglialab.com/static_files/shared/faqs.html#4">http://s.tartaglialab.com/static_files/shared/faqs.html#4</a>). We used three different thresholds in ascending order of stringency: Z greater or equal than 1.50, 1.75 and 2 respectively and for each threshold we then selected the proteins that were unique for each fragment for each threshold.</p></div><div class="section" id="sec-17"><h3>GO terms analysis</h3><p id="p-51"><em>clever</em>GO <sup><a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">35</a></sup>, an algorithm for the analysis of Gene Ontology annotations, was used to determine which fragments present enrichment in GO terms related to viral processes. Analysis of functional annotations was performed in parallel with <em>GeneMania</em> <sup><a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a></sup>.</p></div><div class="section" id="sec-18"><h3>RNA and protein alignments</h3><p id="p-52">We sued <em>Clustal W</em> <sup><a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a></sup> for 62 SARS-CoV-2 strains alignments and <em>Tcoffee</em> <sup><a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">29</a></sup> for spike S proteins alignments. The variability in the spike S region was measured by computing Shannon entropy on translated RNA sequences. The Shannon entropy is computed as follows:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/09/2020.03.28.013789/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/09/2020.03.28.013789/embed/graphic-5.gif"/></noscript></span></span>
</p><p id="p-53">Where <em>a</em> correspond to the amino acid at the position <em>i</em> and P(a,i) is the frequency of a certain amino-acid <em>a</em> at position <em>i</em> of the sequence. Low entropy indicates poorly variability: if P(a,x) = 1 for one <em>a</em> and 0 for the rest, then S(x) =0. By contrast, if the frequencies of all amino acids are equally distributed, the entropy reaches its maximum possible value.</p></div></div><div class="section" id="sec-19"><h2 class="">SUPPLEMENTARY MATERIAL</h2><p id="p-54"><em><strong>Supp. Figure 1.</strong> We employed CROSSalign</em> <sup><a id="xref-ref-12-15" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-54-6" class="xref-bibr" href="#ref-54">54</a></sup> <em>was to compare the Wuhan strain MN908947 with other coronaviruses (2800 strains, including SARS-CoV, MERS-CoV and coronaviruses having as host other species, such as bats). The result highlights that the most conserved region falls inside the spike S genomic locus</em>.</p><p id="p-55"><em><strong>Supp. Table 1.</strong> 1) catRAPID</em> <sup><a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">17</a></sup> <em>score for interactions with fragment 1; 2) GO</em> <sup><a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">35</a></sup> <em>and Uniprot annotations of viral proteins interacting with fragment 1 and ; 3<strong>)</strong> catRAPID score for interactions with fragment 2; 4) GO annotations of viral proteins interacting with fragment 2; 5<strong>)</strong> catRAPID score for interactions with fragment 29; 6) GO annotations of viral proteins interacting with fragment 29;</em></p><p id="p-56"><em><strong>Supp. Table 2.</strong> RBP interactions from Gordon et al.</em> <sup><a id="xref-ref-49-8" class="xref-bibr" href="#ref-49">49</a></sup> <em>classified according to catRAPID scores. GO</em> <sup><a id="xref-ref-35-6" class="xref-bibr" href="#ref-35">35</a></sup> <em>and Uniprot</em> <sup><a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">36</a></sup> <em>annotations are reported</em>.</p><p id="p-57"><em><strong>Supp. Table 3</strong>. RBPs significantly enriched in the 5’ interactions and HIV, HBV and Influenza</em></p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-43">The authors would like to thank Jakob Rupert, Dr. Mattia Miotto, Dr Lorenzo Di Rienzo, Dr. Alexandros Armaos, Dr. Alessandro Dasti, Dr. Elias Bechara, Dr. Claudia Giambartolomei and Dr. Elsa Zacco for discussions.</p><p id="p-44">The research leading to these results has been supported by European Research Council (RIBOMYLOME_309545 to GGT, ASTRA_855923 to GGT), Spanish Ministry of Economy and Competitiveness BFU2017-86970-P, the H2020 projects INFORE_825080 and IASIS_727658, as well as the collaboration with Peter St. George-Hyslop financed by the Wellcome Trust.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-4">- update the list of protein partners - reanalysed HIV interactome</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2>References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.1" data-doi="10.1056/NEJMoa2001017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">382</span>, <span class="cit-fpage">727</span>–<span class="cit-lpage">733</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed.%26rft.volume%253D382%26rft.spage%253D727%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001017%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.2" data-doi="10.1002/lt.25756"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">D’Antiga</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Coronaviruses and immunosuppressed patients. The facts during the third epidemic</span>. <abbr class="cit-jnl-abbrev">Liver Transplant. Off. Publ. Am. Assoc. Study Liver Dis. Int. Liver Transplant. Soc.</abbr> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/lt.25756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLiver%2BTransplant.%2BOff.%2BPubl.%2BAm.%2BAssoc.%2BStudy%2BLiver%2BDis.%2BInt.%2BLiver%2BTransplant.%2BSoc.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Flt.25756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/lt.25756&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.28.013789v3.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cascella</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rajnik</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cuomo</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dulebohn</span>,  <span class="cit-name-given-names">S. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Di Napoli</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-chapter-title">Features, Evaluation and Treatment Coronavirus (COVID-19)</span>. in <span class="cit-source">StatPearls</span> (<span class="cit-publ-name">StatPearls Publishing</span>, <span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.4" data-doi="10.1038/nature12711"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ge</span>,  <span class="cit-name-given-names">X.-Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">503</span>, <span class="cit-fpage">535</span>–<span class="cit-lpage">538</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D503%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12711%26rft_id%253Dinfo%253Apmid%252F24172901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24172901&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327464200045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.5" data-doi="10.1016/j.virol.2006.02.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Follis</span>,  <span class="cit-name-given-names">K. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">York</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Nunberg</span>,  <span class="cit-name-given-names">J. H.</span></span> <span class="cit-article-title">Furin cleavage of the SARS coronavirus spike glycoprotein enhances cell-cell fusion but does not affect virion entry</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">350</span>, <span class="cit-fpage">358</span>–<span class="cit-lpage">369</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DFollis%26rft.auinit1%253DK.%2BE.%26rft.volume%253D350%26rft.issue%253D2%26rft.spage%253D358%26rft.epage%253D369%26rft.atitle%253DFurin%2Bcleavage%2Bof%2Bthe%2BSARS%2Bcoronavirus%2Bspike%2Bglycoprotein%2Benhances%2Bcell-cell%2Bfusion%2Bbut%2Bdoes%2Bnot%2Baffect%2Bvirion%2Bentry.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2006.02.003%26rft_id%253Dinfo%253Apmid%252F16519916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2006.02.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16519916&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238785900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.6" data-doi="10.1101/2020.02.17.951335"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xiao</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Isolation and Characterization of 2019-nCoV-like Coronavirus from Malayan Pangolins</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> 2020.02.17.951335 (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.02.17.951335</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.02.17.951335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMi4xNy45NTEzMzV2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDMuMjguMDEzNzg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.7" data-doi="10.1038/s41594-019-0334-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">Y.-J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Structures of MERS-CoV spike glycoprotein in complex with sialoside attachment receptors</span>. <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">1151</span>–<span class="cit-lpage">1157</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BStruct.%2BMol.%2BBiol.%26rft.volume%253D26%26rft.spage%253D1151%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41594-019-0334-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41594-019-0334-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.8" data-doi="10.1038/nature16988"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>,  <span class="cit-name-given-names">A. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cryo-electron microscopy structure of a coronavirus spike glycoprotein trimer</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">531</span>, <span class="cit-fpage">114</span>–<span class="cit-lpage">117</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D531%26rft.spage%253D114%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature16988%26rft_id%253Dinfo%253Apmid%252F26855426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature16988&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26855426&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.9" data-doi="10.1073/pnas.1712592114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification of sialic acid-binding function for the Middle East respiratory syndrome coronavirus spike glycoprotein</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A.</abbr> <span class="cit-vol">114</span>, <span class="cit-fpage">E8508</span>–<span class="cit-lpage">E8517</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1712592114%26rft_id%253Dinfo%253Apmid%252F28923942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzQwL0U4NTA4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wMy4yOC4wMTM3ODkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.10" data-doi="10.1016/j.virusres.2015.02.025"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Leibowitz</span>,  <span class="cit-name-given-names">J. L.</span></span> <span class="cit-article-title">The Structure and Functions of Coronavirus Genomic 3’ and 5’ Ends</span>. <abbr class="cit-jnl-abbrev">Virus Res.</abbr> <span class="cit-vol">206</span>, <span class="cit-fpage">120</span>–<span class="cit-lpage">133</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes.%26rft.volume%253D206%26rft.spage%253D120%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2015.02.025%26rft_id%253Dinfo%253Apmid%252F25736566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2015.02.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25736566&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.11" data-doi="10.1093/nar/gkw1094"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Delli Ponti</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marti</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Armaos</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Tartaglia</span>,  <span class="cit-name-given-names">G. G.</span></span> <span class="cit-article-title">A high-throughput approach to profile RNA structure</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>, <span class="cit-fpage">e35</span>–<span class="cit-lpage">e35</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D45%26rft.spage%253De35%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1094&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Delli Ponti</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Armaos</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marti</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gian Gaetano</span>  <span class="cit-name-given-names">Tartaglia</span></span>. <span class="cit-article-title">A Method for RNA Structure Prediction Shows Evidence for Structure in lncRNAs</span>. <abbr class="cit-jnl-abbrev">Front. Mol. Biosci.</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">111</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMol.%2BBiosci.%26rft.volume%253D5%26rft.spage%253D111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanchez de Groot</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">RNA structure drives interaction with proteins</span>. <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">3246</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D10%26rft.spage%253D3246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cid-Samper</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">An Integrative Study of Protein-RNA Condensates Identifies Scaffolding RNAs and Reveals Players in Fragile X-Associated Tremor/Ataxia Syndrome</span>. <abbr class="cit-jnl-abbrev">Cell Rep.</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">3422</span>–<span class="cit-lpage">3434.e7</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep.%26rft.volume%253D25%26rft.spage%253D3422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.15" data-doi="10.1038/s41594-019-0223-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cerase</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Phase separation drives X-chromosome inactivation: a hypothesis</span>. <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">331</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BStruct.%2BMol.%2BBiol.%26rft.volume%253D26%26rft.spage%253D331%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41594-019-0223-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41594-019-0223-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Agostini</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">catRAPID omics: a web server for large-scale prediction of protein-RNA interactions</span>. <abbr class="cit-jnl-abbrev">Bioinforma. Oxf. Engl.</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">2928</span>–<span class="cit-lpage">2930</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinforma.%2BOxf.%2BEngl.%26rft.volume%253D29%26rft.spage%253D2928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cirillo</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Quantitative predictions of protein interactions with long noncoding RNAs</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">5</span>–<span class="cit-lpage">6</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D14%26rft.spage%253D5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.18" data-doi="10.1038/nmeth.1611"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bellucci</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Agostini</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masin</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Tartaglia</span>,  <span class="cit-name-given-names">G. G.</span></span> <span class="cit-article-title">Predicting protein associations with long noncoding RNAs</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">444</span>–<span class="cit-lpage">445</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DBellucci%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253D444%26rft.epage%253D445%26rft.atitle%253DPredicting%2Bprotein%2Bassociations%2Bwith%2Blong%2Bnoncoding%2BRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1611%26rft_id%253Dinfo%253Apmid%252F21623348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1611&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21623348&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291031800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.19" data-doi="10.1093/nar/gky967"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Armaos</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Tartaglia</span>,  <span class="cit-name-given-names">G. G.</span></span> <span class="cit-article-title">RNAct: Protein–RNA interaction predictions for model organisms with supporting experimental data</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gky967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky967&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.20" data-doi="10.1186/1471-2180-3-20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kliger</span>,  <span class="cit-name-given-names">Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Levanon</span>,  <span class="cit-name-given-names">E. Y.</span></span> <span class="cit-article-title">Cloaked similarity between HIV-1 and SARS-CoV suggests an anti-SARS strategy</span>. <abbr class="cit-jnl-abbrev">BMC Microbiol.</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">20</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BMicrobiology%26rft.stitle%253DBMC%2BMicrobiology%26rft.aulast%253DKliger%26rft.auinit1%253DY.%26rft.volume%253D3%26rft.spage%253D20%26rft.epage%253D20%26rft.atitle%253DCloaked%2Bsimilarity%2Bbetween%2BHIV-1%2Band%2BSARS-CoV%2Bsuggests%2Ban%2Banti-SARS%2Bstrategy.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2180-3-20%26rft_id%253Dinfo%253Apmid%252F14499001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2180-3-20&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14499001&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.21" data-doi="10.1038/360358a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hallenberger</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Inhibition of furin-mediated cleavage activation of HIV-1 glycoprotein gp160</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">360</span>, <span class="cit-fpage">358</span>–<span class="cit-lpage">361</span> (<span class="cit-pub-date">1992</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHallenberger%26rft.auinit1%253DS.%26rft.volume%253D360%26rft.issue%253D6402%26rft.spage%253D358%26rft.epage%253D361%26rft.atitle%253DInhibition%2Bof%2Bfurin-mediated%2Bcleavage%2Bactivation%2Bof%2BHIV-1%2Bglycoprotein%2Bgp160.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F360358a0%26rft_id%253Dinfo%253Apmid%252F1360148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/360358a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1360148&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JZ63000056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.22" data-doi="10.1128/JVI.01248-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Glowacka</span>,  <span class="cit-name-given-names">I.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Differential downregulation of ACE2 by the spike proteins of severe acute respiratory syndrome coronavirus and human coronavirus NL63</span>. <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">84</span>, <span class="cit-fpage">1198</span>–<span class="cit-lpage">1205</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01248-09%26rft_id%253Dinfo%253Apmid%252F19864379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg0LzIvMTE5OCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDMuMjguMDEzNzg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gultyaev</span>,  <span class="cit-name-given-names">A. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richard</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spronken</span>,  <span class="cit-name-given-names">M. I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olsthoorn</span>,  <span class="cit-name-given-names">R. C. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fouchier</span>,  <span class="cit-name-given-names">R. A. M.</span></span> <span class="cit-article-title">Conserved structural RNA domains in regions coding for cleavage site motifs in hemagglutinin genes of influenza viruses</span>. <abbr class="cit-jnl-abbrev">Virus Evol.</abbr> <span class="cit-vol">5</span>, (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">325</span>–<span class="cit-lpage">328</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft.volume%253D27%26rft.spage%253D325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.25" data-doi="10.1186/1748-7188-6-26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">ViennaRNA Package 2.0</span>. <abbr class="cit-jnl-abbrev">Algorithms Mol. Biol.</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">26</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAlgorithms%2BMol%2BBiol%26rft.aulast%253DLorenz%26rft.auinit1%253DR.%26rft.volume%253D6%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DViennaRNA%2BPackage%2B2.0.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1748-7188-6-26%26rft_id%253Dinfo%253Apmid%252F22115189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1748-7188-6-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22115189&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Madeira</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The EMBL-EBI search and sequence analysis tools APIs in 2019</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">47</span>, <span class="cit-fpage">W636</span>–<span class="cit-lpage">W641</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D47%26rft.spage%253DW636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.27" data-doi="10.1038/s41591-020-0820-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andersen</span>,  <span class="cit-name-given-names">K. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lipkin</span>,  <span class="cit-name-given-names">W. I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Garry</span>,  <span class="cit-name-given-names">R. F.</span></span> <span class="cit-article-title">The proximal origin of SARS-CoV-2</span>. <abbr class="cit-jnl-abbrev">Nat. Med.</abbr> <span class="cit-fpage">1</span>–<span class="cit-lpage">3</span> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41591-020-0820-9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMed.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41591-020-0820-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41591-020-0820-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.28" data-doi="10.1038/s41586-020-2012-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">579</span>, <span class="cit-fpage">270</span>–<span class="cit-lpage">273</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D579%26rft.spage%253D270%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2012-7%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-020-2012-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.29" data-doi="10.1093/nar/gkr245"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Di Tommaso</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">T-Coffee: a web server for the multiple sequence alignment of protein and RNA sequences using structural information and homology extension</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>, <span class="cit-fpage">W13</span>–<span class="cit-lpage">W17</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr245%26rft_id%253Dinfo%253Apmid%252F21558174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21558174&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292325300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Qing</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hantak</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perlman</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gallagher</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">Distinct Roles for Sialoside and Protein Receptors in Coronavirus Infection</span>. <abbr class="cit-jnl-abbrev">mBio</abbr> <span class="cit-vol">11</span>, (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Milanetti</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">In-Silico evidence for two receptors based strategy of SARS-CoV-2</abbr>. ArXiv200311107Phys.Q-Bio (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.32" data-doi="10.1016/j.virol.2009.06.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Galán</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Host cell proteins interacting with the 3’ end of TGEV coronavirus genome influence virus replication</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">391</span>, <span class="cit-fpage">304</span>–<span class="cit-lpage">314</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D391%26rft.spage%253D304%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2009.06.006%26rft_id%253Dinfo%253Apmid%252F19580983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2009.06.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19580983&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.33" data-doi="10.1016/j.jmb.2011.04.029"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heng</span>,  <span class="cit-name-given-names">X.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Summers</span>,  <span class="cit-name-given-names">M. F.</span></span> <span class="cit-article-title">Structural determinants and mechanism of HIV-1 genome packaging</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">410</span>, <span class="cit-fpage">609</span>–<span class="cit-lpage">633</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DLu%26rft.auinit1%253DK.%26rft.volume%253D410%26rft.issue%253D4%26rft.spage%253D609%26rft.epage%253D633%26rft.atitle%253DStructural%2Bdeterminants%2Band%2Bmechanism%2Bof%2BHIV-1%2Bgenome%2Bpackaging.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2011.04.029%26rft_id%253Dinfo%253Apmid%252F21762803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2011.04.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21762803&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fehr</span>,  <span class="cit-name-given-names">A. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Perlman</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Coronaviruses: An Overview of Their Replication and Pathogenesis</span>. <abbr class="cit-jnl-abbrev">Methods Mol. Biol. Clifton NJ</abbr> <span class="cit-vol">1282</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">23</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%2BClifton%2BNJ%26rft.volume%253D1282%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.35" data-doi="10.1186/s12864-015-2280-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Klus</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ponti</span>,  <span class="cit-name-given-names">R. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Livi</span>,  <span class="cit-name-given-names">C. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Tartaglia</span>,  <span class="cit-name-given-names">G. G.</span></span> <span class="cit-article-title">Protein aggregation, structural disorder and RNA-binding ability: a new approach for physico-chemical and gene ontology classification of multiple datasets</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">1071</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D1071%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2280-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2280-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.36" data-doi="10.1093/nar/gky1049"><div class="cit-metadata"><cite><span class="cit-article-title">UniProt: a worldwide hub of protein knowledge</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">47</span>, <span class="cit-fpage">D506</span>–<span class="cit-lpage">D515</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D47%26rft.spage%253DD506%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky1049%26rft_id%253Dinfo%253Apmid%252F30395287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky1049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30395287&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.37" data-doi="10.1016/j.cell.2012.04.031"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Castello</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Insights into RNA biology from an atlas of mammalian mRNA-binding proteins</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">149</span>, <span class="cit-fpage">1393</span>–<span class="cit-lpage">1406</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCastello%26rft.auinit1%253DA.%26rft.volume%253D149%26rft.issue%253D6%26rft.spage%253D1393%26rft.epage%253D1406%26rft.atitle%253DInsights%2Binto%2BRNA%2Bbiology%2Bfrom%2Ban%2Batlas%2Bof%2Bmammalian%2BmRNA-binding%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.04.031%26rft_id%253Dinfo%253Apmid%252F22658674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.04.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22658674&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305119600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.38" data-doi="10.1016/j.tim.2012.11.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bell</span>,  <span class="cit-name-given-names">N. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lever</span>,  <span class="cit-name-given-names">A. M. L.</span></span> <span class="cit-article-title">HIV Gag polyprotein: processing and early viral particle assembly</span>. <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">21</span>, <span class="cit-fpage">136</span>–<span class="cit-lpage">144</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BMicrobiol.%26rft.volume%253D21%26rft.spage%253D136%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2012.11.006%26rft_id%253Dinfo%253Apmid%252F23266279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2012.11.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23266279&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316715200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.39" data-doi="10.1093/nar/gkq537"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Warde-Farley</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The GeneMANIA prediction server: biological network integration for gene prioritization and predicting gene function</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>, <span class="cit-fpage">W214</span>–<span class="cit-lpage">W220</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq537%26rft_id%253Dinfo%253Apmid%252F20576703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq537&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20576703&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284148900035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.40" data-doi="10.1016/j.semcancer.2004.04.018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hayward</span>,  <span class="cit-name-given-names">S. D.</span></span> <span class="cit-article-title">Viral interactions with the Notch pathway</span>. <abbr class="cit-jnl-abbrev">Semin. Cancer Biol.</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">387</span>–<span class="cit-lpage">396</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSeminars%2Bin%2Bcancer%2Bbiology%26rft.stitle%253DSemin%2BCancer%2BBiol%26rft.aulast%253DHayward%26rft.auinit1%253DS.%2BD.%26rft.volume%253D14%26rft.issue%253D5%26rft.spage%253D387%26rft.epage%253D396%26rft.atitle%253DViral%2Binteractions%2Bwith%2Bthe%2BNotch%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.semcancer.2004.04.018%26rft_id%253Dinfo%253Apmid%252F15288264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.semcancer.2004.04.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15288264&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223477400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.41" data-doi="10.1038/sj.leu.2402451"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dudley</span>,  <span class="cit-name-given-names">J. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mertz</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rajan</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lozano</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Broussard</span>,  <span class="cit-name-given-names">D. R.</span></span> <span class="cit-article-title">What retroviruses teach us about the involvement of c-Myc in leukemias and lymphomas</span>. <abbr class="cit-jnl-abbrev">Leukemia</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">1086</span>–<span class="cit-lpage">1098</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLeukemia%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BLeukemia%2BSociety%2Bof%2BAmerica%252C%2BLeukemia%2BResearch%2BFund%252C%2BU.K%26rft.stitle%253DLeukemia%26rft.aulast%253DDudley%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D1086%26rft.epage%253D1098%26rft.atitle%253DWhat%2Bretroviruses%2Bteach%2Bus%2Babout%2Bthe%2Binvolvement%2Bof%2Bc-Myc%2Bin%2Bleukemias%2Band%2Blymphomas.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.leu.2402451%26rft_id%253Dinfo%253Apmid%252F12040439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.leu.2402451&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12040439&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.42" data-doi="10.1101/2020.03.12.988865"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">The architecture of SARS-CoV-2 transcriptome</abbr>. bioRxiv 2020.03.12.988865 (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.03.12.988865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2Barchitecture%2Bof%2BSARS-CoV-2%2Btranscriptome%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.03.12.988865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMy4xMi45ODg4NjV2MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDMuMjguMDEzNzg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.43" data-doi="10.1016/j.virol.2004.09.039"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fang</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A DEAD box protein facilitates HIV-1 replication as a cellular co-factor of Rev</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">330</span>, <span class="cit-fpage">471</span>–<span class="cit-lpage">480</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DFang%26rft.auinit1%253DJ.%26rft.volume%253D330%26rft.issue%253D2%26rft.spage%253D471%26rft.epage%253D480%26rft.atitle%253DA%2BDEAD%2Bbox%2Bprotein%2Bfacilitates%2BHIV-1%2Breplication%2Bas%2Ba%2Bcellular%2Bco-factor%2Bof%2BRev.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2004.09.039%26rft_id%253Dinfo%253Apmid%252F15567440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2004.09.039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15567440&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225779400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.44" data-doi="10.1128/JVI.00392-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The cellular RNA helicase DDX1 interacts with coronavirus nonstructural protein 14 and enhances viral replication</span>. <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">84</span>, <span class="cit-fpage">8571</span>–<span class="cit-lpage">8583</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00392-10%26rft_id%253Dinfo%253Apmid%252F20573827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4NC8xNy84NTcxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wMy4yOC4wMTM3ODkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carlson</span>,  <span class="cit-name-given-names">K. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Molecular characterization of a putative antiretroviral transcriptional factor, OTK18</span>. <abbr class="cit-jnl-abbrev">J. Immunol. Baltim. Md 1950</abbr> <span class="cit-vol">172</span>, <span class="cit-fpage">381</span>–<span class="cit-lpage">391</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BImmunol.%2BBaltim.%2BMd%2B1950%26rft.volume%253D172%26rft.spage%253D381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.46" data-doi="10.1006/jmbi.1999.2663"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ivanov</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Cyclin T1 domains involved in complex formation with Tat and TAR RNA are critical for tat-activation</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">288</span>, <span class="cit-fpage">41</span>–<span class="cit-lpage">56</span> (<span class="cit-pub-date">1999</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DIvanov%26rft.auinit1%253DD.%26rft.volume%253D288%26rft.issue%253D1%26rft.spage%253D41%26rft.epage%253D56%26rft.atitle%253DCyclin%2BT1%2Bdomains%2Binvolved%2Bin%2Bcomplex%2Bformation%2Bwith%2BTat%2Band%2BTAR%2BRNA%2Bare%2Bcritical%2Bfor%2Btat-activation.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1999.2663%26rft_id%253Dinfo%253Apmid%252F10329125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1999.2663&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10329125&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079925700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.47" data-doi="10.1006/jmbi.1999.2664"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kwak</span>,  <span class="cit-name-given-names">Y. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ivanov</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nee</span>,  <span class="cit-name-given-names">E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gaynor</span>,  <span class="cit-name-given-names">R. B.</span></span> <span class="cit-article-title">Role of the human and murine cyclin T proteins in regulating HIV-1 tat-activation</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">288</span>, <span class="cit-fpage">57</span>–<span class="cit-lpage">69</span> (<span class="cit-pub-date">1999</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DKwak%26rft.auinit1%253DY.%2BT.%26rft.volume%253D288%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D69%26rft.atitle%253DRole%2Bof%2Bthe%2Bhuman%2Band%2Bmurine%2Bcyclin%2BT%2Bproteins%2Bin%2Bregulating%2BHIV-1%2Btat-activation.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1999.2664%26rft_id%253Dinfo%253Apmid%252F10329126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1999.2664&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10329126&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.48" data-doi="10.1093/hmg/ddp167"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Locke</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tinsley</span>,  <span class="cit-name-given-names">C. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benson</span>,  <span class="cit-name-given-names">M. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Blake</span>,  <span class="cit-name-given-names">D. J.</span></span> <span class="cit-article-title">TRIM32 is an E3 ubiquitin ligase for dysbindin</span>. <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">2344</span>–<span class="cit-lpage">2358</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddp167%26rft_id%253Dinfo%253Apmid%252F19349376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/hmg/ddp167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19349376&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266961400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.49" data-doi="10.1101/2020.03.22.002386"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gordon</span>,  <span class="cit-name-given-names">D. E.</span></span> <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">A SARS-CoV-2-Human Protein-Protein Interaction Map Reveals Drug Targets and Potential Drug-Repurposing</abbr>. bioRxiv 2020.03.22.002386 (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.03.22.002386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DA%2BSARS-CoV-2-Human%2BProtein-Protein%2BInteraction%2BMap%2BReveals%2BDrug%2BTargets%2Band%2BPotential%2BDrug-Repurposing%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.03.22.002386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMy4yMi4wMDIzODZ2MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDMuMjguMDEzNzg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xing</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kleiman</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">The role of A-kinase anchoring protein 95-like protein in annealing of tRNALys3 to HIV-1 RNA</span>. <abbr class="cit-jnl-abbrev">Retrovirology</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">58</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRetrovirology%26rft.volume%253D11%26rft.spage%253D58%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.51" data-doi="10.1038/embor.2008.72"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Markert</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The La-related protein LARP7 is a component of the 7SK ribonucleoprotein and affects transcription of cellular and viral polymerase II genes</span>. <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">569</span>–<span class="cit-lpage">575</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BRep.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fembor.2008.72%26rft_id%253Dinfo%253Apmid%252F18483487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZW1ib3IiO3M6NToicmVzaWQiO3M6NzoiOS82LzU2OSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDMuMjguMDEzNzg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">C</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Systematic Analysis of the Protein Interaction Network for the Human Transcription Machinery Reveals the Identity of the 7SK Capping Enzyme</span>. <abbr class="cit-jnl-abbrev">Molecular cell</abbr> vol. <span class="cit-vol">27</span> <a href="https://pubmed.ncbi.nlm.nih.gov/17643375/">https://pubmed.ncbi.nlm.nih.gov/17643375/</a> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mendez</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">ChEMBL: towards direct deposition of bioassay data</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">47</span>, <span class="cit-fpage">D930</span>–<span class="cit-lpage">D940</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D47%26rft.spage%253DD930%26rft_id%253Dinfo%253Apmid%252F30398643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=30398643&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.54" data-doi="10.1093/nar/gkw1094"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Delli Ponti</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marti</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Armaos</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Tartaglia</span>,  <span class="cit-name-given-names">G. G.</span></span> <span class="cit-article-title">A high-throughput approach to profile RNA structure</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>, <span class="cit-fpage">e35</span>–<span class="cit-lpage">e35</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D45%26rft.spage%253De35%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1094&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.55" data-doi="10.1093/nar/gkg933"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schlott</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Görlach</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Grosse</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">DNA-dependent protein kinase (DNA-PK) phosphorylates nuclear DNA helicase II/RNA helicase A and hnRNP proteins in an RNA-dependent manner</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DZhang%26rft.auinit1%253DS.%26rft.volume%253D32%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D10%26rft.atitle%253DDNA-dependent%2Bprotein%2Bkinase%2B%2528DNA-PK%2529%2Bphosphorylates%2Bnuclear%2BDNA%2Bhelicase%2BII%252FRNA%2Bhelicase%2BA%2Band%2BhnRNP%2Bproteins%2Bin%2Ban%2BRNA-dependent%2Bmanner%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg933%26rft_id%253Dinfo%253Apmid%252F14704337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg933&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14704337&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188988700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.28.013789v3.56" data-doi="10.3390/ijms160817611"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Siddiqui</span>,  <span class="cit-name-given-names">M. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mukherjee</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manivannan</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Malathi</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">RNase L Cleavage Products Promote Switch from Autophagy to Apoptosis by Caspase-Mediated Cleavage of Beclin-1</span>. <abbr class="cit-jnl-abbrev">Int. J. Mol. Sci.</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">17611</span>–<span class="cit-lpage">17636</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BMol.%2BSci.%26rft.volume%253D16%26rft.spage%253D17611%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fijms160817611%26rft_id%253Dinfo%253Apmid%252F26263979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/ijms160817611&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26263979&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.03.28.013789.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.28.013789v3.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.28.013789v3" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.29.013342v3" title="A one-enzyme RT-qPCR assay for SARS-CoV-2, and procedures for reagent production" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.25.009084v3" title="Non-neuronal expression of SARS-CoV-2 entry genes in the olfactory system suggests mechanisms underlying COVID-19-associated anosmia" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 09, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.28.013789v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.28.013789v3.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.28.013789v3.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.28.013789v3" class="active" data-icon-position="" data-hide-link-title="0">Structural analysis of SARS-CoV-2 and predictions of the human interactome</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1237309" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-nyV876tFZgk8V-QmLRJDJ5zeRZdY5-Y5htuqqKgZdHk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1237309" id="node1237309--2" data-pisa="biorxiv;2020.03.28.013789v3" data-pisa-master="biorxiv;2020.03.28.013789" data-seqnum="1237309" data-apath="/biorxiv/early/2020/04/09/2020.03.28.013789.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Structural analysis of SARS-CoV-2 and predictions of the human interactome</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Andrea</span> <span class="nlm-surname">Vandelli</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Michele</span> <span class="nlm-surname">Monti</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Edoardo</span> <span class="nlm-surname">Milanetti</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Riccardo</span> <span class="nlm-surname">Delli Ponti</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Gian</span> <span class="nlm-surname">Gaetano Tartaglia</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.28.013789; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.28.013789 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.28.013789v3" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013789v3&amp;title=Structural%20analysis%20of%20SARS-CoV-2%20and%20predictions%20of%20the%20human%20interactome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013789v3&amp;title=Structural%20analysis%20of%20SARS-CoV-2%20and%20predictions%20of%20the%20human%20interactome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013789v3&amp;text=Structural%20analysis%20of%20SARS-CoV-2%20and%20predictions%20of%20the%20human%20interactome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013789v3&amp;title=Structural%20analysis%20of%20SARS-CoV-2%20and%20predictions%20of%20the%20human%20interactome" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013789v3&amp;t=Structural%20analysis%20of%20SARS-CoV-2%20and%20predictions%20of%20the%20human%20interactome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013789v3&amp;title=Structural%20analysis%20of%20SARS-CoV-2%20and%20predictions%20of%20the%20human%20interactome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013789v3&amp;title=Structural%20analysis%20of%20SARS-CoV-2%20and%20predictions%20of%20the%20human%20interactome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1237309" id="node1237309--3" data-pisa="biorxiv;2020.03.28.013789v3" data-pisa-master="biorxiv;2020.03.28.013789" data-seqnum="1237309" data-apath="/biorxiv/early/2020/04/09/2020.03.28.013789.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Structural analysis of SARS-CoV-2 and predictions of the human interactome</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Andrea</span> <span class="nlm-surname">Vandelli</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Michele</span> <span class="nlm-surname">Monti</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Edoardo</span> <span class="nlm-surname">Milanetti</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Riccardo</span> <span class="nlm-surname">Delli Ponti</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Gian</span> <span class="nlm-surname">Gaetano Tartaglia</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.28.013789; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.28.013789 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Structural%20analysis%20of%20SARS-CoV-2%20and%20predictions%20of%20the%20human%20interactome&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Vandelli%2C+Andrea&amp;rft.au=Monti%2C+Michele&amp;rft.au=Milanetti%2C+Edoardo&amp;rft.au=Delli+Ponti%2C+Riccardo&amp;rft.au=Gaetano+Tartaglia%2C+Gian"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1237309/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1237309/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1237309/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1237309/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1237309/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1237309/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1237309/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1237309/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1237309/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1237309/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1237309/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1237309/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013789v3&amp;count=horizontal&amp;via=&amp;text=Structural%20analysis%20of%20SARS-CoV-2%20and%20predictions%20of%20the%20human%20interactome&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013789v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.28.013789v3&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.28.013789v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
