<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS Negl Trop Dis</span><span class="iso-abbrev" title="iso-abbrev">PLoS Negl Trop Dis</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plosntds</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS Neglected Tropical Diseases</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1935-2727</span><span class="issn-epub" tagx="issn" title="issn-epub">1935-2735</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6143276">6143276</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/30188905">30188905</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pntd.0006738">10.1371/journal.pntd.0006738</a></span><span class="publisher-id" title="publisher-id">PNTD-D-18-00816</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Processes</span>: <span class="subject" title="subject">Evolutionary Rate</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Evolution</span>: <span class="subject" title="subject">Viral Evolution</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organismal Evolution</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Evolution</span>: <span class="subject" title="subject">Viral Evolution</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Virology</span>: <span class="subject" title="subject">Viral Evolution</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genomics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Genomics</span>: <span class="subject" title="subject">Viral Genomics</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Genomics</span>: <span class="subject" title="subject">Viral Genomics</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Virology</span>: <span class="subject" title="subject">Viral Genomics</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular biology techniques</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequencing techniques</span>: <span class="subject" title="subject">RNA sequencing</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and analysis methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular biology techniques</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequencing techniques</span>: <span class="subject" title="subject">RNA sequencing</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phylogenetics</span>: <span class="subject" title="subject">Phylogenetic Analysis</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Taxonomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phylogenetics</span>: <span class="subject" title="subject">Phylogenetic Analysis</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computer and Information Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Data Management</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Taxonomy</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Phylogenetics</span>: <span class="subject" title="subject">Phylogenetic Analysis</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Techniques</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequencing Techniques</span>: <span class="subject" title="subject">Genome Sequencing</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Techniques</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequencing Techniques</span>: <span class="subject" title="subject">Genome Sequencing</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">People and Places</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Geographical Locations</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Africa</span>: <span class="subject" title="subject">Angola</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Techniques</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequencing Techniques</span>: <span class="subject" title="subject">Nucleotide Sequencing</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Molecular Biology Techniques</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequencing Techniques</span>: <span class="subject" title="subject">Nucleotide Sequencing</span></div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Phylogenomic analysis unravels evolution of yellow fever virus within hosts</div> <div class="alt-title" title="alt-title">
Intrahost phylogenomics of yellow fever virus</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-1765-8899</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Chen</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Writing – original draft</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">Dong</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ni</span><span class="given-names" tagx="given-names" title="given-names">Ming</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Software</span><span class="role" tagx="role" title="role">Visualization</span><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Jing</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Zhihai</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Project administration</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a><a href="#aff005"><sup>5</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Jingyuan</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff006"><sup>6</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">Hanhui</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Validation</span><a href="#aff007"><sup>7</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">Gary</span></span><span class="role" tagx="role" title="role">Validation</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Wei</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Yuanyuan</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Validation</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-3746-0188</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Beibei</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bi</span><span class="given-names" tagx="given-names" title="given-names">Yuhai</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Danying</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Ping</span></span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Validation</span><a href="#aff007"><sup>7</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Xuesen</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Project administration</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kong</span><span class="given-names" tagx="given-names" title="given-names">Yaxian</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Weifeng</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Validation</span><a href="#aff008"><sup>8</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">Pengcheng</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Methodology</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">Gengfu</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff009"><sup>9</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">Juncai</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Validation</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">George F.</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Supervision</span><a href="#aff004"><sup>4</sup></a><a href="#aff010"><sup>10</sup></a><a href="#aff011"><sup>11</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">Jie</span></span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><a href="#aff009"><sup>9</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Fujie</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a><a href="#aff005"><sup>5</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Wenjun</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff004"><sup>4</sup></a><a href="#aff010"><sup>10</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bo</span><span class="given-names" tagx="given-names" title="given-names">Xiaochen</span></span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Writing – original draft</span><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Ang</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Funding acquisition</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeng</span><span class="given-names" tagx="given-names" title="given-names">Hui</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Project administration</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0003-3693-2726</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Di</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Funding acquisition</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff004"><sup>4</sup></a><a href="#aff009"><sup>9</sup></a><a href="#aff010"><sup>10</sup></a><a href="#cor001">*</a></span></div><span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">Institute of Infectious Diseases, Beijing Ditan Hospital, Capital Medical University, Beijing, China</span></span><span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Beijing Key Laboratory of Emerging Infectious Diseases, Beijing, China</span></span><span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">Beijing Institute of Radiation Medicine, Beijing, China</span></span><span class="citation_author_institution" id="aff004">[4], <span class="addr-line" title="addr-line">Institute of Microbiology, Chinese Academy of Sciences, Beijing, China</span></span><span class="citation_author_institution" id="aff005">[5], <span class="addr-line" title="addr-line">Clinical Center for Infectious Diseases, Beijing Ditan Hospital, Capital Medical University, Beijing, China</span></span><span class="citation_author_institution" id="aff006">[6], <span class="addr-line" title="addr-line">Department of Intensive Care Unit, Beijing Ditan Hospital, Capital Medical University, Beijing, China</span></span><span class="citation_author_institution" id="aff007">[7], <span class="addr-line" title="addr-line">Fuzhou Infectious Disease Hospital of Fujian Medical University, Fuzhou, China</span></span><span class="citation_author_institution" id="aff008">[8], <span class="addr-line" title="addr-line">Institute of Pathogen Biology, Taishan Medical College, Taian, China</span></span><span class="citation_author_institution" id="aff009">[9], <span class="addr-line" title="addr-line">Wuhan Institute of Virology, Chinese Academy of Sciences, Wuhan, China</span></span><span class="citation_author_institution" id="aff010">[10], <span class="addr-line" title="addr-line">Medical School, University of Chinese Academy of Sciences, Beijing, China</span></span><span class="citation_author_institution" id="aff011">[11], <span class="addr-line" title="addr-line">Chinese Center for Disease Control and Prevention (China CDC), Beijing, China</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mossel</span><span class="given-names" tagx="given-names" title="given-names">Eric</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Center for Disease Control and Prevention, UNITED STATES</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors have declared that no competing interests exist.</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">liud@wh.iov.cn</span> (DL); <span class="email" tagx="email" title="email">zenghui@ccmu.edu.cn</span> (HZ); <span class="email" tagx="email" title="email">chenchen1@ccmu.edu.cn</span> (CC); <span class="email" tagx="email" title="email">liang@ccmu.edu.cn</span>) (AL)</div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-9-9</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2018-9</span></span><span class="volume" tagx="volume" title="volume">12</span><span class="issue" tagx="issue" title="issue">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e0006738</span><span class="history" title="history"><span class="received" title="received">received: 2018-5-22</span><span class="accepted" title="accepted">accepted: 2018-8-7</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Chen et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></div><a class="self-uri" href="pntd.0006738.pdf" title="self-uri">pntd.0006738.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The yellow fever virus (YFV) recently reemerged in the large outbreaks in Africa and Brazil, and the first imported patients into Asia have recalled the concerns of YFV evolution. Here we show phylogenomics of YFV with serial clinical samples of the 2016 YFV infections. Phylogenetics exhibited that the 2016 strains were close to Angola 1971 strains and only three amino acid changes presented new to other lineages. Deep sequencing of viral genomes discovered 101 intrahost single nucleotide variations (iSNVs) and 234 single nucleotide polymorphisms (SNPs). Analysis of iSNV distribution and mutated allele frequency revealed that the coding regions were under purifying selection. Comparison of the evolutionary rates estimated by iSNV and SNP showed that the intrahost rate was ~2.25 times higher than the epidemic rate, and both rates were higher than the long-term YFV substitution rate, as expected. In addition, the result also hinted that short viremia duration of YFV might further hinder the evolution of YFV.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Author summary</div> <p>The first importation of infections into China in 2016 and the following outbreaks in Africa and Brazil of yellow fever virus (YFV) have raised again the concerns of the potential viral spread into new territories. In this study, we aimed to know the evolution dynamics of YFV by using intrahost phylogenomics and to assess the risk of virus epidemics. Through deep sequencing of consecutive samples from 12 patients, we identified hundreds of genomic variations (iSNVs and SNPs), and noticed the nearly linear accumulation of variations within individuals. The estimated evolutionary rate within host is much higher than the epidemic evolutionary rate. In comparison with Dengue virus (DENV) and Zika virus (ZIKV), which share similar host vectors (Aedes spp.), life cycles, mutation rates and replication strategies to YFV, the lower epidemic evolutionary rate of YFV might have been hindered by the shorter viremia duration, which decreased the accumulated variations to get into the transmission cycle.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Beijing Municipal Science &amp;amp; Technology Commission</span></div><span class="award-id" tagx="award-id" title="award-id">Z161100000116049</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Ang</span></span></span></div> <div class="funding-statement" title="funding-statement">
This work was supported by Grants from China Ministry of Science and Technology Key Research and Development Program [2016YFC1200804 and 2016YFC1200805], Beijing Municipal Science &amp;amp; Technology Commission [Z161100000116049], National Natural Science Foundation of China [91631110 and U1435222], Beijing Municipal Administration of Hospitals Clinical Medicine Development of Special Funding Support [ZYLX201602NHFPC], Beijing Nova program [Z161100004916101, Z181100006218114], Beijing 215 Project [2015-3-107], China National Grand S&amp;amp;T Special Project [2016ZX10004222-010], and ‘863’ Program [2015AA020108 &amp;amp; 2014AA021501]. WS is supported by the ‘Taishan Scholar’ project of Shandong Province. GW is supported by the President’s International Fellowship Initiative from CAS. JC is supported by CAS Pioneer Hundred Talents Program. GFG is a leading principal investigator of the NSFC Innovative Research Group [81321063]. DL is supported by National Program for Support of Top-notch Young Professionals. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="PLOS Publication Stage" value="vor-update-to-uncorrected-proof" /> <meta name="Publication Update" value="2018-09-18" /> <meta name="Data Availability" value="All reads from next generation sequencing were deposited in the NCBI SRA database under the accession no. SRP096859. The genome sequences of YF-BJ1, YF-BJ2, YF-BJ3, YF-BJ4, YF-BJ5, YF-FZ2, YF-FZ4, YF-FZ6, and YF-FZ7 were deposited in the NCBI GenBank database under the accession no. MH633684-MH633692." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>All reads from next generation sequencing were deposited in the NCBI SRA database under the accession no. SRP096859. The genome sequences of YF-BJ1, YF-BJ2, YF-BJ3, YF-BJ4, YF-BJ5, YF-FZ2, YF-FZ4, YF-FZ6, and YF-FZ7 were deposited in the NCBI GenBank database under the accession no. MH633684-MH633692.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Yellow fever is a notorious mosquito-borne viral disease emerged during the 15th-19th centuries in the Americas, Africa and Europe, causing severe hemorrhagic fever and liver injury with high mortality rates. Although control of mosquitoes and the use of the live-attenuated yellow fever virus (YFV)-17D vaccine strain have effectively prevented and controlled the epidemics, YFV is estimated to cause approximately 30,000 deaths out of 200,000 infections annually worldwide, mostly in Africa (<a href="https://www.cdc.gov/">https://www.cdc.gov</a>, last accessed 10<sup>th</sup> April, 2018). The etiological agent of the disease, YFV, is a single-stranded, positive-sense RNA virus with a ~11 kb genome in length. The virus is frequently transmitted between nonhuman primates and mosquitoes in African and American jungles, known as the sylvatic cycle [<a href="#pntd.0006738.ref001">1</a>]. Occasionally, YFV can escape from the sylvatic cycle to infect humans, with subsequent transmission between mosquitoes and humans, forming the urban cycle. Historically, YFV was believed to have originated from Central or East Africa, and transmitted to America during the slave trade [<a href="#pntd.0006738.ref002">2</a>, <a href="#pntd.0006738.ref003">3</a>]. Currently, YFV in Africa and America are classified into seven genotypes, with two in South America, two in West Africa, and three in East and Central Africa [<a href="#pntd.0006738.ref002">2</a>, <a href="#pntd.0006738.ref004">4</a>, <a href="#pntd.0006738.ref005">5</a>].</p> <p>In January 2016, the Ministry of Health of Angola notified the World Health Organization (WHO) of a yellow fever disease outbreak: as of October 28<sup>th</sup>, 4,347 suspected cases, including 377 deaths, were reported from all 18 provinces of Angola. Outbreaks were also reported simultaneously from the Democratic Republic of Congo and Uganda [<a href="#pntd.0006738.ref006">6</a>]. In February 2018, 464 confirmed human cases of yellow fever have also been reported in Brazil, with 154 deaths (<a href="http://www.who.int/">http://www.who.int</a>, last accessed 10<sup>th</sup> April, 2018). Meanwhile, epizootics have expanded to areas previously not considered at risk for yellow fever (<a href="http://www.who.int/csr/don/27-february-2018-yellow-fever-brazil/en/">http://www.who.int/csr/don/27-february-2018-yellow-fever-brazil/en/</a>, last accessed on 10<sup>th</sup> April, 2018). All these recalled the concerns of the evolution of YFV. During the Angolan outbreak, YFV infections were detected in Chinese workers returning to China from Angola [<a href="#pntd.0006738.ref007">7</a>–<a href="#pntd.0006738.ref009">9</a>], marking the first time YFV infections were documented in Asia. By using the consecutive samples from those imported YFV patients, we unraveled the intrahost and epidemic evolutionary dynamics of YFV.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="deepsequencingofyfvgenomesfromclinicalsamples" title="sec"> <div class="title" tagx="title" title="title">
Deep sequencing of YFV genomes from clinical samples</div> <p>We collected samples from twelve out of thirteen YFV patients in China, sequenced the viral genomes, and performed analyses on phylogenetics, sequence comparison, and intra-host dynamics. Of the 12 patients, two had severe disease, in which one survived (YF-BJ3) and one died (YF-BJ1), and the remaining patients displayed mild symptoms (<i><a href="#pntd.0006738.t001">Table 1</a></i>). We tested blood samples of these 12 patients from the first day of admittance to the hospital by real-time reverse transcription PCR (RT-PCR), and four patients (YF-BJ1, YF-BJ2, YF-BJ3, and YF-BJ5) were positive. YFV RNA fragments could be detected in serum until 9, 12, 10 and 6 days, respectively, after the onset of symptoms (<i><a href="#pntd.0006738.t001">Table 1</a>, and <a href="#pntd.0006738.g001">Fig 1</a></i>). Urine samples from all patients were also tested by real-time RT-PCR and all were YFV-positive. Higher viral RNA loads were observed in the urine compared to the blood samples, except for those from the non-survivor (<i><a href="#pntd.0006738.t001">Table 1</a></i>). The virus persisted in urine samples for at least 15 days after the onset of symptoms in survivors. Specifically, the urine sample from patient YF-BJ3 was still PCR-positive when tested 32 days after the onset of symptoms (<i><a href="#pntd.0006738.t001">Table 1</a> and <a href="#pntd.0006738.g001">Fig 1</a></i>). We then sequenced all available YFV-positive samples by using total RNA sequencing, amplicon sequencing and/or Sanger sequencing (<i><a href="#pntd.0006738.t001">Table 1</a> and</i><a href="#pntd.0006738.s004"><i>S</i>1 <i>Table</i></a>). In total, nearly complete virus genomes (&amp;gt;10,222 bp) from 9 patients, and deep-sequenced genome datasets (average sequence depth 34,978x) from 3 patients covering 12 time points were obtained (<i><a href="#pntd.0006738.t001">Table 1</a></i>).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pntd.0006738.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
Cases of twelve documented returning YFV patients in China by date in the outbreak of Angola from March to May 2016.</div> <p>Timeline of events for each patient from symptom onset to the leaving from hospital are shown with different molecular detection methods and results. Dots denote that we did not get any positive molecular detection result in the time point, grey (before admitted to hospital) and blue (after admitted to hospital). The larger size dots means that we performed Real-time PCR detections in that point. The red triangles denote the sample have been detected by PCR and Real-time PCR with positive results. Blue boxes denote the samples that were subjected to NGS analysis at the time points.</p>   </div> <div class="table-wrap_UNKNOWN" id="pntd.0006738.t001" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.pntd.0006738.t001">10.1371/journal.pntd.0006738.t001</a><span class="label" tagx="label" title="label">Table 1</span> <div class="title" tagx="title" title="title">
Patient information, viral loads and genome sequencing in urine samples.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="pntd.0006738.t001g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="center" rowspan="1" colspan="1">Patient ID</th> <th align="center" rowspan="1" colspan="1">Group</th> <th align="center" rowspan="1" colspan="1">Outcome</th> <th align="center" rowspan="1" colspan="1">Days after onset</th> <th align="center" rowspan="1" colspan="1">Blood virus (Ct)<a href="#t001fn001">*</a></th> <th align="center" rowspan="1" colspan="1">Urine virus (Ct)<a href="#t001fn001">*</a></th> <th align="center" rowspan="1" colspan="1">Sequencing Method<a href="#t001fn002">†</a></th> <th align="center" rowspan="1" colspan="1"># Total Reads</th> <th align="center" rowspan="1" colspan="1"># Mapped Reads</th> <th align="center" rowspan="1" colspan="1">Sequencing Depth (x1000)</th> <th align="center" rowspan="1" colspan="1">Genome Coverage (bp)</th> </tr> </thead> <tbody> <tr> <td align="center" rowspan="1" colspan="1">YF-BJ1</td> <td align="center" rowspan="1" colspan="1">Severe</td> <td align="center" rowspan="1" colspan="1">Death</td> <td align="center" rowspan="1" colspan="1">7</td> <td align="center" rowspan="1" colspan="1">31</td> <td align="center" rowspan="1" colspan="1">35.00</td> <td align="center" rowspan="1" colspan="1">AS</td> <td align="center" rowspan="1" colspan="1">1,695,368</td> <td align="center" rowspan="1" colspan="1">1,509,993</td> <td align="center" rowspan="1" colspan="1">30.98</td> <td align="center" rowspan="1" colspan="1">10,732</td> </tr> <tr> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1">8</td> <td align="center" rowspan="1" colspan="1">32</td> <td align="center" rowspan="1" colspan="1">33.50</td> <td align="center" rowspan="1" colspan="1">AS</td> <td align="center" rowspan="1" colspan="1">1,661,085</td> <td align="center" rowspan="1" colspan="1">953,063</td> <td align="center" rowspan="1" colspan="1">19.67</td> <td align="center" rowspan="1" colspan="1">10,650</td> </tr> <tr> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1">9</td> <td align="center" rowspan="1" colspan="1">30</td> <td align="center" rowspan="1" colspan="1">34.42</td> <td align="center" rowspan="1" colspan="1">AS</td> <td align="center" rowspan="1" colspan="1">3,303,973</td> <td align="center" rowspan="1" colspan="1">1,952,626</td> <td align="center" rowspan="1" colspan="1">35.68</td> <td align="center" rowspan="1" colspan="1">10,725</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">YF-BJ3</td> <td align="center" rowspan="1" colspan="1">Severe</td> <td align="center" rowspan="1" colspan="1">Discharged</td> <td align="center" rowspan="1" colspan="1">8</td> <td align="center" rowspan="1" colspan="1">27.6</td> <td align="center" rowspan="1" colspan="1">25.60</td> <td align="center" rowspan="1" colspan="1">AS</td> <td align="center" rowspan="1" colspan="1">1,451,105</td> <td align="center" rowspan="1" colspan="1">1,354,090</td> <td align="center" rowspan="1" colspan="1">27.59</td> <td align="center" rowspan="1" colspan="1">10,650</td> </tr> <tr> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1">10</td> <td align="center" rowspan="1" colspan="1">27.9</td> <td align="center" rowspan="1" colspan="1">25.32</td> <td align="center" rowspan="1" colspan="1">AS</td> <td align="center" rowspan="1" colspan="1">1,941,400</td> <td align="center" rowspan="1" colspan="1">2,369,835</td> <td align="center" rowspan="1" colspan="1">18.97</td> <td align="center" rowspan="1" colspan="1">10,393</td> </tr> <tr> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1">14</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">32.85</td> <td align="center" rowspan="1" colspan="1">AS</td> <td align="center" rowspan="1" colspan="1">4,034,937</td> <td align="center" rowspan="1" colspan="1">2,896,728</td> <td align="center" rowspan="1" colspan="1">55.05</td> <td align="center" rowspan="1" colspan="1">10,405</td> </tr> <tr> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1">17</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">31.66</td> <td align="center" rowspan="1" colspan="1">AS</td> <td align="center" rowspan="1" colspan="1">1,998,163</td> <td align="center" rowspan="1" colspan="1">1,861,631</td> <td align="center" rowspan="1" colspan="1">36.67</td> <td align="center" rowspan="1" colspan="1">10,397</td> </tr> <tr> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1">20</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">34.00</td> <td align="center" rowspan="1" colspan="1">AS</td> <td align="center" rowspan="1" colspan="1">1,920,032</td> <td align="center" rowspan="1" colspan="1">1,416,518</td> <td align="center" rowspan="1" colspan="1">28.89</td> <td align="center" rowspan="1" colspan="1">10,393</td> </tr> <tr> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1">32</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">39.83</td> <td align="center" rowspan="1" colspan="1">AS</td> <td align="center" rowspan="1" colspan="1">1,424,735</td> <td align="center" rowspan="1" colspan="1">1,014,331</td> <td align="center" rowspan="1" colspan="1">21.80</td> <td align="center" rowspan="1" colspan="1">10,650</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">YF-BJ5</td> <td align="center" rowspan="1" colspan="1">Mild</td> <td align="center" rowspan="1" colspan="1">Discharged</td> <td align="center" rowspan="1" colspan="1">6</td> <td align="center" rowspan="1" colspan="1">23.84</td> <td align="center" rowspan="1" colspan="1">23.77</td> <td align="center" rowspan="1" colspan="1">AS</td> <td align="center" rowspan="1" colspan="1">2,001,845</td> <td align="center" rowspan="1" colspan="1">1,866,672</td> <td align="center" rowspan="1" colspan="1">37.82</td> <td align="center" rowspan="1" colspan="1">9,918</td> </tr> <tr> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1">10</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">35.42</td> <td align="center" rowspan="1" colspan="1">AS</td> <td align="center" rowspan="1" colspan="1">1,684,026</td> <td align="center" rowspan="1" colspan="1">1,014,398</td> <td align="center" rowspan="1" colspan="1">18.54</td> <td align="center" rowspan="1" colspan="1">10,674</td> </tr> <tr> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1" /> <td align="center" rowspan="1" colspan="1">14</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">34.23</td> <td align="center" rowspan="1" colspan="1">AS</td> <td align="center" rowspan="1" colspan="1">2,090,128</td> <td align="center" rowspan="1" colspan="1">840,862</td> <td align="center" rowspan="1" colspan="1">16.56</td> <td align="center" rowspan="1" colspan="1">10,394</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">YF-BJ2</td> <td align="center" rowspan="1" colspan="1">Mild</td> <td align="center" rowspan="1" colspan="1">Discharged</td> <td align="center" rowspan="1" colspan="1">16</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">30.95</td> <td align="center" rowspan="1" colspan="1">AS</td> <td align="center" rowspan="1" colspan="1">1,924,145</td> <td align="center" rowspan="1" colspan="1">1,467,048</td> <td align="center" rowspan="1" colspan="1">30.21</td> <td align="center" rowspan="1" colspan="1">10,398</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">YF-BJ4</td> <td align="center" rowspan="1" colspan="1">Mild</td> <td align="center" rowspan="1" colspan="1">Discharged</td> <td align="center" rowspan="1" colspan="1">19</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">35.86</td> <td align="center" rowspan="1" colspan="1">TS</td> <td align="center" rowspan="1" colspan="1">56,728,408</td> <td align="center" rowspan="1" colspan="1">873</td> <td align="center" rowspan="1" colspan="1">0.02</td> <td align="center" rowspan="1" colspan="1">10,393</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">YF-FZ1<a href="#t001fn004">§</a></td> <td align="center" rowspan="1" colspan="1">Mild</td> <td align="center" rowspan="1" colspan="1">Discharged</td> <td align="center" rowspan="1" colspan="1">6</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">TS</td> <td align="center" rowspan="1" colspan="1">77,238,978</td> <td align="center" rowspan="1" colspan="1">129</td> <td align="center" rowspan="1" colspan="1">0.00</td> <td align="center" rowspan="1" colspan="1">1,012</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">YF-FZ2<a href="#t001fn003">¶</a></td> <td align="center" rowspan="1" colspan="1">Mild</td> <td align="center" rowspan="1" colspan="1">Discharged</td> <td align="center" rowspan="1" colspan="1">16</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">39.37</td> <td align="center" rowspan="1" colspan="1">TS</td> <td align="center" rowspan="1" colspan="1">58,570,126</td> <td align="center" rowspan="1" colspan="1">195</td> <td align="center" rowspan="1" colspan="1">0.00</td> <td align="center" rowspan="1" colspan="1">10,243</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">YF-FZ3<a href="#t001fn004">§</a></td> <td align="center" rowspan="1" colspan="1">Mild</td> <td align="center" rowspan="1" colspan="1">Discharged</td> <td align="center" rowspan="1" colspan="1">14</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">39.40</td> <td align="center" rowspan="1" colspan="1">TS</td> <td align="center" rowspan="1" colspan="1">50,462,246</td> <td align="center" rowspan="1" colspan="1">68</td> <td align="center" rowspan="1" colspan="1">0.00</td> <td align="center" rowspan="1" colspan="1">839</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">YF-FZ4<a href="#t001fn003">¶</a></td> <td align="center" rowspan="1" colspan="1">Mild</td> <td align="center" rowspan="1" colspan="1">Discharged</td> <td align="center" rowspan="1" colspan="1">7</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">TS</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">10,222</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">YF-FZ5<a href="#t001fn004">§</a></td> <td align="center" rowspan="1" colspan="1">Mild</td> <td align="center" rowspan="1" colspan="1">Discharged</td> <td align="center" rowspan="1" colspan="1">6</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">TS</td> <td align="center" rowspan="1" colspan="1">56,605,654</td> <td align="center" rowspan="1" colspan="1">68</td> <td align="center" rowspan="1" colspan="1">0.00</td> <td align="center" rowspan="1" colspan="1">2,795</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">YF-FZ6<a href="#t001fn003">¶</a></td> <td align="center" rowspan="1" colspan="1">Mild</td> <td align="center" rowspan="1" colspan="1">Discharged</td> <td align="center" rowspan="1" colspan="1">20</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">32.44</td> <td align="center" rowspan="1" colspan="1">TS</td> <td align="center" rowspan="1" colspan="1">51,627,058</td> <td align="center" rowspan="1" colspan="1">4,722</td> <td align="center" rowspan="1" colspan="1">0.11</td> <td align="center" rowspan="1" colspan="1">10,838</td> </tr> <tr> <td align="center" rowspan="1" colspan="1">YF-FZ7<a href="#t001fn003">¶</a></td> <td align="center" rowspan="1" colspan="1">Mild</td> <td align="center" rowspan="1" colspan="1">Discharged</td> <td align="center" rowspan="1" colspan="1">15</td> <td align="center" rowspan="1" colspan="1">-</td> <td align="center" rowspan="1" colspan="1">33.96</td> <td align="center" rowspan="1" colspan="1">TS</td> <td align="center" rowspan="1" colspan="1">64,345,650</td> <td align="center" rowspan="1" colspan="1">3,945</td> <td align="center" rowspan="1" colspan="1">0.09</td> <td align="center" rowspan="1" colspan="1">10,311</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>* "-�?, Negative results (Ct value &amp;gt;45).</p> </div> <div class="fn-type-" title=""> <p>† AS, amplicon sequencing; TS, total RNA sequencing.</p> </div> <div class="fn-type-" title=""> <p>¶ Genome sequence assembled with Sanger sequencing data.</p> </div> <div class="fn-type-" title=""> <p>§ Genome sequence failed to assemble.</p> </div> </div> </div> </div> <div class="phylogeneticsandsequencecomparisonofyfvgenome" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetics and sequence comparison of YFV genome</div> <p>We performed phylogenetic analysis of the coding region of YFVs (<a href="#pntd.0006738.s005">S2 <i>Table</i></a>), and found that YFV sequences from the returning workers in 2016 outbreak closely clustered with the 1971 Angola strains (<i><a href="#pntd.0006738.g002">Fig 2A</a></i>). Of note, the vaccine strain YF-17D and its derivatives are located far from the Angola strains on the phylogenetic tree. A closer inspection of the Angola strains shows that the viruses in the 2016 outbreak are likely from a single origin and genome sequences from both severely ill patients are closely clustered (<i><a href="#pntd.0006738.g002">Fig 2B</a></i>). There are 188 nucleotide substitutions between the 1971 and 2016 consensus sequences and 6 of them are nonsynonymous substitutions. Comparison of the YFV polypeptides shows that only three amino acid changes that appear to be specific to the Angola 2016 strains (<i><a href="#pntd.0006738.g002">Fig 2C</a></i>). Two are in the capsid protein and one in NS5, the RNA-dependent RNA polymerase. The above data are consistent with previous studies using partial or whole genomes showing that YFV exhibits a slow evolutionary rate [<a href="#pntd.0006738.ref002">2</a>, <a href="#pntd.0006738.ref004">4</a>, <a href="#pntd.0006738.ref010">10</a>].</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pntd.0006738.g002"><span class="label" tagx="label" title="label">Fig 2</span></a></div>   <div class="title" tagx="title" title="title">
Phylogenetics and amino acids comparison of YFV viruses.</div> <p>(A) Maximum-likelihood phylogenetic tree of YFV genome sequences (the first 130 bps in 5’-UTR and the last 370 bps in 3’-UTR sequences are excluded). Bootstrap support values are shown along the branches. Wesselsbron virus, Sepik virus and Entebbe bat virus were used as outgroup. (B) Maximum-likelihood phylogenic tree of YFV genomes of Angola 2016 strains. Viruses from patients with severe disease are highlighted in magenta. Angola 1971(GenBank accession AY968064) was used as outgroup. Bootstrap support values (≥70) are shown. (C) Comparison of amino acids of YFV consensus sequence of each lineage in (A). Identical amino acid to vaccine strain (17D) lineage is denoted as a dot. The amino acid sites that were different in Angola 2016 and 1971, are highlight with red rectangles. Novel amino acid substitutions of Angola 2016 are denoted by arrowheads.</p>   </div> </div> <div class="intrahostgenomicvariations" title="sec"> <div class="title" tagx="title" title="title">
Intrahost genomic variations</div> <p>For the deep-sequenced samples, we identified the iSNVs by using methods in a previous study for Ebola virus [<a href="#pntd.0006738.ref011">11</a>]. The mean sequencing depth of genomic regions is between 16,558x and 37,822x (<i><a href="#pntd.0006738.g003">Fig 3A</a></i>), and a total of 101 iSNV sites were discovered, including 69 in the coding region and 32 in non-coding regions. In each iSNV sites, we only found two types of nucleotides. Taken that sequencing errors are generated randomly and may result in multi-nucleotide heterogeneity in a single site, it is unlikely those iSNVs were the results of sequencing bias. Fewer iSNVs appeared in the 1<sup>st</sup> and 2<sup>nd</sup> codon positions than the 3<sup>rd</sup> codon position (<i><a href="#pntd.0006738.g003">Fig 3B</a></i>) and fewer non-synonymous than synonymous iSNVs (<i><a href="#pntd.0006738.g003">Fig 3C</a></i>), implying that the coding region is generally under purifying selection. We plotted the distributions of all mutated allele frequencies of the iSNVs (<i><a href="#pntd.0006738.g003">Fig 3D and 3E</a></i>). The mean mutated allele frequency of non-coding iSNVs was at 0.18, while that of synonymous and non-synonymous iSNVs was 0.12 and 0.09, respectively (<i><a href="#pntd.0006738.g003">Fig 3D</a></i>). Among them, non-synonymous iSNV is significantly lower than that in non-coding regions (P = 0.02). The distribution of non-coding iSNVs is close to the expected neutrality, whereas curves of synonymous and nonsynonymous iSNVs have higher portions of iSNVs in the area of low mutated allele frequencies (<i><a href="#pntd.0006738.g003">Fig 3E</a></i>). This further supports the notion that the coding region was under purifying selection. Additionally, we discovered two variant types at the 3’ untranslated region (UTR), one with 5 concurrent iSNVs (G10360A, U10365G, C10367U, G10373A, and U10398C) and the other having an additional iSNV (A10425G) (<i><a href="#pntd.0006738.g003">Fig 3F</a></i>). Phasing analysis reveals that these iSNVs tend to be concurrent in the same reads (<a href="#pntd.0006738.s006">S3 <i>Table</i></a><i>and <a href="#pntd.0006738.s001">S1 Fig</a></i>). Predicted RNA structure shows that these substitutions are likely to affect the structure of the 3’ UTR (<i><a href="#pntd.0006738.g003">Fig 3F</a> and <a href="#pntd.0006738.s002">S2 Fig</a></i>), and probably influence viral replication in hosts [<a href="#pntd.0006738.ref012">12</a>–<a href="#pntd.0006738.ref014">14</a>].</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pntd.0006738.g003"><span class="label" tagx="label" title="label">Fig 3</span></a></div>   <div class="title" tagx="title" title="title">
The iSNVs of Angola 2016 YFVs from clinical samples.</div> <p>(A) Sequencing depth across the sequenced genomes. The x-axis represents the YFV genome, with the ORF boundaries indicated by vertical dashed lines. The sequencing depth smoothed by locally weighted smooth regression (LOESS) is shown by the black curve, with 95% confidence interval as shown by shadow. (B) Numbers of iSNVs at non-coding (NC) regions and codon positions of the ORF. (C) Numbers of nonsynonymous (N) and synonymous (S) iSNVs of the ORF. (D) Box plots of the mutated allele frequencies (MuAFs) for non-coding, synonymous and nonsynonymous iSNVs. The MuAF values of three kinds of iSNVs are compared to each other with the Wilcoxon rank-sum test. Dashed lines denote the boundaries of MuAFs in iSNV identification. Boxes denote the interquartile range (IQR) between the first and third quartiles. Lines inside the boxes indicate the median, and the lines outside boxes represent values within 1.5 times the IRQ. Outliers are denoted as dots. (E) Comparison of MuAF spectra of non-coding, synonymous and nonsynonymous iSNVs. MuAFs (dots) are shown with LOESS lines (95% confidence interval in shadow). Expectation under neutral selection is shown by a black line. (F) Prediction of RNA secondary structures of 3’ UTR (with 68 bp downstream of the stop codon). The earliest type of Angola 2016 (wildtype) and two nucleotides variant types are illustrated. Six variation sites are highlighted in magenta. The stop codon is in the gray box.</p>   </div> </div> <div class="intrahostevolutionarydynamics" title="sec"> <div class="title" tagx="title" title="title">
Intrahost evolutionary dynamics</div> <p>Subsequently, we placed all of the polymorphic sites (SNPs and iSNVs) along viral genome (<i><a href="#pntd.0006738.g004">Fig 4A</a></i>). SNPs and iSNVs described the viral difference in two levels of between and within the host, respectively. Within 65 days (from the onset of the first patient to the sample of the last patient in this study) we observed a total of 234 SNPs in 148 sites in 18 samples, including 102 synonymous and 117 nonsynonymous SNPs. A total of 138 SNPs that appear only once are scattered along the viral genome. In particular, the SNPs of 5 sites (T900A, A2352C, C3918A, G6463A, and A7320G) could be used to characterize the 2016 YFVs. All five SNPs were detected in seven patients, four SNPs (without A7320G) were detected in one patient (YF-BJ5), and only one patient (YF-BJ3) does not possess any of the above SNPs (<i><a href="#pntd.0006738.g004">Fig 4A</a></i>). Of note, YF-BJ1 is likely a co-infection with more than one YFV variant, as the virus genome on day 7 does not contain any of the five SNPs, whereas all five SNPs became dominant on day 8 (<i><a href="#pntd.0006738.g004">Fig 4A</a></i>). Of the five SNPs, only G6463A is nonsynonymous, resulting in an amino acid substitution from Val to Ile in NS4A.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pntd.0006738.g004"><span class="label" tagx="label" title="label">Fig 4</span></a></div>   <div class="title" tagx="title" title="title">
Genomic variations of Angola 2016 YFV and evolutionary model.</div> <p>(A) SNPs (in magenta) and iSNVs (in cyan) detected in Angola 2016 strains along the YFV genome. Non-synonymous variations are denoted by ovals, synonymous variations by diamonds, and variations in UTR by rectangles. Non-synonymous site at 6463 is denoted by arrowhead. YFV genome of patient YF-BJ1 at 7 days after the onset (YF-BJ1/7D) was used as the reference sequence (B) Linear regression of number of variations at all variations, synonymous and non-synonymous sites accumulated by days after onset. The smooth shadows denote the LOESS fit with 95% confidence interval.</p>   </div> <p>We then compared the accumulation of polymorphic sites within patients over time (<i><a href="#pntd.0006738.g004">Fig 4B</a></i>). Each sample was set to days after onset of the symptoms of the patients it belongs to. Linear regressions revealed that iSNVs and SNPs accumulated from the day of onset of symptoms at a rate of 0.72 and 0.32 variations/day, respectively. The accumulation rate of iSNV may reflect the intrahost evolution. Intriguingly, the YFV evolutionary rate within humans is roughly 2.48 x10<sup>-2</sup> variations/site/year (0.72 variations/day = 0.72 x 365 / 10591 variations/site/year = 2.48 x10<sup>-2</sup> variations/site/year) in our study. Of those synonymous variations, the intrahost evolutionary rate estimated by iSNV was 0.7 x10<sup>-2</sup> variations/site/year; whereas the rate of nonsynonymous variations was 1.2 x10<sup>-2</sup> variations/site/year. Compared to the rates estimated by SNP within these samples, the intrahost evolutionary rate of non-synonymous sites (by iSNV) is higher, while the rate of synonymous sites is slightly higher than the SNP rate.</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>The evolution of YFV is a basic question to unfold to predict the risk of the pathogens. Phylogenetic analysis suggests that the 2016 outbreak is likely the spillover of YFV viruses in Angola. The high identity between the 2016 strain and the vaccine strain YF-17D would explain the effectiveness of the current vaccines [<a href="#pntd.0006738.ref015">15</a>]. Although there is a diversity of yellow fever virus genotypes, all genotypes are indistinguishable in serological assays. There is not any evidence that the Angola outbreak was due to immune escape and it is accepted among public health professionals that low vaccine coverage in the area prior to the outbreak was the driving factor and the resulting aggressive mass vaccination campaign was vital for containing and eliminating the outbreak. However, it should be noted that the Angola lineage is clearly diversified from the 17D-lineage (<i><a href="#pntd.0006738.g002">Fig 2A</a></i>). Moreover, we did notice 6 patients who had been vaccinated before [<a href="#pntd.0006738.ref016">16</a>], but the limited clinical and epidemiological data cannot tell the immune escape. Future immune escape from current YFV vaccines cannot be fully ruled out, although YFVs evolve very slowly [<a href="#pntd.0006738.ref017">17</a>]. A novel vaccine based on a Central/East African YFV isolate would be necessary as a complement vaccine to prepare against future outbreaks. Owing to the slow evolutionary rate and the past experience of using YF-17D, the novel vaccine is likely to work for another decades.</p> <p>Virus evolution experienced mutation, selection within and between hosts, genetic drift, and transmission, and finally formed the genetic variants (lineages) over long-term evolutionary timescales [<a href="#pntd.0006738.ref018">18</a>]. Deep sequencing of viral genomes can provide novel insights into viral evolutionary dynamics [<a href="#pntd.0006738.ref019">19</a>, <a href="#pntd.0006738.ref020">20</a>]. In this intra-host phylogenomic analysis, we depicted the evolutionary process of YFV from intra-host to epidemic substitutions, which were reflected by iSNV to SNP, respectively. Other than using the metric for long-term evolutionary rate, we applied the substitution/site/day to measure the intrahost evolutionary rate, which appeared more appropriate in outbreak or epidemic scenarios. Short-term rates may provide better insight into the intrahost YFV evolutionary trends (e.g., iSNV dynamics over time) relative to clinical symptomology, severity, and/or co-morbidities. However, further in vitro or in vivo studies would be needed. The estimated evolutionary rates clearly showed that the intrahost evolutionary rate is ~2.25 times higher than that of the epidemic evolutionary rate, and both are much higher (59 times and 26 times) than the substitution rate of YFV along longer evolutionary timescale [<a href="#pntd.0006738.ref010">10</a>]. The higher rate estimated by iSNV reflected purifying selection occurred, when intrahost variations were spreading out. The rates estimated by synonymous iSNV and SNP, respectively, were similar, implying that the synonymous variations were under similar evolutionary constraints during transmission. Meanwhile, only a few of the nonsynonymous variations were spread out in different individuals, and only G6463A (Val to Ile in NS4A) were detected in multiple samples.</p> <p>Compard to Dengue virus (DENV) and Zika virus (ZIKV), YFV shares similar host vectors (<i>Aedes spp</i>.) and life cycles [<a href="#pntd.0006738.ref021">21</a>, <a href="#pntd.0006738.ref022">22</a>], and has similar mutation rates and replication strategies [<a href="#pntd.0006738.ref010">10</a>, <a href="#pntd.0006738.ref023">23</a>, <a href="#pntd.0006738.ref024">24</a>]. However, the long-term substitution rates of DENV (0.77–0.99 x10<sup>-3</sup> substitutions/site/year) and ZIKA (0.98–1.06 x10<sup>-3</sup> substitutions/site/year) are higher than that of YFV [<a href="#pntd.0006738.ref023">23</a>, <a href="#pntd.0006738.ref025">25</a>, <a href="#pntd.0006738.ref026">26</a>]. Previous studies have posed that additional constrained selection forces are probably existed [<a href="#pntd.0006738.ref010">10</a>, <a href="#pntd.0006738.ref027">27</a>]. Based on our results, we noticed the nearly linear accumulation of variations within individuals. Since the transmission of YFV variants needs mosquitoes as vector, the window (viremia duration) that mosquitoes could use is essential, as only the variations within this period could be brought into the transmission cycle. A wider window size would possibly allow more non-synonymous variations into the transmission cycle (Cristina Domingo, EID, 2018). Given that YFV, DENV and ZIKV have ~3, 3–7, and 4–7 days of viremia, respectively, after symptom onset (<i><a href="#pntd.0006738.s007">S4 Table</a></i>), it hinted that the shorter viremia duration in YFV infections might result in the lower evolutionary rate than that of DENV or ZIKV. Most Flaviviruses are vector-borne viruses and the evolutionary rates in mammals and arthropods are different. The intrahost analysis of virus could show the short-term evolution dynamics, especially from serial samples, and this would be a helpful complement to the long-term evolution of Flaviviruses.</p> <p>In addition, deep sequencing and viral genomics have provided new insights into viral evolutionary dynamics, especially in a continuous mode. Our findings strongly suggest that the urine samples should be considered for the clinical diagnosis, genomic surveillance, and large-scale screening of YFV infections. Further investigations are also encouraged to characterize the factors that maintain viremia in vector-borne diseases.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="ethicsstatement" title="sec"> <div class="title" tagx="title" title="title">
Ethics statement</div> <p>The study was conducted in accordance with the Declaration of Helsinki and approved by the ethics committee of Beijing Ditan Hospital (DSRB 2008/00293 and DSRB 2013/00209). Clinical samples and information were obtained after written informed consents from all participants. All human subjects were adult, and if not, whether a parent or guardian of any child participant provided informed consent on their behalf.</p> </div> <div class="patientsanddatacollection" title="sec"> <div class="title" tagx="title" title="title">
Patients and data collection</div> <p>Twelve YFV patients returning from Angola were recruited (<i><a href="#pntd.0006738.t001">Table 1</a></i>). We sampled urine and blood sample of these patients from routine examination and collected their clinical data.</p> </div> <div class="yfvdiagnosisandvirusdetection" title="sec"> <div class="title" tagx="title" title="title">
YFV diagnosis and virus detection</div> <p>YFV infection was diagnosed according to the WHO guidelines (<a href="http://www.who.int/csr/disease/yellowfev/case-definition/en/">http://www.who.int/csr/disease/yellowfev/case-definition/en/</a>, last accessed 10<sup>th</sup> April, 2018), including clinical symptoms (fever, headache, jaundice, etc.) and laboratory detection of virus by using real-time RT PCR. The serum and urine samples were collected and stored at -80 <sup>o</sup>C for further analyses, including real-time RT-PCR, PCR sequencing and RNA sequencing.</p> </div> <div class="nextgenerationsequencing" title="sec"> <div class="title" tagx="title" title="title">
Next generation sequencing</div> <p>RNA from both serum and urine samples were deployed to two approaches of deep sequencing. Approach I: Total RNA sequencing. RNA was purified by Trizol (Thermo Scientific) for each sample, and then total RNA libraries were constructed with 500bps insertion size using the NEBNext UltraTM RNA Library Prep Kit (NEB, MA, USA), and then sequenced by Illumina Hi-seq platform, and generated 2 × 125 bp paired-end reads. Approach II: RNA amplicon sequencing. Two sets of YFV-specific primer pairs were designed (Set A: 5 pairs, amplicon length = 2,248 ± 168 nt; Set B: 12 pairs, amplicon length = 1,018 ± 48 nt; <a href="#pntd.0006738.s008">S5 <i>Table</i></a>). PCR products were sequenced by Illumina platform with 2 × 250 bp paired-end reads. Quality control and error correction were implemented according to previous report [<a href="#pntd.0006738.ref028">28</a>, <a href="#pntd.0006738.ref029">29</a>]. The probably low quality regions in sequencing, including 1) Q&amp;lt;30, and 2) the first 10 bps after primers were removed from the high quality sequences, according to a previous study [<a href="#pntd.0006738.ref028">28</a>]. To minimize sequencing error that affects the accuracy of iSNV calling, we only kept the read-pair that has &amp;gt;100 high quality bases in both ends as clean reads. All reads were deposited in the NCBI SRA database under the accession no. SRP096859.</p> </div> <div class="sequencealignmentandphylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Sequence alignment and phylogenetic analysis</div> <p>Consensus sequences were aligned by MUSCLE [<a href="#pntd.0006738.ref030">30</a>]. Phylogenetic analyses were performed by using RAxML v8.1.6, with the GTR model of nucleotide substitution and γ-distributed rates among sites. Phylogenetic tree was constructed by maximum-likelihood using YFV genome sequences [<a href="#pntd.0006738.ref031">31</a>]. A NJ tree based on YFV genome sequences was also constructed to show the results were robust (<a href="#pntd.0006738.s003">S3 <i>Fig</i></a>). A total of 1,000 bootstrap replicates were performed. All SNPs were listed in <a href="#pntd.0006738.s009">S6 <i>Table</i></a>. The genome sequences of YF-BJ1, YF-BJ2, YF-BJ3, YF-BJ4, YF-BJ5, YF-FZ2, YF-FZ4, YF-FZ6, and YF-FZ7 were deposited in the NCBI GenBank database under the accession no. MH633684-MH633692.</p> </div> <div class="callingofisnvs" title="sec"> <div class="title" tagx="title" title="title">
Calling of iSNVs</div> <p>To minimize the potential sequencing errors generated by system, we performed the following two steps before we called iSNV, including 1) mapping the clean reads to the reference genome, 2) qualification of the samples. For mapping the clean reads to genome, we use the traditional protocol used widely. First, we mapped the clean reads with pair-ended aligned to the assembled genome from YF-BJ1/7D by using Bowtie2 v2.2.5 [<a href="#pntd.0006738.ref032">32</a>] with default parameters. SAMtools v1.2 [<a href="#pntd.0006738.ref033">33</a>] was used to generate ‘mpileup’ files with no limit for the maximum site depth. To find the potential mutation in each site, we do not use the following pipeline in SAMtools. Instead, we developed homemade PERL scripts (available at <a href="http://github.com/generality/iSNV-calling/">http://github.com/generality/iSNV-calling/</a>), which were used for iSNV calling to identify all potential mutations by using the mpileup files as input. The depth of bases in each sites were used to measure the mutation rate of iSNV. Then, we selected the samples with more than 3,000 sites with a sequencing depth ≥300× as candidate samples for iSNV calling. Using this protocol, we called iSNV in 15 samples in this study (<a href="#pntd.0006738.s004">S1 <i>Table</i></a>), and obtained the frequencies of each allele in each genome site. Interestingly, we only found the heterogeneous nucleotides with two types in each sites. Thus, we defined the nucleotide with higher frequencies as the major allele, and fewer one as the minor allele in the following iSNV calling.</p> <p>The iSNV calling was according to the site depth and strand bias with following: 1) remove the ambiguous iSNV and keep the following iSNVs: (1.1) minor allele frequency of ≥5%, a conservative cutoff based on an error rate estimation described before [<a href="#pntd.0006738.ref011">11</a>, <a href="#pntd.0006738.ref028">28</a>]; (1.2) depth of the minor allele of ≥15; and (1.3) strand bias of the minor allele less than tenfold. 2) In our analysis, these two types of nucleotides in each site have contained the nucleotide in the reference genome. Mutated allele frequencies were calculated by the rate of the reads numbers for mutated alleles to the references and total reads. Minor allele and Major allele frequencies were calculated by the majority nucleotides and minor nucleotides in each site respectively. 3) The effect of iSNV to the gene was calculated by the mutation whether caused the amino acid changes.</p> </div> <div class="phasinganalysisofisnvs" title="sec"> <div class="title" tagx="title" title="title">
Phasing analysis of iSNVs</div> <p>The iSNVs that co-occurred in same viral haplotype(s) within quasispecies was defined as phased iSNVs. 1)We screened the iSNV sites along genome with a window of 250 bp, and then identified all the windows which probably containing a phasing iSNVs. 2) All the sequencing reads aligned to YFV genome region harboring these iSNVs were extracted. 3) The stretches of nearest iSNV were determined by the mutations in the reads aligned to the genome region with the same phasing (supporting reads &amp;gt; = 2). 4) The fractions of reads supporting phased and non-phased neighbor iSNVs were respectively calculated.</p> </div> <div class="predictionofrnasecondarystructures" title="sec"> <div class="title" tagx="title" title="title">
Prediction of RNA secondary structures</div> <p>RNA secondary structures were predicted by using RNAfold program of ViennaRNA Package 2 with default settings [<a href="#pntd.0006738.ref034">34</a>]. Nucleotide sequences started at the beginning of 3' UTRs of Angola 2016 strain. The length of window for secondary structure prediction was equal to 70 nucleotides. We also extended the additional 200 nucleotides at the end of the sequence, with a step length equal to 50.</p> </div> <div class="accessionnumbers" title="sec"> <div class="title" tagx="title" title="title">
Accession numbers</div> <p>All reads from next generation sequencing were deposited in the NCBI SRA database under the accession no. SRP096859. The genome sequences of YF-BJ1, YF-BJ2, YF-BJ3, YF-BJ4, YF-BJ5, YF-FZ2, YF-FZ4, YF-FZ6, and YF-FZ7 were deposited in the NCBI GenBank database under the accession no. MH633684-MH633692.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Fig</span> <div class="title" tagx="title" title="title">
Phasing analysis of iSNVs at the 3’ untranslated region (UTR).</div> <p>For a given sample, the sequencing reads aligned to 3’ UTR of the YFV genome were extracted to obtain the 5-mutations (at positions 10360, 10365, 10367, 10373 and 10398) or 6-mutations (10360, 10365, 10367, 10373, 10398 and 10425). The pie charts denote fractions of reads supporting wildtype 5/6-mers and phased mutant 5/6-mers (A10360G, G10365T, T10367C, A10373G, C10398T and G10425A), with read counts (or depth, x) shown in brackets. Sample IDs are indicated at the top of pie charts.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pntd.0006738.s001.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Fig</span> <div class="title" tagx="title" title="title">
RNA secondary structure prediction of the 3’ UTR.</div> <p>RNA secondary structures were predicted based on sequences of YFV 3’ UTR (started from the stop codon) with gradually increased lengths. The sequences were extracted from genomes of wild type Angola 2016 strain and two variant types. Blue arrows indicate variations corresponding to the phased iSNV sites at 3’ UTR, A10360G, G10365T, T10367C, A10373G, C10398T and G10425A. The local structural alterations due to the variations are highlighted in blue shadow.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pntd.0006738.s002.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S3 Fig</span> <div class="title" tagx="title" title="title">
Phylogenetic tree constructed by neighbor-joining method using YFV genome sequences.</div> <p>Viruses from patients with severe disease are highlighted in magenta. Angola 1971(GenBank accession AY968064) was used as outgroup. Bootstrap support values over 70 are shown.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pntd.0006738.s003.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Table</span> <div class="title" tagx="title" title="title">
Sequencing data from continuous clinical samples of YFV.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="pntd.0006738.s004.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Table</span> <div class="title" tagx="title" title="title">
Public sequences used for phylogenic analysis.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="pntd.0006738.s005.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S3 Table</span> <div class="title" tagx="title" title="title">
Phasing analysis for continuous neighbor iSNV.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="pntd.0006738.s006.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S4 Table</span> <div class="title" tagx="title" title="title">
Duration of viremia and evolutionary rates of YFV, ZIKV and DENV.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="pntd.0006738.s007.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S5 Table</span> <div class="title" tagx="title" title="title">
Two primer sets used for amplicon-sequencing.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="pntd.0006738.s008.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S6 Table</span> <div class="title" tagx="title" title="title">
SNPs observed in YFV.</div> <p>(PDF)</p>  <div class="media" title="media"><a href="pntd.0006738.s009.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank Prof. Dr. Gong Cheng from Tsinghua University and Prof. Dr. Bo Zhang from Wuhan Institute of Virology, Chinese Academy of Sciences, for helpful discussion.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pntd.0006738.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Visser</span><span class="given-names" tagx="given-names" title="given-names">LG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roukens</span><span class="given-names" tagx="given-names" title="given-names">AH</span></span>. <span class="mixed-article-title" title="mixed-article-title">Modelling a way out of yellow fever</span>. <span class="source" tagx="source" title="source">Lancet</span><span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">388</span>:<span class="fpage" tagx="fpage" title="fpage">2847</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(16)31330-7">10.1016/S0140-6736(16)31330-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27837922">27837922</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bryant</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barrett</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span>. <span class="mixed-article-title" title="mixed-article-title">Out of Africa: a molecular perspective on the introduction of yellow fever virus into the Americas</span>. <span class="source" tagx="source" title="source">PLoS pathogens</span><span class="year" tagx="year" title="year">2007</span>; <span class="volume" tagx="volume" title="volume">3</span>:<span class="fpage" tagx="fpage" title="fpage">e75</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.0030075">10.1371/journal.ppat.0030075</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17511518">17511518</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mutebi</span><span class="given-names" tagx="given-names" title="given-names">J-P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rijnbrand</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Genetic relationships and evolution of genotypes of yellow fever virus and other members of the yellow fever virus group within the Flavivirus genus based on the 3′ noncoding region</span>. <span class="source" tagx="source" title="source">Journal of virology</span><span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">78</span>:<span class="fpage" tagx="fpage" title="fpage">9652</span>–<span class="lpage" tagx="lpage" title="lpage">65</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.78.18.9652-9665.2004">10.1128/JVI.78.18.9652-9665.2004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15331698">15331698</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stock</span><span class="given-names" tagx="given-names" title="given-names">NK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laraway</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faye</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diallo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Niedrig</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sall</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span>. <span class="mixed-article-title" title="mixed-article-title">Biological and phylogenetic characteristics of yellow fever virus lineages from West Africa</span>. <span class="source" tagx="source" title="source">Journal of virology</span><span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">87</span>:<span class="fpage" tagx="fpage" title="fpage">2895</span>–<span class="lpage" tagx="lpage" title="lpage">907</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01116-12">10.1128/JVI.01116-12</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23269797">23269797</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vasconcelos</span><span class="given-names" tagx="given-names" title="given-names">PF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bryant</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">da Rosa</span><span class="given-names" tagx="given-names" title="given-names">APT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tesh</span><span class="given-names" tagx="given-names" title="given-names">RB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodrigues</span><span class="given-names" tagx="given-names" title="given-names">SG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barrett</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span>. <span class="mixed-article-title" title="mixed-article-title">Genetic divergence and dispersal of yellow fever virus, Brazil</span>. <span class="source" tagx="source" title="source">Emerging infectious diseases</span><span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">1578</span><span class="pub-id"><a href="https://dx.doi.org/10.3201/eid1009.040197">10.3201/eid1009.040197</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15498159">15498159</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kraemer</span><span class="given-names" tagx="given-names" title="given-names">MU</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">NR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reiner</span><span class="given-names" tagx="given-names" title="given-names">RC</span><span class="suffix" tagx="suffix" title="suffix">Jr.</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Spread of yellow fever virus outbreak in Angola and the Democratic Republic of the Congo 2015–16: a modelling study</span>. <span class="source" tagx="source" title="source">The Lancet infectious diseases</span><span class="year" tagx="year" title="year">2016</span>.</span></li> <li tag="ref"><a name="pntd.0006738.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lv</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">A fatal yellow fever virus infection in China: description and lessons</span>. <span class="source" tagx="source" title="source">Emerg Microbes Infect</span><span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">5</span>:<span class="fpage" tagx="fpage" title="fpage">e69</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/emi.2016.89">10.1038/emi.2016.89</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27406389">27406389</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ling</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Yellow Fever in a Worker Returning to China from Angola, March 2016</span>. <span class="source" tagx="source" title="source">Emerging infectious diseases</span><span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">22</span>:<span class="fpage" tagx="fpage" title="fpage">1317</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. <span class="pub-id"><a href="https://dx.doi.org/10.3201/eid2207.160469">10.3201/eid2207.160469</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27314417">27314417</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Yellow fever virus: Increasing imported cases in China</span>. <span class="source" tagx="source" title="source">The Journal of infection</span><span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">73</span>:<span class="fpage" tagx="fpage" title="fpage">377</span>–<span class="lpage" tagx="lpage" title="lpage">80</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jinf.2016.07.003">10.1016/j.jinf.2016.07.003</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27422700">27422700</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sall</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faye</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diallo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Firth</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kitchen</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>. <span class="mixed-article-title" title="mixed-article-title">Yellow fever virus exhibits slower evolutionary dynamics than dengue virus</span>. <span class="source" tagx="source" title="source">Journal of virology</span><span class="year" tagx="year" title="year">2010</span>; <span class="volume" tagx="volume" title="volume">84</span>:<span class="fpage" tagx="fpage" title="fpage">765</span>–<span class="lpage" tagx="lpage" title="lpage">72</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01738-09">10.1128/JVI.01738-09</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19889759">19889759</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ni</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qian</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Intra-host dynamics of Ebola virus during 2014</span>. <span class="source" tagx="source" title="source">Nature microbiology</span><span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">1</span>:<span class="fpage" tagx="fpage" title="fpage">16151</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmicrobiol.2016.151">10.1038/nmicrobiol.2016.151</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27595345">27595345</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Proutski</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gould</span><span class="given-names" tagx="given-names" title="given-names">EA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>. <span class="mixed-article-title" title="mixed-article-title">Secondary structure of the 3' untranslated region of flaviviruses: similarities and differences</span>. <span class="source" tagx="source" title="source">Nucleic acids research</span><span class="year" tagx="year" title="year">1997</span>; <span class="volume" tagx="volume" title="volume">25</span>:<span class="fpage" tagx="fpage" title="fpage">1194</span>–<span class="lpage" tagx="lpage" title="lpage">202</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9092629">9092629</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Villordo</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carballeda</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Filomatori</span><span class="given-names" tagx="given-names" title="given-names">CV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gamarnik</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span>. <span class="mixed-article-title" title="mixed-article-title">RNA Structure Duplications and Flavivirus Host Adaptation</span>. <span class="source" tagx="source" title="source">Trends in microbiology</span><span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">24</span>:<span class="fpage" tagx="fpage" title="fpage">270</span>–<span class="lpage" tagx="lpage" title="lpage">83</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tim.2016.01.002">10.1016/j.tim.2016.01.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26850219">26850219</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goz</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tuller</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>. <span class="mixed-article-title" title="mixed-article-title">Widespread signatures of local mRNA folding structure selection in four Dengue virus serotypes</span>. <span class="source" tagx="source" title="source">BMC genomics</span><span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">16</span>:<span class="fpage" tagx="fpage" title="fpage">S4</span>.</span></li> <li tag="ref"><a name="pntd.0006738.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beck</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tesh</span><span class="given-names" tagx="given-names" title="given-names">RB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">TG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Widen</span><span class="given-names" tagx="given-names" title="given-names">SG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryman</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barrett</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span>. <span class="mixed-article-title" title="mixed-article-title">Comparison of the live attenuated yellow fever vaccine 17D-204 strain to its virulent parental strain Asibi by deep sequencing</span>. <span class="source" tagx="source" title="source">The Journal of infectious diseases</span><span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">209</span>:<span class="fpage" tagx="fpage" title="fpage">334</span>–<span class="lpage" tagx="lpage" title="lpage">44</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jit546">10.1093/infdis/jit546</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24141982">24141982</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">SS</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Late or Lack of Vaccination Linked to Importation of Yellow Fever from Angola to China</span>. <span class="source" tagx="source" title="source">Emerging infectious diseases</span><span class="year" tagx="year" title="year">2018</span>; <span class="volume" tagx="volume" title="volume">24</span>:<span class="fpage" tagx="fpage" title="fpage">1383</span>.</span></li> <li tag="ref"><a name="pntd.0006738.ref017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monath</span><span class="given-names" tagx="given-names" title="given-names">TP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barrett</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span>. <span class="mixed-article-title" title="mixed-article-title">Pathogenesis and pathophysiology of yellow fever</span>. <span class="source" tagx="source" title="source">Adv Virus Res</span><span class="year" tagx="year" title="year">2003</span>; <span class="volume" tagx="volume" title="volume">60</span>:<span class="fpage" tagx="fpage" title="fpage">343</span>–<span class="lpage" tagx="lpage" title="lpage">95</span>. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14689698">14689698</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudas</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andersen</span><span class="given-names" tagx="given-names" title="given-names">KG</span></span>. <span class="mixed-article-title" title="mixed-article-title">The evolution of Ebola virus: Insights from the 2013–2016 epidemic.</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">538</span>:<span class="fpage" tagx="fpage" title="fpage">193</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature19790">10.1038/nature19790</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27734858">27734858</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonaldo</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gómez</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">dos Santos</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Genome analysis of yellow fever virus of the ongoing outbreak in Brazil reveals polymorphisms</span>. <span class="source" tagx="source" title="source">Memórias do Instituto Oswaldo Cruz</span><span class="year" tagx="year" title="year">2017</span>; <span class="volume" tagx="volume" title="volume">112</span>:<span class="fpage" tagx="fpage" title="fpage">447</span>–<span class="lpage" tagx="lpage" title="lpage">51</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1590/0074-02760170134">10.1590/0074-02760170134</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28591405">28591405</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMullan</span><span class="given-names" tagx="given-names" title="given-names">LK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frace</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sammons</span><span class="given-names" tagx="given-names" title="given-names">SA</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Using next generation sequencing to identify yellow fever virus in Uganda</span>. <span class="source" tagx="source" title="source">Virology</span><span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">422</span>:<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">5</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2011.08.024">10.1016/j.virol.2011.08.024</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21962764">21962764</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lee</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>. <span class="mixed-article-title" title="mixed-article-title">Guiding dengue vaccine development using knowledge gained from the success of the yellow fever vaccine</span>. <span class="source" tagx="source" title="source">Cellular &amp;amp; molecular immunology</span><span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">13</span>:<span class="fpage" tagx="fpage" title="fpage">36</span>–<span class="lpage" tagx="lpage" title="lpage">46</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26435066">26435066</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tabachnick</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">Climate Change and the Arboviruses: Lessons from the Evolution of the Dengue and Yellow Fever Viruses</span>. <span class="source" tagx="source" title="source">Annu Rev Virol</span><span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">3</span>:<span class="fpage" tagx="fpage" title="fpage">125</span>–<span class="lpage" tagx="lpage" title="lpage">45</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev-virology-110615-035630">10.1146/annurev-virology-110615-035630</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27482902">27482902</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faria</span><span class="given-names" tagx="given-names" title="given-names">NR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azevedo Rdo</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kraemer</span><span class="given-names" tagx="given-names" title="given-names">MU</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Zika virus in the Americas: Early epidemiological and genetic findings</span>. <span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2016</span>; <span class="volume" tagx="volume" title="volume">352</span>:<span class="fpage" tagx="fpage" title="fpage">345</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.aaf5036">10.1126/science.aaf5036</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27013429">27013429</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mutebi</span><span class="given-names" tagx="given-names" title="given-names">J-P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bryant</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barrett</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span>. <span class="mixed-article-title" title="mixed-article-title">Phylogenetic and evolutionary relationships among yellow fever virus isolates in Africa</span>. <span class="source" tagx="source" title="source">Journal of virology</span><span class="year" tagx="year" title="year">2001</span>; <span class="volume" tagx="volume" title="volume">75</span>:<span class="fpage" tagx="fpage" title="fpage">6999</span>–<span class="lpage" tagx="lpage" title="lpage">7008</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.75.15.6999-7008.2001">10.1128/JVI.75.15.6999-7008.2001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11435580">11435580</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allicock</span><span class="given-names" tagx="given-names" title="given-names">OM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tatem</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Phylogeography and population dynamics of dengue viruses in the Americas</span>. <span class="source" tagx="source" title="source">Molecular biology and evolution</span><span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">29</span>:<span class="fpage" tagx="fpage" title="fpage">1533</span>–<span class="lpage" tagx="lpage" title="lpage">43</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msr320">10.1093/molbev/msr320</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22319149">22319149</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goz</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsalenchuck</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benaroya</span><span class="given-names" tagx="given-names" title="given-names">RO</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Generation and comparative genomics of synthetic dengue viruses</span>. <span class="source" tagx="source" title="source">BMC bioinformatics</span><span class="year" tagx="year" title="year">2018</span>; <span class="volume" tagx="volume" title="volume">19</span>:<span class="fpage" tagx="fpage" title="fpage">140</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12859-018-2132-3">10.1186/s12859-018-2132-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29745863">29745863</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woelk</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>. <span class="mixed-article-title" title="mixed-article-title">Reduced positive selection in vector-borne RNA viruses</span>. <span class="source" tagx="source" title="source">Molecular biology and evolution</span><span class="year" tagx="year" title="year">2002</span>; <span class="volume" tagx="volume" title="volume">19</span>:<span class="fpage" tagx="fpage" title="fpage">2333</span>–<span class="lpage" tagx="lpage" title="lpage">6</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/oxfordjournals.molbev.a004059">10.1093/oxfordjournals.molbev.a004059</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12446826">12446826</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schirmer</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ijaz</span><span class="given-names" tagx="given-names" title="given-names">UZ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">D'Amore</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sloan</span><span class="given-names" tagx="given-names" title="given-names">WT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Quince</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>. <span class="mixed-article-title" title="mixed-article-title">Insight into biases and sequencing errors for amplicon sequencing with the Illumina MiSeq platform</span>. <span class="source" tagx="source" title="source">Nucleic acids research</span><span class="year" tagx="year" title="year">2015</span>; <span class="volume" tagx="volume" title="volume">43</span>:<span class="fpage" tagx="fpage" title="fpage">e37</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gku1341">10.1093/nar/gku1341</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25586220">25586220</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nikolenko</span><span class="given-names" tagx="given-names" title="given-names">SI</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Korobeynikov</span><span class="given-names" tagx="given-names" title="given-names">AI</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alekseyev</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>. <span class="mixed-article-title" title="mixed-article-title">BayesHammer: Bayesian clustering for error correction in single-cell sequencing</span>. <span class="source" tagx="source" title="source">BMC Genomics</span><span class="year" tagx="year" title="year">2013</span>; <span class="volume" tagx="volume" title="volume">14</span><span class="issue" tagx="issue" title="issue">Suppl 1</span>:<span class="fpage" tagx="fpage" title="fpage">S7</span>.</span></li> <li tag="ref"><a name="pntd.0006738.ref030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edgar</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span>. <span class="mixed-article-title" title="mixed-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. <span class="source" tagx="source" title="source">Nucleic acids research</span><span class="year" tagx="year" title="year">2004</span>; <span class="volume" tagx="volume" title="volume">32</span>:<span class="fpage" tagx="fpage" title="fpage">1792</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkh340">10.1093/nar/gkh340</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147">15034147</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stamatakis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2006</span>; <span class="volume" tagx="volume" title="volume">22</span>:<span class="fpage" tagx="fpage" title="fpage">2688</span>–<span class="lpage" tagx="lpage" title="lpage">90</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btl446">10.1093/bioinformatics/btl446</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16928733">16928733</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langmead</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salzberg</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span>. <span class="mixed-article-title" title="mixed-article-title">Fast gapped-read alignment with Bowtie 2</span>. <span class="source" tagx="source" title="source">Nature methods</span><span class="year" tagx="year" title="year">2012</span>; <span class="volume" tagx="volume" title="volume">9</span>:<span class="fpage" tagx="fpage" title="fpage">357</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.1923">10.1038/nmeth.1923</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286">22388286</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Handsaker</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wysoker</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The Sequence Alignment/Map format and SAMtools</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2009</span>; <span class="volume" tagx="volume" title="volume">25</span>:<span class="fpage" tagx="fpage" title="fpage">2078</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btp352">10.1093/bioinformatics/btp352</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943">19505943</a></span></span></li> <li tag="ref"><a name="pntd.0006738.ref034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lorenz</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bernhart</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Honer Zu Siederdissen</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">ViennaRNA Package 2.0</span>. <span class="source" tagx="source" title="source">Algorithms Mol Biol</span><span class="year" tagx="year" title="year">2011</span>; <span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">26</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1748-7188-6-26">10.1186/1748-7188-6-26</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22115189">22115189</a></span></span></li> </ul> </div> </div>  </body></html>