<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/05/05/315457.atom" />
<meta name="HW.pisa" content="biorxiv;315457v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome Sequence of Indian Peacock Reveals the Peculiar Case of a Glittering Bird" />
<meta name="DC.Identifier" content="10.1101/315457" />
<meta name="DC.Date" content="2018-05-05" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The unique ornamental features and extreme sexual traits of Peacock have always intrigued the scientists. However, the genomic evidence to explain its phenotype are yet unknown. Thus, we report the first genome sequence and comparative analysis of peacock with the available high-quality genomes of chicken, turkey, duck, flycatcher and zebra finch. The candidate genes involved in early developmental pathways including TGF-β, BMP, and Wnt signaling pathway, which are also involved in feather patterning, bone morphogenesis, and skeletal muscle development, showed signs of adaptive evolution and provided useful clues on the phenotype of peacock. The innate and adaptive immune components such as complement system and T-cell response also showed signs of adaptive evolution in peacock suggesting their possible role in building a robust immune system which is consistent with the between species predictions of Hamilton-Zuk hypothesis. This study provides novel genomic and evolutionary insights into the molecular understanding towards the phenotypic evolution of Indian peacock." />
<meta name="DC.Contributor" content="Shubham K. Jaiswal" />
<meta name="DC.Contributor" content="Ankit Gupta" />
<meta name="DC.Contributor" content="Rituja Saxena" />
<meta name="DC.Contributor" content="P. K. Vishnu Prasoodanan" />
<meta name="DC.Contributor" content="Ashok K. Sharma" />
<meta name="DC.Contributor" content="Parul Mittal" />
<meta name="DC.Contributor" content="Ankita Roy" />
<meta name="DC.Contributor" content="Aaron B.A. Shafer" />
<meta name="DC.Contributor" content="Nagarjun Vijay" />
<meta name="DC.Contributor" content="Vineet K. Sharma" />
<meta name="article:published_time" content="2018-05-05" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Genome Sequence of Indian Peacock Reveals the Peculiar Case of a Glittering Bird" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;ABSTRACT&lt;/h3&gt;
&lt;p&gt;The unique ornamental features and extreme sexual traits of Peacock have always intrigued the scientists. However, the genomic evidence to explain its phenotype are yet unknown. Thus, we report the first genome sequence and comparative analysis of peacock with the available high-quality genomes of chicken, turkey, duck, flycatcher and zebra finch. The candidate genes involved in early developmental pathways including TGF-β, BMP, and Wnt signaling pathway, which are also involved in feather patterning, bone morphogenesis, and skeletal muscle development, showed signs of adaptive evolution and provided useful clues on the phenotype of peacock. The innate and adaptive immune components such as complement system and T-cell response also showed signs of adaptive evolution in peacock suggesting their possible role in building a robust immune system which is consistent with the between species predictions of Hamilton-Zuk hypothesis. This study provides novel genomic and evolutionary insights into the molecular understanding towards the phenotypic evolution of Indian peacock.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;315457v1" />
<meta name="citation_id" content="315457v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/315457v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/315457v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/315457v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457.full.pdf" />
<meta name="citation_doi" content="10.1101/315457" />
<meta name="citation_num_pages" content="24" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="315457" />
<meta name="citation_author" content="Shubham K. Jaiswal" />
<meta name="citation_author_institution" content="Metagenomics and Systems Biology Group, Department of Biological Sciences, Indian Institute of Science Education and Research Bhopal" />
<meta name="citation_author" content="Ankit Gupta" />
<meta name="citation_author_institution" content="Metagenomics and Systems Biology Group, Department of Biological Sciences, Indian Institute of Science Education and Research Bhopal" />
<meta name="citation_author" content="Rituja Saxena" />
<meta name="citation_author_institution" content="Metagenomics and Systems Biology Group, Department of Biological Sciences, Indian Institute of Science Education and Research Bhopal" />
<meta name="citation_author" content="P. K. Vishnu Prasoodanan" />
<meta name="citation_author_institution" content="Metagenomics and Systems Biology Group, Department of Biological Sciences, Indian Institute of Science Education and Research Bhopal" />
<meta name="citation_author" content="Ashok K. Sharma" />
<meta name="citation_author_institution" content="Metagenomics and Systems Biology Group, Department of Biological Sciences, Indian Institute of Science Education and Research Bhopal" />
<meta name="citation_author" content="Parul Mittal" />
<meta name="citation_author_institution" content="Metagenomics and Systems Biology Group, Department of Biological Sciences, Indian Institute of Science Education and Research Bhopal" />
<meta name="citation_author" content="Ankita Roy" />
<meta name="citation_author_institution" content="Metagenomics and Systems Biology Group, Department of Biological Sciences, Indian Institute of Science Education and Research Bhopal" />
<meta name="citation_author" content="Aaron B.A. Shafer" />
<meta name="citation_author_institution" content="Forensic Science and Environmental &amp; Life Sciences, Trent University" />
<meta name="citation_author" content="Nagarjun Vijay" />
<meta name="citation_author_institution" content="Computational Evolutionary Genomics Lab, Department of Biological Sciences, Indian Institute of Science Education and Research Bhopal" />
<meta name="citation_author" content="Vineet K. Sharma" />
<meta name="citation_author_institution" content="Metagenomics and Systems Biology Group, Department of Biological Sciences, Indian Institute of Science Education and Research Bhopal" />
<meta name="citation_author_email" content="vineetks@iiserb.ac.in" />
<meta name="citation_reference" content="Alexander, D.J. (2000). A review of avian influenza in different bird species. Vet Microbiol 74, 3–13." />
<meta name="citation_reference" content="Asratian, A.A., and Vasil’;eva, V.I. (1976). [Immunological structure of the population of Erevan with regard to Mycoplasma hominis]. Zh Eksp Klin Med 16, 59–61." />
<meta name="citation_reference" content="Balenger, S.L., and Zuk, M. (2014). &quot;Testing the Hamilton–Zuk hypothesis: past, present, and future&quot;. The Society for Integrative and Comparative Biology)." />
<meta name="citation_reference" content="Berg, T.P. (2000). Acute infectious bursal disease in poultry: a review. Avian Pathol 29, 175–194." />
<meta name="citation_reference" content="Bintanja, R., and Van De Wal, R. (2008). North American ice-sheet dynamics and the onset of 100,000-year glacial cycles. Nature 454, 869–872." />
<meta name="citation_reference" content="Bornelöv, S., Seroussi, E., Yosefi, S., Pendavis, K., Burgess, S.C., Grabherr, M., Friedman-Einat, M., and Andersson, L. (2017). Correspondence on Lovell et al.: identification of chicken genes previously assumed to be evolutionarily lost. Genome biology 18, 112." />
<meta name="citation_reference" content="Brack, A.S., Conboy, M.J., Roy, S., Lee, M., Kuo, C.J., Keller, C., and Rando, T.A. (2007). Increased Wnt signaling during aging alters muscle stem cell fate and increases fibrosis. Science 317, 807–810." />
<meta name="citation_reference" content="Chuong, C.M., Chodankar, R., Widelitz, R.B., and Jiang, T.X. (2000). Evo-devo of feathers and scales: building complex epithelial appendages. Curr Opin Genet Dev 10, 449–456." />
<meta name="citation_reference" content="Crowe, R., Henrique, D., Ish-Horowicz, D., and Niswander, L. (1998). A new role for Notch and Delta in cell fate decisions: patterning the feather array. Development 125, 767–775." />
<meta name="citation_reference" content="Denton, J.F., Lugo-Martinez, J., Tucker, A.E., Schrider, D.R., Warren, W.C., and Hahn, M.W. (2014). Extensive error in the number of genes inferred from draft genome assemblies. PLoS computational biology 10, e1003998." />
<meta name="citation_reference" content="Edgar, R.C. (2004). MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res 32, 1792–1797." />
<meta name="citation_reference" content="Ekblom, R., French, L., Slate, J., and Burke, T. (2010). Evolutionary analysis and expression profiling of zebra finch immune genes. Genome Biol Evol 2, 781–790." />
<meta name="citation_reference" content="Fayeye, T., Ayorinde, K., Ojo, V., and Adesina, O. (2006). Frequency and influence of some major genes on body weight and body size parameters of Nigerian local chickens. Livestock research for rural development 18, 37." />
<meta name="citation_reference" content="Groppe, J., Greenwald, J., Wiater, E., Rodriguez-Leon, J., Economides, A.N., Kwiatkowski, W., Affolter, M., Vale, W.W., Izpisua Belmonte, J.C., and Choe, S. (2002). Structural basis of BMP signalling inhibition by the cystine knot protein Noggin. Nature 420, 636–642." />
<meta name="citation_reference" content="Guindon, S., Dufayard, J.F., Lefort, V., Anisimova, M., Hordijk, W., and Gascuel, O. (2010). New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0. Syst Biol 59, 307–321." />
<meta name="citation_reference" content="Hallier, M., Lerga, A., Barnache, S., Tavitian, A., and Moreau-Gachelin, F. (1998). The transcription factor Spi-1/PU.1 interacts with the potential splicing factor TLS. J Biol Chem 273, 4838–4842." />
<meta name="citation_reference" content="Han, M.V., Thomas, G.W., Lugo-Martinez, J., and Hahn, M.W. (2013). Estimating gene gain and loss rates in the presence of error in genome assembly and annotation using CAFE 3. Mol Biol Evol 30, 1987–1997." />
<meta name="citation_reference" content="Hedges, S.B., Dudley, J., and Kumar, S. (2006). TimeTree: a public knowledge-base of divergence times among organisms. Bioinformatics 22, 2971–2972." />
<meta name="citation_reference" content="Huang, Y., Li, Y., Burt, D.W., Chen, H., Zhang, Y., Qian, W., Kim, H., Gan, S., Zhao, Y., and Li, J. (2013). The duck genome and transcriptome provide insight into an avian influenza virus reservoir species. Nature genetics 45, 776." />
<meta name="citation_reference" content="Huerta-Cepas, J., Szklarczyk, D., Forslund, K., Cook, H., Heller, D., Walter, M.C., Rattei, T., Mende, D.R., Sunagawa, S., Kuhn, M., Jensen, L.J., Von Mering, C., and Bork, P. (2016). eggNOG 4.5: a hierarchical orthology framework with improved functional annotations for eukaryotic, prokaryotic and viral sequences. Nucleic Acids Res 44, D286–293." />
<meta name="citation_reference" content="Huxley, T.H. (1968). On the origin of species. University of Michigan P." />
<meta name="citation_reference" content="Kaiser, V.B., Van Tuinen, M., and Ellegren, H. (2007). Insertion events of CR1 retrotransposable elements elucidate the phylogenetic branching order in galliform birds. Mol Biol Evol 24, 338–347." />
<meta name="citation_reference" content="Kaisho, T., and Akira, S. (2006). Toll-like receptor function and signaling. J Allergy Clin Immunol 117, 979–987; quiz 988." />
<meta name="citation_reference" content="Kan, X.-Z., Li, X.-F., Lei, Z.-P., Chen, L., Gao, H., Yang, Z.-Y., Yang, J.-K., Guo, Z.-C., Yu, L., and Zhang, L.-Q. (2010). Estimation of divergence times for major lineages of galliform birds: evidence from complete mitochondrial genome sequences. African Journal of Biotechnology 9, 3073–3078." />
<meta name="citation_reference" content="Kanehisa, M., and Goto, S. (2000). KEGG: kyoto encyclopedia of genes and genomes. Nucleic Acids Res 28, 27–30." />
<meta name="citation_reference" content="Kimball, R.T., Braun, E.L., Zwartjes, P.W., Crowe, T.M., and Ligon, J.D. (1999). A molecular phylogeny of the pheasants and partridges suggests that these lineages are not monophyletic. Mol Phylogenet Evol 11, 38–54." />
<meta name="citation_reference" content="Klaus, A., and Birchmeier, W. (2008). Wnt signalling and its impact on development and cancer. Nat Rev Cancer 8, 387–398." />
<meta name="citation_reference" content="Kumar, P., Henikoff, S., and Ng, P.C. (2009). Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm. Nat Protoc 4, 1073–1081." />
<meta name="citation_reference" content="Li, H., and Durbin, R. (2011). Inference of human population history from individual whole-genome sequences. Nature 475, 493–496." />
<meta name="citation_reference" content="Lim, X., and Nusse, R. (2013). Wnt signaling in skin development, homeostasis, and disease. Cold Spring Harb Perspect Biol 5." />
<meta name="citation_reference" content="Liu, J., Xiao, H., Lei, F., Zhu, Q., Qin, K., Zhang, X.W., Zhang, X.L., Zhao, D., Wang, G., Feng, Y., Ma, J., Liu, W., Wang, J., and Gao, G.F. (2005). Highly pathogenic H5N1 influenza virus infection in migratory birds. Science 309, 1206." />
<meta name="citation_reference" content="Lovell, P.V., Wirthlin, M., Wilhelm, L., Minx, P., Lazar, N.H., Carbone, L., Warren, W.C., and Mello, C.V. (2014). Conserved syntenic clusters of protein coding genes are missing in birds. Genome biology 15, 565." />
<meta name="citation_reference" content="Loveridge, N., Farquharson, C., Hesketh, J.E., Jakowlew, S.B., Whitehead, C.C., and Thorp, B.H. (1993). The control of chondrocyte differentiation during endochondral bone growth in vivo: changes in TGF-beta and the proto-oncogene c-myc. J Cell Sci 105 ( Pt 4), 949–956." />
<meta name="citation_reference" content="Loyau, A., Jaime, M.S., and Sorci, G. (2005a). Intra- and Intersexual Selection for Multiple Traits in the Peacock (Pavo cristatus). Ethology 111, 810–820." />
<meta name="citation_reference" content="Loyau, A., Saint Jaime, M., Cagniant, C., and Sorci, G. (2005b). Multiple sexual advertisements honestly reflect health status in peacocks (Pavo cristatus). Behavioral Ecology and Sociobiology 58, 552–557." />
<meta name="citation_reference" content="Moyle, W.R., Campbell, R.K., Myers, R.V., Bernard, M.P., Han, Y., and Wang, X. (1994). Co-evolution of ligand-receptor pairs. Nature 368, 251–255." />
<meta name="citation_reference" content="Nadachowska-Brzyska, K., Li, C., Smeds, L., Zhang, G., and Ellegren, H. (2015). Temporal dynamics of avian populations during Pleistocene revealed by whole-genome sequences. Current Biology 25, 1375–1380." />
<meta name="citation_reference" content="Nadachowska-Brzyska, K., Burri, R., Smeds, L., and Ellegren, H. (2016). PSMC analysis of effective population sizes in molecular ecology and its application to black-and-white Ficedula flycatchers. Molecular Ecology 25, 1058–1072." />
<meta name="citation_reference" content="Nie, X., Luukko, K., and Kettunen, P. (2006). BMP signalling in craniofacial development. Int J Dev Biol 50, 511–521." />
<meta name="citation_reference" content="Nishimura, R., Hata, K., Matsubara, T., Wakabayashi, M., and Yoneda, T. (2012). Regulation of bone and cartilage development by network between BMP signalling and transcription factors. J Biochem 151, 247–254." />
<meta name="citation_reference" content="O’ leary, N.A., Wright, M.W., Brister, J.R., Ciufo, S., Haddad, D., Mcveigh, R., Rajput, B., Robbertse, B., Smith-White, B., Ako-Adjei, D., Astashyn, A., Badretdin, A., Bao, Y., Blinkova, O., Brover, V., Chetvernin, V., Choi, J., Cox, E., Ermolaeva, O., Farrell, C.M., Goldfarb, T., Gupta, T., Haft, D., Hatcher, E., Hlavina, W., Joardar, V.S., Kodali, V.K., Li, W., Maglott, D., Masterson, P., Mcgarvey, K.M., Murphy, M.R., O’neill, K., Pujar, S., Rangwala, S.H., Rausch, D., Riddick, L.D., Schoch, C., Shkeda, A., Storz, S.S., Sun, H., Thibaud-Nissen, F., Tolstoy, I., Tully, R.E., Vatsan, A.R., Wallin, C., Webb, D., Wu, W., Landrum, M.J., Kimchi, A., Tatusova, T., Dicuccio, M., Kitts, P., Murphy, T.D., and Pruitt, K.D. (2016). Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation. Nucleic Acids Res 44, D733–745." />
<meta name="citation_reference" content="Ouyang, Y.-N., Yang, Z.-Y., Li, D.-L., Huo, J.-L., Qian, K., and Miao, Y.-W. (2009). Genetic Divergence between Pavo muticus and Pavo cristatus by Cyt b Gene. Journal of Yunnan Agricultural University 2, 014." />
<meta name="citation_reference" content="Owens, I.P., and Short, R.V. (1995). Hormonal basis of sexual dimorphism in birds: implications for new theories of sexual selection. Trends Ecol Evol 10, 44–47." />
<meta name="citation_reference" content="Pixley, F.J., and Stanley, E.R. (2004). CSF-1 regulation of the wandering macrophage: complexity in action. Trends Cell Biol 14, 628–638." />
<meta name="citation_reference" content="Pownall, M.E., and Isaacs, H.V. (Year). “Fgf signalling in vertebrate development”, in: Colloquium Series on Developmental Biology: Morgan &amp; Claypool Life Sciences), 1–75." />
<meta name="citation_reference" content="Ramesh, K., and Mcgowan, P. (2009). On the current status of Indian peafowl Pavo cristatus (Aves: Galliformes: Phasianidae): keeping the common species common. Journal of Threatened Taxa 1, 106–108." />
<meta name="citation_reference" content="Rice, P., Longden, I., and Bleasby, A. (2000). EMBOSS: the European Molecular Biology Open Software Suite. Trends Genet 16, 276–277." />
<meta name="citation_reference" content="Serna, M., Giles, J.L., Morgan, B.P., and Bubeck, D. (2016). Structural basis of complement membrane attack complex formation. Nat Commun 7, 10587." />
<meta name="citation_reference" content="Stock, A.D., and Bunch, T.D. (1982). The evolutionary implications of chromosome banding pattern homologies in the bird order Galliformes. Cytogenet Cell Genet 34, 136–148." />
<meta name="citation_reference" content="Sun, L., Zhang, G.H., Jiang, J.W., Fu, J.D., Ren, T., Cao, W.S., Xin, C.A., Liao, M., and Liu, W.J. (2007). A Massachusetts prototype like coronavirus isolated from wild peafowls is pathogenic to chickens. Virus Res 130 121–128." />
<meta name="citation_reference" content="Vilella, A.J., Severin, J., Ureta-Vidal, A., Heng, L., Durbin, R., and Birney, E. (2009). EnsemblCompara GeneTrees: Complete, duplication-aware phylogenetic trees in vertebrates. Genome Res 19, 327–335." />
<meta name="citation_reference" content="Walunas, T.L., Lenschow, D.J., Bakker, C.Y., Linsley, P.S., Freeman, G.J., Green, J.M., Thompson, C.B., and Bluestone, J.A. (1994). CTLA-4 can function as a negative regulator of T cell activation. Immunity 1, 405–413." />
<meta name="citation_reference" content="Wang, N., Kimball, R.T., Braun, E.L., Liang, B., and Zhang, Z. (2013). Assessing phylogenetic relationships among galliformes: a multigene phylogeny with expanded taxon sampling in Phasianidae. PLoS One 8, e64312." />
<meta name="citation_reference" content="Wright, A.E., Harrison, P.W., Zimmer, F., Montgomery, S.H., Pointer, M.A., and Mank, J.E. (2015). Variation in promiscuity and sexual selection drives avian rate of Faster-Z evolution. Molecular Ecology 24, 1218–1235." />
<meta name="citation_reference" content="Yang, Z. (2007). PAML 4: phylogenetic analysis by maximum likelihood. Mol Biol Evol 24, 1586–1591." />
<meta name="citation_reference" content="Yang, Z., Wong, W.S., and Nielsen, R. (2005). Bayes empirical bayes inference of amino acid sites under positive selection. Mol Biol Evol 22, 1107–1118." />
<meta name="citation_reference" content="Zahavi, A. (1975). Mate selection—A selection for a handicap. Journal of Theoretical Biology 53, 205–214." />
<meta name="citation_reference" content="Zhang, J., Nielsen, R., and Yang, Z. (2005). Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level. Mol Biol Evol 22, 2472–2479." />
<meta name="citation_reference" content="Zhang, L., Mo, J., Swanson, K.V., Wen, H., Petrucelli, A., Gregory, S.M., Zhang, Z., Schneider, M., Jiang, Y., and Fitzgerald, K.A. (2014). NLRC3, a member of the NLR family of proteins, is a negative regulator of innate immune signaling induced by the DNA sensor STING. Immunity 40, 329–341." />
<meta name="twitter:title" content="Genome Sequence of Indian Peacock Reveals the Peculiar Case of a Glittering Bird" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The unique ornamental features and extreme sexual traits of Peacock have always intrigued the scientists. However, the genomic evidence to explain its phenotype are yet unknown. Thus, we report the first genome sequence and comparative analysis of peacock with the available high-quality genomes of chicken, turkey, duck, flycatcher and zebra finch. The candidate genes involved in early developmental pathways including TGF-β, BMP, and Wnt signaling pathway, which are also involved in feather patterning, bone morphogenesis, and skeletal muscle development, showed signs of adaptive evolution and provided useful clues on the phenotype of peacock. The innate and adaptive immune components such as complement system and T-cell response also showed signs of adaptive evolution in peacock suggesting their possible role in building a robust immune system which is consistent with the between species predictions of Hamilton-Zuk hypothesis. This study provides novel genomic and evolutionary insights into the molecular understanding towards the phenotypic evolution of Indian peacock." />
<meta name="og-title" property="og:title" content="Genome Sequence of Indian Peacock Reveals the Peculiar Case of a Glittering Bird" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/315457v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The unique ornamental features and extreme sexual traits of Peacock have always intrigued the scientists. However, the genomic evidence to explain its phenotype are yet unknown. Thus, we report the first genome sequence and comparative analysis of peacock with the available high-quality genomes of chicken, turkey, duck, flycatcher and zebra finch. The candidate genes involved in early developmental pathways including TGF-β, BMP, and Wnt signaling pathway, which are also involved in feather patterning, bone morphogenesis, and skeletal muscle development, showed signs of adaptive evolution and provided useful clues on the phenotype of peacock. The innate and adaptive immune components such as complement system and T-cell response also showed signs of adaptive evolution in peacock suggesting their possible role in building a robust immune system which is consistent with the between species predictions of Hamilton-Zuk hypothesis. This study provides novel genomic and evolutionary insights into the molecular understanding towards the phenotypic evolution of Indian peacock." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-05-05" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/315457v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/315457v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/315457v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/315457v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/96725" />
    <title>Genome Sequence of Indian Peacock Reveals the Peculiar Case of a Glittering Bird | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"WxcEtT-P9xahoPInS3SRP7uueOGL1Dw4YUQNBAo4OSY","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"96725","apath":"\/biorxiv\/early\/2018\/05\/05\/315457.atom","pisa":"biorxiv;315457v1","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;315457\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;315457\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;315457v1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;315457\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;315457\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/315457v1","title":"Genome Sequence of Indian Peacock Reveals the Peculiar Case of a Glittering Bird","identifier":"node\/96725"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220tiZmzOVSdz1qsek9izPAtM7yyfwyx8pYtRcYgnJRq5ZuhGHk11oiMyMeESSq3tFDqCm39I3cs1Xvw33DDRMZdtZmqzQQ\\\\\\\/hHatYskFzm\\\\\\\/G7GTCuQ6pkVcfM5vWogbubSZogVkCwKlPe6ntnDUoCdfk63kBrfwAeVZSiDy\\\\\\\/khtmTu\\\\\\\/GjmKZNEdV8QJzUFcv0SggOxykcg+4pUEcQAkSAb6NHD15nXdOAUHbf7zknXYecJd0q+AXXKcedhZscdROyUDf\\\\\\\/sqZR01k9cjpRMQqs5MPNI1qLm7P5dX3llh34NhPkF\\\\\\\/yk6NWv1jCCU4xO8ra87awHqzxCmjW2HsDhLjF9xTKuIyO9\\\\\\\/x\\\\\\\/ZZ6ebrOlHTxzh4SrUBQRriGi14vAwFvVc97\\\\\\\/BLuLlvw5BO077aynAxE2NKF0l0NzV5bPj\\\\\\\/\\\\\\\/PTi906QTiXyU2JzIznEkRJEpkyEU33qUx6NZ45Nta1J8RAFns95WowbnEGDfl2ZNt+B5qZEp6t1IP8609W9FhfA0qYIv9GB5oYJoJxEQ3f\\\\\\\/RO5ebGkSp00CHR0EK5RZDjLcMQXDwzU3S7YXy2b9ss7FMTNa7EV9Petfhh9y\\\\\\\/6ejUfNlqbGXjTRzhW2CJWBcH\\\\\\\/O\\\\\\\/\\\\\\\/iVjctffZTzhKpuZKju9GNG7wLuMKdLAFu+VN\\\\\\\/JCGRx12aLIO0FwDaVsMC4wJYxiEEelQkRv8pF+cgFD8fCw3iaaMq1h7cedLfVAfJYlSB9IOFOIpbMmWXfBx690HJ67GObNNJWJMh2ZUuVO4Ekrl9l5SQcT5JV50QImpf8dj0N6TkNY\\\\\\\/tBwoPx6yp\\\\\\\/LZiDcli6li6gCZOPxNfStkj\\\\\\\/UHOxQjFRNzXk1QlYa6qka3nZiHRGR26GBDvYkLYdZyR\\\\\\\/YYyxipgC75n8PfZyUcBRgkOQPSeY2xnSero8w5pcEduM4IhRLTnQHPnxuNOqIT6vzYwqGdvKfqmzqlh+lsIopq80PkTLfwKn+QWaAerLkIAtpSL7ZrmGmOWRMAMeW+2GSErBihjzb8y4WqDtmGXrZ4OjheY9hjT8bum1QYGrq+BcvBwwEwv3ZV9jiSHuydxZ0TmIemOCBrMiRtsFAdhVT9fewoQ1il0OJR4Hr7b8EEJQgJHgQ6k2aI1\\\\\\\/ZYbkQxZ4gJr5sCqyhkRqP8+jTbLpgPBaHtVac9ALSbaisOd9JfEuIwMsPekGHhogxBgfpz2HHIim9\\\\\\\/V7Y+AnX6d1+fnb2WTxEtjd2RJPHya4ODFIt8MVtlJOBVfdBqIvPs2pHl2hHMPgGkQcPOxqQqmyFc3crDaKBBOmJHYKlGMsDm5GixQ4gZr6NvOc4Mupz+hHLymfyrKI=\\\u0022,\\\u0022iv\\\u0022:\\\u0022HiHQajz5FiLoh7BLKo4kUA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022af7ad5d19105400c7279559787a40968f69862be534c39e04e38b6a8677c9afd\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/315457v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-96725 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/315457v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NtLj9_L3GAXJ3qTpBwl-_AaQm_huIUgX25VGB_ijPWg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="96725" id="node96725" data-pisa="biorxiv;315457v1" data-pisa-master="biorxiv;315457" data-apath="/biorxiv/early/2018/05/05/315457.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Genome Sequence of Indian Peacock Reveals the Peculiar Case of a Glittering Bird</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Shubham K.</span> <span class="nlm-surname">Jaiswal</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Ankit</span> <span class="nlm-surname">Gupta</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Rituja</span> <span class="nlm-surname">Saxena</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">P. K.</span> <span class="nlm-surname">Vishnu Prasoodanan</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Ashok K.</span> <span class="nlm-surname">Sharma</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Parul</span> <span class="nlm-surname">Mittal</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Ankita</span> <span class="nlm-surname">Roy</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Aaron B.A.</span> <span class="nlm-surname">Shafer</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Nagarjun</span> <span class="nlm-surname">Vijay</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Vineet K.</span> <span class="nlm-surname">Sharma</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/315457 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node96725" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shubham K. Jaiswal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Metagenomics and Systems Biology Group, Department of Biological Sciences, Indian Institute of Science Education and Research Bhopal</span>, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shubham%2BK.%2BJaiswal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jaiswal%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShubham%2BK.%2BJaiswal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ankit Gupta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Metagenomics and Systems Biology Group, Department of Biological Sciences, Indian Institute of Science Education and Research Bhopal</span>, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ankit%2BGupta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gupta%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnkit%2BGupta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rituja Saxena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Metagenomics and Systems Biology Group, Department of Biological Sciences, Indian Institute of Science Education and Research Bhopal</span>, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rituja%2BSaxena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saxena%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARituja%2BSaxena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">P. K. Vishnu Prasoodanan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Metagenomics and Systems Biology Group, Department of Biological Sciences, Indian Institute of Science Education and Research Bhopal</span>, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=P.%2BK.%2BVishnu%2BPrasoodanan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vishnu%20Prasoodanan%20PK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AP.%2BK.%2BVishnu%2BPrasoodanan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ashok K. Sharma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Metagenomics and Systems Biology Group, Department of Biological Sciences, Indian Institute of Science Education and Research Bhopal</span>, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ashok%2BK.%2BSharma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sharma%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAshok%2BK.%2BSharma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Parul Mittal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Metagenomics and Systems Biology Group, Department of Biological Sciences, Indian Institute of Science Education and Research Bhopal</span>, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Parul%2BMittal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mittal%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AParul%2BMittal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Ankita Roy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Metagenomics and Systems Biology Group, Department of Biological Sciences, Indian Institute of Science Education and Research Bhopal</span>, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ankita%2BRoy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roy%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnkita%2BRoy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Aaron B.A. Shafer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Forensic Science and Environmental &#38; Life Sciences, Trent University</span>, <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BB.A.%2BShafer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shafer%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BB.A.%2BShafer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Nagarjun Vijay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Computational Evolutionary Genomics Lab, Department of Biological Sciences, Indian Institute of Science Education and Research Bhopal</span>, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nagarjun%2BVijay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vijay%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANagarjun%2BVijay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Vineet K. Sharma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Metagenomics and Systems Biology Group, Department of Biological Sciences, Indian Institute of Science Education and Research Bhopal</span>, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vineet%2BK.%2BSharma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sharma%20VK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVineet%2BK.%2BSharma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vineetks@iiserb.ac.in" class="" data-icon-position="" data-hide-link-title="0">vineetks@iiserb.ac.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/315457v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:96725" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:96725/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/315457v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:96725" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:96725/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/315457v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:96725" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:96725/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/315457v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:96725" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:96725/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/315457v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:96725" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:96725/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/315457v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:96725" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:96725/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-4">The unique ornamental features and extreme sexual traits of Peacock have always intrigued the scientists. However, the genomic evidence to explain its phenotype are yet unknown. Thus, we report the first genome sequence and comparative analysis of peacock with the available high-quality genomes of chicken, turkey, duck, flycatcher and zebra finch. The candidate genes involved in early developmental pathways including TGF-β, BMP, and Wnt signaling pathway, which are also involved in feather patterning, bone morphogenesis, and skeletal muscle development, showed signs of adaptive evolution and provided useful clues on the phenotype of peacock. The innate and adaptive immune components such as complement system and T-cell response also showed signs of adaptive evolution in peacock suggesting their possible role in building a robust immune system which is consistent with the between species predictions of Hamilton-Zuk hypothesis. This study provides novel genomic and evolutionary insights into the molecular understanding towards the phenotypic evolution of Indian peacock.</p></div><div class="section" id="sec-1"><h2 class="">INTRODUCTION</h2><p id="p-5">One of the most glittering bird, the Indian peafowl (<em>Pavo cristatus</em>), is an avian species that had once puzzled the greatest naturalist, Charles Darwin, who wrote - “the sight of a feather in a Peacock’s tail, whenever I gaze at it, makes me sick” (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Huxley, 1968</a>). The presence of an exceptional ornamental plumage with large tail-coverts in peacock, which makes it more visible to predators attack, posed a question for his theory of natural selection. However, later studies showed its significance for the reproductive success of peacock mediated by sexual selection. The <em>Pavo</em> genus from the family Phasianidae has two known species, <em>Pavo cristatus</em> (Blue peafowl) and <em>Pavo muticus</em> (Green peafowl), which diverged about 3 million years ago (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">OUYANG et al., 2009</a>). The Blue peafowl (Indian Peacock) is endemic to the Indian subcontinent, whereas, the Green Peafowl is mostly found in Southeast Asia.</p><p id="p-6">Peacock (male peafowl) is one of the largest known bird among pheasants and flying birds. It shows sexual dimorphism, polygamy with no paternal care to offspring, and an elaborate male display for mating success (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Zahavi, 1975</a>;<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Ramesh and McGowan, 2009</a>). The sexual selection is extreme in peacock, which is dependent upon the ornamental display (glittering train and crest plumage) and behavioral traits (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Loyau et al., 2005a</a>). These ornamental features are also used as an honest signal about their immunocompetence to the peahen, which helps in the selection of individuals with better immunity (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Loyau et al., 2005b</a>). Though, the male masculine traits are testosterone-dependent in peacock, the large train is the default state since the peahen also shows the development of this train after ovirectomy(<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Owens and Short, 1995</a>).</p><p id="p-7">The existence of intricate ornaments in peacock has perplexed the scientists for decades and has led to several ecological and population-based studies (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Zahavi, 1975</a>;<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Loyau et al., 2005a</a>;<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Ramesh and McGowan, 2009</a>). However, the genomic details about the phenotypic evolution of this species are still unknown. Therefore, we carried out the comprehensive comparative genomics of <em>Pavo cristatus</em> (Blue Peafowl) to decipher the genomic evolution of this species. The ornamental and sexual characteristics of peacock are distinct from other birds and are absent in the available closely related species such as chicken and turkey, which makes it intriguing to look for the genomic changes underlying the phenotypic divergence of peacock. Therefore, we also carried out a comprehensive comparative genome-wide analysis of peacock genome (order Galliformes) with the high quality genomes of five other birds under the class Aves: chicken and turkey (order Galliformes), duck (order Anseriformes), and flycatcher and zebra finch (order Passeriformes). The comparative genome-wide analysis of peacock with five other related birds provided novel genomic insights into the intriguing peacock genome evolution.</p></div><div class="section" id="sec-2"><h2 class="">MATERIALS AND METHODS</h2><div id="sec-3" class="subsection"><h3>Sample collection, DNA isolation, and sequencing of peacock genome</h3><p id="p-8">Approximately 2 ml blood was drawn from the medial metatarsal vein of a two years old male Indian peacock at Van Vihar National Park, Bhopal, India and was collected in EDTA-coated vials. The fresh blood sample was immediately brought to the laboratory at 4 °C and genomic DNA was extracted using DNeasy Blood and Tissue Kit (Qiagen, USA) following the manufacturer’s protocol. Sex of the bird was determined to be male by morphological identification and was confirmed using molecular sexing assay (Supplementary Note). Multiple shotgun genomic libraries were prepared using Illumina TruSeq DNA PCR-free library preparation kit and Nextera XT sample preparation kit (Illumina Inc., USA), as per the manufacturer’s protocol. The insert size for the TruSeq libraries was selected to be 550 bp and the average insert size for Nextera XT libraries was ~650 bp. The sequencing library size for both the libraries was assessed on 2100 Bioanalyzer using High Sensitivity DNA kit (Agilent, USA). The libraries were quantified using KAPA SYBR FAST qPCR Master mix with Illumina standards and primer premix (KAPA Biosystems, USA), and Qubit dsDNA HS kit on a Qubit 2.0 fluorometer (Life Technologies, USA) as per the Illumina suggested protocol. The normalised libraries were loaded on Illumina NextSeq 500 platform using NextSeq 500/550 v2 sequencing reagent kit (Illumina Inc., USA) and 150 bp paired-end sequencing was performed for all the libraries on May 11, 2016.</p><div id="sec-4" class="subsection"><h4>Sequence alignment and phylogenetic tree construction</h4><p id="p-9">All sequence alignments (DNA and Protein) used for the phylogenetic tree reconstruction and other sequence divergence analysis were generated using MUSCLE release 3.8.31 (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Edgar, 2004</a>). The likelihood-based tree-searching algorithm was used for phylogenetic tree reconstruction using PhyML version 3.1 (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Guindon et al., 2010</a>). For nucleotide sequences GTR model was used, whereas for protein sequences JTT model was utilized. The bootstrapping value of n=1000 was used to test the robustness of the constructed phylogenetic trees of mitochondrial genome and concatenated nuclear-genes.</p></div><div id="sec-5" class="subsection"><h4>Gene gain/loss analysis</h4><p id="p-10">To estimate the gene gain and loss in gene families, CAFE (v3.1)(<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Han et al., 2013</a>) with a random birth and death model was used (Supplementary Figure 7). The species tree was constructed using NCBI taxonomy, and the branch lengths were calculated using the fossil data in TimeTree as described in Ensembl Compara pipeline(<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Vilella et al., 2009</a>). Simulated data based on the properties of observed data was generated using the command gene family and the significance of two-lambda model (separate lambda values for Galloanserae) was assessed against a global lambda model. The two-parameter model was found to fit the data better because the observed LR [LR = 2*(score of global lambda model – score of multi-lambda model)] was greater than 95% of the distribution of simulated LRs. Therefore, the two-lambda model was used for the following CAFE analysis.</p></div><div id="sec-6" class="subsection"><h4>Identification of CDS with multiple signs of adaptive evolution</h4><p id="p-11">All validated peacock coding gene sequences (&gt;90% valid bases) were analyzed through multiple sequence-based analysis such as dN/dS or ω (ratio of the rate of non-synonymous to the rate of synonymous substitutions) enrichment, positive selection, and unique substitution to assess the adaptive sequence divergence. The functional analysis was performed using KEGG (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kanehisa and Goto, 2000</a>), eggNOGs (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Huerta-Cepas et al., 2016</a>) and NCBI NR (O’Leary et al., 2016) databases. Furthermore, the functional impact of the identified unique substitutions and other sequence variations were evaluated using functional domain analysis and SIFT (Sorting Intolerant from tolerant) analysis (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kumar et al., 2009</a>). SIFT is a homology-based method, where the specific-amino acids of protein sequence conserved across species are considered to be functionally crucial.</p><div id="sec-7" class="subsection"><h5>dN/dS enrichment analysis</h5><p id="p-12">Based on the dN/dS or ω values, the positively selected (ω &gt;1), negatively selected (ω &lt;1) and neutrally selected (ω =1) genes were identified. The dN/dS values for the peacock CDS were calculated using CODEML program of the PAML package 4.9(<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Yang, 2007</a>). The pairwise dN/dS analysis was performed on the orthologous genes for six different pairs: peacock-chicken, peacock-turkey, peacock-duck, peacock-zebra finch, peacock-flycatcher and chicken-turkey using default parameters. To check for the convergence of calculated values, the iterations were performed with three different initial or fixed ω values, i.e. 0.5, 1 and 1.5, and only the coding gene sequences with consensus values were considered. To reduce the false positives and aberrant dN/dS values from analysis all the genes with the dN/dS values above five were not used for the function interpretation of results and for drawing conclusions out of it, although, they were used at the eggNOG and KEGG functional classification stage to reduce the bias.</p></div><div id="sec-8" class="subsection"><h5>Positive selection analysis</h5><p id="p-13">The multiple sequence alignment for each peacock coding gene sequence and the corresponding orthologs identified using reciprocal blast approach in the other five bird genomes were carried out using EMBOSS tranalign program (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Rice et al., 2000</a>). Furthermore, the Maximum Likelihood-based (ML) phylogenetic tree was constructed using the amino acid sequence of these orthologs. Based on the alignment and the phylogenetic tree, the calculations of likelihood scores with revised branch-site model A was performed to identify the signatures of positive selection in peacock for the considered coding gene sequence. This model tries to detect positive selection acting on specific sites on the particular specified branch (foreground branches) (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Yang et al., 2005</a>;<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Zhang et al., 2005</a>). The foreground branch consisted of peacock, and the other branches constituted the ‘background branches’. Codons were categorized into previously assumed four classes in the model based on the foreground and background estimates of dN/dS (ω) values. The alternative hypothesis, according to which the foreground branches show positive selection with ω &gt;1, was compared with the null hypothesis, according to which all branches have the same ω =1 value. The comparison was performed using LRT (Likelihood Ratio Test) values based chi-square test. The genes with P-value &lt; 0.05 were considered to be positively selected in peacock. Additionally, the amino acid sites under positive selection were identified using the Bayesian Empirical Bayes values for the branch-site model A (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Zhang et al., 2005</a>). This positive selection analysis was performed using CODEML program of the PAML package version 4.9 (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Yang, 2007</a>).</p></div><div id="sec-9" class="subsection"><h5>Unique substitution analysis</h5><p id="p-14">The peacock coding gene sequence and its orthologs identified from the five bird genomes were translated using EMBOSS transeq and the protein sequence alignments were performed using MUSCLE release 3.8.31 (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Edgar, 2004</a>). Using custom-made Perl scripts, the positions at which the peacock protein showed amino acid substitutions in comparison to all the other five bird genomes were identified and reported as the unique substitutions in peacock genome.</p></div></div><div id="sec-10" class="subsection"><h4>Estimation of effective population size (N<sub>e</sub>) history</h4><p id="p-15">The demographic history of the peacock was reconstructed by estimating the effective population size (N<sub>e</sub>) over time using pairwise sequentially Markovian coalescent (PSMC) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Li and Durbin, 2011</a>). The autosomal data of the peacock diploid genome sequence was filtered by excluding sites at which the inferred consensus quality was below 20, and the read depth was either one-third or more than twice of the average read depth across the genome. Since, mean coverage and percentage of missing data, both are important filtering thresholds in PSMC analysis, the minimum length of the contigs selected for carrying out the analysis was 5000 bp based on no more than 25% of the missing data as suggested by Krystyna et al. (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Nadachowska-Brzyska et al., 2016</a>) (<strong>Supplementary Figure 5</strong>). The resultant filtered genome sequence used for the analysis was 76% of the total genome. The parameters for PSMC were set to “N30 -t5 -r5 -p4+30*24+610”, which were used previously for 38 bird species (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Nadachowska-Brzyska et al., 2015</a>). Generation time and mutation rate are necessary to scale the results of PSMC analysis to real time. Hence, a generation time of 4 years was used in this analysis and was calculated as twice of the sexual maturity (2 years) [26]. The *mutation rate of 1.33e-09 was used as calculated in a previous study (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wright et al., 2015</a>). It is known that the estimates of Ne from PSMC can be influenced by the quality of the genome and sequencing coverage. To ensure that our results are not strongly influenced by such artefacts, 100 bootstrap runs were performed to estimate the N<sub>e</sub> from different parts of the genome to ascertain variability in the estimates of N<sub>e</sub>.</p></div><div id="sec-11" class="subsection"><h4>Accession codes</h4><p id="p-16">Sequence data for <em>Pavo cristatus</em> has been deposited in Short Read Archive under project number SRP083005 (BioProject accession: PRJNA040135, Biosample accession: SAMN05660020) and accession codes : SRR4068853 and SRR4068854.</p></div></div></div><div class="section" id="sec-12"><h2 class="">RESULTS</h2><p id="p-17">Although more than fifty bird genomes have been sequenced so far, yet the comprehensive and curated gene set is available only for the handful of bird genomes at the Ensembl browser. Thus, the comparative genomics analysis was performed using only the high quality genome assemblies of species relatively closer to pheasants which were available at the Ensembl browser.</p><p id="p-18">The whole genome sequencing of peacock genome yielded 153.7 Gb of sequence data (~136x genomic coverage; <strong>Supplementary Table 1 and Supplementary Figure 1 and 2</strong>). High-quality sequence reads were used to generate a draft genome assembly of an estimated genome size of 1.13 Gb using Abyss, Gapcloser, and Agouti (<strong>Supplementary Table 2</strong>). The de novo genome scaffold and contig N50s were 25.6 Kb and 19.3 Kb, respectively (<strong>Supplementary Table 2</strong>). BUSCO scores assessed the genome assembly to be 77.6% complete (S:63.44%, D:14.2%) and predicted 13.5% as partial, and 8.9% as missing BUSCOs (<strong>Supplementary Table 3</strong>). Using ab initio-based approach, 25,963 coding sequences were identified in peacock, and in addition, 213 tRNAs, 236 snoRNAs, and 540 miRNAs were also identified (<strong>Supplementary Table 4</strong>). The peacock genome was found to have less repetitive DNA (8.62%) as compared to chicken (9.45%) (<strong>Supplementary Table 5</strong>). PSMC analysis suggested that the peacock suffered at least two bottlenecks (around four Million and 450,000 years ago), which resulted in a severe reduction in its effective population size (<strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a></strong>). It was also interesting to note that the results of PSMC analysis of peacock were similar to the demographic history of the tropical bowerbird and turkey vulture that show long-term decrease in the effective population size (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Nadachowska-Brzyska et al., 2015</a>), perhaps because all three birds are native to the tropical rain forests.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effective population size (Ne) estimated from PSMC analysis for Peacock. The changes in effective population size (Ne) for the peacock is shown as the blue line plot. The thick line represents the consensus, and the thin light line corresponds to 100 bootstrapping rounds. Atmospheric and deep ocean temperatures from (Bintanja and Van de Wal, 2008) have been overlaid." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-451948978" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Effective population size (N&lt;sub&gt;e&lt;/sub&gt;) estimated from PSMC analysis for Peacock.&lt;/span&gt; The changes in effective population size (N&lt;sub&gt;e&lt;/sub&gt;) for the peacock is shown as the blue line plot. The thick line represents the consensus, and the thin light line corresponds to 100 bootstrapping rounds. Atmospheric and deep ocean temperatures from (Bintanja and Van de Wal, 2008) have been overlaid.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F1.medium.gif" width="440" height="185"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F1.medium.gif" width="440" height="185"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <span class="caption-title">Effective population size (N<sub>e</sub>) estimated from PSMC analysis for Peacock.</span><p id="p-19" class="first-child">The changes in effective population size (N<sub>e</sub>) for the peacock is shown as the blue line plot. The thick line represents the consensus, and the thin light line corresponds to 100 bootstrapping rounds. Atmospheric and deep ocean temperatures from (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bintanja and Van de Wal, 2008</a>) have been overlaid.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Using a combination of homology and <em>ab initio</em> based approaches, 15,970 protein-coding genes were identified in peacock by utilizing the peacock genome assembly and the filtered high quality reads from previous study (<strong>Supplementary Methods</strong>). The comparison of single nucleotide variants (SNVs) between chicken and peacock revealed 2,051,161 heterozygous SNVs at a rate of 2.05 SNV per Kb. The observed SNV rate in peacock was closer to turkey in comparison to the other avian species (<strong>Supplementary Note and Supplementary Table 6</strong>).</p><p id="p-21">The analysis of gene gain/loss in gene families was also performed for the six bird genomes namely peacock, chicken, turkey, duck, flycatcher and zebra finch. The Venn diagram of the genes families for these bird genomes is shown in <strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a></strong>. Additionally, the phylogenetic tree showing the gene gain/loss for the six bird genomes and the outlier green anole is displayed in <strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a></strong>. It is apparent that the common ancestor to the birds in the phylogenetic tree show a loss of 2,295 genes, which is also supported by a previous report mentioning the loss of around 2000 genes in the ancestor as compared to other vertebrate lineages (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Huang et al., 2013</a>;<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lovell et al., 2014</a>). However, such observations could be an artefact of poor genome coverage in the GC-rich regions and incomplete genome assemblies (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bornelöv et al., 2017</a>). This can also lead to an over or under-estimation of gene counts due to fragmentation of genes on multiple contigs and gaps in the assembly (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Denton et al., 2014</a>). We observed that contraction has been more prominent in comparison to expansion for the common ancestor of Galliformes and Anseriformes and the same pattern has also been observed for turkey and duck (<strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a></strong>). These observations corroborates with the previous study (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Huang et al., 2013</a>). However, an opposite pattern of expansion in gene families was observed for peacock and chicken (<strong><a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a></strong>). The top 20 protein families featuring gain and loss in the peacock genome are listed in <strong>Supplementary Table 7, 8 and 9</strong>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="[A] Venn diagram of gene families identified using TreeFam. A total of 9,545 gene families were common among the five bird genomes. 522 gene families were unique to the genus (Pavo, Gallus and Meleagris) of Galliformes order, whereas, 637 gene families were unique to the genus (Ficedula and Taeniopygia) of Passeriformes order. [B] Gene gain/loss in the six avian species and anole The number of gene gain (+) and loss (&#x2212;) are mentioned on the right of the taxa (branches), for the six avian species and an outlier green anole. The gene gain and loss were calculated using CAFE two-lambda model with &#x3BB; = 0.0055 for Galliformes and &#x3BB;= 0.0014 for the rest of the tree." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-451948978" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;[A] Venn diagram of gene families identified using TreeFam.&lt;/strong&gt; A total of 9,545 gene families were common among the five bird genomes. 522 gene families were unique to the genus (Pavo, Gallus and Meleagris) of Galliformes order, whereas, 637 gene families were unique to the genus (Ficedula and Taeniopygia) of Passeriformes order. &lt;strong&gt;[B] Gene gain/loss in the six avian species and anole&lt;/strong&gt; The number of gene gain (+) and loss (&#x2212;) are mentioned on the right of the taxa (branches), for the six avian species and an outlier green anole. The gene gain and loss were calculated using CAFE two-lambda model with &#x3BB; = 0.0055 for Galliformes and &#x3BB;= 0.0014 for the rest of the tree.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F2.medium.gif" width="440" height="185"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F2.medium.gif" width="440" height="185"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-22" class="first-child"><strong>[A] Venn diagram of gene families identified using TreeFam.</strong></p><p id="p-23">A total of 9,545 gene families were common among the five bird genomes. 522 gene families were unique to the genus (Pavo, Gallus and Meleagris) of Galliformes order, whereas, 637 gene families were unique to the genus (Ficedula and Taeniopygia) of Passeriformes order.</p><p id="p-24"><strong>[B] Gene gain/loss in the six avian species and anole</strong></p><p id="p-25">The number of gene gain (+) and loss (−) are mentioned on the right of the taxa (branches), for the six avian species and an outlier green anole. The gene gain and loss were calculated using CAFE two-lambda model with λ = 0.0055 for Galliformes and λ= 0.0014 for the rest of the tree.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The phylogenetic position of peacock was determined using a maximum likelihood-based analysis performed using the coding sequences of 5,907 orthologous genes identified from the six bird genomes : peacock, chicken, turkey, duck, flycatcher and zebra finch genomes (<strong>Supplementary Note</strong>). From the phylogenetic tree, it was apparent that peacock is closer to chicken than turkey in the Galliformes order, and formed a monophyletic group with duck from Anseriformes order (<strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>).</strong> The genome-wide analysis confirms the earlier studies carried out using limited coding and non-coding sequences, and chromosomal banding patterns (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Stock and Bunch, 1982</a>;<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kaiser et al., 2007</a>;<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wang et al., 2013</a>). The branch-specific ω or dN/dS (ratio of the rate of non-synonymous to synonymous substitutions) values were lower for chicken and peacock in comparison to the other bird genomes (<strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a></strong>). The mitochondrial genome, which evolves independent of the nuclear genome, was also used to infer the phylogenetic relationships using the complete mitochondrial genome sequences of peacock and 22 species from five different classes of Chordates, which included Aves, Mammalia, Reptilia, Actinopterygii and Amphibia (<strong>Supplementary Figure 3</strong>). The phylogenetic positions of the six bird species were found similar in both the trees (<strong><a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>, Supplementary Figure 4</strong>). Furthermore, the distribution of ω values and log-transformed mean ω values for the 5,907 orthologous genes showed the evolutionary closeness of peacock and chicken in comparison to peacock and turkey and supported the observations made from the phylogenetic trees (<strong><a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a></strong>). The phylogenetic analysis carried out using nuclear-genes and mitochondrial genomes revealed that peacock is closer to chicken as compared to turkey, which confirms the phylogenetic position of peacock through a genome-wide analysis, in addition to the earlier reports from limited molecular data (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Stock and Bunch, 1982</a>;<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kimball et al., 1999</a>;<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kan et al., 2010</a>;<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Wang et al., 2013</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="[A] Phylogenetic relationship of peacock with other bird genomes The phylogenetic tree constructed from the concatenated alignments of the orthologous genes across all six species. The divergence time of different bird species was determined using the TIMETREE database (Hedges et al., 2006), which is based on the published reports of molecular and fossil data. The origin of turkey was estimated to be 37.2 mya, whereas the origin of peacock and chicken was estimated to be 32.9 mya. [B] Comparison of the distribution of &#x3C9; or dN/dS values for the pairs of birds in Galliformes order: peacock-chicken (PG) and peacock-turkey (PT). The calculation was performed using 9,078 orthologous genes by employing CODEML program of PAML package v4.9a. The actual values were log-transformed to the base of 2 and mean values for the PG and PT pairs were -4.4 and -3.8, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-451948978" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;[A] Phylogenetic relationship of peacock with other bird genomes&lt;/strong&gt; The phylogenetic tree constructed from the concatenated alignments of the orthologous genes across all six species. The divergence time of different bird species was determined using the TIMETREE database (Hedges et al., 2006), which is based on the published reports of molecular and fossil data. The origin of turkey was estimated to be 37.2 mya, whereas the origin of peacock and chicken was estimated to be 32.9 mya. &lt;strong&gt;[B] Comparison of the distribution of &#x3C9; or dN/dS values for the pairs of birds in Galliformes order: peacock-chicken (PG) and peacock-turkey (PT).&lt;/strong&gt; The calculation was performed using 9,078 orthologous genes by employing CODEML program of PAML package v4.9a. The actual values were log-transformed to the base of 2 and mean values for the PG and PT pairs were -4.4 and -3.8, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F3.medium.gif" width="440" height="208"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F3.medium.gif" width="440" height="208"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-27" class="first-child"><strong>[A] Phylogenetic relationship of peacock with other bird genomes</strong></p><p id="p-28">The phylogenetic tree constructed from the concatenated alignments of the orthologous genes across all six species. The divergence time of different bird species was determined using the TIMETREE database (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hedges et al., 2006</a>), which is based on the published reports of molecular and fossil data. The origin of turkey was estimated to be 37.2 mya, whereas the origin of peacock and chicken was estimated to be 32.9 mya.</p><p id="p-29"><strong>[B] Comparison of the distribution of ω or dN/dS values for the pairs of birds in Galliformes order: peacock-chicken (PG) and peacock-turkey (PT).</strong></p><p id="p-30">The calculation was performed using 9,078 orthologous genes by employing CODEML program of PAML package v4.9a. The actual values were log-transformed to the base of 2 and mean values for the PG and PT pairs were -4.4 and -3.8, respectively.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-13" class="subsection"><h3>Divergence and adaptive evolution</h3><p id="p-31">A comparative genomic analysis was performed using 15,970 peacock genes and their corresponding orthologs present in chicken, turkey, duck, flycatcher and zebra finch. The dN/dS values &gt;1 was shown by 74 genes, of which 25 genes had values above five indicating possible false positives, and were not considered for the functional interpretation (<strong>Supplementary Table 10</strong>). A total of 491 genes displayed the signs of positive selection identified using branch-site model A and the statistical significance was evaluated using likelihood ratio tests with p-value threshold of 0.05 (<strong>Supplementary Table 11</strong>). Unique amino-acid substitutions in peacock were found for 3,238 genes, of which the substitutions in 116 genes were predicted to affect the protein function using SIFT (Sorting Intolerant from Tolerant) analysis (<strong>Supplementary Table 12 and 13</strong>). A total of 417 genes contained amino acid sites, which were under significant positive selection based on the Bayesian empirical Bayes values. In total 99 genes showed positive selection and unique amino acid substitutions that may affect the protein function predicted using SIFT and are referred to as genes with ‘multiple signs of adaptation’ (MSA) in this study (<strong>Supplementary Table 14</strong>).</p><p id="p-32">The functional analysis revealed the role of these genes in key cellular processes such as cell proliferation and differentiation (MAPK, RAS, PI3K-Akt, ErbB, Hippo, Rap1, and Jak-STAT signaling, Wnt signaling, calcium signaling and adrenergic signaling in cardiomyocytes) and immune response (T cell receptor, Toll-like receptor signaling, NOD-like receptor signaling, complement and coagulation cascade and chemokine-chemokine signaling). In addition, multiple genes involved in early development pathways such as TGF-β, Wnt/β-catenin, FGF, and BMP signaling also showed adaptive sequence divergence in peacock. These cellular processes and pathways regulate key features such as early development, feather development, bone morphogenesis, skeletal muscle development, metabolism, and immune response (<strong>Supplementary Table 15</strong>).</p><p id="p-33">An interesting observation was made from the signalling pathways such as Wnt, Rap1, Ras, Jak-Stat, and cAMP-mediated GPCR signalling. It was observed that the ligand and/or receptor, and in some cases the final effector genes showed adaptive evolution, whereas the genes involved in the intermediate signal transduction processes remained conserved perhaps due to their common role in multiple signaling pathways (<strong>Supplementary Note</strong>). Another interesting observation was that in several interacting protein pairs, both the interacting proteins showed sequence divergence hinting towards their co-evolution (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Moyle et al., 1994</a>). These protein pairs were majorly involved in early development pathways such as Wnt, BMP and TGF-β signalling, cell cycle regulation, DNA replication, GPCR signaling, and gene expression regulation (<strong>Supplementary Table 16</strong>).</p></div><div id="sec-14" class="subsection"><h3>Adaptive evolution of early developmental pathways</h3><p id="p-34">The early developmental pathways, which are crucial in guiding the embryonic development in birds such as TGF-β, Wnt, FGF and BMP signaling, showed adaptive divergence in peacock (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Klaus and Birchmeier, 2008</a>). Among these pathways, the TGF-β pathway is known to regulate the cartilage connective tissue development (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Loveridge et al., 1993</a>), and also functions as an activator of feather development in birds. In this pathway, TGFBR3 gene showed MSA, and TGF-β3 preproprotein, TGFBRAP1, and TAB3 genes showed multiple unique substitutions (<strong>Supplementary Note</strong>). The Wnt signaling pathway is involved in development, regeneration, aging process (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brack et al., 2007</a>;<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Klaus and Birchmeier, 2008</a>), and also regulates the initial placement of feather buds and their consolidation within the feather field (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lim and Nusse, 2013</a>). Multiple regulators of Wnt signaling such as WNT2, WIF1, and DKK2 genes had positively selected amino acid sites and showed signs of adaptive evolution. The WIF1 and DKK2 genes also harbored multiple unique substitutions. Furthermore, the DKK2 and WNT2 genes were found to be positively selected in peacock. APCDD1 gene, which is an inhibitor of Wnt signaling pathway, showed MSA. The Bone Morphogenetic Protein (BMP) signaling is involved in the development of skeletal muscles, bone and cartilage connective tissue (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Nie et al., 2006</a>;<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Nishimura et al., 2012</a>), neurogenesis (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Groppe et al., 2002</a>), and feather formation and patterning. Multiple genes such as BRK-3, BMP5, BMP3, BMP10 and CRIM1, which are involved in the regulation of BMP pathways and the corresponding early development, showed unique substitutions that may affect their function in cellular pathways as compared to the other birds.</p><p id="p-35">In addition, the Notch-2 receptor gene of Notch-Delta signaling, which is involved in growth and patterning of feather buds, early development of sensory organs (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crowe et al., 1998</a>), and terminal muscle differentiation also showed five unique substitutions. Unique substitutions were also found in the FGFR3 receptor gene and FGF23 genes, which are part of the FGF signaling involved in limb and skeletal muscle development, feather development and morphogenesis, and regulation of feather density and patterning (Pownall and Isaacs, 2010).</p><p id="p-36">Taken together, the multiple signs of evolution observed in the genes of early development pathways in peacock suggest the adaptive divergence of the early development processes, including feather, bone and skeletomuscle development.</p><div id="sec-15" class="subsection"><h4>Peacock feathers: Clues from early development genes</h4><p id="p-37">Among the distinctive features of a peacock, the large and decorative feathers attract the most attention; particularly the long train, which is useful for their courtship behavior. The feather development in birds is primarily guided by the continuous reciprocal interactions between the epithelium and mesenchyme (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chuong et al., 2000</a>). The analysis of the curated set of 2,146 feather-related genes (<strong>Supplementary Note</strong>) involved in feather development revealed that the activators of feather development including FGF, Wnt/β-catenin and TGF-β and, the inhibitors such as BMP and Notch-delta showed sequence divergence in peacock in comparison to the other bird genomes. The observed divergence in genes related to feather development provides useful genomic clues for the peculiar patterning and structure of peacock feathers.</p></div></div><div id="sec-16" class="subsection"><h3>Adaptive Evolution in Immune-related Genes</h3><p id="p-38">In birds, the rate of sequence divergence in immune-related genes is usually higher than the other genes primarily due to the co-evolution of host-pathogen interactions (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ekblom et al., 2010</a>). Several genes involved in the development of immune system and modulation of immune response have shown sequence divergence and signs of adaptive evolution in the peacock genome.</p><p id="p-39">Multiple components of the innate immune system such as complement system and pathogen recognition system showed adaptive evolution. The C5 protein involved in the recruitment of cellular component of the immune system at the site of infection showed five unique substitutions. The α-subunit of C8 protein involved in forming the membrane attack complex (MAC)(<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Serna et al., 2016</a>) showed MSA. Additionally, the CSF-1R gene, which is crucial for macrophage survival, differentiation, and proliferation (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Pixley and Stanley, 2004</a>), showed positive selection with positively selected sites and unique substitutions. Different components of NF-ĸB signaling such as MYD88, TRADD, SIGIRR, MAP3K14 and TLR5, which regulate the immune response against infections (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kaisho and Akira, 2006</a>), showed signs of adaptations. The MYD88 protein, which is a part of Toll-like receptors (TLRs) mediated signaling, showed MSA and higher divergence from chicken in comparison to turkey among the species of the Galliformes order. Similarly, the genes TRADD, SIGIRR, MAP3K14, and TLR5 showed multiple unique substitutions. Furthermore, the pattern recognition receptors such as NLRC3, which regulates innate immune response by interacting with stimulators of interferon genes (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Zhang et al., 2014</a>), showed positive selection with positively selected sites and unique substitution.</p><p id="p-40">Several genes regulating the T and B-cell response of the adaptive immune system also displayed adaptive evolution in peacock. The SPI-1 gene involved in B and T cell development by regulating the expression as well as alternative splicing of target genes (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hallier et al., 1998</a>) showed MSA. The ITGAV and AQP3 genes, which are involved in T-cell movement and migration, showed unique substitutions and higher (2X) divergence from chicken as compared to turkey. Furthermore, different T-cell receptors and signaling proteins involved in T-cell activation such as SDC4, FLT4, NFATC3, and IL12B subunit showed sequence divergence and multiple signs of adaptation in peacock. CTLA4 gene, which is a negative regulator of T-cell response (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Walunas et al., 1994</a>), also showed multiple unique substitutions. A few other regulator genes of immune response also showed multiple signs of adaptation in peacock and are discussed in Supplementary Note. In addition, the gene family SSC4D involved in the development of immune system and the regulation of both innate and adaptive immunity (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Asratian and Vasil’eva, 1976</a>) showed expansion in peacock in comparison to chicken (<strong>Supplementary Table 8</strong>).</p><p id="p-41">Taken together, it appears that the adaptive evolution of immune-related genes in peacock has occurred primarily in the components of innate immunity such as complement system, pattern recognition receptors, and monocyte development, and in the components of adaptive immunity such as T-cell response. It suggests that the immune system-related genes in peacock genome have significantly evolved to provide a selective advantage in fighting against infections.</p></div><div id="sec-17" class="subsection"><h3>Body Dimensions</h3><p id="p-42">Follicle stimulating hormone receptor (FSHR), which is involved in regulating the cell growth, differentiation, and body dimensions of birds via cAMP-mediated PI3K-AKT and SRC-ERK1/2 signaling (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fayeye et al., 2006</a>), showed multiple unique substitutions. Several genes such as MMP2, BMP7, TRAF6, TNF3, Neurochondrin, IGF, and NOX4, regulating bone morphogenesis and development in birds showed divergence as well as adaptive evolution in peacock. These genes primarily function as ligands or receptors for Wnt-beta-catenin, TGF-beta, p70S6K and PEDF signaling pathways. From these observations, it appears that the adaptive evolution of intracellular signaling and early development genes, which play significant roles in bone and skeletal muscle development, are perhaps beneficial for supporting its body dimensions.</p></div><div id="sec-18" class="subsection"><h3>MSA genes involved in other cellular processes</h3><p id="p-43">Among the other genes that displayed multiple signs of adaptation, BRCA2, DNA-PKcs, FANCC, and INO80 genes were involved in the DNA double-strand break repair and recombination, FBXO15, USP53, and PSMD1-26S were part of ubiquitin-proteasomal protein degradation system, HERPUD1 and HSP90B1 genes were involved in stress response, and METTL5 gene had protein methyltransferase activity. Thus, DNA repair and protein turnover and modification were among the other cellular processes where a notable number of genes showed MSA.</p></div></div><div class="section" id="sec-19"><h2 class="">DISCUSSION</h2><p id="p-44">The most significant results emerged from the adaptive sequence divergence analysis, where a major fraction of genes involved in early development and immune system showed multiple signs of adaptive evolution (<strong><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a></strong>). Similarly, the genes involved in the early development of feathers showed signs of adaptive evolution in the feather-specific gene set. In addition, the adaptive divergence observed in the genes involved in bone morphogenesis and skeletal muscle development perhaps explain the large body dimensions, stronger legs and spurs, and the ability to take short flights despite of a long train. Taken together, the evolution in the early development genes emerges as a prominent factor for explaining the molecular basis of the phenotypic evolution for Indian peacock.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Adaptively evolved signaling pathways in peacock genome The genes highlighted in Red colour showed signs of adaptive evolution such as positive selection and unique substitution. It is apparent that the receptors, ligands and regulators of early development pathways such as Wnt, TGF-&#x3B2; and BMP, showed adaptive sequence divergence in peacock. In the case of NF-KB, cytokine and growth factor signaling pathways, the proteins involved in intermediate signal transduction also showed adaptive sequence divergence. Individual pathways are colour coded separately." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-451948978" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Adaptively evolved signaling pathways in peacock genome&lt;/span&gt; The genes highlighted in Red colour showed signs of adaptive evolution such as positive selection and unique substitution. It is apparent that the receptors, ligands and regulators of early development pathways such as Wnt, TGF-&#x3B2; and BMP, showed adaptive sequence divergence in peacock. In the case of NF-KB, cytokine and growth factor signaling pathways, the proteins involved in intermediate signal transduction also showed adaptive sequence divergence. Individual pathways are colour coded separately.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F4.medium.gif" width="440" height="236"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F4.medium.gif" width="440" height="236"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/05/05/315457/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <span class="caption-title">Adaptively evolved signaling pathways in peacock genome</span><p id="p-45" class="first-child">The genes highlighted in Red colour showed signs of adaptive evolution such as positive selection and unique substitution. It is apparent that the receptors, ligands and regulators of early development pathways such as Wnt, TGF-β and BMP, showed adaptive sequence divergence in peacock. In the case of NF-KB, cytokine and growth factor signaling pathways, the proteins involved in intermediate signal transduction also showed adaptive sequence divergence. Individual pathways are colour coded separately.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">Though birds are the natural host of viruses and are also prone to avian viral infections (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alexander, 2000</a>;<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Berg, 2000</a>;<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Liu et al., 2005</a>), peacocks have a longer average life span, and are also found to be resistant to the new viral strain pathogenic to chicken and turkey (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Sun et al., 2007</a>), pointing towards the presence of a robust immune system. The strong immunity against pathogens and infections could be attributed to the adaptive divergence observed in the components of the innate immune system (complement and pathogen recognition system), adaptive immune response (B and T cell development), and other genes responsible for the overall immune system development. The adaptive evolution observed for immune genes in peacock appears to be indicative of a higher parasite load consistent with Hamilton-Zuk hypothesis (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Balenger and Zuk, 2014</a>). Though the results were obtained from the comparative genomic analysis of peacock, some of the insights may also be applicable to the other related species in the pheasant group. The comparative genomic analysis presented in this work provides novel insights on the phenotypic evolution of Indian Peacock and the genomic clues from this study will serve as leads for further studies to decipher the genotype-phenotype interactions for peacock. In addition, this study will also help in devising better strategies for the management and conservation of peacock population, which is showing a decline mainly because of habitat deterioration, poaching for train-feathers, use of pesticides and chemical fertilizers.</p></div><div class="section COI-statement" id="sec-20"><h2 class="">Competing financial interests</h2><p id="p-48">The authors declare no competing financial interests.</p></div><div class="section" id="sec-21"><h2 class="">Contributions</h2><p id="p-49">VKS conceived and coordinated the project. RS prepared the DNA samples, performed sequencing and the molecular sexing assay. AG performed the <em>de novo</em> and reference-based genome assembly. PM, AKS, AG and SKJ performed the genome annotations. SKJ and PM performed the phylogenetic tree analyses. SKJ performed the dN/dS, positive selection, and statistical analysis. SKJ, AG and AR performed the unique substitution and SIFT analyses. PM performed the gene gain/loss analysis. SKJ, VPPK and AG created figures. SKJ, AG, VKS, NV, and AS analysed the data and wrote the manuscript. All the authors have read and approved the final manuscript.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-47">We thank Dr. Atul Gupta, Wildlife Veterinary Officer, Van Vihar National Park, Bhopal and Director, Van Vihar National Park, Bhopal, India for providing the blood samples of peacock. We also acknowledge the help of Dr. Tista Joseph and Dr. Niraj Dahe, Wildlife Veterinary Officers (Wildlife SOS India) at Van Vihar National Park for carrying out the sample collection procedure. We thank the HPC facility and NGS facility at IISER Bhopal. The authors SKJ, AG and RS thank the Department of Science and Technology for the DST-INSPIRE fellowship. We also thank the intramural research funds provided by IISER Bhopal.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-2">Email addresses of authors: Shubham K. Jaiswal - <span class="em-link"><span class="em-addr">shubhamj{at}iiserb.ac.in</span></span>, Ankit Gupta - <span class="em-link"><span class="em-addr">ankitg{at}iiserb.ac.in</span></span>, Aaron B.A. Shafer - <span class="em-link"><span class="em-addr">aaronshafer{at}trentu.ca</span></span>, Rituja Saxena - <span class="em-link"><span class="em-addr">ritus{at}iiserb.ac.in</span></span>, Vishnu Prasoodanan P. K.- <span class="em-link"><span class="em-addr">vishnup16{at}iiserb.ac.in</span></span>, Ashok K. Sharma - <span class="em-link"><span class="em-addr">ashok{at}iiserb.ac.in</span></span>, Parul Mittal - <span class="em-link"><span class="em-addr">parulm{at}iiserb.ac.in</span></span>, Ankita Roy - <span class="em-link"><span class="em-addr">rankitauga{at}gmail.com</span></span>, Nagarjun Vijay - <span class="em-link"><span class="em-addr">nagarjun{at}iiserb.ac.in</span></span></p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.1" data-doi="10.1016/S0378-1135(00)00160-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alexander</span>,  <span class="cit-name-given-names">D.J.</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">A review of avian influenza in different bird species</span>. <abbr class="cit-jnl-abbrev">Vet Microbiol</abbr> <span class="cit-vol">74</span>, <span class="cit-fpage">3</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bmicrobiology%26rft.stitle%253DVet%2BMicrobiol%26rft.aulast%253DAlexander%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D74%26rft.issue%253D1-2%26rft.spage%253D3%26rft.epage%253D13%26rft.atitle%253DA%2Breview%2Bof%2Bavian%2Binfluenza%2Bin%2Bdifferent%2Bbird%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1135%252800%252900160-7%26rft_id%253Dinfo%253Apmid%252F10799774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1135(00)00160-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10799774&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087148200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Asratian</span>,  <span class="cit-name-given-names">A.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Vasil’;eva</span>,  <span class="cit-name-given-names">V.I.</span></span> (<span class="cit-pub-date">1976</span>). <span class="cit-article-title">[Immunological structure of the population of Erevan with regard to Mycoplasma hominis]</span>. <abbr class="cit-jnl-abbrev">Zh Eksp Klin Med</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">59</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZhurnal%2Beksperimental%2527no%25C4%25AD%2Bi%2Bklinichesko%25C4%25AD%2Bmeditsiny%26rft.stitle%253DZh%2BEksp%2BKlin%2BMed%26rft.aulast%253DAsratian%26rft.auinit1%253DA.%2BA.%26rft.volume%253D16%26rft.issue%253D3%26rft.spage%253D59%26rft.epage%253D61%26rft.atitle%253D%255BImmunological%2Bstructure%2Bof%2Bthe%2Bpopulation%2Bof%2BErevan%2Bwith%2Bregard%2Bto%2BMycoplasma%2Bhominis%255D%26rft_id%253Dinfo%253Apmid%252F136853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=136853&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Balenger</span>,  <span class="cit-name-given-names">S.L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Zuk</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2014</span>). "<span class="cit-article-title">Testing the Hamilton–Zuk hypothesis: past, present, and future</span>". <abbr class="cit-jnl-abbrev">The Society for Integrative and Comparative Biology</abbr>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.4" data-doi="10.1080/03079450050045431"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berg</span>,  <span class="cit-name-given-names">T.P.</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">Acute infectious bursal disease in poultry: a review</span>. <abbr class="cit-jnl-abbrev">Avian Pathol</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">175</span>–<span class="cit-lpage">194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAvian%2Bpathology%2B%253A%2B%2Bjournal%2Bof%2Bthe%2BW.V.P.A%26rft.stitle%253DAvian%2BPathol%26rft.aulast%253DBerg%26rft.auinit1%253DT.%2BP.%26rft.volume%253D29%26rft.issue%253D3%26rft.spage%253D175%26rft.epage%253D194%26rft.atitle%253DAcute%2Binfectious%2Bbursal%2Bdisease%2Bin%2Bpoultry%253A%2Ba%2Breview.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F03079450050045431%26rft_id%253Dinfo%253Apmid%252F19184804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/03079450050045431&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19184804&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088296200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.5" data-doi="10.1038/nature07158"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bintanja</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Van De Wal</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">North American ice-sheet dynamics and the onset of 100,000-year glacial cycles</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">454</span>, <span class="cit-fpage">869</span>–<span class="cit-lpage">872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D454%26rft.issue%253D7206%26rft.spage%253D869%26rft.epage%253D872%26rft.atitle%253DNorth%2BAmerican%2Bice-sheet%2Bdynamics%2Band%2Bthe%2Bonset%2Bof%2B100%252C000-year%2Bglacial%2Bcycles%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07158%26rft_id%253Dinfo%253Apmid%252F18704083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2008123032&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=18704083&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258398600032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.6" data-doi="10.1186/s13059-017-1231-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bornelöv</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seroussi</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yosefi</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pendavis</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burgess</span>,  <span class="cit-name-given-names">S.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grabherr</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Friedman-Einat</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Andersson</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Correspondence on Lovell et al.: identification of chicken genes previously assumed to be evolutionarily lost</span>. <abbr class="cit-jnl-abbrev">Genome biology</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.volume%253D18%26rft.spage%253D112%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1231-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-017-1231-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.7" data-doi="10.1126/science.1144090"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brack</span>,  <span class="cit-name-given-names">A.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Conboy</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roy</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuo</span>,  <span class="cit-name-given-names">C.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keller</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Rando</span>,  <span class="cit-name-given-names">T.A.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Increased Wnt signaling during aging alters muscle stem cell fate and increases fibrosis</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">317</span>, <span class="cit-fpage">807</span>–<span class="cit-lpage">810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBrack%26rft.auinit1%253DA.%2BS.%26rft.volume%253D317%26rft.issue%253D5839%26rft.spage%253D807%26rft.epage%253D810%26rft.atitle%253DIncreased%2BWnt%2BSignaling%2BDuring%2BAging%2BAlters%2BMuscle%2BStem%2BCell%2BFate%2Band%2BIncreases%2BFibrosis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1144090%26rft_id%253Dinfo%253Apmid%252F17690295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTcvNTgzOS84MDciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNS8wNS8zMTU0NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.8" data-doi="10.1016/S0959-437X(00)00111-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chuong</span>,  <span class="cit-name-given-names">C.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chodankar</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Widelitz</span>,  <span class="cit-name-given-names">R.B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">T.X.</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">Evo-devo of feathers and scales: building complex epithelial appendages</span>. <abbr class="cit-jnl-abbrev">Curr Opin Genet Dev</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">449</span>–<span class="cit-lpage">456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DChuong%26rft.auinit1%253DC.%2BM.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D449%26rft.epage%253D456%26rft.atitle%253DEvo-devo%2Bof%2Bfeathers%2Band%2Bscales%253A%2Bbuilding%2Bcomplex%2Bepithelial%2Bappendages.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252800%252900111-8%26rft_id%253Dinfo%253Apmid%252F11023302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(00)00111-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11023302&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088285100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crowe</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Henrique</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ish-Horowicz</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Niswander</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">1998</span>). <span class="cit-article-title">A new role for Notch and Delta in cell fate decisions: patterning the feather array</span>. <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">125</span>, <span class="cit-fpage">767</span>–<span class="cit-lpage">775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DCrowe%26rft.auinit1%253DR%26rft.volume%253D125%26rft.issue%253D4%26rft.spage%253D767%26rft.epage%253D775%26rft.atitle%253DA%2Bnew%2Brole%2Bfor%2BNotch%2Band%2BDelta%2Bin%2Bcell%2Bfate%2Bdecisions%253A%2Bpatterning%2Bthe%2Bfeather%2Barray%26rft_id%253Dinfo%253Apmid%252F9435296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czo5OiIxMjUvNC83NjciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNS8wNS8zMTU0NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Denton</span>,  <span class="cit-name-given-names">J.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lugo-Martinez</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tucker</span>,  <span class="cit-name-given-names">A.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schrider</span>,  <span class="cit-name-given-names">D.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Warren</span>,  <span class="cit-name-given-names">W.C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hahn</span>,  <span class="cit-name-given-names">M.W.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Extensive error in the number of genes inferred from draft genome assemblies</span>. <abbr class="cit-jnl-abbrev">PLoS computational biology</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">e1003998</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bcomputational%2Bbiology%26rft.volume%253D10%26rft.spage%253De1003998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.11" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R.C.</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034147&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220487200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.12" data-doi="10.1093/gbe/evq061"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ekblom</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">French</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Slate</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Burke</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Evolutionary analysis and expression profiling of zebra finch immune genes</span>. <abbr class="cit-jnl-abbrev">Genome Biol Evol</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">781</span>–<span class="cit-lpage">790</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevq061%26rft_id%253Dinfo%253Apmid%252F20884724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evq061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20884724&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fayeye</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ayorinde</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ojo</span>,  <span class="cit-name-given-names">V.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Adesina</span>,  <span class="cit-name-given-names">O.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Frequency and influence of some major genes on body weight and body size parameters of Nigerian local chickens</span>. <abbr class="cit-jnl-abbrev">Livestock research for rural development</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivestock%2Bresearch%2Bfor%2Brural%2Bdevelopment%26rft.volume%253D18%26rft.spage%253D37%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.14" data-doi="10.1038/nature01245"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Groppe</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greenwald</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wiater</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodriguez-Leon</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Economides</span>,  <span class="cit-name-given-names">A.N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kwiatkowski</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Affolter</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vale</span>,  <span class="cit-name-given-names">W.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Izpisua Belmonte</span>,  <span class="cit-name-given-names">J.C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Choe</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">Structural basis of BMP signalling inhibition by the cystine knot protein Noggin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">420</span>, <span class="cit-fpage">636</span>–<span class="cit-lpage">642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGroppe%26rft.auinit1%253DJ.%26rft.volume%253D420%26rft.issue%253D6916%26rft.spage%253D636%26rft.epage%253D642%26rft.atitle%253DStructural%2Bbasis%2Bof%2BBMP%2Bsignalling%2Binhibition%2Bby%2Bthe%2Bcystine%2Bknot%2Bprotein%2BNoggin.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01245%26rft_id%253Dinfo%253Apmid%252F12478285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12478285&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179751800037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.15" data-doi="10.1093/sysbio/syq010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guindon</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dufayard</span>,  <span class="cit-name-given-names">J.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lefort</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anisimova</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hordijk</span>,  <span class="cit-name-given-names">W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Gascuel</span>,  <span class="cit-name-given-names">O.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0</span>. <abbr class="cit-jnl-abbrev">Syst Biol</abbr> <span class="cit-vol">59</span>, <span class="cit-fpage">307</span>–<span class="cit-lpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyq010%26rft_id%253Dinfo%253Apmid%252F20525638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syq010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20525638&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276528300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.16" data-doi="10.1074/jbc.273.9.4838"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hallier</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lerga</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barnache</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tavitian</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Moreau-Gachelin</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">1998</span>). <span class="cit-article-title">The transcription factor Spi-1/PU.1 interacts with the potential splicing factor TLS</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">273</span>, <span class="cit-fpage">4838</span>–<span class="cit-lpage">4842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DHallier%26rft.auinit1%253DM.%26rft.volume%253D273%26rft.issue%253D9%26rft.spage%253D4838%26rft.epage%253D4842%26rft.atitle%253DThe%2BTranscription%2BFactor%2BSpi-1%252FPU.1%2BInteracts%2Bwith%2Bthe%2BPotential%2BSplicing%2BFactor%2BTLS%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.273.9.4838%26rft_id%253Dinfo%253Apmid%252F9478924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzMvOS80ODM4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMDUvMzE1NDU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.17" data-doi="10.1093/molbev/mst100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">M.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomas</span>,  <span class="cit-name-given-names">G.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lugo-Martinez</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hahn</span>,  <span class="cit-name-given-names">M.W.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Estimating gene gain and loss rates in the presence of error in genome assembly and annotation using CAFE 3</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">1987</span>–<span class="cit-lpage">1997</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst100%26rft_id%253Dinfo%253Apmid%252F23709260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23709260&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321820400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.18" data-doi="10.1093/bioinformatics/btl505"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hedges</span>,  <span class="cit-name-given-names">S.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dudley</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">TimeTree: a public knowledge-base of divergence times among organisms</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">2971</span>–<span class="cit-lpage">2972</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl505%26rft_id%253Dinfo%253Apmid%252F17021158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl505&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17021158&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242246300027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.19" data-doi="10.1038/ng.2657"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burt</span>,  <span class="cit-name-given-names">D.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qian</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gan</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">The duck genome and transcriptome provide insight into an avian influenza virus reservoir species</span>. <abbr class="cit-jnl-abbrev">Nature genetics</abbr> <span class="cit-vol">45</span>, <span class="cit-fpage">776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.volume%253D45%26rft.spage%253D776%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2657%26rft_id%253Dinfo%253Apmid%252F23749191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2657&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23749191&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.20" data-doi="10.1093/nar/gkv1248"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huerta-Cepas</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Szklarczyk</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Forslund</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cook</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heller</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Walter</span>,  <span class="cit-name-given-names">M.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rattei</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mende</span>,  <span class="cit-name-given-names">D.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sunagawa</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuhn</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jensen</span>,  <span class="cit-name-given-names">L.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Von Mering</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bork</span>,  <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">eggNOG 4.5: a hierarchical orthology framework with improved functional annotations for eukaryotic, prokaryotic and viral sequences</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">44</span>, <span class="cit-fpage">D286</span>–<span class="cit-lpage">293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1248%26rft_id%253Dinfo%253Apmid%252F26582926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1248&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26582926&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-315457v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huxley</span>,  <span class="cit-name-given-names">T.H.</span></span> (<span class="cit-pub-date">1968</span>). <span class="cit-source">On the origin of species</span>. <span class="cit-publ-name">University of Michigan P</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.22" data-doi="10.1093/molbev/msl164"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kaiser</span>,  <span class="cit-name-given-names">V.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Tuinen</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ellegren</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Insertion events of CR1 retrotransposable elements elucidate the phylogenetic branching order in galliform birds</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">338</span>–<span class="cit-lpage">347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl164%26rft_id%253Dinfo%253Apmid%252F17077154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msl164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17077154&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243069700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.23" data-doi="10.1016/j.jaci.2006.02.023"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kaisho</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Akira</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Toll-like receptor function and signaling</span>. <abbr class="cit-jnl-abbrev">J Allergy Clin Immunol</abbr> <span class="cit-vol">117</span>, <span class="cit-fpage">979</span>–<span class="cit-lpage">987</span>; quiz 988.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAllergy%2Band%2BClinical%2BImmunology%26rft.stitle%253DJournal%2Bof%2BAllergy%2Band%2BClinical%2BImmunology%26rft.aulast%253DKaisho%26rft.auinit1%253DT.%26rft.volume%253D117%26rft.issue%253D5%26rft.spage%253D979%26rft.epage%253D987%26rft.atitle%253DToll-like%2Breceptor%2Bfunction%2Band%2Bsignaling.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jaci.2006.02.023%26rft_id%253Dinfo%253Apmid%252F16675322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jaci.2006.02.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16675322&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237436300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kan</span>,  <span class="cit-name-given-names">X.-Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.-F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lei</span>,  <span class="cit-name-given-names">Z.-P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.-Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.-K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">Z.-C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.-Q.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Estimation of divergence times for major lineages of galliform birds: evidence from complete mitochondrial genome sequences</span>. <abbr class="cit-jnl-abbrev">African Journal of Biotechnology</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">3073</span>–<span class="cit-lpage">3078</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAfrican%2BJournal%2Bof%2BBiotechnology%26rft.volume%253D9%26rft.spage%253D3073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.25" data-doi="10.1093/nar/28.1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kanehisa</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Goto</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">KEGG: kyoto encyclopedia of genes and genomes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">28</span>, <span class="cit-fpage">27</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKanehisa%26rft.auinit1%253DM.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D30%26rft.atitle%253DKEGG%253A%2BKyoto%2BEncyclopedia%2Bof%2BGenes%2Band%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.27%26rft_id%253Dinfo%253Apmid%252F10592173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/28.1.27&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10592173&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084896300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.26" data-doi="10.1006/mpev.1998.0562"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kimball</span>,  <span class="cit-name-given-names">R.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Braun</span>,  <span class="cit-name-given-names">E.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zwartjes</span>,  <span class="cit-name-given-names">P.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crowe</span>,  <span class="cit-name-given-names">T.M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ligon</span>,  <span class="cit-name-given-names">J.D.</span></span> (<span class="cit-pub-date">1999</span>). <span class="cit-article-title">A molecular phylogeny of the pheasants and partridges suggests that these lineages are not monophyletic</span>. <abbr class="cit-jnl-abbrev">Mol Phylogenet Evol</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">38</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DKimball%26rft.auinit1%253DR.%2BT.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D38%26rft.epage%253D54%26rft.atitle%253DA%2Bmolecular%2Bphylogeny%2Bof%2Bthe%2Bpheasants%2Band%2Bpartridges%2Bsuggests%2Bthat%2Bthese%2Blineages%2Bare%2Bnot%2Bmonophyletic.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fmpev.1998.0562%26rft_id%253Dinfo%253Apmid%252F10082609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/mpev.1998.0562&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10082609&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079387200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.27" data-doi="10.1038/nrc2389"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Klaus</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Birchmeier</span>,  <span class="cit-name-given-names">W.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Wnt signalling and its impact on development and cancer</span>. <abbr class="cit-jnl-abbrev">Nat Rev Cancer</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">387</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BCancer%26rft.stitle%253DNat%2BRev%2BCancer%26rft.aulast%253DKlaus%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253D387%26rft.epage%253D398%26rft.atitle%253DWnt%2Bsignalling%2Band%2Bits%2Bimpact%2Bon%2Bdevelopment%2Band%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrc2389%26rft_id%253Dinfo%253Apmid%252F18432252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrc2389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18432252&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255264700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.28" data-doi="10.1038/nprot.2009.86"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Henikoff</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ng</span>,  <span class="cit-name-given-names">P.C.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm</span>. <abbr class="cit-jnl-abbrev">Nat Protoc</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">1073</span>–<span class="cit-lpage">1081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DKumar%26rft.auinit1%253DP.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253D1073%26rft.epage%253D1081%26rft.atitle%253DPredicting%2Bthe%2Beffects%2Bof%2Bcoding%2Bnon-synonymous%2Bvariants%2Bon%2Bprotein%2Bfunction%2Busing%2Bthe%2BSIFT%2Balgorithm.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2009.86%26rft_id%253Dinfo%253Apmid%252F19561590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2009.86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19561590&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268858700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.29" data-doi="10.1038/nature10231"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Durbin</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Inference of human population history from individual whole-genome sequences</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">475</span>, <span class="cit-fpage">493</span>–<span class="cit-lpage">496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DH.%26rft.volume%253D475%26rft.issue%253D7357%26rft.spage%253D493%26rft.epage%253D496%26rft.atitle%253DInference%2Bof%2Bhuman%2Bpopulation%2Bhistory%2Bfrom%2Bindividual%2Bwhole-genome%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10231%26rft_id%253Dinfo%253Apmid%252F21753753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21753753&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293167900039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lim</span>,  <span class="cit-name-given-names">X.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Nusse</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Wnt signaling in skin development, homeostasis, and disease</span>. <abbr class="cit-jnl-abbrev">Cold Spring Harb Perspect Biol</abbr> <span class="cit-vol">5</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.31" data-doi="10.1126/science.1115273"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiao</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lei</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qin</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">X.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">X.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">G.F.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Highly pathogenic H5N1 influenza virus infection in migratory birds</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">309</span>, <span class="cit-fpage">1206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1115273%26rft_id%253Dinfo%253Apmid%252F16000410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDkvNTczOC8xMjA2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMDUvMzE1NDU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.32" data-doi="10.1186/s13059-014-0565-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lovell</span>,  <span class="cit-name-given-names">P.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wirthlin</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilhelm</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Minx</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lazar</span>,  <span class="cit-name-given-names">N.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carbone</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Warren</span>,  <span class="cit-name-given-names">W.C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mello</span>,  <span class="cit-name-given-names">C.V.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Conserved syntenic clusters of protein coding genes are missing in birds</span>. <abbr class="cit-jnl-abbrev">Genome biology</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.volume%253D15%26rft.spage%253D565%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0565-1%26rft_id%253Dinfo%253Apmid%252F25518852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0565-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25518852&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Loveridge</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farquharson</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hesketh</span>,  <span class="cit-name-given-names">J.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jakowlew</span>,  <span class="cit-name-given-names">S.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitehead</span>,  <span class="cit-name-given-names">C.C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Thorp</span>,  <span class="cit-name-given-names">B.H.</span></span> (<span class="cit-pub-date">1993</span>). <span class="cit-article-title">The control of chondrocyte differentiation during endochondral bone growth in vivo: changes in TGF-beta and the proto-oncogene c-myc</span>. <abbr class="cit-jnl-abbrev">J Cell Sci</abbr> <span class="cit-vol">105</span> ( Pt <span class="cit-issue">4</span>), <span class="cit-fpage">949</span>–<span class="cit-lpage">956</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BCell%2BScience%26rft.stitle%253DJ.%2BCell%2BSci.%26rft.aulast%253DLoveridge%26rft.auinit1%253DN.%26rft.volume%253D105%26rft.issue%253D4%26rft.spage%253D949%26rft.epage%253D956%26rft.atitle%253DThe%2Bcontrol%2Bof%2Bchondrocyte%2Bdifferentiation%2Bduring%2Bendochondral%2Bbone%2Bgrowth%2Bin%2Bvivo%253A%2Bchanges%2Bin%2BTGF-beta%2Band%2Bthe%2Bproto-oncogene%2Bc-myc%26rft_id%253Dinfo%253Apmid%252F8227216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6OToiMTA1LzQvOTQ5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMDUvMzE1NDU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.34" data-doi="10.1111/j.1439-0310.2005.01091.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Loyau</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jaime</span>,  <span class="cit-name-given-names">M.S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Sorci</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2005a</span>). <span class="cit-article-title">Intra- and Intersexual Selection for Multiple Traits in the Peacock (Pavo cristatus)</span>. <abbr class="cit-jnl-abbrev">Ethology</abbr> <span class="cit-vol">111</span>, <span class="cit-fpage">810</span>–<span class="cit-lpage">820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEthology%26rft.volume%253D111%26rft.spage%253D810%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0310.2005.01091.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0310.2005.01091.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000230841400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.35" data-doi="10.1007/s00265-005-0958-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Loyau</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saint Jaime</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cagniant</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Sorci</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2005b</span>). <span class="cit-article-title">Multiple sexual advertisements honestly reflect health status in peacocks (Pavo cristatus)</span>. <abbr class="cit-jnl-abbrev">Behavioral Ecology and Sociobiology</abbr> <span class="cit-vol">58</span>, <span class="cit-fpage">552</span>–<span class="cit-lpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehavioral%2BEcology%2Band%2BSociobiology%26rft.volume%253D58%26rft.spage%253D552%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00265-005-0958-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00265-005-0958-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000232792800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.36" data-doi="10.1038/368251a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moyle</span>,  <span class="cit-name-given-names">W.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campbell</span>,  <span class="cit-name-given-names">R.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Myers</span>,  <span class="cit-name-given-names">R.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bernard</span>,  <span class="cit-name-given-names">M.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span> (<span class="cit-pub-date">1994</span>). <span class="cit-article-title">Co-evolution of ligand-receptor pairs</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">368</span>, <span class="cit-fpage">251</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMoyle%26rft.auinit1%253DW.%2BR.%26rft.volume%253D368%26rft.issue%253D6468%26rft.spage%253D251%26rft.epage%253D255%26rft.atitle%253DCo-evolution%2Bof%2Bligand-receptor%2Bpairs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F368251a0%26rft_id%253Dinfo%253Apmid%252F8145825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/368251a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8145825&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NA87000058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.37" data-doi="10.1016/j.cub.2015.03.047"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nadachowska-Brzyska</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smeds</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ellegren</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Temporal dynamics of avian populations during Pleistocene revealed by whole-genome sequences</span>. <abbr class="cit-jnl-abbrev">Current Biology</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">1375</span>–<span class="cit-lpage">1380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BBiology%26rft.volume%253D25%26rft.spage%253D1375%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2015.03.047%26rft_id%253Dinfo%253Apmid%252F25891404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2015.03.047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25891404&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.38" data-doi="10.1111/mec.13540"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nadachowska-Brzyska</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burri</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smeds</span>,  <span class="cit-name-given-names">L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ellegren</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">PSMC analysis of effective population sizes in molecular ecology and its application to black-and-white Ficedula flycatchers</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">1058</span>–<span class="cit-lpage">1072</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D25%26rft.spage%253D1058%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13540&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nie</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luukko</span>,  <span class="cit-name-given-names">K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kettunen</span>,  <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">BMP signalling in craniofacial development</span>. <abbr class="cit-jnl-abbrev">Int J Dev Biol</abbr> <span class="cit-vol">50</span>, <span class="cit-fpage">511</span>–<span class="cit-lpage">521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BInternational%2Bjournal%2Bof%2Bdevelopmental%2Bbiology%26rft.stitle%253DInt%2BJ%2BDev%2BBiol%26rft.aulast%253DNie%26rft.auinit1%253DX.%26rft.volume%253D50%26rft.issue%253D6%26rft.spage%253D511%26rft.epage%253D521%26rft.atitle%253DBMP%2Bsignalling%2Bin%2Bcraniofacial%2Bdevelopment.%26rft_id%253Dinfo%253Apmid%252F16741866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16741866&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239034300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.40" data-doi="10.1093/jb/mvs004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hata</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matsubara</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wakabayashi</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Yoneda</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Regulation of bone and cartilage development by network between BMP signalling and transcription factors</span>. <abbr class="cit-jnl-abbrev">J Biochem</abbr> <span class="cit-vol">151</span>, <span class="cit-fpage">247</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiochem%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjb%252Fmvs004%26rft_id%253Dinfo%253Apmid%252F22253449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jb/mvs004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22253449&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-315457v1.41" data-doi="10.1093/nar/gkv1189"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">O’ leary</span>,  <span class="cit-name-given-names">N.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wright</span>,  <span class="cit-name-given-names">M.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brister</span>,  <span class="cit-name-given-names">J.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ciufo</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haddad</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mcveigh</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rajput</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robbertse</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith-White</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ako-Adjei</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Astashyn</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Badretdin</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bao</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blinkova</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brover</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chetvernin</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choi</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cox</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ermolaeva</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farrell</span>,  <span class="cit-name-given-names">C.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldfarb</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gupta</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haft</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hatcher</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hlavina</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joardar</span>,  <span class="cit-name-given-names">V.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kodali</span>,  <span class="cit-name-given-names">V.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maglott</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masterson</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mcgarvey</span>,  <span class="cit-name-given-names">K.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murphy</span>,  <span class="cit-name-given-names">M.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’neill</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pujar</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rangwala</span>,  <span class="cit-name-given-names">S.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rausch</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Riddick</span>,  <span class="cit-name-given-names">L.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schoch</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shkeda</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Storz</span>,  <span class="cit-name-given-names">S.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thibaud-Nissen</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tolstoy</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tully</span>,  <span class="cit-name-given-names">R.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vatsan</span>,  <span class="cit-name-given-names">A.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wallin</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Webb</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Landrum</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kimchi</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tatusova</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dicuccio</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kitts</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murphy</span>,  <span class="cit-name-given-names">T.D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Pruitt</span>,  <span class="cit-name-given-names">K.D.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">44</span>, <span class="cit-fpage">D733</span>–<span class="cit-lpage">745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1189%26rft_id%253Dinfo%253Apmid%252F26553804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26553804&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ouyang</span>,  <span class="cit-name-given-names">Y.-N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.-Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">D.-L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huo</span>,  <span class="cit-name-given-names">J.-L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qian</span>,  <span class="cit-name-given-names">K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Miao</span>,  <span class="cit-name-given-names">Y.-W.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Genetic Divergence between Pavo muticus and Pavo cristatus by Cyt b Gene</span>. <abbr class="cit-jnl-abbrev">Journal of Yunnan Agricultural University</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BYunnan%2BAgricultural%2BUniversity%26rft.volume%253D2%26rft.spage%253D014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.43" data-doi="10.1016/S0169-5347(00)88967-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Owens</span>,  <span class="cit-name-given-names">I.P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Short</span>,  <span class="cit-name-given-names">R.V.</span></span> (<span class="cit-pub-date">1995</span>). <span class="cit-article-title">Hormonal basis of sexual dimorphism in birds: implications for new theories of sexual selection</span>. <abbr class="cit-jnl-abbrev">Trends Ecol Evol</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">44</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DOwens%26rft.auinit1%253DI.%2BP.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D44%26rft.epage%253D47%26rft.atitle%253DHormonal%2Bbasis%2Bof%2Bsexual%2Bdimorphism%2Bin%2Bbirds%253A%2Bimplications%2Bfor%2Bnew%2Btheories%2Bof%2Bsexual%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252800%252988967-3%26rft_id%253Dinfo%253Apmid%252F21236951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-5347(00)88967-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21236951&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QB76300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.44" data-doi="10.1016/j.tcb.2004.09.016"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pixley</span>,  <span class="cit-name-given-names">F.J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Stanley</span>,  <span class="cit-name-given-names">E.R.</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">CSF-1 regulation of the wandering macrophage: complexity in action</span>. <abbr class="cit-jnl-abbrev">Trends Cell Biol</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">628</span>–<span class="cit-lpage">638</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bcell%2Bbiology%26rft.stitle%253DTrends%2BCell%2BBiol%26rft.aulast%253DPixley%26rft.auinit1%253DF.%2BJ.%26rft.volume%253D14%26rft.issue%253D11%26rft.spage%253D628%26rft.epage%253D638%26rft.atitle%253DCSF-1%2Bregulation%2Bof%2Bthe%2Bwandering%2Bmacrophage%253A%2Bcomplexity%2Bin%2Baction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tcb.2004.09.016%26rft_id%253Dinfo%253Apmid%252F15519852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tcb.2004.09.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15519852&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225270700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-315457v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pownall</span>,  <span class="cit-name-given-names">M.E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Isaacs</span>,  <span class="cit-name-given-names">H.V.</span></span> (Year). “<span class="cit-article-title">Fgf signalling in vertebrate development</span>”, in: <abbr class="cit-jnl-abbrev">Colloquium Series on Developmental Biology: Morgan &amp; Claypool Life Sciences</abbr>), <span class="cit-fpage">1</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ramesh</span>,  <span class="cit-name-given-names">K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mcgowan</span>,  <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">On the current status of Indian peafowl Pavo cristatus (Aves: Galliformes: Phasianidae): keeping the common species common</span>. <abbr class="cit-jnl-abbrev">Journal of Threatened Taxa</abbr> <span class="cit-vol">1</span>, <span class="cit-fpage">106</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BThreatened%2BTaxa%26rft.volume%253D1%26rft.spage%253D106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.47" data-doi="10.1016/S0168-9525(00)02024-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rice</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Longden</span>,  <span class="cit-name-given-names">I.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bleasby</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">EMBOSS: the European Molecular Biology Open Software Suite</span>. <abbr class="cit-jnl-abbrev">Trends Genet</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">276</span>–<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DRice%26rft.auinit1%253DP.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D276%26rft.epage%253D277%26rft.atitle%253DEMBOSS%253A%2Bthe%2BEuropean%2BMolecular%2BBiology%2BOpen%2BSoftware%2BSuite.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252800%252902024-2%26rft_id%253Dinfo%253Apmid%252F10827456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(00)02024-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10827456&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087457200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.48" data-doi="10.1038/ncomms10587"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Serna</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giles</span>,  <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morgan</span>,  <span class="cit-name-given-names">B.P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bubeck</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Structural basis of complement membrane attack complex formation</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">10587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D7%26rft.spage%253D10587%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms10587%26rft_id%253Dinfo%253Apmid%252F26841837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms10587&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26841837&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stock</span>,  <span class="cit-name-given-names">A.D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bunch</span>,  <span class="cit-name-given-names">T.D.</span></span> (<span class="cit-pub-date">1982</span>). <span class="cit-article-title">The evolutionary implications of chromosome banding pattern homologies in the bird order Galliformes</span>. <abbr class="cit-jnl-abbrev">Cytogenet Cell Genet</abbr> <span class="cit-vol">34</span>, <span class="cit-fpage">136</span>–<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetics%2Band%2Bcell%2Bgenetics%26rft.stitle%253DCytogenet%2BCell%2BGenet%26rft.aulast%253DStock%26rft.auinit1%253DA.%2BD.%26rft.volume%253D34%26rft.issue%253D1-2%26rft.spage%253D136%26rft.epage%253D148%26rft.atitle%253DThe%2Bevolutionary%2Bimplications%2Bof%2Bchromosome%2Bbanding%2Bpattern%2Bhomologies%2Bin%2Bthe%2Bbird%2Border%2BGalliformes.%26rft_id%253Dinfo%253Apmid%252F7151485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7151485&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.50" data-doi="10.1016/j.virusres.2007.06.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">G.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">J.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fu</span>,  <span class="cit-name-given-names">J.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ren</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cao</span>,  <span class="cit-name-given-names">W.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xin</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liao</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">W.J.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">A Massachusetts prototype like coronavirus isolated from wild peafowls is pathogenic to chickens</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">130</span> <span class="cit-fpage">121</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DSun%26rft.auinit1%253DL.%26rft.volume%253D130%26rft.issue%253D1-2%26rft.spage%253D121%26rft.epage%253D128%26rft.atitle%253DA%2BMassachusetts%2Bprototype%2Blike%2Bcoronavirus%2Bisolated%2Bfrom%2Bwild%2Bpeafowls%2Bis%2Bpathogenic%2Bto%2Bchickens.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2007.06.003%26rft_id%253Dinfo%253Apmid%252F17629993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2007.06.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17629993&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.51" data-doi="10.1101/gr.073585.107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vilella</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Severin</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ureta-Vidal</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heng</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Durbin</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Birney</span>,  <span class="cit-name-given-names">E.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">EnsemblCompara GeneTrees: Complete, duplication-aware phylogenetic trees in vertebrates</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">327</span>–<span class="cit-lpage">335</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.073585.107%26rft_id%253Dinfo%253Apmid%252F19029536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzIvMzI3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDUvMDUvMzE1NDU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.52" data-doi="10.1016/1074-7613(94)90071-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walunas</span>,  <span class="cit-name-given-names">T.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lenschow</span>,  <span class="cit-name-given-names">D.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bakker</span>,  <span class="cit-name-given-names">C.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Linsley</span>,  <span class="cit-name-given-names">P.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Freeman</span>,  <span class="cit-name-given-names">G.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Green</span>,  <span class="cit-name-given-names">J.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thompson</span>,  <span class="cit-name-given-names">C.B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bluestone</span>,  <span class="cit-name-given-names">J.A.</span></span> (<span class="cit-pub-date">1994</span>). <span class="cit-article-title">CTLA-4 can function as a negative regulator of T cell activation</span>. <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-vol">1</span>, <span class="cit-fpage">405</span>–<span class="cit-lpage">413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.stitle%253DImmunity%26rft.aulast%253DWalunas%26rft.auinit1%253DT.%2BL.%26rft.volume%253D1%26rft.issue%253D5%26rft.spage%253D405%26rft.epage%253D413%26rft.atitle%253DCTLA-4%2Bcan%2Bfunction%2Bas%2Ba%2Bnegative%2Bregulator%2Bof%2BT%2Bcell%2Bactivation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F1074-7613%252894%252990071-X%26rft_id%253Dinfo%253Apmid%252F7882171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/1074-7613(94)90071-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7882171&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PK16800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.53" data-doi="10.1371/journal.pone.0064312"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kimball</span>,  <span class="cit-name-given-names">R.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Braun</span>,  <span class="cit-name-given-names">E.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>,  <span class="cit-name-given-names">B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Assessing phylogenetic relationships among galliformes: a multigene phylogeny with expanded taxon sampling in Phasianidae</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">e64312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De64312%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0064312%26rft_id%253Dinfo%253Apmid%252F23741315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0064312&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23741315&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.54" data-doi="10.1111/mec.13113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wright</span>,  <span class="cit-name-given-names">A.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrison</span>,  <span class="cit-name-given-names">P.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zimmer</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Montgomery</span>,  <span class="cit-name-given-names">S.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pointer</span>,  <span class="cit-name-given-names">M.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mank</span>,  <span class="cit-name-given-names">J.E.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Variation in promiscuity and sexual selection drives avian rate of Faster-Z evolution</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">1218</span>–<span class="cit-lpage">1235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D24%26rft.spage%253D1218%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.55" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">PAML 4: phylogenetic analysis by maximum likelihood</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">1586</span>–<span class="cit-lpage">1591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17483113&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248848400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.56" data-doi="10.1093/molbev/msi097"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">W.S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Bayes empirical bayes inference of amino acid sites under positive selection</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">1107</span>–<span class="cit-lpage">1118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi097%26rft_id%253Dinfo%253Apmid%252F15689528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msi097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15689528&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228139400031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.57" data-doi="10.1016/0022-5193(75)90111-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zahavi</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">1975</span>). <span class="cit-article-title">Mate selection—A selection for a handicap</span>. <abbr class="cit-jnl-abbrev">Journal of Theoretical Biology</abbr> <span class="cit-vol">53</span>, <span class="cit-fpage">205</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DZahavi%26rft.auinit1%253DA.%26rft.volume%253D53%26rft.issue%253D1%26rft.spage%253D205%26rft.epage%253D214%26rft.atitle%253DMate%2Bselection-a%2Bselection%2Bfor%2Ba%2Bhandicap.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-5193%252875%252990111-3%26rft_id%253Dinfo%253Apmid%252F1195756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-5193(75)90111-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1195756&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AR81900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.58" data-doi="10.1093/molbev/msi237"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">2472</span>–<span class="cit-lpage">2479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi237%26rft_id%253Dinfo%253Apmid%252F16107592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msi237&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16107592&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233361500014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-315457v1.59" data-doi="10.1016/j.immuni.2014.01.010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mo</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Swanson</span>,  <span class="cit-name-given-names">K.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wen</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petrucelli</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gregory</span>,  <span class="cit-name-given-names">S.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schneider</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Fitzgerald</span>,  <span class="cit-name-given-names">K.A.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">NLRC3, a member of the NLR family of proteins, is a negative regulator of innate immune signaling induced by the DNA sensor STING</span>. <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-vol">40</span>, <span class="cit-fpage">329</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.volume%253D40%26rft.spage%253D329%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.immuni.2014.01.010%26rft_id%253Dinfo%253Apmid%252F24560620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.immuni.2014.01.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24560620&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F05%2F05%2F315457.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000333422300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/315457v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F315457v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/315234v1" title="HiCAGE : an R package for large-scale annotation and visualization of 3C-based genomic data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;May 05, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/315457v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/315457v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/315457v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/315457v1" class="active" data-icon-position="" data-hide-link-title="0">Genome Sequence of Indian Peacock Reveals the Peculiar Case of a Glittering Bird</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/96725" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-13Ku0WbArvEvNVNk_YbiPiGOf3UObCvjQTlh1iv8SZ8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="96725" id="node96725--2" data-pisa="biorxiv;315457v1" data-pisa-master="biorxiv;315457" data-seqnum="96725" data-apath="/biorxiv/early/2018/05/05/315457.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Genome Sequence of Indian Peacock Reveals the Peculiar Case of a Glittering Bird</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Shubham K.</span> <span class="nlm-surname">Jaiswal</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Ankit</span> <span class="nlm-surname">Gupta</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Rituja</span> <span class="nlm-surname">Saxena</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">P. K.</span> <span class="nlm-surname">Vishnu Prasoodanan</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Ashok K.</span> <span class="nlm-surname">Sharma</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Parul</span> <span class="nlm-surname">Mittal</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Ankita</span> <span class="nlm-surname">Roy</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Aaron B.A.</span> <span class="nlm-surname">Shafer</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Nagarjun</span> <span class="nlm-surname">Vijay</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Vineet K.</span> <span class="nlm-surname">Sharma</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">315457; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/315457 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/315457v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/315457v1&amp;title=Genome%20Sequence%20of%20Indian%20Peacock%20Reveals%20the%20Peculiar%20Case%20of%20a%20Glittering%20Bird" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/315457v1&amp;title=Genome%20Sequence%20of%20Indian%20Peacock%20Reveals%20the%20Peculiar%20Case%20of%20a%20Glittering%20Bird" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/315457v1&amp;text=Genome%20Sequence%20of%20Indian%20Peacock%20Reveals%20the%20Peculiar%20Case%20of%20a%20Glittering%20Bird" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/315457v1&amp;title=Genome%20Sequence%20of%20Indian%20Peacock%20Reveals%20the%20Peculiar%20Case%20of%20a%20Glittering%20Bird" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/315457v1&amp;t=Genome%20Sequence%20of%20Indian%20Peacock%20Reveals%20the%20Peculiar%20Case%20of%20a%20Glittering%20Bird" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/315457v1&amp;title=Genome%20Sequence%20of%20Indian%20Peacock%20Reveals%20the%20Peculiar%20Case%20of%20a%20Glittering%20Bird" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/315457v1&amp;title=Genome%20Sequence%20of%20Indian%20Peacock%20Reveals%20the%20Peculiar%20Case%20of%20a%20Glittering%20Bird" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="96725" id="node96725--3" data-pisa="biorxiv;315457v1" data-pisa-master="biorxiv;315457" data-seqnum="96725" data-apath="/biorxiv/early/2018/05/05/315457.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Genome Sequence of Indian Peacock Reveals the Peculiar Case of a Glittering Bird</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Shubham K.</span> <span class="nlm-surname">Jaiswal</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Ankit</span> <span class="nlm-surname">Gupta</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Rituja</span> <span class="nlm-surname">Saxena</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">P. K.</span> <span class="nlm-surname">Vishnu Prasoodanan</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Ashok K.</span> <span class="nlm-surname">Sharma</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Parul</span> <span class="nlm-surname">Mittal</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Ankita</span> <span class="nlm-surname">Roy</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Aaron B.A.</span> <span class="nlm-surname">Shafer</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Nagarjun</span> <span class="nlm-surname">Vijay</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Vineet K.</span> <span class="nlm-surname">Sharma</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">315457; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/315457 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Genome%20Sequence%20of%20Indian%20Peacock%20Reveals%20the%20Peculiar%20Case%20of%20a%20Glittering%20Bird&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Jaiswal%2C+Shubham+K.&amp;rft.au=Gupta%2C+Ankit&amp;rft.au=Saxena%2C+Rituja&amp;rft.au=Vishnu+Prasoodanan%2C+P.+K.&amp;rft.au=Sharma%2C+Ashok+K.&amp;rft.au=Mittal%2C+Parul&amp;rft.au=Roy%2C+Ankita&amp;rft.au=Shafer%2C+Aaron+B.A.&amp;rft.au=Vijay%2C+Nagarjun&amp;rft.au=Sharma%2C+Vineet+K."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/96725/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/96725/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/96725/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/96725/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/96725/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/96725/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/96725/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/96725/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/96725/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/96725/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/96725/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/96725/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/315457v1&amp;count=horizontal&amp;via=&amp;text=Genome%20Sequence%20of%20Indian%20Peacock%20Reveals%20the%20Peculiar%20Case%20of%20a%20Glittering%20Bird&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/315457v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/315457v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/315457v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">Genomics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20277)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
