<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/12/2020.03.10.986398.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.10.986398v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The SARS-CoV-2 exerts a distinctive strategy for interacting with the ACE2 human receptor" />
<meta name="DC.Identifier" content="10.1101/2020.03.10.986398" />
<meta name="DC.Date" content="2020-03-12" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The COVID-19 disease has plagued over 110 countries and has resulted in over 4,000 deaths within 10 weeks. We compare the interaction between the human ACE2 receptor and the SARS-CoV-2 spike protein with that of other pathogenic coronaviruses using molecular dynamics simulations. SARS-CoV, SARS-CoV-2, and HCoV-NL63 recognize ACE2 as the natural receptor but present a distinct binding interface to ACE2 and a different network of residue-residue contacts. SARS-CoV and SARS-CoV-2 have comparable binding affinities achieved by balancing energetics and dynamics. The SARS-CoV-2–ACE2 complex contains a higher number of contacts, a larger interface area, and decreased interface residue fluctuations relative to SARS-CoV. These findings expose an exceptional evolutionary exploration exerted by coronaviruses toward host recognition. We postulate that the versatility of cell receptor binding strategies has immediate implications on therapeutic strategies.

One Sentence Summary Molecular dynamics simulations reveal a temporal dimension of coronaviruses interactions with the host receptor." />
<meta name="DC.Contributor" content="Esther S. Brielle" />
<meta name="DC.Contributor" content="Dina Schneidman-Duhovny" />
<meta name="DC.Contributor" content="Michal Linial" />
<meta name="article:published_time" content="2020-03-12" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="The SARS-CoV-2 exerts a distinctive strategy for interacting with the ACE2 human receptor" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The COVID-19 disease has plagued over 110 countries and has resulted in over 4,000 deaths within 10 weeks. We compare the interaction between the human ACE2 receptor and the SARS-CoV-2 spike protein with that of other pathogenic coronaviruses using molecular dynamics simulations. SARS-CoV, SARS-CoV-2, and HCoV-NL63 recognize ACE2 as the natural receptor but present a distinct binding interface to ACE2 and a different network of residue-residue contacts. SARS-CoV and SARS-CoV-2 have comparable binding affinities achieved by balancing energetics and dynamics. The SARS-CoV-2 - ACE2 complex contains a higher number of contacts, a larger interface area, and decreased interface residue fluctuations relative to SARS-CoV. These findings expose an exceptional evolutionary exploration exerted by coronaviruses toward host recognition. We postulate that the versatility of cell receptor binding strategies has immediate implications on therapeutic strategies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.10.986398v1" />
<meta name="citation_id" content="2020.03.10.986398v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.10.986398v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.10.986398v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.10.986398v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.10.986398" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.10.986398" />
<meta name="citation_author" content="Esther S. Brielle" />
<meta name="citation_author_institution" content="Department of Biological Chemistry, Institute of Life Sciences, The Hebrew University of Jerusalem" />
<meta name="citation_author_institution" content="The Alexander Grass Center for Bioengineering, The Hebrew University of Jerusalem" />
<meta name="citation_author" content="Dina Schneidman-Duhovny" />
<meta name="citation_author_institution" content="Department of Biological Chemistry, Institute of Life Sciences, The Hebrew University of Jerusalem" />
<meta name="citation_author_institution" content="The Rachel and Selim Benin School of Computer Science and Engineering, The Hebrew University of Jerusalem" />
<meta name="citation_author_email" content="michall@cc.huji.ac.il" />
<meta name="citation_author_email" content="dina.schneidman@mail.huji.ac.il" />
<meta name="citation_author" content="Michal Linial" />
<meta name="citation_author_institution" content="Department of Biological Chemistry, Institute of Life Sciences, The Hebrew University of Jerusalem" />
<meta name="citation_author_email" content="michall@cc.huji.ac.il" />
<meta name="citation_author_email" content="dina.schneidman@mail.huji.ac.il" />
<meta name="citation_reference" content="P. A. Rota et al., Characterization of a novel coronavirus associated with severe acute respiratory syndrome. Science 300, 1394–1399 (2003)." />
<meta name="citation_reference" content="L. E. Gralinski, V. D. Menachery, Return of the Coronavirus: 2019-nCoV. Viruses 12, 135 (2020)." />
<meta name="citation_reference" content="S. R. Weiss, S. Navas-Martin, Coronavirus pathogenesis and the emerging pathogen severe acute respiratory syndrome coronavirus. Microbiol Mol Biol Rev 69, 635–664 (2005)." />
<meta name="citation_reference" content="Z. Q. Zeng et al., Epidemiology and clinical characteristics of human coronaviruses OC43, 229E, NL63, and HKU1: a study of hospitalized children with acute respiratory tract infection in Guangzhou, China. Eur J Clin Microbiol Infect Dis 37, 363–369 (2018)." />
<meta name="citation_reference" content="F. Wu et al., A new coronavirus associated with human respiratory disease in China. Nature, 1–5 (2020)." />
<meta name="citation_reference" content="M. A. Tortorici et al., Structural basis for human coronavirus attachment to sialic acid receptors. Nature structural &amp; molecular biology 26, 481–489 (2019)." />
<meta name="citation_reference" content="K. Kuba et al., A crucial role of angiotensin converting enzyme 2 (ACE2) in SARS coronavirus– induced lung injury. Nature medicine 11, 875–879 (2005)." />
<meta name="citation_reference" content="H. Hofmann et al., Susceptibility to SARS coronavirus S protein-driven infection correlates with expression of angiotensin converting enzyme 2 and infection can be blocked by soluble receptor. Biochemical and biophysical research communications 319, 1216–1221 (2004)." />
<meta name="citation_reference" content="E. C. Mossel et al., Exogenous ACE2 expression allows refractory cell lines to support severe acute respiratory syndrome coronavirus replication. J Virol 79, 3846–3850 (2005)." />
<meta name="citation_reference" content="I. Hamming et al., Tissue distribution of ACE2 protein, the functional receptor for SARS coronavirus. A first step in understanding SARS pathogenesis. The Journal of Pathology: A Journal of the Pathological Society of Great Britain and Ireland 203, 631–637 (2004)." />
<meta name="citation_reference" content="Y. Wan, J. Shang, R. Graham, R. S. Baric, F. Li, Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS. J Virol (2020)." />
<meta name="citation_reference" content="H. Hofmann et al., Human coronavirus NL63 employs the severe acute respiratory syndrome coronavirus receptor for cellular entry. Proc Natl Acad Sci U S A 102, 7988–7993 (2005)." />
<meta name="citation_reference" content="W. Li et al., The S proteins of human coronavirus NL63 and severe acute respiratory syndrome coronavirus bind overlapping regions of ACE2. Virology 367, 367–374 (2007)." />
<meta name="citation_reference" content="N. Wang et al., Structure of MERS-CoV spike receptor-binding domain complexed with human receptor DPP4. Cell research 23, 986 (2013)." />
<meta name="citation_reference" content="Y. Zhang, J. Skolnick, TM-align: a protein structure alignment algorithm based on the TM-score. Nucleic acids research 33, 2302–2309 (2005)." />
<meta name="citation_reference" content="G. Q. Dong, H. Fan, D. Schneidman-Duhovny, B. Webb, A. Sali, Optimized atomic statistical potentials: assessment of protein interfaces and loops. Bioinformatics 29, 3158–3166 (2013)." />
<meta name="citation_reference" content="R. Yan, Y. Zhang, Y. Guo, L. Xia, Q. Zhou, Structural basis for the recognition of the 2019-nCoV by human ACE2. bioRxiv (2020)." />
<meta name="citation_reference" content="Y. Zhang, N. Zheng, Y. Zhong, Computational characterization and design of SARS coronavirus receptor recognition and antibody neutralization. Computational biology and chemistry 31, 129–133 (2007)." />
<meta name="citation_reference" content="W. Li et al., Receptor and viral determinants of SARS-coronavirus adaptation to human ACE2. EMBO J 24, 1634–1643 (2005)." />
<meta name="citation_reference" content="X. Tian et al., Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody. Emerg Microbes Infect 9, 382–385 (2020)." />
<meta name="citation_reference" content="K. Wu, W. Li, G. Peng, F. Li, Crystal structure of NL63 respiratory coronavirus receptor-binding domain complexed with its human receptor. Proceedings of the National Academy of Sciences 106, 19970–19974 (2009)." />
<meta name="citation_reference" content="A. C. Walls et al., Structure, function and antigenicity of the SARS-CoV-2 spike glycoprotein. bioRxiv (2020)." />
<meta name="citation_reference" content="D. Wrapp et al., Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science (2020)." />
<meta name="citation_reference" content="A.-W. Struck, M. Axmann, S. Pfefferle, C. Drosten, B. Meyer, A hexapeptide of the receptor-binding domain of SARS corona virus spike protein blocks viral entry into host cells via the human receptor ACE2. Antiviral research 94, 288–296 (2012)." />
<meta name="citation_reference" content="L. Du et al., The spike protein of SARS-CoV--a target for vaccine and therapeutic development. Nat Rev Microbiol 7, 226–236 (2009)." />
<meta name="citation_reference" content="V. Kunik, Y. Ofran, The indistinguishability of epitopes from protein surface is explained by the distinct binding preferences of each of the six antigen-binding loops. Protein Eng Des Sel 26, 599–609 (2013)." />
<meta name="citation_reference" content="B. Webb, A. Sali, Comparative protein structure modeling using MODELLER. Current protocols in bioinformatics 47, 5.6. 1–5.6. 32 (2014)." />
<meta name="citation_reference" content="S. Pronk et al., GROMACS 4.5: a high-throughput and highly parallel open source molecular simulation toolkit. Bioinformatics 29, 845–854 (2013)." />
<meta name="citation_reference" content="J. Huang et al., CHARMM36m: an improved force field for folded and intrinsically disordered proteins. Nature methods 14, 71–73 (2017)." />
<meta name="twitter:title" content="The SARS-CoV-2 exerts a distinctive strategy for interacting with the ACE2 human receptor" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The COVID-19 disease has plagued over 110 countries and has resulted in over 4,000 deaths within 10 weeks. We compare the interaction between the human ACE2 receptor and the SARS-CoV-2 spike protein with that of other pathogenic coronaviruses using molecular dynamics simulations. SARS-CoV, SARS-CoV-2, and HCoV-NL63 recognize ACE2 as the natural receptor but present a distinct binding interface to ACE2 and a different network of residue-residue contacts. SARS-CoV and SARS-CoV-2 have comparable binding affinities achieved by balancing energetics and dynamics. The SARS-CoV-2–ACE2 complex contains a higher number of contacts, a larger interface area, and decreased interface residue fluctuations relative to SARS-CoV. These findings expose an exceptional evolutionary exploration exerted by coronaviruses toward host recognition. We postulate that the versatility of cell receptor binding strategies has immediate implications on therapeutic strategies.

One Sentence Summary Molecular dynamics simulations reveal a temporal dimension of coronaviruses interactions with the host receptor." />
<meta name="og-title" property="og:title" content="The SARS-CoV-2 exerts a distinctive strategy for interacting with the ACE2 human receptor" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.10.986398v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The COVID-19 disease has plagued over 110 countries and has resulted in over 4,000 deaths within 10 weeks. We compare the interaction between the human ACE2 receptor and the SARS-CoV-2 spike protein with that of other pathogenic coronaviruses using molecular dynamics simulations. SARS-CoV, SARS-CoV-2, and HCoV-NL63 recognize ACE2 as the natural receptor but present a distinct binding interface to ACE2 and a different network of residue-residue contacts. SARS-CoV and SARS-CoV-2 have comparable binding affinities achieved by balancing energetics and dynamics. The SARS-CoV-2–ACE2 complex contains a higher number of contacts, a larger interface area, and decreased interface residue fluctuations relative to SARS-CoV. These findings expose an exceptional evolutionary exploration exerted by coronaviruses toward host recognition. We postulate that the versatility of cell receptor binding strategies has immediate implications on therapeutic strategies.

One Sentence Summary Molecular dynamics simulations reveal a temporal dimension of coronaviruses interactions with the host receptor." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-12" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.10.986398v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.10.986398v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.10.986398v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.10.986398v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1192797" />
    <title>The SARS-CoV-2 exerts a distinctive strategy for interacting with the ACE2 human receptor | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"FugmjjJLm4IPLK4NRNjLKGhfY-JpmCVdRZxTAJKdO8k","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1192797","apath":"\/biorxiv\/early\/2020\/03\/12\/2020.03.10.986398.atom","pisa":"biorxiv;2020.03.10.986398v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.10.986398v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.10.986398v1","title":"The SARS-CoV-2 exerts a distinctive strategy for interacting with the ACE2 human receptor","identifier":"node\/1192797"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223yv7cbaX2Mn51WPJMGYZhEYjhuTbufjRc0+1qIymvKlKOaOF439+C5y1SVFHWPLo5AYpBWs7aOeFn7IX\\\\\\\/NQ2PSLBnjwJd9C1e6X2WF6KaRtGDD49VGlE7WAm\\\\\\\/YaYqzGyvucpFZAf7ryhVkNjZmsKWXu\\\\\\\/DK6QfSvdhkqymgPP7vdhM6aE8b8aeQ1Yhp\\\\\\\/nuZD4HQmGd8deX5aSLjzi\\\\\\\/AEc9RR7ZDB9SGKJdvlhsIyLd34oAGeuKixePvq6iwXFLekLlkhEzIAZf0e30\\\\\\\/KHjs57BKl1KRe8sPafPLLaxmzv7Z+bWNVIVIlsdyAcMmwRjTylvN1K1ri8EJ7IoDIKzDHO6QUyehi8\\\\\\\/3K\\\\\\\/trASmXydePgcBINHab4GBZLjJG+\\\\\\\/aYGt2azFBpNrIeZZ+QbyjdhscWFdXZ0KbLnaTgs9KyAgiihRZ0ynT+s+lsbkGydirvm9Ol9mH1oljlht3sEGyo2+5E3Z47JTZi2MHbYGNKYX8ZOm\\\\\\\/27QHOCxM3Llso\\\\\\\/XZl4qDA+3ahagygsYE8K+pb0mKDa+8pjPGnoQRAW01yfmu7i0VBqgRUAzwA6tOJH4PwixwocVia4ig6I5p3i\\\\\\\/VOp0MZ3wP7W7+tRRBgwSOm0P9xQdTVPZBhVjwxtl+Se7uRn2p+Eb1jljTRB7YIOhVoueWlFuyaPT0UfW8Kl+eZJoF0KH12\\\\\\\/EN8pAFKN4Ad1LL5ZIDrfN\\\\\\\/rRsf\\\\\\\/nL4m1NxW\\\\\\\/0yT49zw\\\\\\\/FoJtpT8VjjAsBAJIGcKYDogvaQokC+uxzPdjQOKu226G\\\\\\\/az8ePRovjAQMd\\\\\\\/5xCkTGZwMXH2zfSI+EQk7Fohv+rSwq7lfLAwBq3axwwmXIkajY1U\\\\\\\/DsPxqL+UK9Lu9nHqlaspXr37hOYRMNijsBGJ1yCDoz8KqSdcSMosqIN5d2MPq3vzA2o2Tu3Grn8O0T+d41Bd020A6atjz1rm2OD1FqoG3gwf2y6Jrz7qH5tlqtiJg4OaIOuhm5jXmFNLHqmd+ZKrshxW6D8k74moCb\\\\\\\/jpZ9iWbZyS02wg1laZy1MweUW0uNjgfqqGJMrh5jFf3dXHik\\\\\\\/YCd0EIS179xOzFjKTGYE9SvkES+gi1pPTKkNTuKN1rNWZjzC3zw3TC\\\\\\\/aYVbM2k2jjX+7LHGzkdxo2YWRx5CMch0bS+mTOgPF4dXTZObUYGjwqXhDrryV1sOU8X3zFEDlwjzaEG6V4iORSzaqPo7aQWXXXMSaju7Gzv2B8j8dmoRdzBK5oRMz9mQY3nH6uYk6PUd9J6qAwmoyD\\\\\\\/kURQA5w5yWDJkF4wzwF9pYEb\\\\\\\/KBDYeXSi0Vw7RGe+HEeuAMPHSLSYpbiAMWKk6ptJQS5ZaotMnCLcs8JP4f\\\u0022,\\\u0022iv\\\u0022:\\\u00223ey2GVeTARarahB5yYWM5w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022674c512b03023dc59b6542f2314950f4288d43df810694b734d2e75dc8fdd463\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.10.986398v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1192797 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.10.986398v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-m_WtS194MeCATunOTnLOGNEJdJFzHFAj9b4anItXgGw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1192797" id="node1192797" data-pisa="biorxiv;2020.03.10.986398v1" data-pisa-master="biorxiv;2020.03.10.986398" data-apath="/biorxiv/early/2020/03/12/2020.03.10.986398.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">The SARS-CoV-2 exerts a distinctive strategy for interacting with the ACE2 human receptor</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Esther S.</span> <span class="nlm-surname">Brielle</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Dina</span> <span class="nlm-surname">Schneidman-Duhovny</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Michal</span> <span class="nlm-surname">Linial</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.10.986398 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1192797" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Esther S. Brielle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biological Chemistry, Institute of Life Sciences, The Hebrew University of Jerusalem</span>, Jerusalem, <span class='nlm-country'>Israel</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>The Alexander Grass Center for Bioengineering, The Hebrew University of Jerusalem</span>, Jerusalem, <span class='nlm-country'>Israel</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Esther%2BS.%2BBrielle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brielle%20ES&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEsther%2BS.%2BBrielle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dina Schneidman-Duhovny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biological Chemistry, Institute of Life Sciences, The Hebrew University of Jerusalem</span>, Jerusalem, <span class='nlm-country'>Israel</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>The Rachel and Selim Benin School of Computer Science and Engineering, The Hebrew University of Jerusalem</span>, Jerusalem, <span class='nlm-country'>Israel</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dina%2BSchneidman-Duhovny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schneidman-Duhovny%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADina%2BSchneidman-Duhovny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:michall@cc.huji.ac.il" class="" data-icon-position="" data-hide-link-title="0">michall@cc.huji.ac.il</a>
<a href="mailto:dina.schneidman@mail.huji.ac.il" class="" data-icon-position="" data-hide-link-title="0">dina.schneidman@mail.huji.ac.il</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michal Linial </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biological Chemistry, Institute of Life Sciences, The Hebrew University of Jerusalem</span>, Jerusalem, <span class='nlm-country'>Israel</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michal%2BLinial%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Linial%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichal%2BLinial%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:michall@cc.huji.ac.il" class="" data-icon-position="" data-hide-link-title="0">michall@cc.huji.ac.il</a>
<a href="mailto:dina.schneidman@mail.huji.ac.il" class="" data-icon-position="" data-hide-link-title="0">dina.schneidman@mail.huji.ac.il</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.10.986398v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1192797" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1192797/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.10.986398v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1192797" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1192797/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.10.986398v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1192797" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1192797/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.10.986398v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1192797" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1192797/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.10.986398v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1192797" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1192797/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.10.986398v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1192797" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1192797/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">The COVID-19 disease has plagued over 110 countries and has resulted in over 4,000 deaths within 10 weeks. We compare the interaction between the human ACE2 receptor and the SARS-CoV-2 spike protein with that of other pathogenic coronaviruses using molecular dynamics simulations. SARS-CoV, SARS-CoV-2, and HCoV-NL63 recognize ACE2 as the natural receptor but present a distinct binding interface to ACE2 and a different network of residue-residue contacts. SARS-CoV and SARS-CoV-2 have comparable binding affinities achieved by balancing energetics and dynamics. The SARS-CoV-2–ACE2 complex contains a higher number of contacts, a larger interface area, and decreased interface residue fluctuations relative to SARS-CoV. These findings expose an exceptional evolutionary exploration exerted by coronaviruses toward host recognition. We postulate that the versatility of cell receptor binding strategies has immediate implications on therapeutic strategies.</p><div id="sec-1" class="subsection"><p id="p-3"><strong>One Sentence Summary</strong> Molecular dynamics simulations reveal a temporal dimension of coronaviruses interactions with the host receptor.</p></div></div><div class="section" id="sec-2"><h2 class="">Main Text</h2><p id="p-4">Coronavirus disease 2019 (COVID-19), initially detected in the Wuhan seafood market in the Hubei province of China (<em><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></em>) is caused by SARS-CoV-2 (referred to as the COVID-19 virus for clarity). The COVID-19 virus already spread within 10 weeks from its appearance to more than 110 countries, resulting in over 4,000 deaths worldwide. The COVID-19 virus is capable of human-to-human transmission and was introduced to humans in a zoonotic event (<em><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></em>).</p><p id="p-5">Currently, seven confirmed coronavirus (CoV) species are known as human pathogens (<em><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></em>). Four CoV are endemic species in humans and cause mild respiratory symptoms, mostly in pediatric patients (<em><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></em>). These are the HCoV-HKU1 and HCoV-OC43 from the betacoronavirus (BCoV) genus and the HCoV-229E, and HCoV-NL63 from the alphacoronavirus (ACoV) genus. The other human CoVs have caused severe outbreaks (<strong><a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table S1</a></strong>). The SARS-CoV (referred to as the SARS-2002 virus for clarity), is a BCoV that emerged in humans in 2002, giving rise to the Severe Acute Respiratory Syndrome (SARS) outbreak. The Middle East Respiratory Syndrome (MERS) BCoV caused an outbreak in 2012-2013. Most recently, SARS-CoV-2, with high homology to the 2002 SARS-CoV, caused the current pandemic-like COVID-19 outbreak (<em><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></em>).</p><p id="p-6">To gain access to host cells, coronaviruses rely on spike proteins, which are membrane-anchored trimers containing a receptor-binding S1 segment and a membrane-fusion S2 segment (<em><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></em>). The S1 segment contains a receptor binding domain (RBD) that recognizes and binds to a host cell receptor. The angiotensin-converting enzyme 2 (ACE2) was identified as the critical receptor for mediating SARS-2002 entry into host cells (<em><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></em>). Binding of the spike protein to the receptor is a critical phase where the levels of the ACE2 expressed on the cell membrane correlates with viral infectivity, and govern clinical outcomes (<em><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></em>). Consistent with the clinical pulmonary manifestation, ACE2 is widely expressed in almost all tissues, with the highest expression levels in the epithelium of the lung (<em><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></em>). Similar to the SARS-2002 virus, the COVID-19 virus enters the host cell by RBD binding to the host cell ACE2 receptor (<em><a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></em>). Host receptor recognition for cell entry is, however, not specified by the CoV genus classification. MERS-CoV is a member of the BCoV genus but does not recognize the ACE2 receptor. In contrast, HCoV-NL63 is a member of the ACoV genus and does recognize the ACE2 receptor (<em><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></em>).</p><p id="p-7">Herein, we analyze the binding of several CoV RBDs to ACE2 with molecular dynamics (MD) simulations and compare the stability, relative interaction strength, and dynamics of the interaction between the viral spike protein and the human ACE2 receptor.</p><p id="p-8">The COVID-19 RBD (residues 319-529) shares a 72.8% sequence identity and high structural similarity with the SARS-2002 RBD (<strong><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a></strong>). In contrast, the RBD of HCoV-NL63 is only 17.1% identical to that of COVID-19 and there are no significant structural similarities between them (<strong><a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. S1</a></strong>). Remarkably, the RBD of MERS-CoV, which is structurally similar to that of COVID-19 (20.1% sequence identity, 65% structure similarity) recognizes a different host receptor (DPP4) for its cell entry and does not bind ACE2 (<em><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></em>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1197589/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1197589/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1197589" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-9" class="first-child">The sequence and structural resemblance of the RBDs of various human coronaviruses.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">We ran 100ns molecular dynamic (MD) simulations of ACE2 in complex with the RBDs of the COVID-19, SARS-2002, and HCoV-NL63 viruses to quantify the energetics and the dynamics of the different RBD—ACE2 interactions. The simulation trajectory snapshots at 10 ps intervals (10,000 frames) were analyzed by a statistical potential to assess the probability of the RBD— ACE2 interaction (SOAP score, (<em><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></em>)), with lower values corresponding to higher probabilities and thus higher affinities. The interaction scores for COVID-19 RBD—ACE2 were comparable to those of SARS-2002, median of −1865.9 and −1929.5, respectively (<strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a></strong>). HCoV-NL63 has RBD—ACE2 interaction scores are higher than both of the SARS-CoVs (median of −941.6). MERS, which is structurally similar to COVID-19 (<strong><a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a></strong>) does not bind ACE2. MERS virus which binds dipeptidyl peptidase-4 (DPP4, also known as CD26 (<em><a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a></em>)), has RBD—ACE2 interaction scores that indicate extremely weak affinity (median of −692.6), as expected from a non-cognate receptor interaction. COVID-19 has the largest buried surface area at the interface (1204Å<sup>2</sup>), followed by the interface area for SARS-2002 (998Å<sup>2</sup>) and HCoV-NL63 (973Å<sup>2</sup>). The number of ACE2 contacting residues maintains the same order, with 30, 24, and 23 for COVID-19, SARS-2002, and HCoV-NL63, respectively (<strong><a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1C</a></strong>). The three RBDs exploit specific binding sites on ACE2 based on the analysis of the MD trajectories (<strong><a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1, C</a> and <a id="xref-fig-1-4" class="xref-fig" href="#F1">D</a></strong>; <strong>Movie S1</strong>). There is a significant overlap of ACE2 interacting residues between COVID-19 and SARS-2002 (at least 73%), while HCoV-NL63 shares only 17% and 36% of contacts with SARS-2002 and COVID-19, respectively. These findings suggest that the coronaviruses exert different interaction strategies with their cognate receptors to achieve the affinity that is required for effective cell entry.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of RBD&#x2014;ACE2 interactions based on MD trajectories. COVID-19, SARS-2002, and HCoV-NL63 are colored blue, red, and green, respectively. (A) Histograms of the RBD&#x2014;ACE2 interaction scores throughout the simulation trajectory. Darker color represents 75% of all frames. (B) The score values along the simulation trajectory, smoothed along the elapsed time. (C) Venn diagram of ACE2 interacting residues for COVID-19, SARS-2002, and HCoV-NL63. An ACE2 residue is considered as part of the interface if one of its atoms is within 4&#xC5; from any RBD atom in at least 10% of the 10,000 MD simulation frames. (D) Overlay of 50 snapshots for each of the three RBDs. The ACE2 is in surface representation (gray). The frames were aligned using the N-terminal fragment of ACE2 that contains the two helices participating in the RBDs binding." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-652534114" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of RBD&#x2014;ACE2 interactions based on MD trajectories.&lt;/span&gt; COVID-19, SARS-2002, and HCoV-NL63 are colored blue, red, and green, respectively. &lt;strong&gt;(A)&lt;/strong&gt; Histograms of the RBD&#x2014;ACE2 interaction scores throughout the simulation trajectory. Darker color represents 75% of all frames. &lt;strong&gt;(B)&lt;/strong&gt; The score values along the simulation trajectory, smoothed along the elapsed time. &lt;strong&gt;(C)&lt;/strong&gt; Venn diagram of ACE2 interacting residues for COVID-19, SARS-2002, and HCoV-NL63. An ACE2 residue is considered as part of the interface if one of its atoms is within 4&#xC5; from any RBD atom in at least 10% of the 10,000 MD simulation frames. &lt;strong&gt;(D)&lt;/strong&gt; Overlay of 50 snapshots for each of the three RBDs. The ACE2 is in surface representation (gray). The frames were aligned using the N-terminal fragment of ACE2 that contains the two helices participating in the RBDs binding.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F1.medium.gif" width="440" height="337"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F1.medium.gif" width="440" height="337"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1.</span> <span class="caption-title">Analysis of RBD—ACE2 interactions based on MD trajectories.</span><p id="p-11" class="first-child">COVID-19, SARS-2002, and HCoV-NL63 are colored blue, red, and green, respectively. <strong>(A)</strong> Histograms of the RBD—ACE2 interaction scores throughout the simulation trajectory. Darker color represents 75% of all frames. <strong>(B)</strong> The score values along the simulation trajectory, smoothed along the elapsed time. <strong>(C)</strong> Venn diagram of ACE2 interacting residues for COVID-19, SARS-2002, and HCoV-NL63. An ACE2 residue is considered as part of the interface if one of its atoms is within 4Å from any RBD atom in at least 10% of the 10,000 MD simulation frames. <strong>(D)</strong> Overlay of 50 snapshots for each of the three RBDs. The ACE2 is in surface representation (gray). The frames were aligned using the N-terminal fragment of ACE2 that contains the two helices participating in the RBDs binding.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">While the sequence identity between the RBDs of COVID-19 and SARS-2002 is 73% (<strong><a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a></strong>), we observe a significantly higher residue substitution rate at the interaction interface with the ACE2 receptor. Out of 29 RBD interface residues, only 10 residues (34%) in COVID-19 are conserved with respect to SARS-2002 (<strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a>, <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table S1</a></strong>, <strong><a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. S1</a></strong>). Similarly, only 12 residues (40%) in SARS-2002 are conserved with respect to COVID-19.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Interaction interfaces of RBD&#x2014;ACE2. A residue is considered as part of the interface if one of its atoms is within 4&#xC5; from any atom of the other partner in at least 30% of the 10,000 MD simulation frames. (A) Interface residue side-chain heavy atoms that vary between COVID-19 (blue) and SARS-2002 (red) are shown with ball-and-stick representations. ACE2 is colored gray. (B) Contacts difference plot between COVID-19 and SARS-2002. Contacts with 50% greater CF in COVID-19 RBD&#x2014;ACE2 vs. SARS-2002 RBD&#x2014;ACE2 are colored blue. Contacts with 50% greater CF in SARS-2002 RBD&#x2014;ACE2 vs. COVID-19 RBD&#x2014;ACE2 are colored red. Similar interface-residue CFs (" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-652534114" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Interaction interfaces of RBD&#x2014;ACE2.&lt;/span&gt; A residue is considered as part of the interface if one of its atoms is within 4&#xC5; from any atom of the other partner in at least 30% of the 10,000 MD simulation frames. &lt;strong&gt;(A)&lt;/strong&gt; Interface residue side-chain heavy atoms that vary between COVID-19 (blue) and SARS-2002 (red) are shown with ball-and-stick representations. ACE2 is colored gray. &lt;strong&gt;(B)&lt;/strong&gt; Contacts difference plot between COVID-19 and SARS-2002. Contacts with 50% greater CF in COVID-19 RBD&#x2014;ACE2 vs. SARS-2002 RBD&#x2014;ACE2 are colored blue. Contacts with 50% greater CF in SARS-2002 RBD&#x2014;ACE2 vs. COVID-19 RBD&#x2014;ACE2 are colored red. Similar interface-residue CFs (&lt;50%) in both RBDs are colored gray. The residue numbering is according to COVID-19 (RefSeq: YP_009724390.1). &lt;strong&gt;(C)&lt;/strong&gt; Difference plots for interface residue CFs that are in the interface for ACE2 (left) and RBD (right). &lt;strong&gt;(D)&lt;/strong&gt; Zoom on the interface contacts unique to each virus.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F2.medium.gif" width="331" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F2.medium.gif" width="331" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2:</span> <span class="caption-title">Interaction interfaces of RBD—ACE2.</span><p id="p-13" class="first-child">A residue is considered as part of the interface if one of its atoms is within 4Å from any atom of the other partner in at least 30% of the 10,000 MD simulation frames. <strong>(A)</strong> Interface residue side-chain heavy atoms that vary between COVID-19 (blue) and SARS-2002 (red) are shown with ball-and-stick representations. ACE2 is colored gray. <strong>(B)</strong> Contacts difference plot between COVID-19 and SARS-2002. Contacts with 50% greater CF in COVID-19 RBD—ACE2 vs. SARS-2002 RBD—ACE2 are colored blue. Contacts with 50% greater CF in SARS-2002 RBD—ACE2 vs. COVID-19 RBD—ACE2 are colored red. Similar interface-residue CFs (&lt;50%) in both RBDs are colored gray. The residue numbering is according to COVID-19 (RefSeq: YP_009724390.1). <strong>(C)</strong> Difference plots for interface residue CFs that are in the interface for ACE2 (left) and RBD (right). <strong>(D)</strong> Zoom on the interface contacts unique to each virus.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">To investigate these interface residues, we construct and overlay the contact maps for the RBD— ACE2 interfaces for COVID-19 and SARS-2002 (<strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2B</a></strong>). We define a residue-residue contact frequency (CF) as the fraction of MD trajectory frames in which the contact appears. Remarkably, only 8 out of the total 72 residue-residue interface contacts have comparable (&lt;50% difference) contact frequencies between the COVID-19—ACE2 and SARS-2002—ACE2 interfaces (<strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2B</a></strong>, colored gray). Furthermore, we find two interaction patches unique to COVID-19 (<strong><a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2B</a></strong>, patches 1 and 3) and another patch unique to SARS-2002 (<strong><a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2B</a></strong>, patch 2). COVID-19 has a significant and unique contact site between residues 500-505 of the RBD and residues 353-357 of ACE2 (<strong><a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2, B</a> and <a id="xref-fig-2-7" class="xref-fig" href="#F2">C</a></strong>). COVID-19 also creates a new interaction patch with the middle of the N-terminal ACE2 helix (<strong><a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2, B</a> and <a id="xref-fig-2-9" class="xref-fig" href="#F2">C</a>)</strong>, while SARS-2002 has a unique interaction patch with the end of the same helix (<strong><a id="xref-fig-2-10" class="xref-fig" href="#F2">Fig. 2, B</a> and <a id="xref-fig-2-11" class="xref-fig" href="#F2">C</a>)</strong>. The rest of the changes in the interface contact frequencies are due to the different interface loop conformations (COVID-19 residue numbers 474-498, SARS-2002 residue numbers 461-484) (<strong><a id="xref-fig-2-12" class="xref-fig" href="#F2">Fig. 2, A</a> and <a id="xref-fig-2-13" class="xref-fig" href="#F2">B</a>, <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table S1</a></strong>). COVID-19 has a significantly higher number of well-defined contact pairs compared to SARS-2002: 52 vs. 28 contacts (with 44 and 20 unique pairs, excluding the ones with similar CFs) were found for RBD—ACE2 of the COVID-19 and SARS-2002, respectively (<strong><a id="xref-fig-2-14" class="xref-fig" href="#F2">Fig. 2B</a></strong>). Results from <a id="xref-fig-2-15" class="xref-fig" href="#F2">Fig. 2</a> expose the accelerated evolution among the key anchoring residues of the RBD—ACE2 interface. This comparison raises the following question: How does SARS-2002 RBD reach an ACE2 binding affinity that is comparable to that of COVID-19 but with fewer contact pairs and a smaller interface area?</p><p id="p-15">The distribution of SOAP scores throughout the simulation trajectory has a larger fluctuation range for SARS-2002, relative to COVID-19 (<strong><a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1, A</a> and <a id="xref-fig-1-6" class="xref-fig" href="#F1">B</a></strong>; <strong><a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. S2A</a>)</strong> suggesting that SARS-2002—ACE2 interaction is fluctuating between several structural states. Moreover, analysis of contact frequencies along the entire trajectory reveals that none of the SARS-2002 contacts are maintained over 90% of the frames while COVID-19 still maintains about half of its contacts at 90% of the trajectory (<strong><a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. S2B</a></strong>).</p><p id="p-16">To investigate the dynamics of COVID-19 binding compared to SARS-2002, we calculate the root-mean-square fluctuation (RMSF) of each residue with respect to the lowest energy snapshot from their respective 100ns MD simulation trajectory. The interface region in the RBD contains two loops (loop1: residues 474-489, loop2: residues 498-505; using COVID-19 numbering, <strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3D</a></strong>) that bind to the ACE2 N-terminal helix on both of its ends. These two loops are highly flexible in the SARS-2002 RBD (<strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3, A</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">D</a></strong>). While loop1 is also fluctuating in the COVID-19 RBD, albeit much less, loop2 remains relatively rigid in the COVID-19 RBD. In addition, we find that in the COVID-19-RBD, a region centered around K417 leads to further stability relative to the corresponding region in SARS-2002. We attribute this difference to the unique interaction of COVID-19 at position K417 with the middle of the N-terminal ACE2 helix, thus serving as an anchor site to the receptor (<strong><a id="xref-fig-2-16" class="xref-fig" href="#F2">Fig. 2C</a></strong> and <strong><a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3A</a></strong>). The contribution of K417 to ACE2 binding is observed in a recent cryoEM structure of the COVID-19 spike protein bound to ACE2 (<em><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></em>). Overall, COVID-19 is more rigid compared to SARS-2002 (<strong><a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3, A</a> and <a id="xref-fig-3-6" class="xref-fig" href="#F3">D</a></strong>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dynamics of the RBD domains. The dynamics of COVID-19 and SARS2002, with a comparison to a designed SARS mutant are shown. The graphs show the root-mean-square fluctuation (RMSF) of each residue along the simulation trajectory with respect to the structure with minimum energy. The residue numbers are according to COVID-19 numbering (RefSeq: YP_009724390.1). The top graph (A) compares COVID-19 with SARS-2002. The middle graph (B) compares COVID-19 with SARS-designed. The bottom graph (C) compares SARS-2002 with SARS-designed. For all three graphs, positions highlighted in red or blue indicate those that have an increase or decrease, respectively, of 50% RMSF with respect to the comparison graph. Contact positions are written in gray, with solid vertical lines denoting the contact residues that exist in both comparison structures, and dashed vertical lines denote contact residues that exist in only one of the comparison structures." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-652534114" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Dynamics of the RBD domains.&lt;/span&gt; The dynamics of COVID-19 and SARS2002, with a comparison to a designed SARS mutant are shown. The graphs show the root-mean-square fluctuation (RMSF) of each residue along the simulation trajectory with respect to the structure with minimum energy. The residue numbers are according to COVID-19 numbering (RefSeq: YP_009724390.1). The top graph &lt;strong&gt;(A)&lt;/strong&gt; compares COVID-19 with SARS-2002. The middle graph &lt;strong&gt;(B)&lt;/strong&gt; compares COVID-19 with SARS-designed. The bottom graph &lt;strong&gt;(C)&lt;/strong&gt; compares SARS-2002 with SARS-designed. For all three graphs, positions highlighted in red or blue indicate those that have an increase or decrease, respectively, of 50% RMSF with respect to the comparison graph. Contact positions are written in gray, with solid vertical lines denoting the contact residues that exist in both comparison structures, and dashed vertical lines denote contact residues that exist in only one of the comparison structures.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F3.medium.gif" width="440" height="359"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F3.medium.gif" width="440" height="359"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3.</span> <span class="caption-title">Dynamics of the RBD domains.</span><p id="p-17" class="first-child">The dynamics of COVID-19 and SARS2002, with a comparison to a designed SARS mutant are shown. The graphs show the root-mean-square fluctuation (RMSF) of each residue along the simulation trajectory with respect to the structure with minimum energy. The residue numbers are according to COVID-19 numbering (RefSeq: YP_009724390.1). The top graph <strong>(A)</strong> compares COVID-19 with SARS-2002. The middle graph <strong>(B)</strong> compares COVID-19 with SARS-designed. The bottom graph <strong>(C)</strong> compares SARS-2002 with SARS-designed. For all three graphs, positions highlighted in red or blue indicate those that have an increase or decrease, respectively, of 50% RMSF with respect to the comparison graph. Contact positions are written in gray, with solid vertical lines denoting the contact residues that exist in both comparison structures, and dashed vertical lines denote contact residues that exist in only one of the comparison structures.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">We investigate the dynamics of a designed SARS (SARS-des) variant (<em><a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a></em>), which differs from SARS-2002 at only 2 positions: Y455F and L486F (The matched position in SARS-2002 residues are 442 and 472, respectively; <strong><a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table S1</a></strong>). The L486F mutation is of special interest for the COVID-19 RBD as well because it has this same substitution. Our MD simulation analysis reveals that the SARS-des has a substantially lower interaction scores with ACE2 (median of −2199.2, <strong><a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. S2</a></strong>), as expected for an optimized human ACE2-binding RBD design. We observed that these two mutations not only enhance the binding affinity to ACE2, but also lead to a substantial stabilization of the interaction interface. The fluctuation signatures along the RBD of SARS-des are surprisingly similar to those recorded for COVID-19 (<strong><a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3, B</a> and <a id="xref-fig-3-8" class="xref-fig" href="#F3">C</a></strong>). Thus, the switch from a flexible binding mode (for SARS-2002) to a stable one (COVID-19 and SARS-des, <strong><a id="xref-fig-3-9" class="xref-fig" href="#F3">Fig. 3B</a></strong>) highlights the remarkable capacity of the RBD to adopt alternative receptor binding strategies driven by a minimal number of amino acid substitutions. This analysis reveals the critical role of L486F (SARS-des residue F472) for stabilizing the COVID-19—ACE2 interface and a reduction in the number of states of the COVID-19 spike protein bound to an ACE2 receptor.</p><p id="p-19">Experimental affinity measurements (e.g. surface plasmon resonance, SPR) confirm the high affinity of SARS-2002 RBD—ACE2 binding, with an equilibrium dissociation constant (K<sub>D</sub>) of ~15 mM (<em><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>–<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></em>), similar to the binding affinity of ACE2 and the COVID-19 RBD (<em><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></em>). Our MD based calculation is consistent with SARS-2002 displaying a similar but slightly higher affinity relative to COVID-19 (<strong><a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1A</a></strong>, <strong><a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. S2</a></strong> and <strong><a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table S2</a></strong>). Binding affinity is achieved through a combination of interface contact optimization and protein stability (<strong><a id="xref-fig-3-10" class="xref-fig" href="#F3">Fig. 3E</a></strong>). While the RBD—ACE2 complex can be resolved at high-resolution by cryo-EM (<em><a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a></em>), MD simulations provide orthogonal information about the interaction dynamics on a nanosecond timescale. In the case of CoVs, MD simulations reveal an exceptional versatility of viral receptor binding strategies (<strong><a id="xref-fig-3-11" class="xref-fig" href="#F3">Fig. 3E</a></strong>). COVID-19 adopted a different strategy for achieving comparable affinity to SARS-2002: the interface of COVID-19 is significantly larger than that of SARS-2002 (1204Å vs. 998Å) with a remarkable number of interacting residues (ACE2: 30 vs. 24, <strong><a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig. 1C</a></strong>). In contrast, SARS-2002 is more flexible in its interaction with ACE2, interacting through fewer contacts that serve as “hot spots”. Therefore, we predict that SARS-2002 RBD neutralizing antibodies will not be effective for COVID-19. The failure of several of these antibodies to neutralize the binding of COVID-19 RBD to its receptor is consistent with our findings (<em><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a></em>). The fluctuation from high- to low-affinity conformations in SARS-2002 leads to an increased efficacy for inhibiting peptides (<em><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></em>) and high-affinity antibodies (<em><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></em>) compared to COVID-19. This implies a therapeutic challenge is attributed to the enhanced rigidity of the COVID-19 RBD relative to that of the SARS-2002.</p><p id="p-20">The geometric and physicochemical properties of RBD—ACE2 interfaces resemble those of antibody-antigen interactions. In both cases the interface benefits from long loop plasticity, bulky aromatic side chains as anchoring sites, and the stabilization of the complex by distributed electrostatic interactions (<em><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></em>). Both COVID-19 and SARS-2002 interfaces contain long flexible loops and nine aromatic residues (Tyr, Trp, Phe) in the interface with ACE2 (<strong><a id="xref-fig-2-17" class="xref-fig" href="#F2">Fig. 2A</a></strong>). Moreover, in the SARS designed variant (SARS-des (<em><a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a></em>)), the addition of an aromatic residue (L486F substitution) significantly improved the interaction scores and interface stability (<strong><a id="xref-fig-3-12" class="xref-fig" href="#F3">Fig. 3, B</a> and <a id="xref-fig-3-13" class="xref-fig" href="#F3">D</a></strong>). Our findings shed light on the accelerated evolution of spike protein binding to the ACE2 receptor similar to the rapid evolution along the antibody-antigen affinity maturation process.</p></div><div class="section" id="sec-4"><h2 class="">Funding</h2><p id="p-22">ISF 1466/18 for DS; HUJI-CIDR for ML and DS.</p></div><div class="section" id="sec-5"><h2 class="">Author contributions</h2><p id="p-23">EB, DS, and ML performed the calculations, analyzed the data, and prepared the manuscript</p></div><div class="section COI-statement" id="sec-6"><h2 class="">Competing interests</h2><p id="p-24">Authors declare no competing interests</p></div><div class="section" id="sec-7"><h2 class="">Data and materials availability</h2><p id="p-25">The MD trajectories are available from <a href="ftp://ftp.huji.ac.il">ftp.huji.ac.il</a></p></div><div class="section" id="sec-8"><h2 class="">SUPPLEMENTARY MATERIALS</h2><p id="p-26">Materials and Methods <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figs. S1</a>–<a id="xref-fig-6-1" class="xref-fig" href="#F6">S3</a></p><p id="p-27"><a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Tables S1</a>–<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">S2</a></p><p id="p-28">Movie S1 References (<em><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>–<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></em>)</p></div><div class="section" id="sec-9"><h2 class="">Materials and Methods</h2><div class="section" id="sec-10"><h3>Structural modeling</h3><p id="p-29">The structural model of the COVID-19 spike protein receptor binding domain (RBD) in complex with ACE2 was generated by comparative modeling using Modeller 9.18 (<em><a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a></em>) with the COVID-19 sequence (RefSeq: YP_009724390.1). We relied on the crystal structure of the spike protein receptor-binding domain from a SARS coronavirus designed human strain complexed with the human receptor ACE2 (PDB 3SCI, resolution 2.9Å) as a template for comparative modeling. The SARS-2002 spike protein RBD and HCoV-NL63 in complex with ACE2 were taken from PDB 2AJF (resolution 2.9Å) and 3KBH (resolution 3.3Å), respectively. Missing residues were added in MODELLER. MERS RBD structure was taken from the complex with the neutralizing antibody CDC2-C2 (PDB 6C6Z, resolution 2.1Å) and structurally aligned onto SARS-2002 RBD in complex with ACE2 receptor. The designed variant is from PDB 3SCI.</p></div><div class="section" id="sec-11"><h3>Molecular dynamic simulations</h3><p id="p-30">The MD simulations were performed with GROMACS 2020 software (<em><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></em>) using the CHARMM36m force field (<em><a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a></em>). Each of the complexes was solvated in transferable intermolecular potential with 3 points (TIP3P) water molecules and ions were added to equalize the total system charge. The steepest descent algorithm was used for initial energy minimization until the system converged at Fmax &lt; 1,000 kJ/(mol · nm). Then water and ions were allowed to equilibrate around the protein in a two-step equilibration process. The first part of equilibration was at a constant number of particles, volume, and temperature (NVT). The second part of equilibration was at a constant number of particles, pressure, and temperature (NPT). For both MD equilibration parts, positional restraints of k = 1,000 kJ/(mol nm<sup>2</sup>) were applied to heavy atoms of the protein, and the system was allowed to equilibrate at a reference temperature of 300 K, or reference pressure of 1 bar for 100 ps at a time step of 2 fs. Following equilibration, the production simulation duration was 100 nanoseconds with 2 fs time intervals. Altogether 10,000 frames were saved for the analysis at intervals of 10 ps. We superimposed several MD snapshots on the recently submitted to the PDB x-ray structure (6VW1, resolution 2.7Å) of COVID-19—ACE2 complex. The average RMSD over the interface Cα atoms is ~1Å. Interaction scores between the virus spike RBD and ACE2 were calculated for each frame of the trajectory using the SOAP statistical potential (<em><a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a></em>). In the interface contact analysis, a residue-residue contact was defined based on the inter-atomic distance, with a cutoff of 4Å.</p></div></div><div id="sec-12" class="subsection"><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multiple sequence alignment of the analyzed RBDs" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-652534114" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Multiple sequence alignment of the analyzed RBDs&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F4.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. S1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F4.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. S1.</span> <p id="p-31" class="first-child">Multiple sequence alignment of the analyzed RBDs</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Numbers of RBD residues in contact with ACE2 vs. the fraction of trajectory frames for COVID-19 (blue) and SARS-2002 (red). (B) Box plots of the SOAP interaction scores for the trajectories frames. The center point is the median score, while 50% of the scores are within the box. The whiskers extend to cover &gt;99% of the scores." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-652534114" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;(A)&lt;/strong&gt; Numbers of RBD residues in contact with ACE2 vs. the fraction of trajectory frames for COVID-19 (blue) and SARS-2002 (red). &lt;strong&gt;(B)&lt;/strong&gt; Box plots of the SOAP interaction scores for the trajectories frames. The center point is the median score, while 50% of the scores are within the box. The whiskers extend to cover &gt;99% of the scores.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F5.medium.gif" width="347" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. S2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F5.medium.gif" width="347" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. S2.</span> <p id="p-32" class="first-child"><strong>(A)</strong> Numbers of RBD residues in contact with ACE2 vs. the fraction of trajectory frames for COVID-19 (blue) and SARS-2002 (red). <strong>(B)</strong> Box plots of the SOAP interaction scores for the trajectories frames. The center point is the median score, while 50% of the scores are within the box. The whiskers extend to cover &gt;99% of the scores.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison between the x-ray structure of COVID-19&#x2014;ACE2 complex (PDB 6VW1, pink) and MD trajectory frames. The MD snapshots are colored gray and blue for ACE2 and COVID-19 RBD, respectively. The average RMSD over the interface C&#x3B1; atoms is ~1&#xC5;." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-652534114" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison between the x-ray structure of COVID-19&#x2014;ACE2 complex (PDB 6VW1, pink) and MD trajectory frames. The MD snapshots are colored gray and blue for ACE2 and COVID-19 RBD, respectively. The average RMSD over the interface C&#x3B1; atoms is ~1&#xC5;.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F6.medium.gif" width="440" height="309"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. S3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F6.medium.gif" width="440" height="309"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. S3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986398/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. S3.</span> <p id="p-33" class="first-child">Comparison between the x-ray structure of COVID-19—ACE2 complex (PDB 6VW1, pink) and MD trajectory frames. The MD snapshots are colored gray and blue for ACE2 and COVID-19 RBD, respectively. The average RMSD over the interface Cα atoms is ~1Å.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1197594/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1197594/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1197594" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S1.</span> <p id="p-34" class="first-child">RBD interface residues that have substitutions between COVID-19 and SARS-2002</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1197588/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1197588/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1197588" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S2.</span> <p id="p-35" class="first-child">RBD—ACE2 interface evaluated by several methods for analysis of protein-protein interactions</p><div class="sb-div caption-clear"></div></div></div><p id="p-36"><strong>Movie S1.</strong> Overlay of 50 random snapshots from the MD trajectories of COVID-19—ACE2, SARS-2002— ACE2, and HCoV-NL63—ACE2 complexes. For clarity only one copy of ACE2 is shown (gray), COVID-19, SARS-2002, and HCoV-NL63 are colored blue, red, and green, respectively.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-21">The authors gratefully acknowledge Barak Raveh for useful suggestions.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES AND NOTES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.1" data-doi="10.1126/science.1085952"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Rota</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Characterization of a novel coronavirus associated with severe acute respiratory syndrome</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">300</span>, <span class="cit-fpage">1394</span>–<span class="cit-lpage">1399</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1085952%26rft_id%253Dinfo%253Apmid%252F12730500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDAvNTYyNC8xMzk0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTIvMjAyMC4wMy4xMC45ODYzOTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.2" data-doi="10.3390/v12020135"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L. E.</span> <span class="cit-name-surname">Gralinski</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V. D.</span> <span class="cit-name-surname">Menachery</span></span>, <span class="cit-article-title">Return of the Coronavirus: 2019-nCoV</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">135</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D12%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv12020135%26rft_id%253Dinfo%253Apmid%252F31991541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v12020135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31991541&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.10.986398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.3" data-doi="10.1128/MMBR.69.4.635-664.2005"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Weiss</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Navas-Martin</span></span>, <span class="cit-article-title">Coronavirus pathogenesis and the emerging pathogen severe acute respiratory syndrome coronavirus</span>. <abbr class="cit-jnl-abbrev">Microbiol Mol Biol Rev</abbr> <span class="cit-vol">69</span>, <span class="cit-fpage">635</span>–<span class="cit-lpage">664</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DWeiss%26rft.auinit1%253DS.%2BR.%26rft.volume%253D69%26rft.issue%253D4%26rft.spage%253D635%26rft.epage%253D664%26rft.atitle%253DCoronavirus%2BPathogenesis%2Band%2Bthe%2BEmerging%2BPathogen%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.69.4.635-664.2005%26rft_id%253Dinfo%253Apmid%252F16339739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI2OS80LzYzNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzEyLzIwMjAuMDMuMTAuOTg2Mzk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.4"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Z. Q.</span> <span class="cit-name-surname">Zeng</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Epidemiology and clinical characteristics of human coronaviruses OC43, 229E, NL63, and HKU1: a study of hospitalized children with acute respiratory tract infection in Guangzhou, China</span>. <abbr class="cit-jnl-abbrev">Eur J Clin Microbiol Infect Dis</abbr> <span class="cit-vol">37</span>, <span class="cit-fpage">363</span>–<span class="cit-lpage">369</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur%2BJ%2BClin%2BMicrobiol%2BInfect%2BDis%26rft.volume%253D37%26rft.spage%253D363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.5"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Wu</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-fpage">1</span>–<span class="cit-lpage">5</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.6"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Tortorici</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Structural basis for human coronavirus attachment to sialic acid receptors</span>. <abbr class="cit-jnl-abbrev">Nature structural &amp; molecular biology</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">481</span>–<span class="cit-lpage">489</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.volume%253D26%26rft.spage%253D481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.7" data-doi="10.1038/nm1267"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kuba</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">A crucial role of angiotensin converting enzyme 2 (ACE2) in SARS coronavirus– induced lung injury</span>. <abbr class="cit-jnl-abbrev">Nature medicine</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">875</span>–<span class="cit-lpage">879</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmedicine%26rft.stitle%253DNat%2BMed%26rft.aulast%253DKuba%26rft.auinit1%253DK.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253D875%26rft.epage%253D879%26rft.atitle%253DA%2Bcrucial%2Brole%2Bof%2Bangiotensin%2Bconverting%2Benzyme%2B2%2B%2528ACE2%2529%2Bin%2BSARS%2Bcoronavirus-induced%2Blung%2Binjury.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm1267%26rft_id%253Dinfo%253Apmid%252F16007097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm1267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16007097&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.10.986398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230964700033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.8" data-doi="10.1016/j.bbrc.2004.05.114"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Hofmann</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Susceptibility to SARS coronavirus S protein-driven infection correlates with expression of angiotensin converting enzyme 2 and infection can be blocked by soluble receptor</span>. <abbr class="cit-jnl-abbrev">Biochemical and biophysical research communications</abbr> <span class="cit-vol">319</span>, <span class="cit-fpage">1216</span>–<span class="cit-lpage">1221</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DHofmann%26rft.auinit1%253DH.%26rft.volume%253D319%26rft.issue%253D4%26rft.spage%253D1216%26rft.epage%253D1221%26rft.atitle%253DSusceptibility%2Bto%2BSARS%2Bcoronavirus%2BS%2Bprotein-driven%2Binfection%2Bcorrelates%2Bwith%2Bexpression%2Bof%2Bangiotensin%2Bconverting%2Benzyme%2B2%2Band%2Binfection%2Bcan%2Bbe%2Bblocked%2Bby%2Bsoluble%2Breceptor.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2004.05.114%26rft_id%253Dinfo%253Apmid%252F15194496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2004.05.114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15194496&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.10.986398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222318700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.9" data-doi="10.1128/JVI.79.6.3846-3850.2005"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Mossel</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Exogenous ACE2 expression allows refractory cell lines to support severe acute respiratory syndrome coronavirus replication</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">79</span>, <span class="cit-fpage">3846</span>–<span class="cit-lpage">3850</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DMossel%26rft.auinit1%253DE.%2BC.%26rft.volume%253D79%26rft.issue%253D6%26rft.spage%253D3846%26rft.epage%253D3850%26rft.atitle%253DExogenous%2BACE2%2BExpression%2BAllows%2BRefractory%2BCell%2BLines%2BTo%2BSupport%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%2BReplication%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.6.3846-3850.2005%26rft_id%253Dinfo%253Apmid%252F15731278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc5LzYvMzg0NiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzEyLzIwMjAuMDMuMTAuOTg2Mzk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.10"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Hamming</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Tissue distribution of ACE2 protein, the functional receptor for SARS coronavirus. A first step in understanding SARS pathogenesis</span>. <abbr class="cit-jnl-abbrev">The Journal of Pathology: A Journal of the Pathological Society of Great Britain and Ireland</abbr> <span class="cit-vol">203</span>, <span class="cit-fpage">631</span>–<span class="cit-lpage">637</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BPathology%253A%2BA%2BJournal%2Bof%2Bthe%2BPathological%2BSociety%2Bof%2BGreat%2BBritain%2Band%2BIreland%26rft.volume%253D203%26rft.spage%253D631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.11"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Shang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Graham</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Baric</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-article-title">Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.12" data-doi="10.1073/pnas.0409465102"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Hofmann</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Human coronavirus NL63 employs the severe acute respiratory syndrome coronavirus receptor for cellular entry</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">102</span>, <span class="cit-fpage">7988</span>–<span class="cit-lpage">7993</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0409465102%26rft_id%253Dinfo%253Apmid%252F15897467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzIyLzc5ODgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xMi8yMDIwLjAzLjEwLjk4NjM5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.13" data-doi="10.1016/j.virol.2007.04.035"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">The S proteins of human coronavirus NL63 and severe acute respiratory syndrome coronavirus bind overlapping regions of ACE2</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">367</span>, <span class="cit-fpage">367</span>–<span class="cit-lpage">374</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D367%26rft.issue%253D2%26rft.spage%253D367%26rft.epage%253D374%26rft.atitle%253DThe%2BS%2Bproteins%2Bof%2Bhuman%2Bcoronavirus%2BNL63%2Band%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%2Bbind%2Boverlapping%2Bregions%2Bof%2BACE2.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2007.04.035%26rft_id%253Dinfo%253Apmid%252F17631932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2007.04.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17631932&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.10.986398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.14" data-doi="10.1038/cr.2013.92"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Structure of MERS-CoV spike receptor-binding domain complexed with human receptor DPP4</span>. <abbr class="cit-jnl-abbrev">Cell research</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">986</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bresearch%26rft.volume%253D23%26rft.spage%253D986%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2013.92%26rft_id%253Dinfo%253Apmid%252F23835475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2013.92&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23835475&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.10.986398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322574900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">15.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.10.986398v1.15" data-doi="10.1093/nar/gki524"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Skolnick</span></span>, <span class="cit-article-title">TM-align: a protein structure alignment algorithm based on the TM-score</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr> <span class="cit-vol">33</span>, <span class="cit-fpage">2302</span>–<span class="cit-lpage">2309</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D33%26rft.issue%253D7%26rft.spage%253D2302%26rft.epage%253D2309%26rft.atitle%253DTM-align%253A%2Ba%2Bprotein%2Bstructure%2Balignment%2Balgorithm%2Bbased%2Bon%2Bthe%2BTM-score%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki524%26rft_id%253Dinfo%253Apmid%252F15849316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki524&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15849316&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.10.986398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228778700028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.16" data-doi="10.1093/bioinformatics/btt560"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G. Q.</span> <span class="cit-name-surname">Dong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Fan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Schneidman-Duhovny</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Webb</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sali</span></span>, <span class="cit-article-title">Optimized atomic statistical potentials: assessment of protein interfaces and loops</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">3158</span>–<span class="cit-lpage">3166</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt560%26rft_id%253Dinfo%253Apmid%252F24078704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt560&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24078704&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.10.986398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.17"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Yan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Guo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Xia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-article-title">Structural basis for the recognition of the 2019-nCoV by human ACE2</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.18"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Zheng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhong</span></span>, <span class="cit-article-title">Computational characterization and design of SARS coronavirus receptor recognition and antibody neutralization</span>. <abbr class="cit-jnl-abbrev">Computational biology and chemistry</abbr> <span class="cit-vol">31</span>, <span class="cit-fpage">129</span>–<span class="cit-lpage">133</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComputational%2Bbiology%2Band%2Bchemistry%26rft.stitle%253DComput%2BBiol%2BChem%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D129%26rft.epage%253D133%26rft.atitle%253DComputational%2Bcharacterization%2Band%2Bdesign%2Bof%2BSARS%2Bcoronavirus%2Breceptor%2Brecognition%2Band%2Bantibody%2Bneutralization.%26rft_id%253Dinfo%253Apmid%252F17374510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17374510&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.10.986398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.10.986398v1.19" data-doi="10.1038/sj.emboj.7600640"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Receptor and viral determinants of SARS-coronavirus adaptation to human ACE2</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">1634</span>–<span class="cit-lpage">1643</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D24%26rft.issue%253D8%26rft.spage%253D1634%26rft.epage%253D1643%26rft.atitle%253DReceptor%2Band%2Bviral%2Bdeterminants%2Bof%2BSARS-coronavirus%2Badaptation%2Bto%2Bhuman%2BACE2%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7600640%26rft_id%253Dinfo%253Apmid%252F15791205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIyNC84LzE2MzQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xMi8yMDIwLjAzLjEwLjk4NjM5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.20"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Tian</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</span>. <abbr class="cit-jnl-abbrev">Emerg Microbes Infect</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">382</span>–<span class="cit-lpage">385</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg%2BMicrobes%2BInfect%26rft.volume%253D9%26rft.spage%253D382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.21" data-doi="10.1073/pnas.0908837106"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Peng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-article-title">Crystal structure of NL63 respiratory coronavirus receptor-binding domain complexed with its human receptor</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">106</span>, <span class="cit-fpage">19970</span>–<span class="cit-lpage">19974</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0908837106%26rft_id%253Dinfo%253Apmid%252F19901337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzQ3LzE5OTcwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTIvMjAyMC4wMy4xMC45ODYzOTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.22"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Walls</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Structure, function and antigenicity of the SARS-CoV-2 spike glycoprotein</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.23"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Wrapp</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. <abbr class="cit-jnl-abbrev">Science</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.24" data-doi="10.1016/j.antiviral.2011.12.012"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.-W.</span> <span class="cit-name-surname">Struck</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Axmann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Pfefferle</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Drosten</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Meyer</span></span>, <span class="cit-article-title">A hexapeptide of the receptor-binding domain of SARS corona virus spike protein blocks viral entry into host cells via the human receptor ACE2</span>. <abbr class="cit-jnl-abbrev">Antiviral research</abbr> <span class="cit-vol">94</span>, <span class="cit-fpage">288</span>–<span class="cit-lpage">296</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2Bresearch%26rft.stitle%253DAntiviral%2BRes%26rft.aulast%253DStruck%26rft.auinit1%253DA.%2BW.%26rft.volume%253D94%26rft.issue%253D3%26rft.spage%253D288%26rft.epage%253D296%26rft.atitle%253DA%2Bhexapeptide%2Bof%2Bthe%2Breceptor-binding%2Bdomain%2Bof%2BSARS%2Bcorona%2Bvirus%2Bspike%2Bprotein%2Bblocks%2Bviral%2Bentry%2Binto%2Bhost%2Bcells%2Bvia%2Bthe%2Bhuman%2Breceptor%2BACE2.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2011.12.012%26rft_id%253Dinfo%253Apmid%252F22265858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2011.12.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22265858&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.10.986398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.25" data-doi="10.1038/nrmicro2090"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Du</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">The spike protein of SARS-CoV--a target for vaccine and therapeutic development</span>. <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">226</span>–<span class="cit-lpage">236</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DDu%26rft.auinit1%253DL.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D226%26rft.epage%253D236%26rft.atitle%253DThe%2Bspike%2Bprotein%2Bof%2BSARS-CoV--a%2Btarget%2Bfor%2Bvaccine%2Band%2Btherapeutic%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2090%26rft_id%253Dinfo%253Apmid%252F19198616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19198616&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.10.986398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.26" data-doi="10.1093/protein/gzt027"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Kunik</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ofran</span></span>, <span class="cit-article-title">The indistinguishability of epitopes from protein surface is explained by the distinct binding preferences of each of the six antigen-binding loops</span>. <abbr class="cit-jnl-abbrev">Protein Eng Des Sel</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">599</span>–<span class="cit-lpage">609</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BEng%2BDes%2BSel%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fprotein%252Fgzt027%26rft_id%253Dinfo%253Apmid%252F23754530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/protein/gzt027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23754530&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.10.986398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325185400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.27"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Webb</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sali</span></span>, <span class="cit-article-title">Comparative protein structure modeling using MODELLER</span>. <abbr class="cit-jnl-abbrev">Current protocols in bioinformatics</abbr> <span class="cit-vol">47</span>, <span class="cit-fpage">5.6. 1</span>–<span class="cit-lpage">5.6. 32</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bprotocols%2Bin%2Bbioinformatics%26rft.volume%253D47%26rft.spage%253D561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.28" data-doi="10.1093/bioinformatics/btt055"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Pronk</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">GROMACS 4.5: a high-throughput and highly parallel open source molecular simulation toolkit</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">845</span>–<span class="cit-lpage">854</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt055%26rft_id%253Dinfo%253Apmid%252F23407358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23407358&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.10.986398.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316695700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.10.986398v1.29"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Huang</span></span> <span class="cit-etal">et al.</span>, <span class="cit-article-title">CHARMM36m: an improved force field for folded and intrinsically disordered proteins</span>. <abbr class="cit-jnl-abbrev">Nature methods</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">71</span>–<span class="cit-lpage">73</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D14%26rft.spage%253D71%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.10.986398v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.10.986398v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.10.986596v1" title="Elastohydrodynamical instabilities of active filaments, arrays and carpets analyzed using slender body theory" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.02.974246v2" title="Multi-dimensional predictions of psychotic symptoms via machine learning" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 12, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.10.986398v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.10.986398v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.10.986398v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.10.986398v1" class="active" data-icon-position="" data-hide-link-title="0">The SARS-CoV-2 exerts a distinctive strategy for interacting with the ACE2 human receptor</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1192797" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-S8zqPK8D01nhCPxQNpAmvHT5fPnIyAMVLpniGexyPJw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1192797" id="node1192797" data-pisa="biorxiv;2020.03.10.986398v1" data-pisa-master="biorxiv;2020.03.10.986398" data-seqnum="1192797" data-apath="/biorxiv/early/2020/03/12/2020.03.10.986398.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">The SARS-CoV-2 exerts a distinctive strategy for interacting with the ACE2 human receptor</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Esther S.</span> <span class="nlm-surname">Brielle</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Dina</span> <span class="nlm-surname">Schneidman-Duhovny</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Michal</span> <span class="nlm-surname">Linial</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.10.986398; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.10.986398 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.10.986398v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986398v1&amp;title=The%20SARS-CoV-2%20exerts%20a%20distinctive%20strategy%20for%20interacting%20with%20the%20ACE2%20human%20receptor" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986398v1&amp;title=The%20SARS-CoV-2%20exerts%20a%20distinctive%20strategy%20for%20interacting%20with%20the%20ACE2%20human%20receptor" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986398v1&amp;text=The%20SARS-CoV-2%20exerts%20a%20distinctive%20strategy%20for%20interacting%20with%20the%20ACE2%20human%20receptor" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986398v1&amp;title=The%20SARS-CoV-2%20exerts%20a%20distinctive%20strategy%20for%20interacting%20with%20the%20ACE2%20human%20receptor" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986398v1&amp;t=The%20SARS-CoV-2%20exerts%20a%20distinctive%20strategy%20for%20interacting%20with%20the%20ACE2%20human%20receptor" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986398v1&amp;title=The%20SARS-CoV-2%20exerts%20a%20distinctive%20strategy%20for%20interacting%20with%20the%20ACE2%20human%20receptor" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986398v1&amp;title=The%20SARS-CoV-2%20exerts%20a%20distinctive%20strategy%20for%20interacting%20with%20the%20ACE2%20human%20receptor" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1192797" id="node1192797--2" data-pisa="biorxiv;2020.03.10.986398v1" data-pisa-master="biorxiv;2020.03.10.986398" data-seqnum="1192797" data-apath="/biorxiv/early/2020/03/12/2020.03.10.986398.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">The SARS-CoV-2 exerts a distinctive strategy for interacting with the ACE2 human receptor</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Esther S.</span> <span class="nlm-surname">Brielle</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Dina</span> <span class="nlm-surname">Schneidman-Duhovny</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Michal</span> <span class="nlm-surname">Linial</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.10.986398; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.10.986398 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=The%20SARS-CoV-2%20exerts%20a%20distinctive%20strategy%20for%20interacting%20with%20the%20ACE2%20human%20receptor&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Brielle%2C+Esther+S.&amp;rft.au=Schneidman-Duhovny%2C+Dina&amp;rft.au=Linial%2C+Michal"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1192797/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1192797/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1192797/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1192797/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1192797/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1192797/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1192797/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1192797/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1192797/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1192797/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1192797/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1192797/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986398v1&amp;count=horizontal&amp;via=&amp;text=The%20SARS-CoV-2%20exerts%20a%20distinctive%20strategy%20for%20interacting%20with%20the%20ACE2%20human%20receptor&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986398v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.10.986398v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.10.986398v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/molecular-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Molecular Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
