<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/04/01/2020.03.30.016931.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.30.016931v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Prediction of SARS-CoV-2 epitopes across 9360 HLA class I alleles"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.30.016931"/>
  <meta name="DC.Date" content="2020-04-01"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Elucidating antiviral CD8 T lymphocyte responses to SARS-CoV-2 may shed light on the heterogeneity of clinical outcomes and inform vaccine or therapeutic approaches. To facilitate the evaluation of antiviral CD8 T cell responses to SARS-CoV-2, we generated a publicly accessible database of epitopes predicted to bind any class I HLA protein across the entire SARS-CoV-2 proteome. While a subset of epitopes from earlier betacoronaviruses, such as SARS-CoV (SARS), have been validated experimentally, validation systems are often biased toward specific HLA haplotypes (notably HLA-A*02:01) that only account for a fraction of the haplotypes of individuals affected by the SARS-CoV-2 pandemic. To enable evaluation of epitopes across individuals with a variety of HLA haplotypes, we computed the predicted binding affinities between 9-mer peptides derived from the annotated SARS-CoV-2 peptidome across 9,360 MHC class I HLA-A, -B, and -C alleles. There were 6,748 unique combinations of peptides and HLA alleles (pMHCs) with a predicted binding affinity of less than 500nM, including 1,103 unique peptides and 1,022 HLA alleles, spanning 11 annotated superfamilies. These peptides were derived from all 11 proteins spanning the SARS-CoV-2 peptidome, including peptides that have previously been validated experimentally. We also show evidence that these previously validated epitopes may be relevant in other HLA contexts. This complete dataset is available publicly: gs://pici-covid19-data-resources/mhci/peptide_predictions."/>
  <meta name="DC.Contributor" content="Katie M. Campbell"/>
  <meta name="DC.Contributor" content="Gabriela Steiner"/>
  <meta name="DC.Contributor" content="Daniel K. Wells"/>
  <meta name="DC.Contributor" content="Antoni Ribas"/>
  <meta name="DC.Contributor" content="Anusha Kalbasi"/>
  <meta name="article:published_time" content="2020-04-01"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Prediction of SARS-CoV-2 epitopes across 9360 HLA class I alleles"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Elucidating antiviral CD8 T lymphocyte responses to SARS-CoV-2 may shed light on the heterogeneity of clinical outcomes and inform vaccine or therapeutic approaches. To facilitate the evaluation of antiviral CD8 T cell responses to SARS-CoV-2, we generated a publicly accessible database of epitopes predicted to bind any class I HLA protein across the entire SARS-CoV-2 proteome. While a subset of epitopes from earlier betacoronaviruses, such as SARS-CoV (SARS), have been validated experimentally, validation systems are often biased toward specific HLA haplotypes (notably HLA-A*02:01) that only account for a fraction of the haplotypes of individuals affected by the SARS-CoV-2 pandemic. To enable evaluation of epitopes across individuals with a variety of HLA haplotypes, we computed the predicted binding affinities between 9-mer peptides derived from the annotated SARS-CoV-2 peptidome across 9,360 MHC class I HLA-A, -B, and -C alleles. There were 6,748 unique combinations of peptides and HLA alleles (pMHCs) with a predicted binding affinity of less than 500nM, including 1,103 unique peptides and 1,022 HLA alleles, spanning 11 annotated superfamilies. These peptides were derived from all 11 proteins spanning the SARS-CoV-2 peptidome, including peptides that have previously been validated experimentally. We also show evidence that these previously validated epitopes may be relevant in other HLA contexts. This complete dataset is available publicly: gs://pici-covid19-data-resources/mhci/peptide_predictions.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.30.016931v1"/>
  <meta name="citation_id" content="2020.03.30.016931v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.30.016931v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.30.016931v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.30.016931v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/01/2020.03.30.016931.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.30.016931"/>
  <meta name="citation_num_pages" content="12"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.30.016931"/>
  <meta name="citation_author" content="Katie M. Campbell"/>
  <meta name="citation_author_institution" content="Department of Medicine, Division of Hematology-Oncology, University of California, Los Angeles (UCLA)"/>
  <meta name="citation_author_email" content="KatieCampbell@mednet.ucla.edu"/>
  <meta name="citation_author_email" content="AnushaKalbasi@mednet.ucla.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6491-4432"/>
  <meta name="citation_author" content="Gabriela Steiner"/>
  <meta name="citation_author_institution" content="Parker Institute for Cancer Immunotherapy"/>
  <meta name="citation_author" content="Daniel K. Wells"/>
  <meta name="citation_author_institution" content="Parker Institute for Cancer Immunotherapy"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2463-3139"/>
  <meta name="citation_author" content="Antoni Ribas"/>
  <meta name="citation_author_institution" content="Department of Medicine, Division of Hematology-Oncology, University of California, Los Angeles (UCLA)"/>
  <meta name="citation_author_institution" content="Parker Institute for Cancer Immunotherapy"/>
  <meta name="citation_author_institution" content="Department Surgery, Division of Surgical Oncology, University of California"/>
  <meta name="citation_author_institution" content="Jonsson Comprehensive Cancer Center"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3669-8458"/>
  <meta name="citation_author" content="Anusha Kalbasi"/>
  <meta name="citation_author_institution" content="Department of Radiation Oncology, UCLA"/>
  <meta name="citation_author_institution" content="Department Surgery, Division of Surgical Oncology, University of California"/>
  <meta name="citation_author_institution" content="Jonsson Comprehensive Cancer Center"/>
  <meta name="citation_author_email" content="KatieCampbell@mednet.ucla.edu"/>
  <meta name="citation_author_email" content="AnushaKalbasi@mednet.ucla.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8692-1457"/>
  <meta name="citation_reference" content="X. Yang et al., Clinical course and outcomes of critically ill patients with SARS-CoV-2 pneumonia in Wuhan, China: a single-centered, retrospective, observational study. Lancet Respir Med, (2020)."/>
  <meta name="citation_reference" content="D. Wang et al., Clinical Characteristics of 138 Hospitalized Patients With 2019 Novel Coronavirus-Infected Pneumonia in Wuhan, China. Jama, (2020)."/>
  <meta name="citation_reference" content="E. J. Wherry, R. Ahmed, Memory CD8 T-cell differentiation during viral infection. Journal of virology 78, 5535–5545 (2004)."/>
  <meta name="citation_reference" content="R. Channappanavar, S. Perlman, Pathogenic human coronavirus infections: causes and consequences of cytokine storm and immunopathology. Semin Immunopathol 39, 529–539 (2017)."/>
  <meta name="citation_reference" content="A. V. Hill, Immunogenetics and genomics. Lancet 357, 2037–2041 (2001)."/>
  <meta name="citation_reference" content="J. Hundal et al., pVACtools: A Computational Toolkit to Identify and Visualize Cancer Neoantigens. Cancer Immunol Res 8, 409–420 (2020)."/>
  <meta name="citation_reference" content="J. Sidney, B. Peters, N. Frahm, C. Brander, A. Sette, HLA class I supertypes: a revised and updated classification. BMC Immunol 9, 1 (2008)."/>
  <meta name="citation_reference" content="R. Vita et al., The Immune Epitope Database (IEDB): 2018 update. Nucleic Acids Res 47, D339–d343 (2019)."/>
  <meta name="citation_reference" content="A. Grifoni et al., A Sequence Homology and Bioinformatic Approach Can Predict Candidate Targets for Immune Responses to SARS-CoV-2. Cell Host Microbe, (2020)."/>
  <meta name="citation_reference" content="S. F. Ahmed, A. A. Quadeer, M. R. McKay, Preliminary Identification of Potential Vaccine Targets for the COVID-19 Coronavirus (SARS-CoV-2) Based on SARS-CoV Immunological Studies. Viruses 12, (2020)."/>
  <meta name="citation_reference" content="N. P. Croft et al., Most viral peptides displayed by class I MHC on infected cells are immunogenic. Proc Natl Acad Sci U S A 116, 3112–3117 (2019)."/>
  <meta name="citation_reference" content="M. Andreatta et al., An automated benchmarking platform for MHC class II binding prediction methods. Bioinformatics 34, 1522–1528 (2018)."/>
  <meta name="citation_reference" content="Y. P. Tsao et al., HLA-A*0201 T-cell epitopes in severe acute respiratory syndrome (SARS) coronavirus nucleocapsid and spike proteins. Biochem Biophys Res Commun 344, 63–71 (2006)."/>
  <meta name="citation_reference" content="S. Ohno et al., Synthetic peptides coupled to the surface of liposomes effectively induce SARS coronavirus-specific cytotoxic T lymphocytes and viral clearance in HLA-A*0201 transgenic mice. Vaccine 27, 3912–3920 (2009)."/>
  <meta name="citation_reference" content="C. Sylvester-Hvid et al., SARS CTL vaccine candidates; HLA supertype-, genome-wide scanning and biochemical validation. Tissue Antigens 63, 395–400 (2004)."/>
  <meta name="citation_reference" content="M. Zhou et al., Screening and identification of severe acute respiratory syndrome-associated coronavirus-specific CTL epitopes. J Immunol 177, 2138–2145 (2006)."/>
  <meta name="citation_reference" content="Y. K. Cheung, S. C. Cheng, F. W. Sin, K. T. Chan, Y. Xie, Induction of T-cell response by a DNA vaccine encoding a novel HLA-A*0201 severe acute respiratory syndrome coronavirus epitope. Vaccine 25, 6070–6077 (2007)."/>
  <meta name="citation_reference" content="L. Rivino et al., Defining CD8+ T cell determinants during human viral infection in populations of Asian ethnicity. J Immunol 191, 4010–4019 (2013)."/>
  <meta name="citation_reference" content="C. X. Chang et al., Conditional ligands for Asian HLA variants facilitate the definition of CD8+ T-cell responses in acute and chronic viral diseases. Eur J Immunol 43, 1109–1120 (2013)."/>
  <meta name="citation_reference" content="Y. D. Wang et al., T-cell epitopes in severe acute respiratory syndrome (SARS) coronavirus spike protein elicit a specific T-cell immune response in patients who recover from SARS. Journal of virology 78, 5612–5618 (2004)."/>
  <meta name="citation_reference" content="J. Liu et al., The membrane protein of severe acute respiratory syndrome coronavirus acts as a dominant immunogen revealed by a clustering region of novel functionally and structurally defined cytotoxic T-lymphocyte epitopes. J Infect Dis 202, 1171–1180 (2010)."/>
  <meta name="citation_reference" content="Y. Lv, Z. Ruan, L. Wang, B. Ni, Y. Wu, Identification of a novel conserved HLA-A*0201-restricted epitope from the spike protein of SARS-CoV. BMC Immunol 10, 61 (2009)."/>
  <meta name="citation_reference" content="B. Wang et al., Identification of an HLA-A*0201-restricted CD8+ T-cell epitope SSp-1 of SARS-CoV spike protein. Blood 104, 200–206 (2004)."/>
  <meta name="citation_reference" content="H. L. Oh et al., Engineering T cells specific for a dominant severe acute respiratory syndrome coronavirus CD8 T cell epitope. Journal of virology 85, 10464–10471 (2011)."/>
  <meta name="citation_reference" content="S. Kohyama et al., Efficient induction of cytotoxic T lymphocytes specific for severe acute respiratory syndrome (SARS)-associated coronavirus by immunization with surface-linked liposomal peptides derived from a non-structural polyprotein 1a. Antiviral Res 84, 168–177 (2009)."/>
  <meta name="twitter:title" content="Prediction of SARS-CoV-2 epitopes across 9360 HLA class I alleles"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Elucidating antiviral CD8 T lymphocyte responses to SARS-CoV-2 may shed light on the heterogeneity of clinical outcomes and inform vaccine or therapeutic approaches. To facilitate the evaluation of antiviral CD8 T cell responses to SARS-CoV-2, we generated a publicly accessible database of epitopes predicted to bind any class I HLA protein across the entire SARS-CoV-2 proteome. While a subset of epitopes from earlier betacoronaviruses, such as SARS-CoV (SARS), have been validated experimentally, validation systems are often biased toward specific HLA haplotypes (notably HLA-A*02:01) that only account for a fraction of the haplotypes of individuals affected by the SARS-CoV-2 pandemic. To enable evaluation of epitopes across individuals with a variety of HLA haplotypes, we computed the predicted binding affinities between 9-mer peptides derived from the annotated SARS-CoV-2 peptidome across 9,360 MHC class I HLA-A, -B, and -C alleles. There were 6,748 unique combinations of peptides and HLA alleles (pMHCs) with a predicted binding affinity of less than 500nM, including 1,103 unique peptides and 1,022 HLA alleles, spanning 11 annotated superfamilies. These peptides were derived from all 11 proteins spanning the SARS-CoV-2 peptidome, including peptides that have previously been validated experimentally. We also show evidence that these previously validated epitopes may be relevant in other HLA contexts. This complete dataset is available publicly: gs://pici-covid19-data-resources/mhci/peptide_predictions."/>
  <meta name="og-title" property="og:title" content="Prediction of SARS-CoV-2 epitopes across 9360 HLA class I alleles"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.30.016931v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Elucidating antiviral CD8 T lymphocyte responses to SARS-CoV-2 may shed light on the heterogeneity of clinical outcomes and inform vaccine or therapeutic approaches. To facilitate the evaluation of antiviral CD8 T cell responses to SARS-CoV-2, we generated a publicly accessible database of epitopes predicted to bind any class I HLA protein across the entire SARS-CoV-2 proteome. While a subset of epitopes from earlier betacoronaviruses, such as SARS-CoV (SARS), have been validated experimentally, validation systems are often biased toward specific HLA haplotypes (notably HLA-A*02:01) that only account for a fraction of the haplotypes of individuals affected by the SARS-CoV-2 pandemic. To enable evaluation of epitopes across individuals with a variety of HLA haplotypes, we computed the predicted binding affinities between 9-mer peptides derived from the annotated SARS-CoV-2 peptidome across 9,360 MHC class I HLA-A, -B, and -C alleles. There were 6,748 unique combinations of peptides and HLA alleles (pMHCs) with a predicted binding affinity of less than 500nM, including 1,103 unique peptides and 1,022 HLA alleles, spanning 11 annotated superfamilies. These peptides were derived from all 11 proteins spanning the SARS-CoV-2 peptidome, including peptides that have previously been validated experimentally. We also show evidence that these previously validated epitopes may be relevant in other HLA contexts. This complete dataset is available publicly: gs://pici-covid19-data-resources/mhci/peptide_predictions."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-04-01"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Prediction of SARS-CoV-2 epitopes across 9360 HLA class I alleles | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1221477 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">Elucidating antiviral CD8 T lymphocyte responses to SARS-CoV-2 may shed light on the heterogeneity of clinical outcomes and inform vaccine or therapeutic approaches. To facilitate the evaluation of antiviral CD8 T cell responses to SARS-CoV-2, we generated a publicly accessible database of epitopes predicted to bind any class I HLA protein across the entire SARS-CoV-2 proteome. While a subset of epitopes from earlier betacoronaviruses, such as SARS-CoV (SARS), have been validated experimentally, validation systems are often biased toward specific HLA haplotypes (notably HLA-A*02:01) that only account for a fraction of the haplotypes of individuals affected by the SARS-CoV-2 pandemic. To enable evaluation of epitopes across individuals with a variety of HLA haplotypes, we computed the predicted binding affinities between 9-mer peptides derived from the annotated SARS-CoV-2 peptidome across 9,360 MHC class I HLA-A, -B, and -C alleles. There were 6,748 unique combinations of peptides and HLA alleles (pMHCs) with a predicted binding affinity of less than 500nM, including 1,103 unique peptides and 1,022 HLA alleles, spanning 11 annotated superfamilies. These peptides were derived from all 11 proteins spanning the SARS-CoV-2 peptidome, including peptides that have previously been validated experimentally. We also show evidence that these previously validated epitopes may be relevant in other HLA contexts. This complete dataset is available publicly: gs://pici-covid19-data-resources/mhci/peptide_predictions.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-4">In a subset of patients, SARS-CoV-2 can result in COVID-19, which can be a deadly disease with hallmarks of acute respiratory distress syndrome (ARDS)(
     <em>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
     </em>). Other patients harboring SARS-CoV-2 can have minimal or no symptoms. While clinical factors such as older age and underlying medical conditions have been reported as potential risk factors for more severe disease(
     <em>
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </em>), the heterogeneity of clinical response to this viral agent is otherwise poorly understood.
    </p>
    <p id="p-5">Cytotoxic CD8
     <sup>+</sup> T lymphocyte responses are crucial for initial viral clearance and development of immunologic memory(
     <em>
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>
     </em>). CD8 T cell responses are a plausible contributor to immunopathology following viral infection(
     <em>
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
     </em>). Excessive cytotoxic CD8 T cell responses and resultant cytokine release by these and other cells may contribute to ARDS, while less effective CD8 T cell responses may allow progression of viral pathology.
    </p>
    <p id="p-6">The heterogeneity in CD8 T cell responses to SARS-CoV-2 may in part be related to the capacity to recognize the viral antigens in the context of class I major histocompatibility complex (MHC) proteins. Indeed, genetic susceptibilities to viral infection have been tied to human leukocyte antigen (HLA) haplotypes(
     <em>
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
     </em>). Meanwhile, functional differences in antigen-specific CD8 T cell responses in symptomatic and asymptomatic patients may also contribute to the biology of at-risk populations. Together, this knowledge may aid in designing and monitoring the impact of antiviral vaccines.
    </p>
    <p id="p-7">As a resource to explore antigen and HLA specific CTL responses, we used a computational approach previously developed to define neoantigen-specific CD8 T cell responses in patients with cancer(
     <em>
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>
     </em>). With this approach, we predicted all potential peptides that bind a globally representative set of MHC I alleles across the entire SARS-CoV-2 proteome. We have made this resource free and available to the public, and the resource will be updated through 
     <a href="https://covid-19.parkerici.org/docs/data_sets/mhc_peptidome.html">https://covid-19.parkerici.org/docs/data_sets/mhc_peptidome.html</a> as additional data becomes available pertaining to the SARS-CoV-2 genome, proteome, and epitome.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Results</h2>
    <p id="p-8">We deployed peptide binding predictions across 9-mer peptides derived from 1,075 protein sequences spanning 166 SARS-CoV-2 genotypes for 9,360 class I HLA alleles (2,987 HLA-A; 3,707 HLA-B; 2,666 HLA-C). Peptide binding predictions were filtered to those with a high-binding affinity (&lt;;500nM) to identify putative class I HLA antigens (
     <strong>
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>
     </strong>). There were 6,748 unique combinations of peptides and HLA alleles (pMHCs) with a predicted binding affinity of less than 500nM (
     <strong>Supplementary Table 3</strong>), including 1,103 unique peptides and 1,022 HLA alleles (295 HLA-A, 614 HLA-B, and 113 HLA-C).
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/01/2020.03.30.016931/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Peptide binding predictions for SARS-CoV-2 A. Overview of the analysis strategy. Using 9-mer peptides across the viral proteome, binding predictions were generated with nine different prediction algorithms (See Methods). B. Peptide-MHC complexes (pMHCs) were filtered to those with binding affinities less than 500nM. This histogram shows the distribution of the number of pMHCs (x-axis) corresponding to each peptide. C. Number of peptides (bars; top x-axis) derived from each protein (y-axis). The number of peptides was normalized to the length of the protein, and is indicated by a single point (points; bottom x-axis). D. Bar charts showing the number of alleles (y-axis) that have a high-binding peptide at the corresponding protein position (x-axis) for six viral proteins. Due to differences in amino acid sequences across annotated proteins, there are 43 peptides not shown. See Supplementary Figure 1 for remaining proteins." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1642075215" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Peptide binding predictions for SARS-CoV-2&lt;;/span&gt;; A. Overview of the analysis strategy. Using 9-mer peptides across the viral proteome, binding predictions were generated with nine different prediction algorithms (See Methods). B. Peptide-MHC complexes (pMHCs) were filtered to those with binding affinities less than 500nM. This histogram shows the distribution of the number of pMHCs (x-axis) corresponding to each peptide. C. Number of peptides (bars; top x-axis) derived from each protein (y-axis). The number of peptides was normalized to the length of the protein, and is indicated by a single point (points; bottom x-axis). D. Bar charts showing the number of alleles (y-axis) that have a high-binding peptide at the corresponding protein position (x-axis) for six viral proteins. Due to differences in amino acid sequences across annotated proteins, there are 43 peptides not shown. See &lt;;strong&gt;;Supplementary Figure 1&lt;;/strong&gt;; for remaining proteins.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/01/2020.03.30.016931/F1.medium.gif" width="440" height="389"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/01/2020.03.30.016931/F1.medium.gif" width="440" height="389"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/01/2020.03.30.016931/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/01/2020.03.30.016931/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1.</span>
      <span class="caption-title">Peptide binding predictions for SARS-CoV-2</span>
      <p id="p-9" class="first-child">A. Overview of the analysis strategy. Using 9-mer peptides across the viral proteome, binding predictions were generated with nine different prediction algorithms (See Methods). B. Peptide-MHC complexes (pMHCs) were filtered to those with binding affinities less than 500nM. This histogram shows the distribution of the number of pMHCs (x-axis) corresponding to each peptide. C. Number of peptides (bars; top x-axis) derived from each protein (y-axis). The number of peptides was normalized to the length of the protein, and is indicated by a single point (points; bottom x-axis). D. Bar charts showing the number of alleles (y-axis) that have a high-binding peptide at the corresponding protein position (x-axis) for six viral proteins. Due to differences in amino acid sequences across annotated proteins, there are 43 peptides not shown. See 
       <strong>Supplementary Figure 1</strong> for remaining proteins.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-10">In order to better understand the class I antigenic profile of the SARS-CoV-2 proteome, we focused on pMHCs with a binding affinity of less than 500nM (n=6,748). Of the 1,103 unique peptides comprising these pMHCs, there were between 1 and 55 (median 3) alleles that could bind each peptide (
     <strong>
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>
     </strong>), indicating that multiple HLA types could present the same antigens. Antigens were annotated by their corresponding protein (
     <strong>Methods</strong>) to identify which viral proteins may be responsible for class I recognition. There were between 12 and 684 (median 31) high-binding peptides observed in each of the 11 viral proteins (Orf1ab, Surface/Spike Glycoprotein, Orf3a, Envelope (Orf4; E), Membrane Glycoprotein, Orf6, Orf7a, Orf7b, Orf8, Nucleocapsid (Orf9), and Orf10; 
     <strong>
      <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>, Supplementary Figure 1
     </strong>). These predicted peptides were distributed throughout each protein (
     <strong>
      <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1D</a>
     </strong>). Orf1ab was the largest protein and had the highest number of peptides (n=684), followed by the Surface/Spike Glycoprotein (n=109) and the Nucleocapsid (Orf9; n=60). When normalized by amino acid length, the largest proteins had the lowest density of peptides.
    </p>
    <p id="p-11">We were additionally interested in the diversity and number of the HLA alleles that could bind each predicted SARS-CoV-2 peptide. HLA alleles were annotated by their gene and structural superfamily (
     <strong>Supplementary Table 4</strong>)(
     <em>
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>
     </em>). In addition, we specifically highlighted the HLA-A*02 allele group, due to the prevalence of this allele in the population and its level of study in model systems. There were more peptides with predicted high binding affinities for HLA-A (n=438 unique peptides) and -B (n=552) alleles, compared to -C (n=243) (
     <strong>Supplementary Figure 3</strong>). HLA alleles within superfamilies tended to bind similar numbers of peptides (left bar plot, 
     <strong>
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, Supplementary Figure 3
     </strong>) and the same sets of peptides (heatmap clusters, 
     <strong>
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>
     </strong>). However, overall, a diverse set of HLA alleles are predicted to bind antigens spanning the viral proteome (
     <strong>
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>
     </strong>). Notably, our predicted peptides with high-affinity binding were all included in the homologous sequences between SARS-CoV and SARS-CoV-2.
    </p>
    <div id="F2" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/01/2020.03.30.016931/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="SARS-CoV-2 Class I epitome This figure displays all combinations (pMHCs; n=6,493) across peptides (x-axis) properly assigned to protein annotations (bottom color bar) with a binding affinity to HLA alleles (y-axis) of less than 500nM. The heatmap shows the binding affinity (fill color, nM), and pairs without high-binding epitopes are black. The bar charts show the number of peptides corresponding to each peptide (top) or the number of peptides corresponding to each allele (side left). Each allele is annotated (side color bars) by its gene (HLA-A, -B, -C), whether it is part of the A02 allele group, and superfamily (if available)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1642075215" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;SARS-CoV-2 Class I epitome&lt;;/span&gt;; This figure displays all combinations (pMHCs; n=6,493) across peptides (x-axis) properly assigned to protein annotations (bottom color bar) with a binding affinity to HLA alleles (y-axis) of less than 500nM. The heatmap shows the binding affinity (fill color, nM), and pairs without high-binding epitopes are black. The bar charts show the number of peptides corresponding to each peptide (top) or the number of peptides corresponding to each allele (side left). Each allele is annotated (side color bars) by its gene (HLA-A, -B, -C), whether it is part of the A02 allele group, and superfamily (if available).&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/01/2020.03.30.016931/F2.medium.gif" width="440" height="272"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/01/2020.03.30.016931/F2.medium.gif" width="440" height="272"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/01/2020.03.30.016931/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/01/2020.03.30.016931/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 2.</span>
      <span class="caption-title">SARS-CoV-2 Class I epitome</span>
      <p id="p-12" class="first-child">This figure displays all combinations (pMHCs; n=6,493) across peptides (x-axis) properly assigned to protein annotations (bottom color bar) with a binding affinity to HLA alleles (y-axis) of less than 500nM. The heatmap shows the binding affinity (fill color, nM), and pairs without high-binding epitopes are black. The bar charts show the number of peptides corresponding to each peptide (top) or the number of peptides corresponding to each allele (side left). Each allele is annotated (side color bars) by its gene (HLA-A, -B, -C), whether it is part of the A02 allele group, and superfamily (if available).</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-13">Due to the partial homology between SARS-CoV-2 with SARS-CoV, we evaluated whether any experimentally validated epitopes from SARS-CoV were identified in our study (
     <strong>Supplementary Table 5</strong>). Using the Immune Epitope Database (IEDB) (
     <em>
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
     </em>), we enumerated 30 HLA class I 9-mer epitopes that were validated by T cell assays (cytokine release assays, tetramer staining, and/or cytotoxicity assays) in clinical samples (including those derived from patients that had recovered from a prior SARS-CoV infection and normal donors) and humanized mouse models (e.g. HLA-A*0201 transgenic mice). Of these 30 epitopes, 10 were also seen in our filtered results (
     <strong>
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>
     </strong>); there were two additional 9-mers that were contained within previously validated 10-mers. These included epitopes corresponding to the HLA-A*02 (n=20), HLA-A*11 (n=1), and HLA-B*40 (n=1) allele groups. Of note, of the 12 predicted epitopes with corresponding experimental validation, six were also predicted as high-binding antigens across other allele groups.
    </p>
    <div id="T1" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1229473/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1229473/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1229473" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table 1.</span>
      <span class="caption-title">Previously validated SARS-CoV peptides.</span>
      <p id="p-14" class="first-child">Allele groups highlighted in 
       <strong>bold</strong> under “Predicted HLA Alleles” indicates that the allele group matches the validated HLA Allele group (Column 1).
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-15">Grifoni, et al. recently used the homology between the SARS-CoV and SARS-CoV-2 proteomes and the existing annotated epitopes of SARS-CoV from the IEDB (
     <em>
      <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>
     </em>) to infer T cell epitopes derived from SARS-CoV-2(
     <em>
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>
     </em>). Our prediction identified 10 of the 25 dominant class 1 HLA 9-mer epitopes from that study. In addition, our study reports 1,088 additional predicted peptides that may serve as a resource for experimental validation.
    </p>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">Discussion</h2>
    <p id="p-16">Our study was designed to evaluate the predicted MHC class I epitope landscape with respect to the SARS-CoV-2 viral proteome across a globally representative set of HLA alleles. We aimed to establish a resource for the scientific community, and have made the entirety of these data publicly available. This work expands upon recent studies that inferred the epitope landscape of SARS-CoV-2 based on its partial homology with SARS-CoV, for which there is an existing annotated epitope landscape(
     <em>
      <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>, 
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>
     </em>).
    </p>
    <p id="p-17">Our analysis was restricted to pMHC complexes with predicted binding affinities of less than 500nM. Subsequent analysis did not treat the predicted binding affinities as a continuous variable (i.e. predicted values of 5nM and 400nM were treated similarly in the remaining analysis). The pMHC binding affinities presented here are computationally defined. In the absence of experimental validation, we did not try to further delineate the association between HLA diversity and the predicted binding affinity. Furthermore, utilizing a threshold of 500nM may result in underestimating the number of alleles associated with the predicted antigenic peptides. Thus, this analysis could be expanded to consider pMHCs with binding affinities slightly greater than 500nM.</p>
    <p id="p-18">We also compared our filtered results to experimental efforts to validate viral epitopes of the homologous virus SARS-CoV over the last two decades. Our prediction successfully identified 12 of 32 previously validated HLA class I peptides across the SARS-CoV viral peptidome. Moreover, we predict that there are additional HLA types that may successfully bind and present these antigens.</p>
    <p id="p-19">This dataset and analysis have limitations. Our search was restricted to perfectly matched amino acid sequences, and there may be peptides associated with SARS-CoV-2 that have mismatches, compared to SARS-CoV. Our search was also limited to 9-mers, which represents most but not all reported HLA class I binding peptides. In addition, we did not further assess peptide sequences in pMHCs with affinities greater than 500nM and refined searches will be necessary to completely assess the homology between the SARS-CoV and SARS-CoV-2 epitomes. Our data also does not account for either the quantity or timing of viral protein expression in a host cell, both of which can impact the immunogenicity of predicted epitopes(
     <em>
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>
     </em>).
    </p>
    <p id="p-20">CD4 T cell responses are also an important aspect of antiviral clearance and memory, but these were not included owing to the limitations in predicting human HLA class II epitopes(
     <em>
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>
     </em>). Lastly, the HLA binding prediction algorithms used in this study are trained based on antigen binding affinities available in the IEDB. Thus, our results are biased toward previously studied epitopes and HLA haplotypes and unlikely to be exhaustive, particularly in our list of filtered high-binding pMHC. As a result, we have deposited data for all predicted epitopes, even those with lower pMHC binding affinity.
    </p>
    <p id="p-21">With the ongoing SARS-CoV-2 pandemic, there are worldwide efforts to collect blood and tissue samples from persons who are asymptomatic, symptomatic with limited or serious complications, and convalescent. It is possible that patients who have the most severe forms of the disease, including ARDS, may have an increased functionality of CD8 T cells recognizing specific antigens presented by one of their HLAs, which could be specifically studied using the predicted epitopes described herein. In addition, preventive vaccines may require monitoring not only the induction of an antibody response, which is usually delayed, but also an earlier CD8 T cell response to specific antigens within the context of the patient’s HLA subtypes. Given the global reach of the SARS-CoV-2 pandemic, this database of predicted HLA class I binding peptides may serve as a guide to identify and monitor phenotypic, functional and kinetic responses of putative SARS-CoV-2-specific CD8
     <sup>+</sup> T cells across patients with a broad range of HLA haplotypes internationally.
    </p>
   </div>
   <div class="section" id="sec-5">
    <h2 class="">Declaration of Interests</h2>
    <p id="p-23">KMC is a shareholder in Geneoscopy LLC. DKW is a founder, equity holder and receives consulting fees from Immunai. AR has received honoraria from consulting with Amgen, Bristol-Myers Squibb, Chugai, Genentech, Merck, Novartis, Roche and Sanofi, is or has been a member of the scientific advisory board and holds stock in Advaxis, Apricity, Arcus Biosciences, Bioncotech Therapeutics, Compugen, CytomX, Five Prime, FLX-Bio, ImaginAb, Isoplexis, Kite-Gilead, Lutris Pharma, Merus, PACT Pharma, Rgenix and Tango Therapeutics, has received research funding from Agilent and from Bristol-Myers Squibb through Stand Up to Cancer (SU2C).</p>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">Methods</h2>
    <div class="section" id="sec-7">
     <h3>SARS-CoV-2 data acquisition and protein annotation</h3>
     <p id="p-24">The protein fasta containing 1,075 protein sequences, spanning 166 genotypes for SARS-CoV-2 were accessed using the NCBI Virus resource (
      <a href="https://www.ncbi.nlm.nih.gov/labs/virus/vssi/">https://www.ncbi.nlm.nih.gov/labs/virus/vssi/</a>) on March 15, 2020 (
      <strong>Supplementary Table 1</strong>). Of the 1,075 accession identifiers for proteins, there were 144 unique amino acid sequences. These were further reduced to 22 groups by overlapping amino acid sequences (i.e. some accessions were regions within annotated proteins). Using the named annotation of these accession identifiers, these 22 groups of accession identifiers were annotated by the 11 named proteins in the SARS-CoV-2 proteome; 8 accession identifiers were labeled ‘UNKNOWN’ because they were not annotated with protein names.
     </p>
    </div>
    <div class="section" id="sec-8">
     <h3>Peptide-MHC binding prediction</h3>
     <p id="p-25">MHC binding predictions were performed on 9-mer peptides derived from the protein fasta, across 9,360 haplotypes of class I HLA-A, -B, and -C, (
      <strong>Supplementary Table 2</strong>) using the pvacbind tool from pVACtools(
      <em>
       <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>
      </em>) and nine prediction algorithms (MHCflurry, MHCnuggetsI, NNalign, NetMHC, PickPocket, SMM, SMMPMBEC, SMMalign, NetMHC). The following parameters were used:
     </p>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1229471/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1229471/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1229471" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
     </div>
     <p id="p-26">These jobs were executed on the Google Cloud Platform using Terra and the griffithlab/pvactools Docker container. The binding affinities for each prediction were summarized by their Median Score.</p>
    </div>
    <div class="section" id="sec-9">
     <h3>Assignment of peptides to proteins</h3>
     <p id="p-27">Peptides were assigned to proteins based upon the protein annotation described above. Peptides per amino acid (Peptides/AA) were calculated by taking the total number of high-binding (&lt;;500nM) peptides associated with the protein accession divided by the length of the protein. These numbers were aggregated by the average Peptides/AA per protein.</p>
    </div>
    <div class="section" id="sec-10">
     <h3>Data Availability Statement</h3>
     <p id="p-28">The results of this study are available in a public Google bucket through the following link: 
      <a href="https://console.cloud.google.com/storage/browser/pici-covid19-data-resources">https://console.cloud.google.com/storage/browser/pici-covid19-data-resources</a>. The folder “gs://pici-covid19-data-resources/mhci/peptide_predictions” in this bucket contains all of the Supplementary Tables corresponding to this document and the unfiltered peptide binding predictions (all_epitopes.tsv results from pvacbind). See either the 
      <strong>Supplementary Note</strong> or the README in the Google bucket for further details.
     </p>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgments</h2>
    <p id="p-22">We are grateful to John Wherry (University of Pennsylvania, Philadelphia, PA) and Bonaventura Clotet, Julia Garcia Prado and Christian Brander (IrsiCaixa Foundation, Barcelona, Spain) for valuable feedback on the manuscript. The computational resources for this study were provided by the Parker Institute for Cancer Immunotherapy (PICI). KMC is supported by the UCLA Tumor Immunology Training Grant (NIH T32CA009120) and the Cancer Research Institute (CRI) Irvington Postdoctoral Fellowship Program. AK is supported by the UCLA CTSI KL2 Award (NCATS TR001882) and Sarcoma Alliance for Research Through Collaboration Career Enhancement Program. AR is supported by R35 CA197633 and The Ressler Family Fund, and is a member researcher at PICI.</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-dataset fn-group-external-links" id="fn-1">
      <p id="p-3">
       <a href="https://covid-19.parkerici.org/docs/data_sets/mhc_peptidome.html">https://covid-19.parkerici.org/docs/data_sets/mhc_peptidome.html</a>
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">REFERENCES</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.30.016931v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Yang</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Clinical course and outcomes of critically ill patients with SARS-CoV-2 pneumonia in Wuhan, China: a single-centered, retrospective, observational study</span>. 
         <abbr class="cit-jnl-abbrev">Lancet Respir Med</abbr>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.30.016931v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Wang</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Clinical Characteristics of 138 Hospitalized Patients With 2019 Novel Coronavirus-Infected Pneumonia in Wuhan, China</span>. 
         <abbr class="cit-jnl-abbrev">Jama</abbr>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.30.016931v1.3" data-doi="10.1128/JVI.78.11.5535-5545.2004">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">E. J.</span>
          <span class="cit-name-surname">Wherry</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Ahmed</span>
         </span>, 
         <span class="cit-article-title">Memory CD8 T-cell differentiation during viral infection</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-vol">78</span>, 
         <span class="cit-fpage">5535</span>–
         <span class="cit-lpage">5545</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DWherry%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D78%26rft.issue%253D11%26rft.spage%253D5535%26rft.epage%253D5545%26rft.atitle%253DMemory%2BCD8%2BT-Cell%2BDifferentiation%2Bduring%2BViral%2BInfection%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.11.5535-5545.2004%26rft_id%253Dinfo%253Apmid%252F15140950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3OC8xMS81NTM1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDEvMjAyMC4wMy4zMC4wMTY5MzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.30.016931v1.4" data-doi="10.1007/s00281-017-0629-x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Channappanavar</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Perlman</span>
         </span>, 
         <span class="cit-article-title">Pathogenic human coronavirus infections: causes and consequences of cytokine storm and immunopathology</span>. 
         <abbr class="cit-jnl-abbrev">Semin Immunopathol</abbr>
         <span class="cit-vol">39</span>, 
         <span class="cit-fpage">529</span>–
         <span class="cit-lpage">539</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSemin%2BImmunopathol%26rft.volume%253D39%26rft.spage%253D529%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00281-017-0629-x%26rft_id%253Dinfo%253Apmid%252F28466096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/s00281-017-0629-x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=28466096&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F01%2F2020.03.30.016931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.30.016931v1.5" data-doi="10.1016/S0140-6736(00)05117-5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A. V.</span>
          <span class="cit-name-surname">Hill</span>
         </span>, 
         <span class="cit-article-title">Immunogenetics and genomics</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>
         <span class="cit-vol">357</span>, 
         <span class="cit-fpage">2037</span>–
         <span class="cit-lpage">2041</span> (
         <span class="cit-pub-date">2001</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%26rft.stitle%253DLancet%26rft.aulast%253DHill%26rft.auinit1%253DA.%2BV.%26rft.volume%253D357%26rft.issue%253D9273%26rft.spage%253D2037%26rft.epage%253D2041%26rft.atitle%253DImmunogenetics%2Band%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252800%252905117-5%26rft_id%253Dinfo%253Apmid%252F11438157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0140-6736(00)05117-5&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=11438157&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F01%2F2020.03.30.016931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000169509200033&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.30.016931v1.6" data-doi="10.1158/2326-6066.CIR-19-0401">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Hundal</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">pVACtools: A Computational Toolkit to Identify and Visualize Cancer Neoantigens</span>. 
         <abbr class="cit-jnl-abbrev">Cancer Immunol Res</abbr>
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">409</span>–
         <span class="cit-lpage">420</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCancer%2BImmunol%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1158%252F2326-6066.CIR-19-0401%26rft_id%253Dinfo%253Apmid%252F31907209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FuaW1tIjtzOjU6InJlc2lkIjtzOjc6IjgvMy80MDkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wMS8yMDIwLjAzLjMwLjAxNjkzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.30.016931v1.7" data-doi="10.1186/1471-2172-9-1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Sidney</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Peters</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Frahm</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Brander</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Sette</span>
         </span>, 
         <span class="cit-article-title">HLA class I supertypes: a revised and updated classification</span>. 
         <abbr class="cit-jnl-abbrev">BMC Immunol</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">1</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2Bimmunology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BImmunol%26rft.aulast%253DSidney%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DHLA%2Bclass%2BI%2Bsupertypes%253A%2Ba%2Brevised%2Band%2Bupdated%2Bclassification.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2172-9-1%26rft_id%253Dinfo%253Apmid%252F18211710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2172-9-1&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18211710&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F01%2F2020.03.30.016931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.30.016931v1.8" data-doi="10.1093/nar/gky1006">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Vita</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">The Immune Epitope Database (IEDB): 2018 update</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">47</span>, 
         <span class="cit-fpage">D339</span>–
         <span class="cit-lpage">d343</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253DD339%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky1006%26rft_id%253Dinfo%253Apmid%252F30357391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gky1006&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=30357391&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F01%2F2020.03.30.016931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.30.016931v1.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Grifoni</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">A Sequence Homology and Bioinformatic Approach Can Predict Candidate Targets for Immune Responses to SARS-CoV-2</span>. 
         <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.30.016931v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">S. F.</span>
          <span class="cit-name-surname">Ahmed</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A. A.</span>
          <span class="cit-name-surname">Quadeer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. R.</span>
          <span class="cit-name-surname">McKay</span>
         </span>, 
         <span class="cit-article-title">Preliminary Identification of Potential Vaccine Targets for the COVID-19 Coronavirus (SARS-CoV-2) Based on SARS-CoV Immunological Studies</span>. 
         <abbr class="cit-jnl-abbrev">Viruses</abbr>
         <span class="cit-vol">12</span>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.30.016931v1.11" data-doi="10.1073/pnas.1815239116">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">N. P.</span>
          <span class="cit-name-surname">Croft</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Most viral peptides displayed by class I MHC on infected cells are immunogenic</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>
         <span class="cit-vol">116</span>, 
         <span class="cit-fpage">3112</span>–
         <span class="cit-lpage">3117</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1815239116%26rft_id%253Dinfo%253Apmid%252F30718433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTE2LzgvMzExMiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAxLzIwMjAuMDMuMzAuMDE2OTMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.30.016931v1.12" data-doi="10.1093/bioinformatics/btx820">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Andreatta</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">An automated benchmarking platform for MHC class II binding prediction methods</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">34</span>, 
         <span class="cit-fpage">1522</span>–
         <span class="cit-lpage">1528</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D1522%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx820&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.30.016931v1.13" data-doi="10.1016/j.bbrc.2006.03.152">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Y. P.</span>
          <span class="cit-name-surname">Tsao</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">HLA-A*0201 T-cell epitopes in severe acute respiratory syndrome (SARS) coronavirus nucleocapsid and spike proteins</span>. 
         <abbr class="cit-jnl-abbrev">Biochem Biophys Res Commun</abbr>
         <span class="cit-vol">344</span>, 
         <span class="cit-fpage">63</span>–
         <span class="cit-lpage">71</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DTsao%26rft.auinit1%253DY.%2BP.%26rft.volume%253D344%26rft.issue%253D1%26rft.spage%253D63%26rft.epage%253D71%26rft.atitle%253DHLA-A%2A0201%2BT-cell%2Bepitopes%2Bin%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2B%2528SARS%2529%2Bcoronavirus%2Bnucleocapsid%2Band%2Bspike%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2006.03.152%26rft_id%253Dinfo%253Apmid%252F16630549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2006.03.152&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16630549&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F01%2F2020.03.30.016931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000237408000011&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.30.016931v1.14" data-doi="10.1016/j.vaccine.2009.04.001">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Ohno</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Synthetic peptides coupled to the surface of liposomes effectively induce SARS coronavirus-specific cytotoxic T lymphocytes and viral clearance in HLA-A*0201 transgenic mice</span>. 
         <abbr class="cit-jnl-abbrev">Vaccine</abbr>
         <span class="cit-vol">27</span>, 
         <span class="cit-fpage">3912</span>–
         <span class="cit-lpage">3920</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVaccine%26rft.volume%253D27%26rft.spage%253D3912%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vaccine.2009.04.001%26rft_id%253Dinfo%253Apmid%252F19490987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.vaccine.2009.04.001&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19490987&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F01%2F2020.03.30.016931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.30.016931v1.15" data-doi="10.1111/j.0001-2815.2004.00221.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Sylvester-Hvid</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">SARS CTL vaccine candidates; HLA supertype-, genome-wide scanning and biochemical validation</span>. 
         <abbr class="cit-jnl-abbrev">Tissue Antigens</abbr>
         <span class="cit-vol">63</span>, 
         <span class="cit-fpage">395</span>–
         <span class="cit-lpage">400</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTissue%2Bantigens%26rft.stitle%253DTissue%2BAntigens%26rft.aulast%253DSylvester-Hvid%26rft.auinit1%253DC.%26rft.volume%253D63%26rft.issue%253D5%26rft.spage%253D395%26rft.epage%253D400%26rft.atitle%253DSARS%2BCTL%2Bvaccine%2Bcandidates%253B%2BHLA%2Bsupertype-%252C%2Bgenome-wide%2Bscanning%2Band%2Bbiochemical%2Bvalidation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0001-2815.2004.00221.x%26rft_id%253Dinfo%253Apmid%252F15104671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.0001-2815.2004.00221.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15104671&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F01%2F2020.03.30.016931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000221620300002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.30.016931v1.16" data-doi="10.4049/jimmunol.177.4.2138">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Zhou</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Screening and identification of severe acute respiratory syndrome-associated coronavirus-specific CTL epitopes</span>. 
         <abbr class="cit-jnl-abbrev">J Immunol</abbr>
         <span class="cit-vol">177</span>, 
         <span class="cit-fpage">2138</span>–
         <span class="cit-lpage">2145</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.stitle%253DJ.%2BImmunol.%26rft.aulast%253DZhou%26rft.auinit1%253DM.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2138%26rft.epage%253D2145%26rft.atitle%253DScreening%2Band%2BIdentification%2Bof%2BSevere%2BAcute%2BRespiratory%2BSyndrome-Associated%2BCoronavirus-Specific%2BCTL%2BEpitopes%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.177.4.2138%26rft_id%253Dinfo%253Apmid%252F16887973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIxMzgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wMS8yMDIwLjAzLjMwLjAxNjkzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.30.016931v1.17" data-doi="10.1016/j.vaccine.2007.05.025">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Y. K.</span>
          <span class="cit-name-surname">Cheung</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. C.</span>
          <span class="cit-name-surname">Cheng</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">F. W.</span>
          <span class="cit-name-surname">Sin</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K. T.</span>
          <span class="cit-name-surname">Chan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Xie</span>
         </span>, 
         <span class="cit-article-title">Induction of T-cell response by a DNA vaccine encoding a novel HLA-A*0201 severe acute respiratory syndrome coronavirus epitope</span>. 
         <abbr class="cit-jnl-abbrev">Vaccine</abbr>
         <span class="cit-vol">25</span>, 
         <span class="cit-fpage">6070</span>–
         <span class="cit-lpage">6077</span> (
         <span class="cit-pub-date">2007</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVaccine%26rft.stitle%253DVaccine%26rft.aulast%253DCheung%26rft.auinit1%253DY.%2BK.%26rft.volume%253D25%26rft.issue%253D32%26rft.spage%253D6070%26rft.epage%253D6077%26rft.atitle%253DInduction%2Bof%2BT-cell%2Bresponse%2Bby%2Ba%2BDNA%2Bvaccine%2Bencoding%2Ba%2Bnovel%2BHLA-A%2A0201%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%2Bepitope.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vaccine.2007.05.025%26rft_id%253Dinfo%253Apmid%252F17629360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.vaccine.2007.05.025&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17629360&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F01%2F2020.03.30.016931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.30.016931v1.18" data-doi="10.4049/jimmunol.1301507">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Rivino</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Defining CD8+ T cell determinants during human viral infection in populations of Asian ethnicity</span>. 
         <abbr class="cit-jnl-abbrev">J Immunol</abbr>
         <span class="cit-vol">191</span>, 
         <span class="cit-fpage">4010</span>–
         <span class="cit-lpage">4019</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BImmunol%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1301507%26rft_id%253Dinfo%253Apmid%252F24058176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE5MS84LzQwMTAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wMS8yMDIwLjAzLjMwLjAxNjkzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.30.016931v1.19" data-doi="10.1002/eji.201243088">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">C. X.</span>
          <span class="cit-name-surname">Chang</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Conditional ligands for Asian HLA variants facilitate the definition of CD8+ T-cell responses in acute and chronic viral diseases</span>. 
         <abbr class="cit-jnl-abbrev">Eur J Immunol</abbr>
         <span class="cit-vol">43</span>, 
         <span class="cit-fpage">1109</span>–
         <span class="cit-lpage">1120</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEur%2BJ%2BImmunol%26rft.volume%253D43%26rft.spage%253D1109%26rft_id%253Dinfo%253Adoi%252F10.1002%252Feji.201243088%26rft_id%253Dinfo%253Apmid%252F23280567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/eji.201243088&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23280567&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F01%2F2020.03.30.016931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.30.016931v1.20" data-doi="10.1128/JVI.78.11.5612-5618.2004">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Y. D.</span>
          <span class="cit-name-surname">Wang</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">T-cell epitopes in severe acute respiratory syndrome (SARS) coronavirus spike protein elicit a specific T-cell immune response in patients who recover from SARS</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-vol">78</span>, 
         <span class="cit-fpage">5612</span>–
         <span class="cit-lpage">5618</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DWang%26rft.auinit1%253DY.-D.%26rft.volume%253D78%26rft.issue%253D11%26rft.spage%253D5612%26rft.epage%253D5618%26rft.atitle%253DT-Cell%2BEpitopes%2Bin%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2B%2528SARS%2529%2BCoronavirus%2BSpike%2BProtein%2BElicit%2Ba%2BSpecific%2BT-Cell%2BImmune%2BResponse%2Bin%2BPatients%2BWho%2BRecover%2Bfrom%2BSARS%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.11.5612-5618.2004%26rft_id%253Dinfo%253Apmid%252F15140958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3OC8xMS81NjEyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDEvMjAyMC4wMy4zMC4wMTY5MzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.30.016931v1.21" data-doi="10.1086/656315">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Liu</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">The membrane protein of severe acute respiratory syndrome coronavirus acts as a dominant immunogen revealed by a clustering region of novel functionally and structurally defined cytotoxic T-lymphocyte epitopes</span>. 
         <abbr class="cit-jnl-abbrev">J Infect Dis</abbr>
         <span class="cit-vol">202</span>, 
         <span class="cit-fpage">1171</span>–
         <span class="cit-lpage">1180</span> (
         <span class="cit-pub-date">2010</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BInfect%2BDis%26rft_id%253Dinfo%253Adoi%252F10.1086%252F656315%26rft_id%253Dinfo%253Apmid%252F20831383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1086/656315&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20831383&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F01%2F2020.03.30.016931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.30.016931v1.22" data-doi="10.1186/1471-2172-10-61">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Lv</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Ruan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Wang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Ni</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Wu</span>
         </span>, 
         <span class="cit-article-title">Identification of a novel conserved HLA-A*0201-restricted epitope from the spike protein of SARS-CoV</span>. 
         <abbr class="cit-jnl-abbrev">BMC Immunol</abbr>
         <span class="cit-vol">10</span>, 
         <span class="cit-fpage">61</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2Bimmunology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BImmunol%26rft.aulast%253DLv%26rft.auinit1%253DY.%26rft.volume%253D10%26rft.spage%253D61%26rft.epage%253D61%26rft.atitle%253DIdentification%2Bof%2Ba%2Bnovel%2Bconserved%2BHLA-A%2A0201-restricted%2Bepitope%2Bfrom%2Bthe%2Bspike%2Bprotein%2Bof%2BSARS-CoV.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2172-10-61%26rft_id%253Dinfo%253Apmid%252F19958537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2172-10-61&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19958537&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F01%2F2020.03.30.016931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.30.016931v1.23" data-doi="10.1182/blood-2003-11-4072">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Wang</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Identification of an HLA-A*0201-restricted CD8+ T-cell epitope SSp-1 of SARS-CoV spike protein</span>. 
         <abbr class="cit-jnl-abbrev">Blood</abbr>
         <span class="cit-vol">104</span>, 
         <span class="cit-fpage">200</span>–
         <span class="cit-lpage">206</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBlood%26rft_id%253Dinfo%253Adoi%252F10.1182%252Fblood-2003-11-4072%26rft_id%253Dinfo%253Apmid%252F15016646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImJsb29kam91cm5hbCI7czo1OiJyZXNpZCI7czo5OiIxMDQvMS8yMDAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wMS8yMDIwLjAzLjMwLjAxNjkzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.30.016931v1.24" data-doi="10.1128/JVI.05039-11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">H. L.</span>
          <span class="cit-name-surname">Oh</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Engineering T cells specific for a dominant severe acute respiratory syndrome coronavirus CD8 T cell epitope</span>. 
         <abbr class="cit-jnl-abbrev">Journal of virology</abbr>
         <span class="cit-vol">85</span>, 
         <span class="cit-fpage">10464</span>–
         <span class="cit-lpage">10471</span> (
         <span class="cit-pub-date">2011</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.05039-11%26rft_id%253Dinfo%253Apmid%252F21813600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4NS8yMC8xMDQ2NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAxLzIwMjAuMDMuMzAuMDE2OTMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">25.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.30.016931v1.25" data-doi="10.1016/j.antiviral.2009.09.004">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Kohyama</span>
         </span>
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Efficient induction of cytotoxic T lymphocytes specific for severe acute respiratory syndrome (SARS)-associated coronavirus by immunization with surface-linked liposomal peptides derived from a non-structural polyprotein 1a</span>. 
         <abbr class="cit-jnl-abbrev">Antiviral Res</abbr>
         <span class="cit-vol">84</span>, 
         <span class="cit-fpage">168</span>–
         <span class="cit-lpage">177</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAntiviral%2Bresearch%26rft.stitle%253DAntiviral%2BRes%26rft.aulast%253DKohyama%26rft.auinit1%253DS.%26rft.volume%253D84%26rft.issue%253D2%26rft.spage%253D168%26rft.epage%253D177%26rft.atitle%253DEfficient%2Binduction%2Bof%2Bcytotoxic%2BT%2Blymphocytes%2Bspecific%2Bfor%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2B%2528SARS%2529-associated%2Bcoronavirus%2Bby%2Bimmunization%2Bwith%2Bsurface-linked%2Bliposomal%2Bpeptides%2Bderived%2Bfrom%2Ba%2Bnon-structural%2Bpolyprotein%2B1a.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2009.09.004%26rft_id%253Dinfo%253Apmid%252F19748524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2009.09.004&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19748524&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F01%2F2020.03.30.016931.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
