<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/01/02/509141.atom" />
<meta name="HW.pisa" content="biorxiv;509141v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Kinetic Analysis of Bacteriophage Sf6 Binding to Outer Membrane Protein A Using Whole Virions" />
<meta name="DC.Identifier" content="10.1101/509141" />
<meta name="DC.Date" content="2019-01-02" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="For successful infection, viruses must recognize their respective host cells. A common mechanism of host recognition by viruses is to utilize a portion of the host cell as a receptor. Bacteriophage Sf6, which infects Shigella flexneri , uses lipopolysaccharide as a primary receptor and then requires interaction with a secondary receptor, a role that can be fulfilled by either outer membrane proteins (Omp) A or C. Our previous work showed that specific residues in the loops of OmpA mediate Sf6 infection. To better understand Sf6 interactions with OmpA loop variants, we determined the kinetics of these interactions through the use of biolayer interferometry, an optical biosensing technique that yields data similar to surface plasmon resonance. Here, we successfully tethered whole Sf6 virions, determined the binding constant of Sf6 to OmpA to be 36 nM. Additionally, we showed that Sf6 bound to five variant OmpAs and the resulting kinetic parameters varied only slightly. Based on these data, we propose a model in which Sf6: Omp receptor recognition is not solely based on kinetics, but likely also on the ability of an Omp to induce a conformational change that results in productive infection." />
<meta name="DC.Contributor" content="Natalia B. Hubbs" />
<meta name="DC.Contributor" content="Mareena M. Whisby-Pitts" />
<meta name="DC.Contributor" content="Jonathan L. McMurry" />
<meta name="article:published_time" content="2019-01-02" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Kinetic Analysis of Bacteriophage Sf6 Binding to Outer Membrane Protein A Using Whole Virions" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;For successful infection, viruses must recognize their respective host cells. A common mechanism of host recognition by viruses is to utilize a portion of the host cell as a receptor. Bacteriophage Sf6, which infects &lt;i&gt;Shigella flexneri&lt;/i&gt;, uses lipopolysaccharide as a primary receptor and then requires interaction with a secondary receptor, a role that can be fulfilled by either outer membrane proteins (Omp) A or C. Our previous work showed that specific residues in the loops of OmpA mediate Sf6 infection. To better understand Sf6 interactions with OmpA loop variants, we determined the kinetics of these interactions through the use of biolayer interferometry, an optical biosensing technique that yields data similar to surface plasmon resonance. Here, we successfully tethered whole Sf6 virions, determined the binding constant of Sf6 to OmpA to be 36 nM. Additionally, we showed that Sf6 bound to five variant OmpAs and the resulting kinetic parameters varied only slightly. Based on these data, we propose a model in which Sf6: Omp receptor recognition is not solely based on kinetics, but likely also on the ability of an Omp to induce a conformational change that results in productive infection.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;509141v1" />
<meta name="citation_id" content="509141v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/509141v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/509141v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/509141v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141.full.pdf" />
<meta name="citation_doi" content="10.1101/509141" />
<meta name="citation_num_pages" content="23" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="509141" />
<meta name="citation_author" content="Natalia B. Hubbs" />
<meta name="citation_author_institution" content="Department of Biology, Hanover College" />
<meta name="citation_author" content="Mareena M. Whisby-Pitts" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, Kennesaw State University" />
<meta name="citation_author" content="Jonathan L. McMurry" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, Kennesaw State University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0096-5094" />
<meta name="citation_reference" content="Flint, S.J.; Enquiest, L.W.; Krug, R.M.; Racaniello, V.R.; Skalka, A.M. Principles of virology, molecular biology, pathogenesis, and control. ASM Press: 2000." />
<meta name="citation_reference" content="Hendrix, R.W. Bacteriophages: Evolution of the majority. Theor Popul Biol 2002, 61, 471–480." />
<meta name="citation_reference" content="Poranen, M.M.; Daugelavicius, R.; Bamford, D.H. Common principles in viral entry. Annul Rev Microbiol 2002, 56, 521–538." />
<meta name="citation_reference" content="Rakhuba, D.V.; Kolomiets, E.I.; Dey, E.S.; Novik, G.I. Bacteriophage receptors, mechanisms of phage adsorption and penetration into host cell. Pol J Microbiol 2010, 59, 145–155." />
<meta name="citation_reference" content="Chatterjee, A. Use of bacteriophage-resistant mutants to study the nature of the bacteriophage receptor site of Staphylococcus aureus. J Bacteriol 1969, 98, 519–527." />
<meta name="citation_reference" content="Cleary, P.P.; Wannamaker, L.W.; Fisher, M.; Laible, N. Studies of the receptor for phage A25 in group A Streptococci: the role of peptidoglycan in reversible adsorption. J Exp Med 1977, 145, 578–593." />
<meta name="citation_reference" content="Wendlinger, G.; Loessner, M.J.; Scherer, S. Bacteriophage receptors on Listeria monocytogenes cells are the N-acetylglucosamine and rhamnose substituents of teichoic acids or the peptidoglycan itself. Microbiology 1996, 142 (Pt 4), 985–992." />
<meta name="citation_reference" content="Sao-Jose, C.; Lhuillier, S.; Lurz, R.; Melki, R.; Lepault, J.; Santos, M.A.; Tavares, P. The ectodomain of the viral receptor Yueb forms a fiber that triggers ejection of bacteriophage SPP1 DNA. J Biol Chem 2006, 281, 11464–11470." />
<meta name="citation_reference" content="Baptista, C.; Santos, M.A.; Sao-Jose, C. Phage SPP1 reversible adsorption to Bacillus subtilis cell wall teichoic acids accelerates virus recognition of membrane receptor Yueb. J Bacteriol 2008, 190, 4989–4996." />
<meta name="citation_reference" content="Young, F.E. Requirement of glucosylated teichoic acid for adsorption of phage in Bacillus subtilis 168. Proc Natl Acad Sci USA 1967, 58, 2377–2384." />
<meta name="citation_reference" content="Gonzalez-Garcia, V.A.; Pulido-Cid, M.; Garcia-Doval, C.; Bocanegra, R.; van Raaij, M.J.; Martin-Benito, J.; Cuervo, A.; Carrascosa, J.L. Conformational changes leading to T7 DNA delivery upon interaction with the bacterial receptor. J Biol Chem 2015, 290, 10038–10044." />
<meta name="citation_reference" content="Marti, R.; Zurfluh, K.; Hagens, S.; Pianezzi, J.; Klumpp, J.; Loessner, M.J. Long tail fibres of the novel broad-host-range T-even bacteriophage S16 specifically recognize Salmonella OmpC. Mol Microbiol 2013, 87, 818–834." />
<meta name="citation_reference" content="Baxa, U.; Steinbacher, S.; Miller, S.; Weintraub, A.; Huber, R.; Seckler, R. Interactions of phage P22 tails with their cellular receptor, Salmonella O-antigen polysaccharide. Biophys J 1996, 71, 2040–2048." />
<meta name="citation_reference" content="Lindberg, A.A.; Wollin, R.; Gemski, P.; Wohlhieter, J.A. Interaction between bacteriophage Sf6 and Shigella flexneri. J Virol 1978, 27, 38–44." />
<meta name="citation_reference" content="Smith, S.G.; Mahon, V.; Lambert, M.A.; Fagan, R.P. A molecular swiss army knife: OmpA structure, function and expression. FEMS Microbiol Lett 2007, 273, 1–11." />
<meta name="citation_reference" content="Morona, R.; Kramer, C.; Henning, U. Bacteriophage receptor area of outer membrane protein OmpA of Escherichia coli k-12. J Bacteriol 1985, 164, 539–543." />
<meta name="citation_reference" content="Hashemolhosseini, S.; Holmes, Z.; Mutschler, B.; Henning, U. Alterations of receptor specificities of coliphages of the T2 family. J Mol Biol 1994, 240, 105–110." />
<meta name="citation_reference" content="Hashemolhosseini, S.; Montag, D.; Kramer, L.; Henning, U. Determinants of receptor specificity of coliphages of the T4 family. A chaperone alters the host range. J Mol Biol 1994, 241, 524–533." />
<meta name="citation_reference" content="Randall-Hazelbauer, L.; Schwartz, M. Isolation of the bacteriophage lambda receptor from Escherichia coli. J Bacteriol 1973, 116, 1436–1446." />
<meta name="citation_reference" content="Hantke, K.; Braun, V. Functional interaction of the TonA/TonB receptor system in Escherichia coli. J Bacteriol 1978, 135, 190–197." />
<meta name="citation_reference" content="Braun, V.; Wolff, H. Characterization of the receptor protein for phage T5 and colicin M in the outer membrane of E. coli B. FEBS Lett 1973, 34, 77–80." />
<meta name="citation_reference" content="Morona, R.; Mavris, M.; Fallarino, A.; Manning, P.A. Characterization of the rfc region of Shigella flexneri. J Bacteriol 1994, 176, 733–747." />
<meta name="citation_reference" content="Porcek, N.B.; Parent, K.N. Key residues of S. flexneri OmpA mediate infection by bacteriophage Sf6. J Mol Biol 2015, 427, 1964–1976." />
<meta name="citation_reference" content="Freiberg, A.; Morona, R.; Van den Bosch, L.; Jung, C.; Behlke, J.; Carlin, N.; Seckler, R.; Baxa, U. The tailspike protein of Shigella phage Sf6. A structural homolog of Salmonella phage P22 tailspike protein without sequence similarity in the beta-helix domain. J Biol Chem 2003, 278, 1542–1548." />
<meta name="citation_reference" content="Parent, K.N.; Erb, M.L.; Cardone, G.; Nguyen, K.; Gilcrease, E.B.; Porcek, N.B.; Pogliano, J.; Baker, T.S.; Casjens, S.R. OmpA and OmpC are critical host factors for bacteriophage Sf6 entry in Shigella. Mol Microbiol 2014, 92, 47–60." />
<meta name="citation_reference" content="Morona, R.; Henning, U. Host range mutants of bacteriophage Ox2 can use two different outer membrane proteins of Escherichia coli K-12 as receptors. J Bacteriol 1984, 159, 579–582." />
<meta name="citation_reference" content="Drexler, K.; Riede, I.; Montag, D.; Eschbach, M.; Henning, U. Receptor specificity of the Escherichia coli T-even type phage Ox2. J Mol Biol 1989, 207, 797–803." />
<meta name="citation_reference" content="Concepcion, J.; Witte, K.; Wartchow, C.; Choo, S.; Yao, D.; Persson, H.; Wei, J.; Li, P.; Heidecker, B.; Ma, W., et al. Label-free detection of biomolecular interactions using biolayer interferometry for kinetic characterization. Comb Chem High Throughput Screen 2009, 8, 791–800." />
<meta name="citation_reference" content="Abdiche, Y.; Malashock, D.; Pinkerton, A.; Pons, J. Determining kinetics and affinities of protein interactions using a parallel real-time label-free biosensor, the octet. Anal Biochem 2008, 377, 209–217." />
<meta name="citation_reference" content="Zhang, H.; de Vries, R.P.; Tzarum, N.; Zhu, X.; Yu, W.; McBride, R.; Paulson, J.C.; Wilson, I.A. A human-infecting H10N8 influenza virus retains a strong preference for avian-type receptors. Cell Host Microbe 2015, 17, 377–384." />
<meta name="citation_reference" content="Yang, H.; Carney, P.J.; Chang, J.C.; Guo, Z.; Villanueva, J.M.; Stevens, J. Structure and receptor binding preferences of recombinant human a(H3N2) virus hemagglutinins. Virology 2015, 477, 18–31." />
<meta name="citation_reference" content="Xiong, X.; Martin, S.R.; Haire, L.F.; Wharton, S.A.; Daniels, R.S.; Bennett, M.S.; McCauley, J.W.; Collins, P.J.; Walker, P.A.; Skehel, J.J., et al. Receptor binding by an H7N9 influenza virus from humans. Nature 2013, 499, 496–499." />
<meta name="citation_reference" content="Casjens, S.; Winn-Stapley, D.A.; Gilcrease, E.B.; Morona, R.; Kuhlewein, C.; Chua, J.E.; Manning, P.A.; Inwood, W.; Clark, A.J. The chromosome of Shigella flexneri bacteriophage Sf6: Complete nucleotide sequence, genetic mosaicism, and DNA packaging. J Mol Biol 2004, 339, 379–394." />
<meta name="citation_reference" content="Bonaparte, M.I.; Dimitrov, A.S.; Bossart, K.N.; Crameri, G.; Mungall, B.A.; Bishop, K.A.; Choudhry, V.; Dimitrov, D.S.; Wang, L.F.; Eaton, B.T., et al. Ephrin-B2 ligand is a functional receptor for hendra virus and nipah virus. Proc Natl Acad of Sci USA 2005, 102, 10652–10657." />
<meta name="citation_reference" content="Wang, Q.; Qi, J.; Yuan, Y.; Xuan, Y.; Han, P.; Wan, Y.; Ji, W.; Li, Y.; Wu, Y.; Wang, J., et al. Bat origins of MERS-CoV supported by bat coronavirus HKU4 usage of human receptor CD26. Cell Host Microbe 2014, 16, 328–337." />
<meta name="citation_reference" content="Yang, D.; Singh, A.; Wu, H.; Kroe-Barrett, R. Comparison of biosensor platforms in the evaluation of high affinity antibody-antigen binding kinetics. Anal Biochem 2016, 508, 78–96." />
<meta name="citation_reference" content="Burguillo, F.J.; Wright, A.J.; Bardsley, W.G. Use of the F test for determining the degree of enzyme-kinetic and ligand-binding data. A monte carlo simulation study. Biochem J 1983, 211, 23–34." />
<meta name="citation_reference" content="Kleinschmidt, J.H.; Wiener, M.C.; Tamm, L.K. Outer membrane protein a of E. coli folds into detergent micelles, but not in the presence of monomeric detergent. Protein Science 1999, 8, 2065–2071." />
<meta name="citation_reference" content="Andres, D.; Roske, Y.; Doering, C.; Heinemann, U.; Seckler, R.; Barbirz, S. Tail morphology controls DNA release in two Salmonella phages with one lipopolysaccharide receptor recognition system. Mol Microbiol 2012, 83, 1244–1253." />
<meta name="citation_reference" content="de Frutos, M.; Letellier, L.; Raspaud, E. DNA ejection from bacteriophage T5: Analysis of the kinetics and energetics. Biophys J 2005, 88, 1364–1370." />
<meta name="citation_reference" content="Killmann, H.; Videnov, G.; Jung, G.; Schwarz, H.; Braun, V. Identification of receptor binding sites by competitive peptide mapping: Phages T1, T5, and Phi 80 and colicin M bind to the gating loop of Fhua. J Bacteriol 1995, 177, 694–698." />
<meta name="citation_reference" content="Mackay, D.J.; Bode, V.C. Events in lambda injection between phage adsorption and DNA entry. Virology 1976, 72, 154–166." />
<meta name="citation_reference" content="Legrand, P.; Collins, B.; Blangy, S.; Murphy, J.; Spinelli, S.; Gutierrez, C.; Richet, N.; Kellenberger, C.; Desmyter, A.; Mahony, J., et al. The atomic structure of the phage Tuc2009 baseplate tripod suggests that host recognition involves two different carbohydrate binding modules. mBio 2016, 7, e01781–01715." />
<meta name="citation_reference" content="Casjens, S.R.; Molineux, I.J. Short noncontractile tail machines: adsorption and DNA delivery by podoviruses. In Advances in experimental medicine and biology, 2012; Vol. 726, pp 143–179." />
<meta name="citation_reference" content="Hu, B.; Margolin, W.; Molineux, I.J.; Liu, J. The bacteriophage T7 virion undergoes extensive structural remodeling during infection. Science 2013, 339, 576–579." />
<meta name="citation_reference" content="Muller, J.J.; Barbirz, S.; Heinle, K.; Freiberg, A.; Seckler, R.; Heinemann, U. An intersubunit active site between supercoiled parallel beta helices in the trimeric tailspike endorhamnosidase of Shigella flexneri phage Sf6. Structure 2008, 16, 766–775." />
<meta name="citation_reference" content="Wang, C.; Tu, J.; Hu, B.; Molineux, I.; Liu, J. Visualizing infection initiation of bacteriophage P22 by cryo-electron tomography. Biophys J, 2017." />
<meta name="citation_reference" content="Andres, D.; Hanke, C.; Baxa, U.; Seul, A.; Barbirz, S.; Seckler, R. Tailspike interactions with lipopolysaccharide effect DNA ejection from phage P22 particles in vitro. J Biol Chem 2010, 285, 36768–36775." />
<meta name="citation_reference" content="Wu, W.; Leavitt, J.C.; Cheng, N.; Gilcrease, E.B.; Motwani, T.; Teschke, C.M.; Casjens, S.R.; Steven, A.C. Localization of the houdinisome (ejection proteins) inside the bacteriophage P22 virion by bubblegram imaging. mBio 2016, 7." />
<meta name="citation_reference" content="Bhardwaj, A.; Sankhala, R.S.; Olia, A.S.; Brooke, D.; Casjens, S.R.; Taylor, D.J.; Prevelige, P.E., Jr.; Cingolani, G. Structural plasticity of the protein plug that traps newly packaged genomes in Podoviridae virions. J Biol Chem 2016, 291, 215–226." />
<meta name="citation_reference" content="Bhardwaj, A.; Olia, A.S.; Cingolani, G. Architecture of viral genome-delivery molecular machines. Curr Opin Struct Biol 2014, 25, 1–8." />
<meta name="citation_reference" content="Pettersen, E.F.; Goddard, T.D.; Huang, C.C.; Couch, G.S.; Greenblatt, D.M.; Meng, E.C.; Ferrin, T.E. UCSF chimera--a visualization system for exploratory research and analysis. J Comput Chem 2004, 25, 1605–1612." />
<meta name="twitter:title" content="Kinetic Analysis of Bacteriophage Sf6 Binding to Outer Membrane Protein A Using Whole Virions" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="For successful infection, viruses must recognize their respective host cells. A common mechanism of host recognition by viruses is to utilize a portion of the host cell as a receptor. Bacteriophage Sf6, which infects Shigella flexneri , uses lipopolysaccharide as a primary receptor and then requires interaction with a secondary receptor, a role that can be fulfilled by either outer membrane proteins (Omp) A or C. Our previous work showed that specific residues in the loops of OmpA mediate Sf6 infection. To better understand Sf6 interactions with OmpA loop variants, we determined the kinetics of these interactions through the use of biolayer interferometry, an optical biosensing technique that yields data similar to surface plasmon resonance. Here, we successfully tethered whole Sf6 virions, determined the binding constant of Sf6 to OmpA to be 36 nM. Additionally, we showed that Sf6 bound to five variant OmpAs and the resulting kinetic parameters varied only slightly. Based on these data, we propose a model in which Sf6: Omp receptor recognition is not solely based on kinetics, but likely also on the ability of an Omp to induce a conformational change that results in productive infection." />
<meta name="og-title" property="og:title" content="Kinetic Analysis of Bacteriophage Sf6 Binding to Outer Membrane Protein A Using Whole Virions" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/509141v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="For successful infection, viruses must recognize their respective host cells. A common mechanism of host recognition by viruses is to utilize a portion of the host cell as a receptor. Bacteriophage Sf6, which infects Shigella flexneri , uses lipopolysaccharide as a primary receptor and then requires interaction with a secondary receptor, a role that can be fulfilled by either outer membrane proteins (Omp) A or C. Our previous work showed that specific residues in the loops of OmpA mediate Sf6 infection. To better understand Sf6 interactions with OmpA loop variants, we determined the kinetics of these interactions through the use of biolayer interferometry, an optical biosensing technique that yields data similar to surface plasmon resonance. Here, we successfully tethered whole Sf6 virions, determined the binding constant of Sf6 to OmpA to be 36 nM. Additionally, we showed that Sf6 bound to five variant OmpAs and the resulting kinetic parameters varied only slightly. Based on these data, we propose a model in which Sf6: Omp receptor recognition is not solely based on kinetics, but likely also on the ability of an Omp to induce a conformational change that results in productive infection." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-01-02" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/509141v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/509141v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/509141v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/509141v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/150617" />
    <title>Kinetic Analysis of Bacteriophage Sf6 Binding to Outer Membrane Protein A Using Whole Virions | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Tz3Bb8fiY--z-9TFUef_2v2v5Go6FfUQzXiRmXHLKF4","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"150617","apath":"\/biorxiv\/early\/2019\/01\/02\/509141.atom","pisa":"biorxiv;509141v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;509141v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/509141v1","title":"Kinetic Analysis of Bacteriophage Sf6 Binding to Outer Membrane Protein A Using Whole Virions","identifier":"node\/150617"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NdFugAbmIEDvzJ6TaUEIjZ5SwKrZ0qrDzBKodNRyonYIOTNYLgkPrVG2cXWLxp8ZOwRMDgiSOKmth+15CsDQGdjVK4daripfcWJMInOPuYdgQaMppE6u7sN39gGlDHlF1\\\\\\\/AkxGtcZt3Ok0mNQLA\\\\\\\/s3hIqBKyRVconiCttEMMatbX3eKZWGc7rcn3NHsiG2oFM6lIOVegcGGjQgItXfoDfbOuO5QpMCETXXgQcCuytwKmUqSYqNel07hAiwKxakpA5H3VIA8Z32w6ddP\\\\\\\/ZRmjhOoA8fQBKhwRWxH6GedHLC5xOFT0Kj1EDZRLVuRPN9gWhlDs1Ssc8nHQl00cuBtuChCp+REKO\\\\\\\/AsZlbMp4Ae1ktJUY\\\\\\\/0C6jZ0o+3Gn+er2deDxdKOzYSbKBQJJRqeICAEqr8jXIGAoodIvt226OyCP+IHWyY4UNc1o\\\\\\\/zkolm7EnApkrTKy8IEccZrEz+hXsXFpW72ynM+CgoVMIHCrH72z2YctLnP+QE7VWiRGE1ry5FB5a9Kb7gfJKzRH+Db2pD8Y6VHJCE9ehg0Cq3o3FSGzqmNcKvYj\\\\\\\/aTJR7U\\\\\\\/do0eKO11TE6iiusr1VXZwLBVLOe2oAL++oKvVBIzs3reQBLQDzhlTAd1eYKevisHiHg03s5XKTukekQ24LVuirmbkTOz5cu0Byp2wEWhe3fiZx1kKb1zB\\\\\\\/Sho2cra8ZTD2H8OCdaobZiv96QyJdNejs4CWwV\\\\\\\/Ko1jEWfhQZ55fZghxYjP3O0MRIZp98r1psiMYjYD3Pam7t+aU32QdXAc6D+yM7icugxs6NM+ev7xRVtO80vnisQtb9BrMnXmbbA0bOoOlc6gywdFeBB\\\\\\\/LDHJElh9PBtjNXrgbMzDjizAfAbliRqTUocqd3r9lLpuonUCsUHmiFRjAg8l5zDgRP4IfnGz+7NSw4cHbbsVlOKJOzvU5ZSuItW7SYv185\\\\\\\/cQqZUoAQjll5I34pms7PbXSaxYH7OgNoHiGEodoVHy81Ly\\\\\\\/\\\\\\\/vc8vfMBpMomGKnD29LkwY078IGMEK8qvZQxewYhwd1A0p2s5wYkS8QE\\\\\\\/L49grpv02VClXqo5SlCWmdNz08wxt\\\\\\\/i\\\\\\\/U1wlCaYFIPaZ33Wd2JsO+s0AqLe9Zsi3godhtWAsojjFYhBETPPAkkBkuzgZQMIbVHDDOvg26ALaB4Hl2fFklK0Cq6fvQRrwpw\\\\\\\/aocca7D6xusP5D1iYtacfcaibuHIZh7PFExM6A\\\\\\\/OZgjc4YYnZwQrwyh1IVpvr42InRU0Ve4uWU5lhwMUx\\\\\\\/5t+6DG61vsne\\\\\\\/P6\\\\\\\/+412iA3eY1u1KHkxHKOXsDTvsKd7\\\\\\\/5wbKPcNAySI=\\\u0022,\\\u0022iv\\\u0022:\\\u0022TqXyiQVFNAimiC3FQW5OSA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00220699ee4c1d2886193e5147f3913607b4753accabf0841c407fa615b4d06e4b8c\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/509141v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-150617 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/509141v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5LBj4AoItC1kXLbnat5eQOcNb5Bcm2e0NNAJTT3AVdo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="150617" id="node150617" data-pisa="biorxiv;509141v1" data-pisa-master="biorxiv;509141" data-apath="/biorxiv/early/2019/01/02/509141.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Kinetic Analysis of Bacteriophage Sf6 Binding to Outer Membrane Protein A Using Whole Virions</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Natalia B.</span> <span class="nlm-surname">Hubbs</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mareena M.</span> <span class="nlm-surname">Whisby-Pitts</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-0096-5094" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jonathan L.</span> <span class="nlm-surname">McMurry</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/509141 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node150617" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Natalia B. Hubbs </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biology, Hanover College</span>, Hanover, IN, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natalia%2BB.%2BHubbs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hubbs%20NB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANatalia%2BB.%2BHubbs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mareena M. Whisby-Pitts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Molecular and Cellular Biology, Kennesaw State University</span>, Kennesaw, Georgia, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mareena%2BM.%2BWhisby-Pitts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whisby-Pitts%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMareena%2BM.%2BWhisby-Pitts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jonathan L. McMurry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Molecular and Cellular Biology, Kennesaw State University</span>, Kennesaw, Georgia, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BL.%2BMcMurry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McMurry%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJonathan%2BL.%2BMcMurry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0096-5094" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jonathan L. McMurry</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/509141v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:150617" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:150617/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/509141v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:150617" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:150617/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/509141v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:150617" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:150617/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/509141v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:150617" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:150617/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/509141v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:150617" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:150617/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">For successful infection, viruses must recognize their respective host cells. A common mechanism of host recognition by viruses is to utilize a portion of the host cell as a receptor. Bacteriophage Sf6, which infects <em>Shigella flexneri</em>, uses lipopolysaccharide as a primary receptor and then requires interaction with a secondary receptor, a role that can be fulfilled by either outer membrane proteins (Omp) A or C. Our previous work showed that specific residues in the loops of OmpA mediate Sf6 infection. To better understand Sf6 interactions with OmpA loop variants, we determined the kinetics of these interactions through the use of biolayer interferometry, an optical biosensing technique that yields data similar to surface plasmon resonance. Here, we successfully tethered whole Sf6 virions, determined the binding constant of Sf6 to OmpA to be 36 nM. Additionally, we showed that Sf6 bound to five variant OmpAs and the resulting kinetic parameters varied only slightly. Based on these data, we propose a model in which Sf6: Omp receptor recognition is not solely based on kinetics, but likely also on the ability of an Omp to induce a conformational change that results in productive infection.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">Virtually all viruses must translocate their genetic information into their respective host cells and replicate via the host cell machinery to produce progeny [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. dsDNA bacteriophages, which infect bacteria, are the most abundant viruses in the biosphere, with a global population estimated to be greater than 10<sup>30</sup> [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. The molecular mechanisms that govern bacteriophage attachment to their hosts are not completely understood. Host recognition must be well coordinated by the virus in order to ensure fitness and progeny formation, as premature genome ejection does not result in a successful infection. One common mechanism bacteriophages employ is to utilize a portion of the host cell as a receptor [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>], and this can be through interactions with surface glycans, proteins, or both.</p><p id="p-5">Teichoic acid, peptidoglycan, and other components of Gram-positive bacteria have been shown to be receptors for many phages [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>-<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. For instance, φ29 recognizes glucosylated teichoic acid in <em>Bacillus subtilis</em> [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. Bacteriophage SPP1, in addition to teichoic acid, also requires recognition of membrane protein YueB to irreversibly adsorb and commit to infection [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>]. Lipopolysaccharide (LPS) and/or proteins localized on the outer surface of Gram-negative bacteria are also used as phage receptors [<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>]. For instance, bacteriophage T7 recognizes the LPS of <em>E</em>. <em>coli</em> [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>] and phage S16 recognizes outer membrane protein C (OmpC) [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. Different forms of LPS are commonly used as phage receptors and attachment is regulated by either the length of LPS or by specific O-antigen modifications, for phages such as P22 and Sf6 [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. Sequence-diverse Omps are also commonly used as receptors by phages that infect Gram-negative hosts; examples include OmpA, OmpC, OmpF, LamB, FhuA, as well as others [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>-<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>].</p><p id="p-6">Bacteriophage Sf6 is a short-tailed dsDNA virus that belongs to a subgroup of the family <em>Podoviridae</em>, the “P22-like” phages [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>]. Sf6 infection of <em>Shigella flexneri</em> is a two-step process that utilizes both glycans and proteins during infection. First, Sf6 reversibly recognizes and then hydrolyzes LPS via its tailspikes [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. Second, Sf6 interacts irreversibly with a protein receptor to commit to infection [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. Sf6 preferentially uses OmpA, but can also use OmpC when OmpA is absent [<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>]. Sf6 can likely utilize a third, as of yet unidentified receptor, as infection still occurs in the absence of both OmpA and OmpC [<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">25</a>]. Bacteriophage Sf6 has an inherent ability to utilize multiple Omps for infection [<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">25</a>]. Although host range studies have generated mutants of other phages that can switch to utilize alternative receptors when under selection pressure [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>,<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>,<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>], an innate ability to recognize multiple receptor types is not a common phenomenon, making Sf6 somewhat unique.</p><p id="p-7">Our previous work showed that OmpA surface loops mediate Sf6 infection and confer host range [<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>]. Individual amino acid substitutions in OmpA loops result in a range of Sf6 infection efficiencies [<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a>]. In an effort to better understand Sf6 interactions with OmpA, and how variations affect binding, we used biolayer interferometry (BLI) to determine the kinetics of these interactions. BLI is an optical biosensing technique used to measure the kinetic parameters of biomolecular interactions [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>,<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]. It works by tethering one binding partner (the ligand, in this study, whole phage) to a fiber optic sensor tip. The ligand-loaded sensor is then dipped into a sample that contains a known concentration of the binding partner (the analyte, in this study, purified receptor proteins). White light reflects off of two optical layers in the tip, establishing an interference pattern, which is measured by a photodetector. Binding between ligand and analyte causes the distance between optical layers to increase, resulting in a shift in the interference pattern. The shift (in nm) plotted against time in an association-then-dissociation experiment allows for determination of rate and affinity constants. Examination of several different analyte concentrations allows for robust global fits.</p><p id="p-8">Previously published BLI kinetic analyses have used purified host receptors and studied interactions with purified viral receptor binding proteins [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>-<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>], yet no such studies are published to date for bacteriophages. Here, we successfully immobilized intact Sf6 virions by amine crosslinking. To our knowledge, this also represents the first study of whole virion immobilization completed on the BLI platform. We determined the equilibrium dissociation constant of Sf6 to OmpA, and found it is 36 nM. Moreover, we showed that Sf6 bound to five variant OmpAs that demonstrated phenotypic changes [<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">23</a>], yet the resulting kinetic parameters vary only slightly when compared to the native <em>Shigella</em> OmpA protein. These results suggest that the altered infection efficiencies observed in vivo are not solely dependent on the rate at which Sf6 interacts with OmpA.</p></div><div class="section" id="sec-2"><h2 class="">Materials and Methods</h2><div id="sec-3" class="subsection"><h3>Media and strains</h3><p id="p-9">Bacterial growth, plating experiments, and preparations of Sf6 phage stocks were all completed in Lysogeny broth (LB). Bacteriophage Sf6 (clear plaque mutant [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>]) was propagated on <em>ompA</em><sup><em>-</em></sup><em>C</em><sup><em>-</em></sup> <em>S</em>. <em>flexneri</em> (dual ompA and ompC gene knock out), as previously described [<a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">25</a>]. Phage used for <em>in vitro</em> genome ejection experiments were stored in phage buffer (10 mM Tris, pH 7.6 and 10 mM MgCl<sub>2</sub>) and phage used for BLI experiments were stored in NaOAc buffer (10 mM sodium acetate, pH 4.0 and 2 mM MgCl<sub>2</sub>). <em>S</em>. <em>flexneri</em> strain PE577 was used for phage plating experiments [<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>]. OmpA-TM proteins (“TM” = transmembrane portion of OmpA that lacks the periplasmic domain, and has been shown to be sufficient to induce genome ejection <em>in vitro</em>) were expressed from <em>E</em>. <em>coli</em> BL21/DE3/pLysS cells, unfolded in 6M GuHCl, purified using Ni-NTA agarose matrix (Qiagen) in the presence of 6M urea, and then reconstituted by slow dialysis into 1.8 mM Triton X-100, as previously described [<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">25</a>]. The initial concentration of purified OmpA-TM was determined using a Bradford assay. We then created 0.2 mg/mL stock solutions, ran a portion of each on a 15% SDS gel, and quantified the resulting bands by gel densitometry (BIORAD Gel Doc XR+) as previously described [<a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">23</a>] to ensure that the final concentration of variant OmpA-TM dilutions were identical to OmpA-TM<sub><em>S</em>.<em>flex</em></sub>.</p></div><div id="sec-4" class="subsection"><h3>LPS extraction and in vitro genome ejections</h3><p id="p-10">Using a BulldogBio kit, <em>S</em>. <em>flexneri</em> LPS was extracted from PE577 as previously described [<a id="xref-ref-25-7" class="xref-bibr" href="#ref-25">25</a>]. Sf6 was incubated at 25, 30, or 37 °C with purified LPS (0.5 mg/mL) and OmpA-TM<sub><em>S</em>.<em>flex</em></sub> (0.05 mg/mL). The “percent remaining virions” is a measurement for the fraction of the population of phages that have not released their genomes after interaction with LPS and OmpA-TM and was calculated by dividing the plaque forming units (PFUs) in each reaction by the PFUs in buffer. Plates were grown overnight at 30°C.</p></div><div id="sec-5" class="subsection"><h3>Biolayer Interferometry</h3><p id="p-11">Kinetic analyses of variant OmpA-TMs binding to Sf6 phage were performed on a FortéBio (Menlo Park, CA) Octet QK BLI instrument using amine reactive sensors (AR2G) at 25, 30, or 37 °C. All final volumes were 200 µL. A stock of Sf6 phage in 10 mM sodium acetate buffer, pH 4.0 at a titer of 1x10<sup>10</sup> phage/mL was used to tether the phage to the sensor. The AR2G sensors were wetted and activated in 10 mM sulfo-NHS (N-hydroxysulfosuccinimide) and 400 mM EDC (1-ethyl-3-(3 dimethylaminopropyl)carbodiimide hydrochloride) for 300 seconds. Sensors were then dipped for 600 seconds in the phage stock to allow crosslinking, which was followed by quenching in 1M ethanolamine, pH 8.5 for 300 seconds. Baseline was established in 1.8 mM Triton X-100 (diluted in water) over a period of 300 seconds. Sensors were then exposed to various OmpA-TM analytes (ranging from 1,000 nM to 7.8 nM) for 300 seconds to measure association. Dissociation was measured for 300 seconds by dipping the sensors into 1.8 mM Triton X-100. Data were reference-subtracted using the signal from crosslinked phage exposed only to 1.8 mM Triton X-100. Nonspecific binding was measured by exposing a sensor without tethered phage to the highest concentration of OmpA-TM<sub>S.flex</sub> and was found to be negligible. Data were fit using GraphPad Prism 7 (GraphPad Software, La Jolla, CA, USA) and BiaEvaluation Software (GE Healthcare, USA). Experiments were performed in triplicate. Global fits were calculated from each set of experimental data, and overall there was relatively little binding variation between separate titrations.</p></div></div><div class="section" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>Temperature does not significantly change the kinetic parameters of Sf6 and OmpA binding</h3><p id="p-12">We purified the transmembrane domain of <em>S</em>. <em>flexneri</em> OmpA (“OmpA-TM<sub><em>S</em>.<em>flex</em></sub>”[<a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">23</a>]), confirmed we had functional OmpA-TMs by our previously reported assays [<a id="xref-ref-23-8" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-25-8" class="xref-bibr" href="#ref-25">25</a>],and then measured the ability of OmpA-TMs to induce Sf6 genome ejection <em>in vitro</em> prior to performing BLI experiments. For all BLI experiments described herein, the ligand, Sf6, was immobilized on amine reactive (AR2G) sensors. OmpA-TMs reconstituted into detergent micelles were used as analytes. To ensure that sodium acetate, pH 4.0 buffer (a low pH buffer in which phage are not typically stored, but which was necessary for tethering to sensors) had no effect on the phage, we monitored the titer of the phage stock over time, comparing it to phage stored in phage dilution buffer, pH 7.6, and found no significant differences. Moreover, we tested the ability of OmpA-TM<sub><em>S</em>.<em>flex</em></sub> to induce genome ejection of Sf6 stored in NaOAc, pH 4.0 buffer and found it to be similar to previously published results [<a id="xref-ref-23-9" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-25-9" class="xref-bibr" href="#ref-25">25</a>]. In addition, Sf6 binding to OmpA-TM<sub><em>S</em>.<em>flex</em></sub> was specific, as phage Sf6 did not bind to other outer membrane proteins.</p><p id="p-13">To determine kinetic parameters of Sf6 and OmpA-TM<sub><em>S</em>.<em>flex</em></sub> binding, we measured the change in interference patterns over time to generate sensorgrams at 25, 30, and 37 °C (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The generated data were fit in GraphPad prism to a global 1:1 association – then – dissociation model (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Calculated kinetic parameters are shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The analyte concentrations tested ranged from 62.5 nM to 1,000 nM. Consistent with our hypothesis and published results from other bacteriophage and host receptor biosensing work [<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>-<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a>,<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>,<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>], OmpA-TM<sub><em>S</em>.<em>flex</em></sub> bound Sf6 with nM affinity that varied only slightly with changes in temperature. Based on the calculated parameters in OmpA-TM<sub><em>S</em>.<em>flex</em></sub> bound Sf6 with relatively fast-on and slow-off kinetics. Overall, these data suggest that temperature differences do not significantly affect Sf6 binding to OmpA.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/708727/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/708727/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/708727" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-14" class="first-child">Kinetic and equilibrium dissociation constants for Sf6 and OmpA-TM<sub><em>S.flex</em></sub></p><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="BLI sensorgrams are shown for crosslinked Sf6 and varying concentrations of OmpA-TMS.flex analyte 62.5 (purple), 125 (green), 250 (red), 500 (orange), and 1,000 (blue) nM at 25, 30, and 37 &#xB0;C. Reference subtracted raw data are shown as points and global 1:1 association-then-dissociation non-linear fits are shown as solid black lines. Association and dissociation times were 300 s. Residuals are shown below the sensorgrams and are less than 10% of the total signal. Kinetic and equilibrium dissociation constants determined from the sensorgrams are shown in Table 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-281522716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;BLI sensorgrams are shown for crosslinked Sf6 and varying concentrations of OmpA-TM&lt;sub&gt;&lt;em&gt;S&lt;/em&gt;.&lt;em&gt;flex&lt;/em&gt;&lt;/sub&gt; analyte 62.5 (purple), 125 (green), 250 (red), 500 (orange), and 1,000 (blue) nM at 25, 30, and 37 &#xB0;C. Reference subtracted raw data are shown as points and global 1:1 association-then-dissociation non-linear fits are shown as solid black lines. Association and dissociation times were 300 s. Residuals are shown below the sensorgrams and are less than 10% of the total signal. Kinetic and equilibrium dissociation constants determined from the sensorgrams are shown in Table 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F1.medium.gif" width="440" height="121"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F1.medium.gif" width="440" height="121"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-15" class="first-child">BLI sensorgrams are shown for crosslinked Sf6 and varying concentrations of OmpA-TM<sub><em>S</em>.<em>flex</em></sub> analyte 62.5 (purple), 125 (green), 250 (red), 500 (orange), and 1,000 (blue) nM at 25, 30, and 37 °C. Reference subtracted raw data are shown as points and global 1:1 association-then-dissociation non-linear fits are shown as solid black lines. Association and dissociation times were 300 s. Residuals are shown below the sensorgrams and are less than 10% of the total signal. Kinetic and equilibrium dissociation constants determined from the sensorgrams are shown in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Sf6: OmpA-TM binding</h3><p id="p-16">Observed binding was complex. As seen in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, a 1:1 association-then-dissociation model (<em>i</em>.<em>e</em>. one-state fit) fits the data relatively well, though a secondary component is noticeable at higher concentrations. Data were collected from broad ranges of concentrations of OmpA-TM<sub><em>S</em>.<em>flex</em></sub> at 37 °C and fit to several other models including two-state parallel and conformational change models, for which the fits were ambiguous or poor (data not shown). Goodness-of-fit is described by χ<sup>2</sup> [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>]; the lower the χ<sup>2</sup>, the better the model describes the fit of the data. In addition to χ<sup>2</sup>, the one-state model was the only one to pass the F test, a standard statistic for ensuring validity of a kinetic model [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>]. Additionally, equilibrium binding analyses of the data shown in <a id="xref-fig-1-4" class="xref-fig" href="#F1">figure 1</a> yielded K<sub>D</sub>s of 82.3, 99.6, and 89.9 nM (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) which are all essentially indistinguishable from each other and in reasonable proximity to the constants derived from the kinetic models, <em>i</em>.<em>e</em>. the secondary component (seen with all OmpA-TM variants) is minor and probably reflective of some artifact rather than biology, e.g. ligand presentation. Another potential explanation is that detergent micelle size varies within a preparation, yet is usually smaller than the membrane-inserted portion of a single OmpA β-barrel (∼19 kDa) [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>]. Therefore, in some cases OmpA stoichiometry within micelles may be a 1:1, but in other cases, multiple micelles likely aggregate around a single OmpA protein, as reported in [<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>]. This variance likely explains at least part of the complex behavior of our data at high analyte concentrations, and why the residuals are not random at these concentrations.</p></div><div id="sec-9" class="subsection"><h3>Sf6 genome ejection efficiency is highest at physiological temperatures</h3><p id="p-17">We were surprised that the binding kinetics did not change greatly with temperature as phage ejection can often be affected by temperature [<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>-<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>]. Therefore, we tested if Sf6 genome ejection was affected <em>in vitro</em> using our standard assay [<a id="xref-ref-23-10" class="xref-bibr" href="#ref-23">23</a>, <a id="xref-ref-25-10" class="xref-bibr" href="#ref-25">25</a>] and measured the efficiency of genome ejection at 25, 30, and 37 °C (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Reactions were incubated for 10 minutes, which is within the timeframe of the lengths of BLI association phases. Consistent with previously reported data, at 37 °C the majority (&gt;95%) of Sf6 virions have lost their genomes at 10 minutes post initiation of ejection [<a id="xref-ref-23-11" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-25-11" class="xref-bibr" href="#ref-25">25</a>]. However, as temperature decreased, the observed genome ejection efficiency <em>in vitro</em> also decreased. For example, at 30 °C, ∼ 40% of virions have lost their genomes and only ∼ 10% at 25 °C. Therefore, all subsequent BLI experiments performed were completed at 37 °C.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sf6 in vitro genome ejection efficiency increases with temperature. Ejection efficiency of Sf6 incubated at 25, 30, or 37 &#xB0;C for 10 minutes with LPS + OmpA-TMS.flex. &#x201C;Percent remaining virions&#x201D; was calculated as the number of PFUs remaining after incubation at each temperature divided by the number of PFUs when treated with buffer only. Each data point is an average of at least three separate experiments; error bars signify one standard deviation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-281522716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sf6 &lt;em&gt;in vitro&lt;/em&gt; genome ejection efficiency increases with temperature. Ejection efficiency of Sf6 incubated at 25, 30, or 37 &#xB0;C for 10 minutes with LPS + OmpA-TM&lt;sub&gt;&lt;em&gt;S&lt;/em&gt;.&lt;em&gt;flex&lt;/em&gt;&lt;/sub&gt;. &#x201C;Percent remaining virions&#x201D; was calculated as the number of PFUs remaining after incubation at each temperature divided by the number of PFUs when treated with buffer only. Each data point is an average of at least three separate experiments; error bars signify one standard deviation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F2.medium.gif" width="440" height="310"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F2.medium.gif" width="440" height="310"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-18" class="first-child">Sf6 <em>in vitro</em> genome ejection efficiency increases with temperature. Ejection efficiency of Sf6 incubated at 25, 30, or 37 °C for 10 minutes with LPS + OmpA-TM<sub><em>S</em>.<em>flex</em></sub>. “Percent remaining virions” was calculated as the number of PFUs remaining after incubation at each temperature divided by the number of PFUs when treated with buffer only. Each data point is an average of at least three separate experiments; error bars signify one standard deviation.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Sf6 binds different OmpA-TMs at similar affinities</h3><p id="p-19">Previously reported amino acid substitutions in OmpA resulted in altered infection efficiencies of Sf6 ejection <em>in vivo</em> and <em>in vitro</em>, [<a id="xref-ref-23-12" class="xref-bibr" href="#ref-23">23</a>] and it was thought that these changes may be due to differences in binding affinities. We purified various OmpA-TMs: one from <em>E</em>. <em>coli</em>, “OmpA-TM<sub><em>E</em>.<em>coli</em></sub>” and four that deviated by single amino acid substitutions from the native <em>S</em>. <em>flexneri</em> sequence (D66A, N67E, P111E, and N155E). These variants were chosen to represent a broad range of phenotypes [<a id="xref-ref-23-13" class="xref-bibr" href="#ref-23">23</a>]. Here, we measured the kinetics of Sf6 binding to these various receptor types using BLI and calculated the kinetic and affinity constants for each (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Again, some complexity was evident but a simple 1:1 binding model fit the data better than two-state parallel or conformational change models. The only exception was the lowest analyte concentration for N67E (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, magenta line), for which global fits did not converge, and was therefore eliminated from the analysis. The calculated parameters were consistent with fast-on and slow-off kinetics. OmpA-TM<sub><em>E</em>.<em>coli</em></sub> and all <em>S</em>.<em>flexneri</em> OmpA-TM variants bound Sf6 with nM affinity; the K<sub>D</sub>s ranged between 6.9 and 65.4 nM. The kinetic parameters of binding for the variants differed only slightly when compared to those of OmpA-TM<sub><em>S</em>.<em>flex</em></sub>. Moreover, the small differences observed do not correspond to the phenotypes previously reported [<a id="xref-ref-23-14" class="xref-bibr" href="#ref-23">23</a>]. For example, Sf6 infection in <em>Shigella</em> cells expressing <em>E</em>. <em>coli</em> OmpA is ten-fold lower than on cells expressing <em>S</em>. <em>flexneri</em> OmpA. OmpA-TM<sub><em>E</em>.<em>coli</em></sub> is unable to efficiently induce genome ejection in vitro of Sf6 [<a id="xref-ref-23-15" class="xref-bibr" href="#ref-23">23</a>], yet the binding affinities of the two proteins are highly similar 25 and 36 nM. Furthermore, N155E exhibited similar characteristics to <em>E</em>. <em>coli</em> OmpA yet displayed the highest affinity (7 nM). Given the small range of affinities observed in the BLI data collected and the lack of correlation to our previous work, we interpret these kinetic differences as not significant, particularly given the experimental and instrumental set-up in BLI [<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>]. Overall these data show that there are no large kinetic differences in the rates at which Sf6 binds various forms of OmpA.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/708827/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/708827/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/708827" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <p id="p-20" class="first-child">Equilibrium dissociation constants for Sf6 and variant OmpA-TMs at 37 °C.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="BLI sensorgrams are shown for crosslinked Sf6 and varying concentrations of various OmpA-TM analytes 62.5 (purple), 125 (green), 250 (red), 500 (orange), and 1,000 (blue) nM at 37 &#xB0;C. Reference subtracted raw data are shown as points. Association and dissociation times were 300 s. Residuals are shown below the sensorgrams and are less than 10% of the total signal. Kinetic and equilibrium dissociation constants generated from the sensorgrams are shown in Table 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-281522716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;BLI sensorgrams are shown for crosslinked Sf6 and varying concentrations of various OmpA-TM analytes 62.5 (purple), 125 (green), 250 (red), 500 (orange), and 1,000 (blue) nM at 37 &#xB0;C. Reference subtracted raw data are shown as points. Association and dissociation times were 300 s. Residuals are shown below the sensorgrams and are less than 10% of the total signal. Kinetic and equilibrium dissociation constants generated from the sensorgrams are shown in Table 2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F3.medium.gif" width="440" height="100"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F3.medium.gif" width="440" height="100"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-21" class="first-child">BLI sensorgrams are shown for crosslinked Sf6 and varying concentrations of various OmpA-TM analytes 62.5 (purple), 125 (green), 250 (red), 500 (orange), and 1,000 (blue) nM at 37 °C. Reference subtracted raw data are shown as points. Association and dissociation times were 300 s. Residuals are shown below the sensorgrams and are less than 10% of the total signal. Kinetic and equilibrium dissociation constants generated from the sensorgrams are shown in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-11"><h2 class="">Discussion</h2><p id="p-22">To test our hypothesis that phenotypic differences [<a id="xref-ref-23-16" class="xref-bibr" href="#ref-23">23</a>] may be due to differences in binding affinities of Sf6 to OmpA, we purified six versions of OmpA-TM (<em>S</em>. <em>flexneri</em>, <em>E</em>. <em>coli</em>, and single amino acid substitutions: D66A, N67E, P111E, and N155E). To determine the kinetic parameters of Sf6 and OmpA-TMs, whole Sf6 virions were crosslinked to AR2G sensors and we measured changes in the interference of white light using BLI to generate sensorgrams. Consistent with BLI and SPR studies published with purified phage proteins and host cells [<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>] or purified receptor proteins [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>], we determined the binding affinity of Sf6 to OmpA-TM<sub><em>S</em>.<em>flex</em></sub> to be nM in affinity (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Kinetics were fast-on and slow-off and fit a simple one-state model reasonably well. Furthermore, OmpA-TM<sub><em>E</em>.<em>coli</em></sub> and <em>S</em>. <em>flexneri</em> OmpA-TM variants bound Sf6 with similar affinities and their calculated kinetic parameters varied only slightly when compared to OmpA-TM<sub><em>S</em>.<em>flex</em></sub> (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-23">Previously published kinetic analyses performed with BLI for animal viruses and their respective host receptors have shown that binding affinities are in the μM – pM range [<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">30</a>-<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">32</a>]. Surface plasmon resonance (SPR), another optical biosensing technique, has also been used to study virus:host interactions. For example, Bonaparte et al. showed that the equilibrium dissociation constant for Hendra virus attachment glycoprotein to its receptor, human ephrin-B2 is 1 nM [<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>]. Another SPR study showed that purified receptor binding proteins of human coronavirus, Middle East respiratory syndrome (MERS-CoV), and a bat coronavirus HKU4 can bind to human CD26 with K<sub>D</sub>s of 18.4 nM and 35.7 μM, respectively [<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>]. Recently, Marti et al. showed via SPR that the binding affinity of the long tail fiber of bacteriophage S16, the phage tail protein that mediates interaction with the host, and its host <em>Salmonella enterica</em> ssp. <em>enterica</em> is ∼ 5 nM [<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">12</a>].</p><p id="p-24">The data presented herein suggest that the previously reported differences in Sf6 infection efficiencies seen <em>in vivo</em> and the differences in Sf6 genome ejection efficiencies <em>in vitro</em> [<a id="xref-ref-23-17" class="xref-bibr" href="#ref-23">23</a>] are not based solely on the kinetics of receptor binding. There are no significant kinetic differences between Sf6 binding to the various OmpA-TMs, nor are there any significant changes when temperature is varied, even though both factors have been shown to cause changes in infection efficiency. Our results are similar to those in another study in which the authors use SPR and demonstrate that amino acid substitutions in the coronavirus receptor binding protein do not greatly affect the overall kinetic parameters when binding to the human receptor CD26 [<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">35</a>]. Their data support the idea that virus: host recognition is more complex and not dependent solely upon binding affinities. Our work, in combination with the coronavirus data, suggest that this may be a common theme throughout virology, and a phenomenon that may be universally conserved across kingdoms from bacteriophage to eukaryotic viruses. Kinetics alone do not explain the changes in infection efficiencies observed when temperature is varied or when amino acid substitutions are present in the receptor protein. Therefore we hypothesize that conformational changes in the phage upon interaction with receptors are key to effective host recognition.</p><p id="p-25">The current working model for <em>Podoviridae</em> attachment is a three step model [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>]. In the first step the virion binds to LPS reversibly. In the second step there is likely an irreversible interaction with a secondary receptor. Third, the genome is translocated from the phage capsid into the cell concurrent with several conformational changes in virion structure. Hu et al. have shown that bacteriophage T7 [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>] undergoes extensive structural remodeling during infection, particularly in the tail machinery. In summation, we propose a model in which Sf6: Omp receptor recognition is not solely based on kinetics, but likely also involves conformational changes induced when docking to a cell surface (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Sf6 interacts with LPS first via its tailspikes [<a id="xref-ref-25-12" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>] likely coming into contact with the host surface at an angle, as work with a closely related phage, P22, has shown [<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>]. Once Sf6 has cleaved enough LPS repeats [<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>, <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>], and is close enough to the surface of the cell it interacts with its secondary receptor, an Omp [<a id="xref-ref-25-13" class="xref-bibr" href="#ref-25">25</a>]. Upon interaction with Omps by the tail machinery, a conformational change in the phage is likely triggered. Amino acid substitutions in the loops of OmpA may affect the ability of the phage to adopt the correct conformation to promote channel formation, which is necessary to translocate the DNA genome [<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">44</a>, <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>-<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>]. Although more work is necessary to discern a complete understanding of Sf6 (and <em>Podoviridae</em>) infection, the data presented here shed light on the kinetics of Sf6 and OmpA binding, which is an important step during host recognition. All of the work presented here was completed solely with purified OmpA. Our previous work showed that keeping LPS constant, but varying different OmpAs resulted in differences of ejection efficiencies and rates <em>in vitro</em> [<a id="xref-ref-23-18" class="xref-bibr" href="#ref-23">23</a>]. Since both components are required for infection [<a id="xref-ref-23-19" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-25-14" class="xref-bibr" href="#ref-25">25</a>], it may be possible that the LPS helps to “prime” the phage to interact with OmpA, by inducing a subtle, initial conformational change that could induce different interactions with OmpA loop variants. Future work includes building on our current platform and will include a much more complex binding landscape. Ultimately, we hope to expand these studies to liposomes containing LPS and OmpA and/or whole <em>S</em>. <em>flexneri</em> cells to better elucidate the mechanistic properties of the Sf6 infection process.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic showing steps in Sf6 attachment (modified from [44]). Step 1: A virion, likely coming in at an angle, binds to lipopolysaccharide (LPS). Step 2: The tailspike proteins (purple) hydrolyze the LPS bringing the virion closer to the outer membrane (OM) surface, where it then interacts with OmpA. The crystal structure of E.coli OmpA (PDB: 1BXW) is depicted as a ribbon diagram using UCSF Chimera [52]. Interaction with OmpA likely triggers a conformational change in the tail machinery. Step 3: dsDNA likely enters the cell through a channel formed by the tail and the ejection proteins. Schematic is not to scale." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-281522716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic showing steps in Sf6 attachment (modified from [44]). Step 1: A virion, likely coming in at an angle, binds to lipopolysaccharide (LPS). Step 2: The tailspike proteins (purple) hydrolyze the LPS bringing the virion closer to the outer membrane (OM) surface, where it then interacts with OmpA. The crystal structure of &lt;em&gt;E&lt;/em&gt;.&lt;em&gt;coli&lt;/em&gt; OmpA (PDB: 1BXW) is depicted as a ribbon diagram using UCSF Chimera [52]. Interaction with OmpA likely triggers a conformational change in the tail machinery. Step 3: dsDNA likely enters the cell through a channel formed by the tail and the ejection proteins. Schematic is not to scale.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F4.medium.gif" width="440" height="241"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F4.medium.gif" width="440" height="241"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/02/509141/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-26" class="first-child">Schematic showing steps in Sf6 attachment (modified from [<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">44</a>]). Step 1: A virion, likely coming in at an angle, binds to lipopolysaccharide (LPS). Step 2: The tailspike proteins (purple) hydrolyze the LPS bringing the virion closer to the outer membrane (OM) surface, where it then interacts with OmpA. The crystal structure of <em>E</em>.<em>coli</em> OmpA (PDB: 1BXW) is depicted as a ribbon diagram using UCSF Chimera [<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>]. Interaction with OmpA likely triggers a conformational change in the tail machinery. Step 3: dsDNA likely enters the cell through a channel formed by the tail and the ejection proteins. Schematic is not to scale.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-12"><h2 class="">Author Contributions</h2><p id="p-28">All authors contributed towards experimental design and discussion regarding results. Natalia B. Hubbs purified Sf6, LPS, and OmpA-TM proteins and performed <em>in vitro</em> genome ejections. Natalia B. Hubbs, Mareena M. Whisby, and Jonathan L. McMurry collected and analyzed the BLI data. Natalia B. Hubbs wrote the paper with help from the other authors.</p></div><div class="section COI-statement" id="sec-13"><h2 class="">Conflicts of Interest</h2><p id="p-29">The authors declare no conflict of interest.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-27">The authors would like to thank Dr. Kristin Parent (MSU) for collaborative assistance, use of her laboratory and helpful comments on the manuscript, Dr. Xuefei Huang (MSU) and Peng Wang for time on their BLI instrument and Dr. Charles Hoogstraten and Senem Aykul for thoughtful discussion and help with kinetic analysis.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-509141v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Flint</span>,  <span class="cit-name-given-names">S.J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Enquiest</span>,  <span class="cit-name-given-names">L.W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Krug</span>,  <span class="cit-name-given-names">R.M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Racaniello</span>,  <span class="cit-name-given-names">V.R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Skalka</span>,  <span class="cit-name-given-names">A.M.</span></span> <span class="cit-source">Principles of virology, molecular biology, pathogenesis, and control</span>. <span class="cit-publ-name">ASM Press</span>: <span class="cit-pub-date">2000</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.2" data-doi="10.1006/tpbi.2002.1590"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hendrix</span>,  <span class="cit-name-given-names">R.W.</span></span> <span class="cit-article-title">Bacteriophages: Evolution of the majority</span>. <abbr class="cit-jnl-abbrev">Theor Popul Biol</abbr> <span class="cit-pub-date">2002</span>, <span class="cit-vol">61</span>, <span class="cit-fpage">471</span>–<span class="cit-lpage">480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DHendrix%26rft.auinit1%253DR.%2BW.%26rft.volume%253D61%26rft.issue%253D4%26rft.spage%253D471%26rft.epage%253D480%26rft.atitle%253DBacteriophages%253A%2Bevolution%2Bof%2Bthe%2Bmajority.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.2002.1590%26rft_id%253Dinfo%253Apmid%252F12167366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.2002.1590&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12167366&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177739500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Poranen</span>,  <span class="cit-name-given-names">M.M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Daugelavicius</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bamford</span>,  <span class="cit-name-given-names">D.H.</span></span> <span class="cit-article-title">Common principles in viral entry</span>. <abbr class="cit-jnl-abbrev">Annul Rev Microbiol</abbr> <span class="cit-pub-date">2002</span>, <span class="cit-vol">56</span>, <span class="cit-fpage">521</span>–<span class="cit-lpage">538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnul%2BRev%2BMicrobiol%26rft.volume%253D56%26rft.spage%253D521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rakhuba</span>,  <span class="cit-name-given-names">D.V.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kolomiets</span>,  <span class="cit-name-given-names">E.I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dey</span>,  <span class="cit-name-given-names">E.S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Novik</span>,  <span class="cit-name-given-names">G.I.</span></span> <span class="cit-article-title">Bacteriophage receptors, mechanisms of phage adsorption and penetration into host cell</span>. <abbr class="cit-jnl-abbrev">Pol J Microbiol</abbr> <span class="cit-pub-date">2010</span>, <span class="cit-vol">59</span>, <span class="cit-fpage">145</span>–<span class="cit-lpage">155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPolish%2Bjournal%2Bof%2Bmicrobiology%2B%252F%2BPolskie%2BTowarzystwo%2BMikrobiolog%25C3%25B3w%2B%253D%2BThe%2BPolish%2BSociety%2Bof%2BMicrobiologists%26rft.stitle%253DPol%2BJ%2BMicrobiol%26rft.aulast%253DRakhuba%26rft.auinit1%253DD.%2BV.%26rft.volume%253D59%26rft.issue%253D3%26rft.spage%253D145%26rft.epage%253D155%26rft.atitle%253DBacteriophage%2Breceptors%252C%2Bmechanisms%2Bof%2Bphage%2Badsorption%2Band%2Bpenetration%2Binto%2Bhost%2Bcell.%26rft_id%253Dinfo%253Apmid%252F21033576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21033576&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-509141v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chatterjee</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Use of bacteriophage-resistant mutants to study the nature of the bacteriophage receptor site of <em>Staphylococcus aureus</em></span>. <abbr class="cit-jnl-abbrev">J Bacteriol</abbr> <span class="cit-pub-date">1969</span>, <span class="cit-vol">98</span>, <span class="cit-fpage">519</span>–<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DChatterjee%26rft.auinit1%253DA.%2BN.%26rft.volume%253D98%26rft.issue%253D2%26rft.spage%253D519%26rft.epage%253D527%26rft.atitle%253DUse%2Bof%2BBacteriophage-resistant%2BMutants%2Bto%2BStudy%2Bthe%2BNature%2Bof%2Bthe%2BBacteriophage%2BReceptor%2BSite%2Bof%2BStaphylococcus%2Baureus%26rft_id%253Dinfo%253Apmid%252F4239385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6ODoiOTgvMi81MTkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMS8wMi81MDkxNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">6.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-509141v1.6" data-doi="10.1084/jem.145.3.578"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cleary</span>,  <span class="cit-name-given-names">P.P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wannamaker</span>,  <span class="cit-name-given-names">L.W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fisher</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Laible</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-article-title">Studies of the receptor for phage A25 in group A <em>Streptococci</em>: the role of peptidoglycan in reversible adsorption</span>. <abbr class="cit-jnl-abbrev">J Exp Med</abbr> <span class="cit-pub-date">1977</span>, <span class="cit-vol">145</span>, <span class="cit-fpage">578</span>–<span class="cit-lpage">593</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BExperimental%2BMedicine%26rft.stitle%253DJEM%26rft.aulast%253DCleary%26rft.auinit1%253DP.%2BP.%26rft.volume%253D145%26rft.issue%253D3%26rft.spage%253D578%26rft.epage%253D593%26rft.atitle%253DStudies%2Bof%2Bthe%2Breceptor%2Bfor%2Bphage%2BA25%2Bin%2Bgroup%2BA%2Bstreptococci%253A%2Bthe%2Brole%2Bof%2Bpeptidoglycan%2Bin%2Breversible%2Badsorption%26rft_id%253Dinfo%253Adoi%252F10.1084%252Fjem.145.3.578%26rft_id%253Dinfo%253Apmid%252F233903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamVtIjtzOjU6InJlc2lkIjtzOjk6IjE0NS8zLzU3OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAxLzAyLzUwOTE0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-509141v1.7" data-doi="10.1099/00221287-142-4-985"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wendlinger</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Loessner</span>,  <span class="cit-name-given-names">M.J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Scherer</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Bacteriophage receptors on <em>Listeria monocytogenes</em> cells are the N-acetylglucosamine and rhamnose substituents of teichoic acids or the peptidoglycan itself</span>. <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-pub-date">1996</span>, <span class="cit-vol">142</span> (<span class="cit-issue">Pt 4</span>), <span class="cit-fpage">985</span>–<span class="cit-lpage">992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DWendlinger%26rft.auinit1%253DG.%26rft.volume%253D142%26rft.issue%253D4%26rft.spage%253D985%26rft.epage%253D992%26rft.atitle%253DBacteriophage%2Breceptors%2Bon%2BListeria%2Bmonocytogenes%2Bcells%2Bare%2Bthe%2BN-acetylglucosamine%2Band%2Brhamnose%2Bsubstituents%2Bof%2Bteichoic%2Bacids%2Bor%2Bthe%2Bpeptidoglycan%2Bitself%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00221287-142-4-985%26rft_id%253Dinfo%253Apmid%252F8936325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00221287-142-4-985&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8936325&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UH16300035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.8" data-doi="10.1074/jbc.M513625200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sao-Jose</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lhuillier</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lurz</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Melki</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lepault</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Santos</span>,  <span class="cit-name-given-names">M.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tavares</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">The ectodomain of the viral receptor Yueb forms a fiber that triggers ejection of bacteriophage SPP1 DNA</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">281</span>, <span class="cit-fpage">11464</span>–<span class="cit-lpage">11470</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M513625200%26rft_id%253Dinfo%253Apmid%252F16481324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvMTcvMTE0NjQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMS8wMi81MDkxNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.9" data-doi="10.1128/JB.00349-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baptista</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Santos</span>,  <span class="cit-name-given-names">M.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sao-Jose</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">Phage SPP1 reversible adsorption to <em>Bacillus subtilis</em> cell wall teichoic acids accelerates virus recognition of membrane receptor Yueb</span>. <abbr class="cit-jnl-abbrev">J Bacteriol</abbr> <span class="cit-pub-date">2008</span>, <span class="cit-vol">190</span>, <span class="cit-fpage">4989</span>–<span class="cit-lpage">4996</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBacteriol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00349-08%26rft_id%253Dinfo%253Apmid%252F18487323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE5MC8xNC80OTg5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDEvMDIvNTA5MTQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.10" data-doi="10.1073/pnas.58.6.2377"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Young</span>,  <span class="cit-name-given-names">F.E.</span></span> <span class="cit-article-title">Requirement of glucosylated teichoic acid for adsorption of phage in <em>Bacillus subtilis</em> 168</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-pub-date">1967</span>, <span class="cit-vol">58</span>, <span class="cit-fpage">2377</span>–<span class="cit-lpage">2384</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DYoung%26rft.auinit1%253DF.%2BE.%26rft.volume%253D58%26rft.issue%253D6%26rft.spage%253D2377%26rft.epage%253D2384%26rft.atitle%253DRequirement%2Bof%2Bglucosylated%2Bteichoic%2Bacid%2Bfor%2Badsorption%2Bof%2Bphage%2Bin%2BBacillus%2Bsubtilis%2B168.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.58.6.2377%26rft_id%253Dinfo%253Apmid%252F4969329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI1OC82LzIzNzciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMS8wMi81MDkxNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.11" data-doi="10.1074/jbc.M114.614222"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Garcia</span>,  <span class="cit-name-given-names">V.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Pulido-Cid</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Garcia-Doval</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bocanegra</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">van Raaij</span>,  <span class="cit-name-given-names">M.J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Martin-Benito</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cuervo</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Carrascosa</span>,  <span class="cit-name-given-names">J.L.</span></span> <span class="cit-article-title">Conformational changes leading to T7 DNA delivery upon interaction with the bacterial receptor</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">290</span>, <span class="cit-fpage">10038</span>–<span class="cit-lpage">10044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M114.614222%26rft_id%253Dinfo%253Apmid%252F25697363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyOTAvMTYvMTAwMzgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMS8wMi81MDkxNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.12" data-doi="10.1111/mmi.12134"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marti</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zurfluh</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hagens</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Pianezzi</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Klumpp</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Loessner</span>,  <span class="cit-name-given-names">M.J.</span></span> <span class="cit-article-title">Long tail fibres of the novel broad-host-range T-even bacteriophage S16 specifically recognize <em>Salmonella</em> OmpC</span>. <abbr class="cit-jnl-abbrev">Mol Microbiol</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">87</span>, <span class="cit-fpage">818</span>–<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BMicrobiol%26rft.volume%253D87%26rft.spage%253D818%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmmi.12134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mmi.12134&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.13" data-doi="10.1016/S0006-3495(96)79402-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baxa</span>,  <span class="cit-name-given-names">U.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Steinbacher</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Weintraub</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Huber</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Seckler</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Interactions of phage P22 tails with their cellular receptor, <em>Salmonella</em> O-antigen polysaccharide</span>. <abbr class="cit-jnl-abbrev">Biophys J</abbr> <span class="cit-pub-date">1996</span>, <span class="cit-vol">71</span>, <span class="cit-fpage">2040</span>–<span class="cit-lpage">2048</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft.stitle%253DBiophys.%2BJ%26rft.aulast%253DBaxa%26rft.auinit1%253DU%26rft.volume%253D71%26rft.issue%253D4%26rft.spage%253D2040%26rft.epage%253D2048%26rft.atitle%253DInteractions%2Bof%2Bphage%2BP22%2Btails%2Bwith%2Btheir%2Bcellular%2Breceptor%252C%2BSalmonella%2BO-antigen%2Bpolysaccharide.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0006-3495%252896%252979402-X%26rft_id%253Dinfo%253Apmid%252F8889178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0006-3495(96)79402-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8889178&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VK29500035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lindberg</span>,  <span class="cit-name-given-names">A.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wollin</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gemski</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wohlhieter</span>,  <span class="cit-name-given-names">J.A.</span></span> <span class="cit-article-title">Interaction between bacteriophage Sf6 and <em>Shigella flexneri</em></span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-pub-date">1978</span>, <span class="cit-vol">27</span>, <span class="cit-fpage">38</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLindberg%26rft.auinit1%253DA%2BA%26rft.volume%253D27%26rft.issue%253D1%26rft.spage%253D38%26rft.epage%253D44%26rft.atitle%253DInteraction%2Bbetween%2Bbacteriophage%2BSf6%2Band%2BShigella%2Bflexner.%26rft_id%253Dinfo%253Apmid%252F357756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjc6IjI3LzEvMzgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMS8wMi81MDkxNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.15" data-doi="10.1111/j.1574-6968.2007.00778.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">S.G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mahon</span>,  <span class="cit-name-given-names">V.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lambert</span>,  <span class="cit-name-given-names">M.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fagan</span>,  <span class="cit-name-given-names">R.P.</span></span> <span class="cit-article-title">A molecular swiss army knife: OmpA structure, function and expression</span>. <abbr class="cit-jnl-abbrev">FEMS Microbiol Lett</abbr> <span class="cit-pub-date">2007</span>, <span class="cit-vol">273</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol%2BLett%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.2007.00778.x%26rft_id%253Dinfo%253Apmid%252F17559395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-6968.2007.00778.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17559395&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247666900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-509141v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morona</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kramer</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Henning</span>,  <span class="cit-name-given-names">U.</span></span> <span class="cit-article-title">Bacteriophage receptor area of outer membrane protein OmpA of <em>Escherichia coli</em> k-12</span>. <abbr class="cit-jnl-abbrev">J Bacteriol</abbr> <span class="cit-pub-date">1985</span>, <span class="cit-vol">164</span>, <span class="cit-fpage">539</span>–<span class="cit-lpage">543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DMorona%26rft.auinit1%253DR%26rft.volume%253D164%26rft.issue%253D2%26rft.spage%253D539%26rft.epage%253D543%26rft.atitle%253DBacteriophage%2Breceptor%2Barea%2Bof%2Bouter%2Bmembrane%2Bprotein%2BOmpA%2Bof%2BEscherichia%2Bcoli%2BK-12.%26rft_id%253Dinfo%253Apmid%252F3902787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTY0LzIvNTM5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDEvMDIvNTA5MTQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.17" data-doi="10.1006/jmbi.1994.1424"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hashemolhosseini</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mutschler</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Henning</span>,  <span class="cit-name-given-names">U.</span></span> <span class="cit-article-title">Alterations of receptor specificities of coliphages of the T2 family</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-pub-date">1994</span>, <span class="cit-vol">240</span>, <span class="cit-fpage">105</span>–<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DHashemolhosseini%26rft.auinit1%253DS.%26rft.volume%253D240%26rft.issue%253D2%26rft.spage%253D105%26rft.epage%253D110%26rft.atitle%253DAlterations%2Bof%2Breceptor%2Bspecificities%2Bof%2Bcoliphages%2Bof%2Bthe%2BT2%2Bfamily.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1994.1424%26rft_id%253Dinfo%253Apmid%252F8027994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1994.1424&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8027994&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.18" data-doi="10.1006/jmbi.1994.1529"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hashemolhosseini</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Montag</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kramer</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Henning</span>,  <span class="cit-name-given-names">U.</span></span> <span class="cit-article-title">Determinants of receptor specificity of coliphages of the T4 family. A chaperone alters the host range</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-pub-date">1994</span>, <span class="cit-vol">241</span>, <span class="cit-fpage">524</span>–<span class="cit-lpage">533</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DHashemolhosseini%26rft.auinit1%253DS.%26rft.volume%253D241%26rft.issue%253D4%26rft.spage%253D524%26rft.epage%253D533%26rft.atitle%253DDeterminants%2Bof%2Breceptor%2Bspecificity%2Bof%2Bcoliphages%2Bof%2Bthe%2BT4%2Bfamily.%2BA%2Bchaperone%2Balters%2Bthe%2Bhost%2Brange.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1994.1529%26rft_id%253Dinfo%253Apmid%252F8057378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1994.1529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8057378&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-509141v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Randall-Hazelbauer</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Schwartz</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Isolation of the bacteriophage lambda receptor from <em>Escherichia coli</em></span>. <abbr class="cit-jnl-abbrev">J Bacteriol</abbr> <span class="cit-pub-date">1973</span>, <span class="cit-vol">116</span>, <span class="cit-fpage">1436</span>–<span class="cit-lpage">1446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DRandall-Hazelbauer%26rft.auinit1%253DL.%26rft.volume%253D116%26rft.issue%253D3%26rft.spage%253D1436%26rft.epage%253D1446%26rft.atitle%253DIsolation%2Bof%2Bthe%2BBacteriophage%2BLambda%2BReceptor%2Bfrom%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Apmid%252F4201774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjExNi8zLzE0MzYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMS8wMi81MDkxNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">20.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-509141v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hantke</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Braun</span>,  <span class="cit-name-given-names">V.</span></span> <span class="cit-article-title">Functional interaction of the <em>TonA/TonB</em> receptor system in <em>Escherichia coli</em></span>. <abbr class="cit-jnl-abbrev">J Bacteriol</abbr> <span class="cit-pub-date">1978</span>, <span class="cit-vol">135</span>, <span class="cit-fpage">190</span>–<span class="cit-lpage">197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DHantke%26rft.auinit1%253DK.%26rft.volume%253D135%26rft.issue%253D1%26rft.spage%253D190%26rft.epage%253D197%26rft.atitle%253DFunctional%2BInteraction%2Bof%2Bthe%2BtonA%252FtonB%2BReceptor%2BSystem%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Apmid%252F353030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTM1LzEvMTkwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDEvMDIvNTA5MTQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.21" data-doi="10.1016/0014-5793(73)80707-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Braun</span>,  <span class="cit-name-given-names">V.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wolff</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">Characterization of the receptor protein for phage T5 and colicin M in the outer membrane of <em>E. coli</em> B</span>. <abbr class="cit-jnl-abbrev">FEBS Lett</abbr> <span class="cit-pub-date">1973</span>, <span class="cit-vol">34</span>, <span class="cit-fpage">77</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DBraun%26rft.auinit1%253DV.%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D80%26rft.atitle%253DCharacterization%2Bof%2Bthe%2Breceptor%2Bprotein%2Bfor%2Bphage%2BT5%2Band%2Bcolicin%2BM%2Bin%2Bthe%2Bouter%2Bmembrane%2Bof%2BE.%2Bcoli%2BB.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0014-5793%252873%252980707-0%26rft_id%253Dinfo%253Apmid%252F4580999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0014-5793(73)80707-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4580999&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1973Q368000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.22" data-doi="10.1128/jb.176.3.733-747.1994"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morona</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mavris</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fallarino</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Manning</span>,  <span class="cit-name-given-names">P.A.</span></span> <span class="cit-article-title">Characterization of the rfc region of <em>Shigella flexneri</em></span>. <abbr class="cit-jnl-abbrev">J Bacteriol</abbr> <span class="cit-pub-date">1994</span>, <span class="cit-vol">176</span>, <span class="cit-fpage">733</span>–<span class="cit-lpage">747</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DMorona%26rft.auinit1%253DR.%26rft.volume%253D176%26rft.issue%253D3%26rft.spage%253D733%26rft.epage%253D747%26rft.atitle%253DCharacterization%2Bof%2Bthe%2Brfc%2Bregion%2Bof%2BShigella%2Bflexneri.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.176.3.733-747.1994%26rft_id%253Dinfo%253Apmid%252F7507920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTc2LzMvNzMzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDEvMDIvNTA5MTQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.23" data-doi="10.1016/j.jmb.2015.03.012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Porcek</span>,  <span class="cit-name-given-names">N.B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Parent</span>,  <span class="cit-name-given-names">K.N.</span></span> <span class="cit-article-title">Key residues of <em>S. flexneri</em> OmpA mediate infection by bacteriophage Sf6</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">427</span>, <span class="cit-fpage">1964</span>–<span class="cit-lpage">1976</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BMol%2BBiol%26rft.volume%253D427%26rft.spage%253D1964%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2015.03.012%26rft_id%253Dinfo%253Apmid%252F25816773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2015.03.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25816773&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.24" data-doi="10.1074/jbc.M205294200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Freiberg</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Morona</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Van den Bosch</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jung</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Behlke</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Carlin</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Seckler</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Baxa</span>,  <span class="cit-name-given-names">U.</span></span> <span class="cit-article-title">The tailspike protein of <em>Shigella</em> phage Sf6. A structural homolog of <em>Salmonella</em> phage P22 tailspike protein without sequence similarity in the beta-helix domain</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-pub-date">2003</span>, <span class="cit-vol">278</span>, <span class="cit-fpage">1542</span>–<span class="cit-lpage">1548</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M205294200%26rft_id%253Dinfo%253Apmid%252F12424253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzgvMy8xNTQyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDEvMDIvNTA5MTQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.25" data-doi="10.1111/mmi.12536"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Parent</span>,  <span class="cit-name-given-names">K.N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Erb</span>,  <span class="cit-name-given-names">M.L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cardone</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Nguyen</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gilcrease</span>,  <span class="cit-name-given-names">E.B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Porcek</span>,  <span class="cit-name-given-names">N.B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Pogliano</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Baker</span>,  <span class="cit-name-given-names">T.S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Casjens</span>,  <span class="cit-name-given-names">S.R.</span></span> <span class="cit-article-title">OmpA and OmpC are critical host factors for bacteriophage Sf6 entry in <em>Shigella</em></span>. <abbr class="cit-jnl-abbrev">Mol Microbiol</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">92</span>, <span class="cit-fpage">47</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BMicrobiol%26rft.volume%253D92%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmmi.12536%26rft_id%253Dinfo%253Apmid%252F24673644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mmi.12536&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24673644&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morona</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Henning</span>,  <span class="cit-name-given-names">U.</span></span> <span class="cit-article-title">Host range mutants of bacteriophage Ox2 can use two different outer membrane proteins of <em>Escherichia coli</em> K-12 as receptors</span>. <abbr class="cit-jnl-abbrev">J Bacteriol</abbr> <span class="cit-pub-date">1984</span>, <span class="cit-vol">159</span>, <span class="cit-fpage">579</span>–<span class="cit-lpage">582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DMorona%26rft.auinit1%253DR%26rft.volume%253D159%26rft.issue%253D2%26rft.spage%253D579%26rft.epage%253D582%26rft.atitle%253DHost%2Brange%2Bmutants%2Bof%2Bbacteriophage%2BOx2%2Bcan%2Buse%2Btwo%2Bdifferent%2Bouter%2Bmembrane%2Bproteins%2Bof%2BEscherichia%2Bcoli%2BK-12%2Bas%2Breceptors.%26rft_id%253Dinfo%253Apmid%252F6378883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTU5LzIvNTc5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDEvMDIvNTA5MTQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.27" data-doi="10.1016/0022-2836(89)90245-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drexler</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Riede</span>,  <span class="cit-name-given-names">I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Montag</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Eschbach</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Henning</span>,  <span class="cit-name-given-names">U.</span></span> <span class="cit-article-title">Receptor specificity of the <em>Escherichia coli</em> T-even type phage Ox2</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-pub-date">1989</span>, <span class="cit-vol">207</span>, <span class="cit-fpage">797</span>–<span class="cit-lpage">803</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DDrexler%26rft.auinit1%253DK.%26rft.volume%253D207%26rft.issue%253D4%26rft.spage%253D797%26rft.epage%253D803%26rft.atitle%253DReceptor%2Bspecificity%2Bof%2Bthe%2BEscherichia%2Bcoli%2BT-even%2Btype%2Bphage%2BOx2.%2BMutational%2Balterations%2Bin%2Bhost%2Brange%2Bmutants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252889%252990245-3%26rft_id%253Dinfo%253Apmid%252F2547973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(89)90245-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2547973&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AD06200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Concepcion</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Witte</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wartchow</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Choo</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yao</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Persson</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Heidecker</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Label-free detection of biomolecular interactions using biolayer interferometry for kinetic characterization</span>. <abbr class="cit-jnl-abbrev">Comb Chem High Throughput Screen</abbr> <span class="cit-pub-date">2009</span>, <span class="cit-vol">8</span>, <span class="cit-fpage">791</span>–<span class="cit-lpage">800</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComb%2BChem%2BHigh%2BThroughput%2BScreen%26rft.volume%253D8%26rft.spage%253D791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.29" data-doi="10.1016/j.ab.2008.03.035"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Abdiche</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Malashock</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Pinkerton</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Pons</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Determining kinetics and affinities of protein interactions using a parallel real-time label-free biosensor, the octet</span>. <abbr class="cit-jnl-abbrev">Anal Biochem</abbr> <span class="cit-pub-date">2008</span>, <span class="cit-vol">377</span>, <span class="cit-fpage">209</span>–<span class="cit-lpage">217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2Bbiochemistry%26rft.stitle%253DAnal%2BBiochem%26rft.aulast%253DAbdiche%26rft.auinit1%253DY.%26rft.volume%253D377%26rft.issue%253D2%26rft.spage%253D209%26rft.epage%253D217%26rft.atitle%253DDetermining%2Bkinetics%2Band%2Baffinities%2Bof%2Bprotein%2Binteractions%2Busing%2Ba%2Bparallel%2Breal-time%2Blabel-free%2Bbiosensor%252C%2Bthe%2BOctet.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ab.2008.03.035%26rft_id%253Dinfo%253Apmid%252F18405656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ab.2008.03.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18405656&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255918000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.30" data-doi="10.1016/j.chom.2015.02.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">de Vries</span>,  <span class="cit-name-given-names">R.P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tzarum</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">McBride</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Paulson</span>,  <span class="cit-name-given-names">J.C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">I.A.</span></span> <span class="cit-article-title">A human-infecting H10N8 influenza virus retains a strong preference for avian-type receptors</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">17</span>, <span class="cit-fpage">377</span>–<span class="cit-lpage">384</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft.volume%253D17%26rft.spage%253D377%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2015.02.006%26rft_id%253Dinfo%253Apmid%252F25766296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2015.02.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25766296&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">31.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-509141v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Carney</span>,  <span class="cit-name-given-names">P.J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">J.C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Villanueva</span>,  <span class="cit-name-given-names">J.M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Stevens</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Structure and receptor binding preferences of recombinant human a(H3N2) virus hemagglutinins</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">477</span>, <span class="cit-fpage">18</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D477%26rft.spage%253D18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.32" data-doi="10.1038/nature12372"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xiong</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">S.R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Haire</span>,  <span class="cit-name-given-names">L.F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wharton</span>,  <span class="cit-name-given-names">S.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Daniels</span>,  <span class="cit-name-given-names">R.S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bennett</span>,  <span class="cit-name-given-names">M.S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">McCauley</span>,  <span class="cit-name-given-names">J.W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Collins</span>,  <span class="cit-name-given-names">P.J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Walker</span>,  <span class="cit-name-given-names">P.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Skehel</span>,  <span class="cit-name-given-names">J.J.</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Receptor binding by an H7N9 influenza virus from humans</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">499</span>, <span class="cit-fpage">496</span>–<span class="cit-lpage">499</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D499%26rft.spage%253D496%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12372%26rft_id%253Dinfo%253Apmid%252F23787694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12372&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23787694&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322157900044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.33" data-doi="10.1016/j.jmb.2004.03.068"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Casjens</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Winn-Stapley</span>,  <span class="cit-name-given-names">D.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gilcrease</span>,  <span class="cit-name-given-names">E.B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Morona</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kuhlewein</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chua</span>,  <span class="cit-name-given-names">J.E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Manning</span>,  <span class="cit-name-given-names">P.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Inwood</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Clark</span>,  <span class="cit-name-given-names">A.J.</span></span> <span class="cit-article-title">The chromosome of <em>Shigella flexneri</em> bacteriophage Sf6: Complete nucleotide sequence, genetic mosaicism, and DNA packaging</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-pub-date">2004</span>, <span class="cit-vol">339</span>, <span class="cit-fpage">379</span>–<span class="cit-lpage">394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DCasjens%26rft.auinit1%253DS.%26rft.volume%253D339%26rft.issue%253D2%26rft.spage%253D379%26rft.epage%253D394%26rft.atitle%253DThe%2Bchromosome%2Bof%2BShigella%2Bflexneri%2Bbacteriophage%2BSf6%253A%2Bcomplete%2Bnucleotide%2Bsequence%252C%2Bgenetic%2Bmosaicism%252C%2Band%2BDNA%2Bpackaging.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2004.03.068%26rft_id%253Dinfo%253Apmid%252F15136040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2004.03.068&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15136040&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221561800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.34" data-doi="10.1073/pnas.0504887102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bonaparte</span>,  <span class="cit-name-given-names">M.I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dimitrov</span>,  <span class="cit-name-given-names">A.S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bossart</span>,  <span class="cit-name-given-names">K.N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Crameri</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mungall</span>,  <span class="cit-name-given-names">B.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bishop</span>,  <span class="cit-name-given-names">K.A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Choudhry</span>,  <span class="cit-name-given-names">V.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dimitrov</span>,  <span class="cit-name-given-names">D.S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Eaton</span>,  <span class="cit-name-given-names">B.T.</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Ephrin-B2 ligand is a functional receptor for hendra virus and nipah virus</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad of Sci USA</abbr> <span class="cit-pub-date">2005</span>, <span class="cit-vol">102</span>, <span class="cit-fpage">10652</span>–<span class="cit-lpage">10657</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2Bof%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0504887102%26rft_id%253Dinfo%253Apmid%252F15998730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzMwLzEwNjUyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDEvMDIvNTA5MTQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.35" data-doi="10.1016/j.chom.2014.08.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Q.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Qi</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xuan</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wan</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ji</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Bat origins of MERS-CoV supported by bat coronavirus HKU4 usage of human receptor CD26</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">16</span>, <span class="cit-fpage">328</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft.volume%253D16%26rft.spage%253D328%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2014.08.009%26rft_id%253Dinfo%253Apmid%252F25211075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2014.08.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25211075&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Singh</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kroe-Barrett</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Comparison of biosensor platforms in the evaluation of high affinity antibody-antigen binding kinetics</span>. <abbr class="cit-jnl-abbrev">Anal Biochem</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">508</span>, <span class="cit-fpage">78</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnal%2BBiochem%26rft.volume%253D508%26rft.spage%253D78%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.37" data-doi="10.1042/bj2110023"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Burguillo</span>,  <span class="cit-name-given-names">F.J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wright</span>,  <span class="cit-name-given-names">A.J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bardsley</span>,  <span class="cit-name-given-names">W.G.</span></span> <span class="cit-article-title">Use of the F test for determining the degree of enzyme-kinetic and ligand-binding data. A monte carlo simulation study</span>. <abbr class="cit-jnl-abbrev">Biochem J</abbr> <span class="cit-pub-date">1983</span>, <span class="cit-vol">211</span>, <span class="cit-fpage">23</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2BJournal%26rft.stitle%253DBiochem.%2BJ.%26rft.aulast%253DBurguillo%26rft.auinit1%253DF.%2BJ.%26rft.volume%253D211%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D34%26rft.atitle%253DUse%2Bof%2Bthe%2BF%2Btest%2Bfor%2Bdetermining%2Bthe%2Bdegree%2Bof%2Benzyme-kinetic%2Band%2Bligand-binding%2Bdata.%2BA%2BMonte%2BCarlo%2Bsimulation%2Bstudy%26rft_id%253Dinfo%253Adoi%252F10.1042%252Fbj2110023%26rft_id%253Dinfo%253Apmid%252F6870821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6ODoiMjExLzEvMjMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMS8wMi81MDkxNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.38" data-doi="10.1110/ps.8.10.2065"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kleinschmidt</span>,  <span class="cit-name-given-names">J.H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wiener</span>,  <span class="cit-name-given-names">M.C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tamm</span>,  <span class="cit-name-given-names">L.K.</span></span> <span class="cit-article-title">Outer membrane protein a of <em>E. coli</em> folds into detergent micelles, but not in the presence of monomeric detergent</span>. <abbr class="cit-jnl-abbrev">Protein Science</abbr> <span class="cit-pub-date">1999</span>, <span class="cit-vol">8</span>, <span class="cit-fpage">2065</span>–<span class="cit-lpage">2071</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%26rft.stitle%253DProtein%2BSci.%26rft.aulast%253DKleinschmidt%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D2065%26rft.epage%253D2071%26rft.atitle%253DOuter%2Bmembrane%2Bprotein%2BA%2Bof%2BE.%2Bcoli%2Bfolds%2Binto%2Bdetergent%2Bmicelles%252C%2Bbut%2Bnot%2Bin%2Bthe%2Bpresence%2Bof%2Bmonomeric%2Bdetergent%26rft_id%253Dinfo%253Adoi%252F10.1110%252Fps.8.10.2065%26rft_id%253Dinfo%253Apmid%252F10548052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1110/ps.8.10.2065&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10548052&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083189400019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.39" data-doi="10.1111/j.1365-2958.2012.08006.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andres</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Roske</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Doering</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Heinemann</span>,  <span class="cit-name-given-names">U.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Seckler</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Barbirz</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Tail morphology controls DNA release in two <em>Salmonella</em> phages with one lipopolysaccharide receptor recognition system</span>. <abbr class="cit-jnl-abbrev">Mol Microbiol</abbr> <span class="cit-pub-date">2012</span>, <span class="cit-vol">83</span>, <span class="cit-fpage">1244</span>–<span class="cit-lpage">1253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DAndres%26rft.auinit1%253DD.%26rft.volume%253D83%26rft.issue%253D6%26rft.spage%253D1244%26rft.epage%253D1253%26rft.atitle%253DTail%2Bmorphology%2Bcontrols%2BDNA%2Brelease%2Bin%2Btwo%2BSalmonella%2Bphages%2Bwith%2Bone%2Blipopolysaccharide%2Breceptor%2Brecognition%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2012.08006.x%26rft_id%253Dinfo%253Apmid%252F22364412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2012.08006.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22364412&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">40.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-509141v1.40" data-doi="10.1529/biophysj.104.048785"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Frutos</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Letellier</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Raspaud</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">DNA ejection from bacteriophage T5: Analysis of the kinetics and energetics</span>. <abbr class="cit-jnl-abbrev">Biophys J</abbr> <span class="cit-pub-date">2005</span>, <span class="cit-vol">88</span>, <span class="cit-fpage">1364</span>–<span class="cit-lpage">1370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophys%2BJ%26rft_id%253Dinfo%253Adoi%252F10.1529%252Fbiophysj.104.048785%26rft_id%253Dinfo%253Apmid%252F15542548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1529/biophysj.104.048785&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15542548&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226750800053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-509141v1.41" data-doi="10.1128/jb.177.3.694-698.1995"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Killmann</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Videnov</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jung</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Schwarz</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Braun</span>,  <span class="cit-name-given-names">V.</span></span> <span class="cit-article-title">Identification of receptor binding sites by competitive peptide mapping: Phages T1, T5, and Phi 80 and colicin M bind to the gating loop of Fhua</span>. <abbr class="cit-jnl-abbrev">J Bacteriol</abbr> <span class="cit-pub-date">1995</span>, <span class="cit-vol">177</span>, <span class="cit-fpage">694</span>–<span class="cit-lpage">698</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DKillmann%26rft.auinit1%253DH.%26rft.volume%253D177%26rft.issue%253D3%26rft.spage%253D694%26rft.epage%253D698%26rft.atitle%253DIdentification%2Bof%2Breceptor%2Bbinding%2Bsites%2Bby%2Bcompetitive%2Bpeptide%2Bmapping%253A%2Bphages%2BT1%252C%2BT5%252C%2Band%2Bphi%2B80%2Band%2Bcolicin%2BM%2Bbind%2Bto%2Bthe%2Bgating%2Bloop%2Bof%2BFhuA.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.177.3.694-698.1995%26rft_id%253Dinfo%253Apmid%252F7836303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTc3LzMvNjk0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDEvMDIvNTA5MTQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.42" data-doi="10.1016/0042-6822(76)90320-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mackay</span>,  <span class="cit-name-given-names">D.J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bode</span>,  <span class="cit-name-given-names">V.C.</span></span> <span class="cit-article-title">Events in lambda injection between phage adsorption and DNA entry</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-pub-date">1976</span>, <span class="cit-vol">72</span>, <span class="cit-fpage">154</span>–<span class="cit-lpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DMackay%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D72%26rft.issue%253D1%26rft.spage%253D154%26rft.epage%253D166%26rft.atitle%253DEvents%2Bin%2Blambda%2Binjection%2Bbetween%2Bphage%2Badsorption%2Band%2BDNA%2Bentry.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252876%252990320-2%26rft_id%253Dinfo%253Apmid%252F779240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(76)90320-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=779240&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1976BX75400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.43" data-doi="10.1128/mBio.01781-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Legrand</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Collins</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Blangy</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Murphy</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Spinelli</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gutierrez</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Richet</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kellenberger</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Desmyter</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Mahony</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The atomic structure of the phage Tuc2009 baseplate tripod suggests that host recognition involves two different carbohydrate binding modules</span>. <abbr class="cit-jnl-abbrev">mBio</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">7</span>, <span class="cit-fpage">e01781</span>–<span class="cit-lpage">01715</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmBio%26rft.volume%253D7%26rft.spage%253De01781%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.01781-15%26rft_id%253Dinfo%253Apmid%252F26814179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.01781-15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26814179&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Casjens</span>,  <span class="cit-name-given-names">S.R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Molineux</span>,  <span class="cit-name-given-names">I.J.</span></span> <span class="cit-article-title">Short noncontractile tail machines: adsorption and DNA delivery by podoviruses</span>. <abbr class="cit-jnl-abbrev">In Advances in experimental medicine and biology</abbr>, <span class="cit-pub-date">2012</span>; Vol. <span class="cit-vol">726</span>, pp <span class="cit-fpage">143</span>–<span class="cit-lpage">179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIn%2BAdvances%2Bin%2Bexperimental%2Bmedicine%2Band%2Bbiology%26rft.volume%253D726%26rft.spage%253D143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.45" data-doi="10.1126/science.1231887"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Margolin</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Molineux</span>,  <span class="cit-name-given-names">I.J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">The bacteriophage T7 virion undergoes extensive structural remodeling during infection</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">339</span>, <span class="cit-fpage">576</span>–<span class="cit-lpage">579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1231887%26rft_id%253Dinfo%253Apmid%252F23306440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzkvNjExOS81NzYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMS8wMi81MDkxNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.46" data-doi="10.1016/j.str.2008.01.019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Muller</span>,  <span class="cit-name-given-names">J.J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Barbirz</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Heinle</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Freiberg</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Seckler</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Heinemann</span>,  <span class="cit-name-given-names">U.</span></span> <span class="cit-article-title">An intersubunit active site between supercoiled parallel beta helices in the trimeric tailspike endorhamnosidase of <em>Shigella flexner</em>i phage Sf6</span>. <abbr class="cit-jnl-abbrev">Structure</abbr> <span class="cit-pub-date">2008</span>, <span class="cit-vol">16</span>, <span class="cit-fpage">766</span>–<span class="cit-lpage">775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%2B%2528London%2529%26rft.stitle%253DStructure%2B%2528London%2529%26rft.aulast%253DMuller%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D766%26rft.epage%253D775%26rft.atitle%253DAn%2Bintersubunit%2Bactive%2Bsite%2Bbetween%2Bsupercoiled%2Bparallel%2Bbeta%2Bhelices%2Bin%2Bthe%2Btrimeric%2Btailspike%2Bendorhamnosidase%2Bof%2BShigella%2Bflexneri%2BPhage%2BSf6.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2008.01.019%26rft_id%253Dinfo%253Apmid%252F18462681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.str.2008.01.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18462681&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tu</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Molineux</span>,  <span class="cit-name-given-names">I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Visualizing infection initiation of bacteriophage P22 by cryo-electron tomography</span>. <abbr class="cit-jnl-abbrev">Biophys J</abbr>, <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.48" data-doi="10.1074/jbc.M110.169003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andres</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hanke</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Baxa</span>,  <span class="cit-name-given-names">U.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Seul</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Barbirz</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Seckler</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Tailspike interactions with lipopolysaccharide effect DNA ejection from phage P22 particles <em>in vitro</em></span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-pub-date">2010</span>, <span class="cit-vol">285</span>, <span class="cit-fpage">36768</span>–<span class="cit-lpage">36775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M110.169003%26rft_id%253Dinfo%253Apmid%252F20817910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvNDcvMzY3NjgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMS8wMi81MDkxNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Leavitt</span>,  <span class="cit-name-given-names">J.C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gilcrease</span>,  <span class="cit-name-given-names">E.B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Motwani</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Teschke</span>,  <span class="cit-name-given-names">C.M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Casjens</span>,  <span class="cit-name-given-names">S.R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Steven</span>,  <span class="cit-name-given-names">A.C.</span></span> <span class="cit-article-title">Localization of the houdinisome (ejection proteins) inside the bacteriophage P22 virion by bubblegram imaging</span>. <abbr class="cit-jnl-abbrev">mBio</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">7</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">50.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-509141v1.50" data-doi="10.1074/jbc.M115.696260"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bhardwaj</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sankhala</span>,  <span class="cit-name-given-names">R.S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Olia</span>,  <span class="cit-name-given-names">A.S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Brooke</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Casjens</span>,  <span class="cit-name-given-names">S.R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Taylor</span>,  <span class="cit-name-given-names">D.J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Prevelige</span>,  <span class="cit-name-given-names">P.E.</span>,   <span class="cit-name-suffix">Jr.</span>.</span>; <span class="cit-auth"><span class="cit-name-surname">Cingolani</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Structural plasticity of the protein plug that traps newly packaged genomes in <em>Podoviridae</em> virions</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">291</span>, <span class="cit-fpage">215</span>–<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M115.696260%26rft_id%253Dinfo%253Apmid%252F26574546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjk6IjI5MS8xLzIxNSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAxLzAyLzUwOTE0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.51" data-doi="10.1016/j.sbi.2013.10.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bhardwaj</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Olia</span>,  <span class="cit-name-given-names">A.S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cingolani</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Architecture of viral genome-delivery molecular machines</span>. <abbr class="cit-jnl-abbrev">Curr Opin Struct Biol</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">25</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BStruct%2BBiol%26rft.volume%253D25%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.sbi.2013.10.005%26rft_id%253Dinfo%253Apmid%252F24878339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.sbi.2013.10.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24878339&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-509141v1.52" data-doi="10.1002/jcc.20084"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pettersen</span>,  <span class="cit-name-given-names">E.F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Goddard</span>,  <span class="cit-name-given-names">T.D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Couch</span>,  <span class="cit-name-given-names">G.S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Greenblatt</span>,  <span class="cit-name-given-names">D.M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Meng</span>,  <span class="cit-name-given-names">E.C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ferrin</span>,  <span class="cit-name-given-names">T.E.</span></span> <span class="cit-article-title">UCSF chimera--a visualization system for exploratory research and analysis</span>. <abbr class="cit-jnl-abbrev">J Comput Chem</abbr> <span class="cit-pub-date">2004</span>, <span class="cit-vol">25</span>, <span class="cit-fpage">1605</span>–<span class="cit-lpage">1612</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DPettersen%26rft.auinit1%253DE.%2BF.%26rft.volume%253D25%26rft.issue%253D13%26rft.spage%253D1605%26rft.epage%253D1612%26rft.atitle%253DUCSF%2BChimera--a%2Bvisualization%2Bsystem%2Bfor%2Bexploratory%2Bresearch%2Band%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.20084%26rft_id%253Dinfo%253Apmid%252F15264254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.20084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15264254&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F02%2F509141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223379100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/509141v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F509141v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/509349v1" title="Nano-enhanced optical gene delivery to retinal degenerated mice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 02, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/509141v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/509141v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/509141v1" class="active" data-icon-position="" data-hide-link-title="0">Kinetic Analysis of Bacteriophage Sf6 Binding to Outer Membrane Protein A Using Whole Virions</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/150617" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-CL5nPKDrvUQa8uklWTKc2d8OxZwaE1fvYmcf4gT9pZw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="150617" id="node150617--2" data-pisa="biorxiv;509141v1" data-pisa-master="biorxiv;509141" data-seqnum="150617" data-apath="/biorxiv/early/2019/01/02/509141.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Kinetic Analysis of Bacteriophage Sf6 Binding to Outer Membrane Protein A Using Whole Virions</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Natalia B.</span> <span class="nlm-surname">Hubbs</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mareena M.</span> <span class="nlm-surname">Whisby-Pitts</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jonathan L.</span> <span class="nlm-surname">McMurry</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">509141; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/509141 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/509141v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/509141v1&amp;title=Kinetic%20Analysis%20of%20Bacteriophage%20Sf6%20Binding%20to%20Outer%20Membrane%20Protein%20A%20Using%20Whole%20Virions" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/509141v1&amp;title=Kinetic%20Analysis%20of%20Bacteriophage%20Sf6%20Binding%20to%20Outer%20Membrane%20Protein%20A%20Using%20Whole%20Virions" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/509141v1&amp;text=Kinetic%20Analysis%20of%20Bacteriophage%20Sf6%20Binding%20to%20Outer%20Membrane%20Protein%20A%20Using%20Whole%20Virions" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/509141v1&amp;title=Kinetic%20Analysis%20of%20Bacteriophage%20Sf6%20Binding%20to%20Outer%20Membrane%20Protein%20A%20Using%20Whole%20Virions" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/509141v1&amp;t=Kinetic%20Analysis%20of%20Bacteriophage%20Sf6%20Binding%20to%20Outer%20Membrane%20Protein%20A%20Using%20Whole%20Virions" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/509141v1&amp;title=Kinetic%20Analysis%20of%20Bacteriophage%20Sf6%20Binding%20to%20Outer%20Membrane%20Protein%20A%20Using%20Whole%20Virions" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/509141v1&amp;title=Kinetic%20Analysis%20of%20Bacteriophage%20Sf6%20Binding%20to%20Outer%20Membrane%20Protein%20A%20Using%20Whole%20Virions" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="150617" id="node150617--3" data-pisa="biorxiv;509141v1" data-pisa-master="biorxiv;509141" data-seqnum="150617" data-apath="/biorxiv/early/2019/01/02/509141.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Kinetic Analysis of Bacteriophage Sf6 Binding to Outer Membrane Protein A Using Whole Virions</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Natalia B.</span> <span class="nlm-surname">Hubbs</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mareena M.</span> <span class="nlm-surname">Whisby-Pitts</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jonathan L.</span> <span class="nlm-surname">McMurry</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">509141; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/509141 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Kinetic%20Analysis%20of%20Bacteriophage%20Sf6%20Binding%20to%20Outer%20Membrane%20Protein%20A%20Using%20Whole%20Virions&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Hubbs%2C+Natalia+B.&amp;rft.au=Whisby-Pitts%2C+Mareena+M.&amp;rft.au=McMurry%2C+Jonathan+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/150617/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/150617/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/150617/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/150617/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/150617/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/150617/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/150617/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/150617/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/150617/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/150617/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/150617/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/150617/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/509141v1&amp;count=horizontal&amp;via=&amp;text=Kinetic%20Analysis%20of%20Bacteriophage%20Sf6%20Binding%20to%20Outer%20Membrane%20Protein%20A%20Using%20Whole%20Virions&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/509141v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/509141v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/509141v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
