<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/21/2020.03.19.998179.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.19.998179v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Molecular characterization of SARS-CoV-2 in the first COVID-19 cluster in France reveals an amino-acid deletion in nsp2 (Asp268Del)"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.19.998179"/>
  <meta name="DC.Date" content="2020-03-21"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="We present the first genetic characterization of a COVID-19 cluster in Europe using metagenomic next-generation sequencing (mNGS). Despite low viral loads, the mNGS workflow used herein allowed to characterize the whole genome sequences of SARS-CoV2 isolated from an asymptomatic patient, in 2 clinical samples collected 1 day apart. Comparison of these sequences suggests viral evolution with development of quasispecies. In addition, the present workflow identified a new deletion in nsp2 (Asp268Del) which was found in all 3 samples originating from this cluster as well as in 37 other viruses collected in England and in Netherlands, suggesting the spread of this deletion in Europe. The impact of Asp268Del on SARS-CoV-2 transmission and pathogenicity, as well as on PCR performances and anti-viral strategy should be rapidly evaluated in further studies."/>
  <meta name="DC.Contributor" content="Antonin Bal"/>
  <meta name="DC.Contributor" content="Grégory Destras"/>
  <meta name="DC.Contributor" content="Alexandre Gaymard"/>
  <meta name="DC.Contributor" content="Maude Bouscambert-Duchamp"/>
  <meta name="DC.Contributor" content="Martine Valette"/>
  <meta name="DC.Contributor" content="Vanessa Escuret"/>
  <meta name="DC.Contributor" content="Emilie Frobert"/>
  <meta name="DC.Contributor" content="Geneviève Billaud"/>
  <meta name="DC.Contributor" content="Sophie Trouillet-Assant"/>
  <meta name="DC.Contributor" content="Valérie Cheynet"/>
  <meta name="DC.Contributor" content="Karen Brengel-Pesce"/>
  <meta name="DC.Contributor" content="Florence Morfin"/>
  <meta name="DC.Contributor" content="Bruno Lina"/>
  <meta name="DC.Contributor" content="Laurence Josset"/>
  <meta name="article:published_time" content="2020-03-21"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Molecular characterization of SARS-CoV-2 in the first COVID-19 cluster in France reveals an amino-acid deletion in nsp2 (Asp268Del)"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;We present the first genetic characterization of a COVID-19 cluster in Europe using metagenomic next-generation sequencing (mNGS). Despite low viral loads, the mNGS workflow used herein allowed to characterize the whole genome sequences of SARS-CoV2 isolated from an asymptomatic patient, in 2 clinical samples collected 1 day apart. Comparison of these sequences suggests viral evolution with development of quasispecies. In addition, the present workflow identified a new deletion in nsp2 (Asp268Del) which was found in all 3 samples originating from this cluster as well as in 37 other viruses collected in England in February and in Netherlands in March, suggesting the spread of this deletion in Europe. The impact of Asp268Del on SARS-CoV-2 transmission and pathogenicity, as well as on PCR performances and anti-viral strategy should be rapidly evaluated in further studies.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.19.998179v1"/>
  <meta name="citation_id" content="2020.03.19.998179v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.19.998179v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.19.998179v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.19.998179v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/21/2020.03.19.998179.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.19.998179"/>
  <meta name="citation_num_pages" content="6"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.19.998179"/>
  <meta name="citation_author" content="Antonin Bal"/>
  <meta name="citation_author_institution" content="Centre National de Référence des Virus respiratoires"/>
  <meta name="citation_author_institution" content="Laboratoire de Virologie, Institut des Agents Infectieux, Hôpital de la Croix-Rousse"/>
  <meta name="citation_author_institution" content="Université Lyon 1, Faculté de Médecine Lyon Est"/>
  <meta name="citation_author_institution" content="Laboratoire Commun de Recherche Hospices Civils de Lyon-bioMérieux, Centre Hospitalier Lyon Sud"/>
  <meta name="citation_author" content="Grégory Destras"/>
  <meta name="citation_author_institution" content="Centre National de Référence des Virus respiratoires"/>
  <meta name="citation_author_institution" content="Laboratoire de Virologie, Institut des Agents Infectieux, Hôpital de la Croix-Rousse"/>
  <meta name="citation_author_institution" content="Université Lyon 1, Faculté de Médecine Lyon Est"/>
  <meta name="citation_author" content="Alexandre Gaymard"/>
  <meta name="citation_author_institution" content="Centre National de Référence des Virus respiratoires"/>
  <meta name="citation_author_institution" content="Laboratoire de Virologie, Institut des Agents Infectieux, Hôpital de la Croix-Rousse"/>
  <meta name="citation_author_institution" content="Université Lyon 1, Faculté de Médecine Lyon Est"/>
  <meta name="citation_author" content="Maude Bouscambert-Duchamp"/>
  <meta name="citation_author_institution" content="Centre National de Référence des Virus respiratoires"/>
  <meta name="citation_author_institution" content="Laboratoire de Virologie, Institut des Agents Infectieux, Hôpital de la Croix-Rousse"/>
  <meta name="citation_author" content="Martine Valette"/>
  <meta name="citation_author_institution" content="Centre National de Référence des Virus respiratoires"/>
  <meta name="citation_author_institution" content="Laboratoire de Virologie, Institut des Agents Infectieux, Hôpital de la Croix-Rousse"/>
  <meta name="citation_author" content="Vanessa Escuret"/>
  <meta name="citation_author_institution" content="Centre National de Référence des Virus respiratoires"/>
  <meta name="citation_author_institution" content="Laboratoire de Virologie, Institut des Agents Infectieux, Hôpital de la Croix-Rousse"/>
  <meta name="citation_author_institution" content="Université Lyon 1, Faculté de Médecine Lyon Est"/>
  <meta name="citation_author" content="Emilie Frobert"/>
  <meta name="citation_author_institution" content="Centre National de Référence des Virus respiratoires"/>
  <meta name="citation_author_institution" content="Laboratoire de Virologie, Institut des Agents Infectieux, Hôpital de la Croix-Rousse"/>
  <meta name="citation_author_institution" content="Université Lyon 1, Faculté de Médecine Lyon Est"/>
  <meta name="citation_author" content="Geneviève Billaud"/>
  <meta name="citation_author_institution" content="Centre National de Référence des Virus respiratoires"/>
  <meta name="citation_author_institution" content="Laboratoire de Virologie, Institut des Agents Infectieux, Hôpital de la Croix-Rousse"/>
  <meta name="citation_author" content="Sophie Trouillet-Assant"/>
  <meta name="citation_author_institution" content="Université Lyon 1, Faculté de Médecine Lyon Est"/>
  <meta name="citation_author_institution" content="Laboratoire Commun de Recherche Hospices Civils de Lyon-bioMérieux, Centre Hospitalier Lyon Sud"/>
  <meta name="citation_author" content="Valérie Cheynet"/>
  <meta name="citation_author_institution" content="Laboratoire Commun de Recherche Hospices Civils de Lyon-bioMérieux, Centre Hospitalier Lyon Sud"/>
  <meta name="citation_author" content="Karen Brengel-Pesce"/>
  <meta name="citation_author_institution" content="Laboratoire Commun de Recherche Hospices Civils de Lyon-bioMérieux, Centre Hospitalier Lyon Sud"/>
  <meta name="citation_author" content="Florence Morfin"/>
  <meta name="citation_author_institution" content="Centre National de Référence des Virus respiratoires"/>
  <meta name="citation_author_institution" content="Laboratoire de Virologie, Institut des Agents Infectieux, Hôpital de la Croix-Rousse"/>
  <meta name="citation_author_institution" content="Université Lyon 1, Faculté de Médecine Lyon Est"/>
  <meta name="citation_author" content="Bruno Lina"/>
  <meta name="citation_author_institution" content="Centre National de Référence des Virus respiratoires"/>
  <meta name="citation_author_institution" content="Laboratoire de Virologie, Institut des Agents Infectieux, Hôpital de la Croix-Rousse"/>
  <meta name="citation_author_institution" content="Université Lyon 1, Faculté de Médecine Lyon Est"/>
  <meta name="citation_author" content="Laurence Josset"/>
  <meta name="citation_author_institution" content="Centre National de Référence des Virus respiratoires"/>
  <meta name="citation_author_institution" content="Laboratoire de Virologie, Institut des Agents Infectieux, Hôpital de la Croix-Rousse"/>
  <meta name="citation_author_institution" content="Université Lyon 1, Faculté de Médecine Lyon Est"/>
  <meta name="citation_author_email" content="laurence.josset@chu-lyon.fr"/>
  <meta name="citation_reference" content="Zhu N, Zhang D, Wang W, Li X, Yang B, Song J, et al. A Novel Coronavirus from Patients with Pneumonia in China, 2019. N Engl J Med 2020;382:727–33. https://doi.org/10.1056/NEJMoa2001017."/>
  <meta name="citation_reference" content="Spiteri G, Fielding J, Diercke M, Campese C, Enouf V, Gaymard A, et al. First cases of coronavirus disease 2019 (COVID-19) in the WHO European Region, 24 January to 21 February 2020. Euro Surveill Bull Eur Sur Mal Transm Eur Commun Dis Bull 2020;25. https://doi.org/10.2807/1560-7917.ES.2020.25.9.2000178."/>
  <meta name="citation_reference" content="Corman VM, Landt O, Kaiser M, Molenkamp R, Meijer A, Chu DKW, et al. Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR. Euro Surveill Bull Eur Sur Mal Transm Eur Commun Dis Bull 2020;25. https://doi.org/10.2807/1560-7917.ES.2020.25.3.2000045."/>
  <meta name="citation_reference" content="Bal A, Pichon M, Picard C, Casalegno JS, Valette M, Schuffenecker I, et al. Quality control implementation for universal characterization of DNA and RNA viruses in clinical respiratory samples using single metagenomic next-generation sequencing workflow. BMC Infect Dis 2018;18:537. https://doi.org/10.1186/s12879-018-3446-5."/>
  <meta name="citation_reference" content="CoV-GLUE. Amino acid analysis for the SARS-CoV-2 outbreak. Available from: http://cov-glue.cvr.gla.ac.uk"/>
  <meta name="twitter:title" content="Molecular characterization of SARS-CoV-2 in the first COVID-19 cluster in France reveals an amino-acid deletion in nsp2 (Asp268Del)"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="We present the first genetic characterization of a COVID-19 cluster in Europe using metagenomic next-generation sequencing (mNGS). Despite low viral loads, the mNGS workflow used herein allowed to characterize the whole genome sequences of SARS-CoV2 isolated from an asymptomatic patient, in 2 clinical samples collected 1 day apart. Comparison of these sequences suggests viral evolution with development of quasispecies. In addition, the present workflow identified a new deletion in nsp2 (Asp268Del) which was found in all 3 samples originating from this cluster as well as in 37 other viruses collected in England and in Netherlands, suggesting the spread of this deletion in Europe. The impact of Asp268Del on SARS-CoV-2 transmission and pathogenicity, as well as on PCR performances and anti-viral strategy should be rapidly evaluated in further studies."/>
  <meta name="og-title" property="og:title" content="Molecular characterization of SARS-CoV-2 in the first COVID-19 cluster in France reveals an amino-acid deletion in nsp2 (Asp268Del)"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.19.998179v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="We present the first genetic characterization of a COVID-19 cluster in Europe using metagenomic next-generation sequencing (mNGS). Despite low viral loads, the mNGS workflow used herein allowed to characterize the whole genome sequences of SARS-CoV2 isolated from an asymptomatic patient, in 2 clinical samples collected 1 day apart. Comparison of these sequences suggests viral evolution with development of quasispecies. In addition, the present workflow identified a new deletion in nsp2 (Asp268Del) which was found in all 3 samples originating from this cluster as well as in 37 other viruses collected in England and in Netherlands, suggesting the spread of this deletion in Europe. The impact of Asp268Del on SARS-CoV-2 transmission and pathogenicity, as well as on PCR performances and anti-viral strategy should be rapidly evaluated in further studies."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-21"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Molecular characterization of SARS-CoV-2 in the first COVID-19 cluster in France reveals an amino-acid deletion in nsp2 (Asp268Del) | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1206495 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">We present the first genetic characterization of a COVID-19 cluster in Europe using metagenomic next-generation sequencing (mNGS). Despite low viral loads, the mNGS workflow used herein allowed to characterize the whole genome sequences of SARS-CoV2 isolated from an asymptomatic patient, in 2 clinical samples collected 1 day apart. Comparison of these sequences suggests viral evolution with development of quasispecies. In addition, the present workflow identified a new deletion in nsp2 (Asp268Del) which was found in all 3 samples originating from this cluster as well as in 37 other viruses collected in England and in Netherlands, suggesting the spread of this deletion in Europe. The impact of Asp268Del on SARS-CoV-2 transmission and pathogenicity, as well as on PCR performances and anti-viral strategy should be rapidly evaluated in further studies.</p>
   </div>
   <p id="p-4">To the Editor,</p>
   <p id="p-5">In December 2019, a novel coronavirus emerged in China, causing outbreaks of unexplained pneumonia [
    <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. The virus was subsequently identified as a beta-coronavirus and named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). SARS-CoV-2 is responsible of the coronavirus disease 2019 (COVID-19) pandemic which includes asymptomatic, upper, and lower respiratory tract infections. Among the first European cases of COVID-19, 6 were associated with a cluster of transmission in the French Alps in late January 2020 [
    <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. The index case of this cluster travelled from Singapore to France and went back to the United Kingdom (UK) where he was tested positive for SARS-CoV-2 on February 6
    <sup>th</sup>. Here, we aimed to investigate the French cases related to this cluster using metagenomic next-generation sequencing analysis (mNGS).
   </p>
   <p id="p-6">Of the 6 contact patients tested positive for SARS-CoV2, the 3 samples with highest viral loads (assessed by RT-PCR targeting the RdRp gene) were selected for mNGS [
    <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. One nasopharyngeal swab was collected from a patient with an upper respiratory tract infection on February 7
    <sup>th</sup> (sample #1, Ct = 31.3). The other two samples were collected from the same asymptomatic patient on February 8
    <sup>th</sup> (sample #2, nasopharyngeal swab, Ct = 31.1) and 9
    <sup>th</sup> (sample #3, nasopharyngeal aspirate, Ct = 28.8).
   </p>
   <p id="p-7">A previously described mNGS protocol was used but for which DNase treatment was performed after nucleic acid extraction in order to increase sensitivity of RNA viruses’ detection [
    <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. Low quality and human reads were filtered out and remaining reads were aligned to the SARS-CoV-2 reference genome (isolate Wuhan-Hu-1, EPI_ISL_402125) using BWA-MEM algorithm. A mean of 19 445 767 reads per sample were generated, of which a mean of 605 243 reads per sample were mapped to the SARS-CoV-2 reference genome. Percentage of genome covered at a minimum depth of coverage of 100X was 38.3% for sample #1, 99.6% for sample #2 and 80.6% for sample #3.
   </p>
   <p id="p-8">The whole genome sequence (WGS) generated from sample #2 was deposited on GISAID (EPI_ISL_410486). The phylogenetic analysis using the 571 WGS of SARS-CoV-2 publicly available (as of March 17
    <sup>th</sup> 2020) found that this sequence clustered with a sequence collected in Jiangsu, China (EPI_ISL_408488) on January 19
    <sup>th</sup>, suggesting a direct introduction of the virus from Asia, as suggested by epidemiological data (
    <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig 1</a>). Compared to the reference SARS-CoV-2 sequence, a 3-nucleotide deletion in open reading frame 1ab (ORF1ab) at position 1607-1609 was identified. This deletion was found in 100% of the reads at this position with a coverage of 1745X around the deletion. Importantly, this deletion was also identified in 100% of the reads of sample #1 and sample #3 with a coverage of 54X and 481X, respectively. Using CoV-GLUE resource, we found that this mutation leads to a deletion of amino acid 268 in non-structural protein 2 (nsp2) [
    <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. Up to March 17
    <sup>th</sup>, this deletion in nsp2 (Asp268Del) was also characterized in 37/571 (6.1%) WGS (England n = 6; Netherlands n = 31). WGS-based phylogenetic analysis found that 15 viruses containing this specific deletion were closed to viruses collected in China between December 2019 and early February 2020, while 23 viruses collected in the Netherlands in March 2020 have slightly diverged (
    <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig 1</a>).
   </p>
   <div id="F1" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/21/2020.03.19.998179/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="WGS-based phylogenetic analysis of SARS-CoV-2 The analysis included 571 WGS of SARS-CoV2 (&gt;;29k nt) collected in humans and available in GISAID from March 17th, 2020. The following sequences were excluded from the analysis: EPI_ISL_406592, EPI_ISL_414588, EPI_ISL_412900, EPI_ISL_408487, EPI_ISL_408483 and EPI_ISL_406595 because they were outliers, and EPI_ISL_413747, EPI_ISL_413695 because of incomplete sequences in ORF1ab. The hCoV19/Wuhan/IPBCAMSWH01/2019 strain was used as an outgroup virus. Genetic distances were calculated using the Kimura[[#x2019]]s 2-parameter model (K80) and pairwise deletion. The tree was constructed by the neighbour-joining method using R seqinr and ggtree packages and validated using 1000 bootstrap pseudo-replicates. Sequence from sample #2 (EPI_ISL_410486) is shown by the arrow. Nucleotide alignment (1600 to 1614) is depicted as a heatmap on the right panel with 3-nucleotides deletion shown in black." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1481863446" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;WGS-based phylogenetic analysis of SARS-CoV-2&lt;;/span&gt;; The analysis included 571 WGS of SARS-CoV2 (&gt;;29k nt) collected in humans and available in GISAID from March 17&lt;;sup&gt;;th&lt;;/sup&gt;;, 2020. The following sequences were excluded from the analysis: EPI_ISL_406592, EPI_ISL_414588, EPI_ISL_412900, EPI_ISL_408487, EPI_ISL_408483 and EPI_ISL_406595 because they were outliers, and EPI_ISL_413747, EPI_ISL_413695 because of incomplete sequences in ORF1ab. The hCoV19/Wuhan/IPBCAMSWH01/2019 strain was used as an outgroup virus. Genetic distances were calculated using the Kimura[[#x2019]]s 2-parameter model (K80) and pairwise deletion. The tree was constructed by the neighbour-joining method using R seqinr and ggtree packages and validated using 1000 bootstrap pseudo-replicates. Sequence from sample #2 (EPI_ISL_410486) is shown by the arrow. Nucleotide alignment (1600 to 1614) is depicted as a heatmap on the right panel with 3-nucleotides deletion shown in black.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/21/2020.03.19.998179/F1.medium.gif" width="370" height="440"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/21/2020.03.19.998179/F1.medium.gif" width="370" height="440"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/21/2020.03.19.998179/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/21/2020.03.19.998179/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure 1.</span>
     <span class="caption-title">WGS-based phylogenetic analysis of SARS-CoV-2</span>
     <p id="p-9" class="first-child">The analysis included 571 WGS of SARS-CoV2 (&gt;;29k nt) collected in humans and available in GISAID from March 17
      <sup>th</sup>, 2020. The following sequences were excluded from the analysis: EPI_ISL_406592, EPI_ISL_414588, EPI_ISL_412900, EPI_ISL_408487, EPI_ISL_408483 and EPI_ISL_406595 because they were outliers, and EPI_ISL_413747, EPI_ISL_413695 because of incomplete sequences in ORF1ab. The hCoV19/Wuhan/IPBCAMSWH01/2019 strain was used as an outgroup virus. Genetic distances were calculated using the Kimura’s 2-parameter model (K80) and pairwise deletion. The tree was constructed by the neighbour-joining method using R seqinr and ggtree packages and validated using 1000 bootstrap pseudo-replicates. Sequence from sample #2 (EPI_ISL_410486) is shown by the arrow. Nucleotide alignment (1600 to 1614) is depicted as a heatmap on the right panel with 3-nucleotides deletion shown in black.
     </p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-10">SARS-CoV2 sequences were not further compared between the 2 patients due to largely incomplete coverage of SARS-CoV2 genome in sample #1. Nonetheless, the longitudinal samples from the asymptomatic patient (sample #2 vs sample #3) were compared using a minimum depth of coverage of 100X in order to make a preliminary assessment of intra-host genetic variability. Three SNVs were noticed between the 2 samples: C366A (nsp1: S34Y), A20475G (synonymous mutation in nsp15), T24084A (protein S: L841H), suggesting intra-host evolution of the virus. For all 3 positions, nucleotides from sample #2 were still detected in sample #3 but as minor variants.</p>
   <p id="p-11">In this short report, we present the first genetic characterization of a COVID-19 cluster in Europe. Despite low viral loads, the mNGS workflow used herein allowed to characterize the whole genome sequences of SARS-CoV2 isolated from an asymptomatic patient, in 2 clinical samples collected 1 day apart. Comparison of these sequences suggests viral evolution with development of quasispecies. Specific studies using high depth of coverage to assess quasispecies are needed to characterize potential intra-host adaptation. In addition, the present workflow identified a new deletion in nsp2 (Asp268Del) which was found in all 3 samples originating from this cluster as well as in 37 other viruses collected in England (February) and in Netherlands (March), suggesting the spread of this deletion in Europe. The impact of Asp268Del on SARS-CoV-2 transmission and pathogenicity, as well as on PCR performances and anti-viral strategy should be rapidly evaluated in further studies.</p>
   <div class="section" id="sec-1">
    <h2 class="">Ethical considerations</h2>
    <p id="p-12">Investigations complied with the General Data Protection Regulation (Regulation (EU) 2016/679 and Directive 95/46/EC) and the French data protection law (Law n°78-17 on 06/01/1978 and Décret n°2019-536 on 29/05/2019). Informed consent to disclosure information relevant to this publication was obtained from the confirmed cases in France.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-13">We would like to thank all the patients, laboratory technicians, and clinicians who contributed to this investigation. We are also grateful to Véréna Landel and Philip Robinson Landel (DRCI, Hospices Civils de Lyon) for help in manuscript preparation. We thank the Authors, the Originating and Submitting Laboratories for their sequence and metadata shared through GISAID on which this research is based. We gratefully acknowledge all the members of CoV-GLUE, Nextrsain.org, and virological.org for sharing their analysis in real-time.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">[1].</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.19.998179v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>
         <span class="cit-pub-date">2020</span>;
         <span class="cit-vol">382</span>:
         <span class="cit-fpage">727</span>–
         <span class="cit-lpage">33</span>. 
         <a href="https://doi.org/10.1056/NEJMoa2001017">https://doi.org/10.1056/NEJMoa2001017</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D382%26rft.spage%253D727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[2].</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.19.998179v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Spiteri</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fielding</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Diercke</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Campese</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Enouf</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gaymard</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">First cases of coronavirus disease 2019 (COVID-19) in the WHO European Region, 24 January to 21 February 2020</span>. 
         <abbr class="cit-jnl-abbrev">Euro Surveill Bull Eur Sur Mal Transm Eur Commun Dis Bull</abbr>
         <span class="cit-pub-date">2020</span>;
         <span class="cit-vol">25</span>. 
         <a href="https://doi.org/10.2807/1560-7917.ES.2020.25.9.2000178">https://doi.org/10.2807/1560-7917.ES.2020.25.9.2000178</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[3].</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.19.998179v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Corman</span>
          <span class="cit-name-given-names">VM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Landt</span>
          <span class="cit-name-given-names">O</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kaiser</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Molenkamp</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Meijer</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chu</span>
          <span class="cit-name-given-names">DKW</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</span>. 
         <abbr class="cit-jnl-abbrev">Euro Surveill Bull Eur Sur Mal Transm Eur Commun Dis Bull</abbr>
         <span class="cit-pub-date">2020</span>;
         <span class="cit-vol">25</span>. 
         <a href="https://doi.org/10.2807/1560-7917.ES.2020.25.3.2000045">https://doi.org/10.2807/1560-7917.ES.2020.25.3.2000045</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">[4].</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.19.998179v1.4" data-doi="10.1186/s12879-018-3446-5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bal</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pichon</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Picard</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Casalegno</span>
          <span class="cit-name-given-names">JS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Valette</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schuffenecker</span>
          <span class="cit-name-given-names">I</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Quality control implementation for universal characterization of DNA and RNA viruses in clinical respiratory samples using single metagenomic next-generation sequencing workflow</span>. 
         <abbr class="cit-jnl-abbrev">BMC Infect Dis</abbr>
         <span class="cit-pub-date">2018</span>;
         <span class="cit-vol">18</span>:
         <span class="cit-fpage">537</span>. 
         <a href="https://doi.org/10.1186/s12879-018-3446-5">https://doi.org/10.1186/s12879-018-3446-5</a>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2BInfect%2BDis%26rft.volume%253D18%26rft.spage%253D537%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12879-018-3446-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/s12879-018-3446-5&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">[5].</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.03.19.998179v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">CoV-GLUE</span>. 
         <span class="cit-source">Amino acid analysis for the SARS-CoV-2 outbreak. Available from</span>: 
         <a href="http://cov-glue.cvr.gla.ac.uk">http://cov-glue.cvr.gla.ac.uk</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
