<?xml version="1.0" encoding="UTF-8"?>
<html xmlns="http://www.w3.org/1999/xhtml">
 <head charset="UTF-8">
  <title/>
  <link rel="stylesheet" type="text/css" href="http://ajax.aspnetcdn.com/ajax/jquery.dataTables/1.9.4/css/jquery.dataTables.css"/>
  <script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.8.2.min.js" charset="UTF-8" type="text/javascript"> </script>
  <script src="http://ajax.aspnetcdn.com/ajax/jquery.dataTables/1.9.4/jquery.dataTables.min.js" charset="UTF-8" type="text/javascript"> </script>
  <script charset="UTF-8" type="text/javascript">$(function() { $("#results").dataTable(); }) </script>
 </head>
 <body>
  <div class="bs-example table-responsive">
   <table class="table table-striped table-bordered table-hover" id="results">
    <thead>
     <tr>
      <th>articles</th>
      <th>bibliography</th>
      <th>dic:country</th>
      <th>dic:disease</th>
      <th>dic:funders</th>
      <th>word:frequencies</th>
     </tr>
    </thead>
    <tbody>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC2259160" target="separate" title="remote PDF on server">PMC2259160</a>
       <a href="PMC2259160/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Emerging diseases at the interface of people, domestic animals and wildlife. The role of wildlife in our understanding of highly pathogenic avian influenza.">
         <i>titl: </i>Emerging diseases at the inter ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="null">
         <i>abst: </i>null
        </span>
       </div>
      </td>
      <td/>
      <td>
       <span>1</span>
      </td>
      <td/>
      <td/>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC3427559" target="separate" title="remote PDF on server">PMC3427559</a>
       <a href="PMC3427559/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Human viruses: discovery and emergence.">
         <i>titl: </i>Human viruses: discovery and e ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="There are 219 virus species that are known to be able to infect humans. The first of these to be discovered was yellow fever virus in 1901, and three to four new species are still being found every year. Extrapolation of the discovery curve suggests that there is still a substantial pool of undiscovered human virus species, although an apparent slow-down in the rate of discovery of species from different families may indicate bounds to the potential range of diversity. More than two-thirds of human viruses can also infect non-human hosts, mainly mammals, and sometimes birds. Many specialist human viruses also have mammalian or avian origins. Indeed, a substantial proportion of mammalian viruses may be capable of crossing the species barrier into humans, although only around half of these are capable of being transmitted by humans and around half again of transmitting well enough to cause major outbreaks. A few possible predictors of species jumps can be identified, including the use of phylogenetically conserved cell receptors. It seems almost inevitable that new human viruses will continue to emerge, mainly from other mammals and birds, for the foreseeable future. For this reason, an effective global surveillance system for novel viruses is needed.">
         <i>abst: </i>There are 219 virus species th ...
        </span>
       </div>
      </td>
      <td/>
      <td>
       <span>13</span>
      </td>
      <td>
       <span>3</span>
      </td>
      <td>
       <span>10</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC3544629" target="separate" title="remote PDF on server">PMC3544629</a>
       <a href="PMC3544629/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Acute care utilization due to hospitalizations for pediatric lower respiratory tract infections in British Columbia, Canada.">
         <i>titl: </i>Acute care utilization due to ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND: Pediatric LRTI hospitalizations are a significant burden on patients, families, and healthcare systems. This study determined the burden of pediatric LRTIs on hospital settings in British Columbia and the benefits of prevention strategies as they relate to healthcare resource demand. METHODS: LRTI inpatient episodes for patients &lt;19 years of age during 2008-2010 were extracted from the BC Discharge Abstract Database. The annual number of acute care beds required to treat pediatric LRTIs was estimated. Sub-analyses determined the burden due to infants &lt;1 year of age and high-risk infants. Population projections were used to forecast LRTI hospitalizations and the effectiveness of public health initiatives to reduce the incidence of LRTIs to 2020 and 2030. RESULTS: During 2008-2010, LRTI as the primary diagnosis accounted for 32.0 and 75.9% hospitalizations for diseases of the respiratory system in children &lt;19 years of age and infants &lt;1 year of age, respectively. Infants &lt;1 year of age accounted for 47 and 77% hospitalizations due to pediatric LRTIs and pediatric LRTI hospitalizations specifically due to respiratory syncytial virus (RSV), respectively. The average length of stay was 3.1 days for otherwise healthy infants &lt;1 year of age and 9.1 days for high-risk infants (P &lt;0.0001). 73.1% pediatric LRTI hospitalizations occurred between November and April. Over the study timeframe, 19.6 acute care beds were required on average to care for pediatric LRTIs which increased to 64.0 beds at the peak of LRTI hospitalizations. Increases in LRTI bed-days of 5.5 and 16.2% among &lt;19 year olds by 2020 and 2030, respectively, were predicted. Implementation of appropriate prevention strategies could cause 307 and 338 less LRTI hospitalizations in &lt;19 year olds in 2020 and 2030, respectively. CONCLUSION: Pediatric LRTI hospitalizations require significant use of acute care infrastructure particularly between November and April. Population projections show the burden may increase in the next 20 years, but implementation of effective public health prevention strategies may contribute to reducing the acute care demand and to supporting efforts for overall pediatric healthcare sustainability.">
         <i>abst: </i>BACKGROUND: Pediatric LRTI hos ...
        </span>
       </div>
      </td>
      <td>
       <span>2</span>
      </td>
      <td>
       <span>13</span>
      </td>
      <td>
       <span>3</span>
      </td>
      <td>
       <span>23</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC3702433" target="separate" title="remote PDF on server">PMC3702433</a>
       <a href="PMC3702433/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="The incidence and aetiology of hospitalised community-acquired pneumonia among Vietnamese adults: a prospective surveillance in Central Vietnam.">
         <i>titl: </i>The incidence and aetiology of ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND: Lower respiratory tract infection (LRTI) including Community-acquired pneumonia (CAP) is a common infectious disease that is associated with significant morbidity and mortality. The patterns of aetiological pathogens differ by region and country. Special attention must be paid to CAP in Southeast Asia (SEA), a region facing rapid demographic transition. Estimates burden and aetiological patterns of CAP are essential for the clinical and public health management. The purposes of the study are to determine the incidence, aetiological pathogens, clinical pictures and risk factors of community-acquired pneumonia (CAP) in the Vietnamese adult population. METHODS: A prospective surveillance for hospitalised adult CAP was conducted in Khanh Hoa Province, Central Vietnam. All adults aged â‰¥15 years with lower respiratory tract infections (LRTI) admitted to a provincial hospital from September 2009 to August 2010 were enrolled in the study. Patients were classified into CAP and non-pneumonic LRTI (NPLRTI) according to the radiological findings. Bacterial pathogens were identified from sputum samples by the conventional culture and polymerase chain reaction (PCR) for Streptococcus pneumoniae, Haemophilus influenzae, and Moraxella catarrhalis; 13 respiratory viruses were identified from nasopharyngeal specimens by PCR. RESULTS: Of all 367 LRTI episodes examined, 174 (47%) were CAP. Older age, the presence of underlying respiratory conditions, and higher index score of smoking were associated with CAP. The one-year estimated incidence of hospitalised adult CAP in our study population was 0.81 per 1,000 person years. The incidence increased considerably with age and was highest among the elderly. The case fatality proportion of hospitalised CAP patients was 9.8%. Among 286 sputum samples tested for bacterial PCR, 79 (28%) were positive for H. influenzae, and 65 (23%) were positive for S. pneumoniae. Among 357 samples tested for viral PCR, 73 (21%) were positive for respiratory viruses; influenza A (nâ€‰=â€‰32, 9%) was the most common. CONCLUSIONS: The current adult CAP incidence in Vietnam was relatively low; this result was mainly attributed to the young age of our study population.">
         <i>abst: </i>BACKGROUND: Lower respiratory ...
        </span>
       </div>
      </td>
      <td>
       <span>5</span>
      </td>
      <td>
       <span>9</span>
      </td>
      <td>
       <span>2</span>
      </td>
      <td>
       <span>13</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC3787546" target="separate" title="remote PDF on server">PMC3787546</a>
       <a href="PMC3787546/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Virus-induced exacerbations in asthma and COPD.">
         <i>titl: </i>Virus-induced exacerbations in ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Chronic obstructive pulmonary disease (COPD) is characterized by chronic airway inflammation and/or airflow limitation due to pulmonary emphysema. Chronic bronchitis, pulmonary emphysema, and bronchial asthma may all be associated with airflow limitation; therefore, exacerbation of asthma may be associated with the pathophysiology of COPD. Furthermore, recent studies have suggested that the exacerbation of asthma, namely virus-induced asthma, may be associated with a wide variety of respiratory viruses. COPD and asthma have different underlying pathophysiological processes and thus require individual therapies. Exacerbation of both COPD and asthma, which are basically defined and diagnosed by clinical symptoms, is associated with a rapid decline in lung function and increased mortality. Similar pathogens, including human rhinovirus, respiratory syncytial virus, influenza virus, parainfluenza virus, and coronavirus, are also frequently detected during exacerbation of asthma and/or COPD. Immune response to respiratory viral infections, which may be related to the severity of exacerbation in each disease, varies in patients with both COPD and asthma. In this regard, it is crucial to recognize and understand both the similarities and differences of clinical features in patients with COPD and/or asthma associated with respiratory viral infections, especially in the exacerbative stage. In relation to definition, epidemiology, and pathophysiology, this review aims to summarize current knowledge concerning exacerbation of both COPD and asthma by focusing on the clinical significance of associated respiratory virus infections.">
         <i>abst: </i>Chronic obstructive pulmonary ...
        </span>
       </div>
      </td>
      <td>
       <span>4</span>
      </td>
      <td>
       <span>23</span>
      </td>
      <td>
       <span>8</span>
      </td>
      <td>
       <span>27</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4142007" target="separate" title="remote PDF on server">PMC4142007</a>
       <a href="PMC4142007/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Current perspectives in transfusion-transmitted infectious diseases: emerging and re-emerging infections.">
         <i>titl: </i>Current perspectives in transf ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="In August 2009, a group from the AABB (Stramer et al., Transfusion 2009;99:1S-29S, Emerging Infectious Disease Agents and their Potential Threat to Transfusion Safety; http://www.aabb.org/resources/bct/eid/Pages/default.aspx) published a Supplement to Transfusion that reviewed emerging infectious disease (EID) agents that pose a real or theoretical threat to transfusion safety, but for which an existing effective intervention is lacking. The necessary attributes for transfusion transmission were outlined including: presence of the agent in blood during the donor's asymptomatic phase, the agent's survival/persistence in blood during processing/storage, and lastly that the agent must be recognized as responsible for a clinically apparent outcome in at least a proportion of recipients who become infected. Without these attributes, agents are not considered as a transfusion-transmission threat and were excluded. Sixty-eight such agents were identified with enough evidence/likelihood of transfusion transmission (e.g., blood phase) and potential for clinical disease to warrant further consideration. In the Supplement, Fact Sheets (FS) were published providing information on: agent classification; disease agent's importance; clinical syndromes/diseases caused; transmission modes (including vectors/reservoirs); likelihood of transfusion transmission, and if proven to be transfusion-transmitted, information on known cases; the feasibility/predicted success of interventions for donor screening (questioning) and tests available for diagnostics/ adapted for donor screening; and finally, the efficacy, if known, of inactivation methods for plasma-derived products. The Supplement included a separate section on pathogen reduction using published data. Agents were prioritized relative to their scientific/epidemiologic threat and their perceived threat to the community including concerns expressed by the regulators of blood. Agents given the highest priority due to a known transfusion-transmission threat and severe/fatal disease in recipients were the vCJD prion, dengue viruses and the obligate red-cell parasite that causes babesiosis (B. microti and related Babesia). Although the focus of the Supplement was towards the United States and Canada, many of the agents (and the process) are applicable worldwide.Since the publication of the Supplement, six new FSs (yellow fever viruses-including vaccine breakthrough infections, miscellaneous arboviruses, XMRV, human parvoviruses/bocaviruses other than B19, and most recently the Middle East respiratory syndrome coronavirus, MERS-CoV) were added and 14 existing FSs updated (Anaplasma, Babesia, Bartonella, Erhlichia, chronic wasting disease-CWD, human prions other than vCJD, vCJD, Coxiella burnetii-the agent of Q fever, dengue viruses, HAV, HEV, Japanese encephalitis-JE complex, tick-borne encephalitis viruses-TBEV, and human parvovirus B19). Also, tables were released outlining pathogen reduction clinical trials/results (published) and availability/commercial routine use of such technologies by country. Of necessity, the list of EID agents is not, and can never be, complete due to the nature of emergence. We recognized that a system of assessing the risk/threat of EIDs for their potential impact on blood safety and availability must include processes for monitoring, identifying, evaluating, estimating severity, assessing risk and developing interventions. Thus, a 'toolkit' containing the necessary 'tools' from EID monitoring (horizon scanning) to validation/effectiveness evaluations of interventions is being developed. The goal is, to develop a systematic approach to risk assessment and intervention development for the impact of emerging infectious upon blood safety intended to educate and provide advise about risks/interventions in a timely/accurate fashion.The process and final product (toolkit) including methods to monitor EID agent emergence, identification/recognition of a transfusion-transmission threat, methods for quantitative risk assessments, and the appropriate management of such threats should be considered for implementation by all blood systems.">
         <i>abst: </i>In August 2009, a group from t ...
        </span>
       </div>
      </td>
      <td>
       <span>13</span>
      </td>
      <td>
       <span>15</span>
      </td>
      <td>
       <span>4</span>
      </td>
      <td>
       <span>4</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC4589665" target="separate" title="remote PDF on server">PMC4589665</a>
       <a href="PMC4589665/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Recovering full-length viral genomes from metagenomes.">
         <i>titl: </i>Recovering full-length viral g ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Infectious disease metagenomics is driven by the question: &quot;what is causing the disease?&quot; in contrast to classical metagenome studies which are guided by &quot;what is out there?&quot; In case of a novel virus, a first step to eventually establishing etiology can be to recover a full-length viral genome from a metagenomic sample. However, retrieval of a full-length genome of a divergent virus is technically challenging and can be time-consuming and costly. Here we discuss different assembly and fragment linkage strategies such as iterative assembly, motif searches, k-mer frequency profiling, coverage profile binning, and other strategies used to recover genomes of potential viral pathogens in a timely and cost-effective manner.">
         <i>abst: </i>Infectious disease metagenomic ...
        </span>
       </div>
      </td>
      <td>
       <span>6</span>
      </td>
      <td>
       <span>4</span>
      </td>
      <td>
       <span>6</span>
      </td>
      <td>
       <span>8</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC5309255" target="separate" title="remote PDF on server">PMC5309255</a>
       <a href="PMC5309255/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Metagenomic Sequencing for Surveillance of Food- and Waterborne Viral Diseases.">
         <i>titl: </i>Metagenomic Sequencing for Sur ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="A plethora of viruses can be transmitted by the food- and waterborne route. However, their recognition is challenging because of the variety of viruses, heterogeneity of symptoms, the lack of awareness of clinicians, and limited surveillance efforts. Classical food- and waterborne viral disease outbreaks are mainly caused by caliciviruses, but the source of the virus is often not known and the foodborne mode of transmission is difficult to discriminate from human-to-human transmission. Atypical food- and waterborne viral disease can be caused by viruses such as hepatitis A and hepatitis E. In addition, a source of novel emerging viruses with a potential to spread via the food- and waterborne route is the repeated interaction of humans with wildlife. Wildlife-to-human adaptation may give rise to self- limiting outbreaks in some cases, but when fully adjusted to the human host can be devastating. Metagenomic sequencing has been investigated as a promising solution for surveillance purposes as it detects all viruses in a single protocol, delivers additional genomic information for outbreak tracing, and detects novel unknown viruses. Nevertheless, several issues must be addressed to apply metagenomic sequencing in surveillance. First, sample preparation is difficult since the genomic material of viruses is generally overshadowed by host- and bacterial genomes. Second, several data analysis issues hamper the efficient, robust, and automated processing of metagenomic data. Third, interpretation of metagenomic data is hard, because of the lack of general knowledge of the virome in the food chain and the environment. Further developments in virus-specific nucleic acid extraction methods, bioinformatic data processing applications, and unifying data visualization tools are needed to gain insightful surveillance knowledge from suspect food samples.">
         <i>abst: </i>A plethora of viruses can be t ...
        </span>
       </div>
      </td>
      <td>
       <span>9</span>
      </td>
      <td>
       <span>6</span>
      </td>
      <td>
       <span>6</span>
      </td>
      <td>
       <span>14</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC5325067" target="separate" title="remote PDF on server">PMC5325067</a>
       <a href="PMC5325067/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Lyssaviruses and rabies: current conundrums, concerns, contradictions and controversies.">
         <i>titl: </i>Lyssaviruses and rabies: curre ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Lyssaviruses are bullet-shaped, single-stranded, negative-sense RNA viruses and the causative agents of the ancient zoonosis rabies. Africa is the likely home to the ancestors of taxa residing within the Genus Lyssavirus, Family Rhabdoviridae. Diverse lyssaviruses are envisioned as co-evolving with bats, as the ultimate reservoirs, over seemingly millions of years. In terms of relative distribution, overt abundance, and resulting progeny, rabies virus is the most successful lyssavirus species today, but for unknown reasons. All mammals are believed to be susceptible to rabies virus infection. Besides reservoirs among the Chiroptera, meso-carnivores also serve as major historical hosts and are represented among the canids, raccoons, skunks, mongooses, and ferret badgers. Â Perpetuating as a disease of nature with the mammalian central nervous system as niche, host breadth alone precludes any candidacy for true eradication. Despite having the highest case fatality of any infectious disease and a burden in excess of or comparative to other major zoonoses, rabies remains neglected. Once illness appears, no treatment is proven to prevent death. Paradoxically, vaccines were developed more than a century ago, but the clear majority of human cases are unvaccinated. Tens of millions of people are exposed to suspect rabid animals and tens of thousands succumb annually, primarily children in developing countries, where canine rabies is enzootic. Rather than culling animal populations, one of the most cost-effective strategies to curbing human fatalities is the mass vaccination of dogs. Building on considerable progress to date, several complementary actions are needed in the near future, including a more harmonized approach to viral taxonomy, enhanced de-centralized laboratory-based surveillance, focal pathogen discovery and characterization, applied pathobiological research for therapeutics, improved estimates of canine populations at risk, actual production of required vaccines and related biologics, strategies to maximize prevention but minimize unnecessary human prophylaxis, and a long-term, realistic plan for sustained global program support to achieve success in disease control, prevention, and elimination.">
         <i>abst: </i>Lyssaviruses are bullet-shaped ...
        </span>
       </div>
      </td>
      <td>
       <span>84</span>
      </td>
      <td>
       <span>8</span>
      </td>
      <td>
       <span>7</span>
      </td>
      <td>
       <span>25</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC5327784" target="separate" title="remote PDF on server">PMC5327784</a>
       <a href="PMC5327784/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Emerging and Neglected Infectious Diseases: Insights, Advances, and Challenges.">
         <i>titl: </i>Emerging and Neglected Infecti ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Infectious diseases are a significant burden on public health and economic stability of societies all over the world. They have for centuries been among the leading causes of death and disability and presented growing challenges to health security and human progress. The threat posed by infectious diseases is further deepened by the continued emergence of new, unrecognized, and old infectious disease epidemics of global impact. Over the past three and half decades at least 30 new infectious agents affecting humans have emerged, most of which are zoonotic and their origins have been shown to correlate significantly with socioeconomic, environmental, and ecological factors. As these factors continue to increase, putting people in increased contact with the disease causing pathogens, there is concern that infectious diseases may continue to present a formidable challenge. Constant awareness and pursuance of effective strategies for controlling infectious diseases and disease emergence thus remain crucial. This review presents current updates on emerging and neglected infectious diseases and highlights the scope, dynamics, and advances in infectious disease management with particular focus on WHO top priority emerging infectious diseases (EIDs) and neglected tropical infectious diseases.">
         <i>abst: </i>Infectious diseases are a sign ...
        </span>
       </div>
      </td>
      <td>
       <span>9</span>
      </td>
      <td>
       <span>21</span>
      </td>
      <td>
       <span>11</span>
      </td>
      <td>
       <span>20</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC5435908" target="separate" title="remote PDF on server">PMC5435908</a>
       <a href="PMC5435908/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Ribonuclease from Bacillus Acts as an Antiviral Agent against Negative- and Positive-Sense Single Stranded Human Respiratory RNA Viruses.">
         <i>titl: </i>Ribonuclease from Bacillus Act ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Bacillus pumilus ribonuclease (binase) was shown to be a promising antiviral agent in animal models and cell cultures. However, the mode of its antiviral action remains unknown. To assess the binase effect on intracellular viral RNA we have selected single stranded negative- and positive-sense RNA viruses, influenza virus, and rhinovirus, respectively, which annually cause respiratory illnesses and are characterized by high contagious nature, mutation rate, and antigen variability. We have shown that binase exerts an antiviral effect on both viruses at the same concentration, which does not alter the spectrum of A549 cellular proteins and expression of housekeeping genes. The titers of influenza A (H1N1pdm) virus and human rhinovirus serotype 1A were reduced by 40% and 65%, respectively. A preincubation of influenza virus with binase before infection significantly reduced viral titer after single-cycle replication of the virus. Using influenza A virus mini genome system we showed that binase reduced GFP reporter signaling indicating a binase action on the expression of viral mRNA. Binase reduced the level of H1N1pdm viral NP mRNA accumulation in A549 cells by 20%. Since the viral mRNA is a possible target for binase this agent could be potentially applied in the antiviral therapy against both negative- and positive-sense RNA viruses.">
         <i>abst: </i>Bacillus pumilus ribonuclease ...
        </span>
       </div>
      </td>
      <td>
       <span>4</span>
      </td>
      <td>
       <span>9</span>
      </td>
      <td>
       <span>5</span>
      </td>
      <td>
       <span>12</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC5493079" target="separate" title="remote PDF on server">PMC5493079</a>
       <a href="PMC5493079/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="36th International Symposium on Intensive Care and Emergency Medicine : Brussels, Belgium. 15-18 March 2016.">
         <i>titl: </i>36th International Symposium o ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="null">
         <i>abst: </i>null
        </span>
       </div>
      </td>
      <td>
       <span>54</span>
      </td>
      <td>
       <span>124</span>
      </td>
      <td>
       <span>100</span>
      </td>
      <td>
       <span>829</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC5605151" target="separate" title="remote PDF on server">PMC5605151</a>
       <a href="PMC5605151/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Cetylpyridinium Chloride (CPC) Exhibits Potent, Rapid Activity Against Influenza Viruses in vitro and in vivo.">
         <i>titl: </i>Cetylpyridinium Chloride (CPC) ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:There is a continued need for strategies to prevent influenza. While cetylpyridinium chloride (CPC), a broad-spectrum antimicrobial agent, has an extensive antimicrobial spectrum, its ability to affect respiratory viruses has not been studied in detail. OBJECTIVES:Here, we evaluate the ability of CPC to disrupt influenza viruses in vitro and in vivo. METHODS:The virucidal activity of CPC was evaluated against susceptible and oseltamivir-resistant strains of influenza viruses. The effective virucidal concentration (EC) of CPC was determined using a hemagglutination assay and tissue culture infective dose assay. The effect of CPC on viral envelope morphology and ultrastructure was evaluated using transmission electron microscopy (TEM). The ability of influenza virus to develop resistance was evaluated after multiple passaging in sub-inhibitory concentrations of CPC. Finally, the efficacy of CPC in formulation to prevent and treat influenza infection was evaluated using the PR8 murine influenza model. RESULTS:The virucidal effect of CPC occurred within 10 minutes, with mean EC50 and EC2log ranging between 5 to 20 Î¼g/mL, for most strains of influenza tested regardless of type and resistance to oseltamivir. Examinations using TEM showed that CPC disrupted the integrity of the viral envelope and its morphology. Influenza viruses demonstrated no resistance to CPC despite prolonged exposure. Treated mice exhibited significantly increased survival and maintained body weight compared to untreated mice. CONCLUSIONS:The antimicrobial agent CPC possesses virucidal activity against susceptible and resistant strains of influenza virus by targeting and disrupting the viral envelope. Substantial virucidal activity is seen even at very low concentrations of CPC without development of resistance. Moreover, CPC in formulation reduces influenza-associated mortality and morbidity in vivo.">
         <i>abst: </i>BACKGROUND:There is a continue ...
        </span>
       </div>
      </td>
      <td>
       <span>2</span>
      </td>
      <td>
       <span>1</span>
      </td>
      <td>
       <span>6</span>
      </td>
      <td>
       <span>7</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC5683535" target="separate" title="remote PDF on server">PMC5683535</a>
       <a href="PMC5683535/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Phylodynamic applications in 21st century global infectious disease research.">
         <i>titl: </i>Phylodynamic applications in 2 ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Background:Phylodynamics, the study of the interaction between epidemiological and pathogen evolutionary processes within and among populations, was originally defined in the context of rapidly evolving viruses and used to characterize transmission dynamics. The concept of phylodynamics has evolved since the early 21st century, extending its reach to slower-evolving pathogens, including bacteria and fungi, and to the identification of influential factors in disease spread and pathogen population dynamics. Results:The phylodynamic approach has now become a fundamental building block for the development of comparative phylogenetic tools capable of incorporating epidemiological surveillance data with molecular sequences into a single statistical framework. These innovative tools have greatly enhanced scientific investigations of the temporal and geographical origins, evolutionary history, and ecological risk factors associated with the growth and spread of viruses such as human immunodeficiency virus (HIV), Zika, and dengue and bacteria such as Methicillin-resistant Staphylococcus aureus. Conclusions:Capitalizing on an extensive review of the literature, we discuss the evolution of the field of infectious disease epidemiology and recent accomplishments, highlighting the advancements in phylodynamics, as well as the challenges and limitations currently facing researchers studying emerging pathogen epidemics across the globe.">
         <i>abst: </i>Background:Phylodynamics, the ...
        </span>
       </div>
      </td>
      <td>
       <span>9</span>
      </td>
      <td>
       <span>5</span>
      </td>
      <td>
       <span>7</span>
      </td>
      <td>
       <span>9</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC5693564" target="separate" title="remote PDF on server">PMC5693564</a>
       <a href="PMC5693564/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="The global spread of Middle East respiratory syndrome: an analysis fusing traditional epidemiological tracing and molecular phylodynamics.">
         <i>titl: </i>The global spread of Middle Ea ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Since its discovery in 2012, over 1700 confirmed cases of Middle East Respiratory Syndrome (MERS) have been documented worldwide and more than a third of those cases have died. While the greatest number of cases has occurred in Saudi Arabia, the recent export of MERS-coronavirus (MERS-CoV) to Republic of Korea showed that a pandemic is a possibility that cannot be ignored. Due to the deficit of knowledge in transmission methodology, targeted treatment and possible vaccines, understanding this virus should be a priority. Our aim was to combine epidemiological data from literature with genetic information from viruses sequenced around the world to present a phylodynamic picture of MERS spread molecular level to global scale.We performed a qualitative meta-analysis of all laboratory confirmed cases worldwide to date based on literature, with emphasis on international transmission and healthcare associated infections. In parallel, we used publicly available MERS-CoV genomes from GenBank to create a phylogeographic tree, detailing geospatial timeline of viral evolution.Several healthcare associated outbreaks starting with the retrospectively identified hospital outbreak in Jordan to the most recent outbreak in Riyadh, Saudi Arabia have occurred. MERS has also crossed many oceans, entering multiple nations in eight waves between 2012 and 2015. In this paper, the spatiotemporal history of MERS cases, as documented epidemiologically, was examined by Bayesian phylogenetic analysis. Distribution of sequences into geographic clusters and interleaving of MERS-CoV sequences from camels among those isolated from humans indicated that multiple zoonotic introductions occurred in endemic nations. We also report a summary of basic reproduction numbers for MERS-CoV in humans and camels.Together, these analyses can help us identify factors associated with viral evolution and spread as well as establish efficacy of infection control measures. The results are especially pertinent to countries without current MERS-CoV endemic, since their unfamiliarity makes them particularly susceptible to uncontrollable spread of a virus that may be imported by travelers.">
         <i>abst: </i>Since its discovery in 2012, o ...
        </span>
       </div>
      </td>
      <td>
       <span>18</span>
      </td>
      <td>
       <span>3</span>
      </td>
      <td>
       <span>14</span>
      </td>
      <td>
       <span>21</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC5735522" target="separate" title="remote PDF on server">PMC5735522</a>
       <a href="PMC5735522/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Evaluation of a field-deployable reverse transcription-insulated isothermal PCR for rapid and sensitive on-site detection of Zika virus.">
         <i>titl: </i>Evaluation of a field-deployab ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="The recent emergence of Zika virus (ZIKV) in Brazil and its precipitous expansion throughout the Americas has highlighted the urgent need for a rapid and reliable on-site diagnostic assay suitable for viral detection. Such point-of-need (PON), low-cost diagnostics are essential for ZIKV control in vulnerable areas with limited resources.We developed and evaluated a ZIKV-specific field-deployable RT-iiPCR reagent set targeting the E gene for rapid detection of ZIKV in ZIKV-spiked human and mosquito specimens, and compared its performance to the Center for Disease Control and Prevention (CDC) and Pan American Health Organization (PAHO) RT-qPCR assays targeting the E and NS2B genes, respectively.These assays demonstrated exclusive specificity for ZIKV (African and Asian lineages), had limits of detection ranging from 10 to 100 in vitro transcribed RNA copies/Î¼l and detection endpoints at 10 plaque forming units/ml of infectious tissue culture fluid. Analysis of human whole blood, plasma, serum, semen, urine, and mosquito pool samples spiked with ZIKV showed an agreement of 90% (kâ€‰=â€‰0.80), 92% (kâ€‰=â€‰0.82), 95% (kâ€‰=â€‰0.86), 92% (kâ€‰=â€‰0.81), 90% (kâ€‰=â€‰0.79), and 100% (kâ€‰=â€‰1), respectively, between the RT-iiPCR assay and composite results from the reference RT-qPCR assays. Overall, the concurrence between the ZIKV RT-iiPCR and the reference RT-qPCR assays was 92% (kâ€‰=â€‰0.83).The ZIKV RT-iiPCR has a performance comparable to the reference CDC and PAHO RT-qPCR assays but provides much faster results (~1.5Â h) with a field-deployable system that can be utilized as a PON diagnostic with the potential to significantly improve the quality of the health care system in vulnerable areas.">
         <i>abst: </i>The recent emergence of Zika v ...
        </span>
       </div>
      </td>
      <td>
       <span>14</span>
      </td>
      <td>
       <span>6</span>
      </td>
      <td>
       <span>5</span>
      </td>
      <td>
       <span>32</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC5765694" target="separate" title="remote PDF on server">PMC5765694</a>
       <a href="PMC5765694/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Haemophilus is overrepresented in the nasopharynx of infants hospitalized with RSV infection and associated with increased viral load and enhanced mucosal CXCL8 responses.">
         <i>titl: </i>Haemophilus is overrepresented ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:While almost all infants are infected with respiratory syncytial virus (RSV) before the age of 2Â years, only a small percentage develops severe disease. Previous studies suggest that the nasopharyngeal microbiome affects disease development. We therefore studied the effect of the nasopharyngeal microbiome on viral load and mucosal cytokine responses, two important factors influencing the pathophysiology of RSV disease. To determine the relation between (i) the microbiome of the upper respiratory tract, (ii) viral load, and (iii) host mucosal inflammation during an RSV infection, nasopharyngeal microbiota profiles of RSV infected infants (&lt;â€‰6Â months) with different levels of disease severity and age-matched healthy controls were determined by 16S rRNA marker gene sequencing. The viral load was measured using qPCR. Nasopharyngeal CCL5, CXCL10, MMP9, IL6, and CXCL8 levels were determined with ELISA. RESULTS:Viral load in nasopharyngeal aspirates of patients associates significantly to total nasopharyngeal microbiota composition. Healthy infants (nÂ =â€‰21) and RSV patients (nÂ =â€‰54) display very distinct microbial patterns, primarily characterized by a loss in commensals like Veillonella and overrepresentation of opportunistic organisms like Haemophilus and Achromobacter in RSV-infected individuals. Furthermore, nasopharyngeal microbiota profiles are significantly different based on CXCL8 levels. CXCL8 is a chemokine that was previously found to be indicative for disease severity and for which we find Haemophilus abundance as the strongest predictor for CXCL8 levels. CONCLUSIONS:The nasopharyngeal microbiota in young infants with RSV infection is marked by an overrepresentation of the genus Haemophilus. We present that this bacterium is associated with viral load and mucosal CXCL8 responses, both which are involved in RSV disease pathogenesis.">
         <i>abst: </i>BACKGROUND:While almost all in ...
        </span>
       </div>
      </td>
      <td>
       <span>3</span>
      </td>
      <td>
       <span>9</span>
      </td>
      <td/>
      <td>
       <span>24</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC5819479" target="separate" title="remote PDF on server">PMC5819479</a>
       <a href="PMC5819479/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Epidemiological characteristics of human-infective RNA viruses.">
         <i>titl: </i>Epidemiological characteristic ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="RNA viruses are a major threat to human health. Here, based on extensive literature searches carried out over a period of 18 years, we provide a catalogue of all 214 known human-infective RNA virus species. We link these viruses to metadata for a number of traits that influence their epidemiology, including the date of the first report of human infection, transmissibility in human populations, transmission route(s) and host range. This database can be used in comparative studies of human-infective RNA viruses to identify the characteristics of viruses most likely to pose the greatest public health threat, both now and in the future.">
         <i>abst: </i>RNA viruses are a major threat ...
        </span>
       </div>
      </td>
      <td/>
      <td>
       <span>5</span>
      </td>
      <td>
       <span>4</span>
      </td>
      <td>
       <span>5</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6043914" target="separate" title="remote PDF on server">PMC6043914</a>
       <a href="PMC6043914/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Meeting report: 30th International Conference on Antiviral Research, in Atlanta, GA, USA.">
         <i>titl: </i>Meeting report: 30th Internati ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="The 30th International Conference on Antiviral Research was held in Atlanta, GA, USA, from 21 to 25 May 2017. Each year, the International Society for Antiviral Research (ISAR) presents three major awards, this year to Mike Sofia (Elion award), David Chu (HolÃ½ award) and Maaike Everts (Prusoff award). Also this year, the inaugural ISAR Women in Science award lecture was presented by Priscilla Yang. For several years, International Conference on Antiviral Research (ICAR) has included at least one Keynote lecture, this year there were four. Although there are accounts of only these eight lectures, they reflect the diversity that is characteristic of ICAR - employment (academia, industry, public health), type of research (virus biology, potential antiviral targets, antiviral drugs, research organisation) and a range of viruses. For example, the viruses included were hepatitis C virus and hepatitis B virus (Mike Sofia), HIV and hepatitis B virus (David Chu), multiple antiviral projects (Maaike Everts), dengue (Priscilla Yang), rhinovirus C (Ann Palmenberg), polio (Mark Pallansch), HIV (Eric Hunter) and Zika virus (Pei-Yong Shi). This report ends with my personal comments giving examples in which this diversity can bring benefits. The 31st ICAR will be in Porto, Portugal, 11-15 June 2018.">
         <i>abst: </i>The 30th International Confere ...
        </span>
       </div>
      </td>
      <td>
       <span>7</span>
      </td>
      <td>
       <span>15</span>
      </td>
      <td>
       <span>22</span>
      </td>
      <td>
       <span>18</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6156540" target="separate" title="remote PDF on server">PMC6156540</a>
       <a href="PMC6156540/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="New Vaccine Technologies to Combat Outbreak Situations.">
         <i>titl: </i>New Vaccine Technologies to Co ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Ever since the development of the first vaccine more than 200 years ago, vaccinations have greatly decreased the burden of infectious diseases worldwide, famously leading to the eradication of small pox and allowing the restriction of diseases such as polio, tetanus, diphtheria, and measles. A multitude of research efforts focuses on the improvement of established and the discovery of new vaccines such as the HPV (human papilloma virus) vaccine in 2006. However, radical changes in the density, age distribution and traveling habits of the population worldwide as well as the changing climate favor the emergence of old and new pathogens that bear the risk of becoming pandemic threats. In recent years, the rapid spread of severe infections such as HIV, SARS, Ebola, and Zika have highlighted the dire need for global preparedness for pandemics, which necessitates the extremely rapid development and comprehensive distribution of vaccines against potentially previously unknown pathogens. What is more, the emergence of antibiotic resistant bacteria calls for new approaches to prevent infections. Given these changes, established methods for the identification of new vaccine candidates are no longer sufficient to ensure global protection. Hence, new vaccine technologies able to achieve rapid development as well as large scale production are of pivotal importance. This review will discuss viral vector and nucleic acid-based vaccines (DNA and mRNA vaccines) as new approaches that might be able to tackle these challenges to global health.">
         <i>abst: </i>Ever since the development of ...
        </span>
       </div>
      </td>
      <td>
       <span>10</span>
      </td>
      <td>
       <span>21</span>
      </td>
      <td>
       <span>3</span>
      </td>
      <td>
       <span>60</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6369834" target="separate" title="remote PDF on server">PMC6369834</a>
       <a href="PMC6369834/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Investing in Public Health Microbiology Laboratories in Western Balkan Countries Enhances Health Security From Communicable Disease Threats in Europe.">
         <i>titl: </i>Investing in Public Health Mic ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="The European Centre for Disease Prevention and Control (ECDC), under the EU enlargement policy, has supported national efforts of Western Balkan countries to strengthen their communicable disease prevention and control systems. The new EU strategy &quot;A credible enlargement perspective for and enhanced EU engagement with the Western Balkans&quot; advocates transformation processes that will build the foundation of EU-oriented national reforms. Well-functioning public health microbiology laboratories are key for early detection and control of infectious diseases, and thus maintaining and enhancing health security in Europe. In order to help Western Balkan countries to improve their national capacities, ECDC facilitated needs assessments and identified key areas for advancement toward effective public health microbiology systems. Countries identified gaps in their laboratory data reporting and exchange systems. Harmonized and effective procedures for handling of highly contagious agents and cross-border transportation of biological samples were often lacking, as well as the systematic use of diagnostic testing at the primary care level or referral of patients, in particular for detection of antimicrobial resistance. There is a clear need to address the financial investment required for sustaining sufficient numbers of skilled laboratory workforce, laboratory supplies, and the development of new methods and techniques, including investment in emerging laboratory technologies, such as molecular typing by whole genome sequencing. This article highlights the key areas for investing in public health microbiology laboratories in Western Balkan countries needed to strengthen health security in Europe.">
         <i>abst: </i>The European Centre for Diseas ...
        </span>
       </div>
      </td>
      <td>
       <span>7</span>
      </td>
      <td>
       <span>3</span>
      </td>
      <td>
       <span>4</span>
      </td>
      <td>
       <span>9</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6509602" target="separate" title="remote PDF on server">PMC6509602</a>
       <a href="PMC6509602/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Advancing Planetary Health in Australia: focus on emerging infections and antimicrobial resistance.">
         <i>titl: </i>Advancing Planetary Health in ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="With rising population numbers, anthropogenic changes to our environment and unprecedented global connectivity, the World Economic Forum ranks the spread of infectious diseases second only to water crises in terms of potential global impact. Addressing the diverse challenges to human health and well-being in the 21st century requires an overarching focus on 'Planetary Health', with input from all sectors of government, non-governmental organisations, academic institutions and industry. To clarify and advance the Planetary Health agenda within Australia, specifically in relation to emerging infectious diseases (EID) and antimicrobial resistance (AMR), national experts and key stakeholders were invited to a facilitated workshop. EID themes identified included animal reservoirs, targeted surveillance, mechanisms of emergence and the role of unrecognised human vectors (the 'invisible man') in the spread of infection. Themes related to AMR included antimicrobial use in production and companion animals, antimicrobial stewardship, novel treatment approaches and education of professionals, politicians and the general public. Effective infection control strategies are important in both EID and AMR. We provide an overview of key discussion points, as well as important barriers identified and solutions proposed.">
         <i>abst: </i>With rising population numbers ...
        </span>
       </div>
      </td>
      <td>
       <span>2</span>
      </td>
      <td>
       <span>5</span>
      </td>
      <td>
       <span>19</span>
      </td>
      <td>
       <span>9</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6593267" target="separate" title="remote PDF on server">PMC6593267</a>
       <a href="PMC6593267/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Multiplex Platforms for the Identification of Respiratory Pathogens: Are They Useful in Pediatric Clinical Practice?">
         <i>titl: </i>Multiplex Platforms for the Id ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Respiratory tract infections (RTIs) are extremely common especially in the first year of life. Knowledge of the etiology of a RTI is essential to facilitate the appropriate management and the implementation of the most effective control measures. This perspective explains why laboratory methods that can identify pathogens in respiratory secretions have been developed over the course of many years. High-complexity multiplex panel assays that can simultaneously detect up to 20 viruses and up to four bacteria within a few hours have been marketed. However, are these platforms actually useful in pediatric clinical practice? In this manuscript, we showed that these platforms appear to be particularly important for epidemiological studies and clinical research. On the contrary, their routine use in pediatric clinical practice remains debatable. They can be used only in the hospital as they require specific equipment and laboratory technicians with considerable knowledge, training, and experience. Moreover, despite more sensitive and specific than other tests routinely used for respiratory pathogen identification, they do not offer significantly advantage for detection of the true etiology of a respiratory disease. Furthermore, knowledge of which virus is the cause of a respiratory disease is not useful from a therapeutic point of view unless influenza virus or respiratory syncytial virus are the infecting agents as effective drugs are available only for these pathogens. On the other hand, multiplex platforms can be justified in the presence of severe clinical manifestations, and in immunocompromised patients for whom specific treatment option can be available, particularly when they can be used simultaneously with platforms that allow identification of antimicrobial resistance to commonly used drugs. It is highly likely that these platforms, particularly those with high sensitivity and specificity and with low turnaround time, will become essential when new drugs effective and safe against most of the respiratory viruses will be available. Further studies on how to differentiate carriers from patients with true disease, as well as studies on the implications of coinfections and identification of antimicrobial resistance, are warranted.">
         <i>abst: </i>Respiratory tract infections ( ...
        </span>
       </div>
      </td>
      <td>
       <span>6</span>
      </td>
      <td>
       <span>11</span>
      </td>
      <td>
       <span>2</span>
      </td>
      <td>
       <span>14</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6594388" target="separate" title="remote PDF on server">PMC6594388</a>
       <a href="PMC6594388/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Overview of Current Therapeutics and Novel Candidates Against Influenza, Respiratory Syncytial Virus, and Middle East Respiratory Syndrome Coronavirus Infections.">
         <i>titl: </i>Overview of Current Therapeuti ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Emergence and re-emergence of respiratory virus infections represent a significant threat to global public health, as they occur seasonally and less frequently (such as in the case of influenza virus) as pandemic infections. Some of these viruses have been in the human population for centuries and others had recently emerged as a public health problem. Influenza viruses have been affecting the human population for a long time now; however, their ability to rapidly evolve through antigenic drift and antigenic shift causes the emergence of new strains. A recent example of these events is the avian-origin H7N9 influenza virus outbreak currently undergoing in China. Human H7N9 influenza viruses are resistant to amantadines and some strains are also resistant to neuraminidase inhibitors greatly limiting the options for treatment. Respiratory syncytial virus (RSV) may cause a lower respiratory tract infection characterized by bronchiolitis and pneumonia mainly in children and the elderly. Infection with RSV can cause severe disease and even death, imposing a severe burden for pediatric and geriatric health systems worldwide. Treatment for RSV is mainly supportive since the only approved therapy, a monoclonal antibody, is recommended for prophylactic use in high-risk patients. The Middle East respiratory syndrome coronavirus (MERS-CoV) is a newly emerging respiratory virus. The virus was first recognized in 2012 and it is associated with a lower respiratory tract disease that is more severe in patients with comorbidities. No licensed vaccines or antivirals have been yet approved for the treatment of MERS-CoV in humans. It is clear that the discovery and development of novel antivirals that can be used alone or in combination with existing therapies to treat these important respiratory viral infections are critical. In this review, we will describe some of the novel therapeutics currently under development for the treatment of these infections.">
         <i>abst: </i>Emergence and re-emergence of ...
        </span>
       </div>
      </td>
      <td>
       <span>9</span>
      </td>
      <td>
       <span>11</span>
      </td>
      <td>
       <span>12</span>
      </td>
      <td>
       <span>42</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6727070" target="separate" title="remote PDF on server">PMC6727070</a>
       <a href="PMC6727070/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Causes of severe pneumonia requiring hospital admission in children without HIV infection from Africa and Asia: the PERCH multi-country case-control study.">
         <i>titl: </i>Causes of severe pneumonia req ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:Pneumonia is the leading cause of death among children younger than 5 years. In this study, we estimated causes of pneumonia in young African and Asian children, using novel analytical methods applied to clinical and microbiological findings. METHODS:We did a multi-site, international case-control study in nine study sites in seven countries: Bangladesh, The Gambia, Kenya, Mali, South Africa, Thailand, and Zambia. All sites enrolled in the study for 24 months. Cases were children aged 1-59 months admitted to hospital with severe pneumonia. Controls were age-group-matched children randomly selected from communities surrounding study sites. Nasopharyngeal and oropharyngeal (NP-OP), urine, blood, induced sputum, lung aspirate, pleural fluid, and gastric aspirates were tested with cultures, multiplex PCR, or both. Primary analyses were restricted to cases without HIV infection and with abnormal chest x-rays and to controls without HIV infection. We applied a Bayesian, partial latent class analysis to estimate probabilities of aetiological agents at the individual and population level, incorporating case and control data. FINDINGS:Between Aug 15, 2011, and Jan 30, 2014, we enrolled 4232 cases and 5119 community controls. The primary analysis group was comprised of 1769 (41Â·8% of 4232) cases without HIV infection and with positive chest x-rays and 5102 (99Â·7% of 5119) community controls without HIV infection. Wheezing was present in 555 (31Â·7%) of 1752 cases (range by site 10Â·6-97Â·3%). 30-day case-fatality ratio was 6Â·4% (114 of 1769 cases). Blood cultures were positive in 56 (3Â·2%) of 1749 cases, and Streptococcus pneumoniae was the most common bacteria isolated (19 [33Â·9%] of 56). Almost all cases (98Â·9%) and controls (98Â·0%) had at least one pathogen detected by PCR in the NP-OP specimen. The detection of respiratory syncytial virus (RSV), parainfluenza virus, human metapneumovirus, influenza virus, S pneumoniae, Haemophilus influenzae type b (Hib), H influenzae non-type b, and Pneumocystis jirovecii in NP-OP specimens was associated with case status. The aetiology analysis estimated that viruses accounted for 61Â·4% (95% credible interval [CrI] 57Â·3-65Â·6) of causes, whereas bacteria accounted for 27Â·3% (23Â·3-31Â·6) and Mycobacterium tuberculosis for 5Â·9% (3Â·9-8Â·3). Viruses were less common (54Â·5%, 95% CrI 47Â·4-61Â·5 vs 68Â·0%, 62Â·7-72Â·7) and bacteria more common (33Â·7%, 27Â·2-40Â·8 vs 22Â·8%, 18Â·3-27Â·6) in very severe pneumonia cases than in severe cases. RSV had the greatest aetiological fraction (31Â·1%, 95% CrI 28Â·4-34Â·2) of all pathogens. Human rhinovirus, human metapneumovirus A or B, human parainfluenza virus, S pneumoniae, M tuberculosis, and H influenzae each accounted for 5% or more of the aetiological distribution. We observed differences in aetiological fraction by age for Bordetella pertussis, parainfluenza types 1 and 3, parechovirus-enterovirus, P jirovecii, RSV, rhinovirus, Staphylococcus aureus, and S pneumoniae, and differences by severity for RSV, S aureus, S pneumoniae, and parainfluenza type 3. The leading ten pathogens of each site accounted for 79% or more of the site's aetiological fraction. INTERPRETATION:In our study, a small set of pathogens accounted for most cases of pneumonia requiring hospital admission. Preventing and treating a subset of pathogens could substantially affect childhood pneumonia outcomes. FUNDING:Bill &amp; Melinda Gates Foundation.">
         <i>abst: </i>BACKGROUND:Pneumonia is the le ...
        </span>
       </div>
      </td>
      <td>
       <span>12</span>
      </td>
      <td>
       <span>9</span>
      </td>
      <td>
       <span>18</span>
      </td>
      <td>
       <span>62</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6911100" target="separate" title="remote PDF on server">PMC6911100</a>
       <a href="PMC6911100/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="A database of geopositioned Middle East Respiratory Syndrome Coronavirus occurrences.">
         <i>titl: </i>A database of geopositioned Mi ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="As a World Health Organization Research and Development Blueprint priority pathogen, there is a need to better understand the geographic distribution of Middle East Respiratory Syndrome Coronavirus (MERS-CoV) and its potential to infect mammals and humans. This database documents cases of MERS-CoV globally, with specific attention paid to zoonotic transmission. An initial literature search was conducted in PubMed, Web of Science, and Scopus; after screening articles according to the inclusion/exclusion criteria, a total of 208 sources were selected for extraction and geo-positioning. Each MERS-CoV occurrence was assigned one of the following classifications based upon published contextual information: index, unspecified, secondary, mammal, environmental, or imported. In total, this database is comprised of 861 unique geo-positioned MERS-CoV occurrences. The purpose of this article is to share a collated MERS-CoV database and extraction protocol that can be utilized in future mapping efforts for both MERS-CoV and other infectious diseases. More broadly, it may also provide useful data for the development of targeted MERS-CoV surveillance, which would prove invaluable in preventing future zoonotic spillover.">
         <i>abst: </i>As a World Health Organization ...
        </span>
       </div>
      </td>
      <td>
       <span>51</span>
      </td>
      <td>
       <span>4</span>
      </td>
      <td>
       <span>12</span>
      </td>
      <td>
       <span>68</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6925075" target="separate" title="remote PDF on server">PMC6925075</a>
       <a href="PMC6925075/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Developing vaccines against epidemic-prone emerging infectious diseases.">
         <i>titl: </i>Developing vaccines against ep ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Today's world is characterized by increasing population density, human mobility, urbanization, and climate and ecological change. This global dynamic has various effects, including the increased appearance of emerging infectious diseases (EIDs), which pose a growing threat to global health security.Outbreaks of EIDs, like the 2013-2016 Ebola outbreak in West Africa or the current Ebola outbreak in Democratic Republic of the Congo (DRC), have not only put populations in low- and middle-income countries (LMIC) at risk in terms of morbidity and mortality, but they also have had a significant impact on economic growth in affected regions and beyond.The Coalition for Epidemic Preparedness Innovation (CEPI) is an innovative global partnership between public, private, philanthropic, and civil society organizations that was launched as the result of a consensus that a coordinated, international, and intergovernmental plan was needed to develop and deploy new vaccines to prevent future epidemics.CEPI is focusing on supporting candidate vaccines against the World Health Organization (WHO) Blueprint priority pathogens MERS-CoV, Nipah virus, Lassa fever virus, and Rift Valley fever virus, as well as Chikungunya virus, which is on the WHO watch list. The current vaccine portfolio contains a wide variety of technologies, ranging across recombinant viral vectors, nucleic acids, and recombinant proteins. To support and accelerate vaccine development, CEPI will also support science projects related to the development of biological standards and assays, animal models, epidemiological studies, and diagnostics, as well as build capacities for future clinical trials in risk-prone contexts.">
         <i>abst: </i>Today's world is characterized ...
        </span>
       </div>
      </td>
      <td>
       <span>12</span>
      </td>
      <td>
       <span>4</span>
      </td>
      <td>
       <span>29</span>
      </td>
      <td>
       <span>6</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6926486" target="separate" title="remote PDF on server">PMC6926486</a>
       <a href="PMC6926486/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Enzootic patterns of Middle East respiratory syndrome coronavirus in imported African and local Arabian dromedary camels: a prospective genomic study.">
         <i>titl: </i>Enzootic patterns of Middle Ea ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:The Middle East respiratory syndrome coronavirus (MERS-CoV) is a lethal zoonotic pathogen endemic to the Arabian Peninsula. Dromedary camels are a likely source of infection and the virus probably originated in Africa. We studied the genetic diversity, geographical structure, infection prevalence, and age-associated prevalence among camels at the largest entry port of camels from Africa into the Arabian Peninsula. METHODS:In this prospective genomic study, we took nasal samples from camels imported from Sudan and Djibouti into the Port of Jeddah in Jeddah, Saudi Arabia, over an almost 2-year period and local Arabian camels over 2 months in the year after surveillance of the port. We determined the prevalence of MERS-CoV infection, age-associated patterns of infection, and undertook phylogeographical and migration analyses to determine intercountry virus transmission after local lineage establishment. We compared all virological characteristics between the local and imported cohorts. We compared major gene deletions between African and Arabian strains of the virus. Reproductive numbers were inferred with Bayesian birth death skyline analyses. FINDINGS:Between Aug 10, 2016, and May 3, 2018, we collected samples from 1196 imported camels, of which 868 originated from Sudan and 328 from Djibouti, and between May 1, and June 25, 2018, we collected samples from 472 local camels, of which 189 were from Riyadh and 283 were from Jeddah, Saudi Arabia. Virus prevalence was higher in local camels than in imported camels (224 [47Â·5%] of 472 vs 157 [13Â·1%] of 1196; p&lt;0Â·0001). Infection prevalence peaked among camels older than 1 year and aged up to 2 years in both groups, with 255 (66Â·9%) of 381 positive cases in this age group. Although the overall geographical distribution of the virus corresponded with the phylogenetic tree topology, some virus exchange was observed between countries corresponding with trade routes in the region. East and west African strains of the virus appear to be geographically separated, with an origin of west African strains in east Africa. African strains of the virus were not re-sampled in Saudi Arabia despite sampling approximately 1 year after importation from Africa. All local Arabian samples contained strains of the virus that belong to a novel recombinant clade (NRC) first detected in 2014 in Saudi Arabia. Reproduction number estimates informed by the sequences suggest sustained endemicity of NRC, with a mean Re of 1Â·16. INTERPRETATION:Despite frequent imports of MERS-CoV with camels from Africa, African lineages of MERS-CoV do not establish themselves in Saudi Arabia. Arabian strains of the virus should be tested for changes in virulence and transmissibility. FUNDING:German Ministry of Research and Education, EU Horizon 2020, and Emerging Diseases Clinical Trials Partnership.">
         <i>abst: </i>BACKGROUND:The Middle East res ...
        </span>
       </div>
      </td>
      <td>
       <span>17</span>
      </td>
      <td>
       <span>1</span>
      </td>
      <td>
       <span>5</span>
      </td>
      <td>
       <span>12</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6926871" target="separate" title="remote PDF on server">PMC6926871</a>
       <a href="PMC6926871/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Exit and Entry Screening Practices for Infectious Diseases among Travelers at Points of Entry: Looking for Evidence on Public Health Impact.">
         <i>titl: </i>Exit and Entry Screening Pract ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="A scoping search and a systematic literature review were conducted to give an insight on entry and exit screening referring to travelers at points of entry, by analyzing published evidence on practices, guidelines, and experiences in the past 15 years worldwide. Grey literature, PubMed. and Scopus were searched using specific terms. Most of the available data identified through the systematic literature review concerned entry screening measures at airports. Little evidence is available about entry and exit screening measure implementation and effectiveness at ports and ground crossings. Exit screening was part of the World Health Organisation's (WHO) temporary recommendations for implementation in certain points of entry, for specific time periods. Exit screening measures for Ebola Virus Disease (EVD) in the three most affected West African countries did not identify any cases and showed zero sensitivity and very low specificity. The percentages of confirmed cases identified out of the total numbers of travelers that passed through entry screening measures in various countries worldwide for Influenza Pandemic (H1N1) and EVD in West Africa were zero or extremely low. Entry screening measures for Severe Acute Respiratory Syndrome (SARS) did not detect any confirmed SARS cases in Australia, Canada, and Singapore. Despite the ineffectiveness of entry and exit screening measures, authors reported several important concomitant positive effects that their impact is difficult to assess, including discouraging travel of ill persons, raising awareness, and educating the traveling public and maintaining operation of flights from/to the affected areas. Exit screening measures in affected areas are important and should be applied jointly with other measures including information strategies, epidemiological investigation, contact tracing, vaccination, and quarantine to achieve a comprehensive outbreak management response. Based on review results, an algorithm about decision-making for entry/exit screening was developed.">
         <i>abst: </i>A scoping search and a systema ...
        </span>
       </div>
      </td>
      <td>
       <span>59</span>
      </td>
      <td>
       <span>13</span>
      </td>
      <td>
       <span>7</span>
      </td>
      <td>
       <span>85</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6935926" target="separate" title="remote PDF on server">PMC6935926</a>
       <a href="PMC6935926/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Metagenomic Nanopore Sequencing of Influenza Virus Direct from Clinical Respiratory Samples.">
         <i>titl: </i>Metagenomic Nanopore Sequencin ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Influenza is a major global public health threat as a result of its highly pathogenic variants, large zoonotic reservoir, and pandemic potential. Metagenomic viral sequencing offers the potential for a diagnostic test for influenza virus which also provides insights on transmission, evolution, and drug resistance and simultaneously detects other viruses. We therefore set out to apply the Oxford Nanopore Technologies sequencing method to metagenomic sequencing of respiratory samples. We generated influenza virus reads down to a limit of detection of 102 to 103 genome copies/ml in pooled samples, observing a strong relationship between the viral titer and the proportion of influenza virus reads (Pâ€‰=â€‰4.7â€‰Ã—â€‰10-5). Applying our methods to clinical throat swabs, we generated influenza virus reads for 27/27 samples with mid-to-high viral titers (cycle threshold [CT ] values, &lt;30) and 6/13 samples with low viral titers (CT values, 30 to 40). No false-positive reads were generated from 10 influenza virus-negative samples. Thus, Nanopore sequencing operated with 83% sensitivity (95% confidence interval [CI], 67 to 93%) and 100% specificity (95% CI, 69 to 100%) compared to the current diagnostic standard. Coverage of full-length virus was dependent on sample composition, being negatively influenced by increased host and bacterial reads. However, at high influenza virus titers, we were able to reconstruct &gt;99% complete sequences for all eight gene segments. We also detected a human coronavirus coinfection in one clinical sample. While further optimization is required to improve sensitivity, this approach shows promise for the Nanopore platform to be used in the diagnosis and genetic analysis of influenza virus and other respiratory viruses.">
         <i>abst: </i>Influenza is a major global pu ...
        </span>
       </div>
      </td>
      <td>
       <span>4</span>
      </td>
      <td>
       <span>7</span>
      </td>
      <td>
       <span>15</span>
      </td>
      <td>
       <span>18</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6939334" target="separate" title="remote PDF on server">PMC6939334</a>
       <a href="PMC6939334/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Development of therapeutic antibodies for the treatment of diseases.">
         <i>titl: </i>Development of therapeutic ant ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="It has been more than three decades since the first monoclonal antibody was approved by the United States Food and Drug Administration (US FDA) in 1986, and during this time, antibody engineering has dramatically evolved. Current antibody drugs have increasingly fewer adverse effects due to their high specificity. As a result, therapeutic antibodies have become the predominant class of new drugs developed in recent years. Over the past five years, antibodies have become the best-selling drugs in the pharmaceutical market, and in 2018, eight of the top ten bestselling drugs worldwide were biologics. The global therapeutic monoclonal antibody market was valued at approximately US$115.2 billion in 2018 and is expected to generate revenue of $150 billion by the end of 2019 and $300 billion by 2025. Thus, the market for therapeutic antibody drugs has experienced explosive growth as new drugs have been approved for treating various human diseases, including many cancers, autoimmune, metabolic and infectious diseases. As of December 2019, 79 therapeutic mAbs have been approved by the US FDA, but there is still significant growth potential. This review summarizes the latest market trends and outlines the preeminent antibody engineering technologies used in the development of therapeutic antibody drugs, such as humanization of monoclonal antibodies, phage display, the human antibody mouse, single B cell antibody technology, and affinity maturation. Finally, future applications and perspectives are also discussed.">
         <i>abst: </i>It has been more than three de ...
        </span>
       </div>
      </td>
      <td>
       <span>5</span>
      </td>
      <td>
       <span>66</span>
      </td>
      <td>
       <span>31</span>
      </td>
      <td>
       <span>72</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6941257" target="separate" title="remote PDF on server">PMC6941257</a>
       <a href="PMC6941257/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Therapeutic applications of nucleic acid aptamers in microbial infections.">
         <i>titl: </i>Therapeutic applications of nu ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Today, the treatment of bacterial infections is a major challenge, due to growing rate of multidrug-resistant bacteria, complication of treatment and increased healthcare costs. Moreover, new treatments for bacterial infections are limited. Oligonucleotide aptamers are single stranded DNAs or RNAs with target-selective high-affinity feature, which considered as nucleic acid-based affinity ligands, replacing monoclonal antibodies. The aptamer-based systems have been found to be talented tools in the treatment of microbial infections, regarding their promising anti-biofilm and antimicrobial activities; they can reduce or inhibit the effects of bacterial toxins, and inhibit pathogen invasion to immune cell, as well as they can be used in drug delivery systems. The focus of this review is on the therapeutic applications of aptamers in infections. In this regard, an introduction of infections and related challenges were presented, first. Then, aptamer definition and selection, with a brief history of aptamers development against various pathogens and toxins were reviewed. Diverse strategies of aptamer application in drug delivery, as well as, the effect of aptamers on the immune system, as the main natural agents of human defense against pathogens, were also discussed. Finally, the future trends in clinical applications of this technology were discussed.">
         <i>abst: </i>Today, the treatment of bacter ...
        </span>
       </div>
      </td>
      <td/>
      <td>
       <span>13</span>
      </td>
      <td>
       <span>4</span>
      </td>
      <td>
       <span>13</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6958603" target="separate" title="remote PDF on server">PMC6958603</a>
       <a href="PMC6958603/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="An evaluation of the Zambia influenza sentinel surveillance system, 2011-2017.">
         <i>titl: </i>An evaluation of the Zambia in ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:Over the past decade, influenza surveillance has been established in several African countries including Zambia. However, information on the on data quality and reliability of established influenza surveillance systems in Africa are limited. Such information would enable countries to assess the performance of their surveillance systems, identify shortfalls for improvement and provide evidence of data reliability for policy making and public health interventions. METHODS:We used the Centers for Disease Control and Prevention guidelines to evaluate the performance of the influenza surveillance system (ISS) in Zambia during 2011-2017 using 9 attributes: (i) data quality and completeness, (ii) timeliness, (iii) representativeness, (iv) flexibility, (v) simplicity, (vi) acceptability, (vii) stability, (viii) utility, and (ix) sustainability. Each attribute was evaluated using pre-defined indicators. For each indicator we obtained the proportion (expressed as percentage) of the outcome of interest over the total. A scale from 1 to 3 was used to provide a score for each attribute as follows: &lt;â€‰60% (as obtained in the calculation above) scored 1 (weak performance); 60-79% scored 2 (moderate performance); â‰¥80% scored 3 (good performance). An overall score for each attribute and the ISS was obtained by averaging the scores of all evaluated attributes. RESULTS:The overall mean score for the ISS in Zambia was 2.6. Key strengths of the system were the quality of data generated (score: 2.9), its flexibility (score: 3.0) especially to monitor viral pathogens other than influenza viruses, its simplicity (score: 2.8), acceptability (score: 3.0) and stability (score: 2.6) over the review period and its relatively low cost ($310,000 per annum). Identified weaknesses related mainly to geographic representativeness (score: 2.0), timeliness (score: 2.5), especially in shipment of samples from remote sites, and sustainability (score: 1.0) in the absence of external funds. CONCLUSIONS:The system performed moderately well in our evaluation. Key improvements would include improvements in the timeliness of samples shipments and geographical coverage. However, these improvements would result in increased cost and logistical complexity. The ISSS in Zambia is largely reliant on external funds and the acceptability of maintaining the surveillance system through national funds would require evaluation.">
         <i>abst: </i>BACKGROUND:Over the past decad ...
        </span>
       </div>
      </td>
      <td>
       <span>4</span>
      </td>
      <td>
       <span>3</span>
      </td>
      <td>
       <span>8</span>
      </td>
      <td>
       <span>18</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6961264" target="separate" title="remote PDF on server">PMC6961264</a>
       <a href="PMC6961264/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="The impact of repeated vaccination using 10-year vaccination history on protection against influenza in older adults: a test-negative design study across the 2010/11 to 2015/16 influenza seasons in Ontario, Canada.">
         <i>titl: </i>The impact of repeated vaccina ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="IntroductionAnnual influenza vaccination is recommended for older adults, but evidence regarding the impact of repeated vaccination has been inconclusive.AimWe investigated vaccine effectiveness (VE) against laboratory-confirmed influenza and the impact of repeated vaccination over 10 previous seasons on current season VE among older adults.MethodsWe conducted an observational test-negative study in community-dwelling adults agedâ€‰&gt;â€‰65â€‰years in Ontario, Canada for the 2010/11 to 2015/16 seasons by linking laboratory and health administrative data. We estimated VE using multivariable logistic regression. We assessed the impact of repeated vaccination by stratifying by previous vaccination history.ResultsWe included 58,304 testing episodes for respiratory viruses, with 11,496 (20%) testing positive for influenza and 31,004 (53%) vaccinated. Adjusted VE against laboratory-confirmed influenza for the six seasons combined was 21% (95%â€‰confidence interval (CI):â€‰18â€‰toâ€‰24%). Patients who were vaccinated in the current season, but had received no vaccinations in the previous 10 seasons, had higher current season VE (34%;â€‰95%CI:â€‰9 to 52%) than patients who had received 1-3 (26%;â€‰95%CI:â€‰13â€‰toâ€‰37%), 4-6 (24%;â€‰95%CI:â€‰15â€‰toâ€‰33%), 7-8 (13%;â€‰95%CI:â€‰2â€‰toâ€‰22%), or 9-10 (7%;â€‰95%CI:â€‰-4â€‰toâ€‰16%) vaccinations (trend test pâ€‰=â€‰0.001). All estimates were higher after correcting for misclassification of current season vaccination status. For patients who were not vaccinated in the current season, residual protection rose significantly with increasing numbers of vaccinations received previously.ConclusionsAlthough VE appeared to decrease with increasing numbers of previous vaccinations, current season vaccination likely provides some protection against influenza regardless of the number of vaccinations received over the previous 10 influenza seasons.">
         <i>abst: </i>IntroductionAnnual influenza v ...
        </span>
       </div>
      </td>
      <td>
       <span>2</span>
      </td>
      <td>
       <span>4</span>
      </td>
      <td>
       <span>10</span>
      </td>
      <td>
       <span>17</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6969405" target="separate" title="remote PDF on server">PMC6969405</a>
       <a href="PMC6969405/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Investigations into the presence of nidoviruses in pythons.">
         <i>titl: </i>Investigations into the presen ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:Pneumonia and stomatitis represent severe and often fatal diseases in different captive snakes. Apart from bacterial infections, paramyxo-, adeno-, reo- and arenaviruses cause these diseases. In 2014, new viruses emerged as the cause of pneumonia in pythons. In a few publications, nidoviruses have been reported in association with pneumonia in ball pythons and a tiger python. The viruses were found using new sequencing methods from the organ tissue of dead animals. METHODS:Severe pneumonia and stomatitis resulted in a high mortality rate in a captive breeding collection of green tree pythons. Unbiased deep sequencing lead to the detection of nidoviral sequences. A developed RT-qPCR was used to confirm the metagenome results and to determine the importance of this virus. A total of 1554 different boid snakes, including animals suffering from respiratory diseases as well as healthy controls, were screened for nidoviruses. Furthermore, in addition to two full-length sequences, partial sequences were generated from different snake species. RESULTS:The assembled full-length snake nidovirus genomes share only an overall genome sequence identity of less than 66.9% to other published snake nidoviruses and new partial sequences vary between 99.89 and 79.4%. Highest viral loads were detected in lung samples. The snake nidovirus was not only present in diseased animals, but also in snakes showing no typical clinical signs. CONCLUSIONS:Our findings further highlight the possible importance of snake nidoviruses in respiratory diseases and proof multiple circulating strains with varying disease potential. Nidovirus detection in clinical healthy individuals might represent testing during the incubation period or reconvalescence. Our investigations show new aspects of nidovirus infections in pythons. Nidoviruses should be included in routine diagnostic workup of diseased reptiles.">
         <i>abst: </i>BACKGROUND:Pneumonia and stoma ...
        </span>
       </div>
      </td>
      <td>
       <span>5</span>
      </td>
      <td>
       <span>9</span>
      </td>
      <td>
       <span>1</span>
      </td>
      <td>
       <span>27</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6970282" target="separate" title="remote PDF on server">PMC6970282</a>
       <a href="PMC6970282/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Tonsillar hypertrophy and prolapse in a child - is epiglottitis a predisposing factor for sudden unexpected death?">
         <i>titl: </i>Tonsillar hypertrophy and prol ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:Tonsillitis, with associated tonsillar hypertrophy, is a common disease of childhood, yet it is rarely associated with sudden death due to airway obstruction. Lethal complications involving the inflamed tonsils include haemorrhage, retropharyngeal abscess and disseminated sepsis. CASE PRESENTATION:We report on a case of sudden and unexpected death in an 8-year-old female who was diagnosed with and treated for tonsillitis. The child was diagnosed with acute tonsillitis 2 days prior to her collapse and was placed on a course of oral antibiotics. There were no signs of upper or lower airway obstruction. She was found to be unresponsive by her caregiver and gasping for air in her bed in the early hours of the second morning after the start of treatment. Autopsy showed massive and symmetrically enlarged palatine tonsils. The tonsils filled the pharynx almost completely. The epiglottis and laryngeal mucosa at the base of the epiglottis in the vicinity of the aryepiglottic membrane and the superior aspect of the larynx displayed red-purple discoloration, with mucosal swelling and edema. Histological examination of the palatine tonsils revealed prominent lymphoid hyperplasia, but no evidence of acute inflammation. CONCLUSION:Palatine tonsillar hypertrophy in infants is a common feature of both viral and bacterial tonsillitis and has been postulated as a possible risk factor for Sudden and Unexplained Death in Infancy (SUDI), based on the theory of mechanical impediment of breathing by narrowing of the upper airway. The rounded shape of the tonsils may facilitate some airflow past the enlarged structures and hence protect against asphyxial death when the enlarged tonsils fill the laryngo-pharynx. Epiglottal and proximal laryngeal edema may play a more significant role in asphyxial unexpected deaths in cases of tonsillitis with tonsillar hypertrophy than previously suspected. This focusses the importance of careful examination of the epiglottis and proximal laryngeal mucosa, as part of a thorough examination of the laryngo-pharynx in cases of sudden death associated with tonsillar hypertrophy.">
         <i>abst: </i>BACKGROUND:Tonsillitis, with a ...
        </span>
       </div>
      </td>
      <td/>
      <td>
       <span>4</span>
      </td>
      <td/>
      <td>
       <span>2</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6979112" target="separate" title="remote PDF on server">PMC6979112</a>
       <a href="PMC6979112/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Proceedings 31st Symposium ESVN-ECVN.">
         <i>titl: </i>Proceedings 31st Symposium ESV ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="null">
         <i>abst: </i>null
        </span>
       </div>
      </td>
      <td>
       <span>11</span>
      </td>
      <td>
       <span>54</span>
      </td>
      <td>
       <span>23</span>
      </td>
      <td>
       <span>103</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6979267" target="separate" title="remote PDF on server">PMC6979267</a>
       <a href="PMC6979267/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Research Communications of the 29th ECVIM-CA Congress.">
         <i>titl: </i>Research Communications of the ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="null">
         <i>abst: </i>null
        </span>
       </div>
      </td>
      <td>
       <span>31</span>
      </td>
      <td>
       <span>110</span>
      </td>
      <td>
       <span>51</span>
      </td>
      <td>
       <span>386</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC6981968" target="separate" title="remote PDF on server">PMC6981968</a>
       <a href="PMC6981968/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Effect of Information Disclosure Policy on Control of Infectious Disease: MERS-CoV Outbreak in South Korea.">
         <i>titl: </i>Effect of Information Disclosu ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="This study examined the effect of disclosing a list of hospitals with Middle East respiratory syndrome coronavirus (MERS-CoV) patients on the number of laboratory-confirmed MERS-CoV cases in South Korea. MERS-CoV data from 20 May 2015 to 5 July 2015 were from the Korean Ministry of Health &amp; Welfare website and analyzed using segmented linear autoregressive error models for interrupted time series. This study showed that the number of laboratory-confirmed cases was increased by 9.632 on 5 June (p &lt; 0.001). However, this number was significantly decreased following disclosure of a list of hospitals with MERS-CoV cases (Estimate = -0.699; p &lt; 0.001). Disclosing the list of hospitals exposed to MERS-CoV was critical to the prevention of further infection. It reduced the number of confirmed MERS-CoV cases. Thus, providing accurate and timely information is a key to critical care response.">
         <i>abst: </i>This study examined the effect ...
        </span>
       </div>
      </td>
      <td>
       <span>5</span>
      </td>
      <td>
       <span>2</span>
      </td>
      <td>
       <span>4</span>
      </td>
      <td>
       <span>2</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC7000305" target="separate" title="remote PDF on server">PMC7000305</a>
       <a href="PMC7000305/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Identifying factors and target preventive therapies for Middle East Respiratory Syndrome sucsibtable patients.">
         <i>titl: </i>Identifying factors and target ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Background:Middle East Respiratory Syndrome (MERS) is a respiratory disease caused by a novel coronavirus that was identified in 2012 in Saudi Arabia. It is associated with significant mortality and morbidity. We identified factors associated with the Middle East Respiratory Syndrome-Coronavirus (MERS-CoV) infection among suspected cases presented with sign and symptoms of upper respiratory infection or exposure to the virus. We also looked at the impact of medication history on virus transmission. Method:We included subjects with suspected MERS-CoV infection and confirmed cases of MERS infection. Subjects were excluded if there were any missing data that prevent the final analysis. Descriptive statistics were used to report demographic data. Percentages and frequencies were used to summarize the categorical variables, while means and standard deviations were calculated for continuous variables. Logistic regression was used to assess the risk of MERS-CoV infection among the suspected cases. A value of pÂ &lt;Â 0.05 was considered statistically significant. Results:A total of 16,189 suspected cases were identified, complete data were analyzed for 3154 to assess factors that are independently associated with MERS-CoV infection. MERS-CoV infection was associated with age (adjusted odds ratio [AOR]Â =Â 1.06; 95% CI [1.02-1.098], P-valueÂ =Â 0.004), male gender (AORÂ =Â 1.617; 95% CI [1.365-1.77], P-valueÂ &lt;Â 0.001) and diabetes (AORÂ =Â 1.68; 95% CI [1.346-1.848], P-valueÂ =Â 0.002. There was no significant association with the other comorbidities. Medication history was not associated with an increase or decrease the likelihood of the infection. Conclusions:MERS-Cov infection is more common in male, advanced age and diabetes. No medications were associated with an increase or decrease the likelihood of the infection. This is important to focus on screening and detection to this patient population.">
         <i>abst: </i>Background:Middle East Respira ...
        </span>
       </div>
      </td>
      <td>
       <span>5</span>
      </td>
      <td>
       <span>5</span>
      </td>
      <td>
       <span>6</span>
      </td>
      <td>
       <span>1</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC7006675" target="separate" title="remote PDF on server">PMC7006675</a>
       <a href="PMC7006675/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Polymorphisms in dipeptidyl peptidase 4 reduce host cell entry of Middle East respiratory syndrome coronavirus.">
         <i>titl: </i>Polymorphisms in dipeptidyl pe ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Middle East respiratory syndrome (MERS) coronavirus (MERS-CoV) causes a severe respiratory disease in humans. The MERS-CoV spike (S) glycoprotein mediates viral entry into target cells. For this, MERS-CoV S engages the host cell protein dipeptidyl peptidase 4 (DPP4, CD26) and the interface between MERS-CoV S and DPP4 has been resolved on the atomic level. Here, we asked whether naturally-occurring polymorphisms in DPP4, that alter amino acid residues required for MERS-CoV S binding, influence cellular entry of MERS-CoV. By screening of public databases, we identified fourteen such polymorphisms. Introduction of the respective mutations into DPP4 revealed that all except one (Î”346-348) were compatible with robust DPP4 expression. Four polymorphisms (K267E, K267N, A291P and Î”346-348) strongly reduced binding of MERS-CoV S to DPP4 and S protein-driven host cell entry, as determined using soluble S protein and S protein bearing rhabdoviral vectors, respectively. Two polymorphisms (K267E and A291P) were analyzed in the context of authentic MERS-CoV and were found to attenuate viral replication. Collectively, we identified naturally-occurring polymorphisms in DPP4 that negatively impact cellular entry of MERS-CoV and might thus modulate MERS development in infected patients.">
         <i>abst: </i>Middle East respiratory syndro ...
        </span>
       </div>
      </td>
      <td>
       <span>5</span>
      </td>
      <td>
       <span>10</span>
      </td>
      <td>
       <span>5</span>
      </td>
      <td>
       <span>17</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC7017508" target="separate" title="remote PDF on server">PMC7017508</a>
       <a href="PMC7017508/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Annual and seasonal patterns in etiologies of pediatric community-acquired pneumonia due to respiratory viruses and Mycoplasma pneumoniae requiring hospitalization in South Korea.">
         <i>titl: </i>Annual and seasonal patterns i ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:Community-acquired pneumonia (CAP) is one of the leading worldwide causes of childhood morbidity and mortality. Its disease burden varies by age and etiology and is time dependent. We aimed to investigate the annual and seasonal patterns in etiologies of pediatric CAP requiring hospitalization. METHODS:We conducted a retrospective study in 30,994 children (aged 0-18â€‰years) with CAP between 2010 and 2015 at 23 nationwide hospitals in South Korea. Mycoplasma pneumoniae (MP) pneumonia was clinically classified as macrolide-sensitive MP, macrolide-less effective MP (MLEP), and macrolide-refractory MP (MRMP) based on fever duration after initiation of macrolide treatment, regardless of the results of in vitro macrolide sensitivity tests. RESULTS:MP and respiratory syncytial virus (RSV) were the two most commonly identified pathogens of CAP. With the two epidemics of MP pneumonia (2011 and 2015), the rates of clinical MLEP and MRMP pneumonia showed increasing trends of 36.4% of the total MP pneumonia. In children &lt;â€‰2â€‰years of age, RSV (34.0%) was the most common cause of CAP, followed by MP (9.4%); however, MP was the most common cause of CAP in children aged 2-18â€‰years of age (45.3%). Systemic corticosteroid was most commonly administered for MP pneumonia. The rate of hospitalization in intensive care units was the highest for RSV pneumonia, and ventilator care was most commonly needed in cases of adenovirus pneumonia. CONCLUSIONS:The present study provides fundamental data to establish public health policies to decrease the disease burden due to CAP and improve pediatric health.">
         <i>abst: </i>BACKGROUND:Community-acquired ...
        </span>
       </div>
      </td>
      <td>
       <span>1</span>
      </td>
      <td>
       <span>4</span>
      </td>
      <td>
       <span>19</span>
      </td>
      <td>
       <span>14</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC7019290" target="separate" title="remote PDF on server">PMC7019290</a>
       <a href="PMC7019290/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Virus Metagenomics in Farm Animals: A Systematic Review.">
         <i>titl: </i>Virus Metagenomics in Farm Ani ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="A majority of emerging infectious diseases are of zoonotic origin. Metagenomic Next-Generation Sequencing (mNGS) has been employed to identify uncommon and novel infectious etiologies and characterize virus diversity in human, animal, and environmental samples. Here, we systematically reviewed studies that performed viral mNGS in common livestock (cattle, small ruminants, poultry, and pigs). We identified 2481 records and 120 records were ultimately included after a first and second screening. Pigs were the most frequently studied livestock and the virus diversity found in samples from poultry was the highest. Known animal viruses, zoonotic viruses, and novel viruses were reported in available literature, demonstrating the capacity of mNGS to identify both known and novel viruses. However, the coverage of metagenomic studies was patchy, with few data on the virome of small ruminants and respiratory virome of studied livestock. Essential metadata such as age of livestock and farm types were rarely mentioned in available literature, and only 10.8% of the datasets were publicly available. Developing a deeper understanding of livestock virome is crucial for detection of potential zoonotic and animal pathogens and One Health preparedness. Metagenomic studies can provide this background but only when combined with essential metadata and following the &quot;FAIR&quot; (Findable, Accessible, Interoperable, and Reusable) data principles.">
         <i>abst: </i>A majority of emerging infecti ...
        </span>
       </div>
      </td>
      <td>
       <span>16</span>
      </td>
      <td>
       <span>10</span>
      </td>
      <td>
       <span>3</span>
      </td>
      <td>
       <span>21</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC7028525" target="separate" title="remote PDF on server">PMC7028525</a>
       <a href="PMC7028525/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Silicone pneumonitis after gluteal filler: a case report and literature review.">
         <i>titl: </i>Silicone pneumonitis after glu ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Liquid silicone (polydimethylsiloxane) is an inert material that is commonly used for cosmetic purpose. Silicone embolization syndrome (SES) can rapidly progress to pneumonitis as a consequence of the injection of nonmedical-grade liquid silicone. We describe a case of severe silicone pneumonitis complicated with acute respiratory distress syndrome and bilateral pneumothorax secondary to silicone gluteal augmentation. In this case report, we aim to discuss our experience and approach in managing an uncommon case of SES.">
         <i>abst: </i>Liquid silicone (polydimethyls ...
        </span>
       </div>
      </td>
      <td>
       <span>1</span>
      </td>
      <td>
       <span>10</span>
      </td>
      <td>
       <span>1</span>
      </td>
      <td>
       <span>5</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC7029452" target="separate" title="remote PDF on server">PMC7029452</a>
       <a href="PMC7029452/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="First cases of coronavirus disease 2019 (COVID-19) in France: surveillance, investigations and control measures, January 2020.">
         <i>titl: </i>First cases of coronavirus dis ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="A novel coronavirus (severe acute respiratory syndrome coronavirus 2, SARS-CoV-2) causing a cluster of respiratory infections (coronavirus disease 2019, COVID-19) in Wuhan, China, was identified on 7 January 2020. The epidemic quickly disseminated from Wuhan and as at 12 February 2020, 45,179 cases have been confirmed in 25 countries, including 1,116 deaths. Strengthened surveillance was implemented in France on 10 January 2020 in order to identify imported cases early and prevent secondary transmission. Three categories of risk exposure and follow-up procedure were defined for contacts. Three cases of COVID-19 were confirmed on 24 January, the first cases in Europe. Contact tracing was immediately initiated. Five contacts were evaluated as at low risk of exposure and 18 at moderate/high risk. As at 12 February 2020, two cases have been discharged and the third one remains symptomatic with a persistent cough, and no secondary transmission has been identified. Effective collaboration between all parties involved in the surveillance and response to emerging threats is required to detect imported cases early and to implement adequate control measures.">
         <i>abst: </i>A novel coronavirus (severe ac ...
        </span>
       </div>
      </td>
      <td>
       <span>2</span>
      </td>
      <td>
       <span>5</span>
      </td>
      <td>
       <span>9</span>
      </td>
      <td>
       <span>7</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC7029759" target="separate" title="remote PDF on server">PMC7029759</a>
       <a href="PMC7029759/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Therapeutic strategies in an outbreak scenario to treat the novel coronavirus originating in Wuhan, China.">
         <i>titl: </i>Therapeutic strategies in an o ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="A novel coronavirus (2019-nCoV) originating in Wuhan, China presents a potential respiratory viral pandemic to the world population. Current efforts are focused on containment and quarantine of infected individuals. Ultimately, the outbreak could be controlled with a protective vaccine to prevent 2019-nCoV infection. While vaccine research should be pursued intensely, there exists today no therapy to treat 2019-nCoV upon infection, despite an urgent need to find options to help these patients and preclude potential death. Herein, I review the potential options to treat 2019-nCoV in patients, with an emphasis on the necessity for speed and timeliness in developing new and effective therapies in this outbreak. I consider the options of drug repurposing, developing neutralizing monoclonal antibody therapy, and an oligonucleotide strategy targeting the viral RNA genome, emphasizing the promise and pitfalls of these approaches. Finally, I advocate for the fastest strategy to develop a treatment now, which could be resistant to any mutations the virus may have in the future. The proposal is a biologic that blocks 2019-nCoV entry using a soluble version of the viral receptor, angiotensin-converting enzyme 2 (ACE2), fused to an immunoglobulin Fc domain, providing a neutralizing antibody with maximal breath to avoid any viral escape, while also helping to recruit the immune system to build lasting immunity. The sequence of the ACE2-Fc protein is provided to investigators, allowing its possible use in recombinant protein expression systems to start producing drug today to treat patients under compassionate use, while formal clinical trials are later undertaken. Such a treatment could help infected patients before a protective vaccine is developed and widely available in the coming months to year(s).">
         <i>abst: </i>A novel coronavirus (2019-nCoV ...
        </span>
       </div>
      </td>
      <td>
       <span>8</span>
      </td>
      <td>
       <span>12</span>
      </td>
      <td>
       <span>3</span>
      </td>
      <td>
       <span>18</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC7033854" target="separate" title="remote PDF on server">PMC7033854</a>
       <a href="PMC7033854/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Characteristics of fever and response to antipyretic therapy in military personnel with adenovirus-positive community-acquired pneumonia.">
         <i>titl: </i>Characteristics of fever and r ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:In 2014, an outbreak of adenoviral pneumonia occurred in the Korean military training center. However, there are limited data on the characteristics of the fever and its response to antipyretic therapy in immunocompetent adults with adenovirus-positive community-acquired pneumonia (CAP). METHODS:The medical records of the patients who were admitted to the Armed Forces Chuncheon Hospital for the treatment of CAP between January 2014 and December 2016 were retrospectively analyzed. The patients were divided into three groups, namely, the adenovirus-positive (Adv) group, the adenovirus-negative (Non-Adv) group and the unknown pathogen group, according to the results of a polymerase chain reaction (PCR) test and sputum culture used to measure adenovirus and other bacteria or viruses in respiratory specimens. We evaluated and compared the demographics, clinicolaboratory findings and radiological findings upon admission between the two groups. RESULTS:Out of the 251 military personnel with CAP during the study periods, 67 were classified into the Adv group, while 134 were classified into the Non-Adv group and 50 were classified into the unknown pathogen group. The patients in the Adv group had a longer duration of fever after admission (3.2â€‰Â±â€‰1.6 vs. 1.9â€‰Â±â€‰1.2 vs. 2.2â€‰Â±â€‰1.5â€‰days, P = 0.018) and symptom onset (5.8â€‰Â±â€‰2.2 vs. 3.9â€‰Â±â€‰2.5 vs. 3.7â€‰Â±â€‰2.0â€‰days, Pâ€‰=â€‰0.006) than patients in the Non-Adv and unknown pathogen groups, respectively. The patients in the Adv group had a higher mean temperature at admission (37.8â€‰Â±â€‰0.3 vs. 37.3â€‰Â±â€‰0.3 vs. 37.3 Â± 0.3, Pâ€‰=â€‰0.005), and more patients were observed over 40 and 39 to 40(14.9% vs. 2.2% vs. 4.0%, 35.8% vs. 3.7% vs. 6.0%, Pâ€‰&lt;â€‰Â 0.001) than those in the Non-Adv and unknown pathogen groups, respectively. The Adv group more commonly had no response or exhibited adverse events after antipyretic treatment compared to the Non-Adv group (17.9% vs. 1.5%, 35.0% vs. 4.3%, Pâ€‰&lt;â€‰Â 0.001, P = 0.05, respectively). In addition, the time from admission to overall clinical stabilization was significantly longer in the patients in the Adv group than in those in the Non-Adv group (4.3â€‰Â±â€‰2.8 vs. 2.9â€‰Â±â€‰1.8â€‰days, P = 0.034, respectively). Furthermore, no significant difference in the length of hospital stay was observed between the two groups, and no patient died in either group. CONCLUSION:In this study, Adv-positive CAP in immunocompetent military personnel patients had distinct fever characteristics and responses to antipyretic treatment.">
         <i>abst: </i>BACKGROUND:In 2014, an outbrea ...
        </span>
       </div>
      </td>
      <td>
       <span>2</span>
      </td>
      <td>
       <span>12</span>
      </td>
      <td>
       <span>7</span>
      </td>
      <td>
       <span>21</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC7034298" target="separate" title="remote PDF on server">PMC7034298</a>
       <a href="PMC7034298/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Multiple Family Members With Delayed Cord Separtion and Combined Immunodeficiency With Novel Mutation in IKBKB.">
         <i>titl: </i>Multiple Family Members With D ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Background: Inhibitor of kappa kinase 2 (IKK2) deficiency is a recently described combined immunodeficiency. It undermines the nuclear factor-kappa B (NF-ÎºB) activation pathway. Methods: The clinical and immunological data of four patients diagnosed with combined immunodeficiency (CID) from two related Saudi families were collected. Autozygosity mapping of all available members and whole exome sequencing of the index case were performed to define the genetic etiology. Results: The patients had early onset (2-4 months of age) severe infections caused by viruses, bacteria, mycobacteria, and fungi. They all had hypogammaglobulinemia and low absolute lymphocyte count. Their lymphocytes failed to respond to PHA mitogen stimulation. A novel homozygous non-sense mutation in the IKBKB gene, c.850C&gt;T (p. Arg284*) was identified in the index patient and segregated with the disease in the rest of the family. He underwent hematopoietic stem cell transplantation (HSCT) from a fully matched sibling with no conditioning. The other three patients succumbed to their disease. Interestingly, all patients had delayed umbilical cord separation. Conclusion: IKK2 deficiency causes CID with high mortality. Immune reconstitution with HSCT should be considered as early as possible. Delayed umbilical cord separation in CID patients may be a clue to IKK2 deficiency.">
         <i>abst: </i>Background: Inhibitor of kappa ...
        </span>
       </div>
      </td>
      <td>
       <span>4</span>
      </td>
      <td>
       <span>11</span>
      </td>
      <td>
       <span>6</span>
      </td>
      <td>
       <span>2</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC7041086" target="separate" title="remote PDF on server">PMC7041086</a>
       <a href="PMC7041086/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Late viral or bacterial respiratory infections in lung transplanted patients: impact on respiratory function.">
         <i>titl: </i>Late viral or bacterial respir ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:Respiratory infections are a major threat for lung recipients. We aimed to compare with a monocentric study the impact of late viral and bacterial respiratory infections on the graft function. METHODS:Patients, who survived 6 months or more following lung transplantation that took place between 2009 and 2014, were classified into three groups: a viral infection group (VIG) (without any respiratory bacteria), a bacterial infection group (BIG) (with or without any respiratory viruses), and a control group (CG) (no documented infection). Chronic lung allograft dysfunction (CLAD) and acute rejection were analysed 6 months after the inclusion in the study. RESULTS:Among 99 included lung recipients, 57 (58%) had at least one positive virological respiratory sample during the study period. Patients were classified as follows: 38 in the VIG, 25 in the BIG (among which 19 co-infections with a virus) and 36 in the CG. The BIG presented a higher initial deterioration in lung function (pâ€‰=â€‰0.05) than the VIG. But 6 months after the infection, only the VIG presented a median decrease of forced expiratory volume in 1â€‰s; -â€‰35â€‰mL (IQR; -â€‰340; +â€‰80) in the VIG, +â€‰140â€‰mL (+â€‰60;+â€‰330) in the BIG andâ€‰+â€‰10 (-â€‰84;+â€‰160) in the CG, pâ€‰&lt;â€‰0.01. Acute rejection was more frequent in the VIG (nâ€‰=â€‰12 (32%)), than the BIG (nâ€‰=â€‰6 (24%)) and CG (nâ€‰=â€‰3 (8%)), pâ€‰&lt;â€‰0.05, despite presenting no more CLAD (pâ€‰=â€‰0.21). CONCLUSIONS:Despite a less severe initial presentation, single viral respiratory infections seem to lead to a greater deterioration in lung function, and to more acute rejection, than bacterial infections.">
         <i>abst: </i>BACKGROUND:Respiratory infecti ...
        </span>
       </div>
      </td>
      <td/>
      <td>
       <span>3</span>
      </td>
      <td>
       <span>2</span>
      </td>
      <td>
       <span>15</span>
      </td>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC7046816" target="separate" title="remote PDF on server">PMC7046816</a>
       <a href="PMC7046816/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="A 25-Year-Old Sample Contributes the Complete Genome Sequence of Avian Coronavirus Vaccine Strain ArkDPI, Reisolated from Commercial Broilers in the United States.">
         <i>titl: </i>A 25-Year-Old Sample Contribut ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="Here, we report the complete genome sequence of Avian coronavirus strain ArkDPI of the GI-9 lineage, isolated from broiler chickens in North Georgia in 1994. This is the complete genome sequence of this vaccine strain, reisolated from broilers in the United States.">
         <i>abst: </i>Here, we report the complete g ...
        </span>
       </div>
      </td>
      <td>
       <span>5</span>
      </td>
      <td>
       <span>1</span>
      </td>
      <td>
       <span>4</span>
      </td>
      <td/>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC7047369" target="separate" title="remote PDF on server">PMC7047369</a>
       <a href="PMC7047369/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="Q&amp;A: TheÂ novel coronavirus outbreakÂ causing COVID-19.">
         <i>titl: </i>Q&amp;A: TheÂ novel coronavirus out ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="null">
         <i>abst: </i>null
        </span>
       </div>
      </td>
      <td>
       <span>5</span>
      </td>
      <td>
       <span>4</span>
      </td>
      <td>
       <span>5</span>
      </td>
      <td/>
     </tr>
     <tr>
      <td title="fixme">
       <a href="http://europepmc.org/articles/PMC7047374" target="separate" title="remote PDF on server">PMC7047374</a>
       <a href="PMC7047374/scholarly.html" target="separate" title="HTML on local server">local</a>
      </td>
      <td>
       <div>
        <span class="titl" title="A mathematical model for simulating the phase-based transmissibility of a novel coronavirus.">
         <i>titl: </i>A mathematical model for simul ...
        </span>
        <br/>
        <span class="date" title="">
         <i>date: </i>
        </span>
        <span class="jour" title="">
         <i>jour: </i>
        </span>
        <br/>
        <span class="abst" title="BACKGROUND:As reported by the World Health Organization, a novel coronavirus (2019-nCoV) was identified as the causative virus of Wuhan pneumonia of unknown etiology by Chinese authorities on 7 January, 2020. The virus was named as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by International Committee on Taxonomy of Viruses on 11 February, 2020. This study aimed to develop a mathematical model for calculating the transmissibility of the virus. METHODS:In this study, we developed a Bats-Hosts-Reservoir-People transmission network model for simulating the potential transmission from the infection source (probably be bats) to the human infection. Since the Bats-Hosts-Reservoir network was hard to explore clearly and public concerns were focusing on the transmission from Huanan Seafood Wholesale Market (reservoir) to people, we simplified the model as Reservoir-People (RP) transmission network model. The next generation matrix approach was adopted to calculate the basic reproduction number (R0) from the RP model to assess the transmissibility of the SARS-CoV-2. RESULTS:The value of R0 was estimated of 2.30 from reservoir to person and 3.58 from person to person which means that the expected number of secondary infections that result from introducing a single infected individual into an otherwise susceptible population was 3.58. CONCLUSIONS:Our model showed that the transmissibility of SARS-CoV-2 was higher than the Middle East respiratory syndrome in the Middle East countries, similar to severe acute respiratory syndrome, but lower than MERS in the Republic of Korea.">
         <i>abst: </i>BACKGROUND:As reported by the ...
        </span>
       </div>
      </td>
      <td>
       <span>4</span>
      </td>
      <td>
       <span>8</span>
      </td>
      <td>
       <span>8</span>
      </td>
      <td>
       <span>6</span>
      </td>
     </tr>
    </tbody>
    <tfoot/>
   </table>
  </div>
 </body>
</html>
