<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="publisher-id" title="publisher-id">biohorizons</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Bioscience Horizons: The International Journal of Student Research</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1754-7431</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7189836">7189836</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1093/biohorizons/hzw010">10.1093/biohorizons/hzw010</a></span><span class="publisher-id" title="publisher-id">hzw010</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Recent advances in diagnostic testing for viral infections</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Souf</span><span class="given-names" tagx="given-names" title="given-names">Selma</span></span><span class="email" tagx="email" title="email">soufselma@gmail.com</span></span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">*</span><b>Corresponding author:</b> College of Science and Technology, Nottingham Trent University, Clifton campus NG11 8NS, Nottingham, UK. Email: <span class="email" tagx="email" title="email">soufselma@gmail.com</span></div> <div class="fn-type-" title=""> <p><b>Supervisor:</b> Dr Jody Winter, College of Science and Technology, Nottingham Trent University, Clifton campus NG11 8NS, Nottingham, UK.</p> </div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2016-11-11</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2016-11-11</span></span><span class="volume" tagx="volume" title="volume">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">hzw010</span><span class="history" title="history"><span class="received" title="received">received: 2016-2-22</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2016</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a href="http://creativecommons.org/licenses/by-nc/4.0/">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>Viral infectious diseases represent an important portion of global public health concerns with thousands of deaths annually. From serious pandemics and highly contagious infections to common influenza episodes, clinical prognosis often relies on early detection of the infectious agent. Thus, effective identification of viral pathogens is needed to help prevent transmission, set up appropriate therapy, monitor response to treatment and lead to efficient disease management and control. The aim of this review is to outline some of the recent technological advances in viral identification, including polymerase chain reaction, mass spectrometry and next-generation sequencing, and how they are applied in the diagnosis and management of viral infections. These powerful tools combine rapidity and efficiency in detecting viral pathogens and have revolutionized the field of clinical diagnostics. However, a number of drawbacks such as high cost have limited their use in many laboratories, particularly in resource-limited settings. On the contrary, the advent of microfluidic technology has attracted increasing interest from biomedical research groups, and could represent a challenging alternative to diagnose viral infections at lower cost.</p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
viral infections</div> <div class="kwd" title="kwd">
laboratory testing</div> <div class="kwd" title="kwd">
immunoassay</div> <div class="kwd" title="kwd">
polymerase chain reaction</div> <div class="kwd" title="kwd">
next-generation sequencing</div> <div class="kwd" title="kwd">
mass spectrometry</div> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Global pandemics are serious threats to human life. While well-established and characterized viruses such as The human immunodeficiency virus (HIV) and Hepatitis are still killing millions of people, the emerging viruses are also problematic and have caused several serious outbreaks in the recent years. For example, the Severe Acute Respiratory Syndrome-Coronavirus (SARS-CoV) in 2002â€"2003, Swine Influenza A (H1N1) in 2009 and Ebola Haemorrhagic fever outbreak in 2014 which has caused thousands of deaths in West Africa.</p> <p>Morbidity and mortality rates are significantly high. Thirty-five million people were infected with HIV in 2013, and 350â€"400 millions chronic carriers of Hepatitis B virus. According to the World Health Organization (WHO) report of 2014 (<a href="#hzw010C82">WHO, 2014</a>); more than 780 000 people die every year of Hepatitis B and up to 500 000 die of Hepatitis C-related liver diseases. The high prevalence of these diseases has raised the efforts for improving clinical diagnostics.</p> <p>Effective prevention and clinical management of infectious diseases are intimately linked to early and accurate screening of pathogens, not only by detecting the infectious particles in the organism but also by elucidating the aspects that confer resistance to therapy and immune escape profiles, including mutations and genotype disparity.</p> <p>Therefore, rapid diagnosis benefits patients in allowing timely therapy to prevent complications; and benefits public health by collecting data for epidemiological studies, to prevent outbreaks and spreading of diseases. In that context, the WHO has established many surveillance programs for disease control such as the global strategy for control and assessment of HIV drug resistance, the Global Influenza Surveillance and Response System for the control and monitoring of Influenza and the Global Policy on Viral Hepatitis.</p> <p>At a smaller scale, clinical laboratories are a crucial point for diagnosis of viral diseases by using a range of tools and machineries varying in cost and efficacy.</p> <p>In a rapidly growing world of technology, the industry is continuously delivering up-to-date instruments but many factors are limiting their implementation in healthcare settings with low income, which unfortunately delays global benefit. This review will describe some of these advanced testing methods, how their specific characteristics have revolutionized the field of laboratory diagnosis and what can be done to overcome their limitations.</p> </div> <div class="generalprinciplesofgoodlaboratorytesting" title="sec"> <div class="title" tagx="title" title="title">
General principles of good laboratory testing</div> <p>Rigorous and accurate interpretation of laboratory results guarantees effective clinical management of a disease and control of its propagation (<a href="#hzw010C40">Lemon <i>et al</i>., 2007</a>). However, erroneous diagnosis could lead to financial and human loss.</p> <p>In clinical testing of infectious diseases, it is crucial to determine precisely the presence or absence of the infectious agent or its corresponding antibodies, to prove current or past exposure. Therefore, the ability to say precisely whether the person is infected or not, and to determine the course of the infection has a positive impact on the therapeutic strategy.</p> <p>The usefulness and reliability of laboratory results depend on the <i>performance</i> and <i>operational</i> parameters of the intended test. The performance parameters directly relate to the results by estimating their <i>Accuracy, Precision, Sensitivity</i> and <i>Specificity</i> (<a href="#hzw010C39">Lalkhen and McCluskey, 2008</a>). While these are statistical values (percentages), they have different explanations and involve comparison with the reference method or â€˜gold standardâ€™ for the desired test (<a href="#hzw010C29">Guzman <i>et al</i>., 2010</a>).</p> <p><i>Accuracy</i> describes how close the obtained results are to those obtained with the reference method and it is expressed as a percentage of correct results. <i>Precision</i> refers to the reliable reproduction of one test on the same sample, and obtaining similar results.</p> <p>These two parameters must be regularly monitored using local quality control (QC) and quality assurance (QA) procedures in order to maintain reliability of the test. In perfect conditions, an ideal test would have 100% accuracy and 100% precision; however, external factors and methodological differences can cause small variations.</p> <p><i>Sensitivity</i> (also called the true positive rate) is the percentage of patients with confirmed infection (by the â€˜gold standardâ€™ method) who will have positive results. It is usually measured by the lower limit of detection of the analyte producing a positive result.</p> <p><i>Specificity</i> (also called the true negative rate) is a qualitative assessment, showing the capability of the test to distinguish target from non-target analyte. This measure is expressed as the percentage of infection-free patients who will have a negative result. The closer the values are to the reference, the higher the sensitivity and specificity of the test.</p> <p>On the contrary, operational parameters concern simplicity and ease in performing the test such as the turnaround time (TAT). TAT is a key performance indicator defined as the interval time between sample registration to result reporting. Sample preparation and any other pre-analytical steps are within this interval. Assay completion in less than 60 min is ideal so manufacturers aim to construct diagnosis instruments allowing shorter TAT, which is particularly beneficial for point-of-care settings (<a href="#hzw010C31">Hawkins, 2007</a>).</p> <p>The WHO has established ASSURED criteria (<i>A</i>ffordable, <i>S</i>ensitive, <i>S</i>pecific, <i>U</i>ser-friendly, <i>R</i>apid and robust, <i>E</i>quipment-free and <i>D</i>eliverable to end users) (<a href="#hzw010C85">Wu and Zaman, 2012</a>) for diagnostics in resources-limited point-of care settings (<a href="#hzw010C8">Blacksell, 2012</a>). The aim is to provide better management of the disease, such as immediate delivery of the results and rapid record of the disease status, to improve clinical decision-making.</p> </div> <div class="traditionallaboratorymethodsforthediagnosisofviralinfections" title="sec"> <div class="title" tagx="title" title="title">
Traditional laboratory methods for the diagnosis of viral infections</div> <p>For long time, clinical laboratories have relied on a diverse range of techniques to diagnose viral infections.</p> <p>In developed countries, <i>electron microscope </i>(<i>EM</i>) has for long time been considered an efficient tool for direct detection of viruses through visualization and counting of the viral particles in body fluids, stools or histopathologic samples. The identification is based on morphological characteristics specific to each virus family and requires a certain amount of viral particles (up to 106 particles/ml). However, specimen preparation that must be performed beforehand (<a href="#hzw010C26">Goldsmith and Miller, 2009</a>) may reduce the virus concentration which makes the analysis harder. In addition, electron microscopy requires substantial technical skills and expertise; the presence of unusual or look-alike structures such as cellular debris and organelles may confuse the electron microscopist with their virus-like shapes (<a href="#hzw010C21">Fauquet <i>et al</i>., 2005</a>).</p> <p>The combination of EM with culture-based methods has shown great contribution in the diagnosis of viral infections, along with serology testing for detection of antibodies targeted against the virus. These conventional methods are still fundamental practices in many hospital laboratories.</p> <p><i>Cell culture</i> is one of the most popular methods for isolating viruses using cell lines. These latter vary according to the targeted viruses (for example; rhesus monkey kidney cells are used for isolation of Influenza A virus).</p> <p>Evidence of virus growth is seen through the cytopathic effect (CPE) exhibiting specific characteristics and alterations of the cells (<a href="#hzw010C66">Robbins, Enders and Weller, 1950</a>). The virus definitive identification is then performed using Immunofluorescence (IF) staining. Nevertheless, virus isolation using cell culture is not ideal in case of viruses not amenable to growth in cell lines (norovirus, hepatitis virus) or producing CPE (<a href="#hzw010C58">Papafragkou <i>et al</i>., 2013</a>).</p> <p>In addition, small volumes of the sample may not allow inoculation of many cell types, and thus compromise the results. For example, the standard inoculation of a cell culture medium with cerebrospinal fluid (CSF) sample requires a minimum of 0.2 ml; however, much more is needed to inoculate a combination of different cell types, which could be invasive to the patient (<a href="#hzw010C49">McIntyre, 2007</a>, <a href="#hzw010C32">Hematiana <i>et al</i>., 2016</a>).</p> <p>The time required for isolating viruses by cell culture is very long (weeks), limiting the usefulness of this technique when rapid diagnosis is needed. Cell culture needs highly skilled and experienced personnel for accurate interpretation of the CPE and adequate facilities for handling mammalian cell lines or highly pathogenic viruses.</p> <p>In the <i>shell vial technique,</i> one of the <i>rapid culture methods</i>, inoculated cells are subjected to centrifugation, incubation and then IF staining with monoclonal antibodies, specific to the range of viruses suspected to cause the infection, such as respiratory viruses, Herpes simplex virus or Varicella zoster virus. The virus-induced antigens are detected 2â€"4 days later.</p> <p>Rapid culture has limited benefits as it does not target a wide range of viruses and has low sensitivity.</p> <p><i>Complement fixation test </i>(<i>CFT</i>) is one of the oldest methods in the history of clinical virology (<a href="#hzw010C11">Casals and Palacios, 1941</a>). The complement reacts only with antigenâ€"antibody complex in a non-specific manner. Thus, in the presence of the complex, the complement is not free to interact with sensitized sheep red blood cells (RBCs) used as an indicator, and which remain unlysed. The test is said to be â€˜positiveâ€™.</p> <p>CFT is supposedly easy to perform, convenient and requires inexpensive material. However, it is labour intensive and lacks sensitivity. In-house standardization through titration of the reactants and preparation of controls is crucial for obtaining effective testing.</p> <p><i>Haemagglutination inhibition</i> test is generally used for detecting arboviruses, influenza and parainfluenza virus subtypes and provides relative quantitation of the virus particles. The principle relies on the capacity of haemagglutinin (HA); a viral protein present in the envelope, to bind to erythrocytes (RBC) and to form a lattice pattern termed â€˜agglutinationâ€™. In the assay, serial dilutions of the sample serum are added to a fixed amount of viral HA and agglutinable RBCs. If Influenza antibodies are present in the serum, the agglutination process is prevented. The corresponding dilution rate at which complete haemagglutination is observed and considered.</p> <p>Variants of the agglutination assay are used for the diagnosis of wider range of viral diseases other than influenza (<a href="#hzw010C27">Grandien <i>et al</i>., 1987</a>; <a href="#hzw010C68">Sandeep <i>et al</i>., 2002</a>). However, the test is time consuming, and demanding, particularly in terms of QC.</p> <p>This list of conventional techniques is not exhaustive, and only the most commonly used were cited. The following sections will discuss more recent developments in the field, which replaced the conventional practices and allowed screening (qualitative test), surveillance (quantitative test) and confirmation of diagnosis.</p> </div> <div class="recentmethodsinthediagnosisofviralinfections" title="sec"> <div class="title" tagx="title" title="title">
Recent methods in the diagnosis of viral infections</div> <p>As stated before, this review will discuss only some of the main developments in diagnostic technologies that have ushered the new era of clinical virology, and lay out their key advantages and limitations that will be summarized in Table <a href="#hzw010TB1">1</a>. <div class="table-wrap_UNKNOWN" id="hzw010TB1" orientation="portrait" position="float"><span class="label" tagx="label" title="label">Table 1.</span> </div></p><p>Summary of the main viral diagnostic methods</p>  <table frame="hsides" rules="all"> <colgroup span="1"> <col align="left" span="1" /></colgroup> <colgroup><col align="left" span="1" /></colgroup> <colgroup><col align="left" span="1" /></colgroup> <colgroup><col align="left" span="1" /></colgroup> <colgroup><col align="left" span="1" /></colgroup> <colgroup><col align="left" span="1" /></colgroup>  <thead> <tr> <th align="left" rowspan="1" colspan="1">Diagnostic technique</th> <th align="left" rowspan="1" colspan="1">Principle</th> <th align="left" rowspan="1" colspan="1">Strengths</th> <th align="left" rowspan="1" colspan="1">Weaknesses</th> <th align="left" rowspan="1" colspan="1">Variants</th> <th align="left" rowspan="1" colspan="1">References</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">Immunoassay</td> <td align="left" rowspan="1" colspan="1">Formation of antigenâ€"antibody through recognition and binding</td> <td align="left" rowspan="1" colspan="1">High sensitivity<br /><br />High specificity<br /><br />High speed throughput<br /><br />Quick TAT (20 min or less)<br /><br />Different types of tags, labels<br /><br />Automated method</td> <td align="left" rowspan="1" colspan="1">Must rely on QC assurance<br /><br />High risk of interferences<br /><br />High cost</td> <td align="left" rowspan="1" colspan="1">RIA<br /><br />EIA (FPIA, MEIA, CLIA)</td> <td align="left" rowspan="1" colspan="1"><a href="#hzw010C28">Gupta <i>et al</i>. (2015)</a>, <a href="#hzw010C52">Mixson-Hayden <i>et al</i>. (2015)</a></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">NAAT</td> <td align="left" rowspan="1" colspan="1">Amplification and detection of sequences from the viral genome (DNA or RNA)</td> <td align="left" rowspan="1" colspan="1">High sensitivity<br /><br />High specificity<br /><br />Multiplexed platforms<br /><br />Genotyping<br /><br />Determination of the viral load<br /><br />New compact and portable formats</td> <td align="left" rowspan="1" colspan="1">Longer run time<br /><br />Requires specific<br /><br />primers for the targets</td> <td align="left" rowspan="1" colspan="1">RT-PCR<br /><br />qPCR<br /><br />NASBA<br /><br />TMA</td> <td align="left" rowspan="1" colspan="1"><a href="#hzw010C24">GarcÃ­a-Arroyo <i>et al</i>. (2016)</a>, <a href="#hzw010C64">Reijans <i>et al</i>. (2008)</a>, <a href="#hzw010C65">Renois <i>et al</i>. (2010)</a>, <a href="#hzw010C2">Afshar and Mollaie (2012)</a>, <a href="#hzw010C50">Mercier-Delarue <i>et al</i>. (2014)</a>, <a href="#hzw010C84">Wu <i>et al</i>. (2014)</a></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">NGS</td> <td align="left" rowspan="1" colspan="1">Polymerization of DNA template by incorporation of labelled dNTPs, and terminate the extension</td> <td align="left" rowspan="1" colspan="1">High sensitivity<br /><br />High specificity<br /><br />Identification of novel genomic sequences<br /><br />Genotyping<br /><br />Accurate detection of mutations and drug resistant mutations</td> <td align="left" rowspan="1" colspan="1">High cost<br /><br />Needs bioinformatics skills for data analysis<br /><br />Delay in use for routine clinical diagnostics</td> <td align="left" rowspan="1" colspan="1">Pyrosequencing<br /><br />Fluorescently labelled dNTP<br /><br />Detection of released hydrogen ion (H+)</td> <td align="left" rowspan="1" colspan="1"><a href="#hzw010C10">Capobianchi <i>et al</i>. (2013)</a>, <a href="#hzw010C5">Bartolini <i>et al</i>. (2015)</a>, <a href="#hzw010C45">Lowe <i>et al</i>. (2016)</a>, <a href="#hzw010C43">Liu <i>et al</i>. (2013)</a>, <a href="#hzw010C76">Van den Hoecke <i>et al</i>. (2015)</a>, <a href="#hzw010C67">Rothberg <i>et al</i>. (2011)</a></td> </tr> <tr> <td align="left" rowspan="1" colspan="1">MS</td> <td align="left" rowspan="1" colspan="1">Ionization of the sample, then separation and detection of the particles according to their mass-to-charge ratio (<i>m</i>/<i>z</i>)</td> <td align="left" rowspan="1" colspan="1">High sensitivity<br /><br />Versatility<br /><br />Cost-effectiveness<br /><br />High workload</td> <td align="left" rowspan="1" colspan="1">Expensive equipment<br /><br />Limited database library</td> <td align="left" rowspan="1" colspan="1">MALDI-TOF MS<br /><br />ESI MS<br /><br />Often combined with other methods: PCR-MS</td> <td align="left" rowspan="1" colspan="1"><a href="#hzw010C42">LÃ©vÃªque <i>et al</i>. (2014)</a>, <a href="#hzw010C89">Mengelle <i>et al</i>. (2013)</a>, <a href="#hzw010C63">Qian <i>et al</i>. (2014)</a></td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>CLIA, chemiluminescent immunoassay; dNTP, deoxyribonucleotide triphosphate; EIA, enzyme immunoassay; ESI, electrospray ionization; FPIA, fluorescence polarization immunoassay; MALDI-TOF, matrix-assisted laser desorption ionization time-of-flight; MEIA, micro-particle enzyme immunoassay; MS, mass spectrometry; NGS, next-generation sequencing; NAAT, nucleic acid amplification test; NASBA, nucleic acid sequence-based amplification; qPCR, quantitative polymerase chain reaction; RIA, radio-immunoassay; RT-PCR, real-time polymerase chain reaction; TMA, transcription-mediated amplification.</p> </div> </div>  <p /> </div> <div class="immunoassay-basedtests" title="sec"> <div class="title" tagx="title" title="title">
Immunoassay-based tests</div> <p>Antibodies produced immediately after invasion of a foreign substance can inform on primary infection, reinfection or a reactivation state. Therefore, measuring the level of immunoglobulins (Ig) is a widely considered approach for the diagnosis of viral infections.</p> <p>Automated immunoassay-based methods are among the most frequently used for testing, and are effective because of the high specificity and binding affinity between antigen and antibody. Therefore, the principle of the test relies in the formation of an immuno-complex between antibody present in the patient sample and synthetic antigen present in the reagent or vice versa, to generate a measurable signal.</p> <p>Immunoassays use labels conjugated to synthetic antibodies or antigens which are linked to a solid phase, and used to capture corresponding antigens or antibodies present in sera samples. These labels could be radioactive isotopes, enzymes that cause change in colour or light-generating substances. Consequently, this principle has generated several methodologies for the testing.</p> <p><i>Radio-immunoassay </i>(<i>RIA</i>) is probably the initiating method (1960s); it uses radioisotopes (such as Iodine 125) to label antigen or antibody. The amount of substance to analyse is determined by the amount of the generated radioactivity. RIA is a highly sensitive method but the main drawback is the handling and disposal of hazardous radioactive substances.</p> <p>The enzymatic labelling alternative using alkaline phosphatase or horseradish peroxidase as markers is, however, the most widely used and was long considered a reference method (<a href="#hzw010C20">Engvall and Perlmann, 1972</a>; <a href="#hzw010C80">Voller, Bidwell and Bartlett, 1976</a>). These enzymes induce emission of signals or change in colour respectively, and allow the amount of analyte of interest to be measured. This <i>enzyme-linked immunoassay</i> (<i>EIA</i>) has numerous variants, including ELISA, and they differ in the enzyme used and the signal detection principle.</p> <p>The main variants of EIA are as follows: </p><ul> <li> <p><i>Fluorescence polarization immunoassay</i> (<i>FPIA</i>): uses fluorescent label and polarized light.</p> </li> <li> <p><i>Micro-particle enzyme immunoassay</i> (<i>MEIA</i>) (<a href="#hzw010C75">Tassopoulos <i>et al</i>., 1997</a>): widely used and relies on alkaline phosphatase enzyme and a corresponding fluorescent substrate.</p> </li> <li> <p><i>Chemiluminescent immunoassay</i> (<i>CLIA</i>), which uses chemiluminescent or light-emitting labels. Companies like ROCHE or Abbott are exploiting this method, and high-volume laboratories are gradually replacing MEIA technology with CLIA for its high-speed throughput and ease of measurement.</p> </li> </ul> <p /> <p>In clinical practice, serological studies of Hepatitis B rely on immunoassay as a key tool for detection of Hepatitis B virus (HBV) markers. The up-to-date versions of immunoassay methods are able to detect the lowest levels of the main marker HBsAg, undetectable by conventional methods, particularly in sera samples of asymptomatic patients. The latest screening standard detects as little as 0.05 IU/ml, the equivalent of 0.2 ng/ml viral antigen (<a href="#hzw010C16">Deguchi <i>et al</i>., 2004</a>) and high sensitivity (&amp;gt;99%) was observed at different stages of the disease or even in patients showing seroclearance.</p> <p>A newly developed CLIA allowed denaturation of the HBV particle, and used monoclonal antibodies against inner and outer structural epitopes in order to obtain higher assay sensitivity (<a href="#hzw010C72">Shinkai <i>et al</i>., 2013</a>) and sufficient information on the HBV genotypes.</p> <p>Automated immunoassay techniques for virus detection overcome some of the limitations encountered with the conventional tests, particularly the delay to response.</p> <div class="limitations" title="sec"> <div class="title" tagx="title" title="title">
Limitations</div> <p>Despite the popularity of immunoassay in clinical testing, erroneous results may occur for many reasons, which confer inconsistencies to the testing. </p><ul> <li> <p>Immunoassays are more prone to interferences than any other assay, which leads to false-positive or false-negative results. In most cases, interferences are due to the presence of agents with structural similarities to the reagents (<a href="#hzw010C51">Miller, 2004</a>), and fluctuate according to the concentration of the interfering substance and the analyte. Endogenous antibodies (autoantibodies, hetero-antibodies or human anti-animal antibodies) are commonly the main culprit in these interferences (<a href="#hzw010C18">Emerson and Lai, 2013</a>). Their binding to capture antibodies and detection antibodies in the absence of antigen (analyte) mainly leads to false-positive result (<a href="#hzw010C7">Berth and Willaert, 2016</a>).</p> </li> <li> <p>Accurate QC guarantees reliable results and assumes that the assay is performed well. Therefore, poor QC measures lead to low accuracy and precision. In immunoassays, QC is assured by regular calibration and control of the reagents, using standardized solutions delivered by the manufacturers.</p> </li> <li> <p>The high cost of reagents and equipment is another drawback in immunoassays, particularly for resource-limited settings, considering that the most accurate and sensitive methods are automated.</p> </li> </ul> <p /> </div> </div> <div class="amplification-basedassays" title="sec"> <div class="title" tagx="title" title="title">
Amplification-based assays</div> <p>Developed by <a href="#hzw010C53">Mullis and Faloona (1987)</a>, nucleic acid amplification by polymerase chain reaction (PCR) has revolutionized the field of molecular diagnosis. The basic PCR assay relies on extraction and purification of the nucleic acid, then exponential amplification of the target sequence, using a thermostable polymerase enzyme and specific primers. The resulting amplicons are then identified using a fluorescence-based detection system, and the result is reported in international units IU/ml.</p> <p>Soon after its invention, modifications in PCR were tested and patented, with the aim of improving the assay capabilities. The term nucleic acid amplification tests (NAAT) was applied to this range of new variants.</p> <p>NAAT are very popular in the diagnosis and management of viral infections (HBV, HCV, HIV, Influenza viruses, â€¦) because they allow determination of the viral load. In other terms, quantitation of the viral nucleic acid by amplifying the target sequence thousands-fold. In most cases, they are now considered a reference, or â€˜gold standardâ€™ method for diagnostic practices such as screening donated blood for transfusion-transmitted viruses (CMV, HIV, HCV, â€¦) (<a href="#hzw010C35">Jackson, 1990</a>).</p> <p>The most widely used variants of conventional amplification are real-time PCR (quantitative PCR) and reverse transcription-PCR (RT-PCR). Both are nowadays becoming benchmarks in assessing the viral load, and while the first method quantifies DNA throughout the reactions in real time (<a href="#hzw010C55">Ntziora <i>et al</i>., 2013</a>); the second performs RT of the mRNA (RNA messenger) and amplifies the resulting cDNA (complementary DNA). It also quantifies RNA. The combination of both techniques increases sensitivity in detecting viruses, particularly influenza viruses. The WHO recently approved a newly developed reverse transcriptase-PCR assay after the first death from MERS-CoV (Middle East Respiratory Syndrome-Coronavirus) infection reported in 2012 (<a href="#hzw010C1">Abd El Wahed <i>et al</i>., 2013</a>).</p> <p>Other amplification-based tests such as nucleic acid sequence-based amplification (NASBA) and transcription-mediated amplification (TMA) are suited for detection of RNA viruses by amplification of the mRNA instead of conversion to cDNA (<a href="#hzw010C50">Mercier-Delarue <i>et al</i>., 2014</a>, <a href="#hzw010C84">Wu <i>et al</i>., 2014</a>).</p> <p>Among the latest enhancements in PCR systems, extraction, amplification and detection steps have been combined within one unit. As an example, BioFire Film Array technology applies two sequential amplifications and can detect a panel of respiratory viruses with high sensitivity in 1 h and within a single instrument (<a href="#hzw010C61">Pierce <i>et al</i>., 2012</a>).</p> <p>On the contrary, microfluidic technology has also benefited PCR-based systems; and allowed further decrease in the detection time (15 min), with high sensitivity and cost-effectiveness. In the 2009 epidemic, detection of Influenza A virus H1N1 greatly relied on a chip-type real-time PCR instead of the tube-based principle. The portable format of such systems is very convenient during epidemics and outbreaks (<a href="#hzw010C73">Song <i>et al</i>., 2012</a>).</p> <p>Multiplexed NAAT were designed to detect multiple viruses or subtypes in a single run. Their detection platforms can comprise up to 20 viruses using diversified panels (<a href="#hzw010C47">Mahony <i>et al</i>., 2007</a>), for example, concurrent detection of HAV, HBV and HCV infections, as well as co-infections (<a href="#hzw010C59">Park <i>et al</i>., 2012</a>).</p> <p>In Hepatitis B, PCR is considered the gold standard method for assessing the HBV DNA level with high accuracy, and the assays are standardized according to the WHO International Standard for Hepatitis B virus (NIBSC), such as the reference Roche Cobas TaqMan HBV test. The test was the first FDA approved assay, with a detection limit of 10.2 IU/ml. The combined system with Cobas AmpliPrep has an automated sample extraction process and highly sensitive detection for all HBV genotypes (<a href="#hzw010C62">Pyne <i>et al</i>., 2012</a>).</p> <p>In HIV patients taking antiretroviral treatment (ART), ultrasensitive assays permit quantitation of low copy numbers of HIV subtypes (&amp;lt;50 copies/ml) and long terminal repeats in latent reservoirs and from cell lysates omitting the nucleic acid extraction step (<a href="#hzw010C77">Vandergeeten <i>et al</i>., 2014</a>). Such assays could be valuable tools in large cohort studies or pandemic situations.</p> <div class="limitations" title="sec"> <div class="title" tagx="title" title="title">
Limitations</div> <p>The limitations of PCR are an important parameter to consider, despite the cost-effectiveness and reliability in the diagnosis of viral infections. The risk of contamination is very high while handling, especially during the sample preparation step, in addition, real-time PCR has a longer run-time (2â€"5 h) by comparison to other techniques.</p> <p>In the case of influenza, many PCR-based assays were designed to detect only a particular subtype of the virus responsible for important pandemics (<a href="#hzw010C30">Hall <i>et al</i>., 2009</a>), and the need for designing specific primers for the target requires handling by experienced operators, capable of detecting errors, particularly that PCR is prone to false-positive results.</p> <p>The high mutation rate of some viruses could trigger mutation within PCR primer regions of the viral genome, which would lead the virus to escape the detection by this assay.</p> </div> </div> <div class="next-generationsequencing" title="sec"> <div class="title" tagx="title" title="title">
Next-generation sequencing</div> <p>Next-generation sequencing (NGS) is one of the greatest achievements of the modern era. Beyond genome sequencing from known organisms, it allowed discovery of novel viruses responsible for unknown human diseases (<a href="#hzw010C56">Palacios <i>et al</i>., 2008</a>), and tracking of outbreaks and pandemics such as influenza (<a href="#hzw010C4">Baillie <i>et al</i>., 2012</a>) to understand their emergence and transmission profiles (<a href="#hzw010C41">Leung <i>et al</i>., 2014</a>; <a href="#hzw010C34">Isakov <i>et al</i>., 2015</a>).</p> <p>The journey began in the 1970s with the works of Sanger and Barrel, followed by Maxam and Gilbert, who first initiated the principle of sequencing oligonucleotides via enzymatic polymerization, using radiolabelled primers. In their experimental protocols, they relied on using dideoxy-nucleotides to terminate the extension of the polymer, hence the name: chain termination or dideoxynucleotide method (<a href="#hzw010C69">Sanger, Nicklen and Coulson, 1977</a>).</p> <p>Since then, the principle has remained virtually the same, but improvements and automation have dramatically increased the speed and accuracy in delivering maximum volume of data comparing to dideoxynucleotide sequencing (<a href="#hzw010C71">Shedure and Ji, 2008</a>). Technically, NGS is inclusive of three main steps: sample preparation, sequencing and data analysis. Systems available in the market differ mostly in their sequencing or reading techniques. Efficient and accurate clinical diagnosis of viral infections using NGS is increasingly aiming to provide accurate longer read-length in a shortest time and at a lower cost. Bioinformatic platforms are key components of the sequencing process. They allow interpretation of the sequencing output through computational analysis (<a href="#hzw010C54">Naccache <i>et al</i>., 2014</a>), and then convert it into useful information on species, genotypes and the occurrence of mutations conferring virulence or resistance to antivirals.</p> <p>Pyrosequencing is currently the variant of choice within NGS systems. Based on the detection of pyrophosphate (PPi) after incorporation of a nucleotide in a DNA polymerization process, it uses luciferase to catalyse light-generating processes, and the collected light is then recorded. Although Illumina is considered the most frequently used pyrosequencing platform, 454 FLX; a subsidiary company of Roche, was the first high-throughput analyser in the market and was used to determine human papillomavirus (HPV) types (<a href="#hzw010C6">Barzon <i>et al</i>., 2011</a>), subtypes and variants, present in cervical specimens. Other variants detect hydrogen ions that are released throughout nucleotide incorporation reaction (<a href="#hzw010C67">Rothberg <i>et al</i>. 2011</a>).</p> <p>Generating high volumes of sequence data has allowed the compilation of viral nucleotide databases and acquisition of de novo sequences to understand the genetic variability of viruses (<a href="#hzw010C74">Szpara, Parsons and Enquist, 2010</a>). HIV is by far the most sequenced because of the global priority of AIDS as a serious endemic, and because of the high mutation rate of the virus.</p> <p>So far, only gene sequencing methods have been successful in genotyping tricky HBV, unlike conventional PCR or serological assays. Therefore, sequencing has allowed better clinical management of HBV infection and related complications (<a href="#hzw010C48">Margeridon-Thermet <i>et al</i>., 2009</a>).</p> <p>In data analysis, recent technical approaches have included adjustment of the software reading platforms for simultaneous detection of genotypes and mutants of clinical importance (<a href="#hzw010C25">Germer <i>et al</i>., 2013</a>), and partial sequencing of the HBV S gene with high sensitivity (98.64%), considering that portion as the site of most drug-resistance mutations (<a href="#hzw010C81">Wang <i>et al</i>., 2013</a>). Nevertheless, implementation of NGS in clinical settings is increasing, particularly for detecting low-abundance drug-resistance patterns such as in HIV and HCV (<a href="#hzw010C57">Palmer <i>et al</i>., 2005</a>; <a href="#hzw010C79">Verbinnen <i>et al</i>., 2010</a>). Nucleotide sequencing of HCV sub-genomic regions is now the method of choice for genotyping.</p> <div class="limitations" title="sec"> <div class="title" tagx="title" title="title">
Limitations</div> <p>The principal requirements for NGS are initially access to a sequencer, and considerable skills in bioinformatics and expertise in data analysis, plus adequate handling systems for storage of generated data. Adding to that, despite the outstanding results delivered by prototypes in trials, many are still at the research level, and not yet approved for use in routine clinical practice (<a href="#hzw010C36">Jiangqin <i>et al</i>., 2015</a>).</p> <p>NGS is undeniably a key technology in specialized clinical laboratories, but its implementation is still a challenge in many countries, where not only their resource-limited settings cannot afford a sequence analyser, sample and library preparation, but the vast majority of the population cannot afford the cost of the test.</p> </div> </div> <div class="massspectrometry" title="sec"> <div class="title" tagx="title" title="title">
Mass spectrometry</div> <p>Mass spectrometry (MS) is nowadays a benchmark of laboratory qualitative and quantitative investigation, particularly in bacteriology (<a href="#hzw010C70">Sauer and Kliem, 2010</a>).</p> <p>The principle of MS relies on converting the sample into charged particles (ions) by ionization process. These ions are separated according to their mass-to-charge ratio (<i>m</i>/<i>z</i>) and analysed by a detector. The result obtained is compared to a reference database (library), existing within the system and delivered as an interpretive spectrum.</p> <p>In clinical laboratories, matrix-assisted laser desorption ionization (MALDI) and electrospray (ES) are the most used ionization methods because they allow processing of considerable amounts of analyte (<a href="#hzw010C19">Emonet <i>et al</i>., 2010</a>). These approaches have extensively been evaluated experimentally and provided excellent results, either used alone or combined with other molecular methods, such as PCR, in order to enhance sensitivity. The combination (RT-PCR/ESI-MS) was able to detect viral pathogens usually undetected by regular testing methods, and provided rapid and detailed data (types and subtypes) within a short time (<a href="#hzw010C42">LÃ©vÃªque <i>et al</i>., 2014</a>).</p> <p>Detection of genomic variations or mutations in Influenza A using MALDI-TOF has been a key tool in the management of outbreaks (<a href="#hzw010C13">Chen <i>et al</i>., 2011</a>). MS application to the structural investigation of biomolecules showed the efficiency of MALDI-TOF-MS coupled to antibody magnetic nanoparticles in detecting influenza viruses (<a href="#hzw010C15">Chou <i>et al</i>., 2011</a>; <a href="#hzw010C87">Yea <i>et al</i>., 2011</a>), through concordant results with the gold standard PCR-based method. This blend of two powerful machineries (PCR-MS) can detect drug resistance to antiviral therapy as well as the presence of multiple viruses within the same sample and diagnose for co-infections, when assays are multiplexed.</p> <p>The increasing interest in MALDI-TOF MS in clinical virology has led to research advances in the diagnosis of Hepatitis B. The method was in fact not only able to detect HBV at relatively low viral load levels (100 HBV DNA copies/ml) (<a href="#hzw010C33">Hong <i>et al</i>., 2004</a>) but it also allowed detection of up to 60 HBV variants and genotypes at a cost lower than $10/sample for all the variants with high throughput (<a href="#hzw010C46">Luan <i>et al</i>., 2009</a>). The same method was applied for genotyping and successfully detected the eight HBV genotypes accurately (<a href="#hzw010C23">Ganova-Raeva <i>et al</i>., 2010</a>), and also minor HCV genotypes occurring at very low level (<a href="#hzw010C37">Kim <i>et al</i>., 2005</a>).</p> <p>Mass spectrometric-based methods are versatile, sensitive, rapid and cost-effective, and do not require interpretation software for data analysis. The automated machinery necessitates easy sample preparation and fewer operators. The analysis capacity can reach up to 960 specimens/day, which makes it suitable for routine diagnosis in high-volume laboratories and large-scale studies. Tests can also be performed efficiently on archived specimen.</p> <div class="limitations" title="sec"> <div class="title" tagx="title" title="title">
Limitations</div> <p>The main limitation of MS is the high cost, particularly in high pandemic areas, which are usually the poorest; not all laboratories can afford a mass analyser for their activities. The second major drawback is within the reference library. The identification is limited by known data from well-identified organisms only; therefore, rare mutations cannot be detected if they do not exist within the reading platform, but there is hope that MS database libraries will rapidly expand.</p> </div> </div> <div class="advantagesoftherecentmethods" title="sec"> <div class="title" tagx="title" title="title">
Advantages of the recent methods</div> <p> </p><ul> <li> <p>High sensitivity and specificity assays approaching 100%.</p> </li> <li> <p>Possibility to combine different methods in one assay enhances detection capability and accuracy (PCR-MS, microfluidic chip technology-based PCR, â€¦).</p> </li> <li> <p>Automation of the assays reduces the number of operators and manual workload.</p> </li> <li> <p>Small sample volumes are needed, so the assays can still be performed in particular cases (CSF, new-borns, etc.)</p> </li> <li> <p>Some assays such as MS remain efficient after several freezingâ€"thawing of the samples (archived samples).</p> </li> <li> <p>Rapid TAT: minimum time required from sample collection to results reporting is 30 min, and microarray tools can deliver results within seconds.</p> </li> <li> <p>Low detection limit: 10â€"100 copies/ml by PCR.</p> </li> <li> <p>Multiplex reactions: detection of a wide range of pathogens in a single run is time saving.</p> </li> <li> <p>Detection of rare drug-resistance patterns.</p> </li> <li> <p>NGS generates many sequencing data per run and sequences long-reads.</p> </li> <li> <p>Low reaction cost when using microfluidic chips, and use in POC settings.</p> </li> <li> <p>Lower risk of contamination by processing single tube within one unit.</p> </li> </ul> <p /> </div> <div class="thelatesttechnologiesandresource-limitedsettings" title="sec"> <div class="title" tagx="title" title="title">
The latest technologies and resource-limited settings</div> <p>The methods described above have shown outstanding performance in saving thousands of lives in developed countries. However, accurate diagnosis is still a challenge in resource-limited settings because of the difficulty in acquiring these equipment and technical expertise. While expensive tools such as PCR-based systems or mass analysers are only available in reference laboratories or in military facilities, point-of-care or near-patients clinics that serve more than 80% of the poor population struggle with the testing devices available. As an example, the WHO has established guidelines for monitoring ART efficacy in HIV-1 infected patients in developing countries, by suggesting only CD4+ cell count and ELISA-based assays as alternatives to the expensive tests (<a href="#hzw010C83">WHO, 2006a</a>). Ultrasensitive p24 (<a href="#hzw010C60">Patton <i>et al</i>., 2008</a>) and the ExaVir assay (<a href="#hzw010C38">Kokkayil <i>et al</i>., 2014</a>), measure the level of p24 antigen and the activity of reverse transcriptase, respectively, to replace viral load monitoring. Unfortunately, their lack of sensitivity has induced failure of treatment and emergence of resistant strains (<a href="#hzw010C78">Vekemans, John and Colebunders, 2007</a>).</p> <p>Hepatitis is another expensive burden to manage in poor countries, where accurate assessment of the epidemiological profile is hard to establish. In the case of HBV, most POC facilities rely on screening for HBsAg solely using rapid diagnosis tests. These low-cost devices do not allow determination of the course of the disease to inform whether to initiate an antiviral treatment or not.</p> </div> <div class="automation,microfluidicsandfutureprospects" title="sec"> <div class="title" tagx="title" title="title">
Automation, microfluidics and future prospects</div> <p>Providing high-level epidemiological monitoring of viral diseases is undeniably a global public health ambition, and despite rapid progress in the development of diagnostic methods in recent years, improvements are needed for better cost, size (<a href="#hzw010C44">Loman <i>et al</i>., 2012</a>; <a href="#hzw010C22">Frey <i>et al</i>., 2014</a>) and TAT (<a href="#hzw010C86">Xu <i>et al</i>., 2013</a>).</p> <p>After the Ebola Haemorrhagic fever in 2014, the US governmental Department of Energy has developed an innovative rapid and portable test to detect specifically Ebola virus within seconds. The test aims to target other RNA and exotic viruses such as Dengue and West Nile for effective management of viral outbreaks. Other examples include fully NAAT systems and small devices to sequence single molecule DNA using nanopore technology (<a href="#hzw010C17">Eisenstein, 2012</a>).</p> <p>Microfluidic technology is regarded with optimism for the management of infectious diseases by allowing timely therapy. It provides a key potential solution for remote areas and near-patients facilities by avoiding turnaround trips of the patients between the clinic and the laboratory. Their use is also beneficial where time is crucial, or when physical spaces do not allow setting up of conventional methods.</p> <p>Lab-on-a-chip (LOC) is a very small device that integrates laboratory processes within a few square centimetres. It uses very small volume of samples to perform immediate reactions within the chip or in a portable device. The reactions vary from nucleic acid amplification and detection, to cell count and immunoassays; therefore, microfluidic diagnostics compete with large instruments in performing laboratory tests at a lower cost, to benefit low-income settings and remote areas.</p> <p>A wide range of LOC devices were approved by the FDA in the diagnosis of viral infections such as Influenza (<a href="#hzw010C9">Cao <i>et al</i>., 2012</a>), HIV (<a href="#hzw010C3">Alyassin <i>et al</i>., 2009</a>) and HBV (<a href="#hzw010C88">Zhi <i>et al</i>., 2014</a>) and further development is in progress. As examples, Daktari Diagnostics use affinity chromatography for CD4+T-cell count as an alternative to flow cytometry to monitor HIV in developing countries (<a href="#hzw010C14">Cheng <i>et al</i>., 2007</a>). The bench top analyser GeneXpert made by Cepheid has an integrated sample preparation and PCR system for molecular diagnosis of influenza and other bacterial infections in a light portable format.</p> <p>Improvement of QC programs, QA and standardization of assays, kits and reagents are important to fulfil requirements for accuracy. The Centre for Disease Control and Prevention (CDC) is continuously implementing laboratory-testing guidelines particularly for HIV, before further tests will be approved (<a href="#hzw010C12">CDC, 2014</a>).</p> <p>The WHO and non-profit organizations such as the Foundation for Innovative New Diagnostics (FIND) aim to maximize efforts to implement surveillance programs and control of communicable diseases (<a href="#hzw010C44">Loman <i>et al</i>., 2012</a>), set up new policies and improve diagnostic services in low resource settings.</p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>The recently developed viral diagnostic methods are reshaping the field of clinical microbiology, and could contribute to reducing the prevalence of serious infectious diseases. However, the technical capabilities alone are insufficient if not supported by health promotion strategies to increase awareness about the importance of early detection and regular screening of persons at high risk.</p> <p>Finally, good quality diagnosis has a cost that only developed countries can afford in routine practice so far, and this is delaying the implementation of new methods in the developing world and the endemic areas. However, there is hope that efforts will continue towards developing new good quality tests affordable in low-income countries, which would substantially strengthen disease control strategies for their populations.</p> </div> <div class="authorbiography" title="sec"> <div class="title" tagx="title" title="title">
Author biography</div> <p>Selma obtained an MSc with Honours in Biomedical sciences from Nottingham Trent University. She works as a biomedical scientist at Institut Pasteur, within the Immunoassay division, and has particular interest in laboratory diagnosis issues in low-income settings and how they can be solved at lower cost. She also has interest in structural biology and microbiology.</p> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="hzw010C1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abd El Wahed</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patel</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heidenreich</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Reverse transcription recombinase polymerase amplification assay for the detection of Middle East respiratory syndrome coronavirus</span>, <span class="source" tagx="source" title="source">PLoS Currents Outbreaks</span>, <span class="volume" tagx="volume" title="volume">5</span>, <span class="fpage" tagx="fpage" title="fpage">e2e8364</span>.</span></li> <li tag="ref"><a name="hzw010C2" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Afshar</span><span class="given-names" tagx="given-names" title="given-names">R. M.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mollaie</span><span class="given-names" tagx="given-names" title="given-names">H. R.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Detection of HBV resistance to lamivudine in patients with chronic Hepatitis B using zip nucleic acid probes in Kerman, Southeast of Iran</span>, <span class="source" tagx="source" title="source">Asian Pacific Journal of Cancer Prevention</span>, <span class="volume" tagx="volume" title="volume">13</span> (<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">3657</span>â€"<span class="lpage" tagx="lpage" title="lpage">3661</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23098450">23098450</a></span></span></li> <li tag="ref"><a name="hzw010C3" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alyassin</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moon</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keles</span><span class="given-names" tagx="given-names" title="given-names">H. O.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Rapid automated cell quantification on HIV microfluidic devices</span>, <span class="source" tagx="source" title="source">Lab on a Chip</span>, <span class="volume" tagx="volume" title="volume">9</span> (<span class="issue" tagx="issue" title="issue">23</span>). <span class="pub-id"><a href="https://dx.doi.org/10.1039/b911882a">10.1039/b911882a</a></span>.</span></li> <li tag="ref"><a name="hzw010C4" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baillie</span><span class="given-names" tagx="given-names" title="given-names">G. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galiano</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agapow</span><span class="given-names" tagx="given-names" title="given-names">P. M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Evolutionary dynamics of local pandemic H1N1/2009 influenza virus lineages revealed by whole genome analysis</span>, <span class="source" tagx="source" title="source">Journal of Virology</span>, <span class="volume" tagx="volume" title="volume">86</span>, <span class="fpage" tagx="fpage" title="fpage">11</span>â€"<span class="lpage" tagx="lpage" title="lpage">18</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22013031">22013031</a></span></span></li> <li tag="ref"><a name="hzw010C5" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartolini</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giombini</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abbate</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Near full length Hepatitis C virus genome reconstruction by next generation sequencing based on genotype-independent amplification</span>, <span class="source" tagx="source" title="source">Digestive and Liver Disease</span>, <span class="volume" tagx="volume" title="volume">47</span> (<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">608</span>â€"<span class="lpage" tagx="lpage" title="lpage">12</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25888234">25888234</a></span></span></li> <li tag="ref"><a name="hzw010C6" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barzon</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Militello</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lavezzo</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Human papillomavirus genotyping by 454 next generation sequencing technology</span>, <span class="source" tagx="source" title="source">Journal of Clinical Virology</span>, <span class="volume" tagx="volume" title="volume">52</span>, <span class="fpage" tagx="fpage" title="fpage">93</span>â€"<span class="lpage" tagx="lpage" title="lpage">97</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21802982">21802982</a></span></span></li> <li tag="ref"><a name="hzw010C7" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berth</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Willaert</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span> (<span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Elimination of complement interference can improve the diagnostic performance of the VIDAS CMV IgG assay in acute cytomegalovirus infections</span>, <span class="source" tagx="source" title="source">Diagnostic Microbiology and Infectious Disease</span>, <span class="volume" tagx="volume" title="volume">85</span> (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">30</span>â€"<span class="lpage" tagx="lpage" title="lpage">35</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26971633">26971633</a></span></span></li> <li tag="ref"><a name="hzw010C8" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blacksell</span><span class="given-names" tagx="given-names" title="given-names">S. D.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Commercial dengue rapid diagnostic tests for point-of-care application: recent evaluations and future needs</span>. <span class="source" tagx="source" title="source">Journal of Biomedicine and Biotechnology</span>, 2012, <span class="fpage" tagx="fpage" title="fpage">12</span>Article ID 151967.</span></li> <li tag="ref"><a name="hzw010C9" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahalanabis</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Microfluidic chip for molecular amplification of Influenza A RNA in human respiratory specimens</span>, <span class="source" tagx="source" title="source">PLoS One</span>, <span class="volume" tagx="volume" title="volume">7</span> (<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">e33176</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22457740">22457740</a></span></span></li> <li tag="ref"><a name="hzw010C10" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Capobianchi</span><span class="given-names" tagx="given-names" title="given-names">M. R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giombini</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rozera</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Next-generation sequencing technology in clinical virology</span>, <span class="source" tagx="source" title="source">Clinical Microbiology and Infection</span>, <span class="volume" tagx="volume" title="volume">19</span> (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">15</span>â€"<span class="lpage" tagx="lpage" title="lpage">22</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23279287">23279287</a></span></span></li> <li tag="ref"><a name="hzw010C11" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Casals</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palacios</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span> (<span class="year" tagx="year" title="year">1941</span>) <span class="mixed-article-title" title="mixed-article-title">The complement fixation test in the diagnosis of virus infections of the central nervous system</span>, <span class="source" tagx="source" title="source">The Journal of Experimental Medicine</span>, <span class="volume" tagx="volume" title="volume">74</span> (<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">409</span>â€"<span class="lpage" tagx="lpage" title="lpage">426</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19871144">19871144</a></span></span></li> <li tag="ref"><a name="hzw010C12" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: CDC</span></span>, Laboratory testing for the diagnosis of HIV infection: updated recommendations. (<span class="year" tagx="year" title="year">2014</span> June). Accessed at <span class="uri" title="uri"><a href="http://www.cdc.gov/hiv/pdf/HIVtestingAlgorithmRecommendation-Final.pdf">http://www.cdc.gov/hiv/pdf/HIVtestingAlgorithmRecommendation-Final.pdf</a></span></span></li> <li tag="ref"><a name="hzw010C13" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">K. F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rothman</span><span class="given-names" tagx="given-names" title="given-names">R. E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramachandran</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Rapid identification viruses from nasal pharyngeal aspirates in acute viral respiratory infections by RT-PCR and electrospray ionization mass spectrometry</span>, <span class="source" tagx="source" title="source">Journal of Virological Methods</span>, <span class="volume" tagx="volume" title="volume">173</span> (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">60</span>â€"<span class="lpage" tagx="lpage" title="lpage">66</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21256867">21256867</a></span></span></li> <li tag="ref"><a name="hzw010C14" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Irimia</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dixon</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">A microfluidic device for practical label-free CD4 (+) T cell counting of HIV-infected subjects</span>, <span class="source" tagx="source" title="source">Lab on a Chip</span>, <span class="volume" tagx="volume" title="volume">7</span> (<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">170</span>â€"<span class="lpage" tagx="lpage" title="lpage">178</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17268618">17268618</a></span></span></li> <li tag="ref"><a name="hzw010C15" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chou</span><span class="given-names" tagx="given-names" title="given-names">T. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hsu</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C. H.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Rapid and specific influenza virus detection by functionalized magnetic nanoparticles and mass spectrometry</span>, <span class="source" tagx="source" title="source">Journal of Nanotechnology</span>, <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">52</span>.</span></li> <li tag="ref"><a name="hzw010C16" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deguchi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamashita</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kagita</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">Quantitation of Hepatitis B surface antigen by an automated chemiluminescent microparticle immunoassay</span>, <span class="source" tagx="source" title="source">Journal of Virological Methods</span>, <span class="volume" tagx="volume" title="volume">115</span>, <span class="fpage" tagx="fpage" title="fpage">217</span>â€"<span class="lpage" tagx="lpage" title="lpage">222</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14667538">14667538</a></span></span></li> <li tag="ref"><a name="hzw010C17" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eisenstein</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Oxford nanopore announcement sets sequencing sector abuzz</span>, <span class="source" tagx="source" title="source">Nature Biotechnology</span>, <span class="volume" tagx="volume" title="volume">30</span>, <span class="fpage" tagx="fpage" title="fpage">295</span>â€"<span class="lpage" tagx="lpage" title="lpage">296</span>.</span></li> <li tag="ref"><a name="hzw010C18" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emerson</span><span class="given-names" tagx="given-names" title="given-names">J. F.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">K. K. Y.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Endogenous antibody interferences in immunoassays</span>, <span class="source" tagx="source" title="source">Laboratory medicine</span>, <span class="volume" tagx="volume" title="volume">44</span>, <span class="fpage" tagx="fpage" title="fpage">69</span>â€"<span class="lpage" tagx="lpage" title="lpage">73</span>.</span></li> <li tag="ref"><a name="hzw010C19" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emonet</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shah</span><span class="given-names" tagx="given-names" title="given-names">H. N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cherkaoui</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Application and use of various mass spectrometry methods in clinical microbiology</span>, <span class="source" tagx="source" title="source">Clinical Microbiology and Infections</span>, <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">1604</span>â€"<span class="lpage" tagx="lpage" title="lpage">13</span>.</span></li> <li tag="ref"><a name="hzw010C20" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Engvall</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perlmann</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span> (<span class="year" tagx="year" title="year">1972</span>) <span class="mixed-article-title" title="mixed-article-title">Enzyme-linked immunosorbent assay, ELISA. 3. Quantitation of specific antibodies by enzyme-labeled anti-immunoglobulin in antigen-coated tubes</span>, <span class="source" tagx="source" title="source">Journal of Immunology</span>, <span class="volume" tagx="volume" title="volume">109</span> (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">129</span>â€"<span class="lpage" tagx="lpage" title="lpage">35</span>.</span></li> <li tag="ref"><a name="hzw010C21" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fauquet</span><span class="given-names" tagx="given-names" title="given-names">C. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mayo</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maniloff</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2005</span>) <span class="source" tagx="source" title="source">Virus Taxonomy: Classification and Nomenclature of Viruses. Eighth Report of the International Committee on the Taxonomy of Viruses</span>, <span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier Academic Press</span>, <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span>, p. <span class="fpage" tagx="fpage" title="fpage">1259</span>.</span></li> <li tag="ref"><a name="hzw010C22" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frey</span><span class="given-names" tagx="given-names" title="given-names">K. G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herrera-Galeano</span><span class="given-names" tagx="given-names" title="given-names">J. E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Redden</span><span class="given-names" tagx="given-names" title="given-names">C. L.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Comparison of three next-generation sequencing platforms for metagenomic sequencing and identification of pathogens in blood</span>, <span class="source" tagx="source" title="source">BMC Genomics</span>, <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">96</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24495417">24495417</a></span></span></li> <li tag="ref"><a name="hzw010C23" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ganova-Raeva</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramachandran</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Honisch</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Robust Hepatitis B virus genotyping by mass spectrometry</span>, <span class="source" tagx="source" title="source">Journal of Clinical Microbiology</span>, <span class="volume" tagx="volume" title="volume">48</span>, <span class="fpage" tagx="fpage" title="fpage">4161</span>â€"<span class="lpage" tagx="lpage" title="lpage">8</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20810764">20810764</a></span></span></li> <li tag="ref"><a name="hzw010C24" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GarcÃ­a-Arroyo</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prim</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">MartÃ­</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Benefits and drawbacks of molecular techniques for diagnosis of viral respiratory infections. Experience with two multiplex PCR assays</span>, <span class="source" tagx="source" title="source">Journal of Medical Virology</span>, <span class="volume" tagx="volume" title="volume">88</span> (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">45</span>â€"<span class="lpage" tagx="lpage" title="lpage">50</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26100363">26100363</a></span></span></li> <li tag="ref"><a name="hzw010C25" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Germer</span><span class="given-names" tagx="given-names" title="given-names">J. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abraham</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mandrekar</span><span class="given-names" tagx="given-names" title="given-names">J. N.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Evaluation of the Abbott HBV RUO sequencing assay combined with laboratory-modified interpretive software</span>, <span class="source" tagx="source" title="source">Journal of Clinical Microbiology</span>, <span class="volume" tagx="volume" title="volume">51</span> (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">95</span>â€"<span class="lpage" tagx="lpage" title="lpage">100</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23100352">23100352</a></span></span></li> <li tag="ref"><a name="hzw010C26" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldsmith</span><span class="given-names" tagx="given-names" title="given-names">C. S.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">S. E.</span></span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Modern uses of electron microscopy for detection of viruses</span>, <span class="source" tagx="source" title="source">Clinical Microbiology Reviews</span>, <span class="volume" tagx="volume" title="volume">22</span> (<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">552</span>â€"<span class="lpage" tagx="lpage" title="lpage">563</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19822888">19822888</a></span></span></li> <li tag="ref"><a name="hzw010C27" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grandien</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pettersson</span><span class="given-names" tagx="given-names" title="given-names">C. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Svensson</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">1987</span>) <span class="mixed-article-title" title="mixed-article-title">Latex agglutination test for adenovirus diagnosis in diarrheal disease</span>, <span class="source" tagx="source" title="source">Journal of Medical Virology</span>, <span class="volume" tagx="volume" title="volume">23</span> (<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">311</span>â€"<span class="lpage" tagx="lpage" title="lpage">316</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2826673">2826673</a></span></span></li> <li tag="ref"><a name="hzw010C28" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pandey</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Correlation between two chemiluminescence based assays for quantification of Hepatitis B surface antigen in patients with chronic Hepatitis B infection</span>, <span class="source" tagx="source" title="source">Indian Journal of Medical Microbiology</span>, <span class="volume" tagx="volume" title="volume">33</span> (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">96</span>â€"<span class="lpage" tagx="lpage" title="lpage">100</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25560010">25560010</a></span></span></li> <li tag="ref"><a name="hzw010C29" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guzman</span><span class="given-names" tagx="given-names" title="given-names">M. G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jaenisch</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaczkowski</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Multi-country evaluation of the sensitivity and specificity of two commercially available NS1 ELISA assays for dengue diagnosis</span>, <span class="source" tagx="source" title="source">PLOS Neglected Tropical Diseases</span>, <span class="volume" tagx="volume" title="volume">4</span> (<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">e811</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20824173">20824173</a></span></span></li> <li tag="ref"><a name="hzw010C30" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">R. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peacey</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Q. S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Rapid method to support diagnosis of swine-origin influenza virus infection by sequencing of real-time PCR amplicons from diagnostic assays</span>, <span class="source" tagx="source" title="source">Journal of Clinical Microbiology</span>, <span class="volume" tagx="volume" title="volume">47</span>, <span class="fpage" tagx="fpage" title="fpage">3053</span>â€"<span class="lpage" tagx="lpage" title="lpage">3054</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19587304">19587304</a></span></span></li> <li tag="ref"><a name="hzw010C31" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hawkins</span><span class="given-names" tagx="given-names" title="given-names">R. C.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Laboratory turnaround time</span>, <span class="source" tagx="source" title="source">The Clinical Biochemist Reviews</span>, <span class="volume" tagx="volume" title="volume">28</span> (<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">179</span>â€"<span class="lpage" tagx="lpage" title="lpage">194</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18392122">18392122</a></span></span></li> <li tag="ref"><a name="hzw010C32" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hematiana</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sadeghifarda</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohebia</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Traditional and modern cell culture in virus diagnosis</span>, <span class="source" tagx="source" title="source">Osong Public Health and Research Perspectives</span>, <span class="volume" tagx="volume" title="volume">7</span> (<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">77</span>â€"<span class="lpage" tagx="lpage" title="lpage">82</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27169004">27169004</a></span></span></li> <li tag="ref"><a name="hzw010C33" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong</span><span class="given-names" tagx="given-names" title="given-names">S. P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">N. K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hwang</span><span class="given-names" tagx="given-names" title="given-names">S. G.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">Detection of Hepatitis B virus YMDD variants using mass spectrometric analysis of oligonucleotide fragments</span>, <span class="source" tagx="source" title="source">Journal of Hepatology</span>, <span class="volume" tagx="volume" title="volume">40</span> (<span class="issue" tagx="issue" title="issue">5</span>), <span class="fpage" tagx="fpage" title="fpage">837</span>â€"<span class="lpage" tagx="lpage" title="lpage">844</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15094233">15094233</a></span></span></li> <li tag="ref"><a name="hzw010C34" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Isakov</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">BorderÃ­a</span><span class="given-names" tagx="given-names" title="given-names">A. V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Golan</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Deep sequencing analysis of viral infection and evolution allows rapid and detailed characterization of viral mutant spectrum</span>, <span class="source" tagx="source" title="source">Bioinformatics</span>, <span class="volume" tagx="volume" title="volume">31</span> (<span class="issue" tagx="issue" title="issue">13</span>), <span class="fpage" tagx="fpage" title="fpage">2141</span>â€"<span class="lpage" tagx="lpage" title="lpage">2150</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25701575">25701575</a></span></span></li> <li tag="ref"><a name="hzw010C35" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jackson</span><span class="given-names" tagx="given-names" title="given-names">J. B.</span></span></span> (<span class="year" tagx="year" title="year">1990</span>) <span class="mixed-article-title" title="mixed-article-title">Polymerase chain reaction in transfusion medicine</span>, <span class="source" tagx="source" title="source">Transfusion</span>, <span class="volume" tagx="volume" title="volume">30</span> (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">51</span>â€"<span class="lpage" tagx="lpage" title="lpage">57</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2404355">2404355</a></span></span></li> <li tag="ref"><a name="hzw010C36" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiangqin</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viswanath</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jikun</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Nanomicroarray and multiplex next-generation sequencing for simultaneous identification and characterization of influenza viruses</span>, <span class="source" tagx="source" title="source">Emerging Infectious Diseases</span>, <span class="volume" tagx="volume" title="volume">21</span> (<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">400</span>â€"<span class="lpage" tagx="lpage" title="lpage">408</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25694248">25694248</a></span></span></li> <li tag="ref"><a name="hzw010C37" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">Y. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">S. O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chung</span><span class="given-names" tagx="given-names" title="given-names">H. J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Population genotyping of Hepatitis C virus by matrix-assisted laser desorption/ionization time-of-flight mass spectrometry analysis of short DNA fragments</span>, <span class="source" tagx="source" title="source">Journal of Clinical Chemistry</span>, <span class="volume" tagx="volume" title="volume">51</span> (<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">1123</span>â€"<span class="lpage" tagx="lpage" title="lpage">1131</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15890891">15890891</a></span></span></li> <li tag="ref"><a name="hzw010C38" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kokkayil</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kurapati</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Negi</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Comparative evaluation of a reverse transcriptase based assay for HIV-1 viral load quantitation in resource-limited settings</span>, <span class="source" tagx="source" title="source">Journal of Virological Methods</span>, <span class="volume" tagx="volume" title="volume">203</span>, <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">4</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24671025">24671025</a></span></span></li> <li tag="ref"><a name="hzw010C39" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lalkhen</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCluskey</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Clinical tests: sensitivity and specificity, continuing education in anaesthesia</span>, <span class="source" tagx="source" title="source">Critical Care &amp;amp; Pain</span>, <span class="volume" tagx="volume" title="volume">8</span> (<span class="issue" tagx="issue" title="issue">6</span>), <span class="fpage" tagx="fpage" title="fpage">221</span>â€"<span class="lpage" tagx="lpage" title="lpage">223</span>.</span></li> <li tag="ref"><a name="hzw010C40" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemon</span><span class="given-names" tagx="given-names" title="given-names">S. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamburg</span><span class="given-names" tagx="given-names" title="given-names">M. A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sparling</span><span class="given-names" tagx="given-names" title="given-names">P. F.</span></span>, <span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2007</span>) Global infectious disease surveillance and detection: assessing the challengesâ€"finding solutions. Workshop Summary. Institute of Medicine (US) Forum on Microbial Threats, National Academies Press (US).</span></li> <li tag="ref"><a name="hzw010C41" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leung</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bull</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lloyd</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">A bioinformatics pipeline for the analyses of viral escape dynamics and host immune responses during an infection</span>, <span class="source" tagx="source" title="source">BioMed Research International</span>, 2014, <span class="fpage" tagx="fpage" title="fpage">264519</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25013771">25013771</a></span></span></li> <li tag="ref"><a name="hzw010C42" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LÃ©vÃªque</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Legoff</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mengelle</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Virological diagnosis of central nervous system infections by use of PCR coupled with mass spectrometry analysis of cerebrospinal fluid samples</span>, <span class="source" tagx="source" title="source">Journal of Clinical Microbiology</span>, <span class="volume" tagx="volume" title="volume">52</span> (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">212</span>â€"<span class="lpage" tagx="lpage" title="lpage">217</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24197874">24197874</a></span></span></li> <li tag="ref"><a name="hzw010C43" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Variant callers for next-generation sequencing data: a comparison study</span>, <span class="source" tagx="source" title="source">PLoS One</span>, <span class="volume" tagx="volume" title="volume">8</span> (<span class="issue" tagx="issue" title="issue">9</span>), <span class="fpage" tagx="fpage" title="fpage">e75619</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24086590">24086590</a></span></span></li> <li tag="ref"><a name="hzw010C44" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loman</span><span class="given-names" tagx="given-names" title="given-names">N. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Misra</span><span class="given-names" tagx="given-names" title="given-names">R. V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dallman</span><span class="given-names" tagx="given-names" title="given-names">T. J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Performance comparison of benchtop high-throughput sequencing platforms</span>, <span class="source" tagx="source" title="source">Nature Biotechnology</span>, <span class="volume" tagx="volume" title="volume">30</span>, <span class="fpage" tagx="fpage" title="fpage">434</span>â€"<span class="lpage" tagx="lpage" title="lpage">439</span>.</span></li> <li tag="ref"><a name="hzw010C45" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lowe</span><span class="given-names" tagx="given-names" title="given-names">C. F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Merrick</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrigan</span><span class="given-names" tagx="given-names" title="given-names">P. R.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2016</span>) <span class="mixed-article-title" title="mixed-article-title">Implementation of next-generation sequencing for Hepatitis B virus resistance testing and genotyping in a clinical microbiology laboratory</span>, <span class="source" tagx="source" title="source">Journal of Clinical Microbiology</span>, <span class="volume" tagx="volume" title="volume">54</span> (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">127</span>â€"<span class="lpage" tagx="lpage" title="lpage">133</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26537448">26537448</a></span></span></li> <li tag="ref"><a name="hzw010C46" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luan</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Multiplex detection of 60 Hepatitis B virus variants by MALDI-TOF mass spectrometry</span>, <span class="source" tagx="source" title="source">Journal of Clinical Chemistry</span>, <span class="volume" tagx="volume" title="volume">55</span> (<span class="issue" tagx="issue" title="issue">8</span>), <span class="fpage" tagx="fpage" title="fpage">1503</span>â€"<span class="lpage" tagx="lpage" title="lpage">1509</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19541863">19541863</a></span></span></li> <li tag="ref"><a name="hzw010C47" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mahony</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chong</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Merante</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">Development of a respiratory virus panel test for detection of twenty human respiratory viruses by use of multiplex PCR and a fluid microbead-based assay</span>, <span class="source" tagx="source" title="source">Journal of Clinical Microbiology</span>, <span class="volume" tagx="volume" title="volume">45</span>, <span class="fpage" tagx="fpage" title="fpage">2965</span>â€"<span class="lpage" tagx="lpage" title="lpage">2970</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17596360">17596360</a></span></span></li> <li tag="ref"><a name="hzw010C48" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Margeridon-Thermet</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shulman</span><span class="given-names" tagx="given-names" title="given-names">N. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmed</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Ultra-deep pyrosequencing of Hepatitis B virus quasispecies from nucleoside and nucleotide reverse-transcriptase inhibitor (NRTI)â€"treated patients and NRTI-naive patients</span>, <span class="source" tagx="source" title="source">Journal of Infectious Diseases</span>, <span class="volume" tagx="volume" title="volume">199</span> (<span class="issue" tagx="issue" title="issue">9</span>), <span class="fpage" tagx="fpage" title="fpage">1275</span>â€"<span class="lpage" tagx="lpage" title="lpage">1285</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19301976">19301976</a></span></span></li> <li tag="ref"><a name="hzw010C49" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McIntyre</span><span class="given-names" tagx="given-names" title="given-names">P. G.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">How many drops of CSF is enough</span>. <span class="source" tagx="source" title="source">Postgraduate Medical Journal</span>, <span class="volume" tagx="volume" title="volume">83</span> (<span class="issue" tagx="issue" title="issue">977</span>), <span class="fpage" tagx="fpage" title="fpage">158</span>.</span></li> <li tag="ref"><a name="hzw010C89" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mengelle</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mansury</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Da Silva</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guerin</span><span class="given-names" tagx="given-names" title="given-names">J. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Izopet</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Evaluation of a polymerase chain reaction-electrospray ionization time-of-flight mass spectrometry for the detection and subtyping of influenza viruses in respiratory specimens</span>. <span class="source" tagx="source" title="source">Journal of Clinical Virology</span>, <span class="volume" tagx="volume" title="volume">57</span> (<span class="issue" tagx="issue" title="issue">3</span>), <span class="fpage" tagx="fpage" title="fpage">222</span>â€"<span class="lpage" tagx="lpage" title="lpage">226</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23557709">23557709</a></span></span></li> <li tag="ref"><a name="hzw010C50" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mercier-Delarue</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vray</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plantier</span><span class="given-names" tagx="given-names" title="given-names">J. C.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Higher specificity of NASBA isothermal technology versus real-time PCR for HIV-1 RNA quantification on dried blood spots</span>, <span class="source" tagx="source" title="source">Journal of clinical microbiology</span>, <span class="volume" tagx="volume" title="volume">52</span> (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">52</span>â€"<span class="lpage" tagx="lpage" title="lpage">6</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24131691">24131691</a></span></span></li> <li tag="ref"><a name="hzw010C51" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">J. J.</span></span></span> (<span class="year" tagx="year" title="year">2004</span>) Interference in immunoassays: avoiding erroneous results. <i>Clinical Laboratory International</i> (Published in CLI April 2004) <span class="uri" title="uri"><a href="http://www.cli-online.com/fileadmin/pdf/pdf_general/interference-in-immunoassays-avoiding-erroneous-results.pdf">http://www.cli-online.com/fileadmin/pdf/pdf_general/interference-in-immunoassays-avoiding-erroneous-results.pdf</a></span></span></li> <li tag="ref"><a name="hzw010C52" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mixson-Hayden</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dawson</span><span class="given-names" tagx="given-names" title="given-names">G. J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teshale</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Performance of ARCHITECT HCV core antigen test with specimens from US plasma donors and injecting drug users</span>, <span class="source" tagx="source" title="source">Journal of clinical virology</span>, <span class="volume" tagx="volume" title="volume">66</span>, <span class="fpage" tagx="fpage" title="fpage">15</span>â€"<span class="lpage" tagx="lpage" title="lpage">18</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25866329">25866329</a></span></span></li> <li tag="ref"><a name="hzw010C53" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mullis</span><span class="given-names" tagx="given-names" title="given-names">K. B.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faloona</span><span class="given-names" tagx="given-names" title="given-names">F. A.</span></span></span> (<span class="year" tagx="year" title="year">1987</span>) <span class="mixed-article-title" title="mixed-article-title">Specific synthesis of DNA in vitro via a polymerase-catalysed chain reaction</span>, <span class="source" tagx="source" title="source">Methods in Enzymology</span>, <span class="volume" tagx="volume" title="volume">155</span>, <span class="fpage" tagx="fpage" title="fpage">335</span>â€"<span class="lpage" tagx="lpage" title="lpage">350</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3431465">3431465</a></span></span></li> <li tag="ref"><a name="hzw010C54" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naccache</span><span class="given-names" tagx="given-names" title="given-names">S. N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Federman</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veeraraghavan</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">A cloud-compatible bioinformatics pipeline for ultra-rapid pathogen identification from next-generation sequencing of clinical samples</span>, <span class="source" tagx="source" title="source">Genome Research</span>, <span class="volume" tagx="volume" title="volume">24</span> (<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">1180</span>â€"<span class="lpage" tagx="lpage" title="lpage">1192</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24899342">24899342</a></span></span></li> <li tag="ref"><a name="hzw010C55" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ntziora</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paraskevis</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haida</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Ultrasensitive amplification refractory mutation system real-time PCR (ARMS RT-PCR) assay for detection of minority Hepatitis B virus-resistant strains in the era of personalized medicine</span>, <span class="source" tagx="source" title="source">Journal of Clinical Microbiology</span>, <span class="volume" tagx="volume" title="volume">51</span> (<span class="issue" tagx="issue" title="issue">9</span>), <span class="fpage" tagx="fpage" title="fpage">2893</span>â€"<span class="lpage" tagx="lpage" title="lpage">2900</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23804383">23804383</a></span></span></li> <li tag="ref"><a name="hzw010C56" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palacios</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Druce</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">A new arenavirus in a cluster of fatal transplant-associated diseases</span>, <span class="source" tagx="source" title="source">The New England Journal of Medicine</span>, <span class="volume" tagx="volume" title="volume">358</span> (<span class="issue" tagx="issue" title="issue">10</span>), <span class="fpage" tagx="fpage" title="fpage">991</span>â€"<span class="lpage" tagx="lpage" title="lpage">998</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18256387">18256387</a></span></span></li> <li tag="ref"><a name="hzw010C57" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palmer</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kearney</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maldarelli</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Multiple, linked human immunodeficiency virus type 1 drug resistance mutations in treatment-experienced patients are missed by standard genotype analysis</span>, <span class="source" tagx="source" title="source">Journal of Clinical Microbiology</span>, <span class="volume" tagx="volume" title="volume">43</span>, <span class="fpage" tagx="fpage" title="fpage">406</span>â€"<span class="lpage" tagx="lpage" title="lpage">413</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15635002">15635002</a></span></span></li> <li tag="ref"><a name="hzw010C58" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Papafragkou</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hewitt</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">G. W.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Challenges of culturing human norovirus in three-dimensional organoid intestinal cell culture models</span>, <span class="source" tagx="source" title="source">PLoS ONE</span>, <span class="volume" tagx="volume" title="volume">8</span> (<span class="issue" tagx="issue" title="issue">6</span>), <span class="fpage" tagx="fpage" title="fpage">e63485</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23755105">23755105</a></span></span></li> <li tag="ref"><a name="hzw010C59" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">B. S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choi</span><span class="given-names" tagx="given-names" title="given-names">K. H.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">A novel multiplex real-time PCR assay for the concurrent detection of Hepatitis A, B and C viruses in patients with acute hepatitis</span>, <span class="source" tagx="source" title="source">Plos One</span>, <span class="volume" tagx="volume" title="volume">7</span> (<span class="issue" tagx="issue" title="issue">11</span>), <span class="fpage" tagx="fpage" title="fpage">e49106</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23145085">23145085</a></span></span></li> <li tag="ref"><a name="hzw010C60" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patton</span><span class="given-names" tagx="given-names" title="given-names">J. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coovadia</span><span class="given-names" tagx="given-names" title="given-names">A. H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyers</span><span class="given-names" tagx="given-names" title="given-names">T. M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Evaluation of the ultrasensitive human immunodeficiency virus type 1 (HIV-1) p24 antigen assay performed on dried blood spots for diagnosis of HIV-1 infection in infants</span>, <span class="source" tagx="source" title="source">Clinical and Vaccine Immunology</span>, <span class="volume" tagx="volume" title="volume">15</span> (<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">388</span>â€"<span class="lpage" tagx="lpage" title="lpage">91</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18057183">18057183</a></span></span></li> <li tag="ref"><a name="hzw010C61" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pierce</span><span class="given-names" tagx="given-names" title="given-names">V. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elkan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leet</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Comparison of the Idaho Technology FilmArray system to real-time PCR for detection of respiratory pathogens in children</span>, <span class="source" tagx="source" title="source">Journal of Clinical Microbiology</span>, <span class="volume" tagx="volume" title="volume">50</span> (<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">364</span>â€"<span class="lpage" tagx="lpage" title="lpage">371</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22116144">22116144</a></span></span></li> <li tag="ref"><a name="hzw010C62" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pyne</span><span class="given-names" tagx="given-names" title="given-names">M. T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vest</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clement</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Comparison of three Roche Hepatitis B virus viral load assay formats</span>, <span class="source" tagx="source" title="source">Journal of Clinical Microbiology</span>, <span class="volume" tagx="volume" title="volume">50</span> (<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">2337</span>â€"<span class="lpage" tagx="lpage" title="lpage">2342</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22535983">22535983</a></span></span></li> <li tag="ref"><a name="hzw010C63" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qian</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Detection of HIV-1 p24 antigen using streptavidinâ€"biotin and gold nanoparticles based immunoassay by inductively coupled plasma mass spectrometry</span>, <span class="source" tagx="source" title="source">Journal of Analytical Atomic Spectrometry</span>, <span class="volume" tagx="volume" title="volume">29</span>, <span class="fpage" tagx="fpage" title="fpage">1477</span>â€"<span class="lpage" tagx="lpage" title="lpage">1482</span>.</span></li> <li tag="ref"><a name="hzw010C64" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reijans</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dingemans</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klaassen</span><span class="given-names" tagx="given-names" title="given-names">C. H.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">RespiFinder: a new multiparameter test to differentially identify fifteen respiratory viruses</span>, <span class="source" tagx="source" title="source">Journal of Clinical Microbiology</span>, <span class="volume" tagx="volume" title="volume">46</span>, <span class="fpage" tagx="fpage" title="fpage">1232</span>â€"<span class="lpage" tagx="lpage" title="lpage">1240</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18256230">18256230</a></span></span></li> <li tag="ref"><a name="hzw010C65" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Renois</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Talmud</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huguenin</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Rapid detection of respiratory tract viral infections and coinfections in patients with influenza-like illnesses by use of reverse transcription-PCR DNA microarray systems</span>, <span class="source" tagx="source" title="source">Journal of Clinical Microbiology</span>, <span class="volume" tagx="volume" title="volume">48</span>, <span class="fpage" tagx="fpage" title="fpage">3836</span>â€"<span class="lpage" tagx="lpage" title="lpage">3842</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20739481">20739481</a></span></span></li> <li tag="ref"><a name="hzw010C66" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robbins</span><span class="given-names" tagx="given-names" title="given-names">F. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enders</span><span class="given-names" tagx="given-names" title="given-names">J. F.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weller</span><span class="given-names" tagx="given-names" title="given-names">T. H.</span></span></span> (<span class="year" tagx="year" title="year">1950</span>) <span class="mixed-article-title" title="mixed-article-title">Cytopathogenic effect of poliomyelitis viruses in vitro on human embryonic tissues</span>, <span class="source" tagx="source" title="source">Experimental Biology and Medicine</span>, <span class="volume" tagx="volume" title="volume">75</span> (<span class="issue" tagx="issue" title="issue">2</span>), <span class="fpage" tagx="fpage" title="fpage">370</span>â€"<span class="lpage" tagx="lpage" title="lpage">374</span>.</span></li> <li tag="ref"><a name="hzw010C67" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rothberg</span><span class="given-names" tagx="given-names" title="given-names">J. M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hinz</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rearick</span><span class="given-names" tagx="given-names" title="given-names">T. M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">An integrated semiconductor device enabling non-optical genome sequencing</span>, <span class="source" tagx="source" title="source">Nature</span>, <span class="volume" tagx="volume" title="volume">475</span>, <span class="fpage" tagx="fpage" title="fpage">348</span>â€"<span class="lpage" tagx="lpage" title="lpage">52</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21776081">21776081</a></span></span></li> <li tag="ref"><a name="hzw010C68" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sandeep</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rajesh</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antony Raj</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title">Rapid particle agglutination test for human immunodeficiency virus: hospital-based evaluation</span>, <span class="source" tagx="source" title="source">Journal of Clinical Microbiology</span>, <span class="volume" tagx="volume" title="volume">40</span> (<span class="issue" tagx="issue" title="issue">4</span>), <span class="fpage" tagx="fpage" title="fpage">1553</span>â€"<span class="lpage" tagx="lpage" title="lpage">1554</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11923395">11923395</a></span></span></li> <li tag="ref"><a name="hzw010C69" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanger</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicklen</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coulson</span><span class="given-names" tagx="given-names" title="given-names">A. R.</span></span></span> (<span class="year" tagx="year" title="year">1977</span>) <span class="mixed-article-title" title="mixed-article-title">DNA sequencing with chain-terminating inhibitors</span>, <span class="source" tagx="source" title="source">Proceedings of the National Academy of Science USA</span>, <span class="volume" tagx="volume" title="volume">74</span>, <span class="fpage" tagx="fpage" title="fpage">5463</span>â€"<span class="lpage" tagx="lpage" title="lpage">5467</span>.</span></li> <li tag="ref"><a name="hzw010C70" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sauer</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kliem</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Mass spectrometry tools for the classification and identification of bacteria</span>, <span class="source" tagx="source" title="source">Nature Reviews Microbiology</span>, <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">74</span>â€"<span class="lpage" tagx="lpage" title="lpage">82</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20010952">20010952</a></span></span></li> <li tag="ref"><a name="hzw010C71" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shedure</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ji</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span></span> (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Next generation DNA sequencing</span>, <span class="source" tagx="source" title="source">Nature Biotechnology</span>, <span class="volume" tagx="volume" title="volume">26</span>, <span class="fpage" tagx="fpage" title="fpage">1135</span>â€"<span class="lpage" tagx="lpage" title="lpage">1145</span>.</span></li> <li tag="ref"><a name="hzw010C72" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shinkai</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsuura</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sugauchi</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Application of a newly developed high-sensitivity HBsAg chemiluminescent enzyme immunoassay for Hepatitis B patients with HBsAg seroclearance</span>, <span class="source" tagx="source" title="source">Journal of Clinical Microbiology</span>, <span class="volume" tagx="volume" title="volume">51</span> (<span class="issue" tagx="issue" title="issue">11</span>), <span class="fpage" tagx="fpage" title="fpage">3484</span>â€"<span class="lpage" tagx="lpage" title="lpage">3491</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23946517">23946517</a></span></span></li> <li tag="ref"><a name="hzw010C73" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">H. O.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">J. H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryu</span><span class="given-names" tagx="given-names" title="given-names">H. S.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Polymeric labchip real-time PCR as a point-of-care-potential diagnostic tool for rapid detection of Influenza A/H1N1 virus in human clinical specimens</span>, <span class="source" tagx="source" title="source">PLOS One</span>, <span class="volume" tagx="volume" title="volume">7</span> (<span class="issue" tagx="issue" title="issue">12</span>), <span class="fpage" tagx="fpage" title="fpage">e53325</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23285281">23285281</a></span></span></li> <li tag="ref"><a name="hzw010C74" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Szpara</span><span class="given-names" tagx="given-names" title="given-names">M. L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parsons</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enquist</span><span class="given-names" tagx="given-names" title="given-names">L. W.</span></span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Sequence variability in clinical and laboratory isolates of herpes simplex virus 1 reveals new mutations</span>, <span class="source" tagx="source" title="source">Journal of Virology</span>, <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">5303</span>â€"<span class="lpage" tagx="lpage" title="lpage">5313</span>.</span></li> <li tag="ref"><a name="hzw010C75" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tassopoulos</span><span class="given-names" tagx="given-names" title="given-names">N. C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Papatheodoridis</span><span class="given-names" tagx="given-names" title="given-names">G. V.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kalantzakis</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">1997</span>) <span class="mixed-article-title" title="mixed-article-title">Differential diagnosis of acute HBsAg positive hepatitis using IgM anti-HBc by a rapid, fully automated microparticle enzyme immunoassay</span>, <span class="source" tagx="source" title="source">Journal of Hepatology</span>, <span class="volume" tagx="volume" title="volume">26</span>, <span class="fpage" tagx="fpage" title="fpage">14</span>â€"<span class="lpage" tagx="lpage" title="lpage">19</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9148005">9148005</a></span></span></li> <li tag="ref"><a name="hzw010C76" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van den Hoecke</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verhelst</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vuylsteke</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2015</span>) <span class="mixed-article-title" title="mixed-article-title">Analysis of the genetic diversity of Influenza A viruses using next-generation DNA sequencing</span>, <span class="source" tagx="source" title="source">BMC Genomics</span>, <span class="volume" tagx="volume" title="volume">16</span>, <span class="fpage" tagx="fpage" title="fpage">79</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25758772">25758772</a></span></span></li> <li tag="ref"><a name="hzw010C77" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandergeeten</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fromentin</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Merlini</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Cross-clade ultrasensitive PCR-based assays to measure HIV persistence in large-cohort studies</span>, <span class="source" tagx="source" title="source">Journal of Virology</span>, <span class="volume" tagx="volume" title="volume">88</span> (<span class="issue" tagx="issue" title="issue">21</span>), <span class="fpage" tagx="fpage" title="fpage">12385</span>â€"<span class="lpage" tagx="lpage" title="lpage">12396</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25122785">25122785</a></span></span></li> <li tag="ref"><a name="hzw010C78" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vekemans</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">John</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colebunders</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span> (<span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">when to switch for antiretroviral treatment failure in resource limited settings</span>. <span class="source" tagx="source" title="source">AIDS</span>, <span class="volume" tagx="volume" title="volume">21</span>, <span class="fpage" tagx="fpage" title="fpage">1205</span>â€"<span class="lpage" tagx="lpage" title="lpage">6</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17502731">17502731</a></span></span></li> <li tag="ref"><a name="hzw010C79" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verbinnen</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Marck</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandenbroucke</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Tracking the evolution of multiple in vitro Hepatitis C virus replicon variants under protease inhibitor selection pressure by 454 deep sequencing</span>, <span class="source" tagx="source" title="source">Journal of Virology</span>, <span class="volume" tagx="volume" title="volume">84</span>, <span class="fpage" tagx="fpage" title="fpage">11124</span>â€"<span class="lpage" tagx="lpage" title="lpage">11133</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20739521">20739521</a></span></span></li> <li tag="ref"><a name="hzw010C80" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Voller</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bidwell</span><span class="given-names" tagx="given-names" title="given-names">D. E.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartlett</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span> (<span class="year" tagx="year" title="year">1976</span>) <span class="mixed-article-title" title="mixed-article-title">Enzyme immunoassays in diagnostic medicine</span>, <span class="source" tagx="source" title="source">Bulletin of the World Health Organization</span>, <span class="volume" tagx="volume" title="volume">53</span> (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">55</span>â€"<span class="lpage" tagx="lpage" title="lpage">65</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1085667">1085667</a></span></span></li> <li tag="ref"><a name="hzw010C81" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Development of a novel DNA sequencing method not only for Hepatitis B virus genotyping but also for drug resistant mutation detection</span>, <span class="source" tagx="source" title="source">BMC Med Genomics</span>, <span class="volume" tagx="volume" title="volume">6</span> (<span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">S15</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23369292">23369292</a></span></span></li> <li tag="ref"><a name="hzw010C82" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="collab'">collab: WHO</span></span>-fact sheet NÂ°204/ July 2014. <span class="uri" title="uri"><a href="http://www.who.int/mediacentre/factsheets/fs204/en/">http://www.who.int/mediacentre/factsheets/fs204/en/</a></span></span></li> <li tag="ref"><a name="hzw010C83" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">WHO. Antiretroviral therapy for HIV infection in adults and adolescents. <span class="year" tagx="year" title="year">2006</span>a. Available at: <span class="uri" title="uri"><a href="http://www.who.int/hiv/pub/guidelines/artadultguidelines.pdf">http://www.who.int/hiv/pub/guidelines/artadultguidelines.pdf</a></span> (25 November 2009).</span></li> <li tag="ref"><a name="hzw010C84" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">Detection of serum HCV RNA in patients with chronic Hepatitis C by transcription mediated amplification and real-time reverse transcription polymerase chain reaction</span>, <span class="source" tagx="source" title="source">Zhong nan da xue xue Bao. Yi xue ban</span>, <span class="volume" tagx="volume" title="volume">39</span> (<span class="issue" tagx="issue" title="issue">7</span>), <span class="fpage" tagx="fpage" title="fpage">664</span>â€"<span class="lpage" tagx="lpage" title="lpage">672</span> = Journal of Central South University. Medical Sciences.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25080913">25080913</a></span></span></li> <li tag="ref"><a name="hzw010C85" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span> and <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaman</span><span class="given-names" tagx="given-names" title="given-names">M. H.</span></span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Low-cost tools for diagnosing and monitoring HIV infection in low resource settings</span>, <span class="source" tagx="source" title="source">Bulletin of the World Health Organization</span>, <span class="volume" tagx="volume" title="volume">90</span>, <span class="fpage" tagx="fpage" title="fpage">914920</span>.</span></li> <li tag="ref"><a name="hzw010C86" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Astion</span><span class="given-names" tagx="given-names" title="given-names">M. L.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Implementation of film-array respiratory viral panel in a core laboratory improves testing turnaround time and patient care</span>, <span class="source" tagx="source" title="source">American Journal of Clinical Pathology</span>, <span class="volume" tagx="volume" title="volume">139</span>, <span class="fpage" tagx="fpage" title="fpage">118</span>â€"<span class="lpage" tagx="lpage" title="lpage">123</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23270907">23270907</a></span></span></li> <li tag="ref"><a name="hzw010C87" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yea</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCorrister</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Westmacott</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">Early detection of Influenza A (H5) viruses with affinity for the human sialic acid receptor by MALDI-TOF mass spectrometry based mutation detection</span>, <span class="source" tagx="source" title="source">Journal of Virological Methods</span>, <span class="volume" tagx="volume" title="volume">172</span>, <span class="fpage" tagx="fpage" title="fpage">72</span>â€"<span class="lpage" tagx="lpage" title="lpage">7</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21195111">21195111</a></span></span></li> <li tag="ref"><a name="hzw010C88" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhi</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="etal" title="etal"><i>et al.</i></span></span> (<span class="year" tagx="year" title="year">2014</span>) <span class="mixed-article-title" title="mixed-article-title">A novel HBV genotypes detecting system combined with microfluidic chip, loop-mediated isothermal amplification and GMR sensors</span>, <span class="source" tagx="source" title="source">Biosensors and Bioelectronics</span>, <span class="volume" tagx="volume" title="volume">54</span>, <span class="fpage" tagx="fpage" title="fpage">372</span>â€"<span class="lpage" tagx="lpage" title="lpage">377</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24292142">24292142</a></span></span></li> </ul> </div> </div>  </body></html>