<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/02/2020.02.27.967422.atom" />
<meta name="HW.pisa" content="biorxiv;2020.02.27.967422v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Strategies for vaccine design for corona virus using Immunoinformatics techniques" />
<meta name="DC.Identifier" content="10.1101/2020.02.27.967422" />
<meta name="DC.Date" content="2020-03-02" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The cutting-edge technology vaccinomics is the combination of two topics immunogenetics and immunogenomics with the knowledge of systems biology and immune profiling for designing vaccine against infectious disease. In our present study, an epitope-based peptide vaccine against nonstructural protein 4 of beta coronavirus, using a combination of B cell and T cell epitope predictions, followed by molecular docking methods are performed. Here, protein sequences of homologous nonstructural protein 4 of beta coronavirus are collected and conserved regions present in them are investigated via phylogenetic study to determine the most immunogenic part of protein. From the identified region of the target protein, the peptide sequence IRNTTNPSAR from the region ranging from 38-47 and the sequence PTDTYTSVYLGKFRG from the positions of 76-90 are considered as the most potential B cell and T cell epitopes respectively. Furthermore, this predicted T cell epitopes PTDTYTSVY and PTDTYTSVYLGKFRG interacted with MHC allelic proteins HLA-A*01:01 and HLA-DRB5*01:01 respectively with the low IC50 values. These epitopes are perfectly fitted into the epitope binding grooves of alpha helix of MHC I molecule and MHC II molecule with binding energy scores −725.0 Kcal/mole and −786.0 Kcal/mole respectively, showing stability in MHC molecules binding. This MHC restricted epitope PTDTYTSVY also showed a good conservancy of 50.16% in world population coverage. This MHC I HLA-A*01:01 allele is present among 58.87% of Chinese population also. Therefore, the epitopes IRNTTNPSAR and PTDTYTSVYLGKFRG may be considered as potential peptides for peptide-based vaccine for coronavirus after further experimental study." />
<meta name="DC.Contributor" content="Anamika Basu" />
<meta name="DC.Contributor" content="Anasua Sarkar" />
<meta name="DC.Contributor" content="Ujjwal Maulik" />
<meta name="article:published_time" content="2020-03-02" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Strategies for vaccine design for corona virus using Immunoinformatics techniques" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The cutting-edge technology vaccinomics is the combination of two topics immunogenetics and immunogenomics with the knowledge of systems biology and immune profiling for designing vaccine against infectious disease. In our present study, an epitope-based peptide vaccine against nonstructural protein 4 of beta coronavirus, using a combination of B cell and T cell epitope predictions, followed by molecular docking methods are performed. Here, protein sequences of homologous nonstructural protein 4 of beta coronavirus are collected and conserved regions present in them are investigated via phylogenetic study to determine the most immunogenic part of protein. From the identified region of the target protein, the peptide sequence IRNTTNPSAR from the region ranging from 38-47 and the sequence PTDTYTSVYLGKFRG from the positions of 76-90 are considered as the most potential B cell and T cell epitopes respectively. Furthermore, this predicted T cell epitopes PTDTYTSVY and PTDTYTSVYLGKFRG interacted with MHC allelic proteins HLA-A*01:01 and HLA-DRB5*01:01 respectively with the low IC50 values. These epitopes are perfectly fitted into the epitope binding grooves of alpha helix of MHC I molecule and MHC II molecule with binding energy scores -725.0 Kcal/mole and -786.0 Kcal/mole respectively, showing stability in MHC molecules binding. This MHC restricted epitope PTDTYTSVY also showed a good conservancy of 50.16% in world population coverage. This MHC I HLA-A*01:01 allele is present among 58.87% of Chinese population also. Therefore, the epitopes IRNTTNPSAR and PTDTYTSVYLGKFRG may be considered as potential peptides for peptide-based vaccine for coronavirus after further experimental study.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.02.27.967422v1" />
<meta name="citation_id" content="2020.02.27.967422v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.27.967422v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.27.967422v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.27.967422v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.02.27.967422" />
<meta name="citation_num_pages" content="20" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.02.27.967422" />
<meta name="citation_author" content="Anamika Basu" />
<meta name="citation_author_institution" content="Assistant Professor, Gurudas College" />
<meta name="citation_author" content="Anasua Sarkar" />
<meta name="citation_author_institution" content="Computer Science and Engineering Department, Jadavpur University" />
<meta name="citation_author_email" content="ashru2006@hotmail.com" />
<meta name="citation_author_email" content="anasua.sarkar@jadavpuruniversity.in" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7365-3924" />
<meta name="citation_author" content="Ujjwal Maulik" />
<meta name="citation_author_institution" content="Computer Science and Engineering Department, Jadavpur University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1167-0774" />
<meta name="citation_reference" content="Lu, R., Zhao, X., Li, J., Niu, P., Yang, B., Wu, H., &amp; Bi, Y. (2020). Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. The Lancet." />
<meta name="citation_reference" content="Chan, J. F. W., Kok, K. H., Zhu, Z., Chu, H., To, K. K. W., Yuan, S., &amp; Yuen, K. Y. (2020). Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan. Emerging Microbes &amp; Infections, 9(1), 221–236." />
<meta name="citation_reference" content="Doyle, N., Neuman, B. W., Simpson, J., Hawes, P. C., Mantell, J., Verkade, P., &amp; Maier, H. J. (2018). Infectious bronchitis virus nonstructural protein 4 alone induces membrane pairing. Viruses, 10(9), 477." />
<meta name="citation_reference" content="Züst, R., Cervantes-Barragán, L., Kuri, T., Blakqori, G., Weber, F., Ludewig, B., &amp; Thiel, V. (2007). Coronavirus non-structural protein 1 is a major pathogenicity factor: implications for the rational design of coronavirus vaccines. PLoS pathogens, 3(8)." />
<meta name="citation_reference" content="Sakai, Y., Kawachi, K., Terada, Y., Omori, H., Matsuura, Y., &amp; Kamitani, W. (2017). Two-amino acids change in the nsp4 of SARS coronavirus abolishes viral replication. Virology, 510, 165–174." />
<meta name="citation_reference" content="Kametani, Y., Miyamoto, A., Tsuda, B., &amp; Tokuda, Y. (2015). B Cell Epitope-Based Vaccination Therapy. Antibodies, 4(3), 225–239." />
<meta name="citation_reference" content="Yasmin, T., &amp; Nabi, A. H. M. (2016). B and T Cell Epitope-Based Peptides Predicted from Evolutionarily Conserved and Whole Protein Sequences of Ebola Virus as Vaccine Targets. Scandinavian journal of immunology, 83(5), 321–337." />
<meta name="citation_reference" content="Edgar RC. MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res. 2004;32(5):1792–7." />
<meta name="citation_reference" content="Guindon S, Dufayard JF, Lefort V, Anisimova M, Hordijk W, Gascuel O. New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0. Systematic biology. 2010 May 1;59(3):307–21." />
<meta name="citation_reference" content="Doytchinova, I.A. and Flower, D.R., 2007. VaxiJen: a server for prediction of protective antigens, tumour antigens and subunit vaccines. BMC bioinformatics, 8(1), p.4." />
<meta name="citation_reference" content="Fieser TM, John A, Tainer H, et al. Influence of protein flexibility and peptide conformation on reactivity of monoclonal anti-peptide antibodies with a protein a-helix. Proc Natl Acad Sci. 1987; 84(23):8568–72" />
<meta name="citation_reference" content="Kolaskar AS, Tongaonkar PC. A semi-empirical method for prediction of antigenic determinants on protein antigens. FEBS Lett. 1990; 276(1–2):172–4." />
<meta name="citation_reference" content="Emini EA, Hughes JV, Perlow DS, et al. Induction of hepatitis A virus-neutralizing antibody by a virus-specific synthetic peptide. J Virol. 1985; 55(3):836–9." />
<meta name="citation_reference" content="Parker JM, Guo D, Hodges RS. New hydrophilicity scale derived from high-performance liquid chromatography peptide retention data: correlation of predicted surface residues with antigenicity and X-ray-derived accessible sites. Biochemistry. 1986; 25(19):5425–32." />
<meta name="citation_reference" content="Karplus PA, Schulz GE. Prediction of chain flexibility in proteins. Naturwissenschaften. 1985; 72:212–3." />
<meta name="citation_reference" content="Larsen, J.E., Lund, O. and Nielsen, M., 2006. Improved method for predicting linear B-cell epitopes. Immunome research, 2(1), p.2." />
<meta name="citation_reference" content="Chou PY, Fasman GD. Prediction of the secondary structure of proteins from their amino acid sequence. Adv Enzymol Relat Areas Mol Biol. 1978; 47:45–148." />
<meta name="citation_reference" content="Nielsen M, Lundegaard C, Worning P, Lauemoller SL, Lamberth K, Buus S, Brunak S, Lund O. 2003. Reliable prediction of T-cell epitopes using neural networks with novel sequence representations. Protein Sci 12:1007–1017." />
<meta name="citation_reference" content="Peters B, Sette A. 2005. Generating quantitative models describing the sequence specificity of biological processes with the stabilized matrix method. BMC Bioinformatics 6:132." />
<meta name="citation_reference" content="Vita R, Overton JA, Greenbaum JA, Ponomarenko J, Clark JD, Cantrell JR, Wheeler DK, Gabbard JL, Hix D, Sette A, Peters B. The immune epitope database (IEDB) 3.0. Nucleic Acids Res. 2014 Oct 9. pii: gku938. [Epub ahead of print] PubMed PMID: 25300482." />
<meta name="citation_reference" content="Tenzer S, Peters B, Bulik S, Schoor O, Lemmel C, Schatz MM, Kloetzel PM, Rammensee HG, Schild H, Holzhutter HG. 2005. Modeling the MHC class I pathway by combining predictions of proteasomal cleavage, TAP transport and MHC class I binding. Cell Mol Life Sci 62:1025–1037." />
<meta name="citation_reference" content="Larsen, M.V., Lundegaard, C., Lamberth, K., Buus, S., Lund, O. and Nielsen, M., 2007. Large-scale validation of methods for cytotoxic T-lymphocyte epitope prediction. BMC bioinformatics, 8(1), p.424." />
<meta name="citation_reference" content="Bui HH, Sidney J, Dinh K, Southwood S, Newman MJ, Sette A. Predicting population coverage of T-cell epitope-based diagnostics and vaccines.BMC Bioinformatics. 2006; 17:153." />
<meta name="citation_reference" content="Thévenet, P., Shen, Y., Maupetit, J., Guyon, F., Derreumaux, P. and Tufféry, P., 2012. PEP-FOLD: an updated de novo structure prediction server for both linear and disulfide bonded cyclic peptides. Nucleic acids research, 40(W1), pp.W288–W293." />
<meta name="citation_reference" content="Shen, Y., Maupetty, J., Derreumaux, P. and Tufféry, P., 2014. Improved PEP-FOLD approach for peptide and miniprotein structure prediction. Journal of chemical theory and computation, 10(10), pp.4745–4758." />
<meta name="citation_reference" content="Kozakov, D., Hall, D.R., Xia, B., Porter, K.A., Padhorny, D., Yueh, C., Beglov, D. and Vajda, S., 2017. The ClusPro web server for protein-protein docking. nature protocols, 12(2), pp.255–278." />
<meta name="citation_reference" content="Kozakov D, Brenke R, Comeau SR, Vajda S. PIPER: an FFT-based protein docking program with pairwise potentials. Proteins: Structure, Function, and Bioinformatics. 2006 Nov 1;65(2):392–406." />
<meta name="twitter:title" content="Strategies for vaccine design for corona virus using Immunoinformatics techniques" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The cutting-edge technology vaccinomics is the combination of two topics immunogenetics and immunogenomics with the knowledge of systems biology and immune profiling for designing vaccine against infectious disease. In our present study, an epitope-based peptide vaccine against nonstructural protein 4 of beta coronavirus, using a combination of B cell and T cell epitope predictions, followed by molecular docking methods are performed. Here, protein sequences of homologous nonstructural protein 4 of beta coronavirus are collected and conserved regions present in them are investigated via phylogenetic study to determine the most immunogenic part of protein. From the identified region of the target protein, the peptide sequence IRNTTNPSAR from the region ranging from 38-47 and the sequence PTDTYTSVYLGKFRG from the positions of 76-90 are considered as the most potential B cell and T cell epitopes respectively. Furthermore, this predicted T cell epitopes PTDTYTSVY and PTDTYTSVYLGKFRG interacted with MHC allelic proteins HLA-A*01:01 and HLA-DRB5*01:01 respectively with the low IC50 values. These epitopes are perfectly fitted into the epitope binding grooves of alpha helix of MHC I molecule and MHC II molecule with binding energy scores −725.0 Kcal/mole and −786.0 Kcal/mole respectively, showing stability in MHC molecules binding. This MHC restricted epitope PTDTYTSVY also showed a good conservancy of 50.16% in world population coverage. This MHC I HLA-A*01:01 allele is present among 58.87% of Chinese population also. Therefore, the epitopes IRNTTNPSAR and PTDTYTSVYLGKFRG may be considered as potential peptides for peptide-based vaccine for coronavirus after further experimental study." />
<meta name="og-title" property="og:title" content="Strategies for vaccine design for corona virus using Immunoinformatics techniques" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.27.967422v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The cutting-edge technology vaccinomics is the combination of two topics immunogenetics and immunogenomics with the knowledge of systems biology and immune profiling for designing vaccine against infectious disease. In our present study, an epitope-based peptide vaccine against nonstructural protein 4 of beta coronavirus, using a combination of B cell and T cell epitope predictions, followed by molecular docking methods are performed. Here, protein sequences of homologous nonstructural protein 4 of beta coronavirus are collected and conserved regions present in them are investigated via phylogenetic study to determine the most immunogenic part of protein. From the identified region of the target protein, the peptide sequence IRNTTNPSAR from the region ranging from 38-47 and the sequence PTDTYTSVYLGKFRG from the positions of 76-90 are considered as the most potential B cell and T cell epitopes respectively. Furthermore, this predicted T cell epitopes PTDTYTSVY and PTDTYTSVYLGKFRG interacted with MHC allelic proteins HLA-A*01:01 and HLA-DRB5*01:01 respectively with the low IC50 values. These epitopes are perfectly fitted into the epitope binding grooves of alpha helix of MHC I molecule and MHC II molecule with binding energy scores −725.0 Kcal/mole and −786.0 Kcal/mole respectively, showing stability in MHC molecules binding. This MHC restricted epitope PTDTYTSVY also showed a good conservancy of 50.16% in world population coverage. This MHC I HLA-A*01:01 allele is present among 58.87% of Chinese population also. Therefore, the epitopes IRNTTNPSAR and PTDTYTSVYLGKFRG may be considered as potential peptides for peptide-based vaccine for coronavirus after further experimental study." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-02" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.02.27.967422v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.02.27.967422v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.02.27.967422v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.02.27.967422v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1177200" />
    <title>Strategies for vaccine design for corona virus using Immunoinformatics techniques | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"lkysDOulm052lSt2bCPXvNUSRSNlTMao_DPB7a9qITM","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1177200","apath":"\/biorxiv\/early\/2020\/03\/02\/2020.02.27.967422.atom","pisa":"biorxiv;2020.02.27.967422v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.02.27.967422v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.02.27.967422v1","title":"Strategies for vaccine design for corona virus using Immunoinformatics techniques","identifier":"node\/1177200"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UCxB5wa5SO9hLmdCzRzdSGf20OpU8jSihaK7tP+UIKznLBCuLcbfr1JpoofeSTbDvme59vJyGaQdMECGhN72POfYqiTisaY\\\\\\\/6\\\\\\\/mGAQYYwGjHFzJVojcm2KIWPq6QqiMuiagHgUQnVCbQEK8TSHH7q5DhRoVpI5yqii7BhLnjLJx9TNXUXz9s+Dn5Gl\\\\\\\/6M8zFe\\\\\\\/0zseAeEHuda1SwFWOIiHCCRE80LilnkYcpf97zAMDQSPn4htnw3WLh1fpv\\\\\\\/TgotEeFjIudNFMcm2bTYrk9aLd\\\\\\\/ldRACwmnNGK6\\\\\\\/xv\\\\\\\/RqHy1kn\\\\\\\/WosASYxeGVOfLmZY6SbXvh8PdB7ZBG0dGbJZWfUK6hRM6eBzdEQ1cbxcPnrEeD9sODUBLZCHcFMjJvgrWMjb3kG2gKG+LiUlvcAbQwnA+C2SbhLUPH4dkKumsj7QS8J6C1wnUHCPRblywHilHybNxGMOPsEMaNz6M+O0W9mBsftmEzj1dkr3I3lbDw5YU\\\\\\\/4wkO8IyYfWyEJ2zUmxCRV4NPY4ZVulTfFDzldQOurAib3yiVKZ5tOl3tByuqilij8tiVfTzTNjkj9Tclt9StTaZAUL91RrVMlDVOivDY66cLfpxcnG87VKG8CBIjpP\\\\\\\/dnkeSaxiKynBukrdhocbeyjGTRwblgTk6L3NjHTHRSXMpBNTojlyWfxktfT6aLI7RC\\\\\\\/KSe+zRmQhivOf2udS8dvzilRt8LMmznxnrrMYqKim+YHvCY+z3AdZB2C+Z4okg1qDrEAkouiBxVJ\\\\\\\/wTwnepBOi1+XzxW75q1+CkSeqeyoS62PmTKXvKepHGIow16nRbYUB7jR\\\\\\\/6t7INlAj+o4velA0yOlwhLxoiSBxpxA+yEU2y7ZFtBFWMAobs41szzbKTBpEq2+mm\\\\\\\/RkEOBz1WxA\\\\\\\/eSIuDQlFcGOCQKtdWvGLpMBW5uLP+LPH7fSS8WmstxNsH7r7qvOCuWnrFv5O7B8nEsWtsZiu4GxpgRbxq65DY4hsMk71GmEzkk8r69Lw+QHyZ+gh4gow+1\\\\\\\/HW1+RVqldgUhrlgu+T3LxWW1alQSfiDB0LBE0qTddJNMMhWHEP6Kzq+YPJid0zJfIYVUHK1j77lwH0Y9KjVR5b8qfY\\\\\\\/jWbLBqX9oYpYhb2zWoxEw4eJuECoMuRDCeurfcas5\\\\\\\/vLGZAcEhlQwefJY5N9rjHjoGdANJSyFX3CmyrbONBXvaNx71cGiPl9XLgU9RiqxNbGOzz5DwHuvBSV\\\\\\\/mA6XVW9n4bgPUnhRvA0Jjc7pZfXbEqLwoSzIo5lf+JfMRd15DBr6yN2QLTa8f\\\\\\\/NHtj9CTZJEAEI9yeKh9bli\\\\\\\/bmGw\\\\\\\/wJVDK4j\\\\\\\/v2VClXWnutMZQGT8\\\u0022,\\\u0022iv\\\u0022:\\\u0022xzokUXXzSjWIH36K+yymiQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00220e7bf5d0b910e2a7032cf0d73ddd36f3b2e14536400b5237023ec36d6eb35557\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.02.27.967422v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1177200 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.02.27.967422v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Oaj3ZbqUREOtuJ2xpNxG-iOtjhbNImIndTh53dgiOeo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1177200" id="node1177200" data-pisa="biorxiv;2020.02.27.967422v1" data-pisa-master="biorxiv;2020.02.27.967422" data-apath="/biorxiv/early/2020/03/02/2020.02.27.967422.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Strategies for vaccine design for corona virus using Immunoinformatics techniques</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Anamika</span> <span class="nlm-surname">Basu</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-7365-3924" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Anasua</span> <span class="nlm-surname">Sarkar</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-1167-0774" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Ujjwal</span> <span class="nlm-surname">Maulik</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.02.27.967422 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1177200" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anamika Basu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Assistant Professor, Gurudas College</span>, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anamika%2BBasu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Basu%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnamika%2BBasu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anasua Sarkar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Computer Science and Engineering Department, Jadavpur University</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anasua%2BSarkar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sarkar%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnasua%2BSarkar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7365-3924" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Anasua Sarkar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ashru2006@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">ashru2006@hotmail.com</a>
<a href="mailto:anasua.sarkar@jadavpuruniversity.in" class="" data-icon-position="" data-hide-link-title="0">anasua.sarkar@jadavpuruniversity.in</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ujjwal Maulik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Computer Science and Engineering Department, Jadavpur University</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ujjwal%2BMaulik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maulik%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AUjjwal%2BMaulik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1167-0774" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ujjwal Maulik</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.02.27.967422v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1177200" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1177200/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.27.967422v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1177200" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1177200/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.27.967422v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1177200" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1177200/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.27.967422v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1177200" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1177200/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.02.27.967422v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1177200" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1177200/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-2">The cutting-edge technology vaccinomics is the combination of two topics immunogenetics and immunogenomics with the knowledge of systems biology and immune profiling for designing vaccine against infectious disease. In our present study, an epitope-based peptide vaccine against nonstructural protein 4 of beta coronavirus, using a combination of B cell and T cell epitope predictions, followed by molecular docking methods are performed. Here, protein sequences of homologous nonstructural protein 4 of beta coronavirus are collected and conserved regions present in them are investigated via phylogenetic study to determine the most immunogenic part of protein. From the identified region of the target protein, the peptide sequence IRNTTNPSAR from the region ranging from 38-47 and the sequence PTDTYTSVYLGKFRG from the positions of 76-90 are considered as the most potential B cell and T cell epitopes respectively. Furthermore, this predicted T cell epitopes PTDTYTSVY and PTDTYTSVYLGKFRG interacted with MHC allelic proteins HLA-A*01:01 and HLA-DRB5*01:01 respectively with the low IC<sub>50</sub> values. These epitopes are perfectly fitted into the epitope binding grooves of alpha helix of MHC I molecule and MHC II molecule with binding energy scores −725.0 Kcal/mole and −786.0 Kcal/mole respectively, showing stability in MHC molecules binding. This MHC restricted epitope PTDTYTSVY also showed a good conservancy of 50.16% in world population coverage. This MHC I HLA-A*01:01 allele is present among 58.87% of Chinese population also. Therefore, the epitopes IRNTTNPSAR and PTDTYTSVYLGKFRG may be considered as potential peptides for peptide-based vaccine for coronavirus after further experimental study.</p></div><div class="section" id="sec-1"><h2 class="">1. Introduction</h2><p id="p-3">According to World Health Organization, Coronaviruses (CoV) are a large family of RNA viruses that cause infection ranging from the common cold to more severe diseases such as Middle East Respiratory Syndrome (MERS-CoV) and Severe Acute Respiratory Syndrome (SARS-CoV). A novel coronavirus (nCoV) is a new strain that has not been previously identified in humans. 2019 Novel Coronavirus (2019-nCoV) is a coronavirus identified as the cause of an outbreak of respiratory illness first detected in Wuhan, China.</p><p id="p-4">Coronaviruses are zoonotic, meaning they are transmitted between animals and people. Detailed investigations found that SARS-CoV has been transmitted from civet cats to humans and MERS-CoV from camels to humans. Several known coronaviruses are circulating in animals that have not yet infected humans. Early on, many of the patients in the outbreak in Wuhan, China reportedly have some link to a large seafood and animal market, suggesting animal-to-person spread. However, a growing number of patients reportedly have not had exposure to animal markets, indicating person-to-person spread is occurring. The 2019-nCoV is spreading from person to person in China and limited spread among close contacts has been detected in some countries outside China. Common signs of infection include respiratory symptoms, fever, cough, shortness of breath and breathing difficulties. In more severe cases, infection can cause pneumonia, severe acute respiratory syndrome, kidney failure and even death. There is currently no vaccine to protect against 2019-nCoV.</p><p id="p-5">According to Lu et al, 2020 [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>], genome sequence of 2019-nCoV is closely related (with 88% identity) to two bat-derived severe acute respiratory syndrome (SARS)-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21 and genetically distinct from SARS-CoV. Two complete virus genomes (HKU-SZ-002a and HKU-SZ-005b) are sequenced from 2019-nCoV infected patients [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. HKU-SZ-002a and HKU-SZ-005b differ from each other by only two bases. One of them is a non-synonymous mutation at amino acid position 336 of non-structural protein 4 (Ser336 for HKU-SZ-002a; Leu336 for HKU-SZ-005b). The amino acid sequence of the N-terminal domain of Spike subunit 1 of this novel coronavirus is around 66% identical to those of the SARS-related coronaviruses, and the core domain of the receptor binding domain of this novel coronavirus has about 68% amino acid identity with those of the SARS-related coronavirus. But the protein sequence of the external subdomain region of receptor binding domain of Spike subunit 1 has only 39% identity, which might affect the choice of human receptor and therefore the biological activity of this virus.</p><p id="p-6">Coronaviruses encode large replicase polyproteins which are proteolytically processed by viral proteases to generate mature nonstructural proteins (nsps) that form the viral replication complex. Positive-strand RNA viruses, such as coronaviruses, can induce cellular membrane rearrangements during replication to form replication organelles which allows efficient viral RNA synthesis. Nonstructural Protein 4 alone induces membrane pairing in infectious bronchitis virus [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. Infection with coronavirus causes rearrangement in the host cell membrane to accumulate a replication and transcription complex in which replication of the viral genome and transcription of viral mRNA can occur. For coronaviruses, a major pathogenicity factor has now been identified with non-structural protein 1 in a murine model of coronavirus infection [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. Sakai et al in 2017 [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>] highlighted the role of nsp4 of SARS coronavirus in viral replication. It has been discovered that only a change in two amino acids in nsp4 protein sequence can abolish viral replication completely. So, to prevent coronavirus infection by designing vaccine, nonstructural protein 4 of beta coronavirus can be selected as target protein in this study.</p><p id="p-7">Prevention of viral diseases by vaccination purposes for controlled induction of protective immune responses against viral pathogens such as small pox, hepatitis etc. Attenuated viral vaccines can be produced by targeting essential pathogenicity factors. This study has implicated for the rational design of live attenuated coronavirus vaccines aiming to prevent coronavirus-induced diseases of human as well as animal origin, including the lethal severe acute respiratory syndrome.</p><p id="p-8">In this study nonstructural protein 4 has been analyzed in order to predict informative epitopes which will help for future vaccine designing. The knowledge of peptide vaccine is used to identify B-cell and T-cell epitopes which can induce specific immune responses [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>], [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. These designed epitopes will offer cost effective, high quality therapeutics against corona virus. By using online algorithms in immunoinformatics, potentially active immunogenic T and B cell epitopes for this viral protein has been identified and modelled to design possible peptide vaccine for coronavirus infection.</p></div><div class="section" id="sec-2"><h2 class="">2. Materials and methods</h2><div id="sec-3" class="subsection"><h3>2.1 Retrieval of Non-structural protein NS4 in Beta coronavirus HKU24</h3><p id="p-9">The protein family information of Coronavirus nonstructural protein NS4 containing 78 protein is retrieved from InterPro database (<a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR005603/">https://www.ebi.ac.uk/interpro/entry/InterPro/IPR005603/</a>). To find conserved region, retrieved sequences were aligned using Muscle tool 3.8.31 [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>] where <em>k</em>-mer clustering is used. In the next step a phylogenetic tree is constructed by the method known as progressive alignment. The evolutionary divergence analysis for all 78 virus proteins are completed by forming a phylogenetic tree using Phy ML 3.1/3.0 aLRT software [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. Here the phylogenetic tree was reconstructed using the maximum likelihood method. The default substitution model was selected assuming an estimated proportional of invariant sites (of 0.008) and 4 gamma-distributed rate categorized to account for rate heterogenicity across sites. The gamma shape parameter was estimated directly from the data (gamma = 1.788). Reliability for internal branch was assessed using the aLRT test (SH-like).</p><p id="p-10">Among these proteins non-structural protein NS4, with accession number A0A0A7UXD8, known as Beta coronavirus HKU24 (due to its Chinese origin) with length 136 is selected in FASTA format.</p></div><div id="sec-4" class="subsection"><h3>2.2 Protein antigenicity determination</h3><p id="p-11">Antigenicity of this protein is predicted by VaxiJen v 2.0, an online prediction server [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>].</p></div><div id="sec-5" class="subsection"><h3>2.3 Potential B cell epitope prediction</h3><p id="p-12">B cell epitopes are part of allergen which come in interaction with B lymphocytes to induce immune response. Among two types of B cell epitopes, linear type of B cell epitopes is predicted.</p><p id="p-13">Various physico-chemical properties e.g. hydrophilicity, flexibility, accessibility, turns, exposed surface, polarity and antigenic propensity of peptides chains have been estimated to identify the locations of linear epitopes of an antigenic protein [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. Thus, different tools from IEDB (<a href="http://www.iedb.org">www.iedb.org</a>), including the classical propensity scale methods such as Kolaskar and Tongaonkar antigenicity scale [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>], E mini surface accessibility prediction [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>], Parker hydrophilicity prediction [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>], Karplus and Schulz flexibilty prediction [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>], Bepipred linear epitope prediction [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>] and Chou and Fashman beta turn prediction tool [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>] are used to predict linear or continous B cell epitopes of nonstructural protein NS4 protein in coronavirus. With the help of graphical findings and prediction scores the most probable B cell epitope of that antigenic protein has been identified. BepiPred prediction method is a combination of Hidden Markov model and propensity scale method to predict score and identification of B cell epitopes of antigenic protein [<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>].</p></div><div id="sec-6" class="subsection"><h3>2.4 Potential T-cell epitope prediction</h3><div id="sec-7" class="subsection"><h4>2.4.1 MHC I T cell epitope prediction</h4><p id="p-14">Linear T-cell epitopes for MHC-I binding for nonstructural protein NS4 protein in coronavirus are recognized by consensus methods using various methods such as Artificial neural network (ANN) [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>], Stabilized matrix method (SMM) [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>] and Scoring Matrices Derived from Combinatorial Peptide Libraries (Comblib) from tools for MHC -I binding prediction methods of Immune Epitope Database (IEDB) (<a href="http://www.iedb.org">www.iedb.org</a>) [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]. This server-based method forecasts the MHC class I binding predication to 26 MHC supertypes as percentile rank. SMM algorithm of MHC-1 binding, transporter of antigenic peptides (TAP) transport efficiency and proteasomal cleavage efficiency are also considered to determine the IC<sub>50</sub> values for processing prediction of epitopes by MHC-I molecules [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. On the basis of low IC<sub>50</sub> values, 5 best epitopes bind with specific MHC-1 molecules are elected for further evaluation.</p></div><div id="sec-8" class="subsection"><h4>2.4.2 MHC II T cell epitope prediction</h4><p id="p-15">CD4+ T-cell receptor responses against concerned nonstructural protein NS4 protein are done by using Peptide binding to MHC class II molecules software using MHC II binding prediction tool in IEDB analysis resource, including a consensus approach which combines NN-align, SMM-align [<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>] and Combinatorial library methods. For this prediction we chose thirty HLA class II alleles from the reference set. For the predicted T-cell epitopes with low percentile rank are identified and their IC<sub>50</sub> values for respective alleles are determined by SMM-align method [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>].</p></div></div><div id="sec-9" class="subsection"><h3>2.5 Analysis of population coverage</h3><p id="p-16">Population coverage for identified T cell epitopes is assessed for world, as well as China and Indian population with the help of IEDB population coverage calculation tool [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>]. This tool calculates the fraction of individuals predicted to reply to a specified set of epitopes with recognized MHC restrictions. This calculation is finished considering the HLA genotypic frequencies assuming non-linkage disequilibrium between HLA loci.</p></div><div id="sec-10" class="subsection"><h3>2.6 Docking study of T cell epitopes</h3><p id="p-17">For docking studies, the T cell epitope PTDTYTSVY (MHC I restricted) and PTDTYTSVYLGKFRG (MHC II restricted) are selected and subjected to PEP-FOLD server [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>], [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>] for 3D structure formation. To identify the molecular interactions with specific HLA protein for respective epitopes, docking studies are performed with ClusPro 2.2 web server [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>]. Cluster scores for lowest binding energy prediction are calculated using the formula-E = 0.40E_{rep} + −0.40E_{att} + 600E_{elec} + 1.00E_{DARS}. Here, repulsive, attractive, electrostatic as well as interactions extracted from the decoys as the reference state, are considered for structure-based pairwise potential calculation in epitope-MHC molecule docking [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. Modified PDB ID 4NQX for HLA-A*01:01 is used as allelic protein for docking study of T cell epitope with MHC I restricted. Similarly, 3D structure (PDB ID 1FV1) for HLA-DRB5*01:01 MHC molecule is used as receptor molecule for T cell epitope docking analysis with MHC II restriction.</p></div></div><div class="section" id="sec-11"><h2 class="">3 Results</h2><div id="sec-12" class="subsection"><h3>3.1 Retrieval of Non-structural protein NS4 in Beta coronavirus HKU24</h3><p id="p-18">From InterPro database, 78 homologous protein sequences are recovered for nonstructural protein NS4 protein for coronavirus as shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1180035/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1180035/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <p id="p-19" class="first-child">List of 78 nonstructural protein NS4 protein for coronavirus</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">A phylogenetic tree illustrating the evolutionary relationship among the 78 homologous non-structural protein 4 of coronavirus is depicted in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1a</a> and the relevant enlarged version is shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 1b</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A phylogenetic tree of non-structural protein NS4" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1038646409" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A phylogenetic tree of non-structural protein NS4&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1a" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F1.medium.gif" width="440" height="377"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1a" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F1.medium.gif" width="440" height="377"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1a" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1a</span> <p id="p-21" class="first-child">A phylogenetic tree of non-structural protein NS4</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="An enlarged version of phylogenetic tree of non-structural protein NS4" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1038646409" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;An enlarged version of phylogenetic tree of non-structural protein NS4&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1b" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F2.medium.gif" width="440" height="219"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1b" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F2.medium.gif" width="440" height="219"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1b" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 1b</span> <p id="p-22" class="first-child">An enlarged version of phylogenetic tree of non-structural protein NS4</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Phylogenetic tree shows that Non-structural protein NS4 of Beta coronavirus HKU24 (AOAO7UXD8_9BETC), rat coronavirus (NS4_CVRSD), murine coronavirus (NS4_CVMJH, NS$_CVMS and NSA_CVMA5) and ORF4 protein of murine hepatitis virus (Q9J3EC) are very similar in their structures (identity vary 42.3% to 38.3%). All these five protein sequences are aligned to identify conserved sequences with varying length using Muscle tool 3.8.31[<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>] is shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 2</a>. Conserved regions are highlighted.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multiple sequence alignment five Non-structural protein NS4" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1038646409" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Multiple sequence alignment five Non-structural protein NS4&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F3.medium.gif" width="440" height="173"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F3.medium.gif" width="440" height="173"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Multiple sequence alignment five Non-structural protein NS4</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>3.2 Potential T-cell epitope prediction</h3><div id="sec-14" class="subsection"><h4>3.2.1 MHC I T cell epitope prediction</h4></div><div id="sec-15" class="subsection"><h4>3.2.2 MHC II T cell epitope prediction</h4><p id="p-25">Among the five peptide sequences, <sup>76</sup> PTDTYTSVY <sup>84</sup> peptide sequence (MHC I restricted) and <sup>76</sup> PTDTYTSVYLGKFRG <sup>90</sup> (MHC II restricted) are selected as most probable T cell epitope for the antigenic protein present in non-structural protein NS4 of beta coronavirus HKU24 on the basis of its interactions with highest number of alleles (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">3</a>). The IC<sub>50</sub> values of these peptides are least for the MHCI HLA-A*01:01and MHC II allele HLA-DRB5*01:01 respectively.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1180034/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1180034/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1180034" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <p id="p-26" class="first-child">CD8+ T cell epitopes</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1180042/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1180042/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1180042" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <p id="p-27" class="first-child">CD4+ t cell epitopes</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-16" class="subsection"><h3>3.3 Analysis of population coverage</h3><p id="p-28">IEDB population coverage tool [<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>] is used to calculate the population coverage of the predicted epitopes. The result for class I MHC restriction for the China, whole world and Indian population with the selected MHC I alleles is shown in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1180032/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1180032/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1180032" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <p id="p-29" class="first-child">Population coverage for predicted T cell epitopes</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>3.4 Potential B cell epitope prediction</h3><p id="p-30">Characteristic features of the B cell epitope contain flexibility, hydrophilicity, surface accessibility and beta-turn prediction. Prediction scores of Chou and Fasman beta turn [<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>], Emini surface accessibility [<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>], Kolaskar and Tangaonkar antigenicity [<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>] and Parker hydrophilicity [<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>] are plotted (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 3</a>). Predicted linear B cell epitopes and the accessibility, hydrophilicity, flexibility and beta-turn prediction score for each residue are summarized in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction scores for each amino acid residues of the virus protein" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1038646409" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Prediction scores for each amino acid residues of the virus protein&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F4.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F4.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">Prediction scores for each amino acid residues of the virus protein</p><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1180040/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1180040/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1180040" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <p id="p-32" class="first-child">Surface accessibility, hydrophilicity, flexibility, beta turn and antigenicity prediction score for each residue of B cell epitope.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Prediction scores of Emini surface accessibility, Parker hydrophilicity, Karplus and Schulz
flexibility, Chou and Fashman beta turn for each residue of peptides <sup>38</sup>IRNTTNPSAR<sup>47</sup> and <sup>56</sup>RTPPTSVVTSGDDGRV<sup>71</sup> reveals that these short stretches of antigenic protein can act as linear B cell epitopes.</p></div><div id="sec-18" class="subsection"><h3>3.5 Docking study of T cell epitopes</h3><p id="p-34">The T cell epitope <sup>76</sup> PTDTYTSVYLGKFRG <sup>90</sup> (MHC II restricted) is selected on the basis of its interactions with large number of alleles and lowest IC<sub>50</sub> value with HLA-DRB5*1:01 MHC II allele. Similarly, <sup>76</sup> PTDTYTSVY <sup>84</sup>, restricted with MHC I allelic protein HLA-A*01:01 is designated as most probable T cell epitope which is also present in conserved region of non-structural protein NS4 of coronavirus. Docking studies are performed with these two epitopes with human HLA-DRB5*01:01MHC II molecule and MHC I allelic protein HLA-A*01:01 respectively.</p><p id="p-35">Docking study of PTDTYTSVYLGKFRG epitope with HLA-DRB5*01:01, shows lowest binding energy −786.0 Kcal/mole, shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 4</a>. Docking structure is stabilized by H bonds, are shown in <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bound structure of T cell epitope with MHC II" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1038646409" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bound structure of T cell epitope with MHC II&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F5.medium.gif" width="440" height="350"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F5.medium.gif" width="440" height="350"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">Bound structure of T cell epitope with MHC II</p><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1180038/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1180038/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1180038" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <p id="p-37" class="first-child">Details of Hydrogen bonding between T cell epitope with MHC II molecule</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Interaction between T cell epitope PTDTYTSVY MHC I allelic protein HLA-A*01:01 with binding energy −725.0 Kcal/mole is shown in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 5</a>.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bound structure of T cell epitope with MHC I molecule" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1038646409" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bound structure of T cell epitope with MHC I molecule&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F6.medium.gif" width="440" height="215"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F6.medium.gif" width="440" height="215"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.27.967422/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-39" class="first-child">Bound structure of T cell epitope with MHC I molecule</p><div class="sb-div caption-clear"></div></div></div><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1180037/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1180037/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1180037" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <p id="p-40" class="first-child">Details of Hydrogen bonding between T cell epitope with MHC I molecule</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-19"><h2 class="">4 Discussions</h2><p id="p-41">At present in China and whole world, considering the emergency situation due to corona virus infection, rapid development in vaccine design is the most argent step to prevent pandemics. Because by using vaccine the mortality rate due to coronavirus can be controlled. This technique is applied successfully for smallpox virus, polio virus etc. Though for some other common viruses such as dengue virus, hepatitis C virus, human immunodeficiency virus and coronavirus, vaccine has not been invented till now. Due to lack of definite information about growth, replication and pathogenesis of these viruses. Therefore, computational techniques are used for epitope mapping, which is the preliminary step for vaccine design to prevent coronavirus infection. This study integrates several immunoinformatics and molecular docking methods to recognize potential epitopes of non-structural protein NS4 in coronavirus.</p><p id="p-42">At first, all seventy-nine non-structural protein 4 from different coronaviruses are retrieved from InterPro database and aligned to detect conserved sequences present in them. A phylogenetic investigation reveals a closed evolutionary relationship among these homologous proteins. Non-structural protein 4 is considered as our protein of interest for vaccine design assuming its role in viral replication during coronavirus infection and considering its antigenic nature in human. Not only that, this selected protein has structural similarity with non-structural protein 4 present in rat, murine coronavirus containing various strains. Therefore, the proposed peptide-based vaccine might be effective to prevent coronavirus infection not only in human, but also in rat, murine etc. Here, computational method for vaccine design is totally safe, rapid and cost effective at this emergency situation, caused due to coronavirus infection.</p><p id="p-43">Five potent T cell epitope having potentiality for binding with MHC molecules are predicted. For MHC I and MHC II molecules both 9 mer and 15 mer peptide structures are projected from IEDB recommended prediction method and modeled by peptide modeling algorithm. The percentile rank and IC<sub>50</sub> values with SMM/ANN method covering all MHC class I supertypes are also analyzed. The five most effective epitopes are presented in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> along with their IC<sub>50</sub> values.</p><p id="p-44">For MHC I binding prediction scores, the peptide with the lowest percentile rank and IC<sub>50</sub> value, is selected for their highest affinity for that interacting MHC I allele. The T cell epitope <sup>76</sup> PTDTYTSVY <sup>84</sup> is considered on the basis of its interaction with large number of alleles and lowest IC<sub>50</sub> value with HLA-A*01:01 MHC I allele. For this epitope, MHC I processing score with that specific allele comprises proteasome score 1.25, TAP score 1.02, MHC IC<sub>50</sub> value is 6.7 nm. This means that this specific epitope has high affinity to MHC I molecule during antigen presentation. Moreover that, this epitope when interacting with selective MHC I allele shows the highest population coverage not only for Chinese population, but also has higher population coverage compared to other epitopes, for Indian and whole world population. Henceforth, this epitope is considered as epitope of choice for CD8 <sup>+</sup> T cells.</p><p id="p-45">Likewise, presence of wider peptide binding groove in MHC II molecule than that of MHC I, 15 mer epitopes are investigated by smm/nn/sturnilo method along with their IC<sub>50</sub> values and are listed in <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>. For MHC II binding prediction method, a 15 mer T cell epitope sequence <sup>76</sup> PTDTYTSVYLGKFRG <sup>90</sup> of non-structural protein 4, displays a percentile rank 22.0 with IC<sub>50</sub> value 22.0 at the time of interaction with HLA-DRB5*01:01MHC II allele. This result approves that this peptide can be selected as T cell epitope with MHC II restriction for our protein of interest. Moreover, this T cell epitope sequence is well conserved among the non-structural protein 4 present in different corona viruses.</p><p id="p-46">B cell epitope identification method, the prediction scores for Emini surface accessibility, Parker hydrophilicity, Chou and Fashman beta turn and Karplus and Schulz flexibility for each residue of peptides IRNTTNPSAR, starting from sequence 38 position and ending at 47 position of that viral antigenic protein, predicts that this is the most potent B cell epitope present in it. Additionally, Kolarskar and Tangaonkar antigenicity prediction values, confirm our estimation.</p><p id="p-47">The predicted T cell epitopes are validated by using molecular docking study. These two epitopes are preferably fitted in the epitope binding grooves of the two MHC protein molecules with highly negative binding energies and stabilized by H-bonds.</p><p id="p-48">An important factor in vaccine design is the distribution of selected HLA allelic protein. This distribution varies among the human population according to the population in different geographic regions of the world. Our predicted T cell epitope, <sup>76</sup> PTDTYTSVY <sup>84</sup>, bound with MHC I HLA-A*01:01 allele, which is present among 58.87% and 45.77% of Chinese and Indian populations respectively and 50.16% of world populations. So, it may be concluded that the predicted T cell epitope must be specifically restricted with the predominant MHC molecule, which is present in target population in India and China, as well as in whole world against coronavirus.</p></div><div class="section" id="sec-20"><h2 class="">5 Conclusion</h2><p id="p-49">Though in general most peptide-based vaccines are developed considering B cell epitopes, in our present study both B cell and T cell epitopes, present in non-structural protein 4, are considered for vaccine design against coronavirus. These two T cell epitopes can stimulate immunogenic response after administration inside the human body. This immunological reaction can prevent coronavirus infection in human as well as rat and murine, when they come in contact with this virus in future. Since these epitopes are well restricted with MHC molecules and at the same time, they are almost conserved among other homologous proteins. These proteins include non-structural protein 4, obtained from novel-coronavirus infected patient in China. Thus, these epitopes can be proceeded for further experimental verification during vaccine designing against coronavirus infection.</p></div><div class="section" id="sec-21"><h2 class="">Funding</h2><p id="p-50">This work is not supported by any funding.</p></div><div class="section" id="sec-22"><h2 class="">Conflict of interest</h2><p id="p-51">The authors report no conflicts of interest in this work.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niu</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">H.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Bi</span>,  <span class="cit-name-given-names">Y.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. <abbr class="cit-jnl-abbrev">The Lancet</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chan</span>,  <span class="cit-name-given-names">J. F. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kok</span>,  <span class="cit-name-given-names">K. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chu</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">To</span>,  <span class="cit-name-given-names">K. K. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Yuen</span>,  <span class="cit-name-given-names">K. Y.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Genomic characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with atypical pneumonia after visiting Wuhan</span>. <abbr class="cit-jnl-abbrev">Emerging Microbes &amp; Infections</abbr>, <span class="cit-vol">9</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">221</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2BMicrobes%2B%2526%2BInfections%26rft.volume%253D9%26rft.spage%253D221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Doyle</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neuman</span>,  <span class="cit-name-given-names">B. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simpson</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hawes</span>,  <span class="cit-name-given-names">P. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mantell</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verkade</span>,  <span class="cit-name-given-names">P.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Maier</span>,  <span class="cit-name-given-names">H. J.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Infectious bronchitis virus nonstructural protein 4 alone induces membrane pairing</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr>, <span class="cit-vol">10</span>(<span class="cit-issue">9</span>), <span class="cit-fpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D10%26rft.spage%253D477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Züst</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cervantes-Barragán</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuri</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blakqori</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weber</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ludewig</span>,  <span class="cit-name-given-names">B.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Thiel</span>,  <span class="cit-name-given-names">V.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Coronavirus non-structural protein 1 is a major pathogenicity factor: implications for the rational design of coronavirus vaccines</span>. <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr>, <span class="cit-vol">3</span>(<span class="cit-issue">8</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.5" data-doi="10.1016/j.virol.2017.07.019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sakai</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kawachi</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Terada</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Omori</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matsuura</span>,  <span class="cit-name-given-names">Y.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Kamitani</span>,  <span class="cit-name-given-names">W.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Two-amino acids change in the nsp4 of SARS coronavirus abolishes viral replication</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>, <span class="cit-vol">510</span>, <span class="cit-fpage">165</span>–<span class="cit-lpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D510%26rft.spage%253D165%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2017.07.019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2017.07.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kametani</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miyamoto</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsuda</span>,  <span class="cit-name-given-names">B.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Tokuda</span>,  <span class="cit-name-given-names">Y.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">B Cell Epitope-Based Vaccination Therapy</span>. <abbr class="cit-jnl-abbrev">Antibodies</abbr>, <span class="cit-vol">4</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">225</span>–<span class="cit-lpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntibodies%26rft.volume%253D4%26rft.spage%253D225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.7" data-doi="10.1111/sji.12425"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yasmin</span>,  <span class="cit-name-given-names">T.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Nabi</span>,  <span class="cit-name-given-names">A. H. M.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">B and T Cell Epitope-Based Peptides Predicted from Evolutionarily Conserved and Whole Protein Sequences of Ebola Virus as Vaccine Targets</span>. <abbr class="cit-jnl-abbrev">Scandinavian journal of immunology</abbr>, <span class="cit-vol">83</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">321</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScandinavian%2Bjournal%2Bof%2Bimmunology%26rft.volume%253D83%26rft.spage%253D321%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fsji.12425%26rft_id%253Dinfo%253Apmid%252F26939891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/sji.12425&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26939891&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.27.967422.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.8" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">RC.</span></span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2004</span>;<span class="cit-vol">32</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">1792</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034147&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.27.967422.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220487200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dufayard</span>  <span class="cit-name-given-names">JF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lefort</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anisimova</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hordijk</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span> <span class="cit-article-title">New algorithms and methods to estimate maximum-likelihood phylogenies</span>: <abbr class="cit-jnl-abbrev">assessing the performance of PhyML 3.0. Systematic biology</abbr>. <span class="cit-pub-date">2010</span> <span class="cit-month">May</span> <span class="cit-day">1</span>;<span class="cit-vol">59</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">307</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253Dassessing%2Bthe%2Bperformance%2Bof%2BPhyML%2B3.0.%2BSystematic%2Bbiology%26rft.volume%253D59%26rft.spage%253D307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.10" data-doi="10.1186/1471-2105-8-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Doytchinova</span>,  <span class="cit-name-given-names">I.A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Flower</span>,  <span class="cit-name-given-names">D.R.</span></span>, <span class="cit-pub-date">2007</span>. <span class="cit-article-title">VaxiJen: a server for prediction of protective antigens, tumour antigens and subunit vaccines</span>. <abbr class="cit-jnl-abbrev">BMC bioinformatics</abbr>, <span class="cit-vol">8</span>(<span class="cit-issue">1</span>), p.<span class="cit-fpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DDoytchinova%26rft.auinit1%253DI.%2BA.%26rft.volume%253D8%26rft.spage%253D4%26rft.epage%253D4%26rft.atitle%253DVaxiJen%253A%2Ba%2Bserver%2Bfor%2Bprediction%2Bof%2Bprotective%2Bantigens%252C%2Btumour%2Bantigens%2Band%2Bsubunit%2Bvaccines.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-4%26rft_id%253Dinfo%253Apmid%252F17207271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17207271&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.27.967422.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.11" data-doi="10.1073/pnas.84.23.8568"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fieser</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">John</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tainer</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Influence of protein flexibility and peptide conformation on reactivity of monoclonal anti-peptide antibodies with a protein a-helix</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci</abbr>. <span class="cit-pub-date">1987</span>; <span class="cit-vol">84</span>(<span class="cit-issue">23</span>):<span class="cit-fpage">8568</span>–<span class="cit-lpage">72</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DFieser%26rft.auinit1%253DT.%2BM.%26rft.volume%253D84%26rft.issue%253D23%26rft.spage%253D8568%26rft.epage%253D8572%26rft.atitle%253DInfluence%2Bof%2Bprotein%2Bflexibility%2Band%2Bpeptide%2Bconformation%2Bon%2Breactivity%2Bof%2Bmonoclonal%2Banti-peptide%2Bantibodies%2Bwith%2Ba%2Bprotein%2Balpha-helix.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.84.23.8568%26rft_id%253Dinfo%253Apmid%252F2446325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODQvMjMvODU2OCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzAyLzIwMjAuMDIuMjcuOTY3NDIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.12" data-doi="10.1016/0014-5793(90)80535-Q"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kolaskar</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tongaonkar</span>  <span class="cit-name-given-names">PC.</span></span> <span class="cit-article-title">A semi-empirical method for prediction of antigenic determinants on protein antigens</span>. <abbr class="cit-jnl-abbrev">FEBS Lett</abbr>. <span class="cit-pub-date">1990</span>; <span class="cit-vol">276</span>(<span class="cit-issue">1–2</span>):<span class="cit-fpage">172</span>–<span class="cit-lpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DKolaskar%26rft.auinit1%253DA.%2BS.%26rft.volume%253D276%26rft.issue%253D1-2%26rft.spage%253D172%26rft.epage%253D174%26rft.atitle%253DA%2Bsemi-empirical%2Bmethod%2Bfor%2Bprediction%2Bof%2Bantigenic%2Bdeterminants%2Bon%2Bprotein%2Bantigens.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0014-5793%252890%252980535-Q%26rft_id%253Dinfo%253Apmid%252F1702393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0014-5793(90)80535-Q&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1702393&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.27.967422.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990EP34200044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Emini</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">JV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perlow</span>  <span class="cit-name-given-names">DS</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Induction of hepatitis A virus-neutralizing antibody by a virus-specific synthetic peptide</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">1985</span>; <span class="cit-vol">55</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">836</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DEmini%26rft.auinit1%253DE%2BA%26rft.volume%253D55%26rft.issue%253D3%26rft.spage%253D836%26rft.epage%253D839%26rft.atitle%253DInduction%2Bof%2Bhepatitis%2BA%2Bvirus-neutralizing%2Bantibody%2Bby%2Ba%2Bvirus-specific%2Bsynthetic%2Bpeptide.%26rft_id%253Dinfo%253Apmid%252F2991600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjU1LzMvODM2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMDIvMjAyMC4wMi4yNy45Njc0MjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.14" data-doi="10.1021/bi00367a013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hodges</span>  <span class="cit-name-given-names">RS.</span></span> <span class="cit-article-title">New hydrophilicity scale derived from high-performance liquid chromatography peptide retention data: correlation of predicted surface residues with antigenicity and X-ray-derived accessible sites</span>. <abbr class="cit-jnl-abbrev">Biochemistry</abbr>. <span class="cit-pub-date">1986</span>; <span class="cit-vol">25</span>(<span class="cit-issue">19</span>):<span class="cit-fpage">5425</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D25%26rft.spage%253D5425%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi00367a013%26rft_id%253Dinfo%253Apmid%252F2430611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi00367a013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2430611&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.27.967422.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986E242300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.15" data-doi="10.1007/BF01195768"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Karplus</span>  <span class="cit-name-given-names">PA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schulz</span>  <span class="cit-name-given-names">GE.</span></span> <span class="cit-article-title">Prediction of chain flexibility in proteins</span>. <abbr class="cit-jnl-abbrev">Naturwissenschaften</abbr>. <span class="cit-pub-date">1985</span>; <span class="cit-vol">72</span>:<span class="cit-fpage">212</span>–<span class="cit-lpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNaturwissenschaften%26rft.volume%253D72%26rft.spage%253D212%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01195768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01195768&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1985AJQ6200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.16" data-doi="10.1186/1745-7580-2-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Larsen</span>,  <span class="cit-name-given-names">J.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lund</span>,  <span class="cit-name-given-names">O.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Improved method for predicting linear B-cell epitopes</span>. <abbr class="cit-jnl-abbrev">Immunome research</abbr>, <span class="cit-vol">2</span>(<span class="cit-issue">1</span>), p.<span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunome%2Bresearch%26rft.volume%253D2%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1745-7580-2-2%26rft_id%253Dinfo%253Apmid%252F16635264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1745-7580-2-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16635264&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.27.967422.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.17" data-doi="10.1002/9780470122921.ch2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chou</span>  <span class="cit-name-given-names">PY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fasman</span>  <span class="cit-name-given-names">GD.</span></span> <span class="cit-article-title">Prediction of the secondary structure of proteins from their amino acid sequence</span>. <abbr class="cit-jnl-abbrev">Adv Enzymol Relat Areas Mol Biol</abbr>. <span class="cit-pub-date">1978</span>; <span class="cit-vol">47</span>:<span class="cit-fpage">45</span>–<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Benzymology%2Band%2Brelated%2Bareas%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DAdv%2BEnzymol%2BRelat%2BAreas%2BMol%2BBiol%26rft.aulast%253DChou%26rft.auinit1%253DP.%2BY.%26rft.volume%253D47%26rft.spage%253D45%26rft.epage%253D148%26rft.atitle%253DPrediction%2Bof%2Bthe%2Bsecondary%2Bstructure%2Bof%2Bproteins%2Bfrom%2Btheir%2Bamino%2Bacid%2Bsequence.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F9780470122921.ch2%26rft_id%253Dinfo%253Apmid%252F364941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/9780470122921.ch2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=364941&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.27.967422.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.18" data-doi="10.1110/ps.0239403"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lundegaard</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Worning</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lauemoller</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lamberth</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brunak</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">O.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Reliable prediction of T-cell epitopes using neural networks with novel sequence representations</span>. <abbr class="cit-jnl-abbrev">Protein Sci</abbr> <span class="cit-vol">12</span>:<span class="cit-fpage">1007</span>–<span class="cit-lpage">1017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%26rft.stitle%253DProtein%2BSci.%26rft.aulast%253DNielsen%26rft.auinit1%253DM.%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D1007%26rft.epage%253D1017%26rft.atitle%253DReliable%2Bprediction%2Bof%2BT-cell%2Bepitopes%2Busing%2Bneural%2Bnetworks%2Bwith%2Bnovel%2Bsequence%2Brepresentations%26rft_id%253Dinfo%253Adoi%252F10.1110%252Fps.0239403%26rft_id%253Dinfo%253Apmid%252F12717023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1110/ps.0239403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12717023&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.27.967422.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182486800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.19" data-doi="10.1186/1471-2105-6-132"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sette</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Generating quantitative models describing the sequence specificity of biological processes with the stabilized matrix method</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DPeters%26rft.auinit1%253DB.%26rft.volume%253D6%26rft.spage%253D132%26rft.epage%253D132%26rft.atitle%253DGenerating%2Bquantitative%2Bmodels%2Bdescribing%2Bthe%2Bsequence%2Bspecificity%2Bof%2Bbiological%2Bprocesses%2Bwith%2Bthe%2Bstabilized%2Bmatrix%2Bmethod.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-6-132%26rft_id%253Dinfo%253Apmid%252F15927070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-6-132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15927070&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.27.967422.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.20" data-doi="10.1093/nar/gku938"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vita</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Overton</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greenbaum</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ponomarenko</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cantrell</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">DK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gabbard</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hix</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sette</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">The immune epitope database (IEDB) 3.0</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2014</span> <span class="cit-month">Oct</span> <span class="cit-day">9</span>. pii: gku938. [Epub ahead of print] PubMed<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID: </span><span class="cit-pub-id cit-pub-id-pmid">25300482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku938%26rft_id%253Dinfo%253Apmid%252F25300482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku938&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25300482&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.27.967422.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.21" data-doi="10.1007/s00018-005-4528-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tenzer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bulik</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schoor</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lemmel</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schatz</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kloetzel</span>  <span class="cit-name-given-names">PM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rammensee</span>  <span class="cit-name-given-names">HG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schild</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holzhutter</span>  <span class="cit-name-given-names">HG.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Modeling the MHC class I pathway by combining predictions of proteasomal cleavage, TAP transport and MHC class I binding</span>. <abbr class="cit-jnl-abbrev">Cell Mol Life Sci</abbr> <span class="cit-vol">62</span>:<span class="cit-fpage">1025</span>–<span class="cit-lpage">1037</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellular%2Band%2Bmolecular%2Blife%2Bsciences%2B%253A%2B%2BCMLS%26rft.stitle%253DCell%2BMol%2BLife%2BSci%26rft.aulast%253DTenzer%26rft.auinit1%253DS.%26rft.volume%253D62%26rft.issue%253D9%26rft.spage%253D1025%26rft.epage%253D1037%26rft.atitle%253DModeling%2Bthe%2BMHC%2Bclass%2BI%2Bpathway%2Bby%2Bcombining%2Bpredictions%2Bof%2Bproteasomal%2Bcleavage%252C%2BTAP%2Btransport%2Band%2BMHC%2Bclass%2BI%2Bbinding.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00018-005-4528-2%26rft_id%253Dinfo%253Apmid%252F15868101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00018-005-4528-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15868101&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.27.967422.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232496600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.22" data-doi="10.1186/1471-2105-8-424"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Larsen</span>,  <span class="cit-name-given-names">M.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lundegaard</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lamberth</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buus</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lund</span>,  <span class="cit-name-given-names">O.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Large-scale validation of methods for cytotoxic T-lymphocyte epitope prediction</span>. <abbr class="cit-jnl-abbrev">BMC bioinformatics</abbr>, <span class="cit-vol">8</span>(<span class="cit-issue">1</span>), p.<span class="cit-fpage">424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLarsen%26rft.auinit1%253DM.%2BV.%26rft.volume%253D8%26rft.spage%253D424%26rft.epage%253D424%26rft.atitle%253DLarge-scale%2Bvalidation%2Bof%2Bmethods%2Bfor%2Bcytotoxic%2BT-lymphocyte%2Bepitope%2Bprediction.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-424%26rft_id%253Dinfo%253Apmid%252F17973982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-424&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17973982&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.27.967422.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bui</span>  <span class="cit-name-given-names">HH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sidney</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dinh</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Southwood</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sette</span>  <span class="cit-name-given-names">A.</span></span>. <span class="cit-article-title">Predicting population coverage of T-cell epitope-based diagnostics and vaccines</span>.<abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>. <span class="cit-pub-date">2006</span>; <span class="cit-vol">17</span>:<span class="cit-fpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D17%26rft.spage%253D153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.24" data-doi="10.1093/nar/gks419"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thévenet</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maupetit</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guyon</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Derreumaux</span>,  <span class="cit-name-given-names">P.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Tufféry</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-pub-date">2012</span>. <span class="cit-article-title">PEP-FOLD: an updated de novo structure prediction server for both linear and disulfide bonded cyclic peptides</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>, <span class="cit-vol">40</span>(<span class="cit-issue">W1</span>), pp.<span class="cit-fpage">W288</span>–<span class="cit-lpage">W293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks419%26rft_id%253Dinfo%253Apmid%252F22581768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks419&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22581768&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F02%2F2020.02.27.967422.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306670900047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maupetty</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Derreumaux</span>,  <span class="cit-name-given-names">P.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Tufféry</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Improved PEP-FOLD approach for peptide and miniprotein structure prediction</span>. <abbr class="cit-jnl-abbrev">Journal of chemical theory and computation</abbr>, <span class="cit-vol">10</span>(<span class="cit-issue">10</span>), pp.<span class="cit-fpage">4745</span>–<span class="cit-lpage">4758</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bchemical%2Btheory%2Band%2Bcomputation%26rft.volume%253D10%26rft.spage%253D4745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kozakov</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hall</span>,  <span class="cit-name-given-names">D.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Porter</span>,  <span class="cit-name-given-names">K.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Padhorny</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yueh</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beglov</span>,  <span class="cit-name-given-names">D.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Vajda</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-pub-date">2017</span>. <span class="cit-article-title">The ClusPro web server for protein-protein docking</span>. <abbr class="cit-jnl-abbrev">nature protocols</abbr>, <span class="cit-vol">12</span>(<span class="cit-issue">2</span>), pp.<span class="cit-fpage">255</span>–<span class="cit-lpage">278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253Dnature%2Bprotocols%26rft.volume%253D12%26rft.spage%253D255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.27.967422v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kozakov</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brenke</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Comeau</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vajda</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">PIPER: an FFT-based protein docking program with pairwise potentials</span>. <abbr class="cit-jnl-abbrev">Proteins: Structure, Function, and Bioinformatics</abbr>. <span class="cit-pub-date">2006</span> <span class="cit-month">Nov</span> <span class="cit-day">1</span>;<span class="cit-vol">65</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">392</span>–<span class="cit-lpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%253A%2BStructure%252C%2BFunction%252C%2Band%2BBioinformatics%26rft.volume%253D65%26rft.spage%253D392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.02.27.967422v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.02.27.967422v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.02.29.971390v1" title="smiFISH and embryo segmentation for single-cell multi-gene RNA quantification in arthropods" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.02.972745v1" title="Fusion pore expansion and contraction during catecholamine release from endocrine cells" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 02, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.02.27.967422v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.02.27.967422v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.02.27.967422v1" class="active" data-icon-position="" data-hide-link-title="0">Strategies for vaccine design for corona virus using Immunoinformatics techniques</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1177200" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-1GKIyRp1amPVzX7mBeAqPGRLNhD0tqHybc1hN75yyOQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1177200" id="node1177200--2" data-pisa="biorxiv;2020.02.27.967422v1" data-pisa-master="biorxiv;2020.02.27.967422" data-seqnum="1177200" data-apath="/biorxiv/early/2020/03/02/2020.02.27.967422.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Strategies for vaccine design for corona virus using Immunoinformatics techniques</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Anamika</span> <span class="nlm-surname">Basu</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Anasua</span> <span class="nlm-surname">Sarkar</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Ujjwal</span> <span class="nlm-surname">Maulik</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.27.967422; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.27.967422 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.02.27.967422v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.967422v1&amp;title=Strategies%20for%20vaccine%20design%20for%20corona%20virus%20using%20Immunoinformatics%20techniques" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.967422v1&amp;title=Strategies%20for%20vaccine%20design%20for%20corona%20virus%20using%20Immunoinformatics%20techniques" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.967422v1&amp;text=Strategies%20for%20vaccine%20design%20for%20corona%20virus%20using%20Immunoinformatics%20techniques" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.967422v1&amp;title=Strategies%20for%20vaccine%20design%20for%20corona%20virus%20using%20Immunoinformatics%20techniques" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.967422v1&amp;t=Strategies%20for%20vaccine%20design%20for%20corona%20virus%20using%20Immunoinformatics%20techniques" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.967422v1&amp;title=Strategies%20for%20vaccine%20design%20for%20corona%20virus%20using%20Immunoinformatics%20techniques" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.967422v1&amp;title=Strategies%20for%20vaccine%20design%20for%20corona%20virus%20using%20Immunoinformatics%20techniques" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1177200" id="node1177200--3" data-pisa="biorxiv;2020.02.27.967422v1" data-pisa-master="biorxiv;2020.02.27.967422" data-seqnum="1177200" data-apath="/biorxiv/early/2020/03/02/2020.02.27.967422.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Strategies for vaccine design for corona virus using Immunoinformatics techniques</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Anamika</span> <span class="nlm-surname">Basu</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Anasua</span> <span class="nlm-surname">Sarkar</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Ujjwal</span> <span class="nlm-surname">Maulik</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.27.967422; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.27.967422 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Strategies%20for%20vaccine%20design%20for%20corona%20virus%20using%20Immunoinformatics%20techniques&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Basu%2C+Anamika&amp;rft.au=Sarkar%2C+Anasua&amp;rft.au=Maulik%2C+Ujjwal"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1177200/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1177200/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1177200/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1177200/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1177200/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1177200/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1177200/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1177200/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1177200/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1177200/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1177200/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1177200/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.967422v1&amp;count=horizontal&amp;via=&amp;text=Strategies%20for%20vaccine%20design%20for%20corona%20virus%20using%20Immunoinformatics%20techniques&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.967422v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.967422v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.02.27.967422v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/immunology" class="highlight" data-icon-position="" data-hide-link-title="0">Immunology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
