<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/05/23/639369.atom" />
<meta name="HW.pisa" content="biorxiv;639369v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Identification of a Nidovirales Orf1a N7-guanine cap Methyltransferase signature-sequence as a genetic marker of large genome Tobaniviridae" />
<meta name="DC.Identifier" content="10.1101/639369" />
<meta name="DC.Date" content="2019-05-23" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Members of the Nidovirales order have (+)RNA genomes amongst the largest in size in the RNA virus world. Expression of their genes is promoted through reading of genomic RNA and mRNA transcripts by the ribosome of the infected cell. The 5’-end of these RNAs is supposedly protected by an RNA-cap structure (m7GpppNm) whose most synthesis steps remain elusive. In Eukaryotes, the RNA-cap structure is methylated by RNA methyltransferases (MTases) at the RNA-cap N7-guanine position as well as the 2’-O methyl position of the first transcribed nucleotide. In Coronaviridae , two separate enzymes (nsp14 and nsp16) perform the N7-guanine and the 2’-OH methylation, respectively. One salient feature of the Nidovirales N7-guanine MTase nsp14 is that it is the only example of non-Rossman fold viral MTase known so far. Conversely, all other Nidovirales nsp16-like MTases have a canonical Rossman fold. Many Nidovirales members lack either any RNA MTase signature sequence (e.g., Arteriviridae ), or lack a N7-guanine MTase signature sequence (e.g., Tobaniviridae , Euroniviridae , Roniviridae , Medioniviridae ). Both nsp14-and nsp16-like enzyme genes are usually located in Orf1b encoding for the replication machinery. Here, we report the discovery of a putative Rossman fold RNA MTase in the Orf1a of ten Tobaniviridae members. Multiple sequence alignments and structural analyses identify this novel gene as a typical RNA-cap N7-guanine MTase with substrate specificity and active-site organization similar to the canonical eukaryotic RNA-cap N7-guanine MTase." />
<meta name="DC.Contributor" content="François Ferron" />
<meta name="DC.Contributor" content="Humberto Julio Debat" />
<meta name="DC.Contributor" content="Etienne Decroly" />
<meta name="DC.Contributor" content="Bruno Canard" />
<meta name="article:published_time" content="2019-05-23" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Identification of a Nidovirales Orf1a N7-guanine cap Methyltransferase signature-sequence as a genetic marker of large genome Tobaniviridae" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Members of the &lt;i&gt;Nidovirales&lt;/i&gt; order have (+)RNA genomes amongst the largest in size in the RNA virus world. Expression of their genes is promoted through reading of genomic RNA and mRNA transcripts by the ribosome of the infected cell. The 5’-end of these RNAs is supposedly protected by an RNA-cap structure (m7GpppNm) whose most synthesis steps remain elusive. In Eukaryotes, the RNA-cap structure is methylated by RNA methyltransferases (MTases) at the RNA-cap N7-guanine position as well as the 2’-O methyl position of the first transcribed nucleotide. In &lt;i&gt;Coronaviridae&lt;/i&gt;, two separate enzymes (nsp14 and nsp16) perform the N7-guanine and the 2’-OH methylation, respectively. One salient feature of the &lt;i&gt;Nidovirales&lt;/i&gt; N7-guanine MTase nsp14 is that it is the only example of non-Rossman fold viral MTase known so far. Conversely, all other &lt;i&gt;Nidovirales&lt;/i&gt; nsp16-like MTases have a canonical Rossman fold. Many &lt;i&gt;Nidovirales&lt;/i&gt; members lack either any RNA MTase signature sequence (e.g., &lt;i&gt;Arteriviridae&lt;/i&gt;), or lack a N7-guanine MTase signature sequence (e.g., &lt;i&gt;Tobaniviridae&lt;/i&gt;, &lt;i&gt;Euroniviridae&lt;/i&gt;, &lt;i&gt;Roniviridae&lt;/i&gt;, &lt;i&gt;Medioniviridae&lt;/i&gt;). Both nsp14-and nsp16-like enzyme genes are usually located in Orf1b encoding for the replication machinery. Here, we report the discovery of a putative Rossman fold RNA MTase in the Orf1a of ten &lt;i&gt;Tobaniviridae&lt;/i&gt; members. Multiple sequence alignments and structural analyses identify this novel gene as a typical RNA-cap N7-guanine MTase with substrate specificity and active-site organization similar to the canonical eukaryotic RNA-cap N7-guanine MTase.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;639369v2" />
<meta name="citation_id" content="639369v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/639369v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/639369v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/639369v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/05/23/639369.full.pdf" />
<meta name="citation_doi" content="10.1101/639369" />
<meta name="citation_num_pages" content="20" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="639369" />
<meta name="citation_author" content="François Ferron" />
<meta name="citation_author_institution" content="Architecture et Fonction des Macromolécules Biologiques, CNRS and Aix-Marseille Université" />
<meta name="citation_author_institution" content="European Virus Bioinformatics Center" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8351-4992" />
<meta name="citation_author" content="Humberto Julio Debat" />
<meta name="citation_author_institution" content="Instituto de Patología Vegetal, Centro de Investigaciones Agropecuarias, Instituto Nacional de Tecnología Agropecuaria" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3056-3739" />
<meta name="citation_author" content="Etienne Decroly" />
<meta name="citation_author_institution" content="Architecture et Fonction des Macromolécules Biologiques, CNRS and Aix-Marseille Université" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6046-024X" />
<meta name="citation_author" content="Bruno Canard" />
<meta name="citation_author_institution" content="Architecture et Fonction des Macromolécules Biologiques, CNRS and Aix-Marseille Université" />
<meta name="citation_author_institution" content="European Virus Bioinformatics Center" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4924-1991" />
<meta name="citation_reference" content="Siddell SG, Walker PJ, Lefkowitz EJ, Mushegian AR, Adams MJ, Dutilh BE, Gorbalenya AE, Harrach B, Harrison RL, Junglen S, et al.: Additional changes to taxonomy ratified in a special vote by the International Committee on Taxonomy of Viruses (October 2018). Arch Virol 2019, 164:943–946." />
<meta name="citation_reference" content="Song Z, Xu Y, Bao L, Zhang L, Yu P, Qu Y, Zhu H, Zhao W, Han Y, Qin C: From SARS to MERS, Thrusting Coronaviruses into the Spotlight. Viruses 2019, 11." />
<meta name="citation_reference" content="Gorbalenya AE, Enjuanes L, Ziebuhr J, Snijder EJ: Nidovirales: evolving the largest RNA virus genome. Virus Res 2006, 117:17–37." />
<meta name="citation_reference" content="Snijder EJ, Bredenbeek PJ, Dobbe JC, Thiel V, Ziebuhr J, Poon LLM, Guan Y, Rozanov M, Spaan WJM, Gorbalenya AE: Unique and conserved features of genome and proteome of SARS-coronavirus, an early split-off from the coronavirus group 2 lineage. J Mol Biol 2003, 331:991–1004." />
<meta name="citation_reference" content="Ramanathan A, Robb GB, Chan S-H: mRNA capping: biological functions and applications. Nucleic Acids Res 2016, 44:7511–7526." />
<meta name="citation_reference" content="Decroly E, Ferron F, Lescar J, Canard B: Conventional and unconventional mechanisms for capping viral mRNA. Nat Rev Microbiol 2011, 10:51–65." />
<meta name="citation_reference" content="van Vliet ALW, Smits SL, Rottier PJM, de Groot RJ: Discontinuous and non-discontinuous subgenomic RNA transcription in a nidovirus. EMBO J 2002, 21:6571–6580." />
<meta name="citation_reference" content="Lai MM, Patton CD, Stohlman SA: Further characterization of mRNA’s of mouse hepatitis virus: presence of common 5’-end nucleotides. J Virol 1982, 41:557–565." />
<meta name="citation_reference" content="Sagripanti JL, Zandomeni RO, Weinmann R: The cap structure of simian hemorrhagic fever virion RNA. Virology 1986, 151:146–150." />
<meta name="citation_reference" content="Decroly E, Imbert I, Coutard B, Bouvet M, Selisko B, Alvarez K, Gorbalenya AE, Snijder EJ, Canard B: Coronavirus nonstructural protein 16 is a cap-0 binding enzyme possessing (nucleoside-2’O)-methyltransferase activity. J Virol 2008, 82:8071–8084." />
<meta name="citation_reference" content="Ferron F, Subissi L, Silveira De Morais AT, Le NTT, Sevajol M, Gluais L, Decroly E, Vonrhein C, Bricogne G, Canard B, et al.: Structural and molecular basis of mismatch correction and ribavirin excision from coronavirus RNA. Proc Natl Acad Sci U S A 2018, 115:E162–E171." />
<meta name="citation_reference" content="Ma Y, Wu L, Shaw N, Gao Y, Wang J, Sun Y, Lou Z, Yan L, Zhang R, Rao Z: Structural basis and functional analysis of the SARS coronavirus nsp14-nsp10 complex. Proc Natl Acad Sci U S A 2015, 112:9436–9441." />
<meta name="citation_reference" content="Chen Y, Su C, Ke M, Jin X, Xu L, Zhang Z, Wu A, Sun Y, Yang Z, Tien P, et al.: Biochemical and structural insights into the mechanisms of SARS coronavirus RNA ribose 2’-O-methylation by nsp16/nsp10 protein complex. PLoS Pathog 2011, 7:e1002294." />
<meta name="citation_reference" content="Decroly E, Debarnot C, Ferron F, Bouvet M, Coutard B, Imbert I, Gluais L, Papageorgiou N, Sharff A, Bricogne G, et al.: Crystal structure and functional analysis of the SARS-coronavirus RNA cap 2’-O-methyltransferase nsp10/nsp16 complex. PLoS Pathog 2011, 7:e1002059." />
<meta name="citation_reference" content="Perdigão N, Heinrich J, Stolte C, Sabir KS, Buckley MJ, Tabor B, Signal B, Gloss BS, Hammang CJ, Rost B, et al.: Unexpected features of the dark proteome. Proc Natl Acad Sci U S A 2015, 112:15898–15903." />
<meta name="citation_reference" content="Zeng C, Wu A, Wang Y, Xu S, Tang Y, Jin X, Wang S, Qin L, Sun Y, Fan C, et al.: Identification and Characterization of a Ribose 2’-O-Methyltransferase Encoded by the Ronivirus Branch of Nidovirales. J Virol 2016, 90:6675–6685." />
<meta name="citation_reference" content="Ivanov KA, Ziebuhr J: Human coronavirus 229E nonstructural protein 13: characterization of duplex-unwinding, nucleoside triphosphatase, and RNA 5’-triphosphatase activities. J Virol 2004, 78:7833–7838." />
<meta name="citation_reference" content="Lehmann KC, Gulyaeva A, Zevenhoven-Dobbe JC, Janssen GMC, Ruben M, Overkleeft HS, van Veelen PA, Samborskiy DV, Kravchenko AA, Leontovich AM, et al.: Discovery of an essential nucleotidylating activity associated with a newly delineated conserved domain in the RNA polymerase-containing protein of all nidoviruses. Nucleic Acids Res 2015, 43:8416–8434." />
<meta name="citation_reference" content="Minskaia E, Hertzig T, Gorbalenya AE, Campanacci V, Cambillau C, Canard B, Ziebuhr J: Discovery of an RNA virus 3’-&gt;5’ exoribonuclease that is critically involved in coronavirus RNA synthesis. Proc Natl Acad Sci U S A 2006, 103:5108–5113." />
<meta name="citation_reference" content="Bouvet M, Imbert I, Subissi L, Gluais L, Canard B, Decroly E: RNA 3’-end mismatch excision by the severe acute respiratory syndrome coronavirus nonstructural protein nsp10/nsp14 exoribonuclease complex. Proc Natl Acad Sci U S A 2012, 109:9372–9377." />
<meta name="citation_reference" content="Lauber C, Goeman JJ, Parquet M del C, Nga PT, Snijder EJ, Morita K, Gorbalenya AE: The footprint of genome architecture in the largest genome expansion in RNA viruses. PLoS Pathog 2013, 9:e1003500." />
<meta name="citation_reference" content="Eckerle LD, Lu X, Sperry SM, Choi L, Denison MR: High fidelity of murine hepatitis virus replication is decreased in nsp14 exoribonuclease mutants. J Virol 2007, 81:12135–12144." />
<meta name="citation_reference" content="Bukhari K, Mulley G, Gulyaeva AA, Zhao L, Shu G, Jiang J, Neuman BW: Description and initial characterization of metatranscriptomic nidovirus-like genomes from the proposed new family Abyssoviridae, and from a sister group to the Coronavirinae, the proposed genus Alphaletovirus. Virology 2018, 524:160–171." />
<meta name="citation_reference" content="Debat HJ: Expanding the size limit of RNA viruses: Evidence of a novel divergent nidovirus in California sea hare, with a ~35.9 kb virus genome. bioRxiv 2018, doi:10.1101/307678." />
<meta name="citation_reference" content="Debat HJ: An RNA Virome Associated to the Golden Orb-Weaver Spider Nephila clavipes. Front Microbiol 2017, 8." />
<meta name="citation_reference" content="Armougom F, Moretti S, Poirot O, Audic S, Dumas P, Schaeli B, Keduas V, Notredame C: Expresso: automatic incorporation of structural information in multiple sequence alignments using 3D-Coffee. Nucleic Acids Res 2006, 34:W604–608." />
<meta name="citation_reference" content="Pettersen EF, Goddard TD, Huang CC, Couch GS, Greenblatt DM, Meng EC, Ferrin TE: UCSF Chimera--a visualization system for exploratory research and analysis. J Comput Chem 2004, 25:1605–1612." />
<meta name="citation_reference" content="Zimmermann L, Stephens A, Nam S-Z, Rau D, Kübler J, Lozajic M, Gabler F, Söding J, Lupas AN, Alva V: A Completely Reimplemented MPI Bioinformatics Toolkit with a New HHpred Server at its Core. J Mol Biol 2018, 430:2237–2243." />
<meta name="citation_reference" content="Gouy M, Guindon S, Gascuel O: SeaView version 4: A multiplatform graphical user interface for sequence alignment and phylogenetic tree building. Mol Biol Evol 2010, 27:221–224." />
<meta name="citation_reference" content="Yachdav G, Kloppmann E, Kajan L, Hecht M, Goldberg T, Hamp T, Hönigschmid P, Schafferhans A, Roos M, Bernhofer M, et al.: PredictProtein--an open resource for online prediction of protein structural and functional features. Nucleic Acids Res 2014, 42:W337–343." />
<meta name="citation_reference" content="Gouet P, Robert X, Courcelle E: ESPript/ENDscript: Extracting and rendering sequence and 3D information from atomic structures of proteins. Nucleic Acids Res 2003, 31:3320–3323." />
<meta name="citation_reference" content="Kelley LA, Mezulis S, Yates CM, Wass MN, Sternberg MJE: The Phyre2 web portal for protein modeling, prediction and analysis. Nat Protoc 2015, 10:845–858." />
<meta name="citation_reference" content="Crooks GE, Hon G, Chandonia J-M, Brenner SE: WebLogo: a sequence logo generator. Genome Res 2004, 14:1188–1190." />
<meta name="citation_reference" content="Martin JL, McMillan FM: SAM (dependent) I AM: the S-adenosylmethionine-dependent methyltransferase fold. Curr Opin Struct Biol 2002, 12:783–793." />
<meta name="citation_reference" content="Byszewska M, Śmietański M, Purta E, Bujnicki JM: RNA methyltransferases involved in 5’ cap biosynthesis. RNA Biol 2014, 11:1597–1607." />
<meta name="citation_reference" content="Fabrega C, Hausmann S, Shen V, Shuman S, Lima CD: Structure and mechanism of mRNA cap (guanine-N7) methyltransferase. Mol Cell 2004, 13:77–89." />
<meta name="citation_reference" content="De la Peña M, Kyrieleis OJP, Cusack S: Structural insights into the mechanism and evolution of the vaccinia virus mRNA cap N7 methyl-transferase. EMBO J 2007, 26:4913–4925." />
<meta name="citation_reference" content="Sutton G, Grimes JM, Stuart DI, Roy P: Bluetongue virus VP4 is an RNA-capping assembly line. Nat Struct Mol Biol 2007, 14:449–451." />
<meta name="citation_reference" content="Tao Y, Farsetta DL, Nibert ML, Harrison SC: RNA synthesis in a cage--structural studies of reovirus polymerase lambda3. Cell 2002, 111:733–745." />
<meta name="citation_reference" content="Ray D, Shah A, Tilgner M, Guo Y, Zhao Y, Dong H, Deas TS, Zhou Y, Li H, Shi P-Y: West Nile virus 5’-cap structure is formed by sequential guanine N-7 and ribose 2’-O methylations by nonstructural protein 5. J Virol 2006, 80:8362–8370." />
<meta name="citation_reference" content="Fox NK, Brenner SE, Chandonia J-M: SCOPe: Structural Classification of Proteins--extended, integrating SCOP and ASTRAL data and classification of new structures. Nucleic Acids Res 2014, 42:D304–309." />
<meta name="citation_reference" content="Ahola T, Kääriäinen L: Reaction in alphavirus mRNA capping: formation of a covalent complex of nonstructural protein nsP1 with 7-methyl-GMP. Proc Natl Acad Sci U S A 1995, 92:507–511." />
<meta name="citation_reference" content="Egloff M-P, Benarroch D, Selisko B, Romette J-L, Canard B: An RNA cap (nucleoside-2’-O-)-methyltransferase in the flavivirus RNA polymerase NS5: crystal structure and functional characterization. EMBO J 2002, 21:2757–2768." />
<meta name="citation_reference" content="Paesen GC, Collet A, Sallamand C, Debart F, Vasseur J-J, Canard B, Decroly E, Grimes JM: X-ray structure and activities of an essential Mononegavirales L-protein domain. Nat Commun 2015, 6:8749." />
<meta name="citation_reference" content="Martin B, Coutard B, Guez T, Paesen GC, Canard B, Debart F, Vasseur J-J, Grimes JM, Decroly E: The methyltransferase domain of the Sudan ebolavirus L protein specifically targets internal adenosines of RNA substrates, in addition to the cap structure. Nucleic Acids Res 2018, 46:7902–7912." />
<meta name="citation_reference" content="Ringeard M, Marchand V, Decroly E, Motorin Y, Bennasser Y: FTSJ3 is an RNA 2’-O-Methyltransferase recruited by HIV to avoid innate immunity sensing. Nature 2019," />
<meta name="citation_reference" content="Rana AK, Ankri S: Reviving the RNA World: An Insight into the Appearance of RNA Methyltransferases. Front Genet 2016, 7:99." />
<meta name="citation_reference" content="Varshney D, Petit A-P, Bueren-Calabuig JA, Jansen C, Fletcher DA, Peggie M, Weidlich S, Scullion P, Pisliakov AV, Cowling VH: Molecular basis of RNA guanine-7 methyltransferase (RNMT) activation by RAM. Nucleic Acids Res 2016, 44:10423–10436." />
<meta name="citation_reference" content="Reinisch KM, Nibert ML, Harrison SC: Structure of the reovirus core at 3.6 A resolution. Nature 2000, 404:960–967." />
<meta name="citation_reference" content="Hausmann S, Zheng S, Fabrega C, Schneller SW, Lima CD, Shuman S: Encephalitozoon cuniculi mRNA cap (guanine N-7) methyltransferase: methyl acceptor specificity, inhibition BY S-adenosylmethionine analogs, and structure-guided mutational analysis. J Biol Chem 2005, 280:20404–20412." />
<meta name="citation_reference" content="Kyrieleis OJP, Chang J, de la Peña M, Shuman S, Cusack S: Crystal structure of vaccinia virus mRNA capping enzyme provides insights into the mechanism and evolution of the capping apparatus. Struct Lond Engl 1993 2014, 22:452–465." />
<meta name="twitter:title" content="Identification of a Nidovirales Orf1a N7-guanine cap Methyltransferase signature-sequence as a genetic marker of large genome Tobaniviridae" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Members of the Nidovirales order have (+)RNA genomes amongst the largest in size in the RNA virus world. Expression of their genes is promoted through reading of genomic RNA and mRNA transcripts by the ribosome of the infected cell. The 5’-end of these RNAs is supposedly protected by an RNA-cap structure (m7GpppNm) whose most synthesis steps remain elusive. In Eukaryotes, the RNA-cap structure is methylated by RNA methyltransferases (MTases) at the RNA-cap N7-guanine position as well as the 2’-O methyl position of the first transcribed nucleotide. In Coronaviridae , two separate enzymes (nsp14 and nsp16) perform the N7-guanine and the 2’-OH methylation, respectively. One salient feature of the Nidovirales N7-guanine MTase nsp14 is that it is the only example of non-Rossman fold viral MTase known so far. Conversely, all other Nidovirales nsp16-like MTases have a canonical Rossman fold. Many Nidovirales members lack either any RNA MTase signature sequence (e.g., Arteriviridae ), or lack a N7-guanine MTase signature sequence (e.g., Tobaniviridae , Euroniviridae , Roniviridae , Medioniviridae ). Both nsp14-and nsp16-like enzyme genes are usually located in Orf1b encoding for the replication machinery. Here, we report the discovery of a putative Rossman fold RNA MTase in the Orf1a of ten Tobaniviridae members. Multiple sequence alignments and structural analyses identify this novel gene as a typical RNA-cap N7-guanine MTase with substrate specificity and active-site organization similar to the canonical eukaryotic RNA-cap N7-guanine MTase." />
<meta name="og-title" property="og:title" content="Identification of a Nidovirales Orf1a N7-guanine cap Methyltransferase signature-sequence as a genetic marker of large genome Tobaniviridae" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/639369v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Members of the Nidovirales order have (+)RNA genomes amongst the largest in size in the RNA virus world. Expression of their genes is promoted through reading of genomic RNA and mRNA transcripts by the ribosome of the infected cell. The 5’-end of these RNAs is supposedly protected by an RNA-cap structure (m7GpppNm) whose most synthesis steps remain elusive. In Eukaryotes, the RNA-cap structure is methylated by RNA methyltransferases (MTases) at the RNA-cap N7-guanine position as well as the 2’-O methyl position of the first transcribed nucleotide. In Coronaviridae , two separate enzymes (nsp14 and nsp16) perform the N7-guanine and the 2’-OH methylation, respectively. One salient feature of the Nidovirales N7-guanine MTase nsp14 is that it is the only example of non-Rossman fold viral MTase known so far. Conversely, all other Nidovirales nsp16-like MTases have a canonical Rossman fold. Many Nidovirales members lack either any RNA MTase signature sequence (e.g., Arteriviridae ), or lack a N7-guanine MTase signature sequence (e.g., Tobaniviridae , Euroniviridae , Roniviridae , Medioniviridae ). Both nsp14-and nsp16-like enzyme genes are usually located in Orf1b encoding for the replication machinery. Here, we report the discovery of a putative Rossman fold RNA MTase in the Orf1a of ten Tobaniviridae members. Multiple sequence alignments and structural analyses identify this novel gene as a typical RNA-cap N7-guanine MTase with substrate specificity and active-site organization similar to the canonical eukaryotic RNA-cap N7-guanine MTase." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-05-23" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/639369v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/639369v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/639369v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/639369v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/744855" />
    <title>Identification of a Nidovirales Orf1a N7-guanine cap Methyltransferase signature-sequence as a genetic marker of large genome Tobaniviridae | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Mo5mwfO_1-UITnUGWeEO09ObMTmNrzstzEfdC3S1JAc","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"744855","apath":"\/biorxiv\/early\/2019\/05\/23\/639369.atom","pisa":"biorxiv;639369v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;639369v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/639369v2","title":"Identification of a Nidovirales Orf1a N7-guanine cap Methyltransferase signature-sequence as a genetic marker of large genome Tobaniviridae","identifier":"node\/744855"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022taBYsnmp8Nucfupgmsde0JWLlklqTFQQH2Cdicc6RqFwT9xH\\\\\\\/wWvo7ywNelLKinjgWwziFtv1dKK80FudKWCevSGODCEhrxeWCpkAiEMKNIvdPB4pERfltGWrc89u7coZRJJcQNONnMvuHT754ujQZpxhNqbTg9XkZhQiALW1mCxuyEbD8x3yHJUYn24xug5pDOKKuGZtKl7BJ\\\\\\\/FgildoHPIdsB4o9Vah6prPc26zgg3MekkD8nr+RPQGDoEp7BUt4rUM5+4hkrvsuFQofGjR1gHrhgBI7LUdQHNebO+Ug42bc9iGSK6D989jTsMxkqVXp15Xf2XkdlNLUtRJ+WkLzSGnYkwKX0g0mpwtT6ricOzPqZ33uh+X\\\\\\\/3av4Xkj8oYptlTLl\\\\\\\/bIi\\\\\\\/KA9BtnpMPyYKBJ20LCkB3MIs972eYJr2WPnDW1IeuxE6F8bgR2s30vIVZ4FobMcn4dK1GZsp6n6R2DDQ+4yVDo7oJta2eGEs3QG4MN9G5h+t\\\\\\\/S1nHj6HmrrIWwE6a22c22CSJZ\\\\\\\/0GmiaS4BgXB0lU7NW9iqckA3+hF651BjSukNpDgfUy4SmHHKpTc6CeWBaZaKkJDcOwQ7qkzHqS+8RgQieibdWkHZrWZwGdgmvJhYofmgrbqqe3fTi+bbqlfOxKsxFwBzwAGbyK\\\\\\\/3S2WKhRUkWmJxjkvUtR5IlJwUfJXHC4DT3EjH7ApcdX+tXJbCQDZ3NpqU1BxnkggGxaeRx0MhJuLaMNrw4fbOs5B3DYJwiBnCEkno\\\\\\\/E\\\\\\\/7NkOwxskZ1pj6OOyOiaFymS1h3HXSccEbtCvLfaFEEWs6PI3mGgZyDxqWJf6sS1iskcmcxgt4s8mwu8DOh+dUgN7yI1B5I1CxsJYaj9HbcP\\\\\\\/\\\\\\\/qiSbNfOfxhHQWGSYle+o7OEERYDGcMD+EvC4efVYs6aCSBlaxiDfkFw27zBOIPweY1tMic49dGTe3M6sa92cmH23nPgRV21kmE3mBSxhsJbdXklK3V1w9a2hYw3PhS6HO7+Pe\\\\\\\/S\\\\\\\/gVXnmB6yYOLAPUVSf4uOVgyGbab02CcoKq9ogD361FKyF1O24bwXiy8JrlR7I9kG2Dn1L94N9w8rbdsdeQKswiuw8ZovZh+FjvMi5L6zDGKAQUY7VKKiC9qbqp5iOA8HQoaIRoXxIU\\\\\\\/Z3x2Iw6HKujys1xkD+Hc8OxhYAGBcqre9Tji2z922tJSU1qOxa8YoCuMbIcg9M8RskBJwZn2\\\\\\\/FCfHm4pA5dGjpyp6fcfGfBIbNcYL6dWWdZyKm4GFg8s5XDkINTQ5UqIgEs6bt2ElFwqFzW\\\\\\\/CknD9RiKA2RhR8CDmV16R9HuV8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022AHrSb\\\\\\\/FMSY03avpeNJJSMw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002212e4e028f9c0d05fa20e8d90936d46815033c1a7c3f938ae8be33a4673a4eaec\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/639369v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-744855 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/639369v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qRVfSpruhx8FM1isparQN7qL-pwuHBAzG3GH1ExJWpw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="744855" id="node744855" data-pisa="biorxiv;639369v2" data-pisa-master="biorxiv;639369" data-apath="/biorxiv/early/2019/05/23/639369.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Identification of a <em>Nidovirales</em> Orf1a N7-guanine cap Methyltransferase signature-sequence as a genetic marker of large genome <em>Tobaniviridae</em></h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8351-4992" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">François</span> <span class="nlm-surname">Ferron</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-3056-3739" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Humberto Julio</span> <span class="nlm-surname">Debat</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6046-024X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Etienne</span> <span class="nlm-surname">Decroly</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-4924-1991" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Bruno</span> <span class="nlm-surname">Canard</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/639369 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node744855" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">François Ferron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Architecture et Fonction des Macromolécules Biologiques, CNRS and Aix-Marseille Université</span>, UMR 7257, Polytech Case 925, 13009 Marseille, <span class='nlm-country'>France</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>European Virus Bioinformatics Center</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fran%C3%A7ois%2BFerron%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferron%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFran%25C3%25A7ois%2BFerron%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8351-4992" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for François Ferron</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Humberto Julio Debat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Instituto de Patología Vegetal, Centro de Investigaciones Agropecuarias, Instituto Nacional de Tecnología Agropecuaria</span>, Córdoba, <span class='nlm-country'>Argentina</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Humberto%2BJulio%2BDebat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Debat%20HJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHumberto%2BJulio%2BDebat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3056-3739" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Humberto Julio Debat</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Etienne Decroly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Architecture et Fonction des Macromolécules Biologiques, CNRS and Aix-Marseille Université</span>, UMR 7257, Polytech Case 925, 13009 Marseille, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Etienne%2BDecroly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Decroly%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEtienne%2BDecroly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6046-024X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Etienne Decroly</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bruno Canard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Architecture et Fonction des Macromolécules Biologiques, CNRS and Aix-Marseille Université</span>, UMR 7257, Polytech Case 925, 13009 Marseille, <span class='nlm-country'>France</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>European Virus Bioinformatics Center</span>, Leutragraben 1, 07743 Jena, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruno%2BCanard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Canard%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABruno%2BCanard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4924-1991" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bruno Canard</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/639369v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:744855" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:744855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/639369v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:744855" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:744855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/639369v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:744855" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:744855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/639369v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:744855" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:744855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/639369v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:744855" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:744855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/639369v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:744855" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:744855/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Members of the <em>Nidovirales</em> order have (+)RNA genomes amongst the largest in size in the RNA virus world. Expression of their genes is promoted through reading of genomic RNA and mRNA transcripts by the ribosome of the infected cell. The 5’-end of these RNAs is supposedly protected by an RNA-cap structure (m7GpppNm) whose most synthesis steps remain elusive. In Eukaryotes, the RNA-cap structure is methylated by RNA methyltransferases (MTases) at the RNA-cap N7-guanine position as well as the 2’-O methyl position of the first transcribed nucleotide. In <em>Coronaviridae</em>, two separate enzymes (nsp14 and nsp16) perform the N7-guanine and the 2’-OH methylation, respectively. One salient feature of the <em>Nidovirales</em> N7-guanine MTase nsp14 is that it is the only example of non-Rossman fold viral MTase known so far. Conversely, all other <em>Nidovirales</em> nsp16-like MTases have a canonical Rossman fold. Many <em>Nidovirales</em> members lack either any RNA MTase signature sequence (e.g., <em>Arteriviridae</em>), or lack a N7-guanine MTase signature sequence (e.g., <em>Tobaniviridae</em>, <em>Euroniviridae</em>, <em>Roniviridae</em>, <em>Medioniviridae</em>). Both nsp14-and nsp16-like enzyme genes are usually located in Orf1b encoding for the replication machinery. Here, we report the discovery of a putative Rossman fold RNA MTase in the Orf1a of ten <em>Tobaniviridae</em> members. Multiple sequence alignments and structural analyses identify this novel gene as a typical RNA-cap N7-guanine MTase with substrate specificity and active-site organization similar to the canonical eukaryotic RNA-cap N7-guanine MTase.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">Up to recently, four viral families, <em>Arteriviridae</em>, <em>Coronaviridae</em>, <em>Roniviridae</em>, and <em>Mesoniviridae</em> constituted the <em>Nidovirales</em> order. The <em>Coronaviridae</em> family was further divided in two subfamilies, <em>Coronavirinae</em> and <em>Torovirinae.</em> The <em>Torovirinae</em> family contained two genus <em>Torovirus</em> and <em>Bafinivirus</em>, plus few unassigned species. Recently, the ICTV proposed a novel classification into the <em>Nidovirales</em> order, with the creation of nine families [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. In this novel ICTV proposal, ratified on February 2019, <em>Coronaviridae</em> do not encompass both <em>Coronavirinae</em> and <em>Toronavirinae</em> subfamilies. Rather, the new suborder <em>Tornidovirinae</em> (including toroviruses and bafiniviruses) contains a novel family, <em>Tobaniviridae</em>, at the same level as the <em>Coronaviridae</em>, which is now within suborder <em>Cornidovirineae</em>. This new taxonomy is used here, as depicted in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/23/639369/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Presence, type, and loci of signature-sequences of Nidovirales RNA MTases : RF MT of unknown specificity (light red), non-RF &#xAB; nsp14-like &#xBB; MT (yellow), and RF &#xAB; nsp16-like &#xBB; MT (green). Both light red and yellow MTases map to the nsp14 C-ter locus, ie., immediately downstrean the ExoN domain, while &#xAB; nsp16-like &#xBB; MT map to the nsp16 locus, at the end of Orf1b. The tree was made based on MAFFT v7.427 multiple sequence alignment with BLOSUM62 scoring matrix and G-INS-i iterative refinement method. The alignments were used as input for maximum likelihood trees generated with the FasTtree v2.1.5 software (best-fit model = JTT-Jones-Taylor-Thorton with single rate of evolution for each site = CAT). Local support values were computed using the Shimodaira-Hasegawa test (SH) with 1,000 replicates. Numbers at the nodes represent FasTtree support values and scale var substitutions per site. The tree included two novel Ronivirus-like and Mesonivirus-like genome sequences: Western Flower Thrips Mesonivirus, and Palaemon Nidovirus, respectively (WFTV, PAN, unpublished). When present in the NCBI viral genomes database in the 92 Nidovirales complete genomes repository (https://www.ncbi.nlm.nih.gov/genomes/GenomesGroup.cgi?taxid=76804), no accession number is indicated. When an accession number is given in parenthesis, it is referring to the GenBank accession number (https://www.ncbi.nlm.nih.gov/genbank/). From top to bottom of the figure: Arteriviridae: HHPAV: Hainan hebius popei arterivirus (MG600021); NGSAV: Nanhai ghost shark arterivirus (MG600024); SHFV : simian hemorrhagic fever virus; EAV: Equine arteritis virus; LDV, Lactate elevating virus; PRRSV-1 and 2, porcine reproductive and respiratory syndrome virus. Tobaniviridae: BToV: Breda virus; GToV: Goat torovirus; EToV: Berne virus (CAA36747); PToV: Porcine torovirus; CSV: Chinook salmon bafinivirus; FHMNV: Fathead Minnow nidovirus; WBV: white bream virus; BoNV: Bovine nidovirus TCH5; INToV: Xinzhou nido-like virus 6; SECToV; Xinzhou toro-like virus 1; GRBS-ToV: Guangdong red banded snake torovirus (MG600030); ShNV: shingleback nidovirus 1 (KX184715); BPNV: Ball python nidovirus 1; MVNV: Morelia viridis nidovirus; BRV: Bellinger River virus (MF685025); Coronaviridae: IBV: Infectious Bronchitis Virus; BWCoV: Beluga Whale coronavirus SW1; HCoV_HKU1: Human coronavirus HK1; MHV: Mouse hepatitis virus; BatCoV_HKU9: Rousettus bat Coronavirus HKU9; SARS-CoV: Severe acute respiratory syndrome coronavirus; MERS_CoV: middle-east respiratory syndrome coronavirus; Ty-BatCoV_HKU4: Tylonycteris bat coronavirus HKU4; Pi-BatCoV_HKU5: pipistrellus bat coronavirus HKU5; FIPV: Feline infectious peritonitis virus; Rh-BatCoV-HKU2: Rhinolophus bat coronavirus HKU2; HCoV_229E: Human coronavirus 229E; HCoV_OC43: Human coronavirus OC43 (YP_003766); HCoV_NL63: Human coronavirus NL63; PEDV: Porcine epidemic diarrhea virus; Sc-BatCoV_512: Scotophilus bat coronavirus 512; Mi-BatCoV-1A: bat coronavirus 1A; Mi-BatCoV_HKU8: Miniopterus bat coronavirus HKU8. Medioniviridae: BlNV: Botrylloides leachii nidovirus; TurrNV: Turrinivirus 1; WFTV: Western Flower Thrips virus. Mesoniviridae: MenoV: Meno virus; NseV: Nse virus; HanaV: Hana virus; KSaV: Karang sari virus; DKNV: Dak Nong virus; CASV: Casuarina virus; NDiV: Nam Dinh virus; CavV: Cavally virus; DMV: Dianke mesonivirus. Roniviridae: PAN: Palaemon nidovirus; GAV: Gill-associated virus; YHV: Yellow head virus (EU487200). Euroniviridae: PagRV: Paguronivirus 1; CharNV: Charybnivirus 1; DecNV: Decronivirus 1. Abyssoviridae: AAbV: Aplysia abyssovirus 1 Mononiviridae: PSCNV: Planidovirus 1" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1087231205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Presence, type, and loci of signature-sequences of &lt;em&gt;Nidovirales&lt;/em&gt; RNA MTases : RF MT of unknown specificity (light red), non-RF &#xAB; nsp14-like &#xBB; MT (yellow), and RF &#xAB; nsp16-like &#xBB; MT (green). Both light red and yellow MTases map to the nsp14 C-ter locus, ie., immediately downstrean the ExoN domain, while &#xAB; nsp16-like &#xBB; MT map to the nsp16 locus, at the end of Orf1b. The tree was made based on MAFFT v7.427 multiple sequence alignment with BLOSUM62 scoring matrix and G-INS-i iterative refinement method. The alignments were used as input for maximum likelihood trees generated with the FasTtree v2.1.5 software (best-fit model = JTT-Jones-Taylor-Thorton with single rate of evolution for each site = CAT). Local support values were computed using the Shimodaira-Hasegawa test (SH) with 1,000 replicates. Numbers at the nodes represent FasTtree support values and scale var substitutions per site. The tree included two novel Ronivirus-like and Mesonivirus-like genome sequences: Western Flower Thrips Mesonivirus, and Palaemon Nidovirus, respectively (WFTV, PAN, unpublished). When present in the NCBI viral genomes database in the 92 &lt;em&gt;Nidovirales&lt;/em&gt; complete genomes repository (https://www.ncbi.nlm.nih.gov/genomes/GenomesGroup.cgi?taxid=76804), no accession number is indicated. When an accession number is given in parenthesis, it is referring to the GenBank accession number (https://www.ncbi.nlm.nih.gov/genbank/). From top to bottom of the figure: &lt;em&gt;Arteriviridae:&lt;/em&gt; HHPAV: Hainan hebius popei arterivirus (MG600021); NGSAV: Nanhai ghost shark arterivirus (MG600024); SHFV : simian hemorrhagic fever virus; EAV: Equine arteritis virus; LDV, Lactate elevating virus; PRRSV-1 and 2, porcine reproductive and respiratory syndrome virus. &lt;em&gt;Tobaniviridae:&lt;/em&gt; BToV: Breda virus; GToV: Goat torovirus; EToV: Berne virus (CAA36747); PToV: Porcine torovirus; CSV: Chinook salmon bafinivirus; FHMNV: Fathead Minnow nidovirus; WBV: white bream virus; BoNV: Bovine nidovirus TCH5; INToV: Xinzhou nido-like virus 6; SECToV; Xinzhou toro-like virus 1; GRBS-ToV: Guangdong red banded snake torovirus (MG600030); ShNV: shingleback nidovirus 1 (KX184715); BPNV: Ball python nidovirus 1; MVNV: Morelia viridis nidovirus; BRV: Bellinger River virus (MF685025); &lt;em&gt;Coronaviridae:&lt;/em&gt; IBV: Infectious Bronchitis Virus; BWCoV: Beluga Whale coronavirus SW1; HCoV_HKU1: Human coronavirus HK1; MHV: Mouse hepatitis virus; BatCoV_HKU9: Rousettus bat Coronavirus HKU9; SARS-CoV: Severe acute respiratory syndrome coronavirus; MERS_CoV: middle-east respiratory syndrome coronavirus; Ty-BatCoV_HKU4: Tylonycteris bat coronavirus HKU4; Pi-BatCoV_HKU5: pipistrellus bat coronavirus HKU5; FIPV: Feline infectious peritonitis virus; Rh-BatCoV-HKU2: Rhinolophus bat coronavirus HKU2; HCoV_229E: Human coronavirus 229E; HCoV_OC43: Human coronavirus OC43 (YP_003766); HCoV_NL63: Human coronavirus NL63; PEDV: Porcine epidemic diarrhea virus; Sc-BatCoV_512: Scotophilus bat coronavirus 512; Mi-BatCoV-1A: bat coronavirus 1A; Mi-BatCoV_HKU8: Miniopterus bat coronavirus HKU8. &lt;em&gt;Medioniviridae:&lt;/em&gt; BlNV: Botrylloides leachii nidovirus; TurrNV: Turrinivirus 1; WFTV: Western Flower Thrips virus. &lt;em&gt;Mesoniviridae&lt;/em&gt;: MenoV: Meno virus; NseV: Nse virus; HanaV: Hana virus; KSaV: Karang sari virus; DKNV: Dak Nong virus; CASV: Casuarina virus; NDiV: Nam Dinh virus; CavV: Cavally virus; DMV: Dianke mesonivirus. &lt;em&gt;Roniviridae:&lt;/em&gt; PAN: Palaemon nidovirus; GAV: Gill-associated virus; YHV: Yellow head virus (EU487200). &lt;em&gt;Euroniviridae&lt;/em&gt;: PagRV: Paguronivirus 1; CharNV: Charybnivirus 1; DecNV: Decronivirus 1. &lt;em&gt;Abyssoviridae&lt;/em&gt;: AAbV: Aplysia abyssovirus 1 &lt;em&gt;Mononiviridae&lt;/em&gt;: PSCNV: Planidovirus 1&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/23/639369/F1.medium.gif" width="397" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/23/639369/F1.medium.gif" width="397" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/23/639369/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/23/639369/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-5" class="first-child">Presence, type, and loci of signature-sequences of <em>Nidovirales</em> RNA MTases : RF MT of unknown specificity (light red), non-RF « nsp14-like » MT (yellow), and RF « nsp16-like » MT (green). Both light red and yellow MTases map to the nsp14 C-ter locus, ie., immediately downstrean the ExoN domain, while « nsp16-like » MT map to the nsp16 locus, at the end of Orf1b. The tree was made based on MAFFT v7.427 multiple sequence alignment with BLOSUM62 scoring matrix and G-INS-i iterative refinement method. The alignments were used as input for maximum likelihood trees generated with the FasTtree v2.1.5 software (best-fit model = JTT-Jones-Taylor-Thorton with single rate of evolution for each site = CAT). Local support values were computed using the Shimodaira-Hasegawa test (SH) with 1,000 replicates. Numbers at the nodes represent FasTtree support values and scale var substitutions per site. The tree included two novel Ronivirus-like and Mesonivirus-like genome sequences: Western Flower Thrips Mesonivirus, and Palaemon Nidovirus, respectively (WFTV, PAN, unpublished).</p><p id="p-6">When present in the NCBI viral genomes database in the 92 <em>Nidovirales</em> complete genomes repository (<a href="https://www.ncbi.nlm.nih.gov/genomes/GenomesGroup.cgi?taxid=76804">https://www.ncbi.nlm.nih.gov/genomes/GenomesGroup.cgi?taxid=76804</a>), no accession number is indicated. When an accession number is given in parenthesis, it is referring to the GenBank accession number (<a href="https://www.ncbi.nlm.nih.gov/genbank/">https://www.ncbi.nlm.nih.gov/genbank/</a>).</p><p id="p-7">From top to bottom of the figure:</p><p id="p-8"><em>Arteriviridae:</em></p><p id="p-9">HHPAV: Hainan hebius popei arterivirus (MG600021); NGSAV: Nanhai ghost shark arterivirus (MG600024); SHFV : simian hemorrhagic fever virus; EAV: Equine arteritis virus; LDV, Lactate elevating virus; PRRSV-1 and 2, porcine reproductive and respiratory syndrome virus.</p><p id="p-10"><em>Tobaniviridae:</em></p><p id="p-11">BToV: Breda virus; GToV: Goat torovirus; EToV: Berne virus (CAA36747); PToV: Porcine torovirus; CSV: Chinook salmon bafinivirus; FHMNV: Fathead Minnow nidovirus; WBV: white bream virus; BoNV: Bovine nidovirus TCH5; INToV: Xinzhou nido-like virus 6; SECToV; Xinzhou toro-like virus 1; GRBS-ToV: Guangdong red banded snake torovirus (MG600030); ShNV: shingleback nidovirus 1 (KX184715); BPNV: Ball python nidovirus 1; MVNV: Morelia viridis nidovirus; BRV: Bellinger River virus (MF685025);</p><p id="p-12"><em>Coronaviridae:</em></p><p id="p-13">IBV: Infectious Bronchitis Virus; BWCoV: Beluga Whale coronavirus SW1; HCoV_HKU1: Human coronavirus HK1; MHV: Mouse hepatitis virus; BatCoV_HKU9: Rousettus bat Coronavirus HKU9; SARS-CoV: Severe acute respiratory syndrome coronavirus; MERS_CoV: middle-east respiratory syndrome coronavirus; Ty-BatCoV_HKU4: Tylonycteris bat coronavirus HKU4; Pi-BatCoV_HKU5: pipistrellus bat coronavirus HKU5; FIPV: Feline infectious peritonitis virus; Rh-BatCoV-HKU2: Rhinolophus bat coronavirus HKU2; HCoV_229E: Human coronavirus 229E; HCoV_OC43: Human coronavirus OC43 (YP_003766); HCoV_NL63: Human coronavirus NL63; PEDV: Porcine epidemic diarrhea virus; Sc-BatCoV_512: Scotophilus bat coronavirus 512; Mi-BatCoV-1A: bat coronavirus 1A; Mi-BatCoV_HKU8: Miniopterus bat coronavirus HKU8.</p><p id="p-14"><em>Medioniviridae:</em></p><p id="p-15">BlNV: Botrylloides leachii nidovirus; TurrNV: Turrinivirus 1; WFTV: Western Flower Thrips virus.</p><p id="p-16"><em>Mesoniviridae</em>:</p><p id="p-17">MenoV: Meno virus; NseV: Nse virus; HanaV: Hana virus; KSaV: Karang sari virus; DKNV: Dak Nong virus; CASV: Casuarina virus; NDiV: Nam Dinh virus; CavV: Cavally virus; DMV: Dianke mesonivirus.</p><p id="p-18"><em>Roniviridae:</em></p><p id="p-19">PAN: Palaemon nidovirus; GAV: Gill-associated virus; YHV: Yellow head virus (EU487200).</p><p id="p-20"><em>Euroniviridae</em>:</p><p id="p-21">PagRV: Paguronivirus 1; CharNV: Charybnivirus 1; DecNV: Decronivirus 1.</p><p id="p-22"><em>Abyssoviridae</em>:</p><p id="p-23">AAbV: Aplysia abyssovirus 1</p><p id="p-24"><em>Mononiviridae</em>:</p><p id="p-25">PSCNV: Planidovirus 1</p><div class="sb-div caption-clear"></div></div></div><p id="p-26"><em>Nidovirales</em> have attracted much attention in the past decades for being causative agents of serious diseases in humans and animals. In humans, Severe Acute Respiratory Syndrome (SARS) and the Middle-East Respiratory Syndrome (MERS) coronaviruses (CoVs) have been associated with high fatality/case ratios of ~10 % ~35 %, respectively (reviewed in [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]). In animals, pathogens of significant economical and societal impact include Equine Arteritis virus (EAV, Alphaarterivirus equid) and Porcine Reproductive Respiratory Syndrome virus (PRSSV 1-2, Betaarterivirus suid 1-2).</p><p id="p-27">Two salient features of these viruses are, amongst others, their complex and still obscure replication/transcription mechanism as well as their broad genome-size range. Genomes of <em>Arteriviridae</em> (12.7 −15.7 kb in length), <em>Mesoniviridae</em> (~20 kb), <em>Medioniviridae</em> (20.2 – 25 kb), <em>Euroniviridae</em> (~24.5 kb), <em>Roniviridae</em> (~26 kb) <em>Tobaniviridae</em> and <em>Coronaviridae</em> (~27-32 kb), <em>Abyssoviridae</em> (35.9 kb), and <em>Mononiviridae</em> (41.1 kb) members, share a similar genomic organisation generating viral mRNAs from a fixed genomic location, the ‘nest’, which has inspired the family name (<em>Nido</em> : latin for ‘nest’) [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. <em>Nidovirales</em> genomic and mRNAs are presumably capped, although direct demonstration of the presence of an RNA cap structure is still missing for many of these <em>Nidovirales</em> members.</p><p id="p-28">In Eukaryotes, the RNA cap structure is thought to be synthesized post-transcriptionally [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. Most of (+)RNA viruses supposedly follow a conventional RNA capping pathway [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>] in which nascent viral 5’-triphosphate genomic (or mRNAs) is thought to be processed through three enzymatic reactions to yield an RNA cap whose structure is indistinguishable from that of cellular mRNAs. The capping pathway implies that the 5’-triphosphate RNA is hydrolyzed into 5’-diphosphate by an RNA triphosphatase, a GMP residue (the « cap », coming from GTP) is next covalently transferred to the 5’-diphosphate RNA in the 5’ to 5’ orientation by a guanlylyltransferase (GTase), releasing inorganic pyrophosphate. Both the cap and the first transcribed nucleotide are then methylated at the N7-guanine (mGpppN-RNA, so-called Cap0 structure) and the 2’-oxygen position (mGpppNm-RNA, Cap1 structure) by one or two RNA methyltransferases (MTases), respectively. The MTase uses S-Adenosyl-Methionine (SAM) as the methyl donor and releases S-Adenosyl-homocysteine (SAH).</p><p id="p-29">There is no high-resolution structural analysis available of <em>Nidovirales</em> RNA caps. The evidence for their presence comes from immunological detection in Torovirus genomic and mRNAs [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>], and co-migration analysis in low-resolution chromatographic techniques for MHV [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>] and Simian Hemorragic fever virus [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. In other instances, <em>Nidovirales</em> cap structures supposedly conform to the canonical eukaryotic RNA capping pathway because enzymes usually deemed necessary to perform RNA capping are present in cognate genomes (reviewed in [<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>]). Bioinformatic analyses have been used to detect signature-sequences typically associated with RNA capping/modifying enzymes <em>i.e</em> guanlylyltransferase (GTase), polymerase, helicase, nuclease, and methyltransferase (MTase). These signature-sequences have served in many instances to isolate corresponding enzymes endowed with the predicted activity. In <em>Nidovirales</em>, this approach has been pioneered for Coronavirus enzymes, yet regarding their RNA capping pathway, it only succeeded in predicting 2’OMTase, while a second non-canonical RNA-cap N7-guanine MTase was later structurally and functionally characterized [<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>–<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. Until then, viral MTases matched a specific structural architecture, the Rossmann fold (RF), which possess a unique interaction geometry that represents a fingerprint of common ancestry. The discovery of a non-Rossmann fold (NRF) MTase unveiled the limitation of annotating “dark proteome” data [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>], lacking structural counterparts. In <em>Nidovirales</em>, enzyme activities have been experimentally evidenced only for coronaviruses and roniviruses [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. The putative first step of the <em>Nidovirales</em> RNA capping pathway, involving a 5’-triphosphatase, is uncertain: such activity exists in the Coronavirus helicase (nsp13, [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>]), but its involvement in RNA capping has not been directly demonstrated. The guanylyltransferase also remains unknown: A nidovirus RdRp-associated nucleotidyltransferase (NiRAN) activity was found to be associated to the N-terminus of the arterivirus nsp9 RdRp [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>], but its role in arterivirus RNA synthesis, if any, remains to be determined.</p><p id="p-30">Despite the conservation of MTases amongst large <em>Nidovirales</em> genomes, their distribution, type, and the function of MTases in the <em>Nidovirales</em> order is not homogenous (summarized in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, see text). Arterivirus or arterivirus-like genomes are not known to carry any obvious MTase signature sequence; Mesonivirus and Ronivirus genomes code for one 2’-O RF-MTase (nsp16-like) signature-sequence; The <em>Tobaniviridae</em> (<em>e.g</em>., toroviruses and bafiniviruses, see <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>) carry a single 2’-O RF-MTase (nsp16-like) signature sequence, whereas <em>Coronaviridae</em> carry both a N7-guanine non-RF-MTase (NRF-MTase, nsp14-like) and a 2’-O RF-MTase (nsp16-like) signature-sequence. A single deviation for this MTase-based grouping is that of Fathead minnow nidovirus 1 (<em>Tobaniviridae</em>) which carries both MTases (see below) as in the case of <em>Coronaviridae</em>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/747087/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/747087/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <span class="caption-title">Distribution and putative substrate specificities of MTases in the four families of the <em>Nidovirales</em> Order.</span><p id="p-31" class="first-child">Only the <em>Tobaniviridae</em> is expanded and genera indicated in italic plus unassigned viruses (see <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1</a>). *: Arteriviruses do not usually carry any MTase, except the two newly identified Hainan Hebius Popei arterivirus and Nanhai gost shark arterivirus (see text).**: although in the nsp14 locus immediately downstream the ExoN gene, the MTase has a Rossmann fold. Virus species names are indicated and genome size corresponds to exemplar virus sequence used in this study.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">The Coronavirus nsp14 is a peculiar, bi-functional enzyme combining in N-terminus a 3’-to-5’ exonuclease domain (ExoN) fused to the N7-guanine NRF-MTase domain in C-terminus [<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>,<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]. First, it is the only known example of a viral SAM-dependent RNA MTase (and more generally, a unique example of an RNA-Cap 0 N7-guanine MTase) that does not have a Rossman fold [<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>]. Since the Rossman fold is extremely widely distributed in all kingdoms of life, why this NRF-MTase fold has appeared or been conserved through evolution is puzzling. Second, the ExoN activity of nsp14 is a marker of large RNA genomes [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. The ExoN domain confers capability of RNA mismatch repair to the replicative complex, ensuring genetic stability [<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>,<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>].</p><p id="p-33">There is still a significant gap of general knowledge on <em>Nidovirales</em> RNA capping. For example, <em>i</em>) the guanylyltransferase step is still elusive in the whole order; <em>ii</em>) if any cap exists in <em>Arteriviridae</em>, how is RNA cap methylation achieved ? and <em>iii</em>) how is the cap N7-guanine methylated in <em>Tobaniviridae</em> and <em>Roniviridae</em>, since there is no N7-guanine MTase clearly identified in these families?</p><p id="p-34">In this paper, we report the discovery of an RNA-cap N7-guanine RF-MTase sequence in the Orf1a of ten members of the <em>Tobaniviridae</em> family. Remarkably, within this family, this MTase is absent from mammalian toroviruses, but may well represent a genetic marker of non-mammalian <em>Tobaniviridae</em>.</p></div><div class="section" id="sec-2"><h2 class="">Material and Methods</h2><p id="p-35">Virus genome sequences were retrieved from the NCBI database (<a href="https://www.ncbi.nlm.nih.gov/genomes/GenomesGroup.cgi?taxid=439488">https://www.ncbi.nlm.nih.gov/genomes/GenomesGroup.cgi?taxid=439488</a>) or Genbank (ncbi.nlm.nih.gov/genbank). The data set (given in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) was that of Lauber <em>et al</em>. [<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>], to which novel or recently described genome sequences (retrieved in NCBI of GenBank) were added purposely. Additionally, the putative and unpublished Palaemon nidovirus and Western flower thrips nidovirus, and the proposed Botrylloides leachii nidovirus [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>], were identified in the Sequence Read Archive (SRA) runs SRR5658389, SRR492945 and SRR2729873, respectively, and the virus sequences were assembled, curated and annotated as described elsewhere [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. The resulting sequences are available upon request to H.D.</p><p id="p-36">The conserved Nidovirales C-terminus RdRp core was used to build phylogenetic trees based on MAFFT v7.427 multiple sequence alignment with BLOSUM62 scoring matrix and G-INS-i iterative refinement method. The alignments were used as input for maximum likelihood trees generated with the FasTtree v2.1.5 software (best-fit model = JTT-Jones-Taylor-Thorton with single rate of evolution for each site = CAT). Local support values were computed using the Shimodaira-Hasegawa test (SH) with 1,000 replicates.</p><p id="p-37">Structural alignment of reference or retrieved Mtases were done using EXPRESSO [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>] and proofed with Chimera [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. N7-Mtases reference motifs were inferred and defined based on visual inspection and analysis of the structures. The subsequent motifs were used as orthogonal validation and not used for the search.</p><p id="p-38">Sequences were analyzed using HHblits and HHpred tools of the BioInformatics tookit ([<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]). When available, authentic cleavage sites were used to predict protein gene products of the Orf1ab, Orf1a, and Orf1b polyproteins. The boundaries were otherwise approximately (+/-10 aa) determined using structural homologies detected using HHPred, except for the N-term boundary of the Orf1b gene product: In <em>Nidovirales</em>, the absence of any structural data nor homology (outside the order) on the N-terminus of the RdRp gene (nsp9 <em>Arteriviridae</em>, nsp12 in <em>Coronaviridae</em>), which was used for phylogenic analysis, precludes precise sequence homology search in this limited area comprised between the nsp10 and nsp12 proteins (Coronavirus gene-product naming).</p><p id="p-39">Multiple sequence alignments (MSA) were generated using Muscle in SeaView [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]. For each sequence of unknown structure, secondary structures were predicted using Predict Protein [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>]. The predicted secondary structures were used to validate the alignment with structural references. The MSA was rendered using ESPript 3.0 [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>], together with appropriate structural models as indicated, to assign secondary structures.</p><p id="p-40">When possible, structural 3D models were generated using Phyre 2.0 [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>]. Conserved patches of amino-acids were generated using WebLogo [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>] and mapped in the structural models rendered in Chimera.</p></div><div class="section" id="sec-3"><h2 class="">Results</h2><div id="sec-4" class="subsection"><h3>All non-arterivirus Nidovirales members carry at least one 2’O-MTase</h3><p id="p-41">Available <em>Nidovirales</em> genome sequences were used to analyze the presence and primary structure of RNA MTases. Any RF-MTase signature sequence is detectable mainly due to the presence of a typical G-X-G-(X)n-G element part of a SAM-binding motif [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>,<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>], whereas the detection of viral NRF-MTases is based on its homology to the MTase domain of Coronavirus nsp14. We made use of the conserved RdRp fold to build a phylogenetic tree along the whole order. From this tree, we first determined that most of <em>Nidovirales</em> code for at least one RF-MTase protein with the canonical K-D-K E catalytic tetrad of 2’O MTase (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig.1</a> green &amp; <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Conversely, as previously reported, this RF-2’O MTase is lacking in EAV, LDV, and PRRSV in the <em>Arteriviridae</em> family. Interestingly, we identified two RF-MTase signature sequences in the genomes of two recently identified arteri-like viruses : the Hainan Hebius Popei Arterivirus (HHPAV, 12,496 nt) and the Nanhai gost shark arterivirus (NGSAV, 13,162 nt). Their genomes carry a <em>bona fide</em> RF-2’-O-MTase signature sequence at the end of their Orf1b, in the gene order RdRp-Hel-(ExoN*)-EndoU-2’OMTase. The NGSAV does not carry a detectable ExoN signature sequence, whereas the HHPAV genome carries a nsp14-like ExoN domain coding region but not a detectable C-terminus NRF-MTase domain, hence the ExoN* labeling (Ferron, Decroly &amp; Canard, unpublished). All these RF-MTase with the K-D-K-E signature are localized in a conserved genomic position at the 3’ end of the Orf1ab, much like their CoV and nsp16 homologues. We performed a MSA of nsp16 from <em>Roniviridae</em> and <em>Tobaniviridae</em> (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1</a>, in green), followed by modeling a typical representative of these nsp16 (not shown). All these nsp16-like enzymes appear to be predicted as canonical RNA 2’-O MTases with a typical K-D-K-E tetrad. As noted by others in the Ronivirus nsp16 model [<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>], small structural differences are observed across the <em>Tobaniviridae</em> family, such as the absence of β3 strand and a shorter loop upstream helix αD. Thus, we conclude that non-arterivirus <em>Nidovirales</em> code for a RF-MTase with the canonical K-D-K E catalytic tetrad of 2’-O MTase, and that these enzymes are distributed in similar position along genomes.</p></div><div id="sec-5" class="subsection"><h3>The distribution of NRF-MTase is uneven along Nidovirales genomes</h3><p id="p-42">As indicated above, many <em>Nidovirales</em> members, such as SARS-CoV, stand out because they possess a NRF-MTase, participating to cap N7-guanine methylation, embedded at the C-terminus of their nsp14 gene product. It have been previously reported that the N7-guanine-MTase domain is not uniformly present in nsp14-containing nidoviruses [<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">21</a>]. The phylogenetic tree of <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a> reports our findings, and confirms that coronaviruses and most mesoniviruses only possess signature-sequences lining the SAM binding site seen in the nsp14 MTase domain structure. Conversely, the other members of the <em>Nidovirales</em> order (<em>Arteriviridae, Medionivirdae, Roniviridae, Euroniviridae, Abyssoviridae, Mononiviridae</em> and <em>Tobaniviridae</em>) do not seem to embed any nsp14-like NRF-MTase. Two notable exceptions are apparent: First, the unique members of <em>Abyssoviridae, Mononiviridae</em> do possess a MTase signature-sequence at the C-terminus of their nsp14-like gene, (i.e., fused to the ExoN domain). Curiously, though, this MTase is readily detectable using HH-Pred as a RF-MTase which does qualify neither as a K-D-K-E 2’-O MTase nor as a N7-guanine MTase (see below). Second, the Fathead Minnow nidovirus 1, belonging to the <em>Tobaniviridae</em>, is also an exception as we detected a nsp14-like NRF MTase at the expected genomic Orf1b position. Against, based on the homology with the nsp14-like MTase domain characterized by the absence of K-D-K-E catalytic residue and NRF folding, we hypothesize that these enzymes might play a role in guanine N7 methylation of the RNA cap structure.</p><p id="p-43">Apart from coronaviruses and most mesoniviruses, how nidoviruses methylate their RNA-cap N7-guanine position is thus still unclear.</p></div><div id="sec-6" class="subsection"><h3>What defines N7-guanine MTase features in Nidovirales?</h3><p id="p-44">Unlike 2’-O MTases and their well-defined K-D-K-E tetrad, and apart from NRF nsp14-like MTases, signature sequences of RNA cap N7-guanine cap N7-guanosine MTases are much less obvious. The wealth of structural data and mechanistic insight on the N7-guanine MTases enzymes has been acquired from the microsporidian parasite <em>Encephalitozoon cuniculi</em> [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>], the poxvirus MTase D1:D12 heterodimer [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>] and <em>Reoviridae</em> [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>,<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>] in the case of RNA viruses. There are only three crystal structure of RF-MTase known to methylate N7-guanine RNA caps of RNA viruses : those of <em>Reoviridae</em> (Rotavirus VP4 and Reovirus Lambda2), and Flavivirus, whose NS5 N-terminus domain is a bi-functional N7-guanine and 2’O-MTase [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>]. Taking together these data, and combining structural information with the human RNA N7-guanine methyltransferase crystal structure (PDB ID: 3BGV), one cannot define a specific N7-guanine MTase signature (Fig. S1).</p><p id="p-45">From this analysis and others [<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>] it is clear that the structure conservation prevailed over sequence conservation. No amino acid motifs typical of an RNA N7-guanine MTase can be clearly identified (Fig. S1). A narrower structural comparison including only <em>Encephalitozoon cuniculi</em>, poxvirus and the human RNA N7-guanine MTase allows detection of 6 amino acids or motifs K / G/D/ HY / E / Y (Fig. S2). Typical SAM binding motifs are defined by a combination of 3 motifs placed at specific structural positions: K within an α helix, followed by a GxGxG motif within a loop and conserved D. The 3 latter motifs are located in the MTase structural pocket where the N7 methylation supposedly occurs.</p><p id="p-46">In order to use an independent method, we performed a structural search using HHpred on PDB and SCOPe databases [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>], using each sequence as a query. This allowed to refine the boundaries of the MTase domain. The structure with the closest hit is a SAM-dependent MTase from <em>Pectobacterium atrosepticum</em> and then <em>Encephalitozoon cuniculi</em>. A quick superimposition of the retrieved structures with N7-guanine MTase shows that these structures can be structurally aligned but with very limited sequence conservation. Surprisingly, apart from a conserved leucine residue 2 amino acid upstream the poorly conserved SAM binding motif (only the third G is strictly conserved) and a glycine-rich motifs in the fifth beta sheet, nothing else is conserved nor defines an active site (Fig. S3).</p><p id="p-47">Taking these criteria into account, there was no obvious indication of RF N7-guanine MTase features in <em>Nidovirales</em> other than the two mentioned above in <em>Abyssoviridae</em> and <em>Mononiviridae</em>. Currently, based on the detection of RF-MTase fold, one can infer a 2’ O-MTase sequence signature when the K-D-K-E tetrad is present, but not a RNA-cap N7-guanine MTase.</p></div><div id="sec-7" class="subsection"><h3>Selected Tobaniviridae members possess a RF-MTase signature-sequence in Orf1a lacking the canonical 2’O catalytic K-D-K-E tetrad</h3><p id="p-48">Then, how do <em>Roniviridae</em> and <em>Tobaniviridae</em> methylate the RNA-cap N7-guanine position? One possibility would be that these nsp16-like were bi-functional, carrying both 2’-O and N7-guanine methylation activity such as evidenced in Flavivirus NS5 protein, with no obvious signature sequence of this bi-functionality. Another possibility would be that another gene would code for an enzyme performing this methylation, and had escaped detection by standard bioinformatic methods.</p><p id="p-49">We thus performed a larger search of MTase signature-sequences along the whole Orf1ab in all <em>Nidovirales</em>: surprisingly, a RF-MTase signature-sequence was detected in Orf1a of 10 members of the <em>Tobaniviridae</em> family (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). Strictly conserved amino-acids in these new viral MTases define three motifs: 3 glycines of the SAM binding site (G54, 56, and 58 in WBV) 2 residues downstream of a 3 amino acid hydrophobic patches in a β-strand structure, a histidine (H117 immediately followed by either F or Y, and Glu175 (Fig. S4). Interestingly the catalytic K-D-K-E tetrad associated to 2’O-MTase is lacking, suggesting that they could correspond to the missing RNA-cap N7-guanine MTase.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/23/639369/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of the Orf1a MTase along the A) Tobaniviridae phylogentic tree (see Fig. 1). B) approximate genome organisation and gene content along the family described in A. The genomes are not drawn to scale. Only the Bostovirus (Remotovirinae) genome is represented smaller than its fellow members to account for its ~20 kb genome size." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1087231205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of the Orf1a MTase along the A) &lt;em&gt;Tobaniviridae&lt;/em&gt; phylogentic tree (see Fig. 1). B) approximate genome organisation and gene content along the family described in A. The genomes are not drawn to scale. Only the Bostovirus (&lt;em&gt;Remotovirinae&lt;/em&gt;) genome is represented smaller than its fellow members to account for its ~20 kb genome size.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/23/639369/F2.medium.gif" width="436" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/23/639369/F2.medium.gif" width="436" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/23/639369/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/23/639369/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-50" class="first-child">Distribution of the Orf1a MTase along the A) <em>Tobaniviridae</em> phylogentic tree (see <a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1</a>). B) approximate genome organisation and gene content along the family described in A. The genomes are not drawn to scale. Only the Bostovirus (<em>Remotovirinae</em>) genome is represented smaller than its fellow members to account for its ~20 kb genome size.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">Remarkably, the novel Orf1a MTase gene was associated with reptiles and fish. Nonetheless, in two cases (Xinzhou Nematode virus 6 and Xinzhou toro-like virus 1, from the <em>Sectovirus 1</em> and <em>Infratovirus 1</em> species respectively), the viruses were isolated from snake-associated nematodes, although their proper host and life style has not been clearly assessed. The mammalian toroviruses (EToV, BToV, Bovine TCH5 nidovirus, GToV and PToV) do not carry the MTase signature sequence, and thus the latter is associated so far to non-mammalian <em>Tobaniviridae</em>.</p></div><div id="sec-8" class="subsection"><h3>The RF-MTase signature-sequence in Orf1a is a putative RNA cap N7-guanine MTase</h3><p id="p-52">We performed a structure-based alignment of this Orf1a MTase with the integration of several structurally-defined eukaryotic RNA-cap N7-guanine MTases (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a>), of which Ecm1 can be considered as prototypic [<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>]. The conserved amino-acids in these new MTase set remain essentially the same: two glycines of the SAM binding site (G54 and 56 in WBV, G74-76 in Ecm1), a histidine (H117 in WBV, H144 in Ecm1) immediately followed by either F or Y. Glu175 is also very conserved, and in the Ecm1 structure, its homosteric counterpart Glu225 is positioned close to His144 and interacts also with the guanine base, strongly suggesting that this MTase is an N7-guanine MTase. The alignment allows to define 5 conserved motifs which are represented in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3B</a>, and structurally aligned onto the vaccinia N7-guanine MTase structure D12 (PDB: 4KCB).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/23/639369/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A) Structural alignment of Tobaniviridae methyltransferases together with various prokaryotic, eukaryotic and viral members; B) Vaccinia Virus N7-guanine methyltransferases structure (PDB : 4CKB) with highlighted motifs 1 to 5 and their corresponding amino-acid frequencies as determined by WebLogo analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1087231205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A) Structural alignment of &lt;em&gt;Tobaniviridae&lt;/em&gt; methyltransferases together with various prokaryotic, eukaryotic and viral members; B) Vaccinia Virus N7-guanine methyltransferases structure (PDB : 4CKB) with highlighted motifs 1 to 5 and their corresponding amino-acid frequencies as determined by WebLogo analysis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/05/23/639369/F3.medium.gif" width="261" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/05/23/639369/F3.medium.gif" width="261" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/23/639369/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/05/23/639369/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-53" class="first-child">A) Structural alignment of <em>Tobaniviridae</em> methyltransferases together with various prokaryotic, eukaryotic and viral members; B) Vaccinia Virus N7-guanine methyltransferases structure (PDB : 4CKB) with highlighted motifs 1 to 5 and their corresponding amino-acid frequencies as determined by WebLogo analysis.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-9"><h2 class="">Discussion</h2><p id="p-54">RNA virus cap-MTases perform two types of reactions: the methylation of the N7-guanine of the RNA cap, and the methylation of the adenosine 2’-O ribose of the first transcribed nucleotide [<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a>]. Regarding the guanine methylation, this reaction can also be achieved on a GTP molecule before capping, in some virus families, and the N7-methyl GTP is then used to cap the RNA [<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>]. This is the case with the alphavirus nsp1 enzyme, whose structure is currently unknown. Conversely, several crystal structures of viral enzymes involved in N7-guanine methylation of RNA caps have been reported, such as that of flaviviruses (NS5, [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]), rotaviruses and reoviruses (VP4 and lambda2, respectively [<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>,<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a>]), and coronaviruses (nsp14, [<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>]).</p><p id="p-55">The viral 2’-O MTases have been better defined than N7-guanine MTases at the structural and functional level. They all use a K-D-K-E catalytic tetrad, and crystal structures have been determined for several virus groups, such as Reovirus, Flavivirus, Coronavirus, and <em>Mononegavirales</em> (reviewed in [<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">6</a>], [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>]). All these enzymes (N7-and 2’O–MTase), with SARS-CoV nsp14 as a notable exception (see below), belong to the Rossmann-fold (RF) type MTase (reviewed in [<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>,<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">35</a>]). They bind the methyl donor (SAM), and are characterized by several structural features: their Rossmann fold is a seven-strands β-sheet surrounded with 6 α-helices; the seventh β-strand is inserted in an anti-parallel orientation between the 5th and 6th strand. The SAM cofactor binds to the first structural motif βαβ, which bears the amino acid sequence motif G-x-G-(x)<sub>n</sub>-G. The RF is an evolutionary ancient fold, which has been widely evolved to perform a variety of chemical reactions. Its structural plasticity is well illustrated by the Flavivirus NS5 MTase, which is able to perform both N7-guanine and 2’-O ribose methylation with the same ~33 kDa domain fused at the N-terminus of the viral RdRp domain [<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>].</p><p id="p-56">The presence of a MTase in <em>Nidovirales</em> Orf1a is unexpected, and had not been reported before, although signature sequences of an uncharacterized MTase had been detected in Ball Python nidovirus, Sectovirus 1 and Infratovirus 1 [<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>]. Both protein and enzyme functions in <em>Nidovirales</em> have been classified into a functional triangle [<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">21</a>], each side of the triangle representing a carrier of the following functions: Orf1a: host defence; Orf1b: genome maintenance; Orfs at the 3’-end: structural products. In this triangle, the MTase-type of activity involved in genome maintenance would usually map to Orf1b [<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">21</a>]. Perhaps indeed this MTase is involved in tasks other than RNA capping, which remain to be determined. Accordingly, it was recently reported that viral or cellular MTases can be recruited by viruses in order to induce internal methylation of their genome [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>] and escape to the antiviral response mediated by MDA5 [<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>]. Out of the gain of this (still putative) enzyme activity in this virus group, the copy number of Orf1a products is expected to be 3-6 time higher than those of Orf1b, raising the following questions: why only these <em>Tobaniviridae</em> members need such additional MTase ? Why do they need it in high amounts, provided that the Orf1a stoichiometry is thought to be higher than that of Orf1b products? Are activity levels provided by this enzyme low? What is its actual substrate specificity?</p><p id="p-57">Also, it is legitimate to question the specificity of single copy MTases in <em>Nidovirales</em>, such as mammalian <em>Tobaniviridae</em> or <em>Roniviridae</em> (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>). Since they do not have any obvious RNA-cap N7-guanine MTase, how is RNA-cap methylation achieved, if any? Are their nsp16-like enzymes of broad or promiscuous specificity?</p><p id="p-58">Our work also uncovers an unexpected 2’-O MTase distribution in viruses seemingly blurring the contour of <em>Arteriviridae</em>, a discovery that, in turn, may have a significant impact on our understanding of genome size evolution in <em>Nidovirales</em>. Two newly identified arteri-like virus genomes (HHPA and NGSA) provide food for thought regarding RNA genome evolution: Phylogenetic branching of both HHPA and NGSA suggests that arteriviruses might not be primitive small version of larger Coronavirus genomes, but may rather originate from size-reduction of a large Nidovirus ancestor genome.</p><p id="p-59">In any case, the type, diversity, and distribution of RNA MTases along the <em>Nidovirales</em> deserves a closer look : SAM-dependent MTases are ancient folds associated with RNA stability and evolution [<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>]. Their presence and properties in a phylogenetic tree may well give interesting clues regarding RNA genome evolution and its associated issue of host defense mechanisms.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-60">This work was supported by the European Union’s Horizon 2020 research and innovation program through the ANTIVIRALS project under the Marie Sklodowska-Curie grant agreement N° 642434.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-3">An improved definition of the occurence of Non-Rossmann fold MTase has been corrected. Nsp14-like N7-guanine MTase are unique in that they are the only example in the viral world, in addition of being the sole example of Non-Rossmann fold RNA-Cap0 Mtases.
This has been corrected throughout the ms.</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.1" data-doi="10.1007/s00705-018-04136-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Siddell</span>  <span class="cit-name-given-names">SG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lefkowitz</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mushegian</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dutilh</span>  <span class="cit-name-given-names">BE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrach</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Junglen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Additional changes to taxonomy ratified in a special vote by the International Committee on Taxonomy of Viruses (October 2018)</span>. <abbr class="cit-jnl-abbrev">Arch Virol</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">164</span>:<span class="cit-fpage">943</span>–<span class="cit-lpage">946</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArch%2BVirol%26rft.volume%253D164%26rft.spage%253D943%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-018-04136-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-018-04136-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">C</span></span>: <span class="cit-article-title">From SARS to MERS, Thrusting Coronaviruses into the Spotlight</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">11</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.3" data-doi="10.1016/j.virusres.2006.01.017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>: <span class="cit-article-title">Nidovirales: evolving the largest RNA virus genome</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">117</span>:<span class="cit-fpage">17</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DGorbalenya%26rft.auinit1%253DA.%2BE.%26rft.volume%253D117%26rft.issue%253D1%26rft.spage%253D17%26rft.epage%253D37%26rft.atitle%253DNidovirales%253A%2Bevolving%2Bthe%2Blargest%2BRNA%2Bvirus%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2006.01.017%26rft_id%253Dinfo%253Apmid%252F16503362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2006.01.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16503362&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236746000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.4" data-doi="10.1016/S0022-2836(03)00865-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bredenbeek</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dobbe</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thiel</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poon</span>  <span class="cit-name-given-names">LLM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rozanov</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spaan</span>  <span class="cit-name-given-names">WJM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>: <span class="cit-article-title">Unique and conserved features of genome and proteome of SARS-coronavirus, an early split-off from the coronavirus group 2 lineage</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-pub-date">2003</span>, <span class="cit-vol">331</span>:<span class="cit-fpage">991</span>–<span class="cit-lpage">1004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DSnijder%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D331%26rft.issue%253D5%26rft.spage%253D991%26rft.epage%253D1004%26rft.atitle%253DUnique%2Band%2Bconserved%2Bfeatures%2Bof%2Bgenome%2Band%2Bproteome%2Bof%2BSARS-coronavirus%252C%2Ban%2Bearly%2Bsplit-off%2Bfrom%2Bthe%2Bcoronavirus%2Bgroup%2B2%2Blineage.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252803%252900865-9%26rft_id%253Dinfo%253Apmid%252F12927536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-2836(03)00865-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12927536&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185036400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.5" data-doi="10.1093/nar/gkw551"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ramanathan</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robb</span>  <span class="cit-name-given-names">GB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">S-H</span></span>: <span class="cit-article-title">mRNA capping: biological functions and applications</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">44</span>:<span class="cit-fpage">7511</span>–<span class="cit-lpage">7526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw551%26rft_id%253Dinfo%253Apmid%252F27317694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27317694&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.6" data-doi="10.1038/nrmicro2675"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Decroly</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferron</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lescar</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canard</span>  <span class="cit-name-given-names">B</span></span>: <span class="cit-article-title">Conventional and unconventional mechanisms for capping viral mRNA</span>. <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr> <span class="cit-pub-date">2011</span>, <span class="cit-vol">10</span>:<span class="cit-fpage">51</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BMicrobiol%26rft.volume%253D10%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2675%26rft_id%253Dinfo%253Apmid%252F22138959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2675&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22138959&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.7" data-doi="10.1093/emboj/cdf635"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Vliet</span>  <span class="cit-name-given-names">ALW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smits</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rottier</span>  <span class="cit-name-given-names">PJM</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Groot</span>  <span class="cit-name-given-names">RJ</span></span>: <span class="cit-article-title">Discontinuous and non-discontinuous subgenomic RNA transcription in a nidovirus</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr> <span class="cit-pub-date">2002</span>, <span class="cit-vol">21</span>:<span class="cit-fpage">6571</span>–<span class="cit-lpage">6580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253Dvan%2BVliet%26rft.auinit1%253DA.%2BL.%2BW.%26rft.volume%253D21%26rft.issue%253D23%26rft.spage%253D6571%26rft.epage%253D6580%26rft.atitle%253DDiscontinuous%2Band%2Bnon-discontinuous%2Bsubgenomic%2BRNA%2Btranscription%2Bin%2Ba%2Bnidovirus%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252Fcdf635%26rft_id%253Dinfo%253Apmid%252F12456663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjEvMjMvNjU3MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzIzLzYzOTM2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patton</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stohlman</span>  <span class="cit-name-given-names">SA</span></span>: <span class="cit-article-title">Further characterization of mRNA’s of mouse hepatitis virus: presence of common 5’-end nucleotides</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-pub-date">1982</span>, <span class="cit-vol">41</span>:<span class="cit-fpage">557</span>–<span class="cit-lpage">565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLai%26rft.auinit1%253DM%2BM%26rft.volume%253D41%26rft.issue%253D2%26rft.spage%253D557%26rft.epage%253D565%26rft.atitle%253DFurther%2Bcharacterization%2Bof%2BmRNA%2527s%2Bof%2Bmouse%2Bhepatitis%2Bvirus%253A%2Bpresence%2Bof%2Bcommon%2B5%2527-end%2Bnucleotides.%26rft_id%253Dinfo%253Apmid%252F6281467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjQxLzIvNTU3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMjMvNjM5MzY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.9" data-doi="10.1016/0042-6822(86)90113-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sagripanti</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zandomeni</span>  <span class="cit-name-given-names">RO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weinmann</span>  <span class="cit-name-given-names">R</span></span>: <span class="cit-article-title">The cap structure of simian hemorrhagic fever virion RNA</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-pub-date">1986</span>, <span class="cit-vol">151</span>:<span class="cit-fpage">146</span>–<span class="cit-lpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DSagripanti%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D151%26rft.issue%253D1%26rft.spage%253D146%26rft.epage%253D150%26rft.atitle%253DThe%2Bcap%2Bstructure%2Bof%2Bsimian%2Bhemorrhagic%2Bfever%2Bvirion%2BRNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252886%252990113-3%26rft_id%253Dinfo%253Apmid%252F2421482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(86)90113-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2421482&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.10" data-doi="10.1128/JVI.00407-08"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Decroly</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Imbert</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coutard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bouvet</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Selisko</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canard</span>  <span class="cit-name-given-names">B</span></span>: <span class="cit-article-title">Coronavirus nonstructural protein 16 is a cap-0 binding enzyme possessing (nucleoside-2’O)-methyltransferase activity</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-pub-date">2008</span>, <span class="cit-vol">82</span>:<span class="cit-fpage">8071</span>–<span class="cit-lpage">8084</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00407-08%26rft_id%253Dinfo%253Apmid%252F18417574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Mi8xNi84MDcxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMjMvNjM5MzY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.11" data-doi="10.1073/pnas.1718806115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ferron</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Subissi</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silveira De Morais</span>  <span class="cit-name-given-names">AT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">NTT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sevajol</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gluais</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Decroly</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vonrhein</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bricogne</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Structural and molecular basis of mismatch correction and ribavirin excision from coronavirus RNA</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">115</span>:<span class="cit-fpage">E162</span>–<span class="cit-lpage">E171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1718806115%26rft_id%253Dinfo%253Apmid%252F29279395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTE1LzIvRTE2MiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzIzLzYzOTM2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.12" data-doi="10.1073/pnas.1508686112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lou</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">Z</span></span>: <span class="cit-article-title">Structural basis and functional analysis of the SARS coronavirus nsp14-nsp10 complex</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">112</span>:<span class="cit-fpage">9436</span>–<span class="cit-lpage">9441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1508686112%26rft_id%253Dinfo%253Apmid%252F26159422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzMwLzk0MzYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8yMy82MzkzNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">13.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-639369v2.13" data-doi="10.1371/journal.ppat.1002294"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ke</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tien</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Biochemical and structural insights into the mechanisms of SARS coronavirus RNA ribose 2’-O-methylation by nsp16/nsp10 protein complex</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-pub-date">2011</span>, <span class="cit-vol">7</span>:<span class="cit-fpage">e1002294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DChen%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002294%26rft.epage%253De1002294%26rft.atitle%253DBiochemical%2Band%2Bstructural%2Binsights%2Binto%2Bthe%2Bmechanisms%2Bof%2BSARS%2Bcoronavirus%2BRNA%2Bribose%2B2%2527-O-methylation%2Bby%2Bnsp16%252Fnsp10%2Bprotein%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002294%26rft_id%253Dinfo%253Apmid%252F22022266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22022266&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.14" data-doi="10.1371/journal.ppat.1002059"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Decroly</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Debarnot</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferron</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bouvet</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coutard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Imbert</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gluais</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Papageorgiou</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharff</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bricogne</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Crystal structure and functional analysis of the SARS-coronavirus RNA cap 2’-O-methyltransferase nsp10/nsp16 complex</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-pub-date">2011</span>, <span class="cit-vol">7</span>:<span class="cit-fpage">e1002059</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DDecroly%26rft.auinit1%253DE.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De1002059%26rft.epage%253De1002059%26rft.atitle%253DCrystal%2Bstructure%2Band%2Bfunctional%2Banalysis%2Bof%2Bthe%2BSARS-coronavirus%2BRNA%2Bcap%2B2%2527-O-methyltransferase%2Bnsp10%252Fnsp16%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002059%26rft_id%253Dinfo%253Apmid%252F21637813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21637813&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.15" data-doi="10.1073/pnas.1508380112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Perdigão</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heinrich</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stolte</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sabir</span>  <span class="cit-name-given-names">KS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buckley</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tabor</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Signal</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gloss</span>  <span class="cit-name-given-names">BS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hammang</span>  <span class="cit-name-given-names">CJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rost</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Unexpected features of the dark proteome</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">112</span>:<span class="cit-fpage">15898</span>–<span class="cit-lpage">15903</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1508380112%26rft_id%253Dinfo%253Apmid%252F26578815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzUyLzE1ODk4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMjMvNjM5MzY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.16" data-doi="10.1128/JVI.00658-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Identification and Characterization of a Ribose 2’-O-Methyltransferase Encoded by the Ronivirus Branch of Nidovirales</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">90</span>:<span class="cit-fpage">6675</span>–<span class="cit-lpage">6685</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00658-16%26rft_id%253Dinfo%253Apmid%252F27170751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI5MC8xNS82Njc1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMjMvNjM5MzY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.17" data-doi="10.1128/JVI.78.14.7833-7838.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ivanov</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>: <span class="cit-article-title">Human coronavirus 229E nonstructural protein 13: characterization of duplex-unwinding, nucleoside triphosphatase, and RNA 5’-triphosphatase activities</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-pub-date">2004</span>, <span class="cit-vol">78</span>:<span class="cit-fpage">7833</span>–<span class="cit-lpage">7838</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DIvanov%26rft.auinit1%253DK.%2BA.%26rft.volume%253D78%26rft.issue%253D14%26rft.spage%253D7833%26rft.epage%253D7838%26rft.atitle%253DHuman%2BCoronavirus%2B229E%2BNonstructural%2BProtein%2B13%253A%2BCharacterization%2Bof%2BDuplex-Unwinding%252C%2BNucleoside%2BTriphosphatase%252C%2Band%2BRNA%2B5%2527-Triphosphatase%2BActivities%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.14.7833-7838.2004%26rft_id%253Dinfo%253Apmid%252F15220459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3OC8xNC83ODMzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMjMvNjM5MzY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.18" data-doi="10.1093/nar/gkv838"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lehmann</span>  <span class="cit-name-given-names">KC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gulyaeva</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zevenhoven-Dobbe</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Janssen</span>  <span class="cit-name-given-names">GMC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruben</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Overkleeft</span>  <span class="cit-name-given-names">HS</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Veelen</span>  <span class="cit-name-given-names">PA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Samborskiy</span>  <span class="cit-name-given-names">DV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kravchenko</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leontovich</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Discovery of an essential nucleotidylating activity associated with a newly delineated conserved domain in the RNA polymerase-containing protein of all nidoviruses</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">43</span>:<span class="cit-fpage">8416</span>–<span class="cit-lpage">8434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv838%26rft_id%253Dinfo%253Apmid%252F26304538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv838&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26304538&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.19" data-doi="10.1073/pnas.0508200103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Minskaia</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hertzig</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campanacci</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cambillau</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>: <span class="cit-article-title">Discovery of an RNA virus 3’-&gt;5’ exoribonuclease that is critically involved in coronavirus RNA synthesis</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">103</span>:<span class="cit-fpage">5108</span>–<span class="cit-lpage">5113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0508200103%26rft_id%253Dinfo%253Apmid%252F16549795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAzLzEzLzUxMDgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8yMy82MzkzNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.20" data-doi="10.1073/pnas.1201130109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bouvet</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Imbert</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Subissi</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gluais</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Decroly</span>  <span class="cit-name-given-names">E</span></span>: <span class="cit-article-title">RNA 3’-end mismatch excision by the severe acute respiratory syndrome coronavirus nonstructural protein nsp10/nsp14 exoribonuclease complex</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-pub-date">2012</span>, <span class="cit-vol">109</span>:<span class="cit-fpage">9372</span>–<span class="cit-lpage">9377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1201130109%26rft_id%253Dinfo%253Apmid%252F22635272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA5LzI0LzkzNzIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8yMy82MzkzNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.21" data-doi="10.1371/journal.ppat.1003500"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lauber</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goeman</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parquet</span>  <span class="cit-name-given-names">M del C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nga</span>  <span class="cit-name-given-names">PT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morita</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE</span></span>: <span class="cit-article-title">The footprint of genome architecture in the largest genome expansion in RNA viruses</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">9</span>:<span class="cit-fpage">e1003500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D9%26rft.spage%253De1003500%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003500%26rft_id%253Dinfo%253Apmid%252F23874204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874204&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.22" data-doi="10.1128/JVI.01296-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eckerle</span>  <span class="cit-name-given-names">LD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sperry</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR</span></span>: <span class="cit-article-title">High fidelity of murine hepatitis virus replication is decreased in nsp14 exoribonuclease mutants</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-pub-date">2007</span>, <span class="cit-vol">81</span>:<span class="cit-fpage">12135</span>–<span class="cit-lpage">12144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01296-07%26rft_id%253Dinfo%253Apmid%252F17804504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MS8yMi8xMjEzNSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzIzLzYzOTM2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bukhari</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mulley</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gulyaeva</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neuman</span>  <span class="cit-name-given-names">BW</span></span>: <span class="cit-article-title">Description and initial characterization of metatranscriptomic nidovirus-like genomes from the proposed new family Abyssoviridae, and from a sister group to the Coronavirinae, the proposed genus Alphaletovirus</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">524</span>:<span class="cit-fpage">160</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D524%26rft.spage%253D160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.24" data-doi="10.1101/307678"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Debat</span>  <span class="cit-name-given-names">HJ</span></span>: <span class="cit-article-title">Expanding the size limit of RNA viruses: Evidence of a novel divergent nidovirus in California sea hare, with a ~35.9 kb virus genome</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-pub-date">2018</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/307678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F307678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIzMDc2Nzh2MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzIzLzYzOTM2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Debat</span>  <span class="cit-name-given-names">HJ</span></span>: <span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">RN</span></span><span class="cit-auth"> <span class="cit-name-given-names">A</span> <span class="cit-name-surname">Virome</span></span> <span class="cit-article-title">Associated to the Golden Orb-Weaver Spider Nephila clavipes</span>. <abbr class="cit-jnl-abbrev">Front Microbiol</abbr> <span class="cit-pub-date">2017</span>, <span class="cit-vol">8</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.26" data-doi="10.1093/nar/gkl092"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Armougom</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moretti</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poirot</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Audic</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dumas</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schaeli</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keduas</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Notredame</span>  <span class="cit-name-given-names">C</span></span>: <span class="cit-article-title">Expresso: automatic incorporation of structural information in multiple sequence alignments using 3D-Coffee</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">34</span>:<span class="cit-fpage">W604</span>–<span class="cit-lpage">608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DArmougom%26rft.auinit1%253DF.%26rft.volume%253D34%26rft.issue%253DWeb%2BServer%2Bissue%26rft.spage%253DW604%26rft.epage%253DW608%26rft.atitle%253DExpresso%253A%2Bautomatic%2Bincorporation%2Bof%2Bstructural%2Binformation%2Bin%2Bmultiple%2Bsequence%2Balignments%2Busing%2B3D-Coffee.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl092%26rft_id%253Dinfo%253Apmid%252F16845081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16845081&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245650200121&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.27" data-doi="10.1002/jcc.20084"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pettersen</span>  <span class="cit-name-given-names">EF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">TD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Couch</span>  <span class="cit-name-given-names">GS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greenblatt</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferrin</span>  <span class="cit-name-given-names">TE</span></span>: <span class="cit-article-title">UCSF Chimera--a visualization system for exploratory research and analysis</span>. <abbr class="cit-jnl-abbrev">J Comput Chem</abbr> <span class="cit-pub-date">2004</span>, <span class="cit-vol">25</span>:<span class="cit-fpage">1605</span>–<span class="cit-lpage">1612</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DPettersen%26rft.auinit1%253DE.%2BF.%26rft.volume%253D25%26rft.issue%253D13%26rft.spage%253D1605%26rft.epage%253D1612%26rft.atitle%253DUCSF%2BChimera--a%2Bvisualization%2Bsystem%2Bfor%2Bexploratory%2Bresearch%2Band%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.20084%26rft_id%253Dinfo%253Apmid%252F15264254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.20084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15264254&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223379100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zimmermann</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nam</span>  <span class="cit-name-given-names">S-Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rau</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kübler</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lozajic</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gabler</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Söding</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lupas</span>  <span class="cit-name-given-names">AN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alva</span>  <span class="cit-name-given-names">V</span></span>: <span class="cit-article-title">A Completely Reimplemented MPI Bioinformatics Toolkit with a New HHpred Server at its Core</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">430</span>:<span class="cit-fpage">2237</span>–<span class="cit-lpage">2243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BMol%2BBiol%26rft.volume%253D430%26rft.spage%253D2237%26rft_id%253Dinfo%253Apmid%252F29258817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=29258817&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.29" data-doi="10.1093/molbev/msp259"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gouy</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O</span></span>: <span class="cit-article-title">SeaView version 4: A multiplatform graphical user interface for sequence alignment and phylogenetic tree building</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-pub-date">2010</span>, <span class="cit-vol">27</span>:<span class="cit-fpage">221</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp259%26rft_id%253Dinfo%253Apmid%252F19854763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msp259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19854763&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273704400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.30" data-doi="10.1093/nar/gku366"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yachdav</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kloppmann</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kajan</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hecht</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldberg</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hamp</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hönigschmid</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schafferhans</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roos</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bernhofer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">PredictProtein--an open resource for online prediction of protein structural and functional features</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">42</span>:<span class="cit-fpage">W337</span>–<span class="cit-lpage">343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku366%26rft_id%253Dinfo%253Apmid%252F24799431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku366&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24799431&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339715000056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.31" data-doi="10.1093/nar/gkg556"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gouet</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Courcelle</span>  <span class="cit-name-given-names">E</span></span>: <span class="cit-article-title">ESPript/ENDscript: Extracting and rendering sequence and 3D information from atomic structures of proteins</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-pub-date">2003</span>, <span class="cit-vol">31</span>:<span class="cit-fpage">3320</span>–<span class="cit-lpage">3323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGouet%26rft.auinit1%253DP.%26rft.volume%253D31%26rft.issue%253D13%26rft.spage%253D3320%26rft.epage%253D3323%26rft.atitle%253DESPript%252FENDscript%253A%2Bextracting%2Band%2Brendering%2Bsequence%2Band%2B3D%2Binformation%2Bfrom%2Batomic%2Bstructures%2Bof%2Bproteins%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg556%26rft_id%253Dinfo%253Apmid%252F12824317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12824317&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183832900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.32" data-doi="10.1038/nprot.2015.053"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">LA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mezulis</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yates</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wass</span>  <span class="cit-name-given-names">MN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sternberg</span>  <span class="cit-name-given-names">MJE</span></span>: <span class="cit-article-title">The Phyre2 web portal for protein modeling, prediction and analysis</span>. <abbr class="cit-jnl-abbrev">Nat Protoc</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">10</span>:<span class="cit-fpage">845</span>–<span class="cit-lpage">858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BProtoc%26rft.volume%253D10%26rft.spage%253D845%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2015.053%26rft_id%253Dinfo%253Apmid%252F25950237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2015.053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25950237&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.33" data-doi="10.1101/gr.849004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crooks</span>  <span class="cit-name-given-names">GE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hon</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chandonia</span>  <span class="cit-name-given-names">J-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">SE</span></span>: <span class="cit-article-title">WebLogo: a sequence logo generator</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-pub-date">2004</span>, <span class="cit-vol">14</span>:<span class="cit-fpage">1188</span>–<span class="cit-lpage">1190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DCrooks%26rft.auinit1%253DG.%2BE.%26rft.volume%253D14%26rft.issue%253D6%26rft.spage%253D1188%26rft.epage%253D1190%26rft.atitle%253DWebLogo%253A%2BA%2BSequence%2BLogo%2BGenerator%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.849004%26rft_id%253Dinfo%253Apmid%252F15173120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE0LzYvMTE4OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzIzLzYzOTM2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.34" data-doi="10.1016/S0959-440X(02)00391-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">FM</span></span>: <span class="cit-article-title">SAM (dependent) I AM: the S-adenosylmethionine-dependent methyltransferase fold</span>. <abbr class="cit-jnl-abbrev">Curr Opin Struct Biol</abbr> <span class="cit-pub-date">2002</span>, <span class="cit-vol">12</span>:<span class="cit-fpage">783</span>–<span class="cit-lpage">793</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bstructural%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BStruct%2BBiol%26rft.aulast%253DMartin%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D783%26rft.epage%253D793%26rft.atitle%253DSAM%2B%2528dependent%2529%2BI%2BAM%253A%2Bthe%2BS-adenosylmethionine-dependent%2Bmethyltransferase%2Bfold.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-440X%252802%252900391-3%26rft_id%253Dinfo%253Apmid%252F12504684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-440X(02)00391-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12504684&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179777100014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.35" data-doi="10.1080/15476286.2015.1004955"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Byszewska</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Śmietański</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Purta</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bujnicki</span>  <span class="cit-name-given-names">JM</span></span>: <span class="cit-article-title">RNA methyltransferases involved in 5’ cap biosynthesis</span>. <abbr class="cit-jnl-abbrev">RNA Biol</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">11</span>:<span class="cit-fpage">1597</span>–<span class="cit-lpage">1607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2BBiol%26rft.volume%253D11%26rft.spage%253D1597%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15476286.2015.1004955%26rft_id%253Dinfo%253Apmid%252F25626080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15476286.2015.1004955&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25626080&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.36" data-doi="10.1016/S1097-2765(03)00522-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fabrega</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hausmann</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shuman</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lima</span>  <span class="cit-name-given-names">CD</span></span>: <span class="cit-article-title">Structure and mechanism of mRNA cap (guanine-N7) methyltransferase</span>. <abbr class="cit-jnl-abbrev">Mol Cell</abbr> <span class="cit-pub-date">2004</span>, <span class="cit-vol">13</span>:<span class="cit-fpage">77</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DFabrega%26rft.auinit1%253DC.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D89%26rft.atitle%253DStructure%2Band%2Bmechanism%2Bof%2BmRNA%2Bcap%2B%2528guanine-N7%2529%2Bmethyltransferase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252803%252900522-7%26rft_id%253Dinfo%253Apmid%252F14731396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(03)00522-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14731396&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220168400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.37" data-doi="10.1038/sj.emboj.7601912"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De la Peña</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kyrieleis</span>  <span class="cit-name-given-names">OJP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cusack</span>  <span class="cit-name-given-names">S</span></span>: <span class="cit-article-title">Structural insights into the mechanism and evolution of the vaccinia virus mRNA cap N7 methyl-transferase</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr> <span class="cit-pub-date">2007</span>, <span class="cit-vol">26</span>:<span class="cit-fpage">4913</span>–<span class="cit-lpage">4925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DDe%2Bla%2BPena%26rft.auinit1%253DM.%26rft.volume%253D26%26rft.issue%253D23%26rft.spage%253D4913%26rft.epage%253D4925%26rft.atitle%253DStructural%2Binsights%2Binto%2Bthe%2Bmechanism%2Band%2Bevolution%2Bof%2Bthe%2Bvaccinia%2Bvirus%2BmRNA%2Bcap%2BN7%2Bmethyl-transferase%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7601912%26rft_id%253Dinfo%253Apmid%252F17989694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjYvMjMvNDkxMyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzIzLzYzOTM2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.38" data-doi="10.1038/nsmb1225"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sutton</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grimes</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stuart</span>  <span class="cit-name-given-names">DI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roy</span>  <span class="cit-name-given-names">P</span></span>: <span class="cit-article-title">Bluetongue virus VP4 is an RNA-capping assembly line</span>. <abbr class="cit-jnl-abbrev">Nat Struct Mol Biol</abbr> <span class="cit-pub-date">2007</span>, <span class="cit-vol">14</span>:<span class="cit-fpage">449</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DSutton%26rft.auinit1%253DG.%26rft.volume%253D14%26rft.issue%253D5%26rft.spage%253D449%26rft.epage%253D451%26rft.atitle%253DBluetongue%2Bvirus%2BVP4%2Bis%2Ban%2BRNA-capping%2Bassembly%2Bline.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb1225%26rft_id%253Dinfo%253Apmid%252F17417654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb1225&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17417654&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246187400019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.39" data-doi="10.1016/S0092-8674(02)01110-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farsetta</span>  <span class="cit-name-given-names">DL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nibert</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">SC</span></span>: <span class="cit-article-title">RNA synthesis in a cage--structural studies of reovirus polymerase lambda3</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-pub-date">2002</span>, <span class="cit-vol">111</span>:<span class="cit-fpage">733</span>–<span class="cit-lpage">745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DTao%26rft.auinit1%253DY.%26rft.volume%253D111%26rft.issue%253D5%26rft.spage%253D733%26rft.epage%253D745%26rft.atitle%253DRNA%2Bsynthesis%2Bin%2Ba%2Bcage--structural%2Bstudies%2Bof%2Breovirus%2Bpolymerase%2Blambda3.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252802%252901110-8%26rft_id%253Dinfo%253Apmid%252F12464184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(02)01110-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12464184&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179594500014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.40" data-doi="10.1128/JVI.00814-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ray</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tilgner</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deas</span>  <span class="cit-name-given-names">TS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">P-Y</span></span>: <span class="cit-article-title">West Nile virus 5’-cap structure is formed by sequential guanine N-7 and ribose 2’-O methylations by nonstructural protein 5</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">80</span>:<span class="cit-fpage">8362</span>–<span class="cit-lpage">8370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DRay%26rft.auinit1%253DD.%26rft.volume%253D80%26rft.issue%253D17%26rft.spage%253D8362%26rft.epage%253D8370%26rft.atitle%253DWest%2BNile%2BVirus%2B5%2527-Cap%2BStructure%2BIs%2BFormed%2Bby%2BSequential%2BGuanine%2BN-7%2Band%2BRibose%2B2%2527-O%2BMethylations%2Bby%2BNonstructural%2BProtein%2B5%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00814-06%26rft_id%253Dinfo%253Apmid%252F16912287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MC8xNy84MzYyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDUvMjMvNjM5MzY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.41" data-doi="10.1093/nar/gkt1240"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">NK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chandonia</span>  <span class="cit-name-given-names">J-M</span></span>: <span class="cit-article-title">SCOPe: Structural Classification of Proteins--extended, integrating SCOP and ASTRAL data and classification of new structures</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">42</span>:<span class="cit-fpage">D304</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1240%26rft_id%253Dinfo%253Apmid%252F24304899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24304899&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.42" data-doi="10.1073/pnas.92.2.507"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ahola</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kääriäinen</span>  <span class="cit-name-given-names">L</span></span>: <span class="cit-article-title">Reaction in alphavirus mRNA capping: formation of a covalent complex of nonstructural protein nsP1 with 7-methyl-GMP</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-pub-date">1995</span>, <span class="cit-vol">92</span>:<span class="cit-fpage">507</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DAhola%26rft.auinit1%253DT.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D507%26rft.epage%253D511%26rft.atitle%253DReaction%2Bin%2Balphavirus%2BmRNA%2Bcapping%253A%2Bformation%2Bof%2Ba%2Bcovalent%2Bcomplex%2Bof%2Bnonstructural%2Bprotein%2BnsP1%2Bwith%2B7-methyl-GMP.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.92.2.507%26rft_id%253Dinfo%253Apmid%252F7831320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI5Mi8yLzUwNyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzIzLzYzOTM2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.43" data-doi="10.1093/emboj/21.11.2757"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Egloff</span>  <span class="cit-name-given-names">M-P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benarroch</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Selisko</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Romette</span>  <span class="cit-name-given-names">J-L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canard</span>  <span class="cit-name-given-names">B</span></span>: <span class="cit-article-title">An RNA cap (nucleoside-2’-O-)-methyltransferase in the flavivirus RNA polymerase NS5: crystal structure and functional characterization</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr> <span class="cit-pub-date">2002</span>, <span class="cit-vol">21</span>:<span class="cit-fpage">2757</span>–<span class="cit-lpage">2768</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DEgloff%26rft.auinit1%253DM.-P.%26rft.volume%253D21%26rft.issue%253D11%26rft.spage%253D2757%26rft.epage%253D2768%26rft.atitle%253DAn%2BRNA%2Bcap%2B%2528nucleoside-2%2527-O-%2529-methyltransferase%2Bin%2Bthe%2Bflavivirus%2BRNA%2Bpolymerase%2BNS5%253A%2Bcrystal%2Bstructure%2Band%2Bfunctional%2Bcharacterization%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F21.11.2757%26rft_id%253Dinfo%253Apmid%252F12032088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjEvMTEvMjc1NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA1LzIzLzYzOTM2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.44" data-doi="10.1038/ncomms9749"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paesen</span>  <span class="cit-name-given-names">GC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collet</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sallamand</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Debart</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vasseur</span>  <span class="cit-name-given-names">J-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Decroly</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grimes</span>  <span class="cit-name-given-names">JM</span></span>: <span class="cit-article-title">X-ray structure and activities of an essential Mononegavirales L-protein domain</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">6</span>:<span class="cit-fpage">8749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D6%26rft.spage%253D8749%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms9749%26rft_id%253Dinfo%253Apmid%252F26549102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms9749&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26549102&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coutard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guez</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paesen</span>  <span class="cit-name-given-names">GC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Debart</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vasseur</span>  <span class="cit-name-given-names">J-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grimes</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Decroly</span>  <span class="cit-name-given-names">E</span></span>: <span class="cit-article-title">The methyltransferase domain of the Sudan ebolavirus L protein specifically targets internal adenosines of RNA substrates, in addition to the cap structure</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">46</span>:<span class="cit-fpage">7902</span>–<span class="cit-lpage">7912</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253D7902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ringeard</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marchand</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Decroly</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Motorin</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bennasser</span>  <span class="cit-name-given-names">Y</span></span>: <span class="cit-article-title">FTSJ3 is an RNA 2’-O-Methyltransferase recruited by HIV to avoid innate immunity sensing</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2019</span>,</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-639369v2.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rana</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ankri</span>  <span class="cit-name-given-names">S</span></span>: <span class="cit-article-title">Reviving the RNA World: An Insight into the Appearance of RNA Methyltransferases</span>. <abbr class="cit-jnl-abbrev">Front Genet</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">7</span>:<span class="cit-fpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront%2BGenet%26rft.volume%253D7%26rft.spage%253D99%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">48.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-639369v2.48" data-doi="10.1093/nar/gkw637"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petit</span>  <span class="cit-name-given-names">A-P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bueren-Calabuig</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fletcher</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peggie</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weidlich</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scullion</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pisliakov</span>  <span class="cit-name-given-names">AV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cowling</span>  <span class="cit-name-given-names">VH</span></span>: <span class="cit-article-title">Molecular basis of RNA guanine-7 methyltransferase (RNMT) activation by RAM</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">44</span>:<span class="cit-fpage">10423</span>–<span class="cit-lpage">10436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw637%26rft_id%253Dinfo%253Apmid%252F27422871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw637&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27422871&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">49.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-639369v2.49" data-doi="10.1038/35010041"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reinisch</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nibert</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">SC</span></span>: <span class="cit-article-title">Structure of the reovirus core at 3.6 A resolution</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2000</span>, <span class="cit-vol">404</span>:<span class="cit-fpage">960</span>–<span class="cit-lpage">967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DReinisch%26rft.auinit1%253DK.%2BM.%26rft.volume%253D404%26rft.issue%253D6781%26rft.spage%253D960%26rft.epage%253D967%26rft.atitle%253DStructure%2Bof%2Bthe%2Breovirus%2Bcore%2Bat%2B3.6%2BA%2Bresolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35010041%26rft_id%253Dinfo%253Apmid%252F10801118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35010041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10801118&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F05%2F23%2F639369.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">50.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-639369v2.50" data-doi="10.1074/jbc.M501073200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hausmann</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fabrega</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schneller</span>  <span class="cit-name-given-names">SW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lima</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shuman</span>  <span class="cit-name-given-names">S</span></span>: <span class="cit-article-title">Encephalitozoon cuniculi mRNA cap (guanine N-7) methyltransferase: methyl acceptor specificity, inhibition BY S-adenosylmethionine analogs, and structure-guided mutational analysis</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-pub-date">2005</span>, <span class="cit-vol">280</span>:<span class="cit-fpage">20404</span>–<span class="cit-lpage">20412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M501073200%26rft_id%253Dinfo%253Apmid%252F15760890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODAvMjEvMjA0MDQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNS8yMy82MzkzNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">51.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-639369v2.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kyrieleis</span>  <span class="cit-name-given-names">OJP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">de la Peña</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shuman</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cusack</span>  <span class="cit-name-given-names">S</span></span>: <span class="cit-article-title">Crystal structure of vaccinia virus mRNA capping enzyme provides insights into the mechanism and evolution of the capping apparatus</span>. <abbr class="cit-jnl-abbrev">Struct Lond Engl 1993</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">22</span>:<span class="cit-fpage">452</span>–<span class="cit-lpage">465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStruct%2BLond%2BEngl%2B1993%26rft.volume%253D22%26rft.spage%253D452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/639369v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F639369v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/640235v2" title="Three-dimensional morphological analysis of the dynamic digestive system in the green brittle star" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;May 23, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/639369v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/639369v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/639369v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/639369v2" class="active" data-icon-position="" data-hide-link-title="0">Identification of a Nidovirales Orf1a N7-guanine cap Methyltransferase signature-sequence as a genetic marker of large genome Tobaniviridae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/744855" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-n43OI78lqCyThVO4exSAlS79FcrTqUnAvAPTO1E789U" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="744855" id="node744855--2" data-pisa="biorxiv;639369v2" data-pisa-master="biorxiv;639369" data-seqnum="744855" data-apath="/biorxiv/early/2019/05/23/639369.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Identification of a <em>Nidovirales</em> Orf1a N7-guanine cap Methyltransferase signature-sequence as a genetic marker of large genome <em>Tobaniviridae</em></div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">François</span> <span class="nlm-surname">Ferron</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Humberto Julio</span> <span class="nlm-surname">Debat</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Etienne</span> <span class="nlm-surname">Decroly</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Bruno</span> <span class="nlm-surname">Canard</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">639369; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/639369 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/639369v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/639369v2&amp;title=Identification%20of%20a%20Nidovirales%20Orf1a%20N7-guanine%20cap%20Methyltransferase%20signature-sequence%20as%20a%20genetic%20marker%20of%20large%20genome%20Tobaniviridae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/639369v2&amp;title=Identification%20of%20a%20Nidovirales%20Orf1a%20N7-guanine%20cap%20Methyltransferase%20signature-sequence%20as%20a%20genetic%20marker%20of%20large%20genome%20Tobaniviridae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/639369v2&amp;text=Identification%20of%20a%20Nidovirales%20Orf1a%20N7-guanine%20cap%20Methyltransferase%20signature-sequence%20as%20a%20genetic%20marker%20of%20large%20genome%20Tobaniviridae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/639369v2&amp;title=Identification%20of%20a%20Nidovirales%20Orf1a%20N7-guanine%20cap%20Methyltransferase%20signature-sequence%20as%20a%20genetic%20marker%20of%20large%20genome%20Tobaniviridae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/639369v2&amp;t=Identification%20of%20a%20Nidovirales%20Orf1a%20N7-guanine%20cap%20Methyltransferase%20signature-sequence%20as%20a%20genetic%20marker%20of%20large%20genome%20Tobaniviridae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/639369v2&amp;title=Identification%20of%20a%20Nidovirales%20Orf1a%20N7-guanine%20cap%20Methyltransferase%20signature-sequence%20as%20a%20genetic%20marker%20of%20large%20genome%20Tobaniviridae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/639369v2&amp;title=Identification%20of%20a%20Nidovirales%20Orf1a%20N7-guanine%20cap%20Methyltransferase%20signature-sequence%20as%20a%20genetic%20marker%20of%20large%20genome%20Tobaniviridae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="744855" id="node744855--3" data-pisa="biorxiv;639369v2" data-pisa-master="biorxiv;639369" data-seqnum="744855" data-apath="/biorxiv/early/2019/05/23/639369.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Identification of a <em>Nidovirales</em> Orf1a N7-guanine cap Methyltransferase signature-sequence as a genetic marker of large genome <em>Tobaniviridae</em></div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">François</span> <span class="nlm-surname">Ferron</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Humberto Julio</span> <span class="nlm-surname">Debat</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Etienne</span> <span class="nlm-surname">Decroly</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Bruno</span> <span class="nlm-surname">Canard</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">639369; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/639369 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Identification%20of%20a%20Nidovirales%20Orf1a%20N7-guanine%20cap%20Methyltransferase%20signature-sequence%20as%20a%20genetic%20marker%20of%20large%20genome%20Tobaniviridae&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Ferron%2C+Fran%C3%A7ois&amp;rft.au=Debat%2C+Humberto+Julio&amp;rft.au=Decroly%2C+Etienne&amp;rft.au=Canard%2C+Bruno"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/744855/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/744855/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/744855/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/744855/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/744855/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/744855/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/744855/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/744855/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/744855/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/744855/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/744855/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/744855/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/639369v2&amp;count=horizontal&amp;via=&amp;text=Identification%20of%20a%20Nidovirales%20Orf1a%20N7-guanine%20cap%20Methyltransferase%20signature-sequence%20as%20a%20genetic%20marker%20of%20large%20genome%20Tobaniviridae&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/639369v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/639369v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/639369v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
