<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/20/2020.03.18.997346.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.18.997346v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Ancient RNA virus epidemics through the lens of recent adaptation in human genomes" />
<meta name="DC.Identifier" content="10.1101/2020.03.18.997346" />
<meta name="DC.Date" content="2020-03-20" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Over the course of the last several million years of evolution, humans likely have been plagued by hundreds or perhaps thousands of epidemics. Little is known about such ancient epidemics and a deep evolutionary perspective on current pathogenic threats is lacking. The study of past epidemics has typically been limited in temporal scope to recorded history, and in physical scope to pathogens that left sufficient DNA behind, such as Yersinia pestis during the Great Plague.

Host genomes however offer an indirect way to detect ancient epidemics beyond the current temporal and physical limits. Arms races with pathogens have shaped the genomes of the hosts by driving a large number of adaptations at many genes, and these signals can be used to detect and further characterize ancient epidemics.

Here, we detect the genomic footprints left by ancient viral epidemics that took place in the past ~50,000 years in the 26 human populations represented in the 1,000 Genomes Project. By using the enrichment in signals of adaptation at ~4,500 host loci that interact with specific types of viruses, we provide evidence that RNA viruses have driven a particularly large number of adaptive events across diverse human populations. These results suggest that different types of viruses may have exerted different selective pressures during human evolution. Knowledge of these past selective pressures will provide a deeper evolutionary perspective on current pathogenic threats." />
<meta name="DC.Contributor" content="David Enard" />
<meta name="DC.Contributor" content="Dmitri A. Petrov" />
<meta name="article:published_time" content="2020-03-20" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Ancient RNA virus epidemics through the lens of recent adaptation in human genomes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Over the course of the last several million years of evolution, humans likely have been plagued by hundreds or perhaps thousands of epidemics. Little is known about such ancient epidemics and a deep evolutionary perspective on current pathogenic threats is lacking. The study of past epidemics has typically been limited in temporal scope to recorded history, and in physical scope to pathogens that left sufficient DNA behind, such as Yersinia pestis during the Great Plague. Host genomes however offer an indirect way to detect ancient epidemics beyond the current temporal and physical limits. Arms races with pathogens have shaped the genomes of the hosts by driving a large number of adaptations at many genes, and these signals can be used to detect and further characterize ancient epidemics. Here, we detect the genomic footprints left by ancient viral epidemics that took place in the past ~50,000 years in the 26 human populations represented in the 1,000 Genomes Project. By using the enrichment in signals of adaptation at ~4,500 host loci that interact with specific types of viruses, we provide evidence that RNA viruses have driven a particularly large number of adaptive events across diverse human populations. These results suggest that different types of viruses may have exerted different selective pressures during human evolution. Knowledge of these past selective pressures will provide a deeper evolutionary perspective on current pathogenic threats.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.18.997346v1" />
<meta name="citation_id" content="2020.03.18.997346v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.18.997346v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.18.997346v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.18.997346v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.18.997346" />
<meta name="citation_num_pages" content="23" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.18.997346" />
<meta name="citation_author" content="David Enard" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Arizona" />
<meta name="citation_author_email" content="denard@email.arizona.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2634-8016" />
<meta name="citation_author" content="Dmitri A. Petrov" />
<meta name="citation_author_institution" content="Department of Biology, Stanford University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3664-9130" />
<meta name="citation_reference" content="Abi-Rached, L., Jobin, M.J., Kulkarni, S., McWhinnie, A., Dalva, K., Gragert, L., Babrzadeh, F., Gharizadeh, B., Luo, M., Plummer, F.A., et al. (2011). The shaping of modern human immune systems by multiregional admixture with archaic humans. Science 334, 89–94." />
<meta name="citation_reference" content="Akbari, A., Vitti, J.J., Iranmehr, A., Bakhtiari, M., Sabeti, P.C., Mirarab, S., and Bafna, V. (2018). Identifying the favored mutation in a positive selective sweep. Nat Methods 15, 279–282." />
<meta name="citation_reference" content="Barreiro, L.B., Ben-Ali, M., Quach, H., Laval, G., Patin, E., Pickrell, J.K., Bouchier, C., Tichit, M., Neyrolles, O., Gicquel, B., et al. (2009). Evolutionary dynamics of human Toll-like receptors and their different contributions to host defense. PLoS Genet 5, e1000562." />
<meta name="citation_reference" content="Breman, J.G., Heymann, D.L., Lloyd, G., McCormick, J.B., Miatudila, M., Murphy, F.A., Muyembe-Tamfun, J.J., Piot, P., Ruppol, J.F., Sureau, P., et al. (2016). Discovery and Description of Ebola Zaire Virus in 1976 and Relevance to the West African Epidemic During 2013-2016. J Infect Dis 214, S93–S101." />
<meta name="citation_reference" content="Castellano, D.U., L.H.; Munch, K.; Enard, D. (2019). Viruses rule over adaptation in conservd human proteins. bioRxiv." />
<meta name="citation_reference" content="Consortium, G.T. (2015). Human genomics. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. Science 348, 648–660." />
<meta name="citation_reference" content="Ebel, E.R., Telis, N., Venkataram, S., Petrov, D.A., and Enard, D. (2017). High rate of adaptation of mammalian proteins that interact with Plasmodium and related parasites. PLoS Genet 13, e1007023." />
<meta name="citation_reference" content="Enard, D., Cai, L., Gwennap, C., and Petrov, D.A. (2016). Viruses are a dominant driver of protein adaptation in mammals. Elife 5." />
<meta name="citation_reference" content="Enard, D., Messer, P.W., and Petrov, D.A. (2014). Genome-wide signals of positive selection in human evolution. Genome Res 24, 885–895." />
<meta name="citation_reference" content="Enard, D., and Petrov, D.A. (2018). Evidence that RNA Viruses Drove Adaptive Introgression between Neanderthals and Modern Humans. Cell 175, 360–371 e313." />
<meta name="citation_reference" content="Ferrer-Admetlla, A., Liang, M., Korneliussen, T., and Nielsen, R. (2014). On detecting incomplete soft or hard selective sweeps using haplotype structure. Mol Biol Evol 31, 1275–1291." />
<meta name="citation_reference" content="Fu, Q., Hajdinjak, M., Moldovan, O.T., Constantin, S., Mallick, S., Skoglund, P., Patterson, N., Rohland, N., Lazaridis, I., Nickel, B., et al. (2015). An early modern human from Romania with a recent Neanderthal ancestor. Nature 524, 216–219." />
<meta name="citation_reference" content="Galtier, N. (2016). Adaptive Protein Evolution in Animals and the Effective Population Size Hypothesis. PLoS Genet 12, e1005774." />
<meta name="citation_reference" content="Garud, N.R., Messer, P.W., Buzbas, E.O., and Petrov, D.A. (2015). Recent selective sweeps in North American Drosophila melanogaster show signatures of soft sweeps. PLoS Genet 11, e1005004." />
<meta name="citation_reference" content="Gene Ontology, C. (2015). Gene Ontology Consortium: going forward. Nucleic Acids Res 43, D1049–1056." />
<meta name="citation_reference" content="Genomes Project, C., Auton, A., Brooks, L.D., Durbin, R.M., Garrison, E.P., Kang, H.M., Korbel, J.O., Marchini, J.L., McCarthy, S., McVean, G.A., et al. (2015). A global reference for human genetic variation. Nature 526, 68–74." />
<meta name="citation_reference" content="Geoghegan, J.L., Duchene, S., and Holmes, E.C. (2017). Comparative analysis estimates the relative frequencies of co-divergence and cross-species transmission within viral families. PLoS Pathog 13, e1006215." />
<meta name="citation_reference" content="Green, R.E., Krause, J., Briggs, A.W., Maricic, T., Stenzel, U., Kircher, M., Patterson, N., Li, H., Zhai, W., Fritz, M.H., et al. (2010). A draft sequence of the Neandertal genome. Science 328, 710–722." />
<meta name="citation_reference" content="Gubler, D.J., Vasilakis, N., and Musso, D. (2017). History and Emergence of Zika Virus. J Infect Dis 216, S860–S867." />
<meta name="citation_reference" content="Hemelaar, J. (2012). The origin and diversity of the HIV-1 pandemic. Trends Mol Med 18, 182–192." />
<meta name="citation_reference" content="Hinch, A.G., Tandon, A., Patterson, N., Song, Y., Rohland, N., Palmer, C.D., Chen, G.K., Wang, K., Buxbaum, S.G., Akylbekova, E.L., et al. (2011). The landscape of recombination in African Americans. Nature 476, 170–175." />
<meta name="citation_reference" content="Holmes, E.C., and Rambaut, A. (2004). Viral evolution and the emergence of SARS coronavirus. Philos Trans R Soc Lond B Biol Sci 359, 1059–1065." />
<meta name="citation_reference" content="Johnson, K.E., and Voight, B.F. (2018). Patterns of shared signatures of recent positive selection across human populations. Nat Ecol Evol 2, 713–720." />
<meta name="citation_reference" content="Jones, B.A., Grace, D., Kock, R., Alonso, S., Rushton, J., Said, M.Y., McKeever, D., Mutua, F., Young, J., McDermott, J., et al. (2013). Zoonosis emergence linked to agricultural intensification and environmental change. Proc Natl Acad Sci U S A 110, 8399–8404." />
<meta name="citation_reference" content="Kern, A.D., and Schrider, D.R. (2016). Discoal: flexible coalescent simulations with selection. Bioinformatics 32, 3839–3841." />
<meta name="citation_reference" content="Kreuder Johnson, C., Hitchens, P.L., Smiley Evans, T., Goldstein, T., Thomas, K., Clements, A., Joly, D.O., Wolfe, N.D., Daszak, P., Karesh, W.B., et al. (2015). Spillover and pandemic properties of zoonotic viruses with high host plasticity. Sci Rep 5, 14830." />
<meta name="citation_reference" content="Luisi, P., Alvarez-Ponce, D., Pybus, M., Fares, M.A., Bertranpetit, J., and Laayouni, H. (2015). Recent positive selection has acted on genes encoding proteins with more interactions within the whole human interactome. Genome Biol Evol 7, 1141–1154." />
<meta name="citation_reference" content="Maclean, C.A., Chue Hong, N.P., and Prendergast, J.G. (2015). hapbin: An Efficient Program for Performing Haplotype-Based Scans for Positive Selection in Large Genomic Datasets. Mol Biol Evol 32, 3027–3029." />
<meta name="citation_reference" content="McDonald, J.H., and Kreitman, M. (1991). Adaptive protein evolution at the Adh locus in Drosophila. Nature 351, 652–654." />
<meta name="citation_reference" content="Metzger, W.G., Kohler, C., and Mordmuller, B. (2015). Lessons from a modern review of the smallpox eradication files. J R Soc Med 108, 473–477." />
<meta name="citation_reference" content="Nakagome, S., Alkorta-Aranburu, G., Amato, R., Howie, B., Peter, B.M., Hudson, R.R., and Di Rienzo, A. (2016). Estimating the Ages of Selection Signals from Different Epochs in Human History. Mol Biol Evol 33, 657–669." />
<meta name="citation_reference" content="Paabo, S. (2015). The diverse origins of the human gene pool. Nat Rev Genet 16, 313–314." />
<meta name="citation_reference" content="Petti, S., and Lodi, G. (2019). The controversial natural history of oral Herpes Simplex Virus Type 1 infection. Oral Dis." />
<meta name="citation_reference" content="Sankararaman, S., Mallick, S., Dannemann, M., Prufer, K., Kelso, J., Paabo, S., Patterson, N., and Reich, D. (2014). The genomic landscape of Neanderthal ancestry in present-day humans. Nature 507, 354–357." />
<meta name="citation_reference" content="Schrider, D.R., and Kern, A.D. (2016). S/HIC: Robust Identification of Soft and Hard Sweeps Using Machine Learning. PLoS Genet 12, e1005928." />
<meta name="citation_reference" content="Schrider, D.R., and Kern, A.D. (2017). Soft Sweeps Are the Dominant Mode of Adaptation in the Human Genome. Mol Biol Evol 34, 1863–1877." />
<meta name="citation_reference" content="Schrider, D.R., and Kern, A.D. (2018). Supervised Machine Learning for Population Genetics: A New Paradigm. Trends Genet 34, 301–312." />
<meta name="citation_reference" content="Siepel, A., Bejerano, G., Pedersen, J.S., Hinrichs, A.S., Hou, M., Rosenbloom, K., Clawson, H., Spieth, J., Hillier, L.W., Richards, S., et al. (2005). Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. Genome Res 15, 1034–1050." />
<meta name="citation_reference" content="Spyrou, M.A., Bos, K.I., Herbig, A., and Krause, J. (2019). Ancient pathogen genomics as an emerging tool for infectious disease research. Nat Rev Genet 20, 323–340." />
<meta name="citation_reference" content="Sugden, L.A., Atkinson, E.G., Fischer, A.P., Rong, S., Henn, B.M., and Ramachandran, S. (2018). Localization of adaptive variants in human genomes using averaged one-dependence estimation. Nat Commun 9, 703." />
<meta name="citation_reference" content="Tataru, P., Mollion, M., Glemin, S., and Bataillon, T. (2017). Inference of Distribution of Fitness Effects and Proportion of Adaptive Substitutions from Polymorphism Data. Genetics 207, 1103–1119." />
<meta name="citation_reference" content="Uricchio, L.H., Petrov, D.A., and Enard, D. (2018). Exploiting selection at linked sites to infer the rate and strength of adaptation. bioRxiv." />
<meta name="citation_reference" content="Uricchio, L.H., Petrov, D.A., and Enard, D. (2019). Exploiting selection at linked sites to infer the rate and strength of adaptation. Nat Ecol Evol 3, 977–984." />
<meta name="citation_reference" content="Voight, B.F., Kudaravalli, S., Wen, X., and Pritchard, J.K. (2006). A map of recent positive selection in the human genome. PLoS Biol 4, e72." />
<meta name="citation_reference" content="Worobey, M., Gemmel, M., Teuwen, D.E., Haselkorn, T., Kunstman, K., Bunce, M., Muyembe, J.J., Kabongo, J.M., Kalengayi, R.M., Van Marck, E., et al. (2008). Direct evidence of extensive diversity of HIV-1 in Kinshasa by 1960. Nature 455, 661–664." />
<meta name="citation_reference" content="Zerbino, D.R., Achuthan, P., Akanni, W., Amode, M.R., Barrell, D., Bhai, J., Billis, K., Cummins, C., Gall, A., Giron, C.G., et al. (2018). Ensembl 2018. Nucleic Acids Res 46, D754–D761." />
<meta name="twitter:title" content="Ancient RNA virus epidemics through the lens of recent adaptation in human genomes" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Over the course of the last several million years of evolution, humans likely have been plagued by hundreds or perhaps thousands of epidemics. Little is known about such ancient epidemics and a deep evolutionary perspective on current pathogenic threats is lacking. The study of past epidemics has typically been limited in temporal scope to recorded history, and in physical scope to pathogens that left sufficient DNA behind, such as Yersinia pestis during the Great Plague.

Host genomes however offer an indirect way to detect ancient epidemics beyond the current temporal and physical limits. Arms races with pathogens have shaped the genomes of the hosts by driving a large number of adaptations at many genes, and these signals can be used to detect and further characterize ancient epidemics.

Here, we detect the genomic footprints left by ancient viral epidemics that took place in the past ~50,000 years in the 26 human populations represented in the 1,000 Genomes Project. By using the enrichment in signals of adaptation at ~4,500 host loci that interact with specific types of viruses, we provide evidence that RNA viruses have driven a particularly large number of adaptive events across diverse human populations. These results suggest that different types of viruses may have exerted different selective pressures during human evolution. Knowledge of these past selective pressures will provide a deeper evolutionary perspective on current pathogenic threats." />
<meta name="og-title" property="og:title" content="Ancient RNA virus epidemics through the lens of recent adaptation in human genomes" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.18.997346v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Over the course of the last several million years of evolution, humans likely have been plagued by hundreds or perhaps thousands of epidemics. Little is known about such ancient epidemics and a deep evolutionary perspective on current pathogenic threats is lacking. The study of past epidemics has typically been limited in temporal scope to recorded history, and in physical scope to pathogens that left sufficient DNA behind, such as Yersinia pestis during the Great Plague.

Host genomes however offer an indirect way to detect ancient epidemics beyond the current temporal and physical limits. Arms races with pathogens have shaped the genomes of the hosts by driving a large number of adaptations at many genes, and these signals can be used to detect and further characterize ancient epidemics.

Here, we detect the genomic footprints left by ancient viral epidemics that took place in the past ~50,000 years in the 26 human populations represented in the 1,000 Genomes Project. By using the enrichment in signals of adaptation at ~4,500 host loci that interact with specific types of viruses, we provide evidence that RNA viruses have driven a particularly large number of adaptive events across diverse human populations. These results suggest that different types of viruses may have exerted different selective pressures during human evolution. Knowledge of these past selective pressures will provide a deeper evolutionary perspective on current pathogenic threats." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-20" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.18.997346v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.18.997346v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.18.997346v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.18.997346v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1203952" />
    <title>Ancient RNA virus epidemics through the lens of recent adaptation in human genomes | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"3zX4Aq_YaNfkrYI3uaEjQV_nozNWo132_qC9YkKUOhY","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1203952","apath":"\/biorxiv\/early\/2020\/03\/20\/2020.03.18.997346.atom","pisa":"biorxiv;2020.03.18.997346v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.18.997346v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.18.997346v1","title":"Ancient RNA virus epidemics through the lens of recent adaptation in human genomes","identifier":"node\/1203952"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/\\\\\\\/2BC3S\\\\\\\/P\\\\\\\/USUy9tAl8qbcv4Sj1HAn3o\\\\\\\/oiDZOiQX5iT\\\\\\\/Xln9AM+WkHmGyopz4telyhSllQua3UpqWt8Y+mG+Dmcd\\\\\\\/ZkA4pOfRo5Y9zp8koTJlCZ2rD906pJemjyQhlskvouW7OFwJM7W1wo1xdB5mwv0HWNEyd866\\\\\\\/Oi6DiJ9ePA3PjJZVplT01ZDf\\\\\\\/V393wG+3jFmOxrw\\\\\\\/Z72FG2wgucCmSvKIE\\\\\\\/W99k5PxqQgaoFvY\\\\\\\/yUf4kYONH0v+OiAOd3EozB+khnDjAg3ronOy4TSykwO2pfGaJZ466LmuQPIuzfjiViUnYJ+mS30dwKtqOz2F2KOuy0XD6klDDkDTi9w\\\\\\\/IdkYSRmACELUZtPQwcaqGE02iZ0w4naFSEoGMVlS3\\\\\\\/q84TS9n6DE9R6OsMdaN7462N9pupLm9qz41g\\\\\\\/PQuqaPca390R\\\\\\\/fo1C1NLOc3+StwMOhKLNA1I7jPbEkCatie4kqlYdvMxYrKNn5167LoftbZC1zGzw+kFTrnHM3MUG92NqiF8kGFXyeSjcG5HASlaVDz3C5ETsqAuEu8zb3kQuCmBju2XYVUMOhOZ2RuIdLjyaPQz19ExuKU8nBP1C26SKx0mfOPmPjjG\\\\\\\/yAFwF23JqIM4CoznfIILkGnbIhEQWoxMJPOL\\\\\\\/kD0LOogsVkmkoBHyNYaHNyRqDHzGU9qdlOVDpdualBYUbIs4TlMvtCKT0ejxR+\\\\\\\/GwaefLeT3g5VMlOF6eCXL9Vy5oHQzJ9Lawa2I3PK5yOGK5P1Q8s076BuBNcQQTtbukRvpPX2\\\\\\\/3TrV+mtQp3Wq1ed39\\\\\\\/4UIQAM60ISdXNpWbRp5OoIPiSBGGxkVS81MLEy57zBdbaJRdZMgGsWz2bNZ6n7agWLPAU1Kf1cmGGvvtHbX7G\\\\\\\/PG\\\\\\\/A+YmQk2gHlk6oMaGUUt4er0P3kDfw9SN24zy\\\\\\\/N+KkeBI7vOMUWrdsytX7pL38G3v13nbaQbfT6zD0\\\\\\\/Dv4YJFbiPJfnW96Hklx8kJoOjtxe98plDyR\\\\\\\/qbOAPanFeBwSmxwCK0rM2iRpTj8\\\\\\\/sP8f4xYuC5vGl5fpI8GjYzEHedaciroccBng2opj6rkpweuKtfmhsmidDXKKWOUEfzCIzPSSgnxW\\\\\\\/zGlxo\\\\\\\/EJcDgtl5JONCH+ZBkFSS\\\\\\\/+jDrwnuc3X8P3GjHORswGkjh6oD3ToQYtuyqFOiK9ZGMWKmMaM1NuwXYw1770tbwPOmcJ2a70wmsjPXsOJJxZ1vxgsiOxbUowdvp5ZwwXnNgmLiJK1mZSKbzPwkASbpr77Kq5usfdH22VHdA80zoE8scm8DYOvZpB302Lm9ImKb0qSDN2MYgnSmMTHFpISrzw3mZMLG1tXybbFfK\\\u0022,\\\u0022iv\\\u0022:\\\u0022f436yfGNd+T3\\\\\\\/w07yYeSHw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022b72c14d52420575c1fb70479c605a13a33f5354928de7786f7dbd6441ca32ec5\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.18.997346v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1203952 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.18.997346v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bGxVmwmj3Y4SAPGWjQD58tTRRKXn4921SeDihgoPRiY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1203952" id="node1203952" data-pisa="biorxiv;2020.03.18.997346v1" data-pisa-master="biorxiv;2020.03.18.997346" data-apath="/biorxiv/early/2020/03/20/2020.03.18.997346.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Ancient RNA virus epidemics through the lens of recent adaptation in human genomes</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-2634-8016" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">David</span> <span class="nlm-surname">Enard</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3664-9130" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Dmitri A.</span> <span class="nlm-surname">Petrov</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.18.997346 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1203952" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David Enard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Ecology and Evolutionary Biology, University of Arizona</span>, Tucson, AZ, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BEnard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Enard%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BEnard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2634-8016" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David Enard</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:denard@email.arizona.edu" class="" data-icon-position="" data-hide-link-title="0">denard@email.arizona.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dmitri A. Petrov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Biology, Stanford University</span>, Stanford, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dmitri%2BA.%2BPetrov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petrov%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADmitri%2BA.%2BPetrov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3664-9130" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dmitri A. Petrov</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.18.997346v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1203952" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1203952/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.18.997346v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1203952" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1203952/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.18.997346v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1203952" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1203952/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.18.997346v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1203952" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1203952/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.18.997346v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1203952" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1203952/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.18.997346v1.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:1203952" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:1203952/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.18.997346v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1203952" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1203952/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Over the course of the last several million years of evolution, humans likely have been plagued by hundreds or perhaps thousands of epidemics. Little is known about such ancient epidemics and a deep evolutionary perspective on current pathogenic threats is lacking. The study of past epidemics has typically been limited in temporal scope to recorded history, and in physical scope to pathogens that left sufficient DNA behind, such as <em>Yersinia pestis</em> during the Great Plague.</p><p id="p-3">Host genomes however offer an indirect way to detect ancient epidemics beyond the current temporal and physical limits. Arms races with pathogens have shaped the genomes of the hosts by driving a large number of adaptations at many genes, and these signals can be used to detect and further characterize ancient epidemics.</p><p id="p-4">Here, we detect the genomic footprints left by ancient viral epidemics that took place in the past ~50,000 years in the 26 human populations represented in the 1,000 Genomes Project. By using the enrichment in signals of adaptation at ~4,500 host loci that interact with specific types of viruses, we provide evidence that RNA viruses have driven a particularly large number of adaptive events across diverse human populations. These results suggest that different types of viruses may have exerted different selective pressures during human evolution. Knowledge of these past selective pressures will provide a deeper evolutionary perspective on current pathogenic threats.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-6">About 40 years ago and for the first time in their evolution, humans eradicated a virus that had claimed countless lives, the Variola virus known as the causal agent of smallpox (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Metzger et al., 2015</a>). Since then, and despite progress in prevention, many new viral threats have emerged and spread, including viruses such as HIV (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hemelaar, 2012</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Worobey et al., 2008</a>), Ebola virus (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Breman et al., 2016</a>), Zika virus (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gubler et al., 2017</a>), and SARS coronavirus (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Holmes and Rambaut, 2004</a>). The number of new viral zoonoses (transmission from animals to humans) keeps increasing as a result of multiple factors notably including higher population density and the disturbance of wild habitats (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jones et al., 2013</a>). Although the current frequency of zoonoses may be unusually high due to large human populations in contact with other species, even much less frequent viral zoonoses during the millions of years of past human evolution are likely to have resulted in many epidemics.</p><p id="p-7">When viruses impose a selective pressure on a host, adaptation happens in response at the host loci that interact with the virus (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Enard et al., 2016</a>). An epidemic can then be detected through the enrichment in signals of adaptation at VIPs that interact with a virus. One obstacle to utilizing this method is that we do not have a direct access to the interactions between a host and an ancient virus. However, host-virus interactions are well known in present viruses. Because phylogenetically related pathogens tend to use the same interactions with their hosts (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Enard et al., 2016</a>), we can use present interactions as proxies for the interactions with ancient viruses from the same phylogenetic family. For example, HIV belongs to the family of lentiviruses, and we can use interactions between HIV and human proteins as proxies for ancient interactions between human proteins and ancient lentiviruses.</p><p id="p-8">Accordingly, we recently found that arms races with ancient viruses have left characteristic signals of abundant, intense adaptation in human proteins that interact with present viruses (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Castellano, 2019</a>; <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Enard et al., 2016</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Uricchio et al., 2019</a>). According to our curation and annotation of the virology literature, in the past three decades, virologists have found that approximately 4,500, or 20% of human proteins physically interact with human-infecting viruses (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Enard and Petrov, 2018</a>). A large proportion of these interactions have known functional consequences for the viral replication cycle, thus making virus-interacting proteins (VIPs) prime candidates for host adaptation in response to viral epidemics (Methods and Table S1). VIPs harbor remarkably high levels of past protein adaptation, with rates of adaptive amino acid changes several times higher than human proteins that are not known to interact with viruses (non-VIPs) (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Enard et al., 2016</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Uricchio et al., 2019</a>). Furthermore, adaptation was not only more frequent at VIPs compared to non-VIPs; it was also stronger, more intense adaptation (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Castellano, 2019</a>; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Uricchio et al., 2019</a>), suggesting that viruses repeatedly imposed strong selective pressures on their human hosts during evolution. Thus, frequent new zoonoses and abundant adaptation at VIPs together suggest that viruses drove many epidemics in past human evolution.</p><p id="p-9">Despite these clear indications that ancient viral epidemics may have been frequent, little is known about which ancient viruses were involved, and how ancient viruses that caused ancient epidemics relate phylogenetically to current viruses infecting humans. The current lack of knowledge of ancient viruses can be explained by the difficulty to recover ancient viral DNA (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Spyrou et al., 2019</a>), and by the fact that many viruses have their genomes coded by RNA that is known to degrade much faster than DNA. In light of the scarcity of molecular remains, how can we identify the viruses that drove ancient epidemics during human evolution?</p><p id="p-10">In order to identify viruses that drove ancient epidemics, we can use present interactions as proxies for the interactions with ancient viruses from the same phylogenetic family. Using this approach, we previously found evidence that 50,000 years ago (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abi-Rached et al., 2011</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fu et al., 2015</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Green et al., 2010</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Paabo, 2015</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Sankararaman et al., 2014</a>), Neanderthals appear to have infected the modern human ancestors of present Europeans with one or multiple RNA viruses, as shown by the fact that European modern humans harbor substantially more and substantially longer introgressed Neanderthal DNA at genes that interact with RNA viruses compared to genes that interact with DNA viruses (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Enard and Petrov, 2018</a>). The key factor in this finding was that ancient RNA viruses drove adaptive introgression of Neanderthal DNA not only at a few loci, but at dozens of loci, thus creating a strongly significant enrichment of Neanderthal DNA at RNA VIPs (VIPs that interact with RNA viruses) compared to DNA VIPs (VIPs that interact with DNA viruses). The magnitude of the adaptive signals is what enabled their assignment to a specific type of viruses, in this case RNA viruses.</p><p id="p-11">Because when they interbred (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Abi-Rached et al., 2011</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Fu et al., 2015</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Green et al., 2010</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Paabo, 2015</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Sankararaman et al., 2014</a>), Neanderthals and modern humans likely infected each other with their respective viruses, there was a strong prior to expect ancient epidemics at the time of interbreeding, and it made sense to start looking for host genomic signals of ancient epidemics at this particular time of human evolution. However, because ancient epidemics were likely frequent during human evolution, it now also makes sense to extend the search for signals of adaptation left by ancient epidemics to any time of human evolution from which we can get signals of adaptation. Recent human adaptation is particularly interesting in this respect, because we now have good genome datasets (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Genomes Project et al., 2015</a>) and statistical tools (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Ferrer-Admetlla et al., 2014</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Schrider and Kern, 2016</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">2017</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">2018</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Voight et al., 2006</a>) to detect signals of recent adaptation genome-wide in the form of recent selective sweeps. Haplotype-based statistics that use the structure of haplotypes along chromosomes have been shown to be particularly useful to detect recent selective sweeps, because they have good statistical power to detect strong, recent incomplete sweeps (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Ferrer-Admetlla et al., 2014</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Garud et al., 2015</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Voight et al., 2006</a>) without suffering from the confounding effect of other processes such as background selection (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Enard et al., 2014</a>). Among available haplotype-based statistics, the iHS or integrated Haplotype Score (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Voight et al., 2006</a>) also has the three advantages, (i) of having been extensively tested (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barreiro et al., 2009</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Enard et al., 2014</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Johnson and Voight, 2018</a>; <a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Voight et al., 2006</a>), (ii) of showing versatility when it comes to detect sweeps from de novo mutations or from standing genetic variation (see power analysis in Methods), and (iii) of having fast implementations that can be used to scan many individual genomes from many human populations in a non-prohibitive amount of time (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Maclean et al., 2015</a>) (Methods).</p><p id="p-12">Here, we examine signals of recent human adaptation in diverse human populations to ask whether viruses drove an enrichment of signals of recent adaptation at VIPs compared to non-VIPs. Specifically, we test whether VIPs are overall enriched for recent selective sweeps detected by the well-established iHS statistic in the 26 human populations represented in the 1,000 Genomes phase 3 dataset (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Genomes Project et al., 2015</a>). The iHS statistic has better power to detect the most recent, incomplete selective sweeps, and no power to detect selective sweeps older than 50,000 years (see power analysis, Methods). This restricts our analysis of recent adaptation in response to viruses to human evolution after the migration out of Africa.</p><p id="p-13">We further ask whether specific types of viruses drove recent adaptation more frequently than others. More specifically, our previous results on adaptive introgression from Neanderthals to modern humans (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Enard and Petrov, 2018</a>), and the known zoonoses from the recent past (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Geoghegan et al., 2017</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kreuder Johnson et al., 2015</a>), suggest that RNA viruses may have driven more recent adaptation than DNA viruses because they jump more often from a species to another. The vast majority of known zoonotic viruses that infected humans in the recent past are RNA viruses from diverse RNA virus phylogenetic families, ranging from lentiviruses such as HIV, flaviviruses (Dengue and Zika virus), filoviruses (Ebola virus), to orthomyxoviruses (Influenza virus). Comparatively, most DNA viruses infecting humans were transmitted from animals much longer ago (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Geoghegan et al., 2017</a>) and thus might cross species barriers less frequently in general. DNA viruses also tend to include less pathogenic viral families such as herpesviruses (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Petti and Lodi, 2019</a>).</p><p id="p-14">Taken together, these diverse lines of evidence suggest the hypothesis that RNA viruses might have imposed a stronger selective pressure than DNA viruses during recent human evolution. To test this, we compare the enrichment in iHS selective sweeps at RNA VIPs and the same enrichment at DNA VIPs. The dataset of 4,500 VIPs is particularly well suited for this comparison, with similar numbers of RNA and DNA VIPs (2,691 and 2,604, respectively, Table S1), providing an evenly powered comparison.</p><p id="p-15">First, using all VIPs compared to non-VIPs, we find a strong enrichment of iHS selective sweeps at VIPs overall, suggesting that viruses were a major selective pressure during recent human evolution that drove multiple strong adaptive events. Second, we further find that the selective sweeps enrichment is much more pronounced at RNA VIPs compared to DNA VIPs. These results validate previous results and are consistent with RNA viruses being a significant selective pressure in the past 50,000 years of human evolution.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Properties of VIPs</h3><p id="p-16">We use a dataset of ~4,500 VIPs that are human proteins known to interact physically with viruses (Table S1). Of these, we previously manually curated 1,920 from the virology literature (Methods). The articles that report these interactions also frequently report their effect on the viral replication cycle, and 66% of the manually curated interactions also have reported proviral (interaction is beneficial to viral proliferation), or antiviral effects (interaction is detrimental to the virus) (Methods). While this percentage is high, it is still likely to be an underestimate due to the fact that it is limited to the cases where the effect of the interaction has been investigated in the first place. This makes VIPs prime candidates for host adaptation, with a strong likelihood of functional mutations affecting the viral replication cycle. The remaining ~2,600 VIPs were identified using high-throughput methods and were retrieved from the VirHostnet 2.0 database as well as from a number of additional studies (Table S1). For this analysis, we pooled the manually curated and the high-throughput VIPs together into one group that we systematically compared to the rest of the genome. In total, 20 viruses that infect humans have more than 10 VIPs and 14 viruses have more than 100 VIPs, with influenza virus (IAV) and HIV having the highest numbers of known VIPs (1,505 and 1,209 respectively; Table S1). Note that VIPs and all other genes used for this analysis are Ensembl v83 genes (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Zerbino et al., 2018</a>).</p><p id="p-17">A simplifying property of this set of VIPs for our analysis is that they are almost exactly evenly distributed between VIPs that interact with RNA viruses (2,691 RNA VIPs) and VIPs that interact with DNA viruses (2,604 DNA VIPs), with 1,134 VIPs interacting with both. This similar number of interactions makes it possible to compare sweeps at RNA VIPs with sweeps at DNA VIPs with no bias in statistical power.</p></div><div id="sec-4" class="subsection"><h3>VIPs are enriched for recent selective sweeps in human populations</h3><p id="p-18">We first test if VIPs are overall enriched for recent selective sweeps compared to non-VIPs. We use the iHS statistic to detect candidate sweeps in each of the 26 populations from the 1,000 Genomes Project (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Maclean et al., 2015</a>) (Methods). The iHS statistic is measured for a focal variant (<a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">Voight et al., 2006</a>), by measuring how far the haplotypes carrying the derived allele of the focal variant extend both upstream and downstream of it, compared to how far the haplotypes carrying the ancestral allele extend. A rapid increase in frequency of the derived focal allele due to strong positive selection results in large and frequent linked haplotypes compared to the haplotypes carrying the ancestral allele, and in elevated values of the iHS statistic. We first measure iHS along chromosomes for all variants in all the 26 populations (Methods), in a way that optimizes the sensitivity of iHS to strong selective sweeps (see power analysis in Methods). We do this because our previous work using a very different methodology to quantify adaptation (the McDonald-Kreitman test) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">McDonald and Kreitman, 1991</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Tataru et al., 2017</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Uricchio et al., 2018</a>) estimated that adaptation at VIPs was not only frequent but also generally driven by strongly adaptive mutations (<a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Uricchio et al., 2019</a>), which predicts strong selective sweeps at VIPs.</p><p id="p-19">We then rank all protein coding genes in the genome from highest iHS values to lowest iHS values. We measure the average iHS in large 1,000kb windows centered on the genomic center of genes. These large windows are more specifically sensitive to strong adaptation compared to smaller windows, and using a constant window size avoids biases due to gene length (Methods). For example, the top 200 genes are then the 200 genes in a population with the highest iHS compared to other genes, the top 1,000 are the 1,000 genes with the highest iHS, and so on. Finally, we count how many VIPs are among the top-ranking genes in a specific population. We then sum the number of VIPs among top-ranking genes across all the 26 populations. That is, if the world included three populations A, B and C, and the respective top 100 of populations A, B and C include 23, 34, and 15 VIPs, then the worldwide top 100 would be 23+34+15=72. We then measure the same sum but in the control sets of non-VIPs, and compute the corresponding fold enrichment in VIPs compared to the average of the control sets of non-VIPs. This implies that the results represent average worldwide trends.</p><p id="p-20">Importantly, the sets of control non-VIPs account for multiple key potential confounding factors (Methods). Indeed, VIPs and non-VIPs not only differ by the fact that VIPs are not to interact with viruses while non-VIPs either do not interact with viruses or are not known to. In addition, VIPs and non-VIPs differ by many other factors (<a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Enard et al., 2016</a>; <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Enard and Petrov, 2018</a>). For instance, VIPs are more highly constrained, more highly expressed than non-VIPs, and have more protein-protein interactions than non-VIPs (<a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Enard et al., 2016</a>; <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Enard and Petrov, 2018</a>). If gene expression, protein-protein interactions (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Luisi et al., 2015</a>), or other factors affect the prevalence of recent sweeps on their own, independently of interactions with viruses, they might confound the comparison of VIPs and non-VIPs by creating differences between the former and the latter that have nothing to do with interactions with viruses. Thus, we build random sets of control non-VIPs that match VIPs for multiple potential confounding factors using a previously described bootstrap test (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Castellano, 2019</a>) (Methods). Furthermore, the control sets of non-VIPs exclude all non-VIPs that are too close to VIPs and may thus be found in the same large sweeps extending over multiple genes. To avoid counting large VIP sweeps also as non-VIP sweeps, or large non-VIP sweeps also as VIP sweeps, we select only control non-VIPs at least 500kb away from VIPs. The enrichment of recent adaptation is therefore tested comparing sweep signals at VIPs compared to sweeps signals far from VIPs. In addition, to avoid the confounding effect of counting the same adaptive event multiple times when VIPs are clustered together in the same selective sweep, we use an approach based on block-randomized genomes to assess the statistical significance of the signals detected (Methods).</p><p id="p-21">We retrieved the number of control non-VIPs among the top-ranking sweep genes for each of the 1,000 control sets, measured the corresponding average, and measured how enriched VIPs were in top-ranking genes compared to this control average. Furthermore, we measured the fold enrichment at VIPs for several sets of iHS top-ranking genes corresponding to different rank thresholds. The top-ranking genes are defined by sliding a rank threshold from the top 2,000 genes, to a much more stringent threshold for the top 20 genes (see <a id="xref-fig-1-1" class="xref-fig" href="#F1">figure 1</a>, x axis). The top 2,000 genes include even weak sweep signals, while the top 20 genes only correspond to strong sweep candidates. Using a sliding rank threshold thus avoids making assumptions on exactly how strong adaptation to viruses needs to be, and enables the detection of enrichments for strong selective sweeps (top 100 or less) as well as enrichments driven by more incomplete or weaker, polygenic hitchhiking signals spread across more genes (see power analysis in Methods).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Worldwide enrichment of iHS sweeps at VIPs compared to control non-VIPs A. Black line: observed fold enrichment at VIPs. Grey area: 95% confidence interval of the fold enrichment. Fold enrichments above ten are represented at ten. When this happens, the confidence interval is not represented. However the lowest edge of the confidence intervals not represented are all above one. Red dots: bootstrap test P" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-389314089" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Worldwide enrichment of iHS sweeps at VIPs compared to control non-VIPs&lt;/span&gt; A. Black line: observed fold enrichment at VIPs. Grey area: 95% confidence interval of the fold enrichment. Fold enrichments above ten are represented at ten. When this happens, the confidence interval is not represented. However the lowest edge of the confidence intervals not represented are all above one. Red dots: bootstrap test P&lt;0.001 (Methods). Dashed line: fold enrichment of one, i.e. no enrichment. Fold enrichment (y-axis) is the number of VIPs in candidate sweeps divided by the average number of control-non-VIPs in candidate sweeps. VIPs and non-VIPs in candidate sweeps are counted if they belong to the top x iHS genes (x-axis), where x is a rank threshold that slides from top 2,000 to top 20, taking in total 25 values (2,000; 1,500; 1,000; 900; 800; 700; 600; 500; 450; 400; 350; 300; 250; 200; 150; 100; 90; 80; 70; 60; 50; 40; 30; 25; 20). A fold enrichment of y=3.51 at top x=100 means that there are 3.51 times more VIPs in the top 100 iHS genes than control non-VIPs on average (over 1,000 control sets of non-VIPs). There are in fact 80 VIPs in the iHS top 100, versus only 22.7 control-non-VIPs. 80 is high compared to 100 because of the summing over all 26 human populations. Specifically, a VIP or non-VIP counts as one in the top x if it is in the top x of at least one of the 26 populations. Note that counting the number of genes instead of counting the number of sweeps ignores the clustering of multiple genes in a single sweep, but that we account for this potential bias when estimating the whole rank threshold curve significance (see Methods).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F1.medium.gif" width="440" height="201"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F1.medium.gif" width="440" height="201"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Worldwide enrichment of iHS sweeps at VIPs compared to control non-VIPs</span><p id="p-22" class="first-child">A. Black line: observed fold enrichment at VIPs. Grey area: 95% confidence interval of the fold enrichment. Fold enrichments above ten are represented at ten. When this happens, the confidence interval is not represented. However the lowest edge of the confidence intervals not represented are all above one. Red dots: bootstrap test P&lt;0.001 (Methods). Dashed line: fold enrichment of one, i.e. no enrichment. Fold enrichment (y-axis) is the number of VIPs in candidate sweeps divided by the average number of control-non-VIPs in candidate sweeps. VIPs and non-VIPs in candidate sweeps are counted if they belong to the top x iHS genes (x-axis), where x is a rank threshold that slides from top 2,000 to top 20, taking in total 25 values (2,000; 1,500; 1,000; 900; 800; 700; 600; 500; 450; 400; 350; 300; 250; 200; 150; 100; 90; 80; 70; 60; 50; 40; 30; 25; 20). A fold enrichment of y=3.51 at top x=100 means that there are 3.51 times more VIPs in the top 100 iHS genes than control non-VIPs on average (over 1,000 control sets of non-VIPs). There are in fact 80 VIPs in the iHS top 100, versus only 22.7 control-non-VIPs. 80 is high compared to 100 because of the summing over all 26 human populations. Specifically, a VIP or non-VIP counts as one in the top x if it is in the top x of at least one of the 26 populations. Note that counting the number of genes instead of counting the number of sweeps ignores the clustering of multiple genes in a single sweep, but that we account for this potential bias when estimating the whole rank threshold curve significance (see Methods).</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Using this strategy, we find that VIPs are enriched for recent selected sweeps with confidence intervals clearly above the no-enrichment mark (fold enrichment of one in <a id="xref-fig-1-2" class="xref-fig" href="#F1">figure 1A,B</a>, dashed horizontal line). VIPs are enriched for genes from the top 2,000 of iHS (<a id="xref-fig-1-3" class="xref-fig" href="#F1">figure 1A,B</a>), suggesting an excess of weak to moderate hitchhiking events. The relative enrichment shown in <a id="xref-fig-1-4" class="xref-fig" href="#F1">figure 1</a> at top 2,000 corresponds to an absolute number of 527 additional VIPs compared to control expectations. VIPs are however particularly enriched for strong selective sweep signals in the top 100 of iHS or less (<a id="xref-fig-1-5" class="xref-fig" href="#F1">figure 1A</a>), suggesting that viruses drove particularly strong adaptive events in recent human evolution. The iHS to 100 includes 80 VIPs in sweeps in different human populations, versus only 22.7 expected by chance according to the bootstrap test. The enrichment observed for the top 100 thus cannot explain alone the enrichment observed for the top 2,000 (57 vs. 527 additional VIPs, respectively). Overall, the enrichments from the top 2,000 to the top 20 are collectively highly significant and robust to issues such as clustering of multiple VIPs in the same sweep (P&lt;0.001; see Methods describing how we estimate the significance of the whole enrichment curve and not just at one rank threshold). These results suggest that viruses may have driven multiple strong adaptive events during that past ~50,000 years of human evolution, and are consistent with our previous results showing a high enrichment of adaptive events at VIPs at different evolutionary time scales (<a id="xref-ref-8-7" class="xref-bibr" href="#ref-8">Enard et al., 2016</a>; <a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Enard and Petrov, 2018</a>; <a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Uricchio et al., 2019</a>).</p></div><div id="sec-5" class="subsection"><h3>RNA viruses drove more recent selective sweeps than DNA viruses</h3><p id="p-24">The strong excess of iHS selective sweeps at VIPs suggests that there may be enough statistical power to cut the set of VIPs further into smaller categories, and ask which types of viruses drove this signal. Specific viruses may have indeed driven more epidemics than others in recent evolution, and a completely homogeneous distribution of sweeps between the VIPs of distinct viruses would be surprising.</p><p id="p-25">As we mentioned previously, RNA viruses have been responsible for the vast majority of recorded zoonoses in human populations (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Geoghegan et al., 2017</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kreuder Johnson et al., 2015</a>) and are often pathogenic (Ebola, SARS, Dengue, Zika, Influenza, HIV, etc.). Since the VIPs of specific viruses can be used as proxies for their ancient viral relatives, we use 2,691 RNA VIPs and 2,604 DNA VIPs as proxies for looking at sweeps left by ancient RNA viruses or by ancient DNA viruses, respectively.</p><p id="p-26">We estimate the enrichment of recent sweeps at RNA VIPs by comparing RNA VIPs with all other protein coding genes far (&gt;500kb) from RNA VIPs, including both non-VIPs and other VIPs that do not interact with RNA viruses. Similarly, we estimate the enrichment of recent sweeps at DNA VIPs by comparing DNA VIPs with all protein coding genes located far away (&gt;500kb). In short, we test if what matters for observing a sweep enrichment is being close to RNA VIPs, or being close to DNA VIPs. We use the bootstrap test again to match confounding factors exactly the same way we did when comparing all VIPs and non-VIPs.</p><p id="p-27">Counting sweeps and summing them over all the 26 populations from the 1,000 Genomes Project, we find a substantial enrichment in strong selective sweeps at RNA VIPs (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>; whole enrichment curve P&lt;0.001). This enrichment is reminiscent of the one observed comparing all VIPs and non-VIPs (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1A,B</a>). Conversely, when we compare DNA VIPs with genes far from them, we do not observe any enrichment of strong selective sweeps (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig 2B</a>). We nevertheless observe some enrichment when using the rank thresholds from top 2,000 to top 500, suggesting weaker sweeps signals at DNA VIPs. These results suggest that RNA viruses exerted a more drastic selective pressure driving a larger number of strong selective events compared to DNA viruses during recent human evolution. The past 50,000 years of human evolution might thus echo the very strong skew of recorded zoonoses toward RNA viruses.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sweep enrichment at RNA VIPs and DNA VIPs Legend same as figure 1. A. Sweep enrichment at RNA VIPs compared to other genes far from RNA VIPs (&gt;500kb). The enrichment exceeds tenfold at iHS top 30 and 25, and is represented at ten without confidence intervals. B. Sweep enrichment at DNA VIPs compared to other genes far from DNA VIPs (&gt;500kb). Red dots: bootstrap test P" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-389314089" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Sweep enrichment at RNA VIPs and DNA VIPs&lt;/span&gt; Legend same as figure 1. A. Sweep enrichment at RNA VIPs compared to other genes far from RNA VIPs (&gt;500kb). The enrichment exceeds tenfold at iHS top 30 and 25, and is represented at ten without confidence intervals. B. Sweep enrichment at DNA VIPs compared to other genes far from DNA VIPs (&gt;500kb). Red dots: bootstrap test P&lt;0.001. Orange dots: P&lt;0.05.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F2.medium.gif" width="440" height="203"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F2.medium.gif" width="440" height="203"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Sweep enrichment at RNA VIPs and DNA VIPs</span><p id="p-28" class="first-child">Legend same as <a id="xref-fig-1-7" class="xref-fig" href="#F1">figure 1</a>. A. Sweep enrichment at RNA VIPs compared to other genes far from RNA VIPs (&gt;500kb). The enrichment exceeds tenfold at iHS top 30 and 25, and is represented at ten without confidence intervals. B. Sweep enrichment at DNA VIPs compared to other genes far from DNA VIPs (&gt;500kb). Red dots: bootstrap test P&lt;0.001. Orange dots: P&lt;0.05.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Host functions do not explain the sweep enrichment at VIPs or at RNA VIPs</h3><p id="p-29">Even though we control for a number of confounding factors in the previous comparisons (VIPs vs. non-VIPs, RNA VIPs vs. other genes and DNA VIPs vs. other genes), we have so far not controlled for the potential confounding effect of the host biological functions on sweep enrichments. Because the representation of specific host functions can be different in VIPs and non-VIPs, these host functions rather than interactions with viruses might explain the sweep enrichment at VIPs, specifically at RNA VIPs. Specific host functions may indeed be enriched in selective sweeps, and also over-represented in VIPs compared to non-VIPs (or RNA VIPs compared to other genes). For example, one might think of a hypothetical case where VIPs are enriched for sweeps not because they interact with viruses, but because they are enriched in cell cycle proteins, and cell cycle proteins are enriched for sweeps regardless of whether they interact with viruses or not.</p><p id="p-30">To test the possibility that host functions may explain our results instead of viruses, we use functional annotations from the Gene Ontology (GO) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gene Ontology, 2015</a>) and first ask which GO biological annotations are enriched in VIPs compared to non-VIPs (at least 100% enrichment, lower enrichments are unlikely to explain the several fold excess of sweeps observed at VIPs). We then build a large group of genes, each gene belonging to at least one of the enriched GO annotations. We then use the bootstrap test to compare sweeps in this group with sweeps in control genes that belong to none of the VIP-enriched GO annotations (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>). If GO annotations rather than viruses explain our results, we expect the genes from the VIP-enriched GO annotations to be significantly enriched for sweeps compared to other genes. Because the confounding effect should be independent from viruses, we should be able to observe the confounding sweep enrichment using only non-VIPs, after having completely excluded VIPs from the comparison test. When we use the bootstrap test to compare non-VIPs within VIP-enriched GO annotations, and control non-VIPs, we do not find any particular sweep enrichment (whole sliding rank threshold curve P=0.27; see Methods). We do not detect any sweep enrichment when testing with GO annotations over-represented (100% or more) in RNA VIPs either (whole curve P=0.62). These tests show that the host intrinsic biological functions are unlikely to explain our results, which further supports the role of viruses, and in particular RNA viruses, in the observed patterns of recent adaptation.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Rationale of the test for confounding host intrinsic functions If host functions cause the enrichment in adaptation at VIPs, then these over-represented host functions in VIPs concentrate the bulk of adaptation in VIPs but also in non-VIPs. This implies that non-VIPs in those host functions should exhibit more adaptation than non-VIPs in other host functions (right side of the table). If there is no difference (left side of the table), then host functions do not confound our analysis, and viruses are likely causal." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-389314089" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Rationale of the test for confounding host intrinsic functions&lt;/span&gt; If host functions cause the enrichment in adaptation at VIPs, then these over-represented host functions in VIPs concentrate the bulk of adaptation in VIPs but also in non-VIPs. This implies that non-VIPs in those host functions should exhibit more adaptation than non-VIPs in other host functions (right side of the table). If there is no difference (left side of the table), then host functions do not confound our analysis, and viruses are likely causal.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F3.medium.gif" width="440" height="270"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F3.medium.gif" width="440" height="270"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Rationale of the test for confounding host intrinsic functions</span><p id="p-31" class="first-child">If host functions cause the enrichment in adaptation at VIPs, then these over-represented host functions in VIPs concentrate the bulk of adaptation in VIPs but also in non-VIPs. This implies that non-VIPs in those host functions should exhibit more adaptation than non-VIPs in other host functions (right side of the table). If there is no difference (left side of the table), then host functions do not confound our analysis, and viruses are likely causal.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-7"><h2 class="">Discussion</h2><p id="p-32">We find a sharp difference in recent sweep enrichment at VIPs compared to non-VIPs, and more specifically at RNA VIPs compared to DNA VIPs worldwide. Even though the enrichment of sweeps at RNA VIPs is very unlikely to have happened by chance alone, or because of confounding factors, our results must be considered as preliminary. Because the enrichment analysis we conducted represents a correlation of sweeps signals with the location of RNA VIPs in the genome, it does not fully establish that viruses caused the sweeps. Establishing causality will require more work in the following future directions.</p><p id="p-33">First, identifying the variants causal to the sweeps will help establishing plausible functional mechanisms of adaptation in response to viruses. For example, several selective sweeps might have been driven by causal non-synonymous variants localized in the contact interface between the host VIP and a viral protein, at a site in the contact interface known to be important for the physical interaction. Alternatively, the sweep causal variants might correspond to expression quantitative trait loci (eQTLs), with the selected alleles corresponding to either alleles increasing the expression of antiviral VIPs, or to alleles decreasing the expression of proviral VIPs that a virus needs to replicate. In short, the identification of causal functional alleles affecting coding or regulatory sequences in a way expected to be detrimental to viral replication will provide clear support to the causal role of viruses behind the sweep enrichment observed so far. A potential obstacle is that we specifically observe strong enrichment in large selective sweeps driven by strong recent adaptation. This may complicate the identification of the causal variants if the large sweeps include many linked variants. That said, new methods were recently published with increased power and accuracy to isolate the causal variants even in large sweeps (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akbari et al., 2018</a>).</p><p id="p-34">Second, we have so far identified a worldwide sweep enrichment at RNA VIPs, considering RNA viruses as one broad category. However, specific viruses are expected to have driven selective sweeps in specific human populations. Detecting sweep enrichments in specific human populations at VIPs that interact with specific RNA viruses may therefore further support causality, in addition to further identifying which viruses were particularly active during ancient epidemics. An important obstacle to detecting the effect of specific viruses in specific human populations is that the set of VIPs that interact with a specific virus can be substantially smaller than the set of all VIPs or RNA and DNA VIPs. For example, of the 2691 RNA VIPs, only 215 interact with Ebola virus. In order to make up for the loss of statistical power that is inevitable when using smaller samples of VIPs, we will have to use sweep detection tools beyond the single iHS statistic, with improved power and accuracy. Specific machine learning approaches that combine multiple summary statistics may offer increased performance compared to the use of a single statistic (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Schrider and Kern, 2016</a>, <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">2017</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Sugden et al., 2018</a>).</p><p id="p-35">Third, if viruses were causal, we expect that the selective sweeps at VIPs of a specific virus would have happened around the same evolutionary time within a specific human population, which would correspond to the time of an epidemic, as opposed to being dispersed randomly across different evolutionary times with no connection to a specific epidemic event. Dating selective sweeps linked to a specific virus, with a significant convergence in individually estimated evolutionary times, may thus strengthen the evidence that viruses were causal, in addition to providing estimates of when they drove adaptation. Using combinations of summary statistics instead of only one may again provide more accuracy when dating selective events with approaches such as Approximate Bayesian Computation (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Nakagome et al., 2016</a>).</p><p id="p-36">Finally, the findings that both recent selective sweeps and adaptive introgression from Neanderthals to Eurasian modern humans were dominated by RNA viruses, raises the important question of whether or not RNA viruses systematically drove more adaptation during most of human evolution, and more broadly during the evolution of other species. More work is now needed to quantify the impact of RNA viruses compared to DNA viruses at deeper evolutionary time scales in humans, using for example recent implementations of the McDonald-Kreitman test (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Galtier, 2016</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Tataru et al., 2017</a>; <a id="xref-ref-43-6" class="xref-bibr" href="#ref-43">Uricchio et al., 2019</a>), and to quantify the impact of RNA viruses in other species both during recent and deeper evolution. It will also be interesting to see if the genes involved in recent human adaptation to RNA viruses, were also more often involved in adaptation over deeper evolutionary time scales, which would highlight genes that may be particularly important for host adaptation. Because RNA molecules from ancient RNA viruses may prove very hard to impossible to recover, our results potentially illustrate the importance of pursuing host genome-based, indirect approaches to uncover important pathogenic players in host evolution.</p></div><div class="section" id="sec-8"><h2 class="">Methods</h2><div id="sec-9" class="subsection"><h3>Measuring iHS and power analysis</h3><p id="p-37">The iHS summary statistic is computed for each variant in a genome with a minor allele frequency greater than 5%, and known derived and ancestral alleles. For our analysis we used the hapbin software (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Maclean et al., 2015</a>) to rapidly measure iHS for variants in all the 26 separate human populations represented in the 1,000 Genomes Project phase 3. We then computed the average of |iHS| (iHS can take both highly negative or positive values that both indicate adaptation) across all the variants within large 1,000kb windows. Large 1,000kb windows are more specifically sensitive to large sweeps driven by strong adaptation (see power analysis below). To assign a iHS rank to each gene, each window was centered at the genomic center of a gene, half-way between the most upstream transcription start site and the most downstream transcription end site. The gen coordinates were obtained from Ensembl v83 (ref). This window configuration avoids introducing biases related to gene length. Indeed, one can imagine an alternative way to rank genes by iHS where the value used to rank genes is the average iHS measured only for variants that overlap a gene. The problem with this is that longer genes would then be more likely to overlap high local |iHS| values just by chance compared to shorter genes, thus biasing power in favor of larger genes. For this reason, we prefer to use a constant window size.</p><p id="p-38">To determine a window size that would more specifically detect strong adaptation in the human genome, we ran population simulations of incomplete selective sweeps and measured the power of iHS to detect weak or strong adaptation when using different window sizes. In particular, we used two window sizes, with small 50kb windows and much larger 1,000kb windows. We ran the population simulations using discoal (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kern and Schrider, 2016</a>). Each simulation included 50 individuals (100 chromosomes) and represented a locus of 1.2 megabases with a uniform recombination rate of 1 cM/Mb, and a total cumulated theta of 1,800 to match the average recombination rate and diversity observed in the 1,000 Genomes Project populations. In order to measure the statistical power of iHS windows, we first had to determine the distribution of the corresponding iHS values in the neutral case without selective sweeps. This distribution was obtained running 10,000 independent neutral simulations. Note we simulated a constant population size of 10,000 since fluctuations in population size are not likely to change the difference in power between small or large windows of iHS. We simulated a large number of neutral loci to get precise power estimates even at low false positive rates. Specifically, we estimated power at a low, 0.1% false positive rate.</p><p id="p-39"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows the power of the average |iHS| to detect a range of incomplete selective sweeps (see figure legend) using either 50kb or 1,000kb windows. From the figure it is clear that 50kb windows have good power to detect sweeps for selection intensities between 2Ns=100 and 1,000, whereas 1,000 kb windows only have good power to detect sweeps with selection intensities greater than 200 (1% selection coefficient in the human genome). Large, 1,000kb windows are therefore more appropriate to detect more specifically strong adaptation events expected from interactions with viruses. For this reason, we used 1,000kb windows for the whole analysis.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Power of iHS windows to detect various sweeps. A. Incomplete selective sweeps from a de-novo mutation that reached a 50% frequency. B. Sweep from a standing, 5% standing variant at the start of selection and that reached a 50% frequency. C. and D. same as A. and B., respectively, but for sweeps that reached a 70% instead of 50% frequency. The y axis represents the statistical power (true positive rate) at 0.1% false positive rate (FPR). The x axis represents the range of simulated selection intensities, ranging from 2Ns=50 to 2Ns=1,000 where N=10,000 in our simulations, and s is the selection coefficient. Blue curve: power with 50kb windows. Orange curve: power with 1,000kb windows. Grey curve: ratio of the power with 1,000kb windows over the power with 50kb windows." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-389314089" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Power of iHS windows to detect various sweeps.&lt;/span&gt; A. Incomplete selective sweeps from a de-novo mutation that reached a 50% frequency. B. Sweep from a standing, 5% standing variant at the start of selection and that reached a 50% frequency. C. and D. same as A. and B., respectively, but for sweeps that reached a 70% instead of 50% frequency. The y axis represents the statistical power (true positive rate) at 0.1% false positive rate (FPR). The x axis represents the range of simulated selection intensities, ranging from 2Ns=50 to 2Ns=1,000 where N=10,000 in our simulations, and s is the selection coefficient. Blue curve: power with 50kb windows. Orange curve: power with 1,000kb windows. Grey curve: ratio of the power with 1,000kb windows over the power with 50kb windows.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F4.medium.gif" width="440" height="347"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F4.medium.gif" width="440" height="347"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/20/2020.03.18.997346/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Power of iHS windows to detect various sweeps.</span><p id="p-40" class="first-child">A. Incomplete selective sweeps from a de-novo mutation that reached a 50% frequency. B. Sweep from a standing, 5% standing variant at the start of selection and that reached a 50% frequency. C. and D. same as A. and B., respectively, but for sweeps that reached a 70% instead of 50% frequency. The y axis represents the statistical power (true positive rate) at 0.1% false positive rate (FPR). The x axis represents the range of simulated selection intensities, ranging from 2Ns=50 to 2Ns=1,000 where N=10,000 in our simulations, and s is the selection coefficient. Blue curve: power with 50kb windows. Orange curve: power with 1,000kb windows. Grey curve: ratio of the power with 1,000kb windows over the power with 50kb windows.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Testing enrichments with the boostrap test</h3><p>The bootstrap test we used to match VIPs with control non-VIPs that match for multiple confounding factors has already been described extensively in a previous manuscript that the reader can refer to for more ample details (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Castellano, 2019</a>). An implementation of the bootstrap test is available at <a href="https://github.com/DavidPierreEnard">https://github.com/DavidPierreEnard</a>, as part of a larger pipeline that also estimates the whole enrichment curve p-value (see below). In brief, the bootstrap test uses a straightforward control set-building algorithm that adds control genes to a progressively growing control set, in such a way that the growing control set has the same range of values of confounding factors as the tested set of genes of interest. For this analysis, the confounding factors that were used in the bootstrap test include the following factors likely to impact the frequency of selective sweeps:
</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-42"><span class="list-label">- </span>the density of coding sequences in 50kb windows centered on genes. Coding sequences are Ensembl v83 coding sequences.</p></li><li id="list-item-2"><p id="p-43"><span class="list-label">- </span>the density of mammalian phastCons conserved elements (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Siepel et al., 2005</a>) (in 50kb windows), downloaded from the UCSC Genome Browser (<a href="https://genome.ucsc.edu/">https://genome.ucsc.edu/</a>).</p></li><li id="list-item-3"><p id="p-44"><span class="list-label">- </span>the density of regulatory elements, as measured by the density of DNASE1 hypersensitive sites (in 50kb windows) also from the UCSC Genome Browser.</p></li><li id="list-item-4"><p id="p-45"><span class="list-label">- </span>GC content (in 50kb windows).</p></li><li id="list-item-5"><p id="p-46"><span class="list-label">- </span>recombination rate from Hinch et al. (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hinch et al., 2011</a>) in 200kb windows. We used 200kb windows instead of 50kb windows to reduce the noise in recombination rate estimates.</p></li><li id="list-item-6"><p id="p-47"><span class="list-label">- </span>average overall expression in 53 GTEx v7 tissues (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Consortium, 2015</a>) (<a href="https://www.gtexportal.org/home/">https://www.gtexportal.org/home/</a>). We used the log (in base 2) of RPKM values.</p></li><li id="list-item-7"><p id="p-48"><span class="list-label">- </span>expression (log base 2 of RPKM) in GTEx lymphocytes. Expression in immune tissues is likely to impact the rate of sweeps.</p></li><li id="list-item-8"><p id="p-49"><span class="list-label">- </span>expression (log base 2 of RPKM) in GTEx testis. Expression in testis is also likely to impact the rate of sweeps.</p></li><li id="list-item-9"><p id="p-50"><span class="list-label">- </span>the number of protein-protein interactions (PPIs) in the human protein interaction network (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Luisi et al., 2015</a>). The number of PPIs has been shown to influence the rate of sweeps (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Luisi et al., 2015</a>). We use the log (base 2) of the number of PPIs.</p></li><li id="list-item-10"><p id="p-51"><span class="list-label">- </span>the proportion of immune genes. The control sets have the same proportion of immune genes as VIPs, as annotated by the GO terms GO:0002376 (immune system process), GO:0006952 (defense response) and GO:0006955 (immune response), to avoid confusing the effect of viruses with broader immune effects.</p></li><li id="list-item-11"><p id="p-52"><span class="list-label">- </span>the proportion of genes that interact with bacteria according to the Intact database as of April 2018 (<a href="https://www.ebi.ac.uk/intact/">https://www.ebi.ac.uk/intact/</a>). Note that we also attempted to match the proportion of genes that interact with Plasmodium (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Ebel et al., 2017</a>), but VIPs and non-VIPs had such different numbers of Plasmodium-Interacting genes that the bootstrap test failed to match them. However, we were able to use the bootstrap test to verify that plasmodium-interacting genes do not have more iHS sweeps than other genes (bootstrap test P&gt;0.05 for all thresholds from top 2,000 to top 20).</p></li></ul><p id="p-53">It is important to note that we match many (11) confounding factors, which inevitably restricts the number of non-VIPs that can be used as controls. Smaller numbers of control non-VIPs increase the false positive rate of the bootstrap test (see below). However, the False Discovery Rate analysis strategy that we use to test the significance of the whole curve enrichment fully takes this limitation into account (see below). Note also that in order to limit the number of false positive sweeps due to low recombination regions included in the bootstrap test, we did not use genes with recombination rates estimated to be lower than 0.2 cM/Mb (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hinch et al., 2011</a>).</p></div><div id="sec-11" class="subsection"><h3>Testing the significance of the whole enrichment curve</h3><p id="p-54">In addition to estimating a p-value for each iHS rank threshold from top 2,000 to top 20 with the bootstrap test, we also estimated how significantly the whole enrichment curve stands above null expectations. We do this for multiple reasons, some of which already mentioned above. First, the different thresholds across the whole curve are not independent from each other. As a consequence, an excess of significant p-values at multiple thresholds may reflect a correlation of false positives due to the dependence between thresholds. Second and most importantly, it is very likely that we used the bootstrap test in conditions that make it non-nominal (meaning a non-uniform distribution of p-values under the null hypothesis of no enrichment). Indeed, as we mentioned in the results, we had to exclude many non-VIPs too close to VIPs (&lt;500kb) to avoid counting VIP sweeps as non-VIP sweeps, and vice versa. We also used only non-VIPs that could match VIPs for a large number of confounding factors. The issue then is that we only had a limited number of non-VIPs that were far enough from VIPs and that could be used as controls. For example, for the VIPs vs. non-VIPs test, we could use only 1,704 control non-VIPs. A smaller set of controls means that in the bootstrap test, the same control non-VIP has to be re-sampled more times (it is a bootstrap), thus resulting in a smaller set of control non-VIPs. Smaller samples of control non-VIPs are inevitably related to (i) a higher variance of the control null distribution, and (ii) also to a higher variance of the overall average of the null distribution. Bias (i) can result in a higher rate of false negatives with a decreased power to detect an significant sweep enrichment, but more alarmingly bias (ii) can on the contrary inflate the rate of false positive bootstrap tests if the average of the small control sets is far from what the average would be for ideal, very large control sets.</p><p id="p-55">This is a very serious limitation of our analysis, but fortunately there is a very simple, yet computationally intensive remedy. Since we know that the bootstrap test is likely not nominal, we can estimate the true False Discovery Rate associated with the whole approach, by re-running the entire analysis pipeline many times on randomized genomes where the iHS ranks have been swapped randomly between genes. We specifically use 10,000 random genomes. We can then estimate how significant the real enrichment curve from top 2,000 to top 20 is compared to the same curved measured on the 10,000 random genomes. As a statistic to estimate the significance of the whole curve, we use the difference of the observed number of VIPs, minus the expected number according to controls, and we sum this difference over all rank thresholds from top 2,000 to top 20 (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a>). We then compare the real value of the statistic for the real genome, with the distribution of 10,000 random values obtained from the 10,000 randomized genomes. For each randomized genome, we use exactly the same VIPs and control non-VIPs as we did when testing the real genome, but these genes are now associated with randomly swapped iHS ranks. The sample size of the control sets of non-VIPs is exactly the same using random genomes as when using the real genome, meaning that the results of the boostrap test reproduce exactly the same biases for the random genomes as when testing the real genome. The p-value obtained for the whole rank thresholds enrichment curve after 10,000 randomized genomes is thus an unbiased, nominal p-value that matches the actual False Discovery Rate.</p><p id="p-56">It is however very important to note that it is true that the randomized genomes provide an unbiased, whole curve test, only because we do not randomize genomes in a completely random fashion. If we just swapped randomly iHS ranks between genes, we would lose the very important property of the real genome that genes with top iHS ranks are likely to be neighbors and to form clusters. Indeed, selective sweeps, and especially strong large selective sweeps, can overlap with multiple genes in the human genome, thus creating clusters of neighboring high-ranking iHS genes. In our case, it was therefore crucial to randomize iHS ranks between genes in a way that conserved the same exact clustering structure between top iHS rank genes. To achieve this, we cut the genes ordered as they are across human chromosomes in 100 blocks of contiguous genes, and then randomly shuffled these blocks. The size of the blocks is much larger than the size of even very large sweeps of multiple megabases, thus ensuring that the clustering structure of the top iHS rank genes is preserved within the blocks. Randomly swapping the blocks still results in randomly swapping iHS ranks between genes within the blocks. Because the randomized genomes obtained this way preserve the clustering of iHS signals, we can still measure enrichments using the number of genes, instead of having to count the number of sweeps. Note that the entire pipeline to get the whole enrichment curve p-value is available at <a href="https://github.com/DavidPierreEnard">https://github.com/DavidPierreEnard</a> with a user manual, together with the bootstrap test.</p></div><div id="sec-12" class="subsection"><h3>Testing the involvement of host intrinsic functions with GO categories</h3><p id="p-57">To test whether host intrinsic functions can explain the sweep enrichment at VIPs and particularly at RNA VIPs, we measured enrichment for sweeps among genes that belong to GO functions that are over-represented in VIPs. Specifically, we tested whether genes that belong to over-represented functions in VIPs are enriched for sweeps completely independently of whether they interact with viruses or not, by completely excluding VIPs and genes at less than 500kb from VIPs (and more likely to overlap VIP sweeps) from the analysis. Thus, we used the bootstrap test to compare iHS sweeps in 718 non-VIPs more than 500kb from VIPs that belong to over-represented GO functions in VIPs (GO functions had to be found in 50 VIPs or more, 100% over-represented or more), with sweeps in 423 non-VIPs more than 500kb from VIPs that do not belong to over-represented GO functions in VIPs. The 718 non-VIPs with over-represented GO functions are themselves at least 500kb from non-VIPs with no over-represented GO function to avoid the issue of sweeps overlapping the two categories. For RNA VIPs, we used 1,726 nonRNA-VIPs within over-represented GO functions, and 357 nonRNA-VIPs with no over-represented GO function. We ran the bootstrap test testing for a deficit of sweeps in the 423 non-VIPs out of over-represented GO functions using the larger number of non-VIPs with at least an over-represented function (the larger the pool of controls compared to the tested set, the better). We also flipped the two groups when testing GO functions for RNA VIPs for the same reason that it is always better to have the larger group being the control group in the bootstrap test. We estimated the significance of the bootstrap test by measuring the whole enrichment curve p-value as already described above. We used the same confounding factors to match during the bootstrap test as we previously did to match VIPs with control non-VIPs.</p></div><div id="sec-13" class="subsection"><h3>Functional relevance of VIPs</h3><p id="p-58">To estimate the functional relevance of VIPs for the viral replication cycle, we counted how many manually curated VIPs from the first 200 rows in Table S1 had clear reported proviral or antiviral effects when their function was experimentally perturbed either via expression perturbation or via protein function perturbation. The experimental perturbations had to be reported in the publications listed in Table S1. We found that 66% of the interactions with VIPs had clear functional consequences for the viral replication cycle.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-dataset fn-group-external-links" id="fn-1"><p id="p-5">
<a href="https://github.com/DavidPierreEnard/Gene_Set_Enrichment_Pipeline">https://github.com/DavidPierreEnard/Gene_Set_Enrichment_Pipeline</a>
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.1" data-doi="10.1126/science.1209202"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Abi-Rached</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jobin</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kulkarni</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McWhinnie</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dalva</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gragert</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Babrzadeh</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gharizadeh</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plummer</span>,  <span class="cit-name-given-names">F.A.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">The shaping of modern human immune systems by multiregional admixture with archaic humans</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">334</span>, <span class="cit-fpage">89</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1209202%26rft_id%253Dinfo%253Apmid%252F21868630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzQvNjA1Mi84OSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzIwLzIwMjAuMDMuMTguOTk3MzQ2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.2" data-doi="10.1038/nmeth.4606"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Akbari</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vitti</span>,  <span class="cit-name-given-names">J.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Iranmehr</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bakhtiari</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sabeti</span>,  <span class="cit-name-given-names">P.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mirarab</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bafna</span>,  <span class="cit-name-given-names">V.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Identifying the favored mutation in a positive selective sweep</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">279</span>–<span class="cit-lpage">282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D15%26rft.spage%253D279%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4606&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.3" data-doi="10.1371/journal.pgen.1000562"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barreiro</span>,  <span class="cit-name-given-names">L.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ben-Ali</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quach</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laval</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patin</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pickrell</span>,  <span class="cit-name-given-names">J.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bouchier</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tichit</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neyrolles</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gicquel</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Evolutionary dynamics of human Toll-like receptors and their different contributions to host defense</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">e1000562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBarreiro%26rft.auinit1%253DL.%2BB.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000562%26rft.epage%253De1000562%26rft.atitle%253DEvolutionary%2Bdynamics%2Bof%2Bhuman%2BToll-like%2Breceptors%2Band%2Btheir%2Bdifferent%2Bcontributions%2Bto%2Bhost%2Bdefense.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000562%26rft_id%253Dinfo%253Apmid%252F19609346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000562&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19609346&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.4" data-doi="10.1093/infdis/jiw207"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Breman</span>,  <span class="cit-name-given-names">J.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heymann</span>,  <span class="cit-name-given-names">D.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lloyd</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCormick</span>,  <span class="cit-name-given-names">J.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miatudila</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murphy</span>,  <span class="cit-name-given-names">F.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muyembe-Tamfun</span>,  <span class="cit-name-given-names">J.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Piot</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruppol</span>,  <span class="cit-name-given-names">J.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sureau</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Discovery and Description of Ebola Zaire Virus in 1976 and Relevance to the West African Epidemic During 2013-2016</span>. <abbr class="cit-jnl-abbrev">J Infect Dis</abbr> <span class="cit-vol">214</span>, <span class="cit-fpage">S93</span>–<span class="cit-lpage">S101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BInfect%2BDis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Finfdis%252Fjiw207%26rft_id%253Dinfo%253Apmid%252F27357339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/infdis/jiw207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27357339&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Castellano</span>,  <span class="cit-name-given-names">D.U.</span></span>, L.H.; <span class="cit-auth"><span class="cit-name-surname">Munch</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Enard</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Viruses rule over adaptation in conservd human proteins</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.6" data-doi="10.1126/science.1262110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Consortium</span>,  <span class="cit-name-given-names">G.T.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Human genomics. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">348</span>, <span class="cit-fpage">648</span>–<span class="cit-lpage">660</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DThe%2BGTEx%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D348%26rft.issue%253D6235%26rft.spage%253D648%26rft.epage%253D660%26rft.atitle%253DThe%2BGenotype-Tissue%2BExpression%2B%2528GTEx%2529%2Bpilot%2Banalysis%253A%2BMultitissue%2Bgene%2Bregulation%2Bin%2Bhumans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1262110%26rft_id%253Dinfo%253Apmid%252F25954001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDgvNjIzNS82NDgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yMC8yMDIwLjAzLjE4Ljk5NzM0Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.7" data-doi="10.1371/journal.pgen.1007023"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ebel</span>,  <span class="cit-name-given-names">E.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Telis</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Venkataram</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petrov</span>,  <span class="cit-name-given-names">D.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Enard</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">High rate of adaptation of mammalian proteins that interact with Plasmodium and related parasites</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">e1007023</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D13%26rft.spage%253De1007023%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1007023%26rft_id%253Dinfo%253Apmid%252F28957326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1007023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28957326&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Enard</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cai</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gwennap</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Petrov</span>,  <span class="cit-name-given-names">D.A.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Viruses are a dominant driver of protein adaptation in mammals</span>. <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">5</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.9" data-doi="10.1101/gr.164822.113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Enard</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Messer</span>,  <span class="cit-name-given-names">P.W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Petrov</span>,  <span class="cit-name-given-names">D.A.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Genome-wide signals of positive selection in human evolution</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">885</span>–<span class="cit-lpage">895</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.164822.113%26rft_id%253Dinfo%253Apmid%252F24619126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI0LzYvODg1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjAvMjAyMC4wMy4xOC45OTczNDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.10" data-doi="10.1016/j.cell.2018.08.034"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Enard</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Petrov</span>,  <span class="cit-name-given-names">D.A.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Evidence that RNA Viruses Drove Adaptive Introgression between Neanderthals and Modern Humans</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">175</span>, <span class="cit-fpage">360</span>–<span class="cit-lpage">371 e313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D175%26rft.spage%253D360%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.08.034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.08.034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.11" data-doi="10.1093/molbev/msu077"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ferrer-Admetlla</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">On detecting incomplete soft or hard selective sweeps using haplotype structure</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">31</span>, <span class="cit-fpage">1275</span>–<span class="cit-lpage">1291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu077%26rft_id%253Dinfo%253Apmid%252F24554778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu077&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24554778&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000335914400020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.12" data-doi="10.1038/nature14558"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fu</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hajdinjak</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moldovan</span>,  <span class="cit-name-given-names">O.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Constantin</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mallick</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skoglund</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patterson</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rohland</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lazaridis</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nickel</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">An early modern human from Romania with a recent Neanderthal ancestor</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">524</span>, <span class="cit-fpage">216</span>–<span class="cit-lpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFu%2BQiaomei%26rft.volume%253D524%26rft.issue%253D7564%26rft.spage%253D216%26rft.epage%253D219%26rft.atitle%253DAn%2Bearly%2Bmodern%2Bhuman%2Bfrom%2BRomania%2Bwith%2Ba%2Brecent%2BNeanderthal%2Bancestor%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14558%26rft_id%253Dinfo%253Apmid%252F26098372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14558&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2015099287&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=26098372&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.13" data-doi="10.1371/journal.pgen.1005774"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Galtier</span>,  <span class="cit-name-given-names">N.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Adaptive Protein Evolution in Animals and the Effective Population Size Hypothesis</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">e1005774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D12%26rft.spage%253De1005774%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005774%26rft_id%253Dinfo%253Apmid%252F26752180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005774&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26752180&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.14" data-doi="10.1371/journal.pgen.1005004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garud</span>,  <span class="cit-name-given-names">N.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Messer</span>,  <span class="cit-name-given-names">P.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buzbas</span>,  <span class="cit-name-given-names">E.O.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Petrov</span>,  <span class="cit-name-given-names">D.A.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Recent selective sweeps in North American Drosophila melanogaster show signatures of soft sweeps</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">e1005004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D11%26rft.spage%253De1005004%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005004%26rft_id%253Dinfo%253Apmid%252F25706129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25706129&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.15" data-doi="10.1093/nar/gku1179"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gene Ontology</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Gene Ontology Consortium: going forward</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">43</span>, <span class="cit-fpage">D1049</span>–<span class="cit-lpage">1056</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1179%26rft_id%253Dinfo%253Apmid%252F25428369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1179&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25428369&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.16" data-doi="10.1038/nature15393"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Genomes Project</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Auton</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brooks</span>,  <span class="cit-name-given-names">L.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Durbin</span>,  <span class="cit-name-given-names">R.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garrison</span>,  <span class="cit-name-given-names">E.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kang</span>,  <span class="cit-name-given-names">H.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korbel</span>,  <span class="cit-name-given-names">J.O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marchini</span>,  <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCarthy</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McVean</span>,  <span class="cit-name-given-names">G.A.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">A global reference for human genetic variation</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">526</span>, <span class="cit-fpage">68</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D526%26rft.spage%253D68%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature15393%26rft_id%253Dinfo%253Apmid%252F26432245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature15393&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26432245&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.17" data-doi="10.1371/journal.ppat.1006215"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Geoghegan</span>,  <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duchene</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E.C.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Comparative analysis estimates the relative frequencies of co-divergence and cross-species transmission within viral families</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">e1006215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D13%26rft.spage%253De1006215%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006215%26rft_id%253Dinfo%253Apmid%252F28178344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28178344&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.18" data-doi="10.1126/science.1188021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Green</span>,  <span class="cit-name-given-names">R.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krause</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Briggs</span>,  <span class="cit-name-given-names">A.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maricic</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stenzel</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kircher</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patterson</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhai</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fritz</span>,  <span class="cit-name-given-names">M.H.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">A draft sequence of the Neandertal genome</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">328</span>, <span class="cit-fpage">710</span>–<span class="cit-lpage">722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGreen%26rft.auinit1%253DR.%2BE.%26rft.volume%253D328%26rft.issue%253D5979%26rft.spage%253D710%26rft.epage%253D722%26rft.atitle%253DA%2BDraft%2BSequence%2Bof%2Bthe%2BNeandertal%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1188021%26rft_id%253Dinfo%253Apmid%252F20448178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjgvNTk3OS83MTAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yMC8yMDIwLjAzLjE4Ljk5NzM0Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.19" data-doi="10.1093/infdis/jix451"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gubler</span>,  <span class="cit-name-given-names">D.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vasilakis</span>,  <span class="cit-name-given-names">N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Musso</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">History and Emergence of Zika Virus</span>. <abbr class="cit-jnl-abbrev">J Infect Dis</abbr> <span class="cit-vol">216</span>, <span class="cit-fpage">S860</span>–<span class="cit-lpage">S867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BInfect%2BDis%26rft.volume%253D216%26rft.spage%253DS860%26rft_id%253Dinfo%253Adoi%252F10.1093%252Finfdis%252Fjix451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/infdis/jix451&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.20" data-doi="10.1016/j.molmed.2011.12.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hemelaar</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">The origin and diversity of the HIV-1 pandemic</span>. <abbr class="cit-jnl-abbrev">Trends Mol Med</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">182</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmolecular%2Bmedicine%26rft.stitle%253DTrends%2BMol%2BMed%26rft.aulast%253DHemelaar%26rft.auinit1%253DJ.%26rft.volume%253D18%26rft.issue%253D3%26rft.spage%253D182%26rft.epage%253D192%26rft.atitle%253DThe%2Borigin%2Band%2Bdiversity%2Bof%2Bthe%2BHIV-1%2Bpandemic.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molmed.2011.12.001%26rft_id%253Dinfo%253Apmid%252F22240486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molmed.2011.12.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22240486&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302455400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.21" data-doi="10.1038/nature10336"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hinch</span>,  <span class="cit-name-given-names">A.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tandon</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patterson</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rohland</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palmer</span>,  <span class="cit-name-given-names">C.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">G.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buxbaum</span>,  <span class="cit-name-given-names">S.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Akylbekova</span>,  <span class="cit-name-given-names">E.L.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">The landscape of recombination in African Americans</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">476</span>, <span class="cit-fpage">170</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHinch%26rft.auinit1%253DA.%2BG.%26rft.volume%253D476%26rft.issue%253D7359%26rft.spage%253D170%26rft.epage%253D175%26rft.atitle%253DThe%2Blandscape%2Bof%2Brecombination%2Bin%2BAfrican%2BAmericans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10336%26rft_id%253Dinfo%253Apmid%252F21775986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10336&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21775986&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293731900028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.22" data-doi="10.1098/rstb.2004.1478"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E.C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">Viral evolution and the emergence of SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Philos Trans R Soc Lond B Biol Sci</abbr> <span class="cit-vol">359</span>, <span class="cit-fpage">1059</span>–<span class="cit-lpage">1065</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DHolmes%26rft.auinit1%253DE.%2BC.%26rft.volume%253D359%26rft.issue%253D1447%26rft.spage%253D1059%26rft.epage%253D1065%26rft.atitle%253DViral%2Bevolution%2Band%2Bthe%2Bemergence%2Bof%2BSARS%2Bcoronavirus%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2004.1478%26rft_id%253Dinfo%253Apmid%252F15306390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2004.1478&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15306390&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222688200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">K.E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Voight</span>,  <span class="cit-name-given-names">B.F.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Patterns of shared signatures of recent positive selection across human populations</span>. <abbr class="cit-jnl-abbrev">Nat Ecol Evol</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">713</span>–<span class="cit-lpage">720</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BEcol%2BEvol%26rft.volume%253D2%26rft.spage%253D713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.24" data-doi="10.1073/pnas.1208059110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">B.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grace</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kock</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alonso</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rushton</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Said</span>,  <span class="cit-name-given-names">M.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McKeever</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mutua</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Young</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McDermott</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Zoonosis emergence linked to agricultural intensification and environmental change</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">110</span>, <span class="cit-fpage">8399</span>–<span class="cit-lpage">8404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1208059110%26rft_id%253Dinfo%253Apmid%252F23671097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzIxLzgzOTkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yMC8yMDIwLjAzLjE4Ljk5NzM0Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.25" data-doi="10.1093/bioinformatics/btw556"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kern</span>,  <span class="cit-name-given-names">A.D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Schrider</span>,  <span class="cit-name-given-names">D.R.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Discoal: flexible coalescent simulations with selection</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">3839</span>–<span class="cit-lpage">3841</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw556%26rft_id%253Dinfo%253Apmid%252F27559153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27559153&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.26" data-doi="10.1038/srep14830"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kreuder Johnson</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hitchens</span>,  <span class="cit-name-given-names">P.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smiley Evans</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldstein</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomas</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clements</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joly</span>,  <span class="cit-name-given-names">D.O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wolfe</span>,  <span class="cit-name-given-names">N.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daszak</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karesh</span>,  <span class="cit-name-given-names">W.B.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Spillover and pandemic properties of zoonotic viruses with high host plasticity</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">14830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D5%26rft.spage%253D14830%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep14830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep14830&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.27" data-doi="10.1093/gbe/evv055"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luisi</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alvarez-Ponce</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pybus</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fares</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bertranpetit</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Laayouni</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Recent positive selection has acted on genes encoding proteins with more interactions within the whole human interactome</span>. <abbr class="cit-jnl-abbrev">Genome Biol Evol</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">1141</span>–<span class="cit-lpage">1154</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv055%26rft_id%253Dinfo%253Apmid%252F25840415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evv055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25840415&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.28" data-doi="10.1093/molbev/msv172"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maclean</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chue Hong</span>,  <span class="cit-name-given-names">N.P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Prendergast</span>,  <span class="cit-name-given-names">J.G.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">hapbin: An Efficient Program for Performing Haplotype-Based Scans for Positive Selection in Large Genomic Datasets</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">3027</span>–<span class="cit-lpage">3029</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv172%26rft_id%253Dinfo%253Apmid%252F26248562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msv172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26248562&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.29" data-doi="10.1038/351652a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McDonald</span>,  <span class="cit-name-given-names">J.H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kreitman</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">1991</span>). <span class="cit-article-title">Adaptive protein evolution at the Adh locus in Drosophila</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">351</span>, <span class="cit-fpage">652</span>–<span class="cit-lpage">654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMcDonald%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D351%26rft.issue%253D6328%26rft.spage%253D652%26rft.epage%253D654%26rft.atitle%253DAdaptive%2Bprotein%2Bevolution%2Bat%2Bthe%2BAdh%2Blocus%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F351652a0%26rft_id%253Dinfo%253Apmid%252F1904993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/351652a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1904993&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FT11200066&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.30" data-doi="10.1177/0141076815605211"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Metzger</span>,  <span class="cit-name-given-names">W.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kohler</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mordmuller</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Lessons from a modern review of the smallpox eradication files</span>. <abbr class="cit-jnl-abbrev">J R Soc Med</abbr> <span class="cit-vol">108</span>, <span class="cit-fpage">473</span>–<span class="cit-lpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BR%2BSoc%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1177%252F0141076815605211%26rft_id%253Dinfo%253Apmid%252F26432815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/0141076815605211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26432815&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.31" data-doi="10.1093/molbev/msv256"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nakagome</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alkorta-Aranburu</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Amato</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Howie</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peter</span>,  <span class="cit-name-given-names">B.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hudson</span>,  <span class="cit-name-given-names">R.R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Di Rienzo</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Estimating the Ages of Selection Signals from Different Epochs in Human History</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">33</span>, <span class="cit-fpage">657</span>–<span class="cit-lpage">669</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv256%26rft_id%253Dinfo%253Apmid%252F26545921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msv256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26545921&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.32" data-doi="10.1038/nrg3954"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paabo</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">The diverse origins of the human gene pool</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">313</span>–<span class="cit-lpage">314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BGenet%26rft.volume%253D16%26rft.spage%253D313%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3954%26rft_id%253Dinfo%253Apmid%252F25982166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3954&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25982166&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Petti</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lodi</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">The controversial natural history of oral Herpes Simplex Virus Type 1 infection</span>. <abbr class="cit-jnl-abbrev">Oral Dis.</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.34" data-doi="10.1038/nature12961"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mallick</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dannemann</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prufer</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kelso</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paabo</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patterson</span>,  <span class="cit-name-given-names">N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Reich</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">The genomic landscape of Neanderthal ancestry in present-day humans</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">507</span>, <span class="cit-fpage">354</span>–<span class="cit-lpage">357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D507%26rft.spage%253D354%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12961%26rft_id%253Dinfo%253Apmid%252F24476815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12961&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24476815&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000333029000033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.35" data-doi="10.1371/journal.pgen.1005928"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schrider</span>,  <span class="cit-name-given-names">D.R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kern</span>,  <span class="cit-name-given-names">A.D.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">S/HIC: Robust Identification of Soft and Hard Sweeps Using Machine Learning</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">e1005928</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D12%26rft.spage%253De1005928%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005928%26rft_id%253Dinfo%253Apmid%252F26977894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005928&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26977894&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.36" data-doi="10.1093/molbev/msx154"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schrider</span>,  <span class="cit-name-given-names">D.R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kern</span>,  <span class="cit-name-given-names">A.D.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Soft Sweeps Are the Dominant Mode of Adaptation in the Human Genome</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">34</span>, <span class="cit-fpage">1863</span>–<span class="cit-lpage">1877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft.volume%253D34%26rft.spage%253D1863%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx154%26rft_id%253Dinfo%253Apmid%252F28482049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx154&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28482049&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.37" data-doi="10.1016/j.tig.2017.12.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schrider</span>,  <span class="cit-name-given-names">D.R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kern</span>,  <span class="cit-name-given-names">A.D.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Supervised Machine Learning for Population Genetics: A New Paradigm</span>. <abbr class="cit-jnl-abbrev">Trends Genet</abbr> <span class="cit-vol">34</span>, <span class="cit-fpage">301</span>–<span class="cit-lpage">312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet%26rft.volume%253D34%26rft.spage%253D301%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2017.12.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2017.12.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.38" data-doi="10.1101/gr.3715005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Siepel</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bejerano</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pedersen</span>,  <span class="cit-name-given-names">J.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hinrichs</span>,  <span class="cit-name-given-names">A.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hou</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosenbloom</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clawson</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spieth</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hillier</span>,  <span class="cit-name-given-names">L.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richards</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">1034</span>–<span class="cit-lpage">1050</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3715005%26rft_id%253Dinfo%253Apmid%252F16024819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE1LzgvMTAzNCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzIwLzIwMjAuMDMuMTguOTk3MzQ2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Spyrou</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bos</span>,  <span class="cit-name-given-names">K.I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Herbig</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Krause</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Ancient pathogen genomics as an emerging tool for infectious disease research</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">323</span>–<span class="cit-lpage">340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BGenet%26rft.volume%253D20%26rft.spage%253D323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.40" data-doi="10.1038/s41467-018-03100-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sugden</span>,  <span class="cit-name-given-names">L.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atkinson</span>,  <span class="cit-name-given-names">E.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fischer</span>,  <span class="cit-name-given-names">A.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rong</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Henn</span>,  <span class="cit-name-given-names">B.M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ramachandran</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Localization of adaptive variants in human genomes using averaged one-dependence estimation</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D9%26rft.spage%253D703%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-03100-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-018-03100-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.41" data-doi="10.1534/genetics.117.300323"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tataru</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mollion</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glemin</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bataillon</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Inference of Distribution of Fitness Effects and Proportion of Adaptive Substitutions from Polymorphism Data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">207</span>, <span class="cit-fpage">1103</span>–<span class="cit-lpage">1119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300323%26rft_id%253Dinfo%253Apmid%252F28951530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNy8zLzExMDMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yMC8yMDIwLjAzLjE4Ljk5NzM0Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Uricchio</span>,  <span class="cit-name-given-names">L.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petrov</span>,  <span class="cit-name-given-names">D.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Enard</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Exploiting selection at linked sites to infer the rate and strength of adaptation</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Uricchio</span>,  <span class="cit-name-given-names">L.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petrov</span>,  <span class="cit-name-given-names">D.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Enard</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Exploiting selection at linked sites to infer the rate and strength of adaptation</span>. <abbr class="cit-jnl-abbrev">Nat Ecol Evol</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">977</span>–<span class="cit-lpage">984</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BEcol%2BEvol%26rft.volume%253D3%26rft.spage%253D977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.44" data-doi="10.1371/journal.pbio.0040072"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Voight</span>,  <span class="cit-name-given-names">B.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kudaravalli</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wen</span>,  <span class="cit-name-given-names">X.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Pritchard</span>,  <span class="cit-name-given-names">J.K.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">A map of recent positive selection in the human genome</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">e72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DVoight%26rft.auinit1%253DB.%2BF.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253De72%26rft.epage%253De72%26rft.atitle%253DA%2Bmap%2Bof%2Brecent%2Bpositive%2Bselection%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040072%26rft_id%253Dinfo%253Apmid%252F16494531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16494531&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.45" data-doi="10.1038/nature07390"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Worobey</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gemmel</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teuwen</span>,  <span class="cit-name-given-names">D.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haselkorn</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kunstman</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bunce</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muyembe</span>,  <span class="cit-name-given-names">J.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kabongo</span>,  <span class="cit-name-given-names">J.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalengayi</span>,  <span class="cit-name-given-names">R.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Marck</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Direct evidence of extensive diversity of HIV-1 in Kinshasa by 1960</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">455</span>, <span class="cit-fpage">661</span>–<span class="cit-lpage">664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWorobey%26rft.auinit1%253DM.%26rft.volume%253D455%26rft.issue%253D7213%26rft.spage%253D661%26rft.epage%253D664%26rft.atitle%253DDirect%2Bevidence%2Bof%2Bextensive%2Bdiversity%2Bof%2BHIV-1%2Bin%2BKinshasa%2Bby%2B1960.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07390%26rft_id%253Dinfo%253Apmid%252F18833279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07390&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18833279&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259639700046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.18.997346v1.46" data-doi="10.1093/nar/gkx1098"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zerbino</span>,  <span class="cit-name-given-names">D.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Achuthan</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Akanni</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Amode</span>,  <span class="cit-name-given-names">M.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barrell</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bhai</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Billis</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cummins</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gall</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giron</span>,  <span class="cit-name-given-names">C.G.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Ensembl 2018</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">46</span>, <span class="cit-fpage">D754</span>–<span class="cit-lpage">D761</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DD754%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1098%26rft_id%253Dinfo%253Apmid%252F29155950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29155950&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F20%2F2020.03.18.997346.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.18.997346v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.18.997346v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.18.996173v1" title="Intracellular Delivery of Nanoparticles via Microelectrophoresis Technique: Feasibility demonstration" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.19.999581v1" title="Hormonal regulation of the BRC1-dependent strigolactone transcriptome involved in shoot branching responses" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 20, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.18.997346v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.18.997346v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2020/03/20/2020.03.18.997346.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.18.997346v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.18.997346v1" class="active" data-icon-position="" data-hide-link-title="0">Ancient RNA virus epidemics through the lens of recent adaptation in human genomes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1203952" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-bXBnYIYH_zNznUtfhyb_8cg3jHbUxyq7Je20i0hpc64" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1203952" id="node1203952--2" data-pisa="biorxiv;2020.03.18.997346v1" data-pisa-master="biorxiv;2020.03.18.997346" data-seqnum="1203952" data-apath="/biorxiv/early/2020/03/20/2020.03.18.997346.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Ancient RNA virus epidemics through the lens of recent adaptation in human genomes</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">David</span> <span class="nlm-surname">Enard</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Dmitri A.</span> <span class="nlm-surname">Petrov</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.18.997346; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.18.997346 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.18.997346v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.997346v1&amp;title=Ancient%20RNA%20virus%20epidemics%20through%20the%20lens%20of%20recent%20adaptation%20in%20human%20genomes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.997346v1&amp;title=Ancient%20RNA%20virus%20epidemics%20through%20the%20lens%20of%20recent%20adaptation%20in%20human%20genomes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.997346v1&amp;text=Ancient%20RNA%20virus%20epidemics%20through%20the%20lens%20of%20recent%20adaptation%20in%20human%20genomes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.997346v1&amp;title=Ancient%20RNA%20virus%20epidemics%20through%20the%20lens%20of%20recent%20adaptation%20in%20human%20genomes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.997346v1&amp;t=Ancient%20RNA%20virus%20epidemics%20through%20the%20lens%20of%20recent%20adaptation%20in%20human%20genomes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.997346v1&amp;title=Ancient%20RNA%20virus%20epidemics%20through%20the%20lens%20of%20recent%20adaptation%20in%20human%20genomes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.997346v1&amp;title=Ancient%20RNA%20virus%20epidemics%20through%20the%20lens%20of%20recent%20adaptation%20in%20human%20genomes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1203952" id="node1203952--3" data-pisa="biorxiv;2020.03.18.997346v1" data-pisa-master="biorxiv;2020.03.18.997346" data-seqnum="1203952" data-apath="/biorxiv/early/2020/03/20/2020.03.18.997346.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Ancient RNA virus epidemics through the lens of recent adaptation in human genomes</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">David</span> <span class="nlm-surname">Enard</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Dmitri A.</span> <span class="nlm-surname">Petrov</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.18.997346; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.18.997346 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Ancient%20RNA%20virus%20epidemics%20through%20the%20lens%20of%20recent%20adaptation%20in%20human%20genomes&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Enard%2C+David&amp;rft.au=Petrov%2C+Dmitri+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1203952/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1203952/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1203952/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1203952/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1203952/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1203952/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1203952/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1203952/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1203952/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1203952/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1203952/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1203952/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.997346v1&amp;count=horizontal&amp;via=&amp;text=Ancient%20RNA%20virus%20epidemics%20through%20the%20lens%20of%20recent%20adaptation%20in%20human%20genomes&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.997346v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.18.997346v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.18.997346v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
