<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2019/01/31/444117.atom"/>
  <meta name="HW.pisa" content="biorxiv;444117v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Using pan RNA-seq analysis to reveal the ubiquitous existence of 5’ end and 3’ end small RNAs"/>
  <meta name="DC.Identifier" content="10.1101/444117"/>
  <meta name="DC.Date" content="2019-01-31"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="In this study, we used pan RNA-seq analysis to reveal the ubiquitous existence of 5’ end and 3’ end small RNAs. 5’ and 3’ sRNAs alone can be used to annotate mitochondrial with 1-bp resolution and nuclear non-coding genes and identify new steady-state RNAs, which are usually from functional genes. Using 5’, 3’ and intronic sRNAs, we revealed that the enzymatic dsRNA cleavage and RNAi could involve in the RNA degradation and gene expression regulation of U1 snRNA in human. The further study of 5’, 3’ and intronic sRNAs help rediscover double-stranded RNA (dsRNA) cleavage, RNA interference (RNAi) and the regulation of gene expression, which challenges the classical theories. In this study, we provided a simple and cost effective way for the annotation of mitochondrial and nuclear non-coding genes and the identification of new steady-state RNAs, particularly long non-coding RNAs (lncRNAs). We also provided a different point of view for cancer and virus, based on the new discoveries of dsRNA cleavage, RNAi and the regulation of gene expression."/>
  <meta name="DC.Contributor" content="Xiaofeng Xu"/>
  <meta name="DC.Contributor" content="Haishuo Ji"/>
  <meta name="DC.Contributor" content="Zhi Cheng"/>
  <meta name="DC.Contributor" content="Xiufeng Jin"/>
  <meta name="DC.Contributor" content="Xue Yao"/>
  <meta name="DC.Contributor" content="Yanqiang Liu"/>
  <meta name="DC.Contributor" content="Qiang Zhao"/>
  <meta name="DC.Contributor" content="Tao Zhang"/>
  <meta name="DC.Contributor" content="Jishou Ruan"/>
  <meta name="DC.Contributor" content="Wenjun Bu"/>
  <meta name="DC.Contributor" content="Ze Chen"/>
  <meta name="DC.Contributor" content="Shan Gao"/>
  <meta name="article:published_time" content="2019-01-31"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Using pan RNA-seq analysis to reveal the ubiquitous existence of 5’ end and 3’ end small RNAs"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;In this study, we used pan RNA-seq analysis to reveal the ubiquitous existence of 5’ end and 3’ end small RNAs. 5’ and 3’ sRNAs alone can be used to annotate mitochondrial with 1-bp resolution and nuclear non-coding genes and identify new steady-state RNAs, which are usually from functional genes. Using 5’, 3’ and intronic sRNAs, we revealed that the enzymatic dsRNA cleavage and RNAi could involve in the RNA degradation and gene expression regulation of U1 snRNA in human. The further study of 5’, 3’ and intronic sRNAs help rediscover double-stranded RNA (dsRNA) cleavage, RNA interference (RNAi) and the regulation of gene expression, which challenges the classical theories. In this study, we provided a simple and cost effective way for the annotation of mitochondrial and nuclear non-coding genes and the identification of new steady-state RNAs, particularly long non-coding RNAs (lncRNAs). We also provided a different point of view for cancer and virus, based on the new discoveries of dsRNA cleavage, RNAi and the regulation of gene expression.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2019/01/01"/>
  <meta name="citation_mjid" content="biorxiv;444117v2"/>
  <meta name="citation_id" content="444117v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/444117v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/444117v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/444117v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/444117.full.pdf"/>
  <meta name="citation_doi" content="10.1101/444117"/>
  <meta name="citation_num_pages" content="15"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="444117"/>
  <meta name="citation_author" content="Xiaofeng Xu"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Veterinary Etiological Biology and Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Science"/>
  <meta name="citation_author" content="Haishuo Ji"/>
  <meta name="citation_author_institution" content="College of Life Sciences, Nankai University"/>
  <meta name="citation_author_institution" content="Institute of Statistics, Nankai University"/>
  <meta name="citation_author" content="Zhi Cheng"/>
  <meta name="citation_author_institution" content="College of Life Sciences, Nankai University"/>
  <meta name="citation_author" content="Xiufeng Jin"/>
  <meta name="citation_author_institution" content="College of Life Sciences, Nankai University"/>
  <meta name="citation_author" content="Xue Yao"/>
  <meta name="citation_author_institution" content="Tianjin General Hospital"/>
  <meta name="citation_author" content="Yanqiang Liu"/>
  <meta name="citation_author_institution" content="College of Life Sciences, Nankai University"/>
  <meta name="citation_author" content="Qiang Zhao"/>
  <meta name="citation_author_institution" content="College of Life Sciences, Nankai University"/>
  <meta name="citation_author" content="Tao Zhang"/>
  <meta name="citation_author_institution" content="College of Life Sciences, Nankai University"/>
  <meta name="citation_author" content="Jishou Ruan"/>
  <meta name="citation_author_institution" content="School of Mathematical Sciences, Nankai University"/>
  <meta name="citation_author" content="Wenjun Bu"/>
  <meta name="citation_author_institution" content="College of Life Sciences, Nankai University"/>
  <meta name="citation_author" content="Ze Chen"/>
  <meta name="citation_author_institution" content="State Key Laboratory of Veterinary Etiological Biology and Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Science"/>
  <meta name="citation_author_email" content="gao_shan@mail.nankai.edu.cn"/>
  <meta name="citation_author_email" content="chenze@caas.cn"/>
  <meta name="citation_author" content="Shan Gao"/>
  <meta name="citation_author_institution" content="College of Life Sciences, Nankai University"/>
  <meta name="citation_author_institution" content="Institute of Statistics, Nankai University"/>
  <meta name="citation_author_email" content="gao_shan@mail.nankai.edu.cn"/>
  <meta name="citation_author_email" content="chenze@caas.cn"/>
  <meta name="citation_reference" content="S. Gao, J. Ou and K. Xiao, R language and Bioconductor in bioinformatics applications(Chinese Edition). 2014, Tianjin: Tianjin Science and Technology Translation Publishing Ltd."/>
  <meta name="citation_reference" content="Y. Ren, Z. Jiaqing, Y. Sun, Z. Wu, J. Ruan, et al., Full-length transcriptome sequencing on PacBio platform (in Chinese). Chinese Science Bulletin, 2016. 61(11): p. 1250–1254."/>
  <meta name="citation_reference" content="S. Gao, Y. Ren, Y. Sun, Z. Wu, J. Ruan, et al., PacBio Full-length transcriptome profiling of insect mitochondrial gene expression. RNA biology, 2016. 13(9): p. 820–825."/>
  <meta name="citation_reference" content="D.R. Garalde, E.A. Snell, D. Jachimowicz, A.J. Heron, M. Bruce, et al., Highly parallel direct RNA sequencing on an array of nanopores. bioRxiv, 2016. p. 068809."/>
  <meta name="citation_reference" content="M. Bouvy-Liivrand, A. Hernández de sande, P. Pölönen, J. Mehtonen, T. Vuorenmaa, et al., Analysis of primary microRNA loci from nascent transcriptomes reveals regulatory domains governed by chromatin architecture. Nucleic Acids Research, 2017. 45(17): p. 9837–9849."/>
  <meta name="citation_reference" content="Z. Chen, Y. Sun, X. Yang, Z. Wu, K. Guo, et al., Two featured series of rRNA-derived RNA fragments (rRFs) constitute a novel class of small RNAs. Plos One, 2017. 12(4): p. e0176458."/>
  <meta name="citation_reference" content="S. Gao, X. Tian, H. Chang, Y. Sun, Z. Wu, et al., Two novel lncRNAs discovered in human mitochondrial DNA using PacBio full-length transcriptome data. Mitochondrion, 2017."/>
  <meta name="citation_reference" content="R. Li, S. Gao, A.G. Hernandez, P.W. Wechter, Z. Fei, et al., Deep sequencing of small RNAs in tomato for virus and viroid identification and strain differentiation. PLoS ONE, 2012. 7(5): p. e37127."/>
  <meta name="citation_reference" content="Y.S. Lee, Y. Shibata, A. Malhotra and A. Dutta, A novel class of small RNAs: tRNA-derived RNA fragments (tRFs). Genes &amp;; development, 2009. 23(22): p. 2639–2649."/>
  <meta name="citation_reference" content="A.G. Matera, R.M. Terns and M.P. Terns, Non-coding RNAs: lessons from the small nuclear and small nucleolar RNAs. Nature Reviews Molecular Cell Biology, 2007. 8(3): p. 209–220."/>
  <meta name="citation_reference" content="Z. Cheng, Y. Sun, X. Niu, Y.C. Shang, Z. Wu, et al., Gene expression profiling reveals U1 snRNA regulates cancer gene expression. Oncotarget, 2017. 8(68): p. 112867–112874."/>
  <meta name="citation_reference" content="O. Rackham, A.M. Shearwood, T.R. Mercer, S.M. Davies, J.S. Mattick, et al., Long noncoding RNAs are generated from the mitochondrial genome and regulated by nuclear-encoded proteins. Rna-a Publication of the Rna Society, 2011. 17(12): p. 2085."/>
  <meta name="citation_reference" content="H. Seligmann, Undetected antisense tRNAs in mitochondrial genomes? Biology Direct, 2010. 5(1): p. 1–25."/>
  <meta name="citation_reference" content="C. Liu, Z. Chen, Y. Hu, H. Ji, D. Yu, et al., Complemented Palindromic Small RNAs First Discovered from SARS Coronavirus. Genes, 2018. 9(9): p. 442."/>
  <meta name="citation_reference" content="J. Montoya and G. Attardi, Identification of Initiation Sites for Heavy-Strand and Light-Strand Transcription in Human Mitochondrial DNA. Proceedings of the National Academy of Sciences of the United States of America, 1982. 79(23): p. 7195–9."/>
  <meta name="citation_reference" content="J. Houseley and D. Tollervey, The Many Pathways of RNA Degradation. Cell, 2009. 136(4): p. 763–776."/>
  <meta name="citation_reference" content="X. Niu, Y. Sun, Z. Chen, R. Li, C. Padmanabhan, et al., Using Small RNA-seq Data to Detect siRNA Duplexes Induced by Plant Viruses. Genes, 2017. 8(6): p. 163."/>
  <meta name="citation_reference" content="A.W. Nicholson, Ribonuclease III mechanisms of double-stranded RNA cleavage. Wiley Interdisciplinary Reviews Rna, 2013. 5(1): p. 31–48."/>
  <meta name="citation_reference" content="M. Buhler, N. Spies, D. Bartel and D. Moazed, TRAMP-mediated RNA surveillance prevents spurious entry of RNAs into the Schizosaccharomyces pombe siRNA pathway. Nature Structural &amp;; Molecular Biology, 2008. 15(10): p. 1015."/>
  <meta name="citation_reference" content="N. Buchon and C. Vaury, RNAi: a defensive RNA-silencing against viruses and transposable elements. Heredity, 2006. 96(2): p. 195–202."/>
  <meta name="citation_reference" content="Z. Cheng, Y. Shang, S. Gao and T. Zhang, Overexpression of U1 snRNA induces decrease of U1 spliceosome function associated with Alzheimer’s disease. Journal of Neurogenetics, 2017. p. 1."/>
  <meta name="citation_reference" content="L. Spraggon and L. Cartegni, U1 snRNP-dependent suppression of polyadenylation: physiological role and therapeutic opportunities in cancer. International journal of cell biology, 2013. 2013."/>
  <meta name="citation_reference" content="F. Wang, Y. Sun, J. Ruan, R. Chen, X. Chen, et al., Using small RNA deep sequencing data to detect human viruses. BioMed Research International, 2016. 2016(2016): p. 2596782."/>
  <meta name="citation_reference" content="M. Zhang, F. Zhan, H. Sun, X. Gong, Z. Fei, et al. Fastq_clean: An optimized pipeline to clean the Illumina sequencing data with quality control. in Bioinformatics and Biomedicine (BIBM), 2014 IEEE International Conference on. 2014. IEEE."/>
  <meta name="twitter:title" content="Using pan RNA-seq analysis to reveal the ubiquitous existence of 5’ end and 3’ end small RNAs"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="In this study, we used pan RNA-seq analysis to reveal the ubiquitous existence of 5’ end and 3’ end small RNAs. 5’ and 3’ sRNAs alone can be used to annotate mitochondrial with 1-bp resolution and nuclear non-coding genes and identify new steady-state RNAs, which are usually from functional genes. Using 5’, 3’ and intronic sRNAs, we revealed that the enzymatic dsRNA cleavage and RNAi could involve in the RNA degradation and gene expression regulation of U1 snRNA in human. The further study of 5’, 3’ and intronic sRNAs help rediscover double-stranded RNA (dsRNA) cleavage, RNA interference (RNAi) and the regulation of gene expression, which challenges the classical theories. In this study, we provided a simple and cost effective way for the annotation of mitochondrial and nuclear non-coding genes and the identification of new steady-state RNAs, particularly long non-coding RNAs (lncRNAs). We also provided a different point of view for cancer and virus, based on the new discoveries of dsRNA cleavage, RNAi and the regulation of gene expression."/>
  <meta name="og-title" property="og:title" content="Using pan RNA-seq analysis to reveal the ubiquitous existence of 5’ end and 3’ end small RNAs"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/444117v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="In this study, we used pan RNA-seq analysis to reveal the ubiquitous existence of 5’ end and 3’ end small RNAs. 5’ and 3’ sRNAs alone can be used to annotate mitochondrial with 1-bp resolution and nuclear non-coding genes and identify new steady-state RNAs, which are usually from functional genes. Using 5’, 3’ and intronic sRNAs, we revealed that the enzymatic dsRNA cleavage and RNAi could involve in the RNA degradation and gene expression regulation of U1 snRNA in human. The further study of 5’, 3’ and intronic sRNAs help rediscover double-stranded RNA (dsRNA) cleavage, RNA interference (RNAi) and the regulation of gene expression, which challenges the classical theories. In this study, we provided a simple and cost effective way for the annotation of mitochondrial and nuclear non-coding genes and the identification of new steady-state RNAs, particularly long non-coding RNAs (lncRNAs). We also provided a different point of view for cancer and virus, based on the new discoveries of dsRNA cleavage, RNAi and the regulation of gene expression."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2019-01-31"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Using pan RNA-seq analysis to reveal the ubiquitous existence of 5’ end and 3’ end small RNAs | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-157716 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">In this study, we used pan RNA-seq analysis to reveal the ubiquitous existence of 5’ end and 3’ end small RNAs. 5’ and 3’ sRNAs alone can be used to annotate mitochondrial with 1-bp resolution and nuclear non-coding genes and identify new steady-state RNAs, which are usually from functional genes. Using 5’, 3’ and intronic sRNAs, we revealed that the enzymatic dsRNA cleavage and RNAi could involve in the RNA degradation and gene expression regulation of U1 snRNA in human. The further study of 5’, 3’ and intronic sRNAs help rediscover double-stranded RNA (dsRNA) cleavage, RNA interference (RNAi) and the regulation of gene expression, which challenges the classical theories. In this study, we provided a simple and cost effective way for the annotation of mitochondrial and nuclear non-coding genes and the identification of new steady-state RNAs, particularly long non-coding RNAs (lncRNAs). We also provided a different point of view for cancer and virus, based on the new discoveries of dsRNA cleavage, RNAi and the regulation of gene expression.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-4">RNA sequencing (RNA-seq), usually based on the Next Generation Sequencing (NGS) technologies is widely used to simultaneously measure the expression levels of genes with higher accuracy than Serial Analysis of Gene Expression (SAGE) and microarray [
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. RNA-seq is also used to annotate genes in sequenced genomes, which is the only basis to study gene transcription, RNA processing and biological functions of these genes, 
     <em>etc</em>. Particularly, RNA-seq or small RNA sequencing (sRNA-seq) is indispensable for the annotation of non-coding genes, while the annotation of protein-coding genes can be conducted based on the analysis of protein codons. However, RNA-seq cannot be used to obtain the full-length transcripts by 
     <em>de novo</em> assembly or alignment. Both of PacBio full-length transcripts (PacBio cDNA-seq) [
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>] and Nanopore cDNA sequencing (Nanopore cDNA-seq) [
     <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>] can be used to obtain the full-length transcripts of mature RNAs or RNA precursors [
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. PacBio cDNA-seq produces reads with lower error rates than Nanopore cDNA-seq, while Nanopore cDNA-seq can produce longer reads than PacBio cDNA-seq. However, neither PacBio cDNA-seq nor Nanopore cDNA-seq can provide the exact 3’-end information of transcripts (
     <em>e.g.</em> polyA length) due to reverse transcription. The reason is that primers anneal to random positions located in the polyA regions or A-enriched regions in the body of transcripts to start reverse transcription. Nanopore direct RNA sequencing (Nanopore RNA-seq), as the only available sequencing technology which can sequence RNA directly [
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>], theoretically can be used to obtain the full-length 3’ ends of transcripts. But it can not be used to obtain the exact 3’-end information of transcripts either, due to the high error rate of Nanopore RNA-seq data. Combined with specific capture or enrichment technologies, several other RNA-seq methods have been developed to extent the use of standard RNA-seq. Parallel Analysis of RNA Ends and sequencing (PARE-seq), Cap Analysis of Gene Expression and sequencing (CAGE-seq) and Precision nuclear Run-On and sequencing (PRO-seq) have been developed to identify 5’ ends of mature RNAs. Polyadenylation sequencing (PA-seq) has been developed to identify 3’ ends of mature RNAs. Global Run-On and sequencing (GRO-seq) has been developed to sequence nascent RNAs [
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>], which helps determining the primary transcripts of genes.
    </p>
    <p id="p-5">The standard RNA-seq, sRNA-seq, PARE-seq, CAGE-seq, PRO-seq, PA-seq, GRO-seq, PacBio cDNA-seq, Nanopore cDNA-seq, Nanopore RNA-seq and 
     <em>et al.</em>, defined as pan RNA-seq, were used to improve gene annotation in our previous studies. Using pan RNA-seq analysis, we reported the corrected annotation of tick rRNA genes, human rRNA genes [
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>], insect mitochondrial genes [
     <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>] and human mitochondrial genes [
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. We also reported two novel long non-coding RNAs (lncRNAs) discovered in human mitochondrial DNA [
     <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>]. In addition, we unexpectedly discovered the existence of 5’ and 3’ end small RNAs (5’ and 3’ sRNAs) in animal rRNA genes [
     <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>] and later proved the ubiquitous existence of 5’ and 3’ sRNAs in mitochondrial and nuclear non-coding genes. In this study, we demonstrated that 5’ and 3’ sRNAs alone can be used to annotate mitochondrial and nuclear non-coding genes with 1-bp resolution and identify new steady-state RNAs. Using public sRNA-seq data from the same species, this method provides a simple and cost effective way for the annotation of mitochondrial and nuclear non-coding genes and the identification of new steady-state RNAs, which are usually from functional genes. The further study of 5’, 3’ and intronic sRNAs help rediscover double-stranded RNA (dsRNA) cleavage, RNA interference (RNAi) and the regulation of gene expression, which challenges the classical theories.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Results</h2>
    <div id="sec-3" class="subsection">
     <h3>Discovery of 5’ and 3’ sRNAs</h3>
     <p id="p-6">In a genome-alignment map of sRNA data, there usually are some peaks or hotspots [
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>], where the depths of the positions are much higher than those of other positions in the genome. In our previous study of human rRNA genes [
      <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a>], we found that some of peaks comprised 5’ and 3’ sRNAs and they were ubiquitously existed in mitochondrial and nuclear non-coding genes. As the current sRNA-seq technologies usually provide sequences with short lengths, 5’ and 3’ sRNAs are defined as sRNA-seq reads with lengths of 15~50 bp, which are precisely aligned to the 5’ and 3’ ends of mature RNAs respectively (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) and they have such features: 
      <strong>1)</strong> 5’ and 3’ sRNAs are degraded fragments from mature RNAs and the lengths of them vary progressively with 1-bp differences. 
      <strong>2)</strong> The cleavage sites between 3’ sRNAs and their downstream 5’ sRNAs are not limited to one (usually three) due to inexact cleavage by enzymes (
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). 
      <strong>3)</strong> 5’ and 3’ sRNAs of steady-state RNAs (
      <em>e.g.</em> 18S, 5.8S and 28S rRNA) are significantly more abundant than the intronic sRNAs of them, while 5’ and 3’ sRNAs of transicent RNAs (
      <em>e.g.</em> Internal Transcribed Spacers of rRNA, ITS1 and ITS2) are not. This criterion can be used to identify new steady-state RNAs, which are usually from functional genes. One example of a new steady-state RNA downstream tRNAs and another example of two novel mitochondrial lncRNAs were introduced in the following paragraphs. Particularly, it was proved that MDL1 and MDL1AS were two steady-state lncRNAs in the human mitochondrial DNA and predicted to have biological functions [
      <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>].
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/444117/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Definition of 5[[#x2019]] and 3[[#x2019]] sRNAs A. 5[[#x2019]] and 3[[#x2019]] sRNAs are defined as sRNA-seq reads with lengths of 15~50 bp, which are precisely aligned to the 5[[#x2019]] and 3[[#x2019]] ends of mature RNAs respectively. The lengths of them vary progressively with 1-bp differences. B. 5-end format is defined to easily identify 5[[#x2019]] ends of mature RNAs using sRNA-seq data. Human rRNA genes (RefSeq: NR_046235.1) were annotated using alignment results in the 5-end format. Among positions 7923, 7924 and 7925 with ratio1s (the 5th column) above 70 %, 7925 with the highest ratio1 was determined as the 5[[#x2019]] end of 28S rRNA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-888302493" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Definition of 5[[#x2019]] and 3[[#x2019]] sRNAs&lt;;/span&gt;; &lt;;strong&gt;;A.&lt;;/strong&gt;; 5[[#x2019]] and 3[[#x2019]] sRNAs are defined as sRNA-seq reads with lengths of 15~50 bp, which are precisely aligned to the 5[[#x2019]] and 3[[#x2019]] ends of mature RNAs respectively. The lengths of them vary progressively with 1-bp differences. &lt;;strong&gt;;B.&lt;;/strong&gt;; 5-end format is defined to easily identify 5[[#x2019]] ends of mature RNAs using sRNA-seq data. Human rRNA genes (RefSeq: NR_046235.1) were annotated using alignment results in the 5-end format. Among positions 7923, 7924 and 7925 with ratio1s (the 5&lt;;sup&gt;;th&lt;;/sup&gt;; column) above 70 %, 7925 with the highest ratio1 was determined as the 5[[#x2019]] end of 28S rRNA.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/444117/F1.medium.gif" width="440" height="274"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/444117/F1.medium.gif" width="440" height="274"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/444117/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/444117/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <span class="caption-title">Definition of 5’ and 3’ sRNAs</span>
       <p id="p-7" class="first-child">
        <strong>A.</strong> 5’ and 3’ sRNAs are defined as sRNA-seq reads with lengths of 15~50 bp, which are precisely aligned to the 5’ and 3’ ends of mature RNAs respectively. The lengths of them vary progressively with 1-bp differences. 
        <strong>B.</strong> 5-end format is defined to easily identify 5’ ends of mature RNAs using sRNA-seq data. Human rRNA genes (RefSeq: NR_046235.1) were annotated using alignment results in the 5-end format. Among positions 7923, 7924 and 7925 with ratio1s (the 5
        <sup>th</sup> column) above 70 %, 7925 with the highest ratio1 was determined as the 5’ end of 28S rRNA.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-8">We used 5’ and 3’ sRNAs from one sRNA-seq dataset to annotate genes and used one CAGE-seq dataset, one GRO-seq dataset and one PacBio cDNA-seq dataset (
      <strong>Materials and Methods</strong>) to validate the annotations. Later, we developed a simplified gene-annotation procedure. Using only 5’ sRNAs, gene annotation can be reduced to the identification of 5’ ends of mature RNAs. By doing so, the 3’ ends of their upstream mature RNAs and their cleavage sites can be derived (
      <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). We have defined a new file format, named 5-end format, to easily identify 5’ ends of mature RNAs. The new format is derived from the Pileup format (
      <strong>Materials and Methods</strong>) to include eight columns (
      <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>) for each line including information from a genomic position: 
      <strong>1)</strong> chromosome ID, 
      <strong>2)</strong> 1-based coordinate of this position, 
      <strong>3)</strong> reference base, 
      <strong>4)</strong> depth (the number of reads covering the position), 
      <strong>5)</strong> ratio1 (the number of positive-stranded reads starting at this position divided by the number of positive-stranded reads), 
      <strong>6)</strong> the number of positive-stranded reads starting at this position divided by the number of positive-stranded reads, 
      <strong>7)</strong> ratio2 (the number of negative-stranded reads starting at this position divided by the number of negative-stranded reads), 
      <strong>8)</strong> the number of negative-stranded reads starting at this position divided by the number of negative-stranded reads. Using the 5-end format, the 5’ end of one mature RNA can be easily identified from two to three candidates (
      <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>). Their ratio1s or ratio2s must be above a threshold (
      <em>e.g.</em> 75%) and significantly higher than the ratio1s or ratio2s of positions surrounding these two to three positions.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>5’ and 3’ sRNAs in non-coding genes</h3>
     <p id="p-9">Using 5’ and 3’ sRNAs, we corrected the annotation of human rRNA genes. For the 5’ end of each mature RNA, we obtained two or three candidates and selected the position with the highest ratio1 or ratio2 as the result. For example (
      <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>), we obtained three positions 7,923, 7,924 and 7,925 to identify the 5’ end of 28S rRNA and selected 7,925 as the result. In the same way, 5’ ends of 18S and 5.8S rRNA were also identified using 5’ sRNAs. Then, 3’ ends of 18S, 5.8S and 28S rRNA were identified using 3’ sRNAs. Finally, the annotations of ITS1 and ITS2 were derived using the annotations of 18S, 5.8S and 28S rRNA. These corrected annotations (
      <strong>Table 2</strong>) were validated using the CAGE-seq dataset and the GRO-seq dataset (
      <strong>Materials and Methods</strong>). Although the depth 1,471,247 at the position 6,601 was much higher than the depth 647,406 at the position 6,596 in the sRNA-seq dataset, the 5’ end of 5.8S rRNA annotated as the position 6,601 with the ratio1 of 35.42% (520,006/1,468,024) was still corrected as the position 6,596 with the ratio1 of 88.11% (569,882/646,805). In addition, the genome-alignment map using the sRNA-seq dataset showed that human rRNA genes had peaks at the position 6,596, 7,925 and 6,756 corresponding to the 5’ ends of 5.8S and 28S rRNA and the 3’ end of 5.8S rRNA, respectively (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The genome-alignment map using the CAGE-seq dataset showed that human rRNA genes had peaks at the position 3,675 and 7,926 corresponding to the 5’ ends of 18S and 28S rRNA, respectively (
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). This suggested that 5’ m
      <sup>7</sup>G or other caps of 18S and 28S rRNA could exist. By the analysis of 3’ sRNAs, we confirmed that rRNA genes did not contain 3’ polyAs.
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/444117/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Genome-alignment maps using sRNA-seq, Cage-seq and GRO-seq This figure shows the count distribution of all aligned reads on the reference rRNA sequence (RefSeq: NR_046235.1). These reads are from one sRNA-seq dataset (A.), one CAGE-seq dataset (B.) and one GRO-seq dataset (C.). The description of these datasets can be seen in the section Materials and Methods. The identified 5[[#x2019]] and 3[[#x2019]] ends of mature RNAs are marked by boxes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-888302493" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Genome-alignment maps using sRNA-seq, Cage-seq and GRO-seq&lt;;/span&gt;; This figure shows the count distribution of all aligned reads on the reference rRNA sequence (RefSeq: NR_046235.1). These reads are from one sRNA-seq dataset (&lt;;strong&gt;;A.&lt;;/strong&gt;;), one CAGE-seq dataset (&lt;;strong&gt;;B.&lt;;/strong&gt;;) and one GRO-seq dataset (&lt;;strong&gt;;C.&lt;;/strong&gt;;). The description of these datasets can be seen in the section Materials and Methods. The identified 5[[#x2019]] and 3[[#x2019]] ends of mature RNAs are marked by boxes.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/444117/F2.medium.gif" width="252" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/444117/F2.medium.gif" width="252" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/444117/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/444117/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <span class="caption-title">Genome-alignment maps using sRNA-seq, Cage-seq and GRO-seq</span>
       <p id="p-10" class="first-child">This figure shows the count distribution of all aligned reads on the reference rRNA sequence (RefSeq: NR_046235.1). These reads are from one sRNA-seq dataset (
        <strong>A.</strong>), one CAGE-seq dataset (
        <strong>B.</strong>) and one GRO-seq dataset (
        <strong>C.</strong>). The description of these datasets can be seen in the section Materials and Methods. The identified 5’ and 3’ ends of mature RNAs are marked by boxes.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-11">In 2009, a novel class of sRNAs named tRNA-derived RNA fragments (tRFs) was introduced and three series of tRFs (tRF-5, tRF-3 and tRF-1) were identified using the sRNA-seq data of the human prostate cancer cell line by 454 deep sequencing [
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. However, these authors did not correctly understand the tRFs due to the technology limit and the small dataset size. Using the pan RNA-seq analysis, we proved that the tRF-5 and tRF-3 series were 5’ and 3’ sRNAs from mature tRNAs and the tRF-1 series were 5’ sRNAs from mature RNAs of their downstream genes (
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). In this study, 13 mature tRNAs and their 43 precursors (
      <strong>Supplementary file 1</strong>) were analyzed, and the 5/3-sRNA ratios of them ranged from 0.1 to 595.8. However, we did not found the characteristics in the size distribution of 5’ and 3’ sRNAs from these tRNAs. As 3’ sRNAs contain detailed 3’-end information of mature RNAs, we acquired more discoveries related to tRNA processing, maturation and degradation. One example was 3’ sRNAs of tRNAs had four types, which were non-tail, C-, CC-, and CCA-tailed. The proportions of these four types were 5.26% (22,906/4,355,95), 12.36% (53,845/4,355,95), 13.81% (60,176/4,355,95) and 68.57% (298,668/4,355,95). In addition, we obtained sequences of full-length mature tRNAs with non-tail, C-, CC-, and CCA-tailed. Among these full-length mature tRNAs, 8,539 TRD-GTC2-1 tRNAs (for Asp) and 16,900 TRE-CTC1-1 tRNAs (for Glu) were obtained. These results suggested that 3’ sRNAs were produced by tRNA degradation during its synthesis, when CCAs were post-transcriptionally added to 3’ ends of tRNAs one nucleotide by one nucleotide. Another example was the correction of TRL-TAG3-1’s annotation. The mature TRL-TAG3-1 was annotated as a 82-nt sequence from the human genome with its 3’ cleavage site ACCGCTGCCA|cacctcagaa. Using 5’ and 3’ sRNAs, the 3’ cleavage site of TRL-TAG3-1 was determined as ACCGCTGCCAC|acctcagaa. Instead of CAA, it was CA that was post-transcriptionally added to the 3’ end of TRL-TAG3-1 ACCGCTGCCAC. The genome-alignment results using the CAGE-seq dataset showed that 5’ m
      <sup>7</sup>G or other caps of tRNA did not exist. By the analysis of 3’ sRNAs, we confirmed that tRNA genes did not contain 3’ polyAs. 5’ and 3’ sRNAs from all the 13 mature tRNAs resided in peaks in the genome-alignment maps, while a few 3’ sRNAs of their upstream genes or 5’ sRNAs of their downstream genes resided in peaks. Among the peaks from these upstream or downstream genes, the highest one was on the downstream of TRS-TGA1-1 (chr10:67764503-67764584). It suggested that this peak was the 5’ end of a new steady-state RNA, which could be from a functional gene and had not been annotated in the current genome version.
     </p>
     <p id="p-12">Small nuclear RNAs (snRNAs) include a class of small RNA molecules that are found within the splicing speckles and Cajal bodies of the cell nucleus in eukaryotic cells [
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. snRNAs are always associated with a set of specific proteins and the complexes are referred to as small nuclear ribonucleoproteins (snRNPs). SnRNAs are also commonly referred to as U-RNAs and one well-known member is U1 snRNA [
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. Using 5’ sRNAs, we confirmed annotations of U1, U2, U3, U4, U5, U6 and U7 (
      <strong>Supplementary file 1</strong>). The genome-alignment results using the CAGE-seq dataset showed that U1, U2, U3 and U4 snRNAs could be capped by 5’ m
      <sup>7</sup>G, but U5, U6 and U7 snRNAs could not. By the analysis of 3’ sRNAs, we confirmed that all the snRNAs did not contain 3’ polyAs. In addition, we did not found any new steady-state RNA on the upstream or downstream of seven snRNA genes.
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>5’ and 3’ sRNAs in mitochondrial genes</h3>
     <p id="p-13">Using pan RNA-seq analysis, we confirmed that nuclear mitochondrial DNA segments (NUMTs) in human genome did not transcribe into RNAs [
      <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">7</a>]. This finding simplified the analysis of mitochondrial genes (
      <em>e.g.</em> mutation detection or quantification) using RNA-seq data. In our previous study, we annotated two primary transcripts and 30 mature transcripts (tRNA
      <sup>Ile</sup>, tRNA
      <sup>Gln</sup>AS, tRNA
      <sup>Met</sup>, ND2, tRNA
      <sup>Trp</sup>, tRNA
      <sup>Ala</sup>AS/tRNA
      <sup>Asn</sup>AS/tRNA
      <sup>Cys</sup>AS/tRNA
      <sup>Tyr</sup>AS, COI, tRNA
      <sup>Ser</sup>AS, tRNA
      <sup>Asp</sup>, COII, tRNA
      <sup>Lys</sup>, ATP8/6, COIII, tRNA
      <sup>Gly</sup>, ND3, tRNA
      <sup>Arg</sup>, ND4L/4, tRNA
      <sup>His</sup>, tRNA
      <sup>Ser</sup>, tRNA
      <sup>Leu</sup>, ND5/ND6AS/tRNA
      <sup>Glu</sup>AS, Cytb, tRNA
      <sup>Thr</sup>, MDL1, tRNA
      <sup>Phe</sup>, 12S rRNA, tRNA
      <sup>Val</sup>, 16S rRNA, tRNA
      <sup>Leu</sup> and ND1) on the H-strand with 1-bp resolution [
      <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">7</a>]. In this study, COI and tRNA
      <sup>Ser</sup>AS were corrected as one mature transcript COI/tRNA
      <sup>Ser</sup>AS that could not be further cleaved. We classified mitochondrial genes into tRNA, mRNA, rRNA, antisense tRNA (
      <em>e.g.</em> tRNA
      <sup>Ser</sup>AS), antisense mRNA (
      <em>e.g.</em> ND6AS), antisense rRNA and lncRNAs (
      <em>e.g.</em> MDL1 and MDL1AS) [
      <a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">7</a>] and improved the "mitochondrial cleavage" model that had been proposed in our previous study [
      <a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">7</a>]. The improved model is that RNA cleavage can be processed: 1) at 5’ and 3’ ends of tRNAs, 2) between mRNAs and mRNAs (
      <em>e.g.</em> ATP8/6 and COIII) and 3) between antisense tRNAs and mRNAs (
      <em>e.g.</em> tRNA
      <sup>Tyr</sup>AS and COI), but cannot be processed: 1) between mRNAs and antisense tRNAs (
      <em>e.g.</em> COI and tRNA
      <sup>Ser</sup>AS) 2) between mRNAs and antisense mRNAs (
      <em>e.g.</em> ND5 and ND6AS), 3) between antisense mRNAs and antisense tRNAs (
      <em>e.g.</em> ND6AS and tRNA
      <sup>Glu</sup>AS) or 4) between antisense tRNAs and antisense tRNAs (
      <em>e.g.</em> tRNA
      <sup>Ala</sup>AS/tRNA
      <sup>Asn</sup>AS/tRNA
      <sup>Cys</sup>AS/tRNA
      <sup>Tyr</sup>AS). This model provided a framework to identify all the full-length coding and non-coding RNAs of animal mitochondrion with 1-bp resolution. For example, the identification of ND5/ND6AS/tRNA
      <sup>Glu</sup>AS, MDL1 and MDL1AS proved that all the reported lncRNAs from ND5, ND6, Cytb [
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>] or the D-loop region were meaningless fragments from RNA degradation or. Another example was that tRNA
      <sup>Ala</sup>AS-tRNA
      <sup>Tyr</sup>AS (NC_012920: 1318-1638) could not be further cleaved, which was against a hypothesis from one previous study [
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>].
     </p>
     <p id="p-14">Among 29 mature transcripts on the H-strand, tRNA transcripts were tailed by 3’ CCAs, while other mature transcripts were tailed by 3’ polyAs. The maximum lengths of polyAs in tRNA
      <sup>Gln</sup>AS, ND2, tRNA
      <sup>Ala</sup>AS-tRNA
      <sup>Tyr</sup>AS, COI/tRNA
      <sup>Ser</sup>AS, COII, ATP8/6, COIII, ND3, ND4L/4, ND5/ND6AS/tRNA
      <sup>Glu</sup>AS, Cytb, MDL1, 12S rRNA, 16S rRNA, and ND1 are 14, 1, 12, 11, 20, 35, 10, 1, 22, 1, 8, 28, 32, 21 and 16, respectively. 3’ sRNAs containing polyAs or CCAs of different lengths were captured to proved that 3’ sRNAs were produced by RNA degradation during its synthesis, when polyAs or CCAs were post-transcriptionally added to 3’ ends of RNAs one nucleotide by one nucleotide. In this study, we also confirmed that both of mRNA and rRNA transcripts were capped by 5’ m
      <sup>7</sup>G [
      <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a>]. Our data supported that MDL1AS, ND5/ND6AS/tRNA
      <sup>Glu</sup>AS and tRNA
      <sup>Ala</sup>AS/tRNA
      <sup>Asn</sup>AS/tRNA
      <sup>Cys</sup>AS/tRNA
      <sup>Tyr</sup>AS could be capped by 5’ m
      <sup>7</sup>G, but tRNA
      <sup>Gln</sup>AS and MDL1 could not be capped. 5’ and 3’ sRNAs of MDL1 and MDL1AS were significantly more abundant than the intronic sRNAs of them, which was one criterion to identify steady-state RNAs. Although MDL1 was not capped by 5’ m
      <sup>7</sup>G as MDL1AS, we still proposed that they were steady-state RNAs and could have biological functions. The further study showed that qPCR of MDL1 provided higher sensitivities than that of BAX/BCL2 and CASP3 in the detection of cell apoptosis [
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>].
     </p>
     <p id="p-15">In our previous study, the first Transcription Initiation Site (TIS) of H-strand (IT
      <sub>H1</sub>) and the TIS of L-strand (IT
      <sub>L</sub>) were determined at the position 561 and 407 on the human mitochondrial genome (RefSeq: NC_012920.1), but the second TIS of H-strand (IT
      <sub>H2</sub>) was not determined [
      <a id="xref-ref-7-8" class="xref-bibr" href="#ref-7">7</a>]. In this study, IT
      <sub>H2</sub> was determined at the position 648, which was also the 5’ end of 12S rRNA. This finding was against the long-standing knowledge that IT
      <sub>H2</sub> was at the position 638 [
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. Using pan RNA-seq analysis, we found that all the TISs (IT
      <sub>H1,</sub> IT
      <sub>H2</sub> and IT
      <sub>L</sub>) could be capped by 5’ m
      <sup>7</sup>G. We also found polyAs before TISs, which suggested that the transcription of mitochondrial genes could be initated by primers containing polyTs. This finding explained why all the TISs were resided in A-enriched regions. However, these explanations need be proved in the future studies.
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Analysis of RNA degradation using 5’, 3’ and intronic sRNAs</h3>
     <p id="p-16">As 5’, 3’ and intronic sRNAs are accumulated RNA degradation intermediates, they can be used to investigate the RNA degradation [
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>], particularly for steady-state RNAs. The result using our data showed that in general, 5’ and 3’ sRNAs were more abundant than intronic sRNAs and short 5’ and 3’ sRNAs were more abundant than longer ones for tRNAs, rRNAs, snRNAs and mitochondrial RNAs. This suggested that these mature RNAs, particularly short RNAs (
      <em>e.g.</em> tRNAs), were mainly degraded by 3’ and 5’ exonucleases to accumulate 5’ and 3’ sRNAs. As for rRNAs and snRNAs, we found many peaks in the body of genes, which were even much higher than the peaks comprising 5’ or 3’ sRNAs in the genome-alignment map. In addition, the peaks comprising intronic sRNAs in rRNAs showed tissue specificities. The liver tissue (SRA: SRP002272) had specific peaks at the positions 12,891. The Plasma (SRA: SRP034590) had specific peaks at the positions 5,431, 9,891 and 11,158. The B-Cell and exosome (SRA: SRP046046) had specific peaks at the positions 3,789 and 9,891. The Platelets (SRA: SRP048290) had specific peaks at the positions 4,384 and 10,627. Further study of these tissue specificities was beyond the scope of this study. Then, we focused on the study of the secondary structures around these peaks in rRNAs and snRNAs and found that some of them involved in dsRNA regions. Particularly, we found a featured peak spanning a 43-bp region from 49 bp to 92 bp of U1 snRNA (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). In this region, 5’ ends of most intronic sRNAs were precisely aligned to 49 bp or 78 bp (
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). We also found that this region formed a hairpin in the secondary structure of U1 snRNA and produced a series of siRNA duplexes [
      <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>] with lengths from 15 bp to at least 25 bp (
      <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). The most abundant reads AGGGCGAGGCTTATC and TGTGCTGACCCCTGC formed a 15-bp duplex structure. The duplex ratio of AGGGCGAGGCTTATC against TGTGCTGACCCCTGC was 2.15 (34,078/15,825) and 99.97% (49,889/49,903) of these duplexes were sequenced from 14 samples of plasma (SRA: SRP034590). It suggested that this dsRNA region was cleaved by the ribonuclease III (RNase III) family [
      <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>] to produce these siRNA duplexes and could induced RNAi. Based on the findings in this study, our hypothesis is: 5’ and 3’ exonucleases are prevalent than endonuceases for the degradation of mature non-coding RNAs. So abundant 5’ and 3’ sRNAs were observed using sRNA-seq data. The longer mature RNAs have more and longer dsRNA regions (
      <em>e.g.</em> 15-bp long for U1) than short ones (
      <em>e.g.</em> 7-bp longest for tRNAs) to induce dsRNA cleavage to produce siRNA duplexes. Although the lengths of siRNA duplexes discovered in this study were only 15 bp, we still hypothesized that they could induce RNAi due to the unbalanced duplex ratio 2.15. As RNAi regulate the expression of these genes through a negative feed-back mechanism, we designed preliminary experiments to over-express U1 snRNA in the HEK293 (human), SY5Y (human) and PC-12 (rat) cell lines to prove our hypothesis. The basic idea was that if the negative feed-back mechanism existed, the expression level of U1 snRNA could decrease rather than be stable when the over-expression of it beyond a threshold. The experimental results showed that the expression level of U1 snRNA decreased after 4X, 9X and 6X dosage (
      <strong>Materials and Methods</strong>) in the HEK293 (human), SY5Y (human) and PC-12 (rat) cell lines, respectively (
      <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3D</a>). Particularly, the results in the HEK293 cell line showed a significant effect caused by the negative feed-back mechanism. Therefore, RNAi could involve in the RNA degradation and gene expression regulation of U1 snRNA.
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/444117/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="siRNA duplexes discovered from U1 snRNAs A. The count distribution of all aligned reads on the reference U1 snRNA (RefSeq: NR_004430.2). B. The above is a mountain plot representation of the MFE structure, the thermodynamic ensemble of RNA structures and the centroid structure. The positional entropy for each position is showed below. C. The secondary structure of U1 snRNA. D. U1 over-expression in the HEK293 (human), SY5Y (human) and PC-12 (rat) cell lines were conducted by virus transfection. The qPCR results showed the relative expression levels of U1 in 12 groups (Materials and Methods). The control group used unprocessed samples." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-888302493" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;siRNA duplexes discovered from U1 snRNAs&lt;;/span&gt;; &lt;;strong&gt;;A.&lt;;/strong&gt;; The count distribution of all aligned reads on the reference U1 snRNA (RefSeq: NR_004430.2). &lt;;strong&gt;;B.&lt;;/strong&gt;; The above is a mountain plot representation of the MFE structure, the thermodynamic ensemble of RNA structures and the centroid structure. The positional entropy for each position is showed below. &lt;;strong&gt;;C.&lt;;/strong&gt;; The secondary structure of U1 snRNA. &lt;;strong&gt;;D.&lt;;/strong&gt;; U1 over-expression in the HEK293 (human), SY5Y (human) and PC-12 (rat) cell lines were conducted by virus transfection. The qPCR results showed the relative expression levels of U1 in 12 groups (&lt;;strong&gt;;Materials and Methods&lt;;/strong&gt;;). The control group used unprocessed samples.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/444117/F3.medium.gif" width="423" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/444117/F3.medium.gif" width="423" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/444117/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/444117/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <span class="caption-title">siRNA duplexes discovered from U1 snRNAs</span>
       <p id="p-17" class="first-child">
        <strong>A.</strong> The count distribution of all aligned reads on the reference U1 snRNA (RefSeq: NR_004430.2). 
        <strong>B.</strong> The above is a mountain plot representation of the MFE structure, the thermodynamic ensemble of RNA structures and the centroid structure. The positional entropy for each position is showed below. 
        <strong>C.</strong> The secondary structure of U1 snRNA. 
        <strong>D.</strong> U1 over-expression in the HEK293 (human), SY5Y (human) and PC-12 (rat) cell lines were conducted by virus transfection. The qPCR results showed the relative expression levels of U1 in 12 groups (
        <strong>Materials and Methods</strong>). The control group used unprocessed samples.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-7">
    <h2 class="">Conclusion and Discussion</h2>
    <p id="p-18">In this study, we used the pan RNA-seq analysis to reveal the ubiquitous existence of 5’ end and 3’ end small RNAs. 5’ and 3’ sRNAs alone can be used to annotate mitochondrial and nuclear non-coding genes with 1-bp resolution and identify new steady-state RNAs. Using 5’, 3’ and intronic sRNAs, we revealed that the enzymatic dsRNA cleavage and RNAi could involve in the RNA degradation and gene expression regulation of U1 snRNA in human. The RNAi’s function in the RNA degradation was reported as a inappropriate event in yeast rRNA and tRNA degradation and only happened when 5’ and 3’ degradation were absent [
     <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]. However, our finding suggested that RNAi’s function in the RNA degradation could be a general mechanism.
    </p>
    <p id="p-19">Based on a previous study, the Rnt1p polypeptide cleave hairpin structures in pre-rRNAs, pre-mRNAs, and transcripts containing noncoding RNAs (
     <em>e.g.</em> snoRNAs) for their maturation in yeast. Rnt1p recognizes the tetraloop [A/u]GNN and cleaves the stem ∼14–16 bp from the structure [
     <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>]. The most abundant read AGGGCGAGGCTTATC discovered in this study contained AGGG and AGGC tetraloops and had a length of 15 bp. It suggested that Rnt1p could produce those siRNA duplexes from U1 snRNAs and cause RNAi, which was against basic knowledge that Dicer is required for RNAi in mammal but produce siRNA duplexes with lengths of ∼20–25 bp. Although our preliminary experiments proved the existence of RNAi, which enzyme caused RNAi in U1 snRNAs is still unclear.
    </p>
    <p id="p-20">The ancestral function of RNAi is generally agreed to have been immune defense against exogenous genetic elements such as transposons and viral genomes [
     <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]. However, our findings help rediscover dsRNA cleavage, RNAi and the regulation of gene expression. That is, both of dsRNA cleavage and RNAi are innate mechanisms rather than immune defense mechanisms. Basically, the enzymatic dsRNA cleavage is responsible for RNA processing, maturation and degradation, while RNAi is responsible for RNA degradation. By the degradation of mature RNAs, RNAi of one gene produces siRNA duplexes to regulate expression levels of itself or other genes. Mature RNAs containing more hairpin structures have more chances to induce RNAi, which is more important for highly expressed genes (
     <em>e.g.</em> U1 snRNA) or virus genes. These genes need RNAi to regulate gene expression though a negative feed-back mechanism. In one of our previous studies, we reported for the first time the existence of complemented palindromic small RNAs (cpsRNAs) and proposed that one cpsRNA from severe acute respiratory syndrome coronavirus (SARS-CoV) could induced RNAi [
     <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>]. This cpsRNA was detected from a 22-bp DNA complemented palindrome in the SARS-CoV genome. As DNA complemented palindromes are prone to produce dsRNA regions, viruses containing more DNA complemented palindromes in their genomes have more chances to induce RNAi and have more abilities for the regulation of gene expression, which is important for their infection or pathogenesis.
    </p>
    <p id="p-21">We also provided a different point of view for the gene expression regulation of U1 snRNA, The primary function of U1 snRNA is its involvement in the splicing of pre-mRNAs in nuclei. In the past 20 years, research of U1 snRNA has focused on its primary function, particularly related to neurodegenerative diseases caused by abnormalities of U1 snRNA [
     <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>]. In one of our previous studies, we reported that over-expression of U1 snRNA induces decrease of U1 spliceosome function associated with Alzheimer’s disease. However, the relationship between U1 snRNA over-expression and U1 snRNP loss of function remain unknown [
     <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. In another previous study, we reported that U1 snRNA over-expression induced cell apoptosis in SY5Y cells, but not in PC-12 cells [
     <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>]. These contradictory phenomena can be explained using the RNAi’s function in the RNA degradation and the negative feedback mechanism.
    </p>
    <p id="p-22">We provided a different point of view for cancer and virus. In one of our previous study, we reported how U1 snRNA over-expression affected the expression of mammal genes on a genome-wide scale and that U1 snRNA could regulate cancer gene expression. This had been explained by that Alternative Splicing (AS) and Alternative Polyadenylation (APA) were deregulated and exploited by cancer cells to promote their growth and survival [
     <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>]. Based on our point of view, the over-expressed U1 snRNA in cancer cells recruit excess RNase III. So the left RNase III is not enough to function in RNA degradation of other genes or genome surveillance and 
     <em>etc</em> [
     <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a>]. Viruses also recruit excess RNase III and the left RNase III is not enough to function in hose defense [
     <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">18</a>].
    </p>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">Materials and Methods</h2>
    <div id="sec-9" class="subsection">
     <h3>Datasets and data analysis</h3>
     <p id="p-23">All sRNA-seq data were downloaded from the NCBI SRA database. Data in four projects (SRP002272, SRP034590, SRP046046 and SRP048290) were selected from the human931 sRNA-seq dataset which had been collected in our previous study [
      <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>]. SRP002272, SRP034590, SRP046046 and SRP048290 were sequenced using Illumina Single End (SE) sequencing technologies with length 35~46, 202, 101 and 101 bp, respectively and they contained 15, 14, 12 and 6 runs of sRNA-seq data, respectively. One CAGE-seq dataset, one GRO-seq dataset [
      <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>] and one PacBio cDNA-seq dataset [
      <a id="xref-ref-7-9" class="xref-bibr" href="#ref-7">7</a>] were used to validate the annotations (
      <strong>Supplementary file 1)</strong>. The cleaning and quality control of sRNA-seq data were performed using the pipeline Fastq_clean [
      <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>] that was optimized to clean the raw reads from Illumina platforms. To simply annotate genes from a sequenced genome, we aligned all the cleaned reads from sRNA-seq, CAGE-seq and GRO-seq data to the reference sequences using the software bowtie v0.12.7 allowing one mismatch. Then, we obtained SAM, BAM, sorted BAM, Pileup and 5-end files using the software samtools. Statistical computation and plotting were performed using the software R v2.15.3 with the Bioconductor packages [
      <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a>].
     </p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>Validation by preliminary experiments</h3>
     <p id="p-24">U1 over-expression in the HEK293 (human), SY5Y (human) and PC-12 (rat) cell lines were conducted by virus transfection using the pLVX-shRNA1 plasmids and the Lenti-X HTX Packaging System (Clontech, USA), which had been described in our previous study [
      <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>]. U1 snRNAs of human and rat used synthetic DNA containing the sequence (RefSeq: NR_004430.2) and the sequence (GenBank: 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=V01266.1&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F444117.atom">V01266.1</a>), respectively. For each experiment, 12 groups of samples named control, 1X, 2X, 3X, 4X, 5X, 6X, 7X, 8X, 9X, 10X and 11X were transfected by 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 and 11 μL U1-packged lentiviruses (
      <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3D</a>). Each group contained three samples for biological replicates and the control samples used unprocessed cells. Each sample contained 10
      <sup>5</sup> cells and virus titer was 10
      <sup>7</sup> TU/mL for 1X. After transfection, RNA extraction, cDNA synthesis and cDNA amplification were performed following the same procedure in our previous study [
      <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">11</a>]. For each sample, total RNA was isolated using RNAiso Plus Reagent (TaKaRa, Japan) and the cDNA was synthesized by Mir-X miRNA First-Strand Synthesis Kit (Clontech, USA). The cDNA product was amplified by qPCR (Thermo Fisher Scientific, USA) using U6 snRNA as internal control under gene-specific reaction conditions. U1 snRNAs of human and rat used the forward and reverse primers GGGAGATACCATGATCAC and CCACTACCACAAATTATGC. U6 snRNAs of human and rat used CGGCAGCACATATACTAA and GAACGCTTCACGAATTTG.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1058358/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1058358/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1058358" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <p id="p-25" class="first-child">Annotation of human rRNA genes with corrections</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">Funding</h2>
    <p id="p-27">Central Public-Interest Scientific Institution Basal Research Fund of Lanzhou Veterinary Research Institute of CAAS to Ze Chen and National Key Research and Development Program of China (2016YFC0502304-03) to Defu Chen.</p>
   </div>
   <div class="section COI-statement" id="sec-12">
    <h2 class="">Competing interests</h2>
    <p id="p-28">Non-financial competing interests are claimed in this study.</p>
   </div>
   <div class="section" id="sec-13">
    <h2 class="">Authors’ contributions</h2>
    <p id="p-29">SG conceived this project. SG and QW supervised this project. SG, ZC and ZW analyzed the data. YS and HC curated the sequences and prepared all the figures, tables and additional files. CL, WS and YH performed qPCR experiments. SG drafted the main manuscript. DY and WB revised the manuscript. 
     <strong>All authors have read and approved the manuscript.</strong>
    </p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgments</h2>
    <p id="p-26">We thank Yangguang Han and Jinke Huang from TianJin KeYiJiaXin Technology Co., Ltd for his professional guidance on the RNAi and cellular experiments.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">REFERENCES</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-book" id="cit-444117v2.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Gao</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Ou</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Xiao</span>
         </span>, 
         <span class="cit-chapter-title">R language and Bioconductor in bioinformatics applications(Chinese Edition)</span>. 
         <span class="cit-pub-date">2014</span>, 
         <span class="cit-publ-loc">Tianjin</span>: 
         <span class="cit-publ-name">Tianjin Science and Technology Translation Publishing Ltd</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Ren</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Jiaqing</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Sun</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Wu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Ruan</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Full-length transcriptome sequencing on PacBio platform (in Chinese)</span>. 
         <abbr class="cit-jnl-abbrev">Chinese Science Bulletin</abbr>, 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-vol">61</span>(
         <span class="cit-issue">11</span>): p. 
         <span class="cit-fpage">1250</span>–
         <span class="cit-lpage">1254</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DChinese%2BScience%2BBulletin%26rft.volume%253D61%26rft.spage%253D1250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Gao</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Ren</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Sun</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Wu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Ruan</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">PacBio Full-length transcriptome profiling of insect mitochondrial gene expression</span>. 
         <abbr class="cit-jnl-abbrev">RNA biology</abbr>, 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-vol">13</span>(
         <span class="cit-issue">9</span>): p. 
         <span class="cit-fpage">820</span>–
         <span class="cit-lpage">825</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DRNA%2Bbiology%26rft.volume%253D13%26rft.spage%253D820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">D.R.</span>
          <span class="cit-name-surname">Garalde</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.A.</span>
          <span class="cit-name-surname">Snell</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Jachimowicz</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.J.</span>
          <span class="cit-name-surname">Heron</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Bruce</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Highly parallel direct RNA sequencing on an array of nanopores</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, 
         <span class="cit-pub-date">2016</span>. p. 
         <span class="cit-fpage">068809</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Bouvy-Liivrand</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Hernández de sande</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Pölönen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Mehtonen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Vuorenmaa</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Analysis of primary microRNA loci from nascent transcriptomes reveals regulatory domains governed by chromatin architecture</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">45</span>(
         <span class="cit-issue">17</span>): p. 
         <span class="cit-fpage">9837</span>–
         <span class="cit-lpage">9849</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D45%26rft.spage%253D9837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Sun</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Yang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Wu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K.</span>
          <span class="cit-name-surname">Guo</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Two featured series of rRNA-derived RNA fragments (rRFs) constitute a novel class of small RNAs</span>. 
         <abbr class="cit-jnl-abbrev">Plos One</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">12</span>(
         <span class="cit-issue">4</span>): p. 
         <span class="cit-fpage">e0176458</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPlos%2BOne%26rft.volume%253D12%26rft.spage%253De0176458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Gao</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Tian</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Chang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Sun</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Wu</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Two novel lncRNAs discovered in human mitochondrial DNA using PacBio full-length transcriptome data</span>. 
         <abbr class="cit-jnl-abbrev">Mitochondrion</abbr>, 
         <span class="cit-pub-date">2017</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.8" data-doi="10.1371/journal.pone.0037127">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Li</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Gao</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.G.</span>
          <span class="cit-name-surname">Hernandez</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.W.</span>
          <span class="cit-name-surname">Wechter</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Fei</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Deep sequencing of small RNAs in tomato for virus and viroid identification and strain differentiation</span>. 
         <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>, 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-vol">7</span>(
         <span class="cit-issue">5</span>): p. 
         <span class="cit-fpage">e37127</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLi%26rft.auinit1%253DR.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37127%26rft.epage%253De37127%26rft.atitle%253DDeep%2Bsequencing%2Bof%2Bsmall%2BRNAs%2Bin%2Btomato%2Bfor%2Bvirus%2Band%2Bviroid%2Bidentification%2Band%2Bstrain%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037127%26rft_id%253Dinfo%253Apmid%252F22623984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037127&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22623984&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F444117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.9" data-doi="10.1101/gad.1837609">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.S.</span>
          <span class="cit-name-surname">Lee</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Shibata</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Malhotra</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Dutta</span>
         </span>, 
         <span class="cit-article-title">A novel class of small RNAs: tRNA-derived RNA fragments (tRFs)</span>. 
         <abbr class="cit-jnl-abbrev">Genes &amp;; development</abbr>, 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-vol">23</span>(
         <span class="cit-issue">22</span>): p. 
         <span class="cit-fpage">2639</span>–
         <span class="cit-lpage">2649</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DLee%26rft.auinit1%253DY.%2BS.%26rft.volume%253D23%26rft.issue%253D22%26rft.spage%253D2639%26rft.epage%253D2649%26rft.atitle%253DA%2Bnovel%2Bclass%2Bof%2Bsmall%2BRNAs%253A%2BtRNA-derived%2BRNA%2Bfragments%2B%2528tRFs%2529%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1837609%26rft_id%253Dinfo%253Apmid%252F19933153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIzLzIyLzI2MzkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wMS8zMS80NDQxMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.10" data-doi="10.1038/nrm2124">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A.G.</span>
          <span class="cit-name-surname">Matera</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.M.</span>
          <span class="cit-name-surname">Terns</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.P.</span>
          <span class="cit-name-surname">Terns</span>
         </span>, 
         <span class="cit-article-title">Non-coding RNAs: lessons from the small nuclear and small nucleolar RNAs</span>. 
         <abbr class="cit-jnl-abbrev">Nature Reviews Molecular Cell Biology</abbr>, 
         <span class="cit-pub-date">2007</span>. 
         <span class="cit-vol">8</span>(
         <span class="cit-issue">3</span>): p. 
         <span class="cit-fpage">209</span>–
         <span class="cit-lpage">220</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DMatera%26rft.auinit1%253DA.%2BG.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D209%26rft.epage%253D220%26rft.atitle%253DNon-coding%2BRNAs%253A%2Blessons%2Bfrom%2Bthe%2Bsmall%2Bnuclear%2Band%2Bsmall%2Bnucleolar%2BRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm2124%26rft_id%253Dinfo%253Apmid%252F17318225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nrm2124&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17318225&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F444117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000244479900016&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.11" data-doi="10.18632/oncotarget.22842">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Cheng</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Sun</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Niu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.C.</span>
          <span class="cit-name-surname">Shang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Wu</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Gene expression profiling reveals U1 snRNA regulates cancer gene expression</span>. 
         <abbr class="cit-jnl-abbrev">Oncotarget</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">8</span>(
         <span class="cit-issue">68</span>): p. 
         <span class="cit-fpage">112867</span>–
         <span class="cit-lpage">112874</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DOncotarget%26rft.volume%253D8%26rft.spage%253D112867%26rft_id%253Dinfo%253Adoi%252F10.18632%252Foncotarget.22842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.18632/oncotarget.22842&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.12" data-doi="10.1261/rna.029405.111">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">O.</span>
          <span class="cit-name-surname">Rackham</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.M.</span>
          <span class="cit-name-surname">Shearwood</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.R.</span>
          <span class="cit-name-surname">Mercer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.M.</span>
          <span class="cit-name-surname">Davies</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.S.</span>
          <span class="cit-name-surname">Mattick</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Long noncoding RNAs are generated from the mitochondrial genome and regulated by nuclear-encoded proteins</span>. 
         <abbr class="cit-jnl-abbrev">Rna-a Publication of the Rna Society</abbr>, 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-vol">17</span>(
         <span class="cit-issue">12</span>): p. 
         <span class="cit-fpage">2085</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DRna-a%2BPublication%2Bof%2Bthe%2BRna%2BSociety%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.029405.111%26rft_id%253Dinfo%253Apmid%252F22028365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMi8yMDg1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDEvMzEvNDQ0MTE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Seligmann</span>
         </span>, 
         <span class="cit-article-title">Undetected antisense tRNAs in mitochondrial genomes?</span>
         <abbr class="cit-jnl-abbrev">Biology Direct</abbr>, 
         <span class="cit-pub-date">2010</span>. 
         <span class="cit-vol">5</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">25</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiology%2BDirect%26rft.volume%253D5%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Liu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Hu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Ji</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Yu</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Complemented Palindromic Small RNAs First Discovered from SARS Coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">Genes</abbr>, 
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-vol">9</span>(
         <span class="cit-issue">9</span>): p. 
         <span class="cit-fpage">442</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenes%26rft.volume%253D9%26rft.spage%253D442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.15" data-doi="10.1073/pnas.79.23.7195">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Montoya</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.</span>
          <span class="cit-name-surname">Attardi</span>
         </span>, 
         <span class="cit-article-title">Identification of Initiation Sites for Heavy-Strand and Light-Strand Transcription in Human Mitochondrial DNA</span>. 
         <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr>, 
         <span class="cit-pub-date">1982</span>. 
         <span class="cit-vol">79</span>(
         <span class="cit-issue">23</span>): p. 
         <span class="cit-fpage">7195</span>–
         <span class="cit-lpage">9</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMontoya%26rft.auinit1%253DJ.%26rft.volume%253D79%26rft.issue%253D23%26rft.spage%253D7195%26rft.epage%253D7199%26rft.atitle%253DIdentification%2Bof%2Binitiation%2Bsites%2Bfor%2Bheavy-strand%2Band%2Blight-strand%2Btranscription%2Bin%2Bhuman%2Bmitochondrial%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.79.23.7195%26rft_id%253Dinfo%253Apmid%252F6185947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNzkvMjMvNzE5NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAxLzMxLzQ0NDExNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.16" data-doi="10.1016/j.cell.2009.01.019">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Houseley</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Tollervey</span>
         </span>, 
         <span class="cit-article-title">The Many Pathways of RNA Degradation</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>, 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-vol">136</span>(
         <span class="cit-issue">4</span>): p. 
         <span class="cit-fpage">763</span>–
         <span class="cit-lpage">776</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHouseley%26rft.auinit1%253DJ.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D763%26rft.epage%253D776%26rft.atitle%253DThe%2Bmany%2Bpathways%2Bof%2BRNA%2Bdegradation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.01.019%26rft_id%253Dinfo%253Apmid%252F19239894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.01.019&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19239894&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F444117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000263688200024&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Niu</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Sun</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Li</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Padmanabhan</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Using Small RNA-seq Data to Detect siRNA Duplexes Induced by Plant Viruses</span>. 
         <abbr class="cit-jnl-abbrev">Genes</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">8</span>(
         <span class="cit-issue">6</span>): p. 
         <span class="cit-fpage">163</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenes%26rft.volume%253D8%26rft.spage%253D163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">A.W.</span>
          <span class="cit-name-surname">Nicholson</span>
         </span>, 
         <span class="cit-article-title">Ribonuclease III mechanisms of double-stranded RNA cleavage</span>. 
         <abbr class="cit-jnl-abbrev">Wiley Interdisciplinary Reviews Rna</abbr>, 
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-vol">5</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">31</span>–
         <span class="cit-lpage">48</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DWiley%2BInterdisciplinary%2BReviews%2BRna%26rft.volume%253D5%26rft.spage%253D31%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Buhler</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Spies</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Bartel</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Moazed</span>
         </span>, 
         <span class="cit-article-title">TRAMP-mediated RNA surveillance prevents spurious entry of RNAs into the Schizosaccharomyces pombe siRNA pathway</span>. 
         <abbr class="cit-jnl-abbrev">Nature Structural &amp;; Molecular Biology</abbr>, 
         <span class="cit-pub-date">2008</span>. 
         <span class="cit-vol">15</span>(
         <span class="cit-issue">10</span>): p. 
         <span class="cit-fpage">1015</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BStructural%2B%2526%2BMolecular%2BBiology%26rft.volume%253D15%26rft.spage%253D1015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.20" data-doi="10.1038/sj.hdy.6800789">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Buchon</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Vaury</span>
         </span>, 
         <span class="cit-article-title">RNAi: a defensive RNA-silencing against viruses and transposable elements</span>. 
         <abbr class="cit-jnl-abbrev">Heredity</abbr>, 
         <span class="cit-pub-date">2006</span>. 
         <span class="cit-vol">96</span>(
         <span class="cit-issue">2</span>): p. 
         <span class="cit-fpage">195</span>–
         <span class="cit-lpage">202</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DBuchon%26rft.auinit1%253DN.%26rft.volume%253D96%26rft.issue%253D2%26rft.spage%253D195%26rft.epage%253D202%26rft.atitle%253DRNAi%253A%2Ba%2Bdefensive%2BRNA-silencing%2Bagainst%2Bviruses%2Band%2Btransposable%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800789%26rft_id%253Dinfo%253Apmid%252F16369574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800789&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16369574&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F444117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000234843500013&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Cheng</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Shang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Gao</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Zhang</span>
         </span>, 
         <span class="cit-article-title">Overexpression of U1 snRNA induces decrease of U1 spliceosome function associated with Alzheimer’s disease</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Neurogenetics</abbr>, 
         <span class="cit-pub-date">2017</span>. p. 
         <span class="cit-fpage">1</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.22">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Spraggon</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Cartegni</span>
         </span>, 
         <span class="cit-article-title">U1 snRNP-dependent suppression of polyadenylation: physiological role and therapeutic opportunities in cancer</span>. 
         <abbr class="cit-jnl-abbrev">International journal of cell biology</abbr>, 
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-vol">2013</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-444117v2.23">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">F.</span>
          <span class="cit-name-surname">Wang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.</span>
          <span class="cit-name-surname">Sun</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Ruan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Chen</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Using small RNA deep sequencing data to detect human viruses</span>. 
         <abbr class="cit-jnl-abbrev">BioMed Research International</abbr>, 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-vol">2016</span>(
         <span class="cit-issue">2016</span>): p. 
         <span class="cit-fpage">2596782</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioMed%2BResearch%2BInternational%26rft.volume%253D2016%26rft.spage%253D2596782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-book" id="cit-444117v2.24">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Zhang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">F.</span>
          <span class="cit-name-surname">Zhan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Sun</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">X.</span>
          <span class="cit-name-surname">Gong</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">Z.</span>
          <span class="cit-name-surname">Fei</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-chapter-title">Fastq_clean: An optimized pipeline to clean the Illumina sequencing data with quality control</span>. in 
         <span class="cit-source">Bioinformatics and Biomedicine (BIBM), 2014 IEEE International Conference</span> on. 
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-publ-name">IEEE</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
