<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Emerg Microbes Infect</span><span class="iso-abbrev" title="iso-abbrev">Emerg Microbes Infect</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Emerging Microbes &amp;amp; Infections</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2222-1751</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/30353004">30353004</a></span><span class="publisher-id" title="publisher-id">172</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41426-018-0172-4">10.1038/s41426-018-0172-4</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Genome-wide characterization of the seasonal H3N2 virus in Shanghai reveals natural temperature-sensitive strains conferred by the I668V mutation in the PA subunit</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">Dong</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">De-Ming</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Ming-jie</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Dong-hua</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">Qi-jian</span></span><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qu</span><span class="given-names" tagx="given-names" title="given-names">Jie-Ming</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">jmqu0906@163.com</span></span><a href="#Aff4">4</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Xin-xin</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">zhangx@shsmu.edu.cn</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a><a href="#Aff5">5</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 0368 8293</span><span class="institution-id" title="institution-id">grid.16821.3c</span><span class="institution" title="institution">Research Laboratory of Clinical Virology, Ruijin Hospital, </span><span class="institution" title="institution">Shanghai Jiaotong University School of Medicine, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 0368 8293</span><span class="institution-id" title="institution-id">grid.16821.3c</span><span class="institution" title="institution">Department of Infectious Diseases, Institute of Infectious and Respiratory Diseases, Ruijin Hospital, </span><span class="institution" title="institution">Shanghai Jiaotong University School of Medicine, </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 0368 8293</span><span class="institution-id" title="institution-id">grid.16821.3c</span><span class="institution" title="institution">Department of Respiratory Diseases, Ruijin Hospital North, </span><span class="institution" title="institution">Shanghai Jiaotong University School of Medicine, </span></div></span><span class="citation_author_institution" id="Aff4">[4], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 0368 8293</span><span class="institution-id" title="institution-id">grid.16821.3c</span><span class="institution" title="institution">Department of Respiratory Diseases, Ruijin Hospital, </span><span class="institution" title="institution">Shanghai Jiaotong University School of Medicine, </span></div></span><span class="citation_author_institution" id="Aff5">[5], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0004 0368 8293</span><span class="institution-id" title="institution-id">grid.16821.3c</span><span class="institution" title="institution">Clinical Research Center, Ruijin Hospital North, </span><span class="institution" title="institution">Shanghai Jiaotong University School of Medicine, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-10-10</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-10-10</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span></span><span class="volume" tagx="volume" title="volume">7</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">171</span><span class="history" title="history"><span class="received" title="received">received: 2018-6-2</span><span class="rev-recd" title="rev-recd">rev-recd: 2018-9-1</span><span class="accepted" title="accepted">accepted: 2018-9-5</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articleâ€™s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articleâ€™s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">Seasonal H3N2 influenza viruses are recognized as major epidemic viruses, exhibiting complex seasonal patterns in regions with temperate climates. To investigate the influence of viral evolution and mutations on the seasonality of influenza, we performed a genome-wide analysis of samples collected from 62 influenza A/H3N2-infected patients in Shanghai during 2016â€"2017. Phylogenetic analysis of all eight segments of the influenza A virus revealed that there were two epidemic influenza virus strains circulating in the 2016â€"2017 winter season (2016â€"2017win) and 2017 summer season (2017sum). Replication of the two epidemic viral strains at different temperatures (33, 35, 37, and 39â€‰Â°C) was measured, and the correlation of the mutations in the two epidemic viral strains with temperature sensitivity and viral replication was analyzed. Analysis of the replication kinetics showed that replication of the 2016â€"2017win strains was significantly restricted at 39â€‰Â°C compared with that of the 2017sum strains. A polymerase activity assay and mutational analysis demonstrated that the PA I668V mutation of the 2016â€"2017win viruses suppressed polymerase activity in vitro at high temperatures. Taken together, these data suggest that the I668V mutation in the PA subunit of the 2016â€"2017win strains may confer temperature sensitivity and attenuate viral replication and polymerase activity; meanwhile, the 2017sum strains maintained virulence at high temperatures. These findings highlight the importance of certain mutations in viral adaptation and persistence in subsequent seasons.</p> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2018" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="introduction"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par2">Seasonal influenza is responsible for an average of approximately 1 billion infections worldwide, including 250,000 to 500,000 deaths annually. Increasing evidence has suggested that seasonal influenza patterns are highly diverse in different zones, especially in Asia, where these patterns can exhibit semiannual or annual epidemic cycles. The complexity of this pattern has hindered the establishment of effective routine immunization programs<sup><a href="#CR1">1</a></sup>. China can be divided into the following three regions according to the seasonality of influenza viruses: the northern provinces experience epidemics during winter, the southern provinces experience epidemics that peak during spring, and provinces at intermediate latitudes experience semi-annual epidemics. Shanghai belongs to the category of provinces at intermediate latitudes, and influenza A epidemics, particularly seasonal A/H3N2 and A/H1N1 viruses, usually appear between autumn and spring, with the influenza activity peaking after October<sup><a href="#CR2">2</a>,<a href="#CR3">3</a></sup>.</p> <p id="Par3">Between 2014 and 2016, only sporadic influenza infections were reported during summer in Shanghai. However, in 2017, the number of infections increased substantially between May and September among inpatients at Ruijin Hospital. To further examine the influenza epidemics during the summer of 2017 in Shanghai, we compared virus samples obtained during the 2016â€"2017 winter season in Shanghai with virus samples obtained during the summer season. An etiological and viral replication analysis was performed to investigate the effects of the mutations in the internal gene segments of the influenza A virus during 2017. The influenza A virus contains eight gene segments, namely, basic polymerase 2 (PB2), basic polymerase 1 (PB1), acidic polymerase (PA), hemagglutinin (HA), nucleoprotein (NP), neuraminidase (NA), matrix (M), and nonstructural protein (NS). Antigenic drift in the surface glycoproteins (e.g., HA and NA) can facilitate viral evasion of the hostâ€™s immune system<sup><a href="#CR4">4</a></sup>. Certain mutations in the polymerases may affect temperature sensitivity, which is characterized by restricted (â‰¥100-fold) replication at a temperature of 39â€‰Â°C or higher<sup><a href="#CR5">5</a>â€"<a href="#CR8">8</a></sup>.</p> <p id="Par4">The aim of this study was to compare the complete genome sequences of two sequential influenza epidemic viruses and to investigate whether the genetic differences in the two epidemic viruses were correlated with temperature sensitivity and viral replication. The results presented here suggest that the mutation in PA I668V may confer temperature sensitivity and attenuate the polymerase activity of the influenza virus, changing the seasonality and adaptation of the virus.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="clinicalcharacterizationofpatientsinfectedwiththeh3n2influenzavirusduring2016â€" title="sec"> <div class="title" tagx="title" title="title">
Clinical characterization of patients infected with the H3N2 influenza virus during 2016â€"2017</div> <p id="Par5">Sixty-two patients were diagnosed with a H3N2 influenza virus infection from Nov. 2016 to Sep. 2017: 28 patients in the 2016â€"2017 winter season (from Nov. 2016 to Mar. 2017) and 34 patients in the 2017 summer season (from May 2017 to Sep. 2017). We divided all of the viral strains isolated from influenza-infected patients into two groups: the 2016â€"2017win virus strains and 2017sum virus strains. The median age of the hosts of the 2016â€"2017win strains was 66, with ages ranging from 19 to 94, while the median age of the hosts of the 2017sum strains was 70, with ages ranging from 33 to 90 years. Of the 62 patients, 33 (53.2%) were male and 48 (77.4%) were elderly (â‰¥60 years). Influenza-like symptoms (i.e., temperatureâ€‰â‰¥â€‰38â€‰Â°C, cough, or sore throat) were observed in all patients who were subsequently diagnosed with influenza A by real-time RT-PCR. No significant differences were observed in gender, age, underlying condition, or outcome between the two groups. The vaccination rate in Shanghai during 2016â€"2017 is unclear, but previous studies have suggested that the rate might be relatively low (approximately 2â€"16%)<sup><a href="#CR3">3</a>,<a href="#CR9">9</a>,<a href="#CR10">10</a></sup>. The clinical characteristics of these patients were excluded as possible causes of the two different epidemics. The detailed patient characteristics are described in TableÂ <a href="#Tab1">1</a>. <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Demographic characteristics and clinical features of 62 inpatients with influenza infection from 2016â€"2017</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Characteristics</th> <th>Inpatients during Nov. 2016â€" Mar. 2017</th> <th>Inpatients during May. 2017â€" Sep. 2017</th> <th><i>P</i> value</th> </tr> </thead> <tbody> <tr> <td>Age (years), median age (range)</td> <td>66 (19â€"94)</td> <td>70 (33â€"90)</td> <td>0.390<sup>a</sup></td> </tr> <tr> <td>Males/females</td> <td>16/12</td> <td>23/11</td> <td>0.438<sup>b</sup></td> </tr> <tr> <td>Elder(â‰¥60), %</td> <td>71.4</td> <td>82.4</td> <td /> </tr> <tr> <td>Fever, %</td> <td>92.9</td> <td>88.2</td> <td /> </tr> <tr> <td>Chest-X ray abnormalities, %</td> <td>85.7</td> <td>82.4</td> <td /> </tr> <tr> <td>Underlying condition, %</td> <td>75.0</td> <td>76.5</td> <td>0.893<sup>b</sup></td> </tr> <tr> <td>Outcome (cured/dead, %)</td> <td>82.1/7.1</td> <td>82.4/8.8</td> <td>0.809<sup>b</sup></td> </tr> <tr> <td>Co-infection with bacteria or fungus, %</td> <td>28.6</td> <td>20.6</td> <td /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><sup>a</sup><i>P</i> value was calculated with one-way analysis of variance</p> <p><sup>b</sup><i>P</i> values were calculated with Chi-square test</p> </div>  <p /> </div> <div class="genome-widephylogeneticanalysisindicatedthatthe2017sumvirusstrainsclearlydifferedfromthe2016â€" title="sec"> <div class="title" tagx="title" title="title">
Genome-wide phylogenetic analysis indicated that the 2017sum virus strains clearly differed from the 2016â€"2017win virus strains, with multiple mutations in several segments</div> <p id="Par6">The gene sequences of all eight influenza virus gene segments were analyzed. Genetic analysis of the eight segments (PB2, PB1, PA, HA, NP, NA, M, and NS) was performed for all of the samples and for vaccine strains isolated from the Northern Hemisphere between 2010 and 2018. According to the phylogenetic analysis, the 2017sum virus strains significantly differed from the 2016â€"2017win virus strains, and all of the 2017sum strains evolved from the 2016â€"2017win strains with several nucleotide alterations (Figs.Â <a href="#Fig1">1</a>â€"<a href="#Fig3">3</a>). The phylogenetic trees exhibited similar topological structures, with the scale bar ranging from 0.002 to 0.001. The amino acid substitutions in the proteins encoded by these gene segments are summarized in TableÂ <a href="#Tab2">2</a>, including the proportion of each substitution. Most amino acid substitutions were observed in the HA1 subunit. Compared with the WHO-recommended vaccine strains for the 2016â€"2018 influenza seasons (A/Hong Kong/4801/ 2014), the 2017sum and 2016â€"2017win virus strains contained 11 amino acid substitutions (T114S, T131K, I140T, R142K, and R261Q in the 2016â€"2017win strains and K92R, T114S, N121K, T135K/N, N171K, D291E, and H311Q in the 2017sum strains). In addition, the amino acid substitutions between the 2017sum and 2016â€"2017win strains occurred at positions 220, 303, 329, and 339 in the NA protein; 127 in the NS1 protein; 668 in the PA protein; and 299 and 340 in the PB2 protein. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <div class="title" tagx="title" title="title">
Phylogenetic analysis of the HA and NA segments circulating between Nov. 2016 and Sep. 2017.</div> <p>The 2016â€"2017win virus strains are designated Win01â€"28, and the 2017sum virus strains are designated Sum01-34. The 2017sum virus strains significantly differed from the 2016â€"2017win virus strains</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <div class="title" tagx="title" title="title">
Phylogenetic analysis of the PA, PB1 and PB2 segments circulating between Nov. 2016 and Sep. 2017.</div> <p>The 2016â€"2017win virus strains are designated Win01â€"28, and the 2017sum virus strains are designated Sum01â€"34. The phylogenetic analysis was consistent with the HA and NA analysis</p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <div class="title" tagx="title" title="title">
Phylogenetic analysis of the NP, M, and NS segments circulating between Nov. 2016 and Sep. 2017.</div> <p>The 2016â€"2017win virus strains are designated Win01â€"28, and the 2017sum virus strains are designated Sum01â€"34. The phylogenetic analysis was consistent with the analysis of the other segments</p>   </div> <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> <p>Comparison of the 2016â€"2017win, 2017sum and 2014 H3N2 virus strains with respect to the WHO recommended vaccine strains (A/Hong Kong/4801/2014)</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>Virus strain</th> <th>Protein</th> <th>Amino acid position</th> <th>Function or domain of mutation (references)</th> <th>Proportion (%)</th> <th>Mutations contained in the 2016â€"2017win isolates (2016â€"2017win02, 16, 21)</th> <th>Mutations contained in the 2017sum isolates (2017sum10, 14, 18)</th> <th>Mutations contained in the 2014 H3N2 virus strain</th> </tr> </thead> <tbody> <tr> <td rowspan="11">2016â€"2017win</td> <td>PB2</td> <td>R299K</td> <td /> <td>100</td> <td>+<sup>a</sup></td> <td>â€"</td> <td>â€"</td> </tr> <tr> <td>PB1-F2</td> <td>R25Q</td> <td>Pathogenesis associated motif<sup><a href="#CR35">35</a></sup></td> <td>100</td> <td>+</td> <td>â€"</td> <td>â€"</td> </tr> <tr> <td /> <td>H75P</td> <td /> <td>100</td> <td>+</td> <td>â€"</td> <td>â€"</td> </tr> <tr> <td>PA</td> <td>I668V</td> <td>PB1 interaction domain<sup><a href="#CR24">24</a></sup></td> <td>95.8</td> <td>+</td> <td>â€"</td> <td>â€"</td> </tr> <tr> <td>PA-X</td> <td>R15K</td> <td /> <td>100</td> <td>+</td> <td>â€"</td> <td>â€"</td> </tr> <tr> <td>HA1</td> <td>T114S</td> <td>HA stalk globular head<sup><a href="#CR36">36</a></sup></td> <td>100</td> <td>+</td> <td>â€"</td> <td>+</td> </tr> <tr> <td /> <td>T131K</td> <td /> <td>100</td> <td>+</td> <td>â€"</td> <td>â€"</td> </tr> <tr> <td /> <td>I140T</td> <td /> <td>35.7</td> <td>â€"</td> <td>â€"</td> <td>â€"</td> </tr> <tr> <td /> <td>R142K</td> <td /> <td>100</td> <td>+</td> <td>â€"</td> <td>â€"</td> </tr> <tr> <td /> <td>R261Q</td> <td /> <td>100</td> <td>+</td> <td>â€"</td> <td>â€"</td> </tr> <tr> <td>NS1</td> <td>N127S</td> <td>PKR-binding domain<sup><a href="#CR26">26</a></sup></td> <td>100</td> <td>+</td> <td>â€"</td> <td>â€"</td> </tr> <tr> <td rowspan="14">2017 sum</td> <td>PB2</td> <td>K340R</td> <td /> <td>80.8</td> <td>â€"</td> <td>+</td> <td>â€"</td> </tr> <tr> <td>PB1-F2</td> <td>R25Q</td> <td>Pathogenesis associated motif<sup><a href="#CR35">35</a></sup></td> <td>100</td> <td>â€"</td> <td>+</td> <td>â€"</td> </tr> <tr> <td /> <td>H75P</td> <td /> <td>100</td> <td>â€"</td> <td>+</td> <td>â€"</td> </tr> <tr> <td>PA-X</td> <td>R15K</td> <td /> <td>100</td> <td>â€"</td> <td>+</td> <td>â€"</td> </tr> <tr> <td>HA1</td> <td>K92R</td> <td>HA stalk globular head<sup><a href="#CR31">31</a></sup></td> <td>91.2</td> <td>â€"</td> <td>+</td> <td>â€"</td> </tr> <tr> <td /> <td>T114S</td> <td /> <td>91.2</td> <td>â€"</td> <td>+</td> <td>â€"</td> </tr> <tr> <td /> <td>N121K</td> <td /> <td>100</td> <td>â€"</td> <td>+</td> <td>â€"</td> </tr> <tr> <td /> <td>T135K</td> <td /> <td>47.1</td> <td>â€"</td> <td>+</td> <td>â€"</td> </tr> <tr> <td /> <td>T135N</td> <td /> <td>52.9</td> <td>â€"</td> <td>â€"</td> <td>â€"</td> </tr> <tr> <td /> <td>N171K</td> <td /> <td>91.2</td> <td>â€"</td> <td>+</td> <td /> </tr> <tr> <td>NA</td> <td>K220N</td> <td /> <td>96.6</td> <td>â€"</td> <td>+</td> <td>â€"</td> </tr> <tr> <td /> <td>V303I</td> <td /> <td>100</td> <td>â€"</td> <td>+</td> <td>â€"</td> </tr> <tr> <td /> <td>N329S</td> <td /> <td>96.6</td> <td>â€"</td> <td>+</td> <td>â€"</td> </tr> <tr> <td /> <td>D339N</td> <td /> <td>100</td> <td>â€"</td> <td>+</td> <td>â€"</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><sup>a</sup>Three isolates of each group and wild type virus were chosen as representative strains and the mutations in these strains were listed. Mutation existed: +, mutation didnâ€™t exist: â€"</p> </div> </div> <p /> <p id="Par7">For the M2 protein, there was no significant substitution associated with amantadine and rimantadine resistance between the 2017sum and 2016â€"2017win strains in either the gene or protein sequences. The following amino acid mutations were associated with adamantane resistance: L26F, V27A/T, A30T/V, S31N/R, G34E, A30V, S193F, and R45H<sup><a href="#CR11">11</a></sup>. Compared with the vaccine strains from the Northern Hemisphere, no important substitutions were observed in the circulating viral strains during 2016â€"2017 in the M and NP segments.</p> </div> <div class="boththe2016â€" title="sec"> <div class="title" tagx="title" title="title">
Both the 2016â€"2017win and 2017sum virus strains replicate efficiently in MDCK cells at 33â€‰Â°C</div> <p id="Par8">To investigate the mechanism by which seasonal H3N2 mutations influence viral replication, we determined the viral titers based on the TCID<sub>50</sub> at a series of time points post infection (12, 24, 48, 72, and 96â€‰h) and on the quantities of viral RNA (vRNA), cRNA and viral mRNA in Madin-Darby canine kidney (MDCK) cells infected with the 2016â€"2017win and 2017sum viral strains at a series of time points post infection (0, 4, 8, 12, 24, 48, 72, and 96â€‰h). Multiple-cycle growth curves of the 2016â€"2017win, 2017sum and 2014 H3N2 viral strains in MDCK cells are shown in Figs.Â <a href="#Fig4">4</a>,Â <a href="#Fig5">5a</a>. All of the viral strains replicated efficiently at 33â€‰Â°C. There were no significant differences among the replication efficiencies of the 2016â€"2017win, 2017sum or 2014 H3N2 viruses at 33â€‰Â°C, with mean titers ranging from 10<sup>7.9</sup> to 10<sup>8.3</sup> TCID<sub>50</sub>/ml at 96hpi, and vRNA, cRNA and mRNA were present at high levels in all of the viruses. The HA titers were determined at 96hpi. The results of the HA assay indicated that the 2016â€"2017win strains may replicate slightly more efficiently than the 2017sum strains at 33â€‰Â°C (Fig.Â <a href="#Fig5">5e</a>). The vRNA levels of all of the strains at 96hpi and 33â€‰Â°C were higher than those at other temperatures (35, 37, and 39â€‰Â°C). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <div class="title" tagx="title" title="title">
Quantity of vRNA, cRNA and mRNA in infected MDCK cells at 33, 35, 37, and 39â€‰Â°C.</div> <p>MDCK cells were infected with 2016â€"2017win, 2017sum and 2014 H3N2 viruses at an MOI of 0.02 and incubated at 33, 35, 37, and 39â€‰Â°C. The viral vRNA, cRNA, and mRNA of the NP segment in each sample were normalized by a standard curve. The error bars indicate the standard error of the mean. The <i>P</i>-values were statistically analyzed by paired <i>t</i> test. <i>P</i>â€‰<div class="â€‰0.05_UNKNOWN" /></p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <div class="title" tagx="title" title="title">
Kinetics of replication by TCID<sub>50</sub> and the HA assay of 2016â€"2017win, 2017sum and 2014 H3N2 viruses at 33C, 35, 37, and 39â€‰Â°C.</div> <p><b>a</b>â€"<b>d</b> The kinetics of replication of the 2016â€"2017win, 2017sum and 2014 H3N2 viruses at different temperatures were determined according to TCID<sub>50</sub>. <b>e</b> Influenza virus replication in MDCK cells at different temperatures was examined by the hemagglutinin assay at 96 hpi. The <i>P</i>-values were statistically analyzed by paired <i>t</i> test. *<i>P</i>â€‰<div class="â€‰0.05_UNKNOWN" /></p>   </div> <p /> </div> <div class="replicationofthe2017sumviruseswasmoreefficientthanthatofthe2016â€" title="sec"> <div class="title" tagx="title" title="title">
Replication of the 2017sum viruses was more efficient than that of the 2016â€"2017win viruses at 35â€‰Â°C and 37â€‰Â°C</div> <p id="Par9">The 2017sum virus strains replicated to a mean titer of 10<sup>7.2</sup> TCID<sub>50</sub>/ml at 35â€‰Â°C and 10<sup>7.6</sup> TCID<sub>50</sub>/ml at 37â€‰Â°C, and the 2016â€"2017win virus strains replicated to a mean titer of 10<sup>7.5</sup> TCID<sub>50</sub>/ml at 35â€‰Â°C and 10<sup>7.3</sup> TCID<sub>50</sub>/ml at 37â€‰Â°C (Fig.Â <a href="#Fig5">5b, c</a>). The final levels of vRNA, cRNA, and mRNA in MDCK cells infected with the 2016â€"2017win, 2017sum and 2014 H3N2 viruses were relatively similar at 96hpi (Fig.Â <a href="#Fig4">4</a>). However, the growth rates of the 2016â€"2017win viruses were lower than those of the 2017sum and 2014 H3N2 viruses, especially after 24hpi. There was an up to 1-log increase in the levels of the 2017sum viruses compared to those of the 2016â€"2017win viruses at 48 and 72hpi. The RNA levels of the 2017sum and 2014 H3N2 viruses peaked at 72hpi, while the RNA levels of the 2016â€"2017win viruses peaked at 96hpi. The HA titers of the 2017sum viruses at 35â€‰Â°C and 37â€‰Â°C were both higher than those of the 2016â€"2017win viruses. These data indicated that the 2017sum viruses had higher growth rates and could replicate more efficiently at 35â€‰Â°C and 37â€‰Â°C than the 2016â€"2017win viruses and that replication of the 2016â€"2017win viruses was attenuated.</p> </div> <div class="replicationofthe2016â€" title="sec"> <div class="title" tagx="title" title="title">
Replication of the 2016â€"2017win viruses was significantly restricted in MDCK cells at 39â€‰Â°C</div> <p id="Par10">To determine whether the amino acid mutations of the 2016â€"2017win and 2017sum viral strains influenced the replication of these viruses at high temperatures, we determined the viral titer and RNA levels of the 2016â€"2017win, 2017sum and 2014 H3N2 viruses at 39â€‰Â°C. The 2017sum viruses maintained a mean titer of 10<sup>6.3</sup> TCID<sub>50</sub>/ml at 39â€‰Â°C, while the 2016â€"2017win viruses replicated to a mean titer of 10<sup>3.4</sup>TCID<sub>50</sub>/ml at 39â€‰Â°C (Fig.Â <a href="#Fig5">5d</a>). The results of the HA assay were consistent with the viral TCID<sub>50</sub> titers; the HA titer of the 2017sum viruses was significantly higher than that of the 2016â€"2017win viruses. Meanwhile, the accumulation of vRNA, cRNA, and mRNA in cells infected with the 2017sum viruses was significantly higher than those in cells infected with the 2016â€"2017win viruses at 39â€‰Â°C after 24 hpi. RNA replication of the 2016â€"2017win viruses was restricted by more than 100-fold at 39â€‰Â°C compared to that at 35â€‰Â°C and 37â€‰Â°C. In addition, the peak vRNA levels of the 2016â€"2017win viruses at 39â€‰Â°C exhibited a 4-log decrease compared to the RNA levels at 33â€‰Â°C (Fig.Â <a href="#Fig4">4</a>). By contrast, the RNA accumulation and growth rates of the 2017sum viruses were stable at different temperatures (33, 35, 37, and 39â€‰Â°C) and did not exhibit any significant decrease. These data demonstrated that replication of the 2016â€"2017win viruses was significantly restricted in MDCK cells at 39â€‰Â°C, indicating that the 2016â€"2017win virus strains were temperature sensitive.</p> </div> <div class="polymeraseactivityofreplicationcomplexescontainingpb2andpamutations" title="sec"> <div class="title" tagx="title" title="title">
Polymerase activity of replication complexes containing PB2 and PA mutations</div> <p id="Par11">We assessed the effects of the viral polymerase in the 2016â€"2017win and 2017sum strains on the replication activity using a minireplicon assay. The pPolI-NP-luciferase plasmid produces a modified influenza virus NP viral RNA in which the NP-coding sequences are replaced by the luciferase gene<sup><a href="#CR12">12</a>,<a href="#CR13">13</a></sup>. Upon co-transfection of 293T cells with the pPolI-NP-luc plasmid and plasmids expressing PB2, PB1, PA, and NP of the 2016â€"2017win and 2017sum strains, we observed that the polymerase activity of all of the 2017sum and 2016â€"2017win strains were higher at 33â€‰Â°C than those at 37â€‰Â°C and 39â€‰Â°C (Fig.Â <a href="#Fig6">6</a>). The polymerases reconstituted with the PB2, PB1, PA, and NP sequences of the 2016â€"2017win strain had significantly lower activities than those of the 2017sum strains at 39â€‰Â°C, whereas the polymerase activities did not differ at 33â€‰Â°C or 37â€‰Â°C. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 6</span></a></div>   <div class="title" tagx="title" title="title">
Polymerase activities of the 2016â€"2017win, 2017sum and SC/09 viruses and their mutations.</div> <p>293T cells were transfected with a mix of PB2-, PB1-, PA-, and NP-containing plasmids from different virus strains plus a control plasmid and luciferase reporter plasmid. After transfection, 293T cells were cultured at 33â€‰Â°C (<b>a</b>), 37â€‰Â°C (<b>b</b>), and 39â€‰Â°C (<b>c</b>). The polymerase activities were measured in cell lysates 24â€‰h post-transfection. Data are expressed as the mean luciferase activityâ€‰Â±â€‰the standard error of the mean of triplicates. The P-values were statistically analyzed by paired <i>t</i> test. *<i>P</i>â€‰<div class="â€‰0.05_UNKNOWN" /></p>   </div> <p /> <p id="Par12">To understand the effects of the PB2 and PA mutations in the 2016â€"2017win and 2017sum virus strains, we introduced three individual mutations, as summarized in TableÂ <a href="#Tab2">2</a> (PB2 R299K, PB2 R340K, and PA I668V), into the reassortant polymerase complex from the 2017sum viruses. In human 293T cells, both mutations in PB2 led to similar activities at 33â€‰Â°C, 37â€‰Â°C and 39â€‰Â°C, which was consistent with the polymerase complex of the 2017sum strains. However, the polymerase reconstituted with the PA I668V mutant exhibited significantly reduced activity, a feature that was generally more pronounced at 39â€‰Â°C than at 33â€‰Â°C or 37â€‰Â°C. To verify the decrease in the activity of the PA I668V mutant at 39â€‰Â°C, an A/Sichuan/1/2009 virus polymerase complex containing the PA I668V mutation (SC/09 PA-I668V) was constructed, and the activity was measured in transfected cells. We compared the activities of the polymerase complexes of the SC/09 virus with the amino acid substitutions in the PA gene via polymerase activity assays. Compared to the polymerase activity of SC/09, the isoleucine-to-valine substitution in the mutant PA I668V of the H1N1 subtype resulted in lower polymerase activity at 39â€‰Â°C, which was consistent with the PA mutant polymerase complex of the H3N2 subtype (Fig.Â <a href="#Fig6">6c</a>). These results suggested that the temperature sensitivity of the 2016â€"2017win viruses was associated with a defect in the replication activity of the polymerase and that the PA I668V mutant might potentially lead to reduced polymerase activity at high temperatures.</p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par13">Seasonal influenza epidemics are known to result in substantial morbidity and mortality worldwide, particularly in regions with temperate climates, and the mechanism of seasonal influenza activity and transmission has not been elucidated. Influenza seasons are determined by climate regions. Worldwide, influenza epidemics always follow low seasonal temperatures, suggesting that low temperatures may change seasonal crowding, viral survival in respiratory droplets and host susceptibility<sup><a href="#CR14">14</a>,<a href="#CR15">15</a></sup>. Moreover, humidity may be a critical driver of influenza outbreaks. Absolute humidity has a regulatory effect on airborne influenza transmission, and this effect can be modulated by temperature<sup><a href="#CR16">16</a></sup>. Generally, the northern and southern regions in China experience different influenza seasonality and evolutionary dynamics. Usually, the northern region experiences an influenza season during the winter months (with a median peak in Jan., ranging from Dec. to Jan.), while the southern region experiences influenza during the summer months (with a median peak from May to Jun.). Because Shanghai has a subtropical climate but is located in the intermediate-latitude region, the influenza seasonality in Shanghai is atypical<sup><a href="#CR3">3</a>,<a href="#CR17">17</a></sup>. Between 2006 and 2016, the annual seasonality in Shanghai was irregular. Two influenza peaks were observed in 2008 and 2010, and only one epidemic occurred during the other years. However, in 2017, the following two influenza epidemics occurred in Shanghai: one epidemic between Nov. 2016 and Mar. 2017 and one epidemic between May 2017 and Sep. 2017. These epidemics were consistent with the influenza epidemics observed in Hong Kong and Guangdong in 2017. Although influenza epidemics occur frequently during summer in the southern region, the mechanism underlying summer influenza epidemics has not been elucidated<sup><a href="#CR18">18</a></sup>. In this study, we performed genomic and replication analyses for viruses obtained during both the winter and summer of 2017 in an attempt to investigate the evolution of the influenza virus during 2017 in Shanghai and the effects of mutations on viral replication in these two epidemic viral strains.</p> <p id="Par14">The results of genetic and phylogenetic analyses indicated that the 2017sum virus strains differed from the 2016â€"2017win virus strains and that there were several amino acid substitutions distributed in different viral proteins. According to the phylogenetic tree of HA, the 2017sum virus strains emerged from the 2016â€"2017win virus strains. Certain HA segments acted as intermediates during the transition from the cluster of the 2016â€"2017win viruses to the cluster of the 2017sum viruses, indicating that certain segments of the 2017sum virus strains were already present in influenza viruses circulating in 2016. This tendency was consistent with the phylogenetic analysis of other segments.</p> <p id="Par15">The influenza virus polymerase consists of three subunits, encoded by the PA, PB1, and PB2 segments, which also express several important viral proteins, such as PA-X and PB1-F2. Efficient nuclear import and assembly of the polymerase subunit RNP complex are critical steps in the viral life cycle and are closely associated with viral temperature sensitivity<sup><a href="#CR19">19</a>,<a href="#CR20">20</a></sup>. In our study, genetic analysis revealed certain unique amino acid substitutions in each group of viral strains. In the PA protein, the 2017sum strains exhibited a V668I substitution located in the PB1 interaction domain. The 2017sum viral strains differed from the 2016â€"2017win viral strains in regard to the K299R and K340R mutations in the PB2 protein. To further investigate the effects of these mutations on the influenza polymerase in the 2016â€"2017win and 2017sum virus strains, we evaluated viral replication via a TCID<sub>50</sub> measurement, an HA assay and the vRNA, cRNA, and mRNA levels of the NP segment in MDCK cells infected with these two viral strains at different temperatures (33, 35, 37, and 39â€‰Â°C). Our data showed that replication of the 2016â€"2017win viruses was significantly restricted at 39â€‰Â°C. A previous study demonstrated the role of the polymerase complex in determining the temperature range over which influenza viruses can replicate<sup><a href="#CR5">5</a></sup>. Bradel Tretheway et al.<sup><a href="#CR21">21</a></sup> showed that the human WSN polymerase complex lost activity at 39â€‰Â°C. The shutoff temperature of most temperature-sensitive viruses is 39â€‰Â°C or higher. These conclusions are consistent with our results for the 2016â€"2017win strains. In this study, replication of the 2016â€"2017win strains was similar to that of temperature-sensitive viruses, exhibiting high viral replication at 33â€‰Â°C and significantly low replication at 39â€‰Â°C.</p> <p id="Par16">Furthermore, we demonstrated that the temperature sensitivity of the 2016â€"2017win viruses was conferred by a mutation in the influenza polymerase PA subunit. Introducing a single substitution at residue 668 of the PA gene induced a strong attenuation of polymerase activity at 39â€‰Â°C, whereas the substitutions at residues 299 and 340 of PB2 had little influence on the polymerase activity at different temperatures. The mechanism by which the PA I668V mutation suppressed polymerase activity at high temperatures was not elucidated. Temperature-sensitive mutations fall into two general classes: those generating thermolabile proteins and those generating defects in protein synthesis, folding or assembly<sup><a href="#CR22">22</a></sup>. The PA amino acid 668 maps to a helix located in the C-terminal region of the PA protein<sup><a href="#CR23">23</a></sup>, suggesting that this mutation may belong to the latter class. Previous research has confirmed that this helix (residues 657â€"716) is required for binding to the N terminus of PB1, suggesting that residue 668 may be involved in proteinâ€"protein interactions and may affect polymerase assembly<sup><a href="#CR24">24</a>,<a href="#CR25">25</a></sup>. This finding is consistent with our results showing that polymerases of both the H1N1 and H3N2 viruses with the PA I668V residue exhibit reduced activity at high temperatures. The I668V mutation may regulate the PA-PB1 interaction or inhibit polymerase function.</p> <p id="Par17">In addition, there was a decrease in the accumulation rate of vRNA, cRNA, and mRNA in MDCK cells infected with the 2016â€"2017win viruses at 35â€‰Â°C and 37â€‰Â°C compared with that in cells infected with the 2017sum and 2014 H3N2 viruses after 24â€‰hpi; this result was consistent with those from the TCID<sub>50</sub> measurement and HA assay. Combined with our genome-wide analysis, we found one mutation in the NS gene that might result in a decrease in the growth rate of the 2016â€"2017win viruses. The N127S mutation identified in NS1 has not been reported previously; however, this position had been confirmed to be part of the PKR-binding site of the NS1 protein (position 123â€"127)<sup><a href="#CR26">26</a></sup>. The influenza virus NS1 protein plays a key role in blocking the activation of PKR during influenza infection and interacts with the viral polymerase to mediate the temporal regulation of vRNA synthesis during infection. Mutants of NS1 with mutations at position 123/124 or 126/127 do not bind PKR, and mutations at position 123/124 can result in the deregulation of vRNA synthesis, resulting in attenuated influenza virus replication<sup><a href="#CR27">27</a></sup>. There has been no report regarding mutations at position 127 of NS1, but our data were consistent with the results of the analysis of mutations at position 123/124 of the NS1 protein. We hypothesize that the decrease in the RNA accumulation and TCID<sub>50</sub> value in cells infected with the 2016â€"2017win viruses may result from the N127S mutation in the NS1 protein. Viruses with the N127S mutation may lose the ability to interact with the viral polymerase and exhibit low synthesis of vRNA at the early infection stage.</p> <p id="Par18">In summary, the influenza A (H3N2) virus strains circulating in the 2016â€"2017 winter season and 2017 summer season in Shanghai were clustered into two different groups according to a phylogenetic analysis. The 2016â€"2017win virus strains were temperature sensitive and attenuated compared with the 2017sum virus strains, and the 2017sum viruses exhibited an adaptation to high temperatures. A mutation in PA (at position 668) appeared to be critical for replication and temperature sensitivity. Adaptation to high temperatures may be the result of the natural evolution of the influenza virus and could explain the second epidemic that occurred during the summer of 2017 in Shanghai. These data further enhance our understanding of the summer influenza epidemic in Shanghai and subtropical Asia and thus provide insights for the prediction of the next epidemic strain of influenza A (H3N2). Our research provides information on the temperature-sensitive signature and further insight into the function of the influenza A polymerase. Future studies investigating the functional properties of viral polymerases should be performed to fully understand the evolution of the influenza virus and pave the way for the prevention of new severe influenza pandemics.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="ethicsstatement" title="sec"> <div class="title" tagx="title" title="title">
Ethics statement</div> <p id="Par19">The Research Laboratory of Clinical Virology of Ruijin Hospital collected human nasopharyngeal swab samples from residual samples that had been obtained for clinical respiratory virus detection. Each patient received a clinical report and was informed of the result of the laboratory diagnostic test before our study began. All procedures complied with the Measures for the Ethical Review of Biomedical Research Involving Human Subjects issued by the National Health and Family Planning Commission of The Peopleâ€™s Republic of China. The Ruijin Hospital Ethics Committee, Shanghai Jiaotong University School of Medicine, approved the study protocol (permit number: Ruijin Hospital Ethics Committee 2018â€"48). All experimental infection procedures were performed in biosafety level 2 facilities.</p> </div> <div class="samplecollectionandvirusisolation" title="sec"> <div class="title" tagx="title" title="title">
Sample collection and virus isolation</div> <p id="Par20">All samples were obtained from the Research Laboratory of Clinical Virology, Institute of Infectious and Respiratory Diseases, Ruijin Hospital. Nasopharyngeal specimens from the 2016â€"2017 winter and 2017 summer influenza outbreaks were collected from influenza-infected inpatients before the administration of antiviral treatment. The samples were classified into the following two groups in chronological order: 2016win (Nov. 2016 to Mar. 2017) and 2017sum (May 2017 to Sep. 2017). Nasopharyngeal swabs were placed into 2â€‰ml cryovials containing virus transport medium and stored at âˆ’80â€‰Â°C until isolation and analysis.</p> <p id="Par21">All samples were identified and subtyped by performing real-time RT-PCR. Eight complete gene segments of 47 samples from 62 influenza-infected inpatients were amplified successfully. For the other 15 samples, we analyzed only partial viral genomes because of the low levels of vRNA. In total, 448 gene segments were included in our genome-wide analysis, including the 47 PB2, 50 PB1, 49 PA, 62 HA, 59 NP, 57 NA, 62 M, and 62 NS segments. Among these 62 samples, 28 were obtained from inpatients between Nov. 2016 and Mar. 2017 (2016â€"2017win 1â€"28) and 34 were obtained from inpatients between May 2017 and Sep. 2017 (2017sum 1â€"34). All segments were submitted to the Influenza Research Database under GenBank accession numbers MG984093 to MG984541.</p> <p id="Par22">The influenza A viruses were isolated from MDCK cells as previously described<sup><a href="#CR28">28</a></sup>. We chose six isolates in this study for viral growth and replication kinetic assays: three isolates represented the 2016â€"2017win virus strains (2016â€"2017win02, 16, 21) and three isolates represented the 2017sum virus strains (2017sum10, 14, 18). The isolates from each group shared more than 99.9% identity in all eight segments. We used an influenza A (H3N2) viral strain isolated in 2014 as our control virus. This control virus strain, isolated in Ruijin Hospital in Dec. 2014, was highly homologous (&amp;gt;99%) to the WHO-recommended vaccine strains A/Texas/50/2012 (H3N2) and was designated as 2014 H3N2. The whole-genome sequences of the 2014 H3N2 viral strains was determined and submitted to GISAID (accession number EPI1230942-EPI1230949).</p> </div> <div class="plasmids" title="sec"> <div class="title" tagx="title" title="title">
Plasmids</div> <p id="Par23">A/Sichuan/01/2009 (H1N1) virus (SC/09) cDNA in pBD vectors was kindly provided by Hualan Chen (Harbin Veterinary Research Institute, Chinese Academy of Agricultural Sciences, China), and the luciferase reporter plasmid pPolI-NP-luc was provided by Ke Xu (Institute Pasteur of Shanghai, Shanghai, China). The H3N2 polymerases of the 2016â€"2017win and 2017sum strains were generated using plasmid-based reverse genetic techniques. The PB2, PB1, PA, and NS viral segments from these two viral strains were cloned into bidirectional pBD vectors. Mutations in PB2 (R229K, R340K) and PA (I668V) were introduced by site-directed mutagenesis.</p> </div> <div class="vrnaamplificationandsequencing" title="sec"> <div class="title" tagx="title" title="title">
vRNA amplification and sequencing</div> <p id="Par24">vRNA was extracted using an AxyPrep Body Fluid Viral DNA/RNA Miniprep Kit (Corning Life Sciences, New York, USA) according to the manufacturerâ€™s instructions. Contaminant DNA was eliminated using DNase. Amplification of all eight influenza genome segments (PB2, PB1, PA, HA, NP, NA, M, and NS) was performed using a HiScript II One Step RT-PCR Kit (Vazyme Biotech, Nanjing, China) and a set of primers (TableÂ <a href="#MOESM1">S1</a>). Amplification was performed under the following conditions: initial reverse transcription at 50â€‰Â°C for 15â€‰min, followed by 40 cycles of denaturation at 94â€‰Â°C for 5â€‰min, denaturation at 94â€‰Â°C for 15â€‰s, annealing at 50â€‰Â°C for 30â€‰s and extension at 72â€‰Â°C for 2â€‰min. All of the PCR products were purified using a QIAquick Gel Extraction Kit (Qiagen, Hilden, Germany). DNA sequencing was performed using the dideoxy chain termination method using a TaqDyeDeoxy Terminator Cycle-Sequencing Kit (Applied Biosystems, Foster City, USA) according to the manufacturerâ€™s instructions. The sequencing reactions were analyzed on an ABI 3730XL DNA analyzer (Applied Biosystems, Foster City, USA).</p> </div> <div class="phylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic analysis</div> <p id="Par25">The sequences of the reference strains and Northern Hemisphere vaccine strains recommended by the WHO that were included in the phylogenetic analysis were downloaded from the GISAID database. MEGA version 7.0 was used to construct phylogenetic trees using the maximum likelihood method under the general time reversibleâ€‰+â€‰proportion of invariant sitesâ€‰+â€‰shape parameter of the Î³ distribution (GTRâ€‰+â€‰Iâ€‰+â€‰Î") model as described<sup><a href="#CR29">29</a>,<a href="#CR30">30</a></sup>. The amino acids in epitopes A to E of the influenza A (H3N2) HA protein referenced in this study have been previously identified<sup><a href="#CR30">30</a></sup>.</p> </div> <div class="viralgrowthandreplicationkinetics" title="sec"> <div class="title" tagx="title" title="title">
Viral growth and replication kinetics</div> <p id="Par26">MDCK cells were infected with the 2014 H3N2, 2016â€"2017win, and 2017sum viruses at a multiplicity of infection (MOI) of 0.02 at 33â€‰, 35, 37 or 39â€‰Â°C. After 1â€‰h of virus adsorption at room temperature, the cells were washed twice with PBS and overlaid with DMEM containing 0.3% BSA, 100â€‰IU/ml penicillin and 2â€‰Î¼g/ml TPCK-treated trypsin<sup><a href="#CR31">31</a></sup>. At the indicated times (0, 4, 8, 12, 24, 48, 72, and 96hpi), the culture supernatants were collected, and the viral titers and replication kinetics were determined by a TCID<sub>50</sub> measurement (0, 12, 24, 48, 72, and 96hpi) and real-time PCR assay (0, 4, 8, 12, 24, 48, 72, and 96hpi). The dilution at which 50% of the wells were infected (TCID<sub>50</sub>) was determined as described previously<sup><a href="#CR32">32</a></sup>, and the titers were expressed as log<sub>10</sub> TCID<sub>50</sub>/ml.</p> </div> <div class="quantificationofvrna,crna,andmrnaininfectedmdckcells" title="sec"> <div class="title" tagx="title" title="title">
Quantification of vRNA, cRNA, and mRNA in infected MDCK cells</div> <p id="Par27">Total RNA was extracted and reverse transcribed according to the Hot-Start protocol as previously described<sup><a href="#CR33">33</a></sup>. Unique tags consisting of an additional 18 to 20 nucleotides unrelated to the influenza virus were added to distinguish the three types of influenza RNAs (vRNA, cRNA, and mRNA). The resulting cDNA was amplified by quantitative PCR with an ABI Prism 7500 system (Applied Biosystems, Life Technologies) using SYBR Green SuperMix (Vazyme Biotech, Nanjing, China). The primers used are listed in Supplementary TableÂ <a href="#MOESM2">S2</a>. Ten-fold serial dilutions (10<sup>8</sup>, 10<sup>7</sup>, 10<sup>6</sup>, 10<sup>5</sup>, 10<sup>4</sup>, 10<sup>3</sup> copies/Î¼l) of the vRNA plasmids were used to generate a standard curve.</p> </div> <div class="haassay" title="sec"> <div class="title" tagx="title" title="title">
HA assay</div> <p id="Par28">The HA assay was performed as previously described<sup><a href="#CR34">34</a></sup>. Briefly, the cell culture supernatants of infected cells at 96hpi were serially diluted with PBS, and an equal volume of 1% cavy erythrocytes was added. The viral titers were calculated 40â€‰min later.</p> </div> <div class="luciferaseassayofviralpolymeraseactivity" title="sec"> <div class="title" tagx="title" title="title">
Luciferase assay of viral polymerase activity</div> <p id="Par29">Monolayers of 293T cells were transfected with 125â€‰ng of the luciferase reporter plasmid pPolI-NP-luc and 12.5â€‰ng of the internal control plasmid pRL-CMV (Promega, Wisconsin, USA) with a mix of PB2-, PB1-, PA-, and NP-containing plasmids at quantities of 125, 125, 125, and 250â€‰ng, respectively. The transfected 293T cells were incubated at 33, 37, and 39â€‰Â°C and lysed 24â€‰h post transfection. The luciferase activity was analyzed with a dual-luciferase assay system (Promega, Wisconsin, USA) according to the manufacturerâ€™s instructions. The results were the averages from three independent experiments.</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p id="Par30">Qualitative and quantitative variables were compared between the two groups (2016â€"2017win and 2017sum) using the Chi-squared test and one-way analysis of variance (one-way ANOVA). No adjustment for multiple testing was performed. The levels of the replication and polymerase activity were statistically analyzed by a paired <i>t</i> test. Statistical analysis was performed using SPSS 18.0 (SPSS, Inc., Chicago, USA).</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <div class="sec" title="sec"> <p> </p><div class="local-data" title="local-data"> <div class="media" title="media"><a href="41426_2018_172_MOESM1_ESM.docx">LINK</a> <p>Table S1</p>  </div> </div> <div class="local-data" title="local-data"> <div class="media" title="media"><a href="41426_2018_172_MOESM2_ESM.docx">LINK</a> <p>Table S2</p>  </div> </div> <p /> </div> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher's note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="electronicsupplementarymaterial" title="sec"> <div class="title" tagx="title" title="title">
Electronic supplementary material</div> <p><b>Supplementary Information</b> accompanies this paper at (10.1038/s41426-018-0172-4).</p> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We acknowledge Professor Hualan Chen for her help to generate the reconstituted polymerase. We thank Professor Ke Xu for her technical support in this study. This study was supported by a Shanghai Key Discipline Grant for Respiratory Diseases (2017ZZ02014) and the Suburban Hospital Clinical Capacity Building Programme of the Shanghai Shenkang Hospital Development Center, China (SHDC12015901).</p> </div> <div class="title" tagx="title" title="title">
Conflict of interest</div> <p id="Par31">The authors declare that they have no conflict of interest.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Global circulation patterns of seasonal influenza viruses vary with antigenic drift</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">523</span><span class="fpage" tagx="fpage" title="fpage">217</span><span class="lpage" tagx="lpage" title="lpage">220</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature14460">10.1038/nature14460</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26053121">26053121</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saha</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chadha</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Divergent seasonal patterns of influenza types A and B across latitude gradient in Tropical Asia</span><span class="source" tagx="source" title="source">Influenza Other Respir. Virus.</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">176</span><span class="lpage" tagx="lpage" title="lpage">184</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/irv.12372">10.1111/irv.12372</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of regional influenza seasonality patterns in China and implications for vaccination strategies: spatio-temporal modeling of surveillance data</span><span class="source" tagx="source" title="source">PLoS Med.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">e1001552</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pmed.1001552">10.1371/journal.pmed.1001552</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24348203">24348203</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zinder</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedford</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gupta</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pascual</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="mixed-article-title" title="mixed-article-title">The roles of competition and mutation in shaping antigenic and genetic diversity in influenza</span><span class="source" tagx="source" title="source">PLoS Pathog.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">e1003104</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1003104">10.1371/journal.ppat.1003104</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23300455">23300455</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Multiple amino acid residues confer temperature sensitivity to human influenza virus vaccine strains (FluMist) derived from cold-adapted A/Ann Arbor/6/60</span><span class="source" tagx="source" title="source">Virology</span><span class="year" tagx="year" title="year">2003</span><span class="volume" tagx="volume" title="volume">306</span><span class="fpage" tagx="fpage" title="fpage">18</span><span class="lpage" tagx="lpage" title="lpage">24</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0042-6822(02)00035-1">10.1016/S0042-6822(02)00035-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12620793">12620793</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kemble</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="mixed-article-title" title="mixed-article-title">The cold adapted and temperature sensitive influenza A/Ann Arbor/6/60 virus, the master donor virus for live attenuated influenza vaccines, has multiple defects in replication at the restrictive temperature</span><span class="source" tagx="source" title="source">Virology</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">380</span><span class="fpage" tagx="fpage" title="fpage">304</span><span class="lpage" tagx="lpage" title="lpage">311</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2008.07.027">10.1016/j.virol.2008.07.027</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18768193">18768193</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kemble</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Imparting temperature sensitivity and attenuation in ferrets to A/Puerto Rico/8/34 influenza virus by transferring the genetic signature for temperature sensitivity from cold-adapted A/Ann Arbor/6/60</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">78</span><span class="fpage" tagx="fpage" title="fpage">995</span><span class="lpage" tagx="lpage" title="lpage">998</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.78.2.995-998.2004">10.1128/JVI.78.2.995-998.2004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14694130">14694130</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noton</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Studies of an influenza A virus temperature-sensitive mutant identify a late role for NP in the formation of infectious virions</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">83</span><span class="fpage" tagx="fpage" title="fpage">562</span><span class="lpage" tagx="lpage" title="lpage">571</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01424-08">10.1128/JVI.01424-08</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18987140">18987140</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Seasonal influenza vaccine supply and target vaccinated population in China, 2004-2009</span><span class="source" tagx="source" title="source">Vaccine</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">6778</span><span class="lpage" tagx="lpage" title="lpage">6782</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.vaccine.2010.07.064">10.1016/j.vaccine.2010.07.064</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20688038">20688038</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Seasonal influenza vaccination coverage rate of target groups in selected cities and provinces in China by season (2009/10 to 2011/12)</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">e73724</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0073724">10.1371/journal.pone.0073724</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24040041">24040041</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carr</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Oseltamivir-resistant influenza A and B viruses pre- and postantiviral therapy in children and young adults with cancer</span><span class="source" tagx="source" title="source">Pediatr. Infect. Dis. J.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">284</span><span class="lpage" tagx="lpage" title="lpage">288</span><span class="pub-id"><a href="https://dx.doi.org/10.1097/INF.0b013e3181ff863b">10.1097/INF.0b013e3181ff863b</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21048522">21048522</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moncorge</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Investigation of influenza virus polymerase activity in pig cells</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">87</span><span class="fpage" tagx="fpage" title="fpage">384</span><span class="lpage" tagx="lpage" title="lpage">394</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01633-12">10.1128/JVI.01633-12</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23077313">23077313</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Regan</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parslow</span><span class="given-names" tagx="given-names" title="given-names">TG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Defective assembly of influenza A virus due to a mutation in the polymerase subunit PA</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">80</span><span class="fpage" tagx="fpage" title="fpage">252</span><span class="lpage" tagx="lpage" title="lpage">261</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.80.1.252-261.2006">10.1128/JVI.80.1.252-261.2006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16352550">16352550</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Azziz Baumgartner</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Seasonality, timing, and climate drivers of influenza activity worldwide</span><span class="source" tagx="source" title="source">J. Infect. Dis.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">206</span><span class="fpage" tagx="fpage" title="fpage">838</span><span class="lpage" tagx="lpage" title="lpage">846</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jis467">10.1093/infdis/jis467</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22829641">22829641</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yaari</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katriel</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huppert</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Axelsen</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stone</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="mixed-article-title" title="mixed-article-title">Modelling seasonal influenza: the role of weather and punctuated antigenic drift</span><span class="source" tagx="source" title="source">J. R. Soc., Interface</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">20130298</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rsif.2013.0298">10.1098/rsif.2013.0298</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23676899">23676899</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deyle</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maher</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hernandez</span><span class="given-names" tagx="given-names" title="given-names">RD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Basu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sugihara</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Global environmental drivers of influenza</span><span class="source" tagx="source" title="source">Proc. Natl Acad. Sci. USA</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">113</span><span class="fpage" tagx="fpage" title="fpage">13081</span><span class="lpage" tagx="lpage" title="lpage">13086</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.1607747113">10.1073/pnas.1607747113</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27799563">27799563</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saha</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Influenza seasonality and vaccination timing in tropical and subtropical areas of southern and south-eastern Asia</span><span class="source" tagx="source" title="source">Bull. World Health Organ</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">92</span><span class="fpage" tagx="fpage" title="fpage">318</span><span class="lpage" tagx="lpage" title="lpage">330</span><span class="pub-id"><a href="https://dx.doi.org/10.2471/BLT.13.124412">10.2471/BLT.13.124412</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24839321">24839321</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">AW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glass</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">Dynamic patterns of avian and human influenza in east and southeast Asia</span><span class="source" tagx="source" title="source">Lancet Infect. Dis.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">543</span><span class="lpage" tagx="lpage" title="lpage">548</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1473-3099(07)70186-X">10.1016/S1473-3099(07)70186-X</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17646027">17646027</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayashi</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wills</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bussey</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takimoto</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of influenza A virus PB2 residues involved in enhanced polymerase activity and virus growth in mammalian cells at low temperatures</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">89</span><span class="fpage" tagx="fpage" title="fpage">8042</span><span class="lpage" tagx="lpage" title="lpage">8049</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00901-15">10.1128/JVI.00901-15</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26018156">26018156</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Functional</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">90</span><span class="fpage" tagx="fpage" title="fpage">2938</span><span class="lpage" tagx="lpage" title="lpage">2947</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.02400-15">10.1128/JVI.02400-15</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26719244">26719244</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bradel-Tretheway</span><span class="given-names" tagx="given-names" title="given-names">BG</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The human H5N1 influenza A virus polymerase complex is active in vitro over a broad range of temperatures, in contrast to the WSN complex, and this property can be attributed to the PB2 subunit</span><span class="source" tagx="source" title="source">J. Gen. Virol.</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">89</span><span class="fpage" tagx="fpage" title="fpage">2923</span><span class="lpage" tagx="lpage" title="lpage">2932</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/vir.0.2008/006254-0">10.1099/vir.0.2008/006254-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19008377">19008377</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gordon</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">King</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic properties of temperature-sensitive folding mutants of the coat protein of phage P22</span><span class="source" tagx="source" title="source">Genetics</span><span class="year" tagx="year" title="year">1994</span><span class="volume" tagx="volume" title="volume">136</span><span class="fpage" tagx="fpage" title="fpage">427</span><span class="lpage" tagx="lpage" title="lpage">438</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8150274">8150274</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawaguchi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nagata</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">De novo replication of the influenza virus RNA genome is regulated by DNA replicative helicase, MCM</span><span class="source" tagx="source" title="source">EMBO J.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">26</span><span class="fpage" tagx="fpage" title="fpage">4566</span><span class="lpage" tagx="lpage" title="lpage">4575</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/sj.emboj.7601881">10.1038/sj.emboj.7601881</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17932485">17932485</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Obayashi</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The structural basis for an essential subunit interaction in influenza virus RNA polymerase</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">454</span><span class="fpage" tagx="fpage" title="fpage">1127</span><span class="lpage" tagx="lpage" title="lpage">1131</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature07225">10.1038/nature07225</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18660801">18660801</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghanem</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Peptide-mediated interference with influenza A virus polymerase</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">81</span><span class="fpage" tagx="fpage" title="fpage">7801</span><span class="lpage" tagx="lpage" title="lpage">7804</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00724-07">10.1128/JVI.00724-07</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17494067">17494067</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">min</span><span class="given-names" tagx="given-names" title="given-names">JY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sen</span><span class="given-names" tagx="given-names" title="given-names">GC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krug</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span></span><span class="mixed-article-title" title="mixed-article-title">A site on the influenza A virus NS1 protein mediates both inhibition of PKR activation and temporal regulation of viral RNA synthesis</span><span class="source" tagx="source" title="source">Virology</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">363</span><span class="fpage" tagx="fpage" title="fpage">236</span><span class="lpage" tagx="lpage" title="lpage">243</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2007.01.038">10.1016/j.virol.2007.01.038</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17320139">17320139</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Robust expression of vault RNAs induced by influenza A virus plays a critical role in suppression of PKR-mediated innate immunity</span><span class="source" tagx="source" title="source">Nucleic Acids Res.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">10321</span><span class="lpage" tagx="lpage" title="lpage">10337</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26490959">26490959</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balish</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katz</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klimov</span><span class="given-names" tagx="given-names" title="given-names">AI</span></span></span><span class="mixed-article-title" title="mixed-article-title">Influenza: propagation, quantification, and storage</span><span class="source" tagx="source" title="source">Curr. Protoc. Microbiol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">Chapter 15</span><span class="fpage" tagx="fpage" title="fpage">11</span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">BG</span></span></span><span class="mixed-article-title" title="mixed-article-title">Building phylogenetic trees from molecular data with MEGA</span><span class="source" tagx="source" title="source">Mol. Biol. Evol.</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">1229</span><span class="lpage" tagx="lpage" title="lpage">1235</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/mst012">10.1093/molbev/mst012</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23486614">23486614</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Castoe</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doan</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parkinson</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Data partitions and complex models in Bayesian analysis: the phylogeny of Gymnophthalmid lizards</span><span class="source" tagx="source" title="source">Syst. Biol.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">53</span><span class="fpage" tagx="fpage" title="fpage">448</span><span class="lpage" tagx="lpage" title="lpage">469</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/10635150490445797">10.1080/10635150490445797</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15503673">15503673</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Da Costa</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Temperature-sensitive mutants in the influenza A virus RNA polymerase: alterations in the PA linker reduce nuclear targeting of the PB1-PA dimer and result in viral attenuation</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">89</span><span class="fpage" tagx="fpage" title="fpage">6376</span><span class="lpage" tagx="lpage" title="lpage">6390</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00589-15">10.1128/JVI.00589-15</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25855727">25855727</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biacchesi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Rapid human metapneumovirus microneutralization assay based on green fluorescent protein expression</span><span class="source" tagx="source" title="source">J. Virol. Methods</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">128</span><span class="fpage" tagx="fpage" title="fpage">192</span><span class="lpage" tagx="lpage" title="lpage">197</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jviromet.2005.05.005">10.1016/j.jviromet.2005.05.005</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15955576">15955576</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawakami</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Strand-specific real-time RT-PCR for distinguishing influenza vRNA, cRNA, and mRNA</span><span class="source" tagx="source" title="source">J. Virol. Methods</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">173</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">6</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jviromet.2010.12.014">10.1016/j.jviromet.2010.12.014</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21185869">21185869</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Transport of influenza virus neuraminidase (NA) to host cell surface is regulated by ARHGAP21 and Cdc42 proteins</span><span class="source" tagx="source" title="source">J. Biol. Chem.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">287</span><span class="fpage" tagx="fpage" title="fpage">9804</span><span class="lpage" tagx="lpage" title="lpage">9816</span><span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M111.312959">10.1074/jbc.M111.312959</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22318733">22318733</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alymova</span><span class="given-names" tagx="given-names" title="given-names">IV</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Immunopathogenic and antibacterial effects of H3N2 influenza A virus PB1-F2 map to amino acid residues 62, 75, 79, and 82</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">85</span><span class="fpage" tagx="fpage" title="fpage">12324</span><span class="lpage" tagx="lpage" title="lpage">12333</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.05872-11">10.1128/JVI.05872-11</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21937639">21937639</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gambaryan</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Receptor-binding profiles of H7 subtype influenza viruses in different host species</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">86</span><span class="fpage" tagx="fpage" title="fpage">4370</span><span class="lpage" tagx="lpage" title="lpage">4379</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.06959-11">10.1128/JVI.06959-11</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22345462">22345462</a></span></span></li> </ul> </div> </div>  </body></html>