<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/10/2020.04.08.031963.atom" />
<meta name="HW.pisa" content="biorxiv;2020.04.08.031963v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="BioLaboro: A bioinformatics system for detecting molecular assay signature erosion and designing new assays in response to emerging and reemerging pathogens" />
<meta name="DC.Identifier" content="10.1101/2020.04.08.031963" />
<meta name="DC.Date" content="2020-04-10" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Background Emerging and reemerging infectious diseases such as the novel Coronavirus disease, COVID-19 and Ebola pose a significant threat to global society and test the public health community’s preparedness to rapidly respond to an outbreak with effective diagnostics and therapeutics. Recent advances in next generation sequencing technologies enable rapid generation of pathogen genome sequence data, within 24 hours of obtaining a sample in some instances. With these data, one can quickly evaluate the effectiveness of existing diagnostics and therapeutics using in silico approaches. The propensity of some viruses to rapidly accumulate mutations can lead to the failure of molecular detection assays creating the need for redesigned or newly designed assays.

Results Here we describe a bioinformatics system named BioLaboro to identify signature regions in a given pathogen genome, design PCR assays targeting those regions, and then test the PCR assays in silico to determine their sensitivity and specificity. We demonstrate BioLaboro with two use cases: Bombali Ebolavirus (BOMV) and the novel Coronavirus 2019 (SARS-CoV-2). For the BOMV, we analyzed 30 currently available real-time reverse transcription-PCR assays against the three available complete genome sequences of BOMV. Only two met our in silico criteria for successful detection and neither had perfect matches to the primer/probe sequences. We designed five new primer sets against BOMV signatures and all had true positive hits to the three BOMV genomes and no false positive hits to any other sequence. Four assays are closely clustered in the nucleoprotein gene and one is located in the glycoprotein gene. Similarly, for the SARS-CoV-2, we designed five highly specific primer sets that hit all 145 whole genomes (available as of February 28, 2020) and none of the near neighbors.

Conclusions Here we applied BioLaboro in two real-world use cases to demonstrate its capability; 1) to identify signature regions, 2) to assess the efficacy of existing PCR assays to detect pathogens as they evolve over time, and 3) to design new assays with perfect in silico detection accuracy, all within hours, for further development and deployment. BioLaboro is designed with a user-friendly graphical user interface for biologists with limited bioinformatics experience.

### Competing Interest Statement



*   bp
    :   base pair (bps, plural base pairs)
    BDBV
    :   Bundibugyo ebolavirus
    BLAST
    :   Basic Local Alignment Search Tool
    BOMV
    :   Bombali ebolavirus
    CFR
    :   Case Fatality Ratio
    COVID-19
    :   Coronavirus Disease 2019
    DNA
    :   deoxyribonucleic acid
    DRC
    :   Democratic Republic of the Congo
    EBOV
    :   Zaire ebolavirus
    EMA
    :   European Medicines Agency
    EVD
    :   Ebola Virus Disease
    FDA
    :   Food and Drug Administration
    FN
    :   False Negative
    FP
    :   False Positive
    GC
    :   guanine cytosine
    GP
    :   glycoprotein
    GUI
    :   Graphic User interface
    ID
    :   identifier
    mAb
    :   monoclonal antibody
    MAFFT
    :   Multiple Sequence Alignment using Fast Fourier Transform
    MCM
    :   medical countermeasure
    N
    :   nucleocapsid phosphoprotein
    N/A
    :   Not available
    NCBI
    :   National Center for Biotechnology Information
    NIAID
    :   National Institute of Allergy and Infectious Diseases
    NP
    :   nucleoprotein
    NPC1
    :   Niemann-Pick C1
    ORF
    :   Open Reading Frame
    PALM
    :   Pamoja Tulinde Maisha study
    PCR
    :   polymerase chain reaction
    PSET
    :   PCR signature erosion tool
    RAM
    :   random access memory
    RESTV
    :   Reston ebolavirus
    RNA
    :   ribonucleic acid
    rRT-PCR
    :   real-time reverse-transcription polymerase chain reaction
    rVSV
    :   recombinant vesicular stomatitis virus
    SARS
    :   severe acute respiratory syndrome
    SNP
    :   single nucleotide protein
    SUDV
    :   Sudan ebolavirus
    TAFV
    :   Taï Forest ebolavirus
    TN
    :   True Negative
    TP
    :   True Positive
    WGS
    :   Whole Genome Sequence
    WHO
    :   World Health Organization" />
<meta name="DC.Contributor" content="Mitchell Holland" />
<meta name="DC.Contributor" content="Daniel Negrón" />
<meta name="DC.Contributor" content="Shane Mitchell" />
<meta name="DC.Contributor" content="Nate Dellinger" />
<meta name="DC.Contributor" content="Mychal Ivancich" />
<meta name="DC.Contributor" content="Tyler Barrus" />
<meta name="DC.Contributor" content="Sterling Thomas" />
<meta name="DC.Contributor" content="Katharine W. Jennings" />
<meta name="DC.Contributor" content="Bruce Goodwin" />
<meta name="DC.Contributor" content="Shanmuga Sozhamannan" />
<meta name="article:published_time" content="2020-04-10" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="BioLaboro: A bioinformatics system for detecting molecular assay signature erosion and designing new assays in response to emerging and reemerging pathogens" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Background: Emerging and reemerging infectious diseases such as the novel Coronavirus disease, COVID-19 and Ebola pose a significant threat to global society and test the preparedness of the public health community to rapidly respond to an outbreak with effective diagnostics and therapeutics. Recent advances in next generation sequencing technologies enable rapid generation of pathogen genome sequence data, within 24 hours of obtaining a sample in some instances. With these data, one can quickly evaluate the effectiveness of existing diagnostics and therapeutics using in silico approaches. The propensity of some viruses to rapidly accumulate mutations can lead to the failure of molecular detection assays creating the need for redesigned or newly designed assays. Results: Here we describe a bioinformatics system named BioLaboro to identify signature regions in a given pathogen genome, design PCR assays targeting those regions, and then test the PCR assays in silico to determine their sensitivity and specificity. We demonstrate BioLaboro with two use cases: Bombali Ebolavirus (BOMV) and the novel Coronavirus 2019 (SARS-CoV-2). For the BOMV, we analyzed 30 currently available real-time reverse transcription-PCR assays against the three available complete genome sequences of BOMV. Only two met our in silico criteria for successful detection and neither had perfect matches to the primer/probe sequences. We designed five new primer sets against BOMV signatures and all had true positive hits to the three BOMV genomes and no false positive hits to any other sequence. Four assays are closely clustered in the nucleoprotein gene and one is located in the glycoprotein gene. Similarly, for the SARS-CoV-2, we designed five highly specific primer sets that hit all 145 whole genomes (available as of February 28, 2020) and none of the near neighbors. Conclusions: Here we applied BioLaboro in two real-world use cases to demonstrate its capability; 1) to identify signature regions, 2) to assess the efficacy of existing PCR assays to detect pathogens as they evolve over time, and 3) to design new assays with perfect in silico detection accuracy, all within hours, for further development and deployment. BioLaboro is designed with a user-friendly graphical user interface for biologists with limited bioinformatics experience.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.04.08.031963v1" />
<meta name="citation_id" content="2020.04.08.031963v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.08.031963v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.08.031963v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.08.031963v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.04.08.031963" />
<meta name="citation_num_pages" content="39" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.04.08.031963" />
<meta name="citation_author" content="Mitchell Holland" />
<meta name="citation_author_institution" content="Noblis" />
<meta name="citation_author" content="Daniel Negrón" />
<meta name="citation_author_institution" content="Noblis" />
<meta name="citation_author" content="Shane Mitchell" />
<meta name="citation_author_institution" content="Noblis" />
<meta name="citation_author" content="Nate Dellinger" />
<meta name="citation_author_institution" content="Noblis" />
<meta name="citation_author" content="Mychal Ivancich" />
<meta name="citation_author_institution" content="Noblis" />
<meta name="citation_author" content="Tyler Barrus" />
<meta name="citation_author_institution" content="Noblis" />
<meta name="citation_author" content="Sterling Thomas" />
<meta name="citation_author_institution" content="Noblis" />
<meta name="citation_author" content="Katharine W. Jennings" />
<meta name="citation_author_institution" content="Noblis" />
<meta name="citation_author" content="Bruce Goodwin" />
<meta name="citation_author_institution" content="Defense Biological Product Assurance Office (DBPAO), Joint Program Executive Office for Chemical, Biological, Radiological and Nuclear Defense (JPEO-CBRND), Enabling Biotechnologies" />
<meta name="citation_author" content="Shanmuga Sozhamannan" />
<meta name="citation_author_institution" content="Defense Biological Product Assurance Office (DBPAO), Joint Program Executive Office for Chemical, Biological, Radiological and Nuclear Defense (JPEO-CBRND), Enabling Biotechnologies" />
<meta name="citation_author_institution" content="Logistics Management Institute" />
<meta name="citation_author_email" content="Shanmuga.Sozhamannan.ctr@mail.mil" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6762-7874" />
<meta name="citation_reference" content="Fauci AS, Touchette NA, Folkers GK: Emerging infectious diseases: a 10-year perspective from the National Institute of Allergy and Infectious Diseases. Emerg Infect Dis 2005, 11(4):519–525." />
<meta name="citation_reference" content="Morens DM, Fauci AS: Emerging infectious diseases: threats to human health and global stability. PLoS Pathog 2013, 9(7):e1003467." />
<meta name="citation_reference" content="Paules CI, Eisinger RW, Marston HD, Fauci AS: What Recent History Has Taught Us About Responding to Emerging Infectious Disease Threats. Ann Intern Med 2017, 167(11):805–811." />
<meta name="citation_reference" content="Fonkwo PN: Pricing infectious disease. The economic and health implications of infectious diseases. EMBO Rep 2008, 9 Suppl 1:S13–17." />
<meta name="citation_reference" content="Bloom DE, Black S, Rappuoli R: Emerging infectious diseases: A proactive approach. Proc Natl Acad Sci U S A 2017, 114(16):4055–4059." />
<meta name="citation_reference" content="World Health Organization: Coronavirus disease 2019 (COVID-19) Situation Report – 50 [https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200310-sitrep-50-covid-19.pdf?sfvrsn=55e904fb_2]" />
<meta name="citation_reference" content="GISAID. [https://www.gisaid.org/epiflu-applications/global-cases-covid-19/]" />
<meta name="citation_reference" content="Dong E, Du H, Gardner L: An interactive web-based dashboard to track COVID-19 in real time. Lancet Infect Dis 2020." />
<meta name="citation_reference" content="World Health Organization: Fact Sheet/Details/Ebola virus disease [https://www.who.int/news-room/fact-sheets/detail/ebola-virus-disease]" />
<meta name="citation_reference" content="Mari Saez A, Weiss S, Nowak K, Lapeyre V, Zimmermann F, Dux A, Kuhl HS, Kaba M, Regnaut S, Merkel K et al: Investigating the zoonotic origin of the West African Ebola epidemic. EMBO Mol Med 2015, 7(1):17–23." />
<meta name="citation_reference" content="World Health Organization: Situation Report-Ebola Virus Disease-10 June 2016. https://www.who.int/csr/disease/ebola/situation-reports/archive/en/." />
<meta name="citation_reference" content="World Health Organization: Ebola in the Democratic Republic of Congo-Health Emergecy Update [https://www.who.int/emergencies/diseases/ebola/drc-2019]." />
<meta name="citation_reference" content="Mulangu S, Dodd LE, Davey RT, Jr., Tshiani Mbaya O, Proschan M, Mukadi D, Lusakibanza Manzo M, Nzolo D, Tshomba Oloma A, Ibanda A et al: A Randomized, Controlled Trial of Ebola Virus Disease Therapeutics. N Engl J Med 2019, 381(24):2293–2303." />
<meta name="citation_reference" content="European Medicines Agency: First Vaccine to protect against Ebola. In: EMA/CHMP/565403/2019. 2019." />
<meta name="citation_reference" content="Food and Drug Administration: First FDA-approved vaccine for the prevention of Ebola virus disease, marking a critical milestone in public health preparedness and response. In.; 2019. https://www.fda.gov/news-events/press-announcements/first-fda-approved-vaccine-prevention-ebola-virus-disease-marking-critical-milestone-public-health." />
<meta name="citation_reference" content="Zawilinska B, Kosz-Vnenchak M: General introduction into the Ebola virus biology and disease. Folia Med Cracov 2014, 54(3):57–65." />
<meta name="citation_reference" content="Baseler L, Chertow DS, Johnson KM, Feldmann H, Morens DM: The Pathogenesis of Ebola Virus Disease. Annu Rev Pathol 2017, 12:387–418." />
<meta name="citation_reference" content="Cantoni D, Hamlet A, Michaelis M, Wass MN, Rossman JS: Risks Posed by Reston, the Forgotten Ebolavirus. mSphere 2016, 1(6)." />
<meta name="citation_reference" content="Miranda ME, Miranda NL: Reston ebolavirus in humans and animals in the Philippines: a review. J Infect Dis 2011, 204 Suppl 3:S757–760." />
<meta name="citation_reference" content="Goldstein T, Anthony SJ, Gbakima A, Bird BH, Bangura J, Tremeau-Bravard A, Belaganahalli MN, Wells HL, Dhanota JK, Liang E et al: The discovery of Bombali virus adds further support for bats as hosts of ebolaviruses. Nat Microbiol 2018, 3(10):1084–1089." />
<meta name="citation_reference" content="Leroy EM, Kumulungui B, Pourrut X, Rouquet P, Hassanin A, Yaba P, Delicat A, Paweska JT, Gonzalez JP, Swanepoel R: Fruit bats as reservoirs of Ebola virus. Nature 2005, 438(7068):575–576." />
<meta name="citation_reference" content="Heeney JL: Ebola: Hidden reservoirs. Nature 2015, 527(7579):453–455." />
<meta name="citation_reference" content="Leendertz SA, Gogarten JF, Dux A, Calvignac-Spencer S, Leendertz FH: Assessing the Evidence Supporting Fruit Bats as the Primary Reservoirs for Ebola Viruses. Ecohealth 2016, 13(1):18–25." />
<meta name="citation_reference" content="Olival KJ, Hayman DT: Filoviruses in bats: current knowledge and future directions. Viruses 2014, 6(4):1759–1788." />
<meta name="citation_reference" content="Pappalardo M, Julia M, Howard MJ, Rossman JS, Michaelis M, Wass MN: Conserved differences in protein sequence determine the human pathogenicity of Ebolaviruses. Sci Rep 2016, 6:23743." />
<meta name="citation_reference" content="Martell HJ, Masterson SG, McGreig JE, Michaelis M, Wass MN: Is the Bombali virus pathogenic in humans? Bioinformatics 2019, 35(19):3553–3558." />
<meta name="citation_reference" content="World Health Organization: WHO Director-General’s opening remarks at the media briefing on COVID-19 - 11 March 2020. In.; 2020. https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19---11-march-2020." />
<meta name="citation_reference" content="Huang C, Wang Y, Li X, Ren L, Zhao J, Hu Y, Zhang L, Fan G, Xu J, Gu X et al: Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet 2020, 395(10223):497–506." />
<meta name="citation_reference" content="Zhou P, Yang XL, Wang XG, Hu B, Zhang L, Zhang W, Si HR, Zhu Y, Li B, Huang CL et al: A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 2020." />
<meta name="citation_reference" content="World Health Organization: Coronavirus disease 2019 (COVID-19)Situation Report – 49 [https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200309-sitrep-49-covid-19.pdf?sfvrsn=70dabe61_4]" />
<meta name="citation_reference" content="Elbe S, Buckland-Merrett G: Data, disease and diplomacy: GISAID’s innovative contribution to global health. Glob Chall 2017, 1(1):33–46." />
<meta name="citation_reference" content="Sozhamannan S, Holland MY, Hall AT, Negron DA, Ivancich M, Koehler JW, Minogue TD, Campbell CE, Berger WJ, Christopher GW et al: Evaluation of Signature Erosion in Ebola Virus Due to Genomic Drift and Its Impact on the Performance of Diagnostic Assays. Viruses 2015, 7(6):3130–3154." />
<meta name="citation_reference" content="Slezak T, Kuczmarski T, Ott L, Torres C, Medeiros D, Smith J, Truitt B, Mulakken N, Lam M, Vitalis E et al: Comparative genomics tools applied to bioterrorism defence. Brief Bioinform 2003, 4(2):133–149." />
<meta name="citation_reference" content="Gardner SN, Lam MW, Smith JR, Torres CL, Slezak TR: Draft versus finished sequence data for DNA and protein diagnostic signature development. Nucleic Acids Res 2005, 33(18):5838–5850." />
<meta name="citation_reference" content="Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ: Basic local alignment search tool. J Mol Biol 1990, 215(3):403–410." />
<meta name="citation_reference" content="Madeira F, Park YM, Lee J, Buso N, Gur T, Madhusoodanan N, Basutkar P, Tivey ARN, Potter SC, Finn RD et al: The EMBL-EBI search and sequence analysis tools APIs in 2019. Nucleic Acids Res 2019, 47(W1):W636–W641." />
<meta name="citation_reference" content="Bao Y, Amarasinghe GK, Basler CF, Bavari S, Bukreyev A, Chandran K, Dolnik O, Dye JM, Ebihara H, Formenty P et al: Implementation of Objective PASC-Derived Taxon Demarcation Criteria for Official Classification of Filoviruses. Viruses 2017, 9(5)." />
<meta name="citation_reference" content="Zulkower V, and Rosser, S.: DNA Features Viewer, a sequence annotations formatting and plotting library for Python. BioRxiv 2020." />
<meta name="citation_reference" content="Patel A, Jernigan DB, nCo VCDCRT: Initial Public Health Response and Interim Clinical Guidance for the 2019 Novel Coronavirus Outbreak - United States, December 31, 2019-February 4, 2020. MMWR Morb Mortal Wkly Rep 2020, 69(5):140–146." />
<meta name="citation_reference" content="Corman VM, Landt O, Kaiser M, Molenkamp R, Meijer A, Chu DKW, Bleicker T, Brunink S, Schneider J, Schmidt ML et al: Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR. Euro Surveill 2020, 25(3)." />
<meta name="citation_reference" content="Forbes KM, Webala PW, Jaaskelainen AJ, Abdurahman S, Ogola J, Masika MM, Kivisto I, Alburkat H, Plyusnin I, Levanov L et al: Bombali Virus in Mops condylurus Bat, Kenya. Emerg Infect Dis 2019, 25(5)." />
<meta name="citation_reference" content="Karan LS, Makenov MT, Korneev MG, Sacko N, Boumbaly S, Yakovlev SA, Kourouma K, Bayandin RB, Gladysheva AV, Shipovalov AV et al: Bombali Virus in Mops condylurus Bats, Guinea. Emerg Infect Dis 2019, 25(9)." />
<meta name="citation_reference" content="Gire SK, Goba A, Andersen KG, Sealfon RS, Park DJ, Kanneh L, Jalloh S, Momoh M, Fullah M, Dudas G et al: Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak. Science 2014, 345(6202):1369–1372." />
<meta name="citation_reference" content="Mbala-Kingebeni P, Pratt CB, Wiley MR, Diagne MM, Makiala-Mandanda S, Aziza A, Di Paola N, Chitty JA, Diop M, Ayouba A et al: 2018 Ebola virus disease outbreak in Equateur Province, Democratic Republic of the Congo: a retrospective genomic characterisation. Lancet Infect Dis 2019, 19(6):641–647." />
<meta name="citation_reference" content="Katoh K, Standley DM: MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol Biol Evol 2013, 30(4):772–780." />
<meta name="citation_reference" content="Page AJ, Taylor B, Delaney AJ, Soares J, Seemann T, Keane JA, Harris SR: SNP-sites: rapid efficient extraction of SNPs from multi-FASTA alignments. Microb Genom 2016, 2(4):e000056." />
<meta name="citation_reference" content="R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. [https://www.R-project.org/]" />
<meta name="citation_reference" content="Untergasser A, Cutcutache I, Koressaar T, Ye J, Faircloth BC, Remm M, Rozen SG: Primer3--new capabilities and interfaces. Nucleic Acids Res 2012, 40(15):e115." />
<meta name="twitter:title" content="BioLaboro: A bioinformatics system for detecting molecular assay signature erosion and designing new assays in response to emerging and reemerging pathogens" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Background Emerging and reemerging infectious diseases such as the novel Coronavirus disease, COVID-19 and Ebola pose a significant threat to global society and test the public health community’s preparedness to rapidly respond to an outbreak with effective diagnostics and therapeutics. Recent advances in next generation sequencing technologies enable rapid generation of pathogen genome sequence data, within 24 hours of obtaining a sample in some instances. With these data, one can quickly evaluate the effectiveness of existing diagnostics and therapeutics using in silico approaches. The propensity of some viruses to rapidly accumulate mutations can lead to the failure of molecular detection assays creating the need for redesigned or newly designed assays.

Results Here we describe a bioinformatics system named BioLaboro to identify signature regions in a given pathogen genome, design PCR assays targeting those regions, and then test the PCR assays in silico to determine their sensitivity and specificity. We demonstrate BioLaboro with two use cases: Bombali Ebolavirus (BOMV) and the novel Coronavirus 2019 (SARS-CoV-2). For the BOMV, we analyzed 30 currently available real-time reverse transcription-PCR assays against the three available complete genome sequences of BOMV. Only two met our in silico criteria for successful detection and neither had perfect matches to the primer/probe sequences. We designed five new primer sets against BOMV signatures and all had true positive hits to the three BOMV genomes and no false positive hits to any other sequence. Four assays are closely clustered in the nucleoprotein gene and one is located in the glycoprotein gene. Similarly, for the SARS-CoV-2, we designed five highly specific primer sets that hit all 145 whole genomes (available as of February 28, 2020) and none of the near neighbors.

Conclusions Here we applied BioLaboro in two real-world use cases to demonstrate its capability; 1) to identify signature regions, 2) to assess the efficacy of existing PCR assays to detect pathogens as they evolve over time, and 3) to design new assays with perfect in silico detection accuracy, all within hours, for further development and deployment. BioLaboro is designed with a user-friendly graphical user interface for biologists with limited bioinformatics experience.

### Competing Interest Statement



*   bp
    :   base pair (bps, plural base pairs)
    BDBV
    :   Bundibugyo ebolavirus
    BLAST
    :   Basic Local Alignment Search Tool
    BOMV
    :   Bombali ebolavirus
    CFR
    :   Case Fatality Ratio
    COVID-19
    :   Coronavirus Disease 2019
    DNA
    :   deoxyribonucleic acid
    DRC
    :   Democratic Republic of the Congo
    EBOV
    :   Zaire ebolavirus
    EMA
    :   European Medicines Agency
    EVD
    :   Ebola Virus Disease
    FDA
    :   Food and Drug Administration
    FN
    :   False Negative
    FP
    :   False Positive
    GC
    :   guanine cytosine
    GP
    :   glycoprotein
    GUI
    :   Graphic User interface
    ID
    :   identifier
    mAb
    :   monoclonal antibody
    MAFFT
    :   Multiple Sequence Alignment using Fast Fourier Transform
    MCM
    :   medical countermeasure
    N
    :   nucleocapsid phosphoprotein
    N/A
    :   Not available
    NCBI
    :   National Center for Biotechnology Information
    NIAID
    :   National Institute of Allergy and Infectious Diseases
    NP
    :   nucleoprotein
    NPC1
    :   Niemann-Pick C1
    ORF
    :   Open Reading Frame
    PALM
    :   Pamoja Tulinde Maisha study
    PCR
    :   polymerase chain reaction
    PSET
    :   PCR signature erosion tool
    RAM
    :   random access memory
    RESTV
    :   Reston ebolavirus
    RNA
    :   ribonucleic acid
    rRT-PCR
    :   real-time reverse-transcription polymerase chain reaction
    rVSV
    :   recombinant vesicular stomatitis virus
    SARS
    :   severe acute respiratory syndrome
    SNP
    :   single nucleotide protein
    SUDV
    :   Sudan ebolavirus
    TAFV
    :   Taï Forest ebolavirus
    TN
    :   True Negative
    TP
    :   True Positive
    WGS
    :   Whole Genome Sequence
    WHO
    :   World Health Organization" />
<meta name="og-title" property="og:title" content="BioLaboro: A bioinformatics system for detecting molecular assay signature erosion and designing new assays in response to emerging and reemerging pathogens" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.08.031963v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Background Emerging and reemerging infectious diseases such as the novel Coronavirus disease, COVID-19 and Ebola pose a significant threat to global society and test the public health community’s preparedness to rapidly respond to an outbreak with effective diagnostics and therapeutics. Recent advances in next generation sequencing technologies enable rapid generation of pathogen genome sequence data, within 24 hours of obtaining a sample in some instances. With these data, one can quickly evaluate the effectiveness of existing diagnostics and therapeutics using in silico approaches. The propensity of some viruses to rapidly accumulate mutations can lead to the failure of molecular detection assays creating the need for redesigned or newly designed assays.

Results Here we describe a bioinformatics system named BioLaboro to identify signature regions in a given pathogen genome, design PCR assays targeting those regions, and then test the PCR assays in silico to determine their sensitivity and specificity. We demonstrate BioLaboro with two use cases: Bombali Ebolavirus (BOMV) and the novel Coronavirus 2019 (SARS-CoV-2). For the BOMV, we analyzed 30 currently available real-time reverse transcription-PCR assays against the three available complete genome sequences of BOMV. Only two met our in silico criteria for successful detection and neither had perfect matches to the primer/probe sequences. We designed five new primer sets against BOMV signatures and all had true positive hits to the three BOMV genomes and no false positive hits to any other sequence. Four assays are closely clustered in the nucleoprotein gene and one is located in the glycoprotein gene. Similarly, for the SARS-CoV-2, we designed five highly specific primer sets that hit all 145 whole genomes (available as of February 28, 2020) and none of the near neighbors.

Conclusions Here we applied BioLaboro in two real-world use cases to demonstrate its capability; 1) to identify signature regions, 2) to assess the efficacy of existing PCR assays to detect pathogens as they evolve over time, and 3) to design new assays with perfect in silico detection accuracy, all within hours, for further development and deployment. BioLaboro is designed with a user-friendly graphical user interface for biologists with limited bioinformatics experience.

### Competing Interest Statement



*   bp
    :   base pair (bps, plural base pairs)
    BDBV
    :   Bundibugyo ebolavirus
    BLAST
    :   Basic Local Alignment Search Tool
    BOMV
    :   Bombali ebolavirus
    CFR
    :   Case Fatality Ratio
    COVID-19
    :   Coronavirus Disease 2019
    DNA
    :   deoxyribonucleic acid
    DRC
    :   Democratic Republic of the Congo
    EBOV
    :   Zaire ebolavirus
    EMA
    :   European Medicines Agency
    EVD
    :   Ebola Virus Disease
    FDA
    :   Food and Drug Administration
    FN
    :   False Negative
    FP
    :   False Positive
    GC
    :   guanine cytosine
    GP
    :   glycoprotein
    GUI
    :   Graphic User interface
    ID
    :   identifier
    mAb
    :   monoclonal antibody
    MAFFT
    :   Multiple Sequence Alignment using Fast Fourier Transform
    MCM
    :   medical countermeasure
    N
    :   nucleocapsid phosphoprotein
    N/A
    :   Not available
    NCBI
    :   National Center for Biotechnology Information
    NIAID
    :   National Institute of Allergy and Infectious Diseases
    NP
    :   nucleoprotein
    NPC1
    :   Niemann-Pick C1
    ORF
    :   Open Reading Frame
    PALM
    :   Pamoja Tulinde Maisha study
    PCR
    :   polymerase chain reaction
    PSET
    :   PCR signature erosion tool
    RAM
    :   random access memory
    RESTV
    :   Reston ebolavirus
    RNA
    :   ribonucleic acid
    rRT-PCR
    :   real-time reverse-transcription polymerase chain reaction
    rVSV
    :   recombinant vesicular stomatitis virus
    SARS
    :   severe acute respiratory syndrome
    SNP
    :   single nucleotide protein
    SUDV
    :   Sudan ebolavirus
    TAFV
    :   Taï Forest ebolavirus
    TN
    :   True Negative
    TP
    :   True Positive
    WGS
    :   Whole Genome Sequence
    WHO
    :   World Health Organization" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-10" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.04.08.031963v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.04.08.031963v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.04.08.031963v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.04.08.031963v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1238478" />
    <title>BioLaboro: A bioinformatics system for detecting molecular assay signature erosion and designing new assays in response to emerging and reemerging pathogens | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"niyDg1jyxcdWGgutUcd8eUeQs6dnFyD1gcwvxurpaLE","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1238478","apath":"\/biorxiv\/early\/2020\/04\/10\/2020.04.08.031963.atom","pisa":"biorxiv;2020.04.08.031963v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.04.08.031963v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.04.08.031963v1","title":"BioLaboro: A bioinformatics system for detecting molecular assay signature erosion and designing new assays in response to emerging and reemerging pathogens","identifier":"node\/1238478"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002251zEhwaBRh4UUEOW\\\\\\\/xSeDAQeUsK80JA4bs5VSXdEHqUJewSP0WxkXJopH2QG9Ams7Z9APImHpwlXVRByLpIpsCAzAwdfPqVhji3y01XvGRMOUzA6mkWWrhM9YSP0GQ3pLRm\\\\\\\/JKUUoFDS5l+fk1NNw8Mu9n3TwOL5XXjz50HEKmYSS0m2\\\\\\\/wh+qQ8fzsG2aTK5ASsATDhEqPLuH2go3xxFS84PdEgfBfRYcIoi5Iv7gtJi6GznjWzLlnng3Zx6\\\\\\\/TCVVs7gYwVuep4urajrv50kiCG+hDO1E\\\\\\\/I1HDhPh\\\\\\\/2QCA2gMj1weZvOaRZoNpVoQxk8e0nvy3qzJ7NGBhSuDNfg+sGe1CgbXzvEkbK7kQA11Yc5mevWm3+cKKXh2LDGxCMvZYILHFOV7Rxj\\\\\\\/1LB05CnDeZSU\\\\\\\/9hxBM87VAzG6\\\\\\\/AXUXkrNiUjfsfRfjF3Co+eeF6m5xIHtpaNcDUOSFQEIN4MsjkqmrJxP0y+I8ySwARnXA1XvF4O4uIQg3mLWNN4LQlItjyKI5EDY+GJ9tkQnIgKQ3K1Fx+Px5F47GkyhSZoLZMI4MQRtZdtItQ6FgHHyODqcvkt\\\\\\\/FMIobI05X2EWsIkR3cd4iKqR9bEFyqsZY83FwEgAjIQbRTxAi1AcPBZGJ6qZwnjKXi3hHqIFwjrXYHvYlQGZvOlR+3csvQAhP7jMXVy9bI\\\\\\\/P6QzMGESPbSxDEIq5RYtX1oDB84smnXB5VaGhxg0naaI\\\\\\\/PzW4j6zIyMZ3QL75j7xKUbmBMbjiXsjZcIpoc9jNvnr4AAKseDR6hINOpvZxf2GOSurzjhfSHiwML2pkRL4UkRJVHnqKOYxIdJQq1cxNR1NqC110mb7ZahRbZU2EBb0\\\\\\\/NRzEfaAJAnnUD8nvqZYzHv3m9DG6E4VEOOvDRC5VtpjDcTxF8dfQDlXaEtcZql\\\\\\\/OgX\\\\\\\/xr6+6n+QjBdn0bO3g2GJzImGIPDW8TuThXXCuI9NnwyEnQOnr8vhTNJAqQovW\\\\\\\/P9mlgjgre9SLYM88VHY5VIBHIuKt9aIiQehHyluEF0vZ0\\\\\\\/46FXCNuFLJryQ2kZOE1\\\\\\\/Vr8ddhH2kbUNpW\\\\\\\/mrembDP2gxBthnP61FkeWF9h3gFtoFFymFfxcNtUC6kToYXG+MnCm2sViBNSwkPutuaLYo+U2i\\\\\\\/h2Iu+uExUxiRdDD3WOu0nN95b2z5rGtgyFaoJgaqgqiw7OWdYUDqRUq1UGOyz9hl0GV\\\\\\\/y8fHG5EOFtOH1CzBZ\\\\\\\/2swjMMho7dKWj4S1MpiPLWzZEkVedaGg5mhB8qt4FRTMBafNWMyOORjXwLf5mV6mprkyNcWfwzxAOGzO4OWxzwOG96f77myADeuHoFjotSQdiKD\\\u0022,\\\u0022iv\\\u0022:\\\u0022hjFgHQ0XaSY3LsFuNivIKQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00228695ba6ae52260171b3c821e35d742e290e22f058e7d20546c3ad64ef6122daf\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.04.08.031963v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1238478 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.04.08.031963v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IgRyDUpDMp6QE5gf4Q4h1Z3fsP8hkZ1JPi2bpxHdUXQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1238478" id="node1238478" data-pisa="biorxiv;2020.04.08.031963v1" data-pisa-master="biorxiv;2020.04.08.031963" data-apath="/biorxiv/early/2020/04/10/2020.04.08.031963.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">BioLaboro: A bioinformatics system for detecting molecular assay signature erosion and designing new assays in response to emerging and reemerging pathogens</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Mitchell</span> <span class="nlm-surname">Holland</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Negrón</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Shane</span> <span class="nlm-surname">Mitchell</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Nate</span> <span class="nlm-surname">Dellinger</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Mychal</span> <span class="nlm-surname">Ivancich</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Tyler</span> <span class="nlm-surname">Barrus</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Sterling</span> <span class="nlm-surname">Thomas</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Katharine W.</span> <span class="nlm-surname">Jennings</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Bruce</span> <span class="nlm-surname">Goodwin</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-6762-7874" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Shanmuga</span> <span class="nlm-surname">Sozhamannan</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.04.08.031963 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1238478" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mitchell Holland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Noblis</span>, Reston, VA, 20191, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mitchell%2BHolland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holland%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMitchell%2BHolland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel Negrón </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Noblis</span>, Reston, VA, 20191, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BNegr%C3%B3n%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Negr%C3%B3n%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BNegr%25C3%25B3n%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shane Mitchell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Noblis</span>, Reston, VA, 20191, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shane%2BMitchell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mitchell%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShane%2BMitchell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nate Dellinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Noblis</span>, Reston, VA, 20191, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nate%2BDellinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dellinger%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANate%2BDellinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Mychal Ivancich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Noblis</span>, Reston, VA, 20191, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mychal%2BIvancich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ivancich%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMychal%2BIvancich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Tyler Barrus </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Noblis</span>, Reston, VA, 20191, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tyler%2BBarrus%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barrus%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATyler%2BBarrus%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Sterling Thomas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Noblis</span>, Reston, VA, 20191, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sterling%2BThomas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thomas%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASterling%2BThomas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Katharine W. Jennings </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Noblis</span>, Reston, VA, 20191, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katharine%2BW.%2BJennings%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jennings%20KW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatharine%2BW.%2BJennings%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Bruce Goodwin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Defense Biological Product Assurance Office (DBPAO), Joint Program Executive Office for Chemical, Biological, Radiological and Nuclear Defense (JPEO-CBRND), Enabling Biotechnologies</span>, Frederick, MD 21702, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruce%2BGoodwin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goodwin%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABruce%2BGoodwin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Shanmuga Sozhamannan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Defense Biological Product Assurance Office (DBPAO), Joint Program Executive Office for Chemical, Biological, Radiological and Nuclear Defense (JPEO-CBRND), Enabling Biotechnologies</span>, Frederick, MD 21702, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Logistics Management Institute</span>, Tysons, VA 22102, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shanmuga%2BSozhamannan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sozhamannan%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShanmuga%2BSozhamannan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6762-7874" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Shanmuga Sozhamannan</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Shanmuga.Sozhamannan.ctr@mail.mil" class="" data-icon-position="" data-hide-link-title="0">Shanmuga.Sozhamannan.ctr@mail.mil</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.04.08.031963v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1238478" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1238478/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.08.031963v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1238478" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1238478/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.08.031963v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1238478" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1238478/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.08.031963v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1238478" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1238478/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.08.031963v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1238478" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1238478/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.04.08.031963v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1238478" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1238478/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><div id="sec-1" class="subsection"><p id="p-2"><strong>Background</strong> Emerging and reemerging infectious diseases such as the novel Coronavirus disease, COVID-19 and Ebola pose a significant threat to global society and test the public health community’s preparedness to rapidly respond to an outbreak with effective diagnostics and therapeutics. Recent advances in next generation sequencing technologies enable rapid generation of pathogen genome sequence data, within 24 hours of obtaining a sample in some instances. With these data, one can quickly evaluate the effectiveness of existing diagnostics and therapeutics using <em>in silico</em> approaches. The propensity of some viruses to rapidly accumulate mutations can lead to the failure of molecular detection assays creating the need for redesigned or newly designed assays.</p></div><div id="sec-2" class="subsection"><p id="p-3"><strong>Results</strong> Here we describe a bioinformatics system named BioLaboro to identify signature regions in a given pathogen genome, design PCR assays targeting those regions, and then test the PCR assays <em>in silico</em> to determine their sensitivity and specificity. We demonstrate BioLaboro with two use cases: Bombali Ebolavirus (BOMV) and the novel Coronavirus 2019 (SARS-CoV-2). For the BOMV, we analyzed 30 currently available real-time reverse transcription-PCR assays against the three available complete genome sequences of BOMV. Only two met our <em>in silico</em> criteria for successful detection and neither had perfect matches to the primer/probe sequences. We designed five new primer sets against BOMV signatures and all had true positive hits to the three BOMV genomes and no false positive hits to any other sequence. Four assays are closely clustered in the nucleoprotein gene and one is located in the glycoprotein gene. Similarly, for the SARS-CoV-2, we designed five highly specific primer sets that hit all 145 whole genomes (available as of February 28, 2020) and none of the near neighbors.</p></div><div id="sec-3" class="subsection"><p id="p-4"><strong>Conclusions</strong> Here we applied BioLaboro in two real-world use cases to demonstrate its capability; 1) to identify signature regions, 2) to assess the efficacy of existing PCR assays to detect pathogens as they evolve over time, and 3) to design new assays with perfect <em>in silico</em> detection accuracy, all within hours, for further development and deployment. BioLaboro is designed with a user-friendly graphical user interface for biologists with limited bioinformatics experience.</p></div></div><div class="section" id="sec-4"><h2 class="">Background</h2><p id="p-6">Emerging and reemerging infectious diseases have serious adverse impacts on society with respect to lives lost and annual economic losses [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>], as being witnessed currently in the COVID-19 outbreak caused by a novel Coronavirus, SARS-CoV-2. As of March 9, 2020, there have been 113,584 confirmed cases of COVID-19 around the globe and 3,996 deaths [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>-<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. Factors such as climate change, urbanization, zoonotic spillover, international travel, lack or breakdown of public health care systems, natural or man-made disasters, and pathogen evolution to name a few, contribute to infectious disease emergence and sustainment [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>].</p><p id="p-7">A unique challenge associated with emerging infectious diseases is rapidly identifying the etiological agent and developing or repurposing existing medical countermeasures (MCMs), including diagnostic assays, to curtail the spread of an outbreak. With respect to reemerging infectious diseases the challenge is to determine whether existing MCMs are effective or not, and in the latter case, develop or modify the MCMs in a timely manner. In some cases, the failure of existing MCMs can be accounted for by genetic drift and shift and the resulting altered genotypic and phenotypic profiles of the newly emergent pathogen. Ebolavirus is one such reemerging infectious disease agent that exhibits high degrees of genetic changes in every new outbreak.</p><p id="p-8">Ebola virus disease (EVD) is one of the deadliest infectious diseases that continues to plague Central and Western Africa. It was discovered in 1976 when two consecutive outbreaks of fatal hemorrhagic fever occurred in different parts of Central Africa [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. The first outbreak occurred in the Democratic Republic of the Congo (DRC, formerly Zaire) in a village near the Ebola River, which gave the virus its name [<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>]. Since 1976 more than 25 outbreaks have been recorded. The average case fatality ratio (CFR) for EVD is around 50% and varies from 25% to 90% in past outbreaks [<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">9</a>]. From 2014 to 2016, the world witnessed the worst-yet EVD outbreak, which originated in Western Africa. That outbreak started in a rural setting of southeastern Guinea, and quickly spread like a wildfire to urban areas and across borders to the neighboring countries of Liberia and Sierra Leone [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. In the end, the outbreak infected 28,616 people and killed 11,310 of the victims (CFR of approximately 39.5%) [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. In the current Democratic Republic of the Congo outbreak that began in August 2018, as of March 4, 2020, a total of 3,444 cases, total deaths 2,264 and 1,169 survivors have been reported [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>].</p><p id="p-9">Until recently, there was no approved vaccine or treatment for EVD although four experimental therapeutics, (Regeneron’s monoclonal antibody REGN-EB3, mAb-114 [Ridgeback Biotherapeutics], Remdesivir GS-5734 [Gilead] and ZMapp [MappBio Pharmaceutical] underwent clinical trials (the Pamoja Tulinde Maisha [PALM] study) during the current DRC outbreak [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. On October 17, 2019, the Committee for Medicinal Products for Human Use granted a “conditional marketing authorization” for the Ebola Zaire vaccine ERVEBO by Merck Sharp and Dohme [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. ERVEBO was reviewed under the European Medicines Agency’s (EMA) accelerated assessment program. ERVEBO (v290) (Ebola Zaire Vaccine (rVSVΔG-ZEBOV-GP, live) is a genetically engineered, replication competent, attenuated live vaccine that received approval from the U. S. Food and Drug Administration (FDA) on December 19, 2019 [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>].</p><p id="p-10">Ebolaviruses are one of three genera of filoviruses belonging to the Family <em>Filoviridae,</em> Cuevavirus and Marburgvirus being the other two. Filoviruses are non-segmented, negative-sense, single-stranded RNA viruses. Six species of Ebolaviruses have been described to date, Zaire (EBOV), Bundibugyo (BDBV), Sudan (SUDV), Taï Forest (TAFV), Reston (RESTV) and Bombali (BOMV). The prototypic viruses - EBOV, BDBV, SUDV and TAFV - have been associated with disease in humans [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>]. The RESTV causes disease in nonhuman primates and pigs [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]. In a 2018 wildlife survey, BOMV RNA was recovered from insectivorous bats, but there are no known cases of human or animal disease caused by this species [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>].</p><p id="p-11">Research on EVD focuses on finding the virus’ natural reservoirs and hosts from which spill over occurs, developing preventive measures such as vaccines to protect at-risk populations, and discovering therapies to improve treatment of infections. Biosurveillance data suggested that the reservoirs of ebolaviruses may be fruit and insect eating bats [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>-<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. The new BOMV was identified in a biosurveillance project conducted in Sierra Leone to identify hosts of EBOV as well as any additional filoviruses that might be circulating in wildlife [<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>]. Oral and rectal swabs were collected from 535 animals (244 bats, 46 rodents, 240 dogs, 5 cats) from 20 locations in Sierra Leone in 2016. Of the 1,278 samples, five samples from three Little free-tailed bats and one Angolan free-tailed bat contained ebolavirus sequences. Two full genome sequences were assembled from two of the samples and they showed nucleotide identity of 55–59% and amino acid identity of 64–72% to other ebolaviruses [<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20</a>]. Based on phylogenetic analyses of sequence data, it was determined that the genome was sufficiently distinct to represent the prototypic strain of a new species, Bombali Ebolavirus (BOMV) [<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">20</a>].</p><p id="p-12">I<em>n vitro</em> studies of the BOMV demonstrated that a recombinant vesicular stomatitis virus (rVSV) encoding the BOMV glycoprotein (GP) gene mediated virus entry into human host cells |20|. Entry and infection of rVSV–BOMV GP was also completely dependent on Niemann-Pick C1 (NPC1) protein, providing additional evidence that this is a universal receptor for filoviruses. Although not conclusive, these data indicated the potential for BOMV to infect humans. Given the high divergence from other Ebolaviruses, we examined whether existing real-time reverse-transcription PCR (rRT-PCR) assays for Ebolaviruses would detect the new BOMV.</p><p id="p-13">Potential determinants of Ebolavirus pathogenicity in humans were identified by analyzing the differentially conserved amino acid positions called specificity determining positions (SDPs) between human pathogenic ebolaviruses and the non-pathogenic Reston virus [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. Recently, this study was extended to include BOMV to assess its pathogenicity to humans [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>]. At SDPs, BOMV shared the majority of amino acids (63.25%) with the human pathogenic Ebolaviruses. However, for two SDPs in viral protein 24 (VP24), which may be critical for the lack of Reston virus human pathogenicity, the BOMV amino acids match those of Reston virus. Thus, BOMV may not be pathogenic in humans [<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>]. Nonetheless, rRT-PCR assays are important for biosurveillance of BOMV and other potential new variants in the wild.</p><p id="p-14">While our study on the BOMV use case for assay design was ongoing, the world witnessed the emergence of a novel respiratory pneumonia disease (COVID-19) epidemic from Wuhan city, Hubei province, China. In the aftermath of its rapid global spread and devastating impact, on March 11, 2020, the WHO declared that COVID-19 can be characterized a pandemic threat [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. COVID-19 was determined to be caused by a severe acute respiratory syndrome (SARS)-like corona virus (SARS-CoV-2) that quickly spread within Wuhan [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>] and crossed borders to &gt;104 countries/territories/areas as of March 09, 2020 [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>].</p><p id="p-15">Using next generation sequencing, the whole genome sequence (WGS) of SARS-CoV-2 are continuously being released and shared (306 complete genomes as of March 09, 2020) with the entire research community through Global Initiative on Sharing All Influenza Data (GISAID) [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]. The release of WGS allowed us to test the BioLaboro pipeline (described in this study) to evaluate currently used diagnostic assays and to rapidly design new assays.</p><p id="p-16">In a previous study, we described a bioinformatics tool called PSET (PCR signature erosion tool) and used it to show <em>in silico</em>, confirmed with wet lab work, the effectiveness of existing Ebolavirus diagnostic assays against a large number of sequences available at that time [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>]. The phrase “signature erosion” used here signifies potential false-positive or false-negative results in PCR assays due to mutations in the primers, probe, or amplicon target sequences (PCR signatures). Signature erosion could also mean failure of medical countermeasures; for example, a change in the genomic sequence resulting in an amino acid change that could potentially alter the efficacy of sequence-based therapeutics [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>].</p><p id="p-17">In this study, we describe an expanded bioinformatics pipeline called BioLaboro in which we have integrated several tools: BioVelocity®, Primer3 and PSET for end-to-end analysis of outbreak pathogen genome sequences to evaluate existing PCR assay efficacy against the new sequences, and to identify unique signature regions (BioVelocity), design PCR assays to these regions (Primer3), and test the new assays’ efficacy (PSET) against current National Center for Biotechnology Information Basic Local Alignment Search Tool (BLAST) standard databases [<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>]. We have used the BOMV and SARS-CoV-2 sequences as proof of concept for BioLaboro to develop and evaluate new PCR assays <em>in silico.</em></p></div><div class="section" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>BioLaboro architecture</h3><p id="p-18">BioLaboro is comprised of three algorithms - BioVelocity, Primer3, and PSET - which are built into a pipeline for <em>user-friendly</em> applications. The user has the option to launch one of four different job types: Signature Discovery, Score Assay Targets, Validate Assay, or New Assay Discovery. Each of the three algorithms can be run individually or together as a complete end-to-end pipeline (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). For the BOMV use case, in the first phase of the pipeline BioVelocity was used to analyze a set of genome sequences for unique regions that are both conserved and signature to the target sequences selected. This was achieved by splitting a chosen representative whole genome sequence into sliding 50 base pairs (bps) k-mers. Each k-mer was then scanned against all target sequences to determine conservation. Conserved k-mers were then elongated based on overlaps and formed into contigs. These contigs were then split into k-mers ≤ 250 bps and scanned against all non-target sequences to determine specificity. All passing sequences were then elongated based on overlaps and the signature contigs were passed to the next step in the pipeline. Primer3 was then used to evaluate the signature contigs to identify suitable primers and probes for assay development. Primer3 was run in parallel against all signatures and the output was ranked by penalty score in ascending order. The top five best primer sets were passed along to the final step in the pipeline, PSET. In this step the primer sets were run through a bioinformatics pipeline which aligned the sequences against large public sequence databases from NCBI using BLAST and GLSEARCH [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>] to determine how well each assay correctly aligned to all target sequences while excluding off-target hits.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="BioLaboro- New Assay Discovery pipeline. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-888010457" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;BioLaboro- New Assay Discovery pipeline.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F1.medium.gif" width="440" height="337"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F1.medium.gif" width="440" height="337"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">BioLaboro- New Assay Discovery pipeline.</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Performance assessment of currently available EBOV PCR assays on BOMV using PSET</h3><p id="p-19">The phylogenetic classification criteria for filoviruses were reported to be 64–77% similarity for species and 41–50% for genera, based on BLAST alignment results [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>]. Compared to other ebolaviruses, the BOMV showed 55–59% nucleotide identity, 64–72% amino acid identity [<a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">20</a>]. Given the sequence diversity, we assessed if the current diagnostic assays were effective against BOMV. We compared the current Ebola assay signatures against the three available complete genome sequences of BOMV (NCBI Accession numbers: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MF319185.1&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom">MF319185.1</a>, MF319186.1, and MK340750.1). Of the 30 Ebola assays examined (assay details in reference [<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a>]), only two met our <em>in silico</em> criteria for successful detection (90% identity over 90% of the length for all primers and probe sequences in the amplicon). Of the two assays that did meet detection criteria for BOMV, neither had perfect matches to the primer sequences. The primer set match percentages (the average identity of all primers/probes) for all 30 Ebola assays against the three BOMV complete genomes are shown in the heat map (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The two assays which pass the <em>in silico</em> criteria are shown in green while the matches in red indicate failure or no alignment. Even the two assays that passed <em>in silico</em> criteria did not have perfect matches raising the possibility that these assays may fail in wet lab testing due to mismatches against currently available BOMV genomic sequences. Hence, as described below, we designed new assays using the BioLaboro platform.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1243258/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1243258/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1243258" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <span class="caption-title">Heat map of PSET results of Ebola assays against BOMV sequences</span><p id="p-20" class="first-child">Table 1 Legend: The average primer set percent identity matches for thirty current Ebola assays tested against the three BOMV genomes. Cells in green have an average primer identity over 90% which indicates likely primer binding while red cells have below 90% average identity or no alignment at all, indicating likely primer binding failure.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Discovery of potential new BOMV assays using BioLaboro end-to-end pipeline</h3><p id="p-21">Using BioLaboro we ran a New Assay Discovery job to discover new BOMV signatures and determine their potential for accurate detection using PSET. In the first phase, BioVelocity was used to search for conserved and signature regions within the selected genomes. We selected the organism of interest by searching for “Bombali ebolavirus” from the database and selecting the three available complete genomes. The MF319185.1 genome was used as the algorithmic reference sequence as it is the same one that NCBI selected for the RefSeq database (Genbank ID: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_039345.1&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom">NC_039345.1</a>) . The algorithmic reference sequence was first split into k-mers of 50 bps each using a sliding window of 1 bp, which amounts to 18,994 k-mers to be evaluated with BioVelocity’s conserved sequence detection algorithm. BioVelocity found 27% (5,237) of these k-mers to be conserved in all three of the BOMV genomes. The conserved k-mers were then evaluated to determine overlapping segments and were combined into 120 conserved contigs. These contigs were next evaluated with BioVelocity’s signature sequence detection algorithm. The contigs were split into signatures with a max size of 250 bps (longer contigs were split into 250 k-mers with a step size of 1). The conserved k-mer sequences were evaluated against 563,843 complete genomes and plasmids from the NCBI GenBank repository. There were 291 k-mers sequences found to be signatures to BOMV. The signatures were then evaluated to determine overlapping reads and combined back into 119 signature contigs. Metrics for the BioVelocity run in phase one are shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1243257/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1243257/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1243257" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <span class="caption-title">BioVelocity run statistics for BOMV.</span><p id="p-22" class="first-child">Table 2 Legend. Total number of conserved and signature contigs along with total bases, average sequence length, median sequence length, and percent coverage of the BOMV genome</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">In the second phase, Primer3 was used to identify potential primer pairs and probes for generating new PCR detection assays. For the BioLaboro pipeline, Primer3 is used to assess the suitability of a sequence to primer creation and generates viable primers which cover the identified signature regions. There were 151 primer sets created from the signatures run through Primer3 and assigned a penalty score to facilitate comparison of the results. The top five primer sets, by lowest penalty score (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>), were formatted and sent to the final step for validation using PSET.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1243262/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1243262/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1243262" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3.</span> <span class="caption-title">Primer3 identified PCR assays for BOMV.</span><p id="p-24" class="first-child">Table 3 Legend. The five new assays identified by Primer3 ranked by lowest penalty score. The Identifier column is an automated ID generated from the pipeline, the Targets column is the Taxonomy ID for BOMV, the Definition column contains the amplicon sequence with the primers in brackets (orange) and the probe in parentheses (blue), and the Penalty Points column contains the score generated after taking into account primer design parameters.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">The signature regions identified by BioVelocity and the five new assays selected by Primer3 were mapped to the BOMV genome (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). As shown, the four assays are clustered in the nucleoprotein (NP) gene and one is in the glycoprotein (GP) gene. The signature segments (blue) indicate that there are many potential assay regions throughout the genome.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linear map of BOMV genome. Figure 2 Legend: Linear map of the BOMV genome with annotations. The BOMV genes are colored in green, the signature segments are colored in blue, and the new assays are colored in red. The figure was created using the DNA Features Viewer Python library [38]." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-888010457" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Linear map of BOMV genome.&lt;/span&gt; Figure 2 Legend: Linear map of the BOMV genome with annotations. The BOMV genes are colored in green, the signature segments are colored in blue, and the new assays are colored in red. The figure was created using the DNA Features Viewer Python library [38].&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F2.medium.gif" width="440" height="108"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F2.medium.gif" width="440" height="108"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Linear map of BOMV genome.</span><p id="p-26" class="first-child">Figure 2 Legend: Linear map of the BOMV genome with annotations. The BOMV genes are colored in green, the signature segments are colored in blue, and the new assays are colored in red. The figure was created using the DNA Features Viewer Python library [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>].</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">In the third phase, PSET was used to test the five newly designed assays identified by Primer3 <em>in silico</em> against publicly available sequences. PSET was used to analyze the primers and probes in the new assays using bioinformatics tools to identify potential false-positive and false-negative matches to NCBIs BLAST nucleotide sequence repositories (nt, gss, and env_nt) comprised of over 220 million sequences (as of last update in September 2019). BLAST+ was used to compare the assay amplicon sequences against these sequence repositories to identify matches. These matches were then used to create a custom library of sequences for GLSEARCH, a global-local sequence comparison tool in the FASTA suite of programs, which was used to search for the individual primers and probes. The resulting output was then processed and filtered based on pre-defined hit acceptance criteria. These criteria require that the assay components all hit to 90% identity over 90% of the component length, primer pairs were on opposite strands, and the total amplicon size was no greater than 1000 bps. The results were then validated by comparing the hits to the target NCBI Taxonomy identifier (ID), and true and false matches were reported. PSET results confirmed that the top five primer sets had true positive hits to all three BOMV genomes and no false positive hits to any other organism (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1243254/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1243254/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1243254" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4.</span> <span class="caption-title">PSET results of BOMV assays</span><p id="p-28" class="first-child">Table 4 Legend: True positive (TP: All assay components hit with &gt;=90% identity over &gt;=90% of the component length to the correct target), true negative (TN: Partial hit to assay amplicon but one or more assay components hit with &lt;90% alignment to an incorrect target), false positive (FP: All assay components hit with &gt;=90% identity over &gt;=90% of the component length to an incorrect target), and false negative (FN: Partial hit to assay amplicon but one or more assay components hit with &lt;90% alignment to the correct target) counts for each of the five new BOMV assays tested. The targets column is the NCBI Taxonomy ID of the target sequence, BOMV.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">There are a high number of true negative (TN) results for 4 of 5 assays due to the similarity of the amplicon sequences with Zaire ebolavirus although these TNs are not expected to produce PCR positive results in wet lab experiments.</p></div><div id="sec-9" class="subsection"><h3>Discovery of potential SARS-CoV-2 assays using BioLaboro end-to-end pipeline</h3><p id="p-30">Using BioLaboro, we ran a New Assay Discovery job to discover SARS-CoV-2 signatures and determine their potential for true positive viral detection. In the first phase, BioVelocity was used to search for conserved and signature regions within the selected genomes. Ninety six complete SARS-CoV-2 whole genome sequences were downloaded from the GISAID and uploaded to BioLaboro as a custom database. These 96 genomes were used as our reference set and EPI_ISL_404253 (Genbank ID: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN988713.1&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom">MN988713.1</a>)  was used as the algorithmic reference sequence. The algorithmic reference sequence was first split into k-mers of 50 bps each using a sliding window of 1 bp, which amounted to 29,833 k-mers to be evaluated with the conserved sequence detection algorithm. BioVelocity found 79% (23,542) of these k-mers to be conserved in all 96 of the SARS-CoV-2 genomes. The conserved k-mers were then evaluated to determine overlapping segments and were combined into 96 conserved contigs. These contigs were next evaluated with BioVelocity’s signature sequence detection algorithm. The contigs were split into signatures with a max size of 250 bps (longer contigs were split into 250 k-mers with a step size of 1). The conserved k-mer sequences were evaluated against 563,941 complete genomes and plasmids from the NCBI GenBank repository. There were 11,152 sequences found to be signature to SARS-CoV-2. The signatures were then evaluated to determine overlapping reads and combined back into 91 signature contigs. Metrics for the BioVelocity run in phase one are shown in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1243246/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1243246/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1243246" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5.</span> <span class="caption-title">BioVelocity run statistics for SARS-CoV-2.</span><p id="p-31" class="first-child">Table 5 Legend: Total number of conserved and signature segments along with total bases, average sequence length, median sequence length, and percent coverage of the SARS-CoV-2 genome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">In the second phase, Primer3 was used to identify potential primer pairs and probes for generating new PCR detection assays as described above for BOMV. There were 330 primer sets created from the signatures which were assigned a penalty score to facilitate comparison of the results. Primer sets were sorted by lowest penalty score and five potential assays were chosen manually in order to distribute potential candidates across the genome. These assays were then formatted and sent to the final step for validation using PSET. Primer sets sent to PSET are shown in <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1243244/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1243244/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1243244" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6.</span> <span class="caption-title">Primer3 identified assays for SARS-CoV-2.</span><p id="p-33" class="first-child">Table 6 Legend: The five new assays identified by Primer3 ranked by lowest penalty score. The Identifier column is an automated ID generated from the pipeline, the Targets column is the Taxonomy ID for SARS-CoV-2, the Definition column contains the amplicon sequence with the primers in brackets (orange) and the probe in parentheses (blue), and the Penalty Points column contains the score generated after taking into account primer design parameters.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">The five new assays were mapped to the SARS-CoV-2 genome presented below (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). This figure shows that four assays are in the ORF1ab and one is located in the spike (S) gene. By comparison, the CDC and Corman group assays are clustered primarily at the 3’ end of the genome in the envelope (E) and nucleocapsid phosphoprotein (N) genes.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linear map of SARS-CoV-2. Figure 3 Legend. A linear map of the SARS-CoV-2 genome with annotations. The genes and open reading frames (ORFs) are colored in green and the new assays are colored in red, Corman assays (blue), CDC assays (purple), and assays discovered in this study (red). The figure was created using the DNA Features Viewer Python library [36]." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-888010457" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Linear map of SARS-CoV-2.&lt;/span&gt; Figure 3 Legend. A linear map of the SARS-CoV-2 genome with annotations. The genes and open reading frames (ORFs) are colored in green and the new assays are colored in red, Corman assays (blue), CDC assays (purple), and assays discovered in this study (red). The figure was created using the DNA Features Viewer Python library [36].&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F3.medium.gif" width="440" height="91"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F3.medium.gif" width="440" height="91"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Linear map of SARS-CoV-2.</span><p id="p-35" class="first-child">Figure 3 Legend. A linear map of the SARS-CoV-2 genome with annotations. The genes and open reading frames (ORFs) are colored in green and the new assays are colored in red, Corman assays (blue), CDC assays (purple), and assays discovered in this study (red). The figure was created using the DNA Features Viewer Python library [<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>].</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">In the third phase, PSET was used to test the five newly designed assays identified by Primer3 <em>in silico</em> against publicly available sequences as described above for BOMV signatures. The results were then validated by comparing the hits to the target NCBI Taxonomy identifier (ID), and true and false matches were reported. PSET confirmed that the top five primer sets had true positive hits to all 145 (SARS-CoV-2) genomes (as of February 28, 2020) and no false positive hits to any other organism. An additional table lists the identifiers and metadata for these 145 complete genome sequences generated from human samples and not from other sources such as bat or pangolin [see Additional Table 2]. Results are shown alongside assays from CDC and Corman et al. [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>] (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>).</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1243259/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1243259/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1243259" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7.</span> <span class="caption-title">PSET results of SARS-CoV-2 PCR assays.</span><p id="p-37" class="first-child">Table 7 Legend: True positive (TP: All assay components hit with &gt;=90% identity over &gt;=90% of the component length to the correct target), true negative (TN: Partial hit to assay amplicon but one or more assay components hit with &lt;90% alignment to an incorrect target), false positive (FP: All assay components hit with &gt;=90% identity over &gt;=90% of the component length to an incorrect target), and false negative (FN: Partial hit to assay amplicon but one or more assay components hit with &lt;90% alignment to the correct target) counts for each of the five new SARS-CoV-2 assays tested. The targets column is the NCBI Taxonomy ID of the target sequence, SARS-CoV-2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">There are a high number of true negative (TN) results for 4 of 5 assays due to the similarity of the amplicon sequences with SARS coronavirus near neighbors. The FP results from some Corman and CDC assays are due to near neighbor hits since these assays are pan assays. The one FN identified for corman_e_gene is to sequence EPI_ISL_410486 (Additional Table 2) which contains a large stretch of Ns over the majority of the amplicon sequence (80%) which is likely due to missing sequences.</p><p id="p-39">We also tested the SARS-CoV-2 assays using PSET on near neighbor sequences that were generated during this outbreak, such as the bat and pangolin sequences. As expected the analyses showed a range of TP from pan assays, FN results due to sequence divergence (<a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a>).</p><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1243255/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1243255/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1243255" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8.</span> <span class="caption-title">PSET results of SARS-CoV-2 PCR assays against bat and pangolin SARS-CoV sequences.</span><p id="p-40" class="first-child">Table 8 Legend: TP, TN, FP and FN definitions are similar to <a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table 7</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Discussion</h3><p id="p-41">Since its discovery in 2016, BOMV RNA has been detected in oral and rectal swabs as well as internal organs of <em>Mops condylurus</em> and <em>Chaerephon pumilus</em> bats in Sierra Leone, Kenya and Guinea [<a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>]. These data add to the body of evidence suggesting that bats are a reservoir for filoviruses. However, these did not conclusively link the presence of viral RNA in bats to human infections with filoviruses. The discovery of BOMV in bats residing near and in human dwellings and residential areas further highlights the gaps in knowledge about ebolavirus diversity and ecology. Given these gaps and the human and economic impacts of ebolavirus disease, there is an ongoing need for ebolavirus biosurveillance and further characterization of BOMV. Availability of efficient viral RNA detection assays is critical for bio surveillance of these reservoirs.</p><p id="p-42">Based on <em>in silico</em> analyses, we determined that current EBOV assays could potentially fail to detect BOMV sequences, and thus there is a need for BOMV specific assays. Using BioLaboro we rapidly designed and evaluated new, more specific assays. An advantage of BioVelocity is that the end user obtains results quickly with high confidence that the output of conserved and unique signature regions are accurate and not based on heuristics and probabilities. Additionally, Primer3 allows an end user to determine which signatures yield the best primers and probes, based on an objective penalty scoring system. PSET tests the PCR assays <em>in silico</em> against the latest versions of public sequence repositories, including newly added strain genomes, to validate that the primers and probes match only to their intended targets. BioLaboro’s easy-to-use Graphical User Interface (GUI) provides full functionality for submitting a new job, viewing the current job queue, checking results from previously completed jobs, and exploring the system database management and settings. The dedicated large RAM system easily supports multiple users with discrete logins and rapid operations. Moreover, the user-friendly GUI allows scientists without command-line experience to design and evaluate an assay for immediate wet lab testing.</p><p id="p-43">Due to the relative novelty of BOMV there are currently only three complete genomes available from NCBI. As more samples are identified and sequenced the genetic diversity will likely increase. During the 2014 - 2016 Western African EBOV outbreak, rapid accumulation of inter- and intra-host genetic variations were observed [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]. Since many of the nucleotide mutations altered protein sequences, it became apparent that the changes should be monitored for impacts on diagnostics, vaccines, and therapies critical to outbreak response. In an earlier study conducted to decipher the impact of the then-available diagnostics, we determined that many of the real-time reverse transcription PCR (rRT-PCR) assays that were in use during that outbreak identified regions outside of those that BioVelocity selected as unique to EBOV, SUDV, and RESTV [<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">32</a>]. In another study, signature erosion of diagnostic assays was identified during the 2018 outbreak in North Kivu and Ituri Provinces of the Democratic Republic of the Congo [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>]. Using the <em>in silico</em> methods described here, only two of the 30 EBOV rRT-PCR assays evaluated against BOMV target sequences met our criteria for successful detection and neither showed perfect matches. For ongoing biosurveillance, we recommend wet lab testing and validation of the rRT-PCR assays described here to ensure detection of BOMV.</p><p id="p-44">We also tested the BioLaboro pipeline with available SARS-CoV-2 viral genomes and rRT-PCR assays. We identified five signature sequences distributed across the genome and in different regions than those of the CDC or German group [<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a>, <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>]. There are seven assays currently in use (3 CDC and 4 German) for SARS-CoV-2 diagnostics and they all produced true positive results against the human-derived sample sequences without any signs of signature erosion. A few whole genome sequences that showed false negative results were from environmental samples (Bat and Pangolin origin) indicating that the diagnostic assays are specific for human isolates. The assays that produced true positive results were pan assays. The lack of signature erosion is in agreement with the whole genome sequence data analyzed thus far (February 28, 2020). A simple pipeline consisting of Multiple Sequence Alignment using Fast Fourier Transform (MAFFT), snp-sites, and R we calculated 228 single nucleotide polymorphisms (SNPs) (159 unique) across 145 genomes [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>-<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>]. Each of the 145 WGS contains less than 10 SNPs with the exception of one having 25. None of the variations impact the diagnostic assay signatures. However, real-time monitoring of these assays against WGS as they become available, will enable rapid identification of signature erosion if it occurs and generation of new assays as needed. The newly designed assays we have described here need to be validated in wet lab testing and with appropriate clinical matrices to determine their performance. However, we have demonstrated that the BioLaboro pipeline can be used effectively and rapidly to validate available assays and to design new assays using genome sequences of newly emerging pathogens.</p></div><div id="sec-11" class="subsection"><h3>Conclusions</h3><p id="p-45">By periodically re-running BioLaboro on emerging and reemerging pathogen sequences as they become available, over time the relative diversity can be monitored, and assays can be updated to remain current with regards to available data. By tracking assay performance measures over time, one can evaluate the efficacy of MCMs on a routine basis. These analyses would ensure that the most accurate MCMs are available when an outbreak response is necessary. In this study we demonstrate the value of real-time genomic sequencing and MCM evaluation to provide actionable information before and during a public health emergency. Combined with an active biosurveillance of zoonotic reservoirs and generation of sequence data to understand the genetic diversity of these pathogens, BioLaboro is broadly applicable for providing effective diagnostics and medical countermeasures during a crisis involving future threats.</p></div></div><div class="section" id="sec-12"><h2 class="">Methods</h2><div id="sec-13" class="subsection"><h3>BioLaboro System Description</h3><p id="p-46">BioLaboro is an application for rapidly designing <em>de novo</em> assays and validating existing PCR detection assays. It is composed of three tools: BioVelocity, Primer3, and PSET which are built into a pipeline for user-friendly new assay discovery via an interactive graphic user interface.</p><p id="p-47"><strong>BioVelocity</strong> is a bioinformatics tool based on an innovative algorithm and approach to genomic reference indices [<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">32</a>]. Using a fast and accurate hashing algorithm, BioVelocity can quickly align reads to a large set of references. BioVelocity takes advantage of large RAM systems (hardware specification described in Additional Table 1) and creates a k-mer index of all selected reference sequences (e.g. GenBank) by identifying all possible base pair sequences of various k-mer lengths. This index is used to determine all possible matches between query sequences and references, simultaneously. The advantage of this approach is that it allows for rapid identification of sequences conserved within or omitted from a set of target references. Thus, the used has high confidence in the conserved and signature (unique) designations because they are not based on heuristics and probabilities.</p><p id="p-48"><strong>Primer3</strong> [<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>] is a tool for designing primers and probes for real-time PCR reactions. It considers a range of criteria such as oligonucleotide melting temperature, size, GC content, and primer-dimer possibilities. Potential new primer sets are identified within the signature regions using Primer3 analysis. For the BioLaboro pipeline, Primer3 has been configured to analyze 22 parameters influencing suitability of a sequence to primer creation and construct viable primers which cover identified signature regions. These primers are scored with a penalty scoring system to attempt to determine the fitness of the resulting primers thus allowing an end user to assess which signatures yield the best primers when the signatures themselves may be of similar size.</p><p id="p-49"><strong>PSET</strong> is configured to test PCR assays <em>in silico</em> against the latest versions of public sequence repositories to determine if the primers and probes still match only to their intended targets. An elaborate description of PSET is provided in ref [<a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">32</a>]. As NCBI’s database and other public databases are updated periodically, newly added genomic sequences can reveal where primers and probes may no longer be functional or where PCR assays may detect previously un-sequenced near neighbors. Using this information, an assay provider can be better aware of potential false hits and design new primers when false hits become an issue. PSET is used to test currently deployed assays as well as new assays designed using BioLaboro’s capabilities.</p><p id="p-50">The BioLaboro application is composed of a fully functional GUI front-end that allows users to submit jobs to the back-end bioinformatics pipeline hosted on a dedicated large RAM system. The system has multi-user capability with discrete logins and a single job queue. The landing screen, shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, gives the user options for submitting a new job, viewing the current job queue, checking results from previously completed jobs, and exploring the system database management and settings.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The landing page of BioLaboro. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-888010457" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The landing page of BioLaboro.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F4.medium.gif" width="440" height="280"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F4.medium.gif" width="440" height="280"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">The landing page of BioLaboro.</span><div class="sb-div caption-clear"></div></div></div><p id="p-51">The BioLaboro application allows job submissions through a simplified user interface designed for scientists with minimal or no command-line experience. The user can search for sequences of interest using the built-in Organism Select tool, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, which allows for searching on free text, NCBI Accession number, or NCBI Taxonomy ID. The results can then be filtered using a “smart filter” which will only include sequences within +/- 10% of the calculated median genome length of the results. This tool is useful for automatically excluding plasmids or sequence fragments which can negatively impact signature identification. Alternatively, custom sequence size filters can also be used if the user wants to target specific plasmids or chromosomes. Once all sequences are selected and added the user can optionally choose a specific sequence to serve as the algorithmic reference.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The Job Submission page for the BioVelocity component of BioLaboro showing the BOMV sequences. BioLaboro employs a queuing system to manage job submissions due to the high computational requirements of the BioVelocity algorithm. The queue page identifies the currently running job, the ordered list of queued jobs, and a list of previously finished jobs with timestamps and completion status, Figure 6. Each finished job can be re-launched from this dialog in the future with previously used parameters while utilizing the newest available datasets." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-888010457" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The Job Submission page for the BioVelocity component of BioLaboro showing the BOMV sequences.&lt;/span&gt; BioLaboro employs a queuing system to manage job submissions due to the high computational requirements of the BioVelocity algorithm. The queue page identifies the currently running job, the ordered list of queued jobs, and a list of previously finished jobs with timestamps and completion status, Figure 6. Each finished job can be re-launched from this dialog in the future with previously used parameters while utilizing the newest available datasets.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F5.medium.gif" width="440" height="290"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F5.medium.gif" width="440" height="290"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">The Job Submission page for the BioVelocity component of BioLaboro showing the BOMV sequences.</span><p id="p-52" class="first-child">BioLaboro employs a queuing system to manage job submissions due to the high computational requirements of the BioVelocity algorithm. The queue page identifies the currently running job, the ordered list of queued jobs, and a list of previously finished jobs with timestamps and completion status, <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. Each finished job can be re-launched from this dialog in the future with previously used parameters while utilizing the newest available datasets.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The BioLaboro job queue. The queue includes three sections: 1) The currently running job is identified with information on the current sub-process, 2) The job queue is shown, which can be re-arranged as needed, and 3) The finished jobs section shows completed jobs with timestamp metadata and an option to re-run the same job with identical parameters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-888010457" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The BioLaboro job queue.&lt;/span&gt; The queue includes three sections: 1) The currently running job is identified with information on the current sub-process, 2) The job queue is shown, which can be re-arranged as needed, and 3) The finished jobs section shows completed jobs with timestamp metadata and an option to re-run the same job with identical parameters.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F6.medium.gif" width="440" height="280"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F6.medium.gif" width="440" height="280"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <span class="caption-title">The BioLaboro job queue.</span><p id="p-53" class="first-child">The queue includes three sections: 1) The currently running job is identified with information on the current sub-process, 2) The job queue is shown, which can be re-arranged as needed, and 3) The finished jobs section shows completed jobs with timestamp metadata and an option to re-run the same job with identical parameters.</p><div class="sb-div caption-clear"></div></div></div><p id="p-54">The Database Management and Settings Page contains many options for maintaining system data and pipeline settings. BioLaboro’s simple user interface design allows the user to manage the genomic reference databases from within the GUI. The GenBank Repository page, <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>, lists the databases loaded on the system along with the number of sequences each contains. These databases are used for selecting sequences for signature creation and can be updated on-demand by submitting a sync job to the queue. Other GenBank databases or custom sequence databases can also be added and maintained through this entry point.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The BioLaboro GenBank Repository located in the Database Management and Settings page. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-888010457" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;The BioLaboro GenBank Repository located in the Database Management and Settings page.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F7.medium.gif" width="440" height="291"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F7.medium.gif" width="440" height="291"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.08.031963/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7.</span> <span class="caption-title">The BioLaboro GenBank Repository located in the Database Management and Settings page.</span><div class="sb-div caption-clear"></div></div></div><p id="p-55">PCR detection assays are also maintained in the system and can be viewed and searched for in the Assay Repository. The system maintains a history of past assay runs so that the user can track performance of assays over time, as well as re-run assays when new sequences of interest are available. Lastly the Database and Management Settings page maintains the default settings for each of the three system tools which can be permanently set here, or specifically tailored for each job at run-time. Results for completed runs are available from the Results page which allows the user to view raw data or system generated reports as well as download them to local copies.</p></div></div><div class="section" id="sec-15"><h2 class="">Supplementary Information</h2><div id="T9" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1243249/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1243249/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1243249" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Additional Table 1.</span> <span class="caption-title">Large RAM System Hardware Specifications.</span><div class="sb-div caption-clear"></div></div></div><p id="p-56">Additional Table 2. (gisaid_sequences.xlsx) List of whole genome sequences downloaded from GISAID and used in this study. The “Signature Creation” column indicates which sequences were used to generate the signatures, and the “Validation” column indicates which sequences were used to validate the signatures with PSET.</p></div><div class="section" id="sec-16"><h2 class="">Declarations</h2><p id="p-57"><strong>Ethics approval and consent to participate:</strong> Not applicable</p><p id="p-58"><strong>Consent for publication</strong>: Not applicable</p><p id="p-59"><strong>Availability of data and materials:</strong></p><p id="p-60">Weekly updates of SARS-CoV assay performance using PSET are posted at Viological.org. (<a href="http://virological.org/t/preliminary-in-silico-assessment-of-the-specificity-of-published-molecular-assays-and-design-of-new-assays-using-the-available-whole-genome-sequences-of-2019-ncov/343/16">http://virological.org/t/preliminary-in-silico-assessment-of-the-specificity-of-published-molecular-assays-and-design-of-new-assays-using-the-available-whole-genome-sequences-of-2019-ncov/343/16</a>).</p><p id="p-61">The datasets analyzed during the current study are available in the following:</p><p id="p-62">Three available complete genome sequences of BOMV NCBI Accession numbers: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MF319185.1&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom">MF319185.1</a> <a href="https://www.ncbi.nlm.nih.gov/nuccore/MF319185.1/">https://www.ncbi.nlm.nih.gov/nuccore/MF319185.1/</a>, MF319186.1 <a href="https://www.ncbi.nlm.nih.gov/nuccore/MF319186.1">https://www.ncbi.nlm.nih.gov/nuccore/MF319186.1</a>, MK340750.1 <a href="https://www.ncbi.nlm.nih.gov/nuccore/MK340750.1">https://www.ncbi.nlm.nih.gov/nuccore/MK340750.1</a></p><p id="p-63">Complete genome sequences of SARS-CoV-2: We gratefully acknowledge the authors, originating and submitting laboratories of the sequences from GISAID’s EpiFlu™ Database on which this research is based. <a href="https://www.gisaid.org/">https://www.gisaid.org/</a></p><p id="p-64">NCBI Taxonomy database: <a href="https://www.ncbi.nlm.nih.gov/taxonomy">https://www.ncbi.nlm.nih.gov/taxonomy</a></p><p id="p-65">EMBL-EBI FASTA GLSEARCH: <a href="https://www.ebi.ac.uk/Tools/sss/fasta/nucleotide.html">https://www.ebi.ac.uk/Tools/sss/fasta/nucleotide.html</a></p></div><div class="section" id="sec-17"><h2 class="">Competing interests</h2><p id="p-66">The authors declare that they have no competing interests.</p></div><div class="section" id="sec-18"><h2 class="">Funding</h2><p id="p-67">BioLaboro was developed by Noblis with funding received from the Defense Biological Product Assurance Office of the US Department of Defense under contract number W911QY-17-C-0016. The funding body did not play any roles in the design of the study and collection, analysis, and interpretation of data and in writing the manuscript. BioVelocity is a patented and trademarked tool developed by Noblis exclusively at its private expense. The research and reports related to the SARS-CoV-2 work also was funded exclusively by Noblis at its private expense.</p></div><div class="section" id="sec-19"><h2 class="">Authors’ contributions</h2><p id="p-68">MH, DN, SM, SS conceptualized the study. MH, DN, SM, ND, MI, ST developed BioLaboro. MH, DN, SM, SS analyzed the data. BG provided study resources. MH, DN, SM, MI, TB, KJ, SS wrote and edited the manuscript.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-69">The views expressed in this article are those of the authors and do not necessarily reflect the official policy or position of the DBPAO, JPEO-CBRND, Department of Defense, the US Government, nor the institutions or companies affiliated with the authors. BG is a US Government employee and this work was prepared as part of his official duties. Title 17 of the United States Code §105 provides that “Copyright protection under this title is not available for any work of the United States Government.” Title 17 of the United States Code §101 defines a US Government work as a work prepared by a military service member or employee of the US Government as part of that person’s official duties. We gratefully acknowledge the Authors and the Originating and Submitting Laboratories for the SARS-CoV-2 sequences and metadata shared through GISAID (<a href="https://www.gisaid.org/">https://www.gisaid.org/</a>) on which SARS-CoV-2 part of this study is based.</p></div><div class="section glossary"><h2 class="">Abbreviations</h2><dl id="def-list-1"><dt id="def-item-1">bp<span class="def-item-dt-sep"></span></dt><dd>base pair (bps, plural base pairs)<span class="def-item-dd-sep"></span></dd><dt id="def-item-2">BDBV<span class="def-item-dt-sep"></span></dt><dd>Bundibugyo ebolavirus<span class="def-item-dd-sep"></span></dd><dt id="def-item-3">BLAST<span class="def-item-dt-sep"></span></dt><dd>Basic Local Alignment Search Tool<span class="def-item-dd-sep"></span></dd><dt id="def-item-4">BOMV<span class="def-item-dt-sep"></span></dt><dd>Bombali ebolavirus<span class="def-item-dd-sep"></span></dd><dt id="def-item-5">CFR<span class="def-item-dt-sep"></span></dt><dd>Case Fatality Ratio<span class="def-item-dd-sep"></span></dd><dt id="def-item-6">COVID-19<span class="def-item-dt-sep"></span></dt><dd>Coronavirus Disease 2019<span class="def-item-dd-sep"></span></dd><dt id="def-item-7">DNA<span class="def-item-dt-sep"></span></dt><dd>deoxyribonucleic acid<span class="def-item-dd-sep"></span></dd><dt id="def-item-8">DRC<span class="def-item-dt-sep"></span></dt><dd>Democratic Republic of the Congo<span class="def-item-dd-sep"></span></dd><dt id="def-item-9">EBOV<span class="def-item-dt-sep"></span></dt><dd>Zaire ebolavirus<span class="def-item-dd-sep"></span></dd><dt id="def-item-10">EMA<span class="def-item-dt-sep"></span></dt><dd>European Medicines Agency<span class="def-item-dd-sep"></span></dd><dt id="def-item-11">EVD<span class="def-item-dt-sep"></span></dt><dd>Ebola Virus Disease<span class="def-item-dd-sep"></span></dd><dt id="def-item-12">FDA<span class="def-item-dt-sep"></span></dt><dd>Food and Drug Administration<span class="def-item-dd-sep"></span></dd><dt id="def-item-13">FN<span class="def-item-dt-sep"></span></dt><dd>False Negative<span class="def-item-dd-sep"></span></dd><dt id="def-item-14">FP<span class="def-item-dt-sep"></span></dt><dd>False Positive<span class="def-item-dd-sep"></span></dd><dt id="def-item-15">GC<span class="def-item-dt-sep"></span></dt><dd>guanine cytosine<span class="def-item-dd-sep"></span></dd><dt id="def-item-16">GP<span class="def-item-dt-sep"></span></dt><dd>glycoprotein<span class="def-item-dd-sep"></span></dd><dt id="def-item-17">GUI<span class="def-item-dt-sep"></span></dt><dd>Graphic User interface<span class="def-item-dd-sep"></span></dd><dt id="def-item-18">ID<span class="def-item-dt-sep"></span></dt><dd>identifier<span class="def-item-dd-sep"></span></dd><dt id="def-item-19">mAb<span class="def-item-dt-sep"></span></dt><dd>monoclonal antibody<span class="def-item-dd-sep"></span></dd><dt id="def-item-20">MAFFT<span class="def-item-dt-sep"></span></dt><dd>Multiple Sequence Alignment using Fast Fourier Transform<span class="def-item-dd-sep"></span></dd><dt id="def-item-21">MCM<span class="def-item-dt-sep"></span></dt><dd>medical countermeasure<span class="def-item-dd-sep"></span></dd><dt id="def-item-22">N<span class="def-item-dt-sep"></span></dt><dd>nucleocapsid phosphoprotein<span class="def-item-dd-sep"></span></dd><dt id="def-item-23">N/A<span class="def-item-dt-sep"></span></dt><dd>Not available<span class="def-item-dd-sep"></span></dd><dt id="def-item-24">NCBI<span class="def-item-dt-sep"></span></dt><dd>National Center for Biotechnology Information<span class="def-item-dd-sep"></span></dd><dt id="def-item-25">NIAID<span class="def-item-dt-sep"></span></dt><dd>National Institute of Allergy and Infectious Diseases<span class="def-item-dd-sep"></span></dd><dt id="def-item-26">NP<span class="def-item-dt-sep"></span></dt><dd>nucleoprotein<span class="def-item-dd-sep"></span></dd><dt id="def-item-27">NPC1<span class="def-item-dt-sep"></span></dt><dd>Niemann-Pick C1<span class="def-item-dd-sep"></span></dd><dt id="def-item-28">ORF<span class="def-item-dt-sep"></span></dt><dd>Open Reading Frame<span class="def-item-dd-sep"></span></dd><dt id="def-item-29">PALM<span class="def-item-dt-sep"></span></dt><dd>Pamoja Tulinde Maisha study<span class="def-item-dd-sep"></span></dd><dt id="def-item-30">PCR<span class="def-item-dt-sep"></span></dt><dd>polymerase chain reaction<span class="def-item-dd-sep"></span></dd><dt id="def-item-31">PSET<span class="def-item-dt-sep"></span></dt><dd>PCR signature erosion tool<span class="def-item-dd-sep"></span></dd><dt id="def-item-32">RAM<span class="def-item-dt-sep"></span></dt><dd>random access memory<span class="def-item-dd-sep"></span></dd><dt id="def-item-33">RESTV<span class="def-item-dt-sep"></span></dt><dd>Reston ebolavirus<span class="def-item-dd-sep"></span></dd><dt id="def-item-34">RNA<span class="def-item-dt-sep"></span></dt><dd>ribonucleic acid<span class="def-item-dd-sep"></span></dd><dt id="def-item-35">rRT-PCR<span class="def-item-dt-sep"></span></dt><dd>real-time reverse-transcription polymerase chain reaction<span class="def-item-dd-sep"></span></dd><dt id="def-item-36">rVSV<span class="def-item-dt-sep"></span></dt><dd>recombinant vesicular stomatitis virus<span class="def-item-dd-sep"></span></dd><dt id="def-item-37">SARS<span class="def-item-dt-sep"></span></dt><dd>severe acute respiratory syndrome<span class="def-item-dd-sep"></span></dd><dt id="def-item-38">SNP<span class="def-item-dt-sep"></span></dt><dd>single nucleotide protein<span class="def-item-dd-sep"></span></dd><dt id="def-item-39">SUDV<span class="def-item-dt-sep"></span></dt><dd>Sudan ebolavirus<span class="def-item-dd-sep"></span></dd><dt id="def-item-40">TAFV<span class="def-item-dt-sep"></span></dt><dd>Taï Forest ebolavirus<span class="def-item-dd-sep"></span></dd><dt id="def-item-41">TN<span class="def-item-dt-sep"></span></dt><dd>True Negative<span class="def-item-dd-sep"></span></dd><dt id="def-item-42">TP<span class="def-item-dt-sep"></span></dt><dd>True Positive<span class="def-item-dd-sep"></span></dd><dt id="def-item-43">WGS<span class="def-item-dt-sep"></span></dt><dd>Whole Genome Sequence<span class="def-item-dd-sep"></span></dd><dt id="def-item-44">WHO<span class="def-item-dt-sep"></span></dt><dd>World Health Organization</dd></dl></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.1" data-doi="10.3201/eid1104.041167"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fauci</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Touchette</span>  <span class="cit-name-given-names">NA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Folkers</span>  <span class="cit-name-given-names">GK</span></span>: <span class="cit-article-title">Emerging infectious diseases: a 10-year perspective from the National Institute of Allergy and Infectious Diseases</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr> <span class="cit-pub-date">2005</span>, <span class="cit-vol">11</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">519</span>–<span class="cit-lpage">525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DFauci%26rft.auinit1%253DA.%2BS.%26rft.volume%253D11%26rft.issue%253D4%26rft.spage%253D519%26rft.epage%253D525%26rft.atitle%253DEmerging%2Binfectious%2Bdiseases%253A%2Ba%2B10-year%2Bperspective%2Bfrom%2Bthe%2BNational%2BInstitute%2Bof%2BAllergy%2Band%2BInfectious%2BDiseases.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1104.041167%26rft_id%253Dinfo%253Apmid%252F15829188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1104.041167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15829188&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227940700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.2" data-doi="10.1371/journal.ppat.1003467"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morens</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fauci</span>  <span class="cit-name-given-names">AS</span></span>: <span class="cit-article-title">Emerging infectious diseases: threats to human health and global stability</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">9</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">e1003467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D9%26rft.spage%253De1003467%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003467%26rft_id%253Dinfo%253Apmid%252F23853589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003467&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23853589&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">3.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.08.031963v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paules</span>  <span class="cit-name-given-names">CI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eisinger</span>  <span class="cit-name-given-names">RW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marston</span>  <span class="cit-name-given-names">HD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fauci</span>  <span class="cit-name-given-names">AS</span></span>: <span class="cit-article-title">What Recent History Has Taught Us About Responding to Emerging Infectious Disease Threats</span>. <abbr class="cit-jnl-abbrev">Ann Intern Med</abbr> <span class="cit-pub-date">2017</span>, <span class="cit-vol">167</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">805</span>–<span class="cit-lpage">811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn%2BIntern%2BMed%26rft.volume%253D167%26rft.spage%253D805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">4.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.08.031963v1.4" data-doi="10.1038/embor.2008.110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fonkwo</span>  <span class="cit-name-given-names">PN</span></span>: <span class="cit-article-title">Pricing infectious disease. The economic and health implications of infectious diseases</span>. <abbr class="cit-jnl-abbrev">EMBO Rep</abbr> <span class="cit-pub-date">2008</span>, <span class="cit-vol">9</span> Suppl <span class="cit-issue">1</span>:<span class="cit-fpage">S13</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2Breports%26rft.stitle%253DEMBO%2BRep.%26rft.aulast%253DFonkwo%26rft.auinit1%253DP.%2BN.%26rft.volume%253D9%26rft.issue%253D1S%26rft.spage%253DS13%26rft.epage%253DS17%26rft.atitle%253DPricing%2Binfectious%2Bdisease%253A%2BThe%2Beconomic%2Band%2Bhealth%2Bimplications%2Bof%2Binfectious%2Bdiseases%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fembor.2008.110%26rft_id%253Dinfo%253Apmid%252F18578017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZW1ib3IiO3M6NToicmVzaWQiO3M6ODoiOS8xUy9TMTMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8xMC8yMDIwLjA0LjA4LjAzMTk2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.5" data-doi="10.1073/pnas.1701410114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">DE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Black</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rappuoli</span>  <span class="cit-name-given-names">R</span></span>: <span class="cit-article-title">Emerging infectious diseases: A proactive approach</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-pub-date">2017</span>, <span class="cit-vol">114</span>(<span class="cit-issue">16</span>):<span class="cit-fpage">4055</span>–<span class="cit-lpage">4059</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1701410114%26rft_id%253Dinfo%253Apmid%252F28396438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE0LzE2LzQwNTUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8xMC8yMDIwLjA0LjA4LjAzMTk2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.08.031963v1.6"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">World Health Organization</span>: <span class="cit-article-title">Coronavirus disease 2019 (COVID-19) Situation Report – 50</span> [<a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200310-sitrep-50-covid-19.pdf?sfvrsn=55e904fb_2">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200310-sitrep-50-covid-19.pdf?sfvrsn=55e904fb_2</a>]</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.08.031963v1.7"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">GISAID</span>. [<a href="https://www.gisaid.org/epiflu-applications/global-cases-covid-19/">https://www.gisaid.org/epiflu-applications/global-cases-covid-19/</a>]</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">L</span></span>: <span class="cit-article-title">An interactive web-based dashboard to track COVID-19 in real time</span>. <abbr class="cit-jnl-abbrev">Lancet Infect Dis</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.08.031963v1.9"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">World Health Organization</span>: <span class="cit-article-title">Fact Sheet/Details/Ebola virus disease</span> [<a href="https://www.who.int/news-room/fact-sheets/detail/ebola-virus-disease">https://www.who.int/news-room/fact-sheets/detail/ebola-virus-disease</a>]</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.10" data-doi="10.15252/emmm.201404792"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mari Saez</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nowak</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lapeyre</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zimmermann</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dux</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuhl</span>  <span class="cit-name-given-names">HS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaba</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Regnaut</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Merkel</span>  <span class="cit-name-given-names">K</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Investigating the zoonotic origin of the West African Ebola epidemic</span>. <abbr class="cit-jnl-abbrev">EMBO Mol Med</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">7</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">17</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BMol%2BMed%26rft_id%253Dinfo%253Adoi%252F10.15252%252Femmm.201404792%26rft_id%253Dinfo%253Apmid%252F25550396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZW1ib21tIjtzOjU6InJlc2lkIjtzOjY6IjcvMS8xNyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzEwLzIwMjAuMDQuMDguMDMxOTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.08.031963v1.11"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">World Health Organization</span>: <span class="cit-article-title">Situation Report-Ebola Virus Disease-10 June 2016</span>. <a href="https://www.who.int/csr/disease/ebola/situation-reports/archive/en/">https://www.who.int/csr/disease/ebola/situation-reports/archive/en/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.08.031963v1.12"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">World Health Organization</span>: <span class="cit-article-title">Ebola in the Democratic Republic of Congo-Health Emergecy Update</span> [<a href="https://www.who.int/emergencies/diseases/ebola/drc-2019">https://www.who.int/emergencies/diseases/ebola/drc-2019</a>].</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.13" data-doi="10.1056/NEJMoa1910993"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mulangu</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dodd</span>  <span class="cit-name-given-names">LE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">RT</span>,   <span class="cit-name-suffix">Jr.</span>.</span>, <span class="cit-auth"><span class="cit-name-surname">Tshiani Mbaya</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Proschan</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mukadi</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lusakibanza Manzo</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nzolo</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tshomba Oloma</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ibanda</span>  <span class="cit-name-given-names">A</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">A Randomized, Controlled Trial of Ebola Virus Disease Therapeutics</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">381</span>(<span class="cit-issue">24</span>):<span class="cit-fpage">2293</span>–<span class="cit-lpage">2303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D381%26rft.spage%253D2293%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1910993%26rft_id%253Dinfo%253Apmid%252F31774950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1910993&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=31774950&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.14"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">European Medicines Agency</span>: <span class="cit-article-title">First Vaccine to protect against Ebola</span>. In: <abbr class="cit-jnl-abbrev">EMA/CHMP/565403/2019</abbr>. <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.08.031963v1.15"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Food and Drug Administration</span>: <span class="cit-source">First FDA-approved vaccine for the prevention of Ebola virus disease, marking a critical milestone in public health preparedness and response</span>. In.; <span class="cit-pub-date">2019</span>. <a href="https://www.fda.gov/news-events/press-announcements/first-fda-approved-vaccine-prevention-ebola-virus-disease-marking-critical-milestone-public-health">https://www.fda.gov/news-events/press-announcements/first-fda-approved-vaccine-prevention-ebola-virus-disease-marking-critical-milestone-public-health</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zawilinska</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kosz-Vnenchak</span>  <span class="cit-name-given-names">M</span></span>: <span class="cit-article-title">General introduction into the Ebola virus biology and disease</span>. <abbr class="cit-jnl-abbrev">Folia Med Cracov</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">54</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">57</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFolia%2BMed%2BCracov%26rft.volume%253D54%26rft.spage%253D57%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.17" data-doi="10.1146/annurev-pathol-052016-100506"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baseler</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chertow</span>  <span class="cit-name-given-names">DS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feldmann</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morens</span>  <span class="cit-name-given-names">DM</span></span>: <span class="cit-article-title">The Pathogenesis of Ebola Virus Disease</span>. <abbr class="cit-jnl-abbrev">Annu Rev Pathol</abbr> <span class="cit-pub-date">2017</span>, <span class="cit-vol">12</span>:<span class="cit-fpage">387</span>–<span class="cit-lpage">418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BPathol%26rft.volume%253D12%26rft.spage%253D387%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-pathol-052016-100506%26rft_id%253Dinfo%253Apmid%252F27959626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-pathol-052016-100506&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27959626&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cantoni</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hamlet</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Michaelis</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wass</span>  <span class="cit-name-given-names">MN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rossman</span>  <span class="cit-name-given-names">JS</span></span>: <span class="cit-article-title">Risks Posed by Reston, the Forgotten Ebolavirus</span>. <abbr class="cit-jnl-abbrev">mSphere</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">1</span>(<span class="cit-issue">6</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.19" data-doi="10.1093/infdis/jir296"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miranda</span>  <span class="cit-name-given-names">ME</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miranda</span>  <span class="cit-name-given-names">NL</span></span>: <span class="cit-article-title">Reston ebolavirus in humans and animals in the Philippines: a review</span>. <abbr class="cit-jnl-abbrev">J Infect Dis</abbr> <span class="cit-pub-date">2011</span>, <span class="cit-vol">204</span> Suppl <span class="cit-issue">3</span>:<span class="cit-fpage">S757</span>–<span class="cit-lpage">760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BInfect%2BDis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Finfdis%252Fjir296%26rft_id%253Dinfo%253Apmid%252F21987747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/infdis/jir296&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21987747&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anthony</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gbakima</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bird</span>  <span class="cit-name-given-names">BH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bangura</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tremeau-Bravard</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belaganahalli</span>  <span class="cit-name-given-names">MN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wells</span>  <span class="cit-name-given-names">HL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dhanota</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">E</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">The discovery of Bombali virus adds further support for bats as hosts of ebolaviruses</span>. <abbr class="cit-jnl-abbrev">Nat Microbiol</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">3</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">1084</span>–<span class="cit-lpage">1089</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMicrobiol%26rft.volume%253D3%26rft.spage%253D1084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.21" data-doi="10.1038/438575a"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leroy</span>  <span class="cit-name-given-names">EM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumulungui</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pourrut</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rouquet</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hassanin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yaba</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delicat</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paweska</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Swanepoel</span>  <span class="cit-name-given-names">R</span></span>: <span class="cit-article-title">Fruit bats as reservoirs of Ebola virus</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2005</span>, <span class="cit-vol">438</span>(<span class="cit-issue">7068</span>):<span class="cit-fpage">575</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLeroy%26rft.auinit1%253DE.%2BM.%26rft.volume%253D438%26rft.issue%253D7068%26rft.spage%253D575%26rft.epage%253D576%26rft.atitle%253DFruit%2Bbats%2Bas%2Breservoirs%2Bof%2BEbola%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F438575a%26rft_id%253Dinfo%253Apmid%252F16319873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/438575a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16319873&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233593100030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">22.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.08.031963v1.22" data-doi="10.1038/527453a"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heeney</span>  <span class="cit-name-given-names">JL</span></span>: <span class="cit-article-title">Ebola: Hidden reservoirs</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">527</span>(<span class="cit-issue">7579</span>):<span class="cit-fpage">453</span>–<span class="cit-lpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D527%26rft.spage%253D453%26rft_id%253Dinfo%253Adoi%252F10.1038%252F527453a%26rft_id%253Dinfo%253Apmid%252F26607539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/527453a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26607539&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">23.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.08.031963v1.23" data-doi="10.1007/s10393-015-1053-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leendertz</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gogarten</span>  <span class="cit-name-given-names">JF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dux</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Calvignac-Spencer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leendertz</span>  <span class="cit-name-given-names">FH</span></span>: <span class="cit-article-title">Assessing the Evidence Supporting Fruit Bats as the Primary Reservoirs for Ebola Viruses</span>. <abbr class="cit-jnl-abbrev">Ecohealth</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">13</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">18</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcohealth%26rft.volume%253D13%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10393-015-1053-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10393-015-1053-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.24" data-doi="10.3390/V6041759"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Olival</span>  <span class="cit-name-given-names">KJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayman</span>  <span class="cit-name-given-names">DT</span></span>: <span class="cit-article-title">Filoviruses in bats: current knowledge and future directions</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">6</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">1759</span>–<span class="cit-lpage">1788</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D6%26rft.spage%253D1759%26rft_id%253Dinfo%253Adoi%252F10.3390%252FV6041759%26rft_id%253Dinfo%253Apmid%252F24747773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/V6041759&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24747773&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000335761600015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.25" data-doi="10.1038/srep23743"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pappalardo</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Julia</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rossman</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Michaelis</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wass</span>  <span class="cit-name-given-names">MN</span></span>: <span class="cit-article-title">Conserved differences in protein sequence determine the human pathogenicity of Ebolaviruses</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">6</span>:<span class="cit-fpage">23743</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D6%26rft.spage%253D23743%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep23743%26rft_id%253Dinfo%253Apmid%252F27009368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep23743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27009368&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martell</span>  <span class="cit-name-given-names">HJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masterson</span>  <span class="cit-name-given-names">SG</span></span>, <span class="cit-auth"><span class="cit-name-surname">McGreig</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Michaelis</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wass</span>  <span class="cit-name-given-names">MN</span></span>: <span class="cit-article-title">Is the Bombali virus pathogenic in humans?</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">35</span>(<span class="cit-issue">19</span>):<span class="cit-fpage">3553</span>–<span class="cit-lpage">3558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D35%26rft.spage%253D3553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.08.031963v1.27"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">World Health Organization</span>: <span class="cit-article-title">WHO Director-General’s opening remarks at the media briefing on COVID-19 - 11 March 2020</span>. In.; <span class="cit-pub-date">2020</span>. <a href="https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19---11-march-2020">https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19---11-march-2020</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.28" data-doi="10.1016/S0140-6736(20)30183-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">X</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-pub-date">2020</span>, <span class="cit-vol">395</span>(<span class="cit-issue">10223</span>):<span class="cit-fpage">497</span>–<span class="cit-lpage">506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D497%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930183-5%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30183-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">XL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">XG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Si</span>  <span class="cit-name-given-names">HR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">CL</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.08.031963v1.30"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">World Health Organization</span>: <span class="cit-source">Coronavirus disease 2019 (COVID-19)Situation Report – 49</span> [<a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200309-sitrep-49-covid-19.pdf?sfvrsn=70dabe61_4">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200309-sitrep-49-covid-19.pdf?sfvrsn=70dabe61_4</a>]</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.31" data-doi="10.1002/gch2.1018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elbe</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buckland-Merrett</span>  <span class="cit-name-given-names">G</span></span>: <span class="cit-article-title">Data, disease and diplomacy: GISAID’s innovative contribution to global health</span>. <abbr class="cit-jnl-abbrev">Glob Chall</abbr> <span class="cit-pub-date">2017</span>, <span class="cit-vol">1</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">33</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob%2BChall%26rft.volume%253D1%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgch2.1018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gch2.1018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.32" data-doi="10.3390/v7062763"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sozhamannan</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">MY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">AT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Negron</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ivancich</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koehler</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Minogue</span>  <span class="cit-name-given-names">TD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">WJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Christopher</span>  <span class="cit-name-given-names">GW</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Evaluation of Signature Erosion in Ebola Virus Due to Genomic Drift and Its Impact on the Performance of Diagnostic Assays</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">7</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">3130</span>–<span class="cit-lpage">3154</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D7%26rft.spage%253D3130%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv7062763%26rft_id%253Dinfo%253Apmid%252F26090727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v7062763&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26090727&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.33" data-doi="10.1093/bib/4.2.133"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Slezak</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuczmarski</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ott</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Medeiros</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Truitt</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mulakken</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vitalis</span>  <span class="cit-name-given-names">E</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Comparative genomics tools applied to bioterrorism defence</span>. <abbr class="cit-jnl-abbrev">Brief Bioinform</abbr> <span class="cit-pub-date">2003</span>, <span class="cit-vol">4</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">133</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBriefings%2Bin%2BBioinformatics%26rft.stitle%253DBrief%2BBioinform%26rft.aulast%253DSlezak%26rft.auinit1%253DT.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D133%26rft.epage%253D149%26rft.atitle%253DComparative%2Bgenomics%2Btools%2Bapplied%2Bto%2Bbioterrorism%2Bdefence%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252F4.2.133%26rft_id%253Dinfo%253Apmid%252F12846395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/4.2.133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12846395&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.34" data-doi="10.1093/nar/gki896"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">SN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">CL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Slezak</span>  <span class="cit-name-given-names">TR</span></span>: <span class="cit-article-title">Draft versus finished sequence data for DNA and protein diagnostic signature development</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-pub-date">2005</span>, <span class="cit-vol">33</span>(<span class="cit-issue">18</span>):<span class="cit-fpage">5838</span>–<span class="cit-lpage">5850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGardner%26rft.auinit1%253DS.%2BN.%26rft.volume%253D33%26rft.issue%253D18%26rft.spage%253D5838%26rft.epage%253D5850%26rft.atitle%253DDraft%2Bversus%2Bfinished%2Bsequence%2Bdata%2Bfor%2BDNA%2Band%2Bprotein%2Bdiagnostic%2Bsignature%2Bdevelopment%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki896%26rft_id%253Dinfo%253Apmid%252F16243783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki896&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16243783&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233046100026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.35" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">EW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">DJ</span></span>: <span class="cit-article-title">Basic local alignment search tool</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-pub-date">1990</span>, <span class="cit-vol">215</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Madeira</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">YM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buso</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gur</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madhusoodanan</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Basutkar</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tivey</span>  <span class="cit-name-given-names">ARN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Potter</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Finn</span>  <span class="cit-name-given-names">RD</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">The EMBL-EBI search and sequence analysis tools APIs in 2019</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">47</span>(<span class="cit-issue">W1</span>):<span class="cit-fpage">W636</span>–<span class="cit-lpage">W641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253DW636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Amarasinghe</span>  <span class="cit-name-given-names">GK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Basler</span>  <span class="cit-name-given-names">CF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bavari</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bukreyev</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chandran</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dolnik</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dye</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ebihara</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Formenty</span>  <span class="cit-name-given-names">P</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Implementation of Objective PASC-Derived Taxon Demarcation Criteria for Official Classification of Filoviruses</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-pub-date">2017</span>, <span class="cit-vol">9</span>(<span class="cit-issue">5</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zulkower</span>  <span class="cit-name-given-names">V</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Rosser</span>,  <span class="cit-name-given-names">S.</span></span>: <span class="cit-article-title">DNA Features Viewer, a sequence annotations formatting and plotting library for Python</span>. <abbr class="cit-jnl-abbrev">BioRxiv</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jernigan</span>  <span class="cit-name-given-names">DB</span></span>, <span class="cit-auth"><span class="cit-name-surname">nCo</span>  <span class="cit-name-given-names">VCDCRT</span></span>: <span class="cit-article-title">Initial Public Health Response and Interim Clinical Guidance for the 2019 Novel Coronavirus Outbreak - United States, December 31, 2019-February 4, 2020</span>. <abbr class="cit-jnl-abbrev">MMWR Morb Mortal Wkly Rep</abbr> <span class="cit-pub-date">2020</span>, <span class="cit-vol">69</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">140</span>–<span class="cit-lpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMMWR%2BMorb%2BMortal%2BWkly%2BRep%26rft.volume%253D69%26rft.spage%253D140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Corman</span>  <span class="cit-name-given-names">VM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Landt</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaiser</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Molenkamp</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meijer</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">DKW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bleicker</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brunink</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">ML</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</span>. <abbr class="cit-jnl-abbrev">Euro Surveill</abbr> <span class="cit-pub-date">2020</span>, <span class="cit-vol">25</span>(<span class="cit-issue">3</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Forbes</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Webala</span>  <span class="cit-name-given-names">PW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jaaskelainen</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abdurahman</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ogola</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masika</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kivisto</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alburkat</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plyusnin</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Levanov</span>  <span class="cit-name-given-names">L</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Bombali Virus in Mops condylurus Bat, Kenya</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">25</span>(<span class="cit-issue">5</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Karan</span>  <span class="cit-name-given-names">LS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makenov</span>  <span class="cit-name-given-names">MT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korneev</span>  <span class="cit-name-given-names">MG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sacko</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boumbaly</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yakovlev</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kourouma</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bayandin</span>  <span class="cit-name-given-names">RB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gladysheva</span>  <span class="cit-name-given-names">AV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shipovalov</span>  <span class="cit-name-given-names">AV</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Bombali Virus in Mops condylurus Bats, Guinea</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">25</span>(<span class="cit-issue">9</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.43" data-doi="10.1126/science.1259657"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gire</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goba</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">KG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sealfon</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kanneh</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jalloh</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Momoh</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fullah</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dudas</span>  <span class="cit-name-given-names">G</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">345</span>(<span class="cit-issue">6202</span>):<span class="cit-fpage">1369</span>–<span class="cit-lpage">1372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1259657%26rft_id%253Dinfo%253Apmid%252F25214632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDUvNjIwMi8xMzY5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMTAvMjAyMC4wNC4wOC4wMzE5NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.44" data-doi="10.1016/S1473-3099(19)30124-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mbala-Kingebeni</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pratt</span>  <span class="cit-name-given-names">CB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wiley</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diagne</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makiala-Mandanda</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aziza</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Di Paola</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chitty</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diop</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ayouba</span>  <span class="cit-name-given-names">A</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">2018 Ebola virus disease outbreak in Equateur Province, Democratic Republic of the Congo: a retrospective genomic characterisation</span>. <abbr class="cit-jnl-abbrev">Lancet Infect Dis</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">19</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">641</span>–<span class="cit-lpage">647</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2BInfect%2BDis%26rft.volume%253D19%26rft.spage%253D641%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252819%252930124-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1473-3099(19)30124-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.45" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Standley</span>  <span class="cit-name-given-names">DM</span></span>: <span class="cit-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">30</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">772</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23329690&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317002300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">46.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.08.031963v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delaney</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soares</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seemann</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">SR</span></span>: <span class="cit-article-title">SNP-sites: rapid efficient extraction of SNPs from multi-FASTA alignments</span>. <abbr class="cit-jnl-abbrev">Microb Genom</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">2</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">e000056</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrob%2BGenom%26rft.volume%253D2%26rft.spage%253De000056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.08.031963v1.47"><div class="cit-metadata"><cite><span class="cit-source">R: A language and environment for statistical computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>. [<a href="https://www.R-project.org/">https://www.R-project.org/</a>]</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.08.031963v1.48" data-doi="10.1093/nar/gks596"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Untergasser</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cutcutache</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koressaar</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faircloth</span>  <span class="cit-name-given-names">BC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Remm</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rozen</span>  <span class="cit-name-given-names">SG</span></span>: <span class="cit-article-title">Primer3--new capabilities and interfaces</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-pub-date">2012</span>, <span class="cit-vol">40</span>(<span class="cit-issue">15</span>):<span class="cit-fpage">e115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks596%26rft_id%253Dinfo%253Apmid%252F22730293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks596&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22730293&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.08.031963.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.04.08.031963v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.04.08.031963v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.04.09.033597v1" title="NUDT21 links mitochondrial IPS-1 to RLR-containing stress granules and activates host antiviral defense" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.04.09.019695v1" title="Litter decomposition rates of biocrust-forming lichens are similar to that of vascular plants and are affected by warming in a semiarid grassland" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 10, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.04.08.031963v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.04.08.031963v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.04.08.031963v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.04.08.031963v1" class="active" data-icon-position="" data-hide-link-title="0">BioLaboro: A bioinformatics system for detecting molecular assay signature erosion and designing new assays in response to emerging and reemerging pathogens</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1238478" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-PifErNk1A_9K_GHVvZyL3COKP8QRND8dUt7jd_kfzUo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1238478" id="node1238478--2" data-pisa="biorxiv;2020.04.08.031963v1" data-pisa-master="biorxiv;2020.04.08.031963" data-seqnum="1238478" data-apath="/biorxiv/early/2020/04/10/2020.04.08.031963.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">BioLaboro: A bioinformatics system for detecting molecular assay signature erosion and designing new assays in response to emerging and reemerging pathogens</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Mitchell</span> <span class="nlm-surname">Holland</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Negrón</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Shane</span> <span class="nlm-surname">Mitchell</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Nate</span> <span class="nlm-surname">Dellinger</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Mychal</span> <span class="nlm-surname">Ivancich</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Tyler</span> <span class="nlm-surname">Barrus</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Sterling</span> <span class="nlm-surname">Thomas</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Katharine W.</span> <span class="nlm-surname">Jennings</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Bruce</span> <span class="nlm-surname">Goodwin</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Shanmuga</span> <span class="nlm-surname">Sozhamannan</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.08.031963; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.08.031963 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.04.08.031963v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031963v1&amp;title=BioLaboro%3A%20A%20bioinformatics%20system%20for%20detecting%20molecular%20assay%20signature%20erosion%20and%20designing%20new%20assays%20in%20response%20to%20emerging%20and%20reemerging%20pathogens" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031963v1&amp;title=BioLaboro%3A%20A%20bioinformatics%20system%20for%20detecting%20molecular%20assay%20signature%20erosion%20and%20designing%20new%20assays%20in%20response%20to%20emerging%20and%20reemerging%20pathogens" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031963v1&amp;text=BioLaboro%3A%20A%20bioinformatics%20system%20for%20detecting%20molecular%20assay%20signature%20erosion%20and%20designing%20new%20assays%20in%20response%20to%20emerging%20and%20reemerging%20pathogens" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031963v1&amp;title=BioLaboro%3A%20A%20bioinformatics%20system%20for%20detecting%20molecular%20assay%20signature%20erosion%20and%20designing%20new%20assays%20in%20response%20to%20emerging%20and%20reemerging%20pathogens" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031963v1&amp;t=BioLaboro%3A%20A%20bioinformatics%20system%20for%20detecting%20molecular%20assay%20signature%20erosion%20and%20designing%20new%20assays%20in%20response%20to%20emerging%20and%20reemerging%20pathogens" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031963v1&amp;title=BioLaboro%3A%20A%20bioinformatics%20system%20for%20detecting%20molecular%20assay%20signature%20erosion%20and%20designing%20new%20assays%20in%20response%20to%20emerging%20and%20reemerging%20pathogens" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031963v1&amp;title=BioLaboro%3A%20A%20bioinformatics%20system%20for%20detecting%20molecular%20assay%20signature%20erosion%20and%20designing%20new%20assays%20in%20response%20to%20emerging%20and%20reemerging%20pathogens" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1238478" id="node1238478--3" data-pisa="biorxiv;2020.04.08.031963v1" data-pisa-master="biorxiv;2020.04.08.031963" data-seqnum="1238478" data-apath="/biorxiv/early/2020/04/10/2020.04.08.031963.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">BioLaboro: A bioinformatics system for detecting molecular assay signature erosion and designing new assays in response to emerging and reemerging pathogens</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Mitchell</span> <span class="nlm-surname">Holland</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Negrón</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Shane</span> <span class="nlm-surname">Mitchell</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Nate</span> <span class="nlm-surname">Dellinger</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Mychal</span> <span class="nlm-surname">Ivancich</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Tyler</span> <span class="nlm-surname">Barrus</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Sterling</span> <span class="nlm-surname">Thomas</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Katharine W.</span> <span class="nlm-surname">Jennings</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Bruce</span> <span class="nlm-surname">Goodwin</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Shanmuga</span> <span class="nlm-surname">Sozhamannan</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.08.031963; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.08.031963 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=BioLaboro%3A%20A%20bioinformatics%20system%20for%20detecting%20molecular%20assay%20signature%20erosion%20and%20designing%20new%20assays%20in%20response%20to%20emerging%20and%20reemerging%20pathogens&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Holland%2C+Mitchell&amp;rft.au=Negr%C3%B3n%2C+Daniel&amp;rft.au=Mitchell%2C+Shane&amp;rft.au=Dellinger%2C+Nate&amp;rft.au=Ivancich%2C+Mychal&amp;rft.au=Barrus%2C+Tyler&amp;rft.au=Thomas%2C+Sterling&amp;rft.au=Jennings%2C+Katharine+W.&amp;rft.au=Goodwin%2C+Bruce&amp;rft.au=Sozhamannan%2C+Shanmuga"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1238478/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1238478/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1238478/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1238478/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1238478/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1238478/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1238478/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1238478/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1238478/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1238478/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1238478/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1238478/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031963v1&amp;count=horizontal&amp;via=&amp;text=BioLaboro%3A%20A%20bioinformatics%20system%20for%20detecting%20molecular%20assay%20signature%20erosion%20and%20designing%20new%20assays%20in%20response%20to%20emerging%20and%20reemerging%20pathogens&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031963v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.04.08.031963v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.04.08.031963v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
