<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/01/25/2020.01.24.919282.atom" />
<meta name="HW.pisa" content="biorxiv;2020.01.24.919282v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Under-the-radar dengue virus infections in natural populations of Aedes aegypti mosquitoes" />
<meta name="DC.Identifier" content="10.1101/2020.01.24.919282" />
<meta name="DC.Date" content="2020-01-25" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The incidence of locally acquired dengue infections increased during the last decade in the United States, compelling a sustained research effort on the dengue mosquito vector, Aedes aegypti, and its microbiome, which has been shown to influence virus transmission success. We examined the ‘metavirome’ of four populations of Ae. aegypti mosquitoes collected in 2016-2017 from Manatee County, Florida. Unexpectedly, we discovered that dengue virus serotype 4 (DENV4) was circulating in these mosquito populations, representing the first documented case of such a phenomenon in the absence of a local DENV4 human case in this county over a two-year period. We confirmed that all of the mosquito populations carried the same DENV4 strain, assembled its full genome, validated infection orthogonally by reverse transcriptase PCR, traced the virus origin, estimated the time period of its introduction to the Caribbean region, as well as explored the viral genetic signatures and mosquito-specific virome associations that potentially mediated DENV4 persistence in mosquitoes. We discuss the significance of prolonged maintenance of these DENV4 infections in Ae. aegypti that occurred in the absence of a DENV4 human index case in Manatee County with respect to the inability of current surveillance paradigms to detect mosquito vector infections prior to a potential local outbreak.

Importance Since 1999, dengue outbreaks in the continental United States (U.S.) involving local transmission have occurred episodically and only in Florida and Texas. In Florida, these episodes appear to be coincident with increased introductions of dengue virus into the region through human travel and migration from endemic countries. To date, the U.S. public health response to dengue outbreaks is largely reactive, and implementation of comprehensive arbovirus surveillance in advance of predictable transmission seasons, which would enable proactive preventative efforts, remains unsupported. The significance of our finding is that it is the first documented report of non-outbreak DENV4 transmission and maintenance within a local mosquito vector population in the continental U.S.in the absence of a human case during a two-year time period. Our data suggest that molecular surveillance of mosquito populations in high-risk, high tourism areas of the U.S., may allow for proactive, targeted vector control before potential arbovirus outbreaks." />
<meta name="DC.Contributor" content="Sean M. Boyles" />
<meta name="DC.Contributor" content="Carla N. Mavian" />
<meta name="DC.Contributor" content="Esteban Finol" />
<meta name="DC.Contributor" content="Maria Ukhanova" />
<meta name="DC.Contributor" content="Caroline J. Stephenson" />
<meta name="DC.Contributor" content="Gabriela Hamerlinck" />
<meta name="DC.Contributor" content="Seokyoung Kang" />
<meta name="DC.Contributor" content="Caleb Baumgartner" />
<meta name="DC.Contributor" content="Mary Geesey" />
<meta name="DC.Contributor" content="Israel Stinton" />
<meta name="DC.Contributor" content="Kate Williams" />
<meta name="DC.Contributor" content="Derrick K. Mathias" />
<meta name="DC.Contributor" content="Mattia Prosperi" />
<meta name="DC.Contributor" content="Volker Mai" />
<meta name="DC.Contributor" content="Marco Salemi" />
<meta name="DC.Contributor" content="Eva A. Buckner" />
<meta name="DC.Contributor" content="John A. Lednicky" />
<meta name="DC.Contributor" content="Adam R. Rivers" />
<meta name="DC.Contributor" content="Rhoel R. Dinglasan" />
<meta name="article:published_time" content="2020-01-25" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Under-the-radar dengue virus infections in natural populations of Aedes aegypti mosquitoes" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;The incidence of locally acquired dengue infections increased during the last decade in the United States, compelling a sustained research effort on the dengue mosquito vector, &lt;i&gt;Aedes aegypti,&lt;/i&gt; and its microbiome, which has been shown to influence virus transmission success. We examined the ‘metavirome’ of four populations of &lt;i&gt;Ae. aegypti&lt;/i&gt; mosquitoes collected in 2016-2017 from Manatee County, Florida. Unexpectedly, we discovered that dengue virus serotype 4 (DENV4) was circulating in these mosquito populations, representing the first documented case of such a phenomenon in the absence of a local DENV4 human case in this county over a two-year period. We confirmed that all of the mosquito populations carried the same DENV4 strain, assembled its full genome, validated infection orthogonally by reverse transcriptase PCR, traced the virus origin, estimated the time period of its introduction to the Caribbean region, as well as explored the viral genetic signatures and mosquito-specific virome associations that potentially mediated DENV4 persistence in mosquitoes. We discuss the significance of prolonged maintenance of these DENV4 infections in &lt;i&gt;Ae. aegypti&lt;/i&gt; that occurred in the absence of a DENV4 human index case in Manatee County with respect to the inability of current surveillance paradigms to detect mosquito vector infections prior to a potential local outbreak.&lt;/p&gt;&lt;h3&gt;Importance&lt;/h3&gt;
&lt;p&gt;Since 1999, dengue outbreaks in the continental United States (U.S.) involving local transmission have occurred episodically and only in Florida and Texas. In Florida, these episodes appear to be coincident with increased introductions of dengue virus into the region through human travel and migration from endemic countries. To date, the U.S. public health response to dengue outbreaks is largely reactive, and implementation of comprehensive arbovirus surveillance in advance of predictable transmission seasons, which would enable proactive preventative efforts, remains unsupported. The significance of our finding is that it is the first documented report of non-outbreak DENV4 transmission and maintenance within a local mosquito vector population in the continental U.S.in the absence of a human case during a two-year time period. Our data suggest that molecular surveillance of mosquito populations in high-risk, high tourism areas of the U.S., may allow for proactive, targeted vector control before potential arbovirus outbreaks.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.01.24.919282v1" />
<meta name="citation_id" content="2020.01.24.919282v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.01.24.919282v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.24.919282v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.24.919282v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.01.24.919282" />
<meta name="citation_num_pages" content="39" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.01.24.919282" />
<meta name="citation_author" content="Sean M. Boyles" />
<meta name="citation_author_institution" content="Emerging Pathogens Institute, University of Florida" />
<meta name="citation_author_institution" content="Department of Infectious Diseases &amp; Immunology, College of Veterinary Medicine, University of Florida" />
<meta name="citation_author_institution" content="CDC Southeastern Center of Excellence in Vector Borne Diseases" />
<meta name="citation_author" content="Carla N. Mavian" />
<meta name="citation_author_institution" content="Emerging Pathogens Institute, University of Florida" />
<meta name="citation_author_institution" content="Department of Pathology, College of Medicine, University of Florida" />
<meta name="citation_author" content="Esteban Finol" />
<meta name="citation_author_institution" content="Institute for Molecular Biology and Biophysics" />
<meta name="citation_author" content="Maria Ukhanova" />
<meta name="citation_author_institution" content="Emerging Pathogens Institute, University of Florida" />
<meta name="citation_author_institution" content="Department of Epidemiology, College of Public Health and Health Professions &amp; College of Medicine, University of Florida" />
<meta name="citation_author" content="Caroline J. Stephenson" />
<meta name="citation_author_institution" content="Emerging Pathogens Institute, University of Florida" />
<meta name="citation_author_institution" content="Department of Environmental and Global Health, College of Public Health and Health Professions, University of Florida" />
<meta name="citation_author_institution" content="CDC Southeastern Center of Excellence in Vector Borne Diseases" />
<meta name="citation_author" content="Gabriela Hamerlinck" />
<meta name="citation_author_institution" content="Emerging Pathogens Institute, University of Florida" />
<meta name="citation_author_institution" content="Department of Geography, College of Liberal Arts &amp; Sciences, University of Florida" />
<meta name="citation_author_institution" content="CDC Southeastern Center of Excellence in Vector Borne Diseases" />
<meta name="citation_author" content="Seokyoung Kang" />
<meta name="citation_author_institution" content="Emerging Pathogens Institute, University of Florida" />
<meta name="citation_author_institution" content="Department of Infectious Diseases &amp; Immunology, College of Veterinary Medicine, University of Florida" />
<meta name="citation_author_institution" content="CDC Southeastern Center of Excellence in Vector Borne Diseases" />
<meta name="citation_author" content="Caleb Baumgartner" />
<meta name="citation_author_institution" content="Manatee County Mosquito Control District" />
<meta name="citation_author" content="Mary Geesey" />
<meta name="citation_author_institution" content="Manatee County Mosquito Control District" />
<meta name="citation_author" content="Israel Stinton" />
<meta name="citation_author_institution" content="Manatee County Mosquito Control District" />
<meta name="citation_author" content="Kate Williams" />
<meta name="citation_author_institution" content="Manatee County Mosquito Control District" />
<meta name="citation_author" content="Derrick K. Mathias" />
<meta name="citation_author_institution" content="CDC Southeastern Center of Excellence in Vector Borne Diseases" />
<meta name="citation_author_institution" content="Florida Medical Entomology Laboratory, Institute of Food and Agricultural Sciences, University of Florida" />
<meta name="citation_author" content="Mattia Prosperi" />
<meta name="citation_author_institution" content="Department of Epidemiology, College of Public Health and Health Professions &amp; College of Medicine, University of Florida" />
<meta name="citation_author" content="Volker Mai" />
<meta name="citation_author_institution" content="Emerging Pathogens Institute, University of Florida" />
<meta name="citation_author_institution" content="Department of Epidemiology, College of Public Health and Health Professions &amp; College of Medicine, University of Florida" />
<meta name="citation_author" content="Marco Salemi" />
<meta name="citation_author_institution" content="Emerging Pathogens Institute, University of Florida" />
<meta name="citation_author_institution" content="Department of Pathology, College of Medicine, University of Florida" />
<meta name="citation_author" content="Eva A. Buckner" />
<meta name="citation_author_institution" content="CDC Southeastern Center of Excellence in Vector Borne Diseases" />
<meta name="citation_author_institution" content="Manatee County Mosquito Control District" />
<meta name="citation_author_institution" content="Florida Medical Entomology Laboratory, Institute of Food and Agricultural Sciences, University of Florida" />
<meta name="citation_author" content="John A. Lednicky" />
<meta name="citation_author_institution" content="Emerging Pathogens Institute, University of Florida" />
<meta name="citation_author_institution" content="Department of Environmental and Global Health, College of Public Health and Health Professions, University of Florida" />
<meta name="citation_author_institution" content="CDC Southeastern Center of Excellence in Vector Borne Diseases" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4069-0539" />
<meta name="citation_author" content="Adam R. Rivers" />
<meta name="citation_author_institution" content="CDC Southeastern Center of Excellence in Vector Borne Diseases" />
<meta name="citation_author_institution" content="Genomics and Bioinformatics Research Unit, Agricultural Research Service, United States Department of Agriculture" />
<meta name="citation_author" content="Rhoel R. Dinglasan" />
<meta name="citation_author_institution" content="Emerging Pathogens Institute, University of Florida" />
<meta name="citation_author_institution" content="Department of Infectious Diseases &amp; Immunology, College of Veterinary Medicine, University of Florida" />
<meta name="citation_author_institution" content="CDC Southeastern Center of Excellence in Vector Borne Diseases" />
<meta name="citation_reference" content="World Health Organization, Special Programme for Research and Training in Tropical Diseases. Report of the scientific working group meeting on dengue: Geneva, 1-5 October 2006. Geneva: World Health Organization on behalf of the Special Programme for Research and Training in Tropical Diseases, 2007." />
<meta name="citation_reference" content="Kraemer MU, Sinka ME, Duda KA, et al. The global compendium of Aedes aegypti and Ae. albopictus occurrence. Sci Data 2015; 2:150035." />
<meta name="citation_reference" content="Rey JR. Dengue in Florida (USA). Insects 2014; 5:991–1000." />
<meta name="citation_reference" content="Reiskind MH, Lounibos LP. Spatial and temporal patterns of abundance of Aedes aegypti L. (Stegomyia aegypti) and Aedes albopictus (Skuse) [Stegomyia albopictus (Skuse)] in southern Florida. Med Vet Entomol 2013; 27:421–9." />
<meta name="citation_reference" content="Romo H, Kenney JL, Blitvich BJ, Brault AC. Restriction of Zika virus infection and transmission in Aedes aegypti mediated by an insect-specific flavivirus. Emerg Microbes Infect 2018; 7:181." />
<meta name="citation_reference" content="Schultz MJ, Frydman HM, Connor JH. Dual Insect specific virus infection limits Arbovirus replication in Aedes mosquito cells. Virology 2018; 518:406–13." />
<meta name="citation_reference" content="Baidaliuk A, Miot EF, Lequime S, et al. Cell-Fusing Agent Virus Reduces Arbovirus Dissemination in Aedes aegypti Mosquitoes. J Virol 2019; 93." />
<meta name="citation_reference" content="Nurk S, Meleshko D, Korobeynikov A, Pevzner PA. metaSPAdes: a new versatile metagenomic assembler. Genome Res 2017; 27:824–34." />
<meta name="citation_reference" content="Buchfink B, Xie C, Huson DH. Fast and sensitive protein alignment using DIAMOND. Nat Methods 2015; 12:59–60." />
<meta name="citation_reference" content="Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods 2012; 9:357–9." />
<meta name="citation_reference" content="Li H, Handsaker B, Wysoker A, et al. The Sequence Alignment/Map format and SAMtools. Bioinformatics 2009; 25:2078–9." />
<meta name="citation_reference" content="Huson DH, Auch AF, Qi J, Schuster SC. MEGAN analysis of metagenomic data. Genome Res 2007; 17:377–86." />
<meta name="citation_reference" content="van den Boogaart GK, Tolosana-Delgado R, Bren M. (2018). compositions: Compositional Data Analysis. R package version 1.40–2. https://CRAN.R-project.org/package=compositions." />
<meta name="citation_reference" content="Fernandes AD, Macklaim JM, Linn TG, Reid G, Gloor GB. ANOVA-like differential expression (ALDEx) analysis for mixed population RNA-Seq. PLoS One 2013; 8:e67019." />
<meta name="citation_reference" content="Fernandes AD, Reid JN, Macklaim JM, McMurrough TA, Edgell DR, Gloor GB. Unifying the analysis of high-throughput sequencing datasets: characterizing RNA-seq, 16S rRNA gene sequencing and selective growth experiments by compositional data analysis. Microbiome 2014; 2:15." />
<meta name="citation_reference" content="Tcherepanov V, Ehlers A, Upton C. Genome Annotation Transfer Utility (GATU): rapid annotation of viral genomes using a closely related reference genome. BMC Genomics 2006; 7:150." />
<meta name="citation_reference" content="Pickett BE, Greer DS, Zhang Y, et al. Virus pathogen database and analysis resource (ViPR): a comprehensive bioinformatics database and analysis resource for the coronavirus research community. Viruses 2012; 4:3209–26." />
<meta name="citation_reference" content="Katoh K, Standley DM. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol Biol Evol 2013; 30:772–80." />
<meta name="citation_reference" content="Katoh K, Standley DM. A simple method to control over-alignment in the MAFFT multiple sequence alignment program. Bioinformatics 2016; 32:1933–42." />
<meta name="citation_reference" content="Nguyen LT, Schmidt HA, von Haeseler A, Minh BQ. IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies. Mol Biol Evol 2015; 32:268–74." />
<meta name="citation_reference" content="Schmidt HA, Strimmer K, Vingron M, von Haeseler A. TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel computing. Bioinformatics 2002; 18:502–4." />
<meta name="citation_reference" content="Trifinopoulos J, Nguyen LT, von Haeseler A, Minh BQ. W-IQ-TREE: a fast online phylogenetic tool for maximum likelihood analysis. Nucleic Acids Res 2016; 44:W232–5." />
<meta name="citation_reference" content="Minh BQ, Nguyen MA, von Haeseler A. Ultrafast approximation for phylogenetic bootstrap. Mol Biol Evol 2013; 30:1188–95." />
<meta name="citation_reference" content="Martin DP, Murrell B, Golden M, Khoosal A, Muhire B. RDP4: Detection and analysis of recombination patterns in virus genomes. Virus Evol 2015; 1:vev003." />
<meta name="citation_reference" content="Rambaut A, Lam TT, Max Carvalho L, Pybus OG. Exploring the temporal structure of heterochronous sequences using TempEst (formerly Path-O-Gen). Virus Evol 2016; 2:vew007." />
<meta name="citation_reference" content="Drummond AJ, Suchard MA, Xie D, Rambaut A. Bayesian phylogenetics with BEAUti and the BEAST 1.7. Mol Biol Evol 2012; 29:1969–73." />
<meta name="citation_reference" content="Drummond AJ, Rambaut A. BEAST: Bayesian evolutionary analysis by sampling trees. BMC Evol Biol 2007; 7:214." />
<meta name="citation_reference" content="Hasegawa M, Kishino H, Yano T. Dating of the human-ape splitting by a molecular clock of mitochondrial DNA. J Mol Evol 1985; 22:160–74." />
<meta name="citation_reference" content="Hall MD, Woolhouse ME, Rambaut A. The effects of sampling strategy on the quality of reconstruction of viral population dynamics using Bayesian skyline family coalescent methods: A simulation study. Virus Evol 2016; 2:vew003." />
<meta name="citation_reference" content="Baele G, Lemey P, Bedford T, Rambaut A, Suchard MA, Alekseyenko AV. Improving the accuracy of demographic and molecular clock model comparison while accommodating phylogenetic uncertainty. Mol Biol Evol 2012; 29:2157–67." />
<meta name="citation_reference" content="Xie W, Lewis PO, Fan Y, Kuo L, Chen MH. Improving marginal likelihood estimation for Bayesian phylogenetic model selection. Syst Biol 2011; 60:150–60." />
<meta name="citation_reference" content="Yu G, Smith DK, Zhu H, Guan Y, Lam TT. ggtree: an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data. Methods Ecol Evol 2017; 8:28–36." />
<meta name="citation_reference" content="Pond SL, Frost SD, Muse SV. HyPhy: hypothesis testing using phylogenies. Bioinformatics 2005; 21:676–9." />
<meta name="citation_reference" content="Pond SL, Frost SD. Datamonkey: rapid detection of selective pressure on individual sites of codon alignments. Bioinformatics 2005; 21:2531–3." />
<meta name="citation_reference" content="Murrell B, Moola S, Mabona A, et al. FUBAR: a fast, unconstrained bayesian approximation for inferring selection. Mol Biol Evol 2013; 30:1196–205." />
<meta name="citation_reference" content="Murrell B, Wertheim JO, Moola S, Weighill T, Scheffler K, Kosakovsky Pond SL. Detecting individual sites subject to episodic diversifying selection. PLoS Genet 2012; 8:e1002764." />
<meta name="citation_reference" content="Steinway SN, Dannenfelser R, Laucius CD, Hayes JE, Nayak S. JCoDA: a tool for detecting evolutionary selection. BMC Bioinformatics 2010; 11:284." />
<meta name="citation_reference" content="Kosakovsky Pond SL, Frost SD. Not so different after all: a comparison of methods for detecting amino acid sites under selection. Mol Biol Evol 2005; 22:1208–22." />
<meta name="citation_reference" content="Weaver S, Shank SD, Spielman SJ, Li M, Muse SV, Kosakovsky Pond SL. Datamonkey 2.0: a modern web application for characterizing selective and other evolutionary processes. Mol Biol Evol 2018." />
<meta name="citation_reference" content="Bennett SN, Holmes EC, Chirivella M, et al. Selection-driven evolution of emergent dengue virus. Mol Biol Evol 2003; 20:1650–8." />
<meta name="citation_reference" content="Gebhard LG, Filomatori CV, Gamarnik AV. Functional RNA elements in the dengue virus genome. Viruses 2011; 3:1739–56." />
<meta name="citation_reference" content="Finol E, Ooi EE. Evolution of Subgenomic RNA Shapes Dengue Virus Adaptation and Epidemiological Fitness. iScience 2019; 16:94–105." />
<meta name="citation_reference" content="Parry R, Asgari S. Aedes Anphevirus: an Insect-Specific Virus Distributed Worldwide in Aedes aegypti Mosquitoes That Has Complex Interplays with Wolbachia and Dengue Virus Infection in Cells. J Virol 2018; 92.45." />
<meta name="citation_reference" content="Zhang G, Asad S, Khromykh AA, Asgari S. Cell fusing agent virus and dengue virus mutually interact in Aedes aegypti cell lines. Sci Rep 2017; 7:6935." />
<meta name="citation_reference" content="Ten Bosch QA, Clapham HE, Lambrechts L, et al. Contributions from the silent majority dominate dengue virus transmission. PLoS Pathog 2018; 14:e1006965." />
<meta name="citation_reference" content="Vaughn DW, Green S, Kalayanarooj S, et al. Dengue viremia titer, antibody response pattern, and virus serotype correlate with disease severity. J Infect Dis 2000; 181:2–9." />
<meta name="citation_reference" content="Buckner EA, Alto BW, Lounibos LP. Vertical transmission of Key West dengue-1 virus by Aedes aegypti and Aedes albopictus (Diptera: Culicidae) mosquitoes from Florida. J Med Entomol 2013; 50:1291–7." />
<meta name="citation_reference" content="Mavian C, Dulcey M, Munoz O, Salemi M, Vittor AY, Capua I. Islands as Hotspots for Emerging Mosquito-Borne Viruses: A One-Health Perspective. Viruses 2018; 11." />
<meta name="citation_reference" content="Blohm G, A Elbadry M, Mavian C, et al. Mayaro as a Caribbean traveler: Evidence for multiple introductions and transmission of the virus into Haiti. Int J Infect Dis 2019." />
<meta name="citation_reference" content="White SK, Mavian C, Salemi M, et al. A new “American” subgroup of African-lineage Chikungunya virus detected in and isolated from mosquitoes collected in Haiti, 2016. PLoS One 2018; 13:e0196857." />
<meta name="twitter:title" content="Under-the-radar dengue virus infections in natural populations of Aedes aegypti mosquitoes" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The incidence of locally acquired dengue infections increased during the last decade in the United States, compelling a sustained research effort on the dengue mosquito vector, Aedes aegypti, and its microbiome, which has been shown to influence virus transmission success. We examined the ‘metavirome’ of four populations of Ae. aegypti mosquitoes collected in 2016-2017 from Manatee County, Florida. Unexpectedly, we discovered that dengue virus serotype 4 (DENV4) was circulating in these mosquito populations, representing the first documented case of such a phenomenon in the absence of a local DENV4 human case in this county over a two-year period. We confirmed that all of the mosquito populations carried the same DENV4 strain, assembled its full genome, validated infection orthogonally by reverse transcriptase PCR, traced the virus origin, estimated the time period of its introduction to the Caribbean region, as well as explored the viral genetic signatures and mosquito-specific virome associations that potentially mediated DENV4 persistence in mosquitoes. We discuss the significance of prolonged maintenance of these DENV4 infections in Ae. aegypti that occurred in the absence of a DENV4 human index case in Manatee County with respect to the inability of current surveillance paradigms to detect mosquito vector infections prior to a potential local outbreak.

Importance Since 1999, dengue outbreaks in the continental United States (U.S.) involving local transmission have occurred episodically and only in Florida and Texas. In Florida, these episodes appear to be coincident with increased introductions of dengue virus into the region through human travel and migration from endemic countries. To date, the U.S. public health response to dengue outbreaks is largely reactive, and implementation of comprehensive arbovirus surveillance in advance of predictable transmission seasons, which would enable proactive preventative efforts, remains unsupported. The significance of our finding is that it is the first documented report of non-outbreak DENV4 transmission and maintenance within a local mosquito vector population in the continental U.S.in the absence of a human case during a two-year time period. Our data suggest that molecular surveillance of mosquito populations in high-risk, high tourism areas of the U.S., may allow for proactive, targeted vector control before potential arbovirus outbreaks." />
<meta name="og-title" property="og:title" content="Under-the-radar dengue virus infections in natural populations of Aedes aegypti mosquitoes" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.01.24.919282v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The incidence of locally acquired dengue infections increased during the last decade in the United States, compelling a sustained research effort on the dengue mosquito vector, Aedes aegypti, and its microbiome, which has been shown to influence virus transmission success. We examined the ‘metavirome’ of four populations of Ae. aegypti mosquitoes collected in 2016-2017 from Manatee County, Florida. Unexpectedly, we discovered that dengue virus serotype 4 (DENV4) was circulating in these mosquito populations, representing the first documented case of such a phenomenon in the absence of a local DENV4 human case in this county over a two-year period. We confirmed that all of the mosquito populations carried the same DENV4 strain, assembled its full genome, validated infection orthogonally by reverse transcriptase PCR, traced the virus origin, estimated the time period of its introduction to the Caribbean region, as well as explored the viral genetic signatures and mosquito-specific virome associations that potentially mediated DENV4 persistence in mosquitoes. We discuss the significance of prolonged maintenance of these DENV4 infections in Ae. aegypti that occurred in the absence of a DENV4 human index case in Manatee County with respect to the inability of current surveillance paradigms to detect mosquito vector infections prior to a potential local outbreak.

Importance Since 1999, dengue outbreaks in the continental United States (U.S.) involving local transmission have occurred episodically and only in Florida and Texas. In Florida, these episodes appear to be coincident with increased introductions of dengue virus into the region through human travel and migration from endemic countries. To date, the U.S. public health response to dengue outbreaks is largely reactive, and implementation of comprehensive arbovirus surveillance in advance of predictable transmission seasons, which would enable proactive preventative efforts, remains unsupported. The significance of our finding is that it is the first documented report of non-outbreak DENV4 transmission and maintenance within a local mosquito vector population in the continental U.S.in the absence of a human case during a two-year time period. Our data suggest that molecular surveillance of mosquito populations in high-risk, high tourism areas of the U.S., may allow for proactive, targeted vector control before potential arbovirus outbreaks." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-01-25" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.01.24.919282v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.01.24.919282v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.01.24.919282v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.01.24.919282v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1121132" />
    <title>Under-the-radar dengue virus infections in natural populations of Aedes aegypti mosquitoes | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"FBUzj6YjlOVMvz342rF-5qBAt8w4uUt0c4MNwkk7WaQ","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1121132","apath":"\/biorxiv\/early\/2020\/01\/25\/2020.01.24.919282.atom","pisa":"biorxiv;2020.01.24.919282v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.01.24.919282v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.01.24.919282v1","title":"Under-the-radar dengue virus infections in natural populations of Aedes aegypti mosquitoes","identifier":"node\/1121132"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HdGblqcGaSbKlxUXBUy39Lc5f7hhquSLWvOC6Of0YwIFME8inLRMTuLi98PAlDcDQ\\\\\\\/Ydr3Z4l4sjX+Fog\\\\\\\/9xQTyxFr5gD8IyImKHHKL4MkUAVE5NB2Z49SuenTx7leEqEB4FCOF02RBTFWiwYGZCl12kTWSTP2i5DOoeeWfxJyjyircQ7Y+SmfSq1mOJ+SJl+ehtUd7cF6wJuXJvxGYIhmbR3j42ZjBZBsdQ89Vqoumblib+dvrbRuzHbidaU1Q3MdwDCJU9O7X4V1Prxvj4tQyfQb9r+247jmrJi\\\\\\\/kI3eJhHhwsZXDdgrzHAwVn7eA82qd4ZonCWT61nZTfh71ZirCRaqeO8XNyRCA+ClGDNMSJqP04KSC0y74wbK1+UzXY6qUnHvo58DxQ5oGFlWnAy7YkoucOwJH8QSegO3y6Wo66MYE2e\\\\\\\/wSEO5xKrL8gOqUFpErwTMyd5RA8KZZZ\\\\\\\/jGnGHc0GljqkCGjHZ3mw1ykbjEBzW8vjhxsPPbyWHLJSsmV1gEVGfSh\\\\\\\/CvMKoi+cDzaYMHpTu148FdcYCBqKJVfKXUGoEk9+7Ljp4lcxPw0ynya0D3h\\\\\\\/wYAE9cZcLOAelo8p\\\\\\\/gz0\\\\\\\/CwGGNx+ZOocARicl2lxwcuvqRuFHG29iSjMUBjh2gBrLs1w1fZE7OQeMDTDO3j0FTJXXy5Pyfj8mnC4SD3N4Z03UdroPqLobTFIHCdZKdBBpuZTh\\\\\\\/\\\\\\\/W1yubfwCNSUH1vFaWregGz\\\\\\\/O0rHdK5BVrK+IRDXPGGzbGK57swyk9iejV9VtQ\\\\\\\/ZwpmmIQEp21hUL3yhAWz7BxcO+qtLUIwJR8cA9PBjSO6ri8xaZhXX8ZeztOzq+B53hqVbfUoC5At3deMloewwuKfVk02p9TbmjZf1dxWK0AA5eS1xoFelDqcYcDhadz4+Q6SXL7gWCn419o4ajWgPR1UDhctM+yaxC8dv7poourKNfogHn\\\\\\\/bcP4jNx3S75YtNrouqRdpEUtOOlFgwrL\\\\\\\/WBoxZF0hcLHTQnlfIMvNs9PsRCT0r+37YGqWdPT8GSvLYY0RHaZJjjKglJ3JUa6kssvJm3w6iCB+u34kWu32TywzcXH+EaUW91jrBoVsclqTWCO6WcLRAERvcFHEKFOawt81\\\\\\\/byAPf2BCMuQNswG6dJWODppcNh6UTsfxw\\\\\\\/UaPhB2wqEjxjWNVJPH6yxc8TJt6Hcj5lCaG\\\\\\\/h\\\\\\\/x+HUOjZRZJ2iehRjbKWnhC6CDNe7sMZeTKDWW\\\\\\\/3A1tta4XuQOiea3ZQO8xHxnNXjTvaKcaA\\\\\\\/FBTOlKWeBSajeO8Yk2B3odmXKyskhQNd1g07zCVOxkD5YmlNpCIQosuq3scOKhDa2RwoV74VeC6y\\\u0022,\\\u0022iv\\\u0022:\\\u0022EjCoWXY\\\\\\\/kkP\\\\\\\/jEa892PelQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022767b1a2a8634a18e8bb7447a2a2ead2afa48562e14bd08399c3264cdc31a35f8\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.01.24.919282v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1121132 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.01.24.919282v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nIVAej0xwrOClzS5SlUnhv7y7WFmO70wpj58AUR80OA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1121132" id="node1121132" data-pisa="biorxiv;2020.01.24.919282v1" data-pisa-master="biorxiv;2020.01.24.919282" data-apath="/biorxiv/early/2020/01/25/2020.01.24.919282.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Under-the-radar dengue virus infections in natural populations of <em>Aedes aegypti</em> mosquitoes</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Sean M.</span> <span class="nlm-surname">Boyles</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Carla N.</span> <span class="nlm-surname">Mavian</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Esteban</span> <span class="nlm-surname">Finol</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Ukhanova</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Caroline J.</span> <span class="nlm-surname">Stephenson</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Gabriela</span> <span class="nlm-surname">Hamerlinck</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Seokyoung</span> <span class="nlm-surname">Kang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Caleb</span> <span class="nlm-surname">Baumgartner</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Mary</span> <span class="nlm-surname">Geesey</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Israel</span> <span class="nlm-surname">Stinton</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Kate</span> <span class="nlm-surname">Williams</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Derrick K.</span> <span class="nlm-surname">Mathias</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Mattia</span> <span class="nlm-surname">Prosperi</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Volker</span> <span class="nlm-surname">Mai</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Marco</span> <span class="nlm-surname">Salemi</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Eva A.</span> <span class="nlm-surname">Buckner</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="16"><a href="http://orcid.org/0000-0003-4069-0539" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">John A.</span> <span class="nlm-surname">Lednicky</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Adam R.</span> <span class="nlm-surname">Rivers</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Rhoel R.</span> <span class="nlm-surname">Dinglasan</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.01.24.919282 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1121132" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sean M. Boyles </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Emerging Pathogens Institute, University of Florida</span>, Gainesville, Florida, 32611, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Infectious Diseases &#38; Immunology, College of Veterinary Medicine, University of Florida</span>, Gainesville, Florida 32611, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>CDC Southeastern Center of Excellence in Vector Borne Diseases</span>, Gainesville, Florida 32611, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BM.%2BBoyles%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boyles%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BM.%2BBoyles%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Carla N. Mavian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Emerging Pathogens Institute, University of Florida</span>, Gainesville, Florida, 32611, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Pathology, College of Medicine, University of Florida</span>, Gainesville, Florida 32611, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carla%2BN.%2BMavian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mavian%20CN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarla%2BN.%2BMavian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Esteban Finol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Institute for Molecular Biology and Biophysics</span>, ETH Zurich, Zurich 8093, <span class='nlm-country'>Switzerland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Esteban%2BFinol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Finol%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEsteban%2BFinol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Maria Ukhanova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Emerging Pathogens Institute, University of Florida</span>, Gainesville, Florida, 32611, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Epidemiology, College of Public Health and Health Professions &#38; College of Medicine, University of Florida</span>, Gainesville, Florida, 32610, <span class='nlm-country'>United States</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BUkhanova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ukhanova%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BUkhanova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Caroline J. Stephenson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Emerging Pathogens Institute, University of Florida</span>, Gainesville, Florida, 32611, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Environmental and Global Health, College of Public Health and Health Professions, University of Florida</span>, Gainesville, Florida, 32610, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>CDC Southeastern Center of Excellence in Vector Borne Diseases</span>, Gainesville, Florida 32611, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Caroline%2BJ.%2BStephenson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stephenson%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACaroline%2BJ.%2BStephenson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gabriela Hamerlinck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Emerging Pathogens Institute, University of Florida</span>, Gainesville, Florida, 32611, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Department of Geography, College of Liberal Arts &#38; Sciences, University of Florida</span>, Gainesville, Florida 32610, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>CDC Southeastern Center of Excellence in Vector Borne Diseases</span>, Gainesville, Florida 32611, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gabriela%2BHamerlinck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hamerlinck%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGabriela%2BHamerlinck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Seokyoung Kang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Emerging Pathogens Institute, University of Florida</span>, Gainesville, Florida, 32611, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Infectious Diseases &#38; Immunology, College of Veterinary Medicine, University of Florida</span>, Gainesville, Florida 32611, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>CDC Southeastern Center of Excellence in Vector Borne Diseases</span>, Gainesville, Florida 32611, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seokyoung%2BKang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kang%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASeokyoung%2BKang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Caleb Baumgartner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>9</span><span class='nlm-institution'>Manatee County Mosquito Control District</span>, Palmetto, Florida 34221, <span class='nlm-country'>United States</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Caleb%2BBaumgartner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baumgartner%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACaleb%2BBaumgartner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Mary Geesey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>9</span><span class='nlm-institution'>Manatee County Mosquito Control District</span>, Palmetto, Florida 34221, <span class='nlm-country'>United States</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BGeesey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Geesey%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMary%2BGeesey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Israel Stinton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>9</span><span class='nlm-institution'>Manatee County Mosquito Control District</span>, Palmetto, Florida 34221, <span class='nlm-country'>United States</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Israel%2BStinton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stinton%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIsrael%2BStinton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Kate Williams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>9</span><span class='nlm-institution'>Manatee County Mosquito Control District</span>, Palmetto, Florida 34221, <span class='nlm-country'>United States</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kate%2BWilliams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Williams%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKate%2BWilliams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Derrick K. Mathias </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>CDC Southeastern Center of Excellence in Vector Borne Diseases</span>, Gainesville, Florida 32611, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>10</span><span class='nlm-institution'>Florida Medical Entomology Laboratory, Institute of Food and Agricultural Sciences, University of Florida</span>, Vero Beach, Florida 32962, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Derrick%2BK.%2BMathias%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mathias%20DK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADerrick%2BK.%2BMathias%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Mattia Prosperi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Epidemiology, College of Public Health and Health Professions &#38; College of Medicine, University of Florida</span>, Gainesville, Florida, 32610, <span class='nlm-country'>United States</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mattia%2BProsperi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Prosperi%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMattia%2BProsperi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Volker Mai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Emerging Pathogens Institute, University of Florida</span>, Gainesville, Florida, 32611, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Epidemiology, College of Public Health and Health Professions &#38; College of Medicine, University of Florida</span>, Gainesville, Florida, 32610, <span class='nlm-country'>United States</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Volker%2BMai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mai%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVolker%2BMai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Marco Salemi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Emerging Pathogens Institute, University of Florida</span>, Gainesville, Florida, 32611, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Pathology, College of Medicine, University of Florida</span>, Gainesville, Florida 32611, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marco%2BSalemi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salemi%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarco%2BSalemi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Eva A. Buckner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>CDC Southeastern Center of Excellence in Vector Borne Diseases</span>, Gainesville, Florida 32611, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>9</span><span class='nlm-institution'>Manatee County Mosquito Control District</span>, Palmetto, Florida 34221, <span class='nlm-country'>United States</span>.</div><div class='author-affiliation'><span class='nlm-sup'>10</span><span class='nlm-institution'>Florida Medical Entomology Laboratory, Institute of Food and Agricultural Sciences, University of Florida</span>, Vero Beach, Florida 32962, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eva%2BA.%2BBuckner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckner%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEva%2BA.%2BBuckner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">John A. Lednicky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Emerging Pathogens Institute, University of Florida</span>, Gainesville, Florida, 32611, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Environmental and Global Health, College of Public Health and Health Professions, University of Florida</span>, Gainesville, Florida, 32610, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>CDC Southeastern Center of Excellence in Vector Borne Diseases</span>, Gainesville, Florida 32611, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BA.%2BLednicky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lednicky%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BA.%2BLednicky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4069-0539" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John A. Lednicky</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Adam R. Rivers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>CDC Southeastern Center of Excellence in Vector Borne Diseases</span>, Gainesville, Florida 32611, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>11</span><span class='nlm-institution'>Genomics and Bioinformatics Research Unit, Agricultural Research Service, United States Department of Agriculture</span>, Gainesville, Florida 32608, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BR.%2BRivers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rivers%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdam%2BR.%2BRivers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-18"><div class="author-tooltip-name">Rhoel R. Dinglasan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Emerging Pathogens Institute, University of Florida</span>, Gainesville, Florida, 32611, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Infectious Diseases &#38; Immunology, College of Veterinary Medicine, University of Florida</span>, Gainesville, Florida 32611, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>CDC Southeastern Center of Excellence in Vector Borne Diseases</span>, Gainesville, Florida 32611, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rhoel%2BR.%2BDinglasan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dinglasan%20RR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARhoel%2BR.%2BDinglasan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.01.24.919282v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1121132" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1121132/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.24.919282v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1121132" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1121132/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.24.919282v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1121132" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1121132/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.24.919282v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1121132" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1121132/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.01.24.919282v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1121132" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1121132/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-8">The incidence of locally acquired dengue infections increased during the last decade in the United States, compelling a sustained research effort on the dengue mosquito vector, <em>Aedes aegypti,</em> and its microbiome, which has been shown to influence virus transmission success. We examined the ‘metavirome’ of four populations of <em>Ae. aegypti</em> mosquitoes collected in 2016-2017 from Manatee County, Florida. Unexpectedly, we discovered that dengue virus serotype 4 (DENV4) was circulating in these mosquito populations, representing the first documented case of such a phenomenon in the absence of a local DENV4 human case in this county over a two-year period. We confirmed that all of the mosquito populations carried the same DENV4 strain, assembled its full genome, validated infection orthogonally by reverse transcriptase PCR, traced the virus origin, estimated the time period of its introduction to the Caribbean region, as well as explored the viral genetic signatures and mosquito-specific virome associations that potentially mediated DENV4 persistence in mosquitoes. We discuss the significance of prolonged maintenance of these DENV4 infections in <em>Ae. aegypti</em> that occurred in the absence of a DENV4 human index case in Manatee County with respect to the inability of current surveillance paradigms to detect mosquito vector infections prior to a potential local outbreak.</p><div id="sec-1" class="subsection"><p id="p-9"><strong>Importance</strong> Since 1999, dengue outbreaks in the continental United States (U.S.) involving local transmission have occurred episodically and only in Florida and Texas. In Florida, these episodes appear to be coincident with increased introductions of dengue virus into the region through human travel and migration from endemic countries. To date, the U.S. public health response to dengue outbreaks is largely reactive, and implementation of comprehensive arbovirus surveillance in advance of predictable transmission seasons, which would enable proactive preventative efforts, remains unsupported. The significance of our finding is that it is the first documented report of non-outbreak DENV4 transmission and maintenance within a local mosquito vector population in the continental U.S.in the absence of a human case during a two-year time period. Our data suggest that molecular surveillance of mosquito populations in high-risk, high tourism areas of the U.S., may allow for proactive, targeted vector control before potential arbovirus outbreaks.</p></div></div><div class="section" id="sec-2"><h2 class="">Introduction</h2><p id="p-10">Approximately 40% of the globe is at risk of infection by flaviviruses, such as dengue virus (DENV): an enveloped, single-stranded RNA virus transmitted primarily by <em>Aedes aegypti</em> mosquitoes [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. Since severe disease from DENV infections can manifest as dengue hemorrhagic fever/dengue shock syndrome [<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>], DENV establishment in the continental United States is a major concern for public health agencies. In the USA, Florida has experienced increases in local DENV transmission since 2009 [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>], driven in part by human and pathogen movement. <em>Ae. aegypti</em> is endemic throughout subtropical Florida and the vector population has resurged recently, following its near displacement by <em>Ae. albopictus</em> [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. Autochthonous DENV infection occurs sporadically, primarily in Southern Florida with limited local cases elsewhere in the state [<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>]. In 2019, 16 cases of locally acquired DENV were reported for the state, including an area along the West-Central Florida Gulf Coast.</p><p id="p-11">Recently, reports have indicated that certain insect-specific viruses (ISVs) can negatively impact or enhance arbovirus (including DENV) infections in insect cells [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>] and mosquitoes [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>], respectively. Although the impacts of many ISVs on arboviral competence have yet to be determined, the evidence to date clearly indicates that the mosquito virome cannot be safely ignored and likely influences the risk of autochthonous DENV transmission once the virus is introduced into an area. Therefore, we conducted a metaviromic study of F<sub>1</sub> (first-generation, lab-raised mosquitoes from wild parents) <em>Ae. aegypti</em> adult females collected as eggs from ovitraps in 2016-2017 from Manatee County to assess the potential risk of flavivirus transmission outside of Southern Florida. Although no indexed human case of DENV4 was reported during 2016-2017 in the county, we detected and sequenced DENV4, which may have been maintained vertically for at least one generation (but potentially more) in these <em>Ae. aegypti</em> mosquito populations along Florida’s Gulf Coast. We followed up this unexpected finding with genetic analyses to determine the DENV4 strain’s likely location of origin, assess the timeframe of virus introduction, and investigate strain-specific mutations that may have enabled adaptation to and/or persistence within local mosquito populations.</p></div><div class="section" id="sec-3"><h2 class="">Methods</h2><div id="sec-4" class="subsection"><h3>Mosquito sample preparation and viral RNASeq</h3><p id="p-12"><em>Ae. aegypti</em> eggs were collected in ovitraps in the summers of 2016-2017 (May 15, 2016, and June 19, 2017) from four Manatee County sites (<strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1a</a></strong>). To avoid cross contamination of mosquito viromes, each year eggs from each site were hatched independently in distilled water, reared to adulthood, speciated and then frozen. Female abdomens were pooled (N= 20/pool) separately for the four collection sites for a total of eight individual pools. Total RNA was extracted using the AllPrep DNA/RNA Mini Kit (Qiagen), and rRNA depleted with the NEBNext rRNA Depletion Kit (New England BioLabs). The NEBNext Ultra II Directional RNA Library Prep Kit (New England BioLabs) was used to prepare shotgun metagenomics libraries. Reverse-transcribed RNA libraries were sequenced using a HiSeq 3000 (Illumina) instrument in 2x101 run mode. The data were deposited into the NCBI Sequence Read Archive and Biosample archive under BioProject PRJNA547758.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Metaviromic analysis of Aedes aegypti mosquito populations from Manatee County, Florida. (a) Locations of ovitraps in four different locations in Manatee County: Palmetto, Cortez, Ana Maria Island and Longboat Key. (b) The relative abundance of reads identified to come from RNA viruses in the 8 metagenomes. The proportion of the sub-composition is summarized at the species level for most viruses; however, some viruses were classified at higher levels if species could not be determined by the lowest common ancestor method." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-665957934" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Metaviromic analysis of &lt;em&gt;Aedes aegypti&lt;/em&gt; mosquito populations from Manatee County, Florida.&lt;/span&gt; &lt;strong&gt;(a)&lt;/strong&gt; Locations of ovitraps in four different locations in Manatee County: Palmetto, Cortez, Ana Maria Island and Longboat Key. &lt;strong&gt;(b)&lt;/strong&gt; The relative abundance of reads identified to come from RNA viruses in the 8 metagenomes. The proportion of the sub-composition is summarized at the species level for most viruses; however, some viruses were classified at higher levels if species could not be determined by the lowest common ancestor method.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F1.medium.gif" width="345" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F1.medium.gif" width="345" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Metaviromic analysis of <em>Aedes aegypti</em> mosquito populations from Manatee County, Florida.</span><p id="p-13" class="first-child"><strong>(a)</strong> Locations of ovitraps in four different locations in Manatee County: Palmetto, Cortez, Ana Maria Island and Longboat Key. <strong>(b)</strong> The relative abundance of reads identified to come from RNA viruses in the 8 metagenomes. The proportion of the sub-composition is summarized at the species level for most viruses; however, some viruses were classified at higher levels if species could not be determined by the lowest common ancestor method.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Initial assembly and metavirome analysis</h3><p id="p-14">BBduk (version 37.75; <a href="https://sourceforge.net/projects/bbmap/">https://sourceforge.net/projects/bbmap/</a>) was used to trim adaptor sequences and remove contaminants. <em>Ae. aegypti</em> sequences were removed using BBsplit (<a href="https://sourceforge.net/projects/bbmap/">https://sourceforge.net/projects/bbmap/</a>) against the <em>Ae. aegypti</em> Liverpool genome (AaegL5.1). Non-mosquito reads were assembled using Spades (3.11.1) in metagenomics mode [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. For each contig, local similarity search in protein space was run using Diamond (0.9.17) [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>] against the NCBI NR database. Reads were mapped against assemblies using Bowtie (2.3.4.1) [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>], then sorted/indexed using Samtools (1.4.1) [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. Megan 6 [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>] was used to assign contigs and read counts to the lowest common ancestor (LCA) and to view viral contigs. To estimate microbial community abundance, Diamond (0.9.17) [<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>] was used to search reads against the NCBI NR database, Megan 6 [<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>] was used to assign read counts to the LCA, and R (3.6.0) package Compositions [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>] (1.40-2) was used to create a sub-composition of RNA (<strong><a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1b</a></strong>). Compositional count data from the Megan [<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>] LCA classification was assessed by ALDEx2 [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>] to estimate the statistical significance of the change in DENV4 reads from 2016 to 2017. ALDEx2 [<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>] uses a Dirichlet multinomial Monte Carlo simulation to estimate the variance of the centered log ratio (CLR) values for taxa amongst the reads. Using the variance of the CLR, ALDEx2 [<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a>] computes P-values using Welch’s t-test and returns an effect size (CLR/variance) for the estimate. For a determination of the statistical significance of the observed decrease in CFAV reads from 2016-2017, a linear regression fitted to the CLRs of the Anna Maria and Cortez site DENV4 reads in 2016-2017 was utilized to yield an R<sup>2</sup> value and a P-value to describe the trend.</p></div><div id="sec-6" class="subsection"><h3>DENV4 refinement and genome-closing assembly</h3><p id="p-15">Two contigs covering most of the genome with a small gap were obtained. To create a closed genome, a dataset of genomes for DENV1-4 (NC_001477.1, NC_001474.2, NC_001475.2, NC_002640.1) and the two assembled contigs were used. We selected reads sharing a 31-mer with the dataset using BBduk (<a href="https://sourceforge.net/projects/bbmap/">https://sourceforge.net/projects/bbmap/</a>), followed by assembly with Spades in <em>meta</em> mode [<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>] and classification using Diamond [<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">9</a>] for a complete DENV4 genome. Read-mapping with Bowtie [<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>] revealed incorrect bases near the 3’ end, which were manually corrected. The genome was annotated using the Genome Annotation Transfer Utility [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>] from the Virus Pathogen Database and Analysis Resource (ViPR) [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>].</p></div><div id="sec-7" class="subsection"><h3>Phylogenetic and Molecular Clock analyses</h3><p id="p-16">Two hundred thirty-four DENV4 genome sequences from GenBank (<strong>Table S1</strong>) were aligned using MAFFT version 7.407 [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>] with the L-INS-I method [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]. IQ-TREE software [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>] was used to evaluate phylogenetic signal in the genomes by likelihood mapping [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>] and to infer maximum likelihood (ML) phylogeny based on the best-fit model according to the Bayesian Information Criterion (BIC) [<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>]. Statistical robustness for internal branching order was assessed by Ultrafast Bootstrap (BB) Approximation (2,000 replicates), and strong statistical support was defined as BB&gt;90% [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>].</p><p id="p-17">To estimate when DENV4 entered Florida, we used 145 strains including all isolates from the Americas, related Asian and African isolates, and randomly reduced oversampled Brazilian isolates. The strains in this dataset were not recombinant, as assessed by scanning the alignments for possible recombination points using the RDP, GENECONV, MaxChi, CHIMAERA, and 3Seq algorithms implemented in the RDP4 software (available from <a href="http://web.cbio.uct.ac.za/~darren/rdp.html">http://web.cbio.uct.ac.za/~darren/rdp.html</a>) [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. Correlation between root-to-tip genetic divergence and date of sampling was conducted [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>] to assess clock signal before Bayesian phylodynamic analysis. Time-scaled trees were reconstructed using the Bayesian phylodynamic inference framework in BEAST v.1.8.4 [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. Markov Chain Monte Carlo (MCMC) samplers were run until 200/250 million generations to ensure Markov chain mixing, assessed by calculating the Effective Sampling Size (ESS) of parameter estimates. The HKY substitution model [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>] was used with empirical base frequencies and gamma distribution of site-specific rate heterogeneity. The fit of strict vs. relaxed uncorrelated molecular clock models, and constant size vs. Bayesian Skyline Plot [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>] demographic models were tested. Marginal likelihood estimates (MLE) for Bayesian model testing were obtained using path sampling (PS) and stepping-stone sampling (SS) methods [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]. The best model was of a strict clock and constant demographic size. The maximum clade credibility tree was inferred from the posterior distribution of trees using TreeAnnotator specifying a burn-in of 10% and median node heights, then edited graphically in FigTree v1.4.4 (<a href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtree/</a>), alongside ggtree available in R [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>].</p></div><div id="sec-8" class="subsection"><h3>Single-nucleotide variation analyses</h3><p id="p-18">The viral RNA sequencing reads were mapped onto the complete genome of seven DENV4 strains. These strains represent all the known DENV4 lineages (accession numbers are provided in <strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3c</a></strong>). We also mapped the reads onto the assembled Manatee DENV4 full genome. The read mapping was performed in the Geneious platform (Geneious Prime® version 2019.2.1) using the “map to reference” function under standard settings (Mapper: Geneious; Sensitivity: Highest Sensitivity/Slow; Fine tuning: Iterate up to 5 times; no trim before mapping). The Single nucleotide variation quantification was performed in the same platform using the “find Variation/SNV” function under default settings.</p></div><div id="sec-9" class="subsection"><h3>DENV4 Genetic Analyses</h3><p id="p-19">From the 234 DENV4 genome alignment, sequences corresponding to the NS2A gene were extracted to investigate selection pressure and mutations that potentially influenced adaptation to and/or persistence in mosquito populations. Comparative selection and mutation analyses revealed NS2A as a relatively strong region of potential selection for the Manatee County genome. HyPhy algorithms were used to estimate non-synonymous (dN) to synonymous (dS) codon substitution rate ratios (ω), with ω&lt;1 indicating purifying/negative selection and ω&gt;1 indicating diversifying/positive selection [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]. Fast, unconstrained Bayesian approximation (FUBAR) [<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>] was used for inferring pervasive selection, and the mixed effects model of evolution (MEME) [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>] to identify episodic selection. Sites were considered to have experienced diversifying/positive or purifying/negative selective pressure based on posterior probability (PP) &gt; 0.90 for FUBAR, and likelihood ratio test ≤ 0.05 for MEME.</p><p id="p-20">To elucidate influential mutations in the Manatee DENV4 genome that potentially enabled persistence in the local mosquito population, a dN/dS analysis of the Manatee DENV4 against the relatively close, but geographically distant 1981 Senegalese DENV4 (MF004387.1) was conducted using JCoDA [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>] with default settings, a 10-bp sliding window and a jump value of 5. To further assess the selective pressure throughout coding sequences in the DENV4 lineage that established transmission in Manatee County, we implemented a Single-Likelihood Ancestor Counting (SLAC) method [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>] on the DataMonkey 2.0 web application [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>]. It combines maximum-likelihood (ML) and counting approaches to infer nonsynonymous (dN) and synonymous (dS) substitution rates on a site-by-site basis for the different DENV4 coding alignments and corresponding DENV4 phylogeny. The measurements were performed on different alignments that included all strains, only genotype II strains, only clade IIa or IIb strains or only strains that are closely related to the DENV4 Manatee strain (multiple DENV4 coding sequence alignments are available as a Mendeley dataset). NS2A and 2K peptide genes were individually aligned and inspected between closely related DENV4 strains (1994 Haitian (JF262782.1), 2014 Haitian #1 (KP140942.1), 2014 Haitian #2 (KT276273.1), 2015 Haitian (MK514144.1), and 1981 Senegalese (MF004387.1) genomes) and the Manatee DENV4 for mutations to identify signals of adaptation of Manatee DENV4 to Floridian <em>Ae. aegypti</em>.</p></div></div><div class="section" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>DENV4 and ISVs in Ae. aegypti mosquitoes from Manatee County, Florida</h3><p id="p-21">Our metaviromic analysis of female <em>Ae. aegypti</em> mosquitoes detected DENV4 alongside several ISVs across four sites in 2016 and only Anna Maria and Cortez sites in 2017 (<strong><a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1a</a></strong>). A full DENV4 genome (MN192436) was constructed with an overall genome coverage of ∼11X across the reads (<strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a></strong>). We observed that the 2017 DENV4 signal was much lower than 2016 for Anna Maria and Cortez (<strong>Fig. S1</strong>) and although Palmetto had the highest proportion of 2016 reads, this signal was virtually absent in 2017. To confirm DENV4 infection, we amplified and confirmed by direct sequencing the NS2A DENV4 amplicon for 2016 Longboat and Palmetto mosquito samples. Cumulatively, the drop in DENV4 relative to the metavirome from 2016-2017 was statistically significant (effect size=-2.026; P=0.035).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mapping of RNASeq reads on the DENV4 genome. Coverage plots for DENV4 genome readings are shown from top to bottom graph panels. Coverage values across the genome for collection site/year combinations. Coverage is depicted on each y-axis and amino acid position on the x-axes. The smoothed central lines on the graphs indicate median values." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-665957934" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Mapping of RNASeq reads on the DENV4 genome.&lt;/span&gt; Coverage plots for DENV4 genome readings are shown from top to bottom graph panels. Coverage values across the genome for collection site/year combinations. Coverage is depicted on each y-axis and amino acid position on the x-axes. The smoothed central lines on the graphs indicate median values.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F2.medium.gif" width="355" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F2.medium.gif" width="355" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Mapping of RNASeq reads on the DENV4 genome.</span><p id="p-22" class="first-child">Coverage plots for DENV4 genome readings are shown from top to bottom graph panels. Coverage values across the genome for collection site/year combinations. Coverage is depicted on each y-axis and amino acid position on the x-axes. The smoothed central lines on the graphs indicate median values.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">The RNA metavirome profile of the Manatee <em>Ae. aegypti</em> indicated an abundance of <em>Partitiviridae, Anphevirus,</em> Whidbey virus, and cell fusing agent virus (CFAV). <em>Partitiviridae</em> are known to primarily infect plants, protozoa, and fungi, but all the abundant groups in the metavirome have previously been detected in mosquitoes. We noted that the highest levels of CFAV (Anna Maria and Cortez sites) in 2016 were associated with DENV4 persistence into 2017 (P=0.07109; R<sup>2</sup>= 0.7943). Additionally, <em>Anphevirus</em> signals were notably abundant in the Palmetto samples in 2016 and 2017, coincident with DENV4 signal loss in Palmetto in 2017.</p></div><div id="sec-12" class="subsection"><h3>DENV4 Phylogenetic and Molecular Clock Analyses</h3><p id="p-24">After analyzing the metavirome, we investigated the genome of the DENV4 strain to determine its likely source and assess the potential timeframe of introduction into Florida. Our first analysis confirmed the phylogenetic signal and absence of nucleotide substitution saturation (<strong>Fig. S2a-b</strong>). We subsequently explored Manatee County DENV4’s phylogeny with a 234-genome DENV4 dataset constructed from GenBank sequences (<strong>Table S1</strong>) by maximum likelihood (ML) phylogenetic inference (<strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3a</a></strong>). The ML phylogeny showed three clades: two Asian clades, and one American clade with two Senegalese strains (MF00438, KF907503) and one Thai (KM190936) at the base (<strong><a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3a</a></strong>). Manatee DENV4 can be classified as DENV4 genotype IIb. The DENV4 genome obtained in Florida most closely clustered with two Haitian isolates from 2014 (KT276273, KP140942) and a cluster of Puerto Rican isolates (<strong><a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3a</a></strong>). Further back, a Haitian isolate (JF262782) collected 20 years earlier also clustered with the Manatee-associated clade (<strong><a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3a</a></strong>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic and phylodynamic analyses of Manatee DENV4. (a) Maximum likelihood phylogenetic analysis of DENV4 full genome sequences. ML tree was obtained using IQ-TREE [20] software, diamonds indicate strong statistical support along the branches defined by ultrafast bootstrap &gt;90. Tips are labeled and colored based on country of origin. (b) Bayesian phylodynamic reconstruction of DENV4 genotype IIb strains. The Maximum Clade Credibility time-scaled phylogenetic maximum clade credibility tree inferred using relaxed clock and constant demographic priors implemented in BEAST v1.8.4. Circles represent branches supported by posterior probability &gt;0.90. Tips are colored based on location of origin. Labeled are nodes A (time of the most recent common ancestor [tMRCA] 2010), B (tMRCA 1992), and C (tMRCA 1981) on the branches. (c) SNVs/read per collection site/year combination of mosquitoes with significant detection by viral RNASeq in comparison to various reference genomes shown as a distance matrix is shown. The total numbers of SNVs were normalized by the total numbers of reads from each sample. Cell values refer to the SNV/read ratios of every sample (column) as compared to every representative sequence (rows). Cells are color-coded in the matrix as red = 0.0 SNV/read; white = 1.5 SNV/read; and blue = &gt; 3 SNV/read." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-665957934" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Phylogenetic and phylodynamic analyses of Manatee DENV4.&lt;/span&gt; &lt;strong&gt;(a)&lt;/strong&gt; Maximum likelihood phylogenetic analysis of DENV4 full genome sequences. ML tree was obtained using IQ-TREE [20] software, diamonds indicate strong statistical support along the branches defined by ultrafast bootstrap &gt;90. Tips are labeled and colored based on country of origin. &lt;strong&gt;(b)&lt;/strong&gt; Bayesian phylodynamic reconstruction of DENV4 genotype IIb strains. The Maximum Clade Credibility time-scaled phylogenetic maximum clade credibility tree inferred using relaxed clock and constant demographic priors implemented in BEAST v1.8.4. Circles represent branches supported by posterior probability &gt;0.90. Tips are colored based on location of origin. Labeled are nodes A (time of the most recent common ancestor [tMRCA] 2010), B (tMRCA 1992), and C (tMRCA 1981) on the branches. &lt;strong&gt;(c)&lt;/strong&gt; SNVs/read per collection site/year combination of mosquitoes with significant detection by viral RNASeq in comparison to various reference genomes shown as a distance matrix is shown. The total numbers of SNVs were normalized by the total numbers of reads from each sample. Cell values refer to the SNV/read ratios of every sample (column) as compared to every representative sequence (rows). Cells are color-coded in the matrix as red = 0.0 SNV/read; white = 1.5 SNV/read; and blue = &gt; 3 SNV/read.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F3.medium.gif" width="304" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F3.medium.gif" width="304" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Phylogenetic and phylodynamic analyses of Manatee DENV4.</span><p id="p-25" class="first-child"><strong>(a)</strong> Maximum likelihood phylogenetic analysis of DENV4 full genome sequences. ML tree was obtained using IQ-TREE [<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">20</a>] software, diamonds indicate strong statistical support along the branches defined by ultrafast bootstrap &gt;90. Tips are labeled and colored based on country of origin. <strong>(b)</strong> Bayesian phylodynamic reconstruction of DENV4 genotype IIb strains. The Maximum Clade Credibility time-scaled phylogenetic maximum clade credibility tree inferred using relaxed clock and constant demographic priors implemented in BEAST v1.8.4. Circles represent branches supported by posterior probability &gt;0.90. Tips are colored based on location of origin. Labeled are nodes A (time of the most recent common ancestor [tMRCA] 2010), B (tMRCA 1992), and C (tMRCA 1981) on the branches. <strong>(c)</strong> SNVs/read per collection site/year combination of mosquitoes with significant detection by viral RNASeq in comparison to various reference genomes shown as a distance matrix is shown. The total numbers of SNVs were normalized by the total numbers of reads from each sample. Cell values refer to the SNV/read ratios of every sample (column) as compared to every representative sequence (rows). Cells are color-coded in the matrix as red = 0.0 SNV/read; white = 1.5 SNV/read; and blue = &gt; 3 SNV/read.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">To estimate the most recent common ancestor (MRCA) for DENV4 entry into Manatee County, Florida, as well as date divergence of the strain with Haitian isolates, we performed a molecular clock analysis using a Bayesian evolutionary framework [<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>] on a reduced dataset including only the “Americas clade.” We first assessed the phylogenetic signal and the absence of nucleotide substitution saturation (<strong>Fig. S2c-d</strong>) and then the temporal signal alone (<strong>Fig. S3</strong>). In the maximum clade credibility (MCC) tree, Manatee DENV4 clustered with the Haitian isolates from 2014 (Node A posterior probability [PP] &gt; 0.9) (<strong><a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3b</a></strong>). The MCC phylogeny showed that the time of the MRCA (tMRCA) for the DENV4 Manatee isolate and Haitian isolates was 2010 (Node A in <strong><a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3b</a></strong>). This 95% high posterior density interval for this tMRCA suggests that DENV4 may have entered Manatee County sometime between 2006-2013. For Node B (<strong><a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig. 3b</a></strong>), the tMRCA of 1992 with a 95% HPD interval 1901-1994 indicated that Floridian and 2014 Haitian strains diverged from the 1994 Haitian DENV4 (JF262782), almost a decade before its arrival to Florida. However, strain divergence may have occurred in Haiti and was not necessarily precipitated by its introduction to Manatee County. Therefore, the introduction timeframe could be more recent than the estimated tMRCA.</p></div><div id="sec-13" class="subsection"><h3>DENV4 SNVs/Read Analyses</h3><p id="p-27">Next, we examined the Manatee DENV4 genome sequences to compare strain variation between years and to identify mutations unique to the strain that potentially enabled local adaptation to and/or persistence in local mosquito populations. Following the MCC phylogenetic analysis, site-specific reads from mosquito populations in Manatee County were analyzed for single-nucleotide variations (SNVs) by the number of SNVs/read against the Manatee consensus genome and other global DENV4 genomes (<strong><a id="xref-fig-3-9" class="xref-fig" href="#F3">Fig. 3c</a></strong>). SNV/read values showed only 22 SNVs across the 11,650-nucleotide Manatee County genome against all reads. SNVs were more substantial per read in the other DENV4 genomes. This indicates the likely persistence of a single strain of DENV4 in Manatee County during 2016-2017 transmission seasons.</p></div><div id="sec-14" class="subsection"><h3>Signatures of Manatee DENV4 adaptation</h3><p id="p-28">We then explored selective pressures on the Manatee County DENV4 strain’s coding sequence that may be functionally important with respect to transmission and persistence of DENV4 in Floridian aegypti. The DENV4 genome has 5’ and 3’ untranslated regions (UTRs) flanking eight protein-coding genes (non-structural [NS] protein 1, NS2A, NS2B, NS3, NS4A, the 2K peptide, NS4B, and NS5) (<strong><a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4a</a></strong>). The protein-coding regions of Manatee DENV4 were compared to four Haitian DENV4 genomes from 1994-2015 and a 1981 Senegalese DENV4 genome. These were analyzed for all amino acid substitutions between strains, and a dN/dS analysis was conducted comparing the Senegalese DENV4 genome with Manatee DENV4 (<strong><a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4a</a></strong> and <strong>Table S2</strong>). The highest proportions of amino acid substitution were seen in NS2A and the 2K peptide; simultaneously, the highest dN/dS values occurred for the NS2A gene, to a point of weak positive selection (dN/dS &gt; 1) that covered a V1238T mutation discussed further herein. We then calculated dN/dS ratios for DENV4 altogether, genotype II, genotype IIa, and genotype IIb with all sequences available, as well as within the Haiti-Florida clade and the Haiti-Florida-Puerto-Rico clades (<strong><a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4b</a></strong>). Purifying selection, which occurs when non-synonymous mutations are deleterious, dominated, but we found weaker purifying selection in NS2A and 2K peptide genes, correlating to the Manatee-to-Senegal dN/dS analysis conducted previously. Values of dN/dS for these genes increased relative to those for flanking genes for genotype IIb and Caribbean/Florida-specific groups as well (<strong><a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4b</a></strong>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="DENV4 amino acid analyses. (a) A dN/dS analysis conducted for the whole coding region of DENV4 Manatee vs. the Senegalese genome from 1981 (MF004387.1). The analysis was conducted utilizing full genome coding sequences in JCoDA using a sliding window analysis with a window size of 10. A small genome schematic is placed below the graph to its scale. Line colors approaching red from orange lie at higher values on the graph to indicate highe dN/dS values. (b) Using all available sequences, a dN/dS comparison was conducted to calculate mean ratio values within DENV4 overall, DENV4 genotype II, DENV4 genotype IIb, DENV4 FL-American-Caribbean clades, and DENV4-Florida-Caribbean-specific clades (respectively moving along Nodes C, B, and A from fig. 3B). Adjacent to the IIb-containing comparison, a comparison between all available DENV4 genome sequences, DENV4 genotype II, and DENV4 genotype IIa is depicted. (c) A comparative amino acid sequence alignment of Manatee County (MN192436), Puerto Rican (AH011951.2), Haitian 1994 (JF262782.1), Haitian 2014 #1 (KP140942.1), Haitian 2014 #2 (KT276273.1), Haitian 2015 (MK514144.1), and 1981 Senegalese (MF004387.1) genome sequences for the NS2A region sequenced in Puerto Rican isolate genomes by Bennet et al. [40]. Amino acid positions are numbered at the top of the figure. Key amino acid changes defining the 1998 DENV4 Puerto Rican outbreak in the NS2A gene are highlighted with boxes. (d) A comparison of the 2K peptide (colored in grey) sequence between the Manatee County (MN192436), 1994 Haitian (JF262782.1), Haitian 2014 #1 (KP140942.1), Haitian 2014 #2 (KT276273.1), Haitian 2015 (MK514144.1), and 1981 Senegalese (MF004387.1) genomes. Uncolored portions of the sequences correlate to portions of NS4A and NS4B." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-665957934" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;DENV4 amino acid analyses.&lt;/span&gt; &lt;strong&gt;(a)&lt;/strong&gt; A dN/dS analysis conducted for the whole coding region of DENV4 Manatee vs. the Senegalese genome from 1981 (MF004387.1). The analysis was conducted utilizing full genome coding sequences in JCoDA using a sliding window analysis with a window size of 10. A small genome schematic is placed below the graph to its scale. Line colors approaching red from orange lie at higher values on the graph to indicate highe dN/dS values. &lt;strong&gt;(b)&lt;/strong&gt; Using all available sequences, a dN/dS comparison was conducted to calculate mean ratio values within DENV4 overall, DENV4 genotype II, DENV4 genotype IIb, DENV4 FL-American-Caribbean clades, and DENV4-Florida-Caribbean-specific clades (respectively moving along Nodes C, B, and A from fig. 3B). Adjacent to the IIb-containing comparison, a comparison between all available DENV4 genome sequences, DENV4 genotype II, and DENV4 genotype IIa is depicted. &lt;strong&gt;(c)&lt;/strong&gt; A comparative amino acid sequence alignment of Manatee County (MN192436), Puerto Rican (AH011951.2), Haitian 1994 (JF262782.1), Haitian 2014 #1 (KP140942.1), Haitian 2014 #2 (KT276273.1), Haitian 2015 (MK514144.1), and 1981 Senegalese (MF004387.1) genome sequences for the NS2A region sequenced in Puerto Rican isolate genomes by Bennet et al. [40]. Amino acid positions are numbered at the top of the figure. Key amino acid changes defining the 1998 DENV4 Puerto Rican outbreak in the NS2A gene are highlighted with boxes. &lt;strong&gt;(d)&lt;/strong&gt; A comparison of the 2K peptide (colored in grey) sequence between the Manatee County (MN192436), 1994 Haitian (JF262782.1), Haitian 2014 #1 (KP140942.1), Haitian 2014 #2 (KT276273.1), Haitian 2015 (MK514144.1), and 1981 Senegalese (MF004387.1) genomes. Uncolored portions of the sequences correlate to portions of NS4A and NS4B.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F4.medium.gif" width="440" height="384"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F4.medium.gif" width="440" height="384"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/25/2020.01.24.919282/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">DENV4 amino acid analyses.</span><p id="p-29" class="first-child"><strong>(a)</strong> A dN/dS analysis conducted for the whole coding region of DENV4 Manatee vs. the Senegalese genome from 1981 (MF004387.1). The analysis was conducted utilizing full genome coding sequences in JCoDA using a sliding window analysis with a window size of 10. A small genome schematic is placed below the graph to its scale. Line colors approaching red from orange lie at higher values on the graph to indicate highe dN/dS values. <strong>(b)</strong> Using all available sequences, a dN/dS comparison was conducted to calculate mean ratio values within DENV4 overall, DENV4 genotype II, DENV4 genotype IIb, DENV4 FL-American-Caribbean clades, and DENV4-Florida-Caribbean-specific clades (respectively moving along Nodes C, B, and A from <a id="xref-fig-3-10" class="xref-fig" href="#F3">fig. 3B</a>). Adjacent to the IIb-containing comparison, a comparison between all available DENV4 genome sequences, DENV4 genotype II, and DENV4 genotype IIa is depicted. <strong>(c)</strong> A comparative amino acid sequence alignment of Manatee County (MN192436), Puerto Rican (AH011951.2), Haitian 1994 (JF262782.1), Haitian 2014 #1 (KP140942.1), Haitian 2014 #2 (KT276273.1), Haitian 2015 (MK514144.1), and 1981 Senegalese (MF004387.1) genome sequences for the NS2A region sequenced in Puerto Rican isolate genomes by Bennet et al. [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>]. Amino acid positions are numbered at the top of the figure. Key amino acid changes defining the 1998 DENV4 Puerto Rican outbreak in the NS2A gene are highlighted with boxes. <strong>(d)</strong> A comparison of the 2K peptide (colored in grey) sequence between the Manatee County (MN192436), 1994 Haitian (JF262782.1), Haitian 2014 #1 (KP140942.1), Haitian 2014 #2 (KT276273.1), Haitian 2015 (MK514144.1), and 1981 Senegalese (MF004387.1) genomes. Uncolored portions of the sequences correlate to portions of NS4A and NS4B.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Next, we further analyzed coding sequences in specific regions of the genome to investigate specific mutations that may have mediated Manatee DENV4 Floridian entry and persistence. The NS2A gene was analyzed in an alignment between Manatee (MN192436), 1994 Haitian (JF262782.1), 2014 Haitian #1 (KP140942.1), 2014 Haitian #2 (KT276273.1), 2015 Haitian (MK514144.1), and 1981 Senegalese (MF004387.1) genomes and a partial DENV4 genome (AH011951.2, Puerto Rico, 1998), with the analysis targeting three mutations that defined the 1998 DENV4 Puerto Rican outbreak [<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>] (<strong><a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4c</a></strong>). The Manatee DENV4 sequence shares these key mutations with 1998 Puerto Rico, 2014 Haiti #1, and 2015 Haiti genomes. Conversely, the 1981 Senegal sequence and the oldest Haitian sequence from 1994 lack these mutations. In a selective pressure analysis utilizing the aforementioned 234-genome assembly, we observed strong background purifying selection with 143 sites that were found under episodic negative/purifying selection within the NS2A gene. Episodic diversifying/positive selection (evolutionarily preferred non-synonymous mutation) was detected in two sites corresponding to amino acids 1,238 and 1,333, both residues localized to transmembrane segments of the protein. This makes V1238T a mutation of note with the previous NS2A-associated analysis, detected in different analyses as a point of possible positive selection. The 2K peptide was next analyzed against the four Haitian genomes and the Senegalese genome from the first NS2A-specific analysis (<strong><a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4d</a></strong>), and we observed that it had the second highest general rate of non-synonymous mutations and had a peak of weaker purifying selection (<strong><a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig. 4a</a></strong>). There was only one non-synonymous mutation among the six genomes, which is significant considering the size of the 2K peptide. This was a T2232A mutation present solely in the Manatee DENV4 sequence.</p></div><div id="sec-15" class="subsection"><h3>DENV4 3’-UTR sequence and secondary structure analysis</h3><p id="p-31">To complete our genomic analysis of Manatee DENV4, we examined the 3’ UTR, as this region and its derivative subgenomic RNA have been implicated in epidemiologic and transmission fitness. Although the DENV4 3’ UTR lacks one of the two flaviviral nuclease-resistant RNAs (fNRs) in Domain I as compared to other DENV 3’ UTRs (3’ UTRs of other DENV serotypes [<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a>–<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a>]), DENV4 has the same conserved secondary structures in its domain II and III: two dumbbells (DB1 and DB2), and a 3′ end stem–loop (3′ SL) (<strong>Fig. S4</strong>). The 3’-UTR, through structural conformations, can affect viral replication in hosts [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>]. We noted several transition substitutions in the DENV4 IIb lineage prior its arrival to Florida (Node B in <strong><a id="xref-fig-3-11" class="xref-fig" href="#F3">Fig. 3b</a></strong> and <strong>Fig. S4b</strong>). Most of these mapped to either the highly variable region (HVR) or the adenine-rich segments that space functional RNA elements in DENV 3’ UTRs [<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>]. The U10318C substitution in fNR2 (fNR1 present only in other DENV serotypes) and the G10588A substitution on the 3’ SL mapped to base-pairing positions. However, these mutations have occurred in both directions in other lineages, suggesting they don’t imply fitness costs. Conversely, Floridian DENV4 underwent a rare transversion (A10478U) in a conserved position in DB2. This substitution favours formation of a new base-pair in DB2 structure. Additionally, an insertion (10467A) occurred in the adenine-rich segment upstream of DB2; this insertion is common for all lineages.</p></div></div><div class="section" id="sec-16"><h2 class="">Discussion</h2><p id="p-32">Our unbiased metavirome analysis of <em>Ae. aegypti</em> from Manatee County has revealed new insight into human arboviruses and ISV maintenance in a state prone to autochthonous flavivirus transmission. The observed drop in DENV4 relative to the mosquito virome (ISVs) between 2016-2017 was statistically significant (P=0.035), suggesting that the ISVs influence persistence of DENV4 in site-specific mosquito populations within the surveyed area. <em>Anphevirus</em> has been shown to reduce DENV viral titers <em>in vitro</em> during coinfections [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]. The abundance of Palmetto <em>Anphevirus</em> alongside the observed Palmetto 2016-2017 DENV4 reduction is consistent with this and suggests that these viruses and their respective abundance or relative proportions within a mosquito impact DENV4 prevalence in the vector population. The role of natural infections by insect-specific flaviruses on the proliferation of pathogenic arboviruses carried by different mosquito vector species is equivocal. A mosquito-specific flavivirus we detected known as cell fusing agent virus (CFAV) is of particular interest. Co-infection studies <em>in vitro</em> with DENV2 and CFAV result in enhanced proliferation in both [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>]. Following this notion, the presence of CFAV in the same mosquito populations as DENV4 may improve viral dissemination and maintenance in mosquitoes. The observed correlation between persistence of DENV4 infection into 2017 in Anna Maria and Cortez mosquitoes with CFAV abundance in 2016 (<strong><a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1b</a></strong>) appears to operate in parallel to the research conducted by Zhang et al. showing the enhanced replication of the two viruses [<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a>]. An important caveat is that Zhang et al.’s research was conducted <em>in vitro</em>. Conversely, Baidaliuk, et al. demonstrated <em>in vivo</em> amplification-restrictive interaction between CFAV and DENV1 [<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>]. How DENV4 genotype X mosquito genotype X CFAV genotype interactions ultimately influence the vector competence of Floridian <em>Ae. aegypti</em> mosquitoes remains to be determined. The observed metavirome patterns sets the stage for follow-up studies to characterize the precise nature of ISV-DENV-mosquito interactions viz. vector competence.</p><p id="p-33">The absence of an index human DENV4 case does not preclude the possibility that DENV4 was transmitted locally. Up to 88% of primary DENV infections are asymptomatic, with DENV4 being widely understood to cause primarily subclinical infections [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>]. Importantly, clinically inapparent infections could contribute to 84% of DENV transmission events through mosquitoes [<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a>], so the threat of local transmission cannot be ruled out. However, it is noteworthy that DENV4 was detected in adult female mosquitoes reared from wild-captured eggs, implicating transovarial transmission (TOT) in local <em>Ae. aegypti</em> as has been shown for DENV1 in Key West, Florida [<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>]. However, since the DENV4 signal measured in 2017 was lower than in 2016, with two sites losing DENV4 prevalence, TOT alone may have been insufficient to maintain DENV4 from 2016-2017. Furthermore, we suspect that despite Manatee DENV4’s divergence from Haitian strains sometime between 2006-2013, it likely did not enter Manatee County until 2014 or after, given its similarity to the 2014-2015 Haitian DENV4 isolates and the fact that TOT is an inefficient process. Tertiary mechanisms, beyond ISV composition profile and TOT, could include inapparent human-mosquito infection cycles during the summer transmission (mosquito) season, which may have also contributed to DENV4 persistence in Manatee County aegypti. The exact mechanisms of maintenance in mosquitoes and proof of local transmission are difficult to elucidate at this juncture, considering all mosquito samples were processed for RNASeq and RT-PCR (i.e., no live virus can be isolated). Importantly, a comprehensive serosurvey with subsequent confirmation by gold-standard neutralization assay of the population from the four sample collection sites was not possible within the estimated mean half-life of detectable anti-DENV4 virion IgM or IgG. This limitation was unavoidable since (i) the complete viral genome assembly and orthogonal confirmation occurred more than two years following the initial mosquito collections, and (ii) there are significant confounders and logistical obstacles working with transient worker and migrant communities in the sampled area (well outside of the current scope of the study). However, the complete assembly and persistence over two years of an individual strain of DENV4, which is supported by results from orthogonal analytical approaches, remains provocative and reveals an unappreciated ecological process for DENV4 transmission in a non-endemic setting.</p><p id="p-34">Tracking and predicting arbovirus movement and introduction into the United States, especially into Florida, can potentially lead to proactive efforts for increased monitoring and vector control at critical points of introduction into the state. DENV4 has been reported throughout the Caribbean, especially in Puerto Rico, Haiti and more recently in Cuba [<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>]. Florida has the largest populations of Puerto Rican, Haitian and Cuban origin and descent in the U.S., and there are ongoing efforts to develop effectively “sentinel” surveillance programs that can prepare Florida to deal with potential local arbovirus transmission. As expected, our analysis suggests a Caribbean origin for the Manatee isolate due to movements of DENV4 into Florida from Haiti, and preceding this, into Haiti from Puerto Rico. These results concur with previous findings depicting the Caribbean as a hotspot for arboviral spread in the Americas [<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">48</a>–<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>]. Diversifying selective pressure in the NS2A gene and the 2K peptide (<strong><a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig. 4a-b</a></strong>) experienced by American/Caribbean DENV4 may have contributed to the fixation of mutations driving the adaptation of DENV4 to environmental/vector conditions in these areas. NS2A mutations that characterized the 1998 DENV4 outbreak in Puerto Rico [<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">40</a>] are conserved between the Manatee, Puerto Rican, and two Haitian (JF262782.1 and KT276273.1) genomes (<strong><a id="xref-fig-4-9" class="xref-fig" href="#F4">Fig. 4c</a></strong>). The 1981 Senegalese strain, the closest-clustering strain to the Manatee strain isolated outside the Americas (<strong><a id="xref-fig-3-12" class="xref-fig" href="#F3">Fig. 3a-b</a></strong>), shares none of these mutations with Manatee DENV4. An in-depth understanding of how putative “hallmark” mutations in arboviruses can lead to increased local aegypti mosquito infections is lacking and compels further study.</p><p id="p-35">We observed the expected 15-nucleotide deletion (Δ15) in the Manatee DENV4 3’ UTR (<strong>Fig. S4</strong>) that is present across all circulating DENV4 strains but absent from the extinct genotype I DENV4 lineage (GQ868594_Philippines_1956). Since the Δ15 deletion maps to the HVR, it does not alter the required secondary structures for sfRNA production. However, the HVR is an adenylate-rich unfolded spacer with poor sequence conservation—where no reliable secondary structure can be predicted, as our previous analyses suggested [<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">42</a>]. It has been speculated that these spacers favor the correct folding of adjacent functional structured RNA elements. The deletion might change the rate of folding of the downstream functional structured RNA and thus alter sfRNA production levels. Clearly, a closer molecular exploration of the exact role of this Δ15 deletion is needed.</p><p id="p-36">The potential implications of our findings are profound; especially considering that arboviral surveillance of mosquito populations during the extended Florida mosquito season (April-October) is limited. To our knowledge, this is the first reported characterization of a DENV4 infection in native mosquito populations in Florida in the absence of an index human case across two years in a specific county. These data highlight the importance of knowing when and where arboviruses are introduced and point to the potential benefit of surveilling local mosquito populations for arbovirus infections prior to an outbreak. Given the increasing number of travel-related arbovirus introductions into Florida alone and the risk of local establishment in the state, we expect that while our report is seminal, it is likely the tip of the iceberg. If our data are any indication, the number of “under-the-radar” arbovirus infections of mosquito populations in migration hotspots across the state remains significantly underestimated.</p></div><div class="section" id="sec-17"><h2 class="">Funding</h2><p id="p-37">This research was supported in part by the United States Centers for Disease Control (CDC) Grant 1U01CK000510-03: Southeastern Regional Center of Excellence in Vector-Borne Diseases: The Gateway Program. The CDC had no role in the design of the study, the collection, analysis, and interpretation of data, or in writing the manuscript. Support was also provided by the University of Florida Emerging Pathogens Institute, the University of Florida Preeminence Initiative, and United States Department of Agriculture, Agricultural research Service, project 6066-21310-005-00-D.</p></div><div class="section" id="sec-18"><h2 class="">Data Availability</h2><p id="p-39">Viral RNASeq read data is available in the NCBI Sequence Read Archive and Biosample archive under BioProject PRJNA547758. Genome sequence data for the Manatee sequence is available in NCBI’s GenBank database (<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN192436&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom">MN192436</a>)  and reference sequences are available in the GenBank database with accession numbers described in the text. Multiple coding DENV4 sequence alignments from the dN/dS analyses and alignments for the RNA secondary structure model in Figure S4 are available with relevant accession numbers in a Mendeley dataset (<a href="https://data.mendeley.com/datasets/kwszjp63rb/draft?a=e11f9b80-bcfb-443b-918d-3016032ef3bd">https://data.mendeley.com/datasets/kwszjp63rb/draft?a=e11f9b80-bcfb-443b-918d-3016032ef3bd</a>). The accession numbers in order from top to bottom for the compared sequences in <a id="xref-fig-4-10" class="xref-fig" href="#F4">Fig. 4c</a> and <a id="xref-fig-4-11" class="xref-fig" href="#F4">4d</a> are MN192436, AH011951.2, JF262782.1, KP140942.1, KT276273.1, MK514144.1, and MF004387.1 (excluding AH011951.2 for <a id="xref-fig-4-12" class="xref-fig" href="#F4">Fig. 4d</a>).</p></div><div class="section" id="sec-19"><h2 class="">Conflicts of Interest</h2><p id="p-40">The authors declare that there are no competing interests.</p></div><div class="section" id="sec-20"><h2 class="">Funding Statement</h2><p id="p-41">This research was supported in part by the CDC (<a href="https://www.cdc.gov/">https://www.cdc.gov/</a>) Grant 1U01CK000510-03: Southeastern Regional Center of Excellence in Vector-Borne Diseases: The Gateway Program. The CDC had no role in the design of the study, the collection, analysis, and interpretation of data, or in writing the manuscript. Support was also provided by the University of Florida Emerging Pathogens Institute and the University of Florida Preeminence Initiative to RRD for this study. Mention of trade names or commercial products in this report is solely for the purpose of providing specific information and does not imply recommendation or endorsement by the U.S. Department of Agriculture.</p><p id="p-42">Correspondence and requests for reprints should be sent to Dr. Rhoel R. Dinglasan. His e-mail address is <span class="em-link"><span class="em-addr">rdinglasan{at}epi.ufl.edu</span></span>. His FAX number is 352-392-9704 and his telephone number is 352-294-8448. His professional address is 2055 Mowry Road, 32611, Gainesville, FL.</p></div><div id="sec-21" class="subsection"><p id="p-43"><strong>Figure S1. RNASeq read-proportion analyses.</strong> To analyze the read abundance of the RNASeq assay conducted on the mosquito samples from Manatee county, the proportion of DENV4-mapped reads to the total number of reads mapped for each site and year pool was calculated. The number of reads that mapped specifically to DENV4 was divided by the total number of reads mapped, the resultant values being shown above the bars on the graph. Proportion values are on the y-axis of the graph.</p><p id="p-44"><strong>Figure S2 Assessment of phylogenetic quality for DENV4 strains. (a,c)</strong> Phylogenetic signal, nucleotide substitution saturation and phylogenetic relationship in HIV envelope sequences from six patients obtained after ATI. Evaluation of the presence of phylogenetic signal satisfying resolved phylogenetic relationships among sequences was assessed by likelihood mapping (IQ-TREE: <a href="http://www.iqtree.org/">http://www.iqtree.org/</a>), which estimates the likelihood of each of the three possible tree topologies for each group of four sequences (quartet) in the data set using the best-fit nucleotide substitution model chosen according to Bayesian Information Criterion (BIC). Quartets are considered “resolved” when the three likelihood are significantly different (phylogenetic signal), unresolved or partially resolved, when all three likelihood values or two of them are not significantly different (phylogenetic noise). Percentage within each triangle, indicate the proportion of resolved quartets (in the three corner areas), as well as the proportion of partially resolved (side areas) or unresolved (center) quartets. Extensive simulation studies have shown that side/center areas including &lt;40% of the unresolved quartets can be considered robust in terms of phylogenetic signal [<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>]. <strong>(b,d)</strong> Substitution saturation, which decreases the phylogenetic information contained in the sequences, was assessed using DAMBE7 (<a href="http://dambe.bio.uottawa.ca/DAMBE/">http://dambe.bio.uottawa.ca/DAMBE/</a>) by plotting pairwise nucleotide (blue) transition (s) and (green) transversion (v) substitutions (y-axis) versus pairwise genetic distance (x-axis) determined with the Tamura and Nei 1993 (TN93) nucleotide substitution model [<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">3</a>].</p><p id="p-45"><strong>Figure S3. Assessment of temporal signal for DENV4 strains.</strong> The plot represents regression analysis of root-to-tip genetic distance assessed using TempEst v1.5. The positive slope (R2=0.7135) indicates presence of temporal signal for the dataset.</p><p id="p-46"><strong>Figure S4. DENV4 3’ UTR Analyses. (a)</strong> Alignment with the 3’-UTR of Manatee DENV4. The 3’-UTR RNA sequence alignment is of DENV4 from Manatee County, DENV4 Haiti 2014 #2 (KT276273.1), and DENV4 Philippines H241 (KR011349.2) genomes. The DENV4 RNA sequence alignment was generated with CLC Sequence Viewer 8.0 (<a href="https://www.qiagenbioinformatics.com/products/clc-main-w">https://www.qiagenbioinformatics.com/products/clc-main-w</a>). Some DENV4 conserved 3’ UTR regions are designated in black boxes in the figure, including repeated conserved sequence 2 (RCS2), conserved sequence 1 (CS1), conserved sequence 2 (CS2), and 3’ upstream AUG region (3’ UAR). Different nucleotides are designated with different colors. <strong>(b)</strong> A diagram of the secondary structure of the DENV4 Manatee County 3’ UTR is depicted with key nucleotides and mutations highlighted and drawn in orange, correlating to nodes A or B from <a id="xref-fig-3-13" class="xref-fig" href="#F3">fig. 3B</a>. Key secondary structure regions of the 3’ UTR are shown in black text: dumbbells 1-2 (DB1 &amp; DB2), pseudoknots 1-5 (PK1-5), flavivirus nuclease-resistant RNA (fNR2), and the 3’ stem loop (3’ SL).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-38">We gratefully acknowledge the support of Carina Blackmore, Danielle Stanek, and Andrea Morrison from the Florida Department of Health, as well as Lisa Conti, Kelly Friend, Davis Daiker, and Adriane Rogers from the Florida Department of Agriculture and Consumer Services for their institutional collaboration with the CDC Southeastern Center of Excellence in Vector Borne Diseases: The Gateway Program. We also thank Heather Coatsworth and Kaci McCoy for useful comments. This research was supported in part by the United States Centers for Disease Control (CDC) Grant 1U01CK000510-03: Southeastern Regional Center of Excellence in Vector-Borne Diseases: The Gateway Program. The CDC had no role in the design of the study, the collection, analysis, and interpretation of data, or in writing the manuscript. Support was also provided by the University of Florida Emerging Pathogens Institute, the University of Florida Preeminence Initiative, and United States Department of Agriculture, Agricultural research Service, project 6066-21310-005-00-D.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-3"><a class="rev-xref" href="#xref-fn-3-1">↵</a><span class="fn-label">***</span> Adam R. Rivers and Rhoel R. Dinglasan are co-senior authors.</p></li><li class="fn" id="fn-4"><p id="p-4"><strong>Disclaimers</strong>: Not applicable</p></li><li class="fn" id="fn-5"><p id="p-5"><strong>Sources of Funding:</strong> This research was supported in part by the United States Centers for Disease Control (CDC) Grant 1U01CK000510-03: Southeastern Regional Center of Excellence in Vector-Borne Diseases: The Gateway Program. The CDC had no role in the design of the study, the collection, analysis, and interpretation of data, or in writing the manuscript. Support was also provided by the University of Florida Emerging Pathogens Institute and the University of Florida Preeminence Initiative to RRD for this study.</p></li><li class="fn-conflict" id="fn-6"><p id="p-6"><strong>Conflicts of Interest:</strong> The authors declare no competing interests.</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-confproc" id="cit-2020.01.24.919282v1.1"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">World Health Organization</span>, <span class="cit-article-title">Special Programme for Research and Training in Tropical Diseases</span>. <span class="cit-conf-name">Report of the scientific working group meeting on dengue: Geneva, 1-5 October 2006</span>. <span class="cit-publ-loc">Geneva</span>: <span class="cit-publ-name">World Health Organization on behalf of the Special Programme for Research and Training in Tropical Diseases</span>, <span class="cit-pub-date">2007</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kraemer</span>  <span class="cit-name-given-names">MU</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sinka</span>  <span class="cit-name-given-names">ME</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duda</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The global compendium of <em>Aedes aegypti</em> and <em>Ae. albopictus</em> occurrence</span>. <abbr class="cit-jnl-abbrev">Sci Data</abbr> <span class="cit-pub-date">2015</span>; <span class="cit-vol">2</span>:<span class="cit-issue">150035</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rey</span>  <span class="cit-name-given-names">JR</span></span>. <span class="cit-article-title">Dengue in Florida (USA)</span>. <abbr class="cit-jnl-abbrev">Insects</abbr> <span class="cit-pub-date">2014</span>; <span class="cit-vol">5</span>:<span class="cit-fpage">991</span>–<span class="cit-lpage">1000</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInsects%26rft.volume%253D5%26rft.spage%253D991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.4" data-doi="10.1111/mve.12000"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reiskind</span>  <span class="cit-name-given-names">MH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lounibos</span>  <span class="cit-name-given-names">LP</span></span>. <span class="cit-article-title">Spatial and temporal patterns of abundance of <em>Aedes aegypti</em> L. (<em>Stegomyia aegypti</em>) and <em>Aedes albopictus</em> (Skuse) [<em>Stegomyia albopictus</em> (Skuse)] in southern Florida</span>. <abbr class="cit-jnl-abbrev">Med Vet Entomol</abbr> <span class="cit-pub-date">2013</span>; <span class="cit-vol">27</span>:<span class="cit-fpage">421</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMed%2BVet%2BEntomol%26rft.volume%253D27%26rft.spage%253D421%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmve.12000%26rft_id%253Dinfo%253Apmid%252F23278304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mve.12000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23278304&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Romo</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kenney</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blitvich</span>  <span class="cit-name-given-names">BJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brault</span>  <span class="cit-name-given-names">AC</span></span>. <span class="cit-article-title">Restriction of Zika virus infection and transmission in <em>Aedes aegypti</em> mediated by an insect-specific flavivirus</span>. <abbr class="cit-jnl-abbrev">Emerg Microbes Infect</abbr> <span class="cit-pub-date">2018</span>; <span class="cit-vol">7</span>:<span class="cit-issue">181</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.6" data-doi="10.1016/j.virol.2018.03.022"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frydman</span>  <span class="cit-name-given-names">HM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Connor</span>  <span class="cit-name-given-names">JH</span></span>. <span class="cit-article-title">Dual Insect specific virus infection limits Arbovirus replication in <em>Aedes</em> mosquito cells</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-pub-date">2018</span>; <span class="cit-vol">518</span>:<span class="cit-fpage">406</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.volume%253D518%26rft.spage%253D406%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2018.03.022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2018.03.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baidaliuk</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miot</span>  <span class="cit-name-given-names">EF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lequime</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Cell-Fusing Agent Virus Reduces Arbovirus Dissemination in <em>Aedes aegypti</em> Mosquitoes</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-pub-date">2019</span>; <span class="cit-vol">93</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.8" data-doi="10.1101/gr.213959.116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nurk</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meleshko</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korobeynikov</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pevzner</span>  <span class="cit-name-given-names">PA</span></span>. <span class="cit-article-title">metaSPAdes: a new versatile metagenomic assembler</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-pub-date">2017</span>; <span class="cit-vol">27</span>:<span class="cit-fpage">824</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.213959.116%26rft_id%253Dinfo%253Apmid%252F28298430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvODI0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMjUvMjAyMC4wMS4yNC45MTkyODIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.9" data-doi="10.1038/nmeth.3176"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Buchfink</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huson</span>  <span class="cit-name-given-names">DH</span></span>. <span class="cit-article-title">Fast and sensitive protein alignment using DIAMOND</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr> <span class="cit-pub-date">2015</span>; <span class="cit-vol">12</span>:<span class="cit-fpage">59</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D12%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3176%26rft_id%253Dinfo%253Apmid%252F25402007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3176&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25402007&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.10" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">SL</span></span>. <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr> <span class="cit-pub-date">2012</span>; <span class="cit-vol">9</span>:<span class="cit-fpage">357</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.11" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2009</span>; <span class="cit-vol">25</span>:<span class="cit-fpage">2078</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.12" data-doi="10.1101/gr.5969107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huson</span>  <span class="cit-name-given-names">DH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Auch</span>  <span class="cit-name-given-names">AF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schuster</span>  <span class="cit-name-given-names">SC</span></span>. <span class="cit-article-title">MEGAN analysis of metagenomic data</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-pub-date">2007</span>; <span class="cit-vol">17</span>:<span class="cit-fpage">377</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5969107%26rft_id%253Dinfo%253Apmid%252F17255551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzMvMzc3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMjUvMjAyMC4wMS4yNC45MTkyODIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van den Boogaart</span>  <span class="cit-name-given-names">GK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tolosana-Delgado</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bren</span>  <span class="cit-name-given-names">M</span></span>. (<span class="cit-pub-date">2018</span>)<span class="cit-article-title">. compositions: Compositional Data Analysis</span>. <abbr class="cit-jnl-abbrev">R package version</abbr> <span class="cit-vol">1</span>.<span class="cit-fpage">40</span>–<span class="cit-lpage">2</span>. <a href="https://CRAN.R-project.org/package=compositions">https://CRAN.R-project.org/package=compositions</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2Bpackage%2Bversion%26rft.volume%253D1%26rft.spage%253D40%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fernandes</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Macklaim</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Linn</span>  <span class="cit-name-given-names">TG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gloor</span>  <span class="cit-name-given-names">GB</span></span>. <span class="cit-article-title">ANOVA-like differential expression (ALDEx) analysis for mixed population RNA-Seq</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-pub-date">2013</span>; <span class="cit-vol">8</span>:<span class="cit-issue">e67019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.15" data-doi="10.1186/2049-2618-2-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fernandes</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">JN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Macklaim</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">McMurrough</span>  <span class="cit-name-given-names">TA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edgell</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gloor</span>  <span class="cit-name-given-names">GB</span></span>. <span class="cit-article-title">Unifying the analysis of high-throughput sequencing datasets: characterizing RNA-seq, 16S rRNA gene sequencing and selective growth experiments by compositional data analysis</span>. <abbr class="cit-jnl-abbrev">Microbiome</abbr> <span class="cit-pub-date">2014</span>; <span class="cit-vol">2</span>:<span class="cit-fpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiome%26rft.volume%253D2%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2049-2618-2-15%26rft_id%253Dinfo%253Apmid%252F24910773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2049-2618-2-15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24910773&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tcherepanov</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ehlers</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Upton</span>  <span class="cit-name-given-names">C</span></span>. <span class="cit-article-title">Genome Annotation Transfer Utility (GATU): rapid annotation of viral genomes using a closely related reference genome</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-pub-date">2006</span>; <span class="cit-vol">7</span>:<span class="cit-issue">150</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.17" data-doi="10.3390/v4113209"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pickett</span>  <span class="cit-name-given-names">BE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greer</span>  <span class="cit-name-given-names">DS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Virus pathogen database and analysis resource (ViPR): a comprehensive bioinformatics database and analysis resource for the coronavirus research community</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-pub-date">2012</span>; <span class="cit-vol">4</span>:<span class="cit-fpage">3209</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DPickett%26rft.auinit1%253DB.%2BE.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D3209%26rft.epage%253D3226%26rft.atitle%253DVirus%2BPathogen%2BDatabase%2Band%2BAnalysis%2BResource%2B%2528ViPR%2529%253A%2B%2BA%2BComprehensive%2BBioinformatics%2BDatabase%2Band%2BAnalysis%2BResource%2Bfor%2Bthe%2BCoronavirus%2BResearch%2BCommunity.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv4113209%26rft_id%253Dinfo%253Apmid%252F23202522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v4113209&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23202522&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.18" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Standley</span>  <span class="cit-name-given-names">DM</span></span>. <span class="cit-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-pub-date">2013</span>; <span class="cit-vol">30</span>:<span class="cit-fpage">772</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23329690&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317002300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.19" data-doi="10.1093/bioinformatics/btw108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Standley</span>  <span class="cit-name-given-names">DM</span></span>. <span class="cit-article-title">A simple method to control over-alignment in the MAFFT multiple sequence alignment program</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2016</span>; <span class="cit-vol">32</span>:<span class="cit-fpage">1933</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw108%26rft_id%253Dinfo%253Apmid%252F27153688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27153688&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.20" data-doi="10.1093/molbev/msu300"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">LT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">HA</span></span>, <span class="cit-auth"><span class="cit-name-surname">von Haeseler</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Minh</span>  <span class="cit-name-given-names">BQ</span></span>. <span class="cit-article-title">IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-pub-date">2015</span>; <span class="cit-vol">32</span>:<span class="cit-fpage">268</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu300%26rft_id%253Dinfo%253Apmid%252F25371430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25371430&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.21" data-doi="10.1093/bioinformatics/18.3.502"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">HA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Strimmer</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vingron</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">von Haeseler</span>  <span class="cit-name-given-names">A</span></span>. <span class="cit-article-title">TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel computing</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2002</span>; <span class="cit-vol">18</span>:<span class="cit-fpage">502</span>–<span class="cit-lpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DSchmidt%26rft.auinit1%253DH.%2BA.%26rft.volume%253D18%26rft.issue%253D3%26rft.spage%253D502%26rft.epage%253D504%26rft.atitle%253DTREE-PUZZLE%253A%2Bmaximum%2Blikelihood%2Bphylogenetic%2Banalysis%2Busing%2B%2Bquartets%2Band%2Bparallel%2Bcomputing%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F18.3.502%26rft_id%253Dinfo%253Apmid%252F11934758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/18.3.502&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11934758&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174708500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.22" data-doi="10.1093/nar/gkw256"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Trifinopoulos</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">LT</span></span>, <span class="cit-auth"><span class="cit-name-surname">von Haeseler</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Minh</span>  <span class="cit-name-given-names">BQ</span></span>. <span class="cit-article-title">W-IQ-TREE: a fast online phylogenetic tool for maximum likelihood analysis</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-pub-date">2016</span>; <span class="cit-vol">44</span>:<span class="cit-fpage">W232</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw256%26rft_id%253Dinfo%253Apmid%252F27084950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27084950&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.23" data-doi="10.1093/molbev/mst024"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Minh</span>  <span class="cit-name-given-names">BQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">von Haeseler</span>  <span class="cit-name-given-names">A</span></span>. <span class="cit-article-title">Ultrafast approximation for phylogenetic bootstrap</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-pub-date">2013</span>; <span class="cit-vol">30</span>:<span class="cit-fpage">1188</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst024%26rft_id%253Dinfo%253Apmid%252F23418397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23418397&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318165700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">DP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murrell</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Golden</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khoosal</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muhire</span>  <span class="cit-name-given-names">B</span></span>. <span class="cit-article-title">RDP4: Detection and analysis of recombination patterns in virus genomes</span>. <abbr class="cit-jnl-abbrev">Virus Evol</abbr> <span class="cit-pub-date">2015</span>; <span class="cit-vol">1</span>:vev003.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">TT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Max Carvalho</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pybus</span>  <span class="cit-name-given-names">OG</span></span>. <span class="cit-article-title">Exploring the temporal structure of heterochronous sequences using TempEst (formerly Path-O-Gen)</span>. <abbr class="cit-jnl-abbrev">Virus Evol</abbr> <span class="cit-pub-date">2016</span>; <span class="cit-vol">2</span>:vew007.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.26" data-doi="10.1093/molbev/mss075"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>. <span class="cit-article-title">Bayesian phylogenetics with BEAUti and the BEAST 1.7</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-pub-date">2012</span>; <span class="cit-vol">29</span>:<span class="cit-fpage">1969</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss075%26rft_id%253Dinfo%253Apmid%252F22367748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22367748&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307171300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.27" data-doi="10.1186/1471-2148-7-214"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>. <span class="cit-article-title">BEAST: Bayesian evolutionary analysis by sampling trees</span>. <abbr class="cit-jnl-abbrev">BMC Evol Biol</abbr> <span class="cit-pub-date">2007</span>; <span class="cit-vol">7</span>:<span class="cit-fpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DDrummond%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D7%26rft.spage%253D214%26rft.epage%253D214%26rft.atitle%253DBEAST%253A%2BBayesian%2Bevolutionary%2Banalysis%2Bby%2Bsampling%2Btrees.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-7-214%26rft_id%253Dinfo%253Apmid%252F17996036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-7-214&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17996036&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.28" data-doi="10.1007/BF02101694"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hasegawa</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kishino</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">T</span></span>. <span class="cit-article-title">Dating of the human-ape splitting by a molecular clock of mitochondrial DNA</span>. <abbr class="cit-jnl-abbrev">J Mol Evol</abbr> <span class="cit-pub-date">1985</span>; <span class="cit-vol">22</span>:<span class="cit-fpage">160</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DHasegawa%26rft.auinit1%253DM.%26rft.volume%253D22%26rft.issue%253D2%26rft.spage%253D160%26rft.epage%253D174%26rft.atitle%253DDating%2Bof%2Bthe%2Bhuman-ape%2Bsplitting%2Bby%2Ba%2Bmolecular%2Bclock%2Bof%2Bmitochondrial%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02101694%26rft_id%253Dinfo%253Apmid%252F3934395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02101694&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3934395&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985AWB1600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woolhouse</span>  <span class="cit-name-given-names">ME</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>. <span class="cit-article-title">The effects of sampling strategy on the quality of reconstruction of viral population dynamics using Bayesian skyline family coalescent methods: A simulation study</span>. <abbr class="cit-jnl-abbrev">Virus Evol</abbr> <span class="cit-pub-date">2016</span>; <span class="cit-vol">2</span>:vew003.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.30" data-doi="10.1093/molbev/mss084"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baele</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lemey</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bedford</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alekseyenko</span>  <span class="cit-name-given-names">AV</span></span>. <span class="cit-article-title">Improving the accuracy of demographic and molecular clock model comparison while accommodating phylogenetic uncertainty</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-pub-date">2012</span>; <span class="cit-vol">29</span>:<span class="cit-fpage">2157</span>–<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss084%26rft_id%253Dinfo%253Apmid%252F22403239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22403239&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308851600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.31" data-doi="10.1093/sysbio/syq085"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">PO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">MH</span></span>. <span class="cit-article-title">Improving marginal likelihood estimation for Bayesian phylogenetic model selection</span>. <abbr class="cit-jnl-abbrev">Syst Biol</abbr> <span class="cit-pub-date">2011</span>; <span class="cit-vol">60</span>:<span class="cit-fpage">150</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyq085%26rft_id%253Dinfo%253Apmid%252F21187451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syq085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21187451&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287255100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.32" data-doi="10.1111/2041&amp;ndash;210X.12628"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">DK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-article-title">Lam TT. ggtree: an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data</span>. <abbr class="cit-jnl-abbrev">Methods Ecol Evol</abbr> <span class="cit-pub-date">2017</span>; <span class="cit-vol">8</span>:<span class="cit-fpage">28</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol%2BEvol%26rft.volume%253D8%26rft.spage%253D28%26rft_id%253Dinfo%253Adoi%252F10.1111%252F2041%2526ndash%253B210X.12628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/2041&amp;ndash;210X.12628&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.33" data-doi="10.1093/bioinformatics/bti079"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pond</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frost</span>  <span class="cit-name-given-names">SD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muse</span>  <span class="cit-name-given-names">SV</span></span>. <span class="cit-article-title">HyPhy: hypothesis testing using phylogenies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2005</span>; <span class="cit-vol">21</span>:<span class="cit-fpage">676</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti079%26rft_id%253Dinfo%253Apmid%252F15509596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15509596&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227241200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.34" data-doi="10.1093/bioinformatics/bti320"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pond</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frost</span>  <span class="cit-name-given-names">SD</span></span>. <span class="cit-article-title">Datamonkey: rapid detection of selective pressure on individual sites of codon alignments</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2005</span>; <span class="cit-vol">21</span>:<span class="cit-fpage">2531</span>–<span class="cit-lpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti320%26rft_id%253Dinfo%253Apmid%252F15713735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti320&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15713735&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229285600054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.35" data-doi="10.1093/molbev/mst030"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Murrell</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moola</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mabona</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">FUBAR: a fast, unconstrained bayesian approximation for inferring selection</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-pub-date">2013</span>; <span class="cit-vol">30</span>:<span class="cit-fpage">1196</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst030%26rft_id%253Dinfo%253Apmid%252F23420840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23420840&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318165700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.36" data-doi="10.1371/journal.pgen.1002764"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Murrell</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wertheim</span>  <span class="cit-name-given-names">JO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moola</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weighill</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scheffler</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kosakovsky Pond</span>  <span class="cit-name-given-names">SL</span></span>. <span class="cit-article-title">Detecting individual sites subject to episodic diversifying selection</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-pub-date">2012</span>; <span class="cit-vol">8</span>:<span class="cit-fpage">e1002764</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMurrell%26rft.auinit1%253DB.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253De1002764%26rft.epage%253De1002764%26rft.atitle%253DDetecting%2Bindividual%2Bsites%2Bsubject%2Bto%2Bepisodic%2Bdiversifying%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002764%26rft_id%253Dinfo%253Apmid%252F22807683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002764&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22807683&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.37" data-doi="10.1186/1471-2105-11-284"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Steinway</span>  <span class="cit-name-given-names">SN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dannenfelser</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laucius</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nayak</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-article-title">JCoDA: a tool for detecting evolutionary selection</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-pub-date">2010</span>; <span class="cit-vol">11</span>:<span class="cit-fpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DSteinway%26rft.auinit1%253DS.%2BN.%26rft.volume%253D11%26rft.spage%253D284%26rft.epage%253D284%26rft.atitle%253DJCoDA%253A%2Ba%2Btool%2Bfor%2Bdetecting%2Bevolutionary%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-284%26rft_id%253Dinfo%253Apmid%252F20507581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-284&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20507581&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.38" data-doi="10.1093/molbev/msi105"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Kosakovsky</span> <span class="cit-name-surname">Pond SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frost</span>  <span class="cit-name-given-names">SD</span></span>. <span class="cit-article-title">Not so different after all: a comparison of methods for detecting amino acid sites under selection</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-pub-date">2005</span>; <span class="cit-vol">22</span>:<span class="cit-fpage">1208</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi105%26rft_id%253Dinfo%253Apmid%252F15703242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msi105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15703242&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228700200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weaver</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shank</span>  <span class="cit-name-given-names">SD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spielman</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muse</span>  <span class="cit-name-given-names">SV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kosakovsky Pond</span>  <span class="cit-name-given-names">SL</span></span>. <span class="cit-article-title">Datamonkey 2.0: a modern web application for characterizing selective and other evolutionary processes</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.40" data-doi="10.1093/molbev/msg182"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">SN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chirivella</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Selection-driven evolution of emergent dengue virus</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-pub-date">2003</span>; <span class="cit-vol">20</span>:<span class="cit-fpage">1650</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg182%26rft_id%253Dinfo%253Apmid%252F12832629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msg182&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12832629&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185810900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.41" data-doi="10.3390/v3091739"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gebhard</span>  <span class="cit-name-given-names">LG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Filomatori</span>  <span class="cit-name-given-names">CV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gamarnik</span>  <span class="cit-name-given-names">AV</span></span>. <span class="cit-article-title">Functional RNA elements in the dengue virus genome</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-pub-date">2011</span>; <span class="cit-vol">3</span>:<span class="cit-fpage">1739</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DGebhard%26rft.auinit1%253DL.%2BG.%26rft.volume%253D3%26rft.issue%253D9%26rft.spage%253D1739%26rft.epage%253D1756%26rft.atitle%253DFunctional%2BRNA%2Belements%2Bin%2Bthe%2Bdengue%2Bvirus%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv3091739%26rft_id%253Dinfo%253Apmid%252F21994804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v3091739&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21994804&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295211400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Finol</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ooi</span>  <span class="cit-name-given-names">EE</span></span>. <span class="cit-article-title">Evolution of Subgenomic RNA Shapes Dengue Virus Adaptation and Epidemiological Fitness</span>. <abbr class="cit-jnl-abbrev">iScience</abbr> <span class="cit-pub-date">2019</span>; <span class="cit-vol">16</span>:<span class="cit-fpage">94</span>–<span class="cit-lpage">105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DiScience%26rft.volume%253D16%26rft.spage%253D94%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Parry</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Asgari</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-article-title"><em>Aedes Anphevirus</em>: an Insect-Specific Virus Distributed Worldwide in <em>Aedes aegypti</em> Mosquitoes That Has Complex Interplays with Wolbachia and Dengue Virus Infection in Cells</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-pub-date">2018</span>; <span class="cit-vol">92</span>.<span class="cit-issue">45</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.44" data-doi="10.1038/s41598-017-07279-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Asad</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khromykh</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Asgari</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-article-title">Cell fusing agent virus and dengue virus mutually interact in <em>Aedes aegypti</em> cell lines</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-pub-date">2017</span>; <span class="cit-vol">7</span>:<span class="cit-fpage">6935</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D7%26rft.spage%253D6935%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-07279-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-017-07279-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ten Bosch</span>  <span class="cit-name-given-names">QA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clapham</span>  <span class="cit-name-given-names">HE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lambrechts</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Contributions from the silent majority dominate dengue virus transmission</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-pub-date">2018</span>; <span class="cit-vol">14</span>:<span class="cit-issue">e1006965</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.46" data-doi="10.1086/315215"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vaughn</span>  <span class="cit-name-given-names">DW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalayanarooj</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Dengue viremia titer, antibody response pattern, and virus serotype correlate with disease severity</span>. <abbr class="cit-jnl-abbrev">J Infect Dis</abbr> <span class="cit-pub-date">2000</span>; <span class="cit-vol">181</span>:<span class="cit-fpage">2</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BInfect%2BDis%26rft_id%253Dinfo%253Adoi%252F10.1086%252F315215%26rft_id%253Dinfo%253Apmid%252F10608744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/315215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10608744&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085229300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.47" data-doi="10.1603/ME13047"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Buckner</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alto</span>  <span class="cit-name-given-names">BW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lounibos</span>  <span class="cit-name-given-names">LP</span></span>. <span class="cit-article-title">Vertical transmission of Key West dengue-1 virus by <em>Aedes aegypti</em> and <em>Aedes albopictus</em> (Diptera: Culicidae) mosquitoes from Florida</span>. <abbr class="cit-jnl-abbrev">J Med Entomol</abbr> <span class="cit-pub-date">2013</span>; <span class="cit-vol">50</span>:<span class="cit-fpage">1291</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BMed%2BEntomol%26rft_id%253Dinfo%253Adoi%252F10.1603%252FME13047%26rft_id%253Dinfo%253Apmid%252F24843934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1603/ME13047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24843934&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F25%2F2020.01.24.919282.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mavian</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dulcey</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munoz</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salemi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vittor</span>  <span class="cit-name-given-names">AY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Capua</span>  <span class="cit-name-given-names">I</span></span>. <span class="cit-article-title">Islands as Hotspots for Emerging Mosquito-Borne Viruses: A One-Health Perspective</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-pub-date">2018</span>; <span class="cit-vol">11</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">49.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.24.919282v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blohm</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A</span> <span class="cit-name-surname">Elbadry M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mavian</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Mayaro as a Caribbean traveler: Evidence for multiple introductions and transmission of the virus into Haiti</span>. <abbr class="cit-jnl-abbrev">Int J Infect Dis</abbr> <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.24.919282v1.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mavian</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salemi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A new “American” subgroup of African-lineage Chikungunya virus detected in and isolated from mosquitoes collected in Haiti</span>, <span class="cit-pub-date">2016</span>. <abbr class="cit-jnl-abbrev">PLoS One 2018</abbr>; <span class="cit-vol">13</span>:<span class="cit-fpage">e0196857</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%2B2018%26rft.volume%253D13%26rft.spage%253De0196857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.01.24.919282v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.01.24.919282v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.01.24.917708v1" title="TNFα/Grnd mediate JNK/MMPs activation during glioma progression and neurodegeneration" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.01.24.918433v1" title="Structural insights into the insecticidal Vip3A toxin of Bacillus thuringiensis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 25, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.01.24.919282v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.01.24.919282v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.01.24.919282v1" class="active" data-icon-position="" data-hide-link-title="0">Under-the-radar dengue virus infections in natural populations of Aedes aegypti mosquitoes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1121132" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ks-bDnDH9tqLRizMOUlMUwNozxM54dLNq9UUcMKHRTo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1121132" id="node1121132--2" data-pisa="biorxiv;2020.01.24.919282v1" data-pisa-master="biorxiv;2020.01.24.919282" data-seqnum="1121132" data-apath="/biorxiv/early/2020/01/25/2020.01.24.919282.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Under-the-radar dengue virus infections in natural populations of <em>Aedes aegypti</em> mosquitoes</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Sean M.</span> <span class="nlm-surname">Boyles</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Carla N.</span> <span class="nlm-surname">Mavian</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Esteban</span> <span class="nlm-surname">Finol</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Ukhanova</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Caroline J.</span> <span class="nlm-surname">Stephenson</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Gabriela</span> <span class="nlm-surname">Hamerlinck</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Seokyoung</span> <span class="nlm-surname">Kang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Caleb</span> <span class="nlm-surname">Baumgartner</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Mary</span> <span class="nlm-surname">Geesey</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Israel</span> <span class="nlm-surname">Stinton</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Kate</span> <span class="nlm-surname">Williams</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Derrick K.</span> <span class="nlm-surname">Mathias</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Mattia</span> <span class="nlm-surname">Prosperi</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Volker</span> <span class="nlm-surname">Mai</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Marco</span> <span class="nlm-surname">Salemi</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Eva A.</span> <span class="nlm-surname">Buckner</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">John A.</span> <span class="nlm-surname">Lednicky</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Adam R.</span> <span class="nlm-surname">Rivers</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Rhoel R.</span> <span class="nlm-surname">Dinglasan</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.24.919282; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.24.919282 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.01.24.919282v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.01.24.919282v1&amp;title=Under-the-radar%20dengue%20virus%20infections%20in%20natural%20populations%20of%20Aedes%20aegypti%20mosquitoes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.24.919282v1&amp;title=Under-the-radar%20dengue%20virus%20infections%20in%20natural%20populations%20of%20Aedes%20aegypti%20mosquitoes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.24.919282v1&amp;text=Under-the-radar%20dengue%20virus%20infections%20in%20natural%20populations%20of%20Aedes%20aegypti%20mosquitoes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.24.919282v1&amp;title=Under-the-radar%20dengue%20virus%20infections%20in%20natural%20populations%20of%20Aedes%20aegypti%20mosquitoes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.01.24.919282v1&amp;t=Under-the-radar%20dengue%20virus%20infections%20in%20natural%20populations%20of%20Aedes%20aegypti%20mosquitoes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.01.24.919282v1&amp;title=Under-the-radar%20dengue%20virus%20infections%20in%20natural%20populations%20of%20Aedes%20aegypti%20mosquitoes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.24.919282v1&amp;title=Under-the-radar%20dengue%20virus%20infections%20in%20natural%20populations%20of%20Aedes%20aegypti%20mosquitoes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1121132" id="node1121132--2" data-pisa="biorxiv;2020.01.24.919282v1" data-pisa-master="biorxiv;2020.01.24.919282" data-seqnum="1121132" data-apath="/biorxiv/early/2020/01/25/2020.01.24.919282.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Under-the-radar dengue virus infections in natural populations of <em>Aedes aegypti</em> mosquitoes</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Sean M.</span> <span class="nlm-surname">Boyles</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Carla N.</span> <span class="nlm-surname">Mavian</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Esteban</span> <span class="nlm-surname">Finol</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Ukhanova</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Caroline J.</span> <span class="nlm-surname">Stephenson</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Gabriela</span> <span class="nlm-surname">Hamerlinck</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Seokyoung</span> <span class="nlm-surname">Kang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Caleb</span> <span class="nlm-surname">Baumgartner</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Mary</span> <span class="nlm-surname">Geesey</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Israel</span> <span class="nlm-surname">Stinton</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Kate</span> <span class="nlm-surname">Williams</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Derrick K.</span> <span class="nlm-surname">Mathias</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Mattia</span> <span class="nlm-surname">Prosperi</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Volker</span> <span class="nlm-surname">Mai</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Marco</span> <span class="nlm-surname">Salemi</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Eva A.</span> <span class="nlm-surname">Buckner</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">John A.</span> <span class="nlm-surname">Lednicky</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Adam R.</span> <span class="nlm-surname">Rivers</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Rhoel R.</span> <span class="nlm-surname">Dinglasan</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.24.919282; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.24.919282 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Under-the-radar%20dengue%20virus%20infections%20in%20natural%20populations%20of%20Aedes%20aegypti%20mosquitoes&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Boyles%2C+Sean+M.&amp;rft.au=Mavian%2C+Carla+N.&amp;rft.au=Finol%2C+Esteban&amp;rft.au=Ukhanova%2C+Maria&amp;rft.au=Stephenson%2C+Caroline+J.&amp;rft.au=Hamerlinck%2C+Gabriela&amp;rft.au=Kang%2C+Seokyoung&amp;rft.au=Baumgartner%2C+Caleb&amp;rft.au=Geesey%2C+Mary&amp;rft.au=Stinton%2C+Israel&amp;rft.au=Williams%2C+Kate&amp;rft.au=Mathias%2C+Derrick+K.&amp;rft.au=Prosperi%2C+Mattia&amp;rft.au=Mai%2C+Volker&amp;rft.au=Salemi%2C+Marco&amp;rft.au=Buckner%2C+Eva+A.&amp;rft.au=Lednicky%2C+John+A.&amp;rft.au=Rivers%2C+Adam+R.&amp;rft.au=Dinglasan%2C+Rhoel+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1121132/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1121132/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1121132/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1121132/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1121132/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1121132/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1121132/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1121132/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1121132/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1121132/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1121132/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1121132/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.24.919282v1&amp;count=horizontal&amp;via=&amp;text=Under-the-radar%20dengue%20virus%20infections%20in%20natural%20populations%20of%20Aedes%20aegypti%20mosquitoes&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.01.24.919282v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.01.24.919282v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.01.24.919282v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/ecology" class="highlight" data-icon-position="" data-hide-link-title="0">Ecology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
